<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/447" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/447.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/447.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/447.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/447.atom" />
<meta name="HW.pisa" content="ggg;4/3/447" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" />
<meta name="DC.Identifier" content="10.1534/g3.113.009316" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Brieuc et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.114.012021" />
<meta name="DC.Description" content="Comparisons between the genomes of salmon species reveal that they underwent extensive chromosomal rearrangements following whole genome duplication that occurred in their lineage 58−63 million years ago. Extant salmonids are diploid, but occasional pairing between homeologous chromosomes exists in males. The consequences of re-diploidization can be characterized by mapping the position of duplicated loci in such species. Linkage maps are also a valuable tool for genome-wide applications such as genome-wide association studies, quantitative trait loci mapping or genome scans. Here, we investigated chromosomal evolution in Chinook salmon ( Oncorhynchus tshawytscha ) after genome duplication by mapping 7146 restriction-site associated DNA loci in gynogenetic haploid, gynogenetic diploid, and diploid crosses. In the process, we developed a reference database of restriction-site associated DNA loci for Chinook salmon comprising 48528 non-duplicated loci and 6409 known duplicated loci, which will facilitate locus identification and data sharing. We created a very dense linkage map anchored to all 34 chromosomes for the species, and all arms were identified through centromere mapping. The map positions of 799 duplicated loci revealed that homeologous pairs have diverged at different rates following whole genome duplication, and that degree of differentiation along arms was variable. Many of the homeologous pairs with high numbers of duplicated markers appear conserved with other salmon species, suggesting that retention of conserved homeologous pairing in some arms preceded species divergence. As chromosome arms are highly conserved across species, the major resources developed for Chinook salmon in this study are also relevant for other related species." />
<meta name="DC.Contributor" content="Marine S. O. Brieuc" />
<meta name="DC.Contributor" content="Charles D. Waters" />
<meta name="DC.Contributor" content="James E. Seeb" />
<meta name="DC.Contributor" content="Kerry A. Naish" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Comparisons between the genomes of salmon species reveal that they underwent extensive chromosomal rearrangements following whole genome duplication that occurred in their lineage 58−63 million years ago. Extant salmonids are diploid, but occasional pairing between homeologous chromosomes exists in males. The consequences of re-diploidization can be characterized by mapping the position of duplicated loci in such species. Linkage maps are also a valuable tool for genome-wide applications such as genome-wide association studies, quantitative trait loci mapping or genome scans. Here, we investigated chromosomal evolution in Chinook salmon (&lt;i&gt;Oncorhynchus tshawytscha&lt;/i&gt;) after genome duplication by mapping 7146 restriction-site associated DNA loci in gynogenetic haploid, gynogenetic diploid, and diploid crosses. In the process, we developed a reference database of restriction-site associated DNA loci for Chinook salmon comprising 48528 non-duplicated loci and 6409 known duplicated loci, which will facilitate locus identification and data sharing. We created a very dense linkage map anchored to all 34 chromosomes for the species, and all arms were identified through centromere mapping. The map positions of 799 duplicated loci revealed that homeologous pairs have diverged at different rates following whole genome duplication, and that degree of differentiation along arms was variable. Many of the homeologous pairs with high numbers of duplicated markers appear conserved with other salmon species, suggesting that retention of conserved homeologous pairing in some arms preceded species divergence. As chromosome arms are highly conserved across species, the major resources developed for Chinook salmon in this study are also relevant for other related species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/447" />
<meta name="citation_id" content="4/3/447" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/447" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/447.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/447.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/447.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009316" />
<meta name="citation_pmid" content="24381192" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="447" />
<meta name="citation_lastpage" content="460" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marine S. O. Brieuc" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020" />
<meta name="citation_author_email" content="mbrieuc@uw.edu" />
<meta name="citation_author" content="Charles D. Waters" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020" />
<meta name="citation_author" content="James E. Seeb" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020" />
<meta name="citation_author" content="Kerry A. Naish" />
<meta name="citation_author_institution" content="School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources;citation_author=A. Abadia-Cardoso;citation_author=A. J. Clemento;citation_author=J. C. Garza;citation_title=Discovery and characterization of single-nucleotide polymorphisms in steelhead/rainbow trout, Oncorhynchus mykiss.;citation_pages=31-49;citation_volume=11;citation_year=2011;citation_pmid=21429161;citation_doi=10.1111/j.1755-0998.2010.02971.x" />
<meta name="citation_reference" content="citation_title=Tetraploidy and the Evolution of Salmonid Fishes in Evolutionary Genetics of Fishes;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. W. Allendorf;citation_author=R. G. Danzmann;citation_title=Secondary Tetrasomic Segregation of MDH-B and Preferential Pairing of Homeologues in Rainbow Trout;citation_pages=1083-1092;citation_volume=145;citation_year=1997;citation_issue=4;citation_pmid=9093860" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. W. Allendorf;citation_author=J. E. Seeb;citation_author=K. L. Knudsen;citation_author=G. H. Thorgaard;citation_author=R. F. Leary;citation_title=Gene-centromere mapping of 25 loci in rainbow trout;citation_pages=307-312;citation_volume=77;citation_year=1986;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=S. J. Amish;citation_author=P. A. Hohenlohe;citation_author=S. Painter;citation_author=R. F. Leary;citation_author=C. Muhlfeld;citation_title=RAD sequencing yields a high success rate for westslope cutthroat and rainbow trout species-diagnostic SNP assays.;citation_pages=653-660;citation_volume=12;citation_year=2012;citation_doi=10.1111/j.1755-0998.2012.03157.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Evol.Appl.;citation_author=I. R. Bradbury;citation_author=S. Hubert;citation_author=B. Higgins;citation_author=S. Bowman;citation_author=T. Borza;citation_title=Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.;citation_pages=450-461;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=N. R. Campbell;citation_author=S. R. Narum;citation_title=Identification of novel single-nucleotide polymorphisms in Chinook salmon and variation among life history types.;citation_pages=96-106;citation_volume=137;citation_year=2008;citation_doi=10.1577/T07-011.1" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=C. Canestroa;citation_author=R. Albalat;citation_author=M. Irimia;citation_author=J. Garcia-Fernandez;citation_title=Impact of gene gains, losses and duplication modes on the origin and diversification of vertebrates.;citation_pages=83-94;citation_volume=24;citation_year=2013;citation_pmid=23291262;citation_doi=10.1016/j.semcdb.2012.12.008" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Psychiatry;citation_author=S. Cichon;citation_author=N. Craddock;citation_author=M. Daly;citation_author=S. V. Faraone;citation_author=P. V. Gejman;citation_title=Genomewide association studies: history, rationale, and prospects for psychiatric disorders.;citation_volume=166;citation_year=2009;citation_pmid=19339359;citation_doi=10.1176/appi.ajp.2008.08091354" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=B. C. Y. Collard;citation_author=M. Z. Z. Jahufer;citation_author=J. B. Brouwer;citation_author=E. C. K. Pang;citation_title=An introduction to markers, quantitative trait loci (QTL) mapping and marker-assisted selection for crop improvement: the basic concepts.;citation_pages=169-196;citation_volume=142;citation_year=2005;citation_doi=10.1007/s10681-005-1681-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Crete-Lafreniere;citation_author=L. K. Weir;citation_author=L. Bernatchez;citation_title=Framing the salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.;citation_pages=e46662-e46662;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071608;citation_doi=10.1371/journal.pone.0046662" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=R. G. Danzmann;citation_title=LINKMFEX: linkage analysis software for diploid and tetraploid outcrossed mapping panels.;citation_pages=10;citation_volume=247;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=R. G. Danzmann;citation_author=M. Cairney;citation_author=W. S. Davidson;citation_author=M. M. Ferguson;citation_author=K. Gharbi;citation_title=A comparative analysis of the rainbow trout genome with 2 other species of fish (Arctic charr and Atlantic salmon) within the tetraploid derivative Salmonidae family (subfamily: Salmoninae).;citation_pages=1037-1051;citation_volume=48;citation_year=2005;citation_issue=6;citation_pmid=16391673;citation_doi=10.1139/g05-067" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. G. Danzmann;citation_author=E. A. Davidson;citation_author=M. M. Ferguson;citation_author=K. Gharbi;citation_author=B. F. Koop;citation_title=Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).;citation_pages=557-557;citation_volume=9;citation_year=2008;citation_pmid=19032764;citation_doi=10.1186/1471-2164-9-557" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=P. C. J. Donoghue;citation_author=M. A. Purnell;citation_title=Genome duplication, extinction and vertebrate evolution.;citation_pages=312-319;citation_volume=20;citation_year=2005;citation_issue=6;citation_pmid=16701387;citation_doi=10.1016/j.tree.2005.04.008" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. J. Doyle;citation_author=L. E. Flagel;citation_author=A. H. Paterson;citation_author=R. A. Rapp;citation_author=D. E. Soltis;citation_title=Evolutionary genetics of genome merger and doubling in plants.;citation_pages=443-461;citation_volume=42;citation_year=2008;citation_pmid=18983261;citation_doi=10.1146/annurev.genet.42.110807.091524" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. V. Everett;citation_author=M. R. Miller;citation_author=J. E. Seeb;citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.;citation_pages=521-521;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23031582;citation_doi=10.1186/1471-2164-13-521" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Gharbi;citation_author=A. Gautier;citation_author=R. G. Danzmann;citation_author=S. Gharbi;citation_author=T. Sakamoto;citation_title=A linkage map for brown trout (Salmo trutta): Chromosome homeologies and comparative genome organization with other salmonid fish.;citation_volume=172;citation_year=2006;citation_pmid=16452148;citation_doi=10.1534/genetics.105.048330" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Guyomard;citation_author=S. Mauger;citation_author=K. Tabet-Canale;citation_author=S. Martineau;citation_author=C. Genet;citation_title=A type I and type II microsatellite linkage map of rainbow trout (Oncorhynchus mykiss) with presumptive coverage of all chromosome arms.;citation_pages=302-302;citation_volume=7;citation_year=2006;citation_pmid=17137492;citation_doi=10.1186/1471-2164-7-302" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Guyomard;citation_author=M. Boussaha;citation_author=F. Krieg;citation_author=C. Hervet;citation_author=E. Quillet;citation_title=A synthetic rainbow trout linkage map provides new insights into the salmonid whole genome duplication and the conservation of synteny among teleosts.;citation_pages=15-15;citation_volume=13;citation_year=2012;citation_pmid=22424132;citation_doi=10.1186/1471-2156-13-15" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=B. C. Hecht;citation_author=F. P. Thrower;citation_author=M. C. Hale;citation_author=M. R. Miller;citation_author=K. M. Nichols;citation_title=Genetic Architecture of Migration-Related Traits in Rainbow and Steelhead Trout, Oncorhynchus mykiss;citation_pages=1113-1127;citation_volume=2;citation_year=2012;citation_issue=9;citation_pmid=22973549;citation_doi=10.1534/g3.112.003137" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. E. Hess;citation_author=A. P. Matala;citation_author=S. R. Narum;citation_title=Comparison of SNPs and microsatellites for fine-scale application of genetic stock identification of Chinook salmon in the Columbia River Basin.;citation_pages=137-149;citation_volume=11;citation_year=2011;citation_pmid=21429170;citation_doi=10.1111/j.1755-0998.2010.02958.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=P. D. Etter;citation_author=N. Stiffler;citation_author=E. A. Johnson;citation_title=Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.;citation_pages=e1000862-e1000862;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20195501;citation_doi=10.1371/journal.pgen.1000862" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. D. Houston;citation_author=J. W. Davey;citation_author=S. C. Bishop;citation_author=N. R. Lowe;citation_author=J. C. Mota-Velasco;citation_title=Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_pmid=22702806;citation_doi=10.1186/1471-2164-13-244" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=M. E. Hudson;citation_title=Sequencing breakthroughs for genomic ecology and evolutionary biology.;citation_pages=3-17;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=21585713;citation_doi=10.1111/j.1471-8286.2007.02019.x" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=A. L. Hufton;citation_author=G. Panopoulou;citation_title=Polyploidy and genome restructuring: a variety of outcomes.;citation_pages=600-606;citation_volume=19;citation_year=2009;citation_issue=6;citation_pmid=19900800;citation_doi=10.1016/j.gde.2009.10.005" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Jorgenson;citation_author=H. Tang;citation_author=M. Gadde;citation_author=M. Province;citation_author=M. Leppert;citation_title=Ethnicity and human genetic linkage maps.;citation_pages=276-290;citation_volume=76;citation_year=2005;citation_issue=2;citation_pmid=15627237;citation_doi=10.1086/427926" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=H. Komen;citation_author=G. H. Thorgaard;citation_title=Androgenesis, gynogenesis and the production of clones in fishes: a review.;citation_pages=150-173;citation_volume=269;citation_year=2007;citation_doi=10.1016/j.aquaculture.2007.05.009" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Canadian Journal of Fisheries and Aquatic Sciences, online early.;citation_title=SNPs identified through genotyping-by-sequencing improve genetic stock identification of Chinook salmon (Oncorhynchus tshawytscha) from western Alaska.;citation_author=W. A. Larson;citation_author=J. E. Seeb;citation_author=C. E. Pascal;citation_author=W. D. Templin;citation_author=L. W. Seeb;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=K. R. Lindner;citation_author=J. E. Seeb;citation_author=C. Habicht;citation_author=K. L. Knudsen;citation_author=E. Kretschmer;citation_title=Gene-centromere mapping of 312 loci in pink salmon by half-tetrad analysis.;citation_pages=538-549;citation_volume=43;citation_year=2000;citation_issue=3;citation_pmid=10902719;citation_doi=10.1139/g00-016" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=K. P. Lubieniecki;citation_author=S. L. Jones;citation_author=E. A. Davidson;citation_author=J. Park;citation_author=B. F. Koop;citation_title=Comparative genomic analysis of Atlantic salmon, Salmo salar, from Europe and North America.;citation_pages=105-105;citation_volume=11;citation_year=2010;citation_pmid=21092310" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=J. Zool.;citation_author=B. K. Mable;citation_author=M. A. Alexandrou;citation_author=M. I. Taylor;citation_title=Genome duplication in amphibians and fish: an extended synthesis.;citation_pages=151-182;citation_volume=284;citation_year=2011;citation_doi=10.1111/j.1469-7998.2011.00829.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. M. Magwire;citation_author=D. K. Fabian;citation_author=H. Schweyen;citation_author=C. Cao;citation_author=B. Longdon;citation_title=Genome-Wide Association Studies Reveal a Simple Genetic Basis of Resistance to Naturally Coevolving Viruses in Drosophila melanogaster.;citation_pages=e1003057-e1003057;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23166512;citation_doi=10.1371/journal.pgen.1003057" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. A. Margarido;citation_author=A. P. Souza;citation_author=A. A. F. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=N. Am. J. Fish. Manage.;citation_author=A. P. Matala;citation_author=S. R. Narum;citation_author=W. Young;citation_author=J. L. Vogel;citation_title=Influences of hatchery supplementation, spawner distribution, and habitat on genetic structure of Chinook salmon in the South Fork Salmon River, Idaho.;citation_pages=346-359;citation_volume=32;citation_year=2012;citation_doi=10.1080/02755947.2012.678961" />
<meta name="citation_reference" content="citation_journal_title=Genetic Maps: Locus Maps of Complex Genomes, Fifth Edition.;citation_title=Composite linkage map of Salmonid fishes Salvelinus Salmo Oncorhynchus. O'brien, S. J. (Ed.).;citation_author=B. May;citation_author=K. R. Johnson;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Semin. Cell Dev. Biol.;citation_author=D. Mayfield-Jones;citation_author=J. D. Washburn;citation_author=T. Arias;citation_author=P. P. Edger;citation_author=J. C. Pires;citation_title=Watching the grin fade: Tracing the effects of polyploidy on different evolutionary time scales.;citation_pages=320-331;citation_volume=24;citation_year=2013;citation_pmid=23466286;citation_doi=10.1016/j.semcdb.2013.02.002" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=I. Mayrose;citation_author=S. H. Zhan;citation_author=C. J. Rothfels;citation_author=K. Magnuson-Ford;citation_author=M. S. Barker;citation_title=Recently formed polyploid plants diversify at lower rates.;citation_volume=333;citation_year=2011;citation_pmid=21852456;citation_doi=10.1126/science.1207205" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. L. Metzker;citation_title=Sequencing technologies - the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=E. K. McClelland;citation_author=K. A. Naish;citation_title=A genetic linkage map for coho salmon (Oncorhynchus kisutch).;citation_pages=169-179;citation_volume=39;citation_year=2008;citation_issue=2;citation_pmid=18318791;citation_doi=10.1111/j.1365-2052.2008.01699.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction-site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. R. Miller;citation_author=J. P. Brunelli;citation_author=P. A. Wheeler;citation_author=S. X. Liu;citation_author=C. E. Rexroad;citation_title=A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.;citation_pages=237-249;citation_volume=21;citation_year=2012;citation_issue=2;citation_pmid=21988725;citation_doi=10.1111/j.1365-294X.2011.05305.x" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=T. Moen;citation_author=B. Hoyheim;citation_author=H. Munck;citation_author=L. Gomez-Raya;citation_title=A linkage map of Atlantic salmon (Salmo salar) reveals an uncommonly large difference in recombination rate between the sexes.;citation_pages=81-92;citation_volume=35;citation_year=2004;citation_issue=2;citation_pmid=15025566;citation_doi=10.1111/j.1365-2052.2004.01097.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. Moen;citation_author=B. Hayes;citation_author=M. Baranski;citation_author=P. R. Berg;citation_author=S. Kjoglum;citation_title=A linkage map of the Atlantic salmon (Salmo salar) based on EST-derived SNP markers.;citation_pages=223-223;citation_volume=9;citation_year=2008;citation_pmid=18482444;citation_doi=10.1186/1471-2164-9-223" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. W. Nachman;citation_title=Variation in recombination rate across the genome: evidence and implications.;citation_pages=657-663;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433578;citation_doi=10.1016/S0959-437X(02)00358-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=K. A. Naish;citation_author=R. B. Phillips;citation_author=M. S. O. Brieuc;citation_author=L. R. Newton;citation_author=A. Elz;citation_title=Comparative genome mapping between Chinook salmon (Oncorhynchus tshawytscha) and rainbow trout (O. mykiss) based on homologous microsatellite loci;citation_pages=2281-2288;citation_volume=2;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. M. Nichols;citation_author=A. F. Edo;citation_author=P. A. Wheeler;citation_author=G. H. Thorgaard;citation_title=The genetic basis of smoltification-related traits in Oncorhynchus mykiss.;citation_volume=179;citation_year=2008;citation_pmid=18562654;citation_doi=10.1534/genetics.107.084251" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=S. Ohno;citation_title=Enormous diversity in genome sizes of fish as a reflection of natures extensive experiments with gene duplication.;citation_pages=120-130;citation_volume=99;citation_year=1970;citation_doi=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=S. Ohno;citation_title=Gene duplication and the uniqueness of vertebrate genomes circa 1970-1999.;citation_pages=517-522;citation_volume=10;citation_year=1999;citation_issue=5;citation_pmid=10597635;citation_doi=10.1006/scdb.1999.0332" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=R. Phillips;citation_author=P. Rab;citation_title=Chromosome evolution in the Salmonidae (Pisces): an update.;citation_pages=1-25;citation_volume=76;citation_year=2001;citation_issue=1;citation_pmid=11325050;citation_doi=10.1017/S1464793100005613" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. B. Phillips;citation_author=K. M. Nichol;citation_author=J. J. DeKoning;citation_author=M. R. Morasch;citation_author=K. A. Keadey;citation_title=Assignment of rainbow trout linkage groups to specific chromosomes.;citation_volume=174;citation_year=2006;citation_pmid=16951085;citation_doi=10.1534/genetics.105.055269" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. B. Phillips;citation_author=K. A. Keatley;citation_author=M. R. Morasch;citation_author=A. B. Ventura;citation_author=K. P. Lubieniecki;citation_title=Assignment of Atlantic salmon (Salmo salar) linkage groups to specific chromosomes: conservation of large syntenic blocks corresponding to whole chromosome arms in rainbow trout (Oncorhynchus mykiss).;citation_pages=46-46;citation_volume=10;citation_year=2009;citation_pmid=19689812;citation_doi=10.1186/1471-2156-10-46" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. B. Phillips;citation_author=L. K. Park;citation_author=K. A. Naish;citation_title=Assignment of Chinook Salmon (Oncorhynchus tshawytscha) linkage groups to specific chromosomes reveals a karyotype with multiple rearrangements of the chromosome arms of rainbow trout (Oncorhynchus mykiss);citation_pages=2289-2295;citation_volume=2;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of experimental zoology. Part B. Molecular and developmental evolution;citation_journal_abbrev=J Exp Zoolog B Mol Dev Evol;citation_author=J. H. Postlethwait;citation_title=The zebrafish genome in context: ohnologs gone missing.;citation_pages=563-577;citation_volume=308B;citation_year=2007;citation_issue=5;citation_pmid=17068775" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Sakamoto;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_author=P. Howard;citation_author=A. Ozaki;citation_title=A Microsatellite Linkage Map of Rainbow Trout (Oncorhynchus mykiss) Characterized by Large Sex-Specific Differences in Recombination Rates;citation_pages=1331-1345;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880492" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Res.;citation_author=J. E. Seeb;citation_author=C. E. Pascal;citation_author=E. D. Grau;citation_author=L. W. Seeb;citation_author=W. D. Templin;citation_title=Transcriptome sequencing and high-resolution melt analysis advance single nucleotide polymorphism discovery in duplicated salmonids.;citation_pages=335-348;citation_volume=11;citation_year=2011;citation_doi=10.1111/j.1755-0998.2010.02936.x" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=M. Semon;citation_author=K. H. Wolfe;citation_title=Consequences of genome duplication.;citation_pages=505-512;citation_volume=17;citation_year=2007;citation_issue=6;citation_pmid=18006297;citation_doi=10.1016/j.gde.2007.09.007" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Shendure;citation_author=H. L. Ji;citation_title=Next-generation DNA sequencing.;citation_pages=1135-1145;citation_volume=26;citation_year=2008;citation_issue=10;citation_pmid=18846087;citation_doi=10.1038/nbt1486" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=P. Slijepcevic;citation_title=Telomeres and mechanisms of Robertsonian fusion.;citation_pages=136-140;citation_volume=107;citation_year=1998;citation_issue=2;citation_pmid=9601982;citation_doi=10.1007/s004120050289" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=C. T. Smith;citation_author=C. M. Elfstrom;citation_author=L. W. Seeb;citation_author=J. E. Seeb;citation_title=Use of sequence data from rainbow trout and Atlantic salmon for SNP detection in Pacific salmon.;citation_pages=4193-4203;citation_volume=14;citation_year=2005;citation_issue=13;citation_pmid=16262869;citation_doi=10.1111/j.1365-294X.2005.02731.x" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=C. T. Smith;citation_author=J. E. Seeb;citation_author=P. Schwenke;citation_author=L. W. Seeb;citation_title=Use of the 5 ’-nuclease reaction for single nucleotide polymorphism genotyping in Chinook salmon.;citation_pages=207-217;citation_volume=134;citation_year=2005;citation_doi=10.1577/T04-019.1" />
<meta name="citation_reference" content="citation_journal_title=N. Am. J. Fish. Manage.;citation_author=C. T. Smith;citation_author=W. D. Templin;citation_author=J. E. Seeb;citation_author=U. W. Seeb;citation_title=Single nucleotide polymorphisms provide rapid and accurate estimates of the proportions of US and Canadian Chinook salmon caught in Yukon River fisheries.;citation_pages=944-953;citation_volume=25;citation_year=2005;citation_doi=10.1577/M04-143.1" />
<meta name="citation_reference" content="citation_journal_title=Taxon;citation_author=D. E. Soltis;citation_author=R. J. A. Buggs;citation_author=J. J. Doyle;citation_author=P. S. Soltis;citation_title=What we still don’t know about polyploidy.;citation_pages=1387-1403;citation_volume=59;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. S. Taylor;citation_author=Y. Van de Peer;citation_author=A. Meyer;citation_title=Genome duplication, divergent resolution and speciation.;citation_pages=299-301;citation_volume=17;citation_year=2001;citation_issue=6;citation_pmid=11377777;citation_doi=10.1016/S0168-9525(01)02318-6" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=W. D. Templin;citation_author=J. E. Seeb;citation_author=J. R. Jasper;citation_author=A. W. Barclay;citation_author=L. W. Seeb;citation_title=Genetic differentiation of Alaska Chinook salmon: the missing link for migratory studies.;citation_pages=226-246;citation_volume=11;citation_year=2011;citation_pmid=21429177;citation_doi=10.1111/j.1755-0998.2010.02968.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. H. Thorgaard;citation_author=F. W. Allendorf;citation_author=K. L. Knudsen;citation_title=GENE-CENTROMERE MAPPING IN RAINBOW TROUT: HIGH INTERFERENCE OVER LONG MAP DISTANCES;citation_pages=771-783;citation_volume=103;citation_year=1983;citation_issue=4;citation_pmid=17246124" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=E. R. Timusk;citation_author=M. M. Ferguson;citation_author=H. K. Moghadam;citation_author=J. D. Norman;citation_author=C. C. Wilson;citation_title=Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout.;citation_pages=68-68;citation_volume=12;citation_year=2011;citation_pmid=21798024" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=Y. Tsumura;citation_author=K. Uchiyama;citation_author=Y. Moriguchi;citation_author=S. Ueno;citation_author=T. Ihara-Ujino;citation_title=Genome scanning for detecting adaptive genes along environmental gradients in the Japanese conifer, Cryptomeria japonica.;citation_pages=349-360;citation_volume=109;citation_year=2012;citation_issue=6;citation_pmid=22929151;citation_doi=10.1038/hdy.2012.50" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. H. Wolfe;citation_title=Yesterday's polyploids and the mystery of diploidization.;citation_pages=333-341;citation_volume=2;citation_year=2001;citation_issue=5;citation_pmid=11331899;citation_doi=10.1038/35072009" />
<meta name="citation_reference" content="citation_journal_title=Isozymes;citation_journal_abbrev=Isozymes Curr Top Biol Med Res;citation_author=J. E. Wright;citation_author=K. Johnson;citation_author=A. Hollister;citation_author=B. May;citation_title=Meiotic models to explain classical linkage, pseudolinkage, and chromosome pairing in tetraploid derivative salmonid genomes.;citation_pages=239-260;citation_volume=10;citation_year=1983;citation_pmid=6354984" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinformatics;citation_author=Y. H. Wu;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Accurate construction of consensus genetic maps via integer linear programming.;citation_pages=381-394;citation_volume=8;citation_year=2011;citation_doi=10.1109/TCBB.2010.35" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Comparisons between the genomes of salmon species reveal that they underwent extensive chromosomal rearrangements following whole genome duplication that occurred in their lineage 58−63 million years ago. Extant salmonids are diploid, but occasional pairing between homeologous chromosomes exists in males. The consequences of re-diploidization can be characterized by mapping the position of duplicated loci in such species. Linkage maps are also a valuable tool for genome-wide applications such as genome-wide association studies, quantitative trait loci mapping or genome scans. Here, we investigated chromosomal evolution in Chinook salmon ( Oncorhynchus tshawytscha ) after genome duplication by mapping 7146 restriction-site associated DNA loci in gynogenetic haploid, gynogenetic diploid, and diploid crosses. In the process, we developed a reference database of restriction-site associated DNA loci for Chinook salmon comprising 48528 non-duplicated loci and 6409 known duplicated loci, which will facilitate locus identification and data sharing. We created a very dense linkage map anchored to all 34 chromosomes for the species, and all arms were identified through centromere mapping. The map positions of 799 duplicated loci revealed that homeologous pairs have diverged at different rates following whole genome duplication, and that degree of differentiation along arms was variable. Many of the homeologous pairs with high numbers of duplicated markers appear conserved with other salmon species, suggesting that retention of conserved homeologous pairing in some arms preceded species divergence. As chromosome arms are highly conserved across species, the major resources developed for Chinook salmon in this study are also relevant for other related species." />
<meta name="og-title" property="og:title" content="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/447" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Comparisons between the genomes of salmon species reveal that they underwent extensive chromosomal rearrangements following whole genome duplication that occurred in their lineage 58−63 million years ago. Extant salmonids are diploid, but occasional pairing between homeologous chromosomes exists in males. The consequences of re-diploidization can be characterized by mapping the position of duplicated loci in such species. Linkage maps are also a valuable tool for genome-wide applications such as genome-wide association studies, quantitative trait loci mapping or genome scans. Here, we investigated chromosomal evolution in Chinook salmon ( Oncorhynchus tshawytscha ) after genome duplication by mapping 7146 restriction-site associated DNA loci in gynogenetic haploid, gynogenetic diploid, and diploid crosses. In the process, we developed a reference database of restriction-site associated DNA loci for Chinook salmon comprising 48528 non-duplicated loci and 6409 known duplicated loci, which will facilitate locus identification and data sharing. We created a very dense linkage map anchored to all 34 chromosomes for the species, and all arms were identified through centromere mapping. The map positions of 799 duplicated loci revealed that homeologous pairs have diverged at different rates following whole genome duplication, and that degree of differentiation along arms was variable. Many of the homeologous pairs with high numbers of duplicated markers appear conserved with other salmon species, suggesting that retention of conserved homeologous pairing in some arms preceded species divergence. As chromosome arms are highly conserved across species, the major resources developed for Chinook salmon in this study are also relevant for other related species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"R2EcTeIhX-ikd94R7RXAzaFJcxxijmmvnbXSJ0QA4d4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459816","apath":"\/ggg\/4\/3\/447.atom","pisa":"ggg;4\/3\/447","ac":{"\/ggg\/4\/3\/447.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/447.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/447"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/447"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/447":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wO6Hgn39aaiduNuxXZUEwphiGWTEyzFTjr7hbHOjl4Rkn60NBE4k0IbmHjmzk5T\\\\\\\/SJHUc4hcOeVcCKcZDRv5WNYxZcwJjAs6Tc+h\\\\\\\/Oe8b7MbY+uQleLje5C\\\\\\\/IYD2qeMvM7bDPSVkLE3lc\\\\\\\/Kyu6qtnegGEB865\\\\\\\/CBYAy4oWxyks8kC3PhfKfDL2GrhL5Ph9m\\\\\\\/V1AskMY0oKYWb2EqEbaULIlWNr8dBWZJtTzT1e4VHJ5hBMEY80\\\\\\\/ZrhUMXErUr72hSpLobXG\\\\\\\/pLxqcGqVncRWNf9R3xij5rdN3atF1ElWeOXRfJaBMwrCpHkd\\\\\\\/hn7NthlG3VztZftBYnSbSOm\\\\\\\/5GzZKIhso00QR6pec17eZpgotAVqzj3qOm0R5mP8SrcXpQbSxqxTuUWtte7ZWaQ7+GBiWO9dnFS06H3gRMkI\\\\\\\/OcZ85DSW9MxE5T\\\\\\\/LN1drx0PD+01Ilx9KRRYC9e5sy1KefWFO0EhGKpIH5ndCgd+lrcPjh9aJmtXMeOF5o6duUPyon7bfkjYAI6JuSW4FGP8ZEu5hCI7TELdB0\\\\\\\/BJVAXX42orN7l4dbS3wZ\\\\\\\/+dzYvo8+shTAZZGZsaLAHC2lIAUVwFIdZDlkPrWy9zKrkWGKcHb+LpYQ69MNytB0QG9mw\\\\\\\/RmRUFB5AX8ssl382CM2FtV\\\\\\\/bqtz5ydux2TTLzxIjUIB+BCPOucs1r6BNsnR9FxFFjduVQgkcyK9xg08LAJeYa\\\\\\\/SEKbJtGdMi2Acqt+hUDid\\\\\\\/GRHoAYMI5OUoy228sKYCG\\\\\\\/o6up950za+ttAsDSxyR4xRf0JVM\\\\\\\/FcT7O6vY9HQEfZpUkeL+fYQSuu2VDjnPy3\\\\\\\/X3LJxZEiF6KOYvI84XPHFiMkAy7T1pcmkQ5vM9P0U\\\\\\\/LjBn70OHWo7KifQxjnzRI1ii9m6PZMswN1bg\\\\\\\/6DrTx0FubpenlkkKnpSAQFFFe4T3OwAEGzFqG7pwriaVAwECY5Eb1L\\\\\\\/gfuByG3Fd+\\\\\\\/XsjdVLq5G9iXY5SFRjtyv8Kb57xRG\\\\\\\/ORTQ3qzUKcYlLRlWRak38T6VBi1DGCjm+GXO8rqezPEiBPLFFafpP5YXwwS4QQI2pEa0VJd7MbOJxpCqVHxJeCU6\\\\\\\/05itnwIHuCtnM7rrW\\\\\\\/D\\\\\\\/27CGWp+5eki9N9bHVxk6REcVlhfJ3WsePd8zhaeGnVR24LwV4J7iTUADO1wnOqAU0PnW0Dp4O8tvEvuRk54ETPhA1BfBx+mkPipgemLQrulUjvcweesvFsMXHgkJ7v7JFVhoaJWcyrm\\\\\\\/FMGwSsTLz1Bgc7KFqQd4IUj1DkEe\\\u0022,\\\u0022iv\\\u0022:\\\u0022JAHMuRsT+m\\\\\\\/qf7iSm+FfOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226c22def931b7e357eb3d19dc63654add9ca5c1fc90d723194dfa8793c3989926\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022weG9nS\\\\\\\/s49gAknDBRkraT48vArLJwY4pgEydUnqg2fk3nc+X5BWugvM\\\\\\\/DA6MDIY\\\\\\\/ZDxM610Bnl1DMr9PPbJ\\\\\\\/jJ7Z9peZ6+DzOf3xl9w03zicMVatMcCaulHtf7iBxfMiC2tZ0e9RiltyFf1LcbIJmJyVaSAa0sEegUm27M19k4ysmvp64m\\\\\\\/kCAZyUkCIJ6OE8jI5LILTyWXG3ZYZz5c\\\\\\\/UOqfnsbnKJ7dhWDIGcEF5kgfsyqET6e0\\\\\\\/UwbosfC8sNADj82qYiGeeBfnnKsKDkOhpZ5tT5245KfTtainmeD3VhAQFptPheE11HROzTshHanen3r\\\\\\\/wOm0WqL5PeLcvfY6XMkF1ULZSUPJ5CFk\\\\\\\/PJF8WLhz6hEMv\\\\\\\/p9myiJVTBWTpo40fHh2E2IRQNlXknCcoxF8ZYX59yK5eoSn4Pgad3admDbfCcUdF2nHHytSFle8fupbMZfsDuiW4g0BpG9qFdTd9m3IL63q3Ozv8IK27szfKZE6CBUFNtfrCRchDsqA82IQT+O\\\\\\\/bxbed0+nzuQOuEufhs\\\\\\\/Llg+Mdhw5nEGcNL5s9qfW7rHqnSEtU9KK3UcBmbpPKnaV0PmH779FFlburgNSKFFvLo1eQUoZFj3OpIb06pWszNwI2GanSH0Fsf05DOzBUjvoQk7T35GXtVu9G0aX8XJ+6CtAPVqfOs9ePoKzbUu8\\\\\\\/G8FHSGqKJOSfBYqKw29qAni\\\\\\\/HnBOvsqyBYUq\\\\\\\/seg4vDWxU7eDL6kfuBXHEEhip6KR0UKjnKpSg471pyMeJutsdqoa61geGeRPvUOZXxyhTOJ6u0oHF41D6vzbmHAzkHEKAZ7oNqRidN7zNMcxFyuAuKiq4\\\\\\\/YU1aXp17L8IPpw3FktRjkzUz7myzCKNS9J86\\\\\\\/PleLRzAmOrMA2VFQ2+g+Prs8FmG5MYWhZD2uFKJNdX7ippcjKm2fRhJJTmUaRLRJYawBfCySdxIwaffGtzYBYdUYiTh2w6Z+Ek52A6lwi03GwFxE5AUM0KWZ1TgIF7pa8UYv0ZspwsQ8OknKiYNltiQJKsZDUQa\\\\\\\/Chu5aiB6VuGrLKgodJPquXsMGj32A2ssu4mG9mwnz72JhzOmMuO+nPa\\\\\\\/RzrckR6Y6GgU24TwOmvGeaRn0yK6d2IGDwQk3pq+3qjRrcevDDEDfbx94wF5erdvNhdIrhdwtMbH7XbjQWQAOiWpySvka7Vb3pIZzpO4PFydVSvq1ZBHSZiG8nmMX\\\\\\\/PmIHdZ1o6yO4R0R3idC0Tp1kWg3KfcH99uB1w7vODh2EjtysO3wiR7XYZmyXFHSz6ArqUHQAPci7+0u4bvhIN6H8P0VwPDMWZAOHClW\\\\\\\/UGRsNiktT5b6FskvoACK4dXP1h13ljN5uYobEpcZvsCHGkz\\\\\\\/EIfXGWbJk+7uIETmTBj1H4ssyISN5XTzndN3C7vAHDrprsPmqa+kbdW\\\\\\\/Gm46ZbGPFr9sMe8prtnw+AAGbHuvasVMSpZOzjFciI61gKHUhFttmkAEUzoxBDL\\\\\\\/cNJHGcW+OCoJD\\\\\\\/lJfv8nZhLtdBML2n4vhEi0p3jl0V2BPp62z5iOFzmPk+FgEo2uTwS0FDo9c3DnrhjljxL+igNFTViQ\\\\\\\/i038OicBpHI5sle+WLO9NAB35lVrlstNGkSdUrnGTJtDzOp4M74xWCt25T8llsxGtp1PrDT4\\\\\\\/ii8vFUjtAeBoIH2CK\\\\\\\/zXD9UcFhQVUPvfPSnEn3J3eG1xYfe60OTKuxrYBUSqrxxOxjBTjvuBucuQL6rYrkAaePyCqWRTm7TOkTEPunCmbv0D20RIez1ZxBYV8xrBAU3R6ccqdpBIe8xreqX4DY5kG7OMi+NP3zDh4foJF0+3gv0UuIsOJ+1tsufnLFwQ6KwoDCbVPBb7+pMTY+qeTtpcX1mveaBZzSCxQhgel\\\\\\\/8mAtdNgiyERshF8DAoYjOVTd4wvZOd3H3QSjnTC8ivqTmLCt1k27mfo0jjzPaMJz6umpNEbo6vhDh56tmoLWneGu5oVTuSHzIFdCL4ncWJEvcL+wT8\\\\\\\/rvLm0QhVbZc366xcCSprsG1UhFQQl\\\\\\\/8cbXTtq6Pks+Vlt58oUEaJayDRYc81Io3eCjxPWFWnHkV05rBWPiRCdWEO9q62lGh2U\\\\\\\/S4tAmaF\\\\\\\/6EfZOpqbEpl+LiSZ+7myB\\\\\\\/Wno99TIqCSAtGS6XmgbDL3RFdk0n27m+7p4Zcmus68MQL4bXqGEo3Nr\\\\\\\/XYP9lSW7z4ghTc+vsbtF8I5uTi27CVxTQ4npaf\\\\\\\/zr+v2Bs1soG1kHdd5OKWNtzrnwoenM65FgqY5u6t47JodLXwgZ4Ytp5u87RGxFp3J5gDKG6gK2qpQqq1eagp2PnJIT0SeCwo09No2i6AGBdPCoBFN2NjUJsQqxjJlBgWsJkjzSnlYUTI1KvMPgo1iK8lXZyzsX2BmdZuoG+1dbkVa9mxxhp6OHOUY9A467UNDLj4uv2UA\\\\\\\/5iwKOZDO8dzICe4jfWKYvvbRTvHSB8KZ187S8lMVpPqMefsO1f7f1z0cP5UNTMu0Ur6T2xj4oJbv+WzXOQt9BPYf3KuZNukZS5Onezl8kMVS4q6b19bCJiGSsOvSGREBfYSCbBqsZC7ZbxZ2G3JrOf17KA0jBkGh9l2UCBM7gXRROvWYOf5M\\\\\\\/NY88MqO0gl\\\\\\\/HBPTBAXy62EZQpBzvq0N4jY1DoRuF9OFtbNlnFPSMn5e9swg+\\\\\\\/C1bzErz5x3ksksHUwHntCDdfhRqC+xf6djJTSJd0A9hA+096clhjt7P6XS3ILfOsxt1\\\\\\\/LguZZ1bmv+ksa9vKALQviMnmd6JU9zRRI1gVjXcwhcVsUET\\\\\\\/UpJCBOqv3FMlzE5ENMB3A\\\\\\\/w\\\\\\\/JrrkC54SNLu+CzevKmBrvDKik1R7xR6o43sEko1i7sTdwLZYYVSBjd9STC0S4aUCQgN9xAklmwfRDMlqNyGCEgN8LHUk58hXvGHRVUyyvmrAH2Ro079JgEmXoimdxkLFaWR\\\\\\\/9XoRz1FWcm13Mw68cOZVRbeDch9RjlWU6KfCkxIXkg4bEY5eQiRMZar73vB9hxTXRB31HAmrhT4YDM5IseS9lzAcBg+zEUv2tfkeHBgJSeKiEY56zW103vxsQAtcBXxZVvd0aHsvw9VDozdGx23Ss0neAM5R6I5QnkSwKTdfmRsPgRZVJHcRFKc3Fnv8AwQjaFBuvECuTQzyS8n7LB466ifvSHdFD\\\\\\\/rWKYYyR4vT8F2k\\\\\\\/TR1IJd5DB\\\\\\\/owHcvZWoL88zq0IylTNV\\\\\\\/qkH+YreeDhNaESxI6THvYVt\\\\\\\/TnDbH8xdASbqmiSqGckmsn4M2K\\\\\\\/gGKXq9kAtk6dVxvC9kmb5gPrx6ERgRUEH\\\\\\\/0pcjjNjULMIysOiPkghry2vdKjNuQadVMFhjcFa+X8OyErfc3pqLAhgzaGTDAJoCkJg09+ldoc5FDQ7c6MNa\\\\\\\/45tC24yiD34P\\\\\\\/7B1o6sUaIidBGR1WvwRQ6UYndGKwiPu7b\\\\\\\/763DrTzR7ahEcMkb1Deo7p1afLC3kIxQbtdGkHFM3HZd2jABA3rU9CzGr3w5+v0i5vcIn5xHhcIznmi8Q4MHuefSC4jR1f7m1J9te1aVJgVZWjiEY677TxIwuewUWNHqAlaL5Oi+7I7t8QY6lwNFskxY\\\\\\\/DDIhJVYzTjlNMd1hZ3JVT3N6buGEGAqZlOMz0jZP4Fzggc5usdRVnvtoM1qjSW7ZQJr9H9Q1Gv3SyAn5zpwJvJ2Q1iKdCvb\\\\\\\/KTbJvyi7JQm5ArfQxTsDLJteRfIHkYpLqawaNuM3kcTRNXa1HVpq8XBRn3ceMFCwdK71PPN4tpcRmhSv29NmS\\\\\\\/v3tP77Vu9Sb4CKbcCBeO7ahIflTNRQHMnwwC4HbRxPYbXfIWA0STyUswzz4+lMJraQGCF4CWJrT7gjAeoNKAgmXRsERgwGn9U9sG7ybptcn5rRp4P\\\\\\\/Y1ldrJLiYcbtstxYpixnDC7I3IfIzSX5y\\\\\\\/kDdC319KowZ9UMoJc7fH2rW5uaFkbHLnmHQKRcOXdlx2gRbQbHEpWVB6NFfi+yYE7eZCm05cPjP9w54cm53P25+BniPA1WJuhLoR0Y0iOBRF8atWexVNGVRLIKXYI9TX9h+LlelOcJ5NRLKoxeIZuyvl4S\\\\\\\/rHF98XcqTYkF6BeML5yArlh5faz7LyieUoS8kKWDkCgxaL091LRjkJWvFNSs9np86ltw3esbvOInDJz5RFNyFvWpPWdqlXJnvhitulx1E1S2M\\\\\\\/wgIdIi\\\\\\\/KDwcfwAXKnjLeN6iPBjCTDpQEupZS2FGmNaCT2GclZ990xnACF34W0PTDDAQewoDqQIL5NwEaORkc7aPNBpfOY9A1IZBVFOevfPjY87Uo80SPco9FjmxMTQrJx9kSAN21bft\\\\\\\/9a1maKOWPP\\\\\\\/\\\\\\\/UEJ8LTRpgFpNa2NGIInhTjyAmMjvWrkwI+SFZ0kEg0xIXW5WiDwVux8IVGQvWpOyRyeQDMZpijYyXKsTnXU+zKuj28v99+cI2An4mqP9hgVvL3AFHjqWnw41UCNFwPyM6qpWgGscxM4IwC0lVrsZPbwkT6hdo59RjVl8FZ1EY+s2pbckuATfmaiLNsWXEzI5tblx4Cehmh0mobYoeDjPFa9Au+6Man+pv7\\\\\\\/xkomM9GTDP1acz7HaN3wpMnjKon1K7jgfT6fb\\\\\\\/E6BFk313XDDVxdcr1hy+9CepCQsIPBbK54ubBC2hqx3cg2HcmPc0EQrU1zx1arPp8wei20m1AGagS0vXHST+46lYbV7Sb3o\\\\\\\/zBiCJX5KyoxHyFqv\\\\\\\/6EkKbHZaM7ihqfGBIR2CHnj4IpX\\\\\\\/BcagooN1gZSHp1kcZxUgQ8qOZ57hlAUyxSsgD51\\\\\\\/Sev3Hk5VG+3PiMNZbnJJJx\\\\\\\/UrzC7iRfbDE3i8Xb4vYVgWoA+8mJEyLE57FVS9NrRv0EoAlOcBfZiiY7MTuPRFG4TUfgg+q3N00XsJ5Zq6oNjuCCK1zPK\\\\\\\/e1kqI7rWECrBlEOHI5tL+RR1SwzD82IWzBRs1BZjroVSCLYCp8PMGV8WZfjuEKo8306Qe9it6\\\\\\\/1e77dyB3oKjtXWWw3dt4GIVV7H1ohVXYUmuQSx4wq2oYDZjs3005xOK1RER5gbXeqfEIsi82HlnVQXgnLIQOmhTvP+BtVKz3LAH6VhvdIevIVzBFXzIKm7a1\\\\\\\/iEC4l6uORB4f8y0Goa1DOg2WVRvvYZNi1yn\\\\\\\/TQxE6DquUlM746Mpn0JrpOqalq0tFpnHpqRtcJQz\\\\\\\/OzOUMn2EoNSEFZ\\\\\\\/wnQz7YEMsfHI12pj\\\\\\\/lpx94D56sj5wnOlvcx40J7sMqX1pbaCtJSBrQzEQrQObh\\\\\\\/Bs6E6HxyP+\\\\\\\/cwUT2u\\\\\\\/TZAOYLD\\\\\\\/a58EptAbeyg3pHlhBbygqG108yUnsZeTcKHXGF5pNr0ppMHDYtNnJ86VnhdU1LzRSFOVcLirfIpOOlQsGlk29FKubocNKWQSaMm0UG8GrqIgoSSIWfy3TKjSLvJ+o8RUla2vVoEtCyf0u0esWgqCj3sC8JATV4Sl4tJOTLi+fdw+v7Gsvt3QRE3mil0ICdEh6uE3Uh3vqqoCXWGri8I4wC6QRdxKWw\\\\\\\/QrlfXxExP5dCho4q5FFh6by2tf28dDNnExoWG9mxKWEXT9VDwulViejAIC5HH4gt\\\\\\\/YFj+RBzM2rz0MAz+exjVIW3UNLISjpnd6\\\\\\\/EvJnyBSJYgKVvA\\\\\\\/99XXWm8FYFS5a\\\\\\\/URxrdr5SbrfwoXUJaPndjbfdyGrUT+enjYybzYX50\\\\\\\/fxN5Y2NgqduFyoDkei+K\\\\\\\/v7lLxmL5bJkwKjZVJKzBAf7VE7J9SdJW3RduPh2M7fk\\\\\\\/v+Bsi5KhjNGBZ2WEvZlIg6Pd9Hawr2XUuW4k\\\\\\\/06lZiIoLrlLJ9ORt8Ja4LP0NP7FwmDhUFhTpm68JECKSWUZpXq9MFnNDxXLTBTbGnW4vhzszWvBz3zeIt5fD\\\\\\\/qrrG8OtraoZn4LA9S+xQBADNNkHBq\\\\\\\/vwB0eyjCAsdk\\\\\\\/wTM3BWXV\\\\\\\/k9n0ma9bWzH7BwDaiLIjibN4nT5V3ZcX2HkQdr5Ekpj9Gx9aBDXsR0QAPg0+g3prxUseYLLyNFn5MeVV8XJrUaEVh6aSVIL\\\\\\\/0w2fVgqyqZN7LA\\\\\\\/B90GHTn5Nmli4PKOOE7HxZL07EccAAK22FpJAgjA0xiMxZRjvrJ1moVHZk1aMgfVelOWSjTMvMdn7Uhght0Kd\\\\\\\/HjVKZQI1WKxFlV06adhsX8ELnwYceTOp\\\\\\\/mZYPeRlAzLsv5NaF\\\\\\\/p7jPqWjrKFtMmieMyV1LiM8nlbve3iyEG2GLTcweMNBLLgx7ETp8nNx7klgF4Zv91SGnmwfc00U+p7MWXoFsYXbjuz1jvo0GQpN\\\\\\\/U+csH+HobXaMKGV3YdLOeNOG7aQoxBPHI+CD\\\\\\\/ierKrfvg\\\\\\\/7wcO2aY+OgZlARQQJRi1ydTfc7yvIaV+LNDetZK9FWxqTMzo+fNLq6s23gyt1IjNweS6wPZ8gRzGxNztiazu17E0P1\\\\\\\/YipbdgzWJ5rs77h3FuAsHh0++6rj2vpM41u6bUEgEPNkHr+83R2cuKIZnh\\\\\\\/G9aKRWhzbJ9jcwwLD0kHfCuO6x\\\\\\\/cTFILmOmC83PnMYSsR5u\\\\\\\/AC93+G\\\\\\\/66YQhJga3yz2iICjxzvkCpPh4sR40ySM0W4RUfkyK2YrmbacrV\\\\\\\/GMUeu6UpJufrnIpdE36zoRLHjVKaLQ6nGpHKv0o456gxrqpbroPFOhz\\\\\\\/+NLoBsPvWqTDri2n22kMxRiio8XD3YRkHVqrXIXZN+2K+5tStBmY7vb3645shU3xsUkQb0PDCEUeHrYmoMkVJZGRgTI20dvmPSYXWybrsf1oWuoflz5oxLiRJs0c5\\\\\\\/At4s21CMRlx+iTQK03JKqSjYt52wSzMXUtpLwthmoQhB5CKcN9gjdAo5Lsv2JJFVZDyhcVbvBiLdNNSV6LCbLR\\\\\\\/ahHN35esHA1xTXcchpMZuqzp9Y7AKihSEPAfpf22estG2LSE5xX8D22H4vX+MP2bSkiEf90tuLn1leafyaKuk8vCs3RelekH8cGPqVl4zKBF001Na0+eNKqg+GgGDCRE6kMpvJI+mzncBh9bjNLs1S8lB+W09kcQYdD3RxkPYOl1LxGItVXapb0PGxXjZYWImYNXQCik1cNrDoaCjnv0Bm0XPf8uX+FtNBvHUhJ17T39DTzImj7wlUvIZs5WIt6otZqEHkdePOcYzqXfyfi41z3UGIzAEzaFLhYB99BvAWlNANoohLfjh6h+eHYiGBgwYIgLCrqZUpMEXEl1TtBE4Y\\\\\\\/VDVyEI3san5Uo9mUcq0FD3b4LiXIinw8fTF+L\\\\\\\/Jz\\\\\\\/mqYPuearBVP1Peiaj+ow4p7\\\\\\\/V4bZQ+LRZGsdoDbmxTkVO3ofkge\\\\\\\/yVoK0SJAjpYtEip6u3Y5PWbZmThCazYODuafGyivSLAJyTJvM8QHS0oabs6yYWqY\\\\\\\/m9JwtAHL7Ocs6nT1\\\\\\\/Z2DHuYg0C7QrG7hJppbw8DIzPpbEsYMYFZJloHRqqj\\\\\\\/Xg\\\\\\\/9CGZ\\\\\\\/gZmmaaoh7qh6vFvSBbay1d6rltohoTJ\\\\\\\/wKxCSgroyN6DzOYI3rGarXPzlvijwPZEPz6Eg0Qw9ILjYxbdftjLD4e\\\\\\\/7kzPqiCYQIHmSqqOeKQ7HQYNT7+hPkV3UXF82l4hoHZ\\\\\\\/5OS8wORQTq7t2GblqTSkHOIfCvkQ7BItXe\\\\\\\/Vhooy6lUpjYhi1LCbAFNTueOU4GC096MvjdtTjE\\\\\\\/CAZ7otGR7YIVJcXwJPXdXCJacqmts\\\\\\\/mxJiTxOgh1qqIXzV4FDeN3wFxzXbFzyX6v30L3moQ\\\\\\\/GZRVSTerEBvHs9mAY5vQhdjhJOVKzipDkynSsQZ3nC9kBha0V5w5I8NPZBlph6Wq3vpLVtM77s91g+fHh34vSmLsf11gXWfxG1h0HgqNmfqA\\\\\\\/Z6GBsJA6+NcjV3KKbF+ukC67UaOnUbBvdqHiYbLl4h0Hu8xzM24igfWB6c5YADFVaYJN\\\\\\\/TpE3575RqgmwA2OBcYLxOnRE+U4loY9Kh84xVexrh8Mt82t3yEKkhMSFrrBWIVXUimuTgJgUJGR35W4BbSl2IttnVFc6PCGJjZdOTKMHXJ7jEs+Y5ynQVUNKNN6qYzGaP4u+aTcdQK8HyNQvBwJ2n5tpN4t1WxCOBJkSlRrayNk7ddqm618WT0MomrrUY3zVIW2SSt0Nw\\\\\\\/Cr0tVzbdcHXqGJ\\\\\\\/cD3q7GlBnraCeS73JFRWXP1ipYmOi8HQATy7jK\\\\\\\/ZtQ5Qi6uQm8jg8d7yFMrOG99Q5pkbAxFak08sIBsJp7jgIRKHQlfCfeiFBufQAPmqSJhGiNcaUKQd0jhGwwAW6lXS+SR0Kx7bCKbZaBs7kbLqNNx6S0EFxEfcH3fm4tRq4SY7hI5n8k6oEVqp6E6B1aJm1E4xLQ\\\\\\\/wrAZ73rIfRu6uduqtnOkclpJcUvMLCSGpL+bXU9FodrLzMuvCA4PMNAmjY17qgIlw6ptl3YWrcuQPyud5AyvJvCQzpPD4SgYWH\\\\\\\/4ZaNVozx+t\\\\\\\/3qXrTH2Y5K9ym7pDEm+Y0FhXEjZGJ861klFELcZ5JUOFA+QLQf5fITijuY9oqjFUrl1d4woJoJSTPuRzwx\\\\\\\/4YllqszzkxqtdM8mmcXGitQRZSYjQ9iqFSVazNjFnGvjXkjAnpfsKrW8BKel8ZrFC4FRQaRYTRUO\\\\\\\/GJK0LuXVD8p9YwdKYyJIPSBZqM9TTO+WA1YATO0y21ykLqgMfOeTvHdcetC3n85duIPjt9shBDho9jy9CQ8tyYgOXS\\\\\\\/txyVweh0sYsWVO+UpsF\\\\\\\/pgjNLF5ilQ1PfP4vLDXG5r1bMXFosyq72Q6ZcJ63EsbH9E9rVnO0UcKagLB\\\\\\\/kTn15dFlkIvLUiu6keSLLvA3LH1QSBMhYYLG8rCz1GO9p\\\\\\\/CImXrk7hvTDk3RG+5SrOhbnRK3MkRrHIU0NyxJ3g3Gj2OcCYJPx9R3iuAI4N\\\\\\\/u+J5JZsb1E\\\\\\\/Yn0WYgdvMtFZ0\\\\\\\/246hkyt3fcjsmKd2f1DgmgzMbtphLqGtfu4I1bWyiwKS4xynzG9Qlc9W3OrZ4YdDgH662ZYIXufDKMm8XIzx\\\\\\\/9vohb6IXgoXnPnD8d+J7w54367A0WFcC8ZWyar7HvYTjOcFINBgjag3hUftjp6VQshz4tgwkpKy4yhhg8obHDv9b8GGl+It5TMagRZ6+0ASgNk35DHLWw5+10lw+cP1qGXwEl1E4hGBBZGzGSeQbUJilgKtNRy0YRtRDHvcjVrb3UGPeUBf6sZMed49VkVAgj64AwtVFDcHFHs0otDOL1gsq8jvEhAmn7VdU7tz4fmkPhLRrDeOaHZeZc4MrxpXvPZYZN7mWrWJWUqjpIvlFkfWNvcadUndFwuh2SdCymt161vI2\\\\\\\/7J9Re4cWs3w0bVEYnNXGb\\\\\\\/m8IrA1d+cy\\\\\\\/jvV75Bg7HBC54FgMVMPIiIrT228vYOIPrkEpPifXqaUllEfBCg7XhV546dd9qZpNoRO3nhmNwwHygebIpY5HgIZk84xT7K35ooYhiUandIMjYvI8h0NA\\\\\\\/LQCrCN\\\\\\\/sHgUuafMFIkKaMus0HIVDes9cP62BbAqrxwTAgsF8BUduVDognuhm3vqIVXbKD5cIqykDAdmVT7OwTXttMu8kD2NLAuIlK6qb1sgUW6Num5EWQZyQTtO93HuDp+gi9LTtcx9NsswhnqdqgDFrN7Y3qAvVs848sqn947HYcNW7T59ya1EM\\\\\\\/ZBMOwW35KCtzIRTx0JEOkLCasqUm6fP\\\\\\\/jzr3z7onEVVgaVBZM62jUpXDkkWBjOItzzfVsyrq7OQX3AjvodVM9nfY60VLoa8l4ZXkCXZKW1qJU54IJnTlaC8YR7FAAsm\\\\\\\/LPYBdJgw6XeHQKGW9RXHD0rsc0qciIyfG7Rpoiu94jEkIc7aUbrsQx71PmaXVf6S+t9NyDo2GDEGBc6JM2tYqfYRGOkYHdYE02xHoykGEEq+IwE18gSNBBqcScDkd9KWr3EwguE86pTNffekHFuiIjWdRWoYFD0kFLRj5W4VnzGlv8kFpOgwH1dnyXZzNCYw1lMF2RpSgILgn4bFryqoS3T0il+aB0OtoMl8E+b0Fbi5pDvKdI5wZsghbRqu\\\\\\\/NEwYtG5vN\\\\\\\/oeHI5ICUuO+uGGmF1AFvw2pQLWW2XaF8pH6FfmzVSp\\\\\\\/4jptO6zIY761e4ooq4arhcV378DZKPZIysGFsklwcQgG6yZ4ynpA9mySnQJ9GFu0P9ACgdDAVlYGnkCm\\\\\\\/ILqAcbBeYBmTHt2SSPEG948aY+aLS4TbVZEfYB012mjm4EZbdAYJgnhl5AeLrjyibfguzGc3EmX7J0tFMGXT+yUavVVFjuvg8oBhUxQTNusFnuckSMDGJE2faCmfKslOho7q9Ro5s\\\\\\\/H\\\\\\\/mQOU0b1fpKfO19BC4FWjzC2vYAjrfMIhhW9pY1q2KGQZDFTf4zKIlIH+QRknHFgk2OBHN4HSc6eqi21wcm47iW8PdxnJ0hTQme9eUCGAoHy7mu4D5crkfz5Sv1WTtelEPJXae8exy3Y1yw9MmGtj4vHdbNz6YeNkXikBcPNH\\\\\\\/oPHXY02C2xx07+bIAnmDwldenecTQkIxZ+tWrMrIgNL+5lcG+1GeTIUvdwZZUZ7fgyLTnFvkamGC+uq2AYQY4W4Hyjjv9zMDJL96D\\\\\\\/6\\\\\\\/t9RNR95VOstkB\\\\\\\/9Tvm\\\\\\\/gx7qdx\\\\\\\/+maO4fkC0mRBHLWgTtQ4LmrFpc4irY7y8xaYacCfltIT8jIfakkHAFWjN1B2mFrNTGjnlqZqJ9RkI8YcaTMKX0wWejEN4C1v5EP\\\\\\\/9NWMU3GWSePKWITbdmmbA\\\\\\\/aABASoGH1cX+nFgQtGs2DYg7TmfEsmZ7d4efGdcp9u84+Cb1T5gAdhmbnwTUnU6oE7nn0z9F2HnvuYv5BvvTE1BgxnzLzBcMZku403KtC8o3GDIGQEKL7usWYwxykDopy+R9oi0NaXRn5+QDaBeUL9Wqwl2ppZmsp9cxN4X\\\\\\\/VDbTn9kS6JubmrmYlrpYSnVvCelmtMp0ZGfQz5\\\\\\\/gZqh8n\\\\\\\/dz6\\\\\\\/jaolWSI6FQT5jT346Dbrm0Cx2uPOqvj+gBwtvWtO0RR3nlLA7Q34yw4GFbJjrCmBftyw+1FdIf7xABhZgYmByKEc6QcvxcYv4Tnt8qKgFb8bq4GCurUpXbs0Xih6FK+Z0TuVHyoYBpHJ+Te6UVKdV\\\\\\\/WK79T9f35HXc\\\\\\\/xkMK82KnkjEn6bMtUJXrmLLDH1fVMk9EAnRPV7MvsNLevQiw+v1FQEqGvG1Ex1Kh76RXms7cdB\\\\\\\/05KLo0h1b5DwURpbSWImo\\\\\\\/4Pkdfzb9k3ooHPuPt3luWJgOegMcxD0VHouhtsWG5q048L+hJrG+tzuPA2ToPQq3BihjJzqvZuxAAlxJMWpZtC3yNPlsmnNMd7\\\\\\\/OSHNLg5IhwxHsoI6L+Jvk3XdX96T9U5YoENmK+vZGN1\\\\\\\/lE9J6ZH2gKgCoFnTjJRBNU5vP\\\\\\\/V1eKKYKKh2KV4KHgCOLh4vJ4qpasTOdJSGUYb++sqX4Uoym2tqI3CG\\\\\\\/ZzXAv\\\\\\\/tbSjg0bcR4ZWPhRQNohDIvnK2YVEYwS0xIRr\\\\\\\/fHtPIfB5bDQ3TzcE9V8AnW+vRyPO1ldd+5fVY1iE7YQ4zCPxUpHTCmhTxR69XXLticIXTIbAdC+KbHy5omyRdiEtruxNfG1IGuGnKneJSn\\\\\\\/ndg1e\\\\\\\/TCoalgGTp8sWqYshUMTXeiE4i67x9GGYDP+iMGjW0jSqwv2whwjY6N\\\\\\\/LW5oKEjZUs6kOLojUoi9XLbbiEF+dHsHSmQgTNb8PoOfeWe8mySuhiwax8tgK3cAvA\\\\\\\/AydQoiQ128DSUTJUzuAjXkMjmhGWXBzac3NYYHtdEElIwV5k1rmvHWob3MO+DHL5el5baJ0n5ldusJ1zyG3OGuBlYg8hcKKjQ\\\\\\\/7XUpmRMTQ7Qt8cwUh+FphgWmc\\\\\\\/x+ZQPDedVbQrz6tVn\\\\\\\/cPHQKg38ta0oVPDOuBR5\\\\\\\/FE5qGXBnaKL5igkFQoNDS03Es02R4\\\\\\\/fhTnVaOOCWCCti2UZKKgDlkllz4oX5YrDqC77Ke6hbJHwIrmCPfplqwFiyisT\\\\\\\/0UjoyI3pykolW4jDlF7ObQ8sFU7vh6nqm0uhPbRjSq7mA7qmTQ9YbahTjwAfAboGiGdqADc0eZ7eVwebc+sg66DZVGOzE4ZtZQr5KaXwC5YCfuK+oNc498Nrzp9QdgemukDDQLoBk15HH0XnAW1Px26b1se8iKzVsI62evDsSZTWE5u1Dxjac8hpgITdaxSOw\\\\\\\/PadazyKIpNBZ7zf6F8KvAHpsYMZkS4wj5Xx9I9oHLVbBcFj1MqtAtaVlb3bO1ldMz58OugeMBRavKXBVOhmabnoqMdMBPvxthveUpEfNrE5KYFEzApatO9Q0h6LRgY3mKEFPe5RswLS6AHS9\\\\\\\/xUsVgR9NnMjs7KFnbslolLnbCYNGvPUm8e4mfKGUbqwqNqLUsyiGy9Ue4xobPWxiMomAnU2slMvCswxozyXvB226A1wT7s+Fu0Qpxk08BZWBCRo2yB3TNnQH01aKY+Dz2PAOan\\\\\\\/eSgnyYy1Wrf9Raj\\\\\\\/mUOyUtKg89n9HCyBInakxO6iuCIv7Ci7l4GIBH7zyK\\\\\\\/IUUfsrstnyCI+lkFIGOV50FP1VvvJGT2BfpTWvvEhYUYDaIE2MZIdv\\\\\\\/RDAU+TVFrpAWmuH\\\\\\\/UARuynr6c8T228dyivEoIcMk2fv\\\\\\\/Z8vNMJoMaUsM2iV+Vbe2HpQOEs79WNBG2BvX1mmwHg70i2rdXCJZln9pgswLnwgHM\\\\\\\/zZKpjBTTE4n47uHeI\\\\\\\/W5jy7jMFqmUHOkHfT1lH0GEtQPzlxnGV4SP3kVOVqFwBbBHkRcyCJBPvxaW41udtp0py7LR+APr1Sv43J\\\\\\\/mz4gM93xT8T+nk2C5raDRLzmOLmd7kDacR7B+LuqVpFIPv6EBEMc+i5K\\\\\\\/OPmapxafYLt1+lFpqp74qxw+HrW4HVTWPZiqIkWO4aIG4NcmW3LOgWtFum1H\\\\\\\/OJJiNZtVj3WAKOFTb71u6f\\\\\\\/cww7k7DBx5Ec1uX89fi5SoQHMr6XG+r+bwU3+2p27dGp+i5WnD91c0GIqwuyx66tzSsEPLYfSuuNt1p93PPseFBlD4nGVrnikbvOGrKZUsW7gXHodEoBhyvMosK3NHdF3L3XOE+51Rag6NKdm5G1\\\\\\\/SVH5jQGnd8aPeSZt7jc4YDIJ\\\\\\\/z+jFpovczimDJe5fOCQ53MXlkDDQtL\\\\\\\/jqhbIyOFFdpSpa2cwPMdeKzJUwWthV\\\\\\\/9UD29AD56G7wTfRQf41UljTJDok1odaM+j2Mj86M3bxSYJklSC9sJbvffSwmCiizffQP\\\\\\\/CtfifLuTuajl6lkZNFDHs1yz\\\\\\\/ccZCrCHPN6cDRXmFx40kdKZ80jBcprE1lbpwnHpshh6QM\\\\\\\/Q7zPcQ5SoI8OufBRLHv1COmLIx2mScGEU3aP7TaPRtBB2CINnvuxZDXOtRF3OCRXzCo4ZgCzq7dtJXas+ajBoDqCPZV\\\\\\\/aTvU8EiHgRrIyCa2fsrtycpbxTPbt77zw4dl\\\\\\\/lmckmweDTHuvXfRBbf6TGRyOWoqoNlVyrN20VOrrMkB0aErbmaMefRxTh42h8P1\\\\\\\/7oW3yvJeHmU0z1qfgvHxjER0Wm7h17dQu8BnC3eURTOLFSSRCbjkogEGt8+pfnXMpTOoYCWDJ300l3HsgZv1nYCdgRshZJIeSmhOL8pJXFFnDa8agUE3OopPV+jp7\\\\\\\/PallgHVZn3RhBl5Fdgb3lrfP\\\\\\\/Zy2Eme5ZJ45OBFLJkBK5BTO7TzHVBcCq4Xv6ILIw9MGTZbjJef\\\\\\\/tFbD9hIr\\\\\\\/WatNWsmGjytCNjPJtd0DMiY3lleGdK\\\\\\\/Eo8b7bs5qQsP+jQzVCHozIC9NBaNEzaf\\\\\\\/2SSggll6dIW6Jkzay6dSJdLPRZhzaLfZDikl6xRX4SplVQki1iPSc2UUNBYruOnFQ1+vS0e6b81JQ5AQ5VJKnCkZ4vVtb\\\\\\\/1ShPlhjSGGsdidk3RI0MU4LHcg+HACTgx3eHCV9a5M2B+YrxS\\\\\\\/chfvE6s03TlG89GDt8snXecPlUx66\\\\\\\/uWA7kLwD+HdzOxbQdpeaMhnHJvFB2Opc2tPEmw+l6iJilslulGBdcdCcexTYukNeNHZHj5uLXpfnpFn8mzFb5+5rk2IRIYpIGdKAZFEEp3e4\\\\\\\/7w1zfMdkch+NUAEdfgsDjtxg8fMNx8aViWnz7H4zLDR5fqkyWVYTP2TEsebda3PwVE23mjwa1MM4ky3oO3gfFSbKbJkg1tfkxMxS\\\\\\\/\\\\\\\/gDG3JdRJkgTyaareK3KwjSwvZY5dQPdrsVHAA3cfK9XFl7cp1WnRgSeZ12eGtsCGm5qGGecB5d\\\\\\\/aCIHQdhaatZmaGMVHNhxKqD45LuP3ciaWVFO7H\\\\\\\/9UrSNInQkRjIyA251dVwXirHTx55hmD6\\\\\\\/CQmOJxLFfdqZAauxFrce\\\\\\\/Gf8aadgVySCzMU9kRsTNSiXwdhEWfwbeNUudBahpNQVMVT8nqJ0lNY2dDJlWroHe1xEmnSVVX8fSyRxc0A0\\\\\\\/DJbBHWAjsV1SWoGcQPolAPIqS4twXnAt\\\\\\\/996zNFHBAf1fh84bvJlZyZkctnXhfheDbI2Nl1073kMS4MDEOpyIyA0jA+nYnvrN\\\\\\\/YNPSBBcj+v\\\\\\\/qUjXktU+jDukd+yq54o4FWlZnL0SLl7TAC+Qs2uEPkWla3vu53N\\\\\\\/VF7eiqixa93OozAAKFK9NroRmTvr5d2pA9CKitce4Dwq3c7aN5wjKv9WuIxIeqNS5DcE75P0VU7S8dTXuW+qj+JHfwQ04rrWmFjHkii9jfEfl3yqMqXptTpU2wFZo2ZUYtQi4eYFFQCZz49KSIHG5x3vQsKcOW7EHShx+WoEqKN1vcA4mSGScnPjWIwnzFmraxiUt6UYbAZOrFZgTJXnXNXIufuxlhtymh3gMcGU7dlfoz74js9RNm7VVaGOSK8dcpVmRI6bhgMZTdYpTid7l7VR+\\\\\\\/EVCW\\\\\\\/GAW5dT5w2yTngFm2yNK+nlnNeTNvOzVimQn2FTjL\\\\\\\/Z\\\\\\\/tRzkgCkNOUckGu8p5KuGXgTWk1llC6H8GXz3cz2TCKjBJjOqtf3gRsiNyTnZ1DRtUST+sJM\\\\\\\/qax5Z5pA0scwmIQ3vOQD2N3KGASA6Jz1lwVI8oaN593Qk5iCTiVBxrXdKyRwZ+6DBTmIVGe8F2HOK4bcaXOWZ0ryD5AoGNl1RQpuKVVBfu1IgcbSosI51J+qtmwypsWfBkaTA4P11Gq99uumyPzRaNRAz6F8qZvDZlEdouF68Cq05eNeIPnDyX+6WyQtGtdkm5zEn2vWncID77zwIK1754Z2QyAPrpWoBCQfB4dPWe6VyuGUYwe99YnerwqM3aVWHC\\\\\\\/vjX2BA21PVnO3I6I6qBx6Y+Ojlvyj4tdxzg65rTFM2iVnO2vzqISpdCq1unWBJDUGmQ3Nca1\\\\\\\/GDkucnRWfUszXGUqctHmMo8HoT51ny87kXJk5szdvLnH8bNecUlU8R6eon+Zj4vjlX3mj27F4VzXINaJoS4IvEu0M\\\\\\\/HAQ4eNWEJ3ASx\\\\\\\/gyi+wpFvTjMNo9BiElBa+WQNUDvgNNZtJQBI\\\\\\\/xZ6\\\\\\\/jl8lAJaPXuuAEuXNh\\\\\\\/qgg\\\\\\\/VZdN3ESOOIgnXEKPK5j4+lJnz3Y0G8+4FMzilrCRN8dhYC5ttoo\\\\\\\/r7v1BYApQq4BCq\\\\\\\/CMwQt9PXqSdRBiAMNK4DDwvqH\\\\\\\/zIDISSHlvYX5k\\\\\\\/IjcolzqKOo3enXFJ0wUl\\\\\\\/vnUX2Q8MY8MxnualBvbW8aCAii+7egKdkQiZ8DU1rwEO5FrS4nKqk\\\\\\\/COjgn+1aZiL2q6KIBNjNfOLwMsTfTjiq2vusr3W0pgvflJVj4xoTT1\\\\\\\/9q7NavPUB1k0\\\\\\\/jWJIEzc1s6SS74T382PTkkUXiL1UfQ6v0G6WJIPkpnjFKt+fCJBSvkgaJLq\\\\\\\/mP+721PB781yh6fyFFyb+RSICeq7Jjy9KX6v73k8eldA6aWaxBoPson+k\\\\\\\/qQOA2cNFxp++8WCpQh9YnhncZmp2liqTSm3Jj+0ls4\\\\\\\/BBGMx\\\\\\\/WUkmsdSI6xOS76YDugge3qnb7ZP5ZKTxMV1\\\\\\\/3uti8kIm3FaUYnJigBLJH1lDs1i0q6G+SBu\\\\\\\/QQnqOyM+kiI5mTyg+9nqiZJ\\\\\\\/eAGGrO2mLOwGZ\\\\\\\/oeg5fdUpGhFro1rRjk70aaYJFtFFnB0H13r3x2ryEzaASANVCyZ2QXYya+OPY94jlsegP+OpJwZWsJ\\\\\\\/w6f2k4rO3eIJM5yeQ1f+j0Hys6JID5ZUUsGsgmb1h9bzQFP8wOZzHbEzzkQ6VTjdDqh2vPqM+StnqVblnSPzLYcEf3i6\\\\\\\/iM\\\\\\\/2dO5oUna9bdspCRZKh8Sjkns1UcOl4GbkJ2XhBXbNrzybGrg\\\\\\\/8B3PYzHU8QPSiPw0taDEEos7bbfNBFwha\\\\\\\/bCLA\\\\\\\/s6aLlXbN41mhDrno3GK2\\\\\\\/Wsw\\\\\\\/zIBjddbL2njzZTSY8Tw5BAnI01wZKLwCdWZwdruyDV01vY1rU\\\\\\\/RBY23BvrIrp4XznE9+T2B9oh05RXWLR9211XfaK8bglDcGBZ7lpg6lY7W\\\\\\\/PCAn3vECz4Ndr81YbqRnHJo\\\\\\\/LCjTOg1PcFXd5Q2nGpCMuj+VVTTr2kL036dBc3BsV0aeoqnHbjFSStj2iw17sngZ87WTB2Kwp\\\\\\\/hZCxlNcM58D5BhnTk6McPvuMDdaF\\\\\\\/hmrgZs82zML3VI5KaRvemfSq\\\\\\\/lQHFnlSjVpWdADsCZYpWtj8tnuh6jrZtuDv7pgxw7pgmAk4LUnxCY9HPu\\\\\\\/jAAHGvvOtarSEE92cJarAnT3daY7i79tgiZka\\\\\\\/PzYiDP3rrBHyCR0d5qXoZuBwZeAE7xhIMhO\\\\\\\/FCr5c32N4E5JZfpMXmJ3\\\\\\\/HdJhFz8zXCAnr7oxpBVGQZOn1qug3Q3d0LP4+AcQmZ5fy9LWAlmj9gsdF73BzKcaNaCWXSA7z4tYSrz7t2fP5Ux0Eg44FGVFSon7V2vyW6fexZisPlnIaAfNkbQYzAZvbmbAEz06Q4tl+mVcyiwRC8aEuSvlwpxtCMoxMO8kPFqM9GJDfMe9A0YotI6KhmTr3b5qWSkEFzF5t2EeY47d5FywMTc0zjsULCTBwASiiSe1Nm\\\\\\\/x8MiOVJLJGmxNmqD7Yh4pxBG\\\\\\\/SEhhWsEnignV8MhnZ\\\\\\\/bOwebCSQoDWIa61dECtkrr4lpMJH4eQ\\\\\\\/ZmuiUypg3Zb6PF+WbykuPj4zLMw47Zl92jhv2z\\\\\\\/1xsqCMwx2fhTdRZCymjDxz9lM78rkQjq7E1ZWnm\\\\\\\/A3WNHgQ7WzvBbCAPsqDWm2otpcMEC4YmiP226s5sEYqbpOXYK\\\\\\\/bbIysWqkM6+5dcufOSobvjjGI+FbOP3WitLOCxis3H6sAYyd1ED0PEqOBCr8rXVPsDexPDALTZ2Q5WxkrDyAyDZCxGRdUIDUz0D0k59WqsoPJ\\\\\\\/soRSa0os1vxM42G2i71mD+VzNVzU6Sye9Jm\\\\\\\/ZEkVHh0DQnxNPXQGpdtb92F1eWGO6yATU8LRaFJ3eQyfFJhCyw6CBITr7lA11vCuXe\\\\\\\/Brp\\\\\\\/o+KxFnfRgmZN9fnnvejXIl5IHuyjY9iLktujUUPCuRrgVgEQGzHQiM4d52\\\\\\\/IG\\\\\\\/2Ma7Njs0spoF8z2YqJqLAVa+diG+Sqou8R6WH2agHzcrmBWfQi0aIB7nnQ3Ug4Rd3H+KVjnAJBmk1tFrrKWoCQ+rIkjUd\\\\\\\/ufGCvtBWh1c35\\\\\\\/rHq04+TWGHEkHKo0EgkcK\\\\\\\/Qk9LUdpBvpe2ezpiuHgDJJq0v9vXf4EGYWNsFLpuOe5DXvBrh8i0YeVqFmbDRbnDZrWVA7m2bIJ+TA3xSgZhzfpnR\\\\\\\/GN9CHIb95V769eQJFC6ZSg9DQRSTD8pRjWTz1CrsnbQZr2U4cir0JjBbXOjPsqL1BuuUAIf+Ba2EDrrsCGFwnf6PcR5zQFO1b+jzxBiHNMGKTO2UGWyyEFx2OzfPsw5YrGAF8foy1e\\\\\\\/HWwz+kT2XyfXjKu3PPZMRxfAV0awXzaG46qj\\\\\\\/Ka5F\\\\\\\/ygXk6ES9J1zeT4QWANPe3kHQMV6cusMJePLz5jmJJ1V7xYZaWdP8fs1lnS0WI7NjDRBxWQQpuBBleg3MNne2Embe4sZooTMFZZRLjykoAzWRckqnWam8ZaedJQkZ3gSL2bDhQ0GyYtOSIqEpyudfxK8h5ko8NDyKSy9r3Z\\\\\\\/Yzfl+IsarVlBsZIcHQu0SGuj3mN5rOEqelB1DQxm2iULAWjWSQ\\\\\\\/1kOtrNLJV5RiXXtO7lKMzwzc3Jdlmk2Q2rF3UYVv1Rlz7WsupEm04O6pj8boAJ9G6304I896Q7fa5CH7IwDmXMB2cOS0Vq5Ref3rNRW1oYaVisgBNaEddWVKPZSZQNDP0L5KNf9oUiU3qXjBXhAaEfXMdC7QL32plSGDUu2Kp2BUftWiD6oa562wlPrg9G1RjDrvkWheILS5qEFNJKyqOGU50++iR1R54cjnrW630OKdvgHqlu\\\\\\\/sQoOZOWJaLggIr80EuK1EraNzu7BN6wIfqJSyzBhMwKhzPs7MHCO\\\\\\\/Jwd055Ckc+GCd3V5f0tPDghI+Jm0CONVcEC\\\\\\\/7xj4EAIUYKdNThYLE8SRzcmgF6cMXwNiV33wD7adjn3I2L1gd6CRKa\\\\\\\/AP92yY9fIrqwVwjOtXSzAhdaybI3G4WK6lhSGYVbz1XsxNk\\\\\\\/7Vqj3X+ZommQdz80m2Wc7+LxXLWuallrtywoRDPwQMKTIYRCqi18Iu4w7HQAWGii4ROv3IdWNdtbUSGP5j3D5h\\\\\\\/kGPe0Ci+Ow+tgbR+2CnI6v9x1++S9uc\\\\\\\/6ugk2rnK8fa1MVEXAhGl+JUowQF3mHtEE\\\\\\\/ZXWEgqH5tlZWS\\\\\\\/\\\\\\\/cN6Xknxqxwd2W8HsvM8Y5xvlGTZGJ\\\\\\\/1ekZ\\\\\\\/bmzjRSGKCBWSx7yNpwHv+mGLvLfPuKyF0E+XDPx5I2SFB7bG+Ptehm9iAv0Xd399yuUsiTdoBkVJeaa292UeeJ7f+nDSqlbxR0AgSXJJ9zsrAykgo1fBD1\\\\\\\/\\\\\\\/trI996txOHqL98vDu86AVV8dKdJli\\\\\\\/rEn9EcVVT+6VVG+f2ADJLcUxuGKp5yaVi7mKZQDOtjZeiRnYnFqU5YWPB4F5GVqQJ75qU\\\\\\\/ZB3zzldN7K6SFc4v4lSYWJmNEZ5YNBu6YcFTWKgkbs52qbtIPIumZR3CXujOtyYyL8LjfJPLnHPgfucghzSn+eY8VX90C7UKTGmx0S3CIQ0rXwNp8QQeTEwN+iZNcAp1f8uQ8pVyzKhQusTywVj0\\\\\\\/UEcUWEEitOdRC2wbrZLB9weQ6SJTtfehCjaJhOMwXnHYLNL7HneI9MmErdzVHoTmmSIzbCjds\\\\\\\/IH8pRIxA19Dmt+KaWI7z6KWtWMyQdU4zzqU+Gj\\\\\\\/pS14T9P5cKFIuqBzGMljxawFknnHZ5tAnv\\\\\\\/r5n1Dj8h\\\\\\\/3m\\\\\\\/QLBzarozA\\\\\\\/fJAsS5ixFJFAoZg3ESdYJQ7U9JaX5Be50Pe3He+WBdH1mAzlJpYLDKJWWIFGTvx9XgQ5TurqgXT09REx7Ur6T6udUASvGWDa1Rbw4HUBJU+Qb03pAUd42c3XDfJcr0mMCASNm6it4VZTLWLyp6\\\\\\\/eg\\\\\\\/4MW5bHja6GzGNs3MgIAi68jaPFnOVzTUNQKD8fpbM7Y\\\\\\\/WE94XkbmVMMO2Do2xyl9tRavIGluHXxhIZqxHZ1gnXb\\\\\\\/ZW4SZmAspDk6K6fbtnjVKe0wWV3ikodAd396SmYIo28U7CTNya\\\\\\\/P+ViQsFCsFBqkE14wds9yuZia6VrKm4zIqvjvt8bNhhPqVAQk\\\\\\\/x2nP0yfkvbBgPK+51qI7IUrwui0CJ2veIs6nKmK+bXKBz4ab0j9lsHc+LlsC7XOoeuuNnn+PVQtXepTGHzAoFMn5dDDLczAFCjCG3jVIscTXvUTtYXRLhFFZC2et4SE03nFXvUtr03ZGtgp7zcnLJzEuprKznH0eoBlOC3vcEqin6UrA7JW6RxIKJQSJ86v82VavrTAdgR986B26uhGsJyhBfZ2gDWIaimfyit+F0xm\\\\\\\/up3sk+HjELOGqYx+YJEvak1qy8lpWMsMmkc0W8LXwTHmBxOjySLqmOs9Ek2975wlCScfPmoc7nBOI7rOHykkzo1F2Voz9iQB1+gQ6SlJJnvFnIluQVy4jlq\\\\\\\/kD4jg0iZQRwzUY1rHnvxzfWQzD4xUTnh2REB5faDd1PTdghdh+\\\\\\\/0cnrtcyHq1ELEEaCFFZTnX8x2LDnpGHvItIoj4KTzfcSW\\\\\\\/FwKrx1DE4FY++KaDSQmvgAQOiuIopcO7FN2bvHyKX9kCrqLnF6OOUxCFmBA4xxD3CoZukcpeIvSfvyRE8RWE8wESyWghh2St63ubWsbNj3NmhidLMw1c0BcBLDQLLH6+Htd5xsnJ6V6aj2aTiYjsOzbu57XzfnzLV17Yu1ai4jS\\\\\\\/Xr\\\\\\\/jO7QGDhBGCyE+4I54jtgvGS\\\\\\\/cPgfvylu\\\\\\\/f4TSehrLyBgu8lFD8dXACPEP\\\\\\\/iUxC0pPlvf9EVTVUdSBN8tNEYPFYWXh+SE6efpVp8oEQmn2Vs\\\\\\\/9BHciUVgd4DB+eiJEcn9kN0Xc2LuGdPyj0yZ\\\\\\\/gUNXUWJOUOYdUMPlriNq4VIKe1CTOOXuBnH9Fbo8yTcReNCurggwl52clupUJnO0p1TNQCEpLdWZkrah5ad0ytrDtRB9cu3p85mTM0HXzO1ReiQ\\\\\\\/gVHV\\\\\\\/jNc2m82ivHqWJxvz2HdfFpGfz62vw4azhVnampIpcCyVlwFaj8n9JHTxfhsmASLy8TtXTlTb3qgfEO35nlc7x7gGkxCd4ywxmgN97tInYismKA8FJp70hZeVg8qnQbl4yXcbyBdM48EeMdxZQODJaCu0VHqO+HYCL4dpoOblI1EX1ENA2GeNizOJT6QKoOHh58GlVT7h1mSthNFXJPOR14s6S0LO1Niq1WUDEQJAU\\\\\\\/\\\\\\\/R1MDNOuUgshsedcyxYZYf+WAGqLD0ltmSEbdOQomwmulMxpqmZtoCmuBk6DUbTOi6Qh4gcDAWPwWu73UeGrbctz\\\\\\\/SJ8EuVM8EX32z+CG6011wKJWy8fMkIAeJISvCMfKM\\\\\\\/kIYOWo7WLFu99+9ow0ywURVMgtz\\\\\\\/rSrScsOxZ4hktcbJTyAD7ZtP\\\\\\\/9enPYPfkQoCxiNZhFWVVSXVK65FHMlMMri4UH\\\\\\\/yVJyaH+Q+Ss1sv4y5C2BAPiY3dWpmvMxC26wUWG8UKelQFi0t9STsybnLwF6Pk1V6Hg3mewo3iSDjmYY+lRxBeSPLT4yS\\\\\\\/hoPszPxjSI1CDN+GfCM9kQ7O1HFil0h+5Vpmym9yFaD\\\\\\\/8\\\\\\\/5Go6Y6ZNvwWh2rzCuGsN3J4FsLgQtfY2hE5IgqEqOlwVxR2UDjR\\\\\\\/bj1Y+3QtO0xMICadlq+XOX05AoMWgwlHm8GCOesZah1SEXXg2TLS+nNzi\\\\\\\/F3X8YSDhO3b5aIKFfQCvg2\\\\\\\/wQKXgOTXsMOzNFCiaMcG2mlDNGIcjgphHHlphD6yItqNwpfE\\\\\\\/F9GWCRZcViHRBqSInJ8++kkP8gKNCgMIvt6TLsiGX3JUW4y4G6JM6buWkxmHMMCvyrsNY48JiShnnX1RR6I6KJlMH41BzDLqPkDIGOaFDG4inFXURkRu9jukuecld93l3anWBF2J13NEwxRH5rRv1XBAdSJy3snGjpjn+IYeuTHFxIHKDZzcteGdKSG3mADuQ1MVVWeZX6I5gT+ANq4TXxrZ3YmSyTHlhgfDi+qlX28R6x+EsK9awQwIKIrMKeIFUh5TYgoiSf7+\\\\\\\/SAs8We1pQKG6EOCh0\\\\\\\/7\\\\\\\/sIrkS8PImrlm83UCEl31xL+rCzFxmIeBHbQkaLYgpKVEYsluTLCcm0CsfLQszVSYFi8YBhmCdvnWPJE2VB+1Xm5SjJBEskfUI9pmIy+hmiJ\\\\\\\/LK\\\\\\\/wx5xEw4lgSB2QBmLbXaSrULsoim501px\\\\\\\/xNsJGZiVog+E0BMAGllBC2DFiVJbVPPZjPbi3CEX4zdiZPtd+At8fH0V+bM7UcFqXG6MLojX+qFImxlF9aSP\\\\\\\/fEOGYGMVDyoqD78x556TMh6Bg0UHDmbJSImuekdFnwMTlWhDmrrIC0l+R7Lai7Ff2x3QsGoOjXCJOb9wMwIlEQQ7\\\\\\\/MrdmDNO+zJxKPopE55SipV9hujnqKyKkOSpptSxSwgurGXxdTeFHukeTm36OAWcuiv3zFv5xR8gS2ekpVAv29Ca3F0iKn\\\\\\\/Fkh20zrpfK2tJzFVnZmDqp9oTlY+BV\\\\\\\/UoVBK\\\\\\\/JNpCLu8raftCvesr4m\\\\\\\/\\\\\\\/t6ocV+hSmDo8t2mi4Ka4nMZuiYTpOTAP8L10LxER2Af4VSrGySON1xmtZ2VQYMV8vLjOhKGMY4LOWdETJ1CPV46egv3k63\\\\\\\/2+67i82dhPTdfZD4wFj21CyeBADUgA1BXLZxkA8w53Ptgo2gFXc2\\\\\\\/DXyRjk05hpioXWmE36Z\\\\\\\/2wuhOAqXPsacdNOWOX8vGIVTyASPqaQDzMfk+PiWGYoE2cTn3JGxhxNIe9XjP9LESxtVSPfzqHbgFQAOG9GJOmiHP45arStsiwt0JfsIy66i38X+JG1juveGoArU9j2J7y3gSlSgwQh5EbI\\\\\\\/vXfV5EWzOc8QgsM4RcIEPXJH\\\\\\\/8dR1LBNlP2VdWsd8VjTHWkoxmFS6+8V2mjeikHS7PxTcNZ9ReBMT2n4FchUFAM7wNc7hUpWRcSfUWLdIFNpjGQ5GOWB5bAM1IySR3+ap5qp8fB0PSr91Yw9\\\\\\\/ECRWbRc2IyLyOPy6rzXzhhbs5xfkAjNWXOAaqYR95eAUlWY5Dhw50VTzIAnQV7CWf+zZhX9lEEL4FlPZCscsepQ7tmQakdXS6tYttHsN0BjpD8gVTnD8hg0EZv7D0SaPY9+KInvO4Eik6knwC7zXx3GtT+p5S+d2\\\\\\\/zibVMrqLJItzIHNWXZtL2avdnVzgCK9r7ROoIAa1lUTe4\\\\\\\/tYsQHdhy3qFlWXAxx5KwCTWzyehAIcPYBIlErCNR4z65L0XxtXQWZ2w9ircJsVfcM+ysHj4wC81uiSj+qtGNDmrT5TOGpy72oM5PSsvynR46NdI86jm9ej+wqNL\\\\\\\/PDVMrWk0WFuysaTbHfePN3gtPtBRiknO0ktmcxzmpXCsxz1\\\\\\\/qoMiQ52YJpRgpiQ5Sqv7BPQpOzxTBIWKWjCfj3cbkVuzxS6tpxVTBPXP7ZIoLIychYPPEcXHiMJ6OQ7k\\\\\\\/WDA5Fq0wcmsJ3Fe\\\\\\\/hRjQcYdPpucniYzWByb\\\\\\\/1a\\\\\\\/uFdU7G8Q4kIP8dy7K6WBTJx1VHiSPEstxaDI\\\\\\\/9ePHMAWQBvJfUqO2jheBbbR0CgqXQ7wC8+GeiWszzoPd4PXVg45V1RCZsKs5pex10otZwINQNTaMo6K2spTiYCGa0U8Va6zrnBnE6ORWdVXYR4ux8NY8Q\\\\\\\/j8NFLfdIuJlox77Wu1a9MSZhQ4Mej\\\\\\\/8vOoVc4FGcsdvSpKk81qmp4GN+NMCp0KxBVIz4J+9BgkFg9fOhLitqbUC1ths8XRVfjMjHTIYAaSBNUXd1ZYzQ8qzA9SHqzd4u9HF\\\\\\\/QPmX8G4cm5QUytkVEH92Xw4bYZzI10uQvIHabpxwXuBt817G8dV\\\\\\\/LbD8cORlztBaG70yezZ9NcJ0KcWEw9mkt2x3I+QGGS\\\\\\\/uYstWaxxju1PQY\\\\\\\/HG3nu4KrRzqhZanM4bgPbMOTNWVVM5DOIJYpLUWKCi2zEUEAhTovzxW5WBKuo9CcvWAVTeAW5zr7n1dIza+pqolOkD2nj2AKdtJe0BHlwy9in26hMH5Plr4HxUTrgnmrg2G2D5PatAIimDQlLKf6DoQVuFeLPLU8GYc+58cc5I3Fi3P9BdPh0lQi5SiRnQi8tEC0RUTO1GcyvINJ08XeOuJoOdkgkCyoBOEQS0r8ked6jXqtzBAi9HFQ5ANRLjkK+NxtVwnpeziWN4sDWt6W02Bnom5NPH0w0aA2HXbjO+gXkf2v47CGvwnTEV93UcO9djJRUjyCdjarAqA8BJWWoeqSoPiCLpvH+q+MttJRoVMWUzWoWfN35eIvnqmDtBzeICaS+6izHOLw5h8T9KK9+v+qlg\\\\\\\/d\\\\\\\/6xaVkTxX8yzChiw4DbSTxfbc+eXANRek6+rQAU+3yirYx9Igwc3soVkHi8KnISGxXIcWxxdMU0eAMxKXpJvdiAY2pvgy+pNeuvxOmTQf+GSbZOHEJbL8m1ZZheUehYxSsRHkAOa7+U+esRE4jQ8NAB4GxhB3n7AgRG5zHPwVtqVG0KW6sbfgkox184yc6bDVZPQAkv3ZrnA3DUFDkrR6iAulWBvhrQseACaDsT4YBalrbpS+xq65Muq2ebQDJhXs57B5eTnZLDYHKIgg7JGxypBzukiceY\\\\\\\/RYwjSMtaJKkwpnQEIxY1YRljKz65F0d83Z54wTwTt9\\\\\\\/lPzw3cfqsrCTsf6l+pvdcFHzQsP+3l790UcrxgDaMJstj2x4WGLu1kXO8486qHErlK\\\\\\\/KPtnG\\\\\\\/adK1NCUBugisAwT9p3sgc4lDrx5c7DQP71k04JhSLpxGTJJLQ9R2w+902tmNmXsdN16tyFqH+2KEti743uCSejA4YQ4Te5UT+X7p7jk6quzibf\\\\\\\/qdaMTmL3jZsX+sT6kvAVp+WIKd5O5wXpfACilr6yDOx6uHH5z8lqi4tSbLZwiDXUHCqX0QOGPQbGe\\\\\\\/wOq9hFM0k1zd2uLO87wVGmG+wwMTmYwOtIzFuwo8FMSHMyD3wHx2CbmOsNVM19zFbxhLukMB6NIH94y3QfIq0W3A29hBjDYaTR7lc+r54crE1434aRzXDIciGYcTd6qMwZujIvi2BEqWF05kxSddQYhfs+plSrR6j6rveP2YnRlGydT1fwhtuerzwwMZ54xrgkYy7XLtHs+dNoy+y99jOxN25fqKKePHhwua4CMwL6sPIpoaYHKEhDTIdrSQwliM+cd5WxtmyIIzOqGZuhc+PRNpubiO+XCxvE6ox3Mu6Ueph9Yso0byDjnwIINf8QGvNABWG5Ysxv571OdPSX+gBC8NOzLfEoqjIlZZ+TWDsKPUYEGMV34TdPu3y6f9vpI2Ou2jTG4c5ifJhZLWMi24NinpDLwVZYm4TQN5ScrA8dGmcMFjI0XRpF8xvUbnGt0v1a9PmnkrQgf6zrbVRoF1eg1eB\\\\\\\/Aw5H+JzLLxniyRGRG+7EUuMS5bmd08hFI\\\\\\\/OVr2ewsII5HdGBrfHQt9BdikVXH9ElbJTyDD\\\\\\\/47\\\\\\\/ZYGHb2efpuei3zR6kww5rluKRpmXWfnQuiLB4vf6JWZBBpgYDVl4s+tIbOdzlhiIkmv988cCL2fewbVTf7SqgT7R1qbnDEusZr3FEmA\\\\\\\/OBKb4ZAnWlh9qtFxZdjuTM8xzy2uhIGg9QtLKXwSJy5W\\\\\\\/g2yeQKkeAv6Bvm2HYrMzCdyc3jqDmxloFtVPVm0HQhHcUi4+u7AU6iJA1kkBnIUDeKb+P8w\\\\\\\/t\\\\\\\/Nvhb7sSQAdEruAl4IUgKGcRWJbKiTRVh+w75fWkTLRPXLYrR7zv9jQ5f1d8VV\\\\\\\/bYx\\\\\\\/ADN68O7qdz\\\\\\\/p+a3j1vOxqTBh3TQdYA75KmE+f6t\\\\\\\/KnqhNhV\\\\\\\/xRmW36VSFT6f+isUGnl9ENp5J5OkppiXF86EJ2fdggELqJukgGFnGnYk6KI7yOE8ZahZVKLmu4izdd1sZ1kjmxaxJO6kblQKktezN08gs0jTx+QV1mGMK+nmK8PMHUlCj6uJFqrJ2zURMKDrVuSqokqkrfXSNv\\\\\\\/ARgeTY3WEVpqktsEvjd\\\\\\\/NdIqm8QcaZsanX5dz6x8\\\\\\\/peCdQVh94AflsSeyTm0RyLjogPhkG6RioX3nLd8yQDEddusj7fVQr2eE65DD7GZUvgm+bgUrSIs6Ux\\\\\\\/ZF1Q0UMVwG5X6v+Sds9xiBjXz4Pssh0dt0+qOpGsiSk7fFyLKAYsF53H4JfzxuU3xbKYOn8NgoLHK2ecAR5J4UPEQ7GbeeOpS6fIN\\\\\\\/kdoSvdCvB5hF1k1PDffADnllocYe0IRSCDTFCbG8\\\\\\\/Oe+KYsuGD1K+iUHDapbKwQHTPlEeY1DT1GYmO2mu3jOt0fg5Lb3bM4EVA0b7TCBGU1ftinDfFdkRsqL1ZgHFZ7Rzgk+pTuvmViREiy+cbXXD5\\\\\\\/+Qw+cdUyQWYYRRWA5lhS+6n94GBgbclLCZ2G1lE6ffFZ7JrBBO486w9jNt5AfmDCprznzNL63B5mrhUyzG\\\\\\\/ahy8lLKKBHvhSMJeLXDSDE8eTj2lmDvEsg6MfKHHIHWlcksFodQS7X8RhSv20E+06rL87tiL3IFBi8s+cxos12HsZCRk1Lk8Akp7cX5jDaUIynveFfpyXio+GXcBudESN4JQwgCo80Jujp1p76A77kdyihzOlw+92kf3nCjpltzCeP1h6VQh57GIW5ERenF4+GidR3YrA12cBI1acon0tHHZbpXJPISTv+sUsGMcYj4Rmw7B04GjnlXu+tUUznjivXo3qJXrfaK3UnOrERFulBijUr7O+P\\\\\\\/I37EuEMG\\\\\\\/tI1aV2Tyb11aMF1f6sVcyBc0aKEThEa84tvEFReQR8quH3vDdEu5\\\\\\\/3QjApyGG\\\\\\\/Oy0ThZgjAfUYJ\\\\\\\/ryRcKf+RxujUWwLriByWqa\\\\\\\/i9GE2XCGnRZeZGj6gYoTehpmVqU1FiTmSnOS6oLfaR3TF6ACXAoPVxBqjXjDWNm051EOXVEXj8HRDx5deccVgOLPJ327XG7is9k8mXaHMMzdg9gMv\\\\\\\/rpgsTZcg8+PN9OB6jVIhL83g8fNCiQU8wwx9tVG5kempnk2pr9eOuucfuYQRsr1aMfAFh5W6VGaoNcFZWVew8p5MkD7WvG9M496aMUK6IgOnzW6KUrfnV6kO2U5pKKrVRc0haATxrDHWK\\\\\\\/TirGvTELabReNp+DqoaSQjMay98hggB03g8r7Rb3wtTamd8pZni9t9fZLdGqhq3tftuz16+Laig2jAQB9ltkoZPHb3wJxPnszv62RU7MNLOAufym1m62FWyPZuFJIZh80RPsJq1tFhMCKdGSrtb9sJosv38ZFa3cJQ9CW+dkqCgyu4f6v8TMP07sdXHVzxWDWbPNiQJ5DHu3RTMTJh+5+WPQBUbt03QQsALad\\\\\\\/sbACwul45\\\\\\\/X3dkL5G6w13UNTnNBrohgYIV\\\\\\\/YPxVvuJh\\\\\\\/+DDQh6scteT0O00+hANSO2RltUTZD2p6aXBgRgaZuRF04OMV2T\\\\\\\/i3UV6z\\\\\\\/hptto4iSOl\\\\\\\/OpVAitiT1jh\\\\\\\/7a95cVUIkOPKFvh7PoVBWQ2v9sO\\\\\\\/WqO9hsqH3jJwd3DosPzgUbCG1VlVb\\\\\\\/2Xnu1jD80QGZm5iE\\\\\\\/HKaz9KSD7PGiJmXFeI\\\\\\\/mi8F01\\\\\\\/yIJukg0gCsxKygJ35hgyy94jxWohQBPn1Xi3AhGc9lcE5pMMdr5H\\\\\\\/x0i1ZRXBNu\\\\\\\/MqUJXRSDaN\\\\\\\/AAwTfWgexJR0KnoexQzwuXaNLZ4NgFRyWEF2AqAUDCBbSKOPYt5s2O1jmrDFws2HOeVDDl0CBdlubTwk1rc2uILP44AfSB15nBozn1Basi5MTPOUF7bv7T7Awo6Nh7eKW\\\\\\\/GnFGW+UbZ+WJUIcM6u\\\\\\\/Wj0rbINxistMV0exjseYnsBuvBUom+sAeZsefaBrUMcQLkAEQvpUlNJcrtxPerNlnZrmKNtj6N53k8h9UgLNPJTkLCcNn\\\\\\\/9XztYAGwRwT9UDm+K2FDhO5rGiYILl3wNKrpN5UUeOo08CAAo3feo\\\\\\\/UoZ64sh5ZVUxtwo1B1MyBf7PUVHTCbtKnXcQBuxe4CmfNv4MQlC7rDS9Q3KvIo0JBRXR4bhxE+arl\\\\\\\/bwLSyQya0xrHH17fKfCZSaW+HObmKjU1xN5onWpY6srZ06CM9fV5nC5u4XUi7V7yqZktf5R1I1JX5LQrIqJZEW6b6almz2DzTxE1AVa07IoMGzbGpKQ\\\\\\\/4bioIS9WNiNqWNcrkXrNgTBKoyZ6TB9CaKQMYPkWwskzZqGZlFu7lIdquhHkoE8Og1gv5bCLnojBs\\\\\\\/GeT9GCItmAfRMvNxnOI9aIJL10\\\\\\\/BbTqdqLPQ7YrROMtBYpVfRFtAJ7ewfNd1QFH1zC18SV2qbMQSfunLEhDnw\\\\\\\/npqUAQVUJxg74Jjx9cUHBhKAN9Tm+1cteQBXbdxoaFTruua64ca+\\\\\\\/tbzhvJMl3f96RfOLTovaNPJWB\\\\\\\/D2dKL97QkFCOwSqUKgTsz5RccMp5qIBDtsA7R6WZj084J7YUcKLKKLwbs2oMylNAPNCyherlEyvAbRSvH7RVJvH5NNpPJ96lYYns7\\\\\\\/Yshzficx2ykgebF+XfLmJu8ToN5EU1HOxCwyAjB6GkVeWDvPGj7oJ53+2l+Z\\\\\\\/ez9VgwnoPGzWE6AQAq\\\\\\\/lnQKy94mUbJpiQszg8\\\\\\\/p4XAmJR6vF8m1pb3xMd85JLmQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aIhVu5ZFQUrSaVFwrqoxRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b7208e4aa6661fbdbaa62e215c4dbf047dfc343dc4bc1debb6d7ee7e156e2340\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L2Ebkn590qA6GbiLwYsj97JtTJOVzGd0N3oP74DEUE\\\\\\\/1Ickl0pBp3WKGWtvJTHUMJTJZkY3NPfkJeNa5MvGulsResTG6\\\\\\\/7cPMvkzCprQJ8+AqByS9k7Ft\\\\\\\/005w4zgMpEAMv\\\\\\\/Vu9ZD70O0+53AL\\\\\\\/1nxLM2uK\\\\\\\/YrrJs0ZWscZmQ0jGH\\\\\\\/jYooqylL5z6wy+1\\\\\\\/w9oLoDMFLPOxRZy2KlsZDcDc31Y2foF2lrfmxM5o1sKldz135O2m1U+Mb8OQvHg0Ccejs0oRbSVYQEcN8FaSgG3x1WAnbPTSb8tJ0PJ2AA1LRI3cCq6+f5VIyRUFCJb87hXPDWJBJg6cnNmmjPklUa8IH1brd5Niqdc\\\\\\\/7QIrl4cH7ac7TC4UtsC9uQReVhXhKnVsTDF3uTM7FuEXk6gqJNSwqlIWzzXVmXd5RV8zLoy3452aXKNh8kmql8z\\\\\\\/XCxYb5GrHCEZjulXpINOLMgPWKyBugL7d3SqqjmJVQ5G35VPwvl9AbH1Fa2\\\\\\\/G1kmj\\\\\\\/UrVV\\\\\\\/20zqWF5Tk+FDmsjcxwdUjxTFi9ywm9JV\\\\\\\/yghUieu9ytiaR9AAdlSVSHEHUURPOibW6ls2lC9CuCsj+C64s6sKwzFtiRO+otmIcTmXZEMpF0CLDRMiiFOpjJAC+ky1ncD30lpBEUXjAPou3VCM4huSW7jYNNQ2AB8awJfkFN88T2dDG3YVes8fzuVPCmajRbI22RnZgRXGPg7Gzuby+7h+x6e2pCIzOXitGrQoeXGKaseuhQqFFkGCilrp7Gd1GeqU7CjJQ1QDxQ2Gvy1nfy3cW1u\\\\\\\/SMZ+9kM0zmqQQVaJaurT1vRjD5ho4j3Z1bg2Y1AKkGFOUBDb0hDQUzd8GZJSI03OG4DyD3Sz6YAbh3kExzVsWBkmtAQElGwybRrP5FQT8TVemn0EzA7oetLPdeksmBcFTg7OV2l5hS+CSYhv9moojkhtZqjwd973KixCMbvXUpSs8RuHcj7hvAKC7ENmmM4rRwghRzFrfa0aYqXQ0ExBc6Qxb6YuJHAI1x2vi\\\\\\\/Y6Ww+3R+EQ9M9CPEYZhl0edZndIa4aqCTPg8PlV9ecYRa1AxmZXBz8fuk07uPNl96flP74UiqvWYP8nlK8NTARLlV6xweF0MEHvGwh1h7YyT9x6w41GQJNAIYw32st2U+Zc5YMZ4hvathzpKKXhJFe97iQkQFMX7pMuniao4ozzBtrp4qq86bAwsH+stop5egQVQbQf9bsAQk37lUIstImhmpp96NtLBhBvV1jFjY++kpVNAtHVSb3ZnYONjITCL6a7\\\\\\\/WAybTsde+o5VyJmjcsTwMP3UaYXdN0sfFVUS1AchLRHkpTWkz\\\\\\\/JSft1ZmTeTmhvFYy2LTbWjYdgCuSBaWLMeG\\\\\\\/cNCVgfZxuWBkhl1jg\\\\\\\/turqHIR7dJPXTAgAaghohXv\\\\\\\/hw7qHxMNQBuZ4Lda8CP3vraYir7RkC2q2qvsM+VL9jmQoljw013sHNx42PYBS\\\\\\\/j0dW1hV7UwdSqu4mYm5h\\\\\\\/agTGvUvoh16KwxF5P8rZiAkIMcNt6eoKu2wuUXvEJcWvea4kl2aWoEEmINinxvEmtIGcrMnysQ0aXXfX7SdC+dTS0J4QQ3kUj8wIfsrjOpvDJCEC\\\\\\\/VS\\\\\\\/BH4unb63uKju1VQFMCotvNoJTcGbO7EhADmxercqMxSZJ97Zg0ZxfIssuynnyNdCag\\\\\\\/S3bDNH0I15VheGDogem1OE82Smi+TY3iLJ0vyCjf4aCAEr392Br1DqyhHiavaLQtZB3ZwJejF6kCpVazQQk\\\\\\\/3feyCixKXeAX6PxPjro4C7wrCW\\\\\\\/HJR6sqZa2Lt8LWIaCY7guZJxVWqGd0ISfCLnTmWyNUTkkX\\\\\\\/2EZqiYkVFE9lu5atgFV2NTYMjqjLR7Sbn5wy3XnVxFePlxS+35Qdrhf412djaA2AUhJbrrbqE5BONdn2\\\\\\\/rKbOWZNuhEibPeaU7qKf5lGdtE1nDwUdS3uJa7T14dSqx+MhqnWcGcQDcKgbKF3fk9bhZMPOvolwoFhjMKF61nOkQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O7WKkYabCVigNmr2\\\\\\\/9xBXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9cb8136e6d6dc93095a59de487e2180bd60cc0ea8b013da1d4ab9b6c54f2266\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459816 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/447" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Ndx-TnlThV7495KcYJdPYDURnySwHjxSqOwgGoIvjLQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-b7Vf12O4axx8xmTtaC96eVwrEZ0sh3vbw1zjHuVgRA8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/433" title="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/461" title="The BAG Homology Domain of Snl1 Cures Yeast Prion [URE3] Through Regulation of Hsp70 Chaperones" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459816" id="node459816" data-pisa="ggg;4/3/447" data-pisa-master="ggg;g3.113.009316" data-apath="/ggg/4/3/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Linkage Map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="1">Charles D. Waters</span>, <span class="highwire-citation-author" data-delta="2">James E. Seeb</span> and <span class="highwire-citation-author" data-delta="3">Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459816" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbrieuc@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mbrieuc@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles D. Waters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BD.%2BWaters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waters%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BD.%2BWaters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/447" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459816" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459816/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/447.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459816" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459816/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/447.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459816" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459816/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/447.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459816" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459816/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/447.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459816" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459816/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-67850 first last"><a href="/content/4/7/1359" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - July 01, 2014</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Comparisons between the genomes of salmon species reveal that they underwent extensive chromosomal rearrangements following whole genome duplication that occurred in their lineage 58−63 million years ago. Extant salmonids are diploid, but occasional pairing between homeologous chromosomes exists in males. The consequences of re-diploidization can be characterized by mapping the position of duplicated loci in such species. Linkage maps are also a valuable tool for genome-wide applications such as genome-wide association studies, quantitative trait loci mapping or genome scans. Here, we investigated chromosomal evolution in Chinook salmon (<em>Oncorhynchus tshawytscha</em>) after genome duplication by mapping 7146 restriction-site associated DNA loci in gynogenetic haploid, gynogenetic diploid, and diploid crosses. In the process, we developed a reference database of restriction-site associated DNA loci for Chinook salmon comprising 48528 non-duplicated loci and 6409 known duplicated loci, which will facilitate locus identification and data sharing. We created a very dense linkage map anchored to all 34 chromosomes for the species, and all arms were identified through centromere mapping. The map positions of 799 duplicated loci revealed that homeologous pairs have diverged at different rates following whole genome duplication, and that degree of differentiation along arms was variable. Many of the homeologous pairs with high numbers of duplicated markers appear conserved with other salmon species, suggesting that retention of conserved homeologous pairing in some arms preceded species divergence. As chromosome arms are highly conserved across species, the major resources developed for Chinook salmon in this study are also relevant for other related species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-duplication-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-duplication" rel="nofollow">whole genome duplication</a></li><li class="kwd"><a href="/keyword/chromosome-homeologies" class="hw-term hw-article-keyword hw-article-keyword-chromosome-homeologies" rel="nofollow">chromosome homeologies</a></li><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/chinook-salmon" class="hw-term hw-article-keyword hw-article-keyword-chinook-salmon" rel="nofollow">Chinook salmon</a></li><li class="kwd"><a href="/keyword/rad-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rad-sequencing" rel="nofollow">RAD sequencing</a></li></ul><p id="p-4">Understanding the stabilization and fate of the polyploid genome following whole genome duplication (WGD) is fundamental to evolutionary analyses (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wolfe 2001</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Soltis <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mayfield-Jones <em>et al.</em> 2013</a>). While genome duplication immediately introduces additional genetic material that can serve as a template for evolutionary innovation, the effect of this variation on adaptation and divergence rates in descendent lineages is debated (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Taylor <em>et al.</em> 2001</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Donoghue and Purnell 2005</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hufton and Panopoulou 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mayrose <em>et al.</em> 2011</a>). Recent duplication is pervasive in plants (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Soltis <em>et al.</em> 2010</a>) and some animals (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mable <em>et al.</em> 2011</a>), but diversification rates in neopolyploids have been shown to be lower than that of related diploids (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Mayrose <em>et al.</em> 2011</a>). Nevertheless, comparative analyses in ancient polyploids (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ohno 1999</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch and Conery 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Canestroa <em>et al.</em> 2013</a>) have revealed a recurring role for WGD in the evolution of eukaryote genomes. Studying the genomes of mesopolyploids-“caught in the act” of diploidization (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Mayfield-Jones <em>et al.</em> 2013</a>)-can provide a basis for understanding the processes governing genome stabilization and the persistence of duplicated regions.</p><p id="p-5">Diploidization can be achieved through chromosomal rearrangements or losses, as well as through sequence deletions and divergence (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hufton and Panopoulou 2009</a>). These processes can act together to reduce the similarity between homeologous chromosomes (ohnologs), resulting in a gradual change from multisomic inheritance based on multivalent formations at meiosis to bivalent formations and disomic inheritance at two diverged loci (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Wolfe 2001</a>). The rate of diploidization is predicted to differ between allopolyploids and autopolyploids, because the changes induced by the latter are likely to be less dramatic (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sèmon and Wolfe 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Doyle <em>et al.</em> 2008</a>). Allopolyploids may attain diploid segregation earlier because the two original genomes are divergent, and thus retain progenitor contributions for longer. On the other hand, the genomes in autopolyploids are more compatible and are expected to display tetrasomic inheritance from the outset, losing parental alleles primarily through segregation. Thus, selection for diploidization may be lower in autopolyploids than in allopolyploids. One way to shed light on the processes of diploidization would be to compare the genomes of intermediate mesopolyploids that are descended from the same WGD event (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Mayfield-Jones <em>et al.</em> 2013</a>). Examining the relationship between chromosomal rearrangements and the distribution of duplicated loci in these lineages would reveal whether diploidization is uniform across the genome and between species.</p><p id="p-6">It is widely accepted that a third round (3R) of genome duplication occurred in the ray-finned fish lineage after two rounds (2R) of duplication in early vertebrates (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Postlethwait 2007</a>). Salmonid fish are descended from an autopolyploid ancestor that underwent an additional (4R) event (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>), recently estimated as occurring 58−63 million years ago (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crête-Lafrenière <em>et al.</em> 2012</a>). Partial diploidy has been restored in this lineage through chromosomal rearrangements and divergence of homeologous chromosomes. Chromosomes have evolved by inversions within the subfamily Thymallinae, whereas chromosome structure within the subfamilies Salmoninae (which includes <em>Salmo</em>, <em>Salvelinus</em> and <em>Oncorhynchus</em>) and Coregoninae has evolved through Robertsonian rearrangements (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ohno 1970</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Phillips and Rab 2001</a>). Extensive chromosomal studies and genome mapping efforts have shown that most chromosome arms are syntenic between species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lubieniecki <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Timusk <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guyomard <em>et al.</em> 2012</a>). The chromosome arm number (NF) has been conserved (around 100) within the Salmoninae subfamily, the exception being Atlantic salmon (<em>Salmo salar</em>) which has NF = 54−58 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Phillips and Rab 2001</a>). However, the number of chromosomes varies extensively between species, from 2n = 52−54 in Pink salmon (<em>O. gorbutscha</em>) to 2n = 84−86 in the Japanese char (<em>Salvelinus pluvius</em>), a result of differences in arm arrangements.</p><p id="p-7">Evidence of tetrasomic inheritance in Salmoninae supports the fact that restoration of diploidy is incomplete in this lineage (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allendorf and Danzmann 1997</a>). A model of secondary tetrasomy has been proposed, in which homologous chromosomes first pair in regions that are proximal to the centromere, followed by pairing between homeologs and recombination in the distal regions (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>). Such pairing results in a greater retention of duplicated loci in regions of the chromosome involved in ongoing recombination. Occasional homeologous pairing can also result in pseudolinkage (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>), characterized by an excess of nonparental progeny types in crosses and identified by observing linkage disequilibrium between physically unlinked loci using two-point linkage analysis (<em>e.g.</em>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>). In salmon species, homeologous pairing is thought to be limited to males (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Wright <em>et al.</em> 1983</a>; <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Allendorf and Thorgaard 1984</a>), with only a few instances inferred in females (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>). A recent linkage map based on single-nucleotide polymorphisms (SNPs) showed that duplicated loci were not randomly distributed among all chromosomes within Atlantic salmon (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>), suggesting that diploidization rates have not been uniform among homeologous pairs. However, the distribution of duplicated loci along each homeologous chromosome pair has yet to be described. Such information in another salmon species will reveal the rates of divergence along the chromosome, and a comparative analysis will identify whether chromosomal divergence is conserved across species descended from the WGD event.</p><p id="p-8">Linkage maps can facilitate genome-wide studies on diploidization that rely on chromosomal position, especially in species whose genomes have not been extensively described. Among salmonids, one such species is Chinook salmon (<em>O. tshawytscha</em>). The characterization of its genome will provide a useful comparison with the better described rainbow trout <em>O. mykiss</em> (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Guyomard <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>) and Atlantic salmon (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>). An existing linkage map comprising 361 microsatellite markers (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>) has been aligned to the 34 chromosomes described in Chinook salmon (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> 2013</a>). Comparative mapping using markers conserved between Chinook salmon and rainbow trout revealed that Robertsonian arm arrangements for 13 chromosomes preceded species divergence within the genus <em>Oncorhynchus</em>. An additional comparison with Atlantic salmon points to two conserved arm arrangements ancestral to the divergence of <em>Oncorhynchus</em> and <em>Salmo</em>. Since most Robertsonian fusions occur at the centromere (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Slijepcevic 1998</a>), determining the positions of the centromeres and increasing the numbers of markers on the Chinook salmon map will strengthen these comparisons and will facilitate an examination of divergence between homeologous arms. Centromere mapping has successfully been implemented in other salmonids (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> 1983</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allendorf <em>et al.</em> 1986</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lindner <em>et al.</em> 2000</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guyomard <em>et al.</em> 2006</a>) and is achieved by studying marker inheritance in gynogenetic diploid crosses (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Komen and Thorgaard 2007</a>). Examining the genomic distribution of markers that are recently diverged or are inherited tetrasomically can be challenging in diploids because the alleles at duplicated loci might not be fixed. However, the use of gynogenetic haploids solves this problem because offspring will only be heterozygous at duplicated loci.</p><p id="p-9">The recent and rapid improvement in sequencing technologies (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hudson 2008</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shendure and Ji 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Metzker 2010</a>) has facilitated the characterization of thousands of variable markers for species, or species with little or no available genetic information (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Davey <em>et al.</em> 2011</a>), such as Chinook salmon. Several of these approaches take advantage of the large amount of information afforded by sequencing a reduced portion of the genome. Restriction-site associated DNA (RAD) sequencing (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> 2008</a>; c.f. <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Miller <em>et al.</em> 2007</a>) targets a consistent portion of the genome across individuals. Interest for RAD sequencing has increased recently for salmon research (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Amish <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hecht <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Houston <em>et al.</em> 2012</a>) and is expected to provide a useful basis for comparative mapping across salmon species. Application of RAD sequencing to genome mapping in Chinook salmon provides an additional opportunity to develop analytical approaches relevant to mapping a species with polyploidy ancestry. The reliable assignment of short reads (60−100 nucleotides, typical of RAD sequencing) to loci that correspond across individuals could be resolved with the creation of a reference database of RAD loci for the species of interest, where duplicated loci would be identified. This database would rapidly facilitate alignment of newly sequenced individuals in related studies, and promote data sharing across research groups.</p><p id="p-10">Our overall aims are to describe the divergence of homeologous chromosome arms in Chinook salmon following a WGD and to compare these findings with those of other salmon to determine whether the processes of diploidization are consistent across species. Our specific objectives are first to construct a reference database of RAD sequences that can be used for alignments of sequences generated in future projects. Second, we will improve the genomic map for Chinook salmon by populating the existing genome map with thousands of RAD markers from the reference database and identify chromosome arms by mapping centromere locations. Third, we will examine divergence of homologous arms by mapping duplicated loci. Finally, we will improve our current understanding of chromosome arm rearrangement between Chinook salmon and rainbow trout using comparative analyses of marker-dense maps for the two species.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection and creation of mapping families</h3><p id="p-11">We used four data sets to develop genomic resources for Chinook salmon; RAD sequences from individuals sampled across a broad geographic range for the reference database, gynogenetic haploid crosses for mapping single and duplicated loci, gynogenetic diploid crosses for placement of the centromere on linkage groups, and a diploid cross for aligning the RAD-based map with the previously identified chromosome arms (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> 2013</a>).</p><p id="p-12">First, 159 individuals from a total of 10 populations from the Columbia River basin, Pacific Northwest, USA were sampled for the preliminary identification of RAD loci and creation of a reference database. Second, three haploid crosses comprising 46, 48, and 72 individuals per family were used to identify duplicated loci in the database and construct the initial linkage map. In haploid offspring, all unique loci will be homozygous; polymorphic duplicated loci will be heterozygous. The haploid crosses were created at the Cle Elum Supplementation and Research Facility by fertilizing eggs with ultraviolet-irradiated milt following the protocol of <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> (1983)</a> and sampled before hatching. Whole embryos were collected and stored in ethanol. Third, we used three gynogenetic diploid families, created at the University of Washington hatchery facility, to map the centromere on each linkage group. The second polar body is retained during the creation of gynogenetic diploid progeny. Therefore, a progeny will be heterozygous at a locus if a crossover event occurred in the female parent between a given marker and the centromere during meiosis I. The percentage of heterozygous offspring at a locus is expected to be 0% at the centromere, increasing to 100% in the telomeric region, because salmonids exhibit complete to near complete interference and typically have one crossover event per chromosome arm (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> 1983</a>). Eggs were fertilized with ultraviolet-irradiated milt and subsequently heat shocked to retain the second polar body (<a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> 1983</a>). Fish were harvested as parr and stored in 100% ethanol. We sampled the dam and 84, 90 and 93 progeny from each gynogenetic diploid family. Finally, we sampled 44 F2 progeny from the diploid cross of <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> (2013)</a> to verify linkage group and chromosomal representation based on the microsatellite markers mapped previously, and to align the maps with the 34 chromosomes identified in <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> (2013)</a>. Recombination rates vary between the sexes in salmonids (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Moen <em>et al.</em> 2004</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">McClelland and Naish 2008</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Moen <em>et al.</em> 2008</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>) we therefore mapped the female meiosis in the diploid cross to obtain accurate marker order.</p></div><div id="sec-3" class="subsection"><h3>DNA extraction and sequencing</h3><p id="p-13">Genomic DNA was extracted using the DNeasy extraction kit (QIAGEN, Valencia, CA) following the manufacturer’s instructions. Each sample was prepared for RAD sequencing, using <em>Sbf</em>I as a restriction enzyme and six-nucleotide individual-specific barcodes, as described in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Baird <em>et al.</em> (2008)</a>. Twelve to 48 individuals were then sequenced per lane on an Illumina platform (GAII or HiSeq) using 100-nucleotide single-read sequencing. Reads were sorted per individual and barcodes were removed using <span class="sc">process_radtags</span> implemented in <span class="sc">Stacks</span> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Catchen <em>et al.</em> 2011</a>). The last 20 nucleotides were subsequently trimmed because the last 20 base pairs of the sequence had a consistently lower quality. For the purpose of this study, we defined a locus as a 74-nucleotide RAD sequence.</p></div><div id="sec-4" class="subsection"><h3>Reference database of RAD loci</h3><p id="p-14">Creation of the reference database of RAD loci was carried out using three steps: the generation of a preliminary database of loci for Chinook salmon, the screening of the preliminary database for loci in repetitive regions and loci with repeat sequences, and the identification of duplicated loci.</p><p id="p-15">Reads for all diploid individuals were sorted into polymorphic and monomorphic loci <em>de novo</em> using <span class="sc">Stacks 0.9995</span> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Catchen <em>et al.</em> 2011</a>) with a minimum of three nucleotide mismatches between loci within an individual. We retained a consensus sequence for every locus that had been sequenced with a depth greater than 5X in more than 135 individuals (85%) as a temporary database of loci: these loci were used for further screening.</p><p id="p-16">The screening steps were aimed at identifying repetitive loci and loci with tandem repeat units, such as microsatellites and minisatellites. We used two alignment-based strategies. First, loci within the temporary database were aligned against themselves using <span class="sc">Bowtie</span> 0.12.9 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Langmead <em>et al.</em> 2009</a>), allowing up to three nucleotide mismatches. We expect that most homeologous loci with three or fewer mismatches between the paralogs would have been identified as a single locus during the creation of the preliminary database. Paralogs with more than three mismatches would not be detected using the <span class="sc">Bowtie</span> alignment criteria we used here. Therefore a locus that aligned to several loci, including itself, was likely a repeat sequence and was excluded from the database. Second, a <span class="sc">Blast</span> search (Basic Local Alignment Search Tool; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Altschul <em>et al.</em> 1990</a>) of the temporary database was conducted against itself using the low-complexity filter implemented in the search algorithm. This filter masks regions of low complexity, such as repeat nucleotides or motifs, within the query sequence. When this filter is used, a <span class="sc">Blast</span> search that compares sequences with low complexity with themselves will rarely return a match or might return a match with another sequence, because the flanking sequence will be short. Therefore, loci within the database that did not return a match or where the best match (E-value less than 10<sup>−15</sup>) for a given locus was not itself were discarded from the temporary database.</p><p id="p-17">Finally, polymorphic duplicated loci were identified using the three haploid families. Reads for all the haploid individuals were sorted into loci by alignment to the temporary database using <span class="sc">Bowtie.</span> Individual reads from the haploids that aligned to more than one locus in the database could not be confidently relied upon in further analyses, and so were removed from the database. Loci with a depth of less than 10 reads for an individual were discarded for that individual. Genotypes for each individual were obtained using <span class="sc">Stacks,</span> which uses a maximum likelihood approach to identify polymorphisms (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Catchen <em>et al.</em> 2011</a>). The presence of a single individual with a heterozygous genotype at a locus within each haploid family was considered as insufficient evidence for duplication, as this genotype could be the result of a potential sequencing error, and the locus was retained in the database. However, if more than one haploid individual was heterozygous at a locus, this locus was identified as being duplicated, since the same error occurring in two individuals was viewed as unlikely. We did not weigh these choices by family sizes, because the recurrence of a heterozygote genotype caused by sequencing error was deemed unlikely, regardless of number of offspring. This final step provided the final database of RAD loci for Chinook salmon, against which all further alignments were made.</p></div><div id="sec-5" class="subsection"><h3>Linkage mapping</h3><div id="sec-6" class="subsection"><h4>Genotyping:</h4><p id="p-18">Genotypes at every non-duplicated polymorphic locus in the haploid crosses were identified during the creation of the reference database. Duplicated markers identified in the haploids (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) during database development were used for mapping when one of the paralogs was polymorphic (one paralog polymorphic, parental genotype aa and ab, or aa and bc) or when both paralogs were polymorphic for different alleles (both paralogs polymorphic, parental genotype ab and ac and ab and cd). We also observed loci with ab and ab parental genotypes, but did not map these loci because heterozygous offspring were uninformative.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464093/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464093/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Types of duplicated loci encountered in this study, expected segregation ratio per paralog, and expected segregation ratio when both paralogs are analyzed as a single locus, which is the case in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-20">All the reads for the diploid cross and gynogenetic diploid crosses were aligned to the reference database using <span class="sc">Bowtie,</span> and the polymorphic loci were identified with <span class="sc">Stacks.</span> Stacks uses a maximum likelihood approach to determine whether a polymorphism in an individual is true, or whether it is due to a sequencing error (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hohenlohe <em>et al.</em> 2010</a>). This approach can be biased against the designation of heterozygous genotypes for individuals that differ in sequence depth between the two alleles. To correct this bias, we developed a Python script (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/009316SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS1.py">File S1</a>) that called a heterozygote if both verified alleles had a depth of more than two and the total read depth at the locus was 10X or greater; this was the minimum depth we designated previously. Parental haplotypes for loci following Mendelian inheritance in the diploid cross were determined using linkage relationships with the previously mapped microsatellite markers.</p><p id="p-21">Finally, we used 5′-nuclease genotyping as in <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Seeb <em>et al.</em> (2011)</a> to screen and map 384 SNPs that originated from other labs (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Smith <em>et al.</em> 2005a</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">b</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Campbell and Narum 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abadia-Cardoso <em>et al.</em> 2011</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Larson <em>et al.</em> 2014</a>) in two haploid families. Many of these loci are polymorphic expressed sequence tags (ESTs) that are used in conservation and management applications for Chinook salmon across Pacific North America (<em>e.g.</em>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Smith <em>et al.</em> 2005c</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hess <em>et al.</em> 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Templin <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Matala <em>et al.</em> 2012</a>).</p></div><div id="sec-7" class="subsection"><h4>Linkage group construction and alignment with Chinook chromosomes:</h4><p id="p-22">We used Onemap 2.0-3 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Margarido <em>et al.</em> 2007</a>) for genome mapping in the haploid crosses and the F2 diploid cross. The Chinook salmon karyotype comprises 34 pairs of chromosomes (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Phillips and Rab 2001</a>). We therefore predicted 34 linkage groups per mapping cross. Linkage groups were identified independently for each haploid and diploid family using Onemap with a maximum recombination fraction of 0.25 and a starting LOD of 3.0. This LOD was subsequently increased by increments of 1.0 until the number of linkage groups identified was 34 or greater. We then used the microsatellite markers previously mapped and the RAD loci polymorphic in the diploid cross and the haploid crosses to identify each chromosome. Markers on each linkage group were subsequently ordered using O<span class="sc">nemap</span> for each haploid family. Individual haploid maps were merged using M<span class="sc">erge</span>M<span class="sc">ap</span> (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Wu <em>et al.</em> 2011</a>) to create a consensus map.</p></div><div id="sec-8" class="subsection"><h4>Centromere mapping:</h4><p id="p-23">We estimated the proportion of heterozygous progeny in each gynogenetic diploid family at every non-duplicated marker mapped on the haploid map and polymorphic in the gynogenetic diploid crosses. This information was used to identify the centromere and the chromosome type (acrocentric or metacentric) for each haploid family. Comparison with the diploid map was used to characterize the short (p) arm and long (q) arm for each chromosome as defined in <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> (2013)</a>.</p></div></div><div id="sec-9" class="subsection"><h3>Analysis of the properties of the Chinook salmon linkage groups</h3><div id="sec-10" class="subsection"><h4>Frequency of recombination:</h4><p id="p-24">Recombination is usually reduced around the centromere in most species (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nachman 2002</a>) and in the telomeric regions in the female in salmonids (<a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>). Reduced recombination will result in a high number of loci mapping to the same position. Here, we examined the distribution of the markers along the linkage groups relative to the center of the centromere to determine recombination frequency.</p></div><div id="sec-11" class="subsection"><h4>Crossover frequency and interference:</h4><p id="p-25">Salmonids are thought to exhibit complete to near-complete interference (<a id="xref-ref-74-5" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> 1983</a>). We estimated the number of crossover events per chromosome arm using <span class="sc">Linkmfex</span> 2.3 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danzmann 2005</a>). Metacentric linkage groups were divided in two chromosome arms. For each chromosome arm we counted the number of progeny with 0, 1, or more crossovers. Absence of double crossovers on all chromosome arms for every progeny would confirm the hypothesis of complete interference.</p></div><div id="sec-12" class="subsection"><h4>Distribution of duplicated markers across the genome:</h4><p id="p-26">Two types of duplicated markers were used in this study. Duplicated loci with both paralogs polymorphic (BPP) were used to infer homeologies, because both paralogs could be mapped (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Occasional homeologous chromosome pairing in salmon may result in reduced divergence between the arms involved. We examined the position of the duplicated loci on the consensus haploid map to determine whether there was a bias in distribution of these loci. We reasoned that this analysis would identify chromosomal regions of reduced divergence between homeologs, indicating possible map positions where homeologs have a tendency to pair. Here, we estimated the relative proportion of duplicated loci along the linkage groups. Because map positions are not uniformly distributed along the chromosomes, we used a kernel smoothing sliding window approach with a bandwidth of 2cM to determine the relative proportion of duplicated loci along the linkage groups.</p></div></div><div id="sec-13" class="subsection"><h3>Comparative mapping with rainbow trout</h3><p id="p-27">To examine differences in chromosomal arrangement between Chinook salmon and rainbow trout, we aligned the 40,649 RAD loci identified in the latter species (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>) with the reference dataset of loci created for Chinook salmon. To achieve this goal, we used <span class="sc">bowtie</span>, allowing a maximum of three nucleotide mismatches per locus. Mapped loci in common between the two species were used to identify homologies between rainbow trout and Chinook salmon and confirm alignment with previous studies (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> 2013</a>).</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Reference database of RAD loci for Chinook salmon</h3><p id="p-28">A total of 62,249 putative loci were sequenced in at least 135 individuals from the Columbia River with a minimum depth of five reads per locus per individual: these sequences formed the temporary database of RAD loci. Of these, 2713 were removed because they did not align uniquely to themselves. After conducting a <span class="sc">Blast</span> search of the temporary database against itself, 1451 loci did not have a BLAST result or the best hit was not itself, mostly due to the presence of repetitive units in the sequence (data not shown). Alignments of all reads for the haploid individuals against the updated temporary database were not unique for 3148 loci and these were therefore removed from the database. Finally, 6409 duplicated loci were identified as heterozygous in more than two progeny in at least one haploid family and were identified as such. The final reference database comprised 48,528 putative non-duplicated loci and 6409 duplicated loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS2">File S2</a>).</p></div><div id="sec-16" class="subsection"><h3>Linkage mapping</h3><div id="sec-17" class="subsection"><h4>Haploid and diploid linkage maps:</h4><p id="p-29">The three haploid families (here, family A, B, and C) had 3528, 3325, and 3403 biallelic polymorphic RAD loci respectively, representing 7146 unique RAD loci. Two families were genotyped using the 384 5′-nuclease panel (family A and B); each had 98 and 92 polymorphic SNPs respectively, 153 of which were unique. We used 2674 informative biallelic RAD loci scored in the diploid cross to develop sex-specific linkage groups. A subset of 1189 loci was polymorphic in the female parent and linked to previously mapped microsatellite markers (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>). We identified 34 linkage groups corresponding to the chromosomes for each haploid cross using 578 RAD loci that were in common between the diploid and haploid families.</p><p id="p-30">We mapped 3485, 3291, and 3273 non-duplicated and duplicated markers within each of the three haploid families (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The map lengths ranged between 2834.9 cM and 3099.6 cM (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). A total of 2319 loci were polymorphic in more than one family and were used to merge the haploid maps. The consensus haploid map comprised 7304 markers and measured 4163.9 cM (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS3.xlsx">File S3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464153/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464153/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of loci mapped and map length for each haploid family and for the consensus map</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Consensus Chinook salmon female linkage map&#x2014;34 linkage groups corresponding to the 34 Chinook salmon chromosomes. Ots01 to Ots16 are metacentric; Ots17 to Ots34 are acrocentric. The size of each linkage group varies from 58 to 173.2 cM. Each line corresponds to the location of one or more markers. The centromere is represented in pink. All the chromosomes are oriented with the shorter arm (p arm) before the centromere, longer arm (q arm) after the centromere." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Consensus Chinook salmon female linkage map&#x2014;34 linkage groups corresponding to the 34 Chinook salmon chromosomes. Ots01 to Ots16 are metacentric; Ots17 to Ots34 are acrocentric. The size of each linkage group varies from 58 to 173.2 cM. Each line corresponds to the location of one or more markers. The centromere is represented in pink. All the chromosomes are oriented with the shorter arm (p arm) before the centromere, longer arm (q arm) after the centromere.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F1.medium.gif" width="440" height="108"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/447/F1.medium.gif" width="440" height="108"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464173" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Consensus Chinook salmon female linkage map—34 linkage groups corresponding to the 34 Chinook salmon chromosomes. Ots01 to Ots16 are metacentric; Ots17 to Ots34 are acrocentric. The size of each linkage group varies from 58 to 173.2 cM. Each line corresponds to the location of one or more markers. The centromere is represented in pink. All the chromosomes are oriented with the shorter arm (p arm) before the centromere, longer arm (q arm) after the centromere.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The diploid map comprised 1101 non-duplicated RAD markers and 242 microsatellite loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS3.xlsx">File S3</a>). All 34 chromosomes were identified, but five chromosomes were represented by 2 linkage groups each (Ots08, Ots15, Ots19, Ots26, and Ots29). The number of individuals scored per locus was variable due to lower DNA quality. As a result, the marker order on the female diploid map was not consistently reliable. Therefore, the diploid map was not merged with the haploid maps. However, the microsatellite markers proved reliable in assigning linkage group arms to chromosomes.</p></div><div id="sec-18" class="subsection"><h4>Centromere mapping:</h4><p id="p-34">Of the 6348 non-duplicated RAD markers placed on the haploid map, 3021 were polymorphic in at least one of the three gynogenetic diploid crosses and were used to identify the centromeres. Placement of the centromere permitted identification of 16 metacentric linkage groups (Ots01 - Ots16) and 18 acrocentric linkage groups (Ots17 to Ots34), corresponding to the known Chinook salmon karyotype (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS4.pdf">File S4</a>). The small (p) chromosome arm of acrocentric chromosomes (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Phillips and Rab 2001</a>) is usually uncharacterized in mapping studies because there are often insufficient markers describing this region. In this study, we identified the small arm for three acrocentric chromosomes (Ots19, Ots20, and Ots33; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). It is interesting to note that the linkage map sizes did not correlate with the sizes of the chromosomes, but the metacentric linkage groups (Ots01 to Ots16) were longer than the acrocentric linkage groups (Ots17 to Ots34). Ots19, Ots20, and Ots33 were the longest acrocentric linkage groups.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of heterozygous offspring in the gynogenetic diploid crosses along three acrocentric chromosomes where the p arm has been identified: Ots19, Ots20, and Ots33. On the x-axis, the distances are oriented from the p arm. Three gynogenetic crosses were used (F978, F981, and F984). The centromere is located where the percentage of heterozygous offspring is about zero." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentage of heterozygous offspring in the gynogenetic diploid crosses along three acrocentric chromosomes where the p arm has been identified: Ots19, Ots20, and Ots33. On the x-axis, the distances are oriented from the p arm. Three gynogenetic crosses were used (F978, F981, and F984). The centromere is located where the percentage of heterozygous offspring is about zero.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F2.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/447/F2.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464072" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Percentage of heterozygous offspring in the gynogenetic diploid crosses along three acrocentric chromosomes where the p arm has been identified: Ots19, Ots20, and Ots33. On the x-axis, the distances are oriented from the p arm. Three gynogenetic crosses were used (F978, F981, and F984). The centromere is located where the percentage of heterozygous offspring is about zero.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-19" class="subsection"><h3>Analysis of the properties of the Chinook salmon linkage groups</h3><div id="sec-20" class="subsection"><h4>Frequency of recombination:</h4><p id="p-36">The distribution of markers across all chromosomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) revealed a bias in marker placement. The greatest numbers of mapped loci were placed at the centromeres and toward the telomeres; the number of markers increased with increasing distance from the centromere regardless of the type of chromosome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This over-representation of markers at distal positions suggests that there is reduced recombination in the telomeres relative to the remaining chromosomal regions in the female.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker distribution across all chromosome arms, examined separately for all chromosomes, metacentric chromosomes, and acrocentric chromosomes (including the p arm or not for Ots19, Ots20, and Ots 33). Distances on the x-axis are represented as the relative distance from the center of the centromeric region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker distribution across all chromosome arms, examined separately for all chromosomes, metacentric chromosomes, and acrocentric chromosomes (including the p arm or not for Ots19, Ots20, and Ots 33). Distances on the x-axis are represented as the relative distance from the center of the centromeric region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F3.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/3/447/F3.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464169" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Marker distribution across all chromosome arms, examined separately for all chromosomes, metacentric chromosomes, and acrocentric chromosomes (including the p arm or not for Ots19, Ots20, and Ots 33). Distances on the x-axis are represented as the relative distance from the center of the centromeric region.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h4>Crossover frequency and interference:</h4><p id="p-38">We used one haploid family (Family A) with 46 progeny to examine the number of crossovers in 50 chromosome arms (2300 chromosome arms). We only observed 60 instances (2.6%) of double crossovers. The occurrences of double crossovers were not randomly distributed between chromosomes. The chromosomes with the highest frequency of double crossovers were acrocentric. Double crossovers occurred in Ots19, Ots20, and Ots33, for 10, 6, and 6 progeny respectively. However, the second crossover always occurred on the short arm of these three chromosomes. The remaining double crossovers occurred on 21 chromosome arms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1/FileS5.xlsx">File S5</a>). Finally, the frequency of double crossovers was not correlated to the number of duplicated loci on the linkage groups (<em>t</em>-test <em>P</em>-value: 0.42).</p><p id="p-39">In gynogenetic diploid progeny, the maximum proportion of heterozygotes (MPH) at a locus in the telomeric region should be 0.67 if there is no interference and 1.00 if there is complete interference (<a id="xref-ref-74-6" class="xref-bibr" href="#ref-74">Thorgaard <em>et al.</em> 1983</a>). Here the average MPH for each chromosome arm was 0.90. The MPH ranged from 0.75 to 0.99, except for Ots11p, where the MPH was 0.49. Here, it was only possible to genotype the non-duplicated loci in the gynogenetic diploids. Given that the distal regions from the centromere of 16 chromosome arms mainly comprised duplicated loci, we did not have full coverage of these arms in the gynogenetic diploid crosses. Indeed, we observed that all the arms with the lowest MPH (&lt;0.85) had a greater proportion of duplicated loci. Therefore, we concluded that the lower MPH observed for those arms was due to a lack of coverage with the gynogenetic diploid rather than absence of interference.</p></div><div id="sec-22" class="subsection"><h4>Duplicated loci and homeologies:</h4><p id="p-40">A total of 799 duplicated loci detected by RAD sequencing were placed on the linkage map. The duplicated loci were not distributed uniformly between the chromosomes. We observed two categories of chromosome arms: those with very few duplicated loci (1−7 loci, corresponding to 0.5–5.6% of all duplicated markers in the data) and those with many duplicated loci (17−62 loci, corresponding to 15–61% of all markers). A total of 89.7% of the duplicated loci were located on 16 chromosome arms (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Homeologies were inferred between eight pairs of chromosome arms using 98 paralogs that were polymorphic at both loci (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Six of the homeologies had been identified by <a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> (2013)</a>, but two were novel (Ots01q/06q and Ots07p/14p). Ots01q and Ots06q had the lowest number of duplicated markers: 15% and 24% of the markers mapping to these linkage groups respectively were duplicated. All other chromosome arms had between 35% and 61% loci that were duplicated. Finally, the duplicated loci were not evenly distributed along the 16 chromosome arms that had a greater number of these loci. The regions distal from the centromere almost exclusively comprised duplicated loci (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of markers on each linkage group, designated by chromosome arm. Non-duplicated loci (RAD loci or SNP loci) are represented by the white bars; Duplicated loci are represented by the light gray bars (loci with only one paralog polymorphic) or dark gray bars (both paralogs polymorphic)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of markers on each linkage group, designated by chromosome arm. Non-duplicated loci (RAD loci or SNP loci) are represented by the white bars; Duplicated loci are represented by the light gray bars (loci with only one paralog polymorphic) or dark gray bars (both paralogs polymorphic).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F4.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/3/447/F4.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464138" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Number of markers on each linkage group, designated by chromosome arm. Non-duplicated loci (RAD loci or SNP loci) are represented by the white bars; Duplicated loci are represented by the light gray bars (loci with only one paralog polymorphic) or dark gray bars (both paralogs polymorphic).</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464086/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464086/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Homeologous chromosome pairs identified for Chinook salmon, Atlantic salmon, or both</span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of duplicated and non-duplicated loci along the 16 linkage groups (denoted by chromosome number) with a high number of duplicated loci. Non-duplicated loci are represented in white; Duplicated loci are represented in gray (loci with one paralog polymorphic) or in dark gray (loci with both paralogs polymorphic). The centromere is represented by the cross-hatched area. All chromosomes are oriented with the short arm (p) on the left where relevant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proportion of duplicated and non-duplicated loci along the 16 linkage groups (denoted by chromosome number) with a high number of duplicated loci. Non-duplicated loci are represented in white; Duplicated loci are represented in gray (loci with one paralog polymorphic) or in dark gray (loci with both paralogs polymorphic). The centromere is represented by the cross-hatched area. All chromosomes are oriented with the short arm (p) on the left where relevant.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F5.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/3/447/F5.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464163" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Proportion of duplicated and non-duplicated loci along the 16 linkage groups (denoted by chromosome number) with a high number of duplicated loci. Non-duplicated loci are represented in white; Duplicated loci are represented in gray (loci with one paralog polymorphic) or in dark gray (loci with both paralogs polymorphic). The centromere is represented by the cross-hatched area. All chromosomes are oriented with the short arm (p) on the left where relevant.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-23" class="subsection"><h3>Comparison with rainbow trout</h3><p id="p-45">A total of 40,649 RAD loci have been described in rainbow trout (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>). More than 50% of these loci (20,436) aligned uniquely to the non-duplicated markers in the Chinook salmon reference database. A total of 317 RAD loci mapped in both species, allowing us to confirm previously described homologies between the two species (<a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We confirmed the speculation in <a id="xref-ref-54-8" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> (2013)</a> that Ck04 (Ots16) is homologous to rainbow trout linkage groups Omy11p and Omy09q, a finding also in agreement with the observations in <a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> (2013)</a>. These earlier studies showed that Ots16p and Ots16q are homologous to a portion of Omy11p and Omy9q respectively. Here we observed one marker from Omy11 on Ots16q (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We were not able to compare the order of the RAD loci between the rainbow trout map and the Chinook salmon map because most of the markers polymorphic in both species on a linkage group mapped to a single position on the rainbow trout map, which is based on a androgenetic doubled haploid cross.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464021/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464021/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <p id="p-46" class="first-child">Homologies between Chinook salmon and rainbow trout chromosome arms and number of RAD markers supporting the homologies in this study</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/447/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage map for Ots16, denoting loci that are homologous with rainbow trout. Loci in common between the two species are underlined and the position on the rainbow trout map is indicated on the left of the chromosome (WS: linkage groups from Miller et al. 2012; Omy: rainbow trout chromosomes). The centromere is represented in gray. The chromosome is oriented with the p arm on top and the q arm at the bottom. Distances are in centimorgans." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-45896087" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage map for Ots16, denoting loci that are homologous with rainbow trout. Loci in common between the two species are underlined and the position on the rainbow trout map is indicated on the left of the chromosome (WS: linkage groups from Miller &lt;em&gt;et al.&lt;/em&gt; 2012; Omy: rainbow trout chromosomes). The centromere is represented in gray. The chromosome is oriented with the p arm on top and the q arm at the bottom. Distances are in centimorgans.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/447/F6.medium.gif" width="289" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/3/447/F6.medium.gif" width="289" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/447/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/447/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464065" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Linkage map for Ots16, denoting loci that are homologous with rainbow trout. Loci in common between the two species are underlined and the position on the rainbow trout map is indicated on the left of the chromosome (WS: linkage groups from <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>; Omy: rainbow trout chromosomes). The centromere is represented in gray. The chromosome is oriented with the p arm on top and the q arm at the bottom. Distances are in centimorgans.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-49">The aim of this study was to characterize the evolution of Chinook salmon chromosomes relative to that of other salmonids following a whole duplication event in a common ancestor. This goal was achieved by improving the genomic resources for the species and by performing comparative mapping. We have developed a reference database of RAD loci for Chinook salmon comprising 48,528 non-duplicated loci and 6409 known duplicated loci. We identified 7151 polymorphic RAD loci in three haploid families that were used, along with 153 SNP loci currently used in conservation and management studies, to create a consensus map with a length of 4163 cM. The map comprised 34 linkage groups, which were anchored to all Chinook salmon chromosome arms using microsatellite loci that have been physically mapped in previous studies. The placement of 799 duplicated loci on the linkage map revealed an uneven distribution of these loci across chromosomes, suggesting that homeologs diverged at different rates following whole genome duplication. Crossover frequency measured in one haploid family confirmed near complete interference across chromosome arms. Finally, the genome map supports previously published homologies among rainbow trout and Chinook salmon chromosome arms, but these homologies are supported using more extensive data and centromere placement.</p><p id="p-50">The reference database of RAD loci in Chinook salmon is extensive and provides a resource against which future RAD sequences generated using <em>Sbf</em>I as a restriction enzyme can be aligned. Markers that were polymorphic in the mapping families have been annotated in the database by chromosome arm. We attempted as far as possible to identify loci that had repeat units or were located in repeat regions using a series of screening tests based on self-alignment. However, the use of three haploid families would not have identified paralogs in the database that were not polymorphic. Therefore, we recommend aligning initial sequences generated in future studies to the reference database, and treating a locus that aligns to more than one of the reference loci as a putative duplicate.</p><p id="p-51">The coverage of the Chinook salmon linkage map, 7304 markers that span all 34 chromosomes, is comparable to published maps in other salmonids. For example, the map for Atlantic salmon (<em>Salmo salar</em>) comprises 5650 SNPs (<a id="xref-ref-36-11" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>), for rainbow trout 4563 RAD markers (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>) and for sockeye salmon 1672 RAD markers (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Everett <em>et al.</em> 2012</a>). The present map had a size of 4163.9 cM, which is significantly larger than the first generation map available for Chinook salmon (2206.2 cM for the sex average map; <a id="xref-ref-54-10" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>). The distances differ from the most recent maps for Atlantic salmon (2402.3 cM for females and 1746.2 cm for males; <a id="xref-ref-36-12" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>) and rainbow trout (3600 cM; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Guyomard <em>et al.</em> 2012</a>). There are three reasons that might explain the differences. Nonrandom missing values (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jorgenson <em>et al.</em> 2005</a>) and genotyping errors (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hackett and Broadfoot 2003</a>) can inflate map distances. We found that missing values were not randomly distributed across individuals (χ<sup>2</sup> test for uniform distribution across individuals: P-value ~0 for each family). We also noted earlier that genotyping RAD markers may be biased against heterozygotes. As a result, the genotyping error for the duplicated loci can be predicted to be higher for such loci. Finally, the extensive addition of duplicated markers at the telomeres might have increased map length. On the other hand, we also acknowledge that relatively few individuals were mapped (four families with 44−72 offspring). While this study demonstrates the possibility of constructing dense and high-resolution maps with relatively small sample sizes, increasing the number of individuals per family and number of families will result in better mapping resolution, with fewer loci mapping to the same position. While the Atlantic salmon map of <a id="xref-ref-36-13" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> (2011)</a> comprised 3297 fish from 143 families, our sample sizes are comparable to other high-density salmon maps [<em>e.g.</em>, rainbow trout, two families of 60 individuals (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Guyomard <em>et al.</em> 2012</a>) and one family of 123 individuals (<a id="xref-ref-50-7" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> (2012</a>); sockeye salmon, one family of 96 individuals to create the initial RAD-EST linkage map, and an additional 13 families with 45 or 93 individuals to increase the number of EST loci (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Everett <em>et al.</em> (2012</a>)].</p><p id="p-52">We located the centromere for 18 acrocentric chromosomes and 16 metacentric chromosomes, using more than 3000 loci for three gynogenetic diploid families. The centromeric regions were sometimes large. The percentage of heterozygote offspring was constrained by the number of progeny in each cross (84−93). Increasing the number of crosses, as well as the number of progeny, would facilitate the narrower placement of the centromere relative to the mapped markers. The location of the centromere allowed us to conclusively support previous findings on chromosome arm arrangement in Chinook salmon (<a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Phillips and Rab 2001</a>; <a id="xref-ref-61-8" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> 2013</a>). Additionally, we confirmed that Ots25 (Chinook salmon linkage group Ck06) was acrocentric and that Ots16 (Ck04) was metacentric, as speculated in <a id="xref-ref-54-11" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> (2013)</a> and <a id="xref-ref-61-9" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> (2013)</a>. Six of the homeologous chromosome pairs detected in this study had been previously identified in Chinook salmon, two were novel (Ots01q/06q and Ots07p/14p) and highly supported, and three previously identified pairings were not observed here (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Eleven homeologous chromosome arm pairs have therefore been identified to date for Chinook salmon.</p><p id="p-53">Our data support the hypothesis of near complete interference in Chinook salmon, where we observed very few occurrences of double crossovers and a maximum proportion of heterozygotes close to one for all chromosomes in the gynogenetic diploid families. This result agrees with previous studies (<em>e.g.</em>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Guyomard <em>et al.</em> 2006</a>) but is supported by a much higher number of markers and recombination events observed. We also observed that the frequency of recombination was reduced in the telomeric regions in females, as suggested in <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Moen <em>et al.</em> (2004)</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Danzmann <em>et al.</em> (2008)</a>, or <a id="xref-ref-36-14" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> (2011)</a>. The greater proportion of markers mapping in the telomeric regions suggest that the map created in this study covers the entire genome, but that the order of the markers in the telomeres is likely not fully resolved, but could be by mapping male meiosis. Indeed, the male-based map based on RAD markers in rainbow trout (<a id="xref-ref-50-8" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>) showed that most recombination events occurred at the telomeres.</p><p id="p-54">Placement of the centromeres on the Chinook salmon linkage map and comparisons with the rainbow trout linkage map (<a id="xref-ref-50-9" class="xref-bibr" href="#ref-50">Miller <em>et al.</em> 2012</a>) confirmed all rearrangements and homologies previously identified (<a id="xref-ref-54-12" class="xref-bibr" href="#ref-54">Naish <em>et al.</em> 2013</a>; <a id="xref-ref-61-10" class="xref-bibr" href="#ref-61">Phillips <em>et al.</em> 2013</a>). Our data also support the fact that Ots16 (Ck04) comprises a fusion between a fragment of one chromosome arm from a metacentric chromosome, Omy11p, and another, Omy9q. However, the greater resolution on the current map shows that markers from Omy11p are found on both arms of Ots16, suggesting that there may have been a centromeric inversion on Ots16. The number of RAD loci shared between Chinook salmon and rainbow trout suggests that determining chromosome evolution across salmonids is increasingly feasible as more species are mapped using RAD loci. Since chromosome arms are mainly conserved across species, this map can also be used for genome-wide studies in other salmon species.</p><p id="p-55">In this study, we examined the chromosomal distribution of duplicated loci that differed only at one, two, or three nucleotide sites. Thus, we assumed that these loci had only recently diverged or were still involved in occasional multivalent pairings. The distribution of this type of duplicated locus varied across chromosomes. Linkage group arms had either almost no duplicated loci, or they had a high density of duplicated loci primarily located in distal regions from the centromere. Population genetic studies based on RAD loci in duplicated regions might therefore be limited, and so we recommend using mapped ESTs or microsatellites to target these regions. Interestingly, all of these pairings involved at least one metacentric chromosome. The results suggest that divergence rates of homeologs following WGD have not been uniform. Comparative mapping shows that the homeologous pairings we identified in Chinook salmon have also been shown in other salmonids (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sakamoto <em>et al.</em> 2000</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gharbi <em>et al.</em> 2006</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Danzmann <em>et al.</em> 2008</a>; <a id="xref-ref-36-15" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> 2011</a>).</p><p id="p-56">Although qualitative, it is possible to speculate whether the relative rates of divergence between homeologous chromosome arms are consistent between Chinook salmon and Atlantic salmon, by comparing this study to that of <a id="xref-ref-36-16" class="xref-bibr" href="#ref-36">Lien <em>et al.</em> (2011)</a>. Both studies mapped loci where both paralogs were polymorphic (here, designated “BPP,” and in Atlantic salmon, “MSV5,” <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) and we can broadly use these loci as a metric for reduced divergence between homeologs. Four pairs of chromosome arms have a large number of polymorphic paralogs mapped in both species (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). An additional four pairs were highly supported in Chinook salmon but had no or few equivalent polymorphic loci in Atlantic salmon (1 or 2 MSV5 or arms were instead confirmed as homeologous by a BLAST search between Atlantic salmon and stickleback <em>Gasterosteus aculeatus</em>). Finally, two homeologies in Atlantic salmon, each supported by only one MSV5, were not observed in Chinook salmon. Importantly, the evidence for reduced divergence or ongoing recombination in the remaining chromosome arms in both species is small. There are three possible explanations for these observations. The first is that the patterns observed may simply be explained by marker density—these reported differences might diminish with extensive sequencing. The second is methodological; the duplicated loci in Atlantic salmon were mapped using SNP markers with two alleles, whereas the present study mapped paralogs that had up to four alleles. Loci in Chinook salmon were considered duplicated if the paralogs had a maximum of three substitutions. Relaxing the alignment parameters, or using SNPs with more alleles in Atlantic salmon, might permit identification of duplicated loci that have comparable polymorphisms. The third explanation is intriguing; namely, that the differentiation between the majority of homeologous pairs and the retention of pairing in some pre-dates the divergence between <em>Salmo</em> and <em>Oncorhynchus</em>. <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">May and Johnson (1990)</a> observed conservation of homeologous pairing between the same chromosomes across different salmon species. Our interpretation supports these observations.</p><p id="p-57">Several studies have shown that the other main process of diploidization—chromosomal arm rearrangement—is extensive subsequent to divergence between the salmonid genera (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Danzmann <em>et al.</em> 2005</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Phillips <em>et al.</em> 2009</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Timusk <em>et al.</em> 2011</a>). Some homeologous pairings may have been prevented by these rearrangements. <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Wright <em>et al.</em> (1983)</a> observed that ongoing homeologous pairing in salmon occurred between one acrocentric and one metacentric chromosome. Our results support this observation and add that metacentric-metacentric pairing also occurs. The involvement of at least one metacentric might provide the stability required the formation of multivalents. In Atlantic salmon, the q arm of some chromosomes has been formed by the fusion of two acrocentric chromosomes. In these cases the distal arm (qb) might be involved in homeologous pairing, but the proximal arm (qa) cannot. In conclusion, it is unclear whether the differences in divergence rates among chromosomes can simply be explained by homeologous pairing or whether selection has acted differentially across the genome following the WGD event (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Mayfield-Jones <em>et al.</em> 2013</a>), but elucidating these mechanisms can be explored by explicitly testing for evidence of selection at the molecular level.</p><p id="p-58">Here, we developed two major genomic resources for Chinook salmon: a reference database of RAD loci and a very dense linkage map anchored to the chromosomes, where arms have been identified by placement of the centromeres. We have also identified homeologous chromosomal arm regions that appear to be less diverged than other pairs, highlighting areas that may be of interest in evolutionary analyses of residual polyploidy. These resources will facilitate genome-wide studies in Chinook salmon, such as genome scans (<em>e.g.</em>, <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tsumura <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bradbury <em>et al.</em> 2013</a>), QTL mapping (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Collard <em>et al.</em> 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Nichols <em>et al.</em> 2008</a>), and genome-wide association analyses (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cichon <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Magwire <em>et al.</em> 2012</a>), as well as studies in related salmon species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">This work was funded by a grant from the Washington Sea Grant Program, University of Washington, pursuant to National Oceanic and Atmospheric Administration Award awarded to K.A.N. and J.E.S. The views expressed herein are those of the authors and do not necessarily reflect the views of NOAA or any of their subagencies. We thank Linda Park, Robin Waples, Paul Moran, David Teel, and Michael Ford for giving us access to the samples from the Columbia River used for the database. We thank the Cle Elum Supplementation and Research Facility (CESRF) for allowing us to create and raise the haploid crosses in their facilities. We thank Todd Seamons, Ken Warheit, and Sewall Young from the Washington Department of Fish and Wildlife, for helping with haploid crosses. We thank James Meyers and Jeff Hard for the gynogenetic diploid and diploid crosses. We thank Ruth Phillips for her invaluable advice throughout. Finally, we thank Isadora Jimenez-Hidalgo, Melissa Baird, Carita Pascal, Daniel Drinan, Miyako Kodama, and Kotaro Ono for their help with lab work, data analysis and paper review.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009316/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-60"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-05"><span class="received-label">Received </span>November 5, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-24"><span class="accepted-label">Accepted </span>December 24, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Brieuc <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.1" data-doi="10.1111/j.1755-0998.2010.02971.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abadia-Cardoso</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemento</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Discovery and characterization of single-nucleotide polymorphisms in steelhead/rainbow trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D11%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02971.x%26rft_id%253Dinfo%253Apmid%252F21429161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02971.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429161&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.447.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Tetraploidy and the Evolution of Salmonid Fishes in Evolutionary Genetics of Fishes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Secondary tetrasomic segregation of MDH-B and preferential pairing of homeologues in rainbow trout.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D145%26rft.issue%253D4%26rft.spage%253D1083%26rft.epage%253D1092%26rft.atitle%253DSecondary%2BTetrasomic%2BSegregation%2Bof%2BMDH-B%2Band%2BPreferential%2BPairing%2Bof%2BHomeologues%2Bin%2BRainbow%2BTrout%26rft_id%253Dinfo%253Apmid%252F9093860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NS80LzEwODMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leary</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Gene-centromere mapping of 25 loci in rainbow-trout.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D77%26rft.issue%253D5%26rft.spage%253D307%26rft.epage%253D312%26rft.atitle%253DGene-centromere%2Bmapping%2Bof%2B25%2Bloci%2Bin%2Brainbow%2Btrout%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijc3LzUvMzA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.5" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.6" data-doi="10.1111/j.1755-0998.2012.03157.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leary</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlfeld</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">RAD sequencing yields a high success rate for westslope cutthroat and rainbow trout species-diagnostic SNP assays.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D12%26rft.spage%253D653%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2012.03157.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2012.03157.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.7" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borza</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence and their consequences for the resolution of spatial structure in an exploited marine fish.</span> <abbr class="cit-jnl-abbrev">Evol.Appl.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">450</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.Appl.%26rft.volume%253D6%26rft.spage%253D450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.9" data-doi="10.1577/T07-011.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification of novel single-nucleotide polymorphisms in Chinook salmon and variation among life history types.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D137%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1577%252FT07-011.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/T07-011.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.10" data-doi="10.1016/j.semcdb.2012.12.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Canestroa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albalat</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irimia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Fernandez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of gene gains, losses and duplication modes on the origin and diversification of vertebrates.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D24%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2012.12.008%26rft_id%253Dinfo%253Apmid%252F23291262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2012.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23291262&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.11" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping Loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.12" data-doi="10.1176/appi.ajp.2008.08091354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cichon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craddock</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faraone</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gejman</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomewide association studies: history, rationale, and prospects for psychiatric disorders.</span> <abbr class="cit-jnl-abbrev">Am. J. Psychiatry</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPsychiatry%26rft_id%253Dinfo%253Adoi%252F10.1176%252Fappi.ajp.2008.08091354%26rft_id%253Dinfo%253Apmid%252F19339359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1176/appi.ajp.2008.08091354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19339359&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265662500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.13" data-doi="10.1007/s10681-005-1681-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B. C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahufer</span>  <span class="cit-name-given-names">M. Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">E. C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An introduction to markers, quantitative trait loci (QTL) mapping and marker-assisted selection for crop improvement: the basic concepts.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D142%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-1681-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-1681-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000228875800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.14" data-doi="10.1371/journal.pone.0046662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crete-Lafreniere</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Framing the Salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrete-Lafreniere%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46662%26rft.epage%253De46662%26rft.atitle%253DFraming%2Bthe%2Bsalmonidae%2Bfamily%2Bphylogenetic%2Bportrait%253A%2Ba%2Bmore%2Bcomplete%2Bpicture%2Bfrom%2Bincreased%2Btaxon%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046662%26rft_id%253Dinfo%253Apmid%252F23071608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071608&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">LINKMFEX: linkage analysis software for diploid and tetraploid outcrossed mapping panels.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">247</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D247%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.16" data-doi="10.1139/g05-067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A comparative analysis of the rainbow trout genome with 2 other species of fish (Arctic charr and Atlantic salmon) within the tetraploid derivative Salmonidae family (subfamily: Salmoninae).</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D48%26rft.issue%253D6%26rft.spage%253D1037%26rft.epage%253D1051%26rft.atitle%253DA%2Bcomparative%2Banalysis%2Bof%2Bthe%2Brainbow%2Btrout%2Bgenome%2Bwith%2B2%2Bother%2Bspecies%2Bof%2Bfish%2B%2528Arctic%2Bcharr%2Band%2BAtlantic%2Bsalmon%2529%2Bwithin%2Bthe%2Btetraploid%2Bderivative%2BSalmonidae%2Bfamily%2B%2528subfamily%253A%2BSalmoninae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg05-067%26rft_id%253Dinfo%253Apmid%252F16391673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g05-067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16391673&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.17" data-doi="10.1186/1471-2164-9-557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Distribution of ancestral proto-Actinopterygian chromosome arms within the genomes of 4R-derivative salmonid fishes (Rainbow trout and Atlantic salmon).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDanzmann%26rft.auinit1%253DR.%2BG.%26rft.volume%253D9%26rft.spage%253D557%26rft.epage%253D557%26rft.atitle%253DDistribution%2Bof%2Bancestral%2Bproto-Actinopterygian%2Bchromosome%2Barms%2Bwithin%2Bthe%2Bgenomes%2Bof%2B4R-derivative%2Bsalmonid%2Bfishes%2B%2528Rainbow%2Btrout%2Band%2BAtlantic%2Bsalmon%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-557%26rft_id%253Dinfo%253Apmid%252F19032764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19032764&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.18" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.19" data-doi="10.1016/j.tree.2005.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">P. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purnell</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome duplication, extinction and vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DDonoghue%26rft.auinit1%253DP.%2BC.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D312%26rft.epage%253D319%26rft.atitle%253DGenome%2Bduplication%252C%2Bextinction%2Band%2Bvertebrate%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2005.04.008%26rft_id%253Dinfo%253Apmid%252F16701387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2005.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701387&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229887800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.20" data-doi="10.1146/annurev.genet.42.110807.091524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary genetics of genome merger and doubling in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DDoyle%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D42%26rft.spage%253D443%26rft.epage%253D461%26rft.atitle%253DEvolutionary%2Bgenetics%2Bof%2Bgenome%2Bmerger%2Band%2Bdoubling%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091524%26rft_id%253Dinfo%253Apmid%252F18983261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983261&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.21" data-doi="10.1186/1471-2164-13-521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEverett%26rft.auinit1%253DM.%2BV.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D521%26rft.atitle%253DMeiotic%2Bmaps%2Bof%2Bsockeye%2Bsalmon%2Bderived%2Bfrom%2Bmassively%2Bparallel%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-521%26rft_id%253Dinfo%253Apmid%252F23031582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23031582&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.22" data-doi="10.1534/genetics.105.048330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A linkage map for brown trout (Salmo trutta): Chromosome homeologies and comparative genome organization with other salmonid fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2405</span>–<span class="cit-lpage">2419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048330%26rft_id%253Dinfo%253Apmid%252F16452148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI0MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.23" data-doi="10.1186/1471-2164-7-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guyomard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabet-Canale</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martineau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A Type I and Type II microsatellite linkage map of Rainbow trout (<em>Oncorhynchus mykiss</em>) with presumptive coverage of all chromosome arms.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGuyomard%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DA%2Btype%2BI%2Band%2Btype%2BII%2Bmicrosatellite%2Blinkage%2Bmap%2Bof%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529%2Bwith%2Bpresumptive%2Bcoverage%2Bof%2Ball%2Bchromosome%2Barms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-302%26rft_id%253Dinfo%253Apmid%252F17137492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17137492&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.24" data-doi="10.1186/1471-2156-13-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guyomard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussaha</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krieg</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hervet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quillet</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A synthetic rainbow trout linkage map provides new insights into the salmonid whole genome duplication and the conservation of synteny among teleosts.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGuyomard%26rft.auinit1%253DR.%26rft.volume%253D13%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DA%2Bsynthetic%2Brainbow%2Btrout%2Blinkage%2Bmap%2Bprovides%2Bnew%2Binsights%2Binto%2Bthe%2Bsalmonid%2Bwhole%2Bgenome%2Bduplication%2Band%2Bthe%2Bconservation%2Bof%2Bsynteny%2Bamong%2Bteleosts.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-13-15%26rft_id%253Dinfo%253Apmid%252F22424132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-13-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22424132&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.25" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.26" data-doi="10.1534/g3.112.003137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thrower</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hale</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of migration-related traits in rainbow and steelhead trout, <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DHecht%26rft.auinit1%253DB.%2BC.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253D1113%26rft.epage%253D1127%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2BMigration-Related%2BTraits%2Bin%2BRainbow%2Band%2BSteelhead%2BTrout%252C%2BOncorhynchus%2Bmykiss%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003137%26rft_id%253Dinfo%253Apmid%252F22973549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjIvOS8xMTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.27" data-doi="10.1111/j.1755-0998.2010.02958.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matala</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparison of SNPs and microsatellites for fine-scale application of genetic stock identification of Chinook salmon in the Columbia River Basin.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02958.x%26rft_id%253Dinfo%253Apmid%252F21429170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02958.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429170&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.28" data-doi="10.1371/journal.pgen.1000862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiffler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genomics of parallel adaptation in threespine stickleback using sequenced RAD tags.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHohenlohe%26rft.auinit1%253DP.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000862%26rft.epage%253De1000862%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bparallel%2Badaptation%2Bin%2Bthreespine%2Bstickleback%2Busing%2Bsequenced%2BRAD%2Btags.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000862%26rft_id%253Dinfo%253Apmid%252F20195501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20195501&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.29" data-doi="10.1186/1471-2164-13-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterisation of QTL-linked and genome-wide restriction-site associated DNA (RAD) markers in farmed Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D13%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DCharacterisation%2Bof%2BQTL-linked%2Band%2Bgenome-wide%2Brestriction%2Bsite-associated%2BDNA%2B%2528RAD%2529%2Bmarkers%2Bin%2Bfarmed%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-244%26rft_id%253Dinfo%253Apmid%252F22702806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702806&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.30" data-doi="10.1111/j.1471-8286.2007.02019.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing breakthroughs for genomic ecology and evolutionary biology.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DHudson%26rft.auinit1%253DM.%2BE.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D17%26rft.atitle%253DSequencing%2Bbreakthroughs%2Bfor%2Bgenomic%2Becology%2Band%2Bevolutionary%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.02019.x%26rft_id%253Dinfo%253Apmid%252F21585713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.02019.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585713&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.31" data-doi="10.1016/j.gde.2009.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufton</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panopoulou</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Polyploidy and genome restructuring: a variety of outcomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DHufton%26rft.auinit1%253DA.%2BL.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D600%26rft.epage%253D606%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Brestructuring%253A%2Ba%2Bvariety%2Bof%2Boutcomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2009.10.005%26rft_id%253Dinfo%253Apmid%252F19900800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2009.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19900800&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273117900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.32" data-doi="10.1086/427926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorgenson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Province</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leppert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ethnicity and human genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DJorgenson%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D276%26rft.epage%253D290%26rft.atitle%253DEthnicity%2Band%2Bhuman%2Bgenetic%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F427926%26rft_id%253Dinfo%253Apmid%252F15627237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/427926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15627237&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226215100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.33" data-doi="10.1016/j.aquaculture.2007.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Androgenesis, gynogenesis and the production of clones in fishes: a review.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D269%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2007.05.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2007.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000249143600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.34" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.3.447.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascal</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SNPs identified through genotyping-by-sequencing improve genetic stock identification of Chinook salmon (<em>Oncorhynchus tshawytscha</em>) from western Alaska.</span> <abbr class="cit-jnl-abbrev">Canadian Journal of Fisheries and Aquatic Sciences, online early.</abbr> DOI: <a href="http://dx.doi.org/10.1139/cjfas-2013-0502">10.1139/cjfas-2013-0502</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.36" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.37" data-doi="10.1139/g00-016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Habicht</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kretschmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene-centromere mapping of 312 loci in pink salmon by half-tetrad analysis.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLindner%26rft.auinit1%253DK.%2BR.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D538%26rft.epage%253D549%26rft.atitle%253DGene-centromere%2Bmapping%2Bof%2B312%2Bloci%2Bin%2Bpink%2Bsalmon%2Bby%2Bhalf-tetrad%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg00-016%26rft_id%253Dinfo%253Apmid%252F10902719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g00-016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10902719&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lubieniecki</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative genomic analysis of Atlantic salmon, <em>Salmo salar</em>, from Europe and North America.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DLubieniecki%26rft.auinit1%253DK.%2BP.%26rft.volume%253D11%26rft.spage%253D105%26rft.epage%253D105%26rft.atitle%253DComparative%2Bgenomic%2Banalysis%2Bof%2BAtlantic%2Bsalmon%252C%2BSalmo%2Bsalar%252C%2Bfrom%2BEurope%2Band%2BNorth%2BAmerica.%26rft_id%253Dinfo%253Apmid%252F21092310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21092310&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.39" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.40" data-doi="10.1111/j.1469-7998.2011.00829.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrou</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome duplication in amphibians and fish: an extended synthesis.</span> <abbr class="cit-jnl-abbrev">J. Zool.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BZool.%26rft.volume%253D284%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-7998.2011.00829.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-7998.2011.00829.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000292262700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.41" data-doi="10.1371/journal.pgen.1003057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabian</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweyen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longdon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies reveal a simple genetic basis of resistance to naturally coevolving viruses in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMagwire%26rft.auinit1%253DM.%2BM.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1003057%26rft.epage%253De1003057%26rft.atitle%253DGenome-Wide%2BAssociation%2BStudies%2BReveal%2Ba%2BSimple%2BGenetic%2BBasis%2Bof%2BResistance%2Bto%2BNaturally%2BCoevolving%2BViruses%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003057%26rft_id%253Dinfo%253Apmid%252F23166512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23166512&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.42" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OneMap: software for genetic mapping in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.43" data-doi="10.1080/02755947.2012.678961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matala</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Influences of hatchery supplementation, spawner distribution, and habitat on genetic structure of Chinook salmon in the South Fork Salmon River, Idaho.</span> <abbr class="cit-jnl-abbrev">N. Am. J. Fish. Manage.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BAm.%2BJ.%2BFish.%2BManage.%26rft.volume%253D32%26rft.spage%253D346%26rft_id%253Dinfo%253Adoi%252F10.1080%252F02755947.2012.678961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/02755947.2012.678961&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Composite linkage map of Salmonid fishes Salvelinus Salmo Oncorhynchus. O'brien, S. J. (Ed.).</span> <abbr class="cit-jnl-abbrev">Genetic Maps: Locus Maps of Complex Genomes, Fifth Edition.</abbr> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>: Plain View, New York, USA. Illus, IV151–IV160.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.45" data-doi="10.1016/j.semcdb.2013.02.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayfield-Jones</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Washburn</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Watching the grin fade: Tracing the effects of polyploidy on different evolutionary time scales.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSemin.%2BCell%2BDev.%2BBiol.%26rft.volume%253D24%26rft.spage%253D320%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2013.02.002%26rft_id%253Dinfo%253Apmid%252F23466286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2013.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23466286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.46" data-doi="10.1126/science.1207205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayrose</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothfels</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnuson-Ford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recently formed polyploid plants diversify at lower rates.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">1257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1207205%26rft_id%253Dinfo%253Apmid%252F21852456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0Ny8xMjU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.47" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequencing technologies - the next generation.</span> <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMetzker%26rft.auinit1%253DM.%2BL.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D46%26rft.atitle%253DSequencing%2Btechnologies%2B-%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.48" data-doi="10.1111/j.1365-2052.2008.01699.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClelland</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genetic linkage map for coho salmon (<em>Oncorhynchus kisutch</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DMcClelland%26rft.auinit1%253DE.%2BK.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D169%26rft.epage%253D179%26rft.atitle%253DA%2Bgenetic%2Blinkage%2Bmap%2Bfor%2Bcoho%2Bsalmon%2B%2528Oncorhynchus%2Bkisutch%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2008.01699.x%26rft_id%253Dinfo%253Apmid%252F18318791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2008.01699.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18318791&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254381100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.49" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction-site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.50" data-doi="10.1111/j.1365-294X.2011.05305.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunelli</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A conserved haplotype controls parallel adaptation in geographically distant salmonid populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BR.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D249%26rft.atitle%253DA%2Bconserved%2Bhaplotype%2Bcontrols%2Bparallel%2Badaptation%2Bin%2Bgeographically%2Bdistant%2Bsalmonid%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05305.x%26rft_id%253Dinfo%253Apmid%252F21988725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05305.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988725&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298845600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.51" data-doi="10.1111/j.1365-2052.2004.01097.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyheim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munck</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Raya</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A linkage map of Atlantic salmon (<em>Salmo salar</em>) reveals an uncommonly large difference in recombination rate between the sexes.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DMoen%26rft.auinit1%253DT.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D92%26rft.atitle%253DA%2Blinkage%2Bmap%2Bof%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Ban%2Buncommonly%2Blarge%2Bdifference%2Bin%2Brecombination%2Brate%2Bbetween%2Bthe%2Bsexes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2004.01097.x%26rft_id%253Dinfo%253Apmid%252F15025566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2004.01097.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15025566&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220273100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.52" data-doi="10.1186/1471-2164-9-223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjoglum</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A linkage map of the Atlantic salmon (<em>Salmo salar</em>) based on EST-derived SNP markers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMoen%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.spage%253D223%26rft.epage%253D223%26rft.atitle%253DA%2Blinkage%2Bmap%2Bof%2Bthe%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Bbased%2Bon%2BEST-derived%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-223%26rft_id%253Dinfo%253Apmid%252F18482444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18482444&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.53" data-doi="10.1016/S0959-437X(02)00358-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Variation in recombination rate across the genome: evidence and implications.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D657%26rft.epage%253D663%26rft.atitle%253DVariation%2Bin%2Brecombination%2Brate%2Bacross%2Bthe%2Bgenome%253A%2Bevidence%2Band%2Bimplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900358-1%26rft_id%253Dinfo%253Apmid%252F12433578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00358-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433578&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newton</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genome mapping between Chinook salmon (<em>Oncorhynchus tshawytscha</em>) and rainbow trout (<em>O. mykiss</em>) based on homologous microsatellite loci</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D2281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.55" data-doi="10.1534/genetics.107.084251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edo</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genetic basis of smoltification-related traits in <em>Oncorhynchus mykiss</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084251%26rft_id%253Dinfo%253Apmid%252F18562654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.56" data-doi="10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Enormous diversity in genome sizes of fish as a reflection of natures extensive experiments with gene duplication.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D99%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1577%252F1548-8659%25281970%252999%253C120%253ATEDIGS%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/1548-8659(1970)99&lt;120:TEDIGS&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.57" data-doi="10.1006/scdb.1999.0332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gene duplication and the uniqueness of vertebrate genomes circa 1970–1999.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DOhno%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D517%26rft.epage%253D522%26rft.atitle%253DGene%2Bduplication%2Band%2Bthe%2Buniqueness%2Bof%2Bvertebrate%2Bgenomes%2Bcirca%2B1970-1999.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fscdb.1999.0332%26rft_id%253Dinfo%253Apmid%252F10597635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/scdb.1999.0332&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10597635&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083733400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-4.3.447.58" data-doi="10.1017/S1464793100005613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rab</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome evolution in the Salmonidae (Pisces): an update.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%26rft.volume%253D76%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D25%26rft.atitle%253DChromosome%2Bevolution%2Bin%2Bthe%2BSalmonidae%2B%2528Pisces%2529%253A%2Ban%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1464793100005613%26rft_id%253Dinfo%253Apmid%252F11325050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1464793100005613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11325050&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.59" data-doi="10.1534/genetics.105.055269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichol</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeKoning</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keadey</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assignment of rainbow trout linkage groups to specific chromosomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.055269%26rft_id%253Dinfo%253Apmid%252F16951085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE2NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.60" data-doi="10.1186/1471-2156-10-46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keatley</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morasch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubieniecki</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Assignment of Atlantic salmon (<em>Salmo salar</em>) linkage groups to specific chromosomes: Conservation of large syntenic blocks corresponding to whole chromosome arms in rainbow trout (<em>Oncorhynchus mykiss</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPhillips%26rft.auinit1%253DR.%2BB.%26rft.volume%253D10%26rft.spage%253D46%26rft.epage%253D46%26rft.atitle%253DAssignment%2Bof%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Blinkage%2Bgroups%2Bto%2Bspecific%2Bchromosomes%253A%2Bconservation%2Bof%2Blarge%2Bsyntenic%2Bblocks%2Bcorresponding%2Bto%2Bwhole%2Bchromosome%2Barms%2Bin%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-10-46%26rft_id%253Dinfo%253Apmid%252F19689812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-10-46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19689812&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assignment of Chinook Salmon (<em>Oncorhynchus tshawytscha</em>) linkage groups to specific chromosomes reveals a karyotype with multiple rearrangements of the chromosome arms of rainbow trout (<em>Oncorhynchus mykiss</em>)</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D2289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The zebrafish genome in context: Ohnologs gone missing.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zoolog. B Mol. Dev. Evol.</abbr> <span class="cit-vol">308B</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bexperimental%2Bzoology.%2BPart%2BB.%2BMolecular%2Band%2Bdevelopmental%2Bevolution%26rft.stitle%253DJ%2BExp%2BZoolog%2BB%2BMol%2BDev%2BEvol%26rft.aulast%253DPostlethwait%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D308%26rft.issue%253D5%26rft.spage%253D563%26rft.epage%253D577%26rft.atitle%253DThe%2Bzebrafish%2Bgenome%2Bin%2Bcontext%253A%2Bohnologs%2Bgone%2Bmissing.%26rft_id%253Dinfo%253Apmid%252F17068775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17068775&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterized by large sex-specific differences in recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSakamoto%26rft.auinit1%253DT.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1331%26rft.epage%253D1345%26rft.atitle%253DA%2BMicrosatellite%2BLinkage%2BMap%2Bof%2BRainbow%2BTrout%2B%2528Oncorhynchus%2Bmykiss%2529%2BCharacterized%2Bby%2BLarge%2BSex-Specific%2BDifferences%2Bin%2BRecombination%2BRates%26rft_id%253Dinfo%253Apmid%252F10880492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzEzMzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.64" data-doi="10.1111/j.1755-0998.2010.02936.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascal</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grau</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome sequencing and high-resolution melt analysis advance single nucleotide polymorphism discovery in duplicated salmonids.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BRes.%26rft.volume%253D11%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02936.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02936.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.65" data-doi="10.1016/j.gde.2007.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Consequences of genome duplication.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DSemon%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D505%26rft.epage%253D512%26rft.atitle%253DConsequences%2Bof%2Bgenome%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2007.09.007%26rft_id%253Dinfo%253Apmid%252F18006297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2007.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006297&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251856900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.66" data-doi="10.1038/nbt1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1135</span>–<span class="cit-lpage">1145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DShendure%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D10%26rft.spage%253D1135%26rft.epage%253D1145%26rft.atitle%253DNext-generation%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1486%26rft_id%253Dinfo%253Apmid%252F18846087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846087&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259926000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.67" data-doi="10.1007/s004120050289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slijepcevic</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Telomeres and mechanisms of Robertsonian fusion.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DSlijepcevic%26rft.auinit1%253DP.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D136%26rft.epage%253D140%26rft.atitle%253DTelomeres%2Band%2Bmechanisms%2Bof%2BRobertsonian%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004120050289%26rft_id%253Dinfo%253Apmid%252F9601982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004120050289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9601982&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074093800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.68" data-doi="10.1111/j.1365-294X.2005.02731.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elfstrom</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Use of sequence data from rainbow trout and Atlantic salmon for SNP detection in Pacific salmon.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4193</span>–<span class="cit-lpage">4203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSmith%26rft.auinit1%253DC.%2BT.%26rft.volume%253D14%26rft.issue%253D13%26rft.spage%253D4193%26rft.epage%253D4203%26rft.atitle%253DUse%2Bof%2Bsequence%2Bdata%2Bfrom%2Brainbow%2Btrout%2Band%2BAtlantic%2Bsalmon%2Bfor%2BSNP%2Bdetection%2Bin%2BPacific%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02731.x%26rft_id%253Dinfo%253Apmid%252F16262869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02731.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16262869&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232774000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.69" data-doi="10.1577/T04-019.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwenke</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">Use of the 5 ’-nuclease reaction for single nucleotide polymorphism genotyping in Chinook salmon.</span> <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D134%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1577%252FT04-019.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/T04-019.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.70" data-doi="10.1577/M04-143.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">U. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">c</span> <span class="cit-article-title">Single nucleotide polymorphisms provide rapid and accurate estimates of the proportions of US and Canadian Chinook salmon caught in Yukon River fisheries.</span> <abbr class="cit-jnl-abbrev">N. Am. J. Fish. Manage.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">944</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BAm.%2BJ.%2BFish.%2BManage.%26rft.volume%253D25%26rft.spage%253D944%26rft_id%253Dinfo%253Adoi%252F10.1577%252FM04-143.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/M04-143.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buggs</span>  <span class="cit-name-given-names">R. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">What we still don’t know about polyploidy.</span> <abbr class="cit-jnl-abbrev">Taxon</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">1387</span>–<span class="cit-lpage">1403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D59%26rft.spage%253D1387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000282924800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.72" data-doi="10.1016/S0168-9525(01)02318-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genome duplication, divergent resolution and speciation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DTaylor%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D299%26rft.epage%253D301%26rft.atitle%253DGenome%2Bduplication%252C%2Bdivergent%2Bresolution%2Band%2Bspeciation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252801%252902318-6%26rft_id%253Dinfo%253Apmid%252F11377777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(01)02318-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11377777&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170644200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.73" data-doi="10.1111/j.1755-0998.2010.02968.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jasper</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barclay</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic differentiation of Alaska Chinook salmon: the missing link for migratory studies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D11%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02968.x%26rft_id%253Dinfo%253Apmid%252F21429177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02968.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429177&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Gene-centromere mapping in Rainbow-trout—high interference over long map distances.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThorgaard%26rft.auinit1%253DG.%2BH.%26rft.volume%253D103%26rft.issue%253D4%26rft.spage%253D771%26rft.epage%253D783%26rft.atitle%253DGENE-CENTROMERE%2BMAPPING%2BIN%2BRAINBOW%2BTROUT%253A%2BHIGH%2BINTERFERENCE%2BOVER%2BLONG%2BMAP%2BDISTANCES%26rft_id%253Dinfo%253Apmid%252F17246124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTAzLzQvNzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timusk</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghadam</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DTimusk%26rft.auinit1%253DE.%2BR.%26rft.volume%253D12%26rft.spage%253D68%26rft.epage%253D68%26rft.atitle%253DGenome%2Bevolution%2Bin%2Bthe%2Bfish%2Bfamily%2Bsalmonidae%253A%2Bgeneration%2Bof%2Ba%2Bbrook%2Bcharr%2Bgenetic%2Bmap%2Band%2Bcomparisons%2Bamong%2Bcharrs%2B%2528Arctic%2Bcharr%2Band%2Bbrook%2Bcharr%2529%2Bwith%2Brainbow%2Btrout.%26rft_id%253Dinfo%253Apmid%252F21798024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21798024&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.76" data-doi="10.1038/hdy.2012.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsumura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriguchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ihara-Ujino</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome scanning for detecting adaptive genes along environmental gradients in the Japanese conifer, <em>Cryptomeria japonica</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DTsumura%26rft.auinit1%253DY.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D349%26rft.epage%253D360%26rft.atitle%253DGenome%2Bscanning%2Bfor%2Bdetecting%2Badaptive%2Bgenes%2Balong%2Benvironmental%2Bgradients%2Bin%2Bthe%2BJapanese%2Bconifer%252C%2BCryptomeria%2Bjaponica.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.50%26rft_id%253Dinfo%253Apmid%252F22929151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22929151&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311098300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.77" data-doi="10.1038/35072009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Yesterday’s polyploids and the mystery of diploidization.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D333%26rft.epage%253D341%26rft.atitle%253DYesterday%2527s%2Bpolyploids%2Band%2Bthe%2Bmystery%2Bof%2Bdiploidization.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35072009%26rft_id%253Dinfo%253Apmid%252F11331899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35072009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11331899&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168467800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Meiotic models to explain classical linkage, pseudolinkage, and chromosome-pairing in tetraploid derivative Salmonid genomes.</span> <abbr class="cit-jnl-abbrev">Isozymes Curr. Top. Biol. Med. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIsozymes%26rft.stitle%253DIsozymes%2BCurr%2BTop%2BBiol%2BMed%2BRes%26rft.aulast%253DWright%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D10%26rft.spage%253D239%26rft.epage%253D260%26rft.atitle%253DMeiotic%2Bmodels%2Bto%2Bexplain%2Bclassical%2Blinkage%252C%2Bpseudolinkage%252C%2Band%2Bchromosome%2Bpairing%2Bin%2Btetraploid%2Bderivative%2Bsalmonid%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F6354984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6354984&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RL29500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.447.79" data-doi="10.1109/TCBB.2010.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinformatics%26rft.volume%253D8%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2010.35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2010.35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/447.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/433" title="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/461" title="The BAG Homology Domain of Snl1 Cures Yeast Prion [URE3] Through Regulation of Hsp70 Chaperones" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zBWmhb-CC1mLnb_9TyM17eMbWmcjaO0WjczpViw42W8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/447" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/447" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/447" class="active" data-icon-position="" data-hide-link-title="0">A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459816" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Fv2yTE8yvimw6UbL5YQ1UNdKnl_LfIUPXAddORXd3WM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/447.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/447" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-AhQ0wJr3Il-4IqeVHAGIH1qVVvDf-kThROFqhJxNTxU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/447" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/447.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459816" id="node459816--2" data-pisa="ggg;4/3/447" data-pisa-master="ggg;g3.113.009316" data-apath="/ggg/4/3/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Linkage Map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="1">Charles D. Waters</span>, <span class="highwire-citation-author" data-delta="2">James E. Seeb</span> and <span class="highwire-citation-author" data-delta="3">Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459816--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbrieuc@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mbrieuc@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles D. Waters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BD.%2BWaters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waters%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BD.%2BWaters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/447" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/447&amp;text=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/447&amp;t=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/447&amp;title=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459816/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459816" id="node459816--3" data-pisa="ggg;4/3/447" data-pisa-master="ggg;g3.113.009316" data-apath="/ggg/4/3/447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Dense Linkage Map for Chinook salmon (<em>Oncorhynchus tshawytscha</em>) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marine S. O. Brieuc</span>, <span class="highwire-citation-author" data-delta="1">Charles D. Waters</span>, <span class="highwire-citation-author" data-delta="2">James E. Seeb</span> and <span class="highwire-citation-author" data-delta="3">Kerry A. Naish</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">447-460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459816--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marine S. O. Brieuc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marine%2BS.%2BO.%2BBrieuc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brieuc%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarine%2BS.%2BO.%2BBrieuc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mbrieuc@uw.edu" class="" data-icon-position="" data-hide-link-title="0">mbrieuc@uw.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Charles D. Waters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BD.%2BWaters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waters%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BD.%2BWaters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James E. Seeb </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BE.%2BSeeb%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seeb%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BE.%2BSeeb%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kerry A. Naish </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>School of Aquatic and Fishery Sciences, University of Washington, Seattle, Washington 98195-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerry%2BA.%2BNaish%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naish%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerry%2BA.%2BNaish%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=447&amp;rft.epage=460&amp;rft.atitle=A%20Dense%20Linkage%20Map%20for%20Chinook%20salmon%20%28Oncorhynchus%20tshawytscha%29%20Reveals%20Variable%20Chromosomal%20Divergence%20After%20an%20Ancestral%20Whole%20Genome%20Duplication%20Event&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Brieuc%2C+Marine+S.+O.&amp;rft.au=Waters%2C+Charles+D.&amp;rft.au=Seeb%2C+James+E.&amp;rft.au=Naish%2C+Kerry+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459816/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459816/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459816/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459816/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459816/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459816/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459816/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459816/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459816/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459816/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459816/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459816/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/447" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/447.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/447.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/447.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/447.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
