<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/399" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/399.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/399.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/399.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/399.atom" />
<meta name="HW.pisa" content="ggg;4/3/399" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection" />
<meta name="DC.Identifier" content="10.1534/g3.113.009365" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Payen et al." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Population adaptation to strong selection can occur through the sequential or parallel accumulation of competing beneficial mutations. The dynamics, diversity, and rate of fixation of beneficial mutations within and between populations are still poorly understood. To study how the mutational landscape varies across populations during adaptation, we performed experimental evolution on seven parallel populations of Saccharomyces cerevisiae continuously cultured in limiting sulfate medium. By combining quantitative polymerase chain reaction, array comparative genomic hybridization, restriction digestion and contour-clamped homogeneous electric field gel electrophoresis, and whole-genome sequencing, we followed the trajectory of evolution to determine the identity and fate of beneficial mutations. During a period of 200 generations, the yeast populations displayed parallel evolutionary dynamics that were driven by the coexistence of independent beneficial mutations. Selective amplifications rapidly evolved under this selection pressure, in particular common inverted amplifications containing the sulfate transporter gene [SUL1][1] . Compared with single clones, detailed analysis of the populations uncovers a greater complexity whereby multiple subpopulations arise and compete despite a strong selection. The most common evolutionary adaptation to strong selection in these populations grown in sulfate limitation is determined by clonal interference, with adaptive variants both persisting and replacing one another.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498" />
<meta name="DC.Contributor" content="Celia Payen" />
<meta name="DC.Contributor" content="Sara C. Di Rienzi" />
<meta name="DC.Contributor" content="Giang T. Ong" />
<meta name="DC.Contributor" content="Jamie L. Pogachar" />
<meta name="DC.Contributor" content="Joseph C. Sanchez" />
<meta name="DC.Contributor" content="Anna B. Sunshine" />
<meta name="DC.Contributor" content="M. K. Raghuraman" />
<meta name="DC.Contributor" content="Bonita J. Brewer" />
<meta name="DC.Contributor" content="Maitreya J. Dunham" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Population adaptation to strong selection can occur through the sequential or parallel accumulation of competing beneficial mutations. The dynamics, diversity, and rate of fixation of beneficial mutations within and between populations are still poorly understood. To study how the mutational landscape varies across populations during adaptation, we performed experimental evolution on seven parallel populations of &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; continuously cultured in limiting sulfate medium. By combining quantitative polymerase chain reaction, array comparative genomic hybridization, restriction digestion and contour-clamped homogeneous electric field gel electrophoresis, and whole-genome sequencing, we followed the trajectory of evolution to determine the identity and fate of beneficial mutations. During a period of 200 generations, the yeast populations displayed parallel evolutionary dynamics that were driven by the coexistence of independent beneficial mutations. Selective amplifications rapidly evolved under this selection pressure, in particular common inverted amplifications containing the sulfate transporter gene &lt;i&gt;SUL1&lt;/i&gt;. Compared with single clones, detailed analysis of the populations uncovers a greater complexity whereby multiple subpopulations arise and compete despite a strong selection. The most common evolutionary adaptation to strong selection in these populations grown in sulfate limitation is determined by clonal interference, with adaptive variants both persisting and replacing one another.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/399" />
<meta name="citation_id" content="4/3/399" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/399" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/399.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/399.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/399.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009365" />
<meta name="citation_pmid" content="24368781" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="399" />
<meta name="citation_lastpage" content="409" />
<meta name="citation_author" content="Celia Payen" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Sara C. Di Rienzi" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Giang T. Ong" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Jamie L. Pogachar" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Joseph C. Sanchez" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Anna B. Sunshine" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="M. K. Raghuraman" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Bonita J. Brewer" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="maitreya@uw.edu" />
<meta name="citation_author_email" content="bbrewer@gs.washington.edu" />
<meta name="citation_author" content="Maitreya J. Dunham" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="maitreya@uw.edu" />
<meta name="citation_author_email" content="bbrewer@gs.washington.edu" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=G. M. Adamo;citation_author=M. Lotti;citation_author=M. J. Tamás;citation_author=S. Brocca;citation_title=Amplification of the CUP1 gene is associated with evolution of copper tolerance in Saccharomyces cerevisiae.;citation_volume=158;citation_year=2012;citation_pmid=22790396;citation_doi=10.1099/mic.0.058024-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. I. Andersson;citation_author=E. S. Slechta;citation_author=J. R. Roth;citation_title=Evidence That Gene Amplification Underlies Adaptive Mutability of the Bacterial lac Operon;citation_pages=1133-1135;citation_volume=282;citation_year=1998;citation_issue=5391;citation_pmid=9804552;citation_doi=10.1126/science.282.5391.1133" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. L. Araya;citation_author=C. Payen;citation_author=M. J. Dunham;citation_author=S. Fields;citation_title=Whole-genome sequencing of a laboratory-evolved yeast strain.;citation_pages=88-88;citation_volume=11;citation_year=2010;citation_pmid=20128923;citation_doi=10.1186/1471-2164-11-88" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Barrick;citation_author=D. S. Yu;citation_author=S. H. Yoon;citation_author=H. Jeong;citation_author=T. K. Oh;citation_title=Genome evolution and adaptation in a long-term experiment with Escherichia coli.;citation_pages=1243-1247;citation_volume=461;citation_year=2009;citation_issue=7268;citation_pmid=19838166;citation_doi=10.1038/nature08480" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Z. D. Blount;citation_author=J. E. Barrick;citation_author=C. J. Davidson;citation_author=R. E. Lenski;citation_title=Genomic analysis of a key innovation in an experimental Escherichia coli population.;citation_pages=513-518;citation_volume=489;citation_year=2012;citation_issue=7417;citation_pmid=22992527;citation_doi=10.1038/nature11514" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Brewer;citation_author=C. Payen;citation_author=M. K. Raghuraman;citation_author=M. J. Dunham;citation_title=Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.;citation_pages=e1002016-e1002016;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437266;citation_doi=10.1371/journal.pgen.1002016" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. J. Brown;citation_author=K. M. Todd;citation_author=R. F. Rosenzweig;citation_title=Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment;citation_pages=931-942;citation_volume=15;citation_year=1998;citation_issue=8;citation_pmid=9718721;citation_doi=10.1093/oxfordjournals.molbev.a026009" />
<meta name="citation_reference" content="citation_journal_title=Proc. Roy. Soc. B.;citation_author=M. K. Burke;citation_title=How does adaptation sweep through the genome? Insights from long-term selection experiments.;citation_volume=279;citation_year=2012;citation_pmid=22833271;citation_doi=10.1098/rspb.2012.0799" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=Z. P. Cakar;citation_author=U. O. Seker;citation_author=C. Tamerler;citation_author=M. Sonderegger;citation_author=U. Sauer;citation_title=Evolutionary engineering of multiple-stress resistant Saccharomyces cerevisiae.;citation_volume=5;citation_year=2005;citation_pmid=15780656;citation_doi=10.1016/j.femsyr.2004.10.010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Cherest;citation_author=J. C. Davidian;citation_author=D. Thomas;citation_author=V. Benes;citation_author=W. Ansorge;citation_title=Molecular Characterization of Two High Affinity Sulfate Transporters in Saccharomyces cerevisiae;citation_pages=627-635;citation_volume=145;citation_year=1997;citation_issue=3;citation_pmid=9055073" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Rep;citation_author=H. H. Chou;citation_author=C. J. Marx;citation_title=Optimization of gene expression through divergent mutational paths.;citation_pages=133-140;citation_volume=1;citation_year=2012;citation_issue=2;citation_pmid=22832162;citation_doi=10.1016/j.celrep.2011.12.003" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. M. Desai;citation_author=D. S. Fisher;citation_author=A. W. Murray;citation_title=The speed of evolution and maintenance of variation in asexual populations.;citation_pages=385-394;citation_volume=17;citation_year=2007;citation_issue=5;citation_pmid=17331728;citation_doi=10.1016/j.cub.2007.01.072" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. G. M. de Visser;citation_author=C. W. Zeyl;citation_author=P. J. Gerrish;citation_author=J. L. Blanchard;citation_author=R. E. Lenski;citation_title=Diminishing Returns from Mutation Supply Rate in Asexual Populations;citation_pages=404-406;citation_volume=283;citation_year=1999;citation_issue=5400;citation_pmid=9888858;citation_doi=10.1126/science.283.5400.404" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=S. C. Di Rienzi;citation_author=K. C. Lancaster;citation_author=R. Lindstrom;citation_author=L. Rolczynski;citation_author=M. K. Raghuraman;citation_title=Genetic, genomic, and molecular tools for studying the protoploid yeast, L. waltii.;citation_pages=137-151;citation_volume=28;citation_year=2011;citation_issue=2;citation_pmid=21246627;citation_doi=10.1002/yea.1826" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=P. J. Gerrish;citation_author=R. E. Lenski;citation_title=The fate of competing beneficial mutations in an asexual population.;citation_pages=127-144;citation_volume=102–103;citation_year=1998;citation_pmid=9720276;citation_doi=10.1023/A:1017067816551" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Gresham;citation_author=M. M. Desai;citation_author=C. M. Tucker;citation_author=H. T. Jenq;citation_author=D. A. Pai;citation_title=The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.;citation_pages=e1000303-e1000303;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19079573;citation_doi=10.1371/journal.pgen.1000303" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Gresham;citation_author=R. Usaite;citation_author=S. M. Germann;citation_author=M. Lisby;citation_author=D. Botstein;citation_title=Adaptation to diverse nitrogen-limited environments by deletion or extrachromosomal element formation of the GAP1 locus.;citation_volume=107;citation_year=2010;citation_pmid=20937885;citation_doi=10.1073/pnas.1014023107" />
<meta name="citation_reference" content="citation_journal_title=Biotechnology letters;citation_journal_abbrev=Biotechnol Lett;citation_author=P. M. Guimaraes;citation_author=J. A. Teixeira;citation_author=L. Domingues;citation_title=Fermentation of high concentrations of lactose to ethanol by engineered flocculent Saccharomyces cerevisiae.;citation_pages=1953-1958;citation_volume=30;citation_year=2008;citation_issue=11;citation_pmid=18575804;citation_doi=10.1007/s10529-008-9779-1" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=F. Hach;citation_author=F. Hormozdiari;citation_author=C. Alkan;citation_author=I. Birol;citation_author=E. E. Eichler;citation_title=mrsFAST: a cache-oblivious algorithm for short-read mapping.;citation_pages=576-577;citation_volume=7;citation_year=2010;citation_issue=8;citation_pmid=20676076;citation_doi=10.1038/nmeth0810-576" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. D. Herron;citation_author=M. Doebeli;citation_title=Parallel evolutionary dynamics of adaptive diversification in Escherichia coli.;citation_pages=e1001490;citation_volume=11;citation_year=2013;citation_pmid=23431270;citation_doi=10.1371/journal.pbio.1001490" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. H. Ho;citation_author=L. Magtanong;citation_author=S. L. Barker;citation_author=D. Gresham;citation_author=S. Nishimura;citation_title=A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.;citation_pages=369-377;citation_volume=27;citation_year=2009;citation_issue=4;citation_pmid=19349972;citation_doi=10.1038/nbt.1534" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=C. S. Hoffman;citation_author=F. Winston;citation_title=A ten-minute DNA preparation from yeast efficiently releases autonomous plasmids for transformation of Escherichia coli.;citation_pages=267-272;citation_volume=57;citation_year=1987;citation_issue=2-3;citation_pmid=3319781;citation_doi=10.1016/0378-1119(87)90131-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Huberman;citation_author=L. D. Spotila;citation_author=K. A. Nawotka;citation_author=S. M. el-Assouli;citation_author=L. R. Davis;citation_title=The in vivo replication origin of the yeast 2 microns plasmid.;citation_pages=473-481;citation_volume=51;citation_year=1987;citation_issue=3;citation_pmid=3311385;citation_doi=10.1016/0092-8674(87)90643-X" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. C. Kao;citation_author=G. Sherlock;citation_title=Molecular characterization of clonal interference during adaptive evolution in asexual populations of Saccharomyces cerevisiae.;citation_pages=1499-1504;citation_volume=40;citation_year=2008;citation_issue=12;citation_pmid=19029899;citation_doi=10.1038/ng.280" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=E. Karakoc;citation_author=C. Alkan;citation_author=B. J. O'Roak;citation_author=M. Y. Dennis;citation_author=L. Vives;citation_title=Detection of structural variants and indels within exome data.;citation_pages=176-178;citation_volume=9;citation_year=2011;citation_pmid=22179552;citation_doi=10.1038/nmeth.1810" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=A. Kohler;citation_author=M. Schneider;citation_author=G. G. Cabal;citation_author=U. Nehrbass;citation_author=E. Hurt;citation_title=Yeast Ataxin-7 links histone deubiquitination with gene gating and mRNA export.;citation_pages=707-715;citation_volume=10;citation_year=2008;citation_issue=6;citation_pmid=18488019;citation_doi=10.1038/ncb1733" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=R. Koszul;citation_author=S. Caburet;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Eucaryotic genome evolution through the spontaneous duplication of large chromosomal segments;citation_pages=234-243;citation_volume=23;citation_year=2004;citation_issue=1;citation_pmid=14685272;citation_doi=10.1038/sj.emboj.7600024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Koszul;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Stability of large segmental duplications in the yeast genome.;citation_volume=172;citation_year=2006;citation_pmid=16489235;citation_doi=10.1534/genetics.105.048058" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Kugelberg;citation_author=E. Kofoid;citation_author=D. I. Andersson;citation_author=Y. Lu;citation_author=J. Mellor;citation_title=The tandem inversion duplication in Salmonella enterica: selection drives unstable precursors to final mutation types.;citation_volume=185;citation_year=2010;citation_pmid=20215473;citation_doi=10.1534/genetics.110.114074" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. I. Lang;citation_author=D. P. Rice;citation_author=M. J. Hickman;citation_author=E. Sodergren;citation_author=G. M. Weinstock;citation_title=Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.;citation_pages=571-574;citation_volume=500;citation_year=2013;citation_pmid=23873039;citation_doi=10.1038/nature12344" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. C. Lee;citation_author=C. J. Marx;citation_title=Repeated, selection-driven genome reduction of accessory genes in experimental populations.;citation_pages=e1002651-e1002651;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22589730;citation_doi=10.1371/journal.pgen.1002651" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Lee;citation_author=C. J. Marx;citation_title=Synchronous waves of failed soft sweeps in the laboratory: remarkably rampant clonal interference of alleles at a single locus.;citation_volume=193;citation_year=2013;citation_pmid=23307898;citation_doi=10.1534/genetics.112.148502" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=A. W. Miller;citation_author=C. Befort;citation_author=E. O. Kerr;citation_author=M. J. Dunham;citation_title=Design and use of multiplexed chemostat arrays.;citation_pages=e50262;citation_volume=Feb 23;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. R. Miller;citation_author=P. Joyce;citation_author=H. A. Wichman;citation_title=Mutational effects and population dynamics during viral adaptation challenge current models.;citation_volume=187;citation_year=2011;citation_pmid=21041559;citation_doi=10.1534/genetics.110.121400" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Notley-McRobb;citation_author=T. Ferenci;citation_title=Experimental Analysis of Molecular Events During Mutational Periodic Selections in Bacterial Evolution;citation_pages=1493-1501;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102352" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell.;citation_author=N. Pashkova;citation_author=L. Gakhar;citation_author=S. C. Winistorfer;citation_author=A. B. Sunshine;citation_author=M. Rich;citation_title=The yeast Alix homolog Bro1 functions as a ubiquitin receptor for protein sorting into multivesicular endosomes.;citation_pages=520-533;citation_volume=25;citation_year=2013;citation_pmid=23726974;citation_doi=10.1016/j.devcel.2013.04.007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Payen;citation_author=R. Koszul;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Segmental duplications arise from Pol32-dependent repair of broken forks through two alternative replication-based mechanisms.;citation_pages=e1000175-e1000175;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18773114;citation_doi=10.1371/journal.pgen.1000175" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry. Biokhimiiï¸ aï¸¡;citation_journal_abbrev=Biochemistry (Mosc);citation_author=F. E. Pryde;citation_author=E. J. Louis;citation_title=Saccharomyces cerevisiae telomeres. A review.;citation_pages=1232-1241;citation_volume=62;citation_year=1997;citation_issue=11;citation_pmid=9467847" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. V. Sonti;citation_author=J. R. Roth;citation_title=Role of Gene Duplications in the Adaptation of Salmonella typhimurium to Growth on Limiting Carbon Sources;citation_pages=19-28;citation_volume=123;citation_year=1989;citation_issue=1;citation_pmid=2680755" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. M. Stoebel;citation_author=K. Hokamp;citation_author=M. S. Last;citation_author=C. J. Dorman;citation_title=Compensatory evolution of gene regulation in response to stress by Escherichia coli lacking RpoS.;citation_pages=e1000671-e1000671;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19798444;citation_doi=10.1371/journal.pgen.1000671" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Tanaka;citation_author=D. A. Bergstrom;citation_author=M. C. Yao;citation_author=S. J. Tapscott;citation_title=Widespread and nonrandom distribution of DNA palindromes in cancer cells provides a structural platform for subsequent gene amplification.;citation_pages=320-327;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711546;citation_doi=10.1038/ng1515" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=Y. C. Tang;citation_author=A. Amon;citation_title=Gene copy-number alterations: a cost-benefit analysis.;citation_pages=394-405;citation_volume=152;citation_year=2013;citation_pmid=23374337;citation_doi=10.1016/j.cell.2012.11.043" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=O. Tenaillon;citation_author=A. Rodriguez-Verdugo;citation_author=R. L. Gaut;citation_author=P. McDonald;citation_author=A. F. Bennett;citation_title=The Molecular Diversity of Adaptive Convergence;citation_pages=457-461;citation_volume=335;citation_year=2012;citation_issue=6067;citation_pmid=22282810;citation_doi=10.1126/science.1212986" />
<meta name="citation_reference" content="citation_journal_title=Advances in biochemical engineering/biotechnology;citation_journal_abbrev=Adv Biochem Eng Biotechnol;citation_author=A. J. van Maris;citation_author=A. A. Winkler;citation_author=M. Kuyper;citation_author=W. T. de Laat;citation_author=J. P. van Dijken;citation_title=Development of efficient xylose fermentation in Saccharomyces cerevisiae: xylose isomerase as a key component.;citation_pages=179-204;citation_volume=108;citation_year=2007;citation_pmid=17846724;citation_doi=10.1007/10_2007_057" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. M. Wahl;citation_author=D. C. Krakauer;citation_title=Models of Experimental Evolution: The Role of Genetic Chance and Selective Necessity;citation_pages=1437-1448;citation_volume=156;citation_year=2000;citation_issue=3;citation_pmid=11063715" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Woods;citation_author=D. Schneider;citation_author=C. L. Winkworth;citation_author=M. A. Riley;citation_author=R. E. Lenski;citation_title=Tests of parallel molecular evolution in a long-term experiment with Escherichia coli.;citation_volume=103;citation_year=2006;citation_pmid=16751270;citation_doi=10.1073/pnas.0602917103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Yona;citation_author=Y. S. Manor;citation_author=R. H. Herbst;citation_author=G. H. Romano;citation_author=A. Mitchell;citation_title=Chromosomal duplication is a transient evolutionary solution to stress.;citation_volume=109;citation_year=2012;citation_pmid=23197825;citation_doi=10.1073/pnas.1211150109" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiology Letters;citation_journal_abbrev=FEMS Microbiol Lett;citation_author=E. Zhang;citation_author=T. Ferenci;citation_title=OmpF changes and the complexity of Escherichia coli adaptation to prolonged lactose limitation.;citation_pages=395-401;citation_volume=176;citation_year=1999;citation_issue=2;citation_pmid=10427722;citation_doi=10.1111/j.1574-6968.1999.tb13689.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Zhong;citation_author=A. Khodursky;citation_author=D. E. Dykhuizen;citation_author=A. M. Dean;citation_title=Evolutionary genomics of ecological specialization.;citation_volume=101;citation_year=2004;citation_pmid=15289609;citation_doi=10.1073/pnas.0404397101" />
<meta name="twitter:title" content="The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Population adaptation to strong selection can occur through the sequential or parallel accumulation of competing beneficial mutations. The dynamics, diversity, and rate of fixation of beneficial mutations within and between populations are still poorly understood. To study how the mutational landscape varies across populations during adaptation, we performed experimental evolution on seven parallel populations of Saccharomyces cerevisiae continuously cultured in limiting sulfate medium. By combining quantitative polymerase chain reaction, array comparative genomic hybridization, restriction digestion and contour-clamped homogeneous electric field gel electrophoresis, and whole-genome sequencing, we followed the trajectory of evolution to determine the identity and fate of beneficial mutations. During a period of 200 generations, the yeast populations displayed parallel evolutionary dynamics that were driven by the coexistence of independent beneficial mutations. Selective amplifications rapidly evolved under this selection pressure, in particular common inverted amplifications containing the sulfate transporter gene [SUL1][1] . Compared with single clones, detailed analysis of the populations uncovers a greater complexity whereby multiple subpopulations arise and compete despite a strong selection. The most common evolutionary adaptation to strong selection in these populations grown in sulfate limitation is determined by clonal interference, with adaptive variants both persisting and replacing one another.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498" />
<meta name="og-title" property="og:title" content="The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/399" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Population adaptation to strong selection can occur through the sequential or parallel accumulation of competing beneficial mutations. The dynamics, diversity, and rate of fixation of beneficial mutations within and between populations are still poorly understood. To study how the mutational landscape varies across populations during adaptation, we performed experimental evolution on seven parallel populations of Saccharomyces cerevisiae continuously cultured in limiting sulfate medium. By combining quantitative polymerase chain reaction, array comparative genomic hybridization, restriction digestion and contour-clamped homogeneous electric field gel electrophoresis, and whole-genome sequencing, we followed the trajectory of evolution to determine the identity and fate of beneficial mutations. During a period of 200 generations, the yeast populations displayed parallel evolutionary dynamics that were driven by the coexistence of independent beneficial mutations. Selective amplifications rapidly evolved under this selection pressure, in particular common inverted amplifications containing the sulfate transporter gene [SUL1][1] . Compared with single clones, detailed analysis of the populations uncovers a greater complexity whereby multiple subpopulations arise and compete despite a strong selection. The most common evolutionary adaptation to strong selection in these populations grown in sulfate limitation is determined by clonal interference, with adaptive variants both persisting and replacing one another.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6S2pMDPvUFrrmskWgwqhNuAct4-Dz-vj714_lRP3t-Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459815","apath":"\/ggg\/4\/3\/399.atom","pisa":"ggg;4\/3\/399","ac":{"\/ggg\/4\/3\/399.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/399.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/399"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/399"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/399":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022H8ALSm2\\\\\\\/JEbSepDHo0izc0ZR6bwAl9mg8z8bKQcYkQdqSa44GF+PbaU7e5y34G\\\\\\\/iPqTVVEdYM03ZukFhsonB+gGufpbCjchbAJ5PODVS\\\\\\\/6yE2Ria\\\\\\\/M+PCf2Ee3A0Rit7qx6UdbDZvTP8nU\\\\\\\/d+XMoDgZEZK+MWHgwM\\\\\\\/UsmOUxVTrpx32QT57VNCxIzA6\\\\\\\/oj9\\\\\\\/occDvWCnLs5lqFRbRykhZtvBahcpIfhQD9NjLf8D7SbNw8waNkpwWa5nKYilTalqCpTFz29hVvBlyftZWHghrce4U0JnLjzfTRTBvhd+p7aD\\\\\\\/f+TDL4qvkGShVQNtfu3cVt84+vmd\\\\\\\/FDo6\\\\\\\/2d7gI7Yc9b+qwRexL+KSWl7sk\\\\\\\/m+XDL64gxXH6AdvR1+qKH+e2na7bra+xQPUevh6g7CxWnvcd\\\\\\\/+uisP4vh\\\\\\\/SiONK8jFQZ5hI\\\\\\\/mb7Y3pTHD4LEIBF6UAdGEA2Cry0GzsXvmjPskiKLP0G3z0yTJDM9NEruEJlryKhy4f5RykdBRAB28CuHFRMc0K81iaW+KVPs2fGBrYPqmMp8pZEZeD6Rybyt\\\\\\\/s4kWyye6R72qOSbjCo4UPy+Z7fiRaJQdy4apFS94KkFuwNoPYHkGizVMkSV8AlqR5\\\\\\\/mWa9nRRN4dDxa3O9fC3KBsHk4QFwHhA\\\\\\\/u8gHw\\\\\\\/xp1iYl5Ojq96gk8lyY2UgmW8+luMc2Pk+d1\\\\\\\/sHnaZZeRl10Yhg9KoInNejURKceSB34+tV+6HTFnsu6FcogCf7MRNd1xpRFJ3rlYRtNmtA3CHeGZnqfU5+MfIl2\\\\\\\/cGFhQegenRM3D3msBT6nD2dyJvh217yM085c16giNkOmrvP\\\\\\\/rdbyVqqtEWySqcI5njaU1cjslWk1BFulHdhApG13fSSaZsAxmlQ3NwwMCu1nBUB0fdD9nR\\\\\\\/lVzE5R3lNN4KOMMw3Mh\\\\\\\/MKRX01VFLNXmtanSnfZIvkuAJcIJvSbqJX8gkN6eIicKz6+ggRupyPi1Iv5hCrsUPOuw7UWfyYNBRVLp0ViNSuRD2Pro6XhxLIgwQnrOtPm\\\\\\\/EFVasrD366B4ID5\\\\\\\/100t7IbxHqO4m\\\\\\\/MomIwPBexyO0qVdBZHgX0kxQgUAZOsVst2oWOGLu86jgK8XJwjIIHFINM7yA7N2Mn9uMT+OeuoJyNM\\\\\\\/zgXQXYmftsRFakj9izHLuqGsA+8KRojYn0VeKkbztFoisjj33LY47Okx93haGTMhaGy6tuBDAg6Gb9xMJnwxd6Ar7jpixfwKD0AWXkZoto2ISJflbGNPC930A7JgT09pbw6pEhC1GfEhHz\\\u0022,\\\u0022iv\\\u0022:\\\u0022rapSxtXtihN70ZgAfC5Nrw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002250cfec9e33387d2aea052b8588b6bb2b1c110bb54ea49ff7b8464680c35e8864\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220Ir8OZdzjneajVk25h6ITfc2pRWX90ejpbHYZpnTpwZTDzFXYm7Gf4JJtBYsHDiUoD7lSBNnHvzVb5TJ1297ztn61gzgF5IABDemlBgbSp8XFIypNQOt87I\\\\\\\/aqzg3nqPk0XP4IfuPa+i4PB564nTpI7Stz2F+ynMkvn5A8LBf5L59eNVYo4B\\\\\\\/E7ko+Iwz9ehMUGC+qEXHljeG+WFeRi2o0Pw\\\\\\\/ET0O6OkzirHbarGWwhIPtOlAXAMIxNlGDT\\\\\\\/dX\\\\\\\/D0U9ar9fmTYmJFIk\\\\\\\/l8JmcSUqnMY5OPGBtTtuLQ2etgG4++9JibWGrKKIc9yU\\\\\\\/EBkxLwmLe\\\\\\\/fUIgEtLQ8gMBXQM0\\\\\\\/q4\\\\\\\/tRlYvvETOCypw0j40+J9NmGip8MYchG5Stl3vbgGiomcyoHiwA2AhK1Tcv\\\\\\\/8gsRGHu72cmnUfL0nmEhUjqZ9yUcw\\\\\\\/6ZQOYQM1ychlvKvkexhCfJOGPOFRS73HkdtJCwlYiezjLuMoPeXX3lKVOBl1TSv9j\\\\\\\/YiejUCyqHXL3Cjigu5QHkQeple9FZ2ZI6\\\\\\\/v6ASO1vG\\\\\\\/LFZSXwdfJflfUXS48gGTQkmXdjVFBpzCHQKWL0zquAcXwfmblQ0GrSpfjmofkIfJcmZFlgp8Db8cFQ6rQc40ihHIp132U8GH+OSF4m7ONsqsGbrAWA\\\\\\\/VjiQEmm0oVA3oZNBav785blThpCyR0a8YvznCCNC2tPXZxzzvOwZlbkpc0qsi3xQZ7MkSqfvSPGVdd2s3Pjusk7DEXIE3VHZ+kIie9MSmDl7kNiDzTvU89+IPpl49DHOAG\\\\\\\/Cofr9P9U04xgczltKaPLcv0a3bcRyYxnkO3o6bLXSkFLaaTsSdptrwyDbTEbNK\\\\\\\/GAscD8DiCqNofcZ1N2g+lr9C8kJvT3FPLy5olxenA0ic01mU07KWgLoDOu0GboiNK9BFIVrt7Cw0ZLldoyOx7oH3mDRPGP7WaplaJUWYYLACiSWDkm69UZ3hixUjxY0txYNclb9fIL5bnbIXEj6Gr6xyDgIbbqxOpK9U\\\\\\\/kDqQzjqED0I\\\\\\\/EDDrkolBHV529PnGDE3GWxO8qKXSenhtU4+OkpW2+bD8gHxKN28q9Cg0S400\\\\\\\/UaLrbdSF+7tlBMBbutphVopGfQUjJuF\\\\\\\/yys9BB8mel54Erjr\\\\\\\/nTQtBu0lzj7Vwm2qFJF0402GgjRQo1xHYg8mxKcdzMVCx6XiGD8o8ifPCF\\\\\\\/VaY2248khEbzBTb2rn3qX4YRYAVBnhZOqPbp7dYXFQUGPNTbwJlfwi9hIInCfu328zEdmS\\\\\\\/ZsK20JM0OaKkrXRf\\\\\\\/8MIof4ZyFMJOCJURs8bvlA3yW\\\\\\\/ZZBD2aCpr0sWDFTsmVdrCGO5uHeamdQ0dSx9+nGD4k3E9Xg0YxwWzw1Uz44\\\\\\\/YGctUQz4VnSYZcgAeClOtt1RuyTVO0a5jnYBK4YJyrz1XbfLxmgLZYmEo5qylEj32Uezf1BNlYQ3ZgklUj9ptT6QeuiNS+IajgCfnQKBNKKvb0P6bud493espNhpCh6btMmetz3IH+77ewfz8nbhkEvGECgbf\\\\\\\/LS2eSNkMPFTI7M+w7xt6y6jLVJ8IVOfPyBA1LN0fLeS+fUlaM+PY3gJee9D19ZHarVWQcnrshfZkkYPscVQn4Ak98BZF6BBAhDsH2ZxX4Qg6kWkcRNS2G+aPgqjLWc8qVjjCG0\\\\\\\/nVGLm\\\\\\\/mPzW09rSOPhOCiKJhlKLq0e2B5AA80O8kamzDxygik8EaZcukhleBqv9cP1Jo4xUD2cDF8P19g3TQQvTUxKpc5aalatIsGmGPLymeE8VVxWznkrYZ7yCg9goP9V2JfVcMVMg7ym1p4SSnhNRsUoTvO2FT+ZLYVysX\\\\\\\/ZQ7+lZQhz8gp6\\\\\\\/Fd\\\\\\\/cMauEpC2V+7Xd5K30W0lOVc3ZI7\\\\\\\/9Zjw0xrIJAi6c6upZUZQgahx+YJs\\\\\\\/CPLFjF\\\\\\\/dGylqH0fLuhThp+\\\\\\\/vn5\\\\\\\/nNiHxHkjylNVvlxzoKonT8VZNJkbpRnVcTTnsvm0DHRXSPSpiifMP2\\\\\\\/UdpJ+ahwn9jLcrdUELLDk9qQcyelr484H\\\\\\\/H\\\\\\\/mPOCmKR2JD72f0Cvkh\\\\\\\/1vYF1SNt+T1rGWKETS80fGR\\\\\\\/NQ80yN6Qs3WXNCozGvu+bhK42tXYanfhBTguNrPebiYDe+RcQkZPYYjONwgsT3nOD3QmRhQcybL1X8Y2RS4uvUxyrhlIhNAD79Wxfh8ZJnRx\\\\\\\/N+nWcjwQBn91V3oYD8R67ihZGuLxc9KM4672LE8fw+VuQpBRWeSVOoPmCXyPz3LR1OIC2tUQguU7uE3xdD1VgYf8R722wMke3E2I+eF56zJTzApHo+gv56UcGb9uC0+pVWL+TbzAk5zCXhqYFinOl8kvyEHPB429HTxsfUuJTxjQBoomhaDpxkLrbT2BhNigfrAp573eNzbnU4h10KTb+aS8G\\\\\\\/c4wQAx+eEQjUMFrUL3VUZtGr\\\\\\\/KZqLCsquEQF+tX1faX8Pmu8Ym5LJyv1CFLfD87FW3s+34qwf4LYhNmEZGlDO9w6bNE2iquh04tMs0v7Hh8DNFtGFAFa3tPgcYHkJiphsHJlPNR52ZUN4pDlg90elNM4lGli0Z6\\\\\\\/G6vfXIJLdI1jwvc6ObIkGGTaQ50\\\\\\\/+WA\\\\\\\/LaGHdP3USCNm3Nd8svKAYq9VeunA0+bREzcYWPfwNZ9\\\\\\\/F95laWqxrpi3dLs1KnObR7rw1R3xDY2T5T9II+3jtLpY\\\\\\\/7wukyMqZhI85bqWroiuwrjUQDf2wo+qFHLKo8EgWNa6fkD2YZZPIqCy7tkPvS6cy+kG+mX+PKZ3CL8yg4YzrSj6DxiOwAq1pCv1Rdh3BRbDlqwQ9vbhs2\\\\\\\/F66tzda6Zuzajx1mYEUP2CYpZtljyYwf4yWrj2SH9+YD4sjcI\\\\\\\/QyjM5cdxzDUTgZ6u2iQ29f0m+5oonqdT4GqDNm10Fy7miBvKKLTgzb2j3aJsW8ucf7Zsh84BIloHILY6sSg11Fnnifizc8kpnA6ptf0uFMt7gMbYYPVBSXgsvQAPGa\\\\\\\/Tb90M62E58aJKR8necJH29llgS53QGe\\\\\\\/GHS6JMAAvmUNB0Q7kvV+xo5XjZyTqcPKNr6dk0jh4HjN5I2KfcQ6pCQwB4WOVglW+wWyz3k0UPo3Awxu3E3owCkDXRQlyoc0RFPZA0k51sp0ctO2CxF7V0l8OxxY0oRWOlWW3+FfM+1ZWTR61AK1A0rUUkp0Y8mDYvorCP\\\\\\\/65FmbXjg4G2ax05EzouOPkC0Xj2b7bCcp8QsNTBV1GR0o\\\\\\\/6w55z6B54Z+Xhb8FNVtTI7nrzVs3D3L3G2dqX4ZQR4nVUZ2GgghYMPtNYyqMLcEHbb1fbezUtVCbBJyiM8ugw1k9NdByTRbH03RlhrxzDTmtBwFsJ\\\\\\\/jYiP0tvQt54m6gRfs8hFu58JCduyv8mSZn1+HHj7m36yogZzwF6SAc\\\\\\\/I+PxlLhYpO54KrupFqyQvmb\\\\\\\/dEzr0qLojds3GZYDHKZJT\\\\\\\/rh12k1h9Lio5JinhILNcopI\\\\\\\/u3jgjgGeaXtqDuAsuV76GAzEsuxCWK8B2FeDnkufgFMEnbwcIE639IWK0XalDeCId4aj6ynLPTjIvbDngVEY4udKzqYY2MWm3BUBxeo7ZwGsU4RyCB4DaxnrzL5gQsNJry\\\\\\\/u8MYDepxAGSvJvNoxLzki+TvGIMa2IIPQhiLiGHZ0wcX9TXsQ3RnHGWgUeB6j73v63IksjsiJ7H7IElCPBEov2tVOnxqqenpHVAK0bAb6Hs2wtHr8svgeoBxBVgxaeLnB3DPz18wETRyk2G\\\\\\\/r+mvE8ylyRYd8oFp7GrFTOfKyVN0QpJZCd2xpIOVZMIiXPUFpYWH\\\\\\\/TzIG56DnAz\\\\\\\/s4D51ymbreEvxV7aPaT3utyTUTcGGi9CN7md\\\\\\\/YVQzlmVGey3JLNsnuMvvXDUOFTgHjE3ysgrjt6DaYdvv6YPwW4iRaCNWAXSx8QD9vySkuL1KSSnvExybh\\\\\\\/\\\\\\\/etMwGGHwwP4Jp7P1GIis\\\\\\\/w9NIWlepwTuQVrsf3e9vg6uB\\\\\\\/A4he9JplicFMgHr8Yi93L1J1mlujFFnXFZHnkJfzwbr7Q8hHVMPjLcbhdjm28XAiraYaFebKG9vdgevRTUdzjnB+4GmyKfzDdMTMjr3M\\\\\\\/XGcNuQuiYYZpLMlmXrG8qO1HnFXx3AKfgCWrTZLwLjnglZgEJU2kF+w05wYuFrGPmmjazFjovQRiXoUMC0gBFJwoHI+LEKSzx227ajQu5KtRND5pf0f3VHeA3AVvACiNeSl6KnfUxlnCC57F2Rjpj8TRWqLihZW\\\\\\\/HZVwLW29HIhj546oTxXJf46qhetZO7TXTHAqfUi\\\\\\\/wZgwvx8wMqjoCcesX7\\\\\\\/X4zg88EivyAmKvXrD3aBxrK7gp7U04sbp+QAAPBe8wbKYvZpU25nf9r2colxYrkg+iK14Gv06fc5eVl6+s\\\\\\\/U2oGc47BayXM0OsrPCwSktUuBCoIjgXBgow1tHAtEN47AqHYs7F+DmVircmoSa5TjE4o0DI09vwCmybo8RCxUVF\\\\\\\/vTXDCoKjPeGyCnOvEMToC7yYphbblzpLRftljvWRNGGnGm0YwD0vsM7D5YVwM7mUTnIf\\\\\\\/PjsY4IaHlCjdnHNB0sCxTcfp2HBy0+FMRzI16ulPBtZYx9KmFdkxwgzsOfVhgeFVWRnQnP7pNyBUbqUvcj4t20T8icExSkWpoEWews7vW2fegk8JfbHEH71IvYaR27DkzHiUHexP+qlePYglS7EH4YMrILVn5QckDcoq9ggE2QBxrUFGZW8jpxJv6gwDfHobGy1pYU+jMNgiTudvuUKbY1pl6VxqvVnAPZPMEEU7uge9MOLY91yG5L6bYExZCjOLc2r210SZtysinbXBz2n6pjyDNsKsb7kMMbpr6stxzllE7fJs76vXs9ACBAJAH1cHRMv9pDv4eoyCUmXiXovDAli4h2cn8ehP3JPBrTcTPmzKTBH8Iib5UodDNfDaYMnI5OuJbheTW0hffz2kh\\\\\\\/Lwj1v2x7UcCJzYlyxmiADzHFxLkCmdZJs+9vwudgT3mrCldM8AW3G4eVBVQu4INyFhC6zI\\\\\\\/DTrrO5CjMnZouNbkayR0mhWOS1O9VL5DAUMw2CnlgF65mrhKkRONBX\\\\\\\/xcu+xr9Lj8uzXuiW5ICcGS0RxOQ\\\\\\\/ah\\\\\\\/Eo9RidV4nsj\\\\\\\/ioC73l3iFfufGNe5sY4W2CEuLk\\\\\\\/YWkkxBSifW4AIduZnGQF6aQ\\\\\\\/Rpt24R+gu9UvLyaVf85CvvR8LdQvvSsQrdRLEwVgYmW+\\\\\\\/uqb\\\\\\\/4\\\\\\\/2DvAXxfZUbfDiEUtutlkz8Uw5kb7xV4ef5E7p+T1rypzFJG1gsyqFVZWMUAs2sM4tkQtlxF0rpLnI1Y6b4UqByTb2fJRjBfrc9NRSPtoufzkzpfYSe72VrCKD7JTb6F1PvN6zQtHxGVDSl3xHgwWpztggVbgcj\\\\\\\/5kTlWp\\\\\\\/teTMdKIx0xn7ugkv8t+SZs4grUFD2bhy2VReGduc48NnhpUyLY5lsYio5gYWqSIo4N6BlfCPQzx1A7F1JTYRqBbPgwO3YMbkAJzS41Rr0y2VFE+vRalkBusAg0ne8wITZwMs78vJNokSzK9ghmxq24p1bFf08kL8aF0YHU3refsH4EsU81u76WiYt7HFyi9Ms3wGLoCeak1omw\\\\\\\/mCyYTFWLPE4bviLJbBd3yn5DU8CN2phgamKRxSY4iB\\\\\\\/J8O0drbj+RfTKVkWX4nPZtELJQ3z06PSKteBJeiuuoiJhxSKUKd+pEyj5kWNbsjTTrLUYk3\\\\\\\/o4KfGrooRcJen3MCDorlmnHBR5y48bJtQjaDtRAiZekn2FKTFNSGonXKNUzcec6oaw32UcXycdD1od83HBP6wL6ir4557Tqzyd1Kzwvswb2LI98HYkDOpsYT0w6TyMLZKe7SFisnCbH7fSKkSV8Q0rUrGVA1yaYHiPdFpXFmuA9a7fD8xOTM6bk1e5xgt9KQ2t\\\\\\\/EM03kZxphtvMUHKVbF2OV9KbQu8voHwTXeQQg6AJGQ3m2cNopsLbvCRNKYczJONKF3FriBfNOTRoGig5nFDzhaFcLDE\\\\\\\/tZzGfgKlU29TqsrplV4rptzm+DK75eDI2IN7efaGC8390Xuc\\\\\\\/e9VN8XJ75sY9nIiAXb5LosCrzAynUQC+aboOVY1hVwLYFtzJ3X+AKgy\\\\\\\/Gjdj3clVXjQDWDmd4p\\\\\\\/\\\\\\\/hzDlE8vywmqxvcBNver+z62PZzIq3+nm\\\\\\\/PnkaVarGnDAnhxxGmwr6bjLCxCgBirHm8Fs7R\\\\\\\/TQDM8rhhfz3jCD7AjKn8lCGELWlr0u6KyFN1GKi6qDJngNz0PNUfLqGgKS+dSRldHOAMME\\\\\\\/P5a6OpOtyoVnHi5TBY7f6VQ9\\\\\\\/J1USwbFIsf+Phlee3y4DFbTR7GhvirqShsC6+D2oMFWcxQIi\\\\\\\/3BuEyfq8C+QtD0vqQe7x+fN7v6LuXbVDU1wbvufwcXyX\\\\\\\/CxPgRyuk\\\\\\\/ZCUce5isbUhuwOfmhzo8ZlUe4\\\\\\\/AjqQGciH04HEEI+t83TmNoSrf6T\\\\\\\/hX+T+TXkD6kjbb5QP4dDEG8oB5vpjiRszegcVr2ZksDI\\\\\\\/WVdoJKqaJwFyQmaSiKZTwvPCWLf+TG2Ar3Ib\\\\\\\/pLR6WYqBfgqEHmwggwA+oETZDFz5j3c7rLp9eAt7uccLxUrRE4QuIS\\\\\\\/s4cMvXUJOIIOwEG0SqQ0NUifsWjy8ADBZQe36Jo1QWlHwAAu8RpfITmzH+EJgus3WYnqDeUyRf0+3bxOh9t\\\\\\\/MVe2fI3ejUXZAtIXNCH4YToTEbPQ9\\\\\\\/adOK4IhKTGLm4sfv5GCZUDhNdqgJljVBIRWtlr3+EUr6OMnSB9deYhEBwhwH\\\\\\\/tWeImUMDaj5xXSkgCHOmrfnvn75+6dDjkilSANouYK82v\\\\\\\/3kSQ5PZhP5sEO3JiaZCO5+inljS8VTWZBCvq7KMiBVN7Sx+Wm4o5Yb7CvnMsxIurh2JI\\\\\\\/K+tc9Q3cfM6sbvtwt4oIE118lr6+jwGx18r\\\\\\\/xrZJMtsiOk+5NhIsfyWl6cWKZXLRZ+\\\\\\\/tQc0cKd+09WLIC4MFhRC2qPF889\\\\\\\/gE9I8E52Z6OzqcwnFhTp2BVht1+f\\\\\\\/s+1QavKBaAX1yB\\\\\\\/CL4ehvfXVFmPi0J5l218Js2J62IeVQUSggqvS\\\\\\\/VqGIk49F46rnGRjRnL68EwlQZBKNuIc9vgf9UsRDqEz9I2R4xBUkuPwaVr8SmkmPRi1z1lpvoN\\\\\\\/+RK5+KdXWy97u9d7\\\\\\\/knNf1R9Q2rwb8JQhmRCh4rjNBMCWMj2xqNotr0S\\\\\\\/aQToqZChTnSNIJ8qdOoXZYfOwugD\\\\\\\/b4Z+bCzEu9oz7tAwU2Grna3KNtIXhgRF3o8+fkJCT6OA9X8dDOB6omuc+lk5ceQroR\\\\\\\/JHX\\\\\\\/hBozmRPuELL5VkuOicIivkhxuQMSVgXto3b1PuGIYWEKNxHl97knTJpk3RVy6dvvarOxAYkbqoqMGwO8hr068ydszvQY9r1QKuxNeN3jyWTIldZ\\\\\\\/Buzqjfz8hQASBRrcrdCNJExobS8KgZlEigdggRXpC6hmbqz\\\\\\\/4JBQZHW\\\\\\\/MRD0w3JgnnnDSOoLjVp2RnnutHbQ1lgDo10\\\\\\\/u6XdUT+buc+tf31kq1pbGat3HTq\\\\\\\/6X2Pe0nLaml\\\\\\\/5Ev3s+HD2Jkx6IRR9EhjBlY8A91lAgFYZCWFL7wFtVQnT\\\\\\\/1liOVCr2As4adSA8lj\\\\\\\/mkmZaOtcyv8OemZBeUcbVjr5Ss8krqVZYNIQ5b0pJhpzg9sPBxqD3MDCtqbYN1zz8ts5kX4T+ELvJq\\\\\\\/U5RM\\\\\\\/cnkKX5ImcY5oTvxNn9GWx\\\\\\\/uTm0KhNYYRbK7WLbHwO6cRbQcjMI4RQIcRgRNuRrPJV5teolDWR\\\\\\\/3qHB5DYix4pIQlgXSJoqUYL0otI3zJ6G\\\\\\\/KYsskUlrMSDlmW\\\\\\\/+g3jjy+8ZuAI\\\\\\\/F5BO+HH1L9LJG+XoRKwjUE\\\\\\\/8Zed4rYVzNrfgZPvMac7vrEG231DI4c7PogX9Y+LXWomTzzSi6kTPvQaIZzLkY48qPHENuIoZCxFZH1562PzhMyOddT1bqpTalxXQTG0o+Z501GR02CKnksS5LY3yJCuT3uiFCXHOSVnYBpUolY4ULgawIRkEFnSRDi+PJDNPuONqS9B7qmtjtLIQcS5SDuwx7bVJ+y9LiGYKCil3eVrE4XS6E2bbdGe+4VtV9QIg8xGBYZeACcZ2TYRYcr3zOvnC2HcFvI0a8Tgd5FGq8UGU8XFD6HpbNQNw2zwnVZ2aoYEnUJZuRpWNBSBtnIJfKj8pHlxCHMyGVoK+O3F0pWF0gFQfpfp2mT6QndvnztVC0chXAFI0VYrdHgc6jekY7Y6kuVEt86BoipKBFh9sLccYgK3J3ublUT5AZIm+g3rD59KufjNEuLeIUGiIa0i\\\\\\\/n3TFKn1IXEI5iFrZ0501Vd2r1NJFEMnDCPVO9hpGbbKcuQLnU3SLmDV7Q1E+aZgyflAYe\\\\\\\/nyRE4gw+7RXHRTC6ihsu\\\\\\\/D9xYKaqD0LgulfX\\\\\\\/ZhfEE66DDcOS6mXFkpU8X+n63V6YO4UWmHbKR7qRmQ2x1G+LLjw8gBISYAk4D2BCi7Aw3S5dze6U45uiQmEiMnXJOkmd3mXQC944wkZQiXLxNaPb9jUICZH8rIJU4eMQREPjzSLz6k89rKC+aaTTXUZ8wJcuhgNpPH5NqR5IWSmgu8YUGY7o7Zu3W34mfrlKbScuimcwG9UwGpjbhkTrpDgjttCKG9Zetzt+8PYdHHBv8JkWXIN8y+EAVFwLSSuJ7ZrdxoXIylct\\\\\\\/HOiFTSA7Va6IwbLVv6D7Y59\\\\\\\/tFSRqAj2LcYfbmBp8eaMoxrOzkkiPnsp7b4TWfXC16mEJKcgde0c\\\\\\\/yPJTSYC1HW\\\\\\\/PY2UG5Ky21OMbFgtA1uMAXSP7110zVImhHGh8\\\\\\\/\\\\\\\/Pc9\\\\\\\/l+RRKDFD+a9lkmEjYASNtyIJ8ftEQvyGPoG4gAIN1RW4WA1q9cHmYK8p3yBROImdduQpmR8HqtdgLTpvji9EC\\\\\\\/\\\\\\\/1VxSTZx7jN9lEnV8VRs4n0WdN0LfCrXlByDFYQs11+VUAAWp5\\\\\\\/Aw0Q+XACatKai8pWLERa4IGDntypvezFbZvYgLaxtbSUMVr+Tpxzn\\\\\\\/DWNmFE2okMt5ZfB8dhIJUsPRx2l0wzYNkA8XTGWy9V0c9qXB65Kr2nGe6kYLeODLXP5XeTZkXNC99PJ6AKEbj1Ql\\\\\\\/bEIveZElI2SnEsgs\\\\\\\/WXJpFH+idxBPHsKeWV39cibv510Y2EBDOvVqbT9X+SmjH4kjcxTqpW+tvePdwcy6qQ39dEWF19SLqN0mX36+B0Xbo6b7r3e7oOAeQKQEXxQZRuem3BrbVtqSQ1sD18Y5TVc4dU5TKS7jEo7L7S27skSjcnmVtI7cQCdewM2mNlhu5epMpLjEYHFs5NnOFSt1tFrudckJHu0ZHY5LB6tiC76bCF92Sp1wKICEt\\\\\\\/LW1MslDFcuxcF\\\\\\\/oZf+1Thb0I+i0dW0YflneiOBQSNd8kH1x+WHuErvP5QSy\\\\\\\/f04ESnJfsBgSv0Y94tz02J5F\\\\\\\/s\\\\\\\/M7807SEWUQZADkhzOQzLaQzSxV2Ih2TXl7CjqD3PJSS\\\\\\\/5u1Yhrq76iUc0YGPGrG6zlGIIAxwxcCRCX16NgNMgoLQYUURxGI3dC\\\\\\\/SGMYLBpQ9L4Kn1kzhwryTvNr638Lp6HYEMIaE1r96k5XR2JNH\\\\\\\/U4EmXIE9G7TvhuBpJ8u4XA0borXwmw63cpFgbm6SCraBintGClzp\\\\\\\/EofraqIYkTbhcCFWmqo236DkmM1y1kd0Y\\\\\\\/M9rXecTFoYgxEYa3Rz\\\\\\\/kJChziEnMLzC6fKt3kIShAAtOZNt2ovPqNJKFDosn9fx+zp\\\\\\\/JvEn8SqRnqqJXyXT0VcitZK9cFzc5n9xoSrakGN38qWg+mQ7hwIlXbzS6b+GBQjrHB6aTR7lZ0Dw+EjtnpxPjKu+Wd8bC6JXMoHwIDS1acg48LtRhSp\\\\\\\/YeMoztc07g44hhGD0neq\\\\\\\/e9DjKZRZZbvVqoejZqXmQtEer7WkbNCjtxaVakWdtX+uu8hmyXcxSdAgw6hpz3gKtIEH07TCiVus0EZP6CkEWNCF\\\\\\\/w8adcGsXS\\\\\\\/ODJAb14xVPsqepoefis4WxUrO9kisGBDqcuDpv0JS7waI5hiHyTtwakBvkNrUFNH6LPi13UTgOX+y2kOnQBhrmNsvHEAv6NaMa7LjWblpGiAnEOcKYaFGnAuNRCa52m0q0sC0WWf9yb7Cz4XOivJUkb47+zkc0ho8VtJNkzBtvtANKRJL0kE1WUHkZlPlobR3BHXk601XO3wGELRUd9YEdGfWsfzp4NlpjXoY6mUP3MgGzj24kJeou0cGS85LesZ6rPDRVDJF86+S4E2qERa7aKPj4+39hGYPGfyxssn0NdTsmbrEA2nwa3syhpk4sNSu\\\\\\\/6e43RDSYzPE48RrUtpTagV2KgSSpvohHIS8NC2b2cQokdM6KzxVIbaO+RX98vm7o+Gi8GIE5r1vMDKMOTpoK0+htcYagvLjRlciRGTTwSg1vDij\\\\\\\/i86mtLvPlNUmZP3w\\\\\\\/zy5cmoqjgMphDSfxDR\\\\\\\/SlLeMJuY1wPnV9TnODfccEOeF9otjZl0RrYNiVZBkhG2WysinNKgR+\\\\\\\/3uiaIP3bZnWEqxvZSBEL+b\\\\\\\/TRrTlDuBtkJsrOD+yfEJ6d2Jpi\\\\\\\/xKUep7Yulw3gEx4JFg7wl+j8wvXhm3oV5gQMs7nWgHYUBG2MRDoWGytDOWX9oYTmI3CjljAmwuqxzJsh0hZEdk8pAAZi3Eu3+kLxsR9gTBzwXem\\\\\\\/ebhdARTfYECJ798rn7l7pbHLhRepI8wMtt1MAl9Ik0HmlFusopULeNy47947WgKJyzomgG+HPA+oTWPaQbNnPt4azDJIkdI\\\\\\\/MbAOTHtwTcKSdbCCM3RTr9\\\\\\\/EwggzYi7lsWVe1S3wqY2xOndNHi151NfYY9yfX6j9eQI0lsfOo4yYXpsuNFfp4NG8vaNm6Cmq4wiWDT+ljZbiYUMUwQYNJY3m1FA1pc5sBqDWwlFNCNLXwuBeJzN56tWDMs9tvqgaHvyHDpoa\\\\\\\/l5wL7xU3XszSM9rU0NRVho\\\\\\\/TOiN936ggdnTGf+1CgiSGPDnJlwo3FlBIizHKIEmJi+oOZxPTxf4N5B5uVdkoIYTdC8gExJehBG2yksOjX9MLvLnbIzsKHqnrMsEzKIyboGr8ePXbSE87ecdro0WlURSO22RYfmGkf8Frt17gUr7SSlM\\\\\\\/rppQWryvMojYB1ESJz4MGft8IknasClTk5VBsuzR7Vb4DhnQcJuLBiW\\\\\\\/kWyZT3GxasPnOjHc5VEHDnGPV3WfbWK7XdxA2TTsiKIG5jUp1V4jfN+8oGOuKxGicBaooDz1\\\\\\\/UnPHMbJTJREusmJNWzWSMB8bZYLjuspblPPsJ4qggWYk+J8SAc2F+lRf7fWl7MpkyR8eSNgX9g+YrC7zNBjtHtJsxf61fMK8TOppU2lKDxYiXMx5CrH4JppqKJqSw5Arv\\\\\\\/pi1H80DXcl\\\\\\\/n7YkKwagvC3S2AdEGdwOVE8KAtKEZ+t5vjn+6yulIJvrfpiNVu+bdUZIx6i\\\\\\\/PzGnjNpgzSHkiXFbqT5MMVkRXqFbZeAbrwTYQlB65V1eBAyRov0izsSCavm0UMTli245qIcZfFCfZXyP8tJfjR+5WCZafdZuPpuzQZROnywQ05\\\\\\\/S48E0y8fIu8r9DWWhcQfzYIqlaB5z7vCb0x9RkJC+jBzWBrJhJlG4QHqhUx20jep8JgrS11NlIKgyv8GD6GakMJYxYVQsv4NFIqfAiMU0NkNCS7V42RQcsbf00nx44sr6C3cWCgjOnwTgM7P3MDARiz7CKUBcysKQDLUTX7jaF5f0YaVwzSGL6uZOyG6bOykqTXCHks2\\\\\\\/kku5wgsYDsRMakQj6CFSAbzM5ruSR9h1vWtGZoE+QLHVkJm7lHM2wn4lsOpN6OcM3oYgRfxn31dgfP7Xcf8GCgTdtyTYhLEibb66lT7XeZHuM4NLVCt6rqav+KCpO6bZttZbNpchjdvCp\\\\\\\/6izQcL7+W0Tw+o4PtEUNw3yHcf7+L7\\\\\\\/AS0XmOnSR72+CwUeeZZX6qvCT91nZG0P5Y8YCgw8exOOuzB\\\\\\\/zgPXfN8EoHfXVMMgo1OjTWEijUht\\\\\\\/nzAuBsm4WoJDlhyHvVnMbiX3c7tk\\\\\\\/bkOJ0c\\\\\\\/YiiFLeD8zxTo5CQmY1WRkBDce7JFACuVwUxyECi2V5ewl7nDzkKmVQMNDIOwdQN2BO+1gWvLNM0x04zNAjyVjvgPcuWMoczsojrbvC5dWEmNkWq++L6SCxfPtlFALaacnCQeyv0iDC+e8Ip+ezMnvf2EYcLSayN3S3R9f9nVqKKxXC4eQr62CBiyVCMOQmUbjvi5zDweJylVjNk1okS1mPVN1fTnZ1Z9Xd4ADy5T3\\\\\\\/7MsMGcbD9TIfWE78LGfjJHS36RGr4MJInVXAUXav8NasqmffRJZsDWsFCNjXo0093V3mzeTyIYkVFmGGbTzX6FFnvIA1KP8aZbKqAxxFs38sRYVbUJpgy\\\\\\\/exauAIaamOuMqMQVj9Ahfm46oLYNUnzKyR2hGaz4HB4Tn0qg0lXMYUgNOE+FyeezW+EpyxHKdEfCC9BxUgwWEDg6vbsO0amKWjRMJkxhpDN0p\\\\\\\/10VLAU617YbxoutErMQFhmzD3nd3wd1x4SoA3RgCDbkeBaLhX9gpNksDt\\\\\\\/wLmoZ1wnRNxiL83X8U1UCwjhFB1yTioFDqUL68e576s8Rz+RnUURa0M9T\\\\\\\/yUJkSA2C3T9XYlNxW73NRxc8gmGG9MhZusIQqK8MnmiOGmvdhj5cYjWz1YIQM9Ho7OIcE6xUtgQLMu5NFPUBJd+uMnJXfL5btYFoX30TitPsCsRgWtDsvUzAcrEoxojKPmJGXfjXK2rJREH+3TqSwbz7ErtsWWCGnRYN+3nTjjuTs7g8yaQpGXpJS7A5XPsea8uu5YlgMEuUgKzF49BmePqhsNb4B2EluQDX3OiktzZ7tLnkeQPG+YdlLkz1\\\\\\\/EfD7e214D7BtBmoJl5AZEHyA4Ps2jjMswE0FaTSb+tQ3r0vUPmND2Pc6H4TXpBKnFLLiHw1fhTUd5KRK698NcccB+aqOfjKpk3qy8WW\\\\\\\/6HJURRUw4jBKFW8a2gg7CY8RRaFOfQSUP2bmv+P0sOg6RXDPewEjg7UzL50TsdyJ5Wy7ZQaThgPSRBWUb5WgMYIBrklkO5nyqCjRk2lEZwKtQgATvG49GeU5lulgad0IsJHLf4XJzwsCPjSn0I2aMF68cNDzMFbDjpBHtdfBT\\\\\\\/rbkuBKn+0a3a4\\\\\\\/aWslGscm0EhGzd1ZcaQJL0klVYbnkdGlzGp7xZzxfrh\\\\\\\/d+hUlEEAI5fWqcaOONaon3AG7F\\\\\\\/Ieo7+Q+IUzPdMWELk2zkL7gTy4VytHpgmxXtKvR09OVIwj+kk4D66urCPs86D1wp3hvpV+X0ehs48aPlwPinwwK5UJzLsHJ9CKMlIuEHclzRiNNlezve+49Ons1+5Cf6JbvpiOdT2TOFszWTWNcmM4p8sZHb5ocFfRXscBOubWImoqs1Th33NhPdq1QpA5B\\\\\\\/UsXzJwVjDntE09LHIud3szbcPP5fdkQRYmqs+DLxt0TQEE\\\\\\\/PF9rEZU+8M0N6TzHBOvxpKjdV0QXszvwQ3nLnHAqshBwz6YEPybiZ0PuccJKGoh\\\\\\\/RmHkJonQUb6WzGnyrWn2mZCJdoGSFXyvFLC0PvOk9xYkeYxrTGCDnrCOGK\\\\\\\/KmlAvQ\\\\\\\/mUBGfZSPqWmy5pR2jWYXq3vDvUWD9+ZNkpbTzDgXV4gbIvgD8kpiU2xkH1Xxv\\\\\\\/lh5U43LYKqjJu8wUmNULWhD4xqLirooak\\\\\\\/z0lV3QE7Tnmq4SYf6XcSvA5Yg10FI1bNIQgz7WexiQvzkKvu4yQ4wfLIdu6tZcBxEJ4Ni8asHzjRjOIEvNhj1YOU8f0djEZH21hmhuJRYgC2qeHhk3NMYb6SAVZd2wdSibyU1DItcJ4pTqCBjF\\\\\\\/IWXmy7U+GxTYBKrD2cdbVA5IJNa+vhnnm3NFVj34ai+lRFaZWQesL1wdt+tPuSGDYvXoYxFMryOwkjZsAL4yGDl2H2uYA6Mk37pyL\\\\\\\/ClanFvSftHVrl+ckCLJjlCYH+CxDCRbBg9DvesyJEza5OzKJkVFAbx2pOM4DrEVClPEZvX7I6xWoiEgxwep+U9KrEvpt9IWjkGD7TqtN35reuaaMgJ24tyb2RiuGoONdgekXWyjRl+q3t5HH4y7NUVN+NZO+or\\\\\\\/NVqASGMqQ9XxomhtQgUpMlgL9m2VvopVuS4M0LXGb466EhNQbjAtY9SQvhUxlBsTbp9xphCyZZ1OdJ7RA9wBOGN14sGknC4J6Kv7zgSD1xef6o8cZQqGb\\\\\\\/jKn2n6GXVxS2BXyaTBQzVhroNbyoA9trRjLdno1Vak3VbtZjZIGH35wec2fyTSHsD3N1yEUGiOpnbf5AzU6sKj3bfAFAl9fhnKRSrBV01+dsyo7UQpmdtU1kmDlhuYuTdZKBzcTy54iHWrGSMIiWFtWFk9JcwcWBb0SA0DvAF8RS2uhg\\\\\\\/qS\\\\\\\/ZCf1zkcSdgT6MCUR5YMZbG4faz9TBz96BTrE+OS4u66IMgwdj3MskD8wjfPPCZ3TWHjuWDf9c1L6g6WmlM\\\\\\\/5lM6omjdA555SK65Ii5odigKnRbk7GaHnLLjnRJuhHJ85fpEgIjAVRq1MO24\\\\\\\/dd3H6j94TfMdgoLmaBOcfKUsWPc9Vu9q5VSM5x5BtCZXJK4JXgV\\\\\\\/hWsHWbLxrkdyQM62gDS6GKASJ4aXUGEtgAufeicn1PtSJIK8oNH8EJhdum0MkcT34r++GmYv3VZmEChWxxVRHPdcLYz1t2PulrXn7t\\\\\\\/OX00hN9cXX\\\\\\\/Ll8IFV6MgU\\\\\\\/JKbhtSlrkp013WL8e+\\\\\\\/ccTKT7TyIvpVrMYEcE4E2FYw5wZ9QZPslP\\\\\\\/QgUcjp0KwKf0Q7i2lVZ3LzzKCn4A8I6loPPfCaii7nRGGfG+gO0t7X+FhrzZ1fxPHeiKI5chFn1h\\\\\\\/YDSoxkUG9C841BohdQ+j12TrvJXLvglRAXCkKl033sfdWSU2smtafGDpZSMRoT0jU81n9oBzcLrLaL8yAW7FIugV2vryWz0oLwAW\\\\\\\/tFcoGAUMTIlwHK6xtFuw7KlmJ+s7iSmKYDOxNNO3puRJpG2nmYHEULbARGIURq613KWepBx2gKFj8lwpLi51xRLUt6QFCcDTcKGJdcR6JdDSvZuJdIIwOVJQRCLFXCpPYAdl\\\\\\\/keQ3CSIP\\\\\\\/npXN7MvU7nlFkiVxjlYgt+q7Nu5ha6ixh\\\\\\\/rgiyLhiEcC\\\\\\\/7IdBdYXmfCmYnrPCV4YAtjzotGMWmPEtXo1iM+JBRGMQvKeXsBnaWLDRpUDltpFW4uB6GSFSYWo\\\\\\\/5x5kQ3YLw6A818GXyN2QEs3e4D8hSoPK8WVYqV\\\\\\\/y5jXrwVJ8jBT075ci66Yj1H6HUf1YbnUKmUK2U+Dk4G8IuIHEvZKlwciUCFdkDM4BGWF9IosJiq+i8aJ525c4ueUPbymSFxiOhxwfDHwc2np62mf4bDcJjFMZP3BrzsDmTV3DrEmD\\\\\\\/1XrUyzir\\\\\\\/e2vFd3CgSbRAtZrEKmqiq8GYnzep67vVGZnKRcXgybhoKfk+\\\\\\\/MFy4q8HBw2xA6mvJWmmPPROqsx2fKw47LURLfLHmRAXt7yAzfhvwbp1oVmyv\\\\\\\/3FAnw3BKEzVju5dB3Mi9D+fWb3E\\\\\\\/yWX8t588xfI0tMZzDxJWbBwdC\\\\\\\/xZSOIKueomIiF0a09rxfrHqaCeRh+uRSctEeaUgObZAplV4lXQIekeYy0CUcK8+HfR8ixpUPBPg\\\\\\\/L+GHUPC0w52GAwg0SCsSAty5ylX7j0zjFIp81q+J0dURJsJxMNrvKw0wNZglw7uRX0XGfA7oBp2caWaObwGtdpppCIQHl1Fm7Ku1BZTZ+fvKbi6vsa5rmJyPr22+1Bo0SMbpCmVcjuMhA5V0K9hm5pAWoy1+lpgnasSX6YLuM9jqWokQQxxWXw7D0o1Hon5t9UexN4seK8cQJK9qrwjhifHmL+J50WUgUWGzkkRLm8Ilf\\\\\\\/hRxyNXV9gSN2AzAHifJsNtZAgYacvMmwSBp4Ac\\\\\\\/NAy0N1e0dGQq4YBvoDPeUrYlftnZPY\\\\\\\/+e7ZCl5gjuuBOJXjT3\\\\\\\/LtpSZ1smkC5YmBg\\\\\\\/UQ9WrsrMZ96lTOIw+zsaKfU67o+ohUu1WhO5ch7n93UX3D\\\\\\\/Uvk8aQ6SDPM2uIynSSAaHpPvGUhhHZnT3hd6RGb2x3n8jOYmmQ68In4coAbJA8ULnkwUuX42RakcqaA2s7qJAB\\\\\\\/UN9feTwKn4KKIf3Le11tyH69C0MgJrEfS6b6x3P7kyr671mzLDxIMImtpLeIzCannkXpAdmRitotVbUoSqsukVxrWpaTqKgo109ecrZGimeYzYY+\\\\\\\/G\\\\\\\/K2sXLqiWANO6R0bD4fV5DwKFfuwOvJ1f6pVRYBMKKzp7FAUXfEvK\\\\\\\/eZan8GYmSKdFEo68sXSkeAzsqCq7FKP6nEdsH59772nmHwEk0jpRYX5YGMPr95PF8rCF3+G8xfWkL4\\\\\\\/vJwRsQcmPk\\\\\\\/zXoY+f+3\\\\\\\/J7e\\\\\\\/06L5GmT0HSJJgsx0csmeOX+ITxeQc1T8\\\\\\\/nchCGwJIlJjVDRndguad9S9vVQQ2kPHc1+0cAuH9DvXKWHZ6y5xZPjGaz0GFdRKLdcANfsOAHhnh7mWM0ao1QLOMdf9YfTqnFmnxlBYc+I5iXaOKhZBEjDFMUI4cQpR914wVnEJXL0ts2oP1vbTz5BMa5N322bW8ioV5My2qj6HGDJPTiWowdsrt3AhAz1VqIakwUfy1PHRHWKEsdODK4VeBxrchvb4dfJiGVI20lh6GMDHFd\\\\\\\/hn9djJNxL3MQ\\\\\\\/\\\\\\\/UiopJsKrtihEsn52SXk9\\\\\\\/t3EsBo61exYWBrbZSf1QtB1q4epOT2mpsWXeum6WZUTHHhIvx9ei6SAQOgl6VQgpeyh30VUAhd9+Xid44Wd\\\\\\\/PwlF3ZqWe\\\\\\\/3+dMgMzJ892mdHPhBY6+TxC7BQLnAmwzrBGJA0y6zE1S8n0jEci3Tbh3HaGCYFSonXMnr21mG7X9PHV7hkLsXfeNsCwZ73JyX59CvHsUqtMP41lm7EVc\\\\\\\/Nzry+YES6oPqRl9MwOpCLMTL59YUlAWoJYTu6EaS9kH9vW4tQLJjQWsT\\\\\\\/ct7SH44WWAQ2cwdtotY3ZxjVbfcLuOU7vuLy0jwalaxfYPqAUdTd7lb9VkHlhy1JAkaQvGHfcVC527cw71Db91g\\\\\\\/nVnlfJ6zj2ejk0gJf766917uGt1xEZpUrebCiqJvYhBLguZckUWUVcLul02u9X80oA+8AAisBdPOF6HqSye8n2Aprwn+toyK9Vi6\\\\\\\/lYuojIHbn2Y8Z6M1jkJLcsinNnFmEkD2CsewcO7zJsOJLvNjT0Ml\\\\\\\/GKaWFJSh42DXw90SDVUp2qviDHy5WpPLh1nweFkZGSzeMPH1dVzp3\\\\\\\/RB9lnK4dOgwVdrp2pKky9T8x2gr+mM9enLTfIbGnDFdV9NrB9iK+rjxZ4X93xkLNrU8OcEVYv9hkmWAyhkw3aKRTSVj34xJx\\\\\\\/YGyJ0kvfwhm+NtGL9fYT6bKJD9vp0XwvVC+l45ofFMdJ94q3n5jMCBBmj\\\\\\\/ssE8ZsR5LjI0BfmAGbVmG7ckLF47TqXwKJUY3uFtNlmRzIE7kNWJWjJvOAFLM8WQ9TX+SRCQL9FP6hw9B1NK8MBu6MLz+NBHld2ovqEGD\\\\\\\/K3Ncd5\\\\\\\/Kj3IMWGrgEbpIgtO5QPK0VYff8Yit7sjQnBKFkAlXU4AHT0MTdOIcjtqobcZyrqfGp3lkb7pYTZj+ldILLt41N7DL3niWod\\\\\\\/sd6cgQrngLiWToWSeit53Jas\\\\\\\/anKsTlQdNjkcBxv5l5Y24nlWKMnQgI+ES7pdWgIdhDlYxuYKNrhohb6tKthhgsN8br5MDW8L8fXzru0Bu9adcYr8dHMbx6E5TrvhDbM2GaT0rGhKmT\\\\\\\/aZHhB2SbrN5lYmPEokBAejk0LpJNNNVjarPY+b8Uf+yTO1Zys2niOsTTFCNxiCf2c76aDj1i738CdXfw5FWp\\\\\\\/GyLix2n2eTXx6B+OkjPk6sK8d\\\\\\\/NP6Tf8m\\\\\\\/MDGGbOJ1WrEJGXAxtNaHrFCxmRO8wJXgDiAR1xmPt9cIsDYNIt\\\\\\\/KOFeBZ8nh8umxCMSl68BkNWqvfO4HSLd24M\\\\\\\/o3z\\\\\\\/VlMd3aeXA8YaTBnVd7PSNJCXiRRQpRBuCSFfOdmRoVS8F1UlrDAUa4kTYcGtdcuGyvg7m5nx6XNkF5ekJGHjCKcvzTK9aUzDTa37xl18L1rdwtSgBGpzzdYz82Ya4lyScU39uZXcFO0cXFc5bsBWHL6BdqumjBOcPlUvIggDfnb4U7338W5aqZNhJQaM4sqiyx6ycGdJriglPSFmSPIcVdh2WzydXtLySdE5pnU\\\\\\\/MF2CbkOkQ83CUnAUW+5Q+IvniNEpB6WG0hY1mXSIxKSCMygeuHLQOTpBRCJvENWQp4bcH8Ojq4mj8uciXQ1nRyaPzHIfxqWj3eenJ7ISBC++zuMAy9M6KubOiZ2pnzZngT2eAb2sGcGMrfr7a4BMFVlgxPa50U2WFn9dQJOGNKD8fwTFCW0QeUDQJlN74TsZp7xpV5dPsZsZP06DGjaTjSm+5nYVVg\\\\\\\/\\\\\\\/p8+C33Q\\\\\\\/iPfoEtK2b0j8DEJPE9F2VwUs91mUxFuAYSsptFCmvtDYXL00nfxirfXrJVGJSC9hUV1maX2aIOK5WI0SQCERothSjBYPCbdKzpg2yMUOWKMprklXcHWA\\\\\\\/4+xBEJoju\\\\\\\/mY9PtCg2uS5jPJflPhu5DmSr2nUXFThWlRc21herj\\\\\\\/qMi52ZF3HE0t4ehsvn\\\\\\\/QSza1Fbt9L0hXxorfiAHtQJmc7e99OnsK+BtOh2zmYAFSEv\\\\\\\/n5iuiQZXYNgqg38dCaZLfsYh0czC5Mo4usEDZlMuiW4Gt+QKHwPFhYe5lH+UtFL6pbp5nwWIVGisVMP7UopQkIRXZMVYkKxaTGwAxHgPUTYCN2pgU8zSLsB1WKSHntxq0P2u8CQymvqMNukB5z\\\\\\\/QAthbFFtdb8g7NUwBUpyUG1lWu3DxU+omrvuE3vUL5yfiHbXbJiL3K9+1KF2+0f02bETCXGGhQpqvP2dIt28GYUa7pYYJodQBo0QYWHx0pNa7OFeg3wsHXNXcGVcFVWVhW3aPcYzdmMiGmo0iku9S19ElQR1+jI9\\\\\\\/UjDS68e8FSkU1lqN8ndI09uIbaNqWb7CTnf2hdl1d4iQnPpfpi8N57eENqHoZTvBg3wXIJqudDAm5F4T\\\\\\\/asAyXiGwGMRbY6LBwHMNvms\\\\\\\/gl2pC7z+DPZ2x6Wm4RS9vHuWxkYYYweUY1OsGz72XjRkHaJD9\\\\\\\/b6jlwtM8DeTMHMvPqRKFK9FGGSH+xc0lpCV0oFOE7zFIJwlMnDvF8UCH+cYFLAKftqydqvS2Vc7S+6Xdet2M11d30CZjN9yszj\\\\\\\/ZRxPcZDe0m7dny4dXCmVMly3Ah10Oacs04uXP4R35QrMdJhjWDCY2T8CpTAHmOIGkb7AO61OedRB4XO4dLdbL11J\\\\\\\/l9Pxy\\\\\\\/Ic08uRQaZXL1diy6Cx2SEkLZIHpLFav9I8jMOERf2NjxGsvXUPjpY3kLLHiIZ2RTX\\\\\\\/H9VVRE6ieJeEtyXILSzKdojzSNcpcD+slQ1VEp\\\\\\\/g6\\\\\\\/Yt2IupTSNhhc6QurdGucPHExL44E8nkSThFw5ZU93qFPA9+urSr5jNat5GKyX7mWD7BFD91aemjeaDylKq4yMI4LRVIUReOcdRJoPZfPTKMhulUAXv8O4Rb5Xq3bP0iZkl61257CPPUAcD\\\\\\\/\\\\\\\/pU0EEiAT6aYrdOkFZw1FIHrdcfsgiONPViAFWVD1uBr9WLdyeLd4QHNXT4MwVrEdQE0EAqdnztL5RKmqaUrdU0gmnf5zuQFBYSUhMOCjafk87KOHv6msAnrz\\\\\\\/OQsjO+RRWLQjwsUl0k2hQBi0BBshz3bxAPijcDHSWRpxo0DvBkwlnoy1K\\\\\\\/mHwtvc3Vl65uW4opUuiuWfLbHnQ54pnKCcLN7Mq4uHUMPoO7alBaBHNvqeBytgUQ2fes7V0UqQ01k7Boyft8WVMYw8u3KnknmXo2vBOVN7XHJ6Clkox5ucU6QCAOaEWJNbmACxJ0GKWYj1VbC1IKKcH2OfJHtIcJktMSfnqIP6u3qumZCqpJvKRaFGpCOkkakWvJutKva4wHK21a4\\\\\\\/qBH86lB39OGk7edZuK7oQluyS\\\\\\\/4QmQr+IL7zERrq3fH37hL7K3P0NzErw5q6gLtUAQnpXAjy1hQeQ5bw578i+uHGXFKI7M7c0J0FBSBOx3QzKY9a3x0tDwm6WCIv63aKTfyCLWyjfl4R7r3RmqQdZW8uXileVAHNSWG1x\\\\\\\/TeSIeRrn0KIQgOJvW3N614bxvd7fjwDlR\\\\\\\/fJemnHtFKhp3hT4e2APlg9ySRTZtQjTETyMCYIkIxSXuoOydheUae34rp9pREBn0OubjMpOMyZcGIuPR5Abllz64eWVLiFe8U\\\\\\\/VZ8wjxF8b8UBjVGhCL0Idqix48uc3fLy4WJpQumBldQnl50RgySeDMiDHYI+yRIUGrldSzYmUVMOLNfxId56BAcvUhshgmxF\\\\\\\/umWtGQlEmUYrqhswSknwJu9\\\\\\\/JvRERJUXXT1TCKvzYNtvLvEMRxBeM\\\\\\\/Z6iVkr3w7GEx6t6WSUS72Z9ZRdw9tEVwgReapI7uB\\\\\\\/HlX3JL\\\\\\\/TqOQWlZ\\\\\\\/4JYYpK8fPagVOrmDyhyEunYwmMYkbIUd3mkcBQ5DjBRh+7IegJ5jt9jIXWv+PW6u0AMdPj59ZsJi5uOz5FFd3K91UzM\\\\\\\/Zi4OOAv9yZcqmENWsDfFTt6yItBvEsrZei8LB6RBXVZF0dq3nVJM3N2\\\\\\\/pTQWQtmJX86\\\\\\\/aMyESPVM0DUDE9RvLbT5pDNDB0lO8V0rF+ZScwoOxmsKgtho7KVGcTzbTvD7JdZia4pbXlWBMj65LqrVaFTYMUvMnJ1iJ\\\\\\\/Gw2eD\\\\\\\/lCJ1yxhnxhpvUkoumZNpBmhQxI2V9wKV02d9dH2u+rIbPMnLS4kXM2wpQTBT6DxDCvQsXwWak57c2l+fhxrZWoiP1RJETxDu6bwdIM\\\\\\\/Qrf\\\\\\\/yoZntEePiahLPoLkmjPuXaBkUKtjJPNNLC8oljYmz8Nw3ecMTVm5q6Dk14Oj8sm+5GVsNoBCQF99sY8YHwgZ7Huv4H7ykwYPpP3uyPY6B\\\\\\\/6iUpVbx50Efcuiv8\\\\\\\/EW53FJ7DICgB5Nm8D6WBAQBXkb19aDFpBdBHHX3oMcE79aREPpdry2GtjmsFiQJhBMDDLo4+BkU6wk0L0rytMjDLaXjkDUMcqeyVc5cjgmJezVBQWpSlZiUkcyOszGrBj0CA75XjnL4nKKMB0KXMsdsttlWQcDTK\\\\\\\/HWayB7AaDIklfFVXDqAx+ACJFN7qv3PQyr4ezI2p3U+2gDylDxUl5\\\\\\\/bMydxMEPR1tjt98Z9rrUOoWnjH+j2Lh3j+Okb1fo+RcYhD+x728rSMiJ2kwWkErIuL96YHlBZVdkgZxA0imdrPeXz3jtSp5GFJ1CNbezaCfhIlGlN238LSwO\\\\\\\/ynhkWqw7P2Vwu1K97mxV\\\\\\\/PxtGtCrurFCPpLr6AvY5SmF+boMd8UlIxL8BsZkkwIRhdGcXNGgs3M7Kyy0Gw2gtUCQjQx0nPy1CsIl5kQ5AOGUAbNI9IPQYLG3nWJi3BF6WuUUQLlcoaqc7a170LYYPWXFxoIROwHLgsVWzccBUB6Ym1fTge6rsqlm\\\\\\\/ME6ebFVlpBq865PzNXnp8vvbFM2uiJhdyN+XXgCL3PuXg6edhrIDZaat3BTs5MmJxU5tY2x8kDtff8gKhvv8Mww+aC5wDUUq3wkGQZzYGZhT2vrb+cGvAQiMqeLyt7q7F23rMi3kvLRdTBBAsNQ2THA7qvXCW5VLlXJxxvh+l0ZoHEMeT39A8QHFjj8eno8nWKqGeRGpk3WanqnhJ0BDG\\\\\\\/h2gEkiVq\\\\\\\/1e1172+sslXKFg8CprnPum5\\\\\\\/3DMpWs+FteGHWS874zZDfTI69Y56O+x3r1e3ZJOD1QpfLnjMLU1j0kNa1w70BUJRrWUUAUQbfnuxwIui1JRtco3658SnWUzoQzxTmypbOWNxHiOZ6+qpiR0fkn8bOyzmfWxio7xqGf72kSxbIU3w0ORDOW3nNvlUQS8x9GpvIObqd9ZJp1P+krD0QFh4A1MmHgFFtre63rFxsNr1CC4o8hThhCl5Aby7O4e4aGXi1FATjFilXKqYrEVzs1MD\\\\\\\/YiKH+85c5iilvKpbhD4t8WfJ41y2Fapx+w4SR\\\\\\\/4htiYK\\\\\\\/u9ZkdVWOEMvntQ4y7yDHvqE\\\\\\\/xmQjSvOlHGlu2HYGbZQABVvC4XzAIp21fLE+BFCMA047pylPDuVtdJWsV7BXJvRKhGzoXOWvwLESF0nZ6W4Uhn30DUXcsbhzxJ6A412mbQPbogQ7cxRbHiCmZJ+bGkypSiBmA0oYMeAVjMWalqfuQbhg8SUwSJbGjhUteNALWg3JG4XZfs9XrZQsqKH3TVHhacXmBYy4d6xj5poPj4wRVBu0tRHWGihUD6JFJ3xFllzNyzUrH4\\\\\\\/0t7lZQX3vKfbI6AYqk3gSNIjJILtEUcgf72bf3GwXGp++LW3rDqfgRxxP9fO5Ua54sMrmxCTXI7qjKWPC6QtO3yNfk5eFXCsM8rgt4njyeb6T0lkUDdTjpX0nJQoTQZ5le65rmzH00K91TXdTXwpxAs9mLkiRHqnbLqAJ\\\\\\\/wepoHqEDs8txHmHRAWO6McmFI7eLVhl6wIlOLvHu1y0Rmb\\\\\\\/wJq50ulBBVSaeAR3QJA4ks\\\\\\\/dY7bGqTurG9eqjLRPPunQ4JpyiPtNy+DxTWFi6r2XU9hd7dF701DrWsjI5rtBJgAvmpJsmMDII06IBfO8\\\\\\\/d+yhRpvlAFXzR6rzCeFIXBt1Q4BW2Itded2zd4o8mzykIUytp3sFUQ0lBMD7rHo\\\\\\\/+RDngm9jEgQggIPa\\\\\\\/kemnk5PN+6+31Org0O64stRdeIxHT0IUW9QoINIhZo3jj\\\\\\\/GO0Nm4Z8zdyQjm3TkZvPC5JWUcLRFsXUnyomoGOJghD6lxgiiAooyiQuJmc1VzZ3bMz4QDawny7S+OK0z5umgXgtOmS\\\\\\\/+Y+pqNhvqZYxcAClHPywxkg8L83iCQ0GWUUIhXs9U\\\\\\\/vZAl5X0HLhyUtlcv7AUgdqxPfVCL\\\\\\\/4v69ua4O6kr8LQsuj5+7Rlo6nRSGpEsHR8Kvwl41DUqlFKER3cdg1dQmbce5dGcB2G623UU0PKCKxDAAULH+0SSMrza2Xjgi86GqCxt3musrH9KiiVSBrAKFKgtsikwG0M+ly3mh2xi+tYJn\\\\\\\/wHWHsQZhWNqIvKp1CMXRnu2loOf1zGt3NeDy99X3emf8f1dyO7brCOR8pRpvDLpzeZvLgQWMw4\\\\\\\/xhv6hAroH5LY1yV3tD+gVH0b6490qy7kylmyx9vxuMcs37kZq3g6s43UKE3NTEaWR+Yaf+X5\\\\\\\/j0mls4LIfnfr8+7U74cfea9tvL6QZzCVCoHY2Tcx3xYHQN4EoaHMiUmqjC+fgPLBT4yZPNyN9x30rV4XBZk401ZC8kZGfYNtTLTilW7Mtlg6odoC2uZ0z9F48za68lyQ1QNhko8pDv411Cbt1RR4lqI0LstBOXNb3ZJ\\\\\\\/ASIslplY+uZf+ms\\\\\\\/2h4QTMatKHLgdwro5TlOZKvDYQvDYKmu\\\\\\\/CzYkcdzpi\\\\\\\/Kg5rObDF7Id0ncZmC2LJ0mPZNZHYBzDIqPq3sawPDk1A7w\\\\\\\/g5byElJ4QWAPsDc9s8yFeuAKQuhhwbIZ2q2QfTYoZUib+49rsETuAwLcl6iJOGQkY3L4tmeSQDF42rpjEKmmbB7le0e\\\\\\\/kidqm3OPZk6uqpL2r+0iJZHvoX4G9ioyjJ\\\\\\\/82CIezmCh0lh9iXzjtwLvX5urvWvmfOez9XjDU2ADnAVw6UwjODVzwfHWBKA7OflKteanWZ2hd86qdskAJB7v+wNdd9vYF7m7mZRtc3bpOzr5HJgJmmEhA24GYZRq0BncrGpvPSWGF2r3LGZFArBTCmwKt73106p9PCQfIa+tHmJVQKhZ1sy4tsOY8j8q7beJqS4Mod98I8+Vof9yOEF6+f5An5lM8Uotz6yQFnPqlJn6wqWXpimQdzT+C6sxabC+w0QhdX6hzHkW2owU\\\\\\\/5zItkAOoysuHFmC250fUzCMPv05iJ7k1h9ztST1Yh68dmESWTB+QPcjkxG7GaiDghopYY5hGqJYXT3qTuJh4TAkUzCtjpzTuQUqwzbV4N7NiFqTxeNQ7rVdueCbiKeXwk8REwwfVMsbQcYWJ8Ge\\\\\\\/cHn32p\\\\\\\/zdGx8hOeXfNEWPEZcEkUDrqHtMMk1I40CzjeeaL8itq\\\\\\\/dkpa0NjFBLSWRV\\\\\\\/ZQtnVoN4EWDeatqrM4MRFNIc2MX+ogQRDMtDNPWgMk13eH8q7OrZavWoVSbQLqzy0OunyrVaij+J5nAGKXWBCnMJDtLojeBRiYCbthFKVPJrl1+EkjBFa22OXufLx4EUy+UmNAlgKA4vKSTuEzQ5zLr5WggSHZY0pTOXPWoDfuWyQkKTPnGwGMZInM6bDs2ja97M5LYOlL4pFBZelpLE+C8LO5n4Gn2F8Vt8rJfhUExDwVcFxYAwN\\\\\\\/A1J4DkwU5RRynaYDWw3vS2m14LEVCYS3PMnz8f6cZvXcylZdoJVuwDCaGcuiCyK2NPF8BAz2tfXZaR3Qc6A1UsVaUwk7SoxHjsgV2GsprMp5wW3Zne9AjKhXzm9zxbvjwmuVlAYf17gbecgR0a98uDhMeS5nB4ATXc1txHL1lItLknLzzT1H8wYIPWFiz\\\\\\\/s5RP5Adb0v0nuAB8COPZwY0b0Ub\\\\\\\/J2GuBZ7NO+hcO\\\\\\\/N9Hk87+R84h4VDnKeNaA+afVlE4ruP1ukhevJ5nQ9kXiZT+myKqYzHgKpCYtd0CyjHAW6xP4y0boGwOjRmNXDxn+9Gl\\\\\\\/h2Yju\\\\\\\/WzSDH4MFLfyVSMavqeoeIzDkzSCFC\\\\\\\/5l6anKLzVANy71EEPVZfbeldidJSP9xdXL3MB7hmS84WiP+QcqK+QIVOCdtjVzR9O\\\\\\\/yl772AmHn3x+NnrvFWc5ipCIkReOySTrt2xhefVA4XS5tCXIYTl+ulWC6kEu1bbh1mx0hMkMqyATpJ1AKPpdI2JPCXM2uKlnPb5bVVMwczLQi3kyo\\\\\\\/j358vOpoWub2fsuNwtkI4DfZSb5oIBR2R4SkHZ4NYc+1gBx+z+kNE+jPVLtDkXnJvGiFCvUgI1aF2COnQXeHbSrrw961lSvAX8Y4tdPteUWEcXND75iS01\\\\\\\/JvOaQim6gc1P2bvAz5dNjURL2fz9\\\\\\\/8mxdz1uaXUyVOMAE2kkhn5xqo\\\\\\\/O1oWq2IOCjCYkhQSI1u\\\\\\\/MHNrp8Qf0yjIGdSsYUy8TrSpRdTEKXN0VvWo9ZEqYaJNIz3laMgRLjlBcOLBAJjOgisijJgAtL\\\\\\\/WG1CP0ztLDe66JfM25eFxBcBFKPX20gWAZ4wnGPYCOLoRJUg4lTIGYDCw1HZ0cFBh4cdceVPsl33vADXyHr6Vu2KxTmEsqaWyNWu\\\\\\\/K62kSh4kD3UMPNDlNRWaazFZQKkXeN6gYI2I+jFY6uy0kwE6p6hs73JmE8ldysDWDhhvUexKppMG\\\\\\\/GQoOcXuso\\\\\\\/lxzc7225UKf9sQH6B3oQ02Jzfa4V4RK0qIOguC3P81hJxhLo1mmocYQjNuSW\\\\\\\/qip2lroP2lCIslWkGUbFvjamvIPkgEwH4zXtN1bPj\\\\\\\/u0Z9+OgxRjRKr6dLA78EhFPmueoE8x46\\\\\\\/llLmCpehpB1XqyvGG9QiGE\\\\\\\/iKUwmn2nfgHpHvQpxfpmvK5MwX1xIQTa1d+Ig\\\\\\\/tAn7geb1o7OmSoNZ+lZSkvzzfzPGtaRYMyw8Y\\\\\\\/yiP5sBif6P5S9Guu\\\\\\\/FaMVmeygJ4oSQjyD12nna55brbS9uiKmrO0uHRngJ5keIZv+PWhvxOezbjWfNKw97fAs\\\\\\\/DOroISmUfX09qNdYZXIIvfiigzmT7Mqa5ltR3qVPmtSEL0Py549Z1x3BXtlmvBYvnW\\\\\\\/OLP+RSvsRCmTxE6F+B5kVL5e+iwWQI1LpWhXQbH3q3qE+oeVxZbCc0X85rkH4TG4dxYnceUDrU4tXTRR8QkLUl1Mp\\\\\\\/oGdFr7GqO0MloW1ct7+yc9qly6dPybz6vDEb37lRLpMmPv+3Oiqk8QePGGSbce0wH5NwgW3RN8dR5h09vX1TkR7wGcAjtqAOL8iEa9oKAa8spTRKi1xoP2h2bULwDrwMQ83bZeF+\\\\\\\/d\\\\\\\/TYvvoUlN8jcQhBYkgL1EfKMSVQuRyx8seseJIKDRMLWORvkFldHkzWcPDq6VSv20lY091LDJOEZN77AP3ubqgIUx\\\\\\\/QnhMx20U2yAERQw1RB3wW4Jbz8KjS9zc4DmkK\\\\\\\/5Rx2Th0JB2kcl8kBfqavwjndglpGQjDdx4xGQcs20cRwhB5qGOdDl64mTJJcGXFP3bOmCfGPLZjOKp0cNhciBv6JVzIGvwJblAp0ZN4lnkirGIPHZSNAPN8hReB32xBjIIBAzbdS\\\\\\\/BGRcJBZFC9xJQja\\\\\\\/QCqewCTDxsF4h0FvxEKB5iG9yXkqWlllPy+TCoJWP8ZqYzJcYZJ6yPSO0rLIXxRBAAyJA4HNrkx63IxQWdWoEXT\\\\\\\/NxYyNatBHDAhyIZaeh5+rYEDl7WMZhjmx+CRmM3hTzQYjPkEIFgtyUJmD7Sf4oKh9Uxr982CH0v8nOwcS5lZtelBC5hdjFd9hg7ttZ9styaHT\\\\\\\/R1RO9KFoVGoFGY94hCwJeTziKczglhKoknDCYByRVBPdz6rRii7cYgaa7UEZCQ94mqEuf4KQz49HJEfNsxEj8XuFESWMATb36uxyF1h11mGT+V10pP8uBSkkaX+c6BX+3Toclq4NGXvT9LuP4yLBeRKP2cU2Y4opPwsHZs\\\\\\\/Iq\\\\\\\/pHKG1aEr5FwFLH\\\\\\\/3bVmP6Ybdeq5vbRjVuQgIWNIVqgrDB45c6eN5Jy8kxsxb6ptXCIlDLkOZrFTJSsp8+Y+5vrzVU3vNYmhSqc+Q1VFQkSa6Z0XOOEzEBoUog4QhNpvrq086Mv8PIU4tBqkZsr9oBiWp3SdQHtZ5\\\\\\\/BUgFYftsrKnhZIcvRMZR1IGeGo\\\\\\\/BL0XgZsckSYyKzAJfjnoxEOLmni\\\\\\\/lmJ7XLSb92y837yD+hUUYGIZZdZNsE+Ob+Kc3jcilEVl4jjw01e5oOS97fHPf0QBpgGGDQvjl8YA8vy\\\\\\\/4kVs6v1EdWmMe17lZcmGkEJ9PLshzbmrpo7HdvnEa9ZDU5AhtbAgf24IpZ2W7CEVhnBAExg5N1T12+TuaKJPGnaVmrM9T5u7mQ56lmmlMz02KYPLJ2wzPI55W20t51UH9oMx\\\\\\\/1egHDt5gwHz6qbLfTWRTYCMoWgPo0bzaOsnsmZGkIg8BP\\\\\\\/JuUuEeGpxHF94N7plWO+dnsUz3EtkQqWNt0E2pZs\\\\\\\/\\\\\\\/ql4Ec+w\\\\\\\/d5e0mtXQZ6g44Mfpj46M7zfrXrvSjG+pIssjniVmAgRVQsUMXALFve08CSzGF2XejBBOz2WC2mKvV0u8XoDNr6E1QanZLYiN3vaaCI34XagElK+pmaAQN+IeTLKzCa1R51K2XoavzE88TGR8E13eqgG8mmHy7hupIePtaOfcx2LsDxZaxa7XLH2+z6GS145f6vL8gSNPcepFMmDjbN8OKbKP+l+Hp+I3UKRLsHPXXyATdbXgRXGlR1fggPNYmlFZdGjH1mZia0TtYkKyywEiMg0rsgPzH9Tg4VWD1ckrfmGT3D6o4aOXJDaVpnIkg2Sd90ZLe5uhU56IefhsS\\\\\\\/1MoE9q0T8bcKFqasrY2IXw120CZ2lfUQeHNro\\\\\\\/02EjNZiIi4pmgOPnclWhQ6lfek6AaRNa2DJvX\\\\\\\/IqyVXMaro+1Zg78\\\\\\\/0sF4b0lcXqZzk5yE1IEedeZRwIe0+MKk1NeJrOUQ51UaDkv5Z9c\\\\\\\/rO5A3j84CTyLZSgs1gd0JEGyxeo9H9TX370YBULh1RRdx2F3smeiDjYN0HTj38oz4YejugQG1FbgRAAevO2sGXbySU+rmtJT6vqrfXLttdQjL3LoMC\\\\\\\/bZHt4oNfMCL62X7xqmmaRxdSbvW6kMmFtUL\\\\\\\/zBx3RkiV9usCVRYsHOrEN\\\\\\\/IjOZfc6x1fzm81uWyFKryMUoPY\\\\\\\/v2fGfobHqZILMOvMPHMFkDKbf4uPJbMRVMr9r2cHFY5bg5z0D50BM2l\\\\\\\/VC22xBL4mVvU9gqSpSUDV\\\\\\\/J7UCpT0\\\\\\\/2BUqcgx\\\\\\\/AfkcBuAZsn9pvWWmClwiBncjhnW9zA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hxf45q\\\\\\\/dKO7thLH4Lhbk\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022234e213c9a770263d8d192094bf5635df817f96a2a7d898429e8b95bd46892ce\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QWRZW84c3FKxfTfJfczegb6lvd+3OKBs22usx7oF\\\\\\\/RhpiOoIuhHJVqXFO0iSPTNIWS3IxgEuNVUIEY+3Sodoo7F9MHNb+fE2fMxGTFy8fM6amsEL3zdz2ed17fF3clTbEpD2WwL7iW27k1CRs7X0VHZ2jk++L98hreSRY+9c6lg+FCJZShx5dOGi905MWlurP\\\\\\\/VWsG78vGTvHgUUnvYpuoRjeuXEREfW1PzbiUUJf5WqQEUO2HPo\\\\\\\/sM+hR9IhD5LxDZvdw2iCH7E34nOO+pREbs3kn2Sftkc8O67kjX1NSkIBxGTffD9jPoqh\\\\\\\/ZSwzRr8rYpoTmSid8O0eNWOZkUbabm1eYGwXY44bmUn+qhagYKSUSCQlbrULqdiDuU51kHup+AGpDHMjzzXI0zaQoBtwxuK6TZ+gVMRH8Q0lhHxz35sfgu1x6Fc9tI3itqMi+63hFh62C2CfkcMJdyewO\\\\\\\/fYrUvWMl4XXm+kAemwcJ+Bp7YdsdjwO6O+P7eT20bUqGur++KiM60g66WEoPdkWEtik3yr4CeSbWFO59jLmY6X4SvnIu0hPCOsMNC\\\\\\\/xpijVxw94SObfSeevR5Ja80E7hoEepbUM565pLDzW7\\\\\\\/+0lC+ofChaG8LMqZ915uKm\\\\\\\/2TjpCOpMw3EmBiyC+lHZaQ82\\\\\\\/sqzxOTHl\\\\\\\/+c\\\\\\\/2UacmgJa7IUS\\\\\\\/Vd8pdik6BmNcpgKN0+LRGBN1NP96F\\\\\\\/JQIztVBYo7SGpLuqaXM13pM6CTqsUsBDyFSS9XrzXubW4ixGDavbMm2h0bL4Ii6F3Ro\\\\\\\/xlxRWm0T+gq0V2qao+ldVzdcB5dl35KCYqGgIRdPtg+OJjW3PiYGAw1H2gNwNPc5LyGKqpoJf+wR22HG4UtNH6dbzh85e6vMWOtlNsEP6jPOIl+VvYsmHYveuqrdSmbcaMw0A4dexNPUetFcI1r3Vgi\\\\\\\/agug+ziV2rBKPg0zwGPOPOPnBGPi\\\\\\\/vGk9Ap\\\\\\\/dralib42DcI0oTVVmnoponvgCJoeYS9bWnSInCn375xif9965RcpZ9ayeSvXmXIZGwNgVsHMOSrxdUdfWGprO+T8uJiJdF9\\\\\\\/94Qd2f1etKcWdmRTKruhYj1jtz+NDe3OtQnmWGVafVMNkrCAGUSq9x96Y+4EJkx7ATErESGSvoSZsexxyOrMJ+4qajkeIYA+hOE8JHlctNSVATvTSDWRUGwk6\\\\\\\/wN9gIrKIZo9e8E5mGMwpEkud47YdSIWseUHeRnuienh3oCEnw\\\\\\\/urF5vRsYkINfvOlG6Yk1WRf4IgyAbYEu8piOy\\\\\\\/WVyXfT8wig6znX1A8nRPYq+pI59bCe2Tj38xaoa4tZW9Vf2JZgnNN2izrvPHdKFWkQn3JpL13ki6+jbD7VNil6WejBXDGTuXGF1GNyhM2IDpcN7UE546apoh\\\\\\\/dnKG8G3kECskaj\\\\\\\/FPfD3Y2lYDcfEyN2efPEmbuBEjjfch1X7v0D4Atq7L5HMzT\\\\\\\/hKuHo4010ulpkunIVtH2Za1irL\\\\\\\/semN7ZQvajswTRzRVcPCCa8cYF1Ix\\\\\\\/qrRSD1HTbCrgoCCteFHdw4HQfEAA6boIKpNPdV21mdY8QWU4PR\\\\\\\/tLGvE4yn4lieV+n90kLFTa\\\\\\\/tYqyeYsnngprFn0ce59kJFlA1fxIIy98aguJBP4OLEthY01RrYOaygz792eDSNiSCEyDEPqDC\\\\\\\/BMiEyiXuhVJFm668eRUP6ZdQEDa+KIZxGrMO1tmvZBf0GUymHpeSrnbvZVZmZjzpw5SRN374FHBHjiX7EOcuZPv7suEMinxI1DcEOU\\\\\\\/bRnGO9R28XZ+5Lk+plNT0xQ98PcJHmWCYihDrZleTZOnNiZ10Gu37YwuvwGU9E2bCEk5S3qLUcNQMqnKVpk2jezjNTFcWdYIqeznyk4qCE2rxFWxGjm4cB9Dtm6LbOseFcqYMes5RAmbGaVLnEhBEA6ZIf42MRTnSlfOE4JDaKb3\\\\\\\/QMoB7GGbyj3Xmd6IEJTgxHbtF1BSTh55v3gLQ75bPb18iWwE6Y5TPu+owuBxauA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pOGSk0WWwWV2rS94v70IGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022302065f937672ea57e4366f80c8394cf886f6a7912604d17140f6e2e5e9f573b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459815 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/399" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ukuJ0yYNAA81PcLNKMfsTxIk4Wlly5w5sKOsB2ZrFks" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/399" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ysmQvMiq2oMDjPMk681MICaf48VVrk92ETerb09R8zY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/389" title="The Reference Genome Sequence of Saccharomyces cerevisiae: Then and Now" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/411" title="Metabolic and Environmental Conditions Determine Nuclear Genomic Instability in Budding Yeast Lacking Mitochondrial DNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459815" id="node459815" data-pisa="ggg;4/3/399" data-pisa-master="ggg;g3.113.009365" data-apath="/ggg/4/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Dynamics of Diverse Segmental Amplifications in Populations of <em>Saccharomyces cerevisiae</em> Adapting to Strong Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Celia Payen</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">Giang T. Ong</span>, <span class="highwire-citation-author" data-delta="3">Jamie L. Pogachar</span>, <span class="highwire-citation-author" data-delta="4">Joseph C. Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Anna B. Sunshine</span>, <span class="highwire-citation-author" data-delta="6">M. K. Raghuraman</span>, <span class="highwire-citation-author" data-delta="7">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="8">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.113.009365" data-atom-uri="/ggg/4/3/399.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459815" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Giang T. Ong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giang%2BT.%2BOng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ong%20GT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiang%2BT.%2BOng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jamie L. Pogachar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BL.%2BPogachar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogachar%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJamie%2BL.%2BPogachar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph C. Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BC.%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna B. Sunshine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB.%2BSunshine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sunshine%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB.%2BSunshine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/399" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459815" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459815/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/399.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459815" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459815/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/399.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459815" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459815/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/399.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459815" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459815/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/399.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459815" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459815/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Population adaptation to strong selection can occur through the sequential or parallel accumulation of competing beneficial mutations. The dynamics, diversity, and rate of fixation of beneficial mutations within and between populations are still poorly understood. To study how the mutational landscape varies across populations during adaptation, we performed experimental evolution on seven parallel populations of <em>Saccharomyces cerevisiae</em> continuously cultured in limiting sulfate medium. By combining quantitative polymerase chain reaction, array comparative genomic hybridization, restriction digestion and contour-clamped homogeneous electric field gel electrophoresis, and whole-genome sequencing, we followed the trajectory of evolution to determine the identity and fate of beneficial mutations. During a period of 200 generations, the yeast populations displayed parallel evolutionary dynamics that were driven by the coexistence of independent beneficial mutations. Selective amplifications rapidly evolved under this selection pressure, in particular common inverted amplifications containing the sulfate transporter gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>. Compared with single clones, detailed analysis of the populations uncovers a greater complexity whereby multiple subpopulations arise and compete despite a strong selection. The most common evolutionary adaptation to strong selection in these populations grown in sulfate limitation is determined by clonal interference, with adaptive variants both persisting and replacing one another.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/evolutionary-genomics" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-genomics" rel="nofollow">evolutionary genomics</a></li><li class="kwd"><a href="/keyword/experimental-evolution" class="hw-term hw-article-keyword hw-article-keyword-experimental-evolution" rel="nofollow">experimental evolution</a></li><li class="kwd"><a href="/keyword/clonal-interference" class="hw-term hw-article-keyword hw-article-keyword-clonal-interference" rel="nofollow">clonal interference</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole genome sequencing</a></li><li class="kwd"><a href="/keyword/gene-amplification" class="hw-term hw-article-keyword hw-article-keyword-gene-amplification" rel="nofollow">gene amplification</a></li><li class="kwd"><a href="/keyword/inverted-triplication" class="hw-term hw-article-keyword hw-article-keyword-inverted-triplication" rel="nofollow">inverted triplication</a></li></ul><p id="p-6">Adaptive evolution in asexual populations depends on the accumulation of genetic variation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gerrish and Lenski 1998</a>). If a single beneficial mutation occurs in a small population and is not lost from drift, its frequency will increase over time to eventually reach fixation (review in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burke 2012</a>). In larger populations, multiple beneficial mutations can occur and interfere with one another’s fixation, a phenomenon referred to as ”clonal interference” (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Burke 2012</a>). In a population in which two beneficial mutations compete, the one conferring greater fitness is more likely to reach fixation (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gerrish and Lenski 1998</a>).</p><p id="p-7">The dynamics by which mutations accumulate within populations are complex as the result of stochastic mutational processes, drift, natural selection, and competition between clones of different overall fitness (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gerrish and Lenski 1998</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de Visser <em>et al.</em> 1999</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Desai <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Miller <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lang <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lee and Marx 2013</a>). Such dynamics can be examined directly by laboratory experimental evolution in microbial populations. Previous long-term studies have contributed to our understanding of genetic and genomic changes that underlie diverse phenotypes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 1998</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Koszul <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cakar <em>et al.</em> 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">van Maris <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guimaraes <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kugelberg <em>et al.</em> 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adamo <em>et al.</em> 2012</a>). These studies have primarily focused on clones isolated either at particular times (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sonti and Roth 1989</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Notley-McRobb and Ferenci 2000</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrick <em>et al.</em> 2009</a>) or at the end of the experiments (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brown <em>et al.</em> 1998</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lee and Marx 2012</a>), and therefore provide limited information about population dynamics. One effective way to understand the dynamics of a population undergoing adaptation is to mark different subpopulations and visualize their change in frequency over time. Neutral fluorescent markers, for example, have been used to monitor the contractions and expansions of subpopulations over 500 generations of evolution (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kao and Sherlock 2008</a>). However, even within these defined subpopulations, multiple beneficial mutations can arise over the course of the evolution experiment, making it difficult to track the extent of clonal interference. A recent study has quantified the temporal dynamics of point mutations over 1000 generations by deep sequencing of populations (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lang <em>et al.</em> 2013</a>). Similar work has been conducted by Lee and Marx in which they examined large deletions and other chromosome rearrangements during the long-term experimental evolution of <em>Methylobacterium extorquens</em> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lee and Marx 2012</a>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">2013</a>). These studies were able to detect up to 17 different large-scale rearrangements in one population.</p><p id="p-8">In a chemostat, large populations of cells grow under a controlled environment for many generations. This system allows us to study adaptation under a defined selective pressure, such as limitation for a nutrient. In both bacteria and yeast grown under nutrient limitations, the target of selection is often a nutrient transport pathway. For example, mutations in <em>ompF</em>, a gene implicated in membrane permeability, have been isolated in <em>Escherichia coli</em> grown in lactose-limited conditions (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhang and Ferenci 1999</a>). In <em>Saccharomyces cerevisiae</em> grown in glucose-, nitrogen-, or sulfate-limited chemostats, amplifications of the glucose, amino acid, and sulfate transporters (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002751">HXT6</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=HXT7">HXT7</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">GAP1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>, respectively) were detected in single clones (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brown <em>et al.</em> 1998</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kao and Sherlock 2008</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gresham <em>et al.</em> 2010</a>). Because amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> occurs repeatedly in independent evolution experiments and confers a large fitness advantage (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>), sulfate-limited chemostat growth provides an excellent model for visualizing the diversity and dynamics of beneficial mutations. Independent mutations affecting the same gene, often called parallelism, have been previously described at the single nucleotide level (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tenaillon <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Herron and Doebeli 2013</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lang <em>et al.</em> 2013</a>) as well as for large and small deletions, amplifications, and insertions (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhong <em>et al.</em> 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stoebel <em>et al.</em> 2009</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blount <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chou and Marx 2012</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lee and Marx 2012</a>). In this work, we determined that the selection for amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> is highly repeatable and provides an excellent example of parallelism at the locus level. We had previously isolated several independently evolved clones in which each amplification event appeared to produce novel junctions on chromosome II (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>), leading us to hypothesize that these unique junctions could be used to identify distinct subpopulations. We now demonstrate that the inverted repeat structure we identified previously in a single clone (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>) is the dominant mode of amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> in haploid yeast. Unlike with the amplifications of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002751">HXT6</a>/<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?locus=HXT7">HXT7</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001747">GAP1</a></em>, which recur using the same repeat elements and are thus difficult to track, each <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon resulted in novel junctions, allowing us to use them as intrinsic markers to follow the trajectory and determine the fate of unique amplifications during the course of ~200 generations in multiple parallel independent cultures. Whole-genome sequencing of several evolved clones also identified a beneficial point mutation with a fitness effect less than that conferred by the amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>. Tracking of multiple subpopulations provides evidence that parallel evolution via clonal interference is the mode of action by which evolutionary adaptation occurs in populations of <em>S. cerevisiae</em> subjected to strong selection for assimilating limiting sulfate.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and continuous culture</h3><p id="p-9">The <em>S. cerevisiae</em> wild-type strain used in this study was FY4, a <em>MATa</em> prototroph of the S288c background. A single colony was inoculated into sulfate-limited chemostat medium (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>), grown overnight at 30°, and 100 μL of the culture was inoculated into ministat chambers (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Miller <em>et al.</em> 2013</a>) containing 20 mL of the same medium. After 30 hr, the flow of medium was turned on at a dilution rate of 0.17 ± 0.01 hr<sup>−1</sup>. Seven chemostats were inoculated in total and cell samples (glycerol stock and dry pellet) were passively collected every day from fresh effluent for ~200 generations. The strain used in the competition experiments is a FY <em>MATa</em> strain where the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a></em> locus had been replaced with <em>eGFP</em>. To test the fitness due to the amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>, we transformed <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em> strains with a low-copy plasmid (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ho <em>et al.</em> 2009</a>) or a 2-μm plasmid containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cherest <em>et al.</em> 1997</a>). Clones at generation approximately 50, 100, and 200 were plated from frozen samples onto sulfate-limiting plates and propagated in sulfate-limiting liquid medium to attempt to maintain selection for the amplicons that arose during chemostat growth.</p><p id="p-10">The strain deleted for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> was obtained from the Yeast Deletion Collection (<em>MATa <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">sgf73</a></em>::<em>KanMX <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005728">his3</a>Δ1 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>Δ0 <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000319">lys2D0</a> <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>Δ0</em>) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Giaever <em>et al.</em> 2002</a>). The strain was backcrossed three times to FY5 (<em>MATα</em>, prototroph) to select for a prototroph clone that contained the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">sgf73</a></em> deletion.</p></div><div id="sec-3" class="subsection"><h3>Genomic DNA extraction, gel electrophoresis assays</h3><p id="p-11">Genomic DNA was extracted from dry, frozen cell pellets via the Smash-and-Grab method (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hoffman and Winston 1987</a>) or the NIB-and-Grab method, which is a hybrid of the Smash-and-Grab procedure and Huberman DNA isolation procedure (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huberman <em>et al.</em> 1987</a>; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/009365SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FileS1.pdf">File S1</a>). The average molecular weights of the two DNA isolation methods yielded DNAs of 10−20 kb and &gt;50 kb, respectively. Gels for analysis of restriction enzyme digested genomic NIB-and-Grab DNA were 20 cm 0.4% ME agarose run at 1−1.5 V/cm for 18−24 hr in 1X TBE. The probes for Southern blot hybridization were an internal fragment of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>, a fragment just centromere proximal to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (“786”), a fragment of chromosome III containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em>, a fragment from chromosome V containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077394">ARS522</a></em> (originally known as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077394">ARS501</a></em>), and the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em> adjacent ORF <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000097">ECM15</a></em>.</p><p id="p-12">For indirect end labeling NIB-and-Grab DNAs were digested first with either <em>ApaL</em>I, which cleaves just centromere-proximal of the 5′-end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> gene with no additional <em>ApaL</em>I sites between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> and the right telomere (~25 kb), or <em>EcoN</em>I, which releases a fragment that extends from the 3′ end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> 19.4 kb toward <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em>. Aliquots (~1 μL) of these digests were incubated with a series of second enzymes that cleave varying distances from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>. For snap-back assays, NIB-and-Grab DNA was digested with either <em>ApaL</em>I or <em>EcoN</em>I, denatured at 100° for 10 min, chilled immediately on ice for 7 min, and then ethanol precipitated and resuspended in 8 μL of H<sub>2</sub>O. S1 nuclease digestion was carried out on the resuspended DNA in a 10-μL reaction in 1X S1 buffer with 1 μL of S1 nuclease for 10 min at room temperature. The reaction was stopped by the addition of a Tris (pH 8)/ethylenediaminetetraacetic acid stop mix.</p><p id="p-13">DNA for contour-clamped homogeneous electric field (CHEF) gel analysis was isolated in agarose plugs as described (J. L. Argueso, personal communication). CHEF gel analysis of yeast chromosomal DNAs was performed in 1% LE agarose gels with a switch time ramped from 47−170 sec at 165 volts for 66 hr in 0.5X TBE at 14° using a BioRad DRII electrophoresis chamber. Southern blots of CHEF gels were probed sequentially with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em> probe and then with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> probe. The ratio of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em> hybridization was used to quantify the number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> repeats on each unique version of chromosome II, setting the ratio to 1.0 for chromosomes at the beginning of the experiment. In each generation sampled, only chromosome IIs that were present at &gt;20% of the total were quantified. In the seven populations we detected 16 new versions of chromosome II that reached this cutoff for at least one sampling interval.</p></div><div id="sec-4" class="subsection"><h3>Polymerase chain reaction (PCR) and quantitative PCR (qPCR)</h3><p id="p-14">Smash-and-Grab DNA was used for qPCR as previously described (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Di Rienzi <em>et al.</em> 2011</a>) after being cleaned using the DNA Clean &amp; Concentrator kit (Zymo Research). For each sample, the copy number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> was determined relative to the copy number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001855">ACT1</a></em>. The copy number of the locus for a given sample was normalized against the copy number of that locus in the original strain used to inoculate the ministat. A site was considered amplified if the copy number was ≥1.5.</p><p id="p-15">Smash-and-Grab DNA was used for PCR amplification to obtain fragments used as probes and for Sanger sequencing. Primers are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/TableS3.xlsx">Table S3</a>. PCR products of interest were purified with DNA Clean and Concentrator (Zymo Research) and primer extension sequencing was performed by GENEWIZ, Inc. (South Plainfield, NJ) using Applied Biosystems BigDye version 3.1. The reactions were run on Applied Biosystem’s 3730xl DNA Analyzer.</p></div><div id="sec-5" class="subsection"><h3>Clones and population array comparative genomic hybridization (aCGH) analysis</h3><p id="p-16">Frozen chemostat samples from generation ~200 were streaked onto limiting sulfate plates (medium as described previously plus 20g/L Difco agar). Single colonies were picked, and DNA was isolated by a modified Smash-and-Grab protocol (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hoffman and Winston 1987</a>). For population analysis, DNA was extracted using the NIB-and-Grab method directly from the frozen sample. aCGH was performed using Agilent 4x44k microarrays with probes spaced every 290 nt on average. Hybridization was executed as described previously (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>). Microarray data from this article have been deposited in the Gene Expression Omnibus repository under accession GSE47854 (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) and in the Princeton Microarray Database (<a href="http://puma.princeton.edu">http://puma.princeton.edu</a>).</p></div><div id="sec-6" class="subsection"><h3>Competition experiment</h3><p id="p-17">The pairwise competition experiments were performed in ministats (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Miller <em>et al.</em> 2013</a>). Each competitor strain was cultured individually. Upon achieving steady state, the competitors were mixed in the indicated ratio. Each competition was conducted in two biological replicates for 15 generations after mixing. Samples were collected and analyzed twice daily. The proportion of GFP+ cells in the population was detected using a BD Accuri C6 flow cytometer (BD Biosciences). The data were plotted with ln[(dark cells/GFP+ cells)] <em>vs.</em> generations. The relative fitness coefficient was determined from the slope of the linear region by the use of linear regression analysis.</p></div><div id="sec-7" class="subsection"><h3>Nextera libraries and whole-genome sequencing</h3><p id="p-18">Genomic DNA libraries were prepared for Illumina sequencing using the Nextera sample preparation kit (Illumina). Barcoded libraries were quantified on an Invitrogen Qubit Fluorometer and submitted for 75 bp paired end sequencing on an Illumina HiSeq 2000. Read data have been deposited at the NCBI under BioSample accessions: SAMN02208069, SAMN02208070, SAMN02208071, SAMN02208072, SAMN02208073, SAMN02208074, and SAMN02208075. The reads were mapped against the genome sequence of the reference strain S288C (SacCer3) using mrsFAST (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hach <em>et al.</em> 2010</a>). The sequence coverage of the nuclear genome ranged from 70 to 300x. Novel junctions and indels were identified with SplitReads (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Karakoc <em>et al.</em> 2011</a>). For single nucleotide variant (SNV) analysis, the reads were aligned with Burrows-Wheeler Aligner (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Durbin 2009</a>) and SNVs were called via use of the Samtools (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009</a>) mpileup command after applying standard filters (quality score &lt;30). SNVs unique to the evolved clones were identified, annotated with a custom Python script (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pashkova <em>et al.</em> 2013</a>), and further prioritized by manual examination with the Integrative Genomics Viewer (IGV) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Robinson <em>et al.</em> 2011</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-19">To visualize the diversity and dynamics of a population during 200 generations of adaptation to a sulfate-limiting environment, we first developed assays to establish clonal identity based on unique amplicons containing the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus. We then took advantage of the specific and beneficial amplification of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus as an intrinsic marker of evolution to study and visualize clonal interference in evolving populations of yeast.</p><div id="sec-9" class="subsection"><h3>Adaptation to sulfate limitation selects for the amplification of <em>SUL1</em> in <em>S. cerevisiae</em></h3><p id="p-20">To characterize the evolutionary paths of yeast populations subjected to a constant selective pressure, in this case sulfate limitation, we performed seven parallel evolution experiments by using chemostat continuous cultures. Each experiment was initiated from a prototrophic haploid <em>S. cerevisiae</em> strain that had never before been exposed to long-term sulfate limitation. In sulfate-limiting conditions, the seven cultures reached steady-state growth with population sizes of ~10<sup>9</sup> cells. Six of the cultures were maintained in continuous growth for ~200 generations; the seventh culture (Pop1) was terminated early (at generation ~90) because the input media line became colonized. At the end of the experiment, a single clone from each population (six clones total) was analyzed for copy number variants and relative fitness.</p><p id="p-21">As expected from previous studies (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>), we detected amplification of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus in each clone as assessed by aCGH (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS1.pdf">Figure S1</a>). The number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> copies varied from three to five per haploid genome with the boundaries of the amplification differing in each individual clone. No other large structural changes were detected in any of the clones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS1.pdf">Figure S1</a>). Relative fitnesses of the six clones were estimated by competition with the wild-type ancestor in the same chemostat environment. All clones showed significantly higher fitness compared to the ancestral strain, with the relative fitness coefficients ranging from 36 to 53% (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Unique SUL1 amplicons are observed in clones isolated from six evolution experiments. The map illustrates the location of SUL1, flanking open reading frames, and the origin of replication, ARS228. The lines above the map show the extent of the amplified segment observed by aCGH (Figure S1) for each clone: blue line, copy number 5; green line, copy number 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Unique &lt;em&gt;SUL1&lt;/em&gt; amplicons are observed in clones isolated from six evolution experiments. The map illustrates the location of &lt;em&gt;SUL1&lt;/em&gt;, flanking open reading frames, and the origin of replication, &lt;em&gt;ARS228&lt;/em&gt;. The lines above the map show the extent of the amplified segment observed by aCGH (Figure S1) for each clone: blue line, copy number 5; green line, copy number 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F1.medium.gif" width="440" height="113"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/399/F1.medium.gif" width="440" height="113"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464117" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Unique <em>SUL1</em> amplicons are observed in clones isolated from six evolution experiments. The map illustrates the location of <em>SUL1</em>, flanking open reading frames, and the origin of replication, <em>ARS228</em>. The lines above the map show the extent of the amplified segment observed by aCGH (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS1.pdf">Figure S1</a>) for each clone: blue line, copy number 5; green line, copy number 3.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of SUL1 amplicons reveals inverted structures. (A) Map of the right telomeric region of chromosome II shows the position of SUL1, the relevant restriction enzyme sites, the deduced structure of Pop4 gen201 clone1, and the probe (SUL1*) used for Southern blot analysis. (B) Southern blot of ApaLI digests of seven clones containing SUL1 amplifications isolated from independent evolution experiments and hybridized with the SUL1 probe. The ancestral fragment corresponds to the telomere-proximal SUL1 fragment. The more prominent bands of variable sizes correspond to fragments with novel chromosomal junctions. (C) Indirect end-labeling of ApaLI double digests. The order of the lanes corresponds to the order in which the sites for the second restriction enzymes are found between ApaLI and the telomere. (D) Southern blot of the double digests. The series of bands of increasing sizes in the Southern blot indicates that the portion of the genome from SUL1 to the telomere is intact. Fragments that contain the amplicon junction comigrate with the expected fragments only up to the position of the junction. Second enzymes whose sites lie distal to the amplicon junction fail to make a second cleavage and produce the amplicon-specific ApaLI junction fragment. (E) A schematic illustrating the snap-back (SB)/S1 nuclease assay (S1). The rapid chilling of denatured ApaLI fragments only permits reformation of dsDNA if the molecule is self-complementary. S1 treatment degrades all single stranded fragments including the ssDNA in the loop. (F) Southern analysis of the snap-back/S1-nuclease assay of population 4 clone1 using the SUL1 probe. The 14.6-kb amplicon-specific ApaLI fragment generates an S1-resistant duplex molecule approximately half of its original size while the single strands of the ancestral fragment are degraded by S1 nuclease." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of &lt;em&gt;SUL1&lt;/em&gt; amplicons reveals inverted structures. (A) Map of the right telomeric region of chromosome II shows the position of &lt;em&gt;SUL1&lt;/em&gt;, the relevant restriction enzyme sites, the deduced structure of Pop4 gen201 clone1, and the probe (&lt;em&gt;SUL1*&lt;/em&gt;) used for Southern blot analysis. (B) Southern blot of &lt;em&gt;ApaL&lt;/em&gt;I digests of seven clones containing &lt;em&gt;SUL1&lt;/em&gt; amplifications isolated from independent evolution experiments and hybridized with the &lt;em&gt;SUL1&lt;/em&gt; probe. The ancestral fragment corresponds to the telomere-proximal &lt;em&gt;SUL1&lt;/em&gt; fragment. The more prominent bands of variable sizes correspond to fragments with novel chromosomal junctions. (C) Indirect end-labeling of &lt;em&gt;ApaL&lt;/em&gt;I double digests. The order of the lanes corresponds to the order in which the sites for the second restriction enzymes are found between &lt;em&gt;ApaL&lt;/em&gt;I and the telomere. (D) Southern blot of the double digests. The series of bands of increasing sizes in the Southern blot indicates that the portion of the genome from &lt;em&gt;SUL1&lt;/em&gt; to the telomere is intact. Fragments that contain the amplicon junction comigrate with the expected fragments only up to the position of the junction. Second enzymes whose sites lie distal to the amplicon junction fail to make a second cleavage and produce the amplicon-specific &lt;em&gt;ApaL&lt;/em&gt;I junction fragment. (E) A schematic illustrating the snap-back (SB)/S1 nuclease assay (S1). The rapid chilling of denatured &lt;em&gt;ApaL&lt;/em&gt;I fragments only permits reformation of dsDNA if the molecule is self-complementary. S1 treatment degrades all single stranded fragments including the ssDNA in the loop. (F) Southern analysis of the snap-back/S1-nuclease assay of population 4 clone1 using the &lt;em&gt;SUL1&lt;/em&gt; probe. The 14.6-kb amplicon-specific &lt;em&gt;ApaL&lt;/em&gt;I fragment generates an S1-resistant duplex molecule approximately half of its original size while the single strands of the ancestral fragment are degraded by S1 nuclease.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F2.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/399/F2.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Analysis of <em>SUL1</em> amplicons reveals inverted structures. (A) Map of the right telomeric region of chromosome II shows the position of <em>SUL1</em>, the relevant restriction enzyme sites, the deduced structure of Pop4 gen201 clone1, and the probe (<em>SUL1*</em>) used for Southern blot analysis. (B) Southern blot of <em>ApaL</em>I digests of seven clones containing <em>SUL1</em> amplifications isolated from independent evolution experiments and hybridized with the <em>SUL1</em> probe. The ancestral fragment corresponds to the telomere-proximal <em>SUL1</em> fragment. The more prominent bands of variable sizes correspond to fragments with novel chromosomal junctions. (C) Indirect end-labeling of <em>ApaL</em>I double digests. The order of the lanes corresponds to the order in which the sites for the second restriction enzymes are found between <em>ApaL</em>I and the telomere. (D) Southern blot of the double digests. The series of bands of increasing sizes in the Southern blot indicates that the portion of the genome from <em>SUL1</em> to the telomere is intact. Fragments that contain the amplicon junction comigrate with the expected fragments only up to the position of the junction. Second enzymes whose sites lie distal to the amplicon junction fail to make a second cleavage and produce the amplicon-specific <em>ApaL</em>I junction fragment. (E) A schematic illustrating the snap-back (SB)/S1 nuclease assay (S1). The rapid chilling of denatured <em>ApaL</em>I fragments only permits reformation of dsDNA if the molecule is self-complementary. S1 treatment degrades all single stranded fragments including the ssDNA in the loop. (F) Southern analysis of the snap-back/S1-nuclease assay of population 4 clone1 using the <em>SUL1</em> probe. The 14.6-kb amplicon-specific <em>ApaL</em>I fragment generates an S1-resistant duplex molecule approximately half of its original size while the single strands of the ancestral fragment are degraded by S1 nuclease.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464017/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464017/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Fitness coefficient of evolved clones</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">The average fitness of clones with five copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (46.3%) did not significantly differ from that of clones with three copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (43.3%). This result contrasts with the significant difference in fitness between the ancestral strains carrying ~20 copies of a 2-μm plasmid with the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> gene <em>vs.</em> one to two copies of a CEN plasmid with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). There was also no significant correlation between the size of the amplicon and the relative fitness (data not shown). These data support the hypothesis that extra copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>, but not their absolute copy number above a minimal threshold nor the extent of flanking sequences, significantly affect the fitness of cells during growth under sulfate limitation. However, any additional mutations carried by these strains could confound our ability to detect such a trend.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464024/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464024/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Fitness associated with an increased copy number of <em>SUL1</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3><em>SUL1</em> amplicons have an inverted repeat structure</h3><p id="p-25">To determine the chromosomal location of the additional copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> in each clone, we performed CHEF gel analysis coupled with Southern blot hybridization using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em> probes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS2.pdf">Figure S2</a>). In each case, chromosome II migrated more slowly and migrated the same distance as the band hybridized by the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> probe (data not shown), consistent with the amplified <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> sequences residing on chromosome II. Because sequence analysis of a previously characterized <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon revealed a tandem inverted structure for the additional copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>), we devised electrophoretic tests to detect potential inverted structures. These approaches allow both qualitative and quantitative characterization of clonal amplicons at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus. DNA from the ancestral strain was digested with <em>ApaL</em>1, a Southern blot was hybridized with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> probe, and the expected band of approximately 26 kb was observed (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Although this fragment also was detected in the amplified clones, an additional, variable fragment was detected as well (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The size of this additional band was roughly consistent with the aCGH data, assuming an inverted repeat orientation; the size of the amplification-specific band was equal to twice the distance from the <em>ApaL</em>I site to the telomere-proximal amplicon junction (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Similar results were found for the centromere-proximal junction using <em>EcoN</em>I digestion (data not shown).</p><p id="p-26">To confirm the inverted structure, we conducted indirect end-labeling using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> as a probe on Southern blots of genomic DNA cleaved with <em>ApaL</em>I and a series of second enzymes that cut at increasing distances from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> toward the telomere (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, C and D</a>). The doubly digested DNA from the ancestral strain produces a ladder of fragments of increasing size reflecting the order of restriction sites in the ancestral genome. Evolved clones with an inverted triplication, such as illustrated in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>, produce the same ladder of fragments because the telomere-proximal copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> is identical to that found in the ancestral strain. However, the <em>ApaL</em>I fragment unique to the inversion junction can only be cleaved by the enzymes that recognize the more centromere-proximal sites. All enzymes that recognize sites beyond the amplification junction leave the amplicon-specific <em>ApaL</em>I fragment intact (Pop4 210 clone1, <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). Indirect end labeling using <em>EcoN</em>I (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS3.pdf">Figure S3</a>) allowed us to map the inversion junction on the centromere-proximal side of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> for many of the clones. We analyzed each of the six clones in an identical manner and obtained results that are consistent with inversion junctions at the boundaries of the amplifications we found using aCGH (data not shown). As a final test of the inverted structure of the amplicons, we carried out snap-back assays (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tanaka <em>et al.</em> 2005</a>). If the amplicons-specific <em>ApaL</em>I fragments were inverted, then denaturation and rapid cooling would produce duplex hairpin fragments that are resistant to digestion with S1-nuclease (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2E</a>). Each of the six clones had an S1-resistant snapback DNA fragment of a size that is approximately half of the original <em>ApaL</em>I fragment (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2F</a> and data not shown). Snap-back analyses of the centromere-proximal junctions confirmed the inverted structure of the amplicons (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS3.pdf">Figure S3</a> and data not shown).</p></div><div id="sec-11" class="subsection"><h3>Sequencing of the <em>SUL1</em> amplicon junctions reveals inverted microhomologies</h3><p id="p-27">To map in detail the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon junctions of six of the evolved clones isolated at generation 200, we applied a split-read sequencing method designed to identify the exact junctions for complex events (tandem duplication, inversion and deletion) (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Karakoc <em>et al.</em> 2011</a>; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The accumulation of balanced split-reads (<em>i.e.</em>, split in the middle of the read) and unbalanced split-reads (<em>i.e.</em>, split on one side of the read) at a specific genomic locus is the signature of a rearrangement junction. For population 4 clone 1, we identified 17 split-reads mapping within the 893 bp windows encompassing the left junction and 18 split-reads within the 507 bp window for the right junctions identified by aCGH (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). All but one of the junctions showed pairs of 5- to 10-bp interrupted palindromic sequences flanking the junction; the twelfth junction was located in a CAG repeat region (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In confirmation of the aCGH data, the 12 junctions occurred at unique sites. The median distance between the two halves of the interrupted palindromic sequences was 40 bp (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The orientations of the split reads also confirmed the inverted structure of the amplicons. The interrupted palindromes are similar in structure to the junction sequences of a previously analyzed, evolved clone with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon bearing an inverted repeat structure (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detection of unique junctions using split-read methods. (A) Schematic diagram for the mapping of paired-end sequences at the junction. In a pair of reads where only one read is mapped (anchored), the second unmapped read is split into two parts and mapped to the genome. (B) Diagram for the split-read at the junction that contains an interrupted inverted repeat (H). (C) Expanded view of the last 35 kb of chromosome II containing the amplification of the SUL1 locus for Pop4 210 clone 1. The blue and pink boxes correspond to the regions in which the junctions of the amplification have occurred. The accumulation of split-reads that include the short inverted repeats (black arrows) indicates the junctions of the rearrangements in the evolved genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Detection of unique junctions using split-read methods. (A) Schematic diagram for the mapping of paired-end sequences at the junction. In a pair of reads where only one read is mapped (anchored), the second unmapped read is split into two parts and mapped to the genome. (B) Diagram for the split-read at the junction that contains an interrupted inverted repeat (H). (C) Expanded view of the last 35 kb of chromosome II containing the amplification of the &lt;em&gt;SUL1&lt;/em&gt; locus for Pop4 210 clone 1. The blue and pink boxes correspond to the regions in which the junctions of the amplification have occurred. The accumulation of split-reads that include the short inverted repeats (black arrows) indicates the junctions of the rearrangements in the evolved genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F3.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/3/399/F3.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464084" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Detection of unique junctions using split-read methods. (A) Schematic diagram for the mapping of paired-end sequences at the junction. In a pair of reads where only one read is mapped (anchored), the second unmapped read is split into two parts and mapped to the genome. (B) Diagram for the split-read at the junction that contains an interrupted inverted repeat (H). (C) Expanded view of the last 35 kb of chromosome II containing the amplification of the <em>SUL1</em> locus for Pop4 210 clone 1. The blue and pink boxes correspond to the regions in which the junctions of the amplification have occurred. The accumulation of split-reads that include the short inverted repeats (black arrows) indicates the junctions of the rearrangements in the evolved genome.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464135/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464135/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Junction signatures</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3><em>SUL1</em> amplification occurs early during the adaptation to sulfate limitation</h3><p id="p-30">To better understand the evolutionary dynamics of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification, we determined when amplicons appeared during the 200 generations of adaptation to sulfate limitation. We used real-time, qPCR on genomic DNA collected from the evolving populations at roughly 50-generation intervals (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). By generation ~50, we detected amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> in approximately half of the populations, and by generation ~100, in all of the populations. The average population copy number of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> in the six completed evolution experiments ranged from 1.9 to 4.2, values consistent with the estimates of copy number in the final clones. Although these results confirm that adaptation to sulfate limitation proceeds via the amplification of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> gene, the kinetics of the amplification and the final copy number achieved varied between the replicate evolution experiments, suggesting that each population experienced different evolutionary trajectories over the course of sulfate-limited adaptation.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary dynamics of SUL1 amplification of experimental populations evolved in sulfate limitation medium. The copy number of SUL1 was assessed using qPCR analysis on samples taken from Populations 2 through 7 every ~50 generations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evolutionary dynamics of &lt;em&gt;SUL1&lt;/em&gt; amplification of experimental populations evolved in sulfate limitation medium. The copy number of &lt;em&gt;SUL1&lt;/em&gt; was assessed using qPCR analysis on samples taken from Populations 2 through 7 every ~50 generations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F4.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/3/399/F4.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Evolutionary dynamics of <em>SUL1</em> amplification of experimental populations evolved in sulfate limitation medium. The copy number of <em>SUL1</em> was assessed using qPCR analysis on samples taken from Populations 2 through 7 every ~50 generations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Clonal interference is commonly observed during the adaptation to sulfate limitation</h3><p id="p-32">Population-level data from experimental evolution can be analyzed for evidence of clonal interference, suggesting the origin and fate of adaptive mutations. Because the qPCR analysis provides only population averages of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification, we performed electrophoresis-based analysis of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicons to track the frequencies of distinct subpopulations. We isolated DNA from chemostat samples at regular intervals over each ∼200 generation experiment and digested aliquots separately with <em>EcoN</em>1 and <em>ApaL</em>1 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A−C</a>). Using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> as the hybridization probe, we were able to detect when new amplification junctions arose (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, B and C</a>). Hybridization of the Southern blots with a probe from a genomic region with a copy number of one (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em>; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, B and C</a>) allowed us to quantify the prevalence of each amplicon junction over the course of the sulfate-limited growth. With this assay, we were able not only to follow the overall dynamics of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification but also to identify subpopulations that carry unique <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification junctions and to track their frequencies in the population. Although we can assess the relative abundance of each amplicon junction (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>), it should be noted that the relative abundance of each subpopulation reported by this assay is necessarily a composite of the subpopulation frequency and the clonal copy number of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon. To disentangle these two variables, we simultaneously isolated genomic DNA in agarose plugs for karyotype analysis using CHEF gel electrophoresis (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>). Using <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006464">CEN2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> as hybridization probes, we could detect when increases in the size of chromosome II occurred (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5E</a>) and the number of copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> on each new version of chromosome II (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5F</a>). During the course of the seven evolution experiments, we detected a minimum of 16 new versions of chromosome II with chromosomes containing three copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> being replaced by chromosomes with higher copy numbers over time. Although we could not measure significant differences in fitness for three <em>vs.</em> five copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> (43.3% <em>vs.</em> 46.3%, respectively), five of the six evolution experiments that reached ~200 generation were being overtaken by higher copy-number clones. Most of the increases in copy number were not accompanied by changes in junction fragments, suggesting that the increase in copy number was a consequence of unequal crossing over that expanded <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> arrays from three to five copies.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Kinetics of SUL1 amplicon formation during ~200 generations of sulfate-limited growth. (A) A map of the SUL1 region of chromosome II showing the positions of ApaLI and EcoNI restriction sites used to digest DNA isolated from different samples of evolution #4 (Pop4). The location of SUL1, a hypothetical structure of an inverted amplicon, and the probe used for Southern hybridization are also shown. (B) EcoNI digestion and electrophoretic separation of chromosomal fragments recovered during sulfate limited growth. Top, control hybridization of the Southern blot with a single copy sequence ARS305. Bottom, hybridization of the EcoNI blot with SUL1. (C) ApaLI digestion and electrophoretic separation of the same DNA samples as in panel B. Top, hybridization of the blot with ARS305. Bottom, hybridization of the blot with SUL1. (D). Quantification of different amplicons during evolution #4 (Pop4) using ARS305 hybridization for normalization. A minimum of four unique amplicons were detected for both digests. Their pattern of abundance, appearance and disappearance identifies which proximal and distal junction fragments make up individual amplicons (for example, 4-2). (E) CHEF gel analysis of population samples of Pop4. The Southern blot was hybridized sequentially with a CEN2 probe (left) and then a SUL1 probe (right). (F) Determination of SUL1 copy number on variant copies of chromosome II. For each form of chromosome II that accounted for at least 20% of the chromosome IIs in the population, the ratio of SUL1/CEN2 was normalized to the ratio at the start of the evolution experiment to determine the copy number of SUL1 for each variant chromosome II. For several evolution experiments multiple different forms of chromosome II transiently coexisted. (G) Chart showing the proportion of clones from Pop4 detected by single clone analysis using ApaL1 digestion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Kinetics of &lt;em&gt;SUL1&lt;/em&gt; amplicon formation during ~200 generations of sulfate-limited growth. (A) A map of the &lt;em&gt;SUL1&lt;/em&gt; region of chromosome II showing the positions of &lt;em&gt;ApaL&lt;/em&gt;I and &lt;em&gt;EcoN&lt;/em&gt;I restriction sites used to digest DNA isolated from different samples of evolution #4 (Pop4). The location of &lt;em&gt;SUL1&lt;/em&gt;, a hypothetical structure of an inverted amplicon, and the probe used for Southern hybridization are also shown. (B) &lt;em&gt;EcoN&lt;/em&gt;I digestion and electrophoretic separation of chromosomal fragments recovered during sulfate limited growth. Top, control hybridization of the Southern blot with a single copy sequence &lt;em&gt;ARS305&lt;/em&gt;. Bottom, hybridization of the &lt;em&gt;EcoN&lt;/em&gt;I blot with &lt;em&gt;SUL1&lt;/em&gt;. (C) &lt;em&gt;ApaL&lt;/em&gt;I digestion and electrophoretic separation of the same DNA samples as in panel B. Top, hybridization of the blot with &lt;em&gt;ARS305&lt;/em&gt;. Bottom, hybridization of the blot with &lt;em&gt;SUL1&lt;/em&gt;. (D). Quantification of different amplicons during evolution #4 (Pop4) using &lt;em&gt;ARS305&lt;/em&gt; hybridization for normalization. A minimum of four unique amplicons were detected for both digests. Their pattern of abundance, appearance and disappearance identifies which proximal and distal junction fragments make up individual amplicons (for example, 4-2). (E) CHEF gel analysis of population samples of Pop4. The Southern blot was hybridized sequentially with a &lt;em&gt;CEN2&lt;/em&gt; probe (left) and then a &lt;em&gt;SUL1&lt;/em&gt; probe (right). (F) Determination of &lt;em&gt;SUL1&lt;/em&gt; copy number on variant copies of chromosome II. For each form of chromosome II that accounted for at least 20% of the chromosome IIs in the population, the ratio of &lt;em&gt;SUL1/CEN2&lt;/em&gt; was normalized to the ratio at the start of the evolution experiment to determine the copy number of &lt;em&gt;SUL1&lt;/em&gt; for each variant chromosome II. For several evolution experiments multiple different forms of chromosome II transiently coexisted. (G) Chart showing the proportion of clones from Pop4 detected by single clone analysis using &lt;em&gt;ApaL&lt;/em&gt;1 digestion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F5.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/3/399/F5.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464050" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Kinetics of <em>SUL1</em> amplicon formation during ~200 generations of sulfate-limited growth. (A) A map of the <em>SUL1</em> region of chromosome II showing the positions of <em>ApaL</em>I and <em>EcoN</em>I restriction sites used to digest DNA isolated from different samples of evolution #4 (Pop4). The location of <em>SUL1</em>, a hypothetical structure of an inverted amplicon, and the probe used for Southern hybridization are also shown. (B) <em>EcoN</em>I digestion and electrophoretic separation of chromosomal fragments recovered during sulfate limited growth. Top, control hybridization of the Southern blot with a single copy sequence <em>ARS305</em>. Bottom, hybridization of the <em>EcoN</em>I blot with <em>SUL1</em>. (C) <em>ApaL</em>I digestion and electrophoretic separation of the same DNA samples as in panel B. Top, hybridization of the blot with <em>ARS305</em>. Bottom, hybridization of the blot with <em>SUL1</em>. (D). Quantification of different amplicons during evolution #4 (Pop4) using <em>ARS305</em> hybridization for normalization. A minimum of four unique amplicons were detected for both digests. Their pattern of abundance, appearance and disappearance identifies which proximal and distal junction fragments make up individual amplicons (for example, 4-2). (E) CHEF gel analysis of population samples of Pop4. The Southern blot was hybridized sequentially with a <em>CEN2</em> probe (left) and then a <em>SUL1</em> probe (right). (F) Determination of <em>SUL1</em> copy number on variant copies of chromosome II. For each form of chromosome II that accounted for at least 20% of the chromosome IIs in the population, the ratio of <em>SUL1/CEN2</em> was normalized to the ratio at the start of the evolution experiment to determine the copy number of <em>SUL1</em> for each variant chromosome II. For several evolution experiments multiple different forms of chromosome II transiently coexisted. (G) Chart showing the proportion of clones from Pop4 detected by single clone analysis using <em>ApaL</em>1 digestion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Using these two gel assays, we detected the first <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicons between 46 and 71 generations (Pop4, <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5, B−E</a> and Pop1−3, 5−7, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS4.pdf">Figure S4</a>). Over the course of all evolution experiments, we observed the presence of multiple subpopulations, each carrying different <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicons. The subpopulations in all of the evolution experiments demonstrated two distinct behaviors: at least one subpopulation persisted throughout the course of the experiment, and additional transient subpopulations rose to different frequencies and then fell below the level of detection before the end of the experiment.</p><p id="p-35">In Population 4, several subpopulations were already observable at generation 59, and most persisted throughout the course of the evolution experiment although they fluctuated in frequency over time (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5B−D</a>). For example, after generation 150, the two predominant subpopulations (4-1 and 4-4) declined in frequency whereas the third subpopulation expanded (4-2; <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5E</a>)—a result consistent with clonal interference. These data clearly demonstrate the presence of multiple adaptive events and reflect a greater diversity within subpopulations.</p><p id="p-36">To further disambiguate clone frequency and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> copy number, we verified these results by examining 45 clones from Population 4 at generations 52 and 202 by using the electrophoretic assay focused on the centromere-proximal junction of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon. The increase in frequency of clones corresponding to subpopulation 4-2 at the expense of subpopulation 4-3 between generations 50 and 200 matches our observations from the population analysis (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5F</a>).</p><p id="p-37">Because all strains generate wild-type <em>ApaL</em>I and <em>EcoN</em>I bands in the electrophoretic assay, we were unable to determine what fraction of the populations did not carry an amplicon, or whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification had become fixed. To specifically investigate the dynamics of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> copy number and population frequency of its amplification throughout the course of each evolution experiment, we used qPCR on a total of 506 independent clones, isolated from each experiment at generations ~50 and ~200 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/TableS1.xlsx">Table S1</a>). At generation ~50 we did not detect any amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> in two of the populations (Population 2 and Population 5) whereas in the four remaining populations, 21–97% of clones contained a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification event. At generation ~200, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification was present at high frequency in four of the six populations (80–98% of clones) and had apparently become fixed in two populations (Pop 2 and 7; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Some of the clones examined may have lost their amplicon by homologous recombination during the period of nonselective growth after removal from the chemostat. However, the CHEF analysis of population samples not subjected to nonselective growth (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS4.pdf">Figure S4</a>) confirmed that in some of the evolution experiments, a small subset of cells (from 1 to 15%) still retained the ancestral-sized chromosome II. In general, the presence or absence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification at generation ~50 could not be used as a predictor for fixation at generation ~200. These data establish the fixation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification at ~200 generations in two populations (Population 2 and 7). Even in these two populations, multiple coexisting subpopulations arose together possibly indicating the presence of other mutations in these populations as well.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/399/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of SUL1 amplification at generations ~50 and ~200. qPCR was used to determine the percentage of clones with SUL1 amplification at generations ~50 and ~200 in six populations (in black) vs. percentage of clones found with only one copy of SUL1 (in gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2051485523" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency of &lt;em&gt;SUL1&lt;/em&gt; amplification at generations ~50 and ~200. qPCR was used to determine the percentage of clones with &lt;em&gt;SUL1&lt;/em&gt; amplification at generations ~50 and ~200 in six populations (in black) &lt;em&gt;vs.&lt;/em&gt; percentage of clones found with only one copy of &lt;em&gt;SUL1&lt;/em&gt; (in gray).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/399/F6.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/3/399/F6.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/399/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/399/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464081" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-38" class="first-child">Frequency of <em>SUL1</em> amplification at generations ~50 and ~200. qPCR was used to determine the percentage of clones with <em>SUL1</em> amplification at generations ~50 and ~200 in six populations (in black) <em>vs.</em> percentage of clones found with only one copy of <em>SUL1</em> (in gray).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Alternative adaptive trajectories are rarely observed in populations evolving under sulfate limitation</h3><p id="p-39">The varying dynamics of amplification observed among different subpopulations, the decrease in the frequency of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification in Population 3, and the fact that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification is not fixed in most of the populations at generation ~200 suggest that additional mutations have occurred that may interfere with the fixation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplifications (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). We looked directly for such possible mutations by isolating nine clones from Population 3 that did not contain a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon at generation ~200 and determining their relative fitness. Except for one clone, the fitness change compared with the ancestral strain was minimal (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). One clone was 26.25% more fit than the ancestral strain. To rule out the possibility that cells with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification were selected during the 15 generations of competition used to determine the relative fitness, we performed qPCR on a sample from the last day of the competition experiment in which 96% of the cells corresponded to the Population 3 clone. Amplification at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus was not detected by qPCR, and the absence of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification or other major variations was confirmed by aCGH (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS5.pdf">Figure S5</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464091/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464091/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Fitness coefficient of clones from Pop3 at generation 210 without <em>SUL1</em> amplification</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">To investigate the genetic changes underlying the fitness increase of this clone, we sequenced its genome at 60x coverage and detected <em>de novo</em> mutations relative to the ancestral strain. Consistent with the aCGH data, no large CNVs were detected in this strain. Our initial analysis predicted a total of three point mutations: one in a telomeric region and two nonsynonymous mutations in coding regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/TableS2.xlsx">Table S2</a>). We validated the two nonsynonymous mutation calls by Sanger sequencing. No mutations were predicted in the coding sequence or the upstream regulatory sequences of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em>, ruling out an increase in the expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> due to a mutation in <em>cis</em>. As a convergent mutation may provide additional evidence of adaptive events, we compared this list of mutations to the genome sequences obtained for the additional evolved clones, plus those we previously detected by tiling arrays (<a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>) and by whole-genome sequencing (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>). We found that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> was mutated in two clones from this study (Population 3 clone described previously and a final clone from Population 7) and two other clones from our previous studies (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>; <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">Sgf73</a> is a subunit of the SAGA histone acetylase complex required for the assembly of the histone deubiquitination module and the yeast ortholog of Ataxin-7 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kohler <em>et al.</em> 2008</a>). We confirmed the mutations in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> in these four clones by Sanger sequencing (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS6.pdf">Figure S6</a>). In every instance, the mutation is a nonsense mutation predicted to truncate the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> gene product, suggesting that inactivation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> may be advantageous in sulfate limitation. Interestingly, two of the evolved clones with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> mutations also carried the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification while two did not. To examine further the physiological effect of this allele, we determined the relative fitness in both sulfate and glucose limitation of a strain deleted for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> [obtained from the Yeast deletion collection (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Giaever <em>et al.</em> 2002</a>)]. This strain had never previously been cultivated in sulfate-limited media. We found that the deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> has a small deleterious effect in glucose limitation (−3.64% ± 0.62) but has an increased fitness of 24.3% in sulfate limitation, demonstrating that the positive fitness effects of this mutation may be specific to sulfate limitation. No previous connection to sulfate metabolism has been reported. Although the role that this gene plays in a sulfate-limited environment has yet to be elucidated, its positive effect on fitness under sulfate-limiting conditions strongly suggests this mutation offers an alternative adaptive trajectory to this strong selective pressure.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-41">The adaptation of <em>S. cerevisiae</em> to limited sulfate conditions during long-term evolution experiments provides a powerful approach to study an adaptive trajectory. In this work, we show that the vast majority of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification products that arise in sulfate-limited growth are <em>in situ</em> inverted amplicons with unique junctions coinciding with genomic sequences that consist of short, interrupted palindromes, a result we first discovered for a single clone (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>). The unique structure of these amplicons and their predictable occurrence during sulfate-limited growth provides a rare opportunity to study potential mechanisms that generate this interstitial, inverted form of gene amplification (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brewer <em>et al.</em> 2011</a>). This system also provides a unique opportunity to study how gene amplification contributes to adaptation under strong selection, a topic of much recent interest (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sonti and Roth 1989</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Koszul <em>et al.</em> 2004</a>; <a id="xref-ref-18-10" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">2010</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kugelberg <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Blount <em>et al.</em> 2012</a>). By following the trajectory of amplification of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> over time with qPCR and electrophoretic techniques, we find that genetically distinct subpopulations arise within ~60 generations, coexist to variable degrees over ~200 generations, and change in relative representation as one adaptive variant replaces another. Thus, these evolution experiments display direct support for clonal interference. Unlike point mutation, amplification as an adaptive strategy has the unique property that it is easily reversible by intrachromosomal homologous recombination (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 1998</a>). Further analysis of the fitness and rates of amplification and contraction at this locus across conditions will be required to resolve the importance of this possibility.</p><p id="p-42">Although we previously demonstrated that strains with increased <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> copy number also showed increased RNA abundance (<a id="xref-ref-18-11" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>), we have not measured RNA content for the particular populations reported here. Quantitative assays of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> RNA and protein levels would allow us to measure the extent of correlation between copy number, mRNA and protein levels, and let us directly measure how these molecular phenotypes correlate with fitness in sulfate limitation and other conditions.</p><p id="p-43">Among rare clones from the last day of Population 3 that had either persisted without <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification or had recently lost their <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicons, we found one clone that had acquired an adaptive point mutation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> gene. The fitness increase of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> mutation was substantially lower than that provided by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification (25% <em>vs.</em> 43%), suggesting that adaptive point mutations, if and when they occur, cannot compete with the added advantage that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification provides. In a previous study, we found that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> mutations rose to 15–20% allele frequency in two populations; however, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification status of these subpopulations was not determined (<a id="xref-ref-18-12" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>).</p><p id="p-44">The observation of subpopulations that are genetically distinct at one locus throughout the course of the evolution experiments is consistent with previous studies in large populations of bacteria that demonstrate periodic selection of more fit clones harboring different beneficial mutations in the same gene (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Notley-McRobb and Ferenci 2000</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lee and Marx 2013</a>). The coexistence of transient large-scale rearrangements at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> locus in all seven of the evolution experiments provides direct evidence of strong competition among the evolving subpopulations. Although <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification was expected in each of the populations, the apparent simultaneous occurrence of multiple independent amplicons within each single culture had not been systematically observed in previous studies. With a starting population size in the chemostats of 10<sup>9</sup> individuals and an estimated rate of amplicon formation of 10<sup>−7</sup>/cell/division (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Payen <em>et al.</em> 2008</a>), we considered the possibility that amplicons may have pre-existed in the inoculum cultures, and that they swept the population, as predicted from theoretical models (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wahl and Krakauer 2000</a>). This possibility may explain why we first detected amplicons at roughly the same time in different chemostat cultures. This finding could also mean that there is a low probability that an advantageous mutation can occur <em>de novo</em> after the chemostat culture has been established or, if it does arise, that it fails to sweep the population.</p><p id="p-45">Ultra-deep sequencing of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> flanking sequences from the ancestral strain inoculum and from populations over the course of the evolution experiment might allow us to detect rare initial and failed <em>de-novo</em> events. In addition to the common form of amplification event, we also note another trend over the course of the evolution experiments: the major amplicon in five of the seven populations increased in copy number from three to five as judged by the jumps in chromosome II size (<a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5, D and E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-46">Recently, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yona <em>et al.</em> (2012)</a> have proposed that large copy-number variants are first acquired during the course of an evolution experiment and are rapidly replaced by a more refined adaptive solution (<em>e.g.</em>, elevated expression of few genes). In our case, we anticipated the appearance of point mutations that would increase <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> expression. The sequencing data of our clones did not reveal any mutations within the promoter of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> and a previous study likewise did not show any increase in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> expression independent of the effect of the copy number in clones isolated at 120−300 generations (<a id="xref-ref-18-13" class="xref-bibr" href="#ref-18">Gresham <em>et al.</em> 2008</a>). Because even small regions of aneuploidy will cause an overproduction of proteins that could lead to an accumulation of misfolded proteins, a condition known as proteotoxic stress (reviewed in <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tang and Amon 2013</a>), we imagined that there would be a fitness cost for retaining large stretches of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> flanking DNA as part of the amplicon. As a consequence, we expected to see that the shortened amplicons would replace larger amplicons over time. However, that was not the case: for example, clone 4-4 (<a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5, B and C</a>) contained the smallest amplicons of the four coexisting subpopulations and was on its way to extinction as other larger amplicons remained. Possible explanations for this result are that additional driver genes might be present on the longer amplicons, or beneficial point mutations could have arisen in the genome. As discussed previously, we detected such a beneficial point mutation in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em> gene in a clone that appeared to have escaped or lost its <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicon. Independent mutations in this gene were found in other cultures, most notably in a final clone from Population 7 that contained both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification and the nonsense mutation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">SGF73</a></em>. The four independent mutations leading to the truncation of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003034">Sgf73</a>, are an example of convergent evolution at the gene level (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Woods <em>et al.</em> 2006</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tenaillon <em>et al.</em> 2012</a>). Although we did not test it directly, it appears that the fitnesses associated with these two types of events are not additive, as the fitness of this particular clone is actually below the average of the six tested clones. Because our experiment was limited to ~200 generations, we cannot rule out the possibility that the beneficial mutations acquired and observed in our study are transient and that they would have been replaced by a more efficient and sustainable solution given more time. Exploring the stability and the persistence of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplification over a longer period of time will be key to understanding the dynamics between transient and possibly costly aneuploidy events and more refined mechanisms of adaptation to nutrient stress.</p><p id="p-47">Our evolutionary studies in sulfate-limited chemostats also provide an efficient experimental tool to explore the mechanism that produces inverted, <em>in situ</em> amplicons. Because <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> is located near the telomere of chromosome II, the intrinsic instability associated with subtelomeric genomic domains may play a pivotal role in amplification of this region. However, the predominant form of instability associated with subtelomeres has been ascribed to their high levels of inter- and intrachromosomal recombination (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pryde and Louis 1997</a>). Little is known about the instability of such inverted repeat structures; however, repeat numbers of three or five could easily resolve to single copy by unequal recombination or pop-out recombination at the alternating, directly repeating copies of the amplicon. A previous study reported that the frequency of loss of direct tandem duplications is positively correlated with the size of the amplicon (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Koszul <em>et al.</em> 2006</a>). The possible role of recombination in the generation of these palindrome-associated, inverted amplification events remains obscure. We favor an alternative mechanism for the generation of these specific inverted amplicons based on aberrant replication fork processing (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brewer <em>et al.</em> 2011</a>). Because we have demonstrated that inverted <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000498">SUL1</a></em> amplicons are the predominant solution to growth in sulfate-limited chemostats, we are now in position to dissect the genetic and molecular requirements for this under-explored and under-appreciated mode of gene amplification.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank the members of the Brewer/Raghuraman and Dunham labs for helpful discussions and Stan Fields, Gilles Fischer, and Ben Kerr for helpful comments on the manuscript. We also thank David Breslow and Jonathan Weissman for their <em>eGFP</em> construct, Barry Dion for constructing the <em>ho</em>::<em>KanMX-GFP</em> strain used for our competition experiments, and Angelika Amon, Eduardo Torres, and the Nickerson lab for assistance with the DNA sequencing. This work was supported by NSF grant 1120425 and NIGMS grants GM18926 to BJB and MKR and GM094306 to MJD. MJD is a Rita Allen Foundation Scholar and a CIFAR Fellow. ABS was supported by T32 AG000057 and F30CA165440.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.009365/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-2">Microarray data from this article have been deposited in the Gene expression Omnibus (GEO) repository under accession GSE47854 (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) and in the Princeton Microarray Database (<a href="http://puma.princeton.edu">http://puma.princeton.edu</a>).</p></li><li class="fn" id="fn-3"><p id="p-3">Sequencing data have been deposited with the National Center for Biotechnology Information under BioSample accessions SAMN02208069, SAMN02208070, SAMN02208071, SAMN02208072, SAMN02208073, SAMN02208074, and SAMN02208075.</p></li><li class="fn" id="fn-6"><p id="p-49"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-08"><span class="received-label">Received </span>November 8, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-21"><span class="accepted-label">Accepted </span>December 21, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Payen <em>et al.</em></li></ul><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.1" data-doi="10.1099/mic.0.058024-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adamo</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamás</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brocca</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Amplification of the <em>CUP1</em> gene is associated with evolution of copper tolerance in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.058024-0%26rft_id%253Dinfo%253Apmid%252F22790396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.058024-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22790396&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309852800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.2" data-doi="10.1126/science.282.5391.1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slechta</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Evidence that gene amplification underlies adaptive mutability of the bacterial <em>lac</em> operon.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAndersson%26rft.auinit1%253DD.%2BI.%26rft.volume%253D282%26rft.issue%253D5391%26rft.spage%253D1133%26rft.epage%253D1135%26rft.atitle%253DEvidence%2BThat%2BGene%2BAmplification%2BUnderlies%2BAdaptive%2BMutability%2Bof%2Bthe%2BBacterial%2Blac%2BOperon%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5391.1133%26rft_id%253Dinfo%253Apmid%252F9804552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5MS8xMTMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.3" data-doi="10.1186/1471-2164-11-88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araya</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing of a laboratory-evolved yeast strain.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAraya%26rft.auinit1%253DC.%2BL.%26rft.volume%253D11%26rft.spage%253D88%26rft.epage%253D88%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Ba%2Blaboratory-evolved%2Byeast%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-88%26rft_id%253Dinfo%253Apmid%252F20128923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20128923&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.4" data-doi="10.1038/nature08480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome evolution and adaptation in a long-term experiment with <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D461%26rft.issue%253D7268%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DGenome%2Bevolution%2Band%2Badaptation%2Bin%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08480%26rft_id%253Dinfo%253Apmid%252F19838166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19838166&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271190800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.5" data-doi="10.1038/nature11514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blount</span>  <span class="cit-name-given-names">Z. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic analysis of a key innovation in an experimental <em>Escherichia coli</em> population.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBlount%26rft.auinit1%253DZ.%2BD.%26rft.volume%253D489%26rft.issue%253D7417%26rft.spage%253D513%26rft.epage%253D518%26rft.atitle%253DGenomic%2Banalysis%2Bof%2Ba%2Bkey%2Binnovation%2Bin%2Ban%2Bexperimental%2BEscherichia%2Bcoli%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11514%26rft_id%253Dinfo%253Apmid%252F22992527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992527&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309167100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.6" data-doi="10.1371/journal.pgen.1002016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrewer%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1002016%26rft.epage%253De1002016%26rft.atitle%253DOrigin-dependent%2Binverted-repeat%2Bamplification%253A%2Ba%2Breplication-based%2Bmodel%2Bfor%2Bgenerating%2Bpalindromic%2Bamplicons.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002016%26rft_id%253Dinfo%253Apmid%252F21437266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437266&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.7" data-doi="10.1093/oxfordjournals.molbev.a026009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D931%26rft.epage%253D942%26rft.atitle%253DMultiple%2Bduplications%2Bof%2Byeast%2Bhexose%2Btransport%2Bgenes%2Bin%2Bresponse%2Bto%2Bselection%2Bin%2Ba%2Bglucose-limited%2Benvironment%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026009%26rft_id%253Dinfo%253Apmid%252F9718721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzgvOTMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.8" data-doi="10.1098/rspb.2012.0799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">How does adaptation sweep through the genome? Insights from long-term selection experiments.</span> <abbr class="cit-jnl-abbrev">Proc. Roy. Soc. B.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">5029</span>–<span class="cit-lpage">5038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BRoy.%2BSoc.%2BB.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.0799%26rft_id%253Dinfo%253Apmid%252F22833271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjc5LzE3NDkvNTAyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.9" data-doi="10.1016/j.femsyr.2004.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cakar</span>  <span class="cit-name-given-names">Z. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seker</span>  <span class="cit-name-given-names">U. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamerler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonderegger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionary engineering of multiple-stress resistant <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsyr.2004.10.010%26rft_id%253Dinfo%253Apmid%252F15780656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjk6IjUvNi03LzU2OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherest</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidian</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ansorge</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular characterization of two high affinity sulfate transporters in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCherest%26rft.auinit1%253DH.%26rft.volume%253D145%26rft.issue%253D3%26rft.spage%253D627%26rft.epage%253D635%26rft.atitle%253DMolecular%2BCharacterization%2Bof%2BTwo%2BHigh%2BAffinity%2BSulfate%2BTransporters%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9055073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ1LzMvNjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.11" data-doi="10.1016/j.celrep.2011.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimization of gene expression through divergent mutational paths.</span> <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DChou%26rft.auinit1%253DH.%2BH.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D140%26rft.atitle%253DOptimization%2Bof%2Bgene%2Bexpression%2Bthrough%2Bdivergent%2Bmutational%2Bpaths.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2011.12.003%26rft_id%253Dinfo%253Apmid%252F22832162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2011.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22832162&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.12" data-doi="10.1016/j.cub.2007.01.072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The speed of evolution and maintenance of variation in asexual populations.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSchuh%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D385%26rft.epage%253D394%26rft.atitle%253DThe%2Bspeed%2Bof%2Bevolution%2Band%2Bmaintenance%2Bof%2Bvariation%2Bin%2Basexual%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.01.072%26rft_id%253Dinfo%253Apmid%252F17331728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.01.072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17331728&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244817000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.13" data-doi="10.1126/science.283.5400.404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Visser</span>  <span class="cit-name-given-names">J. A. G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeyl</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerrish</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Diminishing returns from mutation supply rate in asexual populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DArjan%2BG.%26rft.auinit1%253DJ.%26rft.volume%253D283%26rft.issue%253D5400%26rft.spage%253D404%26rft.epage%253D406%26rft.atitle%253DDiminishing%2BReturns%2Bfrom%2BMutation%2BSupply%2BRate%2Bin%2BAsexual%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.283.5400.404%26rft_id%253Dinfo%253Apmid%252F9888858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNTQwMC80MDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.14" data-doi="10.1002/yea.1826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzi</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolczynski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic, genomic, and molecular tools for studying the protoploid yeast, <em>L. waltii</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DDi%2BRienzi%26rft.auinit1%253DS.%2BC.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D151%26rft.atitle%253DGenetic%252C%2Bgenomic%252C%2Band%2Bmolecular%2Btools%2Bfor%2Bstudying%2Bthe%2Bprotoploid%2Byeast%252C%2BL.%2Bwaltii.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1826%26rft_id%253Dinfo%253Apmid%252F21246627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21246627&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287605600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.15" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.16" data-doi="10.1023/A:1017067816551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerrish</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The fate of competing beneficial mutations in an asexual population.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102–103</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1017067816551%26rft_id%253Dinfo%253Apmid%252F9720276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1017067816551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9720276&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.17" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.18" data-doi="10.1371/journal.pgen.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenq</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGresham%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DThe%2Brepertoire%2Band%2Bdynamics%2Bof%2Bevolutionary%2Badaptations%2Bto%2Bcontrolled%2Bnutrient-limited%2Benvironments%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000303%26rft_id%253Dinfo%253Apmid%252F19079573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079573&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.19" data-doi="10.1073/pnas.1014023107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usaite</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Germann</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation to diverse nitrogen-limited environments by deletion or extrachromosomal element formation of the <em>GAP1</em> locus.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">18551</span>–<span class="cit-lpage">18556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1014023107%26rft_id%253Dinfo%253Apmid%252F20937885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQzLzE4NTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.20" data-doi="10.1007/s10529-008-9779-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domingues</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Fermentation of high concentrations of lactose to ethanol by engineered flocculent <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Lett.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1953</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Bletters%26rft.stitle%253DBiotechnol%2BLett%26rft.aulast%253DGuimaraes%26rft.auinit1%253DP.%2BM.%26rft.volume%253D30%26rft.issue%253D11%26rft.spage%253D1953%26rft.epage%253D1958%26rft.atitle%253DFermentation%2Bof%2Bhigh%2Bconcentrations%2Bof%2Blactose%2Bto%2Bethanol%2Bby%2Bengineered%2Bflocculent%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10529-008-9779-1%26rft_id%253Dinfo%253Apmid%252F18575804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10529-008-9779-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18575804&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.21" data-doi="10.1038/nmeth0810-576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">mrsFAST: a cache-oblivious algorithm for short-read mapping.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DHach%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D576%26rft.epage%253D577%26rft.atitle%253DmrsFAST%253A%2Ba%2Bcache-oblivious%2Balgorithm%2Bfor%2Bshort-read%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0810-576%26rft_id%253Dinfo%253Apmid%252F20676076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0810-576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20676076&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280500000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.22" data-doi="10.1371/journal.pbio.1001490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herron</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebeli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Parallel evolutionary dynamics of adaptive diversification in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001490%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001490%26rft_id%253Dinfo%253Apmid%252F23431270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23431270&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.23" data-doi="10.1038/nbt.1534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magtanong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A molecular barcoded yeast ORF library enables mode-of-action analysis of bioactive compounds.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHo%26rft.auinit1%253DC.%2BH.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D369%26rft.epage%253D377%26rft.atitle%253DA%2Bmolecular%2Bbarcoded%2Byeast%2BORF%2Blibrary%2Benables%2Bmode-of-action%2Banalysis%2Bof%2Bbioactive%2Bcompounds.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1534%26rft_id%253Dinfo%253Apmid%252F19349972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349972&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264971800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.24" data-doi="10.1016/0378-1119(87)90131-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A ten-minute DNA preparation from yeast efficiently releases autonomous plasmids for transformation of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHoffman%26rft.auinit1%253DC.%2BS.%26rft.volume%253D57%26rft.issue%253D2-3%26rft.spage%253D267%26rft.epage%253D272%26rft.atitle%253DA%2Bten-minute%2BDNA%2Bpreparation%2Bfrom%2Byeast%2Befficiently%2Breleases%2Bautonomous%2Bplasmids%2Bfor%2Btransformation%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252887%252990131-4%26rft_id%253Dinfo%253Apmid%252F3319781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(87)90131-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3319781&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K812400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.25" data-doi="10.1016/0092-8674(87)90643-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huberman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spotila</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nawotka</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">el-Assouli</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The in vivo replication origin of the yeast 2μm plasmid.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHuberman%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DThe%2Bin%2Bvivo%2Breplication%2Borigin%2Bof%2Bthe%2Byeast%2B2%2Bmicrons%2Bplasmid.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252887%252990643-X%26rft_id%253Dinfo%253Apmid%252F3311385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(87)90643-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3311385&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K785400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.26" data-doi="10.1038/ng.280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Molecular characterization of clonal interference during adaptive evolution in asexual populations of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1499</span>–<span class="cit-lpage">1504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKao%26rft.auinit1%253DK.%2BC.%26rft.volume%253D40%26rft.issue%253D12%26rft.spage%253D1499%26rft.epage%253D1504%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bclonal%2Binterference%2Bduring%2Badaptive%2Bevolution%2Bin%2Basexual%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.280%26rft_id%253Dinfo%253Apmid%252F19029899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029899&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261215900030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.27" data-doi="10.1038/nmeth.1810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karakoc</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O'Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vives</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detection of structural variants and indels within exome data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D9%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1810%26rft_id%253Dinfo%253Apmid%252F22179552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22179552&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.28" data-doi="10.1038/ncb1733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabal</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nehrbass</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurt</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast Ataxin-7 links histone deubiquitination with gene gating and mRNA export.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DKohler%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D707%26rft.epage%253D715%26rft.atitle%253DYeast%2BAtaxin-7%2Blinks%2Bhistone%2Bdeubiquitination%2Bwith%2Bgene%2Bgating%2Band%2BmRNA%2Bexport.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1733%26rft_id%253Dinfo%253Apmid%252F18488019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18488019&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256350100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.29" data-doi="10.1038/sj.emboj.7600024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caburet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Eucaryotic genome evolution through the spontaneous duplication of large chromosomal segments.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKoszul%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D234%26rft.epage%253D243%26rft.atitle%253DEucaryotic%2Bgenome%2Bevolution%2Bthrough%2Bthe%2Bspontaneous%2Bduplication%2Bof%2Blarge%2Bchromosomal%2Bsegments%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600024%26rft_id%253Dinfo%253Apmid%252F14685272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8xLzIzNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.30" data-doi="10.1534/genetics.105.048058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Stability of large segmental duplications in the yeast genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048058%26rft_id%253Dinfo%253Apmid%252F16489235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzIyMTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.31" data-doi="10.1534/genetics.110.114074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kugelberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofoid</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The tandem inversion duplication in <em>Salmonella enterica</em>: selection drives unstable precursors to final mutation types.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114074%26rft_id%253Dinfo%253Apmid%252F20215473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTg1LzEvNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.32" data-doi="10.1038/nature12344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sodergren</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D571%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12344%26rft_id%253Dinfo%253Apmid%252F23873039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23873039&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323625900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.33" data-doi="10.1371/journal.pgen.1002651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeated, selection-driven genome reduction of accessory genes in experimental populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DM.%2BC.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002651%26rft.epage%253De1002651%26rft.atitle%253DRepeated%252C%2Bselection-driven%2Bgenome%2Breduction%2Bof%2Baccessory%2Bgenes%2Bin%2Bexperimental%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002651%26rft_id%253Dinfo%253Apmid%252F22589730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589730&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.34" data-doi="10.1534/genetics.112.148502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marx</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Synchronous waves of failed soft sweeps in the laboratory: remarkably rampant clonal interference of alleles at a single locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148502%26rft_id%253Dinfo%253Apmid%252F23307898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvOTQzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.35" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy8zOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Befort</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Design and use of multiplexed chemostat arrays.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">Feb 23</span>: <span class="cit-fpage">e50262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D23%26rft.spage%253De50262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.38" data-doi="10.1534/genetics.110.121400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutational effects and population dynamics during viral adaptation challenge current models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121400%26rft_id%253Dinfo%253Apmid%252F21041559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Notley-McRobb</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferenci</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Experimental analysis of molecular events during mutational periodic selections in bacterial evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNotley-McRobb%26rft.auinit1%253DL.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D1493%26rft.epage%253D1501%26rft.atitle%253DExperimental%2BAnalysis%2Bof%2BMolecular%2BEvents%2BDuring%2BMutational%2BPeriodic%2BSelections%2Bin%2BBacterial%2BEvolution%26rft_id%253Dinfo%253Apmid%252F11102352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzE0OTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.40" data-doi="10.1016/j.devcel.2013.04.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pashkova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gakhar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winistorfer</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunshine</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The yeast Alix homolog Bro1 functions as a ubiquitin receptor for protein sorting into multivesicular endosomes.</span> <abbr class="cit-jnl-abbrev">Dev. Cell.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell.%26rft.volume%253D25%26rft.spage%253D520%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2013.04.007%26rft_id%253Dinfo%253Apmid%252F23726974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2013.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23726974&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.41" data-doi="10.1371/journal.pgen.1000175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segmental duplications arise from Pol32-dependent repair of broken forks through two alternative replication-based mechanisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPayen%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000175%26rft.epage%253De1000175%26rft.atitle%253DSegmental%2Bduplications%2Barise%2Bfrom%2BPol32-dependent%2Brepair%2Bof%2Bbroken%2Bforks%2Bthrough%2Btwo%2Balternative%2Breplication-based%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000175%26rft_id%253Dinfo%253Apmid%252F18773114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18773114&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryde</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> telomeres. A review.</span> <abbr class="cit-jnl-abbrev">Biochemistry (Mosc)</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1232</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry.%2BBiokhimii%25EF%25B8%25A0a%25EF%25B8%25A1%26rft.stitle%253DBiochemistry%2B%2528Mosc%2529%26rft.aulast%253DPryde%26rft.auinit1%253DF.%2BE.%26rft.volume%253D62%26rft.issue%253D11%26rft.spage%253D1232%26rft.epage%253D1241%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2Btelomeres.%2BA%2Breview.%26rft_id%253Dinfo%253Apmid%252F9467847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9467847&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.43" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonti</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Role of gene duplications in the adaptation of <em>Salmonella typhimurium</em> to growth on limiting carbon sources.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSonti%26rft.auinit1%253DR.%2BV.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D28%26rft.atitle%253DRole%2Bof%2BGene%2BDuplications%2Bin%2Bthe%2BAdaptation%2Bof%2BSalmonella%2Btyphimurium%2Bto%2BGrowth%2Bon%2BLimiting%2BCarbon%2BSources%26rft_id%253Dinfo%253Apmid%252F2680755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIzLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.45" data-doi="10.1371/journal.pgen.1000671"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoebel</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hokamp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Last</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Compensatory evolution of gene regulation in response to stress by <em>Escherichia coli</em> lacking RpoS.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DStoebel%26rft.auinit1%253DD.%2BM.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000671%26rft.epage%253De1000671%26rft.atitle%253DCompensatory%2Bevolution%2Bof%2Bgene%2Bregulation%2Bin%2Bresponse%2Bto%2Bstress%2Bby%2BEscherichia%2Bcoli%2Blacking%2BRpoS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000671%26rft_id%253Dinfo%253Apmid%252F19798444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19798444&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.46" data-doi="10.1038/ng1515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tapscott</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Widespread and nonrandom distribution of DNA palindromes in cancer cells provides a structural platform for subsequent gene amplification.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTanaka%26rft.auinit1%253DH.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D320%26rft.epage%253D327%26rft.atitle%253DWidespread%2Band%2Bnonrandom%2Bdistribution%2Bof%2BDNA%2Bpalindromes%2Bin%2Bcancer%2Bcells%2Bprovides%2Ba%2Bstructural%2Bplatform%2Bfor%2Bsubsequent%2Bgene%2Bamplification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1515%26rft_id%253Dinfo%253Apmid%252F15711546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711546&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.47" data-doi="10.1016/j.cell.2012.11.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene copy-number alterations: a cost-benefit analysis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.11.043%26rft_id%253Dinfo%253Apmid%252F23374337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.11.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23374337&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314362800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.48" data-doi="10.1126/science.1212986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Verdugo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The molecular diversity of adaptive convergence.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTenaillon%26rft.auinit1%253DO.%26rft.volume%253D335%26rft.issue%253D6067%26rft.spage%253D457%26rft.epage%253D461%26rft.atitle%253DThe%2BMolecular%2BDiversity%2Bof%2BAdaptive%2BConvergence%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1212986%26rft_id%253Dinfo%253Apmid%252F22282810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA2Ny80NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.49" data-doi="10.1007/10_2007_057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Maris</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuyper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Laat</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dijken</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Development of efficient xylose fermentation in <em>Saccharomyces cerevisiae</em>: xylose isomerase as a key component.</span> <abbr class="cit-jnl-abbrev">Adv. Biochem. Eng. Biotechnol.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bbiochemical%2Bengineering%252Fbiotechnology%26rft.stitle%253DAdv%2BBiochem%2BEng%2BBiotechnol%26rft.aulast%253Dvan%2BMaris%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D108%26rft.spage%253D179%26rft.epage%253D204%26rft.atitle%253DDevelopment%2Bof%2Befficient%2Bxylose%2Bfermentation%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bxylose%2Bisomerase%2Bas%2Ba%2Bkey%2Bcomponent.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F10_2007_057%26rft_id%253Dinfo%253Apmid%252F17846724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/10_2007_057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846724&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250578300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Models of experimental evolution: the role of genetic chance and selective necessity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1437</span>–<span class="cit-lpage">1448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWahl%26rft.auinit1%253DL.%2BM.%26rft.volume%253D156%26rft.issue%253D3%26rft.spage%253D1437%26rft.epage%253D1448%26rft.atitle%253DModels%2Bof%2BExperimental%2BEvolution%253A%2BThe%2BRole%2Bof%2BGenetic%2BChance%2Band%2BSelective%2BNecessity%26rft_id%253Dinfo%253Apmid%252F11063715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni8zLzE0MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.51" data-doi="10.1073/pnas.0602917103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winkworth</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Tests of parallel molecular evolution in a long-term experiment with <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">9107</span>–<span class="cit-lpage">9112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0602917103%26rft_id%253Dinfo%253Apmid%252F16751270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzI0LzkxMDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvMzk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.52" data-doi="10.1073/pnas.1211150109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manor</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbst</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal duplication is a transient evolutionary solution to stress.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">21010</span>–<span class="cit-lpage">21015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211150109%26rft_id%253Dinfo%253Apmid%252F23197825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzUxLzIxMDEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.53" data-doi="10.1111/j.1574-6968.1999.tb13689.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferenci</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">OmpF changes and the complexity of <em>Escherichia coli</em> adaptation to prolonged lactose limitation.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiology%2BLetters%26rft.stitle%253DFEMS%2BMicrobiol%2BLett%26rft.aulast%253DZhang%26rft.auinit1%253DE.%26rft.volume%253D176%26rft.issue%253D2%26rft.spage%253D395%26rft.epage%253D401%26rft.atitle%253DOmpF%2Bchanges%2Band%2Bthe%2Bcomplexity%2Bof%2BEscherichia%2Bcoli%2Badaptation%2Bto%2Bprolonged%2Blactose%2Blimitation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.1999.tb13689.x%26rft_id%253Dinfo%253Apmid%252F10427722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.1999.tb13689.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10427722&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F399.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081453200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.399.54" data-doi="10.1073/pnas.0404397101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khodursky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dykhuizen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evolutionary genomics of ecological specialization.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">11719</span>–<span class="cit-lpage">11724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404397101%26rft_id%253Dinfo%253Apmid%252F15289609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMyLzExNzE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzM5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/399.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/389" title="The Reference Genome Sequence of Saccharomyces cerevisiae: Then and Now" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/411" title="Metabolic and Environmental Conditions Determine Nuclear Genomic Instability in Budding Yeast Lacking Mitochondrial DNA" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/399" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NxtssUo88txnC5pUmYeE57whb528pXQuz3xItxJtRQc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/399" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/399" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/399" class="active" data-icon-position="" data-hide-link-title="0">The Dynamics of Diverse Segmental Amplifications in Populations of Saccharomyces cerevisiae Adapting to Strong Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459815" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-On05VVEFz2CPN26A5nqp-DMtzF4fAYqx241iYn3fBko" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/399.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/399" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-waUUiICm_o1Ug3tbBzjqcMHrWWI8AdTroTSWyJmO2t4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/399" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/399.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459815" id="node459815--2" data-pisa="ggg;4/3/399" data-pisa-master="ggg;g3.113.009365" data-apath="/ggg/4/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Dynamics of Diverse Segmental Amplifications in Populations of <em>Saccharomyces cerevisiae</em> Adapting to Strong Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Celia Payen</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">Giang T. Ong</span>, <span class="highwire-citation-author" data-delta="3">Jamie L. Pogachar</span>, <span class="highwire-citation-author" data-delta="4">Joseph C. Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Anna B. Sunshine</span>, <span class="highwire-citation-author" data-delta="6">M. K. Raghuraman</span>, <span class="highwire-citation-author" data-delta="7">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="8">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.113.009365" data-atom-uri="/ggg/4/3/399.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459815--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Giang T. Ong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giang%2BT.%2BOng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ong%20GT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiang%2BT.%2BOng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jamie L. Pogachar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BL.%2BPogachar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogachar%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJamie%2BL.%2BPogachar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph C. Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BC.%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna B. Sunshine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB.%2BSunshine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sunshine%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB.%2BSunshine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/399" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/399&amp;text=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/399&amp;t=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/399&amp;title=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459815/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459815" id="node459815--3" data-pisa="ggg;4/3/399" data-pisa-master="ggg;g3.113.009365" data-apath="/ggg/4/3/399.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Dynamics of Diverse Segmental Amplifications in Populations of <em>Saccharomyces cerevisiae</em> Adapting to Strong Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Celia Payen</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">Giang T. Ong</span>, <span class="highwire-citation-author" data-delta="3">Jamie L. Pogachar</span>, <span class="highwire-citation-author" data-delta="4">Joseph C. Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Anna B. Sunshine</span>, <span class="highwire-citation-author" data-delta="6">M. K. Raghuraman</span>, <span class="highwire-citation-author" data-delta="7">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="8">Maitreya J. Dunham</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">399-409;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009365 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access highwire-citation-access-check" data-pisa-id="ggg;g3.113.009365" data-atom-uri="/ggg/4/3/399.atom" data-request-view="full"></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459815--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Celia Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Giang T. Ong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giang%2BT.%2BOng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ong%20GT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiang%2BT.%2BOng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jamie L. Pogachar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jamie%2BL.%2BPogachar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pogachar%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJamie%2BL.%2BPogachar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph C. Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BSanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanchez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BC.%2BSanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anna B. Sunshine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BB.%2BSunshine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sunshine%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnna%2BB.%2BSunshine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Maitreya J. Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maitreya%2BJ.%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaitreya%2BJ.%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:maitreya@uw.edu" class="" data-icon-position="" data-hide-link-title="0">maitreya@uw.edu</a>
<a href="mailto:bbrewer@gs.washington.edu" class="" data-icon-position="" data-hide-link-title="0">bbrewer@gs.washington.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=399&amp;rft.epage=409&amp;rft.atitle=The%20Dynamics%20of%20Diverse%20Segmental%20Amplifications%20in%20Populations%20of%20Saccharomyces%20cerevisiae%20Adapting%20to%20Strong%20Selection&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Payen%2C+Celia&amp;rft.au=Di+Rienzi%2C+Sara+C.&amp;rft.au=Ong%2C+Giang+T.&amp;rft.au=Pogachar%2C+Jamie+L.&amp;rft.au=Sanchez%2C+Joseph+C.&amp;rft.au=Sunshine%2C+Anna+B.&amp;rft.au=Raghuraman%2C+M.+K.&amp;rft.au=Brewer%2C+Bonita+J.&amp;rft.au=Dunham%2C+Maitreya+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459815/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459815/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459815/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459815/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459815/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459815/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459815/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459815/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459815/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459815/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459815/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459815/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link" >
  
        <h2 class="pane-title"><span class='panels-ajax-pane-title' data-pid='new-37'></span></h2>
    
  
  <div class="pane-content">
    <div class='panels-ajax-pane panels-ajax-pane-new-37' data-pid='new-37'></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/399" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/399.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/399.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/399.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/399.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
