<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/509" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/509.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/509.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/509.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/3/509/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/509.atom" />
<meta name="HW.pisa" content="ggg;4/3/509" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus" />
<meta name="DC.Identifier" content="10.1534/g3.113.008441" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Modliszewski and Willis. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic variation is the fundamental medium of evolution. In allopolyploids, which are the product of hybridization and whole genome duplication, if homologous chromosomes always pair, then all descendants of a single diploid F1 hybrid lineage will be genetically identical. Contrarily, genetic variation among initially isogenic lineages is augmented when homeologous chromosomes pair; this added variation may contribute to phenotypic evolution. Mimulus sookensis is a naturally occurring, small-flowered allotetraploid derived from the large-flowered Mimulus guttatus and small-flowered Mimulus nasutus . Because diploid F1 hybrids between M. guttatus and M. nasutus have large flowers, phenotypic evolution post-polyploidization is implied in M. sookensis . Here, we present genetic and phenotypic analyses of synthetic neoallotetraploid Mimulus derived from a cross between M. guttatus and M. nasutus . Genetic marker data from S2 and BC1N progeny suggest that chromosomes regularly pair with their homologous counterpart. By measuring the phenotype of synthetic neoallotetraploids, we demonstrate that polyploidization per se does not induce the small flowers of M. sookensis . Moreover, phenotypic measurements of synthetic allotetraploid F2s and S4 families suggest that rare homeologous recombination events have a negligible phenotypic effect in the first few generations. In total, the results are consistent with either exceedingly rare homeologous pairing and recombination or spontaneous fragment loss. The low levels of fragment loss and phenotypic variation in neoallotetraploids suggest that homeologous recombination after polyploidization is not a major mechanism of phenotypic evolution in M. sookensis . Rather, it may be that spontaneous mutations or epigenetic changes after allopolyploidization have driven phenotypic evolution in M. sookensis." />
<meta name="DC.Contributor" content="Jennifer L. Modliszewski" />
<meta name="DC.Contributor" content="John H. Willis" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic variation is the fundamental medium of evolution. In allopolyploids, which are the product of hybridization and whole genome duplication, if homologous chromosomes always pair, then all descendants of a single diploid F&lt;sub&gt;1&lt;/sub&gt; hybrid lineage will be genetically identical. Contrarily, genetic variation among initially isogenic lineages is augmented when homeologous chromosomes pair; this added variation may contribute to phenotypic evolution. &lt;i&gt;Mimulus sookensis&lt;/i&gt; is a naturally occurring, small-flowered allotetraploid derived from the large-flowered &lt;i&gt;Mimulus guttatus&lt;/i&gt; and small-flowered &lt;i&gt;Mimulus nasutus&lt;/i&gt;. Because diploid F&lt;sub&gt;1&lt;/sub&gt; hybrids between &lt;i&gt;M. guttatus&lt;/i&gt; and &lt;i&gt;M. nasutus&lt;/i&gt; have large flowers, phenotypic evolution post-polyploidization is implied in &lt;i&gt;M. sookensis&lt;/i&gt;. Here, we present genetic and phenotypic analyses of synthetic neoallotetraploid &lt;i&gt;Mimulus&lt;/i&gt; derived from a cross between &lt;i&gt;M. guttatus&lt;/i&gt; and &lt;i&gt;M. nasutus&lt;/i&gt;. Genetic marker data from S&lt;sub&gt;2&lt;/sub&gt; and BC&lt;sub&gt;1N&lt;/sub&gt; progeny suggest that chromosomes regularly pair with their homologous counterpart. By measuring the phenotype of synthetic neoallotetraploids, we demonstrate that polyploidization &lt;i&gt;per se&lt;/i&gt; does not induce the small flowers of &lt;i&gt;M. sookensis&lt;/i&gt;. Moreover, phenotypic measurements of synthetic allotetraploid F&lt;sub&gt;2&lt;/sub&gt;s and S&lt;sub&gt;4&lt;/sub&gt; families suggest that rare homeologous recombination events have a negligible phenotypic effect in the first few generations. In total, the results are consistent with either exceedingly rare homeologous pairing and recombination or spontaneous fragment loss. The low levels of fragment loss and phenotypic variation in neoallotetraploids suggest that homeologous recombination after polyploidization is not a major mechanism of phenotypic evolution in &lt;i&gt;M. sookensis&lt;/i&gt;. Rather, it may be that spontaneous mutations or epigenetic changes after allopolyploidization have driven phenotypic evolution in &lt;i&gt;M. sookensis.&lt;/i&gt;&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/509" />
<meta name="citation_id" content="4/3/509" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/509" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/509.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/509.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/509.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008441" />
<meta name="citation_pmid" content="24470218" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="509" />
<meta name="citation_lastpage" content="522" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jennifer L. Modliszewski" />
<meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina 27708" />
<meta name="citation_author_email" content="jenmod@live.unc.edu" />
<meta name="citation_author" content="John H. Willis" />
<meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina 27708" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=M. L. Ainouche;citation_author=A. Baumel;citation_author=A. Salmon;citation_title=Spartina anglica C. E. Hubbard: A natural model system for analysing early evolutionary changes that affect allopolyploid genomes.;citation_pages=475-484;citation_volume=82;citation_year=2004;citation_doi=10.1111/j.1095-8312.2004.00334.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=S. Anssour;citation_author=T. Krügel;citation_author=T. F. Sharbel;citation_author=H. P. Saluz;citation_author=G. Bonaventure;citation_title=Phenotypic, genetic and genomic consequences of natural and synthetic polyploidization of Nicotiana attenuata and Nicotiana obtusifolia.;citation_volume=103;citation_year=2009;citation_pmid=19307190;citation_doi=10.1093/aob/mcp058" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=A. Baumel;citation_author=M. Ainouche;citation_author=R. Kalendar;citation_author=A. H. Schulman;citation_title=Retrotransposons and Genomic Stability in Populations of the Young Allopolyploid Species Spartina anglica C.E. Hubbard (Poaceae);citation_pages=1218-1227;citation_volume=19;citation_year=2002;citation_issue=8;citation_pmid=12140233;citation_doi=10.1093/oxfordjournals.molbev.a004182" />
<meta name="citation_reference" content="citation_journal_title=Madrono;citation_author=B. G. Benedict;citation_author=J. L. Modliszewski;citation_author=A. L. Sweigart;citation_author=N. H. Martin;citation_author=F. R. Ganders;citation_title=Mimulus sookensis (Phrymaceae), a new allotetraploid species derived from Mimulus guttatus and Mimulus nasutus.;citation_pages=29-43;citation_volume=59;citation_year=2012;citation_doi=10.3120/0024-9637-59.1.29" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=. Beardsley;citation_author=P. S. E. Schoenig;citation_author=J. B. Whittall;citation_author=R. G. Olmstead;citation_title=Patterns of evolution in western North American Mimulus (Phrymaceae);citation_pages=474-489;citation_volume=91;citation_year=2004;citation_issue=3;citation_pmid=21653403;citation_doi=10.3732/ajb.91.3.474" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=R. J. Bino;citation_author=J. N. De Vries;citation_author=H. L. Kraak;citation_author=J. G. Van Pijlen;citation_title=Flow Cytometric Determination of Nuclear Replication Stages in Tomato Seeds during Priming and Germination;citation_pages=231-236;citation_volume=69;citation_year=1992;citation_issue=3;citation_doi=10.1093/aob/" />
<meta name="citation_reference" content="citation_journal_title=Bot. Gaz.;citation_author=A. F. Blakeslee;citation_author=J. Belling;citation_author=M. E. Farnham;citation_title=Inheritance in tetraploid daturas.;citation_pages=329-373;citation_volume=76;citation_year=1923" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Blanc;citation_author=K. H. Wolfe;citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.;citation_volume=16;citation_year=2004;citation_pmid=15208399;citation_doi=10.1105/tpc.021345" />
<meta name="citation_reference" content="citation_title=Discussions in Cytogenetics;citation_author=C. R. Burnham;citation_year=1962" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Chaudhary;citation_author=L. Flagel;citation_author=R. M. Stupar;citation_author=J. A. Udall;citation_author=N. Verma;citation_title=Reciprocal silencing, transcriptional bias and functional divergence of homeologs in polyploid cotton (Gossypium).;citation_volume=182;citation_year=2009;citation_pmid=19363125;citation_doi=10.1534/genetics.109.102608" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=Z. J. Chen;citation_title=Genetic and epigenetic mechanisms for gene expression and phenotypic variation in plant polyploids.;citation_pages=377-406;citation_volume=58;citation_year=2007;citation_pmid=17280525;citation_doi=10.1146/annurev.arplant.58.032806.103835" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Chester;citation_author=J. P. Gallagher;citation_author=V. V. Symonds;citation_author=A. V. Cruz da Silva;citation_author=E. V. Mavrodiev;citation_title=Extensive chromosomal variation in a recently formed natural allopolyploid species, Tragopogon miscellus (asteraceae).;citation_volume=109;citation_year=2012;citation_pmid=22228301;citation_doi=10.1073/pnas.1112041109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=L. Comai;citation_author=A. P. Tyagi;citation_author=K. Winter;citation_author=R. Holmes-Davis;citation_author=S. H. Reynolds;citation_title=Phenotypic Instability and Rapid Gene Silencing in Newly Formed Arabidopsis Allotetraploids;citation_pages=1551-1568;citation_volume=12;citation_year=2000;citation_issue=9;citation_pmid=11006331;citation_doi=10.1105/tpc.12.9.1551" />
<meta name="citation_reference" content="citation_title=Speciation;citation_author=J. Coyne;citation_author=H. A. Orr;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. B. Crane;citation_author=C. D. Darlington;citation_title=Chromatid segregation in tetraploid Rubus.;citation_pages=869;citation_volume=129;citation_year=1932" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Cui;citation_author=P. K. Wall;citation_author=J. H. Leebens-Mack;citation_author=B. G. Lindsay;citation_author=D. E. Soltis;citation_title=Widespread genome duplications throughout the history of flowering plants.;citation_volume=16;citation_year=2006;citation_pmid=16702410;citation_doi=10.1101/gr.4825606" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Curole;citation_author=D. Hedgecock;citation_title=Estimation of preferential pairing rates in second-generation autotetraploid pacific oysters (Crassostrea gigas).;citation_volume=171;citation_year=2005;citation_pmid=15965239;citation_doi=10.1534/genetics.105.043042" />
<meta name="citation_reference" content="citation_journal_title=Bot. Gaz.;citation_author=H. DeVries;citation_title=Oenothera gigas nanella, a mendelian mutant.;citation_pages=337-345;citation_volume=60;citation_year=1915" />
<meta name="citation_reference" content="citation_title=Genetics and the Origin of Species;citation_author=T. Dobzhansky;citation_year=1937" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. G. Doyle;citation_title=Autotetraploid gene segregation.;citation_pages=139-146;citation_volume=43;citation_year=1973;citation_pmid=24424975;citation_doi=10.1007/BF00306563" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. J. Doyle;citation_author=L. E. Flagel;citation_author=A. H. Paterson;citation_author=R. A. Rapp;citation_author=D. E. Soltis;citation_title=Evolutionary genetics of genome merger and doubling in plants.;citation_pages=443-461;citation_volume=42;citation_year=2008;citation_pmid=18983261;citation_doi=10.1146/annurev.genet.42.110807.091524" />
<meta name="citation_reference" content="citation_journal_title=Ann. Hum. Genet.;citation_author=R. A. Fisher;citation_title=The theoretical consequences of polyploid inheritance for the mid style form of Lythrum salicaria.;citation_pages=31-38;citation_volume=11;citation_year=1941" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. Fishman;citation_author=J. H. Willis;citation_title=Evidence for Dobzhansky-Muller incompatibilites contributing to the sterility of hybrids between Mimulus guttatus and M. nasutus.;citation_pages=1932-1942;citation_volume=55;citation_year=2001;citation_issue=10;citation_pmid=11761055;citation_doi=10.1554/0014-3820(2001)055[1932:EFDMIC]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. Fishman;citation_author=A. J. Kelly;citation_author=J. H. Willis;citation_title=Minor quantitative trait loci underlie floral traits associated with mating system divergence in Mimulus.;citation_pages=2138-2155;citation_volume=56;citation_year=2002;citation_issue=11;citation_pmid=12487345;citation_doi=10.1554/0014-3820(2002)056[2138:MQTLUF]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=L. Flagel;citation_author=J. Udall;citation_author=D. Nettleton;citation_author=J. Wendel;citation_title=Duplicate gene expression in allopolyploid Gossypium reveals two temporally distinct phases of expression evolution.;citation_pages=16-16;citation_volume=6;citation_year=2008;citation_pmid=18416842;citation_doi=10.1186/1741-7007-6-16" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. T. Gaeta;citation_author=J. C. Pires;citation_author=F. Iniguez-Luy;citation_author=E. Leon;citation_author=T. C. Osborn;citation_title=Genomic changes in resynthesized Brassica napus and their effect on gene expression and phenotype.;citation_volume=19;citation_year=2007;citation_pmid=18024568;citation_doi=10.1105/tpc.107.054346" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. U. Gerstel;citation_title=SEGREGATION IN NEW ALLOPOLYPLOIDS OF NICOTIANA. I. COMPARISON OF 6x (N. TABACUM x TOMENTOSIFORMIS) AND 6x (N. TABACUM x OTOPHORA);citation_pages=1723-1734;citation_volume=45;citation_year=1960;citation_issue=12;citation_pmid=17248031" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. U. Gerstel;citation_author=L. L. Phillips;citation_title=SEGREGATION IN NEW ALLOPOLYPLOIDS OF GOSSYPIUM. II. TETRAPLOID COMBINATIONS;citation_pages=783-797;citation_volume=42;citation_year=1957;citation_issue=6;citation_pmid=17247734" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=D. U. Gerstel;citation_author=L. L. Phillips;citation_title=Segregation of Synthetic Amphiploids in Gossypium and Nicotiana;citation_pages=225-237;citation_volume=23;citation_year=1958;citation_issue=0;citation_pmid=13635558;citation_doi=10.1101/SQB.1958.023.01.025" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=R. P. Gregory;citation_title=On the Genetics of Tetraploid Plants in Primula sinensis;citation_pages=484-492;citation_volume=87;citation_year=1914;citation_issue=597;citation_doi=10.1098/rspb.1914.0035" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=Theoretical genetics of autopolyploids.;citation_pages=359-372;citation_volume=22;citation_year=1930;citation_doi=10.1007/BF02984197" />
<meta name="citation_reference" content="citation_title=Techniques for Pollination Biologists;citation_author=C. A. Kearns;citation_author=D. W. Inouye;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. J. Kelly;citation_author=J. H. Willis;citation_title=Polymorphic microsatellite loci in Mimulus guttatus and related species.;citation_pages=769-774;citation_volume=7;citation_year=1998;citation_doi=10.1046/j.1365-294x.1998.00328.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. Y. Lim;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=J. Tate;citation_author=R. Matyasek;citation_title=Rapid chromosome evolution in recently formed polyploids in Tragopogon (Asteraceae).;citation_pages=e3353-e3353;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18843372;citation_doi=10.1371/journal.pone.0003353" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. W. Lindstrom;citation_author=L. M. Humphrey;citation_title=COMPARATIVE CYTO-GENETIC STUDIES OF TETRAPLOID TOMATOES FROM DIFFERENT ORIGINS;citation_pages=193-209;citation_volume=18;citation_year=1933;citation_issue=3;citation_pmid=17246688" />
<meta name="citation_reference" content="citation_journal_title=Bot. Rev.;citation_author=T. M. Little;citation_title=Gene segregation in autotetraploids.;citation_pages=60-85;citation_volume=11;citation_year=1945;citation_doi=10.1007/BF02861164" />
<meta name="citation_reference" content="citation_journal_title=Bot. Rev.;citation_author=T. M. Little;citation_title=Gene segregation in autotetraploids. II.;citation_pages=318-339;citation_volume=24;citation_year=1958;citation_doi=10.1007/BF02872438" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=B. Liu;citation_author=C. L. Brubaker;citation_author=G. Mergeai;citation_author=R. C. Cronn;citation_author=J. F. Wendel;citation_title=Polyploid formation in cotton is not accompanied by rapid genomic changes.;citation_pages=321-330;citation_volume=44;citation_year=2001;citation_issue=3;citation_pmid=11444689" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. Madlung;citation_author=A. P. Tyagi;citation_author=B. Watson;citation_author=H. Jiang;citation_author=T. Kagochi;citation_title=Genomic changes in synthetic Arabidopsis polyploids.;citation_pages=221-230;citation_volume=41;citation_year=2005;citation_issue=2;citation_pmid=15634199;citation_doi=10.1111/j.1365-313X.2004.02297.x" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=N. H. Martin;citation_author=J. H. Willis;citation_title=Geographical variation in postzygotic isolation and its genetic basis within and between two Mimulus species;citation_pages=2469-2478;citation_volume=365;citation_year=2010;citation_issue=1552;citation_pmid=20643736;citation_doi=10.1098/rstb.2010.0030" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Masterson;citation_title=Stomatal Size in Fossil Plants: Evidence for Polyploidy in Majority of Angiosperms;citation_pages=421-424;citation_volume=264;citation_year=1994;citation_issue=5157;citation_pmid=17836906;citation_doi=10.1126/science.264.5157.421;citation_georef=1994028739" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Reductional and equational separation of the chromosomes in bivalents and multivalents.;citation_pages=53-78;citation_volume=30;citation_year=1935;citation_doi=10.1007/BF02982205" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=K. Mather;citation_title=Segregation and linkage in autotetraploids.;citation_pages=287-314;citation_volume=32;citation_year=1936;citation_doi=10.1007/BF02982683" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. Matsushita;citation_author=A. P. Tyagi;citation_author=G. M. Thornton;citation_author=J. C. Pires;citation_author=A. Madlung;citation_title=Allopolyploidization lays the foundation for evolution of distinct populations: Evidence from analysis of synthetic Arabidopsis allohexaploids.;citation_volume=191;citation_year=2012;citation_pmid=22426881;citation_doi=10.1534/genetics.112.139295" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. L. Modliszewski;citation_author=J. H. Willis;citation_title=Allotetraploid Mimulus sookensis are highly interfertile despite independent origins.;citation_pages=5280-5298;citation_volume=21;citation_year=2012;citation_issue=21;citation_pmid=22831280;citation_doi=10.1111/j.1365-294X.2012.05706.x" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=H. J. Muller;citation_title=A new mode of segregation in Gregory’s tetraploid Primulas.;citation_pages=508-512;citation_volume=48;citation_year=1914;citation_doi=10.1086/279426" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=B. P. Oswald;citation_author=S. L. Nuismer;citation_title=Neopolyploidy and diversification in Heuchera grossulariifolia.;citation_pages=1667-1679;citation_volume=65;citation_year=2011;citation_issue=6;citation_pmid=21143472;citation_doi=10.1111/j.1558-5646.2010.01208.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=S. P. Otto;citation_author=J. Whitton;citation_title=POLYPLOID INCIDENCE AND EVOLUTION;citation_pages=401-437;citation_volume=34;citation_year=2000;citation_issue=1;citation_pmid=11092833;citation_doi=10.1146/annurev.genet.34.1.401" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=H. Ozkan;citation_author=A. A. Levy;citation_author=M. Feldman;citation_title=Allopolyploidy-Induced Rapid Genome Evolution in the Wheat (Aegilops-Triticum) Group;citation_pages=1735-1747;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487689;citation_doi=10.1105/tpc.13.8.1735" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Syst.;citation_author=J. Ramsey;citation_author=D. W. Schemske;citation_title=Neopolyploidy in flowering plants.;citation_volume=33;citation_year=2002;citation_doi=10.1146/annurev.ecolsys.33.010802.150437" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. A. Rapp;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Genomic expression dominance in allopolyploids.;citation_pages=18;citation_volume=7;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Salmon;citation_author=M. L. Ainouche;citation_author=J. F. Wendel;citation_title=Genetic and epigenetic consequences of recent hybridization and polyploidy in Spartina (Poaceae).;citation_pages=1163-1175;citation_volume=14;citation_year=2005;citation_issue=4;citation_pmid=15773943;citation_doi=10.1111/j.1365-294X.2005.02488.x" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. Salmon;citation_author=L. Flagel;citation_author=B. Ying;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Homoeologous nonreciprocal recombination in polyploid cotton.;citation_pages=123-134;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19925554;citation_doi=10.1111/j.1469-8137.2009.03093.x" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=F. W. Sansome;citation_title=Chromatid segregation in Solanum lycopersicum.;citation_pages=105-132;citation_volume=27;citation_year=1933;citation_doi=10.1007/BF02984385" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. E. Schranz;citation_author=T. C. Osborn;citation_title=Novel flowering time variation in the resynthesized polyploid Brassica napus;citation_pages=242-246;citation_volume=91;citation_year=2000;citation_issue=3;citation_pmid=10833052;citation_doi=10.1093/jhered/91.3.242" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=M. E. Schranz;citation_author=T. C. Osborn;citation_title=De novo variation in life-history traits and responses to growth conditions of resynthesized polyploid Brassica napus (Brassicaceae);citation_pages=174-183;citation_volume=91;citation_year=2004;citation_issue=2;citation_pmid=21653373;citation_doi=10.3732/ajb.91.2.174" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=H. Shaked;citation_author=K. Kashkush;citation_author=H. Ozkan;citation_author=M. Feldman;citation_author=A. A. Levy;citation_title=Sequence Elimination and Cytosine Methylation Are Rapid and Reproducible Responses of the Genome to Wide Hybridization and Allopolyploidy in Wheat;citation_pages=1749-1759;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487690;citation_doi=10.1105/tpc.13.8.1749" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=K. Skalická;citation_author=K. Y. Lim;citation_author=R. Matyasek;citation_author=M. Matzke;citation_author=A. R. Leitch;citation_title=Preferential elimination of repeated DNA sequences from the paternal, Nicotiana tomentosiformis genome donor of a synthetic, allotetraploid tobacco.;citation_pages=291-303;citation_volume=166;citation_year=2005;citation_issue=1;citation_pmid=15760371;citation_doi=10.1111/j.1469-8137.2004.01297.x" />
<meta name="citation_reference" content="citation_title=Biometry;citation_author=R. R. Sokal;citation_author=F. J. Rohlf;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=J. A. Tate;citation_title=Advances in the study of polyploidy since plant speciation.;citation_pages=173-191;citation_volume=161;citation_year=2003;citation_doi=10.1046/j.1469-8137.2003.00948.x" />
<meta name="citation_reference" content="citation_title=Chromosomal Evolution in Higher Plants;citation_author=G. L. Stebbins;citation_year=1971" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Stift;citation_author=C. Berenos;citation_author=P. Kuperus;citation_author=P. H. van Tienderen;citation_title=Segregation models for disomic, tetrasomic, and intermediate inheritance in tetraploids: a general procedure applied to Rorippa (yellow cress) microsatellite data.;citation_volume=179;citation_year=2008;citation_pmid=18689891;citation_doi=10.1534/genetics.107.085027" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. L. Sweigart;citation_author=N. H. Martin;citation_author=J. H. Willis;citation_title=Patterns of nucleotide variation and reproductive isolation between a Mimulus allotetraploid and its progenitor species.;citation_pages=2089-2100;citation_volume=17;citation_year=2008;citation_issue=8;citation_pmid=18312552" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Sybenga;citation_title=Preferential pairing estimates from multivalent frequencies in tetraploids.;citation_pages=1045-1055;citation_volume=37;citation_year=1994;citation_issue=6;citation_pmid=18470143" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=E. Szadkowski;citation_author=F. Eber;citation_author=V. Huteau;citation_author=M. Lodé;citation_author=C. Huneau;citation_title=The first meiosis of resynthesized Brassica napus, a genome blender.;citation_pages=102-112;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=20149113;citation_doi=10.1111/j.1469-8137.2010.03182.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wang;citation_author=L. Tian;citation_author=A. Madlung;citation_author=H.-S. Lee;citation_author=M. Chen;citation_title=Stochastic and Epigenetic Changes of Gene Expression in Arabidopsis Polyploids;citation_pages=1961-1973;citation_volume=167;citation_year=2004;citation_issue=4;citation_pmid=15342533;citation_doi=10.1534/genetics.104.027896" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wang;citation_author=L. Tian;citation_author=H.-S. Lee;citation_author=N. E. Wei;citation_author=H. Jiang;citation_title=Genomewide nonadditive gene regulation in Arabidopsis allotetraploids.;citation_volume=172;citation_year=2006;citation_pmid=16172500;citation_doi=10.1534/genetics.105.047894" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Wu;citation_author=M. Gallo-Meagher;citation_author=R. C. Littel;citation_author=Z.-B. Zeng;citation_title=A General Polyploid Model for Analyzing Gene Segregation in Outcrossing Tetraploid Species;citation_pages=869-882;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606559" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Yu;citation_author=G. Haberer;citation_author=M. Matthes;citation_author=T. Rattei;citation_author=K. F. X. Mayer;citation_title=Impact of natural genetic variation on the transcriptome of autotetraploid Arabidopsis thaliana.;citation_volume=107;citation_year=2010;citation_pmid=20876110;citation_doi=10.1073/pnas.1000852107" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Genetic variation is the fundamental medium of evolution. In allopolyploids, which are the product of hybridization and whole genome duplication, if homologous chromosomes always pair, then all descendants of a single diploid F1 hybrid lineage will be genetically identical. Contrarily, genetic variation among initially isogenic lineages is augmented when homeologous chromosomes pair; this added variation may contribute to phenotypic evolution. Mimulus sookensis is a naturally occurring, small-flowered allotetraploid derived from the large-flowered Mimulus guttatus and small-flowered Mimulus nasutus . Because diploid F1 hybrids between M. guttatus and M. nasutus have large flowers, phenotypic evolution post-polyploidization is implied in M. sookensis . Here, we present genetic and phenotypic analyses of synthetic neoallotetraploid Mimulus derived from a cross between M. guttatus and M. nasutus . Genetic marker data from S2 and BC1N progeny suggest that chromosomes regularly pair with their homologous counterpart. By measuring the phenotype of synthetic neoallotetraploids, we demonstrate that polyploidization per se does not induce the small flowers of M. sookensis . Moreover, phenotypic measurements of synthetic allotetraploid F2s and S4 families suggest that rare homeologous recombination events have a negligible phenotypic effect in the first few generations. In total, the results are consistent with either exceedingly rare homeologous pairing and recombination or spontaneous fragment loss. The low levels of fragment loss and phenotypic variation in neoallotetraploids suggest that homeologous recombination after polyploidization is not a major mechanism of phenotypic evolution in M. sookensis . Rather, it may be that spontaneous mutations or epigenetic changes after allopolyploidization have driven phenotypic evolution in M. sookensis." />
<meta name="og-title" property="og:title" content="Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/509" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic variation is the fundamental medium of evolution. In allopolyploids, which are the product of hybridization and whole genome duplication, if homologous chromosomes always pair, then all descendants of a single diploid F1 hybrid lineage will be genetically identical. Contrarily, genetic variation among initially isogenic lineages is augmented when homeologous chromosomes pair; this added variation may contribute to phenotypic evolution. Mimulus sookensis is a naturally occurring, small-flowered allotetraploid derived from the large-flowered Mimulus guttatus and small-flowered Mimulus nasutus . Because diploid F1 hybrids between M. guttatus and M. nasutus have large flowers, phenotypic evolution post-polyploidization is implied in M. sookensis . Here, we present genetic and phenotypic analyses of synthetic neoallotetraploid Mimulus derived from a cross between M. guttatus and M. nasutus . Genetic marker data from S2 and BC1N progeny suggest that chromosomes regularly pair with their homologous counterpart. By measuring the phenotype of synthetic neoallotetraploids, we demonstrate that polyploidization per se does not induce the small flowers of M. sookensis . Moreover, phenotypic measurements of synthetic allotetraploid F2s and S4 families suggest that rare homeologous recombination events have a negligible phenotypic effect in the first few generations. In total, the results are consistent with either exceedingly rare homeologous pairing and recombination or spontaneous fragment loss. The low levels of fragment loss and phenotypic variation in neoallotetraploids suggest that homeologous recombination after polyploidization is not a major mechanism of phenotypic evolution in M. sookensis . Rather, it may be that spontaneous mutations or epigenetic changes after allopolyploidization have driven phenotypic evolution in M. sookensis." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1-5xJEn0LPSbC5tVo_aLouf90bvlLqTv1kzNKCIzftY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459820","apath":"\/ggg\/4\/3\/509.atom","pisa":"ggg;4\/3\/509","ac":{"\/ggg\/4\/3\/509.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/509.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/509"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/509"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/509":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022INBgm9wio2Tq+Kw0\\\\\\\/7Q3834WhBe5KBN+C4qoyY68pFh5NIhavtl2ybb64SqqRNK4mNqY6BqlDL7sgO8WkL4eAUUO9JD\\\\\\\/SrcKMmvu4EZuhB1aJpvkOyA1A6VPR3Sv9lusxf2dhHSC2iwRAcf67ES5274roGxZsHvI8U9lZM8rkI+3JwbVlixzypATPY1x3EyjO4N18gPEc1B953G00\\\\\\\/a7YPJiOcJfLc\\\\\\\/cfnn0hkfsLdndtckYzkW41OBwaCfCLhvg50tZXHGmcIR4kVZqJdCbb+miYGmu8nYEgTnqGA2nfntNpHy7eFOm7wb4ovFWNI\\\\\\\/hRhGi5NJKGUjvZYpdTJAK3h26QDB8Xbyuux9ljKCxxnljFHxTpblPrHcOvnUswGkgDXVR+7cuJIWBEma\\\\\\\/WCgiLDkClFfNndCap8B2f7xbeZQ7tgsEVKd77wxjEKBz1bhaxbOWXO7Xs1uBvjby2v8hDuNnlR2FT1BXM6FtxQnW3+6mPjTSL\\\\\\\/1EWL9nrYQcIWms9ze1bg33WPV5YRF5JuqeTtxjhVq3uARBRIeKu7mYmBuaBd1XhoZUAFBJ5hV\\\\\\\/ZLTQ7qL\\\\\\\/VK3KSvRfRHQ0pv6lmrbO\\\\\\\/CRcVViIaMZrQEqaDYdgADUUhuVgmtjvTJunCP47t8kg3kCgD6ratroWeIsnyiCbdVgvDXVYnFMAeQmTaODJ2zrUyGLHrJyyp33W9E5cmSBTtqMKlt9YxRkBoDbz4KgvZhgMUd1sJ9re3dltd59MK153x5NNBEev4Cm1Lf8Y0iZy6bb0kPycOFKEMnFufI4kFWBh1OY0UR8dWdaDTwkMEfW+hntPD\\\\\\\/glb6GTj5JYUnDXkUW8b8g+QXCG6WJM4rKnUAP7tA6X6vNvfD0X0wOcF9k6n3PcTH1cWz67W7sktpVLD0GlAiW9EcHHI7QQzFvrPGs74+dtcv2KceEwJo21vFDzxLKnjmX5xwdRQvR+hKhg3iVzARGNnqeRgMZK\\\\\\\/s8c3ri5m+++keCl4o\\\\\\\/nFmnGx9cTfDFPM+TWiheyKPSDL+NVv2pTOg8GEYzrNQzbGAp5ugG6Kz96vISQECTq2VIEgeiWu\\\\\\\/j0YxNwxjQv16DTXWKxshW2NnqWZBwItoyexduuXsANeA+bkllFVWWw\\\\\\\/jTUPnJsixMzDQGXzF3o1KkPEfuizmA8eH0xE3G2xCd5d7JVBUGlwlTYmw32CP+BBrwhtuwu5Usykg96Gvxqpg0S1BTwLF71A\\\\\\\/ruFbDcueuomUS8rRW1LYI1dgNmEgX4F45BDJsEnN4rD5EGqqWRNZ5G\\\u0022,\\\u0022iv\\\u0022:\\\u0022GTsqsx25m2WmXz4o7Q42tQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223172aa7d39d9d43987a617adadb1c2398c8e9ebb961a9648990ef7344b946c36\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N8rpq9s3rQYwrxt\\\\\\\/PrruzLf0s7Fw\\\\\\\/2y1mymmx5i5X5JbZHXLm9h9oy5Y2m9OOZx44yB+37UZjhRASqfMnWjsriGuR40go+E\\\\\\\/L38JwTYh1Cc+jKcBie\\\\\\\/PvKckKP3yeh7DeCiJ6tOMvuLerurmSRDitwcIGd8OwkVzrjjrq3fV+ksaX8FMmlgytMaKBX8JJV4KZe+qzwXByQFxEp3Nf86G8XkMmwYzaTNCaYLiFgwCIeHiL\\\\\\\/3\\\\\\\/1fh28jdeLmUfpvKuuRBxIZgx\\\\\\\/TMN71VZhY6kS0en\\\\\\\/3vsPRLO5OftabCnOLaSC9sRiI\\\\\\\/mmxqAolYAaDkiSpeCx8bC09VwiJr70IraHzpvgNAMqxgzZasVc6jeuWMcyN038bZwEPW7sThC5IW4z\\\\\\\/Apa2XqGprwuuFrpiwhVs3pJDwAyMRsK2NhwIyMwLroT5\\\\\\\/QuSef1e2GV0a6Sa5M1o46MEWxPcYnrFQFIeF95KIHQ6nnKYd03XMgg6U4efGlvdY2PEU+eOPpyczJjxxTqPyOk5+C6WFu+PKCNRZLv7AwSiwAbSjhx9FwU9qsTr1MqHazIJXrNAtI\\\\\\\/LbNU7naS0wb2oAjnENitqwTZwwuFrJL7cUZhFpBqhv0rJRlkwoLjsIScHzmRjb5tl82rsAGeDHONwjc+Hn5Ogo\\\\\\\/S\\\\\\\/nywgvpL\\\\\\\/2nsz+YENJJTwcxqv6wUFlU2qI0Lys++WfYvF9OPfCf04mnwe2ip0NR34MTa8hHCF+UXq1mgesoeNWBmqKxTXwss\\\\\\\/J0hcZOC2DUbLyRykUns4+I96DMFozooDZWGfT3PL69M0Bk0N7cS3yZmF\\\\\\\/jARgyUjcGq33nvuE9W4AZ\\\\\\\/g2CPCR4SS5cXl+NLnDdCE7vACbgz8ji8HItVUBp3P9BTlUvxEHB406gyWUDnC0diIFjbOLLVsgwGBZ9EFOdHNpPMdeyX\\\\\\\/Jip7vSBQHLIGbvHXUuxA+9NLhiv4x0xOaNeNSxykryvL1vI0EoznshSzp+0WCSCUnDAlFfp2InEcsN07vG0mx1ztewahmDAPkeNfBJe\\\\\\\/TERyelflMZnerCl5P5QqnBekraBDaIoGzE4ZXACxwdubku7v9MK6HroAsNmA6AxKD9JpUy7kuwG2m9HIKwxx6ajl9tSp0UgEqrvtoFCdou1WZh2PwU\\\\\\\/0qMmEN+M\\\\\\\/NhlfNUvJykt9z4xBPvrnm+Whuf0P+0gHn6imBLcbvZP7VVuDRXZv8jeVHViv\\\\\\\/K14mDMUpMZOTLoEw9+pwy47rmu3vaOPjGMcSVCzQOPK\\\\\\\/Y4a2t1F9UqtRueapyBBP18VTHnRQCxKzxHpgin\\\\\\\/GqGUhoNmDRZMq2z+xABUUN3OC6+4RGfERc4hTeKGsEjz2l9F+D5xSvfypIpFg4E3GMKBkhHHKAwK9aNjOGSsicWp8X1E3vFTb\\\\\\\/rzFbn5MMPtSuR9vWkTve150DKkJu0yxF3ddpdbRUPeLeRXi0yNtbkHl\\\\\\\/m3uiaxyzxD+VNQoTlVhNgdvsdqkAMsv6ymD42RyTG82er0lOLhCfcS3dEhS+mdIXt04StEmgMkFfMh5\\\\\\\/5o6t6Rm+hyG+yspKvOCYsIxQLkr+S6yveXUAgOYIOpRL7vYjriy8P4DAd3nZgol0Ycqe8DNc\\\\\\\/+TE+TZbaokxYncgmfwJSZwbpF7xl9Pjfskgzzuc5aD1LbbYp1ovNow9OuKzDaecSDoWc85pn9tzno6r77POl\\\\\\\/e7RutTwfv6lj9x8v4KRn6fIWrOogYCr78FgD9811IcUOYsq6ZouFng27DVElh7XCUX6pPkAiPP8FreJ3vD8BLzQrF5AJkfYANisdF+5G0UwaYlaVrnI3TJqZksdiI4mxgJwc09SC\\\\\\\/jycoYMQRy7SpTkUkfeMXWwVDzIAwQ+1tY7sl8Po55YJe7MJ7FrpR8It\\\\\\\/IlhUXKBIzWnXPep+92+tdqfwTCYSm+mnqsBi6pIih8l5pvxTrUJR3gXOdP0+3Vw89i\\\\\\\/jhQbb1YtUhy9UWclBsXAaB2Ne\\\\\\\/OevuaaXiB5Jd0rVqvu905BX2CyipdZ1bupEps9AticJQF3f4yK2mcX4euGD93VuAnSr2NK5oiQ7jBZ\\\\\\\/2iL7dElr1KBZ3eG\\\\\\\/M1bfKHukwm8UTKef5hlphu6qZOdmDZCfmoGWfWrmL\\\\\\\/MTo2XgDHAupYQ5s\\\\\\\/sAb+rGVKDajYKSpe7KWnJjMX6cGTpvYeqnp88r+t0d265tB0f6jP1rJKmElrfTr7QlAWbYGRNJAhzcm99ekX7m4mYuAKDJZ8beOe\\\\\\\/qEfwUBqJtXsN\\\\\\\/33C6jTiCjM7ShoZ6PoA4iVYZbPatWwFFnD4eObO6VWZ9BpayStNOD7ZlhepvzeT3iMiG27NaUFfVRUHDjbURzWlu8i+wi8tbPzIgY0XHofgn9nieqmfUKif1pKkz5eeFKC8GOFeV0NMHO9dBrUkFTuutV1\\\\\\\/\\\\\\\/4K8Nn5JR93J6E+hOb2KB0jOXIMuq0c3DUU7E9PnfUP\\\\\\\/EvjXXfG9XsLvIKnlvtgublngksKrxWWd9BZGur8+YkCy9ffsD5WQOrH8uSW9M6c4akQqn12CWyNADIqYpdFpUHp7k49gsVFG6sGhHwLzXXWIMKFf70UdTOlxPAIiMZukopZHiHrKMn8x2cty9Bn3A4tJR2yP22MZLoc\\\\\\\/omkI7E8F5NRrSU8VGQGq0XDWReiMissn3zTP78hgtmorVWQs5cGJzxNA8CUoVd2TSgsjr0R0da6zxNlHh32r7+\\\\\\\/gn7xqldCdcdj74UrQQea2B8EDSb6GMupVaEO7aj+H2WJKgZw94YsTMUpQIrGnEF7q0cxpeb8XjkjqFVGUJxrOW76bKCdCjHFyqLdqCrrdpK\\\\\\\/+xQ7hEPxcJf1jYApyEdbbKrrXM\\\\\\\/Jhx1AYYexoM3Nq0vT9FG5llL2gqwYGgs8VOl3xihhD1w0Oo0QfLaAMxOWVWES6vrfCUkRzulai6gOj0Nv2p8bD2hzp2hw5Jr2J1N3gKkiKEFAsRbBVHdmEKsVCBW2Nv\\\\\\\/z1zyiB2k7onKTV6+hDczLe71\\\\\\\/BnyLmxJopYbgd\\\\\\\/guiXw4yn2zDJzQXHIOB8pAXMYM1UE5nny75BZkuVYSj79NRv0h\\\\\\\/S4A\\\\\\\/ECMILVpQM57Jj1zX2Z2XuqJpQ+r4R\\\\\\\/kc6qRrAznlljuLFAY3PBkLix1O4peNfrR8FcVl4L8iYSIrmq7HW830GMgpHtDCSG5yPu4xPo8CXGGZTsSAQY5PSox4lNtoZnptw\\\\\\\/L9EXem166UvnHlBJcpo7UVpXfw2+MEJR5fqMuZ2Mmb+x9UE6DjlD8\\\\\\\/0mA3rTw3\\\\\\\/1tGPtRoDH46gJb80KvTuyOZKQa2rcJ79A6Gjx9DEkO21h8XSjroO1m7YVcOyWnCL1QV66Jcz+PUHg2ONlp1rXxU7f5eiii7PGq9aPJnxehmbL17L0Vp8xJVG3blHx2q96O7w1vauMatyTrFUeuZA7E2XEItW31oUV8X6vc3PhWgNb2ccaKUD0ebRE28SK62KcSqyiHvCiQ9Cjvrc0WA6PlvzCGacoyNr24o7t6tJYNCOp5+9Ep1GXBwJcTZGkBdp8vwiadaSMw4czR\\\\\\\/Rbe8Kxhxzphn1jPu0b4xPNH75EZfa\\\\\\\/wI9orWhFGa2jZuX4Yt8aDF+lImAkwPkNbRLmQ4v3D2+S99GE2D+eiqG6ixyWg9ECumYFYiss4e\\\\\\\/+xpiUk00D0eVuYslz7NInxVBjHM1PWxwYhvun49eQtbLkA\\\\\\\/JUCRyESa2RrCG5Rk8c65C30VBcSEKlI29br2wJyHiLAP5ACF+fFqrwx0c3z0+60vv6EH2Uv6PMx4p28DvDpj72RzNpMcRPoeLldrMuJRU863seJiAWGgpymz8ZHdIVqL1RPmMT8CZarksRK531nGdgI0BZOgwAARi3iOjvPzRcdpWLVV1JD\\\\\\\/Qnbpm1V4bpS37UkQ+CamhEujbVW+kR85Btq9NQyfP+HRd48XV554AUcfn2SIjMLvxxqhA\\\\\\\/LZR47gW9GQqqHvnpzyP7cLCdq4vF8EpXRTXJ8SVQrJMRjZh1elJnN\\\\\\\/BtWTE5Hc7XSrHdO\\\\\\\/tyDa5w2fA1b2Huy\\\\\\\/e+7gAAfE1OARaoXocZ7x+HDMFnOb5TfwtYzqOktuTuDrm3xNcfV4GljMlIEMm2aSI27vU6ZI6vzYi3GQT7sdHIelE9jRyleiwRTU6uBg2DtD96SFksLtyIc0Zg7ZFJsS2fu3Ec0eY+hWvCEBO7fny4gYsWo4r7+wn9q5FlcTD1u+pzhaNE77a+qRJ3Om8\\\\\\\/HoUlIehfmU5x7J\\\\\\\/WrrJf1v\\\\\\\/B1PRcs6UvaAGcqnSw0be8VgIYHF+BmZSGpUdAJ9LEnkRFunhZ1nRVOTdQ7GlPUk8CzMKPjmuqiGAUHyfg6GTskOW8K6UrpoOYFAeOimZq\\\\\\\/qO6y2TOYSpuU\\\\\\\/RTge9UhGkLyeG5tm6T7\\\\\\\/TiFJvOtT86eMUJf6vPd6+O30vR1Vs2vdaRDkEh59t9lH0ZLEkrLCVDB+u7u2BC1CPv4zRAhr7YBisA44wvkflA5NGZ2l5LDlGsq9yckRgDGeKh2eayqkf\\\\\\\/7cK3BRDVABBMkiRnLMrEtoNMve5HL4LE5CkLT8XJwFVQILR+d4d06xYK+G9qk3H0ys2zWIvuBidfYlhiasJk2KMwoPdq5XtIt1eHSttrXuAsH4TMdPyn0+x3lDLFhHx40j5s71nsm9+Z8QqwT2vrdHwfqx7HwZ8pM4zgv7PVauhawraAO9J2p+akNfDWtLF0pw2Ryf1Hn4sfkZUV53gDm6GINTpNXYoGNWtrYFfwnd5UriS0To00ruDgKGtiuBfFwydv+VeDbzsyqlk7EImd+DyLgHkVMgWzztLbt5HvMXB\\\\\\\/z7r5sLTqQcl+5q059Cc\\\\\\\/pFB9Hw0xYXmyAvEXgWV+qMD3akqRkmk5\\\\\\\/ozATDcLgQFDAtlfB6fP\\\\\\\/Gr53y84LGmoi\\\\\\\/EF1sr6okCH0tTMRFjHZb6XQq8EQfkUCjOBE9BKVfVYSySbiDG5+UehAlkZXuMTSO5JDTp9O7tSxcv2OfMkps0QYo3tiT5MrTaCyfGByIqKK7hjh7w82JodWnVyVYqaPi38ptVUEOnI65J17Ywl+FSN0j1dxqyMxSaTLS0fxps2rVu045y8Csigut1YIC5tldaCITQR8Br6RCYuGHre5sBLjlp6U+WxiAHZQVln3YZtRdqBSwKsvUJrLXCpk5ia8tfPAQrweeTPX9LL\\\\\\\/BM6VT72iUMyNGJLjGeIbCyfdCVjyzAvYP5JjeHnm+Ja9DL4\\\\\\\/mcIYFk3g7BpjGwyVWQ6acBuA6R7Jj6cHb0zXy2\\\\\\\/asJo\\\\\\\/0PCJt7lgWRlzw7dpcgncrxUjnVXCqcg7eWwdWKJiVHnj7hVjdMU+jBBnASA9g7rH8pd92v4VTCQyQ6A\\\\\\\/GGHfP5guyirt+TUWxqN9kQKfoMv6jiM+27pUl1aBfjRGKIqd9kGTCqjpL04VBKK7aJnOpqwsbNjqFuFu3BUjxVUKalVwR3y0L1702ENoY1Anfy4xDV\\\\\\\/3Nq\\\\\\\/TjO9kLsvS5\\\\\\\/N1foy9afPnKgZdVZF479GDkLWXPky7cG2DWwKoxO4OBZ19ssOQTHEdBrD9+sLY2nK0VYjWtKRQ4tkl5LHvFAWmMNIOIgL\\\\\\\/5lgm6WIZBW\\\\\\\/PCLURZ\\\\\\\/QYc9\\\\\\\/GdxsjnMxYrGbbhDw2XagAVtplLwtHq8pz0zlJEiCZwD0oHnCsG\\\\\\\/h\\\\\\\/mdE6NLBCtGaHJZKa87Df9Wz2g\\\\\\\/bXTi7q92KspPbl3qyv+1ajj\\\\\\\/2F61xZFzhAf53ruODaXGK0eq3Mf7mhfB46lZh9gJGfID0d8+gXHMjszey6wzuqlC7gtx5tgp+WcyuritndZQx+PN14YQOfZv8e9t042+L+G2Xu9JH0\\\\\\\/gvzf+AIHSaI3sVV26x51cq0\\\\\\\/maz5\\\\\\\/Ib1dH3Ci0Ra1LzL41hHEufqA6z93iyEw8DG+SNwkaObinJUYmOUBr7XGoCrSvVtf4o4TU25qSa6VJvcLJVgVLZUy7+tOBeQMCYoHj5+9uI0kd+xKov9p1X1rWD5zyTzWORm8lTIbEch3JUYe+v5JuOMlXpblXoCvZEz\\\\\\\/Nep1XXBm8fdD7MYmW\\\\\\\/OkHsRQEHpnLaKNDQspGrd\\\\\\\/TyEMwx3gTmdebVwonGj4Iny34l0j6V3HB8tmtG1KU4NlhK6Ofti03tIGCdST8JONJW\\\\\\\/2fWZw0FVRu3MIamEcT24kFwdVB65d0VNM37h9V4ONZmjHOQgQt94QBgM6rEB5zn3cLg\\\\\\\/I7u+9fxcysx1EscshWDv6heU22KaAZ0Wxba5m1yp2suUtq01bq8Q2ioBzXd4H6i8NKxPJRG8xUwrL7EbFwi3RG0hhacTT8rN\\\\\\\/Ajkc6XKbBqf+iMqpZq+EySCabtfE7Te9IQ\\\\\\\/lMOUgA1LObs01Q4UUrNKgqgdMDOLGH9s+ioLeGSKNcxfiXdkQOTVZtrJb2qgjev5pGVbeT\\\\\\\/+F+PLxOfp4Ueezl9x4zGH9h0LZ9BEM3E484wC8cmycqhf58U5BMtWgV+rss4WpGtGvD5wbzX2ZbFM7\\\\\\\/VpPLYieAWulmv6qarcvjbR6X7g0JjLW2Kh3V1c7FjV+Q6+Bgf0leFR4qbUDxFGkZpHM5Y31g7RNg2KPsILdq7iGksqsZ+higFoz9ayIb1Z\\\\\\\/q0x48h+9GoCZxQApzLFDl2lE+OMpbXe77v8\\\\\\\/q+s3LfO\\\\\\\/sq+kBTreJ4NJ2+ifHbBknGD1d7tK\\\\\\\/h5rTJq2xsoMl\\\\\\\/NN7AC7OgH1mlSXVzYBEmW2aimMN0+YCzC\\\\\\\/+r7G8zYKCtjFYEPm63FlWy2XaONEQzCi5gNyFNQMAZT1bnG94V7fI9IybBYe+ft5AuUIkvfvMs9y0iCHFh4kNoBxT8yzHmlqOy+uMOjkvB27fPpwUKlJESgTHwwk+20lkpRW91mcHhNqKxU1A9mBrGfyaJ7d5APTALJ2zQ09YkC4qZGdfuQZM3PGK4QYSK\\\\\\\/1s5MZse5bxC2YI3sdXHp0qDctvcpULdTnASRbx4lpA5bxLPCAWmSt41rI7wsLx5CzkRLLq9FPS9IjttvV7jAe4znGYGW\\\\\\\/d06qJWcsQU4a8j6qzCG2CX4JdJwayglaax01xTVMHYps4GTqlO3woWM8Lw+6dm8XFQjcIkf8200X0eCqWA9ykxC+Iv7S17FD8nvbGzLjpfN9iAQ80paf120K9Lxuw0\\\\\\\/USqKObHFW68Q6VsP4KkvO4runGiY7XuHQ\\\\\\\/GEa2p7F0m0fnVDatxUIpEFFdLD7LvbjU05z8RlJJHbYFhzt670fZYGuxJ\\\\\\\/4HZ7N1sB+YMn0K9lqcO5O6T6ToLuOz4gGjReoLLIPIQll6TvMAbDhKUd6RvdBDCnihS+iMHAkJsvf7GY2VpeGIJ0yRdRlIeCFJiqXYJE4KtzlHnvjGz\\\\\\\/r5eqfJEecSqsgt8FXe4FGxlPut0yExzdY5EcVETVDqhqNl1Bw7gaH8cp3FgNU+pfLnGMJ+WuDXcgIuIcZot\\\\\\\/5WxkteFjHkwxiQd6vQz1DTsEhTyMaF8g0xVCC2WyZOhQN1mNx+MX41QCREzu\\\\\\\/dTC+y4lgK\\\\\\\/S7bj6gC8Tuc9BZ07aa47JhQBn\\\\\\\/krZTszODkjDgFq4npRLRLUx4oFtXe4rG+k\\\\\\\/S2b\\\\\\\/XSqwbijlvQCMcnDWKtOzb\\\\\\\/VfZf\\\\\\\/ilXRm1sX8IpFt4gKBRCuquoFfZvPX0dzdtRAHiBwEi7jW\\\\\\\/UyCFkmHsf86rxkBoIFSBBBp4UlVXRAOjlr2tVlDlhZTPjZRsZNicBCWWsOrNgs7Y\\\\\\\/RrJbOEH5kXevSkZQc04c72uyIswwCk2FzOB8jFDdY74KRiVr2TmzxqvqO3wORndwtAHTluutOoL0FTK+whUHmwapEbsQPXYosHAia5FfBz67lsn3cfDOTdN5jMMPJwUlsDmyaBigQFPFp8GpKtTLvi\\\\\\\/erkswSOcAq80XAMZsIS32WLSFqe3VgxXHk3aGeWKgQ6Ru3HSuNkm2D+1pTqeDET\\\\\\\/zx0t9eSQhQRgryJoimDaBLBF\\\\\\\/tsxsWWoKMnu0rIsZli6E8VhKLpRJ2P3cmzy\\\\\\\/k6UOW5wl7Tk3qcsSCP77FVtW4Kf1\\\\\\\/h8ItnytFO1kZz8i3diDqYNTnevfFJaFMmJnRTlRyoeDpiEvhIRKOIenFqwy3RS\\\\\\\/xzYDTi44EQF4PrNQNqz++7a27MFZglgE+5J5OOPxXQO0z1KDkC98hMtuBK7a7ZRYH3S3iL8FjVPgLhO8uSeNLqNE5\\\\\\\/GXMueveSZyidk8PSxDB2RfNyoRwiZJB6L0\\\\\\\/zLKBXKIBDwogyUr3XC45oSg8MlRckWGqQbXQVkxveDacpwNS2rsxT5L+p74Xw559\\\\\\\/zPbDAZpR3jv1uCFUrmWL2mcktFM0spejJXLoQhgqMcuopFKxlqbCB1Q0L06RJT73eGsl\\\\\\\/j\\\\\\\/wpkEhIvYgWtvF+ocaF+g5TJyFaOkTQcHo\\\\\\\/0rosLG6mJ7z7n2POQJxjfDlx+XYhNv8c5gH0a6LctCY3fSQ8zYXrST38i2kLeWVzEFbW4lT2obH15KXkSVt8GsEugEfHvoE5YHUzywUybtXQNI4agEDGPqA8jSoe6BTbouiZR1kg1c\\\\\\\/Z2lCt8jlMdmx6tPOjNSN9E8+d7mUJwy5ao3LTZw5j9ceWX5BzbeVAH3fPEBerBCk5fVHT7XVPCx43gQfrt9pmYVu3uq5MrjA2AHSFPKHApwRwOoh\\\\\\\/Vo\\\\\\\/7c5rWNydK1J2aGFP6LgOGcUfr01NTDV7\\\\\\\/fg2fqStXh6ezfbAHbwm\\\\\\\/bIirKpArFVtQHwrTBE\\\\\\\/GoR8fRq7Fjzy7mpW0FbH4SW1FLEwuqsgmyZaMY31NhEJPg3IjbeHdplEEozffNwGp9fSTjW+rLR\\\\\\\/E44C6CXe7EtOKp5Gf9coRURj7vZxRaT246pRDIwSxWkD4w28avUan6dTkeoonHVqd7olQUrqq4PuJ\\\\\\\/+Q4f73cZFO6d\\\\\\\/JiN1DduKD2uC7DngwSw\\\\\\\/Q3er5xYkSV6XRFzqU84QHqBrbTx739xJx5W3hGE3WbW5QyWKJseWtnVZ6zHfzsLZq1iw\\\\\\\/Hxk2X2EGxz+02\\\\\\\/qj9uyK+HSKhUxduidVWKf6YBLVUUZb\\\\\\\/rAIwmATZ4SPBrsmclC2o\\\\\\\/iQccBC0ep\\\\\\\/VPOvz5cVChIUTn7zr\\\\\\\/zl6WHmH9X8IyIJ24qnGkg6e7Gl2KfSfymJZuOgwl80AU82XCrycltOWiLKXFCydnrhD4aoDgiVBqU4ktKRIvu21LWti\\\\\\\/wHpv807oujFLPu0OGMEqpBLBE9qIU8X6C\\\\\\\/MC7ZyKezhjGIvLz0b2F3gfWCPf0Gm8RQdiRdUtW9L59C9fcZCsMfpoRfcnoXf8mg3WlmhJiw50MJ\\\\\\\/SUvY+\\\\\\\/xim4g8wiyzweYBaQMrVlqm3XC6iLulRkvYVmefaVhpSKGM\\\\\\\/Jyhme+towM3BosaR6cYzXbJjc8\\\\\\\/7Z7JxQq058USTXimq5tdIDQYPVgtfOznnjrB1UKa0c\\\\\\\/EvXGb5GqY67ydBSCzYbY6cougLKwsWRIE5KC55MRbI5hw26t\\\\\\\/sg6dX6tJvDG1cFDR7GskOqcJJYaONW5hQ9pXRTZibtue\\\\\\\/BovDlKUttgDH6utyo2XEp\\\\\\\/V5O1SRBqhKb9DrwVLUUowYXLoIhc2X4tn3f9+SzJwR5+4TUxYIltUPn0Zyy4PPcRU0huZ42iKBkRUlLtqsDODfUI8jiOZX8gNG7jcasgGibnlj\\\\\\\/jkMGXuqR\\\\\\\/CfyPTRC3UoVsp4FCIKe32Q6frlVB6cFi\\\\\\\/ENIwNr9L3RfivlNA21oV8b88xkKDqL4pGwWSKXbTXRjRi3PRik54EwU7U\\\\\\\/loo+aClYANoohu2y15aqT4ZhwbxpB2RHYzY37BXrRJHFXWjhL1ayunuDpk4qg5IzJd0NOTMITCOebijwqWuDJGrd+vc0SXEvKZS8q15cw0wPR3pGdpjiLY1G9QnA8TX+BJXuxdzb+m1z35m3a6sS\\\\\\\/CRc2mln39AQt9t021Uns+15dDrPtGEzLlFbuuQSmxawKwjkMHplxnIXtLC74A06LKhoRAm7ZuwhJVfFqZ4LbwOhkOAdm3sOr35tWYRJVMr8Sinw9n8shwEEea84ISBZUtJm22KZgnhGyDPrd3X5vVecexj6ke2sTxiy0HJ\\\\\\\/oQvIw15N7GyX7FYcgXi5ujjN6TYWn500XFmzh+Oh57xMdWSgbkVyWEMAU4\\\\\\\/IHsRKJcGGfGzIIzGt3yXPlrHQmlkncv8ojBKFJFqPfMgtD5vJ+Tq2p90mPg7GeJSD+a5dmpNPawBGKzcioAIBqKa1pMMI9gm2TjQ3JXqk61jGrYdNHHC\\\\\\\/7p+Wxkk2IJAIWoVd8I23iVWuQXucekrXiET3U62wsfxyt5JJ2+YFd57gXGgbGlA2IGTZcz9BrgF\\\\\\\/hvhE4cN8xpPMu\\\\\\\/LeQj59It1YaHrRo3xfagAmLMvmhqzQMPZWikhKU0xRAJEjy6Rg5HfoBNH0F3ICFIbB7IgE7VYREB+8F8NKmru0lt283AjS3adeKXlfpEQcA01rrM5sdehUTWQ5eDxWIOjuKWZxvUEXEHoM9U83GmGwME7\\\\\\\/JxzwdYTL0eCl9UdclfKsn2BDxYsZRydFXaGRhSF+vhHMGkb2d1viqAj\\\\\\\/qTcg+zi75Rx5\\\\\\\/ZUxLY0oBbGQGUcwDlgKvKT0\\\\\\\/Lwrq1tByXuyD+mY9O\\\\\\\/Q5krLaE5lETo80LmdFiYKOO6EyrsyOoO5WcrYSRj\\\\\\\/G8MvXXx\\\\\\\/p67f0mEGteBrzGcN\\\\\\\/kWSMdyR\\\\\\\/4mYpdWVBlAgtujIrplsXtg8wLWiwvu\\\\\\\/bMv9mhLB0z+NPNMIpfVpUgxIgslifx5yCCGzRJtf1yq2hMg325P74P4dljDrl08YM8vUmNHasc34fMbPXmKT3iRJx39riyeH81cRCysklMcSe2N+UPVJLCAQgS24XrRJm6Rk\\\\\\\/1Gn4c6SBfM4CBIwNM10VbF1Urh1JDCCYb8bbLTL6Glo6SOzRZU5pXwBew6NyL9SRwf1mPD5UqFOyEoDM1kzUmkFzdgjrc\\\\\\\/7KoGGjlXp2barbUk5aIGUl4EfsvupmJcadjlSnfA2PlQlmRLaQQPqt65JBqhP5Wn3FOikC59FActmZfrHZTkiGis4aY6tUAJ1OhTNwVB90N5vxe6oA2+rveq5WSftFlU\\\\\\\/mrx+Q69YhgZ2v9gpwWyLjvemrOlqduosb2f6U7GV\\\\\\\/3CQ2EKqE616WPxOryoY6pLEtsFAOTe+TNjk4oLWZuqOTlgH4FTcTKTUURTCQUEYWUCkFrD7Jy7SZp6Fr0E+jxmYFAWOonqBqrTYdAX3yL86KR2kmxcHdlwe9hGpr8aJL2fQT22kcNwL3oHLsOzFjxXr4+cu3cuFrmQAVMQuYAPKJQwTV23U4zTl1wgyBnG84hOj2fxf0cxYRmG\\\\\\\/Qvw7qNUMprrYNuceaevanqRycYrHdkQWzIvDd16kwniat6zruPBy2\\\\\\\/\\\\\\\/1S5JB0XrHOyliTV+lxavtjj9RILnBCra1MyM+iDUEJHXavyKw+oFtA+4ZB7HSeE9h+4s3rkLuanGowTZ4UwCbpOg1A2ms\\\\\\\/9lx3w9llEU8LX1NKH2mOyh4rIsL9RNNW3b9w+1T+lrPUw0BiD8SZ0hCi558AlOXNEEflkUYWKfpTfqg3soevoM\\\\\\\/B\\\\\\\/pJlzU0h2uv1pkYbDzVDprfYTLUVDHK2xtbWtWnWglnVDoKdPLZucm2FLEjilqU9HROQueKcKmntfl20i5tsSLhyXsq0n3KMqurTRjdfu505Jiyt4d4zfHlCVkihLFwTs+r8lvG2dCmEFUYHXpygIJhoh96zsklF39vo2SWZS2uZTx6oloeBfWZyAPauQVX7W1\\\\\\\/I8ZKxKwxZba9Sa4VqIEspM8VTjlGjA2ZglWFseNRYR0BrBH\\\\\\\/taEb14+zJKqBlZXhpoDn1ETh6\\\\\\\/Sfc2EksYPLgFzYiMzd7zaqYxp4HxLc3VwM1mFlr36yAj5Zl9e\\\\\\\/lbLicu7Jcd1J4PJ7UBNNl1PQigVSCR3qFyMNdMeAGoAHGZFHNPYFmxoZEi8frOiCw1hbeRlD8mhWH3yBfC\\\\\\\/RqRzKK8+VfVBKZMu33Lx2WnZYlkr8FMKTyzGqWliFRdAJKGDsQ1dhpViUWNN0zVCE+EuF7wqQERj4TvfNX4ZFh+HM1b7uY6Vvc53h9LCDoRbPJqHgyz9wURCrm\\\\\\\/ZMEjceuqzskVyUAWlIv+uKaUgQlSsgTlg5owH1T2KznARcbYv\\\\\\\/VJFbNY\\\\\\\/4sbFPYpu1bWr3DW8t6fZgh0LI+m\\\\\\\/koSpKQsnyCrw7EHiAlXqdk4zAYpbpVAyhcGthS2CAIZ4BbFR+4iLJOZH656FcOVc1UdLVvmFQDJFJME4\\\\\\\/d6Am57PlSEWJiN97AITSi4fuDg5Nx2r6y+eFRHQfCY890F\\\\\\\/Z206AT4S3mgQL5Xd3D2hHoyPcfWzBCLWs9Y46jC+cH9z9G49vcXWymHGClnJgF44TsYc3xrBHzXjDOzFSqw\\\\\\\/4ond0BDCL0oi\\\\\\\/WO7lW4IaBOzopMUqRWW9pxeGA4jB+bWWFhD5p1Dt9WSAyDQSSVO7QFpRxwicILq2C3zyDzIkTez8Qogqz5oQxOxcutyGrckfNKRiTc8m3naOpy66+OgzjZx99T6XaYhzsrHbzBemocBDoXK5N2T4UVmBwiJlZJ3QhIE8SWeI0r1lGYwt1wCX9jFgpWMWa62nw10IaKPIlejmoklmpWY3IoBi4UKqLxRanCqZPPFSEuD8LDkoK84LhWP3NNvA9iXQ2O5c8OQZchl8ljDFDPUHK36ZIyzciwbCh3F8qU\\\\\\\/9ev8eh1XnqqzA58Dq1Pdw2lCeS1dWavTT+7fdQyf0mknr9dtoYGj7DKQhR\\\\\\\/+rm8GFdV1sl1GBj+JUH8KYPfGzuC7VlGRUimBHtE1CJfaHHcjLlKtg3iD3n3Dp+tJ9082oOtXZpC8ZcIqO5giuQJb++e1Y3GWwyeCBblXVtRmUQrixRcCt+hKKluCXSFf7ZGfuW\\\\\\\/ZjWzvIDrEH9xN1Qlge\\\\\\\/ajSEL1j8fsB8u6i98L9neowcJrUUabOfVtr6xFXHlxo93MnbnDlk26oyqGXdffMMiTBdtmi1weNmpwxdzkCx5E3thK\\\\\\\/a9Rer\\\\\\\/MJEK2b7FdWaBLAdzUJ42n9n+f2GVxrQRZ3OSmzKV+edIbPRaZAGdDh+\\\\\\\/\\\\\\\/KO8WstJSUCSanzgkUfF9KlCk\\\\\\\/fYBUNDZlZtU1vCmHp8wrOqLLwx\\\\\\\/nJvWJFH9Urzz5VofZfYml7rZ5HnUPuYFt8L07S5CEP8sgkEKFaGQqalib+T\\\\\\\/UOm5UtQ2sSdF6UdR29cGOiX3bd9sRd7DMao20+DurXSAIza225ai+UI1K3Br1lI308+dpRiiqoSgjBRPN2TZ2RGFNLFLHcAWtJlswULCX2rr6GPegIaKmG9i1704wIa1ESFIOunkGr2OdH9WeMU1K19JidOyFrJsibbhUiC5\\\\\\\/jRy9Sw9yZNa0eOQRyFCpbGhMw0WZOhavxmIGc4BdKKDz1BLp0VINeOI3ZOsjZsIuuOZ6woG4eQoSRNqvO09DtBBtw+s\\\\\\\/hDHfPGWsUZ6YEmD0w0NTQcECOFHvsbUP6yJyIVD0LCaCgqktvUW4yNTHTa021L3dqJTkmnCwcb+Ti16BY5ccUJgzZsAhVAd7sSmoawAmUC7Oio72QpoO7QY1ubD5Tdcc\\\\\\\/XSLAT26dmYRPNkxzcNfQ\\\\\\\/x70Cq5OtOu8LNe4ytr4PXV+izMC1\\\\\\\/\\\\\\\/93Ns0hENbxb6QM4zlQ56J8Ad9NY8CdhvWRY91O6JmineDBjKh\\\\\\\/bGnTPAdzf4R32rvg8vXlnFeyE7WfFXQ19YCn54iEOa\\\\\\\/2ce9yCxueqMAlwIZHjer19yQPzCUP\\\\\\\/B1qnzSIfkV3IoSBOUK0UoP1TVohqeir7VZjQIZ08MM2WbTASyy7ceE4QsuOwqeIGwixYglLtlMFUlzNyQe60wSB2vhBoKR80GKGHdBe7AUaUatBIYedgA3Ug3s1aadEqO5I6wQSNtmwnAcsPbxkwKxlW59xAA9Bh4HliZARadujp5gSuM2mFrr5C6S7x4Egv3Ndpon+3i33ZtuVmzYmBierKfNhnCtV2ShA26lybPsabXGxNIHSntQVMjvcQGeMDSwvz4VYX0x2TA8UPKLF0fbID1243myr5lkMLlEI4V29G6i8sLOLNKDuqEWbmotV\\\\\\\/Q11\\\\\\\/imk\\\\\\\/r1f8Shs3IUDdr1bGU1hT6Yg6Spv+g9t4B1u9I75B8ViT253bJN5Y1JDmHLIbcA2ZlU6ZRGO3M6MWAHz32xr\\\\\\\/Z9N26B0GgusxnkD065BMjpddyfTQEZhPeKh315sjmThlIwWR0Zhx3Yfy0molqbSUGYgkYXpEM4ky4XNspHlBTy5hZou\\\\\\\/TjcCkmLaF5x2ZDCWjeAo2R\\\\\\\/t8Wrfmu5AuYpUlox29MxnKF02QblzffB5VI61sYb+SnhYy7Sb9BzhXpYxgzPqOe9kO3meguvnlrR+B2Iz0QWFhKEp9f2FrFjbVq7alauZG1bsgbgudmzaxlZIFpImHz\\\\\\\/quJZ3l0Uhlw5gOX8Ob3D0zyi1zMiMnQtYNmteky1N1eHTaWDOCEUAhvXnVlUId6\\\\\\\/vEyQPvLH6t0kTktt8\\\\\\\/CgUqvV+oHWZ3aCdbOetsASU7KsZBnX4pcf7JU\\\\\\\/K0SaM7ahulBIPWzMuhI8jvsT2QISAksbGBML\\\\\\\/tNSHFGSWpI0jooKLRaeBChdYGYVWzBDc\\\\\\\/i80rwyGuoxRtbYH23JJDJPjSO5nSQjW6XsaRTzGO4XtGpAFUIjZ8TY5YkhUfJ8XNN77D\\\\\\\/tmvD1jbEcLbvq4WyfaYDQiHnI9xfCZP4E6rq\\\\\\\/st6wEARDj7oJ5H9JmzflzBHev1H6OA2g4dSD7w+3OsEAupZddsT7s+XaZ7joXdrrAYDLq+uVWGMrKfMpjQUKjhozH9CShHIQKA+BfjcRfTECrdTQfFotaL0WHmpHq5KyQ2Clz7okFc+69ylp4xc5N6PCGvvEnja9kBdfxRP3murXtbFTZK+0cBXd8OhRGpTc+VhEMW5xEj9nJ0uQda7SO0HwWiXoLwYIGtHqMc8NF4lRozbHrCs0TdoGSIsuWYREanCkCZtw1LLkxzx0cW7GOXaNEABxYapap7ZoPFRqhaDFxmNese5+u4Vmk8N+Kt8cZkJvvHFK+T3stAD43zY1rkXcN5kr6cwIPZegAMk1l0OpyIMwddM4NA086tWIZz1TMCXgIJKyobGzANiLOkMjn4iFIAeQW2EpZVBz10b3KNcvr\\\\\\\/1p\\\\\\\/o4OvnmzQUTeX6csxJ+C3QbgTlRYfZimbIbHYzWFYe4Mj\\\\\\\/epJNGZpHovpD2BhcPEaKAwn4R9qr4D\\\\\\\/3JbSVrxX+JQ3RLmcT+ggEiQv1xOMLo1GkydizCVPakhU8ZrJaXbGrJ8T5\\\\\\\/iwIy11QR5kL8cwTexJ0oRXJWTpPh6x+2bECgSa+AxeEfE3i2T4x1R1HdZpiT0c00pJJqbUhzODLl5CZqTLy6RObreZkcBzZLkivtVrpP88e+Pst4sVJBmPDyq0983oaSzveHOt3lQa\\\\\\\/K1VPe2paIGZ0+8+o2JonZTop5drv3AXBr7Na6aCRZknQYSpfBFqqYwGKTv86AdbQe8VXMTGOrx3+ulssXhSC9x2KF0sowH5KENL3cCF08SGFqWla31MYEWUGDChEaTdvlGohcyq7QTd7dXXfBz64D6Wnc727vfBvLVZDoUQxtLlnJPeH0mp9zrLyt10li+zqI3xHn+lV351gpYlaaGz3Y5DAKMFx2nb7GfFa5mlbEw+twuSkAdIUuwHeetE3pYLKEptBmvYj7rBJsb+k6rhjh7wC\\\\\\\/aCFlaHR+hzYC8hlwhxUhBdDVwoTSnb37Ohg+Cj3eIamICvtcBV+2Cis3xElXcj6PqhN4CLDfPCbtIlcktYbMQ\\\\\\\/sYF8esvji1TjS\\\\\\\/xOtx9fzdzRThZap\\\\\\\/IXlqEcKCAv6ClniQe9etGpHg9ji7RHSX35557k\\\\\\\/vVLe9aX+UEZZ6eyexmRMyl7DQOYWg1vMWdYtu132jCwW\\\\\\\/+LADsMqjP3eCH9OytK2B1XljVOiMB4MRBnifCsPNOkp8wfzxfwmIA3hlmCdhK50qnnfy7MxUBwTOwknWmhk6r1FIjiHq8W6fScoBZlvSEu2PLBYwGu6qaijqe943uM5yOMb1nrdlpItf\\\\\\\/b78X9bTCrTFzaEJBUEzxTg5rphiDxVl9SzJIxipPD0GXlWDz6CJDKA2ndYxiRwEBnHuTmZorDKr2d4QOBZXBeMxTnznocPsXsJ5we6fQ1ubtKcv4yJ5defDaf2Nq0TWtdqgMoi1LjtmRALMa2w5SXPtEnZoN34LYJOYZeCh7Syj1JCGEI0JuYLcFVVUnr8KkjrgrWaak0ulpwpaKMXIL952ivX6aHecL\\\\\\\/J9MaWhUEdSygSpp7tj2frG9WYGY6UNKze2EJgeNRS6J8nQy5ANO9fYpAK61XyPErqkeJVL\\\\\\\/1\\\\\\\/cMUYTjDif0nQsSuxwbh+zNKjZ4+9ej5Moy375mmlRIYIO0gUC3uVZPxz1ceNk0MkxpuhjeepRJwO85DPT8cSzQ6yuknuQ3u3TKXCLfFFf48ygpIr15JSafY13CFQ+oQ1cQSeRvqGvzVwiBYcoNdW\\\\\\\/QAV\\\\\\\/NgcL8hI0EdTOn5td8gKBzrFc36tPx4zdgkqKNIes+XN+3nzqOQq2sV7u2PUNMeQjnItaC3aTaCKquYkvE49TVv6VlQI9zkeVSPo\\\\\\\/k\\\\\\\/G8CIWBzdh6wriKYyxd7Xf\\\\\\\/+EHoGkV8p79j1C8r0D8vkcWHXZR7Prua4yQ4m70AlJVZiIToTM8flTs0EbVZCXa9hwqIn4g\\\\\\\/a0Kg78zAmllVfB+EOfsoi0HcwfyCkBQwfbmd3ZP5kKgiftbc9XD5wxfZNFx5Q6J9MX2Fc9KwpAYBN1IUYnRfh8J1aakIC08Kr9a\\\\\\\/phDMMCcjOMIael4OSpOTQoLZK1SX16fmqzK9z10tP5jB9joyZFQX7Q6q8CTfDmQuybzLMvX8jBiAHRcV7B\\\\\\\/vqwasYV1ZPzgmU4I9uju3M2\\\\\\\/YqRw8bbXUclpvbEEfZqWHexEwrwhbrWuetzQzNdxv0QmwvdKDLzoLvF2l4haaCDH8fDtFhdFfqcUSsqYK\\\\\\\/9Fvsuy46gemwTrT8EskLnZbU3IwsuhEpbmG3WqvTQSTLOt7aIYkdkLozm4djs1PY55Is5L0J+N48WKYIhJKysL+oMuXLomBA9VOi6CCIKe3SIz94pIQtUp9i3xluzRpjf6e4XGfPLdnO2JAgNf3qtUUBS+DyUHOZfKrG\\\\\\\/IUBxQ2h8btJzXfLaBE2\\\\\\\/j4eb2ETEahMNDwKcGXm0G6X6u4ZO45431Nd1T9BOS20FypxmzodpV\\\\\\\/Tgt81pNyAJfmq6AzT7qD9M8x\\\\\\\/XiWUhzsYCv2cXoQH7y5paVbFgRX57k+aKXQrCz5UgctVHTnyIhz4MaY1qjhy8hFma3f3mlI4WIMGfoGk4vUCTQ\\\\\\\/n1UYsjT4DGKhS9WCGnQJ+glE47TRnMTdS96Z+eCYRU0KngfXHSKeP7UFH1ge0yyLOArc2qzKSojZQk8SYilnwL8aBATp6P0CgC0u8WvH9SoHAKij0+TIGWaaJCbCpx\\\\\\\/Re8tpQPgbEvzo5fqjwGsVEt\\\\\\\/ncKxdGk\\\\\\\/ZrUvwMzg0Oisz7xNSw1VdZN\\\\\\\/hLkyvb2nVxaEpypl8S789xtdwIW8W+0SPAoxZaNl7eIZ3qlgQ282\\\\\\\/03YBobUWLemO4UfCUSvQ8roLwwsekEE8TSp3nQSebOCS+y55P0PEuHjY9G4Q5Do+XpQKCr7F\\\\\\\/uZLlTi\\\\\\\/pGUxKfK+etqLLNNVtNFU+0i\\\\\\\/xUVA8\\\\\\\/PKE6K6Z6\\\\\\\/AD8RurBGvbZ5qOOYWT13CEPWVz6N2\\\\\\\/jdsmUwUuNL\\\\\\\/HCPkBoZYTdQwvy+59O2hs1Q8oqW6IsCBX7qatC7+AX3mbfyNEJ\\\\\\\/p4Yb88oKRquITLFPxQSKMawP5XGFOHjRgay+OwRHDt+GSylZQXra4ng\\\\\\\/HYJU7op5DSBRkxZ6Xsw9eyjqM1FG6ax+uJii7qS+QApxSxGV5bA12IPRRDY++bcayDDU3gbDmJOQPV7pKFk5CZeem\\\\\\\/XfiuyH\\\\\\\/7bp4NxrzaQ87nw90O5ZO1CftNJFNuCJ\\\\\\\/N4FE0NZNtDWT6KLTq3xgCwdVH0Y1Gn8uZ6YhxbJLJqcMRyKDshtNzFfGaGjAT6oduJhRjN8Q69O+3oGWHwtJevtiE7PbpZi+OR43QY14ayA4o1Ozy\\\\\\\/OWwTSxOQWh6KSYbbZlc8xHNiTwDEN316g8vGaZHTWaiaNrmHuHhLlug5g6J5yG0dCPSeYYrjnJ58AM6CrVEw30kKPClTMdMm3Uiu\\\\\\\/\\\\\\\/35RQYe3vYk0d7TfoQC+mUMcIJJxJqx+3glaoS2YzrK+y7UR0jyzCM2RTeZeKHj+YpaewFIt+qdZAPihA2f+bX6pz5d7xCpJV7RWJ4V+x5shq67VhiU8W6a+XIAW2vYW1ZPWBfBrA8JArSAfCah4dJpQudHlPrmgSGP5ORFtKCls1zI0YGqRiiQzAvqE69okWiihL+T1S3ezS7WUZhP2Nwciiftg\\\\\\\/nVe+\\\\\\\/bRP4n\\\\\\\/QOjh5OR+T\\\\\\\/z4Ar6ZTg++zcKHmoQ3RFSoFWV1uYA5A3KTxH4CuPQsBhWyn2ucdm2dMXJ+GZCT3B5SacSljKhgxNdgdqKbkKREdm1onrVPn1pyo2Xtmej6LvlV8S9SOXX6b1NlY\\\\\\\/VBPmOHwFD6zAwY66AlVQ2kNR0KGtCIy1DLdWtWwgZJryNrYX81ucnZlbOvnCxrCF0h6ew6tWovatwZ\\\\\\\/wuvmCVU+kFPclwv0nmyMrVvJnFG\\\\\\\/VyryEBFRxgaqPF2A38\\\\\\\/UJ51r9I2iQTrTkvjYZpPH8EOMwCNRpn0LprZ95qT0W3iHCJIcB9JBnYpdmNCZwofH3kxBmXx\\\\\\\/Jph0RjZPrKMisEIltXIy6dTkqfCYxjj4MPQh6+8nH\\\\\\\/5dPchSIrAEgZ8g5tABJrChHbhuz3nYkyDJ9VCx7fpmlcLeMWqJ+45ZrDljpmccvz3Mx52YsjZQToASFyaErHKalLx3pbFzskKPcmkUme7e\\\\\\\/iGluZ+kYb6DlE8sK3cAu3JvViuXbzPNShjjzdSlvLQ01RDNQylSzjrNm3cpFYRU4hJMxegI71QZGi2jBCCf\\\\\\\/vNA2n3WxGN6YF0aq2f2xytd4im\\\\\\\/N92E6912AGgH8hI5Z\\\\\\\/2a9d3DNfPWCsHWoWNv3KfTLnpqDB\\\\\\\/HYcD+oM7d6Lx2IXwAICcJmiXDbHo3zblZbf+jYctD8A1xIk3QGPEK2YLWYW2K4cISs\\\\\\\/WFk85w03OwxJg1VL3QGLHXXwm6EX35Vutm7Dwn98AlE19V\\\\\\\/FJjeu1OtIbcegQ+6cIvQu3AwG+loMPLG3aeSud9MByOiSJ7Ut\\\\\\\/XpdKrLfHAtPFFzoayubxcf+dfBF9YLEQDg\\\\\\\/eX6vjpBqNzHYN97U3CRCleidn96E1aleEA2GaDi1R7Zm40SYdqpPuSYx8CHdkoRWZbEbMaYRMNSy+sgEsG5gvU8d2q3Bqf3v0CtNXrgsxJh36KHV8JkVtZeTXpwviaeXuJ0NECMvdsuU6i+5IMpc0J\\\\\\\/Wi4B6Hg5ln5gZXUfUlO913dMfGcs4+y2OG66Hw23E8ESetreuj5aTam3tJ7YIWVcg\\\\\\\/9M4ekdsKVdeKu6fuAkNN+9jVxtMozzuU7IeNQicEhxZXilp3MRDtHX0A5zwfiIy7COKou7kGVYfPnUNQW\\\\\\\/ULwnrg1JfWsQ39\\\\\\\/eYKhpDdM0s1IzoTT84XdY7hzyV+anPz+FJZIWCFd3n9+vhVWY+siZCUPevnhdeZA\\\\\\\/E\\\\\\\/k20Duq3TvzPDWlu0TeVNB2220+sg93\\\\\\\/177Lu4poQxYynNWMa2UXsbKV04v4uerKAE16jSFarUzjCLIpIkRC1M3YjIa2nJJ\\\\\\\/ij\\\\\\\/zqRIANk6THg7KuREOl3UmJqEgS9LxHZQDBooUz4Inmy+GcBuUqvsq47AOBm7L2blsheEEcZP\\\\\\\/av7k5DDLL3EIilGxebdMUfhXt+qVA+W1Botvk41\\\\\\\/FCJrUwmTBEszTPvO9WTnJNeXU2DcqYhefnON1hyyZYvOdSDoNbo+x5M\\\\\\\/CRUB6K6Ql1kgMDsSoBt\\\\\\\/RPi2+uGtvC7ykkLpdkqix0KR3pFU600VMmuhSO83iqTy9WnHwdJIFdS1encdaeLZuo\\\\\\\/+oi8ADR8HJxAWjmUEXmWw395C76cCPSEce5ndW7uQlhHgmW4ncwKZhSyrdqo1RNPbutu33EVEwNI6j\\\\\\\/g4CUQnPyKQNWL30mFToHxEKBImlA3NuF3yq0HquQbbNjMGSxlPVO2WRza46wwOgeNjOc0cQRSmUWRz4eAHVrClqGh5P1cXmsXabST9ArA14ecu5hNYCgWz58FjdYiPWGu3MGqLooDSu5RIwub8DuQ0GZcz\\\\\\\/MPOMPWPpQctFB09sChiJtoKqIvQUdTsBqHMdfwnGMBUbVlGhOU26Ac5zcx3XGi10QtJAMzeeXqZTML\\\\\\\/gPvwKL9Ij5llUpdtk2Zand7fvZPMvwFIiaL8fRe2mCg4qx7GBSkn0D40U0mrJ3Ndpnt3XmIcLtCnTaNtw+RzySB29iFme6iYE0lfIMxb3sgMFgAuylu8HVtowpbfcYJu+J8f6HlEHnUelYQJ6ZOH9fWNlM3lmiN5ijMkrq6PwWu15mHEjqjeEuhUpPiudCRwygBWLoToWwyO77fKj9XLJgbPsj5p3mMVTU8v0\\\\\\\/we\\\\\\\/Nw3FkJpbEvyFR9T1qsrzyBoORZ\\\\\\\/AFgztexJXM0XLP3ctYWVobat1KG11OLzimlD1aDufhaQ8dfHlcGb6AP5Gp2eOM1vCn4FbL1aSkIwsXSt3jM0GdoSiqREgMQ5M3OWbAHpHWI884o8BKE\\\\\\\/L1u9bqPhM2XqdaayBz0Bqb6gNkLbi3KBgwnTM0GUHplCdkOgSIVK55eejIhEUt1I9ZD+zQJI9LTYMIaW6Sv+ip\\\\\\\/afBkKiP5HwLI\\\\\\\/IbtEz14ewV7yHwvYdK3qN2oEz9UR9pZp6NGXjUuhSH8Lj8Hvop\\\\\\\/anA6pfCY\\\\\\\/IxB3WN4yz0xj3Gy8BPNvmIMzePGF4RlFgg66dn02lOXKXM0jF1TArg1uyOZHnmYqcGzBJaHFnBqqC6CJ09M+M09EAPgXqYk4odcpxsmnkAtw5dfo56VkdQVkBqQd32Z9TCqnSpKY9XPBC4CHu17+ZETQHiAdqs2egVlc9MFMoU7yf2r+CSgdOVeSZvzOpiw5WxHnNL84bWCp44QzBiGN2hrTwQr646jdDFb57N8ZJw5uaQ+2X3enk1qPA4RtchdhkPsw2Ju24IjEaQoB3Ius0M\\\\\\\/GqR7lYTjm75Sn2oXroutOmvL9RvrcV9WGxWX06lNxWKAaXAzTRdvQfLFkIy7hSPSupqd6wBIhUtCPHBYBfC0TaRwuEjr6svRCTDnZySJGVhWSmTYBXZ4VNAfOCN0Jfwfz\\\\\\\/olunycfmbPvGCBnU3lYdevi8W0tc9O7LNXq74wVK8PCxatZ8TW6FxVebwzCS6mncaBVrygZDQjxClEnOBXgM3Ul6Wd6\\\\\\\/om7D6usQkLqmEWkYAnPMsJ15cPJqTMWdJmJmmd4qbxN3V9ulAu4+rtDDYjGjmbVuuijzvpnzcVEyzjQuz9jOKTDE1Xm9OXrXEDJb9SXbE2nr44t9iHu4nmy7mRbBk3Bwyja7TS2HHJuAYSyHMdJO2jYX\\\\\\\/aki7EEHaOhnvjy82d66C+oT4TplhxSuHq4ZEVVDHatjeXg+ln3nu39L12zgvaV7JAURKf\\\\\\\/a2mj5nANMP6i\\\\\\\/pue6uNtSicGWFVMvuQ4ZtnfrLay+v7571a4v3vq\\\\\\\/EKeNJsJ0OrTGQ4xArZ4iavsr5OSN3H3l\\\\\\\/F4DO\\\\\\\/B+e+uUGVfCe\\\\\\\/8XkyV8YeJ0Z+\\\\\\\/RljNaMGFvOa9H2VFSJdyyIv6hl6BpYGgruDgT+wWE7gBU5L+OLHNE5E2lnjZoeEEfKq2r58GVMPXJboZYUOt+sxrhSBM60iaGQEC3TE5eww6FHxHFTSP4iydJzqFlkg2VM+R8tYaCjcl7t8hPNs+7gMZqP4JOKTdU94t7GQPw\\\\\\\/FqBGcmDSjnUc\\\\\\\/yvYGWHJpTrURZUWl1HrgH9I1qJ\\\\\\\/YFOfuHld38Tt22F+y\\\\\\\/w5lhaVGH9A0ApgN3JuRDf\\\\\\\/Cv6u3hnWFD7tf0ARv5aedsbpKtTTmkKRrVa9MX\\\\\\\/9RFyuDQMYEi58METQHoE4seNYesRPazyNF5hFjBEqE0XldaYSvsNLXI002I8B46vV3yQDJrvENDz71aVgsnbRXpJTZKzvasns7wZUoHGwmyJxfGfctXjvg3PRr8oEX9PDF0Bc74cCTCqZmJsZX90LAYLZaJ9YCqYY3B3wsVCQvbWCIzBLFxtfwt92BE7XYFGBYQmd4UlEXRA52HzRVGkQ9F7fWv7SFyIdlvqoheEyrjdI+NIE9C0xa8B8KWjqMIErXUF2KWxpCE8drSlfAF\\\\\\\/8XGMacJlF2ZWfO8rgjH8R+CiGJGRsw9blP3GzZ1GijpU\\\\\\\/Y+j6I05FSkcLwzLtk3H7zd\\\\\\\/gesHLYUS1qJk38HqpgXmGfJatLItwD6CFfuUZ+rXZVuTGRIjCH3I8XcAuKPGwNXKBPmwDOVaCRz6DlaC\\\\\\\/hSDK0SYYGC8VzGr+85+ccHDnK9HGYe2MzJcLb+h+kd5baCCDfJfj9gqfNS5qk0cIsLevWhM\\\\\\\/sxj8UaPFtuL0F6YeS+BvDLKnPRn6HLdQNXaYDG06PAst7exL7uE9DJFTWGeGgRzuXmuMQUOgP6wPzjSciRHA7wfm7+9jiiizTjwcMlI5\\\\\\\/kHPqZt9rhL96kN\\\\\\\/sp73HDJMROH+N2P8Cz+bz8ZTkSqDEEkmormBKdlJ0CTnHlCp1oKTQkOflX3VPGSQDsk\\\\\\\/zmeK2O6AtLf3u9ygzoVp2AVlh6bPIX0PoF\\\\\\\/2GpEA4LPSzX9aIcNqmxR7uUneEqXL5sQqwAZ\\\\\\\/vVCXHpVd3A4ZhFUu5PPn\\\\\\\/58qOFxvLr1RAO40VojCsNRa42XfdNsCwkhKiayoBm+caY4ixMlvAwkWhcsUgXqxvYczCdSEcDrtUs1x+zdfb5384tuabrKnDRiGil7Md4xaPSC2nrVi2Io7fTl18TxUB90FWA+g4Vmg7CEPJ\\\\\\\/H\\\\\\\/o6u5FFdi7UNAiR\\\\\\\/vNuEZXi0hUohcNMNhRIPIyO1qGO+EvpjOrAH8QVR6aEgcTMejaSUmz15+XyQmvi\\\\\\\/VCnz0CnSeNjMS\\\\\\\/obAczkQ87Eg8caLICGUxJmypCjILJ5H1o4hozrdJ1fnR9ihNKVNw+p9NOvajYFZqoTSffjUoifrFS66X+weMOAzU5\\\\\\\/FtZtQ8Sb\\\\\\\/yoRCxmwQkSQDJXCjhb5jg3k4vgI\\\\\\\/409AwNM+g3PYYzE1sclkq29yo\\\\\\\/yQtklM6koKtM9brFh1a764ujCh6ZsQlMWXhcYu9trx7eGn\\\\\\\/22lI4aDo68dPO7VNMceJWOvCWzMqrM1ZuN2X8\\\\\\\/r+1ivFSxeedcmwdnEIpm3+SApVGojcaNrP\\\\\\\/odcW\\\\\\\/vtQ5EEwfrEt2ikPCfVHp\\\\\\\/7TgDjtzhLJSt76QyTtH0+Pd2mUsPEtpzWqHv5a0QzD5qp7pwoCPocy9Gpa0ouOXLUjZ3hOAhcFkPwIAcO3UimiHfnAb\\\\\\\/6G8D9IU7XiSdeYQWswepWKeBaowJXcpGnIL1NzIcd4itkg5l1Hkc1hVO76kEsC1UcO3EK83IuUBkwWROkmuWDgR0PBGSgPzE\\\\\\\/P1+UedlsudXx14bAwuSVR4lID8J6LR3kIXXglSeDFgj69hcngQqHfDzaTMhudQP65fxycSehUKCUIXb8pnHytjkXnAo3MU7P2gv2owrZuYnxtHozUTjCjuDkdt1PMmKipQ+B9j5Fs8tFwFkrF5zmSsQul751y1FrurwaFR+OuzLPcfXMMWg\\\\\\\/Rs67Sz\\\\\\\/No5EU9WAo+Vx2av\\\\\\\/u3tBBEpqFbMJ0J0xAzje23ZjZBOR\\\\\\\/t\\\\\\\/UFTwQc1Hnqddr9gcJmx8RruKKPNzP2EtDlHwEA1l7zWu3rVg6CA73bb5aMyWuT2YpADK+x0ll2jEedNm8KFiXe85LdV9V4Y4\\\\\\\/o43C1D+oYXfsj04tQmqOthzYEg4A9EfVpglfQf2jFm\\\\\\\/qlwtjyj3U7PKR58g6IDOrh4KPttxxmg\\\\\\\/g5WaoR4r4uWrsTM\\\\\\\/bMH00qF+i7+ECc\\\\\\\/At3GYUQ3EeWfAl5cEShTTSOHDyI2Q3z6FTcTrn6RiLDfc8NPsmMs5oh2gEgWorjEXBJILsjUxSn6AuLAtGljPCsNsKhbFJ60SWxIO9ySgDFcorpXXktiUEhOXW1FYinhIV3vu0K7a8Meugn\\\\\\\/KfLL4nqmLkgNFrHQIpew57zgGKXQoiOlUEb\\\\\\\/N6WA0+780JwgttX\\\\\\\/CGhYisbPDnj88fqOCTxLCNNHimILP44ceIe+X9OhXLiHT\\\\\\\/TsmxryqSBW\\\\\\\/0mO80jlv89UhDMwuN3K7ConmqUrxbI1PCnlWN53zRJ2TLArAkNhvecPpE3VY+ZvBV1CzSm82V8gBqBut9tKv9+riXisXQuHHvvmpsRlrQsu1jUOJdjwf8W51V+vsbPNcxh7h3O5dAY2ZMU9PRmAtHB9a3NsBv4tuBNZduSyx1P6uBoE7bdXDxs7uZ\\\\\\\/FZ7p4FwDP214p2yHwTcHYW5vrjIdKkZDsu0LeaQujjMSLYvT8Fave8ZfF1zYz3R7gIHRWghlyGdHn739fVHW+2auWgOEqRU4dyP2IkEXKiICtk7aMWXfBYzsNPC3oFJ6I0vUZ5EXkoyYwSFjMEkxyDvD23vxoVcR7FghlLehGAiJBUHLFQvt1jU6kxfe35Vkr2NVA5ftVPS8X7eSzDJEvtcSOFDhQTE+XxwT7ffuJlOB1FmaLCxb6fhpTzGISCqxNb4esp2FNyfDVE+d38nuk49ND542HWVk\\\\\\\/OzTQu6MEc2dXkehutQQwWbX1RVPbe7n3S6fgxtwGI8y5ZY73bRQ7oaxEaJmvFi65N5mXm3Stj1AwAoTEVzGEmV0+7DabMxPmwergIyqWZe2wNH8jS1CzYA4bWyW+I7dy+1HMiAd4dpWfBVyhm3+4jCREKQOo3LX83RyzpvAIEkPmzUqHBn9IBqQOHlbIFsYY3O3\\\\\\\/G8mPka1o5mmcv60dSOkSaIqZ1QMDbEIzQ05vcTYCR8vO\\\\\\\/sLhHAoKP1nkhssQoKL1aNSSmHwqcRHipfXOBAb3UwsSczUiSWHeQsA+3GpijQZB4gOp\\\\\\\/XshRnO+XqWkIjgzPs+9On+XjPPduxWTZ8WhTkBMTz+Sv5GOkWQtcRurQY17pTpUghxjeqga0V7pnxfAvI+dcdM4rYYS0mhe7aOsGdq77OuP8H2QFTgxfDgE\\\\\\\/JDlq7KpdZzyrBuB\\\\\\\/0VjFuPe5ORY05kWOi9NorxnKTCgubhYRExrd6hSuQqYT6XlcKXgoiRHevDNNHTAR6fJcV1fgX3cYWzu0Y2rSPIvf0WsQBwUHYKuLVn8bxWqA0snd0+4fVHvo7VoZ+OrAunHiM2od94RNEJdPdFPK+Ag0V5J4rxuBuDC1aWW30slMjU1BfAqte2NW0KVnLxnkHvb1i867lH3zGcTtM0RSrHL4YDaT6xupHOYG1G16ehmJFuuAllFbj9Bx6iDJfwb0LURoPLRhf0TKEYWhHfHX5+3GlNu1HFJWYQQ39KXN5Zy9R9z8kd3V0b2VnAoBqn7Gdt61kNjCLixkbzuY4twWNsEsCFawdqmYRV5XYG1SJwa55SGuwlgHvrq6TrXWIJzJbWOsc9Vi6pu5Zp6vNaa0R2Mk1seDJuxxqds51SkPVnVolod8hlwOPXSKeJvzyNBnpVgkjhhAGCXN8s+bBAwl1LcWptLhNkk4Awtn2BZYNhRvW4Ale9LXiZzMX6dCRJDGwdWE3dShhoetXOM6aLq7RL68RucuEVxCE65AQALqCivw948hDROzLgJSqqlEsdSsdEPFqM677RXh6QHiffE8UTP3ylREh1meVqsDXHiG5flV8xN\\\\\\\/RL47P+t\\\\\\\/LYy4K9ryl3VQ68og5wlcEA31tkcY5bHdcuLdlRg\\\\\\\/zOBt3fxvaBClKK2kObumw3qGXsgwlpHc98MMP7QiIVeIWWa+rzfq\\\\\\\/t2caqlkgmVRgn7m7hlOs2gfFYfy9p5YmcnyPCJAF5OMfLR0ESfrOBHPmi51xefCyIlexyy1isloeueLdk+7JJH9Kjma3SVdIZi43YLJ3aS3B+Uwc2WjvIAZ66aQtDwQ0B0HATeREadaAfCHJTAQzFyLNOhS92qlKrJiVC2\\\\\\\/l5oIQgJiy1siC2jyVKHUWnwF4BLO5wCxQ9EIUKNZnH7EzX9mrC3OcjKSaZ6zzWGFuy51mbuXiTZXQB3dri87QI3fu6Q5SIi1kjAn6ZZ4Z3lClh30eOelKqV4wnG5wk7VuznWVnGZI2Z6CNEYWPKr2RXCkCZBYPlcXQg1SMGVwT3YmNfFmRTE51x2MRASCXtgQt5QCNDB340AZ+tZ\\\\\\\/R+1wN8n9p+LFfsEIT3nFA3xJQCPKG8tfNUNXqyRKlCPBWFAqbh1CnGR6oNmGg26ydtz4dBTH5h\\\\\\\/za1xOdSOl2OFwwXp+7MccAv1ldHbVUEtAA4r5i3mnxcYLU25Xs9vMRrSjNCRmdTp\\\\\\\/Gm\\\\\\\/vYFROuM2phFuzht338x5yWs39NQT9+r0aoICM+xVsNu1x7Ff1dBs5OhzGkpGetya+tNTiRBAKFHbN6ThVNgojvgRxabOXuEY+p59oET+zcutlMllfWNwOOstcMLlkDMK77ro7a\\\\\\\/cUCsktgZglwHDZP7mngmty4YZiEJtco2C+oKHWQWuQGPcHJ0pjKvWKMAHG5v3bfghEXD1tO64cr0bPGBbl83ZcKuyIw1LFSeePhC9JZ++erpHXwjk5nxbzYCowlnyc\\\\\\\/VQS8c5PpfJrDjlMAda+S0LB5iHz9TTYig24gDYEUTy+uboe4AzU9NLdWglgpmwM2HH2y6btvYRpdEo2E5FUnuWBwiamx6aWr4dtIGJNus7N7+6G0DGHku3X\\\\\\\/Dgp2tvqqXro1Pu2qwMyhNdgNELeKs4q5NBVainrvUS0FvJbyOFyBq9277fbEuXjwR5PF+GeAl5ZuP5BeqPyR2HO\\\\\\\/1d9\\\\\\\/gxCC5+L63fpGY\\\\\\\/2IaK8xycE63EUNHxZiPJruLbbP3Z3OXnL7EV9QrwNBa9y6lgvJ03g\\\\\\\/YbiIzkuDtV6CUnJIxmkaJ3o25UuhrgWarCldWOZhE2GT3\\\\\\\/fOcegq1u63eqk7sW5a8+ztJgUTMpGTBm+2sC7klWqbHB3fppwci7bkvw\\\\\\\/jaJuiHyd1uFClUiYsVKU8rSu7rGx7zV7EAnRfFU3yXMXWO0nX7dt5nB3yJZkNLA7FEctc6z2gW0v+S5BT5GmOX0y9zxX0MMjaok8\\\\\\\/I3MXsg46jl9uY7MHPEJZb92gmY9foOp7p1FyOZrCy6k9bkJDhDN7PDslU51YGf4lVJNwYDpCnzHvNCtRd2hjFylHBER6qyvGAPSfaLOAQtjf40\\\\\\\/oNdjYq8pFWiFqTyTP1xP6m5jHgrhFJP9EDehX6QdQ9YLHEMUHzfZQm7mOD3qHJhfStDJx6NBSGwRkZwU8ZXRqNSC+m9X+wFDfgYoaMu6Ml4FvOfB207m5hT2NhiF1Z+G6SCQKpZ6do3DaK8BTP\\\\\\\/s5ozKp0axb\\\\\\\/72Itq9g96J8GEaotFXrYWtvLZL1dw\\\\\\\/atb25\\\\\\\/QWxcleYCj7Mbz1FjY4gpSRHvx+1Xt49ZZgffiCFo2Vv\\\\\\\/lSSw09Z8sgWH+QhcNQx3Dkx9xhoLOn0vqgeTnpP\\\\\\\/q\\\\\\\/C8NgDd4gLmRJCcBaVws\\\\\\\/x45HTXGotSg2hV1biZQwftTcVOqo4wztminIuhvjMD\\\\\\\/xp8pIxPV\\\\\\\/BUxzPgA\\\\\\\/geqHX2EQlnF9JVhX1QjsclO0mtWacsM271pzCx6r2Wo0Y+rdMZDL7G8jTvE4kb\\\\\\\/Lqxx3kgDvlZPc\\\\\\\/1i8RZYr3kAi+v5ScKTK\\\\\\\/HKX8ai3OJx5nYne5gX3FXP4rxpS0iuLxBof7ULOAjAwpW\\\\\\\/eL0bND9jWX4jX0SVf2Pgpmz1ZRe5XcyQjzybpYIY2G\\\\\\\/bXBQe9Buc6fqvix+boYHpiJIfMDPdhx9u6C\\\\\\\/dLh0DT3595JSmtFPIUDDaDvcdmZ3xmTaIK3sO3okGnkpI5ybu8IWVOrPkEbbMDFXMzwYrKBazwqEcEsUOxg5z5S+OnYzzxLmSc5Q\\\\\\\/8a0uQHpSzzXK+vZ79cVaPy4dBr2+LUpbREeSWZ\\\\\\\/TbJHfinPESMNDgCaQNvb+yJpScndz687sQ\\\\\\\/EeRMxGnBUGqzqKKRsO8j3HBk4HkTAURiArG1qcOooJcZXGw7Db3eiHb54eIs7XS4cJS3V2z6KsAbrqk1FBNxF\\\\\\\/ZdwS8M0UXM4Ffgz8HBdYgU9QQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r8O6dXABHoTEWexHVm8iYA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022480fc43e95617dae59376e1fdf5b993caf003097a59dbf051d4f8e22d316df9c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022h2l4N93tkcWJauvJ\\\\\\\/qZYtReXyGivpO34XYe6M1C0E6kDF8TfIxPbNTbt2zMJNyA1fhvph8e8dgtImrPQRQAxoFsgFw4LhWGzdq3PK2W9qCAR+cwz3TUkyzqN006dxE86KHDPJcAA2ksUpzWDvVeuTV7ImuOYFEq\\\\\\\/D7mYJXjPtQk8hGM+muLJ\\\\\\\/g4rWLcbSfoO9e9G0JMp77LLaE8SfJCSTPj19F8FjS3GzJMp0xdqwI+i1SAnYjyTNbq4Qk4ILDW6dkZG8d9nek0VYljY2kr2cmzSz4HbrvDLI1\\\\\\\/nCKZnT5eXb\\\\\\\/klytMpNTyzjy7mxyQWPMjbPmGOu0FeQboQBl59h82RNT8hUsP9V80Yve35PDrZAH4Vkd0dC981VTSsk\\\\\\\/WB8m5HfQxHtw\\\\\\\/Q4wGBtpbpTgo4BxiT94gf9K9dW5023W2zFVdKZ0lrhApCl95DVTzC0cUN8w2h3hQRhs3RyTu0zFoCWHda+Q9dbBA7mgWpfGCftbVhD7N63Xy\\\\\\\/Kc3i10+ru20s9zQOT6XR7GuvqbsV+R2aESK2FFsGHpN4kCMs9nweZT9K3O00Lrk3HXTtI0EQOWKOjd+FPaDk5aL\\\\\\\/50a1QlfmckfjNYDoXAJxi2T4ctVVdxnTa3QH\\\\\\\/goXAVDTkJlQqMAAC6XCGMTzxIVy52Iwy+ncT3TOyp2k\\\\\\\/S8FNBB9X0OJ9+6vJJufS0YCJ8ZiyvgrZz0UF+3HBjqPLJWOrq2oyEaWoa06f45fVO8gmsgQ\\\\\\\/+yd3lc6OnEIupCNHzZhmu4q1\\\\\\\/69Pb4W+j9xEUu95e5jX0ynwm8wRI5D4A\\\\\\\/1iYWTHqSzKC8X\\\\\\\/PnfKxX1YBi7rFz2CCEf097NsG0qy+AB9BgJA1G7rsFx5d54SztZ9cTBpn1dSAHPz\\\\\\\/8njXFtgx3s0voXMnXmLpgYc7zlhNjhyPiEdvmprDML4zpt9p6+R6mLClXPgZVjRU4IdpiUaLo8C9cfnLobAZNmQ22rkAlu6aShIVTcUY8pYAr+CtyejrSic0jhsCK4Pg9W8uOGIIc6HhOwx1ktIzkIFKqF5TDo6JbUhiqq6WPb+oDaer3NsSO9jZ+P2z5HZera7ofPQQ58Jz1+J0+Ic0e3wxioVDpK\\\\\\\/U1o73M6T0kgw5ddYBEpz1mIus5dsh0KgwOFOOmqeEaQ+BiDARhM1BQlEO0j7uAI\\\\\\\/yRI6y8Hks4AOFiEVEHw2f2eDVAcNIspIzut2\\\\\\\/QjDqt1+5u1N4KKDyeWB46DokjM8TjfbTJ4U08Bs2RI2S58JkU1BcfQIUTcYZRNAkY+HkHl1HasZZTLocHuuVnUaMfTrQLN73QtU95gc74bBSEQEpWp4NnsPDkuESAV\\\\\\\/PfIrjbkdzKlH0g4ycalkFipohV7qRqdgsJ9A8Zm9esSzVLKQJUnAVHgKGLtoSKkjPZvs\\\\\\\/o4tTNR9fzCOVtLotj7JO03yR9XaelCWUdo3uhNo0c5PZ+3X7lZXoSteyfR+LlV9VqmP2ve9OoSCL6Eh20DlIhehm3zRlZyIWXVy4qD2o1d9qoesZwWj4tdVitTfvAeP3u0sArzoe4q1\\\\\\\/DrL\\\\\\\/09eex8\\\\\\\/E9gp6R+Y5Z0yQWiL5q9lBLQCK2Sr6Blu0AA0lEyK5wJQvKJVDUyzMvsiDHt94iaJ2qnExXtvC+JXu3gQG4gLKkgCBcT\\\\\\\/CdqEoH1HZZiiLapArcy3aM\\\\\\\/40IeLL2dIl51xkA6PZ8cY67eM18JkaDoEyE4lyGjefWVGIpxHHREk3cTcTbnQLoFBjSsocCkePKkJJtr0ytE\\\\\\\/nzPyjvm7cklZyK6XFlHeteujw8mCMTQ6myKJ\\\\\\\/6kNMKdBvj9xfnWO61KMsDLLoDLphT+E1uAfI3XALklPE+\\\\\\\/KginnhWuGXQ\\\\\\\/\\\\\\\/eSmmQRfz5yGxt8yDLARjEHOv\\\\\\\/scW6u5cVWXJOB1SfdtO+8OC6a3onAU6PZK4wIV1fTsXlEHdInaWjs3z1J1iDhoGzE2eI2yDoIX1tWOHJjLrzhMCGGZP5\\\\\\\/+xOS7ROj142X3a8WtSNuHX1xjLAzqFQXtOMR46Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/Uo\\\\\\\/FtWSIqwswiQ9xPzYVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e8e9b7c69644c0e5b94ab3e4b89e1a37dd09d8f888e8d92aa8ef975813c7434\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459820 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/509" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wrGCB4f8GMaR26Nu23QwMt-vGHmhq5plzRzYAzWUzic" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nkFigHvlDzjxgOf5QEjbFRmsUBaPGdhaIxzgjyHYylM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/497" title="Reduced rDNA Copy Number Does Not Affect “Competitive” Chromosome Pairing in XYY Males of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/523" title="The Yeast Ess1 Prolyl Isomerase Controls Swi6 and Whi5 Nuclear Localization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459820" id="node459820" data-pisa="ggg;4/3/509" data-pisa-master="ggg;g3.113.008441" data-apath="/ggg/4/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer L. Modliszewski</span> and <span class="highwire-citation-author" data-delta="1">John H. Willis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008441 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459820" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer L. Modliszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BModliszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Modliszewski%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BL.%2BModliszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jenmod@live.unc.edu" class="" data-icon-position="" data-hide-link-title="0">jenmod@live.unc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John H. Willis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWillis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWillis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/509" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459820" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459820/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/509.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459820" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459820/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/509.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459820" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459820/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/509.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459820" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459820/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/509.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459820" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459820/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Genetic variation is the fundamental medium of evolution. In allopolyploids, which are the product of hybridization and whole genome duplication, if homologous chromosomes always pair, then all descendants of a single diploid F<sub>1</sub> hybrid lineage will be genetically identical. Contrarily, genetic variation among initially isogenic lineages is augmented when homeologous chromosomes pair; this added variation may contribute to phenotypic evolution. <em>Mimulus sookensis</em> is a naturally occurring, small-flowered allotetraploid derived from the large-flowered <em>Mimulus guttatus</em> and small-flowered <em>Mimulus nasutus</em>. Because diploid F<sub>1</sub> hybrids between <em>M. guttatus</em> and <em>M. nasutus</em> have large flowers, phenotypic evolution post-polyploidization is implied in <em>M. sookensis</em>. Here, we present genetic and phenotypic analyses of synthetic neoallotetraploid <em>Mimulus</em> derived from a cross between <em>M. guttatus</em> and <em>M. nasutus</em>. Genetic marker data from S<sub>2</sub> and BC<sub>1N</sub> progeny suggest that chromosomes regularly pair with their homologous counterpart. By measuring the phenotype of synthetic neoallotetraploids, we demonstrate that polyploidization <em>per se</em> does not induce the small flowers of <em>M. sookensis</em>. Moreover, phenotypic measurements of synthetic allotetraploid F<sub>2</sub>s and S<sub>4</sub> families suggest that rare homeologous recombination events have a negligible phenotypic effect in the first few generations. In total, the results are consistent with either exceedingly rare homeologous pairing and recombination or spontaneous fragment loss. The low levels of fragment loss and phenotypic variation in neoallotetraploids suggest that homeologous recombination after polyploidization is not a major mechanism of phenotypic evolution in <em>M. sookensis</em>. Rather, it may be that spontaneous mutations or epigenetic changes after allopolyploidization have driven phenotypic evolution in <em>M. sookensis.</em></p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mimulus" class="hw-term hw-article-keyword hw-article-keyword-mimulus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mimulus</em></a></li><li class="kwd"><a href="/keyword/neoallotetraploid" class="hw-term hw-article-keyword hw-article-keyword-neoallotetraploid" rel="nofollow">neoallotetraploid</a></li><li class="kwd"><a href="/keyword/segregational-variation" class="hw-term hw-article-keyword hw-article-keyword-segregational-variation" rel="nofollow">segregational variation</a></li><li class="kwd"><a href="/keyword/homeologous-recombination" class="hw-term hw-article-keyword hw-article-keyword-homeologous-recombination" rel="nofollow">homeologous recombination</a></li><li class="kwd"><a href="/keyword/colchicine" class="hw-term hw-article-keyword hw-article-keyword-colchicine" rel="nofollow">colchicine</a></li></ul><p id="p-6">Genetic variation within a species is essential for evolution by natural selection and genetic drift. In plants, polyploidization, or whole genome duplication, is a common and instantaneous mode of speciation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dobzhansky 1937</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Masterson 1994</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Otto and Whitton 2000</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blanc and Wolfe 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coyne and Orr 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cui <em>et al.</em> 2006</a>). A unique aspect of polyploid speciation is that species formation can involve only a single individual, creating an extreme genetic bottleneck. As a consequence of this, when allopolyploids form from diploid F<sub>1</sub> hybrids, it is possible that all descendants of a single individual will be genetically identical, with the exception of spontaneous mutations and ploidy-mediated genetic changes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Comai 2005</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Soltis <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Doyle <em>et al.</em> 2008</a>). Spontaneous mutations and method of polyploid formation aside, the extent to which descendent lineages are genetically identical depends on the pattern of chromosome pairing at meiosis (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stebbins 1971</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Comai 2005</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ramsey and Schemske 2002</a>). Allopolyploids will lack genetic variation if the homologous chromosomes from within each diploid progenitor species pair faithfully during meiosis (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and C</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). However, if any of the homeologous chromosomes from different diploid progenitors pair with one another, then multiple gamete types will be produced, resulting in genetically variable offspring (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and D</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/509/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of expectations under homologous or homeologous chromosome pairing. Chromosome pairings of the same color (black&#x2013;black or gray&#x2013;gray) are homologous and chromosomes of different colors (gray&#x2013;black) are homeologous. (A) Homologous pairing and subsequent fixed heterozygosity. (B) Homeologous pairing and novel genetic variation created by segregation and recombination. (C) Homologous pairing and independent assortment do not further contribute to genetic variation. (D) Homeologous pairing and independent assortment further contribute to genetic variation among offspring." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-143196013" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of expectations under homologous or homeologous chromosome pairing. Chromosome pairings of the same color (black&#x2013;black or gray&#x2013;gray) are homologous and chromosomes of different colors (gray&#x2013;black) are homeologous. (A) Homologous pairing and subsequent fixed heterozygosity. (B) Homeologous pairing and novel genetic variation created by segregation and recombination. (C) Homologous pairing and independent assortment do not further contribute to genetic variation. (D) Homeologous pairing and independent assortment further contribute to genetic variation among offspring.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/509/F1.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/509/F1.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/509/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/509/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Illustration of expectations under homologous or homeologous chromosome pairing. Chromosome pairings of the same color (black–black or gray–gray) are homologous and chromosomes of different colors (gray–black) are homeologous. (A) Homologous pairing and subsequent fixed heterozygosity. (B) Homeologous pairing and novel genetic variation created by segregation and recombination. (C) Homologous pairing and independent assortment do not further contribute to genetic variation. (D) Homeologous pairing and independent assortment further contribute to genetic variation among offspring.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464029/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464029/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Expected gametic ratios, genotypic frequencies, and observed apparent heterozygosity in F<sub>2</sub> and BC<sub>1</sub> progeny in five models of chromosome pairing</span><div class="sb-div caption-clear"></div></div></div><p id="p-9">Homeologous chromosome pairing at meiosis affects genetic variation in gametes and subsequent progeny by segregation, which results in genetic variation among gametes at a single locus, and by independent assortment, which results in genetic variation within a gamete by random alignment of all chromosome pairs during metaphase I of meiosis (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>). More specifically, if one considers a single locus, an allotetraploid individual with the genotype AAaa will only produce Aa gametes across all loci if homologous chromosomes pair, a phenomenon known as fixed heterozygosity (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ramsey and Schemske 2002</a>). If homeologous chromosomes pair, then AA, Aa, and aa gametes will be produced, resulting in five possible zygotic genotypes (AAAA, AAAa, AAaa, Aaaa, and aaaa) and genetically variable offspring. The ratio of gamete types (<em>i.e.</em>, AA, Aa, aa) for a single locus produced by homeologous chromosome pairing depends on whether recombination occurs between the locus and the centromere (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Haldane 1930</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mather 1935</a>). If recombination does not occur, then random chromosome assortment, as modeled by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gregory 1914</a> and <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Muller 1914</a>, and segregation of chromosomes will produce an expected gametic ratio of 1AA:4Aa and 1aa (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Muller 1914</a>). If recombination occurs between the locus and the centromere, as first proposed by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Haldane (1930)</a>, then the expected gametic ratio becomes 2AA:5Aa:2aa (random chromatid assortment) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/FigureS1.pdf">Figure S1</a> and <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Mather 1935</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">1936</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burnham 1962</a>).</p><p id="p-10">Although chromosome pairing models are useful for predicting expected gametic ratios and genotype frequencies, adherence to expectations will vary among loci according to three conditions: the proximity of the locus to the centromere; the particular chromosome pair being considered; and how strictly a particular chromosome associates with its homolog. The last condition, the degree of preferential pairing (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Sybenga 1994</a>), refers to the fact that chromosomes may form any combination of homologous bivalents, homeologous bivalents, or quadrivalents; this has been observed in autotetraploid <em>Rorippa</em> species and <em>Crassostrea gigas</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Curole and Hedgecock 2005</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stift <em>et al.</em> 2008</a>). When only homologous bivalents form, chromosomes are said to preferentially pair.</p><p id="p-11">By measuring the phenotype of polyploid F<sub>2</sub> populations at single-locus traits, early plant breeders, focusing mainly on autotetraploids, were able to reveal that gametic ratios vary by locus in accordance with random chromosome and random chromatid pairing models (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blakeslee <em>et al.</em> 1923</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lindstrom and Humphrey 1933</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fisher 1941</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Doyle 1973</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Little 1945</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">1958</a>). Surprisingly, in allopolyploids homeologous chromosome pairing and recombination have also been inferred (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crane and Darlington 1932</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sansome 1933</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gerstel and Phillips 1957</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">1958</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gerstel 1960</a>). More recently, it has been shown in <em>Brassica napus</em> that allopolyploid lines that began as a genetically homogeneous population can accumulate genetic variation among lineages by the first generation after allopolyploidization as a result of homeologous recombination; most importantly, this genetic variation is associated with variation in flowering time (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Schranz and Osborn 2000</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gaeta <em>et al.</em> 2007</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Szadkowski <em>et al.</em> 2010</a>).</p><p id="p-12">Homeologous recombination in <em>Gossypium</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Salmon <em>et al.</em> 2010</a>) and translocations and reciprocal aneuploidy in <em>Tragopogon</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lim <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chester <em>et al.</em> 2012</a>) may create genetic variation among initially isogenic lines, but these changes have not been linked to phenotypic variation. Gene expression in allopolyploids has also been reported to vary from expectations based on an additive model, suggesting that epigenetic changes may also play a role in phenotypic divergence between polyploid and diploid progenitor (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 2001</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Flagel <em>et al.</em> 2008</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chaudhary <em>et al.</em> 2009</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rapp <em>et al.</em> 2009</a>). Of the studies that have examined genetic and epigenetic changes after polyploidization, surprisingly few have made a concerted effort to record phenotypic variation of synthetic and natural allopolyploids while also examining genomic changes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Comai <em>et al.</em> 2000</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anssour <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Matsushita <em>et al.</em> 2012</a>). In synthetic autotetraploid <em>Heuchera grossulariifolia</em>, polyploidization appears to cause an immediate phenotypic shift toward trait values that may have been beneficial during establishment, although flower size in synthetic <em>Heuchera grossulariifolia</em> is large, unlike in natural tetraploid <em>Heuchera grossulariifolia</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Oswald and Nuismer 2011</a>). Thus, aside from the results in <em>Brassica napus</em>, little is known about how the multitude of genetic and epigenetic changes observed in synthetic polyploids may lead to the phenotype observed in a naturally occurring or crop allopolyploid.</p><p id="p-13"><em>Mimulus sookensis</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benedict <em>et al.</em> 2012</a>) is an allotetraploid derivative of the large-flowered, predominately out-crossing <em>Mimulus guttatus</em> and the small-flowered, highly selfing <em>Mimulus nasutus</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sweigart <em>et al.</em> 2008</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Modliszewski and Willis 2012</a>). <em>M. sookensis</em> is small-flowered, similar to <em>M. nasutus</em> (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Benedict <em>et al.</em> 2012</a>), but diploid F<sub>1</sub> hybrids are somewhat intermediate and phenotypically similar to <em>M. guttatus</em> because of partial dominance of <em>M. guttatus</em> floral characters (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fishman <em>et al.</em> 2002</a>); this suggests that phenotypic evolution occurred following polyploidization. A genetic linkage map constructed from a <em>M. guttatus</em> × <em>M. nasutus</em> F<sub>2</sub> population establishes that chromosome pairing and recombination occur in diploid F<sub>1</sub> hybrids between <em>M. guttatus</em> and <em>M. nasutus</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fishman and Willis 2001</a>). Consequentially, the homeologous chromosomes of <em>M. guttatus</em> and <em>M. nasutus</em> are predicted to be able to readily pair when placed together in an allopolyploid genome, possibly leading to homeologous recombination and rapid recapitulation of the phenotype of natural <em>M. sookensis</em> in synthetic allopolyploids derived from <em>M. guttatus</em> and <em>M. nasutus</em>. Although the chromosomes of <em>M. guttatus</em> and <em>M. nasutus</em> are capable of pairing in diploid F<sub>1</sub>s, in an allotetraploid the two chromosomes of <em>M. guttatus</em> may be more likely to pair with one another, and likewise for the two homologous <em>M. nasutus</em> chromosomes. For this reason, it is unknown how the chromosomes of allotetraploid <em>Mimulus</em> will pair.</p><p id="p-14">Here, our broad objective was to determine how chromosome pairing following polyploidization may affect phenotypic evolution in <em>M. sookensis</em>. We first asked if there was evidence for homeologous chromosome pairing and recombination, using genetic marker data from artificially synthesized neoallotetraploid <em>Mimulus</em> derived from diploid <em>M. guttatus</em> and <em>M. nasutus</em>. We then assessed the immediate phenotypic effects of polyploidization by measuring the floral phenotype of neoallotetraploid <em>Mimulus</em> and comparing it to diploid <em>M. guttatus</em> and <em>M. nasutus</em>, diploid hybrids, and <em>M. sookensis</em>. Finally, to shed light on the evolutionary processes that have shaped flower size in naturally occurring <em>M. sookensis</em>, we asked if homeologous recombination may facilitate an enhanced rate of phenotypic evolution by contributing to phenotypic variation. We address this question by comparing the variance of quantitative traits in F<sub>1</sub> and F<sub>2</sub> hybrids and S<sub>4</sub> families. Both the genetic marker data and phenotypic measurements will reveal any heritable genetic variation in newly formed synthetic allotetraploid <em>Mimulus</em>, which natural selection may act on.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of synthetic tetraploid lines</h3><p id="p-15">To generate synthetic tetraploid lines, diploid seed from <em>M. guttatus</em> IM62, <em>M. nasutus</em> SF, and diploid F<sub>1</sub> hybrid seed generated by reciprocally crossing <em>M. guttatus</em> IM62 and <em>M. nasutus</em> SF was soaked in a 0.0001%, 0.001%, and 0.01% colchicine solution for 6 hr. Both <em>M. guttatus</em> IM62 and <em>M. nasutus</em> SF are highly inbred lines (&gt;12 generations), greatly reducing the likelihood of residual heterozygosity in diploid and autotetraploid lines. Because only F<sub>1</sub> hybrid seeds were transformed into polyploids at the aforementioned concentrations, subsequent treatments of <em>M. nasutus</em> SF and <em>M. guttatus</em> IM62 were performed with concentrations of 0.05% and 0.1% colchicine. After the soak, seeds were rinsed ∼10 times using distilled water and pipetted onto premoistened Fafard 4p potting medium. To identify tetraploid lines, plants were screened using flow cytometry once sufficient leaf and bud material was present, as described previously (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Modliszewski and Willis 2012</a>), using NIB chopping buffer (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bino <em>et al.</em> 1992</a>). To increase the efficiency of the screening process, four individuals and an internal control (<em>Petunia</em> × <em>hybrida</em> “Wave”) were processed simultaneously. If tetraploid peaks were evident, then all four individuals were run separately to identify the tetraploid individual(s).</p><p id="p-16">Two forms of synthetic neoallotetraploid lines and one diploid F<sub>2</sub> population were generated (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). For the purpose of clarity, we hereafter refer to the combination of genotype and ploidy as a class (for example, IM-2x indicates diploid <em>M. guttatus</em> IM62), with reciprocal hybrid classes combined, unless indicated with G or N, which are used to designate the maternal parent (for example, F<sub>1G</sub>-4x is used to indicate tetraploid F<sub>1</sub> hybrids with <em>M. guttatus</em> IM62 as the maternal parent, whereas F<sub>1</sub>-4x plants are tetraploid F<sub>1</sub> hybrids with either <em>M. nasutus</em> SF or <em>M. guttatus</em> IM62 as the maternal parent). Synthetic neotetraploid <em>Mimulus</em> lines are referred to as “neoallotetraploids” (<em>e.g.</em>, F<sub>1</sub>-4x, F<sub>2</sub>-4x, S<sub>4</sub> families) or “neoautotetraploids,” as appropriate. Neoallotetraploid lines were first created by synthesizing neoautotetraploid parents [<em>M. guttatus</em> IM62 (IM-4x) and <em>M. nasutus</em> SF (SF-4x)] and reciprocally crossing them to create F<sub>1</sub> neoallotetraploids (F<sub>1</sub>-4x). First-generation backcross progeny (BC<sub>1N</sub>) were generated for genetic marker analysis by crossing the resulting F<sub>1N</sub>-4x neoallotetraploid to neoautotetraploid <em>M. nasutus</em> SF; the F<sub>1N</sub> plant was used as the maternal parent in this cross. Reciprocal F<sub>1</sub> neoallotetraploids were also selfed to create two F<sub>2</sub> populations for use in phenotypic analyses; reciprocally crossed populations were treated as a single population for most analyses (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Neoallotetraploids were also generated by treating diploid F<sub>1</sub> seed from reciprocal crosses between <em>M. guttatus</em> IM62 and <em>M. nasutus</em> SF with colchicine. The progeny from two reciprocally crossed neoallotetraploid individuals were made into 48 family lines for each cross direction (S-lines), which were then selfed for three generations to create two groups of S<sub>4</sub> families, S<sub>4G</sub> and S<sub>4N</sub> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>), which were used in the phenotypic analysis. Individuals from the S<sub>2</sub> generation (S<sub>2G</sub> and S<sub>2N</sub>) were used for genetic marker analysis. As a control for the neoallotetraploid F<sub>2</sub> population, diploid F<sub>2</sub> populations were constructed from reciprocal crosses between diploid <em>M. guttatus</em> IM62 (IM-2x) and <em>M. nasutus</em> SF (IM-4x); and a single plant of each cross direction in the F<sub>1</sub> generation was selfed to generate the F<sub>2</sub> population (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/509/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossing design for phenotypic analyses. (A) Diploids. (B) Synthetic F2 tetraploids. (C) Synthetic tetraploid S4 families. 2x, diploid; 4x, tetraploid; IM, M. guttatus IM62; SF, M. nasutus SF; subscripted &#x201C;G&#x201D; and &#x201C;N&#x201D; are used to indicate that M. guttatus (G) or M. nasutus (N) was the maternal parent. All S-lines (S1&#x2013;S4) are tetraploid. S2 lines were used in genetic marker analysis; S3 lines were split into &#x201C;a&#x201D; and &#x201C;b&#x201D; subfamilies to test for maternal effects in the S4 families. BC1N lines were generated by backcrossing F1N-4x to SF-4x." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-143196013" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossing design for phenotypic analyses. (A) Diploids. (B) Synthetic F&lt;sub&gt;2&lt;/sub&gt; tetraploids. (C) Synthetic tetraploid S&lt;sub&gt;4&lt;/sub&gt; families. 2x, diploid; 4x, tetraploid; IM, &lt;em&gt;M. guttatus&lt;/em&gt; IM62; SF, &lt;em&gt;M. nasutus&lt;/em&gt; SF; subscripted &#x201C;G&#x201D; and &#x201C;N&#x201D; are used to indicate that &lt;em&gt;M. guttatus&lt;/em&gt; (G) or &lt;em&gt;M. nasutus&lt;/em&gt; (N) was the maternal parent. All S-lines (S&lt;sub&gt;1&lt;/sub&gt;&#x2013;S&lt;sub&gt;4&lt;/sub&gt;) are tetraploid. S&lt;sub&gt;2&lt;/sub&gt; lines were used in genetic marker analysis; S&lt;sub&gt;3&lt;/sub&gt; lines were split into &#x201C;a&#x201D; and &#x201C;b&#x201D; subfamilies to test for maternal effects in the S&lt;sub&gt;4&lt;/sub&gt; families. BC&lt;sub&gt;1N&lt;/sub&gt; lines were generated by backcrossing F&lt;sub&gt;1N&lt;/sub&gt;-4x to SF-4x.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/509/F2.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/509/F2.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/509/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/509/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464067" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Crossing design for phenotypic analyses. (A) Diploids. (B) Synthetic F<sub>2</sub> tetraploids. (C) Synthetic tetraploid S<sub>4</sub> families. 2x, diploid; 4x, tetraploid; IM, <em>M. guttatus</em> IM62; SF, <em>M. nasutus</em> SF; subscripted “G” and “N” are used to indicate that <em>M. guttatus</em> (G) or <em>M. nasutus</em> (N) was the maternal parent. All S-lines (S<sub>1</sub>–S<sub>4</sub>) are tetraploid. S<sub>2</sub> lines were used in genetic marker analysis; S<sub>3</sub> lines were split into “a” and “b” subfamilies to test for maternal effects in the S<sub>4</sub> families. BC<sub>1N</sub> lines were generated by backcrossing F<sub>1N</sub>-4x to SF-4x.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Assessing evidence for homeologous recombination: Genetic marker analysis</h3><p id="p-18">Our first goal was to determine if homeologous recombination was occurring in neoallotetraploid <em>Mimulus</em> by using genetic marker data. DNA was extracted from 48 BC<sub>1N</sub> individuals using a modified CTAB method (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kelly and Willis 1998</a>). These 48 BC<sub>1N</sub> individuals were genotyped at seven microsatellite loci known to differ between <em>M. guttatus</em> IM62 and <em>M. nasutus</em> SF. Each marker is located in the distal portion of its respective chromosome arm. In addition, 48 individuals, each from a separate S<sub>2</sub> family (24 S<sub>2G</sub> families and 24 S<sub>2N</sub> families that were used to create the S<sub>4</sub> families), were genotyped at 13 microsatellite loci, with one marker per linkage group. PCR conditions were as follows: 3 min at 95°, 12 cycles of 30 sec at 94°, 30 sec at 60°, and 45 sec at 72°, with annealing temperature reduced by 1° each cycle, followed by 32 cycles with annealing temperature held constant at 52°, and a final extension time of 20 min; the primer sequence for each marker can be found at <a href="http://www.mimulusevolution.org">http://www.mimulusevolution.org</a>. S<sub>2G</sub>, S<sub>2N</sub>, and BC<sub>1N</sub> progeny were then scored as either heterozygous or homozygous with the assistance of the GeneMarker software program (SoftGenetics LLC, State College, PA). To test for departures from models of chromosome pairing (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), we constructed a 95% confidence interval based on the binomial distribution for each observed proportion of heterozygous individuals using the pooled marker data from each progeny class separately (S<sub>2G</sub>, S<sub>2N</sub>, BC<sub>1N</sub>). For the BC<sub>1N</sub> pooled marker data set, we also conducted a <em>G</em>-test (log-likelihood ratio test) using Equation 17.4 as demonstrated in Box 17.1 of <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sokal and Rohlf (1995)</a>, with the random chromosome model as our null hypothesis. For the six markers that had at least one homozygous individual, a heterogeneity <em>G</em>-test was conducted as per Box 17.4 of <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Sokal and Rohlf (1995)</a>.</p><p id="p-19">To further assess the degree to which homologous chromosomes preferentially pair, we calculated the preferential pairing rate, <em>p</em>, which is a measure of how often homologous chromosomes pair with one another, rather than with their homeologous counterparts (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Sybenga 1994</a>). The preferential pairing rate (<em>p</em>) can vary from 0 to ⅔. At <em>p</em> = 0, each chromosome arm may pair randomly with any other homologous or homeologous chromosome arm, and chromosome arms are assumed to pair independently, allowing for quadrivalents to form. The preferential pairing rate reaches an upper bound at <em>p</em> = ⅔, which is derived from the fact that ⅓ of all bivalent pairings are between homologous chromosomes and ⅔ are between homeologous chromosomes. Thus, under a model of strict homologous pairing, homologous bivalents have a probability of 1, or ⅓ + <em>p</em>. <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wu <em>et al.</em> (2001)</a> developed a model for calculating the preferential pairing rate from genetic marker data rather than chromosome configurations at metaphase, as shown by <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Sybenga (1994)</a>; we used the model of <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Wu <em>et al.</em> (2001)</a> to calculate the preferential pairing rate from the BC<sub>1N</sub> progeny.</p><p id="p-20">In the BC<sub>1N</sub> progeny, there are three possible genotypes, GGNN, GNNN, and NNNN, with the SF-4x parent (NNNN) contributing NN gametes and the F<sub>1</sub>-4x (G<sub>1</sub>G<sub>2</sub>N<sub>1</sub>N<sub>2</sub>) parent contributing GG (G<sub>1</sub>G<sub>1</sub>, G<sub>1</sub>G<sub>2</sub>, G<sub>2</sub>G<sub>2</sub>), GN (G<sub>1</sub>N<sub>1</sub>, G<sub>1</sub>N<sub>2</sub>, G<sub>2</sub>N<sub>1</sub>, G<sub>2</sub>N<sub>2</sub>), or NN (N<sub>1</sub>N<sub>1</sub>, N<sub>1</sub>N<sub>2</sub>, N<sub>2</sub>N<sub>2</sub>) gametes. We directly solved for <em>p</em> using the genetic marker data and equations derived from <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Wu <em>et al.</em> (2001)</a>:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/509/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/509/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/509/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/509/embed/mml-math-2.gif"/></noscript></span> ,<span class="disp-formula-label">(2)</span></span>where <em>p</em> = the preferential pairing factor, α is the double reduction rate, and <em>f</em>(G<sub>i</sub>G<sub>j</sub>), for example, is the observed or inferred frequency of a given gamete. Because our microsatellite data are not fully informative (<em>i.e.</em>, G<sub>1</sub> = G<sub>2</sub> and N<sub>1</sub> = N<sub>2</sub>, and GGNN and GNNN genotypes are indistinguishable), we chose to make some simplifying assumptions that allowed us to calculate the preferential pairing rate directly from the data rather than to infer the missing genotypes using a expectation maximization algorithm, as suggested by <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Wu <em>et al.</em> (2001)</a>.</p><p id="p-21">For each marker, we first assumed that an equal number of GG and NN gametes were created in the F<sub>1</sub>-4x parent, despite the fact that we were not able to infer the occurrence of GG gametes. Second, we assumed either that any NN or GG gametes derived from the F<sub>1</sub> parent were the result of homeologous bivalent pairings or that any NN or GG gametes derived from the F<sub>1</sub> parent were derived entirely from quadrivalent pairings, and a crossover occurred between the locus and the centromere, allowing for double reduction. In the case of the first scenario, equation 1 is set equal to zero, because G<sub>1</sub>G<sub>1</sub>, G<sub>2</sub>G<sub>2</sub>, N<sub>1</sub>N<sub>1</sub>, and N<sub>2</sub>N<sub>2</sub> can only be derived from quadrivalent formation and subsequent double reduction. In the case of the second scenario, we assumed that three-eighths of all GG and NN gametes are derived from double reduction events; this frequency is expected if all nine possible quadrivalent arrangement and separation combinations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/FigureS1.pdf">Figure S1</a>) are considered. Because of our simplifying assumptions, we could not solve for the rate of double reduction (α). To evaluate whether the observed <em>p</em> fit the data better than a model with <em>p</em> = ⅔, likelihood ratio tests were conducted for each marker using the expected gamete frequencies and observed gamete counts, as described by <a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">Wu <em>et al.</em> (2001)</a> and <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Curole and Hedgecock (2005)</a>.</p></div><div id="sec-4" class="subsection"><h3>Common garden experiment and phenotypic analyses</h3><p id="p-22">Before the final common garden experiment in which all lines were measured, diploid and tetraploid parental (<em>M. guttatus</em> IM62 and <em>M. nasutus</em> SF), F<sub>1</sub> and F<sub>2</sub> lines, and two maternal lines for each of the 43 S<sub>4G</sub> and 35 S<sub>4N</sub> families, as well as seed from one maternal family from each of two <em>M. sookensis</em> populations (<em>M. sookensis</em> FAN and <em>M. sookensis</em> ROG from Vancouver Island, British Columbia, Canada, and Mehama, Oregon, respectively) were all generated simultaneously in the greenhouses at Duke University to standardize any environmental effects induced by the greenhouse over time. Neoallotetraploid F<sub>1</sub> seed was freshly regenerated by recreating autotetraploid parental lines and reciprocally crossing them. Previously generated S<sub>2G</sub> and S<sub>2N</sub> seed was also grown in the final grow-out, but the seed was not matured at the same time in the greenhouse as all other seed. At the time of the final common garden experiment, and after the first generation of the F<sub>1</sub> hybrid-derived neoallotetraploid lines (S<sub>2</sub> generation), synthetic polyploid lines were assayed for DNA content as a proxy for ploidy to ensure that the unlikely event of a reversion to diploidy had not occurred. Flow cytometry was performed as described above and as by <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Modliszewski and Willis (2012)</a>.</p><p id="p-23">For each plant grown and phenotyped, one to four seeds were placed onto well-moistened Fafard 4P soil-less potting medium in 2.5-inch pots. Flats were covered with humidity domes and placed at 4° with little ambient light for vernalization. All flats were removed from the cold room and placed in the greenhouse 6–8 days after sowing. After germination, pots were haphazardly randomized by moving pots from one flat to another, and extra seedlings were discarded, such that each pot contained one plant for the remainder of the experiment. Before the date of first flowering, flats were haphazardly moved about daily in the greenhouse to minimize spatial effects.</p><p id="p-24">On the date of first flower, the first or second flower from each plant was phenotyped for the following traits, as described by <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fishman <em>et al.</em> (2002)</a>: corolla width and length; corolla tube width and length; stamen length; and carpel (pistil) length. The width of the lower calyx was also measured for all plants using the distance between the two lower calyx lobes. Stigma-anther separation was calculated by measuring the difference in stamen and pistil length. Flowering time was measured from the date of germination to the date of first flowering. Pollen viability was assayed by macerating all four anthers from the first or second flower in a drop of lactophenol aniline blue (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kearns and Inouye 1993</a>) placed on a microscope slide and then examining a haphazard selection of 100 pollen grains for viability under a light microscope at 100× magnification.</p></div><div id="sec-5" class="subsection"><h3>Testing for an immediate effect of ploidy on phenotype: Analysis of means</h3><p id="p-25">To address whether polyploidization in <em>Mimulus</em> neoallotetraploids has immediate and direct phenotypic consequences, we calculated the mean value of all floral traits. Principal components analyses were first conducted using all phenotypic traits to visualize the distribution of F<sub>1</sub> and F<sub>2</sub> classes relative to parental and <em>M. sookensis</em> phenotypes. Principal component 1 (PC1) and principal component 2 (PC2) were subsequently included in all mean and variance calculations. To test if ploidy has a significant effect on phenotype, Tukey-Kramer HSD tests were performed among all classes (<em>e.g.</em>, IM-2x, IM-4x, F<sub>1</sub>-2x, F<sub>1</sub>-4x). We verified that comparison of means and variances by class (<em>e.g.</em>, F<sub>1</sub>-2x) was appropriate by testing for significant differences in the means within each reciprocal cross (<em>e.g.</em>, F<sub>1G</sub>-2x, F<sub>1N</sub>-2x) using a <em>t</em>-test. To test for normality, a Shapiro-Wilkes W test was conducted on each class for each trait. All statistical analyses of means and distributions were performed in JMP Pro 10.0 (SAS Institute, Inc., Cary, NC).</p></div><div id="sec-6" class="subsection"><h3>Testing for an effect of ploidy on phenotype through homeologous recombination: Analyses of variance</h3><p id="p-26">In a typical diploid F<sub>2</sub> population in which heritable genetic variation for a trait exists, an increase in variance, attributable to segregational variance, should be observed. To test for this increase in variance in the diploid and tetraploid F<sub>2</sub> populations, F-tests, and Levene's test were conducted in R (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">R Development Core Team 2010</a>); 95% confidence intervals for the coefficient of variation were calculated using equation 4.9 in <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Sokal and Rohlf (1995)</a>. To test for a significant family effect in the S<sub>4G</sub> and S<sub>4N</sub> families, a nested ANOVA was conducted separately for each trait and also for PC1 and PC2, with both family and maternal lines (nested within family) as random effects, using the REML computation in JMP Pro 10.0 (SAS Institute, Inc.). Broad-sense heritabilities (<em>H<sup>2</sup></em>) based on the diploid and tetraploid F<sub>2</sub> populations were also calculated for each trait as <em>H<sup>2</sup></em> = V<sub>G</sub>/V<sub>P</sub>, where V<sub>G</sub> = Var(F<sub>2</sub>)−V<sub>E</sub>, and V<sub>E</sub> was calculated from parental and F<sub>1</sub> variance.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Identification of synthetic tetraploid lineages</h3><p id="p-27">From the F<sub>1G</sub>-2x (<em>M. guttatus</em> × <em>M. nasutus</em>) and F<sub>1N</sub>-2x (<em>M. nasutus</em> × <em>M. guttatus</em>) seed treated with 0.01% colchicine, one F<sub>1G</sub>-4x and one F<sub>1N</sub>-4x tetraploid individual was found after screening 59 and 137 individuals, respectively; these two plants were used to generate the S<sub>4</sub> families. No tetraploid <em>M. guttatus</em> IM62 or <em>M. nasutus</em> SF was found among the 81 and 27 individuals (respectively) screened that were treated with 0.0001%–0.01% colchicine. In treatments with 0.05% and 0.1% colchicine, 11 neoautotetraploid lines of <em>M. guttatus</em> IM62 (IM-4x) and two neoautotetraploid <em>M. nasutus</em> SF (SF-4x) were identified after screening 44 and 337 individuals, respectively. Interestingly, although the rate of conversion to tetraploidy was quite low in <em>M. nasutus</em> compared to <em>M. guttatus</em>, the two tetraploid <em>M. nasutus</em> individuals were the result of the 0.05% colchicine treatment. Likewise, more <em>M. guttatus</em> IM62 individuals (seven <em>vs.</em> four) were converted to tetraploids at the 0.05% treatment level. We attempted to convert other lines of <em>M. guttatus</em> and <em>M. nasutus</em> to tetraploids but often had limited success with <em>M. nasutus</em> and had varying success with <em>M. guttatus</em> (data not shown), suggesting that there may be a genetic component to the observed variation. Although all individuals were screened regardless of appearance, by casual observation the leaves of neotetraploids induced via colchicine tended to have a thicker and darker appearance, and pollen grains were larger. Flow cytometric analysis of neoautotetraploid lines revealed that the autopolyploids had approximately double the DNA content of their diploid progenitors, as did neoallotetraploid F<sub>1</sub> individuals (2.04 ± 0.018; n = 7) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS1.pdf">Table S1</a>). A small but significant decrease in 2C DNA content was observed in the neoallotetraploid F<sub>2</sub> population (2.00 ± 0.007; n = 16) when compared to the F<sub>1</sub>-4x class (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS1.pdf">Table S1</a>).</p></div><div id="sec-9" class="subsection"><h3>Genetic marker analysis reveals limited evidence for homeologous recombination</h3><p id="p-28">Depending on the model of chromosome pairing, the expected genotypic frequencies in the S<sub>2</sub> and BC<sub>1N</sub> progeny will vary (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). The progeny examined can be classified into two groups: heterozygous (GGNN, GGGN, and GNNN genotypes, with “G” indicating the <em>M. guttatus</em> allele and “N” indicating the <em>M. nasutus</em> allele) and homozygous (GGGG and NNNN). In the S<sub>2</sub> progeny, all five genotypes can occur, but homeologous pairing can only be confirmed using our genetic marker data if a quadruple homozygote is formed. In BC<sub>1N</sub> progeny, only GGNN, GNNN, and NNNN individuals can be formed, regardless of the chromosome pairing model. BC<sub>1N</sub> progeny scored as heterozygous are either GGNN or GNNN; although GGNN genotypes are the result of homeologous pairing, they cannot be detected using the microsatellite data. Under even the most modest of homeolog pairing models, one-sixth of BC<sub>1N</sub> progeny are expected to be quadruple homozygotes that have lost the <em>M. guttatus</em> allele (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-29">The genetic marker data revealed that homeologous recombination is occurring at an exceedingly low, and in some cases nonexistent, rate. In the S<sub>2</sub> progeny, no instances of marker loss were observed at any of the 13 loci. The probability of failing to observe a quadruple homozygote as calculated by (5/6)<sup>24</sup> was 25% under the least stringent random chromosome model, and the probabilities of failing to observe a quadruple homozygote for the total number of individuals observed over all markers in the S<sub>2G</sub> and S<sub>2N</sub> classes are 2.14×10<sup>−8</sup> and 2.26×10<sup>−8</sup>, respectively. For the combined marker data at S<sub>2G</sub> and S<sub>2N</sub> classes, the 95% confidence interval under a binomial distribution incorporates only the homolog pairing model. On a per-marker basis, no chromosome pairing model can be excluded for either S<sub>2</sub> class (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464033/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464033/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Results of marker screening in S<sub>2</sub> and BC<sub>1N</sub> individuals</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">In the BC<sub>1N</sub> progeny, 10 instances of marker loss were observed over the seven markers surveyed, with one BC<sub>1N</sub> individual contributing to six of the 10 observed instances of marker loss. Fifty-six instances of marker loss (one-sixth of the 336 marker/progeny combination) were expected to occur under the most conservative model of homeologous chromosome pairing. The 95% confidence interval of the proportion of apparent heterozygosity, as calculated using an exact binomial test for the entire set of marker data, is 0.946–0.986. A <em>G</em>-test of the entire BC<sub>1N</sub> data set under the random chromosome model resulted in a rejection of the null hypothesis (p-value = 9.07×10<sup>−6</sup>), indicating that the data do not fit the least stringent model of nonhomologous chromosome pairing; when pooled, the markers did not show significant heterogeneity (G<sub>H</sub> = 1.99, p-value = 0.85). When each marker in the BC<sub>1N</sub> data set was analyzed separately, only one marker (MgSTS_727) had a confidence level overlapping the least stringent nonhomologous chromosome pairing model. Overall, the genetic marker data did not display a pattern of inheritance strictly congruent with any of the models of homeologous chromosome pairing.</p><p id="p-34">To provide further insight into the nature of chromosome pairing in synthetic allotetraploid <em>Mimulus</em>, we calculated the preferential pairing rate (<em>p</em>) (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Sybenga 1994</a>) from the BC<sub>1N</sub> progeny. If chromosome pairings are exclusively homologous, then <em>p</em> = ⅔. Our genetic marker data indicate that <em>p</em> ranges from 0.46 to 0.67 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Assuming either 100% bivalent pairings or 100% quadrivalent pairings has little effect on <em>p</em> (data not shown), and thus the values reported here were all calculated under the assumption of 100% bivalent pairings, which actually results in a slightly smaller value for <em>p</em>.</p></div><div id="sec-10" class="subsection"><h3>Analysis of means: Polyploidization results in large-flowered <em>Mimulus</em></h3><p id="p-35">To determine the effects of polyploidization on floral phenotype in <em>Mimulus</em>, floral traits were measured in neoautotetraploids and neoallotetraploid F<sub>1</sub> and F<sub>2</sub> hybrids and compared to diploid parental and hybrid lines. Phenotypic analysis revealed that F<sub>1</sub> and F<sub>2</sub> neoallotetraploid <em>Mimulus</em> are large-flowered (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Principal components analysis revealed that floral traits are highly correlated, and the first two principal components (PC1 and PC2) capture 77.21% and 10.02% of the phenotypic variation, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS2.pdf">Table S2</a>), with most floral traits positively loaded on the first principal component. Tests for departures from normality revealed that most traits did not fit a normal distribution and that a log-normal distribution also did not fit the data, with the exception of width:length ratio (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS3.pdf">Table S3</a>). Subsequently, with the exception of width:length ratio, no transformations were performed. The non-normality of the data is not expected to have an effect on our tests for differences of the means.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/509/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCA scores for principal component 1 (PC1) and principal component 2 (PC2). Symbols are coded by shape as follows: &#x25B4; and &#x25B5; (diploids); &#x25CB; (synthetic tetraploids); orange x (M. sookensis ROG); and orange + (M. sookensis FAN). Symbols are color-coded to indicate M. nasutus SF (blue), M. guttatus IM62 (black) F1 hybrids (light green), and F2 hybrids (purple)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-143196013" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PCA scores for principal component 1 (PC1) and principal component 2 (PC2). Symbols are coded by shape as follows: &#x25B4; and &#x25B5; (diploids); &#x25CB; (synthetic tetraploids); orange x (&lt;em&gt;M. sookensis&lt;/em&gt; ROG); and orange + (&lt;em&gt;M. sookensis&lt;/em&gt; FAN). Symbols are color-coded to indicate &lt;em&gt;M. nasutus&lt;/em&gt; SF (blue), &lt;em&gt;M. guttatus&lt;/em&gt; IM62 (black) F&lt;sub&gt;1&lt;/sub&gt; hybrids (light green), and F&lt;sub&gt;2&lt;/sub&gt; hybrids (purple).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/509/F3.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/3/509/F3.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/509/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/509/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464119" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">PCA scores for principal component 1 (PC1) and principal component 2 (PC2). Symbols are coded by shape as follows: ▴ and ▵ (diploids); ○ (synthetic tetraploids); orange x (<em>M. sookensis</em> ROG); and orange + (<em>M. sookensis</em> FAN). Symbols are color-coded to indicate <em>M. nasutus</em> SF (blue), <em>M. guttatus</em> IM62 (black) F<sub>1</sub> hybrids (light green), and F<sub>2</sub> hybrids (purple).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Means and 95% confidence intervals for floral traits. IM, M. guttatus IM62; SF, M. nasutus SF; FAN, M. sookensis FAN; and ROG, M. sookensis ROG. Ploidy for parental, for F1, and for F2 populations are indicated with 2x (diploid) or 4x (neotetraploid) as appropriate; both S2 classes are tetraploid. Means are shaded to indicate ploidy as follows: diploids (white), artificial neotetraploids (black), and naturally occurring allotetraploids (gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-143196013" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Means and 95% confidence intervals for floral traits. IM, &lt;em&gt;M. guttatus&lt;/em&gt; IM62; SF, &lt;em&gt;M. nasutus&lt;/em&gt; SF; FAN, &lt;em&gt;M. sookensis&lt;/em&gt; FAN; and ROG, &lt;em&gt;M. sookensis&lt;/em&gt; ROG. Ploidy for parental, for F&lt;sub&gt;1&lt;/sub&gt;, and for F&lt;sub&gt;2&lt;/sub&gt; populations are indicated with 2x (diploid) or 4x (neotetraploid) as appropriate; both S&lt;sub&gt;2&lt;/sub&gt; classes are tetraploid. Means are shaded to indicate ploidy as follows: diploids (white), artificial neotetraploids (black), and naturally occurring allotetraploids (gray).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-4.medium.gif" width="369" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-4.medium.gif" width="369" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464063" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Means and 95% confidence intervals for floral traits. IM, M. guttatus IM62; SF, M. nasutus SF; FAN, M. sookensis FAN; and ROG, M. sookensis ROG. Ploidy for parental, for F1, and for F2 populations are indicated with 2x (diploid) or 4x (neotetraploid) as appropriate; both S2 classes are tetraploid. Means are shaded to indicate ploidy as follows: diploids (white), artificial neotetraploids (black), and naturally occurring allotetraploids (gray)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-143196013" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Means and 95% confidence intervals for floral traits. IM, &lt;em&gt;M. guttatus&lt;/em&gt; IM62; SF, &lt;em&gt;M. nasutus&lt;/em&gt; SF; FAN, &lt;em&gt;M. sookensis&lt;/em&gt; FAN; and ROG, &lt;em&gt;M. sookensis&lt;/em&gt; ROG. Ploidy for parental, for F&lt;sub&gt;1&lt;/sub&gt;, and for F&lt;sub&gt;2&lt;/sub&gt; populations are indicated with 2x (diploid) or 4x (neotetraploid) as appropriate; both S&lt;sub&gt;2&lt;/sub&gt; classes are tetraploid. Means are shaded to indicate ploidy as follows: diploids (white), artificial neotetraploids (black), and naturally occurring allotetraploids (gray).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-5.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-5.medium.gif" width="368" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/509/F4/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464063" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Means and 95% confidence intervals for floral traits. IM, <em>M. guttatus</em> IM62; SF, <em>M. nasutus</em> SF; FAN, <em>M. sookensis</em> FAN; and ROG, <em>M. sookensis</em> ROG. Ploidy for parental, for F<sub>1</sub>, and for F<sub>2</sub> populations are indicated with 2x (diploid) or 4x (neotetraploid) as appropriate; both S<sub>2</sub> classes are tetraploid. Means are shaded to indicate ploidy as follows: diploids (white), artificial neotetraploids (black), and naturally occurring allotetraploids (gray).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Our measurement of floral traits reveals three distinct patterns (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS4.pdf">Table S4</a>). First, the two measured <em>M. sookensis</em> lines cluster significantly with diploid and neoautotetraploid <em>M. nasutus</em>, although they are statistically different from each other and <em>M. nasutus</em> for some traits. Second, dominance of <em>M. guttatus</em> floral characters is upheld in the neoallotetraploid F<sub>2</sub> plants and was also observed in diploid hybrids (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS4.pdf">Table S4</a>) (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Fishman <em>et al.</em> 2002</a>). Third, in <em>M. guttatus</em> and F<sub>1</sub> hybrids, ploidy appears to cause an increase in flower size, although the difference is not always significant, whereas autotetraploid <em>M. nasutus</em> have smaller flowers than diploid <em>M. nasutus</em>. Two other patterns were observed from the phenotypic data. First, whereas overall there was little variation in flowering time (as measured by days from germination to date of first flower), the neoautotetraploid lines of <em>M. guttatus</em> (IM-4x) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS4.pdf">Table S4</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>) flowered more than 8 days later than all other lines. We observed that these individuals lingered as seedlings with only cotyledons for an extended period of time before growing true leaves and bolting. Last, in the neoautotetraploid lines, ploidy appears to cause a decrease in pollen viability, although the difference was not statistically significant in the <em>M. nasutus</em> lines (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS4.pdf">Table S4</a>). In contrast, the neoallotetraploid lines had significantly higher pollen viability than did their diploid counterparts; pollen viability in neoallotetraploid lines was equivalent to pollen viability in diploid <em>M. nasutus</em> and <em>M. sookensis</em>, which had the highest pollen viability among all classes.</p></div><div id="sec-11" class="subsection"><h3>Analysis of variance: Homeologous recombination does not significantly contribute to phenotypic variation</h3><p id="p-39">To determine if the low observed levels of homeologous recombination observed in genetic marker data from BC<sub>1N</sub> progeny and S<sub>2</sub> families could result in phenotypic variation with an underlying heritable genetic basis, variance components of neoallotetraploid F<sub>2</sub>s and S<sub>4</sub> families were calculated. In the diploid F<sub>2</sub> population, a significant increase in variation was observed at 10 of 11 traits, according to Levene's test, which is robust to asymmetry of the mean, with the variation seen in diploid F<sub>2</sub>s for tube length being of marginal significance (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In contrast, only one trait out of 11 (PC2) showed significantly elevated levels of variation in the neoallotetraploid F<sub>2</sub> population after accounting for traits in which subdividing the F<sub>2</sub> class was appropriate (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS5.pdf">Table S5</a>; tube length, corolla width, corolla length, lower calyx width, and PC1).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464053/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464053/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Variance, broad-sense heritability (<em>H</em><sup>2</sup>), and results of tests for homogeneity of variance between F<sub>1</sub> and F<sub>2</sub> classes</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">Broad-sense heritability of floral traits and flowering time calculated from the diploid F<sub>2</sub> cross ranged from 0.41 for tube length to 0.72 for flowering time. In contrast, the heritabilities for floral traits and flowering time ranged from −0.87 for flowering time to 0.37 for corolla width in the tetraploid cross. Negative heritabilities indicate an obvious lack of heritable genetic variation, and heritability of traits that do not demonstrate segregational variance is not meaningful. For PC2, the only trait to display a significant level of segregating variation in the tetraploid F<sub>2</sub> cross, the broad-sense heritability was 0.22, whereas it was 0.62 in the diploid F<sub>2</sub> cross (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-42">For the S<sub>4</sub> families, we tested the hypothesis that individuals from a family were more phenotypically similar to each other than they were to a random individual, a relationship that should arise if heritable genetic variation exists for a given trait. For the S<sub>4</sub> families derived from S<sub>1</sub> tetraploids with <em>M. guttatus</em> as the maternal parent, a significant effect of family was found for six of 11 traits (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). For all six traits, the lower bound of the 95% confidence limit was very close to zero. In contrast, for the S<sub>4</sub> families derived from S<sub>1</sub> tetraploids with <em>M. nasutus</em> as the maternal parent, a significant effect of family was not found in any of the 11 traits examined (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). For lower calyx width and stamen length, a significant maternal effect was found in the S<sub>4N</sub> families, but no significant maternal effects were found in the S<sub>4G</sub> families for any trait. Maternal effects were measured by splitting the S<sub>4</sub> families into subfamilies in the penultimate generation and measuring three individuals from each subfamily during the common garden experiment.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464124/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464124/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Variance components, SE, significance at α = 0.05, and percent total variance for S<sub>4</sub> families calculated separately based on original synthetic tetraploid F<sub>1</sub></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-44">The objective of this study was to assess the effects of polyploidization on genetic and phenotypic variation in <em>Mimulus</em>. Using synthetic neoallotetraploid <em>Mimulus</em> derived from <em>M. guttatus</em> and <em>M. nasutus</em>, we report three key findings. First, we used genetic marker data to show that homeologous pairing and recombination are almost negligible in neoallotetraploid <em>Mimulus</em>. We then showed that polyploidization results in large-flowered neoallotetraploid <em>Mimulus</em>, in contrast to naturally occurring <em>M. sookensis</em>. Finally, we demonstrated that the low levels of homeologous recombination do not contribute to significant levels of phenotypic variation, suggesting that polyploidization <em>per se</em> is not a major driver of genetic and phenotypic variation in neoallotetraploid <em>Mimulus</em>.</p><p id="p-45">Analysis of genetic marker data led us to reject models of homeologous chromosome pairing. Although some fragment loss was observed in BC<sub>1N</sub> progeny, the level of fragment loss observed (10 instances) was far below the level expected (56 instances) if homeologous chromosomes regularly pair, and we observed no instances of marker loss in the S<sub>2</sub> progeny. In addition, the preferential pairing rate was found to be significantly different from a model in which <em>p</em> = ⅔ (<em>i.e.</em>, strict homologous pairings) in three of the seven loci examined in the BC<sub>1N</sub> progeny. The binomial tests, replicated <em>G</em>-test, and preferential pairing rate all suggest that although chromosomes may often preferentially pair with their homologous counterpart, it is not a strict association. In the genetic marker data presented here, we cannot distinguish between rare homeologous recombination events causing the loss of one parental genome fragment with replacement from the other fragment, and deletion of one fragment without replacement. It is thus possible that the 10 instances of observed fragment loss are attributable to fragment deletion caused possibly by ectopic or unequal recombination rather than homeologous recombination. Nevertheless, it is still possible to test if fragment loss, regardless of the mechanism, generates a measurable phenotypic effect.</p><div id="sec-13" class="subsection"><h3>Polyploidization results in large-flowered neoallotetraploid <em>Mimulus</em></h3><p id="p-46">Polyploidization can result in noticeable phenotypic effects, as first observed in the tetraploid “<em>gigas</em>” mutants of <em>Oenothera lamackiana</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">DeVries 1915</a> and references therein). We found that neoallotetraploid <em>Mimulus</em> are large-flowered and do not resemble naturally occurring <em>M. sookensis</em>. The large flower size of neoallotetraploid F<sub>1</sub> and F<sub>2</sub> plants suggests that the dominance relationships of <em>M. guttatus</em> and <em>M. nasutus</em> floral loci remain unchanged after polyploidization. Interestingly, there also appears to be a dosage effect of floral loci; neoautotetraploid <em>M. guttatus</em>, which contains four copies of each of the many loci contributing to large flower size, is larger than diploid <em>M. guttatus</em>, whereas neoautotetraploid <em>M. nasutus</em>, which contains four rather than two small-flower alleles at each of the loci contributing to flower size, is smaller than diploid <em>M. nasutus</em>. Unlike in <em>Arabidopsis</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Madlung <em>et al.</em> 2005</a>), induction of polyploidy did not result in phenotypic instability.</p><p id="p-47">It is unlikely that the use of different lines of <em>M. guttatus</em> and <em>M. nasutus</em> would result in small-flowered neoallotetraploid <em>Mimulus</em>. For this to occur, there would have to be alternative flower size alleles in <em>M. guttatus</em> and <em>M. nasutus</em> that interact to create small-flowered F<sub>1</sub> hybrids. Range-wide crossing experiments between eight pairs of <em>M. guttatus</em> and <em>M. nasutus</em> lines always resulted in intermediate to large-flowered F<sub>1</sub> hybrids (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Martin and Willis 2010</a>; N.H. Martin, personal communication), suggesting that differential epistatic interactions between flower size alleles derived from different populations of <em>M. guttatus</em> and <em>M. nasutus</em> could not combine to create small-flowered neoallotetraploids. It is also unlikely that a small-flowered <em>M. guttatus</em> was the progenitor of <em>M. sookensis</em>. Because <em>M. sookensis</em> has recurrently formed throughout its range (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Modliszewski and Willis 2012</a>), a small-flowered <em>M. guttatus</em> would have to form an unlikely mating with small-flowered <em>M. nasutus</em>; it would also have to be present at a fairly high frequency or produce a disproportionate amount of unreduced gametes. Given that the larger flower size of <em>M. guttatus</em> is likely to be the ancestral state of the <em>M. guttatus</em> and <em>M. nasutus</em> based on phylogenetic relationships within <em>Mimulus</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beardsley <em>et al.</em> 2004</a>), and the fact that no small-flowered <em>M. guttatus</em> was observed, this scenario is highly unlikely. These observations, together with our data, suggest that polyploidization of <em>M. guttatus</em> and <em>M. nasutus</em> hybrids will result in large neoallotetraploids regardless of the <em>M. guttatus</em> and <em>M. nasutus</em> lines used, and that <em>M. sookensis</em> was derived from a large-flowered <em>M. guttatus</em>.</p></div><div id="sec-14" class="subsection"><h3>Low levels of homeologous recombination do not significantly enhance phenotypic variation in neoallotetraploid <em>Mimulus</em></h3><p id="p-48">In the segregating diploid F<sub>2</sub> population, all traits exhibited evidence of segregational variance and had significant levels of broad-sense heritability, as is expected with segregation and independent assortment. Analyses of variance in tetraploid F<sub>2</sub>s and S<sub>4</sub> families suggests that fragment loss (if occurring) is not likely to create substantial genetic variation that has a corresponding phenotypic effect. In support of this conclusion, broad-sense heritability was much lower in the tetraploids relative to diploids. Although broad-sense heritability is expected to be lower in tetraploids relative to diploids with similar phenotypic ranges because of the expected decreased variance in tetraploids, in our study the parental neoautotetraploid lines have a larger phenotypic range than diploid parental lines, which should increase the expected genetic variance in F<sub>2</sub>s and thus should increase the estimate of broad-sense heritability.</p><p id="p-49">An additional line of indirect evidence is suggestive of infrequent homeologous pairing in neoallotetraploid <em>Mimulus</em>. The high levels of pollen viability observed in neoallotetraploids, compared to the decreased levels of pollen viability in neoautotetraploids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1/TableS4.pdf">Table S4</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>.), suggest that chromosomes are regularly pairing in bivalents in the neoallotetraploids, but in quadrivalents in the neoautotetraploids. However, it may be that further assessment of pollen viability or germinability may reveal that the difference in viability of pollen between neoautotetraploids and neoallotetraploids is not as great as originally observed. A negative correlation between quadrivalent formation and fertility has been previously observed in wheat and other species (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ozkan <em>et al.</em> 2001</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ramsey and Schemske 2002</a>).</p></div><div id="sec-15" class="subsection"><h3>Implications for phenotypic evolution in polyploids</h3><p id="p-50">Overall, the nuances of the data suggest that there is not rampant pairing of homeologous chromosomes, but that occasionally homeologous chromosomes may pair or fragment loss through another deletion mechanism may be occurring. Low to nonexistent levels of homeologous recombination will slow the process of phenotypic evolution, unlike the evolution of flowering time observed in <em>Brassica napus</em> (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Schranz and Osborn 2000</a>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">2004</a>). In <em>Gossypium</em> allotetraploids, although homeologous recombination has been confirmed, the timing of homeologous recombination events is scattered over the evolutionary history of cotton (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Salmon <em>et al.</em> 2010</a>). A similar phenomenon may be occurring in <em>M. sookensis</em>. Alternate pathways of formation, such as through an outcrossing autotetraploid progenitor rather than a diploid F<sub>1</sub> hybrid, may contribute to an initial augmentation of genetic variation, but if the polyploid lineage is formed from only a few individuals, then homologous pairing will still limit genetic variation in the allopolyploids. At present, it appears that the best approach to determining the genetic mechanisms responsible for the evolution of flower size in <em>M. sookensis</em> is to examine naturally occurring <em>M. sookensis</em> on a genome-wide level for evidence of homeologous recombination or to directly address the genetic basis of flower size via QTL mapping.</p><p id="p-51">Although polyploidization itself may be widespread among plants (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Masterson 1994</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Otto and Whitton 2000</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Blanc and Wolfe 2004</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Cui <em>et al.</em> 2006</a>), the effects of polyploidization as observed in any single polyploid system do not appear to be universally applicable. For example, the rapid fragment loss and cytosine methylation observed in <em>Aegilops-Triticum</em> wheat (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ozkan <em>et al.</em> 2001</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shaked <em>et al.</em> 2001</a>) do not occur in <em>Gossypium</em>, which instead demonstrates biased expression dominance on a genome-wide level (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 2001</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Flagel <em>et al.</em> 2008</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chaudhary <em>et al.</em> 2009</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rapp <em>et al.</em> 2009</a>). Likewise, rapid genomic changes have not been documented in <em>Spartina anglica</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baumel <em>et al.</em> 2002</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ainouche <em>et al.</em> 2004</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Salmon <em>et al.</em> 2005</a>). In artificial <em>Arabidopsis suecica</em>, ∼11% of the genes exhibit a departure from expression additivity, but the phenotype of early generation synthetic <em>A. suecica</em> is quite similar to natural <em>A. suecica</em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wang <em>et al.</em> 2004</a>), whereas the effects of polyploidization on gene expression differ in autotetraploid <em>Arabidopsis</em> lines (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Yu <em>et al.</em> 2010</a>). Additionally, early-generation synthetic allotetraploid <em>Nicotiana tabacum</em> display evidence of loss of 25%–60% of the repetitive fragments analyzed but appear phenotypically similar to domesticated <em>N. tabacum</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Skalická <em>et al.</em> 2005</a>). Thus, it is not clear if fragment loss and expression nonadditivity observed in synthetic allotetraploids result in genetic variation that has a high (and observable) impact on phenotypic evolution. The findings we present here in <em>Mimulus</em> suggest yet another variation on a theme in polyploid evolution. One cannot assume that homeologous recombination or fragment loss will occur and allow for rapid recapitulation of the phenotype observed in natural or crop allopolyploids. Rather, the trajectory of phenotypic evolution may have been slow in the allotetraploid <em>M. sookensis</em> and perhaps also in other polyploid systems.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">This research was funded by a National Science Foundation (NSF) doctoral dissertation improvement grant (0910296; to J.L.M.) and NSF IOS grant (1024966 to J.H.W.). The authors thank John Kelley for discussion of colchicine transformation technique and Mohamed Noor for helpful comments on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supplemental information for raw phenotypic data is available online at <a href="http://dx.doi.org/10.6084/m9.figshare.904927">http://dx.doi.org/10.6084/m9.figshare.904927</a>.</p></li><li class="fn" id="fn-2"><p id="p-2">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008441/-/DC1</a>.</p></li><li class="fn" id="fn-10"><p id="p-53"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-06"><span class="received-label">Received </span>September 6, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-01-17"><span class="accepted-label">Accepted </span>January 17, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Modliszewski and Willis</li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.1" data-doi="10.1111/j.1095-8312.2004.00334.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Spartina anglica</em> C. E. Hubbard: A natural model system for analysing early evolutionary changes that affect allopolyploid genomes.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D82%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2004.00334.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2004.00334.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223808600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.2" data-doi="10.1093/aob/mcp058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anssour</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krügel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharbel</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saluz</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonaventure</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Phenotypic, genetic and genomic consequences of natural and synthetic polyploidization of <em>Nicotiana attenuata</em> and <em>Nicotiana obtusifolia</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">1207</span>–<span class="cit-lpage">1217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcp058%26rft_id%253Dinfo%253Apmid%252F19307190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDMvOC8xMjA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.3" data-doi="10.1093/oxfordjournals.molbev.a004182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baumel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalendar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulman</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Retrotransposons and genomic stability in populations of the young allopolyploid species <em>Spartina anglica</em> C.E. Hubbard (<em>Poaceae</em>).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1218</span>–<span class="cit-lpage">1227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBaumel%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D1218%26rft.epage%253D1227%26rft.atitle%253DRetrotransposons%2Band%2BGenomic%2BStability%2Bin%2BPopulations%2Bof%2Bthe%2BYoung%2BAllopolyploid%2BSpecies%2BSpartina%2Banglica%2BC.E.%2BHubbard%2B%2528Poaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a004182%26rft_id%253Dinfo%253Apmid%252F12140233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjE5LzgvMTIxOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.4" data-doi="10.3120/0024-9637-59.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modliszewski</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweigart</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganders</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mimulus sookensis (Phrymaceae), a new allotetraploid species derived from <em>Mimulus guttatus</em> and <em>Mimulus nasutus</em>.</span> <abbr class="cit-jnl-abbrev">Madrono</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMadrono%26rft.volume%253D59%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.3120%252F0024-9637-59.1.29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3120/0024-9637-59.1.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.5" data-doi="10.3732/ajb.91.3.474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beardsley</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoenig</span>  <span class="cit-name-given-names">P. S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittall</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Patterns of evolution in western North American <em>Mimulus</em> (Phrymaceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DBeardsley%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D474%26rft.epage%253D489%26rft.atitle%253DPatterns%2Bof%2Bevolution%2Bin%2Bwestern%2BNorth%2BAmerican%2BMimulus%2B%2528Phrymaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.91.3.474%26rft_id%253Dinfo%253Apmid%252F21653403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjkxLzMvNDc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.6" data-doi="10.1093/aob/"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bino</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Vries</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraak</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Pijlen</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Flow cytometric determination of nuclear replication stages in tomato seeds during priming and germination.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DBINO%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D231%26rft.epage%253D236%26rft.atitle%253DFlow%2BCytometric%2BDetermination%2Bof%2BNuclear%2BReplication%2BStages%2Bin%2BTomato%2BSeeds%2Bduring%2BPriming%2Band%2BGermination%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252F%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6IjY5LzMvMjMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blakeslee</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farnham</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1923</span> <span class="cit-article-title">Inheritance in tetraploid daturas.</span> <abbr class="cit-jnl-abbrev">Bot. Gaz.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BGaz.%26rft.volume%253D76%26rft.spage%253D329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.8" data-doi="10.1105/tpc.021345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.021345%26rft_id%253Dinfo%253Apmid%252F15208399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMTY2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burnham</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-source">Discussions in Cytogenetics</span>, <span class="cit-publ-name">Burgess Publishing Company</span>, <span class="cit-publ-loc">Minneapolis, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.10" data-doi="10.1534/genetics.109.102608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaudhary</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stupar</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reciprocal silencing, transcriptional bias and functional divergence of homeologs in polyploid cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102608%26rft_id%253Dinfo%253Apmid%252F19363125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzIvNTAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.11" data-doi="10.1146/annurev.arplant.58.032806.103835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic and epigenetic mechanisms for gene expression and phenotypic variation in plant polyploids.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DChen%26rft.auinit1%253DZ.%2BJ.%26rft.volume%253D58%26rft.spage%253D377%26rft.epage%253D406%26rft.atitle%253DGenetic%2Band%2Bepigenetic%2Bmechanisms%2Bfor%2Bgene%2Bexpression%2Band%2Bphenotypic%2Bvariation%2Bin%2Bplant%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.58.032806.103835%26rft_id%253Dinfo%253Apmid%252F17280525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.58.032806.103835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17280525&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247703600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.12" data-doi="10.1073/pnas.1112041109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symonds</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz da Silva</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavrodiev</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extensive chromosomal variation in a recently formed natural allopolyploid species, <em>Tragopogon miscellus</em> (asteraceae).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112041109%26rft_id%253Dinfo%253Apmid%252F22228301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTE3NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.13" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.14" data-doi="10.1105/tpc.12.9.1551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes-Davis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Phenotypic instability and rapid gene silencing in newly formed <em>Arabidopsis</em> allotetraploids.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D1551%26rft.epage%253D1568%26rft.atitle%253DPhenotypic%2BInstability%2Band%2BRapid%2BGene%2BSilencing%2Bin%2BNewly%2BFormed%2BArabidopsis%2BAllotetraploids%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.12.9.1551%26rft_id%253Dinfo%253Apmid%252F11006331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEyLzkvMTU1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Speciation</span>, <span class="cit-publ-name">Sinauer Associates Inc</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crane</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darlington</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1932</span> <span class="cit-article-title">Chromatid segregation in tetraploid <em>Rubus</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D129%26rft.spage%253D869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.17" data-doi="10.1101/gr.4825606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leebens-Mack</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Widespread genome duplications throughout the history of flowering plants.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">738</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4825606%26rft_id%253Dinfo%253Apmid%252F16702410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzM4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.18" data-doi="10.1534/genetics.105.043042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Curole</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of preferential pairing rates in second-generation autotetraploid pacific oysters (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">859</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.043042%26rft_id%253Dinfo%253Apmid%252F15965239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvODU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeVries</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1915</span> <span class="cit-article-title"><em>Oenothera gigas nanella</em>, a mendelian mutant.</span> <abbr class="cit-jnl-abbrev">Bot. Gaz.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BGaz.%26rft.volume%253D60%26rft.spage%253D337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobzhansky</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-source">Genetics and the Origin of Species</span>, <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.21" data-doi="10.1007/BF00306563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Autotetraploid gene segregation.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D43%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00306563%26rft_id%253Dinfo%253Apmid%252F24424975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00306563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24424975&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.22" data-doi="10.1146/annurev.genet.42.110807.091524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolutionary genetics of genome merger and doubling in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DDoyle%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D42%26rft.spage%253D443%26rft.epage%253D461%26rft.atitle%253DEvolutionary%2Bgenetics%2Bof%2Bgenome%2Bmerger%2Band%2Bdoubling%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091524%26rft_id%253Dinfo%253Apmid%252F18983261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983261&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1941</span> <span class="cit-article-title">The theoretical consequences of polyploid inheritance for the mid style form of <em>Lythrum salicaria</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BHum.%2BGenet.%26rft.volume%253D11%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.24" data-doi="10.1554/0014-3820(2001)055[1932:EFDMIC]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evidence for Dobzhansky-Muller incompatibilities contributing to the sterility of hybrids between <em>Mimulus guttatus</em> and <em>M. nasutus</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1932</span>–<span class="cit-lpage">1942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFishman%26rft.auinit1%253DL.%26rft.volume%253D55%26rft.issue%253D10%26rft.spage%253D1932%26rft.epage%253D1942%26rft.atitle%253DEvidence%2Bfor%2BDobzhansky-Muller%2Bincompatibilites%2Bcontributing%2Bto%2Bthe%2Bsterility%2Bof%2Bhybrids%2Bbetween%2BMimulus%2Bguttatus%2Band%2BM.%2Bnasutus.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1932%253AEFDMIC%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11761055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1932:EFDMIC]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11761055&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172308600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.25" data-doi="10.1554/0014-3820(2002)056[2138:MQTLUF]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Minor quantitative trait loci underlie floral traits associated with mating system divergence in <em>Mimulus</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2138</span>–<span class="cit-lpage">2155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFishman%26rft.auinit1%253DL.%26rft.volume%253D56%26rft.issue%253D11%26rft.spage%253D2138%26rft.epage%253D2155%26rft.atitle%253DMinor%2Bquantitative%2Btrait%2Bloci%2Bunderlie%2Bfloral%2Btraits%2Bassociated%2Bwith%2Bmating%2Bsystem%2Bdivergence%2Bin%2BMimulus.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B2138%253AMQTLUF%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12487345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[2138:MQTLUF]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12487345&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179683100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.26" data-doi="10.1186/1741-7007-6-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Duplicate gene expression in allopolyploid <em>Gossypium</em> reveals two temporally distinct phases of expression evolution.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DDuplicate%2Bgene%2Bexpression%2Bin%2Ballopolyploid%2BGossypium%2Breveals%2Btwo%2Btemporally%2Bdistinct%2Bphases%2Bof%2Bexpression%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-6-16%26rft_id%253Dinfo%253Apmid%252F18416842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-6-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416842&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.27" data-doi="10.1105/tpc.107.054346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaeta</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iniguez-Luy</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic changes in resynthesized <em>Brassica napus</em> and their effect on gene expression and phenotype.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3403</span>–<span class="cit-lpage">3417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.054346%26rft_id%253Dinfo%253Apmid%252F18024568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMS8zNDAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstel</span>  <span class="cit-name-given-names">D. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Segregation in new allopolyploids of <em>Nicotiana</em>. I. Comparison of 6x (<em>N. tabacum</em> × <em>tomentosiformis</em>) and 6x (<em>N. tabacum</em> × <em>otophora</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1723</span>–<span class="cit-lpage">1734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGerstel%26rft.auinit1%253DD.%2BU.%26rft.volume%253D45%26rft.issue%253D12%26rft.spage%253D1723%26rft.epage%253D1734%26rft.atitle%253DSEGREGATION%2BIN%2BNEW%2BALLOPOLYPLOIDS%2BOF%2BNICOTIANA.%2BI.%2BCOMPARISON%2BOF%2B6x%2B%2528N.%2BTABACUM%2Bx%2BTOMENTOSIFORMIS%2529%2BAND%2B6x%2B%2528N.%2BTABACUM%2Bx%2BOTOPHORA%2529%26rft_id%253Dinfo%253Apmid%252F17248031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czoxMDoiNDUvMTIvMTcyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstel</span>  <span class="cit-name-given-names">D. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">Segregation in new allopolyploids of <em>Gossypium</em>. II. Tetraploid combinations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGerstel%26rft.auinit1%253DD.%2BU.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D783%26rft.epage%253D797%26rft.atitle%253DSEGREGATION%2BIN%2BNEW%2BALLOPOLYPLOIDS%2BOF%2BGOSSYPIUM.%2BII.%2BTETRAPLOID%2BCOMBINATIONS%26rft_id%253Dinfo%253Apmid%252F17247734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0Mi82Lzc4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.30" data-doi="10.1101/SQB.1958.023.01.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstel</span>  <span class="cit-name-given-names">D. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Segregation of synthetic amphiploids in <em>Gossypium</em> and <em>Nicotiana</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DGerstel%26rft.auinit1%253DD.%2BU.%26rft.volume%253D23%26rft.issue%253D0%26rft.spage%253D225%26rft.epage%253D237%26rft.atitle%253DSegregation%2Bof%2BSynthetic%2BAmphiploids%2Bin%2BGossypium%2Band%2BNicotiana%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1958.023.01.025%26rft_id%253Dinfo%253Apmid%252F13635558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjIzLzAvMjI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.31" data-doi="10.1098/rspb.1914.0035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">On the genetics of tetraploid plants in <em>Primula sinensis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond., B</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DGregory%26rft.auinit1%253DR.%2BP.%26rft.volume%253D87%26rft.issue%253D597%26rft.spage%253D484%26rft.epage%253D492%26rft.atitle%253DOn%2Bthe%2BGenetics%2Bof%2BTetraploid%2BPlants%2Bin%2BPrimula%2Bsinensis%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1914.0035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJyb3lwcnNiIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny81OTcvNDg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.32" data-doi="10.1007/BF02984197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Theoretical genetics of autopolyploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D22%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200610100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearns</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source">Techniques for Pollination Biologists</span>, <span class="cit-publ-name">University Press of Colorado</span>, <span class="cit-publ-loc">Niwot, CO</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.34" data-doi="10.1046/j.1365-294x.1998.00328.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Polymorphic microsatellite loci in <em>Mimulus guttatus</em> and related species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D7%26rft.spage%253D769%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.1998.00328.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.1998.00328.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074236300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.35" data-doi="10.1371/journal.pone.0003353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matyasek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid chromosome evolution in recently formed polyploids in <em>Tragopogon</em> (Asteraceae).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLim%26rft.auinit1%253DK.%2BY.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3353%26rft.epage%253De3353%26rft.atitle%253DRapid%2Bchromosome%2Bevolution%2Bin%2Brecently%2Bformed%2Bpolyploids%2Bin%2BTragopogon%2B%2528Asteraceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003353%26rft_id%253Dinfo%253Apmid%252F18843372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18843372&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphrey</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Comparative cyto-genetic studies of tetraploid tomatoes from different origins.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLindstrom%26rft.auinit1%253DE.%2BW.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D193%26rft.epage%253D209%26rft.atitle%253DCOMPARATIVE%2BCYTO-GENETIC%2BSTUDIES%2BOF%2BTETRAPLOID%2BTOMATOES%2BFROM%2BDIFFERENT%2BORIGINS%26rft_id%253Dinfo%253Apmid%252F17246688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxOC8zLzE5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.37" data-doi="10.1007/BF02861164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1945</span> <span class="cit-article-title">Gene segregation in autotetraploids.</span> <abbr class="cit-jnl-abbrev">Bot. Rev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BRev.%26rft.volume%253D11%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02861164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02861164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1945YF11500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.38" data-doi="10.1007/BF02872438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1958</span> <span class="cit-article-title">Gene segregation in autotetraploids. II.</span> <abbr class="cit-jnl-abbrev">Bot. Rev.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot.%2BRev.%26rft.volume%253D24%26rft.spage%253D318%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02872438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02872438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mergeai</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Polyploid formation in cotton is not accompanied by rapid genomic changes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLiu%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D321%26rft.epage%253D330%26rft.atitle%253DPolyploid%2Bformation%2Bin%2Bcotton%2Bis%2Bnot%2Baccompanied%2Bby%2Brapid%2Bgenomic%2Bchanges.%26rft_id%253Dinfo%253Apmid%252F11444689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11444689&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.40" data-doi="10.1111/j.1365-313X.2004.02297.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madlung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagochi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic changes in synthetic <em>Arabidopsis</em> polyploids.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMadlung%26rft.auinit1%253DA.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D221%26rft.epage%253D230%26rft.atitle%253DGenomic%2Bchanges%2Bin%2Bsynthetic%2BArabidopsis%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02297.x%26rft_id%253Dinfo%253Apmid%252F15634199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02297.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15634199&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225948400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.41" data-doi="10.1098/rstb.2010.0030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Geographical variation in postzygotic isolation and its genetic basis within and between two <em>Mimulus</em> species.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">2469</span>–<span class="cit-lpage">2478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DMartin%26rft.auinit1%253DN.%2BH.%26rft.volume%253D365%26rft.issue%253D1552%26rft.spage%253D2469%26rft.epage%253D2478%26rft.atitle%253DGeographical%2Bvariation%2Bin%2Bpostzygotic%2Bisolation%2Band%2Bits%2Bgenetic%2Bbasis%2Bwithin%2Band%2Bbetween%2Btwo%2BMimulus%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2010.0030%26rft_id%253Dinfo%253Apmid%252F20643736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU1Mi8yNDY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.42" data-doi="10.1126/science.264.5157.421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masterson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Stomatal size in fossil plants: Evidence for polyploidy in majority of angiosperms.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">264</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMasterson%26rft.auinit1%253DJ.%26rft.volume%253D264%26rft.issue%253D5157%26rft.spage%253D421%26rft.epage%253D424%26rft.atitle%253DStomatal%2BSize%2Bin%2BFossil%2BPlants%253A%2BEvidence%2Bfor%2BPolyploidy%2Bin%2BMajority%2Bof%2BAngiosperms%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.264.5157.421%26rft_id%253Dinfo%253Apmid%252F17836906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjQvNTE1Ny80MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.43" data-doi="10.1007/BF02982205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-article-title">Reductional and equational separation of the chromosomes in bivalents and multivalents.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D30%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.44" data-doi="10.1007/BF02982683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">Segregation and linkage in autotetraploids.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D32%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200612400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.45" data-doi="10.1534/genetics.112.139295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsushita</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madlung</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allopolyploidization lays the foundation for evolution of distinct populations: Evidence from analysis of synthetic <em>Arabidopsis</em> allohexaploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139295%26rft_id%253Dinfo%253Apmid%252F22426881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzIvNTM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.46" data-doi="10.1111/j.1365-294X.2012.05706.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Modliszewski</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allotetraploid <em>Mimulus sookensis</em> are highly interfertile despite independent origins.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5280</span>–<span class="cit-lpage">5298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DModliszewski%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D21%26rft.issue%253D21%26rft.spage%253D5280%26rft.epage%253D5298%26rft.atitle%253DAllotetraploid%2BMimulus%2Bsookensis%2Bare%2Bhighly%2Binterfertile%2Bdespite%2Bindependent%2Borigins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05706.x%26rft_id%253Dinfo%253Apmid%252F22831280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05706.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22831280&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310065600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.47" data-doi="10.1086/279426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">A new mode of segregation in Gregory’s tetraploid Primulas.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D48%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1086%252F279426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/279426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200856200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.48" data-doi="10.1111/j.1558-5646.2010.01208.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oswald</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuismer</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Neopolyploidy and diversification in <em>Heuchera grossulariifolia</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DOswald%26rft.auinit1%253DB.%2BP.%26rft.volume%253D65%26rft.issue%253D6%26rft.spage%253D1667%26rft.epage%253D1679%26rft.atitle%253DNeopolyploidy%2Band%2Bdiversification%2Bin%2BHeuchera%2Bgrossulariifolia.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01208.x%26rft_id%253Dinfo%253Apmid%252F21143472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01208.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21143472&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291270300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.49" data-doi="10.1146/annurev.genet.34.1.401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitton</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Polyploid incidence and evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D401%26rft.epage%253D437%26rft.atitle%253DPOLYPLOID%2BINCIDENCE%2BAND%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.34.1.401%26rft_id%253Dinfo%253Apmid%252F11092833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.34.1.401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11092833&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166299400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.50" data-doi="10.1105/tpc.13.8.1735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Allopolyploidy-induced rapid genome evolution in the wheat (<em>Aegilops</em>-<em>Triticum</em>) group.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1735</span>–<span class="cit-lpage">1747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DOzkan%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1735%26rft.epage%253D1747%26rft.atitle%253DAllopolyploidy-Induced%2BRapid%2BGenome%2BEvolution%2Bin%2Bthe%2BWheat%2B%2528Aegilops-Triticum%2529%2BGroup%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1735%26rft_id%253Dinfo%253Apmid%252F11487689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTczNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-web" id="cit-4.3.509.51"><div class="cit-metadata unstructured">R Development Core Team, 2010 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org">http://www.R-project.org</a>. Accessed: August 31, 2012.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.52" data-doi="10.1146/annurev.ecolsys.33.010802.150437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Neopolyploidy in flowering plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.33.010802.150437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.33.010802.150437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000180007000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic expression dominance in allopolyploids.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D7%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.54" data-doi="10.1111/j.1365-294X.2005.02488.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic and epigenetic consequences of recent hybridization and polyploidy in <em>Spartina</em> (Poaceae).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1163</span>–<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D1163%26rft.epage%253D1175%26rft.atitle%253DGenetic%2Band%2Bepigenetic%2Bconsequences%2Bof%2Brecent%2Bhybridization%2Band%2Bpolyploidy%2Bin%2BSpartina%2B%2528Poaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02488.x%26rft_id%253Dinfo%253Apmid%252F15773943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02488.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15773943&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227721900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.55" data-doi="10.1111/j.1469-8137.2009.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeologous nonreciprocal recombination in polyploid cotton.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D134%26rft.atitle%253DHomoeologous%2Bnonreciprocal%2Brecombination%2Bin%2Bpolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03093.x%26rft_id%253Dinfo%253Apmid%252F19925554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19925554&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.56" data-doi="10.1007/BF02984385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sansome</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Chromatid segregation in <em>Solanum lycopersicum</em>.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D27%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.57" data-doi="10.1093/jhered/91.3.242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schranz</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Novel flowering time variation in the resynthesized polyploid <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DSchranz%26rft.auinit1%253DM.%2BE.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D242%26rft.epage%253D246%26rft.atitle%253DNovel%2Bflowering%2Btime%2Bvariation%2Bin%2Bthe%2Bresynthesized%2Bpolyploid%2BBrassica%2Bnapus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F91.3.242%26rft_id%253Dinfo%253Apmid%252F10833052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkxLzMvMjQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.58" data-doi="10.3732/ajb.91.2.174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schranz</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">De novo variation in life-history traits and responses to growth conditions of resynthesized polyploid <em>Brassica napus</em> (Brassicaceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DSchranz%26rft.auinit1%253DM.%2BE.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D183%26rft.atitle%253DDe%2Bnovo%2Bvariation%2Bin%2Blife-history%2Btraits%2Band%2Bresponses%2Bto%2Bgrowth%2Bconditions%2Bof%2Bresynthesized%2Bpolyploid%2BBrassica%2Bnapus%2B%2528Brassicaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.91.2.174%26rft_id%253Dinfo%253Apmid%252F21653373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjkxLzIvMTc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.59" data-doi="10.1105/tpc.13.8.1749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaked</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashkush</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Sequence elimination and cytosine methylation are rapid and reproducible responses of the genome to wide hybridization and allopolyploidy in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DShaked%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1749%26rft.epage%253D1759%26rft.atitle%253DSequence%2BElimination%2Band%2BCytosine%2BMethylation%2BAre%2BRapid%2Band%2BReproducible%2BResponses%2Bof%2Bthe%2BGenome%2Bto%2BWide%2BHybridization%2Band%2BAllopolyploidy%2Bin%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1749%26rft_id%253Dinfo%253Apmid%252F11487690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTc0OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.60" data-doi="10.1111/j.1469-8137.2004.01297.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skalická</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">K. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matyasek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Preferential elimination of repeated DNA sequences from the paternal, <em>Nicotiana tomentosiformis</em> genome donor of a synthetic, allotetraploid tobacco.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSkalick%25C3%25A1%26rft.auinit1%253DK.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D291%26rft.epage%253D303%26rft.atitle%253DPreferential%2Belimination%2Bof%2Brepeated%2BDNA%2Bsequences%2Bfrom%2Bthe%2Bpaternal%252C%2BNicotiana%2Btomentosiformis%2Bgenome%2Bdonor%2Bof%2Ba%2Bsynthetic%252C%2Ballotetraploid%2Btobacco.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2004.01297.x%26rft_id%253Dinfo%253Apmid%252F15760371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2004.01297.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15760371&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227390500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sokal</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohlf</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">Biometry</span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">W.H. Freeman and Company</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.62" data-doi="10.1046/j.1469-8137.2003.00948.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Advances in the study of polyploidy since plant speciation.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D161%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1469-8137.2003.00948.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1469-8137.2003.00948.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187417700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.509.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stebbins</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-source">Chromosomal Evolution in Higher Plants</span>, <span class="cit-publ-name">Addison-Wesley Publishing Company</span>, <span class="cit-publ-loc">Reading, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.64" data-doi="10.1534/genetics.107.085027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stift</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berenos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuperus</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>  <span class="cit-name-given-names">P. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segregation models for disomic, tetrasomic, and intermediate inheritance in tetraploids: a general procedure applied to <em>Rorippa</em> (yellow cress) microsatellite data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">2113</span>–<span class="cit-lpage">2123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085027%26rft_id%253Dinfo%253Apmid%252F18689891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS80LzIxMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweigart</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patterns of nucleotide variation and reproductive isolation between a <em>Mimulus</em> allotetraploid and its progenitor species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2089</span>–<span class="cit-lpage">2100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DYang%26rft.auinit1%253DS.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D2089%26rft.epage%253D2100%26rft.atitle%253DPatterns%2Bof%2Bnucleotide%2Bvariation%2Band%2Breproductive%2Bisolation%2Bbetween%2Ba%2BMimulus%2Ballotetraploid%2Band%2Bits%2Bprogenitor%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18312552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18312552&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254953900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sybenga</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Preferential pairing estimates from multivalent frequencies in tetraploids.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSybenga%26rft.auinit1%253DJ.%26rft.volume%253D37%26rft.issue%253D6%26rft.spage%253D1045%26rft.epage%253D1055%26rft.atitle%253DPreferential%2Bpairing%2Bestimates%2Bfrom%2Bmultivalent%2Bfrequencies%2Bin%2Btetraploids.%26rft_id%253Dinfo%253Apmid%252F18470143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470143&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.67" data-doi="10.1111/j.1469-8137.2010.03182.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szadkowski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huteau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lodé</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huneau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The first meiosis of resynthesized <em>Brassica napus</em>, a genome blender.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSzadkowski%26rft.auinit1%253DE.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D102%26rft.epage%253D112%26rft.atitle%253DThe%2Bfirst%2Bmeiosis%2Bof%2Bresynthesized%2BBrassica%2Bnapus%252C%2Ba%2Bgenome%2Bblender.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03182.x%26rft_id%253Dinfo%253Apmid%252F20149113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03182.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20149113&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.68" data-doi="10.1534/genetics.104.027896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madlung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Stochastic and epigenetic changes of gene expression in <em>Arabidopsis</em> polyploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1961</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D167%26rft.issue%253D4%26rft.spage%253D1961%26rft.epage%253D1973%26rft.atitle%253DStochastic%2Band%2BEpigenetic%2BChanges%2Bof%2BGene%2BExpression%2Bin%2BArabidopsis%2BPolyploids%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027896%26rft_id%253Dinfo%253Apmid%252F15342533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny80LzE5NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.69" data-doi="10.1534/genetics.105.047894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomewide nonadditive gene regulation in <em>Arabidopsis</em> allotetraploids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047894%26rft_id%253Dinfo%253Apmid%252F16172500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNTA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallo-Meagher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littel</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A general polyploid model for analyzing gene segregation in outcrossing tetraploid species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D869%26rft.epage%253D882%26rft.atitle%253DA%2BGeneral%2BPolyploid%2BModel%2Bfor%2BAnalyzing%2BGene%2BSegregation%2Bin%2BOutcrossing%2BTetraploid%2BSpecies%26rft_id%253Dinfo%253Apmid%252F11606559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvODY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.509.71" data-doi="10.1073/pnas.1000852107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haberer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattei</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K. F. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Impact of natural genetic variation on the transcriptome of autotetraploid <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">17809</span>–<span class="cit-lpage">17814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1000852107%26rft_id%253Dinfo%253Apmid%252F20876110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQxLzE3ODA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/509.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/497" title="Reduced rDNA Copy Number Does Not Affect “Competitive” Chromosome Pairing in XYY Males of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/523" title="The Yeast Ess1 Prolyl Isomerase Controls Swi6 and Whi5 Nuclear Localization" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ufeu68LdOJ02j3g-N3OK2HS1eQEo5Bc6kQ2lyhtaTeg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/509" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/509" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/509" class="active" data-icon-position="" data-hide-link-title="0">Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid Mimulus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459820" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eobyyZujSR7EeBSH6VZYHUuhbclmQzoGuw6RPfq1t0M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/509.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/509" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JQ1n7jbRyI32Q4y_kxO1EsCsozWEyW-nREdWEY1Ebho" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/509" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/509.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459820" id="node459820--2" data-pisa="ggg;4/3/509" data-pisa-master="ggg;g3.113.008441" data-apath="/ggg/4/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer L. Modliszewski</span> and <span class="highwire-citation-author" data-delta="1">John H. Willis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008441 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459820--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer L. Modliszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BModliszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Modliszewski%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BL.%2BModliszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jenmod@live.unc.edu" class="" data-icon-position="" data-hide-link-title="0">jenmod@live.unc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John H. Willis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWillis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWillis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/509" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/509&amp;text=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/509&amp;t=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/509&amp;title=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459820/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459820" id="node459820--3" data-pisa="ggg;4/3/509" data-pisa-master="ggg;g3.113.008441" data-apath="/ggg/4/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Near-Absent Levels of Segregational Variation Suggest Limited Opportunities for the Introduction of Genetic Variation Via Homeologous Chromosome Pairing in Synthetic Neoallotetraploid <em>Mimulus</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jennifer L. Modliszewski</span> and <span class="highwire-citation-author" data-delta="1">John H. Willis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-522;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008441 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459820--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jennifer L. Modliszewski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BL.%2BModliszewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Modliszewski%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BL.%2BModliszewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jenmod@live.unc.edu" class="" data-icon-position="" data-hide-link-title="0">jenmod@live.unc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John H. Willis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, Duke University, Durham, North Carolina 27708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BH.%2BWillis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BH.%2BWillis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=509&amp;rft.epage=522&amp;rft.atitle=Near-Absent%20Levels%20of%20Segregational%20Variation%20Suggest%20Limited%20Opportunities%20for%20the%20Introduction%20of%20Genetic%20Variation%20Via%20Homeologous%20Chromosome%20Pairing%20in%20Synthetic%20Neoallotetraploid%20Mimulus&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Modliszewski%2C+Jennifer+L.&amp;rft.au=Willis%2C+John+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459820/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459820/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459820/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459820/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459820/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459820/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459820/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459820/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459820/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459820/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459820/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459820/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/509" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/509.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/509.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/509.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/509.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
