<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/539" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/539.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/539.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/539.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/3/539/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/539.atom" />
<meta name="HW.pisa" content="ggg;4/3/539" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" />
<meta name="DC.Identifier" content="10.1534/g3.113.010025" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Hofheinz and Frisch. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Ridge regression with heteroscedastic marker variances provides an alternative to Bayesian genome-wide prediction methods. Our objectives were to suggest new methods to determine marker-specific shrinkage factors for heteroscedastic ridge regression and to investigate their properties with respect to computational efficiency and accuracy of estimated effects. We analyzed published data sets of maize, wheat, and sugar beet as well as simulated data with the new methods. Ridge regression with shrinkage factors that were proportional to single-marker analysis of variance estimates of variance components ( i.e. , RRWA) was the fastest method. It required computation times of less than 1 sec for medium-sized data sets, which have dimensions that are common in plant breeding. A modification of the expectation-maximization algorithm that yields heteroscedastic marker variances ( i.e. , RMLV) resulted in the most accurate marker effect estimates. It outperformed the homoscedastic ridge regression approach for best linear unbiased prediction in particular for situations with high marker density and strong linkage disequilibrium along the chromosomes, a situation that occurs often in plant breeding populations. We conclude that the RRWA and RMLV approaches provide alternatives to the commonly used Bayesian methods, in particular for applications in which computational feasibility or accuracy of effect estimates are important, such as detection or functional analysis of genes or planning crosses." />
<meta name="DC.Contributor" content="Nina Hofheinz" />
<meta name="DC.Contributor" content="Matthias Frisch" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ridge regression with heteroscedastic marker variances provides an alternative to Bayesian genome-wide prediction methods. Our objectives were to suggest new methods to determine marker-specific shrinkage factors for heteroscedastic ridge regression and to investigate their properties with respect to computational efficiency and accuracy of estimated effects. We analyzed published data sets of maize, wheat, and sugar beet as well as simulated data with the new methods. Ridge regression with shrinkage factors that were proportional to single-marker analysis of variance estimates of variance components (&lt;i&gt;i.e.&lt;/i&gt;, RRWA) was the fastest method. It required computation times of less than 1 sec for medium-sized data sets, which have dimensions that are common in plant breeding. A modification of the expectation-maximization algorithm that yields heteroscedastic marker variances (&lt;i&gt;i.e.&lt;/i&gt;, RMLV) resulted in the most accurate marker effect estimates. It outperformed the homoscedastic ridge regression approach for best linear unbiased prediction in particular for situations with high marker density and strong linkage disequilibrium along the chromosomes, a situation that occurs often in plant breeding populations. We conclude that the RRWA and RMLV approaches provide alternatives to the commonly used Bayesian methods, in particular for applications in which computational feasibility or accuracy of effect estimates are important, such as detection or functional analysis of genes or planning crosses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/539" />
<meta name="citation_id" content="4/3/539" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/539" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/539.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/539.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/539.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.010025" />
<meta name="citation_pmid" content="24449687" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="539" />
<meta name="citation_lastpage" content="546" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nina Hofheinz" />
<meta name="citation_author_institution" content="Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany" />
<meta name="citation_author" content="Matthias Frisch" />
<meta name="citation_author_institution" content="Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany" />
<meta name="citation_author_email" content="matthias.frisch@uni-giessen.de" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H. J. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize.;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_issue=2;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Hofheinz;citation_author=D. Borchardt;citation_author=K. Weissleder;citation_author=M. Frisch;citation_title=Genome-based prediction of test cross performance in two subsequent breeding cycles.;citation_pages=1639-1645;citation_volume=125;citation_year=2012;citation_issue=8;citation_pmid=22814724;citation_doi=10.1007/s00122-012-1940-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. Kärkkäinen;citation_author=M. J. Sillanpää;citation_title=Back to basics for Bayesian model building in genomic selection.;citation_volume=191;citation_year=2012;citation_pmid=22554888;citation_doi=10.1534/genetics.112.139014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Kumar;citation_author=D. Chagné;citation_author=M. C. A. M. Bink;citation_author=R. K. Volz;citation_author=C. Withworth;citation_title=Genomic selection for fruit quality traits in apple (MalusÃdomestica Borkh.).;citation_pages=e36674-e36674;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22574211;citation_doi=10.1371/journal.pone.0036674" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Li;citation_author=A. K. Lühmann;citation_author=K. Weißleder;citation_author=B. Stich;citation_title=Genome-wide distribution of genetic diversity and linkage disequilibrium in elite sugar beet germplasm.;citation_pages=484-484;citation_volume=12;citation_year=2011;citation_pmid=21970685;citation_doi=10.1186/1471-2164-12-484" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Misztal;citation_author=L. R. Schaeffer;citation_title=Nonlinear Model for Describing Convergence of Iterative Methods of Variance Component Estimation;citation_pages=2209-2213;citation_volume=69;citation_year=1986;citation_issue=8;citation_doi=10.3168/jds.S0022-0302(86)80655-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian Linear Regression Package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manés;citation_title=Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H.-P. Piepho;citation_title=Ridge Regression and Extensions for Genomewide Selection in Maize;citation_pages=1165-1176;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.10.0595" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H.-P. Piepho;citation_author=J. O. Ogutu;citation_author=T. Schulz-Streeck;citation_author=B. Estaghvirou;citation_author=A. Gordillo;citation_title=Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.;citation_pages=1093-1104;citation_volume=52;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Variance Components;citation_author=S. R. Searle;citation_author=G. Casella;citation_author=C. E. McCulloch;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Shen;citation_author=M. Alam;citation_author=F. Fikse;citation_author=L. Rönnegård;citation_title=A novel generalized ridge regression method for quantitative genetics.;citation_volume=193;citation_year=2013;citation_pmid=23335338;citation_doi=10.1534/genetics.112.146720" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. K. Shepherd;citation_author=T. H. E. Meuwissen;citation_author=J. A. Wooliams;citation_title=Genomic selection and complex trait prediction using a fast EM algorithm applied to genome-wide markers.;citation_pages=529-529;citation_volume=11;citation_year=2010;citation_pmid=20969788;citation_doi=10.1186/1471-2105-11-529" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Stich;citation_author=A. E. Melchinger;citation_author=M. Frisch;citation_author=H. P. Maurer;citation_author=M. Heckenberger;citation_title=Linkage disequilibrium in European elite maize germplasm investigated with SSRs.;citation_pages=723-730;citation_volume=111;citation_year=2005;citation_issue=4;citation_pmid=15997389;citation_doi=10.1007/s00122-005-2057-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Wimmer;citation_author=C. Lehermeier;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=Y. Wang;citation_title=Genome-wide prediction of traits with different genetic architecture through efficient variable selection.;citation_volume=195;citation_year=2013;citation_pmid=23934883;citation_doi=10.1534/genetics.113.150078" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Ridge regression with heteroscedastic marker variances provides an alternative to Bayesian genome-wide prediction methods. Our objectives were to suggest new methods to determine marker-specific shrinkage factors for heteroscedastic ridge regression and to investigate their properties with respect to computational efficiency and accuracy of estimated effects. We analyzed published data sets of maize, wheat, and sugar beet as well as simulated data with the new methods. Ridge regression with shrinkage factors that were proportional to single-marker analysis of variance estimates of variance components ( i.e. , RRWA) was the fastest method. It required computation times of less than 1 sec for medium-sized data sets, which have dimensions that are common in plant breeding. A modification of the expectation-maximization algorithm that yields heteroscedastic marker variances ( i.e. , RMLV) resulted in the most accurate marker effect estimates. It outperformed the homoscedastic ridge regression approach for best linear unbiased prediction in particular for situations with high marker density and strong linkage disequilibrium along the chromosomes, a situation that occurs often in plant breeding populations. We conclude that the RRWA and RMLV approaches provide alternatives to the commonly used Bayesian methods, in particular for applications in which computational feasibility or accuracy of effect estimates are important, such as detection or functional analysis of genes or planning crosses." />
<meta name="og-title" property="og:title" content="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/539" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ridge regression with heteroscedastic marker variances provides an alternative to Bayesian genome-wide prediction methods. Our objectives were to suggest new methods to determine marker-specific shrinkage factors for heteroscedastic ridge regression and to investigate their properties with respect to computational efficiency and accuracy of estimated effects. We analyzed published data sets of maize, wheat, and sugar beet as well as simulated data with the new methods. Ridge regression with shrinkage factors that were proportional to single-marker analysis of variance estimates of variance components ( i.e. , RRWA) was the fastest method. It required computation times of less than 1 sec for medium-sized data sets, which have dimensions that are common in plant breeding. A modification of the expectation-maximization algorithm that yields heteroscedastic marker variances ( i.e. , RMLV) resulted in the most accurate marker effect estimates. It outperformed the homoscedastic ridge regression approach for best linear unbiased prediction in particular for situations with high marker density and strong linkage disequilibrium along the chromosomes, a situation that occurs often in plant breeding populations. We conclude that the RRWA and RMLV approaches provide alternatives to the commonly used Bayesian methods, in particular for applications in which computational feasibility or accuracy of effect estimates are important, such as detection or functional analysis of genes or planning crosses." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Chm6OP5wHJgHi4tilzs6B8-F9icseqj59idDSqeO19g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459648","apath":"\/ggg\/4\/3\/539.atom","pisa":"ggg;4\/3\/539","ac":{"\/ggg\/4\/3\/539.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/539.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/539"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/539"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/539.abstract":true,"\/content\/4\/3\/539":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223YqofYtsBg3qjB3RxAU1\\\\\\\/D4yARYzcuQfVQnuyqBSUymye1Gaw\\\\\\\/Mtow11fFp3jEs\\\\\\\/8GVEWMYEs6yzJ5w3X7U7wPStMUNgMGIp+71oqRHtgD89Ysevwv\\\\\\\/r004vziyjIj8JyTLP52S6VEXsZPP6NAkb4Mh+Wac9GkeQQcqruEpyoIA9sDUTLw6+Q54QWW36eHDx6awMPtaqwndNrdoFZQ3A6d7wOUfxZKFLkuETGOsYSyRsVdT7HJJfWwn3YkDtho1bgNCFGF4ZEZfntMNpbY8i5YlaS81+88xVHIZqxwl6XSpMKEKpt+qv19VfSXlGHOqI9t8PLeS+NUAMz0vOYz87BdpPaKKNTpq5XSXKMEl8EMxSlMLojIgweCzzLLWPdawxZZiwZ5zKP4SZd6HRPfobVGVhjbaouGoqNvicSI1sTrgHYYc\\\\\\\/t0UoMJkjYcgLF4+zFCslUaMs\\\\\\\/q17amG4sHxq4WL5YHaEkwbNBDsIMCYjXqU2oOBjzkCZxscZ+rYOM4CKWRn\\\\\\\/E4cJlgzEWquiD8+ZttrJEBVizC8qNgxRYfc4lmnJNmRSyDPahtLq92x1X3gSi7uYwocMWf+684e2qg7KMdGu+XSqNeIOCIME3WBWVH3lqoLkkzPkAsj2V2ohkunMlu1aoX1LPm\\\\\\\/7PNRLSnL5bpP7ePN2Baxc6j3\\\\\\\/Nmo0sJKfuCm3BdsTX9JjvaMg45dHn6f9cW5y\\\\\\\/tm3DaL6wkhjg+JBOGaQkQX1tppngl1sTkA5jtfs5cLvZ1lmeuckmqxgE0u1uk\\\\\\\/Du\\\\\\\/QU+F900l3se+zctA9XsRVPSgUJXFar2aph2JXtYP6PREWD3S+rjBwwB+YAR3e7HRCpEBX+hklyMn0TgaVyTgdILWUpY7xpcbFSvw\\\\\\\/AUyXvhczaYnW+KTffvueDSmFxheQIWueo6isY9FjfHVlIFDAikhVfI4wi+zgEwG97++vW9JY82CjAnd9ZOYmwbtJRIlP5f\\\\\\\/BQsF5T7vy23mBKx1pmR9WexqAK451Pyf\\\\\\\/6YtqzPgMqK1wGOuZmxst5XCOzIbjJuQHsbBOuJemGuDBL8DZhXl1YTLBFr68TuZbPvJEGfDvid9XXdXA1twJgchpIdQFwxg34rXzIWGZ8nP3XZT3Yy2P5EsOQe7BSV2zFxSc2sP73N\\\\\\\/w5yMBBv3xagk6w5y1r7QOO4IRjjnF\\\\\\\/cDPFFG9KSOXCjaWrxiGFLMp+Vqegss87KAJuKK89C5T1XisNe\\\\\\\/XTttjUCiyyOk2MaWevkxvPBKCew4s1\\\\\\\/gwmyOvDkiq2n8WYlzWtzgmW\\\u0022,\\\u0022iv\\\u0022:\\\u0022YSmCVnxt8D4Q13e+UVGgFA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bb12f7fcb66c20971859d22195599a830f38813240e8d60164e55d2ab1d6d522\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vBGpa6SHvhfbTV+mytFDWoECOWqlwn5C0Y4gZWIgEOEtbIIwEfDjkFjBxa2cNJa8D3pqQxBlHXX+h3s+oJRdjVOdqg5m16z84\\\\\\\/gbiQq1uLMiLmUKRsFf32aHq8W1OvUlKBGaDet8dqqW3NHUjcVlBl40aJpObnHLIfBsQ\\\\\\\/3NGbMDjhaPm\\\\\\\/RcmLmyOnlEObOP7lpJlqxu4gPT9\\\\\\\/MC7K9BiiEFGH1NNRxAS+0IvuO+9HkUX4KD1JGkw4fUDc7pA4cgMOj\\\\\\\/4TcrIpBbGVpNM5FFYEwgq6UT5BQlODoIgxFqrO+hz2S70aWUqd4nTIxvckGPCIfV9Zpq3qZ4VmPgta\\\\\\\/hiBx0VwTGr+3qEISUZ0XM9GXLANeEnHyiTFLGN9cdjDEj2Gdr61rBodx6U10D72\\\\\\\/VzZyS+uXsZrzMmX0H8BZZwveHNlc080bkFSPOB5umewB5EFIzKTsTXn44lhK8DiIEjR7vNJ88biQtf\\\\\\\/q+L0Dv6qyoDPUwoJY7aOTucJF5h3Xk2CI6viRdTX5yuXLrH2uBLUfpIRuZGgwAGeMkdwXTWm25X6AUqQEpk2O2PITjD3QFdXmzGl+9n1Uk7zA4V74kMxP2KvX7Vlxa\\\\\\\/vqRKTCzzxbUbWSRKTwRUB1Xp+F59FMVkCuNES4Gi\\\\\\\/+l2iH4IYjQcAzPICg+ki+v2LmFp4xKrPUMB3pbQCPkGOJaQOeeXuWkPp9sHVxKQ+X5MkR3AX568G93w83QX4HYrvNJXQB6RhCaNDNXpsZXidXuyUOLWRa6DDC3O6ec\\\\\\\/OGqjEPcSvqnsaWD0CBpU\\\\\\\/B7UkxEpQv4zjKFbze088aSPb4ZO8bJD2Cf8cZmeFxrNLQ4Qvbgi7asI5ofZWlBToJw16IIQJ4FOx6xxH40AFDD79aKPYCBtMtvpfnQHB5yZSNEZ8yUXbOqo\\\\\\\/u1qEDuUDKWhdXaBY73wcNrEWyqlCmuxyx9oKsUIKviHo1Uw5ih2OvNPPehswWzS\\\\\\\/I5AFBjLqCpM0j68J+a6aQ7R\\\\\\\/nDTCpEmh4eP8W26RINQegrNVqYsYIND\\\\\\\/X5XLNKUKBy3YP1YBwx\\\\\\\/NmcxKL3eg5dWMfsbAzMKw91Q0d\\\\\\\/tYn2wuK9Gbet2m0A7AjacObpJlyQI2ACqviuG4GVZknACkV0\\\\\\\/Q7BXEKUuxaspX5qurwgot+rUmOuwZxV00tkLbTowSsBhzgHlLpfGm5dUd30ONd66oN87y2\\\\\\\/YJY3df8dAH8UqdMe0SFjrhWno5tdqJ7WI6bOD8mIu6wNtnWLeLQG0qRZR8Cpu+RaHHl1kErofPxc1Awj48r5pjTqO3Tyb4RbAsiDx1MCVbLZ8glCMakPrjj7l9OxFsMfbx\\\\\\\/Zf6+V+ojq5zGV3P11teTfCR\\\\\\\/8fMBQofpa56snPY9K3SDzWzLWFpxYn9y7bUG6bX0Wkf5SzsBUdbkYiccBlfBdswTpbSELNYIdH\\\\\\\/rgoozJHzSRs8b3Y+4ICe+fN7X+4ccUtswUrxfC1ZymjOknZ+QuHFcAqwQ0G9YgyX4a2YNdTwVwpUzOFbpi\\\\\\\/8tzLlaGEMD0rcgk02OEJiS41GmxYoPAzuIscGyjRUxvGMUOjzbfHoEQS1MZhIuDFZhG8GV3Jn93mw\\\\\\\/dfxnA3POAs+2a+sql1Kgk6gnnM+ys6s4RX+XkCwB5xFwvp4jZExaWo5EhPEfAB8dvwE9gppEYoxkdqykRdTCB2BvKPWqlWd9u6JaosBkw4pYfSessE64h514QYY04MfsupM09Xc\\\\\\\/X5ogJay56CtF+s\\\\\\\/1Lk1jLqdLqRXAdMg2QqdQcx+6yA3kBLSnEUPrtFVLxp86AbkLZYdCrYFLPuEnywIFBnfbki2y5FbL\\\\\\\/tO8YbnQcJ0z18BANANNxSxkppnAnfE4UYMHFu9TlbCYAOBnFZQBMxa9t4fJ6mRnns3VIEnoAPInzTXfqr32cUc2kuT1jqxLfdy6qoC1B\\\\\\\/TWzj0JtVnQWj51D\\\\\\\/INC5GevdRQ7+IIVnYrssPIfITcutk0g+gnmxbV3Gaf4Yj2NpsbMuYnf0L7JcEfcM+wZWr8hEAFLvqzBFwKNd5F+iMrARmufnXmYp04x4Ca4N+IbChWOnlz\\\\\\\/+bQwiKQ1DDajRaAvgZFdSRkahTQOoqvxdLpFk9fYttuCIqiayqsgzWWYbNotg619ssfCUmDMc\\\\\\\/\\\\\\\/I2r3yd\\\\\\\/92BGgvVyn4v4TbkZu0dgJvh1UcwUsbzNN5dJb7\\\\\\\/hNauEkvJeQ8PS\\\\\\\/qkxw1Zgsv31wtE6be+D69zSqgjbIiKBR1en3cTBkhBis7fZ1q1JWPx9gCNt4nMOSoKJUA7e7\\\\\\\/kVbVZCKxCpvSzldL+arliTfRTH+FbbBS4F7HzK0w2+BSkyJ0tg8vYnKb3QgRTMCW2Nrh5hMpY6ys2tbdke+OnNtCUP16oVmlWBALYG7\\\\\\\/7zkQRXc82kpJNXzQZZCZQHM9cDCp\\\\\\\/7wPgXKc\\\\\\\/\\\\\\\/pvnkhSLAgIsb8K6YJmMBFN2rIW6sng90pjDOK5DZsCaQnNKmhdVedSYkue13jqli7VNFalp+Q2WkFjcxzc7ivZ9rgonyst+4\\\\\\\/hdP+Te6oew\\\\\\\/odm8VK0m+fDcA9BevrJZc\\\\\\\/Mpbqj2zBp6bRb\\\\\\\/eJIBMECkuL0BQCAcazRPa0tlLwFGXVM28k5GOa0oBkhyhma38SeHrOq4fId5hyZao4zwkr6F7tNL+QZPNbmSovKuHoLtRdde8oKZ4naynvoZWKrQnaDBdgSj4g\\\\\\\/pPHM0osf103CrjKb6cCi4+UnjR9MWrfn4BLQzXMvnI5VM4r1kHKlEXQWiWf9qq7Wi+eBTj7v70VSP5JeQWn01o67Kt2CwVcJB1sGPTDINEaTnkjhdz7qH+7aX54\\\\\\\/r\\\\\\\/x9d9hayQDNfe765HOCKjtJaUVLrthBHuaU+ldo2F\\\\\\\/n+PQYbwEV4edjQPyaTDhCHMMXCdV+HhnGRR0WT2cVbP\\\\\\\/M+VVzwiCMpaXdz1dA7WEGF8ClwuHIFjnlhZSwDi5YApIVCysuJ9zMYESYw5Bp9FqbJvSgDPuE7EhUnISEXydRgdYJLrCKVClQORDcA7L63gtc5mB3muk\\\\\\\/otPs9bHOOKcJ9BmYZf0Yev8zv\\\\\\\/13ftUowDQ3BxP9CkAuLX4vpz+W+t\\\\\\\/76iJ+ifuUSOgu2tzBEUIIXsCnQ3oBI+tga63FRIsNcBSOJFrf+dGH8WyL8swgDiCigjExel48XuUFZB\\\\\\\/Rn+goo+cZvMNekMFtQ+RJwXVERS2obnYfjL5LdxNa2n5\\\\\\\/pd\\\\\\\/ll0b\\\\\\\/bSgeTpPxJxpzFYauVGK23J4cRHgrH+VtdKQbUcnMjVymejr+8yxa0Al3wLMg0s42jnmZO1nyzsqO2JZxLFDcWzGEQArikKMj3BVNX\\\\\\\/NcRyAfPAHRWPK\\\\\\\/xlTRTF57i0A\\\\\\\/V4MkZha9ax24TvrJeLgRXR6zVI2NWazpFHiaP+A8dF2xMBT\\\\\\\/pC17Pl1wi5zwhxySEXs4oLwRpG6vWo0nlFat0brGWlptK3aExVW1CE6WxXJFGlFbbhGx3Vxi5YU6VXhg9Lime0syy1dt0L+q675\\\\\\\/3bH3Uuo46pJRjibuOXNzaGxxmr2i+pHs2MAZ26+cWLM0AWUr+12FX8z40ikX66tiYRpV97z8mQjz3HqI34n9n4UEuwyVfqDUoN4NiA3dPh2fDjGLImAMa9K1FkyDfLndjD6Cv+4\\\\\\\/u4nJ5aBnSxYwk7HR\\\\\\\/PJzrrZoNUiVhXxtObKebrD7OhmReSs17ImKgpC8JgrrdrHwPWX\\\\\\\/a9A5vmnOmdDrV8jKHElCcavM6OBe97NyHrO7+l2jX\\\\\\\/q3bXyqjiBlwSzFD16\\\\\\\/PlIowtTgMDknMG+UQWqNef+ofOEqGuxKNIkJQRgOhe0cjXIZkknUAm9ztRu4doXzPsdEbotYpwyZdceHiKYJLM3sDsttLtykPieo2rImup5\\\\\\\/SxVaLWXvcN\\\\\\\/YhuGZxbs\\\\\\\/iFoS0aNcduL9gohD6Ch9LQGpB0jFZLsvKhNppfcH\\\\\\\/kyzzuvfTiHrHXJLSfq7KlY+Fo89IWuGt1Dsu0NUNiKNhNhVN1eqL0y64hT24T8nu97kRPfWMSa71d24RzZlfYbT59FWJIDp0j4h7JATxAPJgeP\\\\\\\/1wACpt06mChJwL0+u0fFM6NaK92lQSorJ2L6zx103fZFzNJ719EpGdwj7Py6qNg3Qy8jMc1l4aWeKpVMThn83y\\\\\\\/oGssUPOB\\\\\\\/GCOlTPzJhvbWBdK\\\\\\\/z3WomZnTgSaCljFIubUvBK\\\\\\\/ABs6bb\\\\\\\/uT9QZEH9ZsKWgSCxrltxa2WQi88COYV3kitc28\\\\\\\/3+JNCDQEWzovqYthQxQmt4Y+BeUEJpwveSEXJ+fDscUzYVlyjfScHlwC6CWHpqdWktpEMQiRbzov+8NrukAX0iDbM9e1qo7I9n9glUDQqDGQLLH84gI7MHc3vfi+feF6NZe7xiJwQc9QwBQZK3Drwx4niYZlLcb0+8T1AeSJlT32YDA20FBg2lMzAoqtxWQ6VtLVIfSWAtdrYGiZW8S7oEqZzlB3hUJoMN6Xg4RxlASnB\\\\\\\/NoE7MLFjVvJC5sYGIfHF9pQ5onVGu7\\\\\\\/MOo6HPJhdY+skoJm20cFzk90BpP7b5nYtFO\\\\\\\/fMb6qIL3SAJR44w0Yc1eaa16+Hdhtsvt3qtKI3\\\\\\\/X5Hc7fv0FRvdHMHusTb\\\\\\\/3jPo2y+brogGwMhRDWBMnwfgBA4tSXskfMWWxvtKrAl9CKGcb2lS5xQvWN7Q9VRugrfAF3VVV5Q2x\\\\\\\/dDiM9grFTd58nx8gAU6\\\\\\\/V7ePPTFjp\\\\\\\/9TCk+81bWsSnyifVh9g66doCU3zjbvex0tTF9ZxrdN4d0iXGHlkEraLmpvx+48i\\\\\\\/6q5f4awaC9T7GjdrJG9N\\\\\\\/cXULEZs6VYvEnNu+5fxAPyO8SiyI5DSXcG49VN+yCNujia9aH7mm3QJk3\\\\\\\/lctPskkfjTu8gQw1\\\\\\\/Ku8tqPNZzQAd0b1+CdTBvq8A4SnAek8dRif1zJSj9bgPCGszE66ol9cEzMkrXEF9PfUhGu\\\\\\\/1LKf07+ccSq2yNLSmaXzPw18V\\\\\\\/6jDZB6TKQfv+Ix\\\\\\\/k9VyL6OboQKUOSK8HJAbjIWOFY1qZ+g6U6UN5pfHeBga\\\\\\\/mYohtLDWYcCyxYdrUvnflhRZ3nplSZ89522\\\\\\\/5Fybb\\\\\\\/dItKl8+QoZ+BWA2A\\\\\\\/YJFpTBLqfL2P+9xbSH1VnbRT6r0F6L5vfle\\\\\\\/dqltmnr4DuNbXi6k20X00eARtZ7kBjM9HXZtE80I7HL0KifAV9UlGbEvPOYeCvRLfNETA4eDaZJV8u0dMtDDRjkcXWt2q7LScCNAkC+yuzh\\\\\\\/i5vS\\\\\\\/\\\\\\\/OAmd4wstWL6cAyYJdrBNY3b2X8RHAGXk160Uvs5DfYyGUUuIxrJEe\\\\\\\/vWwNKYw+VSXzvoYr2MZARc8qhnAj+WwdtVjSaT4RIjtPsU+hRPi76FiSNFWnhb59NpiJwwkzuk6DlBaue5RxyBoQz8NJdXjtxLD4mvNx7B\\\\\\\/vKSnlLWYBpi0XTVOWdoK9+tyUVwgDNLlnZoPfnnoPN9qmcK0v1Y98ajI\\\\\\\/of1yrdqk89vatztZC+mT85r5\\\\\\\/IZbzr7dT3NPb2cJ\\\\\\\/0jXucMqKB8syS+qBjQ1DYwYVmyAlOKgRdBcasmp+u6VcUeGL84u\\\\\\\/z7PWIhbQy97CeFsLdcwwgbh9uqeuQwuoDTO4kEIj8AmwTdsKoF4N6JMq+wWtuJb2Ie7g269H4iCQ5z6Sl5xNSOy8HJCkgXtPdL\\\\\\\/m7T308RGLSkOpwHfdBuJdLV6JsqaFS7fhvgCEN9ssQD\\\\\\\/TGN+tvJ1YxOxOcZ6gxNqiwuOoOVn0zqAoBACMTFSEJ0x42\\\\\\\/EeZHsQxfmUH2uwUKCI7C6AvNX9ut0xHb0es+3TOwv3FWR\\\\\\\/QpyqK9iOfpIX9fnTVHCU8CgSitSXrjxYCztDn6vpe9v215ANyc66CmJU1YmIzCOsnALS0zjy\\\\\\\/RHooKpfWbG+PGOQNafIsrNME3BSXPArZW2jXarWXtQEZ6i952Xwc1hudXfXFJ8Z9KrarAaVqN\\\\\\\/2vStbK8hYCH4X0lcrQKAGXxQnclmmDY2cOhHsq4flR0Xg0wlgmKkpWcUDW59C6s8dFrKyGps8h5r7JkBauKGh5NMMpJWA7Y4LMyxAOEOpz8bU6vyB3ZFft0IinwBohzJ88qihwCngtUJk5ceaqiVZZqzpuA8Nsip\\\\\\\/oss645QTOubxxOIlE67UVd5yyvuO5c59ewCxnBcttKAvt4QFQp355NlRcdTjRCJD1Yge9f+ccBPEpBN1XI3BHHav\\\\\\\/FsH8r2fodUdi3YqScDzU3RU+Ry\\\\\\\/oamVfU\\\\\\\/30GjWcgIylr5TJyMTLdO0gq+oB51BdM0Y4kFFxO+699nIazoc+f277RyCxY8zsxYxPe4VBfDpdiL68v9SIj\\\\\\\/uWJf0FeBumAG0+onIdvsQ\\\\\\\/CXX6Kcbfk+phgoO+zGurd3QAujmaFfE6e7MigIQsGzXQOr9ltqlopCmz4dyio1+wtfI+yT4MAq6AiPX2wR3EZbI6VNBT0AHDhAef\\\\\\\/gaNoCjvfchxPBFdlIGXMB3FFZDUdUWfi2BCSNQoL9XLrZaCF9DRbuC3zYHMSO040iNrRWNI4uCq5X03JzqUTl2tZsafG3jsRixpwSVTOaTzOqDAsfnI2CE2HyqqbCjYNqYnd0PMMX0jiOgxXNPSZDduaMXzdkZYM2t\\\\\\\/LiTZlHGSAvmE6JAzJAmZvnJS6a6r6PinA4atCDpw7o9yp1CgsLOgOhiOWBtQZE1YXvUWe6akcm4kFVnRSHhu3WbZAsnFL1y6rKbNPpG3A6oJhiihQWSOdZbtAD3TaSMfwdYwF3GWYyO2ymWUxph4kj0oeyHauelp3yYslvR2VVspsoZ25PRuRsEYqW4y\\\\\\\/s+vw+wGQ4YT3WD95qTYjjqI\\\\\\\/ON4i6mSpiRhdDP9SK59wn\\\\\\\/\\\\\\\/PPE7c\\\\\\\/2JkxEpab\\\\\\\/gieIE0H+ZyzNNB\\\\\\\/knF+M2fKuV4Qt3wUgXeEr\\\\\\\/RsB+8bHamWAnsoHrR+UVA5JGB0OCLg4A7KiqVbebjmIbMFuVEtylP\\\\\\\/9m4PXx6iqCQ0\\\\\\\/hTsjqTpFJZ3MTz0FUiC7C+1ajbDtDLzKm87HJLGyNN5o4x6yPFS7ktIcZFM74U5zPd9KgoPauVVLJStzB6RMfzhALuhf4eGgkmkqnxMfsRg0MbSesAwxXQ51oOjtCWhc5o8PnpoQi5UltfX+NT3CQD02O\\\\\\\/STrHfnJ0dK85+yYl6hEhhLfpPkAnd2Jzb1IDyrWVSpRrVifFSMResK647hLfVs+kaWh1xmheCSFd\\\\\\\/6FIEtBrm5b4aGi6JYb26fT5IRCmM7dY\\\\\\\/ZMYmz6NI4\\\\\\\/FdODIF8TjlfHODeEPhEBkOIxuEwf7EwC2luiR5DFEho2KTu7vh+2SZn1XeUn4Rd1GfBXJG5UrKcg\\\\\\\/pGNpkhZOH06KSxr06dYSGXeXG+JbRWNqbwimzvUm3F+GHcbYxoTFc2oi6mVf+H9woPG8\\\\\\\/NK7Akl1R8feqCGbkzD60Lp3HC\\\\\\\/T0PbU7eFnExMYw3nouZ8FomMGx2kkrYzoUQRIwChuOhdkjiZ0ElLGBf8SM5Oh5Pwe4BaYCK0IQqhL\\\\\\\/4P1uEPRZzPrVtAAT7DDer7C\\\\\\\/3AVCgu4tV+CgV\\\\\\\/A6fmrqsHi\\\\\\\/f2kqjca51oFgFQbOGRNCxVhYXT\\\\\\\/Yb60R2si8NJ\\\\\\\/4aiNCbzBHIibJQ75CknQ8avHmWzK9LRazsIypVDLQeflthSVHmhB4lfO08P5dNzpaf7M9X3Us9qH85EzIPJCzky6ieUIB3nnf89bs2aBS+2gG+\\\\\\\/GisrzfkWM83O5pKPZuhcNCbvrUL0o0jUucCvrk29qkTPkVr2XQWjntrrs10xPczH4MZBp9k62yujWQMeIC0lbR4b91Fj2V3KtOBPH6iOpA4pnZ9UTptnJfmDGwgC964ZEI5Ua6RrNRjRWgFRFnd7rddJFnfnD79JuduspRuFsxOmQMZqu6HwPMFtrHwJm4dMbmYtrI+lLOZP7j1kuWHL1MDN1Nl5YKEZeXRVBwykoz9P7TswECCWLe9QcArUT\\\\\\\/a7FzSie6b\\\\\\\/5E3yld7FVoYhh9y3TRXfifxvsWmE\\\\\\\/MEiY591VDoS76qrFdvqVhG0iSfsS8PvGiqEA7diFuFnwSzKsekjc6nrOI8kjJs0R85UAIfF5\\\\\\\/e6kPp\\\\\\\/yHHopDZ7z15xSYnnc4i7rD3kEI94+zG3yEBawCPXhS6HFF3iq4jriQXpN2sgOF49diE2Jw9d9kjntmMI5y+GltGj5hmw4fCy8mxFPiyoJ4sBgLy0drvff1M5uw6MfK1I7FFZvLxpTaju8irpx1Hrrg2zFcfoMFKOg7Y8XCxN+QqdZ\\\\\\\/cbrmAsm7JTyaH1Ww1wMnqPtxXD+JRj+vGSaFFnnyztodtPPBjSa+BR1Nkafd40bsZXef\\\\\\\/C0N1HUJSWVrCPfg3hP33UBpX+On7IJshdc\\\\\\\/fOQElY3B4Jn\\\\\\\/SaBtlICCIW4ImyjY50qdyqyzXWxYdnW4abmgtfiGQe8WQ5oD2qnLh3p80Ez+VvK1PS5rjsgNqrZSpN4+apQ2XeXtWaWiwVU2TCN+79GE4lZtoLTQANh6nkKd9IMsajra3+VyVnDG3QvXxSmr+K8jwa5lMtK89PbQ30jGVgJFrkkw09eY50r4od8ZVCx3AKou6YbWoLHwqWCr0Y9phMMGLM4lNgw9D5EIv9BfUd\\\\\\\/D1kF0QDFtcd6q5dpI0RZ\\\\\\\/GWmoCJdYKMr8FxWZoWDrlYqLYgRhaB1On9rhy1KMhN6MWMtU1pkvMRToI3+rw6Ym97LpR6hyhepNC1Dgaalxgu+U16kxhhLv+5i1N7UC8r41B6YeIXx9mCDbQ+3+pkhPU7UvF2x\\\\\\\/nMC2UGdifZixl7CCkZOaF7d+8lK6S8fA6pPjTPJzRXOGrRZrFckRgSgGmCNvq8bN+sl\\\\\\\/3KTWCVzDOBOWRvLjX2diaSBe8F2tpgguTfOSfdmfHaM5fY0JSlhcnpwkp0nXDy3KgYoj0\\\\\\\/G\\\\\\\/GKO1SbtLfAzcrdfGMNnR60ZzlkMIBAHMbZZNO3emFHaeolaWXOvpx64HTwQcsvv36EkFI8Rs+bVfd261X8EQdiSCPUjvM8hUTmMKKmuNnpvfI1SyXUyHSDH3XlfrWa3TZzs5FmGgzaSzgE4P6JrHoz4npQK\\\\\\\/c\\\\\\\/Z3dYsEcxB9fiDMa2APbQUMiVhGVyikNjd12CyPjudO8Td1h2CR3Sh5s1jRkijxfx9Pi55BBm6A8CmWu1ZONgmxgEHlS\\\\\\\/7ThmUOy1DlnzdqY02+Ya12ZvArN1xhDjLDrFFUBZ\\\\\\\/95odYWIrSrS7uw85ETt0zHAgkkAEVLWBOvywk9bFpHR5QpZbRy29lbiJ9L7fCg9w2Md3DIJ0++b+\\\\\\\/3JptnH0n06R9JhLSvAjpHDWjYx6nvuvvEO7oKlDeJByiJivi40eK583XFvFjg+9YdWCpPNJOIaivmSrK+SJPDjnSCs9gt2sgka+yIAy2M8aCTj0e5SRHgNCOLEs52GXePPyDrrnKF1sUkKvLOhvd0IkLE3eKXcxWQhOnYCPiQ2aXkax6v8yx9EHmWSmnQ1P3S7ufgz6SeI\\\\\\\/ukz8zCXPi7E+\\\\\\\/vikkRuedny7pdXW1gdrAFKKULGuSAgMcUgC5IPdNPj3+OZSNuOj9BfZKFY3AyhEKuLyMXVw2c5lJMAb3RduO6Oh+bevFPlnTtpqWgqqIIvaowQ6CCwlsS3idPDyMMnAgPQEUlETBGK9eOI5BbGsxpHbXE8hY2+2CvPCJPPx6KXMqWRcnC\\\\\\\/MaDWmxHptR5BwUqovQ9PKQL5cbcOk59oSV1bMZaXOxOhMZJZ3E\\\\\\\/JQgqjaHPRLJy1kPBS7OJU496coE\\\\\\\/feDqItPMPh09fwHAF+78WIBVAeFT+O3mxJS8inhcqFPJf5dFbtIHkRvHKX0jOeitsgEw6ENt86sL1RNhi6fatY1DNAQJKjA9z0bHueg6V3uDheZGIZwo72oO1szP3MI5R2Kr4QRGM+SDdNUKHyz1e0YrQFNUB\\\\\\\/HppgARHySHlEwFf8SWGl5P\\\\\\\/42I7yL5d5cYMG59Fu6ykh6Xp30uAa1po6wCZCPmMy677SwcZSQUn7zTml1vf9LWfbg7sJmxnlT7RlFXMU81rHK30gdRfGo5\\\\\\\/RDn1xJHBi5FLAn9rdIE4Au4UKUEsjcjzlWRzScSmDXUwcRlr6AUCMGEZ1XRlZ2DgLQ7Uizhh+NPke5XU8QZeBpSELq+WVegHUiZ37qe5DGN4CXYuSuJrwB9GzwFnjawB4eKHTqW0UaF9CKuJjoPCQIgqhhn39nFiMsBNLQv336gqAwHpokGZdo0+BqdqIW8BGwpCunaEeNSQ9l7WGVmIg2hfR20wIYeiwk6l7x6OCVZStfxeIzQvHsAhrVa2fcp86h++0fUW7e6b7pfium4EL\\\\\\\/8RfxgESZNidgv920h817Np7MAt\\\\\\\/mN8bIB\\\\\\\/37FUvxZjMlLcUDkty4LiCaqXUiFOEcp8CREMgH5ykpE6zhSJDChC\\\\\\\/3ZXoTgabIkJDZIC9LFTuKm5pXn7JUxvCtg9XQR2R8tkwftWPaG9ZDPwHR0Bd4hcoONX\\\\\\\/sTRRId8dDGIZrMacfrf\\\\\\\/9bgoEOcsqWDcVNF73jxww4msl683isWrDxUpWhn0emvqfOjTSecbNjty3J49CjC2SGMkP+odyD\\\\\\\/AEpH7Sjx12T64HN9ml3a3cXKJNgR4wIg9lzKZTTf8\\\\\\\/9czyrvIk1W6il2iq3ozMl\\\\\\\/VuBrsey7TnBeyI3WFpG+wBFBGlxEmeHXNMpoirD330sAnlPQnVBNzJCO4PhGm3wQizlxz3lqeG+tnYMpmVXuVyPeJ9YspJZluQtPvBUc6FUzqi\\\\\\\/uOYQisGKmuNMmJGde\\\\\\\/z+H3HXv\\\\\\\/H8ZPHHcNqW\\\\\\\/ePAXK3aGWweUF5zpRIPhGkHOEYdPn+2gBWZPOukIuY5tUe1CuYP3qOkAQuhLIMCT2dwvfvUPShV2oCInwJS9O10MnBSaoHv7Mw0nDTW45LXLhlIRzSJ8b\\\\\\\/WayTbG88wAQIQt64QSTIQwiPgAWm49MuBMJg0oLRLVoWjEDTZcckNq\\\\\\\/Z4\\\\\\\/pQ28GkJnaBJtAxc5CLCUdQiWQIKE+KxioTW0l1OVjCNkw+UxS2VmvLSg+JyGqdLaYgwwurHV6CA8R+Wo2HhJAoRMtx4W2JVjHUw\\\\\\\/k5vvPq8b1otN28BQL5Q8uLkRP3W19sva8to3KPVGLO2hBWUkundnr88xZ1JdZulJqIDrnhNhG7zPqNIXB3cCvDvzhzapPQH6abJpPOjf4xzLSz5zXOlw2Y39gqrcQUXTYpMHQhNicLtqQwSsf786Oz6AdLvz66HzsKqoducMvbKF6taopaTSoemdMxb2thl6WtrRuN5qJlqDiSPQrbw17W+MgB8KsyX1w04ZzvjdL8AIUPwjCWqMBXgtDe2AGuhNURKmdgj5ay+OJZXIRljbC0CXw0rsbitc43AhCAzbLjRNVEnIiLd9aJMBGI6HGTCZtXUd5NeBibkRlaaL5I493GXk0gC\\\\\\\/SDvryr+juA1tR6ysf\\\\\\\/lq4uuHwcPpQUV\\\\\\\/bAPvTw5BAQi7Jn8\\\\\\\/5sD6d4+dtGWlgxt2bOQUiWDUXRHs8ikMePjVOPi7TxJRdjHmIzheLF66lNhEN\\\\\\\/ecGk4W3Ju4kQxVrmPgoiKhdsUop36x1D4JPwDfl29d+l8lzFUQ6rPtDrDsw3Q98VLZhSMZk5nQiNY4BjNI9zNSrYIEwoVSQtq+lG7jc9XgcC9KMxm1q5wAsJAyul6XTMKPnJr2SCeA4Wv\\\\\\\/4SKT9HWLjIu+7ToEVSmANON\\\\\\\/firlUxaQ7SxfutKBf7f6+FnN\\\\\\\/6k1QIukflZowmD\\\\\\\/xE1r6C0HjHuwTuTPbs8xUwaqOwGdfVd5PNvJFrljoTXpqhUh0wMWhLf63S5riZIDcMVCFdjvbTV2\\\\\\\/g8VtMdIkoplDok0uKfjX9BaiKzq6uqYEXeU\\\\\\\/\\\\\\\/BbMv79jb3Zmj1EfhVnI2indYkK\\\\\\\/RUwwexn37qymmCtnqNPMHlycAqMJ+Vkt\\\\\\\/ifwVIYt6zMGVPoaH4CcMa\\\\\\\/NcmNspBxS0nJ+wrebLqwhqoThRuGf32Uh2kp5p6q8yY\\\\\\\/t5wKcJngqAz7BPQ2CCmGWuUeiwXSo+vqdihz0+jRPmzWqz2hoJNrgbKkUp6q88kcpN\\\\\\\/+OEmSxo3QCyVwCBvfqT3vCp2E\\\\\\\/lcssI2QNYk56RUZCrQjDuXMARBPe4MZLPdzQewO2rSADPIPeYaucrhkvYsYUBAqjbBPbQ2IE620zJH\\\\\\\/yey8IzKuQ8dtYw8ppl00wNB8mz\\\\\\\/jPqP7pyzT5EXf2nBlieAzwT2TvW1RCiOvREyqGUWgT7zQ4azwlfloQ2YoeZQ5pvl9+v6N5eQZsyKbWrKAFVZ\\\\\\\/NmBDuRlRQgbWH91H6N3v6qcupG0uuW5SA63ZUgvqFD9SUXv8OLAJUL1UO3VUqSOHUfqhc2G+YlhDWwCR8AKkDaTDqJLUsiPC5VQZ7WgNr\\\\\\\/HJs+cYFIXaPlJyaxD7TU+TZq\\\\\\\/sxA+OaOmLwcuoN+jLyC\\\\\\\/cBSljx+YC0P62XET9Heh8oFylPB\\\\\\\/b\\\\\\\/fQ1dl3y4a110FDCR7w18+a5VnAHPbBSNM75vk0+3E7epbUJZ4+P2pM0henZ8icjjDeP0syUsG68DZ1qju\\\\\\\/c1y+4QZRHX\\\\\\\/yKovzr3pbfB1S25f\\\\\\\/HhSDRcVsCYW139\\\\\\\/yFAbuvAvQuAiRQ4\\\\\\\/LxeAFybCx8S2S\\\\\\\/0dPLiAs995KT2fD\\\\\\\/LBBH1meula\\\\\\\/V5CXDjAOl3x\\\\\\\/96uFHvo56NYFHiX87A63I5HyUObShJOhOIGAldG3saArYGb3Q7fFo4rxfcgxT0hg7WWgtAXQgpuJrnpeew6hvgoKKxu+w0sU0W\\\\\\\/T4SNpC9IhosCsAMhTXY4oTcGxJ02QZR7miJWuJ4Yqc5Zsol4VJzH6k\\\\\\\/pg5PzxKr8AfP\\\\\\\/gneyGIqrrbIge7t5jjTIbF7xapozpPL1tJfopdIeBcbk8Asi7zeqoiDKY2\\\\\\\/YYfBXfpqLJ5b0iFnGSqu2W0KxzYLsYTOPsiNU\\\\\\\/IuP5FRtx8OaJTFRtEWNBSs4kb+FFZbucnQZOLwU5sDd0XH7WzB5TIyC2jXi8DrmZsDSZjd5LNOAinovWXUalT+ytuJGiiAzOk88zkMo201U9no92Dp90Jo94vgAQZzo+5jZrKeMUBeq3ZhohEZtBiGBKCgelOHXl0u3pZXeyyuslCVKE9uLSMA5kLoE1hC6rYKx4jcNN2M2pIo7\\\\\\\/31NnRq81gSNR+f58UQFH8pChyFn28Y9Ko3ytpvqFqsiUuWzLy8rM5+VGKgBHQo3w\\\\\\\/M0HQt8vi3FeUFhCMpB0mZWnx1NsO1nExWs0VeBAuaw9AQppgYXWNLZHZB16XNYqOY\\\\\\\/kZNTCR9HleENQLYrm98VucteJhSyVT84IhSDS+2PGRZCwCAQkQqFZOe7Bd2Xv0r0RPQNuyKDb5qDQPxt6ig\\\\\\\/e1+1alPjSSRD7KGDTOjbDnC\\\\\\\/LVx9oJkd9\\\\\\\/8P2kci\\\\\\\/1URWzDg6hq6b+a45aXRyfKwjHaDhN+Ht925ughUZ1tV3yl674wb9cgQ+yD3H1QyMZDnTiM1JogcFVriYCqCryCvRHX8X0RL+pxtsXvaOfze6QRV\\\\\\\/Vs04e8upJsi7To6DPHCDrMAbfMsjYe+FMRvhU2GEx82F+ZBrUld+LUYre4Vfe84Jgr1jklxMJv67losExc3jhhVsTHrihHlozhlyRKLv3jU9KqsHFaSf3iX0XY5QigZwU6zYWruLmBvP\\\\\\\/KE\\\\\\\/pQHSHYme6AQrVGDO9kdhz1UQFdcFcPsmxl+S5F+ZPtmmXOa4ye3VmjX8jSdA4ylB7BbW5866D5JcANE9D\\\\\\\/eMYsKcRAltzUmToNeRaL1+KoPX7OShxYch17b2oNmq8agZrGFDx46lbXYAWqaS4nUVtz9ktCKwaJclixniJ29U5q9oUyHE4pUoSybTKP4GR6782WhIHZ9I9WERcaXGo4mR2wgDZ6fOoPCUVjx7ewAm1Z\\\\\\\/1CjN\\\\\\\/V0vLc\\\\\\\/myCjQjpS6AiGwz04KHMa5gQa1jsyJD12A\\\\\\\/Co9A83SecBkuNNyA3bEAEq+m7Y5csuRqbCJ76BFeMQFj5ngrFWa6UPL5Amp2MNJ+I4spNu\\\\\\\/rU0eznZ+kRHnlQT7s21bvMl0Hq6QrpFCIiXXqD9VLR\\\\\\\/krxlsvq0oCkaDqg8Qeu9d\\\\\\\/Kyt\\\\\\\/6+CnIOwyblY5kBLb1aSrDklfWKE7v8wU4jbhMGkFkkwUSIhNhXGjiX\\\\\\\/05TmdEsl0b7eQ9Jb3OnV3J0gpnd39MpW7BTVFAOlncgcbJ+PKVvH4MuZ7++um8kOuaNvdEtR6E\\\\\\\/st+RlKpWZaKWjzeu7BBFVw5aIOY8vHmOqetfmrQdqiA8XYtqHaq184skMyd2JYcTLaMOEDdIP1Y2OuiCPh2B50uCA5YdO7nZbRefX9DOEcA6If3vGjp\\\\\\\/cKpaqCSNpeFjgcKXdKImZEY2vNU7kfjWNjo8uSJCrWnjVGG0FzJnXXVy+PpDyb\\\\\\\/NbbLAl2bIQSU0cHDJlbzW61ZLFdND8CruhCozl\\\\\\\/u7MUMU4I2xslPji\\\\\\\/YiP5frQ3EFU7M4n93JiQ+nNqZxZSw0ib6m+\\\\\\\/4Kj\\\\\\\/5oaZ157Ljs2J2o9XYry8DFdXHuLXXG2anakYK8XKVh0+1gPAIyTK6ZezWyyHMxizJNR7w\\\\\\\/JYMbxQ5e21Y2xFPp+oUe0gr7uZ0QldxXI\\\\\\\/97eWfGU+wZ50fpLhT0288omuSG4QDWeQVnNbKP+XpDmuMHQ9JWiP8j6rYX0gApbo+SrjR6kFrxyv\\\\\\\/C7I5na5vdmZ9v9+LLrCd\\\\\\\/Uh6GbUAaYH+MvECP++nbD\\\\\\\/GmkB5lUq2lRS0vrpfDrqTJeplh2xBG6\\\\\\\/162F8SEahDbrpBqpuQQ+rQxnlzjUV2qOdc5\\\\\\\/C177Bz9FQeb0ETL3qHuMuqBzirsg0BInhHGZ\\\\\\\/wiRm3HwONF90J46YTckCK92hmPtvYZPzIF10fiC\\\\\\\/mBxzmxFqk5IY53tpaPu8uv\\\\\\\/SQBE02Il+97l8BDztgi5XhCnbUlYZORcad4nkqIvkJ7HjfBUHbHMjYmt0jAuDb4V3IVhRYgduqdaFdNuQ3qo0yc9KxTXUolWhpXbbDEXwookxbxrO5blRPEJJ43gPsEl1pWUgVFWpx5hEQRYIrK3W60g0Pqq0F2qTt3wyAbk8f\\\\\\\/UAS46eW6E4fBw8yapQrHI8S8Hxx42007vOlmYegxCRKrm37jJPEX5bLth3I7YpW3vjk7tbtL43l181P2ehQgA3sb\\\\\\\/L0zeTSxedV6Rhs9ErVlPSMm\\\\\\\/tpUhnL0c8hl8J4SG2DobQz0he55jCN165yPEzXEBmTj+OpiTE6tKdLbcJWMn9mPOjnkuF1C+wQTX9k1Z6S8+KC7nG7eb5sLxYBv2F5HVBLv1F\\\\\\\/935EoxeltlDh4dNO7H+RkKufFAoUKmSziPnyscRB2y1nD\\\\\\\/tWkwJzrnpM8dsbyBAFfaDEzdL4rJwijZiwx\\\\\\\/PYAufetrFxL\\\\\\\/MU3TxpDkSjCXVoWLcXksTzWXD0g4YOlBiu3HVbad986yptR\\\\\\\/cOYvL7Ezx8h4sSks1Qu6nr6uxj4Lrkav4VGqdqBk8Q+Xy3hDxO0Co3+KEsZ+8yS1YH9G6IzNHSpmxrhXWr7SnFvjYdSsCIbDFzljoFWZ8xFDxJMhOPZ7qKzGvNF\\\\\\\/2PYqwTO+Y9tsAhT8Ekt91nr7BEI+4Ra68yMhhXdNKUFzlf0038XgLLOOmhLLog5MgMzObtIJPjenH+5TtYHg74RwQzuQnu7lqZ1jRU6699KQuMVBg9DqHKD4of5B6CBTdGdTvM2aSQsfQo86GgKkN6igtnvfhz1MtplTj+4stpMghkBfszqG\\\\\\\/rrkZYSH+lqZ5xOTvPqpXFtwE+OQ58aaM3+cLf83jNfLEFaPZ5UFU0eBBw9N42Ta0e3jiWY0SEugW0GSOv60x7YcMTRqNRTqNv9H3dNxvowy8CTJREZMbWendPpSZ\\\\\\\/NGF\\\\\\\/P4Fxs4pcdKtag6yZtk5nryQXl8h8T3FA4lfizhXFv2weM4w\\\\\\\/hFog78\\\\\\\/eM5XQi0muaJ9mBTdV2SXuuKQjXTvIppc6fqM34xHWpnH0ReO3\\\\\\\/C8tAc65jdyi2fSO0kV0FR7ZZ9wiQITIYxAyK+altmqFqGL36piebr25Oza4+hvr4g1ju+sK\\\\\\\/gm1JL4D0VbDj25rRps11odcNss6Aa0H+jGXuLU0\\\\\\\/zuJZO7RvdBH4aAy0S6wXG20ccudVRETvd2YF66UW\\\\\\\/CFIBWQvlqByU66qrib1\\\\\\\/tMFGPyofZCIw\\\\\\\/yKk76Fu6iVOvqVM4cUjrJA+yNJknobIHerdB0ea2kWqkkxMG0sfdJ4whSd98002\\\\\\\/8i2jld3glRn+fKoJ3Mblkh5dzuD09aNn30OKX\\\\\\\/X\\\\\\\/EpBhLVOM8bi7ZKEYzvI8\\\\\\\/6hhoODSTONExIsVn3eB9YYK5O3YqunFRIb4fzs\\\\\\\/a6lQD6ydGYme6AmpZRcoNj9eTK1kPp21I5S2e1zOruDOWkaZSMuzCzMkDEfwIT6c\\\\\\\/PbjBvQZtrVlswBv5+3Q0jF3AY6ksQwROCHhvZ7ec5lCCya\\\\\\\/k023EXska9N3hbXcmYTVcg9gbQZYY49IpYN5vCAe9tweJUQm4JHtcS9XhQBOcfUZvh5u5yZIBN5yIr\\\\\\\/kSZ7nmXH+KxrsbvRPWRlACN\\\\\\\/lpw0yJOMaRDu8jABsFTgqx3fbtAE0wCoeU1nQORm4i3g\\\\\\\/\\\\\\\/cEqNnGGGMqxd0kAniPw4Pt0KeEsYpJDBGEOqHSJKzotQwYFKWSJACrp938uerkg05SQPg61n4dpmqnszcNzkzNSnrSgH2mUUzIgUH9UpX\\\\\\\/DssYEmq8wlXS93Fzdcv608zA6MVdcmbFhOUhVm+uWP4m5Ctr8EEukphmWTO7C0OL\\\\\\\/Bcrk7Lf3mWuHzKoF7oaqF\\\\\\\/yOs4gczlmODEUcax4LTwI7EJ\\\\\\\/\\\\\\\/Qy31amQG8UvcVG4uXAFMNmzrhD0\\\\\\\/grixDYphJREgOvbGb6k6OYhqFxMWHjYPu+Dbpmrcz4njCde2pK3d0f+5BDFUCZHlIrVohLCw0AeBnTP+Kq7k2MUDLV4+nLNKHzrlHb9Nzuk640ZbxD1XakFUZtSSnO77LR5dNBN2PQp7N1NHP0Y0VFY4udcOm8YQLU3P1sem1CGeNs3LcOJ11liLxwl0u+jf96abMjjXjdOh7eeFkugShD3rrcA9igAyaFYVECA0umIfRYyx+hcYGCHBs+YnGKMGwhE7spH4EeEnlCMUAKErdl0rPCLgzXDwu1XmmVpalekapvSax6wiRRqIIPqoBLgulyKsw+uJKkrAt5R2tDGCTfJDuyBmLmTM9pMoANKmQedVchUdEcS18XLJJT4IHSAOzzOs8aIZQemNJ+FwAx+A9kH4+K0AMi195XVWXk3B\\\\\\\/WD6NKTlpr8kmOgPDFSLjNZQdrSm1Xy1QyqxnOK\\\\\\\/\\\\\\\/w6W4Ewn+GBKe37k3FESKCLO9rbSrpaWlGWe6P9jCPARMWcwQCjEW8R2a6M1slsD51FI3BlA8ZpAArc3kkY8cZgVgNTqWjsj0KvEhGoU9HH6r8\\\\\\\/A7BoynPMd7B+H3aZPfNLmfJsmVs66yX0Wn55wzTHyYg8dYL\\\\\\\/ARSaMgpBvudZmEsOeegEEzX6dDoF9IrjA+NcRWX+S+OMgVKzln8h1sb98H25zyomt39v7BRo9K\\\\\\\/uwFwanIHT4GoGa8tGi4CXymkQ724Q874AmPYpPiJ4tRlGCiQpJdUG0RLcG+nx9P+A7Zbgd8Sk7\\\\\\\/1zjFfm65hitmrryOPFAPp2zYRHDqPAgtVvE3scWmhTgE8r7cZqT4tIqDpz4cWExmB27LvbAdGUAFg2Hr9btWNdo+U33jT6lLwNCLJ5YlmgNG+Mv7EvrFVN0eqhwutZclnuMwDi4rmLVseVf8Ukkn5FmgOEWh4scAVgy\\\\\\\/4xhSIQfz9PLX1UJ+wvFOcp0XsCQRCXCKagJkhoRhrmhB7PvFItgULQUsyvFrD8y54QGcquFMcIiQxVnBoPxln85oSuiROhUsoN6y5P+KQ7BwKDMSHr31QYSUEo0C4g5we3y8IIBunnhcCCkjNmTtCyXxzS7RllaoIfckWeqiSkt6szmOowJC1yXfD48O7+cSI4LdmqUlQd+6eIRCfmJsDdD0KsTOnbHrGx12provNTYNOvU96TswdDXz1Puh30HYM+TNP9mIbrNcQoa6+uWvwOqRwdZ8XOTD+OmqutmKS7zd4mQxdXhxuqxDvfM6t8R1fDp1pTfF9\\\\\\\/hF6GzKYhPrldejF2lqXw7ITGq+Hocw3\\\\\\\/83BGw9sVOoKCjug6da85akshte7QBJdfSKoT+VvF09O\\\\\\\/ymBKSdzIEKZ5YhNxy4x1DzVkAWzMiFvK5nuIims+j8uct7Fq3tE9Rn0WM+FSNv1YET1biL2xK8RSy0CBKJhWqU62wmCf74YucCcUXQIx7EeG9IcwvO6A1mnUU9lt5nm9wffYQXkV9G+wtd84ZwDQR6nJxrRdTlmXihRq3MecTCFVbWlbCMpwso0tLc9Vgw2JbrqrhLsAcPD5MRqsBBknH8j7CGD+qAI\\\\\\\/SRdXsOs0smcK6vaRlRzbHBuL2tLr47KNj75vEa3wc47dHw+9kdjEqo7UO+dk+ZSDD5ukiPNgww363j8TFjS0PwD24LMjhnrAA8sNokOJub2Hpc8\\\\\\\/EKfAcTZXumwCuX+aAFLPlanGJA6UuZ4DJtZF0ftrGVT4j7Cumr3emucUaZslF3zbQWX7PcVVydb7fjz6zKlthHpPDJTGgDmgdi9pQMozbwKrn57hmJkcfX6YnVQ18GzFmsCeRcElx\\\\\\\/K5L+L9MP3OdU95tSydgqH1NXKOD8Ezcu84M9PFmgnLM5hNICL3Ezr0Bit7zhH9pdwsKHqmMCynPNn2\\\\\\\/+OEOpwekJ\\\\\\\/WG5sCZTkfCNX9KQ1BSnXpfsjozMZiaSOjdztUrKhmKNDjbNlMEymqajvfQ67LkTVD3moSgqLcZWiMnuQ6i8Hr3jBuQ9igFnZXgg3ojCtnFgyL2IiVy7BKF3rYLq6XS1Zgw8WmdlRYYHTX\\\\\\\/ytlwoA74wUveCkBi11jXJ7+o\\\\\\\/G3M0id1mSESH0BWqrBSJ5POlJSJzqLqzjLHYUkvTozFGU0wBLrJn2WMU7awtMTsfraGwJ449xGi7YcJo8d08j1bxpue7QSdPafNiT3oRRGX+WWU9xhoJb9C8EuPADpzem4Bb6EYAOrhMuCgVa4IrYdmXsK5Kt1GKP3yGWhnIbxWH9w95USLL74wN6tJQDk0zqERcIJ1vXfoeOtscnXJEPdhER6Gr9VF6Vkz4hfrWRvCclnzBvHzwtdsbC2RUmOrMFWj++s1H8v9ILbSSjP3pYizv4hH6CzOl1W78TwYMigshvmPZW0eNsYmu6TJ9zuHVjym7nErrvoNEzjCTDgCpDLz+lWBCuWUQCf8MfgN\\\\\\\/jycAbRhmdNY9yhmDrxsKVuJUqxaBUv8QmjA\\\\\\\/zoIYUkjpUbpzwXcuwgoqitu8TIlzL8LI4RMrDCh\\\\\\\/Vh+6IhX8GqOkpkA3hQgyQe5wy5A33x607OxqSq1p8O8d+1IUszrIN8miICjv75B4O\\\\\\\/tsrWYnMTU+03vQ8cFdjrHRtRa6J8EDIBbJ3MmhSlmuOHX14DoXCxKors+\\\\\\\/QuhsPOMxlTpGuhZ+j+44l+onuXR\\\\\\\/Rh7jyUXnqNbLUdaIgiLEs7aElCeMK+64hvPVbPtGepck7ufAK+U4TePlDv4zQcUTjnIZ6NS7uiyRENWEeEr\\\\\\\/wj4nSMu5wpZHZR3eM03ZmFXfcXACc4V6gS7Dfxu5AUpot0BtGJzAoIUuCLCM+yGnhrtxUAgmwM2hY3WODlFD6nA\\\\\\\/qX16lC0VYty2p5Comp96RymP3x6451cZOGnU2tgOAJFvRgC6WLDhOvnaG8iVP0M5S7Sqcjyd2zLsGHnnf+l6kt8uhR65tJB7uzLTmQ+Kfc+H2sYRSkTp5p3wt72GPXhaLLnEbSi\\\\\\\/pam5KPgdERB1UiQPXoFtsrGWzhBeZ\\\\\\\/wmMxL2uSFVHLAak88T2ZLZMO8NwfH2uXvIYXWos\\\\\\\/yQgMlnSyN\\\\\\\/U5yuX3ycNkA\\\\\\\/XjWAX5o1TYxP278K7JPOCL70HRZQ2uyHIQcl0xmL16M4JLUD6bQrY0qUSPP36GV4PyHTvROv84qpv29Xf5tSdbc7Hy0kjDvUFF6+vJX+iV+xvn1vdb3MGfXw66mpH7bcNwhEmWiTQuKfzod7eyNmHeLRFFVzD0\\\\\\\/el5dCQWk+hjxyt\\\\\\\/y08IjCzVSpkOLspVj\\\\\\\/CpBiqxyJI3GLcvPru99AnjJFnJU9wJfQjzWT79Iytv8Ua5PvE740KsSH782EXf0gFxfmT9wXQQG6B4X4EFws5iwzj4xTxOr+RnI8xBIpqn5TdwFw+VRhXgcpX3DUjDeZCs+yZHLmINUFdlowDswL4zkhdOUPfYAk\\\\\\\/7CFnivdlbj6M0gH+eHlOxyLc5sMVJF2il98v+f3MB+HJhZxDSOT05rd1xc0fp5UcN7imq9RNAltbKjVPFTGIgoZPUFKlq1g1Wan4vD94pPhPqL6vVo4nO4wo0pJxfUdyvkvO2PNiK41L78aYjrfOR0jSgQ+YOWN1jUXIuKTAnk33ywCZ9C7yDuuVFuzA1VYkfIr3F\\\\\\\/otfccrfFt1lFcHFwRqjrAjwi870gctR0mWnxbPua1h6B+KbPxBuPjTTrOmr9lqPomaXAmnN70bdk8H\\\\\\\/xoGzIDB\\\\\\\/oq9Lo1zhOq2oG0\\\\\\\/4l7Qp72Tb0N\\\\\\\/q8IeUGDrIyEoFkXNtaBwy0vGjmm7ndEx1p5kgPpIMyswC9qWuEO3BcaBmmbNfbpPO+WF1cQPcmP5mswIBaarQp\\\\\\\/qcBoTLJgayH\\\\\\\/gGif6guCA9dAts4gGVcqMsH8Z6yHz7qH6gtZ+I707ibUs9K6wFErI2HiwpqWHfStg9A+eqozW7z6RthAWL6RszoPuWHRtHaEqckAgQt1p6J1310zpfhCRbqTy\\\\\\\/GL8FZWwWPGNrPNNagQq7oUU12QC9iKrEP3zqQfwxMCpGWy5RNJYVPCLEuAbuLYUiZe2TybusWHHPEdiNG3d4rWR1YW5M2GrvoOEGeCc23TylLLCreao4AHbRCZTjQhALJuf3PjjT\\\\\\\/tqtQDa0p1dAYkhG9FmMzwHXuM1WsdX0lVeFH1MCys23FIXo7SmicpH9QUgJs\\\\\\\/uYemDrYbpO41s6bCRkxsyXsOxKa6z10Lpe3S6H9Z8mUNWSmEDCvnFY3nJxGo191rQ\\\\\\\/ly0yZjFNvdYk809K4rTvzxizEsK1N4d\\\\\\\/aKxbrfIiyxtks6CanNw18b8dCydvI8HZkjVqHbRwswXXQmALhMC7wFveIUwamGmSQiZPG7V1mnOlMppcHpFmb2q7tP1MP4d+G00prpCyOx8zJjXJi8UEjQPPicmzYlrSglrASqb02cZnHUHVozTa8FaO0lOz5M+gw9+Wc7nmI4T5f9+ooWFY\\\\\\\/FvuOeFYARHxnhgqV3wmDb2SUVzEPAv\\\\\\\/zhoawxxlalOLZeDyHcF\\\\\\\/MZWOyKRMR19ae7UsYue128jNC2T6opfrvtViwcDEUylVH8S+Qu+OPNaQ6N7h41QGYKANmChgBOCkz22n49WWOKvRa7S6DzxyoQ+Wv5tYZZrTOHkhEaoDVOGLXiiINlTeShdP1hMUwIOHuymEH0WqgTdwmsPnWUgZPKIiVHA2HUDK+n5iiQOjU\\\\\\\/nD\\\\\\\/g7ogreLa0tx++z5sNrWaKJTJKtzjwLwOWuTeW4xGD\\\\\\\/JIgROQzUGjFs4Iq3Qs3Sq0KJjPXGYQGNbCblskocQnsPTf\\\\\\\/m+lcY+UmitkF4KjQqZx3i8hLF3oo5ncsCiClCR6OxqjgTn4d1+FndFtftupB+rbcblgtxt4ZpLv\\\\\\\/HgzvarJw5vSZMsPSj4wtLB0u7\\\\\\\/gvsCUsSaQ6Qz1xszAAqrO7vkd6eD+C8X93cgY\\\\\\\/jkTO2DlRRyhbAgCJAxYMYQCYR2C7F4Hdm0qdAUFNwOwpyj35pvL9zEXpRpho5tenscc5vnMAW41GKwP2RfeSvNAS\\\\\\\/7xIZuztfb1n26I913bEhlf7fkzAjzWGQ1OMZsuHhnt13QggmvJeCBnhPK1Vsqd\\\\\\\/pXGjryoZadUgP+zJahnQ\\\\\\\/Oiy1M5tXvI0g8ZSIvP2wxIr+FdcUld\\\\\\\/k5bIsfpzCExdxFcDNWD6XReGindAo2+XNkmGt9xQfjQB\\\\\\\/wFKEbu3ejTdcHiiQ3Yj12xRKVW5wX\\\\\\\/qm0yBBWtOJG0NCOrmLt5HfaL1I9c5cDi5BoeDK4JcM1ay3X2EanAl2zORKf3NLm5WmoUMaQgAnRSA+IqiE8GjWLNUvqdh4r2MxqWklzK\\\\\\\/4+4YJRQ1lJ0XCO0wyyJddvloet60ksekaNhLuDVqxe1KaXdQpOM4zchi19wxhKAwFS2zdj84\\\\\\\/5ZEP7fjLZmiVvqSEPOh+LqdBEycVWEqKAFvRBK7kmipVsBNNrJ2jMMEZcSXlfclSpbanvCl5mDRCEl3M10CaToeUwTGszdVfGqOa37wFJmm2FSrm7v1C\\\\\\\/FR90Q2xy00QtTy+F6FylJUHpVYi5J4\\\\\\\/02dMrURGeXwsdEEJqOplzO78K9G4eIyZmN6Rh7BfEf5NMEWefF8cL4EIIZ+0YZk0Nb8+qyCUl+qDzWzFEWDr183MLc5aUQd0FV3bquLlb4KOfOqTpT1QlHIu3+6V4DHXy+bfbIPKB5Z40O807l2ulSSOinuYSO8JMgLKqCrxgui25rP3huKYo46Ni\\\\\\\/4KZmk0OYggDlfa2Vn2Pv0IbZmSxc2z3XeK2F7Vq67ZOKP7ZBPrWpBwOytgeH6nHObuYdgUpTDYzaQQ8T93Rvc0OiffjMNfnOGk3oDopsYT+S7EkyosUJh6fKNdC6IcH6b9ny9w92CMrJEwDk5vp6EprryEjEZMEDRcOZ9dfKrK3KxydBffeuA9Y1O1DKQ1\\\\\\\/8eCwzXbXpTWvh+CNGrLVEa\\\\\\\/6gJrva7SFgCRrWN3EU7HHXMGjHv4XnkH4reTRZWAWb5u\\\\\\\/1hByiJna5XWTIZRcpFCPXkkCH+fA14iaJ4rs518JNxHaylQtkjEAnT5Qj\\\\\\\/bm4MePU0U0HqGP\\\\\\\/6\\\\\\\/tlRzJXl4SdH5AYmrfkZbkIdpbgTcjDjCEJYvme+YqJ\\\\\\\/UfVoDbf24KFc\\\\\\\/8om8DuvhOgBhDbT0xYsEA\\\\\\\/2blne8booSzLIeXx21RgobQD3cs0nI0ggLTkZC3De6ky9UuqRZ5DykQadqf3p8Q4VRN88hKo\\\\\\\/ogJgGsoClYb0JGX+A0zVXxan+HP3Uu8asq8vyKv7Ut\\\\\\\/rgGNpUYPonPYf7YRQpzsEbVw4DEw+R0cVf1xEszhtr4QFqO2A9cXJCGOA9YgQcIAk7TXO9DRXZoqBzVA\\\\\\\/CJggfZX3cQpyMU9jMBs4RApk3C+GrWZ2XzFJ\\\\\\\/DolGr0jhjE3NzPAeMmSwdLCL+nK8JkQCPY9838QDlSvwq9IVX5CEKkKhrAPA8DwxhfY5pTA840bgweI7rvuTjCmHSwp050sm1dI4qS2HM9TbGUNdEK8xyHb4EEUQ88ZgcKyRal\\\\\\\/rLPoTmJHH\\\\\\\/UgwRZfqJzAWwc48r8e3uUEwb5lOL4HMe4R6Wh+65xC8f+TnFrAGpdVuecxTThKy0h0YbTaE4uupDX3YPxv774PEdWmg4g9fbavsa7enEROZFhFjp1lJ+cFxVU3Ia2UQHqAyw7Qh75iIJbFGBR3BjeoYFHWAMWxZzk6rUZ9gdYcqWdCUhF04Y+11LIfMFXkO0KJ\\\\\\\/ZQusIIY8qCeHL8N9ogFzeSNG1QsnV61bmcUEsBDQMnB\\\\\\\/X\\\\\\\/wcvigIvdhVnB4xSZZTydy+9vXG9z0GKDG\\\\\\\/9MwfMtQLFeyiZMUXsY5\\\\\\\/BgnsdWnmiO\\\\\\\/na0OC5twqBgDE0uIk8jNAtaWAoBFKcOfHbDOqS3\\\\\\\/eWKjv+vJx4EIBzTTTCctBS0cROadeZpFKdGEP0ZOpxqwZWhKbDegsK3yhrWIieFrpA7j5eZPpAvdfeUZOnyNbx0WDp8oE7qUT+E4LtHgdSfgU9R5kTIFOwePSAMiGf2lRegHgiFu1ovniM+wAkccszfQZkwgXOyDkQMF3OznmBnvf8AXNhN\\\\\\\/RM\\\\\\\/HAv3fWUWTvZBtOdFMSVDiAllielfqGb5gl+SftvDBMkVXL8nJk0lWeiFzYGJj4T5\\\\\\\/khoXI9HiQX0VrYmFjVN+WyK7sKjgxtOOyfe1GhYC3LTedFTUj4yjCEJN91MwjMqRgKT1UQSj9zac3zB+cc7CYgLHLfnTzhkd4APEHOOMNouZ5lBnucuehPImbJeA7L60U77rE5aWloBvQMxU764ThSG+urCU9L1neijV19bdeqF6wu4yMRD99UJjJXc6bE9arkZbOFdlJcp\\\\\\\/seLa4NgxVWUVnpgUE4qzCdwfibFJ18IWGrxmBdmY3VvakaVBlw+L4BvAPYUYxkAfyLbVGbk2hNCkLnW2UItNTrRIkB4N6VJGHaHOmpp1SktxuV4c9knVlhvgwdhDAR8WyzrWr27c3NEkNtSUnsmh5kotZudDSsAhJ2LbKp86pilSIk\\\\\\\/rTJfayPezVC7TW0PJ0v0fmh9uC30jv1ZPiFVtoWYT2V3qjgmyzRYMtKooucUBJ2oK3XTSIhwfEh0HFoUwR9wb2pNEulpyz552f2mfNLIgRa++fR8ARqFl56B5s7l1c+Vo3mrsNWsgpqRRAkaf2wIxRl2jDXvU0x0A3GsQ1Bw1wg7zxr3sRRhUo0rnYigcI6Z7bD4iRtVAG3Ucx+VKgvTJaBJDUcIWy7\\\\\\\/q9JCNRizkQyRsKZQVgYIJ4dUTxCEeNe60J4P\\\\\\\/ptS1ugu5O\\\\\\\/GjVs+2Mi8U3nbNG3p\\\\\\\/NjNXk9KW0lXnt01iTcLzrEF9DOUoimqdvmSbXzE92SRQ54LrzY9L8GRE2zeZlVs1ex3+aRf5RELZKAS4xbPvgQldwFcRR8CTTQ8zbHlwpO58tlIbkoS2QcjcJ7+4finkNzoFJcHQ5DZ4Ycmj\\\\\\\/NYUVEbpV85cDft1Asc+2ukBb6CBKuLN7clwwPpKlCQep3OPj5SkoRUli9cMj\\\\\\\/ssPcTbwKMOENSQOjNv0yW69Q2WBK7\\\\\\\/b0DdeKcjGj24d5f7qTJqce5qmyJD8aMiBq2e+G47aTEef+a+CQWZ6blxGo5QIcluO\\\\\\\/rBCTWeglpBqSCprIrgT27Los5y5rkQBp5VSZM4GdQdUQbUIQyBj6yh7JSzwXK2Fz+1Q8xeXZlDgZPjdv3iRXTTzm7SS9NB5rK+LU7IKKU9BxrEMVfC3I1EOdLWr06SUJq+S8EWQI7A9q6lFJJOFV3yA83XY6NMIGP+fE+G2rmkAdH3hSRZ1fqPb8gS0Fr01i2OKEhHzWRzb7LEIyG9Rs29NJz\\\\\\\/5t2bnEOw4MPiAmP00VT09JyMBD2wgRLSHCC92k\\\\\\\/b3YlCFGraIWdwmd22+vtmT8HFGryN3lqgcOESiun21hhejTpYQ8VF3L0JmhkCBBblHswt2tNL+g5X717lqgsnOOlHafAmzZuIijb0\\\\\\\/CCbYy4lLYUNv8WUu+d1fyIr6R6zQZpCBp3R4BK+\\\\\\\/Q8G59B7Xa64bFeUPshJ3JuxFs2fy\\\\\\\/LIf0RYwwLKCnRddYf6wuz35S+gQmcv9KmOxibtrkzN7hMADOiOk+2CM8V+6BOptHEbgT5+bNpLqVT1Stwt2MAHFJYNy5CjaPe5GKRv8PcOTTDa6OaMvgvhtXpoBJeX7fvFtRwyGQecf1MORilLXvPvcoY+JRtyeicf3c26k\\\\\\\/BbCeWjKKhb37mdCmyPHUA2M3hkXLABgKe4YUfvDwBHkmZzMcfyQeMgzYOSeHMuGFHACIBQB4rHCMjdj3SVEOoNq2rlqiBCbeml8AqIL+KIsLlD8zbl3tWCZLUtHBREHkZXy6Br9l6\\\\\\\/bmJdhDIaFSFf+mbLnj\\\\\\\/UrsT\\\\\\\/882ivjEfB+kEjwMtzjm25pIb2xxec7meny0L4FvTk7KJxzkJH80J6wdBHW+KRDc6tZqfZZCsLtYJK+K2phBl\\\\\\\/lvH7KIzlMIOGBlvXwE1\\\\\\\/4UQgoCsnToQeeVi1zOMX59Twp0ZihEDFOAbFCdu22qKnFVCyxIqrZd7gpUrHdp1h2Mt5EVVt3TFeDVqCZgVWG5KSp4TnoOFAa7MDEVAg26K6TDxLGfzIHosgs846pOT+BwO74hLqdfYxBx3B3LErdUcLNUaeRdBRhYc4k4rS\\\\\\\/LFtG3d7jPo3bPETLyIuVPCgUorga2XwthmyxL2EbLhLTeqg1sjceeqXZ3Pmvmw+vMvu9LjUUYxB\\\\\\\/Y\\\\\\\/jzZlbthN1qxJM\\\\\\\/Y6fka2jZYUPj0HypyXeteuytChuM2R+b8xyV8eyhv\\\\\\\/Of6e3r3iubuhe51xNr33unxO6BJleWvcjgooKNcbIcGX3lZU\\\\\\\/98eNraip5NtL4lkSlEspRjselikyCRkAcBPYr3mL70e+Lt7\\\\\\\/o2+V\\\\\\\/0llQXEcPAOqXFZmDRTkrvKzyVeoxlpNA12no7OPCqUkqRUEoY\\\\\\\/G9CNgbXm7DABhQvVXlUmNET6t8CE7S\\\\\\\/+KKeYphyKVb\\\\\\\/8LvsSHHWxnsz78bBO7t9rkU0fqWQmkpqNo64BplJcc+gul2nuQcszC6VjzwAvEgGbebaXGCqaRGePvQF97n5HcgKwRHE0p6NChGCwMKOuPuny7qLnEGWdy5kCdoxSgcdz+9q2q08v17JtmYrw9YExBM3mpXCSgOW63zS+1\\\\\\\/kwmwLctSFBU92UV34DnZobUOZW\\\\\\\/XtCNBCAXlCcOOMDt9abdj9+PSD73UmzA\\\\\\\/wH+4xf49pARGEEyqkjxFBhUV3qpRlYTl6ub5G3yEnxnqz9E7PY7riITsDKjwosTSpJNSQ15hgoLIopj2M\\\\\\\/Wo1kxnRNxyRTOd\\\\\\\/fSSmDWWuSH91ygHSD6z+DO4erSM\\\\\\\/NjfGtlZHuoTohAwgqlggGG2xL+YyUtXEEBxdoCFqBEYLoEvcRBkHIKsTZRYwbhnBu+\\\\\\\/fcKlcCPaOHVLzs0NbvKh2ObOBedjUD5dH8BSzr6tvbZV6y3kHhywcvQCfhK1pUr1r8\\\\\\\/IzYHFKDN9HRnsVB\\\\\\\/uhlJ8Ws1IZOAVVDiU56JtHLnll7W4ZMpRzSxFBEmSbj0qHhU9vK8DdIYeOBeLOIh53AoGGwIKJF1yrO0pcBWpJmOPcvVBS0Ubif+avXWqJIbQ7OaLgTLhrFiUl2dghUhaS6dIKnvejxQMeWmJXPLJdMTtL99Eipy\\\\\\\/O\\\\\\\/iotvBstqrOkT\\\\\\\/7LsQGwpKU8283Ve\\\\\\\/5DViKIapDGF++UUqdsT5zdO2D4dF3jUKW5+Ar+U\\\\\\\/x\\\\\\\/aCJFcnMGDz0Bl1+MYXwKz3ZwKUAw3fQW4YEb3ogo7qcOwggf9jx5tD\\\\\\\/xUBvdnfjIMMJEbImZ8T\\\\\\\/bo8+swKVJpbIUX+HabadUsgOuDLeZseVDBqjiJ29qqdGfo8po3zc9SOPBZ\\\\\\\/UFYa7n6Ce4r+FHvBJy4O7wM3ec+RZ9EpcuH8JjzsZJGuO2Z5oqRX\\\\\\\/PFrr1bMYW+EhGRyz0LcFBtp9ht14Q2D1UKktvEZCnDosliXGkHr9c1YQd2d1\\\\\\\/X6FRW0wtRaNB\\\\\\\/tixH7JeT+TBkUZh5TaJwo++dQzdClzYs24U0WDRooHGmN+3clsbtoUQg1caHhxpD96jJdyJRiucMvD\\\\\\\/epDOtzcMxGOTUTVKRAbVj05US10gvanlOGWP0YD4qeZfd7eR22KERQt0Rvht0ZJdI0WO3adsR6JuOX1o7H3fEbeGmPXybcuQF7wBH9SMQO9++m9gUvxgNzJKPr+5lx+zWT5SWdNshooXbwUvvY\\\\\\\/EvkVgw7MN5ozpSlZoi9RPI+MQ17kx2l+CoEG0FgtTh2hnKuKIxdL5rem5zdD5NzKg2U7atl7kkYpgPcjwuzVR6odIBbkCdT1XxCHF2mkLOinQswtfan4NK+zCGR8zkh5XlaqQ9Hne+d743AgpZFhaNWL+GEiSucVw\\\\\\\/mRBZRsJVbprW0AwwM7mlEA9YA8O5kSSuLMab10mp3HM4CQrjG5prvMNymFyPEKeFnZMHOEmozJ7IM2r0y7UGj14HoICCxMfLlDtLtiVhUeH\\\\\\\/MhyLbhRUFX9Gr1fSc7Ap5k4Y1ajoMvU4yS7PDRmlb6nA\\\\\\\/bfqIWUeBep55nXilvepdn+b0okS5tlt2LyUZ5V8a5y7RBkFZtLNw46WDRwKfretGeeO1A\\\\\\\/ULke9yOOqX1NtfsbX8ozmQnhZPNbHeRZe8CScWixYpmyFsHwr8Ec8EwtZbsu0oC9W1P+sCS01ODLvo8HCqEwBF3Xy9PymmYoWWRK5T1LEZYj\\\\\\\/r\\\\\\\/xLZQGSIgmeJK5Wtzx+\\\\\\\/5rak0c3oAicFPkyW4yCCjvjooxcMR9qu4YSm7R8cly+vQwVvqa6M0Yp6uQUzMFmJhUejy0QM3dcwhRbz3janoIpAwKsdSa0K9hngDWmCv4q4VWGyTtLvXAf\\\\\\\/B0AXIOj+NG\\\u0022,\\\u0022iv\\\u0022:\\\u0022dcs1hWYzGv8DUP4yw4hNhg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f6381208488132256d2e58c274a5a9e74fd59b7b07b4bcacfe82384f5335046\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022q\\\\\\\/x1fYO1pMUkDg4+MATLLce7z0X99p\\\\\\\/T1TcOv8IMowaEH3h4TfammOYPLPugJ6TCK8DnPhr7Db12H8ULg5BSYl3n8R4eikQObp12VROsWzSWx1oXxYfp5KnGmiyunyDaBvo++8kUrNG2zZ\\\\\\\/3q93M8XEmJaaffL\\\\\\\/y2A41VC7tnypf\\\\\\\/vSY\\\\\\\/JbvDc6dcX2MWO2TeiTyiJpKCxnZCull8Z51aEh8k5VwUZlQnaUnZ0gEyUKiTRkTKNyfzjgmkbRXR\\\\\\\/ADJ66zjqn51MwLzTOV2D0QbUhemQru2vMI6NqYMs4MphGG9L\\\\\\\/j0AMe4MNfxYEo8X\\\\\\\/L3i99wqxI\\\\\\\/tIVrMvMXIDP5g85TpAqvN0flUdXCBuBwW5bMDRA2tVKPy\\\\\\\/zywT0UYVtD9y9McHWMxckc6XxHTH\\\\\\\/AHleuGf5t4I4xXX28EykKjp+X9G+QtzCxVoR6i2Uw7Cjua0FU\\\\\\\/+9cBA5mire7AYZTx0APVLhHdcf3yd9T7zsr7EZML4s3k7IFauxio1OP7eUYrdYxJ4b39A\\\\\\\/O9F54wL2lJ4Z\\\\\\\/9baOdOEzW09zpH0TM8Ja0STG6lilGKxsSbHnYE9E0yQp+VKQwetk5D3KtEbvmilw4gPL5DLz7W0CkeZ3nMDdcNOT0oPRWEHR9NygL99tvEUEd917YshHSrltBAc\\\\\\\/gAcWpABJzFjhH\\\\\\\/EhPWgSzNcWK3X82BLMin\\\\\\\/usX2qq0xLaG6rmKTVdXqwrgji2iT2JDD\\\\\\\/cHou\\\\\\\/p5tcLhtoTgwQXTyfVn5dZWTB9fxg5\\\\\\\/Md\\\\\\\/FD0JU1upjjIyAYP7GetewQzSoRxmYZVOKFIbVzYvlvw2JftdVfXLOjJwdqrkE3b+G6Aa+GmnTItHEttRUzrPaP8nS9b2Gp\\\\\\\/9Z0qgIySbG7R\\\\\\\/gQn9XALdeKlz\\\\\\\/P9Gay+LSTmH29l2p2JKedbkqbA\\\\\\\/vArj5BQ+iPbAEGLWZOMpH\\\\\\\/36atNtT96bPo976vyhxs4NcMQbgVkqLI0Pr93J4WRX7gZAWojfwVdzyTmPgk5Yk\\\\\\\/TB+jHR1stjLOZB7khIKk5QeHZ8YKrYxCc4eIwk+1YAZsWBQKoEPFYcW+3SUN1OGY78YBId4RsuPUb6kwoHKGXf2KicbYXTerQN9VdCaMx5m3CSCSEjJJXqCEB6xgHIObvnFbB8c+dpcvEX12ZdF4UjBJqDR2qEG9b9hsEk9Y796wZD7iUqPDSfYtEt7LgHCvUbk3JRLazk\\\\\\\/56qNdOHLUg4ESPQlxy6BdKJ4dWS+mLSagMrpj8nJWoKYBLArA\\\\\\\/Gw2X8vKDXtGBXzIwT6HpUnUkRI15cGScKJYOdvJr8Lku6\\\\\\\/hFO7P5RhGayai0C2RuH\\\\\\\/e0SfRoEpIn9NmmhC4+A+HRsGP8kGeQZ8WJ6iMWKq3ObsZ5auRBZg6hWlZpO+mbVHskkOj+3RL0imQsCErnEk0n6sfPlk6j15OKRqKl3EZeSKx5BFbY\\\\\\\/7rQ7Rj13zG12rMobExxcI6Z+IV94ZAnMnQ6LV0ue6LdlZQkNJhd85q\\\\\\\/WQWNHaKAFb4u7qdhY+ciZP3GBNB9QA3cssBzOoqtM3Q3ZFcm\\\\\\\/hE5FHcHQh\\\\\\\/PT5mABPvOSd\\\\\\\/BL534OL8OmKsenoVN3z\\\\\\\/4RwS03F2ddND2A54rjfNGW07YWmcl9HrIoE3ji+\\\\\\\/jEnN3tVLeCT53k7trrySA3Vq+9Ly0Qt8oEaU7SoHOr3FNsvN9LwDWLdUQ7q\\\\\\\/uVnZj7uadLCEJo0qZbTFJNmxMSVDUNVYmix\\\\\\\/O5Up1yOc63n9sZQ6aBUcc66WJ+Ayox7sl0SWxpCTLfF7hKH1ts4FM8EZYi7xjKvuFApZc50LHIXMYZeBEUTZsux7QimHTZ\\\\\\\/Ffbz40mYR9fuSJXN\\\\\\\/DYw4aR0CKIAmMWZM7+Mw4+ow3A5IbtSwDA2HCzMjfSJAaKREsA5HY8dYMnkKhNYb0TOt0DSyQZBLTef2kZTHLiledKeOG0YFKwc7KDu5fYLoRgA77NmVr6uGUAq6vboWpQcS6+DruUHb2vdhuLBdj77TfUjinyYVodRO5ifMtPJtWcP31owKYeMWQFYZ1Gy5WW5tdk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022YZ6Ty7EAy2m1Z2bv5Z1XSg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289646554abb41bb58785b764914287a946849188aeb897a4d9a1d161181d2166\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459648 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/539" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-crLBT_ebFXuozPntzDGaVmWp2TrZUB1GG0ppOx4_pCU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/539" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Z2K--LHspnU8YUG80M5jhBfWhFDr8WEviVr3zPnGz1Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/523" title="The Yeast Ess1 Prolyl Isomerase Controls Swi6 and Whi5 Nuclear Localization" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/547" title="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459648" id="node459648" data-pisa="ggg;4/3/539" data-pisa-master="ggg;g3.113.010025" data-apath="/ggg/4/3/539.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nina Hofheinz</span> and <span class="highwire-citation-author" data-delta="1">Matthias Frisch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">539-546;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010025 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459648" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nina Hofheinz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BHofheinz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofheinz%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BHofheinz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthias Frisch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BFrisch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frisch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthias%2BFrisch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthias.frisch@uni-giessen.de" class="" data-icon-position="" data-hide-link-title="0">matthias.frisch@uni-giessen.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/539" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459648" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459648/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/539.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459648" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459648/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/539.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459648" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459648/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/539.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459648" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459648/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/539.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459648" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459648/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Ridge regression with heteroscedastic marker variances provides an alternative to Bayesian genome-wide prediction methods. Our objectives were to suggest new methods to determine marker-specific shrinkage factors for heteroscedastic ridge regression and to investigate their properties with respect to computational efficiency and accuracy of estimated effects. We analyzed published data sets of maize, wheat, and sugar beet as well as simulated data with the new methods. Ridge regression with shrinkage factors that were proportional to single-marker analysis of variance estimates of variance components (<em>i.e.</em>, RRWA) was the fastest method. It required computation times of less than 1 sec for medium-sized data sets, which have dimensions that are common in plant breeding. A modification of the expectation-maximization algorithm that yields heteroscedastic marker variances (<em>i.e.</em>, RMLV) resulted in the most accurate marker effect estimates. It outperformed the homoscedastic ridge regression approach for best linear unbiased prediction in particular for situations with high marker density and strong linkage disequilibrium along the chromosomes, a situation that occurs often in plant breeding populations. We conclude that the RRWA and RMLV approaches provide alternatives to the commonly used Bayesian methods, in particular for applications in which computational feasibility or accuracy of effect estimates are important, such as detection or functional analysis of genes or planning crosses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-wide-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-prediction" rel="nofollow">genome-wide prediction</a></li><li class="kwd"><a href="/keyword/ridge-regression" class="hw-term hw-article-keyword hw-article-keyword-ridge-regression" rel="nofollow">ridge regression</a></li><li class="kwd"><a href="/keyword/heteroscedastic-marker-variances" class="hw-term hw-article-keyword hw-article-keyword-heteroscedastic-marker-variances" rel="nofollow">heteroscedastic marker variances</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/plant-breeding-populations" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding-populations" rel="nofollow">plant breeding populations</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-4">Best linear unbiased prediction (BLUP) and Bayesian approaches were suggested by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Meuwissen <em>et al.</em> (2001)</a> for predicting genotypic values with DNA markers. These genome-wide prediction (GWP) approaches have proven to be useful in plant breeding populations (<em>cf</em>. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Hofheinz <em>et al.</em> 2012</a>). To overcome the problem of overparameterization triggered by more available marker data (p) than number of observations (n), shrinkage factors (ridge regression; BLUP) or variable selection (Bayesian approaches) can be used. Shrinkage factors can be constant for all markers or marker-specific with the use of homo- or heteroscedastic genetic variances.</p><p id="p-5">Homoscedastic genetic variances at all markers in the linear model are regarded as a major shortcoming of the BLUP approach because many traits are assumed to be controlled by only a subset of the genes of an individual, not by all of them. This shortcoming motivated the development of Bayesian approaches that allow for heteroscedastic marker variances but at the expense of being computationally demanding (<em>cf</em>. <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Shepherd <em>et al.</em> 2010</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Kärkkäinen and Sillanpää 2012</a>). To avoid the computational demands of Bayesian approaches, a linear model approach that uses heteroscedastic marker variances for data sets with more genotypes than markers was proposed by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Piepho (2009)</a>. The generalized ridge regression (heteroscedastic effects model, or HEM) of <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> (2013)</a> also allows marker-specific shrinkage for overparameterized situations. These authors emphasized the need for computationally efficient GWP approaches with heteroscedastic marker variances.</p><p id="p-6">The accuracy of the predicted genotypic values for GWP approaches with homoscedastic and heteroscedastic marker variances was compared, <em>e.g.</em>, for fruit traits in apple (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Kumar <em>et al.</em> 2012</a>), <em>Fusarium</em> head blight resistance in barley (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lorenz <em>et al.</em> 2012</a>), 13 traits important in wheat breeding (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Heffner <em>et al.</em> 2011</a>), and for eight data sets in wheat, barley, <em>Arabidopsis</em>, and maize data sets (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Heslot <em>et al.</em> 2012</a>). The common conclusion was that in most instances the accuracy of predicting genotypic values was comparable for the investigated approaches. In particular, (1) none of the approaches was clearly superior under a broad range of applications; and (2) the BLUP approaches proved to provide good prediction accuracies, even for traits that are not supposed to follow closely the infinitesimal model of quantitative genetics, such as resistances. This finding was confirmed in a simulation study by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Wimmer <em>et al.</em> (2013)</a>, who recommend the use of BLUP in plant breeding populations with large linkage disequilibrium (LD) extent, small sample sizes, and medium trait heritabilities.</p><p id="p-7">The focus of the aforementioned studies was on the prediction of genotypic values of the individuals of a prediction set, and high prediction accuracies were observed when the individuals of the training and the prediction set were related (<em>cf</em>. <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Hofheinz <em>et al.</em> 2012</a>). If training and prediction sets are a finite population of related individuals, then long chromosome stretches are expected to be in LD. In such populations, it is sufficient for a high prediction accuracy of genotypic values that the effects of chromosome stretches in LD are estimated with high accuracy. A high accuracy of estimating the effects of single markers is not necessary. Even if the estimated effects of single markers might be different for the different GWP approaches, the sum of the effects on a chromosome stretch in LD might be of similar size. This can be regarded as an explanation why different GWP approaches with homoscedastic and heteroscedastic variances result in a prediction of gentoypic values of similar accuracy.</p><p id="p-8">The focus of this research lies on the accuracy of GWP approaches with respect to estimating the effects of single markers. This accuracy is important for the identification and functional analysis of genes, for the identification of target genes for marker-assisted gene introgression programs, and for the prediction of the performance of crosses. Predicting crosses, <em>i.e.</em>, estimating expectation and variance of the performance of a population derived from a cross of two parental genotypes, is an application of GWP in which plant breeders have high expectations, but no reports of successful implementations have been published. Predicting crosses builds on modeling the breaking up of existing LD and the recombination of favorable alleles originating from the two parents of a cross. Both the accurate localization of markers linked to the investigated trait and the accurate estimation of the effects via a GWP approach are of central importance for the success of such a prediction.</p><p id="p-9">Our objectives were (1) to present novel heteroscedastic ridge regression approaches that improve existing approaches with respect to computational efficiency or accuracy of effect estimates and (2) to demonstrate their properties with computer simulations and with data sets of maize, wheat, and sugar beet.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Linear model</h3><p id="p-10">For estimating the genetic effects of <em>m</em> biallelic single-nucleotide polymorphism (SNP) markers, a linear model, as follows, can be used:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span><strong>y</strong> is the vector of <em>N</em> phenotypic values, <em>β</em><sub>0</sub> is a fixed intercept, <strong>Z</strong> is the design matrix relating the marker data to genotypes, <strong>u</strong> is the vector of genetic effects, and <strong>e</strong> is the vector of residuals. The elements of <strong>Z</strong> are coded as linear regression on the number of one of the two alleles, <em>i.e.</em>, as 0,1,2. The genetic effects <em>u<sub>l</sub></em> (<em>l</em> = 1…<em>m</em>) and the residuals are normally distributed with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-3.gif"/></noscript></span></span> (<em>k</em> = 0…<em>N</em>). Furthermore, cov(<em>u<sub>i</sub></em>, <em>u<sub>j</sub></em>) = 0 (<em>i</em> ≠ <em>j</em>) and cov(<em>e<sub>k</sub></em>, <em>e<sub>l</sub></em>) = 0 (<em>k</em> ≠ <em>l</em>).</p><p id="p-11">In ridge regression, the genetic effects <em>u<sub>l</sub></em> are predicted by solving the following mixed-model equations<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <strong>Λ</strong> is a diagonal matrix that defines the amount of shrinkage. If its elements <em>λ<sub>l</sub></em> (<em>l</em> = 1…<em>m</em>) are defined as <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-6.gif"/></noscript></span></span> for all <em>l</em>, <em>k</em> ∈ {1…<em>m</em>}, then the predictions <em>u<sub>l</sub></em> are the BLUPs (<em>cf</em>. <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Piepho 2009</a>). This approach uses typically variance components <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-8.gif"/></noscript></span></span> estimated from the data set under investigation.</p><p id="p-12">In an approximative approach, preliminary rule of thumb estimates of the heritability <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-9.gif"/></noscript></span></span> can be used to define <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-10.gif"/></noscript></span></span> (ridge regression employing preliminary estimates of the heritability (RIR), <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Hofheinz <em>et al.</em> 2012</a>). In the following, we suggest approaches to determine marker-specific shrinkage parameters <em>λ<sub>l</sub></em> for ridge regression.</p></div><div id="sec-3" class="subsection"><h3>Shrinkage by single-marker variance component estimates</h3><p id="p-13">A moment estimator of the variance component for each marker can be obtained from a random single-factor analysis of variance (ANOVA) as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>MQM<em><sub>l</sub></em> and MQE<em><sub>l</sub></em> are the mean squares due to the marker and the error in the ANOVA for the <em>l</em>-th marker, <em>N</em> is the total number of individuals, and <em>n<sub>i</sub></em> (<em>i</em> = 1,2,3) are the numbers of individuals in the three marker classes.</p><p id="p-14">The <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-12.gif"/></noscript></span></span> are not independent and, therefore, they do not sum up to the genetic variance, which means that they cannot be used directly to determine the shrinkage factor. However, they can be used to partition the total genetic variance to the individual markers:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Here, the proportion of the genetic variance that is assigned to a marker <em>l</em> is proportional to the contribution of the single-marker ANOVA variance component of marker <em>l</em> to the sum of the single marker variance components of all markers. This results in shrinkage factors<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>The approach used to determine the shrinkage factors in Equation 5 is abbreviated as RMLA (<em>i.e.</em>, estimation of the error and genetic variance components with restricted maximum likelihood and partitioning according to ANOVA variance components).</p><p id="p-15">The estimation of the genetic and error variance components from the data set under consideration can be replaced by using preliminary estimates of the heritability <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-15.gif"/></noscript></span></span> as suggested by <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Hofheinz <em>et al.</em> (2012)</a>. This results in shrinkage factors<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-16.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>We abbreviate this procedure RRWA (<em>i.e.</em>, ridge regression with weighing factors according to ANOVA variance components).</p></div><div id="sec-4" class="subsection"><h3>Shrinkage by fixing the residual variance in variance component estimation</h3><p id="p-16">BLUPs of <strong>u</strong> in a linear model as defined by Equation 1 can be obtained with an iterative procedure on basis of the expectation-maximization algorithm (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Searle <em>et al.</em> 1992</a>) that consists of solving the mixed-model equations in Equation 2 for the parameter vector and then solving the following,<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>for the variance components until convergence is reached (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Misztal and Schaeffer 1986</a>). Here <em>q<sub>l</sub></em> is the number columns of the design matrix <strong>Z</strong> that correspond to the variance component <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-18.gif"/></noscript></span></span> and tr<strong>C</strong><em><sub>ll</sub></em> is the trace of the inverse of the coefficient matrix of Equation 2 that corresponds to the variance component.</p><p id="p-17">If <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-19.gif"/></noscript></span></span> (<em>l</em>, <em>k</em> ∈ {1…<em>m</em>}) is the constant variance of marker effects, <strong>C</strong><em><sub>ll</sub></em> is the complete coefficient matrix, and <em>q<sub>l</sub></em> the number of columns of <strong>Z</strong> (assuming full column rank), then the procedure can be used to obtain the variance components that yield the BLUPs.</p><p id="p-18">A modification can be used to determine marker-specific shrinkage factors for ridge regression. First, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-20.gif"/></noscript></span></span> is estimated as with BLUP. Then, the iterative procedure is repeated, but with two modifications: (1) The residual error <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-21.gif"/></noscript></span></span> is not updated in each iteration round but instead the residual variance is held fixed for the value estimated in the first round. (2) For each marker, a different <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-22.gif"/></noscript></span></span> is estimated. This results in <em>m</em> values for <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-23.gif"/></noscript></span></span> and those are used to define the shrinkage factor for ridge regression as <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/539/embed/mml-math-24.gif"/></noscript></span></span>. We abbreviate this procedure RMLV (<em>i.e.</em>, modification of the restricted maximum likelihood procedure that yields heteroscedastic variances).</p></div><div id="sec-5" class="subsection"><h3>Software</h3><p id="p-19">We implemented the RIR, RMLA, RRWA, and RMLV approaches in our software SelectionTools (<a href="http://www.uni-giessen.de/population-genetics/downloads">www.uni-giessen.de/population-genetics/downloads</a>), which was also used for computer simulations. To perform reparametrized BLUP we used the R package rrBlupMethod6 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Piepho <em>et al.</em> 2012</a>). The package BLR (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Pérez <em>et al.</em> 2010</a>) was applied for performing the Bayesian LASSO (BL). We used 1500 iterations and discarded the first 500 iterations as burn-in. The R package bigRR (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> 2013</a>) was used for the HEM approach. A summary of all approaches used in the present study is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The code for all calculations is available in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FileS1.zip">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FileS2.zip">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FileS3.zip">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FileS4.zip">File S4</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464047/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464047/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of GWP approaches organized by the assumption of marker variances in the present study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Experimental data sets</h3><p id="p-21">Three experimental data sets were used to investigate the prediction accuracy, size of effect estimates, and computing time of GWP approaches. The first data set consisted of 300 tropical maize lines from the International Maize and Wheat Improvement Center (CIMMYT), which were genotyped with 1148 SNP markers (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Crossa <em>et al.</em> 2010</a>). The traits grain yield (GY), female flowering, male flowering, and anthesis-silking interval were analyzed. Each trait was evaluated under severe drought stress and well-watered conditions.</p><p id="p-22">The second data set consisted of 306 elite wheat lines from CIMMYT, which were genotyped with 1717 diversity array technology markers (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Pérez-Rodríguez <em>et al.</em> 2012</a>). The averages of all employed environments for the traits GY and days to heading were analyzed. The maize and the wheat data sets are available as an online supplement to the publications. The third data set consisted of 310 inbred lines from a commercial sugar beet breeding program, which were genotyped with 300 SNP markers (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Hofheinz <em>et al.</em> 2012</a>). The traits sugar content and molasses loss were analyzed. Genotypic and phenotypic data for both traits are available in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FileS4.zip">File S4</a>.</p><p id="p-23">To assess the accuracy of predicting genotypic values, we used repeated random subsampling to divide the data for cross validation. The first subset was used to estimate the marker effects and contained 80% of the data. The second subset contained 20% of the data and was used to validate the effects. The correlations between observed and predicted values were averaged over 100 cross validation runs.</p></div><div id="sec-7" class="subsection"><h3>Simulations</h3><p id="p-24">Computer simulations were used to investigate prediction accuracy of GWP approaches with respect to map position and effect size. To investigate the effect of high and low LD, we simulated random intermating of a large F1 population for either three or 19 generations (ngen = 3, 19). From the last intermating generation, 600 random doubled haploid lines were developed. We simulated 10 chromosomes, each of 1.6 M length, which were evenly covered with markers. To investigate the effect of high, medium, and low marker density, we considered distances between two adjacent markers of 1 cM, 2 cM, or 5 cM (md = 1, 2, 5). Two genes affected the trait on each chromosome; they were 0.401 M and 1.201 M distant from the telomere. Each had a positive effect of 2.5 on the trait. Both favorable alleles originated from the same parental line of the F1 population. To obtain phenotypic values, for each of the 600 doubled haploid lines, a random normally distributed residual was added to the genotypic value. The residual effect was chosen such that the heritability of the trait was <em>h</em><sup>2</sup> = 0.5 or <em>h</em><sup>2</sup> = 0.8. Estimation of marker effects in the simulated data set was replicated 50 times for each GWP approach and the estimated effects sizes for each marker were averaged over the replications.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Computational efficiency</h3><p id="p-25">The computing time required to estimate marker effects with the simulated and experimental data sets was compared with a Linux workstation with 8 GB RAM and an Intel Core Quad 2.80 GHz processor. Among the approaches with homoscedastic marker variances, RIR was the fastest, and among those with heteroscedastic marker variances, RRWA was the fastest (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). With both approaches, marker effect estimation took less than a second for all investigated data sets. RMLV was the slowest approach; in particular, for large data sets, the required computing time was considerable greater than that required for the other approaches.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464051/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464051/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Computing time (sec) required for the estimation of marker effects with different GWP approaches</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Prediction accuracy of GWP approaches</h3><p id="p-27">For the approaches BLUP, RRWA, RMLA, BL, and HEM, the correlation between predicted and observed phenotypic values ranged between 0.31 for flowering time in the maize data set and 0.86 for molasses loss in the sugar beet data set (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The differences in prediction accuracy between the data sets were pronounced; however, a clear trend with respect to differences between the GWP approaches was not observable. Prediction accuracies were nearly identical for the approaches BLUP, RIR, and RRBlupM6; therefore, only the results for BLUP are presented. The RMLV approach showed considerable lower prediction accuracies than the other approaches, ranging from <strong><em>r</em></strong> = 0.19 to 0.82. Similar trends were observed with the simulated data (data not shown).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464034/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464034/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Correlation between observed and predicted phenotypic values determined with cross validation for different traits in the maize, wheat, and sugar beet data sets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Size of effect estimates in the wheat data set</h3><p id="p-29">In the wheat data set for the trait GY, markers for which the effects estimated with the BLUP approach were high had even greater effects with the RMLA approach (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). With RMLV, the differences in size between small and large effects were even greater. Most marker effects were shrunken to zero, and only a subset of markers had remarkably high effect estimates. The approaches RRWA and RMLA estimated marker effects of identical effect sizes. HEM and RRWA estimated marker effects of comparable magnitude. Both shrank many marker effects toward zero and estimated greater effects for the remaining markers. However, the marker effects shrunken near zero were not the same for both approaches.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/539/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the estimated marker effects for grain yield (GY) in the wheat data set for the best linear unbiased prediction (BLUP), ridge regression with weighing factors according to analysis of variance components (RRWA), estimation of the error and genetic variance components with restricted maximum likelihood and partitioning according to analysis of variance components (RMLA), modification of the restricted maximum likelihood procedure that yields heteroscedastic variances (RMLV), and heteroscedastic effects model (HEM) approaches." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1748386231" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the estimated marker effects for grain yield (GY) in the wheat data set for the best linear unbiased prediction (BLUP), ridge regression with weighing factors according to analysis of variance components (RRWA), estimation of the error and genetic variance components with restricted maximum likelihood and partitioning according to analysis of variance components (RMLA), modification of the restricted maximum likelihood procedure that yields heteroscedastic variances (RMLV), and heteroscedastic effects model (HEM) approaches.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/539/F1.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/539/F1.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/539/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/539/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464043" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Comparison of the estimated marker effects for grain yield (GY) in the wheat data set for the best linear unbiased prediction (BLUP), ridge regression with weighing factors according to analysis of variance components (RRWA), estimation of the error and genetic variance components with restricted maximum likelihood and partitioning according to analysis of variance components (RMLA), modification of the restricted maximum likelihood procedure that yields heteroscedastic variances (RMLV), and heteroscedastic effects model (HEM) approaches.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Simulation study on accuracy of marker effect estimates</h3><p id="p-31">For all combinations of marker distance (md = 1, 2, 5) and LD (ngen = 3, 19) the BLUP approach estimated the true marker effects with the least accuracy and the RMLV approach with the greatest accuracy (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The BL, HEM, and RMLA approaches reached greater accuracies than the BLUP approach but still were outperformed considerably by the RMLV approach.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/539/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker effects (blue circles) estimated with different GWP approaches in the simulated data set plotted against marker locations [M] for the first chromosome. The positions of the simulated quantitative trait loci are symbolized by open red diamonds, ngen is the number of random intermating generations, and md is the marker distance [cM] of two adjacent markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1748386231" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker effects (blue circles) estimated with different GWP approaches in the simulated data set plotted against marker locations [M] for the first chromosome. The positions of the simulated quantitative trait loci are symbolized by open red diamonds, ngen is the number of random intermating generations, and md is the marker distance [cM] of two adjacent markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/539/F2.medium.gif" width="369" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/539/F2.medium.gif" width="369" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/539/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/539/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464074" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Marker effects (blue circles) estimated with different GWP approaches in the simulated data set plotted against marker locations [M] for the first chromosome. The positions of the simulated quantitative trait loci are symbolized by open red diamonds, ngen is the number of random intermating generations, and md is the marker distance [cM] of two adjacent markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The accuracy of the BLUP approach was in particular low for the combination of small marker distances (md = 1) and high LD (ngen = 3). Here only RMLV resulted in usable effect estimates. With decreasing marker distances and decreasing LD the accuracy of the effect estimates obtained by the BLUP approach increased. However, the other approaches still provided effect estimates with considerable greater accuracy.</p><p id="p-34">The greatest accuracy of effect estimates was achieved for large marker distances (md = 5) and low LD (ngen = 19), but still the BLUP showed a considerable underestimation of the true effects.</p><p id="p-35">In addition to the simulations with a heritability of <em>h</em><sup>2</sup> = 0.8 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), we performed the same set of simulations with a heritability of <em>h</em><sup>2</sup> = 0.5. The accuracy of effect estimates was lower but showed the same trends as with <em>h</em><sup>2</sup> = 0.8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1/FigureS5.zip">File S5</a>).</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Heteroscedastic marker variances</h3><p id="p-36">For highly polygenic traits that follow closely the infinitesimal model of quantitative genetics, like yield, GWP approaches assuming homoscedastic marker variances are expected to be efficient for predicting genotypic values. However, GWP approaches with heteroscedastic marker variances model better the genetic basis of traits when the number of markers is substantially greater than number of genes underlying the trait. This is the case for SNP maps with high marker densities or for traits that are controlled by only a few genes. Bayesian models were the first heteroscedastic GWP approaches. Their two main drawbacks are that choosing a suitable prior is required and that they are computationally very demanding. Dense marker maps have become state of the art and aggravate the problem of high computing times required for Bayesian approaches. Hence, fast and efficient heteroscedastic GWP approaches are necessary (<em>cf</em>. <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> 2013</a>).</p><p id="p-37">Our RMLA approach, as well as the HEM approach of <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> (2013)</a>, provides computational efficient alternatives to Bayesian approaches. The core of both approaches is to determine an individual shrinkage factor for each marker and then apply these shrinkage factors in ridge regression. The shrinkage factors for HEM are determined on basis of a BLUP estimate of the marker effects <em>u<sub>i</sub></em>, whereas RMLA uses a single-marker ANOVA. From a computational point of view, obtaining the BLUP estimates requires iterative procedures, whereas RMLA requires only the calculation of sums of squares. Consequently, determining shrinkage factors for RMLA is simpler and faster than for HEM. A second property that distinguishes RMLA from HEM is that the shrinkage factors for HEM are based on a first approximation, which uses homoscedastic marker variances; in contrast, the shrinkage factors for RMLA are based on a first approximation using heteroscedastic marker variances.</p><p id="p-38">The computational efficiency of HEM was similar to that of RMLA for the data set of <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Crossa <em>et al.</em> (2010)</a>, but HEM was faster than RMLA for the data set of <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Pérez-Rodríguez <em>et al.</em> (2012)</a> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This advantage can be attributed to the optimized fitting algorithm of HEM, which makes its running time proportional to the number of individuals and not to the number of markers, as is the case for RMLA. Adopting a similar approach for RMLA might provide increased performance for dense marker maps. We chose to implement a different strategy for obtaining better performance. Approximating RMLA with RRWA uses preliminary estimates of the heritability instead of estimating the genetic and the residual variance from the data set under investigation. This results in a heteroscedastic ridge regression approach that does not need iterative procedures at all. RMLA and its approximation RRWA yielded the same effect estimates (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and estimating the marker effects with RRWA took less than 1 sec for medium-sized data sets (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). RRWA outperformed the other investigated approaches by factors between 10 and 100.</p></div><div id="sec-15" class="subsection"><h3>Prediction of genotypic values and size of estimated effects</h3><p id="p-39">The accuracy of predicting genotypic values was comparable for homo- and heteroscedastic genetic GWP approaches (<em>cf</em>. <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Heffner <em>et al.</em> 2011</a>). Our results confirm that in general no advantage of a particular approach can be observed with respect to predicting genotypic values (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The size of effect estimates, however, was clearly different in the wheat data set of <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Pérez-Rodríguez <em>et al.</em> (2012)</a> for the five investigated GWP approaches (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The estimated effects for grain yield were greater for RMLA and HEM than for BLUP. RMLV resulted in the greatest effects and the most effects shrunken near zero. Hence, the similarity of GWP approaches with respect to predicting genotypic values is not caused by similar estimated marker effects.</p><p id="p-40">We conclude that a high accuracy of estimated marker effects is not a prerequisite for high prediction accuracies of genotypic values, as long as marker alleles that were in positive LD in the estimation set are still to a large extent in positive LD in the individuals for which the genetic values were predicted. However, because there are considerable differences in the estimated marker effects between the GWP approaches, the choice of the GWP approach is expected to have an impact on the success of such applications of GWP that rely on the accuracy of estimates of single marker effects.</p></div><div id="sec-16" class="subsection"><h3>Importance of accurate effect estimates</h3><p id="p-41">Identification of known candidate genes in <em>Arabidopsis</em> (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> 2013</a>) and apple (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Kumar <em>et al.</em> 2012</a>) was possible with effect estimates obtained by heteroscedastic GWP approaches. In contrast, no successful identification of genes was reported with results from homoscedastic BLUP estimates. This can be regarded as an indication that the greater effects obtained by the heteroscedastic approaches (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) are modeling the genetic basis of traits controlled by few genes better than homoscedastic BLUP and that accurate marker effect estimates are a prerequisite for the identification and fine mapping of functional genes.</p><p id="p-42">An application of GWP that is most anticipated by plant breeders is planning crosses. In planning crosses, the probability distribution of the genotypic values of a population is investigated, which was derived from the cross of two parents with known phenotype and marker genotype. This distribution depends on the recombination between loci in the two parents of the cross, which breaks up the LD present in the parents. Here, it is not sufficient that the sum of genotypic values on a chromosome stretch in LD is correctly estimated. Instead, the effect of each single marker needs to be estimated with high accuracy. These two applications demonstrate that there is a need for GWP that provide accurate effect estimates for single markers.</p><p id="p-43">With experimental data sets, the differences between GWP approaches with respect to effects sizes can be investigated (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>), but it is not possible to evaluate which of the different effects at a marker is in fact the better estimate of the true (but unknown) effect. The importance of the two aforementioned applications and the fact that with experimental data the true effects are unknown motivated our simulation study.</p></div><div id="sec-17" class="subsection"><h3>Accuracy of effect estimates depending on the GWP approach</h3><p id="p-44">In breeding populations of crop species, the level of LD is typically high. <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Li <em>et al.</em> (2011)</a> observed 20.6 cM for sugar type inbreds of sugar beet, and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Stich <em>et al.</em> (2005)</a> found an average LD length of 33 cM in European elite maize germplasm. The simulations with high marker density (md = 1) and high LD (ngen = 3) represent such a genetic situation (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Here BLUP estimates of the genetic effects of traits controlled by two genes are underestimated. The underestimation is so severe that a useful application of the BLUP effect estimates seems unrealistic. Although there is still a considerable underestimation of RMLV in this scenario, this approach was the only that provided an effect estimate useful for applications like prediction of crosses and identification of functional genes.</p><p id="p-45">In conclusion, our results confirm the results of previous studies that the BLUP can provide accurate predictions of genotypic values. However, for dense markers and strong LD, the effect estimates of BLUP are very imprecise. For applications of GWP that rely on accurate effect estimations, heteroscedastic approaches are superior. In particular, the RMLV approach is a promising approach for providing accurate GWP effect estimates.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.010025/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-46"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-10-15"><span class="received-label">Received </span>October 15, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-01-19"><span class="accepted-label">Accepted </span>January 19, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Hofheinz and Frisch</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-based prediction of testcross values in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAlbrecht%26rft.auinit1%253DT.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btestcross%2Bvalues%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.2" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.3" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.4" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.5" data-doi="10.1007/s00122-012-1940-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofheinz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borchardt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissleder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-based prediction of test cross performance in two subsequent breeding cycles.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHofheinz%26rft.auinit1%253DN.%26rft.volume%253D125%26rft.issue%253D8%26rft.spage%253D1639%26rft.epage%253D1645%26rft.atitle%253DGenome-based%2Bprediction%2Bof%2Btest%2Bcross%2Bperformance%2Bin%2Btwo%2Bsubsequent%2Bbreeding%2Bcycles.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1940-5%26rft_id%253Dinfo%253Apmid%252F22814724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1940-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22814724&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.6" data-doi="10.1534/genetics.112.139014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kärkkäinen</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Back to basics for Bayesian model building in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139014%26rft_id%253Dinfo%253Apmid%252F22554888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.7" data-doi="10.1371/journal.pone.0036674"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volz</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Withworth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for fruit trait quality in apple (<em>Malus</em> x <em>domestica</em> Borkh.).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKumar%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36674%26rft.epage%253De36674%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bfruit%2Bquality%2Btraits%2Bin%2Bapple%2B%2528Malus%25C3%2597domestica%2BBorkh.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036674%26rft_id%253Dinfo%253Apmid%252F22574211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574211&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.8" data-doi="10.1186/1471-2164-12-484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lühmann</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weißleder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide distribution of genetic diversity and linkage disequilibrium in elite sugar beet germplasm.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D12%26rft.spage%253D484%26rft.epage%253D484%26rft.atitle%253DGenome-wide%2Bdistribution%2Bof%2Bgenetic%2Bdiversity%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Belite%2Bsugar%2Bbeet%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-484%26rft_id%253Dinfo%253Apmid%252F21970685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-484&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21970685&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.9" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.11" data-doi="10.3168/jds.S0022-0302(86)80655-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Nonlinear model for describing convergence of iterative methods of variance component estimation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">2209</span>–<span class="cit-lpage">2213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DMisztal%26rft.auinit1%253DI.%26rft.volume%253D69%26rft.issue%253D8%26rft.spage%253D2209%26rft.epage%253D2213%26rft.atitle%253DNonlinear%2BModel%2Bfor%2BDescribing%2BConvergence%2Bof%2BIterative%2BMethods%2Bof%2BVariance%2BComponent%2BEstimation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252886%252980655-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(86)80655-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.12" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian Linear Regression Package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manés</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.14" data-doi="10.2135/cropsci2008.10.0595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ridge regression and extensions for genomewide selection in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1176%26rft.atitle%253DRidge%2BRegression%2Band%2BExtensions%2Bfor%2BGenomewide%2BSelection%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.10.0595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.10.0595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estaghvirou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D52%26rft.spage%253D1093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.539.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Variance Components</span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.17" data-doi="10.1534/genetics.112.146720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fikse</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel generalized ridge regression method for quantitative genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146720%26rft_id%253Dinfo%253Apmid%252F23335338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEyNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.18" data-doi="10.1186/1471-2105-11-529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepherd</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wooliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection and complex trait prediction using a fast EM algorithm applied to genome-wide markers.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DShepherd%26rft.auinit1%253DR.%2BK.%26rft.volume%253D11%26rft.spage%253D529%26rft.epage%253D529%26rft.atitle%253DGenomic%2Bselection%2Band%2Bcomplex%2Btrait%2Bprediction%2Busing%2Ba%2Bfast%2BEM%2Balgorithm%2Bapplied%2Bto%2Bgenome-wide%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-529%26rft_id%253Dinfo%253Apmid%252F20969788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20969788&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.19" data-doi="10.1007/s00122-005-2057-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckenberger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Linkage disequilibrium in European elite maize germplasm investigated with SSRs.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DStich%26rft.auinit1%253DB.%26rft.volume%253D111%26rft.issue%253D4%26rft.spage%253D723%26rft.epage%253D730%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2BEuropean%2Belite%2Bmaize%2Bgermplasm%2Binvestigated%2Bwith%2BSSRs.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-2057-x%26rft_id%253Dinfo%253Apmid%252F15997389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-2057-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15997389&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F539.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231313100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.539.20" data-doi="10.1534/genetics.113.150078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide prediction of traits with different genetic architecture through efficient variable selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150078%26rft_id%253Dinfo%253Apmid%252F23934883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzUzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/539.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/523" title="The Yeast Ess1 Prolyl Isomerase Controls Swi6 and Whi5 Nuclear Localization" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/547" title="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/539.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-buh5nDUMRUncm0HpqAuvHNeEyjmEstr5a2rkd9pbAIo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/539" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/539" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/539" class="active" data-icon-position="" data-hide-link-title="0">Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459648" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-y2ETGOgsiKUDDbncg2LLRODsBsPRQ4wI-DpZKjYagp8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/539.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/539" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3tfV_N1CWjK7riGpFL0HDKAozS9Zz2OhwoWpxoq7E7k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/539" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/539.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459648" id="node459648--2" data-pisa="ggg;4/3/539" data-pisa-master="ggg;g3.113.010025" data-apath="/ggg/4/3/539.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nina Hofheinz</span> and <span class="highwire-citation-author" data-delta="1">Matthias Frisch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">539-546;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010025 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459648--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nina Hofheinz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BHofheinz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofheinz%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BHofheinz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthias Frisch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BFrisch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frisch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthias%2BFrisch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthias.frisch@uni-giessen.de" class="" data-icon-position="" data-hide-link-title="0">matthias.frisch@uni-giessen.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/539" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/539&amp;text=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/539&amp;t=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/539&amp;title=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459648/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459648" id="node459648--3" data-pisa="ggg;4/3/539" data-pisa-master="ggg;g3.113.010025" data-apath="/ggg/4/3/539.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nina Hofheinz</span> and <span class="highwire-citation-author" data-delta="1">Matthias Frisch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">539-546;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.010025 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459648--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nina Hofheinz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nina%2BHofheinz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofheinz%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANina%2BHofheinz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthias Frisch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Agronomy and Plant Breeding II, Justus Liebig University, 35392 Giessen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BFrisch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Frisch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthias%2BFrisch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthias.frisch@uni-giessen.de" class="" data-icon-position="" data-hide-link-title="0">matthias.frisch@uni-giessen.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=539&amp;rft.epage=546&amp;rft.atitle=Heteroscedastic%20Ridge%20Regression%20Approaches%20for%20Genome-Wide%20Prediction%20With%20a%20Focus%20on%20Computational%20Efficiency%20and%20Accurate%20Effect%20Estimation&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hofheinz%2C+Nina&amp;rft.au=Frisch%2C+Matthias"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459648/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459648/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459648/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459648/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459648/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459648/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459648/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459648/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459648/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459648/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459648/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459648/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/539" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/539.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/539.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/539.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/539.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
