<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/547" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/547.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/547.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/547.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/3/547/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/547.atom" />
<meta name="HW.pisa" content="ggg;4/3/547" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" />
<meta name="DC.Identifier" content="10.1534/g3.113.009431" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Wagih and Parts. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Colony-based screens that quantify the fitness of clonal populations on solid agar plates are perhaps the most important source of genome-scale functional information in microorganisms. The images of ordered arrays of mutants produced by such experiments can be difficult to process because of laboratory-specific plate features, morphed colonies, plate edges, noise, and other artifacts. Most of the tools developed to address this problem are optimized to handle a single setup and do not work out of the box in other settings. We present gitter, an image analysis tool for robust and accurate processing of images from colony-based screens. gitter works by first finding the grid of colonies from a preprocessed image and then locating the bounds of each colony separately. We show that gitter produces comparable colony sizes to other tools in simple cases but outperforms them by being able to handle a wider variety of screens and more accurately quantify colony sizes from difficult images. gitter is freely available as an R package from &lt;http://cran.r-project.org/web/packages/gitter&gt; under the LGPL. Tutorials and demos can be found at &lt;http://omarwagih.github.io/gitter&gt;" />
<meta name="DC.Contributor" content="Omar Wagih" />
<meta name="DC.Contributor" content="Leopold Parts" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Colony-based screens that quantify the fitness of clonal populations on solid agar plates are perhaps the most important source of genome-scale functional information in microorganisms. The images of ordered arrays of mutants produced by such experiments can be difficult to process because of laboratory-specific plate features, morphed colonies, plate edges, noise, and other artifacts. Most of the tools developed to address this problem are optimized to handle a single setup and do not work out of the box in other settings. We present gitter, an image analysis tool for robust and accurate processing of images from colony-based screens. gitter works by first finding the grid of colonies from a preprocessed image and then locating the bounds of each colony separately. We show that gitter produces comparable colony sizes to other tools in simple cases but outperforms them by being able to handle a wider variety of screens and more accurately quantify colony sizes from difficult images. gitter is freely available as an R package from http://cran.r-project.org/web/packages/gitter under the LGPL. Tutorials and demos can be found at http://omarwagih.github.io/gitter&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/547" />
<meta name="citation_id" content="4/3/547" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/547" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/547.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/547.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/547.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.009431" />
<meta name="citation_pmid" content="24474170" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="547" />
<meta name="citation_lastpage" content="552" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Omar Wagih" />
<meta name="citation_author_institution" content="European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SD, UK" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada" />
<meta name="citation_author_email" content="wagih@ebi.ac.uk" />
<meta name="citation_author_email" content="leopold.parts@utoronto.ca" />
<meta name="citation_author" content="Leopold Parts" />
<meta name="citation_author_institution" content="Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada" />
<meta name="citation_author_email" content="wagih@ebi.ac.uk" />
<meta name="citation_author_email" content="leopold.parts@utoronto.ca" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=J. Angulo;citation_author=J. Serra;citation_title=Automatic analysis of DNA microarray images using mathematical morphology;citation_pages=553-562;citation_volume=19;citation_year=2003;citation_issue=5;citation_pmid=12651712;citation_doi=10.1093/bioinformatics/btg057" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=T. Baba;citation_author=T. Ara;citation_author=M. Hasegawa;citation_author=Y. Takai;citation_author=Y. Okumura;citation_title=Construction of Escherichia coli k-12 in-frame, single-gene knockout mutants: the Keio collection.;citation_pages=2006.0008;citation_volume=2;citation_year=2006;citation_pmid=16738554;citation_doi=10.1038/msb4100050" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. Babu;citation_author=A. Gagarinova;citation_author=A. Emili;citation_title=Array-based synthetic genetic screens to map bacterial pathways and functional networks in Escherichia coli.;citation_pages=99-126;citation_volume=781;citation_year=2011;citation_pmid=21877280;citation_doi=10.1007/978-1-61779-276-2_7" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Bariamis;citation_author=D. K. Iakovidis;citation_author=D. Maroulis;citation_title=M3G: maximum margin microarray gridding.;citation_pages=49-49;citation_volume=11;citation_year=2010;citation_pmid=20100338;citation_doi=10.1186/1471-2105-11-49" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=G. J. Bean;citation_author=T. Ideker;citation_title=Differential analysis of high-throughput quantitative genetic interaction data.;citation_pages=R123;citation_volume=13;citation_year=2012;citation_pmid=23268787;citation_doi=10.1186/gb-2012-13-12-r123" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=G. Butland;citation_author=M. Babu;citation_author=J. J. Daz-Meja;citation_author=F. Bohdana;citation_author=S. Phanse;citation_title=eSGA: E. coli synthetic genetic array analysis.;citation_pages=789-795;citation_volume=5;citation_year=2008;citation_issue=9;citation_pmid=18677321;citation_doi=10.1038/nmeth.1239" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. E. Carpenter;citation_author=T. R. Jones;citation_author=M. R. Lamprecht;citation_author=C. Clarke;citation_author=I. H. Kang;citation_title=CellProfiler: image analysis software for identifying and quantifying cell phenotypes.;citation_pages=R100-R100;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=17076895;citation_doi=10.1186/gb-2006-7-10-r100" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Collins;citation_author=M. Schuldiner;citation_author=N. J. Krogan;citation_author=J. S. Weissman;citation_title=A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.;citation_pages=R63-R63;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16859555;citation_doi=10.1186/gb-2006-7-7-r63" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=J. Bellay;citation_author=Y. Kim;citation_author=E. D. Spear;citation_title=The Genetic Landscape of a Cell;citation_pages=425-431;citation_volume=327;citation_year=2010;citation_issue=5964;citation_pmid=20093466;citation_doi=10.1126/science.1180823" />
<meta name="citation_reference" content="citation_title=The Radon Transform and Some of Its Applications;citation_author=S. R. Deans;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. C. Dittmar;citation_author=R. J. Reid;citation_author=R. Rothstein;citation_title=ScreenMill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.;citation_pages=353-353;citation_volume=11;citation_year=2010;citation_pmid=20584323;citation_doi=10.1186/1471-2105-11-353" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W.-K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Hens;citation_author=J.-D. Feuz;citation_author=A. Isakova;citation_author=A. Iagovitina;citation_author=A. Massouras;citation_title=Automated protein-DNA interaction screening of Drosophila regulatory elements.;citation_pages=1065-1070;citation_volume=8;citation_year=2011;citation_issue=12;citation_pmid=22037703;citation_doi=10.1038/nmeth.1763" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W.-K. Huh;citation_author=J. V. Falvo;citation_author=L. C. Gerke;citation_author=A. S. Carroll;citation_author=R. W. Howson;citation_title=Global analysis of protein localization in budding yeast.;citation_pages=686-691;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562095;citation_doi=10.1038/nature02026" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D.-U. Kim;citation_author=J. Hayles;citation_author=D. Kim;citation_author=V. Wood;citation_author=H.-O. Park;citation_title=Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe.;citation_pages=617-623;citation_volume=28;citation_year=2010;citation_issue=6;citation_pmid=20473289;citation_doi=10.1038/nbt.1628" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Lawless;citation_author=D. J. Wilkinson;citation_author=A. Young;citation_author=S. G. Addinall;citation_author=D. A. Lydall;citation_title=Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.;citation_pages=287-287;citation_volume=11;citation_year=2010;citation_pmid=20509870;citation_doi=10.1186/1471-2105-11-287" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Pau;citation_author=F. Fuchs;citation_author=O. Sklyar;citation_author=M. Boutros;citation_author=W. Huber;citation_title=Ebimage—an r package for image processing with applications to cellular phenotypes.;citation_volume=26;citation_year=2010;citation_pmid=20338898;citation_doi=10.1093/bioinformatics/btq046" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. W. Powers;citation_author=M. Kaeberlein;citation_author=S. D. Caldwell;citation_author=B. K. Kennedy;citation_author=S. Fields;citation_title=Extension of chronological life span in yeast by decreased TOR pathway signaling;citation_pages=174-184;citation_volume=20;citation_year=2006;citation_issue=2;citation_pmid=16418483;citation_doi=10.1101/gad.1381406" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Roguev;citation_author=S. Bandyopadhyay;citation_author=M. Zofall;citation_author=K. Zhang;citation_author=T. Fischer;citation_title=Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast.;citation_volume=322;citation_year=2008;citation_pmid=18818364;citation_doi=10.1126/science.1162609" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=L. Rueda;citation_author=I. Rezaeian;citation_title=A fully automatic gridding method for cDNA microarray images.;citation_pages=113-113;citation_volume=12;citation_year=2011;citation_pmid=21510903;citation_doi=10.1186/1471-2105-12-113" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=N. A. Shah;citation_author=R. J. Laws;citation_author=B. Wardman;citation_author=L. P. Zhao;citation_author=J. L. Hartman;citation_title=Accurate, precise modeling of cell proliferation kinetics from time-lapse imaging and automated image analysis of agar yeast culture arrays.;citation_pages=3-3;citation_volume=1;citation_year=2007;citation_pmid=17408510;citation_doi=10.1186/1752-0509-1-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Y. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. H. Tong;citation_author=G. Lesage;citation_author=G. D. Bader;citation_author=H. Ding;citation_title=Global mapping of the yeast genetic interaction network.;citation_pages=808-13;citation_volume=303;citation_year=2002;citation_issue=5659" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=F. J. Vizeacoumar;citation_author=N. van Dyk;citation_author=F. S. Vizeacoumar;citation_author=V. Cheung;citation_author=J. Li;citation_title=Integrating high-throughput genetic interaction mapping and high-content screening to explore yeast spindle morphogenesis.;citation_volume=188;citation_year=2010;citation_pmid=20065090;citation_doi=10.1083/jcb.200909013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=O. Wagih;citation_author=M. Usaj;citation_author=A. Baryshnikova;citation_author=B. VanderSluis;citation_author=E. Kuzmin;citation_title=Sgatools: one-stop analysis and visualization of array-based genetic interaction screens.;citation_volume=41;citation_year=2013;citation_pmid=23677617;citation_doi=10.1093/nar/gkt400" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Colony-based screens that quantify the fitness of clonal populations on solid agar plates are perhaps the most important source of genome-scale functional information in microorganisms. The images of ordered arrays of mutants produced by such experiments can be difficult to process because of laboratory-specific plate features, morphed colonies, plate edges, noise, and other artifacts. Most of the tools developed to address this problem are optimized to handle a single setup and do not work out of the box in other settings. We present gitter, an image analysis tool for robust and accurate processing of images from colony-based screens. gitter works by first finding the grid of colonies from a preprocessed image and then locating the bounds of each colony separately. We show that gitter produces comparable colony sizes to other tools in simple cases but outperforms them by being able to handle a wider variety of screens and more accurately quantify colony sizes from difficult images. gitter is freely available as an R package from &lt;http://cran.r-project.org/web/packages/gitter&gt; under the LGPL. Tutorials and demos can be found at &lt;http://omarwagih.github.io/gitter&gt;" />
<meta name="og-title" property="og:title" content="gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/547" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Colony-based screens that quantify the fitness of clonal populations on solid agar plates are perhaps the most important source of genome-scale functional information in microorganisms. The images of ordered arrays of mutants produced by such experiments can be difficult to process because of laboratory-specific plate features, morphed colonies, plate edges, noise, and other artifacts. Most of the tools developed to address this problem are optimized to handle a single setup and do not work out of the box in other settings. We present gitter, an image analysis tool for robust and accurate processing of images from colony-based screens. gitter works by first finding the grid of colonies from a preprocessed image and then locating the bounds of each colony separately. We show that gitter produces comparable colony sizes to other tools in simple cases but outperforms them by being able to handle a wider variety of screens and more accurately quantify colony sizes from difficult images. gitter is freely available as an R package from &lt;http://cran.r-project.org/web/packages/gitter&gt; under the LGPL. Tutorials and demos can be found at &lt;http://omarwagih.github.io/gitter&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IHbrE2r1afU842bH4jB1r1Futr6lu62NUp5vt1CfgIQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459649","apath":"\/ggg\/4\/3\/547.atom","pisa":"ggg;4\/3\/547","ac":{"\/ggg\/4\/3\/547.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/547.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/547"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/547"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/547":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PTNr6R5L6aQgiGZV\\\\\\\/xGPwDTGTB8n7ngSAfD3YBH\\\\\\\/R5KnqcbSo8p2fNSPIHKBtaE\\\\\\\/F3P5hR5Suh82AwHJy2FV\\\\\\\/qxyvIJUr4Rxy9xBr6kL5ZdJ7bTeGUfjbdb3S36cfQNuoQ\\\\\\\/uBKeqrC6zKyLdN5OvXjgkU6y6Bt4bduRVtRvK9rw03FB5ZGChvMN2GtZIeD53BZsUR2zsJCUI3cftnGK0XQx1MoVnAbTvpFtvgtJfPPrbyLGbLGiefn53dw1\\\\\\\/FkBVCTC\\\\\\\/Ywnar7NsgLs3JHIYFzui0z5x4SA7d+l\\\\\\\/8S1p5Q3ZKS2dOU1cCMMmiXehjkamvDqfDkDNnJSRavg7\\\\\\\/VkI\\\\\\\/kNc3ytjoTtCQD0JapJQ48QB2RlAbbVFpea\\\\\\\/FpzJW10IM2SvkzIxS3ccva3nJ6aQuaYzq5AOltrBtjtg8JxyxCPpiSLtw10ockcsUc52OofR0qqvQOcfoAh7eGi4uXDcX\\\\\\\/YnV32nDgBkWTgC0XvAQhL\\\\\\\/18sPL9UP+vgMJr3IL1q515USO7\\\\\\\/8f0eraCS\\\\\\\/xOkNYu8EAOh4QKNAPmvddhk7ylsHW22Ix\\\\\\\/Y+UMoYIPk55HyzDoo4RRkvpng3dVimdZpCZothXnubH+CyZ1AmvtLZZAS1ObcNqB9Z5IZFd03NptgLG13Xw+pBVUI7cWXorTrjCwdpn3aHEnsPvwMOrYdk7cVTTHYLmcGwx0Any4z\\\\\\\/5RKtUuUMf9p2Z\\\\\\\/E4cokCLJOT+9UePGlhct058gmm2fySCcFPyVPOf\\\\\\\/BmLuXkEz0Py0yo\\\\\\\/T0HdpbmUySUBxcJtNOXVh8AIeRJaj6DpU5KWWDq30\\\\\\\/pMYUR8DIRaNxVz6azev2u2R7gCqqv3ubANd6eFrOL0MJdCzJDPEPN3bSsCC\\\\\\\/ghKoXqn0e05EGoGkigxZprvouenoYXAKR\\\\\\\/Lq3sf5z6K6zg0Aa529hxdjpcBRMcr5A4ca1o0\\\\\\\/SDaJdmjdCw0MBXHKFXKhuKSBz98PneG7r0kjLZG9ev0b8IrMCfsTTzrTE0NRmVSqmY3AOvQfvPEUt69KbNoGdL8y0+6fRi8wptNSj4M17zFiJqy\\\\\\\/x\\\\\\\/Fu5kQovjm+d7tRiuGencqNfdCBAq\\\\\\\/ikRRqEf8VtmYEwkJQBQuV5pT0Bm\\\\\\\/kOD06XXLm8\\\\\\\/v4fEX2vUcAxC3cyVI5UjnMLuqQ6o4uMClx34AKjTPpdmxh0WAmszoBWCDtZh59ntKLl+tMxXxgiWas8UJ8CPVzE0870aAoQC4vdp1VIv5jRUifHRGn4pNyLQ5nDlNhsVXwP1D04B4pFe5Z1PhhkSFObXmBJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022TlS2z8RBZvdsV8GADX2\\\\\\\/cQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223c02e9d812b4f7fd4e00746b98f2f8c9e20a9d51004c93a483ab6e32cb09a9a8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JgEIOsIEV1gRtarySd7kNNPxf1lf7NoWWTq+RlxOllMdumOh3s9sM8wVmKVtr543rcMbF2AW+DsQBPpRs8mQk7FDgMQMeQu+ZCSQyLLbIyxlakz5n+Qr7sFALgoNf7DnOtfajWOjBtqTU5mT+jh5uEjWEc34RA4lvi\\\\\\\/74ymrBSZ0rJ1EVsggBRdlUYJ6Bgtpa0GspWmDfgPTQ5E4tqT4kP+HRMx9uNaxK3w0oUqbUw+Ik0+KxedI48OmS\\\\\\\/lC4yiMyMi5EVsO6sm0b0Jz+3I7nBSGi5dqXZKiW0fNdfYu9MSBWDtXrGOItGpJ\\\\\\\/UpvBsSlMbTnBbh60aSVfVDm5u6mTpDqWxHSI02zQ+0Gw\\\\\\\/dXva1QiO9rCT3zBCtrVX+WCzWIsqQFd\\\\\\\/r13pK\\\\\\\/Ie2l202xuSvjOgvsmlvrRyReC29k36gzY1BkAZlgLn46fRX2FgH+ehWb8cscbF\\\\\\\/L49+EouilfBR6FoLKCfh1DwRUGyTk\\\\\\\/7IQlqmsccItXcE4ZxeVbPzCPcilDzLTXt+Q7Cx3u+MWfLiEjp45\\\\\\\/YRpj2PvX7bj0aLgqs0v2Cp4Uf85CHjYYnPJLGwlKonKlKzK0WjnlIGgvA+YcWYgv9wQtYo\\\\\\\/vJ172WAe3faqXlvTfNShV29ksDrae9eaNX7EaTOFlwf9A0+PXxqwPTCZA2f13TMnm4BoBRkOEOtfHUmO8yUt\\\\\\\/NvCevyxpQpGaNIN9PeMRHiqPErCw3Hb1XUYj13h1tc7hIzFfwE2Qno\\\\\\\/CLJHKicIT8\\\\\\\/v3lkA16UCfm8kGkMMCDqq0lAD3PKCCm\\\\\\\/1bj7sFay8+KGTd8i0Ayk0ew8SgC34at+tApPK2F1WuZcNfTlJXhQo888R71CtwbMPHCtgIWO9ucAC3X8FLks4w3eWjmkj7ZdTKcw7OXamgElFlBGS8uR20joo0bFDf8CbVG5dFQC5EUsP4pdZvy2X8+b\\\\\\\/\\\\\\\/DDycQfyH11eWgPIY\\\\\\\/aHHByHxofD\\\\\\\/rc+04Y0DGKnKWircRQa\\\\\\\/j82kfUSUUjAa431EnUfgrkCoxcL9g\\\\\\\/Oe3F8NeZEDj\\\\\\\/NsYMX8OPY3BrB0P2WKtckdnoWPQ+36Sj4bgBb+tyx6PVmHLmzYztKm8SyQ97PExSOpJQ0ocPh48kHWo\\\\\\\/Y1T+6v35Uk3k6xXRmHjV063nQGIyhg38bDa6xwGYNNnK0pJKL5BQ0R\\\\\\\/0QevoGdBDZerJXUOG5vrIpeAQ+I9gCG+VbntW\\\\\\\/WLNGD1TyupjiOYoS+lKrDOaM8lBN+1zk0kdk0tuBlqLEhBIw3H3Ne2MbU8UNpp3xpdsCiYBhLno3DkG5qaEblrI7ELy9Q8\\\\\\\/o9V0MUjpsd6UB2jlgVtzaPVlDcN35TSZESOF5v+ybi83FMpLI70p6xRXl7hB\\\\\\\/bdlqBrFqskbszs5rfkv9Zf5zYNLVzeb5++0s3IXHbK856p70tdazAHbUwWphVCiEmkFb5Wm5YOqY7Qx7O5lQImvurU7jNyxmSmAKgTp11KThjuidTI1EWZnX4r+rrhEVWf2vEAMyRS6SrWSeSaZrylDVumlrcYT8PmxubYHLsIW1bqUtiLjgzDfz9SjDTD\\\\\\\/sZ5uiXxIys+2HVFs57lsIISAI1IbkkHJuSPfmCdVDGOXeE1r3wxh+R5tmBf78HBmJi9H+b3GxtAlkHVzLZfcKMxym1F5E09iKXubt3TNRUMSKRzI0SC5MnusT2oivu3QSBfEyYZ59VZRz79leMMDuSqie7unF\\\\\\\/92GAjTsmPO91CjayORxwwThAea2i\\\\\\\/Dk3dGMJ4Ev5DEKheUB5ElCtxLW+UF84w2PqbPhIA9hpoIruB4kAgpFvwNN6ypegJyAzl+1iPwgA3Lec0bQsTpLC\\\\\\\/bsHrVEb5uvhV6hWp0nYITCmA\\\\\\\/\\\\\\\/9QyvJExhrj2Dr9hsySMzTYOQC08cSD7Hi5fPr9C4dYxeyOrHbrU4PgtSf+gd9b3FQhgRKzUwyDk\\\\\\\/AlMy74PbJEJSIPrTbdIwyHR3VHwYpq1rOq9kkG7vzgjMbiXjlrZT7xMVjpKzpmtNiZfuTzsJyrGgafg2UiMhWL44H3ju2vbGrSJgrdnPuooyxP\\\\\\\/iW5scs2k0BD3+IWwQqmzrSvHP8PtqQV5HDG5ipkU7jcpXN4lWiqCZiBywut4VrrdUGu4Vtp9t7q7+Qmbxf6bjKJuxLEpflbdOGiy9taMS8r7Zc5dfXW834L+CU1HIm9xbDQE6\\\\\\\/g4U0uEHia6eQ0GcFqSAUAVztwj5grZn0vNZJCilhuAVi54NGw4sfTi0++MejXa10YXB6wD2YbdywKi1nRuspK9c9IY9HQxXzL4gfT6sJkzOpbpcOIvg2nWgPy6H5qYvwieFpUYnX7Rg0oAHYyE7fJUUlut+Zb6VZQ4z74EoWnEayLhJtKSCNefd1o3lU4ti65dKBSMq0+SX9mfeA+u279LiEGUGo1\\\\\\\/V1URo8eluVz1QV3XDcS8FMjC2AvnMfIXHRMHBSU7bPMPGqVWK\\\\\\\/yHadNo6pC+q5Feigv+XaUPTs3dhUg3ET6m8a4O3TZlWqSrkgtHLtxRYlCobvS6uTG7yVF1lm2U2W0N5oPlCmNfwl8gkZA0NMBNBoQaBDfysboJlhV1I3F28mTUdEHx+ZyTZU5Fv4BG0v\\\\\\\/d8tweN0fClyr+00pCLna7EKIQkwgfotzAbusfZ7UdZrvloJ5RvHrQyBKZZX21yG+jlsrk9whVGMuqY7Pni74AW9cMRQyfVgmi27Wnx0Q9T5Jq9aBdJprZ4lqpCtCGU3ZuAotFToMTk\\\\\\\/XfbsfYNMenlZMBfjKaZA\\\\\\\/Rmc0qH\\\\\\\/QHmS9AnW+sWdbJoYwSlG3bv0Y7lONOE6\\\\\\\/+YaCGPGDuaPwyYgI+ZUTN1+WgeSdCIGTpNc8LQxCiYm2LkX5vFStvFaJ5aeYVGnZ4y+cuqpFbjJSwm1BqEpYkTfDZWjz8LZJ3riQQyYsRstTRILh3NZvZuWrbTkatQOr03jC0IhCCRoxWjinlBSsaluivEx7odXkKs9StOM3CwIHCqEUw0HbnbOJHrTWDgq+eJBL+2LXlySXe6stb\\\\\\\/gx5Nx0BN\\\\\\\/6jXQIgnmRH+4VYfv8Yh95d89rs4kc\\\\\\\/fjadYVDxr0kNPOovIo43FwXZY8gs5bn+vbjXBBccGRoRczkG0DK8Fq7rSZQReXu\\\\\\\/XOPOyP48J8Fyz8XrULNXOmcmN5DTflJpbWbDuBNtl4zj1OokKuD6RyilO41Ha2eYUmzLKvzMF4fd6E0+Gs3G8e\\\\\\\/nM1lblvMXP33QCXGANCQf1FWEIwUZNpBD2mWtZG5gbKdAxuYuwY\\\\\\\/ju2MoXhepLmGgR2PMXTe7fyzv1Rs6dASeoxSEWWzxvGMCC2dyN8qhmmTT7zCwP\\\\\\\/vryx8f\\\\\\\/VwWPYuy7xnvG0YK4ZNeOqYuva67zdBuwg1w9+Au2aK7mh\\\\\\\/yKqBusdJmwhTJkTcx6l1tlQJGAL5sjbwiXT3aSOb3hBf6P7Niz55vNrDrBym5GruJe0MO\\\\\\\/QDCa6oq3BtC\\\\\\\/Mn3XuNV0lyTNdlQPHRE1MPO4lNUJDLBmCkVfXPXUwkuEkLLQPrayWt7uSfjueFAgvy9BU1EGinVyet3ldxLaE0y5Zr\\\\\\\/cI3he+362SBfXhuPvZScVJvp9s1t76XeNJAlKLXVhYfrxdlpd7Q3m54gR9KNrih+kYwSlkR2b4+F9tMpmqRXg+pUechX5Mo3ff67YlxDG5TMIHcSMB6VuMk49huQWk8IcSMxBrA6ZMv3w\\\\\\\/f2HjTCsKUbQIflKcylnqaiHViashWEmNkbPSpfb+g7psGOoqcvXfkuZXW9qTT6uSYFRtN3WK36r5nYxmsPgD9UqoRVaC3JwxrCS6MQ2M4QmOOK6vnYkdE5MUua\\\\\\\/OW\\\\\\\/SW9XN4BGF6PIoxDgQBVOqCksdeB1ahwd5tjumCup3HGWIq3jL0ScEkzwzqedRXAecFDzBmrKwRplzVYmM9M7gK0umsfFBhcUam8p\\\\\\\/KMSSOIryoCdBjDYOIJtaRCxw\\\\\\\/h2AAQl18OpdIVEkKU5+17qslgfXgMdrhp3duztk3EuLj2kMegv0NlvheRLDgXDtr4ZCD6Wp8mlzvz+mYhjsJq6r9e\\\\\\\/8Xz06sOPJU5mVIcxqRYfclgJzc6B7Zmigm8lKDjb5ayqcf6qyzu0fFt+xQJ3XcoNuVNQktwOkQZbSPhEFMN1WOaYFWnvbbZ8K0dRaih\\\\\\\/xhLoUhXmAG8Q31+hpbCpRpTDrcsInimrYPGxrMwWxMxrdWXJparBSTr8lQApdKIUsiYVwK1qfvAtsJeKjy3+E3PV1rH\\\\\\\/XroQODLYPfjYch87N7Jobxv5LKxy2G8F1LLvyJW\\\\\\\/nWx0iWJ\\\\\\\/6ghWiSbMC8S7L1BgXtkK7h40wqiU5DUZJ1IP7XmACkScWGOzadGOUmKL3YEWBFD\\\\\\\/KiAfV8Y30xTERNVG6SLz8zWBhiJQo\\\\\\\/VdC6\\\\\\\/QCcypBF+e3tT0yW51o+Z36LNPAXUHLRuls81\\\\\\\/9enFWqek39ZjIHIGyBbWKl\\\\\\\/ROIvCXtXWFybejuR9G3k0XUwrxKXMRyHebqTv1Gt2j95LbHbbzVGOxAa0Bxp0ooAysQYLKhwdjGEc5JaYkbU\\\\\\\/v04cMJ2VXG492WzsL3LUJWY1X49dXuY3LUJF5USbAEpltL00Ly7Xa62TrbQC6vbzjn0BcqV6f9U1qiAH1mIATWHyur6eDcZzVDWEFBkZeo3TX2JpUQZwyyc8WTZw3lyihTMdMnibleWJ0zicdvXDrPM59N7kzDmR4kZl9tmum72n6YZ+i8\\\\\\\/DI8elI4FFKQNCHZa5Nu9fZMD34xKkb1dwM605W6510XI+6x36PqN\\\\\\\/sRr37ln65fXR1w2CSJ4Bd3Eo2rCWH3\\\\\\\/r8HPNO\\\\\\\/5uL2SMe6mhIunwQqqVr0glkLrnHnlTIk2OTe2+LOw4Cpb2SZZpaEbF57koVANDC5GiMMdU+tHVQJIrfpyxwwCaTmD4aDz6Vao9BllwNnAVxWhzr+C1qRe2R39aEvHq2mUjkQjQcnOO0iXiC2p3GpID+4RSkOIMqrbKk692TDwhVUvR4h+iksZAle8pvkFicTb5kzjmPY6KXfGbKoc7GjgQim2Zp+vp6aKhtZNAV8PnBHyWfEoF8NVqONQMWHYkGT7dC85or+1YVnuQc2m7LeQHjuUDtcQ239sM0j1P0SHZnK52fTKeEd89xYPameiqQzssh5TAkeQ\\\\\\\/qsF+ClgFOX+vorB5G9HK0GEYvXvbei0vcU7eL2Qm9Q5nFbkBzHzPTk8ffdqhBuDKJCAxkFznig0HDIB1P0JwJXvtx0RaTb\\\\\\\/rZV6Dmvr1KuLp7Fm77CIgFdu6KtLUZRWaHHOyHjfgjkL7tGdRelqtzF3at8xIV5gGaiCmQ1Mr7AChHNpXfpy+1dQPTySgpDprZtqns\\\\\\\/wC2Embn+m+asGY3WyRFNV3f3u1sq0fkfSXPMwbPjpu0HXbMs9TeJB\\\\\\\/LU6GxvJATG3cpfVXmLLKG6YQXd6MjJhIMH1cjCdH1xDcq8PQzBPYD1Lr4EEKP95vX141L3rPJfzlxYhA6PqxrOAiPPaNAVY5DUWxkI5f2Yfig6\\\\\\\/vtzQnfimcwaNa5F3kejepZYyqFGwCNfBUf6NzVSa7NXZlBnLuvK9CdvYsdPXspHS5Z\\\\\\\/tsv2R\\\\\\\/\\\\\\\/NvK1Pghr6K5bdainz1cVnIaUUkHY1BD68PKmuRpNwLo1XffTeLRnMY+jUXZU8lMChzWQIPacYEYuymAHHsbcW2JkgaPjQHnCM5KmLx3\\\\\\\/jSMVpEZhUMOz3enCqJuor\\\\\\\/pwPyeAfCO3\\\\\\\/1zXHwZnMPzifvbBuiUgJIuUHu+PiHZE0BhlhzsHlT+kcb6zkYxx\\\\\\\/iNx47RTWjwuT+OJ4YaxTF\\\\\\\/bPEOgt7DwVqUaPSvZ2gmOZoPGNKgdkOlUi3QpLs6D5951NOPiptfSg0dEaGW\\\\\\\/0A3dZepkdu080BA\\\\\\\/tKQ3f9x7OrJ4Dt+9xtFp\\\\\\\/d6Q5OfbYMWkAwnHF+4GAnYFgiS9YRLdIgGW59kCtocWeK1Bs5sRGwQmu8zJ9ZkQwEhOV2XXUElUQu+umdZ5c8ElhYg3\\\\\\\/+L2tfKaG7tHd39B+88pN9oXRXvjhOt69wx6PGwoD5KKLcd\\\\\\\/dlYWWZ9trsigzleLtehCCy0Be1idNaro48a8J3WOHuiZ1BJ9ngC5ld42jzIXVo9BSH4LO\\\\\\\/r2qhllA7nZrTD4NmhMnb3n1tB6XzCPp6Q6RWDdzlgGA+BlbrqRKE5StckhnLM9+Yq6Zi5jZIPM+R1OtgcfCE3tMWz0LTY\\\\\\\/wXrfX8f0JVEa7NmyfDIqBDBdyT+1+9Q+7Q6o4EuSLqc+CJjhWdXapmabG2VQkBBP+bVrfS96lxzXAipD9Uj+Z8BxTKE1wBmILOzghH6tep4yIv2COanxd\\\\\\\/9Bu9Et4c1t+48ZYjt4oLA6z7CPK8+2LvbeP+jkqGudbQkTisH0a5DCjS2DdoLurH0aaMUA+MmwP3cXDK+ZFKYKGeHQP+qX1LjMdLrmg6hTHdWFNGsIAzNMYcL8Iahz\\\\\\\/VZJDVP\\\\\\\/V5xmQ0Wje94sEU7Qvkj8Z0\\\\\\\/BSVjiVi0kx4J9X3xzbsIwbhShVN\\\\\\\/bKWmR7iLJrG5Uyk2COTjVi1s8w9hRkzKPHfxPQNr2VoA4JtYHj1Zwfl66MLRl4wg0cUmbUPq9Q8QfeUjkdegMOAIFS5auSimRBxr1RFoooRL2UCdCR5BAs7hLxHyBaogpWIjtjfJemGPWhhbpLLL2+gxtLLtibNHSW5tEG6QonuHr08lV3lerYgoqtRNUxUhalWG33bVT\\\\\\\/Gz5E7jO5479F6ZOr8ZnztUOw1tvAljHe579m3Y5JASzEOfnWsJw5UBzu3D4er5cayWRZUvZGlXVOeO6MMU1eKNhckXzMA8FCbC6yG6zVFkqqcKuvh8yu3GoICNbK8hauNwzOGQszSfKAKGFwGPZSykMU+75CWxDKT0aNBlgTL9KzD\\\\\\\/IscEeOv5fljtC9i3PdYOKb40ylMfbmfabrlLPqvEvXS51BjIum9VXsAzymW3XlariPZ1vkMjRvr21rKepD\\\\\\\/j11yA7law67aRuZtakiNV9uzTE8I83uMqmogA3mBmJ+S4ygkEtAhDyb5pe8esoOto37BkwOqNpdBj2BfE9K2IJe5Hh+5I9r\\\\\\\/OhiUqRNZMWSGOH+UzySjvcWE5r3jNQmT0XjQmnrCOts2CiZRFofVjdJhyspCnk0yNvHzmHH+0c5iZacoCep\\\\\\\/bcG2f7lhRaNgR2eEXN9ydoNb8BYFA9ZGoIGtzWCR9+jOMaqpQ0WsuaB3lcqBr2WUCxdYu9f\\\\\\\/X8wO0b3HOyh48Xa2vm46WvvM7XPep6BFHTO8DYkQcGaUU7+SofynAMnxYqFNB5DO488srBWJM3a90WYD4wXlHpzHEMMUMmQLsq9mi83VppcG2NqslkpD0EoAwPC8fa6ED6cRNs00HZr5C7OxmsJ54ihQrJJsOfaNmOMsind1rlDdtyCnNsZ+0Z1PqjTKjLToHH2syLZ8oFD6pX1B9uW1eAyTQJB7OcW5Uo4muuREu1tFnqZSDsUyzzW5ptmrzO25CupaFSwIWiJF22OV9BoNeY1T99STsMXAwOh+xHbT8sOU3kP\\\\\\\/vnQlX6U11UbodS5e0XnbybeknVo84Y9H51wrAdSlXlgrKO9nfVlTGbwvcP9iL\\\\\\\/BT2+2pHP4H6jmAEiWNsBOuoc04HfcBjQ7wQPJqScIY5n1mKgz8GnjeZc0qzes2HShLe0hsWmXunRNLAdJWtgoYIfdSZIHr+svpBrUjWSlRmfbiJIyVtCuwRTpNPCjIEF3\\\\\\\/eSdAkoYYyGrKFG77pA7oenlD9bxg5UcsP82TDCdHj+xFcHswIxoR8uTap4dJLU6MMyOm1VRvIY9PwlzS4uea+YLzU3YajYVLcFLE2GvAifdIgqajNwepYC7cUA1JBtYp80mMDBCEq\\\\\\\/mXtcj7P2nwAVLgRlDccgzpRrR3vKQlaXRxqhcj6LmYCxxEln9KM4vE3XbPjq5SeH6yRBHUnAjNEGBgNXCnxC+MlaBRoS95Guf+kbin5nH0390FZC5U1TXTWrReynphpmNAiF5YzStUbNbyCMr2Vyf+Y+q8Hj+pnDmaKXFGfd7pi4mRjA27d5+t5OYJAfvR3823SbUWwUOBRfk3w3xcGJ2E9l8QH6XfuqgCjwQhTqUpsDwgJv\\\\\\\/NWwD+T2U+ej04QZ6haFcMh\\\\\\\/nKBss3Psy8SzvOtynNVFHz283BdcYTwnVIQ5cSjLZ3qU2Ir\\\\\\\/Kq8bhtfNNDapHBRLGL9f3dlzec\\\\\\\/EVXSLt0b06XWivp7vH7PX86GfS4zHl\\\\\\\/GOg\\\\\\\/8bbxtA4gAr\\\\\\\/2sIC\\\\\\\/DJOzeCcCY5h7SSzNUW06rm++Po0U+rttZgwgwFzgtyaub0G\\\\\\\/2Xi6XDx77zVlEjO6goTudaEalWC+Xg08cNMHlYo7daRh0Sh5bOUmrTFh8xrmcstApewDntdmTqechzzLSi6\\\\\\\/hCQFa+0maJtU6t8kOqlSR7ymjxEY7Qva+LDHn3X8KITut8oVQmW1\\\\\\\/x56C\\\\\\\/FXDuxU6R6AhRdYLUvm9zRJjxlNWzek6SUY21iI8XQGAQpHoRM1fY4wLpOlt5gfQMT5\\\\\\\/zis1BeZZs7vUBS8huJEHaFuQmtsnkffF5OOa6ATsd6625Qe45VAgdXHW1VKc8+RbQ7NwPEwSkeafSCJ8lHPzpNga4\\\\\\\/fhX08qkGJ0AT2P5USGZCsLedQrsJumQauKa6ilMsSSeAqdKu8ZIcoI1PkOhEJfbTsexhopMCNfflisJ34wfqOnHmVShhgVpeNfJR8I0FrDUnACMFXlg54ppsbf6l6tgeIhOZ6lFZzgCRFUtVFbjIN7fZ3IV0RqzyJtkF+1PpUJeH60rOOKmmtkdwZ7WNI5OFxwm7OwH5ZCbn3i5qGWa8u8oGO0fDkn+xDMtDQOx5U32t2MaahuDr3+ehlmPytXu1od33Con4Ig\\\\\\\/W6KZLiQDu7Scip8r9eF8o5AVCj88qsKXl8Gg2QTNret9WN8lqjICpJiB+kScxevLHn8tDlLC7vtx\\\\\\\/HcyFnBLJ2QKy3YryK+YOAHFNMVwToKc8vcSbQ2Y0EgzsR5ehW7ez\\\\\\\/c0KkzoX9X7BYYoYCIxmRFd42hnvu8+zqTndac8HsQfk2fa+6jINc7kgrLLUtPPeG6PmQaOERB2KZO13xZH6+uI\\\\\\\/0124XbMCybAQEg4xY70odd6erkkA+kbb2ef1Mz15TAN+MhCZcMLWItxHLm38bs5wAQbTIhkniMA\\\\\\\/c0sK1M3UicHl\\\\\\\/r+MY5N5O70LmNOOjoEcoFeVB4pi1uLJsvx6i4Pm8dRrgtyb0nsjjBndJXXzaCmMUWsfbbLF301k1fMHgJilBOY5slIOKolBfBi5S0gAJIDgBMZb7m+3tC7nf64sXhCCZ7mqv1+lgHFN0k6XnTklwQhROcN7XElyswyJQby3fCPVZwl\\\\\\\/JRHe44L6oIcosfQUMcVovc92t831WWj6CG9MNLmcyQenuDqPxkOOr9IaktlKFraa\\\\\\\/AAwMEHo3foBcpacaJ7cwqGEG2k7U\\\\\\\/rkhcXa5Cqx4DHHk+Hwq4ts7tzWiZn1BlDZw1+luYD+wQkmuLAYSIPPrMXEqPP\\\\\\\/rUZHTHBR5jWY9g5gpy4KmttopEavEKebLnDaIbQt+KqbauLXxZV5pYo8j+50EQo2oeFaU7qrT2\\\\\\\/O4jc14o1qit64ex6zavH1xchUOsKHF4SojymRHP73xO16af5dlLtPAKIPz1be4vxegzVlQNh3vVC\\\\\\\/1KFUyLJi8hXsZssXKUpzYnhgAZ6UawN1MgwriVMWOO62u9ov3vA2i+j1\\\\\\\/3KVdUn\\\\\\\/OCsdS+wraD7FP4WVvFMHxB8SBewnx4UuR9In7We6cZI4JP8+KfPO9ncwlwhMC+MKxoghtzfIJL67h8GUNuy\\\\\\\/B9DukunW\\\\\\\/ZwkZ95tDJ+IRDkpgont1iMwm3qdK4j6d\\\\\\\/V7nBXD\\\\\\\/MQXahuU0kww0PNvK9koa7UUde5coJpjXjRA3tfLBzTcb7\\\\\\\/vRrVZWbdozIp2weesw0b\\\\\\\/XZ5Cga0YvSQ19g1Sv09KJpD\\\\\\\/yFjHuAdgNHdrk1Eiycniq7X53IISAIslKlX8rdbtmx8rOj7dD+XhRZf3T0sniiO\\\\\\\/adlwpczQHHsecQnjV1Odhd2BerFNSmjF6q11DI7sVjCjL43p\\\\\\\/uLZfhec5xrUTB0GPqjoD1dku77sPMeDpsm\\\\\\\/k9WtPb1H0BdGlXez\\\\\\\/E6YS+FBBjjk2dp5H4XfBf7kQG3vFu6GIg+g122moVyzVkCiYf7E9pue9oZkvJs6tpefep7LtGpgiVo4Wrn95z7WINTtcGJc8pLFEZC6oxBreEvHB2oR7cZvoz88+UNPet3y42yLRztB4DvL61\\\\\\\/RcezGNSe0AqD+3ihXak4nJTV9Y2uc1TzVmuMnUEudUnEd+L3zlFDWX2TudeYKaG38zOQ\\\\\\\/74R\\\\\\\/whHwlCs7+IlGzCYR9JJjVaivg1J\\\\\\\/IN34wpEseNIk0OtQLsgsY83bFkv9PmNigbvYr00fLKU1Xs\\\\\\\/LoArMLhrri85bfl+ImBniBEJo3JWqwP\\\\\\\/FzYXs\\\\\\\/ubNnxL+VlZx4hIAtdgSb+gOxa+cIxo+4IuihexDVrwXXyFwgk5FGl34olIzkcjkl3OMI6vzI0UPcL+W0WyYNr38aTbxl+juSkSPpQBe2e+Lg4AsPQMOZWFONyjP\\\\\\\/ik+Bg9zgwf47v7IieT+vQ3OvUw7RdaBfOArUEkj2Tf9hVM+NgzDIxjCQstGoq8QYCoIdK9DXZSaEx9SvdppDcNFf+DiZNwzswtb3hZAUZa5SeD9ZuhPseewu94HTLKKHGdZ6FQXWTowFeigHi+UUhdXVL9kiePp5wfyU5lyGuPjEw6cYYxaGJYW7w9g0Rp4PQ3ZjaPP4wQxe8wCDytbewsUmXEwzDEaJQf5F0fSIf0FWFjf24yZS3vWsAnyiS\\\\\\\/W+n1J0Qih9n\\\\\\\/LyhpcJSC4r39W6kc3HcV0JegsgWNQi87vsD2uCmGhy87GeBXCPkaZ3mKghgMKa0MJbo\\\\\\\/z18GlxJTpcusHaTdIcOcXFRIwepK\\\\\\\/mtRMyuLWz6Zi9Dm1szSjmU+ZxmH8wJ3h9LbWWPXsNxtuISIBm5evoTOzh\\\\\\\/O46I34\\\\\\\/m8jLWQNDlFnOhUCWTX+HeaY26n4x0ulcECMm7SSnFUyrsrdz51I7kJ1vKU7XeDY3HuQmjMdp\\\\\\\/BJ9\\\\\\\/jU+4WOaunC\\\\\\\/PNCPGXeD0msqOp6aq8CgmRLbXU86prMLVMmJgU\\\\\\\/awY8bpXmCLWDzqlMzlVH\\\\\\\/Wo5GM55uKLWQDExY9zdCTP2F5q\\\\\\\/JvqmQNF+Ib+E9KX+vHfQ0VaZgIPrQurRww+H98aT+L7X1JkM89pcrK740PnlwGW92VHs1Xk\\\\\\\/31mvSmzDGiqX8KnFRTSQLD9o1avmD1LefVaaki\\\\\\\/dzZhOCaT3aDdcBPBziVykZ2uaO0F2WZZm\\\\\\\/fBq9aEEXJ7ouFFS7a7HdbpR2LFY+bDHajGD9yFSM2bpQkD202YNwJamDq5d5mC+sbHZY3kSOoliN0+ALQeTPBN\\\\\\\/ozXmCvtY6I1HdTJ2+UY+\\\\\\\/BfFYDwvI\\\\\\\/3+woLBTjTkYId\\\\\\\/DuD6LsmWxQCESFIUmjm5iAGtBFKWp\\\\\\\/3mCPi\\\\\\\/JvsxbYpMjQiVxDaiujAeOcKzbzpr\\\\\\\/AT53F68eL1kQHmHYSCzJyht8KjFepxB5lMjwfmaqc4lSjXivC5QpFqLKKK4T8b6jIwRv4PyRVU1ui+nBnN2GfWq+6goYryanLEQkalSd2sm0dW4WsmUatZQ1vtcTeLBGk2q51GfSP2Crh0i5GCgcAwEZFRKRl\\\\\\\/HyuR31SYNuQ6MiOpm2KrbbOurO4YgwoyNEjUL0IAMPMiEMmSoFb56mwhX9BeGkSHR9JzS6OFAU36wfYJ6NcXMHP6wdcxWoaCTgfv\\\\\\\/hT4pKEjNYFTJ18cXl0ExbIM\\\\\\\/1xJ7br9XOB5hs+I0vnV1djDLhgGMl292vAlTjiiSiMO6jUxJZ94kUAGhuox7gAKgKRv\\\\\\\/hrUflsYKW9+g9FBY+n7xvH56n7obgWbUyqZTBGGNORiYTPi0whye6Wzd2391ZuvSrleCCuMYoGdyo4H2B1wXCh\\\\\\\/Kj0KNJCNopHFpcaYOuyEsQpOtvy4qHugK1wTORdPPCs73anU4DUEZhRVJMqFyjh\\\\\\\/7ilsKOXvvoirtq\\\\\\\/ORA17kO43UUdBwiKSx9thFtDPKsuDGP4jjFdrO5f1KlwuxIATJHEmmWULdCSxWlFtolP5daEd9eBPp8fjTYvouB5kE1CPmT8vjDHa0X3O9vA77r\\\\\\\/NerJZdm1lRuFaAaMpTBTZFXuPenBDsS0ZST9IteebxDnlwAbOCVZyIYqQcbhwAohXsW68ajOLmyGt9pik4qxpqvGdCoDtJZgg\\\\\\\/eu0VJdcODbjlsr+twHKJpJ6kCtZIZk7mhcA10FG6ru39fHGPLnLS2duTTVZ3OTa+xTfwYIiprZN2Awpj8foN1c7sK9H4bvDCC9GMwj0k8DaEITRgRk0XtDzKU3O5SYvw6Eb4xPe\\\\\\\/1ecKWdxTwZ9FvdWT00KbVKtxA6tYLpFxWRohrYZTy5jjdgv\\\\\\\/aA8s2587DpUz9D8bYwmnmV6MW1QJIdebhYcVp47NbqrNGwHgquBEwSrCW\\\\\\\/tsK3Xxc8TeHWfvk\\\\\\\/qIa5hro0ASSaI59XjBgOQGoyl9MiLqInlmkYgNjodchELyl+W6cy\\\\\\\/9jhZ0Glhjv2ACSxCDEwv5xVPTtxM\\\\\\\/yzD5vrhwgHZhaUecROsFqOY2c+qo3ObQOf34P7VaMPaNTHgi1vUH\\\\\\\/8O1le1rrKkun9mOKyBzFqif0e\\\\\\\/lLpmmZThnOFJ2Z86Kc\\\\\\\/jAHKcn7kZhd6CIFMMOmmCiDbmIdFJgQBgquq+xj75BL+FewFel7UJZq8hmWWwdKgaVEVxwnZ5lZ5jCQAwMUkb\\\\\\\/hUt+nsYbT6qg5udVJgpM48qGdZFl+KdVZdEJTP4+WUPtA+z6J73ME6\\\\\\\/6HtM3oH7mppi8qcZ+TAitSkq48b5F47eZzAdQG533h41lscksYf8oM5Oxa0P0IJVxfAYs9G0tLcDPhjmGEdL0Z7l0qVyHquMHkZs76iwrnIoOf6Gc3tUvAiOebiKhK2TgGBLhlfx3Axrkh4ZvN4qUEqtyl5RV\\\\\\\/p4ZtQwxvrZ7zDtXrI\\\\\\\/f6pl2fm8MSlDRqNHP3e3KkkF0cfHYcaOriITT85J4MyIv5Ugg\\\\\\\/GOxOpLeeyCzTUFItqGQF8oBQpy8lHT5Glp2T4c6vC5BLwpYWBYIVb2lOKPAUQKmajuvduzcj6h09fhjV\\\\\\\/Mt27FMluywPhspXFBzsLy\\\\\\\/ymgo8UgB8SaCfYGHc0GsR6\\\\\\\/iFNPohgahbDyCuQLuRg6NvANSEceiiV\\\\\\\/KEpImRfwsqskKShq6QuqAuhRge6oU+iewnEjiQpzGqzeFjlMoqiREkDN7\\\\\\\/+V2iNu1PhFNAuyKMsjaw1pD4ckyUF9uX3M6BW7GO4u0FIic\\\\\\\/W6KjsTeLYi62eai9CKaepvLC+fW\\\\\\\/MY5gpW\\\\\\\/wEKF+SpL6FJPGpdDCO83gqNnomWFaRoqpTwPf4VksOAuqGh1oYWRzWg+d9ctdaVewQtYPXyWh4TjuDnnAjEkriPDGkggLE4yil8Zh0DhG9CSt4GL+YszJJbOOQt4h1DCUADRvOW044osnOrTKqJ6aeFwNPa5hu5Exg+jNXtI3YV4M2We2uiUc4MbTq+RGU1OFb9ETMVLdRY5dc7ONkhipImjKmWwFWlBxf4tEIgBVf55L7I8VflRWqxXscn9F+bsyoSDR3oZ9Dcal5iYCyfNXebCtrEeP9kTZimggWu0\\\\\\\/UmbdAWMJe1sTCyTaOjsxTq304qbeJnHQAdRbpZwzVUltUXDWqvUD0U+NBTtRVe64aTBvK+07amHVfZp0lHZhCb4sxFrDaaSXL4Ak620H+65002MKCGNfGkQj\\\\\\\/fRGMOA7CrumsUDIRg8LClPVl7niqAZMOrjdxq9f+q+I1bQ3siMk3xgTW\\\\\\\/7ZFX2FpXu9bitysIIOHVGgycORA+NGG+7ZB0MNmI6nu2z\\\\\\\/XQow5s0jVJpujymjoVCO0tyvuseOrp+teq\\\\\\\/Gyf7+mUA8d5lbd3sh2tTSpe3ukb0VWLgoiZJahzDP9k9E+mBaxGcEWCuN701HHm3qZsog3TO8c\\\\\\\/euqii5S15koLMJxhQniGOhQLHOylnoxj+E+LfjDkC9UzDyp9B4mccx3KimXtx8QGYg8KuYsxTuxOhe0nlYZVYJQtn7t7z2Kz0Q5ZdFlPKwQwA2VpOp+eL3gO9QCjmRZQoOKQQmUNYV2\\\\\\\/kX0mU9VP9VYJRXow8Br5JcWQnwIbZc4qrvwYuFAPpSMI3OEDaOkDpJuBjAytmFATY2YR0taYj8sn4sFJJK4E3abxvy9leT1N+JkTnMnTr675bkxLKdvHzxKb9g0aCxgH9whad7G5Q5RbBKAytUmBF3FHOw2rUT7z84ryzvLTiMIvUWViIPJMAJTUqdtw5A1vRBwYCyGYtDIpY8oelK1bS2uVpN7p4NvmltkFWT27rApO8JFxz7JU7X+EFFyB7MiTCbA+7FQlf+g664L4pJeNJeOYlntZnRBf+M\\\\\\\/pUlnavw0TtoHLZ8QKxkgOV3Os7uxbOmtt0qH7NYt\\\\\\\/D3uANasNSFk4T8KqSAvSBpIVpa2S6Mhwf4XoPz3v3gPskNn8nb8bcHNcM8gxWrzVX8a9Tpq2cdAiUNhhM0+jVTGr3gb+7BxO\\\\\\\/2zkmDvq9xRXxREkDl82Ok0Yn94q6eHCXidt1ZIwJXDOnZLRlBv+GZ3XBC4GUfaVeOy2b8cX3Hm7Muyp0Hls1EfpCgCFfVDliANuaYIQVIdwyCo34PFVURvoHT\\\\\\\/cYX4r5GbbYKkJN3fHDteHzX\\\\\\\/kFE5MCi79zvaZ76AnO89hHXbeizCTxca5VDdN7SdWTqA4U5IcdZX+9nPYSsoUW+Z2wIQCTeTnE+EyoEPpapeMP4szGRA1pB2DwtVXPjhXBW4lUQtiyXn2r1D8zXP21skfoe4uvAZhRcfnt6\\\\\\\/6riUOFQFu2TR60Ad0VbwRCTB54KNfAbPNbz5BqUa0Slqyf6ML0FCQ2dXCoU2Xp+QT8VTJV75W26hRFwj1MJm77sIaAec7i7bxgizoADvvtSDVG2ew03zISeM77edrafGmdEY0oSZIGq0pIVWKDwVkpJOwNu48lKoKV0GG\\\\\\\/04QxBn7Dwf4ZdhCDvqt1Mk1S+dQvetrcF86zab0LiEv72oY1J6RSeNf2+8gxYKq1X5+rGMYu52ocstAMXl84xaQ11WfJTx4tOEzQa\\\\\\\/u1onebcfEnC3ZcWMX3N2pea8H55jYPrvjMH3ct0o353cEs3Ihz5vS0ijv13sjNyMWwFK0kpAtMTthlGYTyxq4GmjoogxWCXw7EUzPwso6jM1no+D0Wulw1IoyB59SW9MvpcMOmFLr9qNCrgj5vZ\\\\\\\/l4O02bcgULXhh7OwwM6tj2mvNqL6Au4hO0guYiMnZzvKYUDOu4V9zoPqfJXBtckIlevGiyPuwzuz14NR9z0OXEUkTkVbSX8x7RbTm8nZtWZqpOkxF7jYTeJ8eSoYt1ZfEMdPMGwMgDsVZSdQVutYhF5Ji5xxnltXhzAdDvgVrsr8ZPqtOjVweYjVVvzJIjx8CBNmBHR0qtNXi2NDOxgAshnLn+oI6IoA8PSLuKnueaUA+8999sfpwQm3MlOJkSVntB+ir02WRP0tZJdew+HT\\\\\\\/b9L9ys+EjRGMa7x2G4Ukm5utWezSjoKM4GFnWs7fhKOYY2yTEGsY0Rzum2cq4vcd8bDtQ67Cx\\\\\\\/usrA3fAQdvjymqlJOc922hrYkLFGQBxUS+9dfFwrWeXeKeBilG3vqRxp63YAHNgFiwGFwNk4uvb36cR2wrqwk+sgSk2gCooYSmGK2zXawvGBr\\\\\\\/tmARE6uAmvsIqHX6XlM8yTH3f+5VgWm7efiAyEoFaSy2gfgDmEhggSbcz6JbYxtVcZs+HJOSICvuVdi+vPNc0flfwTYGpS+mQ0GdhA+v9FtlLlzc\\\\\\\/4Bdf6eCVrFEeRFIVZA9rvXYJPdKT9oNYCe4ZqW6Js+FFjnnYPhGHT7gjL6\\\\\\\/33rCipUwBMJ1g5YyPvqCLey2uy5AGTcoffOQznJ+D6EWgzM3uhjx0IqrzEmth7XcLcvA886JziOGZknca4jOxgcPqEZXnmo3LtPILqU4eLvKIQZMBnvCnsNKZ1alo17TgOcpFBiyZtbF0H3hNL85nWVNdjd5ho0hNoU3jG8xrdwnR3RLocqf5ug6K1ASqCOB0FFNjd+uSAMq8r8CImkmjwqmXEfIxAxMTng7\\\\\\\/oznl6EKtDSWsI9sNSdb2qj5eE3ucBFaN1GFqkIij59rMnlD7zI2777hzqGa9IwBSdGSfhiOtKF0cXkV2S2\\\\\\\/Ls5krJTBRNdmh0jafllhLbq8x6jAo\\\\\\\/CS2Ke\\\\\\\/7Smk820HRGBNl3nxPLBoI1Hq0VSfDWvKnV6AFLMxfmf48x0g+zJ8sE1OYLbD7ZI4RGI3zmxAjw4G9k8UQBVUyRLkjzid6K1rGCkwNtmgFNilay3cLUkCaxKPg139GVYEuXDjwCnohQ+BHvJSwrUp9HLFZmHopmBkJeBCyBJvjPa3bZbNJ7M+txlZhiNtfek7H2v4PGLIOCAMv+azzVvnOhVv\\\\\\\/kEspJVjsvELxmMbMvfJIB0YA+waQakXBh5yOCUNG7\\\\\\\/njMCmMtRqybDgr9oG9RWV\\\\\\\/v3TGXmC67tzvkxpK5pYruvjBkgwm\\\\\\\/NP5aBu\\\\\\\/\\\\\\\/OrnX7+x\\\\\\\/OnyeBtuQf2aCIFflfxCtKHWprLd0ByS83lMtTjWPeRn0bCbYST1SJlQf40q5On18xE2eXfwOzfkP1j0+ddciB2jYuQDiGo1nBQ4mHuswjZ3Bmm6kKG1m9eAwoNCojLYwc6LfUM4tWSbUSo43YjnSqATkDeQ1nQUCPjbs\\\\\\\/NUnV8VhzpaeB5N91C5l4FqqBikhjCfkAhOLAZ4O9mu3auaaTTzqE9UuIhX0tLqGBlVTFcew9jHn4sds1YVI6JVYHeeHSt6xY1icF+8SBRed5txjoLj++6tEV8CR7\\\\\\\/f8kDrYt5iQH9r+jWqP1XsdmteF5qTlubOyyObUckKuTVTHXpRzQbhGxU1pUEjgcIMjLtUI\\\\\\\/zFjMM3JwwGn8PwgrRc2BZkyWZLcbWY0z982gon5EBEmy8IG0asNjsPs9pbMgN6udRuiP7R890GrSxmlEJFHtsl\\\\\\\/6pd+6p\\\\\\\/qQR5hVJ7v4I4uEU8xUs8OM4CT5idBhrhBMwKNUvD5ywgKYF8Kmg2uEQ5Ne7QirFnSL5lqLDIQddGKJ4LXd+7m60nymvCjh9KO+9m0nvypWBtfKisKTPFFpg+dF0jqPrZDF0R2pRPLQDisd3Hy03yVjss7AmiOwie5nwygm\\\\\\\/Oju4LY+LyeZT7q+2F26Ji+iBBspl1dSR098+GIVRrgt2CQSV8Vjk\\\\\\\/keS1cxePvIqrkoAvW178cbEX4lTTh80l7EGvNqj2iuZaLCWCvrU6RJPWOKmnE9WuzdygHu7uIFV\\\\\\\/buUBbXADWIcON9rhEIDY8gb93jYJpXDzdA94JZumOgsWDj\\\\\\\/hZnzE8+CSHeYTMGzCUd9minWotiVGHHfYE6Itu\\\\\\\/PYlRxAI8Jbh6yX1\\\\\\\/MbQaDoduTZIcyBYDUmTF8avMW6PtSuyY5vRXHN9qh9eB1WwZeGHc98zv6g1R4Yc+dKIrjU1Jh07wCn9ubkjy6UpCnfHNteOPyKVXDpBDGNb84W8ulFQJ4tnPRZR0LKvlf17\\\\\\\/3lOsYawFHWWuc7n7ht5m\\\\\\\/I7V9H1ooKU1L+ma+R7BzGGV01SJmqWje\\\\\\\/MNcp43W3M1cVlswDSvqcMg8VoIZgTKJV+GMvzL1QxXOTAd927HEhyVMxMacaCThz8MNsx56TpVCAWKvH6MK4QV4Kodx44YS3TdHUtc8CVCM2wQqb8kBOXXhhXtU0if0LRqYqSI6F2YNpV64HkGRW3G78y5p4woffGeBXS6GaaP8tuLODrMwRAAXkHKtosbrn1pM1sl7uj\\\\\\\/5bkK0A2gRbakgmr1TuE9\\\\\\\/4BGxoFuaZZlrqPf0xj7jaC8\\\\\\\/JTURpNRIPlgnDBo4ONQiRmkUES+ijNBQDjrP+rxE\\\\\\\/LNHXfbi6sCShMxeJV4jIj2rpEF3LpSBAjKgtSzJXDS18vByavpPq29sTO7DU\\\\\\\/P+j8bx\\\\\\\/nI20zKmH1A3Ul1FRxPRRE9MbypOUVNeyabWhzYyJhLCV4sGx\\\\\\\/\\\\\\\/A0cDcaFtD\\\\\\\/01R3tpyMRmEG70k0sEZXFJzviL2p1Ox2+fabFtRfVN4B4lyS3i\\\\\\\/f1eGAUVHnhP8IOyBNBdVrlHdvH+zHVZ6871gFu9MDQIy3idS8Tl\\\\\\\/cWp9DSHXpkSTZNsc4m0F0Kn+bBiYZD+BP0nyxOuwgvd+y56HFMjWQTPox6i1GHmpEp20MO4yf7SCrme5Hav9JwXO2+eOFfJ9CS+P2iq81DGxjJXaBMTHqlFNBo+2Upqaf4XXo4+RQD98YnoQ9pdibkTLlMp6TwgLf8s4mgXq1v9v11WMpWeTcqQCTrsM2InIQoxgTeaerFeyq9T2TNxBumWj+eU+UP9CyZFZLoftJO0xc888dftPIQhWaG+qWBPtkGjNIcLOHN3W1582y\\\\\\\/V\\\\\\\/oQv0OtpzLUkiFS03Rsgy1NgD3MTkKTobkOGN27ZmjzvX7yMlTGGhRYsEn8bRpLZmSzt0qgLMkv729s\\\\\\\/Jvo9G5ANIM1z1eRGtPPHPumI8tDag4vIIM0+rQ4ZrmlRTTaWa8iOFlBX0++1y\\\\\\\/YlLhCGRaDgTBD+hnPos8iYOq+Kj7KTK6N8voMxfkjrSOLOphfeLOU9lfE\\\\\\\/vh04E7RGzLaqIcuBSF5cYB3x1Z51SPw67tr0ORcGgRBZ5OdITxPg72EWWUPpM2\\\\\\\/vajhGOoZi1QhmhbXHp68in4+pACco\\\\\\\/91jjnZsBu80vggUcNb5b6ljy8mZbMUAonQK0fbWxr4J4HLVuQ9Rf04lX4jdMdu0+KfMuzK9jqBffdryf1C5RIzpnfte0RKSwISDV\\\\\\\/kE6jgpB+GcJgOZnGZkqbOeDSfkduPAsOg3nDFIuN9pYkhb+7UdBNNHa8wz7Rgr49eVK7SeyZgK2LUhmgIcgNPgOMN+GRAkX06p+zoicWGweyHkPlYaPNYg8WtKFTSlijvSlIizA\\\\\\\/W+WGwqOoYaFLkvC1vEnYIOMesjvPP6ui3osm3G7H2Xbq2PvGk8a1AFlDnmrksS5mzQVIZ4DphoQxt2hELSf3XACPpL7RvWmIVTY+ax1A7NuLQ1FYnnDjRtb6zqoLNi1mZwCrMiGlHwFIk+BrYkYphmXz8gAJVYgmBW13qaXKpgMlfADAAcexdn79T+ghUEpDKyMkcZ9EXPYb+HIvjOCxmras2B5sFycPTZOoF2C+b+npVve6lkST8R1PtsyPptUUIQqC3foakbpqRNNjMd\\\\\\\/DT4f3ME6vTwEJGmfYFKCoQUBHSEEdOWvrMQ6wTtp91Dt1g\\\\\\\/xm4nZu8pq1AkkZsphJG0JngXc74uPeDiQXRYzrtxjJ2v303fWcTaTH2nqwEXmBdeLZDLt2cbSY7tJjVA1mRHwnNo5Z0nxK0jo6sYaN1ht6r6ER5waflbLdpnlyFPlD2RGqsZcIWWO1l9EVZna9RL5+u5ucj3a\\\\\\\/1HjtJ5nwmG80GgFU7D\\\\\\\/9buzo2Ogfeqik8Nf6\\\\\\\/vjJSkQF05dqrvY11MJH3F\\\\\\\/SkERW56F\\\\\\\/qrvuuSphVSvYJV71GZfvwo4ov2l5QcRfiw4XYqwSW3GQNNM5ZHzLjUvD\\\\\\\/kwpmjQ+k8euCo0vic+YExUVkZl1ecM5dYSTwn8gQVHhxDxT21xiFhNOJCZY6plAFi29KEazPle4hSlXJcU13oklT8h\\\\\\\/a0cZeeEetDX3SagYQQ8VUulJQ21xBsg80kDAiF0funEj5rt31thoMZWU5g89wNhooQAnjmAaBvZp3p\\\\\\\/movzpRtLrbUdmiPzfLXCdC7zPrMSjAtdBE+UbOZ\\\\\\\/1XGJHwFZ+lZDVNe4hPFWOwgY3zSqSb7yLnUME6vE3q2oU2u3OJ8kJF9dSSK2tsWIGWyb8P1eM99vUEAJ4sXdxr9soFbNw0SwTKUnx6tIlQijXsaBET2KgQjmaQnZwlVL8VcdQx3k5oWPWv\\\\\\\/7tPW81+z9mdJHJ6qX85Cf+N+qkKS79uEEugqwd+JTA5y+PdKAhMAnvr0Hj+rkfyzgg1jBNrNYHFIZ+EkKfdqSdNDCOsUG8vaYbnpTdr+Vesgec+izm\\\\\\\/Wg+jMMNgIM0tEBDcQnB\\\\\\\/RbWp24pW8uKKOvPO0MLDVr+ueok2hyyumY3npxjfuJu0Vw2vzniY+bvaW2x4JTgjiCMLWsmKsozmgKd1yczkzZVyE5rfJ\\\\\\\/luhtrC8DgkyBfvYT13MIZoHm2OjrMYHav3\\\\\\\/1TzutpUUCiCztHVgdTF+oP7YqWTZMmctX+YNuNCM1A4Kdb7hG5Hx++Kurdqqr7\\\\\\\/LKGP2e4gapZqejGff5FdV5bPL5icw0Iq3DTTHz5JIh\\\\\\\/1oYlwQ57wreaia0nmFxLOBKXgzD1WWvbfyJLFT7\\\\\\\/PJufdNLET4E4o4twwhDErnVVocZJtRDgZ+Dkd0+p+63S4acb6n+fGvMVRyKqO75qsnfotwzYBLqIGkg8w2bqnSGz1P4mED7J8ZnLk559PLWZh2qXKuS33EcTLY6AXikKDA886IoMVQiPs6oNVvn\\\\\\\/YCqZUtbhdPq1j+Lij7\\\\\\\/C5zy+5ESM+uy2cxTiV9Sfamvan377jAyC6LZPA3WTUI1we0E0fdo\\\\\\\/NADHef0GYWT9c1ZxH8dBZz3I7M63UejeSqTTBYYuHENnvvd5DtpmzolWq+97MDj9mCP\\\\\\\/QADzwgv3QlOcnbe+2m2NKmJ3T172dmrBK6+7gh7xzRnfAp+mNnuPhnfC0xs5+5VG+IwgWR0uMw2O3bPfea15RU3wZxdckgJZPAIFzCu7CQlGrNVZ+1wo3I2PYQLu\\\\\\\/sitVmb2jWmPPe0lgGgw\\\\\\\/FIvcuqYsuzk6WAtn9ZhVBbnY\\\\\\\/mjZ0Z+7YUzePtoIjM3ntLMH7VqlKbZ0t5X14gzHVz5Vzv4Thg4UEJG8VEJ+UFFSBBjOyWYBpaLJe421e2I9ZcNPqyfyvzZ3srMvgGkrk5P2ypRul3F9oGorWlomVfq7l0BEEm+oneCXiPpZJXluGUB4J5ZToSIIFrf8yLDSrgtvyRGfn6yZnpertOYDRRHudrXhF990+ISxQh5VPOGFAzR8YkGPiizHNCGkDwEo7Ez8y3PYd6Bz\\\\\\\/Mt4xLJSGw1MpwwLYQ4gQOWqrhf0ghaChkbtsF97Zh02miXTD0\\\\\\\/uIsCgizRNJsERh1QBtJyEyONM71JGR8bQ9\\\\\\\/QDyaO2oXGtI36uQe3b+7YLXMj176\\\\\\\/oiFJn04HNMPtNN6WcR0REHFiL4ruy+bw\\\\\\\/XEHgjUpvu9k5PGg3daVK4EFC+LgVa\\\\\\\/EL4FLvubJs9\\\\\\\/Glykv2f2ENed8Tw956UPvjtPVIFDdWR9gPG3zZKNROsy7cVYVbXB5AK1E+TjIuvpV0XtpsKx0GLtCRREz5vri4YdmjsxWyCV+vZC4qV\\\\\\\/1QMSLyJJAtb9KxWXyV6lsDnXN4e9eDMhGXMVT5nvxqq5yTdpWKjKL4iZzV4cOhOghcdnZkbDmUAxXDzH+2WA4+5C1Igbq+4hr\\\\\\\/QoLPhX4qaapWmO+fqli9IQ9N4EhhzxJu\\\\\\\/cX1xzKtTzzGmZC2ADiiicZE6+vM+LAMDISTi8YZZ0moMARQVNJOyK9uaNmzBpxL8KOmREccSP20TmDFi2dhVIN4JtGBElNGhBCvAq\\\\\\\/aT3Crjff4MTSYNpBGo0L697u\\\\\\\/PtuyETg1l63tLtOxZHIMsJENXB\\\\\\\/pZV0YOCXEJ2l1FxhO8jt4CKhrmwqdV6atQbiqjh8X\\\\\\\/EGPbqs5QTyBJBGCHjc+vgsCwUFI17eZy5zvwh3M+CywSnIqRbYmM7BYRg5Fayar1pYpF1S2BG7kIEcPxA1dtRCf77qLy9NvmfGMe19UIv8TAC+S44aj3+4hxZEEYvxwuUEuidrUzGBI8wWYIxUXiPxKK8kPoJqK7bMzR18KzkzqojLNy2nh3gEYq6EN4wUCdgzqn4CR11baz+Wbgp0qrhWdTFhZLtPaKg5AW8oshOd3wsOsXClLlTYDf4fjsygjrbQRHH1CobYnSjtxUF7k8cJiNLwwJq2bamZwXtMdASn0ZpsHwigF7jnwNiJzL1HliqLttwUS4PPHJ6Ru4qS4V3GMCn1AIgXAD5WtWXMGxMxZHPJFcosFyCZChbFcggOe8PHNSt7kkGKXWwa+wSdFsWEYqAHPW62dYq1Es7snqAmXImzZWCvWU69bZCSaWhg7LsdYbGBqaln71j9OcvUOUwlSUOfWb5yGh8mvNS4A1AtOJsmygmY0BcM5aOBQRGsCO1HajKB+AOPezQvlCtQq6ubtHzwtn4dsp+gvmazeJmWI6x9WK7+3jwOrRcvkY1I5O2+0lITXrWnidZke5GsQCNDjEGLWJlurH\\\\\\\/XFHInD9\\\\\\\/yAB9f+ZOBa20gla0QTbvj4SMG5ImZr4fjw5Sgi0asvOEtdH5vgISYlE8wirDgkyKvQGoyUmLQ5EpIuUHxS0ANGfpHV3TvN+MSM3pqXM8Pnl6VC19me1bwtIrOo5NlJCWH4rPSPMGhvkJJW2bFFIFQWMg5L3BBq2sJqUQ+BCy9W6xFii0LietNeLyjRi0xauFWPVFKuj1KdS9V0wn3koabhYmEZohea81H0Hnwl\\\\\\\/w8D9P9g595rAqKT+ymMsGa33K8\\\\\\\/cZYYrJV1o41+j\\\\\\\/E4SAjUi76pEAP4EI4ThiKbPoX7ZKXkghrJT3wXdYZMgwEOiHM\\\\\\\/Fp+aoI3zdr+o5Yhz0BTvi1Or1lunyApUz8Gr\\\\\\\/X55PeVDeJSnu36R\\\\\\\/1z5czc8j4R8y9Vihc\\\\\\\/JCh1mos+vlZvjFr5Tg0prXjFvbjlaLwxsMJuSz1oWxy1YedLs0F9u0vA5W3ZgeaphHC2TRQVy9WLMw1Kk0nuMY10L9s4M8VZLiXqCtbXbwxjHvaQi35FZIwwT1LpGbUVyVSWu5IJOcVqHIB9HszIWEssXKk9RsM59JaWvq4ieXOPQTe+JgKRSKV3SNx3Bmm1BsvBAcO4g4XrgvDtWQsV5uDRI0S\\\\\\\/7+N0PUHWYJaAtRtMv6+mDVnNqeGdheHg37j4A1nZs3ew4cKFceZTeFunB7KDqqqY2S6Y16ihbMK7T7h+NDz7Y19qH781HKJs+je+tai1RE9jAlEFxx6gkhzHi8H0LuIFd7YrMZwijGOV3CBbtD+cOTqK98h\\\\\\\/wuVSBrUBFRCJ3qAdqvz0JqEkV3jCVx+srjUUW4uDLNWZ8IR8vh\\\\\\\/sh1ScqgyKtkyhNvE7j6aG8ieF3ZWGgeHHgrT+PswNI2k9fOTylCoafxUpDiXp7s7kBEyjcD96Wa+p\\\\\\\/Idi1nY86lC37hY04bY7jhDJO87mBnpgb7LUHnEDB1k8yeOE72MUfD+sjaSI9IHo2FTT5NQfmBMTxUn7xWbTz5642X1AzTCWeuvBo3KrTo1GmsJQJcV97GomeZMBQeKAm0Aaxut5rFWWKYA\\\\\\\/HWgnH3w0tISvpjH4LQC2oP+6Wva5ZgK4q+Uo55rKVzo7Hybc\\\\\\\/GvS2sPra9OOBq1mMaf2rjWD1WWBosG6XWRiuvfZbzm968Be3k5taxlR\\\\\\\/dhjY5Pu1JRPxRJwQNUL19D+w\\\\\\\/LJADUWiIixvWuSalsU\\\\\\\/jGyTt03Q+HrJQ98ctF4UOE4ulz36AyYJkuxmmvnLdm6HHZJIpIq9vDJraWafYtWvc0mIP0IQBkS\\\\\\\/2Gi2HryuoiU7ExUhAGLKHANjNAiKw6R9IMhcr\\\\\\\/zr1CI1ksH7vNmRoVOsC7s+U8bpg325nWjsyc2NucJtJY8RwxoA9YfMwckVEcdKZv0FSRAAQcT2922cpbemQj+I9Oj0PRPb9aUNY\\\\\\\/BTZfY\\\\\\\/HFiq2\\\\\\\/rLtbsXpV8z88fI1CdgGQcqmX6yerYKvlTgVNwtUuRF8w5wLs5iLktC4VFyR71nQEhMvigdlkDpeyNks\\\\\\\/gx4G4ClH9C2839AgWEtvLWT+s1T3Hp\\\\\\\/lkRZYj5J4Iz78looitZpMF5BdJcl1joHwfkSza9BcvbpNy2ucBQVE934kjLJuhNQZedmKHlqrgE2dxFxjOEgbcUgfM\\\\\\\/JZUb+6G7ZTHHXLxHgRUJDyKMUl0QCXkXhHfSkWYDZe4PJGw2GuaiL1FterVBnAc5Ey5Wo3cX8OirwxadUS2cjAChaCLV5D+EokZSytxOz1F0\\\\\\\/EurFMX4WMMdVnBDdEX++iWiM8V1QMtzEmwVuUrgFRiaFTrSRDUBEZqeHNOPqsme3Qczb6suyemgLZjBe8DrOEyh+GjYBLomIs+8kKC5x16+3Uv+3a5JI3riGUo0ICBZh6lCVrv7pHZ4vAqgOq4\\\\\\\/5BPqZlvQFXs1bG1c2rlgFepiKyK+NaWURGvOApM1zUzgCrfL9uvJ04nRMAvGv+BG8RozJz7XaxH5Qmrgk4kkW2LQVoYUktBZc+ti\\\\\\\/87AlRk\\\\\\\/MY3XCfIQkiVL75P\\\\\\\/oz1d7utndl3S+09so3u8KR1ZSZ5TqYCBwb6pCivqZZhich9RUbcr6BAaVyZsryMK8VoEjai8jIgDQY6jxdDyNJvKKbJDpDq77uLNs1lTXRHKEAQ\\\\\\\/esOFvsYRJcZRX\\\\\\\/6MXOwxtjX56\\\\\\\/DEhQWhSzzX741X2NSb7HlKwbTzILzhTJwtacXaF7+ZaUv0S3d\\\\\\\/BOIRf39KeJTe+EQcOkgcrQ4jwog7XYJ6YQd+aALn3iGmYjUlr1h3xa3JnodzBS+mHVoUocJaoa2y7mdeP9Yp22XrMdJeP30jsHcr9Rus1\\\\\\\/Ush\\\\\\\/LtGFtQi1gqM+lEfmyZ9PL3RQxToiV0XN1Dyuv1OB4q6U2S2eLHYQIsKdB85eefhT0bZ0v9sk8e8nDVXe2fhp4BOPM6z8uvQfursDInhvx9eid9O5Jhz3X6pHehRAi2sFlmfZNlvLYDJ+aIjF04JqFIlrAReU8sBLmcUTitk29XcmO8+SNT0ziDgyCDQGV7z+pXn7OJDcHD9WFutDv5EMyI7lVbCUynLeTmtr\\\\\\\/AF8tUMfCepRNcKuaCELq3BCeoKqx+EF9L0ayWUovExwLESgVWnSD0XMpD+Ta+H4k9MpLDuP6gRsh+069AhJXzjPi3swPr4hCqKnVZM0U70kpAcakd\\\\\\\/K7SpRvclTZWgWwdE0eaOx2qmaZkQMGRH6x4M626Jr2vCjQR5Tsl5HXrixEztMSfFBPPXar++YE2+DlccYDFUFMIgGw713JPUsMMRob\\\\\\\/znjewLESVmbZHej8seYH4ZRJpf2MIJ3\\\\\\\/DhUg79MwHOlUj0WnYUWbVTuThCunGCKL70fLvg1s7S36jXKr21DJGuyq+1t7pfgzfkNGnx\\\\\\\/ewxH44qlvcO+rIEI6Zk27PDpfKkqsSRSeQXTHhWT+S6Pj6vlF7XIRF4flKwT5ZDJ6FeQIdqy+V1azmvi9LMFixuV4QXMCBJCIvzCYwQuZz261a2VsmrfSuWYRHsphxE3J\\\\\\\/SWRuHmxZDlQsO+XoQyPy07XEZNaBrP1lsuusSs+0B7x0pTmgPtT4chjabHi0xZNhobbdGkBU7gFVHGZfrHqrK93mX5+iiEnS+vLCbDbCnp7PId++siE5nmJGPQHnLwmtgKd\\\\\\\/KkH8\\\\\\\/rvu31FaD+w\\\\\\\/yghOMcrECI6wVvwUS7jNekB\\\\\\\/KvUXl+c8rGoLIH1B3FlnhHCWa++V0cs7RESPbu1aJvAIisCHSA6nlstO18vOPpQJaJfqMunV9fQ0G8F+BDfqBBbI9lCXq+7ExsS0mRsVWhtpfy5EMSc2SaAw2WVTH5q08Bk\\\\\\\/jSGgyvY7n2Ddq8kFHlyWPk1n7\\\\\\\/gvK6NzJ6375xabw+yW1Uz\\\\\\\/WL9GbufaGeKr9pPtRLrhS38xS9HsxdO2xnMFRfZVAvX1azkJjGgMrG+XlX5dRyLjvYP8sJzFIE86RWE+wnxh7MgD6Vpm4VAbvhMc5DM40E9mOvbsUuIWLZwjf0vPadqw2GAzQl2FDoG5qpcKzPoHGf7HN0PVQGTXhza1iZHKz8POVNmSDVFPCtR9N5ZiCmoWyEdvWE\\\\\\\/Yd77R+aQqT141UKHhOpHPL8mV6hSaPgwsH5OT\\\\\\\/avboSSxiD5tsV\\\\\\\/osTdHXseHb7vYO9b6gwleuFa5aUeFbFvyOfVZuvO83cFoxFAleuSD8lwT1AwasrOf\\\\\\\/qTP4tTh5i2avqZ4qW18F7p+pRFhNcEiUEz\\\\\\\/wJuDxWOTU4q26QnwY4jNjrXDRM+WC0VFLdSQCYta7idYR5z4csiEY5+K0TrNpdYBHXcgSsYr8NQx6nThlZeV6EAiX1cs04SKoE88bNoWlrP0F5gruut2BesbLUFR4Wznw+jXaKo5JKqwV\\\\\\\/V\\\\\\\/5Xcwh3XgMkOEleOEtS\\\\\\\/vT+bh\\\\\\\/YCyNRxRfSZXPBurcuSPS1Kr5wFLz8zh4ouV\\\\\\\/KOfBjGbd\\\\\\\/A3zeMczUgmmi4QlXxUiu7cH9Mje\\\\\\\/hfCJLsV5JVRk9a60VHciMO5HhJ4UrpHKL4XXWgc3woaTPbtWIkiCfsDOrB1VBhV+zlVXSK+j6oCt\\\\\\\/4Ir73n4sot\\\\\\\/5VEfpMjvehilrv2XmP+z\\\\\\\/Gm7PHnCjBq4sRXkKE07yRJwOUgwtf0tB2V4cE7lYtH+9uAEcpmGozbw9xfc09jvLpAHX6+fpgZBJnuEzdcBFy8rRTbhN84HEhHDSoXiMuTZ69mV68r2y+POXuNVs82a6tpY\\\\\\\/I7fm8aVcUgoukE2fFxMAmYk4ei1XAq\\\\\\\/YynC\\\\\\\/\\\\\\\/TvUkC9OqKNCSINp\\\\\\\/o42yzIljmSZUSItOYhrEqJTRlL+Dj665P\\\\\\\/8FR2Hgz1NByfHmQvWHLyDe81fK3EUpfrVQQcKNA0PyLhZjKd6PkTFCH33cw9Yz62q5QiungbG+MgTE9Uqk+NkF8NSJOhGkzF9KAhikTXpVJXQuVJUJCT9VUEEwb65wfeeCmWYYfCabV5wcyeJOCgrvRsqIdgi8BLoIVt741kncBfarkf8\\\\\\\/6ywbh++HO1RXHHV\\\\\\\/2f3lTRa\\\\\\\/afjggk\\\\\\\/VllD2ADe+zs7WX+tnbFxSSaoSXpgf9MUeBporb8wanHTLDwSMVbZGMWsgODQmdri8dKfhs3ShkT2pURLIq5T\\\\\\\/ATQNSTSYDxjQ\\\\\\\/o+ZjTy6wQOTr7MPSFF0mSebUJUtQxp5YDY\\\\\\\/TjjpfT4WM4ummfpsTkzPezTeG3Q3WYGgXlNnWRXSYYqvuGnfTAJSTUCkdM5JJr7su\\\\\\\/dAmERSlc42j7zIfDvDeONpe3LwBTw9M7dBuxozgznTBc+dhhKXTU0THS5i73DLYI24r6Vfffh8am6cxSjek9JqpMO2zGsWQaQMljAyDwpG\\\\\\\/140xzRCyynklPPxs8XldOMztK\\\\\\\/jAUOUvNXx8fyy3hCRhy4kgq1GkcVirlBiQXinIU7tFx43EiJkiHAy9tlQV2SFf3JGdFfkcL95BKiytbGQK\\\\\\\/czGrMUzfQnSmHRGDDBVKw4aKvcKb0RwtmMV\\\\\\\/b6Ja98rc4ScUShX0WqKRkMwZ6rU02jFaCESg2CKRtpIdo\\\\\\\/XOhoICtgQ+EjrUiNbet8jtpS1BuLNVA3SwPclEa22OE6nGolMfjSen9JUBsrfcffqvzGKrKnG7BN9Bd8OoPFsNgC4dJSyJaX14GdgT96m7U8vlS\\\\\\\/L3l4ORYNmBj+hu6Wpl++ZeV8KIdxwXHZfqBGdlq1lB\\\\\\\/MOhZAqA6FPu\\\\\\\/5\\\\\\\/2TwhoD46tp+bjfVSsbPs7eG\\\\\\\/6ElR2dqfzHF2TKBpIaBuSk5pG1l4qtQXS0tpanRSMl0kQvjFlm7nknHLWJkbRu0e7i19tfja0cJlL1HCtGCCZ9Rk4xj856y1E0CcYXStVeaLL8ydwkEMsoiZuoMW3FCZt5BP8W3LGVYoX4DN6T5WwpN0YZz83gudoeKfQqFTrT7A0yyDpqEP5XiGs+36HgL81wnP80M\\\\\\\/pidJxiZW8ztf4ueudRvubNr8bWtPovU5em0C7mq\\\\\\\/xxaQI2Ak10CIIzTltDlm6NRuGIQvo8CUzki+\\\\\\\/yvcfSa3nXs9bmy4ehcda5oQpRB+9J+lrDXcMw8nvV6n12WSQLrvrIaJzO8bXUXlrPxaX7q8Z9fHteQfEnCf8EV+6KUK4c3PtiE4v5I01MRpSjDhKa9YuaOiWXo+9Qa6BqZR2DNJ+w3iDWBMeCJH+dVBhSk79LEdmmKEWl\\\\\\\/pWaGTGpcESeV5\\\\\\\/VLdP+JskCTHXZQdy31m+QpxGGmULUpDfA3JVuBYR2HSoq5SANtwprHm8S1RkKi80GYYOcOYFy3AO2nFr9N54Dsi1SMWeQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q\\\\\\\/JHkN9CGiteV+tCw0KtTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227dfdd4f041d22fb27f46157873b27b6600522e21337a0f48f05b248a4ef771eb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r2aAruASI1wk2L4ou74D0rpvHcDhs1m7TZOdI4WLjBfBP+XdwiEtSJoBnNUUpAv1KWLdFJXmM4Dx8fF076pLMKx1VyG01Tyi7DBc8NAaJwsDvoe+Dbp5xm3noRo+eW6RWc0cVEQFo4kImcHSkrhAGXOupgSROt1yQtpc7+b1K7GSnf+CkOBG\\\\\\\/nRjRuRa3a4uBCbLMsFiDsvoO4XJZKO\\\\\\\/\\\\\\\/njKZs4UCtjWGf7Zx85e0fbc6cnOwhH6\\\\\\\/7LIlxQDVpoExrnGlbEcyWZDKPl\\\\\\\/0Oh5w+kgWZTIeHInscGlEt8KaMw\\\\\\\/+VO4Id7ff+RDgwPoMWw30PcUJxHCjyqkDJabHUp4jqFhOR+EQCuvCUmTKYkHx0KWSz\\\\\\\/7z\\\\\\\/3Uw\\\\\\\/wr9e\\\\\\\/IfUToLCClXLTYC\\\\\\\/rtOPJj4FFBp6whQJJuJ2oxg6aP2Qj2JpRCNhoWd1xdKxo6fpW7KDt9KsdJM\\\\\\\/uJoJyQyG\\\\\\\/RKSW6LauZcQlIN9IeKRT\\\\\\\/EjN5oJNpCSzcJQ5MQ6+BmqIGBlZEqRGWZdOIRLLZE+PbgLir0Ktik7YOCyD+JWo1udYtDdtuq7FQq1NvurXdqQ4EAZHHreyadoxnhIVCpNYpWYDV7AyT9omK5Xt8G1oK\\\\\\\/SlHPnIs17LyIgyIxnuOCzq\\\\\\\/J7Nx1DocPBESx\\\\\\\/w4gvINsI4cRjDRk5O31VlCr6mxnjsRiFRrgZx3UC5yFeabN7vZadNiGqDAH6zk7ChaW9kgGx3jxU6xCSyKTHEl9bnBwKmFH0Yaop3EvfiCeCcQa5y1BasDBCrtyubxbHxs72qZa0rO7nZYr0nze90hlums5Bor0ntCEwSuoii+MQ3gv6qjk52RBwDFq6eCPPcXEreoA\\\\\\\/w58TOoJ2\\\\\\\/6VYjI3MyltJLcKzePdbqO\\\\\\\/tKhkz+jqZfW7eZnxv5LTTwX2XTYF7TbMrueT0jPri5NuoRta3680UNjHXCkOe1\\\\\\\/QKh5qVWKsymHsYu2GTr0m2rPlYqey\\\\\\\/0qaqDfV6zV80ksOlSuZIRjw+paC3mwjxtN9RLtNcFXVEVmkbg9VfJoFukNk6NBNZTvoAMUnDzYEav+Orus9SK\\\\\\\/BNS3NcH0Qv6sN4YGE4B0c3+EQxYye03zv1iVhdk5Yk8gMY9WTqeX157tht2AADEGf1YU1YSh5H4gmWKESa3ZMkazcMajT15Cn6oporeg4q7KQaYXjcwsvaZUp7EIS6dMp\\\\\\\/TzZpRg\\\\\\\/Jq4QTKcvZtcjccgv7qwDiBegSA8Y31OUj+xMLCCPVzXzzz42ksujvg5Qr8rhdUGIwLfZZeDKzb74Aj20pWRhBnihQosDQjdG7s6RiRr+ZsTNjurx\\\\\\\/9iW\\\\\\\/B5siQb7ynGrAO0hatTydMVMy6F7G62hb+iH\\\\\\\/MSolJBXzwnWekNCD5AyfisF5D4AZ3jlrOAWQVDmnytSJiaFVHuN5u1Qf04edfgvT6YuVg4J6ZNXn0+8LL9Xo2IuSAoAEuyqB95kF9DvTO9o+guQd28kB4UNLweBoy\\\\\\\/BhnURs00d3yeBGcdnpIwmEjrjxVEa137cWRFbPSNYFFtnjWfVYtOrZKPOL9hQugR16UX\\\\\\\/K3Pt0j6atKdeRTFj2N925mzcUSd4FkXDIuOI1jhN2JzTZ7QsyGly0nkGmTrgmDSHsDdHp\\\\\\\/\\\\\\\/OXUKNDjSKHGtUKz0K2mkVzTpRhrUOj\\\\\\\/mb9ylv3kXPS0J96O7brMzOwx7+FvdAZ2f0mokcMmne32ZMj+wjFmglJo+zBkm\\\\\\\/Pl2UdEgdoUrewHeGC3BTJt+XKc1E89d89r8tR\\\\\\\/TuUpFa5gwEq20Mh9LSgKh3E0zrPEtc3XsjOZPPt0AO7JZ\\\\\\\/oniRji8pupOgREeUU8RNsFugVrRAp8ICPAMch1nGD3ukEeUHe9de2VMVoG+XPS+axgSMJEFlH5hP1TAkYN6FnTJ5pOamVLjVj\\\\\\\/\\\\\\\/6KihV\\\\\\\/2gsTRI5esfaNhLFPuCXa4aGWUrJuKgfDhe6C2sQaPlVq7LWM5XrWh7359zqJPaxatCnSv6DQrKAkbfJ27MevNSN4hnq\\\\\\\/83ndQC9ElqZ\\\\\\\/eovA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022VFBwqAA3X7n5akWX0y62Zg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229507af882bbba6754a372f6870a3f04fb56bfa3e084bb701790bc2c54b376bd7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459649 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/547" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KuT_L6asr5ZeesRn8MOTQO0xZPOXpWq4GsUmKzmjRys" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/547" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5Y5NGCcmlRugqtT9v3Dho0xrNszWK-bPF9CR94HN-Cw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/539" title="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/553" title="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459649" id="node459649" data-pisa="ggg;4/3/547" data-pisa-master="ggg;g3.113.009431" data-apath="/ggg/4/3/547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Omar Wagih</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">547-552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009431 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459649" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omar Wagih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SD, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omar%2BWagih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wagih%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmar%2BWagih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/547" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459649" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459649/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/547.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459649" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459649/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/547.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459649" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459649/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/547.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459649" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459649/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/547.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459649" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459649/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Colony-based screens that quantify the fitness of clonal populations on solid agar plates are perhaps the most important source of genome-scale functional information in microorganisms. The images of ordered arrays of mutants produced by such experiments can be difficult to process because of laboratory-specific plate features, morphed colonies, plate edges, noise, and other artifacts. Most of the tools developed to address this problem are optimized to handle a single setup and do not work out of the box in other settings. We present gitter, an image analysis tool for robust and accurate processing of images from colony-based screens. gitter works by first finding the grid of colonies from a preprocessed image and then locating the bounds of each colony separately. We show that gitter produces comparable colony sizes to other tools in simple cases but outperforms them by being able to handle a wider variety of screens and more accurately quantify colony sizes from difficult images. gitter is freely available as an R package from <a href="http://cran.r-project.org/web/packages/gitter">http://cran.r-project.org/web/packages/gitter</a> under the LGPL. Tutorials and demos can be found at <a href="http://omarwagih.github.io/gitter">http://omarwagih.github.io/gitter</a></p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">Genetics</a></li><li class="kwd"><a href="/keyword/image-analysis" class="hw-term hw-article-keyword hw-article-keyword-image-analysis" rel="nofollow">Image Analysis</a></li><li class="kwd"><a href="/keyword/r" class="hw-term hw-article-keyword hw-article-keyword-r" rel="nofollow">R</a></li><li class="kwd"><a href="/keyword/synthetic-genetic-array" class="hw-term hw-article-keyword hw-article-keyword-synthetic-genetic-array" rel="nofollow">Synthetic Genetic Array</a></li></ul><p id="p-3">A well-controlled approach to query the function of genes and the effects of environment or small molecules is to study single genetic perturbations in appropriate conditions. Publicly available genome-scale reagents in microorganisms, such as gene deletion collections in yeast (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Tong <em>et al.</em> 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Giaever <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2010</a>) and bacteria (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baba <em>et al.</em> 2006</a>), as well as collections of tagged proteins (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huh <em>et al.</em> 2003</a>), combined with low technological requirements for using them, allow almost any laboratory to conduct genome-wide studies in a comprehensive and unbiased fashion. Indeed, much has been learned about genetic interactions (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Roguev <em>et al.</em> 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Costanzo <em>et al.</em> 2010</a>), mechanisms of small molecule effects (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hillenmeyer <em>et al.</em> 2008</a>), modulators of protein localization (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Vizeacoumar <em>et al.</em> 2010</a>), determinants of aging (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Powers <em>et al.</em> 2006</a>), microbial pathogenicity (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butland <em>et al.</em> 2008</a>), and almost all other aspects of cell biology using high-throughput genetic screens.</p><p id="p-4">A typical screen consists of growing an array of strains on agar plates, forming a grid of colonies on each. The raw data from a screen are a collection of high-resolution images of plates containing the colonies. A proxy for the effect of a genetic perturbation in the surveyed setting can then be estimated as the size of the corresponding colony at a particular time point or as the rate of growth over time (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Shah <em>et al.</em> 2007</a>). The primary, and perhaps the most challenging analysis task, is to accurately quantify colony sizes from these images for reliable phenotypic measurements.</p><p id="p-5">Although performing the screens is accessible to many groups, the image analysis step can be a limiting factor. There are currently no tools that are able to robustly quantify the colony sizes from a wide range of experimental setups. The images of screen plates vary in format, shape, illumination, background, plate edges, typical colony sizes, and the distribution of colonies on the plate. Several existing software packages [<em>e.g.</em>, HT colony grid analyzer (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Collins <em>et al.</em> 2006</a>), Colonyzer (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lawless <em>et al.</em> 2010</a>), Cell Profiler (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carpenter <em>et al.</em> 2006</a>), Colony Imager (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tong <em>et al.</em> 2002</a>), ScreenMill (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dittmar <em>et al.</em> 2010</a>) and YeastXtract (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Shah <em>et al.</em> 2007</a>)] work well in some settings, usually for the images from the laboratory in which they were developed. However, when looking for a universal solution to a general screen analysis pipeline (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wagih <em>et al.</em> 2013</a>), we could not find a single tool that was able to robustly handle a large variety of different plate types. Some of the tools are limited to graphical user interfaces, which makes incorporating them into new computational pipelines difficult, or require extensive customization, which is undesirable for an automated solution. Furthermore, processing irregular images, or underestimating colony sizes can be a problem, which we also demonstrate herein.</p><p id="p-6">To address these issues, we developed gitter, an image analysis pipeline to robustly and accurately process images from colony-based genetic screens. gitter first applies a range of preprocessing steps, including automatic plate rotation, contrast adjustment, and background correction, followed by identification of the colony layout on the plate, by the use of techniques similar to those used for microarray image analysis (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Angulo and Serra 2003</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Antoniol <em>et al.</em> 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berlemont <em>et al.</em> 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bariamis <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rueda and Rezaeian 2011</a>). gitter then fits the boundaries of individual colonies separately, which results in more accurate size estimates for irregular and larger-than-average colonies. We show that gitter is able to process a wide variety of plate images and is more accurate in quantifying the colony size compared to other tools. gitter can be freely obtained as an R package from CRAN under the LGPL.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-7">There are six main steps to gitter: image preprocessing, thresholding, determining the grid of colonies on plate, identifying individual colonies, quantification of colony sizes, and visualization of the resulting data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/547/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The workflow of gitter. The input image is preprocessed to account for global lighting effects and small rotations and thresholded to identify foreground pixels corresponding to colonies. The colony grid layout is fitted on the thresholded image, and individual colonies are quantified at each grid cell. Finally, the quantified colony sizes can by visualized." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1323124364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The workflow of gitter. The input image is preprocessed to account for global lighting effects and small rotations and thresholded to identify foreground pixels corresponding to colonies. The colony grid layout is fitted on the thresholded image, and individual colonies are quantified at each grid cell. Finally, the quantified colony sizes can by visualized.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/547/F1.medium.gif" width="440" height="92"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/547/F1.medium.gif" width="440" height="92"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/547/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/547/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464080" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">The workflow of gitter. The input image is preprocessed to account for global lighting effects and small rotations and thresholded to identify foreground pixels corresponding to colonies. The colony grid layout is fitted on the thresholded image, and individual colonies are quantified at each grid cell. Finally, the quantified colony sizes can by visualized.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Image preprocessing</h3><p id="p-9">gitter preprocesses the initial image in several steps to make it appropriate for analysis. First, the grayscale intensity <em>I<sub>ij</sub></em> in row <em>i</em> and column <em>j</em> is calculated from the color image as 0.2<em>R<sub>ij</sub></em> + 0.72<em>G<sub>ij</sub></em> + 0.07<em>B<sub>ij</sub></em>, where <em>R<sub>ij</sub></em>, <em>G<sub>ij</sub></em>, and <em>B<sub>ij</sub></em> are the pixel intensities (ranging from 0 to 1) of the red, green, and blue channels, respectively (also known as the luminosity method). gitter assumes that colonies in the image have a greater pixel intensity compared with the background. If the colonies are dark, the user should choose to invert the image to obtain <em>I<sub>ij</sub></em> = 1 − <em>I<sub>ij</sub></em>. To improve running time, the fast option allows resizing images to a specified width (default 1500 pixels), while maintaining the aspect ratio.</p><p id="p-10">The image is then rotated to make sure the rows of colonies are horizontal. To do so, we first calculate its Radon transform (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Deans 1983</a>), which has been successfully used in microarray image analysis (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Angulo and Serra 2003</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Antoniol <em>et al.</em> 2005</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Berlemont <em>et al.</em> 2007</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bariamis <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-1.gif"/></noscript></span></span>Here, <em>r</em> is the <em>y</em>-axis offset of a line from the plate center, <em>α</em> is the angle of the line, and <em>R</em>(<em>r</em>, <em>α</em>) is the sum of pixel intensities <em>I<sub>ij</sub></em> along the line defined by <em>r</em> and <em>α</em>. The lines that pass through colonies and spaces between colonies will have large and small pixel intensity sums, respectively. The lines angled parallel to the rows of colonies will therefore vary between the large and small total intensity values, and <em>R</em>(<em>r</em>, <em>α</em>) will have large variance for that particular <em>α</em>. Thus, we choose the rotation angle as argmax<em><sub>α</sub></em> (var<em><sub>r</sub> R</em>(<em>r</em>, <em>α</em>)), where we vary <em>α</em> in 0.2-degree increments. To reduce the computational complexity of this step, we resize the image to a width of 500 pixels before applying the transform. In practice, images are rarely rotated more than 5 degrees because the plate position is standardized before the photo is taken. However, we allow rotation angles of up to 30 degrees.</p><p id="p-11">Artifacts such as variation in lighting, condensation, and noise can make it difficult to distinguish colonies from the plate. We estimate the background by eroding and then dilating the image using a <em>w</em> × <em>w</em> window and then subtract it from original image to remove the broad artifacts. We picked the window size <em>w</em> as 1.5 times the width of the image divided by the number of colony columns in the plate. This ensures colonies and plate edges are smaller than the window, and will be collapsed in the erosion step.</p></div><div id="sec-3" class="subsection"><h3>Thresholding the image</h3><p id="p-12">Next, the image is thresholded to identify foreground pixels corresponding to colonies and other bright objects in the plate. We use k-means clustering with two clusters to find the threshold value <em>t</em> that distinguishes the two classes. We initialize <em>t</em> to the mean of <em>I<sub>ij</sub></em>, and then iteratively recalculate cluster means as the average intensity of pixels above and below <em>t</em>, respectively, and threshold <em>t</em> as the midpoint of cluster means. These steps are repeated until the value of <em>t</em> does not change.</p></div><div id="sec-4" class="subsection"><h3>Finding the grid of colonies</h3><p id="p-13">In an image of a screen plate, pixel intensities in rows and columns corresponding to the colonies tend to have greater values than elsewhere (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). This feature can be used to determine the colony locations. To do so, we first calculate the total number of foreground pixels in row <em>i</em> as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-2.gif"/></noscript></span></span>, yielding a set of characteristic peaks. Some of these peaks correspond to colonies (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, green lines), rest to plate edges or other artifacts (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, red lines). The number of foreground pixels in the rows corresponding to one round colony is expected to be <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-3.gif"/></noscript></span></span>, where <em>w</em> ranges from −<em>W</em> to <em>W</em>, the colony radius. To quantify how well the intensity profiles match the expectation at each potential colony center <em>i</em>, we compute the Pearson correlation <em>r<sub>i</sub></em> between the expected intensity <em>λ</em> = (<em>λ</em><sub>−</sub><em><sub>W</sub></em>,…,<em>λ<sub>W</sub></em>) and the observed intensities in the window (<em>T<sub>i</sub></em><sub>−</sub><em><sub>W</sub></em>,…,<em>T<sub>i</sub></em><sub>+</sub><em><sub>W</sub></em>) centered on <em>i</em>. Here, <em>W</em> is estimated as half the median distance between local maxima of the middle 50% in smoothed intensity profiles. We determine all <em>N</em> high peaks in the correlation profiles by finding the local maxima <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-4.gif"/></noscript></span></span>, and calculate the distance <em>δ<sub>k</sub></em> = <em>x<sub>k</sub></em> − <em>x<sub>k</sub></em><sub>−1</sub> between them. Given the start index of the true colony peaks <em>s</em>, and the number of colonies, <em>n</em> in the row, we then calculate the likelihood <em>L</em>(<em>s</em>) for different starting positions, assuming normal distributions for distances <em>δ<sub>k</sub></em> between the peaks, and the correlations to expected colony shape <em>r<sub>i</sub></em> at the peak centers:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-5.gif"/></noscript></span></span>Here, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-6.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-7.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-8.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-9.gif"/></noscript></span></span>. We pick the most likely starting peak <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-10.gif"/></noscript></span></span>, and the corresponding true peaks <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-11.gif"/></noscript></span></span> that follow. The aforementioned procedure is repeated similarly for columns and true peaks obtained from rows and columns are used to define the initial colony locations for the next analysis step. We also calculate the characteristic distance between true peaks <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-12.gif"/></noscript></span></span>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/547/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Obtaining the grid of colonies. (A) Total pixel intensities Ti for rows (to the left of image) and columns (above image) of the thresholded input image are used to find the centers of the peaks that correspond to colony locations (green lines) that define the colony grid. Peaks from greater pixel intensities near plate edges and other sources of confounding signals (red lines) are discarded by fitting the grid. (B) Identifying the best-fit boundaries for a spot using local intensity profiles. Magenta and blue lines represent the x and y-coordinates of the boundaries, respectively. Gray dashed lines represent the local maxima from (A) that define the colony center." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1323124364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Obtaining the grid of colonies. (A) Total pixel intensities &lt;em&gt;T&lt;sub&gt;i&lt;/sub&gt;&lt;/em&gt; for rows (to the left of image) and columns (above image) of the thresholded input image are used to find the centers of the peaks that correspond to colony locations (green lines) that define the colony grid. Peaks from greater pixel intensities near plate edges and other sources of confounding signals (red lines) are discarded by fitting the grid. (B) Identifying the best-fit boundaries for a spot using local intensity profiles. Magenta and blue lines represent the x and y-coordinates of the boundaries, respectively. Gray dashed lines represent the local maxima from (A) that define the colony center.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/547/F2.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/547/F2.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/547/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/547/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464064" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Obtaining the grid of colonies. (A) Total pixel intensities <em>T<sub>i</sub></em> for rows (to the left of image) and columns (above image) of the thresholded input image are used to find the centers of the peaks that correspond to colony locations (green lines) that define the colony grid. Peaks from greater pixel intensities near plate edges and other sources of confounding signals (red lines) are discarded by fitting the grid. (B) Identifying the best-fit boundaries for a spot using local intensity profiles. Magenta and blue lines represent the x and y-coordinates of the boundaries, respectively. Gray dashed lines represent the local maxima from (A) that define the colony center.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Quantification of colony size</h3><p id="p-15">First, small speckles are optionally removed from the thresholded image by eroding with a 3 × 3 window, which eliminates single pixel noise. Larger windows would start eroding colonies and substantially reduce their measured size. We then find, for each colony, the minimum rectangle enclosing it in the square of width 1.5 × <em>δ</em> around the colony center. This width is chosen to allow for variations in the colony’s morphology outside its typical boundary. We then identify the local minimum of the colony pixel intensity profiles that are nearest to the colony centers, but at least <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-13.gif"/></noscript></span></span> away (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). If the pixel defining the center of the colony is a background pixel, we assume that the colony is not present, and a square with width <em>δ</em> around the center of the colony is used as the bounding rectangle. Finally, the size or area <em>A</em> of the colony is computed as the number of foreground pixels within the boundary. For each colony, we also report its circularity <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3/547/embed/mml-math-14.gif"/></noscript></span></span>, where <em>P</em> is the number of foreground pixels neighboring a background pixel.</p><p id="p-16">If the boundary contains foreground pixels, there is no clean separation between neighboring colonies, suggesting they are at least touching. In this case, the colony is flagged as potentially overlapping in the output. Similarly, colonies with low circularity are flagged to draw users’ attention to them, and allow automatic filtering. Entire plates are flagged if at least 10% of colonies are smaller than 0.1 of the median colony size, or have circularity below 0.6. Plate level flags indicate to the user that the output should be manually inspected.</p></div><div id="sec-6" class="subsection"><h3>Processing sparse images</h3><p id="p-17">Sparse plates with a majority of small or dead colonies, such as those seen in suppression screens, usually fail to process properly, as there is not enough information to establish the grid. To overcome this, the typical distance between colonies <em>δ</em> and colony center coordinates {<em>x<sub>k</sub></em>} can be precomputed from a reference image taken with same dimensions, sizes and settings as the sparse image. To account for small translational shifts, we calculate total pixel intensities <em>T<sub>i</sub></em> for both the reference and the sparse image, and offset the reference image to maximize the correlation between the <em>T<sub>i</sub></em> profiles. After obtaining the colony center coordinates and typical distances between them from the reference, the sparse image can be processed without fitting its grid independently.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-18">gitter was developed to overcome the two main drawbacks of existing tools—limited range of experimental setups that can be easily analyzed, and accuracy in quantifying the colony sizes in nontrivial cases. Thus, we evaluated the performance of gitter for robustness of handling plates from different laboratories, and correctly estimating the size of colonies.</p><div id="sec-8" class="subsection"><h3>Robustness</h3><p id="p-19">We tested gitter on images from a range of experimental setups and organisms, including 840 images from <em>Saccharomyces</em><strong> </strong><em>cerevisiae</em> (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Costanzo <em>et al.</em> 2010</a>), 82 from <em>Schizosaccharomyces pombe</em>, 87 from <em>Escherichia coli</em> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Butland <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Babu <em>et al.</em> 2011</a>) and nine from <em>Drosophila melanogaster</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hens <em>et al.</em> 2011</a>). All <em>D. melanogaster</em> and <em>E. coli</em> images were successfully gridded and quantified without any adjustments to the default parameters. Only 2 of 87 (2.3%) of <em>S. pombe</em> images and 5 of 840 (0.6%) <em>S. cerevisiae</em> images had an edge being mistaken for a row or column. When processing images in batch using a reference image, all tested images were correctly gridded.</p><p id="p-20">The tested images included many that pose difficulties to existing image analysis tools. The sizes of irregular colonies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) are difficult to quantify accurately by counting pixels inside a fixed circular template, but gitter overcomes this problem by fitting the bounds of each colony separately. gitter can also process noisy and inverted images where colonies are represented by darker, rather than brighter colors (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and C</a>). Images with uneven lighting or cloudiness caused by condensation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, D and E</a>) are problematic because brighter regions often are mistaken for foreground pixels. gitter’s background correction subtracts these effects out so they do not interfere with gridding and colony quantification. Images of plates with cracked agar caused by drying, dropping, etc. (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3F</a>) often are discarded because it is assumed that the image analysis software will not be able to successfully identify the colonies from the deformed grid. gitter overcomes small variations in the grid by refitting individual colony bounds.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/547/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sections of five example images and their thresholded gridded output. gitter can quantify images with (A) irregular colonies, (B) noise, (C) inverted colonies, (D) variable illumination, (E) cloudy background, and (F) cracked plates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1323124364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sections of five example images and their thresholded gridded output. gitter can quantify images with (A) irregular colonies, (B) noise, (C) inverted colonies, (D) variable illumination, (E) cloudy background, and (F) cracked plates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/547/F3.medium.gif" width="238" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/3/547/F3.medium.gif" width="238" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/547/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/547/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Sections of five example images and their thresholded gridded output. gitter can quantify images with (A) irregular colonies, (B) noise, (C) inverted colonies, (D) variable illumination, (E) cloudy background, and (F) cracked plates.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Accuracy</h3><p id="p-22">We compared the results of gitter to Colonyzer (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lawless <em>et al.</em> 2010</a>), HT colony grid analyzer (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Collins <em>et al.</em> 2006</a>), and Colony Imager (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Tong <em>et al.</em> 2002</a>). Three images showing irregular colonies and that were successfully processed by the four tools were considered for the comparison. Establishing a ground truth segmentation and boundary for colonies is difficult because thresholding is subjective. We thus first confirmed that the colony size estimates are concordant for a large fraction of colonies, and then manually inspected discordant ones.</p><p id="p-23">The colony sizes calculated by different tools are similar for well-behaved images and colonies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), but the estimates differ for some classes. Although Colonyzer also optimizes the bounds of each spot, it does so by fitting a fixed width square, resulting in underestimating sizes of spots larger than the square (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). In addition, in some rare cases, the bounds are misplaced for colonies on the edge of a plate (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). HT colony grid fits a grid of fixed-size circles to the image. Due to not adjusting the bounds for each colony, the sizes are sometimes underestimated, as a substantial portion of the colony remains outside of the fixed circle (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Colony Imager approximates the edges of each spot; however, we have found the defined bounds to be eroded (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>), resulting in the underestimation of colony sizes. This becomes an issue for very small colonies, as they get a quantified size of zero, and very large colonies, for which the erosion can account for a large part of the total area.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/547/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of gitter and other tools. (A) Comparison of colony sizes calculated by the four compared tools for three irregular images that pass each tool. Colony sizes are normalized to a range of 0&#x2212;1 by dividing by the maximum colony size. (B) Fitted colony bounds for the four compared tools for three colonies. The dashed red boundary designates the region of the colony that is used for quantification. Sizes of these colonies are marked on the scatter plots (A) with a red marker." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1323124364" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of gitter and other tools. (A) Comparison of colony sizes calculated by the four compared tools for three irregular images that pass each tool. Colony sizes are normalized to a range of 0&#x2212;1 by dividing by the maximum colony size. (B) Fitted colony bounds for the four compared tools for three colonies. The dashed red boundary designates the region of the colony that is used for quantification. Sizes of these colonies are marked on the scatter plots (A) with a red marker.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/547/F4.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/3/547/F4.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/547/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/547/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464085" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Comparison of gitter and other tools. (A) Comparison of colony sizes calculated by the four compared tools for three irregular images that pass each tool. Colony sizes are normalized to a range of 0−1 by dividing by the maximum colony size. (B) Fitted colony bounds for the four compared tools for three colonies. The dashed red boundary designates the region of the colony that is used for quantification. Sizes of these colonies are marked on the scatter plots (A) with a red marker.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other1" id="sec-10"><h2 class="">Implementation</h2><p id="p-25">gitter is implemented in an open-source R package available from <a href="http://cran.r-project.org/web/packages/gitter">http://cran.r-project.org/web/packages/gitter</a> and uses the EBImage (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pau <em>et al.</em> 2010</a>) package for image manipulation and the ggplot2 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Wickham 2009</a>) package for visualizations. A single function call quantifies the colony sizes from an image:</p><blockquote id="disp-quote-1" class="disp-quote"><p id="p-26">gitter(file, plate.format, remove.noise=F,</p><p id="p-27">inverse=F, autorotate=F,</p><p id="p-28">contrast=F, fast=1500, verbose=p)</p></blockquote><p id="p-29">where file is the path to the plate image, plate.format is a length-2 vector of the number of rows and columns of colonies in the plate or a known plate density (<em>e.g.</em>, 1536 or 384), remove.noise is a boolean determining whether noise and speckles should be removed before quantification, inverse determines whether the image should be inverted, autorotate determines whether the image should be autorotated, contrast is a contrast factor (if any), that should be applied to the image, fast is the width (in pixels) that the image should be resized to before processing, and verbose indicates whether the progress should be written out to the console. A set of images can be processed in batch using the batch function:</p><blockquote id="disp-quote-2" class="disp-quote"><p id="p-30">gitter.batch(files, file.reference, …)</p></blockquote><p id="p-31">where files is the directory containing images to be processed or a list of file paths, file.reference is the path to a reference image to process all images in file (optional), and … is any of the previously mentioned arguments. Additional documentation can be found online through the documentation on the package website.</p><p id="p-32">The result of calling gitter on a single file is a data frame containing the row and column of the colony on the plate, the quantified size, circularity, and any warning flags (potential overlap with neighbor and low circularity). This data frame is output to a tab-delimited file, and the thresholded image showing the boundaries of each colony (as in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>) is created. Users can also visualize the quantified colony sizes as a heatmap or bubble plot.</p></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-33">Software for analyzing images of genetic screens does not always work out of the box for a new experimental setup. Here, we present gitter, a tool that provides a robust and accurate way to quantify colony sizes via a simple interface. We apply methods to deal with heterogeneity in plate formats, as well as confounding factors such as rotation and illumination differences, colony morphology, and plate edges. As a result, the vast majority of the tested images can be processed without requiring any input from the user to tweak the parameters, and all tested images are successfully processed in batch provided one well-behaving reference.</p><p id="p-34">gitter optimizes the boundaries for each colony separately, resulting in more accurate colony size estimates. This is especially beneficial for screens with irregular or larger-than-average colonies, and we have shown that comparable tools often underestimate colony sizes in such cases. As the downstream analyses look for the effects of genetic perturbations by comparing colony sizes, these minute differences in estimates can have a substantial impact on the biological interpretation of the screen.</p><p id="p-35">The image analysis approach used by gitter required choosing several algorithms and variables <em>a priori</em>. For example, we used the Radon transform as a natural approach to quantify the variation in different directions of the image. We also tested the 2-dimensional Fourier transform but found extracting the optimal rotation angle from it to be more error-prone. The 3 × 3 kernel was used for noise removal as the smallest that eliminates one pixel speckles, since larger kernels start compromising colony size by erosion. We picked the background correction window size <em>w</em> to be larger than a colony diameter so that erosion and dilation remove the foreground objects. The choice of even-larger <em>w</em> results in further blurring of the signal and reduction of correction efficacy, whereas choosing a much smaller <em>w</em> results in subtracting off parts of colonies. The typical width <em>W</em> is estimated from the central 50% of intensity profiles to include only well-behaved portions of the image, and exclude edges. Finally, we used a window of 1.5 × <em>δ</em> around colony centers to search for the optimal boundary while allowing for variations in the colony’s morphology. Smaller windows produce boundaries that cut off parts of irregular colonies, and larger ones start encompassing neighboring colonies. All these parameters were manually optimized for best performance on a large set of test images.</p><p id="p-36">Automated tools for extracting understanding from biological data ought to be powerful and flexible on one hand, but intuitive and approachable for bench scientists on the other. Although to use gitter requires one to have a basic knowledge of R, we have also made it accessible via the online SGAtools normalization, scoring, and visualization suite (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Wagih <em>et al.</em> 2013</a>), so that images can be processed in a web application without a command line interface. gitter’s output is compatible with standard formats used in colony processing software, allowing integration into existing data analysis pipelines (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bean and Ideker 2012</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Wagih <em>et al.</em> 2013</a>). Future work includes adding support for spot dilutions, tetrad plates, densitometric images, and multiplate images; the latter can currently be processed with other existing tools (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Dittmar <em>et al.</em> 2010</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Lawless <em>et al.</em> 2010</a>). We hope that with gitter and SGAtools the transition from images of a screen to understanding the effects of the perturbation is straightforward for experimentalists and not limited by computation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We thank Mohan Babu, Conor Lawless, Peter Birchman, and Feuz Jean-Daniel for providing sample images and testing. We also thank Gary D. Bader, Jüri Reimand, Jürgen Janes, and Jared Simpson for comments on the manuscript. L.P. is a Canadian Institute for Advanced Research Global Scholar and supported by a Marie Curie IOF.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-38"><em>Communicating editor: A. M. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-11-17"><span class="received-label">Received </span>November 17, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-01-22"><span class="accepted-label">Accepted </span>January 22, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Wagih and Parts</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.1" data-doi="10.1093/bioinformatics/btg057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angulo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automatic analysis of DNA microarray images using mathematical morphology.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DAngulo%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D553%26rft.epage%253D562%26rft.atitle%253DAutomatic%2Banalysis%2Bof%2BDNA%2Bmicroarray%2Bimages%2Busing%2Bmathematical%2Bmorphology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg057%26rft_id%253Dinfo%253Apmid%252F12651712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS81LzU1MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.3.547.2"><div class="cit-metadata unstructured">Antoniol, G., M. Ceccarelli, and A. Petrosino, 2005 Microarray image addressing based on the radon transform, in <em>IEEE International Conference on Image Processing</em>, <em>2005. ICIP 2005</em>, volume 1. IEEE International Conference on Image Processing. New York City, NY, USA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.3" data-doi="10.1038/msb4100050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Construction of <em>Escherichia coli</em> k-12 in-frame, single-gene knockout mutants: the Keio collection.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2006.0008</span>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D2%26rft.spage%253D2006.0008%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100050%26rft_id%253Dinfo%253Apmid%252F16738554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb4100050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16738554&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.4" data-doi="10.1007/978-1-61779-276-2_7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagarinova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emili</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Array-based synthetic genetic screens to map bacterial pathways and functional networks in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">781</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBabu%26rft.auinit1%253DM.%26rft.volume%253D781%26rft.spage%253D99%26rft.epage%253D126%26rft.atitle%253DArray-based%2Bsynthetic%2Bgenetic%2Bscreens%2Bto%2Bmap%2Bbacterial%2Bpathways%2Band%2Bfunctional%2Bnetworks%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-276-2_7%26rft_id%253Dinfo%253Apmid%252F21877280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-276-2_7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21877280&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.5" data-doi="10.1186/1471-2105-11-49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bariamis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iakovidis</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maroulis</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">M3g: maximum margin microarray gridding.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBariamis%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D49%26rft.epage%253D49%26rft.atitle%253DM3G%253A%2Bmaximum%2Bmargin%2Bmicroarray%2Bgridding.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-49%26rft_id%253Dinfo%253Apmid%252F20100338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100338&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.6" data-doi="10.1186/gb-2012-13-12-r123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ideker</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential analysis of high-throughput quantitative genetic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D13%26rft.spage%253DR123%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-12-r123%26rft_id%253Dinfo%253Apmid%252F23268787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-12-r123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23268787&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.3.547.7"><div class="cit-metadata unstructured">Berlemont, S., B. Aaron, F. Cloppet, and J.-C. Olivo-Marin, 2007 Detection of linear structures in biological images, pp. 1279–1283 in <em>Conference Record of the Forty-First Asilomar Conference on Signals</em>, <em>Systems and Computers</em>, <em>2007</em>. <em>ACSSC 2007</em>. Asilomar Conference on Signals, Systems &amp; Computers. Asilomar Grounds, Pacific Grove, California, USA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.8" data-doi="10.1038/nmeth.1239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butland</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daz-Meja</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohdana</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phanse</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">eSGA: <em>E. coli</em> synthetic genetic array analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DButland%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253D789%26rft.epage%253D795%26rft.atitle%253DeSGA%253A%2BE.%2Bcoli%2Bsynthetic%2Bgenetic%2Barray%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1239%26rft_id%253Dinfo%253Apmid%252F18677321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677321&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258912700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.9" data-doi="10.1186/gb-2006-7-10-r100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cellprofiler: image analysis software for identifying and quantifying cell phenotypes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BE.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253DR100%26rft.epage%253DR100%26rft.atitle%253DCellProfiler%253A%2Bimage%2Banalysis%2Bsoftware%2Bfor%2Bidentifying%2Band%2Bquantifying%2Bcell%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-10-r100%26rft_id%253Dinfo%253Apmid%252F17076895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-10-r100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17076895&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.10" data-doi="10.1186/gb-2006-7-7-r63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCollins%26rft.auinit1%253DS.%2BR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253DR63%26rft.epage%253DR63%26rft.atitle%253DA%2Bstrategy%2Bfor%2Bextracting%2Band%2Banalyzing%2Blarge-scale%2Bquantitative%2Bepistatic%2Binteraction%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-7-r63%26rft_id%253Dinfo%253Apmid%252F16859555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-7-r63&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16859555&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.11" data-doi="10.1126/science.1180823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spear</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetic landscape of a cell.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCostanzo%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5964%26rft.spage%253D425%26rft.epage%253D431%26rft.atitle%253DThe%2BGenetic%2BLandscape%2Bof%2Ba%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180823%26rft_id%253Dinfo%253Apmid%252F20093466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2NC80MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.547.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deans</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Radon Transform and Some of Its Applications</span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.13" data-doi="10.1186/1471-2105-11-353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dittmar</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Screenmill: a freely available software suite for growth measurement, analysis and visualization of high-throughput screen data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDittmar%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D11%26rft.spage%253D353%26rft.epage%253D353%26rft.atitle%253DScreenMill%253A%2Ba%2Bfreely%2Bavailable%2Bsoftware%2Bsuite%2Bfor%2Bgrowth%2Bmeasurement%252C%2Banalysis%2Band%2Bvisualization%2Bof%2Bhigh-throughput%2Bscreen%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-353%26rft_id%253Dinfo%253Apmid%252F20584323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20584323&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.14" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.15" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.16" data-doi="10.1038/nmeth.1763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hens</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuz</span>  <span class="cit-name-given-names">J.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isakova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iagovitina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Automated protein-dna interaction screening of Drosophila regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DHens%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D1065%26rft.epage%253D1070%26rft.atitle%253DAutomated%2Bprotein-DNA%2Binteraction%2Bscreening%2Bof%2BDrosophila%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1763%26rft_id%253Dinfo%253Apmid%252F22037703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22037703&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297931700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.17" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.18" data-doi="10.1038/nature02026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falvo</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein localization in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHuh%26rft.auinit1%253DW.%2BK.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Blocalization%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02026%26rft_id%253Dinfo%253Apmid%252F14562095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562095&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.19" data-doi="10.1038/nbt.1628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.-U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysis of a genome-wide set of gene deletions in the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKim%26rft.auinit1%253DD.%2BU.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D617%26rft.epage%253D623%26rft.atitle%253DAnalysis%2Bof%2Ba%2Bgenome-wide%2Bset%2Bof%2Bgene%2Bdeletions%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1628%26rft_id%253Dinfo%253Apmid%252F20473289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20473289&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278820200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.20" data-doi="10.1186/1471-2105-11-287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Colonyzer: automated quantification of micro-organism growth characteristics on solid agar.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLawless%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D287%26rft.epage%253D287%26rft.atitle%253DColonyzer%253A%2Bautomated%2Bquantification%2Bof%2Bmicro-organism%2Bgrowth%2Bcharacteristics%2Bon%2Bsolid%2Bagar.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-287%26rft_id%253Dinfo%253Apmid%252F20509870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20509870&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.21" data-doi="10.1093/bioinformatics/btq046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sklyar</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutros</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ebimage—an r package for image processing with applications to cellular phenotypes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">981</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq046%26rft_id%253Dinfo%253Apmid%252F20338898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzk3OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy81NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.22" data-doi="10.1101/gad.1381406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeberlein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Extension of chronological life span in yeast by decreased tor pathway signaling.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPowers%26rft.auinit1%253DR.%2BW.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D184%26rft.atitle%253DExtension%2Bof%2Bchronological%2Blife%2Bspan%2Bin%2Byeast%2Bby%2Bdecreased%2BTOR%2Bpathway%2Bsignaling%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1381406%26rft_id%253Dinfo%253Apmid%252F16418483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvMi8xNzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.23" data-doi="10.1126/science.1162609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roguev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zofall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162609%26rft_id%253Dinfo%253Apmid%252F18818364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjIvNTkwMC80MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNTQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.24" data-doi="10.1186/1471-2105-12-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rueda</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rezaeian</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fully automatic gridding method for cDNA microarray images.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRueda%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DA%2Bfully%2Bautomatic%2Bgridding%2Bmethod%2Bfor%2BcDNA%2Bmicroarray%2Bimages.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-113%26rft_id%253Dinfo%253Apmid%252F21510903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21510903&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.25" data-doi="10.1186/1752-0509-1-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laws</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wardman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accurate, precise modeling of cell proliferation kinetics from time-lapse imaging and automated image analysis of agar yeast culture arrays.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DShah%26rft.auinit1%253DN.%2BA.%26rft.volume%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DAccurate%252C%2Bprecise%2Bmodeling%2Bof%2Bcell%2Bproliferation%2Bkinetics%2Bfrom%2Btime-lapse%2Bimaging%2Band%2Bautomated%2Bimage%2Banalysis%2Bof%2Bagar%2Byeast%2Bculture%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-1-3%26rft_id%253Dinfo%253Apmid%252F17408510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-1-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17408510&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F547.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.26" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Global mapping of the yeast genetic interaction network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>(<span class="cit-issue">5659</span>):<span class="cit-fpage">808</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D303%26rft.spage%253D808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.28" data-doi="10.1083/jcb.200909013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dyk</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizeacoumar</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrating high-throughput genetic interaction mapping and high-content screening to explore yeast spindle morphogenesis.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200909013%26rft_id%253Dinfo%253Apmid%252F20065090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE4OC8xLzY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.3.547.29" data-doi="10.1093/nar/gkt400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagih</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usaj</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanderSluis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sgatools: one-stop analysis and visualization of array-based genetic interaction screens.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>(Web Server issue)<strong>:</strong> <span class="cit-fpage">W591</span>–<span class="cit-lpage">W596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt400%26rft_id%253Dinfo%253Apmid%252F23677617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS9XMS9XNTkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzU0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-4.3.547.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite> <span class="cit-pub-date">2009</span> <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/547.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/539" title="Heteroscedastic Ridge Regression Approaches for Genome-Wide Prediction With a Focus on Computational Efficiency and Accurate Effect Estimation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/4/553" title="Biparental Resequencing Coupled With SNP Genotyping of a Segregating Population Offers Insights Into the Landscape of Recombination and Fixed Genomic Regions in Elite Soybean" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/547" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MIwxOyz8oaChndkSp_XDIImrT5s2opPnVAGS5osGk-w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/547" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/547" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/547" class="active" data-icon-position="" data-hide-link-title="0">gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459649" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tfCckfwbTDNlc7O8Sy-XeagRoXNnS7x-QV4nMjStTA8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/547.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/547" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-owtLoqdbWSNFAP5IfwzXue7l_g2UWo8C9OwRbH23ZvE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/547" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/547.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459649" id="node459649--2" data-pisa="ggg;4/3/547" data-pisa-master="ggg;g3.113.009431" data-apath="/ggg/4/3/547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Omar Wagih</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">547-552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009431 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459649--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omar Wagih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SD, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omar%2BWagih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wagih%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmar%2BWagih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/547" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/547&amp;text=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/547&amp;t=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/547&amp;title=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459649/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459649" id="node459649--3" data-pisa="ggg;4/3/547" data-pisa-master="ggg;g3.113.009431" data-apath="/ggg/4/3/547.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">gitter: A Robust and Accurate Method for Quantification of Colony Sizes From Plate Images</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Omar Wagih</span> and <span class="highwire-citation-author" data-delta="1">Leopold Parts</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">547-552;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.009431 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459649--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Omar Wagih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton CB10 1SD, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Omar%2BWagih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wagih%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOmar%2BWagih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leopold Parts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, M5S 3E1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leopold%2BParts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parts%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeopold%2BParts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wagih@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">wagih@ebi.ac.uk</a>
<a href="mailto:leopold.parts@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">leopold.parts@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=547&amp;rft.epage=552&amp;rft.atitle=gitter%3A%20A%20Robust%20and%20Accurate%20Method%20for%20Quantification%20of%20Colony%20Sizes%20From%20Plate%20Images&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wagih%2C+Omar&amp;rft.au=Parts%2C+Leopold"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459649/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459649/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459649/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459649/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459649/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459649/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459649/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459649/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459649/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459649/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459649/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459649/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/547" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Implementation</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/547.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/547.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/547.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/547.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
