<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/3/433" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/3/433.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/3/433.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/3/433.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/3/433.atom" />
<meta name="HW.pisa" content="ggg;4/3/433" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" />
<meta name="DC.Identifier" content="10.1534/g3.113.008722" />
<meta name="DC.Date" content="2014-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2014 Spivakov et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Oryzias latipes (medaka) has been established as a vertebrate genetic model for more than a century and recently has been rediscovered outside its native Japan. The power of new sequencing methods now makes it possible to reinvigorate medaka genetics, in particular by establishing a near-isogenic panel derived from a single wild population. Here we characterize the genomes of wild medaka catches obtained from a single Southern Japanese population in Kiyosu as a precursor for the establishment of a near-isogenic panel of wild lines. The population is free of significant detrimental population structure and has advantageous linkage disequilibrium properties suitable for the establishment of the proposed panel. Analysis of morphometric traits in five representative inbred strains suggests phenotypic mapping will be feasible in the panel. In addition, high-throughput genome sequencing of these medaka strains confirms their evolutionary relationships on lines of geographic separation and provides further evidence that there has been little significant interbreeding between the Southern and Northern medaka population since the Southern/Northern population split. The sequence data suggest that the Southern Japanese medaka existed as a larger older population that went through a relatively recent bottleneck approximately 10,000 years ago. In addition, we detect patterns of recent positive selection in the Southern population. These data indicate that the genetic structure of the Kiyosu medaka samples is suitable for the establishment of a vertebrate near-isogenic panel and therefore inbreeding of 200 lines based on this population has commenced. Progress of this project can be tracked at &lt;http://www.ebi.ac.uk/birney-srv/medaka-ref-panel&gt;." />
<meta name="DC.Contributor" content="Mikhail Spivakov" />
<meta name="DC.Contributor" content="Thomas O. Auer" />
<meta name="DC.Contributor" content="Ravindra Peravali" />
<meta name="DC.Contributor" content="Ian Dunham" />
<meta name="DC.Contributor" content="Dirk Dolle" />
<meta name="DC.Contributor" content="Asao Fujiyama" />
<meta name="DC.Contributor" content="Atsushi Toyoda" />
<meta name="DC.Contributor" content="Tomoyuki Aizu" />
<meta name="DC.Contributor" content="Yohei Minakuchi" />
<meta name="DC.Contributor" content="Felix Loosli" />
<meta name="DC.Contributor" content="Kiyoshi Naruse" />
<meta name="DC.Contributor" content="Ewan Birney" />
<meta name="DC.Contributor" content="Joachim Wittbrodt" />
<meta name="article:published_time" content="2014-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Oryzias latipes&lt;/i&gt; (medaka) has been established as a vertebrate genetic model for more than a century and recently has been rediscovered outside its native Japan. The power of new sequencing methods now makes it possible to reinvigorate medaka genetics, in particular by establishing a near-isogenic panel derived from a single wild population. Here we characterize the genomes of wild medaka catches obtained from a single Southern Japanese population in Kiyosu as a precursor for the establishment of a near-isogenic panel of wild lines. The population is free of significant detrimental population structure and has advantageous linkage disequilibrium properties suitable for the establishment of the proposed panel. Analysis of morphometric traits in five representative inbred strains suggests phenotypic mapping will be feasible in the panel. In addition, high-throughput genome sequencing of these medaka strains confirms their evolutionary relationships on lines of geographic separation and provides further evidence that there has been little significant interbreeding between the Southern and Northern medaka population since the Southern/Northern population split. The sequence data suggest that the Southern Japanese medaka existed as a larger older population that went through a relatively recent bottleneck approximately 10,000 years ago. In addition, we detect patterns of recent positive selection in the Southern population. These data indicate that the genetic structure of the Kiyosu medaka samples is suitable for the establishment of a vertebrate near-isogenic panel and therefore inbreeding of 200 lines based on this population has commenced. Progress of this project can be tracked at http://www.ebi.ac.uk/birney-srv/medaka-ref-panel.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/03/01" />
<meta name="citation_mjid" content="ggg;4/3/433" />
<meta name="citation_id" content="4/3/433" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/3/433" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/3/433.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/3/433.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/3/433.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008722" />
<meta name="citation_pmid" content="24408034" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="433" />
<meta name="citation_lastpage" content="445" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mikhail Spivakov" />
<meta name="citation_author_institution" content="European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK" />
<meta name="citation_author" content="Thomas O. Auer" />
<meta name="citation_author_institution" content="Centre for Organismal Studies, Heidelberg University, Germany" />
<meta name="citation_author" content="Ravindra Peravali" />
<meta name="citation_author_institution" content="Karslruhe Institute of Technology, Karlsruhe, Germany" />
<meta name="citation_author" content="Ian Dunham" />
<meta name="citation_author_institution" content="European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK" />
<meta name="citation_author" content="Dirk Dolle" />
<meta name="citation_author_institution" content="European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK" />
<meta name="citation_author_institution" content="Centre for Organismal Studies, Heidelberg University, Germany" />
<meta name="citation_author" content="Asao Fujiyama" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan" />
<meta name="citation_author" content="Atsushi Toyoda" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan" />
<meta name="citation_author" content="Tomoyuki Aizu" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan" />
<meta name="citation_author" content="Yohei Minakuchi" />
<meta name="citation_author_institution" content="Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan" />
<meta name="citation_author" content="Felix Loosli" />
<meta name="citation_author_institution" content="Karslruhe Institute of Technology, Karlsruhe, Germany" />
<meta name="citation_author_email" content="felix.loosli@kit.edu" />
<meta name="citation_author_email" content="naruse@nibb.ac.jp" />
<meta name="citation_author_email" content="birney@ebi.ac.uk" />
<meta name="citation_author_email" content="jochen.wittbrodt@cos.uni-heidelberg.de" />
<meta name="citation_author" content="Kiyoshi Naruse" />
<meta name="citation_author_institution" content="National Institute for Basic Biology, Laboratory of Bioresources, Okazaki, Japan" />
<meta name="citation_author_email" content="felix.loosli@kit.edu" />
<meta name="citation_author_email" content="naruse@nibb.ac.jp" />
<meta name="citation_author_email" content="birney@ebi.ac.uk" />
<meta name="citation_author_email" content="jochen.wittbrodt@cos.uni-heidelberg.de" />
<meta name="citation_author" content="Ewan Birney" />
<meta name="citation_author_institution" content="European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK" />
<meta name="citation_author_email" content="felix.loosli@kit.edu" />
<meta name="citation_author_email" content="naruse@nibb.ac.jp" />
<meta name="citation_author_email" content="birney@ebi.ac.uk" />
<meta name="citation_author_email" content="jochen.wittbrodt@cos.uni-heidelberg.de" />
<meta name="citation_author" content="Joachim Wittbrodt" />
<meta name="citation_author_institution" content="Centre for Organismal Studies, Heidelberg University, Germany" />
<meta name="citation_author_email" content="felix.loosli@kit.edu" />
<meta name="citation_author_email" content="naruse@nibb.ac.jp" />
<meta name="citation_author_email" content="birney@ebi.ac.uk" />
<meta name="citation_author_email" content="jochen.wittbrodt@cos.uni-heidelberg.de" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Aida;citation_title=ON THE INHERITANCE OF COLOR IN A FRESH-WATER FISH, APLOCHEILUS LATIPES TEMMICK AND SCHLEGEL, WITH SPECIAL REFERENCE TO SEX-LINKED INHERITANCE;citation_pages=554-573;citation_volume=6;citation_year=1921;citation_issue=6;citation_pmid=17245975" />
<meta name="citation_reference" content="citation_journal_title=Ichthyol. Explor. Freshwat.;citation_author=T. Asai;citation_author=H. Senou;citation_author=K. Hosoya;citation_title=Oryzias sakaizumii, a new ricefish from northern Japan (Teleostei: Adrianichthyidae).;citation_pages=289-299;citation_volume=22;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjalmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_title=Transplantation;citation_journal_abbrev=Transplantation;citation_author=D. W. Bailey;citation_title=Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes.;citation_pages=325-327;citation_volume=11;citation_year=1971;citation_issue=3;citation_pmid=5558564;citation_doi=10.1097/00007890-197103000-00013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. S. Bloom;citation_author=I. M. Ehrenreich;citation_author=W. T. Loo;citation_author=T. L. Lite;citation_author=L. Kruglyak;citation_title=Finding the sources of missing heritability in a yeast cross.;citation_pages=234-237;citation_volume=494;citation_year=2013;citation_pmid=23376951;citation_doi=10.1038/nature11867" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Boc;citation_author=D. A. Diallo;citation_author=V. Makarenkov;citation_title=T-REX: a web server for inferring, validating and visualizing phylogenetic trees and networks.;citation_pages=W573579;citation_volume=40;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Chen;citation_author=B. Li;citation_author=Z. Zeng;citation_author=S. Sanna;citation_author=C. Sidore;citation_title=Genotype calling and haplotyping in parent-offspring trios.;citation_volume=23;citation_year=2013;citation_pmid=23064751;citation_doi=10.1101/gr.142455.112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. Drmanac;citation_author=A. B. Sparks;citation_author=M. J. Callow;citation_author=A. L. Halpern;citation_author=N. L. Burns;citation_title=Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays.;citation_volume=327;citation_year=2010;citation_pmid=19892942;citation_doi=10.1126/science.1181498" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Y. Durand;citation_author=N. Patterson;citation_author=D. Reich;citation_author=M. Slatkin;citation_title=Testing for ancient admixture between closely related populations.;citation_volume=28;citation_year=2011;citation_pmid=21325092;citation_doi=10.1093/molbev/msr048" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. E. Bernstein;citation_author=E. Birney;citation_author=I. Dunham;citation_author=E. D. Green;citation_author=C. Gunter;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Flint;citation_author=T. F. Mackay;citation_title=Genetic architecture of quantitative traits in mice, flies, and humans;citation_pages=723-733;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411597;citation_doi=10.1101/gr.086660.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Fu;citation_author=T. J. Wen;citation_author=Y. I. Ronin;citation_author=H. D. Chen;citation_author=L. Guo;citation_title=Genetic dissection of intermated recombinant inbred lines using a new genetic map of maize.;citation_volume=174;citation_year=2006;citation_pmid=16951074;citation_doi=10.1534/genetics.106.060376" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=M. Furutani-Seiki;citation_author=T. Sasado;citation_author=C. Morinaga;citation_author=H. Suwa;citation_author=K. Niwa;citation_title=A systematic genome-wide screen for mutations affecting organogenesis in Medaka, Oryzias latipes.;citation_pages=647-658;citation_volume=121;citation_year=2004;citation_issue=7-8;citation_pmid=15210174;citation_doi=10.1016/j.mod.2004.04.016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. B. Gabriel;citation_author=S. F. Schaffner;citation_author=H. Nguyen;citation_author=J. M. Moore;citation_author=J. Roy;citation_title=The structure of haplotype blocks in the human genome.;citation_volume=296;citation_year=2002;citation_pmid=12029063;citation_doi=10.1126/science.1069424" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=X. Gan;citation_author=O. Stegle;citation_author=J. Behr;citation_author=J. G. Steffen;citation_author=P. Drewe;citation_title=Multiple reference genomes and transcriptomes for Arabidopsis thaliana.;citation_pages=419-423;citation_volume=477;citation_year=2011;citation_issue=7365;citation_pmid=21874022;citation_doi=10.1038/nature10414" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Gautier;citation_author=R. Vitalis;citation_title=rehh: an R package to detect footprints of selection in genome-wide SNP data from haplotype structure.;citation_volume=28;citation_year=2012;citation_pmid=22402612;citation_doi=10.1093/bioinformatics/bts115" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Green;citation_author=J. Krause;citation_author=A. W. Briggs;citation_author=T. Maricic;citation_author=U. Stenzel;citation_title=A Draft Sequence of the Neandertal Genome;citation_pages=710-722;citation_volume=328;citation_year=2010;citation_issue=5979;citation_pmid=20448178;citation_doi=10.1126/science.1188021;citation_georef=2010072445" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. A. Groenen;citation_author=A. L. Archibald;citation_author=H. Uenishi;citation_author=C. K. Tuggle;citation_author=Y. Takeuchi;citation_title=Analyses of pig genomes provide insight into porcine demography and evolution.;citation_pages=393-398;citation_volume=491;citation_year=2012;citation_issue=7424;citation_pmid=23151582;citation_doi=10.1038/nature11622" />
<meta name="citation_reference" content="citation_journal_title=Zool. Mag.;citation_author=Y. Hyodo-Taguchi;citation_title=Establishment of inbred strains of the teleost Oryzias latipes.;citation_pages=283-301;citation_volume=89;citation_year=1980" />
<meta name="citation_reference" content="citation_title=Biology of the Medaka;citation_title=Inbred strains of the Medaka and their characteristics, pp. 129-142 in;citation_author=Y. Hyodo-Taguchi;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Mammal Study;citation_author=N. Ishiguro;citation_author=Y. Inoshima;citation_author=M. Sasaki;citation_author=A. Matsui;citation_author=H. Hongo;citation_title=mtDNA variation and human-mediated introgression of indigenous Sus populations on several Indonesian islands.;citation_pages=1-10;citation_volume=37;citation_year=2012;citation_doi=10.3106/041.037.0101" />
<meta name="citation_reference" content="citation_journal_title=Brain Behavior and Evolution;citation_journal_abbrev=Brain Behavior and Evolution;citation_author=Y. Ishikawa;citation_author=M. Yoshimoto;citation_author=N. Yamamoto;citation_author=H. Ito;citation_title=Different brain morphologies from different genotypes in a single teleost species, the medaka (Oryzias latipes);citation_pages=2-9;citation_volume=53;citation_year=1999;citation_issue=1;citation_pmid=9858800;citation_doi=10.1159/000006577" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. C. Jones;citation_author=M. G. Grabherr;citation_author=Y. F. Chan;citation_author=P. Russell;citation_author=E. Mauceli;citation_title=The genomic basis of adaptive evolution in threespine sticklebacks.;citation_pages=55-61;citation_volume=484;citation_year=2012;citation_issue=7392;citation_pmid=22481358;citation_doi=10.1038/nature10944" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kasahara;citation_author=K. Naruse;citation_author=S. Sasaki;citation_author=Y. Nakatani;citation_author=W. Qu;citation_title=The medaka draft genome and insights into vertebrate genome evolution.;citation_pages=714-719;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554307;citation_doi=10.1038/nature05846" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=T. Katsumura;citation_author=S. Oda;citation_author=S. Mano;citation_author=N. Suguro;citation_author=K. Watanabe;citation_title=Genetic differentiation among local populations of medaka fish (Oryzias latipes) evaluated through grid- and deme-based sampling.;citation_pages=170-177;citation_volume=443;citation_year=2009;citation_pmid=19397956;citation_doi=10.1016/j.gene.2009.04.017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Kimura;citation_author=A. Shimada;citation_author=N. Sakai;citation_author=H. Mitani;citation_author=K. Naruse;citation_title=Genetic Analysis of Craniofacial Traits in the Medaka;citation_pages=2379-2388;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073435;citation_doi=10.1534/genetics.106.068460" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila Synthetic Population Resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Li;citation_author=R. Durbin;citation_title=Inference of human population history from individual whole-genome sequences.;citation_pages=493-496;citation_volume=475;citation_year=2011;citation_issue=7357;citation_pmid=21753753;citation_doi=10.1038/nature10231" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Mechanisms of development;citation_journal_abbrev=Mech Dev;citation_author=F. Loosli;citation_author=R. W. Koster;citation_author=M. Carl;citation_author=R. Kuhnlein;citation_author=T. Henrich;citation_title=A genetic screen for mutations affecting embryonic development in medaka fish (Oryzias latipes).;citation_pages=133-139;citation_volume=97;citation_year=2000;citation_issue=1-2;citation_pmid=11025214;citation_doi=10.1016/S0925-4773(00)00406-8" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. G. MacArthur;citation_author=S. Balasubramanian;citation_author=A. Frankish;citation_author=N. Huang;citation_author=J. Morris;citation_title=A Systematic Survey of Loss-of-Function Variants in Human Protein-Coding Genes;citation_pages=823-828;citation_volume=335;citation_year=2012;citation_issue=6070;citation_pmid=22344438;citation_doi=10.1126/science.1215040" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Meyer;citation_author=M. Kircher;citation_author=M. T. Gansauge;citation_author=H. Li;citation_author=F. Racimo;citation_title=A high-coverage genome sequence from an archaic Denisovan individual.;citation_volume=338;citation_year=2012;citation_pmid=22936568;citation_doi=10.1126/science.1224344" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. Mongin;citation_author=T. O. Auer;citation_author=F. Bourrat;citation_author=F. Gruhl;citation_author=K. Dewar;citation_title=Combining computational prediction of cis-regulatory elements with a new enhancer assay to efficiently label neuronal structures in the medaka fish.;citation_pages=e19747-e19747;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21637758;citation_doi=10.1371/journal.pone.0019747" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=J. L. Peirce;citation_author=L. Lu;citation_author=J. Gu;citation_author=L. M. Silver;citation_author=R. W. Williams;citation_title=A new set of BXD recombinant inbred lines from advanced intercross populations in mice.;citation_pages=7-7;citation_volume=5;citation_year=2004;citation_pmid=15117419;citation_doi=10.1186/1471-2156-5-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of hypertension;citation_journal_abbrev=J Hypertens;citation_author=M. Pravenec;citation_author=P. Klir;citation_author=V. Kren;citation_author=J. Zicha;citation_author=J. Kunes;citation_title=An analysis of spontaneous hypertension in spontaneously hypertensive rats by means of new recombinant inbred strains.;citation_pages=217-221;citation_volume=7;citation_year=1989;citation_issue=3;citation_pmid=2708818;citation_doi=10.1097/00004872-198903000-00008" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=M. Rembold;citation_author=K. Lahiri;citation_author=N. S. Foulkes;citation_author=J. Wittbrodt;citation_title=Transgenesis in fish: efficient selection of transgenic fish by co-injection with a fluorescent reporter construct.;citation_pages=1133-1139;citation_volume=1;citation_year=2006;citation_issue=3;citation_pmid=17406394;citation_doi=10.1038/nprot.2006.165" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=J. D. Retief;citation_title=Phylogenetic analysis using PHYLIP.;citation_pages=243-258;citation_volume=132;citation_year=2000;citation_pmid=10547839" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_reference" content="citation_journal_title=Experimental animals / Japanese Association for Laboratory Animal Science;citation_journal_abbrev=Exp Anim;citation_author=T. Sasado;citation_author=M. Tanaka;citation_author=K. Kobayashi;citation_author=T. Sato;citation_author=M. Sakaizumi;citation_title=The National BioResource Project Medaka (NBRP Medaka): an integrated bioresource for biological and biomedical sciences.;citation_pages=13-23;citation_volume=59;citation_year=2010;citation_issue=1;citation_pmid=20224166;citation_doi=10.1538/expanim.59.13" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Sasaki;citation_author=C. C. Mello;citation_author=A. Shimada;citation_author=Y. Nakatani;citation_author=S. Hashimoto;citation_title=Chromatin-associated periodicity in genetic variation downstream of transcriptional start sites.;citation_volume=323;citation_year=2009;citation_pmid=19074313;citation_doi=10.1126/science.1163183" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=D. H. Setiamarga;citation_author=M. Miya;citation_author=Y. Yamanoue;citation_author=Y. Azuma;citation_author=J. G. Inoue;citation_title=Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.;citation_volume=5;citation_year=2009;citation_pmid=19586967;citation_doi=10.1098/rsbl.2009.0419;citation_georef=2013074188" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=N. Sharopova;citation_author=M. D. McMullen;citation_author=L. Schultz;citation_author=S. Schroeder;citation_author=H. Sanchez-Villeda;citation_title=Development and mapping of SSR markers for maize.;citation_pages=463-481;citation_volume=48;citation_year=2002;citation_issue=5-6;citation_pmid=12004892;citation_doi=10.1023/A:1014868625533" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=A. Struyf;citation_author=M. Hubert;citation_author=P. J. Rousseeuw;citation_title=Integrating robust clustering techniques in S-PLUS.;citation_pages=17-37;citation_volume=26;citation_year=1997;citation_doi=10.1016/S0167-9473(97)00020-0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=H. Takeda;citation_author=A. Shimada;citation_title=The art of medaka genetics and genomics: what makes them so unique?;citation_pages=217-241;citation_volume=44;citation_year=2010;citation_pmid=20731603;citation_doi=10.1146/annurev-genet-051710-151001" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=Y. Takehana;citation_author=N. Nagai;citation_author=M. Matsuda;citation_author=K. Tsuchiya;citation_author=M. Sakaizumi;citation_title=Geographic variation and diversity of the cytochrome b gene in Japanese wild populations of medaka, Oryzias latipes.;citation_pages=1279-1291;citation_volume=20;citation_year=2003;citation_issue=10;citation_pmid=14569151;citation_doi=10.2108/zsj.20.1279" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=Y. Takehana;citation_author=S. R. Jeon;citation_author=M. Sakaizumi;citation_title=Genetic structure of korean wild populations of the Medaka Oryzias latipes inferred from allozymic variation.;citation_pages=977-988;citation_volume=21;citation_year=2004;citation_issue=9;citation_pmid=15459457;citation_doi=10.2108/zsj.21.977" />
<meta name="citation_reference" content="citation_journal_title=Rep. Jap. Breed. Soc.;citation_author=K. Toyama;citation_title=On some Mendelian characters.;citation_pages=1-9;citation_volume=1;citation_year=1916" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=H. Nijveen;citation_author=X. Rao;citation_author=T. Bisseling;citation_author=R. Geurts;citation_title=Primer3Plus, an enhanced web interface to Primer3.;citation_pages=W7174-74;citation_volume=35;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=A map of recent positive selection in the human genome.;citation_pages=e72-e72;citation_volume=4;citation_year=2006;citation_issue=3;citation_pmid=16494531;citation_doi=10.1371/journal.pbio.0040072" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Wittbrodt;citation_author=A. Shima;citation_author=M. Schartl;citation_title=Medaka--a model organism from the far East.;citation_pages=53-64;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823791;citation_doi=10.1038/nrg704" />
<meta name="citation_reference" content="citation_journal_title=J Exp Zool;citation_author=T-O. Yamamoto;citation_title=Artificially induced sex-reversal in genotypic males of the medaka (Oryzias Latipes).;citation_pages=571-594;citation_volume=123;citation_year=1953;citation_doi=10.1002/jez.1401230309" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Zhao;citation_author=P. Zheng;citation_author=S. Dong;citation_author=X. Zhan;citation_author=Q. Wu;citation_title=Whole-genome sequencing of giant pandas provides insights into demographic history and local adaptation.;citation_pages=67-71;citation_volume=45;citation_year=2013;citation_pmid=23242367" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
<meta name="twitter:description" content="Oryzias latipes (medaka) has been established as a vertebrate genetic model for more than a century and recently has been rediscovered outside its native Japan. The power of new sequencing methods now makes it possible to reinvigorate medaka genetics, in particular by establishing a near-isogenic panel derived from a single wild population. Here we characterize the genomes of wild medaka catches obtained from a single Southern Japanese population in Kiyosu as a precursor for the establishment of a near-isogenic panel of wild lines. The population is free of significant detrimental population structure and has advantageous linkage disequilibrium properties suitable for the establishment of the proposed panel. Analysis of morphometric traits in five representative inbred strains suggests phenotypic mapping will be feasible in the panel. In addition, high-throughput genome sequencing of these medaka strains confirms their evolutionary relationships on lines of geographic separation and provides further evidence that there has been little significant interbreeding between the Southern and Northern medaka population since the Southern/Northern population split. The sequence data suggest that the Southern Japanese medaka existed as a larger older population that went through a relatively recent bottleneck approximately 10,000 years ago. In addition, we detect patterns of recent positive selection in the Southern population. These data indicate that the genetic structure of the Kiyosu medaka samples is suitable for the establishment of a vertebrate near-isogenic panel and therefore inbreeding of 200 lines based on this population has commenced. Progress of this project can be tracked at &lt;http://www.ebi.ac.uk/birney-srv/medaka-ref-panel&gt;." />
<meta name="og-title" property="og:title" content="Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/3/433" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Oryzias latipes (medaka) has been established as a vertebrate genetic model for more than a century and recently has been rediscovered outside its native Japan. The power of new sequencing methods now makes it possible to reinvigorate medaka genetics, in particular by establishing a near-isogenic panel derived from a single wild population. Here we characterize the genomes of wild medaka catches obtained from a single Southern Japanese population in Kiyosu as a precursor for the establishment of a near-isogenic panel of wild lines. The population is free of significant detrimental population structure and has advantageous linkage disequilibrium properties suitable for the establishment of the proposed panel. Analysis of morphometric traits in five representative inbred strains suggests phenotypic mapping will be feasible in the panel. In addition, high-throughput genome sequencing of these medaka strains confirms their evolutionary relationships on lines of geographic separation and provides further evidence that there has been little significant interbreeding between the Southern and Northern medaka population since the Southern/Northern population split. The sequence data suggest that the Southern Japanese medaka existed as a larger older population that went through a relatively recent bottleneck approximately 10,000 years ago. In addition, we detect patterns of recent positive selection in the Southern population. These data indicate that the genetic structure of the Kiyosu medaka samples is suitable for the establishment of a vertebrate near-isogenic panel and therefore inbreeding of 200 lines based on this population has commenced. Progress of this project can be tracked at &lt;http://www.ebi.ac.uk/birney-srv/medaka-ref-panel&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/3.cover-source.jpg" />
    <title>Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"J8BGgyBu5SPb4BAy5mnEXPkqbeueWrXA7dB2k47ZpZQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459644","apath":"\/ggg\/4\/3\/433.atom","pisa":"ggg;4\/3\/433","ac":{"\/ggg\/4\/3\/433.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/3\/433.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/3\/433"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/3\/433"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/3\/433":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qyWlwZZEsU7J8qg8V6BoqbQ2Fw3+SBNAi3kKcxPTRMoSIl\\\\\\\/+U8usLU4R7pjX9RsAJpZJKnQvBoYdBnBYPhQU6QYeL+XY79syefFU83QB+YB5ahZOdWUCG2y27eKN\\\\\\\/EAulKFF9EbDnSyWO9fiteknpHYKpfu+It0lAtS5SdroJIxteAJbf8x1vLDG8lE9fohMVp5vahTDMvTsTZAKB8MiHTk9T+us6F5zSSic9DZu0okqLLwe4D0Gg71\\\\\\\/mU3njFnWaOXMUrBFPbkbHZvJ31IWQQ0SkArk27NBsR78vR3nFlA7y3qdyhQpSKHxGtbkyc77zhpNZHpg\\\\\\\/f9uX1ox6\\\\\\\/SerSvUhquXMV7Q7cK\\\\\\\/RdHQuYVcaZ6eIuqPSTJbdvoMxA5+BLSXDp+k1AfvmwYQU+\\\\\\\/1KrjS+oLv16U79Dd6g\\\\\\\/OyN5n7N68PZM0EoWWOtXGQHTy7spEJ6BVCTsumDtFWQLCcE\\\\\\\/RIKl2KQStFAByIxmJ8+kCWaDlejiIugsaHLf\\\\\\\/5t3qTZ3IH9dwrP1TXwr1UUA\\\\\\\/3PqlLqvFGneJn6PCYvSwcjHJQ1NSXo5A6DfWkZ1mHJ560\\\\\\\/5LG22Rgw\\\\\\\/Q5BumTmN0JLr+QQZt4ct5ZjObxSNpFTSQnNpNJUSdNlaf5A3hs2uu77T3jvQE3zHVt19dkH++erwq79H8ZJgTWX46BEmP0GmeNfBRdkhPCLydvPr6gLl59k0FcEclcSahJpcIMZ8FgC\\\\\\\/Xde5ru35iVhnoFa7GHypfElmBk0SW5DdFSJXX4A1FHNFpfWF3eXu4AOhUCpIFtZ9i8gL\\\\\\\/UEx8SQ7Wn9p0UDRYc9T9PANru1leIm6V1WKjXyPJmGYR+EXQcmNMf+QrXWHg8Q6\\\\\\\/69bV47cwSY9FmR79UYKO8lwyXe6HcQxfDGxiOrl8HvDrahxH3RxZahTxelGd7mN7aDOn0DCcqx6iexmHhAmS\\\\\\\/1csr3GK+a3ftpA7ASC7\\\\\\\/cdobizIGNYtvWxMzaYa37PsMjK9WCAznMehaZYR9Rhe2KelY9WLuSaUdOjF8Ig\\\\\\\/L7JRDfdQSrGu1LzIS\\\\\\\/OTM4UAWYNsXNQ\\\\\\\/6lHT1jL6rj+r66UP84\\\\\\\/bnnECVibfaDpBF6wTXT6Z4NeWFvSa0sPSmi\\\\\\\/CfLCAfsGJAB9S4\\\\\\\/Nd8+awi\\\\\\\/91uvIZWau+Y3wDok7Vk4Xfexj+LY7UPfJRVDfqYOuqZqcpKy3FuE6vx9Y901QLDNDSQfKV\\\\\\\/QCvChEjqXblG9MAN6EJ4wiRn38uLmLKFu1WItokS1+pKo9pm\\\\\\\/50cD94UHbq\\\\\\\/7qrqaWA4X0OC\\\u0022,\\\u0022iv\\\u0022:\\\u0022bCOe1bXI3kZSYY6M27pddg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bc5c2677a99a56ba9a7bba0efe0005013a41c6dfe0a0bbdd1076816548250d03\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zCgHy8OZj57l61nMZeTsF6BAr6P+KO\\\\\\\/eJVyCRlVMoVf9xCQXbxcv4LFBCLQBe3wOg8rma9PCZIM\\\\\\\/sEag94C5agQpSdncZojkgwz+pdPP52ktxOCX+C4ATXwNZ4VP1SkNf6XCMCw\\\\\\\/4fNAbsQ9zjamowkJVHgK92cVMJ+nKTPzVU+uTlhM18U\\\\\\\/hDqgR+qe988v0FXbtNfJfCdwnDF\\\\\\\/mibHlzWPkaPJ4H6ysofQjRbIpOPOUxndTWKubxxjLTDYJloCOCM\\\\\\\/enzAN1NzcrvUor7ucN+m497xnjdSqvpcOgNtxwxqFCFeBkQX9g3LcoftLrhT2+mi6oKXNrYG0rgdK1o0bMv5Tr3jEi1ne6tsKBZxp83bOiar2A6b9\\\\\\\/z6hZCbOXvA+pxvCGyMwYfa457GomOwG4RT\\\\\\\/nMYj1IZZPV6aausm69N86PTwGRdp5w0tcApFQNWr\\\\\\\/BtGyGEd\\\\\\\/8jT9csqBOdKy2VS6mRVnEAaSWiIImiM9bR76lTjrbmu3FwBIxI7ivnXMhqeCPyUn2IT4l\\\\\\\/DxqL7yW0ei8PIgo3n4OXLTYn9rNDzt+OjrkoNE5uUUQvf50WftrMvs13\\\\\\\/IrUUHlYr73fncMm66kMtcCuXBarf3vX56cfeLl2Cshw7qn\\\\\\\/HbGvj+LiO5HYSRh5ctDu+vvl0KOTleeai+OXUYR+E8DFhFKIWzfv8daSvxgFL2j1\\\\\\\/iHpXf2I045Bua5J+OF12mcY7\\\\\\\/RM1eHe7MEvtkptunY8ks\\\\\\\/lUIpcBt5YfzL4nmc4FPoHyWSPiTLScqGgJgqvSEFbeEHTT3sZrlpq2LWc60+mMzGs2iWHBai1atEU2QKUIcjYqsPiDnnxPZm1PvZGcliMLoM0LkVfr5AKw14\\\\\\\/z4dZXmM9hAy0\\\\\\\/pAJkqgIzCXpshwHXsfNj9LxyOlUncyHSsSQZfmQYuAw6noD4MpXakJE4jXArk6HrFDtH6147eNLvoHimBsJQWvWu1x3Ul9g6u1xZPMnlDzd66nGlhkPZb0m4nf9tHXBX7\\\\\\\/qmmZ3fpT3XrMqH3pkSXmHS6L+B8vywIcawPIxRnZVPspUYUIFFCcstj8uURmdeLu3bko3dsqrZkyHGfn+t6srgrvLwVP+xI1sqx\\\\\\\/ffmQTuB49HfEVVQ5MNF9y07olbw1MoyBjx63C3\\\\\\\/MWVWEPa5MU5VWdATVchvLDsHS1XGayWhQXgtuBgnFvLhf4QRBGQ2MIP1Fv7QUBwCtzUm69DIstljqCBBtOL1Nw1kwBB681oF7ZpTwjk3TEYBMHLpihKkMHQ\\\\\\\/KvQ24ObiAIoc9DudTa\\\\\\\/uugDNrcNEf8hrUkcUDgQM4cIz0aScyb5A2\\\\\\\/WNhSDEmOkuwRY5vN2wIt6b\\\\\\\/kBQZYsDMiIevvAfBUbQGGzvrsV2MljOR+MlM+MXlR5IDI3302Y6VewdQdiN3fZPjpzTDeJqbE1cbHO6vBCNTBPWmDHQb85geQXzWRMWz3YXxDP+Si8QB5AgSmTSeNKJdmXmcfufGmLLZueL4n6u14TcE14I\\\\\\\/63iYf2CjDIXrZUK4lsUNtSVG6KqZHmDspeIosbZ4uD2x8uLG5dhboMnFmADsROYBKxIyvlNDsp91agTZemPFHyr5RUbrq\\\\\\\/9Lhz1LEJu8f9tfH9C9jYWzhWqfufZVTBZUP0oHXJrqOYNMTnmxClDUFMHtzqxZVCxikvCwKhsMf78Hv04ks7k05ZvrLvCHBKATm0wjcGTMflVHPgF7g2pysAozqcgc9Vs8UT3lN\\\\\\\/B3sopzAHO+lCKVuIpg5LFKI6fDdAefkeiM2yCUi7sSnG0Ug\\\\\\\/syg4nTIr\\\\\\\/rxO4KjFbneu\\\\\\\/wciJpzhQfeknIiugV2wC6oA+m91qcxWbJC65bV\\\\\\\/urvxWC5hwbhXCEHRdj0lmlxf4iicSSC9MgMvZAFpJuarPuVV8JPxJ5E5kSuRZ7ER9mSW2M9bzd6eTGcotWmlk8vja\\\\\\\/jpOOdANl7GXvfm8DPdS+UQaU5\\\\\\\/VOcLJDsoecqKHbyNKY8pRMT7LTSQPQbKloQ70hzulbEHRKZ3D\\\\\\\/VdywSUgJrRw7pl1p4AKmPjkLtaAcuwVLE9lCPR0cmndsFlUmUfGBD7g8wpQlzCGvKs3lwXd2v+Z3LMHv\\\\\\\/jtmKOzpevWpfjbo\\\\\\\/hc3iKkWjJ3X182HxXMkDZY1YuSBubzAruAN6EG1I17qFO+oWt1c6S1Fjc\\\\\\\/deiHZaK6rVWacQ3iTQDurJI4eaEIl\\\\\\\/sR\\\\\\\/yn8qV6bHtuEjJuvl00xhE9w5Oe756aptWr03bDJXd5so2Tx\\\\\\\/kHkb5HkO3F3vg5T9hHrweVrt+WiX38sY3K9taRNcNpOMinOrMTlP4HMe4Nou6BU13jG4BtaMsdQBm0Tr67Azq6NH5La8ppoNrGipyR7Ufu2nXZXCJji6w4XxEpsGKoJncbEsQ1F1R2zD\\\\\\\/Vwa3zM5uAQLYhac1Zj7fX4vPgByIn9vxYIMBsVnuTlws3ABuoyi\\\\\\\/o5yqwcewJQ7vqzI3EyzgEZ+lMIMUez8wbMVRs1VdMw8c5NjSwIX\\\\\\\/OXdwW\\\\\\\/HF5S8o6EttvHZEtmDeBAyBpEonxwSMT8qJCJdAC5pdwbgVxDmHBOgH5SByp+8zxjFkIMFRIBEHqHHnuzcYn1I56LiSiJ+79lGfr5xiEil\\\\\\\/ORSuVdI9crlIhBeid7NrwvwTerN7EgAJOIVVY+zf3JvlJ9I1ACAPlIvitnsu8kPQWDTfVCN7mQSrzO6hX+sTO6R1AjV2xuThYPQMIIXDYbNoFIRXAfIS1WvQ26aB12Q9hd4rOANk64SGb\\\\\\\/hGYUi0vrr2xX4tBLaJRTaJaKhpjY35CeXGiT1SBQdEBDIs1gSjFQQHljl5L8tXGHT4+2KBb5HDGmHOziWlWs8FrxrgeaPUwLoq7YV1MQ4UUaiQR8iSxzqi70QwcJtzbhXsoVyf6gNmh4AuREkzvNCoE\\\\\\\/FgEyajHG2P2p+o1B6QjHA1iNhBTaFNkADckTH08JynpuJHEWDGia1xundtTtnY\\\\\\\/Nzhr8SAcmxr2m9WMQ2s\\\\\\\/dKPjSkwz947+HfRqeeXcNYuttrBSXiWwjjzIm0qWEoXLH2XDR\\\\\\\/l1wq2rlfWiX4PVdMnbFNb8tvYidaEcfU\\\\\\\/kJrcJIx1Y7AfFZ6PX8\\\\\\\/folLZ5zNL3pBgAaE58ngoDu7Eq4rE5HxICfVwYTxRlh\\\\\\\/iDxlLGedwqYGOgeZaugL187BoIFliXmafZotDo6dI\\\\\\\/GTq7FTyNeomJlkXD\\\\\\\/kGAvuqSjNnuPbLip8av\\\\\\\/d4tDWKwhcBZKJJ4TbjyG7Fy1\\\\\\\/C7+U8jcc7ExXcfTpdDNv5NbamZ+BEusK8qZ6AKhEawrIFD\\\\\\\/kWN+fdrq\\\\\\\/8XN\\\\\\\/i59DAdLDlDQKKFAjc2BoJwN4VH5wcFIFeidiQXURqF7YD4jiNNbDP\\\\\\\/e5yQ4pPPHa+p7dM\\\\\\\/eNTHX+B\\\\\\\/nW3neIZGhfnIPBvCC0Gh3TEkKHwxv0gA9DzTuFyoj0LZ+bLh6eotZxutuLVqWBNZntdi3xIqysXbtconwjOrMfugQ3sj1dS8AzzRHa7qOWBmc3n06jm54KKI1kzy3WIN1oZOblRdqe8Lr14msdUmJLpX8Mitgcaz16TtXr16M4vrTN0xIP3o6xfOdCSHNZZFdwZ3CeKNOyqcXrXbK2WhCUzPQPb63M58N5wag0tR7u3vReFSN\\\\\\\/DlzNf4NokeK2r6U759CZqZL6Qx11IkPAmQK5aATn7WCkqaPikGfjOtYUsAxBIA5TMxZ8ib87CRRQ7\\\\\\\/HI4FL5kwDXnaAhpfGdCOJHEyfiFdi2Jya8qUyZFhfLvYKF5EBCImh4ehLD\\\\\\\/ik+IfsspoM9om+RXaMkqck9qvEv4anf1wn3yv0TwT8TmSnPN4EaP0mjS1AB2v67Sed4BD6jc6dAbExr4ybNl9xoEg1kRx7TqDCVrKg6vFPARKLhW+05yo0JwGbkSJ83ZVGj4YDRnbN\\\\\\\/utSxs0UfYJ8zoZDUHPSzO6HA+4\\\\\\\/ST6fHDsTwxdF+4YYyrATBoBGU1KX4fqjoIxrcYkL5NB2Q5FtoMRBg1voJwrhOMLVBz+oFYJH4RgeqvkuWOSAiluTUei9XDQpoQdN5rrSJ\\\\\\\/rxzZz5x7k+\\\\\\\/gFNIMBawAubB1SMbkECzo8t1Vv\\\\\\\/\\\\\\\/bQgyfGO0lthgSWl6Z5vZnpopa1IHd1IGIbxpp3z8O84ft6M8EexQji0mT0dIlnRxlzB56MgWQXrKkCUeCxOLmQTaOtXxRgkTJbo0cdWpOWDJGGagqjasTLphpIiSIrvt6FpVU10OwhjC2WB7+RVaWlf+iFzXH9a2zRnRHbQflcszZv9OI7gShZZNMRq5n8q4++Dgitu0MUFAbqkx+Z8eSy33jGS8gByzM3JEjTuScxsuYn6+kSInWijrM3pFlFj\\\\\\\/TBT06yZXxV3xGIFhs9S4HwuHsG74lDB2evBrVT6Ndocs0hJFabO5+jSNZBwkamWOnuf1uy4vRMkCVd6OSaR7Ydy1qjidvkr+A5\\\\\\\/vAkNvq3Sy+1o7KG9nI\\\\\\\/boA2SNlOj0NG5b2tk88V0BRg905L8Q8aJRV+4TGILTHdqLsHBtGZSocLgDf+IBhke8\\\\\\\/EGI0GMORyshsTz9QfQ8PPBR9k8fsn58GP8Nbq\\\\\\\/siMC+FbdwfNqhNMYT3QS+ryk\\\\\\\/hxrlL0gMihH54YC3Dyo17pqZWUhS81AH6BjoSBleQie9E0m\\\\\\\/MufNUbgq7lnVPzujKZ+rsPBBKrS7NYKX2pt6HHqH\\\\\\\/1kM0glKrIONINUFNnqobJZ1p2Px58eWfRebgqB0zzLxItVGyJIQs++7dyRUwXNnB79dDk3XSLrtd5rFdJ4nM\\\\\\\/yBGFhGW5tBiecGUz1b2Ou\\\\\\\/y5HXL8XHUwFRdSYxv7QbxlQx3OgWpaAJqHg5iHcOCP+LyixYPylQCnkjv2HYC9hAE6qHiWV2DsZGNNQwydSRPQ9c8yJoSTEiGTp4\\\\\\\/ocR45GuyovNL1sjQE6wNXBERoBG+aTxCS2YRgJYstYBGBYg+mnksJDAGlakEdEHduNYJASJoyVW2K2c5oXH9S3BXqLUpjDkocQyX2Hu\\\\\\\/NxUzpApk3FbJh2UOj\\\\\\\/qqkSnG3tl4JsRyWFidZ5X1jnhfa1RI2cRT2nW5qFdDxDs7ocRebYU8uEgCy9HEJQ9Psewfo6X+kG4LBi6ZzJk8+lKoFDrCgFrTExFXVmVdEkSeoItqco+hyKq0dsCju\\\\\\\/qe+yaGEcwxdklrhblvFFzD2FOqBfVyKGSZysdhIhrg8nt4UU0+cb\\\\\\\/VcnD+kR3tf77P8s5s\\\\\\\/\\\\\\\/T6ydQMjBLkzuvVeRMsxPN2iRk4Naz+gD0fGEbtgfwn\\\\\\\/U3ca+IDKcn+O84YYiiVA\\\\\\\/AA8cbXrpGeQId0SDeOFsLPBNp\\\\\\\/454+K\\\\\\\/nnHftFROqJEYz8wG8DZydHdXSbSbNAO2AT6He1uR66Fpi\\\\\\\/gKZuGPh9zXofSiotZAhlC5ipPl\\\\\\\/UajL595g5UVeQQWo\\\\\\\/ZSICtn7bdUJg9Lj2n4BPsb4eEscqZseAf70No9J+6hB8wqTrPrzXdi5LRsL9MYfxCOYRi1mu7BXR03eM2r9AH67EBNaadZE3X3+E5BvJEMoUQr53zLoSgf2W3hTh6xrDj1XAnZmlxivXn7WL+6hzOgWhCG2RxyU2psnxiiBE9GF95gKMTL+hK4ZxWvaGYlU\\\\\\\/hZWAGflYT2occX5MzU4X9jq1ImthNd4SklIWrSnTFyV73J+eRr2Wyiha46d8nUBXgdocj\\\\\\\/R4pJ2GL0m6xgElvjkJKDo+R9QP+KPQtcpBI1c2aGegs1xTs9kR834OrGPZs2m16hx6FFFnqNVE\\\\\\\/o5xt52QOqiaP3dGICz74G3T5ieZoAKCRaW4HRl1LiRLopShliD6Gc8baVD0VnEWT+L8LcU8tzR0lc3o31yA3kJJVHGf9xVdPKdKoMSwOFa2JIV65BVkZvpvKTXkQB2kY0veIcRs3Jcz99NtyTnHf7gj42XWtFXZ8zVFWP5LG7WR\\\\\\\/CNW5v4WgvDf10VaFiMBORbBmYDvBr\\\\\\\/VpXgNWb3IlrGnpEl01JA\\\\\\\/O5yNoe4B9tGPK1s4EaYfqDsv64BUXQDbZjkwJH7RLduiSAPEhIRH5wbPvx5f7xyhGdj9HsiYrs\\\\\\\/\\\\\\\/z1jpMceRnNRd9C8l+8y5J\\\\\\\/UrcF3MFN49RFKsrlYYDwmzXbCjT5btwRsHI0T2UV8bbquFqBSFlKyDShrrt2qI3aT2mA4W6CPIcpeJPN4Q+fwFPvMTV5HOGOriFgwN2K14QDJOhPqWA6h+ERuAcTcYeRKlFDKLYrGDlhp\\\\\\\/QhaUfCTu2b4CWPd3BnJZuYkST9DcjQv6f8niEKQPAjJNDUU6\\\\\\\/xRGXoz7dnTrmjzf2KBjGinfNROtqh6DbGdpi8qKJLn67SrdildwInf5LBbG3xnTfJ2HYanfm7f9vIf56unGZ0OHcLLn73xVFD3IAcf+v\\\\\\\/kjVDUHz1zxwwK3EoFjyZWXeiI37witfv1LJpGlxbrrtKrKwW13jQYMuOB+tb1tjn+PcJqZ508tru+DWqFYeUKn4ObW2zWDoC7\\\\\\\/NZSzH37OLnIrAkmchKhEPpHCCddwdX5y2YgLe0wE35j2WONKox8\\\\\\\/hio6o8aLG0eQCXMprv+t7OkEuR6dRDt9qrH6KX9gxuEH+UKHQ9S+fLfeZK9LhIaKoOhQ67oh\\\\\\\/qJ1CJAVOyfudLd+ar2272X1aOTh2jNg1ZbuBOOSXndYoLEPJgYrDSOaTZo1Lg8bIGUz01huok6BfbPTtDO\\\\\\\/cDB+PIjGRe\\\\\\\/lgttWGyuAyvybTLZBDyMWhqzr0VU5udbkLlEWkSAPKki7u6x\\\\\\\/8uezgxsJ1z2iadDJLoUhfjNhKYQr1xL4Xn0wkGVFc+Bjvw4h\\\\\\\/zZJiJ7u4Qdb5Yz2bRxxfD5ilW\\\\\\\/9CSgBpdMbsk7VFZ9GW7dLlU3TXAEv1fZGkgMq5A9aLdwlxDSWKRc4VV9AOx3DJEVNZeHwRmlxi4oRZqyrFYShrfQq4J4mlWXz7Xe4S\\\\\\\/S9Bdgw\\\\\\\/swnNa4XwRsw1GsVWrGMHZwT5SLpXdD7LpUAZJvSXJ21P7NdfyiCORW\\\\\\\/TnAjlUSh1w5JThg1JMkIrgJWr8uJOt7J7pXdWkL11JQdbh0MQc1sD14cWsMQKLVA75Zx2dFNY7M9x5GMpI7FRK2kOf+HOIWd50EJOLfd+QKUgArFkAfRhx8fpWJZ1VvtWK5SJycfWlHYGXZ6XH63GbQoNLifQRdrId8WbWbJUkkq7GUs56hNO6GkjcpbLTJqvlFBTtN+VqZQOC8xMDO8dqP+tmG2D+26L1j0MXIKVUZzhF9+M9Aq6MgWbPceaxXwmszgEj86QH81zBrVgboXy8NB77\\\\\\\/vGMMhlO5gwgoc3DiRLJHVsDVXdGwAWsmt0DHdCNDdw7LoyNhC0RY4WZ65bHrolOe1t8B1FQYJikFWJIwZHdCAMGKUHGOJiYXSgcYXG5SSKt1wHKx1bKXv7SdmavPKrCd\\\\\\\/FMCXtPZLBsM+k3LSGRtgPFzyqCcKi6PpmUAo0SQ6rWr7BUctVFzSCiKQJKS6CMF1eOAo4MMJZCRMg0ZB4LpTkTjjBuQjS03XFVCzcZc19wPSFp\\\\\\\/yIERcXCadszyCEoCjrmaqsQy5AqIBCZy8\\\\\\\/IAThyENd5B5CfsYQB+Tr9KBHcD9C7QCUEA4NmdixmT5gGmRBXwWVBKLL1YWAg7xHNMnQ7v4zGN69CG2Y0gUzmjiSl0gfqV81n2vNA\\\\\\\/bnuNUI0errX6jQa+W3EeQrL1JrTcuB3EFG\\\\\\\/LUV7Ab53k2xNvj49477sGzxsiWePIcSUXQHuiQHyQ\\\\\\\/Gyc2IyjhkEH0xFIlo6AifUKqUNRDMhxNTsr+PydikvMVECT0kzgDNNI55FYH2QfioxImdNvd+PAWjmJJvb4ewLNZkkfuQsZLSWsUebHBuiCABpOgsUBQUwegZ8F0Rln+QjcdYtNL72Gn5xfAOzciNVfgx+yQO55rt\\\\\\\/g0TDt\\\\\\\/QY5OUtqAYZb3v9JXMhswTRgaroW2gE0POlFAHtVzB3VKlQHqXE+61WM7feKJoflNaAelI8eJKH7BHKWlL\\\\\\\/bnK0GhIuOr4neFdAU6VsqCGQyZ+HATkP0th4huUpXjQhRFAQFnJXcpNVNJrTSZJlUgXTKPyD3cPySoIDoAHZQMkWgrUsh0\\\\\\\/XUmeiKWS31sDCYVQHtKDOl5CwZ1Q4iu19zbbwY27o\\\\\\\/Gd2aTFeyBHVAe\\\\\\\/5q8l5LIsnDFS2FFiEm0S0yqevRQg0d\\\\\\\/BPfzHsaFPRw+uaGJ2zSFX+z\\\\\\\/oAa3tNEpQQ6HtM+94JwGd07RZcAKpGBk3WoEH1hMUG\\\\\\\/PkztCwAxnWOjW6QnFN9ldF6ACdIROQatx95N+JGkyPN1raxMB+F3hWD\\\\\\\/iwe0u+hu+pSD3a0jYLEPYYzc6wJZg8tTtPMuLUdjE8QBtHCxozG2EF+kq\\\\\\\/xMuycg3wxnnO2aA5bmNrfP5N\\\\\\\/JbAjDWippiOuGr9PKvLPvXHB712UAJ1mBhtci0JIPKYUQIPTMbmbXwsnfVcNIvHOHuRTw\\\\\\\/WNtTPoYAx7Gy76UVk26FWXPyjoGiLOP2RdM8nOUj25UP1Zzsiujh4aZjhZ9KhZ\\\\\\\/QUPQ+KPv\\\\\\\/jekkzgJCeKzLikde4V9zrLjB3KSpHQciayw+YcRtZAYoimz+B6kNZiHab3hNqA6KWUoW9vQb61hSgeQoRETD76xKzQL3CRwiyt1Gg+MQNor7rMuxfRNuGnXx3iZhcnh1JTcfeEcDhb8HsgsWVSBcE2KEuNn5ngl7caKHTsP3cRWITj4NDx\\\\\\\/xXE72+LFrk5yueBpMIhxgIATfmU3IJqXInDEPtb2AAUJuppzsGb0T9QnPpvCPe6Jq4zQxSp8bNo0\\\\\\\/Z6wZ0\\\\\\\/Pz1x2lVG7jBNzgf4R9c5Eyold7ZbvBOyFFnR0sLL0nGGTKW\\\\\\\/5AfKXygkaorYI\\\\\\\/Jf8exZAFUSl2j8EGZ1qWsOUugoU6IhqDWPLXCLeFNM\\\\\\\/3KkZYlCoLGTgPa4pS4wtRoMnjPmpZvKpiUXRjh1BCYR1l56Pi0vBwpVE7GpfNfhP7dHtSdSdkNLs3YOT246ncT\\\\\\\/wfNRxs+ODc\\\\\\\/7HF8hqeK5eiNXN\\\\\\\/5r1CvAkPyZzEnG9byo1\\\\\\\/ynjrG\\\\\\\/WR1f1zkZGvTLI\\\\\\\/GjFUtEX4lCJSv6qJCAnKbugM5QhQbuhi45+dbxbXElxDCfOUyMH0CRwJCgE9xbJH7lQf71vbAfFVQvUlp\\\\\\\/blvwZv+jAfzGd0qiNzpD62KHcpm2JEqeUZNFKvD0UFrjYLCEz6tur\\\\\\\/xn4wqf50zvCtp1rm5YFYSf9PlIQUo+xKjjtqy2CHudxCsP0qR\\\\\\\/22fIO7o86whjhIOxhV\\\\\\\/r3WzQefCh9v2HGuLaUvXHHRcRHiT9ZNi4yz9iD+IQET6s04oVrtWDODrv9jUfuW5x10WTS6Fy8eHvzuluuCJ2UI3AWnC2MxuixJi85nCadZt9rrWRgaEsbrEtlYGWn7e4971CnjIMgQOvV3GwEfBY4ZP2VpxUb7fq7RFiwg\\\\\\\/iTa+9Ora+C9Lw3sTqY6NvLKs7asfLXP1zlWTqF6vFs4dD\\\\\\\/q22cRzRMlcucoICoekdMNxV+Z49H\\\\\\\/2JgbVIAEBSST+xxgtOyXKrWkkWeSm2Jw9nyvd1IYiktT45KVt\\\\\\\/u6vGK5GfGkaaxOT\\\\\\\/lxtUAmFXTRgQTsdA1l7qG3eLMualQtpAzo80JasUHgvzF1OLMKkzt0rgUEKuoWsT72ImSsFFjaMxxpywDKlxI3Kcc+nzpv8g5kWa2ZKrh92blaa3xFMguMTcJwGSSGpb7UU7fXIdv7Ncom7tiH8PexTJprhkSiGBCNPbv8x8NAxjljttiGhacIoIkIIq7v+vZ1Zis81TjQbaKekr0qydqMweEktsn8ywu4sZrDrP9YoCb9JfpzIJQ\\\\\\\/Z8KF2GuI1kaEykFFiis7r9tq0\\\\\\\/81x9rf4eqtqt3tT4CZD9AGkhhH+ZC+N3z1Amc6upq14vCWJBWKUJtlCetyXY7p3gDhdF9LrCg\\\\\\\/5azodoPH9FwtCfIaMhYNiE6f0a3Fmh9W0oqUu0TMauVP0hVfJ3coYA3ygEGD4AVQYdK6\\\\\\\/fBD69ThQgrOhiUnQlGrZPWFkO\\\\\\\/SDKf2FmNCihFKCZzjhx3yCiT34xy4LuS+VQIGtBaEX0aRY6dm3QzlIPI3CeqzZqwL1dj9BxJAXPKx\\\\\\\/mmn25SnL0XhUmHfsbKZP4vx2MVtXEZMiZxdOcOWprivC8GR8f3qq15B1Kbn0psRwvL2fXyz++ZNlJes36I1S5qYiubnJcoh\\\\\\\/41\\\\\\\/L40OeMTHDmy7dlYTwQPnrb\\\\\\\/pRc7nF3GkxqWZBzkJ8nJZQxJjsQPZXHFclU\\\\\\\/ma8pwLcihHjZrWSUnlJqjr+Ak3ST874SdlTwXSJUcbrXGU3jkI8edoDAeaxeCbdlVaa4fxrddla7oEq3s414YEa99DhPrPwi9MNosiIVTC9zLKN4I3hekFXYCH+AAca+mRo+zAtKhEZr\\\\\\\/OlBz03xNPv\\\\\\\/TDAvrk2uzAaC2Bnm63O1B5FKQnQ6rki1TTYFxlHJqBRchysLP670PfgWhMCG\\\\\\\/G2Out1fnj79hxxq0LkTISZ0SdEisWl+aCWdRYlsiWgjk8obPeFbFyLfsU+jYZIBNG7Cv7idFSiKBbIyZ3S18kaEW\\\\\\\/3T0+sllZFnW6BqVZ2Qf6R3LyZ+SMcgnBSRkLBJIqr09zKjschCBPssOgNiOTWwXPaKCAcoB2yteZaRtgsJp3t0AEqDVBwZUQeYk8jI81i9xPjYgCGhAm8K5WPoEejRa\\\\\\\/X4Fz4UGCbUm2lVN2iF0oi0Gapl9r4UNFnHMHaY+vThfhaVrsDMVb1TH\\\\\\\/QBbsGE661uus1e\\\\\\\/hqyLUmiBPrZYMbAM2IntEOGOUVUXr0EmpzL4LdgLGrL7C4Diqph5TMY445fd0GqoCI+JrRJyaJVmeaiRlp43GoyToUy+pvtDGk9H4Wj7cvMM6dlu5uD4fhanI5RpNqPwM8KI0CgMkhNcIrsDP\\\\\\\/Bd60GBqlNo+GcbgegpQJxh8eCs5ddHI9ou\\\\\\\/W2SvqgDHdHSe4MkY1W1pmlUXgjOVquIYf6H8hT5nFk9Rm6beX6js8ISE51eZeFyAMEXzxd5yTzV4pxXKp8MNL5LeNbWTZ5uUYPbsV\\\\\\\/WQ9BNGX1Rl+gq3frgnlUGSQMqPsyFmk7d8tLgb1tQFQ8r+WUmJCaeRhp78nELhVGnvVkzbP3kwyHX3qSG1zP7uWiGAGTLIqGXxEskSeEug3BWpWOUU3+vk5o2RIzMVHT\\\\\\\/m2Co3zONLrdwU6aIJWyItw\\\\\\\/d62eYO4VFlrW+qedXvZOYvVqybVwe73JT1C5+NSc0J38em8kcyJnHEaZiHjLKeh1PVVvKeg6uX8UlWZjkaPz3tknEAvBOBGgPZTd8e2Ah\\\\\\\/Mk8BhECdmu7N3ChIzv0zalpdeEpW3393+6aDIsDkXswna+0KikTLOB86SW5HYozwmNYj1efxFkXEZWqJ6z4GmBTLJCcNrRPFXS3WakiMFmAknlEogko7SEXh\\\\\\\/lGzR6LfDuxoG7YCCfuS61Y89k5Dzq8gzQ4LeTp95LTlmEvruNlp56lf5ZBY99acIAeTCgRXqQ5kk8xzuXsZu5cbmLmFB1AvvDneFaZ3hj713OVKTCJBEnIw2\\\\\\\/HZjuPjiSuHQIJ6uXUSURgSBjpd+cDzbaJfZIwNqR8RQwmMzTVmXV+yWLGJbxtzt8tiRF8TNbW3wv9Oh8iWyJ85rWZQr7+FzqcbA1UCXuO\\\\\\\/TCwK\\\\\\\/+wW942e8gR+vDM3V+knUBOOwQCm7SgN8i4nA7W0asOcRUL+SaG+zL4fqpLRM2WiZ4YqxFXZriVVd\\\\\\\/bCR4UKIot2ICS1+9oYDe3mQ8oMuaHSZmIRU3Q9ePw240PQCLLPfR3ErvPXj0BqOJmIKOmt\\\\\\\/D+a5N6OpB3YaC0TGfgRJbS14YLiw955b7\\\\\\\/3uOuzKjpMYWUcNnGZ0Tm5MMoiNkOFJYqRvaoq8LWw4oju2GgGCv9QXunNgpNoyr0k+FhWjSsz9nDKpnkukI5nMn2O1KDNvBYAtAbPHFbpkANsbx+JpWtmI6UdkQ0KRmT1V39TdmPH1rfahUenj9te\\\\\\\/HpvWOyMC7+dYDoEP4Jv7tddJkw1x9D3LmHv+\\\\\\\/tyl8xhEQGHK2LNMEK5dY7G4+uIH8PXwjXVd\\\\\\\/hLtzzPtJB0XH\\\\\\\/BlBA9HafcEguF5nJu21exr5aMVLoV0s6qOSA+aUJlNxuU9596MCO9NpOh+taaAZkOHAYfemgvBF9IucDBEcQ\\\\\\\/DodZDYdiWBT\\\\\\\/WcUfObTZeTZAj7FPaJiDadgcYtir\\\\\\\/A9nx+m\\\\\\\/4\\\\\\\/cvjPkDrFBwZt5FtEUSfHKW2HsuTF7bB1K7h\\\\\\\/3TUtH6KGtpgH2jOfDx8kOuwQczf4DFsxc+02FNFu1O+JrsVAaJtzQ\\\\\\\/VdplxJjPCQLWjahzwjsQK9wH9H5fmx+x844yYDrLiUu50P2ONTiR2lJmaxBfEPR3a9G+gySbsJQxUdcM8c7zeKpBoJRT5q4hePqqxj97lsE9trdLNw86cHtv9hFuTDtY9Cj85O0rrzFiZo8lk2lsHSElVzhvds7pHmDuC7ZoapRJTFOMww5uksSO5D0yZmjlfLLgsDNZ0J8CIorRTr87wixmFbyGre8TY0+dMfQOmrATMP0sAdJOnwjRnNui5mT++XwPyEo6HHEaJjP5Rr6ViTvJbPL5P0InI3h6eJLUYgvc9WFIB0vzHWj8zo3M9aJWBS+\\\\\\\/jSGrv\\\\\\\/bMIBnSkscXZUG+pNmI1h7jVSJ0a7rXPPe+zoi+T19mSSPTZq7gMRzfgt1xag1avtTdm4socp73RtGMpwfirL6Dw+SLFUZdez77xVkawtry8UTf2MG4QM3bbvbC1ow5X4GpZDCUyM6HnOACqCZXkLy7si6tqJPiIJT5xL48U9bZwAwq51HT+3Nu5BPp81sLhgDqyN3FFFn+9WI1N49Dr5A9BfoVrp\\\\\\\/s8xcUzd1Xq5yLnk0SzQOlSoC1PDgWB8M5Oxtt\\\\\\\/zRNrO84Bi6VohKzYHufOKJpmSdNEGTmH\\\\\\\/8dIviFnXR1zV9Fv00WqsS3SsHPgFiTWq4kHLcyC7wyr9b2dJSBlw2coS\\\\\\\/jzMvGl2XkSMYzbh0T\\\\\\\/+zg\\\\\\\/mwlm72e\\\\\\\/Na6IULbSWCuX3z8hGgQTlbzDjO4S+jCSm42lSC9hi+F2LBxZwtVeMl3kvSl24J1jvU5IdK4jcl2cm5dhw\\\\\\\/XjfXXvLHEoqFer4f2iHfO\\\\\\\/jSCwIow2Hvgvn3sOzu5rP6gAIXhpteQdQp3E+MHeuH3fwX8oe4eZpml2C9BaYzhKjk3bAULUlf4FZOFmM9PmskXwNXfh6TygTgTxW73wTtW5T19iHNS81cCefqS37ko\\\\\\\/Ky\\\\\\\/N9QAVXHknfDLNsB7zR9cwQ0l9OBijCLNjssXfstvDjEBKrApz2u0iaf4v5kmvnF5qfptCdwqwyw0AAHlInYehyWmRy5KySz9iMy4QIBQ5LQeWwkQdQxX\\\\\\\/LSurnJf61u+t4W2X0bTfgUNv\\\\\\\/a1ZR7U5Pai4UzESvPbhwbtEFj1l\\\\\\\/11AdvTIOYHmTR7AxrGbVjgN03NHKmyk2h4Uedd8jk4cV\\\\\\\/zTYEBzhk5GlBOG7Li+CFrEV5msXiXzjvrNL+ZkehrFn09RU3lJ9QsyYbeNcTohQ0QLaxor8NC7xpqsA7k+Y2hLKIM9o5cJZFs1A2gGTt8ads5BhkUSSIdLLw1\\\\\\\/G+2vjNN7htBci\\\\\\\/7Sk20FOJmshz9NOaHcPh\\\\\\\/tW1YWPziJ6T3Zf3COoTxtAwRhCpujlfJr7rUWZO2dx0FcymDZP3U5ps2UAxBxRFjmg73w16UR8e4MBNXGQSjyMcUEqugyf6o2PhDKx\\\\\\\/DDPCtdBRY5DJhtbuGg9hWNnQPhqyDY+3LrMbl6f0zsQ8bJtwI6Z0d10yPsXAGEHZFdu6N1OqzdOYZOru+32S4pBrGtkhz3hXHIpHkwfqNjo4+hpP4pnLF5wtOklyFl2nySOqvOatCkulo7YilwipB13\\\\\\\/N541Bq7mzGDMcacB3D+zrZ+XKLjrW6j7\\\\\\\/X1R\\\\\\\/xR9i3wq0VO5XbpwSvUL9mpNuL85bgFVDM7D2nsN35GC5W0qkzjyvh1c1oosIGVZduP6OnGUQrSlnLgC80UCskKEd3j64Ja1NFOWKxOEzlZTagJiC\\\\\\\/OBU2ljiAMaBpwpGDBmHToOaeD04gzjfV4I+bWLK2KqAvha9+baUduPfB6E7P3etrZ3POEyMuXsPG6qKG8pRYLIzTrC2uBIndQ1gyRGax30sScbtngVAR+6wiLfBJzwXiWzIQgcnKxGwSQ0pUYpN+cshkkPnEcJNJCbLdh6RscyQ5FUOk4\\\\\\\/ghpbuoLDMrlx2S9evHipZeEWnBd4dljRvH2jbqWX07S5Jlz\\\\\\\/kbqwiyYQb9oVpWeBrRckqC3XbkWpI79Et7sSqgq7VKkgE1E6cTK1ezA81Rf2ROZhJSMS\\\\\\\/92S+qX4iH0G4NyQmOjyYoZuTBzAkraf2SMy6VuTbzmw0eA8h+rruOolxTrWV5UImEu5OggIgewBulZpxsTlJlS3Zp947w6LV+7CPTHhFBfMbI4sX8XvX0JDkmjQLv4QVH5dCmYiHLtnFvv9FCAtvaIMy8CeFKh19bNyk3RrHh5USFNy6fw6PcuJ9UsJoBcZ1UUNFTX2sNs0g22khBW2P5pYyG+HNO01vvBFxCfAWmXAXpeeJcAcJoPkteJQ1riBGYiE4my5ohiq7cBaURX\\\\\\\/tPgIlUMrncjZRe7uBM7MuoHJK+fOmrWnryzJcmCCIFnecbD\\\\\\\/UCpL18yC\\\\\\\/1Jn4P2L9TfYW34SICODiobzhaPA1v1\\\\\\\/YlEHlZ6ecz9cZl4QXjGsuX+7yb9p4XvcMohubtv3ftZiONJ0Xx4VdXhmMVxp9d73jungEMZeDWHT8c8EYBBa4v8vUfTV4aP4kyC9swO6x7yd4q0uooBwFvYNpxOmHAJfu6Kw7mfoXJp1tAAHa03YFgQvAyAxM7Kbn796qPoDuvzKdIQ0s28t4+zoQPSk5bz95nbLWSBXl3Y7uj5Pil\\\\\\\/eFA2uyCEwXrI0jFYFAI5OFllFBH69vme7QTGsZMqQ2xOJ8agYqNTCrYnhjGBfRNJ0wm5QAQPs63tAO2MruUfmwnOTs3AcpWsnFdS16ktkttmRQ2TNwdzsmnIt9bH7VqJwDTANm7lflm\\\\\\\/jhu+us7RL54JO9+X4p2OP\\\\\\\/NghdQ6gpc3zQMPX2th9GfOOfOZChrncNw+jjNVlFTpUNEd\\\\\\\/lAnOwFXqGuKvBbrBqyjKl+bEKWrgY47WFsdbLlvdvYBfduQlJlEGsHEoVwMJ0odNVwF8+v8EIOFrQXoA+1qSaRGeUX6Sc5ohlHlTSNpiCb8mgJktViCGS+xEyWO98r2Ow4na6c16CS32DQjmykjEgvuTP6Om6oIcUiCH0GJu6LJkunVVH+pdPuuX+6HRp1MJ8idgiNTNmXUSBDuKxhMizaiAaAn91xruLqmGEBHm4aA6PTdBGmPi5Vge+IeCP2\\\\\\\/snuuZ11AlDmRsAGQ1am072Gzt6K1QYsp1SLLdISdav2VODc1yn+QwjG2ObBH5nDTXYkkKYPzkZILG\\\\\\\/YteffXrQkKhmj\\\\\\\/9cPoGfzHjz1RhK\\\\\\\/OVR1kjAolLrd\\\\\\\/dlmEUhekxpOCnlR\\\\\\\/6+Aq9\\\\\\\/WK9Uiu8BseAw2Z+q8vNRsmrBuQgMfVwrMKHzM9gx6Rk5teL4wptebJIQJYT335kQgaCDlgfBvkAqZE5XMz2LlC5VKIIinLSFVgjzm6kV1fz\\\\\\\/Fk6lvInWBEevIv5uJvK7Oa4fuUIUEOcUlZVuJ+JhMN94i\\\\\\\/61FXPh2FRl+Xh5aXMm6CSoghNXu4NUqkY31nFZMSrdqGZKASIxvpa7fB\\\\\\\/oj8t25vLYkd44xHCvbc32DYUWhkk\\\\\\\/WDb4\\\\\\\/zBFJ8g1t0kSUmcU3mMOI\\\\\\\/XW01uVBGQmI+0L9DWADuy55oFZ+T5P+zzdFM2Hf+d3gF5yS9CWd6cWsPafd+wJwOdkmfz6qKWzOavdTwOQ2KgBHC7sl1LW0r26MKKBnpggAFD44w6HeHXKX\\\\\\\/lfvOpAct41HagftaP5kyv4jSQyaPzzzAAsKmFBydu+PCo5fXydmUo+p84LDHJEPAQO\\\\\\\/L4Cay0KMgEI3N2xRoegrmh9mrNrsu30BuVZkyh9aZOBg7GBReVmfvUJrTBDp1luRDCbkKihcj\\\\\\\/ohKeMLYJBcgrub96v0L+oZHI2SaHA8N3HxxY6hw7PBc85t2MgVLUbdUmE7BZ+XKuAf3Vqd8pxidpEAj7NatmAEtp8O4bgmz49n8CUUIFA5irPT11g89ZWNzGnjKNLQDtjneaCku2+qFFeJzdwazewJTC0U5U6\\\\\\\/xsMuFxSIqIB4u2S\\\\\\\/HspDCuv5Ji84MFgakJlOHf86E0aGhv9W1SatzQRSCaW7tSFs3w0+oRuRRSBiZ1QpES2QUfPfk+VCE3KkPOrgKgAJ3B6Q1skTVpBZblG1H0Dgf7ENcjam9ZkYZH4my0qWo57gmcfkB4DgbuWklcDR5OQlFQeNccvNonOhDrwasnT+dNLC0PAdgAnoEXbwimV1MGHFnoyc\\\\\\\/tb6+0DDCX3YB7eTMiV3EnHtVA\\\\\\\/zi4bEgFlicTJKSuv8ZOIj2Cb03jgMnT5juxRgBQFWsJuPAbQArfEpy8odOd5RR2H2MyPHFTMOrhjB\\\\\\\/C7xG+U3Hynvs5gsZB4rsn+71DXwt\\\\\\\/CjHdDhLINmf\\\\\\\/QhsX3+K8\\\\\\\/hqEiM6\\\\\\\/JenlgqQizwZII6Cpib3Hz3sl7wcz0ImS11YfTfQP30n1rVnEJo\\\\\\\/LjsVcXUNwjdt4NDymC+1Zb53GOwudOd+zyrrKmvt0jBhvPseLPGInD0mnfKs4+bulOsQg3yj9c4CiJGTdQbmEVNu0bDI3WlmiDclkmnOjUQeCFc\\\\\\\/MDIoGoZl72op9k4QJmppU4R+C8vruJFzZ8IzJdG+Rn7Fk9WWUWx1yAmljVRMCXDmU+1EEk6CUBVEZrYveEp3SVpUaoKdg71sYI6G8g9S7p7uRFVj7RrKEc0eVc1upwA+V01CKtEVg0NA8HLFc\\\\\\\/OC5For9hLuLufk09rsqDQB2LYVQjGjm0opB75VAmPa8MNJcG7RYdGWaHXhHM0rplbkFJvptZphOBus8DuStafPye2I5limAPOhpihtcacIRuKBkNLsupW9OkJPlbYQNjF4h6EPPX635YL\\\\\\\/oHtGjI0TfZvaD0tloQT8yYLeme+cmImSUY5PixmsUn1vhoJ\\\\\\\/jp1XG7Lzy\\\\\\\/ycn9ztCH4M5JmKxjanYyusObz2KqUNO0ErvpXvbvptmqHUQDJ2ySIVc4bCNctBglqbDwDxkVQagDFvZyEqURCLyQry1z7fRul4en+uNt0HD6urk5CCC+ommmBzSMUNdQ7I1N\\\\\\\/1WvCH54TnHdSoeb2tEfcS1PpuC1Xl8LK9Vh3ZAAjE6UNWxo5YvYvMdQgXnSGM3mugWlRRrfyz5zQ+xoj9HuTnwunI2iaYkqmYsTIN2VUBMaUtTVdT\\\\\\\/avjchq\\\\\\\/917gwHwNgiYmmWa5L1vVmIKLIjJVyl7NaW+Rl\\\\\\\/kOnHlPzB71XQgOXi4RzPO4xLc1o4lSZ1ChSB0yur4d4wuBJ\\\\\\\/HIC9eUbv2kyBZ3bH+JOqJRtjO9KvdBgsG59ZVTJcJDiGEmxBmSyANuWIFSB1sNot9NYsiLZQRyCHZFyognmw+EQmHeMnC86dhA7L5JJjOsQW65eD2DWnMdOSJHLqT2fplNj2psgHMMzl4pHwNaHxhTDGMELQABw0vJLOF6YB7slruKpAMZUFPvK7NHkTzRfd3h7w8UfIW9f7\\\\\\\/PnJxz47SVDVhYlWqVnruoRjV2WTaLGt38hyOPDeDVY41LaptfB7L5EQGvooWFarYluj9A\\\\\\\/ReYJXqIo\\\\\\\/4LEV3jQpa6vFHbjQ2vdeIT49jbeIUum6+\\\\\\\/sNfdEmh0OjX\\\\\\\/e9iBqfxDJxCBiu7AP+IPVv9QwHO2+p1TbrIvx6Ce6tiPmsSoRj66tkbCeCcvJd43iGur2O9vHz1LY2jU0iK5NT0duqSoSkFQDrTMAhg+ZB7iSrjdYgokgncQz76rEdMPiLjZTnlw1duvNoGFqW1IkIqAA03IrDsIPeLnb023p+HnCSHfsWrqsQzK+y8+vIjoCcW\\\\\\\/2Ey1TQYpcHocaTXWevrl5lQywPpWadMrKw2i0UBUg\\\\\\\/n8AcPrE0pTSoHZ+s19G71It0DC2cdkQ\\\\\\\/Ml1S9JCC8zObv21B7l0Tuwv19+Y48pJ4tGiJLGN6uO56Z0lvoTseB75E0Z7ggdBKDP6uvm17kmXcoG5Aym9aX7EP21Rg5MRvWaRTicql2l+iMJRAFPig7mjOUdPz9ur2LxL38inyFSFLHmto1UTxeXcBoSVAAeQfGNGhzTYel0FnLicMYmxOMSao\\\\\\\/btRSd6qoUX1bT2pFmu\\\\\\\/29AAy4rqkGj\\\\\\\/zGWTKvDVYiCcn91+qz0GLJ0\\\\\\\/cpM8Df2gqpKBPLgXzSpAO+D3hEwhNtS755KIYOMsZ8vL9E+q3WedbJ0cGA1wHSkvHp4K3WDbqy\\\\\\\/5a4VUISMC6uiAZ0I54GeYXqgt8Sxvk+7H8b2MDaGyd1c+dEyRlqDcVkJq7N+bIbRoBNxAgK382tbOkRvHhxalP\\\\\\\/847cmBtLiNeOdpbJ7dcoc8bBeayloLoflfH6lCvQVAVSsQpUW+LkA6SYMj51lHaeTRuqYMmdlk6SFQI0l9qQ3R2oKcFUEmNxU2VJvq+mn+\\\\\\\/H01Eu6LwTXOWYByr4Yr6w0fHmVyB3lumkW3o05QtmSGQhWagyPBiYkovGpV4lf8ynQOi+m\\\\\\\/AbwOyxVSSYbvng1plHmrnbnSLiSlAjJAvVdVe4cKYLbfWnX30w7Lbl51455GfTFa289xxj3XGWIWR9P3gJ2i5A1b1qgD50CEt2gM2H+KweoxK6kvggk2mIjh8eeju1SAjUwdXlse0rrDa\\\\\\\/PA+luc1Xq9iPMxwXtcedqbtcj1QGSXg6cAeK4epA98P\\\\\\\/UALqpppP9JCYP0WISPgR6e34B9GuXcK+ffHRmFG2zHEGp4spmy7UUwKPgJFbRxxkHAVtj6xqVZ+6OdJ+Z9rbb5cqrNOwRDxEa7sw9otbcb8d3V70vVcyYdixC5+7pgWxr0aXn5UxXe16DQ5lbLenBZBYb2rPCPuAhup7MODPk0rjuLK6bxy8hRZkfWtxfyvN6Dj8adE8NDYItIkkuVVnUl4Iud+9Tuag7DxlVsKdtOEtfixOA3D6ZlKr4C9HOaph7yX7EhwyrNMqh47nGc28M6ZQhZRZTj5dJh38zfcaqOyydJ8IazDCUaNyx06zRRE\\\\\\\/T0XxMqcj0PgrqbwdUw6ihx7OkPjR6+zgYZShcrZu6d\\\\\\\/NIe0aMQaRRDJe9L4DTxSZk8K7GenNF8TytaUJJuwr11R+XLxZJK2wnyouYqJVn7ngRqt4OfOEn3lFXaGJesVpHXU6M0zxRrk5veypzmW5QXkU\\\\\\\/0lpMSQNAYwcBA5+xe\\\\\\\/YvZdMHdFon1F9v8Rcn0mayDYxWa3tqb8PtSqSDsgaMRt4yDSwZYtdBmzaybWnM\\\\\\\/U2FhD6CklDwsQdBuLfgzXHLCoyQqcnlvfQNYJUEZXfRAMB7Znf6S+EDsq5dQVgI\\\\\\\/uMGcJFIZ+8uPyhGA4+nWMXFps\\\\\\\/kzE7xrWLx4nmhr6eBXbRp3qWInhlvg64RAljE7P5a6v8eWNK3MK8RAxD4Vr3adDg\\\\\\\/TsZjvh+52nUsOMWy+4bGtf+q4de4fTefxI\\\\\\\/76c2PTUq9Zk57FUGYuNA66VU7myowmH5HPNxz4X0ksurYtD1Yhj\\\\\\\/BZt2YmQmdLT8tC3VZs2rq4VNUC+bAK8\\\\\\\/FGM7HZyqJnzCuBhUhlv5QwwzT4mmMDwlEAy\\\\\\\/0kuuQKa9DDkgYbpE3hDcyAbcGafZnz9btzyfjb7byiezlxmYYTDqPorWLb8HaT9cx+INjnPDbJdCt1Ppz+LkNwO6SvieIOSWNaPkgYUYlI2hrTB2aoRfQ8DFQPH+ezx4TSbMkth2HHoV\\\\\\\/Qg42TPXn+womCSHOxL6s1+V3UmYJA4fw2PkEiQGWKMso6fZ3kwc1LAON4MADv7BSMSGwb1iWZFrZEPmxG72tpYib7Refgf5OgHdiH8Un7yQv4xaIzWYJZJXK8vVZBoAKm4yzFMHPS00FOKZMRKOHWx9SYvfgPzsORN1TcUj+4jPtffthbA9YvOa6HnMc1jjBUXqhv+TP2I3HRIwxOjkwGhXZblRCGb4ORfEXSSg829PlD1HnTrEcZaPcBjrKI7\\\\\\\/hmSD7iTFswfFGydBXpTXZkV\\\\\\\/oOqjazX6tMU5vOc2oWRnF72fOxXbTEPosl6tU6zX0FiiVieA9YS\\\\\\\/SH33D8\\\\\\\/xB5IGVckoaeWvcLyLleBBTkMEvkCJiWerIXRKeh7JERC9bVaC6og7Xa16Lo53XXNMQwyLG71YBblscbZYyNq7KfNjiL6MWXRwp7ZzJvhk0BrJMD00X74Myz3iQYRs8s5W8bf\\\\\\\/SppHPq\\\\\\\/a8pIx0TP8HK56\\\\\\\/NBnBSeh5bOoKEMBFR4wZGqAEAIqWN\\\\\\\/O60mxeh0L3eVB8PkGkZQwHRBzmUA+uZ8r6i80jKKCpQH1I1v0C0flCyzkHQVTcY9IK2nui5XIxqrKFMB5fau7yOcaBgVKmUpjQtXQ4AII8giT8SOu8S+yj82V7Xoda1gitShjAcu\\\\\\\/V1loH5hmDVnqxztJbDMlZ4ab3u\\\\\\\/\\\\\\\/h3dJymXat2coF5NvtlGtawXw526c7\\\\\\\/IbuOgvGuBSqbZlucoj+Tdgx60cQ885LMqlcuVfl1iD0mDt5CeLJVUrcI830pYiO6uy1IhA3OiPee9xoTGQd\\\\\\\/btubzNyf+HLE92y7tuWSQs8ZKCXoyR5CiKvSEeCq\\\\\\\/4wlgJcnbPCPCXvY3rUB\\\\\\\/SZ6WKpLV7XGXJ\\\\\\\/T3pPrYaOlDTcLGhoGZZHkS0PUm7ypEkI6H21FJ26QomjfWt0Z3P6V5AOqGWmKA5Ir2ONrldFIxfJk\\\\\\\/HKywjX5lrbaM+Spom5u5oCB3Sp0gofyLQ8RRhfMvu7kel6He5dXpKHqmGazQJ8K+xcTqbQ7N6oFnpSo4XVJL28s0duRETijeJINT+C3AjhOiNTO9bh\\\\\\\/N5xf3dU+vgqpD2ltEmCtNstDve6gSSkidsLxTFMyMHo+zZmu10zFTua6vgJ\\\\\\\/N5+SsT1zZ0UPUml+KtL6LG2fKYhHVyluMQ1zbXAqeOoJ7IKC4q0KTiyW4Nry2pbQgoznBbwEJHaxhx5WY9CJWw1E41sQ8FxJU94fhba8+C\\\\\\\/HjG1XNsNyY0aBX73ViTM91ya3QQHKpdtIwxiWhpTIlffwpd3pICRB2369Uw9VBujFWugn05HCE7siuPcyDchNCNEVkkkeWcidCQoZIcqEFSC7Tlt6gG+wfJSJs09cZcWkP0yLeLai3wsBFGkNgdmzRR\\\\\\\/Ukqh8a5ViJxY0lfqeTEb5KTrg232T2mnx3JQ9at8kDjizs4s9FUIiTV03WT6bRoQv5C6e8GQ1Eqy5jZYuCEG6rsGSSHvUC6PCZA4+5yJTCaXijqBbebbg0dO7OlEANzzJvWiaa2jhstZ5tl45yuPWj+F3klUxv+mJJ3Xxppbc7rT8tu\\\\\\\/XO827F1\\\\\\\/3yMYOGoNeXUybK0WWrz+fO5j2DPgLI8exTKut4HtWlz67EIfK861\\\\\\\/rY26afwLi3HAnvSqaXMoLnqEaTE0vA6d9qWnwZmm6cYp9D39buIM\\\\\\\/PfV3VJryeP15PBWNgcMflaOAiqRYCtJNAk2sAmxOIgEqKJDcZoq7arnyjTr2cllMPcIs\\\\\\\/u1qYOTMAywyBJZ52Yzd4Dp3QwSssrc0Fqmbu45GkrfZQIEMLNhv346uPMlRc7S0x4Tf0jJKc762PcVA22LvCryLXuH4ymWmuNEetqAOyXuMlUl+bqdxr8Er1NrCt1tWPdxgyP1\\\\\\\/niEyAKBy1gwCfekWOuB8JccLt0H9Xi+GD2gO7Di3zBxBm9pXD369ptl6TMYSGlNxDGehh+e3+ATAvksUJH6GaqtShpCazWhXZ8lBbdReECWNmXnlx9rXCutCTjD0KJWSzYx8KTK5s6eLg4kkY0TuqFvio4a9V9mVjBOutec6nG4MEI+n7EJmnkYHBYMF9h4b+AMhTazgjIrp5wmfOlqs7vFSyLxDzg6A+sH4mmUiC659RQEkjYnIeDxLS2IeSPKwFfcLLU8zW9IGkB5jNArCzu3uqiQw0lDlPVw2xZpB7oxhu5h0zXSKa9gqpE0Gwvw54kpmpn07i3r1+E6iJwzkeVUspnFEiLjAtYKj7zqhExncuc97PAvLwEtWLYs89eCWi6G1mzhtgEvBYVOyqEnKujmT5o5HqDFAidr3AM+X36mU9HzURqZy7qpq87CQ6xnGdSrRn5SXEm5vusvZxVMS5M+q2B5R6yLynieMnpS4OonQrfn0pnfkyRFt7T17NIbWHc1\\\\\\\/410RrkFsrUkjgf4i0bPdi9pjPyUCQz8DtZyoVLEX7KB8pK8ANovHPkpGv6G2c8ApB5Kvm2o0XpfptyzNnrLlpXBkLxdfyxghGrZtO\\\\\\\/ChNrML701P9W5uppNEtxKz7kIGv0LnKZQSv9W26jEP2h0ZeV5TACzkzRo9mpPZhJfYX9foH4XR4k2X\\\\\\\/cgG1s7F6jjFOpPCt92ZpFMIImfoEJuxD78fDK3QyCETZsB8Plamhsk2AKRH2rlbQ6yXtMYxEDpGNHr0PBMgkj+W4Gi31YByrpTDJSr\\\\\\\/859gAwZJf78u718ZDOwyOq3hw2oHz7dNbdb13WQXHtX9KKX8s\\\\\\\/6oTBL3LIQ2cXbnagLC2qKzPZ0GSrflAQILn\\\\\\\/dc3bp4LHQ4ly+m+M0S5Th27p6S2duYqZhgUNpa0ldFKOW4lq0HU4aiYnhPbIM9GFqzfCfvZ\\\\\\\/SBsu1QCl8PFmVscbaCpeyBeDvFmYnBDQ9uzapRd5GFlSEgYzf\\\\\\\/ggdNBdWuk\\\\\\\/QV8iur8HGWMV\\\\\\\/LM3omFCdMimPmCRx3NhKkWFQfSln4zxn8HIvGYHw8wJjJ+BN6xQKLD\\\\\\\/CnI+rUl+YOrOEfAAy5aT9Gf+rgScEsPEd1W3FOXszz\\\\\\\/bSDyyWYUwhqXYb9z0xWE4r8CH\\\\\\\/T7cwsp1jluy2ex24zoUPqF9HzlRwbbs5yy4rpX+wEIlQmbU7mtbTwQKsJcpgG+fBUyiXPnCFep9ePDxE7m5v90JRFCAaGPxQzqoCZOoGwygBPLz5GxNwfX\\\\\\\/nJkPWPHCRF1FGiEuHhyNLfoRaZjYR74oWO+rmbAnojq7pTIanGdLMwJId6HrMYBsTppBfr521eyv6GMyz0RLlL\\\\\\\/4CoR\\\\\\\/5nFMCNy\\\\\\\/BfLYWV88013QdipJiu8kAGHeSbWmw0Oq6kTKC2kDSGG27gl8rWbUuQAFRe8qXdtfkWDW9rQJKorojuwZVTsv7Sfd+Xq5Tjg6IFNY7YZOMcSC14FZ+nfnWfNQ3u5rwSF2Adm+Bkkx7SV5G\\\\\\\/0VJ4c+eep0eUz4ptOkxfICtMnpQyL6TnONBalAgDvi3h8cZETy2tK\\\\\\\/exZ99wpXl9PpCB520haYMp+5s\\\\\\\/3K8g28H4xjqIlwdJ7AIwFMxSvSk+FaxS99BqXS4YAEqKPuN\\\\\\\/ZR0rawLrpbt96OApv5GTbKmsqwNSZ7jL4IyO4t6i0t\\\\\\\/QjiXTPblOHd73KP0ZNzAnHfWOmp24LigKikPCz1OanonkNAXIoJ5c7BIunrParBd+RzGx7UAPiMZ8cpaOAf4vHNwcQU8LgUEraZGNbFT2+xQJQejUwALA+5WXR8R2YMNQGTzclo65nhP7U04ozRTBwZglUvAtBTCv01HZsKfMbn2tIBIqgAZjgmcsH\\\\\\\/d2bohXDyBG0\\\\\\\/qmyhfVeX\\\\\\\/EnuiQDWkPONIpRez8Kpj\\\\\\\/T1VDcU1HuP4L94vT21+FSb0rYypgLnK24RTqVP1Pd\\\\\\\/WUpbvGNePgO0vfU\\\\\\\/HuOAf2Iq4+atd04Vaw4WyhQ3lxPxIbJ5d6x\\\\\\\/sjsiHwcpsYmwhAbuFCZrq41gzBzyHi+VZ4uLXe9AVHLfQZoxRQ6J8DcUTZ8exYLoy\\\\\\\/LWng0DjukK+iYucGoUri8rsXIuyGgtVTNpC4zkyP9KgHNYVprmk\\\\\\\/0MkZ8vx44g4TtSBZqgoAdjjTdcsPcRDj3JYuDgvzQa6W\\\\\\\/G5slP2MuwIQ\\\\\\\/FRY7hvDFFlR3BFTb+C4ZWC\\\\\\\/Z2vo87QicQZjtUXqKItEQFFcPPV0LBOrEoBYMb\\\\\\\/VYDWAIp6EULYa67IOMY2JewiDVVikETLC0lSs+GqA0513w3fUKGv4h+Upg8qSJZF0mgPYm1W+OlQVPLjOO1Xe95LAUnWBHYZXbXspgtGvVN0oW6GVdH73brPhsmpOqpdEf7MfjYzctdnO3km1E7bCnfqXeg2VZZ795ez8BeuX2O\\\\\\\/ymb9kJaEPpg7X0inzDj1Qgw4RXMub9PeYJp2PkBqOeRVrxuEry82+PkYtAW2elyhffL\\\\\\\/tzJ\\\\\\\/NHuyxvjMXP7U5xy4rYfWNGvcqD5r89SyZK2cdxlpWzy1UAQ3DGPaJKOYgEj0ZHJ0ubKqA\\\\\\\/opvGV7Z++08MFeBSSort280WK3ZjSBun0JMswDfmaidW1DhA\\\\\\\/nVgkfq6yamVvKpTCAHjUQ2Y8z7mW3DYRSYpPF7lpVMBq7oI6g6jtWKPKyWuclttZAw8Lkkcxduqtz1FUWiNkbasJYa587Xov9gzZP6op\\\\\\\/KvGZeN84N8rJaXpi7w3AEbbbbou9oK23svFfhuv1HolQc0qDVVwDRtcjcGEB6cmQYKqqqT3Gr5GtadvFxtubIYPx95j+ALWvR5sA3\\\\\\\/QPYu2seqqxXpg8gyzxO3evzVuNf4nSHOmgB3mIAREtI11qYrtNT4Ho4uEkgKrBuOQhAUn3cPqePjeG+4Ff5RaUZYGexrNxvRnazIdAc+cijet2tMZpY\\\\\\\/BiMJT4atvTeunH6p5GK8ZZF+68DuMsYfqeDz+whJIKJwX5Occsk9DvmEU8dxgbd1wZBNae9707Hv9Ptbv8KG9SNq4TVXCz4tU0OK\\\\\\\/JGHwsXQ4Q6EOO4ff8V963fQEtsmhYYs7lFcLGi1QoKdm6vQYeNBKmExzS2I8nspNhkzSNyi5xRQxu5ykFFQCmbstFjVqzyVk6aRYQDwKKxmUnmdIllc5SW10bsEX26B92flgW3alBJMivs7tsk2gl918wJFxxvSgVUAjqi1Po8R+SXERaCnQlTFlmfsFL23E5IGfHbd+tn7NwrZa4xYxRk2eWl3qxLhqyt+dwPPHUxpxnN71qCGTcfSLXgU6huhuxaJTa+GDV6RaNafSaGPr177VxdQME+\\\\\\\/w+\\\\\\\/D5\\\\\\\/esDJv8mjQ1wO\\\\\\\/M+MknDxfHHp2HKFQxnqDk10JrGJ8AnkOzbO3knQy70oM3OFoIAvRlmuAd7RlFxezCGaTdOUYHppaZpHFf5rKfgE86yhGeC8AbmqyXxie\\\\\\\/cfgVi9LMSZOEy4w\\\\\\\/H19XGR53QnZJcHhVFF2gHfr0ZiTyd4IAHmV0OiJ63vXHXSXoYyIya33hzImM1AeG37QlZUTkqnCEmRVXBDP5+vTS0IMGH5XVvJK5cLESg5ek0pvCZC\\\\\\\/AHcbaWIybO9ijlU5oVqYGYletecSxirhGQm+crJgKzGSuSTrZbC+MS16yTESmqCD3OK6iT\\\\\\\/kPscP3IepFk7h5iMIu8NFnNoRZZdhTLpy3Qot\\\\\\\/HxxEx86P\\\\\\\/oytAMV05NmLja3JDMgs3t1I\\\\\\\/\\\\\\\/fOgIqJZ7RcVf1jUk2fO500YDHEURpKsspbJIBrJHS6AU+ezpCc6BjTr27rGz369iv5kr1yVzUwlRpRnrHgoBZFrOgQjIuUt6PWlP16epnM0gTb3n2x16012tT1eZifUfPVuz3fRfpxkCaLrAqFBX7wfHOx210K0KMOuoVgYm4VopRz\\\\\\\/3thSsrAguVtQQT6lI\\\\\\\/xpfNkzYFbbHFgzqu2M\\\\\\\/W4sTy59VcI32ZbpEzg9Okq\\\\\\\/N77qk2wflteVwGuT4yx9isSo46bzUlIDe6r5dzwpwhD4770IHL2DSG58R5Hak6pyN8H7z\\\\\\\/szOGUQpCVzqBl7XFso7QKgM+Ku+RFgOZqKzjbsGDDywEUhVIy2uLmPR2JbrzyzYktjQQSPZGGPPW\\\\\\\/P6ebsh45jEe+PU3y3gOh1AvrucrFInSxb87xiOxzwGFDwu0UZBOS7Un23WN5py9DPQnuqwcGgvC3oeNeAyiHTZrRPtQkEfOs1rGuP7OyPDHTXE1h4PmRl\\\\\\\/ynTUU2NW2XY28paGvOeKUxf9aiAVmHyNuxIAbt30qxtSjZqDYEwxhzzbTPskFeFwtElt8U\\\\\\\/nfAnIDkppsk4cUyS7gc8zeW0gAvh9lHMs2A3PF+h+Y\\\\\\\/KppOGv90OeHt37+iKbCMM5zJuUp10h6j6vauEGjmhc\\\\\\\/iFZdz7Ybm\\\\\\\/MUkJLBcDcki8CMrjwS\\\\\\\/6PZSpdKBDCEpdpWtby4veKaBzuHaB0NA\\\\\\\/EGSlP9u1psOGz30XHqbf4Ro5Le0lTsdhX9y9p9q3LtrvVYx9yOy303t5ja4ScTosfVEe+PXrOkyq1AweycBADKy0HqHYcUppTZWiWLxWdRuoA2vmgWw2x5yZ\\\\\\\/Xx8Kh1Yuv8dRmrLpQlrd1UtJCW6ptnY3T5aN4OVUGrOqQvmhxON9nqJo31C9sCMMzn7kTwnawY7gNtOlku8JFusFPHVp5WSAimcT0r+dlJBEnHFrmhXQCkVdfTkrTGMGBKJ8jjh2EajQkuhfonvbiuh8fxtF2EZcAWvra9rj2bEruTut+v\\\\\\\/e\\\\\\\/JFwZlptrkN\\\\\\\/aWk0p2cgNOKcU+d\\\\\\\/drRuUMPfK7nZcKzbGeNxJFKbmRIyG8aS2yL929p34SnqCKt0+d0z7PEnLmlNAEot\\\\\\\/JPRREY5s7I\\\\\\\/j2ER6vj3tVvQCm0Li9QwpqdWD5YEFBZB2tPqHBDp5dkyEbNJCBnpuabiRSD07C6peSdYiC6vOm8Sl\\\\\\\/XMOJKJgNjFkl4Ja1m2Eabq+Chyia5jwbn5rjmCfn+iOczpY0VohLaQaZuLmgBcoWkSx\\\\\\\/7Zf68NMXaVOm0YZLUM63MnGmVgbIVWAcz3UttudT\\\\\\\/d2yzoA2wKY+zIRY2tL1Pol6ZjgdyiaiM\\\\\\\/W+bTO\\\\\\\/tzWtijTCYIsv9sy745cMZtJ3mcg2EYyLEkFC1gJ1c4Wx5niMtTjFc0V5LSPHsOy9pFV9tdCQAbh\\\\\\\/EMZc8Q6y49GRL25KX9SxXZRIkfcXimTKWpEQQqOs9sg3XkAhcKwP8dlXfTI5eZtT4I9AGSH7YxSgv7VBp3UoCA8lNqKo1tMZKFe1vyDdSIlSZ49Gf0Q0hTFuq5SFBvOaNVSL868i0MuyaBrpiQyINinbV4cqGgxFjKiMgJXgnTTC2U3Vf48CiDmxgl2eB9QWzIQSrk\\\\\\\/Q4CZz1PsPJ5QbprdM+UC8xg1Od95fygQ0DHqDxoudOVKvipWCqGBjnreULw4xz\\\\\\\/faiXMZeBhiLhTQlBqvV1gf3JOTQB36wH8Y8SiW9U2uWCgjN2bLLbBA4UIJ23c8SWPvwlmruYaV4e56mdHzPejaApnBebh\\\\\\\/19z4tEHXFsXUYy7deyw2UY6ojsJ9U\\\\\\\/Abv5YEmO9Ng3gF2\\\\\\\/IJcTczmVbMokXiAEgwaHIfz+tRyHVJQ2ELMP1mBxa9HLE\\\\\\\/sk\\\\\\\/\\\\\\\/Gn2OFve1m3VlV0XZzgU++4VahMnspyzCWKvYREpw17ymoFlaUQd58QcXxLb2Hwx\\\\\\\/Am2NkAST5J9IVbFoNiJaXEeXmaIx+V\\\\\\\/f1WrgcJkocGOWx8uyz8DEaGWkF\\\\\\\/CCSM1r35ctegjv3Q2JNP0O+iI7dKr639m0ezxcdqgLW1MKtU8Gfv6rfR9mEDNvYs6hsQro\\\\\\\/P3DPua+A5ObnXsJe0LuSkzwUClb6oiW40jW0urg70Dzw\\\\\\\/b0UNKtwYZXz8O6V3SEjVzKy2I5srga88cc7F45xl1iVehsssbaa3ksX27mfswy1k8O4RI9ozaJz+MvRMuYJAWPpxaav190SSV5R2KkXbEHVEU8dzKsW3g+il6gdOu70POZiRjOPnS9NZD4NOOVB4VclHTtDZQLWwcHkgg+Q2dkzONgevn\\\\\\\/DRdNGvzZTqpL25PRFhgOVVmh4cBbuI+td\\\\\\\/D5HBjoRABz9e5lw0gBo+1os7F0iYsbbtHKvUyOrKiT0h+8W6qoxBn8UNgYmHwBkofjZQQSpOAcvmqcP0ckySxTo+l8z7GdCe5SvX9SstxPwF0bKHQSGd0iVzYoKwedg6EHqFzJG\\\\\\\/bObrFT3Gog1+U5LPLqo+ftfO9LcQ6gh2IT+7nyD61YlOKQFE7+JXTK0opuqWr7SO+VqQAzAj1IR2WDKTw4yFl9f0raPGFu0qiwY7DZ+QPRe19Zcz+bD3sfW7dYqPkIy0xjYaSUmNS1BhJwz2pcBshv9z7aNhtoBd7CV\\\\\\\/LV\\\\\\\/OUvNFwB9\\\u0022,\\\u0022iv\\\u0022:\\\u0022SU\\\\\\\/mv0BgWiFjkrCujFaXlw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac674286259f5fc5e0640ab9f1705c617a8f4508c930d803205d06e447b099eb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YJnr2CQCEgDA4qwiTqAo4zHAVilueRkbdZZo\\\\\\\/XLKTAZ5afUdJoGlRuvJZQj8TWJd0IwyHjl\\\\\\\/9QxTcv517Qbaox46aqz2E9apC78JLrW9IQuyJ85UhtzfUAn5+oOokVjFEC348vrks2kC+ms6xa1lNgv0TIgsPbWQVmuDnJtaEUZQunAoxHaTolDwlZut9sTh2KpghAXegrY5HtEyYyZyNeWU7Vxwlzwivb8aCWNXSoEojfL2mbOiD\\\\\\\/\\\\\\\/Rrqs+pR4cW66BKJnkuoOUXIWwk\\\\\\\/DugMhNc6a9nniBsOxDMcEiQgzR7PR70bQyRQ8\\\\\\\/9YkSFHAVFthmReZ62MdNWx7PNtFPMLKNfAWLwjVeesxQ+3BW5HPbgcOGqgDisrSJuRWM1KOSY+qDqNEPOfWXy43zJEpJ9M9QuYkEBC7D0WWM89z1Uy8Miuke6aguWk+WStkAtB5rLELkJz0HghPma8eE62ysSs9QLpo4kC45iFJhTcWS9hlK34HaWvAfCIw4ywiKvX3luivm0szucAlRn8+gvMt+LDChADBOiMrq\\\\\\\/rq3ztJO4V7m+nm8nj2iaq\\\\\\\/tONHdlTZiVrbyAr3aJFZYV3Cw6+0GNLV5dgCeIl5iS4JRIOD\\\\\\\/5u\\\\\\\/Zd9ObiuRaVxg9rBm49BEtyun3Q5ObRGsQ9nHK2CNqLp4FQMZk22dD5LysE1qqVH\\\\\\\/y1xChTHqxjAHYX7yTsRd3VH11bKwtTT1U9iHCoyfsSXStImrz\\\\\\\/UEINFL9JFtrSZvjSqXFl\\\\\\\/wYon0KfEH8ZwRjzygjB+OZhmX5rHDTVsdsAuM1rPXO3MTgxBvPYhasuL37964iVLW0kKt8yB72rJVBZQfa+\\\\\\\/mHsCGsz2bX8QfBo14K4iWmtWdqDLetSLeKXMDPMr58\\\\\\\/gDwErzhvYK2uHjdQoJb5dab4ajLzVKzE63yONIjTSTvdEN\\\\\\\/36dtapP7jt+2Vqk\\\\\\\/n+BhCnv5TGDAt\\\\\\\/40nRmhxEW3MZxSp\\\\\\\/MrJM7AMsp7C\\\\\\\/FK4E4uiKu5jT7LK5f+jRDNV06olaU8AfsyBO4z7ycrh4ExbbiKGLoBQzpQ2tlwpjOHN3As+v0sccYYSbsVfg5Rbcacdr5op8eypJw2C+sEXEY2LhwIcIHXp1cA\\\\\\\/E9cTWbnRdza4KyiQ5LYqy0FvFylaFl5pogtbb0QY0\\\\\\\/9Q9KNPqpT2oiaa5WTwJRVSC5as3Qp+sktoUEWmz5J2\\\\\\\/J9VgItYmJfNsuCnlRUs8QNNY8CYKrxPG4E0p51kihd3lyYpbP1lCoRIUMCpDDeb693TNiRspBdGLUik+e40kpgXJrzCOCvCvqtz2vCw0rT8xz7fuhw2WhFIOcNo7y7+tyCPs7ULjxTCR2PdkXChydhAk8KP95OfMSvmuf0JDGRTHCn+LTMMsBYV4OjEk49fQ3DtRjYbQ2ckOCvMmG7L5\\\\\\\/8qWpnVLKXYUUKOG2iK7Tx6Wa50WMFKZ9jy4zb7LcrHmE2iovz8cMUuFlgqOBUK8MgjsutE3dcmTp8ALR1eG8EgSpRl29v7wCVPD7qFtFqyMjelTf3ms+6jagpdnSnPC6rhmBq0X\\\\\\\/gBGoXuu49PZaV4o0kpaPEk5eC\\\\\\\/W+ft8A6teGXWOzHTZxDES6ogFBMtVn\\\\\\\/nkfYfDAO20I3pjqY\\\\\\\/7M+7d0C4B7ScJvwlwnpeuky+bt7061COPrIc1c0AmMVHtlVaTcrbHzwKTCIcaZrkL6ItPM0khbXMIoazRCk9qFLfz70wBguh3Y1z0uxZh\\\\\\\/ITt\\\\\\\/Cs+PVXpRdUGwpnl6EWXCNGlggyC6Go\\\\\\\/EIPXz0f0pvRSs9+0f+WXh8cQZJ+cYmyNxiEAL2IeYAWWoeraXJbn3QsVV9KAMsqOr9n0VqagB0SCdOLClEOhxy8uBTZtbKB\\\\\\\/Nb61JEo7g3YYs1ZkQ94unAsH\\\\\\\/7YHiUZEZloFh680ZNKDflKjpr8ygq2P8ykJ5SYmRqpZNRtI9FL1zw+to4GJUeZ8Dad4poTbaUxkiDGCLdVbWjxviYH\\\\\\\/7IUzfCsLejFVM414SK9hEoqUZ3UhnWN3RSNA6sYRfS20RvPWBTQSExYZWWS\\\\\\\/8LMqrRoT+5nIEJNeCf\\\u0022,\\\u0022iv\\\u0022:\\\u0022y7\\\\\\\/qNAK5ej0HpP63b3HMPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022359fbf52597f96b12bc3eb7d5eb19dde3c237325fac475d95f9406b4a3421444\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459644 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/433" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rMxZcKncQ8qNth41yzB5MRldgH2fCjSGfcgp1P6lv5g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/433" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JjU9F2tYMtsgTrm-BqN-ps1uO9VSr31QYiW07jmL3GE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/425" title="Double-Strand Break Repair Assays Determine Pathway Choice and Structure of Gene Conversion Events in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/447" title="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459644" id="node459644" data-pisa="ggg;4/3/433" data-pisa-master="ggg;g3.113.008722" data-apath="/ggg/4/3/433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mikhail Spivakov</span>, <span class="highwire-citation-author" data-delta="1">Thomas O. Auer</span>, <span class="highwire-citation-author" data-delta="2">Ravindra Peravali</span>, <span class="highwire-citation-author" data-delta="3">Ian Dunham</span>, <span class="highwire-citation-author" data-delta="4">Dirk Dolle</span>, <span class="highwire-citation-author" data-delta="5">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="6">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="7">Tomoyuki Aizu</span>, <span class="highwire-citation-author" data-delta="8">Yohei Minakuchi</span>, <span class="highwire-citation-author" data-delta="9">Felix Loosli</span>, <span class="highwire-citation-author" data-delta="10">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="11">Ewan Birney</span> and <span class="highwire-citation-author" data-delta="12">Joachim Wittbrodt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">433-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008722 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459644" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mikhail Spivakov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikhail%2BSpivakov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivakov%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikhail%2BSpivakov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas O. Auer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BO.%2BAuer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Auer%20TO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BO.%2BAuer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ravindra Peravali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravindra%2BPeravali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peravali%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavindra%2BPeravali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ian Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dirk Dolle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk%2BDolle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk%2BDolle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tomoyuki Aizu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BAizu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aizu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BAizu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yohei Minakuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yohei%2BMinakuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minakuchi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYohei%2BMinakuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Felix Loosli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Felix%2BLoosli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loosli%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFelix%2BLoosli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Institute for Basic Biology, Laboratory of Bioresources, Okazaki, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ewan Birney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewan%2BBirney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birney%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEwan%2BBirney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joachim Wittbrodt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BWittbrodt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittbrodt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BWittbrodt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/3/433" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459644" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459644/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/433.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459644" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459644/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/433.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459644" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459644/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/3/433.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459644" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459644/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/3/433.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459644" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459644/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6"><em>Oryzias latipes</em> (medaka) has been established as a vertebrate genetic model for more than a century and recently has been rediscovered outside its native Japan. The power of new sequencing methods now makes it possible to reinvigorate medaka genetics, in particular by establishing a near-isogenic panel derived from a single wild population. Here we characterize the genomes of wild medaka catches obtained from a single Southern Japanese population in Kiyosu as a precursor for the establishment of a near-isogenic panel of wild lines. The population is free of significant detrimental population structure and has advantageous linkage disequilibrium properties suitable for the establishment of the proposed panel. Analysis of morphometric traits in five representative inbred strains suggests phenotypic mapping will be feasible in the panel. In addition, high-throughput genome sequencing of these medaka strains confirms their evolutionary relationships on lines of geographic separation and provides further evidence that there has been little significant interbreeding between the Southern and Northern medaka population since the Southern/Northern population split. The sequence data suggest that the Southern Japanese medaka existed as a larger older population that went through a relatively recent bottleneck approximately 10,000 years ago. In addition, we detect patterns of recent positive selection in the Southern population. These data indicate that the genetic structure of the Kiyosu medaka samples is suitable for the establishment of a vertebrate near-isogenic panel and therefore inbreeding of 200 lines based on this population has commenced. Progress of this project can be tracked at <a href="http://www.ebi.ac.uk/birney-srv/medaka-ref-panel">http://www.ebi.ac.uk/birney-srv/medaka-ref-panel</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/medaka" class="hw-term hw-article-keyword hw-article-keyword-medaka" rel="nofollow">Medaka</a></li><li class="kwd"><a href="/keyword/inbreeding" class="hw-term hw-article-keyword hw-article-keyword-inbreeding" rel="nofollow">inbreeding</a></li><li class="kwd"><a href="/keyword/population-genomics" class="hw-term hw-article-keyword hw-article-keyword-population-genomics" rel="nofollow">population genomics</a></li><li class="kwd"><a href="/keyword/strain-specific-features" class="hw-term hw-article-keyword hw-article-keyword-strain-specific-features" rel="nofollow">strain specific features</a></li></ul><p id="p-7">Defined genetic reference panels of inbred lines with divergent genotypes often have been exploited in genetics (reviewed in <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Flint and Mackay 2009</a>). Broadly there are two approaches to creating such panels. The first involves crossing a small number of genetically distinct founders, followed by a number of interbreeding steps leading to an outcrossed population (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey 1971</a>). The outcrossed population is then inbred again to form recombinant inbred lines. The second approach involves capturing wild individuals from a population and inbreeding them to give near-isogenic wild lines. Both methods result in a panel of inbred lines that has specific benefits for genetic mapping compared with the use of outbred individuals. First, as the same genotype can be produced multiple times from the panel, genotypic and phenotypic resources can be shared on the same panel between research groups and experiments. Second, the ability to make measurements from different individuals of the same genotype can overcome much of the nongenotypic study variance. Finally, the environment can be systematically varied between many individuals of the same genotype, allowing dissection of interactions between genotype and environment, something that is not achievable in standard outbred populations. Recombinant inbred lines provide a more straightforward route to full characterization of the complete genetic components of a population, often relating phenotypes back to the founder strains. However, mapping resolution is restricted by the low number of recombinations available in the panel, and the overall diversity of the panel is limited by the diversity of the input founders, which can limit analysis of interesting traits. In contrast, near-isogenic wild lines usually have both greater diversity of alleles (ideally with a consequent diversity in phenotypes) and have far better recombination patterns, allowing for the discovery of more genetic influences and finer mapping of these loci. With the falling price of sequencing, determining the complete genotype for each line is no longer so onerous, leaving the inbreeding process itself and obtaining sufficient distinct lines to overcome multiple testing issues as the two major limitations of a near-isogenic panel.</p><p id="p-8">Recombinant inbred lines or near-isogenic lines have been developed in many different species during the last 50 years. For example, in the maize genetics community the set of 302 IBM maize strains has been a cornerstone of both basic and applied research (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sharopova <em>et al.</em> 2002</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fu <em>et al.</em> 2006</a>). In the <em>Arabidopsis</em> community, the collection of 107 different wild accessions has allowed the exploration of the genetic determinants of a number of phenotypes and their relationship to the environment (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atwell <em>et al.</em> 2010</a>). The development in <em>Drosophila</em> of both recombinant inbred lines (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">King <em>et al.</em> 2012</a>) (&gt;1700 lines) and near-isogenic wild lines (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mackay <em>et al.</em> 2012</a>) allows the genetic dissection of phenotypes coupled with the excellent transgenic and other resources in this organism. The yeast research community have used crosses between wild and laboratory strains (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bloom <em>et al.</em> 2013</a>), or surveys of wild species in related yeasts (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Liti <em>et al.</em> 2009</a>) to explore genotype to phenotype associations. In vertebrates, the emphasis has been more on recombinant inbred lines. These include the BNxSHR cross in rats (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pravenec <em>et al.</em> 1989</a>) and the Black6/DBA cross in mouse (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Peirce <em>et al.</em> 2004</a>), both of which lead to a number of interesting traits being mapped in these species. The Mouse Collaborative Cross is the largest recombinant inbred line experiment undertaken in vertebrates (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Collaborative Cross Consortium 2012</a>) and is already showing promising results, although the mapping resolution will remain in the megabase range. So far the long generation times and difficulty in laboratory husbandry of wild individuals has prevented, to our knowledge, the establishment of a near-isogenic panel from the wild in any vertebrate species.</p><p id="p-9">During the last decade, the model vertebrate medaka (<em>Oryzias latipes</em>) has been rediscovered beyond Japan for its developmental genetics, genomics, and evolutionary biology (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wittbrodt <em>et al.</em> 2002</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Takeda and Shimada 2010</a>). The physiology, embryology, and genetics of medaka have been extensively studied for the past 100 years. The long history of medaka research and its amenability to inbreeding make this species very well suited for genetic studies and especially for establishing a reference panel of inbred lines. A large number of wild catches have been collected to establish laboratory strains and highly inbred lines, which form a unique repository for genomic and population genetic studies. Because of this and the easily accessible habitat of medaka, it is possible to collect, analyze, and evaluate new wild catches and establish newly inbred strains.</p><p id="p-10">From 1913 onwards, medaka was used to show Mendelian inheritance in vertebrates and in 1921 it was the first vertebrate in which crossing over between the X and Y chromosomes was detected (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Toyama 1916</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aida 1921</a>). In Japan there are two divergent wild populations of medaka separated by the Japanese Alps dividing the main island of Honshu (the Northern and Southern populations, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ishikawa <em>et al.</em> 1999</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Takehana <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Setiamarga <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asai <em>et al.</em> 2011</a>). These two populations are not in sympatry (<em>i.e.</em>, do not overlap in the wild) and have many different phenotypic features; they can, however, produce fertile offspring when mated in the laboratory (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kimura <em>et al.</em> 2007</a>). A critical feature of medaka laboratory husbandry has been the routine inbreeding of wild individuals from the Southern medaka population to isogenic strains pioneered by Hyodo-Taguchi in the 1980s (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hyodo-Taguchi 1980</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">1990</a>). Some of these strains are now in their 80th brother-sister mating, and importantly, there are routine protocols for creating an inbred strain from the wild. At least eight isogenic strains derived from single wild catches are available from the medaka NBRP stock center (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sasado <em>et al.</em> 2010</a>). Furthermore, the availability of standard transgenesis protocols (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rembold <em>et al.</em> 2006</a>), mutant lines (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Furutani-Seiki <em>et al.</em> 2004</a>), a 700-Mb reference genome sequence combined with a detailed linkage map (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kasahara <em>et al.</em> 2007</a>), and tools for enhancer and chromatin analysis (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sasaki <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mongin <em>et al.</em> 2011</a>) make medaka a powerful vertebrate organism for developmental and molecular studies (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">The Encode Project Consortium 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographical localization of the sampling sites for inbred strains and the Kiyosu population. (A) Medaka consists of four major populations in East Asia. The Nilan strain comes from Ilan-City, Taiwan, and is a representative of the Chinese/West Korean population; the HSOK strain is from Sokcho-City, Gangweon-do, Korea, and is a member of the East Korean population. The Japanese populations are divided by the Japanese Alps into Northern and Southern strains. Whereas Kaga and HNI belong to the Northern populations, HdrR is the Southern reference strain that was used for the Medaka genome sequencing project (Kasahara et al., 2007). In red is the site in Kiyosu where the founders for the inbreeding panel were sampled (see Materials and Methods for the GPS location). (B) Representative pictures of lateral and dorsal views of 10 days&#x2019; postfertilization-old larvae of the different Japanese inbred strains. Icab, HNCMH2, and HO5 are further Southern Japanese inbred strains whose original sampling site is not shown in (A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Geographical localization of the sampling sites for inbred strains and the Kiyosu population. (A) Medaka consists of four major populations in East Asia. The Nilan strain comes from Ilan-City, Taiwan, and is a representative of the Chinese/West Korean population; the HSOK strain is from Sokcho-City, Gangweon-do, Korea, and is a member of the East Korean population. The Japanese populations are divided by the Japanese Alps into Northern and Southern strains. Whereas Kaga and HNI belong to the Northern populations, HdrR is the Southern reference strain that was used for the Medaka genome sequencing project (Kasahara et al., 2007). In red is the site in Kiyosu where the founders for the inbreeding panel were sampled (see &lt;em&gt;Materials and Methods&lt;/em&gt; for the GPS location). (B) Representative pictures of lateral and dorsal views of 10 days&#x2019; postfertilization-old larvae of the different Japanese inbred strains. Icab, HNCMH2, and HO5 are further Southern Japanese inbred strains whose original sampling site is not shown in (A).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F1.medium.gif" width="280" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/3/433/F1.medium.gif" width="280" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464025" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Geographical localization of the sampling sites for inbred strains and the Kiyosu population. (A) Medaka consists of four major populations in East Asia. The Nilan strain comes from Ilan-City, Taiwan, and is a representative of the Chinese/West Korean population; the HSOK strain is from Sokcho-City, Gangweon-do, Korea, and is a member of the East Korean population. The Japanese populations are divided by the Japanese Alps into Northern and Southern strains. Whereas Kaga and HNI belong to the Northern populations, HdrR is the Southern reference strain that was used for the Medaka genome sequencing project (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kasahara et al., 2007</a>). In red is the site in Kiyosu where the founders for the inbreeding panel were sampled (see <em>Materials and Methods</em> for the GPS location). (B) Representative pictures of lateral and dorsal views of 10 daysâ postfertilization-old larvae of the different Japanese inbred strains. Icab, HNCMH2, and HO5 are further Southern Japanese inbred strains whose original sampling site is not shown in (A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Here, we characterize molecular, genetic, and phenotypic variation in a single Southern Japanese population of medaka to assess its suitability to found a near-isogenic wild panel. For comparison, we characterize a number of existing inbred strains, including the âreferenceâ HdrR strain, and explore the population genomics of this species. In addition, we set the groundwork for routine high-throughput phenotyping of medaka strains to quantify morphometric features, which is one of many possible traits that can be measured in this vertebrate. On the basis of these results, we have begun the inbreeding cycles of 200 lines for establishment of a medaka inbred isogenic panel.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fish stocks and husbandry</h3><div id="sec-3" class="subsection"><h4>National Institute for Basic Biology (NIBB), Japan:</h4><p id="p-13">The Kiyosu population was sampled at a location with the following GPS coordinates: 34.780885Â° N, 137.347818Â° E (Kiyosu, Toyohashi, Aichi Prefecture). The Takashi Hongo population (showing mitotypes indicative of a human-mediated dispersal and not included in further analyses) was sampled 8.4 km (5.2 miles) to the southeast at 34.715135Â° N, 137.392623Â° E. The HdrR inbred line was generated from the original d-rR strain (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yamamoto 1953</a>), which was established from commercially available orange-red and white varieties with unknown sampling sites. Phylogenetic analysis of the <em>cyt b</em> gene has suggested that HdrR belongs to the Southern Japanese population as has been confirmed here. The HNI inbred line is established from a wild population collected in Niigata city, Niigata prefecture, Japan. The Kaga inbred line is established from a wild population collected in Kaga, Ishikawa Prefecture, Japan. The HNI and Kaga lines belong to the Northern Japanese population. The HSOK inbred line is established from a wild strain of the West Korean population from Sokcho, Korea. The Nilan strain, which is not fully inbred (F16) is established from a wild population collected in Ilan, Taiwan, and belongs to the China-West Korean population. The precise GPS coordinates for each inbred line are not recorded. Classification of each inbred line to each population is according to <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Takehana <em>et al.</em> (2003</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">2004</a>). The numbers of brother-sister matings in the HdrR, HNI, Kaga, HSOK, and Nilan strains used for genome resequencing are F80, F70, F35, F30, and F16, respectively. The two additional inbred lines used in morphometric analyses, Icab and HNCMH2, were purchased from Carolina Biological Supply (Burlington, NC); these lines were derived from Southern Japanese populations through more than 20 generations of inbreeding. All inbred lines have been maintained by brother-sister mating. All procedures in these experiments were carried out by permission number 10A007 by the experimental animal committee at the NIBB.</p></div><div id="sec-4" class="subsection"><h4>Karslruhe Institute of Technology, Germany:</h4><p id="p-14">The following strains from the NIBB were kept as described previously under 14-hr light/10-hr dark conditions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Loosli <em>et al.</em> 2000</a>): HdrR, Icab, HNCMH2, HO5, Kaga, HNI, and Kiyosu. Animal husbandry and experimental procedures were performed in accordance with the German law on Animal Protection and were approved by Local Animal-Protection Committee (RegierungsprÃ¤sidium Karlsruhe, Az.35-9185.64).</p></div></div><div id="sec-5" class="subsection"><h3>Genomic DNA extraction</h3><p id="p-15">Genomic DNA was extracted from one male specimen of each inbred strain at the age of about 8 weeks using Phenol/Chloroform/Isomylalcohol and RNA was removed using RNase A (Fermentas). For analysis of the wild catches DNA was extracted from caudal fins.</p></div><div id="sec-6" class="subsection"><h3>Mitotyping</h3><p id="p-16">Polymerase chain reaction (PCR)-restriction fragment length polymorphism analysis of the mitochondrial cytochrome B gene was performed as described in <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Takehana <em>et al.</em> (2003)</a>. In brief, a 1241-bp segment including the complete cytochrome B gene was amplified, amplicons were digested with five restriction endonucleases (<em>Hae</em>III, <em>Mbo</em>I, <em>Msp</em>I, <em>Rsa</em>I, and <em>Taq</em>I), and mitotypes assigned by inspection.</p></div><div id="sec-7" class="subsection"><h3>D-loop sequencing</h3><p id="p-17">D-loop sequencing was performed as described in <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Katsumura <em>et al.</em> (2009)</a>. The 616-bp region of the mitochondrial D-loop was amplified from caudal fin clip genomic DNA by PCR using the following pair of primers (5â² to 3â²): Medaka_D-loop_F1: CCCAAAGCCAGGATTCTAA; Medaka_D-loop_R1: AACCCCCACGATTTTTGTC. Sequences were determined on both strands and then trimmed to 508 bp for comparison and aligned and analyzed with Geneious Pro 5.3.4 software (Biomatters).</p></div><div id="sec-8" class="subsection"><h3>Design and analysis of microsatellite markers</h3><p id="p-18">To identify potential microsatellite regions, we compared the HdrR and the HNI genome by using Sputnik (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Katsumura <em>et al.</em> 2009</a>) and identified regions with a maximal repeat unit length of 2 and the minimum length of simple sequence repeat set to 20. We designed primers with primer3plus (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Untergasser <em>et al.</em> 2007</a>) to amplify a region of about 200 bp flanking the microsatellite repeat. We searched for one marker per chromosome but finally settled on nine high-quality microsatellite markers. Alleles were annotated from chromatograms after PCR amplification with fluorescently labeled oligonucleotides using GeneMarker software (Softgenetics). For a complete list of primers, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-9" class="subsection"><h3>Inbreeding coefficient calculation</h3><p id="p-19">The Inbreeding coefficient F was calculated (in a single population) as F = 1 â (<em>HOBS</em> / HEXP) = (<em>HEXP</em> â HOBS) / HEXP, where HOBS is the observed heterozygosity and HEXP is the expected heterozygosity calculated on the assumption of random mating.</p></div><div id="sec-10" class="subsection"><h3>Short-read sequencing, reference assembly âpatching,â and single-nucleotide polymorphism (SNP) calling</h3><p id="p-20">Eight paired-end libraries (insert size 300 bp) and three mate-pair libraries (insert size 3K) from the HdrR reference strain were sequenced using an Illumina HiSeq machine to a median coverage of 144X. Reads were aligned using Bowtie2 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Langmead and Salzberg 2012</a>) and SNPs called using SAMtools (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2009</a>). Raw sequence reads and annotations were submitted to DDBJ (accession DRA000588). The HdrR reference genome sequence was âpatchedâ using base differences passing a quality threshold of 100 and with additional bases not defined in the reference (<em>i.e.</em>, originally listed as âNâ). However, when using this patched sequence as reference for calling SNPs in other inbred lines, we noticed that a small number of SNP calls (~0.14â0.2% total) were consistent with the base identified from the short-read HdrR sequence at quality scores below this threshold. The use of an unthresholded HdrR dataset as the reference resolved this problem, albeit resulting in a marginal addition of 0.01â0.017% SNPs that were consistent with the original reference. On the balance of the false-positive and false-negative rates estimated this way, we opted for using the unthresholded âpatchedâ reference for calling SNPs in all other strains. Furthermore, we supplemented the final thresholded patched sequence with 89,227 bases that were consistent with the other four inbred line sequences and submitted the final sequence as a Third-Party Annotation record to the European Nucleotide Archive (ENA; HF933207-HF933230). For 41 loci, SNPs were validated by direct sequencing after PCR amplification using the primers listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS5.pdf">Table S5</a>. Samples from the other inbred strains were sequenced with a combination of paired-end and mate-pair Illumina sequencing to a median coverage of 108-125X. Reads were aligned using Bowtie2 and SNPs called using SAMtools based on the patched assembly. Refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS6.pdf">Table S6</a> for details.</p><p id="p-21">A total of 24 wild individuals of the Southern population in eight mother-father-offspring trios were sequenced using paired-end sequencing to a median coverage of 9X. The sequence was aligned to the patched reference sequence and SNPs were called using SAMtools. Alleles segregating within the population were then phased and annotated taking into account family structure using TrioCaller (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2013</a>). SNPs were filtered by the empirical <em>r<sup>2</sup></em> between inferred and expected values (<em>r<sup>2</sup></em> â¥ 0.6) and the observed Mendelian error (ERATE â¤ 0.1). Raw reads were submitted to ENA (accession no. ERP001016).</p></div><div id="sec-11" class="subsection"><h3>Analysis of protein-coding mutations</h3><p id="p-22">Exon mappings for all regions mapping to chromosome scaffolds in the medaka 2005 reference assembly were obtained from Ensembl. SNPs were annotated as leading to either nonsense, stop-codon, missense, or synonymous substitutions (note that no stop codon mutations were in fact observed). Ambiguities in this annotation due to overlapping transcripts were resolved in this order of preference (<em>i.e.</em>, if a SNP produced a nonsense mutation in at least one overlapping transcript, it was annotated as such, and so on). Expected relative frequencies of each coding mutation type were estimated on the basis of all theoretically possible single-nucleotide substitutions at each codon and the observed frequencies of each codon in the medaka exome.</p></div><div id="sec-12" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-23">Phylogenetic relationships between the wild Southern population, five inbred strains, and stickleback were assessed using a neighbor-joining algorithm based on Kimura two-parameter distances implemented in PHYLYP, <em>i.e.</em>, the <em>PHYL</em>ogeny <em>I</em>nference <em>P</em>ackage (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Retief 2000</a>). Genotypes for the wild population were called using a âmajority voteâ across the 48 haplotypes. Differences between medaka samples and stickleback were too large to estimate distances. Hence, the length of the stickleback branch shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> is an underestimate. The tree was rooted using the midpoint method and plotted using the T-REX online tool (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boc <em>et al.</em> 2012</a>). The relationships between the wild samples and the reference strain (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>) were analyzed in R using a hierarchical clustering with Gower distances (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Struyf <em>et al.</em> 1997</a>) based on the SAMtools genotypes.</p></div><div id="sec-13" class="subsection"><h3>Estimation of linkage disequilibrium (LD)</h3><p id="p-24">LD (<em>r<sup>2</sup></em>) was computed from the TrioCaller SNPs from the 16 wild Southern trio founders as well as in 16 parents from human trios sequenced and annotated by Complete Genomics (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Drmanac <em>et al.</em> 2010</a>). Computations were performed using VCFtools (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Danecek <em>et al.</em> 2011</a>) with the following options: âld-window-bp 50000, âmax-alleles 2, âmin-alleles 2, âmin-r2 0.001, âgeno 0.8. Haplotype blocks in the 16 medaka founders were called using HaploView (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2005</a>) using the default parameters for the method described by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gabriel <em>et al.</em> (2002)</a>. Trials involving varying the minor allele frequency cut-off and the fraction of informative markers required to be in strong LD indicated that block size was relatively stable to these parameters (data not shown).</p></div><div id="sec-14" class="subsection"><h3>Population size history analysis</h3><p id="p-25">To estimate changes in population size over time, we used the method of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li and Durbin (2011)</a>, implemented in the software package psmc. The following options were used with the psmc executable: -N25 -t15 -r5 -p â4+25*2+4+6,â ensuring that the predicted number of recombinations occurring in each specified time interval were sufficiently high (&gt;10) to prevent overfitting. The output was scaled assuming a doubling time of 0.67 years per generation and a mutation rate of 2.5 Ã 10<sup>â8</sup>. Results for all individuals were combined by binning the results into log<sub>10</sub> time intervals of length 0.1.</p></div><div id="sec-15" class="subsection"><h3>Detection of signals of recent positive selection</h3><p id="p-26">A method proposed by <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Voight <em>et al.</em> (2006)</a> and implemented in the R package rehh (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gautier and Vitalis 2012</a>) was used. The standardized output statistic (integrated haplotype score, iHS) was centered around zero, as expected, and was filtered using the following criteria to identify robust and consistent signals: (1) |iHS| â¥ 0.1 (corresponding to the top 0.05% of the distribution); (2) within a 20-kb window surrounding each SNP: (a) standard deviation (iHS) â¤ 0.34 across all SNPs (bottom 50% of the distribution), (b) the number of SNPs with an iHS of at least 80% max(|iHS|) is at least 4, and (c) the number of SNPs with an iHS of at least 80% max(|iHS|) constitutes at least 10% of all SNPs in the window. Thus, clusters of SNPs with highly negative iHS values (indicative of derived alleles under recent positive selection) are detected. None of the positive iHS values (potentially corresponding to hitchhiker alleles or a âswitchedâ directionality of selection (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Voight <em>et al.</em> 2006</a>) passed this filter.</p></div><div id="sec-16" class="subsection"><h3>Phenotyping of morphometric features</h3><p id="p-27">Morphometric features of four Southern (HdrR, Icab, HNCMH2, HO5), two Northern (Kaga, HNI) medaka lines, and the F1 offspring of two Kiyosu brother-sister mating pairs (<em>i.e.</em>, one generation of brother-sister inbreeding) were analyzed. All embryos were raised under identical environmental conditions. The embryos were imaged at 10 days post fertilization by mounting them in 85% glycerol. Imaging was performed using a Leica MZ 16 FA stereomicroscope with a Planapo 1.0Ã objective and 20Ã zoom factor. To extract morphological features, a manual and an automated algorithm were developed. Although the manual algorithm allows the user to select features of interest, the automated algorithm automatically segments predefined landmarks. It is based on contour detection, morphological image filtering; and connected component labeling. The automated algorithm was implemented in MATLAB, and its accuracy was verified with the manual algorithm.</p></div><div id="sec-17" class="subsection"><h3>Assessment of heritability</h3><p id="p-28">Broad-sense heritability was calculated as the proportion of variance assigned to strain factors in an analysis of variance. The different morphometric measurements were divided by body length to remove the strongest source of variance. The ratios of the other measurements were analyzed using the <em>aov</em> function in R, and the proportion of variance explained determined as the square of the residuals of the strain factor over the total sum of squares. Between 50 and 100 individuals were analyzed in each strain.</p></div><div id="sec-18" class="subsection"><h3>Data submissions</h3><p id="p-29">Whole-genome sequencing of the 5 medaka inbred strains was submitted to the DNA Data Bank of Japan under accession DRA000588. Whole-genome sequencing of the eight medaka wild catch trios from the Kiyosu population was submitted to the ENA under accession ERP001016. The revised genome sequence was submitted to ENA as  accessions <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HF933207&amp;atom=%2Fggg%2F4%2F3%2F433.atom">HF933207</a>-<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HF933230.&amp;atom=%2Fggg%2F4%2F3%2F433.atom">HF933230.</a> The SNPs discovered are currently being submitted to the Single Nucleotide Polymorphism Database (dbSNP), and identifiers will be available.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><div id="sec-20" class="subsection"><h3>Refining the medaka reference genome sequence and genotyping the existing inbred lines</h3><p id="p-30">In preparation for the analysis of a wild population, we used high-throughput sequencing to assess the genetic diversity across the existing inbred strains, including the HdrR strain that was used to generate the medaka reference sequence.</p></div><div id="sec-21" class="subsection"><h3>Refining the medaka reference genome sequence by high-throughput sequencing</h3><p id="p-31">The current medaka reference assembly (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kasahara <em>et al.</em> 2007</a>) is based on Sanger sequencing of the HdrR inbred strain derived from a Southern Japanese population. To verify and refine the reference genome sequence, we applied paired-end high throughput short read sequencing to genomic DNA from the HdrR inbred line, to a median depth of 144Ã (see <em>Materials and Methods</em> for details). Consistent with the high quality expected from the reference sequence, we identified 184,318 single nucleotides (0.026% of bases) unresolved in the reference assembly (labeled as âNâ) and 102,432 single nucleotides (0.014% of bases) that were discrepant between the reference genome assembly and the newly obtained sequence above our quality threshold. Further investigation showed that revision of discrepant bases in the reference sequence toward the newly determined sequence resulted in 89% of revised sites matching all individuals in an independently sequenced wild Southern Japanese population (described in <em>Selecting and genotypically characterizing a founder population from Southern Japan</em>), with 96% matching at least one of the analyzed individuals in this population. In addition, revision of these discrepant positions resulted in a striking 2.5-fold increase in consistency with the presumed ancestral allele as determined by comparison to stickleback. Using direct sequencing after PCR amplification, we were able to confirm the new sequence for 67.5% (27) of a set of 40 discrepant bases, whereas the additional 32.5% (13/40) showed traces of both possible sequences and could not be disambiguated. These observations support the hypothesis that these divergent positions represent errors in the original reference sequence. The discrepancies span 490 genes and suggest 804 alternative amino acid sequences and 11 previously unmapped stop codons.</p><p id="p-32">Given the large depth of coverage of the newly obtained genome sequence, and the successful verification of a subset of bases in conflict, we accounted for revisions to the HdrR reference sequence based on the short read data in the subsequent analysis of all other samples in this study. Further refining the reference calls based on consistency with other inbred strains (sequenced as described in <em>Whole-genome genotypic characterization of existing medaka inbred lines</em>), we produced a revised reference assembly, which we submitted to the ENA as a Third-Party Annotation (accession nos. HF933207-HF933230; see <em>Materials and Methods</em> for more details).</p></div><div id="sec-22" class="subsection"><h3>Whole-genome genotypic characterization of existing medaka inbred lines</h3><p id="p-33">We next used high-throughput short read sequencing to characterize the genotypes of four additional inbred lines derived from three different locations: Northern Japan [HNI and Kaga (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hyodo-Taguchi 1980</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hyodo-Taguchi 1990</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Loosli <em>et al.</em> 2000</a>)], South Korea [HSOK (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Sasado <em>et al.</em> 2010</a>)], and Taiwan [Nilan (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Sasado <em>et al.</em> 2010</a>)]. We discovered 20,522,880 single-nucleotide variations within the four strains with divergence from the reference strain at 1.3â1.37% depending on the strain (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), confirming the highly polymorphic nature of medaka consistent with the broad geographical spread of its natural habitat (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Setiamarga <em>et al.</em> 2009</a>). The inbred lines are predominantly highly homozygous (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; 0.1% of divergent sites are heterozygous) with most inbred strains having heterozygosity levels around 100-fold lower (average around 3 Ã 10<sup>â5</sup>) than a wild catch (1.5 Ã 10<sup>â3</sup>) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and, with the exception of the Nilan strain and chromosome 1, there are only a few blocks of high heterozygosity consistent with a wild heterozygous origin. The Nilan strain is not as inbred as the others (heterozygosity at 2 Ã 10<sup>â4</sup>, more blocks of wild-like heterozygosity, 1% of divergent sites are heterozygous), as expected from far fewer cycles of inbreeding (Nilan, 16 inbreeding generations; Kaga, 35; HSOK, 30; HNI, 70). Chromosome 1 harbors the sex determination locus, and as the individuals sequenced for Nilan, Kaga, and HNI were male, the stretch of heterozygosity is centered on the known sex determination locus (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). We conclude that inbreeding in medaka can create highly homozygous individuals, with the expected exception of the sex chromosome.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464030/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464030/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequence variation in four medaka inbred strains and the Kiyosu wild population from Southern Japan compared with the reference HdrR strain</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Levels of heterozygosity in medaka inbred lines. Four plots showing regions of heterozygosity consistent with a wild origin in the HNI, HSOK, Kaga, and Nilan inbred strains. Black blocks show windows of heterozygosity &gt;7.1 &#xD7; 10&#x2212;4. At this threshold, 90% of the genome in the wild catches is classified as heterozygous. Gray blocks are assembly gaps. Chromosome 1 in medaka is the known location of the sex determination locus as indicated by the black arrow heads (Katsumura et al. 2009)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Levels of heterozygosity in medaka inbred lines. Four plots showing regions of heterozygosity consistent with a wild origin in the HNI, HSOK, Kaga, and Nilan inbred strains. Black blocks show windows of heterozygosity &gt;7.1 &#xD7; 10&lt;sup&gt;&#x2212;4&lt;/sup&gt;. At this threshold, 90% of the genome in the wild catches is classified as heterozygous. Gray blocks are assembly gaps. Chromosome 1 in medaka is the known location of the sex determination locus as indicated by the black arrow heads (Katsumura &lt;em&gt;et al.&lt;/em&gt; 2009).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F2.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/3/433/F2.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464176" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Levels of heterozygosity in medaka inbred lines. Four plots showing regions of heterozygosity consistent with a wild origin in the HNI, HSOK, Kaga, and Nilan inbred strains. Black blocks show windows of heterozygosity &gt;7.1 Ã 10<sup>â4</sup>. At this threshold, 90% of the genome in the wild catches is classified as heterozygous. Gray blocks are assembly gaps. Chromosome 1 in medaka is the known location of the sex determination locus as indicated by the black arrow heads (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Katsumura <em>et al.</em> 2009</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Of the 20,522,880 SNPs detected across the inbred lines, 529,009 were mapped to 16,925 of 17,442 medaka genes on chromosome contigs, of which 33.5% led to changes in amino acid sequence, including 175,856 missense mutations in 15,821 genes and 1682 nonsense mutations (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). As expected, missense and particularly nonsense mutations were strongly depleted in favor of synonymous SNPs compared with random expectation, presumably due to purifying selection: 33.2% (175,856) missense mutations <em>vs.</em> 70.8% expected at random and ~0.3% (1682) nonsense mutations <em>vs.</em> ~4.4% expected at random (binomial test <em>P</em> &lt; 1 Ã 10<sup>â300</sup> in both cases; expected values were estimated accounting for codon frequencies in the medaka exome). Of these nonsense mutations, we confirmed 15 of 16 selected loci within the Kaga, HNI, and Nilan strains by targeted PCR and sequencing. In addition, we validated 9 of 10 noncoding Kaga strain SNPs, confirming the high reliability of our sequence data. The phylogenetic relationship between the strains based on the sequencing data (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>) was consistent with the geographical localization of the sampling sites of the inbred strains (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>) and earlier data based on a limited number of SNPs (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Kasahara <em>et al.</em> 2007</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships of medaka inbred strains and the Kiyosu population. (A) Phylogenetic relationships between the wild Southern population, five inbred strains, and stickleback based on high-throughput genome sequencing. PHYLIP clustering of inbred strains using a neighbor-joining algorithm based on Kimura two-parameter distances is shown. Wild Southern is a synthetic sample based on the sequencing of the Kiyosu population and determining bases using majority vote across 48 haplotypes. Differences between medaka samples and stickleback were too large to estimate distances. (B) Hierarchical clustering with Gower distances of genotypes of Kiyosu population founder samples, and representative Southern and Northern strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic relationships of medaka inbred strains and the Kiyosu population. (A) Phylogenetic relationships between the wild Southern population, five inbred strains, and stickleback based on high-throughput genome sequencing. PHYLIP clustering of inbred strains using a neighbor-joining algorithm based on Kimura two-parameter distances is shown. Wild Southern is a synthetic sample based on the sequencing of the Kiyosu population and determining bases using majority vote across 48 haplotypes. Differences between medaka samples and stickleback were too large to estimate distances. (B) Hierarchical clustering with Gower distances of genotypes of Kiyosu population founder samples, and representative Southern and Northern strains.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F3.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/3/433/F3.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464035" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">Phylogenetic relationships of medaka inbred strains and the Kiyosu population. (A) Phylogenetic relationships between the wild Southern population, five inbred strains, and stickleback based on high-throughput genome sequencing. PHYLIP clustering of inbred strains using a neighbor-joining algorithm based on Kimura two-parameter distances is shown. Wild Southern is a synthetic sample based on the sequencing of the Kiyosu population and determining bases using majority vote across 48 haplotypes. Differences between medaka samples and stickleback were too large to estimate distances. (B) Hierarchical clustering with Gower distances of genotypes of Kiyosu population founder samples, and representative Southern and Northern strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Selecting and genotypically characterizing a founder population from Southern Japan</h3><div id="sec-24" class="subsection"><h4>Selecting the founder population:</h4><p id="p-38">We set out to identify a genetically diverse, stable medaka founder population for the establishment of a panel of further isogenic lines. For this purpose we sampled irrigation canals in the Takashi Hongo and the Kiyosu areas near Toyohashi (Aichi Prefecture) in July 2010. We first analyzed the mitotype of 50 and 109 individuals from the two sites, respectively, as described by <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Takehana <em>et al.</em> (2003)</a>. The most likely source of population structure in these samples is human-mediated dispersal of fish, <em>e.g.</em>, fish discarded from aquariums. We detected an unusual mitotype in the Takashi Hongo population, normally present only in Northern Japanese medaka and that is a likely marker of human-mediated dispersal. Hence, we decided to focus on the Kiyosu population in further analysis.</p></div><div id="sec-25" class="subsection"><h4>Estimating the population diversity by sequencing variable genomic regions:</h4><p id="p-39">We sequenced the highly variable D-loop region of the mitochondrial cytochrome B gene in 105 Kiyosu individuals and detected eight different sequence variants. This degree of D-loop sequence diversity was comparable with an earlier study in which <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Katsumura <em>et al.</em> (2009)</a> detected 11 sequence types in 124 individuals in the most diverse and three types in 159 individuals in the least diverse of three wild medaka populations. Compared with the established medaka inbred strains, the Kiyosu population clustered with the southern HdrR, HO5, and HNCMH2 strains as expected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/FigureS1.pdf">Figure S1</a>). We analyzed microsatellite markers designed for 9 of the 24 medaka linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS1.pdf">Table S1</a>) and detected up to 21 different alleles within 105 individuals (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS2.pdf">Table S2</a>). An ideal founder population for establishing a medaka population genomics panel would be without significant population structure. We calculated inbreeding coefficients (F) from these microsatellite allele frequencies. Negative values of F indicate excess heterozygosity (outbreeding), whereas positive values indicate heterozygote deficiency (inbreeding) compared with Hardy-Weinberg expectations. In the case of our founder population, F ranged from â0.188 to 0.203 with seven positive and two negative values and an average of 0.107. Based on this preliminary analysis, we concluded that there is a tolerable degree of inbreeding within our population and decided to proceed with deeper analysis.</p></div><div id="sec-26" class="subsection"><h4>Genome-wide sequencing of eight trios from the founder population:</h4><p id="p-40">To analyze the Kiyosu population in greater detail, eight mating pairs and their first-generation progeny (<em>i.e.</em>, eight trios) were selected. The genomes of these individuals were profiled by the use of high-throughput short read sequencing, with ~9X mean depth of coverage per line and compared with the Southern Japanese reference HdrR sequence. Approximately 23% of variants detected were monoallelic across Kiyosu, distinguishing this population from the reference HdrR strain. Using a trio-aware algorithm [TrioCaller (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2013</a>)], we scored and phased the remaining 77% of variants, giving a total of 4,797,962 SNPs segregating within the Kiyosu population (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Of these, 107,950 mapped to 15,727 coding regions, with 59,700 synonymous, 47,639 missense, and 611 nonsense substitutions. As in the inbred lines, the latter two mutation types were considerably depleted in favor of synonymous SNPs compared with random expectation, as expected under purifying selection (44.1% missense mutations <em>vs.</em> 70.8% expected at random and 0.56% nonsense mutations <em>vs.</em> 4.4% expected at random, respectively, binomial test <em>P</em> &lt; 1 Ã 10<sup>â300</sup> in both cases). Similar observations have been made recently in other population surveys of species such as Arabidopsis and human (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gan <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">MacArthur <em>et al.</em> 2012</a>). We note that the proportion of nonsense mutations observed in the Kiyosu population is nearly twice as high as in the inbred lines (0.56% in Kiyosu compared with ~0.3% in the inbred strains). One explanation for this is the greater tolerability of nonsense mutations when present in a heterozygote. Indeed, only 228 of the 611 (37.3%) nonsense mutations detected in the Kiyosu wild population are homozygous in at least one individual.</p><p id="p-41">This polymorphism set provides the opportunity for a far more powerful test of population structure than the aforementioned microsatellite approach. Clustering the sample genotype differences by distance metric, we found that the parental genotypes were largely equidistant from each other in the tree without visible structure (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). As expected, the wild individuals clustered closest to each other near the Southern reference strain HdrR collected in the same region of Japan, confirming the results from aforementioned directed genotyping.</p></div></div><div id="sec-27" class="subsection"><h3>Linkage disequilibrium estimates suggest medaka as an attractive model for genome-wide association studies</h3><p id="p-42">Linkage disequilibrium (LD) is the deviation from independent segregation of alleles between loci and is dependent on recombination rate and population history. The extent of LD is an important parameter for mapping genotype-phenotype associations within a population. We estimated the LD for the medaka Kiyosu population from the 16 parental sample genotypes expressed as <em>r<sup>2</sup></em>, which is robust to smaller sample sizes and expresses the key relationship in association mapping. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> illustrates that median <em>r<sup>2</sup></em> between pairs of loci gradually drops with increasing distance, reaching a minimum at a distance of approximately 12.5 kb. To compare this with human LD estimates obtained in the same setting, we selected 16 independent parents from trios sequenced by Complete Genomics (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Drmanac <em>et al.</em> 2010</a>). In this human population <em>r<sup>2</sup></em> stabilizes at ~37 kb, which is ~3 times larger. For our medaka data, ~85% of SNP pairs with <em>r<sup>2</sup></em> &gt; 0.8 mapped to the same gene, with ~37% mapping to the same exon, indicating that fine mapping in medaka should largely be possible to the resolution of single genes and frequently even to a single exon. For comparison, only 56% and 29% of the equivalent SNP pairs mapped to the same gene and exon in the equivalent human population, respectively. We also estimated haplotype block size across 15 of the medaka chromosomes by using the method of <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gabriel <em>et al.</em> (2002)</a>. The mean haplotype block size was 712 bp (median 259 bp) with a maximum of ~78 kb. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> shows the LD map of a representative region of chromosome 11. We conclude from the detailed genetic characterization of this Kiyosu population sample that these fish will be suitable for establishment of a population genomics resource.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium (LD) in the Kiyosu medaka population. (A) Decay of LD for a sample of medaka SNPs. LD (r2) was calculated for all SNP pairs in 1-Mb windows overlapping by 500 kb across medaka chromosome 1, and r2 was plotted against distance for a sample of 0.03% (21976) of pairs. The main plot shows a blow-up of distances between 0 and 25 kb, with data for pairs of SNPs up to 1 Mb apart shown in the inset region. Multiple sampling and analysis of other chromosomes gave similar results (not shown). (B) Pairwise LD plot for a representative region of chromosome 11. Pairwise LD (D&#x2032;) for the 25-kb region 11:375000&#x2212;399999 as displayed in Haploview is shown. The plot shows the LD of pairs of SNPs by squares joining the two SNPs via the diagonal and is color coded as described for the standard LD color scheme at http://www.broadinstitute.org/science/programs/medical-and-population-genetics/haploview/ld-display such that pairs of SNPs in LD at D&#x2032; = 1 are colored blue at LOD " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium (LD) in the Kiyosu medaka population. (A) Decay of LD for a sample of medaka SNPs. LD (&lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) was calculated for all SNP pairs in 1-Mb windows overlapping by 500 kb across medaka chromosome 1, and r2 was plotted against distance for a sample of 0.03% (21976) of pairs. The main plot shows a blow-up of distances between 0 and 25 kb, with data for pairs of SNPs up to 1 Mb apart shown in the inset region. Multiple sampling and analysis of other chromosomes gave similar results (not shown). (B) Pairwise LD plot for a representative region of chromosome 11. Pairwise LD (D&#x2032;) for the 25-kb region 11:375000&#x2212;399999 as displayed in Haploview is shown. The plot shows the LD of pairs of SNPs by squares joining the two SNPs via the diagonal and is color coded as described for the standard LD color scheme at http://www.broadinstitute.org/science/programs/medical-and-population-genetics/haploview/ld-display such that pairs of SNPs in LD at D&#x2032; = 1 are colored blue at LOD &lt;2 and red at LOD &#x2265; 2, whereas those with D&#x2032; &lt; 1 are white at LOD &lt; 2 and shades of red/pink at LOD &#x2265; 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F4.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/3/433/F4.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464165" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Linkage disequilibrium (LD) in the Kiyosu medaka population. (A) Decay of LD for a sample of medaka SNPs. LD (<em>r<sup>2</sup></em>) was calculated for all SNP pairs in 1-Mb windows overlapping by 500 kb across medaka chromosome 1, and r2 was plotted against distance for a sample of 0.03% (21976) of pairs. The main plot shows a blow-up of distances between 0 and 25 kb, with data for pairs of SNPs up to 1 Mb apart shown in the inset region. Multiple sampling and analysis of other chromosomes gave similar results (not shown). (B) Pairwise LD plot for a representative region of chromosome 11. Pairwise LD (Dâ²) for the 25-kb region 11:375000â399999 as displayed in Haploview is shown. The plot shows the LD of pairs of SNPs by squares joining the two SNPs via the diagonal and is color coded as described for the standard LD color scheme at <a href="http://www.broadinstitute.org/science/programs/medical-and-population-genetics/haploview/ld-display">http://www.broadinstitute.org/science/programs/medical-and-population-genetics/haploview/ld-display</a> such that pairs of SNPs in LD at Dâ² = 1 are colored blue at LOD &lt;2 and red at LOD â¥ 2, whereas those with Dâ² &lt; 1 are white at LOD &lt; 2 and shades of red/pink at LOD â¥ 2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Whole-genome polymorphism data provide insights into medaka population history and evolution</h3><p id="p-44">The determination of polymorphism data in a wild medaka population allows us to examine a number of questions in medaka population genetics, such as probe for interbreeding between different medaka populations, infer population history, and identify loci that are likely to undergo a recent positive selection.</p><div id="sec-29" class="subsection"><h4>Probing for potential interbreeding between medaka populations:</h4><p id="p-45">Phylogenetically, the Northern Japanese medaka strains lie between the Southern Japanese and the Chinese and Korean medaka populations, which form an outgroup to the two Japanese clades (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Takehana <em>et al.</em> 2004</a>). There are several phenotypes unique to the Northern strains compared with all other medaka, including a requirement for shallow water tanks for good viability, and differences in behavior, pigmentation (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>) as well as in morphometric parameters (as will be presented in <em>Morphometric analysis as a promising tool to assess medaka phenotypic diversity</em>). Recently these divergent phenotypes have even led to suggestion that the Northern and Southern medaka populations are distinct species (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Asai <em>et al.</em> 2011</a>).</p><p id="p-46">To clarify the genetic relationship between the Northern and Southern Japanese medaka clades, we investigated potential wild interbreeding events drawing information from the Kiyosu population and the inbred strains. When interbreeding occurs, genotypes will be exchanged between populations that have separated some time ago, events that are referred to as introgression. Recent studies show that whole-genome data can illuminate ancient introgression events that are not visible by methods typing more limited numbers of loci (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Green <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ishiguro <em>et al.</em> 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meyer <em>et al.</em> 2012</a>).</p><p id="p-47">The test for introgression proposed by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Durand <em>et al.</em> (2011)</a> is based on the expectation that the ratio of ancestral alleles (<em>e.g.</em>, those that medaka has in common with stickleback) to derived alleles is likely to be equal across populations descending from a common ancestor (<em>e.g.</em>, two Southern populations). If, however, one of these populations exchanged genes (introgressed) with a third population that is further away on the phylogenetic tree (<em>e.g.</em>, a Northern population), this symmetry would be perturbed. Specifically, at positions where this Northern population has derived alleles, the Southern population that introgressed with it is more likely to also have derived alleles compared with its nonintrogressed Southern counterpart.</p><p id="p-48">Using this approach, we found limited evidence of introgression between the Northern (HNI) and Southern Japanese samples. Surprisingly, however, the observed effect was stronger in the Southern inbred HdrR strain than in the Kiyosu population. Since these two samples share a high sequence similarity, we asked whether the result for the Kiyosu population is in fact a âghost populationâ artifact (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Durand <em>et al.</em> 2011</a>) of its similarity with HdrR. Indeed, when the analysis was limited to just SNPs that differentiate the Kiyosu population from HdrR, no evidence for the introgression of Kiyosu with HNI was obtained (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS3.pdf">Table S3</a>).</p><p id="p-49">Applying a similar logic, we found evidence of introgression between the Taiwanese (Nilan) inbred strain and HdrR, but less so between Nilan and the Kiyosu population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS3.pdf">Table S3</a>).</p><p id="p-50">The results based on the Kiyosu population and the Northern strains indicate that there was either extremely limited or no interbreeding between the Northern and Southern medaka lineages (at least to the point of the most recent common ancestor of the wild Kiyosu population and the Northern strains). At the same time, we have obtained evidence for introgression between pairs of some laboratory lines. Although these observations could reflect the population history of the wild populations from which the inbred lines were sampled, they may also be the results of historical laboratory interbreeding. Further, well sampled, wild individual sequences from Northern Japan, Korea, and Taiwan would be needed to fully resolve this issue.</p></div><div id="sec-30" class="subsection"><h4>The population history of Southern Japanese medaka:</h4><p id="p-51">It has been shown in humans, pigs, and giant pandas that it is possible to determine a population history in terms of effective population size from a single diploid individual based on the local density of heterozygous sites (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Li and Durbin 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Groenen <em>et al.</em> 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhao <em>et al.</em> 2013</a>). This approach exploits the presence of many thousands of independent segments in each individual genome separated by ancestral recombination events. A Hidden Markov model integrates the uncertainty of segments to provide a consistent view of the effective population size as a function of time. We performed a similar analysis on all the individuals in our Kiyosu wild catch sample. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the distribution of population size estimates from the Kiyosu individuals. Except for the expected variability of estimates for very recent history, the population history profile was consistent between the individuals and is a different shape from the other species analyzed previously. This profile suggests a very large older population that went through a relatively recent bottleneck approximately 10,000 years ago (assuming a position-wise mutation rate of 2.5 Ã 10<sup>â8</sup> per generation, generation time = 1 year) followed by a modest expansion in recent history. In addition to its immediate significance for medaka evolution, this result highlights the applicability and reproducibility of effective population size over time estimation for single-individual diploid genomes of model organisms determined using short-read sequencing.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of population size estimates from the Kiyosu individuals. Population size estimates over time were calculated using the psmc package for the founder individuals in the Kiyosu wild catch as described in Materials and Methods. The graph shows the distribution of estimates after combining by binning." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of population size estimates from the Kiyosu individuals. Population size estimates over time were calculated using the psmc package for the founder individuals in the Kiyosu wild catch as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. The graph shows the distribution of estimates after combining by binning.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F5.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/3/433/F5.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464115" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">Distribution of population size estimates from the Kiyosu individuals. Population size estimates over time were calculated using the psmc package for the founder individuals in the Kiyosu wild catch as described in <em>Materials and Methods</em>. The graph shows the distribution of estimates after combining by binning.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h4>Evidence of recent selection in the Southern Japanese medaka population:</h4><p id="p-53">We have used the wild genotypes to look for patterns of recent positive selection in the Southern population. An established footprint of recent selection is unusually long haplotypes of low diversity surrounding the allele under selection, whereas the alternative allele(s) are associated with haplotypes of a more typical length (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sabeti <em>et al.</em> 2002</a>). A metric termed extended haplotype homozygosity (EHH) describes this effect for a given locus representing the probability that two randomly chosen chromosomes carrying the core haplotype of interest are identical by descent for the entire interval from the core region to any point <em>x</em>. EHH thus detects the transmission of an extended haplotype without recombination (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sabeti <em>et al.</em> 2002</a>). We applied an established statistical approach based on the ratio of EHH decay with distance (known as iHS for integrated haplotype score; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Voight <em>et al</em>. 2006</a>) for the ancestral and derived allele using the parental genotypes of the newly established Southern strain trios. After applying stringent iHS filters (see <em>Materials and Methods</em>), we found that 192 SNPs, most of which map to 12 broad domains, showed evidence for recent positive selection of the derived haplotype (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). As expected, there were no cases with an unusually long ancestral haplotype at this cutoff. These domains contained 17 genes, and in three cases the implicated SNPs mapped to their coding regions [<em>thioredoxin reductase 3</em> (defense against oxidative stress), <em>kelch-like 10</em>, <em>solute carrier family 41</em>], causing nonsynonymous amino acid changes. Interestingly, nearly 20% of SNPs indicative of recent positive selection localized to highly conserved, potentially regulatory genomic regions (1.4-fold enrichment over random expectation, <em>P</em> = 0.03, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/TableS4.pdf">Table S4</a>). This indicates that recent selection events have resulted in potentially significant changes in the coding and noncoding sequence, paving the way for a further examination of their impact on the molecular function and physiology of the organism as whole.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464049/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464049/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Regions under positive selection in the medaka genome</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-32" class="subsection"><h3>Morphometric analysis as a promising tool to assess medaka phenotypic diversity</h3><p id="p-55">An important feature of a population panel is that it is expected to exhibit appreciable phenotypic variation across individual lines. To characterize one set of phenotypes, we took advantage of the Southern inbred lines already in existence because these are likely to be representative of lines generated from the Kiyosu population, allowing us to easily characterize broad-sense heritability of phenotypes. We used light microscope-based imaging combined with an automatic annotation algorithm to derive a number of morphometric features across four Southern (HdrR, Icab, HNCMH2, HO5), two Northern strains (Kaga, HNI), and the Kiyosu population (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1/FigureS3.pdf">Figure S3</a>) from both lateral and dorsal viewpoints. The morphometric analysis of the six inbred lines shows reassuringly that the body length measurement was almost perfectly correlated between the two viewpoints. As expected in a fish, the majority of the variance between individuals correlates with body length, and so we normalized the remaining morphometric phenotypes by body length. Of the seven phenotypes analyzed, four show greater than 30% broad-sense heritability <em>i.e.</em>, the differences between strains explained 30% or more of the variance (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). An example is shown in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>, where a fish from the HdrR strain has substantially smaller eyes relative to body length than the Icab strain (compare box plot in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>). The broad-sense heritability estimate is similar to analogous morphometric measurements between inbred mouse strains, suggesting that medaka populations have similar levels of phenotypic variation as observed between laboratory mouse strains. Many other established phenotypes that differ between medaka strains have been observed during the previous century of research on this fish (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hyodo-Taguchi 1990</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Ishikawa <em>et al.</em> 1999</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kimura <em>et al.</em> 2007</a>), and further work will be needed to examine the suitability of this population for each phenotype. Given the high diversity of alleles in the Southern population, we can expect many phenotypes to have at least some genetic variance in this population.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/3/433/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Morphometric analysis of medaka inbred strains. (A) Four lateral (L1-L4) and five dorsal (D1-D5) morphometric distances were extracted and analyzed for each inbred strain. (B) A bar chart showing the proportion of variance explained by the difference between strains as a fraction of the total variance. The variables are the measurements corrected by the appropriate body length measurement (L4 and D5, respectively). (C) Example pictures showing substantial differences in lateral length and eye diameter between fish from the two Southern inbred strains HdrR and Icab. (D) Box plots showing the distribution of eye diameter, lateral length, and the ratio between eye diameter and lateral length for HdrR and Icab. Eye diameter and lateral length: y-axis = value in pixel. HdrR (n = 70), Icab (n = 78)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-788837149" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Morphometric analysis of medaka inbred strains. (A) Four lateral (L1-L4) and five dorsal (D1-D5) morphometric distances were extracted and analyzed for each inbred strain. (B) A bar chart showing the proportion of variance explained by the difference between strains as a fraction of the total variance. The variables are the measurements corrected by the appropriate body length measurement (L4 and D5, respectively). (C) Example pictures showing substantial differences in lateral length and eye diameter between fish from the two Southern inbred strains HdrR and Icab. (D) Box plots showing the distribution of eye diameter, lateral length, and the ratio between eye diameter and lateral length for HdrR and Icab. Eye diameter and lateral length: y-axis = value in pixel. HdrR (n = 70), Icab (n = 78).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/3/433/F6.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/3/433/F6.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/3/433/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/3/433/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464133" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-56" class="first-child">Morphometric analysis of medaka inbred strains. (A) Four lateral (L1-L4) and five dorsal (D1-D5) morphometric distances were extracted and analyzed for each inbred strain. (B) A bar chart showing the proportion of variance explained by the difference between strains as a fraction of the total variance. The variables are the measurements corrected by the appropriate body length measurement (L4 and D5, respectively). (C) Example pictures showing substantial differences in lateral length and eye diameter between fish from the two Southern inbred strains HdrR and Icab. (D) Box plots showing the distribution of eye diameter, lateral length, and the ratio between eye diameter and lateral length for HdrR and Icab. Eye diameter and lateral length: y-axis = value in pixel. HdrR (n = 70), Icab (n = 78).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-33"><h2 class="">Discussion</h2><p id="p-57">Advances in genome sequencing have made it possible not only to expand the range of species for which reference genomes are available but also to sample the genetic diversity of individuals within the same species. Here we have explored individual genetic diversity in the model vertebrate, medaka (<em>Oryzias latipes</em>), by using inbred strains derived from various geographically distinct populations and from wild catches obtained from a single Southern Japanese population.</p><p id="p-58">Our primary motivation for this study was to characterize a single medaka population from Southern Japan as a potential source for the first vertebrate near-isogenic wild panel. The sampled individuals are genetically highly diverse, as expected for a large population size. There is no discernible population structure in our sample, which is beneficial for mapping by association. Both medaka and human have substantially shorter LD than structured mouse populations. In the past, longer LD profiles were generally desirable in genetic panels for association analysis because this reduced the number of markers that needed to be typed to detect association. However, in the modern setting of inexpensive complete sequencing, an LD profile extending for shorter distances allows higher resolution and finer mapping of causal variants. This relatively tight LD will prove invaluable for the isolation of causative variants with 85% of SNP pairs in LD (from this limited sample set) mapping to single genes and 37% mapping to single exons. A larger panel will sample more polymorphic sites, and so we can expect this panel to nearly always have resolution to a single gene, and often to a single exon or regulatory region. As expected, the sample possesses considerable phenotypic diversity, with the phenotypic differences similar in magnitude to differences between inbred mouse strains.</p><p id="p-59">During characterization of the progenitors of the panel, we also determined the sequence of a number of existing isogenic medaka strains sampled from several wild populations. Our additional deep sequencing data provide an estimate of the original reference sequence error rate at around 2 Ã 10<sup>â5</sup> assuming that there has been no sequence divergence in this strain since 2002. This is better than the estimated error rates for draft quality genome sequencing. We are in the process of generating an improved reference assembly using the additional sequence generated in this project and other sources in the future. All the sequence data are freely available, and the differences identified have been submitted to the Single Nucleotide Polymorphism Database (dbSNP) and will be available on the Ensembl browser and other resources in the future. The list of variants provides a resource for current medaka researchers, in particular those aiming to map and understand the different phenotypes present in these strains. As expected, among the wild catches and the inbred lines there are 2235 nonsense mutations representing natural knockouts of particular protein coding genes.</p><p id="p-60">These initial data have allowed us to further explore medaka population genetics. We lend genetic support to the current hypothesis that the Northern medaka population is very likely to be a distinct species that has not significantly interbred with the Southern medaka population since the Southern/Northern population split. The population history of Southern medaka suggests a recent bottleneck from a substantially larger population approximately 10,000 years ago. Two events present themselves as possible explanations for this bottleneck. First, it is tempting to speculate that this might be associated with the development of rice cultivation on the Japanese islands. Second, there may have been transgression of medaka populations after the last glacial age. 18,000 years ago, the sea level was about 120 m lower than at present, and Japan was directly connected to the Asian continent. Around 10,000 years ago, the sea level rose and the present plain fields were beneath sea level. To investigate this issue further, we need to contrast the Southern Japanese medaka population history to other histories from other geographic locations. The identification of three genes with putative amino acid changes due to positive selection is the start of exploring the impact of positive selection in medaka. The <em>thioredoxin reductase-3</em>, <em>kelch-like 10</em>, and <em>solute carrier family 41</em> (<em>slc41a3</em>) genes, involved in counteracting oxidative stress within the cell, spermatogenesis, and magnesium transport, respectively, might play fundamental roles for fitness and reproductive success. Intronic mutations also were found in other genes associated with membrane transport (<em>rims1</em>, <em>slc39a10</em>) and carbohydrate metabolism (<em>chst13</em>), as well as signaling (<em>wnt4a</em>) and chromatin (<em>histone 2ay</em>), suggesting that they may well be under noncoding, potentially regulatory positive selection, consistent with the evidence from stickleback (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jones <em>et al.</em> 2012</a>).</p><p id="p-61">Overall our study shows that a near-isogenic wild medaka panel has excellent potential for exploring vertebrate biology, analogous to the role of the Arabidopsis and Drosophila panels for plants and invertebrates, respectively. At the time of writing we have initiated the fourth generation of an inbreeding program for 200 independent medaka lines derived from this Kiyosu population. On completion of this program, these lines will be made broadly available to the community, with all data freely distributed. Of interest is that medaka can be housed in similar facilities to zebrafish, and the same phenotyping protocols can be applied to both fish species. Progress of this project can be tracked at <a href="http://www.ebi.ac.uk/birney-srv/medaka-ref-panel/">http://www.ebi.ac.uk/birney-srv/medaka-ref-panel/</a>, and we welcome participants both from established medaka and teleost laboratories and from the statistical genetics community.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank Stephen Fitzgerald and Javier Herrero for discussions on comparative alignments, Justin Paschall and Nima Pakseresht for assistance with database submissions, Hilary Browne at the Sanger Institute Sequencing Facility for sequencing of the inbred trios, Tetsuaki Kimura and Yusuke Takehana for assistance with fish sampling, Wei Chen for advice on TrioCaller, and the National BioResource Project Core Facility Upgrading Program and Genome Information Upgrading Program for financial support of the resequencing of medaka inbred lines. R.P. and F.L. thank Nadeshda Wolf, Nadine Eschen, Natalja Kusminski, and Nils Trost for assistance in inbreeding and morphometric studies.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008722/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-63"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-10-01"><span class="received-label">Received </span>October 1, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-12-23"><span class="accepted-label">Accepted </span>December 23, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2014 Spivakov <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aida</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span>â<span class="cit-article-title">On the inheritance of color in a fresh-water fish, APLOCHEILUS LATIPES Temmick and Schlegel, with special reference to sex-linked inheritance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">554</span>â<span class="cit-lpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAida%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253D554%26rft.epage%253D573%26rft.atitle%253DON%2BTHE%2BINHERITANCE%2BOF%2BCOLOR%2BIN%2BA%2BFRESH-WATER%2BFISH%252C%2BAPLOCHEILUS%2BLATIPES%2BTEMMICK%2BAND%2BSCHLEGEL%252C%2BWITH%2BSPECIAL%2BREFERENCE%2BTO%2BSEX-LINKED%2BINHERITANCE%26rft_id%253Dinfo%253Apmid%252F17245975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI2LzYvNTU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosoya</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title"><em>Oryzias sakaizumii</em>, a new ricefish from northern Japan (Teleostei: Adrianichthyidae).</span> <abbr class="cit-jnl-abbrev">Ichthyol. Explor. Freshwat.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIchthyol.%2BExplor.%2BFreshwat.%26rft.volume%253D22%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.3" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>â<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.4" data-doi="10.1097/00007890-197103000-00013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span>â<span class="cit-article-title">Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes.</span> <abbr class="cit-jnl-abbrev">Transplantation</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">325</span>â<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransplantation%26rft.stitle%253DTransplantation%26rft.aulast%253DBailey%26rft.auinit1%253DD.%2BW.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D327%26rft.atitle%253DRecombinant-inbred%2Bstrains.%2BAn%2Baid%2Bto%2Bfinding%2Bidentity%252C%2Blinkage%252C%2Band%2Bfunction%2Bof%2Bhistocompatibility%2Band%2Bother%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00007890-197103000-00013%26rft_id%253Dinfo%253Apmid%252F5558564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00007890-197103000-00013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5558564&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971I836800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.5" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>â<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMS8yLzI2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.6" data-doi="10.1038/nature11867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lite</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Finding the sources of missing heritability in a yeast cross.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">234</span>â<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11867%26rft_id%253Dinfo%253Apmid%252F23376951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376951&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315137700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diallo</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarenkov</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">T-REX: a web server for inferring, validating and visualizing phylogenetic trees and networks.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">W573â579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253DW573579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.8" data-doi="10.1101/gr.142455.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genotype calling and haplotyping in parent-offspring trios.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">142</span>â<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.142455.112%26rft_id%253Dinfo%253Apmid%252F23064751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzEvMTQyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.9" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>â<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.10" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>â<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.11" data-doi="10.1126/science.1181498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drmanac</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpern</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">78</span>â<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1181498%26rft_id%253Dinfo%253Apmid%252F19892942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS83OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.12" data-doi="10.1093/molbev/msr048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Testing for ancient admixture between closely related populations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2239</span>â<span class="cit-lpage">2252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr048%26rft_id%253Dinfo%253Apmid%252F21325092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjIzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.13" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">ENCODE Project Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>â<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.14" data-doi="10.1101/gr.086660.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic architecture of quantitative traits in mice, flies, and humans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">723</span>â<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D723%26rft.epage%253D733%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bquantitative%2Btraits%2Bin%2Bmice%252C%2Bflies%252C%2Band%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086660.108%26rft_id%253Dinfo%253Apmid%252F19411597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvNzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.15" data-doi="10.1534/genetics.106.060376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Genetic dissection of intermated recombinant inbred lines using a new genetic map of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1671</span>â<span class="cit-lpage">1683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.060376%26rft_id%253Dinfo%253Apmid%252F16951074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE2NzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.16" data-doi="10.1016/j.mod.2004.04.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furutani-Seiki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morinaga</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suwa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niwa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">A systematic genome-wide screen for mutations affecting organogenesis in Medaka, <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">647</span>â<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DFurutani-Seiki%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D7-8%26rft.spage%253D647%26rft.epage%253D658%26rft.atitle%253DA%2Bsystematic%2Bgenome-wide%2Bscreen%2Bfor%2Bmutations%2Baffecting%2Borganogenesis%2Bin%2BMedaka%252C%2BOryzias%2Blatipes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mod.2004.04.016%26rft_id%253Dinfo%253Apmid%252F15210174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mod.2004.04.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15210174&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222571200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.17" data-doi="10.1126/science.1069424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">The structure of haplotype blocks in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">2225</span>â<span class="cit-lpage">2229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069424%26rft_id%253Dinfo%253Apmid%252F12029063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTYvNTU3Ni8yMjI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNC8zLzQzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.18" data-doi="10.1038/nature10414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegle</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drewe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">419</span>â<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGan%26rft.auinit1%253DX.%26rft.volume%253D477%26rft.issue%253D7365%26rft.spage%253D419%26rft.epage%253D423%26rft.atitle%253DMultiple%2Breference%2Bgenomes%2Band%2Btranscriptomes%2Bfor%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10414%26rft_id%253Dinfo%253Apmid%252F21874022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874022&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295080500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.19" data-doi="10.1093/bioinformatics/bts115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitalis</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">rehh: an R package to detect footprints of selection in genome-wide SNP data from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1176</span>â<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts115%26rft_id%253Dinfo%253Apmid%252F22402612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyOC84LzExNzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.20" data-doi="10.1126/science.1188021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maricic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenzel</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A draft sequence of the Neandertal genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">710</span>â<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D328%26rft.issue%253D5979%26rft.spage%253D710%26rft.epage%253D722%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BNeandertal%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188021%26rft_id%253Dinfo%253Apmid%252F20448178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3OS83MTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.21" data-doi="10.1038/nature11622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uenishi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuggle</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Analyses of pig genomes provide insight into porcine demography and evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">393</span>â<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGroenen%26rft.auinit1%253DM.%2BA.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D393%26rft.epage%253D398%26rft.atitle%253DAnalyses%2Bof%2Bpig%2Bgenomes%2Bprovide%2Binsight%2Binto%2Bporcine%2Bdemography%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11622%26rft_id%253Dinfo%253Apmid%252F23151582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23151582&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyodo-Taguchi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-article-title">Establishment of inbred strains of the teleost <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Zool. Mag.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">283</span>â<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BMag.%26rft.volume%253D89%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-book" id="cit-4.3.433.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyodo-Taguchi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Inbred strains of the Medaka and their characteristics, pp. 129-142 in</span> <span class="cit-source">Biology of the Medaka</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Egami</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Yamagami</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Tokyo University Press</span>, <span class="cit-publ-loc">Tokyo (in Japanese)</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.24" data-doi="10.3106/041.037.0101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishiguro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoshima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hongo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">mtDNA variation and human-mediated introgression of indigenous Sus populations on several Indonesian islands.</span> <abbr class="cit-jnl-abbrev">Mammal Study</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammal%2BStudy%26rft.volume%253D37%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3106%252F041.037.0101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3106/041.037.0101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.25" data-doi="10.1159/000006577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Different brain morphologies from different genotypes in a single teleost species, the medaka (<em>Oryzias latipes</em>).</span> <abbr class="cit-jnl-abbrev">Brain Behav. Evol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">2</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrain%2BBehavior%2Band%2BEvolution%26rft.stitle%253DBrain%2BBehavior%2Band%2BEvolution%26rft.aulast%253DIshikawa%26rft.auinit1%253DY.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D9%26rft.atitle%253DDifferent%2Bbrain%2Bmorphologies%2Bfrom%2Bdifferent%2Bgenotypes%2Bin%2Ba%2Bsingle%2Bteleost%2Bspecies%252C%2Bthe%2Bmedaka%2B%2528Oryzias%2Blatipes%2529%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000006577%26rft_id%253Dinfo%253Apmid%252F9858800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000006577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9858800&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.26" data-doi="10.1038/nature10944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">55</span>â<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.27" data-doi="10.1038/nature05846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasahara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The medaka draft genome and insights into vertebrate genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">714</span>â<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKasahara%26rft.auinit1%253DM.%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D714%26rft.epage%253D719%26rft.atitle%253DThe%2Bmedaka%2Bdraft%2Bgenome%2Band%2Binsights%2Binto%2Bvertebrate%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05846%26rft_id%253Dinfo%253Apmid%252F17554307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05846&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554307&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.28" data-doi="10.1016/j.gene.2009.04.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katsumura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suguro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic differentiation among local populations of medaka fish (<em>Oryzias latipes</em>) evaluated through grid- and deme-based sampling.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">170</span>â<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D443%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2009.04.017%26rft_id%253Dinfo%253Apmid%252F19397956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2009.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19397956&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268084100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.29" data-doi="10.1534/genetics.106.068460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitani</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genetic analysis of craniofacial traits in the medaka.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2379</span>â<span class="cit-lpage">2388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DT.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2379%26rft.epage%253D2388%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BCraniofacial%2BTraits%2Bin%2Bthe%2BMedaka%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068460%26rft_id%253Dinfo%253Apmid%252F18073435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.30" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genetic dissection of a model complex trait using the Drosophila Synthetic Population Resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>â<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.31" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>â<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.32" data-doi="10.1038/nature10231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Inference of human population history from individual whole-genome sequences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">493</span>â<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753753&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293167900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.33" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.34" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>â<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.35" data-doi="10.1016/S0925-4773(00)00406-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loosli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koster</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhnlein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrich</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">A genetic screen for mutations affecting embryonic development in medaka fish (<em>Oryzias latipes</em>).</span> <abbr class="cit-jnl-abbrev">Mech. Dev.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">133</span>â<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bdevelopment%26rft.stitle%253DMech%2BDev%26rft.aulast%253DLoosli%26rft.auinit1%253DF.%26rft.volume%253D97%26rft.issue%253D1-2%26rft.spage%253D133%26rft.epage%253D139%26rft.atitle%253DA%2Bgenetic%2Bscreen%2Bfor%2Bmutations%2Baffecting%2Bembryonic%2Bdevelopment%2Bin%2Bmedaka%2Bfish%2B%2528Oryzias%2Blatipes%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0925-4773%252800%252900406-8%26rft_id%253Dinfo%253Apmid%252F11025214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0925-4773(00)00406-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11025214&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000090089900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.36" data-doi="10.1126/science.1215040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacArthur</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankish</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A systematic survey of loss-of-function variants in human protein-coding genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">335</span>: <span class="cit-fpage">823</span>â<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMacArthur%26rft.auinit1%253DD.%2BG.%26rft.volume%253D335%26rft.issue%253D6070%26rft.spage%253D823%26rft.epage%253D828%26rft.atitle%253DA%2BSystematic%2BSurvey%2Bof%2BLoss-of-Function%2BVariants%2Bin%2BHuman%2BProtein-Coding%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1215040%26rft_id%253Dinfo%253Apmid%252F22344438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzUvNjA3MC84MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.37" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The Drosophila melanogaster Genetic Reference Panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>â<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.38" data-doi="10.1126/science.1224344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gansauge</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A high-coverage genome sequence from an archaic Denisovan individual.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">222</span>â<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1224344%26rft_id%253Dinfo%253Apmid%252F22936568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzgvNjEwNC8yMjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.39" data-doi="10.1371/journal.pone.0019747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mongin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auer</span>  <span class="cit-name-given-names">T. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourrat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruhl</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Combining computational prediction of cis-regulatory elements with a new enhancer assay to efficiently label neuronal structures in the medaka fish.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMongin%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19747%26rft.epage%253De19747%26rft.atitle%253DCombining%2Bcomputational%2Bprediction%2Bof%2Bcis-regulatory%2Belements%2Bwith%2Ba%2Bnew%2Benhancer%2Bassay%2Bto%2Befficiently%2Blabel%2Bneuronal%2Bstructures%2Bin%2Bthe%2Bmedaka%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019747%26rft_id%253Dinfo%253Apmid%252F21637758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637758&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.40" data-doi="10.1186/1471-2156-5-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">A new set of BXD recombinant inbred lines from advanced intercross populations in mice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D5%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DA%2Bnew%2Bset%2Bof%2BBXD%2Brecombinant%2Binbred%2Blines%2Bfrom%2Badvanced%2Bintercross%2Bpopulations%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-7%26rft_id%253Dinfo%253Apmid%252F15117419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15117419&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.41" data-doi="10.1097/00004872-198903000-00008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pravenec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klir</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kren</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zicha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunes</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">An analysis of spontaneous hypertension in spontaneously hypertensive rats by means of new recombinant inbred strains.</span> <abbr class="cit-jnl-abbrev">J. Hypertens.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">217</span>â<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bhypertension%26rft.stitle%253DJ%2BHypertens%26rft.aulast%253DPravenec%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D217%26rft.epage%253D221%26rft.atitle%253DAn%2Banalysis%2Bof%2Bspontaneous%2Bhypertension%2Bin%2Bspontaneously%2Bhypertensive%2Brats%2Bby%2Bmeans%2Bof%2Bnew%2Brecombinant%2Binbred%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00004872-198903000-00008%26rft_id%253Dinfo%253Apmid%252F2708818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00004872-198903000-00008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2708818&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T885700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.42" data-doi="10.1038/nprot.2006.165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rembold</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahiri</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulkes</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wittbrodt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Transgenesis in fish: efficient selection of transgenic fish by co-injection with a fluorescent reporter construct.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1133</span>â<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DRembold%26rft.auinit1%253DM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D1133%26rft.epage%253D1139%26rft.atitle%253DTransgenesis%2Bin%2Bfish%253A%2Befficient%2Bselection%2Bof%2Btransgenic%2Bfish%2Bby%2Bco-injection%2Bwith%2Ba%2Bfluorescent%2Breporter%2Bconstruct.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2006.165%26rft_id%253Dinfo%253Apmid%252F17406394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2006.165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17406394&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Retief</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Phylogenetic analysis using PHYLIP.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">243</span>â<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DRetief%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D132%26rft.spage%253D243%26rft.epage%253D258%26rft.atitle%253DPhylogenetic%2Banalysis%2Busing%2BPHYLIP.%26rft_id%253Dinfo%253Apmid%252F10547839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10547839&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.44" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">H. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>â<span class="cit-lpage">837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.45" data-doi="10.1538/expanim.59.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The National BioResource Project Medaka (NBRP Medaka): an integrated bioresource for biological and biomedical sciences.</span> <abbr class="cit-jnl-abbrev">Exp. Anim.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">13</span>â<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2Banimals%2B%252F%2BJapanese%2BAssociation%2Bfor%2BLaboratory%2BAnimal%2BScience%26rft.stitle%253DExp%2BAnim%26rft.aulast%253DSasado%26rft.auinit1%253DT.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D23%26rft.atitle%253DThe%2BNational%2BBioResource%2BProject%2BMedaka%2B%2528NBRP%2BMedaka%2529%253A%2Ban%2Bintegrated%2Bbioresource%2Bfor%2Bbiological%2Band%2Bbiomedical%2Bsciences.%26rft_id%253Dinfo%253Adoi%252F10.1538%252Fexpanim.59.13%26rft_id%253Dinfo%253Apmid%252F20224166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1538/expanim.59.13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224166&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.46" data-doi="10.1126/science.1163183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mello</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Chromatin-associated periodicity in genetic variation downstream of transcriptional start sites.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">401</span>â<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1163183%26rft_id%253Dinfo%253Apmid%252F19074313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMi80MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy80LzMvNDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.47" data-doi="10.1098/rsbl.2009.0419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Setiamarga</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miya</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamanoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azuma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">812</span>â<span class="cit-lpage">816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2009.0419%26rft_id%253Dinfo%253Apmid%252F19586967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNS82LzgxMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzQvMy80MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.48" data-doi="10.1023/A:1014868625533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharopova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Villeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Development and mapping of SSR markers for maize.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">463</span>â<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DSharopova%26rft.auinit1%253DN.%26rft.volume%253D48%26rft.issue%253D5-6%26rft.spage%253D463%26rft.epage%253D481%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2BSSR%2Bmarkers%2Bfor%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1014868625533%26rft_id%253Dinfo%253Apmid%252F12004892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1014868625533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12004892&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174606400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.49" data-doi="10.1016/S0167-9473(97)00020-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Struyf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousseeuw</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Integrating robust clustering techniques in S-PLUS.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">17</span>â<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D26%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0167-9473%252897%252900020-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0167-9473(97)00020-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997YL35300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.50" data-doi="10.1146/annurev-genet-051710-151001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The art of medaka genetics and genomics: what makes them so unique?</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>â<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DTakeda%26rft.auinit1%253DH.%26rft.volume%253D44%26rft.spage%253D217%26rft.epage%253D241%26rft.atitle%253DThe%2Bart%2Bof%2Bmedaka%2Bgenetics%2Band%2Bgenomics%253A%2Bwhat%2Bmakes%2Bthem%2Bso%2Bunique%253F%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-051710-151001%26rft_id%253Dinfo%253Apmid%252F20731603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-051710-151001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20731603&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286042600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.51" data-doi="10.2108/zsj.20.1279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuchiya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Geographic variation and diversity of the cytochrome b gene in Japanese wild populations of medaka, <em>Oryzias latipes</em>.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1279</span>â<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D1279%26rft.epage%253D1291%26rft.atitle%253DGeographic%2Bvariation%2Band%2Bdiversity%2Bof%2Bthe%2Bcytochrome%2Bb%2Bgene%2Bin%2BJapanese%2Bwild%2Bpopulations%2Bof%2Bmedaka%252C%2BOryzias%2Blatipes.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.20.1279%26rft_id%253Dinfo%253Apmid%252F14569151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.20.1279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14569151&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186945600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.52" data-doi="10.2108/zsj.21.977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeon</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakaizumi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Genetic structure of Korean wild populations of the Medaka Oryzias latipes inferred from allozymic variation.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">977</span>â<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D977%26rft.epage%253D988%26rft.atitle%253DGenetic%2Bstructure%2Bof%2Bkorean%2Bwild%2Bpopulations%2Bof%2Bthe%2BMedaka%2BOryzias%2Blatipes%2Binferred%2Bfrom%2Ballozymic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.21.977%26rft_id%253Dinfo%253Apmid%252F15459457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.21.977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15459457&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toyama</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1916</span>â<span class="cit-article-title">On some Mendelian characters.</span> <abbr class="cit-jnl-abbrev">Rep. Jap. Breed. Soc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">9</span> <span class="cit-comment">(in Japanese)</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRep.%2BJap.%2BBreed.%2BSoc.%26rft.volume%253D1%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisseling</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Primer3Plus, an enhanced web interface to Primer3.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W71</span>â<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253DW7174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.55" data-doi="10.1371/journal.pbio.0040072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A map of recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVoight%26rft.auinit1%253DB.%2BF.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DA%2Bmap%2Bof%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040072%26rft_id%253Dinfo%253Apmid%252F16494531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16494531&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.56" data-doi="10.1038/nrg704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittbrodt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schartl</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Medakaâa model organism from the far East.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">53</span>â<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWittbrodt%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D64%26rft.atitle%253DMedaka--a%2Bmodel%2Borganism%2Bfrom%2Bthe%2Bfar%2BEast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg704%26rft_id%253Dinfo%253Apmid%252F11823791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823791&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.57" data-doi="10.1002/jez.1401230309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">T-O.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span>â<span class="cit-article-title">Artificially induced sex-reversal in genotypic males of the medaka (<em>Oryzias Latipes</em>).</span> <abbr class="cit-jnl-abbrev">J Exp Zool</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">571</span>â<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BZool%26rft.volume%253D123%26rft.spage%253D571%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1401230309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1401230309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1953XZ84100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.3.433.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Whole-genome sequencing of giant pandas provides insights into demographic history and local adaptation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">67</span>â<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D67%26rft_id%253Dinfo%253Apmid%252F23242367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23242367&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F3%2F433.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/3/433.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/3/425" title="Double-Strand Break Repair Assays Determine Pathway Choice and Structure of Gene Conversion Events in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/3/447" title="A Dense Linkage Map for Chinook salmon (Oncorhynchus tshawytscha) Reveals Variable Chromosomal Divergence After an Ancestral Whole Genome Duplication Event" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/3"><span>Volume 4 Issue 3, March 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/3.cover-source.jpg?itok=6-EFRnwQ" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/3/433" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZJjdQ6i_5y8P1odP9Kl1pVr5hcJlT1MJLV_NGV3mcqw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/3/433" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/3/433" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/3/433" class="active" data-icon-position="" data-hide-link-title="0">Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459644" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JGd9O0TiZbnBORDF-dv7aoi5YJrY_W6Xbfc5KGmzXxQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/433.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/3/433" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-cwGMDS2iIEKA-7wVV5fw-vbVC0eEWuBePxyHaSNMfhc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/3/433" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/3/433.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459644" id="node459644--2" data-pisa="ggg;4/3/433" data-pisa-master="ggg;g3.113.008722" data-apath="/ggg/4/3/433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mikhail Spivakov</span>, <span class="highwire-citation-author" data-delta="1">Thomas O. Auer</span>, <span class="highwire-citation-author" data-delta="2">Ravindra Peravali</span>, <span class="highwire-citation-author" data-delta="3">Ian Dunham</span>, <span class="highwire-citation-author" data-delta="4">Dirk Dolle</span>, <span class="highwire-citation-author" data-delta="5">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="6">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="7">Tomoyuki Aizu</span>, <span class="highwire-citation-author" data-delta="8">Yohei Minakuchi</span>, <span class="highwire-citation-author" data-delta="9">Felix Loosli</span>, <span class="highwire-citation-author" data-delta="10">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="11">Ewan Birney</span> and <span class="highwire-citation-author" data-delta="12">Joachim Wittbrodt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">433-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008722 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459644--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mikhail Spivakov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikhail%2BSpivakov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivakov%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikhail%2BSpivakov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas O. Auer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BO.%2BAuer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Auer%20TO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BO.%2BAuer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ravindra Peravali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravindra%2BPeravali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peravali%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavindra%2BPeravali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ian Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dirk Dolle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk%2BDolle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk%2BDolle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tomoyuki Aizu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BAizu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aizu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BAizu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yohei Minakuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yohei%2BMinakuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minakuchi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYohei%2BMinakuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Felix Loosli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Felix%2BLoosli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loosli%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFelix%2BLoosli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Institute for Basic Biology, Laboratory of Bioresources, Okazaki, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ewan Birney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewan%2BBirney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birney%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEwan%2BBirney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joachim Wittbrodt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BWittbrodt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittbrodt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BWittbrodt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/3/433" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/3/433&amp;text=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/3/433&amp;t=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/3/433&amp;title=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459644/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459644" id="node459644--3" data-pisa="ggg;4/3/433" data-pisa-master="ggg;g3.113.008722" data-apath="/ggg/4/3/433.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mikhail Spivakov</span>, <span class="highwire-citation-author" data-delta="1">Thomas O. Auer</span>, <span class="highwire-citation-author" data-delta="2">Ravindra Peravali</span>, <span class="highwire-citation-author" data-delta="3">Ian Dunham</span>, <span class="highwire-citation-author" data-delta="4">Dirk Dolle</span>, <span class="highwire-citation-author" data-delta="5">Asao Fujiyama</span>, <span class="highwire-citation-author" data-delta="6">Atsushi Toyoda</span>, <span class="highwire-citation-author" data-delta="7">Tomoyuki Aizu</span>, <span class="highwire-citation-author" data-delta="8">Yohei Minakuchi</span>, <span class="highwire-citation-author" data-delta="9">Felix Loosli</span>, <span class="highwire-citation-author" data-delta="10">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="11">Ewan Birney</span> and <span class="highwire-citation-author" data-delta="12">Joachim Wittbrodt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">433-445;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008722 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459644--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mikhail Spivakov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mikhail%2BSpivakov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivakov%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMikhail%2BSpivakov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas O. Auer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BO.%2BAuer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Auer%20TO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BO.%2BAuer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ravindra Peravali </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravindra%2BPeravali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peravali%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavindra%2BPeravali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ian Dunham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BDunham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunham%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BDunham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dirk Dolle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dirk%2BDolle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dolle%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADirk%2BDolle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Asao Fujiyama </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Asao%2BFujiyama%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fujiyama%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAsao%2BFujiyama%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Atsushi Toyoda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Atsushi%2BToyoda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Toyoda%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAtsushi%2BToyoda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Tomoyuki Aizu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomoyuki%2BAizu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aizu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATomoyuki%2BAizu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yohei Minakuchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Comparative Genomics Laboratory, Center for Information Biology, National Institute of Genetics, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yohei%2BMinakuchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minakuchi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYohei%2BMinakuchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Felix Loosli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Karslruhe Institute of Technology, Karlsruhe, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Felix%2BLoosli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Loosli%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFelix%2BLoosli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Institute for Basic Biology, Laboratory of Bioresources, Okazaki, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Ewan Birney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridgeshire, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ewan%2BBirney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Birney%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEwan%2BBirney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Joachim Wittbrodt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Centre for Organismal Studies, Heidelberg University, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joachim%2BWittbrodt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wittbrodt%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoachim%2BWittbrodt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:felix.loosli@kit.edu" class="" data-icon-position="" data-hide-link-title="0">felix.loosli@kit.edu</a>
<a href="mailto:naruse@nibb.ac.jp" class="" data-icon-position="" data-hide-link-title="0">naruse@nibb.ac.jp</a>
<a href="mailto:birney@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">birney@ebi.ac.uk</a>
<a href="mailto:jochen.wittbrodt@cos.uni-heidelberg.de" class="" data-icon-position="" data-hide-link-title="0">jochen.wittbrodt@cos.uni-heidelberg.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=433&amp;rft.epage=445&amp;rft.atitle=Genomic%20and%20Phenotypic%20Characterization%20of%20a%20Wild%20Medaka%20Population%3A%20Towards%20the%20Establishment%20of%20an%20Isogenic%20Population%20Genetic%20Resource%20in%20Fish&amp;rft.volume=4&amp;rft.issue=3&amp;rft.date=2014-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Spivakov%2C+Mikhail&amp;rft.au=Auer%2C+Thomas+O.&amp;rft.au=Peravali%2C+Ravindra&amp;rft.au=Dunham%2C+Ian&amp;rft.au=Dolle%2C+Dirk&amp;rft.au=Fujiyama%2C+Asao&amp;rft.au=Toyoda%2C+Atsushi&amp;rft.au=Aizu%2C+Tomoyuki&amp;rft.au=Minakuchi%2C+Yohei&amp;rft.au=Loosli%2C+Felix&amp;rft.au=Naruse%2C+Kiyoshi&amp;rft.au=Birney%2C+Ewan&amp;rft.au=Wittbrodt%2C+Joachim"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459644/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459644/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459644/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459644/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459644/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459644/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459644/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459644/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459644/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459644/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459644/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459644/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/3/433" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-33" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/3/433.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/3/433.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/3/433.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/3/433.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
