<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/855" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/855.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/855.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/855.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/855.atom" />
<meta name="HW.pisa" content="ggg;9/3/855" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" />
<meta name="DC.Identifier" content="10.1534/g3.118.200789" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Fablet et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="All genomes contain repeated sequences that are known as transposable elements (TEs). Among these are endogenous retroviruses (ERVs), which are sequences similar to retroviruses and are transmitted across generations from parent to progeny. These sequences are controlled in genomes through epigenetic mechanisms. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences. The tirant ERV of Drosophila simulans displays intra-specific variability in copy numbers, insertion sites, and transcription levels, providing us with a well-suited model to study the dynamic relationship between a TE family and the host genome through epigenetic mechanisms. We show that tirant transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, unlike what was previously described following divergent crosses. In addition, we describe tirant insertion polymorphism in the genomes of three D. simulans wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs." />
<meta name="DC.Contributor" content="Marie Fablet" />
<meta name="DC.Contributor" content="Angelo Jacquet" />
<meta name="DC.Contributor" content="Rita Rebollo" />
<meta name="DC.Contributor" content="Annabelle Haudry" />
<meta name="DC.Contributor" content="Carine Rey" />
<meta name="DC.Contributor" content="Judit Salces-Ortiz" />
<meta name="DC.Contributor" content="Prajakta Bajad" />
<meta name="DC.Contributor" content="Nelly Burlet" />
<meta name="DC.Contributor" content="Michael F. Jantsch" />
<meta name="DC.Contributor" content="Maria Pilar García Guerreiro" />
<meta name="DC.Contributor" content="Cristina Vieira" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;All genomes contain repeated sequences that are known as transposable elements (TEs). Among these are endogenous retroviruses (ERVs), which are sequences similar to retroviruses and are transmitted across generations from parent to progeny. These sequences are controlled in genomes through epigenetic mechanisms. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences. The &lt;i&gt;tirant&lt;/i&gt; ERV of &lt;i&gt;Drosophila simulans&lt;/i&gt; displays intra-specific variability in copy numbers, insertion sites, and transcription levels, providing us with a well-suited model to study the dynamic relationship between a TE family and the host genome through epigenetic mechanisms. We show that &lt;i&gt;tirant&lt;/i&gt; transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, unlike what was previously described following divergent crosses. In addition, we describe &lt;i&gt;tirant&lt;/i&gt; insertion polymorphism in the genomes of three &lt;i&gt;D. simulans&lt;/i&gt; wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/855" />
<meta name="citation_id" content="9/3/855" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/855" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/855.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/855.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/855.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200789" />
<meta name="citation_pmid" content="30658967" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="855" />
<meta name="citation_lastpage" content="865" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marie Fablet" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_author_email" content="marie.fablet@univ-lyon1.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7819-4541" />
<meta name="citation_author" content="Angelo Jacquet" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_author" content="Rita Rebollo" />
<meta name="citation_author_institution" content="BF2i, Univ Lyon, INSA-Lyon, INRA, UMR 0203, Villeurbanne 69621, France" />
<meta name="citation_author" content="Annabelle Haudry" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_author" content="Carine Rey" />
<meta name="citation_author_institution" content="Laboratoire de Biologie et Modélisation de la Cellule, UnivLyon, ENS de Lyon, Univ Claude Bernard, CNRS UMR 5239, INSERM U1210, Lyon 69007, France" />
<meta name="citation_author" content="Judit Salces-Ortiz" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_author" content="Prajakta Bajad" />
<meta name="citation_author_institution" content="Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria" />
<meta name="citation_author" content="Nelly Burlet" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_author" content="Michael F. Jantsch" />
<meta name="citation_author_institution" content="Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria" />
<meta name="citation_author" content="Maria Pilar García Guerreiro" />
<meta name="citation_author_institution" content="Grup de Genòmica, Bioinformàtica i Biologia Evolutiva, Departament de Genètica i Microbiologia, Universitat Autònoma de Barcelona, Bellaterra-Barcelona 08193, Spain" />
<meta name="citation_author" content="Cristina Vieira" />
<meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=A. Akkouche;citation_author=T. Grentzinger;citation_author=M. Fablet;citation_author=C. Armenise;citation_author=N. Burlet;citation_title=Maternally Deposited Germline PiRNAs Silence the Tirant Retrotransposon in Somatic Cells.;citation_pages=458-464;citation_volume=14;citation_year=2013;citation_doi=10.1038/embor.2013.38" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=A. Akkouche;citation_author=R. Rebollo;citation_author=N. Burlet;citation_author=C. Esnault;citation_author=S. Martinez;citation_title=Tirant, a Newly Discovered Active Endogenous Retrovirus in Drosophila Simulans.;citation_volume=86;citation_year=2012;citation_pmid=22278247;citation_doi=10.1128/JVI.07146-11" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=C. Antoniewski;citation_title=Computing SiRNA and PiRNA Overlap Signatures.;citation_pages=135-146;citation_volume=1173;citation_year=2014;citation_pmid=24920366;citation_doi=10.1007/978-1-4939-0931-5_12" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. A. Aravin;citation_author=G. J. Hannon;citation_author=J. Brennecke;citation_title=The Piwi-piRNA Pathway Provides an Adaptive Defense in the Transposon Arms Race;citation_pages=761-764;citation_volume=318;citation_year=2007;citation_issue=5851;citation_pmid=17975059;citation_doi=10.1126/science.1146484" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Biémont;citation_author=C. Vieira;citation_title=Genetics: junk DNA as an evolutionary force.;citation_pages=521-524;citation_volume=443;citation_year=2006;citation_issue=7111;citation_pmid=17024082;citation_doi=10.1038/443521a" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Mol. Cell Biol.;citation_author=S. Boisvert;citation_author=F. Laviolette;citation_author=J. Corbeil;citation_title=Ray: Simultaneous Assembly of Reads from a Mix of High-Throughput Sequencing Technologies.;citation_pages=1519-1533;citation_volume=17;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Brennecke;citation_author=A. A. Aravin;citation_author=A. Stark;citation_author=M. Dus;citation_author=M. Kellis;citation_title=Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila.;citation_pages=1089-1103;citation_volume=128;citation_year=2007;citation_issue=6;citation_pmid=17346786;citation_doi=10.1016/j.cell.2007.01.043" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Carver;citation_author=U. Böhme;citation_author=T. D. Otto;citation_author=J. Parkhill;citation_author=M. Berriman;citation_title=BamView: Viewing Mapped Read Alignment Data in the Context of the Reference Sequence.;citation_volume=26;citation_year=2010;citation_pmid=20071372;citation_doi=10.1093/bioinformatics/btq010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=E. Casacuberta;citation_author=J. González;citation_title=The Impact of Transposable Elements in Environmental Adaptation.;citation_pages=1503-1517;citation_volume=22;citation_year=2013;citation_pmid=23293987;citation_doi=10.1111/mec.12170" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Chambeyron;citation_author=A. Popkova;citation_author=G. Payen-Groschêne;citation_author=C. Brun;citation_author=D. Laouini;citation_title=PiRNA-Mediated Nuclear Accumulation of Retrotransposon Transcripts in the Drosophila Female Germline.;citation_volume=105;citation_year=2008;citation_pmid=18809914;citation_doi=10.1073/pnas.0805943105" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Cordaux;citation_author=M. A. Batzer;citation_title=The impact of retrotransposons on human genome evolution.;citation_pages=691-703;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19763152;citation_doi=10.1038/nrg2640" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. J. Daborn;citation_author=J. L. Yen;citation_author=M. R. Bogwitz;citation_author=G. Le Goff;citation_author=E. Feil;citation_title=A Single P450 Allele Associated with Insecticide Resistance in Drosophila;citation_pages=2253-2256;citation_volume=297;citation_year=2002;citation_issue=5590;citation_pmid=12351787;citation_doi=10.1126/science.1074170" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=T. A. Elliott;citation_author=T. R. Gregory;citation_title=What’s in a Genome? The C–Value Enigma and the Evolution of Eukaryotic Genome Content.;citation_volume=370;citation_year=2015;citation_pmid=26323762;citation_doi=10.1098/rstb.2014.0331" />
<meta name="citation_reference" content="citation_journal_title=FASEB J. Off. Publ. Fed. Am. Soc. Exp. Biol.;citation_author=M. Fablet;citation_author=E. Lerat;citation_author=R. Rebollo;citation_author=B. Horard;citation_author=N. Burlet;citation_title=Genomic Environment Influences the Dynamics of the Tirant LTR Retrotransposon in Drosophila.;citation_pages=1482-1489;citation_volume=23;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. Fablet;citation_author=J. F. McDonald;citation_author=C. Biémont;citation_author=C. Vieira;citation_title=Ongoing loss of the tirant transposable element in natural populations of Drosophila simulans.;citation_pages=54-62;citation_volume=375;citation_year=2006;citation_pmid=16626897;citation_doi=10.1016/j.gene.2006.02.008" />
<meta name="citation_reference" content="citation_title=Evolutionary Biology: Self/Nonself Evolution, Species and Complex Traits Evolution, Methods and Concepts;citation_title=Self and nonself from a genomic perspective: transposable elements;citation_author=M. Fablet;citation_author=J. Salces-Ortiz;citation_author=B. Fraga Menezes;citation_author=M. Roy;citation_author=C. Vieira;citation_pages=111-128;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. Fablet;citation_author=S. Souames;citation_author=C. Biémont;citation_author=C. Vieira;citation_title=Evolutionary pathways of the tirant LTR retrotransposon in the Drosophila melanogaster subgroup of species.;citation_pages=438-447;citation_volume=64;citation_year=2007;citation_issue=4;citation_pmid=17390093;citation_doi=10.1007/s00239-006-0108-9" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Garrell;citation_author=J. Modolell;citation_title=The Drosophila extramacrochaetae locus, an antagonist of proneural genes that, like these genes, encodes a helix-loop-helix protein.;citation_pages=39-48;citation_volume=61;citation_year=1990;citation_issue=1;citation_pmid=1690605;citation_doi=10.1016/0092-8674(90)90213-X" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=D. Garrido-Martín;citation_author=E. Palumbo;citation_author=R. Guigó;citation_author=A. Breschi;citation_title=ggsashimi: Sashimi plot revised for browser- and annotation-independent splicing visualization.;citation_pages=e1006360;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pcbi.1006360" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=D. C. Hancks;citation_author=H. H. Kazazian;citation_title=Roles for Retrotransposon Insertions in Human Disease.;citation_pages=9;citation_volume=7;citation_year=2016;citation_pmid=27158268;citation_doi=10.1186/s13100-016-0065-9" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=D. Jangam;citation_author=C. Feschotte;citation_author=E. Betrán;citation_title=Transposable Element Domestication As an Adaptation to Evolutionary Conflicts.;citation_pages=817-831;citation_volume=33;citation_year=2017;citation_doi=10.1016/j.tig.2017.07.011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=K. P. Johnson;citation_author=K. K. O. Walden;citation_author=H. M. Robertson;citation_title=Next-Generation Phylogenomics Using a Target Restricted Assembly Method.;citation_pages=417-422;citation_volume=66;citation_year=2013;citation_pmid=23000819;citation_doi=10.1016/j.ympev.2012.09.007" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. S. Kelleher;citation_author=D. A. Barbash;citation_title=Analysis of PiRNA-Mediated Silencing of Active TEs in Drosophila Melanogaster Suggests Limits on the Evolution of Host Genome Defense.;citation_volume=30;citation_year=2013;citation_pmid=23625890;citation_doi=10.1093/molbev/mst081" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT–the BLAST-like Alignment Tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Lassmann;citation_author=Y. Hayashizaki;citation_author=C. O. Daub;citation_title=SAMStat: Monitoring Biases in next Generation Sequencing Data.;citation_volume=27;citation_year=2011;citation_pmid=21088025;citation_doi=10.1093/bioinformatics/btq614" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=A. Le Thomas;citation_author=A. K. Rogers;citation_author=A. Webster;citation_author=G. K. Marinov;citation_author=S. E. Liao;citation_title=Piwi Induces PiRNA-Guided Transcriptional Silencing and Establishment of a Repressive Chromatin State.;citation_volume=27;citation_year=2013;citation_pmid=23392610;citation_doi=10.1101/gad.209841.112" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Pervasive epigenetic effects of Drosophila Euchromatic transposable elements impact their evolution.;citation_author=Y. C. G. Lee;citation_author=G. H. Karpen;citation_volume=6;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Lerat;citation_author=M. Fablet;citation_author=L. Modolo;citation_author=H. Lopez-Maestre;citation_author=C. Vieira;citation_title=TEtools Facilitates Big Data Expression Analysis of Transposable Elements and Reveals an Antagonism between Their Activity and That of PiRNA Genes.;citation_pages=e17;citation_volume=45;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map Format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_title=The Genome of Drosophila Melanogaster;citation_author=D. L. Lindsley;citation_author=G. G. Zimm;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated Estimation of Fold Change and Dispersion for RNA-Seq Data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=R. M. Marsano;citation_author=R. Moschetti;citation_author=C. Caggese;citation_author=C. Lanave;citation_author=P. Barsanti;citation_title=The complete Tirant transposable element in Drosophila melanogaster shows a structural relationship with retrovirus-like retrotransposons.;citation_pages=87-95;citation_volume=247;citation_year=2000;citation_issue=1-2;citation_pmid=10773447;citation_doi=10.1016/S0378-1119(00)00115-3" />
<meta name="citation_reference" content="Martin, M., 2011 Cutadapt Removes Adapter Sequences from High-Throughput Sequencing Reads. EMBnet.journal 17: 10–12. 10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Mi;citation_author=X. Lee;citation_author=X. Li;citation_author=G. M. Veldman;citation_author=H. Finnerty;citation_title=Syncytin is a captive retroviral envelope protein involved in human placental morphogenesis.;citation_pages=785-789;citation_volume=403;citation_year=2000;citation_issue=6771;citation_pmid=10693809;citation_doi=10.1038/35001608" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=L. Modolo;citation_author=E. Lerat;citation_title=UrQt: An Efficient Software for the Unsupervised Quality Trimming of NGS Data.;citation_pages=137;citation_volume=16;citation_year=2015;citation_pmid=25924884;citation_doi=10.1186/s12859-015-0546-8" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=F. Mohn;citation_author=G. Sienski;citation_author=D. Handler;citation_author=J. Brennecke;citation_title=The Rhino-Deadlock-Cutoff Complex Licenses Noncanonical Transcription of Dual-Strand PiRNA Clusters in Drosophila.;citation_pages=1364-1379;citation_volume=157;citation_year=2014;citation_pmid=24906153;citation_doi=10.1016/j.cell.2014.04.031" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Naito;citation_author=F. Zhang;citation_author=T. Tsukiyama;citation_author=H. Saito;citation_author=C. N. Hancock;citation_title=Unexpected consequences of a sudden and massive transposon amplification on rice gene expression.;citation_pages=1130-1134;citation_volume=461;citation_year=2009;citation_issue=7267;citation_pmid=19847266;citation_doi=10.1038/nature08479" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Rebollo;citation_author=M. M. Karimi;citation_author=M. Bilenky;citation_author=L. Gagnier;citation_author=K. Miceli-Royer;citation_title=Retrotransposon-induced heterochromatin spreading in the mouse revealed by insertional polymorphisms.;citation_pages=e1002301-e1002301;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21980304;citation_doi=10.1371/journal.pgen.1002301" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=K. Saito;citation_author=K. M. Nishida;citation_author=T. Mori;citation_author=Y. Kawamura;citation_author=K. Miyoshi;citation_title=Specific Association of Piwi with RasiRNAs Derived from Retrotransposon and Heterochromatic Regions in the Drosophila Genome.;citation_volume=20;citation_year=2006;citation_pmid=16882972;citation_doi=10.1101/gad.1454806" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Schmieder;citation_author=R. Edwards;citation_title=Quality Control and Preprocessing of Metagenomic Datasets.;citation_volume=27;citation_year=2011;citation_pmid=21278185;citation_doi=10.1093/bioinformatics/btr026" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=K.-A. Senti;citation_author=D. Jurczak;citation_author=R. Sachidanandam;citation_author=J. Brennecke;citation_title=piRNA-guided slicing of transposon transcripts enforces their transcriptional silencing via specifying the nuclear piRNA repertoire.;citation_volume=29;citation_year=2015;citation_pmid=26302790;citation_doi=10.1101/gad.267252.115" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Shpiz;citation_author=S. Ryazansky;citation_author=I. Olovnikov;citation_author=Y. Abramov;citation_author=A. Kalmykova;citation_title=Euchromatic Transposon Insertions Trigger Production of Novel Pi- and Endo-SiRNAs at the Target Sites in the Drosophila Germline.;citation_pages=e1004138;citation_volume=10;citation_year=2014;citation_pmid=24516406;citation_doi=10.1371/journal.pgen.1004138" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Sienski;citation_author=D. Dönertas;citation_author=J. Brennecke;citation_title=Transcriptional silencing of transposons by piwi and maelstrom and its impact on chromatin state and gene expression.;citation_pages=964-980;citation_volume=151;citation_year=2012;citation_issue=5;citation_pmid=23159368;citation_doi=10.1016/j.cell.2012.10.040" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: Assessing Genome Assembly and Annotation Completeness with Single-Copy Orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=M. C. Siomi;citation_author=K. Sato;citation_author=D. Pezic;citation_author=A. A. Aravin;citation_title=PIWI-interacting small RNAs: the vanguard of genome defence.;citation_pages=246-258;citation_volume=12;citation_year=2011;citation_issue=4;citation_pmid=21427766;citation_doi=10.1038/nrm3089" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. K. Slotkin;citation_author=R. Martienssen;citation_title=Transposable elements and the epigenetic regulation of the genome.;citation_pages=272-285;citation_volume=8;citation_year=2007;citation_issue=4;citation_pmid=17363976;citation_doi=10.1038/nrg2072" />
<meta name="citation_reference" content="Smit, A. F., R. Hubley, and P. Green, 2013 RepeatMasker Open-4.0. http://www.repeatmasker.org." />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. K. Teixeira;citation_author=M. Okuniewska;citation_author=C. D. Malone;citation_author=R.-X. Coux;citation_author=D. C. Rio;citation_title=PiRNA-Mediated Regulation of Transposon Alternative Splicing in the Soma and Germ Line.;citation_pages=268-272;citation_volume=552;citation_year=2017;citation_doi=10.1038/nature25018" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=M. I. Tenaillon;citation_author=J. D. Hollister;citation_author=B. S. Gaut;citation_title=A triptych of the evolution of plant transposable elements.;citation_pages=471-478;citation_volume=15;citation_year=2010;citation_issue=8;citation_pmid=20541961;citation_doi=10.1016/j.tplants.2010.05.003" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=C. Terzian;citation_author=A. Pélisson;citation_author=A. Bucheton;citation_title=Evolution and phylogeny of insect endogenous retroviruses.;citation_pages=3-3;citation_volume=1;citation_year=2001;citation_pmid=11591216;citation_doi=10.1186/1471-2148-1-3" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. Vieira;citation_author=D. Lepetit;citation_author=S. Dumont;citation_author=C. Biémont;citation_title=Wake up of transposable elements following Drosophila simulans worldwide colonization;citation_pages=1251-1255;citation_volume=16;citation_year=1999;citation_issue=9;citation_pmid=10486980;citation_doi=10.1093/oxfordjournals.molbev.a026215" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Wicker;citation_author=F. Sabot;citation_author=A. Hua-Van;citation_author=J. L. Bennetzen;citation_author=P. Capy;citation_title=A unified classification system for eukaryotic transposable elements.;citation_pages=973-982;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=17984973;citation_doi=10.1038/nrg2165" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="All genomes contain repeated sequences that are known as transposable elements (TEs). Among these are endogenous retroviruses (ERVs), which are sequences similar to retroviruses and are transmitted across generations from parent to progeny. These sequences are controlled in genomes through epigenetic mechanisms. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences. The tirant ERV of Drosophila simulans displays intra-specific variability in copy numbers, insertion sites, and transcription levels, providing us with a well-suited model to study the dynamic relationship between a TE family and the host genome through epigenetic mechanisms. We show that tirant transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, unlike what was previously described following divergent crosses. In addition, we describe tirant insertion polymorphism in the genomes of three D. simulans wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs." />
<meta name="og-title" property="og:title" content="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/855" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="All genomes contain repeated sequences that are known as transposable elements (TEs). Among these are endogenous retroviruses (ERVs), which are sequences similar to retroviruses and are transmitted across generations from parent to progeny. These sequences are controlled in genomes through epigenetic mechanisms. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences. The tirant ERV of Drosophila simulans displays intra-specific variability in copy numbers, insertion sites, and transcription levels, providing us with a well-suited model to study the dynamic relationship between a TE family and the host genome through epigenetic mechanisms. We show that tirant transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, unlike what was previously described following divergent crosses. In addition, we describe tirant insertion polymorphism in the genomes of three D. simulans wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"e3nc17rT_l8tIuH09US7iYa3CVgY21oQrqyASBbbr_4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492323","apath":"\/ggg\/9\/3\/855.atom","pisa":"ggg;9\/3\/855","ac":{"\/ggg\/9\/3\/855.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/855.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/855"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/855"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/855":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qvcmW7pKUtNjlFlNrqeDxxot+56VPvZQkPIY16AkYHm+ZKfk3W43Pen4LHdjWuO+w8XMQAmo5TsX9F+zpfC96IsGNO5TTcN1s8stxsawx3kVD457k\\\\\\\/mILujHu8YzkeObhr1awElEO4pm15PM78Um4S6pVnwHTUkmEvCl35Lx7a2X3w7sPUuwtCVJKTM4AWrTOUoVixNN79+uDmytOxwfZWat4w5x5tG2LhLqL\\\\\\\/nstJSIRiD1oWDsLC+JYBN95+KgUaVzjNu8ZSZor7TBXmjb8uzmx0nK3riEc\\\\\\\/zQxtAXbqxKanLPub4Ykn8GUWzvVW24DyL7t\\\\\\\/m7FlSn5AOcWV9u4XfODGP7cwUJIxAugKOwJaTvzF\\\\\\\/vHuugCR8b0UQrlRUBJc\\\\\\\/wGey3L3MsZccPQPH+G9jJ0CHWWRnXNPDwZB6vUQ3vLZXneqL6o1YhmJwRdFrWYrtIHjD7ALitlTcLXKo7QOhtaAdSVMoqokxXDWExr1xlR7lRccqdJzRpRh+uZ1iOt4XqWReRPslQBc6G2zZG9uwpgayEZA44f3QU\\\\\\\/ahHderqEDzW0q3R2B3y\\\\\\\/QKOhT5zgP2KFWkVhshY9mFISLDA5iP2IfER7hmbBjyQwUUZVOLWuvngSyNBmlAfkQV5xpmDlkNLqaW9+nYfU3T0hQBO0a0rKB43FheJ4JoimC5uJFSZ0xAC1VmtRAKuCSmDvesVx6m+FvcifEG2GZluPKaKSFP7RkcJDty8zuiWCGzZLLlXhVbddI3iL5B7cdieAa6AdL6S785dIyWEzKmHtGAPmLNicP344FfHio3b2fHYngF5dhi42164UkhCtnoYw3GFdWgQcppw9qu7DlKn6Cq1kIiBLxmtDC0stB7Rb5B0frXrRgYf8VWOwMncW9kcHIAgj\\\\\\\/KoMR91aIYu00sLWmzufwvuo23Wgl2PCn5+LqHvQu2b1sI9Xjmhq5J0UguiW1b1IpLnTrGK+wFtBFebI6AQBaetU\\\\\\\/vugZ1ssB2uqXRlmTDDhsYAEou1E16sN5OzUiiWK+Tq3vfLF+q5ofCrd450em9sKwCjtgPICQJziFTfdYeTf66d2y1deC8MtqyOj\\\\\\\/F2LsPzDnv6x+Tgee1CjoqJq1licP7BoVTYjEQqloYvQPCIakBVQ0xcVtrgXJdSayWdcuHXjhoR1dvDkL3qxaOzx3zPhWuijTuuFxv1m0z0p8XPHOLZEFuabKrzZQ5lfQSRX6YWTvTG4WTG4L8kePt+\\\\\\\/IQrIc+gr38Ngo6hK9k8wUJJUfEl612Ij2e3srSzG+Ig\\\u0022,\\\u0022iv\\\u0022:\\\u0022M0SJrXVQFCXN5RpE3pvNwg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9c6bd811cf1cfff09098e0c8074edb470a1382356493f9896cca49afb102145\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022X1QpDAQ15swZAYjlFGCIdxSqRL9CLAa5Ajtji4dciE7NYsAE7CDJ0Gl2raRF6NlLgxZOqSH3Zvxx\\\\\\\/lbPzmalGKoirBtapxMYxDLmxkv09Cbn+sjXynsWoCDjl9wG1ZUCZuu5sx8tX2IMiUw\\\\\\\/sLC6U6J1wJ20oio44tL1i9y4k8xN58zHTiVBTosvM53q\\\\\\\/EEvIMpttuyIeBXt2XfpDqBHh60yy0TBAzEQSotqC6\\\\\\\/jqJIKKEeve1+D+sqIUiGqKA1dIr\\\\\\\/0\\\\\\\/m0hijZOsmJnV2yLL7xHqrzoKlb8q35nOLsgj2Kd6LPf8nRs1SMEZ2x5qZSlEd\\\\\\\/gZMgs\\\\\\\/Yxgr\\\\\\\/+OYSL\\\\\\\/kgBAi\\\\\\\/rOwHyif9eKJ9b4uBLI7+ZmJkhYU7jBKQA\\\\\\\/KipaLLU\\\\\\\/iso6QA\\\\\\\/qmVuTB6uArFvvuVGG9oTqqqoYQvGIVvB8XoMCRmYIhndonbXYVegO65zw5Kf2Vk0PtyZ\\\\\\\/0SMfl0fAh1gJtWpqJkKrdJwIT2c7eWvgjPRoKQb8snwesHcQILFpEdfdrghvmCF3HzlkGJXk5NHGOE8\\\\\\\/PH9Z0hpzanWv9tYSS5cLbRhacXC\\\\\\\/3l8LZ9P+tpHczm0359SFh5vm5ob1Pq2RgSpfiRkjeLjSJn2zeK9lQKh\\\\\\\/x81JNtwk270jkTPIA1objK9AsGosirUN6tQy9PqhrZHxQMM5oYNGXU4FJzPY65Vd3H52rmKB7lO6FsUBp5ifSlhslpxioX+WCjCzP2oPPpGlKFc2T6fJkGEDveuRu9Q+7K50M6f5jegVfJMPE3H45oTTe8laFraFzwdyswnHfG71mXH\\\\\\\/LR9Sz2fMO\\\\\\\/XEljL33KEw46qivdDIK0LjgwB+vm4PhO+gfsU8SkIEIGIeYJdZ5q23z924LkYR2eGU1W2cLEUWThrHTcmWgroZbX2MTKGhSGNj6yqYvWTQvofc9GBUGHt3X\\\\\\\/2cRY04NXtOiaQOKUctN6rsQyActdoKkWarNHfoAYbVlCGgf0Mdam5paQxAMW8hffvNAKBTdvubUHcI8Z185H3n89LoZR4SJxEt7QP95gloCSucxlrev8TdIUgjKpe33xF7OltAvAcDRupv3lEdxul0zRZBIo0tgEmPNojYqPDvkRtk9CvauND3xpDbahNuXen99Am4Glg98T\\\\\\\/ncBMnB6B\\\\\\\/WJayXRIgwtCJTa4nKyeTo4EpUEy\\\\\\\/S5atznh79GJVKmaORfP1jATG6ukQ2HfEZ4E18Ks7sAWPta+\\\\\\\/iox5DYOakefnMshvje\\\\\\\/2idDmM6t4bheYI+IgU\\\\\\\/IBmKW73wQY2Yvg+J3xV9n+6VUEYjbdkQY4+zYFX\\\\\\\/eq8Wse+qykS6t1QhgNj0DlhQuIzLwNhtRUvcxeS5gZCioVYGtrBqQNl\\\\\\\/17AZNeRlaPXOfsOneK4m1QfJgGADZdtOMW0Hs8EDmFlFTMKTEQ5W45ztppfD4MrqIACpscvWfDS6VqoMQgSSh+mvaqQ6AQ\\\\\\\/7hyOrvFWrYgZ1kWw+5ztqkMkoiVF99O3PFwpY3PJM\\\\\\\/qYLE95bI2dz2fyiLbYc2MjyAkogDYtgZCs9rpE5b6Lp4aEJfXd+KqIw\\\\\\\/fboLwim9E6SeumbySFBZnf4KP7KaDsdktu2R+dxBwmIrOKQbq7XLI6RcQsxnBCR\\\\\\\/78cjvkgYXNicSRgdy3JeBK7wTOkBnKBeWH6BY855yP2dG41SDgd1n5w6KQqrUu1UHVQsVitakCW0CxZ6tZJ5FZX6P3SW\\\\\\\/0sJgQ7KbgmNXLWyaSiSi4WMb+Rljdgiu2lyI7joZge9XvploJ6DAMg5YS8yz70J6UL8JlQhrebwciEOCXQMfILY\\\\\\\/Wpdn4CN2iYhc6PFCIXUVb90pNVhZQHk9tfeWRUE7+ZXZciD8LemZqI3S0itSRD4t6Ifidj0PFfFdUvhfNhfyaz6d78kAj2elfZ+P9TgAwpklZcDcf1S7ua61aihh6CSxUV2L3Q7PWz5ZY9ILuS0P0c1\\\\\\\/GMjt5+\\\\\\\/STgYXPkmnrF9AYajBqT3DHH1me4h3cTRULhwoUyu6mQkP+9bENL3YwH9lQQ9GIml7e8gOPMCXp0jbwL6Cq9qd21xDKYWQsZnXu5IaNTFcyMTVrm1zFXH2Yg4pEbAuP4zLt4796DQQQzMdKMfDmdTArxqr+Pw\\\\\\\/eFmOQsJ8ncQq1fky0sQB0sLHJ\\\\\\\/12xggDuJYklcgP0ZfdJQLyy8LqgLgwH3l7G1u4g\\\\\\\/ooNzH0WpaICu8uOKP4xfZj6XECybdqNCdTTarA6aKvTMRcYNBiyEjoOsJ\\\\\\\/EHnBpx1dn2rDothVwoNKkWbeQ2lnxtP69NRD2XHBO4g+np9OV0uN0NcRW7zjT+FAz50fParauethbnocUna8THMzNcTBjDylT6wdPCQgnxM4aoHgldnv6xj3zgn1NcX0OKoDcIF4YTGknvgKd1sL9THHubB4oumYGUOffCNp4Zn32eAM6LKhUwXFHyLJVIX+2ACeNoYfji7iqBBdiR0yvpnMWoW\\\\\\\/cBShwnQflzGvelkQIIh87s0iya42L5ysOM7D\\\\\\\/1ExVvwIykJOVTjULZqMSjdOjHjzo680RwGmFkba90EHVKMpbkHbvykcF2PUQ6MWt0O0xxJCTncXy\\\\\\\/1BuixXHX8TKnhu3jkEm0Z9CVwgzkwT1GKr\\\\\\\/bEsy0KGfgHY0NjTOp\\\\\\\/UuQHTp427A20PzjutoEbGq8ylpQKhz5Sym87Y6Mt053xAVOPVcI7B9Qhpmc81wi+nCUeeED2X3QoQY9Vv49bxW5qOqoVUj6Wdws8MC8GnwIi4m2pJxUGM8\\\\\\\/8Ca+woFo\\\\\\\/dzPmlgV3cXEaXMlAI4RNlj7zQqX6Fzb8gZ3yZ8LS4W8nPdqKM4wcQaQtKBrIYI02nUcVweZ9QTr53gfChsJi1nUIym0Cdmkh2baFlaI6JrqiGJb7AT6fwuSTdY\\\\\\\/a4n8Elg4G8aikMSnGaLTEhV7NWvlWnJmIxKe4UDymzz\\\\\\\/pSxiv0BE3+awZq6dhKdmPSrYq1ZLzwQGD+rzGXLYIRK68ruITavmEEt6Z2kyviuN2NbXT44qynabmH8z7b80JyJrSbXqm+6CcFSU6mDVlt3ez8yyZ9w4mBoZe9xhy99RXjGY9P9Ws69I6TlPgzB83E9\\\\\\\/I14O\\\\\\\/86cr8EyUstIL0NbPY\\\\\\\/X2v91qBLthY\\\\\\\/gHswD3aUSoO4T8\\\\\\\/NhEMzVmecfmoyUrtZXKCIjd1Mj4l7PqdmP1kJRV3Oynu8TqL+Aith+E6bOPJCVcLhb3bfYlonepIvQ7auZcDRTbyZpDwo+Tc5URNSghyi4HlSUzC37lBU3vudUzgHh5QxEfde9L9E0+Zwc8xDFbDMglyJ1\\\\\\\/+C\\\\\\\/Gr8YzOWUPZjqmO0ihJZ20G2\\\\\\\/xN6zSPJ4BJHoTGil3H4pse10XL04cubQ3GLcUxTbAkpYpaKb63IAPx8kNP5lZC1HV33UPZuo5nQ454m3CoS+YRNPUkmzdtjxF0jC1E9rRf4kXkGO5Ugf50p2wPm0Jyf6Yzw5AdoFBGWiguNy2+ak7Y2mNiDsrUZoQxbm90n3PFrEWyVm+nX4vDOxDEyfLp\\\\\\\/fPSHMD4VTvRVd40tiiI1gga1BF08JrryobcRrz\\\\\\\/gBvHSrMHCBBtoLnAQ1w+kSeU+sDYXJt5qPxO6LlNKjXh7UH7AQa6UgFv9LFyjE0GUXD04lHxEQ5zRtrmgCQPPfLq3mmskXtxFCVV5mg6eGYp3eIXmWwei9s4EVeeazDkMmfbx0OqAWpgN\\\\\\\/GNNbYflFREErTxSAfIjCYVoYoEKxMn2phIVRBQGmpaRRB94PU0\\\\\\\/2cZjJyqAeyS5PAOd2BcpXmiNpC2dz\\\\\\\/6NRApHeHjqo5+aXJhYBbo9Xx12RREXPYy9a3U99wJJOzhW1dazBPxkG9YsS2tugc1TnvHx3xcX6rUXbiMt0sBhXe1I2r\\\\\\\/8NwEcI4X8Mqcq1rUcZOnTBshQejuz2CDiuXeUoilYCQf+3LjcRJz04nyNxUNN2cFJm92PX2jjiZBvuHWgS3Sm9L7iBuWnLD5Xj9YBSRBJLu5gvSjGj7NTaqiEvHlk5DDXBiYG1q12Sqi09S7g8gDfNMXP7\\\\\\\/56F+LqTY0Ly\\\\\\\/3I8OpaBvQ1tXP\\\\\\\/B22z04xE9ypGm3YqBiUZZLcuOKYk+KzmaxvcIOp0CSsRJpHpe4tHRUdIT3PlpRd+ViicQW0zG7WZqLSyHbL0fyS1\\\\\\\/8A1aa65s5lTah1qs7hjy\\\\\\\/lmn+TTPX2C3bt3Pcvx9he\\\\\\\/qzWwyx9mhMlPk+Zj8cpkVjyazpZM0ogf4pcAyjWTZC92UKtMfgAN8yROpt4Bjf3TIZ92OY3GjmYjXDp1F9SmVAdSybxVaz0bcilYQuQveedR0nVehpSEceRw041OOlbTF4PY8zl4f6q14tPsYXS2xM\\\\\\\/Vktn8LnS77gLKgCuDPqooeJcrCkfMb0qjo\\\\\\\/xXa7o4Xfg5e02JtOJY1\\\\\\\/nsXW\\\\\\\/LFs1xQbvQ4EsN8r70TkIohh+AFsSoCMiLlF2ay+qfKA7LBCmksti4++jbals03dl73\\\\\\\/9M+MwhBc5FPtPArBvQVRw77SJX6nAqo9FZfz6OOE2QpeGsNOOF3zACAdBcd28rkQEP21+kGuErO8Qmtk+Pef0QuEMOqTvYGbCFBm3MSqAGo683ULClA095aEvWqLqioszeGgmIpJdm3\\\\\\\/0Qc4kdKVGAPM19Miou2Qb1fs6W2gN0uDJGB6O6rax8mI3QIaO70hB5ATY78acaDSOb7JQ\\\\\\\/OQNfpussCGxcy4TbfbmUwS\\\\\\\/+JaajF+KAQ+wqJNvt6qUQAgqvJAjvxNLzUpFog4ggS0rrGtNgXu8zyxoa5pPUpNqM+x844S4qDmXCC7FRn33zXl2hc2I7UkVUulc6yE8Xdq249xv8DQ53aOcdIFSPAyY8G8\\\\\\\/ewqX2T2i3c042gZJiPB3Fhg5jZo3gbby7FYLkOC5WcGookQI\\\\\\\/u\\\\\\\/jB33qfDxCZpxIsejjoqc6qRmvi+z\\\\\\\/K9VK5\\\\\\\/0ci0I5o2mcIj87nuXxj3j2BidrBWDjK4ZxrABriuyLwsljlpJGk5awi5Q7x5rJR5iB6a00\\\\\\\/c76P+x7Rk7XCQjNdsgSlwVWUzwIzTiO\\\\\\\/PEbv\\\\\\\/e26FvT6uLfu1hLN3OtbolHOMnX6N6Xzop+LThLN4s9N8V4bavmZpLdTq\\\\\\\/oRYI1NXPGiin81Toh0e9J03oAjzJV5dq8bFJqYEveirf862HYJfS2wX8QfsX2fh3vycKEaYnXYE\\\\\\\/R4TiJzU1mpsLwFSNY+q0S3vCzI+fDcDe5UPkf5uaUjRkTGVVXvhw\\\\\\\/bElzO9gxtlhKEyiduKUiQr+C+VhoVC\\\\\\\/nZJcWC4hXKHFXLgIYjjf5ESbkRLHFjb4CcD6\\\\\\\/Cr2lt+n4FeY\\\\\\\/fe7c538+pdC4hRS6OvwhIkKYHMIc0DaZOmcuU+TEfxDE+D6PaAynS5+9aMwNlQ+0vbnK6hqlW0jKlfL12x9Zk6VdGuojg\\\\\\\/Qf4KCkBzVVPJEsLFhyCn0QMpYVCwqiM7gLMG59DnhRNvZScBVrYwqDs7IfZWzBa\\\\\\\/RTcpTN+9vw+m+YZhHLsLr1FO0cXUrCjO+0QX7A9sEhneQWR\\\\\\\/jl4G+DCwd0VYbT2IF5MiWqGEVhHNLERgb9iYFjHQkMtZ9ThIt9PcvRq73+WsJNofS0OO+yTm0N3zczaCFzY9XnPsVWs1Ro8R\\\\\\\/jduEsQNgAz9C92Q0mC3ZGKjoyOmPEA9vkqXFSfJ8d76Ert4P6raBUGnsAU9Qvc3s1z+Fporkx7jI1s+GIHaqCm3AFsC53ZQIk04jd5qPlKjtYBi0KFB13FGYTDT9TMmed5+1dl3XRFTdM2CuY6Ywcc6Dkq0xc9kILmjNUddoGkavJKZsApaPzhUVvn1rugB2UhsE\\\\\\\/yTEqpeyb6pPWbWLVZTps5l\\\\\\\/QDWxosTGArXROJJiT8U6gSbBWAD7EUn8yRSMi5ghl3MPdAMIeW4PoaFf387\\\\\\\/4Vu\\\\\\\/Tks4tNzs9qiy8cSAUcEpC7lkynqVkWap4gOOlx8b9OCBJ9RGiDL838NC1YxqNUoLbMtzBSP9jzyPovZxg7RsE7x75ye1VQx63VVd9NAFOZtIoNozt3jM75yKwXKEUPYzdLMQYL6Le6PcybCcJJKy7hDfTrpzY1rmhRHKdtoe7oK2EXFWuwb07epC\\\\\\\/mfFKKTVHnWBjTqqm\\\\\\\/r207xQV1H+RtzGz3443QxqUpWZeTkJ5yJXv17S8TYTEoHZhv0lk1dOE2e+dM\\\\\\\/Mh3WTNfQNxOokiKsNmumM\\\\\\\/GDiXv5LZbQ8jtSjaMfU4oVvnJp4Wod\\\\\\\/1iT+oG0XUMd0OaI0iWQHe1NKxMr2dWxJLWFlJ7clvFqoUgbzRd+dkFNp109tn4Q4mr9T6coTUhHV5B7W56qMszrjjjPpvLUfak4k\\\\\\\/qzfMI1aZMDgLEuH4uIQNgGueJRPzUcmdMy7sT8nYLavrQLzXdmjJjerqcMX1HUdg20ctdzrtL5bBYW1xQTimAcMhEvvQgI0Lhb1pFI0RSF93Nqgg6fvhvJZvVE1xciTKdOkk5op1PiamaeIaiBKqVclFv9yNVGwmQu7Xe7yFgdQ\\\\\\\/nxD\\\\\\\/54JsJZKv7A7s5sPDfE+0i5KTnmsubhsHlpV+EiaMUMFj8VZL\\\\\\\/uOtfVEQv6lH\\\\\\\/tbh4\\\\\\\/ogFzb4EcAl6aHVAeF\\\\\\\/vuzsO9HXpYxEr+lVaWQBbiwYdpkhT08p3NPdoOwoFMp+b2Hj6q7y0P\\\\\\\/tdBri9L7B61m5sYJlX0Xv1YZnS9tuiipEuE80bZNwtGRnTGH38PFu0f4z5B+zTJv7mByTtth6s1EXVGXvv+3DCHt18Fm+hzoVAsU6YpZVwCSKUdmTo5t+Jw9Bua6N0\\\\\\\/lAZ\\\\\\\/mw9KwIMAwUFs\\\\\\\/N9seOs\\\\\\\/DDExcq7SCt3Zhuz2W1aeBMLcpDNdqZZLKY1\\\\\\\/geuS5FUPsHZWUVUqQ5NwyEMW3L6q0qpMpzAXImrShuU6KvE0sTrgN9sxJye1S9WmoOxq9oOqPWXukuA4X1gdBizvPJuCclxVeyYna+7PZJMtISr\\\\\\\/8mUrU0VL6VbjaUGBXDU6k\\\\\\\/5yCjo8U8D66Fa8elqnziR6rYNfprAXxElp3hAvVxNczVYOhVhk75MurGSBuTCL+3pO02key2ddYsrlq+qp4+gtpzlfNHsaiBf737GpqWJsJCMLyIPEyOOSdlLIp\\\\\\\/S5EsB8CFtvx9giq1EYgvA3uoZFy0dFSr10Vhg+j3jA6jj7CX41b2nEt96bVqeqkV7xyTpk5PUlvpwxAeDMIyAuZf5WxM4RiNHX0jNzSmtT4n85NWl7GQTPHtyEJwHi+4URu5FiKCjzJuUiH\\\\\\\/PQxPV0E5SaY7UqitondeFOc8\\\\\\\/JVGjSwhmN3P1M6W2viQQkIyRW\\\\\\\/GCdcS\\\\\\\/lPkDLPjPlEE\\\\\\\/q\\\\\\\/TpeGbchM7Sb611grJWt9cm2l6N\\\\\\\/UficrqWkYUWI2Aqty28ayYixewAD\\\\\\\/LxYmdZDg6UjHipT23CHcwcBpH3eijnaqlZabLBED+I16FMiEzf9fnsQkM1r1xvC9t3EFqt7Efm3sFkMyNyeyZtA06WM+vjkScKOaSnCmArhcKYG7FR17imjoXGwehVpc\\\\\\\/U6d5bOKj1derEZsC6kDqnsDK8Uupf3Q0VaW95yMfKE1pCJdOFxEUTNOX\\\\\\\/Dvb9iZCBUrxW8yCS5MPynx0uyuJFVsfXiKzL6eH6PlSuztiffX35e6GIAc2Q2DFkeB6LI0+ZRHCc6FQL5EiaaiTdxt2x0bMX+NUM1b+fpbS+zB2OoiPT+zbzoUzv6lY\\\\\\\/hSB8eTJdMSgT1osrPmvDXDsSpZ9EjLj+i\\\\\\\/K3dXLSl\\\\\\\/SXbH0r10r1wW2fz3zTIB\\\\\\\/faIJdzsJOfaw6p0PH4Q5jviGjbQbpekB2\\\\\\\/A7poHTafoCzWFtlZLrfk6bHCZ4jSyvoXkFUQnvJrCCZDDOLu2wHhiU8nnVMymCZmAqr1MEkMbRsJyF82nAQnUvUVPbQklRqVfWfcanD+SgoV9cQH\\\\\\\/RjNf63FXJn5vxHCw47oPtiMtrys8vo+LTwhCi2nb9Z5Klb4x2v8Ez3n+r1yF7+kVvOglcyRjAwVS6xe0dWUW9VGSzuQeD5gnBE2gYDaA4rns8EV\\\\\\\/LywP75WAyNmgq5heQZNhezdlAoaNvKxx5QgSRSFlv19KX6ifXxlbLWbx8HBHPNqOlCVdx8\\\\\\\/rNbkix2ECT0B2SGRZjudaHXEa8up5FwQG04fjHMEm+inhltyBD\\\\\\\/woc4w4RESv2V0JRHpFvvPeRjkTHSBB\\\\\\\/1YfkdzdwKHsSh4fGTASrko5GJOQ6LMGJOA9nr4Bml0Cgx7FknBK1qfxoNELVxZvQml+rQHpuZmrwMiju4HhGRCtaLLi+9ep8uB9GLH8\\\\\\\/OG5HtCPV9n2NpzC+csi6RQIuHsa8birJahIj7XyiER1SXtEqHcPMoBJdvNDyyGWCUPTchHs\\\\\\\/zLU\\\\\\\/nOfZht4k8mi68HHDnamV\\\\\\\/oVg\\\\\\\/l4n3jBi3wxsamXu9Ub4vdTfOQgbueloLepGJmAkqYjmXX9\\\\\\\/9N4TyekbWaFgFogczc\\\\\\\/CpXX8WkyOUy4BKkC5AAnM9bjyF0vyjPtb3s0C8rIGD3gExxHhkSDCm\\\\\\\/okkY2xeVF7qr74P1wA60kX1iXtkTJQ0wAIiuPgo07ZM08nbNjYvuytSPAiZ8L\\\\\\\/fUi6TALA0CI11i5WsR7UL9G\\\\\\\/LC\\\\\\\/9ztYmg2FkI6EWI5MgzuAiiZbWhemF9V\\\\\\\/hIRO0oiuOlj0hO5wDYLTwYd8826ka9HRtxfh+mQgIBgxDZZLIpkVcZvuYXxR+KK\\\\\\\/ocL3JXSw63r74rLXVqHTuUk\\\\\\\/tcCBKeH76F4tMZBWPk+0g+Xox3n5dEydMYHexsDFQ4WXaRH3q3qqbF+CzcLoZpkPM8xZ9zGLA6h2DrdhCbdwDY55a9EGmWJJSRJ3O3IDohf\\\\\\\/i7Kmvep3T4FX4nuTo7KPnQgABShcV\\\\\\\/rPwE2j6o3b4HORbQCqZP4Ci4RQp+SUDJmL+DfoXGjwDasljt6\\\\\\\/d0N87IMV8PY+FFH2J5XCi+qYngXUioducefWdzK2I6c8rnETG5AvpN0SrADwdha6ugxDToSd\\\\\\\/YOveWSivTVeLTVAkCSOS\\\\\\\/ORCgRI+FLMo2g2CNUQXd2prQABGJGPkZZTptt0MkW4In6Y9+zbIbEH2TjvKTQ6OoMx09cxujzyxCicy8FdPvtfuL1J2irdazDK050arj4u8raUU5L6tACDWZH1u+cAOq+RKg+JTu5aETwebAUx1v\\\\\\\/qTWG5rZ+ZRIr7jnFdVG7StTRu2jCqR67jNmF2PJpF0F9xYAMB1W4FCOSnz09vaXjC+nEZ6w7UaAKW68zmfLwrb8fTVzZEvtIGhhloqWfbUrxTkidBgVPb7\\\\\\\/r5L1eYh2ohovT0nsA6kLg6nC\\\\\\\/MT8IH89C533H6+r2P4IO8YzLOIWup75Zqt0Cg8pMdzjrqbn\\\\\\\/xeJNHc5fLUaU5uwz5ZCsCkix2PdWghBB81O0LA5AmuC\\\\\\\/rgO2\\\\\\\/+bbX0Yt+sHnUSsURT\\\\\\\/E8GU9CH1ITWciKmkbAvWA8woD66T8lWNj6eCzwZ1d5Z4tVdeDPFqge6AEpGeOEMH0nwbtZo5lYgjH5tI1bSjwQeN3qWmW0lk7dQCYZd7VyLv+Eo9\\\\\\\/bWoKhmTBqI4Y0vHzMudA+rOzPj\\\\\\\/lpFMVWDLYA2jX3LPwfHVtp97RhVXplMuUZC\\\\\\\/E\\\\\\\/DRSuyOs2+MXQBcSbR5uSIdZNNtnhQzgOuom2rKGPlJ+GU1KTksTsLoIit6r5Yl8I9UJeH4J3cjv1uhxV9BMmkegdgA8J9gsVOcFK4FZhUxgZQwCVJdXgis3ASWG18nkRIIFbxc8OgHmT91X6vTwBJGDSOR48hYjhK5JNPZwaIqiw1D41w0hCgMxOleS\\\\\\\/FwhVBh8JBSO9WywS6h4VnKIGQz4zG6c\\\\\\\/Ub94pYK1zb5sQnMXlkzcmeZox1q6YfcVoy4iM9vXE29NcmEXN7QUy8ofiW\\\\\\\/UcWfK8UxuJde33VjLbpsELSyMohZ6dKaS9qgfuUhThMe92bKdMdFMsxBuzuBo8VuMy29a5H6t8Sx0AhMwnqn1C7Ubnpy7F\\\\\\\/yTKV9khukdNsdpAuK0Aj6knIuKbue0ZMzyNnmRCEAY0ovgtVqmoaEhm1N9qLSqLtIFqBPOxy4Km5X9vOrtkEwwMeySejV99hAk0PtmTq6CW50ISYq5lvq1\\\\\\\/ypsKhwUk0C6O3T8UZhouisUQljmkCgeFjAEsNuYCiBgFE6ouHTpPRn0KYAdeReho2V\\\\\\\/EA6S42HFhSSoi\\\\\\\/3cTnBO0mgo4UDM5MdKALF5vZbJ8zdmDzPqiIeZKnDbhd\\\\\\\/5B7ZrIwgKfUAFIfb4++SdNpWhLT3YkhtIEutP5M33xHSmlo+1cofHjsLk1H4ynJTkFAjcqlcfmfrrS7Ox6BvLPaIugiz1ssRHmRn9C5U5vte9nwaygJxrujgL5ny2EWEl0bV1BYBnZ5fkKt7HeG91K7ljIbVCgldyWN1n0ccUUwEQvZFGVmbD3M8zVAkoIuzchIRH\\\\\\\/gadxMsAT2FxPNU9kW+MoK87xK4\\\\\\\/fx9oOa0BK6Wu6iHusJxWGqURXWEjLRQdST8IfP3lk+Qlmv54LX3PqQNPmoYr0POaMjLi4wv1VARZUZYFIBk9tevFF+iGQUx+SfX6qmN29KuxHRicA2F5j1EGvJrncRZLtuflR\\\\\\\/2yvoKi2xOzl38E8apKpKvJDKqs9DNeZjXjKfwiujVAxBQ7UgHxtbT7y7FhvXXQ7zrrWIJkgxY7TJPN+CLa\\\\\\\/ntyUDePPtOCnjuAK\\\\\\\/K0YPbZKb59LRJoNz7BU\\\\\\\/TzsocMQPxIQ724pb\\\\\\\/3Ab4WHpMRGpfnzjkIHRi3rp4z3cQxPjCxDTFLv57SjPAU4rfuWlHE60PMlno9l6W8MV0VUnAdJ6zaM+XEgBm2g9gZKm7cD7IQ59iMjMv4dmR+61\\\\\\\/h5umHX2+Vgj3ODgKVHnoSg0uMFs1oINQmxomsd5NlAB0yRZ7Q1MydR3trVRgAnEja\\\\\\\/G6yhVvJuwKFYRla+Ya1RP0QClHJoTa3FHrYE5wGderHRuya\\\\\\\/3MDPqqf1zGE7wWXXxn2yYw\\\\\\\/1MaxYxdW83CYA0ix3t6B1PZOPW0+Wf9Q1JMi+JhlHXm9+y9mcpCIeDpy7d7zuS7fHo6R\\\\\\\/RGQSgdU51nuFttzqNSrTg6E9L\\\\\\\/7Lx6LMIG7wphZC8OXRbYE1eYdDJHGNn5cuk0nONLEDDhS97GmrAfwBoRSMEAAD5pC4++nLhJ5IOn15tC0LOx8v7oADkORHs2AT0yJRRd\\\\\\\/6yNzQszL0kHtwJL\\\\\\\/cQ+x6wNeupqFMXhh0BINHv4VN6rTXCqIY2BcrIGhNz4DYfLels7+DTj\\\\\\\/znf+68QZQoxuBI0X5jn+KtDyOS8kw3gB0pA6QCrjFRvZ3\\\\\\\/b4lsLwXfBqt2CoOdgWU8\\\\\\\/E75dFjtntpaueoGPeMlxT59OPBECdhvSFHTDtmc2crqHLczsOBKb0XBGq8yCM+BC1xbKvX1QJoV1Qdh0eNUNyutYBb1fqhTFGJnMygs5gzNmPAAAHGLMRLx1lLLY+ALKxJhMOImvNM6hH\\\\\\\/Dp7lsIaXrG9+q7KCm\\\\\\\/jxPcQvsBxE7IvxEQTb4zE8Bbefvt1QHhd214+KjIwYwnnODVswbPo5tLEnOiWRnuCfuENn5fMM\\\\\\\/dM562r1evpTJQGowUJp3w++wsaYBRuHP89cuh\\\\\\\/4VQbkbWZUo100PkRgx\\\\\\\/6IjCHUOs69z2xzPdgfIDAH7bSVOBMz9hhTMDGloCFa1naqNlXVrxt68wBWZhQPWZU7ECSqBB2joj9pq9JNSW7MniZTc6QAKY3EsJrqGWPEm4LncMGIyIJvjZbfq6nH3QSyeXl41KG16b9KUd6sdCxCEg8kpMeWHzSaI45Rcu6ed7lMVvn\\\\\\\/SyInHqp\\\\\\\/XHFdIzL1W00\\\\\\\/cxbJB2PjVu7mfgvgRedhKV0ou\\\\\\\/lBNKujOfWJy8EpOJqp1ZgeoEuO1dC9LmuMoKZnkCm0W3hfxCua1TjjNOfiPxdkDP\\\\\\\/AvWUM4pedqT8DDUecO1y+qcCNkfD8Ll\\\\\\\/s9sjbiXGbVdh7iQPh+bmeTMY7oJbbxTCSpsWMEq0lWd+SW6UEZ6Y9AgtVGggHZ7wscniHrVai0d2biNRHws8S2npGOw\\\\\\\/YFtSXF5jKlJqsw67zYJVcipwF090Zt1JYC1Vn\\\\\\\/ol22kTRoqjqAIYmUfy2OcYPH\\\\\\\/deljoDMc7Jd5FyrGTVlpwLbjaDnaSJ47Ru6Nh6Odfg1yvuiTPGzYNRWF2ATdQMF6j8+qN5uLuW\\\\\\\/mbJzcg64nq80T7tUz2RF8d\\\\\\\/lS7VM0iindw+tqwA0iEvSlI3oN8UTj1is8y\\\\\\\/xpcWkSJOmYfV+b07b56K4wXZ9eISaL6r+bUc\\\\\\\/81WfNRpUVs986Bttqdt1vzNfVSd9rv3+9dMYu0Nz3zHdVnOjIz923x9S2BJkOlJRPvwu05z6oLHfxrI3vAcxXpyQIwKngRIhK0jMDKhnPLFfe04YTOlVO4c8jIA3v7sxFy3l99Sx8vhHspRQIqa8Wpjqgx6srBsDn+\\\\\\\/77Dl8aB8F30ZGXp8p9NjXXEeJy1Sv2PVzAcNcRALUCUg\\\\\\\/hrco7vyyxsKewn\\\\\\\/C\\\\\\\/jyo1iaL2VFQU3JY\\\\\\\/6\\\\\\\/I7voLD4kxBRHu06k4qWx4qm8j4hTmDfy2S7N9FN3CdhM8RChKHI5A+MYrhiUCMT1rmlxckkj+rbtFxwItbU49PoQSuhEtOiwRpfSUj+TIuoAzEYIZfLg\\\\\\\/ncDKal+3aMVRAcN2Wl68mZli8HSpY7pCOmDslwPzMJi9APq3qxQI5UZHyby1uV+yzMjBVD5DFe7cwHfyQh3phQk7fgDYg9hActDCXP2PWyPUKMYt4GZ\\\\\\\/iADY4jJk9mum\\\\\\\/tARD+jRjIbQ3JWYYFt3Ze5oXd4rNXRnThhvxCILEwBTIki0MKj7uf2TpBG7ac6S2asOZWRhw6OQUoBoGgBmUaAfesqAxvOcPw\\\\\\\/UwRd4NGo3ljh2BYUmdCNAyi4f0HjFYWpp8M5wvn0fmE7ZI1iaMdWaUm4u1f5xD\\\\\\\/47\\\\\\\/E8oKmG4wfVNCtjIPyu\\\\\\\/4MV357AN7r4zHrIucyDW0mZtZfjwV1CqUxDBqMKU1HTozezv3HYLl1zV9P+hYhtu6UW3rx\\\\\\\/xHqocwuZ7Il1Mc+ONBTxh+2gQP\\\\\\\/xmmO966bZqjLyKYx03wHakaLIdVtJsqZOGI+qfUFEdPpfu48uSU7nsa9Q1jB8otM+drUpstmVhx9x4FUQDbUUJreZxPWW4g6DUswOhSxopGxViA7gD7fzMXw49Jdq0VNKaFRwdH2ACpYDg7zHSsiR2O+6jyANkbL5wvQAMjjubOR5uQfLgzPvsYOQ\\\\\\\/QXsKeet02aoTiIVTNDimqGitxiOMwqndzXsvkrhdmofyLon+GIYs\\\\\\\/dKWPBbz8aZqj9KcIsANP+yVpSYt72DD5KOFR8ne4ZYdXIBgEBisib45KzLjWLHMPMhi6eGCKZH4noLb5CifStVri\\\\\\\/EA3+koGEGGS9KRQ4rnFNyexDDmr5vxbzvLs0UVR4xKDJqQbSzpfiBWDO01BRM\\\\\\\/i2aP+1gIBKPuBB1MnJUBAJlK8HEIazlP5RuR55\\\\\\\/ddrEvFgxdobjOQCdtEfIcaZj\\\\\\\/2mgb9aYM\\\\\\\/Ss6Ox99i\\\\\\\/fMmD31DFfd1xw+iozx1py4D4Z7nNi9SBePpJ2KRQIkwu6OrGRKWfQdTlfuO23yqFq7o7JEx15cde5c2VIFtPVdjRGTkzhDXuh8MUhZy\\\\\\\/yDIMB6LtFD3hbU9MU+xC1\\\\\\\/+Vm9D9B9ZEA3Jg9KjNuNsMdBDLiGlRqQChHZV5JqQ1AJ8+2mkruluslOz\\\\\\\/v7Zxtiq1X2K\\\\\\\/LJhJr4DpvTsajepmlWdhWByixxP9MbScIon0RAMoo01PZk+d2kFKEjQVrMDtswiyWfg9RVv6wUInXW9AbxTv36XnErifcGuRicJSRgqIKL\\\\\\\/0l3BNiuUQaFpVVx\\\\\\\/7wigVERUibDcOxEpMyMLeSvFOXQrDT1oN6FgjEkRcd2sVBRxJJLe3NRPuKxiN+SZ5ftO3wFgikPtZOfXgHO4HbbWwkFHODmEz2o4YH0Ww3E8ZEmtzXGLUWaU91RvF\\\\\\\/Uz0abm4G7qLNdZZf4KHrsS5wVsHWgW3LTm5vmZGgHa71g5lUGojw2zDhS+lTK\\\\\\\/fyOxAP5x8K9b4GgttE4scAgstzvJsqlTGE6A6Y8leo1V2yy2AnTQDJ5ANzsLr8ydIHev9wcKzY4Yqk849lAyWH6IFiOWfWh04lNfsci1qaKLJnn1K9nVwGo+XFmJmuGthM3b2FVTcyJxb+OEk2f\\\\\\\/BdnSBjAUQOURaGOha8V0Y0JO95PfLXy+bpotbu44o1hXmlT0JAQiYpB1Ii8Leq7kXsnQSaT+jefk0wh320ELLEC6nnLwBsl4dTTIpefudJHgfgLLz9t8j+SCGjdNXQ9IyTgvqCuaaxJZ3paB+9qLKTCJoi+3Zqx6vIRJrgz917YleyzjSzaxzDYhzPwm17Rx0LiIWLy\\\\\\\/dJP6LJC2b7msIL7is4VYFJhIDsj2qY1kTupxO8F6oRz5Ml48sz8tZN5DIpCazJaRoZNlQtfyfY97CO+F6s7u7njuW3Va4Lx3ZAkZhCFlc9UoTB2aILaNghGS2Sxf2lN7R3S76\\\\\\\/2NOyBV1EUKTbeTCyyUgTrka4xmKTpqJcpOF9vgW2QttKFKQsTBBUgnNQ7OiHE9KPgqlCsrjNnbxv8Jwv+jLLFiULCEdr3LiE46ViXiGmmQIdLaX9LzzX+HhktJZo7T1S8rqy9OxXaZTwBInZOq36ErT\\\\\\\/yc+s7GmTv1TdvqFoeOAPSW3oaCn56\\\\\\\/\\\\\\\/RYsz5GfxlrzJdi\\\\\\\/Ewd2\\\\\\\/dkSUKGUJE7kovTqlqgsmY5N\\\\\\\/X3zM+0C4jfRmxmbOUp5eBeTj2iL63VBfcLbvlEWxHdDEfmTLDk2ljq6HAPMdAXL8Glokl+RdYZWSvdXkVYdx4ZI9TFQc+0aPCoFsLWcX\\\\\\\/cIyGQnXg02GYjdYqPvzlxrULOV+V0IJLU8ad0BZw+tcOqU3UlHVA5qA6GHxYMyS+tAjhJNY9XaSR+JK+4PHLaDH1agghExacZtpR68JX5L0LB2U4xtAkrSLe9gCfxo\\\\\\\/irCc6RUFEnZixao86kXqq3UMhQmgg5jSzatvY5ZhszVEGwJprYB6sbGYibZWt+WIXMZ43A1sSWNxxCxpYF02eYAXOkDODhXz+CRCxOQJKjjWli3omKr8ilKnUOrsmUNTBti466SrcO4fhCviDxkvW9nnS+dtqp\\\\\\\/tl4wSWzr4IRilLlXRvTORT3e7YGSRrmgiJN9T2mY2XVC5fLOgqyDgjLP21eXz2ZRkU7a6ndhJEXZgdD38U2ibA6ruYuf8ZzsmMUO1cd0ocWW0SXaWROKyU7+SpxtoFJArlSIO429Y\\\\\\\/iBGKV1cVx\\\\\\\/+1\\\\\\\/GHyCLsYYez3U8QJfIzhfCUb18yjib0VRDvHScaSPKG2uw60W+00Dw58ofDVJaA+Ja6BaHbL8fN7bpGaIIFLicm0anxh6uIJuycxSkauOjlivuApQOtnA+oBpWvsSg8QYymc4h\\\\\\\/FyNnyZ1vOYlsWP3dtTED+xDIPdvjNrh46jg0iwzO7Nwke7CTVoaslafZH4f987aOXsZtCG7NhcVJqODpM2IS2B2uxEYPQxEAmEY83zyD3Wbgo0\\\\\\\/VWlArkT+rXX8qfLPOm9OHurak+uVMZrnhIuhause11MJb9h2C7HxDJoS3eGrGZvYZHTKkQnDQvebgSJyO9+mYHWQTDBaXZ0633zmLFcN1mlqIrYakTjHlM7Q\\\\\\\/ODUUevcJe3vPHjrysOqWyANuxPeIPrZu6CqgqPcQM25AxhSrbUOl2bc6TxijLxGkPVyEtoWARQkDP2FJ5U2trkKuYYHDJse6kwGH8vQnP1MJhCvKA+ICKiOBilyGZNDbcA30FvoRZkLsJdzW2QfmNMLmzdp\\\\\\\/VRuJhgfpWRNwH0OgHQzg6FSRWS6Bm3cxab6DccBs00sF+ofPIBRURIvM2SvAQpVkqVBrw+VivIrcrrOHmnvfnEzpFqW8ngDuhhjeOtn2Hzyk5jN3w8\\\\\\\/Bl0iE8cet6PjxWUB8Sn5lwMUSS\\\\\\\/JZIsneJmfG9ldwNOE93GKPefo3MLvdlVmJhswxWVomF1pyAZT7k1iUz03Ybsj\\\\\\\/y7H8Wjd5\\\\\\\/1Wf5Z2WUTCCtVFggZSFW\\\\\\\/5c9ornmA\\\\\\\/ecBos3ogePUw5zhdQsVm\\\\\\\/C+OYJo4nAM4YUyAcy0i4akUovh+F3d0sve7UezyQhRF4vzooYa3E1doAc7C\\\\\\\/QRD7CtSg5mfiwkcBcWml87CeTksQd9w9EhVqUgG4iFJAIsXXw5xqy8z3OBz3eJapKMIDpYuhPHQG1\\\\\\\/sKgu6kjXlx+sSox0Ln7Cfevxfnrb4D36WJwQO6VfXjlFLfl5pVis9rgP\\\\\\\/EKOa+THy1P+XYpjztbMEXrYgJTVFvzsLYzqNEo7LlT6qo0AxaWE5SUWo+4YINuSfbXFWl\\\\\\\/g+OdG18mOTmzo7rHwtw2uRbadCYkEJLa6Wy4jKjYptmPoruCFyIl3FeAfs\\\\\\\/KU0c2cOCUYBYhmUj0aEoIEcB2gBdMse+V9HxwzIAHyoGAd6I9PUqt4QWdLh+D7t8OUwV8jMJoA9nBVKPey8KoFLuRmd4gNGkTo9m0pgfMM6\\\\\\\/CwQOsTLmqb74DWLM+3TP9ZMJvqv78J+\\\\\\\/YYJhMSEHtDUvGxSR6lrd6k7v2QEDmapWn21imq5FVRMDjikbroRZsy2\\\\\\\/edNTlieHvRYcjALkV\\\\\\\/cJSo++fC2Uy8eYT3j9u7a13ActUXI+sZmJoX2tbV9FnzmkZNx7tXZZA\\\\\\\/sQXJ\\\\\\\/80qqCHALunfgxX8dFquNV\\\\\\\/\\\\\\\/6A0wYwYl4rdJmUHEkl+wtxtNp1iBeqrOpHkhm+rDipUhuS438OhYbXTSFd5JQt6GhBgC4Tdx3i7Rf3lLczR2bQD1EbyEWWx6jUGBxKZ1N1cFCtHbOK5VPI9LHEmKerwli5uRTFBzzOxTharNRB38xjokPrSPAqmrcJFWetvCpk7E2SB8pR705xGIJbOldme0jHIcpSjJrO7fgisVznxPy\\\\\\\/TM+gxx2ybKyUPxbtI2wj3A8VMLPze8JhdUQT3Yl1waMTXWpLUuzmTCiWq\\\\\\\/YSTYnthmq75Bp3cWX66z8DNbGFoczErp0M6IlFHGP2YVLxXdMrTk+Kci2ysU9L2gRYknJ0JlFQT2cHoD74JxZV24siRFVlAhS14M3UYYoRzTyb2YmM01Afo21dm\\\\\\\/SBzltbKWFzeccGytzb4uoFJoydE8QO9LzELZtayuVIp840j7pERwia+Q8xDgFOs3HmzSym0zd5Y7JCHAG7h3z4Tu8aSc71GUcGOhVTwrBPAioBW1bIiDBhErkbYp0WDn1BLabX+eHjFrCtgpcTbY+nQeXtaFGh2UjiVO4tg1WXjoxxdQy1LKE0Vilh4\\\\\\\/cVetmKINNP8iIIdn+WOdGKE5ttIaZd9yC3Oarl5oaHLlIzQxgRIvMgeIEPtbri0bSCderbVwk0+ZGvZxhKDzzzNXM8je9oDE9zUCq9dMwfZIS2UGZHHZHuNYGBPPGzSGOkGd+KPnsf3bCwqLa2NGN9iXzl5kme+UM+AuLwm3T\\\\\\\/5cBFKgZYz8eH\\\\\\\/AzKb3PnZJGQgjZ6CUcPZqNo3ZopN9lUBD4AWfRxYNLmngEC3E0Ee4h5mixXAPCohZmjX9NOh\\\\\\\/qRPCzfUU4P9AY4POQGSROg4mVEBnb\\\\\\\/t02ek\\\\\\\/uK7xm7xW+ESvo8SBQ1OE3Rr\\\\\\\/aFYL8vtAMVNE\\\\\\\/8LiR4ivi1YVgKTXWFPXRvDe4dtRpY3XpfkI832bKkgquWIy\\\\\\\/md+8YxnrvX\\\\\\\/\\\\\\\/Rjg3QQoQ1vbAjamVI9bMcQ9ESb84z\\\\\\\/fAga\\\\\\\/K0aLZVoT2r3xUnIf4dlHN2z38+Jvh52n\\\\\\\/QhNpvk5HTznl6jCAC7CDNg7aPITHs9lHsX2Ck0yPL++1ivMrmXsiDUx7I+Shw0muc9zJ11kppfgdYp5sdbKfUrbFpEaTNqMbjlz\\\\\\\/w3AS43bHJRQsYskkpZedGBb53hloHc2umiy+yRQcNBO\\\\\\\/g0S2fQHlxR4YMBL5f9LlongrDC7wz\\\\\\\/5Gm8EIYeM3UAl08cIV9hXAM86g6ZnE9WTxB0wfpxwEGEdzt0XQbgJJ0s5OlGoamAqV2\\\\\\\/cBTzKwijSpWe9F1Rotq2udn1zJycmBBlj+3G1DI0KIX1+g8WcIKQ7AQ5VPxcfaom6OXVUXLmuAcourqRpWw37fg47pzMUs2Es4DAMy1Y50pvtmcdFxgGvVZwJrLnO58xLyNifALrKKZu9jyc07K1ID+9pHx6\\\\\\\/7grBAzuEBwxMeHjh0glYjNM+Ymu8KanEv9WlPlrsEYn\\\\\\\/7nEh4HakTadR2Uf59l8d1MXdAcEEecWxdvwRA7xXxkxmn397RbSMA+swwbrOYSLF81PvzZmx5XUJsYkycwgpRcgV5dRrpyqyk\\\\\\\/DVpMeDQI7\\\\\\\/Lp4fNS8o+FlrQj8Acv2hg4YcOaSXCrvn4MvKTLFoBv5AnN4s1Yg77GMAVSCssOzFj18Qx2La\\\\\\\/qCUgcEQevnWAoGvlqnQ0uC4lDmWibmJF6jaDiM1m\\\\\\\/npt7N3b2c2V8zvzi5DYbjlNhRQrK\\\\\\\/BfYSjZLor8otMWFWCbiq5yr+6\\\\\\\/rq2qWoNGqg\\\\\\\/xcuU6Zo8RKzToCK\\\\\\\/f2dd0CbzeVotQYGLHGSXiqVlkKPeBY9xoZsti0XSUu15WqPBKJrmSs0BHRg7EWcaO925Nlrbcr\\\\\\\/I7Tz\\\\\\\/X5dcwDrwbt1yt5X10nZEWMP8LlsQWl7P6IFqcB0lnB93eI3qCRZ6BsOaX78w850hslCA8GKPKr6bKgVGFcUAE+MbwEUh3vwFGtcKjfKs99qQE7eCyQsa7utq\\\\\\\/BrTb4qxQDAXGmSFjNHRnj+oNtGtb0IMJhQ63Sw0f1yz\\\\\\\/7Ook4ENk3KryU1rPSQ2yco9SBsHyhKx3gUts5ltLmIF4QkvoJR\\\\\\\/3iq55GGh0Nb3Fk4mUOtugMuKcShkFyImmCI0po1EOOTM55yH146VDrCm82\\\\\\\/0nLRRB1th5Dys\\\\\\\/HbGBipwDY2CZlpegImLkeQPnLYhWdhGpE1+UuNOA4g+1pEoUKX76YK\\\\\\\/Sbmr1WnDJhGkqr5EAZhDudW9y\\\\\\\/TLBhyp62kqXwRtt3RGJNfybN1oTxabZ5Aczn8n+2EOGKL\\\\\\\/rE8Jijd1JzhjXDG6uVkQpNLfTTWjrDkcmp+fffBCd+DPHDOKfntt0cp6ekRKF8E7QovxIafNwuczJ1M3Tnq\\\\\\\/MDpB\\\\\\\/DfCJFqyPwPDv8TXqRHyUeO9LSZ5rUU3S6de+2T0KnwFI54kUKN931nb+br26VYf38VUUz8tqv9orUxfVcg9jcaqiYiDcY5KjP\\\\\\\/m2l8vwrhivXkftCQlyYeJb77j6W+p2v9qGnVcCxlre\\\\\\\/upq\\\\\\\/YsO9gzjlzDVuUdxxQDWNwooYeYb2Td5RSU4Ji2EUm0m67ZtRSGAEBO0ZiEDwzU0uxIqDaBsR+6VyiNn4Zwpy9QVuYWtB+A+vZrusZy9qYbtUPEFyaMCzC6yTnl2Q6VnUY\\\\\\\/gcjXmqSNNREINjOTNzZe3d9wo\\\\\\\/8O\\\\\\\/ag+yWNGy\\\\\\\/h1fJhUDbt1B2G0phJXAEMt5JsBDu\\\\\\\/6BnFw\\\\\\\/4xCmIezSG4nGtPGQ9FLugbaiwbywpnMhMEqWJhika9LqN2OB64UOwPyrPXR7CshPmBNFIsUmBwpIISqfEmpPgHUJCXpHAhTtJhMSMmAojqtpM6HoSmkLURa+NrrYUMEgGX7dErN54a3A7Xjrmbx10+IZtgA5m9GhYUcwqTt1v80H73rxEJkUZy5XkkdlxkQXbScBpd5E3+3bIhbkGvMlYART2pgNfK26DwuJcxeiTPT04hoGn+HPOuiF9TgAcjGDAxedak\\\\\\\/aG\\\\\\\/Zyv7pjE1PVl9tQ45yegGXj6mzJ\\\\\\\/olgHXpzKhnKQK7Gw84jufrKiPB1u9oN8Mpu\\\\\\\/1wubnk\\\\\\\/5B6MWFsz8oKzjonVVGXabtxzVM\\\\\\\/lpHNUe1gqQt2S1TO9guqbjhfajgI6P3hTlnEuwezEtk3K8zdxNSSv9Bn9tA4Um\\\\\\\/gksCZ+OnCO0l5tHZ4q1iw6box+cS0yHAJVfomM5jaPwa+a4KqoVMyvpLhWQGYNYcvY1H6Sr2chazJpvY7gapl+7mrsMpEKx\\\\\\\/PUr13MAnHSgNQmTn33Qxc67p1OnV2f93rkujA00L0NC5kjHq7zhR+sh1SLcj80+WXct4q1UJGCfBySP\\\\\\\/NZv+6Tipzn6Ulav9R2uSlhsLyvWbfeZ7uEf1sVkuvrio6hTPNlZXPbe76nmbQVGF4USJsSCmcN0bQ2bHAj+43nRg74rknsmOVG+Ms88UZoCzbQEKdXgK3CwMaAnKHjRtyz7qzMGSb3hejWUKcOG6bW0QChqnpY93p728P\\\\\\\/BY4zoiiJ43rgYnOxKngcRidAIDcE0kKlmtG6PydmtePkv2ou4cZb0UJ59XLO9kx2WKM87JK2B4D44aolWdkdqIuUch1GuEkcT+HRKzj8JfhUXfzLNMk42\\\\\\\/BqymKarUhixFQCqTepgoRywvRhAYVb+X5W6xKk+ohuCPNWiVsQ6p2QoRoI88EAnR\\\\\\\/KBH52DhPBpoom1+XFTa8eLJZTGIf2DoYeO8OBiGKyg+j0pEMqPXOMJs0qF629BjzYvncBjMOYwsdU31kB5bh6PvrCEkTlQJ1KEzsN9iVFP5r9iNFBVg5o9F3dN3Bgt8XJhuXg1rTuA8yxbA0nZp9\\\\\\\/DWygT2IOcGL6p2DhJUlvPR\\\\\\\/2aFxPznX1zaF8sBywEcZcFjJ0Vfbzrl9rlFOJmAInVZFNtMwqHOyo8fRaSHCunAqLgz0KvcvJWID82s3SIksz4QLYHsmZEW3yVMXOV8Gofnni5xMWvUXo8g3u4N6mnWvQHaA4P9epFX2gEdym+2t7p0DeZiyqs9RFFcenxzerkpc9PtgAlJ9AHhXKdHr0KG5ji614kkZtu0nzzazzebYXui577DeKoc7yftuU+4Pvf5dEJZU6ukJonZai6aX+hJ8XN0qI\\\\\\\/2LpsN85VwyxT\\\\\\\/4tB2Z14RqfdH0iC7ipcLpApvvYaz\\\\\\\/HVSL85ev8bgac7GVhBSMydHsV0L\\\\\\\/VEuE\\\\\\\/NcpIcDP1W\\\\\\\/MSFU3pIwo\\\\\\\/bYLnQzkiEVHHG79YMbCVjNQzDAZ534\\\\\\\/hV+qOuTaVSw1Ck6NAut9o64iPFlF79ylxRQcD1mlnBA1N6CNkQzzTV2hm6kkWLTaj\\\\\\\/A+9jje6ZCSBdD\\\\\\\/AyNJiGowaDdZU2TZW2ZspJDivBE1eBngBVrQ90VSxpggYgNhk5st9kzMb1OzIlxDiVNJU\\\\\\\/EPBWk8+ROQDZwzsIV\\\\\\\/tdce2V9nDOeHBZKcg7yA5jw5HImvjY0zoNd70Vz0X+O2XcRVwDLmev8lq7+fnw9ZMb5qEzsFY3fR1y4zOQZaw20RDp4XVmnHmSp06VJ9XWHW+ab80cr4ZUpieNZ4St7zz9q+qR1Di7HBX3PR\\\\\\\/hTphO5D9MiJL1TtcpdweYKRg1LGRiIaV8BAYZvhh89HhNP2bBInpSFiyfD6Ubr\\\\\\\/F8fRoWVw0yrXYn\\\\\\\/feX15KnEIUOobrKI9C0AoG6XzoGgGE3tlVWna8\\\\\\\/SLQ6IeGDWsQ9p4Pz\\\\\\\/+QXC2fJFXtVdX\\\\\\\/\\\\\\\/x50LUankl2AOetsgkQCA6E2Hf2f1NJceUXt4Vg7S1PO1FZd\\\\\\\/JWe4mMHN11uau6wNXVIrDm0CfM7N\\\\\\\/GOLyzJa0b+\\\\\\\/VrwTgHQ8bllAYj\\\\\\\/woimZqwnyOUxFR+\\\\\\\/F12GjplRIBfcJRXdELxGCXtzmh26VrfP6KNgUuHIN8Iy7G0j59cQVPb\\\\\\\/XXo5nKKiSXV4GLM1QCmX\\\\\\\/MQaIjf+WA2yq8f4b0BlBNgQplphF1kVh0FBjjFMqyiyu0hwO+9ohg7WQlTUFN1iUXfCKDSZhuLY4KqEg6MH6iLFfaW0l0XBTnJ2A8CqChnHNkfbetMnsFWT\\\\\\\/hCaTTR+A23bFCac1qtN6eUiuFxy5\\\\\\\/XQjyQ1rYnm3lCQOIGHPQuxDrHGHtoX9eYHO9X2qPpOPoTvzJmYYqGfs+Vn2wYmWjC9Xg106nN85zAsih8JnNZmIZ8toa\\\\\\\/u29jHF25zRdMtUpFJ95XdymBED2Kpxy3zpQEuSTxYH07T5JeOfJF+vgaP11EntaDLKOGVZoEICyg64hOyhzdy8vKEfMeZgyZFOZdu7v4\\\\\\\/\\\\\\\/tOGAMFU4xl1riBRsJQuYabeDmiJERT0ZpB7heyQaVFNIonkKlpUiJeHPeKGk\\\\\\\/w8\\\\\\\/NRdsVGPIlNfZp1JAJYM9WsIWUDA8WHAGEnhgntA3Aa59t7GJGmaUbVwO566WZ40zE+LSaSfkRd\\\\\\\/AjVS8i4L8O1h\\\\\\\/oH8EWfrJGIDhESzHVbvngXm\\\\\\\/v\\\\\\\/SX2F\\\\\\\/XaAL9\\\\\\\/t+wTLkXXEbk3jcQn5yPu0XIePnT3pH7m1QoJcKPJ74bisCaCtBhkWf+P33X3hGJ3MvQ63ebN2j7yAZ9ls76710He0Xy\\\\\\\/Zzy7S5S2kUjwN7FVQs+iNDluSJfIjo7O4hMiC3H+OM6LAbHB\\\\\\\/MpgitwswrDV+5Q5FfOiY1flpxzDOCgaQYITb4K2gtrAznxzABKrDrM5hVgY1q0ERwFBXlbD+Yf9To\\\\\\\/X3vM3KG++668YSmkH+2HvfQ5wqYhTCk\\\\\\\/t5oN+OxjL70KSdmdSOK2tU2LxLthMSzVvsrKkAyWL8nfFhA2m7DlgIcxCxlRxahA8ABV5cQ4eFXW9Z+GtpsXmS84EFOYGCrmGV\\\\\\\/CGJFaTlxUA1BadymUoBV\\\\\\\/1iW1ZwBo0pTADSIi9sEj2TvsovAUSWBk\\\\\\\/vmPE1OJu7FlpY8lyMM4d6\\\\\\\/dZQjka7Fba3LpjPn6ZVjFgBEx2rCxORIc851kvQnyyIJGUMb4BH6uVJtUOEGGAQLGRHnC0mTDXBUSGg9asyprn5HwqzMi5mWfMXhuFGxKJJ7Txx6TffRbvnJhb\\\\\\\/8Ku88JBslHCaehQheyXJo2MgusJqlKMDF4bSZaci7Dt7ZesqYiXxnKSSYrlDPg9bcTLUYV8LemWpXc\\\\\\\/Au9OWaOU6B5bo7QRMMTqvK93IcK9gjQwgT9o3OHv0fU1wo1TeLcAqKHrTqopxOr+dhoDGGC\\\\\\\/ZGvmJZZO\\\\\\\/bMVEnFLquyQoaf3+so2Fps4EMWlQ2G\\\\\\\/G1CHrA7iXLP3AKUo49nT\\\\\\\/\\\\\\\/mB3si0rNUwTRBewNCPFHGGoBRhtkuKYFZ5x89IZu1wmtF2sx8pdPoSAGcj+vuGQMcHedS50FwTFP2EorzTtMBnS+GE8uwL0k25Wa39JrcjHFNfAR2ajMH+AO1NYw80CmJHfmOXmv\\\\\\\/xRbJqg+JzhqaRvMwAxhMzvvJ2RqnVMi0SOESlwK\\\\\\\/bUO+8RTvJFJZV8vzRJ3ckQAigYY6ohU\\\\\\\/KZgMRKEinf0ZKw2BFlGW2dW0rFmNaKr\\\\\\\/xYIJXflLPjn6xkStbzs\\\\\\\/a74m0fZvmOlZSHfoT7hXuo2\\\\\\\/eiWhjhsyqa4\\\\\\\/eBpw7Fvi5CidEteCZEIlCDdGm13Hz\\\\\\\/zsv7G4+HK0bmCHKBcgyk6WXsb1jGc4Lx\\\\\\\/5+MWzwp7CLvScBkzmNlITJM1vQa4i02ZTIDHTKKKO01FdRsHr9pWxCJt7PC6VpjNrq13etSiZnwLBoUYHtw6JXJmX9uoQ2KVZXbjwThntbf+L4N41FKrrUMdJxSqyT2eXhkVXjPQegYH8uQ0Y9dVS5phfLeg9P\\\\\\\/EnYT+s2BZUgl45M9X9MCFmW5VqDsV891LZRTzrIBKb3BF58hkxdsh3\\\\\\\/3Bxfu5KUCc8rn7\\\\\\\/BjQbMJfOU97xtbrya0BZmw7tnXl\\\\\\\/ECdaDzGwklIan7kG6SzXxXLfbcH69qxjYQ8rrERhtdvzmrJngvuVhyhpcBvaHAQYQtgWc5HWYmf7lEPvCNlBp07nmXh729tj+jtRbh86OGpx1MIXcKz1AP3RJChvVY3YtwCGCLNrNV3VMz4B0MVXs7mvv8WLfSSOknAW6hIIYZjcwlxD0sjxi9p71rRtHeuBwCE\\\\\\\/e4xD+24wSZPApjUgqEBFkcfpDnu3\\\\\\\/353OqJFBaITAPw1hPmDC4EnaOra0FbQ2ZCqk2YM0hmGArUTD9TLY1AjSvBwMkO\\\\\\\/i9SyhDOs6lTRJB54OIHAjeREYZWmimh7hQYlLyKkNqsJwG0wWTeQD9gsblVeqaiVhzo2+kc0\\\\\\\/FUmvEwtRWpChsxy7h1qs0RwtdrX7p10iCNp1XpFSXY\\\\\\\/cHqQJEUh2QAuzUotlymnIqycjpogGCO6CrjZeCmCZcANCok6yHg08RvBI2XWmym4pS3bWDd3AW550V5uzUo0nPwVmSNZXCW+iH6y641yMmHmywMSsVd+o\\\\\\\/DFiWP7ziU7BQ6P+DuS2c3ytCoudJPQfyRfN5kFl+3AK3JJrHkw1V5C0HHHVgFX\\\\\\\/nzScLYZFIJ9YdL\\\\\\\/ADhwIYa0TbPeLCj\\\\\\\/Wv0o+6Uacy8ecqo+bSV7Nt5gsmSx+62H9DqK7V4jRKdgtkoZlbrWOp0MhiI2zFZBoHh+eYH2HEXKMU1nLoKDxDKo2UbijqfJYMmPmtgVWS9KmaMrKYOe8+ZKoXveYmIBgwC27TOMbWNKPXNNLOYOPcxt53f1apG2emlKPief6rIpPszU461N6HF\\\\\\\/2A\\\\\\\/cCp6\\\\\\\/jS1LA7Apb7eiqxSEC8AlCJK2AxUugfBCUDLQe8dqhKY+t5tiAGN3xua+riw+fIUbp5sDM86Jo7XymD3CbZ\\\\\\\/bZ1FjRUWIxuTxoHzZ2yqtrw3L1\\\\\\\/y2BWm+FwYmtL7cod+1KkQBQx5Ba53RyRDW4UnzT0g1BgcgTZhuOdewTEFuPfgsDBjgNz1ejJLroKdJWFVYpKqHix4jSalJWtKGC+Qe+N4RsJr1aHj3ekgbULWmUiqJFEM2Xa3Y\\\\\\\/MAIX1uCmKCryfphhdxBkmHQoZSqV5tmRNTZmc6AVixs6hMRXdtAfqopiNj7IjOHQBNSkzMn+zV6ctCvSIAR7GnG8yY7iVXN8nhUpcIqZSjwS8j3skLMPlrdQ7Fgm7Xct7HslUg5EeLax2Pw9nINive7erkX0fHf37Dq+pufY5bkYJFxOEcv1vhfmM5vCltkaBHWpBOjXjxNetY4KUbRhaoL6HRuthl2d+VSYNcQ8Zh3DI9iJuX5nIjm\\\\\\\/5SH5yQD21GjgeC5\\\\\\\/DUsn8+dJMkSnR4ru57Sq+RcMtwBODSEZrRhR5tGxV8P3zkNpRKkeEJpYShc3s7ozu\\\\\\\/bmsDO6XnaHCfbpzRPsoenhNp8NARzgQuwyxym\\\\\\\/auvvAuCreeUHCWGrX0JLR+HcfvW+4kK0abD0uJtZJ6w4NT\\\\\\\/FePyl2l07LnWJpXika6HlyhVMe2L6vrZeXN74ENZrPo+4TyU27v\\\\\\\/xG0FdbhuvYkuej\\\\\\\/BTzMd9g4A3qnjE9SHpxzWgC\\\\\\\/daQUYaWaicX5Yynl0+8tSk7ZSjqpmmRV6ZrmgnPej41QZpT7a2+x5M5TttUXEgWydExcBb+Iy81VvVU8e6176VxVzyXAOORqP7tEyEILZ+pVM5vJFYghR+O6FZkWugDehHGYvMahkX6LretvfpdeclseJEi77pBeGFJKajNT38QiKYBaeEoa89NRcScnIgU4KbVOSKV7MrSrx8OGyIHS1ZkCC+n8v\\\\\\\/BPB\\\\\\\/9+mjhtcs0tMZaE3m2KzjeKM+cZl1gQALnZhmerLXDLNlkLyIkq3uQ8PgwEtfXRhqwqMRqYxw7iAPePk6Mu2tOsKHLSzns8JEuYuayh1KBrh7n0uzVwCVX4YRc4rUC\\\\\\\/fIQLfJ5GIRbKLmFJG1TfnnPbheV6ZNyDHATu7yHMJPPDUvyaI\\\\\\\/rHbcZVtk7lYglgDpBytkdIhR0pwITRvZlpwunQguab7uR8nftB8ZBhGlLRbQv2H7tgbilC5ATDaYRLuinsey62LLTAlwI677BlbwjYFjf58gEdcbiKvTSEdkT0YPXG8EBNCzJrr1dr7O4bA3NYjhkD8REvEg387yFV0oxZmr8OGcxCsgZzz9Alxe968diX7NZ2vuDy3IdSHLpNiFSH0UCOV45fR7JTRxE65L7sXSUJmaKhuRdh3mOSzuFwC3jhB5zkQeQymLSDZA65URERdqQIE09VhL9m38v3jiRxj52mQp1LdbhaU4g6XklwgIgreL+E7DqtEvKc2++IGm0CwOZUhXvCjdGPm07csUe2kiEYI5CHWRQAYH3lRJVcQa1GjrykopboSLdk54jqgWSR3SCe5sumrUejGEwoheRFwrRsHfulmi6zHCnEeNKjBTGo1izl+YdHbe\\\\\\\/rrcCEsOmjURw4eZ5h\\\\\\\/G\\\\\\\/W7fK54xwQDlHfS590zOai0kNQD1+9\\\\\\\/KIOT7rPUhQ2BCTZNf22Qu\\\\\\\/A6zPMBG0xrrs8QKW6AN9HlX\\\\\\\/NFYRLXlSbJk9xz2xZbMmg352xaIv8FWUxCRQB9JMZ+2jddogmXHB3scLiPkotZgCsAmHTvxI0YoDo9+TlL49EKBLZfqO4IKDqPvcbg7rLi2VpuF8j\\\\\\\/k12jo\\\\\\\/8\\\\\\\/gMcLXJ0Lolnx69KqZAVzqKi4NlB8HJiRjCxV8FOgcE81ulvlwz8PzAHS8EyfuWH\\\\\\\/yXwr6O4PYFSLk9S3bmhqUNE175+l+kDmOiy7EnzRlxbQXJQJ2L2yShewQNRUpHsPWbQwtbLZtRv4fFYsebBTYC\\\\\\\/PlLn6b1Tddwoe+nNJR2i2BHWl4coDuskcPvKP7IuWKQYsg1QHmfNAvja2s6yBqDvvqcg0U\\\\\\\/8lfRIi70IZGWSh5+a7YmgfoTbfK9EqvNDqvO4qGpGirFG4xN0Ym7F0cUsxsfhMao\\\\\\\/I5z2sHpuxEV7HOh9PT2JeBK8Bssc\\\\\\\/vke103vCrha4wkPJF3D2IU11EAu4zjo1kbpRLiRuL0O5TpoeehKLLLrHJm4g5bTBHsOBeYUqY8EuKKg5enuYjv5xXsqQ8AJNg+U0+zRlq\\\\\\\/rQJAUplOMA3SZ1N8MeoObBixf8DrZYHGVwJwXNur5l+GSAr2LkLXUmdP6ZzFwhRI0VIu5Xe3WIZsJTc2I0E\\\\\\\/Dn4XtqU3PoVC8fDmcA4oCKrBduwe8dCPFT2Eb9jEKvPPcwihTPAhfParJ93krnSRBpA9KPuS114wjb6w+8RbGBIkF532s4dUi9KEGH0Iq0ynAIrkMcVv3k18zOjI6QlGkYbD2ie21v6kGA7pMOg3K3iiPnnCU83rziJ3hF7msJsNvUToNprxD2Yk1RNs1z3KAE2qwUr5ONjo\\\\\\\/PMW7dUbnoMtz6gI5W7I3CL0OQ648nb8N4l\\\\\\\/cnI+MctI4arDjFFAUaGykviptgRTa5siA6yhymMjOZ07CTICuzYHve1HtNfTqpL6P7589wMM8X7C6DX8vy240HfYnrmR5sfR1Xk8ME1jxaviurID2qa66\\\\\\\/K6FYR3vV5FDr9ALV6P3UObS8W6IkpYzBnncWbldHCj7UJxbJMM6FsVe6fnWurk5k5kcSuQ1QmbH+wILr64BwrLV\\\\\\\/F1hhQYHRTBtMVXVOvZTawjGjWLdzWjWGngKz7uxGGfGsiVc+3Uxqpc6vdEUGAZ8XLYD3VQVmA7RoxMXDdtD18VcYv4icl3vnteyeuLh9Wl7IfmTNQkCEFQIHPRl6aTKba8pdIFHhnLlwCJj4mKsB9z6V4BuSkLu5bo9p\\\\\\\/CrnFcK\\\\\\\/8wF9uO1mmEazqaBhbmoGzfzXmbNABuNe9pNbfFTlzQi1fPmAMXHfNTMnC\\\\\\\/m4u2GCzjyZichmYgj30\\\\\\\/dw3VSSw+BmzW3KC2XmC8swnm8jExWppu4WNNZBqbGH3Xt15A654JO+uO8P783J4pzAcY2mHRuTjfoAVyjnfvq4CADcNFiHspHaE7LL0E3r\\\\\\\/s5GzBvAcxi3CPul3fjNZsgcRlFUGZNtRe1XmS04Nw8U+WRfQMucUp8uYgV\\\\\\\/FpPsd\\\\\\\/FKsQfOAxEe4pT+vU9shb2KR442l0MmvnHJkVP9BHw4nsRctI6Rn\\\\\\\/K3eIOr1dDOu8\\\\\\\/aX7Q97SLd4jaueP9kaXN6u0U4py4fWgvbAIVfQZ6eSAV\\\\\\\/LvxNBc8VcnPI0d8exi8fQf1vQ5TwQ\\\\\\\/xeMXudlhLVZlREXbHlPDGL61Ne15fkn5s1OqIdHn1\\\\\\\/1MxRvH9gotHa9p3Rw9Yg\\\\\\\/VLG4BpHc2K2r6tbNOGOPBjNb3PcZjXVObpaEb88VwXOxknJconM6961s0O149Im8ZhDisnbSM5XdqJWZR9yjbTulT5yz+E\\\\\\\/N8w3\\\\\\\/McqC97h2hJFRatMaPwSO7FxpsX\\\\\\\/vDBUej0QcGGML8khiB8hRHQBpf0z3m4Pr7DxEA0UUu0RrB\\\\\\\/tPaldVAEdZQnmVwiUIkiGUS3d3UdNOsBaE7qwigxIwADMBm3eGkHZHjW6VQsDN\\\\\\\/EBIaG7vD4WSuNQrKnzAAOhKO4WGmW\\\\\\\/bjiYGFXN2+En9dWl3PHFMlPlWF+eBQZkUdQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bC2qfBdwL\\\\\\\/UhrOIx\\\\\\\/Hq4lg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fab61adde9b55c25bdc8a785e55ff4ff9e0da5266458f111253b3b89287b4f82\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ed7lSQKhw9+YErcEXUDWiSv69VGKjFRFupaVM6mR1us9qMK7mPzgudm+Hgy\\\\\\\/aFAJwrGSqz1dH6Fbd3nofiuU6LhbAHfnRRNaV9rdeg+qfM++nwxkHrnJjDCQYd9y\\\\\\\/ACju21BhdcwBPjRC58CfUlRUyuHRTibVdJSpxX1E5j3uvNxyaynaqlAVsqtgpoPkty1zVLTg95Lu+Kn70X\\\\\\\/5kAgy0+mMLV7HbwYKzas8RnBb2QJHCGnD4D0j8C2FHL4kkJjyo+v672YEnE15+VZijcApK4jYWPybFdqScHHin95IyygVXSyMyZRAOdJ7OULNJ5yK4UyWHDGlSn2KRr6oK6evnKPNO6eiJCFu4qiMtXQeiyoEkWRYwtJM\\\\\\\/JboMozKFZMPa\\\\\\\/Mf\\\\\\\/WnF5c07pVeAih3VhbuHk7DfofnZrgs4BsIDcyUicUo\\\\\\\/LJuUSka9CjxsAXU2alzRd9EVfAQCqFBV4sDWhcds\\\\\\\/RDG8D8B+e6F\\\\\\\/MNO+DWRE908JNS2u1cI0v4M4DXO0VgHtjYc08YhrwnROi6+pmy85k1rM\\\\\\\/8igXWtwXk5z3S6MB2Ru54pflzOHOsbDCj7Jp8uw\\\\\\\/50wKXZL5D9DIcUnrfb3pqk8eGAzXAmMQINLKFigRkFbvUC\\\\\\\/MHtQOnVIDJ251SQYsBKNTqWWAEvuMSNBuVUJLic0T6rCN77Khzccs4\\\\\\\/Urd8PCVZOwkyJbP0VSWcH5GXGz+pIob4D6ZbhVTx+huZiQF8S\\\\\\\/4rizT1RuePR5LtDSikVlS1oei7i9NYSz3XctBMMyaiHfLvk5CjjUgEvUoT0Q1OKO+rCURkhrOvnQ08C9bkWXaezSpguoZtEOKvlbLqyVbfQ+iVOnIqxlb14YcWyjoeTengpvNr0gdvd+yz+8NlzhSVZQt6QHn\\\\\\\/g8onRhO1ytpj8vRpjdZI4nlaS7SzqXoPpfoma1lvXtX2OrfxymiDkab8dHGfPqLx7rt7RNQK8uKkEd1Kb2QLbaDXkauuftejXhY\\\\\\\/quDQMzk\\\\\\\/linekVPtGjQUKzDf3bcMtqGLq1tv+lkwrOQsesRy2JpqjptbCO2l8KESFP90DVqjmGsIC3k3bg2Pdv69YrQQBp0QOUUQRP1ZC45IhHcK64rGvp0I0KJkp7O5SNs4CkOLyO\\\\\\\/ocY8wrN3OmAtUiYD7876W1I8+NATPNPOwl3D6HqMFLeXT9+Qv54xecD8wSydi5hidMZ93XFYQfnAJNzGUfr6DG8U+KHIXLKsaciy\\\\\\\/3kfpU\\\\\\\/9Y9Kzyc\\\\\\\/2JyIXcwN+W+QGDbvZWcInuFHTHTn4XXACq3tG1aaW5AsvlddANjsdK5MH7UvXtC7RwnV9qPtRd0dawbRS7yZgb+1gi+axTLYZ4Nsm\\\\\\\/4EoH0c+xBr\\\\\\\/QBqA55AbzSZTyRavYMjLZnkvwvojR4+fcHDxEJ8Jathn3Gn9im431eaen78UmQpOja0k9iNlz0\\\\\\\/\\\\\\\/LebT5lbaf3x0tXPGhChTe0iqGsSlM0WrrsNzwYTN93hfV3\\\\\\\/62hVivjI2yZFXmMuqlt2Ym0RKNMH1m9AEg9wdLBaWzaeb9N9ss08ON6OnauCeO4GKt2IygraH0J+EGVwnw\\\\\\\/YfQ\\\\\\\/9ZL1+YD7b02mNX\\\\\\\/PrrxV9m8h75ULOvU5DX7naWNn68UVyL86lBomfBeJQJywxdyLkR5aWWyl6wSWVvAF18IFwJRA2ofSz2qmiqc2hhjk95uUHwz5SD08bgSli7PB4hFEEAbXh6IPsAY9Sd7+3LAm7C8VtwyiyUl4eeOmZvOe+nopDEJENu6zom9VAo0JaBeGghZU+kbqpc\\\\\\\/4gEc7UT\\\\\\\/omXTkbI3Ki9YmIeekH2XHyCaBioI9qXbRKSO29kAR7XNUEElUCbeDqnHCvJroQM54E\\\\\\\/9Ghwb0FhMrck1QvluhSHiPQWMjGgOoG6TP2cpXktbT\\\\\\\/HSzxNzh5RXsz2TF0az0FLWtbueDHfpUNc4vvmlzp6djdIxcY4XqWjUKKM+3RctCVx8tcynohM+CTkKKB8161BJLtYRsjVUCbZb2A+R+BnsCJOr0RFMQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022E44PtnxjkXQV+U35AsHDzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022134bc985bc1e58765a8f039565f81618bdb815391075e8c8c92166372052fd32\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492323 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/855" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Tn0sl7tuu--BxvNkJ-7xP05L0WqNIa8gPrr7Udu5Z70" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/855" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-67RlbhdQKKjwKijTJstY5rEaHlYlfM6xkyG0FeT-Mco" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/841" title="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/867" title="Identification of Suppressor of Clathrin Deficiency-1 (SCD1) and Its Connection to Clathrin-Mediated Endocytosis in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492323" id="node492323" data-pisa="ggg;9/3/855" data-pisa-master="ggg;g3.118.200789" data-apath="/ggg/9/3/855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamic Interactions Between the Genome and an Endogenous Retrovirus: <em>Tirant</em> in <em>Drosophila simulans</em> Wild-Type Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marie Fablet</span>, <span class="highwire-citation-author" data-delta="1">Angelo Jacquet</span>, <span class="highwire-citation-author" data-delta="2">Rita Rebollo</span>, <span class="highwire-citation-author" data-delta="3">Annabelle Haudry</span>, <span class="highwire-citation-author" data-delta="4">Carine Rey</span>, <span class="highwire-citation-author" data-delta="5">Judit Salces-Ortiz</span>, <span class="highwire-citation-author" data-delta="6">Prajakta Bajad</span>, <span class="highwire-citation-author" data-delta="7">Nelly Burlet</span>, <span class="highwire-citation-author" data-delta="8">Michael F. Jantsch</span>, <span class="highwire-citation-author" data-delta="9">Maria Pilar García Guerreiro</span> and <span class="highwire-citation-author" data-delta="10">Cristina Vieira</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">855-865;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200789 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492323" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie Fablet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BFablet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fablet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarie%2BFablet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marie Fablet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marie.fablet@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">marie.fablet@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angelo Jacquet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angelo%2BJacquet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquet%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngelo%2BJacquet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita Rebollo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BF2i, Univ Lyon, INSA-Lyon, INRA, UMR 0203, Villeurbanne 69621, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BRebollo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rebollo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BRebollo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annabelle Haudry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabelle%2BHaudry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haudry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnabelle%2BHaudry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carine Rey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire de Biologie et Modélisation de la Cellule, UnivLyon, ENS de Lyon, Univ Claude Bernard, CNRS UMR 5239, INSERM U1210, Lyon 69007, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BRey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BRey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Judit Salces-Ortiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judit%2BSalces-Ortiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salces-Ortiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudit%2BSalces-Ortiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prajakta Bajad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prajakta%2BBajad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bajad%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrajakta%2BBajad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nelly Burlet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nelly%2BBurlet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burlet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANelly%2BBurlet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael F. Jantsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BJantsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jantsch%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BJantsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Maria Pilar García Guerreiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grup de Genòmica, Bioinformàtica i Biologia Evolutiva, Departament de Genètica i Microbiologia, Universitat Autònoma de Barcelona, Bellaterra-Barcelona 08193, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BPilar%2BGarc%C3%ADa%2BGuerreiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guerreiro%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BPilar%2BGarc%25C3%25ADa%2BGuerreiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Cristina Vieira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cristina%2BVieira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vieira%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACristina%2BVieira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/855" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492323" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492323/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/855.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492323" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492323/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/855.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492323" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492323/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/855.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492323" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492323/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/855.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492323" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492323/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">All genomes contain repeated sequences that are known as transposable elements (TEs). Among these are endogenous retroviruses (ERVs), which are sequences similar to retroviruses and are transmitted across generations from parent to progeny. These sequences are controlled in genomes through epigenetic mechanisms. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences. The <em>tirant</em> ERV of <em>Drosophila simulans</em> displays intra-specific variability in copy numbers, insertion sites, and transcription levels, providing us with a well-suited model to study the dynamic relationship between a TE family and the host genome through epigenetic mechanisms. We show that <em>tirant</em> transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, unlike what was previously described following divergent crosses. In addition, we describe <em>tirant</em> insertion polymorphism in the genomes of three <em>D. simulans</em> wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transposable-element" class="hw-term hw-article-keyword hw-article-keyword-transposable-element" rel="nofollow">Transposable element</a></li><li class="kwd"><a href="/keyword/retrotransposon" class="hw-term hw-article-keyword hw-article-keyword-retrotransposon" rel="nofollow">Retrotransposon</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/epigenetic-control" class="hw-term hw-article-keyword hw-article-keyword-epigenetic-control" rel="nofollow">epigenetic control</a></li><li class="kwd"><a href="/keyword/pirna" class="hw-term hw-article-keyword hw-article-keyword-pirna" rel="nofollow">piRNA</a></li><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li></ul><p id="p-4">All genomes contain repeated sequences that are known as transposable elements (TEs). These are sequences that can move and multiply along the chromosome arms, generating most of the time deleterious mutations. A few decades ago, TEs were only acknowledged as parasitic “junk DNA”; however, we now have evidence that host-TE relationships may range from parasitism to mutualism, as do all symbiotic interactions (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Biémont and Vieira 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jangam <em>et al.</em> 2017</a>). TEs may be of various structures (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wicker <em>et al.</em> 2007</a>), among which are endogenous retroviruses (ERVs). ERVs are sequences similar to retroviruses and are transmitted across generations from parent to progeny. Due to the potential harmful activity of TEs, mechanisms that allow to control them have been favored by natural selection, and consist mainly of epigenetic processes (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Slotkin and Martienssen 2007</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Siomi <em>et al.</em> 2011</a>). In <em>Drosophila</em>, knowledge of such processes is increasingly accumulating. At the center of the epigenetic control of TEs are small interfering RNAs of the piRNA class, which trigger heterochromatinization of TE sequences through trimethylation of histone 3 lysine 9 (H3K9me3) (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sienski <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Le Thomas <em>et al.</em> 2013</a>).</p><p id="p-5">TE impacts on genomes have varied nature and extents, from deleterious to adaptive (reviewed in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cordaux and Batzer 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casacuberta and González 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fablet <em>et al.</em> 2017</a>). For instance, TE insertions may cause diseases (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hancks and Kazazian 2016</a>), whereas in some cases, TEs may be recruited by the host genome and perform endogenous functions. The most famous example is the <em>syncytin</em> gene in mammals, which is an ancient TE gene and is now essential for placenta formation (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mi <em>et al.</em> 2000</a>). TEs may also be involved in adaptation. A<em>s</em> an example, the <em>Accord</em> TE inserted into the 5′ end of the <em><a href="http://flybase.org/reports/FBgn0025454.html">Cyp6g1</a></em> gene of <em>Drosophila melanogaster</em> is found in some strains and provides them with increased resistance to insecticides (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daborn <em>et al.</em> 2002</a>).</p><p id="p-6">Despite their potential harmful impacts, TEs may reach high proportions of genome sequences (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Biémont and Vieira 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Tenaillon <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elliott and Gregory 2015</a>). Therefore, it is of fundamental interest to precisely understand TE dynamics within genomes, especially at the transcriptional level, which is the preliminary stage before mobilization. In this study, we propose to use the <em>tirant</em> element of <em>Drosophila simulans</em> to tackle this important issue.</p><p id="p-7"><em>Tirant</em> belongs to the ERV class of retrotransposons (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Terzian <em>et al.</em> 2001</a>). It is made of three open reading frames (ORFs), namely <em>gag</em>, <em>pol</em>, and <em>env</em>, which provide all proteins necessary to fulfill the retroviral cycle, and is bordered by two long terminal repeats (LTRs), which include regulatory signals. Two subfamilies can be distinguished based on their sequence similarity and the structure of a minisatellite located in the 5′ untranslated region (UTR) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>). Subfamily C is responsible for <em>tirant</em> activity while subfamily S is heterochromatic, low-copy-number, and silent (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2009</a>). <em>Tirant</em> was first discovered in <em>D. melanogaster</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garrell and Modolell 1990</a>) and found to occur in related species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fablet <em>et al.</em> 2007</a>). Its number of copies and insertion sites, together with expression profiles vary across <em>D. simulans</em> wild-type strains (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Vieira <em>et al.</em> 1999</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akkouche <em>et al.</em> 2012</a>), providing a powerful system to investigate the impacts of ERVs on the host genome, particularly regarding neighboring gene expression and epigenetic signatures.</p><p id="p-8">The analyses of RNA-seq and small RNA-seq data indicate that <em>tirant</em> transcript amounts and piRNA amounts are positively correlated in ovaries in normal conditions, while they were previously found to be negatively correlated in some divergent crosses leading to <em>tirant</em> deregulation. These results allow us to propose that the shape of the correlation between the amounts of TE transcripts and the corresponding piRNAs are indicative of genome stability. In addition, we describe <em>tirant</em> insertion polymorphism in the genomes of three <em>D. simulans</em> wild-type strains, which reveals a limited number of insertions that may be associated with gene transcript level changes through heterochromatin spreading and have phenotypic impacts. Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Drosophila stocks</h3><p id="p-9"><em>Drosophila</em> strains were maintained in the laboratory at 24° as small-mass cultures. Chicharo, Mayotte, Makindu and Zimbabwe are wild-type strains, anciently sampled from the field; w501 is the major strain of the <em>D. simulans</em> sequenced genome.</p></div><div id="sec-3" class="subsection"><h3>Small RNA analysis</h3><p id="p-10">We used small RNA-seq data obtained from ovaries from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> (2013)</a> and <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> (2017)</a>, corresponding to the Chicharo, Mayotte and Makindu strains. We first removed adapter sequences using cutadatp (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Martin 2011</a>) -a CTGTAGGCACCATCAA. Using PRINSEQ lite version 0.20.4 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schmieder and Edwards 2011</a>), we filtered reads of size 23 to 30 nt and considered these as piRNAs. Using a modified version of the TEcount module of TEtools (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>), we mapped the reads against a list of TE sequences either in the sense or antisense directions. The list of TE sequences is made of the TE insertions retrieved from <em>D. simulans</em> sequenced genome (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>). Read count numbers were normalized to miRNA read count numbers (miRNA sequences retrieved from FlyBase: dsim-all-miRNA-r2.02.fasta.gz).</p><p id="p-11">We looked for ping-pong signatures using signature.py with the options min_size = 23 and max_size = 30 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Antoniewski 2014</a>). We used as input files SAM alignments obtained from cleaned small RNA reads against our C-subfamily complete (8.5 kb) reference <em>tirant</em> sequence: accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AC0054444&amp;atom=%2Fggg%2F9%2F3%2F855.atom">AC0054444</a>, positions 50,203 to 58,729, extracted from the <em>D. melanogaster</em> genome, as already described in <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> (2006)</a>. We used bowtie (–best) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langmead <em>et al.</em> 2009</a>). We analyzed base composition at each position for piRNAs aligned against <em>tirant</em> using SAMStat (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lassmann <em>et al.</em> 2011</a>).</p><p id="p-12">To visualize read mapping along <em>tirant</em> sequence, we sampled 40,000,000 reads from each small RNA-seq sample using the fastq-sample program from fastq-tools 0.8 (<a href="https://homes.cs.washington.edu/%E2%88%BCdcjones/fastq-tools/">https://homes.cs.washington.edu/∼dcjones/fastq-tools/</a>), and then we selected 23-30 nt-long reads as mentioned above. We used bowtie (–best) (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Langmead <em>et al.</em> 2009</a>) to map these small RNA-seq reads against our C-subfamily reference <em>tirant</em> sequence (see above). We removed the 3′ LTR, so that reads corresponding to LTRs should map at a unique location. We filtered out alignments with a mapq score below 20 using samtools (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>). Visualization of the alignments was performed using BamView (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carver <em>et al.</em> 2010</a>).</p></div><div id="sec-4" class="subsection"><h3>Cell culture and reporter gene assays</h3><p id="p-13">We used reporter gene assays to test the sense and antisense promoter potentials of the LTRs of both subfamilies. LTR sequences were PCR amplified from the Mayotte strain (see Supplementary Material S1 for primer sequences). Primers were designed to include <em>Kpn</em>I and <em>Hin</em>dIII restriction sites, to allow subsequent directional cloning into the pGL4.10 plasmid (Promega, luciferase reporter gene) using T4 DNA ligase (NEB). S2 <em>Drosophila</em> cells were transfected using Cellfectin (Invitrogen) and lyzed for analysis after 24 h of incubation (6 replicates per construct). Luminescence was then estimated using the Dual-Luciferase Reporter Assay System (Promega).</p></div><div id="sec-5" class="subsection"><h3>Genome sequencing and assemblies</h3><p id="p-14">Genomic DNA samples were treated with RNAse A at 37° for 30 min and purified using Phenol:Chloroform extraction. The purified genomic DNA samples were fragmented using Covaris S220 sonicator to get fragments of size 300-800 bp. Sequencing libraries were prepared from these fragmented DNA samples using NEBNext UltraTM DNA Library Prep Kit (Illumina, # E7370L). All three libraries were sequenced in one lane of an Illumina HiSeqV4 with 125 bp paired-end reads at the Next Generation Sequencing unit of the Vienna Biocenter Core Facilities (VBCF http://vbcf.ac.at).</p><p id="p-15">Approximately 100 million read pairs (24 Gb) from three accessions (Mayotte, Makindu and Chicharo) were obtained and raw reads were filtered using the unsupervised quality trimming program UrQt (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Modolo and Lerat 2015</a>). Contigs were then generated by the Ray assembler v2.3.1 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boisvert <em>et al.</em> 2010</a>) using kmer length varying from 25 to 45. Best assemblies were selected by maximizing both NG50 and the number of contigs larger than 10 kb. For each accession, the assembled genome size was over 100% of the expected genome size (146.7 Mb). The final contig NG50 were 5.38 kb (k = 43), 11.89 kb (k = 39) and 7.36 kb (k = 35), with maximum scaffold length of ∼130 kb, 495 kb and 495 kb for Mayotte, Makindu and Chicharo, respectively. In order to assess completeness of genome assemblies, we searched for conserved genes across arthropods using BUSCO (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Simão <em>et al.</em> 2015</a>). Among the set of 2,675 genes, 2,254 (84%), 2,403 (90%) and 2,269 (85%) were recovered as complete and single copy in the assemblies of Mayotte, Makindu and Chicharo, respectively.</p></div><div id="sec-6" class="subsection"><h3>Genome and transcriptome analyses</h3><p id="p-16">We ran RepeatMasker (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Smit <em>et al.</em> 2013</a>) on the above assemblies (default parameters, -species Drosophila) and retrieved “Tirant” accessions in the output file in order to identify <em>tirant</em> insertion sites. Then, we extracted the flanking sequences for the insertions found above and located them on droSim1 assembly using the BLAT tool (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kent 2002</a>) of UCSC Genome Browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>). We checked that shared <em>tirant</em> insertions had the same breakpoints in all strains.</p><p id="p-17">RNA-seq data from ovaries and the corresponding computed read counts for all TE families, including <em>tirant</em>, were obtained from <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> (2017)</a> for five strains (Chicharo, Makindu, Mayotte, Zimbabwe, and w501). Differential expression analysis was performed using DESeq2 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Love <em>et al.</em> 2014</a>), which model internally corrects for library size, and provides normalization. Read counts obtained for genes on one hand and TEs on the other hand (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>) were concatenated to make the complete read count table that was analyzed at once using DESeq2.</p><p id="p-18">The <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> gene had no assigned <em>D. simulans</em> ortholog in the FlyBase gene list that we used for RNA-seq analysis at that time (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>). In 2018, FBgn0194057 has been assigned as <em>D. simulans tkv</em> ortholog (OrthoDB v9.1). Thus, we retrieved the FBgn0194057 sequence, performed the same procedure as for the other genes (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>), and added the corresponding read counts to our count table.</p><p id="p-19">To visualize read mapping along <em>tirant</em> sequence, we sampled 90,000,000 reads from each DNA-seq sample and 25,000,000 reads from each RNA-seq sample (one biological replicate from each strain) to ensure even abundance of reads. In addition, for RNA-seq data, we only used reads longer than 20 nt (PRINSEQ filter, as described above). We used the fastq-sample program from fastq-tools 0.8 (<a href="https://homes.cs.washington.edu/%E2%88%BCdcjones/fastq-tools/">https://homes.cs.washington.edu/∼dcjones/fastq-tools/</a>) and the fastq_sampler.py script (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Modolo 2018</a>). Then, we used bowtie2 (–very-sensitive) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langmead and Salzberg 2012</a>) and mapped DNA-seq and RNA-seq reads against our C-subfamily reference <em>tirant</em> sequence (see above). We removed the 3′ LTR, so that reads corresponding to LTRs should map at a unique location. We filtered out alignments with a mapq score below 20 using samtools (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Li <em>et al.</em> 2009</a>). Visualization of the alignments was performed using BamView (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Carver <em>et al.</em> 2010</a>).</p><p id="p-20">Transcript structure analysis: we used apytram (apytram v1.1 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rey <em>et al.</em> 2017</a>)) to assemble independently <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> in Chicharo, Makindu and Mayotte RNA-seq samples (replicates #1). Apytram is an implementation of the Target Restricted Assembly Method (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Johnson <em>et al.</em> 2013</a>). This method allows to focus on the assembly of only one gene of a transcriptome and not the whole transcriptome. It is based on an iterative process: at the first iteration, a reference gene from another species is used as bait sequences to fish reads in RNA-seq data using BLAST (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Camacho <em>et al.</em> 2009</a>). Paired reads are used to enlarge this batch of reads. Reads are then <em>de novo</em> assembled using Trinity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Grabherr <em>et al.</em> 2011</a>), and a new iteration begins with the reconstructed sequences as baits. Here, we used the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> genes from <em>D. melanogaster</em> as references. This iterative process allows to reconstruct transcripts step by step from homologous regions between <em>D. melanogaster</em> and <em>D. simulans</em>, and thus non homologous regions can be reconstructed in <em>D. simulans</em> transcripts. In addition, if ever <em>tirant</em> is exonized in one of the strains, apytram will be able to catch it. Then, to analyze <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> transcript structures in our samples, we aligned output transcripts on <em>D. simulans</em> genome (ASM75419v3.41) using exonerate (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Slater and Birney 2005</a>). Finally, we manually checked the hit distribution against <em>D. simulans</em> genome to search for the presence of <em>tirant</em> in transcript sequences. We used TopHat2 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2013</a>) and ggsashimi (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Garrido-Martín <em>et al.</em> 2018</a>) to view transcript structures using sashimi plots.</p></div><div id="sec-7" class="subsection"><h3>ChIP assays</h3><p id="p-21">Biological duplicates were produced. For each replicate, seventy ovary pairs were dissected from three to five day-old females and flash frozen to maintain chromatin integrity. Chromatin extraction and sonication were carried out as described in <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> (2013)</a> with the following modifications: 30 sonication cycles (30 sec on and 30 sec off on high) were performed to obtain chromatin fragments comprised between 200 and 500 bp. The chromatin fragments were then aliquoted and diluted into six IP samples of 500 µL along with a 10% Input sample. IP samples were then immunoprecipitated following the manufactor’s protocol (Magna ChIP, Millipore) with 3 µg of antibodies (H3K4me3: Abcam # ab8580; H3K9me3: Actif Motif # 39161).</p><p id="p-22">Quantitative PCR (qPCR) was used to estimate IP enrichment as previously described (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fablet <em>et al.</em> 2009</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> 2013</a>) and performed as technical triplicates. For each of the studied <em>tirant</em> insertions, we used qPCR primers located at the insertion site and ∼2 kb away, in both directions. At the insertion site, we designed primer pairs displaying one primer within <em>tirant</em> and the other primer in the flanking sequence. For strains devoid of <em>tirant</em> insertion, we used the same primer in the flanking region as mentioned above and designed a second primer in the flanking region beyond the site of <em>tirant</em> insertion. These two primer pairs allow us to amplify <em>tirant</em> insertion sites in strains where <em>tirant</em> is present as well as in strains were <em>tirant</em> is absent. Primer sequences are provided in Supplementary Material S1. As controls, we used <em>light</em> for H3K9me3 enrichment (Fw: 5′-GCT AGG CAA TGA CAA AGT CCT TTG GG-3′ and Rv: 5′-GCA TTC GTC TGA AGT CGG CAG ATA G-3′), and <em>rpl32</em> for H3K4me3 enrichment (Fw: 5′-CAG CTT CAA GAT GAC CAT C-3′ and Rv: 5′-GTT CGA TCC GTA ACC GAT GT-3′). Raw data from qPCRs are provided in Supplementary Material S2.</p></div><div id="sec-8" class="subsection"><h3>Long PCR</h3><p id="p-23">We PCR amplified <em>tirant</em> insertion at the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> locus using DreamTaq (Fermentas), with a 7 min elongation time and a 68° hybridization. Primer sequences are 5′- GCAACACTGACAGCAACTACA -3′ and 5′- TCCTTGCTAGCTACATGGAAC -3′.</p></div><div id="sec-9" class="subsection"><h3>Data Availability</h3><p id="p-24">Genomic raw reads are available under the SRA accession number SRP128969. Genome assemblies, modified versions of TEcount.py and the list of TE sequences are available at <a href="ftp://pbil.univ-lyon1.fr/pub/datasets/Fablet2018">ftp://pbil.univ-lyon1.fr/pub/datasets/Fablet2018</a>. SupplMat S1 contains primer sequences. SupplMat S2 contains ChIP qPCR results. SupplMat S3 is an alignment of reference C and S sequences. Alignments of piRNAs against <em>tirant</em> reference sequence are provided Figure S1. Figure S2 displays graphics of piRNA base composition. Gels obtained from PCR products are shown Figure S3. Sashimi plots for <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> and <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> splicing are provided Figure S4. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7578380">https://doi.org/10.25387/g3.7578380</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Tirant is differentially expressed across wild-type strains</h3><p id="p-25">Using RNA-seq data obtained from ovary samples, we were able to quantify <em>tirant</em> transcript levels in <em>D. simulans</em> strains (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). As expected (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">2009</a>), subfamily S transcript levels were found at background level in all strains, and so were subfamily C transcript levels in w501, Chicharo and Zimbabwe strains. <em>Tirant</em> produces significant amounts of transcripts only in Makindu and Mayotte strains (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). The mapping of <em>tirant</em> reads against the reference sequence shows enrichment in the <em>env</em> region (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). This pattern could reflect either the existence of a master copy deleted for <em>gag</em> and <em>pol</em>, or the prevalence of splicing events of the <em>gag</em> and <em>pol</em> ORFs, as described by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marsano <em>et al.</em> (2000)</a> in <em>D. melanogaster</em>. The higher abundance of <em>env</em> reads in the Mayotte strain is in agreement with the production of the <em>tirant</em> Env protein in this strain, which we previously observed using immunostaining (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akkouche <em>et al.</em> 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tirant trancripts analyzed from RNA-seq data. A. Tirant transcript normalized read counts. B. Mapping of tirant reads against the reference tirant copy (subfamily C). The upper part is tirant structure, to scale (in bp). The lower part is read coverage along tirant sequence, obtained from samples of 25,000,000 paired-end reads. We removed LTR 3&#x2032; to get rid of multi-mapping issues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Tirant&lt;/em&gt; trancripts analyzed from RNA-seq data. A. Tirant transcript normalized read counts. B. Mapping of tirant reads against the reference tirant copy (subfamily C). The upper part is tirant structure, to scale (in bp). The lower part is read coverage along tirant sequence, obtained from samples of 25,000,000 paired-end reads. We removed LTR 3&#x2032; to get rid of multi-mapping issues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F1.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/855/F1.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child"><em>Tirant</em> trancripts analyzed from RNA-seq data. A. Tirant transcript normalized read counts. B. Mapping of tirant reads against the reference tirant copy (subfamily C). The upper part is tirant structure, to scale (in bp). The lower part is read coverage along tirant sequence, obtained from samples of 25,000,000 paired-end reads. We removed LTR 3′ to get rid of multi-mapping issues.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Tirant displays specific sense and antisense piRNAs</h3><p id="p-27">We specifically analyzed small RNAs for the two strains that show significant <em>tirant</em> transcript levels (<em>i.e.</em>, Makindu and Mayotte), and one strain presenting background levels (<em>i.e.</em>, Chicharo). We found sense and antisense <em>tirant</em>-specific piRNAs, corresponding to both the C and S subfamilies. Sequence similarity between subfamilies C and S varies along <em>tirant</em> sequence: it is the lowest in the UTR 5′ region (∼65% (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>)) and displays higher values within coding sequences (76–85% global similarity rate (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Fablet <em>et al.</em> 2009</a>)). Although we cannot exclude that some reads may be mis-attributed to the other subfamily, it should not happen for the majority. Except for a peak in the LTR region, <em>tirant</em> piRNAs evenly map along <em>tirant</em> reference sequence in all three strains (Figure S1). <em>Tirant</em> piRNAs displayed comparable proportions of antisense piRNAs to total piRNAs in all three strains and <em>tirant</em> subfamilies, ranging from 0.798 to 0.869 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In addition, we looked for ping-pong signatures, which are 10 nt overlaps between sense and antisense piRNAs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brennecke <em>et al.</em> 2007</a>). We found clear, similar enrichments for 10 nt overlaps in all three strains (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>), indicative of similar, functional, secondary piRNA pathways against <em>tirant</em> in all three strains. In addition, these piRNAs showed expected enrichment in 1U and 10A (Figure S2) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Saito <em>et al.</em> 2006</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brennecke <em>et al.</em> 2007</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tirant-specific piRNAs. A. Relative proportions of sense and antisense piRNAs. B. Positive correlation between tirant transcript normalized read counts and tirant piRNA read counts in wild-type strains. C. Ping-pong signatures: there is a clear enrichment in 10 nt overlaps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Tirant&lt;/em&gt;-specific piRNAs. A. Relative proportions of sense and antisense piRNAs. B. Positive correlation between tirant transcript normalized read counts and tirant piRNA read counts in wild-type strains. C. Ping-pong signatures: there is a clear enrichment in 10 nt overlaps.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F2.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/855/F2.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492498" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child"><em>Tirant</em>-specific piRNAs. A. Relative proportions of sense and antisense piRNAs. B. Positive correlation between tirant transcript normalized read counts and tirant piRNA read counts in wild-type strains. C. Ping-pong signatures: there is a clear enrichment in 10 nt overlaps.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Moreover, our data allowed us to reveal a positive correlation between <em>tirant</em> piRNA read counts and <em>tirant</em> transcript read counts (Pearson correlation, r = 0.963, p-value = 0.002, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The correlation remains strong using only sense (Pearson correlation, r = 0.965, p-value = 0.002), or antisense (Pearson correlation, r = 0.946, p-value = 0.004) piRNAs.</p></div><div id="sec-13" class="subsection"><h3>Tirant LTRs display antisense promoter activity</h3><p id="p-30">As new TE insertions may behave as dual-strand piRNA clusters (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Mohn <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Shpiz <em>et al.</em> 2014</a>), we wondered whether <em>tirant</em> copies could be able to produce antisense piRNA precursors. To investigate such hypothesis, we tested the C and S LTRs for their sense and antisense promoter abilities. We known from a previous work that sequence similarity between C and S LTRs is 65% on average, while sequence similarity within each subfamily is higher than 98% (<a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>). It appears that the different constructs behave significantly differently regarding gene expression promotion (Kruskal Wallis test, df = 3, p-value = 2.10<sup>−4</sup>), with LTR C sense being the strongest promoter and LTR S sense being the weakest. In addition, we find that both subfamily LTRs display weak antisense promoter activity (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) (Wilcoxon tests, C antisense <em>vs.</em> S sense: p-value = 0.002, S antisense <em>vs.</em> S sense: p-value = 0.002, C antisense <em>vs.</em> S antisense: p-value = 0.485). Therefore, the results suggest that, although subfamily S is not significantly involved in the production of <em>tirant</em> transcripts, and albeit the weak activity observed <em>in vitro</em>, it may rather be involved in the production of antisense piRNA precursors.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tirant LTR promoter activity. LTRs of C and S subfamilies in sense and antisense orientations were cloned upstream of a luciferase reporter gene and tested for promoter activity in S2 cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Tirant&lt;/em&gt; LTR promoter activity. LTRs of C and S subfamilies in sense and antisense orientations were cloned upstream of a luciferase reporter gene and tested for promoter activity in S2 cells.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F3.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/855/F3.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child"><em>Tirant</em> LTR promoter activity. LTRs of C and S subfamilies in sense and antisense orientations were cloned upstream of a luciferase reporter gene and tested for promoter activity in S2 cells.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Tirant displays insertional polymorphism</h3><p id="p-32">The influence of a TE on the host genome depends on its insertion sites. To investigate <em>tirant’s</em> impacts, we produced genome assemblies for Chicharo, Makindu and Mayotte, and used RepeatMasker (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Smit <em>et al.</em> 2013</a>) to retrieve <em>tirant</em> insertion sites. We excluded scaffolds exclusively made of <em>tirant</em>. Nested <em>tirant</em> insertions into other TE sequences were also excluded since we are not able to locate them precisely on the genome. The final number of retrieved <em>tirant</em> insertions is congruent with our knowledge of <em>tirant</em> copy numbers (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>, <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">2009</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Akkouche <em>et al.</em> 2012</a>), indicating that we did not missed a significant number of loci.</p><p id="p-33">Overall, we localized precisely 15 insertions, the majority of which are unique to one strain (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Among these, six are located in introns and two upstream transcription start sites (TSSs), which are the insertions the most likely to have an impact on genes. Insertion #10 is located 242 bp upstream of <em><a href="http://flybase.org/reports/FBgn0010015.html">CanA1</a></em> TSS, and insertion #11 533 bp upstream of <em><a href="http://flybase.org/reports/FBgn0032949.html">Lamp1</a></em> TSS. We identified five additional insertions, attributed to the U chromosome (Unknown chromosome), and which likely correspond to heterochromatic regions. Among these, one is shared across strains.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492334/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492334/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><em>Tirant</em> insertion sites as identified using RepeatMasker on genome assemblies. +: insertion found in the genome assembly of the strain. Due to the fragmented nature of our assemblies, we are not able to interpret “absence of detection” as “absence of tirant”. These data do not distinguish tirant subfamilies. TSS: Transcription Start Site. U: Unkown chromosome. ≥ LTR: we detected one LTR (417 bp) at the extremity of a scaffold, which means that we do not know whether the insertion is a solo LTR or longer. Insertion #10 is located 242 bp upstream of CanA1 TSS, and insertion #11 533 bp upstream of Lamp1 TSS. D. simulans orthologs as assigned in OrthoDB v9.1 (retrieved via FlyBase in november 2018) are mentioned in brackets and italics.</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">It is to be noted that, following this procedure, the <em>tirant</em> insertion into the <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> gene (insertion #12) is only found in the Makindu genome, while we have experimental evidence that it also exists in the Mayotte strain. This stresses on the non exhaustive nature of this approach, which strongly relies on scaffold assemblies. Indeed, we used a PCR approach to confirm <em>tirant</em> insertion polymorphism for insertions #3 and #12, which are studied in more details below. (See Supplementary Material S1 for primer sequences). For insertion #3, PCR profiles were congruent with <em>tirant</em> being present only in Mayotte and absent in Makindu and Chicharo (Figure S3A). For insertion #12, PCR profiles indicated <em>tirant</em> presence in Makindu and Mayotte, and <em>tirant</em> absence in Chicharo (Figure S3B). However, we could also detect a very faint band indicating <em>tirant</em> absence in Makindu. This suggests that this insertion may be heterozygous or even absent in few individuals.</p><p id="p-35">Accessing the precise structure of each <em>tirant</em> insertion is difficult without long reads. Therefore, we mapped genomic reads against our <em>tirant</em> reference sequence to have an idea of the representativeness of each region within the genomes of our strains. We found that reads map evenly along the reference sequence for Chicharo and Makindu, suggesting the presence of full-length copies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Mayotte displays the highest number of reads after normalization, in agreement with a higher number of <em>tirant</em> copies. We note a slight depression in the center of the <em>pol</em> and <em>env</em> ORFs (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), suggesting the existence of internally deleted copies. The drop of mapping coverage within the 5′ UTR region is due to our mapq filter (see Material and Methods), which prevents multi-mapping against repeated regions, such as the minisatellite that is described within <em>tirant</em> 5′ UTR (<a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of genomic reads against tirant reference sequence. The upper part represents tirant structure, to scale (in bp). The lower part depicts read coverage along tirant sequence (subfamily C), obtained from samples of 90,000,000 paired-end reads. We removed LTR 3&#x2032; to get rid of multi-mapping issues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping of genomic reads against &lt;em&gt;tirant&lt;/em&gt; reference sequence. The upper part represents tirant structure, to scale (in bp). The lower part depicts read coverage along tirant sequence (subfamily C), obtained from samples of 90,000,000 paired-end reads. We removed LTR 3&#x2032; to get rid of multi-mapping issues.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F4.medium.gif" width="440" height="65"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/3/855/F4.medium.gif" width="440" height="65"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492500" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Mapping of genomic reads against <em>tirant</em> reference sequence. The upper part represents tirant structure, to scale (in bp). The lower part depicts read coverage along tirant sequence (subfamily C), obtained from samples of 90,000,000 paired-end reads. We removed LTR 3′ to get rid of multi-mapping issues.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Mild impact of tirant on neighboring genes</h3><p id="p-37">For those insertions located near or within genes (within introns), we tested whether <em>tirant</em> could have an effect on gene expression through modifications of chromatin structure. To do so, we used data produced from ovaries, and we focused on the insertions of <em>tirant</em> that fulfill the following criteria: i) the gene into which <em>tirant</em> is inserted displays a high enough number of reads in our RNA-seq dataset (&gt; 100 reads), ii) its expression pattern is contrasted across strains with and without the <em>tirant</em> insertion (<a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>). We found only two of such insertions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The two insertions of interest are located within the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> genes, respectively (insertions #3 –only in Mayotte– and #12 –in Mayotte and Makindu– in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, respectively). In both cases, gene transcript levels are lower for strains where <em>tirant</em> is present compared to strains devoid of <em>tirant</em> at the considered locus (DESeq2 padj values: <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em>: Chicharo <em>vs.</em> Mayotte: 0.0157, Makindu <em>vs.</em> Mayotte: 0.0916; <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em>: Chicharo <em>vs.</em> Makindu: 0.0617, Chicharo <em>vs.</em> Mayotte: 2.10<sup>−8</sup>). Moreover, FlyBase <em>D. simulans</em> genome annotation indicates the presence of additional genes within <em>Hs6st</em> (FBgn0190828, FBgn0270486, FBgn0190826, and FBgn0269631) and <em>tkv</em> (FBgn0270637, FBgn0194717, FBgn0270509, and FBgn0194718) introns. However, virtually none of the reads of our data sets map against these sequences.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RNA-seq results for genes at or near tirant insertions. Normalized read counts. Please refer to Table 1 for tirant insertion details. Error bars are standard deviations on biological duplicates. When tirant insertion is polymorphic, we provide adjusted p-values comparing strains with and without tirant insertion, as calculated by DESeq2. Gene CG17684 (insertion #15) has no D. simulans ortholog in the list of genes used for RNA-seq data mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RNA-seq results for genes at or near &lt;em&gt;tirant&lt;/em&gt; insertions. Normalized read counts. Please refer to Table 1 for tirant insertion details. Error bars are standard deviations on biological duplicates. When tirant insertion is polymorphic, we provide adjusted p-values comparing strains with and without tirant insertion, as calculated by DESeq2. Gene CG17684 (insertion #15) has no D. simulans ortholog in the list of genes used for RNA-seq data mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F5.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/3/855/F5.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">RNA-seq results for genes at or near <em>tirant</em> insertions. Normalized read counts. Please refer to <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> for tirant insertion details. Error bars are standard deviations on biological duplicates. When tirant insertion is polymorphic, we provide adjusted p-values comparing strains with and without tirant insertion, as calculated by DESeq2. Gene CG17684 (insertion #15) has no D. simulans ortholog in the list of genes used for RNA-seq data mapping.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">We investigated chromatin structure to look for epigenetic modifications that could spread on neighboring regions. We performed ChIP experiments on H3K4me3 and H3K9me3. The former is characteristic of open chromatin conformation, particularly found at promoters; the later is the heterochromatic mark known to be involved in TE transcriptional silencing. We used qPCR primers located at the insertion site and ∼2 kb away, in both directions (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">6B</a>). Primers at the insertion sites were the same as those mentioned above to detect <em>tirant</em> presence (Supplementary Material S1). No obvious effect of <em>tirant</em> presence on either chromatin mark at the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> gene is observed: Mayotte enrichment levels (<em>tirant</em> present) lie in between Chicharo and Makindu levels (<em>tirant</em> absent). On the contrary to <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> insertion, it appears that <em>tirant</em> neighborhood is moderately enriched in H3K9me3 at the <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> insertion site compared to strains without <em>tirant</em> (fold change = 1.43 ; <em>t</em>-test, p-value = 0.098). In addition, these histone post-translation modifications spread at least 2 kb away downstream <em>tirant</em> (fold change = 1.41 ;<em>t</em>-test, p-value = 0.002). Regarding H3K4me3 enrichments, we could not detect differences between strains with and without <em>tirant</em> at the <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> locus.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/855/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of two tirant insertions within introns. A. and B. Histone mark enrichments at two tirant insertion sites. P-values in the upper left corners are produced from t-tests between strains with the tirant insertion (yellow) and strains without tirant (uncolored). Left panel: A. tirant insertion #3 into Hs6st, right panel B.: tirant insertion #12 into tkv. Genes are drawn to scale in the middle of each panel; thin lines: introns, thick boxes: exons. H3K4me3 and H3K9me3 enrichments are shown in green and red, respectively, and quantified relative to control genes (see Material and Methods section). They are displayed in boxes corresponding to the location of the qPCR amplicons: either at the tirant insertion site or 2 kb away. Biological duplicates were produced for ChIP experiments. &#x201C;+&#x201D; indicates the mean between biological replicates. Normalized transcript levels for the considered genes are provided in the upper right corners (as already shown Figure 5). Error bars are standard deviations. Strains in which tirant is present at the considered locus are highlighted in yellow. Statistical significance for RNA-seq results was assessed by padj values provided by the DESeq2 analysis. Padj values: Hs6st: Chicharo vs. Mayotte: 0.0157, Makindu vs. Mayotte: 0.0916, tkv: Chicharo vs. Makindu: 0.0617, Chicharo vs. Mayotte: 2.10&#x2212;8. C. and D. Vein phenotype in the Mayotte strain. C. We observed thick L5 veins (red arrows) in seven out of 97 flies in the Mayotte strain. D. Wild-type vein phenotype observed in 90 out of 97 flies in Mayotte (the same phenotype is also observed in all examined flies in Makindu and Chicharo). E. and F. Transcript structures. Reference structures from D. melanogaster, as retrieved from the UCSC Genome Browser, are depicted in dark green, thin lines: introns, thick boxes: exons. Reconstructed exons are shown in light blue. E. Hs6st transcript structures. All three strains display the same transcript structure as described in D. melanogaster. F. tkv transcript structures. We observe a Makindu-specific transcript of isoform D." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-386695520" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of two &lt;em&gt;tirant&lt;/em&gt; insertions within introns. A. and B. Histone mark enrichments at two tirant insertion sites. P-values in the upper left corners are produced from t-tests between strains with the tirant insertion (yellow) and strains without tirant (uncolored). Left panel: A. tirant insertion #3 into Hs6st, right panel B.: tirant insertion #12 into tkv. Genes are drawn to scale in the middle of each panel; thin lines: introns, thick boxes: exons. H3K4me3 and H3K9me3 enrichments are shown in green and red, respectively, and quantified relative to control genes (see Material and Methods section). They are displayed in boxes corresponding to the location of the qPCR amplicons: either at the tirant insertion site or 2 kb away. Biological duplicates were produced for ChIP experiments. &#x201C;+&#x201D; indicates the mean between biological replicates. Normalized transcript levels for the considered genes are provided in the upper right corners (as already shown Figure 5). Error bars are standard deviations. Strains in which tirant is present at the considered locus are highlighted in yellow. Statistical significance for RNA-seq results was assessed by padj values provided by the DESeq2 analysis. Padj values: Hs6st: Chicharo &lt;em&gt;vs.&lt;/em&gt; Mayotte: 0.0157, Makindu &lt;em&gt;vs.&lt;/em&gt; Mayotte: 0.0916, tkv: Chicharo &lt;em&gt;vs.&lt;/em&gt; Makindu: 0.0617, Chicharo &lt;em&gt;vs.&lt;/em&gt; Mayotte: 2.10&lt;sup&gt;&#x2212;8&lt;/sup&gt;. C. and D. Vein phenotype in the Mayotte strain. C. We observed thick L5 veins (red arrows) in seven out of 97 flies in the Mayotte strain. D. Wild-type vein phenotype observed in 90 out of 97 flies in Mayotte (the same phenotype is also observed in all examined flies in Makindu and Chicharo). E. and F. Transcript structures. Reference structures from D. melanogaster, as retrieved from the UCSC Genome Browser, are depicted in dark green, thin lines: introns, thick boxes: exons. Reconstructed exons are shown in light blue. E. Hs6st transcript structures. All three strains display the same transcript structure as described in D. melanogaster. F. tkv transcript structures. We observe a Makindu-specific transcript of isoform D.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/855/F6.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/3/855/F6.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/855/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/855/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492418" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Analysis of two <em>tirant</em> insertions within introns. A. and B. Histone mark enrichments at two tirant insertion sites. P-values in the upper left corners are produced from t-tests between strains with the tirant insertion (yellow) and strains without tirant (uncolored). Left panel: A. tirant insertion #3 into Hs6st, right panel B.: tirant insertion #12 into tkv. Genes are drawn to scale in the middle of each panel; thin lines: introns, thick boxes: exons. H3K4me3 and H3K9me3 enrichments are shown in green and red, respectively, and quantified relative to control genes (see Material and Methods section). They are displayed in boxes corresponding to the location of the qPCR amplicons: either at the tirant insertion site or 2 kb away. Biological duplicates were produced for ChIP experiments. “+” indicates the mean between biological replicates. Normalized transcript levels for the considered genes are provided in the upper right corners (as already shown <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Error bars are standard deviations. Strains in which tirant is present at the considered locus are highlighted in yellow. Statistical significance for RNA-seq results was assessed by padj values provided by the DESeq2 analysis. Padj values: Hs6st: Chicharo <em>vs.</em> Mayotte: 0.0157, Makindu <em>vs.</em> Mayotte: 0.0916, tkv: Chicharo <em>vs.</em> Makindu: 0.0617, Chicharo <em>vs.</em> Mayotte: 2.10<sup>−8</sup>. C. and D. Vein phenotype in the Mayotte strain. C. We observed thick L5 veins (red arrows) in seven out of 97 flies in the Mayotte strain. D. Wild-type vein phenotype observed in 90 out of 97 flies in Mayotte (the same phenotype is also observed in all examined flies in Makindu and Chicharo). E. and F. Transcript structures. Reference structures from D. melanogaster, as retrieved from the UCSC Genome Browser, are depicted in dark green, thin lines: introns, thick boxes: exons. Reconstructed exons are shown in light blue. E. Hs6st transcript structures. All three strains display the same transcript structure as described in D. melanogaster. F. tkv transcript structures. We observe a Makindu-specific transcript of isoform D.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Since <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> mutations are known to alter wing vein phenotypes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lindsley and Zimm 1992</a>), we examined wing morphologies in our flies (we did not separate sexes). We found seven flies out of 97 in the Mayotte strain which displayed thick L5 veins (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>, <a id="xref-fig-6-4" class="xref-fig" href="#F6">6D</a>), while we did not find any out of 100 Makindu flies nor 100 Chicharo flies (Fisher exact test, p-value = 0.014). It is to be noted that Mayotte is the strain that shows the strongest reduction in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> transcript levels.</p><p id="p-42">As it was recently shown that piRNAs may also regulate TEs through splicing (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Teixeira <em>et al.</em> 2017</a>), we analyzed transcript structures in <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em>. <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> displays virtually no transcript isoform variation in the reference <em>D. melanogaster</em> genome. Unsurprisingly, we found the described <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> transcript to be produced in all three strains (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6E</a>). In contrast, four <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> isoforms are described in <em>D. melanogaster</em>. Isoforms B and C are not found to be produced in our samples, and isoform A displays the same reference structure in all three strains. Variability was observed regarding isoform D: the first exon is 1,371 bp longer in the Makindu strain compared to the others (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6F</a>). Sashimi plots are provided Figure S4. Interestingly, this variable splice site is located less than 800 bp from <em>tirant</em> insertion in Makindu and Mayotte, and thus in the region where we find chromatin structure to be modified in association with <em>tirant</em> insertion. It is thus tempting to speculate that <em>tirant</em>-induced chromatin modifications may affect splice site definition. In addition, we note that we do not find evidence of <em>tirant</em> exonization neither in <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> nor in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em>.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>The repertoire of tirant-specific piRNAs</h3><p id="p-43">Our in-depth analysis of <em>tirant</em>-specific small RNAs revealed the presence of both sense and antisense piRNAs. While the primary piRNA pathway exclusively produces antisense piRNAs, the secondary piRNA pathway leads to the production of both sense and antisense piRNAs, characterized by the so-called ping-pong signature (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Brennecke <em>et al.</em> 2007</a>). Thus, the detection of sense <em>tirant</em> piRNAs indicates that the secondary pathway is involved in <em>tirant</em> control, as already proposed by previous experimental analyses (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> 2013</a>). This is reinforced by the identification of ping-pong signatures, which are similar across strains. However, we cannot exclude the involvement of the primary pathway as well, especially considering the high proportions of antisense <em>tirant</em>-specific piRNAs.</p><p id="p-44">So far, we do not know which piRNA clusters are involved in the production of <em>tirant</em>-specific piRNAs in <em>D. simulans</em> strains; however, our observation of antisense promoter properties for both subfamilies, albeit weak, indicates that some copies may have the ability to behave as dual-strand piRNA clusters, as proposed by two research studies (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Mohn <em>et al.</em> 2014</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Shpiz <em>et al.</em> 2014</a>). While the S subfamily is not transcribed and apparently not involved in <em>tirant</em> activity, this potential role in piRNA precursor production may also explain why it is found to be conserved across strains (<a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Fablet <em>et al.</em> 2006</a>). However, although we provide <em>in vitro</em> evidence that some <em>tirant</em> LTR sequences may behave as antisense promoters, we do not know whether the antisense <em>tirant</em> piRNAs that we detect are indeed produced from these promoters or from promoters outside of the element itself.</p><p id="p-45">Moreover, while we observed significant differences in <em>tirant</em> piRNA amounts across strains, we could not detect differences in sense <em>vs.</em> antisense piRNA ratios neither in ping-pong signatures, indicating that <em>tirant</em> control by the piRNA pathway may take place the same way and with comparable efficiencies in the three considered strains.</p></div><div id="sec-18" class="subsection"><h3>Dynamics of tirant control by piRNAs</h3><p id="p-46">Our survey in wild-type strains revealed a positive correlation between the amounts of <em>tirant</em> transcripts and <em>tirant</em> piRNAs in ovaries in normal conditions. This is in agreement with what we found using all TE families in the same strains (<a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Lerat <em>et al.</em> 2017</a>). This is also congruent with data obtained by others from <em>D. melanogaster</em> laboratory strains in normal conditions (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kelleher and Barbash 2013</a>), and in accordance with the accepted model claiming that TE transcripts fuel the ping-pong loop (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Senti <em>et al.</em> 2015</a>).</p><p id="p-47">Nevertheless, these results contrast with what was found in other studies. Notably, in a previous work, in order to understand <em>tirant</em> control, we performed crosses between Makindu and Chicharo, because they display contrasted <em>tirant</em> copy numbers and activities (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> 2013</a>). When the mother comes from Makindu and the father comes from Chicharo, <em>tirant</em> is properly controlled in the progeny. This direction of cross is called “RT” (Regulated Tirant). However, when the reciprocal cross is performed, <em>tirant</em> shows a strong accumulation of transcripts in the somatic follicle cells of the progeny’s ovaries (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> 2013</a>). This direction of cross is called “NRT” (Non-Regulated Tirant). This disruption of <em>tirant</em> control is associated with a lack of <em>tirant</em>-specific piRNAs, while these piRNAs remain relatively abundant in the RT direction of cross (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Akkouche <em>et al.</em> 2013</a>). It results in a negative correlation between <em>tirant</em> transcript amounts and <em>tirant</em>-specific piRNA amounts.</p><p id="p-48">We propose that the nature of the correlation between TE transcripts and piRNAs reflects genome stability. In normal conditions – which corresponds to a balanced relationship between the genome and its TEs, the correlation is positive. When the equilibrium is broken – and so far we do not know which sensor could detect such event –, the correlation becomes negative. Such patterns were also observed in the case of the <em>I</em> non-LTR retrotransposon in <em>D. melanogaster</em> strains (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chambeyron <em>et al.</em> 2008</a>). Indeed, Chambeyron <em>et al.</em> described that strain JA has no <em>I</em> transcripts nor <em>I</em>-specific piRNAs while strain HT2 displays high amounts of both <em>I</em> transcripts and piRNAs (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chambeyron <em>et al.</em> 2008</a>). This corresponds to the positive correlation we propose at equilibrium. On the other hand, when dysgenic crosses are performed and disrupt genome / <em>I</em> stable relationship, an opposite correlation is observed: SF females –which suffer from sterility– accumulate <em>I</em> transcripts due to a lack of <em>I</em> piRNAs; on the contrary, RSF females –in which fertility is restored– lower <em>I</em> transcript levels due to a high production of <em>I</em> piRNAs. Thus, the results obtained from <em>tirant</em> in the present work may reflect a behavior common to all TE families.</p></div><div id="sec-19" class="subsection"><h3>Tirant’s soft influence on neighboring genes</h3><p id="p-49">In most organisms, TE control is mainly achieved by epigenetic mechanisms (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Slotkin and Martienssen 2007</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Siomi <em>et al.</em> 2011</a>). In <em>D. melanogaster</em>, these essentially consist in piRNAs and H3K9me3 (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Saito <em>et al.</em> 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aravin <em>et al.</em> 2007</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Sienski <em>et al.</em> 2012</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Le Thomas <em>et al.</em> 2013</a>). We know that this heterochromatic mark may spread to flanking regions up to a 5 kb distance (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rebollo <em>et al.</em> 2011</a>). Rebollo <em>et al.</em> used mouse strains and did not find that heterochromatin spreading recurrently affected gene activity, probably due to the fact that such TE insertions would be too deleterious (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Rebollo <em>et al.</em> 2011</a>). In a <em>Drosophila</em> system, Sienski and colleagues also observed such spreading, which in addition did impact neighboring genes in mutants in cell cultures (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Sienski <em>et al.</em> 2012</a>). In the present study, we used the <em>tirant</em> system to investigate the spreading of TE chromatin marks in a natural context. We found a large level of insertion polymorphism across the strains we investigated (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). We identified two cases, corresponding to insertions into introns of the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> genes, associated with changes in gene transcript levels. While we could not detect any effect of <em>tirant</em> insertion on chromatin structure in the case of <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em>, <em>tirant</em> insertion into the <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> gene was associated with a moderate enrichment in H3K9me3 marks at the insertion site and up to 2 kb away from the insertion site, compared to the strain devoid of <em>tirant</em>. In addition, we noted that the <em>tirant</em> effect that we detect in Makindu may be underestimated since there may be heterozygosity or absence of the insertion in some individuals. We searched for other TEs in the genome assemblies of all strains in a ∼10 kb window around this particular <em>tirant</em> insertion, and could not find any. This suggests that the effects observed on chromatin structure and gene expression may be attributed to the insertion of <em>tirant</em>. Moreover, we observed a thick veins phenotype in 7% of Mayotte flies while we could not observe it in Makindu nor Chicharo flies. Mayotte is the strain displaying the strongest reduction in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> transcript levels (Mayotte <em>vs.</em> Chicharo: log2FC = -0.80). Although we cannot exclude other differences in genetic backgrounds, we propose that the <em>tirant</em> insertion into <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> in Mayotte may be responsible for this thick veins phenotype, through <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> expression reduction induced by heterochromatin formation. Nevertheless, variation in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> transcript levels is observed –probably associated with micro-environmental variability–, which may explain why most flies do not show the phenotype.</p><p id="p-50">In addition, as it was recently shown that piRNAs may also regulate TEs through splicing (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Teixeira <em>et al.</em> 2017</a>), we analyzed transcript structures in <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> and <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em>. We found no effect of <em>tirant</em> insertion in <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> transcript structures. This observation parallels the absence of <em>tirant</em> effect on <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> chromatin conformation. In contrast, we observed variation in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> splice sites nearby <em>tirant</em> insertion in the Makindu strain: the first exon of isoform D is 1,371 bp longer than expected. It is tempting to speculate that <em>tirant</em>-induced chromatin modifications disrupt splice site definition; however, we do not find the same pattern in the Mayotte strain, which also displays <em>tirant</em> insertion. Further investigation is needed at a larger scale to determine the impacts of TEs on splice site definition.</p><p id="p-51"><em>Tirant</em> insertions have either no or moderate effects on nearby gene expression. The insertion into <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> may have an impact on chromatin structure and gene expression whereas the <em>tirant</em> insertion into <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> does not. Our results are congruent with the recently published study of Lee and Karpen, although regarding a different histone mark. Based on a <em>D. melanogaster</em> genome-wide study, they report the spreading of TE-associated H3K9me2 marks on regions flanking the insertions and an effect on the expression level of flanking genes only for about half of the TE insertions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lee and Karpen 2017</a>). So far, we have not identified which properties of a TE insertion make it affect nearby genes or not. We know from a previous work that <em>tirant</em> is full length at the <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> locus in the Makindu strain (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Fablet <em>et al.</em> 2009</a>). We performed a long PCR at the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> locus and observed a band congruent with a full-length insertion as well (Figure S3C). In addition, both <em>tirant</em> sequences are oriented the same way as the gene. We may note that the <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em> locus displays insertion polymorphism for helitron <em>N1_Dbi</em>, which is found approximately 1 kb away from <em>tirant</em> in Chicharo and Mayotte but not in Makindu. Apart from this helitron and <em>tirant</em>, no other TE sequences are found within a 4 kb window around <em>tirant</em> insertion site. To sum up, we still do not know why <em>tirant</em> affects <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> but not <em><a href="http://flybase.org/reports/FBgn0038755.html">Hs6st</a></em>.</p><p id="p-52"><em>Tirant</em> effects are of small size: on average, a 30% reduction in <em><a href="http://flybase.org/reports/FBgn0003716.html">tkv</a></em> transcript amounts and an increase of 33–42% in H3K9me3 enrichment. If it were not a small effect, we speculate that the insertion would be too deleterious and removed from the genome by natural selection. In addition, cases were also reported in which TE insertions had only a modest impact on neighboring genes in standard conditions, but significant effects under stress conditions (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Naito <em>et al.</em> 2009</a>).</p></div><div id="sec-20" class="subsection"><h3>Conclusion</h3><p id="p-53">In the present study, we use natural variability regarding the <em>tirant</em> ERV of <em>D. simulans</em> to identify potential impacts of TEs on the host genome. We show that, to a moderate extent, a <em>tirant</em> insertion may spread heterochromatin to flanking regions where genes lie, and be associated with transcript level reduction for the gene, with potential phenotypic impacts. Our results also illustrate that TE effects on the genome may not be as pervasive as recently proposed (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lee and Karpen 2017</a>) since we detected them only for one insertion and not for the others. In addition, our data suggest the existence of a dynamic relationship between TE transcripts and the piRNAs that control them. This is illustrated by a positive correlation between these two variables in normal conditions, which is reverted to a negative correlation in the case of dysgenic crosses. We speculate this may happen when asymmetry is too high between ping-pong partners abundances, –<em>ie</em> maternally transmitted piRNAs on one hand and TE transcripts on the other hand, for the piRNA pathway to be efficient.</p><p id="p-54">Taken together, our results participate in the understanding of the equilibrium between the host genome and its TEs. This study opens the way to the investigation of the transition from the equilibrium disruption to a restored balance between the genome and its TEs, which are fundamental aspects of the understanding of TE biology and genome stability.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">This work was performed using the computing facilities of the CC LBBE/PRABI, the luminometer of the IUT Lyon 1 Biology Department, and the Vienna Biocenter Campus Facility.</p><p id="p-56">We thank Justine Picarle, Sonia Martinez, Nabila Berabez and Anaïs Alquezar for technical assistance. We thank Philippe Veber, Matthieu Boulesteix, and attendees of the XX<sup>th</sup> French National Congress on Transposable Elements for fruitful discussions. We thank Labex ECOFECT.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-57">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7578380">https://doi.org/10.25387/g3.7578380</a>.</p></li><li class="fn" id="fn-3"><p id="p-58"><em>Communicating Editor: J. Birchler</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-10"><span class="received-label">Received </span>October 10, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-14"><span class="accepted-label">Accepted </span>January 14, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Fablet <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.1" data-doi="10.1038/embor.2013.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akkouche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grentzinger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armenise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Maternally Deposited Germline PiRNAs Silence the Tirant Retrotransposon in Somatic Cells.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/embor.2013.38</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft.volume%253D14%26rft.spage%253D458%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2013.38%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/embor.2013.38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.2" data-doi="10.1128/JVI.07146-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akkouche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebollo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esnault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tirant, a Newly Discovered Active Endogenous Retrovirus in Drosophila Simulans.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">3675</span>–<span class="cit-lpage">3681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JVI.07146-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.07146-11%26rft_id%253Dinfo%253Apmid%252F22278247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzcvMzY3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.3" data-doi="10.1007/978-1-4939-0931-5_12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antoniewski</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Computing SiRNA and PiRNA Overlap Signatures.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1173</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-0931-5_12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1173%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-0931-5_12%26rft_id%253Dinfo%253Apmid%252F24920366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-0931-5_12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24920366&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.4" data-doi="10.1126/science.1146484"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannon</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Piwi-PiRNA Pathway Provides an Adaptive Defense in the Transposon Arms Race.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1146484</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAravin%26rft.auinit1%253DA.%2BA.%26rft.volume%253D318%26rft.issue%253D5851%26rft.spage%253D761%26rft.epage%253D764%26rft.atitle%253DThe%2BPiwi-piRNA%2BPathway%2BProvides%2Ban%2BAdaptive%2BDefense%2Bin%2Bthe%2BTransposon%2BArms%2BRace%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1146484%26rft_id%253Dinfo%253Apmid%252F17975059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg1MS83NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.5" data-doi="10.1038/443521a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biémont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetics: Junk DNA as an Evolutionary Force.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/443521a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBiemont%26rft.auinit1%253DC.%26rft.volume%253D443%26rft.issue%253D7111%26rft.spage%253D521%26rft.epage%253D524%26rft.atitle%253DGenetics%253A%2Bjunk%2BDNA%2Bas%2Ban%2Bevolutionary%2Bforce.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F443521a%26rft_id%253Dinfo%253Apmid%252F17024082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/443521a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17024082&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240988200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boisvert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laviolette</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ray: Simultaneous Assembly of Reads from a Mix of High-Throughput Sequencing Technologies.</span> <abbr class="cit-jnl-abbrev">J. Comput. Mol. Cell Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BMol.%2BCell%2BBiol.%26rft.volume%253D17%26rft.spage%253D1519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.7" data-doi="10.1016/j.cell.2007.01.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discrete Small RNA-Generating Loci as Master Regulators of Transposon Activity in Drosophila.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2007.01.043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIwase%26rft.auinit1%253DS.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D1089%26rft.epage%253D1103%26rft.atitle%253DDiscrete%2Bsmall%2BRNA-generating%2Bloci%2Bas%2Bmaster%2Bregulators%2Bof%2Btransposon%2Bactivity%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.01.043%26rft_id%253Dinfo%253Apmid%252F17346786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.01.043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17346786&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245396200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.8" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avagyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papadopoulos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: Architecture and Applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.9" data-doi="10.1093/bioinformatics/btq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Böhme</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhill</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berriman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BamView: Viewing Mapped Read Alignment Data in the Context of the Reference Sequence.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq010%26rft_id%253Dinfo%253Apmid%252F20071372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20071372&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.10" data-doi="10.1111/mec.12170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casacuberta</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Impact of Transposable Elements in Environmental Adaptation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1517</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D1503%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12170%26rft_id%253Dinfo%253Apmid%252F23293987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23293987&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316209500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.11" data-doi="10.1073/pnas.0805943105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chambeyron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen-Groschêne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brun</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laouini</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">PiRNA-Mediated Nuclear Accumulation of Retrotransposon Transcripts in the Drosophila Female Germline.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">14964</span>–<span class="cit-lpage">14969</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0805943105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0805943105%26rft_id%253Dinfo%253Apmid%252F18809914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzM5LzE0OTY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.12" data-doi="10.1038/nrg2640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordaux</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzer</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Impact of Retrotransposons on Human Genome Evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCordaux%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D691%26rft.epage%253D703%26rft.atitle%253DThe%2Bimpact%2Bof%2Bretrotransposons%2Bon%2Bhuman%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2640%26rft_id%253Dinfo%253Apmid%252F19763152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19763152&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.13" data-doi="10.1126/science.1074170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daborn</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogwitz</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Goff</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A Single P450 Allele Associated with Insecticide Resistance in Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">2253</span>–<span class="cit-lpage">2256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1074170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDaborn%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D297%26rft.issue%253D5590%26rft.spage%253D2253%26rft.epage%253D2256%26rft.atitle%253DA%2BSingle%2BP450%2BAllele%2BAssociated%2Bwith%2BInsecticide%2BResistance%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1074170%26rft_id%253Dinfo%253Apmid%252F12351787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU5MC8yMjUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.14" data-doi="10.1098/rstb.2014.0331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">What’s in a Genome? The C–Value Enigma and the Evolution of Eukaryotic Genome Content.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">370</span>: <span class="cit-fpage">20140331</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2014.0331</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2014.0331%26rft_id%253Dinfo%253Apmid%252F26323762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2014.0331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26323762&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebollo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic Environment Influences the Dynamics of the Tirant LTR Retrotransposon in Drosophila.</span> <abbr class="cit-jnl-abbrev">FASEB J. Off. Publ. Fed. Am. Soc. Exp. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1482</span>–<span class="cit-lpage">1489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ.%2BOff.%2BPubl.%2BFed.%2BAm.%2BSoc.%2BExp.%2BBiol.%26rft.volume%253D23%26rft.spage%253D1482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.16" data-doi="10.1016/j.gene.2006.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biémont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Ongoing Loss of the Tirant Transposable Element in Natural Populations of Drosophila Simulans.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">375</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2006.02.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DFablet%26rft.auinit1%253DM.%26rft.volume%253D375%26rft.spage%253D54%26rft.epage%253D62%26rft.atitle%253DOngoing%2Bloss%2Bof%2Bthe%2Btirant%2Btransposable%2Belement%2Bin%2Bnatural%2Bpopulations%2Bof%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2006.02.008%26rft_id%253Dinfo%253Apmid%252F16626897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2006.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16626897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.855.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salces-Ortiz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraga Menezes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Self and nonself from a genomic perspective: transposable elements</span>, pp. <span class="cit-fpage">111</span>–<span class="cit-lpage">128</span> in <span class="cit-source"><em>Evolutionary Biology: Self/Nonself Evolution</em>, <em>Species and Complex Traits Evolution</em>, <em>Methods and Concepts</em></span>, edited by <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Cham</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.18" data-doi="10.1007/s00239-006-0108-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souames</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biémont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolutionary Pathways of the Tirant LTR Retrotransposon in the Drosophila Melanogaster Subgroup of Species.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-006-0108-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFablet%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D438%26rft.epage%253D447%26rft.atitle%253DEvolutionary%2Bpathways%2Bof%2Bthe%2Btirant%2BLTR%2Bretrotransposon%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bsubgroup%2Bof%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-006-0108-9%26rft_id%253Dinfo%253Apmid%252F17390093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-006-0108-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17390093&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.19" data-doi="10.1016/0092-8674(90)90213-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modolell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The Drosophila extramacrochaetae locus, an antagonist of proneural genes that, like these genes, encodes a helix-loop-helix protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(90)90213-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGarrell%26rft.auinit1%253DJ.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D48%26rft.atitle%253DThe%2BDrosophila%2Bextramacrochaetae%2Blocus%252C%2Ban%2Bantagonist%2Bof%2Bproneural%2Bgenes%2Bthat%252C%2Blike%2Bthese%2Bgenes%252C%2Bencodes%2Ba%2Bhelix-loop-helix%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252890%252990213-X%26rft_id%253Dinfo%253Apmid%252F1690605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(90)90213-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1690605&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CY38200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.20" data-doi="10.1371/journal.pcbi.1006360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrido-Martín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guigó</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breschi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">ggsashimi: Sashimi plot revised for browser- and annotation-independent splicing visualization.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1006360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006360</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1006360%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.21" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-Length Transcriptome Assembly from RNA-Seq Data without a Reference Genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.22" data-doi="10.1186/s13100-016-0065-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancks</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Roles for Retrotransposon Insertions in Human Disease.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13100-016-0065-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D7%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-016-0065-9%26rft_id%253Dinfo%253Apmid%252F27158268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-016-0065-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27158268&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.23" data-doi="10.1016/j.tig.2017.07.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jangam</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betrán</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transposable Element Domestication As an Adaptation to Evolutionary Conflicts.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.07.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D33%26rft.spage%253D817%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.07.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.24" data-doi="10.1016/j.ympev.2012.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walden</span>  <span class="cit-name-given-names">K. K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Next-Generation Phylogenomics Using a Target Restricted Assembly Method.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2012.09.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D66%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.09.007%26rft_id%253Dinfo%253Apmid%252F23000819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23000819&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.25" data-doi="10.1093/molbev/mst081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbash</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Analysis of PiRNA-Mediated Silencing of Active TEs in Drosophila Melanogaster Suggests Limits on the Evolution of Host Genome Defense.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1816</span>–<span class="cit-lpage">1829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst081</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst081%26rft_id%253Dinfo%253Apmid%252F23625890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23625890&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321820400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.26" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT–the BLAST-like Alignment Tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.27" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-4-r36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.28" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast Gapped-Read Alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.29" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and Memory-Efficient Alignment of Short DNA Sequences to the Human Genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2009-10-3-r25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.30" data-doi="10.1093/bioinformatics/btq614"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashizaki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SAMStat: Monitoring Biases in next Generation Sequencing Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq614</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq614%26rft_id%253Dinfo%253Apmid%252F21088025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21088025&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285626300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.31" data-doi="10.1101/gad.209841.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Thomas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinov</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Piwi Induces PiRNA-Guided Transcriptional Silencing and Establishment of a Repressive Chromatin State.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.209841.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.209841.112%26rft_id%253Dinfo%253Apmid%252F23392610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjcvNC8zOTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpen</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pervasive epigenetic effects of Drosophila Euchromatic transposable elements impact their evolution.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>: e25762.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fablet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Modolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Maestre</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TEtools Facilitates Big Data Expression Analysis of Transposable Elements and Reveals an Antagonism between Their Activity and That of PiRNA Genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">e17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.34" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map Format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.855.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimm</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">The Genome of Drosophila Melanogaster</span>, <span class="cit-publ-name">Academic Press, Inc., San Diego</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.36" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated Estimation of Fold Change and Dispersion for RNA-Seq Data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.37" data-doi="10.1016/S0378-1119(00)00115-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsano</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moschetti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caggese</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanave</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barsanti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The Complete Tirant Transposable Element in Drosophila Melanogaster Shows a Structural Relationship with Retrovirus-like Retrotransposons.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">247</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(00)00115-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMarsano%26rft.auinit1%253DR.%2BM.%26rft.volume%253D247%26rft.issue%253D1-2%26rft.spage%253D87%26rft.epage%253D95%26rft.atitle%253DThe%2Bcomplete%2BTirant%2Btransposable%2Belement%2Bin%2BDrosophila%2Bmelanogaster%2Bshows%2Ba%2Bstructural%2Brelationship%2Bwith%2Bretrovirus-like%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900115-3%26rft_id%253Dinfo%253Apmid%252F10773447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00115-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10773447&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086689100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.855.38" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><cite>Martin, M., 2011 Cutadapt Removes Adapter Sequences from High-Throughput Sequencing Reads. EMBnet.journal 17: 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.39" data-doi="10.1038/35001608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veldman</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finnerty</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Syncytin Is a Captive Retroviral Envelope Protein Involved in Human Placental Morphogenesis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">403</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35001608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMi%26rft.auinit1%253DS.%26rft.volume%253D403%26rft.issue%253D6771%26rft.spage%253D785%26rft.epage%253D789%26rft.atitle%253DSyncytin%2Bis%2Ba%2Bcaptive%2Bretroviral%2Benvelope%2Bprotein%2Binvolved%2Bin%2Bhuman%2Bplacental%2Bmorphogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35001608%26rft_id%253Dinfo%253Apmid%252F10693809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35001608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10693809&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085423100056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.40" data-doi="10.1186/s12859-015-0546-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Modolo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">UrQt: An Efficient Software for the Unsupervised Quality Trimming of NGS Data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0546-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0546-8%26rft_id%253Dinfo%253Apmid%252F25924884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0546-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25924884&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.855.41"><div class="cit-metadata"><cite>Modolo, L., 2018 <em>l-modolo/fastq_sampler: everything is working</em>. Zenodo.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5281/zenodo.1227899</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.42" data-doi="10.1016/j.cell.2014.04.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sienski</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Rhino-Deadlock-Cutoff Complex Licenses Noncanonical Transcription of Dual-Strand PiRNA Clusters in Drosophila.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1364</span>–<span class="cit-lpage">1379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.04.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1364%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.04.031%26rft_id%253Dinfo%253Apmid%252F24906153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.04.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24906153&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340881400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.43" data-doi="10.1038/nature08479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naito</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukiyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Unexpected Consequences of a Sudden and Massive Transposon Amplification on Rice Gene Expression.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">1130</span>–<span class="cit-lpage">1134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNaito%26rft.auinit1%253DK.%26rft.volume%253D461%26rft.issue%253D7267%26rft.spage%253D1130%26rft.epage%253D1134%26rft.atitle%253DUnexpected%2Bconsequences%2Bof%2Ba%2Bsudden%2Band%2Bmassive%2Btransposon%2Bamplification%2Bon%2Brice%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08479%26rft_id%253Dinfo%253Apmid%252F19847266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19847266&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270987600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.44" data-doi="10.1371/journal.pgen.1002301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebollo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karimi</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bilenky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miceli-Royer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Retrotransposon-Induced Heterochromatin Spreading in the Mouse Revealed by Insertional Polymorphisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRebollo%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002301%26rft.epage%253De1002301%26rft.atitle%253DRetrotransposon-induced%2Bheterochromatin%2Bspreading%2Bin%2Bthe%2Bmouse%2Brevealed%2Bby%2Binsertional%2Bpolymorphisms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002301%26rft_id%253Dinfo%253Apmid%252F21980304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21980304&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.855.45"><div class="cit-metadata"><cite>Rey, C., T. Lorin, M. Sémon, and B. Boussau, 2017 <em>apytram v1.1</em>. Zenodo.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5281/zenodo.804416</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.46" data-doi="10.1101/gad.1454806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyoshi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Specific Association of Piwi with RasiRNAs Derived from Retrotransposon and Heterochromatic Regions in the Drosophila Genome.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2214</span>–<span class="cit-lpage">2222</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1454806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1454806%26rft_id%253Dinfo%253Apmid%252F16882972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIwLzE2LzIyMTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.47" data-doi="10.1093/bioinformatics/btr026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmieder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quality Control and Preprocessing of Metagenomic Datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">864</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr026%26rft_id%253Dinfo%253Apmid%252F21278185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21278185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.48" data-doi="10.1101/gad.267252.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senti</span>  <span class="cit-name-given-names">K.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurczak</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachidanandam</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">piRNA-guided slicing of transposon transcripts enforces their transcriptional silencing via specifying the nuclear piRNA repertoire.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1747</span>–<span class="cit-lpage">1762</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.267252.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.267252.115%26rft_id%253Dinfo%253Apmid%252F26302790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI5LzE2LzE3NDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.49" data-doi="10.1371/journal.pgen.1004138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shpiz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryazansky</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olovnikov</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abramov</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalmykova</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Euchromatic Transposon Insertions Trigger Production of Novel Pi- and Endo-SiRNAs at the Target Sites in the Drosophila Germline.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004138%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004138%26rft_id%253Dinfo%253Apmid%252F24516406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24516406&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.50" data-doi="10.1016/j.cell.2012.10.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sienski</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dönertas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptional Silencing of Transposons by Piwi and Maelstrom and Its Impact on Chromatin State and Gene Expression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">964</span>–<span class="cit-lpage">980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2012.10.040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSienski%26rft.auinit1%253DG.%26rft.volume%253D151%26rft.issue%253D5%26rft.spage%253D964%26rft.epage%253D980%26rft.atitle%253DTranscriptional%2Bsilencing%2Bof%2Btransposons%2Bby%2Bpiwi%2Band%2Bmaelstrom%2Band%2Bits%2Bimpact%2Bon%2Bchromatin%2Bstate%2Band%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.10.040%26rft_id%253Dinfo%253Apmid%252F23159368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.10.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159368&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311423500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.51" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: Assessing Genome Assembly and Annotation Completeness with Single-Copy Orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.52" data-doi="10.1038/nrm3089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siomi</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pezic</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PIWI-Interacting Small RNAs: The Vanguard of Genome Defence.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrm3089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DSiomi%26rft.auinit1%253DM.%2BC.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D246%26rft.epage%253D258%26rft.atitle%253DPIWI-interacting%2Bsmall%2BRNAs%253A%2Bthe%2Bvanguard%2Bof%2Bgenome%2Bdefence.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3089%26rft_id%253Dinfo%253Apmid%252F21427766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21427766&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.53" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-6-31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.54" data-doi="10.1038/nrg2072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slotkin</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martienssen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transposable Elements and the Epigenetic Regulation of the Genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">272</span>–<span class="cit-lpage">285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSlotkin%26rft.auinit1%253DR.%2BK.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D272%26rft.epage%253D285%26rft.atitle%253DTransposable%2Belements%2Band%2Bthe%2Bepigenetic%2Bregulation%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2072%26rft_id%253Dinfo%253Apmid%252F17363976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17363976&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245036100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-9.3.855.55"><div class="cit-metadata unstructured">Smit, A. F., R. Hubley, and P. Green, 2013 RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.56" data-doi="10.1038/nature25018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuniewska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coux</span>  <span class="cit-name-given-names">R.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rio</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">PiRNA-Mediated Regulation of Transposon Alternative Splicing in the Soma and Germ Line.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">552</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D552%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.57" data-doi="10.1016/j.tplants.2010.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollister</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A Triptych of the Evolution of Plant Transposable Elements.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2010.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DTenaillon%26rft.auinit1%253DM.%2BI.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D471%26rft.epage%253D478%26rft.atitle%253DA%2Btriptych%2Bof%2Bthe%2Bevolution%2Bof%2Bplant%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.05.003%26rft_id%253Dinfo%253Apmid%252F20541961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20541961&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281176900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.58" data-doi="10.1186/1471-2148-1-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terzian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pélisson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucheton</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evolution and Phylogeny of Insect Endogenous Retroviruses.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-1-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DTerzian%26rft.auinit1%253DC.%26rft.volume%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DEvolution%2Band%2Bphylogeny%2Bof%2Binsect%2Bendogenous%2Bretroviruses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-1-3%26rft_id%253Dinfo%253Apmid%252F11591216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-1-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11591216&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.59" data-doi="10.1093/oxfordjournals.molbev.a026215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepetit</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biémont</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Wake up of Transposable Elements Following Drosophila Simulans Worldwide Colonization.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a026215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DVieira%26rft.auinit1%253DC%26rft.volume%253D16%26rft.issue%253D9%26rft.spage%253D1251%26rft.epage%253D1255%26rft.atitle%253DWake%2Bup%2Bof%2Btransposable%2Belements%2Bfollowing%2BDrosophila%2Bsimulans%2Bworldwide%2Bcolonization%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026215%26rft_id%253Dinfo%253Apmid%252F10486980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10486980&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082431300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.855.60" data-doi="10.1038/nrg2165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua-Van</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A Unified Classification System for Eukaryotic Transposable Elements.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F855.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/855.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/841" title="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/867" title="Identification of Suppressor of Clathrin Deficiency-1 (SCD1) and Its Connection to Clathrin-Mediated Endocytosis in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/855" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dgia0r066pf3847aktJbUcA9CccejP3s9G4DLsRPpl8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/855" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/855" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/855" class="active" data-icon-position="" data-hide-link-title="0">Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492323" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-58ah3hVzsd9WqHAJF_hn2ox_3bi_51JYvf1sZMZBvgE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/855.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/855" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-w26lS4sH7rC9WvPE43MFXn7mEyCFp2HF7UpB9v5b8PA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/855" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/855.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492323" id="node492323--2" data-pisa="ggg;9/3/855" data-pisa-master="ggg;g3.118.200789" data-apath="/ggg/9/3/855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamic Interactions Between the Genome and an Endogenous Retrovirus: <em>Tirant</em> in <em>Drosophila simulans</em> Wild-Type Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marie Fablet</span>, <span class="highwire-citation-author" data-delta="1">Angelo Jacquet</span>, <span class="highwire-citation-author" data-delta="2">Rita Rebollo</span>, <span class="highwire-citation-author" data-delta="3">Annabelle Haudry</span>, <span class="highwire-citation-author" data-delta="4">Carine Rey</span>, <span class="highwire-citation-author" data-delta="5">Judit Salces-Ortiz</span>, <span class="highwire-citation-author" data-delta="6">Prajakta Bajad</span>, <span class="highwire-citation-author" data-delta="7">Nelly Burlet</span>, <span class="highwire-citation-author" data-delta="8">Michael F. Jantsch</span>, <span class="highwire-citation-author" data-delta="9">Maria Pilar García Guerreiro</span> and <span class="highwire-citation-author" data-delta="10">Cristina Vieira</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">855-865;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200789 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492323--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie Fablet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BFablet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fablet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarie%2BFablet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marie Fablet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marie.fablet@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">marie.fablet@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angelo Jacquet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angelo%2BJacquet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquet%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngelo%2BJacquet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita Rebollo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BF2i, Univ Lyon, INSA-Lyon, INRA, UMR 0203, Villeurbanne 69621, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BRebollo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rebollo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BRebollo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annabelle Haudry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabelle%2BHaudry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haudry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnabelle%2BHaudry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carine Rey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire de Biologie et Modélisation de la Cellule, UnivLyon, ENS de Lyon, Univ Claude Bernard, CNRS UMR 5239, INSERM U1210, Lyon 69007, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BRey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BRey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Judit Salces-Ortiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judit%2BSalces-Ortiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salces-Ortiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudit%2BSalces-Ortiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prajakta Bajad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prajakta%2BBajad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bajad%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrajakta%2BBajad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nelly Burlet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nelly%2BBurlet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burlet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANelly%2BBurlet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael F. Jantsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BJantsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jantsch%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BJantsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Maria Pilar García Guerreiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grup de Genòmica, Bioinformàtica i Biologia Evolutiva, Departament de Genètica i Microbiologia, Universitat Autònoma de Barcelona, Bellaterra-Barcelona 08193, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BPilar%2BGarc%C3%ADa%2BGuerreiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guerreiro%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BPilar%2BGarc%25C3%25ADa%2BGuerreiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Cristina Vieira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cristina%2BVieira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vieira%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACristina%2BVieira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/855" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/855&amp;text=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/855&amp;t=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/855&amp;title=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492323/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492323" id="node492323--3" data-pisa="ggg;9/3/855" data-pisa-master="ggg;g3.118.200789" data-apath="/ggg/9/3/855.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dynamic Interactions Between the Genome and an Endogenous Retrovirus: <em>Tirant</em> in <em>Drosophila simulans</em> Wild-Type Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marie Fablet</span>, <span class="highwire-citation-author" data-delta="1">Angelo Jacquet</span>, <span class="highwire-citation-author" data-delta="2">Rita Rebollo</span>, <span class="highwire-citation-author" data-delta="3">Annabelle Haudry</span>, <span class="highwire-citation-author" data-delta="4">Carine Rey</span>, <span class="highwire-citation-author" data-delta="5">Judit Salces-Ortiz</span>, <span class="highwire-citation-author" data-delta="6">Prajakta Bajad</span>, <span class="highwire-citation-author" data-delta="7">Nelly Burlet</span>, <span class="highwire-citation-author" data-delta="8">Michael F. Jantsch</span>, <span class="highwire-citation-author" data-delta="9">Maria Pilar García Guerreiro</span> and <span class="highwire-citation-author" data-delta="10">Cristina Vieira</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">855-865;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200789 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492323--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie Fablet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BFablet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fablet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarie%2BFablet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7819-4541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marie Fablet</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marie.fablet@univ-lyon1.fr" class="" data-icon-position="" data-hide-link-title="0">marie.fablet@univ-lyon1.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Angelo Jacquet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angelo%2BJacquet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jacquet%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngelo%2BJacquet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rita Rebollo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>BF2i, Univ Lyon, INSA-Lyon, INRA, UMR 0203, Villeurbanne 69621, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rita%2BRebollo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rebollo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARita%2BRebollo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Annabelle Haudry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Annabelle%2BHaudry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haudry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnnabelle%2BHaudry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Carine Rey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire de Biologie et Modélisation de la Cellule, UnivLyon, ENS de Lyon, Univ Claude Bernard, CNRS UMR 5239, INSERM U1210, Lyon 69007, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carine%2BRey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rey%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarine%2BRey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Judit Salces-Ortiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Judit%2BSalces-Ortiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salces-Ortiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJudit%2BSalces-Ortiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prajakta Bajad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prajakta%2BBajad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bajad%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrajakta%2BBajad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Nelly Burlet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nelly%2BBurlet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burlet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANelly%2BBurlet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael F. Jantsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Center for Anatomy and Cell Biology, Division of Cell- and Developmental Biology, Medical University of Vienna, Vienna 1090, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BJantsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jantsch%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BF.%2BJantsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Maria Pilar García Guerreiro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Grup de Genòmica, Bioinformàtica i Biologia Evolutiva, Departament de Genètica i Microbiologia, Universitat Autònoma de Barcelona, Bellaterra-Barcelona 08193, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BPilar%2BGarc%C3%ADa%2BGuerreiro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guerreiro%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BPilar%2BGarc%25C3%25ADa%2BGuerreiro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Cristina Vieira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon; Université Lyon 1; CNRS; UMR 5558, Villeurbanne 69622, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cristina%2BVieira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vieira%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACristina%2BVieira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=855&amp;rft.epage=865&amp;rft.atitle=Dynamic%20Interactions%20Between%20the%20Genome%20and%20an%20Endogenous%20Retrovirus%3A%20Tirant%20in%20Drosophila%20simulans%20Wild-Type%20Strains&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fablet%2C+Marie&amp;rft.au=Jacquet%2C+Angelo&amp;rft.au=Rebollo%2C+Rita&amp;rft.au=Haudry%2C+Annabelle&amp;rft.au=Rey%2C+Carine&amp;rft.au=Salces-Ortiz%2C+Judit&amp;rft.au=Bajad%2C+Prajakta&amp;rft.au=Burlet%2C+Nelly&amp;rft.au=Jantsch%2C+Michael+F.&amp;rft.au=Guerreiro%2C+Maria+Pilar+Garc%C3%ADa&amp;rft.au=Vieira%2C+Cristina"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492323/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492323/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492323/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492323/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492323/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492323/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492323/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492323/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492323/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492323/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492323/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492323/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/855" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/855.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/855.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/855.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/855.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
