<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/755" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/755.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/755.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/755.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/755.atom" />
<meta name="HW.pisa" content="ggg;9/3/755" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development" />
<meta name="DC.Identifier" content="10.1534/g3.118.201008" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Minio et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transcriptomics has been widely applied to study grape berry development. With few exceptions, transcriptomic studies in grape are performed using the available genome sequence, PN40024, as reference. However, differences in gene content among grape accessions, which contribute to phenotypic differences among cultivars, suggest that a single reference genome does not represent the species’ entire gene space. Though whole genome assembly and annotation can reveal the relatively unique or “private” gene space of any particular cultivar, transcriptome reconstruction is a more rapid, less costly, and less computationally intensive strategy to accomplish the same goal. In this study, we used single molecule-real time sequencing (SMRT) to sequence full-length cDNA (Iso-Seq) and reconstruct the transcriptome of Cabernet Sauvignon berries during berry ripening. In addition, short reads from ripening berries were used to error-correct low-expression isoforms and to profile isoform expression. By comparing the annotated gene space of Cabernet Sauvignon to other grape cultivars, we demonstrate that the transcriptome reference built with Iso-Seq data represents most of the expressed genes in the grape berries and includes 1,501 cultivar-specific genes. Iso-Seq produced transcriptome profiles similar to those obtained after mapping on a complete genome reference. Together, these results justify the application of Iso-Seq to identify cultivar-specific genes and build a comprehensive reference for transcriptional profiling that circumvents the necessity of a genome reference with its associated costs and computational weight." />
<meta name="DC.Contributor" content="Andrea Minio" />
<meta name="DC.Contributor" content="Mélanie Massonnet" />
<meta name="DC.Contributor" content="Rosa Figueroa-Balderas" />
<meta name="DC.Contributor" content="Amanda M. Vondras" />
<meta name="DC.Contributor" content="Barbara Blanco-Ulate" />
<meta name="DC.Contributor" content="Dario Cantu" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transcriptomics has been widely applied to study grape berry development. With few exceptions, transcriptomic studies in grape are performed using the available genome sequence, PN40024, as reference. However, differences in gene content among grape accessions, which contribute to phenotypic differences among cultivars, suggest that a single reference genome does not represent the species’ entire gene space. Though whole genome assembly and annotation can reveal the relatively unique or “private” gene space of any particular cultivar, transcriptome reconstruction is a more rapid, less costly, and less computationally intensive strategy to accomplish the same goal. In this study, we used single molecule-real time sequencing (SMRT) to sequence full-length cDNA (Iso-Seq) and reconstruct the transcriptome of Cabernet Sauvignon berries during berry ripening. In addition, short reads from ripening berries were used to error-correct low-expression isoforms and to profile isoform expression. By comparing the annotated gene space of Cabernet Sauvignon to other grape cultivars, we demonstrate that the transcriptome reference built with Iso-Seq data represents most of the expressed genes in the grape berries and includes 1,501 cultivar-specific genes. Iso-Seq produced transcriptome profiles similar to those obtained after mapping on a complete genome reference. Together, these results justify the application of Iso-Seq to identify cultivar-specific genes and build a comprehensive reference for transcriptional profiling that circumvents the necessity of a genome reference with its associated costs and computational weight.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/755" />
<meta name="citation_id" content="9/3/755" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/755" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/755.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/755.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/755.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.201008" />
<meta name="citation_pmid" content="30642874" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="755" />
<meta name="citation_lastpage" content="767" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrea Minio" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California Davis, Davis, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2643-9209" />
<meta name="citation_author" content="Mélanie Massonnet" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California Davis, Davis, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0993-9148" />
<meta name="citation_author" content="Rosa Figueroa-Balderas" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California Davis, Davis, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3321-1376" />
<meta name="citation_author" content="Amanda M. Vondras" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California Davis, Davis, CA" />
<meta name="citation_author" content="Barbara Blanco-Ulate" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California Davis, Davis, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8819-9207" />
<meta name="citation_author" content="Dario Cantu" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California Davis, Davis, CA" />
<meta name="citation_author_email" content="dacantu@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4858-1508" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Afoufa-Bastien;citation_author=A. Medici;citation_author=J. Jeauffre;citation_author=P. Coutos-Thévenot;citation_author=R. Lemoine;citation_title=The Vitis vinifera sugar transporter gene family: phylogenetic overview and macroarray expression profiling.;citation_pages=245-245;citation_volume=10;citation_year=2010;citation_pmid=21073695;citation_doi=10.1186/1471-2229-10-245" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=K. C. H. Amrine;citation_author=B. Blanco-Ulate;citation_author=S. Riaz;citation_author=D. Pap;citation_author=L. Jones;citation_title=Comparative transcriptomics of Central Asian Vitis vinifera accessions reveals distinct defense strategies against powdery mildew.;citation_pages=15037;citation_volume=2;citation_year=2015;citation_doi=10.1038/hortres.2015.37" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Ashrafi;citation_author=T. Hill;citation_author=K. Stoffel;citation_author=A. Kozik;citation_author=J. Yao;citation_title=De novo assembly of the pepper transcriptome (Capsicum annuum): a benchmark for in silico discovery of SNPs, SSRs and candidate genes.;citation_pages=571-571;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23110314;citation_doi=10.1186/1471-2164-13-571" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. F. Au;citation_author=J. G. Underwood;citation_author=L. Lee;citation_author=W. H. Wong;citation_title=Improving PacBio Long Read Accuracy by Short Read Alignment.;citation_pages=e46679-e46679;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056399;citation_doi=10.1371/journal.pone.0046679" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=E. Bellucci;citation_author=E. Bitocchi;citation_author=A. Ferrarini;citation_author=A. Benazzo;citation_author=E. Biagetti;citation_title=Decreased Nucleotide and Expression Diversity and Modified Coexpression Patterns Characterize Domestication in the Common Bean.;citation_volume=26;citation_year=2014;citation_pmid=24850850;citation_doi=10.1105/tpc.114.124040" />
<meta name="citation_reference" content="citation_author=B. Blanco-Ulate;citation_author=K. C. Amrine;citation_author=T. S. Collins;citation_author=R. M. Rivero;citation_author=A. R. Vicente;citation_pmid=26450706;citation_doi=10.1104/pp.15.00852" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=B. Blanco-Ulate;citation_author=H. Hopfer;citation_author=R. Figueroa-Balderas;citation_author=Z. Ye;citation_author=R. M. Rivero;citation_title=Red blotch disease alters grape berry development and metabolism by interfering with the transcriptional and hormonal regulation of ripening.;citation_pages=1225-1238;citation_volume=68;citation_year=2017;citation_doi=10.1093/jxb/erw506" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=B. Blanco-Ulate;citation_author=E. Vincenti;citation_author=A. L. T. Powell;citation_author=D. Cantu;citation_title=Tomato transcriptome and mutant analyses suggest a role for plant stress hormones in the interaction between fruit and Botrytis cinerea.;citation_pages=1-16;citation_volume=4;citation_year=2013;citation_pmid=23346092;citation_doi=10.3389/fpls.2013.00366" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. E. Bowers;citation_author=C. P. Meredith;citation_title=The parentage of a classic wine grape, Cabernet Sauvignon.;citation_pages=84-87;citation_volume=16;citation_year=1997;citation_issue=1;citation_pmid=9140400;citation_doi=10.1038/ng0597-84" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. R. Brent;citation_title=Steady progress and recent breakthroughs in the accuracy of automated genome annotation.;citation_pages=62-73;citation_volume=9;citation_year=2008;citation_issue=1;citation_pmid=18087260;citation_doi=10.1038/nrg2220" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Brett;citation_author=H. Pospisil;citation_author=J. Valcárcel;citation_author=J. Reich;citation_author=P. Bork;citation_title=Alternative splicing and genome complexity.;citation_pages=29-30;citation_volume=30;citation_year=2002;citation_issue=1;citation_pmid=11743582;citation_doi=10.1038/ng803" />
<meta name="citation_reference" content="citation_title=Current Protocols in Bioinformatics;citation_title=Genome Annotation and Curation Using MAKER and MAKER-P;citation_author=M. S. Campbell;citation_author=C. Holt;citation_author=B. Moore;citation_author=M. Yandell;citation_pages=4.11.1-4.11.39;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. S. Campbell;citation_author=M. Law;citation_author=C. Holt;citation_author=J. C. Stein;citation_author=G. D. Moghe;citation_title=MAKER-P: A tool kit for the rapid creation, management, and quality control of plant genome annotations.;citation_volume=164;citation_year=2014;citation_pmid=24306534;citation_doi=10.1104/pp.113.230144" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. T. Carey;citation_author=K. Holt;citation_author=S. Picard;citation_author=R. Wilde;citation_author=G. A. Tucker;citation_title=Tomato Exo-(1-&gt;4)-[beta]-D-Galactanase (Isolation, Changes during Ripening in Normal and Mutant Tomato Fruit, and Characterization of a Related cDNA Clone);citation_pages=1099-1107;citation_volume=108;citation_year=1995;citation_issue=3;citation_pmid=7630937;citation_doi=10.1104/pp.108.3.1099" />
<meta name="citation_reference" content="citation_author=S. D. Castellarin;citation_author=L. Bavaresco;citation_author=L. Falginella;citation_author=M. I. V. Z. Gonçalves;citation_author=G. Di Gaspero;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Chang;citation_author=Z. Wang;citation_author=G. Li;citation_title=The impacts of read length and transcriptome complexity for de ovo assembly: A simulation study.;citation_pages=1-8;citation_volume=9;citation_year=2014;citation_pmid=24551193;citation_doi=10.1371/journal.pone.0089680" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C.-S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C.-S. Chin;citation_author=P. Peluso;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=G. T. Concepcion;citation_title=Phased diploid genome assembly with single-molecule real-time sequencing.;citation_pages=1050-1054;citation_volume=13;citation_year=2016;citation_pmid=27749838;citation_doi=10.1038/nmeth.4035" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Cipriani;citation_author=A. Spadotto;citation_author=I. Jurman;citation_author=G. Di Gaspero;citation_author=M. Crespan;citation_title=The SSR-based molecular profile of 1005 grapevine (Vitis vinifera L.) accessions uncovers new synonymy and parentages, and reveals a large admixture amongst varieties of different geographic origin.;citation_pages=1569-1585;citation_volume=121;citation_year=2010;citation_issue=8;citation_pmid=20689905;citation_doi=10.1007/s00122-010-1411-9" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=M. Corso;citation_author=A. Vannozzi;citation_author=E. Maza;citation_author=N. Vitulo;citation_author=F. Meggio;citation_title=Comprehensive transcript profiling of two grapevine rootstock genotypes contrasting in drought susceptibility links the phenylpropanoid pathway to enhanced tolerance.;citation_volume=66;citation_year=2015;citation_pmid=26038306;citation_doi=10.1093/jxb/erv274" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=D. J. Cosgrove;citation_title=Growth of the plant cell wall.;citation_pages=850-861;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16261190;citation_doi=10.1038/nrm1746" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Cosgrove;citation_title=Loosening of plant cell walls by expansins.;citation_pages=321-326;citation_volume=407;citation_year=2000;citation_issue=6802;citation_pmid=11014181;citation_doi=10.1038/35030000" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. Da Silva;citation_author=G. Zamperin;citation_author=A. Ferrarini;citation_author=A. Minio;citation_author=A. Dal Molin;citation_title=The high polyphenol content of grapevine cultivar tannat berries is conferred primarily by genes that are not shared with the reference genome.;citation_volume=25;citation_year=2013;citation_pmid=24319081;citation_doi=10.1105/tpc.113.118810" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. G. Deluc;citation_author=D. R. Quilici;citation_author=A. Decendit;citation_author=J. Grimplet;citation_author=M. D. Wheatley;citation_title=Water deficit alters differentially metabolic pathways affecting important flavor and quality traits in grape berries of Cabernet Sauvignon and Chardonnay.;citation_pages=212-212;citation_volume=10;citation_year=2009;citation_pmid=19426499;citation_doi=10.1186/1471-2164-10-212" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=G. Di Gaspero;citation_author=G. Cipriani;citation_author=M. T. Marrazzo;citation_author=D. Andreetta;citation_author=M. J. Prado Castro;citation_title=Isolation of (AC)n-microsatellites in Vitis vinifera L. and analysis of genetic background in grapevines under marker assisted selection.;citation_pages=11-20;citation_volume=15;citation_year=2005;citation_doi=10.1007/s11032-004-1362-4" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=A. Di Genova;citation_author=A. M. Almeida;citation_author=C. Muñoz-Espinoza;citation_author=P. Vizoso;citation_author=D. Travisany;citation_title=Whole genome comparison between table and wine grapes reveals a comprehensive catalog of structural variants.;citation_pages=7;citation_volume=14;citation_year=2014;citation_doi=10.1186/1471-2229-14-7" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Doi;citation_author=T. Monjo;citation_author=P. H. Hoang;citation_author=J. Yoshimura;citation_author=H. Yurino;citation_title=Rapid detection of expanded short tandem repeats in personal genomics using hybrid sequencing.;citation_volume=30;citation_year=2014;citation_pmid=24215022;citation_doi=10.1093/bioinformatics/btt647" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Dong;citation_author=H. Liu;citation_author=J. Zhang;citation_author=S. Yang;citation_author=G. Kong;citation_title=Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research.;citation_pages=1039;citation_volume=16;citation_year=2015;citation_pmid=26645802;citation_doi=10.1186/s12864-015-2257-y" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Ellinghaus;citation_author=S. Kurtz;citation_author=U. Willhoeft;citation_title=LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.;citation_pages=18-18;citation_volume=9;citation_year=2008;citation_pmid=18194517;citation_doi=10.1186/1471-2105-9-18" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=L. Falginella;citation_author=G. Di Gaspero;citation_author=S. D. Castellarin;citation_title=Expression of flavonoid genes in the red grape berry of “Alicante Bouschet” varies with the histological distribution of anthocyanins and their chemical composition.;citation_pages=1037-1051;citation_volume=236;citation_year=2012;citation_pmid=22552639;citation_doi=10.1007/s00425-012-1658-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Fasoli;citation_author=S. Dal Santo;citation_author=S. Zenoni;citation_author=G. B. Tornielli;citation_author=L. Farina;citation_title=The grapevine expression atlas reveals a deep transcriptome shift driving the entire plant into a maturation program.;citation_volume=24;citation_year=2012;citation_pmid=22948079;citation_doi=10.1105/tpc.112.100230" />
<meta name="citation_reference" content="citation_author=M. Fasoli;citation_author=C. L. Richter;citation_author=S. Zenoni;citation_author=E. Bertini;citation_author=N. Vitulo" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. A. Filichkin;citation_author=Mi. Hamilton;citation_author=P. D. Dharmawardhana;citation_author=S. K. Singh;citation_author=C. Sullivan;citation_title=Abiotic stresses modulate landscape of poplar transcriptome via alternative splicing, differential intron retention, and isoform ratio switching.;citation_pages=5;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00005" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Foissac;citation_author=M. Sammeth;citation_title=ASTALAVISTA: Dynamic and flexible analysis of alternative splicing events in custom gene datasets.;citation_volume=35;citation_year=2007;citation_pmid=17485470;citation_doi=10.1093/nar/gkm311" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=G. Gambino;citation_author=A. Dal Molin;citation_author=P. Boccacci;citation_author=A. Minio;citation_author=W. Chitarra;citation_title=Whole-genome sequencing and SNV genotyping of ‘Nebbiolo’ (Vitis vinifera L.) clones.;citation_pages=17294;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-17405-y" />
<meta name="citation_reference" content="citation_journal_title=RNA Biol.;citation_author=S. Gao;citation_author=Y. Ren;citation_author=Y. Sun;citation_author=Z. Wu;citation_author=J. Ruan;citation_title=PacBio full-length transcriptome profiling of insect mitochondrial gene expression.;citation_pages=820-825;citation_volume=13;citation_year=2016;citation_doi=10.1080/15476286.2016.1197481" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Garber;citation_author=M. G. Grabherr;citation_author=M. Guttman;citation_author=C. Trapnell;citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq.;citation_pages=469-477;citation_volume=8;citation_year=2011;citation_issue=6;citation_pmid=21623353;citation_doi=10.1038/nmeth.1613" />
<meta name="citation_reference" content="citation_journal_title=Curr. Plant Biol.;citation_author=M. Geniza;citation_author=P. Jaiswal;citation_title=Tools for building de novo transcriptome assembly.;citation_pages=41-45;citation_volume=11–12;citation_year=2017;citation_doi=10.1016/j.cpb.2017.12.004" />
<meta name="citation_reference" content="citation_author=D. G. Gilbert;citation_pages=47405;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=F. Giordano;citation_author=L. Aigrain;citation_author=M. A. Quail;citation_author=P. Coupland;citation_author=J. K. Bonfield;citation_title=De novo yeast genome assemblies from MinION, PacBio and MiSeq platforms.;citation_pages=3935;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-03996-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. Maccallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. A. Golicz;citation_author=J. Batley;citation_author=D. Edwards;citation_title=Towards plant pangenomics.;citation_pages=1099-1105;citation_volume=14;citation_year=2016;citation_doi=10.1111/pbi.12499" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. A. Golicz;citation_author=P. E. Bayer;citation_author=G. C. Barker;citation_author=P. P. Edger;citation_author=H. R. Kim;citation_title=The pangenome of an agronomically important crop plant Brassica oleracea.;citation_pages=13390;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms13390" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Gordon;citation_author=J. Huddleston;citation_author=M. J. Chaisson;citation_author=C. M. Hill;citation_author=Z. N. Kronenberg;citation_title=Long-read sequence assembly of the gorilla genome;citation_pages=aae0344-aae0344;citation_volume=352;citation_year=2016;citation_issue=6281;citation_pmid=27034376;citation_doi=10.1126/science.aae0344" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Han;citation_author=S. R. Wessler;citation_title=MITE-Hunter: A program for discovering miniature inverted-repeat transposable elements from genomic sequences.;citation_volume=38;citation_year=2010;citation_pmid=20880995;citation_doi=10.1093/nar/gkq862" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. A. Honaas;citation_author=E. K. Wafula;citation_author=N. J. Wickett;citation_author=J. P. Der;citation_author=Y. Zhang;citation_title=Selecting superior de novo transcriptome assemblies: Lessons learned by leveraging the best plant genome.;citation_pages=e0146062;citation_volume=11;citation_year=2016;citation_pmid=26731733;citation_doi=10.1371/journal.pone.0146062" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=D. W. Hopper;citation_author=R. Ghan;citation_author=K. A. Schlauch;citation_author=G. R. Cramer;citation_title=Transcriptomic network analyses of leaf dehydration responses identify highly connected ABA and ethylene signaling hubs in three grapevine species differing in drought tolerance.;citation_pages=118;citation_volume=16;citation_year=2016;citation_doi=10.1186/s12870-016-0804-6" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=R. Hu;citation_author=G. Sun;citation_author=X. Sun;citation_title=LSCplus: a fast solution for improving long read accuracy by short read alignment.;citation_pages=451;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-1316-y" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Huang;citation_author=X. G. Chen;citation_author=P. A. Armbruster;citation_title=Comparative performance of transcriptome assembly methods for non-model organisms.;citation_pages=523;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2923-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Huang;citation_author=Z. Chen;citation_author=G. Huang;citation_author=T. Yu;citation_author=P. Yang;citation_title=HaploMerger: Reconstructing allelic relationships for polymorphic diploid genome assemblies.;citation_volume=22;citation_year=2012;citation_pmid=22555592;citation_doi=10.1101/gr.133652.111" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Huddleston;citation_author=M. J. Chaisson;citation_author=K. M. Steinberg;citation_author=W. Warren;citation_author=K. Hoekzema;citation_title=Discovery and genotyping of structural variation from long-read haploid genome sequence data.;citation_volume=27;citation_year=2017;citation_pmid=27895111;citation_doi=10.1101/gr.214007.116" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Huson;citation_author=A. F. Auch;citation_author=J. Qi;citation_author=S. C. Schuster;citation_title=MEGAN analysis of metagenomic data.;citation_volume=17;citation_year=2007;citation_pmid=17255551;citation_doi=10.1101/gr.5969107" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Enology and Viticulture;citation_journal_abbrev=Am. J. Enol. Vitic.;citation_author=J. Ibáñez;citation_author=A. M. Vargas;citation_author=M. Palancar;citation_author=J. Borrego;citation_author=M. T. De Andrés;citation_title=Genetic Relationships among Table-Grape Varieties;citation_pages=35-42;citation_volume=60;citation_year=2009;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=O. Jaillon;citation_author=J.-M. Aury;citation_author=B. Noel;citation_author=A. Policriti;citation_author=C. Clepet;citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla.;citation_pages=463-467;citation_volume=449;citation_year=2007;citation_issue=7161;citation_pmid=17721507;citation_doi=10.1038/nature06148" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Jiao;citation_author=M. Gao;citation_author=X. Wang;citation_author=Z. Fei;citation_title=Transcriptome characterization of three wild Chinese Vitis uncovers a large number of distinct disease related genes.;citation_pages=223;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1442-3" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Jones;citation_author=D. Binns;citation_author=H.-Y. Chang;citation_author=M. Fraser;citation_author=W. Li;citation_title=InterProScan 5: genome-scale protein function classification.;citation_volume=30;citation_year=2014;citation_pmid=24451626;citation_doi=10.1093/bioinformatics/btu031" />
<meta name="citation_reference" content="citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinforma.;citation_author=C. Ju;citation_author=Z. Zhao;citation_author=W. Wang;citation_title=Efficient approach to correct read alignment for pseudogene abundance estimates.;citation_pages=522-533;citation_volume=14;citation_year=2017;citation_doi=10.1109/TCBB.2016.2591533" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshimoto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. Koren;citation_author=M. C. Schatz;citation_author=B. P. Walenz;citation_author=J. Martin;citation_author=J. T. Howard;citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads.;citation_pages=693-700;citation_volume=30;citation_year=2012;citation_issue=7;citation_pmid=22750884;citation_doi=10.1038/nbt.2280" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k -mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=I. Korf;citation_author=P. Flicek;citation_author=D. Duan;citation_author=M. R. Brent;citation_title=Integrating genomic homology into gene structure prediction;citation_pages=S140-148;citation_volume=17;citation_year=2001;citation_issue=90001;citation_pmid=11473003;citation_doi=10.1093/bioinformatics/17.suppl_1.S140" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=N. Kuhn;citation_author=L. Guan;citation_author=Z. W. Dai;citation_author=B. H. Wu;citation_author=V. Lauvergeat;citation_title=Berry ripening: Recently heard through the grapevine.;citation_volume=65;citation_year=2014;citation_pmid=24285825;citation_doi=10.1093/jxb/ert395" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. I. Kuo;citation_author=E. Tseng;citation_author=L. Eory;citation_author=I. R. Paton;citation_author=A. L. Archibald;citation_title=Normalized long read RNA sequencing in chicken reveals transcriptome complexity similar to human.;citation_pages=323;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3691-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Lacombe;citation_author=J. M. Boursiquot;citation_author=V. Laucou;citation_author=M. Di Vecchi-Staraz;citation_author=J. P. Péros;citation_title=Large-scale parentage analysis in an extended set of grapevine cultivars (Vitis vinifera L.).;citation_pages=401-414;citation_volume=126;citation_year=2013;citation_pmid=23015217;citation_doi=10.1007/s00122-012-1988-2" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=S. Lê;citation_author=J. Josse;citation_author=F. Husson;citation_title=FactoMineR : An R Package for Multivariate Analysis.;citation_pages=1-18;citation_volume=25;citation_year=2008;citation_doi=10.18637/jss.v025.i01" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=F. Lecourieux;citation_author=C. Kappel;citation_author=D. Lecourieux;citation_author=A. Serrano;citation_author=E. Torres;citation_title=An update on sugar transport and signalling in grapevine.;citation_volume=65;citation_year=2014;citation_pmid=24323501;citation_doi=10.1093/jxb/ert394" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Lecourieux;citation_author=C. Kappel;citation_author=P. Pieri;citation_author=J. Charon;citation_author=J. Pillet;citation_title=Dissecting the biochemical and transcriptomic effects of a locally applied heat treatment on developing Cabernet Sauvignon grape berries.;citation_pages=53;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.00053" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Leng;citation_author=J. A. Dawson;citation_author=J. A. Thomson;citation_author=V. Ruotti;citation_author=A. I. Rissman;citation_title=EBSeq: An empirical Bayes hierarchical model for inference in RNA-seq experiments.;citation_volume=29;citation_year=2013;citation_pmid=23428641;citation_doi=10.1093/bioinformatics/btt087" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=B. Li;citation_author=C. N. Dewey;citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.;citation_pages=323-323;citation_volume=12;citation_year=2011;citation_pmid=21816040;citation_doi=10.1186/1471-2105-12-323" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Li;citation_author=A. Godzik;citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_author=J. Liu;citation_author=X. Chen;citation_author=X. Liang;citation_author=X. Zhou;citation_author=F. Yang" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=X. Liu;citation_author=W. Mei;citation_author=P. S. Soltis;citation_author=D. E. Soltis;citation_author=W. B. Barbazuk;citation_title=Detecting alternatively spliced transcript isoforms from single-molecule long-read sequences without a reference genome.;citation_pages=1243-1256;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12670" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. A. Lodhi;citation_author=B. I. Reisch;citation_title=Nuclear DNA content of Vitis species, cultivars, and other genera of the Vitaceae.;citation_pages=11-16;citation_volume=90;citation_year=1995;citation_doi=10.1007/BF00220990" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Lomsadze;citation_author=V. Ter-Hovhannisyan;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene identification in novel eukaryotic genomes by self-training algorithm;citation_pages=6494-6506;citation_volume=33;citation_year=2005;citation_issue=20;citation_pmid=16314312;citation_doi=10.1093/nar/gki937" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. S. Lopes;citation_author=K. M. Sefc;citation_author=E. Eiras Dias;citation_author=H. Steinkellner;citation_author=M. Laimer Câmara Machado;citation_title=The use of microsatellites for germplasm management in a Portuguese grapevine collection.;citation_pages=733-739;citation_volume=99;citation_year=1999;citation_issue=3-4;citation_pmid=22665212;citation_doi=10.1007/s001220051291" />
<meta name="citation_reference" content="citation_journal_title=Phytochemistry;citation_journal_abbrev=Phytochemistry;citation_author=J. Lücker;citation_author=P. Bowen;citation_author=J. Bohlmann;citation_title=Vitis vinifera terpenoid cyclases: functional identification of two sesquiterpene synthase cDNAs encoding (+)-valencene synthase and (-)-germacrene D synthase and expression of mono- and sesquiterpene synthases in grapevine flowers and berries.;citation_pages=2649-2659;citation_volume=65;citation_year=2004;citation_issue=19;citation_pmid=15464152;citation_doi=10.1016/j.phytochem.2004.08.017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. H. Majoros;citation_author=M. Pertea;citation_author=S. L. Salzberg;citation_title=TigrScan and GlimmerHMM: Two open source ab initio eukaryotic gene-finders.;citation_volume=20;citation_year=2004;citation_pmid=15145805;citation_doi=10.1093/bioinformatics/bth315" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Marquez;citation_author=J. W. S. Brown;citation_author=C. Simpson;citation_author=A. Barta;citation_author=M. Kalyna;citation_title=Transcriptome survey reveals increased complexity of the alternative splicing landscape in Arabidopsis.;citation_volume=22;citation_year=2012;citation_pmid=22391557;citation_doi=10.1101/gr.134106.111" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. M. Martin;citation_author=O. Toub;citation_author=A. Chiang;citation_author=B. C. Lo;citation_author=S. Ohse;citation_title=The bouquet of grapevine (Vitis vinifera L. cv. Cabernet Sauvignon) flowers arises from the biosynthesis of sesquiterpene volatiles in pollen grains.;citation_volume=106;citation_year=2009;citation_pmid=19359488;citation_doi=10.1073/pnas.0901387106" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Martin;citation_author=Z. Wang;citation_title=Next-generation transcriptome assembly.;citation_pages=671-682;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21897427;citation_doi=10.1038/nrg3068" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. Massonnet;citation_author=M. Fasoli;citation_author=G. B. Tornielli;citation_author=M. Altieri;citation_author=M. Sandri;citation_title=Ripening Transcriptomic Program in Red and White Grapevine Varieties Correlates with Berry Skin Anthocyanin Accumulation.;citation_volume=174;citation_year=2017;citation_pmid=28652263;citation_doi=10.1104/pp.17.00311" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=M. Massonnet;citation_author=R. Figueroa-Balderas;citation_author=E. R. A. Galarneau;citation_author=S. Miki;citation_author=D. P. Lawrence;citation_title=Neofusicoccum parvum Colonization of the Grapevine Woody Stem Triggers Asynchronous Host Responses at the Site of Infection and in the Leaves.;citation_pages=1117;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01117" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Minio;citation_author=J. Lin;citation_author=B. S. B. S. Gaut;citation_author=D. Cantu;citation_title=How single molecule real-time sequencing and haplotype phasing have enabled reference-grade diploid genome assembly of wine grapes.;citation_pages=826;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.00826" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. Moreton;citation_author=A. Izquierdo;citation_author=R. D. Emes;citation_title=Assembly, assessment, and availability of De novo generated eukaryotic transcriptomes.;citation_pages=1-9;citation_volume=6;citation_year=2016;citation_doi=10.3389/fgene.2015.00361" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Myles;citation_author=A. R. Boyko;citation_author=C. L. Owens;citation_author=P. J. Brown;citation_author=F. Grassi;citation_title=Genetic structure and domestication history of the grape.;citation_volume=108;citation_year=2011;citation_pmid=21245334;citation_doi=10.1073/pnas.1009363108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. P. Nawrocki;citation_author=S. W. Burge;citation_author=A. Bateman;citation_author=J. Daub;citation_author=R. Y. Eberhardt;citation_title=Rfam 12.0: updates to the RNA families database.;citation_volume=43;citation_year=2015;citation_pmid=25392425;citation_doi=10.1093/nar/gku1063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. P. Nawrocki;citation_author=D. L. Kolbe;citation_author=S. R. Eddy;citation_title=Infernal 1.0: Inference of RNA alignments.;citation_volume=25;citation_year=2009;citation_pmid=19307242;citation_doi=10.1093/bioinformatics/btp157" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. Ohmi;citation_author=A. Wakana;citation_author=S. Shiraishi;citation_author=M. Alexandria;citation_title=Study of the parentage of grape cultivars by genetic interpretation of GPI-2 and PGM-2 isozymes.;citation_pages=195-202;citation_volume=65;citation_year=1993;citation_doi=10.1007/BF00023083" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Online;citation_author=M. C. Palumbo;citation_author=S. Zenoni;citation_author=M. Fasoli;citation_author=M. Massonnet;citation_author=L. Farina;citation_title=Integrated network analysis identifies fight-club nodes as a class of hubs encompassing key putative switch genes that induce major transcriptome reprogramming during grapevine development.;citation_volume=26;citation_year=2014;citation_pmid=25490918;citation_doi=10.1105/tpc.114.133710" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. Parra;citation_author=E. Blanco;citation_author=R. Guigó;citation_title=GeneID in Drosophila;citation_pages=511-515;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779490;citation_doi=10.1101/gr.10.4.511" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=C. Pastore;citation_author=S. Zenoni;citation_author=M. Fasoli;citation_author=M. Pezzotti;citation_author=G. B. Tornielli;citation_title=Selective defoliation affects plant growth, fruit transcriptional ripening program and flavonoid metabolism in grapevine.;citation_pages=30-30;citation_volume=13;citation_year=2013;citation_issue=1;citation_pmid=23433030;citation_doi=10.1186/1471-2229-13-30" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Pertea;citation_author=G. M. Pertea;citation_author=C. M. Antonescu;citation_author=T.-C. Chang;citation_author=J. T. Mendell;citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.;citation_pages=290-295;citation_volume=33;citation_year=2015;citation_pmid=25690850;citation_doi=10.1038/nbt.3122" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Price;citation_author=C. Gibas;citation_title=The quantitative impact of read mapping to non-native reference genomes in comparative RNA-Seq studies.;citation_pages=e0180904;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0180904" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. P. Pryszcz;citation_author=T. Gabaldón;citation_title=Redundans: An assembly pipeline for highly heterozygous genomes.;citation_volume=44;citation_year=2016;citation_pmid=27131372;citation_doi=10.1093/nar/gkw294" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. S. N. Reddy;citation_author=Y. Marquez;citation_author=M. Kalyna;citation_author=A. Barta;citation_title=Complexity of the alternative splicing landscape in plants.;citation_volume=25;citation_year=2013;citation_pmid=24179125;citation_doi=10.1105/tpc.113.117523" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=N. Ricker;citation_author=S. Y. Shen;citation_author=J. Goordial;citation_author=S. Jin;citation_author=R. R. Fulthorpe;citation_title=PacBio SMRT assembly of a complex multi-replicon genome reveals chlorocatechol degradative operon in a region of genome plasticity.;citation_pages=239-247;citation_volume=586;citation_year=2016;citation_doi=10.1016/j.gene.2016.04.018" />
<meta name="citation_reference" content="citation_journal_title=Comput. Mol. Cell Biol.;citation_author=Y. Safonova;citation_author=A. Bankevich;citation_author=P. A. Pevzner;citation_title=dipSPAdes: Assembler for Highly Polymorphic Diploid Genomes. J. Comput. Biol. A J.;citation_pages=528-545;citation_volume=22;citation_year=2015;citation_doi=10.1089/cmb.2014.0153" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Salmela;citation_author=E. Rivals;citation_title=LoRDEC: Accurate and efficient long read error correction.;citation_volume=30;citation_year=2014;citation_pmid=25165095;citation_doi=10.1093/bioinformatics/btu538" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. Savoi;citation_author=D. C. J. Wong;citation_author=P. Arapitsas;citation_author=M. Miculan;citation_author=B. Bucchetti;citation_title=Transcriptome and metabolite profiling reveals that prolonged drought modulates the phenylpropanoid and terpenoid pathway in white grapes (Vitis vinifera L.).;citation_pages=67;citation_volume=16;citation_year=2016;citation_pmid=27001212;citation_doi=10.1186/s12870-016-0760-1" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. Savoi;citation_author=D. C. J. Wong;citation_author=A. Degu;citation_author=J. C. Herrera;citation_author=B. Bucchetti;citation_title=Multi-Omics and Integrated Network Analyses Reveal New Insights into the Systems Relationships between Metabolites, Structural Genes, and Transcriptional Regulators in Developing Grape Berries (Vitis vinifera L.) Exposed to Water Deficit.;citation_pages=1124;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01124" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. M. Sefc;citation_author=H. Steinkellner;citation_author=J. Glössl;citation_author=S. Kampfer;citation_author=F. Regner;citation_title=Reconstruction of a grapevine pedigree by microsatellite analysis.;citation_pages=227-231;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050889" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Seo;citation_author=A. Rhie;citation_author=J. Kim;citation_author=S. Lee;citation_author=M. Sohn;citation_title=De novo assembly and phasing of a Korean human genome.;citation_pages=243-247;citation_volume=538;citation_year=2016;citation_pmid=27706134;citation_doi=10.1038/nature20098" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Serrano;citation_author=C. Espinoza;citation_author=G. Armijo;citation_author=C. Inostroza-Blancheteau;citation_author=E. Poblete;citation_title=Omics Approaches for Understanding Grapevine Berry Development: Regulatory Networks Associated with Endogenous Processes and Environmental Responses.;citation_pages=1486;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01486" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=S. Simon;citation_author=J. Zhai;citation_author=R. S. Nandety;citation_author=K. P. McCormick;citation_author=J. Zeng;citation_title=Short-read sequencing technologies for transcriptional analyses.;citation_pages=305-333;citation_volume=60;citation_year=2009;citation_pmid=19575585;citation_doi=10.1146/annurev.arplant.043008.092032" />
<meta name="citation_reference" content="Smit, A. F. A., and R. Hubley, 2008 RepeatModeler Open-1.0. http://www.repeatmasker.org." />
<meta name="citation_reference" content="Smit, A. F. A., R. Hubley, and P. Green, 2013 RepeatMasker Open-4.0. http://www.repeatmasker.org 2013–2015." />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. D. Smith-Unna;citation_author=C. Boursnell;citation_author=R. Patro;citation_author=J. M. Hibberd;citation_author=S. Kelly;citation_title=TransRate: reference-free quality assessment of de novo transcriptome assemblies.;citation_volume=26;citation_year=2016;citation_pmid=27252236;citation_doi=10.1101/gr.196469.115" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=F. Sparvoli;citation_author=C. Martin;citation_author=A. Scienza;citation_author=G. Gavazzi;citation_author=C. Tonelli;citation_title=Cloning and molecular analysis of structural genes involved in flavonoid and stilbene biosynthesis in grape (Vitis vinifera L.).;citation_pages=743-755;citation_volume=24;citation_year=1994;citation_issue=5;citation_pmid=8193299;citation_doi=10.1007/BF00029856" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=O. Keller;citation_author=I. Gunduz;citation_author=A. Hayes;citation_author=S. Waack;citation_title=AUGUSTUS: ab initio prediction of alternative transcripts.;citation_pages=W435-W439;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845043;citation_doi=10.1093/nar/gkl200" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Steinbiss;citation_author=U. Willhoeft;citation_author=G. Gremme;citation_author=S. Kurtz;citation_title=Fine-grained annotation and classification of de novo predicted LTR retrotransposons.;citation_volume=37;citation_year=2009;citation_pmid=19786494;citation_doi=10.1093/nar/gkp759" />
<meta name="citation_reference" content="citation_journal_title=Vitis;citation_author=M. S. Strefeler;citation_author=N. F. Weeden;citation_author=B. I. Reisch;citation_title=Inheritance of chloroplast DNA in two full-sib Vitis populations.;citation_pages=183-187;citation_volume=31;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=R. Swarup;citation_author=M. Crespi;citation_author=M. J. Bennett;citation_title=One gene, many proteins: mapping cell-specific alternative splicing in plants.;citation_pages=383-385;citation_volume=39;citation_year=2016;citation_doi=10.1016/j.devcel.2016.11.002" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Enology and Viticulture;citation_journal_abbrev=Am. J. Enol. Vitic.;citation_author=A. M. Tapia;citation_author=J. A. Cabezas;citation_author=F. Cabello;citation_author=T. Lacombe;citation_author=J. M. Martínez-Zapater;citation_title=Determining the Spanish Origin of Representative Ancient American Grapevine Varieties;citation_pages=242-251;citation_volume=58;citation_year=2007;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. R. Thatcher;citation_author=O. N. Danilevskaya;citation_author=X. Meng;citation_author=M. Beatty;citation_author=G. Zastrow-Hayes;citation_title=Genome-wide analysis of alternative splicing during development and drought stress in maize.;citation_volume=170;citation_year=2016;citation_pmid=26582726;citation_doi=10.1104/pp.15.01267" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Tombácz;citation_author=Z. Csabai;citation_author=P. Oláh;citation_author=Z. Balázs;citation_author=I. Likó;citation_title=Full-length isoform sequencing reveals novel transcripts and substantial transcriptional overlaps in a herpesvirus.;citation_pages=e0162868;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0162868" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Ungaro;citation_author=N. Pech;citation_author=J. F. Martin;citation_author=R. J. S. McCairns;citation_author=J. P. Mévy;citation_title=Challenges and advances for transcriptome assembly in non-model species.;citation_pages=e0185020;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0185020" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Velasco;citation_author=A. Zharkikh;citation_author=M. Troggio;citation_author=D. A. Cartwright;citation_author=A. Cestaro;citation_title=A high quality draft consensus sequence of the genome of a heterozygous grapevine variety.;citation_pages=e1326-e1326;citation_volume=2;citation_year=2007;citation_issue=12;citation_pmid=18094749;citation_doi=10.1371/journal.pone.0001326" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Venturini;citation_author=A. Ferrarini;citation_author=S. Zenoni;citation_author=G. B. G. B. Tornielli;citation_author=M. Fasoli;citation_title=De novo transcriptome characterization of Vitis vinifera cv. Corvina unveils varietal diversity.;citation_pages=41;citation_volume=14;citation_year=2013;citation_pmid=23331995;citation_doi=10.1186/1471-2164-14-41" />
<meta name="citation_reference" content="citation_journal_title=PLOS Genet.;citation_author=S. Vij;citation_author=H. Kuhl;citation_author=I. S. Kuznetsova;citation_author=A. Komissarov;citation_author=A. A. Yurchenko;citation_title=Chromosomal-level assembly of the asian seabass genome using long sequence reads and multi-layered scaffolding.;citation_pages=e1005954;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006500" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=N. Vitulo;citation_author=C. Forcato;citation_author=E. C. Carpinelli;citation_author=A. Telatin;citation_author=D. Campagna;citation_title=A deep survey of alternative splicing in grape reveals changes in the splicing machinery related to tissue, stress condition and genotype.;citation_pages=99;citation_volume=14;citation_year=2014;citation_pmid=24739459;citation_doi=10.1186/1471-2229-14-99" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=B. Wang;citation_author=E. Tseng;citation_author=M. Regulski;citation_author=T. A. Clark;citation_author=T. Hon;citation_title=Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing.;citation_pages=11708;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms11708" />
<meta name="citation_reference" content="Warnes, G. R., B. Bolker, L. Bonebakker, R. Gentleman, W. Huber et al., 2016 Package “gplots”: Various R programming tools for plotting data. R Packag. version 2.17.0 https://CRAN.R-project.org/package=gplots." />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. L. Weirather;citation_author=P. T. Afshar;citation_author=T. A. Clark;citation_author=E. Tseng;citation_author=L. S. Powers;citation_title=Characterization of fusion genes and the significantly expressed fusion isoforms in breast cancer by hybrid sequencing.;citation_volume=43;citation_year=2015;citation_pmid=26040699;citation_doi=10.1093/nar/gkv562" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Single molecule, full-length transcript sequencing provides insight into the extreme metabolism of ruby-throated hummingbird Archilochus colubris.;citation_author=R. E. Workman;citation_author=A. M. Myrka;citation_author=E. Tseng;citation_author=G. W. Wong;citation_author=K. C. Welch;citation_pages=117218;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Xi;citation_author=L. Ma;citation_author=G. Liu;citation_author=N. Wang;citation_author=J. Wang;citation_title=Transcriptomic Analysis of Grape (Vitis vinifera L.) Leaves after Exposure to Ultraviolet C Irradiation.;citation_pages=e113772;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0113772" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=H. Xin;citation_author=J. Zhang;citation_author=W. Zhu;citation_author=N. Wang;citation_author=P. Fang;citation_title=The effects of artificial selection on sugar metabolism and transporter genes in grape.;citation_pages=1343-1349;citation_volume=9;citation_year=2013;citation_doi=10.1007/s11295-013-0643-7" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=K. Yan;citation_author=P. Liu;citation_author=C. A. Wu;citation_author=G. D. Yang;citation_author=R. Xu;citation_title=Stress-Induced Alternative Splicing Provides a Mechanism for the Regulation of MicroRNA Processing in Arabidopsis thaliana.;citation_pages=521-531;citation_volume=48;citation_year=2012;citation_issue=4;citation_pmid=23063528;citation_doi=10.1016/j.molcel.2012.08.032" />
<meta name="citation_reference" content="citation_journal_title=Genomics Inform.;citation_author=I. S. Yang;citation_author=S. Kim;citation_title=Analysis of whole transcriptome sequencing data: Workflow and software.;citation_pages=119-125;citation_volume=13;citation_year=2015;citation_doi=10.5808/GI.2015.13.4.119" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Ye;citation_author=L. Fang;citation_author=H. Zheng;citation_author=Y. Zhang;citation_author=J. Chen;citation_title=WEGO: a web tool for plotting GO annotations.;citation_pages=W293-W297;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845012;citation_doi=10.1093/nar/gkl031" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=S. Zenoni;citation_author=S. Dal Santo;citation_author=G. B. Tornielli;citation_author=E. D’Incà;citation_author=I. Filippetti;citation_title=Transcriptional responses to pre-flowering leaf defoliation in grapevine berry from different growing sites, years, and genotypes.;citation_pages=630;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.00630" />
<meta name="citation_reference" content="citation_journal_title=Bioinform. Biol. Insights;citation_author=C. Zhang;citation_author=H. Yang;citation_author=H. Yang;citation_title=Evolutionary character of alternative splicing in plants.;citation_pages=47-52;citation_volume=9;citation_year=2015;citation_doi=10.4137/BBI.S33716" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Zhou;citation_author=M. Massonnet;citation_author=J. S. Sanjak;citation_author=D. Cantu;citation_author=B. S. Gaut;citation_title=Evolutionary genomics of grape (Vitis vinifera ssp. vinifera) domestication.;citation_volume=114;citation_year=2017;citation_pmid=29042518;citation_doi=10.1073/pnas.1709257114" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. V. Zimin;citation_author=D. Puiu;citation_author=M. C. Luo;citation_author=T. Zhu;citation_author=S. Koren;citation_title=Hybrid assembly of the large and highly repetitive genome of Aegilops tauschii, a progenitor of bread wheat, with the MaSuRCA mega-reads algorithm.;citation_volume=27;citation_year=2017;citation_pmid=28130360;citation_doi=10.1101/gr.213405.116" />
<meta name="citation_reference" content="citation_journal_title=Genom. Data;citation_author=M. M. Zulkapli;citation_author=M. A. F. Rosli;citation_author=F. I. M. Salleh;citation_author=N. Mohd Noor;citation_author=W. M. Aizat;citation_title=Iso-Seq analysis of Nepenthes ampullaria, Nepenthes rafflesiana and Nepenthes × hookeriana for hybridisation study in pitcher plants.;citation_pages=130-131;citation_volume=12;citation_year=2017;citation_doi=10.1016/j.gdata.2017.05.003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Transcriptomics has been widely applied to study grape berry development. With few exceptions, transcriptomic studies in grape are performed using the available genome sequence, PN40024, as reference. However, differences in gene content among grape accessions, which contribute to phenotypic differences among cultivars, suggest that a single reference genome does not represent the species’ entire gene space. Though whole genome assembly and annotation can reveal the relatively unique or “private” gene space of any particular cultivar, transcriptome reconstruction is a more rapid, less costly, and less computationally intensive strategy to accomplish the same goal. In this study, we used single molecule-real time sequencing (SMRT) to sequence full-length cDNA (Iso-Seq) and reconstruct the transcriptome of Cabernet Sauvignon berries during berry ripening. In addition, short reads from ripening berries were used to error-correct low-expression isoforms and to profile isoform expression. By comparing the annotated gene space of Cabernet Sauvignon to other grape cultivars, we demonstrate that the transcriptome reference built with Iso-Seq data represents most of the expressed genes in the grape berries and includes 1,501 cultivar-specific genes. Iso-Seq produced transcriptome profiles similar to those obtained after mapping on a complete genome reference. Together, these results justify the application of Iso-Seq to identify cultivar-specific genes and build a comprehensive reference for transcriptional profiling that circumvents the necessity of a genome reference with its associated costs and computational weight." />
<meta name="og-title" property="og:title" content="Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/755" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transcriptomics has been widely applied to study grape berry development. With few exceptions, transcriptomic studies in grape are performed using the available genome sequence, PN40024, as reference. However, differences in gene content among grape accessions, which contribute to phenotypic differences among cultivars, suggest that a single reference genome does not represent the species’ entire gene space. Though whole genome assembly and annotation can reveal the relatively unique or “private” gene space of any particular cultivar, transcriptome reconstruction is a more rapid, less costly, and less computationally intensive strategy to accomplish the same goal. In this study, we used single molecule-real time sequencing (SMRT) to sequence full-length cDNA (Iso-Seq) and reconstruct the transcriptome of Cabernet Sauvignon berries during berry ripening. In addition, short reads from ripening berries were used to error-correct low-expression isoforms and to profile isoform expression. By comparing the annotated gene space of Cabernet Sauvignon to other grape cultivars, we demonstrate that the transcriptome reference built with Iso-Seq data represents most of the expressed genes in the grape berries and includes 1,501 cultivar-specific genes. Iso-Seq produced transcriptome profiles similar to those obtained after mapping on a complete genome reference. Together, these results justify the application of Iso-Seq to identify cultivar-specific genes and build a comprehensive reference for transcriptional profiling that circumvents the necessity of a genome reference with its associated costs and computational weight." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mfhNSfa9_SIIve8aHHqIZvpA8rNjE62BBfIj8JNSf-w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492489","apath":"\/ggg\/9\/3\/755.atom","pisa":"ggg;9\/3\/755","ac":{"\/ggg\/9\/3\/755.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/755.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/755"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/755"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/755":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229gp\\\\\\\/1eqJw9tf3qwGyG4OU89LeISs88v8mNoTjFfoPWawgKmPKZ3exSlQZeZ6IGAIa+kU7ohXBOt0OoKAiQ4Bh\\\\\\\/xJ3LuWs1My55vx7ReagtL\\\\\\\/xVqQYCd5DYHPoUd0gu+qTo9i79nDwAKnzafxIbP7d4JLkmaAqQzn1MGvvj\\\\\\\/NaDI2ICiZV14KP3rirRuPfzm9YSxcGuDLuBZXWPdf8yx6X6l8EJXWpGPcCT33tivGRS0zEescXA+HdugvbGQUSd6lnJXN8aY+L3eAjc5TxMZGQnfEN4XRugBy\\\\\\\/nzygQqmISrxAlpl6cVzs70FiQv72CbX6gT05PrIvyV+BKLXxuyk\\\\\\\/MJhXxNGZM6QgZnEv5mLLIBL+xc1mkC8RQ\\\\\\\/0NQeADzWqenYAGdVEJgbOuRoAYyoTK9hzgomZ1mIsyR0OE9+TjLIxtXIFevKmqBdS95y17IEEFLUH5lrA5ejqCcvPJr94eaO1mrVQH7GSR+KyeCj3ci4cyB1mFcgMYFgQ\\\\\\\/ahB6YEOXlUBMDAwbVDyye1Qe08DCskY00tgRM7dVgis43PhCdXc4flNzPMDFdPafqtd2U5hdCzQAC8I78MHI6uB0QVX\\\\\\\/UroqM2bBTkdZHxwyyswgzX\\\\\\\/aByfksB7085GmxJAIeASvlQzNjXiC+jJeikCoJTCc35qjmq\\\\\\\/lxq4DJcJal66glOpRbA0rPZJsGVIAWwxbKa95yI8MZLhJFQzJAuQPsgLck+gG9z5gMOqK3XSNBg5cfooGW2Y1Yx9+sHH7hTW1W62r5ByRJK3xlr9pVJwNbts48yF8p9WZtyCJjhDiKk5CG8xFd+ktVr+ai\\\\\\\/rTUUIv2OvXFSgrep\\\\\\\/NDbgiwNHUCwJlg79WbWfxepTEQ+MZHPMeYbOE8WO5dUKmeUfaK4qT6wbP3a8QHd\\\\\\\/CD5DVnRDMUwz0JnrRvuNMUqCIqx6xzFj3L+ECJNks6e6biY0DWmUJ77ZEmKU4uQQ9MytCLOik7hj3dRVB4y\\\\\\\/pRXxLSaPuFCw+q6NEYWGbMtfVrpkEoduBv5p7tyNk8Nh+uG9w71lvg1+EqpGajmUStggmqSEixpfICsNoMyPZ52G9fgJ6243yzBUxm+zWntpoFBLp5+p+jCVPhQJTT3XBu96EEsZ+xDPOcrSkUHLZqQQcqHPg12xUDnFQDmg1DwDoCtYSd9bgBY24NFnyhpc+gEUKFVwo2yFYdVeHYSFw1kughJaioZsInUlTX\\\\\\\/kjw1gfJ0uSgUqcgyta4SHnqJo9nghXd1hi7Ee4ikoDa7uxD9XgKoly1jTKEyE\\\u0022,\\\u0022iv\\\u0022:\\\u0022QKDRxjP6cNODqyVITebvqw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227a20cef23502a8a37f9f93cd1ca74b65aea2ec608fd5b74a4db86967a26399fd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022icAppJXcrvJFLrPirrVaBGhKhsc3SLwJBSjToRToo9ytKxRl6wT\\\\\\\/gprvtHhT2lAbycv9xdkS4oniONzvnvqe91b7tqlRTF\\\\\\\/fCy3HesxNcBtw8p\\\\\\\/6+qWn1J36kKbSGGQrj2q82M83uy8l75zLDqfnnAfkIgphn0D3lfrF3tSz8cWi8qZKrnP3JCa56DlNJeuLvORh47jDt7ekeUq6g3yzGUq\\\\\\\/y8Ncv2n+bgE18JOjUVqXVW6kyVzqmxEYYPR1jrTTuSR\\\\\\\/cl2xe7AfdCbDAwfE5ploGIZ3Mr49hs0uroq06WEa1H5XvdJTEMnpafXfimS1\\\\\\\/Rv4bwVgcQDi+ExaNBknyWGxisw2vqFHwsjDZzwA1YyvOpT2wCSfJRNWsUjZZjazYoXPwhszQYGgrgfT2iXbWRB+V75EMuTePgQxKLQ2qTzf5m0Bhx1DgCa\\\\\\\/3L1jD5CVKjrvUaW3ZKEo9eqOOYClEBuiNPDV+FcRI9lWpoHSYkAt2ov6co4M2P00J77hVzIJL14vtYJMRHqis+rW9MSRs5FhmuPkFwEoew6pjQZltPOF8YHcSSAAhMYFobJhisJDQInoTHZGJHpT9E5HwopwB+3Lr8JCV7vchK3q58AZdjF64zuGAVq4EUylDFmsy9C2\\\\\\\/nKnT7XTwUv5WjoFLqb85ZlZbFzYuAg76HUPFl42j4CaJL7\\\\\\\/6kOxO+b\\\\\\\/F\\\\\\\/En7XSzSDM\\\\\\\/kR\\\\\\\/PoCG7rwi43OYAjmu26Vmzu3Mw+j6AZxGOyBd1568slMeUiNZlS1HXV5T69qWUUXE7AqCxUjdL88nvvsScoOPT8cgXrtUPo5VCkuVKcjMq6GoGZrwANmRrZDs7EJSl18kJsdgDqzfEiKMWCEVDNnCtppeeZe0yBxaLsAAyEMyGgd1drdzclnlEvRmm4jTf9t5nS0kIBEq+YxIwuFtCk4RetacJvs4lGgqJ32FybejzqgeJTTbyLxOgfcp6SKdz55fkdeerLM6zy3FFsgfytEqPBad3skDQyBSkTzLzYVYu3tJitgxtlfP80dP5bXmFSyZ+eiBbqMjXOQSo0WHecxNs3\\\\\\\/DTusVeGNhFY71ovsmaJSR84Pq43ikyNpGNCRP6VwVxpNi4CY+Wj5H7FUEA\\\\\\\/hPCdsK752NL56BBPGtq0rL76uiFOo0JCUUInRonY\\\\\\\/Fm4EWaG2QURyyNVHikAnsAXaVq26601aJZGv9qtXWUVVR1gSgipNWL\\\\\\\/eDJj1i6YfGB1RVrCChunb9X+mt8w4VDUhuV9Op19P4WLCaM20u8x5pjateGC7ZYxZSFELKyQ1mFh7D+AhLka9EtkUvgoAKRBoeXERsns9tSJ2Md0DftFBn8N\\\\\\\/QhqJCn9tuL\\\\\\\/z\\\\\\\/gSTi+j26A6laYhqjSJ\\\\\\\/Gfkm4y\\\\\\\/b0Z4t6FLJbhrFC8RPUMDZ6f+fDFuS\\\\\\\/fuIAhTVn\\\\\\\/YIBLp42s3q8NONVBfIlPLAJRvrfvLXLPWgwFvN8BgjYnydrG9xspJdQEqnB03hYX4baELObxnXiZTar10acOH3E8OGRu7TMi8Sa6xbirSnF3XgGl\\\\\\\/XeTDm2qnoO\\\\\\\/7\\\\\\\/FRM4ptqA6Zb9LGUCF5chLlxPFsRdFYUm64J2\\\\\\\/zYPOPfm93yqkkC4LTfdweJVPuGCyzS53vbEalnCUBGTv2JS+3KESP0bi+XZq+5kz0+1OaNMGK6U8g8FGJclm32qUdWzXXsUOQIX85cOYg5cTWo7poGo3aDBoU5B2hgulRg\\\\\\\/30LcI11uXlv06hIH9q8chO7yaHra8x3z4yrBBkrMcrBrrQXwyO1efT5xwbfkEi6BAo0eXgM6SgzFdjzuIHHd3C6X+yRacesXMkL+scN\\\\\\\/LA9ZUbYt\\\\\\\/wDO6IZ+t\\\\\\\/Pka7OsfAB4nZDi+0LojveNb7UoMK7R6jvNZhP1By+HbGAXwFFtRA2Ym01Kz70ks2qDc8Nv6k2lbn9eY4FzsYfbKXcXkC0rcKyEUe\\\\\\\/7WqPIHufqsSd\\\\\\\/C8iS1859B1yyyXUShHzjyra1L8qbmA2hejjqQD2WIo448RRW0nodGyfGq7\\\\\\\/czyn6WMbLIn69QeZYokA4N7BjasZGU1UEs8rVppxEn867KSul6VsoXE3JX+NGUUb0p9rd9s7VFHb5Wo0EpFc73KVUCq8qku7r6d+8jKBMrPTrIbUWwdsK\\\\\\\/2nYpf1C1pNS2kNTKtwTomAoGcd\\\\\\\/iH+aBxv1KgYw6BspaJM9h\\\\\\\/sGVukDtpfNNnNsiO1cDQma4nlKNq2DXYRWoxaEI+iIBB32mf2JG3ssCxulXAbtvMCMJGig+YCUiUT8JhhDrRQ7GbtkNE3fsjkSgRczeG7z5pckOF4lpI8Boqy5Auk4Lz+fPzigNAj0V058j9M8yEx+9pxQeqU+jHdMOZ54K9lc9x7MD7\\\\\\\/fedAewyUchAPgxZeIegcdxaB01v2+VZKntoGBSEmJ8MK2Gin9m4bSUyRArb8zNh1WIDmdVsUXSNxwWpwicyYUwMD6GlFGaH3H+\\\\\\\/ZJj4wmmQGhKL7fngxvITJavV90CcE2d9Wh+V7h4y8WTMtWhGIUfiZOw6tbHjwilNjD4qCbVOGrib3NO1D4Dt8tvJGSGn\\\\\\\/hFDfefxBrQfwIAg0FF\\\\\\\/+AeUT8IUfhGLhMLucaypdxIN60\\\\\\\/P4bCnSDGTGo5wQ9AEnhIDdvGpa6Pv2WAGu6B5eV8WfJVKOFarUp7\\\\\\\/5H\\\\\\\/D2kpLbBgHMmXy4o8EjJaHeLHJrXLNGNcg7xjBrbPuxGPpmHJ95raPUkVl2CccEhn8E7np29B5dgm9OMtEu9paUh3+JnHZeRlaZyG4tT833AHDWI\\\\\\\/kHVIWxDkssrSnWEU0jEZTjdJHxOW3kssKakGHE5KOpxtQtl5guBCdXf7trfR26EmjKXdrK0oOzN5ftU\\\\\\\/gECwHD85HVDUNHqcCE99rPXb1QhUGgxaLPZo+iKWjinMSgxl4N12wrTeWcZ0PlKpPaH0SPp9LDYSTj33P74hmusGaRGjM2HhGHDuM\\\\\\\/matmrzSTZOZxsZjXB041PwnEy1qyg+fV1rAFqsqp+PMLU83Bb3a6+xiBWuHDlv3S24XvytiZpVQn4QNLTnQWGcV2FqfdHr7SfUzmp4bJBBwpOEVGlp8cVSSjoSo8e6pVksYOZfTvu0wfvOxlG2sa4ThGIv38hegsfrQj\\\\\\\/Hwvn8xuXb5pBZHvk\\\\\\\/7M9Lhk0IC8Spze7evXb7ERn+rfFO9ugZw4PcKaX3Lv+hqIvTznJ7pZ2qDfQUbpPVVdufCfrZTmdfn0yyyKPhhgTdEIfmZSWb7q+cCiwpHgGe34EI7xpGqwn2vSHuagom39I1J6a78X+2zbRhytMkcDOYuhgEunJE\\\\\\\/6mO2XOgxVfTFG990vnOMvaQUfxlH87+BOk6GBV0+IeKKlZppLzrht8I9XVHirtSRyqcNQsTq3eydWijlbF6cvN0I3\\\\\\\/n5lNjui3dDwHp9dum90tmA1Y310kykpS9JGmyoFnIaG8g9LxYRbQeGocdeTEPu40RuUyOB0VPLOaOnArKFrx7WOlwY\\\\\\\/4BT5I9OUqfGJVCLu+2YYWKJGuP59QaAMoR\\\\\\\/HID3wMqJaND9GG9Ih1V40QsWXveEuG9qsMhn8RtMm6Bd2vHfALrLRABlpcPMQ3KR1ymkOFU4KZSXxvUkYQC9pgm3kl0W4mUJa30soLKo58A1IwW1xZsXUuwyuTLRwZfSSs97XR\\\\\\\/993vAdCGeSChfm8Q8o\\\\\\\/TmnTCfe0Ip6ZUI62ySGzxBvVGGpV3sXE53wIpV5qqUwT8Z1Oma3xlrUIQ0ASjcOA8OQvw1p+0fZ4kcj+e6TJB\\\\\\\/us+DVMjEVHNZHFtWgMW3aQ7ZBmT2GHwE4+gFDHsIpcaSLyWWkCZ9LxyPqwo0HRXm1+SGMaw4pY9jW9a4G53woCKM6tglzmqtPBwBQ1b\\\\\\\/hawrfTCiDWZ+E\\\\\\\/CDeXSgq7Wz6zZKGawA7iLkt9UEDIpgNXl+CxIVmek\\\\\\\/pgnaORW+jLWINFlV\\\\\\\/XNnV1qO4h5UwWaCh\\\\\\\/Sq71SsarDOc0Pg3WUOP4kDLKCsMohNTQorHhZL8fAdFWpM+WxF\\\\\\\/UNMzcmvVqDZcURYYZdzogdmQdqsa5Z2WULvgJ\\\\\\\/Vzi6AM\\\\\\\/0+znbRO5CcPgQdFnMbWu8wTlsXs4PN\\\\\\\/3YdfNWUdWbiYH4IXBMnznch9KG1WqSzxE4jClQLj0dYobWUusdkyPmVe34wu7MeBinZ66Kp+swcm0ryU23L21Qryqndtt4wS0srjrl8yMPw1YiR9ijVICWy2AQ6ZxXJfHle9rdKzDAxDkNerm1\\\\\\\/f97m3bvQNnWAqVYYXp4oIcawRIbhbrTEPTIId7sFXIEM4jGlq2gYuGcwIeOj+ctstWLGREUHebiI5K6UG7MhlxS2mVrdG0d8CMbgLyqz6TiC9zZxjBDBm8dSAqP49kOfv952O5LUEATzVSgD5oZfXQu7B7UWRcW8KvmH4i7PRQX0t2w9d7RcQZEM+kEauw9jSr68YzCeAECcB+HlUMafOO\\\\\\\/ZgdHRxhsxDmXNtscDDCehZyABUgi69o0w+wRjoCUmMfrnmO7r9n8WZ0P1g0ZsOUkge4cbOkejxKH+VJ20236I+EeXfGSnq3kFRTecto67WMLztI+HTtUucNJbpTNkfX+Jh2UrEu7p90oVfTPSzdcbiRPfQUF8K3XkF7ubKTHx6MycejffYbIQ7afi8uGa6DsHL7iC3h8UxjarvObBYCzmnD4fFNR5timVbViNWY0f8BITDjcSN23zzEt5b\\\\\\\/fLv6wr8E4Caac8A6l5pqy0wkJn3SaZ6rK0fBKqf094K+JVldnzO5R8NjqU8yL7b3jvwz0K5fYdqRnaewLZ4bs+IGXg8jAevIU58GHRZNPpU5\\\\\\\/w3zSlAlFckwjkD+IO8ARBhX+RnYv2EJ951GJIYFPLf5aYVYSgiET4ROZLCJmsVfdMmofTxpxx03F19OO4TwXz4ZlUi3ajZ\\\\\\\/fZcy0tZR8aFnG4e69Mh1\\\\\\\/t\\\\\\\/P96HCztBRpBfI5YXKHqAhkNst2W0ce+bjJQ0ajtFG1pGI6RmXKXKFBopWVBzTYZGxrvGIgwCdGDWX02W8mkyykR3I88qXlULfHvrGRIKC6BMwsRrfhMeFDOy8Ry4ptOl0BWniM09oe\\\\\\\/JIbIaKTBFdZ2wpNSljMdPcxR5W6wzsKtv1+5e3HVJ+0GxMMtMP1Je30wvd02ywm92jBQ7nxPmod0cpc4D5oUq\\\\\\\/hEjGtK93N0RTG0faB4hBtxLdc2zMjZRLwCP1juyTdEXiyuJORGOMVQWKaWQEIhJXWkpXf6IvT5h5C6L3H6obNPTNKGzzsiouxoNUiifWG511Q7pa\\\\\\\/0ifyScqVY+gOHbK47oBLULZKlAGBU5QoeZVm0mD2Ru+OyP32UFDBoPJ\\\\\\\/+\\\\\\\/KiYYZ9hTsW4D2\\\\\\\/w7jY\\\\\\\/LvXmB9m84f1z4zBrN0TcraAfOYh2qN6FIyrXDfPdqjT8hZn26g660\\\\\\\/YNmJDF\\\\\\\/GG0pjJw+2FD21w974O3L3Mpqd\\\\\\\/BnP2e09W1HX8nNbMWCJrNe5pdNMbxoJmqcIcH4MoMl7TNwXJapqf751Pi4oQOf0zFzL0CijcKxiho\\\\\\\/rk6MGC0SmxwSQ3MWzx7Apvm\\\\\\\/vixQTFi7JYfghCx64lGg7IkzzBKRfXU9ks4idoJmEonX5p+4aT+IXNK9Vb9\\\\\\\/It4cWvFs1Bu1kX1UAh8tb1wJrUv\\\\\\\/0C9TXUzhxXQPPgLJgHBwVC\\\\\\\/b6zOkk3xWq27jRb1O5dI9ZVWDi2W8x\\\\\\\/9Y4FUp9GTP+vMTq+2dGHOebFtiQGslwIOuUrJu4\\\\\\\/Yl\\\\\\\/nJfyafYnOFdZwnF357905xwOFyGTmll6jxfzfjZztnwCB90zNjQyebLbwo2PrJZ9yYc57cUBTXglw+gnGUGMF4E+i6ubJhWO9cxSghMFT8RKl0j4IQ2Kn5JgmPLtO5u4CChtt\\\\\\\/axn\\\\\\\/7mK6AarOLtKhj+o\\\\\\\/LkzseeYf8H82O8ekM+UuVFvZAdhjn+hKA0iqsT\\\\\\\/CJWXC2CpWeReTgtueiDQNPObdxwlbvyOGbA4PS8nvSHQnyP4Y8U4ujShmwf2jAZhhAzuyuTrPEC\\\\\\\/KJJqlqSJs7IKsJp426UB8eRwmAW1glqHJLetxqNFdDe6vFNnSBhp0UyOMMoOM9duLaNho5d\\\\\\\/LQ1ZDg0M6JWAFyBFXzKiBt6vaU0HUwgOw+uJUc+cvP1nxRRgqe8ARQKGEomLJ76\\\\\\\/rXyyYrS3wpH3NcGs3R\\\\\\\/7F8skJX\\\\\\\/sajTe9bPGK6vBJG7ho64OJ\\\\\\\/Sx22i3gt1uWpKBBy+xZw51+vKhrMCHScZq607jJHCKocZt+cwRglxtDmp8PI715omhXOobg4K1POy06Ui5r6AiLeqwN2u1EfHKMH3P44hRWS7Z2MNJaw8Mj4xcwD8ufuHgdFtyBU0cRjbqtHQLmGXqY2IAdXXuqkfksS3tItX42VGKawURPdcVUSKYKRipsYjOLbVOD2SCc\\\\\\\/mzP54+WEy0tR0Cn6RJMlpO7r73+s2w3c9apn1lLvbHYZS2CV4AFllpNKRQwFsjeU+pMYvKOaNqXiMj50tmRlmaNAIValepQZ4cG4vGvPiTJqpvKKLThcAx1YHkAmjrt5dXQu2ZZPXWIMGRiTJbs0OZUnLqlfIKmP+VQE98EEAC\\\\\\\/2g43cVcZ0choc2xo3VWGqzoY5\\\\\\\/\\\\\\\/PXgHABmA5LE2SjebmTgQeCJc2ByfnW2zqWnCTwOJcn1hsQGe3STvuDu5iDem5Yk1kxc\\\\\\\/MtJ+P+JXrohXZ2Ye0P2Z5mEk3x68+1eAACmvWPfEd14ATMzT+z7fHLLV+CNejMY8fynW8MCKIPx4aAgUJCfT68McuiCXifzacpxPxahxircl8s4Xtr8t\\\\\\\/YX7j05Hk4H8o1uK7+3TO5IFbFgOT\\\\\\\/qqjE3I7jzBYUaZMM9xPO\\\\\\\/FdLzYwK8AJJ\\\\\\\/BFa3v3MRBAEZg+iiGTWJzgrS1nVIPLFcqgAQUpzBMXuG74MoU+s52\\\\\\\/G6urlxiO7OMZ1gtbLoRHko\\\\\\\/I0Yp8Xk9cOqAzYtFTx1llfkzihLb0QiGcl9BfLG\\\\\\\/balSywPtW6HGEULly0ypTK5UOBUkPouN8FwZDq8l1QafhUy9CTJxcQsverNInCtD3AlDEoxEQZFBFnFqXDfmuz9doijLOmYR1etAViwbOyt0ZceBdKW57HBW7ZDr\\\\\\\/OHmrPPjHJgUEfpy+U7DKGiagDMr8aiJxlXZmtuDCwvHRCWKM5YydA3Zn4O48HrJiJffh0IPg1wrEnMautu+syqB6nZ8UxyqiqUCq91WLCaAx7XEZTCquJSWb2KFCGa8fA\\\\\\\/ejIjMetNdvRkC7T6s\\\\\\\/+J3NjruA88ZFc188OzxH+1OEKml+UXvZy9DXJXAt0h6wNFWWOOQ7paZIUaDjSz\\\\\\\/yzkkbwzBFfkNwCLojcsagNiEMKAs57EGJxHMhtY5TdpdKx6QWqSJmPoD+9Eg7J58Bj3l4eqOJw3TtXYLagvQ4bwHhcbR99STetXSkUcr5irlh6gzzNH2WAhzNRgN1ndtYlVyU4Ck5lXGV0pk9KzsM2Y6KvkSQsQtbSKIWIQp\\\\\\\/chJrtuKRopPGo9euQkIwusttip5xbDDnaLfU\\\\\\\/TB0vECCHTdEcSfiZ5SBKnrPnjzU7cx9ioW0RTlEDs5xzM49vfWIF81\\\\\\\/oJtevVscS\\\\\\\/zwfKSMsE25389U\\\\\\\/N3Hr5oli2WhJfxLrFfnsx+HY3hhJln7FsNlBCk4KbZBoeUHJ4C7J53JnvKjYgozHj4jdkVjyDTfDLRwfcntbWpVfConF5Hb2kYSi7m6UjfPtFtAkjzyR2wM0VsOJpWqPvM58bKZqRJNBxDaL\\\\\\\/ZMJb++NKoYi0S5pxGjc9xiSy2cX9cXQ86fuIHRwR3eqAgD3Ay9JmzvEYJmpqBLb\\\\\\\/ESmqKYvks3Fvkj1iED+YrgrBy84\\\\\\\/O3A+vLP2MhP\\\\\\\/B9NARWQn4abs4hd+Z0q69nCW9PVlXz05abS1opyxZfGcEaR8phFpEySshxsa3WabzBXUsuMiIQNaQTbUOw\\\\\\\/uXrr3xtRMaWrHPQ06umKJu8B99XizxBalUNBp4uHMcPvaOxzzZTvoOAwK8B+Hngy44jfT6bXgsZpWIAmCmdFVyA4IVGQDC6vdjoAfpaY9Vvkoj2DI+1yzmQlE5uwe0yp2g69yCLC4XYpEKcVhOHrzuG+FQeB0PXGzj4ue1oTYfKknU+7+AVYHFENAtIfgTdgW92Es8cX2gSXkCJ44WKOsGu9XQVAeBZos\\\\\\\/wqqL\\\\\\\/d\\\\\\\/R5G2sijOhJPIDC3T4AUirz21nQ43QSFzrEYFLfhXs7QIbz2GeT7P4HFYpetEhQvnQBED8+g+1oWilmaUC1HxlMZ\\\\\\\/nL6sUfutnGZp2bMCPfFt77cUTJnC31IM+kUEDP0UZQeDsi16qfDffHzmSc31i9ZrzHY4IxvW\\\\\\\/+3cT1HzNZvojDBOQQwDJk1OaLMj9EiUjOhu\\\\\\\/f9UWyP01iHSqsWeG0YnUjjwDC1t8PhFmSK\\\\\\\/DPh7GlCwDW2qxHX9fXCe2+XJesoRfMxg8+GGEbMQT4ZTjoeOOxqx3ZBznye3CKyyyPnhFOE7wniAtHtorvt57ETejWhFV\\\\\\\/fx19iprQhXpsHe0EwyzcFUweSEkqldZXoAbAodXDIDtr30fsSWb9J2Nx0INlGtnzrZ18tnGZZALcZ1y6wRz0vBoOhUYq9XKeMy0Gf\\\\\\\/jUF5iO6sB9NiQIl91dmCpq2N8xgpEmGEvMzXVAr7rjLG8kn3QPZtYHODK7xFeFj0IJCY7TWhYSZYSe5SiBdKEl46fMYJwdFosVgW7iTDZZwGQU6T1k\\\\\\\/36O8iGVOTS3HtX39wOTn2qNJa1qMZM4TYRGH4uGFWlFFKsv8u175gYoYHLDURIpdm40Og+ZLLUmwpRckb+Xsf217xm0nnArvBcRJAXTiiVLqnxfAAXbASxMDGqwf8Rp9OxGLN2+KcCDHH+54+iqyvUyZa8yANHcTOWCIS+hXdlAZ\\\\\\\/lZNVlMZUV1rqPN2teQRosvzAE\\\\\\\/EMjYQh+ink0xYnHEn+4b34skfKVkUJSavy5evLfRstYmsPIzEx32gf7z4HwjfIIHutjDeeBqLxufGpmhwPjGMNxo9Ho8gwWPJSN5ij1O2lhdDLbQUeNpiHlOhGvv9KXsE8JiMS28eUy6YhOTFlH2GBFVrOMxvvISjfBZIQzWZgsBsvRem0pYGkSgy6SZxAjV2SOxGHesg8JoXzmjt5sKeYLU\\\\\\\/EOMhD2r7l7sDuEKVaD9npA0IuWCCvJx1+BWEMfXks89O7LSaFq+7hqj32GijTjs0nCyaDH5M2K7Y6JnkoEz1R5Z5O+VXv8M7bD1iga\\\\\\\/UXlR2ATz3nioN7aFGlooEgRjjO5UmA+FK2QeviF9vEkRIxc8ctY4+XOPC+vBgwcMmTUXPWPrOS5j0bXM\\\\\\\/jjq7uHErIE6DO9fpV0joO+6BH\\\\\\\/qiuN7VOjSKcQrfMh\\\\\\\/81vBmQoCn6uFaVtyOrRy8ig+LWR0uSkpQlNlLREu69R3VcpiLvv\\\\\\\/ZfxD1XsgHDsGDrSU6xihP2XilmrhWTaVfGayjB2N+yOgigvn6uHj\\\\\\\/VWPTemFSGJ8k2TIPwf63eFMkeO\\\\\\\/EkzQkWAkubLe9wkGHXyza5rAE086rz1NNFMDcrXn5zCnGk9ZxQmmKN1MMLbPd9beKT9d+va7NMvl9tbjkoroC9S+9udqKlhjEEbuYhQHegTk6Oo\\\\\\\/I2wxLFy01e8A0v80wCIHaHnn65akjSjcLZ68pFVHBKhuHrCyN1eqzmUD0uNeL9HN1dsuRsuIz3X7oUQk+kA1PMFTk0aDY2jDquzEKd1UmuXR3Gil+lvajJmBUSvf9lUKOPyZjqBFoWKcGO46Owso8sdsHlr3IfcUeRuOOsdI18b47LCYSRm8OKloecTBZnNhbhlRKcwQjRq7kQJAhux\\\\\\\/tqsHwEEEOitzlSP5QD+Z5DwWGgU5a2J4aNhW1nAfoXOcGogjS33jn0dqC+TXI05DR\\\\\\\/TaVmMfLWMz3\\\\\\\/vT\\\\\\\/mbb4qurosYjVCWaptmTxe8PigKgzk3mJkzlQ9iQamRYsb+fNX76nJBn2Et19s\\\\\\\/eRtEZP1fgiH2mfO4c8mb3BudkJtJaDtXt5eX1WOXggfgwjNs2L3oPn5g1GgAvLnG8bwmarYOMDQgrZqnaqmbAmVoUIgFKTde8SHgx3ytUxbtGRsVexJjN23\\\\\\\/cdkSM\\\\\\\/ioP\\\\\\\/kTSRX1KiNUWsYz\\\\\\\/6l4se+OG0lo4Gv1q3cxrgV9Yd7v8J5oFUGDSRc5kmNBs1jW2r+8bqYUcKzdCfdi3yZ3r5AUYU5JDyFPyv4Uf8MiE+3AnqyWhCUL80A9N\\\\\\\/SI+\\\\\\\/llRXBhsg9bErCde+NUUsgMkFfehYYiM5jEqhbmu6WG1W6kMmdIyqBvVhp2N8KwLFfROT4Gn0tlrrfhyORBZ5Y7UfwZoCf8fvDwDV0TgFZ1KzsHQ2sZWx+IGG65d6p4KJQCW\\\\\\\/MGqaUYBrL\\\\\\\/cijF5dKQtKFHCAslOBazaAg\\\\\\\/Hp0UOJER\\\\\\\/0V8yuXD2XlVXgAR2SRk0yPqxGWmnWqwem9nTzhaTev+YHAIzkKWRgZ38EYFcDHHTdNzTnmcYzfJMwKBkqrGb0PZ2XS5LsNG8uO4knYecmD2rFHeNPg\\\\\\\/3h1tF2aY+0SucoSAN+rEB9mrlBs7gm1GC5GRK9EO8QCHmagZ56gcg3f7jtzjKzMWpbs5ftSZTMwwfppAEwy26pnhLxfypzwTOUA32TvKmVpVkY53vVMAg+\\\\\\\/heHBMWLgBxQ16RXj0pXS\\\\\\\/NIQnwT+Vt7N\\\\\\\/jx1ZRxOM+\\\\\\\/6GyaLEsJV9NrX7JtiYxVLIdpMs8Qu+p4x1ZYIU1B099wBOiHBgwoESXqtSEQJUxCQrXZqcfxl0P+fBpoKprENpkoqRxKbsxQ7cVzshLprNZZgh9Md2KGdac9QR5YY3o6Gu4lk2yOSdjh\\\\\\\/WEEKmCP0ezv7xEcitgY4gStrZcDi71V19k8ljyij+L3QhlkwcFnvTxT47mMjb088YogpIuDuhygFJWFHZzpiFWTyECs1V5ccLF07KZXNJYKXZlcWMTYe0v6Z3BHmprY2pLK\\\\\\\/8WYVHpI54zxZ9G1bWhLuYQE0rsyPMUKIWP3vE2Sz\\\\\\\/LD6ziZ\\\\\\\/PKmt\\\\\\\/QRqmA2Ys1Q2prdEIO126CwSxyFzkjdIhMrQ+j0ThLRZVmP7QmHYoqaEeakN+b+Hhj\\\\\\\/NXJU6oHGSyCDV6qat0OyeBMW+M3l6FxGASV4+pqj8HSzlxVui0RdRxzXAoTBA5\\\\\\\/unu5YHpbS1S0FlpW1Vd4aQxdk4KmLYI3PLBnP78HAblzOxPnXF53LsnjH9drgpdFNplfcW+waqJCXRmYw+spsE49fM16nK+vCFEChWWtqnL50J2agZgA1K\\\\\\\/ejZNs5Mdir2wzTANx0Ba4cXygxptkXb8z\\\\\\\/H1lNcIt1LnY7+TdPZeP4bDwtXB6dLq8YpH1aK+iPkFuxuYycGzYThzrVs6RYrjtmVY\\\\\\\/cup0nsMw9AC4TIcx5j3f8RJ6vP1JtoQRisEKcKpkgLoCiVvE9BhvLYtEM7\\\\\\\/1oMrXr8XJ4sJgH+1O8DHjaGvUIzCCJaHzrwQe\\\\\\\/jS7Fw6mOwlCTBFcO\\\\\\\/sZ1AlCTQH0W8h8SINz2qCVR1bi4yBvKatPW2oC28QW0qMRUIH1R\\\\\\\/4RjJwPVoglD6YrDBzenqwYlEp6aM5QW9Eg4D5WQGGB0Tm5pN8xC+k66L9Z7KEnLnDUuEJLDIZJheb+fdkap63TOukujEjIuMLx\\\\\\\/JS\\\\\\\/ZoM2w3dFNmIJ4SXrLstrtnfhvG50MbuYB+oAVyyr5UL8+xDA\\\\\\\/V6WiZwC+eWdQvPDv+xfuTkCN5WsDv1SK2nt1U6giEnro6Kb9SDjiDXfSh3i5lA5M\\\\\\\/voW6jfYShng6rlDs\\\\\\\/7JiBJu4RmAWE5Ol8sFC0\\\\\\\/ubokwC0kx0OLKJm6m5C\\\\\\\/OhKPXJRS1JyAVmthZUaBy4IWMA013k2Rt9PDbmjEEA6sxCMX0cgnQRbxhEMlaaQI5F2U6sbLyKl5viV6MeCCs1yT7KkDMpG8IR+FQ7EhuuRmvGC7USBcikx+JrPT3hbTrbZt05dpQGm1En55hjpIVvaNLUJftnh7PaBgeOq6Br\\\\\\\/8yA7IjUoXSeK+GjhtsiS95jGSnxeWUlFVWKSN+U5vg3JUIe0Woquc2lsdOjQKxlAQtfVI6Hn+VWuOaJAIftWDEwL7dFEmaEtd8wPcw6M2IGINhwsFf3QSWUPuRCVkieaGJSO1krsIRX+zavdEVbxtqQUFawnIlBI\\\\\\\/JXZPAO+rT8M+egAPSb9zHxq41LNLQE5wToEudXFdvHb1BSC9bXP\\\\\\\/60NmRya0LN0PyFKh0b6SU8OrrzRjT9wDCjvtaW8X2eHwGMrMhk0Hcw3iP7L7sFa3FL1CGvlT+HT4RGgmySMLD1n\\\\\\\/SJWc6UhP58jvDlElBA1DgCmbMJVwtwlqgra48SA1JYY62X+d+zhFrdMYTdx4RRiW3RtZv1haKnHWYfEIWANOljpNX\\\\\\\/bGK92Nv84dCCVx34ILU17quR\\\\\\\/kdwKuLmddZc0nccG+P5SbZkHdJ\\\\\\\/doN90hfr\\\\\\\/oK3Z13QTS6gaF4TD0ZwHc5OoIojTnu+oKmmzUJ+Hfi+PaNZ4xv16DlzkqsGG5DDPrPs5davruBLfFz4c6FiG97FjKwhn0RBT9dDNqw5WOFNk8RyXAuo9uIvqrq5CaQFr4Z+QPtAA6VSppDOcYKuhSR16MiBmeSfGQYHXh92O7OcAvcE+2MBONdUDmdxOFOGVm6FqHtMTuOf4hZiypoAIiTfBJ5g3qSBYVjyBR0+HerAOb2Y\\\\\\\/0dDgVnrAjpZ3xrX1fKInQo0WD\\\\\\\/V7lP\\\\\\\/dsWTEcCwMS4+k6QmgkA9ePbL87fyQFbXKXtMGTqH\\\\\\\/P4eCBBJEs2v3wNq+VvSotyHrG+nIIuDSgCy\\\\\\\/FjHpd7nM9gfk7SL+nTWUIVTU6CJdQYRvseFnUDBsIc4ArWdskrGic3lyyZlCYeaGhbw6rmnH6m\\\\\\\/ngtzs97HViN41o+0M4S8Velo45kDpGLZ328La322WhEAU+3HJUqKmDsn35cqu\\\\\\\/G5yd8DLzGiZHFrkbP0GSvTWHrwwSk6GjpKJizQLkQnKOMsu\\\\\\\/+5Ref5JgNoZ2WfagZ6e9Y7D7e09yxLAVkh3svphFPLKZ9\\\\\\\/acmebWG3WyUJBTrqophbZJtXu9MiO0d4vQTADJQNRunZ6dafr5WxuwhxbeYophI6C6lwTwmyGudo13qhjKT1T\\\\\\\/+1+xSha0FgOC6e0hyD4LkUJF\\\\\\\/r2Y22ngXyc6jLBRMR7ZAyS\\\\\\\/4dssBMfVOlFBVwfitFxa75\\\\\\\/OBwpVe\\\\\\\/ZeIlPFKm\\\\\\\/K9+G68KZdwc4IXJYBYH6sVId55Qu3EOvD+N12QKHI22HywdYg6oHyiC0NYQFCG6ifRwMfXN7IykYoYTEIeQ8epdUA\\\\\\\/eRX8EY+RAnI5XlrOKrUEBgypCdY0Fcn6O\\\\\\\/9AxKsLB5TL7txYyKhKhNLSGfH86ENd9HNXVbGLAAwLJjxFDlLx7Hetm9xEK3vQY1c22ZitEUXIFwwxCIX+\\\\\\\/Y6qVBTdKfKnoTrRKgDqpsm9Us\\\\\\\/9Fjnez8\\\\\\\/tcy\\\\\\\/PiD2krpyLWx2f6861497hBQvgK\\\\\\\/A\\\\\\\/ahu50SOqEwtKqFCzD5uXEUVs+1XWbW+QmJEanBzEOlJOxNQi+u9WVZqHwlC+c0knhG4EfosrNMGBztYMqCRD4FpQ7FOV7BlHBLrXay77vU1sw1ZjpuIL7RpYNZIMY9kMwH3RAYyT\\\\\\\/q2s66Q8p8WcDmQm3kYyW5nIfBe0BlLleiSSBuG4\\\\\\\/7uNZS5trtYJrTDK37\\\\\\\/7shuJm4Fd6mMW9qjMet6UM\\\\\\\/aUExhOzc9xfdANqyOz17QhFn\\\\\\\/jSzGqfQcn++Hi71RVv8ogA5HqO2PWMGVkPXKw7R9BwXne8zj6\\\\\\\/vJMima1r6U+MOT8t2F9B03FZgqWrM\\\\\\\/7tuqyuQa\\\\\\\/hNCwaIKwLx1M0tpFmTIndlZHCFKIzrBXvtH9MMU20R4pTGDdzyk2o0rGjQdtDZqFKnpyNETHwEZg22ZaIl0Rgkx696zYkquHJHYgxrH7bV2y+uXy6q6l\\\\\\\/PvC3Yf66DyJcY60S3wmIVfXrb9jGkYgHM3Fhk+4n\\\\\\\/vEdHBaMZOch5bx6pLqPYsY65AmntNvxVaf+SAzvhgne\\\\\\\/5mpCRANxoM21R\\\\\\\/rXVXNl\\\\\\\/quuY8CO1s65RLE9cRz5T+vZR+iJmF0m+gwZG4qiKTo8k8Nvb5W8n10ZZI4OdKp0bcpnQxpBr7x3GJXvyZTcMvLb4\\\\\\\/0\\\\\\\/Y8uCnmp5xmtXnHc04\\\\\\\/z2VKUI\\\\\\\/YjloktTspkEAR9JLZklpCbGDsctRmLtAFrYG9HNqiarI1cEjFJy0vvz77lW9VXarZwby8QTZkyn41NuefsYKkJkpuyrtMitIV23cWvY0YSlnjM5Ob4apUU3TkSMhIMefRjkW1bvfoENDE5Cz6dRSuJEf1b8S+NU4V\\\\\\\/BU8uRDtTGtv3LC2D+vwmkRGgtc5WGhZ7Q7tN4UlU+vPJMyJmuWngIvtUBserUg85kw+F\\\\\\\/nS3XHQrIM6NWmfa60ClQA7IpuoKhY09Q2FbAR2XfO4v9\\\\\\\/o+OxLMDRiFFRyOGL6sFYMJB143Eiy0COKkBA2ZDhn8FVvugVmwFJ1C999RiKlWNVPrsqN9imvX6wUvB7vK1o0QX7p6o7pBVW1MuV1QzEsDkYpjVy9gqVZ4DTMP7JziatYtlWzqTBFW\\\\\\\/yd\\\\\\\/H2buR0jXm+yPqssxslfTI8oMvOf5IIBwbGD4uhSDZmJrPNyYCGRgbmRxBjtYukenjAU0IMjzjZc8wc6q8BClBcFNkb9+71Kvp6qmSHitl6csvwEIueHq3cZpPliytGbZP\\\\\\\/pJqVlxyRT1EQsvmT1JQHVo9O8CQTmLn20s36Ytk6XupFOZCDiho6GRUbD1wcQd1NEeoYpkcoFywijGocU7dD6FgMXF2hlGD+Ve0jJk7mw2fFiKf2gPBw13GAbxnC6p\\\\\\\/OW1pDlvuMZrvrt0BSTp9ub+6Te6NIozUWgWNDfJuZ7C369HIvh0mfZiKuENAZ+gCdjvjWybKFGaqs0e0nfQHBv9pYbJYseg9Kg5u9AATRawksxBzPin2sgWl\\\\\\\/NdBlHWGhwFUI9G3oqR3rUD2N3RpkdEw5JINS7yW6LES7FfAe1bFMEh5yqsqQkq8ZQe8uEWGOtbhPNlpHQ6BKInI6p8ynQouaiQGFpAd41KHPiCUA2ZLmhf3mVDE2eYcO9XgNIRr3yAhUUpRhu9n9U6BEAoKgZk7b+iowGAG\\\\\\\/qCyfSLHimYmNH0VifcyARP\\\\\\\/kbA8WPokGuE\\\\\\\/Lshj36iNwBACY7PneV2nS86irE5lyxyqQtCty2uVAQtwK5J7MntJzzuvcYuyGRg6bPHw8dFwf9zOOgMH+fjRooBW4hZK6+\\\\\\\/3rt8Ti9aP+t82E1c\\\\\\\/VMleHbpezDTXG50BtyB6YBLIuDkNTRthzqgBsjnBTIt56OZAuo2Cn3ezi5dO0xH8ZvVIbea\\\\\\\/an9d1J4LRTEh0jXJqYUOWycIofaeap033dxsd7aFSBZHn1YDUs9ry54Kt27\\\\\\\/vpAmVnCbk1ASRMeomq9XVlsGL0LEa30CDx7iP0rwJObwzyB8h8jan53eDqKi6FrNtAU4MUzE0Jbq9eMRpnfXLtFw7fkwBdjdEAZ8GLkp1IqORod1eRGrs83qUvTTHrqIIiP9gDF1nzpcc5vQpK97a3s1dggUunO169bnlot3vjwv9bgxDAmMtzIASvjc8UbUMPpUQnyMweLxRZFpvTngAV6K4wHSCYM22z5NYvi7AAH28dAjTXKO4dJh7ajSDuap1l6768s\\\\\\\/jfq9FwS\\\\\\\/KOoFBH0pQSzHXeBDha+oBzxSYkfwU88z72IJSKgBGefQ10MAz6uD3R+q3wQ2swr5sNQ6ck+1ZPzgcMJu3iefCMh1UYS2NmM95UoFUxnv7966BXRmiRsg++c3JfjBDkCb9pP2+pgLGMEU822xFFVKiRlUn1\\\\\\\/z4zps2pNlawytWVGFOYPzm1hRkZIvnE1NuCB5TDo\\\\\\\/6C+ct\\\\\\\/38kQVU+OoYA5M7K7aR9VaOqJSlbAbJTKHgh1bIxt\\\\\\\/vO6b14SGTx96aElpFKs7x8B5R6CWsEqEssD56fYIKzsO01wBw0qny9FkoPPg1\\\\\\\/wwhfvJJlUYGv\\\\\\\/Kgw+UomeMzAkvW9+eejNVsfhGr1mtPKxO77J85ghuPyGQodDQ9SNU6mHIfuN8c8UtB9Z3Dojv+OO+DBvkhvkKmSA3cbXHx04ixo5RHvjQXj7KOi8fjs\\\\\\\/gqzqyqRHYUL9oqVMLT2+NBNjNjUGartyQg2+oiWbafDwXRVFVdfq5kTeOXBy5dtkAQAWC1uev44E9HXs3mopN9lE\\\\\\\/sq5plz70gQ2injX8rVyi1jMGGaTaMYWnMFxkBWbFiAumzb4shm8MEjSzUSUvHKxTA6a2n4l+60ix+o\\\\\\\/GAbYU9UrMoAz8XeP6NOROmRqOCqWf3ydslKuKxcT+3vxTlhdSeUNm07Ug3CB4H+TibQ4DwpE9xx6c9T4soGuE54dV5DKAfzZOMmUWeu\\\\\\\/atfY1zBlTEeB79iZxTMzUineutdUpaUVgMg2mXMDW8krzKB8OB2hiKF1ZYC44xAJ\\\\\\\/hf0h\\\\\\\/0UuIjToiV7613jzFRtc9CaP3PzeSDNdzlhWi3EaBFJQuiyjiZXqPcc2QUaY\\\\\\\/YCQ\\\\\\\/fFJSqpp1tJYsXYt94Fn2DXtouNnzzhX9a1sZc6HjRqT2VYcl4g2p6OOsswm67H\\\\\\\/8GvWvU5zGJSz27Sppb9lzI0s2MJAo4QcgJNY4gpd\\\\\\\/mRj61bKSxe\\\\\\\/TyKqe63cHCaRsbaT6hGO8EtDCNyM63HAV9c9ZHk3trmze7fa2CWLpvTr4rWYQ0U+Uj3zBgBJGlYBKbZNXQbKG\\\\\\\/2zcJ9S27NJ9vgjGAtQWVmR4Im3yFSrjlJDFsDcg6FVIlM4Ht6ApSzjioWxdcPLvkOpFso6U5rk886t0+1rp69pM0JhtfmpfeD07wBi74MXFPNIx784EpeCPA5E9xv7bVnsaz+L5uCZxKg8QX77Yn43X5R03t0QloKUKJ\\\\\\\/VNvau95bd+zC47ZpkYEP9X6K2Gbar4bqv0cUH3NJmao5Y4uEXHcEloQKNrVypQO5qZosQ2cMo7v2md63vIkDG7d39sd5VW7CIr3g2IheaL64FTWQ1LefDjGKDnXScAYkf1KSHyyVfHjL7qJWiMzxFKrYpRhfSHW+acPdPqvhG2yeJVlt3P1HZ\\\\\\\/Dd4OVfWdzdqljHke3hypL3\\\\\\\/\\\\\\\/n457si2cNOctmltjV\\\\\\\/2TO0Tw3em+JkXsEsXuEHCP+CtecQdtaNoPAQGF79fYXy7NFIoS1mDakNWIR3DDJ\\\\\\\/\\\\\\\/s0+V+YQIgI9BfVPttv0OkmqMH\\\\\\\/6+GW0IEIpXBnsF8N5nJEMMFnR6Kj4GLaTjHWmP7dLIJkEQNZQ1KCKxvPdNzb9W5jR7WgGHFXCrE7Dde9YBoXNsM4jeQVYNuJGC2YQVgXC2225zjD\\\\\\\/C6U4231CQl\\\\\\\/JiYw23HhDJfp3gNc2+Lhh1gW6imOuGMo6rWNzvIWMmsSYqSbSmJP\\\\\\\/NLxBFvEWtsFv8cFsf09+LMjaeej5qdhEB6bKWJSHCVfAz+T69BFNT+mXHEn129A6\\\\\\\/x2y43poE1zj1uGtXNxFUZocamhveJi4Gkd4hNhTZru\\\\\\\/ZjDI43yH8zyuiv0RhKImuhW+9ewO533UJF8VKs2RmsjJhujuTVT7b1BBcEwCR9cnRusjz1ldPvy+I2vNCR0l+E\\\\\\\/sF+1iBVTlCB9pwRUx+OFIhBehVpCUmdTuslDNC3tyg1SL7b0ly5Fur9vIslYF9UlDEwDLF4LZrkjP8MuPVl5zLaqLFMaXQV8eykhZ4h9SVrdzUyw5wpXY1FK78XCiRq0zAfeMuz1KkGBdTV\\\\\\\/YosOiev2MdeGG3H4ddocHbl2335y3giIHVTGhDcyxRj1uNnP8ZArO32myVy7JKjgcW3TpRK2CqfA0NZVpauS3BVd4LCUggFcIbLuUFLt7NvQsS0ss7dv0BtdRN5gTMbmAdGEvXoyzNxC5ICDc5DYxGzJy2Ze+U0hkKOM+\\\\\\\/H89VXx9U93od3ziVp+fQEMTrEjl+qjyLpj9AL6zAreI4wbzD1D4lkqUwQmvN07l6yfhV+E1PqENyMJG\\\\\\\/J8HOMhulUK++bIDRgrYKajHnXhg0b7xw1iRm2LloaLjH6GJRo2cboHmZ3QYB3zKxhw9bvN8YNI3nMITjit8cO\\\\\\\/5oYcYWQYXbcnzyDC3HA2pYqzC4rGd4\\\\\\\/JwdetvIit4vj2Im4UNYPsoCRwY5i8DHSU3gzN9sEA6NI39pQc1JJfioudqmXPzX8c3Dqk44cSwgPIILKlSgYrOvsJw6+hOG\\\\\\\/ZGdcqLftGVY5cd1Qeljl\\\\\\\/HYIM5uMEXVZ5O4gGPQbz9I6FYG0PM4PUGtWZtPSNaAUK0bhGf5vJP5KFjUUpXuOZD9DtLeYVTAz7x+Msak+e8o3XK1WixCMUVucmzxDdUV3gxsIn6lNCf8\\\\\\\/jzy0SeH44oXymFW7TzJwaxNvEdbRCgEFvh6hEd+C4mdKQTWAJZ\\\\\\\/bYPwkuQ+ngMMxU6quCfkRuCrCP6FXAQ95CKYmdvOQd+kN+NgfrPfyRXSfJUb2J9EQ8qbPTHFT3sqQJ9hXA8O+uSO8vcPz39hzSyQVAdeYgWif8Bi48rs49\\\\\\\/dYKc1MJPc6TKLBXoarOMJpoV5ApA6RwE1HeswN7f78fMFxaUVsKxXi4NrOAKpg9+3FBduAXU9tWYKnc4TFB5rfvDty5tLsv+do\\\\\\\/V8oe6gDX2Xa8KrOPrJQA0H9+rBwRmeI5rZkqP2SPiSk7j5GXdpe15VxKHHdWY5VNTw5W3Ufq3bCf2r+qRmkHEsnRBym8vzuBiW37o6h\\\\\\\/Z+oMw0PMDSRHe9\\\\\\\/EBGNsjRDkreox3Glf72L8RkgSsJE7L2n1kG99SBPT2GD\\\\\\\/jd0jkiYXYYEIhr9IHtoOHwvTqZFOGGd8OQW0T2uHGIsh9Hrft1yg6UnQy2J3+msk5qd9f16rbLTTmbqIvdkv36T4OlBv4pz\\\\\\\/O46pR4Z83NkjWEFCjiLe9NJ3844iO7+s4qUqeUMhKyEaJCnRvaxLg9Sip5SESFdmCeYOWw3JYEVdZuVjF42l186tpZTvd0hlFvluHjpBrYz0s\\\\\\\/z8E1VuXaool2Pmh8Odoi5wv7P2e4jxO1ianiZeTnSM1HErpDRuPwtu36osdE3UoQH\\\\\\\/DbvXCI1eRYHaZuORSEdIbE1zIdc2PNGWcLgU\\\\\\\/dx+0pcUqYHfT\\\\\\\/3iWeOvIK5V\\\\\\\/kjOo8D6TzEjLIM97sl+h0O3ZDlSWjZmM031aBJPz+cgRqK639W6RFnWIBQ40tC9z1j\\\\\\\/O\\\\\\\/RuU21rj6U9gYIs57oXzklJ4h5oUX91n2UjJ6xpsubxs+eaj6kIa3EDabw1Ka5GGFX5\\\\\\\/YQgV47VJmCIw5jci+u\\\\\\\/vyRbnikzy9GuHWx7zSxWufHfnfZJpUw6rl0CnSufJ7W45vbw4xKioFlKJB+jo+mLwD7ftLWCzyo5cSiNB6urgveXmIClED\\\\\\\/goLmt+COIJiccL1HI6c7qx+fmuGR4PbYp\\\\\\\/WOZuw8zw3ZHkeSe0ByKGbvMd0ekrs67id7Sq6GbhsOMaQONTjbbjnivS5F\\\\\\\/nxdClElZOffcD2kR2L2Xuwqs0bGcSHPuJq+Yy2Z2qDk7pjdTRjHv155DDgZlBrdq3acAhEJYjSKjOwoC2MXEJe8XBnprEA8WuPjCVis92PVywFakNuAUm11r9V+cFw9+A9a9VhAR9uvm8f7nYYNUrm2i1CVaclWZl28gURMLTwy9RWlkh+YJLSuh5q\\\\\\\/mscgNb+fszGZ0JSnfMTzfHuMYgIB56ZoeTGsgVvkqYrE\\\\\\\/vOnnWw9YuiCqxsR28WmvA+CXO3e1ECNruPJ9yy4p3c+IAbCx4OagkAfhr1cVSviwVjHcwuS4M7jFpIKbS51VK7PpWkf8boL6xIRTqndAu2GiY2J+ppFrnkgZQ7w6ZDo39uJR7S0K4Xz6LscqMq+1VRTFtfQz2oe5vOHMnD4B3HOGPdGhBe9tY3uERL1apIUZa8fk\\\\\\\/dXJj+qQPX9yX9Rfb+LbwxIgaNANUpH2mdQhUiKIr+vOUXBmrpWS0BxmTwSRdAyIPKst64yrYMrhumBWR35TjSaZ2JUKMSp\\\\\\\/hpndEcf\\\\\\\/1vPk2P0WNRCLllr0Im3MRU2PI6OUqTXx\\\\\\\/Z6af3ASH+2QQHa+MvsbngEmA8L3FPKMrRtCBkJc\\\\\\\/RIMpFGw0G0n9fGm5HXidjeGDNhWIwnLPlSTGauJt3Wj7q2LT6h4Su7aUfDGwjB1m4+dNLYIZ3hAEgMzG\\\\\\\/8NqOdcCmyDlnaGogjBOsjiIPWkcTyq+ml1XsokM3jx79998GJrDHA5r6sfg34MohQpOLe08mkCFjce0WktM1pbGDbHEfnBq2xOce+8YVPRi4nVbl9sPFiznYZwbbh2c4C0EQahtBsg9welyuy7sFxGnn7S1E4B9GV1z2rzwiQJ0VbGROqr6a5PaJS5sKmwYcMYJ+xTDF7qMjE\\\\\\\/sfXPK3ia+HrJYN1KEFkBIDuGVicJKCRwW1\\\\\\\/3B\\\\\\\/LvI\\\\\\\/y70beZojCr0WYNLOkUpWRSHCuJdFlrmhOiiDWuOkLiFa9B8nwZeX84knYNOfNUbrLd9SbtJ1cADVsNlSOt2I6Dt0dJ95mUdV033yWrXAlzPwUPuPRSpeH02qs890S\\\\\\\/2K8DjihNI2h4CcwF6mVTEuXmUHyhUQQi1Bl9J0reTqZzWrJdLN9RMrrYp8VRciChHS43oDPsZH6Pv7G4G5uYcMsgwFTMaBYpi3uB\\\\\\\/0IUpYNsYRiSvcDG8out29ZUjgxfCTR3sdtdMVp62rHq3UTn0WQRfGr453zXisszm+5+AqJTC2hkJoUiaekcXvZXqSmZC8ZiCIIWO5s1KyjfCvIvO0w2zw648nZnxluWsusLwDHBsGbriYtRnlcHTDKsDyWnc2kjXixcyuMeaFd1EPGRJGcnInoc398Y88fCuCXYol+2rzEAUgfEasKHDL+p5eQC1AMyioC1vJD0Z4+oPNvpEN7xqeXU0g+ZvHYEod8wrWUkcypgfysQFanRl6CoYW4WMlSsOjQZl7CYX7zrXs3QqgxArE3gNxuCfMoTxlHHgPM6JmtfBlWCR\\\\\\\/PTQXcGbBGd9xcHhxiGhUFj4o7Ll328yfxC847qC0xNe\\\\\\\/G4NwErDB3CdOmCXwFqSMhYNDW\\\\\\\/1PI0t9cFkp+NxzGRWox\\\\\\\/oXma9Xys27PzuhVkW2QIP7UWWXG2V+V5HLkfaBXK9C5sj2b8J+WJ6hwPs0TMQ5ZGY4d+ZPUjR2T0bhzbBa38PXjEN42E+GQ0eGYIgRb9IxI312zVet3k2YhncR2BSkKCGhMEBrTz3XOFaaniqSx+CmiCIBvJbSk5H9evNH1ZUy+COj9jlv0fzEOh2ri+RnAm9ST\\\\\\\/8DYUHmhrSWbVCrc9ZwNNCtUfTvuPwVwmvsNVTflVsGGcaY8HiPx\\\\\\\/uX\\\\\\\/hXb8c6\\\\\\\/1g2t\\\\\\\/SozC43yhuoQmJDGQFNpE8ayvkVJpuQqxT8rWbVqCPnzDOZYouIVGmsZIAUfvVswkeOTn5NYMpJDWLx9eQek98C2Lerig9+uOPH2b8gYqZO0hRommccczh1G6YlRh4uPd2DonNIQboECv38nOWzTSGNoE0zo6DjYR5WkfsNPV03TNpWycWY2olOK7QrpsCjO7kuUBaLxTvO5xCvI1ZAhHr3IYFh1F\\\\\\\/CXfS1WzKxWq2g8Z7rZIbBwtYBkwSOm+7\\\\\\\/4D\\\\\\\/ativcHBEUeMZLOfYyYBRLni0JGJBqRqkNIIbk6Zd0iAg0KXNpNsx4i6XpQpPtfobFaaVfwzW5aBrDtzBs90rLwMZmrE5V\\\\\\\/3YnHHsiWjMRf5gHq0yTj263M+4VzfWGkiVxqGsbET3YQ5Uj9ufAqcCxfkuAA67YJxRyMqCdkMYhumwKrLekxKzOyfuym9X6eW9tpPhVu7dd2KWJ\\\\\\\/gPWv0ZvPRRgHwb42du1JKBwMj2xbCyEohz6FOf\\\\\\\/hje\\\\\\\/LTaOvrRda+2NG9WbwDOXf8kb5dAkdm6X+l\\\\\\\/QbmyJ38r4mwUGBqTByCH8W+VQywDZbIqOtu0kgEWdM6s+hTobj2jKEnOVEO+eyJ3FJxEXgINXdv+Cw9rYATLfLW1nls9SzXULcLFxSnoN6\\\\\\\/\\\\\\\/utoMyj+pzy1NjPHFfVdnnXkeTloIbEEQr+7StNnKz37EmIhU8ssbNgKxT\\\\\\\/mGpswDNs1zXZdlOlV0NvvlzGR9q7N4KVRCeOVIQ379X+feDirssEOWd60nA2H3xJpohKKNDP6GocjpXvgqg244Zd\\\\\\\/UY7BZg\\\\\\\/49w2bAN8ksYX+orUBD\\\\\\\/voW5LiL8mTi+HuY7u69uMwnxbZbytSqLNIzX\\\\\\\/rRw3XRsOdm6UQuKBbgF6CZxNITglbQ8\\\\\\\/dIKQVSSkIdEQ27tgIocbSHBKym5vWTNyI535rCQ02aluz5yMUlvwaZ5W3hy2VrK7c7jaT8MvWt+EKUUxoIPSLE1V3CGEfcli4MOf4wAkTrjw0R4uYePcQD8iUGBtbSqgU7je7a8cBaRM2LsJXLax9p3AGWrNtgWzH+67CSh79LFx\\\\\\\/Y1Dx4fNW+Bj5d3bjd84OO4T7JMlc9iuYL2+Q3Y3BGl+bJFLsLxgSN9\\\\\\\/C+z4MbhDEu5pTnD2K1pPDPEHFbyitxJrHjgnmj5HUIkAhrkznBL1zsasi81BOQZSsvGAThYmU4wTRxV9GsxPWISRaRRaNUgjpmM7T5HATP7eMpkovlB9RgBq9D8tI\\\\\\\/6Yb7oien60O57oOiuk0MTI0uRoIRbj2++D+Bxnphddj+2sc8IRCqS0y6E8kq2wL3Rg3MFZp3nEmjTeOwm3X0DUZWoL39aM3l7CHRdPucAxS3NErdNokR9JgG8AIWO5GDdWFcovoGRGOLDQEcJE6NboKhmv7fTgFeVj4GO43PmQHGSBPslpZf6VvQpcKOw+UraxF611Lov4phj9wayZl1rFjrvZE+KGbrWuo3bIMe9mQbovn0rzsy8M6VdD0Nzl9+AsvAWL29E2XOL51\\\\\\\/usC8ACZIjRbQ7caAvhuxVKoy4kw4IG+Law\\\\\\\/5tn3X+K84HQTeGkq743WsTsCMSkaAHx82zjADY\\\\\\\/iJFoc1o99tKb2G+ov1tuu1wCmGZeaOk\\\\\\\/Uny4+XXnjPtZePFc+XCcS2vAQKHQMGRdeyAB60epq5kbKy\\\\\\\/10EN7tP+Bpje72JaZrMgn2etXaVgbFwuJgZv7kFAOdlZVQvFTWx6561ywFvXYycwGYs+miRfmKNB6SIdvTYNeP6\\\\\\\/iC4Fq0vgBRs0UpnC7tiIWBiYy88ypSU56yy9fmOpluoimJM2DKRigr+rIbmRBWspUs65Jk2WdaHYG9bjWqLXbGImJldJGU1l24qMOGUw8ikRcC3FFzfTNbs2HopctVjpd2mj1UHQpbtSpyn9LvBHu5eiXrvj5tScsSdWqGi10Ym2M0I5uHcWzA7GlajNT56MgRn1XmtqSFvLcQoyCAnayB9UsBTD7767KjI64EQJuipVq9erXY8Wc39nfrtjxI1n0fkw1gP9fTIVCWvhZwHG5naWnuCePriPX2WauNYtf1EwqobWMShGu2C5UZjGChtpIAnBoBm5OTa1KZnU1Mf9rVoQjMi3XhFVgx\\\\\\\/1tmf9MmWNI55\\\\\\\/w2lcfmPz+gIkx7s8s4BwHsi4SdbG4\\\\\\\/6i2+FMLf0UyBvQqnJPQh1tnKboo7z2wCj3KzwNJy8mPHlRT+uWvQgRCWvP1juooZXl2Ru4oFJY+wsmOADBHjhGGPEZSvCTOPcxGVTkX4F25oWYQkTw0mGBFmvircMFAGEauhRZ0fghJv8SMmGfDV2F\\\\\\\/feAJmhuo\\\\\\\/RJLa89IPpwCOWQmVKCp4\\\\\\\/HfFzoFG8KDT7gxItRi2D+5KVwuCmZQKZpjctYfqMPGvrRTyzP5Mz9s20MnBu6WrIbxC33Umxfa7E5qg10dlGLVCrW+qB\\\\\\\/5C6e1N7ZuT+aIiDihy3fNcahiHc7TE0bOk5cO45e+CTiB4FwCHyHURAaJ0cwjBSkubL8+9+4yJWcZI8LpEmFZ485aS++pdSMk\\\\\\\/b\\\\\\\/+6JUSbMxlcXr0ke2tOz9WXfRrhzoNfwIwgF1jpze+ppE0gr02gEPq0H6fEUiWhrCpfre2ZfqN04oC0iX6lpgIxbWMhRlPEub91v77p2lRXV2HIzZVPgKV+E7HcfTGqLFrM73jW+zd4o3EqcL8UDv9Q8aOXgb2ee\\\\\\\/WeE0mt\\\\\\\/6ct++xymnFRezh0d\\\\\\\/sAnyTp7xn3mlJjCSdhfddB2SS8SLR0xX6AsZxnEdunKqnt6BIsmCQbWu7qT1\\\\\\\/7gvnxVd3PzktpsBXzlDW5gugL+GxHME8j5+z6HOPHt68LUY+PRH5BF5PlnNJqt5mTfg9uj68n\\\\\\\/TPV1nCQ0HSBKz6fBnPLXil6TrEzpDBE58s7CIFDK1HHhThHRF9G1ztzzia4Ig3fq3hA6agGQkGii1Tejws6bxFdJJxPbMNbVySEHF6PbDdZ9FmALwVzUjr25gGo\\\\\\\/zQv4Guq+ysUiEWuO+IoXOhIOKSWFrDcVTmbVVg0aMyhB6merYl7yImUlHhxydEidJK3y5C1mpGOBHmnmXnBJlpmiDs7KijXoTliQPPpUdTk0gctaLvB02fHLkpltohUZpW6t0KReM\\\\\\\/WOGpRbN9ztYAGKlZb7yDrAd8C0IwlUXH7noC8tP6Jtzvj78m\\\\\\\/OoL+PuvoyOY7rYHfcOni7UHPq2oXeIa47\\\\\\\/2tWzG7k6LdTfghzJcE6oxBW5GTBWyDT5N6ff1N6D0ueCtTgiWpVbuwpPh\\\\\\\/33kOL5HmmU753fEefWs8XrxZb2LCNASW8efs+VRqN4A5zZOxrvnOLEgaiK1b+IWZEOmbdnhp9r\\\\\\\/q5rHIh0hmj46OQeQnh1joPO4hQAG3nRscu3J3bttg6D55BcLw2XJin7hf\\\\\\\/mepsMWGaU15ZCHhtN0O\\\\\\\/drkjATFwO4Kxgd42CQxEWEmqaScbKiyrl0x9Di2nYD+K6qstkT1poHNHKEKYSU2qGi+jA3Tfws+fWHWmI\\\\\\\/TzzrnelSusMy3r3Wm6N9qAWz\\\\\\\/W0qXZJRgRpawoBoVYrG2EI317nfjUu+oE0M9GjIFtjSByebzDtrXkwBeq5\\\\\\\/Sh2BdWudwcsboPbJHPGUfkoJnfyutLaBTh2QlS1Nlw9H\\\\\\\/1SLZRLFS11KQMyy3jFb7wak06LGi9DU\\\\\\\/+6Ts5ZNhczvAjAYjsbcykdYG7tQnh+xECtgOEgP+eu3KJqEwUqPW3sB4op5wx3ak1a882opinW8tEfvLLQnMfBMYBiKHikcQUM7bih+dRt\\\\\\\/82mzOkO44dNKpX\\\\\\\/wZDgXaU9XyoL\\\\\\\/iMR\\\\\\\/Yo+sfvh3otzB0c6mkUwzEVv8UJSObWJi1WwjM6ju76qEP\\\\\\\/SnOOiXDqNWgzi0JaHSz7zW6d5Kk3flGyXXBr\\\\\\\/BUa\\\\\\\/6J3fM\\\\\\\/lBhmjMbdhe7ofK6sZFOKGA4ZpDsLC1zM9n95sILmP2UrbA6EIf0et7aawgs+Z3KBBLLk10TxNnfJX7ia\\\\\\\/t52gRL7g5Edj+zFGvxr2J6Vb5G0M1SIwLfdPXH8JnSSUQ5GiLtXhKFAv\\\\\\\/2tVQ5Xow34JyeDsVdjvZMa219nII2h94X+je9I+IEZcQAHv4jYrGD53q4uLXvD64xg0gt64vZzJYuyGd5GI1fy8deyTM1C85WeGB7YCKafzLDXPM6\\\\\\\/7JA\\\\\\\/8EiChE2tBvqzrrDYbaBU1M0nzsdeCU5XV2OUxr4gB2tCvMQUJZQTAtrIcjIU+vfNunAsk1eO2NfRQ4Yal3\\\\\\\/SRp3lu+w1r71hAJTughfHr9jngHb5IziD3\\\\\\\/XZK4gYfLSn1hEGVYd7PVkl0HMhjBuTbMFljLdm0ht5uZVZarfi5wNPR9rdBa2Jip3zpSVjniVeHJQktFjs3bkNGUQa44NMXHozG1brjtOb8Goy9LQdj\\\\\\\/CXbPGCBd+Sef9Z0QCscc3Mf87LJ02PEm\\\\\\\/NKc2BRXbmxXjKrpCeiYFmh56j073PMyEa1jUG12HxEWClIsncd57mTDsIqFQ8DGx8BrBuLrWtddovyOKFZu482TSyzGBe0mTufQfiaDKEHelLWAoMmtdG6+AzfLg9M\\\\\\\/L+dRvC2Wp71zYkb3OqLx\\\\\\\/xN0YX46ZgPGsMJi8ZflxZ89hTvOkVky9klaJltjcnIINuEdgiZzLbKB8SATO5ruPbg7kvK8IRC10+SwFX49nWpnEliHVFKbUJLxCWPx2GgUXRXGLEIo5eUG8nL7fOSVxGwykyTWD85A6p\\\\\\\/hrl2ZnWBu8w1wS0CmjOAD6Zc\\\\\\\/d6nThMOtiOOAuxr5v+fbrbOmNmwhiEv+DSI1FXAZUe6F4QcnwfT688Zp95\\\\\\\/HU4yilzpCHTogD2fec5mTChcJto7i\\\\\\\/kYSErho+JxAWgCz9LOBPqojqajGy1GX2d7+LhcEnwMlE3W7rhZ1blKjcv5Tc8gC5AEkupCBsIsoEQG4KmVJfp3UNutZRHKVPFiaOhU1T3tb\\\\\\\/Yn111IS8Dxwly5JYcbO37F2MFMezQtHN7jgIiNhUxh\\\\\\\/5SuUup0Fwyu0Rrwv4mNA58QxfDYh3NahkOzxMyqmYMmNl1Lkm1jv4nhPNLSycYCFaqHwB9e7nCXOIFazsfEJ9VT98ptpKKtJVBnTVf1HNLUpmUpexbQdBR+zWWAtEfuHW3C6uAciPgLm7ns8AWd9ObFrtwgzLC3XrZM\\\\\\\/mOgdJuELTOHw3SBlfqC6iVAtCY54l2GtkBGJX6ao4n95b++Q0qksrzFQKIMtc9XS4NYc0F7F+OmZQG+Q2z4NsT0BaPAtkHbMrB7Z98w3GkrOoKkRWys+lYSmyjzaQO2lIlUpx27pezPIK\\\\\\\/3zUDAJwDWqeVdkhUtDMvECNNR0vd8lL\\\\\\\/xlwPqKIrBQoQyQctHjh6q3PubOXLghh7LoDFQgYFsbUh3PKHjJvxYuv6Fa2R6DnclUv8LzJovU3gc9XJJZgQpwLjxjVytQKezL2jhRRvveXSG63KUJ8rpbaDQJEh1Jh57V5S27zcTvUOLRRNLRquQYbv3GO1OTV7BRGplpOi6G4+WYQxWrCpNEA2U1rbr20CZBit2dkNReJPcujEQvMRYY6N4V1B+kSL96\\\\\\\/1l+4eD21hAh16twtkmcbMa3IvTR3AKeapsYPTwOPwxh8vI1Aez67SkIJlfZBwD6AtOWS+HGzDq7+J7Q2ufjsmGFSJs9Jp68ILSxu3d7XimgnFu+lzrqGO8TulCNJmG0JheH1FQLqLrn7wUJViRHLNb6LznIr5Yr0VPuP+WNUFU9grb6AEkFM\\\\\\\/CaLt\\\\\\\/feG5MgsQeZ0nal1P9mxlsH\\\\\\\/Y41t3uNJxL8aV2au\\\\\\\/xd0qZwPsR+Bb3kq3azX6Xh3WpCZkrHCDfD9zHGA2bZzDF+zm3M6+HTkn3n2zOCxmU23phMWG\\\\\\\/d5kRHt4MzTwiKa2W+vSynA49IPa6tUZNjqSUBATB6Mgwdn7Fj5ttDM19FJdmYrvxQxEmWqXX8JvlG+6dyrgpA0O6IcKBqAG6jUMlpA+DVmByCc+WStHVKoB4DsRFtIfQIRkvatg5F4W92tc+uzkLS8rrCCGoF5R+NXMI39ltJicopTRTIR6FP\\\\\\\/BhJHnel4cEVcQtOuuAE6sikmORWUJ9wsQToPA7QXjIw7OalQXN53kB5h667p9rb3iOztxl7BxB29iFBcYlo0lALoi5ttD\\\\\\\/LnqBpkDnvORsW9DLXFkoRyUxnmwjgjubgMbHiUvverIvipk1OOXxbGz946a7CJxUy\\\\\\\/5D2KpVTbt+XNLUudx1bAfFjim\\\\\\\/P0Mb5G5tTtxfUWQIgtFkroLQNY837uRL2jKUSLF02orc3ul7TjV1YGPQlWqHVSEHIzm77iSD2e04R87iPU8257\\\\\\\/JqKg6veGACGuTJGbo37bCjU0tm6ZtMCT6m6DTXqYQRJkSxVx6awNFELcDvMdB+t0gUPqAlFy8ehPw5sYtk5i+nTncktRTDBIc9X4lZ7Y68RbuSOQbIH0FLQEpZYk6fLRU\\\\\\\/9m0+AEViwe+eEfX2H6lUnS4YOjdYLh+aDM0Pr9SWUgqttOFE9RlkpSmD3ZhCj3H9De7J+ZTGuggP0TNbe305LAb30\\\\\\\/77mOWDFdzL2\\\\\\\/4iJdVLXh2bTn6ZWzkr12OM2vppx68B0Kx0+J3Wy5Hb5tI0jL\\\\\\\/lUsJsLuAz7tQaoNDolPht0hUx3rBY8IhsuDMfuSrcaBMXs0GMfhU2Eb3Vv\\\\\\\/DKC\\\\\\\/8wEfveV4x7CJhPjNvTxZ73CTkRKTMuBdlYT4hMomRaQ6lkYvWbfEREDuepcNBBnH4s2o4j8KlvuIPf7NQkcATKJDDiQ8Ib4PadPZwzWTSij6kSqW9FPnFw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EtEa0gw5XILA4ihL8Zx9Kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b29013cc09b1ef20098896d58596696795495eafdb0ecdbc9d52bc02e406af31\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220QdtIox5yny8lgll5EDATFIuJlYSzAHPZqHSJn3sJGFUA+ishkzA5PdkIUCidc4zT4NGcBLgM0XtRWHxLPg10oUIliFuJBiafox36HVkvnfhAGrIFnSE\\\\\\\/XHWVGTN7GxXFRFj0Q8JGS+498r9GBrdvuQrzoTg2BpRSQmMRV\\\\\\\/i8IPzRXuJf5+oe1bMpGhUf5nQKDi4Zvv5t1kCTVbvm3ZzKU1unDeKh+awee1dV9ZGVllPfl943SW0MkbcvLH+OUu3yrq89Xzb0CclVNaqSvmODu+LCFIwZ4jz2FtqpG5VIy+mNcYqNv7lvHlfDbwIK+trkfD65n0UdCJzSG5ZJHGtSEpnrai\\\\\\\/F1maGhcnm\\\\\\\/sDPWVIgqLEm4o1P\\\\\\\/yzzGzk67SgnuNEWfRW228SWDSISS7Lb1KQFyQX4rVIAdcyW5jWM9GkNDwxn0kBHefecezccRuYmLBjBBBAUPf8215un1Dbj5W1Keh9DdXB3dcLNfaQS90OLK1kFYJJdwoIO+R6JQgPfijufHuC\\\\\\\/vmPnMAvqGOlGZN0aFdhdrJ3CriFTIeYkwFsn5wqIiRPl50AtBtLHw94XGzVQSSkcTTQbXRQClwEWTROypPuEWfzGw4Jyu2kgcN9l6UOFjk6e93Arrhcgz68722L7gHI0cCfYkhPmc5H+E3cYkgon8eWX6DmvrvdH2QM4H5BKQueFiPCtl\\\\\\\/8cvFOzeZaUW9WBxg\\\\\\\/nbZVL8VQzyWuWH8yV9eJpRKYdMATOXTGe+M9pYn1x1KYtfluNWzewAQYrrvIz+KTKZv3XMKU3QuYThXlj49VpWCMWDDCEolTY6XrdVRunIDrKdRXud2OJlx6swD4h54rtjITurXxIhEBuKurmw9vi\\\\\\\/rm9x1yZ7+xjJUOgo9jvCtoOo4IRTQPwmH5FBVSLT\\\\\\\/AIQxNyK0m9Dv23pbRzhnTGRxJhzzNt4C7kxp\\\\\\\/nQfHl2+zy1MT22Ru3cB3D8JyyGDOfzhntZH0Lu0d8CrFdL+QkxKmmFlDadMO2\\\\\\\/W2JyQLDkAIT4OGgLhp5Rt02+9Y05aLnPvimhUpSQAunCKU1VkfKb51+sYYuguQLFn8O\\\\\\\/Qarqk7cDR9ypkk1PxF2Ft12auvJRYmX8fAlAGwrC1rZDAT0pmeAXpMzSFmxFmVtowrhXSUBXDbN9+wd6DhpiYAweiIRqTOD2ShWHtPvcAjPS3IDCPTMja2MbqAQoIBNfvgHGpZ6UcB8S9l7Yt8vai1sSBj7AEmE6pohMG4HYs8VwS7vuUGffLSR0C+KVVmmxYhjcueSsbUDfAFPWoYDKgqQRYWWzARqb9b\\\\\\\/iUeXdSOStjgmS9Lxk30GjMLPH\\\\\\\/6fCNtYjOUkzeJbNVQzTtN2i6IloR6zdFyywH+ESgPcF0\\\\\\\/7Tfsk+gfwdL\\\\\\\/Vi\\\\\\\/DzXe4X4RQqia1huajMe+9OuIui4l\\\\\\\/x9CZZVBPsUQlOGzYD3p+IlKtBMSVQvdROZTFD5ApV6lPsDBmepa1JbzwdtuUGS0JYG\\\\\\\/Plbz1GLVaM3EUEnrNiDDy2ogYWjgdT37nDElNYcXmYvIYVMIqB+FD9hDXMW6Ze8MEnlMMIkhrARmlc4WF316eJWLBgfvKUKjuJQD8g7lsX2RiUlAjTBZt\\\\\\\/XCWiEntaMikbq66V9LGxsPQl7Jdla5uBVrPXZXVqjKMzX8EUM9WC\\\\\\\/tJzxa3\\\\\\\/iP8\\\\\\\/xMVq45Lhkkfv5Fvzqvd7uFkPD2mjRbBGEGIm\\\\\\\/jrWHyD46G4JK2q1eG7QnSZyEnii0ss8OTNHCA86fRR6v8QZ2A0f5CzCytYzOfO0MKRnwTO25+ZTQxA0wocpe\\\\\\\/ZDHTFZfoFgBj\\\\\\\/qJi3b720a787n0EOzdsvirh3Cb1JfrY81Iteu+sMFYUlITuaf7p\\\\\\\/ZMBArzAa3S96GCTQ+JbIPTiMQ5NWJdQWxJpVR8E5U5fczuC4BuPikw74O8ZA8SNCH1jXj2h5n9DWAmrp2ihME13\\\\\\\/Ion9dd3hXBWoiTAd+LGmykTRPj2NCPI4+efQWJ1DODTT71oLX6PgDbFIAvzNlsYggQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LDFC7NX6fbdBJzW1nQLvWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ef8072e5a8168bd02fbf15fa22ccbe385217f498f490ae9e96b97a0cea57c20\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492489 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/755" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QuEZdLODsR73OlOK7fta542nw_YppxAlyMGmGTmBvEo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/755" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JBiZhgMXvROyhJAYcs5jlnYN2324rfnmAaUNKXwiHA0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/749" title="Shep RNA-Binding Capacity Is Required for Antagonism of gypsy Chromatin Insulator Activity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/769" title="The Role of UV-B light on Small RNA Activity During Grapevine Berry Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492489" id="node492489" data-pisa="ggg;9/3/755" data-pisa-master="ggg;g3.118.201008" data-apath="/ggg/9/3/755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea Minio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mélanie Massonnet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosa Figueroa-Balderas</span>, <span class="highwire-citation-author" data-delta="3">Amanda M. Vondras</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Blanco-Ulate</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario Cantu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">755-767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.201008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492489" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Minio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BMinio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BMinio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea Minio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mélanie Massonnet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BMassonnet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Massonnet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM%25C3%25A9lanie%2BMassonnet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mélanie Massonnet</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rosa Figueroa-Balderas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosa%2BFigueroa-Balderas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figueroa-Balderas%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosa%2BFigueroa-Balderas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosa Figueroa-Balderas</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Amanda M. Vondras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BVondras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vondras%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BVondras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Barbara Blanco-Ulate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BBlanco-Ulate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco-Ulate%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BBlanco-Ulate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Blanco-Ulate</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario Cantu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dacantu@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dacantu@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/755" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492489" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492489/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/755.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492489" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492489/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/755.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492489" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492489/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/755.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492489" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492489/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/755.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492489" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492489/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Transcriptomics has been widely applied to study grape berry development. With few exceptions, transcriptomic studies in grape are performed using the available genome sequence, PN40024, as reference. However, differences in gene content among grape accessions, which contribute to phenotypic differences among cultivars, suggest that a single reference genome does not represent the species’ entire gene space. Though whole genome assembly and annotation can reveal the relatively unique or “private” gene space of any particular cultivar, transcriptome reconstruction is a more rapid, less costly, and less computationally intensive strategy to accomplish the same goal. In this study, we used single molecule-real time sequencing (SMRT) to sequence full-length cDNA (Iso-Seq) and reconstruct the transcriptome of Cabernet Sauvignon berries during berry ripening. In addition, short reads from ripening berries were used to error-correct low-expression isoforms and to profile isoform expression. By comparing the annotated gene space of Cabernet Sauvignon to other grape cultivars, we demonstrate that the transcriptome reference built with Iso-Seq data represents most of the expressed genes in the grape berries and includes 1,501 cultivar-specific genes. Iso-Seq produced transcriptome profiles similar to those obtained after mapping on a complete genome reference. Together, these results justify the application of Iso-Seq to identify cultivar-specific genes and build a comprehensive reference for transcriptional profiling that circumvents the necessity of a genome reference with its associated costs and computational weight.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptome-reference" class="hw-term hw-article-keyword hw-article-keyword-transcriptome-reference" rel="nofollow">transcriptome reference</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-seq</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/vitis-vinifera" class="hw-term hw-article-keyword hw-article-keyword-vitis-vinifera" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Vitis vinifera</em></a></li><li class="kwd"><a href="/keyword/berry-ripening" class="hw-term hw-article-keyword hw-article-keyword-berry-ripening" rel="nofollow">berry ripening</a></li></ul><p id="p-4">Grape berries undergo a series of complex physiological and biochemical changes during their development that determine their characteristics at harvest (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Kuhn <em>et al.</em> 2014</a>). Genome-wide expression studies using microarray and, more recently, RNA sequencing (RNA-seq) revealed that berry development involves the expression and modulation of approximately 23,000 genes (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Massonnet <em>et al.</em> 2017a</a>) and that the ripening transition is associated with a major transcriptome shift (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Fasoli <em>et al.</em> 2012</a>). Transcriptomic studies characterized the ripening program across grapevine cultivars (<a id="xref-ref-127-1" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Jiao <em>et al.</em> 2015</a>; <a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Massonnet <em>et al.</em> 2017a</a>), identifying key ripening-related genes (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Palumbo <em>et al.</em> 2014</a>; <a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">Massonnet <em>et al.</em> 2017a</a>) and determining the impact of stress and viticultural practices on ripening (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Deluc <em>et al.</em> 2009</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Pastore <em>et al.</em> 2013</a>; <a id="xref-ref-135-1" class="xref-bibr" href="#ref-135">Xi <em>et al.</em> 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amrine <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blanco-Ulate <em>et al.</em> 2015</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2017</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Corso <em>et al.</em> 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Hopper <em>et al.</em> 2016</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Savoi <em>et al.</em> 2016</a>; <a id="xref-ref-140-1" class="xref-bibr" href="#ref-140">Zenoni <em>et al.</em> 2017</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Lecourieux <em>et al.</em> 2017</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Massonnet <em>et al.</em> 2017b</a>). This knowledge increases the possibility of exerting control over the ripening process, improving fruit composition under suboptimal or adverse conditions, and enhancing desirable traits in a crop with outstanding cultural and commercial significance (<a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Savoi <em>et al.</em> 2016</a>, <a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">2017</a>; <a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Serrano <em>et al.</em> 2017</a>; <a id="xref-ref-140-2" class="xref-bibr" href="#ref-140">Zenoni <em>et al.</em> 2017</a>).</p><p id="p-5">These genome-wide expression analyses were possible because a highly contiguous assembly for the species was produced (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>); this first effort used a grape line (PN40024) created by several rounds of backcrossing to reduce heterozygosity, facilitating genome assembly (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>). Though poor by current standards, this pioneering, chromosome-resolved assembly served as the basis for numerous publications. However, the structural diversity of grape genomes makes using a single one-size-fits-all reference genome inappropriate (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Golicz <em>et al.</em> 2016a</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">2016b</a>). There is substantial unshared gene content between cultivars, with 8–10% of the genes missing when two cultivars are compared (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>). Although many of these genes are not essential for plant survival, they can account for 80% of the expression within their respective families and expand key gene families possibly associated with cultivar-specific traits (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>).</p><p id="p-6">Assembling genome references for all interesting cultivars is impractical, in part because its cost remains prohibitive and because of genomic features that impede the development of high-quality genome assemblies for any grape cultivar. Although the <em>V. vinifera</em> genome is relatively small (∼487Mb) (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Lodhi and Reisch 1995</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>) and as repetitive as other plant genomes of similar size (41.4%) (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Michael and Jackson 2013</a>), it is highly heterozygous (∼13%) (<a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>; <a id="xref-ref-126-1" class="xref-bibr" href="#ref-126">Velasco <em>et al.</em> 2007</a>). Most domesticated grape cultivars are crosses between distantly related parents; this and clonal propagation cause the high heterozygosity observed in the species (<a id="xref-ref-120-1" class="xref-bibr" href="#ref-120">Strefeler <em>et al.</em> 1992</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Ohmi <em>et al.</em> 1993</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bowers and Meredith 1997</a>; <a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Sefc <em>et al.</em> 1998</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Lopes <em>et al.</em> 1999</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Di Gaspero <em>et al.</em> 2005</a>; <a id="xref-ref-122-1" class="xref-bibr" href="#ref-122">Tapia <em>et al.</em> 2007</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ibáñez <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cipriani <em>et al.</em> 2010</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Myles <em>et al.</em> 2011</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Lacombe <em>et al.</em> 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Chin <em>et al.</em> 2016</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Minio <em>et al.</em> 2017</a>; <a id="xref-ref-142-1" class="xref-bibr" href="#ref-142">Zhou <em>et al.</em> 2017</a>). Earlier attempts using short reads struggled to resolve complex, highly heterozygous genomes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Gnerre <em>et al.</em> 2011</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Di Genova <em>et al.</em> 2014</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Kajitani <em>et al.</em> 2014</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Safonova <em>et al.</em> 2015</a>). A limited ability to call consensus polymorphic regions yields highly fragmented assemblies where structural ambiguity occurs and alternative alleles at heterozygous sites are excluded altogether (<a id="xref-ref-126-2" class="xref-bibr" href="#ref-126">Velasco <em>et al.</em> 2007</a>). Single Molecule Real Time (SMRT) DNA sequencing (Pacific Biosciences, California, USA) has emerged as the leading technology for reconstructing highly contiguous, diploid assemblies of long, repetitive genomes that include phased information about heterozygous sites (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Chin <em>et al.</em> 2013</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2016</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Doi <em>et al.</em> 2014</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Gordon <em>et al.</em> 2016</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Pryszcz and Gabaldón 2016</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Ricker <em>et al.</em> 2016</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Seo <em>et al.</em> 2016</a>; <a id="xref-ref-128-1" class="xref-bibr" href="#ref-128">Vij <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Huddleston <em>et al.</em> 2017</a>). Recently, we used <em>Vitis vinifera</em> cv. Cabernet Sauvignon to test the ability of SMRT reads and the FALCON-Unzip assembly pipeline to resolve both alleles at heterozygous sites in the genome (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Chin <em>et al.</em> 2016</a>). The assembly produced was significantly more contiguous (contig N50 = 2.17 Mb) than the original PN40024 assembly (contig N50 = 102.7 kbp) and provided the first phased sequences of the diploid <em>V. vinifera</em> genome (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Minio <em>et al.</em> 2017</a>).</p><p id="p-7">Despite recent advances in genome reconstruction methodologies, assembling a complex plant genome is still costly. Transcriptome reconstruction is the only alternative strategy to depict known and unknown gene content information (<a id="xref-ref-127-2" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Jiao <em>et al.</em> 2015</a>). <em>De novo</em> assembly of RNA-seq reads is widely used for this purpose (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashrafi <em>et al.</em> 2012</a>; <a id="xref-ref-127-3" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bellucci <em>et al.</em> 2014</a>). SMRT technology was recently deployed to investigate expressed gene isoforms (Iso-Seq) in a variety of organisms, including a handful of plant species (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Liu <em>et al.</em> 2017</a>; <a id="xref-ref-144-1" class="xref-bibr" href="#ref-144">Zulkapli <em>et al.</em> 2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Filichkin <em>et al.</em> 2018</a>). Long reads delivered by this methodology report full-length transcripts sequenced from their 5′-ends to polyadenylated tails (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-132-1" class="xref-bibr" href="#ref-132">Weirather <em>et al.</em> 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Gao <em>et al.</em> 2016</a>; <a id="xref-ref-124-1" class="xref-bibr" href="#ref-124">Tombácz <em>et al.</em> 2016</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Kuo <em>et al.</em> 2017</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Price and Gibas 2017</a>; <a id="xref-ref-133-1" class="xref-bibr" href="#ref-133">Workman <em>et al.</em> 2017</a>), making Iso-Seq an ideal technology for reconstructing a transcriptome without a reference genome sequence and without assembling fragments to resolve the complete isoform sequence (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Honaas <em>et al.</em> 2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ju <em>et al.</em> 2017</a>). Moreover, alternative transcripts that contribute to the gene space complexity (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brett <em>et al.</em> 2002</a>) and vary with cell type (<a id="xref-ref-121-1" class="xref-bibr" href="#ref-121">Swarup <em>et al.</em> 2016</a>), developmental stage (<a id="xref-ref-123-1" class="xref-bibr" href="#ref-123">Thatcher <em>et al.</em> 2016</a>), and stress (<a id="xref-ref-137-1" class="xref-bibr" href="#ref-137">Yan <em>et al.</em> 2012</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Liu <em>et al.</em> 2016</a>) cannot be definitively characterized without full-length transcript information.</p><p id="p-8">The objective of this study was to test whether full-length cDNA sequencing with Iso-Seq technology is a suitable alternative to traditional genome sequencing, assembly, and annotation for reconstructing a grape transcriptome reference for transcriptional profiling. We compared how Cabernet Sauvignon’s Iso-Seq transcriptome fares as a reference for RNA-seq analysis <em>vs.</em> its annotated genome. We sequenced the full-length transcripts of ripening berries with Iso-Seq and Illumina RNA-seq reads. The high-coverage short-read data were used to profile gene expression and to error-correct low-expression isoforms that would have been otherwise lost by the standard Iso-Seq pipeline. The transcriptome reference built with Iso-Seq data represented most of the expressed genes in the grape berries and included cultivar-specific or “private” genes. When used as the reference for RNA-seq, Iso-Seq generated transcriptome profiles quantitatively similar to those obtained by mapping on a complete genome reference. These results support using Iso-Seq to capture the gene space of a plant and build a comprehensive reference for transcriptional profiling without a pre-defined reference genome.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Plant material and RNA isolation</h3><p id="p-9">Grape berries from Cabernet Sauvignon FPS clone 08 were collected in Summer 2016 from vines grown in the Foundation Plant Services (FPS) Classic Foundation Vineyard (Davis, CA, USA). Between 10 and 15 berries were sampled at pre-véraison, véraison, post-véraison, and at commercial maturity. Table S1 provides weather information for the sampling days. The ripening stages were visually assessed based on color development and confirmed by measurements of soluble solids (Figure S1; Table S2). On the day of sampling, berries were deseeded, frozen in liquid nitrogen, and ground to powder (skin and pulp). Total RNA was isolated using a Cetyltrimethyl Ammonium Bromide (CTAB)-based extraction protocol as described in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blanco-Ulate <em>et al.</em> (2013)</a>. RNA purity was evaluated with a Nanodrop 2000 spectrophotometer (Thermo Scientific, Hanover Park, IL, USA). RNA was quantified with a Qubit 2.0 Fluorometer using the RNA broad range kit (Life Technologies, Carlsbad, CA, USA). RNA integrity was assessed using electrophoresis and an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA). Only RNA with integrity number (RIN) greater than 8.0 was used for SMRTbell library preparation.</p></div><div id="sec-3" class="subsection"><h3>Library preparation and sequencing</h3><p id="p-10">RNAs from four biological replicates per developmental stage were pooled in equal amounts. One µg of the pooled RNA was used for cDNA synthesis and for SMRTbell library construction using the SMARTer PCR cDNA synthesis kit (Clontech Laboratories, Inc. Mountain View, CA, USA). First-strand cDNA synthesis was performed using the SMRTScribe Reverse Transcriptase (Clontech Laboratories, Inc. Mountain View, CA, USA). Each developmental stage was individually barcoded (Table S3). To minimize artifacts during large-scale amplification, a cycle optimization step was performed by collecting five 5 µl aliquots at 10, 12, 14, 16, and 18 PCR cycles. PCR reaction aliquots were loaded on an E-Gel pre-cast agarose gel 0.8% (Invitrogen, Life Technologies, Carlsbad, CA, USA) to determine the optimal cycle number. Second-strand cDNA was synthesized and amplified using the Kapa HiFi PCR kit (Kapa Biosystems, Wilmington, MA, USA) with the 5′ PCR primer IIA (Clontech Laboratories, Inc. Mountain View, CA, USA) following the manufacturer’s instructions. Large-scale PCR was performed using the number of cycles determined during the optimization step (14 cycles). Barcoded double-stranded cDNAs were pooled at equal amounts and used for size selection. Size selection was carried out with a BluePippin (Sage Science, Beverly, MA, USA) and 1-2 kbp, 2-3 kbp, 3-6 kbp, and 5-10 kbp fractions were collected. After size selection, each fraction was PCR-enriched prior to SMRTbell template library preparation. cDNA SMRTbell libraries were prepared using 1-3 µg of PCR enriched size-selected samples, followed by DNA damage repair and SMRTbell ligation using the SMRTbell Template Prep Kit 1.0 (Pacific Biosciences, Menlo Park, CA, USA). A second size selection was performed on the 3-6 kbp and 5-10 kbp fractions to remove short contaminating SMRTbell templates. A total of 8 SMRT cells were sequenced on a PacBio Sequel system (DNA Technologies Core, University of California, Davis, USA). Demultiplexing, filtering, quality control, clustering and polishing of the Iso-Seq sequencing data were performed using SMRT Link ver. 4.0.0 (Table S4). Iso-Seq read error rates were estimated using the identity of the best alignment on the diploid Cabernet Sauvignon genomic assembly (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Chin <em>et al.</em> 2016</a>). Alignment was performed with GMAP ver. 2015-09-29 (<a id="xref-ref-134-1" class="xref-bibr" href="#ref-134">Wu and Watanabe 2005</a>) using the parameters “-K 20000 -B 4 -f 2”. Coding sequences (CDS) were identified using Transdecoder (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Haas <em>et al.</em> 2013</a>) as implemented in the PASA ver. 2.1.0 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Haas <em>et al.</em> 2003</a>).</p><p id="p-11">RNA-seq libraries were prepared using the Illumina TruSeq RNA sample preparation kit v2 (Illumina, San Diego, CA, USA), following the low-throughput protocol. Each biological replicate was barcoded individually. Libraries were evaluated for quantity and quality with the High Sensitivity chip on a Bioanalyzer 2100 (Agilent Technologies, Santa Clara, CA, USA) and sequenced on an Illumina HiSeq4000 (DNA Technologies Core Facility, University of California, Davis, USA; Table S5). Quality filtering and adapter trimming were performed with Trimmomatic ver. 0.36 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bolger <em>et al.</em> 2014</a>) using the following parameters: “ILLUMINACLIP:2:30:10 LEADING:7 TRAILING:7 SLIDINGWINDOW:10:20 MINLEN:36” (Table S5). Error correction of the Full-length Non-Chimeric (FLNC) Iso-Seq reads was carried out using high-quality Illumina reads and LSC ver. 2.0 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Au <em>et al.</em> 2012</a>) with a minimum coverage threshold of 5 reads (“–short_read_coverage_threshold 5”).</p></div><div id="sec-4" class="subsection"><h3>Transcriptome reconstruction and annotation</h3><p id="p-12">Isoforms identified by the SMRT Link pipeline and error-corrected Iso-Seq reads were merged using EvidentialGene (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gilbert 2013</a>) in order to obtain a non-redundant transcriptome (ISNT). Contaminant sequences were searched by parsing blastn (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) alignments against the NCBI NT database (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/db/">ftp://ftp.ncbi.nlm.nih.gov/blast/db/</a>, retrieved January 17<sup>th</sup>, 2017) using Megan ver. 6.6.5 (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Huson <em>et al.</em> 2007</a>) with default parameters. Sequences detected as non-viridiplantae were removed. Isoforms with no RNA-seq read mapping on their sequence over the 16 samples (662 isoforms) were considered as putative artifacts and were also discarded. ISNT sequences are provided in File S1. Functional annotation was performed with blastx (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) using RefSeq plant proteins as database (<a href="ftp://ftp.ncbi.nlm.nih.gov/refseq">ftp://ftp.ncbi.nlm.nih.gov/refseq</a>, retrieved January 17<sup>th</sup>, 2017) imposing an HSP length cutoff of 50 amino acids. Functional domains were identified with InterProScan ver. 5.28-68.0 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Jones <em>et al.</em> 2014</a>; File S2). Tree view of identified GO terms was generated using WEGO ver. 2.0 (<a id="xref-ref-139-1" class="xref-bibr" href="#ref-139">Ye <em>et al.</em> 2006</a>). Iso-Seq reads were considered derived from repetitive regions when showing a RepeatMasker (<a id="xref-ref-115-1" class="xref-bibr" href="#ref-115">Smit <em>et al.</em> 2013</a>) hit with coverage ≥ 75% and an identity ≥ 50% using the custom-created repeat library (see below). Non-coding RNAs were identified with Infernal ver. 1.1.2 (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Nawrocki <em>et al.</em> 2009</a>) using the Rfam database ver. 12.2 (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Nawrocki <em>et al.</em> 2015</a>). Secondary overlapping alignments and structures with an <em>e</em>-value ≥ 0.01 were rejected. Hits on the minus strand of the Iso-Seq reads were rejected as well as matches that were truncated or covering less than 80% of the entire read.</p></div><div id="sec-5" class="subsection"><h3>Cabernet Sauvignon genome annotation</h3><p id="p-13">Cabernet Sauvignon primary contigs and haplotigs (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Chin <em>et al.</em> 2016</a>) were used as genomic reference. Primary contigs are the backbone sequence of the genome assembly and haplotigs are the alternative haplotype sequences where the assembler could phase the two haplotypes. A repeat library was created <em>ad hoc</em> for Cabernet Sauvignon. MITEs were identified with MITEHunter ver. 11.2011 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Han and Wessler 2010</a>); LTRs and TRIMs were identified with LTRharvest (GenomeTools ver. 1.5.7; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ellinghaus <em>et al.</em> 2008</a>) and LTRdigest (GenomeTools ver. 1.5.7; <a id="xref-ref-119-1" class="xref-bibr" href="#ref-119">Steinbiss <em>et al.</em> 2009</a>). RepeatModeler ver. 1.0.8 (<a id="xref-ref-114-1" class="xref-bibr" href="#ref-114">Smit and Hubley 2008</a>), and RepeatMasker ver. open-4.0.6 (<a id="xref-ref-115-2" class="xref-bibr" href="#ref-115">Smit <em>et al.</em> 2013</a>) were used to generated a custom library of repeat models. Repetitive elements in Cabernet Sauvignon genome were then identified with RepeatMasker (<a id="xref-ref-115-3" class="xref-bibr" href="#ref-115">Smit <em>et al.</em> 2013</a>) using both custom and plant repeat models altogether (Table S6).</p><p id="p-14"><em>Ab initio</em> trainings and predictions were carried out with SNAP ver. 2006-07-28 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Korf 2004</a>), Augustus ver. 3.0.3 (<a id="xref-ref-118-1" class="xref-bibr" href="#ref-118">Stanke <em>et al.</em> 2006</a>), GeneMark-ES ver. 4.32 (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Lomsadze <em>et al.</em> 2005</a>), GlimmerHMM ver. 3.0.4 (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Majoros <em>et al.</em> 2004</a>), GeneID ver. 1.4.4 (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Parra <em>et al.</em> 2000</a>) and Twinscan ver. 4.1.2 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Korf <em>et al.</em> 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brent 2008</a>) using TAIR10 annotation for Arabidopsis as informant species. MAKER-P ver. 2.31.3 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Campbell <em>et al.</em> 2014a</a>) was used to integrate the <em>ab initio</em> predictions with the experimental evidence listed in Table S7 using the parameters reported in File S3. Only MAKER-P models showing an Annotation Edit Distance (AED) &lt; 0.5 were kept.</p><p id="p-15">Gene structure refinement was carried out with PASA ver. 2.1.0 (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Haas <em>et al.</em> 2003</a>), parameters can be found in File S4. As transcriptomic evidence, we used the Iso-Seq data as well all the publicly available grape transcriptomic data (Table S7). Public Cabernet Sauvignon RNA-seq data (Table S7) were <em>de novo</em> assembled separately using HISAT2 and Stringtie ver. 1.1.3 (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Pertea <em>et al.</em> 2015</a>). <em>De novo</em> transcript sequences were then clustered in a non-redundant dataset using CD-HIT-EST ver. 4.6 (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Li and Godzik 2006</a>) with an identity threshold of 99%. Genome annotation summary is reported in Table S8. Alternative splicing forms were classified using AStalavista ver. 3.0 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Foissac and Sammeth 2007</a>).</p><p id="p-16">Non-coding RNAs were searched with Infernal ver. 1.1.2 (<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Nawrocki <em>et al.</em> 2009</a>) as described above (Table S9). The predicted transcripts’ functional annotation was made with blastp search using the RefSeq plant protein database. Functional domain identification was done with InterProScan as described above for the ISNT. For each gene locus, a non-redundant list of the GO terms attributed to all the alternative transcripts was generated.</p></div><div id="sec-6" class="subsection"><h3>Gene expression analysis with RNA-seq</h3><p id="p-17">For expression profiling, short reads were aligned on transcript sequences using Bowtie2 ver. 2.26 (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Langmead and Salzberg 2012</a>) with options “–sensitive–dpad 0–gbar 99999999–mp 1,1–np 1–score-min L,0,-0.1”. Evaluation of expression at isoform and gene locus levels was carried out using RSEM ver. 1.1.14b3 (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Li and Dewey 2011</a>) with default parameters. Differential expression analysis was performed using EBSeq ver. 1.16.0 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Leng <em>et al.</em> 2013</a>). For each pairwise comparison of consecutive growth stages, size factors were calculated with median normalization using five iterations of the EM algorithm. Genes were considered as significantly differentially expressed if they had a minimum <em>posteriori</em> estimate probability (PPDE) threshold of 0.95 and mean RPKM greater than 1 in at least one of two growth stages. Pearson correlation matrices and PCAs were performed using the log<sub>2</sub>-transformed RPKM values. Heatmaps of the Pearson correlation matrices and dendrograms were generated using heatmap.2 function from the gplots R package ver. 3.0.1 (<a id="xref-ref-131-1" class="xref-bibr" href="#ref-131">Warnes <em>et al.</em> 2016</a>). PCAs were carried out in R with the FactoMineR package ver. 1.41 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Lê <em>et al.</em> 2008</a>).</p><p id="p-18">In order to compare expression values of ISNT transcripts and gene loci in the Cabernet Sauvignon genes, ISNT transcripts were aligned to the Cabernet Sauvignon genome using GMAP as described above. ISNT transcripts were associated with gene loci annotated on the Cabernet Sauvignon genome if they aligned at least for 66% of their length. Alignments with translocation were excluded (Table S10). For each ISNT-gene locus association, gene expression was calculated as the sum of the mean expression values of all ISNT transcripts and all Cabernet Sauvignon gene loci, separately, using RSEM ver. 1.1.14b3 (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Li and Dewey 2011</a>). Differential expression analysis at cluster level was performed using EBSeq ver. 1.16.0 (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Leng <em>et al.</em> 2013</a>).</p></div><div id="sec-7" class="subsection"><h3>Data Availability</h3><p id="p-19">Sequencing data are accessible through NCBI (<strong>PRJNA433195</strong>) and other relevant datasets, such as protein-coding gene and repeat coordinates, can be retrieved from the Cantu lab github repository (<a href="http://cantulab.github.io/data.html">http://cantulab.github.io/data.html</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7291943">https://doi.org/10.25387/g3.7291943</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Isoform sequencing of the grape transcriptome during berry development</h3><p id="p-20">To obtain a comprehensive representation of the transcripts expressed during berry development, we isolated RNA from Cabernet Sauvignon berries (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) before the onset of ripening (4.35 ± 0.39 °Brix), at véraison (10.94 ± 0.26 °Brix), after véraison (18.38 ± 0.61 °Brix), and at commercial ripeness (20.33 ± 0.76 °Brix). To avoid loading bias, cDNAs were fractionated based on their length to produce four libraries at each developmental stage in size ranges of 1-2 kbp, 2-3 kbp, 3-6 kbp, or 5-10 kbp (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Libraries derived from different developmental stages were barcoded and libraries with similar cDNA size were pooled together. Each library pool was sequenced independently on two SMRT cells of a Pacific Biosciences Sequel system generating a total of 23.6 Gbp. In parallel, the same samples were sequenced using Illumina technology (25,655,771 ± 3,512,980 high-quality reads per sample) to provide high-coverage sequence information for error correction and for gene expression quantification (Table S5). Demultiplexing, filtering and quality control of SMRT sequencing data were performed using SMRT Link as described in the Methods section. A total of 672,635 full-length non-chimeric (FLNC; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) reads with a maximum length of 14.6 kbp and a N50 of 3.5 kbp were generated (Table S4). FLNC reads were further polished and clustered into 46,675 single representatives of expressed transcripts (henceforth, polished-clustered Iso-Seq reads or PCIRs) ranging from 400 bp to 8.8 kbp with a N50 of 3.6 kbp (Table S4). The alignment of FLNC reads and PCIRs to the genomic DNA contigs of the same Cabernet Sauvignon clone (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Chin <em>et al.</em> 2016</a>; <a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">Minio <em>et al.</em> 2017</a>) confirmed that sequence clustering and polishing successfully increased sequence accuracy, whose median values were 95.4% in FLNC reads and 99.6% in the PCIRs. The increase in sequence accuracy was also reflected by the significantly longer detectable coding sequences (CDS) in the PCIRs compared to the short and fragmented CDS found in the FLNC reads (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The residual sequence discrepancy between PCIRs and the genomic contigs could be explained by heterozygosity and/or sequencing errors, but unexpectedly not by expression level (Figure S2).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/755/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Full-length cDNA sequencing (Iso-Seq) of Cabernet Sauvignon berries. Representative pictures of berry clusters at the four growth stages used in this study and the read length distribution of the Iso-Seq libraries by cDNA size fraction. FLNC, Full-Length Non-Chimeric." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-902267164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Full-length cDNA sequencing (Iso-Seq) of Cabernet Sauvignon berries. Representative pictures of berry clusters at the four growth stages used in this study and the read length distribution of the Iso-Seq libraries by cDNA size fraction. FLNC, Full-Length Non-Chimeric.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/755/F1.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/755/F1.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/755/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/755/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492487" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Full-length cDNA sequencing (Iso-Seq) of Cabernet Sauvignon berries. Representative pictures of berry clusters at the four growth stages used in this study and the read length distribution of the Iso-Seq libraries by cDNA size fraction. FLNC, Full-Length Non-Chimeric.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/755/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram depicting the main steps of Iso-Seq read analysis. Raw Iso-Seq reads were processed to obtain Full-Length Non-Chimeric (FLNC) reads and clustered isoform reads (PCIRs). FLNC reads that did not cluster in any of the PCIRs were error-corrected using RNA-seq data (C-FLNC). The final dataset described in this study included PCIRs and C-FLNC reads. For each step, sequencing accuracy and predicted coding sequence (CDS) length distributions are reported." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-902267164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Diagram depicting the main steps of Iso-Seq read analysis. Raw Iso-Seq reads were processed to obtain Full-Length Non-Chimeric (FLNC) reads and clustered isoform reads (PCIRs). FLNC reads that did not cluster in any of the PCIRs were error-corrected using RNA-seq data (C-FLNC). The final dataset described in this study included PCIRs and C-FLNC reads. For each step, sequencing accuracy and predicted coding sequence (CDS) length distributions are reported.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/755/F2.medium.gif" width="170" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/755/F2.medium.gif" width="170" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/755/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/755/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492381" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Diagram depicting the main steps of Iso-Seq read analysis. Raw Iso-Seq reads were processed to obtain Full-Length Non-Chimeric (FLNC) reads and clustered isoform reads (PCIRs). FLNC reads that did not cluster in any of the PCIRs were error-corrected using RNA-seq data (C-FLNC). The final dataset described in this study included PCIRs and C-FLNC reads. For each step, sequencing accuracy and predicted coding sequence (CDS) length distributions are reported.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Over 18.5% of the FLNC reads did not cluster with any other reads and were discarded by the SMRT Link pipeline. When mapped on the genomic contigs, the uncorrected reads displayed a sequence accuracy that reflected the typical error rate of 10–20% of the technology (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Giordano <em>et al.</em> 2017</a>; <a id="xref-ref-143-1" class="xref-bibr" href="#ref-143">Zimin <em>et al.</em> 2017</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Koren <em>et al.</em> 2017</a>). High error rates also resulted in short and fragmented detectable CDS (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). To recover the information carried by these 124,185 uncorrected FLNCs, which represented an important fraction of the transcriptome (see below), we error-corrected their sequences with LSC (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Au <em>et al.</em> 2012</a>) using the short reads generated with Illumina technology. As for the PCIRs, error correction resulted in greater sequence accuracy and longer CDS (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). PCIRs and error-corrected FLNC (C-FLNC) reads were finally combined into a single dataset of 170,860 corrected Iso-Seq isoforms (CISIs). As low as 1.7% (2,826) of the CISIs showed significant homology with interspersed repeats. LTRs and LINEs were the most abundant orders with 778 and 729 representatives, respectively. Chloroplast and mitochondria genes represented a small fraction of the CISIs with only 89 isoforms (0.05%) having a significant match (50% identity and mutual alignment coverage). CISIs were also searched for non-coding RNA (ncRNA) using the covariance models of the Rfam database; only 182 isoforms were annotated as ncRNAs and were all ribosomal RNA (145 attributed to the large subunit, clan CL00112, and 37 to the small subunit, clan CL00111). Excluding these transcribed isoforms, only 164 CISIs (0.1%) failed to align to the Cabernet Sauvignon genomic contigs, confirming the completeness of the genome assembly and the negligible biological contamination of the berry samples.</p><p id="p-24">To reconstruct an Iso-Seq non-redundant transcriptome (ISNT) that would be tested as a reference for expression profiling, CISIs were clustered with EvidentialGene (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Gilbert 2013</a>) to reduce the redundancy between stages and libraries. One representative for each expressed transcript was retained for a total of 28,721 isoforms (File S1). Less than four percent of these isoforms did not contain a complete open reading frame, likely due to residual errors in their sequence. Mapping on the Cabernet Sauvignon genome showed that they potentially derive from 24,378 gene loci (<em>i.e.</em>, protein-coding gene sequences) over both primary contigs (13,583 loci) and haplotigs (10,795 loci; File S5). Most isoforms (61.2%) aligned to both a locus on a primary contig and a locus on its associated haplotig, suggesting that despite the differences between haplotypes the mapping could not differentiate between alleles. For 10,727 loci (44%), multiple isoforms (2.03 ± 3.49 isoforms/locus) mapped on the same locus possibly due to alternative splicing and structural differences between alleles. ISNT included 77.6% of the BUSCO (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Simão <em>et al.</em> 2015</a>) conserved orthologous; while far from representing the complete grape transcriptome, the ISNT dataset included a remarkably high fraction of these conserved genes, particularly considering that ISNT was constructed using expression data from berries only. Interestingly, 301 BUSCO complete gene models (20%) were found in multiple copies in the ISNT, suggesting that alternative isoforms of these highly conserved genes are expressed during ripening. Putative functions were assigned to the ISNT transcripts as described in the Methods section (Table S11). Only four sequences did not match any sequence in the databases used. Gene Ontology (GO) terms were assigned to 23,386 transcripts with an average of ∼6.9 GO terms per transcript (Table S11, File S6-S8).</p></div><div id="sec-10" class="subsection"><h3>Isoform sequencing allows the discovery of private Cabernet Sauvignon genes</h3><p id="p-25">Previous analyses of gene content in a limited number of grape cultivars showed that up to 10% of grape isoforms were not shared between genotypes. Some of these “dispensable” genes were associated with cultivar-specific characteristics (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>). To identify protein-coding transcripts characteristic of Cabernet Sauvignon (<em>i.e.</em>, private genes), we looked for homologous sequences among the ISNT transcripts in the PN40024 genome (<a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Jaillon <em>et al.</em> 2007</a>; <a id="xref-ref-129-1" class="xref-bibr" href="#ref-129">Vitulo <em>et al.</em> 2014</a>) and in the transcriptomes of Corvina (<a id="xref-ref-127-4" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>), Tannat (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>), and Nebbiolo (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gambino <em>et al.</em> 2017</a>). Approximately five percent of the ISNT (1,501 isoforms) did not have a homologous copy in any of the four datasets (Table S12). These putative Cabernet Sauvignon private isoforms were involved in various biological processes of berry development and ripening like phenylpropanoid/flavonoid biosynthesis (a chalcone synthase, a flavanone 3-hydroxylase, and a flavonoid 3′-hydroxylase (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Falginella <em>et al.</em> 2012</a>)), sugar accumulation and transport (ten sucrose-phosphate synthases, a phosphofructokinase, a glucose-6-phosphate dehydrogenase, a sucrose transport SUC4-like, a polyol transporter, and an inositol transporter (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Afoufa-Bastien <em>et al.</em> 2010</a>; <a id="xref-ref-136-1" class="xref-bibr" href="#ref-136">Xin <em>et al.</em> 2013</a>)), water transport (eleven aquaporins), and cell wall metabolism and loosening (six cellulose synthases, a xyloglucan galactosyltransferase, one xyloglucan glycosyltransferase 9-like, two expansins, two xyloglucan endotransglucosylase/hydrolases, two pectinesterases, a pectin methylesterase/invertase inhibitor, seven polygalacturonases, and two β-galactosidases (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Carey <em>et al.</em> 1995</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Cosgrove 2000</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2005</a>)).</p></div><div id="sec-11" class="subsection"><h3>Protein-coding gene models in the Cabernet Sauvignon genomic assembly</h3><p id="p-26">To evaluate the non-redundant Iso-Seq transcriptome’s completeness and usefulness as a reference for RNA-seq analysis, the protein-coding genes in the Cabernet Sauvignon genome were predicted as described in Figure S3. First, the repetitive regions of the genome were masked using a custom-made library of Cabernet Sauvignon MITE, LTR, and TRIM information. Overall, ∼51% of the assembly consisted of repetitive elements (Table S6), with 412,994 repetitive elements on the primary assembly (313 Mb) and 274,123 on the haplotigs (177Mb), LTRs were the most abundant class, covering over 335 Mb of the genomic sequences, with Gypsy and Copia families accounting for 201 Mb and 104 Mb, respectively. Next, MAKER-P (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Campbell <em>et al.</em> 2014b</a>) identified putative protein-coding loci, combining the results of six <em>ab initio</em> predictors trained <em>ad hoc</em> with publicly available experimental evidences. <em>Ab initio</em> predictors were trained using a custom set of 4,000 randomly selected gene models out of the 5,636 high-quality, non-redundant, and highly conserved gene models of the PN40024 V1 transcriptome (4,459 multiexonic and 1,177 monoexonic). Experimental evidence from public databases (Table S7) was incorporated and used to validate the predicted models. The final MAKER-P prediction included 38,227 high-quality gene models (AED &lt; 0.5) on the primary contigs and 26,789 on the associated haplotigs. Using the covariance models from the Rfam database, 5,780 non-overlapping putative long non-coding RNAs (ncRNAs; 3,239 on primary contigs and 2,541 on haplotigs) belonging to 275 different families were annotated (Table S9). Gene models were further improved using the information from all Iso-Seq full-length datasets (PCIRs, C-FLNC, FLNC), RNA-seq, and the publicly available grapevine transcriptome assemblies. This final refinement improved the annotation of the UTRs and added isoform information. PCIRs helped identify 155 new loci not detected by MAKER-P, update the structure of 10,801 gene models, and add 2,712 alternative transcripts. C-FLNC reads introduced 830 additional missing loci and added 3,738 alternative transcripts to the annotation. Together, 14,388 gene models were updated. FLNC reads introduced 14 new loci and 20,493 alternative transcripts, bringing the number of updated model structures to 24,945. Predicted genes without similarity to known proteins in the RefSeq database and without any functional domains identified by InterProScan (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Jones <em>et al.</em> 2014</a>) were removed. The final predicted transcriptome included 55,887 transcripts on 36,689 loci on primary contigs and 40,444 transcripts on 25,479 loci on haplotigs (Table S8). GO terms were assigned to 80,752 transcripts (83.8%) based on homology with protein domains in RefSeq and InterPro databases (Table S13, File S9-S11). A genome browser for Cabernet Sauvignon, its annotation, and an associated blast tool are available at <a href="http://cantulab.github.io/data">http://cantulab.github.io/data</a>.</p><p id="p-27">For 2,995 (11%) of the 25,479 protein-coding gene loci identified on the haplotigs, we could not find a corresponding homolog in the primary contigs, likely reflecting the diversity in gene content between parental genomes (Cabernet Franc and Sauvignon Blanc (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bowers and Meredith 1997</a>)), as in Corvina (2,321 transcripts; <a id="xref-ref-127-5" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>), Tannat (1,873 transcripts; <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>), and Nebbiolo (3,961 transcripts; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Gambino <em>et al.</em> 2017</a>). We could not find homologous genes in PN40024, Corvina, Tannat, and Nebbiolo for 1,714 protein-coding gene loci (Table S14). Those genes included likely members of the phenylpropanoid/flavonoid biosynthesis pathway: four phenylalanine ammonia-lyase, three chalcone synthases, a chalcone isomerase, a dihydroflavonol-4-reductase, a flavanone 3-hydroxylase, a flavonoid 3′,5′-hydroxylase, a leucoanthocyanidin dioxygenase, and an anthocyanin acyltransferase. Other private Cabernet Sauvignon genes were associated with terpenoid biosynthesis, including six valencene synthases that may play a role in grapevine flower aroma, three vinorine synthases (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Lücker <em>et al.</em> 2004</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Martin <em>et al.</em> 2009</a>), and a (E,E)-geranyllinalool synthase.</p><p id="p-28">The incorporation of Iso-Seq data in the gene prediction pipeline also allowed the structural annotation of alternative transcripts. Twenty five percent (15,509) of the 62,168 annotated gene loci had two or more alternative isoforms, an average of 1.55 ± 1.29 alternative transcripts per locus, confirming previous reports in PN40024 (<a id="xref-ref-129-2" class="xref-bibr" href="#ref-129">Vitulo <em>et al.</em> 2014</a>). The frequency of splicing variant types was similarly observed in other plant species (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Reddy <em>et al.</em> 2013</a>). Intron retention was the most abundant type, accounting for over 44% (File S12), similar to what was observed for rice (45-55%; <a id="xref-ref-141-1" class="xref-bibr" href="#ref-141">Zhang <em>et al.</em> 2015</a>), Arabidopsis (30-64%; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Marquez <em>et al.</em> 2012</a>; <a id="xref-ref-103-2" class="xref-bibr" href="#ref-103">Reddy <em>et al.</em> 2013</a>; <a id="xref-ref-141-2" class="xref-bibr" href="#ref-141">Zhang <em>et al.</em> 2015</a>) and maize (40-58%; <a id="xref-ref-141-3" class="xref-bibr" href="#ref-141">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-130-1" class="xref-bibr" href="#ref-130">Wang <em>et al.</em> 2016</a>). Alternative acceptor sites (13%), alternative donor sites (10%), and exon skipping (8%) were the other most abundant types of alternative splicing found in the Cabernet Sauvignon genome; a full description of the selected splicing events is reported in File S12.</p></div><div id="sec-12" class="subsection"><h3>Iso-Seq transcriptome as reference for RNA-seq analysis</h3><p id="p-29">The final step of the analysis was to evaluate the effectiveness of the reconstructed ISNT as reference for RNA-seq analysis of berry development compared to the gene space predicted on the Cabernet Sauvignon genome. Comparisons between the predicted transcripts and the reconstructed ISNT as references for RNA-seq are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Only about three percent more RNA-seq reads mapped on the Cabernet Sauvignon predicted transcriptome (90.6 ± 0.8%) than on the ISNT (87.2 ± 0.8%), suggesting that Iso-Seq reconstructed most of the transcripts detectable by RNA-seq at a coverage of ∼26 M reads/sample. Approximately 75% of the ISNT (21,494 transcripts) and ∼49% of the predicted gene space (30,501 gene loci) was detected as expressed (mean RPKM ≥ 1) in at least one stage (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, Table S15-S16). In both datasets, the number of expressed genes was slightly higher at pre-véraison stage than at later developmental stages, consistent with previous observations of ripening Cabernet Sauvignon berries (Fasoli <em>et al.</em> 2018). For both datasets, Pearson’s correlation matrix and Principal Component Analysis (PCA) showed a clear distinction between pre-véraison stage and the three ripening stages, as well as a stronger correlation between post-véraison and full-ripe berry transcriptomes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), confirming the well-known transcriptional reprogramming associated with the onset of ripening (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Fasoli <em>et al.</em> 2012</a>; <a id="xref-ref-88-4" class="xref-bibr" href="#ref-88">Massonnet <em>et al.</em> 2017a</a>) and suggesting that similar global transcriptomic dynamics of berry development can be obtained using either Iso-Seq or the whole genome as reference. We then applied a sequence clustering approach to define associations between ISNT isoforms and gene loci to directly compare the expression values of each gene in the two transcriptomes. Based on reciprocal overlap of the alignment, we were able to associate 25,306 ISNT transcripts with 26,873 gene loci in the Cabernet Sauvignon genome (Table S10). Gene expression levels measured on the two transcriptomes were well-correlated (<em>R</em> = 0.92; <em>P</em>-value &lt; 2.2 e-16; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>; Tables S17-18). Differential gene expression analysis identified 14,477 ISNT transcripts and 18,600 Cabernet Sauvignon genes significantly differentially expressed (PPDE ≥ 0.95) at least once during berry development (Table S15-S16). More genes were differentially regulated between pre-véraison and véraison than during ripening for both transcriptomes, (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A &amp; B</a>) as previously observed (<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Palumbo <em>et al.</em> 2014</a>; <a id="xref-ref-88-5" class="xref-bibr" href="#ref-88">Massonnet <em>et al.</em> 2017a</a>). Ninety one percent of the differentially expressed ISNT isoforms were also differentially expressed when RNA-seq data were mapped on genomic loci. Similar relative amounts of Biological Process GO terms among differentially expressed genes were observed between the two transcriptomes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C &amp; D</a>). Interestingly, 302 Cabernet Sauvignon private isoforms (transcripts not found in other cultivars) were differentially expressed during berry development, including transcripts encoding a polyol transporter, an inositol transporter, and five aquaporins.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492351/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492351/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the RNA-seq results when Cabernet Sauvignon berry ISNT and the Cabernet Sauvignon genome were used as reference for short-read mapping</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/755/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene expression analysis during berry development using the ISNT as reference and comparison with the Cabernet Sauvignon genome. (A) Number of ISNT transcripts (ISNTs) and gene loci expressed overall and at each ripening stage. Abbreviation of berry growth stages: PRV, Pre-v&#xE9;raison; V, V&#xE9;raison; PV, Post- v&#xE9;raison; H, harvest. (B) Scatterplot showing the correlation between the gene expression level (log2(RPKM + 1)) of ISNT isoforms and Cabernet Sauvignon (CS) gene loci. Line of best fit and correlation coefficient factor (R) are provided." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-902267164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene expression analysis during berry development using the ISNT as reference and comparison with the Cabernet Sauvignon genome. (A) Number of ISNT transcripts (ISNTs) and gene loci expressed overall and at each ripening stage. Abbreviation of berry growth stages: PRV, Pre-v&#xE9;raison; V, V&#xE9;raison; PV, Post- v&#xE9;raison; H, harvest. (B) Scatterplot showing the correlation between the gene expression level (log&lt;sub&gt;2&lt;/sub&gt;(RPKM + 1)) of ISNT isoforms and Cabernet Sauvignon (CS) gene loci. Line of best fit and correlation coefficient factor (&lt;em&gt;R&lt;/em&gt;) are provided.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/755/F3.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/755/F3.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/755/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/755/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Gene expression analysis during berry development using the ISNT as reference and comparison with the Cabernet Sauvignon genome. (A) Number of ISNT transcripts (ISNTs) and gene loci expressed overall and at each ripening stage. Abbreviation of berry growth stages: PRV, Pre-véraison; V, Véraison; PV, Post- véraison; H, harvest. (B) Scatterplot showing the correlation between the gene expression level (log<sub>2</sub>(RPKM + 1)) of ISNT isoforms and Cabernet Sauvignon (CS) gene loci. Line of best fit and correlation coefficient factor (<em>R</em>) are provided.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/755/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global transcriptomic changes during berry development ripening using the ISNT and Cabernet Sauvignon gene space as references. (A, B) Pearson correlation matrices of the 16 berry transcriptomes. (C, D) PCA plots of the 16 berry transcriptomes. Each point represents a biological replicate and ellipses define confidence areas (95%) for each berry growth stage. All analyses were performed using the log2-transformed RPKM values of expressed features (mean RPKM &#x2265; 1 at least at one stage)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-902267164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Global transcriptomic changes during berry development ripening using the ISNT and Cabernet Sauvignon gene space as references. (A, B) Pearson correlation matrices of the 16 berry transcriptomes. (C, D) PCA plots of the 16 berry transcriptomes. Each point represents a biological replicate and ellipses define confidence areas (95%) for each berry growth stage. All analyses were performed using the log&lt;sub&gt;2&lt;/sub&gt;-transformed RPKM values of expressed features (mean RPKM &#x2265; 1 at least at one stage).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/755/F4.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/3/755/F4.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/755/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/755/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Global transcriptomic changes during berry development ripening using the ISNT and Cabernet Sauvignon gene space as references. (A, B) Pearson correlation matrices of the 16 berry transcriptomes. (C, D) PCA plots of the 16 berry transcriptomes. Each point represents a biological replicate and ellipses define confidence areas (95%) for each berry growth stage. All analyses were performed using the log<sub>2</sub>-transformed RPKM values of expressed features (mean RPKM ≥ 1 at least at one stage).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/755/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the differentially expressed (DE) ISNT transcripts and Cabernet Sauvignon gene loci during berry development. (A, B) Number of up- and down-regulated genes between each pairwise comparison of developmental stages (PPDE &#x2265; 0.95 and mean RPKM &#x2265; 1 at least at one stage). (C, D) Relative amount of Biological Process Gene Ontology (GO) terms among differentially expressed genes. Abbreviation of berry growth stages: PRV, Pre-v&#xE9;raison; V, V&#xE9;raison; PV, Post- v&#xE9;raison; H, harvest." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-902267164" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the differentially expressed (DE) ISNT transcripts and Cabernet Sauvignon gene loci during berry development. (A, B) Number of up- and down-regulated genes between each pairwise comparison of developmental stages (PPDE &#x2265; 0.95 and mean RPKM &#x2265; 1 at least at one stage). (C, D) Relative amount of Biological Process Gene Ontology (GO) terms among differentially expressed genes. Abbreviation of berry growth stages: PRV, Pre-v&#xE9;raison; V, V&#xE9;raison; PV, Post- v&#xE9;raison; H, harvest.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/755/F5.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/3/755/F5.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/755/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/755/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Comparison of the differentially expressed (DE) ISNT transcripts and Cabernet Sauvignon gene loci during berry development. (A, B) Number of up- and down-regulated genes between each pairwise comparison of developmental stages (PPDE ≥ 0.95 and mean RPKM ≥ 1 at least at one stage). (C, D) Relative amount of Biological Process Gene Ontology (GO) terms among differentially expressed genes. Abbreviation of berry growth stages: PRV, Pre-véraison; V, Véraison; PV, Post- véraison; H, harvest.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-36">Full-length cDNA sequencing with SMRT technology (Iso-Seq) can be used to rapidly reconstruct the grape berry transcriptome, enabling the identification of cultivar-specific isoforms, refinement of the Cabernet Sauvignon genome annotation, and the creation of a reference for transcriptome-wide expression profiling. In contrast to transcriptome reconstruction using short-read sequencing that requires <em>de novo</em> assembly, Iso-Seq delivers full-length transcripts that eliminate the introduction of assembly errors and artifacts like chimeric transcripts and incomplete fragments due to PolyA capture (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chang <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Huang <em>et al.</em> 2016</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Moreton <em>et al.</em> 2016</a>; <a id="xref-ref-116-1" class="xref-bibr" href="#ref-116">Smith-Unna <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Geniza and Jaiswal 2017</a>; <a id="xref-ref-125-1" class="xref-bibr" href="#ref-125">Ungaro <em>et al.</em> 2017</a>). The incorporation of high-coverage short-read sequencing is still necessary to benefit from the complete transcript sequencing enable by Iso-Seq. Although Iso-Seq provides much longer reads than second-generation sequencing platforms and as a result is excellent in resolving transcript structure, its sequencing error rate is high (10–20%) and throughput is still relatively low (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Giordano <em>et al.</em> 2017</a>; <a id="xref-ref-143-2" class="xref-bibr" href="#ref-143">Zimin <em>et al.</em> 2017</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Koren <em>et al.</em> 2017</a>). Here we show that combining Iso-Seq with Illumina sequencing at high coverage enables expression profiling and sequence error correction of Iso-Seq reads, particularly those derived from low-expression genes. The clustering analysis of the SMRT link pipeline discarded ∼18.5% of the FLNC reads, likely caused by low sequence accuracy. To overcome this technical issue, we applied a hybrid error correction pipeline consisting in performing the error correction of the unclustered FLNC reads, followed by an additional clustering step of both to resolve redundancies. Error correction with Illumina reads recovered a significant amount of Iso-Seq reads that would have otherwise been removed by the standard Iso-Seq pipeline, highlighting the importance of integrating multiple sequencing technologies with complementary features (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Koren <em>et al.</em> 2012</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Au <em>et al.</em> 2012</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Salmela and Rivals 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Hu <em>et al.</em> 2016</a>).</p><p id="p-37">Transcriptome reconstruction has been widely used to develop references for genome-wide expression profiling in the absence of an annotated genome assembly (<a id="xref-ref-113-1" class="xref-bibr" href="#ref-113">Simon <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Garber <em>et al.</em> 2011</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Martin and Wang 2011</a>; <a id="xref-ref-138-1" class="xref-bibr" href="#ref-138">Yang and Kim 2015</a>). Though a genome reference is available for grape, transcriptome reconstruction overcomes the limitations of a cultivar-specific reference that lacks the gene content of other cultivars. Although cultivar-specific genes appear nonessential for berry development, those private genes could contribute to cultivar characteristics. For example, the wine grape Tannat accumulates unusually high levels of polyphenols in the berry; its cultivar-specific genes account for more than 80% of the expression of phenolic and polyphenolic compound biosynthetic enzymes (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>). <em>De novo</em> transcriptome assembly from short RNA-seq reads has been used to explore the gene content diversity in Tannat (<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Da Silva <em>et al.</em> 2013</a>), Corvina (<a id="xref-ref-127-6" class="xref-bibr" href="#ref-127">Venturini <em>et al.</em> 2013</a>), and Nebbiolo (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Gambino <em>et al.</em> 2017</a>). Iso-Seq identified 1,501 Cabernet Sauvignon transcripts expressed during berry development that were found in neither the genome of PN40024 nor the transcriptomes of Tannat, Nebbiolo and Corvina. Some private Cabernet Sauvignon transcripts have functions potentially associated with traits characteristic of Cabernet Sauvignon grapes and wines like their color and sugar content. These transcripts included three sugar transporter-coding genes, which could be involved in the accumulation of glucose and fructose during berry ripening (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Lecourieux <em>et al.</em> 2014</a>), and a chalcone synthase, a flavanone 3-hydroxylase, and a flavonoid 3′-hydroxylase, all involved in the flavonoid pathway. Chalcone synthases catalyze the first committed step of the flavonoid biosynthesis pathway (<a id="xref-ref-117-1" class="xref-bibr" href="#ref-117">Sparvoli <em>et al.</em> 1994</a>), which produces different classes of metabolites in grape berry, including flavonols (yellow pigments), flavan-3-ols and proanthocyanidins (mouth-feel and smooth sensory perceptions), and anthocyanins (red, purple, and blue pigments). In addition, products of the flavonoid 3′-hydroxylase can lead to the synthesis of cyanidin-3-glucoside, a red anthocyanin (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Castellarin <em>et al.</em> 2012</a>). The analysis of the gene space in the genome assembly showed that private Cabernet Sauvignon genes identified using Iso-Seq are only a fraction of the private Cabernet Sauvignon transcriptome. As in other transcriptome reconstruction methods, Iso-Seq can only identify transcripts that are expressed in the organs and developmental stages used for RNA sequencing. Obtaining the full set of private transcripts without genome assembly would require sequencing additional organs and developmental stages. In addition, it is challenging to differentiate isoforms derived from close paralogous genes, alleles of the same gene, and alternative splicing variants, in any transcriptome obtained by RNA sequencing (including Iso-Seq); this potentially leads to an overestimation of the genes in the final transcriptome reference. This study could not resolve isoform redundancy in the final transcriptome for about 37% of the gene loci in the Cabernet Sauvignon genome. This is a limitation of Iso-Seq as well as of all transcriptome references that cannot be overcome without a complete genome assembly.</p><p id="p-38">In this study, we tested whether the transcriptome reconstructed using Iso-Seq can be used for expression profiling. Only an approximately 3% difference in read alignment between ISNT and the genome reference was observed, implying that at high coverage, ISNT detects almost all genes expressed during berry development. The slight difference in mapping rate between the two references can be explained by either the absence of some low-expression transcripts in the ISNT or the residual error rate in isoform sequences. Gene expression analysis using the ISNT as reference showed similar results compared to the Cabernet Sauvignon genome assembly, with a very high correlation of expression level and differential gene expression, and with similar global transcriptomic changes. However, we observed differences in the number of expressed and differentially expressed features that depend on the reference used. Those differences could be explained by the diploid phasing of the Cabernet Sauvignon genome assembly and that multiple ISNT transcripts might correspond to a single gene locus. Nonetheless, similar relative amounts of Biological Process GO terms were found among the differentially expressed genes, confirming that the transcriptome obtained using Iso-Seq captured the transcriptional reprogramming underlying the main physiological and biochemical changes during grape berry development. In addition, gene expression analysis revealed that some private isoforms (20%) are significantly modulated during berry development, indicating that in addition to identifying the private gene space, the ISNT reference makes it possible to observe its expression.</p><p id="p-39">In conclusion, this study demonstrates that Iso-Seq data can be used to create and refine a comprehensive reference transcriptome that represents most genes expressed in a tissue undergoing extensive transcriptional reprogramming during development. In grapes, this approach can aid developing transcriptome references and is particularly valuable given diverse cultivars with private transcripts and accessions that are genetically distant from available genome references, like the non<em>-vinifera Vitis</em> species used as rootstocks or for breeding. The pipeline described here can be useful in efforts to reconstruct the gene space in plant species with large and complex genomes still unresolved.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by J. Lohr Vineyards and Wines, E. &amp; J. Gallo Winery, the Louis P. Martini Endowment in Viticulture, and the NSF grant #1741627. Part of this work was carried out in collaboration with UC Davis Chile and funded by the Chilean Economic Development Agency (CORFO).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Equal contributors</p></li><li class="fn" id="fn-5"><p id="p-41">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7291943">https://doi.org/10.25387/g3.7291943</a>.</p></li><li class="fn" id="fn-6"><p id="p-42"><em>Communicating editor: T. Slotte</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-31"><span class="received-label">Received </span>October 31, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-09"><span class="accepted-label">Accepted </span>January 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Minio <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.1" data-doi="10.1186/1471-2229-10-245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Afoufa-Bastien</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medici</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeauffre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coutos-Thévenot</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemoine</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Vitis vinifera sugar transporter gene family: Phylogenetic overview and macroarray expression profiling.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-10-245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DAfoufa-Bastien%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.spage%253D245%26rft.epage%253D245%26rft.atitle%253DThe%2BVitis%2Bvinifera%2Bsugar%2Btransporter%2Bgene%2Bfamily%253A%2Bphylogenetic%2Boverview%2Band%2Bmacroarray%2Bexpression%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-10-245%26rft_id%253Dinfo%253Apmid%252F21073695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-10-245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21073695&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.3" data-doi="10.1038/hortres.2015.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amrine</span>  <span class="cit-name-given-names">K. C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco-Ulate</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riaz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pap</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative transcriptomics of Central Asian Vitis vinifera accessions reveals distinct defense strategies against powdery mildew.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">15037</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hortres.2015.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D2%26rft.spage%253D15037%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhortres.2015.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hortres.2015.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.4" data-doi="10.1186/1471-2164-13-571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">De novo assembly of the pepper transcriptome (Capsicum annuum): a benchmark for in silico discovery of SNPs, SSRs and candidate genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-571</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAshrafi%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D571%26rft.epage%253D571%26rft.atitle%253DDe%2Bnovo%2Bassembly%2Bof%2Bthe%2Bpepper%2Btranscriptome%2B%2528Capsicum%2Bannuum%2529%253A%2Ba%2Bbenchmark%2Bfor%2Bin%2Bsilico%2Bdiscovery%2Bof%2BSNPs%252C%2BSSRs%2Band%2Bcandidate%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-571%26rft_id%253Dinfo%253Apmid%252F23110314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23110314&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.5" data-doi="10.1371/journal.pone.0046679"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Au</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving PacBio long read accuracy by short read alignment.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46679</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046679</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAu%26rft.auinit1%253DK.%2BF.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46679%26rft.epage%253De46679%26rft.atitle%253DImproving%2BPacBio%2BLong%2BRead%2BAccuracy%2Bby%2BShort%2BRead%2BAlignment.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046679%26rft_id%253Dinfo%253Apmid%252F23056399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046679&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056399&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.6" data-doi="10.1105/tpc.114.124040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellucci</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bitocchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrarini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benazzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biagetti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Decreased Nucleotide and Expression Diversity and Modified Coexpression Patterns Characterize Domestication in the Common Bean.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1901</span>–<span class="cit-lpage">1912</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.124040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.124040%26rft_id%253Dinfo%253Apmid%252F24850850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzUvMTkwMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.7" data-doi="10.1104/pp.15.00852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanco-Ulate</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amrine</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivero</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vicente</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, 2015 Developmental and metabolic plasticity of white-skinned grape berries in response to Botrytis cinerea during noble rot. Plant Physiol. 169: pp.00852.2015.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.00852</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00852%26rft_id%253Dinfo%253Apmid%252F26450706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY5LzQvMjQyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.8" data-doi="10.1093/jxb/erw506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanco-Ulate</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopfer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa-Balderas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivero</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Red blotch disease alters grape berry development and metabolism by interfering with the transcriptional and hormonal regulation of ripening.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1225</span>–<span class="cit-lpage">1238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erw506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft.volume%253D68%26rft.spage%253D1225%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.9" data-doi="10.3389/fpls.2013.00366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanco-Ulate</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincenti</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">A. L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantu</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Tomato transcriptome and mutant analyses suggest a role for plant stress hormones in the interaction between fruit and Botrytis cinerea.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2013.00142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2013.00366%26rft_id%253Dinfo%253Apmid%252F23346092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2013.00366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23346092&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.10" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.11" data-doi="10.1038/ng0597-84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meredith</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The parentage of a classic wine grape, Cabernet Sauvignon.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">84</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng0597-84</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBowers%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D84%26rft.epage%253D87%26rft.atitle%253DThe%2Bparentage%2Bof%2Ba%2Bclassic%2Bwine%2Bgrape%252C%2BCabernet%2BSauvignon.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0597-84%26rft_id%253Dinfo%253Apmid%252F9140400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0597-84&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9140400&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WX02400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.12" data-doi="10.1038/nrg2220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Steady progress and recent breakthroughs in the accuracy of automated genome annotation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrent%26rft.auinit1%253DM.%2BR.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D73%26rft.atitle%253DSteady%2Bprogress%2Band%2Brecent%2Bbreakthroughs%2Bin%2Bthe%2Baccuracy%2Bof%2Bautomated%2Bgenome%2Bannotation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2220%26rft_id%253Dinfo%253Apmid%252F18087260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18087260&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252065700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.13" data-doi="10.1038/ng803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pospisil</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valcárcel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Alternative splicing and genome complexity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng803</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBrett%26rft.auinit1%253DD.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D30%26rft.atitle%253DAlternative%2Bsplicing%2Band%2Bgenome%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng803%26rft_id%253Dinfo%253Apmid%252F11743582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11743582&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173105600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.755.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome Annotation and Curation Using MAKER and MAKER-P</span>, pp. <span class="cit-fpage">4.11.1</span>–<span class="cit-lpage">4.11.39</span> in <span class="cit-source">Current Protocols in Bioinformatics</span>, <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.15" data-doi="10.1104/pp.113.230144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">MAKER-P: A tool kit for the rapid creation, management, and quality control of plant genome annotations.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.113.230144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.230144%26rft_id%253Dinfo%253Apmid%252F24306534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMi81MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.16" data-doi="10.1104/pp.108.3.1099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilde</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Tomato exo-(1–4)-b-D-galactanase. Isolation, changes during ripening in normal and mutant tomato fruit, and characterization of a related cDNA clone.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.108.3.1099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DCarey%26rft.auinit1%253DA.%2BT.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D1099%26rft.epage%253D1107%26rft.atitle%253DTomato%2BExo-%25281-%253E4%2529-%255Bbeta%255D-D-Galactanase%2B%2528Isolation%252C%2BChanges%2Bduring%2BRipening%2Bin%2BNormal%2Band%2BMutant%2BTomato%2BFruit%252C%2Band%2BCharacterization%2Bof%2Ba%2BRelated%2BcDNA%2BClone%2529%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.3.1099%26rft_id%253Dinfo%253Apmid%252F7630937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzMvMTA5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castellarin</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bavaresco</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falginella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">M. I. V. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Gaspero</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Phenolics in grape berry and key antioxidants | BenthamScience. Biochem. Grape Berry 89–110.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.18" data-doi="10.1371/journal.pone.0089680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The impacts of read length and transcriptome complexity for de ovo assembly: A simulation study.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0094825</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0089680%26rft_id%253Dinfo%253Apmid%252F24551193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0089680&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24551193&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.19" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klammer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.20" data-doi="10.1038/nmeth.4035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nattestad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Concepcion</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phased diploid genome assembly with single-molecule real-time sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1050</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D1050%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4035%26rft_id%253Dinfo%253Apmid%252F27749838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27749838&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.21" data-doi="10.1007/s00122-010-1411-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cipriani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spadotto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Gaspero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crespan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The SSR-based molecular profile of 1005 grapevine (Vitis vinifera L.) accessions uncovers new synonymy and parentages, and reveals a large admixture amongst varieties of different geographic origin.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1585</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-010-1411-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DCipriani%26rft.auinit1%253DG.%26rft.volume%253D121%26rft.issue%253D8%26rft.spage%253D1569%26rft.epage%253D1585%26rft.atitle%253DThe%2BSSR-based%2Bmolecular%2Bprofile%2Bof%2B1005%2Bgrapevine%2B%2528Vitis%2Bvinifera%2BL.%2529%2Baccessions%2Buncovers%2Bnew%2Bsynonymy%2Band%2Bparentages%252C%2Band%2Breveals%2Ba%2Blarge%2Badmixture%2Bamongst%2Bvarieties%2Bof%2Bdifferent%2Bgeographic%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1411-9%26rft_id%253Dinfo%253Apmid%252F20689905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1411-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20689905&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.22" data-doi="10.1093/jxb/erv274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vannozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitulo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meggio</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comprehensive transcript profiling of two grapevine rootstock genotypes contrasting in drought susceptibility links the phenylpropanoid pathway to enhanced tolerance.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">5739</span>–<span class="cit-lpage">5752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erv274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferv274%26rft_id%253Dinfo%253Apmid%252F26038306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erv274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26038306&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.23" data-doi="10.1038/nrm1746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cosgrove</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Growth of the plant cell wall.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">850</span>–<span class="cit-lpage">861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrm1746</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DCosgrove%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D850%26rft.epage%253D861%26rft.atitle%253DGrowth%2Bof%2Bthe%2Bplant%2Bcell%2Bwall.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1746%26rft_id%253Dinfo%253Apmid%252F16261190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16261190&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233194900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.24" data-doi="10.1038/35030000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cosgrove</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Loosening of plant cell walls by expansins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">407</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35030000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCosgrove%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D407%26rft.issue%253D6802%26rft.spage%253D321%26rft.epage%253D326%26rft.atitle%253DLoosening%2Bof%2Bplant%2Bcell%2Bwalls%2Bby%2Bexpansins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35030000%26rft_id%253Dinfo%253Apmid%252F11014181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35030000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11014181&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089390700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.25" data-doi="10.1105/tpc.113.118810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamperin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrarini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dal Molin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The high polyphenol content of grapevine cultivar tannat berries is conferred primarily by genes that are not shared with the reference genome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4777</span>–<span class="cit-lpage">4788</span> (erratum Plant Cell 29: 913).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.118810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.118810%26rft_id%253Dinfo%253Apmid%252F24319081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMi80Nzc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.26" data-doi="10.1186/1471-2164-10-212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deluc</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quilici</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decendit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimplet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheatley</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Water deficit alters differentially metabolic pathways affecting important flavor and quality traits in grape berries of Cabernet Sauvignon and Chardonnay.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-10-212</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDeluc%26rft.auinit1%253DL.%2BG.%26rft.volume%253D10%26rft.spage%253D212%26rft.epage%253D212%26rft.atitle%253DWater%2Bdeficit%2Balters%2Bdifferentially%2Bmetabolic%2Bpathways%2Baffecting%2Bimportant%2Bflavor%2Band%2Bquality%2Btraits%2Bin%2Bgrape%2Bberries%2Bof%2BCabernet%2BSauvignon%2Band%2BChardonnay.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-212%26rft_id%253Dinfo%253Apmid%252F19426499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19426499&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.27" data-doi="10.1007/s11032-004-1362-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Gaspero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cipriani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marrazzo</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreetta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prado Castro</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isolation of (AC)n-microsatellites in Vitis vinifera L. and analysis of genetic background in grapevines under marker assisted selection.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-004-1362-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D15%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-004-1362-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-004-1362-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.28" data-doi="10.1186/1471-2229-14-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Genova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Espinoza</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vizoso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travisany</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome comparison between table and wine grapes reveals a comprehensive catalog of structural variants.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-14-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.29" data-doi="10.1093/bioinformatics/btt647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monjo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoang</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshimura</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurino</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rapid detection of expanded short tandem repeats in personal genomics using hybrid sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt647%26rft_id%253Dinfo%253Apmid%252F24215022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24215022&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333249900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.30" data-doi="10.1186/s12864-015-2257-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-molecule real-time transcript sequencing facilitates common wheat genome annotation and grain transcriptome research.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2257-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1039%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2257-y%26rft_id%253Dinfo%253Apmid%252F26645802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2257-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26645802&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.31" data-doi="10.1186/1471-2105-9-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellinghaus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEllinghaus%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DLTRharvest%252C%2Ban%2Befficient%2Band%2Bflexible%2Bsoftware%2Bfor%2Bde%2Bnovo%2Bdetection%2Bof%2BLTR%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-18%26rft_id%253Dinfo%253Apmid%252F18194517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18194517&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.32" data-doi="10.1007/s00425-012-1658-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falginella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Gaspero</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castellarin</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Expression of flavonoid genes in the red grape berry of “Alicante Bouschet” varies with the histological distribution of anthocyanins and their chemical composition.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">236</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00425-012-1658-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D236%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-012-1658-2%26rft_id%253Dinfo%253Apmid%252F22552639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-012-1658-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22552639&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309229300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.33" data-doi="10.1105/tpc.112.100230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dal Santo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornielli</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farina</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The grapevine expression atlas reveals a deep transcriptome shift driving the entire plant into a maturation program.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3489</span>–<span class="cit-lpage">3505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.112.100230</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.100230%26rft_id%253Dinfo%253Apmid%252F22948079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMzQ4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.3.755.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitulo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, 2018 Timing and order of the molecular events marking the onset of berry ripening in grapevine. Plant Physiol. 17: pp.00559.2018.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.35" data-doi="10.3389/fpls.2018.00005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filichkin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">Mi.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dharmawardhana</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Abiotic stresses modulate landscape of poplar transcriptome via alternative splicing, differential intron retention, and isoform ratio switching.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.36" data-doi="10.1093/nar/gkm311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foissac</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">ASTALAVISTA: Dynamic and flexible analysis of alternative splicing events in custom gene datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W297</span>–<span class="cit-lpage">W299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm311%26rft_id%253Dinfo%253Apmid%252F17485470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.37" data-doi="10.1038/s41598-017-17405-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gambino</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dal Molin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boccacci</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitarra</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Whole-genome sequencing and SNV genotyping of ‘Nebbiolo’ (Vitis vinifera L.) clones.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">17294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-17405-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D17294%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-17405-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-17405-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.38" data-doi="10.1080/15476286.2016.1197481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PacBio full-length transcriptome profiling of insect mitochondrial gene expression.</span> <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">825</span> (erratum: RNA Biol. 13: 1323).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15476286.2016.1197481</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol.%26rft.volume%253D13%26rft.spage%253D820%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1197481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1197481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.39" data-doi="10.1038/nmeth.1613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Computational methods for transcriptome annotation and quantification using RNA-seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1613</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGarber%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D469%26rft.epage%253D477%26rft.atitle%253DComputational%2Bmethods%2Bfor%2Btranscriptome%2Bannotation%2Band%2Bquantification%2Busing%2BRNA-seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1613%26rft_id%253Dinfo%253Apmid%252F21623353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291031800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.40" data-doi="10.1016/j.cpb.2017.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geniza</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaiswal</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Tools for building de novo transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">Curr. Plant Biol.</abbr> <span class="cit-vol">11–12</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cpb.2017.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BPlant%2BBiol.%26rft.volume%253D1112%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpb.2017.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpb.2017.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.3.755.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Gene-omes built from mRNA-seq not genome DNA, pp. <span class="cit-fpage">47405</span> in <em>7th Annual Arthropod Genomics Symposium</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.42" data-doi="10.1038/s41598-017-03996-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giordano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aigrain</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonfield</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">De novo yeast genome assemblies from MinION, PacBio and MiSeq platforms.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3935</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-03996-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D3935%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-03996-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-03996-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.43" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1017351108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.44" data-doi="10.1111/pbi.12499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golicz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Towards plant pangenomics.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12499</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D1099%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.45" data-doi="10.1038/ncomms13390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Golicz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edger</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">The pangenome of an agronomically important crop plant Brassica oleracea.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms13390</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D13390%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms13390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms13390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.46" data-doi="10.1126/science.aae0344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronenberg</span>  <span class="cit-name-given-names">Z. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Long-read sequence assembly of the gorilla genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">aae0344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aae0344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGordon%26rft.auinit1%253DD.%26rft.volume%253D352%26rft.issue%253D6281%26rft.spage%253Daae0344%26rft.epage%253Daae0344%26rft.atitle%253DLong-read%2Bsequence%2Bassembly%2Bof%2Bthe%2Bgorilla%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aae0344%26rft_id%253Dinfo%253Apmid%252F27034376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTIvNjI4MS9hYWUwMzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.47" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.48" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14500829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185600400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.49" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2013.084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.50" data-doi="10.1093/nar/gkq862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MITE-Hunter: A program for discovering miniature inverted-repeat transposable elements from genomic sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq862%26rft_id%253Dinfo%253Apmid%252F20880995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20880995&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.51" data-doi="10.1371/journal.pone.0146062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honaas</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wafula</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickett</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Der</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Selecting superior de novo transcriptome assemblies: Lessons learned by leveraging the best plant genome.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0146062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0146062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0146062%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0146062%26rft_id%253Dinfo%253Apmid%252F26731733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0146062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26731733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.52" data-doi="10.1186/s12870-016-0804-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hopper</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlauch</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cramer</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcriptomic network analyses of leaf dehydration responses identify highly connected ABA and ethylene signaling hubs in three grapevine species differing in drought tolerance.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-016-0804-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D16%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-016-0804-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-016-0804-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.53" data-doi="10.1186/s12859-016-1316-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">LSCplus: a fast solution for improving long read accuracy by short read alignment.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1316-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1316-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1316-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.54" data-doi="10.1186/s12864-016-2923-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armbruster</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative performance of transcriptome assembly methods for non-model organisms.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2923-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2923-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2923-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.55" data-doi="10.1101/gr.133652.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HaploMerger: Reconstructing allelic relationships for polymorphic diploid genome assemblies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1581</span>–<span class="cit-lpage">1588</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.133652.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.133652.111%26rft_id%253Dinfo%253Apmid%252F22555592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU4MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.56" data-doi="10.1101/gr.214007.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinberg</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekzema</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Discovery and genotyping of structural variation from long-read haploid genome sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.214007.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.214007.116%26rft_id%253Dinfo%253Apmid%252F27895111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNjc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.57" data-doi="10.1101/gr.5969107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auch</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEGAN analysis of metagenomic data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5969107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5969107%26rft_id%253Dinfo%253Apmid%252F17255551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvMzc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vargas</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palancar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borrego</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Andrés</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic relationships among table-grape varieties.</span> <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BEnology%2Band%2BViticulture%26rft.stitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft.aulast%253DIbanez%26rft.auinit1%253DJ.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D42%26rft.atitle%253DGenetic%2BRelationships%2Bamong%2BTable-Grape%2BVarieties%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo3OiI2MC8xLzM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.59" data-doi="10.1038/nature06148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Policriti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clepet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06148</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D449%26rft.issue%253D7161%26rft.spage%253D463%26rft.epage%253D467%26rft.atitle%253DThe%2Bgrapevine%2Bgenome%2Bsequence%2Bsuggests%2Bancestral%2Bhexaploidization%2Bin%2Bmajor%2Bangiosperm%2Bphyla.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06148%26rft_id%253Dinfo%253Apmid%252F17721507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17721507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249724800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.60" data-doi="10.1186/s12864-015-1442-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fei</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome characterization of three wild Chinese Vitis uncovers a large number of distinct disease related genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1442-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D223%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1442-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1442-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.61" data-doi="10.1093/bioinformatics/btu031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binns</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">InterProScan 5: genome-scale protein function classification.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu031%26rft_id%253Dinfo%253Apmid%252F24451626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451626&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.62" data-doi="10.1109/TCBB.2016.2591533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ju</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient approach to correct read alignment for pseudogene abundance estimates.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinforma.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TCBB.2016.2591533</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinforma.%26rft.volume%253D14%26rft.spage%253D522%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2016.2591533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2016.2591533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.63" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toshimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.170720.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.64" data-doi="10.1038/nbt.2280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid error correction and de novo assembly of single-molecule sequencing reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">700</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2280</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKoren%26rft.auinit1%253DS.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D693%26rft.epage%253D700%26rft.atitle%253DHybrid%2Berror%2Bcorrection%2Band%2Bde%2Bnovo%2Bassembly%2Bof%2Bsingle-molecule%2Bsequencing%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2280%26rft_id%253Dinfo%253Apmid%252F22750884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22750884&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.65" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k -mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.66" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.67" data-doi="10.1093/bioinformatics/17.suppl_1.S140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Integrating genomic homology into gene structure prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">S140</span>–<span class="cit-lpage">S148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.suppl_1.S140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D17%26rft.issue%253D90001%26rft.spage%253DS140%26rft.epage%253D148%26rft.atitle%253DIntegrating%2Bgenomic%2Bhomology%2Binto%2Bgene%2Bstructure%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.suppl_1.S140%26rft_id%253Dinfo%253Apmid%252F11473003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.suppl_1.S140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11473003&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.68" data-doi="10.1093/jxb/ert395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lauvergeat</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Berry ripening: Recently heard through the grapevine.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">4543</span>–<span class="cit-lpage">4559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ert395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert395%26rft_id%253Dinfo%253Apmid%252F24285825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ert395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24285825&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.69" data-doi="10.1186/s12864-017-3691-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eory</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paton</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Archibald</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Normalized long read RNA sequencing in chicken reveals transcriptome complexity similar to human.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3691-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3691-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3691-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.70" data-doi="10.1007/s00122-012-1988-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacombe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boursiquot</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laucou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Vecchi-Staraz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Péros</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large-scale parentage analysis in an extended set of grapevine cultivars (Vitis vinifera L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">414</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1988-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1988-2%26rft_id%253Dinfo%253Apmid%252F23015217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1988-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23015217&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.71" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.72" data-doi="10.18637/jss.v025.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lê</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Josse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husson</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FactoMineR : An R Package for Multivariate Analysis.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v025.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v025.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v025.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.73" data-doi="10.1093/jxb/ert394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lecourieux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kappel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecourieux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An update on sugar transport and signalling in grapevine.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ert394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fert394%26rft_id%253Dinfo%253Apmid%252F24323501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ert394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24323501&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.74" data-doi="10.3389/fpls.2017.00053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lecourieux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kappel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pieri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dissecting the biochemical and transcriptomic effects of a locally applied heat treatment on developing Cabernet Sauvignon grape berries.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.75" data-doi="10.1093/bioinformatics/btt087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leng</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruotti</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rissman</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">EBSeq: An empirical Bayes hierarchical model for inference in RNA-seq experiments.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt087%26rft_id%253Dinfo%253Apmid%252F23428641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23428641&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.76" data-doi="10.1186/1471-2105-12-323"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-323</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.spage%253D323%26rft.epage%253D323%26rft.atitle%253DRSEM%253A%2Baccurate%2Btranscript%2Bquantification%2Bfrom%2BRNA-Seq%2Bdata%2Bwith%2Bor%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-323%26rft_id%253Dinfo%253Apmid%252F21816040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21816040&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.77" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btl158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, 2016 Alternative splicing of rice WRKY62 and WRKY76 transcription factor genes in pathogen defense. Plant Physiol. 171: pp.01921.2015.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.79" data-doi="10.1111/1755-0998.12670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Detecting alternatively spliced transcript isoforms from single-molecule long-read sequences without a reference genome.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D1243%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.80" data-doi="10.1007/BF00220990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lodhi</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisch</span>  <span class="cit-name-given-names">B. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Nuclear DNA content of Vitis species, cultivars, and other genera of the Vitaceae.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00220990</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D90%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220990&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QF11000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.81" data-doi="10.1093/nar/gki937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernoff</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene identification in novel eukaryotic genomes by self-training algorithm.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">6494</span>–<span class="cit-lpage">6506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki937</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLomsadze%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D20%26rft.spage%253D6494%26rft.epage%253D6506%26rft.atitle%253DGene%2Bidentification%2Bin%2Bnovel%2Beukaryotic%2Bgenomes%2Bby%2Bself-training%2Balgorithm%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki937%26rft_id%253Dinfo%253Apmid%252F16314312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16314312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233843600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.82" data-doi="10.1007/s001220051291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sefc</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eiras Dias</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinkellner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laimer Câmara Machado</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The use of microsatellites for germplasm management in a Portuguese grapevine collection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">733</span>–<span class="cit-lpage">739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220051291</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLopes%26rft.auinit1%253DM.%2BS.%26rft.volume%253D99%26rft.issue%253D3-4%26rft.spage%253D733%26rft.epage%253D739%26rft.atitle%253DThe%2Buse%2Bof%2Bmicrosatellites%2Bfor%2Bgermplasm%2Bmanagement%2Bin%2Ba%2BPortuguese%2Bgrapevine%2Bcollection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051291%26rft_id%253Dinfo%253Apmid%252F22665212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22665212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.83" data-doi="10.1016/j.phytochem.2004.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lücker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Vitis vinifera terpenoid cyclases: Functional identification of two sesquiterpene synthase cDNAs encoding (+)-valencene synthase and (-)-germacrene D synthase and expression of mono- and sesquiterpene synthases in grapevine flowers and berries.</span> <abbr class="cit-jnl-abbrev">Phytochemistry</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">2649</span>–<span class="cit-lpage">2659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.phytochem.2004.08.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochemistry%26rft.stitle%253DPhytochemistry%26rft.aulast%253DL%25C3%25BCcker%26rft.auinit1%253DJ.%26rft.volume%253D65%26rft.issue%253D19%26rft.spage%253D2649%26rft.epage%253D2659%26rft.atitle%253DVitis%2Bvinifera%2Bterpenoid%2Bcyclases%253A%2Bfunctional%2Bidentification%2Bof%2Btwo%2Bsesquiterpene%2Bsynthase%2BcDNAs%2Bencoding%2B%2528%252B%2529-valencene%2Bsynthase%2Band%2B%2528-%2529-germacrene%2BD%2Bsynthase%2Band%2Bexpression%2Bof%2Bmono-%2Band%2Bsesquiterpene%2Bsynthases%2Bin%2Bgrapevine%2Bflowers%2Band%2Bberries.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.phytochem.2004.08.017%26rft_id%253Dinfo%253Apmid%252F15464152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.phytochem.2004.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15464152&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224655400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.84" data-doi="10.1093/bioinformatics/bth315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majoros</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">TigrScan and GlimmerHMM: Two open source ab initio eukaryotic gene-finders.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2878</span>–<span class="cit-lpage">2879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth315%26rft_id%253Dinfo%253Apmid%252F15145805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15145805&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225250100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.85" data-doi="10.1101/gr.134106.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marquez</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalyna</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome survey reveals increased complexity of the alternative splicing landscape in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1184</span>–<span class="cit-lpage">1195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.134106.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134106.111%26rft_id%253Dinfo%253Apmid%252F22391557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTE4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.86" data-doi="10.1073/pnas.0901387106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toub</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohse</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The bouquet of grapevine (Vitis vinifera L. cv. Cabernet Sauvignon) flowers arises from the biosynthesis of sesquiterpene volatiles in pollen grains.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">7245</span>–<span class="cit-lpage">7250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0901387106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901387106%26rft_id%253Dinfo%253Apmid%252F19359488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE3LzcyNDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.87" data-doi="10.1038/nrg3068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Next-generation transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMartin%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D671%26rft.epage%253D682%26rft.atitle%253DNext-generation%2Btranscriptome%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3068%26rft_id%253Dinfo%253Apmid%252F21897427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21897427&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.88" data-doi="10.1104/pp.17.00311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massonnet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornielli</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altieri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Ripening Transcriptomic Program in Red and White Grapevine Varieties Correlates with Berry Skin Anthocyanin Accumulation.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">2376</span>–<span class="cit-lpage">2396</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.17.00311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.17.00311%26rft_id%253Dinfo%253Apmid%252F28652263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTc0LzQvMjM3NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.89" data-doi="10.3389/fpls.2017.01117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massonnet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa-Balderas</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galarneau</span>  <span class="cit-name-given-names">E. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Neofusicoccum parvum Colonization of the Grapevine Woody Stem Triggers Asynchronous Host Responses at the Site of Infection and in the Leaves.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1117%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> The first 50 plant genomes. Plant Genome 6: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2013.03.0001in</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.91" data-doi="10.3389/fpls.2017.00826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S. B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantu</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">How single molecule real-time sequencing and haplotype phasing have enabled reference-grade diploid genome assembly of wine grapes.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">826</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00826</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D826%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.92" data-doi="10.3389/fgene.2015.00361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izquierdo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emes</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assembly, assessment, and availability of De novo generated eukaryotic transcriptomes.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.93" data-doi="10.1073/pnas.1009363108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grassi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic structure and domestication history of the grape.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">3530</span>–<span class="cit-lpage">3535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1009363108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1009363108%26rft_id%253Dinfo%253Apmid%252F21245334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzkvMzUzMCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.94" data-doi="10.1093/nar/gku1063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rfam 12.0: updates to the RNA families database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D130</span>–<span class="cit-lpage">D137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1063%26rft_id%253Dinfo%253Apmid%252F25392425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25392425&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.95" data-doi="10.1093/bioinformatics/btp157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawrocki</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolbe</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Infernal 1.0: Inference of RNA alignments.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp157%26rft_id%253Dinfo%253Apmid%252F19307242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19307242&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265950600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.96" data-doi="10.1007/BF00023083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohmi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakana</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiraishi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandria</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Study of the parentage of grape cultivars by genetic interpretation of GPI-2 and PGM-2 isozymes.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00023083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D65%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00023083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00023083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.97" data-doi="10.1105/tpc.114.133710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palumbo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massonnet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farina</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrated network analysis identifies fight-club nodes as a class of hubs encompassing key putative switch genes that induce major transcriptome reprogramming during grapevine development.</span> <abbr class="cit-jnl-abbrev">Plant Cell Online</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4617</span>–<span class="cit-lpage">4635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.133710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BOnline%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.133710%26rft_id%253Dinfo%253Apmid%252F25490918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi80NjE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.98" data-doi="10.1101/gr.10.4.511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guigó</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">GeneID in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DParra%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DGeneID%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.511%26rft_id%253Dinfo%253Apmid%252F10779490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.99" data-doi="10.1186/1471-2229-13-30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pastore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pezzotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornielli</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Selective defoliation affects plant growth, fruit transcriptional ripening program and flavonoid metabolism in grapevine.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-13-30</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DPastore%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D30%26rft.atitle%253DSelective%2Bdefoliation%2Baffects%2Bplant%2Bgrowth%252C%2Bfruit%2Btranscriptional%2Bripening%2Bprogram%2Band%2Bflavonoid%2Bmetabolism%2Bin%2Bgrapevine.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-13-30%26rft_id%253Dinfo%253Apmid%252F23433030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-13-30&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23433030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.100" data-doi="10.1038/nbt.3122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonescu</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">T.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3122%26rft_id%253Dinfo%253Apmid%252F25690850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.101" data-doi="10.1371/journal.pone.0180904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibas</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The quantitative impact of read mapping to non-native reference genomes in comparative RNA-Seq studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0180904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0180904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0180904%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0180904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0180904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.102" data-doi="10.1093/nar/gkw294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Redundans: An assembly pipeline for highly heterozygous genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw294%26rft_id%253Dinfo%253Apmid%252F27131372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27131372&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.103" data-doi="10.1105/tpc.113.117523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">A. S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquez</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalyna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barta</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complexity of the alternative splicing landscape in plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3657</span>–<span class="cit-lpage">3683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.113.117523</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.117523%26rft_id%253Dinfo%253Apmid%252F24179125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8zNjU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.104" data-doi="10.1016/j.gene.2016.04.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricker</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goordial</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulthorpe</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PacBio SMRT assembly of a complex multi-replicon genome reveals chlorocatechol degradative operon in a region of genome plasticity.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">586</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2016.04.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D586%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2016.04.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2016.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.105" data-doi="10.1089/cmb.2014.0153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Safonova</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pevzner</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">dipSPAdes: Assembler for Highly Polymorphic Diploid Genomes. J. Comput. Biol. A J.</span> <abbr class="cit-jnl-abbrev">Comput. Mol. Cell Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2014.0153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BMol.%2BCell%2BBiol.%26rft.volume%253D22%26rft.spage%253D528%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2014.0153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2014.0153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.106" data-doi="10.1093/bioinformatics/btu538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivals</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LoRDEC: Accurate and efficient long read error correction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3506</span>–<span class="cit-lpage">3514</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu538%26rft_id%253Dinfo%253Apmid%252F25165095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25165095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.107" data-doi="10.1186/s12870-016-0760-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savoi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arapitsas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miculan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucchetti</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcriptome and metabolite profiling reveals that prolonged drought modulates the phenylpropanoid and terpenoid pathway in white grapes (Vitis vinifera L.).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-016-0760-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D16%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-016-0760-1%26rft_id%253Dinfo%253Apmid%252F27001212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-016-0760-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27001212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.108" data-doi="10.3389/fpls.2017.01124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savoi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrera</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucchetti</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multi-Omics and Integrated Network Analyses Reveal New Insights into the Systems Relationships between Metabolites, Structural Genes, and Transcriptional Regulators in Developing Grape Berries (Vitis vinifera L.) Exposed to Water Deficit.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01124</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1124%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.109" data-doi="10.1007/s001220050889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sefc</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinkellner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glössl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kampfer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regner</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Reconstruction of a grapevine pedigree by microsatellite analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050889</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075377300030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.110" data-doi="10.1038/nature20098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">De novo assembly and phasing of a Korean human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature20098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature20098%26rft_id%253Dinfo%253Apmid%252F27706134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature20098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27706134&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.111" data-doi="10.3389/fpls.2017.01486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espinoza</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inostroza-Blancheteau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poblete</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Omics Approaches for Understanding Grapevine Berry Development: Regulatory Networks Associated with Endogenous Processes and Environmental Responses.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1486%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.112" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-113-1" title="View reference  in text" id="ref-113">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.113" data-doi="10.1146/annurev.arplant.043008.092032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nandety</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Short-read sequencing technologies for transcriptional analyses.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.arplant.043008.092032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DSimon%26rft.auinit1%253DS.%2BA.%26rft.volume%253D60%26rft.spage%253D305%26rft.epage%253D333%26rft.atitle%253DShort-read%2Bsequencing%2Btechnologies%2Bfor%2Btranscriptional%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.043008.092032%26rft_id%253Dinfo%253Apmid%252F19575585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.043008.092032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19575585&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268071800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-114-1" title="View reference  in text" id="ref-114">↵</a><div class="cit ref-cit ref-web" id="cit-9.3.755.114"><div class="cit-metadata unstructured">Smit, A. F. A., and R. Hubley, 2008 RepeatModeler Open-1.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-115-1" title="View reference  in text" id="ref-115">↵</a><div class="cit ref-cit ref-web" id="cit-9.3.755.115"><div class="cit-metadata unstructured">Smit, A. F. A., R. Hubley, and P. Green, 2013 RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a> 2013–2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-116-1" title="View reference  in text" id="ref-116">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.116" data-doi="10.1101/gr.196469.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith-Unna</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boursnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hibberd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">TransRate: reference-free quality assessment of de novo transcriptome assemblies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1134</span>–<span class="cit-lpage">1144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.196469.115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.196469.115%26rft_id%253Dinfo%253Apmid%252F27252236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMTEzNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-117-1" title="View reference  in text" id="ref-117">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.117" data-doi="10.1007/BF00029856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sparvoli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scienza</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gavazzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonelli</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Cloning and molecular analysis of structural genes involved in flavonoid and stilbene biosynthesis in grape (Vitis vinifera L.).</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00029856</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DSparvoli%26rft.auinit1%253DF.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D743%26rft.epage%253D755%26rft.atitle%253DCloning%2Band%2Bmolecular%2Banalysis%2Bof%2Bstructural%2Bgenes%2Binvolved%2Bin%2Bflavonoid%2Band%2Bstilbene%2Bbiosynthesis%2Bin%2Bgrape%2B%2528Vitis%2Bvinifera%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00029856%26rft_id%253Dinfo%253Apmid%252F8193299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00029856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8193299&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NL59500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-118-1" title="View reference  in text" id="ref-118">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.118" data-doi="10.1093/nar/gkl200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunduz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waack</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">AUGUSTUS: ab initio prediction of alternative transcripts.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W435</span>–<span class="cit-lpage">W439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW435%26rft.epage%253DW439%26rft.atitle%253DAUGUSTUS%253A%2Bab%2Binitio%2Bprediction%2Bof%2Balternative%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl200%26rft_id%253Dinfo%253Apmid%252F16845043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845043&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200087&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-119-1" title="View reference  in text" id="ref-119">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.119" data-doi="10.1093/nar/gkp759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinbiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gremme</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fine-grained annotation and classification of de novo predicted LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">7002</span>–<span class="cit-lpage">7013</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkp759</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp759%26rft_id%253Dinfo%253Apmid%252F19786494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19786494&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272688400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-120-1" title="View reference  in text" id="ref-120">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strefeler</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeden</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisch</span>  <span class="cit-name-given-names">B. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Inheritance of chloroplast DNA in two full-sib Vitis populations.</span> <abbr class="cit-jnl-abbrev">Vitis</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVitis%26rft.volume%253D31%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1992KL57900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-121-1" title="View reference  in text" id="ref-121">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.121" data-doi="10.1016/j.devcel.2016.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarup</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crespi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">One gene, many proteins: mapping cell-specific alternative splicing in plants.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.devcel.2016.11.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D39%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2016.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2016.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-122-1" title="View reference  in text" id="ref-122">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tapia</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabezas</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabello</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacombe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Zapater</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Determining the Spanish origin of representative ancient American grapevine varieties.</span> <abbr class="cit-jnl-abbrev">Am. J. Enol. Vitic.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BEnology%2Band%2BViticulture%26rft.stitle%253DAm.%2BJ.%2BEnol.%2BVitic.%26rft.aulast%253DTapia%26rft.auinit1%253DA.%2BM.%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D242%26rft.epage%253D251%26rft.atitle%253DDetermining%2Bthe%2BSpanish%2BOrigin%2Bof%2BRepresentative%2BAncient%2BAmerican%2BGrapevine%2BVarieties%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWpldiI7czo1OiJyZXNpZCI7czo4OiI1OC8yLzI0MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-123-1" title="View reference  in text" id="ref-123">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.123" data-doi="10.1104/pp.15.01267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thatcher</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danilevskaya</span>  <span class="cit-name-given-names">O. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beatty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zastrow-Hayes</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide analysis of alternative splicing during development and drought stress in maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">586</span>–<span class="cit-lpage">599</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.01267</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.01267%26rft_id%253Dinfo%253Apmid%252F26582726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzAvMS81ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-124-1" title="View reference  in text" id="ref-124">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.124" data-doi="10.1371/journal.pone.0162868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tombácz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csabai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oláh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balázs</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Likó</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Full-length isoform sequencing reveals novel transcripts and substantial transcriptional overlaps in a herpesvirus.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0162868</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0162868</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0162868%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0162868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0162868&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-125-1" title="View reference  in text" id="ref-125">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.125" data-doi="10.1371/journal.pone.0185020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pech</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCairns</span>  <span class="cit-name-given-names">R. J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mévy</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Challenges and advances for transcriptome assembly in non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0185020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0185020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0185020%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0185020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0185020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-126-1" title="View reference  in text" id="ref-126">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.126" data-doi="10.1371/journal.pone.0001326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zharkikh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high quality draft consensus sequence of the genome of a heterozygous grapevine variety.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e1326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0001326</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVelasco%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De1326%26rft.epage%253De1326%26rft.atitle%253DA%2Bhigh%2Bquality%2Bdraft%2Bconsensus%2Bsequence%2Bof%2Bthe%2Bgenome%2Bof%2Ba%2Bheterozygous%2Bgrapevine%2Bvariety.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001326%26rft_id%253Dinfo%253Apmid%252F18094749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18094749&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-127-1" title="View reference  in text" id="ref-127">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.127" data-doi="10.1186/1471-2164-14-41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venturini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrarini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornielli</span>  <span class="cit-name-given-names">G. B. G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcriptome characterization of Vitis vinifera cv. Corvina unveils varietal diversity.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-41</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-41%26rft_id%253Dinfo%253Apmid%252F23331995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-41&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23331995&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-128-1" title="View reference  in text" id="ref-128">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.128" data-doi="10.1371/journal.pgen.1006500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vij</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuznetsova</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komissarov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurchenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Chromosomal-level assembly of the asian seabass genome using long sequence reads and multi-layered scaffolding.</span> <abbr class="cit-jnl-abbrev">PLOS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005954%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-129-1" title="View reference  in text" id="ref-129">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.129" data-doi="10.1186/1471-2229-14-99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitulo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forcato</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpinelli</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telatin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campagna</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A deep survey of alternative splicing in grape reveals changes in the splicing machinery related to tissue, stress condition and genotype.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">99</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-14-99</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-99%26rft_id%253Dinfo%253Apmid%252F24739459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24739459&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-130-1" title="View reference  in text" id="ref-130">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.130" data-doi="10.1038/ncomms11708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regulski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms11708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11708%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-131-1" title="View reference  in text" id="ref-131">↵</a><div class="cit ref-cit ref-web" id="cit-9.3.755.131"><div class="cit-metadata unstructured">Warnes, G. R., B. Bolker, L. Bonebakker, R. Gentleman, W. Huber <em>et al.</em>, 2016 Package “gplots”: Various R programming tools for plotting data. R Packag. version 2.17.0 <a href="https://CRAN.R-project.org/package=gplots.">https://CRAN.R-project.org/package=gplots.</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-132-1" title="View reference  in text" id="ref-132">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.132" data-doi="10.1093/nar/gkv562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirather</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Afshar</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterization of fusion genes and the significantly expressed fusion isoforms in breast cancer by hybrid sequencing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">e116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv562</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv562%26rft_id%253Dinfo%253Apmid%252F26040699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26040699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-133-1" title="View reference  in text" id="ref-133">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myrka</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single molecule, full-length transcript sequencing provides insight into the extreme metabolism of ruby-throated hummingbird Archilochus colubris.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">117218</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-134-1" title="View reference  in text" id="ref-134">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.134" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15728110&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-135-1" title="View reference  in text" id="ref-135">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.135" data-doi="10.1371/journal.pone.0113772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptomic Analysis of Grape (Vitis vinifera L.) Leaves after Exposure to Ultraviolet C Irradiation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e113772</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0113772</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De113772%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0113772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0113772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-136-1" title="View reference  in text" id="ref-136">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.136" data-doi="10.1007/s11295-013-0643-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effects of artificial selection on sugar metabolism and transporter genes in grape.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1343</span>–<span class="cit-lpage">1349</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-013-0643-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D1343%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-013-0643-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-013-0643-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000324514800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-137-1" title="View reference  in text" id="ref-137">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.137" data-doi="10.1016/j.molcel.2012.08.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Stress-induced alternative splicing provides a mechanism for the regulation of microRNA processing in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2012.08.032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DYan%26rft.auinit1%253DK.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D521%26rft.epage%253D531%26rft.atitle%253DStress-Induced%2BAlternative%2BSplicing%2BProvides%2Ba%2BMechanism%2Bfor%2Bthe%2BRegulation%2Bof%2BMicroRNA%2BProcessing%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.08.032%26rft_id%253Dinfo%253Apmid%252F23063528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.08.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23063528&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311919500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-138-1" title="View reference  in text" id="ref-138">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.138" data-doi="10.5808/GI.2015.13.4.119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis of whole transcriptome sequencing data: Workflow and software.</span> <abbr class="cit-jnl-abbrev">Genomics Inform.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5808/GI.2015.13.4.119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BInform.%26rft.volume%253D13%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.5808%252FGI.2015.13.4.119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5808/GI.2015.13.4.119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-139-1" title="View reference  in text" id="ref-139">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.139" data-doi="10.1093/nar/gkl031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">WEGO: a web tool for plotting GO annotations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W293</span>–<span class="cit-lpage">W297</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DYe%26rft.auinit1%253DJ.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW293%26rft.epage%253DW297%26rft.atitle%253DWEGO%253A%2Ba%2Bweb%2Btool%2Bfor%2Bplotting%2BGO%2Bannotations.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl031%26rft_id%253Dinfo%253Apmid%252F16845012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845012&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-140-1" title="View reference  in text" id="ref-140">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.140" data-doi="10.3389/fpls.2017.00630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zenoni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dal Santo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornielli</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Incà</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filippetti</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transcriptional responses to pre-flowering leaf defoliation in grapevine berry from different growing sites, years, and genotypes.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.00630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D630%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.00630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.00630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-141-1" title="View reference  in text" id="ref-141">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.141" data-doi="10.4137/BBI.S33716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary character of alternative splicing in plants.</span> <abbr class="cit-jnl-abbrev">Bioinform. Biol. Insights</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4137/BBI.S33716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinform.%2BBiol.%2BInsights%26rft.volume%253D9%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.4137%252FBBI.S33716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4137/BBI.S33716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-142-1" title="View reference  in text" id="ref-142">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.142" data-doi="10.1073/pnas.1709257114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massonnet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanjak</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolutionary genomics of grape (Vitis vinifera ssp. vinifera) domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">11715</span>–<span class="cit-lpage">11720</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1709257114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1709257114%26rft_id%253Dinfo%253Apmid%252F29042518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQ0LzExNzE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-143-1" title="View reference  in text" id="ref-143">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.143" data-doi="10.1101/gr.213405.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Hybrid assembly of the large and highly repetitive genome of Aegilops tauschii, a progenitor of bread wheat, with the MaSuRCA mega-reads algorithm.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.213405.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.213405.116%26rft_id%253Dinfo%253Apmid%252F28130360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-144-1" title="View reference  in text" id="ref-144">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.755.144" data-doi="10.1016/j.gdata.2017.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zulkapli</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosli</span>  <span class="cit-name-given-names">M. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salleh</span>  <span class="cit-name-given-names">F. I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohd Noor</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aizat</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Iso-Seq analysis of Nepenthes ampullaria, Nepenthes rafflesiana and Nepenthes × hookeriana for hybridisation study in pitcher plants.</span> <abbr class="cit-jnl-abbrev">Genom. Data</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gdata.2017.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenom.%2BData%26rft.volume%253D12%26rft.spage%253D130%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gdata.2017.05.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gdata.2017.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/755.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/749" title="Shep RNA-Binding Capacity Is Required for Antagonism of gypsy Chromatin Insulator Activity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/769" title="The Role of UV-B light on Small RNA Activity During Grapevine Berry Development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/755" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WnD33-IIgcQD0RyODimfrQM8gVSm6WOwlZYRLR8zFGY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/755" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/755" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/755" class="active" data-icon-position="" data-hide-link-title="0">Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492489" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zet9OUt6-zNmqB-wdefF4SOxJSW_lyFa2PDmJCnrGDo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/755.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/755" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WYu9YEHsb5dLp269yZnEFWUcgbUCoRkjgf940DCvX3M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/755" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/755.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492489" id="node492489--2" data-pisa="ggg;9/3/755" data-pisa-master="ggg;g3.118.201008" data-apath="/ggg/9/3/755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea Minio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mélanie Massonnet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosa Figueroa-Balderas</span>, <span class="highwire-citation-author" data-delta="3">Amanda M. Vondras</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Blanco-Ulate</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario Cantu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">755-767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.201008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492489--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Minio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BMinio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BMinio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea Minio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mélanie Massonnet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BMassonnet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Massonnet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM%25C3%25A9lanie%2BMassonnet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mélanie Massonnet</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rosa Figueroa-Balderas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosa%2BFigueroa-Balderas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figueroa-Balderas%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosa%2BFigueroa-Balderas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosa Figueroa-Balderas</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Amanda M. Vondras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BVondras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vondras%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BVondras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Barbara Blanco-Ulate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BBlanco-Ulate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco-Ulate%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BBlanco-Ulate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Blanco-Ulate</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario Cantu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dacantu@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dacantu@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/755" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/755&amp;text=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/755&amp;t=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/755&amp;title=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492489/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492489" id="node492489--3" data-pisa="ggg;9/3/755" data-pisa-master="ggg;g3.118.201008" data-apath="/ggg/9/3/755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Iso-Seq Allows Genome-Independent Transcriptome Profiling of Grape Berry Development</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrea Minio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mélanie Massonnet</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rosa Figueroa-Balderas</span>, <span class="highwire-citation-author" data-delta="3">Amanda M. Vondras</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Blanco-Ulate</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dario Cantu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">755-767;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.201008 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492489--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Minio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BMinio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Minio%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BMinio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2643-9209" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea Minio</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mélanie Massonnet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BMassonnet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Massonnet%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM%25C3%25A9lanie%2BMassonnet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0993-9148" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mélanie Massonnet</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rosa Figueroa-Balderas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rosa%2BFigueroa-Balderas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Figueroa-Balderas%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARosa%2BFigueroa-Balderas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3321-1376" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rosa Figueroa-Balderas</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Amanda M. Vondras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BM.%2BVondras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vondras%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmanda%2BM.%2BVondras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Barbara Blanco-Ulate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Sciences, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BBlanco-Ulate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanco-Ulate%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BBlanco-Ulate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8819-9207" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Blanco-Ulate</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Viticulture and Enology, University of California Davis, Davis, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4858-1508" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dario Cantu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dacantu@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">dacantu@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=755&amp;rft.epage=767&amp;rft.atitle=Iso-Seq%20Allows%20Genome-Independent%20Transcriptome%20Profiling%20of%20Grape%20Berry%20Development&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Minio%2C+Andrea&amp;rft.au=Massonnet%2C+M%C3%A9lanie&amp;rft.au=Figueroa-Balderas%2C+Rosa&amp;rft.au=Vondras%2C+Amanda+M.&amp;rft.au=Blanco-Ulate%2C+Barbara&amp;rft.au=Cantu%2C+Dario"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492489/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492489/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492489/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492489/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492489/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492489/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492489/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492489/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492489/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492489/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492489/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492489/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/755" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/755.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/755.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/755.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/755.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
