<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/955" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/955.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/955.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/955.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/955.atom" />
<meta name="HW.pisa" content="ggg;9/3/955" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis" />
<meta name="DC.Identifier" content="10.1534/g3.118.200910" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Sutherland et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7586681" />
<meta name="DC.Description" content="Networks of co-expressed genes produce complex phenotypes associated with functional novelty. Sex differences in gene expression levels or in the structure of gene co-expression networks can cause sexual dimorphism and may resolve sexually antagonistic selection. Here we used RNA-sequencing in the salmonid Brook Charr Salvelinus fontinalis to characterize sex-specific co-expression networks in the liver of 47 female and 53 male offspring. In both networks, modules were characterized for functional enrichment, hub gene identification, and associations with 15 growth, reproduction, and stress-related phenotypes. Modules were then evaluated for preservation in the opposite sex, and in the congener Arctic Charr Salvelinus alpinus . Overall, more transcripts were assigned to a module in the female network than in the male network, which coincided with higher inter-individual gene expression and phenotype variation in the females. Most modules were preserved between sexes and species, including those involved in conserved cellular processes ( e.g. , translation, immune pathways). However, two sex-specific male modules were identified, and these may contribute to sexual dimorphism. To compare with the network analysis, differentially expressed transcripts were identified between the sexes, revealing a total of 16% of expressed transcripts as sex-biased. For both sexes, there was no overrepresentation of sex-biased genes or sex-specific modules on the putative sex chromosome. Sex-biased transcripts were also not overrepresented in sex-specific modules, and in fact highly male-biased transcripts were enriched in preserved modules. Comparative network analysis and differential expression analyses identified different aspects of sex differences in gene expression, and both provided new insights on the genes underlying sexual dimorphism in the salmonid Brook Charr." />
<meta name="DC.Contributor" content="Ben J. G. Sutherland" />
<meta name="DC.Contributor" content="Jenni M. Prokkola" />
<meta name="DC.Contributor" content="Céline Audet" />
<meta name="DC.Contributor" content="Louis Bernatchez" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Genetics of Sex" />
<meta name="citation_title" content="Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Networks of co-expressed genes produce complex phenotypes associated with functional novelty. Sex differences in gene expression levels or in the structure of gene co-expression networks can cause sexual dimorphism and may resolve sexually antagonistic selection. Here we used RNA-sequencing in the salmonid Brook Charr &lt;i&gt;Salvelinus fontinalis&lt;/i&gt; to characterize sex-specific co-expression networks in the liver of 47 female and 53 male offspring. In both networks, modules were characterized for functional enrichment, hub gene identification, and associations with 15 growth, reproduction, and stress-related phenotypes. Modules were then evaluated for preservation in the opposite sex, and in the congener Arctic Charr &lt;i&gt;Salvelinus alpinus&lt;/i&gt;. Overall, more transcripts were assigned to a module in the female network than in the male network, which coincided with higher inter-individual gene expression and phenotype variation in the females. Most modules were preserved between sexes and species, including those involved in conserved cellular processes (&lt;i&gt;e.g.&lt;/i&gt;, translation, immune pathways). However, two sex-specific male modules were identified, and these may contribute to sexual dimorphism. To compare with the network analysis, differentially expressed transcripts were identified between the sexes, revealing a total of 16% of expressed transcripts as sex-biased. For both sexes, there was no overrepresentation of sex-biased genes or sex-specific modules on the putative sex chromosome. Sex-biased transcripts were also not overrepresented in sex-specific modules, and in fact highly male-biased transcripts were enriched in preserved modules. Comparative network analysis and differential expression analyses identified different aspects of sex differences in gene expression, and both provided new insights on the genes underlying sexual dimorphism in the salmonid Brook Charr.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/955" />
<meta name="citation_id" content="9/3/955" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/955" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/955.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/955.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/955.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200910" />
<meta name="citation_pmid" content="30692150" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genetics of Sex" />
<meta name="citation_firstpage" content="955" />
<meta name="citation_lastpage" content="968" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ben J. G. Sutherland" />
<meta name="citation_author_institution" content="Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada" />
<meta name="citation_author_email" content="ben.sutherland.1@ulaval.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2029-9893" />
<meta name="citation_author" content="Jenni M. Prokkola" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, University of Liverpool, L69 7ZB Liverpool, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2987-4417" />
<meta name="citation_author" content="Céline Audet" />
<meta name="citation_author_institution" content="Institut des Sciences de la Mer de Rimouski, Université du Québec à Rimouski, Rimouski, QC G5L 3A1, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3366-210X" />
<meta name="citation_author" content="Louis Bernatchez" />
<meta name="citation_author_institution" content="Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8085-9709" />
<meta name="citation_reference" content="citation_title=Evolutionary genetics of fishes;citation_author=F. W. Allendorf;citation_author=G. H. Thorgaard;citation_title=Tetraploidy and the evolution of salmonid fishes;citation_year=1984;citation_doi=10.1007/978-1-4684-4652-4_1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=N. J. Barson;citation_author=T. Aykanat;citation_author=K. Hindar;citation_author=M. Baranski;citation_author=G. H. Bolstad;citation_title=Sex-dependent dominance at a single locus maintains variation in age at maturity in salmon.;citation_pages=405-408;citation_volume=528;citation_year=2015;citation_pmid=26536110;citation_doi=10.1038/nature16062" />
<meta name="citation_reference" content="citation_author=H. Blackmon;citation_author=Y. Brandvain;citation_year=2017;citation_pmid=29021279;citation_doi=10.1534/genetics.117.300382" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. J. Blanchfield;citation_author=M. S. Ridgway;citation_author=C. C. Wilson;citation_title=Breeding success of male brook trout (Salvelinus fontinalis) in the wild.;citation_pages=2417-2428;citation_volume=12;citation_year=2003;citation_issue=9;citation_pmid=12919479;citation_doi=10.1046/j.1365-294X.2003.01917.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Environ. Biol. Fishes;citation_author=D. Boula;citation_author=V. Castric;citation_author=L. Bernatchez;citation_author=C. Audet;citation_title=Physiological, endocrine, and genetic bases of anadromy in the Brook Charr, Salvelinus fontinalis, of the Laval River (Québec, Canada).;citation_pages=229-242;citation_volume=64;citation_year=2002;citation_doi=10.1023/A:1016054119783" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Böhne;citation_author=T. Sengstag;citation_author=W. Salzburger;citation_title=Comparative transcriptomics in East African Cichlids reveals sex- and species-specific expression and new candidates for sex differentiation in fishes.;citation_volume=6;citation_year=2014;citation_pmid=25364805;citation_doi=10.1093/gbe/evu200" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=D. M. Bryant;citation_author=K. Johnson;citation_author=T. DiTommaso;citation_author=T. Tickle;citation_author=M. B. Couger;citation_title=A tissue-mapped Axolotl de novo transcriptome enables identification of limb regeneration factors.;citation_pages=762-776;citation_volume=18;citation_year=2017;citation_doi=10.1016/j.celrep.2016.12.063" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=W. S. Bush;citation_author=J. H. Moore;citation_title=Chapter 11: Genome-Wide Association Studies.;citation_pages=e1002822;citation_volume=8;citation_year=2012;citation_pmid=23300413;citation_doi=10.1371/journal.pcbi.1002822" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Carruthers;citation_author=A. A. Yurchenko;citation_author=J. J. Augley;citation_author=C. E. Adams;citation_author=P. Herzyk;citation_title=De novo transcriptome assembly, annotation and comparison of four ecological and evolutionary model salmonid fish species.;citation_pages=32;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-017-4379-x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Sexual dimorphism in gene expression and regulatory networks across human tissues.;citation_author=C.-Y. Chen;citation_author=C. M. Lopes-Ramos;citation_author=M. L. Kuijjer;citation_author=J. N. Paulson;citation_author=A. R. Sonawane;citation_pages=1-34;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Integr. Comp. Biol.;citation_author=Z. A. Cheviron;citation_author=D. L. Swanson;citation_title=Comparative transcriptomics of seasonal phenotypic flexibility in two North American songbirds.;citation_pages=1040-1054;citation_volume=57;citation_year=2017;citation_doi=10.1093/icb/icx118" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. A. Christensen;citation_author=E. B. Rondeau;citation_author=D. R. Minkley;citation_author=J. S. Leong;citation_author=C. M. Nugent;citation_title=The Arctic charr (Salvelinus alpinus) genome and transcriptome assembly.;citation_pages=e0204076;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0204076" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Crête-Lafrenière;citation_author=L. K. Weir;citation_author=L. Bernatchez;citation_title=Framing the salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.;citation_pages=e46662-e46662;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071608;citation_doi=10.1371/journal.pone.0046662" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=R. Dean;citation_author=J. E. Mank;citation_title=The role of sex chromosomes in sexual dimorphism: discordance between molecular and phenotypic data.;citation_volume=27;citation_year=2014;citation_doi=10.1111/jeb.12345" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. B. Eisen;citation_author=P. T. Spellman;citation_author=P. O. Brown;citation_author=D. Botstein;citation_title=Cluster analysis and display of genome-wide expression patterns;citation_pages=14863-14868;citation_volume=95;citation_year=1998;citation_issue=25;citation_pmid=9843981;citation_doi=10.1073/pnas.95.25.14863" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Ellegren;citation_author=J. Parsch;citation_title=The evolution of sex-biased genes and sex-biased gene expression.;citation_pages=689-698;citation_volume=8;citation_year=2007;citation_issue=9;citation_pmid=17680007;citation_doi=10.1038/nrg2167" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Filteau;citation_author=S. A. Pavey;citation_author=J. St-Cyr;citation_author=L. Bernatchez;citation_title=Gene coexpression networks reveal key drivers of phenotypic divergence in Lake Whitefish.;citation_volume=30;citation_year=2013;citation_pmid=23519315;citation_doi=10.1093/molbev/mst053" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Fish. Aquat. Sci.;citation_author=I. A. Fleming;citation_title=Pattern and variability in the breeding system of Atlantic salmon (Salmo salar), with comparisons to other salmonids.;citation_pages=59-76;citation_volume=55;citation_year=1998;citation_doi=10.1139/d98-009" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=C. Gaiteri;citation_author=Y. Ding;citation_author=B. French;citation_author=G. C. Tseng;citation_author=E. Sibille;citation_title=Beyond modules and hubs: the potential of gene coexpression networks for investigating molecular mechanisms of complex brain disorders.;citation_pages=13-24;citation_volume=13;citation_year=2014;citation_pmid=24320616;citation_doi=10.1111/gbb.12106" />
<meta name="citation_reference" content="citation_journal_title=Gen. Comp. Endocrinol.;citation_author=N. Garcia-Reyero;citation_author=B. S. Jayasinghe;citation_author=K. J. Kroll;citation_author=T. Sabo-Attwood;citation_author=N. D. Denslow;citation_title=Estrogen signaling through both membrane and nuclear receptors in the liver of Fathead Minnow.;citation_pages=50-66;citation_volume=257;citation_year=2018;citation_doi=10.1016/j.ygcen.2017.07.019" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=J. Gillis;citation_author=P. Pavlidis;citation_title=&quot;Guilt by association&quot; is the exception rather than the rule in gene networks.;citation_pages=e1002444-e1002444;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22479173;citation_doi=10.1371/journal.pcbi.1002444" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=J. Guðbrandsson;citation_author=S. R. Franzdóttir;citation_author=B. K. Kristjánsson;citation_author=E. P. Ahi;citation_author=V. H. Maier;citation_title=Differential gene expression during early development in recently evolved and sympatric Arctic charr morphs.;citation_pages=e4345;citation_volume=6;citation_year=2018;citation_doi=10.7717/peerj.4345" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=J. L. Horreo;citation_title=Revisiting the mitogenomic phylogeny of Salmoninae: new insights thanks to recent sequencing advances.;citation_pages=e3828;citation_volume=5;citation_year=2017;citation_doi=10.7717/peerj.3828" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=D. W. Huang;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_author=M. Kodama;citation_author=M. S. O. Brieuc;citation_author=R. H. Devlin;citation_author=J. J. Hard;citation_author=K. A. Naish;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=Intake_epis_food(): An R Function for Fitting a Bivariate Nonlinear Measurement Error Model to Estimate Usual and Energy Intake for Episodically Consumed Foods.;citation_pages=1-17;citation_volume=46;citation_year=2012;citation_issue=c03;citation_pmid=22837731;citation_doi=10.18637/jss.v046.i11" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Langfelder;citation_author=S. Horvath;citation_title=WGCNA: an R package for weighted correlation network analysis.;citation_pages=559-559;citation_volume=9;citation_year=2008;citation_pmid=19114008;citation_doi=10.1186/1471-2105-9-559" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. Langfelder;citation_author=R. Luo;citation_author=M. C. Oldham;citation_author=S. Horvath;citation_title=Is my network module preserved and reproducible?;citation_pages=e1001057-e1001057;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21283776;citation_doi=10.1371/journal.pcbi.1001057" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Lien;citation_author=B. F. Koop;citation_author=S. R. Sandve;citation_author=J. R. Miller;citation_author=M. P. Kent;citation_title=The Atlantic Salmon genome provides insights into rediploidization.;citation_volume=533;citation_year=2016;citation_doi=10.1038/nature17164" />
<meta name="citation_reference" content="citation_journal_title=Oikos;citation_author=R. L. Lochmiller;citation_author=C. Deerenberg;citation_title=Trade-offs in evolutionary immunology: just what is the cost of immunity?;citation_pages=87-98;citation_volume=88;citation_year=2000;citation_doi=10.1034/j.1600-0706.2000.880110.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=T. F. Mackay;citation_title=THE GENETIC ARCHITECTURE OF QUANTITATIVE TRAITS;citation_pages=303-339;citation_volume=35;citation_year=2001;citation_issue=1;citation_pmid=11700286;citation_doi=10.1146/annurev.genet.35.102401.090633" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. F. C. Mackay;citation_author=E. A. Stone;citation_author=J. F. Ayroles;citation_title=The genetics of quantitative traits: challenges and prospects.;citation_pages=565-577;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19584810;citation_doi=10.1038/nrg2612" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=M. D. MacManes;citation_title=On the optimal trimming of high-throughput mRNA sequence data.;citation_pages=13;citation_volume=5;citation_year=2014;citation_pmid=24567737;citation_doi=10.3389/fgene.2014.00013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=D. J. Macqueen;citation_author=I. A. Johnston;citation_title=A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.;citation_volume=281;citation_year=2014;citation_pmid=24452024;citation_doi=10.1098/rspb.2013.2881" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Mähler;citation_author=J. Wang;citation_author=B. K. Terebieniec;citation_author=P. K. Ingvarsson;citation_author=N. R. Street;citation_title=Gene co-expression network connectivity is an important determinant of selective constraint.;citation_pages=e1006402;citation_volume=13;citation_year=2017;citation_pmid=28406900;citation_doi=10.1371/journal.pgen.1006402" />
<meta name="citation_reference" content="citation_title=Report on the performance evaluation of the Fluidigm BioMark platform for high-throughput microbe monitoring in salmon;citation_author=K. M. Miller;citation_author=I. A. Gardner;citation_author=R. Vanderstichel;citation_author=T. Burnley;citation_author=S. Li;citation_pages=1-293;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=NPJ Syst. Biol. Appl.;citation_author=A. J. Mueller;citation_author=E. G. Canty-Laird;citation_author=P. D. Clegg;citation_author=S. R. Tew;citation_title=Cross-species gene modules emerge from a systems biology approach to osteoarthritis.;citation_pages=13;citation_volume=3;citation_year=2017;citation_doi=10.1038/s41540-017-0014-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Sex Dev;citation_author=B. Nicol;citation_author=Y. Guiguen;citation_title=Expression profiling of Wnt signaling genes during gonadal differentiation and gametogenesis in rainbow trout.;citation_pages=318-329;citation_volume=5;citation_year=2011;citation_issue=6;citation_pmid=22116193;citation_doi=10.1159/000334515" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. C. Oldham;citation_author=S. Horvath;citation_author=D. H. Geschwind;citation_title=Conservation and evolution of gene coexpression networks in human and chimpanzee brains.;citation_volume=103;citation_year=2006;citation_pmid=17101986;citation_doi=10.1073/pnas.0605938103" />
<meta name="citation_reference" content="citation_journal_title=Nature Publishing Group;citation_author=J. Parsch;citation_author=H. Ellegren;citation_title=The evolutionary causes and consequences of sex-biased gene expression.;citation_pages=83-87;citation_volume=14;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Pasquier;citation_author=C. Cabau;citation_author=T. Nguyen;citation_author=E. Jouanno;citation_author=D. Severac;citation_title=Gene evolution and gene expression after whole genome duplication in fish: the PhyloFish database.;citation_pages=368;citation_volume=17;citation_year=2016;citation_pmid=27189481;citation_doi=10.1186/s12864-016-2709-z" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. D. Poley;citation_author=B. J. G. Sutherland;citation_author=S. R. M. Jones;citation_author=B. F. Koop;citation_author=M. D. Fast;citation_title=Sex-biased gene expression and sequence conservation in Atlantic and Pacific salmon lice (Lepeophtheirus salmonis).;citation_pages=483;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2835-7" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=J. M. Prokkola;citation_author=M. Nikinmaa;citation_author=M. Lewis;citation_author=K. Anttila;citation_author=M. Kanerva;citation_title=Cold temperature represses daily rhythms in the liver transcriptome of a stenothermal teleost under decreasing day length.;citation_volume=221;citation_year=2018;citation_pmid=29361589;citation_doi=10.1242/jeb.170670" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Q. Qiao;citation_author=S. Le Manach;citation_author=B. Sotton;citation_author=H. Huet;citation_author=E. Duvernois-Berthet;citation_title=Deep sexual dimorphism in adult medaka fish liver highlighted by multi-omic approach.;citation_pages=32459;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep32459" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Anim. Behav.;citation_author=T. P. Quinn;citation_author=C. J. Foote;citation_title=The effects of body size and sexual dimorphism on the reproductive behavior of sockeye salmon, Oncorhynchus nerka.;citation_pages=751-761;citation_volume=48;citation_year=1994;citation_doi=10.1006/anbe.1994.1300" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=A. Roberts;citation_author=L. Pachter;citation_title=Streaming fragment assignment for real-time analysis of sequencing experiments.;citation_pages=71-73;citation_volume=10;citation_year=2013;citation_pmid=23160280;citation_doi=10.1038/nmeth.2251" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=F. M. Robertson;citation_author=M. K. Gundappa;citation_author=F. Grammes;citation_author=T. R. Hvidsten;citation_author=A. K. Redmond;citation_title=Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.;citation_pages=111;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-017-1241-z" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. D. Robinson;citation_author=A. Oshlack;citation_title=A scaling normalization method for differential expression analysis of RNA-seq data.;citation_pages=R25-R25;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20196867;citation_doi=10.1186/gb-2010-11-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=N. H. Rose;citation_author=F. O. Seneca;citation_author=S. R. Palumbi;citation_title=Gene networks in the wild: identifying transcriptional modules that mediate coral resistance to experimental heat stress.;citation_volume=8;citation_year=2015;citation_pmid=26710855;citation_doi=10.1093/gbe/evv258" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=L. Rowe;citation_author=S. F. Chenoweth;citation_author=A. F. Agrawal;citation_title=The Genomics of Sexual Conflict.;citation_pages=274-286;citation_volume=192;citation_year=2018;citation_doi=10.1086/698198" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=C. Sauvage;citation_author=M. Vagner;citation_author=N. Derôme;citation_author=C. Audet;citation_author=L. Bernatchez;citation_title=Coding Gene Single Nucleotide Polymorphism Mapping and Quantitative Trait Loci Detection for Physiological Reproductive Traits in Brook Charr, Salvelinus fontinalis;citation_pages=379-392;citation_volume=2;citation_year=2012;citation_issue=3;citation_pmid=22413092;citation_doi=10.1534/g3.111.001867" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=C. Sauvage;citation_author=M. Vagner;citation_author=N. Derôme;citation_author=C. Audet;citation_author=L. Bernatchez;citation_title=Coding Gene SNP Mapping Reveals QTL Linked to Growth and Stress Response in Brook Charr (Salvelinus fontinalis);citation_pages=707-720;citation_volume=2;citation_year=2012;citation_issue=6;citation_pmid=22690380;citation_doi=10.1534/g3.112.001990" />
<meta name="citation_reference" content="citation_journal_title=Biol. Reprod.;citation_author=R. Sreenivasan;citation_author=J. Jiang;citation_author=X. Wang;citation_author=R. Bártfai;citation_author=H. Y. Kwan;citation_title=Gonad differentiation in Zebrafish is regulated by the canonical wnt signaling pathway.;citation_volume=90;citation_year=2014;citation_pmid=24174574;citation_doi=10.1095/biolreprod.113.110874" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. J. G. Sutherland;citation_author=T. Gosselin;citation_author=E. Normandeau;citation_author=M. Lamothe;citation_author=N. Isabel;citation_title=Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.;citation_volume=8;citation_year=2016;citation_doi=10.1093/gbe/evw262" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. J. G. Sutherland;citation_author=K. C. Hanson;citation_author=J. R. Jantzen;citation_author=B. F. Koop;citation_author=C. T. Smith;citation_title=Divergent immunity and energetic programs in the gills of migratory and resident Oncorhynchus mykiss.;citation_pages=1952-1964;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12713" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. J. G. Sutherland;citation_author=K. W. Koczka;citation_author=M. Yasuike;citation_author=S. G. Jantzen;citation_author=R. Yazawa;citation_title=Comparative transcriptomics of Atlantic Salmo salar, Chum Oncorhynchus keta and Pink Salmon O. gorbuscha during infections with salmon lice Lepeophtheirus salmonis.;citation_pages=200;citation_volume=15;citation_year=2014;citation_pmid=24628956;citation_doi=10.1186/1471-2164-15-200" />
<meta name="citation_reference" content="citation_author=B. J. G. Sutherland;citation_author=C. Rico;citation_author=C. Audet;citation_author=L. Bernatchez;citation_year=2017;citation_pmid=28626004;citation_doi=10.1534/g3.117.040915" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Cell Dev. Biol.;citation_author=D. Thompson;citation_author=A. Regev;citation_author=S. Roy;citation_title=Comparative analysis of gene regulatory networks: from network reconstruction to evolution.;citation_pages=399-428;citation_volume=31;citation_year=2015;citation_doi=10.1146/annurev-cellbio-100913-012908" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Tritchler;citation_author=E. Parkhomenko;citation_author=J. Beyene;citation_title=Filtering genes for cluster and network analysis.;citation_pages=193-193;citation_volume=10;citation_year=2009;citation_pmid=19549335;citation_doi=10.1186/1471-2105-10-193" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Vainio;citation_author=M. Heikkilä;citation_author=A. Kispert;citation_author=N. Chin;citation_author=A. P. McMahon;citation_title=Female development in mammals is regulated by Wnt-4 signalling.;citation_pages=405-409;citation_volume=397;citation_year=1999;citation_issue=6718;citation_pmid=9989404;citation_doi=10.1038/17068" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=S. van Dam;citation_author=U. Võsa;citation_author=A. van der Graaf;citation_author=L. Franke;citation_author=J. P. de Magalhães;citation_title=Gene co-expression analysis for functional classification and gene-disease predictions.;citation_volume=19;citation_year=2017;citation_pmid=28077403;citation_doi=10.1093/bib/bbw139" />
<meta name="citation_reference" content="citation_journal_title=Funct. Ecol.;citation_author=P. J. van der Most;citation_author=B. de Jong;citation_author=H. K. Parmentier;citation_author=S. Verhulst;citation_title=Trade-off between growth and immune function: a meta-analysis of selection experiments.;citation_pages=74-80;citation_volume=25;citation_year=2010;citation_doi=10.1111/j.1365-2435.2010.01800.x" />
<meta name="citation_reference" content="citation_journal_title=Endocrinology;citation_author=A. van Nas;citation_author=D. Guhathakurta;citation_author=S. S. Wang;citation_author=N. Yehya;citation_author=S. Horvath;citation_title=Elucidating the role of gonadal hormones in sexually dimorphic gene coexpression networks.;citation_volume=150;citation_year=2009;citation_pmid=18974276;citation_doi=10.1210/en.2008-0563" />
<meta name="citation_reference" content="Warnes, G. R., B. Bolker, L. Bonebakker, R. Gentleman, W. Hubert et al., 2016 gplots: Various R Programming Tools for Plotting Data." />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. J. Wijchers;citation_author=R. J. Festenstein;citation_title=Epigenetic regulation of autosomal gene expression by sex chromosomes.;citation_pages=132-140;citation_volume=27;citation_year=2011;citation_issue=4;citation_pmid=21334089;citation_doi=10.1016/j.tig.2011.01.004" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Y. Wong;citation_author=M. M. McLeod;citation_author=J. Godwin;citation_title=Limited sex-biased neural gene expression patterns across strains in Zebrafish (Danio rerio).;citation_pages=905-910;citation_volume=15;citation_year=2014;citation_pmid=25326170;citation_doi=10.1186/1471-2164-15-905" />
<meta name="citation_reference" content="citation_journal_title=Evolution Letters;citation_author=A. E. Wright;citation_author=M. Fumagalli;citation_author=C. R. Cooney;citation_author=N. I. Bloch;citation_author=F. G. Vieira;citation_title=Male-biased gene expression resolves sexual conflict through the evolution of sex-specific genetic architecture.;citation_pages=52-61;citation_volume=2;citation_year=2018;citation_doi=10.1002/evl3.39" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Networks of co-expressed genes produce complex phenotypes associated with functional novelty. Sex differences in gene expression levels or in the structure of gene co-expression networks can cause sexual dimorphism and may resolve sexually antagonistic selection. Here we used RNA-sequencing in the salmonid Brook Charr Salvelinus fontinalis to characterize sex-specific co-expression networks in the liver of 47 female and 53 male offspring. In both networks, modules were characterized for functional enrichment, hub gene identification, and associations with 15 growth, reproduction, and stress-related phenotypes. Modules were then evaluated for preservation in the opposite sex, and in the congener Arctic Charr Salvelinus alpinus . Overall, more transcripts were assigned to a module in the female network than in the male network, which coincided with higher inter-individual gene expression and phenotype variation in the females. Most modules were preserved between sexes and species, including those involved in conserved cellular processes ( e.g. , translation, immune pathways). However, two sex-specific male modules were identified, and these may contribute to sexual dimorphism. To compare with the network analysis, differentially expressed transcripts were identified between the sexes, revealing a total of 16% of expressed transcripts as sex-biased. For both sexes, there was no overrepresentation of sex-biased genes or sex-specific modules on the putative sex chromosome. Sex-biased transcripts were also not overrepresented in sex-specific modules, and in fact highly male-biased transcripts were enriched in preserved modules. Comparative network analysis and differential expression analyses identified different aspects of sex differences in gene expression, and both provided new insights on the genes underlying sexual dimorphism in the salmonid Brook Charr." />
<meta name="og-title" property="og:title" content="Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/955" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Networks of co-expressed genes produce complex phenotypes associated with functional novelty. Sex differences in gene expression levels or in the structure of gene co-expression networks can cause sexual dimorphism and may resolve sexually antagonistic selection. Here we used RNA-sequencing in the salmonid Brook Charr Salvelinus fontinalis to characterize sex-specific co-expression networks in the liver of 47 female and 53 male offspring. In both networks, modules were characterized for functional enrichment, hub gene identification, and associations with 15 growth, reproduction, and stress-related phenotypes. Modules were then evaluated for preservation in the opposite sex, and in the congener Arctic Charr Salvelinus alpinus . Overall, more transcripts were assigned to a module in the female network than in the male network, which coincided with higher inter-individual gene expression and phenotype variation in the females. Most modules were preserved between sexes and species, including those involved in conserved cellular processes ( e.g. , translation, immune pathways). However, two sex-specific male modules were identified, and these may contribute to sexual dimorphism. To compare with the network analysis, differentially expressed transcripts were identified between the sexes, revealing a total of 16% of expressed transcripts as sex-biased. For both sexes, there was no overrepresentation of sex-biased genes or sex-specific modules on the putative sex chromosome. Sex-biased transcripts were also not overrepresented in sex-specific modules, and in fact highly male-biased transcripts were enriched in preserved modules. Comparative network analysis and differential expression analyses identified different aspects of sex differences in gene expression, and both provided new insights on the genes underlying sexual dimorphism in the salmonid Brook Charr." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"D5rK1s_TzgkzlwA7WAbs9i8dppflsZL6GoPwx-XDKks","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492490","apath":"\/ggg\/9\/3\/955.atom","pisa":"ggg;9\/3\/955","ac":{"\/ggg\/9\/3\/955.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/955.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/955"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/955"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/955":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ggv4gVrDd+jszC8oGVS7ayAK5gD84yWb0iYo4whlqPc4dIp2EvtjGtP53MzYW7aLz3Xtxt5klKaqW3OWvV6WdK6+rMEoQ42Y0Ew6qRndZmd1oPXGE4JtwjIg9qOFj1pfJFdC2I1GGPY\\\\\\\/REfCk70ubrxuVCiGZChRAgOBiZvCh0zQ+SAbRISUnd7WHCIWUiXhqhurpzlke6NEea+aOUuBMKmIDTwA6xXAlRs52OrUj0DBspYUp9i5rJxhN3PfeRQW5v1RpcgKOBS2EDf4PjemdWRgfKIrlMuNapcJWGpm8STllfYBuXK1YyKTSdBbISKgioAjzJcoPz6CRzqNxGDeLEjUMwzMe0FRhVIgh7LGL6V3JMkueANW8Tc3uqyFD\\\\\\\/MVgWFoSdeWPrWpwQRzjXuwcxeddTbaFBB8a4dhrxkf6aAF98bQgK9vniqp6cUGfdmf25Geb2zv2EVnk5K42MbbkljrP8anpT85WpRct\\\\\\\/7QafjVw9OrmRsxvTHegyvwPu6xuWkLhJw7m6PBBTrDfEs00yqyxLYipoe3J5qEUT8\\\\\\\/cWbOXBL18vhqb2FkX0JDsxy5eNbX6nM0Xnxx5zHouW4\\\\\\\/iUhBvQn0ni9M8kWPm6rE5SpWrm3wj+vtarSWIJ1dlmBf8\\\\\\\/7eVZeX8XJeoazSGUwLo3tBLwMWmeewOeZW74nhvc2mtODRwpen3UE0tl1sR\\\\\\\/XFxW8Q1wJKHk6W8C6iPS7rU6yw8qhZxe1s7CAP+1XWmXzNwOfhjwXVWLR\\\\\\\/dKs5RIGW5A+sSZxpWHcIFns8VnMyQDJPmHRzZayXPJo0KHX0SiiWHfbUKLPctoeF+Kr8vDlnd5RpiuxvcWnxIyyfSkKotk0qJTISd3I0so7sT5NES8bg0xUsyxF6Tas3jQXbKWBAY\\\\\\\/LVTIaBuZH9JMjj7gh84S30nCiKyyVMOOesRltctFJegTHeBr01OmvhTaYvLs8NY7c86edGXZ+cZQ+IdZoFa9DrYWdqHXNWlUqKc\\\\\\\/MEPh+8y10yqJgLeK8Hk0Q01YvqjJmt1TMuyMVxMmMCC1pFirnxtlkahM\\\\\\\/wUsscoK7y9gFByXxukd4og9Xn6fboFxzNCEfWv8Lz+DuwxozlTfTBiM0NIzNY\\\\\\\/lSzPbYMIDkwV9zwYnZylqDS++uGEh\\\\\\\/HiomcGeWWGNl3FLAl2lRgTWcG7X0vlfXquBcUyuDuiyeCsWi5oeKFtX1\\\\\\\/hSw0k4F3KfpGI6qOXmdVgUOGbGptuaYIwbkLHgs4jS2rgNSsaUSfdT8Li6xRG7xkDlqcLBTh8pe4\\\u0022,\\\u0022iv\\\u0022:\\\u0022lXzJOhrTxHU8V59aOF35lQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266698dcb53bc4a1240ea0ef63d83577de9d35b447acbd82eafc33b40d1d3c44b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OadVivl+c8ksn6wUrih+ZCZqZv7jQrQd09Q4bvMOh7rjRoVB8+HL83Njx\\\\\\\/PTF1RwT6CWJmDSNBf8s+KGceVA\\\\\\\/3UyHfzkA7S4NGjmfKQV0gztLPTDR2q5ioRTMEBJrk7LsNyaxiseU4TXpL6\\\\\\\/So\\\\\\\/Jv931XFJZNWXvu18eTyoZB26CJ1G8AH8erqCi+NUVuFRixnYsPxVGN6I4vnxNCDDwfv44cOHltKGZa9Y\\\\\\\/+XVsvr9hDzhvpsfI4B+M8IXQzb27JLaLbnaCk7+NDCmPkuf06eC59fHje9cYNSHraxkdPAD1ke+7x8NVvumcRqlpsdAYP+Mk17nmyxVNmYG+Bs\\\\\\\/M9G0RZqV7jHaQc9nCufHuLY08Cem2t8a6WQNRnEDKVpAuw5wbKr0mPuqHFRSQCM5wy4Q1rVW3kDX\\\\\\\/LMm\\\\\\\/REJwFUN+9N2FYGoRMrVzXTlqC\\\\\\\/JdpGvuLVd8uOHD\\\\\\\/CHs42NyHyPrvfmg9q0vWl7idYI7ALBylZ01Zoaa+T51Q23NNm4WYJesmh6\\\\\\\/KwbJUjsULeCWuOchvpWrjSKk\\\\\\\/TDoelzXjKXV+jlFKaylnwIuyjo1xUs8Vqlhxtt\\\\\\\/\\\\\\\/ftYN5qQ7osaFPVMwidDsj\\\\\\\/AnhCXV\\\\\\\/VGrL7oIZVeMw+LMPDHVEaXvluR3Bp03yFpU8r5ltQ0gJ52YQuiJ94pS95i9koncn6VT6uxH2jyDYNrfBvI1fi1uCYjkKut074ZuomATnlWdONkqfQW0zYE2rKOTbCTtWpwJSfhNSpVOyD1S1R4LmUP7iOppA6+5dQJCniKrpdxW1WqEoqG4ggQ+j2lz6lzGgcjk4oINKdDjS3vJgbhycUBvRpKxReoiwoeehAJB8DvwLE\\\\\\\/5XJQ3\\\\\\\/q\\\\\\\/oaT5r51joeVzyF0l1ev+eB9+tWtzLgxYG\\\\\\\/UK96FKOY2z8nTL0y0H0kGHe81grlNBJPt86dmNxvvR3x+vIyo\\\\\\\/vt0HouwGUrcIlyYJZIg+lyWRDTrieSzHXEoL6LHFn9Jjj6Y\\\\\\\/MNQK3rJ6+Os2ZOe4Lwwj+fPplUYoO0oYsGGAPBemrze8wPL3walv6fueVhBe2+NJN6T1qt3\\\\\\\/vfftWaVAfIdfPV\\\\\\\/MHQanKHdsv0wg0l6OJQXSubTg4IdEoqk6TiJn3LoRsCd9XfP6IeONYkUIWBUD+vW5FZ\\\\\\\/c9geHd+23LxEpKelbd3UWCNHBhIx64Vvyd58NtvyK7QMIoJlAueCGYaqIaA0dAsvGEvT5KpMeCtAFljfJrHjtrby\\\\\\\/oSWJqG4oEfS3WTAOHHeuAMzp1jKsyaOmuC0XKHIwbd1tiUnmNIaMAavfCKwWSeiNgnPz1JJm2tPjWqzD0InCC99CvPr6X8ahbsMRpm92zsIRmw4N93iVcUkG1kg8eOyfaoQ6ALEFze5yIua\\\\\\\/uAnRVhMmD6eWnM5yQJdy6iM6T3vlOvifWE1ocDLLvb1NxLKjj7IHNqjfQ2FGQ6GvXka5jVub8lD7amUuAOHzkWN8RMIYMOEM4DuG6XyG8oWl7D689y6M6vEEgZ6V\\\\\\\/WocsBltQq58cxcmXFAGQbh4TZKFVNnETJegUp24+vTKWn3NiRMY8rKHmLgNWc5cm4eeTG9682iMshumfkSVTELIFiezDEMFuCFhQhV5tkWudDAOKbDRs5PBAoMZms1tH0J01vrYvuf4I9yZsN+jpUgzIzsC+dJOa79IIwcz9nSi0IoaU7emQTDGytlBpKjRC\\\\\\\/xiBFpBVlejDBmW542F6nLZzuZ2owgps1LpkFvElNXVhb+4jF7hldbZKgB7eDEqbDY1a8OnNEQySTkCZY\\\\\\\/2MOdoXvm0vL\\\\\\\/DXT1afL+Egtsb+p+MUNxHAfXpDweBzYelSyfmL0zg1pUYYZg94Ech7IC46xBs9NyVcBOj5OzInzC2Oa4\\\\\\\/kkGMbKsR1y3ggj1BKNaqal7Usl435KWjdpeF6D6MkZhrM4ztpqSzh\\\\\\\/SD+zdisPman\\\\\\\/ZWDrjm\\\\\\\/8vWFxQWFhOfgg3OnYFXbNjDA8i\\\\\\\/sa3PyG28W5NNTKpR4lz3DW84U\\\\\\\/GktOuYwKzS32fhH5Zhbsjbik5I4NoyfrKYUMANKk+K+V\\\\\\\/BZgtkaMsBdKjgFWDBQ+Xb\\\\\\\/sCLmdzNTPKZz22B7ipkXZHGC1HHiu8Ws+lkUfVxUM0k5U1isuCzhcAWUQ8bTfILbrrX7mt9fI+3yzRK7CNJH0eOAq0Ax9U8S+oN6liDQfsDiyU2mUOM0Wh4497FByBTg5Fe+g\\\\\\\/MbEc6MGU78p+nnId9rFu9lSJUK\\\\\\\/xrU3W0kSMa2lUSOgP\\\\\\\/fN8TpJg9mjdZ5kcUU4ELsQUi1AtpzyGh3kH1TmECZ2grg\\\\\\\/XKuWRCTODf9YL9A8ZAWeOJh0zV\\\\\\\/X0ceCvQQlWcf0\\\\\\\/6ag5zY\\\\\\\/BW4RR0Z\\\\\\\/Ox6aw19AAZ3IWTMW7eXuv3UX0UeMDkmZtSqrWIh46E6A8TTcry6CSNgED653x9M42eTsyybUT5pE3JE94NOwYDwWcOYi+Dho2z20TRIGEjXj+EshzsI1hltP02eeTEpfnz23KB7kHWdRC9qhvvQukc\\\\\\\/7bTqS2he1KVFvAsen\\\\\\\/y8oW+SwEI\\\\\\\/a4An04MK66kZ+vdInQHNOe2\\\\\\\/1kJaZOZUpmu7dJ0p0dLHcPHwYYbEW05nfcw6\\\\\\\/KtqoV6zDz73ibl+oyaRkImRU29Zgk+m\\\\\\\/WWiCbOUcAxcW5CZbX3+\\\\\\\/NUgctffR2\\\\\\\/8UCkAaSrkG59H3nNFvhtd52EMgWn6r2ThFd+O79Bjb4InrNtpXVJ0EtDROvnOCEfWwacZ9oJszMqm\\\\\\\/oEiWKkI6s3bKp5tNALmxq6H+sm89\\\\\\\/XtOzWlY56QvDgoJZcBi9\\\\\\\/BNKowe1+DjnpEFSo5PZ1Fbm8+O1ILz3eNsz77CJmej9sCKpRX+ZVAhgxNfUKmqrYWYSi2oax108B0lPZv+XTXLk+v7a6SV3prp5GmX1GeF9CCb29ccONOjB9SabJjw3y1jjCs5ruvixCOPZLra+\\\\\\\/2ZAHuE2Ng6ZjU6ld8RxeMdt6BrBEtbNZQpBptZB6t9uLXpSgm5zDzQ9AYo1qeJja+yiMCPWyFGLXceDlo5Zd2kT2sdiv3NMKL8oJt9sIh+Eze03njsPghPKOv7nAnuORV6zX3rabcDGn9OFly8RyhMA3BY0rJG\\\\\\\/+YfpqUOsW4wTXwuzlAAYgOs55fVzTsOoUonK5th9mute6LdZitYXA7NbMPnjCZZitusnaDwR9aBw11JcpOylYfLRRcBZ9JVOgCghP9y7cXAVYCEPzmNHd9PTL3sHd1C9lZN9z5QBmOZ+kjfZmlvOBuudT\\\\\\\/OBB2\\\\\\\/WilIBb34aBWvAnxOkpbyH2yJpKjGwgXzvk3v9PvLlD3nRwfqOOD2AaafBZnw5BJkhm\\\\\\\/KRAakIZ\\\\\\\/th+GWGHmPPhyaslK2rwFTsYxjnYcMiWZwLJ2xx6aoEez0sMIx0oNAIuemXaY81KhQjEBzD1sO+2EXPNGu+5KDl5XJ3Y7IyHzKlDGD\\\\\\\/KhF4WKieP+fq3vf8w6LAslKnq7uIqvxzNoyn6HXZMLOv+RIqlEsQfaREJNz7EqbgBOon6B28JwPfPLo8jvqlPvyytF6bRMYoMOezdbv2TmGGpjllYoEj7FhHCGU0cZVkRbfzx\\\\\\\/DYGG8ZyDrgdoWs6oNEI4a0hkpdax\\\\\\\/PtZf8KIF8iwBFxO+80iOq2L6TYI34dBnrBzFPr5jaYo9xv7bdpBmnrhgDN2vCgYx+UgvYoFzMx53XOGnV\\\\\\\/EkejA58NIILw3c+YTWx3uxLN92kXJXpaYU1SvkrNCcnNZzZd3MAsnEF0+K\\\\\\\/SO9gTLXlxz\\\\\\\/tAv2K5SCg+X5m3B5jAxmzE8gBs4HiK955dk+bLxvXCKG\\\\\\\/01BHbEKLVtqCq9s9t0YNP5YL2FM5PgeOXn\\\\\\\/frs70UPBW\\\\\\\/VcaW4lkZ6+gUePxyAOt6N83F9zhPyCwzO8swXDfchi34XYgphlFz9bBz8HPPqC0HeyUQZX5oQpdqOAXljcK6DtBI\\\\\\\/63ePz6NmxLfisPaWndrm2zAwBTLQWxAhKzyQpcATqJ8cBWS0cSX7i1ta085X+6g9Xj2fJfSSLwuwIpkLl9kdHHRC8E\\\\\\\/7LanNn3Y+VgM01ZGyFV1WCRFcKXpvQBYxlofFDM7HlqUTsBnilYIFq\\\\\\\/ajuxtjkHjH9P+U1mWSO\\\\\\\/sCQqOP1\\\\\\\/ENik07d9GeIb6k58AaXXOedw5JuDrRzRAgnbA4Z2mNYgSHqoG41yTjcoppaM\\\\\\\/Z8GWb4oER4RoK9QMYrArFQclvUf50UCtreaYbnjsTzbLilqjbinLSAlTOBP3XxwpKQib5qnl1P38D3my2YsJLzZmM45b9WBc6vh21zX6\\\\\\\/0xypi5g69vfaT\\\\\\\/F60A2ogYKSz4etkoIjKpBcZJfMsNFzj7tuNANmwx7uJ\\\\\\\/yugpKR9HVDXz6WPsDHnXZLvH7VJjmZyfNG9eM7DF7tFk06Zi4YGT1uudmN84hHjUOf9brRD0tyqLuaV9g\\\\\\\/ZpA84KqgSnwsS8UrRwLErSd\\\\\\\/VyCKts+DSB99A\\\\\\\/FDZ5Jv6XY\\\\\\\/IkUo+LC\\\\\\\/momD0lzvkQnZPHSBNKT2Pc2kUQqvVB1j77jKTMMEO+mU6NBRQj2E7Qu2BOUkUjo6coBa9tK2Wg2gC9CfjSnqmnt+D88oY5Dt8E2wQRuV3AILvaFuW7VfFTcZEPHnnG3QG3QAseVELeymUDHZONUfztocXGksqLtiZols7xDzIOGd\\\\\\\/s5Ewqs+DBWOrmpOl9eRunxiQZjmr3izGNsDyFxY1JnTtXC3MCxcbNZQMmmkpZ8fLDpDlAc\\\\\\\/KqsaYNYgbKZhGwRbWOqECqEzGaUAyQUnJX1KFYBGiC88zTLC+OXPxQY7+CvhHwDtfCoBWcSlG1zEsOTVx58JItXeLvNMs7IAq808eisfobOF56xC8DzwPPPew30uJGYMbTBKt8JJJs7EHxZ7+Orvxtxxa5WFfz8QxL7ktvsJw3+4EJs6h997ePKIo\\\\\\\/1gysg93Azg1154PS6qRc2iJL\\\\\\\/q1hdwgww7poEdX7TanNfnuX71gmtI0D0Fu9ybyCgliKKGceVsMDpgqBF2XxA4xn7LJ5A7n6QrOd6LPrJRaODQfsIAkh2NF\\\\\\\/EsRmV0O5VKqC8sb5JHb3pOrhxQk3wTXFOGJgqRkV8reOewR\\\\\\\/OIVmv7jW9XGWb1Xtb\\\\\\\/H+w3uMuOugFwffalMZk3x42hDbKBi7StqOL+\\\\\\\/doUQlkE+VCA8gPwNPFybsaDhnFBIXb6REHOnEvGfKkbjZNIqCGE+GwlaNC36feSS0ealx\\\\\\\/SW26oSj7uBIwtbNM7yRNIk5TdjTOWitA6Yo2f13KPpjh77GdE8zi11D2v1bMRqbfwu3Hh8pdRkx3cbTLPhv+j+hn8XnvrUqXk1IEUZVQCitT1bsTykBqfsxkIE7eQlehK7mLXjruwMcbYhIC5HcTzjgq1saU+EHJxqMb2\\\\\\\/WWIm92+gqaHGFI9L0kkJDU0FSD7n14TzyIEWXYWg775sbZL0OfLZCCJgvZRfXfy6AGDEW6RuRUFj\\\\\\\/3fBZ9r0vdymtjmVitcr6zC0UKLWV\\\\\\\/K5soattFe5AJ9T0pvGQgTg0DM7v4AAI\\\\\\\/Q0HWNzOxT7z+FIWQnGNNH+tx8+ienM0IkSWWtvqvtwPIrMSh63CX\\\\\\\/AhuJua2obWLJethhGumNoe4+psohxF5geR5kNAp8KgWMBWQXoWtgpAQC\\\\\\\/4Q5G4WWsxx6WFFdRZFbpHyrsuSfC1kerxvKnuHGRfN8q0KoWg++GCwUXi9MLZsnvz+z0E03ZyWKGLzRcNNCEJo\\\\\\\/dfubECIGk5IB4BvQj3OM9BLNeK1sbTjx1VIZKyLf8KUNWFaraHDG17\\\\\\\/G79bRdHC+8H6gqCYXV43XI+GOT9ZY7zALbSg7cz0rY5OzZPHvG9yBwfokOYXRCIeZ1qnSFYeNP8wqn8qUl6G6yQ6AtlpC7J7W8VTA4VB2wbHasVlqCFJK8jRFR52sm+0phI00+90Y7rWqxjYIQg5Rw8TwsV\\\\\\\/YvBhrCnnag4BcNzrDbFQ0rf9ApYWervNd99PP3cwP260G3gjbCGNIsJG\\\\\\\/sGcyQBjLPWPuJssOHpkCRONJdi+dqjX0IIrzOyAi5E6lPNjw3KF7SqGC3im16WuMoxqPLjg5HE27BIEMIMoNy\\\\\\\/IdTeHxh3I\\\\\\\/QWVko8rOHKuiJxCPIFuW8ThUUJrEFkblCUhnWiDdTCDv01oRlRTZWr26kWX6akS0X8enmoc+4+yrsAivLLR37FE+mfBXmOObXebE4eZj8QYe2Pld9d\\\\\\\/CFHR9mL3XODOUduH6BLNY\\\\\\\/tbSIsyr8aAKpvHZQwtYkUh6sK2cls7ke+LMIiZ7MSUthdl4B5qoCMVJOZF5ryqf2NjvoMR5f9aAYtD1omOWaTP8p75ol08iB2MVOwkaO0kysLxYmz7wV9hOGiOGsWy\\\\\\\/BA8+C5suWHWREunO+WEAJ+3\\\\\\\/XQ0qpYFUByNXHGtHEToD+5L8\\\\\\\/UY4AhglS1oWtHLGuSE8LTen4+MywAZDpKyZu3hlimINoa3OoquGQTJjWdSbhGBz9VX6zPNYiltNZHJScf8hagz0gKY4Tv30kR1vl+Hk7Wzp65F3fahV8l3W01R\\\\\\\/ptgeuWmFm6PvkJd3J3waOVogSVF6RvVMkKz\\\\\\\/yK2nn32maf27Nz5bY8uYFo2\\\\\\\/rqDcbBI3Y\\\\\\\/hca8VSZOVfWHe6c3iaSLrEvEmXp5c8DHtdOUJjtSOq\\\\\\\/vTf+7p+zbsjNheCee3fe1K71AzCc56RWAJ30ZywqRSmGPnSYBIOZjc+EleB3WBDM8IeMt6N57Wz1D5FjDluDUKqj1paK1EE\\\\\\\/2bUk4Kr96K6MnquqvPkMn8FHJZKu3TA75a78fH99TImIoeOq2P6iD9g7\\\\\\\/CIDTV34WeCtg2I7ERX7vAC3qZma0NnUG0Rk1LjFFmUIJItRJgOP7cP7O0pddt9HTjmY04id93i+7Oe5u8r7kGrRZt88NyIBDjEVJH4RMEP3h599OD5LiQHutvjwbIdx8XOhRfrmuSFCTfCAN\\\\\\\/D\\\\\\\/ORiZGY+N8A8kwLF9yxBRerlsTpJog5x\\\\\\\/JGKcOOwrz6DvsYlgJPgr7yjzOEYOAxljcSFIVL0NjAOzsu+2NMHoRyYyG5jr4++bjl+T+b\\\\\\\/UkegDpSeyLmJYfurwMc8C3K4orDT8bVC35BXN9DlE8xBdhHyWoONMPKjfwADhWOOmhBooRK5+15s8maXpseebRBroVg6CP\\\\\\\/iDuCUK8r73AUJQ59TUennNnKSG6Ck5m0ixpQ7nSQDSst3cfJD\\\\\\\/vRZOR+C5fBzFRUFOy78Xe6eKct7NGsC0TPfL354wYQkYB4y7OcaXhhudJFIFjM0RviTw3o4vDIiPMMwmEe+5SdeQEj0EDnhqq9DOc3xbovbl7kewpte3vLi7yvyl5eaGUP7dilqjoMm6k63UrwifBhzW+jlkVraANbFEqTPafOTr1RsWQyhdh1Rq52IWcSVD7ZYw33Hd7wgNnZ\\\\\\\/E0Dy\\\\\\\/QbaaE4euu2qK2l6Hqlzrj2e3okpl1wWfR8TBaUBQKN5WWoDYLEm8IXpU\\\\\\\/t6XTFL0+d5hymq4jRBYsAPH4ZsypOh1dhG2XyQFT7TlxvOLv2NzW8fdWVVbqQ3ECeiMj9X0Y76XLBwMIQkxlgrwKyiXvwve7Z4q4kV6rR2O90Gm1Qkx9Ga4\\\\\\\/4pL0u1MCTJWm460GKp21PprYyr3ptKuo2a9yavf1LCt1AZR9UJEnvG0E7ABIqGeoJGfE6FZWnRG0bUIT95jZyzkIvrY9JP66PwmqxTZy3a0OeTmHgQLoZEpcja4moqWd7Y8oCF3awb+4ZzuhN6VfSwWEzAO4BMalh1pn2Jn9HlKIuLoRVYCzpO3OF3Y7CRbCitciCIec940olb+Ir4f6wTvcj6F7PctU+lywP0JtkZIFGaH\\\\\\\/99YfWX2JrVr8DxXwNzWDfT7HUedcMh+qmfSvT1hfLBUBvoMRKYhFQgRO+2G4cSKGnrPFdi50fDv2fQXtwP+EwyCuRUITBaQNBaSzdZbK5zdvFhOCC8hpvFmF4GgnFwIHQQU1ukMC1jf0iL8U+0gNbqyaaBXyErtr\\\\\\\/LdhPA1jiD7Lm+z4p0eMsXd\\\\\\\/mHSthyczh6Fub8cH8YSqlKm\\\\\\\/IGc46QsqGFmE34Iz5HgfXsfqms4t9TmSJgf2PgrJP3ywqXNgQ5Kpny\\\\\\\/FphivGPzOVORzn+h+OR9jfKxs4nR1KfE6ubeTTkN6LyFA\\\\\\\/KkhZDSstaITtPVRbgy0r1omhW23FcKz+I1xAAUQaFd9xuon6boF61GvOISnKVzr3M+VdxMasn74wn8wD6XarFQvyZ1SfwBOVNSxqsR4oRg89E\\\\\\\/l47IicgiFL9hYzQtx9YcgmRne36ZOhzPaqq5l31s41Ur\\\\\\\/0baTa3ipZKQkO2KKYIj+6zFX2\\\\\\\/tySolZepw4q9Sc5V1hjroQoovj8C\\\\\\\/teeee+qJ33Egi6TVCoBeSiw4yQloI9oXgNjv9miDp\\\\\\\/AdR9EsWqH0hcUV7ui72kp3bsS7IVqGiUyT79JZp5tmjJ7OgNMv3Ni9xglCnWZWuVdenxdiaaAHiWj+ZvK591f03WuzzyUMJwDvPbo+En6UqmNvpJz9GpYFunat6Cntq+YE\\\\\\\/1JO3PQ+3csCumF2wFTmg3M6lLTdrMk4CrNJvg8TkRMPyahShZMarvhvfYw+S\\\\\\\/oJRPPhbUo9Qbf3t\\\\\\\/AdTDqzyoz3jzR5gRleYbyAWBXhJKxbo\\\\\\\/3xJOmfOAvqEU9gVQEaMlSgtAZdgn2jfXz6\\\\\\\/eBRq6LLnrr3nOprvD+R16P\\\\\\\/NrP\\\\\\\/UYkVkSWijOfFATirbu6mVQy3ath7CyP5COQjY4Kfqo5D4dHxzkW+cXTdUbeKKZ82qiFZyq0m5GOMlOSfR2C4rk1VR\\\\\\\/v4\\\\\\\/8Fpl34rEA+tXcDNvbdgZTi63wP5KTg1ORCWNOnYgYnjicqHPLVbNmdMpLHnEJrNXnsMLpzTcaATViitSgceNKdL8udXJ2\\\\\\\/6lVc7Thx1gtTVC0M6qGGU0G0tmFcgXi3KPNsdShB0Orc7L+8chhrCZATCm1Jfwj5wOafaKvSRHJkik3glmwphFoUgDedBf088flGciioFpik\\\\\\\/vhFPUuk5R1cAMYGQDUMNTYkq\\\\\\\/uM5Hr\\\\\\\/kY+Xix1W\\\\\\\/uKK5CRZs3fd09H4BSJ4TtT3QvbTy9G64wzHD1eyZNsn8MCUVpcpRHZz2f5+FpFrgBM\\\\\\\/Kd6vclbsR0t7QMyE6+KoXc4d3QwOBdce\\\\\\\/NvFuuRgIPtIufROsLPhUiGT55Z5m1EDwdlNS\\\\\\\/F+0+s5trtH\\\\\\\/rdRgb6P64w+QAq5Y4sWm0E7N2uHF6L5oKRpE4GMexOoJbYcLaUhPQfRuPL3nmcCW3Q40arU2CFtJdMGdZMAHentg7ytW09nKwr8aJSNGB6LFyrhGzH1WV0kUkx4toSU1gYY+AFZOwQEFsVtmkGNDti0Lleurfhz4Z\\\\\\\/bSaYFrH3l8Bj3cMkSG\\\\\\\/Sg\\\\\\\/Grlx2SxkUzpJ3gL3JqmFYYGpj\\\\\\\/vSf1Vcx\\\\\\\/REAMoou1YgRjn1W50qthfOa5UUvslaBqdZRfSYKFj8IcxjapXkorgqcN+hSl4fhRWNogEOwZa78t0UXkTkoxlcbLU9V2gDZWuOPNtjrxpgHZZd4\\\\\\\/2YduFu5u1OZzuYdGDwQs9ioeghtIpO12At0SB+iOxs\\\\\\\/TsQP3qPOBzBfX5eXGORFh2yivJxpsaNse4Jo24kqZ2WZCbO6bLJCmwv\\\\\\\/31JrW466pvVyX5HMPC4Gp9tpcEWJKpKwg7MIhjzje7PfVnGMaGJnLYbfV0c9x\\\\\\\/GCIeYGQXChAcayvCtrfWVN2RrSE1uNNSLnntLNnCLAGxTTV2o8tlHN8L\\\\\\\/LljQmWKAT9j96bfxB4I4pqIdd5lhTF52sDIXBxJdfQlmegzDzenhtrlKvvbniX9sBwRXsPRzJVZzCm9xmy6076nCxSeLNzQZL7bvMjIT+ytTSPF1T0mJieOaanttcwmKGHlRU6CVUU41QV7nO++nc\\\\\\\/4uNoVbBJ4fohuSS53660M+WQzUlviZgyVm+ghl1K4se1M5GPQUsvYKxyKpZQBpDA1h1npf4kcsG6LSFzJFI8hRo8erLcuIAJUUkVgPpaIl8y2RCkcQUveN19POfrRWN7aX+Ep7KrMlfaIoTgYOVzdFaxU8kJeBiK3uuOV\\\\\\\/fQmz+C+gLPTjq8oJVUTfshrVlmRrmzZjHYsxV0MUNuCZKmGj\\\\\\\/OYN1T8R5cqsc\\\\\\\/6BMuFkRtIDnLpBmCjbq2rEBOW47d+rUvGOmA4kBubdnnGZhbznkDtptwdMKk00LO0Ryu9M5NK5+T6nW\\\\\\\/DU41GNZFG08ffqKptV4leh2reJQ+wBnexnmED9nVnsn4H\\\\\\\/Rtj6Y8ZePzPKrPLd0WANXggMLnwHIZlKXrNTVMrdsnJUjGiW4qTA5k2hfkZsBNV\\\\\\\/9d93k2E9z3uLByprW8+ZMtpQ6OkdJ02jbWaPZPFDVzaOfTDkgaWBERh1bzSlppj53yD\\\\\\\/ATzKmJZiN+Ma9GmuYbnw4SefYvPNrVq1XbNQmXCQ8EEtzL5Q6\\\\\\\/ugIpS9USVTPgkrfYyj3vdcRJMgihIKzDxRwmmpg6CWs1bnry+uAKoHzG\\\\\\\/rOFLev8SvoBinhgx98+EBSwXV1f9iNdMljTCLWxts\\\\\\\/2KJLSuJvh1Pla4RmPIdTEEPsSRMMKECs5DRaA5xgVlxvOXV2njL6j9B5yqk1AKlKyt34XixQUoXbz6RUu+EJU\\\\\\\/B3oYmNIBLwzWJ6q62vjVMLja\\\\\\\/2CrUUrJJCqwyX51QohSjC\\\\\\\/v7vuV\\\\\\\/23UcVk2ziV2CQBYAQmtQhQc3lCaJaoL+UjGDESXyF40jh18qd3Er0NYuI5lNvFsM+\\\\\\\/dGwbTmsMmzrSJ2fMn+PrUmKxRTE6exBSrEhCzRVffXxxjXKt5CULK\\\\\\\/sQFWDrowTo6txCJVWJfet6XD+DwMDzzkwuG5eDAxk0KiTRxcXjuNJviXYr9DrnlDcl8IITEYkzXyaZch8wsL\\\\\\\/vC2AQUHFj2uMYryUurcFGSbsRTHFx5hdJKMzDo8cFfR3kO6y5LckWbIpNKPYQBo8NY8uOvBNeB3bJWnwcvExmXOTuvwFONWks3l3aVKm5YhvqRu096BA1HKwVWUH5PoZmltLm8CN6mDFK+vAFpaplJc5Gy3hDyXsO01Az+A4wrmHWKAjLNnACVCiRlWDTPVcDR+D0x7d5rVQEZOVcDCSnp59weA8J9nVKLX1sCS7xObjEW1+tEJ8ouf4bvk0toOgsKlcKOEquaaoA0jamXtUXD0CW4hhsZoJwn78vzV93MCeZQuHUNrcVNkBvTSKWGY5vcAVZMVWXeUt3yR1DZq3y79ErUClfPnMvgqaAkwFnOHRx6oSzQ\\\\\\\/nDRZb0oNgkWj1dhx2x6FSOUmNVEeVFF2ShB5Sx+jBbe4JOpvXSHaG+sJfMxlzIgK35TlTLZGHnB70AdkpuQQpetSW\\\\\\\/FwCWhGHc2ZRqX5TnL+5YzR9Id3F5iJsMWWnnhHNd3er5vQjkGm1KBtg9pJ1JNB5oxrynNIBV4fbqNGgsSP83xtjsGulrR9S5iyvRJPIpaK1UCMt0D3dzKg6ZEKe6+CNFY0qi2qfOcb3Zn\\\\\\\/asDE5ASbB\\\\\\\/V2s4PN1xjWx\\\\\\\/oToCFP7E4S4m7IQgKvETtj9DQWLJq4lVTMnRX1q3ScoSDOxNh8uYEjNLb16HHNGQYvdW7C7mwryKEXDrqAu2SmLABMFYfcq\\\\\\\/K1Pr95i4B5AAG81yqoNf1q8LqfRkxbfqWzYWc18pzryVt64mMQ14ANCR081HWeoSCZltaliUCocCShyNxNWWwPx\\\\\\\/coDh8AfEdepPOS7wbVo0OksVAZT88qOmZ\\\\\\\/bqMvLjj4CZH0wOQ5IfXSVZbfF7tiOKbHReBc9nru6hjfBq3Gnlbsp3t7bxKtd6SJk6Iw8lyE0my0sTe8dAF19XOalCxcxdL5zryozAmgak4HdGyepMxq3k8fSYy1nejtUdEDK9ABjoDsF\\\\\\\/VJvdBA92j0CEYhNQpsc8\\\\\\\/VtWUI8UybOZqwcOttfEWLCx2Zdp01YzGBZPIB5pCELMRGKfJVO6Bgct90dEy47RqZGsGr3QZNbin\\\\\\\/EwLNaE7awl35U2dnCHGkWvveTPlQ38pCjXTJTbLWsqN8lbpnGpYH9ybY9clbEO\\\\\\\/dP9lfCrOCrVJMK6iJ9cf3XTcX4N4WRvM6HQLytZuKmDhw2mKy9lshxwka4cD0gRYLrCTccXSysyMapkB1Lv8Dgx24yBRKe1RNlGnWYqICNAf+X7fU10Iph3SBKqec4NtMbjV0U1DRnjwJyvx+ieq3pnM97nfTZWteHPCGnCgfK1y2UmXLNFLV7p6AKmUudsVoxMiWJfrxSB9wTSoEQzEA8NRg0ibrRBH1OzfNsxEu41n8tMsYZY3EPLSKx\\\\\\\/k8mdpmuqsgnlaSZDmYMoXG9GAT5+IkaZQLow4P\\\\\\\/68ZBfWQloFVoLW\\\\\\\/U7vA5310rvJkW3HWEVZ8rMxu\\\\\\\/0Y5Dif1WqELqY3l06NX2r\\\\\\\/7d5HLFui4XQw3qFbY7ktIDQ6UU8QI3dHlihCvnxcee1YA5ai5BeqvTWtzC9SiyhdQ8JaLlQGXmzLA0XK3pNh0cSuWS82rOlwgiQZukmuuSHGsnAup5H6k4uYZYAayxs\\\\\\\/h6W05Bz54SXnApuioapRmGR35mg8gPRBXRr1L88La2OM4lrc\\\\\\\/bGU0ccJFcb558yBHHTNsVbspf8vpuiG069ceiOjiWkWzfz\\\\\\\/HMoY7iQ5Ad\\\\\\\/YnpqarHp4C+Dpakdqtw9yZMK\\\\\\\/K34rT+bPdUn1sfJMqY3mpUA7UcKthvIrHT23wZm2UyHblMFc43b6pXZZguWkDH8CT\\\\\\\/Vnjuq2hJeKw6OUhYt9siiZ881kF+qaSmWEL4T1HrjGGn+Pm0nbvXa1vlt\\\\\\\/k26z9fq0OD8HiadS4G4TT2vy6oV3v8OoVI8F7O+8Byyj6ooQzibKpiuytFPTth1m+FMHMuMFjYIZCsCWLjfZWuaCPmHVFVm67vbcWNrjgKWPM\\\\\\\/J3341CVZ4j4u0yK47prxx8J+UXRfdaFfXopktcLTTZdOmTev4MlcjG+yQeeG4sNfG\\\\\\\/phuBh2Tybhwo1Hi9bUc3Q5QCh8uaTUVAGUmNaIN9hvT2qIYO\\\\\\\/Fz6\\\\\\\/GPCvh7lEYJpj40Pp9+lAMYM9rvnopQL0P3nlSpBdlxMRyAvRmFXWGIX9ttRdebS\\\\\\\/rhKxxHHvVm8OaNT4EMlSQUWHDJntxrAu4BdEKoN1H5WqzEUVI7Up3kTgLrFHX\\\\\\\/rpRSRG5T8vybhqStciE+ccwhNEZvq9gt+rmsEQYd1OQhADm9BXNXJSh88h7ukIOhudSFcTp9egtx+nMjE9+obPwJQJOScAw18RXHUlC8hF\\\\\\\/nDUG0uhkNQBEJ2Yc4s+ldAZ4GGXI23ANT7sArMPbKXOTGX+Hb8vL+1ipPz7eKrVWDjtpXBzhB7Ttg1UeMPUKsxUG7tRMeURRYSO071kREs1prnzQeTSdJx9FbV\\\\\\\/2iVHkd8lJX8+Yt4KtPC6XOTwsOm5le11SlmToiICB6J+dZrO9wxn3euBHfx2CzsEXGM92oR1aVZIIAXiwuZlhUbZmv4T5wYRE3X9q5oTJBpb+x509C1K\\\\\\\/4PoiAJ4CUc6GqJq+j4BeitlDWt1ITCu4SPyXDIj9xeCrtj0cNn3sZ30i9mUjQ05c98y\\\\\\\/ydqCOaMQ74AmfRn2h0vhwsQMDjSDocKG0ITUftgZJWeppQ7i43Fnx478\\\\\\\/Aj4g3fpzCHcY3\\\\\\\/AqbH0y5Z1CUepXfPZQdL7ldIgT\\\\\\\/OuRzP\\\\\\\/\\\\\\\/FnZnDK+cWrEpKgtSVdeW32cv\\\\\\\/8eb3AawSvLC89sS\\\\\\\/5ocGy9PfK7gE1PrYfQSSarMesuSRv9jJeRGEmvBKI8i5xP\\\\\\\/yTWP3SWm+xnQWbyGF8xHPRy7Oj68yglYHfxmLDsp5GhI4hUbRhswzmdM7Sty5Na4pVpdQ4\\\\\\\/T8f3l+OMRwqTuuqnyIyQZKOcaZGm6yaXfgrxaqMYO+TfWC\\\\\\\/FoAundYYI8LQ6pFvOTkJmYhpugS7LlGm00m8CIzxKVKdV+Qw+63lTzHqiHeQsG6xfuIukSis67w5dJH4lMkmW2oh3rSuCMSlAEclMR7P8DNXhnD8GssLFOwWSBTk2reS4JvkBZaDHilZsNQzMPYR7dY1EgRAiCDFmbPPR7gQqDqlPXpsc5uqei5\\\\\\\/KiA4Nnsxq8TmB8hZlaFzA\\\\\\\/z7WBTZp7cWCf1Y+dPOVhJnLQp5lKShNpQcS+bOsT14DYVpDYeWz+KVHu6IT6kxfIw3Xd4ntSdQCjjvNU3vRXjGD+YKORgqbBqYsqN7C0txKWq3+C7t+ev1FpkKOtY\\\\\\\/2BLiVOMFztOlDO7CuLPdAiTY+qfRp0ZZRiksaKZkGI8n1uLpQuFKCRIZqQU61L+OOEeF03antn\\\\\\\/84+OAUtNW6tmrUJCjn2fn5lpTXPC\\\\\\\/Lfsvv\\\\\\\/Wc5lkLk7j5njlcgWlzrziEolTnL704Y19FF5KTrIFQCjdev99MP30HaBWJ8U3KHB69kSPLuge0i5qhE5ZOvAcofLpI9npO2a1y\\\\\\\/OTUCg+kwKD7655IKQckjbr4qoiAT5eJZaL7pibltuWuLbtQ7b5IiCv13lJA3Sdfl0+Yv26qNGBx5ACf\\\\\\\/fZyM0wi2hsVC8iWRWyQAOb4\\\\\\\/O0A71nwrIlgVIGdYDBJaVIcJCQR4lVP\\\\\\\/0kVEL3\\\\\\\/YxpxF2uqj7Cj6ws4Peb8mmRGyn3E\\\\\\\/TEkjZz1WSiZWI+bM5vCd7zejqI5pojF8wNewb5r+jm1NAy3pB6xUopS2DHWmuATu8ZMzRbLHVMrZuMd7wuzUpevFIbExkeH8EJQQxpdp2DACtsJaq0ZzTIjTLS4L2wbtMHZkchJmG8ZZah5JNnx0bjkDmeRtEC9nM\\\\\\\/sm+zDZnK3rBmmCcZimqWe0vzKAw3wiiyKcZgosmIYR1fHD\\\\\\\/cf8CNeOHucV2cBr00ZNWwJqD2ngM9UJh0p21wsfmMae+ChAaBeIBOL66DQninpa8mleL7fACC7HKhXYgSUBs+nordCjfN5tfMydZR5n+xHjnOZY5PaileERA3k9e7wX+2Fl1wEf4oktSvablDhyQXFdcCbsT5\\\\\\\/qKGlSDUYTxPZiIo\\\\\\\/VcTRTMF5Uidw0GicW5TDLAV3JkU0JUuhK6hmx+H9wnBgebAD7B1fCS7K64DPqBfPcxzs4TnCd8ITd\\\\\\\/tmzmutju9MJNjwr4CdPOpGgI4TutijYgpN5VpGPq9nm28r8PzTFxWKoSj+MHqK6UM4T8KwYdS2GHam9XMstTVCuEHNqII5s2wdaCS\\\\\\\/nC+5QFoVLQ3ioNO4bOLDo8H76v3QLmqU7lHRsKZZEzXXp4E2I7nt4QBcsrtD4sMR9i5d0luV2\\\\\\\/D\\\\\\\/BLPwm25HOxX6cOKByaWudJLkWIu8MNv6dzJbJMzZefRm7gek4Le5aVVq4ZJmdwfpjp\\\\\\\/gXfuqfuiTgW5VSDeWeVAzIFMBqMEvwpnqzZICx4qxcDs3m0JHp9HO35UWaHDyjVrGsUK3oyq+sgoUWRA5iA5s\\\\\\\/ccufMozgxyarQUJOqL3KNbHPMMqC19F2B54GaE+uKT4ExYdlKL0lLL\\\\\\\/bwQMIVgNsv7rBIP3vhE6b8fTbGshkyIncQ7KL8BN+OvEVjLH+9hoPIQCl5RveA2V82AZbj2lxj+OwG7pRsPOVsUH5FfmeOZFUkGb587\\\\\\\/45X1ktKrPIecPm8Uab+VMjOaAO33dKO2R5VpNOnypSipYuGSVh4rZAJCaoXcgtwN9hnzLSJ6zyjFzKitGGSuBXZHAp8n2Q0TNYWAnUVSaEe+O+XHD8OjdTCgJ2pChXUhfb9rYcY3BPrd7ZZ\\\\\\\/iftl2SYPW3GxBrbvGu0jLt7VPYz0vSTQpMxPH4OtAZ3pW5Hf7aJahCw3yxaSwFl8IaRxVQiovrhTv+WmEIvxF9NfGSUXY7vTfQROFxN5B3q8e6a6xe+icep801QFwFpJ0XpkyRDrw11FQNMbkC23tjZlK88b8JdoB4DBd9sbr4IjcPSsaHUvXMzXX3k36JPIfs65Qx1\\\\\\\/4zGlYsAzNXghRPC0W4oWh1lw8SvF4DH+\\\\\\\/QuQc2JejvnMUgEIUf4Aaca6PiFszMrC9l\\\\\\\/+1Dk6CBC4s20jmdmOzLqVCqM\\\\\\\/YjGFbRrodUxqy5DBBAQHRQY8FGA4jY9r2FhNEtR3QvDUrQvgdwVLRfyFIvtNU42yZq3wDVe3jl6wGkDBv6ZqU1unaNnQCC5asjmWXRrSy8vVzOd+dNkJOKeejBzeZ2sh8kEBAUyauEtuBUyjpZ9L2sgQH79nWY8PSVuFqNBPozXoRrhugXf69bYVKzB6e5AhmIettz\\\\\\\/pmuCtdmndMQ9Odu7YCYabHgPoohoZaDuM+w\\\\\\\/cqFA\\\\\\\/DG8h973GpmdnLr1Z9PeVoKmbeyniyefr2u7kb5+wywc\\\\\\\/yxYUGwOUlsAp7Nd5Nyyfy9uyA6dEZfQQCVYE8dlsiFmvezgR4wZWPLg6ZtINgac9qArMspGLxuc1vSXzorOkiSj6\\\\\\\/2NEONv1+cok5+K31hvijtjY0f8nVPBrXtqxn6zzjJH8Lu\\\\\\\/slIOf0lPcqEDYsEzkdXk9qiYyftOvmhZzO9dbba60GC6Axzjg+Thi7P2HgnVIiUmYfUcYEYmHtAn\\\\\\\/lTAX7nK4j\\\\\\\/5kemcE5aj2cwc1hdmOMDYqP1ZJbCZRAymrZopyy0xT6RPenyR54F91K\\\\\\\/fc3XKQ\\\\\\\/fRyu70YT29u1Yddwk2YrGV6PB8phwxwq7PqXkLUFi8KoGsg4lwtQCBME81+TMmvDKKihsYQpMONPRk3ZDQWtobPiVozy2m2O\\\\\\\/ujxmcLMukEmN9oNV\\\\\\\/8yreOjTMMCIy2ckBSQGEBpCAitxfywwg7VUZ1YaaFR85L+57Yh3llPHZYTa0YyuvS384\\\\\\\/lSTrPvn1dtPNkxSSM48xN4gepNyKoDPm5dg8kXmqnBMsHxG0\\\\\\\/DtllW\\\\\\\/vHzLfgn3QqeIiML\\\\\\\/UVQBfwciE6nUjR5CKNHXz2EQ8FP7lthC2rN4dNoBNUzM5x\\\\\\\/yhR8NohUdMOsjEyaipDDgwvsUewEo4aQ40aNy5bapQXPbSRkSpyHDMWStKrBkY0ZAoSbAeu9Ax6kFY00nFx7+\\\\\\\/c+nd2ArbWb52OfqLV8T2YBtieQw45jeBBccgdlAFDSonmhVmof5OZOkSVi0afMU8Cwp0HZpTxLv5QM0aAWiGcgVEEbfTuIxG5\\\\\\\/9Hlacj2noHnKj8OChkWVyDFG8qOyw42rBpLVCqU9vih3X6zXB0aIlxdgmlT6gPUlOHMxlwipUX8pmrwQvvOw6v+V+Wme5Fqh\\\\\\\/wwEaQZ\\\\\\\/5\\\\\\\/NvOfKzqHUaqSMGD6npQveD80MW9J4tG7eFLSinEk0ZuNi3h3fEybcwd4ZJsFs06qCOtHaSMyVxlji\\\\\\\/Umo+cPqyfoNJyHqi6S3Zz6NFWr4UsiIHtAMMc1o6vS4oDa2bncEsUrB7U90TzBcOPthe+KgahjsM\\\\\\\/gnaucZGyBqXdRIB8bZ2dS9QxgmC9SWPFpMcEemov2lTITcTvmNkQGzLhJ\\\\\\\/0c50pTidKMqOEC9aFKRwe4mulJhQUmamfH4SK\\\\\\\/av+bIz8uarhccR4zfHIKLmLNuDd7YWA4t+ZOwMxzDzTAi0r+tnojXqQCwaK0Ly9ZrKHkDxhFJxYdQk4r44bAIcOcUGDInA1X9d5ZLDQRY49K3sDQt0YaWbTEeJ5+5gM5BZsvFhaS8pr0sbSvevHA4xTDqPjaJjvxFfIuulxPvivwEtbx41\\\\\\\/6AfDDgcxItIn90Yr\\\\\\\/aGpSjbrktadyqngmiRwaYkveUUWNmPHyoWWRKmT3TsTeli1YIzMfYovQeUW4nSN7azA\\\\\\\/S7RRJOL\\\\\\\/ftNRG6tY+TLrzNaAfMT7S1+DDRqiXEDhfdSJ30tuJedx9\\\\\\\/Ja3KghQkUqYCwPawXowmQzRvJ1179JXhSFf\\\\\\\/kXpG7OByOU\\\\\\\/lclAHFoFgUWgqFsQ34xFLi\\\\\\\/MZkaR6uh0XZwV7RVM2jvyd213aIu\\\\\\\/3ndsQQ4IvEv7yWgSXjHT9G25EY2haxkOSFP16BpiapLN+vnJTXYWxhUZvSmqm\\\\\\\/HW4BBBOOl3+4+U4JBpFfh\\\\\\\/SXTv+wyopRFDZUuTP\\\\\\\/hA2s9xscF8dxA0LA9l7+HWTazLjYnabQVszw+kBoU\\\\\\\/U7f4ZKz3PeZ2vSVgdRAYNAje\\\\\\\/NzbooE0cQgzUXxOHrC6frFA+8oD3ze2k44+Lc6kTLsyuCwg9AkmWZ1FIz0ZjpMZHOUKtjHNSGFs5vsX3oGSL\\\\\\\/1i\\\\\\\/LeiopBKHuCCNGJovZv1bHR7lSWNIXsF0BOZsQP4oDJmXrJ0loM5npykbtRqvvynOhvxSYKeDTnOgkpgnMSoXvuSc1eG1t0lBAZ1dNRwg9urcTTV3gmu+8BBH\\\\\\\/Axl4lPdQVb2HHWICYVfY7qal9p\\\\\\\/2Zx+GlXK1PUF\\\\\\\/lAxy7CRXS7lRlohD04dWhweFkbDCmiYTX\\\\\\\/QbFCvU0pi9RXNcCGuX0d6SbdUOpJg9ntLcjJQFwVZWysGu9FiMUSjcCLJ9G2U4FlX4FnrV+fD\\\\\\\/GS7xuNOARp1\\\\\\\/NaBW4fjhJT4VVsmu1gLqWls5LlXFMwgmbZA602mwRb4nRc0dVH7outp4NzOudGnhNBmF+lENfwCRJvIClkDuWtjsMWsRskeAJgWzwitLvNXU4N0OaxghJRfXfP5oTmtS8av9fsZ9iNOIsaEkLTEHie\\\\\\\/\\\\\\\/CRxRZ35fJmW+cc07RLhtiszM7pi4yGYeE4yiNRCTjtCwNLnodQyzX5QFDBMqRBdhlsQ8zNW2Z4d1AXsKVYl9GkxPT1Rs\\\\\\\/aeuIzXByagxsYddeDDThjsPnYrLJ4ts02S6FpASXbsZJtGVHYY4PnR0kYBflMCPeWdxYJZWmTPqL9po13pV6y6JvJiddNR5sZWHOUsODjO\\\\\\\/kc+yvoOxADhvWxp16pes9BhXLIW1cI0Ya1IuM9hG4etyMqeqDf3N7NKK90C57xQF50bu6kyasQYL7XI1ABMyxzg2bXHbAfkNlsI+RJA8XB1uesFht9\\\\\\\/FZkPShrtu9b1Zec2P+FDGQxUBRKLcKhSc2kPf5\\\\\\\/6ZovehnO32LxkyoLiEW1oEymNt\\\\\\\/8kMUrm9Jks7M27Cgz1mcvCNF2EXUmKsIAg3Sj1tkvCuiIcAE+asZ+4NHpZAhUHP9dLDGgtZJy8h0VCSWvfAroa\\\\\\\/mKGeyAVH6NQlvfl7ZmyDLxZ7TO\\\\\\\/d4V53mhj7GRyUWNIpjve1AGgufSRIilX4mt7jZIBvKwe5EydHScJZh1uwzoQ49NWLGB1opSuziDRmFe5dpCVocpZRROr85pW5CI61wDaCKRjZaekab1T76h6U1Z+v3DfLXq4qUxBh\\\\\\\/UXkRHErikVTUBvgH5x3Ue5TYzZa3UxRPeSBaiI+38TYDSdoV44ZMZavLVRC7\\\\\\\/6G2OAAVjtnX+UK+kbAR5BZKWZNbkVitT6+qHfpBzlj5+\\\\\\\/L2KGG0z8Mngt7L7XyaAqQGByuGTuYUUZ1VbQdHsjSr9HeRxj8BOEr5dPwqTiQwqo59KaFrr73PzPQcn5SCFe55CA8T0n7NZfpIXeUDlIieLv4xYZfevZaBQDEUh1\\\\\\\/kixI7X0SpMp0GRWgDM7t+8z7x3ibwNnjXJPbyQl1oAyvdVJ79jOhk974eF\\\\\\\/lGhg6C+fdmIpyvSZOEMx5Uw9GpsIKnNK+xqyxuo\\\\\\\/UFQIMcjL6ZXYCwXTqJ93u7081ujm5POuVbyN7ZCbx2EE6GPWZawtWWFK1MUmZsbQIuhhM0\\\\\\\/WS3f2xn3LuEcl\\\\\\\/VQrgZ3Oy6htcldHajntxGhbjKw7mTJdSqAp5syYniEp4CI5SZJ0DzR7WKB5rTqxUEJdISNGxuxA8Gs0H752V5sf9vCUZIiKfao2YYiiITEMJqsm8ILPGAXm2LyMOlFksUHoJ8sVRGYCI69eoyOTh+kI2mv\\\\\\\/vJO4yf3QvNjql5fYKvz8qDKeSkswjl5MWdmQ+A3Pad4qcjyn8uIOBN1ij6t+Ov4f\\\\\\\/D20NrJdle8fPHQYhRxH8ovTH3zzZkcOPX1FB6IPDBFfD2J2TV\\\\\\\/1qrD2hrPxhBgwtimIFAPIMajRKpSYiVkAo16T3Oy24Tqj326c3kyrSBo\\\\\\\/ctOBuJibY5nIHieQ5AnxIA1HBs88UDs2ZMGNIWjh0QVU01OPi1ib+WF5xkCqI7RdZekpX1neoS3ZLouZPO3OuWsLdjXkrwTH02Ssi9DG3srty1wG+V554TCfZsUIE80ZIxVOqG39vulUzSARqAb1xWPx8xZmn7TKBRL6OGTaWEfuL0mdhTz6zRwelzD4GB0kkMLMGkns3DSch7DtVoVhpzOfl+4oS6C9CPTRsT7yLJPvbC+3H\\\\\\\/lHwm9j8QmK71YjuDc3ucehf6NMyWI+b1GgMULwstV90Dogm8\\\\\\\/7fkmuwGMdQzPDIk8UV0mRa8u\\\\\\\/MJCXiIAnAbLFeqUvLx\\\\\\\/lJqzejrZh18a1QpMnlCK0V585HIvSWKc73VEL\\\\\\\/+jdRGuoqnFoapuedymUtgSXx03\\\\\\\/FfSnJiW6VAeJjBkOWecfdTFcathgbnqD+jhfDPasB\\\\\\\/XUYrUz2GFYdTrOhtF6Nr7bMDcgMndIrWyOUxcSDpK91tLPSGZDduUW7Ls934fmYX2N3USqqSLkW0k+nisBwv\\\\\\\/VDufFZYS0W4dYxg4uUrLyar+SYs1U1aGYG6cN3xDjOtwACQmP8\\\\\\\/j4cvp4SoTem4yrqye660tJ\\\\\\\/1nnMA+OZvhcgWAnWnPTmmdTWr4tXyK86ZVkvWih0a1QweK8AjMgUklU3UzS76Zvuj9yykWu7H2jFS4B65D24KA3YOHgtiSWl8EeLqSyJvFKl4h6sUkg0S11e\\\\\\\/6OeAL2mnmEimLOF+Ev3hxq61X4S5OWuw0N+A3\\\\\\\/R4vSPZ8tPQfrSxntxzhPf4\\\\\\\/xZP\\\\\\\/JTbNuaGQUgRbgTV0J+\\\\\\\/U2neGl5cbEkgf3wc+TbCU3pLS6jm1Y5NJlXAXNIxsFK4TAc1Cm6Bpjdac7BRaUAvEsOh0KUid\\\\\\\/OnzL0iEWnow5TTH1y2QKhpKgbBdx0tyTDjxkilka3zPc4iuwFHOgQrVwWVC3WLXJ6bnBXVYKznBPD13jZbA1MM\\\\\\\/ez3zaUMWCk6FSd39jpuVfom7sTdoJu4KHkVPi2IMZMUJ4XaaJiy+ZEUc4CdRCku0j70pEISSNV8EQ8EpfJgZQRXRHrDBKAcleaZRP2XB7OD07TROEkxFwhDq6HnRy9XKmvD8o+FbTuG5TxrR4ELjfojPBDRmOFv+1AIy4j9CwzLCmbaPpaeEyvsvWaQMOJDaD8WXKxNYQpoRAxJ4Nxsd0wvkDhuO4Nebfc8F5wBdDU7MT4UwltLSc+gNODfnlO4ecJBjY2OD+RUKVjMwCpbEVdnln9KQROjwASJYCotHK6QsYUrW03vpT3+eRg\\\\\\\/8XHla5HOm1E5Iik5KGMojDRp4f6q9bdpULPObRQ1dgGqWJBGeZWVpm3lYsXg9qiJjkI2\\\\\\\/7RQILpRL22lBwT77uVDUT0ql0JCzaArd99oP1\\\\\\\/5eoItQOxdqnP9B1GL560XV3ALA7xhmI0Fa1fiveIHD8bFLpDSsx033U0ol7VA5g5gQzVViYispHwL2k4glygJJWvNuBxxd2Y6\\\\\\\/gJ3E\\\\\\\/vYkYLpGS9\\\\\\\/Xdp2LeJYUAA4lBpcElicumchEKNd5kRCfmYKW5aDd6owj8j8XMGb4qcJmiqJ+0FKV0Tl2r8Ik7w2sv8RxwC3WccCu++jFtpMArorYVzoLc3nD63ZzXKe5XorreC6rEIzIN7X4tIFHn4oFc+0w+cULbl+Z+rf2LD6Q6m8nhBdRbUpQqIpKiPtJAIBaDt+248w8naqmnkbFRmlM+kQqG9Jbe6XKm3u16sNbOWuMQeHpcCsdyDj9\\\\\\\/6dSbxhqiKD0vxPoMAQ56uSzts9V04boUSFKRK\\\\\\\/Od+QrgPDe02cs2O3ttwarvy71fk3OIodl8b4AGoduzxzv+bg2xrcLWCCGc9u6JxC+CTn7Kk13zre43l2xtX22+R09uUQkOGdVJPDJGWk6zxqhIXngSXVQltR2E5R94LjqbUTqi74z4FwMwYbGYNom\\\\\\\/69vkvYkYQ5sAhsWO4bcHEIyOqEy1zxhwr2BIJBC6acZUQ1hPX\\\\\\\/+kpUxDZA63PdexhNY+t9qYqcRIGjTS6SIMEFo3lQCgtVmwymOLEKzu0SZ1rsKy653ZPqQzpQv85CS8\\\\\\\/56RCiTfjSM7i6fEau+zHrvzCQ+LO+Oi5xGz37cIdb\\\\\\\/HXY4Vjc+kr5YqrVefJxZAdYJJuM9B0DdEYh5cD9HPVlspo7BGMV3BdWNqdu5EUwlcv\\\\\\\/1VHwTmtW1W0UvPlN51oEmXeDXsDkwQOBXbiBK6vYPP7dAtNcQmjV28gNOEPJes5QZ8dj5Mje+4+1OWloypItUCBIFkrdWwyM9AB3Yo8Gi\\\\\\\/N7OqfOa41l9qpyEu8BEhKIe+LhgLkfu4t+J9BmxWjS3gvNRgDe9RxjQTZp4vY1dxKkinNaf2eCC8OMnfU1KDfZZI1cmBaGSMG4NpGA5\\\\\\\/cvF92AF8GwqFYx0d5c2m3+vStWh2dZBGyJ\\\\\\\/YQz2xw3tnaUC31n+OfGCe9k1y5aZPY1tTWkrJAIJrMRnjAHILuxK3p2dVDXE\\\\\\\/szycy2xHaPP6c2aXjysoaZVoJjv8V3BKqR5WpvPOl0EdEHzZN8OVRf6Fdl9w3pC5t49JOqmaW2D5yhWzjMSqyIJDtT+LHTsGNJ5b6Xpqc5\\\\\\\/pTEqbqxcf\\\\\\\/Ic0jYRgoed7oAX9YcyiABHJd0zaLPxYwlQ\\\\\\\/Hs1ApYcflCW+Mg6zwx5fjHLhEw4kvMjXeT3ftHJtKNVVc+U8oJ5GkTnZwcId2sjtICmzM8OI\\\\\\\/BaaVuZWxC2s2ZQCqiPn0a25yEf\\\\\\\/s0zIPCd5jScCctUzShnUIL0Eq3ZfoVQq5f0CLZPNMxCzq6UTq9Kd1Mll1xsEYg+Imbgwe7N78ji7idU1ukZuPZS75XWrWQYW6od9Pa78pTBlXnpgE+0hsm+YeEJRwcmN5PdI+8qpzOinqQDaWW3+W2fDjg7z3P7vREOduURvb5AOV+3ZQZIPMSDWo0UpldToi2KMnQmDIiLSxtjqye1yi0uyqECrTajjyZhETOFj0sMukfhT7lGR+k0eP1PHkhu7GwCiRvTON9QrY8rOXo2qgDiZKzbtel5Y9E9eeVsvRr\\\\\\\/AFjXiyrhg\\\\\\\/XBMAvROjEo2pzmHuTnMPBIhKJlJQejMs40qJ9kMUcfqUiXvBSoZ0jPf7jq88VZEsoKxZLDG9KC6kJH8M4Dj037qO1xH7yLjr5VVyDIHMLc3DpN1Aeo5ZWRJcLmwgKEaOsV5HtN\\\\\\\/5aHNMmhU7YmXoJ\\\\\\\/klMCPOeKt2yuf8zwVf0nFquS1RO8umD+NaCB8MVHXvOVMaGMYinS1ZudrT6GWpYdZ527mKFxnN9sBe3PiE22grN848qC6qvlL6qxr8untSMWakyTgQaLwalrfbbwYVe6yRaIYwL3j8AWrpNu5ltGCOQxAmRN9O1xA0lPxKLTLQKInaUGVNoZhQgKS1oy4uHROhjMWJl8XcYahpkACxUBHDKiPhqq5u0G8I\\\\\\\/MR48fLxnHkAKDRJB52h2rGw8goHnpqdT7vJ+1LD6R9Jnq9Xv\\\\\\\/FL\\\\\\\/S2WFm0NWxnQei8XA9pETOYOidisF4NNo335R\\\\\\\/yOQiyhCso71fWD97LHfx3pd8loritGsd23snxvREAx5CvD21EVgjJJbYX\\\\\\\/tStum9X+MrMCr\\\\\\\/nV+P4VzOtG2VhbECo2MA3fuyhpVWtiXBnm3TnFW2PUyyprpD\\\\\\\/z+d+Lk6ZcCHcI7MEOolu9r05MMB9SzerRn4gTaBQn9XMjqtdzplZEpxJpy5peAE0iLm5Ydd73CCIpSIZHfQPtn9zie11+smoRCILd4PTutBiHOzVKNz1bLiN+\\\\\\\/uiNJjsxAJE1MlUfU4enhzEw7fLqYzssg7OE\\\\\\\/8zNzlFeBiJ0tISgMGkvV0lnnv4zKKI3ByVptcffUQYilr5sdwR4hD00pP1rk0dcYenO3PcAsadDrd\\\\\\\/oda017ynPE4D30a6CmdNaI6vV57XRT1HClsHXMgfPShJaLy0Y+wjI7Ll4DNdVQOCoa5f51fCIuqwUXZZ3C0wYWdiXfhECwZyTz5igLheQDBbhK4K+ZwtwsOkKWlC\\\\\\\/+75J0VRDCM5JYADtu2C5Lf4xo7gWIHLR\\\\\\\/h25BGSXbGWbSnfgb\\\\\\\/bU4Jw0F5JqKSuz6WuJaTymdTy85TvgFLL2rx2sm3uNo5IfJdQWn0pWmEew1CaZbKa3VFMHj2eQqySFFA5Xz98cBS8XJ1HK5o+J5IRUrJ6IXfliRbo22rnWzGxC6HSEgy7D2zoGyXY71BTgOP63\\\\\\\/blKCDLChWLm2E\\\\\\\/Wrj2n\\\\\\\/g5VB5bRK5zCctWJM0z3AK6ROtAuFHVqyT4MH7Uxd2c0AOUbwrpVp2rHov8\\\\\\\/EdmR18oMDnsgIkD\\\\\\\/L6Eku4ZvUo5n5e\\\\\\\/h7+sHelNwh2KR1u4LNtonXqGcH+ssUA+9NRFtJPpFgH3Uyc4zuChOjMC\\\\\\\/2rVA\\\\\\\/TG+CnpTfH1GOxETn4tsceTO4bJndQBe8q+oqxeXgU8LLUF4PUJ3rYi2w9cvz5+o725fqLT4yhaqusg+JJLUu9mfAypqWrsG2gBA59KxqnBLvX4Tdnk9oYqqFQ72z+oq98FYIZF2ttS1b1LME\\\\\\\/ds6u8EacH2\\\\\\\/nDLcJtBNu8Npu78+nfThPccOMhRZ1RPJuYCUyponQUgpEy0xE03L1Jp0D7QlmCi0etjX5BzZAE4kzfjUpZPDmUoszVhIfFWH0pqKjnKtjinpuWm9XJyaE8IlYxin5w0DnWmt4\\\\\\\/h5aM+F++Y5Lr\\\\\\\/oFZ3BKjGtlY+GDG9eQSt864cJW\\\\\\\/T3N882keUzQDo+NxHYa\\\\\\\/3IbzZe97b5cAJZWCgqDfST\\\\\\\/m50wPljKEeIvv9TTrWFXeS24+8AD1KlnRSiwM5RNWf+7XSKHK0JJ549qxfFRNzP1PZLYlb4tghPTMtw\\\\\\\/ad5a6Mf2TzPn\\\\\\\/aDztu9raOqLPvVL8hxElE5WMTfcYPfvDPSC3JHRYb0Bdh5Ry8V1n+paGEXKjatce5lw+jyZAXhsywEZdK9JVGA4pt+8dUER\\\\\\\/AZ5Q10cw+mzKz1DiYomH3UTM16c19ajefrVJeugkXiazEsp21g\\\\\\\/IsNNBRgIt0aU943tXqF7bLkiLFpSKW2EPNhrxnDusVoCvWtCKYA0QmAiaVS6875qJyBchZtkcOs1\\\\\\\/hbci\\\\\\\/DuZL\\\\\\\/XM+FlGjXDXEsrHR4\\\\\\\/E672\\\\\\\/IbehMa4+HcQWOSTpbSCeg06m17bGnfOX4SIE6mLAj8vyVLiUpqKTUU+4rvNSOSDl\\\\\\\/OntU\\\\\\\/wgpqU0FruDYMhL+oGIUz3zE5TzSGZ4yg91rfZSqweeMVyBr5gCPdcs9iKH4NdCcnxD86gIXOqIHBdB3ulw8mkwiZg0\\\\\\\/S8i+O3rQD5gynXh3Ep9kJiK29zoXbnlsQrLB3JFNBIlXASiA8NQS0cLGj1QROgi7DYd3P6wpLp0X6qGwGigjmpzyixrlTf72Ds0Bb0NbiZ475p2mamHniCzNYOgpU1\\\\\\\/YJTwESEu9QH2JFgJFurziQ2zXcNyS9\\\\\\\/GZA8Q3PeUVyvHcEHu8xFniO8O6B7YT639+qPHYjJWu5cx1pL7VSsD4FXpIf6HUqlu1wVLik1YiubgL\\\\\\\/9TdlJsEkN5wGSTlPCNrCs8ER69g+rWi6mm\\\\\\\/wU4UqWX+xuLIx7tgL5nu1DYrnMMjK9jaXlqLe5bMkXU6VExn+Bv7ZVFnOUvZutTVxwX10WNqK6siWuDVSCnFTKkxSR0rB0P1DhS9ASuOwA1ACqIQQBQPTle4T\\\\\\\/B0YCSEPJep5unWhoR8eieZrA7\\\\\\\/nh1l4oXCXP2rtjdnzfdqqcfstpwiHNX+GvoanZpBWxuczfj4eInQBVVBdX6Sl56YTLDbarI00vGxJ41dFdUaRbRScJzWi4XO5QVTG6UhSgQpz4Wp9vDCk\\\\\\\/bch2ulszt+BMF7Rkuv6uXubJA1VPytDYNX8w9UByPRySMO2E7ahhWE\\\\\\\/UQFsW6BcLkSmmMF3E8OJ0Ol7gYfcbSgkvnN0WS+\\\\\\\/WMQw\\\\\\\/B+v0FFcg6g0n7fH5yn0F8bMrWDzu5earPPnzg+zkgm\\\\\\\/ToBMkvF1D2NL2Qb1lkYHCfqtTINIa4tfvK+UOZltGRVj\\\\\\\/YSc5nxGY6GSsmbt1hcmhh4zGX+8AMUStjNoXAX0s4zz453c9sWILLkGbSVdGS\\\\\\\/xgcgFpZK5HnHOG2wVXX7d5gmqsav5WYMKCSl4+zGfNAqkBaKYu+a9sUv2IVjNQVYzB6fvKQ6bBIzslGGrnrmnO8e8ELH2pPBNlpd\\\\\\\/vqneG31A2ajd4r3jLBAwKDMDWaGlN4\\\\\\\/9U4AbNw0RjTEsZ0t\\\\\\\/ImefLYpEdKtVNPkxtLWsNi3VUe4fmw+wfccpRMjxMPxoyhH7utdPzg0d3FRbnPSgYbK0hsR6oBvqpVD3j1tSomjCubMwNrnPEL9SSSrq\\\\\\\/XFvj17MN+OP0saGdLPJBta24iCzTeMN0q1MSCEmWoG4BWlE6bCGSwZFsWgNaMqCZdC+ARjXBHw\\\\\\\/dW0bhUQ0VUyda9J4Xz1GJB2kU8ZvfdyMLoc4Ve1sUOBjDsi3gD8ShAHyWJAlD59rCu2bdRvtqUTln037Gbc4TetKe1O6XlRw+CPzC5Cqg82mvZ1eaAoTbBP20TL9SXg26cOwf8JWONPXrs0Eroyv4P5Sb2HlCMxnHlGq2Dr\\\\\\\/VqWIPH\\\\\\\/IYSC1GGQn4i6D7PEMwC\\\\\\\/7+jV4Urqak9gr6Gn9cE7FonbPU5nX+5OrAWQgIzGPNr8j2+Gp2nqAWXld2ok\\\\\\\/J9ADZ+R\\\\\\\/W7m\\\\\\\/u8XnymhGt8S6ym51aoQxZmB\\\\\\\/wt\\\\\\\/DTxgW1Q8KZHbHJ+PFldh3UrsXWDzR6zXVPvAXYzmCSDhYGboMJKKoXmhfTZFUm7Dxs2R0lG\\\\\\\/2NaJf7NZ\\\\\\\/u0ENcFgvlkRHDrslkjUxyajy4Pe0nG3ZySvzSig1305XHrzqL62\\\\\\\/h1T5xCsMm7uOdBRnz9\\\\\\\/RXKuAkDWbNYV+BbNty48EPdipYgsNLd6vmEyBRb1SAZqVnYKVCMLHrCoxf6TFyOsgl0+QXJjNJfj9wgsHw7f5syONmERITt3HABK3B+5M67hA0SrESQtZoymoiDtouZ99T7WreCgbQxrTdMvaz6bNJXUHIymsSNf46Sh6DmpSqSZonRISqX9e8kHDwkprmu5CV5h7zggEcyfp6MlZIdewGKMGBS2wnpTwPfZvaPUeUGEZh5WSZjI6rRPwr5ELlhvrepJy74cun57\\\\\\\/wn5586LsF46VoPntpAC3nyOXEOuf0srGLA9FBgGhuAnTGl71tGSPBJFRFdNmpk6s29Zs8fmScWnN6Azvrf8dIz11k2h4+0OIk+KZ2uFMdgJPyoo8UyPe8EbHR12rNg96NZL34WGpZXONxRuHvbtYgTEleb8jQPMpTBTKT8eWptJEqNP68CGr\\\\\\\/xm5mx2Zz8e52N5ROeTFETqE6\\\\\\\/+fMl3GLlz6v4OWCH9gWxPeMwbxX7llSvtCHogmuXsMPUwJgbxHqGXOK+98fEczqfSIzILBDLqHQSERxnnAIzGC03g3V\\\\\\\/HyUtj0frUBELyjskl9lIAJTRQfIfyCXUG4XvgZwYb5aIGoKfmgKBUflyXqa88LQ8mnRwPP5adDgZ7ejrkCVLxYv6OD878KXRJFIE8HXxyANMg9C5umXxDZ8XGKw4fvgaNxpSuY9vqQ\\\\\\\/cQdm2xRJCu2oRMeTqjIecAGqp6iQn5XsY1fr5WD4YtWsNw0Wjt5fNXZTtufDjtXFojBCcwtExgbLMaNxo\\\\\\\/s1IDbigUmYRnRKU+lLr9OV8cLNpnpvul6Z5jyGM46QNuiuifAEQfYp3+D6ovgLawzN3AaxXPNkHmWXL4NpyX1ub1wk7K964d7Z14jBS4mCdIVm7eM\\\\\\\/7WMyj\\\\\\\/bUWP5epUN6chsBxkfdnFApWL2gonReNHKN0RiyhHCPYqySfNScTzGweGdfyJT5YWxFxlgXVOYk8wm1o04Aty8rnXsKileoMkRA28pgfRAoaTd7riiibapLQwXlXXhG2NVwBL5eeEwvo5IFODSLddlhMMSGkPlo5ZTbjq6WScjkuwcWWCAbXE9GYrUCvTOE1rSvwcgWhGXmiIN6XJVK87z1dLM+FdCiM+ukIKXfH+fnj3f7Qx3DUtS+l3nm80EtA2HerK5iV0KoivNsHemR\\\\\\\/NUkvS4LgRwbWJ+HOpzClZNRyGugFAm9lcfOZwYodEuOvfKre+kieGA4K3uHmAgLHtW10El5LN8FEPcWqKip7B2Pe4eXcmRAXLosZrfeExucz1nMbW1+SZnm9cIuIv01rgk9cuDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022A1ohAJgydOtEJldFB4waQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002262021815d8f73227d054bfc7024a329e0238f68dd30149c25eaf568c52888baa\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224ymLdagF0ISPphaIkoe+UiQk8TeAV6Em\\\\\\\/nq2QqnilBaPkUNdrRfg4Zc\\\\\\\/1W2eJwz\\\\\\\/ccTWL0MPqWvEP2CBZekWLTcAPhCvAUYwfqMUHWMxBgncMXHBzXakyt4kTG3a74bvGf0ngQ5dA1h34WMnncNbFTLiR0RATNn1e4+yHF1MEzEp+q1RAF4RxsQMJb2t5SZsKLYl2RGn1D4CohZ9eiihpe3jvsk\\\\\\\/hwxfpklj\\\\\\\/M1WfapfltbgRGvV9A2sJ65VJlOK\\\\\\\/sP7PHeNo7PUVru\\\\\\\/fjaEe9xj1RFhvTGw9wytKqR38GwLA\\\\\\\/1d0ag91Ru1BgVKYYk4HnnibX4wxGiDI1UMDUSJLLprsfp34GES4ybv2ERyqkf8jNjcBWa3wwugpciyyRx\\\\\\\/ZPielLkIR+dC\\\\\\\/dJMWwtO\\\\\\\/c7nsS7DTGQTpBDhhMRKdwtM9yedRxMr5lW47D0QQotlVniWraBXykYsnJo+p3gsFqKekEymDE\\\\\\\/CR5TuPXUgr4IrjNI8UMBUZxfaEPbOYmySUcF1HGc3QDUINhrfNHxQRrdKcWDW2zPHqgcD6Fnz8M12lP6umLRO2tFepPgnZZP18Rj+YlDDWMN9aXrA1kIoFEFw\\\\\\\/dHJJOKADmqp1fFh2BacNRYsuFqW1u4qUDa0H3NMzx4NC3PwHOf0SSfdcoroc1q9LQIkZrD\\\\\\\/kfw2K33kP6TI+8msiSCNk+Ljm7Y\\\\\\\/6H+iUSr9voTIlrp\\\\\\\/eau7q6pDYyLyDNNtDP8ZefTIsYlDv\\\\\\\/Jy6VcT6k4xv7uUOwrIi+NCfRCmlUsoDeQ2D004Qi6Tt5bBLiTDkREsTvU+XBOS0d+B77vXuw7Gpkz6qJ36KyAVmE\\\\\\\/nmpigiOI9Hbfw2RQ\\\\\\\/NfGoleyZnhNLx4Go2SiCOIRgpzME\\\\\\\/BeHnpfzV99KQsBYAl+4ZX93ekVRdcGSGPsKGiDchb+C15tSNQw1epk9IWtIlpvurAOOcN8kJA560t2Z1hGbbPw4juPUC1RNl7O3JjBJGjpt56UkyNuBIabLWDVhwupnlSo00etKCBYz4qDuYnwPx7ZcjbSl1VqXexzoJylqnzqgMLQG8fBq3G\\\\\\\/VI1RBR8FoCX2NId4qbz6v0kzGTLaODJQHAESHVQV1Hve8iRNJFCNnRdrCJqzPJ3b1JNdUzREyh4jw5mjvuaBdShXuMrhrkrvIp07vC2o7HB5AJ3JRn4dN6o\\\\\\\/fvgsfqjNjPgp34QzpD+Sv0Qz0pkv+W5LkK4fSm9LFgKCr2EyRNrstyYwpUEsP5eOYD1WIxa1J6PhbwLhYITM6jY8sl3U0c5KaqDfD+wLgmdBhEK2mhCXGE0Pq2Dq3PKMKxlQEMd6\\\\\\\/TOlfWkkD4ULFI5ZiiFOXntFVbrtFXd5q8sPZgJWNb\\\\\\\/CA+lTObkMbImzyHSU+5UxM2N7C\\\\\\\/gDT+tVOwGpLt2s5o+dkL0Yuv6w56liZdWaVFmFcMbKfN8OxY1pj07ewsFAb7jVt+735oux8X9rKuBzFMj1hxsNdtPIf7csg2RaVrzu77o4rWNLnL7WNtDuWQU7I1O9\\\\\\\/A76W26TzgX1vtDP4mALqidpOJIiFvZcFahYzPvTGniLyST8\\\\\\\/BjsgPY0BBNkMU\\\\\\\/NCcuTbZ3LaKqm9Jv6vbv+TcWXcRFdiW+CJGXDf75in7D\\\\\\\/4+kbSdYSZ0p4qBbjordMQ9xekMjtcmuk69cPbrfHAR4w++Sjn2XRQyh4MWvtfT+qQphEpRz5b6rraQSBPWJalMjvRNQFwt6tSjJbmSp2srU9GrJnojka9yRPBFz8c+O4wEJGR8q+U2faZ4yZW9eNWhCXNenNuuRKdezvDZjt1EVQhoHVduRg0tnubcZZvxrmsCvVXN2FlT1ugoaHBSnHS\\\\\\\/XuHuROqNoJq2RK\\\\\\\/LG3qSGJUhvOD5l3+3WyHDjNexKizQVthf1ALJxSARo2bC6\\\\\\\/\\\\\\\/Baf3saU3UTZaubfP6pbqhrriU81zEtkrZGa9I4xqX3zpFrRRd3kabkOizAMPU3DGfpDx4d0PItwqf0evoF4MoaLqjZjj8Cy4J4Vy\\\\\\\/sK06w==\\\u0022,\\\u0022iv\\\u0022:\\\u00228TLQ\\\\\\\/nbQghil49ErJFPo1A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220207a93899e28e1716823f9e203fd78ce51a1c49e9bb5c62862cbffd4065b372\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492490 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genetics-of-sex">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/955" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ALzxR66yh9wGwYfLf80LRdNImHOC_8kU67JZYU3_eAo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/955" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kryjL8XkyCLYJl4fsPglhdtL97V_j-nSxuDiyRgfCx0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/943" title="Arginine to Glutamine Variant in Olfactomedin Like 3 (OLFML3) Is a Candidate for Severe Goniodysgenesis and Glaucoma in the Border Collie Dog Breed" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/969" title="Environmental and Evolutionary Drivers of the Modular Gene Regulatory Network Underlying Phenotypic Plasticity for Stress Resistance in the Nematode Caenorhabditis remanei" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492490" id="node492490" data-pisa="ggg;9/3/955" data-pisa-master="ggg;g3.118.200910" data-apath="/ggg/9/3/955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr <em>Salvelinus fontinalis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ben J. G. Sutherland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jenni M. Prokkola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Céline Audet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Bernatchez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">955-968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200910 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492490" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ben J. G. Sutherland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ben%2BJ.%2BG.%2BSutherland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sutherland%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABen%2BJ.%2BG.%2BSutherland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ben J. G. Sutherland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ben.sutherland.1@ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">ben.sutherland.1@ulaval.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenni M. Prokkola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Integrative Biology, University of Liverpool, L69 7ZB Liverpool, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenni%2BM.%2BProkkola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prokkola%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenni%2BM.%2BProkkola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jenni M. Prokkola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Céline Audet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut des Sciences de la Mer de Rimouski, Université du Québec à Rimouski, Rimouski, QC G5L 3A1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BAudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Audet%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9line%2BAudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Céline Audet</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Louis Bernatchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BBernatchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernatchez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BBernatchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Bernatchez</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/955" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492490" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492490/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/955.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492490" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492490/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/955.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492490" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492490/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/955.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492490" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492490/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/955.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492490" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492490/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Networks of co-expressed genes produce complex phenotypes associated with functional novelty. Sex differences in gene expression levels or in the structure of gene co-expression networks can cause sexual dimorphism and may resolve sexually antagonistic selection. Here we used RNA-sequencing in the salmonid Brook Charr <em>Salvelinus fontinalis</em> to characterize sex-specific co-expression networks in the liver of 47 female and 53 male offspring. In both networks, modules were characterized for functional enrichment, hub gene identification, and associations with 15 growth, reproduction, and stress-related phenotypes. Modules were then evaluated for preservation in the opposite sex, and in the congener Arctic Charr <em>Salvelinus alpinus</em>. Overall, more transcripts were assigned to a module in the female network than in the male network, which coincided with higher inter-individual gene expression and phenotype variation in the females. Most modules were preserved between sexes and species, including those involved in conserved cellular processes (<em>e.g.</em>, translation, immune pathways). However, two sex-specific male modules were identified, and these may contribute to sexual dimorphism. To compare with the network analysis, differentially expressed transcripts were identified between the sexes, revealing a total of 16% of expressed transcripts as sex-biased. For both sexes, there was no overrepresentation of sex-biased genes or sex-specific modules on the putative sex chromosome. Sex-biased transcripts were also not overrepresented in sex-specific modules, and in fact highly male-biased transcripts were enriched in preserved modules. Comparative network analysis and differential expression analyses identified different aspects of sex differences in gene expression, and both provided new insights on the genes underlying sexual dimorphism in the salmonid Brook Charr.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/co-expression" class="hw-term hw-article-keyword hw-article-keyword-co-expression" rel="nofollow">Co-expression</a></li><li class="kwd"><a href="/keyword/salmonid" class="hw-term hw-article-keyword hw-article-keyword-salmonid" rel="nofollow">Salmonid</a></li><li class="kwd"><a href="/keyword/sex-bias-0" class="hw-term hw-article-keyword hw-article-keyword-sex-bias" rel="nofollow">Sex-bias</a></li><li class="kwd"><a href="/keyword/sexual-dimorphism" class="hw-term hw-article-keyword hw-article-keyword-sexual-dimorphism" rel="nofollow">Sexual Dimorphism</a></li><li class="kwd"><a href="/keyword/transcriptomics" class="hw-term hw-article-keyword hw-article-keyword-transcriptomics" rel="nofollow">Transcriptomics</a></li><li class="kwd"><a href="/keyword/weighted-gene-co-expression-network-analysis-wgcna-0" class="hw-term hw-article-keyword hw-article-keyword-weighted-gene-co-expression-network-analysis-wgcna" rel="nofollow">Weighted Gene Co-expression Network Analysis (WGCNA),</a></li><li class="kwd"><a href="/keyword/genetics-sex" class="hw-term hw-article-keyword hw-article-keyword-genetics-of-sex" rel="nofollow">Genetics of Sex</a></li></ul><p id="p-4">Understanding how sex bias in gene expression contributes to sexually dimorphic phenotypes, and how this affects fitness is an important area of study to understand genotype-phenotype interactions (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Parsch and Ellegren 2013</a>). The development of sex differences in phenotypes can be attributed to differences in the expression of both sex-specific and autosomal genes, caused by hormonal and epigenetic effects (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ellegren and Parsch 2007</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wijchers and Festenstein 2011</a>), heterogamy and imperfect dosage compensation (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Parsch and Ellegren 2013</a>), epistatic interactions, and transcriptional network structural differences (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). Sex-biased gene expression is pervasive in many species and may alleviate sexually antagonistic selection, <em>i.e.</em>, the selection for different phenotypes in the different sexes (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wright <em>et al.</em> 2018</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rowe <em>et al.</em> 2018</a>). Transcriptome regulatory architecture differences between the sexes may contribute to the development of sex-biased gene expression (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Wright <em>et al.</em> 2018</a>). Sex-biased expression underlies much of phenotypic sexual dimorphism, which can occur without sexually antagonistic selection (<em>e.g.</em>, imperfect dosage compensation; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Parsch and Ellegren 2013</a>). To connect alleles to phenotypes, typically association studies are applied (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mackay 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bush and Moore 2012</a>), but these bypass important intermediate regulatory steps such as transcriptome regulation (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mackay <em>et al.</em> 2009</a>). To characterize transcriptome regulation, it is important to consider the underlying structure of the network in which genes are co-regulated (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mähler <em>et al.</em> 2017</a>).</p><p id="p-5">Constructing gene co-expression networks is often based on correlating transcript abundance across samples (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Langfelder and Horvath 2008</a>). A network is comprised of modules, each of which is comprised of a group of genes with correlated expression patterns. Co-expression clustering is a valuable approach to classify and visualize transcriptomic data (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eisen <em>et al.</em> 1998</a>). Clustering often groups genes together that have similar cellular functions and regulatory pathways (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Eisen <em>et al.</em> 1998</a>), although this is not always the case (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gillis and Pavlidis 2012</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">van Dam <em>et al.</em> 2017</a>). Module functions can be predicted based on phenotypic correlations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Filteau <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rose <em>et al.</em> 2015</a>) or functional enrichment analysis (<em>e.g.</em>, Gene Ontology). Genes that are highly connected and central to a module (<em>i.e.</em>, hub genes) may be upstream regulators of the module, and potentially more related to the module function (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">van Dam <em>et al.</em> 2017</a>). Hub genes may also be under higher selective constraint than other less connected genes, and as a result may show lower genetic variation and higher phylogenetic conservation (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mähler <em>et al.</em> 2017</a>). Network information thus provides novel insight into both gene activity and evolution.</p><p id="p-6">Comparative network analysis across species can advance the understanding of species-specific innovations. For example, comparing brain transcriptome networks between chimpanzee <em>Pan troglodytes</em> and human <em>Homo sapiens</em> indicates low preservation of modules found in specific brain regions associated with human evolution such as the cerebral cortex (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Oldham <em>et al.</em> 2006</a>). Comparing networks can highlight potential drivers of phenotypic changes associated with adaptive divergence that can lead to ecological speciation (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Filteau <em>et al.</em> 2013</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Thompson <em>et al.</em> 2015</a>). Cross-species network comparisons have also been used to detect gene modules associated with disease (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mueller <em>et al.</em> 2017</a>) or with seasonal phenotypic changes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheviron and Swanson 2017</a>). These insights are often not possible to obtain through standard gene-by-gene differential expression analysis, which captures a smaller proportion of the variation than differential co-expression analysis (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Oldham <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gaiteri <em>et al.</em> 2014</a>).</p><p id="p-7">Network comparisons can also provide insight on sex differences (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). Differing structure of networks between the sexes may resolve sexual antagonism through gene regulation (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). Other genetic architecture solutions to this conflict may include sex-dependent dominance (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barson <em>et al.</em> 2015</a>) or maintaining alleles associated with sexual antagonism on sex chromosomes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blackmon and Brandvain 2017</a>). Comparisons between the sexes are complicated by the fact that sex bias in networks can be tissue-specific, with modules more preserved between sexes in brain and muscle networks than in liver or adipose tissue (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">van Nas <em>et al.</em> 2009</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wong <em>et al.</em> 2014</a>). Liver tissue is considered a highly sexually dimorphic tissue, particularly in oviparous species at a reproductive stage (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Qiao <em>et al.</em> 2016</a>). Although the extent of differences may depend on the tissue of study, network comparisons between sexes can provide new insight into the regulatory underpinnings of sexual dimorphism and antagonism.</p><p id="p-8">Salmonids are an important species of commercial, ecological, and cultural value. This species group is also a model system for studying genome evolution after a whole genome duplication event that occurred approx. 80-100 million years ago (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf and Thorgaard 1984</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crête-Lafrenière <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Macqueen and Johnston 2014</a>). Charr (<em>Salvelinus</em> spp.) are a phenotypically diverse group of salmonids that are less characterized in terms of transcriptomic and genomic data than other genera (<em>e.g.</em>, <em>Salmo</em> or <em>Oncorhynchus</em>; but see <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carruthers <em>et al.</em> 2018</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Christensen <em>et al.</em> 2018</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guðbrandsson <em>et al.</em> 2018</a>). Brook Charr <em>S</em>. <em>fontinalis</em> is a primarily freshwater species native to Eastern North America. Arctic Charr <em>S. alpinus</em> has a circumpolar distribution mainly in the Arctic, and these two lineages diverged approximately 10 million years ago (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Horreo 2017</a>). Sexual dimorphism in body size and secondary sexual characteristics is associated with reproductive success in Brook Charr and other salmonids (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Quinn and Foote 1994</a>). The largest males are typically dominant and fertilize the majority of a brood (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blanchfield <em>et al.</em> 2003</a>), but smaller sneaker males can also contribute in fertilization. In females however, large body size is more constrained as it is highly associated to fecundity, and smaller life history variants are not expected (reviewed by <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fleming 1998</a>). With different optimal reproductive-associated phenotypes between the sexes, this could give rise to sexual dimorphism as well as sexual antagonism.</p><p id="p-9">Here we profile liver transcriptomes of 100 Brook Charr offspring from a single family by RNA-sequencing to characterize co-expression patterns. Transcriptome profiling was conducted shortly (3 h) after the application of an acute handling stressor to all individuals during the reproductive season, increasing variance among individuals. The goals of this study are to i) characterize the sex-specific or preserved modular structure of gene co-expression in liver tissue in Brook Charr; ii) characterize module preservation in the congener Arctic Charr to investigate evolutionary conservation of the networks; iii) connect phenotype and functional category associations to the identified modules; and iv) integrate results from the network analyses with a gene-by-gene differential expression analysis to determine whether the two methods provide different insights on sexual dimorphism in transcriptome architecture.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Animals and sample collection</h3><p id="p-10">Brook Charr used in this study were originally used to construct a low-density genetic map for reproductive (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sauvage <em>et al.</em> 2012a</a>), growth and stress response QTL analyses (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sauvage <em>et al.</em> 2012b</a>). They were used to construct a high-density genetic map that was integrated with the other salmonids (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sutherland <em>et al.</em> 2016</a>), then used to identify QTL, sex-specific recombination rates, and the Brook Charr sex chromosome (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sutherland <em>et al.</em> 2017</a>). The 192 F<sub>2</sub> individuals were full sibs from a single family resulting from a cross of an F<sub>1</sub> female and F<sub>1</sub> male that were from an F<sub>0</sub> female from a wild anadromous population (Laval River, near Forestville, Québec) and an F<sub>0</sub> male from a domestic population (Québec aquaculture over 100 years).</p><p id="p-11">F<sub>2</sub> offspring were raised until 65-80 g and then 21 phenotypes were collected along with several repeat measurements to determine growth rate. Full details on these phenotypes are previously described (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Sauvage <em>et al.</em> 2012a</a>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">2012b</a>), including sex-specific phenotype averages, standard deviations, and phenotype correlations for all 192 offspring (see Table S1 and Figure S2 from <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sutherland <em>et al.</em> 2017</a>). The 15 phenotypes used in the present study to correlate with co-expression modules were maturity, length, weight, growth rate, condition factor, liver weight, post-stress cortisol, osmolality and chloride, change in cortisol, osmolality and chloride between one week before and three hours after an acute handling stress, egg diameter, sperm concentration, and sperm diameter. Fish were anesthetized with 3-aminobenzoic acid ethyl ester and killed by decapitation as per regulations of Canadian Council of Animal Protection recommendations approved by the University Animal Care Committee, as previously reported (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Sauvage <em>et al.</em> 2012a</a>). A total of 87% of the 47 females and 96% of the 53 males used for transcriptome profiling were in a reproductive state at the time of the dissection, which was in the fall (smolting occurs in this strain in the spring; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boula <em>et al.</em> 2002</a>). Phenotypic sex was determined by gonad inspection (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Sauvage <em>et al.</em> 2012b</a>). Immediately after decapitation, liver tissue was excised, flash frozen then kept at -80° until RNA extraction.</p></div><div id="sec-3" class="subsection"><h3>RNA extraction and library preparation</h3><p id="p-12">A total of 100 of the 192 individuals were used for liver transcriptome profiling. Prior to extraction, samples were assigned random order to reduce batch effects on any specific group of samples. Total RNA was extracted from equal sized pieces of liver tissue from approximately the same location on the liver for all samples (0.4 × 0.2 × 0.2 cm; ∼1 mg). This piece was rapidly immersed in 1 ml TRIzol (Invitrogen), then placed on dry ice until all samples per batch were prepared (6-12 per extraction round). When all samples were ready, the samples immersed in frozen TRIzol were allowed to slightly thaw for approximately 1 min until beads within the vials were able to move, then the samples were homogenized for 3 min at 20 hz, rotated 180°, and homogenized again for 3 min at 20 hz on a MixerMill (Retsch). The homogenate was centrifuged at 12,000 × g for 10 min at 4°. The supernatant was transferred to a new 2 ml tube and incubated for 5 min at room temperature. Chloroform (200 μl) was added to the tube, the tube was shaken vigorously for 15 s and incubated 3 min at room temperature, then centrifuged at 12,000 × g for 15 min at 4°. Finally the aqueous layer was carefully transferred to a new centrifuge tube, and into an RNeasy spin column (QIAGEN), as per manufacturer’s instructions with the optional on-column DNase treatment. All samples were quality checked using a BioAnalyzer (Agilent), where all samples had RIN ≥ 8.3 (mean = 9.5), and were quantified using spectrophotometry on a Nanodrop-2000 (Thermo Scientific).</p><p id="p-13">Libraries were prepared using 1 μg of total RNA in the randomized order using TruSeq RNA Sample Prep Kit v2 (Illumina) to generate cDNA as per manufacturer’s instructions using adapters from both Box A and Box B, AMPure XP beads (Agencourt) and a magnetic plate in batches of 8-16 samples per batch. Fragmentation times of 2, 4 and 6 min were tested for optimal size fragmentation and consistency, and as a result of this test, all samples were processed using a 6 min fragmentation time. PCR amplification to enrich cDNA was performed using 15 cycles, as per manufacturer’s instructions. All libraries were quantified using Quant-iT PicoGreen (ThermoFisher) and quality-checked using the BioAnalyzer on High Sensitivity chips (Agilent) for consistent size profiles. Once all samples were confirmed to be high quality and of approximately the same insert size, eight individually tagged samples were pooled in equimolar quantities (80 ng per sample) and sent to McGill Sequencing Center for 100 bp single-end sequencing on a HiSeq2000 (Illumina; total = 13 lanes). Parents (F<sub>1</sub> individuals) were sequenced in duplicate in two separate lanes.</p></div><div id="sec-4" class="subsection"><h3>RNA-seq mapping</h3><p id="p-14">Quality trimming and adapter removal was performed using Trimmomatic (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>), removing adapters with the <em>-illuminaclip</em> (2:30:10) option and removing low quality reads (&lt; Q2) using <em>-slidingwindow</em> (20:2), <em>-leading</em> and <em>-trailing</em> options. Q2 was used for optimal quantification as previously demonstrated (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">MacManes 2014</a>). A reference transcriptome for Brook Charr was obtained from the Phylofish database (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pasquier <em>et al.</em> 2016</a>). Trimmed reads were mapped against the reference transcriptome with <em>bowtie2</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Langmead and Salzberg 2012</a>) using <em>–end-to-end</em> mode reporting multiple alignments (<em>-k</em> 40) for optimal use with eXpress for read count generation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Roberts and Pachter 2013</a>). The multiple alignment file was converted to bam format and sorted by read name using SAMtools (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2009</a>) and input to eXpress (see full bioinformatics pipeline in <em>Data Availability</em>).</p><p id="p-15">Read counts were imported into edgeR (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Robinson <em>et al.</em> 2010</a>) for normalization and low expression filtering. The smallest library (lib87, 8,373,387 aligned reads) was used to calculate a low-expression threshold. A threshold of 10 reads per transcript in this library defined the threshold for transcript retention (count-per-million &gt; 1.19), as suggested in the edgeR documentation. Any transcript passing this threshold in at least five individuals was retained through the first filtering step for initial data visualization and annotation. Additional low expression filtering was conducted in the differential expression analysis and network analysis (see below). Although transcripts were previously annotated in the Phylofish database (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pasquier <em>et al.</em> 2016</a>), each transcript was re-annotated using trinotate (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bryant <em>et al.</em> 2017</a>) and <em>tblastx</em> against Swissprot (cutoff = 1e<sup>-5</sup>) to obtain as many identifiers as possible for Gene Ontology enrichment analysis. For individual gene descriptions, the re-annotated Swissprot identifier was used primarily, and the Phylofish annotation secondarily.</p></div><div id="sec-5" class="subsection"><h3>Differential expression analysis between sexes</h3><p id="p-16">To reduce the number of transcripts with very low expression in the differential expression analysis, we applied a low expression filter of cpm &gt; 1.19 in at least 65% of the individuals from each sex (<em>i.e.</em>, ≥ 31 / 47 females or 35 / 53 males). The data were then normalized using the weighted trimmed mean of M-values (TMM; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Robinson and Oshlack 2010</a>) to generate normalized log<sub>2</sub> expression levels. Using <em>model.matrix</em> from edgeR with sex as the data grouping, a genewise negative binomial generalized linear model (<em>glmFit</em>) was fit to each gene. Genes with false discovery rate of ≤ 0.05 and linear fold-change ≥ 1.5 were defined as differentially expressed. These genes were binned into low sex bias (<em>i.e.</em>, 1.5 ≤ FC ≤ 4) or high sex bias (<em>i.e.</em>, FC &gt; 4) for each sex (negative FC for females, positive for males), as per previous delimitations (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Poley <em>et al.</em> 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Weighted gene co-expression network analysis (WGCNA) in Brook Charr</h3><p id="p-17">To best estimate associations between modules and phenotypes of interest, sexes were analyzed separately, and then the preservation of each module was evaluated in the opposite sex (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). Due to the independent analysis of each sex, low expression filters (<em>i.e.</em>, cpm &gt; 0.5 in at least five individuals) were conducted separately in each sex, then the data normalized as described above. The average library size was 27,896,535 alignments, indicating cpm &gt; 0.5 corresponds to at least 13.95 reads aligning to the transcript for this mean library size. Sex-specific data were used as an input for weighted gene correlation network analysis (WGCNA; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Langfelder and Horvath 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2012</a>) in R (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">R Core Team 2018</a>).</p><p id="p-18">Within each sex, sample outliers were detected and removed by clustering samples based on transcript expression by Euclidean distance and visually inspecting relationships with the <em>hclust</em> average agglomeration method of WGCNA (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Langfelder and Horvath 2008</a>). Removal of outliers prevents spurious correlations of modules due to outlier values and improves network generation (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). Remaining samples were then correlated with trait data using <em>plotDendroAndColors</em>. Network parameters for both female and male networks were defined as per tutorials using unsigned correlation networks (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Langfelder and Horvath 2008</a>). Unsigned networks consider the connectivity between identical positive or negative correlations to be equal, and thus genes in the same module may have similar or inverse expression patterns. An optimal soft threshold power (6) was identified by evaluating effects on the scale free topology model fit and mean connectivity by increasing the threshold power by 1 between 1-10 and by 2 between 12-20 (Figure S1), as suggested by <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Langfelder and Horvath (2008)</a>. An unsigned adjacency matrix was generated in WGCNA to identify the 25,000 most connected transcripts to retain for reducing computational load. Then, to further minimize noise and spurious associations, adjacency relationships were transformed to the Topological Overlap Matrix using the <em>TOMdist</em> function (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Langfelder and Horvath 2008</a>).</p><p id="p-19">Similarity between modules was evaluated using module eigengenes (<em>i.e.</em>, the first principal component of the module). Dissimilarity between eigengenes was calculated by signed Pearson correlation as suggested by <a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Langfelder and Horvath (2008)</a> and plotted using <em>hclust</em>. When modules were more than 0.75 correlated (dissimilarity 0.25), they were merged as suggested by <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Langfelder and Horvath (2008)</a>. Merged module eigengenes were then tested for associations with phenotypes by Pearson correlation. Notably the sign of the correlation does not necessarily indicate the direction of the relationship between the expression of specific genes in each module and the phenotype because the modules were built using unsigned networks.</p><p id="p-20">Module membership (<em>i.e.</em>, the module eigengene-gene correlation) was used to define the top central transcripts for each module (<em>i.e.</em>, hub genes). Gene significance (<em>i.e.</em>, the absolute value of the trait-gene correlation) was calculated for each transcript against traits weight, specific growth rate, condition factor, hepatosomatic index, change in cortisol, osmolality and chloride from the brief handling stressor, female egg diameter, and male sperm concentration and diameter. Module eigengenes were tested for correlation against traits using Pearson correlation (<em>P</em> ≤ 0.01).</p><p id="p-21">Gene Ontology enrichment analysis of transcripts within each module was conducted using the re-annotated Swissprot identifiers in DAVID Bioinformatics (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang <em>et al.</em> 2009</a>). Heatmaps for modules of interest were generated by using the package <em>gplots</em> using the normalized log<sub>2</sub> cpm data (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Warnes <em>et al.</em> 2016</a>). Expression values were standardized across samples for each transcript and Pearson correlation was used to cluster transcripts and samples.</p><p id="p-22">To determine sex-specific or sex-conserved modules, module preservation was evaluated by comparing male transcript expression to the generated female modules, and visa-versa, using the <em>modulePreservation</em> function of WGCNA. A total of 200 permutations of randomly assigned module labels were used to calculate module preservation rank and Zsummary (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). Low Zsummary scores indicate no preservation (≤ 2), intermediate indicate moderate preservation (2-10) and high scores (≥ 10) indicate strong module preservation (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). The similarity in ranking of modules in terms of preservation in the two different comparative networks (<em>i.e.</em>, the opposite sex in Brook Charr and the male Arctic Charr data) was performed by testing the correlation of the module preservation statistic using Spearman rank correlation in R. Module quality was also determined for each module as a measure of module robustness that is characterized by conducting the analysis on multiple random subsets of the original data (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). In addition, cross-tabulation of the proportions of female modules in male modules and visa-versa were performed in R. Cross-tabulation requires similar modular structures of the compared networks, whereas adjacency comparisons directly compare co-expression independent of network topology. All pipelines to analyze the current data are documented and available on GitHub (see <em>Data Availability</em>).</p></div><div id="sec-7" class="subsection"><h3>Module preservation in Arctic Charr</h3><p id="p-23">To compare module preservation between Brook Charr and Arctic Charr <em>S. alpinus</em> we used RNA-seq data from 1+ year-old Arctic Charr. The broodstock of this population was reared in hatchery conditions for three generations after being collected from a subarctic, land-locked population in Finland (Lake Kuolimo, 61°16′ N; 27°32′ E). The data were collected from nine male liver samples (fish relatedness not known) from each of 8° and 15° (total = 18 samples), but due to a large effect of temperature on the transcriptome, and differences between sampling times in the 15° group, only the nine samples from the 8°, group were used here (normal rearing temperature during summer at the fish hatchery; Figure S2) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Prokkola <em>et al.</em> 2018</a>). Fish body mass at 8° was on average 24.2 g ± SD (S.D.) 10.4 g.</p><p id="p-24">Sample processing was explained fully by <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Prokkola <em>et al.</em> (2018)</a> and briefly described here. In August 2013, fish were killed using 200 ppm sodium bicarbonate-buffered tricaine methanesulfonate (MS-222), after which liver samples were collected and flash frozen in liquid nitrogen (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Prokkola <em>et al.</em> 2018</a>). RNA was extracted from approximately 10 mg of liver tissue using Tri-reagent (Molecular Research Center), and quality checked using a BioAnalyzer (Agilent), with an average identified RNA integrity number of 9.95. Strand-specific cDNA library preparation and sequencing were conducted at Beijing Genomics Institute (BGI Hong Kong) using TruSeq RNA Sample Prep Kit v2 (illumina) and sequenced on an Illumina HiSeq2000 instrument to generate paired-end 100 bp reads. All samples were pooled with unique barcodes across four sequencing lanes. Adapters were removed at BGI, and reads trimmed with Trimmomatic (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>) using options <em>leading</em> and <em>trailing</em> (5) <em>slidingwindow</em> (4:15) and <em>minlen</em> (36). From samples included in this study, on average 41.7 ± SD 7.4 million reads were obtained.</p><p id="p-25">Transcript expression was calculated as above, including using the Brook Charr reference transcriptome for ease of cross-species comparisons. Low expression filtering and normalization for the Arctic Charr data were conducted as above (cpm &gt; 0.5 in at least five individuals). However, a network was not constructed for these samples. Using samples from both temperatures, modules were previously identified (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Prokkola <em>et al.</em> 2018</a>). Once normalized and input to WGCNA, read counts in Arctic Charr 8° samples were used to build a gene adjacency matrix, which was then compared against modules generated for female and male Brook Charr samples using the <em>modulePreservation</em> function as described above. Caveats regarding this data should be noted, including the smaller sample size, immature state of Arctic Charr, minor differences in rearing environments (albeit both were reared in hatchery conditions), and unknown relatedness.</p></div><div id="sec-8" class="subsection"><h3>Identifying one transcript per gene and assigning chromosome positions</h3><p id="p-26">The Brook Charr reference transcriptome (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Pasquier <em>et al.</em> 2016</a>) possibly contains multiple isoforms for individual genes (Y. Guiguen, <em>pers. comm</em>.). Therefore an approach was taken here to reduce multiple genes to a single transcript per gene for enrichment analyses on the sex chromosome or in sex-specific modules. First, the Brook Charr reference transcriptome was aligned to the Atlantic salmon <em>Salmo salar</em> chromosome-level genome assembly RefSeq GCF_000233375.1 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lien <em>et al.</em> 2016</a>) using GMAP (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wu and Watanabe 2005</a>). Alignments were converted to an indexed bam retaining only high quality (-q 30) alignments using samtools (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2009</a>). The indexed bam was converted to a bed file using bedtools <em>bamtobed</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Quinlan and Hall 2010</a>). Second, the lengths of Brook Charr transcripts were calculated using custom python scripts (see <em>Data Availability</em>). With the alignment position, lengths, and expression status (expressed or not expressed), a single transcript per contiguous (or overlapping) alignment block on the reference genome was retained using a custom R script (see <em>Data Availability</em>). For each contiguous alignment block, this script preferentially retained the longest, expressed transcript. All other redundant transcripts, and all that did not align, were not retained. In some cases, a single transcript can align to multiple locations with high mapping quality (MAPQ ≥ 30). Since there was no reason to retain one alignment over another, both alignments were retained in the baseline set for these cases.</p><p id="p-27">The alignment information per retained Brook Charr transcript was used to assign an Atlantic Salmon chromosome identifier to each retained unique transcript. The chromosome information was combined with the module information for all transcripts in each sex-specific network. This analysis was conducted separately for females and males, as expressed genes were in some cases different between the two sexes and therefore so would be the selection of which transcript to retain. The correspondence between the Atlantic Salmon genome assembly accession identifier and the Atlantic Salmon chromosome identifier were obtained from the NCBI genome assembly website (see <em>Data Availability</em>). In general, correspondence of gene synteny is expected to be similar between Atlantic Salmon and Brook Charr (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Sutherland <em>et al.</em> 2016</a>). Using the chromosome correspondence table in <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Sutherland <em>et al.</em> (2016)</a>, the Atlantic Salmon chromosome containing the sex chromosome of Brook Charr was identified. For each sex-specific co-expression module, the proportions of Brook Charr genes located on the Atlantic Salmon chromosome that corresponds to the Brook Charr sex chromosome were characterized and compared to the total list of all non-redundant Brook Charr transcripts identified. Fisher exact tests were then used to determine significance for each sex-specific module-sex chromosome combination (<em>i.e.</em>, two tests). The chromosome information was also combined with the sex bias fold change values. Finally, the proportion of highly or moderately sex-biased transcripts for females and males were investigated for overrepresentation on the sex chromosome and in sex-specific or conserved modules using Fisher’s exact tests in custom R scripts (see <em>Data Availability</em>).</p></div><div id="sec-9" class="subsection"><h3>Data Availability</h3><p id="p-28">Brook Charr raw sequence data are available on SRA under BioProject PRJNA445826, accession SRP136537. The bioinformatics pipeline to analyze this data is available at <a href="https://github.com/bensutherland/sfon_wgcna">https://github.com/bensutherland/sfon_wgcna</a>. Additional results that support the main text are available in the Supplemental Results section. Module assignments for transcripts in networks, differential expression results, and gene expression counts are provided in the supplemental information. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7586681">https://doi.org/10.25387/g3.7586681</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Transcriptome overview</h3><p id="p-29">Of the total 69,440 transcripts in the Brook Charr reference transcriptome, 51,911 passed initial low expression filters when using all samples together. Low-expression filtering for differential expression analysis (<em>i.e.</em>, expressed in at least 65% of individuals from one of two sexes) resulted in the retention of 42,622 transcripts. Low-expression filtering for sex-specific network analysis (<em>i.e.</em>, expressed in at least five individuals of the sex of interest) resulted in 50,748 and 50,530 transcripts passing filters in females and males, respectively. When considering each sex individually, most of the expressed genes were expressed in a majority of the samples: females expressed 35,461 transcripts in &gt; 90% of the samples; males expressing 35,714 transcripts in &gt; 90% of the samples (Figure S3).</p><p id="p-30">Hierarchical clustering of samples by gene expression indicated a large effect of sex, where 35 of 47 F<sub>2</sub> females clustered with the F<sub>1</sub> female, and 52 of 53 F<sub>2</sub> males were clustered together with the F<sub>1</sub> male (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). As described in the Methods, outliers were removed to avoid spurious network correlations (<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>), and this included the removal of one male leaving 52 males remaining, and the removal of one group of females that had large liver weight, leaving 35 females remaining (see phenotype liver weight in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; Figure S4). When these outlier samples were included while constructing the female network, many modules correlated with the liver weight phenotype (<em>data not shown</em>), suggesting that these samples were having a large impact on the network.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/955/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Brook Charr individual samples clustered by gene expression similarity in the liver using all genes with corresponding quantitative trait values shown in the heatmap below the dendrogram with intensity of red reflecting the normalized trait value for that sample. Sex was the largest factor affecting the data (see heatmap sex row; white = females; red = males). Parents were sequenced in duplicate, and clustered with the offspring of their respective sex (see 101ab for mother and 102ab for father; parents have gray missing data values for all phenotypes but sex and RIN). Females with large liver weight clustered outside the other female samples (see on the right-hand side on the liver weight row), and were removed as they were considered outliers (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1871659904" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Brook Charr individual samples clustered by gene expression similarity in the liver using all genes with corresponding quantitative trait values shown in the heatmap below the dendrogram with intensity of red reflecting the normalized trait value for that sample. Sex was the largest factor affecting the data (see heatmap sex row; white = females; red = males). Parents were sequenced in duplicate, and clustered with the offspring of their respective sex (see 101ab for mother and 102ab for father; parents have gray missing data values for all phenotypes but sex and RIN). Females with large liver weight clustered outside the other female samples (see on the right-hand side on the liver weight row), and were removed as they were considered outliers (see Methods).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/955/F1.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/955/F1.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/955/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/955/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492367" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Brook Charr individual samples clustered by gene expression similarity in the liver using all genes with corresponding quantitative trait values shown in the heatmap below the dendrogram with intensity of red reflecting the normalized trait value for that sample. Sex was the largest factor affecting the data (see heatmap sex row; white = females; red = males). Parents were sequenced in duplicate, and clustered with the offspring of their respective sex (see 101ab for mother and 102ab for father; parents have gray missing data values for all phenotypes but sex and RIN). Females with large liver weight clustered outside the other female samples (see on the right-hand side on the liver weight row), and were removed as they were considered outliers (see Methods).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Interestingly, females displayed higher inter-individual variance in gene expression than males, as indicated by the multiple smaller sample clusters of females in the hierarchical clustering relative to the fewer and larger sample clusters of the males (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Likewise, six of the phenotypic traits also displayed higher variance in females (significant heteroscedasticity at <em>P</em> &lt; 0.05, Levene’s test): liver weight, cortisol level, osmolality, and change in cortisol, osmolality and chloride levels due to handling (Figure S5). Some of the higher variance in these traits in females was likely explained by maturity, but maturity did not explain all of the variance within the female gene expression, as the different sample clusters observed had a variety of maturity states, and samples that were all determined to be mature were present in different sample clusters (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). In the following, the sex-specific networks will be presented and compared against the alternate sex and the congener Arctic Charr.</p></div><div id="sec-12" class="subsection"><h3>Network construction and phenotype correlations: female Brook Charr</h3><p id="p-33">Highly correlated module eigengenes (r &gt; 0.75) were merged, combining 81 modules into 14 (Figure S6A; Figure S7A). Assigned female modules each contained a range of 77-10,533 transcripts (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The largest module was <em>darkred</em>, with 10,533 transcripts (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), which included more transcripts than even the unassigned <em>grey</em> module (second largest; 5,892 transcripts).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492451/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492451/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Female modules shown with the number of transcripts within the module (n), the general category of traits correlated with the module (<em>P</em> ≤ 0.01), the most significantly enriched Gene Ontology category (Biological Process), the Zsummary for preservation of the module in Brook Charr males (BC m) and Arctic Charr males (AC m), as well as the module quality (robustness). Zsummary &lt; 2 is not preserved, 2 &lt; Zsummary &lt; 10 is moderately preserved, and &gt; 10 is preserved. The <em>grey</em> module includes unassigned genes and the <em>gold</em> module is a random selection of 1000 genes from the assigned modules for testing preservation metrics. Full module-trait correlations are shown in</strong> <a id="xref-fig-2-1" class="xref-fig" href="#F2"><strong>Figure 2</strong></a><strong>, full GO enrichment in Additional File S1, and expanded summaries of this table in Additional File S2.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-34">Correlations of module eigengenes with specific phenotypes (n = 15) indicate potential functional associations of the modules (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The strongest associations of phenotypes to modules were with maturity index, for example with <em>thistle2</em> (r &gt; 0.81), and <em>coral1</em> (r = -0.83). Although the large liver weight outlier samples were removed prior to network generation, liver weight remained highly correlated with <em>indianred4</em> (r = 0.73), <em>salmon</em>, <em>coral</em> (r = 0.52), and <em>blue2</em> (r = -0.64). Growth rate showed similar module correlations to liver weight (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Osmolality change was also correlated with <em>indianred4</em> (r = 0.56) and <em>blue2</em> (r = -0.58), as well as with <em>darkred</em>, <em>thistle3</em> (r ≥ 0.51), <em>darkorange</em>, <em>green</em> and <em>darkmagenta</em> (r ≥ |-0.49|). Chloride change had no significant associations, but post-stress chloride was correlated with <em>thistle3</em> (r = 0.56) and <em>lightsteelblue</em> (r = -0.53). Although no modules were significantly associated with cortisol (change or post-stress; <em>P</em> ≥ 0.01), <em>ivory</em> was close (r = -0.38; <em>P</em> = 0.03).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/955/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Module-trait relationships for Brook Charr females, estimated with Pearson correlation r-values and p-values. The boldness of color indicates the strength of the relationship. Module-trait correlations are also shown in Table 1 with more general grouping of traits alongside other metrics such as module size and enriched Gene Ontology categories. The male network module-trait relationships are shown in Figure S9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1871659904" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Module-trait relationships for Brook Charr females, estimated with Pearson correlation r-values and p-values. The boldness of color indicates the strength of the relationship. Module-trait correlations are also shown in Table 1 with more general grouping of traits alongside other metrics such as module size and enriched Gene Ontology categories. The male network module-trait relationships are shown in Figure S9.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/955/F2.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/955/F2.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/955/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/955/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492540" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Module-trait relationships for Brook Charr females, estimated with Pearson correlation r-values and p-values. The boldness of color indicates the strength of the relationship. Module-trait correlations are also shown in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> with more general grouping of traits alongside other metrics such as module size and enriched Gene Ontology categories. The male network module-trait relationships are shown in Figure S9.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In order to understand the gene composition of modules, Gene Ontology functional enrichment analysis of the transcripts within each module was conducted (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>; Additional File S1). The <em>salmon</em> module (correlated with liver weight) was enriched for erythrocyte development. The <em>blue2</em> module (liver weight) was associated with ribonucleoprotein complex. <em>Darkorange</em>, <em>green</em>, and <em>darkmagenta</em> modules (all correlated with osmolality change) were enriched for small molecular metabolic process, translation and metabolism functions, respectively. One module did not have significant enrichment of biological processes, <em>lightsteelblue</em> (correlated with chloride change).</p></div><div id="sec-13" class="subsection"><h3>Preservation of co-expression: female Brook Charr network</h3><p id="p-37">The preservation of co-expression of female Brook Charr modules in Brook Charr males was primarily evaluated using network adjacency comparisons, which are more sensitive and robust than cross-tabulation methods (<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Langfelder <em>et al.</em> 2011</a>). Most female modules were preserved in males and only <em>darkred</em> had weak evidence for preservation (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). <em>Green</em> was the most conserved (Zsummary = 100), followed by <em>blue2</em>, <em>salmon</em>, <em>lightsteelblue</em> and <em>darkorange</em> (Zsummary ≥ 48; <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). Modules associated with translation activities were among the highest conserved modules (<em>e.g.</em>, <em>green</em> and <em>blue2</em>).</p><p id="p-38">Published male Arctic Charr liver transcriptome data were then compared to the network to evaluate cross-species module preservation (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Prokkola <em>et al.</em> 2018</a>). Even with caveats regarding sample size (see Methods), several female Brook Charr modules were highly preserved in Arctic Charr males, including <em>blue2</em> (Zsummary = 34), <em>green</em> (Zsummary = 24), and <em>salmon</em> (Zsummary = 17), also the most conserved in male Brook Charr (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). Other female Brook Charr modules with moderate evidence for preservation in male Arctic Charr included <em>darkorange</em> and <em>lightsteelblue</em> (Zsummary &gt; 8), which were also highly preserved in male Brook Charr. It is noteworthy that the ranking of preservation of female modules in the Arctic Charr and Brook Charr males is highly similar (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>; Spearman rho = 0.895; <em>P</em> &lt; 0.00005; Figure S8).</p></div><div id="sec-14" class="subsection"><h3>Network construction, phenotype correlations, and sex-specificity: male Brook Charr</h3><p id="p-39">Highly correlated male modules (eigengene correlation r &gt; 0.75) were merged, reducing 44 assigned male modules to 25 (Figure S6B; Figure S7B). Unlike the female network, a large proportion of the male data could not be assigned to a module. The unassigned <em>grey</em> module contained 72% of the analyzed transcripts (17,992 transcripts). Assigned modules each contained between 54-1,732 transcripts (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>; Additional File S2). Phenotypic correlations with male module eigengenes were tested (Figure S9; see Supplemental Results; summarized in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Of note were several modules enriched for immunity-related functions, including <em>darkmagenta</em> (defense response to virus) and <em>steelblue</em> (positive regulation of innate immune response) (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and Additional File S1). These two immune processes were found to belong to different modules that were not just inversely regulated but rather having somewhat decoupled regulation, given that the network constructed was unsigned and thus the sign of the correlation does not affect whether the genes are grouped (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). However, these modules were still correlated even if not grouped into a single module (Figure S7B).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492475/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492475/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>Male modules with the number of transcripts within the module (n), the general category of traits correlated with the module (<em>P</em> ≤ 0.01), the most significantly enriched Gene Ontology category (Biol. Proc.), the Zsummary for preservation of the module in Brook Charr females (BC f) and Arctic Charr males (AC m), as well as the identified module quality (robustness). Zsummary &lt; 2 is not preserved, 2 &lt; Zsummary &lt; 10 is moderately preserved, and &gt; 10 is preserved. The <em>grey</em> module includes unassigned genes and the <em>gold</em> module is a random selection of 1000 genes from the assigned modules for testing preservation metrics. Full module-trait correlations are shown in Figure S9, full GO enrichment in Additional File 1, and expanded summaries of this table in Additional File S2.</strong></span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/955/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmaps of normalized transcript expression values, clustering both samples and transcripts using Pearson correlation within (A) two immunity-related male modules, steelblue and darkmagenta, which are related to innate immunity and innate antiviral immunity, respectively, (B) the preserved male module yellow (translation), and (C) the male-specific ivory (transcription factor activity). Samples are shown on the horizontal with colors corresponding to the three categories of samples in the legend. The two modules shown in (A) are colored on the vertical based on the cluster in which the transcript is contained, as shown in the legend." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1871659904" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmaps of normalized transcript expression values, clustering both samples and transcripts using Pearson correlation within (A) two immunity-related male modules, &lt;em&gt;steelblue&lt;/em&gt; and &lt;em&gt;darkmagenta&lt;/em&gt;, which are related to innate immunity and innate antiviral immunity, respectively, (B) the preserved male module &lt;em&gt;yellow&lt;/em&gt; (translation), and (C) the male-specific &lt;em&gt;ivory&lt;/em&gt; (transcription factor activity). Samples are shown on the horizontal with colors corresponding to the three categories of samples in the legend. The two modules shown in (A) are colored on the vertical based on the cluster in which the transcript is contained, as shown in the legend.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/955/F3.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/955/F3.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/955/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/955/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492399" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Heatmaps of normalized transcript expression values, clustering both samples and transcripts using Pearson correlation within (A) two immunity-related male modules, <em>steelblue</em> and <em>darkmagenta</em>, which are related to innate immunity and innate antiviral immunity, respectively, (B) the preserved male module <em>yellow</em> (translation), and (C) the male-specific <em>ivory</em> (transcription factor activity). Samples are shown on the horizontal with colors corresponding to the three categories of samples in the legend. The two modules shown in (A) are colored on the vertical based on the cluster in which the transcript is contained, as shown in the legend.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Preservation of male modules in females was also evaluated, but in contrast to what was observed in the preservation of female modules in males (see above), many of the male modules were weakly to moderately preserved in the females. Highly preserved modules included <em>yellow</em> (Zsummary = 58; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>) and <em>brown</em> (Zsummary = 38), <em>tan</em> (Zsummary = 43), and <em>lightcyan</em> (Zsummary = 34). Some modules were less preserved, and therefore more sex-specific, than even the randomly generated <em>gold</em> module (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) and the unassigned <em>grey</em> module, including <em>green</em> (translation and size; Zsummary = 4.5), <em>darkgrey</em> (mitochondrial membrane; Zsummary = 1.8) and <em>ivory</em> (transcription factor activity; Zsummary = 0.8; <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Preserved modules <em>yellow</em> and <em>brown</em> were enriched for ribosomal or translation-related functions, as was the more sex-specific <em>green</em> module (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). <em>Ivory</em>, the most sex- and species-specific male module (see below; <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) was enriched for neurogenesis in GO biological process, but also transcription factor activity in GO molecular function (Additional File S1). Other non-preserved or lowly preserved modules were enriched for membrane activity including <em>darkgrey</em> (mitochondrial inner membrane) and <em>lightcyan1</em> (membrane organization; Additional File S1). Preservation of male Brook Charr modules was also explored in Arctic Charr males. Similar to that observed in the female modules, when a male Brook Charr module was preserved in female Brook Charr, it was also often preserved in male Arctic Charr (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>; Spearman rho = 0.69; <em>P</em> &lt; 0.0005; Figure S8).</p></div><div id="sec-15" class="subsection"><h3>Sex-biased transcripts, sex-specific modules, and the sex chromosome</h3><p id="p-42">To further understand the relation between sex-bias in gene expression and sex-specificity in network architecture, a gene-by-gene differential expression analysis between the sexes was conducted. Of the 42,622 expressed transcripts, 6,983 (16.4%) were differentially expressed (FC ≥ 1.5; glmFit FDR ≤ 0.05). Female-biased genes included 3,989 moderately (1.5-fourfold) and 236 highly biased (&gt;fourfold) transcripts. Highly biased transcripts included known sex-biased genes such as <em>vitellogenin</em>, and <em>zona pellucida sperm-binding proteins</em>. Male-biased genes included 2,638 moderately and 120 highly biased transcripts, including <em>semaphorin-3F</em> (most highly male-biased transcript). For a complete list, see Additional File S3.</p><p id="p-43">Interestingly, sex-biased transcripts were not overrepresented in female or male sex-specific modules (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). However, unexpectedly, highly male-biased genes were overrepresented in highly preserved modules (36 transcripts in highly preserved modules, 97% of the highly sex-biased transcripts) in comparison to the overall percentage in the network (4,886 transcripts in highly preserved modules, 76.5% of network transcripts; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>; two-sided Fisher’s exact test <em>P</em> = 0.0013). The female data were more similar between the highly sex-biased transcripts and the entire network (45.2% and 46.6%, respectively). Sex-specific modules were not enriched on the sex chromosome (Fisher’s exact test <em>P</em> &gt; 0.5), including male modules <em>ivory</em> and <em>darkgrey</em> (Table S1).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492526/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492526/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><strong>Sex-biased transcript presence in modules that are either unique to each sex (low module preservation), or moderately or highly preserved, along with the number and percentage of the transcripts within each sex’s sex-biased transcript category (<em>e.g.</em>, female high sex-bias). These counts only include expressed transcripts that are assigned to modules in the network analysis for each sex.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-44">Of the 47 non-overlapping, highly male-biased transcripts assigned to chromosomes, five were on the sex chromosome (10.6%), relative to 779 on the sex chromosome of the 12,934 expressed in males (Ssa09; 6.0%). However, this difference was not significant (one-sided Fisher’s exact test <em>P</em> = 0.15). Furthermore, moderately male-biased transcripts were not enriched on the sex chromosome (5.3%) relative to all expressed transcripts (6%), nor were highly or moderately female-biased transcripts (high bias = 4.1%; moderate bias = 5.8%).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-45">Gene co-expression produces complex phenotypes and may underlie key aspects of phenotypic evolution (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Filteau <em>et al.</em> 2013</a>) and sexual dimorphism (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). One of the main challenges in producing females and males is developing different phenotypes from largely the same set of genes (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rowe <em>et al.</em> 2018</a>). Transcriptome architecture may provide a solution to this challenge. In this study, co-expression networks for both female and male Brook Charr liver transcriptomes were characterized and compared to each other. Although the female network had much higher module assignment of expressed transcripts, in general most modules were preserved between the sexes. Two sex-specific modules were identified in males that may provide insight on the evolution of gene expression and phenotypic sexual dimorphism. Sex bias was observed in 16% of the expressed transcripts, and surprisingly these sex-biased transcripts were not overrepresented in sex-specific modules. This indicates the value of using these two different approaches given that different information was obtained from each.</p><div id="sec-17" class="subsection"><h3>Sex differences in co-expression networks and cross-species preservation</h3><p id="p-46">Females and males clustered distinctly in unsupervised clustering by gene expression, as has been observed in other studies of fish liver at a reproductive stage (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Qiao <em>et al.</em> 2016</a>). Interestingly, there was a large difference in the number of transcripts assigned to modules between the sexes; of the 25 k most connected transcripts, females had 76% transcripts assigning to a module, and males only 28%. Importantly, this observation coincides with higher inter-individual variation in gene expression in females than in males (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) and higher inter-individual variation in six phenotypes in females than in males (Figure S5). When inter-individual variation in gene expression is low, transcripts will not be as well clustered, because without variance there can be no co-variance for clustering algorithms to act upon (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tritchler <em>et al.</em> 2009</a>). Therefore, the lower variance in male gene expression may have resulted in the observed lower module assignment. The observation of lower variation in phenotypes is an interesting result that highlights similarities between transcriptomic and phenotypic expression. Building networks in both sexes allowed for the identification of this lower assignment to modules in males, which may have not been noted otherwise, as female modules generally were scored as preserved in males. Furthermore, this allowed the identification of many male network modules with seemingly important and different functional associations that in the female network were all grouped together into one very large module (<em>i.e.</em>, <em>darkred</em>). The reason for the grouping of these multiple male modules all together into a single female module is not clear, but could be due to an increased effect of maturity on the data in females that may be swamping out the other more subtle covariances in the data. This further indicates the value of doing separate analyses in each sex, in order to avoid signal being overwhelmed by phenotypes that impact the data more in one sex than the other. It will be valuable to inspect sex-specific module generation in other salmonids and in tissues other than liver to understand the generality of these sex differences and associations to inter-individual variance in gene expression and phenotypes.</p><p id="p-47">Our observations confirm previous findings that co-expression patterns are often preserved between sexes or closely related species (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">van Nas <em>et al.</em> 2009</a>; <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Wong <em>et al.</em> 2014</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cheviron and Swanson 2017</a>). Here, highly preserved modules between the sexes were often comprised of genes within pathways involved in conserved functions. The most preserved modules between the sexes and species were involved in basic cellular processes and included many co-expressed subunits of a multiple subunit protein complex, such as translation machinery. Multiple subunits and functionally related genes have long been known to cluster together by co-expression (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Eisen <em>et al.</em> 1998</a>). Immunity-related modules were also preserved between the sexes, with co-expression patterns similar to those observed previously in salmonids (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sutherland, <em>et al.</em> 2014b</a>). Considering the importance of immune function to both sexes, it is not surprising that immunity modules are preserved between sexes.</p><p id="p-48">The male-specific module <em>darkgrey</em> and the lowly preserved <em>green</em> module were both associated with size, which can be sexually dimorphic in salmonids and is associated to breeding success in males (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blanchfield <em>et al.</em> 2003</a>). In comparison, no female modules were associated to length and weight, further suggesting that these more male-specific modules could have a role in producing a sexually dimorphic phenotype. The other male-specific module, <em>ivory</em>, may contribute to sexual dimorphism and resolution of sexual antagonism by being an upstream controller of different programs, as it is enriched for transcription factor activity and hub genes as putative transcription factors. Hub genes of <em>ivory</em> include genes from the <em>wnt</em> protein family. Wnt signaling is associated with gonad differentiation and shows sex-specific expression in several studies in mammals and fish (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Vainio <em>et al.</em> 1999</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nicol and Guiguen 2011</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sreenivasan <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Böhne <em>et al.</em> 2014</a>). Future studies could investigate whether transcription factors from the sex-specific <em>ivory</em> module control expression of transcripts found to be sex-biased here once transcription factor binding sites are characterized in this species.</p><p id="p-49">The presence of sex-specific modules and sexually dimorphic gene expression in the liver corresponds with what is known about sex hormones produced in the gonads, as these hormones have been shown to regulate a significant proportion of the liver transcriptome in mouse (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">van Nas <em>et al.</em> 2009</a>). In a large-scale transcriptome study in humans, the liver was not one of the most sexually dimorphic in terms of sex-biased genes (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). In oviparous species at a reproductive stage however, this tissue is highly sexually dimorphic, given the role in females for producing oocyte constituents (<em>e.g.</em>, <em>vitellogenins</em>, <em>zona pellucida</em> proteins) (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Qiao <em>et al.</em> 2016</a>). Some of the strongest phenotypic associations of female modules were to maturity. These associations may reflect the effects of sex hormones such as estradiol, which controls reproduction and has a strong influence on transcription in fish (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Garcia-Reyero <em>et al.</em> 2018</a>). There were no male modules associated to maturity, but there were only six females and two males retained in the analysis that were immature, which prevents a comparison of maturation-related transcripts between the sexes.</p><p id="p-50">The ranking of module preservation levels in both the opposite sex and in Arctic Charr was often similar, suggesting evolutionary conservation for many gene co-expression modules. Even with the lower sample size in Arctic Charr, moderate and high preservation was identified for eight and three of the female modules (n = 14), respectively, and for seven and 14 of the male modules (n = 25), respectively. Modules preserved across species with significant phenotypic correlations may be worthwhile to investigate further regarding their contribution to phenotypes such as growth rate, reproduction, stress response, and immunity. For example, the preserved module in the male network, <em>turquoise</em>, was enriched for immunity and marginally associated with growth (<em>P</em> = 0.05), phenotypes known to trade-off (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lochmiller and Deerenberg 2000</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">van der Most <em>et al.</em> 2010</a>).</p><p id="p-51">Many sex-biased transcripts were identified (n = 6,983 transcripts), but only 154 transcripts were found in sex-specific modules identified through the network comparison approach. Sex-biased expression and sex-specific networks are not always overlapping phenomena (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2016</a>). This highlights the large differences between these approaches, but in general they together provided a more comprehensive result than either in isolation. The sex bias analysis found that neither male-biased nor female biased genes were significantly overrepresented on the sex chromosome, but power to detect this may have been reduced by the use of a reference genome of a related species rather than the target species. In other species, male-biased transcripts are more often associated with migration to the sex chromosome (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rowe <em>et al.</em> 2018</a>), and although there was a trend toward this for the highly male-biased transcripts here, it was not significant. The relationship of sex chromosomes, sex-biased gene expression and sexual dimorphism is not yet well established (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dean and Mank 2014</a>), and this study is an example of integrating these multiple aspects for improved understanding of the role of transcriptomics in generating sex differences.</p></div><div id="sec-18" class="subsection"><h3>Case study: modules separated by immune response type</h3><p id="p-52">To demonstrate the utility of this network approach in investigating specific phenotypes, the following is an analysis of modules associated with immunity in the male network. Separate modules were identified for immune functions involving innate antiviral genes (<em>i.e.</em>, male <em>darkmagenta</em>) and innate immunity C-type lectins (<em>i.e.</em>, male <em>steelblue</em>). This is of large interest considering that these types of immune responses have been observed to respond inversely, where pathogen recognition receptors (<em>e.g.</em>, C-type lectins) are up-regulated and innate antiviral genes down-regulated in the anterior kidney during ectoparasite infection (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Sutherland <em>et al.</em> 2014b</a>) and pathogen recognition receptors are up-regulated and innate antiviral genes are down-regulated in gill tissue during out-migration of steelhead trout <em>Oncorhynchus mykiss</em> smolts (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sutherland <em>et al.</em> 2014a</a>). Even if the genes are not the same between these studies and ours (<em>i.e.</em>, no 1:1 association of orthologs has been done for these datasets), the observation of similar functions in two different modules in the present study may indicate that these functions are hardwired into different modules given that no known infection is occurring within these samples. It is important to note that here unsigned networks were used, and therefore if the two immune response types were completely inversely regulated, they would belong to the same module, which was not observed here. These two modules may therefore not be completely under the same regulatory control as they are not completely inversely correlated. This is a new observation in the regulation of these different immune system processes in salmonids. This is an important avenue for further study given the relevance of these genes to immune responses against pathogens, and the potential response outcomes of co-infection occurring between parasitic and viral agents in nature.</p><p id="p-53">The immune response modules observed here (<em>i.e.</em>, male <em>darkmagenta</em>, <em>steelblue</em>, and <em>turquoise</em>) were all considered as highly preserved between the sexes and moderately to highly conserved in Arctic Charr. It will be valuable to see if these three modules or the genes within them have conserved expression patterns in other species as these may have important roles in defense responses. The tissue was in a post-stress state, which could affect the induction of immune responses, and so additional observations, such as in a resting state, will be valuable. It is possible that the co-expression viewed in these (and other) modules comes from the occurrence of a specific cell type that is present in different levels in the sampled tissue in different individuals. Single-cell RNA-sequencing of immune cells, or <em>in situ</em> gene expression hybridization techniques could address some of these questions. Further, to better understand the immunity related modules, it may also be valuable to use a microbe profiling platform alongside transcriptome studies of wild sampled individuals to best understand co-infection details (<em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Miller <em>et al.</em> 2016</a>). Nonetheless, the characterization of salmonid co-expression modules will be strengthened when additional analyses are conducted with a broader range of species, once orthologs are identified among the species.</p></div><div id="sec-19" class="subsection"><h3>Future comparative approaches and salmonid transcriptome network evolution</h3><p id="p-54">When similar datasets are produced in other salmonids, it will be valuable to identify whether the preserved modules are conserved outside of the genus <em>Salvelinus</em>. However, importantly this will require identification of 1:1 orthologs among the species, which would enable cross-species analyses. Salmonid ortholog identification across reference transcriptomes has recently been conducted for Atlantic Salmon, Brown Trout <em>Salmo trutta</em>, Arctic Charr, and European Whitefish <em>Coregonus lavaretus</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carruthers <em>et al.</em> 2018</a>), as well as Northern Pike (<em>Esox lucius</em>), Chinook Salmon (<em>O. tshawytscha</em>), Coho Salmon (<em>O. kisutch</em>), Rainbow Trout (<em>O. mykiss</em>), Atlantic Salmon, and Arctic Charr (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Christensen <em>et al.</em> 2018</a>). This type of approach, combined with non-redundant reference transcriptomes will be invaluable in future studies to enable cross species comparisons.</p><p id="p-55">If modules are indeed largely conserved between species, as our study suggests within <em>Salvelinus</em> liver, this would indicate that large-scale rewiring of baseline transcription networks has not occurred since the base of the lineage. Species-specific modules will be highly valuable to investigate to better understand transcriptional architecture underlying phenotypic differences between the species. The largest amount of rediploidization is thought to have occurred in the salmonids at the base of the lineage (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kodama <em>et al.</em> 2014</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lien <em>et al.</em> 2016</a>), although a substantial proportion of ohnologs experienced lineage-specific rediploidization post-speciation events later in evolutionary time (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Robertson <em>et al.</em> 2017</a>). Given the large potential impact that divergence in regulatory regions or epigenetic signatures can have on gene expression, one could expect large lineage-specific changes in co-expression networks. The impact of the genome duplication and rediploidization on transcriptome networks, including lineage specific changes are important avenues for future study.</p></div></div><div class="section conclusions" id="sec-20"><h2 class="">Conclusions</h2><p id="p-56">Co-expression networks and sex-biased expression of female and male Brook Charr liver in a reproductive season shortly after an acute handling stressor were characterized in the present study. Results support previous observations of moderate to high preservation of modules between sexes and closely related species. Highly preserved modules were involved in basic cellular functions and immune functions. Sex-specific modules identified only in the male network were enriched for transcription factor activities and associated with sex-biased or potentially sexually antagonistic phenotypes, such as body size. Higher assignment of transcripts to modules was identified in the female network, potentially due to higher inter-individual variance in gene expression and phenotypes. Important physiological functions such as immunity response types were captured by this analysis, identifying not only inverse regulation between two immunity responses but potentially decoupled regulation, which has implications for responses to co-infections and requires further study. This dataset has provided new insights into the transcriptome network structure differences between sexes and has pointed toward individual genes and gene modules that may be involved with generating sexually dimorphic phenotypes and potentially alleviating sexually antagonistic selection.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">Thanks to Guillaume Côté for laboratory assistance, Jérémy Le Luyer and Marie Filteau for discussions about WGCNA, Eric Normandeau for discussion on gene annotation, Yann Guiguen for discussion on details of transcriptome assembly, Phineas Hamilton for statistical discussion, and Aimee-Lee Houde, three anonymous reviewers and the Editors for comments on the manuscript. This work was funded by a Fonds de Recherche du Québec (FRQ) Nature et Technologies research grant awarded to Céline Audet, Louis Bernatchez, and Nadia Aubin-Horth; a grant from the Société de Recherche et de Développement en Aquaculture Continentale (SORDAC) awarded to L.B. and C.A. J.M.P. is supported by the Finnish Cultural Foundation. During this work, B.J.G.S. was supported first by a Natural Sciences and Engineering Research Council of Canada postdoctoral fellowship and subsequently by an FRQ Santé postdoctoral fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">1</span> Current Address: Pacific Biological Station, Fisheries and Oceans Canada, Nanaimo, BC, V9T 6N7, Canada</p></li><li class="fn" id="fn-2"><p id="p-58">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7586681">https://doi.org/10.25387/g3.7586681</a>.</p></li><li class="fn" id="fn-3"><p id="p-59"><em>Communicating editor: D. Macqueen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-19"><span class="received-label">Received </span>November 19, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-21"><span class="accepted-label">Accepted </span>January 21, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Sutherland <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.955.1" data-doi="10.1007/978-1-4684-4652-4_1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorgaard</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tetraploidy and the evolution of salmonid fishes</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">53</span> in <span class="cit-source">Evolutionary genetics of fishes</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">B. J.</span></span> <span class="cit-publ-name">Plenum Publishing Corporation</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4684-4652-4_1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2Bgenetics%2Bof%2Bfishes%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4684-4652-4_1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4684-4652-4_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.2" data-doi="10.1038/nature16062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aykanat</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolstad</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sex-dependent dominance at a single locus maintains variation in age at maturity in salmon.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">528</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">408</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature16062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D528%26rft.spage%253D405%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16062%26rft_id%253Dinfo%253Apmid%252F26536110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26536110&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.3" data-doi="10.1534/genetics.117.300382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blackmon</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Long-term fragility of Y chromosomes is dominated by short-term resolution of sexual antagonism. Genetics.300382.2017–9.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300382%26rft_id%253Dinfo%253Apmid%252F29021279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy80LzE2MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.4" data-doi="10.1046/j.1365-294X.2003.01917.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blanchfield</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ridgway</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Breeding success of male brook trout (<em>Salvelinus fontinalis</em>) in the wild.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2417</span>–<span class="cit-lpage">2428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-294X.2003.01917.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBlanchfield%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D2417%26rft.epage%253D2428%26rft.atitle%253DBreeding%2Bsuccess%2Bof%2Bmale%2Bbrook%2Btrout%2B%2528Salvelinus%2Bfontinalis%2529%2Bin%2Bthe%2Bwild.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2003.01917.x%26rft_id%253Dinfo%253Apmid%252F12919479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2003.01917.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12919479&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.6" data-doi="10.1023/A:1016054119783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boula</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castric</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audet</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Physiological, endocrine, and genetic bases of anadromy in the Brook Charr, <em>Salvelinus fontinalis</em>, of the Laval River (Québec, Canada).</span> <abbr class="cit-jnl-abbrev">Environ. Biol. Fishes</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1016054119783</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BBiol.%2BFishes%26rft.volume%253D64%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1016054119783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1016054119783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.7" data-doi="10.1093/gbe/evu200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Böhne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sengstag</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzburger</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative transcriptomics in East African Cichlids reveals sex- and species-specific expression and new candidates for sex differentiation in fishes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2567</span>–<span class="cit-lpage">2585</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evu200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu200%26rft_id%253Dinfo%253Apmid%252F25364805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25364805&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.8" data-doi="10.1016/j.celrep.2016.12.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiTommaso</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tickle</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Couger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A tissue-mapped Axolotl <em>de novo</em> transcriptome enables identification of limb regeneration factors.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2016.12.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D18%26rft.spage%253D762%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2016.12.063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2016.12.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.9" data-doi="10.1371/journal.pcbi.1002822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chapter 11: Genome-Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D8%26rft.spage%253De1002822%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002822%26rft_id%253Dinfo%253Apmid%252F23300413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.10" data-doi="10.1186/s12864-017-4379-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carruthers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yurchenko</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Augley</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herzyk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title"><em>De novo</em> transcriptome assembly, annotation and comparison of four ecological and evolutionary model salmonid fish species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-4379-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-4379-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-4379-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes-Ramos</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuijjer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulson</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonawane</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sexual dimorphism in gene expression and regulatory networks across human tissues.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">1</span>–<span class="cit-lpage">34</span>. <a href="https://doi.org/10.1101/082289">https://doi.org/10.1101/082289</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.12" data-doi="10.1093/icb/icx118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheviron</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative transcriptomics of seasonal phenotypic flexibility in two North American songbirds.</span> <abbr class="cit-jnl-abbrev">Integr. Comp. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/icb/icx118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr.%2BComp.%2BBiol.%26rft.volume%253D57%26rft.spage%253D1040%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252Ficx118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/icx118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.13" data-doi="10.1371/journal.pone.0204076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rondeau</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minkley</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nugent</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Arctic charr (<em>Salvelinus alpinus</em>) genome and transcriptome assembly.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0204076</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0204076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0204076%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0204076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0204076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.14" data-doi="10.1371/journal.pone.0046662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crête-Lafrenière</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Framing the Salmonidae family phylogenetic portrait: a more complete picture from increased taxon sampling.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrete-Lafreniere%26rft.auinit1%253DA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46662%26rft.epage%253De46662%26rft.atitle%253DFraming%2Bthe%2Bsalmonidae%2Bfamily%2Bphylogenetic%2Bportrait%253A%2Ba%2Bmore%2Bcomplete%2Bpicture%2Bfrom%2Bincreased%2Btaxon%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046662%26rft_id%253Dinfo%253Apmid%252F23071608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071608&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.15" data-doi="10.1111/jeb.12345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The role of sex chromosomes in sexual dimorphism: discordance between molecular and phenotypic data.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1443</span>–<span class="cit-lpage">1453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.16" data-doi="10.1073/pnas.95.25.14863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Cluster analysis and display of genome-wide expression patterns.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">14863</span>–<span class="cit-lpage">14868</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.95.25.14863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEisen%26rft.auinit1%253DM.%2BB.%26rft.volume%253D95%26rft.issue%253D25%26rft.spage%253D14863%26rft.epage%253D14868%26rft.atitle%253DCluster%2Banalysis%2Band%2Bdisplay%2Bof%2Bgenome-wide%2Bexpression%2Bpatterns%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.25.14863%26rft_id%253Dinfo%253Apmid%252F9843981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMjUvMTQ4NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.17" data-doi="10.1038/nrg2167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolution of sex-biased genes and sex-biased gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D689%26rft.epage%253D698%26rft.atitle%253DThe%2Bevolution%2Bof%2Bsex-biased%2Bgenes%2Band%2Bsex-biased%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2167%26rft_id%253Dinfo%253Apmid%252F17680007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17680007&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248882400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.18" data-doi="10.1093/molbev/mst053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filteau</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St-Cyr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene coexpression networks reveal key drivers of phenotypic divergence in Lake Whitefish.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1396</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst053%26rft_id%253Dinfo%253Apmid%252F23519315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23519315&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.19" data-doi="10.1139/d98-009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Pattern and variability in the breeding system of Atlantic salmon (<em>Salmo salar</em>), with comparisons to other salmonids.</span> <abbr class="cit-jnl-abbrev">Can. J. Fish. Aquat. Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">76</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/d98-009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFish.%2BAquat.%2BSci.%26rft.volume%253D55%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fd98-009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/d98-009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.20" data-doi="10.1111/gbb.12106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaiteri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sibille</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Beyond modules and hubs: the potential of gene coexpression networks for investigating molecular mechanisms of complex brain disorders.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gbb.12106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D13%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgbb.12106%26rft_id%253Dinfo%253Apmid%252F24320616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gbb.12106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24320616&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.21" data-doi="10.1016/j.ygcen.2017.07.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Reyero</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayasinghe</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroll</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabo-Attwood</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denslow</span>  <span class="cit-name-given-names">N. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Estrogen signaling through both membrane and nuclear receptors in the liver of Fathead Minnow.</span> <abbr class="cit-jnl-abbrev">Gen. Comp. Endocrinol.</abbr> <span class="cit-vol">257</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygcen.2017.07.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGen.%2BComp.%2BEndocrinol.%26rft.volume%253D257%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygcen.2017.07.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygcen.2017.07.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.22" data-doi="10.1371/journal.pcbi.1002444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">“Guilt by association” is the exception rather than the rule in gene networks.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002444</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGillis%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002444%26rft.epage%253De1002444%26rft.atitle%253D%2522Guilt%2Bby%2Bassociation%2522%2Bis%2Bthe%2Bexception%2Brather%2Bthan%2Bthe%2Brule%2Bin%2Bgene%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002444%26rft_id%253Dinfo%253Apmid%252F22479173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22479173&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.23" data-doi="10.7717/peerj.4345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guðbrandsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franzdóttir</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristjánsson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahi</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">V. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Differential gene expression during early development in recently evolved and sympatric Arctic charr morphs.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e4345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.4345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D6%26rft.spage%253De4345%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.4345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.4345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.24" data-doi="10.7717/peerj.3828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horreo</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Revisiting the mitogenomic phylogeny of Salmoninae: new insights thanks to recent sequencing advances.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e3828</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.3828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D5%26rft.spage%253De3828%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.3828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.3828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.25" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2008.211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kodama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brieuc</span>  <span class="cit-name-given-names">M. S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naish</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Comparative mapping between Coho Salmon (<em>Oncorhynchus kisutch</em>) and three other salmonids suggests a role for chromosomal rearrangements in the retention of duplicated regions following a whole genome duplication event. G3 - Genes|Genomes|Genetics (Bethesda) 4: 1717–1730.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.27" data-doi="10.18637/jss.v046.i11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast R functions for robust correlations and hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v046.i11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DPerez%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253Dc03%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DIntake_epis_food%2528%2529%253A%2BAn%2BR%2BFunction%2Bfor%2BFitting%2Ba%2BBivariate%2BNonlinear%2BMeasurement%2BError%2BModel%2Bto%2BEstimate%2BUsual%2Band%2BEnergy%2BIntake%2Bfor%2BEpisodically%2BConsumed%2BFoods.%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v046.i11%26rft_id%253Dinfo%253Apmid%252F22837731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v046.i11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22837731&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.28" data-doi="10.1186/1471-2105-9-559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WGCNA: an R package for weighted correlation network analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.spage%253D559%26rft.epage%253D559%26rft.atitle%253DWGCNA%253A%2Ban%2BR%2Bpackage%2Bfor%2Bweighted%2Bcorrelation%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-559%26rft_id%253Dinfo%253Apmid%252F19114008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19114008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.29" data-doi="10.1371/journal.pcbi.1001057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langfelder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oldham</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Is my network module preserved and reproducible?</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1001057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DLangfelder%26rft.auinit1%253DP.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De1001057%26rft.epage%253De1001057%26rft.atitle%253DIs%2Bmy%2Bnetwork%2Bmodule%2Bpreserved%2Band%2Breproducible%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001057%26rft_id%253Dinfo%253Apmid%252F21283776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21283776&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.30" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.31" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.32" data-doi="10.1038/nature17164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Atlantic Salmon genome provides insights into rediploidization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">533</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature17164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.33" data-doi="10.1034/j.1600-0706.2000.880110.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lochmiller</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deerenberg</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Trade-offs in evolutionary immunology: just what is the cost of immunity?</span> <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1034/j.1600-0706.2000.880110.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D88%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1600-0706.2000.880110.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1600-0706.2000.880110.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000085424600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.34" data-doi="10.1146/annurev.genet.35.102401.090633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The genetic architecture of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.35.102401.090633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%2BC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D339%26rft.atitle%253DTHE%2BGENETIC%2BARCHITECTURE%2BOF%2BQUANTITATIVE%2BTRAITS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.35.102401.090633%26rft_id%253Dinfo%253Apmid%252F11700286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.35.102401.090633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172991500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.35" data-doi="10.1038/nrg2612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of quantitative traits: challenges and prospects.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2612</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D565%26rft.epage%253D577%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bquantitative%2Btraits%253A%2Bchallenges%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2612%26rft_id%253Dinfo%253Apmid%252F19584810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584810&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.36" data-doi="10.3389/fgene.2014.00013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacManes</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On the optimal trimming of high-throughput mRNA sequence data.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00013%26rft_id%253Dinfo%253Apmid%252F24567737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567737&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.37" data-doi="10.1098/rspb.2013.2881"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macqueen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A well-constrained estimate for the timing of the salmonid whole genome duplication reveals major decoupling from species diversification.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20132881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2013.2881</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2881%26rft_id%253Dinfo%253Apmid%252F24452024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24452024&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.38" data-doi="10.1371/journal.pgen.1006402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mähler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terebieniec</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gene co-expression network connectivity is an important determinant of selective constraint.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006402</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006402%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006402%26rft_id%253Dinfo%253Apmid%252F28406900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28406900&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.955.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanderstichel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnley</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source">Report on the performance evaluation of the Fluidigm BioMark platform for high-throughput microbe monitoring in salmon</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">293</span> Canadian Science Advisory Secretariat (CSAS) Research Document, Fisheries and Oceans Canada, Pacific Region.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.40" data-doi="10.1038/s41540-017-0014-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canty-Laird</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tew</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Cross-species gene modules emerge from a systems biology approach to osteoarthritis.</span> <abbr class="cit-jnl-abbrev">NPJ Syst. Biol. Appl.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41540-017-0014-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNPJ%2BSyst.%2BBiol.%2BAppl.%26rft.volume%253D3%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41540-017-0014-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41540-017-0014-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.41" data-doi="10.1159/000334515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guiguen</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Expression profiling of wnt signaling genes during gonadal differentiation and gametogenesis in Rainbow Trout.</span> <abbr class="cit-jnl-abbrev">Sex Dev.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">329</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000334515</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DSex%2BDev%26rft.aulast%253DNicol%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D318%26rft.epage%253D329%26rft.atitle%253DExpression%2Bprofiling%2Bof%2BWnt%2Bsignaling%2Bgenes%2Bduring%2Bgonadal%2Bdifferentiation%2Band%2Bgametogenesis%2Bin%2Brainbow%2Btrout.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000334515%26rft_id%253Dinfo%253Apmid%252F22116193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000334515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22116193&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.42" data-doi="10.1073/pnas.0605938103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oldham</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geschwind</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Conservation and evolution of gene coexpression networks in human and chimpanzee brains.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">17973</span>–<span class="cit-lpage">17978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0605938103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605938103%26rft_id%253Dinfo%253Apmid%252F17101986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ3LzE3OTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzk1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The evolutionary causes and consequences of sex-biased gene expression.</span> <abbr class="cit-jnl-abbrev">Nature Publishing Group</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPublishing%2BGroup%26rft.volume%253D14%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.44" data-doi="10.1186/s12864-016-2709-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasquier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jouanno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severac</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Gene evolution and gene expression after whole genome duplication in fish: the PhyloFish database.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2709-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D368%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2709-z%26rft_id%253Dinfo%253Apmid%252F27189481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2709-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27189481&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.45" data-doi="10.1186/s12864-016-2835-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poley</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fast</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sex-biased gene expression and sequence conservation in Atlantic and Pacific salmon lice (Lepeophtheirus salmonis).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2835-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D483%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2835-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2835-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.46" data-doi="10.1242/jeb.170670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prokkola</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikinmaa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anttila</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanerva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Cold temperature represses daily rhythms in the liver transcriptome of a stenothermal teleost under decreasing day length.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">221</span>: <span class="cit-fpage">jeb170670</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.170670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.170670%26rft_id%253Dinfo%253Apmid%252F29361589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjE1OiIyMjEvNS9qZWIxNzA2NzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.47" data-doi="10.1038/srep32459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Manach</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sotton</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huet</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duvernois-Berthet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep sexual dimorphism in adult medaka fish liver highlighted by multi-omic approach.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">32459</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep32459</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D32459%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep32459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep32459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.48" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.49" data-doi="10.1006/anbe.1994.1300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinn</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foote</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The effects of body size and sexual dimorphism on the reproductive behavior of sockeye salmon, <em>Oncorhynchus nerka</em>.</span> <abbr class="cit-jnl-abbrev">Anim. Behav.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">751</span>–<span class="cit-lpage">761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/anbe.1994.1300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBehav.%26rft.volume%253D48%26rft.spage%253D751%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbe.1994.1300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbe.1994.1300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PN87100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.955.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.51" data-doi="10.1038/nmeth.2251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Streaming fragment assignment for real-time analysis of sequencing experiments.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2251%26rft_id%253Dinfo%253Apmid%252F23160280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160280&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312810100040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.52" data-doi="10.1186/s13059-017-1241-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gundappa</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grammes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hvidsten</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redmond</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Lineage-specific rediploidization is a mechanism to explain time-lags between genome duplication and evolutionary diversification.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-017-1241-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1241-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1241-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.53" data-doi="10.1186/gb-2010-11-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshlack</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A scaling normalization method for differential expression analysis of RNA-seq data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2010-11-3-r25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRobinson%26rft.auinit1%253DM.%2BD.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DA%2Bscaling%2Bnormalization%2Bmethod%2Bfor%2Bdifferential%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-3-r25%26rft_id%253Dinfo%253Apmid%252F20196867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20196867&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.54" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.55" data-doi="10.1093/gbe/evv258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seneca</span>  <span class="cit-name-given-names">F. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Gene networks in the wild: identifying transcriptional modules that mediate coral resistance to experimental heat stress.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv258%26rft_id%253Dinfo%253Apmid%252F26710855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26710855&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.56" data-doi="10.1086/698198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Genomics of Sexual Conflict.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">274</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/698198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D192%26rft.spage%253D274%26rft_id%253Dinfo%253Adoi%252F10.1086%252F698198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/698198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.57" data-doi="10.1534/g3.111.001867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derôme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> Coding gene single nucleotide polymorphism mapping and quantitative trait loci detection for physiological reproductive traits in Brook Charr, <em>Salvelinus fontinalis</em>. G3 - Genes|Genomes|Genetics (Bethesda) 2: 379–392.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001867</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D379%26rft.epage%253D392%26rft.atitle%253DCoding%2BGene%2BSingle%2BNucleotide%2BPolymorphism%2BMapping%2Band%2BQuantitative%2BTrait%2BLoci%2BDetection%2Bfor%2BPhysiological%2BReproductive%2BTraits%2Bin%2BBrook%2BCharr%252C%2BSalvelinus%2Bfontinalis%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001867%26rft_id%253Dinfo%253Apmid%252F22413092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvMy8zNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.58" data-doi="10.1534/g3.112.001990"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Derôme</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> Coding gene SNP mapping reveals QTL linked to growth and stress response in Brook Charr (<em>Salvelinus fontinalis</em>). G3 - Genes|Genomes|Genetics (Bethesda) 2: 707–720.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.001990</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D707%26rft.epage%253D720%26rft.atitle%253DCoding%2BGene%2BSNP%2BMapping%2BReveals%2BQTL%2BLinked%2Bto%2BGrowth%2Band%2BStress%2BResponse%2Bin%2BBrook%2BCharr%2B%2528Salvelinus%2Bfontinalis%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.001990%26rft_id%253Dinfo%253Apmid%252F22690380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNi83MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.59" data-doi="10.1095/biolreprod.113.110874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sreenivasan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bártfai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gonad differentiation in Zebrafish is regulated by the canonical wnt signaling pathway.</span> <abbr class="cit-jnl-abbrev">Biol. Reprod.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1095/biolreprod.113.110874</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BReprod.%26rft_id%253Dinfo%253Adoi%252F10.1095%252Fbiolreprod.113.110874%26rft_id%253Dinfo%253Apmid%252F24174574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImJpb2xyZXByb2QiO3M6NToicmVzaWQiO3M6NzoiOTAvMi80NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy85NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.60" data-doi="10.1093/gbe/evw262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Normandeau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamothe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isabel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Salmonid chromosome evolution as revealed by a novel method for comparing RADseq linkage maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3600</span>–<span class="cit-lpage">3617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.61" data-doi="10.1111/mec.12713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jantzen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Divergent immunity and energetic programs in the gills of migratory and resident Oncorhynchus mykiss.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1952</span>–<span class="cit-lpage">1964</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12713</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D1952%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.62" data-doi="10.1186/1471-2164-15-200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koczka</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuike</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jantzen</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yazawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Comparative transcriptomics of Atlantic <em>Salmo salar</em>, Chum <em>Oncorhynchus keta</em> and Pink Salmon <em>O. gorbuscha</em> during infections with salmon lice <em>Lepeophtheirus salmonis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-200%26rft_id%253Dinfo%253Apmid%252F24628956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24628956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.63" data-doi="10.1534/g3.117.040915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">B. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rico</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Sex chromosome evolution, heterochiasmy, and physiological QTL in the salmonid Brook Charr <em>Salvelinus fontinalis</em>. G3 - Genes|Genomes|Genetics (Bethesda) 7: 2749–2762.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.040915</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.040915%26rft_id%253Dinfo%253Apmid%252F28626004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzk1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.64" data-doi="10.1146/annurev-cellbio-100913-012908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparative analysis of gene regulatory networks: from network reconstruction to evolution.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-cellbio-100913-012908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BCell%2BDev.%2BBiol.%26rft.volume%253D31%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-cellbio-100913-012908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-cellbio-100913-012908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.65" data-doi="10.1186/1471-2105-10-193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tritchler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkhomenko</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Filtering Genes for Cluster and Network Analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DTritchler%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.spage%253D193%26rft.epage%253D193%26rft.atitle%253DFiltering%2Bgenes%2Bfor%2Bcluster%2Band%2Bnetwork%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-193%26rft_id%253Dinfo%253Apmid%252F19549335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19549335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.66" data-doi="10.1038/17068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vainio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heikkilä</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kispert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Female development in mammals is regulated by Wnt-4 signalling.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">397</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/17068</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVainio%26rft.auinit1%253DS.%26rft.volume%253D397%26rft.issue%253D6718%26rft.spage%253D405%26rft.epage%253D409%26rft.atitle%253DFemale%2Bdevelopment%2Bin%2Bmammals%2Bis%2Bregulated%2Bby%2BWnt-4%2Bsignalling.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F17068%26rft_id%253Dinfo%253Apmid%252F9989404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/17068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9989404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078461700038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.67" data-doi="10.1093/bib/bbw139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Dam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Võsa</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Graaf</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franke</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Magalhães</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Gene co-expression analysis for functional classification and gene-disease predictions.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbw139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbw139%26rft_id%253Dinfo%253Apmid%252F28077403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbw139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28077403&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.68" data-doi="10.1111/j.1365-2435.2010.01800.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Most</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmentier</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verhulst</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Trade-off between growth and immune function: a meta-analysis of selection experiments.</span> <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2435.2010.01800.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D25%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2435.2010.01800.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2435.2010.01800.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.69" data-doi="10.1210/en.2008-0563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Nas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guhathakurta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehya</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Elucidating the role of gonadal hormones in sexually dimorphic gene coexpression networks.</span> <abbr class="cit-jnl-abbrev">Endocrinology</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1210/en.2008-0563</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEndocrinology%26rft_id%253Dinfo%253Adoi%252F10.1210%252Fen.2008-0563%26rft_id%253Dinfo%253Apmid%252F18974276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1210/en.2008-0563&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18974276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263613200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.955.70"><div class="cit-metadata unstructured">Warnes, G. R., B. Bolker, L. Bonebakker, R. Gentleman, W. Hubert <em>et al.</em>, 2016 gplots: Various R Programming Tools for Plotting Data.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.71" data-doi="10.1016/j.tig.2011.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wijchers</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Festenstein</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Epigenetic regulation of autosomal gene expression by sex chromosomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2011.01.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DWijchers%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D132%26rft.epage%253D140%26rft.atitle%253DEpigenetic%2Bregulation%2Bof%2Bautosomal%2Bgene%2Bexpression%2Bby%2Bsex%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.01.004%26rft_id%253Dinfo%253Apmid%252F21334089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21334089&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289699400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.72" data-doi="10.1186/1471-2164-15-905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLeod</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godwin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Limited sex-biased neural gene expression patterns across strains in Zebrafish (Danio rerio).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">910</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-905</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D905%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-905%26rft_id%253Dinfo%253Apmid%252F25326170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25326170&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.73" data-doi="10.1002/evl3.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooney</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloch</span>  <span class="cit-name-given-names">N. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieira</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Male-biased gene expression resolves sexual conflict through the evolution of sex-specific genetic architecture.</span> <abbr class="cit-jnl-abbrev">Evolution Letters</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/evl3.39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2BLetters%26rft.volume%253D2%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fevl3.39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/evl3.39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.955.74" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15728110&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228783000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/955.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/943" title="Arginine to Glutamine Variant in Olfactomedin Like 3 (OLFML3) Is a Candidate for Severe Goniodysgenesis and Glaucoma in the Border Collie Dog Breed" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/969" title="Environmental and Evolutionary Drivers of the Modular Gene Regulatory Network Underlying Phenotypic Plasticity for Stress Resistance in the Nematode Caenorhabditis remanei" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/955" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3yNxGDDAu9G1-l8Mt85xoAA04X11mSEpeYpzj1wsZA8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genetics-sex" class="" data-icon-position="" data-hide-link-title="0">Genetics of sex</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/955" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/955" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/955" class="active" data-icon-position="" data-hide-link-title="0">Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr Salvelinus fontinalis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492490" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1_I1ZMWHpEcVICgX-_bFGtYGQXSUqtwnjqMnOghpMnM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/955.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/955" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-f8G8HICNwwue7C3nvAq7eBBaIZOUL5lHq5XLPKRvKdI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/955" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/955.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492490" id="node492490--2" data-pisa="ggg;9/3/955" data-pisa-master="ggg;g3.118.200910" data-apath="/ggg/9/3/955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr <em>Salvelinus fontinalis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ben J. G. Sutherland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jenni M. Prokkola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Céline Audet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Bernatchez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">955-968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200910 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492490--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ben J. G. Sutherland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ben%2BJ.%2BG.%2BSutherland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sutherland%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABen%2BJ.%2BG.%2BSutherland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ben J. G. Sutherland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ben.sutherland.1@ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">ben.sutherland.1@ulaval.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenni M. Prokkola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Integrative Biology, University of Liverpool, L69 7ZB Liverpool, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenni%2BM.%2BProkkola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prokkola%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenni%2BM.%2BProkkola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jenni M. Prokkola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Céline Audet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut des Sciences de la Mer de Rimouski, Université du Québec à Rimouski, Rimouski, QC G5L 3A1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BAudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Audet%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9line%2BAudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Céline Audet</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Louis Bernatchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BBernatchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernatchez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BBernatchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Bernatchez</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/955" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/955&amp;text=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/955&amp;t=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/955&amp;title=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492490/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492490" id="node492490--3" data-pisa="ggg;9/3/955" data-pisa-master="ggg;g3.118.200910" data-apath="/ggg/9/3/955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Sex-Specific Co-expression Networks and Sex-Biased Gene Expression in the Salmonid Brook Charr <em>Salvelinus fontinalis</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ben J. G. Sutherland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jenni M. Prokkola</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Céline Audet</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Louis Bernatchez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">955-968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200910 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492490--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ben J. G. Sutherland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ben%2BJ.%2BG.%2BSutherland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sutherland%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABen%2BJ.%2BG.%2BSutherland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2029-9893" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ben J. G. Sutherland</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ben.sutherland.1@ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">ben.sutherland.1@ulaval.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenni M. Prokkola </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Integrative Biology, University of Liverpool, L69 7ZB Liverpool, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenni%2BM.%2BProkkola%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prokkola%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenni%2BM.%2BProkkola%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2987-4417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jenni M. Prokkola</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Céline Audet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institut des Sciences de la Mer de Rimouski, Université du Québec à Rimouski, Rimouski, QC G5L 3A1, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BAudet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Audet%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9line%2BAudet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3366-210X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Céline Audet</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Louis Bernatchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institut de Biologie Intégrative et des Systèmes (IBIS), Université Laval, Québec, QC G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BBernatchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernatchez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouis%2BBernatchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8085-9709" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louis Bernatchez</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=955&amp;rft.epage=968&amp;rft.atitle=Sex-Specific%20Co-expression%20Networks%20and%20Sex-Biased%20Gene%20Expression%20in%20the%20Salmonid%20Brook%20Charr%20Salvelinus%20fontinalis&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sutherland%2C+Ben+J.+G.&amp;rft.au=Prokkola%2C+Jenni+M.&amp;rft.au=Audet%2C+C%C3%A9line&amp;rft.au=Bernatchez%2C+Louis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492490/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492490/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492490/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492490/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492490/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492490/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492490/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492490/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492490/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492490/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492490/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492490/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495849" id="node495849" data-pisa="ggg;10/3/967" data-pisa-master="ggg;g3.119.400963" data-apath="/ggg/10/3/967.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/967" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The <em>Drosophila</em> Post-mating Response: Gene Expression and Behavioral Changes Reveal Perdurance and Variation in Cross-Tissue Interactions</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495800" id="node495800" data-pisa="ggg;g3.119.400923v1" data-pisa-master="ggg;g3.119.400923" data-apath="/ggg/early/2020/02/12/g3.119.400923.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/12/g3.119.400923" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Sex Chromosome Evolution in Muscid Flies</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494619" id="node494619" data-pisa="ggg;9/11/3867" data-pisa-master="ggg;g3.119.400656" data-apath="/ggg/9/11/3867.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/11/3867" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mapping of the Sex Determining Region on Linkage Group 12 of Guppy (<em>Poecilia reticulata</em>)</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genetics%20of%20Sex" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genetics of Sex</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/955" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/955.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/955.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/955.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/955.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
