<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/889" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/889.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/889.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/889.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/889/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/889.atom" />
<meta name="HW.pisa" content="ggg;9/3/889" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" />
<meta name="DC.Identifier" content="10.1534/g3.118.200752" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Meng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7110440" />
<meta name="DC.Relation" content="10.1534/g3.119.400288" />
<meta name="DC.Description" content="Pedigree-derived relationships for individuals from an intercross of several lines cannot easily account for the segregation variance that is mainly caused by loci with alternative alleles fixed in different lines. However, when all founders are genotyped for a large number of markers, such relationships can be derived for descendants as expected genomic relationships conditional on the observed founder allele frequencies. A tabular method was derived in detail for autosomes and the X-chromosome. As a case study, we analyzed litter size and body weights at three different ages in an advanced mouse intercross (29 generations, total pedigree size 19,266) between a line selected for high litter size (FL1) and a highly inbred control line (DUKsi). Approximately 60% of the total genetic variance was due to segregation variance. Estimated heritability values were 0.20 (0.03), 0.34 (0.04), 0.23 (0.03), 0.41 (0.03) and 0.47 (0.02) for litter size, litter weight and body weight at ages of 21, 42 and 63 days, respectively (standard errors in brackets). These values were between 12% and 65% higher than observed in analyses that treated founders as unrelated. Fields of applications include experimental populations (selection experiments or advanced intercross lines) with a limited number of founders, which can be genotyped at a reasonable cost. In principle any number of founder lines can be treated. Additional genotypes from individuals in later generations can be combined into a joint relationship matrix by capitalizing on previously published approaches." />
<meta name="DC.Contributor" content="Jilun Meng" />
<meta name="DC.Contributor" content="Manfred Mayer" />
<meta name="DC.Contributor" content="Erika Wytrwat" />
<meta name="DC.Contributor" content="Martina Langhammer" />
<meta name="DC.Contributor" content="Norbert Reinsch" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Pedigree-derived relationships for individuals from an intercross of several lines cannot easily account for the segregation variance that is mainly caused by loci with alternative alleles fixed in different lines. However, when all founders are genotyped for a large number of markers, such relationships can be derived for descendants as expected genomic relationships conditional on the observed founder allele frequencies. A tabular method was derived in detail for autosomes and the X-chromosome. As a case study, we analyzed litter size and body weights at three different ages in an advanced mouse intercross (29 generations, total pedigree size 19,266) between a line selected for high litter size (FL1) and a highly inbred control line (DUKsi). Approximately 60% of the total genetic variance was due to segregation variance. Estimated heritability values were 0.20 (0.03), 0.34 (0.04), 0.23 (0.03), 0.41 (0.03) and 0.47 (0.02) for litter size, litter weight and body weight at ages of 21, 42 and 63 days, respectively (standard errors in brackets). These values were between 12% and 65% higher than observed in analyses that treated founders as unrelated. Fields of applications include experimental populations (selection experiments or advanced intercross lines) with a limited number of founders, which can be genotyped at a reasonable cost. In principle any number of founder lines can be treated. Additional genotypes from individuals in later generations can be combined into a joint relationship matrix by capitalizing on previously published approaches.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/889" />
<meta name="citation_id" content="9/3/889" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/889" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/889.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/889.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/889.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200752" />
<meta name="citation_pmid" content="30718274" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="889" />
<meta name="citation_lastpage" content="899" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jilun Meng" />
<meta name="citation_author_institution" content="Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Manfred Mayer" />
<meta name="citation_author_institution" content="Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Erika Wytrwat" />
<meta name="citation_author_institution" content="Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Martina Langhammer" />
<meta name="citation_author_institution" content="Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany" />
<meta name="citation_author" content="Norbert Reinsch" />
<meta name="citation_author_institution" content="Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany" />
<meta name="citation_author_email" content="reinsch@fbn-dummerstorf.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7435-4088" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=The Journal of reproduction and development;citation_journal_abbrev=J Reprod Dev;citation_author=H. Alm;citation_author=S. Kuhlmann;citation_author=M. Langhammer;citation_author=A. Tuchscherer;citation_author=H. Torner;citation_title=Occurrence of polyovular follicles in mouse lines selected for high fecundity.;citation_pages=449-453;citation_volume=56;citation_year=2010;citation_issue=4;citation_pmid=20519833;citation_doi=10.1262/jrd.09-224H" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=H. Bakker;citation_author=J. H. Wallinga;citation_author=R. D. Politiek;citation_title=Reproduction and Body Weight of Mice after Long-Term Selection for Large Litter Size;citation_pages=1572-1580;citation_volume=46;citation_year=1978;citation_issue=6;citation_doi=10.2527/jas1978.4661572x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. K. Beniwal;citation_author=I. M. Hastings;citation_author=R. Thompson;citation_author=W. G. Hill;citation_title=Estimation of changes in genetic parameters in selected lines of mice using REML with an animal model. 2. Body weight, body composition and litter size.;citation_pages=361-371;citation_volume=69;citation_year=1992;citation_pmid=1428953;citation_doi=10.1038/hdy.1992.136" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. E. Bradford;citation_title=SELECTION FOR LITTER SIZE IN MICE IN THE PRESENCE AND ABSENCE OF GONADOTROPIN TREATMENT;citation_pages=283-295;citation_volume=58;citation_year=1968;citation_issue=2;citation_pmid=5664389" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_title=Compatibility of pedigree-based and marker-based relationship matrices for single-step genetic evaluation.;citation_pages=37-37;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=23206367;citation_doi=10.1186/1297-9686-44-37" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. F. Christensen;citation_author=M. S. Lund;citation_title=Genomic prediction when some animals are not genotyped.;citation_pages=2-2;citation_volume=42;citation_year=2010;citation_pmid=20105297;citation_doi=10.1186/1297-9686-42-2" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J.-J. Colleau;citation_author=I. Palhière;citation_author=S. T. Rodríguez-Ramilo;citation_author=A. Legarra;citation_title=A fast indirect method to compute functions of genomic relationships concerning genotyped and ungenotyped individuals, for diversity management.;citation_pages=87;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0363-9" />
<meta name="citation_reference" content="citation_title=An introduction to population genetics theory;citation_author=J. F. Crow;citation_author=M. Kimura;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=D. Cruden;citation_title=THE COMPUTATION OF INBREEDING COEFFICIENTS: For Closed Populations;citation_pages=248-251;citation_volume=40;citation_year=1949;citation_issue=9;citation_pmid=18143392;citation_doi=10.1093/oxfordjournals.jhered.a106039" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Arch. Anim. Breed.;citation_author=G. Dietl;citation_author=M. Langhammer;citation_author=U. Renne;citation_title=Model simulations for genetic random drift in the outbred strain Fzt:DU.;citation_pages=595-604;citation_volume=47;citation_year=2004;citation_doi=10.5194/aab-47-595-2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. J. Eisen;citation_title=SINGLE-TRAIT AND ANTAGONISTIC INDEX SELECTION FOR LITTER SIZE AND BODY WEIGHT IN MICE;citation_pages=781-811;citation_volume=88;citation_year=1978;citation_issue=4;citation_pmid=17248819" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=L. O. Emik;citation_author=C. E. Terrill;citation_title=SYSTEMATIC PROCEDURES FOR CALCULATING INBREEDING COEFFICIENTS;citation_pages=51-55;citation_volume=40;citation_year=1949;citation_issue=2;citation_pmid=18116093;citation_doi=10.1093/oxfordjournals.jhered.a105986" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=D. S. Falconer;citation_title=Selection for large and small size in mice.;citation_pages=470-501;citation_volume=51;citation_year=1953;citation_doi=10.1007/BF02982939" />
<meta name="citation_reference" content="citation_journal_title=J. Cell. Comp. Physiol.;citation_author=D. S. Falconer;citation_title=The genetics of litter size in mice.;citation_pages=153-167;citation_volume=56;citation_year=1960;citation_pmid=13697995;citation_doi=10.1002/jcp.1030560414" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. L. Fernando;citation_author=M. Grossman;citation_title=Genetic evaluation with autosomal and X-chromosomal inheritance.;citation_pages=75-80;citation_volume=80;citation_year=1990;citation_doi=10.1007/BF00224018" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. A. Garcia-Baccino;citation_author=A. Legarra;citation_author=O. F. Christensen;citation_author=I. Misztal;citation_author=I. Pocrnic;citation_title=Metafounders are related to Fst fixation indices and reduce bias in single-step genomic evaluations.;citation_pages=34;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0309-2" />
<meta name="citation_reference" content="Gilmour, A. R., B. J. Gogel, B. R. Cullis, S. J. Welham, and R. Thompson, 2015 ASReml User Guide Release 4.1. VSN Int. Ltd Hemel Hempstead UK 375." />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. P. Gutiérrez;citation_author=B. Nieto;citation_author=P. Piqueras;citation_author=N. Ibáñez;citation_author=C. Salgado;citation_title=Genetic parameters for canalisation analysis of litter size and litter weight traits at birth in mice.;citation_pages=445-462;citation_volume=38;citation_year=2006;citation_issue=5;citation_pmid=16954039;citation_doi=10.1186/1297-9686-38-5-445" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=S. C. Heath;citation_author=G. Bulfield;citation_author=R. Thompson;citation_author=P. D. Keightley;citation_title=Rates of change of genetic parameters of body weight in selected mouse lines.;citation_pages=19-25;citation_volume=66;citation_year=1995;citation_issue=1;citation_pmid=8575666;citation_doi=10.1017/S0016672300034352" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Holt;citation_author=T. Meuwissen;citation_author=O. Vangen;citation_title=Long-term responses, changes in genetic variances and inbreeding depression from 122 generations of selection on increased litter size in mice.;citation_pages=199-209;citation_volume=122;citation_year=2005;citation_issue=3;citation_pmid=16130472;citation_doi=10.1111/j.1439-0388.2005.00526.x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Mol. Biol.;citation_author=M. G. Kramer;citation_author=T. T. Vaughn;citation_author=L. S. Pletscher;citation_author=K. King-Ellison;citation_author=E. Adams;citation_title=Genetic variation in body weight gain and composition in the intercross of Large (LG/J) and Small (SM/J) inbred strains of mice.;citation_pages=211-218;citation_volume=21;citation_year=1998;citation_doi=10.1590/S1415-47571998000200007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lande;citation_title=THE MINIMUM NUMBER OF GENES CONTRIBUTING TO QUANTITATIVE VARIATION BETWEEN AND WITHIN POPULATIONS;citation_pages=541-553;citation_volume=99;citation_year=1981;citation_issue=3-4;citation_pmid=7343418" />
<meta name="citation_reference" content="citation_journal_title=Reproduction;citation_journal_abbrev=Reproduction;citation_author=M. Langhammer;citation_author=M. Michaelis;citation_author=M. F. Hartmann;citation_author=S. A. Wudy;citation_author=A. Sobczak;citation_title=Reproductive performance primarily depends on the female genotype in a two-factorial breeding experiment using high-fertility mouse lines;citation_pages=361-368;citation_volume=153;citation_year=2017;citation_issue=3;citation_pmid=28096494;citation_doi=10.1530/REP-16-0434" />
<meta name="citation_reference" content="citation_journal_title=Reproduction;citation_author=M. Langhammer;citation_author=M. Michaelis;citation_author=A. Hoeflich;citation_author=A. Sobczak;citation_author=J. Schoen;citation_title=High-fertility phenotypes: two outbred mouse models exhibit substantially different molecular and physiological strategies warranting improved fertility.;citation_volume=147;citation_year=2014;citation_pmid=24248751;citation_doi=10.1530/REP-13-0425" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=A. Legarra;citation_title=Comparing estimates of genetic variance across different relationship models.;citation_pages=26-30;citation_volume=107;citation_year=2016;citation_doi=10.1016/j.tpb.2015.08.005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. Legarra;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=A relationship matrix including full pedigree and genomic information;citation_pages=4656-4663;citation_volume=92;citation_year=2009;citation_issue=9;citation_pmid=19700729;citation_doi=10.3168/jds.2009-2061" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=O. F. Christensen;citation_author=Z. G. Vitezica;citation_author=I. Aguilar;citation_author=I. Misztal;citation_title=Ancestral Relationships Using Metafounders: Finite Ancestral Populations and Across Population Relationships.;citation_volume=200;citation_year=2015;citation_pmid=25873631;citation_doi=10.1534/genetics.115.177014" />
<meta name="citation_reference" content="citation_title=First Course in Population Genetics;citation_author=C. C. Li;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. L. Lo;citation_author=R. L. Fernando;citation_author=M. Grossman;citation_title=Covariance between relatives in multibreed populations: additive model.;citation_pages=423-430;citation_volume=87;citation_year=1993;citation_pmid=24190314;citation_doi=10.1007/BF00215087" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=A. C. Peripato;citation_author=R. A. D. Brito;citation_author=S. R. Matioli;citation_author=L. S. Pletscher;citation_author=T. T. Vaughn;citation_title=Epistasis affecting litter size in mice.;citation_pages=593-602;citation_volume=17;citation_year=2004;citation_issue=3;citation_pmid=15149402;citation_doi=10.1111/j.1420-9101.2004.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Perry;citation_author=S. Palmer;citation_author=A. Gabriel;citation_author=A. Ashworth;citation_title=A Short Pseudoautosomal Region in Laboratory Mice;citation_pages=1826-1832;citation_volume=11;citation_year=2001;citation_issue=11;citation_pmid=11691846;citation_doi=10.1101/gr.203001" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Pszczola;citation_author=T. Strabel;citation_author=H. A. Mulder;citation_author=M. P. L. Calus;citation_title=Reliability of direct genomic values for animals with different relationships within and to the reference population.;citation_pages=389-400;citation_volume=95;citation_year=2012;citation_issue=1;citation_pmid=22192218;citation_doi=10.3168/jds.2011-4338" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Rönnegård;citation_author=F. Besnier;citation_author=Ö. Carlborg;citation_title=An Improved Method for Quantitative Trait Loci Detection and Identification of Within-Line Segregation in F2 Intercross Designs;citation_pages=2315-2326;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430952;citation_doi=10.1534/genetics.107.083162" />
<meta name="citation_reference" content="citation_title=Matrix algebra useful for statistics;citation_author=S. R. Searle;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=S. G. Self;citation_author=K.-Y. Liang;citation_title=Asymptotic Properties of Maximum Likelihood Estimators and Likelihood Ratio Tests under Nonstandard Conditions.;citation_pages=605-610;citation_volume=82;citation_year=1987;citation_doi=10.1080/01621459.1987.10478472" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. Tuchscherer;citation_author=M. Mayer;citation_author=N. Reinsch;citation_title=Identification of gametes and treatment of linear dependencies in the gametic QTL-relationship matrix and its inverse.;citation_pages=621-642;citation_volume=36;citation_year=2004;citation_issue=6;citation_pmid=15496284;citation_doi=10.1186/1297-9686-36-6-621" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_title=Evolution and Selection of Quantitative Traits;citation_author=B. Walsh;citation_author=M. Lynch;citation_year=2018;citation_doi=10.1093/oso/9780198830870.001.0001" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Stat.;citation_author=A. Wiencierz;citation_author=S. Greven;citation_author=H. Küchenhoff;citation_title=Restricted likelihood ratio testing in linear mixed models with general error covariance structure.;citation_pages=1718-1734;citation_volume=5;citation_year=2011;citation_doi=10.1214/11-EJS654" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=S. P. Wilson;citation_author=H. D. Goodale;citation_author=W. H. Kyle;citation_author=E. F. Godfrey;citation_title=Long Term Selection for Body Weight in Mice;citation_pages=228-234;citation_volume=62;citation_year=1971;citation_issue=4;citation_pmid=5137356;citation_doi=10.1093/oxfordjournals.jhered.a108156" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. A. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Pedigree-derived relationships for individuals from an intercross of several lines cannot easily account for the segregation variance that is mainly caused by loci with alternative alleles fixed in different lines. However, when all founders are genotyped for a large number of markers, such relationships can be derived for descendants as expected genomic relationships conditional on the observed founder allele frequencies. A tabular method was derived in detail for autosomes and the X-chromosome. As a case study, we analyzed litter size and body weights at three different ages in an advanced mouse intercross (29 generations, total pedigree size 19,266) between a line selected for high litter size (FL1) and a highly inbred control line (DUKsi). Approximately 60% of the total genetic variance was due to segregation variance. Estimated heritability values were 0.20 (0.03), 0.34 (0.04), 0.23 (0.03), 0.41 (0.03) and 0.47 (0.02) for litter size, litter weight and body weight at ages of 21, 42 and 63 days, respectively (standard errors in brackets). These values were between 12% and 65% higher than observed in analyses that treated founders as unrelated. Fields of applications include experimental populations (selection experiments or advanced intercross lines) with a limited number of founders, which can be genotyped at a reasonable cost. In principle any number of founder lines can be treated. Additional genotypes from individuals in later generations can be combined into a joint relationship matrix by capitalizing on previously published approaches." />
<meta name="og-title" property="og:title" content="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/889" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Pedigree-derived relationships for individuals from an intercross of several lines cannot easily account for the segregation variance that is mainly caused by loci with alternative alleles fixed in different lines. However, when all founders are genotyped for a large number of markers, such relationships can be derived for descendants as expected genomic relationships conditional on the observed founder allele frequencies. A tabular method was derived in detail for autosomes and the X-chromosome. As a case study, we analyzed litter size and body weights at three different ages in an advanced mouse intercross (29 generations, total pedigree size 19,266) between a line selected for high litter size (FL1) and a highly inbred control line (DUKsi). Approximately 60% of the total genetic variance was due to segregation variance. Estimated heritability values were 0.20 (0.03), 0.34 (0.04), 0.23 (0.03), 0.41 (0.03) and 0.47 (0.02) for litter size, litter weight and body weight at ages of 21, 42 and 63 days, respectively (standard errors in brackets). These values were between 12% and 65% higher than observed in analyses that treated founders as unrelated. Fields of applications include experimental populations (selection experiments or advanced intercross lines) with a limited number of founders, which can be genotyped at a reasonable cost. In principle any number of founder lines can be treated. Additional genotypes from individuals in later generations can be combined into a joint relationship matrix by capitalizing on previously published approaches." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Turning Observed Founder Alleles into Expected Relationships in an Intercross Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rFTJ4GLR4VaabbqYuWQmfsqWNZKJUrgKIfQYCuof8b0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492531","apath":"\/ggg\/9\/3\/889.atom","pisa":"ggg;9\/3\/889","ac":{"\/ggg\/9\/3\/889.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/889.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/889"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/889"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/889":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TizhhP\\\\\\\/j1EzhRpVwr2iIj5l\\\\\\\/YHz73ArWb7fTrIrRO6UYJTrR175jvjufv+F36854cv1podAg\\\\\\\/MATx2BL8ZDeHR7fXJby4mwRTpEe\\\\\\\/qCFv2yqVU920vrQjOPnhjwaImvXWIu0G5DmxwAQtvNFfiAiYbKc1rafEAhgtmaUmJo7zsuMfRxXa4ff85xi4Mstyhu3lJkU\\\\\\\/Fw4rz8Y4ycJ+tQb05eht1FHJMRXTQdaAemL2wWnDs+I44EgqsfR2QgfbSOAetQMCVtDaagISsNCjCk6eFdBrC515daT1EHSARH15pFo2cy\\\\\\\/alJPrQ+i9dC3x19eQFHIrYiQgGxgz0GA0C\\\\\\\/r0BpKzdKYEQHdArUQyvonnxjLFMqNRkkQ6p99sMiJO8\\\\\\\/vOYMdA7ACpZOi7Qudy\\\\\\\/fvWLMowhMXPOVpYM86d2mg2EteJlaJ4nMyb\\\\\\\/3ixBRF0BfnoYemSu3utv3bTTkX1mxrMp0TzMwP5Ijk2d2uyGLLJjOTZWmUBP\\\\\\\/UKp0GWKTUO7EZw3AlUD25yPjFs8gSwaqp9yIo19toIxHTC9BUxAlAeNrv\\\\\\\/9hSUuRLw99plt3d9JjQHGuku9TuazyAdGJe+AchOWXnodN8Jupzl6b9YkTe4rHelvAFJhvNjKsTaA6+e9CuS63kvZKKWeZSGtDuwCrTg7S4i\\\\\\\/r+lLRcEsLiDNktV8+CZginJqvZDiDJ7sqAlvmAJaKeAZ3\\\\\\\/OMv\\\\\\\/mnLQE1wEi3j0kYd\\\\\\\/2\\\\\\\/gncbKktmMCfzbMK36KxCtDw0OEa7OpBW\\\\\\\/nz1lAL2ufdHMiQ+NxfOsw761215JO9JNzl6gzJsKpV1S2KIM5KhIh\\\\\\\/E8z6HsKUjznFY5U71lEjSCVfulnEUhCDcXjpnmcCrG2xLhk0cKSaJ\\\\\\\/n6pHm9RMd4rKVTXOZAO\\\\\\\/qB+wbATn85PGnRNOCFDbdh9WGJB9FaJYaBOdjMjUbck1byx7pedmhaevMrrzSUyx+BTlim0s1SYPXY+ZBNFyhfu5R4rlOrFcZlXHg+We6JTvbqbY3C\\\\\\\/9WouuOmJEV6pbyVDsfE688vVzITVOIcluYHOubf9Dnj6Qb\\\\\\\/hJoZJC8mGfXsJpN27cC8k1jt8F\\\\\\\/hzXj\\\\\\\/iUMsHF0748EP4E3\\\\\\\/Kfwem7KsG5LMB07ECZNHVm3g99cc2HYNOCfcq2JUCvPMOwEwbF81Mk197Gb7n9+x+H9noluPcM3UOhoyupyI3fSFkmgtZb3gdElxzDD9ttFy+S2wlxU8z9Ah5K4CQKmeOBQtG2C54q+wvqdLDDN9Iw+zeXg1Aqe\\\\\\\/eD2VGWocbUm5N3t\\\u0022,\\\u0022iv\\\u0022:\\\u00220bBc\\\\\\\/p4ZKrE4yPpoXmOaXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aabcdf2b5031f1343b7836f4d881ea60d38698b2579dd19689ef132756fdae26\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229LEVmyK9vDMFBItJQSTkwbYinp3IBaW+EUY2eISZZV7HWhu8rZGLQSsL7F6zxfYvelwjb4X5JIbjYlPEvnrjSiVN67FOWXB3anvyzj09IZPxauBjLLTPnyIuDziWRC+cBgJqnEC5d8EBZ8AK6HtcTyC\\\\\\\/4LeGP\\\\\\\/j8Zf0bXnTUCS\\\\\\\/vJjZPpNwOZ3TsovTVZw8pBny7pQazeEjVqJy00ESz0Ewa3SXo09mVHnDWm5Q+oEU07DBEEe+hVIuRDKFePVl\\\\\\\/\\\\\\\/ksMJt\\\\\\\/9Nk1Rd+u7IF1NTsul0DvxpUP9rmcqCnTKrhLLKpjf43HBb6INDXhKo\\\\\\\/DoXqt5L5JtP61fxk7f0T+5YRogXYWEHkG0GG\\\\\\\/Uh7eRJUlbvQ2Z64cjGWY+1TFaunib3ZGHqVfbblUudK8\\\\\\\/0f8y8RPQsvcE5udXp6HKHluKfTLEREe\\\\\\\/arP9ZBfdAfalNfRMQ9v2pG7P4lRu2dGui56dDd73EUXFnqlNmr\\\\\\\/uXS3ENppW9MSV+fp52bI22cy3pKHklVE4SKG6bYYz5WVw4ZIv2ETjmoZa7sfwbqUBXY+sbz2uYjjNlKgRPGhc\\\\\\\/vzCilMFcb7dLibV6BBnxmomlEpm1c4tAoYcwsg7Mf74IodVTVxW9dj8cJNkJcAOz53zCjPjXK8Zxiq6zWAhAJiO42IwTgesIizwsFR0Awvt0M3pCs\\\\\\\/e+5ndYTKUxqUENGjQusQT+b4Ds128\\\\\\\/GMlUTPfDcw9AYdUHmF445I4vu2Akw49SOak03FYGIMKrUjSMkexgqt7gL32vfVe+0VDtfsMWY8pzbRlTa9WebD303lER6eOcKVBIo9T3U+sD1FgNhGlIBJFPPeEdD9ZC4wOX53jET3R\\\\\\\/Le6hvb1jzh64V0Gi8paO5S+uOXws5EzkGDkv3Et\\\\\\\/tyNuc48Bk8sUuLoXW49yl7iJ6rlynK3v0+5PAYJ0Br9NDBW23sp8WFo05wlKlfelKP\\\\\\\/jSkia+KApMQDmBnxOU+5IClhjGsqxg\\\\\\\/L6N7\\\\\\\/sdzK+eHMyy9OjoYRmfzJtP4AMmD2LUxpcIoAY9b9RzXm4BAp+lxo0HQogDFYJOkFGNKFdo3yt0TK22fW4IvMnVY65Wa\\\\\\\/rLo1dn7cHya3yVfcX1aQBehqVcNJW3tHmLSRE346BIM9GZvM91O5GDysY8SrLn5lmw81j9O\\\\\\\/6EuDiih2ZjBb16daxdvxlJ1OTxAhQlkWNq6PRNO4QbPa+OYqbU5Kb\\\\\\\/XnuMo7NRPyRSFmIGgNZEsOFf6Hhg7Ti07Xzads5vAuCkfT2oEaI31ZEMDDWBJ6Rw7+BLV1jtjGpGA12bKWwYxP2hL2Tmff09yyyDem1a\\\\\\\/DoJzNWMELFHa5KlDfayJmquKsps1qrqouQuUE8+AiKhrh7VbFhRX1e2yEUWoiZwd7+W3Zd3FLiwj6svogxZnV4D7iEFvBM7r4KdaiMFhfwSkPH+1jnUgn7YpHbHnh+fEqHaojNGXcntjn99iJ2E8q+WFHFKeaMXq4lpVGxrvuYCUkOoGNNZvzvcQj3rzpqGh2YVkhLWw6vbhfMIO2r3jyH542XQEKntlMW8FctBuS13IE+\\\\\\\/gQfNWbUfoA4al+0sy8jfNMcmeJsYsjx8OC14ceFzB0xaqvRQwI9tp1LagDAurAN6UiKhcjYIzs+7cEKJaJ1YZlIDw5DPyqJFVEKFiOJWKxPG0LubH5YBM4YhfZ50Nzf0NoJjWdcv\\\\\\\/Kad6uACFskwUcbCrthrYr+KcGZVANF1\\\\\\\/JytQrABwTH3Dee\\\\\\\/1jlzTlpoX9EAR5DfyIj1YMMgSMR6USurLmZ2Why1Wd5epsJ+R6j6CZWQvmZghi1pDKAV6dBFYIclxgdghzuEQcy4xJOEkZqSoD+YZGM7KAw9LFyrXaW12HHI8zKNf+BEUSnY+jNjJ95Ieas5Gwkcrc4cH1qkGsUXNoTh3wcG2dKu4ywr6PUUCiW0zobv3bRUtXJEWbkAr0hgylo+3e7fialf+pFlBA39PBKWQ2HMr2yF0khQ41W8kcRJYeHMSxnOv5TLDaT1xWn\\\\\\\/e5wvflKxzATjh8ppkH0D8EOViJIIM4UaRgiKrFi0I2vcZOaVfAIjx1LBLThDEZm+teyZJSopW6CPNONuoKWgtP82FzwhamsFxdEm+3FyPPHxMO7INMBVewsfCvYAA2upl4U3V6up1HDL63PVa34KrvGzix509mNrMMwHpccuvAymuqmHRRmLgeP3MZEsGMVtrQhYcGG7cTaPZoMTCsGj3hu75ot+5XK3AO2CFzczJLZi1uWvxVi2cfhts5\\\\\\\/qxVGjF\\\\\\\/O8uG3lqOgcMxOi2RyX2rkwUb12cIRomSJn+k08ZWR+C2r985UfzaRpG2VCkfg\\\\\\\/OW+CiVeRlMQxB2lfR7paWVlloVt2VOizIkJMxLSBe90NVoDgwQzqfL6JUS9dHyNLGz3Ch5CRApo5pWgPoOqlPZV\\\\\\\/f5OoIGZ0SKLxU2g03fGo2NomTwrj4mrAOV6lgA9pdae3XKfABj\\\\\\\/WJipvrc3xyD6ftHL0pm2Relsd4XvggoT42IAT0kndNFq1hSOayu2q\\\\\\\/IOWk+Qw6rntdV0pqgIV8w3FrX7KG4xxkdNo1hYS9O4Qio3FB+ELA2Pjf1t5atBxNQp4rYO7npUgKEdLO2Ja8t7s\\\\\\\/8xIFNlGLY8qoF6MYVdGu5fLS+7xQbvkMlePGZ5xeKn\\\\\\\/6IL6AKMKMGuRy8JNrqFkWN+\\\\\\\/w4LXaF0UqQtmho6bJLzTXqVoOSthVmsmg7sLr8GZaZhTyirERKg2hMG2VNjiNZAVrl\\\\\\\/T3Q\\\\\\\/BexLshQJCTMDm0KeMuJVC4obL8mllhV\\\\\\\/K7kmhByAoJBJdCyBgY0rodkQTZJZahXWAOjhIok0Y4uDe7Q1B3C98ibEp\\\\\\\/IYo+4\\\\\\\/pWLqp4Qa6ra2i5UJSm3R7qn+PkKTvRG+0PaxTEC2Gy3MtBwdlTS6x9I54ygyLecnMsnAx\\\\\\\/5sGkO6n\\\\\\\/jkGn\\\\\\\/u0Id0ggalXLo+0Woddx6q98JzrbkJTtlQRgVXb9OAb794ql57OXtOLMSaBZDDNdOSSiz6gdiupm5bEjMNTbyiOax04ZHOVrT9Z18kTtRqy4kFNK5sppbFpyeDsOWERY\\\\\\\/huzpZuvirTH7MH4fnifGCKDKEHswxQFZ45iSJ+u4y6nlvreQbfG2sdUBPzxHrSMekX3W95U+681jwmGiZ3gsNLvDDX2byyjiqKmx\\\\\\\/GSqviezhVhQzKC1nuFnhuAP16lFCuvjzL3iQWsFogt0+TdhNuu3KLsYARgl6id7\\\\\\\/gMfPwcesRyyA+saVUiWT4UsbrckvQQl6u\\\\\\\/kWMvYPFKykhf+pxsBfVYd2gIC39oNUtc2nhUSXp1N10qICBObh+LX48HpcjcQKhMx94iV3QOH1XPAd3qFWpoeMEAjYfj14JgUjTwOrwbYNR7L\\\\\\\/uCiJ4YBWrpHvKbr0aoWOGHW8rTs5LfmlurQ3jiiUHWBtvPqMbGo1MNGjZWPsQPei0LjsQhcVAM\\\\\\\/mtPFpTXwomOiBV6I4VCy28xCGCUtL18W9zrZmN4u57Imb3lw+Q\\\\\\\/1FgXNcf0e18Sld7zBn6fT3ZxZWpE6UHbZYmIpxEIXDcmNz6x2B4SgRHd9emPqr3E10tFBh60OrmU+zLViMbK5YpLFy8ajfS6t3jvqJlD3DIdwkeotS74cnV1P+L5z+B+R\\\\\\\/5KwLh0IBOQSLGvYAOCvfPQmPwkWtZpF1uk65hP\\\\\\\/zBmOAcpKqetmWRhVJcxbegieu1jN43naHnE9Zp01B9y5baO8IJKW+JRE4dIpFoCb09F84MfruP7hmPZi3zCDFoFNXpfgaf17aDed1u9Ce\\\\\\\/YjU7OTM9VnkwoZTWdSQO5oMcbiSJXF0\\\\\\\/pNO+PYFwrBuK7F+30x2u6j4ekfaPT6eYiGKR3HBkyVZVjorHxmXpvmUW+OAMij0728OzgMdoccCQmRDfTJ3PcLKOMVD\\\\\\\/AuVMqaUC2T6wCCPf6zdof3v1YsTRdnbxw4Uxfukve\\\\\\\/v\\\\\\\/Urr4mBSSJJPUzxuF6ZV7GFW8qEGzHrR+bWDNboZtLYQYbLPeoARWJjQl+y2vKzj8BcHReRbUNgGdS6yrP5sj4OM5sClzGy0cAKtuxjxkSs+u+wxkqlxe+Bket5ed0fleDZGTwrrwd6ejUiUdbwxF9E4ASpbYBgJtv+gvlFmuHx4s+MLKg3GAW7I0rakBle\\\\\\\/LQw0vhsBPt\\\\\\\/y5FFvNHzbDZPW0Q8h\\\\\\\/bg0MLK2QBUm6GzSHT+ZLoA3TMgUbcbrSeVWtCkAR9hwKigheD2i4545hhvdK7VoecKHOIXi9oJHpaSZB8Mlvg6SvaAXZ45eXqglOCf3sXfXBk3ksfzzSilQp0ehvz+OTSofztt2uuPFBT0XfOHuVRH7\\\\\\\/E0mZkhrFG5sHHqt7XL71ByQJpnnlHbBZSDe5sR6ukdFhk8+J5H3efIFy4C2QzOJ4MnROIcbshD\\\\\\\/m+UKbt1O5\\\\\\\/UaFvV84UQhwX2L\\\\\\\/aSpuuQEIsM6lLub+DyepSibSMcoCzw\\\\\\\/NjNDAk9foaD5cEiaJRkQZ5P9McmOLUjns7+SNuBt0fL4FEiu3RVHhYKtmsCp9IXJyetbx\\\\\\\/fJmsH6iU5Q89cdyHdXBX420td79Z0ohZJxCdDFJRLn6r\\\\\\\/ythBrjIpXUyu4aAU\\\\\\\/TbWVUDSrYMV1sU8DR\\\\\\\/Nd3H27Vl2SiC7cSomy8F9+4uaCFJLIKLytylFRpsrl\\\\\\\/bO0CcpklgxjwcZwtxc7g8oAI7UNeFy6M30uJVq1NUrJtp7eR1TBUon\\\\\\\/8Xb3Vs04699JhywDprDX2n8LBF3JG5MuK3f6WghLXeq3IXhlS2MDIfS5R6Qn1ClEOykAsOawajJnmKQ+QMMk84KAbZDcsfjHcbpEj41FlW4x6G0iudj7p36FOR0XWFvoH594d7DF0Tx+1+Au+e5c7Ggw+DLo7sTHAYSHGnfCt42dVLMulmdR\\\\\\\/l9Xly9AIkVXdREpKsH7lrcS4Xm8lDKmlMFV8ckV+2RSyVZIdQ4xi0vO14Q\\\\\\\/3O5j93aZ8YWhf9qYupWeKbq0qxcmBArfx9IRot\\\\\\\/et\\\\\\\/bflToBja1myeLhVDhY6wTbaJvRvdpwboYzYtlG847LJccXpYHS0KTW6DEKU5PkpUbRLWm2iMtfiOkqDssGzC0BJe5x81QmmK0HZ+xL4gyYRqG0J4QjQI6FVQORyiLbxC9ClDx\\\\\\\/pss3dUGeKnrNtP7lNe1yfi4qFm6\\\\\\\/uKMEYZTflAE4bpIcYXJtC45c5S3Z2jnL3elGTXGbQi9sbYd2hSb9J0rz9sNAE9wvnqiuMLQe7bq85G20MF0XWDzKtM1qJlX0XoRLSw7y3iXEP6RAJ3XZfRJwFC+RDA4rvCh74ATTxWNCRkhlYrxfHzrj91Dmy7nRnf2lDVn32j0\\\\\\\/Q0pMiK80xCcO1tg+VrWDMDANf0NJpP8x023PKjwWZgvR+FLchH0yxjQNnBMRiTP5R0FCV0395QLBVmtUYIgrW26OHjWneLQDHqj9O9Zp6mscoWNAt0qDHzFuDSjN0F8iQZRFLlcrnRijtHon0sqgyAJoXhytRjOuc0ENFhpiT4YDWT5Zi+weO5\\\\\\\/3wN4fBlq5D3gDKEgwDTPmnDo3mjjLripwzKZBrALqrAWDJEurcmkM92\\\\\\\/1MzYeb1hNIN9gZ5todo9qp6aKVFXO7vPK4mO5FMB93SJusD7fuqLYx1tUTCjprIOERuax8RtE54kzpJqkPn2AneevXfHmJ9n0xfXKUEy\\\\\\\/n2S7oltNfTB4QsDWjnF0pljhhdvXP7bep66M9UulqxIIhyAnNvGZRF14UzGi53OfU9ptWtHvciqeSus\\\\\\\/TD2ksCK3r9J0lU0yh17MzPPfa3eWwt11mTlWL99tnB8x+uJ9BbANaguVk5ZRHv+3VXC6T8yt6QEslmhk1V6gwpex72nDLfog5YRiCxcZMJ0rfWRRprIAiHDwZlSIsc\\\\\\\/gBuutPZE1j\\\\\\\/w4j+7E0ZpfUPYoh7FcyPtEdZiKBowNcAwP\\\\\\\/SfBQHRwipUhQaCYirziQyMoIMGPIqEgSayb0hC2m6IfsWMopA8VserVufGauutCDxlAmeHaphG2aMk0XRCB2UroQMvRlkT\\\\\\\/LJjXpZsZ5o+INjvCs+R\\\\\\\/xgnyIX+G3akJqUQcX12PWbJxw82rJSpEV6qiyEYwdb7Za6FOgRT0I3MEPt8IU0ScLoa47yWx3iEXEnHrXK73pD5PcrSGfyzCbRX9TDbMWUdaXKGB4qmcqn1J8NL4116BGsxd5IPb6FJ860J1bPFUwzUH+YY7ew+XJ\\\\\\\/EDaqA\\\\\\\/az926lTIFZaqZjhpMG9lm4W5n6Slu63Wl\\\\\\\/\\\\\\\/sziC46AdPPjLFrlSpGcI\\\\\\\/4FrzWZnZ1FTLzidH+qG\\\\\\\/TgxuRfImVrIIQRfJdlFG\\\\\\\/0Fg\\\\\\\/Ju48vbJbKKJKXJceTfjKNi2FsV1jntUjECfCczARYVf21nMar16prycfArsAvD5MuUdHgRh4DwJ2L8KsewKDq9SRjuRUvLN2CBF\\\\\\\/GD9\\\\\\\/vVqt\\\\\\\/+H71X6t0OWqAOK2I30ZQzX1sLNIPvsYe6+N0Diz9YOnFK2GiJYSdh24Vxj7Uid+m\\\\\\\/R7DsnqNePoVJG5PSESS7A+UbKnOihC2AdXWyD\\\\\\\/t\\\\\\\/lNwk8jvloJFJpCwGPzYB+bSEwAKPn19OE3\\\\\\\/cU3iEuCoG38PM2Qw2r9mku7Do+9jWtYIGdIu4UtX4UAgIwlJLHD6DbtDja8sxzEECnyl8ZKWj18G8QZ\\\\\\\/Cv29nx\\\\\\\/917I5wH6LWMFUhfnSAjInQss1GNHieIMWPIzP6narE2ITLgf8uU1031Di9vf\\\\\\\/aTJO5eOPqbhPvfJOlYoLC4qV1A3mnfWl6FixO+VhjJLz9BFX8qPBTbTYnSJFCncWSpCODqFcTafwPWaeQlpUB6FD0iY0D0FMvlzfnLWHYkelly2Z0Q7a5LUo+LPFA8tV6q00LD0io9UP10aH1ZLlGIse3\\\\\\\/gQbt11iW9cPtW63PIqh5\\\\\\\/dbURpn+5HUUZVlLwxlaIxLRi+4x+25+64FbrkZ3vbNGRxGPxQuGZOQ7GtkIIJXTx0\\\\\\\/L3mu6XeTZsamT25f6n3tIR39KYDObocCgICbMjPIVd9sKXuUSCL3R3OaY1swCdKj1k7tSW9ZqVixpgVNTtp1hjr8juXoddG\\\\\\\/XGZ1ebkwb4BbE9lpHxMFmQgAb360t7UnLUVESYZuKAyeexliOhQDfXnv8ftkHTOKzGnzFXiyzXkTERyI0LHdc1\\\\\\\/bJMkJwSKnTA\\\\\\\/o99gGeWZMO5bF1B8ZQnuzCf9XDHC0LfgXs1aCxHse7rjTjLI9SIPMerEdFYbvvbRvdm8JM7ANEDDY3GnCMwwyQ8kLx\\\\\\\/czNg+oG+qHJZ7kIoisLFxRfYtUOhTWssAQrY03HuqUrfq0plMN8mib7LDlKxSA8tefMSV8mxBKwEEVXqg\\\\\\\/F\\\\\\\/6o6mXu8xzmwrw6ruMS8Xpm4KIDtCydjGEMINxR4uz1l6ESmvFi6Jemjg2pXOuMk6Wh\\\\\\\/ukvtXnX5bEySkfctI5BcXmuj\\\\\\\/RoD3Cl6KvmXbAbjmxJTxuGpaEBItMoxuMr0ixlnzer3YmVprnqvdzNjxFHEml9u4PmMuTzOSj3QCCyUPRvVO5cQrMhSJ5C9ozdyV0ekAOvyWvc\\\\\\\/cOAoKx55XiGisz\\\\\\\/EDcSd7gEtPGqxJJr3rekGZ1+ClLscmqzLNqMATtjUrGr\\\\\\\/DPah69ZDz6HgII0zSU9Ora4vpXwXcYR8ggg8UXyFS7bSe7kvswvCnLsNkACiukmCl\\\\\\\/r3dkgJ0FpkjPi9agq2ST2iJGhaWh1ypu0kQ3kYLzUEc+Z9PKrnY6sGyMn6fZg6xi3X+rCEbqoG18F3X7iQgPgzGJzoXBZwQ2hU8u8kPrxUwC42+VDRcDx6jo6++j5eUaq7E2rJCzCK4ojaSP4wjHmYdij95P\\\\\\\/3SocKrNZHgXS3h7Nig6B7mZQnt0PCkdC9Y+uVWOln2s+Xjvoav8bs4gDghJS0lfwOymoN70UyN2wr\\\\\\\/Py3HdfNGtq3xfT6myKzbgQjWG+B4Qc7sezcQEQssyeaYVZyIyGbG3uhVxj1uisCMpcWsUdGIFE9qC4cQ3DTs1Ob0BC42\\\\\\\/Y2KKRbYAJSpT0Wkw8zndu4zqHJ9Xq3kNfOwTCtTpSQSzMXBNehFlzeXoylxp5N3jMvtZTQoDh6FMcQthLtg2vHSdv2t5UFh48R\\\\\\\/zjnMRKPMkj1dE9pvtHe0bGTbg+3Y2Qg\\\\\\\/WYlDKMZRArdJkWEA+EADXPxYV\\\\\\\/0+Vd9ss4hkZc3WWtOjvUfhbhriZC7C23b8gvz25XKKZqri62IZnjhQ9quo+EyzE4oWitwdCUnGnaZqL00fgn20dYEGlumNQp05JE\\\\\\\/RVzV7bYGWecCuqFF\\\\\\\/68uGxM6QB+FXgXveH0EGk+fWmHbmu5mfZV3xqSRQuJ8ux7Ycq4Fe+pwZtvuYcZGPFx1oECx0R4DyYTMLcyt5r10qx\\\\\\\/TqzWqL\\\\\\\/4C40WjnZggokhP4SbkPHHeKgeHblsX8dDDtNnjCDiJSiE8yz4Eaw43DMRUnGVNcdK7Fj5LBEhvdQR9BtY5NAGVdJPA6\\\\\\\/\\\\\\\/+ehPzmcWur+v+1MeWOxRGEzHlOCSrQUd3GFEel7Rxj\\\\\\\/qRbBA2qITv2zr4AW5OheO6pC0X9xCwkqJ089kZlJD3SFUKtSn9Lar+Sv6Db+cgM3h1eZ+1xP6OR+OpAY+6EA4wZmmel16dCX\\\\\\\/MUxAuC8JV4IyZ3tzj1\\\\\\\/dTowsrulSsYNcveFwGPUeBzkbeKVtDN9ntIayvPZ9Z2UTcpwYSZRYANUroT+kg5Bc\\\\\\\/T1mSlRzxze\\\\\\\/vpcf6CpXuQBKODE\\\\\\\/nbrGsGfk4hLg+T7YeJiA3Bf345xReiZtqVJGiR7kp1uc+YMULdxUCj3yt\\\\\\\/KBB5nCRcv1sTsJoCU3ZZa50Zx0shhlhkPyjv8FZoMn6TYOpqJnKbQ7rQkl6HHffRypyzpuH1DEjJmnuzOkhd8RuNpQfF37ORDSbC+Gdj9muKS5baDBF\\\\\\\/BxARd8HnSVwsU4ewX0+VYj7XMPHPvl+d0xKDaSkMG1WlyNdWGVsXd7pPl1dLjOcR7x5i24VhdQhFFsxnIm9UBvORKqMNebelulfzssCa7ZrwesfYcCa7wKnSaYB3i6paN9cMauBlvn\\\\\\\/kqCV3zsUiEI8myU7PYOjQWe43avDcGlO\\\\\\\/6APqbiH\\\\\\\/RRlkr\\\\\\\/k5PtpMsj0CnwOrrBI4s4FsBcZgre8VajYrM\\\\\\\/CeiSSBInRspnughTJ2WlUlyDKEKK7D29WUqlM10t2dNjYRkCE3PR\\\\\\\/09hDkVIouB3CWk\\\\\\\/kbOkfFOj24N0x4jnpiAiyti+IX8+TzkAVFv96csuKEH31gZq6JB\\\\\\\/GYwQGBgTbfBwywF9VoL2b+x3M7UK0mhxc4sWJBzZPTLETfLyolfuwNONOQFxLXEoVeYqGR8ASbKLEQKi3hjaf1XSWzOB3VfMLhO4NumIutRph9DwLbwYerIajCBiLe+9yXA\\\\\\\/SYk+mkTp7rbAQgPznGfDRUf2RsMVyQmFgHFhbZPG\\\\\\\/JF5eAyX4YeUmPBcBtgGyejyLunXaHRqGp7LwpKwsJ1aD3zYhESBfdBXxZZuLGdVWK7r3ey79zflVQEqwJJ7WcMqKbQHxRzw1Sg+n2wtnKH4WFgkaaGxZI80d0FodFFlBj7IyLut8BH\\\\\\\/xviTkhV0H6j2p7R86atL8rdG\\\\\\\/M2TOYzo1xQGrv5VYp\\\\\\\/DH86ZkgydNnV4NWSjXABTDR13jPr4OmCUPVzrBA1GWM19Th7\\\\\\\/bgRmCqXi441UKobTHc4jpRc8eZLttCZf\\\\\\\/JIWl8teoPSPwRPO6FCwZNuNx8Zy\\\\\\\/UcZT5qezNQYJvmx8PlCIqPXZmp7fNE5uVY\\\\\\\/2wrEkFXFlC08QPdDGsWXDsnz5aXGYr4DmpfQCs\\\\\\\/fVm3S5rzmxp1YBSWOLvE7uMyFvuLKFyoul7WLfgNhBwXV4pkG\\\\\\\/5yDUUqWLnGSVBR3UpaT6DozCQLBzmsrX3\\\\\\\/oxZtslVfWGPDgUA7vb2oAI1i3yFmbUzUk0AdYPro30U2TW1g0cFKqDlunDvinRNJDlZdfdGJdAmhHA1ysOiP3\\\\\\\/8UtReoC6t+Flm98cLBWOAn8vs9kCVSntAJIoO\\\\\\\/r9x2AhXX0PE9CedPjUVznoDiI3nrUGK+glGHsgt84suzinNVe35J8dyp2x2iSUyin7Hvxr0Ju8iBPrBw6CVwUbst41XmABJiQnw24c\\\\\\\/7EE48ZNIFS\\\\\\\/rTJvu39ON6GoTQSggwXy32LDj94LGyAeKS8Cf7cebYTcvUU5MT5LrbGQYvxCTRMPx44rPSoIbTsbngV1ed8rB7ZdrKx5kImGG6h+gSLZMvnq7WM6tRBaj4vkVnn0P0SC+25aJDMByAQbL0KV5J9X+Mvs5QcGBiCVL3dnWF0boVF1WGM16JSRgDRj3HV70AFLVdHVZExmSjPwTbDKJY7BGniiI0jdb3VTGAFIgwX2xrWgrB2r0cY6+N2uNMGVUylDROkJ3pv9+gvmJ2ArW8UOFssDkbZAr\\\\\\\/zkjSSMsaq423g6PaVKMGJdVPOW207JT0ay0GNP9drmET5f\\\\\\\/YiTQ6vr9vCJuCDGVqFk5WQC7FuDzWQ9dDgoybub2KF6tmX+0IztZ8LyylU8xWDhil7S7+rAv4Hzx\\\\\\\/VT0jqqfF3Bm8DEVu+ymTXHdsVh+MS3keIjrd\\\\\\\/2SaNS1ImB3VAGD5KFKlFOKEbt2i+yS6OIypF2BF5I1UU4Sc7FkYY73uNTXoYI2IX593DJD\\\\\\\/tpqRdBKMl60bLx13NSt+SpuacRx7SIVCG+E2mKLZAZAePmqJzh8At+zUlWalNF+u8oFJ84oTH6vf3Wjj0vYOZ2hHhZ9gZsATUiECyinfgisUSVlJOgxXwuxFHvISX2pnBg8rQ+sBIMErWzSsgsyCQ5BiD7M95HLrAVsLJ9\\\\\\\/vpC0eu\\\\\\\/dbLF3ouoZoE3SYOv7p4xNEsVqMMlsSURhmKGpc2AtszQ5pfRmdHRrOpCV9hVt+qxNAte2epMxrpSNG7iMuo1LYXKvS4iEP2P9\\\\\\\/VPaQBDU1pmLW4PF4yloKmOI1GZXY8tONfD65f+7\\\\\\\/X7wYu1nm30wr6VgS1nJRu1E18MPK0oM4k5lJPGNHyjFJoTYqgMnjrBAZhQF2Gb3pCsQfCWbX46Mb9JRXxXFQvGaqQq++oEJe5W1bi8x7VKYCB3fkVlWxXpeVqiSb+HHkAWs4+N1+4DRaMArX\\\\\\\/ApugrSAZ2aladJPt\\\\\\\/xuLWyNfluAEJlhYkQbypAQ9WsamZJZCzsqlGwT\\\\\\\/axA++xVVJLFRXFhSwLFk5GOlbIMxhYiu0sLBsH9+4MgIBZ1IWuxnvp5S9n2dZmlADAwwvNOQU4HHALoY2Cago9CdXBFHMDdz1iLhehDSa3Vj8fLeaDGyOeWiXu+vpUwZ6k+FOYiMAZWu9bBpAYg8jiOQSS6Sz+aw\\\\\\\/A4aFAB0rgJzOPQ8yo5r+gcbpaHVtpHjBRJshwPcfORWDPz07jZWfJYTIN8jokDVIl\\\\\\\/tHcEjRT7kkOzmy\\\\\\\/Vc\\\\\\\/cy1a+x0QFHcfKMhGnzVffbC9+JpgmufriIJ3SAYj+ACvM7vCzxT95pRBt3hQGF3p2AA\\\\\\\/7jNbk\\\\\\\/+YuAtCf1UEAl6umP7kWiN5pPhcnWpipYvnPerr+6zfv+CAgQZGE\\\\\\\/abpivWQIpEJ5Ip2kWhm4IUO7Rq\\\\\\\/BElmmmNLfPd2D\\\\\\\/BY\\\\\\\/ylL2zgQPEZuQbh2c+kvl6BTyLjT6xYgL03H8TBONnSEPZNP5lnfCn+bf3pJ1Rp8nXtqaiiOTzhV8SDEJgNqRDMGxdY9E71VZWfGOinabBE5CrbXN+qPMF3T+UiNg9fkUnyHRMklkRWjsTRc0jrggDBX1h\\\\\\\/l6aYyvZiU84KzdDvIn+zoBLdcziVX07TxtEJlZZC72RI3KJbCbf8dFhXD5FFWvF5CVmxMioaXOo9dUQ6px\\\\\\\/cK8h7vjfJ41jnRafFQcv89XuLC+wdotHG7DkGVMPhxPx73m9SggoIgrWpdqbKb0mQWDFYv\\\\\\\/vJG6E+XBKfCh2dk3Bzl2QLMxmanistsGHlAAbCmmNFDC3s1sBfzxEfmPDDheeMTvjz9aIqCeW5ZYa\\\\\\\/DSSrYV0oWt0VH31Ru5IRx1RCBTNTU24vcY17JeUdmzUpuHX974ZMPYkA3YlSDWVPKYCaWkrM6vGsCvj1NNOgSaawQNIej9CcWu7jJHrF3oYtJm1XTjadbSot6DCFBTnSVrSS03WYrQjXbeJlm1znI+y1gtR6AcjYwCE7HzHm4vthoFKyN3nZ60jzcs3JdpKOTbrmoWSIVEsMIVYSDLt+FIZPOAtCBkWak0yY80xgODw4D+imZ8E\\\\\\\/DtREByQnuEqn9F1Ad0st6eb26kV+Fs8PFj2Fvt2KGz8e08+JMVwXlqQsnbTiYzYzn9MCewwXikHWm6kj\\\\\\\/UeQUgwmgalZocOdd0loAEE\\\\\\\/D32PIB7DxafVTNeHdpXAuTozIIc0zjvVHbixLo92WTSfyqc3Q0ytSxPcYJnQcYMWiGGW16ag8QLN31WGjrl3ABBrNaoHKt3aW8JORn\\\\\\\/yZeut3EXolbu9thefVDbQ\\\\\\\/8cb2arVLWH2hA18V5cp3SK9G5fzQdbQk5K4pbw0riVpweF3dc9qUdfxMFYop0CbirpIca1GoE1mmSHVt14gQ7VITYgLm4TWgqAO+rZHvQZt0SZGXBicO7qlH3nFgrByasw9LTuJAVBK8JML3OGN28HVn\\\\\\\/uiFLqFh71aMU2RUNLsok+ohp\\\\\\\/NRC0aSqiAg3nRaoAK0Wts1NkfrcPDLIbqRMSgHq8y9qgVRncZKjrLs3OY8rsh7TKPLPm1SMuxP+Zv14MDjhpiEN6IeIw4zRnU66z2XNbd5OpwKzOCxUOl4sXLJMkUz7eQOR\\\\\\\/3vHeta2q3uYNbYOVUd2fNrFh64ow2zcdfGbgmTdjJ55qnu\\\\\\\/++iEfzyWlC5FEdvQPd0VilUlA4yUInGvdCM4HnXMWGuJL8i7BnkjBqFAO5whjrIKZVPUCsAOWSn898XpDUZZYETuI0oQPNyx9bD\\\\\\\/K3i5R6zhxn1sbgyF1yO15yS6jIVHDplZ1HFWGnMZlAYwEvj6Ud+KdutEgQyGGQ\\\\\\\/ECI28c08+YVqxf624Ip1e6ZpcAj26120sI0gQ\\\\\\\/kiDqTjwBMuJLmzmuq4PqpthJxjPeupEV8JkZG6lQWvNnJwX8gEykKOLUMQHLpj4Oi565MLtrTEpDlfNWqz7WuZNEdY9ttnyON2M7StOZ77Z3eq4BnNCCU3DOKkWR+RQZpXj0w\\\\\\\/C0AgMHIKZKHZtdQzblFL3gBXysl73Y\\\\\\\/Qe6PS5wxTPsppiv5HTYoA8T88O+JmBXpNmrnXgJz8uK9nF79H7ivvRC7mQppv5pVySDoS2cQha8H6hU2kjlJIrns1eXlmlmiiJQ4FNtDTAccgSMWsb7GUsYdJXcgnXW6cIYjHqngNCYSla\\\\\\\/DwhgtlsJKLzAA4EhEzWmXsWFh03Pwg4ZLfOrOZctyxE0tCkQ+WFBALiLYTzgrIesPlYWzs6y9sXCih\\\\\\\/FPoOggKtodj1sEA94xVpDn\\\\\\\/PDuErelVOBBcWLNvpBTGQ4RBr+fMMVsxM7ziMN5u6zwbmEBqZScP+0mkqqmnszzPmad5adgrkGqVNqbHQlDHN2mIDxEczGxeOQ5cwzDzMZ1YR7w26SrqvEqb9yC3AhPk8UqD7epJ3CkJTBLdhunURFy\\\\\\\/tzp5pBALr8aNLjTaEUKJB6R7iGT0noQ957qE+oqLuGIZ3j7CQOdj9abLRPRt1sg23tGc2rJ9uzBJbwE9VwAZljA8a0Vd+U9MVNG9lcxtux0GtVyxvSiZ\\\\\\\/b1RW8kiyTnIqV6EHH72RNS+DkWb8UL6Cn2RKalYtL3PQNDgCdAvoM967+0eFPoZhyyq6kEB34Zt9NK7LXT7WYhL1h+2qTmubqOF4P9wLu\\\\\\\/hl51DtOGi2CZ7asIgS1i+tlozTtY81D2VXF+J30Pw4RItbBGwqkpCrhv\\\\\\\/q8G5XtqD1s\\\\\\\/uIcyQUNZnwZ5V+pTf9C7Hg+x4x8nNySV7ZDPYbcRef49CzbFUZx4rCoFdXHOneqXjYNu4vXyvE7ca2GWWt61s1MtsCQhyC9agU+25dhp40OLQltMnaQJFL1C6TDs6v0uZbTBvyqpzz3CF2y3yb1+ZHudUywfBRoCz70fhKoq1yQcK5\\\\\\\/szzuHsCdeobsVWuGcmJrR3iBVH\\\\\\\/IvfsOR6WMZ18rA2HHIqhzYr9iLx8TcwatYJJ7YfqMvelFH4Q2WDN6sXwkP5HbW0fRioghB3TU7\\\\\\\/LGkalyLrIVMgGfHe5io\\\\\\\/ZGyv\\\\\\\/ny3VPYP1xqheayJdoSBi9ku+Kju2MFZDl8TCurS3gkhS7DgE3jSbZ7rVAKDiGtWVC7gcN5Zld60Nk5fPBQyQr5Kuq2nIOaCzpq5vMqiSzlMg5mDaA87xtPFCyIpxSqICLLGZV2+HBaJVBAKtfw3j\\\\\\\/iMtDaO6zkW7dJLYm6G1TcycNgD1dZBSVY4CNEdqZMIU83GcGeYAte2Gh7JJTdwh95XozFzmCDCUFaQe52\\\\\\\/WqrNoy\\\\\\\/TSlsqKb3NYdvOfAiy5ie7L6mxrb+gBRdLQMP+Nq+BLVBZcm9Kz\\\\\\\/Y+0euzacX31LB7nIiM5sly+WHm51b2+fVyQ7MedlL2yQX3hiw9iAZOKDvI9oiy4q2Ht0mIkNGrAmc5jTo2YyqEirbNmNAMJjgWAQEY\\\\\\\/qwwqDrvK7DLFWNg53gXtE98GkloYspL+mTnyPSdumrL3mgszWHr8\\\\\\\/Hwi+r4efE8+zZfERgfxMxyaoGoHc+\\\\\\\/RccgvyAHAeDIspWCX8Asq\\\\\\\/BiHB7l7JSfv0maZkhMX4uFE6\\\\\\\/moSRe78viA1of9f95xzZul3z8Id1p92XMDfyjiqryKLgOh4\\\\\\\/tmETMOr0m6uTMMagzllVTnlNGPZQc3wGuIP5rCSyZX4H\\\\\\\/MrIJiBRoIhV5qMbL\\\\\\\/q+6XABkfDD04CskC8XPfcIkq1dhUSV6hI6H4e\\\\\\\/pOv5gQa89AJiwZ7A\\\\\\\/AUWh0ujaLjHs3tYPyb+SEErb7vbyOqVDmuYU4IMtcgV4nbhpaSF7JP0hM3kDiSW51\\\\\\\/SoaauhAtBYlYfFsNNxu3edrb4YWMPOM3GXpff5ptSx+FoKpaq6EOih37nPI6luCRxBDLVZqY4Paqe1tFujXXsWqZxacz5Pffl9GLobBU2eMiOHTSXUzPTZlAf4ieMKNN\\\\\\\/4tWp3L\\\\\\\/xVOCGhH+RF\\\\\\\/nZ\\\\\\\/hf+fHpSfy3brOpMwevigTkWpLyLq7XKhnXVmmG72jjq7NImTOrLvMEOZstvq0OzvvILqfYT06mxBecqx0f96edEjWLDZl2XIDMkqM5NK9TtZXv\\\\\\\/IlML7FTPYvY3dLJqAAAE94cQF\\\\\\\/tUqivx21iB0FmK2NsEB8NXXR+hOEKAtd39\\\\\\\/c5ADOAr2S8GGSTqg4gfC56cJKs58gdUXC2tKnbM6DCt4uTTjLbyMfaRTTsF1uu90hcldQdIVD5rRwWzBxqYNnxOVEwXUVMSQ+T+c48J5NZi3AsNUF6xFZfFqYaeWIF0I1mtU\\\\\\\/wzjkC2T55KpC9MFQD3SN\\\\\\\/rkTODVZk7DsmINEhI9FApDRer3wPqAean7y1M5jZNWKhq44eZerkI+1pY8KdB\\\\\\\/hQjcUrmGtz9c5BSLYbuB4XuQy5l58SHr7JjoOtmJ71PLVDSZSTgFfrZpk5Qs5i7TyYcDdP6UOJEeJ6zaY21xkA8h5h72\\\\\\\/FwuZa2TrgatisBzGFtFnEnGCMgANRRDsJiG74cHJXg2i3MUwpU64Yx6SiGoEzxLSvf6skkgMGN5Pj3Dz2B1TX3p2NPvTWy27AjsnTqnhRoAnLNHvHwPUd5KvmulUkqB36Jx9Rj0c0FN8hFMvNTcmPBfOYAn0XZJCdAsJtV8b\\\\\\\/SCTwcWmBxMH80m48jQUmeQti9fmy0kitK3uVTL\\\\\\\/LaGuSYJWd7+dp7qiIkBfPBxw5oxH\\\\\\\/\\\\\\\/rAMoBGY6ooy4Js6qWUEESAKokCgWt\\\\\\\/4osD9VsP87oC+CQ7czTuF91WiUH4TH8yYwRg2pZiAqLiBjAjy3zAUW169M\\\\\\\/SB3AlKl7J38INLj61zA93g0qPydooaDXJ\\\\\\\/ARPcYjkUbggrN8A2vLZFV5jVGYIcxLYOijn9nF0xHUWIR+r0hW2RWhoB0q0VBweFmYW4xbldVyGg\\\\\\\/g0LOgLM5jQnJE7CShTyfjdvqS6eQl6tU\\\\\\\/KIhTEXm5QyQlgwXbUjLYpWlCCwDTpkOp5rZYEkIz3lWCXG7NYJuhzoSiOH5fgrIywdkDbqr4PlVhmLOcjIUIOB3qj7xjBdrMMXlOSnZmMxNO4N5Tg\\\\\\\/vLDZ4W3G2XBRNJg7NrYK5IXJRtTutZDvFrjOeSQHNH8qTuJuPjNeztLZ19owmwPxLh\\\\\\\/ApaodcDceIdqXuqIDKtZG1CS4kpAPe0o3XqACTAapra19A2qrshCZZatyZjVRW0m9hPVPCC5ypAv4ee\\\\\\\/o5LECOUpGWmHwgdDHXMPCFJMJSQTJiaRPifa6GNzgw6FzxCmI3f0uoOnQsRHjiXol4ZqOGuGF\\\\\\\/3l5u\\\\\\\/bRUHeCyGBpxT8IL1qkuoKm1lD\\\\\\\/+NAFEcg7WZjLX5+bGEEXNIOj36pWGJaR7lqWUsL8FsduZbZYq5uy1dsLsXqyicT+SAdmZ+NfYlOzCzExZ3zWM+OgQOHqhNA5xex6lLFcpKcy7qTABRTMjjkYYJhmZRiJITYF6Lrt83ARcOC5A0fW82EzdWovqadNs7QYf27Z\\\\\\\/nb\\\\\\\/FZT9Gs91H2PRjiuJfrHKHNuIIVJ\\\\\\\/b0BMqk9GMOemOOZAkAvh50bH4XB5lMW6TDxktiCurWPNhWsrZPLEJuD05ReYO603RurBjRvsv00y6V2634LAMcj5rVreG1j+1strLn9lCNrbxj\\\\\\\/pgnUT3fJnIPCrFYfGODRXC0Tg4PyThAtbMvsd7hGHr34pJcTALbAoXPiCIxxU7WZZIe23dcGHhHXAx1GtiRtjfnUYUjhDt2FLnrHLxQLHJrCi3TH5QFaL2jK+Ou1mi6Z5G0HnKdC9Ux2LLtIjyXRG9Vz7kmBbSLqT704XJ3k9dYJM9WJEqj9ukNiQfVap+LE43njCCBgtUqgV6nrtmhe4ZXNi1O4aZ8a8uTL8v2vX2hNFBJFRehpcqEidwCamV6+UcBor+U3nETKv0hzejMGLcI\\\\\\\/BDfye+KVyH0SxOGIfUCdePSz8Su0UoaI4xAkRVBWuZFeCeu0TPJZF3\\\\\\\/\\\\\\\/zd0bkXp0UlD1+LvPzmeuvBReu++8mREsPyUUm1Gx04ukWTZv3bKZTwb3b9rP9Y1Dmz0zkgwVwvZ+NaGfXDOPzP23h53eGgogYPkMdFej0aFOob1\\\\\\\/x9D9H\\\\\\\/Y2y5guECIsYJW4r2iY2Vui90JGHxRx9HWt8CX6qd1o6qPyUHsiKA2Hw7wUrWdVuUpoTODVcjqq9GSKnHGUai2WWbBl346R5Q+pkuxrZPgyRwoBC2RGj3nqXyTGGea\\\\\\\/HKQfLnZZh\\\\\\\/qMPZ2B61x7yjGkSc+MzEs+ALZzenSwYAW5BRK0s04S+MwWiPep7zxYbcqczzDv3M2SJTYOFEdqfX9cfRo0zUCsKfufJ1qvA\\\\\\\/zo2cpL+3BigIjMrOAQ2mOkZxzIdht8nOvy86do3CgHTyROQDPsmDOM0LYHCzVlWjKnx\\\\\\\/fcsiqa5CD7LyXocUcN\\\\\\\/KIaoi5lm0L9eQulUsLZf7nOsYKwQqdAUzrnuhoOlS51dsvXP4g5owEBkzDeTOWMLGtq8BSWtHJiYiV1NKCYT8LXdxYBV5t4OMW8stsCg72aS0\\\\\\\/tfm+3+DBe+zH3RiUXCMVuoQ9mygQeh0kbx7pmlZjU1dnnAr4lh\\\\\\\/mH5DaKS3UoWFyqAh\\\\\\\/nLV8KdPKyEM3\\\\\\\/SioMUod+p8DqW4sBgMIaX8U9bnr\\\\\\\/iW96DJXRlczc8uQ7y1aUZFo\\\\\\\/89GUXwcXUQx\\\\\\\/Xq0pFTRuBvKjXO\\\\\\\/0ciTr1fj2lICHzx+BzLgPs17Z3EHKIsVrpG1LPaWe3pVr1Te8rZQjxpztTDhkUzQRW+oz3f8TjNcFcmcVR1XwUg89loPPL7+Q7tl6NUFwcTDaGLmENzvhxcfQ5757\\\\\\\/ckaSWxOqeT66\\\\\\\/EK+vvLJLxzvcgApLdk9c6fr53EM3OPmf\\\\\\\/cUYT9FspGjUdIFID5L7g4UOg0JxlMfRforPWf3LxYPvFP1FNSq2w451VioEjXICXHOhCiovfsbysxm8a6shpiEUc7vbon9s2+NMNfjXluRGRpIL+lvp4x\\\\\\\/UFNkH3mbu340YTg58yBNV5fI590VsQCBtYN8m\\\\\\\/BXsU77Z7X7ths9UZraMLTXdFMxVYGQWvzmSgwrJPeI1FJMkzBc7jlFXHhPemGedF74cJj3LeGu5jrKwCTsYOKKDuK3BD58DlqqEQ5fA1njlnYGzJwB97l44Cjb58iA7Hp717JETPI8VfxluL+LirXmY4n4dHtn5JRql42E9hogtAt4WxiUn3+eIIg+D8J1F+MhnPj8Mt9bhyK1OlrS+XxLx8kY7901fDcllNERlUHmcFOwxLxAbjh+K1wZ9bQBnDLp9fV92\\\\\\\/CH1beVR0vqtvHcAojfUI9cgxTuRTFbg3RTSP\\\\\\\/1aBQDxNyvD9zRo18RuuAIE9ulUemBgSnSltcGI35jiyRJqIyzTQQE3nw8Wl\\\\\\\/B4+Foann7t4DQJ57qsTnzGRExehTv2Rw+E+OGVV6O\\\\\\\/A9wenFAYrialPGg2ZTmSWSN9LWZ+IkYPRBmaN7YG2ANc7mYjyThTnVokOtCCe4mUxazHJzkM0DbHI7qF3YGN8eU+v5RKc0ScVZk9jdbzOqq71xyuxO5eNgxliZt94h7h0Qg41P6hhegM2Mw3K1VnOfBq+bFlGkwIxbyHt5ZTnb563WEuEcu\\\\\\\/YrJZucIVgJJvirIaBPZsM5rTRWith2jwyA0ByRZ1tQ+\\\\\\\/7+Gj8df9GhH\\\\\\\/0g+ieTr2iciHVWuchrLAk6EoLlcobQGb3j29NLFaANJeVNTZFAI0p73Yum9zAPrYeNT4G0qSW76IfVy\\\\\\\/cE8G\\\\\\\/4Mh28YlA66h62UcnXqXeg\\\\\\\/S6FgDDdGrED9bDrMHuCXNrkgn+L20+BzyDKVmI+Eo\\\\\\\/DqNUvuD2Zcz9rWmm5n7Oc6DEVa8XqEnQI+b6G0OF3XBO99ZBX77LN5k6LZdu1OiYwrp67G9PKKURW7vfPfe2G6pwkH12CzipoDdAvoAFhjcNdSVFUdyFhq9JiOinwTzaGxuQgHIPHJ37FQ2qY0MMjmmn6nKC2snCPTRH6qEYxU851jXFmIfJBHsPAu+2b8nxTOukB1rbF2FpZtP\\\\\\\/z3AD6mipnRVcBJBevrERHPp+NEjyww+wlTkULR5FQuhjxGblAGFFFgnlyDIKfhyMAiiKehV7YzW148if5xSMYF8P2QQI7w6KnbgQt+hOwENkn+cDnzOKVsY7PIViYh5TUMQ+I9jtkmBoX8r\\\\\\\/6qTfmGZ5tx61EUeaVkfumrf23da1qykqOKA5sECZt\\\\\\\/ErZ17UtVCTGLDu8QFZOlH750A3DIpeVUVevougsbMj1bV5RnWV9\\\\\\\/i2PEsIFS\\\\\\\/JUJSFMdpXGuT9JXm4ZF9mnRdmM8AKW+7QTLnhUxxxvRd6QPbJufQ5FACbnhuaesLoClY1etME3+dWUTf3PBWbOSWTF9rfrw5YT7\\\\\\\/+JjYFwG3GhjNHuZZIX+8uxzorM1wBUg4TmkxjOoH7qXNz1Yc5iV\\\\\\\/wxquLBL\\\\\\\/sD9dqPO9stC3uYg01H8PfLlrCgKQPp7vs9qRYud3fnle0bKyi0dj5J8RtPNEgvXbi9t1lRy6qHjLEfaVjabPcv5h6tNibN7eDLbZNgiks9mI\\\\\\\/kS3gKlD5zKJrf9e5iu1LsOvrdGbAec9X+ExWNKe8K3dTCKEXKTmgEXjHzEMYC+KKHM5Y7xiXIcM\\\\\\\/VD\\\\\\\/0F7qAnykq7HQfmb\\\\\\\/AuI7JdTb4qzkiroYi\\\\\\\/o5z1z3YpKArR\\\\\\\/5gA8xQbbuAHzHUn+9gdz2\\\\\\\/an2fKgKLUaDH1eLwYSW\\\\\\\/P7yKNkHbQN6hzbQCdbaBoMVTo2ygsfSlXLWX4mpDvjAD7lKLpUWMvCBCUq3pf7gDQlUAo0yJwyLqIShL\\\\\\\/7XM6X1T+d6KTkZVwbZ4S4rbZYCTizgQ3vV8pJLXISZxPbodK8ZiHBLzAqlTAEiUCOSBnl45rcnvgEpY+ZV0WQHT5FVvpTPEIA6bp33Ri03MHA1aVd7tY8O4cIaLavvWmI\\\\\\\/4FK+Omx7oYbZtc4wFfoog66YqHMl6Ck9BxLgEdCosHM2Co5Pan3Kunl1UEpL6sWhMQIsGIOlnzYyOQ5LBH330eLgOsp\\\\\\\/5HRz5KX4SRgViQoNIPU6E+CYyxUlsvGwIcar\\\\\\\/G8k2AWr9MAdKzrY7uix2zgsBXS6RDZbxSeyZL8fR5qQJlyb8Hsy+KT5hM6UJYq0nMjJ0Yh\\\\\\\/mPeXxYGaOQX3NHpbOBIxlonGIk4d\\\\\\\/JG68qtZL6A+2FygDktZjul1d2QQNpEDB5Z3Ty4L7XWDUnWiAJbVh004g5ETrCvU6CHsfVkmTriWL3b9WkhWlm7xGDofYBT8\\\\\\\/X1Ik3yuOjxi+AMdtU0TywhdqYVwWr0IqbIz9BQjmE+gvVKJXvOTv\\\\\\\/rO1dD3YFndjnHT8Gl9v9meQ7MpOtOrAFl3Nn18M0zOk3cOPuwzkQgxl7NzE15MFU8FYNuxYX9TAGXUvSfGZebulr8IZzPZ7EC5b8+yfc4K6WbEfoWpVKRrTPItiaxGyuo+BYQb8ojoi29zfIIQvD9Rm+z18zN2iwQjqN0O4Xh7xJeRzwyydFEhdbWxtITrayQL3XLwrp\\\\\\\/Q8Xhyq9QkclWOLIKLP8BEPcUdTY5S5CGDfBNoJqS6I5Xe0s7BNlFObEqJxs60qhDmB4s0fAxkOSIkwfM\\\\\\\/U+DFv8MyW7JDWkdXRrtMQLj6OS7WJr4sk+pzEeqDl4W1fz0+jVGmaYENCQyDEpSGSJLe5wOBrOW70g6CzBd0BGuSZTtmHUC6qz3SQ3CJP2\\\\\\\/XzmEZhSFk2uNce3VID1xIsSnjXTIB13Vv4lXMo3nPZcMwgTkKCZUDxrtFX6sr3oEogB+J9xn0OAJlHu4\\\\\\\/9k0Gm0o19REFWRGFatYddNugXLacM8ROMPO+RjFm4\\\\\\\/bhaDubl51RYTGsKL5yvnAHDqO8XYliDZaz9hRLKYstSmLLRJaT1Xch72iPxGgTrFblIcf8Dd0+jobFJQdIZs\\\\\\\/bYIkfDFKBUiBc\\\\\\\/Cjnz22yZZieQwJ3jTTf\\\\\\\/xe9cZo35\\\\\\\/MC1XcSVwh0XrfEN7cAHwoNNWPylSglB8RAwBubaE5PIF5iw3lmWsr8a7EdVuxfYTmxtov8YrvM2kAlE7EtzBztST0Cs6nCaKomwb94jZYO9SVWqzKk6oSEw5QljETwDXVYvz1sgfpJZMwYs3bNPEonoGvY1XoqHYeW2yOq3ps7Wkglh0xTNQ2UpRkLK8H\\\\\\\/uIBuDvekm45IfwDrR+jsv2eOLZ6GFw8ulRKu95lE5conpfBPRJPx4nqZG7s9DIdpCWTgKr4oUfI63uRQtJrnlBTvkBVtamAC80MKz7kQngNi3FQp07LbiK1r2BSfePkra2XmdcsTEuSTy1NMC4O6rWtw98DsPdJIpbD1Zu3GzFCTBfHMI2tWDWCDUbPavNj3GAKaug\\\\\\\/5VEdzk2mHG67L6PbSmPj\\\\\\\/86Hzbvc25SbUp0vGKbsGvFfxZTLq2qjp8BG76RMdE55rfsq+opv5dq8jjsvpqKVXIkyygMueXiwM3RCYe\\\\\\\/\\\\\\\/ukIj1wLe9JN3dJ7KQCMA4viaUL4O+2MJ+ErS8Lw9pbnpCO5MKS7367pFUm8\\\\\\\/oF8Al1yarkfYheofhpRl6UTQjV6kWrzA+BV8akR74NZ1cywaAXRlTp4HBsu0AW9yNSEqN2LGk4\\\\\\\/m+QTA3dqEV0D6RMIuOYa\\\\\\\/7jDzvzZ6hnFaZdy2QkmTDkC9ie6aGU44wkDZ7ysf9sS3ysCT1D9V7MPa+HkHPcvJdmyCcLyjwX1tIy9khkAaalrWZqj8NqMC8uheIS\\\\\\\/WtIGqpgOpYnPs3J9H\\\\\\\/VZXHOFWsL2anBdJtQoUsdubFZ0giqWj+3ImjjlRtWdrJMvSlnYDm0NKbZxVca2V1TxfpIBZjmQK3mh6MpYh7C6NxzhiNjvBD4fcNeOAk2GdRQbprN62PYUsR4AZCozAdwVgR4Mqqq7PiNsjQIP2rR0Jkw+18GHUvW71HwbIb3lZqvFpO+BopRe2m9865Zi5SXzayo6\\\\\\\/kGnZFx0QDbGGj+aN59a4av9hVTFxxT\\\\\\\/Cfs4Ncc7kNpl98Te0fohoutnjouUu7akMMlZkXQ7oMCoBZjBSLjIng+P0l7E+JSC2ljiWor+dQVKJhw+yvp\\\\\\\/v3csUZzWgvLUrf6AlJ0kEV\\\\\\\/zyc8LlOCiSvluuhTEmjG1xixVT57XZVdDLVWsp9jXmKiSGOYSa8+ZF1IsIvH8SLtf5i\\\\\\\/L8KQeeNtouCwaUPEe6Y9zgsR6tll9FELxHGXjY8iYbI77XDBnUErX3VIUIz7T2fWKZrR4q4I1TReA4Pe8Kj0EtF5Rbnn7kHLhJeqXHtwEGKnsov1qO8AlIEbeInXaUlYX98NQQHEchjiokCdeVTzkTNSPoTSZ\\\\\\\/jgg5TpgnnFsOrKLvX+Ke\\\\\\\/FhIp16DfXkGKLY6MZepR5pRYnW43zb2DJWq0ZIRUvB1grJ2SQmdVPuGONOlexzskzwafls7Cxd3x9ol6kLh+5feYLDDMJwl+XXPs3z0fgOpQ6XfqhEA79DWvG5y5o4O+7G6\\\\\\\/F9Y7RSpVntw44D2qF9hfhLYSapK+9vlMTFBCPO3Z0R2nebQ2LDWEa5Jvs0Q+Ah2Gnl8\\\\\\\/UhTJB9+BQf0HOTAv2r3D6wV4sT765qrXo\\\\\\\/NmqxvBRNSMX+3xdZ138dr82pI1PKXAOj6iQqvxwn3y7e8O2gbVQ\\\\\\\/3lrPhXBi6ryYauyFBtkpLbUl\\\\\\\/4Hdz2aBLoaES9hucWDCY\\\\\\\/uZFoNmpygdQGzPI\\\\\\\/V4KwNSzkdC7IF2lM\\\\\\\/e0TR\\\\\\\/jgl+YCa9DDiv0Cvd1sgArwVc7pLvQQmIauQYGDki1bm4asqBLiOjv3bTS7P1cRy5TPshKUWkktF0KlDptalGiXrmc3es+4h52Dsg479\\\\\\\/BBm+VcLuwBWYc1Zd\\\\\\\/xSP+z8lrKcRHAwOHTzTyVliqp67RQuadMQ\\\\\\\/kcgcWAyaCQMqRxst+yTbfLpuaiEpv2+vJXi0KV2X7AqwsWFlyqFPC99iUo0XnMDy72HlKBhIvBfgVCuN+ivy5UM4AukN1A1eQnxijyKiI5oiHpaqOfWd\\\\\\\/2itwLR6GvsOgcjsbqeewiKx\\\\\\\/CPN+9+n3G8eewKnO43zqdIYCpR+oHHDvN\\\\\\\/jiVxGhRtQsQ1bstlPyfDg7rJcl4yUzhdYjCPQR9ZVbgCKs17KlsKSNXoRGiStIS\\\\\\\/WSJAIEfPsjNb1d5rcPt8GEVnG0lFugsvFDp5rG\\\\\\\/h\\\\\\\/0V1BNxNoVKCujIoQYdBegKH79QurSMIclUPOzabObqbomiTzhW52sa8EbM2YOOM4Io0ABBqTMmbUIFSoDJSYTuEuhPDd2WjurLt6VbkHEYPYPhbXhfFkQJ0Lnth0mzj0MIppyHWvYcManKxdtgtVSqieNfdmijghWkYaU\\\\\\\/n78CuJiSyBAhrmEVQ7zHF0kcDFAoXC1b1o3dZtMNuWiVHterkw453VvgioX4A4Hi\\\\\\\/tKsrWuzZUVhfW0okQH5hnhlOiFftml0LufoEjgmoLGLV00e7idUoC8LIBlaXZE4E2w2yAbIt8GDdJiXqDfCh0pI6y4HUYkbkw62OyWm1BvUC5EUf\\\\\\\/HoryJlZk4BkvCvAgNLnCe40hWeJTD3z0EXlzFH3maTRyNjhmb4Gcd3o0g6\\\\\\\/K1hcybelwYz+V6m4HdQth1uurcx3\\\\\\\/U9zu8XLlQMJd\\\\\\\/xGmWhDLtKsGVsV5lbQg+DFaQJhHjLUKNUgLjRXYPc0\\\\\\\/OTWxnqq84N4cTdAc0b9w\\\\\\\/eH4+jHblp5KXaiehZyzTZPbaN5H9JOiH\\\\\\\/L7U2Pi3zGsbVgXp+rSeAlk0WWABWPzU93TNgashOTjMESM0H94h9lU6W2tAQPh54KXMYA1UrfM5s3i907biemd70KtUaE0KWoHvx4rlXGvV3AwWkk7nkRcp7BxCKcWC+Ate96DrHhsmp\\\\\\\/+G\\\\\\\/3XZZ\\\\\\\/9XCfqPblgeZMXavL8zKT4TODilQaOLWNbQpeplZINB8cxHGhfA9PAW5fJQO29ZSn4JBBNbtfvEsZ3V2apMqSwRhllkWJQMjW7m07k+NJmXHlzUw6ygHMpfndINCPWiJpGSybt1WhnWRzvHektDbWwVWoFhMvQnkfTvfkWRmqnPtF\\\\\\\/sBhof+pLJhTntcQelHZpv+aiL31NdDRc+Kjk\\\\\\\/LVyXgjTeYZjn5Jo6fRLS8F8NUvu8lAW10d1RXMmO8fFiWyo+IxDH7geZyTmhI+AdG3KHjESuMu4ds2qgRVBkycVEoBLD7YDduYgiA12XBRO\\\\\\\/B4COI0\\\\\\\/2Nl+8xkdgWOMjJ3uyU4topwxSOsk6HbmllLXKKLrGFtCzvyTvWI2qaqbRSEDd8Nu\\\\\\\/sqngeTDnmJIMEpgr3AAMZUiFLakjmkcC2ZiJguBjzt+S3Uamq24OZiCdv36XCkue0g1aJm7P+GprgxvBuCcVMyuMVM1dQEfiR+3BEaPS62XoRKo7\\\\\\\/EaeaSffT0Va30mn5adY5dL2UFwve1T8l+iF3uAAZa989G5lWCaya3BU6Ur+iwwuMcBwdfL0GUXPT4nb87wXAXe8dOJiapWTQXcdIwgbpiEHVhzL7ZhX\\\\\\\/3EGESIGlTisdeAkwoCp0uVOxsyCvGATVsNGmvvgQr8sRmXsvkrnGXU1lMt2YrN9MZxHgPji7OOy7EhJ1D\\\\\\\/tWnZigl1WOQ\\\\\\\/8CU4GsLImY0+OAcTjkhFyPCwdswZa9qBnojblV2bYnGF6KmxDOEY8QFvhnvPJtv1ZOly23sDkC2njHdD2ZrtS11EmmaosDrMusjTap1SPhCRoVCg9VG\\\\\\\/Z3mCeg1DRHcQLnHy\\\\\\\/facuT6BVMTwquRXa4ewSQDHkOmfkXMR3jBLeveVA5xZEf9nc4b\\\\\\\/fxTB47\\\\\\\/yNs0w+LaWJnYCxIDsZ\\\\\\\/WqOmd+z5V2A4++dl9VckYMFf7q5z2NqSByMZgbbDS4Az5geMgZyY127cvLlLx+Y\\\\\\\/waTfgPHsg0LC1t231Iyo7ggQwpNlztExkSHRMMxQjFboA1bE9yAv2a7KyMZBpME2M2NLwDnjLAXFhIkq5BU+ugFljd3k8e7ZFKFixuGW0A3BPToODhW5m+htUKcV23tiUE4HG4LXBPXpDNDmRLoMvLA1co2Wtk+6SYLXD8wqbUhRBu4SK+y0iu6uFqUxEOuXRBZs6vwFFo9qNscdB76kAiQMFB7Rv4WSXGvDOIcJZMGlzlRqjd49tuQzZm7SSG0DTWBOhZ1yjy06U54x41vDDwW\\\\\\\/Ac4Pq+zqETZ1XuVgs86et3XGqisjgrPjYju85pZaHpWyt6aQbBuUmFqq1XVKRlezhvN0tEznoiDtuuX2Bf80072EBHjgGz\\\\\\\/NSWQRVcB6JZ1KTaWOQ0ZYQOdWXF3AiNKDLuKOFS4M4JJJCWsO\\\\\\\/4n874xji7PiBvFz7vJ2a1Df7srDqbR\\\\\\\/RwhyVS0D+0XiGst9B4JFaTSKZzp69jwo9Ck1UW8wd9igYsYa\\\\\\\/3aBjOcm+ow5k8XCLqrVp5pBkR+jUbSfpKRYyKdW\\\\\\\/xFcSLjkZAfGexm5jdhKso7fHSQFSAlVWR83yDqjKCRKj+wkHlNcAy\\\\\\\/vDbaxmIjN+GCbYcRPWa5\\\\\\\/hIGoAy8T7vk\\\\\\\/Z+4B5q\\\\\\\/hpbhpors6lHTz9Jj1YkcvLAKL9V4Q76GF6BI\\\\\\\/TQVyFeupyJQ1mxXIJrqlSoqoX4SzcpYL8u6RygsMAV9ALgaD4FpodPAQXcWyNK3qnpOc+LkG50x6C97j0eqx37XIbWZP1lmrxfjoHHRGvQUj\\\\\\\/fLO3HyO+owYO0dFMIRqKflZ7sZG1q4SkJ5x6WxuHCoy\\\\\\\/JzuURlzkbnee70KyjbY3BTwET3XzifziMcZbnB4BunMwrl4ZHOFzPl9XtDQSOF05cxXHyY4ryB2xXeI059f6RnUVuB76J5AqifRg6QBWL2+yykP1uURCqpFgT9GPUpEJYzluy\\\\\\\/nsDGgzFF9OqNYGMsk99DvnxQyQB3Xf0Q2RN12H5vE2g2QPljaJ6zkd4AhB0fHIz+KVbL1Cru3DjfVorF8fQe4\\\\\\\/D9AWfPKDWJUxBpIXqYO2I\\\\\\\/FOIyfJB6rTuDRuuYyC6ggSMzFyY25JCH5v0My4YKxh7yi1t74uI5WzfBDmWBg7+wsdeBMzLMC9VPmz1nGY7nejH671Y8box8Icb1ltc6sVEfLDcaX8NdMk9DTWT9TZpXVwBNIP49vYt8pCz\\\\\\\/awQDyB1VyWe7yz2b02SFtfd3L5c44IT3J9YEMdVKg3nyF3rK+lGOl\\\\\\\/f8GFso+prYtJr2wZZY0Zl+Luy99CdjMQ6YhkF97LE6WXN8mpFQJKnrhNYax1dAuTlbYq9ameXTUQOEcrTIlxqFyPROuPCC1XMGijtim\\\\\\\/7qFKubWC\\\\\\\/jd9oiIfI8neU2aE324dFTUotD8d4kIzdyrV9Lo+XZp6StUWznwiOAQ3kFU\\\\\\\/ZDVyadux2lnym1p8FGVUU+o42YcgFbDIl7gTv1n1HN7ODW\\\\\\\/ZtBupo0Vd1vKfs1k+Ci6q56VU0pv2NjS9gTLCeqK+aepR4VMIiqrbIAxH8aDuiVyLFqeMk90Sx+kY6cFgQBzCVjRpsUgEHQeYFqdFJnmg+iBoJJgpvQqn3VL\\\\\\\/IEmuhw30XFt+2a\\\\\\\/K3+AOEC8xFfwzabHeOGahd6GFmKFnKpkWonOiKzl0MlwieHrYZav2\\\\\\\/yIxvnKEjRqU8AQZ1hCAgoUl\\\\\\\/XxzB7wWo5QbUeJL1iy125mwkdj+XPA+c0prvNAL0MQEFwZMW6Q94v7ff\\\\\\\/kETZozPEmKXoaDfZDH+wxWXTI4g1hs2C8MP+fD6mRYg2PP+JInyTG6KXMrpg0s4XRYrJ5s4aGMBKxW1bSBSNtVn9BdYq2WrWYPdGEg5AbFSjvoeAs4CwbKNvluvXbnZbUVUh5Mlf\\\\\\\/Iq9LdFDDhVdt6WkiNhMPwHHdM+toIvnVLDIcmvW4apM4DG1Vpbb912KeYLALNpGfqAksAecVifZtRnjhWT\\\\\\\/BjhvcWm5+HsmgB8G2XgxqsEpbjCrWNUQgMgkny3DwAQTi1TqqSPFyMrs6x21GCt47LuA3wSJ1iOezSob1jJcW64sA2IPrm7pAit9C5zaAT7Fq8Z1ROs7xcJC12d38b1UfaManoH8ArWoIN+nv\\\\\\\/c0tn0PgttNnS9roxhqCNoUR73\\\\\\\/k3kT3Lr+JHv6NMUUP7FtBYUKDAw1FWMk8KyVV2hXuDhEkBgjJ8hIlMDS9ykH0Q\\\\\\\/qok2Qy4pqSn0KiFTY0NSppqPwuCjlEVD+UJv9RFeBQgAdMsgV70AHyLcnMD2IvWn3tz60fL8MBn\\\\\\\/RjDffvL7bHpzXsUVs6FvO\\\\\\\/xerfgAfaBiFiD2aPSdhI5ozX9AElSAXhGmd3w3c71LtzM29lA4s8B6iaVETgXUozyk9skEvrQ7j2YMhYG6taLbc3WLTi7CpPKhYihz\\\\\\\/\\\\\\\/UD9uDEa65r91iBVHpcnpNQimLcBLD2GkDIl7s9UU0pUXqyxHe+l8ry7lhj3LEZrJfK6kIxYNVR7xaQa1qWxy1aT7YOH4b\\\\\\\/nK2qyqkNnhujhr\\\\\\\/dWKqAaOI99Uv\\\\\\\/7toAWANPUmeKZk+RXZyrCILTu2h43i6P6rklwzByqn8Np9THKtel4AMUAzO09DFE6f0yv7oGxy+8XrnR5DnrzhwdeiQ5T6VtcG9UxK6GKRMvpBfA\\\\\\\/Dj4WathWwttocp+W3ZdPTMeFCKCTTRFKplj29vAL9uv2GTEPy11xZN4v2+12s496ETTxmF3zlUi6Iyt8NT3cE+i6PsWoyjfwbHJ4wmtYKR4Qr0M+wrrPnLSStO4+N+zvrrrZN2BBOnG+NZq1YqXqudHevrfQHoyftkY+8t82OQm\\\\\\\/l3vBWgNgXhaPDo0uUUqMvrnJAw9DscMojTyXjS+OM+e0JQfS4Nw5+P3Vq4o+48exzPOVg5vEvg8aDkQvc3h\\\\\\\/sek63vOgMD9KGirotn0lK1ZOFdNMg5lrRWD81uyFFjif3P68ZKFkj+rhjwThrDBzTtU\\\\\\\/bK82cJFAh77jfHvMuhEh59JCqGroNJL1WPPK15DT2D\\\\\\\/gfms8O\\\\\\\/Het+dxruI3z4zxNVimCcFSiLff7kufyV1Ui+Fy3QGyTJtGpsxGFV7VuabMIEklzt7m966ZdT1OP0yLt+bzkiNFSeLVprUSyL7IRL1sQVxoJk3QUG4Jhj9vCnvP8sN63Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RbJH8ItBJnkoZYyJ8JjqJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c3d03cfd8affae7f1625d77d8f1714cea0ffb8fe26c679a4d49883ffa2299c25\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220fxMTsvJBCEvJU756zav54kDQ6IqA22ufueyA9V3al\\\\\\\/EQflUaakcTteemJ+8slO87l33ochQUlFfAYzjiu7\\\\\\\/VQi6EtzF7gaR0XhIEPXQT8Ic4TE9N+qzyi8cCZDLQdrei8bvgBtlc7GQqysY09No5ecClpGREQQN1NyfdgOVWzRTM8HUQbVjc\\\\\\\/47YB16MA71B3xw9AlhoXjhcuIvyzc0lSsUPklKoGOdvasCEWqhV7Fhc5R1nWSVKz1y0yzTg6oxFxx8hyyk6s\\\\\\\/YcGhW7sgKnOCJl9KyFU0vd1h2d5gvEt8C1y\\\\\\\/NyzWrE31DOgnUKG8JskFa8kh3BcGWE7t\\\\\\\/MOaxsL1Uxq6N7daLvARNhHwXODJxoTsC\\\\\\\/u3SJKqlmkabN5vja\\\\\\\/En\\\\\\\/uWAVEA36y5OwoLAO1g6jYPErZkl8s1LTsr170laMaFQFwnGKeOcOdgZvrmp3u00riqYKei5iQjw5kZ\\\\\\\/TwxyV8wqK9eS\\\\\\\/coxaJdVZwlANO\\\\\\\/1pVrNVZRMnCA4i7Y8QZuFddAORfCIjt74iuo7oYALLvou9h1GWCKzVwBx7RHgd8ZJhkjHVUNHPxnPy0L4sEPGO14Z0aw6hAT7EZfv5m6wfxRanl9mA7GYQMWjILuzExamG9HGMMgx+MmpSff8C9rVXQYS9H7x6fZYrSMcDHqORWDxCPuUgVW9gFSDLckJE+Mi5\\\\\\\/vyLzGjeDFtnRg2xsfxy9AX+\\\\\\\/uXE+tzWHfhd1Zq5nvHVixc9aaa11tf9U2y6a7JnrgWU0+LCZLUhsWt0pbUByLkvBxB\\\\\\\/Hy8OsnyRXrjN5rH24kAO3rZmBWHBwr\\\\\\\/E2d4bSReqDpd4uskjQNDBi8Hj3QGdqdsSSLgCBE1UBRPi7PiIhhHjyMa76E5NNtHYoXdnAZiOKYTmRBYPCj6kkt9r\\\\\\\/W1A+FriuzCEVfHgFOmERunZNCsdAUWnEUYHnP3yTsFXw6yv6SeeqnkkJELxGO2LZbjg\\\\\\\/sz4n8ZsWzBlverq\\\\\\\/Z+yLhETiCY5CNlglXsAFYPDcqVI9DyvmmsTXw+b6kREH6tesC\\\\\\\/By8+X5R39ublUL59AxwSH9RB+CDaPkK0k9kblqgUnyx2VyILpMbiRvNV3cWiGm9f7rzEOshQl+u1Js\\\\\\\/giS5xB\\\\\\\/54w1TVi3JNldfOQzMwiq2BytOmPN8jEHaV2ann\\\\\\\/idEmpGYfwhJpXsydxTW443SaVIiLYqYDBozmGOUFbZQkEFJaRaqh4qIrvXyjuvEHLjTe3L+fej8tWpPoXpPyOWaz3CdKl895zu+P\\\\\\\/34ukm06RGu\\\\\\\/s5wEkf76voza2Qdu\\\\\\\/arUC8IDptY+RK1RAbpvRvwd6cCsmIwfnUcuCZ5kSd\\\\\\\/Ye8woPi6Kl\\\\\\\/c4TNAPPZDA3+yTTTzWYBw\\\\\\\/u8y6nduOYdJhSfU\\\\\\\/rsfEuD3OvdCGQuCQr\\\\\\\/VISQLf+JlY7Bru6BulPOZ75mrs40IDooqgYvP\\\\\\\/nMb2hOTAiqFDDVDXjUogVffB0HCpZSN0Qb1KHffwOQDMu3GyUyNwW7MpXOmiqXI1T9IVI8fC9+HB3Xq2bLW+DqfOeBynUK0V8tB3I5nniHyfbQGQfNG2CSa0T5QsgUPt+4MiIxt\\\\\\\/G4mr4PbRo6IWtn2tjpFVVkQBoyLA4jlHVeecMs32VPZDuBPCYhpevJjgyUnt1mdbM5NdEg26VfcwgFUMyO\\\\\\\/qiShnKwyEzKKJXZD\\\\\\\/\\\\\\\/CM3ggtIBwBQel0I3l2UM8oBza2BSRhoef8\\\\\\\/DOwzhZq4LDMKn2WIzKWA3zp\\\\\\\/Uqn\\\\\\\/SRPwcUjTOGA0Pw9Od0QKTPcZBrwC\\\\\\\/O078OaIha36gwwrptcPj8IzpQbAmHkKBpk90fuHYA19oUDDITQ4VqKdNgqz1LhzA9zZz7lsGCay\\\\\\\/1t6jruKyYlCvH5PixScz5K+RuZYmTH1m9l4Dy8FIaWQmA9eNQ007++mi6ZQ9OK+yvlhdziwv5WcVmafCK1N8qAmWWIVQNB0UPcc9U069q4Hh99Dppe+UY7Voh8VlIajCrjhe9QRRaXvVP9IqgvqAXaKdOWWTRUlLB\\\\\\\/zg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DclkXhfSIhUYVtdcSCS5rQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002205f85a2a748148498a8f020042d5c63c10354cf4b4ad3c5a2d0260b01c584e35\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492531 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/889" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fFoTCYOxebaQwxQolWOV85e89AGXmgAeq2KV1lX1OY0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/889" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RvOQ7jq9QZYRhfgNokc08dci5BMdrj87x1lpryRq3sg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/879" title="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/901" title="The mir-35 Family Links Maternal Germline Sex to Embryonic Viability in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492531" id="node492531" data-pisa="ggg;9/3/889" data-pisa-master="ggg;g3.118.200752" data-apath="/ggg/9/3/889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Turning Observed Founder Alleles into Expected Relationships in an Intercross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jilun Meng</span>, <span class="highwire-citation-author" data-delta="1">Manfred Mayer</span>, <span class="highwire-citation-author" data-delta="2">Erika Wytrwat</span>, <span class="highwire-citation-author" data-delta="3">Martina Langhammer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">889-899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492531" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jilun Meng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jilun%2BMeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJilun%2BMeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erika Wytrwat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erika%2BWytrwat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wytrwat%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErika%2BWytrwat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martina Langhammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BLanghammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langhammer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartina%2BLanghammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Reinsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reinsch@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">reinsch@fbn-dummerstorf.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/889" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492531" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/889.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492531" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/889.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492531" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/889.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492531" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/889.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492531" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492531/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-74998 first last"><a href="/content/9/6/2037" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - June 01, 2019</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Pedigree-derived relationships for individuals from an intercross of several lines cannot easily account for the segregation variance that is mainly caused by loci with alternative alleles fixed in different lines. However, when all founders are genotyped for a large number of markers, such relationships can be derived for descendants as expected genomic relationships conditional on the observed founder allele frequencies. A tabular method was derived in detail for autosomes and the X-chromosome. As a case study, we analyzed litter size and body weights at three different ages in an advanced mouse intercross (29 generations, total pedigree size 19,266) between a line selected for high litter size (FL1) and a highly inbred control line (DUKsi). Approximately 60% of the total genetic variance was due to segregation variance. Estimated heritability values were 0.20 (0.03), 0.34 (0.04), 0.23 (0.03), 0.41 (0.03) and 0.47 (0.02) for litter size, litter weight and body weight at ages of 21, 42 and 63 days, respectively (standard errors in brackets). These values were between 12% and 65% higher than observed in analyses that treated founders as unrelated. Fields of applications include experimental populations (selection experiments or advanced intercross lines) with a limited number of founders, which can be genotyped at a reasonable cost. In principle any number of founder lines can be treated. Additional genotypes from individuals in later generations can be combined into a joint relationship matrix by capitalizing on previously published approaches.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/founder-genomic-relationships" class="hw-term hw-article-keyword hw-article-keyword-founder-genomic-relationships" rel="nofollow">founder genomic relationships</a></li><li class="kwd"><a href="/keyword/x-chromosomal-genomic-relationships" class="hw-term hw-article-keyword hw-article-keyword-x-chromosomal-genomic-relationships" rel="nofollow">X-chromosomal genomic relationships</a></li><li class="kwd"><a href="/keyword/sex-linked-inheritance" class="hw-term hw-article-keyword hw-article-keyword-sex-linked-inheritance" rel="nofollow">sex-linked inheritance</a></li><li class="kwd"><a href="/keyword/litter-size" class="hw-term hw-article-keyword hw-article-keyword-litter-size" rel="nofollow">litter size</a></li><li class="kwd"><a href="/keyword/growth-traits" class="hw-term hw-article-keyword hw-article-keyword-growth-traits" rel="nofollow">growth traits</a></li><li class="kwd"><a href="/keyword/selection-experiments" class="hw-term hw-article-keyword hw-article-keyword-selection-experiments" rel="nofollow">selection experiments</a></li></ul><p id="p-3">The founders of a pedigreed population are the very first individuals with no further recorded ancestors. They are usually treated as unrelated and non-inbred for setting up relationship matrices. However, treating founders of a genealogy as related has been shown to be a useful concept (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>) when genomic relationships (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">VanRaden 2008</a>) and pedigree information are to be combined into a joint relationship matrix (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>). This has led to the notion that identity by descent (IBD) of founder alleles arises with a certain probability as a consequence of a limited effective population size. The main achievement of taking founder relatedness into account is a suitable scaling of pedigree relationships (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>), which makes them compatible with genomic relationships. Other benefits are reasonably interpretable estimates of genetic variance components and the prediction of genetic trends (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>). Founder relationships can be estimated from marker data of genotyped individuals (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Christensen 2012</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Colleau <em>et al.</em> 2017</a>), which are usually only available for younger generations in ongoing breeding programs. In the context of mapping of quantitative trait loci (QTL) in line-cross experiments inferences on within-line relationships between QTL-genotypes of founders can also be made from variance components and related likelihoods (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rönnegård <em>et al.</em> 2008</a>).</p><p id="p-4">In the context of crossbreeding, founders comprise individuals from two or more genetically distinct populations. This requires relationship coefficients for each single population, in addition to a combination of populations (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>). Here, the aim is to model relationships between purebreds and also between purebreds and crossbreds, most frequently from the F<sub>1</sub> generation. Applications are in genetic evaluations, were purebred and crossbred performances are treated as genetically correlated traits (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pszczola <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Garcia-Baccino <em>et al.</em> 2017</a>). For this purpose, the interest is in the genetic (co-)variance components for these traits in the purebred populations, where selection takes place.</p><p id="p-5">A somewhat different focus exists when composite populations are generated, <em>e.g.</em>, for selection experiments in laboratory animals (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holt <em>et al.</em> 2005</a>) or when building advanced intercross lines (AIL, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darvasi and Soller 1995</a>) for fine-mapping purposes. In this case, two or more genetically distinct lines, in some cases inbred lines, are intercrossed. The population is further developed from generation to generation by inter-mating crossbreds. Only performance traits of the intercross are entered into the analyses of, for example, selection experiments. This is an undertaking for which the use of mixed models, with an appropriate relationship matrix, has been recommended (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Walsh and Lynch 2018</a>, p. 631-668). The genetic variance in the intercross generations later than F<sub>1</sub> includes the so-called segregation variance (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lande 1981</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lo <em>et al.</em> 1993</a>), which is caused by loci that are fixed for different alleles in the founder lines but begin to segregate from the F<sub>2</sub> generation onwards. The proportion to which the segregation variance contributes to the total genetic variance in the F<sub>2</sub> and later generations can, in principle, vary between zero and one. However, this proportion cannot be derived from pedigree data alone.</p><p id="p-6">As a solution, we propose a relationship matrix that takes account of known marker allele frequencies of founders. Those markers that are fixed for alternative alleles in different lines largely determine the extent of the role of segregation variance at an average locus. Rules for the Mendelian transmission of these relationships to later generations were derived for both autosomal and X-chromosomal relationships. These matrices can then be combined with information on observed genotypes, which may include non-founders, and be used for the estimation of variance components and genetic trends. The associated genetic variance is thereby defined as the variance among unrelated individuals in the first generation of the composite population (<em>i.e.</em>, the F<sub>2</sub> in a two line cross). In addition, an application is presented to obtain estimates of genetic paramters for litter size and growth in an advanced intercross between a long-term selected, high fecundity mouse line and a highly inbred control line.</p><div class="section other1" id="sec-1"><h2 class="">Theory</h2><div id="sec-2" class="subsection"><h3>Underlying assumptions</h3><p id="p-7">We assume two distinct founder populations; A and B, that contribute to a composite crossbred population. All founders are assumed to be genotyped and line specific founder allele frequencies are known. For each marker i, founder frequencies are denoted as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-2.gif"/></noscript></span></span>. Under the condition that all founders contribute equally to the composite population, the expected allele frequency in the F<sub>2</sub> generation <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-3.gif"/></noscript></span></span>, is fully determined as the average, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-4.gif"/></noscript></span></span>, of the two line specific frequencies.</p></div><div id="sec-3" class="subsection"><h3>Observed genomic founder relationships</h3><p id="p-8">The genotypes of the founders can be summarized into a centered genotype matrix, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-5.gif"/></noscript></span></span>, with one row per individual and one column per marker.</p><div id="sec-4" class="subsection"><h4>Autosomes:</h4><p id="p-9">For autosomal markers, entries into the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-6.gif"/></noscript></span></span> matrix are <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-7.gif"/></noscript></span></span> for genotypes AA, Aa and aa, respectively. The observed genomic relationship matrix, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-8.gif"/></noscript></span></span>, between founders is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where S is the scaling factor; <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-10.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-11.gif"/></noscript></span></span> is a standard genomic relationship matrix, except when using <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-12.gif"/></noscript></span></span> for centering and scaling, as previously described by Van Raden (2008).</p></div><div id="sec-5" class="subsection"><h4>X-chromosome:</h4><p id="p-10">The observed X-chromosomal genomic relationship matrix is set up in accordance with the rules for autosomal markers. Extra details apply to the definitions of average gene frequencies and the treatment of male (hemizygous) individuals. For X-chromosome markers we define the mean allele frequency <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-13.gif"/></noscript></span></span>, again, as <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-14.gif"/></noscript></span></span>. However, on the X-chromosome this is only equal to the gene frequency to be ultimately reached in later generations if the two founder lines contribute equally through males and females to the genetic makeup of the composite population.</p><p id="p-11">Genotype codes have to be transformed to gene counts; <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-15.gif"/></noscript></span></span>, for male founders and then centered by <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-16.gif"/></noscript></span></span> instead of <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-17.gif"/></noscript></span></span>. For matrix <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-18.gif"/></noscript></span></span> the entries for X-chromosome markers are <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-19.gif"/></noscript></span></span>, for genotypes A and a, respectively. The X-chromosome genotypes for female founders are, in contrast, treated in the same way as autosomal markers. The observed X-chromosomal genomic relationship matrix can then be calculated by <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a>, using X-chromosomal gene counts and the scaling factor S, as defined above.</p></div></div><div id="sec-6" class="subsection"><h3>Expected founder genomic relationships</h3><div id="sec-7" class="subsection"><h4>Autosomes:</h4><p id="p-12">An expectation of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-20.gif"/></noscript></span></span> (denoted <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-21.gif"/></noscript></span></span>) can be derived under the assumption that alleles at independent loci are randomly sampled from each founder line’s particular gene pool (Binomial sampling), as defined by their known founder allele frequencies. We consider two gametes randomly chosen from base population A. The 2×2 matrix <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-22.gif"/></noscript></span></span> has the expected sum of squared centered coefficients <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-23.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-24.gif"/></noscript></span></span>, for alleles A and a, respectively) for each of the gametes on its diagonal and the corresponding expected sum of cross products on its off diagonal,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>For base population B, the equivalent matrix <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-26.gif"/></noscript></span></span> can be derived from its line specific allele frequencies <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-27.gif"/></noscript></span></span>. In the following matrices, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-28.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-29.gif"/></noscript></span></span> are referred to as the expected covariance matrices between gametes from the same founder line.</p><p id="p-13">Furthermore, we can set up <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-30.gif"/></noscript></span></span> as the equivalent relationship matrix of two randomly chosen gametes from populations A and B:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>From the distinct elements in <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-32.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-33.gif"/></noscript></span></span> (also <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-34.gif"/></noscript></span></span>) we can compute all necessary expected relationships between individuals, which may occur in <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-35.gif"/></noscript></span></span>. First, we have the expected self-relationship of an individual from a particular founder line (<em>e.g.</em>, line A)<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>The expected relationship between two individuals from the same founder line is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>The expected relationship between two individuals from two different founder lines is</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span></div><div id="sec-8" class="subsection"><h4>X-chromosome:</h4><p id="p-14">With regard to expected X-chromosomal relationships, different combinations between either males or females may occur, in addition to the same or different founder lines. In total, there are eight different kinds of possible relationships: the self-relationship of a male originating from a certain line, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-39.gif"/></noscript></span></span>; the expected relationship between two males from the same line, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-40.gif"/></noscript></span></span>, or <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-41.gif"/></noscript></span></span> from different lines; the self-relationship, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-42.gif"/></noscript></span></span>, of a female from a certain line; the relationships <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-43.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-44.gif"/></noscript></span></span> between two females from the same line and different lines, respectively, and finally the relationships <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-45.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-46.gif"/></noscript></span></span> between a male and a female from different lines. Formulas for all eight cases are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492382/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492382/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Formulas for calculating the different types of expected X-chromosomal relationships of males and females in a crossbred population derived from two founder lines A and B</span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-9" class="subsection"><h3>Extending expected genomic relationships to later generations</h3><p id="p-17">Expected founder genomic relationships can be extended to all descendants by following the paths of Mendelian transmission, as specified in the pedigree. The resulting expected genomic relationship matrix is denoted by Ã. The diagonal elements <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-83.gif"/></noscript></span></span> and off-diagonal elements <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-84.gif"/></noscript></span></span> of matrix Ã are computed by a modified version of the tabular method (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Emik and Terrill 1949</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cruden 1949</a>).</p><div id="sec-10" class="subsection"><h4>Autosomes:</h4><p id="p-18">The expected autosomal self-relationships (diagonal elements of Ã) consist of three parts; the expected self-relationships of gametes inherited from the sire, from the dam and the gametic relationship between these parental gametes. Relationships between individuals (off-diagonal elements) are an average of the relationships of one candidate and the parents of another, as known from the tabular method. The expected self-relationships and relationships are:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-88.gif"/></noscript></span></span> are the parents of individual k, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-89.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-90.gif"/></noscript></span></span> are the parents of individual <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-91.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-93.gif"/></noscript></span></span> are the expected self-relationships of gametes that individual k inherits from their parents (see supplement for a derivation of <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">equations 7</a> and <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">8</a>).</p><p id="p-19">These formulas are applicable from the F<sub>1</sub> generation onwards, whereby their components depend on generation number. For F<sub>1</sub> individuals <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-94.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-95.gif"/></noscript></span></span> if we assume a male founder from line A and a female founder from line B. The <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-96.gif"/></noscript></span></span> is the expected relationship between two founders, <em>i.e.</em>, the parents of an F<sub>1</sub> individual, which in our case gives <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-97.gif"/></noscript></span></span>. Generally, for an F<sub>1</sub> individual the expected self-relationship is <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-98.gif"/></noscript></span></span>. Individuals from the F<sub>2</sub> generation receive gametes from each parent with a 50% probability for line A and line B alleles. Therefore, in the F<sub>2</sub>, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-99.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-100.gif"/></noscript></span></span>.</p><p id="p-20">With the two founder lines used in our case, the sum of <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-102.gif"/></noscript></span></span> is equal to 1 but <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-104.gif"/></noscript></span></span> are usually different (see supplement). The expected self-relationship for a gamete that an F<sub>2</sub> individual inherits from one of its parents is always equal to 0.5 and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-105.gif"/></noscript></span></span>. The same applies to later generations.</p></div><div id="sec-11" class="subsection"><h4>X-chromosome:</h4><p id="p-21">For the X-chromosome, equations (<a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">7</a>) and (<a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">8</a>) are modified for females to give<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-106.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-107.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>and for males to give<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-108.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-109.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>Note that <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fernando and Grossman (1990)</a> introduced similar equations for calculating the pedigree-derived relationships for the X-chromosome, were the self-relationship for males, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-110.gif"/></noscript></span></span>, is 0.5 in all generations, meaning <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-111.gif"/></noscript></span></span> must also be 0.5. The underlying assumption is that allele frequencies of both sexes are equal, as it is the case if the population is in an equilibrium state (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Self and Liang 1987</a>). Equations (<a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">9</a>) to (<a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">11</a>), in contrast, allow <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-113.gif"/></noscript></span></span> to fluctuate along with male and female marker frequencies in early generations, after mating male and female founders with differing allele frequencies.</p></div></div><div id="sec-12" class="subsection"><h3>Expected genomic co-variances in an infinitely large F<sub>2</sub> population</h3><p id="p-22">In a cross between two populations, A and B, the gametes that an individual receives in the F<sub>2</sub> generation are of types A and B, with equal probability. Such an individual will have a probability of 0.25 of inheriting two A gametes, a probability of 0.25 of inheriting two B gametes and a probability of 0.5 of inheriting one A gamete and one B gamete.</p><p id="p-23">The average self-relationship of an individual in an infinitely large F<sub>2</sub> population is, therefore<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-114.gif"/></noscript></span></span>as <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-115.gif"/></noscript></span></span> in a cross of two lines.</p><p id="p-24">The average covariance between F<sub>2</sub> individuals can be derived as the weighted average of covariances in nine possible combinations of two individuals, were each of them may carry two (AA), one (AB) or no (BB) gametes from the A line. These nine single pair-wise covariances can be expressed in terms of relationships between gametes, <em>i.e.</em> <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-116.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-118.gif"/></noscript></span></span>. The weights are the probabilities of the occurrence of all these combinations:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-119.gif"/></noscript></span></span>The expected covariance matrix Ã for unrelated and non-inbred F<sub>2</sub> individuals is, therefore, an identity matrix. This means that such a hypothetical population can be viewed as a reference for the actual composite population derived from the genotyped founders. Variance components estimated with an Ã matrix can, after proper adjustment for founder relationships (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Legarra <em>et al.</em> 2015</a>), be interpreted as the genetic variance in such a population. When defined in this way the genetic variance includes the segregation variance, <em>i.e.</em>, the difference between the genetic variance in the F<sub>2</sub> and the F<sub>1</sub> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lande 1981</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lo <em>et al.</em> 1993</a>). The segregation variance can be expressed as function of the self-relationships of non-inbred individuals in the F<sub>2</sub> and the F<sub>1</sub> generations:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-120.gif"/></noscript></span></span>Note that in the case of a cross between two inbred lines <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-121.gif"/></noscript></span></span> and the last formula correctly flags all genetic variance as segregation variance.</p></div><div id="sec-13" class="subsection"><h3>Accounting for observed genotypes</h3><div id="sec-14" class="subsection"><h4>Combined relationship matrix H:</h4><p id="p-25">Expected genomic founder relationships will generally differ from those observed. This can be taken into account by applying a previously developed theory (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Legarra <em>et al.</em> 2009</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Christensen and Lund 2010</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>) for combining pedigree-derived relationships (A) and genomic relationships into a joint matrix, H. In our case we used H as a modification of Ã that is corrected for the observed founder relationships in <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-122.gif"/></noscript></span></span>. We denote Ã as the expected genomic relationships of founders. In terms of the inverse of H (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Christensen and Lund 2010</a>; <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>), we then get<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-123.gif"/></noscript></span></span>The inverse of matrix Ã is described in the supplement. The observed founder genomic relationship matrix, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-124.gif"/></noscript></span></span>, may be singular. In this case, one may capitalize on the idea of blending (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Garcia-Baccino <em>et al.</em> 2017</a>). We used <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-125.gif"/></noscript></span></span>, instead of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-126.gif"/></noscript></span></span>, when computing <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-127.gif"/></noscript></span></span>.</p></div><div id="sec-15" class="subsection"><h4>Joint relationships from simulated genotypes:</h4><p id="p-26">For the sake of comparison with H, a joint relationship matrix, G, was generated from observed genotypes of founders plus simulated genotypes of non-founders. The alleles were randomly sampled from observed founder genotypes and simulated marker genotypes of offspring in later generations were derived by gene-drop. The expected self-relationships for autosomes were calculated as<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-128.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-129.gif"/></noscript></span></span> is the expected relationship between individuals <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-130.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-131.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-133.gif"/></noscript></span></span> are the gene counts of individuals <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-135.gif"/></noscript></span></span> at locus i. For self-relationship we used <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-136.gif"/></noscript></span></span>. Expectations were obtained by averaging over 10000 replicates of the gene-drop simulation.</p><p id="p-27">For X-chromosomes, the sex of the descendants must also be considered. For female descendants <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">equation (13)</a> remains. For male descendants <a id="xref-disp-formula-17-2" class="xref-disp-formula" href="#disp-formula-17">equation (13)</a> will be modified by applying the centered X-chromosomal genotypes (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-137.gif"/></noscript></span></span>), which gives <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-138.gif"/></noscript></span></span>.</p></div></div></div><div class="section other2" id="sec-16"><h2 class="">Application Example</h2><div id="sec-17" class="subsection"><h3>Animals, pedigree, phenotypes and genotypes</h3><p id="p-28">The advanced intercross mouse line (AIL) bred in the Leibniz Institute for Farm Animal Biology (FBN) was established by randomly choosing and intercrossing four females from the long-term selected, high-fecundity line, FL1 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Langhammer <em>et al.</em> 2014</a>), and four males from a highly inbred (theoretical inbreeding coefficient &gt; 0.999) control line, DUKsi (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alm <em>et al.</em> 2010</a>). Both lines were derived from the same initial gene pool (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dietl <em>et al.</em> 2004</a>).</p><p id="p-29">The high fecundity FL1 line was selected for an index trait that combines litter size (LS0) and litter weight (LW0) at birth in primiparous females (Index I = 1.6 × LS0 + LW0) up to generation 131. As a result of selection over 131 generations an average of 17.14 ± 3.25 pups per litter had been reached. This is a 1.8 fold higher fecundity than observed in the control line (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). An outbred control line, DUKs, was maintained at approximately the same population size for 79 generations by random mating and without any selection pressure. The inbred derivative DUKsi was split from DUKs in generation 79.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492330/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492330/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean &amp; SD of litter size and litter weight of two founder lines</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">Four male founders were chosen for the experiment after 38 generations of full sib mating in the DUKsi line. Each of four females from generation 131 of the FL1 selection line was mated with one male from the control line. The F<sub>1</sub> litters were standardized to four male and eight female pups immediately after birth in order to maintain a surplus of females for further reproduction. Full sibs from the four initial F<sub>1</sub> families were then repeatedly (at least four times) inter-mated by rotating males and females within the family. Thus, each of the four pairs of founder parents constitutes a family of its own, with descendants up to generation F<sub>3</sub>. Offspring of only one of these families were then maintained and became the ancestors of all further generations of the AIL.</p><p id="p-32">A total of 19266 mice (9453 males and 9813 females) were used for this study. They were distributed unevenly across all generations; 44 in F<sub>1</sub>, 1483 in F<sub>2</sub>, 5235 in F<sub>3</sub>, 1025, 1058 and 1070 in F<sub>23</sub>, F<sub>24</sub> and F<sub>25</sub>, respectively, and between 312 and 431 for other generations.</p><p id="p-33">Reproductive ability was measured as litter size at birth (LS0) and litter weight at birth (LW0). The litter traits were recorded for 4430 females (from 9813 females) for their first litter. Among these females, 1481 also had a record for their second litter (there were no second litter records for generations from F<sub>3</sub> to F<sub>21</sub>). Growth traits that were recorded for all generations were body weight at day 21, 42 and 63 (BM21, BM42, BM63), in addition to body weight at first mating (BMM).</p><p id="p-34">The six fertility and growth traits (see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> for summary statistics) were analyzed using different kinds of relationship matrices, as described below.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492503/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492503/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Descriptive statistics for traits related to litter size and growth. Litter size and litter weight was recorded in first (LS0_1, LW0_1) and partially also in second (LS0_2, LW0_2) parity females. Body mass of both sexes was measured at days 21 (BM21), 42 (BM42), 63 (BM63) after birth as well as at day of mating (BMM)</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">All eight founders of this intercross line were genotyped with the JAX Mouse Diversity Genotyping Array (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yang <em>et al.</em> 2009</a>) at the genotyping facility of the Jackson Laboratory, The Jackson Laboratory, Bar Harbor, Maine, USA.</p></div><div id="sec-18" class="subsection"><h3>Comparative estimation of variance components</h3><p id="p-37">Two fertility and four growth traits were comparatively analyzed with mixed models that comprised of different kinds of relationship matrices. Fertility traits (LS0, LW0) were analyzed as traits of females (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Langhammer <em>et al.</em> 2017</a>). For both traits, the model for the i<sub>th</sub> observation (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-139.gif"/></noscript></span></span>) of animal a was<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-140.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>where <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-141.gif"/></noscript></span></span> is the fixed generation effect for females born in generations from F<sub>2</sub> to F<sub>29</sub> (g = 1, ... 28) and γ is the linear regression of the body weight <em>w<sub>a</sub></em> at mating of each female’s mother. The random part of the model comprised the additive autosomal and X-chromosomal genetic effects <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-142.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-143.gif"/></noscript></span></span> of animal a (a=1, ..., 19266), the common litter environmental effect <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-144.gif"/></noscript></span></span> (c=1, ..., 2420), the permanent environmental effect <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-145.gif"/></noscript></span></span> (p=1, ..., 4430) and the residual <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-146.gif"/></noscript></span></span>. The covariance matrices of the common litter environmental and permanent environmental effects were assumed as equal to an identity matrix of proper size times the respective variance component.</p><p id="p-38">Observations of growth traits (BM21, BM42, BM63, and BMM) were made from males and females from generations F<sub>2</sub> to F<sub>30</sub>. Therefore, the fixed part of the model also included an additional sex effect, <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-147.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-148.gif"/></noscript></span></span>), and the number of levels was 29 for the generation effect. For BM21 and BMM no permanent environmental effect could be fitted because these traits were only measured once for all animals. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> gives more details of the model we applied.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492484/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492484/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Numbers of observations (N) and number of levels for fixed and random effects in models fitted to litter size (LS0), litter weight (LW0) and body mass traits at ages of 21, 42, 63 days and at mating (BM21, BM42, BM63, BMM). Fixed effects are generation number, linear regression on mother’s weight at mating, litter number and sex; random effects are additive genetic, litter and permanent environmental effects</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">All random effects were assumed to be mutually independent. Three different kinds of relationship matrices were compared for autosomal and X-chromosomal genetic effects: pedigree-derived relationship matrices that assumed unrelated founders, H matrices, as explained above, and G matrices, based on gene-drop simulations (denoted as “A”, “H” and “G”, respectively). Model variants “A<sub>a</sub>”, “G<sub>a</sub>” and “H<sub>a</sub>” include only an autosomal relationship matrix of one of the specified types. Model variants “A<sub>a+x</sub>”, “G<sub>a+x</sub>” and “H<sub>a+x</sub>” additionally include the X-chromosomal relationship matrix of the same type. The Restricted maximum likelihood (REML) estimates of all variance components were obtained from the ASReml package (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gilmour <em>et al.</em> 2015</a>). All estimated genetic variance components from model variants, which included founder relationships, were corrected for non-independence of founders by multiplication with a correction factor, <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-149.gif"/></noscript></span></span> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Searle 1982</a>. p. 355; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Legarra 2016</a>):<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-150.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-151.gif"/></noscript></span></span> are elements of either the observed founder relationship matrix, <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-152.gif"/></noscript></span></span> (after blending), or a respective diagonal matrix for pedigree-derived relationships, and <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-153.gif"/></noscript></span></span> is the number of founders.</p><p id="p-40">The significance of X-chromosomal genetic effects was evaluated by comparing the full model for each trait with a reduced model without X-chromosomal genetic effects. Error probabilities were derived via restricted likelihood ratio tests (RLRT), with a single degree of freedom (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wiencierz <em>et al.</em> 2011</a>).</p></div><div id="sec-19" class="subsection"><h3>Animal welfare declaration</h3><p id="p-41">The animal experiments were performed following national and international guidelines and were approved by the local authorities (Landesamt für Landwirtschaft, Lebensmittelsicherheit und Fischerei, Mecklenburg-Vorpommern, Germany).</p></div><div id="sec-20" class="subsection"><h3>Data availability</h3><p id="p-42">Marker genotypes of founders, a pedigree file for all AIL animals and a data file with observed phenotypes for the six analyzed traits can be found in the RADAR (research data repository) repository under <a href="https://doi.org/10.22000/88">https://doi.org/10.22000/88</a>. An R-program that sets up the Ã matrix for a two-line cross according to the rules explained above (autosomal and X-chromosomal) is available from the first author. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7110440">https://doi.org/10.25387/g3.7110440</a>.</p></div></div><div class="section results-discussion" id="sec-21"><h2 class="">Results and Discussion</h2><div id="sec-22" class="subsection"><h3>Founder relationship matrices</h3><div id="sec-23" class="subsection"><h4>Marker data:</h4><p id="p-43">The number of polymorphic autosomal SNPs (single nucleotide polymorphisms) was 140,532 in all founders (see <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The number that segregated in the FL1 line alone was 44,827, while 67,450 segregated within the DUKsi control line. The numbers on the X-chromosome (non-pseudoautosomal, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Perry <em>et al.</em> 2001</a>) were 2,009 for all founders, 191 in the FL1 line and 1,055 in the control line. Opposite alleles with frequencies at hundred and zero per cent in the two lines (line-specific alleles) occurred with 38.9% on autosomes as well as the X-chromosome. Polymorphic markers were evenly distributed across the genome (see Figure S1 in supplement file) and the density (number per 1 Mbp) was between 34.0 and 70.4 (see Table S1 in supplement file).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492407/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Numbers of SNP-markers which were segregating within all eight founders, and within both founder lines (FL1, control). Line-specific SNPs do not segregate within founders of the same line but have alternative kinds of alleles in each line</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Observed genomic relationships:</h4><p id="p-44">The observed 8×8 genomic founder relationship matrices are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> as triangular matrices for autosomes (above) and X-chromosomes (below). Observed autosomal self-relationships were fairly uniform in both the control (between 1.382 and 1.488) and the FL1 line (between 1.405 and 1.452). The expected self-relationships (lower triangle of 4×4 matrix, same panel) were 1.525 and 1.669 for the control and FL1 founders, respectively. The lower observed relationships (approximately 7% in controls and 14% in the FL1 line) indicate an excess of heterozygosity in both lines relative to within line Hardy-Weinberg proportions at observed allele frequencies. These deviations can be explained by the sampling of rare alleles, which are more likely to occur in a heterozygous condition when compared with more frequent alleles. Observed self-relationships are, however, considerably larger than one due to elevated homozygosity relative to the non-inbred F<sub>2</sub> individuals that define the base population. Despite some fluctuations, relationships between founders of the same line (expected: 1.193) and different lines (expected: -1.193) barely deviate from the expectations. On the X-chromosome, the observed self-relationships of female control founders (range: 1.347 - 1.520) deviate more (about 26%) from the expectation of 1.906, while in contrast, observed self-relationships of male control founders and all observed X-chromosomal relationships between male and female founders agree well with the expectations. On both kinds of chromosomes, large negative relationships between individuals from different lines are predominantly a result of SNPs with line-specific alleles. Their high proportion and the resulting negative between-line relationships reflect the long lasting separation of the two lines and their selection for different goals. Consequently, the expectation of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-154.gif"/></noscript></span></span> translates into a proportion of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-155.gif"/></noscript></span></span> of the total genetic variance that can be attributed to segregation variance.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/889/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the observed and expected founder genomic relationship matrices. The upper panel shows the autosomal observed genomic founder relationship matrix G (marked in blue) and the autosomal expected founder relationship matrix GE (marked in red). The lower panel shows X-chromosomal G (marked in blue) and X-chromosomal GE (marked in red). Diagonal elements are in bold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322190101" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the observed and expected founder genomic relationship matrices. The upper panel shows the autosomal observed genomic founder relationship matrix &lt;strong&gt;G&lt;/strong&gt; (marked in blue) and the autosomal expected founder relationship matrix &lt;strong&gt;G&lt;/strong&gt;&lt;sup&gt;&lt;strong&gt;E&lt;/strong&gt;&lt;/sup&gt; (marked in red). The lower panel shows X-chromosomal &lt;strong&gt;G&lt;/strong&gt; (marked in blue) and X-chromosomal &lt;strong&gt;G&lt;/strong&gt;&lt;sup&gt;&lt;strong&gt;E&lt;/strong&gt;&lt;/sup&gt; (marked in red). Diagonal elements are in bold.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/889/F1.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/889/F1.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/889/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/889/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-45" class="first-child">Comparison of the observed and expected founder genomic relationship matrices. The upper panel shows the autosomal observed genomic founder relationship matrix <strong>G</strong> (marked in blue) and the autosomal expected founder relationship matrix <strong>G</strong><sup><strong>E</strong></sup> (marked in red). The lower panel shows X-chromosomal <strong>G</strong> (marked in blue) and X-chromosomal <strong>G</strong><sup><strong>E</strong></sup> (marked in red). Diagonal elements are in bold.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Both the autosomal and X-chromosomal observed genomic relationship matrices are singular, with rank seven. In the case of the autosomal markers, this is caused by the relationship of 1.36 between the third and fourth founders (<span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-156.gif"/></noscript></span></span>), which is close to the self-relationship of the same animals (1.38). This translates into a very close correlation of almost one. The background is that the inbred control line actually consisted of several sublines that can be traced back to the same pair of ancestors. Sublines were generated by branching the main line in different generations and maintained by repeated full-sib mating. Unintentionally, two male founders were sampled from the same subline and the other male founders from two different sublines. In the case were all control founders had been drawn from the same subline, the rank for the observed relationship matrix is expected to be five, as almost no genetic variation is expected within the subline.</p><p id="p-47">As a consequence of this rank deficiency, the observed relationship matrices are not invertible. This was solved by blending them with their expected counterparts (see Theory). Alternatively, one could have averaged the two columns and rows of the highly correlated animals and used this average as a replacement in a 7×7 relationship matrix, thereby assigning a single genetic effect to both founders (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tuchscherer <em>et al.</em> 2004</a>).</p></div><div id="sec-25" class="subsection"><h4>Evolution of self-relationships over generations:</h4><p id="p-48">The mean self-relationships, as derived from different kinds of relationship matrices, develop differently over generations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The classical pedigree-derived matrix A has diagonal elements of one in the base generations and the F<sub>1</sub>, followed by a jump to 1.25, which indicates inbreeding of F<sub>2</sub> animals due to full-sib mating in the F<sub>1</sub>. From then on there is only a very slight increase of the mean inbreeding coefficient. This pattern is present for autosomal relationships (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, upper left), as well as for X-chromosomal self-relationships in females (lower left panel). Self-relationships from Ã, in contrast, show strong fluctuations from high position values in founders to high negative values in the F<sub>1</sub> (same panels). Autosomal self-relationships from the Ã matrix reach an average of larger than one in the F<sub>2</sub>, which increases only slightly in further generations (upper left panel). The Ã matrix is scaled in such a way that non-inbred F<sub>2</sub> animals would have a self-relationship of one. Therefore, larger values are a sign of a higher expected homozygosity when compared with this reference population. The fluctuations of average generational autosomal self-relationships come to an end from generation F<sub>2</sub> onwards (upper left), while they continue, albeit with decreasing amplitudes, for X-chromosomal self-relationships in males (middle left) and females (lower left). The underlying reason is that the genetic equilibrium is reached after two generations for autosomal markers, when initial allele frequencies differ in males and females (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crow and Kimura 1970</a>). This process takes longer for X-chromosomal loci (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li 1976</a>, p. 137). In line with this, the amplitudes for male X-chromosomal self-relationships have the opposite sign to those for females.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/889/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the generation mean of the self-relationships for the relationship matrices derived in the study. A is the pedigree-derived relationship matrix. &#xC3; is the relationship matrix derived from the pedigree and the allele frequencies of all founder lines. G is the pedigree-genotype-combined relationship matrix derived by &#x201C;gene drop&#x201D;. H is the relationship matrix derived by Legarra&#x2019;s method (2008) using matrix &#xC3; instead of matrix A. The X-chromosomal self-relationships are divided by sex. The oscillatory approach of the allele frequency of the X-linked markers can be observed in the first few generations of the curves for matrices &#xC3;, G and H, when compared with the matrix A and the autosomal cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322190101" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the generation mean of the self-relationships for the relationship matrices derived in the study. A is the pedigree-derived relationship matrix. &#xC3; is the relationship matrix derived from the pedigree and the allele frequencies of all founder lines. G is the pedigree-genotype-combined relationship matrix derived by &#x201C;gene drop&#x201D;. H is the relationship matrix derived by Legarra&#x2019;s method (2008) using matrix &#xC3; instead of matrix A. The X-chromosomal self-relationships are divided by sex. The oscillatory approach of the allele frequency of the X-linked markers can be observed in the first few generations of the curves for matrices &#xC3;, G and H, when compared with the matrix A and the autosomal cases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/889/F2.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/889/F2.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/889/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/889/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492525" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-49" class="first-child">Comparison of the generation mean of the self-relationships for the relationship matrices derived in the study. A is the pedigree-derived relationship matrix. Ã is the relationship matrix derived from the pedigree and the allele frequencies of all founder lines. G is the pedigree-genotype-combined relationship matrix derived by “gene drop”. H is the relationship matrix derived by Legarra’s method (2008) using matrix Ã instead of matrix A. The X-chromosomal self-relationships are divided by sex. The oscillatory approach of the allele frequency of the X-linked markers can be observed in the first few generations of the curves for matrices Ã, G and H, when compared with the matrix A and the autosomal cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The mean X-chromosomal self-relationships of males stabilize at around 0.54, which is somewhat larger than 0.5. The reason is that the actual equilibrium allele frequencies approach <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-157.gif"/></noscript></span></span> instead of <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-158.gif"/></noscript></span></span>, since all founders from the FL1 line were females with two alleles and all founders from the control line had only a single allele at each X-chromosome locus. The X-chromosomal Ã matrix was, however, computed under the assumption of equal contributions of both founder lines to the F<sub>2</sub>, which would require equal numbers of male and female founders from both lines. Values of 0.54 therefore indicate somewhat more X-chromosomal variability, as in a reference population were <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-159.gif"/></noscript></span></span>.</p><p id="p-51">Average self-relationships from H and G types of relationship matrices can be seen on the panels to the right of <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. The initial fluctuation patterns already described for Ã are also present in the average self-relationships of these two matrices. The three curves for the G matrices are similar but not identical to Ã. In comparison, averages for the H matrices are lower in all cases, which is a result of correcting Ã to lower observed homozygosity than expected, under the assumptions made for the construction of Ã.</p></div><div id="sec-26" class="subsection"><h4>Genetic parameters:</h4><p id="p-52">The genetic variance components and heritability for six selected traits can be found in the <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> and Table S2. The X-chromosomal genetic variance proved to be significant at the 5% level for the three growth traits; BM21, BM42 and BM63, regardless of what type of relationship matrix was used as part of the model (Table S3). The additive genetic variance component for the sex chromosome was almost zero for BMM (see Table S2) and was also not significant for litter traits LS0 and LW0 (Table S3).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492429/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492429/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Estimates of genetic parameters for all traits obtained by three different kinds of relationship matrices</span><div class="sb-div caption-clear"></div></div></div><p id="p-54">The proportion of the total genetic variance that was attributed to the sex chromosome was approximately 3% for BM21 in all analyses (data in <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). When founders were assumed to be related, the same proportion was 9% and 12% for BM42 and BM63, respectively. These were lower than the comparative values of 12% and 17%, when the assumption of unrelatedness was applied. Over all traits, a standard pattern emerged (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>) were genetic variance components and heritability were larger when either an H or a G matrix was part of the model, compared to an A matrix analysis. In contrast, results from H and G matrices were almost equal for all traits. Estimated residual and common litter environmental variances were barely affected by the choice of genetic relationship matrix for any of the traits analyzed. The same was true for the permanent environmental variance component for BM42 and BM63, whereas both litter traits displayed lower estimates for the permanent environmental variance component from H and G matrices, compared with A matrices. This was accompanied by considerably larger estimates for autosomal additive genetic variances. Consequently, heritability for LS0 was 20% when founder relationships were taken into account (matrix <strong>H</strong>), vs. 12% when they were not taken into account (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). For LW0 and growth traits, the respective comparisons were 34% and 32% vs. 23% (LW0), 23% and 22% vs. 17% (BM21), 41% and 40% vs. 35% (BM42), 47% and 45% vs. 41% (BM63), and 52% and 50% vs. 45% (BMM). For LS0, in particular, the increase in the estimates of the genetic variance using <strong>H</strong> and <strong>G</strong> matrices is larger than 60% (64% and 78%). To that effect estimated genetic standard deviations changed and the corresponding range of genotypes with very low and very high litter size, defined as six times the estimated genetic standard deviation, rose from seven to approximately nine pups per litter. In essence, the results from <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a> demonstrate that the chosen scaling of H and G matrices provided higher estimates for the genetic variance components for all traits. This increase can be interpreted as caused by correctly including the segregation variance, which is part of the genetic variance from generation F<sub>2</sub> onwards and is expected to be prominent for the trait LS0, on which the FL1 line has been selected.</p><p id="p-55">Our own estimated heritability values for LS0 are within the wide range of results reported in older studies. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falconer (1960)</a> reported 8.3% heritability for upward selection and 22.9% for downward selection. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradford (1968)</a> presented realized heritability from 0.13 to 0.39 for litter size in several lines, while <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bakker <em>et al.</em> (1978)</a> found a realized heritability of 0.11. More recent investigations tend toward lower values, compared with our result of 19%, <em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beniwal <em>et al.</em> (1992)</a> reported a comparatively low heritability for litter size (0.181 ± 0.093 for control and 0.166 ± 0.043 overall), as did <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peripato <em>et al.</em> (2004)</a>, with <em>h</em><sup>2</sup> = 12%. Similarly, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gutiérrez <em>et al.</em> (2006)</a> published heritability values for litter size from 0.099 to 0.101. <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gutiérrez <em>et al.</em> (2006)</a> also reported considerably lower heritability, from 0.112 to 0.148 (derived with different models), for litter weight.</p><p id="p-56">Estimated heritability values for body weight traits in mice vary widely in the literature. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer (1953)</a> reported the heritability for body weight measured at day 60 as approximately 20% for upward selection and 50% for downward selection. Interestingly, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wilson <em>et al.</em> (1971)</a> found that the realized heritability for body weight at day 60 declined from 0.32 in the first 10 generations to 0.08 between generation 61 and 70, in a selection experiment. <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eisen (1978)</a> reported a heritability of 0.44 (or 0.55, depending on the method of estimation) for six week body weight. <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heath <em>et al.</em> (1995)</a> reported a heritability of 0.25 before selection and a mean heritability of 0.216 ± 0.0077 (from 0191 ± 0.016 to 0.242 ± 0.014 for different pairs of lines) for six week body weight. In an intercross-population, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kramer <em>et al.</em> (1998)</a> estimated high heritability values of 0.54 ± 0.24 for three week body weight, 0.76 ± 0.04 for six week body weight and 0.81 ± 0.01 for nine week body weight in a cross-fostering experiment. Although they are within the upper ranks, our estimates of approximately 40–50% at ages from 42 to 63 days may be seen as well within the range of literature values.</p></div><div id="sec-27" class="subsection"><h4>Estimated genetic trends:</h4><p id="p-57"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows estimated genetic trends for LS0 and BM42 from models with different relationship matrices. The upper panels (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) show genetic trends from models with autosomal relationships only, while the other panels (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) depict those from applying the autosomal and X-chromosomal relationship matrices, simultaneously. In general, trends are very similar by shape and also by level, in accordance with the absence of any serious trend. An exception is the jump at generation F<sub>3</sub> for LS0. This reflects the fact that from generation F<sub>3</sub> onwards only the descendants of a single family were maintained in order to breed the later generations, which makes the differences between the four founder families manifest. See also Figure S2 for genetic trends in other traits.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/889/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Autosomal (above and middle) and X-chromosomal (below) genetic trends for litter size (LS0, left) and body mass at day 42 (BM42, right) obtained by different kinds* of relationship matrices (A, G and H) and models with (subscript a+x) and without (subscript a) taking X-chromosomal genetic variation into account. *) Kinds of relationship matrices: pedigree-derived numerator relationship matrix (A), gene-drop derived (G), combined expected and observed genomic relationships (H); subscripts indicate that autosomal relationships only (a) or both autosomal and X-chromosomal relationships (a+x) were fitted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322190101" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Autosomal (above and middle) and X-chromosomal (below) genetic trends for litter size (LS0, left) and body mass at day 42 (BM42, right) obtained by different kinds* of relationship matrices (A, G and H) and models with (subscript a+x) and without (subscript a) taking X-chromosomal genetic variation into account. *) Kinds of relationship matrices: pedigree-derived numerator relationship matrix (A), gene-drop derived (G), combined expected and observed genomic relationships (H); subscripts indicate that autosomal relationships only (a) or both autosomal and X-chromosomal relationships (a+x) were fitted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/889/F3.medium.gif" width="414" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/889/F3.medium.gif" width="414" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/889/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/889/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-58" class="first-child">Autosomal (above and middle) and X-chromosomal (below) genetic trends for litter size (LS0, left) and body mass at day 42 (BM42, right) obtained by different kinds* of relationship matrices (A, G and H) and models with (subscript a+x) and without (subscript a) taking X-chromosomal genetic variation into account. *) Kinds of relationship matrices: pedigree-derived numerator relationship matrix (A), gene-drop derived (G), combined expected and observed genomic relationships (H); subscripts indicate that autosomal relationships only (a) or both autosomal and X-chromosomal relationships (a+x) were fitted.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h4>Underlying assumptions:</h4><p id="p-59">In deriving Ã, the expected overall heterozygosity, as described by the scale parameter S, is taken as being fully determined by the observed allele frequencies of the genotyped founders and the assumption that both lines contribute equally to the new composite population. The latter can easily be adapted to more than two founder lines, even with unequal contributions, by an alteration of the definition of the average allele frequency <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-172.gif"/></noscript></span></span>. With more than two founder lines, the equilibrium state with foreseen autosomal heterozygosity, <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-173.gif"/></noscript></span></span>, may also be reached later than in the second crossbred generation. This depends on the mating scheme that generates the new composite population and, as with two founder lines, is an asymptotic process for X-chromosome markers. The initial numbers of male and female founders may be different in each line to be crossed later. Autosomal and X-chromosome reference populations should be comparable, especially in the interpretation of genetic parameters. Therefore, it seems reasonable to define <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/889/embed/mml-math-174.gif"/></noscript></span></span> for X-chromosome markers, and hence S, as if males and females from all founder lines contribute equally. However, this does not need to be the case, as demonstrated by our mouse example. A further assumption entered into Ã is that founder genotype frequencies meet line-specific Hardy-Weinberg equilibrium. Observed founder genomic self-relationships that deviate from their expected counterparts indicate an excess or lack of heterozygosity relative to this assumption. Actual marker heterozygosity values are then accounted for using the combined relationship matrix H.</p><p id="p-60">Matrix Ã reflects average genomic relationships that result from repeatedly sampling alleles at observed line-specific frequencies from founders and their forward transmission to later generations, in accordance with the pedigree. No attempts were made to estimate IBD-based founder relationships (<em>e.g.</em>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Powell <em>et al.</em> 2010</a>) within lines. There are typically only a few founders of experimental populations, as in our mouse example, and they may provide information on within-line IBD-relationships only with high sampling errors, unless a larger sample is genotyped. However, we did not treat founders as a sample from their respective lines but their genotypes were treated as a complete inventory of all possible alleles that can be further transmitted to the F<sub>2</sub> generation and beyond. Therefore, they fully determine the genetic makeup of the later crossbred population, as mirrored by the construction of Ã. The usefulness of Ã, and hence H, for estimating the genetic variance will depend on how well the frequency spectrum of markers reflect the frequency spectrum of QTL (quantitative trait loci) for a trait (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Walsh and Lynch 2018</a>, p. 631-668). The latter requirement is likely to be largely fulfilled in a cross of two divergently selected lines, where a large contribution by segregation variance can be expected to be picked up by a large proportion of markers with line-specific alleles.</p></div><div id="sec-29" class="subsection"><h4>Fields of application:</h4><p id="p-61">In the analysis of selection experiments, mixed models have largely replaced other methods due to their flexibility (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Walsh and Lynch 2018</a>, p. 631-668). Data from AIL lines can be seen as a special case, with no artificial selection applied. Pedigree-based relationship matrices traditionally treat founders as unrelated and non-inbred. As only a limited number of founders are often genotyped, due to cost, this unrealistic assumption can be overcome by taking genomic founder relationships into account. Using mixed models, with the described Ã based version of H, accounts for the initial disequilibrium in allele and genotype frequencies, which exists for more generations on the X chromosome, compared with only two generations for autosomal loci. Meaningful estimates of genetic variances and heritability may be calculated in crosses of two lines in which line-specific alleles can be expected to prevail both at marker loci and QTL due to divergent selection histories. Moreover, in contrast to a gene-drop derived matrix, additional genotypes from later generations can easily be integrated by setting up a joint genomic relationship matrix for all genotyped individuals and joining it with Ã into H. Thus Ã leads to more realistic assumptions and more flexibility in the analysis of selection experiments if all founders are genotyped.</p></div></div><div id="sec-30" class="subsection"><h3>Conclusion</h3><p id="p-62">An approach for constructing expected autosomal and X-chromosomal genomic relationship matrices for founders from an arbitrary number of founder lines was developed. Extension to non-genotyped individuals in later generations can be performed by an adapted version of the tabular method using pedigree information. The resulting matrix Ã expresses relationships as average genomic relationships, as one would expect from repeated random sampling of alleles from founders at the observed frequencies. Implicitly, Ã accounts for any proportion of segregation variance between zero and one, which is not possible using only pedigree data. Observed marker data of founders and non-founders can then be combined into a joint relationship matrix, H, and its inverse can be used in mixed models for estimating the genetic variance in the crossbred population.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">Many thanks are to our colleagues from the mouse facility (LIN) in Dummerstorf for their technical assistance. J Meng gratefully acknowledges the financial support by the China Scholarship Council.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-64">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7110440">https://doi.org/10.25387/g3.7110440</a>.</p></li><li class="fn" id="fn-7"><p id="p-65"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-20"><span class="received-label">Received </span>September 20, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-16"><span class="accepted-label">Accepted </span>January 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Meng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuruta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hot topic: A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score1.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.2" data-doi="10.1262/jrd.09-224H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alm</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhlmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langhammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuchscherer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Occurrence of Polyovular Follicles in Mouse Lines Selected for High Fecundity.</span> <abbr class="cit-jnl-abbrev">J. Reprod. Dev.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1262/jrd.09-224H</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Breproduction%2Band%2Bdevelopment%26rft.stitle%253DJ%2BReprod%2BDev%26rft.aulast%253DAlm%26rft.auinit1%253DH.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D449%26rft.epage%253D453%26rft.atitle%253DOccurrence%2Bof%2Bpolyovular%2Bfollicles%2Bin%2Bmouse%2Blines%2Bselected%2Bfor%2Bhigh%2Bfecundity.%26rft_id%253Dinfo%253Adoi%252F10.1262%252Fjrd.09-224H%26rft_id%253Dinfo%253Apmid%252F20519833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1262/jrd.09-224H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20519833&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.3" data-doi="10.2527/jas1978.4661572x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallinga</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Politiek</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Reproduction and Body Weight of Mice after Long-Term Selection for Large Litter Size.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1978.4661572x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DBakker%26rft.auinit1%253DH.%26rft.volume%253D46%26rft.issue%253D6%26rft.spage%253D1572%26rft.epage%253D1580%26rft.atitle%253DReproduction%2Band%2BBody%2BWeight%2Bof%2BMice%2Bafter%2BLong-Term%2BSelection%2Bfor%2BLarge%2BLitter%2BSize%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1978.4661572x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1978.4661572x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FJ56300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.4" data-doi="10.1038/hdy.1992.136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beniwal</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of changes in genetic parameters in selected lines of mice using REML with an animal model. 2. Body weight, body composition and litter size.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1992.136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D69%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.136%26rft_id%253Dinfo%253Apmid%252F1428953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1992.136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1428953&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JQ49200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Selection for Litter Size in Mice in the Presence and Absence of Gonadotropin Treatment.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBradford%26rft.auinit1%253DG.%2BE.%26rft.volume%253D58%26rft.issue%253D2%26rft.spage%253D283%26rft.epage%253D295%26rft.atitle%253DSELECTION%2BFOR%2BLITTER%2BSIZE%2BIN%2BMICE%2BIN%2BTHE%2BPRESENCE%2BAND%2BABSENCE%2BOF%2BGONADOTROPIN%2BTREATMENT%26rft_id%253Dinfo%253Apmid%252F5664389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1OC8yLzI4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.6" data-doi="10.1186/1297-9686-44-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Compatibility of pedigree-based and marker-based relationship matrices for single-step genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">37</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DCompatibility%2Bof%2Bpedigree-based%2Band%2Bmarker-based%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Bgenetic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-37%26rft_id%253Dinfo%253Apmid%252F23206367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23206367&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.7" data-doi="10.1186/1297-9686-42-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic prediction when some animals are not genotyped.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DChristensen%26rft.auinit1%253DO.%2BF.%26rft.volume%253D42%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DGenomic%2Bprediction%2Bwhen%2Bsome%2Banimals%2Bare%2Bnot%2Bgenotyped.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-2%26rft_id%253Dinfo%253Apmid%252F20105297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105297&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.8" data-doi="10.1186/s12711-017-0363-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colleau</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palhière</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Ramilo</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A fast indirect method to compute functions of genomic relationships concerning genotyped and ungenotyped individuals, for diversity management.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0363-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0363-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0363-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.889.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">An introduction to population genetics theory</span>. Harper and Row, New York.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.10" data-doi="10.1093/oxfordjournals.jhered.a106039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruden</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The computation of inbreeding coefficients for closed populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a106039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DCRUDEN%26rft.auinit1%253DD.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D248%26rft.epage%253D251%26rft.atitle%253DTHE%2BCOMPUTATION%2BOF%2BINBREEDING%2BCOEFFICIENTS%253A%2BFor%2BClosed%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a106039%26rft_id%253Dinfo%253Apmid%252F18143392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a106039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18143392&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.12" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.13" data-doi="10.5194/aab-47-595-2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dietl</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langhammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renne</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Model simulations for genetic random drift in the outbred strain Fzt:DU.</span> <abbr class="cit-jnl-abbrev">Arch. Anim. Breed.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">604</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5194/aab-47-595-2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch.%2BAnim.%2BBreed.%26rft.volume%253D47%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.5194%252Faab-47-595-2004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/aab-47-595-2004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Single-Trait and Antagonistic Index Selection for Litter Size and Body Weight in Mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEisen%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D88%26rft.issue%253D4%26rft.spage%253D781%26rft.epage%253D811%26rft.atitle%253DSINGLE-TRAIT%2BAND%2BANTAGONISTIC%2BINDEX%2BSELECTION%2BFOR%2BLITTER%2BSIZE%2BAND%2BBODY%2BWEIGHT%2BIN%2BMICE%26rft_id%253Dinfo%253Apmid%252F17248819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODgvNC83ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.15" data-doi="10.1093/oxfordjournals.jhered.a105986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emik</span>  <span class="cit-name-given-names">L. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terrill</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">Systematic procedures for calculating inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a105986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DEMIK%26rft.auinit1%253DL.%2BO.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D51%26rft.epage%253D55%26rft.atitle%253DSYSTEMATIC%2BPROCEDURES%2BFOR%2BCALCULATING%2BINBREEDING%2BCOEFFICIENTS%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a105986%26rft_id%253Dinfo%253Apmid%252F18116093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a105986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18116093&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.16" data-doi="10.1007/BF02982939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Selection for large and small size in mice.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02982939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D51%26rft.spage%253D470%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02982939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02982939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1953XX17800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.17" data-doi="10.1002/jcp.1030560414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">The genetics of litter size in mice.</span> <abbr class="cit-jnl-abbrev">J. Cell. Comp. Physiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jcp.1030560414</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell.%2BComp.%2BPhysiol.%26rft.volume%253D56%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.1030560414%26rft_id%253Dinfo%253Apmid%252F13697995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.1030560414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13697995&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.18" data-doi="10.1007/BF00224018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Genetic evaluation with autosomal and X-chromosomal inheritance.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D80%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.19" data-doi="10.1186/s12711-017-0309-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Baccino</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pocrnic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Metafounders are related to Fst fixation indices and reduce bias in single-step genomic evaluations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0309-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0309-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0309-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.889.20"><div class="cit-metadata unstructured">Gilmour, A. R., B. J. Gogel, B. R. Cullis, S. J. Welham, and R. Thompson, 2015 ASReml User Guide Release 4.1. VSN Int. Ltd Hemel Hempstead UK 375.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.21" data-doi="10.1186/1297-9686-38-5-445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieto</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piqueras</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salgado</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic parameters for canalisation analysis of litter size and litter weight traits at birth in mice.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">445</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-38-5-445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGutierrez%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D445%26rft.epage%253D462%26rft.atitle%253DGenetic%2Bparameters%2Bfor%2Bcanalisation%2Banalysis%2Bof%2Blitter%2Bsize%2Band%2Blitter%2Bweight%2Btraits%2Bat%2Bbirth%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-38-5-445%26rft_id%253Dinfo%253Apmid%252F16954039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-38-5-445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16954039&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.22" data-doi="10.1017/S0016672300034352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulfield</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Rates of change of genetic parameters of body weight in selected mouse lines.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300034352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHeath%26rft.auinit1%253DS.%2BC.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D25%26rft.atitle%253DRates%2Bof%2Bchange%2Bof%2Bgenetic%2Bparameters%2Bof%2Bbody%2Bweight%2Bin%2Bselected%2Bmouse%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300034352%26rft_id%253Dinfo%253Apmid%252F8575666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300034352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8575666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RU68400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.23" data-doi="10.1111/j.1439-0388.2005.00526.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vangen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Long-term responses, changes in genetic variances and inbreeding depression from 122 generations of selection on increased litter size in mice.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2005.00526.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DHolt%26rft.auinit1%253DM.%26rft.volume%253D122%26rft.issue%253D3%26rft.spage%253D199%26rft.epage%253D209%26rft.atitle%253DLong-term%2Bresponses%252C%2Bchanges%2Bin%2Bgenetic%2Bvariances%2Band%2Binbreeding%2Bdepression%2Bfrom%2B122%2Bgenerations%2Bof%2Bselection%2Bon%2Bincreased%2Blitter%2Bsize%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2005.00526.x%26rft_id%253Dinfo%253Apmid%252F16130472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2005.00526.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16130472&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.24" data-doi="10.1590/S1415-47571998000200007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pletscher</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King-Ellison</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic variation in body weight gain and composition in the intercross of Large (LG/J) and Small (SM/J) inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genet. Mol. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S1415-47571998000200007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMol.%2BBiol.%26rft.volume%253D21%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1415-47571998000200007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1415-47571998000200007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075463600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">The Minimum Number of Genes Contributing to Quantitative Variation Between and Within Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D99%26rft.issue%253D3-4%26rft.spage%253D541%26rft.epage%253D553%26rft.atitle%253DTHE%2BMINIMUM%2BNUMBER%2BOF%2BGENES%2BCONTRIBUTING%2BTO%2BQUANTITATIVE%2BVARIATION%2BBETWEEN%2BAND%2BWITHIN%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F7343418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6Ijk5LzMtNC81NDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.26" data-doi="10.1530/REP-16-0434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langhammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartmann</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wudy</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobczak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Reproductive performance primarily depends on the female genotype in a two-factorial breeding experiment using high-fertility mouse lines.</span> <abbr class="cit-jnl-abbrev">Reproduction</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1530/REP-16-0434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%26rft.stitle%253DReproduction%26rft.aulast%253DLanghammer%26rft.auinit1%253DM.%26rft.volume%253D153%26rft.issue%253D3%26rft.spage%253D361%26rft.epage%253D368%26rft.atitle%253DReproductive%2Bperformance%2Bprimarily%2Bdepends%2Bon%2Bthe%2Bfemale%2Bgenotype%2Bin%2Ba%2Btwo-factorial%2Bbreeding%2Bexperiment%2Busing%2Bhigh-fertility%2Bmouse%2Blines%26rft_id%253Dinfo%253Adoi%252F10.1530%252FREP-16-0434%26rft_id%253Dinfo%253Apmid%252F28096494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjE1My8zLzM2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.27" data-doi="10.1530/REP-13-0425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langhammer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoeflich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobczak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-fertility phenotypes: two outbred mouse models exhibit substantially different molecular and physiological strategies warranting improved fertility.</span> <abbr class="cit-jnl-abbrev">Reproduction</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1530/REP-13-0425</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproduction%26rft_id%253Dinfo%253Adoi%252F10.1530%252FREP-13-0425%26rft_id%253Dinfo%253Apmid%252F24248751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicmVwcm9kIjtzOjU6InJlc2lkIjtzOjk6IjE0Ny80LzQyNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.28" data-doi="10.1016/j.tpb.2015.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparing estimates of genetic variance across different relationship models.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2015.08.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D107%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2015.08.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2015.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.29" data-doi="10.3168/jds.2009-2061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A relationship matrix including full pedigree and genomic information.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">4656</span>–<span class="cit-lpage">4663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2061</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DLegarra%26rft.auinit1%253DA.%26rft.volume%253D92%26rft.issue%253D9%26rft.spage%253D4656%26rft.epage%253D4663%26rft.atitle%253DA%2Brelationship%2Bmatrix%2Bincluding%2Bfull%2Bpedigree%2Band%2Bgenomic%2Binformation%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2061%26rft_id%253Dinfo%253Apmid%252F19700729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19700729&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156600059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.30" data-doi="10.1534/genetics.115.177014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misztal</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ancestral Relationships Using Metafounders: Finite Ancestral Populations and Across Population Relationships.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177014%26rft_id%253Dinfo%253Apmid%252F25873631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzIvNDU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.889.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-source">First Course in Population Genetics</span>, <span class="cit-publ-name">The Boxwood Press</span>, <span class="cit-publ-loc">Pacific Grove, CA</span>. Reprinted with corrections, 1978.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.32" data-doi="10.1007/BF00215087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Covariance between relatives in multibreed populations: additive model.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">430</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00215087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D87%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00215087%26rft_id%253Dinfo%253Apmid%252F24190314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00215087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24190314&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.33" data-doi="10.1111/j.1420-9101.2004.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peripato</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brito</span>  <span class="cit-name-given-names">R. A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matioli</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pletscher</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis affecting litter size in mice.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">602</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1420-9101.2004.00702.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DPeripato%26rft.auinit1%253DA.%2BC.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D593%26rft.epage%253D602%26rft.atitle%253DEpistasis%2Baffecting%2Blitter%2Bsize%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2004.00702.x%26rft_id%253Dinfo%253Apmid%252F15149402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2004.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15149402&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221197900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.34" data-doi="10.1101/gr.203001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashworth</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A Short Pseudoautosomal Region in Laboratory Mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1826</span>–<span class="cit-lpage">1832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.203001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DPerry%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1826%26rft.epage%253D1832%26rft.atitle%253DA%2BShort%2BPseudoautosomal%2BRegion%2Bin%2BLaboratory%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.203001%26rft_id%253Dinfo%253Apmid%252F11691846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMS8xODI2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.35" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2865</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.36" data-doi="10.3168/jds.2011-4338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reliability of direct genomic values for animals with different relationships within and to the reference population.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-4338</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPszczola%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D400%26rft.atitle%253DReliability%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bfor%2Banimals%2Bwith%2Bdifferent%2Brelationships%2Bwithin%2Band%2Bto%2Bthe%2Breference%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4338%26rft_id%253Dinfo%253Apmid%252F22192218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192218&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.37" data-doi="10.1534/genetics.107.083162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnegård</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved method for quantitative trait loci detection and identification of within-line segregation in F2 intercross designs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2315</span>–<span class="cit-lpage">2326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.083162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRonnegard%26rft.auinit1%253DL.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2315%26rft.epage%253D2326%26rft.atitle%253DAn%2BImproved%2BMethod%2Bfor%2BQuantitative%2BTrait%2BLoci%2BDetection%2Band%2BIdentification%2Bof%2BWithin-Line%2BSegregation%2Bin%2BF2%2BIntercross%2BDesigns%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083162%26rft_id%253Dinfo%253Apmid%252F18430952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIzMTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.889.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-source">Matrix algebra useful for statistics</span>, <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.39" data-doi="10.1080/01621459.1987.10478472"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Self</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">K.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Asymptotic Properties of Maximum Likelihood Estimators and Likelihood Ratio Tests under Nonstandard Conditions.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1987.10478472</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D82%26rft.spage%253D605%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1987.10478472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1987.10478472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.40" data-doi="10.1186/1297-9686-36-6-621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuchscherer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinsch</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of gametes and treatment of linear dependencies in the gametic QTL-relationship matrix and its inverse.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-36-6-621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DTuchscherer%26rft.auinit1%253DA.%26rft.volume%253D36%26rft.issue%253D6%26rft.spage%253D621%26rft.epage%253D642%26rft.atitle%253DIdentification%2Bof%2Bgametes%2Band%2Btreatment%2Bof%2Blinear%2Bdependencies%2Bin%2Bthe%2Bgametic%2BQTL-relationship%2Bmatrix%2Band%2Bits%2Binverse.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-36-6-621%26rft_id%253Dinfo%253Apmid%252F15496284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-36-6-621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15496284&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.41" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient Methods to Compute Genomic Predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.889.42" data-doi="10.1093/oso/9780198830870.001.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">Evolution and Selection of Quantitative Traits</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oso/9780198830870.001.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Band%2BSelection%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foso%252F9780198830870.001.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oso/9780198830870.001.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.43" data-doi="10.1214/11-EJS654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiencierz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greven</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Küchenhoff</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Restricted likelihood ratio testing in linear mixed models with general error covariance structure.</span> <abbr class="cit-jnl-abbrev">Electron. J. Stat.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1718</span>–<span class="cit-lpage">1734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/11-EJS654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BStat.%26rft.volume%253D5%26rft.spage%253D1718%26rft_id%253Dinfo%253Adoi%252F10.1214%252F11-EJS654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/11-EJS654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.44" data-doi="10.1093/oxfordjournals.jhered.a108156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodale</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyle</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godfrey</span>  <span class="cit-name-given-names">E. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Long Term Selection for Body Weight in Mice.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">228</span>–<span class="cit-lpage">234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a108156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DWILSON%26rft.auinit1%253DS.%2BP.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D228%26rft.epage%253D234%26rft.atitle%253DLong%2BTerm%2BSelection%2Bfor%2BBody%2BWeight%2Bin%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a108156%26rft_id%253Dinfo%253Apmid%252F5137356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a108156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5137356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971K705400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.889.45" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F889.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/889.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/879" title="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/901" title="The mir-35 Family Links Maternal Germline Sex to Embryonic Viability in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/889" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RmHBBEgZCR2AFm23seXvKjvjQkkQe2FlmKbsl-EMwO0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/889" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/889" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/889" class="active" data-icon-position="" data-hide-link-title="0">Turning Observed Founder Alleles into Expected Relationships in an Intercross Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492531" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MLe-wIMw3WY2bG0tUJIw1hbcWkXYpf-Pbt3V4wabC9I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/889.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/889" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-P_OHM1K7d5Q1f5w_FpfJ6IedHDGk6VH-qswIzg5mZik" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/889" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/889.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492531" id="node492531--2" data-pisa="ggg;9/3/889" data-pisa-master="ggg;g3.118.200752" data-apath="/ggg/9/3/889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Turning Observed Founder Alleles into Expected Relationships in an Intercross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jilun Meng</span>, <span class="highwire-citation-author" data-delta="1">Manfred Mayer</span>, <span class="highwire-citation-author" data-delta="2">Erika Wytrwat</span>, <span class="highwire-citation-author" data-delta="3">Martina Langhammer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">889-899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492531--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jilun Meng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jilun%2BMeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJilun%2BMeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erika Wytrwat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erika%2BWytrwat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wytrwat%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErika%2BWytrwat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martina Langhammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BLanghammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langhammer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartina%2BLanghammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Reinsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reinsch@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">reinsch@fbn-dummerstorf.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/889" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/889&amp;text=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/889&amp;t=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/889&amp;title=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492531/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492531" id="node492531--3" data-pisa="ggg;9/3/889" data-pisa-master="ggg;g3.118.200752" data-apath="/ggg/9/3/889.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Turning Observed Founder Alleles into Expected Relationships in an Intercross Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jilun Meng</span>, <span class="highwire-citation-author" data-delta="1">Manfred Mayer</span>, <span class="highwire-citation-author" data-delta="2">Erika Wytrwat</span>, <span class="highwire-citation-author" data-delta="3">Martina Langhammer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Norbert Reinsch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">889-899;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200752 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492531--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jilun Meng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jilun%2BMeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJilun%2BMeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Manfred Mayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BMayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mayer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AManfred%2BMayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erika Wytrwat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erika%2BWytrwat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wytrwat%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErika%2BWytrwat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martina Langhammer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martina%2BLanghammer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Langhammer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartina%2BLanghammer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Norbert Reinsch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Genetics and Biometry, Leibniz Institute for Farm Animal Biology (FBN), 18196 Dummerstorf, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Norbert%2BReinsch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinsch%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANorbert%2BReinsch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7435-4088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Norbert Reinsch</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reinsch@fbn-dummerstorf.de" class="" data-icon-position="" data-hide-link-title="0">reinsch@fbn-dummerstorf.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=889&amp;rft.epage=899&amp;rft.atitle=Turning%20Observed%20Founder%20Alleles%20into%20Expected%20Relationships%20in%20an%20Intercross%20Population&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Meng%2C+Jilun&amp;rft.au=Mayer%2C+Manfred&amp;rft.au=Wytrwat%2C+Erika&amp;rft.au=Langhammer%2C+Martina&amp;rft.au=Reinsch%2C+Norbert"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492531/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492531/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492531/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492531/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492531/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492531/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492531/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492531/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492531/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492531/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492531/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492531/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/889" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Theory</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application Example</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/889.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/889.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/889.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/889.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
