<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/789" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/789.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/789.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/789.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/789/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/789.atom" />
<meta name="HW.pisa" content="ggg;9/3/789" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample" />
<meta name="DC.Identifier" content="10.1534/g3.118.200969" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Ramstein, Casler. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic prediction is a useful tool to accelerate genetic gain in selection using DNA marker information. However, this technology typically relies on standard prediction procedures, such as genomic BLUP, that are not designed to accommodate population heterogeneity resulting from differences in marker effects across populations. In this study, we assayed different prediction procedures to capture marker-by-population interactions in genomic prediction models. Prediction procedures included genomic BLUP and two kernel-based extensions of genomic BLUP which explicitly accounted for population heterogeneity. To model population heterogeneity, dissemblance between populations was either depicted by a unique coefficient (as previously reported), or a more flexible function of genetic distance between populations (proposed herein). Models under investigation were applied in a diverse switchgrass sample under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is entirely excluded from the calibration set. First, we showed that using fixed effects, from principal components or putative population groups, appeared detrimental to prediction accuracy, especially in cross-population calibration. Then we showed that modeling population heterogeneity by our proposed procedure resulted in highly significant improvements in model fit. In such cases, gains in accuracy were often positive. These results suggest that population heterogeneity may be parsimoniously captured by kernel methods. However, in cases where improvement in model fit by our proposed procedure is null-to-moderate, ignoring heterogeneity should probably be preferred due to the robustness and simplicity of the standard genomic BLUP model." />
<meta name="DC.Contributor" content="Guillaume P. Ramstein" />
<meta name="DC.Contributor" content="Michael D. Casler" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic prediction is a useful tool to accelerate genetic gain in selection using DNA marker information. However, this technology typically relies on standard prediction procedures, such as genomic BLUP, that are not designed to accommodate population heterogeneity resulting from differences in marker effects across populations. In this study, we assayed different prediction procedures to capture marker-by-population interactions in genomic prediction models. Prediction procedures included genomic BLUP and two kernel-based extensions of genomic BLUP which explicitly accounted for population heterogeneity. To model population heterogeneity, dissemblance between populations was either depicted by a unique coefficient (as previously reported), or a more flexible function of genetic distance between populations (proposed herein). Models under investigation were applied in a diverse switchgrass sample under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is entirely excluded from the calibration set. First, we showed that using fixed effects, from principal components or putative population groups, appeared detrimental to prediction accuracy, especially in cross-population calibration. Then we showed that modeling population heterogeneity by our proposed procedure resulted in highly significant improvements in model fit. In such cases, gains in accuracy were often positive. These results suggest that population heterogeneity may be parsimoniously captured by kernel methods. However, in cases where improvement in model fit by our proposed procedure is null-to-moderate, ignoring heterogeneity should probably be preferred due to the robustness and simplicity of the standard genomic BLUP model.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/789" />
<meta name="citation_id" content="9/3/789" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/789" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/789.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/789.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/789.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200969" />
<meta name="citation_pmid" content="30651285" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="789" />
<meta name="citation_lastpage" content="805" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Guillaume P. Ramstein" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706" />
<meta name="citation_author_email" content="gr226@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7536-1113" />
<meta name="citation_author" content="Michael D. Casler" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706" />
<meta name="citation_author_institution" content="Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706" />
<meta name="citation_reference" content="citation_title=Pocketbook of mathematical functions;citation_author=M. Abramowitz;citation_author=I. A. Stegun;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. H. Alexander;citation_author=K. Lange;citation_title=Enhancements to the ADMIXTURE algorithm for individual ancestry estimation.;citation_pages=246-246;citation_volume=12;citation_year=2011;citation_pmid=21682921;citation_doi=10.1186/1471-2105-12-246" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H.-J. Auinger;citation_author=M. Schönleben;citation_author=C. Lehermeier;citation_author=M. Schmidt;citation_author=V. Korzun;citation_title=Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).;citation_pages=2043-2053;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2756-5" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Bryc;citation_author=A. Auton;citation_author=M. R. Nelson;citation_author=J. R. Oksenberg;citation_author=S. L. Hauser;citation_title=Genome-wide patterns of population structure and admixture in West Africans and African Americans.;citation_volume=107;citation_year=2010;citation_pmid=20080753;citation_doi=10.1073/pnas.0909559107" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual. The State of Queensland;citation_author=D. G. Butler;citation_author=B. R. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Carillier;citation_author=H. Larroque;citation_author=C. Robert-Granié;citation_title=Comparison of joint vs. purebred genomic evaluation in the French multi-breed dairy goat population.;citation_pages=67;citation_volume=46;citation_year=2014;citation_doi=10.1186/s12711-014-0067-3" />
<meta name="citation_reference" content="citation_title=Statistical inference;citation_author=G. Casella;citation_author=R. L. Berger;citation_year=2002" />
<meta name="citation_reference" content="citation_title=Switchgrass: A Valuable Biomass Crop for Energy;citation_author=M. D. Casler;citation_title=Switchgrass Breeding, Genetics, and Genomics;citation_year=2012;citation_doi=10.1007/978-1-4471-2903-5_2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_title=Selection for Biomass Yield in Upland, Lowland, and Hybrid Switchgrass.;citation_pages=626-636;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.04.0239" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. P.Vogel. Casler;citation_author=A. C. Beal;citation_title=Registration of WS4U and WS8U Switchgrass Germplasms;citation_pages=998-999;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.04-0020" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_title=AN EXTENSION OF THE CONCEPT OF PARTITIONING HEREDITARY VARIANCE FOR ANALYSIS OF COVARIANCES AMONG RELATIVES WHEN EPISTASIS IS PRESENT;citation_pages=859-882;citation_volume=39;citation_year=1954;citation_issue=6;citation_pmid=17247525" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=M. P. Conomos;citation_author=A. P. Reiner;citation_author=B. S. Weir;citation_author=T. A. Thornton;citation_title=Model-free Estimation of Recent Genetic Relatedness.;citation_pages=127-148;citation_volume=98;citation_year=2016;citation_pmid=26748516;citation_doi=10.1016/j.ajhg.2015.11.022" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Coop;citation_author=J. K. Pickrell;citation_author=J. Novembre;citation_author=S. Kudaravalli;citation_author=J. Li;citation_title=The role of geography in human adaptation.;citation_pages=e1000500-e1000500;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19503611;citation_doi=10.1371/journal.pgen.1000500" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=H. D. Daetwyler;citation_author=A. A. Swan;citation_author=J. H. J. van der Werf;citation_author=B. J. Hayes;citation_title=Accuracy of pedigree and genomic predictions of carcass and novel meat quality traits in multi-breed sheep data assessed by cross-validation.;citation_pages=33-33;citation_volume=44;citation_year=2012;citation_pmid=23146144;citation_doi=10.1186/1297-9686-44-33" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. C. Dawson;citation_author=J. B. Endelman;citation_author=N. Heslot;citation_author=J. Crossa;citation_author=J. Poland;citation_title=The use of unbalanced historical data for genomic selection in an international wheat breeding program.;citation_pages=12-22;citation_volume=154;citation_year=2013;citation_doi=10.1016/j.fcr.2013.07.020" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=G. de Los Campos;citation_author=Y. Veturi;citation_author=A. I. Vazquez;citation_author=C. Lehermeier;citation_author=P. Pérez-Rodríguez;citation_title=Incorporating Genetic Heterogeneity in Whole-Genome Regressions Using Interactions.;citation_pages=467-490;citation_volume=20;citation_year=2015;citation_pmid=26660276;citation_doi=10.1007/s13253-015-0222-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix.;citation_pages=1405-1413;citation_volume=2;citation_year=2012;citation_pmid=23173092;citation_doi=10.1534/g3.112.004259" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. J. Hayes;citation_author=L. K. Matukumalli;citation_author=S. Goswami;citation_author=P. J. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Evans;citation_author=E. Crisovan;citation_author=K. Barry;citation_author=C. Daum;citation_author=J. Jenkins;citation_title=Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.;citation_pages=800-815;citation_volume=84;citation_year=2015;citation_pmid=26426343;citation_doi=10.1111/tpj.13041" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Evans;citation_author=J. Kim;citation_author=K. L. Childs;citation_author=B. Vaillancourt;citation_author=E. Crisovan;citation_title=Nucleotide polymorphism and copy number variant detection using exome capture and next-generation sequencing in the polyploid grass Panicum virgatum.;citation_pages=993-1008;citation_volume=79;citation_year=2014;citation_pmid=24947485;citation_doi=10.1111/tpj.12601" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=J. Fan;citation_author=Y. Liao;citation_author=M. Mincheva;citation_title=Large Covariance Estimation by Thresholding Principal Orthogonal Complements.;citation_pages=603-680;citation_volume=75;citation_year=2013;citation_issue=4;citation_doi=10.1111/rssb.12016" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Sparse inverse covariance estimation with the graphical lasso.;citation_volume=9;citation_year=2008;citation_pmid=18079126;citation_doi=10.1093/biostatistics/kxm045" />
<meta name="citation_reference" content="citation_journal_title=Softw. Pract. Exper.;citation_author=T. M. J. Fruchterman;citation_author=E. M. Reingold;citation_title=Graph drawing by force-directed placement.;citation_pages=1129-1164;citation_volume=21;citation_year=1991;citation_doi=10.1002/spe.4380211102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive Genetic Variability and the Bayesian Alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. P. Grabowski;citation_author=G. P. Morris;citation_author=M. D. Casler;citation_author=J. O. Borevitz;citation_title=Population genomic variation reveals roles of history, adaptation and ploidy in switchgrass.;citation_pages=4059-4073;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12845" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=C. J. Basten;citation_author=H. Gandhi;citation_author=E. Ersoz;citation_title=The impact of population structure on genomic prediction in stratified populations.;citation_pages=749-762;citation_volume=127;citation_year=2014;citation_pmid=24452438;citation_doi=10.1007/s00122-013-2255-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_title=An alternative covariance estimator to investigate genetic heterogeneity in populations.;citation_pages=93;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0171-z" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=W. G. Hill;citation_author=A. Mäki-Tanila;citation_title=Expected influence of linkage disequilibrium on genetic variance caused by dominance and epistasis on quantitative traits.;citation_pages=176-186;citation_volume=132;citation_year=2015;citation_doi=10.1111/jbg.12140" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=C. D. Hirsch;citation_author=J. Evans;citation_author=C. R. Buell;citation_author=C. N. Hirsch;citation_title=Reduced representation approaches to interrogate genome diversity in large repetitive plant genomes.;citation_volume=13;citation_year=2014;citation_pmid=24395692;citation_doi=10.1093/bfgp/elt051" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Jarquin;citation_author=J. Specht;citation_author=A. Lorenz;citation_title=Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions.;citation_volume=6;citation_year=2016;citation_pmid=27247288;citation_doi=10.1534/g3.116.031443" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Karoui;citation_author=M. J. Carabaño;citation_author=C. Díaz;citation_author=A. Legarra;citation_title=Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.;citation_pages=39;citation_volume=44;citation_year=2012;citation_pmid=23216664;citation_doi=10.1186/1297-9686-44-39" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Lond. B Biol. Sci.;citation_author=O. Kempthorne;citation_title=The correlation between relatives in a random mating population.;citation_pages=102-113;citation_volume=143;citation_year=1954;citation_pmid=13224653;citation_doi=10.1098/rspb.1954.0056" />
<meta name="citation_reference" content="Langholtz, M. H., B. J. Stokes, L. M. Eaton, C. C. Brandt, M. R. Davis et al., 2016 2016 Billion-ton report: Advancing domestic resources for a thriving bioeconomy, volume 1: Economic availability of feedstocks: Oak Ridge National Lab. (ORNL), Oak Ridge, TN (United States)." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Lehermeier;citation_author=C.-C. Schön;citation_author=G. de Los Campos;citation_title=Assessment of Genetic Heterogeneity in Structured Plant Populations Using Multivariate Whole-Genome Regression Models.;citation_volume=201;citation_year=2015;citation_pmid=26122758;citation_doi=10.1534/genetics.115.177394" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. Li;citation_author=D. D. Serba;citation_author=M. C. Saha;citation_author=J. H. Bouton;citation_author=C. L. Lanzatella;citation_title=Genetic linkage mapping and transmission ratio distortion in a three-generation four-founder population of Panicum virgatum (L.).;citation_volume=4;citation_year=2014;citation_pmid=24637352;citation_doi=10.1534/g3.113.010165" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Lipka;citation_author=F. Lu;citation_author=J. H. Cherney;citation_author=E. S. Buckler;citation_author=M. D. Casler;citation_title=Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.;citation_pages=e112227;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0112227" />
<meta name="citation_reference" content="citation_title=Advances in Agronomy;citation_title=Chapter Two - Genomic Selection in Plant Breeding: Knowledge and Prospects;citation_author=A. J. Lorenz;citation_author=S. Chao;citation_author=F. G. Asoro;citation_author=E. L. Heffner;citation_author=T. Hayashi;citation_pages=77-123;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. S. Lund;citation_author=G. Su;citation_author=L. Janss;citation_author=B. Guldbrandtsen;citation_author=R. F. Brøndum;citation_title=Genomic evaluation of cattle in a multi-breed context.;citation_pages=101-110;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Mäki-Tanila;citation_author=W. G. Hill;citation_title=Influence of gene interaction on complex trait variation with multilocus models.;citation_volume=198;citation_year=2014;citation_pmid=24990992;citation_doi=10.1534/genetics.114.165282" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. R. Martin;citation_author=D. D. Kinnamon;citation_author=M. A. Schmidt;citation_author=E. H. Powell;citation_author=S. Zuchner;citation_title=SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.;citation_volume=26;citation_year=2010;citation_pmid=20861027;citation_doi=10.1093/bioinformatics/btq526" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=R. B. Mitchell;citation_author=K. J. Moore;citation_author=L. E. Moser;citation_author=J. O. Fritz;citation_author=D. D. Redfearn;citation_title=Predicting Developmental Morphology in Switchgrass and Big Bluestem;citation_pages=827-832;citation_volume=89;citation_year=1997;citation_issue=5;citation_doi=10.2134/agronj1997.00021962008900050018x" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=C. Nadeau;citation_author=Y. Bengio;citation_title=Inference for the Generalization Error.;citation_pages=239-281;citation_volume=52;citation_year=2003;citation_doi=10.1023/A:1024068626366" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Novembre;citation_author=T. Johnson;citation_author=K. Bryc;citation_author=Z. Kutalik;citation_author=A. R. Boyko;citation_title=Genes mirror geography within Europe.;citation_pages=98-101;citation_volume=456;citation_year=2008;citation_issue=7218;citation_pmid=18758442;citation_doi=10.1038/nature07331" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=U. Ober;citation_author=M. Erbe;citation_author=N. Long;citation_author=E. Porcu;citation_author=M. Schlather;citation_title=Predicting genetic values: a kernel-based best linear unbiased prediction with genomic data.;citation_volume=188;citation_year=2011;citation_pmid=21515573;citation_doi=10.1534/genetics.111.128694" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Okada;citation_author=C. Lanzatella;citation_author=M. C. Saha;citation_author=J. Bouton;citation_author=R. Wu;citation_title=Complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions.;citation_volume=185;citation_year=2010;citation_pmid=20407132;citation_doi=10.1534/genetics.110.113910" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Z. Ou;citation_author=R. J. Tempelman;citation_author=J. P. Steibel;citation_author=C. W. Ernst;citation_author=R. O. Bates;citation_title=Genomic Prediction Accounting for Residual Heteroskedasticity.;citation_volume=6;citation_year=2015;citation_pmid=26564950;citation_doi=10.1534/g3.115.022897" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=F. Phocas;citation_author=D. Laloë;citation_title=Should genetic groups be fitted in BLUP evaluation? Practical answer for the French AI beef sire evaluation.;citation_pages=325-345;citation_volume=36;citation_year=2004;citation_issue=3;citation_pmid=15107269;citation_doi=10.1051/gse:2004004" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. P. Ramstein;citation_author=J. Evans;citation_author=S. M. Kaeppler;citation_author=R. B. Mitchell;citation_author=K. P. Vogel;citation_title=Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium.;citation_volume=6;citation_year=2016;citation_pmid=26869619;citation_doi=10.1534/g3.115.024950" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Ruiz-Linares;citation_author=K. Adhikari;citation_author=V. Acuña-Alonzo;citation_author=M. Quinto-Sanchez;citation_author=C. Jaramillo;citation_title=Admixture in Latin America: geographic structure, phenotypic diversity and self-perception of ancestry based on 7,342 individuals.;citation_pages=e1004572;citation_volume=10;citation_year=2014;citation_pmid=25254375;citation_doi=10.1371/journal.pgen.1004572" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Efficient Use of Historical Data for Genomic Selection: A Case Study of Stem Rust Resistance in Wheat.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.09.0046" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_author=K. P. Smith;citation_title=Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.;citation_pages=1-15;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=Bioresour. Technol.;citation_author=M. A. Sanderson;citation_author=R. L. Reed;citation_author=S. B. McLaughlin;citation_author=S. D. Wullschleger;citation_author=B. V. Conger;citation_title=Switchgrass as a sustainable bioenergy crop.;citation_pages=83-93;citation_volume=56;citation_year=1996;citation_doi=10.1016/0960-8524(95)00176-X" />
<meta name="citation_reference" content="citation_title=Variance Components;citation_author=S. R. Searle;citation_author=G. Casella;citation_author=C. E. McCulloch;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=D. Speed;citation_author=G. Hemani;citation_author=M. R. Johnson;citation_author=D. J. Balding;citation_title=Improved heritability estimation from genome-wide SNPs.;citation_pages=1011-1021;citation_volume=91;citation_year=2012;citation_pmid=23217325;citation_doi=10.1016/j.ajhg.2012.10.010" />
<meta name="citation_reference" content="citation_title=Principles and procedures of statistics A biometrical approach;citation_author=R. Steel;citation_author=J. H. Torrie;citation_author=D. A. Dickey;citation_pages=334-381;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=F. Technow;citation_author=A. Bürger;citation_author=A. E. Melchinger;citation_title=Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups.;citation_pages=197-203;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.112.004630" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Thornton;citation_author=H. Tang;citation_author=T. J. Hoffmann;citation_author=H. M. Ochs-Balcom;citation_author=B. J. Caan;citation_title=Estimating kinship in admixed populations.;citation_pages=122-138;citation_volume=91;citation_year=2012;citation_issue=1;citation_pmid=22748210;citation_doi=10.1016/j.ajhg.2012.05.024" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. A. M. L. Uitdewilligen;citation_author=A.-M. A. Wolters;citation_author=B. B. D’hoop;citation_author=T. J. A. Borm;citation_author=R. G. F. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Wang;citation_author=S. Sverdlov;citation_author=E. Thompson;citation_title=Efficient Estimation of Realized Kinship from Single Nucleotide Polymorphism Genotypes.;citation_volume=205;citation_year=2017;citation_pmid=28100587;citation_doi=10.1534/genetics.116.197004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. J. Wientjes;citation_author=P. Bijma;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_title=An Equation to Predict the Accuracy of Genomic Values by Combining Data from Multiple Traits, Populations, or Environments.;citation_volume=202;citation_year=2016;citation_pmid=26637542;citation_doi=10.1534/genetics.115.183269" />
<meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.;citation_author=T. Zhao;citation_author=H. Liu;citation_author=K. Roeder;citation_author=J. Lafferty;citation_author=L. Wasserman;citation_title=The huge Package for High-dimensional Undirected Graph Estimation in R.;citation_pages=1059-1062;citation_volume=13;citation_year=2012;citation_pmid=26834510" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=L. Zhou;citation_author=M. S. Lund;citation_author=Y. Wang;citation_author=G. Su;citation_title=Genomic predictions across Nordic Holstein and Nordic Red using the genomic best linear unbiased prediction model with different genomic relationship matrices.;citation_pages=249-257;citation_volume=131;citation_year=2014;citation_pmid=24750283;citation_doi=10.1111/jbg.12089" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Genomic prediction is a useful tool to accelerate genetic gain in selection using DNA marker information. However, this technology typically relies on standard prediction procedures, such as genomic BLUP, that are not designed to accommodate population heterogeneity resulting from differences in marker effects across populations. In this study, we assayed different prediction procedures to capture marker-by-population interactions in genomic prediction models. Prediction procedures included genomic BLUP and two kernel-based extensions of genomic BLUP which explicitly accounted for population heterogeneity. To model population heterogeneity, dissemblance between populations was either depicted by a unique coefficient (as previously reported), or a more flexible function of genetic distance between populations (proposed herein). Models under investigation were applied in a diverse switchgrass sample under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is entirely excluded from the calibration set. First, we showed that using fixed effects, from principal components or putative population groups, appeared detrimental to prediction accuracy, especially in cross-population calibration. Then we showed that modeling population heterogeneity by our proposed procedure resulted in highly significant improvements in model fit. In such cases, gains in accuracy were often positive. These results suggest that population heterogeneity may be parsimoniously captured by kernel methods. However, in cases where improvement in model fit by our proposed procedure is null-to-moderate, ignoring heterogeneity should probably be preferred due to the robustness and simplicity of the standard genomic BLUP model." />
<meta name="og-title" property="og:title" content="Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/789" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic prediction is a useful tool to accelerate genetic gain in selection using DNA marker information. However, this technology typically relies on standard prediction procedures, such as genomic BLUP, that are not designed to accommodate population heterogeneity resulting from differences in marker effects across populations. In this study, we assayed different prediction procedures to capture marker-by-population interactions in genomic prediction models. Prediction procedures included genomic BLUP and two kernel-based extensions of genomic BLUP which explicitly accounted for population heterogeneity. To model population heterogeneity, dissemblance between populations was either depicted by a unique coefficient (as previously reported), or a more flexible function of genetic distance between populations (proposed herein). Models under investigation were applied in a diverse switchgrass sample under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is entirely excluded from the calibration set. First, we showed that using fixed effects, from principal components or putative population groups, appeared detrimental to prediction accuracy, especially in cross-population calibration. Then we showed that modeling population heterogeneity by our proposed procedure resulted in highly significant improvements in model fit. In such cases, gains in accuracy were often positive. These results suggest that population heterogeneity may be parsimoniously captured by kernel methods. However, in cases where improvement in model fit by our proposed procedure is null-to-moderate, ignoring heterogeneity should probably be preferred due to the robustness and simplicity of the standard genomic BLUP model." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"siCcGj5T9bgyQZlPnLQ-Q4Macr8QbzLk9jxHEy8XzdA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492412","apath":"\/ggg\/9\/3\/789.atom","pisa":"ggg;9\/3\/789","ac":{"\/ggg\/9\/3\/789.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/789.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/789"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/789"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/789":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OtLLA32Id97Gy75C9MM7T35l0jr+PC7IzllWWjtBHdPAtPoNuG7vwFL1HraVmZwlzCLRFx+UTe19t4Uo4Sk8kosW9P6ba\\\\\\\/0qRBt+OBPQMWiT0bauy2CtTksjg1DiEiWCJCGZTJwaXGF3G7BCAwQn1fFhy248dbo\\\\\\\/HhblXlFUo0NzZBnaEOCw+jlnBGy9hbo\\\\\\\/9QUl+wgDevyLOtNgObmh0jdcwrCKP5jQCKfQu5C9LPOVzwCa8NrOZanBDK981CMjRga+l6shbvkaU0LhjmQvdqyozFRkg4pFHAzikq3YugtpNeZ+fax72qWg0Als+37JiTv6tOVpMAC4RCJOTaoYTbbkMbcTUJ\\\\\\\/d1403IEs\\\\\\\/wyEDRElIphCd\\\\\\\/ZahO+su\\\\\\\/4f2mNyGFe\\\\\\\/zH73sXXhERLLoxcHJ9wdcHTz7GWmgEEwUh8mR7RHizi96CRxXN8tjhtSzvozNDvHTVxjCMd7gUOJONBXqoXYmr2OzzmBvOnCU43+YNRCOuslCpS1clhMpJBrfLPvPqix5hmbjCKD0s8sG7jq3oXCZx+QeS59ZvPKtol30G3XoghmLaI1JNKDZfjO3TzHaNUH52JPK5ISOOuSRMyxf90yCowzlvUhJwfUltPCgGapCNiQ8Ba5oqpuJNTYvIQWY4BvJIWMMS4XNWy\\\\\\\/+Qj3gzCOOmbTOIsRHU9bXjE8th2TxSdOtxeKKbDGFjmFsRVrTtY0Z64iIzplOKc70uzQqSCAKBQ65QH96kF+dHmpM2kEAsu+dvDumHTxsYmugn5UdbdRsa4lqcaenNC1P9qXWcdNNne+6fgvSukjQV87vIGADlg8gxHY2\\\\\\\/JPXGNMsrDHTQQTI8KeqHI45f+Tp7ky1IKbTxPUIfOL+HgofZENSS7690nfuy4Bo1rYWwufjFD+c4Q\\\\\\\/giJCYGbglcb13T8STvO+28OhvqKPaBa8Zpyd2RD4tviroqAfpZ1YW0YKvu8JWhEcygGf0ExvnNBXqRXMq0Dxjja0XrqNUkv9tkA+RNvqOQCmu9A1lkc\\\\\\\/sqRnkzXC8KbtcUZaznCGG9bm0AlXMB4nKcvCfx1ISlGR74W\\\\\\\/OMgXZI9F0fopRKd5Bs+vUJOZF83rGwcFTyRySuSIHUFMmvrqMn8iGquATyF42xfUJftgpUTvoQ\\\\\\\/bUitUrpeOquFJ11B6Lrbxl5hZidKkB7q6l+UIgMLmPIxbJuWlb9pYI\\\\\\\/0UtobYvYxDFqNuRA1HzHYdt47R9slQQBvkWV3GO8K1zS0XDjTha1GbJitWPHbG4BK58bFPZYTnSpeixB\\\\\\\/\\\\\\\/K\\\u0022,\\\u0022iv\\\u0022:\\\u0022pnQJupVA04wmFQsbGqZvOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231575a410047b0994ae863a5afbf6499233b1b18d43e88e614cddf1f6cae3b5a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NlgyA+B4OcnXIDuGEqv48QilAoRFVKH+hy3DCv3WhdLnSW6TWc\\\\\\\/6JcG2zGAfqsmB1O38E0pMYYidqhi90J2ClCzSDhnYO3pVz\\\\\\\/NIAo\\\\\\\/p2wekbaqyaVNnYCpz80h22u1V1Usns8TELH\\\\\\\/YbY5hQirA2Xypj5KjZc+FiHXor5LCoTebS0Vp\\\\\\\/0MS1k\\\\\\\/ARdsyNlG5mBSCR26Nwk8RAdC4mYTunBLCeTmUjI37Dn8AFy3Vx0JWqBvj1g603eKO9FNFVu7XkH5KcMxdahG8VxixHCqHgrr1o8V8Jm5O5Jq8bdzFjGQS9yEEAAVZNTb8ZkvQ6iPWetQrdNRoSd68qJGYaBOkwN\\\\\\\/eeTSiCZI09wrhU9VXPgJz5LnJhYaNKpcM2MMNOZwv4dRsB\\\\\\\/P6RO659qme3dJq6GD4aOvRhX2DJQE3dLMvxvD6WsuIhwDMKla2tWgLscn2r7ZiNuMfqoNG++E0eTHY\\\\\\\/pIOQ82ZQT39R4\\\\\\\/ENhlD9OMLSs4+874hq1kQu+o0gFOh0BYW5xjXwjbKogXA46U7xAhSRZq1U\\\\\\\/GsRUmWEH9XCIL7AW2A7X7PCELV\\\\\\\/lvBLnuo2qyfMfytqyxYg5Ld0+dpSQrJkZoAUp60vxDrhF\\\\\\\/85lUlAv\\\\\\\/Y6MI9k3OkbYdEh07yM3Q9b5XilrzpCuNcco4NJQQ+HjYbAP67LcF3DkxwYLT5c7LYvOMkXegAN2L9C4NsOV3ESpA\\\\\\\/dRaPdMNbXTczlIaVueT1s+rFiYaOJlIfS+U18cdVKEVLmChAZeJRay\\\\\\\/SK8dZ4QFrrL3Eqr2g9vtNfjOFE7beBNg3uSFAOkGSw1FASafXnH9BEyXJWqOJ4ZXjHP2q5f13WkZfhH3NcH74CpoS6bbtYVE5wgTqsnAT\\\\\\\/2E7Hj3dEEHeUHOvPIcyRDVYtoJKfshiGE3rR0nFT4X0T8pZpwWdF4QpXBXQb3udNw\\\\\\\/Y5gDuHuz1Zif3SNlFQxT2eDm\\\\\\\/QZcMPakzg+PXNgmQeLBvh2xa4sOS2k3ArG32pFojXIX0\\\\\\\/u6ftcMZMIkEnyOAt7Cx5JQ4MSlF02af0cv31v356gIEBp5Gs0qiivYJyKxAO6WivP1r\\\\\\\/YEZOnW2S9GmhGeIOxaDESSCjMD3svRyLhswlhwDZaad+X2fvJAg+OxcGZGUll901Z4ahwFL+yrUD\\\\\\\/yKsRXibrFJu7RqvH93wVG3CXH6vBdKvY9VahUYo7hhPfHxGt\\\\\\\/5HV3NQa6egUwXwui5eFAVV2M8srx4+HL0mJR47BIv5E5+\\\\\\\/QpgmSAPUYHOdmhjHB1U\\\\\\\/qtituCBigtZ3fTiBEpmoMReQ7+MrskjEIuJconeAlcyEWSj6EJINemxj8bG0XWcEo+Kz\\\\\\\/2+tR7\\\\\\\/SX+Q+yh1H07BzkXMY8x4Z+uxj9CA70KXKIuE0iniLeNQ71nI0k1fIV7ByBOlTlJQt4N2y7hJQAQJjvjaQPyC48SzVwCNZ1lzLG\\\\\\\/wEYIpz0HyRPvbHDXBNNTX5woOrY8aKV1xgP2UQw7K\\\\\\\/HRcIaRkNuuJFUEe9WGoCTBmPqzdZ1sJwRpyfOEKBNqFms9c72qMsg0\\\\\\\/gUKPgHdEOPvGOE\\\\\\\/uDwJSirIK0oTh2VXeue2IszaC0I7phxg5D\\\\\\\/tXSeWzfyp6yCIAb9qNfgB+BgrvahLhkB5rRWoBlk5b1AyJWehHpS9x+YQgRipJICr26UbnNvido0dRykeD7KZ6veqtKMF0cX9qHqNPCiSgYS1nLshsYWVWgCGJ4clVc6+E4\\\\\\\/bsQHHpnPD0xRmAvm3MkKW7dVdGuXw29jjHUtsBZdPpuJ5ry1meZVmjM\\\\\\\/cMg8YHPIfsrDjYApPKLea8TEeW\\\\\\\/R4ekvrA2b0jLcUGv0WwUemuWaaXH6S+zjIveHoifSRpXBY6DjrKYrVOJ6qJIe0qVwUwv5cym3TIysXWlqUMRM4yik\\\\\\\/nqfVxbOnMgMS+6E2ksMZOGUKvZYotzDCjfuADG4D4WMEhMc1VX39wfX8\\\\\\\/vGJvNDXpPnXI9NA+9xtikAMpQMO3Wmj3h8YG+XmXMQkVJaHOQVijxMCTbmHQPhdjnaMMudmzM8f6h1YlVJwOhx4LBSWbwFriBrS6q\\\\\\\/9lfgOiIMtlKCnGM6\\\\\\\/3OfZCnxCgItN0jnT8BGC5ik66qtNHkrrbAmhREPwkh10DnTYGOVr0zeLtYH9eWgMXeet\\\\\\\/RMS6Zb3vcOmn07Ruu3pEhRs5nzfa6zVbj4avGZbgKYS+FkQCYylvXtGrAfVugBEGSdazEud6xp4Z19Npzkd7hBwMV1ZqZ3vC\\\\\\\/xXnSahi1Y3Obc15m0+HD8rI5ctwO\\\\\\\/8DBaHx1NFhtM+Fs20+L8XUGPIzLRywrPTZ41zegq3nshZ45O6rs+w5cr9OU0OJuRbWDUlwbyf6HVmfypzrO8EqmtuYbjBAQLTTj5EhqaWOzYHFIkyAdMim7BsaF0KZPL19OZdQp8wf2nVPwQVcHBqE4Z8fIVCvLh0V2egP7FUvKbr2hiYFgtShfU40TVe1H+BC4irAqnhp5lhBh2vKY7SmCwLgv\\\\\\\/IjaAtGhEpAyULshUfUShxmijCvgblpEjSmLxmr0is4vq2RO4IYfh0DUSk8WG8Vxfo6o+nhqZ6Ih6q\\\\\\\/tmvNuPzRUkoG+QiQ1KIP6I4ymUWAS4Z4y6vgcfKySpvsuG1\\\\\\\/juqPR7zGvN9BEicT9pooJ63d2\\\\\\\/PLLM85xVDefuTk4xC91\\\\\\\/dNg13CRVXZDsJAXyTgKSEiX54pwpAuNyWwQcYeCBmXxgw\\\\\\\/G1\\\\\\\/FWyaeZ4c4isQFkDdbfOS9JINdykPkk7Rpwqv5pfCIX3iHrzX7DOhoauiRFDrQJgdQwLwTOwvQbnZKfnuDqCEtpvJWS8ysNKlntz7f9+9BXSiGTO049k85jGpcvxe6kLm1WrQuzAZSrUESU9AR6bmF\\\\\\\/0\\\\\\\/t65u6NevFXgUkM7jvbr9HU+ssdSDpUwZNc\\\\\\\/qu8IOV0fwePUxel+UgnpAauvEgHlD41au6EpxH9lDkfHZ\\\\\\\/O9cZQ\\\\\\\/OQXg25YQ\\\\\\\/K2c2CmjfY7mcUoyGYAtZeBjiHVzj90IKwV6hh9Idn3GwvrlMMMuuQqI\\\\\\\/ZoQ5wU+NHXQJQgsMTfF9ZkxYwJiwLKkYEgBCba5AUZJB4\\\\\\\/vCVotbUYvnOMt23KU9TZuKT1\\\\\\\/idoeVzSMFHADaWf1FX9L6ormprmKDjM91vu4k9HjGcUSerFPznPeHg4lZJ3TSL3LypwtpyZk\\\\\\\/ybigaamgtuKElrWa7bl2YUdps1ST8oCi9vPJVVznhMXxmIBTkGuWt3kAULHOVl195Hl8MLhQ3pZu4kMkDXOjLTS9YvAvj2+sRR\\\\\\\/if0G3AilB3kqtwBP1wf79LgEzfT6jpqcoExlzexWNJQtcXxN7uYSs7aFAx1zOFon617QunRK4T1Cc20bVzQn+TXnVNJYKviYyeiR6Ge9L5vZurzt6ViZmKlxSES1IW+IGKNXjDOqb3Macnd23W4fgVy\\\\\\\/7aRFhQJzeUGFy0BrCIK+gG+Mz6xW3+79MwJ5TOG36BEe8hvxjjK\\\\\\\/PhMhi1Oe8mk1pmmfi8CbVNyfMutagZvg+3xGii4zJxCb3StztMPc058DrNBDCeZcAwBEH+sHYFckD7vtums3HB1uIG4LWI4\\\\\\\/lZfZTl8CH1GJOTwxxVbgLsI69cUEJjT2ry5m3qDnXCEIYkfJ5ARPMIFb\\\\\\\/YtXNQ0AWmQT5P4ahMBFmgjE0SnHooJrORif+yMbfnLcusXrZGcjiiVbE\\\\\\\/u+PeJ5wRaxld1awvWN1hLFSSgAt3DZtQJ3BzX\\\\\\\/+xsI77sH3L9ODGSGmgrf4jxjQk6BlQrMlpCFWQVUa6YZf0Rx22QneaTey9R4TetIFFN2sU\\\\\\\/a8ZfJPWanuSYK6hB80T0ha5WOe3Rk2WjW6yzayCj1gJ1QtswWY1YG\\\\\\\/RzxkO\\\\\\\/PWuCCekOHBR\\\\\\\/v1Br5BVhqM4WPZFYJpa7EtHZDN2dAFFZdGeScE7Y6On2ADMH9W63ruzgHenD4STvGBrao1tCJvNQ\\\\\\\/cq8GnxbwcFR4a078\\\\\\\/+hacGr3vWUtgocc2KBNhJjyeMvO3SYEajsTjC89WZkrj2R++RVpaHuHjA9lhmiS+98DfCOC6JrHvAsCgU+6zSbTi5OGjI9cd3UE4erlHXF+rRU1HF2sCQ7vhz+zYlWdi3O34vxyRcRaSzJNkWrZqjkWjthC5PXAmXsn8a+guW8OjnXWDjIP\\\\\\\/rt3pY9U7td+I2KZiqaFhUyBt88DFL9zk6GF0SVFVlZ2qS2RdB0+1AyIp2GBAkQuGclwfuo1w5x\\\\\\\/vXLeNVDXvWtxbtcUvY9D322Z7z5pU\\\\\\\/PfgRomatNwnSLwaTmwVAebU8ASxIi0OyLqKdXsbaLjh0JBktgvnrJX2Q8MrmYtCtK+A0eQBa8OdGosfjMrKPByo918c2n3CJNJfmv2BsACt0ZOKEDeZ7oE\\\\\\\/3f+OUfgZx92soy3v2hOte9EiLAuIzKxb2G\\\\\\\/oX\\\\\\\/G2gdminAm6kyFrM7xYaXMHTAWUmSEpSkUlwT5TXVHFZKZtq4b1wes56HKrxqfQvhHd7Rf7ljvgkWkDHuL+M\\\\\\\/b7VoQ28\\\\\\\/0oKLov6xVGv0LGf3MYAFuImbTgqT6mEeWWkbjit1hqvfY4gBSr1lRN8CJld59uMzD7e724fpFQKKdBbxwvGr\\\\\\\/vHSq1aPve8tm99X8Ydw9i77BnbcAURyswyKbasXxZaKLd2cq9DnzybdJSCiqfWGu\\\\\\\/Y1e6WpITXnBK5KNfRX85R+XGAkUrj4MjzbbIO2Y55c+YgyvT+YCJh659s7lvmJjyg+aYIBu\\\\\\\/0EqHqowtK+3OgSTbIXuXPl0A14qVVr1WlT3da7m5GCNgm+d4RadJ0sDqTJlmL7moAJaJjrjgwddshElowelWGmZ4BkuJRxdLlfjrpYQOcYpcGY+6u53p38AMzhe\\\\\\\/Fm\\\\\\\/1fKZpl58xRUEMweoNZmmDTApzP\\\\\\\/Ida4THEwaidOdssi4x7OM6Q0R3uXfFYZ5CYpf7i+b\\\\\\\/4iMTDJDk6m4AG\\\\\\\/hh9qBfXpVO\\\\\\\/KxQs5z9\\\\\\\/ORRVy729KKGW9T3ridofPO4fsi4c1eQSvtDQRAWfIMqjOREBRAQePhxnyTA\\\\\\\/+UtWP\\\\\\\/vWHN1bx3diW\\\\\\\/c8mVAXNyYYiTT95PQ6kcxMRD4K76GMY+BVP+kt5FLCxp9fY96yIgb4\\\\\\\/N3\\\\\\\/MSxTyiHuTdlJgO9RAEW8DsOvBGmOxIVLaJGBFN9SXqctMoyCUOyyvu+c0xuVlPuwrRUglQBfrF7YzsgxD2to17gJ82ipJaNdbNKtCKt3b5ZeLxJOaIwDh4DmtR0nAPl0hvWSRfdN70mrHm4ynwHrPcIdcqOtNwpHpmFPuihK+O8SYjmUShYHuRBGsX6BAg2JHH4b37JEBdozGALd0PI02c0Q5M18FbabZkZzHSmIRhy0kP7N6QevYHr0hWaZ7W1uCcOsoeUtXXsCk0Qz3bZyuCTtRPoJPlTb3aLiUBWdg7tfTkIuVieN\\\\\\\/QmIlwVlzyes0XVpTU4TAIGMYxNY0\\\\\\\/Bw2CdrvLWsOKGm+L5tlURdJVowodXLIOBenfoi4tSWChdTTJMdpziFViabpt8E4ca7+o1X4j+iD2MfbskqlheRx7uEa465inhThs04wXp2\\\\\\\/ps\\\\\\\/h\\\\\\\/fqhGsgqCOeuwv9bg7dLhtsLQ9kem8O1LJlUkHXjeA0hnz7+mTnx20DnNKz996\\\\\\\/EnPpSHNuNjNSQluLSz15kf3xYdfAicSyPiPp\\\\\\\/z727HY\\\\\\\/7ejinH6IZBtsXTV0gjrcSwZABHvtJjhF64\\\\\\\/DrG+EOwResiOGUsAKk2rrbK0wvDgZGHZXdwSaycVGBWnaUPap\\\\\\\/LA2p7ax89fjLblLjDAMEyHn4RayQgO1Cl4GBWdIY48P6CoqmcCCY6Gw+nqEeI6DYYBHcp9NChnkstaroWLYkka5mPZf2LImp1pjG48OyTz\\\\\\\/bgrJQBbXY\\\\\\\/G61SXALGV8B\\\\\\\/2UqGIxosFPsz9nhG0TOphWCllzKlDouEgoby7Laj5dnPUmJWjOSMQ++Pw92rWX1qC\\\\\\\/c78OkQGvt9gm9sFGDH2DI\\\\\\\/6Kz0M+VkVGPaM\\\\\\\/LdpUVAJM4Cwnn+vkEcXCzpMDGaaPJ3cVCAVtXU8Y8akyml4Kkp4gdgw8IUrXVkkG5fhjY2JxQfyowoiK3hfVaEgV0u8RaCkAhMI4rQuS9MXs1ua5rwh\\\\\\\/PQ2KvfLbibm7PkBEqottE0plTQQcfkbV2\\\\\\\/rRsReEyQX1kUkWW84aeWswkp4a9ROxhSNFiLPYzxctlV3kh7Llu+9YP+k6FSDnCE3oMH+xw0NfloJg5s7PFwhFQO2DPx9pwHeLmHeQ8pRQWGowWfKvkPlmFJolYol3Norx2r3eB4FQmrgpIHHgakJbXFFfyf\\\\\\\/1ufBHBP1vGRHQcEd5+vnDA4MR9ChtU3Jev4hYbj0+nqOcbUMbvTOSI7EfrJzS4a7G06xlWQUKEEwu4mh+i2X4GpHdC1ApkzRLQMjAVzkBwszQiNhvIYO1YEK0I1FP0LXYN9qZzSgsuUVfV\\\\\\\/38GIRpkLIanC0HrceV7mQTgDprAfpqazOMsbfNSpRSA450fiuBE0hxhXaBN5i9AiRxnBSAnyhq+SrBYsU+rybPZtxusxJmvJ4EzasUNnqHMsTxodjoOG84lB4MUyD0JwkL+ICr3HVuNaZW8g+91Vk3oykqgV2zqxZAxM1WSfE5z+Ff0qvoyyZpmRUziKHr+y5+t+0uAyPBYgEZ5wfoqi+Kz7kqXfeJVl7oteT\\\\\\\/K5TaH+R7kiBNW\\\\\\\/y9uNim\\\\\\\/MFgRAqdVl6vGm2898hDSZcXDsmELq5qH9HvqovoI1V4+GxCfVB1vG\\\\\\\/aFioM\\\\\\\/8uhADagqPZL89tkN0suO6oFmlwqzDw+irxlrHGvk\\\\\\\/wiLMdZpl7zEWxLLSB1Lzo8Kv+ZCY8Um\\\\\\\/M+2uLY1avtEi5FSWIBRKE58R+HhP0p6FOW6tH2irF9ltAxj+7Jes8LKI7G\\\\\\\/kkppzQZkykCCr52xtpenCvvYL4q9qCjEXvsj5rA86sVSdK5uHWvvCLswOSLlgdO3Qk8Bsx\\\\\\\/OmTFOQ7vp5sOnPmxpcH7KkvEfJhQYHJ2KI\\\\\\\/AP52hADfw\\\\\\\/weu7H8sOAPyu+XM5xHCOWuxc30mGOriJCF3nwRa9VdfS\\\\\\\/9T9207FnJR5PIrhxDWeypYD8HZksUWtbsilLYgCcKiDmHPxMu6kWAv3gOB2o8ATxBV6QdWEr8UhpHZ9KXeW4tLDC4Y239g\\\\\\\/SPzYpaBW8LiObRK1xmoONQa3Z0JyMcA+4P1uTsmAPBJUIBxjpTv8aoQsJ4eEMt+1y\\\\\\\/fQEQb2oUOTRHLQXIuBST6d1ru3Z2EIErt4vPNOauM0F5+PgQE8mQfGOP6w2NxyZuuFe+01fvI6aQKSZLb8IExSqwwz66cf3BmnTk5l4TqURj2EocphItGl4Cfw95vet\\\\\\\/aYofXBfDpSqW4plqmhdg5DeCRy6Z1CLkFulcoNcz9E\\\\\\\/lwqpbO0Qyhrbud5da+8r6cNFfVuiPY45O3LdUfo1wNtNhmCSpG1upwNONwkheFpj1V94WPh2kWOhFWiCsgCBt0KtyhmathahvSdIube9AFXxAlu4qhlcTjW6c8PyN3WPthj36YvLwFF0Ugv7pBILUSzyNoCW0CBa2Jh5IfnWP7LN6r4XRv34jre2DBdrtGA\\\\\\\/UQcMqxTKcfvT3p06t3\\\\\\\/DwPa6mcvajYtFeFWOD\\\\\\\/Z8e0ucPFFZjGNiji14ESafYbvns3fpIguvXMaUrLZ62M6BrKZLCvqMqGDbhepc4L9K4cYA7wNvhX+U\\\\\\\/p2WUnLURXEI0QrvwwRnrJSJkuhBKcJbf\\\\\\\/kGd8vbY5ygZAm8S9nceVMUkdSlp30xooKe9oFddi8NKA6ARIOYGurua\\\\\\\/NqwdDS0T8sSX+ltgx3LBeYsWtzKnBEFHjueliNfJAU5ljUvtgnbYhNwVsrgO+hFeru6KbSNGE\\\\\\\/dO30QU02kBjcOTaunk8edO1ZaV1kQ1K64d8qt1LsFMR3TKq9Byjw4Qy2IDziwLaCt5uOwXG0s41i1wWNdcYkdeLjL1hzoa\\\\\\\/kGHb\\\\\\\/vDFDDLsMjJW\\\\\\\/q20d+VM5RzteWcpVJmDEpFr7ZLpp6g79nmIjDKUTCKMVaAskw3g9bU1MkGdwh8XjgZZrcKk0nPRNUgG6Yzd+U18a3mqG75uuQ9UI9v0ksrCBh24suqGs6V\\\\\\\/hbcx46teXoTIzHHnKiZEEs4a5uCQJaSt1MOxo1xyiFkg\\\\\\\/7e12DIelBDgnb8ttFhywZwz0Bpdq9Qsfs1QK6TupmBLNEHlhNTqTaqPBe7qOpSpJQJAIUiM7R1Xko6Gj8S3\\\\\\\/g\\\\\\\/Lq8c\\\\\\\/r5zHy1jPoAAcsjBmFLahIfvFmgOVTmNaDMgANwmDoIQJdO7t+8NNyTiwGeYrMDVlONqKxg52JV3cbWE5sBi3hISNHvu4ntnbzQ32vQBgYsXCf5ZhROQMgDwg6qjZFABbRsEmqculTAeCOtkAKlP0lSXCDOnGjMZtiX79vINWCYCXmUsgHY0wE+RnTVSvdZcGEsPOWIpAPBdN4LA6nXvIbQr5rB57WE73qod0cJms\\\\\\\/8QKaILB+FIqUStnSu7IJb80R7aVoAnhdmRsxoHbq2yA96NPqzQSZTLhW0pBiIo+6nPAPg04dVytcsZ5FRITBRKTcJHwMTdzHMoM2kv8n\\\\\\\/wuGn6+QVwc3VEyCNv4VUTXupHw03RwmrD\\\\\\\/mgg1C4ylll4nsMLtzgCDN9S6IMSvTBeldwoSpC1LMq+MvJuZYzYCSnEpU+o+mzwqXxT2MIRHlnzQGQko9FkmSYITv1ePdPJ\\\\\\\/ofcA0qZsAsJpfGiJUgyb+4dbwyZM7oxYLAl0G+a6\\\\\\\/UlGg43Gmlkqj0c4zpyXeh9YkyhGuDvVAUDBBrUhESZYSh4ENg\\\\\\\/xthdge0E\\\\\\\/gnAEirssR2pjBaJQPLW7rZqQ2NpSEEJ+wEqD5E+2UbKx\\\\\\\/YLWrTKo+sR5scnWSSeaJkZRhl+M5BZGcsPaA8hXK5ZCuJQQX0bzPhKc\\\\\\\/RPftKXUYLhVHo73p+3zfBbEKLf8B8fuPLJ9rKCePk1JAc4WAWqMVcCYNSAQYfLP9mWVPxzfUa\\\\\\\/c4BxUeyfHw170R7cxN0a1qk1f5pTI9BBK0+wc\\\\\\\/PtZJPUWrlG+oruRNLCqNLFtnc\\\\\\\/lhTOA7aJ\\\\\\\/LXfVDWnzz+zPbVzEcdS96PpxNMuToHcSu4wpdg8YYEGBoHRpHRdjNSXjHhVi4QHpmeiZZy+6MkyKAtxu1bO5Uhy5WV\\\\\\\/6ciqcJHOPxzT2pYMU4M39osIvVt0NOAAhe9jaFMQZSsPSIORrL8ohlzbk8vM0\\\\\\\/cRrLeFGdyMDcrju0bdoJsHd0WVclRo9SDAKp49BDw3bjyznSJifMHGAyo7RRm435TOtIbq21k0P4uAcbkgUqvhruGQmQoSfEQtflJO\\\\\\\/7Ho1OzthTVGvCJOCgDmazEQFJ\\\\\\\/Zlyuy17hbXiOEFpkzDUYuCyE2OB1NKBUlEYtQR5rqWq6Lu4PMeqG2oyust9XtWoDZEQVLTcO6K5j\\\\\\\/b4+OZLPcLyfuiDaMQCr2kd4H9kFPSLuICsv7PMsQXXhrkmk+aWYaZo+mL0iz6oNBooAp7YYySsqJym9Xcllq\\\\\\\/xQzeG3Oej6yDKjsFR77ddEgnEcmEF14IpgSibVTt4FCR1c9DY1RuIUY\\\\\\\/H9YGickwWBWujFKQbgIvfNpUFWt2zLEloWYpfYLpx5UKsF18TqrkBWBh0LI3WrTjPZ9cdRgJ7fG5Js7PegFZs2guJ6ReNfgzP\\\\\\\/Ec0zccerw\\\\\\\/17hIgdph5IlcTwftgRr2tStpEmyQNxcKoE64sAtT8\\\\\\\/t5dbuwuA65JSCXLDTvb3B0wFatxBbFqQauM8igdJg4kwwVp8Oaxmu\\\\\\\/WmAn0cRCAWhwdwESbJUXNq93bADOn2\\\\\\\/W0U92+RNravoOFyZW\\\\\\\/JZTJHVQRA9cliHndWVtwGcRSFEQstB1B3SFGocwkrNlLBzo10mdM46i\\\\\\\/tuCXudWDyl5v4edEEXHaPcuRuhJ85G+3pHD4PFbvMtz2+kNy4dXDE1CdoIaeRkpmF4Md8xfwDV4IrmQ+10w2qg1qau3vMhrbHDW5RMJShlgxYWx4eImMhRiyr4vDWEQMuKQyF\\\\\\\/J1jQsmVs0Vcptxnp9wYbQbWhehxzsBVGXVprcS5yOYOpB1smea3hMJmcmgLIIZ1StCeL\\\\\\\/KKnKuhmB4y+jzUzxCejZllkCvptbzNFGNu3u9Gd1Bx49jhJ\\\\\\\/AX6SaJ98loV3qfIKOzMt57liydVoAQZV662ydIp1F0Nm1NTXAe0YB9tF+P1PqNiCZctUpFHTpFtDB\\\\\\\/0sPNk0QiztpTQfJholrvREN1PwRRSPeRmJmwMKUKPJEBBSGzBrnvANp0WqjnhyuAvBuWJ2icaNurFawVZMdXVrfxBHdkoZOmlk+B3fEaXjzGbPos\\\\\\\/MzPtoz0XNQ6kbEY9KPopPvkIU6rWWOMFLNVWm8IfOX8bRpcVdNaAnLZYo9ZbUgTK9CELBNItk8Ik0mGyWFSDTzRP1VFT2YCYE9MjJb7wI1DkCzU911Un3GZC1rkWrpk\\\\\\\/bQpr5aWhhnYuXYRT+GxVnoXGL6m1jgtRbHef+XNIeodsOBwRxugeMOagk\\\\\\\/AMipKzh+0ji7jFLdzmVBtQxeeXbDhvS5xFi4Ce1geO0NM2V3pQmcF5LFmgXVnHbShycRH7uxVi0yeiEv623FbZGYeYU+uCpcZrUkmKVoe741NlaUuSW36yp5\\\\\\\/aRzL0QtP4gfOT4mCGqjFtOd0tqLnnQR8hhaYGcFkkscVg5XKZkgMZtP99dDp5SZw8yj\\\\\\\/bN9teooQc8PX\\\\\\\/rDeiJcF8SNjTMIbQiLI805RKUxpx9qRicdqDhIp2CWDRw67qqfpJWpeCoB05Uu0t6V0jYylOr++D2WOzpvs3\\\\\\\/KlvisIGlzxpbRiMggJ1va8YlaMq1vrD8cSCGnoZyDl7e2Z25rENmhiHsZo1aW5kbnBwltinbCZp3l3lCN0JJbj\\\\\\\/DTIcl0Q7pKCar8oe4lnqpXG06\\\\\\\/DBNcQxTP2bdq45MQZaDYpo9njDz6e3EFwE6UdJKLK4Oh+xXKg\\\\\\\/YDdK5K2gkML1l7ch0rgTMpkEP7BtSieM9UZaN\\\\\\\/Vafx5Q+cwYclWhh\\\\\\\/ySpMsQmNX7\\\\\\\/tjgX3FYYuw7QrjdUACFnk8fEaA3Gy5RS6bwtScwlBB4Q4+S2y9dMaCRBILX3W7ve3TdqmbzRq4mz\\\\\\\/NckMLALg88QTJMu+Pz509ds9FHkH6W5YqKyAe6TARHH+Q\\\\\\\/NBJa6tSQ6dgOgJOVecaXXHMYfvM+dpOvJTbFZ2LprojJS5u8pI5KnPalCh5FcmQANVKgP9yyaLA+T+KlNxdFFwzq5IDuOyQFsJ6srSRFoM3ffl2cqTBito2iRlNfgOgop+fjlTsB039ipyuXD4EavSqlz1AZToK6Mraj+KYOrfsjE4\\\\\\\/rnyux8jvas9WicHHQZYQpBIPaUQz04sM30cxfa0Y6FXxS5Im0LUOXO\\\\\\\/SFuufVm6qfVmrH0OYPR3b6xfnFfcuL8WfuTULW8qf9qvJTYSlu7Ftzh8RUTh3fDPNQgbzkqrH+bTYkwhoK3255EpuSG7xaAaSUgR4BwMOytKooekNJNqhTmWlk2otRy6JaxdrI0jVHlxnhF4bg+PKgUT8aT4hMtjsw8dl4WqDO1NPr2qa7K0ywtF21\\\\\\\/tXgVMdNwAoojqJJ+3h2iUdVKPx5kb+WAPknJCr1WTKj8lm0JOR5MTn7rirzmPxqK7QaPJANzSM8f0eQqqpiajrC3YOs28nUXhJQJY3v4O68mbHf3sBDtlActV63Z2qqdznToTosIX9O\\\\\\\/0w6OrCuATrGdVbAahWw+aM\\\\\\\/SGlAbmyPoHsftnGoC1NCMziaCmKKdOHe0Xpl2+hHojFirZHIxYy+b2robrUDqR7CMfvChYEAFhq6Qf2arwvkp+GfA5QaucYWT49+HHspZ0OoMFuHvFXotRF0qBrXyDVqqlRJVue\\\\\\\/iHvg7FZWxFvXib8MGBIEWzXZGIghnDAp8gTcXSZURpv+e\\\\\\\/Ocph\\\\\\\/NAcuxGuFiFHxINOhCJ6vqLQUOZQ0DZk7roIAwHh3lVbpXO5mSvvyrzdEN4JEUmHJ3Wc5BMCY3X+PAWmv8VysUlj1ib4de+0tCJhqyZn8KOOLzszbtQcfd2Iwlh2P5UZQPybLqLzKE6385EL2\\\\\\\/\\\\\\\/odacCyPIV6Hc86oBm8t1CdfANkgkFboVgfuh2Ws7kMalZT2Mw+dqr3VzFLxaFmvm4aaewYckr48JE\\\\\\\/X9fpO5nF+r9CHp7oR6+hsuYxcFRivq8CpHBlHK5qgzHjgLYEn8ug0Ufpw8etP+KIzf188cyf1V5rdTrEr1ygFA1F4xhoeK5qlHYNUd+v\\\\\\\/O9nGthw+vzdQc5JuL455O6m9Uk8yRv3mo6uYOx\\\\\\\/iDzXfmL4kXL7tTQ+tYd42QijgzSCiHiL\\\\\\\/mXba3fkMgdfvIo7lLTeGqT3y4RcC8\\\\\\\/bP2eceqos0aYdFbA8vmS7scIc0qz+lDO75ycH+N4ZI93XbnC\\\\\\\/rVfHHkseds8UjwVlarxtOK4Nt5dmzF\\\\\\\/Z\\\\\\\/yzGxrY+ij++\\\\\\\/K6IR6Q1SDg1PXgIxef1Ifg0ieZJiEX39Tqg+YUshTes9GAdpY8U1x92ltfUAG\\\\\\\/w3e1FLtARRvg6JJWDlHUHd5IDrE3pyaQgB+ugsBsHea8pFvWJbdZgXuP\\\\\\\/3pjiKCjAfkPrPIi6u4L1szsd4EMCpp1\\\\\\\/0Jiorxdvk1Yjy3BVdwJe9Fy8VPPkUoz5xphi3UdJXZQf8j4Xq8cCWixMoRv1BS9GPcNdQyqpLrKZnKQMviO1VEjizo7xxQcZ3vjUCWMYCb8YgIwJ7iZf2x3\\\\\\\/N35Hbbl3dZjyM+V4Iot5L2XDqCGSkYgnX\\\\\\\/3rbrQwlMJ2I8GzcAI9MNoKnPKsootirzqtzjJi1DDJlcY5nZn13od6IGWQ7akHFPm9d15P5\\\\\\\/iUkVPF9FbSjZZ1t1Gh6\\\\\\\/oBbiPYGLaSBsa8\\\\\\\/lPixqOwGSvtu0pNpI3qRQpBHjG1PH8BjVcCey+Jx9QNP9UdEab5BLotKo71yV2R1giJzdJQdVKOdKTWIGfx4ZkHGJ2obR3xDZbdtlsWDqcxQ0o7JawuAnlIpgqam1Ovp\\\\\\\/sZgEv\\\\\\\/Jo5tQefjmuGa0zaTo0zb0JnCeYoEMNF48Cxm+mM9KUNRO0T\\\\\\\/TDHhNBszO7irBJJM+\\\\\\\/s\\\\\\\/RRehqr2OKsNJkMJlX9tL57WOH14KhoqU1zGCHX8TjGzb6ePqYeHYoaLi8be2bqMPhRQ+alugMDjF9JJZTMicTIqxuOnuh3jSvvxipnHa\\\\\\\/EjNEB9uABYlqFnxOh52kRqC2dFDxI1C5wvDaQDiTCTtjmRnsJ7d5JoV2HayBihvfut+yMgqkDJer5JRUTVjw4uN8hhD2G5OGHllCAf1Hws9Pypx98kvFCLOFFcoBou0Okei6m7d7+Ip07UydhfhisI7z6yl0zifXMNWCOrEiBoO5i0qUDXGAgsne0vcZipqAr9gLz89R7z8LywmNhkAM4AsJPdNJyEKAVpycgvXeMuKplt+sDU4bYLRooio6rf2ReIHcXxg\\\\\\\/WXMBYhJS\\\\\\\/ynx9g4WoBnJ8ypLN4cbaptdENiX85lKhre25ICSCzWezujSnWEYgF3nCZxGw8Na7zw11bR7KZ6bSUQDPLDJjKER\\\\\\\/HeBlq5L1KHRkzigGvKVLIv7Uqh0KhvTi5TapzJ8NtE8xRfQsLGPLk7XoXELmYEMicZc4vnEua7XbgXYVvepr2T3W9HgjWE+RS\\\\\\\/yKHqRchMVQICPM8Q\\\\\\\/Pg1c8z4Ibd\\\\\\\/NH2+9PvWdvZiqvCN44VASVdl23C8Wx4V5pVv3G7\\\\\\\/btMta+viFfdL8\\\\\\\/h\\\\\\\/ybBmM4tkpvSF3hTTkmxEyFhwtFmgnujGLgNgSdq0T8\\\\\\\/nJv7ZtH99awaEOXSPitYjHb+fGY4P9Z\\\\\\\/VhjrmTn4UR\\\\\\\/qaDFI9wEQwBGZbTEAx3w55gJXJqtT2jaGwbPJKp08bAZvQdmUhdLoD2UshNFE0I7hez\\\\\\\/N2nzzBNKu9xQNupAqIqLWmSeFMskNUy4ajYGxvZaZOCJFLPfNhZUkm0ti98RsM20FLpS8WiLPsAP2r0wewF5dKEful8oqiIkNwiKwiSTHi+m7sgYjYevUvvXbXphJ2vlfzrdAXYzbgw3aHzyC\\\\\\\/wcrjogS3XX4HDxzB1USGkK7m2KBK650NGGXmIYPFsbH40DyuK4AdMyMnViDwqbGo\\\\\\\/RuRu9W9Fl56O+BS+pHwUaHiGu4Xg1wXML3ZO1KDoxpQ\\\\\\\/32TA67QnPOgX04eFOubdmVLoxZZ5eSftOOOqE+lKk3tkMNj8vrLPuK3tMeXHLEOC98BiUjKeGFFgZMJO3CV9AnWWOwBbvH3imvyCMOgF6eqbgZM5rL\\\\\\\/FjsiUXy1Q0J1vPnVTuvDctyQ5oQ3EYeBbGMEYRZ1Vx0QzLNfWUQNbIm6fX9793Fp8ahSd3rBukHbp+oIqeG9ShS8MFItDrUcCh4xdeReQntyLB9kxSr9QbTgeBFGyUVjkQACoRSDKwlAwWTZ4kH9WmQ8U9+ssD53r7709YpItFMW3nTmm6YX1jaZmJCMjjA4CqLyb6s53nXzO8EEP+QhLfqtRuvaUQ0f7fuZQDSt6NX1hZaQO2nqkwfVZs0KrEYIx7a9ZkUtjzLFf314IwpiDghN3QDsyutIJyXEiVMu0tjpx\\\\\\\/VuRTv3zDM4W3SkiL1N0i\\\\\\\/YcxFCARDajvj+oKy69KzBiVkAgo22WitiM2+qpihGpes4J9p2HjDHTFEtq+THwa02X7S\\\\\\\/VmYZQGNx888giVjjNnsfWoIkZmrZr9bQnQMEd+QWx8SOOLUllhPsiOv\\\\\\\/+DNm44F\\\\\\\/KXGQusVHS39TH+N0KSBTc+sBMorgpdq8dmkRvAOuL\\\\\\\/InBzioJ10X2oxgi+y5TXH6mo8u5hLf9m9kGO3uMh7sJTHNsP7YUr8zU0S6ZWRIms9CadVRvCyYp6FaxPsXE77iNUyqWQg7kEI+TDe\\\\\\\/3Hoe2sf\\\\\\\/XF+ScdsAzRA30QGzlETvccQQFeyv7+r0j66Z2uvfPNI1UdZdVAwrQ1hrl9lD8ost+\\\\\\\/url7KGuXaXyQg86lJ31+3AYQEKeOGIVulOHDZW4uLGg1u95N2QXbYWYTPmLKy7tsEFLLYIFvXUUmzUH1DkBK2DJ2ytWxd+B2qGigpKpEDK+xFxyPSy+F4q0J2cCtpYU1MqOeFT6RZ9vPgjmkGr06Yotdpr98pxIKsnKVOlway2LfH78K3B4fTectitE8qYs33JtTpKnoAwdwhnAN4TKhaMFGyaQuGkeCm\\\\\\\/0H6gfwDKr\\\\\\\/Y+vk7yaKt1cnjPsuWMo\\\\\\\/1w1Z5V47J849MdXtRrxU1zqeumkxyCW1\\\\\\\/BEtYAtV8My1lMw4YQzpEtyfeIgjH1kdfWICuGmN0f\\\\\\\/+z83A8BO3cYJb+bpT\\\\\\\/U+Nw7lyW9U0j9y4aTG58UxHXWYuEYTx9FqE5Jcnrwd+zcRSNI5LLtosXPyBCBYnSxQAiDEj5T3sLcciYhUbYXHK5jehsegUOZuG\\\\\\\/sacwOYgepNM8MavmWyXmDVYaCG0Z9+0EvFVkrZClNu08vYHXp3h9YwH2rWODvfhD82sEQK5IvFQX1KCnqI\\\\\\\/H3NYDpxydHjW9uDk0qlL2SSi01p8jMf+pd9BqvnAhbfVQ0P5hjshL6DZS1F0e06PkDCR3FvGZcYgHjIkRPWK8Ui3h6y3Ng08bcspWeDv4LjZiP8NcIoEvvyD+ecta+7igIpKvI8\\\\\\\/UPWd8JH1Hf48PFJhk74jUMxPSGlO5+J1875T+y68sJLQG6j9Hl5tiXrRdgz3Ny\\\\\\\/0ErDUiL8bFLu1A9\\\\\\\/JoDPC4xB3DJ0mzTGnW6QtZ8iK57Mp48AEury2T7WK6fmdI5c5UkAR+OJqAq\\\\\\\/jCELrdeFRTE3HmkRYhmdOOzllf5L88tlsGbSUFP9SRyf4JgwATls5sPVoQH5IX09TyCiV40604NJvVRvNWamBk8Wq8YI5n36WDKN56MsendXEzUiqjHTh8934Kr84xc+HFDLB8FnkEsE2Vl56WsfedNVGMivh3B0DBASDv8OYoOkSa2a\\\\\\\/Km0ux0KMmMn3hMrIIZctXnjreQ4fgcoCXjslnKxy+kiAQuoUF2E4Ng7GPw4CPeQ2A\\\\\\\/Z1J4VXBQwMDaobt\\\\\\\/gbG0hPeJI58JCpumO5ro7u83T4ttyq81EGEdJaU3up4K8s\\\\\\\/JzJ5EjOFOKWsBg9aPkg5l9a62uNUC50gcx7py8Nqdvq5u2SNl8vAYJ7qoE4n+yewMehsbYGPrQpiHzrZrD3tnNxc4OJ1\\\\\\\/4cI4tX05gxLM5gXy2R5ORriBVIEeEoTGQ+dBbzI36NoZl6s+3T0fYFQNCPdTEzbizbwujPs3O2iED43a6lDpvl8q3YVQBOkhQc3BntQqQjHyDpknyNgAmFar4pDtqaFDJBtmSFH4oDJa005ya\\\\\\\/JbDL6ljcbt4r6JRY5iDfs1KdoEc9vSYWgzP18uVTlWQEJWxWyyZLlzvulJ55Tp7dpTyWZJWrs59\\\\\\\/\\\\\\\/OFnQQzXquDZz2G+foijOrn9IhrN4BT9YVzm6I75l+dp8l5xpTlP3FbRaJsFaeIALpWN7EtYsA1G86tmew\\\\\\\/FHEP3TjIeW5HNePDpwKdxDAyt1DOnnvy8HoyD9NGAtSg7JATCuX6ws2fz8AtGac0pW9pAlnRjXvnYExJ97jb8HJDq0DXIFr1TNXcpUHSiDXL0hxPuPVwCxYHOiyDgu1hT47nZcGghN3aLBNv9LPmT0EtIA6Bv+N1sEYiQV0TNUnNusWgVfZtN\\\\\\\/aKMQQjB+gXzi0wAHuoOK1mg+Ya3euqU\\\\\\\/X3zyhqhkWE08xwjy\\\\\\\/XskKr+p1RFiF46cxXCNMI4prI0Eq7\\\\\\\/R303PEsgbn0oV9ml1UnbRZUMRy\\\\\\\/PukRBu7JIh99KcYnHKcpmuf7d+FXgCq7gfqEe5\\\\\\\/MlKxRUUa6ZZGPuJ\\\\\\\/SmQBE\\\\\\\/XD5k8xXAuAU6FYrs0yM5hueoI35C0YgAy7htnJZioqzrSBAgk5czKbeLGsIGAoWZIWI3HmbScXjyhsxJim5BM0u9t3maSkdtJy2RA9gDMrUkLMvxbMsQE3v4pPVP5dUuEWXMSYs4ETx4UjJDXSkVFn65WqdTLB+0M7bl6yWfa3++RvP8Xu0NasjRSAZjfX3Imgdo+p7NmhUHbUVGMPnUmfGwr6Jpu5iETBGhYMyp+PcqLVAviaMsudLU9prB6dDmJ4wWjl\\\\\\\/hstuB\\\\\\\/I9a\\\\\\\/wcPlLx8AUwn1AzJ2xj1kWayQ3zp+yVl6SeN9mn7LJ8BdlsHbyJMYdRelIaZFyLMhxBNLzjp6ZSXvGNp6B6d3C6xRQbEobMR0lwEBVPXLzTfuD03QfssftS4hm+8THa563L42GU6BlQ8VoXp9hP8KxCZwbuyDenYw9ZtCWJkleT13H8GDphwyT+zbfk10k1nWdDVagp9v7\\\\\\\/GMZs5p6b3WDcAjwFebpRxGtw2+BUJNX7XtxE4epGwv3\\\\\\\/pvnPsi6xPMbmzlg6LTTcBU+OHZE\\\\\\\/bKqCJu4Vgb78fTqZDsRcTsnjLXrBU+7RcXq7ERBb3zfx1iBazBSvM42r13fjsefFBL28LBSopqN31NcNOLBAbqq5wlikVX6ewY2nc0zuC5B3TlWiTcZfzU+oIlgl\\\\\\\/mILUbOD463v1yMzYK9ru0lvV4vDn0ahOeBAzAKAuHRji8evhX8ewo7N7txaincLxL5Js6xGPa1uAGJiJNzTTyyoGnWR7a14nciHjiao3QhyfGNJTDes8I9Zf7XXOfm\\\\\\\/\\\\\\\/fwxMq\\\\\\\/nV1WIHOooE7kpBXqgJt3CTbszEFhE2P4QGA6pmdW2+9S3BI6OYhvCbNTYHq2d0+YTXBVur6QxM9SR7taDjwHG50QkADH3zATkMU+tetWpIFyuerci7XjLyeiV3x6cweL5X+c2rFX57Jg7EJt2Vm1nP6T8RaclawADU1pQ\\\\\\\/sofXoDHXS9pZ4KjTqpxhuAVeRpc9A8f9RPCZlcTd1XKZHFEY1HLqARG+liY1T8firSqJYAVo7Vrycput1sW7dV3yntHTkK\\\\\\\/\\\\\\\/tcJ9dsXhT5H1L5DDWZpwgM3LM2Lelq8MysJX6RGZRFz3Uhavf87vut5WcPkfySl1KLTfXY1VoylpyLFhnm3tTmcWKc9oCZjSy3Og9YOFLCVXQxKWYydgCzKyzRD3a5MLsfQE3h8Ci6f7PHqVIeZ97xvWz0Ag\\\\\\\/ggFCgR5fdmG5e+WgCj08mzG0sVXOuIueU36lsJoH9+ac0S3STGZq4JbLiynAfPspeznLsQ\\\\\\\/0r9UHCNrmLMDd8Gm4J3p9Hn+YA7owTbqv5m1nGjV+yYD3pWmUEBYL1vXUdoYtl68VJF59RpMPq4Rg5+iB4QiNQvMaL5R44nY47c53pfylZLfG7nNVuSVOv6rZWWLNrRiNEPALKLW8PdJ4Di3JI7E21spLaBjrF0zmmTEuLeKjPqFGwgVQorRM2sRDRKN0n1TzO9FgTKefIK1s08RG4lckGO1\\\\\\\/IO19QCvvKnCtvAAdpmPua5m4Vvh\\\\\\\/qmX14jOQjpjP6p3Q+Kp9cNE+LjM4UcaLJULfqxTTXNGsylee7wvXXjEH\\\\\\\/BPyH7TIRf9MrySmvxjd97rgj5b65XEY2VxoZQOi8GfUkU7WIgf5RLRp+eOPhwBc1Ioan\\\\\\\/nZcZxoLK6T3kWyDYidHpoO3eSS8YcYrB\\\\\\\/3+M+uXaRPPi+LJK2ggjQox1dCsEDNvhA0RUn8PVsNMHbOa34IH\\\\\\\/pE36yuzv38OWfLv0GIgZ4qrXvSyk4qHsmXFY1IKZPGxAXe76SZ21FaSA6qM8IbEhv9HBLIf6cFk0ooQyksjFIKXuNJgj8NDRkdUasBbD1UGHPO1zRSAt7dXZRt+1z7UFvOZDQoGCGShEMWTwPgCwawcKglB8pl9g082s1MlTgn6BmXcXtNBs3JMpzkb5vwXEdZOR1dMDrRIPLK\\\\\\\/WvQ9CRjigvE49r2mNUDilp1WnleAJGIlyN7VYewafpdgIj206r6wkx4U9q6ckxaUiH59UmThktgSsxuvzFfC6UOxnbItY8r+75bjCa8QE2MBrnh1OfrUQgchu0qR2jWYdLKo4Hc9dZmIPBzewdiLPcIGBQAfIK2xqR38pgKmkiS6lo8zKsfowGmHa8df0oUkaOUu8rc61gLgowFFYxUcL4u3ehQ+3bch4OuhklwxFGx78o9mUyYBfadLliHKSSGfCFFi0F3H9238KGug+xqoKXKZ5sUKdt0gdxi8M2qCzP3TnGTIVJHcP0ASp0PLg4V8ISQTLODXtCvZOByZzYL6FkgIBXjvqU2Wr5d2Z2YjPbJv0oZ\\\\\\\/cjL6DI2FMxQo++jFnXZgmsesy8ep0Ztny23wxVl\\\\\\\/gBTTdIN1JkzyzQ7n2JG\\\\\\\/sTgP7lkV9uXiioHV306wAQ8oGR9GuWiNkBfiYzi3yS3nt2PSYXqd8Em\\\\\\\/srFzshUtWr4T4yc+Ev+MdznsCJ8uDAS8oY2UY80W4WqNpBKxs52oaKf7s7xiA+ESopoKMUEPJgHWcdkDZhGzhGdL5AWUGLuZ+7\\\\\\\/Q4TWoiOvNajDvOkH3yFEiMYOJmea6USCLZ+FPm6AtfYzHXRF8irFMnZGPOgLEbLFw\\\\\\\/2l86y5oocZIdB9nDXkxyXSn10xAQs96ZSk0z4M2QrOyrSmz+3pgPXngEBjmNxuUbWpxJrxKGP92eS\\\\\\\/GUzYaNuTNERphSkyi7N3frZcOSf2xgxuPFYYvhcg+3Knbf6C6fpphT9g9eAP0y5WVRchgI\\\\\\\/HEFETH9lrFdNuZkaN0sEt4FEMSxk10zNHvdaD4K33W3ZfFdgL6rD3qBhyY\\\\\\\/EhNuu8zSS3E2+8pMHSKQjtvKALlFn9GGA46vsEJLI5XCfUrDClIolGzi2GDKrS0KfbBZR5nxT\\\\\\\/f41ILY5iHyb+50VhPtuMIeHlwge5Nv72kGShyG2VxPjMuzo\\\\\\\/0hocv8oh\\\\\\\/vb8d6h0g37vD7NgcDDzYQpDCGGH\\\\\\\/YzYV8PAroSEdDmQ14x7V7ei8P5MbULngA28gCf225Ifb0y3be66vL10oyakHF4HxHPIQ4gAh86q7AiDbtwcrp6njLQ0kKl6gZqEDRpqyMRKEecaqqcrOGM0eGnjQvOI3gci0JWqWcw6CHyZY5u6\\\\\\\/e8xQOgQ\\\\\\\/n4tNJLvLsaYEUBL6VZCbhNZhg2EcsEyhAyGrIn4hSiIbCUddOprEnYzt3Sre9b966D4tREVHTsdE0iUhTncdOb4uXWEOkJmvw7iER5iPS\\\\\\\/TLwL3ADh05QyXtRsH0EnEGEhF4dtkBLOm7U1ba5ym7AawNj7W6VtgOCpEYlhB57rJJRr8hXcJOw+bn0+YdHh7udrQNcvVdz02hZPH0VL4qwmG\\\\\\\/q+IjDeOp8P\\\\\\\/lGk255zXSgL8eJJ9lEguC2c8ugaeGCKHcnPGe6lBLmiKW9GfzPRe18iafQDGTIJcgVjUJPKIvVHzt0\\\\\\\/S0pieMHH2\\\\\\\/2AlsTMvTTjdmnTa7PzvOL0ISy6mE3eZlNrq5+S0yrm84eQSCXLl4qPpBF33JI+VF6ZFHWMy5vauQFo6QLM4kZqNgimo3i2cI6bqZYxLYH7YMmsFjCgn4yXDmhF91eIMzjhsxo5ntG2FjSfNtOrdSBn6A3bgaCmEXge5lRW6qVNRwBUGWlO\\\\\\\/FPWUR2zFnBxbGbybUjRDqd6gF+fpdzU4Yhqsr0VTQe+D7JyYP6YvtBfNGesy4KZqVCN00q1IITpNHPVpp5sSjkJR\\\\\\\/jFtdqq40S\\\\\\\/KDcLED8xvEPpZcy9a2T4Uf\\\\\\\/lw\\\\\\\/UPKb7RUMmBZYApsLveFuYaR7IwdKPO1C4W97nwb1d8a3mlJjMPm7lM89dAbnGeXEwbfuM5LbLJyNkOgzKBc5s7mXPk7XVCSEDyOcUJBvdX3y8FeXjW284HYJrpIOBPYMjFKECJxNph1K7dNeY2Cg4Zn\\\\\\\/gr6Bi\\\\\\\/X6e1njVBkyC\\\\\\\/mpjiZuib2\\\\\\\/AkEC06LUQd0HZcUdAI6xB5CBXw8l5NcYnZUhSEqeh+Y64Dc8AyNZz4Q87IHW1SY1pvxUL+3ItmQyugMqPfLlzEfZrlJW44nBLr0K7jgxK6jDWuc9Q3pMuz770pvY8rhBz8vaUuFjyjoTGGmGWovq0Ile+TEQWf6pdkf9EiBHZgxO\\\\\\\/Kg94KHRrjuCbFJjvWo3xYSXmNlN65fncpCseoY8sWylby3mj9ZzjV7c6fRTic57ZTnrr5lgkXEHsZStsIOnvFb69VglM4\\\\\\\/01HCI1sfsYSzhfuHpsr9V59ctt9GZ87rfoOhSHlQs8iwB7mUMGh6+tg113Ij69NZTGqESNZ0YMczNp34wyf40xcVQOKS9cREqWtK2cbLd7NM57gDl8tAUjfIjYp4dLHMCPED5PsUStbqxdspqxHeBlCJacAdIRNJMG5Kns\\\\\\\/UQFI9fyKrjqZdYKx8VuxVsH78AEFZCZFZTZOtFaUtu1TKHQdKRd1tsTrbHum2yLiD3\\\\\\\/Clg7xO9Y1\\\\\\\/Z2VEZ5Q1OBu7F3c+DYkEq\\\\\\\/6NyvITwJnT38Q65ByCIZzmVKCsX5HrFC11Enbv2vpZrOgwjU1jXzlFr3qIk61fCWxlgDG5HXHOmDNjA7efL8oe29Glz2hMWpXp6qnKarGmnZwlAGbrfrQ18REbAsBte7ChZbLHDymcOXplMaze8s1unUdX41Bd\\\\\\\/oU\\\\\\\/zUMAwPwbWClQrZHLEJw9t6kUzw6X71EzocIlzZMFeHgJX\\\\\\\/CAR8y+ULsPdPOcdnRn8CNu4drgb9nE\\\\\\\/z0WYU48fPHL0DZZAUu7hCy+rDQJrUxpZQ5uqJENtuCxqK47Dp\\\\\\\/HCVJBHdW7a2F06mm06va6a82C4zTOvN\\\\\\\/4HenX14ZXQc75i\\\\\\\/4Ai+PPnofSuCKyDXMs+EVwkhN1O94XwxsWdL8PwlIeKMl1yxLXQfRd+dffiJ3wXgjJ8LTAKaN3VpCmwqtD7t\\\\\\\/exSigKbEvaho6KmN8SAILzu1o44LCBjJQXav5gPLFqhHwnNW\\\\\\\/GhFwA+VFmSUej+C2Cqbvj505Q1Ry2Z7v+fohqUAaOY\\\\\\\/sKmuwX6zbxILJyVBGkDRCE4uC13sp4Wyy2Teyz+mZL+zfwl9582+qelCyjdqDiZUol5j9uOKYWIpK2lYQYDt+6DNBhTU9uJdR7BJhorqVXu\\\\\\\/ckf0VIGiZi5eoU0sylpVD1HOnezB0cq2GS9ZHYKIbXvtK61HWUARQ+qy8BT39DaCFgBMFczH5PSxZruJt2GLmuctIHT5x\\\\\\\/Z0E+Ovl7YPmk1uQDsYaw6Ce1eS2JpyAma7K8Yrgn\\\\\\\/oIj3v0lkQON9Ans2\\\\\\\/eZwwos\\\\\\\/uxgFPZTTDV\\\\\\\/RalMvN23z4rQCGb+JebBeuI1isOIoqLdxMRaJeYhIcX6qpNDlD0ndRZ+uy0GutSOs54L8kdV84YYn8LBV4HvUQaOAG3ZvtKV\\\\\\\/F1kACt6XDnLWJ+UIGXdaRZ3qEWoS9MXNdVtk2Y6bJQ+\\\\\\\/CbK+lQ8YV5U7TqZjLBe4VaOS1wkcG4xHuLZ0tUYXSmRjVMIgBm1xW3rs3YbIaovKW2xGTjaQDm0lovqjMLc3msHk5lxIOuIQByaMZXRCVHPPSsVj5CPKuFc\\\\\\\/tYMnQVn63JedMUdUX6xlNPbxwb52VdLYdF+\\\\\\\/jKEa9ukvWiO5zByLYw\\\\\\\/m2ZbgGvX5b4N43G2cuSQZQ7sGuvd2kl4t73PuCuque0IpzZopdXWHlqU0LGQ79cfRzmLDke+PUEYXqGkd2PCFSZbRUGYIk8D6Cepy9pOi1MesLcvjzeXVMhzPnRFJKGMZnGgBQfN2WrGusAWtN7PzKUvhplhE2M2qbsqFFGG7hQIJqrkRHchx2DOtCXZvl60dgHQfxbsyZW8T1MSXZjqNSsVHZ0SVeQOK7MPHHse06+5tP5vbeUQzqea327Nd3\\\\\\\/uYWbPK175pQ38B1p7NevSWoDFyf+\\\\\\\/AlJNmTdIKn3ra9RdjEV5fMx\\\\\\\/wfgRREf8hpDFMidk3DvbgUDKkc7OOypuAbMKvKJV8p+sY7DIG7vlidG10CUEYExjohpxmsXnlPu1RhEaoN8kCaNxZUTCdTj4h\\\\\\\/yPA7i4RseSqQmLQNT\\\\\\\/ASorSB419uf7nooKcP5nwZh4vgyf7lSXgHQaKwPYnB3aSmKZzr38LFnITSxxiu1RrbTZyyafDCR16AQZexHLjIM8sfn1LpseSxhpZh9rrm3Qcb33cKI0yqhVtUCeoxMFL2KmgeigogbxKfLxGlZwDY8RV4r2OhG9ztBuAIsHEF113s52YElZ2EsP87juaMNylunOdbW9kqC+MHviVAS+JWz5msnokufISdUg4erzKQnXTEFX3RSEuUFoQ0kO4HMIkvpzHXmuO6FQK8pWf2fZf3iePUZ0MaB5XCYCQhbPRF4tXZ57I2EvM6gTupwqraSNbO2H5XN7SfLL7U15qEOh4VNFd2udu6dq\\\\\\\/+aSWweNlTPRarVqGCVN+zBmlRUMVortSOgsdP6TqgDhH84At\\\\\\\/sAXWIqkKQkwsXX7LhtFsyn8pMdbNs1+Iu6To\\\\\\\/RETIBNEhL2tJq9\\\\\\\/s4tZqvnDkUIof+kaGPDy2d58PZJU3f1S8reZ2sQ9fmSyfdHBS6DwG3iM+tYrt8\\\\\\\/6CCuVHAYr4nzVFV5KSbihY\\\\\\\/mzW2wPAWIsJ7P+2V7JgtQPkBsuTCSekqGg\\\\\\\/Lq5dOE1FuVJ0F\\\\\\\/t8W7AewiYhPR2ADvVB4DVT3SMh0duFY92GkftOintuHvQkCP7W72c0IkYVuwGrXp53S4tiHj9Qyq5AL2AIgQe6zjqzyIUKliUOP7IW6zc6DfpNSgw9bqChv0TAFDSMy430RGf4QCZXTsf3R4Iq42aPy47T2df9PMzcaCN\\\\\\\/fnOsNf9MrWNYAOha\\\\\\\/7Qx94+hlyNQdveIa1qfiIEc6GGhM4dHh\\\\\\\/gdc36gKPWz\\\\\\\/JNav2h0NcVQ4LVcWzmepNTHedQnK8grotyBtI3yogrrI4vWtqGGsTEhv1WlWLiNubdZHb2T45aiOYrDDKB9JIws9z\\\\\\\/FtHtqiVQZ8G\\\\\\\/5oa60REcefVJedCjnNuyI7JO\\\\\\\/sBcklDkAX32rLnVzj1SB7vpoTp1NkDzAhW3H69OscY1L9Ey4x46iW398bjidgMH6ZuiiSvYfIvImnGXWgMeDpE5D9gfdL1tcHOcw+Ygl42YzRseP2teZm0IdIe0kWYltJC0EDBWcRCo+nwnRb7KmPWwA8i1+mQpfIydhbqmFon2FrxSCkAodlt4DoONGsqh\\\\\\\/xLDsltA+H+WCVCBeaYFBawFkbySOnTPZ7zgemJ+kPdY52i0xFWUqNoDrnhra9mNMRKPXsVFS3ys5Xx+jl5Bmb02uGapB9L86rBucr4YB1nR\\\\\\\/rHc+x7lTVsUiD7Syo6YXHNYWCR0e9mymI6MqN4HwVotGAhw2Po02PKV9QvYCJHIrbql9c6O6cQKHxOFiZgVL3PaasFWAf8HUKAv+JNEuVeqJN6zy+j9ca+Osa13UtnypD4u2Psfy\\\\\\\/28O7\\\\\\\/rLCxj0u1yq0NIWS96nXXJBrEpxaybaMF8iKJUAeRLoi4OUmtCDR+KXsHS3gSZTf6Da0Zf9MiufJyosqHsE1eP1Hj2OgYWOoufhi7vbvTBsWHOLAbw0fxBiqKcLP9bUPrHg8O9q8GJR9fI8ONaU+Avms3owAJZWWvsLwCYAhjp6MtTHmUx97AfIsqOMqvExBUv3tKfE8vx8KLsbilB+EOuxpJV7v0NBpwH6sq12kSOoRWW8sEF332i\\\\\\\/TlIdhj4FBZWIdIhb64u9H1r44H8XTXRf9GxlD9dc9yChry5hTfHTQR9YfQC8j\\\\\\\/ATr3DQhmlIUbPMASK54jc0A+a\\\\\\\/uKtqJNOhQM75p1CgkzDsnRXKWZ3ISbjah6gm\\\\\\\/IRJOs\\\\\\\/iEjpevISNAU5UyefJqvj7P2LUyYV5yQcaq19KQ4l5xyEPvtcMojeCrw\\\\\\\/ND+L3PynrunHAJg4mP9ITBHky+Bz7qNyTWum2hSUqd6ZNIgG+8bapldizCmV6s3NS2q66uAf16dAhiQgFkevHQHuGbADxbUehsVK69wn3AeA6NGjFmRgZhK3qTpy9Pi5mDIKmyeLH\\\\\\\/AiV9C5hBo\\\\\\\/VdnNMqmVaGz2w+yGfqwvUipVGpSOJ+iEXGpbICk0+zfZpvaO\\\\\\\/EfoArqVxkoxyAyug2hTeSsfeV\\\\\\\/itG54ZWJclEu4a3pQVDM\\\\\\\/4eW7WEYmvGnHdbJaMhY\\\\\\\/4fcdVNzUqo74+03iAZ0oCkXdd3kKZA6UQGR21te190b5Ey+faIdUlr1inQhJqlTkn8i\\\\\\\/Nc9LEbG\\\\\\\/EZLEYbMQEMl4ZiXMjh0QrdyRKEUcO\\\\\\\/CEF7ag6r\\\\\\\/RmRxEBOXUu7dCZFC\\\\\\\/FeoXnEXz6fsb3uxUGv4l4sUOisVvHG6FiQx2\\\\\\\/eMLmb7IJRHUyn8Y5m9hFir8dVZ+813cH+O1pTwfpHtsCVnIXFTaYtx8ptfSRdIgI35lBUP0agrPt74GSZruwW0Ae4db1rrx5p5ky7FySC6q41oPeG+3LZ48I1ujgxTsopAkGCbfNKzfVr3gB8NAsdlXzbZBItVVkGP4IWHJ9WUb1L3\\\\\\\/d+jwGGWpTmUbXcjrknH1FMzOIaJCoeQNh6YUduMJmoGwgv4TJPU\\\\\\\/6oVKiHsRpu63XdOtNOfWPpEX207ca0EiLGUcVom5DAg49xxBzRMsfuu1YsjGT5NGmizwYSzK29G43D+af+A+qlD6Gw4sRiloclCXYDfGFrhWaLAmr2iPgvJbPXsZSCTRbLqlWqPzofEuDx\\\\\\\/CRB7OeB83TYyZUaoKpSoHsq0W2uIkQInw3cpHCTreGRa7PpheSQlP0jn70oDPEfY8i7lwnAckZsS6A5E8pXoH2CfLo3ioyGzszGAfCXJrRwRrKQ6Rbul1GUsO+7pB0Bnl+qsI30ku87n83P1n7+Fgx3JzDYPNtU+6S0HWDnfHAUxTkzCBPzlVck4tayYfevhGUBB41t82I5mDMu8e0A0RC22SPNHUU07sPnl8zeC1lD6oTH3zFR+CRNXFysYDS8JLmEER3zH7tGR204hSXVOJ6EmJN60S2NzwspWgpgmMg9CQzIJ2G6QRJAZb2bvtYWU8szRru7A\\\\\\\/birYoaRReCaxPxzaC8n19YegpIF2c6RkFF2DtffUYZS+++N8rhCOWSjestLbzHHSb9LwoxMF\\\\\\\/0EvN8+I\\\\\\\/GrLQQrbIXPqtqY\\\\\\\/kny2ZKaHxfyslwtYEhzKtXTNkdIQ7wBMAeL2jpcSCmejFlNCXXIFnA8\\\\\\\/GwTTufgQG7b6K9s2AZ7D7SwyFd7p\\\\\\\/yGGy3ox8SO7CPxebc5NOSP+oCWQdft4CHYtorQIeaLGOvgd7uT2aizRnqWmssqBPztVbFJJRxyzvjF0bxLYYQR0fnUV\\\\\\\/pzJ+dKO3wJDrMRGcUcJ6qn\\\\\\\/lCWPwLyaLV\\\\\\\/jUIGXmQdbt7bAa+J71znJU1pcfLjCldYxqZjCcnNuD+rPY7a0\\\\\\\/94RUb2LKn71MZ64JdUmXoOFglIJ6gLIjzOBjurV+uJCTCKuBbtTft\\\\\\\/q8QQHXIpDMZNDd532curoLJUD\\\\\\\/Yg5UBirt0rnInH+FsPV7AgQbU1x4ps58autIe0+AK\\\\\\\/5r8htkxTCUGzZLHuNOyPy8IYqHcBLPWRA2oPoOSkuZfk2nZMAXA9rP3gbrHiF8MOMFl7qBAOuS+hBak0f0fopjHNsmol84nxJEXnxlJv8Pvq9XiGXNm80g3w2bhvLZ4YUJl2lF3GalWQKBz1vv22RQ+GhbdM+Gdijp0VWAnqb0OEodX\\\\\\\/MOzrMocy15kT36ehbngFs0tEjqATBdeZ7aorQosTO9Gbtr42SpCkVWDrlNIRhsCgWckc5zPGmfqYWlS8GD3TeWZQ4vgYXPHhXvarkC4FQfFWQPGJl+VnPEFh7JzitgjtQy5lwk5kju7aMfi+Z1pIuKUeqh0wrTWJ8nV9iZ5X8Y8OATRvvpRxKNJI\\\\\\\/jmsZHuxeOTYGxq41hxI6bUNWDr4ROjRTfRtUcRDtTmGfbutQsWtlTdzDyvtFmjpihopPVzWzIOjs\\\\\\\/zua3YC3n28HphDIulHYtK502uAe7vPrVRsFLG1rK8RfyxEb+SBTBodXaA1beidRqNyUfAI4VWEdKF0IUZ9NqRllu6YKop8hvYsuUTTyDR3JKwYpt84DILeAVHMH1iVI2Jjgv6v89gSpRN4pP23yb5Q82C0XVHFBA78u2oICvuoP\\\\\\\/aEjuuklK5ONsrYB59Q9OMF1sOlmR2tWZEZCsQv9SBva7Jk\\\\\\\/36epZT7P\\\\\\\/+hxX3eRf3tyQDhHsXvv5rlDHXljv2ApMqJElk6Zkkj\\\\\\\/AvGSz1rDSJESAFDoviZSZ6XjWCKaSV204ADoi7+2jrnEVStDigOK7dRL\\\\\\\/s51mqSdtHl0Uat0sBMMVNNXnDQdAYfpA0PQPlE3G99I1\\\\\\\/Q5hRpravLwlNQsSjcxvVWqQL6sssp7wyAi4jpP7X0fZ0ZXdaMsXOzu3VF\\\\\\\/lhZWFcJ\\\\\\\/dC8Rs2MOXqfQuv4jQ3hYrWvuOJD87+riqf9wKhnrU9Qy\\\\\\\/ZcmHMdmNanSaupSDK5VOPHK+mz3kkn+j446eMp\\\\\\\/ibz9RyKedrNMoMN9QyEw+HlXLZuBLOZKEeo0g\\\\\\\/HH2ZZ3oxJGfUBo+B\\\\\\\/qZMuiMNOr72AvZyti2S0AV2QMpChYsxD5btQQSiVmgS7BototbKr0DsQiGQcYaXTaKDex1PQjd3PyTEXz0PBYiZ\\\\\\\/A8y27F5ISbX9LH+qo0FO2Ut9LQWE9jgz8qoDhXQqtw+XpRwGVkpJNlGy7luCZG9M8SiDFR3pksbjCbgt0ygLGnZObes24P4LEDv+PmsQBt2ciy5zcA90lWio7lkwCNx7ZgbPMGsFYqgYbB\\\\\\\/DqbsqRvT1\\\\\\\/U3tYKb1HDP\\\\\\\/bV8V3jJce2VTQPFIRJzdM1Hlhir4Um5rs4Uc8CYrBMrjLAYiVjtaBqp8iklXhbhDU\\\\\\\/yoISoI7\\\\\\\/w\\\\\\\/\\\\\\\/gqhFcR72Qa9pZdEdAXyQ2yrSvVKHvuSvrRlxqfLSOkfw+ixzqlbQVp\\\\\\\/W+xHckzNf07sXNJ3bN6\\\\\\\/boDuT3WJ5m4tSqQsLHbWBjgINw2Y88eKKdMYlKp5PNgnA1VjVo0k\\\\\\\/9Kx\\\\\\\/7Jfp0KFnSmrF2JrXucTvjXE2+pMKye1vZMt6c1D6qLUxE6dwmshMtu\\\\\\\/4wGs3PKOv3LinQ14MdhRGz3t4\\\\\\\/1ZnnbUQuxPZ5WaJGrru147TXJJapJ3YublXa6lrDheUL0HVWs2fpNMzg9oE0FYyfYmGXC3S3YCmjjdhIcUwwFfvsepiUgtPneBKkqz2Gs+B4W1qaI7iPhxI81wFSNXZi6V60YqwYAnEr9SSAZspMw1ocKuMIumQrWM5ru4E\\\\\\\/x43yzwgXvkz79ULwmIFPkU2YeL3IYJkTZKYM4iv0Nx5cueaw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022rVMpr\\\\\\\/utTfAYq5jl+Yj66Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b4c21f2741d336f26832f8915ab851ba548f6218aba39b001a900b70bf169bb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022D066rNVLloKtIep5yVve6Vifq2bq5HX\\\\\\\/\\\\\\\/79VW8CEqBMtLEVdXNZENPknPDZc8ffB0Dk6y0yB1czMhW2iurlqc+Ms0RWVJwNqZenkcf6k7eCMy0fosNU9h+eIj5\\\\\\\/UReVwZY3Gr2v24fUq5qef8dVK9gtkMwjwbikMfxOO\\\\\\\/XKyHnIDKj0igz\\\\\\\/YZyVDVqvmeUejjoeHPBiCmvu6T5mHa6Dq+qSFLJkckg\\\\\\\/t\\\\\\\/qP185ApEUneHHooE1D+pQY++nmiHxg7R1WU\\\\\\\/qzLG9LAjL70XHR2VGsWMYvTtahwYSz9CAxLYfNnqUg1wL+SJKDQ0S7lHh4hbACgayjDWjA\\\\\\\/dZvSq6ygPMAQx+DRdGoTQzFD2mZdrRXYni4Lx9oxqIC5cka2Rb8ACsphNDSwFVblbUkCfsZidndudmJrB8+siJ1kZEZSVXnqVT1JZQD\\\\\\\/nCJQZqTVZiUfrNFHKQPyU64cA2elVDX\\\\\\\/iL9V5L0lBbHFP3xwW2zHMmaiTSzQ2LcuSapiTE\\\\\\\/uGLdnf4DY3XyYDhYdCnajmPikmMmcH8\\\\\\\/aGjrr9lwPCka\\\\\\\/V190n2AwZUkyERrUqipXI6YWcd\\\\\\\/l9XrKZxSzLOQu+2FIhBuQ78ohsfRgfek\\\\\\\/koVBu5KzODUaXtk5tn5lScfD7jo9wB9Bg7LEHZHeCKpQL6CY4QewkHMdRtNmfZR5NgktQqsUveDOzQ8oyg\\\\\\\/6inUfhnglKMeYlv3lwKVpDTDQ1HF5Bvyhg4CJjYNfFKKPtJCQ5E8cEWiYeH8ZXjUgTlNQX\\\\\\\/HBjO6SReJ3RNdZF1tdSJFvBiIVriNha8zDh9TxiVpWw2AuPB\\\\\\\/k3jivbc2JXk4UKRl67\\\\\\\/EMRJ7ZdUTDrpjHzVOLJwgsXYlNiRMU1mOhNtD2UXn\\\\\\\/ZYCUlhKL06QLPClin9wfg+JZSlWsUqQoxUfTFtpIRhMMGIH8+gJGt4WnboGZXNxSuqw1VUdSMkMfBnlsiNsdpHVICLqwAjzo6gfbZDroD4aj+DlmcH\\\\\\\/gaTjN4Gezoxnu4QS\\\\\\\/7feDKy1k2vMbnMU3zhZoonH1TpXeG4REgrUj0HByXMqFiY7Ju2B+Vh2t954fj5mpT0oFP1jdiv1Q0wa8bX6BLhamNSAZKjBiyIbvAb\\\\\\\/t4oirF3vECHDpG0feeeoIrkPNrh+bBQlRne2e3NxKC8Qfsa9tQKgSHDXu0mQxgZQd\\\\\\\/GnUkCcs3DBF5IJhDFKjDfx3dc3pvY7hp8+CMThNzX5KFfcNiuZzZVMpJ2OtPOF7S\\\\\\\/EHbg3SDhiZQORhHHRx85a2nNXutIlrC9zzY+AMsZOTaOyFoALCtBteq06+3mKPdnpHJd9TEb8fjwnkZxHkbiNI1lAYtE\\\\\\\/IDlnNwoSujegBEEwGYV4Hrwov5b8Dio\\\\\\\/bIVz5of6qjc89op4uv2VuSCRCfd3th8ZQtn\\\\\\\/ndCOMTt0W3nZsUMZKr2Dpm7LVMktryJmNyU19H0Py9MlJ8qWp5ixdpiOS0z8xmbg28ANpwkYbKxb9PUewNnTaBO3Dto+JQgfIm+WjO1GDVUva3F+wPno5bXtIZPRLw9cL\\\\\\\/XrVTh0GoyaM64S8dmZDAu9Iq5Dy+5\\\\\\\/sYFMAJV8KV+v1HEWwtybSrHJmmCW7NdsL118f4RG1R6bZoqhil2rS4KzyuuyntbdrHcp6rrA\\\\\\\/NlKOjRgYW\\\\\\\/sBufiR2+i3jrae5Mfk0lWTD+F7wptb8SRKUQ5eIxGrw63mGh6t\\\\\\\/CKnue4w5Un+FhYaby0RWOjHPxNC069XJd5TqT5Y\\\\\\\/Ki2fjP7E\\\\\\\/ouMC8ZOQCPyU0RGaixViUy8dPXSslpQ3Szu5ZhtWfTY3yPLVU9VLniQVOs9Ax\\\\\\\/iu0OFIVfsAegI6Bvdlu\\\\\\\/UrrZ0if+xVcVbtgl2S1CFhG8kv50VvNX4UvFI7vmZgYwpnn1DFuv+zphYHNnKgzRMRWjBYPDpCGI8K\\\\\\\/R6tgwAVH6mEid0KdsA1S4JN3lSUV\\\\\\\/VdtBaxZwqr5Lj2FiIuid7USDxipgKUix+VT3PJk6Q244culSQp7pn9Wrry8VlKlLO9+sbw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fUjGRzQhfjGirrgi93RmAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254536e38db9043946cd8d1f53d1d2f8213601725e6460c10f5662cb00a47a8da\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492412 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/789" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QGe4UodARibIN5JYM5n7Ux_GIVjTrliz5b4qV2HjU0Y" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/789" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KlBGDRbJ8PJj5XmYOSGrVmqOPbzA2W2MlKAEUJYi7MM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/769" title="The Role of UV-B light on Small RNA Activity During Grapevine Berry Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/807" title="Tissue-Specific Transcriptome Analysis Reveals Candidate Genes for Terpenoid and Phenylpropanoid Metabolism in the Medicinal Plant Ferula assafoetida" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492412" id="node492412" data-pisa="ggg;9/3/789" data-pisa-master="ggg;g3.118.200969" data-apath="/ggg/9/3/789.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guillaume P. Ramstein</span> and <span class="highwire-citation-author" data-delta="1">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">789-805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200969 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492412" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guillaume P. Ramstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gr226@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">gr226@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/789" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492412" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/789.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492412" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/789.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492412" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/789.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492412" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/789.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492412" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492412/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic prediction is a useful tool to accelerate genetic gain in selection using DNA marker information. However, this technology typically relies on standard prediction procedures, such as genomic BLUP, that are not designed to accommodate population heterogeneity resulting from differences in marker effects across populations. In this study, we assayed different prediction procedures to capture marker-by-population interactions in genomic prediction models. Prediction procedures included genomic BLUP and two kernel-based extensions of genomic BLUP which explicitly accounted for population heterogeneity. To model population heterogeneity, dissemblance between populations was either depicted by a unique coefficient (as previously reported), or a more flexible function of genetic distance between populations (proposed herein). Models under investigation were applied in a diverse switchgrass sample under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is entirely excluded from the calibration set. First, we showed that using fixed effects, from principal components or putative population groups, appeared detrimental to prediction accuracy, especially in cross-population calibration. Then we showed that modeling population heterogeneity by our proposed procedure resulted in highly significant improvements in model fit. In such cases, gains in accuracy were often positive. These results suggest that population heterogeneity may be parsimoniously captured by kernel methods. However, in cases where improvement in model fit by our proposed procedure is null-to-moderate, ignoring heterogeneity should probably be preferred due to the robustness and simplicity of the standard genomic BLUP model.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/population-heterogeneity" class="hw-term hw-article-keyword hw-article-keyword-population-heterogeneity" rel="nofollow">population heterogeneity</a></li><li class="kwd"><a href="/keyword/marker-population-interaction" class="hw-term hw-article-keyword hw-article-keyword-marker-by-population-interaction" rel="nofollow">marker-by-population interaction</a></li><li class="kwd"><a href="/keyword/kernel-functions" class="hw-term hw-article-keyword hw-article-keyword-kernel-functions" rel="nofollow">kernel functions</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/panicum-virgatum" class="hw-term hw-article-keyword hw-article-keyword-panicum-virgatum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Panicum virgatum</em></a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-4">Genomic prediction has proved a useful tool to predict genetic merit in plant and animal breeding (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hayes <em>et al.</em> 2009a</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lorenz <em>et al.</em> 2011</a>). This technology consists of learning relationships between DNA markers and phenotypes, which arise from the non-random association (linkage disequilibrium; LD) between DNA markers and causal genetic variants having direct effects on the trait studied (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Meuwissen <em>et al.</em> 2001</a>). Standard genomic prediction models, including genomic BLUP (GBLUP; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">VanRaden 2008</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hayes <em>et al.</em> 2009b</a>) or Bayesian linear regression (BLR) models (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Meuwissen <em>et al.</em> 2001</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gianola <em>et al.</em> 2009</a>), assume that the effects of causal variants are linear and purely additive, so estimated effects do not capture any dependence on context, arising for example from interactions of causal variants with environmental or genetic backgrounds. Initially, genomic prediction models have been proposed for applications in populations that are relatively homogeneous with respect to LD patterns and interactions involving causal variants (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Meuwissen <em>et al.</em> 2001</a>). In such situations, increasing the size of the calibration set (CS) – the set of individuals used to estimate the model’s parameters – would typically benefit accuracy of the models (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lorenzana &amp; Bernardo 2009</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">VanRaden <em>et al.</em> 2009</a>). However, in practice, increasing the CS size may often involve calibrating prediction models on individuals with inconsistent LD patterns and/or backgrounds, which may result in reduced accuracy (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wientjes <em>et al.</em> 2016</a>). This issue will arise in the typical situation where an initially homogeneous CS is augmented with individuals from extraneous populations, that is, multi-population – or (in the animal literature) multi-breed – calibration (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lund <em>et al.</em> 2014</a>). Recently, studies in both plant and animal breeding have assessed the usefulness of combining populations from different genetic backgrounds in genomic prediction.</p><p id="p-5">Under standard genomic prediction models (where population heterogeneity is ignored), the simulation study of <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">de Roos <em>et al.</em> (2009)</a> suggested that adding an extraneous population to a CS may benefit prediction accuracy if the added population is not too dissimilar (in terms of divergence time) from the initial CS. These authors also suggested that high enough marker density could prevent prediction accuracy from decreasing, even in cases of strong divergence between populations. Consistently, most empirical studies of multi-population calibration with high marker density, based on standard GBLUP and/or BLR, have reported little or no gain in accuracy under strong population structure (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lehermeier <em>et al.</em> 2015</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jarquin <em>et al.</em> 2016</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hayes <em>et al.</em> 2009c</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Erbe <em>et al.</em> 2012</a>). In contrast, only a few studies have reported substantial increases in accuracy from multi-population calibration in similar conditions (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Technow <em>et al.</em> 2013</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Daetwyler <em>et al.</em> 2012</a>).</p><p id="p-6">In multi-population prediction models (where marker-by-population interactions are explicitly accounted for), studies have proposed to fit, to the whole set of available individuals, models that were capable of accommodating population heterogeneity explicitly. This type of models includes multi-trait GBLUP models, with “traits” corresponding to population backgrounds (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Karoui <em>et al.</em> 2012</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carillier <em>et al.</em> 2014</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lehermeier <em>et al.</em> 2015</a>), and random regression models based on markers interacting with discrete population cluster coefficients (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Los Campos <em>et al.</em> 2015</a>, with an extension of a standard BLR model). To our knowledge, the implementation of these methods has not been adapted to contexts of admixture, where population structure variables are continuous. Furthermore, when calibration involves many populations, the increase in model complexity of these methods will make them computationally intractable and statistically inefficient. Parsimonious multi-population models, based on only a few parameters to capture population heterogeneity, have also been proposed (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zhou <em>et al.</em> 2014</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heslot and Jannink 2015</a>). In the presence of many populations, such models are more practical and potentially more useful than multi-trait and random interaction models. Also, since they generally assume some underlying basis for population heterogeneity (<em>e.g.</em>, inconsistency in LD patterns), they may generate insight about the causes of marker-by-population interactions.</p><p id="p-7">In this study, we first considered a standard GBLUP model and evaluated different types of fixed effects to reflect population structure, then we investigated the usefulness of standard GBLUP and two kernel-based extensions for coping with population heterogeneity. These procedures were compared to a standard GBLUP model under two validation schemes: whole-sample calibration, where all individuals except selection candidates are included in the calibration set, and cross-population calibration, where the target population is excluded from the calibration set. The two multi-population GBLUP extensions are one previously-reported model, derived from <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Heslot and Jannink (2015)</a>, and one proposed model, based on a flexible kernel function of population principal components. We applied the procedures to the analysis of three traits (plant height, heading date, and standability) in switchgrass (<em>Panicum virgatum</em> L.), an herbaceous biomass crop showing good promise for bioenergy production (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sanderson <em>et al.</em> 1996</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Langholtz <em>et al.</em> 2016</a>). This species is characterized by an extensive diversity which makes it particularly suitable for studying population heterogeneity (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casler 2012</a>). The sample under study comprised seven population clusters from two diverse panels, assayed in the Midwestern region of the United States, which represent differentiation by ecotype (upland or lowland), geographical origin (latitudinal and longitudinal gradients) and ploidy level (tetraploid or octoploid). This sample exemplified the heterogeneity of data available for practical applications of genomic prediction, which pose both opportunities (by increased sample sizes) and challenges (by inconsistencies in marker effects across populations) for breeding based on DNA markers.</p><p id="p-8">In this study, we did not fit BLR models (usually based on Markov chain Monte Carlo optimizations), since we focused on deterministic methods for model fit and considered only models based on computationally efficient best linear unbiased predictors (BLUP). Further research would be needed to develop kernel-based extensions of this type of models.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and methods</h2><div id="sec-2" class="subsection"><h3>Panels and populations</h3><p id="p-9">In this study, two multi-population panels were assayed and considered together in one sample. The first panel was the breeding panel (BP) described in <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ramstein <em>et al.</em> (2016)</a>, comprising two tetraploid breeding populations of half-sib families: WS4U-C2, which consisted of 137 half-sib families derived from a diverse upland-ecotype pool of 162 plants (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Casler <em>et al.</em> 2006</a>), and Liberty-C2, which consisted of 110 half-sib families derived from the lowland-upland cultivar Liberty (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casler and Vogel 2014</a>). The second panel was the association panel (AP) described in <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lu <em>et al.</em> (2013)</a> and <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> (2015)</a>, comprising six putative populations of clonally propagated genotypes of different ecotypes (U: upland; L: lowland), ploidy levels (4X: tetraploid; 8X: octoploid) and geographical origins (S: South; W: West; N: North; E: East): U4X-N (135 plants), U8X-W (129 plants), U8X-E (97 plants), U8X-S (10 plants), L4X-NE (106 plants) and L4X-S (37 plants). These populations corresponded to 66 diverse accessions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lu <em>et al.</em> 2013</a>, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2015</a>) with up to 10 individuals per accession.</p><p id="p-10">In WS4U-C2, one individual was discarded so as to avoid assigning it to a population in AP, since it was too distantly related to the other individuals in BP (based on principal component analysis). In total, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-1.gif"/></noscript></span></span> individuals were considered in this analysis. The main goal of this study was to assess different methods for accommodating genetic heterogeneity when predicting phenotypic means in a given target population. Four targets were chosen, with a defined focus on tetraploid populations with at least 100 relatively homogeneous individuals: WS4U-C2 and Liberty-C2 (from BP), and U4X-N and L4X-NE (from AP).</p></div><div id="sec-3" class="subsection"><h3>Marker data</h3><p id="p-11">Exome capture sequencing of individuals (parents in BP and clonally propagated plants in AP) was performed using the Roche-Nimblegen protocol for preparation of SeqCap EZ Developer libraries using the Roche-Nimblegen probeset ‘120911_Switchrass_GLBRC_R_EZ_HX1’ as described previously (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2014</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">2015</a>). Reads from sequencing were aligned to the hardmasked <em>P. virgatum</em> v1.1 reference genome (<a href="http://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Pvirgatum">http://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Pvirgatum</a>). Counts of reads corresponding to alternate and reference alleles for each individual were then determined as described previously (for BP, <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Ramstein <em>et al.</em> 2016</a>; for AP, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2014</a>, <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">2015</a>) at 2,179,164 single nucleotide polymorphism (SNP) loci, which were identified as polymorphic in two diversity panels: the Northern Switchgrass Panel, corresponding to AP (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2014</a>, <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">2015</a>), and a southern switchgrass panel (E. C. Brummer, unpublished data). The numbers of alternate allele at the SNP loci were then called by using the expectation-maximization algorithm of <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Martin <em>et al.</em> (2010)</a> fitted in each population (in BP) or accession (in AP) separately, under the assumption of disomic inheritance. Although this assumption is supported in switchgrass for tetraploid genotypes (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Okada <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li <em>et al.</em> 2014</a>), it does not hold for octoploid genotypes, which would presumably exhibit tetrasomic inheritance. However, we did not adapt the algorithm of <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Martin <em>et al.</em> (2010)</a> to accommodate possible tetrasomic inheritance, as the sequencing depth of ∼24 was deemed insufficient for calling intermediate heterozygotes (simplex and triplex) with high enough accuracy (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2015</a>). Indeed, a sequencing depth of at least 60-80 was recommended by <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Uitdewilligen <em>et al.</em> (2013)</a> for accurately calling tetrasomic genotypes. Therefore, for all individuals, the resulting marker-data matrix consisted of expected allelic dosages (sums of alternate-allele counts weighted by their posterior probabilities, for every individual and SNP) between 0 and 2.</p><p id="p-12">The SNP data were filtered based on the following criteria: (i) proportion of missing values strictly lower than 2%, a stringent threshold given prior filtering of SNPs on read depth ≥ 5 (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2014</a>, <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">2015</a>); (ii) minor allele frequency strictly greater than <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-2.gif"/></noscript></span></span> and variance of allelic dosages strictly greater than <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-3.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-4.gif"/></noscript></span></span> individuals; (iii) <em>p</em>-value for Hardy-Weinberg equilibrium strictly greater than 10<sup>−4</sup> in each BP population; (iv) availability of genomic-location information (as per v1.1 of the reference genome of <em>P. virgatum</em>). Missing values at SNPs were imputed by their mode in the whole sample. The resulting <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-5.gif"/></noscript></span></span> filtered and imputed marker-data matrix <strong>X</strong> consisted of allelic dosages at <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-6.gif"/></noscript></span></span> SNP markers.</p></div><div id="sec-4" class="subsection"><h3>Phenotypic data</h3><p id="p-13">Populations in BP were assayed each year between 2012 and 2014, in Arlington, WI (USA), in a randomized complete block design, with four replicates for WS4U-C2 and three replicates for Liberty-C2. Populations in AP were assayed each year between 2009 and 2011 in Ithaca, NY (USA), in a sets-in-reps design, with two replicates per individual and 10 sets within each replicate, with each set comprising at most one individual from each of the 66 accessions in AP (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Lu <em>et al.</em> 2013</a>, <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2015</a>). In each panel, three phenotypic traits were considered: plant height, heading date and standability. Plant height (PH) was measured in centimeters as the height from the ground to the top of the tallest panicle. Heading date (HD) was measured in growing degrees days as the cumulated sum of daily average temperatures (in degrees Celsius; °) above 10 °, from January 1<sup>st</sup> to the day of heading, defined as the emergence of at least half of the panicles from the boot (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mitchell <em>et al.</em> 1997</a>); daily average temperatures were estimated by the average of the minimum and maximum daily temperatures. Standability (St) was measured on a 0-10 scale to describe plants’ stature and stiffness, with 0 qualifying plants that are prostrate and 10 qualifying upright and rigid plants (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lipka <em>et al.</em> 2014</a>).</p><p id="p-14">Not all traits were measured every year in any given population: only HD was measured in all three years in AP populations and Liberty-C2. For all other cases, measurements were available for only a subset of years (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492356/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492356/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of populations and trait measurements</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">In BP, observational units were plants within half-sib families from a given genotype (maternal parent) <em>i</em>. Half-sib families were arranged in a randomized complete block design and assayed in multiple years; so the following mixed model was fitted to phenotypic measurements <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-7.gif"/></noscript></span></span>, to estimate half-sib family effects <em>f<sub>i</sub></em>’s:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-8.gif"/></noscript></span></span>where μ is the population mean; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-9.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-11.gif"/></noscript></span></span> are the effects of half-sib family i (fixed), block j (random) and year l (random) respectively; × indicates interactions (random); <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-12.gif"/></noscript></span></span> are residuals for plant <em>m</em> within plot <em>ij</em> in year <em>l</em>. In AP, observational units were clones of a given genotype <em>i</em>. Genotypes were arranged in a sets-in-reps design and assayed in multiple years; so the following model was fitted to measurements <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-13.gif"/></noscript></span></span> to estimate genotype effects <em>g<sub>i</sub></em>’s:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-14.gif"/></noscript></span></span>where μ is the panel mean; <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-16.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-18.gif"/></noscript></span></span> are the effects of genotype i (fixed), replicate j (random), set <em>k</em> within replicate <em>j</em> (random) and year l (random) respectively; × indicates interactions (random); <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-19.gif"/></noscript></span></span> are residuals for clone <em>ij</em> in year <em>l</em>.</p><p id="p-17">The models described above conform to analyses of strip-plot (split-block) designs (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Steel <em>et al.</em> 1996</a>), in which years and genotype classes (half-sib families in BP, individual genotypes in AP) are whole-plot factors in cross-classification and sub-plot factors are combinations of years and genotype classes. For each random term, the corresponding effects were modeled as independent and identically normally distributed. The linear mixed models described above were fitted using ASREML-R (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butler <em>et al.</em> 2009</a>).</p><p id="p-18">Effects <em>f<sub>i</sub></em>’s in BP are transmitting abilities of genotypes (the mean of their half-sib progeny in their respective breeding population), so <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-20.gif"/></noscript></span></span>, where <em>BV<sub>i</sub></em> is the breeding value of genotype <em>i</em>. In comparison, effects <em>g<sub>i</sub></em>’s in AP are genotypic values, such that <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-21.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-22.gif"/></noscript></span></span> is the deviation from additivity due to dominance and/or epistasis. Thus, outcomes of interest for genomic prediction were set to be genotype means <em>y<sub>i</sub></em>’s such that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-23.gif"/></noscript></span></span> in BP and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-24.gif"/></noscript></span></span> in AP, with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-25.gif"/></noscript></span></span> the estimated population mean in BP or estimated panel mean in AP.</p></div><div id="sec-5" class="subsection"><h3>Population structure and relatedness</h3><div id="sec-6" class="subsection"><h4>Admixture analysis:</h4><p id="p-19">The soft clustering model from the ADMIXTURE software was fitted on the whole sample and the whole set of SNPs, <em>i.e.</em>, without selection on individuals or markers (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alexander <em>et al.</em> 2009</a>). Based on the 10-fold cross-validation implemented in ADMIXTURE (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander and Lange 2011</a>), the number of population clusters in the admixture model was set to <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-26.gif"/></noscript></span></span>, as cross-validation error reached a plateau at that value (Figure S1). The resulting <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-27.gif"/></noscript></span></span> matrix <strong>A</strong> of admixture coefficients comprised inferred membership probabilities at each cluster (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). For convenience (in prediction models), minimum values in <strong>A</strong> (10<sup>−5</sup>) were set to zero while ensuring that each row still summed to one (by dividing each element in <strong>A</strong> by its row sum).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/789/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure in the sample (A) Admixture plot of the whole sample, with colors designating the seven inferred population clusters, which roughly matched populations, with the exception of U8X-S which displayed strong admixture; (B) Principal component analysis (PCA) plot of the whole sample of 760 individuals, with colors designating the eight populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-201583134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population structure in the sample (A) Admixture plot of the whole sample, with colors designating the seven inferred population clusters, which roughly matched populations, with the exception of U8X-S which displayed strong admixture; (B) Principal component analysis (PCA) plot of the whole sample of 760 individuals, with colors designating the eight populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/789/F1.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/789/F1.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/789/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/789/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Population structure in the sample (A) Admixture plot of the whole sample, with colors designating the seven inferred population clusters, which roughly matched populations, with the exception of U8X-S which displayed strong admixture; (B) Principal component analysis (PCA) plot of the whole sample of 760 individuals, with colors designating the eight populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h4>Principal component analysis:</h4><p id="p-21">Principal component analysis (PCA) was performed on the whole sample and the whole set of SNPs. The number of principal components (PCs) to choose for depicting population structure was chosen based on the proportion of variance explained and the grouping patterns captured by PCs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>). The resulting <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-28.gif"/></noscript></span></span> PC matrix <strong>P</strong> consisted of coordinates for each individual at the first <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-29.gif"/></noscript></span></span> PCs.</p></div><div id="sec-8" class="subsection"><h4>Recent relationships among individuals:</h4><p id="p-22">Let <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-30.gif"/></noscript></span></span> be the genomic relationship matrix as defined by <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">VanRaden (2008)</a>, where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-31.gif"/></noscript></span></span> is the centered marker-data matrix and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-32.gif"/></noscript></span></span> is a scaling factor depending on allele frequencies <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-33.gif"/></noscript></span></span>′s (estimated on the whole sample), where <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-34.gif"/></noscript></span></span> is the number of SNP markers. Following <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fan <em>et al.</em> (2013)</a>, <strong>G</strong> was decomposed as <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-35.gif"/></noscript></span></span>, where P consisted of the first <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-36.gif"/></noscript></span></span> PCs, as described above. Matrix <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-37.gif"/></noscript></span></span> is the dense part of the relationship matrix <strong>G</strong>, representing resemblance among individuals through population structure, whereas matrix <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-38.gif"/></noscript></span></span> represents recent relationships conditional on population structure, similarly to the adjusted relationships introduced by <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Thornton <em>et al.</em> (2012)</a> and <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conomos <em>et al.</em> (2016)</a>, with the difference that here coefficients in <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-39.gif"/></noscript></span></span> are not scaled for direct estimation of recent-kinship coefficients. Here the graphical LASSO was applied to <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-40.gif"/></noscript></span></span> to infer a graph of recent relationships among individuals, according to a regularization parameter <em>λ</em>. Parameter <em>λ</em> was chosen to maximize the restricted likelihood in a GBLUP model based on the regularized genomic relationship matrix <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-41.gif"/></noscript></span></span>, fitted to the whole sample (see Appendix 1 for technical details and discussion on graph inference). The GBLUP model depicting relationships through <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-42.gif"/></noscript></span></span> was fitted for each trait separately, so different optimal values of <em>λ</em> were inferred for different traits.</p></div></div><div id="sec-9" class="subsection"><h3>Genomic prediction models</h3><p id="p-23">All linear mixed models described below were fitted using the R package rrBLUP (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Endelman 2011</a>).</p><p id="p-24">For a given marker-data matrix <strong>X</strong> and vector <strong>y</strong> of outcomes, the standard GBLUP model is described as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <strong>y</strong> is the <em>n</em>-vector of genotype means (<em>y<sub>i</sub></em>′s, as described above); <strong>X</strong> is the <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-44.gif"/></noscript></span></span> marker-data matrix (here consisting of allelic dosages), and <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-45.gif"/></noscript></span></span> is the variance of marker effects; <strong>Q</strong> is the model matrix for fixed effects <strong>α</strong>; <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-46.gif"/></noscript></span></span> is the covariance matrix for errors considered independent and identically distributed.</p><p id="p-25">Hereafter, the testing set TS is defined as the set of individuals left out for model validation. The calibration set CS is the set of individuals used to fit the prediction models, which excludes the TS but does not necessarily consist of all remaining (available) individuals.</p><p id="p-26">We defined the mean structure in fitted models by <strong>Q</strong> being one of the following: <em>(Intercept</em>) a <em>n</em>-vector of ones <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-47.gif"/></noscript></span></span>, such that fixed effects consisted of a single intercept; (<em>PCA</em>) the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-48.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-49.gif"/></noscript></span></span> of column vector of ones and first four PCs; (<em>Panel</em>) the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-50.gif"/></noscript></span></span> model matrix attributing observations to panel AP or BP, such that fixed effects reflected differences in genetic compositions and environments across panels; or (<em>Group</em>) the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-51.gif"/></noscript></span></span> matrix model matrix attributing individuals to the following putative population groups: WS4U-C2, Liberty-C2, U4X-N, U8X-W+U8X-S, U8X-E, L4X-NE and L4X-S. Genotypes from U8X-S were grouped with U8X-W on the basis of their proximity according to the first 4 PCs, to avoid having one group with too few observations.</p><p id="p-27">In this study, we first compared mean structures with respect to prediction accuracy under the standard prediction procedure (<em>GBLUP</em>, as described below). Then, we focused on <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-52.gif"/></noscript></span></span> and compared prediction procedures for accommodating population heterogeneity (see below; <em>GBLUP</em>, <em>GBLUP-Target</em>, <em>MPM-Mixture</em>, <em>MPM-Matérn</em>). Prediction accuracy of models (differing either by mean structure or prediction procedure) was assessed by cross-validation as described in the next subsection (Validations).</p><div id="sec-10" class="subsection"><h4>Whole-sample model: GBLUP:</h4><p id="p-28">In the whole-sample model (<em>GBLUP</em>), we fitted model (1) to all available individuals, thereby assuming that the whole sample consists of only one population. This method consists of ignoring population heterogeneity and relying on robustness of standard GBLUP to interactions between markers and population backgrounds.</p></div><div id="sec-11" class="subsection"><h4>Target-population model: GBLUP-Target:</h4><p id="p-29">In the target-population model (<em>GBLUP-Target</em>), we fitted model (1) to individuals belonging to the same population as the TS, when possible (see below). This method corresponds to a typical choice of reducing population heterogeneity and basing predictions only on individuals that have genetic backgrounds that are <em>a priori</em> similar to those in the TS.</p></div><div id="sec-12" class="subsection"><h4>Multi-population models: MPM-Mixture, MPM-Matérn:</h4><p id="p-30">Multi-population models (<em>MPM</em>) were extensions of model (1) intended to accommodate population heterogeneity. The following general model was fitted:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where ○ is the element-wise (Hadamard) product, and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-54.gif"/></noscript></span></span> is a <em>n</em> × <em>n</em> covariance matrix depicting population differentiation among individuals (see Appendix 2 for derivations and technical details). To parsimoniously estimate <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-55.gif"/></noscript></span></span>, we used two different procedures: <em>MPM-Mixture</em> (based on <strong>A</strong>) and <em>MPM-Matérn</em> (based on <strong>P</strong>). In both procedures, we did not model any heteroscedasticity for additive genetic effects <strong>u</strong>.</p><p id="p-31">In <em>MPM-Mixture</em> (the reference <em>MPM</em> procedure), <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-56.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-57.gif"/></noscript></span></span> is the <em>n</em> × <em>n</em> matrix of ones and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-58.gif"/></noscript></span></span> is a <em>K</em> × <em>K</em> matrix depicting relationships among population clusters as inferred in <strong>A</strong>. Here, we simply set <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-59.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-60.gif"/></noscript></span></span> is the <em>K</em> × <em>K</em> identity matrix), so <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-61.gif"/></noscript></span></span>. Therefore in this procedure, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-62.gif"/></noscript></span></span> set a trade-off between the case where relationships were cluster-specific (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-63.gif"/></noscript></span></span>) and the case where relationships assumed one single homogeneous population for all individuals (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-64.gif"/></noscript></span></span>). This approach is similar (but not exactly equivalent) to the <em>K</em>-kernel method of <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Heslot and Jannink (2015)</a>, which set a similar balance between cluster-specific and overall relationships, but using G for relationships (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">VanRaden 2008</a>), instead of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-65.gif"/></noscript></span></span>, and considering only discrete population clusters (in which case values in <strong>A</strong> would then be only 0 or 1). Alternatively, <em>MPM-Mixture</em> may be viewed as a multi-kernel model where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-67.gif"/></noscript></span></span> are the variance components respectively associated to cluster-specific and main marker effects.</p><p id="p-32">In <em>MPM-Matérn</em> (the proposed <em>MPM</em> procedure),<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-68.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-69.gif"/></noscript></span></span> is a Matérn kernel function of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-70.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-71.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-72.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-73.gif"/></noscript></span></span> is the Euclidean distance between the <em>d</em>-vectors of PC coordinates for any pair (<em>i</em>, <em>j</em>) of individuals, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-74.gif"/></noscript></span></span> is a shape parameter, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-75.gif"/></noscript></span></span> is a scale parameter, and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-76.gif"/></noscript></span></span> is the modified Bessel function of the second kind, of order <em>ν</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramowitz and Stegun 1984</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ober <em>et al.</em> 2011</a>). Matérn functions have been used in various contexts, including in genomic prediction for depicting relationships among individuals (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Ober <em>et al.</em> 2011</a>). Here, we used Matérn functions to depict relationships among populations, with the input <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-77.gif"/></noscript></span></span> representing differentiation with respect to population structure in <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-78.gif"/></noscript></span></span> orthogonal directions. We used Matérn functions instead of more typical kernel functions (<em>e.g.</em>, an exponential or Gaussian kernel function) to allow for some flexibility in the shape of the correlation in <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-79.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-81.gif"/></noscript></span></span> correspond respectively to the exponential and Gaussian kernels as special cases, while different shapes can also be fitted (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Ober <em>et al.</em> 2011</a>).</p><p id="p-33">The parameter <em>ρ</em> in <em>MPM-Mixture</em> was estimated by maximizing the restricted likelihood of model (2) using the optimization algorithm implemented in the R function <em>optimize</em>. The parameters <em>ν</em> and <em>h</em> in <em>MPM-Matérn</em> were estimated by maximizing the restricted likelihood of model (2) using the Nelder-Mead algorithm implemented in the R function <em>constrOptim</em>, with constraints for positivity. In order to control (to some extent) for the possible presence of local maxima in the restricted likelihood surface in <em>MPM-Matérn</em>, we used four different starting points <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-82.gif"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-83.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-84.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-85.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-86.gif"/></noscript></span></span>, with <em>D<sub>max</sub></em> the maximum distance <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-87.gif"/></noscript></span></span> observed over pairs of individuals (<em>i</em>, <em>j</em>). In cross-validation (see next section), parameters <em>ρ</em>, <em>ν</em> and <em>h</em> were estimated in each CS separately.</p></div></div><div id="sec-13" class="subsection"><h3>Validations</h3><p id="p-34">We assessed the accuracy of our prediction procedures by cross-validation (CV) under two schemes: whole-sample calibration, where all individuals except the TS are included in the CS, and cross-population calibration, where the target population (the population to which the TS belongs) is excluded from the CS. The target-population model <em>GBLUP-Target</em> was only assayed in whole-sample calibration, since this model could only rely on individuals from the target population for calibration (in <em>GBLUP-Target</em>, the CS could only consist of individuals in the target population, which was not possible in cross-population calibration).</p><p id="p-35">For each target population (L4X-NE, U4X-N, Liberty-C2 or WS4U-C2), we used as the TS a random subset of the target sample. The size of the TS was one fifth of the target sample size. All remaining individuals were used as input to the prediction procedures. Such validations were replicated <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-88.gif"/></noscript></span></span> times for each target.</p><p id="p-36">Prediction procedures were evaluated for accuracy by <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-89.gif"/></noscript></span></span>, <em>i.e.</em>, the correlation between actual and predicted outcomes in a given TS. To assess the significance of differences in prediction accuracy between two procedures, we performed a <em>t</em>-test on <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-90.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-91.gif"/></noscript></span></span> is the average of <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-92.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-93.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-94.gif"/></noscript></span></span>) is the vector of prediction accuracies over testing sets for the tested procedure (baseline procedure); and z is the Fisher transformation. The standard error of the mean difference in prediction accuracy, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-95.gif"/></noscript></span></span>, was estimated in two different ways: (liberal <em>t</em>-test) <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-96.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-97.gif"/></noscript></span></span> is the standard deviation of δ, with all testing sets assumed to be independent datasets; (conservative <em>t</em>-test) based on the first method of <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nadeau and Bengio (2003)</a>, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-98.gif"/></noscript></span></span>, where redundancy over testing sets is accounted for by the additional term <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-99.gif"/></noscript></span></span>, with <em>o</em> being the expected fraction of overlap among testing sets; here <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-100.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-101.gif"/></noscript></span></span> because testing sets were random subsets consisting of a fifth of any given target sample. We considered that this method for estimating <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-102.gif"/></noscript></span></span> was conservative in whole-sample calibration because <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Nadeau and Bengio (2003)</a> derived it by assuming that the CV criterion (the “loss function”, analog here to <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-103.gif"/></noscript></span></span>, for a given TS) did not depend on the CS instances, given a particular CS size. Therefore the adjustment from <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Nadeau and Bengio (2003)</a> may have overestimated the correlation among values of the CV criterion across replicates, in whole-sample calibration, since prediction procedures are probably quite sensitive to differences in the composition of the CS. In all comparisons between procedures, we reported the results from both tests in order to characterize the significance of differences in prediction accuracy.</p></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-37">Population information (population assignment and geographical origin of genotypes, when available), raw phenotypic data (trait measurements at individual plants) and estimated genotype means (for maternal parents in BP and individuals in AP) are available in Files S1, S2 and S3, respectively. These supplementary files as well as the marker data (allelic dosages at the 717,814 selected SNP markers; in .rds format readable in R) are available from figShare. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7464863">https://doi.org/10.25387/g3.7464863</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Population structure in the sample</h3><div id="sec-17" class="subsection"><h4>Population-level differentiation:</h4><p id="p-38">Seven population clusters were inferred from the ADMIXTURE software (Figure S1; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alexander <em>et al.</em> 2009</a>). These clusters corresponded roughly to populations L4X-NE, L4X-S, Liberty-C2 and U4X-N, WS4U-C2, U8X-E, U8X-W. One population with little representation in our sample, U8X-S, appeared to be of mixed origin (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1a</a>). The other populations generally displayed a low level of admixture, with relatively few individuals having intermediate admixture coefficients. There seemed to be some admixture involving upland populations (WS4U-C2 and U4X-N, WS4U-C2 and U8X-W, U8X-E and U8X-W), with even some shared ancestry between WS4U-C2 and U4X-N. The PCA confirmed that population structure was relatively discrete (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1b</a>). Expectedly, the first PC separated genotypes by ecotype while the second PC reflected geographical origin within the lowland ecotype (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Lu <em>et al.</em> 2013</a>, <a id="xref-ref-21-9" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2015</a>). The third and four PCs discriminated upland genotypes by geographical origin and ploidy level, and distinguished L4X-S from the two other lowland populations (L4X-NE and Liberty-C2).</p><p id="p-39">Differences in mean and range among populations were quite typical of previously reported differences between ecotypes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Casler 2012</a>). Indeed, L4X-S and Liberty-C2 (populations of lowland origin) had high mean values and range values for PH, HD and St, compared to upland populations (excluding U8X-S). However, L4X-NE stood out as a lowland population for being relatively short, early-flowering, and prone to lodging, with corresponding values for PH, HD, and St more similar to those of the upland populations.</p></div><div id="sec-18" class="subsection"><h4>Recent relationships in the sample:</h4><p id="p-40">Here, marginal genomic relationships were defined as the elements of <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-104.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-105.gif"/></noscript></span></span> consisting of centered marker variables, and <em>ν</em> being some scaling factor. The strong and quite discrete population structure in the sample translated into multimodal marginal genomic relationship coefficients, with the multiple peaks in off-diagonal elements of <strong>G</strong> reflecting differentiation of population with respect to allele frequencies (Figure S2a). Conditioning relationships on population structure (as depicted by the first four PCs of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-106.gif"/></noscript></span></span>) yielded the matrix <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-107.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-108.gif"/></noscript></span></span> and <strong>P</strong> reflecting structure in <strong>G</strong> due to population-level variation (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fan <em>et al.</em> 2013</a>). The conditional genomic relationships seemed sparser, in the sense that they appeared to cluster around zero, so most individuals could be assumed to be unrelated after accounting for population structure in the sample (Figure S2b). Conditional relationships in <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-109.gif"/></noscript></span></span> were particularly relevant in this study, since among-population variation, captured by <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-110.gif"/></noscript></span></span>, contributed little to variation within any given TS. Indeed, any TS generally consisted of selection candidates from a relatively homogeneous target sample (made of individuals from WS4U-C2, Liberty-C2, U4X-N or L4X-NE), where variation with respect to P was minimal. Graphs of recent relationships, inferred by the graphical LASSO, were rather dense, with average degrees (number of neighbors by node/individual in the graph) ranging from 217 to 458 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). However, some noticeable features of populations emerged from the inferred graphs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>): WS4U-C2, U4X-N and U8X-E appeared quite connected to one another; U8X-W also showed some connection with other upland populations but seemed more distinct, as reflected by a relatively lower average degree (Figure S3); Liberty-C2 and L4X-S were somewhat connected to both upland and lowland populations, which certainly explains why their individual degrees were generally high (Figure S3); most notably, L4X-NE displayed an outstandingly low level of connection with the other populations, which translated in a clear separation of this population in the graph, after placing the nodes based on a force-directed algorithm (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fruchterman and Reingold 1991</a>). These features exemplify the usefulness of conditional relationships and their associated graphs for describing relationships among individuals.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/789/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Inferred graphs of relationships, conditional on population structure Each graph represents the relationships as depicted by the graphical LASSO applied to the whole sample of individuals. The parameter &#x3BB; represents the degree of regularization on conditional relationships, fitted by maximum restricted likelihood for each trait, in a GBLUP model based on regularized relationships (Appendix 1). Nodes (individuals) were positioned using the force-directed placement algorithm of Fruchterman and Reingold (1991), as implemented in function ggnet (R package GGally), so aggregation of nodes reflects connectedness." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-201583134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Inferred graphs of relationships, conditional on population structure Each graph represents the relationships as depicted by the graphical LASSO applied to the whole sample of individuals. The parameter &lt;em&gt;&#x3BB;&lt;/em&gt; represents the degree of regularization on conditional relationships, fitted by maximum restricted likelihood for each trait, in a GBLUP model based on regularized relationships (Appendix 1). Nodes (individuals) were positioned using the force-directed placement algorithm of Fruchterman and Reingold (1991), as implemented in function ggnet (R package GGally), so aggregation of nodes reflects connectedness.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/789/F2.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/789/F2.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/789/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/789/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492528" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Inferred graphs of relationships, conditional on population structure Each graph represents the relationships as depicted by the graphical LASSO applied to the whole sample of individuals. The parameter <em>λ</em> represents the degree of regularization on conditional relationships, fitted by maximum restricted likelihood for each trait, in a GBLUP model based on regularized relationships (Appendix 1). Nodes (individuals) were positioned using the force-directed placement algorithm of <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fruchterman and Reingold (1991)</a>, as implemented in function ggnet (R package GGally), so aggregation of nodes reflects connectedness.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-19" class="subsection"><h3>Impact of mean structure on prediction accuracy</h3><p id="p-42">Prior to assessing different prediction procedures accommodating population heterogeneity, models were compared for different fixed-effect specifications used to characterize population structure (mean structures of models). Mean structures were tested for prediction accuracy under <em>GBLUP</em>, in which the whole sample, excluding the TS, was used to fit a standard GBLUP model. In whole-sample calibration (where the target population was included in the CS), the various mean structures assayed differed marginally with respect to their prediction accuracy. There were improvements over <em>Intercept</em> (only intercept as fixed effect) by mean structures which explicitly captured population structure, <em>i.e.</em>, <em>PCA</em> (intercept and effects of PCs), <em>Panel</em> (effects of panels AP and BP) and <em>Group</em> (effects of putative population groups). However, those were small, inconsistent and moderately significant (hereafter “moderately significant” refers to <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-111.gif"/></noscript></span></span> based on the liberal “naïve” <em>t</em>-test) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2a</a>). Conversely, in cross-population calibration (where the target population was excluded from the CS), fixed effects explicitly depicting population structure resulted in highly significant decreases in prediction accuracy compared to <em>Intercept</em> (hereafter “highly significant” refers to <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-112.gif"/></noscript></span></span> based on the conservative <em>t</em>-test adapted from <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Nadeau and Bengio 2003</a>; see <em>Material and methods</em> for details). In particular, prediction accuracy was substantially lower with <em>PCA</em> for L4X-NE (HD, St), as well as with <em>Group</em> for U4X-N (PH, HD), L4X-NE (PH, HD) and Liberty-C2. Mean structure <em>Panel</em> was not as sensitive to cross-population calibration compared to <em>PCA</em> and <em>Group</em>, and even showed one highly significant increase in prediction accuracy compared to <em>Intercept</em>, for Liberty-C2 (PH). But it also showed decreases in prediction accuracy, with U4X-N (PH) and L4X-NE (HD), which were stronger and highly significant (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2b</a>). Interestingly, the deterioration of prediction accuracy in cross-population calibration by <em>PCA</em> and <em>Group</em> may be due to different factors. Indeed, while <em>PCA</em> may fail to properly extrapolate effects of PCs outside the set of populations represented in the CS, <em>Group</em> would fail to capture any difference due to population differentiation in the TS (since all individuals in the TS belong to the same unobserved population group). Due to the relative stability in performance of <em>Intercept</em>, hereafter we chose to focus on this mean structure when comparing prediction procedures.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492374/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492374/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average prediction accuracy by mean structure</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Impact of prediction procedures on prediction accuracy</h3><div id="sec-21" class="subsection"><h4>Target-population model:</h4><p id="p-44">For prediction in a given TS, the target-population model (<em>GBLUP-Target</em>) consisted in restricting the CS to the subset of the sample belonging to the same population as the TS. Compared to <em>GBLUP</em>, the target-population model yielded decreases in prediction accuracy which appeared moderately significant for PH (WS4U-C2, U4X-N) and St (Liberty-C2) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3a</a>, Table S1). However, prediction accuracy for St (WS4U-C2) was higher, with a moderately significant difference. More intriguing is the consistent increase in prediction accuracy with L4X-NE, with differences being small yet highly significant for PH and HD, and moderately significant for St. It is unclear whether these differences are due to the consistently higher accuracies achieved with L4X-NE (in <em>GBLUP-Target</em>) compared to other populations, or a result of L4X-NE being relatively under-connected to the other populations in the sample (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Figure S3). Both factors could very well contribute to the observed decreases in accuracy when incorporating information from the whole sample.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492337/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492337/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Average prediction accuracy by prediction procedure</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>Multi-population models and marker-by-population interactions:</h4><p id="p-46">The inferred mixing parameter <em>ρ</em> from the <em>MPM-Mixture</em> model was null (or close to null), low and intermediate, for PH, St and HD respectively, with estimations being quite consistent over CV replicates (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The improvement in fit, relatively to <em>GBLUP</em>, was non-significant for PH, rather significant (<em>P</em> &lt; 0.05) for St, and strongly significant (<em>P</em>  &lt;  0.001) for HD (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). In <em>MPM-Matérn</em>, the inferred correlation functions differed substantially across traits (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>), while being quite consistent over CV replicates in whole-sample calibration and across validation schemes, with similar shapes of the correlation function <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-113.gif"/></noscript></span></span> in whole-sample calibration and cross-population calibration (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>): <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-114.gif"/></noscript></span></span> roughly resembled an exponential kernel with PH and HD, and was more similar to a Gaussian kernel with St, for which a “shoulder” maintained high correlation in marker effects for individuals that were relatively close to each other, based on their PCs. Remarkably, the shapes of inferred correlation functions were quite consistent in cross-population calibration, despite entire populations being left out from one CS to another (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>). Inferences regarding among-population correlations (<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-115.gif"/></noscript></span></span>) in <em>MPM-Matérn</em> were weakly significant for PH and St, with <em>p</em>-values close to 0.05; in contrast, inferences regarding <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-116.gif"/></noscript></span></span> for HD were strongly significant, with <em>P</em> &lt; 0.001 (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). Interestingly, distances based on PCs may be equivalent to distances based on allele frequencies. Specifically, <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-117.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-118.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-119.gif"/></noscript></span></span>) is the <em>m</em>-vector of individual-specific allele frequencies of individual <em>i</em> (<em>j</em>) as described by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Conomos <em>et al.</em> (2016)</a>, with population structure described by <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-120.gif"/></noscript></span></span> (Appendix 3). Therefore, the significant relationship between PC-based distances and correlations in marker effects (depicted by <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-121.gif"/></noscript></span></span>) for HD in <em>MPM-Matérn</em> indicates that marker effects for this trait were highly sensitive to variation in allele frequencies across genetic backgrounds.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492327/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492327/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Multi-population model fit: parameter estimates, likelihood-ratio test statistic and <em>p</em>-value, by trait and procedure</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/789/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shape of the inferred correlation functions in MPM-Mat&#xE9;rn Validation scheme: (A) whole-sample calibration, where individuals in the target population (except the selection candidates) are included in the calibration set; (B) cross-population calibration, where all individuals from the target population are omitted from the calibration set. In (A), dashed curves depict correlation functions inferred in cross-validation replicates (where a part of the target population is included in the testing set), while solid curves depict correlation functions inferred in the whole sample. In (B), solid curves depict correlation functions inferred while omitting one of the four target populations in the calibration set. Correlations are functions of , where D is the Euclidean distance between population-structure PCs for any pair of individual (i,j) and Dmax is the maximum of D over the whole sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-201583134" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Shape of the inferred correlation functions in &lt;em&gt;MPM-Mat&#xE9;rn&lt;/em&gt; Validation scheme: (A) whole-sample calibration, where individuals in the target population (except the selection candidates) are included in the calibration set; (B) cross-population calibration, where all individuals from the target population are omitted from the calibration set. In (A), dashed curves depict correlation functions inferred in cross-validation replicates (where a part of the target population is included in the testing set), while solid curves depict correlation functions inferred in the whole sample. In (B), solid curves depict correlation functions inferred while omitting one of the four target populations in the calibration set. Correlations are functions of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, where &lt;em&gt;D&lt;/em&gt; is the Euclidean distance between population-structure PCs for any pair of individual (&lt;em&gt;i&lt;/em&gt;,&lt;em&gt;j&lt;/em&gt;) and &lt;em&gt;D&lt;sub&gt;max&lt;/sub&gt;&lt;/em&gt; is the maximum of &lt;em&gt;D&lt;/em&gt; over the whole sample.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/789/F3.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/789/F3.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/789/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/789/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-48" class="first-child">Shape of the inferred correlation functions in <em>MPM-Matérn</em> Validation scheme: (A) whole-sample calibration, where individuals in the target population (except the selection candidates) are included in the calibration set; (B) cross-population calibration, where all individuals from the target population are omitted from the calibration set. In (A), dashed curves depict correlation functions inferred in cross-validation replicates (where a part of the target population is included in the testing set), while solid curves depict correlation functions inferred in the whole sample. In (B), solid curves depict correlation functions inferred while omitting one of the four target populations in the calibration set. Correlations are functions of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/F3/embed/mml-math-124.gif.backup.1551969685.7972"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/F3/embed/mml-math-124.gif.backup.1551969685.7972"/></noscript></span></span>, where <em>D</em> is the Euclidean distance between population-structure PCs for any pair of individual (<em>i</em>,<em>j</em>) and <em>D<sub>max</sub></em> is the maximum of <em>D</em> over the whole sample.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">In whole-sample calibration, the performance of <em>MPM-Mixture</em> was very similar to that of <em>GBLUP</em>, with differences in accuracy ranging from -0.018 to +0.009 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3a</a>). Quite surprisingly, <em>MPM-Mixture</em> displayed slightly deteriorated accuracies for HD (with the exception of U4X-N), despite the strongly significant improvement in fit for this trait. In contrast, <em>MPM-Matérn</em> yielded larger differences in accuracy, ranging from -0.019 to +0.060 in whole-sample calibration (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3a</a>). With the two upland target populations (WS4U-C2 and U4X-N), noteworthy increases in prediction accuracy (+0.060 and +0.030 respectively) were observed for HD. But with the two other target populations (Liberty-C2 and L4X-NE), smaller differences in accuracy (-0.008 and +0.007 respectively) were observed for HD.</p><p id="p-50">In cross-population calibration, <em>MPM-Mixture</em> showed more differences in accuracy compared to <em>GBLUP</em>, with differences in accuracy ranging from -0.150 to 0.032 (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3b</a>). Again, <em>MPM-Mixture</em> displayed deteriorated accuracy for HD despite a strong improvement in fit, with a dramatic decrease by 0.15 for U4X-N. Such decrease in accuracy may be due to the lack of flexibility of <em>MPM-Mixture</em> in depicting among-population resemblance, as it only fits one correlation coefficient for all pairs of populations. In cross-population calibration, <em>MPM-Matérn</em> also resulted in large differences in accuracy compared to <em>GBLUP</em>, ranging from -0.304 to 0.147. The dramatic decreases in prediction accuracy with PH (-0.304 and -0.250 for WS4U-C2 and L4X-NE, respectively) could be explained by the relatively weak improvement in model fit from <em>GBLUP</em> to <em>MPM-Matérn</em> (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). Interestingly, large and significant improvements in prediction accuracy were observed with HD, similarly to the results from whole-sample calibration, with nonetheless more dramatic increases in accuracy. While <em>MPM-Mixture</em> simply estimates a general coefficient for among-population resemblance, based on the CS, <em>MPM-Matérn</em> may be more suitable for extrapolation to unobserved population backgrounds, as it estimates the resemblance between any two populations as a function of their specific properties (here, PC coordinates). Consistently, the relative improvement in accuracy from <em>GBLUP</em> to <em>MPM-Matérn</em> seemed more predictable based on the relative improvement in model fit. Specifically, a decrease in Bayesian information criterion (BIC) seemed to discriminate cases where an improvement in accuracy could be achieved by <em>MPM-Matérn</em>, especially in cross-population calibration where a correct depiction of population heterogeneity seemed more critical (Figure S4).</p></div></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><div id="sec-24" class="subsection"><h3>Conclusions</h3><p id="p-51">The present study assessed different mean structures to represent population differentiation and evaluated various procedures to accommodate population heterogeneity in diverse samples, with an application in switchgrass. We considered different approaches to reflect population structure, <em>i.e.</em>, characterizing it implicitly by random marker effects, using only an intercept as fixed effect (<em>Intercept</em>), or characterize it explicitly, by continuous differentiation (<em>PCA</em>) or discrete effects at the level of panels (<em>Panel</em>) or putative population groups (<em>Group</em>). Furthermore, we employed three typical strategies for dealing with marker-by-population interactions, <em>i.e.</em>, ignoring (<em>GBLUP</em>), reducing (<em>GBLUP-Target</em>), or modeling (<em>MPM</em>) the source of heterogeneity in the data.</p><p id="p-52">Our assessment of mean structures points to a simple fixed-effect specification being preferable in genomic prediction analyses, since accuracies from <em>Intercept</em> were relatively high across populations and traits, and relatively stable across validation schemes (whole-sample calibration or cross-population calibration). These conclusions are consistent with those of <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Phocas and Laloë (2004)</a>, who showed larger prediction errors in cattle when including putative genetic groups as fixed effects (comparable to <em>Group</em> and <em>Panel</em>). Notably, deteriorations of prediction accuracies from <em>PCA</em> and <em>Group</em> were especially large in cross-population calibration in which entire populations were excluded from the CS. Moreover, these were often noted for L4X-NE which was under-connected to other populations in the sample (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Decreases in accuracy with <em>PCA</em> suggest that linear fixed effects capturing population structure may fail to properly extrapolate on unobserved populations whose genetics may differ markedly from other populations in the sample (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1b</a>). However, it is worth noting that the switchgrass sample under study was highly structured. Samples in other species, <em>e.g.</em>, in maize or rice, may not display such discrete population differentiation and therefore may not suffer as much from fixed effects at population level (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Guo <em>et al.</em> 2014</a>).</p><p id="p-53">In whole-sample calibration, <em>GBLUP</em> often seemed robust to population heterogeneity, regarding prediction accuracy (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3a</a>). This robustness was certainly due to the ability of GBLUP models to combine information from individuals according to the specified relationship matrix, by transferring information preferentially from the more related individuals (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Searle <em>et al.</em> 2009</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Habier <em>et al.</em> 2013</a>). Furthermore, GBLUP models were probably all the more robust as marker density was high, such that genomic relationships were accurately estimated (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Casella and Berger 2002</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Endelman and Jannink 2012</a>). However, some decreases in prediction accuracy compared to <em>GBLUP-Target</em> suggest that robustness of <em>GBLUP</em> may have been affected by other factors. Such factors may be related to relationships within the sample, <em>i.e.</em>, under-connectedness of some populations with others (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>), or differences in accuracy of the prediction model across populations, as reflected by <em>GBLUP-Target</em> being more accurate in certain populations (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3a</a>).</p><p id="p-54">In whole-sample calibration, prediction was mostly determined by individuals in the CS belonging to the same population as the TS. Consistently, <em>MPM</em> procedures, which shrink relationships involving individuals from distantly-related populations, did not dramatically affect prediction accuracy (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3a</a>). However, in cross-population calibration, decreasing the contribution of individuals from distantly-related populations must have been more pertinent, so that there were more opportunities for improvement of prediction accuracy by <em>MPM</em> procedures. In this context, <em>MPM-Matérn</em> proved more useful than <em>MPM-Mixture</em>, especially with HD for which <em>MPM-Matérn</em> resulted in a dramatic improvement of fit (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3b</a>). Importantly, this relative superiority may be due to the fact that <em>MPM-Matérn</em> extrapolated correlations between populations, through leading PCs, while <em>MPM-Mixture</em> merely interpolated such correlations, by estimating a common coefficient of correlation across populations. This lack of flexibility, and the subsequent inability to extrapolate to unobserved populations, must have resulted in high sensitivity of <em>MPM-Mixture</em> to the composition of the CS, making it particularly inappropriate in a cross-population context.</p><p id="p-55">Marker-by-population interactions captured by <em>MPM-Mixture</em> and <em>MPM-Matérn</em> were presumably not confounded by marker-by-environment interactions, since interactions between panel and markers were not significant (<em>P</em> &gt; 0.25 in a model, similar to <em>MPM-Mixture</em>, which depicted correlation in marker effects between BP, assayed in WI, and AP, assayed in NY; Figure S5). Therefore, models analyzed in this study would reflect actual differences in genetic bases across populations. Moreover, for every trait, genomic variability (variance of marker effects) would be similar across panels. Indeed, the non-significant improvement in fit from an extension of a GBLUP model where genomic variance can vary by panel (<em>P</em> ≥ 0.18), suggested limited differences in variance of marker effects by panel (Figure S6). This result further implied that estimation of genotype effects and half-sib family effects (in AP and BP, respectively) and scaling of half-sib family effects (multiplied by two, so they corresponded to breeding values) were effective to ensure concordance in genomic variability across panels.</p><p id="p-56">Here, we modeled interactions between markers and population structure through products of relationships at markers, which were linear (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-125.gif"/></noscript></span></span>), and relationships about population structure, which were linear in <em>MPM-Mixture</em> and nonlinear in <em>MPM-Matérn</em> (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-126.gif"/></noscript></span></span>; Appendix 3). Using kernel functions to estimate relationships dispensed us from fitting effects of many variables, by estimating instead <em>n</em> breeding values directly from relationships. A similar strategy was adopted quite recently by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jarquín <em>et al.</em> (2014)</a>, who modeled genotype-by-environment interactions for genomic prediction, through products of linear kernels at markers and linear kernels at environmental covariates. As noted by these authors, such decomposition with respect to interactions had been introduced in quantitative genetics much earlier, by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kempthorne (1954)</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cockerham (1954)</a> for depicting epistatic effects, based on expected relationships under an infinitesimal model. Importantly, relationships about population structure not only allowed us to efficiently specify a genome-wide marker-by-population interaction model, but they also enabled the use of nonlinear kernels at the population level (with <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-127.gif"/></noscript></span></span> produced by nonlinear functions in <em>MPM-Matérn</em>). Matérn kernels, introduced to genomic prediction research by <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Ober <em>et al.</em> (2011)</a>, were used here to estimate covariance among individuals in a flexible yet parsimonious way (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), while still using simple linear kernels for depicting within-population variability (by <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-128.gif"/></noscript></span></span>). Our results exemplify the potential usefulness of parsimonious multi-population models, which are all the more interesting that they can be applied on samples comprising many populations. In contrast, typical multi-trait models would be computationally intractable or statistically inefficient here, since those would rely on one parameter for each population pair to model correlations among populations in <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-129.gif"/></noscript></span></span> (<em>e.g.</em>, 21 parameters for <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-130.gif"/></noscript></span></span> population clusters). As a matter of fact, multivariate genomic BLUP models fitted by ASREML-R to estimate such correlations among putative population groups (WS4U-C2, Liberty-C2, U4X-N, U8X-W+U8X-S, U8X-E, L4X-NE and L4X-S) failed to converge.</p></div><div id="sec-25" class="subsection"><h3>Improvement of procedures</h3><p id="p-57">Our results suggest that a very high increase in quality of fit, as was observed for HD with <em>MPM-Matérn</em>, may allow for an increase in accuracy, especially in a cross-population context. In the analysis of <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Heslot and Jannink (2015)</a> across various multi-population contexts, there seemed to be a positive relationship between differences in quality of fit, as measured by the Akaike information criterion (AIC), and differences in prediction accuracy. Although this relationship was quite loose, it could be noted that for very high increases in AIC (<span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-131.gif"/></noscript></span></span>), gains in accuracy were null to high, similarly to the situation of <em>MPM-Matérn</em> with HD, for which increases in AIC varied from 28.68 to 42.88, across CV replicates in whole-sample calibration, and from 24.57 to 36.25 in cross-population calibration. Therefore, stringent thresholds on AIC increases could probably be used in <em>MPM</em> to avoid relative decreases in accuracy. In this study, one criterion more conservative than the AIC, the BIC, could discriminate cases where prediction accuracy was improved by <em>MPM-Matérn</em>, compared to <em>GBLUP</em> (Figure S4). Therefore, a possible improvement of <em>MPM</em> procedures could simply come from model selection as an integral part of the fitting process, based for example on the BIC. The BIC differences relative to <em>GBLUP</em> were almost always negative for PH and St in <em>MPM</em>. For these two traits, differences in prediction accuracy from <em>GBLUP</em> to <em>MPM</em> were quite inconsistent, especially with <em>MPM-Matérn</em>, so model selection could probably have made <em>MPM</em> procedures more robust. However, such conclusions are based on a restricted set of populations and genetic architectures. So future studies on other datasets would certainly be necessary to test this <em>post hoc</em> hypothesis and determine whether criteria such as the BIC can indicate cases where <em>MPM-Matérn</em> should be used instead of <em>GBLUP</em>.</p><p id="p-58">Another way of potentially improving <em>MPM</em> procedures would be to use other types of kernels than those used here. For example, one may use linear kernels based on population-level covariates (<em>e.g.</em>, PCs) in place of <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-132.gif"/></noscript></span></span> in <em>MPM-Mixture</em>, hence taking an approach similar to that of <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Jarquín <em>et al.</em> (2014)</a>. Besides, modeling resemblance among population clusters in <em>MPM-Mixture</em>, by <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-133.gif"/></noscript></span></span> in place of <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-134.gif"/></noscript></span></span> (where <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-135.gif"/></noscript></span></span> captures similarity based on metrics at the population level), could be useful to increase quality of fit, and possibly prediction accuracy. Finally, an interesting way of extending the <em>MPM</em> procedures described here would be to incorporate more information at the population level. Here in <em>MPM</em>, population homogeneity was captured through admixture coefficients (<em>MPM-Mixture</em>) or differences in PC coordinates (<em>MPM-Matérn</em>), the latter reflecting differences in allele frequencies (Appendix 3). However, marker-by-population interactions may also be due to differences in LD patterns (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Wientjes <em>et al.</em> 2016</a>). Therefore metrics depicting such differences could be particularly appropriate for capturing population heterogeneity. Further research would be necessary to determine the type of metrics to use for reflecting differences in LD patterns, and the appropriate way to parsimoniously combine the different types of information regarding population differentiation in <em>MPM</em>. Interestingly, geographical distance may succinctly depict population differentiation, due to differences in allele frequencies and/or differences in LD patterns. Fitting population-level correlations in <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-136.gif"/></noscript></span></span> as a function of distance of origin would then be particularly useful in species under strong geographical structure, which include switchgrass (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Grabowski <em>et al.</em> 2014</a>), but also human (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Coop <em>et al.</em> 2009</a>), as was clearly shown in samples from Europe (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Novembre <em>et al.</em> 2008</a>), Africa (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bryc <em>et al.</em> 2010</a>) and Latin America (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ruiz-Linares <em>et al.</em> 2014</a>). Models such as <em>MPM-Matérn</em>, which are parsimonious yet flexible in the shape of the fitted correlation function (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>), are promising in various applications on diverse samples, in prediction studies, but also in inferential studies aiming at characterizing the basis for population differentiation.</p><p id="p-59">In this study, marker data were based on exome capture sequencing, which targets a selected subset of exons for sequencing and subsequent SNP calling (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hirsch <em>et al.</em> 2014</a>, <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2014</a>). The potential lack of representation of causal variants by our assay may have resulted in loss of prediction accuracy. While total lack of representation of some genomic regions imposes a limit on prediction accuracy achievable by our procedures, the relative overrepresentation of some genomic regions could be, to some extent, alleviated by genomic relationships which account for correlation among markers and differential degrees of tagging of loci in the marker data (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Speed <em>et al.</em> 2012</a>, <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Ramstein <em>et al.</em> 2016</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wang <em>et al.</em> 2017</a>).</p><p id="p-60">Another limitation in our study is the assumed homogeneity of genetic and residual variances across populations. Here we focused on parsimonious models estimating genetic correlations (not covariances) between populations. Extending <em>MPM</em> models to capture variance heterogeneity across populations and/or environments would certainly deserve further investigation. Such models ought to fit functions of variance over genetic and/or environmental variables, similarly to <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ou <em>et al.</em> (2015)</a> who reported improvements in fit and marginal gains in prediction accuracy in swine, by modeling residual variance over sexes and slaughter dates. Indeed, a decisive advantage of models like <em>MPM-Matérn</em> (for correlations) and those of <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Ou <em>et al.</em> (2015)</a> (for variances) is their ability to extrapolate population covariances to genotypes from unobserved population backgrounds.</p></div><div id="sec-26" class="subsection"><h3>Applications and prospects</h3><p id="p-61">Based on our case study, we would recommend using <em>MPM</em> whenever a strong improvement in model fit is achieved. Otherwise <em>GBLUP</em> would be the method of choice, since it is often robust enough to perform at least as well as <em>GBLUP-Target</em>. However, fitting a GBLUP model to a CS restricted to the target population may be preferred when making predictions on “outlier populations” such as L4X-NE, which are under-connected to other populations and are characterized by relatively high prediction accuracy in a single-population context. Nevertheless, more empirical studies on population heterogeneity should follow to support the conclusions from our specific application. Such studies could apply to various contexts: in particular, predictions on diverse samples and dynamic breeding programs. The former includes analyses similar to our case study as well as analyses on more complex data, such as historical datasets, in which not only population heterogeneity but also genotype-by-environment interactions must be taken into account (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dawson <em>et al.</em> 2013</a>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rutkoski <em>et al.</em> 2015</a>). The latter involves selection across multiple breeding generations, which might not necessarily suffer from strong population heterogeneity (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sallam <em>et al.</em> 2015</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Auinger <em>et al.</em> 2016</a>) but could nonetheless benefit from robust multi-population models for potential increase in persistency of accuracy over generations (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Habier <em>et al.</em> 2007</a>). In the context of diverse samples or dynamic breeding, simulation studies could also be useful for assessing the suitability of procedures to accommodate population heterogeneity. Differences in allele frequencies and differential LD patterns could be simulated by various genealogies, as was done for example by <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">de Roos <em>et al.</em> (2009)</a>. Additionally, dependency of marker effects on allele frequencies could be simulated through underlying non-additive genetic effects, as well as allele fixation in specific populations. Indeed, marginal additive marker effects, as captured by linear models such as standard GBLUP, depend on allele frequencies at the loci with which they interact (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hill <em>et al.</em> 2008</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mäki-Tanila and Hill 2014</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hill and Mäki-Tanila 2015</a>). Therefore, dominance and epistatic effects could be simulated to generate dependency of marker effects on allele frequencies, which may then be captured by methods such as <em>MPM-Matérn</em> (Appendix 3). Though investigations based on simulations would be complex and, to some extent, arbitrary by their choice of genealogies and genetic architectures, they would provide useful frameworks for assessing procedures, such as those presented in this study, in contexts of population heterogeneity.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">We are grateful to Jeremy Schmutz of the Department of Energy Joint Genome Institute and Hudson Alpha for his work on the switchgrass genome. We are also grateful to Nick Baker and Joseph Halinar, USDA-ARS, Madison, WI for assistance with field operations and data collection. This research was funded in part by the following agencies and organizations: the U.S. Department of Energy Great Lakes Bioenergy Research Center, DOE Office of Science BER DE-FC02-07ER64494 (laboratory operations, genotyping, and bioinformatics), the U.S. Department of Energy Joint Genome Institute supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231 (sequencing), Agriculture and Food Research Initiative Competitive Grant No. 2011-68005-30411 from the USDA National Institute of Food and Agriculture (CenUSA; field operations and phenotypic measurements), USDA-ARS Congressionally allocated funds (field operations, technical support, and logistics), and the University of Wisconsin Agricultural Research Stations (field operations). Mention of commercial products and organizations in this manuscript is solely to provide specific information. The USDA is an equal opportunity provider and employer. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section ack" id="ack-2"><h2 class="">Conflict of interest</h2><p id="p-86">The authors declare that they have no conflict of interest.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section" id="sec-27"><h3>Inference of recent relationships by the graphical LASSO</h3><p id="p-62">Let <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-137.gif"/></noscript></span></span> be a regularized form of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-138.gif"/></noscript></span></span>, with G the genomic relationship matrix as defined by <a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">VanRaden (2008)</a>, and <strong>P</strong> the matrix of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-139.gif"/></noscript></span></span> PCs. We applied the graphical LASSO to <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-140.gif"/></noscript></span></span> to infer a sparse matrix <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-141.gif.backup.1556799206.0257"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-141.gif.backup.1556799206.0257"/></noscript></span></span>, which yielded a graph of relationships among individuals. Indeed, a zero <em>ij</em>-element in <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-142.gif.backup.1556799206.0554"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-142.gif.backup.1556799206.0554"/></noscript></span></span> indicates that individuals <em>i</em> and <em>j</em> are unrelated conditionally on all other individuals, which corresponds to no edge between individuals <em>i</em> and <em>j</em> in the underlying graph of recent relationships.</p><p id="p-63">For a given sample covariance matrix Σ, the graphical LASSO infers a sparse precision matrix <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-143.gif.backup.1556799206.0813"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-143.gif.backup.1556799206.0813"/></noscript></span></span> by maximizing the Gaussian likelihood of the data (as represented by Σ), penalized by a <em>L<sub>1</sub></em>-norm penalty <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-144.gif.backup.1556799206.1164"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-144.gif.backup.1556799206.1164"/></noscript></span></span>, where <em>λ</em> is the regularization parameter and <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-145.gif.backup.1556799206.1486"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-145.gif.backup.1556799206.1486"/></noscript></span></span> is the sum of absolute values in <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-146.gif.backup.1556799206.176"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-146.gif.backup.1556799206.176"/></noscript></span></span> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Friedman <em>et al.</em> 2008</a>).</p><p id="p-64">In this study, regularization of <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-147.gif"/></noscript></span></span> was performed as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-65">Standardizing <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-148.gif.backup.1556799206.2094"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-148.gif.backup.1556799206.2094"/></noscript></span></span> to obtain the corresponding correlation matrix <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-149.gif.backup.1556799206.2391"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-149.gif.backup.1556799206.2391"/></noscript></span></span>: <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-150.gif.backup.1556799206.2634"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-150.gif.backup.1556799206.2634"/></noscript></span></span></p></li><li id="list-item-2"><p id="p-66">Applying the graphical LASSO algorithm to <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-151.gif.backup.1556799206.2839"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-151.gif.backup.1556799206.2839"/></noscript></span></span>, to obtain the regularized correlation matrix <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-152.gif.backup.1556799206.3126"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-152.gif.backup.1556799206.3126"/></noscript></span></span></p></li><li id="list-item-3"><p id="p-67">Rescaling <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-153.gif.backup.1556799206.3412"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-153.gif.backup.1556799206.3412"/></noscript></span></span> to obtain <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-154.gif.backup.1556799206.3759"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-154.gif.backup.1556799206.3759"/></noscript></span></span></p></li></ol><p id="p-68">The graphical LASSO algorithm was run using the R package huge (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhao <em>et al.</em> 2012</a>). The regularization parameter <em>λ</em> was chosen to maximize the restricted likelihood for the following model:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-155.gif.backup.1556799206.4174"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-155.gif.backup.1556799206.4174"/></noscript></span></span>where <strong>y</strong> is the <em>n</em>-vector of genotype means at a given trait; <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-156.gif.backup.1556799206.4522"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-156.gif.backup.1556799206.4522"/></noscript></span></span>, depending on <em>λ</em>, consists of regularized relationships; <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-157.gif.backup.1556799206.4796"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-157.gif.backup.1556799206.4796"/></noscript></span></span> is the variance of breeding values (here not equal to the variance of marker effects due to scaling and regularization on <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-158.gif.backup.1556799206.515"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-158.gif.backup.1556799206.515"/></noscript></span></span>); <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-159.gif.backup.1556799206.5468"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-159.gif.backup.1556799206.5468"/></noscript></span></span> is the <em>n</em>-vector of intercept values; <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-160.gif.backup.1556799206.5724"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-160.gif.backup.1556799206.5724"/></noscript></span></span> is the covariance matrix for errors considered independent and identically distributed. Here we chose the value of <em>λ</em> for which the corresponding matrix <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-161.gif.backup.1556799206.6032"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-161.gif.backup.1556799206.6032"/></noscript></span></span> resulted in the highest restricted likelihood for the aforementioned model fitted to the whole sample. Possible values of <em>λ</em> were the <em>q</em>-quantiles of absolute values in <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-162.gif.backup.1556799206.64"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-162.gif.backup.1556799206.64"/></noscript></span></span>, with <em>q</em> varying from 0.05 to 1 by step of 0.05. The restricted likelihood as a function of <em>λ</em> depended on <strong>y</strong>, so optimization of <em>λ</em> was performed for each trait separately.</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix 2</h2><div class="section" id="sec-28"><h3>Multi-population GBLUP models for heterogeneous calibration sets</h3><p id="p-69">In this section, <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-163.gif.backup.1556799206.6813"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-163.gif.backup.1556799206.6813"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-164.gif.backup.1556799206.716"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-164.gif.backup.1556799206.716"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-165.gif.backup.1556799206.7404"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-165.gif.backup.1556799206.7404"/></noscript></span></span> refer to the vector of ones, identity matrix, and matrix of ones, respectively, of dimensions t, <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-166.gif.backup.1556799206.7725"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-166.gif.backup.1556799206.7725"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-167.gif.backup.1556799206.798"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-167.gif.backup.1556799206.798"/></noscript></span></span>(where <em>t</em> is specified).</p><p id="p-70">Consider the following model for population-specific marker effects with respect to <em>K</em> populations and <em>n</em> genotypes:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-168.gif.backup.1556799206.842"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-168.gif.backup.1556799206.842"/></noscript></span></span>where ⊗ indicates the Kronecker product; <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-169.gif.backup.1556799206.8677"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-169.gif.backup.1556799206.8677"/></noscript></span></span> is the <em>Kn</em> × <em>p</em> design matrix for the <em>p</em>-vector α of fixed effects; <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-170.gif.backup.1556799206.8913"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-170.gif.backup.1556799206.8913"/></noscript></span></span> is the <em>Kn</em> × <em>Km</em> marker-data matrix for the <em>Km</em>-vector <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-171.gif.backup.1556799206.9357"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-171.gif.backup.1556799206.9357"/></noscript></span></span> of marker effects at each of the <em>K</em> populations, with variance <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-172.gif.backup.1556799206.9573"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-172.gif.backup.1556799206.9573"/></noscript></span></span>. The matrix <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-173.gif.backup.1556799206.982"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-173.gif.backup.1556799206.982"/></noscript></span></span> reflects covariances in marker effects between populations. The <em>Kn</em>-vector <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-174.gif.backup.1556799207.0173"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-174.gif.backup.1556799207.0173"/></noscript></span></span>, containing the phenotypic values for the <em>n</em> genotypes at each of the <em>K</em> populations, is hypothetical (and ill-defined from a practical standpoint), since genotypes typically do not belong to more than one population. The <em>Kn</em>-vector of residuals <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-175.gif.backup.1556799207.0739"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-175.gif.backup.1556799207.0739"/></noscript></span></span>, with unspecified variance <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-176.gif.backup.1556799207.0947"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-176.gif.backup.1556799207.0947"/></noscript></span></span>, is assumed to be uncorrelated to marker effects <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-177.gif.backup.1556799207.1155"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-177.gif.backup.1556799207.1155"/></noscript></span></span>.</p><p id="p-71">Let <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-178.gif.backup.1556799207.1411"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-178.gif.backup.1556799207.1411"/></noscript></span></span> be the <em>Kn</em>-vector of additive genetic effects at each of the <em>K</em> populations, as a linear combination of a normally-distributed vector (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-179.gif.backup.1556799207.1608"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-179.gif.backup.1556799207.1608"/></noscript></span></span>), <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-180.gif.backup.1556799207.1914"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-180.gif.backup.1556799207.1914"/></noscript></span></span> follows a normal distribution with expectation and variance as follows (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Lehermeier <em>et al.</em>, 2015</a>):<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-181.gif.backup.1556799207.2155"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-181.gif.backup.1556799207.2155"/></noscript></span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-182.gif.backup.1556799207.2649"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-182.gif.backup.1556799207.2649"/></noscript></span></span>So a multi-population model for breeding values that is equivalent to the model described above, by identical mean and variance structures, is as follows:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-183.gif.backup.1556799207.2896"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-183.gif.backup.1556799207.2896"/></noscript></span></span>Now assume that <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-184.gif.backup.1556799207.3106"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-184.gif.backup.1556799207.3106"/></noscript></span></span>, and each population corresponds to the specific genetic background of each individual separately. By considering only observations at every individual’s specific genetic background, the above model reduces to:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-185.gif.backup.1556799207.341"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-185.gif.backup.1556799207.341"/></noscript></span></span> <span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-186.gif.backup.1556799207.3681"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-186.gif.backup.1556799207.3681"/></noscript></span></span>where ○ is the Hadamard product; y is the typical <em>n</em>-vector of observed phenotypic values; u and e are the corresponding additive genetic effects and residuals, respectively. Individual-specific marker effects are therefore accounted for by multiplying each element of the relationship matrix <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-187.gif.backup.1556799207.3933"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-187.gif.backup.1556799207.3933"/></noscript></span></span> by the corresponding element of <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-188.gif.backup.1556799207.415"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-188.gif.backup.1556799207.415"/></noscript></span></span>, thereby reflecting correlations in marker effects among individuals’ genetic backgrounds.</p><p id="p-72">In general, we propose to infer <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-189.gif.backup.1556799207.4438"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-189.gif.backup.1556799207.4438"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-190.gif"/></noscript></span></span>, where ϕ is some function of the <em>m</em>-vectors of marker variables <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-191.gif.backup.1556799207.4773"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-191.gif.backup.1556799207.4773"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-192.gif.backup.1556799207.5144"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-192.gif.backup.1556799207.5144"/></noscript></span></span>, for any pair of individuals <em>i</em> and <em>j</em>, and <em>κ</em> is a valid kernel function guaranteeing that <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-193.gif.backup.1556799207.5345"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-193.gif.backup.1556799207.5345"/></noscript></span></span> be positive semi-definite. In <em>MPM-Mixture</em>, <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-194.gif.backup.1556799207.5817"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-194.gif.backup.1556799207.5817"/></noscript></span></span> (<em>K</em>-vector of admixture coefficients for <em>i</em>) for any individual <em>i</em>, and the kernel function is <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-195.gif.backup.1556799207.603"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-195.gif.backup.1556799207.603"/></noscript></span></span>, so that <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-196.gif.backup.1556799207.6253"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-196.gif.backup.1556799207.6253"/></noscript></span></span> is a “mixture” between a matrix of correlations restricted to population clusters and a matrix allowing full exchange of information across clusters, as in a standard GBLUP model. More generally, one could define the kernel function as <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-197.gif.backup.1556799207.6438"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-197.gif.backup.1556799207.6438"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-198.gif.backup.1556799207.6677"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-198.gif.backup.1556799207.6677"/></noscript></span></span> is a <em>K</em> × <em>K</em> matrix depicting relationships among clusters. Here, we simply set <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-199.gif.backup.1556799207.6849"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-199.gif.backup.1556799207.6849"/></noscript></span></span> and adjusted the kernel function (by restricted maximum likelihood) for <em>ρ</em> only.</p><p id="p-73">In <em>MPM-Matérn</em>, <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-200.gif.backup.1556799207.7127"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-200.gif.backup.1556799207.7127"/></noscript></span></span> (<em>d</em>-vector of PC coordinates for <em>i</em>) for any individual <em>i</em>, and the kernel function is a Matérn function <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-201.gif.backup.1556799207.7483"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-201.gif.backup.1556799207.7483"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-202.gif.backup.1556799207.7718"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-202.gif.backup.1556799207.7718"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-203.gif.backup.1556799207.7977"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-203.gif.backup.1556799207.7977"/></noscript></span></span> is the Euclidean norm. Notably, it can be shown that <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-204.gif.backup.1556799207.8188"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-204.gif.backup.1556799207.8188"/></noscript></span></span> is proportional to <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-205.gif.backup.1556799207.8436"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-205.gif.backup.1556799207.8436"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-206.gif.backup.1556799207.8653"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-206.gif.backup.1556799207.8653"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-207.gif.backup.1556799207.8862"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-207.gif.backup.1556799207.8862"/></noscript></span></span>) is the <em>m</em>-vector of individual-specific allele frequencies for individuals <em>i</em> (<em>j</em>), defined by projection of matrix <strong>X</strong> onto the column space of <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-208.gif.backup.1556799207.9051"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-208.gif.backup.1556799207.9051"/></noscript></span></span> (Appendix 3). So <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-209.gif.backup.1556799207.9245"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-209.gif.backup.1556799207.9245"/></noscript></span></span>, which reflects differentiation with respect to coordinates at the leading PCs of <strong>X</strong>, also reflects differentiation with respect to individual-specific allele frequencies, with an underlying population structure represented by the same PCs. The allele frequencies <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-210.gif.backup.1556799207.9483"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-210.gif.backup.1556799207.9483"/></noscript></span></span> have been introduced by <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Conomos <em>et al.</em> (2016)</a>, in a study where they also recommended using principal components from a subset of unrelated individuals in <strong>X</strong>. Here, we simply applied PCA on the whole matrix <strong>X</strong>.</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix 3</h2><div class="section" id="sec-29"><h3>Relationship between Euclidean distance based on principal components and Euclidean distance based on individual-specific allele frequencies</h3><p id="p-74">In this section, <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-211.gif.backup.1556799207.975"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-211.gif.backup.1556799207.975"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-212.gif.backup.1556799208.0046"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-212.gif.backup.1556799208.0046"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-213.gif.backup.1556799208.0335"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-213.gif.backup.1556799208.0335"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-214.gif.backup.1556799208.0573"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-214.gif.backup.1556799208.0573"/></noscript></span></span> refer to the vector of ones, identity matrix, matrix of ones and matrix of zeros, respectively, of dimensions t, <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-215.gif.backup.1556799208.0888"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-215.gif.backup.1556799208.0888"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-216.gif.backup.1556799208.122"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-216.gif.backup.1556799208.122"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-217.gif.backup.1556799208.1681"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-217.gif.backup.1556799208.1681"/></noscript></span></span>(where <em>s</em> and <em>t</em> are specified).</p><p id="p-75">We will consider the case where the PC matrix <strong>P</strong> consists of the first <em>d</em> PCs of <strong>X</strong>, and individual-specific allele frequencies are defined as (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Conomos <em>et al.</em> 2016</a>):<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-218.gif.backup.1556799208.2134"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-218.gif.backup.1556799208.2134"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-219.gif.backup.1556799208.2571"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-219.gif.backup.1556799208.2571"/></noscript></span></span> represents population structure through an intercept and the effects of the first <em>d</em> PCs of <strong>X</strong>. Vector <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-220.gif.backup.1556799208.2922"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-220.gif.backup.1556799208.2922"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-221.gif.backup.1556799208.3265"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-221.gif.backup.1556799208.3265"/></noscript></span></span>) then consists of individual-specific allele frequencies (with respect to <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-222.gif.backup.1556799208.3589"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-222.gif.backup.1556799208.3589"/></noscript></span></span>) for individual <em>i</em> (<em>j</em>), such that:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-223.gif.backup.1556799208.3848"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-223.gif.backup.1556799208.3848"/></noscript></span></span>and similarly for <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-224.gif.backup.1556799208.4236"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-224.gif.backup.1556799208.4236"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-225.gif.backup.1556799208.4743"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-225.gif.backup.1556799208.4743"/></noscript></span></span> refers to the <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-226.gif.backup.1556799208.5051"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-226.gif.backup.1556799208.5051"/></noscript></span></span>-vector of population-structure variables from <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-227.gif.backup.1556799208.5415"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-227.gif.backup.1556799208.5415"/></noscript></span></span> for individual <em>i</em>).</p><p id="p-76">We will show that <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-228.gif.backup.1556799208.5696"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-228.gif.backup.1556799208.5696"/></noscript></span></span> for any pair (<em>i</em>, <em>j</em>), <em>i.e.</em>, Euclidean distances based on <em>d</em> PCs are equivalent, by proportionality, to those based on <em>m</em> individual-specific allele frequencies, with such frequencies as defined above.<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-229.gif.backup.1556799208.607"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-229.gif.backup.1556799208.607"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-230.gif.backup.1556799208.6513"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-230.gif.backup.1556799208.6513"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-231.gif.backup.1556799208.6734"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-231.gif.backup.1556799208.6734"/></noscript></span></span>.</p><p id="p-77">Below, we will specify <strong>M</strong> more explicitly, to subsequently show that <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-232.gif.backup.1556799208.7088"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-232.gif.backup.1556799208.7088"/></noscript></span></span>.</p><p id="p-78">Let <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-233.gif.backup.1556799208.7396"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-233.gif.backup.1556799208.7396"/></noscript></span></span> be the matrix of marker variables centered around their respective overall mean. Assuming <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-234.gif.backup.1556799208.7785"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-234.gif.backup.1556799208.7785"/></noscript></span></span>, by eigenvalue decomposition <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-235.gif.backup.1556799208.8088"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-235.gif.backup.1556799208.8088"/></noscript></span></span>, with <strong>U</strong> the <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-236.gif.backup.1556799208.8512"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-236.gif.backup.1556799208.8512"/></noscript></span></span> matrix of eigenvectors and Λ the <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-237.gif.backup.1556799208.8816"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-237.gif.backup.1556799208.8816"/></noscript></span></span> diagonal matrix of eigenvalues of <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-238.gif.backup.1556799208.9332"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-238.gif.backup.1556799208.9332"/></noscript></span></span>; and <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-239.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-240.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-241.gif"/></noscript></span></span> matrix of leading eigenvectors and <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-242.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-243.gif"/></noscript></span></span> diagonal matrix of corresponding singular values, assumed strictly positive.</p><p id="p-79">Because <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-244.gif"/></noscript></span></span> consist of left-eigenvectors of a column-centered matrix (associated with strictly positive eigenvalues), <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-245.gif"/></noscript></span></span> so <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-246.gif"/></noscript></span></span>.</p><p id="p-80">Besides, <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-247.gif"/></noscript></span></span>.</p><p id="p-81">So:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-248.gif"/></noscript></span></span>Moreover <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-249.gif"/></noscript></span></span> because <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-250.gif"/></noscript></span></span>.</p><p id="p-82">So<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-251.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-252.gif"/></noscript></span></span></p><p id="p-83">Finally, <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-253.gif"/></noscript></span></span></p><p id="p-84">with:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-254.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-255.gif"/></noscript></span></span>Therefore, for any pair of individuals (<em>i</em>, <em>j</em>):<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-256.gif"/></noscript></span></span>So:</p><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/789/embed/mml-math-257.gif"/></noscript></span></span></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-87">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7464863">https://doi.org/10.25387/g3.7464863</a>.</p></li><li class="fn" id="fn-7"><p id="p-88"><em>Communicating editor: G. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-13"><span class="received-label">Received </span>December 13, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-10"><span class="accepted-label">Accepted </span>January 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Ramstein, Casler</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramowitz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stegun</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Pocketbook of mathematical functions</span>, <span class="cit-publ-name">Verlag Harri Deutsch</span>, <span class="cit-publ-loc">Thun</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.2" data-doi="10.1186/1471-2105-12-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Enhancements to the ADMIXTURE algorithm for individual ancestry estimation.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DAlexander%26rft.auinit1%253DD.%2BH.%26rft.volume%253D12%26rft.spage%253D246%26rft.epage%253D246%26rft.atitle%253DEnhancements%2Bto%2Bthe%2BADMIXTURE%2Balgorithm%2Bfor%2Bindividual%2Bancestry%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-246%26rft_id%253Dinfo%253Apmid%252F21682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21682921&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293266800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.3" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.4" data-doi="10.1007/s00122-016-2756-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model training across multiple breeding cycles significantly improves genomic prediction accuracy in rye (Secale cereale L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">2043</span>–<span class="cit-lpage">2053</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2756-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D2043%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2756-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2756-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.5" data-doi="10.1073/pnas.0909559107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oksenberg</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide patterns of population structure and admixture in West Africans and African Americans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">786</span>–<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0909559107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909559107%26rft_id%253Dinfo%253Apmid%252F20080753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml-R reference manual. The State of Queensland</span>, <span class="cit-publ-name">Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.7" data-doi="10.1186/s12711-014-0067-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carillier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larroque</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparison of joint <em>vs.</em> purebred genomic evaluation in the French multi-breed dairy goat population.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-014-0067-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-014-0067-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-014-0067-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Statistical inference</span>, <span class="cit-publ-name">Duxbury Pacific Grove</span>, <span class="cit-publ-loc">CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.9" data-doi="10.1007/978-1-4471-2903-5_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Switchgrass Breeding, Genetics, and Genomics</span>, pp. <span class="cit-fpage">29</span>–<span class="cit-lpage">53</span> in <span class="cit-source">Switchgrass: A Valuable Biomass Crop for Energy</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Monti</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-publ-name">Springer London</span>, <span class="cit-publ-loc">London</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4471-2903-5_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSwitchgrass%253A%2BA%2BValuable%2BBiomass%2BCrop%2Bfor%2BEnergy%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4471-2903-5_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4471-2903-5_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.10" data-doi="10.2135/cropsci2013.04.0239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selection for Biomass Yield in Upland, Lowland, and Hybrid Switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">626</span>–<span class="cit-lpage">636</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.04.0239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D626%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.04.0239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.04.0239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.11" data-doi="10.2135/cropsci2005.04-0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">K. P.Vogel</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Registration of WS4U and WS8U Switchgrass Germplasms.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">998</span>–<span class="cit-lpage">999</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.04-0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.D.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D998%26rft.epage%253D999%26rft.atitle%253DRegistration%2Bof%2BWS4U%2Band%2BWS8U%2BSwitchgrass%2BGermplasms%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.04-0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.04-0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">An Extension of the Concept of Partitioning Hereditary Variance for Analysis of Covariances among Relatives When Epistasis Is Present.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D859%26rft.epage%253D882%26rft.atitle%253DAN%2BEXTENSION%2BOF%2BTHE%2BCONCEPT%2BOF%2BPARTITIONING%2BHEREDITARY%2BVARIANCE%2BFOR%2BANALYSIS%2BOF%2BCOVARIANCES%2BAMONG%2BRELATIVES%2BWHEN%2BEPISTASIS%2BIS%2BPRESENT%26rft_id%253Dinfo%253Apmid%252F17247525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIzOS82Lzg1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy83ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.13" data-doi="10.1016/j.ajhg.2015.11.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model-free Estimation of Recent Genetic Relatedness.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.022%26rft_id%253Dinfo%253Apmid%252F26748516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748516&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.14" data-doi="10.1371/journal.pgen.1000500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The role of geography in human adaptation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000500</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCoop%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000500%26rft.epage%253De1000500%26rft.atitle%253DThe%2Brole%2Bof%2Bgeography%2Bin%2Bhuman%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000500%26rft_id%253Dinfo%253Apmid%252F19503611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503611&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.15" data-doi="10.1186/1297-9686-44-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swan</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of pedigree and genomic predictions of carcass and novel meat quality traits in multi-breed sheep data assessed by cross-validation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-33</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D44%26rft.spage%253D33%26rft.epage%253D33%26rft.atitle%253DAccuracy%2Bof%2Bpedigree%2Band%2Bgenomic%2Bpredictions%2Bof%2Bcarcass%2Band%2Bnovel%2Bmeat%2Bquality%2Btraits%2Bin%2Bmulti-breed%2Bsheep%2Bdata%2Bassessed%2Bby%2Bcross-validation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-33%26rft_id%253Dinfo%253Apmid%252F23146144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23146144&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.16" data-doi="10.1016/j.fcr.2013.07.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The use of unbalanced historical data for genomic selection in an international wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2013.07.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D154%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2013.07.020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2013.07.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.17" data-doi="10.1007/s13253-015-0222-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veturi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Incorporating Genetic Heterogeneity in Whole-Genome Regressions Using Interactions.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">490</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13253-015-0222-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-015-0222-5%26rft_id%253Dinfo%253Apmid%252F26660276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-015-0222-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26660276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.18" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge Regression and Other Kernels for Genomic Selection with R Package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.19" data-doi="10.1534/g3.112.004259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1405%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004259%26rft_id%253Dinfo%253Apmid%252F23173092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23173092&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.20" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goswami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2011-5019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.21" data-doi="10.1111/tpj.13041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and population structure of northern switchgrass as revealed through exome capture sequencing.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D84%26rft.spage%253D800%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13041%26rft_id%253Dinfo%253Apmid%252F26426343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426343&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.22" data-doi="10.1111/tpj.12601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaillancourt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nucleotide polymorphism and copy number variant detection using exome capture and next-generation sequencing in the polyploid grass Panicum virgatum.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">993</span>–<span class="cit-lpage">1008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12601</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D79%26rft.spage%253D993%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12601%26rft_id%253Dinfo%253Apmid%252F24947485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24947485&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.23" data-doi="10.1111/rssb.12016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mincheva</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Large Covariance Estimation by Thresholding Principal Orthogonal Complements.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">75</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">603</span>–<span class="cit-lpage">680</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/rssb.12016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D75%26rft.spage%253D603%26rft_id%253Dinfo%253Adoi%252F10.1111%252Frssb.12016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/rssb.12016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.24" data-doi="10.1093/biostatistics/kxm045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sparse inverse covariance estimation with the graphical lasso.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/biostatistics/kxm045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxm045%26rft_id%253Dinfo%253Apmid%252F18079126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxm045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18079126&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256977000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.25" data-doi="10.1002/spe.4380211102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fruchterman</span>  <span class="cit-name-given-names">T. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reingold</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Graph drawing by force-directed placement.</span> <abbr class="cit-jnl-abbrev">Softw. Pract. Exper.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/spe.4380211102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftw.%2BPract.%2BExper.%26rft.volume%253D21%26rft.spage%253D1129%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fspe.4380211102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/spe.4380211102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.26" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive Genetic Variability and the Bayesian Alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.27" data-doi="10.1111/mec.12845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabowski</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population genomic variation reveals roles of history, adaptation and ploidy in switchgrass.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4059</span>–<span class="cit-lpage">4073</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4059%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.28" data-doi="10.1007/s00122-013-2255-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandhi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The impact of population structure on genomic prediction in stratified populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D749%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2255-x%26rft_id%253Dinfo%253Apmid%252F24452438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2255-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24452438&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.29" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.30" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.31" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.32" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.33" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">c</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672308009981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.34" data-doi="10.1186/s12711-015-0171-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An alternative covariance estimator to investigate genetic heterogeneity in populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0171-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0171-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0171-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.35" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.36" data-doi="10.1111/jbg.12140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäki-Tanila</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Expected influence of linkage disequilibrium on genetic variance caused by dominance and epistasis on quantitative traits.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D132%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.37" data-doi="10.1093/bfgp/elt051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reduced representation approaches to interrogate genome diversity in large repetitive plant genomes.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elt051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felt051%26rft_id%253Dinfo%253Apmid%252F24395692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elt051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24395692&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.38" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.39" data-doi="10.1534/g3.116.031443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2329</span>–<span class="cit-lpage">2341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.031443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.031443%26rft_id%253Dinfo%253Apmid%252F27247288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOC8yMzI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.40" data-doi="10.1186/1297-9686-44-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karoui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carabaño</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Díaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint genomic evaluation of French dairy cattle breeds using multiple-trait models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D44%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-39%26rft_id%253Dinfo%253Apmid%252F23216664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23216664&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.41" data-doi="10.1098/rspb.1954.0056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">The correlation between relatives in a random mating population.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft.volume%253D143%26rft.spage%253D102%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1954.0056%26rft_id%253Dinfo%253Apmid%252F13224653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1954.0056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13224653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.789.42"><div class="cit-metadata unstructured">Langholtz, M. H., B. J. Stokes, L. M. Eaton, C. C. Brandt, M. R. Davis <em>et al</em>., 2016 2016 Billion-ton report: Advancing domestic resources for a thriving bioeconomy, volume 1: Economic availability of feedstocks: Oak Ridge National Lab. (ORNL), Oak Ridge, TN (United States).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.43" data-doi="10.1534/genetics.115.177394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessment of Genetic Heterogeneity in Structured Plant Populations Using Multivariate Whole-Genome Regression Models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177394%26rft_id%253Dinfo%253Apmid%252F26122758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.44" data-doi="10.1534/g3.113.010165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serba</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic linkage mapping and transmission ratio distortion in a three-generation four-founder population of Panicum virgatum (L.).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.010165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.010165%26rft_id%253Dinfo%253Apmid%252F24637352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNS85MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.45" data-doi="10.1371/journal.pone.0112227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Chapter Two - Genomic Selection in Plant Breeding: Knowledge and Prospects</span>, pp. <span class="cit-fpage">77</span>–<span class="cit-lpage">123</span> in <span class="cit-source">Advances in Agronomy</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.47" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.48" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.49" data-doi="10.1016/j.livsci.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic evaluation of cattle in a multi-breed context.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.livsci.2014.05.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.50" data-doi="10.1534/genetics.114.165282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mäki-Tanila</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Influence of gene interaction on complex trait variation with multilocus models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165282%26rft_id%253Dinfo%253Apmid%252F24990992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMzU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.51" data-doi="10.1093/bioinformatics/btq526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinnamon</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuchner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2803</span>–<span class="cit-lpage">2810</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq526%26rft_id%253Dinfo%253Apmid%252F20861027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20861027&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283919800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.52" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.53" data-doi="10.2134/agronj1997.00021962008900050018x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redfearn</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Predicting Developmental Morphology in Switchgrass and Big Bluestem.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1997.00021962008900050018x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DMitchell%26rft.auinit1%253DR.%2BB.%26rft.volume%253D89%26rft.issue%253D5%26rft.spage%253D827%26rft.epage%253D832%26rft.atitle%253DPredicting%2BDevelopmental%2BMorphology%2Bin%2BSwitchgrass%2Band%2BBig%2BBluestem%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1997.00021962008900050018x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1997.00021962008900050018x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997YD12000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.54" data-doi="10.1023/A:1024068626366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengio</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference for the Generalization Error.</span> <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1024068626366</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D52%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024068626366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024068626366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.55" data-doi="10.1038/nature07331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutalik</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genes mirror geography within Europe.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">101</span> (addendum: Nature 456: 274).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07331</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D456%26rft.issue%253D7218%26rft.spage%253D98%26rft.epage%253D101%26rft.atitle%253DGenes%2Bmirror%2Bgeography%2Bwithin%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07331%26rft_id%253Dinfo%253Apmid%252F18758442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18758442&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260674000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.56" data-doi="10.1534/genetics.111.128694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porcu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlather</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Predicting genetic values: a kernel-based best linear unbiased prediction with genomic data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.128694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128694%26rft_id%253Dinfo%253Apmid%252F21515573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNjk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.57" data-doi="10.1534/genetics.110.113910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzatella</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complete switchgrass genetic maps reveal subgenome collinearity, preferential pairing and multilocus interactions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.113910</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.113910%26rft_id%253Dinfo%253Apmid%252F20407132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvNzQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.58" data-doi="10.1534/g3.115.022897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Prediction Accounting for Residual Heteroskedasticity.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022897</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022897%26rft_id%253Dinfo%253Apmid%252F26564950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjU6IjYvMS8xIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.59" data-doi="10.1051/gse:2004004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phocas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloë</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Should genetic groups be fitted in BLUP evaluation? Practical answer for the French AI beef sire evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-36-3-325</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DPhocas%26rft.auinit1%253DF.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D345%26rft.atitle%253DShould%2Bgenetic%2Bgroups%2Bbe%2Bfitted%2Bin%2BBLUP%2Bevaluation%253F%2BPractical%2Banswer%2Bfor%2Bthe%2BFrench%2BAI%2Bbeef%2Bsire%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2004004%26rft_id%253Dinfo%253Apmid%252F15107269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2004004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15107269&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.60" data-doi="10.1534/g3.115.024950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramstein</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Accuracy of Genomic Prediction in Switchgrass (Panicum virgatum L.) Improved by Accounting for Linkage Disequilibrium.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.024950</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024950%26rft_id%253Dinfo%253Apmid%252F26869619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNC8xMDQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.61" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.62" data-doi="10.1371/journal.pgen.1004572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz-Linares</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adhikari</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acuña-Alonzo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinto-Sanchez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Admixture in Latin America: geographic structure, phenotypic diversity and self-perception of ancestry based on 7,342 individuals.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004572%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004572%26rft_id%253Dinfo%253Apmid%252F25254375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004572&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25254375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.63" data-doi="10.3835/plantgenome2014.09.0046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient Use of Historical Data for Genomic Selection: A Case Study of Stem Rust Resistance in Wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.09.0046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.09.0046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.09.0046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.64" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing Genomic Selection Prediction Accuracy in a Dynamic Barley Breeding Population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.65" data-doi="10.1016/0960-8524(95)00176-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conger</span>  <span class="cit-name-given-names">B. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Switchgrass as a sustainable bioenergy crop.</span> <abbr class="cit-jnl-abbrev">Bioresour. Technol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0960-8524(95)00176-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioresour.%2BTechnol.%26rft.volume%253D56%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0960-8524%252895%252900176-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0960-8524(95)00176-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VG11600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Variance Components</span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.67" data-doi="10.1016/j.ajhg.2012.10.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improved heritability estimation from genome-wide SNPs.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1021</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2012.10.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D91%26rft.spage%253D1011%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.10.010%26rft_id%253Dinfo%253Apmid%252F23217325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.10.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23217325&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.789.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torrie</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickey</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>, pp. <span class="cit-fpage">334</span>–<span class="cit-lpage">381</span> in <span class="cit-source">Principles and procedures of statistics A biometrical approach</span>, <span class="cit-edition">Ed. 3rd</span>. <span class="cit-publ-name">McGraw Hill Book Company Inc.</span>, <span class="cit-publ-loc">New York, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.69" data-doi="10.1534/g3.112.004630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.70" data-doi="10.1016/j.ajhg.2012.05.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochs-Balcom</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caan</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating kinship in admixed populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2012.05.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DThornton%26rft.auinit1%253DT.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D122%26rft.epage%253D138%26rft.atitle%253DEstimating%2Bkinship%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.05.024%26rft_id%253Dinfo%253Apmid%252F22748210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.05.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748210&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.71" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G. A. M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A.-M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span> (erratum: PLoS One 10: e0141940).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.72" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.73" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: reliability of genomic predictions for North American Holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1514</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.74" data-doi="10.1534/genetics.116.197004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sverdlov</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient Estimation of Realized Kinship from Single Nucleotide Polymorphism Genotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.197004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197004%26rft_id%253Dinfo%253Apmid%252F28100587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS8zLzEwNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.75" data-doi="10.1534/genetics.115.183269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An Equation to Predict the Accuracy of Genomic Values by Combining Data from Multiple Traits, Populations, or Environments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183269%26rft_id%253Dinfo%253Apmid%252F26637542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNzk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzc4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The huge Package for High-dimensional Undirected Graph Estimation in R.</span> <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1059</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes.%26rft.volume%253D13%26rft.spage%253D1059%26rft_id%253Dinfo%253Apmid%252F26834510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26834510&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.789.77" data-doi="10.1111/jbg.12089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic predictions across Nordic Holstein and Nordic Red using the genomic best linear unbiased prediction model with different genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12089%26rft_id%253Dinfo%253Apmid%252F24750283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24750283&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/789.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/769" title="The Role of UV-B light on Small RNA Activity During Grapevine Berry Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/807" title="Tissue-Specific Transcriptome Analysis Reveals Candidate Genes for Terpenoid and Phenylpropanoid Metabolism in the Medicinal Plant Ferula assafoetida" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/789" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vlbgaGsA51TYKthwDAZnrzXySwlUq8eoYpr9VUw4wqw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/789" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/789" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/789" class="active" data-icon-position="" data-hide-link-title="0">Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492412" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iIo7Av5sc8LJhP9mwKropwTKSQqpocM4ObS12iQlGIQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/789.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/789" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FYWrwjWfS4RQTBx7Yep07me6VaRH8DEKujzrqB5vuVE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/789" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/789.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492412" id="node492412--2" data-pisa="ggg;9/3/789" data-pisa-master="ggg;g3.118.200969" data-apath="/ggg/9/3/789.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guillaume P. Ramstein</span> and <span class="highwire-citation-author" data-delta="1">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">789-805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200969 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492412--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guillaume P. Ramstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gr226@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">gr226@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/789" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/789&amp;text=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/789&amp;t=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/789&amp;title=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492412/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492412" id="node492412--3" data-pisa="ggg;9/3/789" data-pisa-master="ggg;g3.118.200969" data-apath="/ggg/9/3/789.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Extensions of BLUP Models for Genomic Prediction in Heterogeneous Populations: Application in a Diverse Switchgrass Sample</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guillaume P. Ramstein</span> and <span class="highwire-citation-author" data-delta="1">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">789-805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200969 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492412--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Guillaume P. Ramstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guillaume%2BP.%2BRamstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramstein%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuillaume%2BP.%2BRamstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guillaume P. Ramstein</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gr226@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">gr226@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Agricultural Research Service, United States Department of Agriculture, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=789&amp;rft.epage=805&amp;rft.atitle=Extensions%20of%20BLUP%20Models%20for%20Genomic%20Prediction%20in%20Heterogeneous%20Populations%3A%20Application%20in%20a%20Diverse%20Switchgrass%20Sample&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ramstein%2C+Guillaume+P.&amp;rft.au=Casler%2C+Michael+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492412/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492412/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492412/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492412/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492412/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492412/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492412/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492412/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492412/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492412/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492412/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492412/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/789" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#ack-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conflict of interest</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 3</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/789.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/789.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/789.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/789.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
