<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/685" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/685.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/685.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/685.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/685/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/685.atom" />
<meta name="HW.pisa" content="ggg;9/3/685" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.118.200613" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Santantonio et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6870110" />
<meta name="DC.Description" content="Whole genome duplications have played an important role in the evolution of angiosperms. These events often occur through hybridization between closely related species, resulting in an allopolyploid with multiple subgenomes. With the availability of affordable genotyping and a reference genome to locate markers, breeders of allopolyploids now have the opportunity to manipulate subgenomes independently. This also presents a unique opportunity to investigate epistatic interactions between homeologous orthologs across subgenomes. We present a statistical framework for partitioning genetic variance to the subgenomes of an allopolyploid, predicting breeding values for each subgenome, and determining the importance of inter-genomic epistasis. We demonstrate using an allohexaploid wheat breeding population evaluated in Ithaca, NY and an important wheat dataset from CIMMYT previously shown to demonstrate non-additive genetic variance. Subgenome covariance matrices were constructed and used to calculate subgenome interaction covariance matrices for variance component estimation and genomic prediction. We propose a method to extract population structure from all subgenomes at once before covariances are calculated to reduce collinearity between subgenome estimates. Variance parameter estimation was shown to be reliable for additive subgenome effects, but was less reliable for subgenome interaction components. Predictive ability was equivalent to current genomic prediction methods. Including only inter-genomic interactions resulted in the same increase in accuracy as modeling all pairwise marker interactions. Thus, we provide a new tool for breeders of allopolyploid crops to characterize the genetic architecture of existing populations, determine breeding goals, and develop new strategies for selection of additive effects and fixation of inter-genomic epistasis." />
<meta name="DC.Contributor" content="Nicholas Santantonio" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Mark Sorrells" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Whole genome duplications have played an important role in the evolution of angiosperms. These events often occur through hybridization between closely related species, resulting in an allopolyploid with multiple subgenomes. With the availability of affordable genotyping and a reference genome to locate markers, breeders of allopolyploids now have the opportunity to manipulate subgenomes independently. This also presents a unique opportunity to investigate epistatic interactions between homeologous orthologs across subgenomes. We present a statistical framework for partitioning genetic variance to the subgenomes of an allopolyploid, predicting breeding values for each subgenome, and determining the importance of inter-genomic epistasis. We demonstrate using an allohexaploid wheat breeding population evaluated in Ithaca, NY and an important wheat dataset from CIMMYT previously shown to demonstrate non-additive genetic variance. Subgenome covariance matrices were constructed and used to calculate subgenome interaction covariance matrices for variance component estimation and genomic prediction. We propose a method to extract population structure from all subgenomes at once before covariances are calculated to reduce collinearity between subgenome estimates. Variance parameter estimation was shown to be reliable for additive subgenome effects, but was less reliable for subgenome interaction components. Predictive ability was equivalent to current genomic prediction methods. Including only inter-genomic interactions resulted in the same increase in accuracy as modeling all pairwise marker interactions. Thus, we provide a new tool for breeders of allopolyploid crops to characterize the genetic architecture of existing populations, determine breeding goals, and develop new strategies for selection of additive effects and fixation of inter-genomic epistasis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/685" />
<meta name="citation_id" content="9/3/685" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/685" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/685.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/685.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/685.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200613" />
<meta name="citation_pmid" content="30455185" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="685" />
<meta name="citation_lastpage" content="698" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicholas Santantonio" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and" />
<meta name="citation_author_email" content="ns722@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4351-4023" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and" />
<meta name="citation_author_institution" content="USDA ARS, Robert W. Holley Center for Agriculture &amp; Health, Ithaca, NY 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_author" content="Mark Sorrells" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7367-2663" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. Abel;citation_author=C. Möllers;citation_author=H. Becker;citation_title=Development of synthetic brassica napus lines for the analysis of “fixed heterosis” in allopolyploid plants.;citation_pages=157-163;citation_volume=146;citation_year=2005;citation_doi=10.1007/s10681-005-3364-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_author=J. Isidro-Sánchez;citation_title=Locally epistatic models for genome-wide prediction and association by importance sampling.;citation_pages=74;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0348-8" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=D. Akdemir;citation_author=J. I. Sánchez;citation_title=Efficient breeding by genomic mating.;citation_volume=7;citation_year=2016;citation_doi=10.3389/fgene.2016.00210" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=R. Bernardo;citation_author=A. M. Thompson;citation_title=Germplasm architecture revealed through chromosomal effects for quantitative traits in maize.;citation_pages=1-11;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2015.04.0025" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. Bingham;citation_author=R. Groose;citation_author=D. Woodfield;citation_author=K. Kidwell;citation_title=Complementary Gene Interactions in Alfalfa are Greater in Autotetraploids than Diploids;citation_pages=823-829;citation_volume=34;citation_year=1994;citation_issue=4;citation_doi=10.2135/cropsci1994.0011183X003400040001x" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. A. Birchler;citation_author=H. Yao;citation_author=S. Chudalayandi;citation_author=D. Vaiman;citation_author=R. A. Veitia;citation_title=Heterosis.;citation_volume=22;citation_year=2010;citation_pmid=20622146;citation_doi=10.1105/tpc.110.076133" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=N. K. Blake;citation_author=B. R. Lehfeldt;citation_author=M. Lavin;citation_author=L. E. Talbert;citation_title=Phylogenetic reconstruction based on low copy DNA sequence data in an allopolyploid: the B genome of wheat.;citation_pages=351-360;citation_volume=42;citation_year=1999;citation_issue=2;citation_pmid=10231966;citation_doi=10.1139/g98-136" />
<meta name="citation_reference" content="Butler, D., 2009 asreml: asreml() fits the linear mixed model. R package version 3.0." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Cockerham;citation_author=Z.-B. Zeng;citation_title=Design III With Marker Loci;citation_pages=1437-1456;citation_volume=143;citation_year=1996;citation_issue=3;citation_pmid=8807314" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=L. Comai;citation_author=A. Madlung;citation_author=C. Josefsson;citation_author=A. Tyagi;citation_title=Do the different parental 'heteromes' cause genomic shock in newly formed allopolyploids?;citation_pages=1149-1155;citation_volume=358;citation_year=2003;citation_issue=1434;citation_pmid=12831481;citation_doi=10.1098/rstb.2003.1305" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="de los Campos, G. and P. Pérez Rodriguez, 2015 BGLR: Bayesian Generalized Linear Regression. R package version 1.0.4." />
<meta name="citation_reference" content="citation_title=Heterosis: feeding people and protecting natural resources;citation_author=D. N. Duvick;citation_pages=19-29;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Dvořák;citation_author=P. Terlizzi;citation_author=H.-B. Zhang;citation_author=P. Resta;citation_title=The evolution of polyploid wheats: identification of the A genome donor species.;citation_pages=21-31;citation_volume=36;citation_year=1993;citation_issue=1;citation_pmid=18469969;citation_doi=10.1139/g93-004" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=C. Eckart;citation_author=G. Young;citation_title=The approximation of one matrix by another of lower rank.;citation_pages=211-218;citation_volume=1;citation_year=1936;citation_doi=10.1007/BF02288367" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=N. C. Ellstrand;citation_author=K. A. Schierenbeck;citation_title=Hybridization as a stimulus for the evolution of invasiveness in plants?;citation_pages=7043-7050;citation_volume=97;citation_year=2000;citation_issue=13;citation_pmid=10860969;citation_doi=10.1073/pnas.97.13.7043" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Eshed;citation_author=D. Zamir;citation_title=Less-Than-Additive Epistatic Interactions of Quantitative Trait Loci in Tomato;citation_pages=1807-1817;citation_volume=143;citation_year=1996;citation_issue=4;citation_pmid=8844166" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=M. Feldman;citation_author=A. A. Levy;citation_author=T. Fahima;citation_author=A. Korol;citation_title=Genomic asymmetry in allopolyploid plants: wheat as a model.;citation_volume=63;citation_year=2012;citation_pmid=22859676;citation_doi=10.1093/jxb/ers192" />
<meta name="citation_reference" content="citation_journal_title=Earth and Environmental Science Transactions of the Royal Society of Edinburgh;citation_author=R. A. Fisher;citation_title=Xv–the correlation between relatives on the supposition of mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1919;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. L. Gage;citation_author=D. Jarquin;citation_author=C. Romay;citation_author=A. Lorenz;citation_author=E. S. Buckler;citation_title=The effect of artificial selection on phenotypic plasticity in maize.;citation_pages=1348;citation_volume=8;citation_year=2017;citation_doi=10.1038/s41467-017-01450-2" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=D. Garrick;citation_title=Associations Among Body Condition Score, Body Weight, and Reproductive Performance in Seasonal-Calving Dairy Cattle;citation_pages=376-391;citation_volume=90;citation_year=2007;citation_issue=1;citation_pmid=17183106" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Association for the Advancement of Animal Breeding and Genetics;citation_author=A. Gilmour;citation_title=Asreml for testing fixed effects and estimating multiple trait variance components.;citation_pages=386-390;citation_volume=12;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=Tassel-gbs: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Goddard;citation_author=B. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=J. Haldane;citation_title=The part played by recurrent mutation in evolution.;citation_pages=5-19;citation_volume=67;citation_year=1933;citation_doi=10.1086/280465" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. Henderson;citation_title=Best Linear Unbiased Prediction of Nonadditive Genetic Merits in Noninbred Populations;citation_pages=111-117;citation_volume=60;citation_year=1985;citation_issue=1;citation_doi=10.2527/jas1985.601111x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=W. Huang;citation_author=T. F. Mackay;citation_title=The genetic architecture of quantitative traits cannot be inferred from variance component analysis.;citation_pages=e1006421;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006421" />
<meta name="citation_reference" content="IWGSC, I. W. G. S. C., 2018  Shifting the limits in wheat research and breeding using a fully annotated reference genome by the international wheat genome sequencing consortium (iwgsc). Science 361: eaar7191." />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J.-L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. Jiang;citation_author=R. H. Schmidt;citation_author=Y. Zhao;citation_author=J. C. Reif;citation_title=A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.;citation_pages=1741-1746;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3974" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with burrows–wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and regression by randomforest.;citation_pages=18-22;citation_volume=2;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="Lynch, M. and J. S. Conery, 2003 The origins of genome complexity. Science 302: 1401–1404." />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=J. Mac Key;citation_title=Significance of mating systems for chromosomes and gametes in polyploids.;citation_pages=165-176;citation_volume=66;citation_year=1970;citation_issue=2;citation_pmid=5525824" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Marcussen;citation_author=S. R. Sandve;citation_author=L. Heier;citation_author=M. Spannagl;citation_author=M. Pfeifer;citation_title=Ancient hybridizations among the ancestral genomes of bread wheat;citation_pages=1250092-1250092;citation_volume=345;citation_year=2014;citation_issue=6194;citation_pmid=25035499;citation_doi=10.1126/science.1250092" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. Martini;citation_author=V. Wimmer;citation_author=M. Erbe;citation_author=H. Simianer;citation_title=Epistasis and covariance: how gene interaction translates into genomic relationship.;citation_pages=963-976;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2675-5" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=B. McClintock;citation_title=The significance of responses of the genome to challenge.;citation_pages=792-801;citation_volume=266;citation_year=1984;citation_doi=10.1126/science.15739260" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Meuwissen;citation_author=B. J. Hayes;citation_author=M. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=A. Nejati-Javaremi;citation_author=C. Smith;citation_author=J. Gibson;citation_title=Effect of total allelic relationship on accuracy of evaluation and response to selection;citation_pages=1738-1745;citation_volume=75;citation_year=1997;citation_issue=7;citation_pmid=9222829;citation_doi=10.2527/1997.7571738x" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_author=S. Ohno;citation_year=1970;citation_doi=10.1007/978-3-642-86659-3" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=H. Ozkan;citation_author=A. A. Levy;citation_author=M. Feldman;citation_title=Allopolyploidy-Induced Rapid Genome Evolution in the Wheat (Aegilops-Triticum) Group;citation_pages=1735-1747;citation_volume=13;citation_year=2001;citation_issue=8;citation_pmid=11487689;citation_doi=10.1105/tpc.13.8.1735" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. E. Rutkoski;citation_author=J. Poland;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Imputation of unordered markers and the impact on genomic selection accuracy.;citation_pages=427-439;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.112.005363" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. Salamini;citation_author=H. Özkan;citation_author=A. Brandolini;citation_author=R. Schäfer-Pregl;citation_author=W. Martin;citation_title=Genetics and geography of wild cereal domestication in the near east.;citation_pages=429-441;citation_volume=3;citation_year=2002;citation_issue=6;citation_pmid=12042770;citation_doi=10.1038/nrg817" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=A low resolution epistasis mapping approach to identify chromosome arm interactions in allohexaploid wheat.;citation_volume=9;citation_year=2018a;citation_pmid=30455184;citation_doi=10.1534/g3.118.200646" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.;citation_year=2018b;citation_doi=10.1101/376731" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Segovia-Lerma;citation_author=L. Murray;citation_author=M. Townsend;citation_author=I. Ray;citation_title=Population-based diallel analyses among nine historically recognized alfalfa germplasms.;citation_pages=1568-1575;citation_volume=109;citation_year=2004;citation_issue=8;citation_pmid=15372154;citation_doi=10.1007/s00122-004-1784-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. J. Stekhoven;citation_author=P. Bühlmann;citation_title=Missforest––non-parametric missing value imputation for mixed-type data.;citation_volume=28;citation_year=2011;citation_pmid=22039212;citation_doi=10.1093/bioinformatics/btr597" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Strandén;citation_author=D. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. W. Stuber;citation_author=S. E. Lincoln;citation_author=D. Wolff;citation_author=T. Helentjaris;citation_author=E. Lander;citation_title=Identification of genetic factors contributing to heterosis in a hybrid from two elite maize inbred lines using molecular markers.;citation_pages=823-839;citation_volume=132;citation_year=1992;citation_issue=3;citation_pmid=1468633" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. Van Raden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. Visscher;citation_author=S. Macgregor;citation_author=B. Benyamin;citation_author=G. Zhu;citation_author=S. Gordon;citation_title=Genome partitioning of genetic variation for height from 11,214 sibling pairs.;citation_pages=1104-1110;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924350;citation_doi=10.1086/522934" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominant variance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Wang;citation_author=M.-C. Luo;citation_author=Z. Chen;citation_author=F. M. You;citation_author=Y. Wei;citation_title=Aegilops tauschii single nucleotide polymorphisms shed light on the origins of wheat d-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.;citation_pages=925-937;citation_volume=198;citation_year=2013;citation_pmid=23374069;citation_doi=10.1111/nph.12164" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. D. Wolfe;citation_author=P. Kulakow;citation_author=I. Y. Rabbi;citation_author=J.-L. Jannink;citation_title=Marker-based estimates reveal significant non-additive effects in clonally propagated cassava (manihot esculenta): implications for the prediction of total genetic value and the selection of varieties.;citation_volume=6;citation_year=2016;citation_pmid=27587297;citation_doi=10.1534/g3.116.033332" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=T. A. Manolio;citation_author=L. R. Pasquale;citation_author=E. Boerwinkle;citation_author=N. Caporaso;citation_title=Genome partitioning of genetic variation for complex traits using common SNPs.;citation_pages=519-525;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21552263;citation_doi=10.1038/ng.823" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Whole genome duplications have played an important role in the evolution of angiosperms. These events often occur through hybridization between closely related species, resulting in an allopolyploid with multiple subgenomes. With the availability of affordable genotyping and a reference genome to locate markers, breeders of allopolyploids now have the opportunity to manipulate subgenomes independently. This also presents a unique opportunity to investigate epistatic interactions between homeologous orthologs across subgenomes. We present a statistical framework for partitioning genetic variance to the subgenomes of an allopolyploid, predicting breeding values for each subgenome, and determining the importance of inter-genomic epistasis. We demonstrate using an allohexaploid wheat breeding population evaluated in Ithaca, NY and an important wheat dataset from CIMMYT previously shown to demonstrate non-additive genetic variance. Subgenome covariance matrices were constructed and used to calculate subgenome interaction covariance matrices for variance component estimation and genomic prediction. We propose a method to extract population structure from all subgenomes at once before covariances are calculated to reduce collinearity between subgenome estimates. Variance parameter estimation was shown to be reliable for additive subgenome effects, but was less reliable for subgenome interaction components. Predictive ability was equivalent to current genomic prediction methods. Including only inter-genomic interactions resulted in the same increase in accuracy as modeling all pairwise marker interactions. Thus, we provide a new tool for breeders of allopolyploid crops to characterize the genetic architecture of existing populations, determine breeding goals, and develop new strategies for selection of additive effects and fixation of inter-genomic epistasis." />
<meta name="og-title" property="og:title" content="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/685" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Whole genome duplications have played an important role in the evolution of angiosperms. These events often occur through hybridization between closely related species, resulting in an allopolyploid with multiple subgenomes. With the availability of affordable genotyping and a reference genome to locate markers, breeders of allopolyploids now have the opportunity to manipulate subgenomes independently. This also presents a unique opportunity to investigate epistatic interactions between homeologous orthologs across subgenomes. We present a statistical framework for partitioning genetic variance to the subgenomes of an allopolyploid, predicting breeding values for each subgenome, and determining the importance of inter-genomic epistasis. We demonstrate using an allohexaploid wheat breeding population evaluated in Ithaca, NY and an important wheat dataset from CIMMYT previously shown to demonstrate non-additive genetic variance. Subgenome covariance matrices were constructed and used to calculate subgenome interaction covariance matrices for variance component estimation and genomic prediction. We propose a method to extract population structure from all subgenomes at once before covariances are calculated to reduce collinearity between subgenome estimates. Variance parameter estimation was shown to be reliable for additive subgenome effects, but was less reliable for subgenome interaction components. Predictive ability was equivalent to current genomic prediction methods. Including only inter-genomic interactions resulted in the same increase in accuracy as modeling all pairwise marker interactions. Thus, we provide a new tool for breeders of allopolyploid crops to characterize the genetic architecture of existing populations, determine breeding goals, and develop new strategies for selection of additive effects and fixation of inter-genomic epistasis." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jFi3R43EtvaWzhuNSdX2bSjN2JNCgLeUzRQtiKTMTZg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492493","apath":"\/ggg\/9\/3\/685.atom","pisa":"ggg;9\/3\/685","ac":{"\/ggg\/9\/3\/685.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/685.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/685"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/685"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/685":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wB1wBr3NCPBNYaOwodNSUHy+RPQFiKBsllCZgA3w1LXGioBDXC0Aa607VuVkjxTAgTf8ISrZJB7mGGx8MrO0RoG1imjP3ffe2uo7YDM1Un18FcJDhuNqAVv6bmm4CVkb8CVb+ia3a6qOyQdz9xI1fyAsLJxHSey3i\\\\\\\/4WIXaPd1wmnf3D\\\\\\\/ACk3K12i1xHT7mIICcriekW0AWd3v9tnQBHtOWf57QB3\\\\\\\/9Ay2zymIrSehXl1y+jj\\\\\\\/byv90N9\\\\\\\/QwESk0\\\\\\\/afI8wv9DoDaw0gr17WsLkoYeTHODxvQEAhpNtTE5DeTqv5heZ6sgDg7yPHPOF2vdK6rigRlgaxwZd75jweLUG3n4dJ8+6zKZoYTcuMHeDhDbk7zaBX+XHxFw\\\\\\\/pwrFC3RY7m4aLQH0e36YGrK3vVCRsGYtdGDIgvr2tjbbc77gwajrV5wpm3jHrw8xXq3rkRyjDac04293XmrV0ocsPYwcKjWKOad6evMzG3zDkeoGuNWqw+LHiGPC8cbD6yyfRbf6eBRHn5hfMVjWDoHuRXGFKWBuAEkybTCOaKGGO0j93usUJVjgpAajO6mZFGDEfcKvWQnGXgqi8lTm7Zt0ImI9wIRRqywxODA71M+ZsX4UZMceDdzxwJ3qKoa7alQGnw\\\\\\\/Q8Gb5IbdngfrNFejjBUtzrMIG45zK2Znyk5SvQDtpBMzx78uLAP+b7zGonRGtur2aMMHLDmEIwB27vZxZ3TJApQgTp58djoSndX0\\\\\\\/w+opECsJZ92NGmlnbsZ7o+h1aWh0oIpm0oEPBeAv60Zwj92z9KEImeXJoY43xuHe0stbT030wLR7sCEh9q1Cr2BpD6URnx9NRmdlSxBl9\\\\\\\/FVP8jBecPbfUhCI5D+MSIn6cQvW6fZucntYupT8dEfBIIU\\\\\\\/Xyt8Z7Y\\\\\\\/izJOrKc7RkVKQ5wYGLX2rgJ00cO7uR1tYYWYqHnP0xklaUtoyw3MOti7QoOob5CHnruHHtnnUZuA3xzHY9zh2d6SmRmBOJswjqyg3CSqusImEqYILCeceaYHdvrbydXqVEEUYV\\\\\\\/RF7FWoAH6gHsurD9lsGo4Z3UD9ug6bt22nQ89enVg1lrC5CAywPOJcDL8asWdVVPq1MSGPp5vkI+45Ei6f3ebEMfuQiDw7SjeAAYx3MLyp7+Z1bXjHe871QZkO349vJ9lktvOSTLSL0U7QLCfPjmkg6d\\\\\\\/1oBAN+wCUYzzsgxsdndls7DaArb5qsAhjv\\\\\\\/EWKoQtH2Spdk0O6HslYbWC5KvlgeNbAkTJGek5jRKV4ieplHsBmPIC\\\u0022,\\\u0022iv\\\u0022:\\\u0022u0oc0GtOvS909DuKzoKetA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022aae05fed07598fd4acde7f6e37cfb62f36e07d262c62c7508bbbabe104777ee8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022krUM3VrlS6xe4I1VExEoEqVKCbdWf9CoZ8khqjvkD\\\\\\\/+uSG\\\\\\\/JSGTfFqTQracZ+g08FOLyekpTOCBEQxT5sDDYEZB442AwYk6Thv2XYbrYMlVI8IrPzXdMJxAayFil3OV8hRUVEg\\\\\\\/CQZ3J0F0fu5gwttPH3S9sEuS3cyxdd1HuM6Ql3Tj1L01gd9SkcXs4wOxKahDSvqCREPJzyRFC8Hejt\\\\\\\/YSXvmZC01RH0h2ML0rKr6aAPkyFYbwyqKwmExsD7R4eY13at0yLPudmgxbURlvrid0E3K9jwMYe\\\\\\\/SJN60qCA5OZjEW4EONcK0KzF+jM\\\\\\\/FjKhS9tojVVRiYgD4DRX9X7LVF9aCcAYEQuBb+AalagXQNL2W6IXc9xuwaKwxIMKgwmNcz6M3zBKvUbb2180VhtEdH31noVXMRk7yDbgqO0P6NdByQ1hheuiZWcMe+RpXYUdkt3CHqA14ljGGc+LcRPDFsiYhtKqWT6FLtbdNS97Pkb3yap2UqHgemdEFJ8u7MDodpOMXCxyMVO3xnJeM49McHZ1okCbd\\\\\\\/Ev+pG8PljZnd2UEIJ7Rjq+1FJAD6le6pVBqIJElySrzDWZB8in2oGRB5E5uLsQSFCBoW5FzdOMiny4CkmfxmQ2OhSPnvJM0KSwnjU3JFsQoo8xK6JhdOk6G5Q0qH5j7VljEoDw88PYfA+CNDhVTCIv4lA\\\\\\\/htXUTolWY9huCFDtxc7kd0GdwfCDFh4gspb0pkQZ7MMM0OY37ZhWksyC2csAn98QDVv9qQ+6kfr+4VtCgGyDxBCmrk4oKKSs\\\\\\\/ou7kUdx7JTQFvI6AX\\\\\\\/Ck4Xm0QsntIIp+5eOOVevJl0fDDMu0XVuGVV2yKNT1Wb9jzbf\\\\\\\/QcRnvo8EGOEGh0rz56GL94jvGiMhbSx7WDBm7q2I1WUw7idKTMWJrPfkopUc0nrz3boHdvDyYTp7gKYmgM47XgbE2w0INGqrrxu9RMfbQMy0huJKzFEplVn7UZhE72ewTX1Yhbg37lM03NOOGcvTozLu\\\\\\\/CXwSEw6ujQlZROkO9Zp3\\\\\\\/Vql5rFm69uBJiYsTeifdA2qBgpK5vq5EY\\\\\\\/spynnS8wnUt912QvDmEpWSIcImZhpD\\\\\\\/Wq9NfA3tMtDLnFQzuFMKN0t\\\\\\\/U6A14+x9f8HUdJa7\\\\\\\/x3nQ0\\\\\\\/GeKVeMVZ7+FwPS\\\\\\\/U0Wurn\\\\\\\/sbtPtj6\\\\\\\/Xshvzz978OcH81LiPoh+eEcJzpQxQMSQinrdTYIi3fr8NNB6+u0lifS8YBUzIdXYRmNvjQgX6CPwLBl3wwFni+zJP1yRB9v4eytgYlaAAP48bKSEK\\\\\\\/lVfF9hVSPqJkRPtI6W7dypnUu3rfsB+xOVNhlZR8zLNI7dssKPEt60ppfr96XpGl+X5tzY\\\\\\\/a7RoUKlbtqW52m39TVsniVu1zf\\\\\\\/sTWlBI8FFDx5Rln\\\\\\\/MQtwqc++zkBl4IxDqPdUE03X2a6UuFSp9Par0OCfGnbM8OCG+fqV26NevgtXWG4EHKhJIlm\\\\\\\/aGbN4Sir756PRkTJray5MWJio+0kTdfPp4iTWybYSiWoRC2Xqg50xdIYYSG7tKomDgiv+nE33wxC0hL\\\\\\\/X3GhtNKwXHmuyqV374tTrfuhltMFGj\\\\\\\/nOrY5NzrllCKWC5pc7t8skh7AQL412Lvv9gwgeMJ6V6xo7xWYKbAm9yE2iSde0FGXpP\\\\\\\/n2ezZGv5RsJOH+PtiiRTd0k+7V2W2Wz36D2YXtKa9PTv1ldzin3d6rLUU+cn9Ja47ir5uqJY\\\\\\\/FoQp+AJ8cRZoWcFMLoTgnwZ3yYn5GL+MgHACPExk2UT96h0ANk\\\\\\\/k9zvCGoPyHl04KeDxJi\\\\\\\/r50IkPHYSYmbhmCD0Y4JL4eL6queicTUVLphrlnvGX8yPP6mQ7d7X6XpycHgQrIKrK0ig5sTNo6NJK0GGDFlHeHPgBSBoHTrEp+W6h5b\\\\\\\/T97Yox3CoMyznHJ\\\\\\\/6Ozemit9A41bkI2kepCRl4OWWM0eF6ZpN6VBpVpIGJ72ea26GQESTwu\\\\\\\/kiLiQiWCqd+pb+2YGqQFFaYbZM\\\\\\\/5qb4WliAXMG4hRJKXWBNviiVxOBxScdNwTCrbuVbNw9IlutTASF3Pa15vW0Beha\\\\\\\/5AfYKKxNZh2fazMFVPXxgQnVg\\\\\\\/jsI8KiPRxZtPUJxGkGqq8WiotLn0NJMGaZwovHrwrR1ugqYswOwO+iLD9wjs+cFx27tgBO4N26BtSsbaG8fPJeKLDhrw+mzbIqLezg31NbcYAG1iSE\\\\\\\/DXFYXMdFiXD4AUTuWC85zixjn1xx2MsXsOVwQdAdsK5avq7WuOTHbOMCd6xxcc2xQUOopuT7K1KqSOTrYH5N\\\\\\\/FcCarUjW6HyABvoF6VQRRY+YKKgnfTEd4enyonwOfOr7ml\\\\\\\/GeNUj42lwMS1m6fmOmH3Ud6DyaYHYxpnV5kZviOZAJ1JQeHgsw0tBqFieNi8l2L4s0ES154PZMag47LZ6qeaJ0qLUYKFtBu\\\\\\\/UdV\\\\\\\/zjpIhS+DOo7NFeh\\\\\\\/VKXlbeyv0xKVTbWpL6LqoKBFQL7G1U1bV4eaDrB4cSducWTUmg+tB77AiCXXtkX6B+bUfFtG3PfbNj46yLLzMVd1JmS+wlCmyhOlz35FjieJVLgF3CP+L1pGy+GS1WA5lx7VkZXh\\\\\\\/s1RMbTPhiYTOj7jukBe8x5AxfqgrvWcYEWpVHvjrVZi7nrTk6duulBsSNZsLXR1DEDqGYVkF2xL6YzTjC0SF5AmyZurujpiaZMo2+2fg7mgwgFm0OjqmxkK0J7EXgyLBpwnUNpJ19FLPTjnJHVhkpq9UzH2mupWRByoJkGsox+SNY3VOP59b7IlTRWsLHuoihFDttWHou5p1rrXtk2FbPAQNwmeZMoalD6xag04Pguk6jylcX4e4VJEBfxWMJZ9oUNnix\\\\\\\/tC7510sPpgiUaBv0gb79zKKoyFw8rVMAftKhqchjSWJLzIbfk0AcI7zKf5vE5\\\\\\\/o9OMG5U0W2REI0Gs7keBEIyGej+lRmsbvRjxSS8OihEQURxWDSrcuHO\\\\\\\/SqcUMX66NBID2k9q5ZWVbrlYVPHKO3WzUcxzAo0hiMgGw\\\\\\\/WFzvvZaHbt0Vo3WGXaiF44n7D6MTH0ZnewdnmEhO4Wzy4wldDesBUQf96LGhF1x9UWpnfhGdKT3D6ZBAPGcdGBy3chCVr5Xzb1pu1eB59s3e6JnlMioCjouxV91QEKoNHNqEABbjOhLCIC7aSaX8GO4ozpLB9a6Yo+I6UibHlQ7nyeGRYYO0GWXMA4M3Pygd+WSAytugb+XOkE5Hlm4jYpke0Me8GLIqoDGyycWupK+Vdk956Ht5o6GyUzYFJvVKZku82DVVNqp0QPpwdqGPbyXO3bwjYGGtDfQlXQ6RXptW7yA69GLWZJPFRY9CtLHm1mcvas7avtFsaZQ0LijsMSMNt8CWc+dC+JubFBh370s1hwJ3oAMXBLhZCbMF1uu+gV7BW8vLy2as8peNo+hC0oQOLisEvPumYGKZS3Kezv9snlxDD4ImB2OYhrHQ97ia64vP3Ht4K+I0siauAd1apcLNYk4Jbr1GvlOKGla4T7SEl5wgB3wUK8ltYuEm\\\\\\\/utg7+XkBpefvS7k5565H6JpWX13jI+QOMq2HSP5b0Ebo7t\\\\\\\/bIDwB5jFrfBbiFgejFZxQzCcjZp1S+DYJH0PrDX50AWOdzUEOXTeRegn2AkPVvPqK\\\\\\\/uR7kuV6XfPn9D1st4+WptnjAejZu2sRMGXM+JUCskHkonglwe6P4A5wZrIi6AhIgpkpXGSNN3Yy8tYQ4S6IFJ6HqzwMu8hzr+4MsGluPA+D\\\\\\\/63yVLruhyUJjGl4xaJAKe6RSVc+CnU6uloc2Q1IfpdrYuwlvNz8rqNbmQwSiPOcE574RswNr52G7HpPmiQr4lAQEd6jfG8t\\\\\\\/rRqmeltI27z1JaWUeGj7KoQfbVZJH33kzlG5BjU\\\\\\\/oMmpv8kJ753tnz0\\\\\\\/Kr0vmhTUWTYiBXCQ+Bhp3V\\\\\\\/BZZCFtkz2t7gmZeUieNxhwjJ8RYUff+uojzkPP7gJtKFkWR4iMhLHt6cDT2N3yYl1NRvlgHCFqvDbFh5BlYWbi6DByi6Y3C46YAE+eLFF8a1ZHKr5WfIgkMklANb8VLE6o+US816Zhv2e2JzfyZeJqw869KWZyvMF1xMcEXZpOe06s2JTP6A3zwZPXeRvu5A+iimixbvt2M8yJNrY8ESm4+sOQe659OSjfc38bItTGiXCAnkfj4e3sayFajxWI9fXASJoP7p84+CX\\\\\\\/A5uleb\\\\\\\/9awRCGBR6On9a5L\\\\\\\/wJljcKoHjnTdYto8GTINth3MXf7Eyx4c7IGBZ00veRtwY1Z+PEXvvdmxDrKB9GgukmuGM9qh3uDSXY\\\\\\\/+0wnuoV5hzuNWzhunR4FNQcx7UXTaFX4Sak2PPjgPJ89Hq4HOdca6LAWcpB7660yPacwkJa6wcYzfbIZ5EdbAiRzCfVAWI85xWrN\\\\\\\/cko7T1bXBmLkpYWR2pdmEWkpseSewZ\\\\\\\/KqweYYF\\\\\\\/T\\\\\\\/PO5UNk+kS\\\\\\\/6ef6KsMC5jwvtcSgY5llCaGkcmcdhaxDwhl7CojC+cp\\\\\\\/1j7PgH\\\\\\\/FYStQX5JJHHuySsIdtE5xDGG3EHLKqhalH\\\\\\\/ksB7W4O7HDZfU0qAHDQ4nyysAmF4SxJMDcJVgKMiMiU87+2N2WMHdGp\\\\\\\/Mzn4X5Pqqw97vwGl2Qb9Md\\\\\\\/HgsfXasWf81SbbAUmzHWloTjMDUKDf1Ay8Xf8jb37K8Fa3M8foBQ+mPWsmUyIxIZ\\\\\\\/0eJo2XCVRnl4IKc5SqvB+3O3od2hE+4ixPb8LuTyFfzFwXMr0kvtY1rSfeU8hCvJ7ZdiXlScMVA2Oxt6IHahIX7+2NmkZYhYYRT4RNHFOoeScSa01U8Iv4qnPOMJfe7KKok\\\\\\\/qjLquIl8Q5E9hT4MRq212VX2tFfo8\\\\\\\/sEAYJKAuylovtk7rrML\\\\\\\/9pV0wFqY+SevNL0hZ5saxu1GAI0APp9zxrEHxjHLFEp6fPqqw3XCVzQGi2z2oTuhFnOj0pzdV4380VkFKo94ufeJwyJ7AwCKmsTb2YneSlE+yMJnQ6Hz0XNS6UAsGw4BWiwspPPJyGLdbLGORMqUviBTPhURSnfz0JBD989GcyxuVeYmS54bl2ehFieVAYPuktMp7HuA7buUscQ8aZQc8\\\\\\\/1DnoQ3HADP1B4dbjQ62MBSK\\\\\\\/xzd6qSBg+PWzlwI76kVX6krtAX943+IMJHb8+sT\\\\\\\/RE7EP\\\\\\\/JtO3IdGTV5no8VI3jtPI0u044iR7zOyZq2+zuoBI0yjRlyP993QSSbLkVYk6UNEimTcrUMFqqMzNOoNrZeSVPoVmsqopvv+KeO22YQLE5lNH74AKHIerM4cwDOU3SXbtMGdbknurBYgTibUIuxDlmOT0edDbHJUp4fLcVkV4k9yU\\\\\\\/nVMvrDCiYC\\\\\\\/tVV+F+0zvdEkK7MOm6yNK2R8LkB81b39sLUf+rhRlKEu9SQcSJiXl2KZ1v3\\\\\\\/FRNuUbjb3I242FRz6aA6\\\\\\\/Qblwz3KRAeFpp1E+Jw+AUbM36t9lj6tHhGDicNLfozbazmsXlis\\\\\\\/btR+28BUbJLPKU+OxOTBEg6bW1ZdZInXq+usZ928IKgZxkprWGQwEkF9wldSYcCPzJIGmyqphLXAANeSgPMy98PwXyKoUbR9S9KpwsVHtQcmoe5SmyyaJSmDhEWx7fEXzjO3hQb+CMTDOUdYnuYTLPMgczcoRt4LU+Nwk+PicGyHhWy54taRECfOCIw4DORC1kgJWAm0TFc1OkmlpPG3J49hsD+FHtFkqA\\\\\\\/DEauwXVQraFUkpXqFHN+is0XcWku8dXamWmXMeBXsGWyrPt4hede5tHOLzu3aTzpQRW2QkT56rA7JuelvHi7iCkqA4EaOLiE0kemoHAap4HFL3B6vPUTgPpGCR6zfq\\\\\\\/OaYXSC2yjKOLcUclTEvAX5qU2yYXGO363aZ6P+YuUAgtLLTaCNK8k4NvYGnRe1wW5nXB76+8f8VDgKq9pi1htnC865ZVR46XUP6dwg3dK3JiVX7hjMv+qXjlPwJUG0aVJ58X2uJnRZxamACyNG3eJU4Z9jE9GYI4o8bsnw6G8c+mlmk9+nohCxF3RACcHmbe1dYT2\\\\\\\/GPWD1\\\\\\\/bdlgwTs5+3Mr\\\\\\\/OR2O3mUrobvIGYRx+JSRhtkkxGyEErBs1WQQkjRS8msZ1iFa8\\\\\\\/58fFDwjBfshg9me2IpKDWFE8WvN5zin3KX+Zp2K7d1to0HHsdjPLxlOG1gZIdeOYtH3wtA5Oumavjtb1CdbC4ZVrGe0nOJvtghfQRnz97kFJiBxOrtDdRkGeQ60iWpi1p+Fzus\\\\\\\/3unlE30+M+l6dnz\\\\\\\/ZNzvRHMEZsWzZGy2gqcWYaeNefaCHzbx0CL9lfEpwsMD0cpBI\\\\\\\/xBQJ9P13mUbbG4l+BSds55HQ1izygUjpBu5BbYzNTQ8uH4j0JduvNfFs0x1kfs7\\\\\\\/sb8Z7nrAmG8QWoErvaUysNUh9ER3vtsOzqPPDfxjGxvfBI\\\\\\\/0YMoZlv\\\\\\\/H2n9uGwlvj66UGrCUfgrDUfhW4m2yx9o8+SK1UWt+CtdT6A0c7A4jbZoPdeXfQ7CuxQpBwwiKxe01kSHf5ZMPTltFoLusdIICXASxcHnAjzqVSoLxX4LdXNDt3fCK+o47d7f4hm7A0tVoK3ne32JE7yIHwwCqpfRfTZCTBUah7FHLiAnSZ3wvMKxETL7hqb19GdbQi4X8Vps90sGZCrorUDNQ9M\\\\\\\/XNesp5l\\\\\\\/78d1ZENMxzhGubiRLZ9sYrEwhk9APow2BudAdWES962DApdsGZFrg646RHebVCy3HwAAF1glXCdhaUn1u9e+coxmEo3qrRTNZk+cFQFwAcXK\\\\\\\/dW\\\\\\\/hTEqLnqgTfG2wXt5eyF9l+JtVXJG+03uklC9wdVuVYq7uaPoJIuGIUTpKl0bY\\\\\\\/0vQSvum4acDEfyZ5b16yT\\\\\\\/eTgPgLrE52+ZI\\\\\\\/JmcZwG5trZtENPysFXdd0aCoTJFvM+o59tyJOqzYVKRP0gGnYPaWo4rHP2z2kKTuqlSYeSdeuZcI71L\\\\\\\/2KNpTVSwqcS1GjHIx7WPOtrbctO0q+0WY6cOhChUBpAEQHh\\\\\\\/ZXtDU\\\\\\\/UKE1HumbKCQLcfnmCfz1QyPh07AZrYlaHmkESbpDQqfNJ52dLMUAyewLMfdiBlxGlBHtiICxiTY6WTey4CzUYo5w+73psHtxAod52Sq+V8rc2aVgYHxKbkaKUTLhN0\\\\\\\/fo0EF6HqVOr1QuEMnyNXWz97qdh4BVThbeiRrbi9L6AB1NuSnqcQZS6SFitmA\\\\\\\/58athqkM816Wqxb1qdn2s59C0X9dMONyzEvj3+AuCGvV3GBNNSKDHSzPornCBH27JorKgCQcSLExmy2Ng8+J6PWzYXjiHV1sj\\\\\\\/43Vwe75l2CI4+GkGDxs7ACUS34bLyPK8mhPptfsG2gMmz8nmPyAQrHoz\\\\\\\/h7U5y8uY0B8XsG17YVguF69KcbXnYQuvJ5SeLlbpW1erDHbnFhD3g+GDnPeclcIsjcYX8OVayViA8yJW7Wsnith5H9b\\\\\\\/C\\\\\\\/TzRUz6K2VqOG8GJ4NY4iZpOO35BKgLwEPLmN6iOH55RCx7GyDH0iPkdwxkjIBNUXyH107s\\\\\\\/iKcKY4rBPkgaKQPL0j0OTXZnDwupwlnz5\\\\\\\/c6GvIKqohvHsF+EP8WSWS9NktrSniI4NvsNw41ezFvspqpwvQxIXEJtX9GHn3o7M3bQTLy+B0W0qvdPjzRPTv679g1nvvDpQaZaovYWdOZ+YCVLKAqP1htHrzFvCJ5BtqTNZGnOC2Omn6Lt3bcSaJNZF+M7YJ1yD\\\\\\\/qw2U9Jcy4Ug8MIZnkHcu9UTcowp5dCG2Y09FdsypEH8FQjqN3nHj8fODf5QMsDdeBWqUs0yzzZ3hbZ9OWFTPQtXYTsBAe6H9mwfljd1s6eFQC6wzMX4NMces\\\\\\\/0nzrWAafcNeMNz60z3wfx34xabM7NzOj3pBSj5iE7tFHmV2p+tkDnlVboqPkZNsOSNDsP8VWA4T\\\\\\\/j4GalxmjuBRJCiucv6cepa6Fkg2dYYFCwfzwg4Pexs2jZ75iLn+3zoNjYNvHnAOCZ7JD8+8FpzsMJUd8Bn\\\\\\\/p65ZshalG48NOr0D4pW\\\\\\\/s2APM4u3MwhJFOhNvyS6O6oQwYf1viku1lCYiDawE3HbjkcJF+fByUxQFTwM6h8KBz1d29yLl08H84pupsQzpYc3fv\\\\\\\/4bdSNzGwa2rDeV7lK\\\\\\\/KNvZ4rfSlSzSAw2pSAP2zn5Rz5omVShOuY01Qi+EAmQrJoy1sKa+0f78SucthQfY+VEGhr7a6LQL6wh\\\\\\\/2Ybrke8edBNJRKW+eO3oxG8fT4HVLiMTBfPt0PpPIAeV0ov2EsiNjLNQGBNHRiqmYZ18i0Kvet4lw6q1Wct2VxbpZjV1DeplAksfF4EKuiGIS8c0w7vyegdqThDQzI6EzsMo9ASMirkoDMvMfJUemftOkBcp7lo+vSJQdQGyH4AaEL1+SMkTmmthhTuWX2eznpvmFXUxGU5t\\\\\\\/crTY\\\\\\\/OoVbfsswMmkkd489\\\\\\\/I53QbJBFt4iOLPrhqjI+sCvl3vU5F0UNa3WKZVgMeAjR40+rsFl35DzkYoFYsQXKIzMi7nfKKqV2wJq9k9nqAo3FZZ8MQTqQ+m+7CJd4sTA0mnYX50DyGUQPY2jOUE\\\\\\\/\\\\\\\/R3D65ELNPw1\\\\\\\/zfrNiAQ44T1VapFT38FB5\\\\\\\/hEx6OS5m9adWOt1v8YsKycFAHdIjTUUy+U7IrVlP5i63t0ky6V9LjA6eC6sPng2nqvRUoMeyCO93f+Zdm7CsXUytdDA7kUUytrv\\\\\\\/2pZPC\\\\\\\/w3PYCflPANbiP1gdRRHgD0XYRsSThrbiAuDbgkGlm5JpMGIfYb4+TaZ8fzU+EJr3KQkIMZzBpRVanRQnTEO\\\\\\\/WSIFcMXIw1A9HgDa6Fkl9kZ9+LXku76c\\\\\\\/oJdxypsOk7P1hZG2MyKdAQHIwXj6L5R9Othgm0pc4NLCYJGOIUOzIVa9717poBa0lnP0meB0Nrx1sWs8lpVBvqbMZkM24Z5rmugax8aEr4GMblGHNFzIyRXCZKqbr3Edc1tLqyoY61N2Vd4qkfwYt3cQitgnppikf0xc2862aYbZQdESM6SJpw+akSlH2OhBrs5VI1XBsN95mpJgJjmn8jKHc\\\\\\\/1KC8Oq8QdvFOmu4mDx694FOxNWZxY76IlcWB\\\\\\\/Y9DB+bjmFTDwBZiwAJ3OVAn6pWvKePxAIHPfU8Du7DZzaLtJpm5pddNmcGHZ7T6NCQ6D3AT8Z2fkytwU44D3mXQ0\\\\\\\/947Qk2x\\\\\\\/biQ\\\\\\\/folIm94esu0H0OHVU02b8dtCOkJpRc40j0w484NWZTYiz22T7ofsRAJu0Vn7e6jOzwA804SRp6sDKArbTvLJZHFGi38w+VNPLYQJ5vagxgD9e4hMFl5PxVPMhPg37eQM92fjMIoqDgkPMZihElC1rWg9mkcDjJq9bDMzD0M04AMexndbvaoMFDWMehgq3ueE9zh1rGV3rN2AOp0ihCtGAZswpqEVWlZLjHeP9pbmRsfPEgI0c3eZSxQwiCEKEnp9l\\\\\\\/lnl7Mf27uWAwybIaVs9Vfj3VLdzJGzcwWQK5ot4y6X32yPY2sbYYwQsCCahGm1wpexuGV0\\\\\\\/nEiTeEyCN8ovay5Pv+cU+b7hcqxpX9++TEvRh4Ctujh05r4CVgluZ6LHNd52lwkdLbacaXzw+xOGSB9LHQXdb9vRjMSVPINxW7bfMJFlTIbc0Kk+vKOiWbh\\\\\\\/4q+sCTdqQ9N2V3EoQJZRR+GEFVaHpbczYYDXAuFbAQ9if9XvmeOntR+\\\\\\\/UW13X5LJWw0FXx5CnOnw0DRKi6Cc7LQwp9\\\\\\\/romVM5C7LTbhotRA3WgoUQuhddJpqcaLp961d9UOri+a\\\\\\\/9PU0vRRX++snExGOh6dgUnjKJKyVjgKJvK4E1XtKVaVKiyiFGZwBuPMciXpPc154Qt\\\\\\\/OVh\\\\\\\/HZ\\\\\\\/S2zS0diSKk+Yz3S+Iw7lC7EI+atvtyLbsKGyBNLW+lIOi8P47aQhRia1eCSXWvoZbAuE7G334M0WGEz6JNJozsNL2FAL1iHIJwnCQkPkKjHX44L6J50K9rSHekJVDjNc73qonrvIbmi50gc2Wg0nou0OJzUoXxDs5sSUDr0Q7x6qekVrgMcRfazbszZ40jHCBbUN+qT3qIld9X+Brq4ghoYar3XAHbaa0kSTbjDvT6vZ3ZrQL5HFHDyV04uSF4bcx\\\\\\\/4mvk60iy4dclB86TTm1MJyygHvJqjQmK5mi\\\\\\\/UBAa3f+r8EpB6mADzEl9dwsNJbMnRdwMKSSM0Ca78iAPqNzm0tmToHlM0cTh92W7+vCACggXrZsz5Xxzlzh8DNpD\\\\\\\/b9nVgMOWKxxMB8uMwRePBHGZDewuYvDrPUK3dEHKiTr7oyWFA9dnqSmudJSgU\\\\\\\/TU2DlDwGyqwkLFXoqI3gu+prlAzI9t3i+dGolfEWde9dnxGKdTZAI3ftN3rfGHhhjsVzQz7GXDIr4C\\\\\\\/JHyz3wYpKlN5ljTbG5DNEPyyHr2gbiX1C38J0kKFo0DO3cD54kGJSWCreTtwzMWQ4MiekSDxB9a13H1ocE30dLf72uHyiw8mvdWWZUbfOS4AggrnpRGW79naGfeEpm+ftr95rMe+MGt82GXtCogPDvqqIVdp7g4Er4nJfdz3GiJBPgcg1NDwg1J4nGszF6Fx0xJak1AiLPD+MUUBau2Uily6UYfOQzpzAdrHH\\\\\\\/eHaCwXWb6YRwh8Mddry5SWDPjqsOCLu0kRL6QrV9Pe12ZRtEUWBg8d1enFWnUAg53iOamyOLeCzZVI1GCSxPyfEBnvO+R7XAWeiVwVn24lYAdLCDGhH3VHxI3dbC92FDqRIwrSQnIJPRMy+eICPjYVHow13fBAgS1qFqWi6mrpOsS1KV5nHCtr13kTnXeaUHlIgd0G72bdB+H3i6rEutq7tebFt1d6jizHW5bwUTA4aep0T2zgD6OcFr9tyoYMwitV5GF8c9Vxts82JCYlDiZt9wjVHczIwnNmWiWyWmpy6E404jBDeh+qO8I8w885lgibZmYzra0SKiDw45QpV6raAvKTxyPgout3dbS1F3qcJtDRIddNxgY4I89RpDLd+BmaoPqQoXpnIgv9VeYVWcnTUbB2FjTq68IHFCF3LTP0b7c1bJQNiNWe3k8YlWJsaTZVt\\\\\\\/lUMw4C6fh7WayAm20udo1yzYq8C7Oy8F8p\\\\\\\/ZvGoOxJ5e72XLhh+Pw3Fv8q\\\\\\\/UtwZjCeGcZgHRM7b3Zbe4\\\\\\\/MLovE1fRG1fAyUZELqP2BJuFAILl7vdUnYSNaFXE8el7AoAOBDfkJO1dvkkswjRb1BiBVSjGOWV4cSCRBFAxHeK7IQvBvL+AKZrITDhkFpAi0ZZk70M0v4xaNQi1gTufn42k5jkjzL1D+E5n4iY1kg1+jZPSuiF4QAC6HWMQOprtZMNIiYTh8g\\\\\\\/Evp\\\\\\\/iQspmQnMjZLz+88tft9ONObTutFZmINiXz0bjUU+BNIxzXecNz7t08MHuW1n6E+8xafVa3KsoKAaEzanthz3hOYStxOV1BY9pWFz4zw3esdMAYWBKTJocVw0HKTlDo5HMc457epui8ijgox5bX3zxHo8BfnQ0HmSjxk9mCxoAaMgZEMBTdA8TfKqJPqdUNQrNr+ATQd7kS3WQAWiFcHDWJMMQnajDD4QG3fGVGl0hdJEmfeLzjGm+aXKUy07hbCsG3OPpL7d1nit\\\\\\\/frIhQrxLlvAoKMVCGUkJ4MTg1hnyF4f1VDniGxQHD19ME1UJGR+TM\\\\\\\/LvjUh75Tcfo3Ni6iXbqLMBC87loZf+FoYx8iUBP67wEyZEVVYxPFLsvEYtM2Fd\\\\\\\/\\\\\\\/3X9R8+glVewy0if1ASYK99nnAaHbBMbrUtV\\\\\\\/QP31OG1HgqRqbibnHEcdXWzYIfEQ39JAnTsyH+1a7zS7L\\\\\\\/UeOVvSb6aDEwvo7KqxkHK1nqYoCjOu+8VA2sG7BdM9k6\\\\\\\/5LwnPBUo1kST5JgYCeLoU2fivQIWMgaNN\\\\\\\/IEisbiIfttSofTHtysmZAbVgj+ETrlgmULezbUNBfUw3G5OBHeYfaxDDOkUh6KTSGag2exwNe9TXo5BM6ipTFA+05FfAM+2c2aUsPSFp1uasHweDKEUL84369DQ+jGMHm2TBQiIQJuaZ9eW2D0R\\\\\\\/EKA8tqGyqKfPWogQMtVZYlgfoLtLPBy5rkrYCqIBY+EYsnZoGbysBXaw7p3EwktJbMaJNkvPiXdK7TTlwcmfTMXZe3lvGGNHDxhZROxiD0ZWQ6\\\\\\\/rX+Cb13szDyVv+n\\\\\\\/KWNVqnu2u30LjdOZwLxmJAmZeQbhikdHggKU+5cNpVuNwlHIQwPKWym+yeW\\\\\\\/vbqrz5A9REXpK6EgYCKJjJj60Nj731b\\\\\\\/9FNiuDIww8U8TbLzg9d1QV8DochEbOGY\\\\\\\/VGFgjet47Yi69r9Cf7zu25KbpprIGIGSq9HZq4zyJ0E+wbv4pVPtZ5c5T0QofgzLyAXwYW0sF36ZTdV4Jy2S4gxRKA+e1iHG7Livw\\\\\\\/WVL6I6ibe0s7WveTX3YwSKJnF65F4lIDjGKcca29SMye9GST88G3ZfUU1HeU1v4kvuDmrQTJ8Vtv0dqRlHLXxPwvmgNmt+9ysy2Ri824MlZjdNMwy5UqZegVqfcOXKsbpE00PnGZCKtQ4BESYft90cykhzq2iCgA28r9TluHn80LSpaWJefVZnHj7U2fN1CCgDx10Uvt2HZKX3iszQHKWz4rEszUq02m+8wSas+4kFVuSEqAw4BrnGmv0LW4h4mbl4PnX59SXJ9nNSMHMhnEO6HnSVrhcUFE7nqzi6W1FC6KgG0RiE5ggwW80nVxBeHhDYcAB9NsrUfLnIBEzO5RUsYBQmupJwas7UxhQvlKk24H9Gl55hbnnh7lYUcjjy7QO9Udgv3yHUjsxDdJ06gqBjntwReKB4ly1aEuU5UZj677yKXWSm+yd92p2zRRfmV2jQaPkEsojwCAMwkfuR8U0KQQJjPU8neVT8YopZdgEave8Unrj3wB8RYwY1fXlzG16Up8vpOnuS\\\\\\\/HH58nHH11ywSmQFNLGy9dXm5grNEwYEv+V8T+tbyg\\\\\\\/NGMPumtoNqogClUiWfCv1O4j0Az82QQ4XA9RLwa7B\\\\\\\/vuHnHESHqaIMoxlwuXvtQSRn7AXDWCsOMiOxMA7oyyU4IxT7FTeaYBCyQ\\\\\\\/vLLCMJLwk3tghCEN2zsEz+PbKc7ac3QZP4rlF5BBVOD6Sk6NAFPFyQjaI+RKEWeccqsmYfmzxX6W+gCxsydOfedzOeZ6FHE826VxdRgTruBmAUBsUhjcudjz1Qz8PGaMMjX60Ok2k8r0gVl72z3jzAEk6xo35ONYJ6Dd7dY6NnVUOAVDomSAsQmDes76ZEeBvmoqGOu8TY6xsJzQ5tmLZKRChOioRdqlStFbHsrR04q2cjUZto+WjO1ds9lDoADG21Px9++4pwTCuchZUMNGjMTfM5EdqAhXcXV4gD5OVvBOZDmzlRE5KH50G\\\\\\\/Ozp0MEvik5m9XAoSf+eqXy+gACvs0X\\\\\\\/uQ\\\\\\\/X45NK0cEyn2qbWgAPEACkuGbMxJBYnT13bW0YNeLOjqEoPvxB\\\\\\\/0tn1Fsapr54Wlmd6HV8sIOxeA6bl4uoCKQbV8QJgiETtffB+GCjrBjOwG\\\\\\\/I\\\\\\\/x4C3UWIFB2C5XhiG5G7m2z+eNpQQHa3zwAfhtWK7Yvbczw4Ut5D8z03typKGF+2LsVWXQwe3xaUthQA5t+Cm+lsU+8Pw33l3kxYomq5lW1CoFVT2xYR702PJjMJIoL4Z7xtPbd+wDo9aj6EieIZDMF9CABUkhLx8a7zu9kOklj8mwfiehYPItgLAfhW+EwjZpze6Cg42Mt844qI82En74K+h4XEV4irFLRX8GNlqyYpxjP7cZlQIn2YQ\\\\\\\/p5ZNPYrRbrO4v3SOi3vsEkw4f9OL9r\\\\\\\/4S98p6b7r8PNPOytaFU6kyWhOohPZQCGwQVm7+67vpdp\\\\\\\/0FnFbdZXw0pj2LNVaMvrlhzQKcdieDghorlSfaJLvmcIRNmilo+v2LSGV+cjw3kK3PDIWAu5rYfxIhw\\\\\\\/fUAdCU6l85E7E3ZG197rNK25oR42WJY2cbxdasSBh6GAA7H+\\\\\\\/P0S78OGVcXsnkxOvzYyeYXFYNGxN4rh0rf0bq8Sx3AIQIt98SlrjAueK\\\\\\\/F20bwfLcFvA3NY3Cdfz3W05GmRzqaDLLsD7592BvfMVI3cZcD6PohfQ991bBuGmMa6NTcC+E0IhkwtXh+22TVkbzPumnxMcYmiH\\\\\\\/+XyBTHilG5IThUZaqkYlitIq8ah42ph4KLwV9tkcEeN5TXrZlu1aGJwcqVz+ARgSoIsFVR0f\\\\\\\/s5ypGE4CDKHN8+VV9Sie7Xg9iR\\\\\\\/31BzaJ\\\\\\\/BVH63HKhwNY+bn3ccj9+PcANzGMLER1vIPxQb8Y9tWXdTrHyS1EfK8JHglQjSyos49oRQb7gKkMgCJco96KbpdpjIXBs458pEbzLpW9+MsVAseZVwfhAb5NaskldxzFjvpujE8UmRCzG1Gaq1LyCIOgQIXkMVmy542Ftv20RpLGlfExelI6LvY7emC\\\\\\\/A1DLtGAIOZL44i3TuhW4DILpDDOEZvBekedc7zeXuEbiGso20aVoTRWQgbF7KxRZeHB7CSgJFs7caU1\\\\\\\/A7oQ76DlT53B9N5PB1pA6J64FhvcKAM269TahK4ieNcDoq77TI5hjuqKysnmMg26+WRk6K3vDqzpyuHm74QA4yvn\\\\\\\/yVN2qcVpBM4NZojo\\\\\\\/UcG\\\\\\\/VHwuC2IOqsQTe4Z3pp9fd2+lWD8tXKcqMkfzKF4jd+7yfDDHCncCvJUl1T27cTedA+\\\\\\\/xxhzOegqgpunvjA26\\\\\\\/3x3X43HNP4CeoG1FK6INKSfkxZ6okzEMqiwp0CW\\\\\\\/0ndZxUxGy6qcGGdk3PvB3g3tfaBgb1\\\\\\\/EbsgFTgT46yDDVHLvhi+yqZlyxx88QgnBQAhNiWdOmQm26i2h04zdV2kL3jcSuX5NqG9CAcR1KbLDc4\\\\\\\/1hSckgLuLR5x5hUX\\\\\\\/x86oMZXm8k+FaD8h9A2o5XxfPrWcVpRdZfaNGATPRYw\\\\\\\/BOGNRtbPJZoxI5z3SRcHp8yJ7jFmDMtq2lkv5O4YcKanBWYQoKcrJUbKkUSoSwprBDDfD\\\\\\\/i+QBvQIfA1USq31jQF6BKHJHiLwOwrmol1Wl8Jc5UoYOUYgw\\\\\\\/ld5\\\\\\\/kdnFEbSBoMCwOaq9UMpGYqdP\\\\\\\/Ln0JzRY7CDTRovhpyqF1nQTUMtwJV0BVZZSwRh5\\\\\\\/ADNFSu8M9Ciyl7uZrgBebX8rico\\\\\\\/MFKa7Qy5XT9rDtsc3lBoxLtnkBEVsERDp4DWzhQDSF6La+o3T7dWKPXEPJ22SOnuJKEmREy0XocPRVWoVTj0EfEsqSc8XQbrZ3hGW93DGp4A+pbH9+Te7JgKMfY8WiZD94IoFEydlMWlz5VTiYlTPR\\\\\\\/RLQPfgfegI1wyKBq2XDl9\\\\\\\/HxL4rUyIrJpaci5dnoi5SLuINF50j0rQQgLdLk0RpgAOUmuD2dUMBpdsWorYPpkeDRKO1ghhjsEkRTILCZXKqWak0mGFkByIpwRiRK0Fc++7Y4QuYorKOFiT7lDZBrkWBpAdVsB7KF0g10cuUd46dVF+gbJUvmUPDcJ90CFFpgRnDKnlEaJgQeJ4pQgpbW2MT9IQRRWEhhY2\\\\\\\/lrtauqSKaWYqMifMwKrzlD96lDgjorNdsbkMQFSO\\\\\\\/4KTVMI5YNPgMA1wet\\\\\\\/nadHZjHozMrdxu8IyAj6H70hWZb6AAFSd3Jj0W7fA8kGlS\\\\\\\/a84WLlg2GvROATLWqAuAnbq4ddVzjdK+t46CoUHz1Lt5935j8UyrB2cw8\\\\\\\/2g+YcXDuiS8BFTemlKZu8OtQ4ST4p4fyYECOuuB4K3BYfGVFqnZRD8ur4rMvYF7GjgUoZZNCSv1MvV5zvLn4oC7pBozJsR45OKurj+SCgPibIfuD8xAVe8qNBQ78fa+\\\\\\\/9kqHHlOb2dL9FhJduSNcyAIFqeh58KXfyNd6BfXOMTWxJTEH9Ti\\\\\\\/1UAR6Hd8oYJ3xqkNCyDEIimHT5CUI12SeiU0HJ2xXyWaigAIfBHggxXJDJSwi98l5K2tkyfk\\\\\\\/wWGzRKEQRWlnF0YtaDqVopIkovcUQQ6A1ySuVf2gzRGMG6kHbUnGAsfgi+\\\\\\\/vwSoL7LL192Tg8u\\\\\\\/NwV5UlqkSZgR9uOhOReLNjUD7R691tjOEw9eHSsHStbt4ZmoBaoC61gamOAXpbohsO3vjtD4g3tEmqXlhLWyhbSh0LB\\\\\\\/rLOqTp2It+J9uvb1yvpumBG5rcbmBahHSmqF6Y0UMNNSA7cdmisigQe9b6eGFp8dRBoZi\\\\\\\/jjuTzJ9RE+GcjTWPbrFK8Y\\\\\\\/QZcDcr3lan2ARnHYljE9JBQv4L8l9WB0yOxs0PE6z0xfCEEKYHLEyduI2W07\\\\\\\/fmEwmibwD\\\\\\\/VmMsLK4FlWiIPmEkG5jjhYCI1I2AVss8mMxVlpkKUjW4MwNV7+Tl20sXTlg+ongOHfHWRmKGRM+D23OFZPSP1yqRhYrZTOhpEBbw2QRc3aZ7RGODACmFrCkmcEwykcSEU0EHtz8UISYHa2+TAVCJ\\\\\\\/NG4WkgP+L+JeJ5bUL65jLKEyOBvdPXOxvPcJgHyqfR9qXZJzHA8ro1NegTdOI\\\\\\\/Jzk4euYpsvdZFuIIPCJaqdy+nHgdEjSVK0bl3NaadBdSUB+nKdgOMz6wWJS9BUPtXIFBuRlQXAX9Gyv2oCopaH0evmYL9YhiO9TmBnvGZ1ujq22gF0lfqabXT5BUaeyiV6Kqe7RXUh31bbbxGa9x5typcBTTcv+iwuN6yp6HwFDU\\\\\\\/D1qxCG9D\\\\\\\/ud\\\\\\\/GHP6P8wEKqd2BUDtCy\\\\\\\/TN9v76PC2ZXm6Bjhbtfi0dLD7duViqH7VqCV4XWEIaLDIShZNCmDaVHVSaJfJTwBBmzaD+E3S05W\\\\\\\/u6qsINrhRV+PUcba1vwH\\\\\\\/drBsVZ2uhJ1C1dj17U57GsNEsZB6ljs3ujRoafXX3POJ8roI++aoBvgz2FXpk+CxbIJw5zKt+szfj8IQ27wjXzVjrO7kQCiGg9bNwoFHd1c7tQ1aKLww+osaKowDFCFnNboYoflvIvWuKA4oL5uYdMD7GXvIAfCWeWSHvjVOcRvu4gc2yoPVq5\\\\\\\/SxhnbUigmwG6CoRP2vpuRQNw5L6MlxkMzRLLARoymmI7tpBwSrttcu1Bk+\\\\\\\/Ww8jPLhtq0oXf3byY1d9qm5SPUs75Xq816Cce8g8yTZTOWWR7FjQhVzWICPVdyXJJtM8Rv2d11HuvJkxU1GBaxkfzT4TooQkdp\\\\\\\/gCVCI5h8ThTVh564TUEpWGViUmfyHmMR+od6sncFp0CyRz163+cFqvAAvA84GwS0zkvCkgqUEMW2goE5QCtlnvLRGQzSbl2FpVGbjo8NpklYZqpjxJgifpegSvfjzCCWKdl5oS6zlTeZvs4TDF8nlJVAVOUnESLftGkanH86kB0ucjLwmwhpIXSUjel5ngW7FC4ii24oS19Ao46YjbuYpJUbZvPVv+Vsca0mVsAeihlNq2ki1YAB6DskOEw6ZVBhIPZh\\\\\\\/9458kScn87HM4ILoNSCyvzVcKUi92ZDa+VejvvZXmf88mtKT5Azge4nduaRFj7z98Io5pvXz\\\\\\\/UR64QH+lW9ERgvRtSX9scIVDX0u\\\\\\\/1z41r5FfxpPE65ju\\\\\\\/o7sDx91Co8dzgzAoLWqA99USsgnsPwm8FwNCRKshMJzLli8kbSPZpqgxOsvlrp21xp02BnawWiMvwI0Qr\\\\\\\/HXuq0sdgoDnWj+LBMpVPprKWMfFW1pTlmMkEvikJuXgosn6yhvSYqi1pJkiIqcvQTQwT6Ko\\\\\\\/\\\\\\\/GOsUWm3MfhFhuoLs6CmWWORObmvIBm\\\\\\\/Ohew3kJI1SlrMqmUckgOdsOOoZiVJMPNokJR3NatxN2lOuvkCtnfIdxbImihfLYXpnZR0rJl60+ZPlpxIvn5T4HhTHxz5z\\\\\\\/pZd9YFVrsmc0Uk0ek5hWc28RwIW8bvwqwX4YM+n4HqeR\\\\\\\/OwJnD5Fbn2UiNLQOfLYpHOHcmbhBRFYfyC6GJmZ0SzKPukbwoWMHz+N1qaAVBa7XHQMXvJdHj+fnLiMLILj3fuM3slibjlHFcvwSmxYa7uZ\\\\\\\/Yj2F4Wqc8i08cquUHA\\\\\\\/G+U2ndg5+U7PCR9\\\\\\\/ik6qDuJvJYOGU\\\\\\\/7vcIHrpfvoFTJvQf1mIXybfM8VzFhu2lNXSLQ5AA9RBKtPDfItuManNfA\\\\\\\/03fdV6QOA+LruzwQMctvL\\\\\\\/3FX5Zl1XxAoHPQh4ZRVN0PUUvdIchjwH9FTvrwzn3hLIRRr9wdhDhRKwQumbgHMNDxKUOqcdyYOfLTuMumZjTzJibSSLhUTg0WCty1Q4crqT30MAcLzWwWHfeBuY7YJVxecNZeP4neHkSJvM+DpkIWBchBDrEt6tIFvSxDLKtCFK6qyzRO5YLj+Yw1p7+go1qJ9j2o87uMoWTyUJ3t2rnYqg7C6K7JKyg5dfLFmYtV6ygqL2Lg+Wf41olU2+PGzMSVSWKoAhYzRJ9S8zfKxo0zeBA9mzEobeuXxbBdcSYwM+YnMwfE0orNGXkr93kMEBgjixQCrzXUUCzqaHk9n62tZwDjiEHnGVQmaU9DQyt0ch9y5hV\\\\\\\/zt0nWcwb6BAYfPa5924UjcBpuI2ErEKqG8z+0lwUMbASK35BFDVeKLwZw+3EPgb1qr2xB8FTOXHAaZUlSLsVVA5d5Fm4IXV1dIfYXPvUhSifVyICUfZOK6\\\\\\\/GDEJgt\\\\\\\/LrZDLD+wnA9kfkdHKnVCXFym0iP8\\\\\\\/tdtdYAhVGBv5uX94wk2gyPEv9kCnUWTPjNnw1BswpPCMpyYq6EYiLxsP1O6WYDCm3zlWUSw49PxJbYIbmm2E8CMERDkoYZuFDMXylXIsA8i0q\\\\\\\/TkK8oq6QZ0ZUG7hBcC6KKEvpvTnqyBsaoQhm+6PtPMv8IKBe6l\\\\\\\/bcpu57Nk0z8M44XzD4NQcmAijFs07alahT8BUm2Vwh9rt0fczz80i0rQ2EAAVaJBlbcgfx1r+pTOrgbeP\\\\\\\/yKPSONG1GzxjtEdSNaF6tfiQ5WcKFbBn+\\\\\\\/6bPPEFiGTJy8sAnlcNyEX\\\\\\\/xRitmf7AaiLijIS1y2Wqfttw9+BnwGYH5Sn04PpA044TH1MVgrGS3CPCSsXbG6QgsT9i3htu4iFtT1WZW1UCpVZ0hukgcwHGflqdRZoras8HJz3vekf\\\\\\\/dUVSxCiDbnqxUeNYJHwkmnfvquqDY9wpycsxjYD9udwX3PysRMzrpW+sJe\\\\\\\/d\\\\\\\/Ons0ImkIMeX7C129djmU\\\\\\\/qsi57gK\\\\\\\/wGZ5oO0mShNWSGTRQp2Gi78hQM5HwH29IE8GMHezOppBBbW472UxAw1SskTO2BYSK9opm\\\\\\\/NR2R+wAWLSUu5\\\\\\\/fAKU3jWy09skdYKRTHyND6aFIwpfkTWlhGzncfb2WcHyZUXn\\\\\\\/dVqi9WZ+PnmmQczNQvoiJ0lLBVJx97RaPk1BnyLz\\\\\\\/viyf50oXOwR7TAu5JQX4qzLsOBz58TVD1\\\\\\\/SPJh00hbXPQ9KoDGmPg1zHd7rsb+9a+U7+k0AiDqXjjDudIDV1Nk020SsEn2CcFScEG\\\\\\\/as6TaHCER2ONfrJqvB3Rmknd3ZNE4R57FKUYS4aKi2N\\\\\\\/FmVcnMSfO62Wdq9ugoMJn26Un6WI9gxT7O8zt+e3gTE+LK0rRaiPt3RJWs8oPvenucqe+BE7AADW+mWqskI9PRmZeIIQpYeDUo37raGDbFMRb9DEtjZ9AaIWi3X1jCQvABBRRtwQP0xwYg1LKOT+MAdLXUjdtr9zBYKEFUfP9ids8yW3M3nAu2jf5OwlAFQ81\\\\\\\/IX\\\\\\\/Z0QRmjvk3ptDT3IDqdAp5osoKU6f8zGZtfrH6HDc5cxaZFmGgou7Z+2yhST9YEg0htep3uhiJZVY2mXNagGmQKwDvutZPDE0URpBpJqVac9PVMdvlZlbnK3eTfCx7Q7CsXegy1HDOzPSUkiEkS1sL9lUHWZWwHF2KeE7seuFlGzUvu0qIMl6e+RD2y4R+65jW+LMZmXw4CA1z7et8CQWVye5jH3I5JxYS9xyvzYQmAdDhYx2WBRyHWx6cvKYiJdGVdKOtH23faAyw\\\\\\\/t1YqBEUy8e58\\\\\\\/McJEdpXAHxi0SudGMJWKNCdeOwquslhcvRZ22dapJ0VxidhvJFLAexFEcYCXZOpRMpSse4h5wdIOfE\\\\\\\/y1OQdU+qx1GEjSaqVdjek47lnJyaJ4dfnEt\\\\\\\/23t6+p+1lKZmSYTkXw87E+dr4E4H+6rQbiizih2pKMzC9nDfJ8OHBLT0\\\\\\\/+5XW9f9\\\\\\\/DCL7semvffCaFZ0T\\\\\\\/TOXPGo8+Zk8ordka5eqwnbke0UmGpeHm+4eixoYFZbmq6l3eIhnMcJffh7QkGiA8eLB0baQlQVupWLvsA6O96oHVTlPRlRpMeDT7pSbZ4nEhIAgMGPFiQs34XxaG2g0Pf7jXatySZnOyNDGBWwvOSDzzPjRWDvp30kyEE2JKZBnFdOr30AJMCqT+tMyhRbpooPcN8hnVUhdATCa0M8pOjx6WoYzw3IDBaNcX8ZxmzDEiO5agebtrxo+b+YJbAY6Q1YB5+E8TuPFA8YsQrPFsya3nOEjcWNDG61+DFiV5Nq017pTAIgagY1KWnUE2H7VCEfDzzPrA2zU7gvT6no4UO4OmhMXlYW+J7ApoeBfB3hWySGPK+Kpll+KOwG8lvK1bA1TEQAcvlHetzsD50vtZ9C\\\\\\\/fzwOPElIQGEkUSoq0ByiuQkVHKwAKYVS3+7dxis6HDyd2hWQBvnxNSHuKGpQqjJud9RXyJAzXRc8cd5MCwNh4qW4uWZeoVq9++CFsFJ+4GzL7MWUdv6YtxmWTBSNL85vLBtCTlxcGZPnThQFUentv2bnVgBVjvZQOPWubrIAoUlMYl\\\\\\\/FsVC1EpneT7F5fPZO7KrEnHPJVWCQY1HcPh3K0vPV\\\\\\\/ybEJfq0tYFBjwafVjrxckxjdUnwwaW5Hz9gBTrtKG2sdGz1+DwzMA\\\\\\\/UGBdtzPEJEFCsnku6wH9pXGCdh6Qf\\\\\\\/t6q2KGAZHj+X1OeouWdKs59bn2R5HHIWxHX\\\\\\\/8BjOQBVWNb5PLqH9etLO48+PcweiLrt\\\\\\\/IcPlaOe8T\\\\\\\/qHxESK0hNO2CW\\\\\\\/SpVCXWXk5xjxf3AQo6ARcQejwhYUSW8UADBiPRjfSmyrADRqw5+20eRehXimThVecoJPouwpMI3IH4ksCOjq4Hj+41K0C+CcitDUzC1rOP0gFDOzsNdYbAyApealwQRL5KpcAWKc4xcPBS9yMyfHoqRGJY7Uo0xS5RmOELphz\\\\\\\/AZnmfGHTSmSq7APlLvbxaqZxfLXIjHYauFzokgHPRNt9ap4FP3ZmV6n++FlVy78Y4XetZ+\\\\\\\/6JOqnIDM610rKClUoaCTgweilq+XNoptJK3SmHlr+4tFeAU40VbKra6XVWMvrWq3AIuGPbNnpc55S1YGbHBJb9tRMiXAHv2KtTo0yN+i9XY5zUM4hR0ZROtFnqHnF3A5KcuoRtLsFzcMyNkHiIrn09vpG4QEL1MggGiU6EdN1sdQpQJBqgvKF+UOas\\\\\\\/mn3AYiTaMHhk\\\\\\\/JeNL\\\\\\\/kaQ9o9HlCxMJWMRxoWd2VDz\\\\\\\/yirFLXWawuxGfGRbixmfNP1LM6OPEpOKhREDVUOOt9iaHgY19NMrl7UhTLju\\\\\\\/M7NImknRQ3bEE9zUPXkTfOmm8G+Vfgj\\\\\\\/9hn\\\\\\\/GOAoxc4EaA+ynmszwVlkaLVS872lGsEon39kf\\\\\\\/M9OIcHn+zC9LiA74O7mlV+xSnzhuGaUVZyaD0MBtB1feaYiEYA5qDgJS9c9ANeyutimX6p7ypR7RtNZ9BNIrzTI2yPgGQaKHvwUQ7g2CWXKlkJfeUp3ZS6s66KL95mkYe3zO15Qodtcjz+jR\\\\\\\/ibJzx6YXwDG0f5teFmBdbPSSh0rlmD2cI1olF3Z9kmid3twIS3EBtMnJbHSiAx1iwgs234bbEneMeUwQYibl7coMLuYxKmQgbJA7qWusaTDZb86h02G2KyLS3aB2z2xX5p3uuj5eRnS0EVuJo7eTqDOgP\\\\\\\/pUr2CBU8Cey1dtQTDThZB9Wcl0GCYFBY+bnuZfkhcMO0JIwjCifmZUR7hDUw+CTR3ELo5OnRDAhOZCOahJkTmf67aCFbfIH6zemYg2fUb2saQM2T+biqBm+30MAa6LqaW\\\\\\\/754nXOU7nDIVbXq0Oewx7vQTlLpgXseTUIjY\\\\\\\/bwjiu5SxRdL2RpeUk35aewTJQJnntH8CuEDfwF+NO2mSE5IyE7\\\\\\\/E29NdgDcSop\\\\\\\/wqI7YgEzwBjiaUqocndPFaF768BAb9m46N3lCI7lEZZU63stIOjzmNGlrtOx+uXvpYwL8LQ+3\\\\\\\/d15hB2EQImmzj7wxGY6IYIYgq7PYrWfMe\\\\\\\/eyMjylbMXvnbT+xduUYxOhH8l1nZobxswipM+xDKIZIbm29qYBID0fUUbqz190YJcZI5PIlHEBcosJdC9QS\\\\\\\/IN01v3vBWzfct\\\\\\\/MG8YTjfcb081BmkFwuvf+5hiTipqfejEhltVujKsmcZ8HNTSFBe6WdIUVgPe3mJTwMI20eDWSDWFV1n8A0aoCl8t28\\\\\\\/uk3xjj4LiRQ0eDG50tIGxwRc05R2Jp+HfrcsXXwBwlg0kHMEnkpCPj1rgBo0Id\\\\\\\/2lewvwTObIaNFibO6ZuszCwE24K0XQeOdW8YU1I5JNmq0ppKC4TsBuoD+COh7acvwXnOvLObX6Sl4fLmdbxDJy9dtu9wmJfwC6ihb073w82yVacqC6\\\\\\\/EHDUpaq2ubEO7oJXT69r7OQJvC0Rsr1a6Eq4+6RAgY+ZeKX+Q4e50xtlvllUom6FB5IPg7aux0f9Xbitrl38j1\\\\\\\/daKposK+Coe432kVz+xXJX5ouisf4h8wfb1G3Ok+CmmidnKHWn0w505afapbCPvm9NH7E620HrJaS12Bt6BaDB+TOQJXS+H4bXAxxEOYq3UgKTQHkPzrBnbJ7k4wh91UzrcHLNsphtrB\\\\\\\/Lm\\\\\\\/uvhWjOyNPcKR+YWxFMoV8xgRNPeLwqo\\\\\\\/9TkDvkdA4OY9nX35cJ4g+7ijZGKUedwlApcSPeYzjvu\\\\\\\/wrMVz3jJZ1zI6QVYgzrh1dYzHFxocNBslbebzEYSq\\\\\\\/j\\\\\\\/zx0TW+qpNw8QZq+xEen+sv9+4oR6gI3Dcv4jMHgqooJ9fxYZMywHb60n38XG\\\\\\\/QZCZjw+slzoL9\\\\\\\/gd21e8JUGUtqzMRGCh9dIQZHhoak480r94ffoPe+Au2ZBXX4ZRoKQoJ68M4AeEf4+0PoaHGNaN0KfMGhJN9e6Sl36QgdrHmL2P+UNPb\\\\\\\/JHDd1kOEqiefOUFmpJt4Qivxsd7X1qFn7fz7j6LgJeZKEMrfCIjU7qhiEbvbgFqQho67Q3iXPNoda\\\\\\\/KqlPFgkDlPaCNs9zvDib5oZ5XYd9noAO6nUTqAbawDy3NrAayrTi0gfYMMZmseeZZf0Sw9TvPDBfXNbFcmlwZwQnOaylkiMA6k5A5AwKOdzecvYPwo4+G5A404\\\\\\\/8YgFl6tWBsj53wPAzn0t5nxNB2mS1bwRe0fMGGTplI1zzqxIYg1dTTDHK1XyYE9MEXcRtpflGocl8GwIH\\\\\\\/46saupSU1zhLPIrUuIamI4QIW3psrJm\\\\\\\/4OTeErlgL\\\\\\\/PKCeCc6Lq3lVsl07onX1dDylAk6gwFZffpYEgWYY4XkKtdLAXGlz56x1zIV3DzwrxLeO9S8JEvf+Hyu3VxdfAD+r\\\\\\\/B7+s\\\\\\\/VLN7aZlinrFJ487n2GGGJRfJpbxZ37jtf9fjYv8NSZzxpFOgI4gD62GIJpofPp3snxDQ3yPMpP2tIA0tHplucdKmWlQt1v0Pl\\\\\\\/iiDCcWEtr7EDAihTojJVbG6otrccoU9JnR\\\\\\\/XZ8Gt082UnMhMqgdGotwA0EWKJT3xdSfjd84PeeSeMZFEeqSH4XoRYgkO5Qhlnd1vQgeS9exjWlmzzf7skwhi1doT31oVwBZN7POV1tKCM3KYIKjgf00ztVE9ex0FjoXfOCxkeZUWbEINffUvAMOM4XKhh1xBkhX7Bm2xTiOxlxY3Aj4hG8ZRHJV4iP4XuVD07HIUq3T8xzcIzuY1KgHVYMaLRzr7U833Sh3r3jxlamgtSs\\\\\\\/PIFzTJoWd92Dm2ADMD9LQ3k6umarOKYlVjoQbe3KyZ47VHCQ8a77IeAngbKGbph5RPNBthHmkYGWrRKBzq\\\\\\\/DOZu3716FqxbJex\\\\\\\/WyfVhz81oqMeFDioNTbtTi725TVBMEcaLlfE6Fn6c0Pv\\\\\\\/+JhI983zC5CWuI7OJsUt3wa5+xj8VLN0kxRwACjRUrnH4aFdf3UQPaj5UcqFc\\\\\\\/RvQWFmR+SoR8IBeotLOs1AMinPzNI+lUjUNX0S0VQc\\\\\\\/QdD6pU36Y8c51BJSEWvaB9rc4f4h4tdMbBUvQJrleDafWp7JbjQ+ko+Z1d62XXJDpxsI3XbMHE71GFbilGQ12ugYQyAFPkmlkxPWBz9VELOGg1TtxYZ6tSFOhu\\\\\\\/wV4p5UK++wfzblFLQZV2XMPtjCywguwH5pWD4AXnza3u8JM4IbSfRTaVwNOTOjA2qKnW1AqIL6gJYKH22fVN5mhd6Q6xBeG9ibDgmPz8f0lVV2pjG\\\\\\\/vd5JVGW0pqJUMbGqytDTLf3gAYlr3BywbNKwrXNulPfMjaGqSAqe8mu\\\\\\\/DMn\\\\\\\/0S\\\\\\\/PO42ms5K5FE+7eiIw8EdVFBqx\\\\\\\/NFX2aGwDS3WvwI1WHskTx\\\\\\\/98abDgs0rRDJ4tganmUtD6JFiA2pA+LwUwYkDEBh5r8wNQZ6uj8p6qvcBaM8UsyzB0EBtt7IQdvVpM0gF4q3f8NL0tw7Z9UXIPCS1z+4nrK3cgu5a\\\\\\\/NodpNtd2aPV2e0xgT3xRIwjY9gLgPxg5Slz4MIi38jL2RM5WNb+hLV\\\\\\\/wb031eBLoHZ4CqGig2sa38\\\\\\\/v3tWGiCtMRpuBCecgEEl7UaoTmqtZylAvq7vf4mTUMCdBZU1\\\\\\\/pwkturjnydRVMuCkXWa8GiIvph+PJ4lCmTQAhUSM1xN\\\\\\\/\\\\\\\/0wtuTm1DWd53r7D9+QGDBkztOA80dawr43a1Osjlhkr4ny0KRapZeCNm\\\\\\\/mY54fiqKrxqgs6G5wnf0PD4YmTrNf33Fk4Y5o\\\\\\\/SSFKspOncAJBC8HfHrEWnAdRHiPoneXdJf1Jf1kkR5bOgklGP8CO5zQizPp+BloOFNV5IllALcsDLFr8hEPguu2U3sn+g1D9kH+qm2Y+YwbkwxL0MuVKD71gldSQYe3ei37t0Qsyc8SQSdG\\\\\\\/8WfidJeh8yQcCbm6CGbHTG6G+PunRZKs+qgoDEYGyky+n+1JbEs\\\\\\\/lV8Hv0LZNaONTEaAnrwfTnbTamNCZ7PcubuIZ0FJHcVc0isbuOIQW0OTPZCZiIM2g02YprRqOZmMjUQmvkhwTlrM3p7gwm7o5AOAjhNtzsnuPhMkpU42R6wmu10H2TTsRl2ryiHdy2xkv1JPz\\\\\\\/GmAjdATwYloOMRECaCzNSokhtBwfvrzRvhSggdxhqlGMPGMh7uR7slqpl0lU83sd5\\\\\\\/MDpx35L1\\\\\\\/0HkMAWPuYLyuByhg9lT8TpZPwgck9daHN2BXdJ7RhHyYYTudtkMjNfvpr7WdGETUTARO0J619iVvrE+Ya8gjJu4nIhql\\\\\\\/4mndM5zdMNhI0nKK643JW08M1OvUcGyR4+a5y6HfvSjqJuxDcTMyYV5QbyOo94CQ0+jhlomxfXO1hYkwyNSh6cxisVuuT9dsTFgEv7H2cnbpetkZIe9dTDcCEn5ken9mXbFIWQ7RWXiW6\\\\\\\/eqMlKWNtYlPwIats21Gmkgew9AJ9a3MhqWCnanP7hWPpPAOU7XSMmsK38XMGRnDFEa9OVRhdDm8phaQ81shtC3O6OWjXalw9CMXhxYVyADsqlNvwPUesipscvbfs5rltrSvtF3mzBR6p9yOOh+\\\\\\\/yFgcAdq537cYUxFXspmdgTYEPh3M2PpInNwkMMGp8z5rDEApXn8PylNC0WyAtxGOP5kwuxG4Fb1kZ6KJz7roOhR53OjLd6htMZq35DS12k6iX9b1nQocBgehtAuAT2sURPiOlhhUhqHODH2ajgY9PWK01Am0uCzXoDEgXSYerCzwhjKZgNJ5sUZiQkQHmQhU\\\\\\\/uSNMzxQGw5RjFeS02xH6m+NF83SsB1VJQfO7tyRZALCPoLI\\\\\\\/aDO18gYWR\\\\\\\/gYQCAm48QNQqEF5M68eA6mNDzfSkVTq2NIol9sxrhvNybzugpCFPkjWtepOH8\\\\\\\/TevXENoWmEJNv4O5F0xOx7IzH37Cf0WjtgxLla4CqhOVYVt3oHtbcQid03Qpmm\\\\\\\/jlvnqUEacpnhrerawdGLxNJHvWz\\\\\\\/2c5tPWFZOpdgOIIAGKTQy3ZiqmvT89WUjewwbIblE8ltqXGefZgCNRSmYTOo1VtoUGWy9Z1cKFURoJwIM1XAhB0e3Faf8brih2jp6Z4\\\\\\\/ZGyp5Mjxhd+lxPfYhMZMZr4ulQwvkcDU4dHanwh809fKIUhY\\\\\\\/UXr+Z2SQx0X1wLaRXPAa6IsmoGYMDP6gH6NgGyAuovXxDBuqgd5I2bprdUl8Xw\\\\\\\/2S6rVUItRSv7rMyTeh+8ZlMfIFUtqgtWijJqB\\\\\\\/K3w9BYb7yiqg\\\\\\\/0D+npXoSSpRe6Z01iygLHNmW\\\\\\\/SKce3avlrMl5z\\\\\\\/bbo1KBe\\\\\\\/T6xa8st4h+9vJJAdwj2sHyH2iJ0qxJcZehbYKlKYG0cZqK+pSEJLxdXBRdu5\\\\\\\/\\\\\\\/S7j9ofqMmWt3guSgKA0SbcoZhN0jSC0opTjysNmwE3xqX4Iz8sFMcU5ArwR8F5B+8lhGcASqmoCOyFpTdAkv2yCivWdbvvCmk6TQzp3eDJ7FWstlg7r7lrTlQmoSJtvM+bBPAt+dVbjQZgBWG0IvwFyjmThOv4tWmS5T8CS2HbO1gQ2iISl9HklItti4lsKH5QXpgQ2WcovwtESInRmRQi0jpR7l7hr\\\\\\\/zk48z745sg2iqV7sjiiYtKHutcDcPaM5VR6wVgZOLS1e5R4vExHPDX\\\\\\\/RDuB7WCRTha6XRArGUoidXh9X2N19opEtEBYzQ2AqXh0cl8qZKgr65z\\\\\\\/1d6evVv61tfVlq3XAMi9Mlcf4ssbL\\\\\\\/tWf9OdwRbz2y1\\\\\\\/iKxELgbZVPfcLtK\\\\\\\/CjdnSM6tM1kP\\\\\\\/hmU948bDE4DJYY0ZtArL2m50D0bnvhmQ32ZVp29Tt+Clk0xtVp8l7ps9hByW+jBqdg3UON4mpf9YiD+FDiH4DzSLK6TArNn6+1AE+ZMFSpQgjtBQV3dM1SoT0vqSiUIDMF7QVhuOL3ucGZb1znV1wqIDqhqm2Rjm3ugjbSJOj6pOZFEY9Nol8KmYJfNAI8Hw2V9a730a6+s5QVVU3g4ybdObAC409gZERFeT\\\\\\\/IQZyHCRt9ZB0wBKPVWKj8O55Co8OYYOR4Znjvmjk5tn3YS6FlowfxlThfZyCqwg5RYE\\\\\\\/lFGQunpoKLQduJ58FfTIJaU7FroZuR7q1WKAX5cFl0aYU8iAImBTDHWg53JAfyRM1VgokbA0Ysd0Ac\\\\\\\/Pen\\\\\\\/Ew+ejuEpxri6LJE4rX0Wh0fkfjs\\\\\\\/n14C3xMkIq2Y0hFb6DPq5dJRPRbOW4q4+E\\\\\\\/1sBSYYow2j2\\\\\\\/7EIix33jNr7yJthctOiR1KFBaMoDLi5v78Y0fkZ1cAFh+AmEVouJJUTFPfosxVFijIDPrLoQmO1Uqo9OHaI4u4YCKgfF6+RcSbAV2NdW12\\\\\\\/grk4dXgo04eQGlEL\\\\\\\/S4bjNpk4GNmeqkin0PJnEdAp2kmoYgCKUjZ+66CoRyPWdfsd6OxTJE0IqMDmrqXYHx20KxfrQ3HqFHejbSe5yGYSsnuFhnk6EbtUIEhu4SvkBICAcsHkAczlxUkyDH5PC6q+0AzZK0Nba2DtybUJ3sZGgYjHhVNX0ilNl0V\\\\\\\/+aEQ6kbE+Zhm+hBsDMFDFl70vh2MGUo7x3h9JGp4tcr3saDl\\\\\\\/dKccP36TU7ubgNA+RtN9gFh\\\\\\\/E35LGrYHxIX54P+UlfcKPFc8uok7RPjXW0hVmfrOyGpU1w0JHSXoC+WrlhG0lZOeXMrMNkzsEcRsq72BhM25K7XGJoDkCva2XU4zPXg5+GTScFhouK0p+w6g85ibXV75emDt8ZLz7vr4AHFgB0weleD9GaB++\\\\\\\/CcIEmw8AcS+Lkzi4YINK9grpPWWI\\\\\\\/0t+gpE4ksR1krhmPRBhEeokkzNNQ8LOzE66+XsgJkBCXoHSo32P\\\\\\\/UxCiL\\\\\\\/QS+NyciyRVn0\\\\\\\/GnVsgnA8eBdPAYIi+sMi9tzJi\\\\\\\/+HI577rPCZOGpryZcHY4d+9rfoipcbKI50wuPi5zjtlmTuF8V\\\\\\\/37GGRTlaeyg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tIkk6eydg\\\\\\\/JOc2w5hW+5lA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d65032c6482b9638201e6e6a3320f324d9121f38891cbb3d5b7030f698f530c6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226pew8bpoAgWUR4vwqWwElL3Pk+63pwuD+jEFPOn3K9c\\\\\\\/DuTji6CA+yCXiXk6TGsGrS81128ahsmaDM02T85zyEx1tL3xmRaUGaDa24r7m0e+w3P7hKBmW8rILiQ184fdL54GLk5BEwLThKgG1VqEcXu4I8WJNPq\\\\\\\/GL\\\\\\\/yCuoBNgXzuW\\\\\\\/FDbHEquyLtm0mndGWWrUiEcAHNuI42vBWqbIfqS43EmtjKgyudnHsXCDkZBhqsmPcVl460qy\\\\\\\/fcX\\\\\\\/fcmjXE3bq0PhyBt\\\\\\\/d37qW869B+B8peBVDUlaJd6oCBHpWk4\\\\\\\/v9+5Ugi7586w46imRplXt3NXGzgcyKpn9kw2kWZpNO21GQZaD6XV1qdaPfW3bntsK7WWbYp0sTIqTeIenBNCClzljvdrTq41CgjSY\\\\\\\/CD90vvWz8C404MXPvXkcM31swT6zvb9Q+8IHLFNcw\\\\\\\/0\\\\\\\/b00FLE5l0p3mKz4zhSfB2efuMcHGLJyUWJK4U+7ArkoUA+33MSULzbYO\\\\\\\/sujJxPFxU+6kDfVpEryFhmaTPySZ6kfA5h9yo0+jlaNq5iZA\\\\\\\/PP2S0S5zKsZlvjvm7KhaJZz2oxgRzioGuEO0UnvNFemYRjHJ7oIfqhrpgm419sSwsjtU\\\\\\\/Wz74rP00oYKoClKtPBw9EwJ2LAP8VVzr\\\\\\\/s2RQoCuMcHvefm5MR7n9sZ1UEcSxTor3hkfrZSqzgApWLejbSkwNHlXAuDf57nsTHGIF4CqOtR2gCRem4aO+rKbFH3wCo72+9MM1ra6+GEUl+UisE4ma70vPKwtg6Pr+i5wV1jmlLXT0f37TfkO9ufi2CFnVMBxnjT6rFZxP5\\\\\\\/ujK+FMO2NMHimEgV4yvDiQlYPPsrf+tNlwLytIpgh8BY9kxGfqvQZhEM+UmOlN4txMY1kQPvsSEpKEBietbcyN2HzepJpGGr35HcsekOosR7upXXHMlk8AR2Yn8OXRkgPvCT0w0aQcAwUQm\\\\\\\/IdGn3ZpRRKiUi5vxJ5BQvEbKiUH+P0S3I8twnf+7BVJe8a\\\\\\\/DPuw7VW4PSLbJNjIlilZvE9HOo0J1TMGnBmu5ZOSZMNqZ1mqC96NTfYHOvQ\\\\\\\/vo+kdArUQcfGgtPYjmOt8w4LOffO28SAqXoKlnPj8mL41umBchqSvke31sHVON\\\\\\\/J1\\\\\\\/p33sbKZUidcNND\\\\\\\/3FCqyw8OeFVBRbenJeeXQXvop1t3a8KIPb99slJ1ovPfT80b5cCXIgmHOe0QERnqi2sW24NVkr1CT3WJc819SxqYF\\\\\\\/z3fiYWXIwwDMFTJmOfC78elnD1FwpYBwoy99LcZw6Tt5mrx5HOGQjdBPvq\\\\\\\/EfmZ8YPi8DkDC1D57+qfdHfXGB1X7Cz9Ur5BThGTXJpv+WH\\\\\\\/1jMVnQPWk5yyH2uxXlUKs6PKYpxjKGmiBWricjp1TCiJDhtsFgc6Yx6ZesUyZeM485sySwXPM01J0uZQGIlmRx5Q36igouHdL4vRRuD9Tz4DjEWU2ZO\\\\\\\/atXP29KmUwae8ViHwQICGkV7\\\\\\\/IfTb0mrViL5Le53uapqs56f2FKXvsHfXGhSS6TKO6VdQ42LlaP+YrzxO3DmKGU9dTCnb8VvN7FW6cQH+N1cHk0rRA46P97KPHNm96w2\\\\\\\/JiaUotTHjBz8sbpVzvXhiUt0O8GElCSX4Ae2V2d9XTipAZZVyzgw4Qx8Jy7e1GjeSDi\\\\\\\/J7j+EgBNIZ5d9OMpxUSjmg2NDDbQ4Iq5nAiA2q2o73V\\\\\\\/1YQyDNqXNkpVDiD\\\\\\\/vYUO17vM9QDEorGg22S0G6hisZRRu\\\\\\\/WToZHqI5\\\\\\\/Eu21W\\\\\\\/N1gd5rQY9ia8GET\\\\\\\/ECOhmc2tjQxsboZ4W1AZjhm+NuXm0uyXrF3hSVwxOc7Lu+m66S1IHejhGpjuqNzEzO3tdEsaBoKdDe64scEhwl3qmBA7NhyNjJIjnr3k3yASQpJmgi9\\\\\\\/s+vEXrFn7Xtjjh04pjcqdPmykjszPYch8uVjOjYm+\\\\\\\/\\\\\\\/c\\\\\\\/DsxuOoEeesY03EF3eChj+1LANo+P7M+gW\\\\\\\/t1n0+sxELSUjJXJ8kfSxXIOg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022N7mOEfvV4ZoEstxgC5qcbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220bcfe88cd754fc72c069873794c7c73fdeb09cbfb6cef44d7b3efb89a4f3ce77\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492493 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/685" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WupSPcYcbEkGo9BqbdCjhs-FXiEtRDXBAM9JJ5rjeH4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/685" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zf33iqh0HtUoyYORBOaK4bzEzioSbf5Ai5SAF8Wl068" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/675" title="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/699" title="Genetic Effects on Dispersion in Urinary Albumin and Creatinine in Three House Mouse (Mus musculus) Cohorts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492493" id="node492493" data-pisa="ggg;9/3/685" data-pisa-master="ggg;g3.118.200613" data-apath="/ggg/9/3/685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">685-698;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200613 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492493" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/685" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492493" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/685.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492493" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/685.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492493" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/685.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492493" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/685.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492493" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492493/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Whole genome duplications have played an important role in the evolution of angiosperms. These events often occur through hybridization between closely related species, resulting in an allopolyploid with multiple subgenomes. With the availability of affordable genotyping and a reference genome to locate markers, breeders of allopolyploids now have the opportunity to manipulate subgenomes independently. This also presents a unique opportunity to investigate epistatic interactions between homeologous orthologs across subgenomes. We present a statistical framework for partitioning genetic variance to the subgenomes of an allopolyploid, predicting breeding values for each subgenome, and determining the importance of inter-genomic epistasis. We demonstrate using an allohexaploid wheat breeding population evaluated in Ithaca, NY and an important wheat dataset from CIMMYT previously shown to demonstrate non-additive genetic variance. Subgenome covariance matrices were constructed and used to calculate subgenome interaction covariance matrices for variance component estimation and genomic prediction. We propose a method to extract population structure from all subgenomes at once before covariances are calculated to reduce collinearity between subgenome estimates. Variance parameter estimation was shown to be reliable for additive subgenome effects, but was less reliable for subgenome interaction components. Predictive ability was equivalent to current genomic prediction methods. Including only inter-genomic interactions resulted in the same increase in accuracy as modeling all pairwise marker interactions. Thus, we provide a new tool for breeders of allopolyploid crops to characterize the genetic architecture of existing populations, determine breeding goals, and develop new strategies for selection of additive effects and fixation of inter-genomic epistasis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/allopolyploidy" class="hw-term hw-article-keyword hw-article-keyword-allopolyploidy" rel="nofollow">Allopolyploidy</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">Epistasis</a></li><li class="kwd"><a href="/keyword/heterosis" class="hw-term hw-article-keyword hw-article-keyword-heterosis" rel="nofollow">Heterosis</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic prediction</a></li></ul><p id="p-3">Gene duplication is known to be a primary driver of evolution by providing the raw genetic material for gene diversification through sub- and neofunctionalization (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Haldane 1933</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ohno 1970</a>). Whole genome duplication events, in which an entire set of genes is duplicated, occurs either through duplication of the same genome (autopolyploidy) or the union of two closely related genomes (allopolyploidy). Both types of polyploids can exhibit non-additive genetic variation from the presence of multiple alleles (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Segovia-Lerma <em>et al.</em>, 2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Birchler <em>et al.</em>, 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em>, 2017</a>), although how these non-additive effects are classified needs clarification.</p><p id="p-4">Statistical deviations from additivity (<em>i.e.</em>, interactions) are important contributors to genetic variation. Homologous gene interactions, also known as dominance, are deviations from an additive expectation due to different allele combinations at a single locus. Non-homologous gene interactions, commonly referred to as epistasis, are deviations from an additive expectation due to different allele combinations at two or more loci (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fisher 1919</a>). When epistasis occurs between non-homologous loci with similar function, such as across orthologs or paralogs, these interactions are comparable to dominance effects. If interactions occur between homeologous orthologs on separate subgenomes of an allopolyploid, should we call this epistasis or dominance?</p><p id="p-5">In classical hybrid variety production, divergent sets of alleles are intentionally isolated into heterotic groups and then brought back together to form a hybrid. This establishes heterozygosity (by descent) at all loci to form a homogeneous population. The union of two divergent suites of genes during the formation of an allopolyploid also results in a homogeneous population, but heterozygosity is established across homeologs rather than homologs. Diploid hybrids lose heterozygosity through segregation in following filial generations, but heterozygosity across homeologous genes is subsequently preserved through selfing in the allopolyploid (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mac Key 1970</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ozkan <em>et al.</em> 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abel <em>et al.</em> 2005</a>). Allelic interactions contribute to dominance variance in the diploid hybrid, whereas homeoallelic interactions will be present as part of the additive by additive epistatic variance in an inbred allopolyploid population. As such, allopolyploids may be thought of as an immortalized hybrid (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ellstrand and Schierenbeck 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Feldman <em>et al.</em> 2012</a>), although it is not yet clear that these exhibit a true heterotic response as traditional hybrids have demonstrated.</p><p id="p-6"><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Birchler <em>et al.</em> (2010)</a> note that newly synthesized allopolyploids often outperform their sub-genome progenitors, and that the heterotic response appears to be exaggerated in wider inter-specific crosses. This seems to hold true even within species, where autopolyploids tend to exhibit higher vigor from wider crosses (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bingham <em>et al.</em> 1994</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Segovia-Lerma <em>et al.</em> 2004</a>). Complementation of deleterious recessive alleles (or pseudo-dominance) has long been the primary explanation of the heterotic response (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Stuber <em>et al.</em> 1992</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cockerham and Zeng 1996</a>). However, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Birchler <em>et al.</em> (2010)</a> indicate evidence against this, where purging deleterious alleles has increased the additive value of inbred parents but has not reduced the heterotic response observed in the hybrid (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duvick 1999</a>). Complementation also seems an unlikely driver of a heterotic response in allopolyploids, as the inbred subgenome progenitors would supposedly need functional copies of these genes to survive.</p><p id="p-7">The availability of affordable genome-wide markers has sparked a revolution in selection on additive variation through the use of genomic prediction models. The additive genetic merit of an individual can be estimated as the sum of its additive marker effects to produce a genomic estimated breeding value (GEBV) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> 2001</a>). When the number of markers is large, marker effects are typically considered random and normally distributed such that only one parameter need be estimated. Alternatively, the additive genetic covariance between individuals can be estimated from the same genome-wide markers and used to predict additive genetic values of individuals based on relatedness (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nejati-Javaremi <em>et al.</em> 1997</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Van Raden 2008</a>). These models are equivalent for prediction under the same set of assumptions (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Garrick 2007</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Van Raden 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Strandén and Garrick 2009</a>). Genomic prediction models have since become popular for their ability to predict the performance of genotyped individuals with no phenotypic observations. Selections on unobserved individuals allows for reduction in the cost of phenotyping and breeding cycle time, increasing the rate of genetic gain (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goddard and Hayes 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Heffner <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jannink <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Heslot <em>et al.</em> 2015</a>).</p><p id="p-8">The potential utility of genome-wide markers has also drawn renewed interest in non-additive genetic variation in recent years (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jiang and Reif 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Huang and Mackay 2016</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> 2017</a>). Genomic prediction models that use genome-wide markers can incorporate non-additive genetic components to obtain better estimates of individual performance than based on additivity alone (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Technow <em>et al.</em> 2012</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Jiang and Reif 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir and Jannink 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akdemir <em>et al.</em> 2017</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wolfe <em>et al.</em> 2016</a>). In outcrossing species such as maize, prediction of dominance effects is key to harnessing heterosis in unobserved hybrids (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Technow <em>et al.</em> 2012</a>). In inbred species, additive by additive epistatic effects have been shown to significantly increase genomic prediction accuracy (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>). Epistatic effects can be added to the prediction model by extending the method of expected epistatic covariance estimation <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Henderson (1985)</a> to marker based covariance estimation (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Jiang and Reif 2015</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>).</p><p id="p-9">The use of genome-wide markers has allowed for the partitioning of genetic variance to specific units of chromatin, previously infeasible with phenotypes alone (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Visscher <em>et al.</em> 2007</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yang <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo and Thompson 2016</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gage <em>et al.</em> 2017</a>). Allopolyploids have been traditionally treated as diploids because they undergo disomic inheritance (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Mac Key 1970</a>), such that the contribution of each subgenome to the genetic variance is ignored. By assigning markers to each subgenome, an additive genetic covariance based on each subgenome can be calculated. Using these covariances in a genomic prediction model, the genetic merit of an allopolyploid individual can be assigned to each of its subgenomes. These subgenomic estimated breeding values (SGEBV) can then be used to identify parents with complementary subgenome effects for crossing.</p><p id="p-10">Under Hardy Weinberg equilibrium, subgenomes segregate independently, and realized estimates of additive covariance of individuals based on each subgenome will be independent. However, this does not generally hold true in breeding programs, where population structure from non-random mating is inherent. As a consequence, the estimates of additive covariance between individuals based on different subgenomes will not be independent, potentially leading to confounding of effects from each subgenome and problems partitioning variance reliably. In an attempt to circumvent this obstacle, we present an approach for removing the largest sources of genetic variance (<em>i.e.</em>, population structure) using singular value decomposition of the matrix of marker scores.</p><p id="p-11">Common wheat (<em>Triticum aestivum</em>) is a staple allopolyploid crop which has undergone two allopolyploid events, resulting in three genomes, denoted A, B and D. The A genome ancestor, <em>Triticum uratu</em>, still exists today and was an early domesticate from the fertile crescent important in the neolithic revolution (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dvořák <em>et al.</em> 1993</a>). The B genome ancestor (an <em>Aegilops spp</em>.) is believed to have since gone extinct (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blake <em>et al.</em> 1999</a>), but the tetraploid formed by these two genomes, <em>Triticum turgidum</em>, is still cultivated today primarily as emmer and durum wheat. The D genome comes from a goat grass, <em>Aegilops tauschii</em>, which may have been incorporated in a single hybridization event as recently as 10,000 years ago (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Salamini <em>et al.</em> 2002</a>). However, recent evidence based on sequence divergence of the D genome from the A and B genome has suggested a much earlier D genome incorporation around 400,000 years ago (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Marcussen <em>et al.</em> 2014</a>). Other evidence shows that limited gene flow into the D genome may have occurred after the polyploidization event, but appears to be from a single lineage (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2013</a>). As a result, the D genome has significantly lower genetic variation than either the A or B genome.</p><p id="p-12">We demonstrate methodology to partition subgenome additive variance to estimate SGEBVs as well as subgenome interactions using two allohexaploid wheat data sets, the Cornell small grains breeding program soft winter wheat breeding population dataset presented here (CNLM) and the W-GY wheat data from <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> (2010)</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Empirical data sets</h3><div id="sec-3" class="subsection"><h4>CNLM population:</h4><p id="p-13">The CNLM dataset consists of 8,692 phenotypic records of 1,447 soft winter wheat inbred lines evaluated at four locations near Ithaca, NY from 2007 to 2016, representing 26 environments. These phenotypic evaluations serve primarily as a first round of evaluation for grain yield and other agronomic traits before relatively few are selected for replicated regional trials around New York State. Lines are introduced and then removed after they are deemed either fit for advanced field trials or to be discarded or recycled in the breeding program. As such, this dataset is unbalanced in nature. Most lines were not replicated within a given trial (<em>i.e.</em>, year and location), but various check varieties were used throughout these years and are typically replicated several times within a given trial.</p><p id="p-14">Data were recorded for four agronomic traits, grain yield (GY), plant height (PH), test weight (TW) and heading date (HD). GY and TW have no missing data, but 842 records are missing PH and 246 records are missing HD. To facilitate comparison across traits, all phenotypes were standardized by subtracting the mean and dividing by the standard deviation of the phenotype vector (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table A1</a>). Data were not standardized within environment to preserve all relationships within the data.</p><p id="p-15">The population was genotyped with genotyping by sequencing (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">GBS; Elshire <em>et al.</em> 2011</a>) markers aligned to the International Wheat Genome Sequencing Consortium (IWGSC) RefSeq v1.0 wheat genome sequence of ‘Chinese Spring’ (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">IWGSC 2018</a>). Markers were filtered for minor allele frequency of at least 0.01 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure A1</a>), no more than 30% missing scores, and no more than 10% heterozygous calls. Missing marker scores were imputed using categorical random forest imputation by chromosome, and all heterozygous calls (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-1.gif"/></noscript></span></span>% of all calls) were subsequently replaced with the population mode (<em>i.e.</em>, homozygous major allele). Marker scores are presented as boolean indicators of the minor allele. Further details of the CNLM dataset can be found in Appendix 1.</p></div><div id="sec-4" class="subsection"><h4>W-GY population:</h4><p id="p-16">The W-GY wheat dataset of 599 historical wheat lines from the CIMMYT Global Wheat Breeding program (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>) was used in this study due to its importance in genomic prediction of an inbred population with non-additive variation (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>). The W-GY dataset consists of genotypic values of all lines for grain yield in each of four environments. The genetic correlations between these environments ranged from -0.19 to 0.66 and can be found in <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> (2016)</a>. As performance between these environments is not highly correlated, we refer to grain yield performance in each environment as a trait. The dataset was used in its entirety with one exception. Of the 1,279 available DArT markers, only the 1,188 with known chromosomal positions as indicated by (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>) were utilized in this study. This information was required to know which markers belonged to which subgenome, such that subgenome specific relationship matrices could be calculated.</p></div></div><div id="sec-5" class="subsection"><h3>Statistical framework</h3><div id="sec-6" class="subsection"><h4>Subgenome additive effects:</h4><p id="p-17">To illustrate, we begin with a linear mixed model depicting environments (<em>i.e.</em>, trials) as fixed effects and genotypes as random<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>μ</em> is the population mean, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-4.gif"/></noscript></span></span> are the fixed environmental and random genetic effects, respectively, of the <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-5.gif"/></noscript></span></span> genotype evaluated in the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-6.gif"/></noscript></span></span> environment, and ε is the error associated with the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-7.gif"/></noscript></span></span> observation. Using matrix notation, model 1 (denoted G) can be rewritten<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where 1 is a vector of ones, X is the design matrix of dummy variables for each trial, and β is the vector of fixed environmental effects. Z is the incidence matrix linking observations in the vector y to their respective genotype effects, in the vector <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-9.gif"/></noscript></span></span>. Normality was assumed for genotype effects and the residuals, where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-11.gif"/></noscript></span></span> The genetic covariance, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-12.gif"/></noscript></span></span>, can be derived from the expectation (or coefficient) of co-ancestry between individuals from a pedigree (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Henderson 1985</a>), or by an empirical estimation of the realized genetic relationship calculated with genome-wide markers (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Van Raden 2008</a>). When genome-wide markers are used to estimate <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-13.gif"/></noscript></span></span>, the genomic prediction model initially suggested by <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Nejati-Javaremi <em>et al.</em> (1997)</a> and <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Meuwissen <em>et al.</em> (2001)</a> is obtained.</p><p id="p-18">Given an <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-14.gif"/></noscript></span></span> matrix, M, of <em>m</em> markers scored as reference allele counts (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-15.gif"/></noscript></span></span>) for <em>n</em> individuals, method I of <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Van Raden (2008)</a> finds the genetic relationship K as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-17.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-18.gif"/></noscript></span></span> and p is the vector of allele frequencies. The small coefficient of 0.01 was added to the diagonal to recover full rank after centering the matrix, such that <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-19.gif"/></noscript></span></span> is invertible.</p><p id="p-19">We use allohexaploid wheat to illustrate, but this method is easily truncated to allotetraploids, or extended to higher level allopolyploids. If we allow the total genetic effect, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-20.gif"/></noscript></span></span> to be decomposed into individual additive effects for each subgenome, such that <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-21.gif"/></noscript></span></span> the following model (ABD) is obtained<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>In model 4, each subgenome is allowed to have its own additive genetic variance and covariance between individuals, such that <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-23.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-25.gif"/></noscript></span></span> The realized additive genetic covariances for each subgenome, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-26.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-28.gif"/></noscript></span></span>, are estimated using only markers corresponding to the respective subgenome, and calculated as described above.</p></div><div id="sec-7" class="subsection"><h4>Subgenome epistatic interactions:</h4><p id="p-20">Following <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Henderson (1985)</a>, the epistatic covariance of individuals can be calculated as the Hadamard product of the component covariance matrices. <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Jiang and Reif (2015)</a> and <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> (2016)</a> provide proofs of Henderson’s method using genome-wide markers to estimate the additive by additive covariance matrix, H. An additional linear kernel can then be added for an additive by additive epistatic interaction term, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-29.gif"/></noscript></span></span> once the additive covariance is estimated to obtain the following model (G×G)<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-31.gif"/></noscript></span></span> As shown by <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Jiang and Reif (2015)</a> and <a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> (2016)</a>, H is calculated from the marker data as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-33.gif"/></noscript></span></span> <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Jiang and Reif (2015</a>, Appendix A1) prove that H is asymptoically equivalent to <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-34.gif"/></noscript></span></span> when the number of markers is large.</p><p id="p-21">The additive by additive epistatic interaction term, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-35.gif"/></noscript></span></span> can also be decomposed into across subgenome interactions and within subgenome epistatic interactions such that <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-36.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-37.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-39.gif"/></noscript></span></span> are the subgenome interaction effects and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-40.gif"/></noscript></span></span> is the remaining epistatic effects due to within subgenome epistasis. Since no markers are shared across subgenomes, subgenome interaction covariances can be estimated by the Hadamard product of their component covariance matrices (<a id="xref-ref-43-8" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>). These interactions can then be incorporated in the following model (ABD×ABD)<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-42.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-43.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-45.gif"/></noscript></span></span> The three way interaction is included here for biological completeness, but was found to be estimated on the boundary (<em>i.e.</em>, zero) for all traits, and was therefore dropped from further analyses.</p></div></div><div id="sec-8" class="subsection"><h3>Accounting for population structure</h3><p id="p-22">Under Hardy Weinberg equilibrium, subgenomes segregate independently, such that for subgenome effects, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-47.gif"/></noscript></span></span> A breeding program, however, intentionally violates this assumption, and therefore may contain significant population structure. <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Price <em>et al.</em> (2006)</a> demonstrated that the first <em>k</em> largest principal components (PCs) of the kinship matrix can be used to control for population structure in genome-wide association studies, and its use has since become wide spread. Because most realized estimates of additive covariance are proportional to <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-48.gif"/></noscript></span></span> singular value decomposition of M, instead of <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-49.gif"/></noscript></span></span> can be used to separate the population structure as the first few principal components from the entire matrix of marker scores before it is divided into its subgenome components. This is accomplished by first extracting the first <em>k</em> principal components in the <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-50.gif"/></noscript></span></span> matrix Q. The marker matrix can then be reconstructed by setting the first <em>k</em> singular values of the diagonal matrix to zero and multiplying to produce a matrix <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-51.gif"/></noscript></span></span> with the population structure removed from each subgenome simultaneously (Appendix 2)</p><p id="p-23">Additive covariance matrices with reduced collinearity can then be calculated for each subgenome from <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-52.gif"/></noscript></span></span> and incorporated into the model as previously described. Q can then be added to the model as a set of fixed covariates, with slopes γ, such that the model will now be of the form<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>for all <em>l</em> genetic terms in the model. Genomic estimated breeding values are then predicted by summing the centered population structure and genetic effects. For this study, a population structure of dimension <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-54.gif"/></noscript></span></span> was chosen for both the CNLM and W-GY datasets, and used to compare to the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-55.gif"/></noscript></span></span> models that do not correct for population structure. The number of PCs, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-56.gif"/></noscript></span></span>, was chosen based on Supplementary Figure S1, where the correlation of additive covariance estimates between subgenomes appeared to level off in both populations.</p></div><div id="sec-9" class="subsection"><h3>Genomic prediction</h3><p id="p-24">To determine the predictability of genetic effects and the variability of variance component estimates, five-fold cross-validation was performed with 10 replications. For each replicate, the set of individuals was randomly split into five groups, with 4 groups of 289 and one of 291. For each fold, records of individuals in the fold were removed (<em>i.e.</em>, masked) from the dataset. Each model was subsequently fit with the remaining lines and used to predict the whole genome effect of the masked lines in the fold. Predictions for all five folds were gathered and correlated to the “true” genetic values once for each replicate. In this way, prediction results are directly comparable between the different models, and not subject to differences in the individuals sampled. The whole genome values were calculated as the sum of the genotypic additive and epistatic effects in the model as previously described. Due to the unbalanced nature of the CNLM dataset, “true” genetic values were calculated as in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">equation 2</a> but were considered independent with a covariance <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-57.gif"/></noscript></span></span>.</p></div><div id="sec-10" class="subsection"><h3>Software</h3><p id="p-25">Models were fit using restricted maximum likelihood (REML) with the software ASReml (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gilmour 1997</a>) implemented in R (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Butler 2009</a>). The Tassel 5.0 GBS pipeline v2 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Glaubitz <em>et al.</em> 2014</a>) along with the ‘bwa’ alignment tool (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li and Durbin 2009</a>) were used for aligning GBS markers to the reference genome. All additional computation, analyses and figures were made using base R (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">R Core Team 2015</a>) implemented in the Microsoft Open R environment 3.3.2 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Microsoft 2017</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-26">Phenotypes for the CNLM population are included in the file ‘pheno.txt’. Marker information and imputed marker scores for the CNLM population are included in files ‘snpInfo.txt’ and ‘snpMatrix.txt’, respectively. Best Linear Unbiased Predictors (BLUPs) for whole and subgenome additive effects (GEBVs and SGEBVS, respectively), as well as non-additive whole and subgenome interaction effects can be found in the ‘effectTable.txt’ file. Genotype and phenotype data for the W-GY population can be found in the ‘BGLR’ package of R (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos and Pérez Rodriguez 2015</a>), and marker chromosome information can be found in <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> (2010)</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6870110">https://doi.org/10.25387/g3.6870110</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Model fit and variance components</h3><p id="p-27">Model fit was assessed using Akaike’s Information Criterion (AIC). Whole genome models tended to have the lowest AIC values, with the exception of the PH and HD traits for the epistatic ABD×ABD models in the CNLM population. When whole genome models had lower AIC values, the comparable subgenome models had only marginally higher AIC values (Supplementary Tables S1 and S2). Whole genome predictions between comparable whole genome and subgenome models were correlated at <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-58.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-59.gif"/></noscript></span></span> for traits within the CNLM and W-GY populations, respectively. This indicates that little, if any, genetic information was lost by splitting the whole genome into biologically relevant subgenome effects. The lack of perfect correlation is at least partially due to floating point rounding errors during model fitting and summation of genotype effects.</p><p id="p-28">Subgenome additive variance parameter estimates were positive for all models, but subgenome interaction variance parameter estimates were often estimated on the boundary (<em>i.e.</em>, near zero). Variance parameters estimated on the boundary were thus considered to be exactly zero. Shifts in variance component importance were seen when the epistatic terms were added in the model. For example, for the TW and E1 traits in the CNLM and W-GY populations, respectively, the A genome component was the largest in the additive only model, but was reduced to less than that of the B genome component in the epistatic model. Additive variance components were generally reduced in epistatic models compared to additive only models, but this reduction in additive variance was accompanied by non-zero subgenome interaction components. The B genome contributed the greatest amount of additive variance in the epistatic ABD×ABD models for all traits except HD. While the D genome variance component was far smaller than the A subgenome component for GY and TW in the CNLM population, it was comparable to the A subgenome component for all traits in the W-GY population.</p><p id="p-29">The A×B component was particularly important for the W-GY traits, E1, E3 and E4, as well as the HD and TW in the CNLM population. The A×D component also featured prominently for the PH and TW traits in the CNLM population. The B×D component appeared to be less important, having the largest effect for PH. No epistatic terms were significantly greater than zero for the E2 trait in the W-GY population. Addition of epistatic interactions resulted in a significant likelihood ratio test at p <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-60.gif"/></noscript></span></span> for all traits except GY, which was significant at p <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-61.gif"/></noscript></span></span>. Despite the significant addition of epistatic terms, additive GEBVs were highly correlated with whole genome predictions from the epistatic models, at <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-62.gif"/></noscript></span></span> for the CNLM population and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-63.gif"/></noscript></span></span> for the W-GY population. A model containing the three-way subgenome epistatic term was fit for all traits, but estimates of the three-way interaction variance parameter were zero for all traits.</p><p id="p-30">The distributions of variance component estimates from repeated sub-sampling of the data during <em>k</em>-fold cross-validation were centered near the point estimate from the full model fit. These distributions were either as wide (≈ 2 standard errors from the center) or tighter than expected based on the standard error from the full model fit (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Supplementary Figures S2 and S3). Standard errors were generally larger for epistatic variance components relative to their magnitude than additive variance components. Standard errors relative to their respective parameter estimates tended to be larger for all terms in models with more estimated variance parameters (Supplementary Tables S1 and S2).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/685/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates and standard errors of variance components from the full model (red) compared to the sampling distribution of variance component estimates from the cross-validation scheme (black violins). Two traits from the CNLM population, (A) PH and (B) HD, with contrasting genetic architectures are shown. G&#xD7;G (5) and ABD&#xD7;ABD (7) models are shown to the left and right of the dotted line, respectively. The sum of the additive  and epitstatic  variance components is also shown for the ABD&#xD7;ABD model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693366348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates and standard errors of variance components from the full model (red) compared to the sampling distribution of variance component estimates from the cross-validation scheme (black violins). Two traits from the CNLM population, (A) PH and (B) HD, with contrasting genetic architectures are shown. G&#xD7;G (5) and ABD&#xD7;ABD (7) models are shown to the left and right of the dotted line, respectively. The sum of the additive &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-64&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and epitstatic &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-65&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; variance components is also shown for the ABD&#xD7;ABD model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/685/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/685/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/685/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/685/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492468" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Estimates and standard errors of variance components from the full model (red) compared to the sampling distribution of variance component estimates from the cross-validation scheme (black violins). Two traits from the CNLM population, (A) PH and (B) HD, with contrasting genetic architectures are shown. G×G (5) and ABD×ABD (7) models are shown to the left and right of the dotted line, respectively. The sum of the additive <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F1/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F1/embed/mml-math-64.gif"/></noscript></span></span> and epitstatic <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F1/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F1/embed/mml-math-65.gif"/></noscript></span></span> variance components is also shown for the ABD×ABD model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Subgenome additive effects</h3><p id="p-32">Subgenome estimated breeding values (SGEBVs) were moderately correlated with the whole genome effect, but weakly correlated with one another (Supplementary Tables S3 and S4). The individuals with the highest SGEBV for one subgenome never had the highest SGEBV for the other two subgenomes, and were often not in the top 95% quantile of the population based on the other two subgenomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Supplementary Figures S4 and S5). For example, the individuals with the highest A, B and D SGEBV for GY in the CNLM population ranked 43rd, 39th and 60th for the whole genome effect, respectively. In contrast, the individual with the best A SGEBV for GY ranked 1067th, 952nd for the B and D genome, respectively. The individual with the highest B genome breeding value for GY ranked 221th and 1393rd for the A and D genomes, respectively. The individual with the highest D genome breeding value for GY ranked 347th and 123rd for the A and B subgenome, respectively. The individual with the highest whole genome GEBV for GY ranked 6th, 22nd and 519th for the A, B and D SGEBV, respectively. In several cases, the top individual based on a SGEBV was not in the top 95% quantile based on their whole genome effect, particularly in the W-GY population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/685/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of whole genome additive effects (GEBV) by subgenome additive effects (SGEBV) for GY in the CNLM populations. The dotted line indicates the 95% quantiles for whole or subgenome effects. Blue squares, triangles and diamonds indicate the line with the highest SGEBV for each of the A, B and D subgenomes, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693366348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of whole genome additive effects (GEBV) by subgenome additive effects (SGEBV) for GY in the CNLM populations. The dotted line indicates the 95% quantiles for whole or subgenome effects. Blue squares, triangles and diamonds indicate the line with the highest SGEBV for each of the A, B and D subgenomes, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/685/F2.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/685/F2.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/685/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/685/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Plot of whole genome additive effects (GEBV) by subgenome additive effects (SGEBV) for GY in the CNLM populations. The dotted line indicates the 95% quantiles for whole or subgenome effects. Blue squares, triangles and diamonds indicate the line with the highest SGEBV for each of the A, B and D subgenomes, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Prediction accuracy</h3><p id="p-34">Including epistasis kernels significantly improved genomic prediction accuracy for all traits except GY and E2 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Subgenome models had either comparable or slightly lower mean prediction accuracy than whole genome models for all traits except HD, for which subgenome models had superior accuracy. The variability in the prediction accuracy based on the individuals sampled was either the same (GY and TW) or lower (PH and HD) for the epistatic models compared to the additive models in the CNLM population, but was similar in the W-GY population (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The variability in prediction accuracy was increased for the subgenome models compared to the whole genome models in the W-GY population for some traits (E2 and E3), but was either the same or decreased in the the CNLM population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492444/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492444/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Table of genomic prediction accuracies for eight traits in the CNLM (GY, PH, TW and HD) or W-GY (E1, E2, E3, E4) populations with <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-67.gif"/></noscript></span></span>. <em>k</em> is the number of principal components removed from the marker matrix prior to calculating subgenome covariance matrices. The first 1 to <em>k</em> principal components were included as fixed effects in the model fit for <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/T1/embed/mml-math-68.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Adjustment for population structure</h3><p id="p-38">The first two principal components explained 17% and 19% of the variance of <strong>M</strong> in the CNLM and W-GY populations, respectively, indicating that some population structure exists in both populations (Supplementary Figure S6). The correlation of additive genetic covariance estimates between individuals based on the three subgenomes declined as PCs were removed from M, but appeared to level out between 5 to 10 PCs (Supplementary Figure S1). Correlation of whole genome effects between additive models, G and ABD, for <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-70.gif"/></noscript></span></span> was <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-71.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-72.gif"/></noscript></span></span> for the CNLM and W-GY populations respectively. Whole genome effect correlations were lower between epistatic models G×G and ABD×ABD, with coefficients of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-73.gif"/></noscript></span></span> in the CNLM population and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-74.gif"/></noscript></span></span> in the W-GY population.</p><p id="p-39">Removing population structure with <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-75.gif"/></noscript></span></span> reduced most of the SGEBV effect correlation coefficients by up to 0.06 in the CNLM population, but there was one instance in which one correlation coefficient increased from 0.15 to 0.19 between A and B SGEBVs for PH (Supplementary Tables S3 and S4). This was not the case for the W-GY population, where many of the SGEBV effect correlations increased by up to 0.21.</p><p id="p-40">Additive variance generally decreased as <em>k</em> was increased from 0 to 10 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Supplementary Figure S7). Ranks of additive variance components relative to one another were stable for most traits, while epistatic variance components were more sensitive to changes in <em>k</em>. Significant epistatic variance component rank changes occurred for the PH, TW and E4 traits. For PH, the A×D term was comparable in magnitude with the additive variance components for A and D when <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-76.gif"/></noscript></span></span> but declined as <em>k</em> increased. The reduction in A×D variance for PH was accompanied by an increase in both the A×B and B×D terms. Similarly, a decline in A×B variance was followed by an increase in B×D for TW and A×D for E4 as <em>k</em> was increased.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/685/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subgenome additive and interaction variance parameter estimates from the ABD&#xD7;ABD (7) model correcting for population structure with  principal components as fixed effects. Models were fit with four traits for the CNLM population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693366348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Subgenome additive and interaction variance parameter estimates from the ABD&#xD7;ABD (7) model correcting for population structure with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-77&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; principal components as fixed effects. Models were fit with four traits for the CNLM population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/685/F3.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/685/F3.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/685/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/685/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492348" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Subgenome additive and interaction variance parameter estimates from the ABD×ABD (7) model correcting for population structure with <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F3/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/F3/embed/mml-math-77.gif"/></noscript></span></span> principal components as fixed effects. Models were fit with four traits for the CNLM population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Correlations of variance component estimates were calculated from the average information matrix for models <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-78.gif"/></noscript></span></span> (Supplementary Figures S8 and S9). Correlations between subgenome additive variance estimates were generally low (0.2-0.4), while correlations of subgenome interaction variance estimates were high (0.8-0.95), and correlations between the two were moderate (0.4-0.6). Despite a small reduction in the correlation of SGEBVs as <em>k</em> was increased from 0 to 5, little reduction in variance component estimate correlations was observed as <em>k</em> was increased from 0. Generally, correlations of additive variance parameter estimates were slightly reduced while correlations between interaction variance parameter estimates increased slightly.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Model fit and variance components</h3><p id="p-43">While whole genome models tended to be the most parsimonious, subgenome models are worth consideration because they provide insight into the biology of the allopolyploid organism. Given the stability of variance component estimation and that no genetic information appears to be lost by partitioning the whole genome into its individual subgenome additive effects, such a partition is informative.</p><p id="p-44">The method presented here could be used for any set of independent loci, such as estimating a variance component and breeding value for each chromosome (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Yang <em>et al.</em> 2011</a>). However, this will become computationally burdensome as the number of variance components to be estimated increases. If the number of variance parameters to estimate is high and the data set is small this may become infeasible. It is also unclear if the estimates from larger numbers of additive kernels would be reliable.</p><p id="p-45"><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bernardo and Thompson (2016)</a> assigned a breeding value for each of the 10 maize chromosomes by fitting a single ridge regression model to estimate marker effects. They subsequently summed marker effects by chromosome to produce a breeding value for each chromosome. However, this method does not allow for direct estimation of variance components for each unit of chromatin. By fitting each unit simultaneously, variance attributable to sets of loci will be split, and the sum of the variance estimates should not exceed the total genetic variance (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Yang <em>et al.</em> 2011</a>). It is unclear what effect linkage disequilibrium across chromosomes has on the variance parameters estimated.</p><p id="p-46">Here we assumed that the subgenome effects are independent, but this is clearly not the case. Generally, we can express the genetic variance due to the three sub genomes as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-79.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where S is the subgenome covariance matrix, J is an <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-80.gif"/></noscript></span></span> matrix of ones for <em>n</em> genotypes, and K is the additive relationship matrix for within and across subgenomes. In this report, we have assumed that S is diagonal with <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-81.gif"/></noscript></span></span> for the <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-82.gif"/></noscript></span></span> subgenome, and K is a block diagonal with the <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-83.gif"/></noscript></span></span> diagonal block represented by the subgenome additive covariance matrix.<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>An unstructured covariance matrix, S, could be estimated, with correlation coefficients between subgenomes. The subgenome effects would be allowed to have a correlation such that<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>However, it is unclear what the covariance structure should be between subgenomes (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-86.gif"/></noscript></span></span>). If consensus haplotypes from uniquely identifiable sequences could be determined with two or more alleles segregating in at least two subgenomes, a covariance across the subgenomes could be constructed. Polymorphisms that predate speciation would be used to identify the consensus haplotypes, while post speciation polymorphisms would be used to identity the subgenome origin. Individuals would then receive a score based on the number of consensus haplotypes they have in common between two subgenomes. This could prove to be a formidable challenge given the evolutionary time between the subgenome ancestors. The Hadamard product of the two additive covariance matrices is a tempting candidate for these off diagonal blocks, however, this would substitute a covariance between additive effects in place of an epistatic variance. It is unclear to these authors if epistasis variance can be thought of and modeled as a covariance between additive effects.</p></div><div id="sec-19" class="subsection"><h3>Genetic architecture</h3><p id="p-47">The genetic architecture of grain yield (GY, E1, E2, E3, E4) in the two populations investigated here are markedly similar, despite the divergent genetic backgrounds of the two populations. The CNLM population is primarily comprised of breeding lines and varieties derived from germplasm historically grown in the North East, in contrast to the W-GY population which has a broader pedigree.</p><p id="p-48">The D genome is known to have low genetic diversity due to limited gene flow from a single <em>Ae. tauschii</em> lineage after the most recent allopolyploidization event (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Wang <em>et al.</em> 2013</a>), estimated to have taken place as recently as 10,000 years ago (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Salamini <em>et al.</em> 2002</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Marcussen <em>et al.</em> 2014</a>). The International Maize and Wheat Improvement Center (Centro Internacional de Mejoramiento de Maíz y Trigo, CIMMYT) has introgressed some genetic material from the D genome ancestor, <em>Ae. tauschii</em>, through the use of synthetically produced hexaploid wheat to increase the genetic diversity of the historically bottle-necked D genome. The higher proportion of D genome variance in the W-GY population may be due to the increased use of wild <em>Ae. tauschii</em> in their breeding program, highlighting the merit of the strategy.</p><p id="p-49">Many of the subgenome epistatic variance parameters were estimated at zero, possibly due to a lack of power to detect them. Greater genetic diversity, larger numbers of individuals, and higher allele frequencies would allow for increased power to detect true interactions. <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hill <em>et al.</em> (2008)</a> emphasized the effect of low allele frequencies on epistatic interactions, proving that as allele frequencies (and therefore joint frequencies of alleles at two loci) approach zero or one, most of the epistatic variance becomes additive. For example, suppose two loci have a large interaction, such that one pair of alleles is selected. Once one locus becomes fixed, all remaining variance is due to segregation at the other locus, and becomes strictly additive. The low joint frequency is magnified in the three way interactions, likely causing the inability to detect any three way epistatic interaction signal between the three subgenomes.</p><p id="p-50">This is also apparent in the reduction of additive variance components upon the addition of epistatic terms to the model. These epistatic components were often estimated to be rather large compared to the additive components, but did not change the final whole genome value drastically. This suggests that the additive terms absorb much of the epistatic variance in the absence of the epistatic kernels.</p><p id="p-51">The A×B epistatic terms were the most important for many of the traits, reflecting the greater genetic variation of these two subgenomes. Subgenome interaction terms including the D genome were notably more important for traits known to have important loci on the D genome. PH is partially governed by two dwarfing genes, <em>Rht-1D</em> and <em>Rht-1B</em> on 4B and 4D, respectively. These two genes have been shown to exhibit a less-than-additive (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eshed and Zamir 1996</a>) epistatic interaction, where the double wildtype is less tall than expected based on the additive effects of the two semi dwarfs from the double dwarf (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Santantonio <em>et al.</em>, 2018b</a>). The B×D term was large for PH, particularly after correction for population structure. Population structure is common for these genes, as many breeding programs primarily utilize one or the other dwarfing gene to avoid producing double dwarfs during crossing, which are often agronomically undesirable.</p></div><div id="sec-20" class="subsection"><h3>Selection on SGEBVs</h3><p id="p-52">Partitioning genetic variance to the subgenomes of an allopolyploid provides a method for identifying individuals with complementary subgenomes as potential parents for crossing. If we consider the upper 95% quantiles as candidates for parental selection, many of the top candidates based on subgenome breeding values would not be considered candidates based on their whole genome breeding value. When they would be considered, they were typically not the top candidates. The low correlation between SGEBVs highlights the opportunity to identify individuals with complementary subgenomes for crossing. These individuals may or may not be among the top performing selection candidates, demonstrating that the optimum set of crosses are not always between the top performing individuals (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Akdemir and Sánchez 2016</a>).</p><p id="p-53">While it is difficult to evaluate the predictability of SGEBVs <em>per se</em>, the sum of their values appears highly predictive. The low correlation of SGEBVs also suggests that individual subgenomes may be directly manipulated as never before. Prior to the discovery and use of genetic markers to track genomic regions, the phenotype (or some summary statistic thereof) was the only indicator of the genetic structure of a genetically distinct individual. Variety releases still demonstrate this legacy, with phenotypic descriptors that define a new variety as genetically distinct from other similar varieties. One breeding strategy will be selecting parents for crossing that have complementary SGEBVs to increase the potential of transgressive segregation in the resulting offspring. We envision other breeding strategies beyond simply choosing parents with complementary subgenomes, and see an opportunity to weight SGEBVs according to some breeding goal.</p><p id="p-54">For example, a newly formed population could undergo several rounds of genomic selection only on the D genome SGEBVs (<em>i.e.</em>, weights of 0, 0 and 1 for the A, B and D subgenomes respectively) before phenotypic or whole genome selection. Because the D genome contributes the least to the total genetic variance, phenotypic selection on D genome loci is challenging. Selection will act on the largest sources of genetic variance first, potentially leading to fixation of small effect loci in the D genome by drift, while selection acts on the large effect loci on the A and B genomes first. By selecting on D genome SGEBVs, gains can be made to the D genome directly with little to no selection on the A and B genomes, a feat previously impossible with phenotypic selection. Signatures of selection on the D genome under this scheme may also help establish the accuracy of SGEBV prediction.</p></div><div id="sec-21" class="subsection"><h3>Subgenome interactions</h3><p id="p-55">Genomic prediction of GY and E2 did not appear to benefit from including epistatic interactions as it did for the other six traits. This may be due in part to the highly polygenic nature of grain yield, which is the culmination of essentially all functional genetic variants subjected to stress throughout the growth cycle. The E2 trait in the W-GY population has previously been shown to be invariant to the addition of various epistatic terms (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-43-9" class="xref-bibr" href="#ref-43">Martini <em>et al.</em> 2016</a>), and it is unclear why this population does not exhibit non-additive variation in this environment. It may be that important epistatic interactions of GY in the CNLM population are too small to detect or are involved with differing performance across years or locations, such that they are lacking in a model that does not include genotype by environment interactions.</p><p id="p-56">Subgenome epistatic terms increased genomic prediction accuracy comparable to modeling all pairwise interactions across the subgenomes, suggesting that the most important interacting loci are on different subgenomes. This result is consistent with the observation that newly formed allopolyploids undergo considerable changes in gene expression, known as genome shock (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">McClintock 1984</a>). This shock has been suggested to be caused by incompatibilities of genetic pathways across the subgenomes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comai <em>et al.</em> 2003</a>). Residual subgenome incompatibility may still be affecting the germplasm pool, even thousands of years after the last polyploidization event. Decay of negative duplicated gene interactions may take hundreds or thousands of generations before all interacting genes are lost or silenced (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch and Conery 2000</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2003</a>).</p><p id="p-57">It is unclear what proportion of this non-additive signal is due to homeoallelic interactions. The proposed method models all pairwise interactions across subgenomes, of which homeoallelic gene interactions are a small minority in number. Smaller homeoallelic regions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Santantonio <em>et al.</em> 2018a</a>) or homeoallele specific marker sets (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Santantonio <em>et al.</em> 2018b</a>) have been constructed to determine the relative importance of these interactions relative to other gene interactions across the subgenomes. The usefulness of the epistatic subgenome interactions is currently unclear and warrants further investigation.</p><p id="p-58">Regardless of the source of the epistasis, we suggest that a breeding scheme should be designed to take advantage of beneficial subgenome interactions. If a suitable training set related to the breeding material can be established, subgenome interactions can be predicted in new, genotyped breeding materials. We suggest that a series of small bi-parental populations be constructed from important contributors to the breeding program, and be used in the development of a training population to balance high genetic diversity and high allele frequencies. This training population would be used to predict SGBEVs and subgenome interactions in individuals formed from new crosses. Individuals that contain favorable interactions could then be selected such that they are fixed in early filial generations. After fixation in a given line, phenotypic, whole genome, or subgenome selection could be used for further line development until complete homozygosity is reached.</p></div><div id="sec-22" class="subsection"><h3>Adjustment for population structure</h3><p id="p-59">The efficacy of the proposed method to handle population structure may need to be improved, or a different approach may need to be taken. While this method reduced the correlation of additive genetic subgenome covariance estimates across the three genomes, variance parameter estimate correlations were not drastically reduced. The correlation of subgenome interaction variance parameter estimates tended to increase slightly when accounting for higher levels of population structure, counter to the assumption that removing this structure should result in better estimates of subgenome interactions.</p><p id="p-60">The lower correlation between epistatic models that correct and do not correct for population structure is likely due to removing Q from the marker matrix. Correcting for population structure also had a small, but negative effect on genomic prediction accuracy for epistatic models. The population structure fixed effect predictors are strictly additive and the loss of accuracy may be due to epistasis variance associated with these PCs (<em>i.e.</em>, population structure epistasis). Epistatic variance related to these PCs may be recovered by using the squares of the PC scores, although this was not done in this study. At least for the additive models, it appears little to no genetic information is lost using the population structure adjustment proposed here.</p><p id="p-61">Determining the best value for <em>k</em> will be at the crux for implementing this methodology for various traits and populations. The same population may need different values of <em>k</em> for different traits, depending on the covariance of the trait and population structure. Traits such as PH or HD may have lower covariance with population structure than traits such as TW or GY, due to different marker effect distributions and the history of the breeding population. Several methods might be used to determine <em>k</em> empirically from the marker matrix (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Patterson <em>et al.</em> 2006</a>, <em>e.g.</em>), however, these methods do not account for the covariance of trait and structure. We used the first one to <em>k</em> PCs in this study, but there is no reason why we must include all PCs up to some value <em>k</em>. There may be certain PCs that are important for a given trait, and could be tested as fixed effects for inclusion or exclusion.</p><p id="p-62">This method may have better performance in populations with greater degrees of population structure than in the populations presented here. Use of this method for partitioning genetic variance to biologically important sets of chromatin and estimating epistatic interactions will need further testing and validation before widespread use.</p></div></div><div class="section conclusions" id="sec-23"><h2 class="">Conclusion</h2><p id="p-63">To our knowledge, we provide the first attempt to assign a breeding value to each subgenome of an allopolyploid crop. With estimates of subgenome additive effects, parents with complementary subgenomes can be selected for crossing. Weighted selection of subgenomes using genomic prediction could be key to increasing the diversity of the D genome in wheat germplasm. Direct selection on the D genome may allow targeted introgression from <em>Ae. tauschii</em> while mitigating the effects of introducing unimproved alleles. Subgenome additive genetic variances appear to be estimated well, and no genetic information is lost partitioning the genome into its subgenome components. This demonstrates that partitioning genetic variance to the subgenomes of an allopolyploid can provide useful information for genomics assisted breeding efforts.</p><p id="p-64">Subgenome interactions increase prediction accuracy, but it is unclear how well the epistatic variance is partitioned to the three interaction terms and what proportion of that variance is due to homeologous gene interactions. Because the homeologous interactions make up relatively few of the possible interactions across subgenomes, they may only explain a small portion of the observed epistatic variance. Yet, seeing as how homeologous genes likely operate in the same or similar physiological pathway, the likelihood for interactions between homeologous loci is high. Further research is needed to investigate the efficacy of modeling subgenome interaction terms, and to what degree this is explained by interactions between homeologous orthologs.</p><p id="p-65">Allopolyploids have traditionally been treated as diploids in breeding programs because they undergo disomic inheritance. With modern DNA marker technology and ever increasing computational power, breeders of allopolyploids can further exploit the genetic complexity of their crops.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-78">Funding of this research was provided by the USDA National Needs Fellowship for N. Santantonio, in partial fulfillment of the requirements for a Ph.D in Plant Breeding and Genetics at Cornell University. Additionally, the field trials comprising the phenotypic data for the CNLM population were funded in part by the Hatch Project # 149-447. Genotyping was funded by the Wheat Coordinated Agricultural Project (WheatCAP). We are also grateful to Jesse Poland’s research group at Kansas State University for their contribution to genotyping of CNLM materials. The authors thank the International Wheat Genome Sequencing Consortium for pre-publication access to IWGSC RefSeq v1.0. Finally, we would like to acknowledge the Cornell small grains staff, particularly David Benscher and James Tanaka, who were vital in implementing, collecting and processing the materials used to build the CNLM dataset.</p><p id="p-79"><em>Note added in proof:</em> See Santantonio <em>et al.</em> 2019 (pp. 675–684) in this issue and (pp. 1105-1122) in the Genetics March 2019 issue, for related works.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1 Cnlm Dataset</h2><p id="p-66">Field plots 1.5 m by 3 m in size were planted with 100 g of seed in September or October of the year prior to the harvest year. Data were recorded for four agronomic traits: grain yield (GY), plant height (PH), test weight (TW) and heading date (HD). Plots were harvested for grain with a plot combine after physiological maturity and oven dried to a grain moisture of approximately 12%. Dried grain was cleaned, weighed and measured for moisture content using a grain moisture analyzer (GAC 2100, Dickey-John). GY was standardized to a uniform grain moisture of 12%. PH was measured as the distance from the ground to the top of the grain head at full extension. TW is used as a measure of grain quality and was measured as the mass of a volume of grain (<span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-87.gif"/></noscript></span></span>) using the grain moisture analyzer (GAC 2100 Dickey-John) which corrects TW for moisture content. HD is a proxy for flowering time and was defined as the number of Julian days until 50% of the primary grain heads have extended out of the boot.</p><p id="p-67">The data set initially consisted of 1,552 lines evaluated in 10,069 1.5 m by 3 m plots planted in September to October of the year prior to the harvest year. Thirty one lines from 2007 were not harvested for GY, nor were they genotyped, and were dropped from the data set. Because GY is of primary interest for breeding, plots that were not harvested or had missing values for GY were dropped, resulting in 9,090 plots with GY measurements. This also caused two additional lines with missing GY measurements to be dropped from the dataset.</p><p id="p-68">Due to the reasonable size of the dataset, small physical area of most trials, lack of replication within environment for most lines and the availability of genetic markers, raw plot observations were used and no attempt was made to correct plot level data for various spatial effects or otherwise. Preliminary results also indicated relatively high genomic prediction accuracy, suggesting that spatial correction, such as an AR1 × AR1 row column autocorrelation structure, would be unlikely to reduce error variance drastically. Instead, 59 plots that included breeder comments about bad seed or significant damage to the plot, via animal or otherwise, were removed from the dataset. Observations outside of a four standard deviation interval from the grand mean of uncorrected GY phenotypic observations were also removed to account for any significant undocumented damage, grain spillage or other undocumented mistakes. This included two observations that were deemed too high, and 20 observations that were deemed too low.</p><p id="p-69">Observations of 11 lines lacking at least one phenotypic record in at least two separate trials, 60 lines that were missing at least 50% of their genotypic calls and one line that had greater than 20% heterozygous genotype calls were also removed from the dataset. This resulted in 8,692 phenotypic observations of 1,447 lines across 26 environments, representing 96.6% of the plots with grain yield measurements. HD was not recorded for the 246 observations from 2007, and PH was not recorded for the 840 observations from 2009. Two additional plots were missing PH measurements from the Ketola location, one that was recorded at 2 meters in 2008 which was likely a recording mistake, and one in 2010 which was simply missing a record.</p><p id="p-70">While most of the genotypes were directly from the Cornell small grains breeding program, a few varieties and breeding lines from other breeding programs that had been genotyped and evaluated were not excluded from the dataset as long as they met the previous criteria. This included 75 lines from The Ohio State University wheat breeding program and 93 lines from the Michigan State University wheat breeding programs that were part of the Allele Based Breeding initiative, among other lines from various breeding programs.</p><p id="p-71">Genotyping by sequencing (GBS) libraries (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Elshire <em>et al.</em> 2011</a>) of 1,521 CNLM were developed using the protocol described by <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Poland <em>et al.</em> (2012)</a> at Kansas State University, and subsequently sequenced at the Genomic Diversity Facility at Cornell University. Genotyping calls were accomplished using standard parameters of the Tassel 5.0 GBS v2 Pipeline (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Glaubitz <em>et al.</em> 2014</a>) and were aligned to the International Wheat Genome Sequencing Consortium (IWGSC) RefSeq v1.0 wheat genome sequence of ‘Chinese Spring’ (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">IWGSC 2018</a>). Following Poland <em>et al.</em> (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Poland <em>et al.</em> 2012</a>), 64 bp sequence tags containing no more than three Single Nucleotide Polymorphisms (SNPs) per tag were included to increase the likelihood of obtaining subgenome specific markers. Only markers with a minor allele frequency of at least 0.01 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure A1</a>), no more than 30% missing scores, and no more than 10% heterozygous calls were kept for the following analyses. Then individuals with greater than 20% heterozygous calls and individuals with more than 50% missing genotype calls were excluded from the dataset. The process was repeated iteratively, starting by filtering on markers until the number of markers and genotypes converged. This resulted in 11,604 available GBS markers distributed throughout the three subgenomes (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure A2</a>). Of the 61 lines removed, 60 were removed due to missing marker information and 1 was removed due to high heterozygosity in the final iteration. The 11 lines with a single observation and the two lines without grain yield observations were subsequently removed to produce genotypic information for the 1,447 lines.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/685/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of minor allele frequencies for 11,604 GBS markers in the CNLM population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693366348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of minor allele frequencies for 11,604 GBS markers in the CNLM population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/685/F4.medium.gif" width="440" height="399"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A1" src="https://www.g3journal.org/content/ggg/9/3/685/F4.medium.gif" width="440" height="399"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/685/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/685/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A1</span> <p id="p-72" class="first-child">Distribution of minor allele frequencies for 11,604 GBS markers in the CNLM population.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/685/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of 11,604 GBS markers on the 21 wheat chromosomes comprised of 7 homeologs of three subgenomes, A, B and D, for the CNLM population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-693366348" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of 11,604 GBS markers on the 21 wheat chromosomes comprised of 7 homeologs of three subgenomes, A, B and D, for the CNLM population.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure A2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/685/F5.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure A2" src="https://www.g3journal.org/content/ggg/9/3/685/F5.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/685/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/685/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure A2</span> <p id="p-73" class="first-child">Distribution of 11,604 GBS markers on the 21 wheat chromosomes comprised of 7 homeologs of three subgenomes, A, B and D, for the CNLM population.</p><div class="sb-div caption-clear"></div></div></div><p id="p-74">Marker scores were coded using <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-88.gif"/></noscript></span></span> for homozygous major allele, heterozygous and homozygous minor allele, respectively. Categorical marker imputation was done independently for each chromosome using random forest imputation via the R package ‘missForest’ (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stekhoven and Bühlmann 2011</a>) which relies on the R package ‘randomForest’ (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Liaw and Wiener 2002</a>). Random forest has been shown to be effective for genotype imputation in wheat (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rutkoski <em>et al.</em> 2013</a>). To allow all individuals to be considered completely inbred, the remaining heterozygous calls (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-89.gif"/></noscript></span></span>% of all marker scores) were conservatively replaced with the population mode for that marker (<em>i.e.</em> the homozygous major allele, -1). Marker scores were then converted to <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-90.gif"/></noscript></span></span> coding for presence of the minor allele.</p><p id="p-75">Genetic correlations of traits were estimated in a multivariate model fit (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table A2</a>). This was accomplished by treating genotypes as independent, or having a realized additive covariance structure calculated from genetic markers.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492421/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492421/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">Means (<em>μ</em>) and standard deviations (<em>σ</em>) of four traits in the CNLM population</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492541/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492541/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A2</span> <span class="caption-title">Estimated genetic correlation of traits with additive (below diagonal) and independent genetic relationships (above diagonal). Genetic standard deviations of scaled traits estimated with a realized additive covariance between individuals and assuming independence are shown in parentheses on the diagonal, respectively</span><div class="sb-div caption-clear"></div></div></div></div><div class="section app" id="app-2"><h2 class="">Appendix 2 Removal Of Population Structure From M</h2><p id="p-76">Let M be the <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-91.gif"/></noscript></span></span> matrix of <em>m</em> marker scores for <em>n</em> genotypes. Markers can be sorted into their respective genome, such that<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-92.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>M can be factored using singular value decomposition as follows:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-93.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>where U, and V are unitary matrices of left and right singular vectors, and D is a diagonal matrix of singular values.</p><p id="p-77">The first <em>k</em> principal components of the marker matrix can be extracted by selecting the first <em>k</em> columns of U and the first <em>k</em> rows and columns of D and multiplying.<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-94.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>In a manner similar to Eckart and Young (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eckart and Young 1936</a>), an approximation, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-95.gif"/></noscript></span></span>, of the marker matrix, M with the first <em>q</em> principal components removed can be reconstructed by setting the first <em>k</em> singular values in D to zero (denoted <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-96.gif"/></noscript></span></span>).<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/685/embed/mml-math-97.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span></p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-80">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6870110">https://doi.org/10.25387/g3.6870110</a>.</p></li><li class="fn" id="fn-5"><p id="p-81"><em>Communicating editor: F. van Eeuwijk</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-07-26"><span class="received-label">Received </span>July 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-14"><span class="accepted-label">Accepted </span>November 14, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Santantonio <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.1" data-doi="10.1007/s10681-005-3364-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möllers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Development of synthetic brassica napus lines for the analysis of “fixed heterosis” in allopolyploid plants.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-005-3364-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D146%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-3364-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-3364-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.2" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.173658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.3" data-doi="10.1186/s12711-017-0348-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isidro-Sánchez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Locally epistatic models for genome-wide prediction and association by importance sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0348-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0348-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0348-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.4" data-doi="10.3389/fgene.2016.00210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient breeding by genomic mating.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2016.00210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2016.00210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2016.00210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.5" data-doi="10.3835/plantgenome2015.04.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Germplasm architecture revealed through chromosomal effects for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.03.0028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.6" data-doi="10.2135/cropsci1994.0011183X003400040001x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bingham</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groose</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodfield</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Complementary gene interactions in alfalfa are greater in autotetraploids than diploids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1994.0011183X003400040001x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBingham%26rft.auinit1%253DE.%2BT.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D823%26rft.epage%253D829%26rft.atitle%253DComplementary%2BGene%2BInteractions%2Bin%2BAlfalfa%2Bare%2BGreater%2Bin%2BAutotetraploids%2Bthan%2BDiploids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400040001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400040001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NV51600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.7" data-doi="10.1105/tpc.110.076133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chudalayandi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaiman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heterosis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.076133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.076133%26rft_id%253Dinfo%253Apmid%252F20622146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMjEwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.8" data-doi="10.1139/g98-136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehfeldt</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Phylogenetic reconstruction based on low copy dna sequence data in an allopolyploid: the b genome of wheat.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g98-136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBlake%26rft.auinit1%253DN.%2BK.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D360%26rft.atitle%253DPhylogenetic%2Breconstruction%2Bbased%2Bon%2Blow%2Bcopy%2BDNA%2Bsequence%2Bdata%2Bin%2Ban%2Ballopolyploid%253A%2Bthe%2BB%2Bgenome%2Bof%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg98-136%26rft_id%253Dinfo%253Apmid%252F10231966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g98-136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10231966&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.685.9"><div class="cit-metadata unstructured">Butler, D., 2009 <em>asreml: asreml() fits the linear mixed model</em>. R package version 3.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Design iii with marker loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1437</span>–<span class="cit-lpage">1456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D143%26rft.issue%253D3%26rft.spage%253D1437%26rft.epage%253D1456%26rft.atitle%253DDesign%2BIII%2BWith%2BMarker%2BLoci%26rft_id%253Dinfo%253Apmid%252F8807314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My8zLzE0MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.11" data-doi="10.1098/rstb.2003.1305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madlung</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Josefsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Do the different parental ‘heteromes’ cause genomic shock in newly formed allopolyploids?</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">358</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2003.1305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D358%26rft.issue%253D1434%26rft.spage%253D1149%26rft.epage%253D1155%26rft.atitle%253DDo%2Bthe%2Bdifferent%2Bparental%2B%2527heteromes%2527%2Bcause%2Bgenomic%2Bshock%2Bin%2Bnewly%2Bformed%2Ballopolyploids%253F%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2003.1305%26rft_id%253Dinfo%253Apmid%252F12831481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2003.1305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12831481&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183891600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.12" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.685.13"><div class="cit-metadata unstructured">de los Campos, G. and P. Pérez Rodriguez, 2015 <em>BGLR: Bayesian Generalized Linear Regression</em>. R package version 1.0.4.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.685.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duvick</span>  <span class="cit-name-given-names">D. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Heterosis: feeding people and protecting natural resources</span>. Genetics and Exploitation of Heterosis in Crops, Coors J.G., Pandey S., eds Madison, WI: American Society of Agronomy and Crop Science Society of America, pp. <span class="cit-fpage">19</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.15" data-doi="10.1139/g93-004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvořák</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terlizzi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resta</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The evolution of polyploid wheats: identification of the a genome donor species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g93-004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DDvorak%26rft.auinit1%253DJ.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D31%26rft.atitle%253DThe%2Bevolution%2Bof%2Bpolyploid%2Bwheats%253A%2Bidentification%2Bof%2Bthe%2BA%2Bgenome%2Bdonor%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg93-004%26rft_id%253Dinfo%253Apmid%252F18469969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g93-004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18469969&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.16" data-doi="10.1007/BF02288367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">The approximation of one matrix by another of lower rank.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02288367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D1%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02288367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02288367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000203817500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.17" data-doi="10.1073/pnas.97.13.7043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellstrand</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenbeck</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hybridization as a stimulus for the evolution of invasiveness in plants?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7043</span>–<span class="cit-lpage">7050</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.97.13.7043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DEllstrand%26rft.auinit1%253DN.%2BC.%26rft.volume%253D97%26rft.issue%253D13%26rft.spage%253D7043%26rft.epage%253D7050%26rft.atitle%253DHybridization%2Bas%2Ba%2Bstimulus%2Bfor%2Bthe%2Bevolution%2Bof%2Binvasiveness%2Bin%2Bplants%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.13.7043%26rft_id%253Dinfo%253Apmid%252F10860969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTMvNzA0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.18" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Less-than-additive epistatic interactions of quantitative trait loci in tomato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1807</span>–<span class="cit-lpage">1817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEshed%26rft.auinit1%253DY.%26rft.volume%253D143%26rft.issue%253D4%26rft.spage%253D1807%26rft.epage%253D1817%26rft.atitle%253DLess-Than-Additive%2BEpistatic%2BInteractions%2Bof%2BQuantitative%2BTrait%2BLoci%2Bin%2BTomato%26rft_id%253Dinfo%253Apmid%252F8844166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0My80LzE4MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.20" data-doi="10.1093/jxb/ers192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic asymmetry in allopolyploid plants: wheat as a model.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">5045</span>–<span class="cit-lpage">5059</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ers192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fers192%26rft_id%253Dinfo%253Apmid%252F22859676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ers192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22859676&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308235100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.21" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">Xv–the correlation between relatives on the supposition of mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Earth and Environmental Science Transactions of the Royal Society of Edinburgh</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0080456800012163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2Band%2BEnvironmental%2BScience%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BEdinburgh%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.22" data-doi="10.1038/s41467-017-01450-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The effect of artificial selection on phenotypic plasticity in maize.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-017-01450-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D1348%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-01450-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-01450-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DRoche%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D376%26rft.epage%253D391%26rft.atitle%253DAssociations%2BAmong%2BBody%2BCondition%2BScore%252C%2BBody%2BWeight%252C%2Band%2BReproductive%2BPerformance%2Bin%2BSeasonal-Calving%2BDairy%2BCattle%26rft_id%253Dinfo%253Apmid%252F17183106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17183106&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Asreml for testing fixed effects and estimating multiple trait variance components.</span> <abbr class="cit-jnl-abbrev">Proceedings of the Association for the Advancement of Animal Breeding and Genetics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">386</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BAssociation%2Bfor%2Bthe%2BAdvancement%2Bof%2BAnimal%2BBreeding%2Band%2BGenetics%26rft.volume%253D12%26rft.spage%253D386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.25" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tassel-gbs: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.26" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00702.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.27" data-doi="10.1086/280465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">The part played by recurrent mutation in evolution.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/280465</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D67%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1086%252F280465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/280465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000201328000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.28" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.29" data-doi="10.2527/jas1985.601111x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Best linear unbiased prediction of nonadditive genetic merits in noninbred populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1985.601111x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BNonadditive%2BGenetic%2BMerits%2Bin%2BNoninbred%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1985.601111x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1985.601111x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AAM9700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.30" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.31" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.32" data-doi="10.1371/journal.pgen.1006421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of quantitative traits cannot be inferred from variance component analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006421%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.685.33"><div class="cit-metadata"><cite>IWGSC, I. W. G. S. C., 2018  Shifting the limits in wheat research and breeding using a fully annotated reference genome by the international wheat genome sequencing consortium (iwgsc). Science <span class="cit-vol">361</span>: eaar7191.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.34" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elq001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20156985&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.35" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.36" data-doi="10.1038/ng.3974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1741</span>–<span class="cit-lpage">1746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.37" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with burrows–wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and regression by randomforest.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.39" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.290.5494.1151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.685.40"><div class="cit-metadata"><cite>Lynch, M. and J. S. Conery, 2003 The origins of genome complexity. Science <span class="cit-vol">302</span>: 1401–1404.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mac Key</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Significance of mating systems for chromosomes and gametes in polyploids.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DKey%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D176%26rft.atitle%253DSignificance%2Bof%2Bmating%2Bsystems%2Bfor%2Bchromosomes%2Band%2Bgametes%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Apmid%252F5525824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=5525824&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970I602800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.42" data-doi="10.1126/science.1250092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marcussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandve</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spannagl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ancient hybridizations among the ancestral genomes of bread wheat.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1250092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1250092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarcussen%26rft.auinit1%253DT.%26rft.volume%253D345%26rft.issue%253D6194%26rft.spage%253D1250092%26rft.epage%253D1250092%26rft.atitle%253DAncient%2Bhybridizations%2Bamong%2Bthe%2Bancestral%2Bgenomes%2Bof%2Bbread%2Bwheat%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1250092%26rft_id%253Dinfo%253Apmid%252F25035499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDUvNjE5NC8xMjUwMDkyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.43" data-doi="10.1007/s00122-016-2675-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Epistasis and covariance: how gene interaction translates into genomic relationship.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2675-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D963%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2675-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2675-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.44" data-doi="10.1126/science.15739260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The significance of responses of the genome to challenge.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">266</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.15739260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D266%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.15739260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.15739260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.45" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.685.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Microsoft and R Core Team, Microsoft R Open 2017 Microsoft Corporation. Redmond, Washington. </span></li></ol><cite><a href="https://mran.microsoft.com">https://mran.microsoft.com</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.47" data-doi="10.2527/1997.7571738x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nejati-Javaremi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Effect of total allelic relationship on accuracy of evaluation and response to selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1738</span>–<span class="cit-lpage">1745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1997.7571738x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DNejati-Javaremi%26rft.auinit1%253DA.%26rft.volume%253D75%26rft.issue%253D7%26rft.spage%253D1738%26rft.epage%253D1745%26rft.atitle%253DEffect%2Bof%2Btotal%2Ballelic%2Brelationship%2Bon%2Baccuracy%2Bof%2Bevaluation%2Band%2Bresponse%2Bto%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1997.7571738x%26rft_id%253Dinfo%253Apmid%252F9222829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1997.7571738x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9222829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XH54200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.685.48" data-doi="10.1007/978-3-642-86659-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by Gene Duplication</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-86659-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Bby%2BGene%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-86659-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-86659-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.49" data-doi="10.1105/tpc.13.8.1735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Allopolyploidy-induced rapid genome evolution in the wheat (aegilops–triticum) group.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1735</span>–<span class="cit-lpage">1747</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.13.8.1735</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DOzkan%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1735%26rft.epage%253D1747%26rft.atitle%253DAllopolyploidy-Induced%2BRapid%2BGenome%2BEvolution%2Bin%2Bthe%2BWheat%2B%2528Aegilops-Triticum%2529%2BGroup%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1735%26rft_id%253Dinfo%253Apmid%252F11487689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTczNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.50" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.51" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0032253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.52" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.685.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.54" data-doi="10.1534/g3.112.005363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation of unordered markers and the impact on genomic selection accuracy.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.005363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.005363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.55" data-doi="10.1038/nrg817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Özkan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandolini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäfer-Pregl</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetics and geography of wild cereal domestication in the near east.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg817</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSalamini%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253D429%26rft.epage%253D441%26rft.atitle%253DGenetics%2Band%2Bgeography%2Bof%2Bwild%2Bcereal%2Bdomestication%2Bin%2Bthe%2Bnear%2Beast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg817%26rft_id%253Dinfo%253Apmid%252F12042770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12042770&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175942600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.56" data-doi="10.1534/g3.118.200646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018a</span> <span class="cit-article-title">A low resolution epistasis mapping approach to identify chromosome arm interactions in allohexaploid wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200646%26rft_id%253Dinfo%253Apmid%252F30455184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjkvMy82NzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.57" data-doi="10.1101/376731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018b</span> <span class="cit-article-title">A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">376731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/376731</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F376731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIzNzY3MzF2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.58" data-doi="10.1007/s00122-004-1784-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segovia-Lerma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Townsend</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population-based diallel analyses among nine historically recognized alfalfa germplasms.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1568</span>–<span class="cit-lpage">1575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1784-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSegovia-Lerma%26rft.auinit1%253DA.%26rft.volume%253D109%26rft.issue%253D8%26rft.spage%253D1568%26rft.epage%253D1575%26rft.atitle%253DPopulation-based%2Bdiallel%2Banalyses%2Bamong%2Bnine%2Bhistorically%2Brecognized%2Balfalfa%2Bgermplasms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1784-8%26rft_id%253Dinfo%253Apmid%252F15372154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1784-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15372154&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225054800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.59" data-doi="10.1093/bioinformatics/btr597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stekhoven</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bühlmann</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Missforest––non-parametric missing value imputation for mixed-type data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr597%26rft_id%253Dinfo%253Apmid%252F22039212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298383800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.60" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolff</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helentjaris</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Identification of genetic factors contributing to heterosis in a hybrid from two elite maize inbred lines using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D132%26rft.issue%253D3%26rft.spage%253D823%26rft.epage%253D839%26rft.atitle%253DIdentification%2Bof%2Bgenetic%2Bfactors%2Bcontributing%2Bto%2Bheterosis%2Bin%2Ba%2Bhybrid%2Bfrom%2Btwo%2Belite%2Bmaize%2Binbred%2Blines%2Busing%2Bmolecular%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F1468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzMvODIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.62" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1905-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.63" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Raden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.64" data-doi="10.1086/522934"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macgregor</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome partitioning of genetic variation for height from 11,214 sibling pairs.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1104</span>–<span class="cit-lpage">1110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/522934</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1104%26rft.epage%253D1110%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bheight%2Bfrom%2B11%252C214%2Bsibling%2Bpairs.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F522934%26rft_id%253Dinfo%253Apmid%252F17924350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/522934&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924350&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.65" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominant variance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.66" data-doi="10.1111/nph.12164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aegilops tauschii single nucleotide polymorphisms shed light on the origins of wheat d-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D198%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12164%26rft_id%253Dinfo%253Apmid%252F23374069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.67" data-doi="10.1534/g3.116.033332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Marker-based estimates reveal significant non-additive effects in clonally propagated cassava (manihot esculenta): implications for the prediction of total genetic value and the selection of varieties.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3497</span>–<span class="cit-lpage">3506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.033332</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.033332%26rft_id%253Dinfo%253Apmid%252F27587297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTEvMzQ5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82ODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.685.68" data-doi="10.1038/ng.823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasquale</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caporaso</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome partitioning of genetic variation for complex traits using common snps.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DGenome%2Bpartitioning%2Bof%2Bgenetic%2Bvariation%2Bfor%2Bcomplex%2Btraits%2Busing%2Bcommon%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.823%26rft_id%253Dinfo%253Apmid%252F21552263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552263&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F685.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/685.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/675" title="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/699" title="Genetic Effects on Dispersion in Urinary Albumin and Creatinine in Three House Mouse (Mus musculus) Cohorts" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/685" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EZh8egDdEtS_Abeb8hMacHpGeWf5SzPcyo0y1Vcpi7A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/685" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/685" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/685" class="active" data-icon-position="" data-hide-link-title="0">Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492493" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lAE7HKhnkjmmq5CJpAjQ3fid2amsbu84CiKFEthWPcI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/685.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/685" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zru7222UjwHfLQhOKVTL-T5ASjoNOeNW19UeXrYPn1s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/685" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/685.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492493" id="node492493--2" data-pisa="ggg;9/3/685" data-pisa-master="ggg;g3.118.200613" data-apath="/ggg/9/3/685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">685-698;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200613 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492493--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/685" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/685&amp;text=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/685&amp;t=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/685&amp;title=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492493/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492493" id="node492493--3" data-pisa="ggg;9/3/685" data-pisa-master="ggg;g3.118.200613" data-apath="/ggg/9/3/685.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">685-698;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200613 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492493--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, 240 Emerson Hall, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=685&amp;rft.epage=698&amp;rft.atitle=Prediction%20of%20Subgenome%20Additive%20and%20Interaction%20Effects%20in%20Allohexaploid%20Wheat&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Santantonio%2C+Nicholas&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Sorrells%2C+Mark"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492493/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492493/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492493/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492493/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492493/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492493/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492493/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492493/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492493/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492493/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492493/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492493/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/685" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1 Cnlm Dataset</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 2 Removal Of Population Structure From M</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/685.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/685.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/685.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/685.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
