<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/841" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/841.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/841.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/841.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/841/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/841.atom" />
<meta name="HW.pisa" content="ggg;9/3/841" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" />
<meta name="DC.Identifier" content="10.1534/g3.118.200921" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Xu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7607060" />
<meta name="DC.Description" content="Numerous quantitative trait loci (QTL) have been mapped in tetraploid and hexaploid wheat and wheat relatives, mostly with simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers. To conduct meta-analysis of QTL requires projecting them onto a common genomic framework, either a consensus genetic map or genomic sequence. The latter strategy is pursued here. Of 774 QTL mapped in wheat and wheat relatives found in the literature, 585 (75.6%) were successfully projected onto the Aegilops tauschii pseudomolecules. QTL mapped with SNP markers were more successfully projected (92.2%) than those mapped with SSR markers (66.2%). The QTL were not distributed homogeneously along chromosome arms. Their frequencies increased in the proximal-to-distal direction but declined in the most distal regions and were weakly correlated with recombination rates along the chromosome arms. Databases for projected SSR markers and QTL were constructed and incorporated into the Ae. tauschii JBrowse. To facilitate meta-QTL analysis, eight clusters of QTL were used to estimate standard deviations (![Formula][1]&lt;/img&gt;) of independently mapped QTL projected onto the Ae. tauschii genome sequence. The standard deviations ![Formula][2]&lt;/img&gt; were modeled as an exponential decay function of recombination rates along the Ae. tauschii chromosomes. We implemented four hypothesis tests for determining the membership of query QTL. The hypothesis tests and estimation procedure for ![Formula][3]&lt;/img&gt; were implemented in a web portal for meta-analysis of projected QTL. Twenty-one QTL for Fusarium head blight resistance mapped on wheat chromosomes 3A, 3B, and 3D were analyzed to illustrate the use of the portal for meta-QTL analyses.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif" />
<meta name="DC.Contributor" content="Jiale Xu" />
<meta name="DC.Contributor" content="Xiongtao Dai" />
<meta name="DC.Contributor" content="Ramesh K. Ramasamy" />
<meta name="DC.Contributor" content="Le Wang" />
<meta name="DC.Contributor" content="Tingting Zhu" />
<meta name="DC.Contributor" content="Patrick E. McGuire" />
<meta name="DC.Contributor" content="Chad M. Jorgensen" />
<meta name="DC.Contributor" content="Hamid Dehghani" />
<meta name="DC.Contributor" content="Patrick J. Gulick" />
<meta name="DC.Contributor" content="Ming-Cheng Luo" />
<meta name="DC.Contributor" content="Hans-Georg Müller" />
<meta name="DC.Contributor" content="Jan Dvorak" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Numerous quantitative trait loci (QTL) have been mapped in tetraploid and hexaploid wheat and wheat relatives, mostly with simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers. To conduct meta-analysis of QTL requires projecting them onto a common genomic framework, either a consensus genetic map or genomic sequence. The latter strategy is pursued here. Of 774 QTL mapped in wheat and wheat relatives found in the literature, 585 (75.6%) were successfully projected onto the &lt;i&gt;Aegilops tauschii&lt;/i&gt; pseudomolecules. QTL mapped with SNP markers were more successfully projected (92.2%) than those mapped with SSR markers (66.2%). The QTL were not distributed homogeneously along chromosome arms. Their frequencies increased in the proximal-to-distal direction but declined in the most distal regions and were weakly correlated with recombination rates along the chromosome arms. Databases for projected SSR markers and QTL were constructed and incorporated into the &lt;i&gt;Ae. tauschii&lt;/i&gt; JBrowse. To facilitate meta-QTL analysis, eight clusters of QTL were used to estimate standard deviations (&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;σ&lt;/mi&gt;&lt;mo&gt;^&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt;) of independently mapped QTL projected onto the &lt;i&gt;Ae. tauschii&lt;/i&gt; genome sequence. The standard deviations &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;σ&lt;/mi&gt;&lt;mo&gt;^&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt; were modeled as an exponential decay function of recombination rates along the &lt;i&gt;Ae. tauschii&lt;/i&gt; chromosomes. We implemented four hypothesis tests for determining the membership of query QTL. The hypothesis tests and estimation procedure for &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me3&quot; overflow=&quot;scroll&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;σ&lt;/mi&gt;&lt;mo&gt;^&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt; were implemented in a web portal for meta-analysis of projected QTL. Twenty-one QTL for &lt;i&gt;Fusarium&lt;/i&gt; head blight resistance mapped on wheat chromosomes 3A, 3B, and 3D were analyzed to illustrate the use of the portal for meta-QTL analyses.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/841" />
<meta name="citation_id" content="9/3/841" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/841" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/841.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/841.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/841.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200921" />
<meta name="citation_pmid" content="30670607" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="841" />
<meta name="citation_lastpage" content="853" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jiale Xu" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7602-5824" />
<meta name="citation_author" content="Xiongtao Dai" />
<meta name="citation_author_institution" content="Department of Statistics, Iowa State University, Iowa," />
<meta name="citation_author" content="Ramesh K. Ramasamy" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Le Wang" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Tingting Zhu" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Patrick E. McGuire" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Chad M. Jorgensen" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Hamid Dehghani" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author_institution" content="Department of Plant Breeding, Faculty of Agriculture, Tarbiat Modares University, Tehran, Iran, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3267-5144" />
<meta name="citation_author" content="Patrick J. Gulick" />
<meta name="citation_author_institution" content="Department of Biology, Concordia University, Montreal, Quebec, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7587-4616" />
<meta name="citation_author" content="Ming-Cheng Luo" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author" content="Hans-Georg Müller" />
<meta name="citation_author_institution" content="Department of Statistics, University of California, Davis, California," />
<meta name="citation_author" content="Jan Dvorak" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California," />
<meta name="citation_author_email" content="jdvorak@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. D. Akhunov;citation_author=C. Nicolet;citation_author=J. Dvorak;citation_title=Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay.;citation_pages=507-517;citation_volume=119;citation_year=2009;citation_issue=3;citation_pmid=19449174;citation_doi=10.1007/s00122-009-1059-5" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. D. Akhunov;citation_author=A. R. Akhunova;citation_author=O. D. Anderson;citation_author=J. A. Anderson;citation_author=N. Blake;citation_title=Nucleotide diversity maps reveal variation in diversity among wheat genomes and chromosomes.;citation_pages=702-702;citation_volume=11;citation_year=2010;citation_pmid=21156062;citation_doi=10.1186/1471-2164-11-702" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. D. Akhunov;citation_author=A. W. Goodyear;citation_author=S. Geng;citation_author=L. Qi;citation_author=B. Echalier;citation_title=The organization and rate of evolution of wheat genomes are correlated with recombination rates along chromosome arms.;citation_volume=13;citation_year=2003;citation_pmid=12695326;citation_doi=10.1101/gr.808603" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. M. Allen;citation_author=M. O. Winfield;citation_author=A. J. Burridge;citation_author=R. C. Downie;citation_author=H. R. Benbow;citation_title=Characterization of a Wheat Breeders’ Array suitable for high-throughput SNP genotyping of global accessions of hexaploid bread wheat (Triticum aestivum).;citation_pages=390-401;citation_volume=15;citation_year=2017;citation_doi=10.1111/pbi.12635" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Arbelbide;citation_author=R. Bernardo;citation_title=Mixed-model QTL mapping for kernel hardness and dough strength in bread wheat.;citation_pages=885-890;citation_volume=112;citation_year=2006;citation_issue=5;citation_pmid=16402188;citation_doi=10.1007/s00122-005-0190-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Avni;citation_author=M. Nave;citation_author=O. Barad;citation_author=K. Baruch;citation_author=S. O. Twardziok;citation_title=Wild emmer genome architecture and diversity elucidate wheat evolution and domestication;citation_pages=93-97;citation_volume=357;citation_year=2017;citation_issue=6346;citation_pmid=28684525;citation_doi=10.1126/science.aan0032" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Beales;citation_author=A. Turner;citation_author=S. Griffiths;citation_author=J. W. Snape;citation_author=D. A. Laurie;citation_title=A pseudo-response regulator is misexpressed in the photoperiod insensitive Ppd-D1a mutant of wheat (Triticum aestivum L.).;citation_pages=721-733;citation_volume=115;citation_year=2007;citation_issue=5;citation_pmid=17634915;citation_doi=10.1007/s00122-007-0603-4" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Börner;citation_author=J. Plaschke;citation_author=V. Korzun;citation_author=A. J. Worland;citation_title=The relationships between the dwarfing genes of wheat and rye.;citation_pages=69-75;citation_volume=89;citation_year=1996;citation_doi=10.1007/BF00015721" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. Dai;citation_author=H. Wang;citation_author=H. Zhou;citation_author=L. Wang;citation_author=J. Dvorak;citation_title=Birth and Death of LTR Retrotransposons in Aegilops tauschii.;citation_volume=210;citation_year=2018;citation_pmid=30158124;citation_doi=10.1534/genetics.118.301198" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. M. Devos;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_author=C. N. Chinoy;citation_author=M. D. Gale;citation_title=Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination.;citation_pages=282-288;citation_volume=91;citation_year=1995;citation_pmid=24169776;citation_doi=10.1007/BF00220890" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Dvorak;citation_author=M.-C. Luo;citation_author=Z. Yang;citation_title=Restriction Fragment Length Polymorphism and Divergence in the Genomic Regions of High and Low Recombination in Self-Fertilizing and Cross-Fertilizing Aegilops Species;citation_pages=423-434;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475752" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Dvorak;citation_author=P. Terlizzi;citation_author=H. B. Zhang;citation_author=P. Resta;citation_title=The evolution of polyploid wheats: identification of the A genome donor species.;citation_pages=21-31;citation_volume=36;citation_year=1993;citation_issue=1;citation_pmid=18469969;citation_doi=10.1139/g93-004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Dvorak;citation_author=E. D. Akhunov;citation_title=Tempos of gene locus deletions and duplications and their relationship to recombination rate during diploid and polyploid evolution in the Aegilops-Triticum alliance.;citation_volume=171;citation_year=2005;citation_pmid=15996988;citation_doi=10.1534/genetics.105.041632" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=J. Dvorak;citation_author=L. Wang;citation_author=T. Zhu;citation_author=C. M. Jorgensen;citation_author=K. R. Deal;citation_title=Structural variation and rates of genome evolution in the grass family seen through comparison of sequences of genomes greatly differing in size.;citation_pages=487-503;citation_volume=95;citation_year=2018;citation_doi=10.1111/tpj.13964" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Dvorak;citation_author=L. Wang;citation_author=T. Zhu;citation_author=C. M. Jorgensen;citation_author=M.-C. Luo;citation_title=Reassessment of the evolution of wheat chromosomes 4A, 5A, and 7B.;citation_pages=2451-2462;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3165-8" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. Dvorak;citation_author=H.-B. Zhang;citation_title=Variation in repeated nucleotide sequences sheds light on the phylogeny of the wheat B and G genomes.;citation_pages=9640-9644;citation_volume=87;citation_year=1990;citation_issue=24;citation_pmid=11607134;citation_doi=10.1073/pnas.87.24.9640" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Forde;citation_author=J.-M. Malpica;citation_author=N. G. Halford;citation_author=P. R. Shewry;citation_author=O. D. Anderson;citation_title=The nucleotide sequence of a HMW glutenin subunit gene located on chromosome 1A of wheat (Triticum aestivum L.);citation_pages=6817-6832;citation_volume=13;citation_year=1985;citation_issue=19;citation_pmid=2997729;citation_doi=10.1093/nar/13.19.6817" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=D. Fu;citation_author=P. Szűcs;citation_author=L. Yan;citation_author=M. Helguera;citation_author=J. S. Skinner;citation_title=Large deletions within the first intron in VRN-1 are associated with spring growth habit in barley and wheat.;citation_pages=54-65;citation_volume=273;citation_year=2005;citation_issue=1;citation_pmid=15690172;citation_doi=10.1007/s00438-004-1095-4" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=M. D. Gale;citation_author=G. A. Marshall;citation_title=Insensitivity to Gibberellin in Dwarf Wheats;citation_pages=729-735;citation_volume=37;citation_year=1973;citation_issue=4;citation_doi=10.1093/aob/" />
<meta name="citation_reference" content="citation_title=Progress in plant breeding;citation_author=M. D. Gale;citation_author=S. Youssefian;citation_author=G. Russell;citation_title=Dwarfing genes in wheat;citation_year=1985;citation_doi=10.1016/B978-0-407-00780-2.50005-9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Gale;citation_author=K. M. Devos;citation_title=Plant Comparative Genetics after 10 Years;citation_pages=656-659;citation_volume=282;citation_year=1998;citation_issue=5389;citation_pmid=9784118;citation_doi=10.1126/science.282.5389.656" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Gao;citation_author=W. Wen;citation_author=J. Liu;citation_author=A. Rasheed;citation_author=G. Yin;citation_title=Genome-wide linkage mapping of QTL for yield components, plant height and yield-related physiological traits in the Chinese wheat cross Zhou 8425B/Chinese Spring.;citation_pages=1099;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.01099" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=B. Garciadeblás;citation_author=M. E. Senn;citation_author=M. A. Bañuelos;citation_author=A. Rodríguez-Navarro;citation_title=Sodium transport and HKT transporters: the rice model.;citation_pages=788-801;citation_volume=34;citation_year=2003;citation_issue=6;citation_pmid=12795699;citation_doi=10.1046/j.1365-313X.2003.01764.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Goffinet;citation_author=S. Gerber;citation_title=Quantitative Trait Loci: A Meta-analysis;citation_pages=463-473;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790417" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Griffiths;citation_author=J. Simmonds;citation_author=M. Leverington;citation_author=Y. Wang;citation_author=L. Fish;citation_title=Meta-QTL analysis of the genetic control of ear emergence in elite European winter wheat germplasm.;citation_pages=383-395;citation_volume=119;citation_year=2009;citation_issue=3;citation_pmid=19430758;citation_doi=10.1007/s00122-009-1046-x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Hanocq;citation_author=M. Niarquin;citation_author=E. Heumez;citation_author=M. Rousset;citation_author=J. Le Gouis;citation_title=Detection and mapping of QTL for earliness components in a bread wheat recombinant inbred lines population.;citation_pages=106-115;citation_volume=110;citation_year=2004;citation_issue=1;citation_pmid=15551039;citation_doi=10.1007/s00122-004-1799-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Hanocq;citation_author=A. Laperche;citation_author=O. Jaminon;citation_author=A. L. Lainé;citation_author=J. Le Gouis;citation_title=Most significant genome regions involved in the control of earliness traits in bread wheat, as revealed by QTL meta-analysis.;citation_pages=569-584;citation_volume=114;citation_year=2007;citation_issue=3;citation_pmid=17171391;citation_doi=10.1007/s00122-006-0459-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Hou;citation_author=X. Chen;citation_author=M. Wang;citation_author=R. D. See;citation_author=S. Chao;citation_title=Mapping a Large Number of QTL for Durable Resistance to Stripe Rust in Winter Wheat Druchamp Using SSR and SNP Markers.;citation_pages=e0126794;citation_volume=10;citation_year=2015;citation_pmid=25970329;citation_doi=10.1371/journal.pone.0126794" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Q. Huang;citation_author=H. Cöster;citation_author=M. W. Ganal;citation_author=M. S. Röder;citation_title=Advanced backcross QTL analysis for the identification of quantitative trait loci alleles from wild relatives of wheat ( Triticum aestivum L.).;citation_pages=1379-1389;citation_volume=106;citation_year=2003;citation_issue=8;citation_pmid=12750781;citation_doi=10.1007/s00122-002-1179-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Huang;citation_author=S. Cloutier;citation_author=L. Lycar;citation_author=N. Radovanovic;citation_author=D. Humphreys;citation_title=Molecular detection of QTLs for agronomic and quality traits in a doubled haploid population derived from two Canadian wheats (Triticum aestivum L.).;citation_pages=753-766;citation_volume=113;citation_year=2006;citation_issue=4;citation_pmid=16838135;citation_doi=10.1007/s00122-006-0346-7" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=S. Huang;citation_author=W. Spielmeyer;citation_author=E. S. Lagudah;citation_author=R. Munns;citation_title=Comparative mapping of HKT genes in wheat, barley, and rice, key determinants of Na+ transport, and salt tolerance.;citation_volume=59;citation_year=2008;citation_pmid=18325922;citation_doi=10.1093/jxb/ern033" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=B. Hussain;citation_author=S. J. Lucas;citation_author=L. Ozturk;citation_author=H. Budak;citation_title=Mapping QTLs conferring salt tolerance and micronutrient concentrations at seedling stage in wheat.;citation_pages=15662;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. Jia;citation_author=S. Zhao;citation_author=X. Kong;citation_author=Y. Li;citation_author=G. Zhao;citation_title=Aegilops tauschii draft genome sequence reveals a gene repertoire for wheat adaptation.;citation_pages=91-95;citation_volume=496;citation_year=2013;citation_pmid=23535592;citation_doi=10.1038/nature12028" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. L. Jiang;citation_author=J. R. Shi;citation_author=R. W. Ward;citation_title=QTL analysis of resistance to Fusarium head blight in the novel wheat germplasm CJ 9306. I. Resistance to fungal spread.;citation_pages=3-13;citation_volume=116;citation_year=2007;citation_issue=1;citation_pmid=17898987;citation_doi=10.1007/s00122-007-0641-y" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=C. M. Jorgensen;citation_author=M.-C. Luo;citation_author=R. Ramasamy;citation_author=M. Dawson;citation_author=B. S. Gill;citation_title=A high-density genetic map of wild emmer wheat from the Karaca Dağ region provides new evidence on the structure and evolution of wheat chromosomes.;citation_pages=1798;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01798" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kikuchi;citation_author=K. Satoh;citation_author=T. Nagata;citation_author=N. Kawagashira;citation_author=K. Doi;citation_title=Collection, Mapping, and Annotation of Over 28,000 cDNA Clones from japonica Rice;citation_pages=376-379;citation_volume=301;citation_year=2003;citation_issue=5631;citation_pmid=12869764;citation_doi=10.1126/science.1081288" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Kunert;citation_author=A. A. Naz;citation_author=O. Dedeck;citation_author=K. Pillen;citation_author=J. Léon;citation_title=AB-QTL analysis in winter wheat: I. Synthetic hexaploid wheat (T. turgidum ssp. dicoccoides x T. tauschii) as a source of favourable alleles for milling and baking quality traits.;citation_pages=683-695;citation_volume=115;citation_year=2007;citation_issue=5;citation_pmid=17634917;citation_doi=10.1007/s00122-007-0600-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. R. Lin;citation_author=K. F. Schertz;citation_author=A. H. Paterson;citation_title=Comparative analysis of QTLs affecting plant height and maturity across the Poaceae, in reference to an interspecific sorghum population.;citation_pages=391-411;citation_volume=141;citation_year=1995;citation_issue=1;citation_pmid=8536986" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=M. P. Lindsay;citation_author=E. S. Lagudah;citation_author=R. A. Hare;citation_author=R. Munns;citation_title=A locus for sodium exclusion (Nax1), a trait for salt tolerance, mapped in durum wheat.;citation_pages=1105-1114;citation_volume=31;citation_year=2004;citation_doi=10.1071/FP04111" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=W. Liu;citation_author=M. Frick;citation_author=R. Huel;citation_author=C. L. Nykiforuk;citation_author=X. Wang;citation_title=The stripe rust resistance gene Yr10 encodes an evolutionary-conserved and unique CC-NBS-LRR sequence in wheat.;citation_volume=7;citation_year=2014;citation_pmid=25336565;citation_doi=10.1093/mp/ssu112" />
<meta name="citation_reference" content="citation_journal_title=Feddes Repert.;citation_author=Á. Löve;citation_title=Conspectus of the Triticeae.;citation_pages=425-521;citation_volume=95;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M.-C. Luo;citation_author=K. R. Deal;citation_author=E. D. Akhunov;citation_author=A. R. Akhunova;citation_author=O. D. Anderson;citation_title=Genome comparisons reveal a dominant mechanism of chromosome number reduction in grasses and accelerated genome evolution in Triticeae.;citation_volume=106;citation_year=2009;citation_pmid=19717446;citation_doi=10.1073/pnas.0908195106" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M.-C. Luo;citation_author=Y. Q. Gu;citation_author=F. M. You;citation_author=K. R. Deal;citation_author=Y. Ma;citation_title=A 4-gigabase physical map unlocks the structure and evolution of the complex genome of Aegilops tauschii, the wheat D-genome progenitor.;citation_volume=110;citation_year=2013;citation_pmid=23610408;citation_doi=10.1073/pnas.1219082110" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M.-C. Luo;citation_author=Y. Q. Gu;citation_author=D. Puiu;citation_author=H. Wang;citation_author=S. O. Twardziok;citation_title=Genome sequence of the progenitor of the wheat D genome Aegilops tauschii.;citation_pages=498;citation_volume=551;citation_year=2017;citation_pmid=29143815;citation_doi=10.1038/nature24486" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=M. Maccaferri;citation_author=A. Ricci;citation_author=S. Salvi;citation_author=S. G. Milner;citation_author=E. Noli;citation_title=A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.;citation_pages=648-663;citation_volume=13;citation_year=2015;citation_pmid=25424506;citation_doi=10.1111/pbi.12288" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Marone;citation_author=G. Laidò;citation_author=A. Gadaleta;citation_author=P. Colasuonno;citation_author=D. B. M. Ficco;citation_title=A high-density consensus map of A and B wheat genomes.;citation_pages=1619-1638;citation_volume=125;citation_year=2012;citation_issue=8;citation_pmid=22872151;citation_doi=10.1007/s00122-012-1939-y" />
<meta name="citation_reference" content="citation_journal_title=J. Cereal Sci.;citation_author=W. Maruyama-Funatsuki;citation_author=K. Takata;citation_author=H. Funatsuki;citation_author=T. Tabiki;citation_author=M. Ito;citation_title=Identification and characterization of a novel LMW-s glutenin gene of a Canadian western extra-strong wheat.;citation_pages=47-57;citation_volume=41;citation_year=2005;citation_doi=10.1016/j.jcs.2004.07.003" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Masci;citation_author=R. D. Ovidio;citation_author=D. Lafiandra;citation_author=D. Kasarda;citation_title=A 1B-coded low-molecular-weight glutenin subunit associated with quality in durum wheats shows strong similarity to a subunit present in some bread wheat cultivars.;citation_pages=396-400;citation_volume=100;citation_year=2000;citation_doi=10.1007/s001220050052" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=C. A. McCartney;citation_author=D. J. Somers;citation_author=D. G. Humphreys;citation_author=O. Lukow;citation_author=N. Ames;citation_title=Mapping quantitative trait loci controlling agronomic traits in the spring wheat cross RL4452x'AC Domain'.;citation_pages=870-883;citation_volume=48;citation_year=2005;citation_issue=5;citation_pmid=16391693;citation_doi=10.1139/g05-055" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=E. McFadden;citation_author=E. Sears;citation_title=THE ORIGIN OF TRITICUM SPELTA AND ITS FREE-THRESHING HEXAPLOID RELATIVES;citation_pages=81-89;citation_volume=37;citation_year=1946;citation_issue=3;citation_pmid=20985728;citation_doi=10.1093/oxfordjournals.jhered.a105590" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Metzger;citation_author=B. Silbaugh;citation_title=Inheritance of Resistance to Stripe Rust and its Association with Brown Glume Color in Tritieum aestivum L., 'P.I. 178383';citation_pages=567-568;citation_volume=10;citation_year=1970;citation_issue=5;citation_doi=10.2135/cropsci1970.0011183X001000050035x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Naruoka;citation_author=K. A. Garland-Campbell;citation_author=A. H. Carter;citation_title=Genome-wide association mapping for stripe rust (Puccinia striiformis F. sp. tritici) in US Pacific Northwest winter wheat (Triticum aestivum L.).;citation_pages=1083-1101;citation_volume=128;citation_year=2015;citation_pmid=25754424;citation_doi=10.1007/s00122-015-2492-2" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Paillard;citation_author=T. Schnurbusch;citation_author=R. Tiwari;citation_author=M. Messmer;citation_author=M. Winzeler;citation_title=QTL analysis of resistance to Fusarium head blight in Swiss winter wheat (Triticum aestivum L.).;citation_pages=323-332;citation_volume=109;citation_year=2004;citation_issue=2;citation_pmid=15014875;citation_doi=10.1007/s00122-004-1628-6" />
<meta name="citation_reference" content="citation_journal_title=J. Sci. Food Agric.;citation_author=P. I. Payne;citation_author=K. G. Corfield;citation_author=L. M. Holt;citation_author=J. A. Blackman;citation_title=Correlations between the inheritance of certain high‐molecular weight subunits of glutenin and bread‐making quality in progenies of six crosses of bread wheat.;citation_pages=51-60;citation_volume=32;citation_year=1981;citation_doi=10.1002/jsfa.2740320109" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Z. Peleg;citation_author=T. Fahima;citation_author=A. B. Korol;citation_author=S. Abbo;citation_author=Y. Saranga;citation_title=Genetic analysis of wheat domestication and evolution under domestication.;citation_volume=62;citation_year=2011;citation_pmid=21778183;citation_doi=10.1093/jxb/err206" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Peng;citation_author=Y. Ronin;citation_author=T. Fahima;citation_author=M. S. Roder;citation_author=Y. Li;citation_title=Domestication quantitative trait loci in Triticum dicoccoides, the progenitor of wheat.;citation_volume=100;citation_year=2003;citation_pmid=12604784;citation_doi=10.1073/pnas.252763199" />
<meta name="citation_reference" content="citation_journal_title=Cereal Res. Commun.;citation_author=J. H. Peng;citation_author=D. F. Sun;citation_author=Y. L. Peng;citation_author=E. Nevo;citation_title=Gene discovery in Triticum dicoccoides, the direct progenitor of cultivated wheats.;citation_pages=1-22;citation_volume=41;citation_year=2013;citation_doi=10.1556/CRC.2012.0030" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=M. J. Pinthus;citation_author=M. D. Gale;citation_author=N. E. Appleford;citation_author=J. R. Lenton;citation_title=Effect of Temperature on Gibberellin (GA) Responsiveness and on Endogenous GA1 Content of Tall and Dwarf Wheat Genotypes;citation_pages=854-859;citation_volume=90;citation_year=1989;citation_issue=3;citation_pmid=16666888;citation_doi=10.1104/pp.90.3.854" />
<meta name="citation_reference" content="citation_journal_title=J. Cereal Sci.;citation_author=N. Pogna;citation_author=D. Lafiandra;citation_author=P. Feillet;citation_author=J. Autran;citation_title=Evidence for a direct causal effect of low molecular weight subunits of glutenins on gluten viscoelasticity in durum wheats.;citation_pages=211-214;citation_volume=7;citation_year=1988;citation_doi=10.1016/S0733-5210(88)80001-8" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=U. M. Quraishi;citation_author=C. Pont;citation_author=Q.-U. Ain;citation_author=R. Flores;citation_author=L. Burlot;citation_title=Combined genomic and genetic data integration of major agronomical traits in bread wheat (Triticum aestivum L.).;citation_pages=1843;citation_volume=8;citation_year=2017;citation_doi=10.3389/fpls.2017.01843" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. Rawat;citation_author=M. O. Pumphrey;citation_author=S. X. Liu;citation_author=X. F. Zhang;citation_author=V. K. Tiwari;citation_title=Wheat Fhb1 encodes a chimeric lectin with agglutinin domains and a pore-forming toxin-like domain conferring resistance to Fusarium head blight.;citation_pages=1576-1580;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3706" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=M. S. Röder;citation_author=V. Korzun;citation_author=B. S. Gill;citation_author=M. W. Ganal;citation_title=The physical mapping of microsatellite markers in wheat.;citation_pages=278-283;citation_volume=41;citation_year=1998;citation_doi=10.1139/g98-009" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=P. J. Rousseeuw;citation_author=C. Croux;citation_title=Alternatives to the median absolute deviation.;citation_pages=1273-1283;citation_volume=88;citation_year=1993;citation_doi=10.2307/2291267" />
<meta name="citation_reference" content="citation_title=Chromosome Manipulations and Plant Genetics;citation_author=E. R. Sears;citation_title=Nullisomic-tetrasomic combinations in hexaploid wheat;citation_year=1966;citation_doi=10.1007/978-1-4899-6561-5_4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. J. Somers;citation_author=P. Isaac;citation_author=K. Edwards;citation_title=A high-density microsatellite consensus map for bread wheat (Triticum aestivum L.).;citation_pages=1105-1114;citation_volume=109;citation_year=2004;citation_issue=6;citation_pmid=15490101;citation_doi=10.1007/s00122-004-1740-7" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X.-J. Song;citation_author=W. Huang;citation_author=M. Shi;citation_author=M.-Z. Zhu;citation_author=H.-X. Lin;citation_title=A QTL for rice grain width and weight encodes a previously unknown RING-type E3 ubiquitin ligase.;citation_pages=623-630;citation_volume=39;citation_year=2007;citation_issue=5;citation_pmid=17417637;citation_doi=10.1038/ng2014" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=P. Sourdille;citation_author=G. Charmet;citation_author=M. Trottet;citation_author=M. Tixier;citation_author=C. Boeuf;citation_title=Linkage between RFLP molecular markers and the dwarfing genes Rht‐B1 and Rht‐D1 in wheat.;citation_pages=41-46;citation_volume=128;citation_year=1998;citation_doi=10.1111/j.1601-5223.1998.00041.x" />
<meta name="citation_reference" content="citation_journal_title=Biotechnol. Biotechnol. Equip.;citation_author=P. T. Thanh;citation_author=C. I. Vladutu;citation_author=S. F. Kianian;citation_author=P. T. Thanh;citation_author=T. Ishii;citation_title=Molecular genetic analysis of domestication traits in emmer wheat. I: Map construction and QTL analysis using an F2 population.;citation_pages=3627-3637;citation_volume=27;citation_year=2013;citation_doi=10.5504/BBEQ.2013.0008" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. Tyagi;citation_author=R. R. Mir;citation_author=H. S. Balyan;citation_author=P. K. Gupta;citation_title=Interval mapping and meta-QTL analysis of grain traits in common wheat (Triticum aestivum L.).;citation_pages=367-380;citation_volume=201;citation_year=2015;citation_doi=10.1007/s10681-014-1217-y" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=R. Tzarfati;citation_author=V. Barak;citation_author=T. Krugman;citation_author=T. Fahima;citation_author=S. Abbo;citation_title=Novel quantitative trait loci underlying major domestication traits in tetraploid wheat.;citation_pages=1613-1628;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0182-4" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J.-B. Veyrieras;citation_author=B. Goffinet;citation_author=A. Charcosset;citation_title=MetaQTL: a package of new computational methods for the meta-analysis of QTL mapping experiments.;citation_pages=49-49;citation_volume=8;citation_year=2007;citation_pmid=17288608;citation_doi=10.1186/1471-2105-8-49" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=B. L. Waldron;citation_author=B. Moreno-Sevilla;citation_author=J. A. Anderson;citation_author=R. W. Stack;citation_author=R. C. Frohberg;citation_title=RFLP Mapping of QTL for Fusarium Head Blight Resistance in Wheat;citation_pages=805-811;citation_volume=39;citation_year=1999;citation_issue=3;citation_doi=10.2135/cropsci1999.0011183X003900030032x" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Wang;citation_author=M.-C. Luo;citation_author=Z. Chen;citation_author=F. M. You;citation_author=Y. Wei;citation_title=Aegilops tauschii single nucleotide polymorphisms shed light on the origins of wheat D‐genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.;citation_pages=925-937;citation_volume=198;citation_year=2013;citation_pmid=23374069;citation_doi=10.1111/nph.12164" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Wang;citation_author=D. Wong;citation_author=K. Forrest;citation_author=A. Allen;citation_author=S. Chao;citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.;citation_volume=12;citation_year=2014;citation_pmid=24646323;citation_doi=10.1111/pbi.12183" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. P. Wilhelm;citation_author=A. S. Turner;citation_author=D. A. Laurie;citation_title=Photoperiod insensitive Ppd-A1a mutations in tetraploid wheat (Triticum durum Desf.).;citation_pages=285-294;citation_volume=118;citation_year=2009;citation_issue=2;citation_pmid=18839130;citation_doi=10.1007/s00122-008-0898-9" />
<meta name="citation_reference" content="citation_title=The world wheat book: A history of wheat breeding;citation_title=Genetic basis of worldwide wheat varietal improvement;citation_author=T. Worland;citation_pages=59-100;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=PLOS ONE;citation_author=Q.-H. Wu;citation_author=Y.-X. Chen;citation_author=S.-H. Zhou;citation_author=L. Fu;citation_author=J.-J. Chen;citation_title=High-density genetic linkage map construction and QTL mapping of grain shape and size in the wheat population Yanda1817 × Beinong6.;citation_pages=e0118144;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=X. Xu;citation_author=G. Bai;citation_author=B. F. Carver;citation_author=G. E. Shaner;citation_title=A QTL for early heading in wheat cultivar Suwon 92.;citation_pages=233-237;citation_volume=146;citation_year=2005;citation_doi=10.1007/s10681-005-9017-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Yan;citation_author=A. Loukoianov;citation_author=G. Tranquilli;citation_author=M. Helguera;citation_author=T. Fahima;citation_title=Positional cloning of the wheat vernalization gene VRN1.;citation_volume=100;citation_year=2003;citation_pmid=12730378;citation_doi=10.1073/pnas.0937399100" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Yu;citation_author=H. Zhang;citation_author=X.-L. Zhou;citation_author=D.-B. Hou;citation_author=G.-Y. Chen;citation_title=Quantitative trait loci associated with agronomic traits and stripe rust in winter wheat mapping population using single nucleotide polymorphic markers.;citation_pages=105;citation_volume=37;citation_year=2017;citation_doi=10.1007/s11032-017-0704-y" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=G. Zhao;citation_author=C. Zou;citation_author=K. Li;citation_author=K. Wang;citation_author=T. Li;citation_title=The Aegilops tauschii genome reveals multiple impacts of transposons.;citation_pages=946-955;citation_volume=3;citation_year=2017;citation_pmid=29158546;citation_doi=10.1038/s41477-017-0067-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Y. Zhou;citation_author=B. Conway;citation_author=D. Miller;citation_author=D. Marshall;citation_author=A. Cooper;citation_title=Quantitative trait loci mapping for spike characteristics in hexaploid wheat.;citation_pages=2;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.10.0101" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Zou;citation_author=K. Semagn;citation_author=M. Iqbal;citation_author=A. N’Diaye;citation_author=H. Chen;citation_title=Mapping QTLs controlling agronomic traits in the ‘Attila’ × ‘CDC Go’ spring wheat population under organic management using 90K SNP array.;citation_pages=365-377;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2016.06.0459" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Numerous quantitative trait loci (QTL) have been mapped in tetraploid and hexaploid wheat and wheat relatives, mostly with simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers. To conduct meta-analysis of QTL requires projecting them onto a common genomic framework, either a consensus genetic map or genomic sequence. The latter strategy is pursued here. Of 774 QTL mapped in wheat and wheat relatives found in the literature, 585 (75.6%) were successfully projected onto the Aegilops tauschii pseudomolecules. QTL mapped with SNP markers were more successfully projected (92.2%) than those mapped with SSR markers (66.2%). The QTL were not distributed homogeneously along chromosome arms. Their frequencies increased in the proximal-to-distal direction but declined in the most distal regions and were weakly correlated with recombination rates along the chromosome arms. Databases for projected SSR markers and QTL were constructed and incorporated into the Ae. tauschii JBrowse. To facilitate meta-QTL analysis, eight clusters of QTL were used to estimate standard deviations (![Formula][1]&lt;/img&gt;) of independently mapped QTL projected onto the Ae. tauschii genome sequence. The standard deviations ![Formula][2]&lt;/img&gt; were modeled as an exponential decay function of recombination rates along the Ae. tauschii chromosomes. We implemented four hypothesis tests for determining the membership of query QTL. The hypothesis tests and estimation procedure for ![Formula][3]&lt;/img&gt; were implemented in a web portal for meta-analysis of projected QTL. Twenty-one QTL for Fusarium head blight resistance mapped on wheat chromosomes 3A, 3B, and 3D were analyzed to illustrate the use of the portal for meta-QTL analyses.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif" />
<meta name="og-title" property="og:title" content="Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/841" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Numerous quantitative trait loci (QTL) have been mapped in tetraploid and hexaploid wheat and wheat relatives, mostly with simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers. To conduct meta-analysis of QTL requires projecting them onto a common genomic framework, either a consensus genetic map or genomic sequence. The latter strategy is pursued here. Of 774 QTL mapped in wheat and wheat relatives found in the literature, 585 (75.6%) were successfully projected onto the Aegilops tauschii pseudomolecules. QTL mapped with SNP markers were more successfully projected (92.2%) than those mapped with SSR markers (66.2%). The QTL were not distributed homogeneously along chromosome arms. Their frequencies increased in the proximal-to-distal direction but declined in the most distal regions and were weakly correlated with recombination rates along the chromosome arms. Databases for projected SSR markers and QTL were constructed and incorporated into the Ae. tauschii JBrowse. To facilitate meta-QTL analysis, eight clusters of QTL were used to estimate standard deviations (![Formula][1]&lt;/img&gt;) of independently mapped QTL projected onto the Ae. tauschii genome sequence. The standard deviations ![Formula][2]&lt;/img&gt; were modeled as an exponential decay function of recombination rates along the Ae. tauschii chromosomes. We implemented four hypothesis tests for determining the membership of query QTL. The hypothesis tests and estimation procedure for ![Formula][3]&lt;/img&gt; were implemented in a web portal for meta-analysis of projected QTL. Twenty-one QTL for Fusarium head blight resistance mapped on wheat chromosomes 3A, 3B, and 3D were analyzed to illustrate the use of the portal for meta-QTL analyses.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif
 [3]: /embed/mml-math-3.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"j40q0SVyUG2fmU55bQkUlDfSnvHZu4KBo2hWuCDXtKA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492432","apath":"\/ggg\/9\/3\/841.atom","pisa":"ggg;9\/3\/841","ac":{"\/ggg\/9\/3\/841.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/841.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/841"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/841"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/841":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yXaiOcLBYUPhmbKxaTDYWi5pGyxXvIthgeT\\\\\\\/9Bxuj0a5nc31O5rwbWxP0\\\\\\\/vr\\\\\\\/TcDxypYxTrYMj8aITe1ObUeD0LMjG7MPEprQvSZ6t7xyrJXnUAcdg8SfglomrX9DFUzURJo1p5WAUgUtRPgX2AEK\\\\\\\/9xiwuGcWjQWmhQt3RPLHCp03BnMDtE7338cIeilVs153Nsri+qtg4uvrFxTlB2zCwEqSCa8RmcnGM5b16PkidxURam5yOlv3MkrNXL64nSyX7uxdb3YCIcp5B5rBj8YvnlQPrGp4\\\\\\\/6liCeN\\\\\\\/R34J7JL0kLT0Ojof4j2nN7kJDbEI7pWdgt6Q6BR7CtlHOwgr4rNZn88PiBfQNWUCeaD9sI9UHibB15M+MsULo3zZPU5v9lnB1ezt72F78lwvSMT\\\\\\\/KUZWUIB4f9udKU4pAFRX41e6o+Jl25YSWIpInlM+ltw6bvAcBhi5ZH13rgq3KThgxzUr3rYQk1PUajOE++gFznC8DOxIXX2CXWKin\\\\\\\/D6ib4YD8iXnQowin7D3npf\\\\\\\/35jJ+IRSzjbSTIav9te+lD79zs4as4oxj\\\\\\\/qmsefAjBLm3EpM6+0gD4GgBR5RccHPlrYqAcVtRrSnhi54bseiQqpKHlGMkbJqa\\\\\\\/UgaA3\\\\\\\/rh33eTLoj6Do5qgwL2xPM8TBlvtJDo\\\\\\\/9oBSJzViWXwpt7G5K1qqbJF1xgD3e24jnz8Mqg5luOUYPBZM9p4trstBDmrLuUwnXfyKC83Ju12Aa25Xf1EV812Fwsj1R+tuvX84UsFhN1WQfdFzX61HxLOJZMLgnO89akAcRAp2xbMznoz06TA6poVX8sYkLpnlFNGtfFlJW48H18JWrgKpkGxTpEzs14nsjEo8uHSQNtkssMrlpYYa5\\\\\\\/KIPrGehygI2OoWe7szPW9n+YGt92Gxnp0RrPoIMgGMxyenwTPM6UQ15yigFq7wCeXCdYU00D3Et79g2xFU0g2cRfMp3daCZmWCyZZ7dz+y\\\\\\\/ceEdexF5E4kActZJeGG+FVs+f19LTuOewCig5AaBQwFhh670Ho9a4gJmyF7YFtcl35cYu\\\\\\\/EeTmiRO8oZPcf4Brl1cFmX+U\\\\\\\/ZrVcHHoDz9N9dSnTtyd0CnS8PsyY5GLROnLfPOB9tZsqLU8LbCfLfQuIBpveglgdpIJ3RsAZLr1RjMPoRSf4Ix+kTfZ1klBbA2FPZl+oaYhuJSycvcCwZJnNt+SOlDA5EUFWMeXYPjqxkVD5dHRUt1VbX3G0cGoh0odFUzbOkPq5fhUdjP4tPcZrCVi95E+KFLDIy6\\\u0022,\\\u0022iv\\\u0022:\\\u0022y\\\\\\\/UjGbWM8YEAWzNDJ+K3dQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ecab9fd23daa46703f714dfd1b3ed865425c9bc27308e8f6be6a74fbc33cae48\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WXhnwBl2E2cdqmcLFrsoQ36OpDg4xhgQGi4\\\\\\\/0GOMaur2H+114+xNs1uvgsFpDC6bncJ5RrP2P2KelnN7eFl6rfU26zYQb9MwJucoETJua6ekhaNtX8+4RpXAUx6ICnzua2EP0Q+cqkwlx8eIiaWYh+s23XAGHViwxvoffN6hR+8KV8nRu5AGqDG1RAJ66mBW1NOjdRyBpxiQ27hF9n9549fR+\\\\\\\/V\\\\\\\/Aaw1zpIzwHTwhJVeUo6KYRtcvGPzsEj4YQ5fzuqB9kvz6+yLX8WVflTobTp6Qp\\\\\\\/Xvdt230N3ph2p0KlV\\\\\\\/NVsMMswWjA1UOLTISK725k+tzCyXoHwnTR4g7hKDDJXXFiHvdTNm\\\\\\\/R9hzEQV57EFPLeMgHI4LKerX1ZrBSe4BqyZxaWD\\\\\\\/Bb1KJdePlO06obAP6v8lC\\\\\\\/QJpK1dZPzJfk3coNzSu3Vrln53fM\\\\\\\/gIqbsM+rRhXQDvq\\\\\\\/OYDNMH6WwfgunDGGWONUz\\\\\\\/RaRkOuYmmM4p4MKLzUlM2+OvIUxsGKlFDw+Ao8OQh8i6puR6TB6AjEzQ9hB9FO8624pR32hyu8T++495pAPI4yqAvQLTnfSBrKHGGeYdjcsUdC0IF1wPGf+F+5nPH\\\\\\\/A5lAbzVKM\\\\\\\/SCqkmRHx2MJL4x+Swq3CoGnSRsKIP8gNdJuBnKoqOwzz61AaqbAKCPVeIgdE\\\\\\\/EU6WkJ+98Mm6YGTmEgsTsNVK72s6qZbWmjNcw15oDbUKL8YBr\\\\\\\/rMt808F72VAQnum2bnV758gwuVQRhN9gMjxRVmEHel\\\\\\\/Qagz4z4R+j7+CNvseTsRV0VnGbTv\\\\\\\/ZcWu3V7QiHk7DIWs6B5yT81g+X+uiR2GfG29z54pXIDtM3aXaIi2tP7CbcKWCsK4QKZihStqZFVRQ4AIt7NDbF4Ld7wXjAn4uQe3b8i1U0P45PLECqFSRZtCO4DwETo\\\\\\\/l33li5T2l3KCnAXcOYRfnd2Eu6\\\\\\\/mAMQURBvdTXINuga84S4IQUzmcVVuTrbLtgt\\\\\\\/Wne0A653wvz5znB912\\\\\\\/l6WpI0llhnM3ii97VrienhARO16Or7gjgxvQ+YLDz18qGGGfqexdPfTOc5o6cM7gvWCBgoVHYiKK\\\\\\\/mwMoLP8zO+DZYS2uCE6P+Om\\\\\\\/+yTLQDM3vdaPKj+nge8\\\\\\\/3VL6jOTd1Np5hrw9a+ZRBDzjJrdEssjn1n71jmntTt9WVEuX094sJNSCVhV7H8B1oYUabm1qhJt8CYcXe7HDjeO1UQCkMereAVk9MY\\\\\\\/8rTQSKExaPZfqkDEGrzOCb2aiAPBuPla62\\\\\\\/4qwK+7oqCphRAvrzocf53uFd1losmEAhetqjhckFxn9gOw5LjSW5MvqbBvgPFyUv2WLdtlJhbZDuQzjfhP+Cht6i2FU9Ei\\\\\\\/DkLUZYMsgclYeRlnURIWvK1k\\\\\\\/Wyi\\\\\\\/ox0zkbc9GO2Y2bhXJXTua9RHNUU3V2evZjZ60zvDcf7Ns6j1Gqfqq87uoBXkWTNxvZzY119KxAVxuB32BhE4h5vXTw2UpN3DG1UOoZaDRFK8kOZT2m3ow2eeCjhkIRQBy0iE9SZ9zx9Oy\\\\\\\/qv6nW\\\\\\\/egRUx7jmBbgDrgIlsrcyKmVgb9Krv6ohoHyRC8n0P\\\\\\\/pAMwkSJ7aisRM2AuMR29aJDAvZHkI9oDq2vhy+8LhczNI+3\\\\\\\/FhHQ1fVXGUq3jSr\\\\\\\/cbseCHTAtW0aWWd+mdOIbajM4i7eKmzRBvjxa\\\\\\\/QF0eOO59wKLGW9bQAliJEL2SALwVCJmrQTLaSXvod7fKbfpy0F8KRprfZ\\\\\\\/rGBPhTF4JmVL2kc+33fr3uG6tfNXFA\\\\\\\/rR9tC5omjoAjpbisrDudzU+xVw2\\\\\\\/fMi3EvoB40rhkVU57hAz3j6\\\\\\\/Slzid67t7TH4SFRIdDTYeTAPGS+8Jo+zxFmtaGoLEdIhrsxxUle3Dke9dk53G8znpwjDf8s4hzIK5WyIC5kplxoxyBZTilPYBnxQOXogVicuV13FjH0QhMGcxsLX2BPqodW\\\\\\\/kfUOfwt\\\\\\\/4RR0AIi5mNK03QpZKXp0KknPrnSwHPXwz\\\\\\\/E+qaP3f9HnF+l\\\\\\\/rIi72V8SwfoMpFx780cPvHX9I8RRwq+ER2+mMttFUm6MCEXvaWHtEBkcEocp6GQ+M3NCfJsWERK7Dy8RP8+gEJ7efHLjgjL+dXDROEmyrnIOhOzxUczfcTBcPiUaU\\\\\\\/rCmcStva31QkCroi3jlb2iE6qMDq9MirAKm+5rPA7ZCijD+cK76HkAaQ03p8cwFSy\\\\\\\/xebOMhIW9371MvFNhkNVCnVLZRP18\\\\\\\/Ov2I8ns361QWiDHN3v8DZ58tcZHqSnjJEbAWMC4rCOJG6T7Vi4wP9oAdIBPqhiY7AD4+pglJgVxzT7vHT\\\\\\\/01tgqp0ClNnQSOZOsy0\\\\\\\/ykK1VlZhZzmOyYPedw1iN6STc4yXiwZl1atcdvAX\\\\\\\/jckLAKqNye2i+q0Ja7FYrluBOsXePjXmC0aV60lSuwGX1Mi68rkB58\\\\\\\/cyFR6Ksk8zTtpLd+BKehIbDSiaQ2fAUlbZZZC68N0H8vIkyQCtQxNVh9RndiYa0\\\\\\\/H2Jp9CRrCTTAfR9yRvRRKjPOpb9s8ubYttnmRIxPiOekqpCsQ\\\\\\\/09cbZSUwLoZ+Zm3dKUnzCJKTZFnHG06N3lIaFilt\\\\\\\/OAwtXmzT8zRlhyYBX+E+3PTOAKkij00\\\\\\\/1Ec37NbdiZjABkKHL6ecbnS5x1NADuzMHxnAqTEIx7KVM+9kMuB9TCOUMmjo4KA5T9R9sUfpBVPmkOrYqa6Okqyt+XNui5K0SPCcfjCQ5G3I\\\\\\\/2B4dtbz7GX4DIPTNFsVe3yVVWhWV3PeY8DuVYb2p1WAGFC0tHH6JlCBjFaXBOoGFuepAW\\\\\\\/LfWI2X1CiGUaQmYVV70jAxaphGtkeHjfctdrd09t4jUAbRN4KKyG2orknQ2e2iVxFbyCVqGrOthLz91EarRvVOwicDk2UscZh72B6ApaWQ7iOc7BBQWu3UskMQlm3haMF7S+UZ0mEFefEiu31K51vy\\\\\\\/gboNJ4GIKB1wrJelrbb6qBWqr19jVxkRlmE5ZXWeQsiXmEfSCUGWSqMQQjwNjqI9hwPzG4R\\\\\\\/n5EwjYIrEEVUm0j5\\\\\\\/WqHVfN9icTEy4ArSBiawwTS35mYmiF6mB9\\\\\\\/l2OwN1WRJGa9x3T7RSj3zMlGc5NQB37dwgOsT78eeBllcpyson\\\\\\\/mDYdLqKgU+FzvJ7NsL\\\\\\\/bKnNc00ap5Gz1SBxHhkPVPxR3+8m640d2vvbmpVBb7ANIGMjzCUKkcR\\\\\\\/0\\\\\\\/8kEdoxcMdvvUaHGteWo5l9pOdV+tV5ipTLoYvZxJcTFfeU89nGjzso4pUZk\\\\\\\/GjsPxH\\\\\\\/nHF5AUMHFRlZfibsqM++qS+\\\\\\\/gYhdYKLB9A4zVIPAoB7Sfz6nv7ju58WMZv2bfevrVWjbXpjW2ZFmH7AVD6z7eI8P2rIsvIU5GgsYB1F+ln7uPdl\\\\\\\/1s2wiWn+DFCpx60RERhM6AiWoCHiUBGtqoLZTubfe70E7x7KXHmHudPK+dwiSQh0NZ0QUOzUQoLn\\\\\\\/vSs7T4Lx\\\\\\\/BtxWDyAEzQjIdmF6DzfIuiuIRWZBMt1eq6kgqICSNaA9pRrZ6WahPpnUGKNuVqw+rnAjny4IZYNIK4ufV9RACiBs5De5HaWUiAQQ9Zd9XrbSKIdrA3KeASOUOTMNw1DmnVOFo1yqEzWaFW7eqYHpU\\\\\\\/KE0PPIedx6Q6YwvPkAuKi88u6QBS6Hm72OugoRhMz1W5TMCAaLI2I6S3vxJDwNjC+c+87tsJ9rtx80LSOAMJ046xJT5dMt\\\\\\\/VM2bwFsqViIETxQpPMXysOdX5\\\\\\\/6QVS17bjlCSMh8wTsy8REfAnVMmSxUWmAVYfVaI3CQWCXitheg\\\\\\\/gqrAZhMWqYdMevSM2NyXs2CLlVbcDUOIqhJY0OtveVE86opZPslLwPvqAas4oghA0BsokpIm7rZXuASG3BAoYZAJ2SEI6vtw4Y6UQd8hBGxLtJ840cXiNO8VJwCt4SnCyD\\\\\\\/EchtZQjJcAz9wUJZxV2ozgRZZxuRvxHiytXNvivPyqENT1GTI5LhCxBkDLInD+CuCPhabvIJaL5l\\\\\\\/\\\\\\\/NonsD3ZnFvU+PmQdFTbx4hhCMWPHy3OW186EdHdzBMcoWDVpiCSdrjFod5Tu1C7bpPsWCSCKzz7rthcpkyBZtuzIInEej4T9QGvUZfyvTYZU3PDorTU+GI+jsQfFo\\\\\\\/7NHvongS+aEy6knrquwlMuCGQ5KSNgNeVcjBuT5KdV\\\\\\\/m4yPc0HQ3K5WuKbGZdAb3Fc18ZdUah4Bt3hTaYPEQaZO3H9\\\\\\\/2tFC9HNjqnHehKQBd+F9C1fsYglQZcZIX8kY7XjmBAqKWGfMV\\\\\\\/O7tCglfbPVs52Pf9zbyFhuTUdtTW6oUl0PXR6YPcAWGhYe80cpQL9Y0Pv2NU0tXVrtP5ctTKIg2h4Jxrdlw7vC58zp0+k25S2alxxk2KdmGQv4AhRiLLjgRM8fVu3NAJzAam04SMNNjXbZxNLiQT8DfNkYLmFj50STVwVbUGKyCMQjfVrnr92mxby930hbtZXihNdvrWV5PhyQTJ\\\\\\\/RrrAkRgluHbLQx9DY2lriazfT\\\\\\\/VeNw+NkLUFshIJFhsI\\\\\\\/24knRsLqhsFogMOlo4ZTFJb\\\\\\\/0DlGajDQqFtA+ikWMe2v1bQhH7863ge7nRa7TrssqB+ye5wFt6GQiI\\\\\\\/OoJ97JHMYRiYbe3hK3yGmjDBieHx6F+By8C193J0OCi1DZhGcWz4IWK65eFJ6VGWMail0R\\\\\\\/vQOPPLr5Okx\\\\\\\/D5Wvsbe6+Jt0iOl\\\\\\\/+4H7DHYCrm5DnpM21X0HKXpSFGiEYzPSAsebpT+m7lK9rfjl+ejbsxC0VG7i5nw48uarGWsJUDRcMntkqCo\\\\\\\/Vrr04lrthMjP4MpZZtna+Op829CwfHhfsfrR7x2d0Rwnd8LbmILsKVqTLJNzOj1DW3sXsU7WLO+AFKDVI69s2jEsid3uzoc68KWA+etWdstaGwvHqtMdKIP7eI759AiF2jE7MNo6cmpvVMbIbBNwdz9h6Y\\\\\\\/OADS4voHtwnYXdV7xhbj+qmLtUe1VRyrsgnbVm8dOF4v\\\\\\\/0PEbqPWL3wdTc04NXP5SbiLmFaiFFmanNQBKe8fP5VpCMR9uZozLYUKZXTe23mFiA8R4AQlnxbQkFQ5wuM0cIYDWEGBPFOsqkfRJUqS0xA71IChS7sVzIkh1YjYmqvcWfGRfZpGft3zDJccHZtI52gJAM98qn3mMMPyRmZew+56\\\\\\\/stBGakwGvXPZpjmOf6r0G8OEZCPXAiexv\\\\\\\/P\\\\\\\/\\\\\\\/KoQ6his2PjDlDOdTqRSTDgr\\\\\\\/KCLopjQtfRvWZ2xnFL6hQvS5i9VgfdrnkvR5U3vuJ1cDifd4Qyc1F8bLXmtlZizcgvm8Vc1MrVN9bQyg\\\\\\\/5XHBderO1d5Yiaixwvn3TzGTgH4bMcAL0je8Caaao3+5jilHkFVESbCbLL4IXbbJqeNxBYyXr63wySAkHWpSF8y0LI6qEX8ZDraE754Gr0fdATRUvcCYfkNj2\\\\\\\/wpyoEs\\\\\\\/K3Yz6ifZVxongS\\\\\\\/yhMcQSTejVgeoWs2OXNbLVpC\\\\\\\/2FImq310+AoAo1tqhBjtFxG3dKJv18cLo+qEzXyylLfCWzLgtd2+XvbeHzb6ZSXFU8kkmXKohMP11PMdcPSOlrQ8KrATEY8xZ0fuOj\\\\\\\/sHtySvHL+wa3AjeUhFmHPHS+DBGTcery0SyqJjTolgGBiZbdHz7pgmOkJdcukuEMqafYJZWPMiIaEKyFuayfJ4ByP6bGmJaL9HLyxgdeu04wf4Yhe9ElvgNfzuFskjwSlkXqtPIcOqTl6Kr96oxqlgKtnX7ZXQJZy\\\\\\\/WOybAAEMc5nPrrVO2hgDZidu9b+xNvi0iddPBSXm136pBm4\\\\\\\/o\\\\\\\/DTjWNcXxQX4eXSfzuw6VeX8wrAqV1QPWI7EGL+93Kggycvtn\\\\\\\/IudHyjHzvPhZ0aNclrghFuWSov0p6mkOGTKRzeaWnQ9WfOOH3P5\\\\\\\/6Loely8e8mPYMc0z2VxhZnc9N35K5TK3hBETb2AJN4vPkUtC1yFIz2MQal6QeeXCZW53V1nFl+2g7p904Log21T9Acgom4SdCs+VZxasfeY9SUQ79Is3q7ipcIkISgpJ\\\\\\\/3T2EGKgEAOyP4UZo9oHB4aW8gGBz9p6H\\\\\\\/WVFrdovrgk4mJllrKULsljbu4T+a+hVrJ7SdEAHHndpjrJIAzUbV00LL9kUUb3EX9P2INtu9Sa46R00AAg4qoixt8axRbgFYgEBrX9d5G0xGTrHAJJ\\\\\\\/aRgbeYtqFdFaV8Jay\\\\\\\/D2Y9Z3581+xZOC2DFLPfVBDXRL5R6CxBhsVms6od\\\\\\\/+e\\\\\\\/oqRD4Hega3gQjvS6PuFT+H+cG15SbCKZxQjgiN6A88seCETfnLLLIQmXKievu0tQkMpnSC33r69w9oBKNe4HVUCVfHqhxiH8YhMfXws9vlSSpdV9L2LE+aKgug4xMBxIbtpOLy91kN3GTzchEngdTRpfxlsL+6lp9kaZ9QKNyOFbZjomQ\\\\\\\/z4lCJpu3G2xfoSupzxV1J1R\\\\\\\/arD5IOxvn0sf57MfKVZaqovd+fu0nPt61IHt3CjTtx2gXw911BpnTs1fSbHdmr15K4sP1PamKeyw1ypEB21QqtSxrevZo2Z8Teuy74J2pbxH4n2pnDaap5MLl9jUu53Hr+GG355HDxjpSZmESD3N+1AtPkxznzXbAE5cTJD0hr1w8L1du5TLp+WsUMlz33EJRYdeS78OQqgPbqgXirvqs8EJ\\\\\\\/MPbBnQPDVxAw52rSTGPAODCekb8RtWTNMeMnz\\\\\\\/06gxj6n6d2HnrltLuCi6Rf9JWe4SDTIVPDfx8YCXtE3LxBte\\\\\\\/+ec\\\\\\\/tU63tCCplr0Q5vCvnpSTNGHgMGBhZU9hF7WSZXrDZSey99heU4ZFME\\\\\\\/izOKsb9bCbA8FRP8x9omc4IDJN8uNQXUqSMS3poEafCa5hiVYa+RZCfwN\\\\\\\/OWjgEJ5Gtg7vl9HkrZugupQz7710szV4P6sn9g1DSaToaTG0BGWPaOIhXloPUkojB6W+Lv4FgfIfLQ8VC6oEqO+kEpFr5zWd12oIE67ry3QS2lH8ZXIJxYYp5zq+B4G6qg10N7UP9TTyB9HTdOo9UxWT+8Rziq0N1I8TxbH+SgQAebT12u5GSVZUiDuR\\\\\\\/cXsc2hsIsbggTQp\\\\\\\/LqBEUnzg4+M13dJpzskuAvucrKp1fUaphsUfwkuVj6BxNPH\\\\\\\/n6To\\\\\\\/VqdqcmhzZg0BQtKo85p3dOdsBo2TYQ0Nv6+pDTvagCD43evKSYJpL8x6UBl0fj3lltjDNjgMu1y5n6mmHoT1nLgu6gFLbnPBlwHke1L806FM\\\\\\\/4UgZyw3+RYKqSYOXy2bAK0tkBYdL\\\\\\\/H4BEo4UyhiqRe5UqoooqCpDs16sOF2ddW5PWTOhis\\\\\\\/p4J46n6Z3t8Y1uCBsfFQ7I8H0QWKPBx1CF8fYgmSIHbX\\\\\\\/0i4kDf2zjz2dTl69BA833OU0o0PW8OUfh5ZzaMAAkj8uvI4YEV7tLcvuWmx5NYZtSv3c\\\\\\\/MwXnFaYzEbQbdXfSiZHlTCxrojIn8UUKKvtwu4yoerBDLc+eU\\\\\\\/9IVQ0HqZuMofV38papGFF0AdU\\\\\\\/EthFBHO8EVqvQ1LjgbWXRbHcsaRNnErG1WOArV61XfwIvBpqb2T9EEfz3NmlmAejm\\\\\\\/jy69S0CzLVes17G4Ss2i4BEI2adrvpM9IlIZle\\\\\\\/s7302uV+7UPyu6zWEWjNlnClphxqAvMnmdXAS+VM6cx7o+4kourAghJHH4hp0Ccvd\\\\\\\/VY4v1lZde9YkFx9pIdluKk8dTu8pNUS5tMdqJboA+\\\\\\\/btlDAAkBu4qa3EX26zJZwF6p5RV6vKqQ47f5YalnwHex8KRv0DA4r9r+3O0Czv3kNLHsZs6DQFujDMRNwWIVK+6me4HJN0mI+U9v2ZCWNAhJZ0Q1yHy0s38zE0pBhKnMW4xuNU1ZvxGZprrQjdEnPrTdapvc8CJ72kkDL17IfHpeR77MZu7uhqNihCmJY70JKBb\\\\\\\/pbGz61Z\\\\\\\/aQ2Y+gNVJuwXCiU3DKHkS3cGEF0mCYjTzCd7uUpLnUfspGfacRWFeZfdCX+qIB4s3cyBE6z9UiiCzbfo+IQRTKUo2LDbp++Vs5Za5ffY2kiazCiswwV\\\\\\\/\\\\\\\/ydcB23cGqlrfG2rIc7YyaL7eWUi4zE8No0oe3PsYJaaWfcdnRTRG0fDGrGK38frzh2Vy8HQ3\\\\\\\/ViPY+d1k2NAkrUKuKual\\\\\\\/dTY0Z3piyYGOORAfqtDOebvX3SWh4nw3g6VCMZTY+ULNMSb2xXC4ysvUPjv5wmPLN\\\\\\\/uHeOMDbX3ViOLBKETRQaEjusY4NQZP2QHPy1efN4aiuX7XQeOkl2BV9qQ9Rn456mmnO7wsb8JpnM5Bgfc9b7sKXQBiLyKCOZ3JkWXna24xJ9gVYmlgRqAD8+eDj\\\\\\\/Lmbo4aEcSRW1bZXrSkJsqkHNHGAXNgl0OMwKiTFCfbnK81LkC30ZORX70Ksia6RWXA4HxfXIKDzuoK+TL6kHgvJiRPQqH\\\\\\\/mRt87hVj0catLdPeOL3LC12DATh9ENUu40D07EBTKLW8DRKXEw7\\\\\\\/hRuzPBlewgZ5q2Y2x30Iabazu2VFzCU9YOYUZZvf1a4NFt8mtoY0NzkCHgdIE0LfZVw9wYeWL+DW5HGyDLUqNCMaAfNMeoURyMDoZO0HHOewrNlxwH\\\\\\\/0ynjxLpH3xuoym2heoh7n6w+v+C08yGu1jbGseIQ+nLwWWJ11fyKZyX5+N\\\\\\\/l0QHPIuSt\\\\\\\/LEMnIrxPXZLkOWiU1TI+0fRrei7cy3pmHWLCBe+SzZs8j3slJhmWUzxb2QD3dPH+h6k8dA\\\\\\\/idkej5+U0R2fAwPrmme+oyCMkM1G0VMGA7Mf18wz0hcjnMwGY+jgXV5u1OdL9uvn4rhfk8svR4Di1XV1HwycAViOKs+aei+duTQMrsJiQ295sHLNuoIr\\\\\\\/W1o5k9Ki4yErhF0Tt3B+1VDntgeNXeR29PbfXShoJKS9wjGu76S0iDWFxxZYSsU4NzcjYhI4YgFz4BeNhWJ8Dk9o+2owgvmE7alHek6V+g97QvHVdbV1nOUtZ\\\\\\\/Uxnu\\\\\\\/N72t+xOohVZuGIIqTbPsBB5vnHoD4+BVeYbtTwnEUT8BLbagVC2wDUrnD8GCAEoyETxh7G7GsR4oGCaIVHWN5xSWCYXkI4y2P4ThGxKfunfmlVKT3eMS8Ye2JL8x9iNhDC1E82NGY+egRR5q2j1PI0\\\\\\\/k8g\\\\\\\/kbz2yYjAwacZX0JWmznGObBgLI0V26Ug+I4EnZGX5e352cPIpgufrpdTWwYBhwInbMCFgCDcQ2FNbaNUikyJatkWwZ4sTT9BF7wQZJc\\\\\\\/dCVXe60P2LIeslxjoQolXkH0SpRcVrIlnyDrCPxJZKb+odducx9QePU6YVlaMLKLJopNR7rsxrSw6tetqbSGSxPFeXldfAieGS6M372e9gB0pfqBzfl4Zjpx8Dd2fYPFssUwM0AYema58dPktGrDKiTjTRBJ9ey\\\\\\\/jCVo0N+S6Yc5UqNENfKcxbWnsw91s46vZFUIJEAUozc9eJLFDLLMUkny+ySb8UOQm9ehSA8utsqPyKPQ2R4hCKYcwqBb996IIy8G+\\\\\\\/JWLpsFMOV5403lvTnuYkzC+xmCBXMc4nh4kSK79t1LVa7ioNTmUXqpOupdIijIG\\\\\\\/fq04QfrLQ9\\\\\\\/xygOm5CmbxdWQASS62RRafh\\\\\\\/1aYl+1LuUZntd\\\\\\\/y8Cs7JUpovAZflvKdchOUhbwTMAEtS\\\\\\\/9M+X1yYl\\\\\\\/ftGnMJaKHzzXV4aAl88XRiCceae\\\\\\\/MyiC6xdERghsfx7dBQ80hIJ4Oim\\\\\\\/dzs+q0xT2v8L25Bqrn+ZUuuHuOBLh8bVRuhtRT9DOIKxji3OqVU6vM+sGHEp0hkONCCM+UwS+IGRe7ACmujwoAu\\\\\\\/pBCTrHwLRs4IVkWep+0+u7cjLUAaijsCMGTIEw81iDYvtLrw9+NSv3Fd8Xhe7mXegUz1NWZA5JhBjOhUx5uZQTs3w38nrVIQaXnY3i+KuddRO0ypjQUZ6NGfhkY4KkUWVJYMkewFcts9k75987bID\\\\\\\/jiZCSa1fnacJOiYf0JFQQ+gMx7yKl\\\\\\\/98e143t8mChYeUAXOnw7mGL9T\\\\\\\/zS7gcbIwFosVivwWP7HqRIlDNTph1lCDJy5dIe+OSxYcL9P1GHkS1vuEAdYVhTCMg\\\\\\\/AoIORRjSxRms5K4Y9DhwQT6MUd\\\\\\\/9fxGAQ0ISY7+rEodIiD8JCjThvWCQntHjBSdLMQMCxSiyeJw7OSPnMYW+CrN\\\\\\\/di\\\\\\\/\\\\\\\/xYkk4Z+E05sD4MYTPAJUSOHvwcl8BOWZjlRAgwvkW1f8\\\\\\\/JxOvspLtwPy4Ljnp5JOs8lImVww+D3PPW1bJNZHMRSEANnQF3k5auhl4KMxXWDEDhPjzMJiHn6Pw3sJRc8WsidcCmtEdFXqE3E6PUxqT4BJRKbULjfhiGgmTzFAvl8uIswKHpwpYw3gcox7nzi2AwsR5jhMOvvTwThfxNSO6dh4PiD5lRytr9kF9lV58pZIr1HBgXIQ1DUVqVLTfptECyJ1BuRpJTJ4Sswx++xqR7aM89tRoudWNdIOkyPaYNc1gNPdIX1tSE0BPbHf52joMNr4BMfBrQRC1dwEmXE9ArPGDonRmGdO3J2hEIXS6F4WjcAAeiPfD68A4fqIk+M6nZnkj\\\\\\\/pBnjg1bLEOSzZgc3w0bS5LwZviUtkGprBdH3E4wp4ZWRtP57hQulcjRusDuV6JZXQYa0U3xB+Sjm4eiY9x81euzbWXwXqzmgDqSIC0c3OsFzNac+hiDHKnwQyAUDAbml6dgEQKV+6ETUCUooM3e9TIIGhjkmo4BnhDGWNtIjytsPdYLS1IyKrtg\\\\\\\/0kMppxNVsNVY96b++Y1VXK0XUDPd\\\\\\\/VuFTEMo8Z71P7ivg6sGQ+OdHZKEZXJ\\\\\\\/a5oaItj\\\\\\\/9hDaXxKqK+HmBkUhBYYJHasWrigrbHj3raAnvkGewhi0aLuLR\\\\\\\/JqDGZjQkJmStqYti3\\\\\\\/\\\\\\\/fFWM5le1mmpRp9O7NqZ9s7foJaNQumyOqv6dZzP1GhfRcM8B6Nzba50XiUxhBzl6luHv\\\\\\\/8X0jWpHdTYjvT17SRy0wa63thw2kR0dfCgH6+dXC\\\\\\\/FWBr6IYjkmNkVHVWGCaRSc0DhgAJaO4q8VvdPx3N4KROFUA4a61fs5EjBMvkFfY\\\\\\\/ycoUooBmEgUo4gqvt3hvTQGOHBTJAXVpzFSTaYWmfv+2oMYa2f0wWRjksuCPNal8hRiLm1VxZk6RErvLVADsyYlP+FgKgexkM30NpNXF7w8uZowQygbaP+96T6Yu1V5cr5HnuMts2WJVycnRRA2LS\\\\\\\/XC\\\\\\\/Lvz+2mFmnC8ceosAgC4IEq2rBCL9kppEkwctFxoHH0StHixV3X0q4VyJTKpTHkyFz0\\\\\\\/+r65e7ALAI+I0slKQjbbkNyPW6nI0c+mNLJJZhk1NkqyWtLA1d1sWYyq7eHGaoWue6\\\\\\\/eQ4wpdlWMzR42VBdnBIpuUu\\\\\\\/ybdxuEcWavPWCkiYLxRZHO0\\\\\\\/uWDBtWpZhVDIMJNZIgNkNnNSXkSBlnB9uOKJ3DLfDujVbZXDKQlcbCkqOYGUGCNjARcEf\\\\\\\/1TgtU1ulbl8CAjg7srC1DGsAgPHGx8vB55XfBsmnD2FRXloik5PrehULDhus1SL39jM4yGZYK\\\\\\\/w\\\\\\\/NtdCcHHOC1+xCfsIZGXCN0FCtHwbAssamxlS3WgSvir\\\\\\\/+3ax533nAogDuF+WU3\\\\\\\/EHiaY02KQbp3hrgoYzal6n23q1xzqwjL6aTQeeb6dQv54WpLHZ02TDUYtTvDtsJ8j5RFllqrz7OWkcezbpoF\\\\\\\/toRuopZLf\\\\\\\/SFjuffW7Ls5M53BHQ0cJ0yV5WBXWZhzp40YepGxTnSmnuMLO03To+Q+Yhlg6OtQNJbuQI8YNgodBco1PROHzmqXbnKzUAfQsINRHHa2wC12HH9JPhq1EbResPQBEVg8LlPB1lxToC\\\\\\\/2enRiXBM4bUV8hPV3vebAc73IYYpWGlifh1r7Q5s\\\\\\\/6mH1E9aS7NkSGkRatR+cGljaAXtMjyEuBXCa4uyvvVUT1TjjM41lWn2iZ1KCnCy4FD7Lb0jnDrX72vQP+UxrQ2Agi1KXX0tTLT2YT6fL7mbAfvmKHTklEytM1lr7QFhVUspNnHrDDvK8XDcx3WFX2JiYw1YXrqbkNGzdzruMAdRrsZ0Vb7jbYqPk0M9Um8hxhKS9K6iubzWTusJiC2Bt5m41LQTBVZbowqay2sEQmRp\\\\\\\/WO5Nyg6VbOX8xFPlK2JMNaM437zpoemXn7uM3hBAg9TKtl+9Iq6TKyzUI7+H3XBhpcwhaLEnIyJcYAZ8lWIwB7ySbSXEDBVqbfryd7Xph1enKyNNuKdAGUF50jG6yYEN3dTc3GV1B+vra6kpu4CvlqdFDbVsV56Rp\\\\\\\/ajquXxEFkPf6kT2k\\\\\\\/rb14m6buy5\\\\\\\/BchixDBnGrSE\\\\\\\/fKCQF5N1YlFYBXIaGFDwt61ybvoa3b42IVWPidau9TQHOldviLQMLkoUBGHNuUnVBAnYvz5yldTZuH1RFXPmxVfrUrkp3TtwBHscMJ382iDINvm7aYEoTyqwKFHlWK98gfbsdnNfEzkUOiEcg1aF3kDadhiBWL1tCUbKYq65vHi57J8yz2sha9HWvWcF0jeVcUw2IrDgY1bEGqopKMTeVwduQVwddIE1OBDh2o15o2akT3zdYNgFarn9KJSrtiMv7vPNTnT3mqE9bcPVzsNrD3yrvn8MTqRRqsS3nRGI0htnd+O9jIuWNNOZxHtFyg+irSDzc\\\\\\\/7\\\\\\\/fKopMdki3ee\\\\\\\/wsn\\\\\\\/Tm7bgYGdjpxxLJsps3cGbUVG6gCTva7uUbrfusx9ksRW1c+mACVGwjT9ocgl0Nl6u\\\\\\\/54lHZU5wDugC9PkUBENYuSa1HHPb5387i8GJ3c5YaQYu\\\\\\\/hjALqVUu+NetpyLQbDhi1sToyw5\\\\\\\/ORY2vI50KfIXGRgKp\\\\\\\/xEYOE3F0eXU+vgGzH8tm6YsYETK6uQxiJNN+YtMvmqNSw1nFOPkOQNfJR4BeZyOgg7Mvp5j1uPMKqUmUgR97LoC79kbyk9CF5Sn1cKYqGfkBJibVxlRMUA6ouwdr29Ik7jqf6\\\\\\\/JR7jerN\\\\\\\/xVTCuYxsZsdmIEPPb17k6kwA6TZEW7r8IfnuYyPcmCppJ3mPhd4GL+bHfIMBg1S1FAPUliM2bfVhMjYIoLs3vRlc8N1c48Iz7GaT21PFvbdjFj9r+DwOB1f+5vFnsgcWXJWxp4WC27Crtocz5Fkw1ZsV12gLe\\\\\\\/8yCBOHAbZIybvZ50e0sg2yt+g5baNxm7bGMFPX0Iw+nPopyCd5krHzYu4nI44x0GuIAvow1dKJEZ+Qtlvx2OyOY61gsHKsZq87fon6Q7YN786TWqX\\\\\\\/LoMLtL9JkcVYNhgUM0x0UeHsoRW9zOv4qkX8JxggQk0DmybG+lz9xFS5XWd4KGhUPlPJ6QZg5Oye6pK+qo4e1f+1rNeu0k2v0a+XIj4u4Q5pivLp99Q5idVk45zH5yEYUsDpQjl84YiYmlYg+t+iiYLf4q64CB19JWd8lqohmEI2ETo0lc60cVK8P3ULtR94AL8Cjlw4MNzMsWE8SENkqwPCx4gyvsdtAlqKmbND19cUv2B0aJVSggEErO2Jh3F7G2JOvsNPS6CX8HbbBRMAkHIAyBh146bGJVaZkYXXVgQ7JYKef9KTLzPRTY0QmShz7tZg+3vLCrskjjceeyso5OVbXdcZjOuDyzHukJCjyiskEhditWuWR1s2c2Lcorfd68N1PMX\\\\\\\/nmULYITaANn8gfV9ZmjoLQWaAlg5GVkcmp9kppzshyFvZxX+yZICBtfgOSQdmCNqeyZjZW3Q7lA4dKxTRW76m+PMItds5vhdvJHaWmGdUvMz\\\\\\\/17EKzNzLLwtW5\\\\\\\/MQ5tm62h2kEEAmdNAZJihfByUt045Hi2UDd8eYC0RYgnWx0MM9N8bKulIP1uy6YOFroOiZfnFfcuEttM9ystRAGRnIXj+79ey\\\\\\\/H9Q6d466EBG7\\\\\\\/NP7aLLkpkPGJCyXsbKyN08sbeD6LysKmB7hP2mL0O0AkCj1Pf1v2M92qmmJU9Tz90+ummRdqripuFK0sTDbwLkX+9KtgpUyYUytL6RsZ10+ZPPzGluN60opHTUx4KjfRNUB6v2L2Vgg4SzHKCbm1q2huffxj19ZArxF1TY1nL2q5HnJ0JanHnmT5QM5MHYk9KVWr+cpjs5XUfLTVXkdMCRWTmXwm3A98GgnBpnEqEjicye8LRb6QgJs629g\\\\\\\/wTFM10Julxs8TUvVOoIXLI0x3w5rUFZUK5l3zYLPvg7suw+uqsMt1UQ1UjhkiHBDuWYktujBMDrHLLFidwOhfhyIxfohyxhCHwOs76nLsbS2iSehic2DENr+HuvduoFpQ6xZlviwhHB00GfeeRKDVoz5DYX+H29+PIPaObhg\\\\\\\/b9SqkBAUqitlG0ZOScdFnqmhMfOv\\\\\\\/0w+FMCkQq68y1UboQ7iSaoUET8rzEuqTdUU55iI4gWVKa8I8+gqsHVMNKiideY7LhfzLDBCriYq\\\\\\\/PFyQhOfac+O3WUZhMlPf\\\\\\\/A0JAfwTc+rNApmT2P6k3+Af3yDrLpjHUbitXLOZ6et4KNleOXw5mF3vsfk4JKabEfJQq7f+AihxhJ1znO5Oj5n48F6JGUNbQb8U+NDe2lkFDldj9h24NWY6xjCNXVhQD1gGgLxCovZ0Yd2ZwgBWAiq77gEfJ5fYNh6DPwV4zjwwDhCQzEv+I0njaO31Iy6WrxHdcAyxoJbdn13fEOkEhnerL7q2pku63BhfQ2mZEQJFYQn7l3n8gOUB0Oi7uqPDdx5yp1ONQgQadXFbRXU4xZTGCGrrcDT7csRX1mR\\\\\\\/Xy44FR2gSWWv9VcuL8wUCY6KZ0MC1CApVxEK\\\\\\\/OoBx9D9vylknNoc5a7MjE\\\\\\\/evAALekNFIGML8zzEwwgOCW3XVMnxgvoqb5\\\\\\\/01fRVkEnhwC9tBR+FlSuT1yBzHAmLNeoMaD94eP+Bq7nqsAc5KiH0FBzcvb7D\\\\\\\/RKI47eHQKcMJdpnSbWbphhyLV7aJPP+jcQ2ZjW8O7ybAGT5SNGZlsFavrsxMUqTEBS28RsJh1C+dTSgk5FPZ\\\\\\\/ZZm+McNxUS3Po1M9le+rGTQAiru4tzqovPWqp+llgC7wCFNhdSCNTYUEGznaKWyNM7V7yGWrK8435NCFWYJFK7YcRnwM6kAowNtbuqJyFP+aQi0CVpdEm3hD1u8zBEjGKftW5tAt7G6G6TJaHqaKhh8jjfMb1wvJLs3Ap\\\\\\\/6MPWSkWMXxrpFyWbhnc5ckNGnhvJMlLJRQOAMUxUotxtoAfG5Ojk6Oi4CK4NqfALnCO+iKDdkoM1unBNMKbirEsJbesRbavUC\\\\\\\/Fhz6HFI2tUpQJ+QkpbnFWksIYfWLGkjs6xJGdFbUY5WghdEYksb5lMYx+GG3MNyd5y0aYaVpsf8OHNFcFE1N0IifRvuPUywEziwgT7d1icmW4DUAuJHcZMuGAbx9TxfsMfQRumXo39bke7jC3YbqoqSb6w+hxRQk4BIE+YKjoufxDrkrlcqsSBFRHo13OkMXEC83GSSkWrhRJOo07Jm6jiGFsq7u2tfUfXIQtXmpwYLu9rt8RXZsDgZkAlisPP4jsVaeSdr5FQn+HznYgmrTIgoG3FXHeSjE6fRaIN6xwUlFsh9PgOIOL9uJjO1Xxe4Ug\\\\\\\/hx+uSIJTRu4nM3NTiDhSDFykTrm26ujRmVJRFvEpBsc\\\\\\\/pOQpKmnpWA718BSIYi3V4h\\\\\\\/fBc6i3fNjFONGWaCcqqsvnD5TXG6S7EDRN8LEUoj63JGbAXEMCiy3ZB4+4MxomDFdRnAX1426dtb\\\\\\\/2vBG5\\\\\\\/f8n775z0KUrv+tIbdvSO0aZFT+wrTtUZ0si3BpyKUX8nDteTd3AD003Y86ktssbozb7ITPkAjgu5zsygeR+StDUPd2mBwpZ\\\\\\\/JRc+PWKeLXJK5HYyzKRuI1MiztYgEm8VCWHOJ7yZoRfP4xjd1xOEzi5C2NIX9GW2PviZ09UBuM5oYzRyolu\\\\\\\/oiww7xnwbBNFDc8B5Qfy6myO4JXWBmQg3UXcLNHHpq1xQhr\\\\\\\/lD2yyKYZ9OoBp1nuK3Mxs379NZBPa7hLtAampvWNnBHXuxnnEEBTLlO2qcglvsdMmiL9o3VvJuI5nYilsRyqFyvnubKgRLrjsqyn2ntUAwJ64EnYkVPn5DO1C4MTn91Da2JoQ9hxZXrQjroqNBGE0UNu5u1a06lOAxioOad7G6qr6KBWtcMWpldTHb1yG7EdjeT3jBO7xP9RKP\\\\\\\/eL1gqZlAf3qr7pQNKlobg7aR4wSVxlg6bDwRSXkgm8tZ0BwCwRhbXPvaa+BPx53WjZXgqyLJCejkoFI4rTYyJFnQZ\\\\\\\/IHcTazYy6oSxJQdZIOdJZ4uVOvbkvLhngVXBJuJylDMx6jkn0EKhTmRNnYXMJsy9dRK7vDkTDUOdIsPauFNsXP4OTVsjNXOqIjcT+OJDkxwml5EOT8SCiqXBLTk61qrxNQvG\\\\\\\/CIB2RvHk\\\\\\\/fUjkoF4dmnjAcOegMoOVtArQYZYc6uFOegcNsR8ee6Cqz1cLu5zOz4XiQI1ShYA8hZg+9hMkaOEHiRU3LLEdih4yKTZHqtVjM4lpIgLed0db\\\\\\\/PqpytSszKGQPPFKNryGenAztnjrPB9MVRM6ELTjmZ8y2hA\\\\\\\/18Nuh3tHL9gBNPGE6FawPusnrj14q1sqhawC6k3CINirHQiPzdPIyUQxVhUI0WBlv5oqYT3cuHCMsbQaDEnjFwmG6qoZMPvCya5cYXq8HossqFm2RfTV4z79Kl4Paceo7rU7L8MeoH+F2ntlK2dr52Xam74f6P5pdSOiM9sghttepGr1tV7Kuq6gvC1lmcuZ8jHNvL8zWiO4nnmCSWT8n426Wkma3L+tDEntPkVeSHiiO0Lnl\\\\\\\/8LE6bXRIucGN7uvBvaCKyO1YRPIdNDPd0k47KdOBhRoEB\\\\\\\/8ZK7nT2dLPf5eZsixgW0VXXLUHKD6bkmZxkRico60Nr+8L76lhN9eQczcUVz6Hx19d62MFpjCCgnmzEIPUL7osBmJWPnNzreZSFDCeci+tcYC2M99nglJI7OQqK8PVfGN+zRPHPeWwksa0IQNujnCvhDm+dcSa6gmtNwSwpMZKE15fNj9fCHjZS9ZKeYqYeJgiLd\\\\\\\/OuWAxF2wKnTXfdM68GiHUBr0Dx+jHpMqXhgSNM5EABJjfCbQ6lsdtoJ1LXSToN6NLwjAq914Znc6XDSrRHbSJ7HMZFcNdVyOe2rhpJ5xKABdS0g4Cr8eQ1fgMzdLf10LFh+EYwAz6nJnVm5uu+ondCv9pFpPrfIp\\\\\\\/7Ol0RYsXicKXM8Ry8HblugaOA02BLC8gKOySGNEcJVNrenK2PbMgphbqKy74GpWUysEA6gGVoZkFRXmmWgKJWBHZTo64n0Brp6m6V8Oz\\\\\\\/hDZo44Nc+d9AQJoezavUodLNenNN27YYKmb4AtbV0fBz4pxR3yp10UkwlKW0fR\\\\\\\/ntuT1FM+mCbQ5iy3LymGgE9+c+KaDYa5A1dAP6+fy437UdBgkBzjjFZs9tIOFY+iAenra0aAItCaigP7Tqm9KnL\\\\\\\/1XufTA3vIwamh0Tt4IiO4TcYNyUM3lrZtm\\\\\\\/tbUw4oGnT9yVNunFYWa6\\\\\\\/R1UYcB7OC\\\\\\\/nP9TilEiiy1wayNnVm1iEnt0CHXF7+b8Yd8jVxnFX4ZbRDrUdobIUnq9B15+UQDYfs0lti\\\\\\\/UqNQ8PoqVqLc9AiJ8HGCq9qEZQ0Ee6qYU4sevlnEebhjuhv1QIolLo+xXL9CZj7idiLsbo+ejKYVMDoC\\\\\\\/CSYdpuzJwMsSIhFjLnIRtlVWMgkjWlalxVhybkK3tFzrf9BIMKu5L7zLQQZgDplAU6GMnd3NtzOd4+OWehp+tvgnZqXOqXtsp2IDVXuwOX6B+hR0EZgQ8nJVbb\\\\\\\/BPNDPdoPDbNnqy5F9JCVMf6lyGk1VCkMMR2eP\\\\\\\/MBt4tIrBHFSzPBGdVIPmsX3NxpTQr2pA4wxiE808lneu6idf7w\\\\\\\/CZnjeMqYe\\\\\\\/OSA9M5aDWAOcZMCXIduVa+HUXl5crATDViR0hW2\\\\\\\/wos0YJx8sYxTR\\\\\\\/lhm\\\\\\\/K\\\\\\\/OtG0u+0TEYcq0nKGzmA+slRQjikyKptSxGasRiPw+IMQvLsdyGDSBylhPt9MZIU9Au86rWcehYMUZMfHestNc3rja++aYZvzPTVEoqsdZQnbKKZxWKQHbeS0NhiYp6rimI6LPbBhlxFwZwwQgTivGyHdIPESWXPmt4IigDM7+p3bdq6N5OXK82RTkbh5smPRD1tkltsqJKnXjWOr5ayPzJYzEdnd2XW8jCOeQIcZqX1gcXclgwqyzMQUOSItAJFmDOsg4ZqM61OGCt10r2hs6FnzIWlHwu9MY6E5n\\\\\\\/nsvwmn4EgMdM+kQY5aFe\\\\\\\/NMqbD0TMp4Ee\\\\\\\/pue9M9uQWgAT\\\\\\\/aesgb1pQYBMWB\\\\\\\/GtFNxE2Gh9HTJsuV32QGrskkW5v12p49VlJpZkuP2Aq6ypV8ccW4mP1zfpOpBZS1OMFRBblv3jet7ZVJWQdd\\\\\\\/7nCwV9BNo8\\\\\\\/r2JFai\\\\\\\/dwJ9ohKgpwGhYNWC3aaqbCkm7mLktUA9lpn5fclAlZX4TWzDLaqPDsVGq7VvusP\\\\\\\/5yFRFPf9fIv94M9dNgApfRr37qdWXTh7NcvgkrHq5omWhsnSPaLnpJfd9FZtDhxMj7IEuCngEC\\\\\\\/ZxxFhKCpCxsEeFSfAHsUAgPRt+wFTgdiZGfGG1H+onMLiyLcy3pbY3WoMhxWryJ0Ef6pZWtL1PNm+6LQQaa\\\\\\\/qoSnjlaqHq6NtneHYuPfJJq2b6ttM2VjJ1iYeKGCPEQwGNqthKhvcslwcpbgp\\\\\\\/DrZh+0Ol0PeE+ZGIkuLl8MXhLPuzKrRPGf+\\\\\\\/ZuOGPT29\\\\\\\/YxRgeto\\\\\\\/EgwrKW+6RIOy9BB4std9nvO3H0ep1\\\\\\\/R7gjbOgZIOaL+YtgKY6PE8x\\\\\\\/VlEYhjMj65TItDzh640ZiFOpccjXOD0mBEhUWz+ce00oVwFH0vMMY8ID2S6Ztinv6PZvXmng8mJAMcos4SzlxKNMgzAwYGdXSLiioGR+Vg130mIqn9cCsNr31yMVlDErsAqIyyXOf8DbgQaJ276Wv8xTUqz25ORn1gk\\\\\\\/\\\\\\\/MXPqmhavFPzSrCPYtxmdGO92yAfAJPuIK5CwFTpsZaf4DzDrKc+r69+cKGzGZykJvut+kdcf2oVL4rYx+VyuAwoV1GNps\\\\\\\/oVUQm+F5ylqZZ2fo4QuIh9RH63nXq3kzE3b4k2OybyO7yrVU6HzwbhURiO\\\\\\\/5ogLNJj+5W4XOw\\\\\\\/vi6LDltJLnpCjL\\\\\\\/dInfGy2BCTw163DlSFx2npHeudeLbH5LZDG2VgyJnjfr7LGuO0YV88jLeW105vb1DArV4a+xczU9czHPvR5205HL7RHGLgtV8qh3xKCSYL8MwS3JnFCUzakEC6t4p1AKS5Nr7u\\\\\\\/ckMjmwDVF0h4te36IKoRIMeFf+wlGZLzUtOETF\\\\\\\/HBk8rphCUsvM+aXOEs7OSx9Aka\\\\\\\/GOCp4NhIAQdCOSSAIJTtjLkWB9J4i005EtN+\\\\\\\/Qfsf\\\\\\\/YLjEjERvPjDvqEiOJYXm+6vlkM\\\\\\\/k83jq0INUV7psGybE8OPjGyKbS66WN4Q7x36z7qALFEqAgvXds4BB4WWHSGlIRaNfT5K7Z5zz0aCkIbeLB\\\\\\\/pl3pRj7agDAzFvDNlJH6tICtvPYLVPEZ4B4gF4UkCp\\\\\\\/qbiTr6lSCV8bGapzh2wooEHYfaITM3q4DlrPqMaKgQFlo28Df1tLSP45Li1JcgwOIpnwl\\\\\\\/GkOWYhyvZzGIKTqB5obJOF2iV6UM\\\\\\\/5qzKDYiK0ASFmxZQi2bwtTdCjMu34dDEfsokmNk\\\\\\\/YdtxlAol3\\\\\\\/ZyvSGlo15jFED1VKpDbPuiSEvVh2AGR4zmd73ZGUEgl5yuOKjLuGRtA+n90PAU0SKfNuBQFFIGLpRaB0zPFDtUIHdDUutn7uF2E5RNZ4x8vc+iSa9EtX7tcqqifEtZnd4VnAfOQKQF9M8rRsYkHAoYvIJMc5H\\\\\\\/bFkPo\\\\\\\/jfya\\\\\\\/Dgjkjy8HEwH1o4OMtXZd2XgzBJv6VGm5DeA+FyOYhV7+3IfSiOzCGc20JM28LmNkVnTf7rnLutVX920SH9cV0APsP07UWxKTDLllKflSoZVtlpT6c0LDXxSQo4YH8GjbcT+xShA2b0cgRAzW\\\\\\\/wl892H1O\\\\\\\/wj2RQShcOeLon8kKrKzqPg09fR3PfFvtmTuQAistY6t20tTGhnXTuQLFJOzkLnIR+jGC0BaNDjl31mhRbzWBh1rJr5Axn6s0K1N50gbF4rlaj+hhO+wE4Goeqn6BThpwut7NxcHVSN+K8aP3k8m+Y4I95dIntD3cPzvImVrBsdagIpT6rwtyqi3RF6pGnAAt2KG\\\\\\\/2tHAw9QdU5JuaZzmL6Ke5jeX3YiWVs7aIlUJcFY2KmRH5O2WVUqY3J2\\\\\\\/w7yj+4ofJMwLvhr0WcFLnai3QCVU99oY2sY8NhtiaEyI0Qz9r6hgBwLGidDsITWI4CdDdDBpfrdntBH4ZCEsNvQXrgMrBPA9pX+mjWM4XnEQIOBuqJVfcsLCouIRuEtOnFibaVESSFpMS82LPNPfFlVEbYrZKnp\\\\\\\/03ik4mXMActdPtaO4ggl8oKalZh6Zu9wYCKrZOq5Ob\\\\\\\/00F3DVwxn4OyPCO1ZySn0n+Vpt9ruoo1l6tLiV+MMGkRn\\\\\\\/Ab9pEOqZwkJEZqF0T13rpinpNMDUeMa\\\\\\\/Rrs\\\\\\\/yVx75SVDf6Lj5ld6Vs41ZTwFbqRR6Dh6PNLE5aJ2PYMc8fBQEFYxFTAAp0pGC9Xu8pxdHBuMbnFPkNKZ06buViZKj03isstlbVrpbKZyfQC5aoQ\\\\\\\/48Hcwvr6QOs\\\\\\\/IbqDsM5Vx+1j86SVrJteTszJrnu9qXhM8E9Rc67e9bO\\\\\\\/VV7ffTY\\\\\\\/SSbngDU9QbG0HfmtXiZFXeca5Pwrzfo9v7j74415rWXZx8NW9otacawYEYXmGDDP8\\\\\\\/RxIgJLtriS\\\\\\\/w0ePTx1EBwg93EkvMfLsUbyKUggxSAn688XDUGrDIMiNF485yNcMQWRZqDqwaJF0wvruyN9neUBUNA3Ly0Z+3naGJnyNCgdYddhx0CNSQr94fZT2hKE8XO1y0Tlti5K1t0d0cC7L+JMcCoCaDLBm85fds\\\\\\\/451Eo\\\\\\\/0lvgO4z9lxxJykZGAHZZCMEmSAZp4+tunKvNZemzf1KJdwutUWIZGyMap3Bh0ay5kqwz5bA0qzM+EO6ANFLKXRBYEOJJ\\\\\\\/zUoQIXS0lO42+uLYOq+Go\\\\\\\/fnc3D7uaVwkFxyYs7\\\\\\\/aKsl9ifArkGrBxgl+8CBywj1plRun+vFq2m4fo6ysz1EhA4r+GY\\\\\\\/\\\\\\\/TfrAnQ+cM\\\\\\\/9nVplOy4RoRt3yy466EA+eIwd7FaLzrleMiTa266UHUnZt4VX8NoFjoDKvygYn4EIY8WgAv+Ky2hZlZ5\\\\\\\/cn\\\\\\\/E4dBLnXaDH6Igl\\\\\\\/n9JoGZS1tPrUiam6qHpd+xZuQcqMNZL8oRz2kTYukCOl\\\\\\\/loFanmPuXq6++MM6F\\\\\\\/U98qn8lpX4QF9tU++lvh5BHXyde4tysFCnrPrV0osCyV45oHGB4PVNV8dF8AvjBrJtsH+b0lLUnam\\\\\\\/+L5eGpIQYtUefELfIznA9ftKh1DZDSX1Z3nmJreMOzPsq\\\\\\\/VHAZ5YUJySZPTLGKwCCkut3vZnAYX47pZkhWrIxyb5KkKcpeDuvbct5KmMI1i70bhK2gSXvpI4HRuMDxW37WU9Czb7K8m4\\\\\\\/w+REx4YBapsDE6d1HoxU270kyu16wEyPXZxtUaYOEZuCAIit3bE02SoqSINlR5XzoSX45Fwdx+cFwO0xNAzMnSPNeQQA6O77EWiAWvNXJDfwuZt6po1bBme0CvVoXhoog0fD1c8MPClAHdyuZXdUBM8TY1OTnH49o+k3v0yJBz7iodBgLPCWSuuPvzG3XvHHtZoSf6B0GHugHTU3nRbmi3o8hXx2eQ2CZnK9d3spyWcDfv\\\\\\\/Mlp5nsm2tG4JtHDLWCl1KuE8+29EGZvJbDTLK7VXCwqn1VabnH1ZQ4eMZr\\\\\\\/eICuIUWmbySq3ORPaN2nDdgcwSFgXnTkP9B4GC\\\\\\\/4fR2XxnDTWklT5ufvXHG4B9tJvzUnPvAEllew8kq5f3V\\\\\\\/qZU\\\\\\\/Av1hZzxIpPNhrgnDcPsaZCJQZvH6Umn5ApysEuzCtfcO6WuBBfr7nCnjAQ7Bm5\\\\\\\/R7Gd87OBt4OR4+KwYtllGooVLN3U35A\\\\\\\/VIdR3U0oczfgJ2aVUlzQ4MU\\\\\\\/6icMtgNq1EWbP\\\\\\\/BLpKOIv+UCN2Agl2AUxp5nLVAlISPh+qbztlWo+zpYN0ApmJdLnqOuZV+Zk3q9q9vs0DZPTkLcsV4KQB9uY8gj0eK6Ru+\\\\\\\/vtJUXoXVglUJfCPRiFEMmZbwee3TeniJQnu1Sn1CpzkEvIGbs1wopbXLE422pT6lb+XjlBeOlgoG+0s9K9nXgHR+W1mS1nqkWvgxnx2mFsS75SEe08Tzpm+Ghmvu69R12zJtzFi+78uBWNeKYKrSoyalxhR0MIzw22FSF3VDEWByV8sDYynk+\\\\\\\/yx0rM0gX8dSfwgmzhBuh3N1j1Z5rbVh0WIPxgl5vTYQ6m1beaeqD\\\\\\\/kG7lJPWRovNQEqvtLriijXc+AndoHG1RkQ9ndr\\\\\\\/qz5jcJ13yEAFOqxN9Mj0DXHRRpml2AnMgAOcKRy6JZE6Dm+MvLVItSNVgpznmvYI3yt9LQ9eY16yHddXkpEDww4rkmbEJGSbgEz74tcWeQmMGxOTw8URYi0alIYbP9yxYiFs2fS1Pkt5nC09ezWlj0csgc4\\\\\\\/ckOm1ZCGFgKS7l5pnxrIc0ymAe2XK93FtbYykh9DDhq+\\\\\\\/4ed7JvvJHxCmjMfVb7VAFGU6RawJIcmjQGd1YoFRcNiYOvbQw7sVS\\\\\\\/eTXuDCerovSyRldV8xHQnTkUXoZBy3YrnQCUp2FDlBMgKXNuiGa+x1HhxKFPjj99AuJEmPO14DIvet2VV5ulh2jgZEmEaffB2yDuhYSInU1X3lh2B9pdE\\\\\\\/5aBSdgt1esEDULzRxG9+2ETrZ3UTrYOA7JZNpVSMWfSr+L9Rj4\\\\\\\/w031BEuZsM7tf4WgC28lMv2Ocx2Ii2iqyx1HhudNCg3CXEIibjVdUZb3fqFsA4bqwVq+MslKvRBl+SHlL69a1puXK3Ut\\\\\\\/mdgWGSz6MJ0PeR04lWH7rH5cua\\\\\\\/7dlm6UyCMl3in3Yh0BoLsLQ\\\\\\\/dUZUuupxT6\\\\\\\/lHOa\\\\\\\/zuwj\\\\\\\/cujj6GvSXXpiyQ\\\\\\\/POaDWMegOUimltvNKhUMEPMnqcapCc4eWy5WZfAkqL3nPj23Rw+bEWYd\\\\\\\/kZ7xaKe4jE6vJ7mxvTU89X8Ba9tLaaxOiATSIO53l\\\\\\\/VJqT3XIYs4VhEsXn16gnl35BIrgzEmtyqn8VlTxtBcCL4HwSubZ\\\\\\\/OCGah2867FsA0bxMeU7\\\\\\\/k8CzCnnAfyrnSfC9OvUMDVI1lFqhgEuWaeph+EgoWnd4A0aYLp31vgNo0\\\\\\\/VaaWiZxtojWu3uG9a7CIyt7ZKJOylgaznQq+TTiENeWWArle0ymisskcrR2Up67\\\\\\\/bpk1dxPjgPzt4+H0f4lq6JDRLdsLnHC2jU2tcD4QSZzsFkL8tzU0+Jf7QKKC9rK0Xf9JNHIEt3WX0fahKjcn4AuYG8bNYz5wxhArbudc71yPwIO2mPodueZokWAAWoa4em2JLaaLYyp1gVQUoNA6wkhsJ1x+buUDrVzbmUoGf0NCP9iuaO2eoCFwKTpTFGoqrExx8fEyRBZSpXvP0f09Ws99d3ninddNASDW7BJFHYFQoCoc5i2Mp0kBvZq4PV\\\\\\\/bZuI6Mp2dTI0n5n0QLXI7Be7T3SzQVCMd6jje36vNwv56DZQS7x2TZWI1GhQhWyjDaZr\\\\\\\/Hv7qAN9nG1my1hXuOmvYwyY9GKspKK6tfzi1p9rwJcURVHS1z9D66BFB3rhqObJx57RO4gvEkDKe6NsJKiFCOn4R5YqKNIcO7Kd\\\\\\\/exH1it\\\\\\\/0BhSZtG53x+bChoFWzr+9yDBx3XfjGp\\\\\\\/06Ryw5\\\\\\\/m8UATDPPhSqbLI9Rdw\\\\\\\/dUaNixsojE3Hvzvq93NT6ulmSTeod42KeJB+T9uvTi1Ebe30IQjYvnoSTZndI\\\\\\\/YNH5T7wWUGYudiKaPXmwkz9qhFhUnpyZNKEVJECdmnkY5cwNCWnDOlE0S4W\\\\\\\/wQHORBt13qEm1\\\\\\\/qm\\\\\\\/wq95FvCMMXMPhq3ypQHneuOYaz+LC2DlbAThOS3IgL42KSyEiBlsIRGNeKl4S80hxfB\\\\\\\/tmbqEIBfE8I\\\\\\\/myj0lecyexS4pVRWIWgwn9rjhJOTJetEMtnlXygHMbkV7pFeBfbJm+3PJ6PQxSHLcynzxTBWWXL3fnYWyNgieWMbRjVSqsClFH96CL9PLJuECyTMLaOX9DD8unSHTWIW6uI7SANiQsb5PDZb0sDBQSdjBQmAYzrhxoND\\\\\\\/Uzd6aIXMUd5noqFyxVsrNk3VFmFSiZrzZdX1FZW7F3ubtbjK5nyEVciVphBSSYz5th3i0Gm244Vy+Kh9rjUUj5FCrFMdK9ytJ6PyoxoWqfhHOqJFtmChb1QZnSFkGz1B7uTfUS563IAwg4qLlV7GWuML8kgjyPXG14v6zsIVoH1LVYBVKZyzW8hT1hWVtui3C64KPWleIUYP\\\\\\\/5JwChGasOhUHXY1VIqzRYccUokqKLrSr0gkIL9IeM7UZYrqJjUyhFp5MITxUGeq2BfU\\\\\\\/ut6\\\\\\\/b2SvnrAyvpnSvw3tlYJ8JMrEestSbrKgX1oJlsYC9HXnPK1uEPz8o09f0wwKrU2oHFU\\\\\\\/0FOXbTYT4XgLnXV0+zyaEIs5KPm7SxQM6\\\\\\\/6pvrobU+4wTxq5y5FWPLp70ApLyNtFvNg\\\\\\\/hHXm8HcnCv0G33knEEI2b8\\\\\\\/NzruKJT+19fJKH+KUS1VKVpx+co9qLc3FWuw1c\\\\\\\/YDr7RoE5hB+GLr4shBjdHSyEyrdpZyCW\\\\\\\/NIJCmw0G8rHHhVOmSj7QN6PeqF0wvfPyT\\\\\\\/4jk5HovWyOEnXrozUdvJfGA+Mz73Nj29n832Ylz5x+M2FRfo3y4fjsqfwjEDU1WbSDdGPprceXoihnmoJB3zfhVBrWJb7u4mMVaYTR0EbO05bn0SbDyTJmJysG1Z49I64AmOaTkf\\\\\\\/PAp1I4A2u4TMn7UXLB85J35btONmGjv\\\\\\\/yK5XBVI8SrG4ctaOu8+vuxo8TcAcOoTk\\\\\\\/5\\\\\\\/X\\\\\\\/1g1C9nPH1fMxtBZzSbwkrz4vMlYtUosNV+rvH7J63sK\\\\\\\/Gw9i8NWdHOJKTQbbxf0aiYvF0X54kpeDzydqTWazQC2HoQN1Bipr40M5I+pALc\\\\\\\/ys9je6sb5PdReWYIbR9ocgOzslVj5XGzUrB5PLBDcKXWiZ6j4TO\\\\\\\/7jIABOBorDi\\\\\\\/JaOsa5CYSim24UNEbiPkVnlRFVw0NomqPoUGQXJrmVUNiF91NsD\\\\\\\/Cye9cZo79JuOuAcRImLoDpxI6u9VYI+CvyvUmp3y4DWS7dtQOoNo\\\\\\\/oOvWjrQOSaGMs88+271c3zpCs9VYCM6alj7HfjuH0J16LbsX167mh14P5vND2CDaWbI4vj4B0wl8mJ0scWJoINpaZ7bk\\\\\\\/kjKcirEnFefdFrLUbGaD7apC4bpFl4R7rN6nhHa3APxXPSqnw1wY+0VFNSJqgOfrwsX8TNIi9zN\\\\\\\/XcZdaYSdAUiSUOj6Z6r6gqdFSfR5wkIXCgV60w0Zd4m8rouWQ8dd2FozgQx1oT37kGY1pwfDS2xiizO7uomqgMGuHqmAh2lVdmTzd31bF89Jv5w+awH0mI2XO\\\\\\\/wZ0nWR6F40rhO1dBSDvAWVK95+48us+d4HlFm5vo2z8duh8fXu1TCBh\\\\\\\/bZKvhIp+jEU0lRX7rFcIlvxedTKoXUa1etr72AtN\\\\\\\/v8E9\\\\\\\/Zhh7IpVe+w49xPYL9CgacMki2LFHtlUJaBodK\\\\\\\/6PBFEL5F3b\\\\\\\/aKzKUr5U\\\\\\\/wY\\\\\\\/+uZA1NO6Qrb4Ul2vPcjH3pclWfebxbq1dmyCyTrJSNAEYmsvTSxuAihhNRFkRqJEk7mD8gLv0dqaQkT4w8Kp2QSkrChvvrswfOfctISuIMtpycHeQESVfyyjoNC3tqkqOABc3NLiLLl+MwxxmnPPtxBxkvZxrXYCHHxb7iKEwTNU3Y6os\\\\\\\/7F5D3baP2iAKOqmZ0ZJt3D3YNS2KxdjrvGG5kp3fAC1K+8VRfJTY7YNENTDTPri9YxZfHDvLjtJs\\\\\\\/rTB4IMJ4RJjBXe8MYLGjJkpvra2JRMxh+30P7zzp1x03k8NG4dedcmqfZsA3d7VhcIBe8XxCoa34B3b+nLXRjkB4bmf+LdP3rFs8fgjDYO\\\\\\\/buzxnzqDdlWAXcFkW77Vlu1quImAY7Vjuo9dqXK\\\\\\\/h09AtmADHyM91MrznTlsttuVIbqwtOUrEvH6\\\\\\\/Xp4PM035y35Q0ljAFqboNYxL\\\\\\\/GeQaJDfrPBadv2BzKGOsGZn2Ao6JjESrxbWfUNC0nfV1M67rDDQ8qCDJKUzYPH3Kzn11+GuqAY3Mr\\\\\\\/aiXYer2B3SN9MjxP4blBJbm2mCXykOjslVw\\\\\\\/wFZMoZLZIZ3VuwG1Kgw3d\\\\\\\/fkfpA8xysRaSYF5d4YdYqk615MeUj8OW0+sIG0KHdjj8W5JXmdmMtfBsyaG\\\\\\\/b7L5dnJRRzFC8mbUaSGOkc3eZrNeZ3GwQsuYWerlG7ZOyq\\\\\\\/RlYZLGre08DUbNcczzGRMh+QfjASKZDRnzGU86u7W5p0C2X\\\\\\\/SL21wBmL5yiIZNSZv9NALVabPXgr8SUcjs1m\\\\\\\/Y5NvZ\\\\\\\/5ZFmFPaDjBQ1ZUs3x+Y1bIBin4jrLsmVoscDjSHVkvIPZ8pDgJmUxrVkPO\\\\\\\/mc0AS\\\\\\\/7LX1iYoR9sPgb9WAS4kEbPCSpcicUhN0ePBOq1qxKEC2qXamg6bCv4+ulm4oNS0L6g9fjN2yOAhnbWfLP9NurDiSXmgC3VSCQGBlaJxtcZnJ2rMtGZv008giD3OCTK1JWZEjy6Qtgaci4S8AUr65730WnMSacpulT0yuZl+iqbJJirlPX8GIvHd1dtStRFh+99dzw2xSvffLfcVnS1ZIbKQa+fjLNlfK65sCq1ykHhSwtytQ2lvLE5LYvqZ5vLQYvbqnjM7vKVB+V0Wyf0qntiTiViI51pmYA0nLEmdMyqQDhJdTZYhVvtsVVURXE6rD8gyNGW9NddugNXZ32jy\\\\\\\/Y5NOeOhLYvQ3JFJNFnhyT0pp+LWWKus7jaV41QGTdS4YOZ\\\\\\\/pKVUSBq3TQB7802Op9+ZwTMvMFR3k8x9kQmcIx+18DgQt7JWaXdTKs129y5jMZWke5HJ8G0P2zdd8VvoY0kVcwEtLwoBGuzRHSZfciEFj+djHoIc+px8Yxx9820Uwt5niG3bH0SRgyezFaUS0xyahRlFQctvStwNXfQpTAAA2TTCEPz9Ox4rrq9B1s6+AjxN38Z1rCUmJpKNrUyz9QsfjWi+wDQKPJJKEQ1Olkuy+G2eGl9EGBrVtpy6zHdC6LH4gUPUpK8zY5RLHi76EJOsFmlxkjsqcIUUpUbijVJqHJ9EdWNi3X7sZ3h6qYa5nT78RBO9vnawEfyk9gNE+MppaBH7V0Z23wQ7fGmbbygS+4v4lWewnWq3AVh0EyX2Pln\\\\\\\/\\\\\\\/Qnp8omgM60axvQw1xuVaxvvP9OfV+iLNT1cg+zFntUC\\\\\\\/bZRyHGtqCIdbXvd11Ly9iD\\\\\\\/EDUG8fvsJsG\\\\\\\/C+IT8IpPXyQieEC95OxMdJ3NzAbwQRS5QrcmR3+wJj0ota+gm+jgBqke7wWYbolbsSmnkcOBXRTxzPbzvwv5wf6JCYwnaIKi4ZAwTdn6pnODXfQ7Hf0m8yd\\\\\\\/YGltcdb0PnrO3aJUjOM9xQlxAxXU\\\\\\\/PTxx6awo4kWBHRemwsOSOAQCIWLZVuqQbdmYN5h+8qgm8+wduanBtaCsiQyVywahqNCi+nO\\\\\\\/v7uvo9G+WvjCx\\\\\\\/fmbk4lgnmFQ9lT0u+H0cT9\\\\\\\/mD0ZVeGjqX6Xh0iDUW4KJVaAcOPg62aggx3kssqrrZ8i6aytRfI96MCa7Lk1+DFUaiLhJ9Wg86kgmV2M\\\\\\\/CGuqf4tGL8ZImt0ZoCqHCZQvWmsBKg2UPj6Y6HJd18M3puPJyHOks0aeZCQxRF1EkBht\\\\\\\/C+FFINTVYs4h\\\\\\\/ld\\\\\\\/n+zbwtrN8iZ6GOhrTDj9rHOM+WD8Qu22KLtjyzrbFIea1TM3Sjs1qftbN8xoLzsXs6WptTcay9CPWxZJwb9JdThtKg93885Jk5O0N\\\\\\\/hPwVdF6hZmGrAvjxDk6AVEBqDcBNHckgq89J\\\\\\\/vZ041ML\\\\\\\/3A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AgX1v4TGoNMyP22tull\\\\\\\/Aw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9ae31b0cef649dfaefb045597e8d4b97268448172673143a70dd5a8ba22e890\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Y8o8Gq9sRmfJYSLg2H7aENdmhGEbM4DIa+gFzeR\\\\\\\/j9aOL4yStShEjwySYWHSIU1egmgBbdJ1\\\\\\\/HEkq7UbBaB6IBQL7y+KHc37fmalB8x+Za2246bk07\\\\\\\/ejPSua4HKxFK6T0+JPJW8Oig9pKjAKI6WcfcW0WVeV7RKszcFUtk96Tx86xJhieZFW90IQjCapOROHMlst2\\\\\\\/jaJPfKB3IMpq6sa+bCf7l\\\\\\\/FujlhQysoRQCgOd7tDhQXEc0uATI7aM\\\\\\\/b82uYLoiHYrPdsuiWyZB+Qbv1Q5vnRj4XTm9XqcKmoNhqPohHblyJXPRZed7LdSCeRaB6Oxi0VGbJIi14Q0D+yl2WQIaBDtc952S+n7Iku4QIZ6zQBYicVIml0+YP+AxBbFJCTM6JdJILO9qD+4XqpCa\\\\\\\/TWmFtgdWC4D8XZEm7yPPX1EmePok6o9E\\\\\\\/3nSI\\\\\\\/TvEjAIzqH2sQnD3SfYS1cBBUp5SIj7Ou\\\\\\\/bOhlSMD1wSIr7Yg3RC1v5BfIImdLz1o+NPD5a4sKLZ7p5UqpBztS4m+eYgJwjNqLWxycslfDfLYvjrMXDE8mIdqMqeRJnbA3MLA8QHMXcLngwQDCpPua7AxFzXKpj8ZYBOtuJ6sebCPFP1eMKpbTEWNXPaA7j4pqRIPeE1DGlr0+CXgfSVpUZgwzkHoG6tNJb2B3Q2vyL1ZL10nL5NmWPfeuPNeQQY8PgpyDtfP7rysnPz7HvJm+0KsC5CEVC65tIbEoHt9eY6OZY\\\\\\\/2wN842sVUaL4Ty6q3mo35isWUynocXSWMCCx4l2kbDH6KWHAtAtbeXFalrAeQw01rT2pW0NHjRW7h2Gp9BaHUnsVKVTLdPPgdZkJTcAwj7RL7mFb9OdUWfdNwjPaSBkq5ABZBrHUe+WLefchHXkmKWLl+ZvPeMi01m358addJ7WnaqscQ1LzrrV\\\\\\\/Sj9ASxUysKF\\\\\\\/D\\\\\\\/SHnwXgpL4Ny6GNOw4G0ZY3Wn2FI3PFfXYrRJot\\\\\\\/97YbGlujZYuldTIGxAApKFYlHLR3UiozNu9mVlU2V3bLTBHvmUUm9xKtCg+NW4BiW3RQVdoDej7zK1\\\\\\\/ambZ0XR3g9QPA0i96n\\\\\\\/sJ4ZqGoMrHNf5sHN\\\\\\\/vbPVQDD2gkhmuys+oPG3tHLa+c0VasgJxB2n\\\\\\\/IoX84T8yufTiCmuPKEhD8PuLWFb\\\\\\\/bZdMxEYtrgIzJfbEFzNAwaykaEsYyCbRieyWyJWN5S1DWD4GGxBEmWIXuE+KHEu0BLf2DV6JoDHOptYCAK+6+b+QBo14+zZmfUVGKTMLinlK6IsA3VLy7aAeVcBwaeiq6vCvWTR\\\\\\\/2YTJdvWO3F2JBCzDXf8EFvMGDoo14PGdMAkbmHUALu9uIEbQqspgNkFN5f2E9owbEZnhhbPytqDEugOQ4lUCpfp\\\\\\\/VzDr1t+oLGtntI\\\\\\\/9OC4suBMMNCw3TfaU3YBPfDBumpIlhXuucdSMSt2KIn5PHQ\\\\\\\/pbZ8lPIQRId\\\\\\\/ydxZtuhFYJcZ\\\\\\\/BW5EIqab3wSaq+JGrUGXcEjIv4ram84JMbmWrJ4TRMWbRlqrEX9Tvqjps3Y2sLC3XMJAbEBI8em6W9NmumQYblaleFuy86wfndyttQ1LgkLK5rMh8ZnYO7CVUi5Ib9jq5+FYCIBVCkAHO4NyzP\\\\\\\/52j+PsGXJw1oopouTa5W\\\\\\\/S0gJLGtPPEKFjAe\\\\\\\/Wy5SHBp3TF5LQ8Eqqsvj6xuKpME2acNNPbIErp2fnqD2eGjmA+STsIJSPGHOJJOxzowLHEUl9JcAEvbhQtc5gMKUo4RB3\\\\\\\/ha4ysmzICp53sPdfksKxRgF66E1uFHMqsTyRs\\\\\\\/8Vl3fVjsM7gcUfWCj4sAsqP4FU1LgEMJSUJqWZ56OYjHbLOo36Ob3NKbYsC9dc+tMuN+sCAckbKUDhvuIJwPSoBOiwL3irNZnvCbZIZo8SsfGms2Koo14kXw18cgsxaAghO+5IhxMnW14npbG7d2jbJpv3TucDpWHSSJHS31SUa5gtXbz0D4e9xXz\\\\\\\/ZGvlPIq\\\\\\\/1dvH\\\\\\\/78GLOP3pZ7kZeCQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CRnr6JmKjSAuPraJpoUj4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002206dec65288e509acdea8bb104cde62431d130c1f0fd0605faa2506cc3a20d9bc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492432 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/841" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qqz2k4k4nTEbiDGGDiboXBVjRweRajhSm0luOCWJAfE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/841" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5ypcGOpHDG_-kCrS9nWx8jwxJOJD9svGbXDAZx5AOzs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/829" title="Auxin-Inducible Depletion of the Essentialome Suggests Inhibition of TORC1 by Auxins and Inhibition of Vrg4 by SDZ 90-215, a Natural Antifungal Cyclopeptide" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/855" title="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492432" id="node492432" data-pisa="ggg;9/3/841" data-pisa-master="ggg;g3.118.200921" data-apath="/ggg/9/3/841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Aegilops tauschii</em> Genome Sequence: A Framework for Meta-analysis of Wheat QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiale Xu</span>, <span class="highwire-citation-author" data-delta="1">Xiongtao Dai</span>, <span class="highwire-citation-author" data-delta="2">Ramesh K. Ramasamy</span>, <span class="highwire-citation-author" data-delta="3">Le Wang</span>, <span class="highwire-citation-author" data-delta="4">Tingting Zhu</span>, <span class="highwire-citation-author" data-delta="5">Patrick E. McGuire</span>, <span class="highwire-citation-author" data-delta="6">Chad M. Jorgensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Dehghani</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Gulick</span>, <span class="highwire-citation-author" data-delta="9">Ming-Cheng Luo</span>, <span class="highwire-citation-author" data-delta="10">Hans-Georg Müller</span> and <span class="highwire-citation-author" data-delta="11">Jan Dvorak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">841-853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200921 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492432" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiale Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiale%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiale%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiale Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiongtao Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Iowa, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiongtao%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiongtao%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ramesh K. Ramasamy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BK.%2BRamasamy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramasamy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BK.%2BRamasamy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Le Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tingting Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tingting%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATingting%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick E. McGuire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BE.%2BMcGuire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGuire%20PE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BE.%2BMcGuire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chad M. Jorgensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BM.%2BJorgensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jorgensen%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BM.%2BJorgensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hamid Dehghani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Breeding, Faculty of Agriculture, Tarbiat Modares University, Tehran, Iran, and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BDehghani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dehghani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BDehghani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Dehghani</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick J. Gulick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Concordia University, Montreal, Quebec, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BGulick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gulick%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BGulick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Gulick</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ming-Cheng Luo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ming-Cheng%2BLuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMing-Cheng%2BLuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hans-Georg Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Georg%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Georg%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jan Dvorak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BDvorak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dvorak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BDvorak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdvorak@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jdvorak@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/841" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492432" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492432/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/841.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492432" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492432/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/841.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492432" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492432/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/841.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492432" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492432/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/841.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492432" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492432/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Numerous quantitative trait loci (QTL) have been mapped in tetraploid and hexaploid wheat and wheat relatives, mostly with simple sequence repeat (SSR) or single nucleotide polymorphism (SNP) markers. To conduct meta-analysis of QTL requires projecting them onto a common genomic framework, either a consensus genetic map or genomic sequence. The latter strategy is pursued here. Of 774 QTL mapped in wheat and wheat relatives found in the literature, 585 (75.6%) were successfully projected onto the <em>Aegilops tauschii</em> pseudomolecules. QTL mapped with SNP markers were more successfully projected (92.2%) than those mapped with SSR markers (66.2%). The QTL were not distributed homogeneously along chromosome arms. Their frequencies increased in the proximal-to-distal direction but declined in the most distal regions and were weakly correlated with recombination rates along the chromosome arms. Databases for projected SSR markers and QTL were constructed and incorporated into the <em>Ae. tauschii</em> JBrowse. To facilitate meta-QTL analysis, eight clusters of QTL were used to estimate standard deviations (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-1.gif"/></noscript></span></span>) of independently mapped QTL projected onto the <em>Ae. tauschii</em> genome sequence. The standard deviations <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-2.gif"/></noscript></span></span> were modeled as an exponential decay function of recombination rates along the <em>Ae. tauschii</em> chromosomes. We implemented four hypothesis tests for determining the membership of query QTL. The hypothesis tests and estimation procedure for <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-3.gif"/></noscript></span></span> were implemented in a web portal for meta-analysis of projected QTL. Twenty-one QTL for <em>Fusarium</em> head blight resistance mapped on wheat chromosomes 3A, 3B, and 3D were analyzed to illustrate the use of the portal for meta-QTL analyses.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl-distribution" class="hw-term hw-article-keyword hw-article-keyword-qtl-distribution" rel="nofollow">QTL distribution</a></li><li class="kwd"><a href="/keyword/meta-qtl" class="hw-term hw-article-keyword hw-article-keyword-meta-qtl" rel="nofollow">meta-QTL</a></li><li class="kwd"><a href="/keyword/qtl-database" class="hw-term hw-article-keyword hw-article-keyword-qtl-database" rel="nofollow">QTL database</a></li><li class="kwd"><a href="/keyword/probability" class="hw-term hw-article-keyword hw-article-keyword-probability" rel="nofollow">probability</a></li><li class="kwd"><a href="/keyword/web-portal" class="hw-term hw-article-keyword hw-article-keyword-web-portal" rel="nofollow">Web portal</a></li><li class="kwd"><a href="/keyword/recombination-rate" class="hw-term hw-article-keyword hw-article-keyword-recombination-rate" rel="nofollow">recombination rate</a></li><li class="kwd"><a href="/keyword/fusarium-head-blight" class="hw-term hw-article-keyword hw-article-keyword-fusarium-head-blight" rel="nofollow">fusarium head blight</a></li></ul><p id="p-3">Wheat is the most widely grown field crop and an important source of protein and carbohydrates in the human diet. Intelligent manipulation of genetic variation of economically important traits in wheat, most of them quantitative, is essential for sustaining wheat genetic improvement and meeting demands for wheat grain in the future.</p><p id="p-4">Commercially important wheats are hexaploid bread wheat (<em>Triticum aestivum</em>, subgenomes BBAADD) and tetraploid durum wheat (<em>T. turgidum</em> ssp. <em>durum</em>, subgenomes BBAA). Wheat subgenomes were contributed by <em>T. urartu</em> (A subgenome), a close relative of <em>Aegilops speltoides</em> (B subgenome), and <em>Ae. tauschii</em> (D subgenome) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">McFadden and Sears 1946</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dvorak and Zhang 1990</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dvorak <em>et al.</em> 1993</a>). The durum A and B subgenomes, as well as those of wild tetraploid emmer wheat (<em>T. turgidum</em> ssp. <em>dicoccoides</em>), are equivalent to the A and B subgenomes of bread wheat.</p><p id="p-5">The bread wheat genome consists of 21 chromosomes. They were allocated into seven groups of three homeologous chromosomes, one from each subgenome (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sears 1966</a>). The ability of the three chromosomes within each homeologous group to replace each other in the nullisomic-tetrasomic aneuploid lines indicated that the majority of wheat orthologous genes have the same or similar functions. The same is true for genomes of most other species in the tribe Triticeae, as compensation in the numerous disomic substitution lines that have been developed in the wheat genetic background has revealed. In addition to wheat, the tribe includes other cereals, such as barley and rye, and nearly 300 grass species (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Löve 1984</a>). Most of them can be hybridized with wheat and represent an important genetic resource for wheat improvement. Although wheat cannot be sexually hybridized with grasses outside of the tribe, they are nevertheless important for wheat genomics (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lin <em>et al.</em> 1995</a>). The functions of genes in other grass genomes suggest functions of wheat genes and facilitate the discovery of candidate genes for wheat qualitative and quantitative traits.</p><p id="p-6">The importance of genome comparisons for wheat genetics and breeding became evident at the onset of wheat molecular mapping (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gale and Devos 1998</a>), which was based on restriction fragment length polymorphism (RFLP) markers. RFLP markers had numerous drawbacks but were comparative. Subsequently, more than 8,000 simple sequence repeat (SSR) markers have been developed in wheat (<a href="https://wheat.pw.usda.gov/GG3/">https://wheat.pw.usda.gov/GG3/</a>). The primers for most of them were engineered to be subgenome specific to facilitate mapping at the polyploid level. Wheat SSR markers are therefore poorly suited for comparative mapping. The use of single nucleotide polymorphism (SNP) markers is currently gaining importance because of their abundance in wheat and wheat relatives and their suitability for genotyping automation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akhunov <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Luo <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akhunov <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Luo <em>et al.</em> 2013</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2014</a>). Gene-based SNP markers can be detected by homology searches in related genomes and are therefore more comparative than SSR markers.</p><p id="p-7">Many genetic maps have been constructed with RFLP, SSR, and SNP markers and some have been integrated into consensus maps (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Somers <em>et al.</em> 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Marone <em>et al.</em> 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Maccaferri <em>et al.</em> 2015</a>). Consensus genetic maps create a framework for integration of quantitative trait loci (QTL) from different maps and their meta-QTL analyses (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Quraishi <em>et al.</em> 2017</a>). A meta-QTL analysis is a method derived from meta-analysis, a statistical concept of integrating independently obtained data into a single analysis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Goffinet and Gerber 2000</a>). Major assets of meta-QTL analyses built from independent QTL studies (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Veyrieras <em>et al.</em> 2007</a>) include increased QTL detection power, reduced size of confidence interval in which a QTL is located, and better control of false positive candidate-gene discovery. Meta-QTL analysis has been successfully deployed in wheat (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hanocq <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Griffiths <em>et al.</em> 2009</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tyagi <em>et al.</em> 2015</a>). A consensus genetic map involving 140,000 molecular markers hosting 32 meta-QTL has been built by combining four wheat genetic maps and 27 wheat QTL studies (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Quraishi <em>et al.</em> 2017</a>).</p><p id="p-8">The backbone for integrating markers, Mendelian genes, and QTL should ideally be a genome sequence, not a genetic map (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jia <em>et al.</em> 2013</a>). A genome sequence is comparative and markers can be incorporated into it on a large scale by searches for homology. QTL can be projected onto a genome sequence relative to each other and relative to genes annotated in the genome sequence, thus streamlining candidate-gene discovery.</p><p id="p-9">This strategy for meta-QTL analysis in Triticeae has been until recently largely hypothetical because no reference-quality genome sequence was available in the tribe. The publication of genome sequences for both wild ancestors of wheat, the tetraploid wild emmer wheat (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Avni <em>et al.</em> 2017</a>) and <em>Ae. tauschii</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zhao <em>et al.</em> 2017</a>) has changed this situation. <em>Ae. tauschii</em> genome sequence Aet v4.0 (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Luo <em>et al</em>. 2017</a>) has been used as a backbone for large-scale projections of markers from a genetic map onto the genome sequence (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jorgensen <em>et al</em>. 2017</a>). Of 10,914 A- and B-subgenome SNP markers mapped on an ultra-dense genetic map of wild emmer wheat, 9,131 (83.6%) were successfully projected onto the <em>Ae. tauschii</em> Aet v4.0 pseudomolecules (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jorgensen <em>et al.</em> 2017</a>) providing proof of concept. Zhao <em>et al.</em> (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Zhao <em>et al.</em> 2017</a>) later also reported success with projecting molecular markers and QTL onto another <em>Ae. tauschii</em> genome sequence.</p><p id="p-10">Based on our success with projecting SNP markers onto the Aet v4.0 genome sequence, we evaluated here the projection of wheat SSR markers onto the <em>Ae. tauschii</em> pseudomolecules. We then used projected markers to project QTL onto the <em>Ae. tauschii</em> pseudomolecules. We performed a meta-analysis of eight clusters of projected QTL clustering about their causative Mendelian loci to estimate standard deviations of mapping and projecting QTL onto the <em>Ae. tauschii</em> genome sequence. We observed that the size of standard deviation decayed exponentially as the meiotic recombination rate along the <em>Ae. tauschii</em> chromosome arms (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>) increased and derived a model for computing standard deviation of projected QTL onto the <em>Ae. tauschii</em> Aet v4.0 pseudomolecules. Three hypothesis tests were proposed to investigate the association of a new (query) QTL with an existing individual QTL (QTL <em>vs.</em> QTL), a Mendelian gene (QTL <em>vs.</em> Mendelian locus), or known cluster of QTL (QTL <em>vs.</em> cluster mean). A fourth test for detecting multiple clusters within a set of QTL was constructed from the QTL-vs-QTL test. Finally, we developed a portal for computing standard deviations adjusted for local recombination rates and probabilities for meta-QTL analyses and performed a case study for 21 QTL for <em>Fusarium</em> head blight resistance mapped on the three chromosomes of homeologous group 3 to illustrate the use of the portal and hypothesis testing.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>QTL data retrieval</h3><p id="p-11">Data for 258 SNP-associated QTL and 500 SSR-associated QTL were extracted from 97 studies in wheat and wheat relatives. In addition, 16 additional QTL mapped with some other marker types, <em>e.g.</em>, EST, were also collected. The QTL were for traits related to biotic stress resistance, abiotic stress tolerance, grain yield, grain quality, and various physiological and morphological traits. For QTL mapped relative to a single marker, nucleotide sequence of the marker was retrieved from the relevant marker database. For QTL mapped relative to two flanking markers, sequences for both flanking markers were retrieved from the database.</p><p id="p-12">Of the 258 QTL mapped with SNP markers, 226 were mapped with an SNP array, such as the 90K Illumina iSelect Wheat Infinium SNP array (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2014</a>), 9K iSelect Wheat BeadChips array (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>), and the 35K Wheat Breeder’s Array (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allen <em>et al.</em> 2017</a>). The remaining 32 QTL were mapped by genotyping by sequencing (GBS) (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zhou <em>et al.</em> 2017</a>). Sequences of the 90K and 9K SNP markers were downloaded from the Kansas University SNP marker database (<a href="http://wheatgenomics.plantpath.ksu.edu/snp/">http://wheatgenomics.plantpath.ksu.edu/snp/</a>) and those of the 35K SNP markers were downloaded from the University of Bristol CerealsDB (<a href="http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/indexNEW.php">http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/indexNEW.php</a>). For the 32 GBS markers, sequences of the relevant reads were retrieved from the Chromosome Survey Sequence published by IWGSC (<a href="https://www.wheatgenome.org/">https://www.wheatgenome.org/</a>). Primer sequences of 6,385 SSRs were downloaded from the GrainGenes database (<a href="https://wheat.pw.usda.gov/GG3/">https://wheat.pw.usda.gov/GG3/</a>). In addition to marker sequences, positions on a genetic map, mapping population, and journal reference were recorded.</p></div><div id="sec-3" class="subsection"><h3>Projection of wheat SNP and SSR markers onto the Ae. tauschii genome sequence</h3><p id="p-13">SNP marker sequences and GBS reads were used as queries against the <em>Ae. tauschii</em> reference genome sequence Aet v4.0 (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>). Homology searches were performed (<a href="https://www.cyverse.org/acknowledge-and-cite-cyverse">https://www.cyverse.org/acknowledge-and-cite-cyverse</a>) with BLASTN in the NCBI BLAST+ v2.7.1 package using the default search parameters except for word-size, which was adjusted to 8. A hit was considered successful if it was located on the <em>Ae. tauschii</em> chromosome homeologous to the wheat chromosome on which the SNP or GBS marker was originally mapped. Hits on other <em>Ae. tauschii</em> chromosomes or multiple hits were discarded. If there was more than a single hit on the <em>Ae. tauschii</em> homeologue, the SNP marker was discarded.</p><p id="p-14">Because SSR primers are short, BLASTN homology search with SSR primers as queries was first performed against the <em>Triticum aestivum</em> cv Chinese Spring (CS) pseudomolecules (<a href="http://aegilops.wheat.ucdavis.edu/ATGSP/blast.php">http://aegilops.wheat.ucdavis.edu/ATGSP/blast.php</a>), to retrieve the entire amplicon. The hits were filtered as follows. (1) A hit had at least a 95% identity. (2) The distance between the forward and reverse primer sequences was &lt; 1 kb (changing this filter to &lt; 10 kb was explored and is reported). (3) The hit was located on the wheat pseudomolecule corresponding to the location of the SSR marker in the GrainGenes database. The sequence of the amplicon was then used as a query in a BLASTN homology search against the <em>Ae. tauschii</em> genome sequence. If the search resulted in hits on several <em>Ae. tauschii</em> pseudomolecules the hit on the homeologous pseudomolecule was selected. If there was more than a single hit on the <em>Ae. tauschii</em> homeologues, the marker was discarded.</p></div><div id="sec-4" class="subsection"><h3>Projecting and validating QTL on the Ae. tauschii genome sequence</h3><p id="p-15">A QTL was projected onto an <em>Ae. tauschii</em> pseudomolecule Aet v4.0 by projecting either a single marker near the LOD score peak or a pair of flanking markers associated with the QTL LOD score. If a pair of markers was projected, the QTL was assumed to be located at the midpoint between the two marker locations. To validate a QTL projection based on a single marker, another marker in the proximity of the projecting marker was selected from the linkage map in the original study or from a genetic map in the GrainGenes database. This additional marker was projected onto the <em>Ae. tauschii</em> genome sequence. If the projection of the additional marker was in the vicinity of the original marker, this additional marker was considered to be a validating marker and the QTL location was considered validated. For QTL projected onto the sequence by a pair of flanking markers, a QTL was considered validated if both markers were located in the <em>Ae. tauschii</em> genome sequence in the vicinity of each other. If one of the two flanking markers could not be projected onto the sequence, another marker was selected and used for validating the QTL as described above.</p></div><div id="sec-5" class="subsection"><h3>Distribution of projected QTL</h3><p id="p-16">In the <em>Ae. tauschii</em> genome, gene density increases about fourfold from the centromere toward the telomere (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>). To assess the distribution of QTL along chromosome arms, the numbers of QTL were expressed per bin containing equal numbers of genes rather than per Mb. Genes in an arm were divided into five bins, each containing 20% of genes in the arm ordered along the centromere-telomere axis. The frequency of QTL (the number of QTL per bin) was computed for each bin. Frequencies of QTL across different chromosome arms were therefore comparable and could be averaged.</p><p id="p-17">The analysis of QTL distribution was performed separately for the long arms and short arms. In addition, we analyzed the effects of marker (SSR or SNP) used for QTL mapping on the QTL distribution along the long and short arms.</p></div><div id="sec-6" class="subsection"><h3>JBrowse tracks and QTL naming</h3><p id="p-18">Databases containing projected markers and QTL were included into the <em>Ae. tauschii</em> Aet v4.0 genome sequence JBrowse database (<a href="http://aegilops.wheat.ucdavis.edu/jbrowse/index.html?data=Aet/data">http://aegilops.wheat.ucdavis.edu/jbrowse/index.html?data=Aet/data</a>). Marker and QTL tracks were added to the JBrowse graphical display (File S1). To incorporate projected markers and QTL into the JBrowse, each QTL received an identification number (ID). ID starts with the letter “Q” followed by a four-digit number. The first digit indicates the <em>Ae. tauschii</em> pseudomolecule (1D to 7D). The second digit indicates a 100 Mb section starting at the tip of the short arm of the <em>Ae. tauschii</em> pseudomolecule. The third digit places QTL into ten sections and the last digit subdivides each section into 10 sub-sections, starting at the midpoint. Therefore, a 100 Mb section can contain up to 100 ordered QTL. QTL that are sharing the same four digits because of the same associated marker are differentially designated by using one or two digits after a decimal point. For example, for QTL Q1028.1, the first digit indicates that the QTL was on pseudomolecule 1D, the second digit indicates that it was in the section 0-100 Mb from the tip of the short arm, the third and fourth digits (2 and 8) indicate that the QTL was arbitrarily placed in the third sub-section and ninth sub-sub-section of the 100-Mb section. The decimal digit (0.1) indicates that there are several QTL, which are projected at the same location of pseudomolecule 1D.</p></div><div id="sec-7" class="subsection"><h3>Meta-QTL hypothesis testing</h3><p id="p-19">Testing meta-QTL hypotheses requires an estimate of the standard deviation of independent QTL mapping attempts. To estimate the standard deviation, eight clusters of QTL for which the underlying Mendelian loci were known were constructed.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-20"><strong><em>GW2</em> cluster</strong> <em>GW2</em> encodes E3 ubiquitin-protein ligase and regulates grain weight in rice (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kikuchi <em>et al.</em> 2003</a>). The wheat ortholog <em>GW2</em> on 6A negatively regulates grain width and thousand kernel weight (TKW) (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Song <em>et al.</em> 2007</a>). Three A- and B-genome QTL affecting TKW (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Huang <em>et al.</em> 2006</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wu <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gao <em>et al.</em> 2015</a>) projected in the proximity of <em>GW2</em> were used as a <em>GW2</em> cluster.</p></li><li id="list-item-2"><p id="p-21"><strong><em>Glu-1</em> cluster</strong> <em>Glu-1</em> encodes high-molecular-weight glutenin subunits (HMW-GS) (Forde <em>et al.</em> 1985). The locus is proximally located in the long arms of wheat chromosomes 1A, 1B, and 1D and is a major determinant of wheat baking quality (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Payne <em>et al.</em> 1981</a>). Ten A- and B-genome QTL for various measures of wheat baking quality, such as falling number (FN), test weight (TW), grain protein content (GPC), mixograph peak height (PH), sedimentation volume (SED), and dough strength (DS), known to be associated with <em>Glu-1</em> were used as the <em>Glu-1</em> cluster.</p></li><li id="list-item-3"><p id="p-22"><strong><em>GLEN42K</em> cluster</strong> <em>GLEN42K</em> is a member of a multigene locus <em>Glu-3</em> located in the distal ends of chromosomes 1A, 1B, and 1D and encodes a LMW-s (serine) subunit, one of the components of low-molecular-weight glutenin (LWM-GS) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Maruyama-Funatsuki <em>et al.</em> 2005</a>). This subunit is thought to be associated with good pasta-making quality (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pogna <em>et al.</em> 1988</a>;<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Masci <em>et al.</em> 2000</a>). Five A-, B-, and D-genome QTL (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">McCartney <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arbelbide and Bernardo 2006</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yu <em>et al.</em> 2017</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zou <em>et al.</em> 2017</a>) for several measures of wheat baking quality, such as SED, TW, and grain hardness (GH), were used as this cluster.</p></li><li id="list-item-4"><p id="p-23"><strong><em>Vrn-B1</em> cluster</strong> <em>Vrn-B1</em> encodes MADS-box protein TaVRT-1 (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Yan <em>et al.</em> 2003</a>), and deletions within this gene are associated with altered vernalization requirements and spring growth habit in wheat (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fu <em>et al.</em> 2005</a>). Four A- and B-genome QTL (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hanocq <em>et al.</em> 2004</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hanocq <em>et al.</em> 2007</a>) for vernalization requirement (VR) projected in the proximity of <em>Vrn-1</em> were used as the <em>Vrn-1</em> cluster.</p></li><li id="list-item-5"><p id="p-24"><strong><em>Yr10</em> cluster</strong> <em>Yr10</em> encodes a CC-NBS-LRR sequence in wheat (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liu <em>et al.</em> 2014</a>) that confers stripe rust resistance (SRR) in wheat cultivars (<em>e.g.</em>, Moro) and was located at the tip of the short arm of chromosome 1B (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Metzger and Silbaugh 1970</a>). Three B- and D-genome SRR QTL (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hou <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Naruoka <em>et al.</em> 2015</a>) projected in the proximity of <em>Yr10</em> were used as the <em>Yr10</em> cluster.</p></li><li id="list-item-6"><p id="p-25"><strong><em>HKT4</em> cluster</strong> <em>HKT4</em> encodes cation transporter that mediates potassium (K<sup>+</sup>) uptake in rice (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garciadeblás <em>et al.</em> 2003</a>). Comparative mapping in rice, barley and wheat revealed that the wheat ortholog was on the long arm of chromosome 2B (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Huang <em>et al.</em> 2008</a>). Two QTL (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lindsay <em>et al.</em> 2004</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hussain <em>et al.</em> 2017</a>) for shoot K<sup>+</sup> concentration (SKC) were projected in the proximity of <em>HKT4</em> gene and were used as the <em>HKT4</em> cluster.</p></li><li id="list-item-7"><p id="p-26"><strong><em>Ppd-1</em> cluster</strong> <em>Ppd-1</em> encodes a key floral regulator. The gene was located on wheat chromosomes 2A, 2B, and 2D (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Worland 2001</a>;<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beales <em>et al.</em> 2007</a>;<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wilhelm <em>et al.</em> 2009</a>). Its mutations confer photoperiod insensitivity. Six QTL (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Huang <em>et al.</em> 2003</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Xu <em>et al.</em> 2005</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Huang <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kunert <em>et al.</em> 2007</a>) projected in the proximity of <em>Pdp-1</em> and affecting photoperiod sensitivity and several other related traits, such as heading date (HD), days to maturity (DTM), ear emergence time (EET), and flowering time (FT), were used as the <em>Ppd-1</em> cluster.</p></li><li id="list-item-8"><p id="p-27"><strong><em>Rht-1</em> cluster</strong> <em>Rht-1</em> encodes in rice the DELLA protein and acts as a repressor of gibberellin (GA) responsive growth (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gale and Marshall 1973</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Pinthus <em>et al.</em> 1989</a>). Wheat orthologs <em>Rht-B1 and Rht-D1</em> were mapped on the short arms of 4BS and 4DS (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sourdille <em>et al.</em> 1998</a>). Allelic variants of these two genes result in dwarfing with a broad range of plant height in wheat (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gale <em>et al.</em> 1985</a>;<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Börner <em>et al.</em> 1996</a>). Three QTL (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">McCartney <em>et al.</em> 2005</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gao <em>et al.</em> 2015</a>) affecting plant height (PTH) and projected in the proximity of <em>Rht-1</em> were used as the <em>Rht-1</em> cluster.</p></li></ul><p id="p-28">Each QTL was projected onto the <em>Ae. tauschii</em> genome sequence, and pairwise distances of QTL associated with the same Mendelian locus were used to estimate the standard deviation of a QTL cluster. Because distances between markers on Triticeae genetic maps are dependent on recombination rates along chromosomes, the local recombination rate must be taken into account in computing the <em>p</em>-values in meta-QTL hypothesis testing. To enumerate the local recombination rate, each pseudomolecule was divided by 10,000 equally spaced grid points. Based on the <em>Ae. tauschii</em> genetic map (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Luo <em>et al.</em> 2013</a>), the recombination rate at each grid point was computed by applying local cubic smoothing with Gaussian kernel and 5-Mb bandwidth (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Luo <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dai <em>et al.</em> 2018</a>).</p><p id="p-29">Within a cluster, the standard deviation σ of the distance between the projected QTL location and the mean location of QTL mapping attempts for the same Mendelian locus was estimated by a robust scale estimator (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rousseeuw and Croux 1993</a>) <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-4.gif"/></noscript></span></span>. The average recombination rate (Rt) was estimated by using recombination rates of the grid points in the cluster. A preliminary exploratory analysis found that the relationship between <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-5.gif"/></noscript></span></span> and rate Rt was properly described by an exponential decay. This suggested the use of the linear regression model<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-6.gif"/></noscript></span></span>using the logarithm of the standard deviation estimate <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-7.gif"/></noscript></span></span> as response and Rt in the chromosome region spanning the QTL cluster as predictor. After obtaining least square estimates <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-9.gif"/></noscript></span></span>, the regression model was exponentiated and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-10.gif"/></noscript></span></span> was used for computing the recombination-dependent standard deviation within a cluster.</p><p id="p-30">To compare dispersion of QTL mapped with SSR markers with those mapped with SNP markers, the adjusted distance of each QTL to the mean location of QTL in a cluster was computed as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-12.gif"/></noscript></span></span> is the adjusted distance of the <em>i</em>th QTL in the <em>j</em>th cluster, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-13.gif"/></noscript></span></span>is the mean location of QTL in the <em>j</em>th cluster, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-14.gif"/></noscript></span></span> is the standard deviation of the <em>j</em>th cluster predicted by the regression model. QTL were then divided into two groups based on whether mapping was based on SSR markers or on SNP markers. A <em>t</em>-test based on the adjusted distances was performed for the null hypothesis <em>H<sub>0</sub></em>: the mean adjusted deviations of QTL mapped by two different marker types are the same, which would hold if the size of standard deviation were not affected by marker type.</p></div><div id="sec-8" class="subsection"><h3>Model applications</h3><p id="p-31">The following methods can be used to assess whether a new (query) QTL belongs to the same cluster as a single known QTL (Test 1), a Mendelian gene (Test 2), or a set of QTL forming a known cluster (Test 3). A Mendelian gene in Test 2 is a gene of known location in the <em>Ae. tauchii</em> genome sequence. A known cluster in Test 3 is a group of QTL for which there is evidence that they are independent estimates of a single Mendelian locus of unknown location in the <em>Ae. tauschii</em> genome sequence. We consider the following hypothesis tests for these scenarios:</p><ul class="list-simple " id="list-2"><li id="list-item-9"><p id="p-32">Test 1: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-15.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-16.gif"/></noscript></span></span> and Y are QTL of the same gene <em>vs.</em> <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-17.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-18.gif"/></noscript></span></span> is not true,</p></li><li id="list-item-10"><p id="p-33">Test 2: <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-19.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-20.gif"/></noscript></span></span> is associated with a Mendelian gene of known location in the genome sequence <em>vs.</em> <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-21.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-22.gif"/></noscript></span></span> is not true,</p></li><li id="list-item-11"><p id="p-34">Test 3: <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-23.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-25.gif"/></noscript></span></span> are QTL of the same Mendelian gene <em>vs.</em> <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-26.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-27.gif"/></noscript></span></span> is not true,</p></li></ul><p id="p-35">where the location (in Mb) of the new QTL is denoted as X, that of the known (respectively set of) QTL as Y (respectively <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-28.gif"/></noscript></span></span>, and the mean location (in Mb) of the QTL cluster associated with underlying Mendelian locus corresponding to the new QTL X as <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-29.gif"/></noscript></span></span> and that of the known QTL(s) as <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-30.gif"/></noscript></span></span>, respectively. The null hypotheses of interest can then be written as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-31.gif"/></noscript></span></span> Assuming <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-32.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-33.gif"/></noscript></span></span>, respectively, where σ is the standard deviation of the locations of the QTL associated with the same gene, we used the <em>z</em>-test to obtain the <em>p</em>-value for testing <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-34.gif"/></noscript></span></span> against <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-35.gif"/></noscript></span></span>. The test statistic T and two-sided <em>p</em>-value for each test were</p><ul class="list-simple " id="list-3"><li id="list-item-12"><p id="p-36"><span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-36.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-37.gif"/></noscript></span></span>,</p></li><li id="list-item-13"><p id="p-37">Test 2: <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-38.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-39.gif"/></noscript></span></span>.</p></li><li id="list-item-14"><p id="p-38"><span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-40.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-41.gif"/></noscript></span></span>.</p></li></ul><p id="p-39">The standard deviation σ was estimated by <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-42.gif"/></noscript></span></span> where Rt is the average recombination rate of the region spanning the QTL cluster. We assume that this estimation procedure is consistent, which then justifies the application of the <em>z</em>-test for large sample sizes.</p><p id="p-40">Consecutive applications of Test 1 can also assess whether a given set of QTL at location <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-43.gif"/></noscript></span></span> form a single cluster. Specifically, to test the hypothesis</p><ul class="list-simple " id="list-4"><li id="list-item-15"><p id="p-41">Test 4: <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-44.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-45.gif"/></noscript></span></span> are QTL of the same gene <em>vs.</em> <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-46.gif"/></noscript></span></span>:<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-47.gif"/></noscript></span></span> is not true,</p></li></ul><p id="p-42">one can apply Test 1 on neighboring QTL pairs <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-48.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-49.gif"/></noscript></span></span> is the <em>i</em>th smallest QTL location, and reject the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-50.gif"/></noscript></span></span> hypothesis if the <em>p</em>-value for any of the neighboring QTL pairs is smaller than <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-51.gif"/></noscript></span></span>. The type I error for testing this single hypothesis is controlled to be smaller than α without corrections for multiple testing, for which empirical evidence is provided in a simulation study that is included in the Supplemental File S2.</p></div><div id="sec-9" class="subsection"><h3>Web portal</h3><p id="p-43">To implement the meta-QTL hypothesis testing, we constructed an open source single page web application using HTML5 and JavaScript, hosted on GitHub pages (<a href="https://pages.github.com/">https://pages.github.com/</a>). The source code is available at our GitHub “ramesh8v” repository (<a href="https://github.com/ramesh8v/tools">https://github.com/ramesh8v/tools</a>). At the portal created from this source code (<a href="https://tools.omicsspeaks.com/consmap">https://tools.omicsspeaks.com/consmap</a>), a tab was built for each of the first three tests (QTL <em>vs.</em> QTL, QTL <em>vs.</em> Mendelian gene, and QTL <em>vs.</em> cluster mean). The portal provides a menu for selection of the <em>Ae. tauschii</em> chromosome and two input boxes to enter query and target locations in bp. The web application retrieves recombination rates for the query and target locations from the database of recombination rate grid points (Table S1). The mean of the retrieved recombination rates is computed and entered into the regression model to estimate the standard deviation, which together with the query and target locations is used to compute the <em>p</em>-value.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-44">Table S1 contains the computed recombination rates at grid points on each of the seven <em>Ae. tauschii</em> pseudomolecules. Table S2 contains the roster and location of SSR markers projected onto the <em>Ae. tauschii</em> genome reference sequence. Table S3 contains the database of QTL projected onto the <em>Ae. tauschii</em> pseudomolecules. The projected markers and QTL were included into the Aet v4.0 JBrowse database. Supplemental File S1 provides information about JBrowse (<a href="http://aegilops.wheat.ucdavis.edu/jbrowse/index.html?data=Aet/data">http://aegilops.wheat.ucdavis.edu/jbrowse/index.html?data=Aet/data</a>) databases and tracks containing 1,952 projected SSR markers, 284 projected SNP markers, and 585 projected QTL. Supplemental File S2 includes simulation testing the hypothesis that error rate is controlled to be smaller than α without corrections for multiple <em>p</em>-values. A link to the web portal for computing <em>p</em>-values for the three null hypotheses was included at the <em>Ae. tauschii</em> website hosting the JBrowse (above). All supplemental material (Tables S1, S2, and S3; Figures S1 and S2; and Files S1 and S2) are available at Figshare: <a href="https://doi.org/10.25387/g3.7607060">https://doi.org/10.25387/g3.7607060</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Projecting markers and QTL onto the Ae. tauschii pseudomolecules</h3><p id="p-45">Primers for 6,213 unique SSR markers were downloaded from the GrainGenes database. Of the 12,426 primers, 11,431 (92.0%) produced one or more hits in BLAST searches. A total of 907,476 hits with an average of 73 hits per primer were obtained. For 3,212 (51.6% of the initial number) primer pairs the hits were within 1 kb. The outcome did not appreciably change when the 1kb requirement was relaxed to 10 kb, in which case hits were within the prescribed interval for 3,277 primer pairs (52.7%). The set of 3,212 amplicons was used in BLAST searches against the <em>Ae. tauschii</em> genome sequence. Of them, 1,952 (31.4% of the initial number) were projected onto <em>Ae. tauschii</em> pseudomolecules (Table S2).</p><p id="p-46">For other types of markers, only those that had been used for locating a QTL on a wheat genetic map were projected onto the <em>Ae. tauschii</em> pseudomolecules. A total of 284 SNP markers from the wheat 90K Infinium assay, 26 from the wheat 35K Infinium assay, and 61 from the wheat 9K Infinium assay, in addition to 34 SNP-containing GBS reads, two RFLP markers, four EST markers, and one SCAR marker were projected onto the <em>Ae. tauschii</em> genome sequence.</p><p id="p-47">Data for 774 QTL mapped in wheat or wheat relatives were extracted from the literature, and 585 (75.6%) QTL for 154 traits were successfully projected onto the <em>Ae. tauschii</em> pseudomolecules (Table S3). Of 500 QTL originally mapped with SSR markers, 331 (66.2%) could be projected onto <em>Ae. tauschii</em> pseudomolecules, and of 258 QTL originally mapped with SNP markers 238 (92.2%) were successfully projected. Of the three wheat subgenomes the highest success rate with projecting QTL was with those in the D subgenome, 150/174 (86.2%), followed by those in the A subgenome, 168/232 (72.4%). The lowest success rate was with QTL in the B subgenome, 201/302 (66.5%).</p><p id="p-48">Of the 585 QTL, 519 were discovered in purely wheat mapping populations whereas 66 QTL were discovered in wheat × wild relative mapping populations. Of the latter QTL, 30 involved <em>Ae. tauschii</em>, 33 involved wild emmer wheat, 2 involved <em>Triticum monococcum</em>, and 1 involved <em>Lophopyrum ponticum</em>. A total of 255 QTL were projected with two validated flanking markers and 330 were projected with a single validated marker. Of the latter, 24 QTL that were originally mapped relative to two flanking markers were anchored with only one validated flanking marker in this study, due to a failure to project the other marker onto the sequence. Fifteen QTL were projected onto the sequence with a single marker that could not be validated by any supporting marker and seven were projected by two adjacent markers on both sides of the single marker (Table S3).</p><p id="p-49">The 585 QTL were classified into six groups based on the relationships among quantitative traits (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Most QTL belonged to trait group 3 (grain yield) followed by group 2 (biotic stress resistance) and group 4 (milling and baking quality).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492455/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492455/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Distribution of projected QTL across the <em>Aegilops tauschii</em> genome</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Distribution of projected QTL in the Ae. tauschii genome sequence</h3><p id="p-50">The highest number of QTL was projected onto pseudomolecule 2D and the lowest onto pseudomolecule 6D (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), which mirrored the relative sizes of these chromosomes in the <em>Ae. tauschii</em> genome, the largest and the smallest, respectively. Three QTL mapped onto wheat chromosome 5A were projected onto pseudomolecule 4D whereas two other QTL mapped on wheat chromosome 4A were projected onto pseudomolecule 5D. Two QTL with pairs of flanking markers were included into our database but not assigned to any pseudomolecule since one marker was mapped on pseudomolecule 4D and the other on pseudomolecule 5D. These apparently incongruous projections were caused by the known 4A/5A translocation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Devos <em>et al.</em> 1995</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dvorak <em>et al.</em> 2018b</a>).</p><p id="p-51">A slightly larger number of QTL (320) was projected onto the long arms than onto the short arms (273 QTL) of the pseudomolecules. Pseudomolecule 2D was exceptional by having a greater number of QTL on the short arm than on the long arm. The average distance between projected QTL varied among the pseudomolecules, ranging from 7.2 Mb on pseudomolecule 2D to 14.3 Mb on pseudomolecule 7D. Genome-wide, the distances between QTL averaged 9.9 Mb.</p><p id="p-52">To test the null hypothesis that every gene had the same probability to become a QTL, the genes in an arm were allocated into five bins containing equal number of genes, and QTL projected onto each bin were counted. If each gene had an equal chance to become a QTL, the numbers of projected QTL in the five bins would be equal. Only one of the 14 chromosome arms, 5DS, had similar numbers of QTL in all bins (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the remaining 13 chromosome arms the numbers of QTL increased in the proximal-to-distal direction or increased to about the midpoint and then declined toward the distal end. A similar distribution of QTL frequencies was found by using a non-overlapping window of 200 genes (Figure S1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/841/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of QTL along the Ae. tauschii chromosome arms. The numbers of QTL (vertical axis) are expressed per bin each containing 20% genes in the arm. The bin closest to the centromere is to the left and that closest to the telomere is to the right in the histograms for the long (A) and short (B) arms of the seven Ae. tauschii chromosomes (horizontal axis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-166556552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of QTL along the &lt;em&gt;Ae. tauschii&lt;/em&gt; chromosome arms. The numbers of QTL (vertical axis) are expressed per bin each containing 20% genes in the arm. The bin closest to the centromere is to the left and that closest to the telomere is to the right in the histograms for the long (A) and short (B) arms of the seven &lt;em&gt;Ae. tauschii&lt;/em&gt; chromosomes (horizontal axis).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/841/F1.medium.gif" width="440" height="93"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/841/F1.medium.gif" width="440" height="93"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/841/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/841/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492527" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-53" class="first-child">Distribution of QTL along the <em>Ae. tauschii</em> chromosome arms. The numbers of QTL (vertical axis) are expressed per bin each containing 20% genes in the arm. The bin closest to the centromere is to the left and that closest to the telomere is to the right in the histograms for the long (A) and short (B) arms of the seven <em>Ae. tauschii</em> chromosomes (horizontal axis).</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">The null hypothesis that each gene in the genome had an equal chance to be mapped as a QTL was formally tested by applying a <em>χ<sup>2</sup></em> test for the five bins. This gave <em>P</em> = 9 × 10<sup>−6</sup> for the short arm and <em>P</em> = 1 × 10<sup>−4</sup> for the long arm. Based on these low <em>p</em>-values, we concluded that genes along chromosome arms differ in the likelihood to be mapped as a QTL.</p><p id="p-55">Since the average frequency of QTL appeared to increase along chromosome arms (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), we investigated whether the probability of a gene to be mapped as a QTL was correlated with recombination rate in its vicinity. The linear correlation of the number of QTL per bin and mean recombination rate in the bin was computed (<em>r</em> = 0.28, <em>P</em> = 3 × 10<sup>−4</sup>, <em>N</em> = 158). We concluded that the probability for a gene to be mapped as a QTL was related to the recombination rate in its vicinity. However, the numbers of QTL declined (long arms) or plateaued (short arms) in the most distal bins while recombination rates increased. The distributions of QTL in the last bin was similar for QTL mapped with SNP markers and SSR markers (Figure S2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/841/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between frequencies of projected QTL, recombination rates, and location on chromosome arms. HC genes annotated in each Ae. tauschii chromosome arm were allocated into five bins, each containing 20% of the genes. The bins were arranged along the centromere-telomere axes. The centromere is to the left and the telomere is to the right. The numbers of QTL projected into each of the five bins were averaged across the seven chromosome arms (Figure 1) and plotted (blue lines). Recombination rates in cM/Mb in each of the five bins were averaged across the seven long arms (A) and seven short arms (B) and plotted (red lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-166556552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between frequencies of projected QTL, recombination rates, and location on chromosome arms. HC genes annotated in each &lt;em&gt;Ae. tauschii&lt;/em&gt; chromosome arm were allocated into five bins, each containing 20% of the genes. The bins were arranged along the centromere-telomere axes. The centromere is to the left and the telomere is to the right. The numbers of QTL projected into each of the five bins were averaged across the seven chromosome arms (Figure 1) and plotted (blue lines). Recombination rates in cM/Mb in each of the five bins were averaged across the seven long arms (A) and seven short arms (B) and plotted (red lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/841/F2.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/841/F2.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/841/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/841/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-56" class="first-child">Relationship between frequencies of projected QTL, recombination rates, and location on chromosome arms. HC genes annotated in each <em>Ae. tauschii</em> chromosome arm were allocated into five bins, each containing 20% of the genes. The bins were arranged along the centromere-telomere axes. The centromere is to the left and the telomere is to the right. The numbers of QTL projected into each of the five bins were averaged across the seven chromosome arms (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) and plotted (blue lines). Recombination rates in cM/Mb in each of the five bins were averaged across the seven long arms (A) and seven short arms (B) and plotted (red lines).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Meta-QTL hypothesis testing</h3><p id="p-57">Standard deviation estimates <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-52.gif"/></noscript></span></span> (in Mb) were obtained for eight clusters of QTL with their known causative Mendelian loci projected onto the <em>Ae. tauschii</em> genome sequence (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The clusters were located on five of the seven <em>Ae. tauschii</em> pseudomolecules and all involved QTL originally mapped on two or all three wheat homeologous chromosomes (for descriptions of the eight Mendelian loci and QTL see Methods). Several clusters involved QTL mapped in wild relatives of wheat. The estimates of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-53.gif"/></noscript></span></span> greatly varied, ranging from 0.6 Mb to 43 Mb among the eight clusters (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492501/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492501/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>Clusters of QTL used for estimating standard deviation</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-59"> The local density of genes and SSR markers along the <em>Ae. tauschii</em> chromosome arms (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>) and the local density of markers on genetic maps in Triticeae correlate with recombination rate along chromosome arms. We therefore hypothesized that the sizes of deviations of independent attempts to map a QTL with different markers on different maps would be larger for QTL proximally located than for those distally located and would be related to the local recombination rate. The relationship between <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-55.gif"/></noscript></span></span> and Rt was fitted by an exponential relationship (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), for which the log response was linearly related to the predictor (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The variable <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-56.gif"/></noscript></span></span> was related to Rt by the linear regression equation<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-61.gif"/></noscript></span></span> The intercept α and slope β were estimated as <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-62.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-63.gif"/></noscript></span></span> (<em>P</em> = 0.006, one-sided <em>t</em>-test) for the regression effect, rejecting the null hypothesis of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-64.gif"/></noscript></span></span> for the regression line (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>) in favor of a negative slope of the regression line and a decaying exponential relationship of the standard estimation with Rt, which confirmed the above-mentioned relationship. These parameter fits and Rt along the <em>Ae. tauschii</em> chromosomes can be used to compute <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-65.gif"/></noscript></span></span> adjusted for recombination rate and the estimates can then be used for conducting the first three hypothesis tests described in Methods.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/841/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between recombination rate (Rt) and the size of standard deviation (). (A) The standard deviation of independently mapped QTL for a Mendelian locus () in Mb is on the vertical axis and the mean meiotic recombination rates (Rt) in cM/Mb in the vicinity of the projected QTL within a cluster are on the horizontal axis. The size of  exponentially decreases as Rt increases. (B) The relationship between log() and Rt is linear." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-166556552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between recombination rate (Rt) and the size of standard deviation (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). (A) The standard deviation of independently mapped QTL for a Mendelian locus (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-58&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) in Mb is on the vertical axis and the mean meiotic recombination rates (Rt) in cM/Mb in the vicinity of the projected QTL within a cluster are on the horizontal axis. The size of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-59&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; exponentially decreases as Rt increases. (B) The relationship between log(&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-60&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) and Rt is linear.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/841/F3.medium.gif" width="440" height="122"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/841/F3.medium.gif" width="440" height="122"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/841/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/841/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-60" class="first-child">Relationship between recombination rate (Rt) and the size of standard deviation (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-57.gif"/></noscript></span></span>). (A) The standard deviation of independently mapped QTL for a Mendelian locus (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-58.gif"/></noscript></span></span>) in Mb is on the vertical axis and the mean meiotic recombination rates (Rt) in cM/Mb in the vicinity of the projected QTL within a cluster are on the horizontal axis. The size of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-59.gif"/></noscript></span></span> exponentially decreases as Rt increases. (B) The relationship between log(<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/F3/embed/mml-math-60.gif"/></noscript></span></span>) and Rt is linear.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">To assess the effect of marker type on the sizes of standard deviations estimated by the statistical model, a <em>t</em>-test based on adjusted distances of QTL mapped by SSR and SNP markers was performed for the null hypothesis <em>H<sub>0</sub></em>: adjusted deviations of QTL mapped by two different marker types are the same. The mean adjusted distance between QTL mapped with SNP markers was 1.68 Mb and that of QTL mapped with SSR markers was 1.54 Mb. The test yielded <em>P</em> = 0.846. We therefore could not reject the null hypothesis and concluded that there was no evidence that our model would not be applicable to QTL mapped by either type of markers.</p></div><div id="sec-15" class="subsection"><h3>Web portal for hypothesis testing</h3><p id="p-62">A web portal (<a href="https://tools.omicsspeaks.com/consmap">https://tools.omicsspeaks.com/consmap</a>) was constructed to facilitate the computation of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-66.gif"/></noscript></span></span> adjusted for recombination rate and <em>p</em>-values for the three hypotheses about QTL projected onto the <em>Ae. tauschii</em> genome sequence. The portal contains tabs, each of which implements one of the hypothesis tests described in Methods. The tab “Test1: QTL vs QTL” was designed for testing the null hypothesis that two QTL are independent estimates of a single locus. The tab “Test2: QTL vs Mendelian locus” was designed for testing the null hypothesis that a QTL is an estimate of variation in QTL mapping of a specific locus annotated on or projected onto the Aet v4.0 sequence. The tab “Test3: QTL vs cluster mean” was designed to test the null hypothesis that a projected QTL is a member of an existing QTL cluster. QTL cluster should be based on biological evidence that the QTL are independent estimates of the same locus, albeit of unspecified location on an Aet v4.0 pseudomolecule. Each <em>Ae. tauschii</em> chromosome has a slightly different distribution of recombination rates, which necessitates inputting the respective <em>Ae. tauschii</em> chromosome. Next, the user should input the pseudomolecule coordinates of the projected query QTL. Finally, the user should input the pseudomolecule coordinates of the projected target QTL for Test 1, a coordinate for the Mendelian locus for Test 2, and the coordinates of all QTL in the cluster for Test 3. A <em>p</em>-value for the requested test is then computed.</p></div><div id="sec-16" class="subsection"><h3>A case study</h3><p id="p-63">We used QTL for resistance to <em>Fusarium</em> head bight (FHB), also known as scab, to illustrate the utility of projecting QTL on the <em>Ae. tauschii</em> genome sequence and the use of our Web portal for meta-QTL hypothesis testing. A total of 64 QTL for FHB resistance was projected onto the <em>Ae. tauschii</em> pseudomolecules (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Tables 3</a> and S3). A QTL mapped on 7E chromosome of <em>Thinopyrum ponticum</em> was not considered. The largest number of them were on chromosomes in the homeologous group 3. In total, 3, 16, and 2 QTL were on chromosomes 3A, 3B, and 3D, respectively. The disproportionally greater number of QTL on 3B was due to numerous attempts to map the <em>Fhb1</em> gene, which provides the most effective resistance to FHB in wheat. The gene is in the distal region of 3BS (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Waldron <em>et al.</em> 1999</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492499/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492499/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><strong>Minimum number of genes affecting <em>Fusarium</em> head blight resistance in wheat and projected onto the <em>Ae. tauschii</em> genome sequence</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-64">A meta-QTL analysis of the 21 QTL on chromosomes 3A, 3B, and 3D was conducted to assess the minimum number of genes on these three chromosomes and to find if some QTL are duplicated on homeologous chromosomes. Using the QTL <em>vs.</em> QTL hypothesis testing option in the portal, <em>p</em>-values between neighboring QTL were computed (Tests 1, Methods). The resulting <em>p</em>-values were above 0.05, except for intervals Q3088-Q3427 and Q3505-Q3695 for which the <em>p</em>-values were &lt;0.0001 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Based on these <em>p</em>-values we rejected the hypothesis that all 21 QTL form a single cluster, leading to the conclusion that there are at least three different genes among wheat chromosomes 3A, 3B and 3D affecting FHB resistance (Test 4, Methods). We should emphasize that failing to reject the null hypothesis <em>H<sub>0</sub></em> for a group of neighboring QTL with <em>p</em>-values &gt; 0.05 would not warrant the statement that these QTL form a cluster equivalent to a single Mendelian gene, as this would require an additional equivalence test in order to accept the null hypothesis with confidence. Hence, only the minimal but not the exact number of FHB genes needed to account for the QTL clusters can be determined.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/841/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tests of the null hypothesis that neighboring FHB QTL correspond to the same gene. The tests involved 21 QTL for various phenotypes of resistance to Fusarium head blight (FHB) mapped on wheat chromosomes 3A, 3B, and 3D and individually projected onto pseudomolecule 3D. The QTL vs. QTL option implemented in the Web portal was used to compute the p-values for testing whether each pair of neighboring QTL were from the same cluster. The approximate projections of QTL (green ovals) on pseudomolecule 3D are indicated on the horizontal axis. Computed p-values (blue vertical columns) are on the vertical axis. The null hypothesis was rejected for two pairs of neighboring QTL (red arrows): Q3088-Q3427 and Q3505-Q3695 (Table S3) but could not be rejected for the remaining intervals between neighboring QTL. The location of PFT, the causative gene for Fhb1 (Rawat et al. 2016) is indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-166556552" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Tests of the null hypothesis that neighboring FHB QTL correspond to the same gene. The tests involved 21 QTL for various phenotypes of resistance to &lt;em&gt;Fusarium&lt;/em&gt; head blight (FHB) mapped on wheat chromosomes 3A, 3B, and 3D and individually projected onto pseudomolecule 3D. The QTL &lt;em&gt;vs.&lt;/em&gt; QTL option implemented in the Web portal was used to compute the &lt;em&gt;p&lt;/em&gt;-values for testing whether each pair of neighboring QTL were from the same cluster. The approximate projections of QTL (green ovals) on pseudomolecule 3D are indicated on the horizontal axis. Computed &lt;em&gt;p&lt;/em&gt;-values (blue vertical columns) are on the vertical axis. The null hypothesis was rejected for two pairs of neighboring QTL (red arrows): Q3088-Q3427 and Q3505-Q3695 (Table S3) but could not be rejected for the remaining intervals between neighboring QTL. The location of &lt;em&gt;PFT&lt;/em&gt;, the causative gene for &lt;em&gt;Fhb1&lt;/em&gt; (Rawat &lt;em&gt;et al.&lt;/em&gt; 2016) is indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/841/F4.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/3/841/F4.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/841/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/841/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-65" class="first-child">Tests of the null hypothesis that neighboring FHB QTL correspond to the same gene. The tests involved 21 QTL for various phenotypes of resistance to <em>Fusarium</em> head blight (FHB) mapped on wheat chromosomes 3A, 3B, and 3D and individually projected onto pseudomolecule 3D. The QTL <em>vs.</em> QTL option implemented in the Web portal was used to compute the <em>p</em>-values for testing whether each pair of neighboring QTL were from the same cluster. The approximate projections of QTL (green ovals) on pseudomolecule 3D are indicated on the horizontal axis. Computed <em>p</em>-values (blue vertical columns) are on the vertical axis. The null hypothesis was rejected for two pairs of neighboring QTL (red arrows): Q3088-Q3427 and Q3505-Q3695 (Table S3) but could not be rejected for the remaining intervals between neighboring QTL. The location of <em>PFT</em>, the causative gene for <em>Fhb1</em> (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Rawat <em>et al.</em> 2016</a>) is indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">We then used Test 4 (Methods) to estimate the minimum number of genes underpinning the set of 64 FHB QTL. We estimated that a minimum of 18 different genes for FHB have been detected in wheat (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-67"><em>Fhb1</em> locus was isolated and shown to the equivalent to gene <em>PFT</em> encoding a wheat agglutinin protein (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Rawat <em>et al.</em> 2016</a>). Blast with the <em>PFT</em> gene (AOZ21512.1) against the AL8/78 genome sequence detected a locus on 2D at 634,015,182 to 634,016,319 bp (annotated as AET2Gv21236400 (<a href="http://aegilops.wheat.ucdavis.edu/ATGSP/index.php">http://aegilops.wheat.ucdavis.edu/ATGSP/index.php</a>). The gene was 88% identical with <em>PFT</em> indicating that it was a paralogue of <em>PFT</em>. No locus was detected on 3D. To predict an approximate location of the <em>PFT</em> in the Aet v4.0 genome sequence, we projected gene AOZ21513.1, a neighbor of <em>PFT</em> on 3B (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Rawat <em>et al.</em> 2016</a>), onto the <em>Ae. tauschii</em> genome sequence. The gene was located at 7,382,740 bp on the 3D pseudomolecule.</p><p id="p-68">Since no gene corresponding to <em>PFT</em> was on 3D, it was unexpected to find QTL QFhs.fal-3DS reported on 3DS (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Paillard <em>et al.</em> 2004</a>). The 3D QTL (Q3014) was projected at 4,887,606 bp. To test the hypothesis that QFhs.fal-3DS is a QTL of a <em>PFT</em> ortholog on wheat 3D, the “QTL vs Mendelian gene” option in the Web portal (Test 2, Methods) was used (<em>P =</em> 0.53). Based on this <em>p</em> value we could not reject the hull hypothesis. It is therefore possible that chromosome 3D in the Arina × Forno RIL population (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Paillard <em>et al.</em> 2004</a>) segregated for an ortholog of <em>PFT</em>.</p><p id="p-69">Another biological question that emerged from the meta-QTL analysis was whether QFhs.nau-2DL (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jiang <em>et al.</em> 2007</a>) projected on 2D at 522,697,513 could be a QTL of the AET2Gv21236400 locus on 2D. The QTL <em>vs.</em> Mendelian gene option in the portal (Test 2, Methods) yielded <em>P</em> &lt; 0.0001. We therefore rejected the null hypothesis of equivalence of the FHB resistance QTL QFhs.nau-2DL on 2D and the expression of the AET2Gv21236400 agglutinin gene.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Markers</h3><p id="p-70">Of the 6,385 random wheat SSR markers downloaded from GrainGenes, only 30.6% were successfully projected onto the <em>Ae. tauschii</em> genome sequence. This success rate was lower than the 83.6% obtained with projecting A- and B-subgenome SNP markers mapped in the wild emmer onto the <em>Ae. tauschii</em> genome sequence (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Jorgensen <em>et al.</em> 2017</a>). A likely reason for this great difference was subgenome specificity, which was emphasized in the design of SSR markers (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Röder <em>et al.</em> 1998</a>) but not in the design of SNP markers for Illumina assays (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akhunov <em>et al.</em> 2009</a>).</p><p id="p-71">Projecting QTL mapped with SSR markers was more successful, about 66.2%, which was primarily due to that it was sufficient to project only one of two SSR markers that were used to map a QTL on a genetic map. A marker that failed to be projected was often replaced by another SSR marker mapped in its vicinity, provided that the alternative marker survived the validation steps described in Methods.</p><p id="p-72">The greater success rate of projecting QTL mapped with SNP markers, 92.2%, than with SSR markers indicated that meta-analyses and discovery of candidate genes will likely be more successful for QTL mapped with SNP markers than for those mapped with SSR markers. If possible, it is therefore preferable to use SNP markers rather than SSR markers in QTL discovery and mapping in wheat and its relatives.</p></div><div id="sec-19" class="subsection"><h3>QTL distribution along chromosomes</h3><p id="p-73">A total of 585 of the 774 QTL accessed in the literature were projected onto the <em>Ae. tauschii</em> pseudomolecules. The projections of this large number of QTL onto a single genomic reference made it possible to test the hypothesis that the QTL distribution was the same as the distribution of genes along the centromere-telomere axes of wheat chromosome arms. This hypothesis was rejected with very low <em>p</em> values for both short and long arms. Compared to gene distribution, QTL distribution increased in the centromere-to-telomere direction, peaking in the third or fourth bin and then plateaued or declined in the fifth bin. To account for this pattern, we considered the possibility that QTL locations were often equated with midpoints between flanking markers, which may have shifted the locations of QTL away from the telomeric ends. We hypothesized that if this were true, QTL mapped with SNP markers would show more even distribution among the five bins than QTL mapped with SSR markers because SNP maps are usually more marker dense than SSR maps, and the distances between markers flanking LOD maxima are often shorter on SNP maps than on SSR maps. However, our comparisons of the distribution of SNP-mapped QTL with the distribution of SSR-mapped QTL showed no meaningful difference.</p><p id="p-74">Except for the anomalous distribution of QTL in the distal portions of chromosome arms, the distribution of QTL along the rest of the chromosome arms mirrored recombination rates along chromosome arms and correlated with them. This is perhaps not surprising since QTL are a form of genetic diversity in the genome and genetic diversity correlates with recombination rate in wheat and its relatives (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dvorak <em>et al.</em> 1998</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akhunov <em>et al.</em> 2010</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wang <em>et al.</em> 2013</a>). Furthermore, gene duplications and deletion, which contribute to the evolution of genetic novelty in wheat and its diploid relatives also correlate with recombination rates (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dvorak and Akhunov 2005</a>). Disease resistance genes, which represented a large portion of QTL in our study, are preferentially located in high-recombination regions in the <em>Ae. tauschii</em> genome (<a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>) and likely also in the wheat genome.</p></div><div id="sec-20" class="subsection"><h3>Meta-QTL analysis</h3><p id="p-75">Standard deviations of QTL projections on the <em>Ae. tauschii</em> genome sequence varied from 0.6 to 43 Mb among the eight QTL clusters in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, which was mostly caused by variation in recombination rates along chromosomes. There may be several reasons for the relationship between <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-67.gif"/></noscript></span></span> and recombination rate. Gene density is highest in distal, high-recombination regions of wheat and <em>Aegilops</em> chromosomes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akhunov <em>et al.</em> 2003</a>; <a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>). The density of markers within or near genes, the preferred locations of most of the wheat SSR and SNP markers, will therefore also be highest in the distal, high-recombination regions. Physical distances between neighboring markers on a genetic map are therefore shorter in the distal, high-recombination chromosome regions than in the proximal, low-recombination chromosome regions. The deviations of projected QTL from the mean QTL location on the pseudomolecule will therefore be smaller in high-recombination regions than in low-recombination regions and the size of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-68.gif"/></noscript></span></span> of QTL mapping and projection will be related to local recombination rate. The same relationship is also expected due to the distortion of genetic maps caused by preference of crossovers for distal chromosome regions over the proximal chromosome regions in wheat and its relatives.</p><p id="p-76">Based on the exponential decay relationship between <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-69.gif"/></noscript></span></span> and recombination rate, we developed a model for adjusting <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-70.gif"/></noscript></span></span> for local recombination rate and deployed it in a portal with a graphical user interfase (GUI) for testing hypotheses about a QTL projected onto an <em>Ae. tauschii</em> pseudomolecule. The model and its application assume that no other factor affects the sizes of deviations among independently mapped and projected QTL. For example, we made no distinction between QTL mapped with SSR and SNP markers in the model. If <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-71.gif"/></noscript></span></span> were actually smaller for QTL mapped with SNP markers than for those mapped with SSR markers, the model would overestimate the <em>p</em>-value for QTL mapped with SNP markers and underestimate that for QTL mapped with SSR markers. Our test of this hypothesis failed to find a difference between QTL mapped with SSRs and SNPs. However, that does not mean that this will continue to be true. Improvements in mapping technology may necessitate re-evaluation of the model and adapting it to the increased precision of QTL mapping. Another factor that the model does not take into account is the size of the sample of gametes that were employed in QTL mapping. If sample size and type of mapping population, <em>e.g.</em>, F<sub>2</sub> compared to recombinant inbred lines, would systematically affect the size of deviations, the model would again be unrealistic. As more data become available, it will be possible to evaluate the various variables affecting the precision of QTL mapping and modify the model accordingly.</p><p id="p-77">To facilitate meta-analyses of QTL projected onto the <em>Ae. tauschii</em> genome sequence, we constructed a portal for testing hypotheses about a projected query QTL, namely, whether it may be an estimate of another projected QTL (Test 1), whether it may be an estimate of a specific Mendelian gene (Test 2), or whether it may belong to a QTL cluster (Test 3). The test of whether a set of QTL belong to the same cluster (Test 4) can be constructed from consecutive applications of Test 1 and requires no multiple-test adjustments as demonstrated by a simulation (Supplemental File S2). The rejection of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-72.gif"/></noscript></span></span> in Test 4 provides statistical evidence for the existence of multiple clusters but not the number of clusters. It should also be emphasized that the failure to reject the null hypothesis for a group of tests of neighboring QTL should not be interpreted as evidence that all such neighboring QTL belong to a single locus. Thus, the 13 QTL surrounding the <em>Fhb1</em> locus and projected onto the tip of the short arm of the 3D pseudomolecule spanning an interval 4,887,606 to 50,957,578 bp in our case study may not all be QTL of <em>Fhb1</em>, even though all of the tests of neighboring QTL failed to reject the null hypothesis.</p><p id="p-78">In our analyses of the eight QTL clusters used for estimating <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/841/embed/mml-math-73.gif"/></noscript></span></span> and the FHB QTL in the case study, we conducted meta-analyses only with QTL for related traits. The tools we developed and made available here could also be used for meta-analyses between collocated but superficially unrelated traits. Such QTL collocations may arise from pleiotropy or tight linkage of causal genes. The former was suggested to account for the collocation of QTL for several traits with a major QTL for spike shattering on the long arm of wild emmer chromosome 2A (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Peng <em>et al.</em> 2003</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Peng <em>et al.</em> 2013</a>). Similar collocated QTL for superficially unrelated morphological traits were encountered on other wheat chromosomes and also considered to be pleiotropy (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Peng <em>et al.</em> 2003</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Peng <em>et al.</em> 2013</a>). Other studies of the genetic bases of wheat domestication added additional QTL to the collocated QTL reported by Peng <em>et al.</em> (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Tzarfati <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thanh <em>et al.</em> 2013</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Peleg <em>et al.</em> 2011</a>). Projecting QTL on the <em>Ae. tauschii</em> genome sequences or wild emmer genome sequence now available (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Avni <em>et al.</em> 2017</a>) will greatly improve the use of high-resolution genetic mapping and facilitates reverse genetic approaches to decide whether pleiotropy or tight linkage is responsible for specific cases of QTL collocation in wild emmer × domesticated wheat and similar QTL collocations in other wheat QTL mapping populations.</p><p id="p-79">The database of projected QTL on the <em>Ae. tauschii</em> genome sequence is an open-ended resource to which new QTL can be easily added using BLASTN of marker sequences against the <em>Ae. tauschii</em> Aet v4.0 genome sequence (<a href="http://aegilops.wheat.ucdavis.edu/ATGSP/blast.php">http://aegilops.wheat.ucdavis.edu/ATGSP/blast.php</a>) to find pseudomolecule coordinates for new QTL. The coordinates can then be used to compare new QTL with already discovered QTL using the portal or search for genes identified in other grass genomes using collinearity databases for the <em>Ae. tauschii</em> genome (<a id="xref-ref-45-9" class="xref-bibr" href="#ref-45">Luo <em>et al.</em> 2017</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dvorak <em>et al.</em> 2018a</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-80">This material is based upon work supported by the US National Science Foundation under Grant No. IOS-1238231, US Department of Agriculture A, 58-2090-8-071-0, and the USDA NIFA Hatch Program 1002302.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-81">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7607060">https://doi.org/10.25387/g3.7607060</a>.</p></li><li class="fn" id="fn-3"><p id="p-82"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-30"><span class="received-label">Received </span>November 30, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-12"><span class="accepted-label">Accepted </span>January 12, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Xu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.1" data-doi="10.1007/s00122-009-1059-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">517</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1059-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAkhunov%26rft.auinit1%253DE.%26rft.volume%253D119%26rft.issue%253D3%26rft.spage%253D507%26rft.epage%253D517%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bgenotyping%2Bin%2Bpolyploid%2Bwheat%2Bwith%2Bthe%2BIllumina%2BGoldenGate%2Bassay.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1059-5%26rft_id%253Dinfo%253Apmid%252F19449174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1059-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19449174&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268329400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.2" data-doi="10.1186/1471-2164-11-702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunova</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleotide diversity maps reveal variation in diversity among wheat genomes and chromosomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">702</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAkhunov%26rft.auinit1%253DE.%2BD.%26rft.volume%253D11%26rft.spage%253D702%26rft.epage%253D702%26rft.atitle%253DNucleotide%2Bdiversity%2Bmaps%2Breveal%2Bvariation%2Bin%2Bdiversity%2Bamong%2Bwheat%2Bgenomes%2Band%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-702%26rft_id%253Dinfo%253Apmid%252F21156062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21156062&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.3" data-doi="10.1101/gr.808603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodyear</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echalier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The organization and rate of evolution of wheat genomes are correlated with recombination rates along chromosome arms.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.808603</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.808603%26rft_id%253Dinfo%253Apmid%252F12695326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzUvNzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.4" data-doi="10.1111/pbi.12635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winfield</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burridge</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Downie</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benbow</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Characterization of a Wheat Breeders’ Array suitable for high-throughput SNP genotyping of global accessions of hexaploid bread wheat (<em>Triticum aestivum</em>).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.5" data-doi="10.1007/s00122-005-0190-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arbelbide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mixed-model QTL mapping for kernel hardness and dough strength in bread wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-005-0190-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DArbelbide%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D5%26rft.spage%253D885%26rft.epage%253D890%26rft.atitle%253DMixed-model%2BQTL%2Bmapping%2Bfor%2Bkernel%2Bhardness%2Band%2Bdough%2Bstrength%2Bin%2Bbread%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0190-1%26rft_id%253Dinfo%253Apmid%252F16402188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0190-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16402188&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.6" data-doi="10.1126/science.aan0032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avni</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nave</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barad</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baruch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twardziok</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Wild emmer genome architecture and diversity elucidate wheat evolution and domestication.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">357</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aan0032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAvni%26rft.auinit1%253DR.%26rft.volume%253D357%26rft.issue%253D6346%26rft.spage%253D93%26rft.epage%253D97%26rft.atitle%253DWild%2Bemmer%2Bgenome%2Barchitecture%2Band%2Bdiversity%2Belucidate%2Bwheat%2Bevolution%2Band%2Bdomestication%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan0032%26rft_id%253Dinfo%253Apmid%252F28684525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTcvNjM0Ni85MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.7" data-doi="10.1007/s00122-007-0603-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snape</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A pseudo-response regulator is misexpressed in the photoperiod insensitive Ppd-D1a mutant of wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0603-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBeales%26rft.auinit1%253DJ.%26rft.volume%253D115%26rft.issue%253D5%26rft.spage%253D721%26rft.epage%253D733%26rft.atitle%253DA%2Bpseudo-response%2Bregulator%2Bis%2Bmisexpressed%2Bin%2Bthe%2Bphotoperiod%2Binsensitive%2BPpd-D1a%2Bmutant%2Bof%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0603-4%26rft_id%253Dinfo%253Apmid%252F17634915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0603-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634915&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249019700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.8" data-doi="10.1007/BF00015721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Börner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plaschke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Worland</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The relationships between the dwarfing genes of wheat and rye.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00015721</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D89%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00015721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00015721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK82100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.9" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1217133110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.10" data-doi="10.1534/genetics.118.301198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Birth and Death of LTR Retrotransposons in <em>Aegilops tauschii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1051</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301198%26rft_id%253Dinfo%253Apmid%252F30158124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMC8zLzEwMzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.11" data-doi="10.1007/BF00220890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinoy</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00220890</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D282%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220890%26rft_id%253Dinfo%253Apmid%252F24169776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24169776&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RL40300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Restriction fragment length polymorphism and divergence in the genomic regions of high and low recombination in self-fertilizing and cross-fertilizing <em>Aegilops</em> species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDvorak%26rft.auinit1%253DJ.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D423%26rft.epage%253D434%26rft.atitle%253DRestriction%2BFragment%2BLength%2BPolymorphism%2Band%2BDivergence%2Bin%2Bthe%2BGenomic%2BRegions%2Bof%2BHigh%2Band%2BLow%2BRecombination%2Bin%2BSelf-Fertilizing%2Band%2BCross-Fertilizing%2BAegilops%2BSpecies%26rft_id%253Dinfo%253Apmid%252F9475752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNDIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.13" data-doi="10.1139/g93-004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terlizzi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resta</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The evolution of polyploid wheats: identification of the A genome donor species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g93-004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DDvorak%26rft.auinit1%253DJ.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D31%26rft.atitle%253DThe%2Bevolution%2Bof%2Bpolyploid%2Bwheats%253A%2Bidentification%2Bof%2Bthe%2BA%2Bgenome%2Bdonor%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg93-004%26rft_id%253Dinfo%253Apmid%252F18469969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g93-004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18469969&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.14" data-doi="10.1534/genetics.105.041632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Tempos of gene locus deletions and duplications and their relationship to recombination rate during diploid and polyploid evolution in the <em>Aegilops</em>-<em>Triticum</em> alliance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.041632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041632%26rft_id%253Dinfo%253Apmid%252F15996988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzEvMzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.15" data-doi="10.1111/tpj.13964"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deal</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Structural variation and rates of genome evolution in the grass family seen through comparison of sequences of genomes greatly differing in size.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13964</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D95%26rft.spage%253D487%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13964&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.16" data-doi="10.1007/s00122-018-3165-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Reassessment of the evolution of wheat chromosomes 4A, 5A, and 7B.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">2451</span>–<span class="cit-lpage">2462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3165-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D2451%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3165-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3165-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.17" data-doi="10.1073/pnas.87.24.9640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Variation in repeated nucleotide sequences sheds light on the phylogeny of the wheat B and G genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">9640</span>–<span class="cit-lpage">9644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.87.24.9640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDvorak%26rft.auinit1%253DJ.%26rft.volume%253D87%26rft.issue%253D24%26rft.spage%253D9640%26rft.epage%253D9644%26rft.atitle%253DVariation%2Bin%2Brepeated%2Bnucleotide%2Bsequences%2Bsheds%2Blight%2Bon%2Bthe%2Bphylogeny%2Bof%2Bthe%2Bwheat%2BB%2Band%2BG%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.87.24.9640%26rft_id%253Dinfo%253Apmid%252F11607134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODcvMjQvOTY0MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.3.841.18" data-doi="10.1093/nar/13.19.6817"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malpica</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halford</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shewry</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">The nucleotide sequence of a HMW glutenin subunit gene located on chromosome 1A of wheat (Triticum aestivum L.).</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">6817</span>–<span class="cit-lpage">6832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DForde%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D19%26rft.spage%253D6817%26rft.epage%253D6832%26rft.atitle%253DThe%2Bnucleotide%2Bsequence%2Bof%2Ba%2BHMW%2Bglutenin%2Bsubunit%2Bgene%2Blocated%2Bon%2Bchromosome%2B1A%2Bof%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F13.19.6817%26rft_id%253Dinfo%253Apmid%252F2997729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/13.19.6817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2997729&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ASV3100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.19" data-doi="10.1007/s00438-004-1095-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szűcs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helguera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skinner</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Large deletions within the first intron in VRN-1 are associated with spring growth habit in barley and wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-004-1095-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DFu%26rft.auinit1%253DD.%26rft.volume%253D273%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D65%26rft.atitle%253DLarge%2Bdeletions%2Bwithin%2Bthe%2Bfirst%2Bintron%2Bin%2BVRN-1%2Bare%2Bassociated%2Bwith%2Bspring%2Bgrowth%2Bhabit%2Bin%2Bbarley%2Band%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1095-4%26rft_id%253Dinfo%253Apmid%252F15690172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1095-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15690172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228187500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.20" data-doi="10.1093/aob/"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Insensitivity to gibberellin in dwarf wheats.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.aob.a084741</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DGALE%26rft.auinit1%253DM.%2BD.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D729%26rft.epage%253D735%26rft.atitle%253DInsensitivity%2Bto%2BGibberellin%2Bin%2BDwarf%2BWheats%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252F%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.841.21" data-doi="10.1016/B978-0-407-00780-2.50005-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youssefian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Dwarfing genes in wheat</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">35</span> in <span class="cit-source">Progress in plant breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">G. E.</span></span> <span class="cit-publ-name">University Press</span>, <span class="cit-publ-loc">Cambridge</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-407-00780-2.50005-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bplant%2Bbreeding%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-407-00780-2.50005-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-407-00780-2.50005-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.22" data-doi="10.1126/science.282.5389.656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Plant comparative genetics after 10 years.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.282.5389.656</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGale%26rft.auinit1%253DM.%2BD.%26rft.volume%253D282%26rft.issue%253D5389%26rft.spage%253D656%26rft.epage%253D659%26rft.atitle%253DPlant%2BComparative%2BGenetics%2Bafter%2B10%2BYears%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5389.656%26rft_id%253Dinfo%253Apmid%252F9784118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTM4OS82NTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.23" data-doi="10.3389/fpls.2015.01099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasheed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide linkage mapping of QTL for yield components, plant height and yield-related physiological traits in the Chinese wheat cross Zhou 8425B/Chinese Spring.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1099</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.01099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D1099%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.01099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.01099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.24" data-doi="10.1046/j.1365-313X.2003.01764.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garciadeblás</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senn</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bañuelos</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Navarro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sodium transport and HKT transporters: the rice model.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">788</span>–<span class="cit-lpage">801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-313X.2003.01764.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGarciadebl%25C3%25A1s%26rft.auinit1%253DB.%26rft.volume%253D34%26rft.issue%253D6%26rft.spage%253D788%26rft.epage%253D801%26rft.atitle%253DSodium%2Btransport%2Band%2BHKT%2Btransporters%253A%2Bthe%2Brice%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.2003.01764.x%26rft_id%253Dinfo%253Apmid%252F12795699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.2003.01764.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12795699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183443000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Quantitative trait loci: a meta-analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoffinet%26rft.auinit1%253DB.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D463%26rft.epage%253D473%26rft.atitle%253DQuantitative%2BTrait%2BLoci%253A%2BA%2BMeta-analysis%26rft_id%253Dinfo%253Apmid%252F10790417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzEvNDYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.26" data-doi="10.1007/s00122-009-1046-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leverington</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fish</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Meta-QTL analysis of the genetic control of ear emergence in elite European winter wheat germplasm.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1046-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGriffiths%26rft.auinit1%253DS.%26rft.volume%253D119%26rft.issue%253D3%26rft.spage%253D383%26rft.epage%253D395%26rft.atitle%253DMeta-QTL%2Banalysis%2Bof%2Bthe%2Bgenetic%2Bcontrol%2Bof%2Bear%2Bemergence%2Bin%2Belite%2BEuropean%2Bwinter%2Bwheat%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1046-x%26rft_id%253Dinfo%253Apmid%252F19430758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1046-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19430758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268329400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.27" data-doi="10.1007/s00122-004-1799-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanocq</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niarquin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heumez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gouis</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Detection and mapping of QTL for earliness components in a bread wheat recombinant inbred lines population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1799-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHanocq%26rft.auinit1%253DE.%26rft.volume%253D110%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D115%26rft.atitle%253DDetection%2Band%2Bmapping%2Bof%2BQTL%2Bfor%2Bearliness%2Bcomponents%2Bin%2Ba%2Bbread%2Bwheat%2Brecombinant%2Binbred%2Blines%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1799-1%26rft_id%253Dinfo%253Apmid%252F15551039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1799-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15551039&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226629900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.28" data-doi="10.1007/s00122-006-0459-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanocq</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laperche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaminon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lainé</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gouis</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Most significant genome regions involved in the control of earliness traits in bread wheat, as revealed by QTL meta-analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-006-0459-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHanocq%26rft.auinit1%253DE.%26rft.volume%253D114%26rft.issue%253D3%26rft.spage%253D569%26rft.epage%253D584%26rft.atitle%253DMost%2Bsignificant%2Bgenome%2Bregions%2Binvolved%2Bin%2Bthe%2Bcontrol%2Bof%2Bearliness%2Btraits%2Bin%2Bbread%2Bwheat%252C%2Bas%2Brevealed%2Bby%2BQTL%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0459-z%26rft_id%253Dinfo%253Apmid%252F17171391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0459-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17171391&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243691700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.29" data-doi="10.1371/journal.pone.0126794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">See</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping a Large Number of QTL for Durable Resistance to Stripe Rust in Winter Wheat Druchamp Using SSR and SNP Markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0126794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0126794</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0126794%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0126794%26rft_id%253Dinfo%253Apmid%252F25970329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0126794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25970329&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.30" data-doi="10.1007/s00122-002-1179-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cöster</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Advanced backcross QTL analysis for the identification of quantitative trait loci alleles from wild relatives of wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1379</span>–<span class="cit-lpage">1389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-002-1179-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D106%26rft.issue%253D8%26rft.spage%253D1379%26rft.epage%253D1389%26rft.atitle%253DAdvanced%2Bbackcross%2BQTL%2Banalysis%2Bfor%2Bthe%2Bidentification%2Bof%2Bquantitative%2Btrait%2Bloci%2Balleles%2Bfrom%2Bwild%2Brelatives%2Bof%2Bwheat%2B%2528%2BTriticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-1179-7%26rft_id%253Dinfo%253Apmid%252F12750781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-1179-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12750781&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000207051000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.31" data-doi="10.1007/s00122-006-0346-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lycar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radovanovic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular detection of QTLs for agronomic and quality traits in a doubled haploid population derived from two Canadian wheats (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-006-0346-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D113%26rft.issue%253D4%26rft.spage%253D753%26rft.epage%253D766%26rft.atitle%253DMolecular%2Bdetection%2Bof%2BQTLs%2Bfor%2Bagronomic%2Band%2Bquality%2Btraits%2Bin%2Ba%2Bdoubled%2Bhaploid%2Bpopulation%2Bderived%2Bfrom%2Btwo%2BCanadian%2Bwheats%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0346-7%26rft_id%253Dinfo%253Apmid%252F16838135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0346-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16838135&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239301500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.32" data-doi="10.1093/jxb/ern033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spielmeyer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagudah</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munns</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparative mapping of HKT genes in wheat, barley, and rice, key determinants of Na<sup>+</sup> transport, and salt tolerance.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/ern033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fern033%26rft_id%253Dinfo%253Apmid%252F18325922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/ern033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18325922&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254293800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozturk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budak</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mapping QTLs conferring salt tolerance and micronutrient concentrations at seedling stage in wheat.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">15662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D15662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.34" data-doi="10.1038/nature12028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Aegilops tauschii</em> draft genome sequence reveals a gene repertoire for wheat adaptation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">496</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D496%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12028%26rft_id%253Dinfo%253Apmid%252F23535592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23535592&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316993400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.35" data-doi="10.1007/s00122-007-0641-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">QTL analysis of resistance to Fusarium head blight in the novel wheat germplasm CJ 9306. I. Resistance to fungal spread.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0641-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DJiang%26rft.auinit1%253DG.%2BL.%26rft.volume%253D116%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D13%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bresistance%2Bto%2BFusarium%2Bhead%2Bblight%2Bin%2Bthe%2Bnovel%2Bwheat%2Bgermplasm%2BCJ%2B9306.%2BI.%2BResistance%2Bto%2Bfungal%2Bspread.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0641-y%26rft_id%253Dinfo%253Apmid%252F17898987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0641-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17898987&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251154100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.36" data-doi="10.3389/fpls.2017.01798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramasamy</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A high-density genetic map of wild emmer wheat from the Karaca Dağ region provides new evidence on the structure and evolution of wheat chromosomes.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01798</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1798%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.37" data-doi="10.1126/science.1081288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawagashira</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">301</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1081288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKikuchi%26rft.auinit1%253DS.%26rft.volume%253D301%26rft.issue%253D5631%26rft.spage%253D376%26rft.epage%253D379%26rft.atitle%253DCollection%252C%2BMapping%252C%2Band%2BAnnotation%2Bof%2BOver%2B28%252C000%2BcDNA%2BClones%2Bfrom%2Bjaponica%2BRice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1081288%26rft_id%253Dinfo%253Apmid%252F12869764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDEvNTYzMS8zNzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.38" data-doi="10.1007/s00122-007-0600-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dedeck</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Léon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">AB-QTL analysis in winter wheat: I. Synthetic hexaploid wheat (<em>T. turgidum ssp. dicoccoides</em> × <em>T. tauschii</em>) as a source of favourable alleles for milling and baking quality traits.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0600-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKunert%26rft.auinit1%253DA.%26rft.volume%253D115%26rft.issue%253D5%26rft.spage%253D683%26rft.epage%253D695%26rft.atitle%253DAB-QTL%2Banalysis%2Bin%2Bwinter%2Bwheat%253A%2BI.%2BSynthetic%2Bhexaploid%2Bwheat%2B%2528T.%2Bturgidum%2Bssp.%2Bdicoccoides%2Bx%2BT.%2Btauschii%2529%2Bas%2Ba%2Bsource%2Bof%2Bfavourable%2Balleles%2Bfor%2Bmilling%2Band%2Bbaking%2Bquality%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0600-7%26rft_id%253Dinfo%253Apmid%252F17634917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0600-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634917&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Comparative analysis of QTLs affecting plant height and maturity across the Poaceae, in reference to an interspecific sorghum population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLin%26rft.auinit1%253DY.%2BR.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D391%26rft.epage%253D411%26rft.atitle%253DComparative%2Banalysis%2Bof%2BQTLs%2Baffecting%2Bplant%2Bheight%2Band%2Bmaturity%2Bacross%2Bthe%2BPoaceae%252C%2Bin%2Breference%2Bto%2Ban%2Binterspecific%2Bsorghum%2Bpopulation.%26rft_id%253Dinfo%253Apmid%252F8536986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzEvMzkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.40" data-doi="10.1071/FP04111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagudah</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hare</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munns</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A locus for sodium exclusion (Nax1), a trait for salt tolerance, mapped in durum wheat.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/FP04111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D31%26rft.spage%253D1105%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP04111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP04111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225301200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.41" data-doi="10.1093/mp/ssu112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nykiforuk</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The stripe rust resistance gene <em>Yr10</em> encodes an evolutionary-conserved and unique CC-NBS-LRR sequence in wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1740</span>–<span class="cit-lpage">1755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/ssu112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssu112%26rft_id%253Dinfo%253Apmid%252F25336565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssu112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25336565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Löve</span>  <span class="cit-name-given-names">Á.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Conspectus of the Triticeae.</span> <abbr class="cit-jnl-abbrev">Feddes Repert.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFeddes%2BRepert.%26rft.volume%253D95%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.43" data-doi="10.1073/pnas.0908195106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deal</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunova</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">O. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome comparisons reveal a dominant mechanism of chromosome number reduction in grasses and accelerated genome evolution in Triticeae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">15780</span>–<span class="cit-lpage">15785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0908195106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908195106%26rft_id%253Dinfo%253Apmid%252F19717446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM3LzE1NzgwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.44" data-doi="10.1073/pnas.1219082110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deal</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A 4-gigabase physical map unlocks the structure and evolution of the complex genome of <em>Aegilops tauschii</em>, the wheat D-genome progenitor.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7940</span>–<span class="cit-lpage">7945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1219082110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1219082110%26rft_id%253Dinfo%253Apmid%252F23610408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc5NDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.45" data-doi="10.1038/nature24486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twardziok</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome sequence of the progenitor of the wheat D genome <em>Aegilops tauschii</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">551</span>: <span class="cit-fpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature24486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24486%26rft_id%253Dinfo%253Apmid%252F29143815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29143815&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.46" data-doi="10.1111/pbi.12288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milner</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">663</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12288%26rft_id%253Dinfo%253Apmid%252F25424506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25424506&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.47" data-doi="10.1007/s00122-012-1939-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marone</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laidò</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gadaleta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colasuonno</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ficco</span>  <span class="cit-name-given-names">D. B. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-density consensus map of A and B wheat genomes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1638</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1939-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMarone%26rft.auinit1%253DD.%26rft.volume%253D125%26rft.issue%253D8%26rft.spage%253D1619%26rft.epage%253D1638%26rft.atitle%253DA%2Bhigh-density%2Bconsensus%2Bmap%2Bof%2BA%2Band%2BB%2Bwheat%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1939-y%26rft_id%253Dinfo%253Apmid%252F22872151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1939-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22872151&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.48" data-doi="10.1016/j.jcs.2004.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruyama-Funatsuki</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funatsuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabiki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification and characterization of a novel LMW-s glutenin gene of a Canadian western extra-strong wheat.</span> <abbr class="cit-jnl-abbrev">J. Cereal Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jcs.2004.07.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCereal%2BSci.%26rft.volume%253D41%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcs.2004.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcs.2004.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.49" data-doi="10.1007/s001220050052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masci</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovidio</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafiandra</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasarda</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A 1B-coded low-molecular-weight glutenin subunit associated with quality in durum wheats shows strong similarity to a subunit present in some bread wheat cultivars.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D100%26rft.spage%253D396%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.50" data-doi="10.1139/g05-055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCartney</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humphreys</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukow</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ames</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mapping quantitative trait loci controlling agronomic traits in the spring wheat cross RL4452 × ‘AC Domain’.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">883</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g05-055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DMcCartney%26rft.auinit1%253DC.%2BA.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D870%26rft.epage%253D883%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bcontrolling%2Bagronomic%2Btraits%2Bin%2Bthe%2Bspring%2Bwheat%2Bcross%2BRL4452x%2527AC%2BDomain%2527.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg05-055%26rft_id%253Dinfo%253Apmid%252F16391693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g05-055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16391693&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.51" data-doi="10.1093/oxfordjournals.jhered.a105590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McFadden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sears</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1946</span> <span class="cit-article-title">The origin of <em>Triticum spelta</em> and its free-threshing hexaploid relatives.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.jhered.a105590</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DMCFADDEN%26rft.auinit1%253DE.%2BS.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D81%26rft.epage%253D89%26rft.atitle%253DTHE%2BORIGIN%2BOF%2BTRITICUM%2BSPELTA%2BAND%2BITS%2BFREE-THRESHING%2BHEXAPLOID%2BRELATIVES%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.jhered.a105590%26rft_id%253Dinfo%253Apmid%252F20985728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.jhered.a105590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20985728&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1946UX17700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.52" data-doi="10.2135/cropsci1970.0011183X001000050035x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silbaugh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Inheritance of Resistance to Stripe Rust and its Association with Brown Glume Color in <em>Tritieum aestivum</em> L.,‘PI 178383’1.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">568</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1970.0011183X001000050035x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMetzger%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D567%26rft.epage%253D568%26rft.atitle%253DInheritance%2Bof%2BResistance%2Bto%2BStripe%2BRust%2Band%2Bits%2BAssociation%2Bwith%2BBrown%2BGlume%2BColor%2Bin%2BTritieum%2Baestivum%2BL.%252C%2B%2527P.I.%2B178383%2527%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1970.0011183X001000050035x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1970.0011183X001000050035x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970H686700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.53" data-doi="10.1007/s00122-015-2492-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naruoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garland-Campbell</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association mapping for stripe rust (<em>Puccinia striiformis F. sp. tritici</em>) in US Pacific Northwest winter wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2492-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D1083%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2492-2%26rft_id%253Dinfo%253Apmid%252F25754424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2492-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25754424&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.54" data-doi="10.1007/s00122-004-1628-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paillard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnurbusch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messmer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">QTL analysis of resistance to Fusarium head blight in Swiss winter wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1628-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DPaillard%26rft.auinit1%253DS.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D323%26rft.epage%253D332%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bresistance%2Bto%2BFusarium%2Bhead%2Bblight%2Bin%2BSwiss%2Bwinter%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1628-6%26rft_id%253Dinfo%253Apmid%252F15014875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1628-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15014875&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223011800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.55" data-doi="10.1002/jsfa.2740320109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corfield</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlations between the inheritance of certain high‐molecular weight subunits of glutenin and bread‐making quality in progenies of six crosses of bread wheat.</span> <abbr class="cit-jnl-abbrev">J. Sci. Food Agric.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jsfa.2740320109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSci.%2BFood%2BAgric.%26rft.volume%253D32%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjsfa.2740320109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jsfa.2740320109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.56" data-doi="10.1093/jxb/err206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peleg</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korol</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saranga</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of wheat domestication and evolution under domestication.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">5051</span>–<span class="cit-lpage">5061</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/err206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferr206%26rft_id%253Dinfo%253Apmid%252F21778183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/err206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21778183&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295983800028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.57" data-doi="10.1073/pnas.252763199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Domestication quantitative trait loci in <em>Triticum dicoccoides</em>, the progenitor of wheat.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">2489</span>–<span class="cit-lpage">2494</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.252763199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.252763199%26rft_id%253Dinfo%253Apmid%252F12604784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzUvMjQ4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.58" data-doi="10.1556/CRC.2012.0030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevo</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene discovery in <em>Triticum dicoccoides</em>, the direct progenitor of cultivated wheats.</span> <abbr class="cit-jnl-abbrev">Cereal Res. Commun.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1556/CRC.2012.0030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCereal%2BRes.%2BCommun.%26rft.volume%253D41%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1556%252FCRC.2012.0030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1556/CRC.2012.0030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.59" data-doi="10.1104/pp.90.3.854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinthus</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Appleford</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenton</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Effect of temperature on gibberellin (GA) responsiveness and on endogenous GA1 content of tall and dwarf wheat genotypes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">859</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.90.3.854</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DPinthus%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D854%26rft.epage%253D859%26rft.atitle%253DEffect%2Bof%2BTemperature%2Bon%2BGibberellin%2B%2528GA%2529%2BResponsiveness%2Band%2Bon%2BEndogenous%2BGA1%2BContent%2Bof%2BTall%2Band%2BDwarf%2BWheat%2BGenotypes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.90.3.854%26rft_id%253Dinfo%253Apmid%252F16666888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiI5MC8zLzg1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.60" data-doi="10.1016/S0733-5210(88)80001-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pogna</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafiandra</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feillet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autran</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Evidence for a direct causal effect of low molecular weight subunits of glutenins on gluten viscoelasticity in durum wheats.</span> <abbr class="cit-jnl-abbrev">J. Cereal Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0733-5210(88)80001-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCereal%2BSci.%26rft.volume%253D7%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0733-5210%252888%252980001-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0733-5210(88)80001-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.61" data-doi="10.3389/fpls.2017.01843"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quraishi</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pont</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ain</span>  <span class="cit-name-given-names">Q.-U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flores</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burlot</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Combined genomic and genetic data integration of major agronomical traits in bread wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2017.01843</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D1843%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2017.01843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2017.01843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.62" data-doi="10.1038/ng.3706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawat</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pumphrey</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Wheat Fhb1 encodes a chimeric lectin with agglutinin domains and a pore-forming toxin-like domain conferring resistance to Fusarium head blight.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1576</span>–<span class="cit-lpage">1580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3706</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1576%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.63" data-doi="10.1139/g98-009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Röder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The physical mapping of microsatellite markers in wheat.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g98-009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D41%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg98-009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g98-009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.64" data-doi="10.2307/2291267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousseeuw</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croux</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Alternatives to the median absolute deviation.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1993.10476408</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D88%26rft.spage%253D1273%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2291267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MG97300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.841.65" data-doi="10.1007/978-1-4899-6561-5_4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sears</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Nullisomic-tetrasomic combinations in hexaploid wheat</span>, pp. <span class="cit-fpage">29</span>–<span class="cit-lpage">45</span> in <span class="cit-source">Chromosome Manipulations and Plant Genetics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">K. R.</span></span>. <span class="cit-publ-name">Oliver &amp; Boyd</span>, <span class="cit-publ-loc">Edinburgh</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4899-6561-5_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2BManipulations%2Band%2BPlant%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4899-6561-5_4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4899-6561-5_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.66" data-doi="10.1007/s00122-004-1740-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaac</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A high-density microsatellite consensus map for bread wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1740-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSomers%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D1105%26rft.epage%253D1114%26rft.atitle%253DA%2Bhigh-density%2Bmicrosatellite%2Bconsensus%2Bmap%2Bfor%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1740-7%26rft_id%253Dinfo%253Apmid%252F15490101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1740-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15490101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224614100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.67" data-doi="10.1038/ng2014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">M.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.-X.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A QTL for rice grain width and weight encodes a previously unknown RING-type E3 ubiquitin ligase.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSong%26rft.auinit1%253DX.%2BJ.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D623%26rft.epage%253D630%26rft.atitle%253DA%2BQTL%2Bfor%2Brice%2Bgrain%2Bwidth%2Band%2Bweight%2Bencodes%2Ba%2Bpreviously%2Bunknown%2BRING-type%2BE3%2Bubiquitin%2Bligase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2014%26rft_id%253Dinfo%253Apmid%252F17417637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17417637&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245971300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.68" data-doi="10.1111/j.1601-5223.1998.00041.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sourdille</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charmet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trottet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tixier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeuf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Linkage between RFLP molecular markers and the dwarfing genes Rht‐B1 and Rht‐D1 in wheat.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-5223.1998.00041.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D128%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1998.00041.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.1998.00041.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.69" data-doi="10.5504/BBEQ.2013.0008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thanh</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vladutu</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kianian</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thanh</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular genetic analysis of domestication traits in emmer wheat. I: Map construction and QTL analysis using an F<sub>2</sub> population.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Biotechnol. Equip.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3627</span>–<span class="cit-lpage">3637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5504/BBEQ.2013.0008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BBiotechnol.%2BEquip.%26rft.volume%253D27%26rft.spage%253D3627%26rft_id%253Dinfo%253Adoi%252F10.5504%252FBBEQ.2013.0008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5504/BBEQ.2013.0008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.70" data-doi="10.1007/s10681-014-1217-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyagi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mir</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balyan</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Interval mapping and meta-QTL analysis of grain traits in common wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-014-1217-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D201%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-014-1217-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-014-1217-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.71" data-doi="10.1007/s11032-014-0182-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tzarfati</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krugman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel quantitative trait loci underlying major domestication traits in tetraploid wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1613</span>–<span class="cit-lpage">1628</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-014-0182-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D1613%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0182-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0182-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.72" data-doi="10.1186/1471-2105-8-49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veyrieras</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goffinet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MetaQTL: A package of new computational methods for the meta-analysis of QTL mapping experiments.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-8-49</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVeyrieras%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D8%26rft.spage%253D49%26rft.epage%253D49%26rft.atitle%253DMetaQTL%253A%2Ba%2Bpackage%2Bof%2Bnew%2Bcomputational%2Bmethods%2Bfor%2Bthe%2Bmeta-analysis%2Bof%2BQTL%2Bmapping%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-49%26rft_id%253Dinfo%253Apmid%252F17288608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17288608&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.73" data-doi="10.2135/cropsci1999.0011183X003900030032x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waldron</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Sevilla</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frohberg</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">RFLP mapping of QTL for fusarium head blight resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">805</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1999.0011183X003900030032x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWaldron%26rft.auinit1%253DB.%2BL.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D805%26rft.epage%253D811%26rft.atitle%253DRFLP%2BMapping%2Bof%2BQTL%2Bfor%2BFusarium%2BHead%2BBlight%2BResistance%2Bin%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.0011183X003900030032x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.0011183X003900030032x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000080943700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.74" data-doi="10.1111/nph.12164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Aegilops tauschii</em> single nucleotide polymorphisms shed light on the origins of wheat D‐genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">937</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D198%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12164%26rft_id%253Dinfo%253Apmid%252F23374069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.75" data-doi="10.1111/pbi.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of polyploid wheat genomic diversity using a high-density 90 000 single nucleotide polymorphism array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12183%26rft_id%253Dinfo%253Apmid%252F24646323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646323&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.76" data-doi="10.1007/s00122-008-0898-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Photoperiod insensitive Ppd-A1a mutations in tetraploid wheat (<em>Triticum durum</em> Desf.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-008-0898-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWilhelm%26rft.auinit1%253DE.%2BP.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D285%26rft.epage%253D294%26rft.atitle%253DPhotoperiod%2Binsensitive%2BPpd-A1a%2Bmutations%2Bin%2Btetraploid%2Bwheat%2B%2528Triticum%2Bdurum%2BDesf.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0898-9%26rft_id%253Dinfo%253Apmid%252F18839130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0898-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18839130&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261656800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.841.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Worland</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic basis of worldwide wheat varietal improvement</span>, pp. <span class="cit-fpage">59</span>–<span class="cit-lpage">100</span> in <span class="cit-source">The world wheat book: A history of wheat breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bonjean</span>  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Angus</span>  <span class="cit-name-given-names">W. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Lavoisier</span>, <span class="cit-publ-loc">Paris</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-density genetic linkage map construction and QTL mapping of grain shape and size in the wheat population Yanda1817 × Beinong6.</span> <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0118144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0118144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.79" data-doi="10.1007/s10681-005-9017-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaner</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A QTL for early heading in wheat cultivar Suwon 92.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-005-9017-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D146%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-9017-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-9017-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.80" data-doi="10.1073/pnas.0937399100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loukoianov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tranquilli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helguera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Positional cloning of the wheat vernalization gene VRN1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">6263</span>–<span class="cit-lpage">6268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0937399100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0937399100%26rft_id%253Dinfo%253Apmid%252F12730378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEwLzYyNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvODQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.81" data-doi="10.1007/s11032-017-0704-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">D.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Quantitative trait loci associated with agronomic traits and stripe rust in winter wheat mapping population using single nucleotide polymorphic markers.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-017-0704-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D37%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-017-0704-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-017-0704-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.82" data-doi="10.1038/s41477-017-0067-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The <em>Aegilops tauschii</em> genome reveals multiple impacts of transposons.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">955</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-017-0067-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D3%26rft.spage%253D946%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-017-0067-8%26rft_id%253Dinfo%253Apmid%252F29158546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-017-0067-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29158546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F841.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.83" data-doi="10.3835/plantgenome2016.10.0101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conway</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Quantitative trait loci mapping for spike characteristics in hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.10.0101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.10.0101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.10.0101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.841.84" data-doi="10.2135/cropsci2016.06.0459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">N’Diaye</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mapping QTLs controlling agronomic traits in the ‘Attila’ × ‘CDC Go’ spring wheat population under organic management using 90K SNP array.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.06.0459</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D365%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.06.0459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.06.0459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/841.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/829" title="Auxin-Inducible Depletion of the Essentialome Suggests Inhibition of TORC1 by Auxins and Inhibition of Vrg4 by SDZ 90-215, a Natural Antifungal Cyclopeptide" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/855" title="Dynamic Interactions Between the Genome and an Endogenous Retrovirus: Tirant in Drosophila simulans Wild-Type Strains" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/841" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IyEBgu4MLPz53thENEo0XOhtmJ8Y_edVhhKnX46Cdlk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/841" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/841" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/841" class="active" data-icon-position="" data-hide-link-title="0">Aegilops tauschii Genome Sequence: A Framework for Meta-analysis of Wheat QTLs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492432" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9N5R5pwkPd-qfXgspFZ3Ze8PHw9MxJbiDHa3KrOUTxg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/841.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/841" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-BwmA3R_ONgZXNNADC0A9X6eg8SqeVlmwSIVgIey7_8o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/841" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/841.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492432" id="node492432--2" data-pisa="ggg;9/3/841" data-pisa-master="ggg;g3.118.200921" data-apath="/ggg/9/3/841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Aegilops tauschii</em> Genome Sequence: A Framework for Meta-analysis of Wheat QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiale Xu</span>, <span class="highwire-citation-author" data-delta="1">Xiongtao Dai</span>, <span class="highwire-citation-author" data-delta="2">Ramesh K. Ramasamy</span>, <span class="highwire-citation-author" data-delta="3">Le Wang</span>, <span class="highwire-citation-author" data-delta="4">Tingting Zhu</span>, <span class="highwire-citation-author" data-delta="5">Patrick E. McGuire</span>, <span class="highwire-citation-author" data-delta="6">Chad M. Jorgensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Dehghani</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Gulick</span>, <span class="highwire-citation-author" data-delta="9">Ming-Cheng Luo</span>, <span class="highwire-citation-author" data-delta="10">Hans-Georg Müller</span> and <span class="highwire-citation-author" data-delta="11">Jan Dvorak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">841-853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200921 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492432--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiale Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiale%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiale%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiale Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiongtao Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Iowa, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiongtao%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiongtao%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ramesh K. Ramasamy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BK.%2BRamasamy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramasamy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BK.%2BRamasamy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Le Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tingting Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tingting%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATingting%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick E. McGuire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BE.%2BMcGuire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGuire%20PE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BE.%2BMcGuire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chad M. Jorgensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BM.%2BJorgensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jorgensen%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BM.%2BJorgensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hamid Dehghani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Breeding, Faculty of Agriculture, Tarbiat Modares University, Tehran, Iran, and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BDehghani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dehghani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BDehghani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Dehghani</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick J. Gulick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Concordia University, Montreal, Quebec, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BGulick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gulick%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BGulick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Gulick</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ming-Cheng Luo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ming-Cheng%2BLuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMing-Cheng%2BLuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hans-Georg Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Georg%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Georg%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jan Dvorak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BDvorak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dvorak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BDvorak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdvorak@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jdvorak@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/841" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/841&amp;text=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/841&amp;t=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/841&amp;title=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492432/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492432" id="node492432--3" data-pisa="ggg;9/3/841" data-pisa-master="ggg;g3.118.200921" data-apath="/ggg/9/3/841.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>Aegilops tauschii</em> Genome Sequence: A Framework for Meta-analysis of Wheat QTLs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jiale Xu</span>, <span class="highwire-citation-author" data-delta="1">Xiongtao Dai</span>, <span class="highwire-citation-author" data-delta="2">Ramesh K. Ramasamy</span>, <span class="highwire-citation-author" data-delta="3">Le Wang</span>, <span class="highwire-citation-author" data-delta="4">Tingting Zhu</span>, <span class="highwire-citation-author" data-delta="5">Patrick E. McGuire</span>, <span class="highwire-citation-author" data-delta="6">Chad M. Jorgensen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hamid Dehghani</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patrick J. Gulick</span>, <span class="highwire-citation-author" data-delta="9">Ming-Cheng Luo</span>, <span class="highwire-citation-author" data-delta="10">Hans-Georg Müller</span> and <span class="highwire-citation-author" data-delta="11">Jan Dvorak</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">841-853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200921 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492432--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiale Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiale%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiale%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7602-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jiale Xu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiongtao Dai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Iowa, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiongtao%2BDai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dai%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiongtao%2BDai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ramesh K. Ramasamy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramesh%2BK.%2BRamasamy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramasamy%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamesh%2BK.%2BRamasamy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Le Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tingting Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tingting%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATingting%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrick E. McGuire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BE.%2BMcGuire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McGuire%20PE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BE.%2BMcGuire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Chad M. Jorgensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chad%2BM.%2BJorgensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jorgensen%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChad%2BM.%2BJorgensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Hamid Dehghani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Breeding, Faculty of Agriculture, Tarbiat Modares University, Tehran, Iran, and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BDehghani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dehghani%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHamid%2BDehghani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3267-5144" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hamid Dehghani</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Patrick J. Gulick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biology, Concordia University, Montreal, Quebec, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BJ.%2BGulick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gulick%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatrick%2BJ.%2BGulick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7587-4616" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patrick J. Gulick</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ming-Cheng Luo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ming-Cheng%2BLuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMing-Cheng%2BLuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Hans-Georg Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Georg%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Georg%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Jan Dvorak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California, </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BDvorak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dvorak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BDvorak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jdvorak@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jdvorak@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=841&amp;rft.epage=853&amp;rft.atitle=Aegilops%20tauschii%20Genome%20Sequence%3A%20A%20Framework%20for%20Meta-analysis%20of%20Wheat%20QTLs&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xu%2C+Jiale&amp;rft.au=Dai%2C+Xiongtao&amp;rft.au=Ramasamy%2C+Ramesh+K.&amp;rft.au=Wang%2C+Le&amp;rft.au=Zhu%2C+Tingting&amp;rft.au=McGuire%2C+Patrick+E.&amp;rft.au=Jorgensen%2C+Chad+M.&amp;rft.au=Dehghani%2C+Hamid&amp;rft.au=Gulick%2C+Patrick+J.&amp;rft.au=Luo%2C+Ming-Cheng&amp;rft.au=M%C3%BCller%2C+Hans-Georg&amp;rft.au=Dvorak%2C+Jan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492432/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492432/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492432/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492432/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492432/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492432/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492432/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492432/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492432/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492432/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492432/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492432/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/841" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/841.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/841.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/841.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/841.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
