<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/879" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/879.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/879.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/879.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/879/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/879.atom" />
<meta name="HW.pisa" content="ggg;9/3/879" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" />
<meta name="DC.Identifier" content="10.1534/g3.118.200745" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Douglass et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7575536" />
<meta name="DC.Description" content="Illumina sequencing has revolutionized yeast genomics, with prices for commercial draft genome sequencing now below $200. The popular SPAdes assembler makes it simple to generate a de novo genome assembly for any yeast species. However, whereas making genome assemblies has become routine, understanding what they contain is still challenging. Here, we show how graphing the information that SPAdes provides about the length and coverage of each scaffold can be used to investigate the nature of an assembly, and to diagnose possible problems. Scaffolds derived from mitochondrial DNA, ribosomal DNA, and yeast plasmids can be identified by their high coverage. Contaminating data, such as cross-contamination from other samples in a multiplex sequencing run, can be identified by its low coverage. Scaffolds derived from the bacteriophage PhiX174 and Lambda DNAs that are frequently used as molecular standards in Illumina protocols can also be detected. Assemblies of yeast genomes with high heterozygosity, such as interspecies hybrids, often contain two types of scaffold: regions of the genome where the two alleles assembled into two separate scaffolds and each has a coverage level C , and regions where the two alleles co-assembled (collapsed) into a single scaffold that has a coverage level 2 C . Visualizing the data with Coverage- vs. -Length (CVL) plots, which can be done using Microsoft Excel or Google Sheets, provides a simple method to understand the structure of a genome assembly and detect aberrant scaffolds or contigs. We provide a Python script that allows assemblies to be filtered to remove contaminants identified in CVL plots." />
<meta name="DC.Contributor" content="Alexander P. Douglass" />
<meta name="DC.Contributor" content="Caoimhe E. O’Brien" />
<meta name="DC.Contributor" content="Benjamin Offei" />
<meta name="DC.Contributor" content="Aisling Y. Coughlan" />
<meta name="DC.Contributor" content="Raúl A. Ortiz-Merino" />
<meta name="DC.Contributor" content="Geraldine Butler" />
<meta name="DC.Contributor" content="Kevin P. Byrne" />
<meta name="DC.Contributor" content="Kenneth H. Wolfe" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Illumina sequencing has revolutionized yeast genomics, with prices for commercial draft genome sequencing now below $200. The popular SPAdes assembler makes it simple to generate a &lt;i&gt;de novo&lt;/i&gt; genome assembly for any yeast species. However, whereas making genome assemblies has become routine, understanding what they contain is still challenging. Here, we show how graphing the information that SPAdes provides about the length and coverage of each scaffold can be used to investigate the nature of an assembly, and to diagnose possible problems. Scaffolds derived from mitochondrial DNA, ribosomal DNA, and yeast plasmids can be identified by their high coverage. Contaminating data, such as cross-contamination from other samples in a multiplex sequencing run, can be identified by its low coverage. Scaffolds derived from the bacteriophage PhiX174 and Lambda DNAs that are frequently used as molecular standards in Illumina protocols can also be detected. Assemblies of yeast genomes with high heterozygosity, such as interspecies hybrids, often contain two types of scaffold: regions of the genome where the two alleles assembled into two separate scaffolds and each has a coverage level &lt;i&gt;C&lt;/i&gt;, and regions where the two alleles co-assembled (collapsed) into a single scaffold that has a coverage level 2&lt;i&gt;C&lt;/i&gt;. Visualizing the data with Coverage-&lt;i&gt;vs.&lt;/i&gt;-Length (CVL) plots, which can be done using Microsoft Excel or Google Sheets, provides a simple method to understand the structure of a genome assembly and detect aberrant scaffolds or contigs. We provide a Python script that allows assemblies to be filtered to remove contaminants identified in CVL plots.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/879" />
<meta name="citation_id" content="9/3/879" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/879" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/879.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/879.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/879.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200745" />
<meta name="citation_pmid" content="30674538" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="879" />
<meta name="citation_lastpage" content="887" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alexander P. Douglass" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author" content="Caoimhe E. O’Brien" />
<meta name="citation_author_institution" content="School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author" content="Benjamin Offei" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author" content="Aisling Y. Coughlan" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8614-7780" />
<meta name="citation_author" content="Raúl A. Ortiz-Merino" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4186-8941" />
<meta name="citation_author" content="Geraldine Butler" />
<meta name="citation_author_institution" content="School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author" content="Kevin P. Byrne" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author" content="Kenneth H. Wolfe" />
<meta name="citation_author_institution" content="School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland" />
<meta name="citation_author_email" content="kenneth.wolfe@ucd.ie" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4992-4979" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=S. Braun-Galleani;citation_author=R. A. Ortiz-Merino;citation_author=Q. Wu;citation_author=Y. Xu;citation_author=K. H. Wolfe;citation_title=Zygosaccharomyces pseudobailii, another yeast interspecies hybrid that regained fertility by damaging one of its MAT loci.;citation_pages=foy079;citation_volume=18;citation_year=2018;citation_doi=10.1093/femsyr/foy079" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. D. Donovan;citation_author=G. Gonzalez;citation_author=D. G. Higgins;citation_author=G. Butler;citation_author=K. Ito;citation_title=Identification of fungi in shotgun metagenomics datasets.;citation_pages=e0192898;citation_volume=13;citation_year=2018;citation_doi=10.1371/journal.pone.0192898" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Dujon;citation_author=E. J. Louis;citation_title=Genome diversity and evolution in the budding yeasts (Saccharomycotina).;citation_volume=206;citation_year=2017;citation_pmid=28592505;citation_doi=10.1534/genetics.116.199216" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Gordon;citation_author=D. Armisén;citation_author=E. Proux-Wéra;citation_author=S. S. ÓhÉigeartaigh;citation_author=K. P. Byrne;citation_title=Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents.;citation_volume=108;citation_year=2011;citation_pmid=22123960;citation_doi=10.1073/pnas.1112808108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. D. Jackman;citation_author=B. P. Vandervalk;citation_author=H. Mohamadi;citation_author=J. Chu;citation_author=S. Yeo;citation_title=ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter.;citation_volume=27;citation_year=2017;citation_pmid=28232478;citation_doi=10.1101/gr.214346.116" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. R. Love;citation_author=K. A. Shah;citation_author=C. A. Whittaker;citation_author=J. Wu;citation_author=M. C. Bartlett;citation_title=Comparative genomics and transcriptomics of Pichia pastoris.;citation_pages=550;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2876-y" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Lu;citation_author=S. L. Salzberg;citation_title=Removing contaminants from databases of draft genomes.;citation_pages=e1006277;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pcbi.1006277" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Stand. Genomic Sci.;citation_author=S. Mukherjee;citation_author=M. Huntemann;citation_author=N. Ivanova;citation_author=N. C. Kyrpides;citation_author=A. Pati;citation_title=Large-scale contamination of microbial isolate genomes by Illumina PhiX control.;citation_pages=18;citation_volume=10;citation_year=2015;citation_pmid=26203331;citation_doi=10.1186/1944-3277-10-18" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=C. Neuveglise;citation_author=H. Feldmann;citation_author=E. Bon;citation_author=C. Gaillardin;citation_author=S. Casaregola;citation_title=Genomic Evolution of the Long Terminal Repeat Retrotransposons in Hemiascomycetous Yeasts;citation_pages=930-943;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12045146;citation_doi=10.1101/gr.219202" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=S. O’Boyle;citation_author=S. A. Bergin;citation_author=E. E. Hussey;citation_author=A. D. McLaughlin;citation_author=L. R. Riddell;citation_title=Draft genome sequence of the yeast Nadsonia starkeyi-henricii UCD142, isolated from forest soil in Ireland.;citation_pages=0054918e;citation_volume=6;citation_year=2018;citation_doi=10.1128/genomeA.00549-18" />
<meta name="citation_reference" content="citation_journal_title=J. Basic Microbiol.;citation_author=I. Y. Pavlov;citation_author=K. S. Bobrov;citation_author=A. D. Sumacheva;citation_author=A. E. Masharsky;citation_author=D. E. Polev;citation_title=Scytalidium candidum 3C is a new name for the Geotrichum candidum Link 3C strain.;citation_pages=883-891;citation_volume=58;citation_year=2018;citation_doi=10.1002/jobm.201800066" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. P. Pryszcz;citation_author=T. Gabaldon;citation_title=Redundans: an assembly pipeline for highly heterozygous genomes.;citation_volume=44;citation_year=2016;citation_pmid=27131372;citation_doi=10.1093/nar/gkw294" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. P. Pryszcz;citation_author=T. Nemeth;citation_author=E. Saus;citation_author=E. Ksiezopolska;citation_author=E. Hegedusova;citation_title=The genomic aftermath of hybridization in the opportunistic pathogen Candida metapsilosis.;citation_pages=e1005626;citation_volume=11;citation_year=2015;citation_pmid=26517373;citation_doi=10.1371/journal.pgen.1005626" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. S. Schröder;citation_author=K. Martinez de San Vicente;citation_author=T. H. Prandini;citation_author=S. Hammel;citation_author=D. G. Higgins;citation_title=Multiple origins of the pathogenic yeast Candida orthopsilosis by separate hybridizations between two parental species.;citation_pages=e1006404;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006404" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=X. X. Shen;citation_author=X. Zhou;citation_author=J. Kominek;citation_author=C. P. Kurtzman;citation_author=C. T. Hittinger;citation_title=Reconstructing the backbone of the Saccharomycotina yeast phylogeny using genome-scale data.;citation_volume=6;citation_year=2016;citation_pmid=27672114;citation_doi=10.1534/g3.116.034744" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=A. A. Stavrou;citation_author=V. Mixao;citation_author=T. Boekhout;citation_author=T. Gabaldon;citation_title=Misidentification of genome assemblies in public databases: The case of Naumovozyma dairenensis and proposal of a protocol to correct misidentifications.;citation_pages=425-429;citation_volume=35;citation_year=2018;citation_doi=10.1002/yea.3303" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=P. R. Sternes;citation_author=D. Lee;citation_author=D. R. Kutyna;citation_author=A. R. Borneman;citation_title=Genome sequences of three species of Hanseniaspora isolated from spontaneous wine fermentations.;citation_pages=e0128716;citation_volume=4;citation_year=2016;citation_doi=10.1128/genomeA.01287-16" />
<meta name="citation_reference" content="citation_journal_title=J. Biotechnol.;citation_author=L. Sturmberger;citation_author=T. Chappell;citation_author=M. Geier;citation_author=F. Krainer;citation_author=K. J. Day;citation_title=Refined Pichia pastoris reference genome sequence.;citation_volume=235;citation_year=2016;citation_doi=10.1016/j.jbiotec.2016.04.023" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=A. Venkatesh;citation_author=A. L. Murray;citation_author=A. B. Boyle;citation_author=L. Quinn Farrington;citation_author=T. J. Maher;citation_title=Draft genome sequence of a highly heterozygous yeast strain from the Metschnikowia pulcherrima subclade, UCD127.;citation_pages=e0055018;citation_volume=6;citation_year=2018;citation_doi=10.1128/genomeA.00550-18" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=J. Watanabe;citation_author=K. Uehara;citation_author=Y. Tsukioka;citation_title=Can interspecies hybrid Zygosaccharomyces rouxii produce an allohaploid gamete?;citation_pages=e0184517;citation_volume=84;citation_year=2018;citation_doi=10.1128/AEM.01845-17" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Wu;citation_author=A. Elliston;citation_author=G. Le Gall;citation_author=I. J. Colquhoun;citation_author=S. R. A. Collins;citation_title=Yeast diversity in relation to the production of fuels and chemicals.;citation_pages=14259;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-14641-0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Illumina sequencing has revolutionized yeast genomics, with prices for commercial draft genome sequencing now below $200. The popular SPAdes assembler makes it simple to generate a de novo genome assembly for any yeast species. However, whereas making genome assemblies has become routine, understanding what they contain is still challenging. Here, we show how graphing the information that SPAdes provides about the length and coverage of each scaffold can be used to investigate the nature of an assembly, and to diagnose possible problems. Scaffolds derived from mitochondrial DNA, ribosomal DNA, and yeast plasmids can be identified by their high coverage. Contaminating data, such as cross-contamination from other samples in a multiplex sequencing run, can be identified by its low coverage. Scaffolds derived from the bacteriophage PhiX174 and Lambda DNAs that are frequently used as molecular standards in Illumina protocols can also be detected. Assemblies of yeast genomes with high heterozygosity, such as interspecies hybrids, often contain two types of scaffold: regions of the genome where the two alleles assembled into two separate scaffolds and each has a coverage level C , and regions where the two alleles co-assembled (collapsed) into a single scaffold that has a coverage level 2 C . Visualizing the data with Coverage- vs. -Length (CVL) plots, which can be done using Microsoft Excel or Google Sheets, provides a simple method to understand the structure of a genome assembly and detect aberrant scaffolds or contigs. We provide a Python script that allows assemblies to be filtered to remove contaminants identified in CVL plots." />
<meta name="og-title" property="og:title" content="Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/879" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Illumina sequencing has revolutionized yeast genomics, with prices for commercial draft genome sequencing now below $200. The popular SPAdes assembler makes it simple to generate a de novo genome assembly for any yeast species. However, whereas making genome assemblies has become routine, understanding what they contain is still challenging. Here, we show how graphing the information that SPAdes provides about the length and coverage of each scaffold can be used to investigate the nature of an assembly, and to diagnose possible problems. Scaffolds derived from mitochondrial DNA, ribosomal DNA, and yeast plasmids can be identified by their high coverage. Contaminating data, such as cross-contamination from other samples in a multiplex sequencing run, can be identified by its low coverage. Scaffolds derived from the bacteriophage PhiX174 and Lambda DNAs that are frequently used as molecular standards in Illumina protocols can also be detected. Assemblies of yeast genomes with high heterozygosity, such as interspecies hybrids, often contain two types of scaffold: regions of the genome where the two alleles assembled into two separate scaffolds and each has a coverage level C , and regions where the two alleles co-assembled (collapsed) into a single scaffold that has a coverage level 2 C . Visualizing the data with Coverage- vs. -Length (CVL) plots, which can be done using Microsoft Excel or Google Sheets, provides a simple method to understand the structure of a genome assembly and detect aberrant scaffolds or contigs. We provide a Python script that allows assemblies to be filtered to remove contaminants identified in CVL plots." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"45Qo74CYSU4qFzdS0oyFlBZ0VDd8UcXNDSB0BLltYkw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492416","apath":"\/ggg\/9\/3\/879.atom","pisa":"ggg;9\/3\/879","ac":{"\/ggg\/9\/3\/879.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/879.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/879"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/879"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/879":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j78XQJrRn\\\\\\\/Hgmin6\\\\\\\/isfGEY7uSrQA6ddS8IX\\\\\\\/i2ZAbn2buj0TI3wi2YrwNJMlQsIqGWfwZ8QkCDtJdw7EhKb7NEWBSiX\\\\\\\/NCQYTUCbnkkzOIjM662O53tq0nACKzYBYw6P0VctO5KCXlwzIkgJfzFSYPu6pKul77V4EiSbj66TFanWI3Ae7FzQh0Tx7wCcCbWBPfxgLwVAXDQYOAk531io5MeIQjmaBZqdHCRZJ9I5BeYI18dl45JN88IDnR52SUILIJWArbHp12ZR4tE\\\\\\\/wOcAR4AaN7U10W2GpZ2mSIa1p9C9jLAVfCKc7p6ghTNRuddZVzWxz5CFrZkY3OI1PCy07TTZujkb1px57B4Snj+ugWnZ9Ez5saKTwGvZzu\\\\\\\/wFodYSqX1ixk9LMwcQ1crZhV4v2r\\\\\\\/lcy6suCl\\\\\\\/6nIrl1JhF3\\\\\\\/KxO\\\\\\\/blANGSD1wtBTDT3udapxtTiH6d9TcabS3AXl31FsvGRwQpyBWUlbAyHtZybXGb513fthSnQLJY6+aT2K42ClYFyLL32u+Qqoti7OiK5K7pD4elWYAqZ6Z\\\\\\\/MtA7Kh0Al+gQ0jfCjjiid\\\\\\\/+toge5Pxiv6b8eD5+e3ABpiMVWEeuKhEBkC4xMUBH28JFSKgNERm4oPUjJim+zSh0k+SeJVIy6vJbVn+wydKug+Ot67jNP0XUmQHa+rYjzdum5y++0MhVbY9082t7A+qQiaUwoq7FSWRpNX0Uoa64ygLGi1sHeVw\\\\\\\/VL6zVwsdw1tyWdaYjBBrLR1pKxrYm8zqu9+S75HiRP4sAHpMEArjvi5m6PI1+HGC5bTpo5KbucfPsK7aAsZCAEWUjbKl6cRDjjAuMBJY28uVBbzkvfxz0885UPqtLY1at+XW3ZKrdNcqd\\\\\\\/6+xb\\\\\\\/klcC\\\\\\\/c77DCJYmEFr5EINVvGCuFT2g8Cxr\\\\\\\/CKZllgCsJazStq7cbGkvKLs\\\\\\\/\\\\\\\/MkdJQhBxgz+MDUVB8R81gXgECsrRv0ehW4O+JmZ491jIF2jnSQS6zbN3JEGcZQXyYTUvu2XHCNHorOOp+oJiw5h7gHRjQ\\\\\\\/Fnt5E824zKXH6kF8Bif06PW4+Jbm5Yo\\\\\\\/e27X6z7nTE\\\\\\\/md56uWuBwil0xZE0kXq3\\\\\\\/WY7nKEKf0serxeHTap7W9uPel7B\\\\\\\/NbHQxpPxIJ0KqcHareHy5xiLqaYUGbdPASMipwgOuIp\\\\\\\/RXi4QEi\\\\\\\/A4xCi3oUdhDi+CI9l+RXgMKXXFoM7xdjWbf7BX3f7pQNT41fO15kNejj\\\\\\\/mCX2mNI2dR4ZCLuhpbfi05uZtjpwLKBC\\\\\\\/uzWF5x7HZuNB\\\u0022,\\\u0022iv\\\u0022:\\\u0022TvW9qo+DVhG5eK3BAnXJTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bbc69eba71b8920f0a89af73562ce22d8ddd3270688f179690ba827bf2119e15\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UHmPHqGOJqn4FdbIXG3BpmPGXRrWcqGpTlC+qe7Sd\\\\\\\/RGQ1BgfF3Uf3WwLw8Dv6EytDrnNhXUhAGMUCYvO6yxyPiNIObnN155N11qPENsO3diitwacK1qrm+pemsAnLW6stWZfIZL6AkdD\\\\\\\/WX\\\\\\\/eSf1QsXv6G671Cuyth8WgVtPoWZPcBPHc01xjVIPwr6G6MgQbSq0nrEdshQ1Cf02pX7yOC+gwKcsUPxmvjenUFzW7d4IXvtZ\\\\\\\/72zpceJL6y1YA+T57K0tSzfm2aV9wRQl9wM121ho8U4ZSu7YPKPGcEAgQhzv72WDI1aebCG5\\\\\\\/Q+xbDwAVi\\\\\\\/hSDxMa\\\\\\\/HHOTNjaHcYu4HVtKfzMEQZzqlweyJRzS1CKBl6hilQp25gpE+oQyGi2iE\\\\\\\/E1MJ1J0KTZhU2tNwdIhRL91m5\\\\\\\/SuyfQiSORrNCLe1R4YkGMEonNYH66unKLkXHsPVnIh0rF2g9VQVbuYaEwaOJlf+9dBoiiByBfhnIi\\\\\\\/cvLLYBeYAVqIPbB0i1hZ0iMeB8DBmZbDLsXqrnWxolWQXiEUxlHENR3L47ieHGGMoWBM52WCEIpJR8SnzYYTIcBYy8yOZXitbTnufkBS7Obt+TK+eZXU0mMecxF8+MW2A7n8ElIRjfGnyRuy5DRouFUXvQ0BjQCahcoEBW6yhfvY2KLrOCmyjSV8af\\\\\\\/0u78IDoVKepb+sG3JyEiRwkl\\\\\\\/hyRyupsqg4mFuVO6X7tU4XFX5GCKnrSv8+wW4zTSmce0regSMr2mRSTIbbgIGzZwDqxTjaOBUK1BNNL3YLbIIE\\\\\\\/uy+O7e\\\\\\\/jE1XCkHKshv\\\\\\\/aj7M8acacQ4EJEkA+wUWyeRORhgHjY2ehpUGOAcDgQreGh03oAZqPJTtosWmZyuBst49lkSfC1DHC9OOtp\\\\\\\/GXlgswiKioaVYOQWI16AFD8BeAX0OAG8vrVjUWTxONsKONnlQ0crCGsM7pKQn2omtBPiJFFqfpnDcWJ9+yav0\\\\\\\/m674sVQuvQfrykEXcEOAlcYFX0DTVdtCdkiLXDatHFz1OudAhU5M\\\\\\\/DKY3\\\\\\\/B1Hwll7L\\\\\\\/j7cZpY\\\\\\\/hPgl8HYtuihnkbxT8YQsGTCimN+sR44NIgbOO3JfloM0zvS8xFu3MaTYwtqX9v5aJU85wqdjV\\\\\\\/Zj+s8bKhyIKB8zaXpmgME3BSsHxbC7Imx5QuGG\\\\\\\/nteOrwfsX1KbJ4svOJSLKNVN7UIGBcq5hiHOeLXLvihZyL5Ka4AlCUHK\\\\\\\/DZ9odJ7w9TjWPt29yBMAE7Zk2lHDcwO2Nt6WNZcavvUszKf5lUulBNtCqaNptM3GjXAjLDH0vn6ePMZndThiXct6tQEHzFupQfEfd+nxsatiA\\\\\\\/MNzCL9mQ3Y2y0P3bzPWwW5wjOOJ0G6vbmVcaPKYoxcsKkQPi\\\\\\\/PRFlV0cuGFWllTPIy8fVD1G0BNsuAldYi5gl\\\\\\\/qZOHnRp9vbfVfX5ELJ3FgOn\\\\\\\/Qpd5lV4DU+iCgt9Xr4XI7xm1GoFCR5yx37+qaWJ9F0tgvNFjZcnXkTEixWprSc51Gp6hcs6ZPwXEIj7ZMXG8Ofw2RHFHsnLrACUaG9AWWu\\\\\\\/S9u\\\\\\\/sJP\\\\\\\/XXttdEhwEDmYusd\\\\\\\/oj9Ej0GsQoZbDHTqV5AzVTcUCHz35dp37Eh5Isfc76j8n1Jmq41LraoFGhVr\\\\\\\/9p0jLUKnJODtVJ9FEr0GFgPDlx1JOEsbqptcjAHL6\\\\\\\/tkzVx9Tk6negZVdowKbhWukAI76D\\\\\\\/lRE1EFI264Wx8c+da5TjVavFpNKtnR6nHFYnhDZO9Wo2GM33sS+8ALs6HEoJZzHvc7S+N1vJQhBBN7cISQU4RoL42ieJj\\\\\\\/yxoFxC2Kmxs9gVzmJlH3xoyKVWsBmKKnHiVvNhBO6UVa5b1Fm\\\\\\\/xAWp9pgJ46WEEQ4iQq\\\\\\\/lFjoa3tf9y+2wwDuVO8mSaKUxcPct5ajlZwR80Zl9eI\\\\\\\/2fBBiXzOOGXcSjAjY\\\\\\\/hWEqbV909AP4pHUw9dbPCo18On2nRgS4\\\\\\\/k0OtxN8JvgSEJMQ2EWT4UMMgu0yxUfcsNl2MhbC7HmndwRSjT+acV9lZwBky9TswxEQRUf+HiTDIFiOwf7Y0fFT1KlqDnTJ2nVb+cchf8+PMWM4YRTm5cPaJIXelmy2f54AJmSGEEG5MsHQj9EN0enB0C4F7DKHQ1ncQDqDRr551eCPTcrNGJc+q8aMRswcZpqSv9Gni+bHpaR1zQDjWge9\\\\\\\/9Tva8\\\\\\\/99nre9SZXXk0XuszXHG3RgxJKPKbULHpI70yy84zj9I\\\\\\\/EQ3ykae7HRXAQxO4Xeo3KWC0sb+Req6RtzVhgpwLQFzjE\\\\\\\/hekdEvSEKci6gEtGGmFb3QtsROBLYsRByIETqS2YdKLFIRSKmcqYif3N+gWOlmvEVWRGXFXVJ24RiLv\\\\\\\/w8AjcwtwTcD8HVzX5ypRp\\\\\\\/xJA+HJsxeS9IdHj\\\\\\\/qmiita+uRuGo7UletsPXVzZ2Wf4+qkcEUDbdWQRoSbFvZ5Dt5uaBsQJR+enU0yHVNZUHlkrF93LdjixyxPsXSz3HFq0XSux5M4HCkWaqckvvfJxIVgQTHGN\\\\\\\/00gTshJR6umIzN9Da5EsklFt84lY+Feltrpl1ttavjKgsf9Wt7dT1EqLNZ4QfNRcofTnyvaYAIDPvwM\\\\\\\/pnX5OlOBgqKiCs7xRs6y4j+A8nS6w10BXxrICMTEiyO941tTIat4+HSpUekyu17Ro23ZJq33nVuD949RuJgcMyybabRhYymxiiR\\\\\\\/Irg8HHj\\\\\\\/tAykKwYJNEtdks4lL6X3\\\\\\\/liEKC9C4G3vPqFJsMrTyCYPpeFbht0ubzBi23nK1nv9JXNi9DSgNILwL+Zvz2CoGPVoJIoYtsTYvE+6tGjCQFF+nc6dZ0xx2OPP73ppcPSVi7Dnwn15sr+XUgotsF6DGN4tr252Asj\\\\\\\/X7gR3GUtcFRriZ5pqHTW0Sok5NOyBkM85mbx9aSniMav0Ma\\\\\\\/TLFgQuZkesuCbHyyPcZ\\\\\\\/K2Ke\\\\\\\/xEMnmEQt3sTXcCXLIInX4x0umImMYDHsGWzlT8+npnVDZYu+RL9dfp81sMuy5NiXwFXnPWuRpOxBlg6EjasHGrwCrx2vZFjZ+IYVJCbIH+dPrK6\\\\\\\/yZ2\\\\\\\/ZgI86WeoeNc+NFDBdsAKd3kQLYe+3xeNtqmyAWZW5IPNvzR12GB3NRBr8o0D\\\\\\\/eLgZfhqLlJIr1hPRZx0KfolVQlv9odgdQk22mGwLFwLU4HOxnY5MWfx4BgZAtfAF7Iqyfgz+9RkKbcTFjKKb\\\\\\\/wACT37vmrDglwEw7opCPEOxYeAiiN5GgRAqwKa1V5A9sBUg9aU\\\\\\\/30Qb\\\\\\\/SKVcg22y1VXcPDvrla2OsbQCMmfvLxVFhdUfSzJy7TU\\\\\\\/zWNSKO4gRudwWJixiSebNrjmfiTdRN\\\\\\\/6RdkcaCrNNQU\\\\\\\/Ou+1okOWqJezCnnTARZzbMaZBK8Pk45jCwFmBT2H8fj8cTYzOsdJBof12hLO\\\\\\\/DHDzpyD3nNeHx1BktpQ8za6dlF4qLxWmPWtxIyWJQ2f6PLXZBXfzfPkvwK\\\\\\\/g\\\\\\\/\\\\\\\/HFjYFzzsPIkkiFXp4LKpTJfz6ZazYsbiv+KECjYHrpJmnIJfvMMBHcDaYpF0hhmvBANwHSo89SxrAwAlKFSurRQ+1tcFztmPAAdkmHWtGqB\\\\\\\/XbOllgsyclIenC\\\\\\\/gISTkGdmVPduxULmjU8kR9fxU6yTD70KzUDq+xzgvi\\\\\\\/m+OAOaRgsrECIwScV228q1DDrKrJ7ihwMrO28fSu4tx1sS2r961r6KQj9fROgMjpbHzU9KL8giFqSjVxEr1pLRSit3yymcEm8EGP0woYSGs+l5t19lNjg8i8pv66uvmlEEXQhSzqgde3QJur6Mz4aXeoo2vVdat2S5dB8Kc13B71zv4zgOzDW0nt3MIIF4PhP9GsdvQIWRHp3hfqUz16LHC1oSS5RcfQl\\\\\\\/D95OBN7t7IcFizXhXKyTXJLe6iBYAoDlVeh5NWenLKN\\\\\\\/UgdqGLhKOByqu4usdWTobVDm\\\\\\\/4\\\\\\\/Gd\\\\\\\/9j09Tgm74Djff11qBpMGt0P8j8+\\\\\\\/ztnjKF1fbEiwuNus\\\\\\\/wFnv9UMbebcZb+9VajHLGWmyFfMZ7CGUEoV+RFBoZ5J1hu91O7lnLVEdlwcbZ4VZsvbhXYs\\\\\\\/mMSK5UCGsx7X\\\\\\\/XtULF1i4K1VSuh7Ag7HV586oxP3F8+ALawtHrs0PnDA9Ox9p0+lIZb1JI0rNm\\\\\\\/ydg4LEzfClLqgZJusBZaq7HaHZ5HvwSj7sxFhpuvt3aGjqAU90LV4\\\\\\\/S50JgHB\\\\\\\/pHGln+OwQffEC1IWI3W65cM1Wn5vz\\\\\\\/3jzkeCdDgD8gmKrNzktOeCLBMk9aSLd8z\\\\\\\/rvJfAmP\\\\\\\/yGWWp3F5fZ0LkLq+1lLO2DSdMRQUnoC8Mnz1lFtgRC29u3V6UukX+QAGmpD\\\\\\\/qDXVRYqXn1YLesypuKg0HO3IgEWtUWrBpVR8w3EShhw9bo9k17U3DS3SltyZU8DTNS0ELVwnVz4T\\\\\\\/m1fnmuhyiGufSO8bAfxfzdGdlehrd\\\\\\\/BPeEAQ2kliy3rPyngP+5VfUUJaWMI7jyoHKGaJPOMx5NI7lhviUKwadV77OvbLGGR5281jZEQKu5+0ohSYrNBr+\\\\\\\/dmZ7IUhwEg4Mc2JS91cb7H2vz1gxVuAklxkXKFRe5artKoYpeh1tzqTo3uiMm6hvIFqvtjbWSKTGC7ZDE5ToKjSVxuoxzRsK0iw8hz2I0j+NP0QxMpQu+LsNRlmJxipL9QoqmhQTPrgoaD6qHrJKFTxJAqyfVSD5uCaf+YZfZyB8cb2qpwIPH7gr0DKg2ga4SS1ZzZ+QFsCuNh0P1lgyuCcCQFDxGXAWRNL\\\\\\\/7EK1CmoLExe\\\\\\\/Pj2gEsy1Fx13NRw2lhV0\\\\\\\/1ocCTWyCQZ9XgAeMZZsUa1xGZFOazdmn5zQ0uWf5qp33amvMKtPjWoTlWVjqgvMz4z5icZEokBRpfcKvKrYnOn0xZoKpp+vcOhARDKg1An2TIWu8E79zeyUbrd0G1zWzkTH9tWN0yn0R+DfCwRYUS0nLiP45XTn+41a0SawNOS+fQ69r7B8R00dOOcfW4tDndHi7mLENFeNJKBKv1wiZNp57+8UYG6b5VwhqKpKG8hHkTyNh\\\\\\\/LjvJXDQEFlQpCQIWmOM8i\\\\\\\/zs78u5O+g8wvPggkekWNNn418KzejzEHYFRFMLm8RW+MDCYqUbYUr09VvDGt\\\\\\\/oki1ldfLJzH\\\\\\\/iKb82XBbQ+wb7CHoJUUoFvSe0FKv6h6zzELD3ZWNgmjnkEqDVwQErSRJ8ZnrcP\\\\\\\/VY7c8+C3oPk6vhMhQTFgrE06wH\\\\\\\/Sfrf\\\\\\\/2RqM8UcPx5IczCy4G82HAiQ4Ey3lRGh4HakgaYAt4OaGA4HYWJr1wRM5oheXGty05lCtrRh0kUtWti8PlI5j\\\\\\\/hgnxiWBDkwdv5h1dZEwrPXpHtuwkSzhIXU606uPcU7fW1MrS+vN64MsYXsBhC3zJDCO9L5JnoQ0yecnD5MVVWGYm7KN9hDMwjxOAh1owdaUMQbW28OMwQxYrdyptydDnAVk+WGNTLzt\\\\\\\/s2MdkyoWacKLEmMpWf6GTt5aL5WOgGRcwltjUnj+v2mU6JiWEUlvQIrxt2+LNTzZZTIMUqkvfGg6vGxO6Ap\\\\\\\/kqO7bo7Bv7lDBmPygN89jecKHBCtYIQiK5Pq\\\\\\\/KRPmaase1Pm5WXI4kL\\\\\\\/2A6dZ8Srq1Ib65GseoaV6Gq4NdCfHdg3BDjCAh3lNEuAwKkXW+jq65nkrP3Xq6owfzvNn7VZoiotrZdeIcWjpsLEVBSrYhbjl4dbOZGp\\\\\\\/rYI4k2AXD45LEJjWsWT6Q90d9jlQ0DQNGDp0chE+3lRqut1zu+HIzzzWJEEUk4doYVSJA6oedqDSfTE4f7eLLYIpjN3kBoPTTLl60tw0uXF64ergYzRKQKCh4yfqhnPlLbxtq0OfNvISANAaSHJJWZKbwlwHFRyh3DHO7luNEoqTYzgCoQ4vQnCcu3LThFXVOLZcfAlfO+CWfQZBqV4ZcMayp2dfwM2UDP+OoDR4\\\\\\\/8zCqZAVOOBTx66LGwOu+pPNuwqDkuAG2wDS3D0xDaCU1aegsgf2Xh7ScENr3uvf\\\\\\\/PeGoYC5yR45YY3XKLsIF0VI3UNj9uEa2hrbPeVqtlpOlj9XgEBodPOhwdZEyLJEM8ZM\\\\\\\/G7LpuVSgj3Msd4So\\\\\\\/f8ZTygYgGyJIOrzoaO5plKqaD\\\\\\\/4p0lNz7YfyhDZ2Cc6rLKW+29e0xpHGWDnEcjmH\\\\\\\/aHkGhli8164Q7uffI0S9cb0PgNxftmaMh1\\\\\\\/jrb8IvGqzv6CgmRw4ia6a+gcx9sfD8w16UPcPEI3ebckr\\\\\\\/FUeUXFb\\\\\\\/auOfQgyw2M\\\\\\\/IVAyXOMsLmuUvMaj5kBMdGiZpmS1AC\\\\\\\/S1irUTK\\\\\\\/5gJmg3ywhtja1MvsV5dF7Bajz1jfFd\\\\\\\/H4REJy3tLgbDtCVm4jZvPIWT+qALAeJ3\\\\\\\/WXumoDvmx\\\\\\\/w2qnp2LnUE9XBUEZYWlvpJVu6hpbrpHHfdeUirUrnpFNTJKn4zKN+CQqlWrjeOZKZi+zonrBI1Hhz7EsEOiY80qQ3Fia4bEmj9Y+Mq6bwxml33JIGGU4J9p7y6uRRK5JLMjGqSw6iuDf9v88ZcHZtGQ4HLAb6DR6BlzDPLm9IhUs9CTLZORTzU7+pRVDvIj5v5Z2EtL2TBJObSV6K50KpMJTWvDjg3ZNcg7UiBQ8yQYD56Xe1MuwdwRRl7tdcSeF1TNb+WaNJeDI\\\\\\\/XvE5CXKvHmYbXbAalPXxwmHFhU+GTgP08tU3hhwVTC9MSvPkJR3y1lqEH4zHXFpQAn9fojh9D52J9xuWVKTioe2eaJwF6OqKvfPpITHbC9sp0KSmcyLN5m5sxobwBYdndJFrKJDwLCYpo1hfa9tsIawphPS\\\\\\\/0ELhPSp\\\\\\\/rTs6V9xVBFUQXBDQEexStS8SxscZmqlcO5iPX5mC5s6+rF+l8C6Lt2xIX0zKCcSKopMhaLVSQp0Rme7jHagV11+8+8MoFvJHl6au85D2znxf3\\\\\\\/iYKLHZzZFQrvwihM1ZKQHGtpGVE6S\\\\\\\/hwItiDUnsjh8oFmG827cmx+bMBSfNMvowJTjpxrHR7wAaBukY4rSOYbYSVJmZuxm5M1Rd5b0JC+OOeUcBVK6szh3+gKdCkhyueNMa0rb8W6mDLXPEk126HFq0SbPlugwd4zLp\\\\\\\/ZMEVHkvNiD6+MumWeR+R\\\\\\\/6TD1nzWw\\\\\\\/9gVj3IGrKa7uqruLV73eAB1ungA5dB\\\\\\\/iafWboVcWnWwCM20x\\\\\\\/i6qfbtyKF4vY3fC+oBFKZyoDLDnJhw4WdH2mstISFcwy6mQzSILY5\\\\\\\/r2tVKjG5rUIWdBHYy9GQAqcTjk3EXUK8dZTFFVuOyjcmd8a5Yh9VFJaDr30O262vuhbiEqg1lbcsTXiRnHXiKUg4o66zXTjrrfRMJlTyWtXKQQHOCz3ME3HvEo\\\\\\\/\\\\\\\/7An2aSdiWZrECucDKGZ18IwVbOGdlh\\\\\\\/fX+wSlt2ucq9qpGia8J2y\\\\\\\/QN5dqbpM8pGnv4dJjdGVKob4oULlU7EJGI4sWcmQT6EcCEwUq0duvS+6y\\\\\\\/w22KJNMN9jGbLTzqBiFxho0dNo6dIaXgfi5lzrdlAsl2qERvlaGDwwW2qs2chagHKfhXzoRzLMihFHuYa00nUAiWsW+H8dcAVLUXShDXKS56NcoXAevFZgnkipGjmj9f2mKBsCGEsEB7tGZk+eeKBSNJ06NrK+x+AeAvsNpylnGxgpFYFEdfosspjMIWcvaMewUZ0c9m54D\\\\\\\/vX8ng\\\\\\\/fgqnv45RbCA+m9oL30j\\\\\\\/ZjF9isGerksokg4ehGZhH\\\\\\\/ivWtPWAxYJ0UrkL0CHYNGiDEAx5WPSvLdBVapsk\\\\\\\/UFGM+oX7qwxa31SLc935RCUl78irInmfWfQz0eLT2Xprl2AQHFl6zSI4Q\\\\\\\/of\\\\\\\/fVGNdmcN90uV3fq8GaXTQOvvhawwOO82GcxB9ibWz4kPZgnWMKv8kcsuyGngJtpciZ\\\\\\\/LDCa4B6EiNVrm0Y5wiF3ISY2qG8ua8lJgViQlOYDt+NssVR0FUcxEEMErG4G3SKSBkB7iYrOFwqsP7m+BO6olJeNAByUKKVO9NnjRc1HGipr0CFkygzFJ0Wqc7aA5rL7MqouP4hRt1aL0xYP7cbA2GslonnEppqnmQJc6PJzYx\\\\\\\/cNGeGZvSeVLBdxCNEffFOFDI5GWwbaoGn5y9VF1xke\\\\\\\/klI7o\\\\\\\/02RZt8MMoz\\\\\\\/w3qmavNLo6bonVS12k1aAGLgnQ1HxH9TBCI3Qr11P7ZjEVdfc6IuqwYb68GTMSwAq9Oxx6I1NkImpcORPrA6miRf0tatFNX7TJAVLssYuZMaAhQrvsg9KZ3WD9pCZm6X9h0wGqxiA9FEUPXcTfB8ecyCqhslHotBtUelgxudp3qZYZs\\\\\\\/7LuyXq14CigEzbuWqPtbqhO\\\\\\\/MioWHoSsSb6O6gxyLKSZkBUkGkg6uC5c5tFHebIbE65H90nef\\\\\\\/wpLHhWcRNhQFDllPMUz9e7kdqNJP9TcXeSvHcT05+R4eRppRETP5qpIGgkduTHCck4XO7jprMiHdPe5GEYdICpPPMS5H6y2xHhQdzFqnvULr4xILksjc7UNtISwORLvLIM6Tr5+JxKC1ZtqZLAa5\\\\\\\/rh0YJ76XKPyqKaBFeDAwfCpzQ8D9NRIpe39VKj+ZKZp2Ex0fvHkXtJLIXi7Zb84dHjsDnDObWdQdKD3pPhUwqnX3zaw435OD8OkJhIjXYY7\\\\\\\/63t+PGJPp1dYJpOOlTjXBbRmQhfidnO16NpNwoD24eXIJEuV4zqIhfbSTeFnWN0q4WwD1rUMt\\\\\\\/01thgENfrleDf40kBpKARski3ge2jYU+SMlhiitPbvUXWHdlRvF6uzYakepCIoMNuHzICqc8Sk\\\\\\\/bRhdY6m2CuPWqQ8CO\\\\\\\/jWewKi6c7Q+1oRva5PDvWByuCnIiBkSSh5Wu0fJbesqf8cJP6oWM10qulWF8Ru3Kod38IbA3WAoRS+cSm\\\\\\\/dDtItlkiYQn1Y+yEuVmZPEE43lCXsZOG4lp2UrRUJAH1wORD\\\\\\\/uwx9aTArN3K6GtazR81RG3YQ07oeDJSc6zsBZxo7StccGlL0fwSe7Vnc\\\\\\\/G2+8tXhXzTFkOohxgJCATBNxFh5O8hFf\\\\\\\/\\\\\\\/iJlhVNtEcQv6N2p5e3pa+kAUGleGYM+N\\\\\\\/RkD7HyAs2pyWhWzkznZt94KVqHOCHluMVfOhHtyL6kl7apgKDy397ekw0JxiXBWHRwyOuru7GEYidvKLafAA2+j+\\\\\\\/9kJhEnCA+Kc3XI7fX0YR1Max4G8GnqLX7PD5d2zyrk4umVeCyuMByaS8pzbF1VhnvcgXU+nY7hPKpXVsKkKVRgzuiMgoPTWwZIO+GkRneFRGd2fR7nZria6moccEC+ZrP+ec4jf9gINRkjOS5ZXOvkKSr9cwNhx0N+bk6sHo4Fk3oNltmVpjuaF9YH4ffRxvzeHuAM1qWgGPREOw+KoorHb9LvoZSqMEJ683z7yqSwz2iGA2uJQz8JnnI0FTOkhlEnzYBUQbL5bilPKGR93DrvcqRe3jBMxhsx1i\\\\\\\/jF3n3Dg6VMe62TMcpGHxpb\\\\\\\/\\\\\\\/bsTtegtXlG1bCGf68BMxPqDMDM0QpJnMpNUoH3ljXMn5Ynwg4h3ML+cphXhoY3+qjLckqgDCmuSElAMUe\\\\\\\/mEqXTV56iMk89mc1l0vflsUoDlUQ0so+qn1m54hn6M5LWJx324gmWJDrwM\\\\\\\/oSxolNqtC8Oiy7UJAYIKuC4mPZhiaFpt\\\\\\\/6Uw\\\\\\\/P+G\\\\\\\/muv8\\\\\\\/YYRZO37I6em3f4CernyKF\\\\\\\/mVnIamTpHz6no0bKf63D5ASeKS2TEy\\\\\\\/uwro5POnubKCaD0j0HdR2mtGKfo\\\\\\\/oufhJOFlK81y2bZbc5\\\\\\\/ZewghkdIEJiyRXW7meIxsLi01xEXZ0CB9rop5NyZx+tfQ2JlGYqXC6eNu0d6f5RplLVvxyT1SUtjlEZZYwbRiwBG64Or36PGLexble18lURsVLlpb\\\\\\\/wjcmyFeVNiNJLRFh99eLOulVIhrk8ATIDrpGp73FgVjDvg4xMtVWo9Q7bPIpvUPN5YkyibnL1Eh0nQnZAjtUPxMtuD1rVbHnl9VT02h03Fm0n7O1GwCy2udehUMCy0tcIeFPYAtlNlFjsTyirEmHPsF2YBwd1uNR\\\\\\\/QHDqgUZhmo1SLjzfvPJF49ktuWs1ovqlax\\\\\\\/XLc9v1QvSyRVZhz0T4hYb9mWUi9r6+B9u29+zuVBjdPZz+xXPVA1fWixwoqRffBHAKjdWpkMMto4KEXPlUQbHSL0117Fk6xN8jHtN5+EMbPLzUnusx1D+QuHizkn8c+ANs6DtgnDvTSkkh4RYQKG6SIiMk6OzQmqrLWrDgIzz\\\\\\\/QwaYBDAH9buSp5jtNYve9jCdReM1V0XSkFvvfeb8dVF2JaJvvoW1\\\\\\\/MzlC1P9su6T0pcBZrfX5Nn41mfVzHbHviY7lqrkPeDc+VyoHZ3u30pqEGV5BuU+JSJBFAugNEizOXgWswrGoy4D2+W1KMxZOW\\\\\\\/JRW7UiqT298Tm+hzzJogKjlT8eb1nSr7B37Ba3vhj6hESpcZStEbcxDaO2ZlFcefXzX5K742enDjYklQIRVF7wMi6TnDpcV+HYwTUz0tgD81nT7DGZtYjHf0CuGb47pozHiGqZJRm9xWeNtvivhTxadpwBBdmGXPVjigEK9fN9kk8DvPkOd01P0hKVY0f\\\\\\\/YZwl+E4l8JJ5\\\\\\\/f+199KMjLdfUcL6q3xs7N7Xg0uafAgWQTl3YlwV9Eohr5t\\\\\\\/kqYWrMQHR8h2KBqd9A2KJbBFRHZU7cyefwJjJPByILPHoTTCmQmCFGdB4g80kP+LCXwwK+izk\\\\\\\/j2ycP33eaeECYRixKRe6Ojsuv1Wo0KamWXHXcr5lwOcXi19sqNJ3lnbVi4PfzY00bFTKEaW\\\\\\\/SiHW+otNz2yOxzFpAYW7yFA92BcXV0+KAWT8cvvKs7IwIw5Tdf\\\\\\\/nbrMZOzTHZQ8Tcn3r\\\\\\\/Nb9aBGzS2Y5Gd7Qb4nZ9jOy4yJt3cTKWrJIpKQ99TvuabH\\\\\\\/QCRImEq5c5NWty6iZ1VGbqzgmO6v1r5f5wbeuNcvQYfcZDp+dcUZdYlUHtXjptKxO6ouD6Gj\\\\\\\/7X3VenM08vBEVBeeRMXmGaLaU+fwv5imKE2F3Py6JPRlaDHL80yrG+3MJZeIzaxgbPN19QLsk\\\\\\\/VG7yPhs93qKSXKkln65qjpqDrMVtoWAmaL\\\\\\\/eLge0zxADqzVNZGOvekdoJSuhVxHnKjMxRxqUIWamXkyZg73Porz3brqflo72uvNnqIH33sxRoHCa08Vm67L5u58uy4KUWiUxs7P2CicNLRZ07kUw9WGV6FUfF6v4wQf0jRDdCw92L1pVKjrPeFxOIqOTW2A4QXUFHrQ2E8ukrtrI2T7NPJ9L5IY+m981x3xZ7u7KZQJeAXzq8zzyPItvH0YeWULBHsqgil\\\\\\\/J3yO5kSAkTI1KLvsFMzRLQXDKam6j8SqmMT3fercvk8bm6IORGp3yx2SiQiF3G+3BhtmFaxWISDEgFGvJGkp6GLyRH3LCuFdQTas1xz2NdvdB\\\\\\\/QhmVrkqYUv9gtIYr9J3VzCIS1wn6J2C\\\\\\\/UtgGl8j7aIhw16pD3l2CNwxHZ559aaHiO4bK+zGfSTFMyrFA7QbJWKxJ3gsLubHpisTQc4Akh2Z4PJU89cZLTdRHMl931HTXNcQ+X8w3eiuSDjFnhziMSm6y7zMEmT17qxqbczzL2bm6QCu2r5hKaYvwNH9WLI4GQ+a9X5iIUoDEJV\\\\\\\/uRe43qq+ZWjY4TIqqIF7TxoFDv\\\\\\\/X7eUtFhJXNYDIZHjKLVD8lvruyzjz1nrZZrXOEr9G4MtxfEDNsY3RDQMIJd5pKNDFVe8EJfc6iOr6a\\\\\\\/Tp4yd0ksZs9uSUETcAOr+1siivMWSMJZgJslYb93GjLtgD6TFkBX\\\\\\\/hN3TiSwXqapt9e8Zf6bs6Mp5CZ0TiLYi6i7uJd4U77ARhO2VoO2IzttVr3+hhqdfKwDC+z0FoONlVqDOJHUqNDS2RVvtluseS3dQdMKIpTOcImPNM1pL7l+NtkyuGVZlAxqL9Xw5EKGej7IOwcxDhN+Fiu9kV\\\\\\\/7CFd5F+W\\\\\\\/89hwvkPemyhxkGSdsZoKdbeNeRpt0V5nrHkzwGvS50cbfKRdgL\\\\\\\/ltvoLWIRGPQY+t3colEKr64YvnaOoJpzTEc1MLN08BIHTiCbjVFmJLBuB9LQY8nhQ0Kr5Q+MTXBzGOVXQu6jb21wymkOJwwR0Dnk3yQf7AHfFEvj2chwAn0+a9WQmzS9xhmZBPSBtQyjB9jKVv7e6\\\\\\\/goPPUJraW++AE+8UpMhvcCT8WjPI4JlwJIjSPlUFJXYQmg\\\\\\\/IMWJGGv8XuCj9c4wa\\\\\\\/rYP2xTPQ2lRrhq1pCuq3eeu+IOXqHQIK3DxXtqCoXjKypwzI7oMMzfsBT7Y\\\\\\\/m3Tho7aHu0wd0lxGEeFRvsnaNTD2o9rXFSlLBwTp46ylPfMfypv0NNiSN9RvPeJxFiW32BJfqgETZsIDNPYNXYml3z8MSnJVWX0EYgCdNQCsa9SsDVdVHYnMcbXzBg+7Aq7f3spQ\\\\\\\/G9bC1JCHXl3jYmH9hQZkZzCWo0vhPht864B+87+50SJ3V+IsskuN3Kb1gMTuAhj\\\\\\\/e16rE1cmf85KpmIlAdttemPgmejKV6iFRv7uUC7AosL\\\\\\\/8Tb+yo8BoZJrdXq1YMYyLzheGip7gx2+ALn8xRwrS5l1HuFgSsiKfPYi4epD7JU\\\\\\\/Co7GWTzDyw3KWGM929+schoHmKsoJm2nriVQvNF+pVwW8h7EP6fDU+Ft3BOXjJrqre5hcT3hufyMTRb\\\\\\\/5ACNnQkZhqLD39adz3PoS5oj1zJHYGgB2cidZkx47HDRYJH7EVxEDwaDFw+g1IfWJH6rrv6r2bCxkDWvuR71\\\\\\\/NK5lhkCR36tsWz8UfybG+Ef73aKQ9rFFEsMDMHtnnNOlLbTdcVvdamAHzjm6T7OtEESulKYR3AjcMAPGwbdMJ44yIZpAQKeQfPtZkOA93dw3EFmJ3LlMhxs3X92dsrSfyAjqzJ0bv9s5Eq\\\\\\\/V0muEGybdnF3gtJzCiXajahFdItykJS3C5I3LG0PxEatYer2PQx\\\\\\\/W7eDMRBtDg9XLmxG1uPq5L87PIiel12BtP3+z1JWj\\\\\\\/8ixsEiU9aVmDxzKcSTnS3NB2KUbD8DkRM32EbRnAIAYPrZ0HrZ3Cc\\\\\\\/bHK\\\\\\\/3ZNe\\\\\\\/9r7SJVSFD7TWUCh4dkejv6s2nFs5QqgSHAZnm55FqophHpQcMeFjj4TNdoL7AyEI1ZcFOm5HOzTSKCOBgisF1gpE668D1l8TtbxWOnTKq1VL\\\\\\\/4BHkjTzrosXsfhQ5FL62xomMYtNUEzaZFY4GvFLK\\\\\\\/1P69kp71\\\\\\\/8rxWOpC4G7gLZY87+0uc3\\\\\\\/lFAhn\\\\\\\/ynx0T2GwaShSl68clxaujYvhNv6ym\\\\\\\/3qEh2KuKIub9C25lnsUKHCR4qtEW6dYjenjChhtBwi6l15jbDIbv1hc\\\\\\\/1JDy\\\\\\\/SEay1WMwAMTADUg4pvxuC+TA+DJ2GVXv4usSd9F6j1FrN\\\\\\\/8Nk3wMZP5H3v7zJznEUdw2eDe183pM4nkq\\\\\\\/WoDamNSN1K3gH5jIKRzADxQ0wCNfxV6ZudK6lWAZsx142JywPV55t0Bdul2LwVdT4Uy6GGLZ3xF9bHDvYXaQjHEuVu57B1fR9aLMXN\\\\\\\/uZhCsUR10RaGnQbIHLoajP2BgmQ+HWS57FOnwZ+OpvjI8L0gqhuhAod\\\\\\\/zy+1ndvbJh68xfDr5g1Hd1n1oOCMjDE0405zJlG3qPhtsU\\\\\\\/0gL8bTcXVPT+xVZIi39A44UVM8AeKdVMs\\\\\\\/cMu3GFyIhZuSL8yMSDvXWWoNUoBjnmpmAp\\\\\\\/pPDPWa7xZot2sfyl4kJqhuKMLt4j+xRVmk2zfnlkDy0Wq6j8Trsdk3Y9l+MUs80FdbAC0nqoCAxknrXKGL2HIgQc9wFNLpjLdoOzmx34+98p9DSEFI2A6XCUNbPZsUWdPEreon3vIfgwS1gK2rEu8tcSxhm1RbMAmYGpphrTFhlZ5GtPUyQ4uHKVc6GifQIgB9QNClXj2Y694NCv6XA0eBHCW+4WSbivVz8q+0sSiRVEt3GS3qaqUgg3d\\\\\\\/wk7BvBgiu6Zjby6LlPZ\\\\\\\/MK6SAf3WBCIX8y3mjMztTeywopdykgZ1y6pgyzoR4ym+dzamrVajTyAtIq3bqbdts\\\\\\\/YKrKrhgs8ClVQSEnFWWRs0JEwb8mD2vCzMv0FjfidRrFonIt6RhgljKLh1W2oQOqAYDuhy+Kp3uJT4XxLJfAzHsKZy8rQdqpTwWe29XHDixingmZTW\\\\\\\/w8VMQv2iN4SF484kE4OYvpD8rFDRMVXlGOVPFQHNy5ZVp8+cGvIi7YMBvjGtTMjon9tAkMQwLz0q5Gl1qtQO0\\\\\\\/KqCgHyIVyui018gObikTzKGkPlr746rmL0QYl7U\\\\\\\/+pOzeHCN15soyogD5yskJ4m2+myyW+zERElzU5CCAfXCdHDvOGephdGjw8V7m7zzStxhulZacUELHUhxXXu5tqxcVaMEDrusO4EYzfbrfdFvaSou1geGv93mu7kGUzloIAeOvQlegOVgfY6ySdjNGWapM0ReriaATMyIZpPE33Sc6VW48LIY5VeeBlaj2o793b\\\\\\\/sP+mOI1KQt3stsNINuMx97TgmFxNarmNRiPy28\\\\\\\/Uls5vi2CeBMAXRV\\\\\\\/OgGpOE9PvTKbUUerUFy8Rna0sulXcKs9ufn3WwYJL\\\\\\\/IUwbZC6DFCZ5nL1ClCjW255fNRZp5hFpXKVz\\\\\\\/g3aEIR+XInQo4xsUjE3ynXGdOw3Ih+qxEjEq7LDVn+C5XbY6wsk8DUnYCasCeI+WqxAjH2S\\\\\\\/jtYJKcTfJiI2o0PbgGT56zqZXZ3hBlbu82vlGNNvbweE9HZGJMOlJvFE7biFqDuZE\\\\\\\/4UTfcsTD8n5AufFBuySLhIk2dTXfTXQLRuMNlwozt7bMmQ3IxJhFntkr03Hm1XLtcMp\\\\\\\/trxdyeOVec31nH9FBEckDkmX5Ao5oa19o2Rl0TzLvXZnnQeH+Trianf0glkYPmQ3+OUVIQ4wuKfCTrByYnHQMnzMLMFg7Ywud8hGdYkJRqTfE2VwX8eUCkzdYE96rYEO9YjJ2AIdxMD68XD4Lg+qUvX5Tz15t6e9NMuTbss454BvzHAzs7r0e728KtL66aMV6IFeI8k++U43bSDE0NvaZeOy+MReCGyHeWs0Q8R2tvOTsGqcvR70SxFTeu7oij22Z\\\\\\\/EidZtoJpprkvvXdF9nYv5JiW02yrz42EYxzVzqZ6LHxOCisw59KmEQrxo4ZkOdxRpUjbeOtSGqx\\\\\\\/6Ym981Uh8P0X\\\\\\\/UBbqK0faLyWUHMofrvZWthfb4n1dkOJPzAjMqUINU4tLCfaB116m1jj9\\\\\\\/N6cDyrueiD9y7c\\\\\\\/3YUctH9mnkWuw\\\\\\\/y9j635nMQy7GrHrJVnLME5MypWRdvBM9QdGXQg27JHsjkHSyk9g3slyTAqt7yr68wIfgd0Hz3RMput4330FDDrZkKZ9mdefbpYmy1epfV1cF2z5MPKliAwa7Htfe67jtXu070Nv3xBeNpd9\\\\\\\/5iE8qvD4qzq86pvhF\\\\\\\/4f9sGdLkn7McJfDpFHmKwoIAIN9\\\\\\\/GV8rRXkq3BOV9aJl9pH\\\\\\\/CNzTVUER7Bpw7WLY2oiucAgskkh7eqJFdDXp7Y0G5MRuoixAZgSAZUrrCioD2mq7s9hBfYqPpIySc1tI1UVzlb9pPSb\\\\\\\/gPJxz6ggcQQs30pimomAT95FY3o8cl14elBjMoi5gWlNSRHnIL9thaPr4S9siYP+2UfirK6LUb87XxIx8hF0O7yJFM19LxgtgcC1lAlhLXAH860fC0GKITUD\\\\\\\/Zd58CnmRfHor5RBQfJdj8eygaScXFU9Gvpoyt2Yq8vrWVxZNOCs0YQC3tIkBt+VpWl1BgzE7cGsivFZs+U9frNtSPDNk4RgvtNiBffwQ8dmVGub81ln+bGKEc7g2d1tn3\\\\\\\/VgE++RLiiEG56XbUl99Ym+BKbsb6O5a+qM2H4p1HoYxO41mSHoIZLeAfgupx1s0o1sqGOMJ7EHvPOpO8N9sHNHgfEJ5YCNJjZW+dqD7kDdfbRf4taAgtN+i4tJxEw1DjxnRtQwbFFpWAIm1\\\\\\\/TU5uo2Z7I4zIBOoq92gGIhWngHy\\\\\\\/dGwEyxjgheFvBTD1b3uiYqbHELP7xiRSF9PfyBgSx7clbiS3NhA\\\\\\\/TEOjPM65hz\\\\\\\/ZFNMYbPlbB6Gr10Ex9KkXmMWoLQczeLrm2zDkYklkjlXzOEKZii3kqlWQtmbsZ85FuVeCIOpAqduivH+o4oD8ZInZZm7lXjI3RiiYSAzQSOr127wwpTW4M9MXjI6LMfhFpglv5CQ0Fx\\\\\\\/Freb0wyDJ7LZJJszHqmClklV8c7NhBQs+MEl9ZC2KqzBNuognz\\\\\\\/Uhdtd2y2qHs37z\\\\\\\/eopmTyVWBAcPblnAhB9P6xVnziy\\\\\\\/6S1r1cOofZ5hIDSrJ1QtqQT2flVXpkbxmu5caQqA4F+FH2e+VKXLjRJaZ3ZauvOfSl1M6+HaPwEsa6SsSRfMs4IcVZvsNCXmHSGWN\\\\\\\/EshXfZTPzgz8DZI1Ioc2lEjRZ1honWQB1jAiQI6z2oCkTvpLuOQmQwd7bqA9\\\\\\\/kPhnHOgBsbinhiXiAkW7tsbk8zaoM0ev9EjiQnZ1+BiUS0aeDf0zpZt5dfe1qDzoPasJe3Uxhky55SsX56CRyROsu6a3fsTxKscBqOWSZJ9sJdtMbtLAD2nh6E7DD3+jB5+m7OHd\\\\\\\/SWRUaoMDPk7HxEy9l4LjOBbKb5polfJaAAW7gRDyDJ9G6W09JoUbjaVCnaFYUsw5K6mTBTPJJCl06qrRviC+UecjtqcQ7flYOGgGMjs0zJvnUcI2WYhR50+4\\\\\\\/UXesWqg6ElP+X5FANRQ1A29\\\\\\\/jW+ok2jx\\\\\\\/WKKuwW13cSZJE5s5BcXjL1nocRz7CQYWxmv6tw3L9iVQyAgpgL1kezsPDW3R9bsERlAJlZnkm1nVFEjBMciEtrJkYEjq6BGLgOdKYMWB4QoUNQAjUL\\\\\\\/SpuOQrwDbSfFwdldKG8eUTjRqtsY4Snc3IzxdcK7Xr1sseJitj60KnbVISQu7KQMGaId5VxFP8R2i7MYahIes3ls+saNOBxq9ISKye8aDYdr\\\\\\\/+sc4xxF190YypNSNqJ8mKlcyUIf9RZ1i9RVL6C3r+L6eVG9E5hud\\\\\\\/6lB6bLZ4pCz3QJwz849J+Jc4QRRMRze9do0JvZjK7kVbMM8bTW7eQCzIwKBsEx0LEaNyIz5aEIwuE68++COao62jsoHHGrqdLEV+kLWu1SiV2PLYlpgoPey8cfILZcGaZuaqS4CvhQXPTeg8pRkqrorpK1I1UGOdP\\\\\\\/6sVLVhYOPiCQ7Wc28dupvN\\\\\\\/vjkA8Il9Ze3kzRGZazXXs9q9XaS7Zz5fkoC8jP8u5oeKBe+Ag4BujuGQR\\\\\\\/QfJBdeK930rG7OjNrHQK5jRdkgsjcYDrhgoFr2Rr\\\\\\\/bQKS2ub0bjWP5oPJyhIDyBrenwQd6q6UMYB\\\\\\\/Hg7Rix1+QKCzfg82j3+TQH1YU10hsAA27eshZ16DhUl7V7M1M5ufQMJUMLEaVv\\\\\\\/ZZUCx9tel5SOdPUIRhvatEzDX6SM7IXN30V0H2Mf4c4SZn09pQLZIYIvZ4bq8XEUQ6w7x+O+uXDirvwcBFrqp3tpLIH1yPvM4mMw0pbwfQowdu6+HKJlYWtPQJ+1nqDKG1cKe2Qti+jA7JW+aOp8CBSwXU0t0NBQY4zJWbb2SNU2MoVH6YZU+v\\\\\\\/kCvkVXsbhvXWNnZrFHeP+aUqMVpJ9BuMUYuvf6xxCt5m94Om62GZHEO3xMRGez5nPHJllCmJUnZM2pWY8ELh3fGclemEMMd1BqEaMTbfJuAgwxjgnODB7Nbvmtsg\\\\\\\/iQ4dtYZxZZzSmllUzla8dw1maHJhSL0\\\\\\\/wQzqTfAWJSzBjx4z4C+GCBhd1Te+\\\\\\\/BJq+C4ipku71T+GoI6wJlSNHJPdXPvw38JcKM3Wb1BRADj2\\\\\\\/cIQk0TWrPFC+oXErVmMVhR66\\\\\\\/ZImrUXTDTM90QqcGi7zvtJk+mn5HEGCSXPrOmY+S2svH5td16bQXspJx8Xp2B0NZu2a45uCLZpN47CS8wIJm5gfLxw8Pupj2g88JYbOuWclZPzQAJvwK2rmrciZzHk\\\\\\\/e3He1O\\\\\\\/hUF2GmFGyyKeHVjaUyEPcRmQ4m0oXcpkqB1Dw66m30ahpdcTlx4frMjCoxSqigCJdYFD9+Ihz9q0VR0\\\\\\\/WfWUXFmTywLCdDoul2wtKHZ95T70j0hsZp0FLXYJlEDGkwgec48Ai9jUVmDCllBjimBYhtcr6J\\\\\\\/N3JE9tw13x5AOETQnxHJoZLULK5+ZKIPS5MVYiTyoSBfDsMZWwDR89Y1f8vmMwPjgiq+gPgF4aLgCYU54DnfWMG+hPhX8AUQU742Yfa+9\\\\\\\/o3EkZ8j7KIPtvGaezIiaIZyU6Wc7BOmDsZ5JD4Sq\\\\\\\/dZPC9ZxCC1EaTXaBCWex6lJtfUwbRsuTrkxm7QTJkjQNl39HFFj\\\\\\\/sUVtqeZPXBnYeO9itW2x4hbAUSfJ8wnVUzUvimCQKtg+k0j0MrDvsdCVlyZpXwZ0WV9jKaJZnM4OMBTrTVthMswsCWtNWF6JsWhahvW84GBCrgoWv0MfdZ4mwtOL8iG8qD0iWS8Owbf\\\\\\\/MGCU71IpCc29iZXVF2ZlCaeLvJoU61JWeJpRYZmZcLerPo2BRCtOtI2tprUacKbm1AqO742\\\\\\\/FkJliKLqhSTAYSMqVJ80p4li8lW57hN92udlu1ElXbbmri7g4lRaUHkYLrig+RjDnHYOonmYC66dgjvOFrdDxT5AYbdr0xY+xGs5M\\\\\\\/yuyGr7u+86ytFsfrBwWZMI\\\\\\\/wJugXfDAMqlB4dsYWMZIGexjIqQvvhHvcDf\\\\\\\/15WyK4sNLtMmPiTbzLu97JiAFWF8AalNb++TjeMpwckxJ2ghbAptzndKNLFHKVK2KgdxzXEV0p7chUzVrZwXU1fdVhuY1x2nIDyRfzT+4ezYAlZRQ2cYZbKvDF\\\\\\\/abHBrbEH9rPa5\\\\\\\/JyLiwz1u27r4xOQg3oiOKKRNwzOIpfZEKj6H2Sk5fR7NcP6a7095aUpJdMdrAETQ1X1YbMvOZboM+y8n0Nu3HM6q\\\\\\\/Y6NFsa9\\\\\\\/GK\\\\\\\/UgTLXk7O9heSzxYyIHpuojeBHmP\\\\\\\/AMfSHirfyUHqd\\\\\\\/gNuEVcdDd9RTcMjCjMjBUdzTjxDFCx4nowdZpa4QI6NVyvysC5EjBkne1773toYCJMLybiRCdBKRziQrH4gngivS+BkM6CqkjRaTQFypvxorwMMH6G56Fdrv1uVT1YZRueM2TmtnTYZigST\\\\\\\/rszzNsBoXxjKb1wkVOmHv3OHyfDMsbfriBsmdk+xfvc6RggLzJgwlIu5lAHdb1TEpenqCL+aHCh9w2YBrBLGgLs5Ry3BBSD+BGpKaq6u9UbS37Vu96h8OydJ0kQ+AkYq2ksgr6XaZse60mQ5oOZzug+FT+xTGn37enUsGCJ0CcrTgZzUCyDt3DnZBdd+Fr20GdzLfJ1XxtEsX1vqa+4vNH3Y4fBNv4r6\\\\\\\/dalPhvSCEkRpJ0rCQtr\\\\\\\/Wboq7isDSrkAjbAmYKNY1Mc3prYJiUwXUEUuZN4eoi8ulin+BMMPgkVng7m1B\\\\\\\/\\\\\\\/iFIZTg7Jx1eGNx3DhGl\\\\\\\/yyFyH6LbzJAHF7H22fI0LPZe8FC0x5isU4oaPkntrwpGMEYOw6pDVAizGqQ4S1OM2n6VuYIXZB5SW\\\\\\\/TqQpRbenImpOLpXflZkLK4WOouGLloc7Kllld9fYw3xf4ufow2zHDGSp2TnAgZqTxw2AB\\\\\\\/qE\\\\\\\/DIYb1uriQmMUb7cmCWUojY62TCoRJXq7lU522lQ0qgrrfSSnlKmqlxef5KsWxJqDhPHNQWxAQUxWk4v\\\\\\\/9cM5ESE5AXx68hXX9g2A1wXuGte54RzBziKB57KZqe4cdB2H\\\\\\\/pBavqluaJuM\\\\\\\/XyZ25b9K79gRfu8SEBJ0Rv1Bsz\\\\\\\/AM8\\\\\\\/whcKIAkumA0O\\\\\\\/0W3sdzvfK40SSthTZviooiUQBnrFeABMqhOLvrNvBYOYl0BTYdGHttYGXzzUoRu5NFsKkS0l2\\\\\\\/WC373hNDNDf78KnO0lFrllviH+owHME9Zp91AiK2st5c2xsnfuurNzqCs6wfRvN2Qkema+U6127qbMXvlfXyzyWKm4u4mpfxhV\\\\\\\/OoXLTdVBQtEjgdpFM\\\\\\\/y9N\\\\\\\/kmxEAnvZZIRq4xWO7mkKxK\\\\\\\/w0tYBVpdfweeHHjuB65vOs8Arukj4V7y\\\\\\\/udu6Ro8M7grjEd28pr6QasAle6VorUxDfrk8V0aEBJXC9OvDuf7ahvIDO\\\\\\\/JohI9aNtzhao8Kla9tSYAQW968a2IYdN+wQtOnkwiXL8Wihk7fZz2zTtMP7cuKNfr+T\\\\\\\/sgEinHRSp1kHthi0J0BsO5n63Uziq1Ku9Yq\\\\\\\/6u+p7hrKoTIrhfEozsLf7csHKT+VulpszuiAESz1wVaSQAtzKhDdqc69AVSw2W8IOuPcfpSAyYNq0+4tZ5gkc\\\\\\\/pdm37yojWVITpnv2ETwNMEQPuFxCJLEkQVjhWoOH2F8yWJTm8wNorzz6yRtbGY2+HWqWlcEMZEPgYTjhu6hqXpHZNJct8GACoyV0TyPt+t6Y+mI\\\\\\\/an38xo48qaAIsIuhTgVC1aOahZBGQ3rM3hx7RhlWoDICGJw6YevzMsip9dCF\\\\\\\/hj0hph\\\\\\\/LzppYbjpBeMvWxd9k\\\\\\\/d1Vo66\\\\\\\/zaGbEMhbPxZwk2VFbLM8fGrS3KpOrNqPiqflDU32MoKeBOJN20fuLDD3gFCLwoPW\\\\\\\/ROXwgA3VLfOtalXJU2cEN0hgNBmejA1C\\\\\\\/oiFsPBFVkyFIwCB0a05dS2+JJGfF1T\\\\\\\/gsrHKX9PVufdjiSqPqZjN8XdvVufYqAIwxxSgEyY3vmlvhs9JUaVQfyEnYIZh7UI0qwd0BVi9JwophyGxvLtnrIqwFbno13fbTQGOqDOJpaDxGaeeFNW4+Yr1VBZSqP+t0NoVxl5YoBgd5fwwL52kr+Zu6RRjn7OMtDxO9NyslL7BwtHtefj9NP3VIv6JyXDGF53XWJsCjCouJXyyUVHu0VwDIX3ZVK2H96gBDTLYQFMFSiCM+P7K3Ikqb7zEw+wy8X\\\\\\\/FvdR7VjbI7z4w7ODx2QI0TaE3kTGEV+MeLwnqyLiReQvHb3Uqk39nCeqN9KzLdNHyMK\\\\\\\/aFeiExFkFgMyU9ehUePuEu6gA4ylhOvXliQm0YbFcYAw0flrG2WrLXzXaRuPIQfPIo6E1QoTOBmOme3FGfYO4X75eNCgnZVJhJFXJamLuVrn4Q0DuvF65IjKVZiGBxLug6dQmSsoqBoCq1nE7GItKiaSE3x9BS0d1Oxk\\\\\\\/qvrb5C4hM6aQpWlzsuo3nlftdj3l3crOc2oItNu0pnMaQqoG6x6A2mv1uysyJU7WobeeSn2pb99savJ6OxMDWI2aqq9xYLKkczssx+kh1XsKBhtfsGYK8Tg+buvtJ+sgrTO+CBz8A+DWf0soanDJygprnqsf6kb\\\\\\\/c2qxW0RSFQTEV2T96w+JSuaDrRfIk5kFd5dBo8rOd\\\\\\\/rUauqArLpaUQ\\\\\\\/wv3j+JQQjuUgUf48EPf8QeGbc\\\\\\\/xaBCFbkNlVJ2OWEFKKS5PAafHSDQIQvOtAKagWJoNoYmqjUxwsyQ+YIPm5Idt4TaRIkQ5+F1E2AP+o+zPUtRLemIbfgdiwvu+AMKA8gEIHOuS6zxf7jTQGittdJ+rME+2Oe3iXwCpmF2QSajMGji5pMiVYiL5gex+7h\\\\\\\/W7xdxcu6LH8QHqD0LYDfYSC1k34lI2Pe3\\\\\\\/j8eTi0KvUQP+CBmtEBnJjD4FJfTsCD1IqoQof2wIg+\\\\\\\/6JN7pO13c3D6wDy2f2KLgi5510iKW+tpyGm6dZzKUNLLdnHLTqBb+iOgzEFPgavyix4+pWitoSN\\\\\\\/XPpI5HBmw64aJmyUYvaIIORyer5BvF4xjd322qS8ZF+PK4LkF7g3rAkOTtCDs4DceF7nNlFM83RNU8DMqEnWdy\\\\\\\/Zu4CCdcVkbLeZP8dlcVXzovrRnTrZFNmoiEhq\\\\\\\/yGuGPtJnGGgzrYzZq9UyfQkwCsoIGhPLRRZpmgxMJu3S0oDRJf2J8uexq9SWxPN0vhyKIP7JN2zhC4w\\\\\\\/02PcXoFpe4YZfIEmLBH6IeC+xeceH6wmbqpwSNmHFFKEhicgyYGll\\\\\\\/GGnBidjSLSqbKAWTt\\\\\\\/RBNQ05resYpGcelbIN8CmioY7MFSMHh1ehXXGe14QdyUrVtTi7LBQtApswNmmOi1+XIOBQv6SVRwygsATW8Jg7Mxf8JLJnmuW0NT+Vm1SoafXM1YlF\\\\\\\/iTP03caHxT0wuL5U5ChS57JPRdeaY1Y2tiQeD05V4QXEHeH9\\\\\\\/ps7bkaT69b82Y563oYldDa6mZqLIuVM0hAoCwXFJv6R4XQx4pEo6Ny9ZIkhxmMIHiYgp2GwhG43QAC\\\\\\\/EZA\\\\\\\/ddDMGSls9\\\\\\\/rkcInkhaMT5KCv02hZ5aBMfeaI2WL3JW25Y7lfLV582SpXUGjzDbTExuozgMcubiLzfPnqjkVNahWMpfGb5n71AjoapS\\\\\\\/fBvNCkpzGokVvQO5nAWQbP4LGxbIMfVxVdbERAvj3YCntrvnPK4vI1h\\\\\\\/pUzAO9O3SI8i8lWPiNmYpKUCyBkBR5TPCZ5y3oZGDsrtHc9NorhW0vTvmrbxpqZOGbxL6ztiCtdfB6eTAmDpmDBeU7tKw4hV8euRqIrk6ptmrL0JtGCYfkRu6+01LedZWV0MMv7gtUhr+ltRD1eYYEa9n\\\\\\\/H7O0NvTVCt2VlvO6fTvHDgsVPoLTt2Ydgykkp\\\\\\\/ltv3d80AufjVkbEDg00xYT\\\\\\\/NgzFRT5n+it\\\\\\\/NpQaw3wHC9Y06bTVisSQBzpYFLJTh\\\\\\\/qxc8enyHf3hd+SEa+EaO4LLTUHubIgWDhszwlDS4Fg5R34jPyKkK0ZVW8ZeSnMRGI\\\\\\\/6o8WgRCUTY36bMnnpMDs9l\\\\\\\/dlUY5ANMOD6yazdn\\\\\\\/z0vvk9I9Uz65qQGR+jifUMyT7T\\\\\\\/yvdxPKbaSSOT4ToZx++II956Jj6WZh3ihR09bZU69WUyNOOHbqmZlHTNp5BxeFc\\\\\\\/303pllVdHj\\\\\\\/AUICsaKEtuFQKl4UbjxbxSdndlnzcflE4fJO3fxfPxffDtNTSqYzwx1YwfKBddUlIYAR+2yuh0Cmn0MokxWgQor\\\\\\\/bjgjU5nugbGQqZNv44isXl+ntTSuLcYwsfHaUdjeM1XK5ypnEPvxI9bnR1y18jZWa9UDWBAJJrTCiRoLy+HjCLhBeLcjJNuZM\\\\\\\/BqhBK09AkktmMljHbK9WoimM9AuFx1N9U0NXyzzPDyFSZzzj9IIhcKOKJz2UwUC5eJW7+Nj80DZkWEtdpP\\\\\\\/p5qowXIezd+RHZYNM14luwDjvVRRjcHHEQ+tbF2GJYEZKOpn8TbNIgw9gGt0MwWegnWAKeCvupArTO+O0YKaU3jkP6vzrqNd4\\\\\\\/awdlIOk9TK43CC1MkzeDK2gn17rtKpq6TFICmjQAuOBCmWZLmTLTHrIxEvMP+u1KnjLA3usiEuAvvB1znIjv\\\\\\\/CzGHwJlFUbUMNZ\\\\\\\/I+9ufCbxV0bDswWhCg4nVmlviX1u9Oq3kd5\\\\\\\/+3QrqGD8v\\\\\\\/We4lti1sC2Pat05P47DNS+sB4i0ic\\\\\\\/HqOmIIqr5gn4GrSbQTx6BFBOAXfIiiUwEuRI0zk4SzGxj3oqyzazqP9VdqkSgXT4Z4x7nsTsKQE8gdlAN4PlD\\\\\\\/phAbv6Z\\\\\\\/t4\\\\\\\/jvPSRzObKEqZBNOsQJMklMruQ8RZdozmTfvXCkzIEujAQeIDn3a4NBB4sAWCkleHzdmAOMWDVqKGf6OXlcoBD98YJeBZi\\\\\\\/86ILz+jx54NomYG9N4ScTmJUbUJni4ooM3tNvR9v0ZmNyC1rlTE3cTCBqZOll6Txv43qbEo8Q0woUetObtBhBCsjSLhcvvTikOL4+8F8\\\\\\\/b4H4nC9z9fG4oMXYX4ZOMaNDjUR\\\\\\\/WJzBkFVvm1XmWnPdK41sdG8G5WA27hl\\\\\\\/B2zeaysu4pOAmv455Uk6x92kvnzEvNGSQ7b4MMsLhNzzuu8MJzDkwXaRuyfVm05bFblQAz0p\\\\\\\/EOfsXn4CWBZY\\\\\\\/GR9hqC671rC9P6BBrFMJxBIRNOhSegRDrofI1CcY6h8fvtu0a3Iu7vxhoydr9RKEVVwYieyx6RjuVCzthziYaMu2mBL3H14zwbREvrxJKPnrvAudTQMbxVSEJO0k\\\\\\\/5fzeRwCxMtPP3\\\\\\\/aUccXpoe6WVyEJlA1ukkZD\\\\\\\/0g2O9ICmhTLBA6YZ1b5eXqF\\\\\\\/yRrwNCs88xzFrVa7jCFAlr2tiz3Ap5wnPy7tPoBawApeDW400bEDl4DkEbOIwUYAVzRrElUUaalpYhyzHoTaor+A2X3XWEKWtggdlL8FK5NX1NplIw7cm7D50DVewXPjm5JpoSKh\\\\\\\/Y9FBx0VmjJYuzKHr3DdxaYTIGRhJgprXS\\\\\\\/LkbBN6QhcAxtr1tn9VqSDa+47FQmVSXXciHrWiZp4Tkhv6+5ZXgo\\\\\\\/4+kR1Gsxw96OzWPLwVEuM3rOTQw1WR7RHdbpoVAvOZjMztKzsisxYWs\\\\\\\/Pu+Uawdb7+1pp8Flp1XSkMBjnna\\\\\\\/yxMWleobiESH3kKFq7T48FiFF598fuFAxt4ffSy3fBWgPBHrRGkCw1jPV8T1ia\\\\\\\/p7CnuhXKWYh9iTqK\\\\\\\/lfK5c+3BW7ncPobfLc7BQt80+Y46tQBtnTpqu\\\\\\\/iX\\\\\\\/eFLxB6MEjV++zjf78bVrQALbZ7lXemabmY5vublQMLjlFeg0hpGDoD9D83SSAxI88PdvCTkGolK966aL3pDaRRvdrbQ7GuxyZZJZvcRGJBfEKMbdawg7fVqyx4ogpXn0z0aX4x4JRwP3h\\\\\\\/muKeJdMJx9bYJZeDakZjjPA72WX4IxvmI\\\\\\\/3\\\\\\\/IeZLgi9eu\\\\\\\/thmA6IxtF28sbMFiEv7KBbxgqBj2+j8ak\\\\\\\/cB5tpBYR7QVWH64dFQ0MhBscS0ayMDlKf9l0gpBHZqp2Zn2Wmz\\\\\\\/+zdPsAZxn+ZbxsMr7haJpT3vkpvRyUecYmegRtQVlyiNBLhBpPPbKW4eZNyetPd\\\\\\\/V3ZxL5PA2Lq0WISq7azoUPBi49zbF+\\\\\\\/D7Z5LF1kyl9HduuH6U7XPDzxjbPxHjuQJZEr\\\\\\\/ZigeKSVez8QjBUgHBOLU5leu62QGURELyv13BOzSVTgvvI95iUqJxfEVIeMe5\\\\\\\/gZvhPXMlX370T2PZW+\\\\\\\/WMS4lIaelpLdI\\\\\\\/B9FQutT693h5MCECmwYdy7CF4bKCRI99vacOLDDo9P75932lAdf1sEpqUph6kzcwasMQMvrTdy\\\\\\\/eD3gPFAVEluY77YOaHm2oD\\\\\\\/K+B+AxMYN5zE0iGHEPsI9wuIv0407O\\\\\\\/yy2kpFbVPLBo8K\\\\\\\/aEgIVMltF3LkMV+LqHo29NJMv1U7f6rUdgmJTIfSY1bQYIf9\\\\\\\/OHyqD7f+NG9K1ISdNGEiONa4rKhR8Q6aZDVYKOlNrZAEZ9SQ4wMVPzECp8YcBmZFEX0l65+JV3SmAY8Rm6CDEcdifJdMZQtwIj1ozD8AGWGA7a9yaRWvfLkMebxMwvUpCrAGV9H1dnriGlZ5NR89TuZfxf0KsGgd0\\\\\\\/Cog098r35YfwHxFEXC9\\\\\\\/hiCQS\\\\\\\/LFG6kjO0KceuTCyx1XzbicaTJLbVbkuqMEFRmWOLt+z2CTaQP6pSO8b7RyPGHHTJGNWV3cAUQOXSy9N\\\\\\\/Scud91P4HSNQ0K4w3B0XfbzuVEhXoaIRo\\\\\\\/BjVKp\\\\\\\/zS\\\\\\\/JDqRo00DWGfOxFbPAEpHv0e9gRNTVIqOf3CMaLmaeMt9b0MQe752ss5eWm\\\\\\\/ZHuQYoAdMMZIkHHjNOEElKr\\\\\\\/txotAryK9gscKmvfZQKyzhOiR\\\\\\\/Ps8o\\\\\\\/ChxGe6+n+btsyWCP2Mnpvy4LcsMLCxS7aI9NqU2tsJ0LaaH\\\\\\\/dTdueqp9bFQBoPijasaPPzTnTeMH2D5a\\\\\\\/oRb76UlFz7ovfH92MCeQ7KEyYzqsMjGx54B9cJYsxRVI4vX2Srdo2FUMuL9hoCzBHfaOp74S\\\\\\\/0kuzxlrIQ61Mf8hNzppy791mtdM6ptFQUYq70VE5v98MO1uIyYtdbaKeWGKZSlERE0LhyNDvq9zYhvwYZcZTT\\\\\\\/oGTja0Fj8N+4QB0ii6QnwALaQ1BaBOa8JIMb2vrjwLfCgEM\\\\\\\/eD2IDmOItcY63Xxy5ZQzeR+3Ir8ilKwO990K+hi01jqHQylGS2bNg4kyuvS+Q\\\\\\\/avcNnxycXXfbPzYxv76bd09TCYMxpkCVlzsCg\\\\\\\/VmHpyf05QYAyvARNdvcKVCT4Ydhue2\\\\\\\/IqQLPuZ++wFIVCo+K7BduSzrH\\\\\\\/PHVmNuxPEGN3EDRDey8q0oyYqBxn3JOMCnx6U02x6H05XkjuZXPSQZDuZThc5B8+R0PBhL1nBLJs\\\\\\\/P1ppsL6N5RwWi12EOqbPZv32mZ\\\\\\\/06P7pQM9lfSKgLFR1\\\\\\\/g48f1YPzE8SDTY\\\\\\\/cy+OfGWJnqyuCCmtdx1Z1ib3TC5O1ZaPeybsQdbTSBGmVjuuBFrK+qjg7eFB6CXSWY\\\\\\\/pA7a9bwbXLRCHOo0heYEb2+x3MFD8yPZNMVfuzXsarlhnovWp6C2gqsYhlvNWFFtfrETC6cu2m0hwBsLzp58HSCvjfmCdvCJXXZGh9XP5ZJ3EDsOzR4a+6SaEoagVLFJ4rGDvF8flPfcVHquqUAWcexeGORtR\\\\\\\/LjR3G7nEjI0OMBytzcYrr3JYqZqBZohl\\\\\\\/YYx8dxnu1SNTrHssfT1lZW2ZTE07RLMRQ6V3XoXKAw520XHb8Qw7b6Y\\\\\\\/ljKdpHIyhq8BAUDRqgPWTxM4+h3u6GXmelwTcdzBCyx+023u9pWPz0rMikYrv1wxafuZuDd3pWERuuVAP98Caq2dFr7kMpciP4J0HBRBn4FXrKAXcjSbnhfJ\\\\\\\/nOzy5qdRgvYIjoa\\\\\\\/K\\\\\\\/O2GGxG1G73S+iZ5aAPhRYMgXoBgI5FdmoT6uPsJzTJZMuYhhbGpszAketwfwdStJGv\\\\\\\/Ma0\\\\\\\/sfqiDqCrMe3yqju9EVEfQlHYXDh5CFaPQR1KCIAaq23cT1ICaTyCHLHe61Sk+Tl39PZYTpaiM9hvP7CV7kEa3sQw4IqT5UnDpcd\\\\\\\/6Y4dTnEZCA9oWw6PofBO8A+O3MFNx7yrFDtppIMo8IVgQBm5fgA1ysbZBO8+4aUzbAIKqWf2vTB4IeiEux88o1Ym5MS8qSfnIO\\\\\\\/OPu7Dji4zk1wEU90E1qfnID+O4iJTurCSTKqlzYibee0f\\\\\\\/qWshjiWjea98mpRLBZr6ZQOLVKDdBpDnVGnFfynu6pzX13TjoUxzztUX0kUaWD\\\\\\\/LiSvE4bBCGD9f8qZ0723astSpvTxLecOvrajW13alY6k0Y8oiMdW0vz4NigXC2T5TlyS7tXzxJvPA6EQBpDL5L87GC0Ug8QiCJDaY+zFJvXZpfkw3cQpCQOT7SjjXddh0wnz+BIgCW5LD3FtFpD\\\\\\\/U2j\\\\\\\/eNlPAHI9aizOHZ0Uoe5W1LNmt2M9qPJPnCEza9lz9yhiwcyl5vt9wYpOXRB55JcM\\\\\\\/Rj\\\\\\\/TkgJG2WWzs7LUWcxxDGwg7FtBBhJic4AK42GoQ8uLh\\\\\\\/am6OJo74\\\\\\\/bCiOTRaMp0d\\\\\\\/6IJiqMx6bcgOFvBuJlT7DtdqhraVj0LnOuhyTTfU2HbCRuFUndZxSpmlQh3ZuRpySXxFBirhsSyu5fzgE1FOIy2bi7d\\\\\\\/VBtqhY3WaqKeTlmCl6PcFjX8biMx7ozGd+W2y3Zlm268PjVzFhvy7tUXfBs6IGsr\\\\\\\/xEEfYpJE1977eifddgfelJ4SZK8kFSXJDwD9HUIlTEY684L2cQO+ZCU2gK1LabJBVS679uv7fo\\\\\\\/ehezgIvc63l6jr1O+M6bhvGP7Mgy1p\\\\\\\/5Z38zwpRzLrB5AJT0rONBVgFqmrYPDzvngkv9FwcIqPdOuZqEs4wfn9QyaQCgFOJdWHbmG3bWrxeGFZFy9hsQqjH6lBI9FXTZrk8w3Zo3Sz86TT5YDE6zpLSzLwopPhdu9QmtuAtq2FczfgsUtKYa0sYox127Hu2gBA674Xu77OKCq1o8ZMBc6RTmr63vOChvfRM9flr9WzlfaAtNMVeyBK7+abN0frjaeVpuu74tUWv8MRHxvOb0v3WscGvsR\\\\\\\/Nc5bom7RbQNdtwIXcJRMCfzgnLAH0aKcSmN2dvshdX3\\\\\\\/phaFflvB3qUy7nXjr8PnZ6bA6kWc27wSaNFo+6u2pbFhLyd\\\\\\\/ZLyCFN1MfwdApZy7VpYoQStdVJTn1A2Lt5LsOm+RyVek9U4Uipd2y4bLlrdbHMmDdfFxfILT4KtPF\\\\\\\/7M2dawzcccb5B+njSvK4fYG4R6HqWcGHVrgRxVptbQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/H428IH\\\\\\\/LfdaOIdooLc2xQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002296561135aee0d23da5c0548f1e36b0bf672166ccf83b216162397fda9ff828d4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oVzEoOTh4eOIae++QfdKjWTHgRrrhL5Omr8SOetUQjTKe6\\\\\\\/h6FjuTO4DtFnP0uxiNLSe663GYrDJqGSOiIGZtNb7yR+nlCszFcKVMIg+BmdTQ50EfxgWLzWFxBCqGzcrdUFmESRowe7Je2QlPZ\\\\\\\/0hF9yPEjDsnehtz6FYHEGuv1R05SFJsUtfggGsUWF3erWscHv7lsyv9tKU8u5sZuJVl8ijWLEQy4krkvEGaqiKIwj77IUQ8v3XqpPicSwhNvz6p9RaEKuxWFmPXGTcNO59cYoqywIEMqKeSnjlH06g9NWUl+8LJkVwA3\\\\\\\/6RlGLpAgnJIexnU26ajjE31Vc4GpQYhtU\\\\\\\/T86FpJ0bkdROcftHOhHGtbvc4bNxs9HEc24NicJT6nZqUclb87e4+Urd7TP9qchpU2aVEJb9XNIeJ7x6pLRFOORlOoK35ztTH8bXFDUpw0HnPL7EJeZBIG7FFad3uCnmITTk61Z8tQ80l414I2VCyH+1J7HqmzTfUGwxvL3K3d2KVbbBf4VqNcBuo6OgCPM+wpML8UO3z1ZaCwCCqfo9nrkCK8XXsoooOppGjviKbtn\\\\\\\/PaS\\\\\\\/Dd0nuvV9ze2El9lTLXqV3kzP1U84cq26RTXFXSmz32KdJo5cIm8K0ze8BIBXDq5VwET7JBmx0d\\\\\\\/jBsEUGuslNWv11DTmGTIT+IMIFtRZrBiwhjrkJ2YINx7RUaHt6A9evinzJY83k9EAnHhHemJ+LCbJhRPqddhE5Nyrt+DmJqByTsUQ+\\\\\\\/2dPEA0h86FvmJCgKbm+82dFk0EHHgC67xJk5q\\\\\\\/84kmlWLswAAM\\\\\\\/GrnWVX7n7\\\\\\\/fDHpzpKDO1hxM+pBJJP4UhSf3m7f1kae\\\\\\\/GsoZ0W77zQ9nuehtY5LpL9uAp38JIX5rxSAtwzM3zDHqAosQC4MIxwpiaI3VvzhQq\\\\\\\/Db0Z865\\\\\\\/9IgtQsH1Yn86fVj8BZwk3u4E9dc\\\\\\\/PpmYdJrJNsi0a43jRx\\\\\\\/5tSCC0GAPPreC9qZd4WmEV+dfWESfbMqpmf2Pc1EXq8Ei+CmNAnU9bnmqhqaR2PF4PvRLylXJhA5eUunKBr6GT7xEQmMFulDthkzOc9UzGF0nRb0lLQdXo0PLYY2gl+n6Nt3mYdVfhal4nkVtHgdn7tVIR6e1C7MLbFDGAOirV\\\\\\\/l9ypYIbO4U0onizQgCKhRzkCnK9tmnFL\\\\\\\/Xdhdlv7AqDJVqemIoNRaGuqXvkx+K+5UHpnouUKluvp360YocaHg\\\\\\\/pqafLgd9TqpM5M6wRpm84RnX8mrH4vAc01I4wFqM5q7KwOGV40hXRszANRzDfw3LIM+MnoUTdV0c9Q8VzRdPIAqjrPz71aUxYsZqgBjq5qv6D2YxHw1aT5H28WS9PboTViF\\\\\\\/7kZIUKCCXPCRxJ9uTmw4xIsZkabjgi+ZqnKfuWb0J7dlVSlBruGd8IhsbNyTcbTeanPqBO7xsUvcwXFU\\\\\\\/jNGJ2BA1GWDTp1caZmW0IkXzOHo9CQlh+UxeaGFxPspDXpABbmsIWr3TKbYqG1tbRj+5boesVryYkJZdAWsXvG1Le+WGBDklyfH5fNjWmPibONRxzV9pAElcq1yqHe8WBNXG0DSm9\\\\\\\/xVD\\\\\\\/PhECluIACXSfPM\\\\\\\/AciP+eXfXRHcsRywZ1\\\\\\\/\\\\\\\/grp6l7MVtG18Sr7t4PPTNHwGP6gHM8jjw5hrvF0IDOcgreBSHCxJa6jhwTHJpZ1Cc8688Evza4RJKaTR9sfADyOxRGZYvaMH06g3YTNkNIlsWZam3xDwHUUhbOSQLi7Z8C44oDGedPc2Um5hu6KNh\\\\\\\/gq3TlItigsg8gweoLW8NTSSwOBM830ciH+qf9m1IJGtLAI7O2kHUbGq6Q\\\\\\\/FnLmZP2ihWKbK\\\\\\\/FiP92joK5YIrTvfA4pUImOoD3uW+CSKGLuKwu\\\\\\\/oNYPUXyd1QrQMMA86RUSYT4Z3ogIP+oDPdRYnfTAl3HpXk7XlLXVcWl0N1sEjYMaNpINLE3rw622EmQEa1DlucyoO4avVbgl7c4t1ewzwMv5QA5CWA6iCakA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022JLPqcVDmgBkt\\\\\\\/Je8fioyIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b887b50b68f6c4626804528dc68b6efc01d333a93d2c103319234eb7fceed7b3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492416 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/879" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BMkKDqX0eFu5WoMm9Z-AqUZO2QTL5XoS7iFbHeCa3cg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/879" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9krpstEfXXWQheC15eFHCrT9WTlCjkZtN0W-XYV-Nok" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/867" title="Identification of Suppressor of Clathrin Deficiency-1 (SCD1) and Its Connection to Clathrin-Mediated Endocytosis in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/889" title="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492416" id="node492416" data-pisa="ggg;9/3/879" data-pisa-master="ggg;g3.118.200745" data-apath="/ggg/9/3/879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Coverage-Versus-Length Plots, a Simple Quality Control Step for <em>de Novo</em> Yeast Genome Sequence Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander P. Douglass</span>, <span class="highwire-citation-author" data-delta="1">Caoimhe E. O’Brien</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Offei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aisling Y. Coughlan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raúl A. Ortiz-Merino</span>, <span class="highwire-citation-author" data-delta="5">Geraldine Butler</span>, <span class="highwire-citation-author" data-delta="6">Kevin P. Byrne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kenneth H. Wolfe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">879-887;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492416" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander P. Douglass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BP.%2BDouglass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglass%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BP.%2BDouglass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Caoimhe E. O’Brien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caoimhe%2BE.%2BO%E2%80%99Brien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Brien%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaoimhe%2BE.%2BO%25E2%2580%2599Brien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Offei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BOffei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Offei%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BOffei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aisling Y. Coughlan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aisling%2BY.%2BCoughlan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coughlan%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAisling%2BY.%2BCoughlan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aisling Y. Coughlan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Raúl A. Ortiz-Merino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ra%C3%BAl%2BA.%2BOrtiz-Merino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ortiz-Merino%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARa%25C3%25BAl%2BA.%2BOrtiz-Merino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raúl A. Ortiz-Merino</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Geraldine Butler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geraldine%2BButler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butler%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeraldine%2BButler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin P. Byrne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BByrne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byrne%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BByrne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth H. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BH.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20KH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BH.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kenneth H. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.wolfe@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">kenneth.wolfe@ucd.ie</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/879" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492416" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492416/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/879.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492416" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492416/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/879.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492416" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492416/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/879.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492416" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492416/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/879.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492416" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492416/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Illumina sequencing has revolutionized yeast genomics, with prices for commercial draft genome sequencing now below $200. The popular SPAdes assembler makes it simple to generate a <em>de novo</em> genome assembly for any yeast species. However, whereas making genome assemblies has become routine, understanding what they contain is still challenging. Here, we show how graphing the information that SPAdes provides about the length and coverage of each scaffold can be used to investigate the nature of an assembly, and to diagnose possible problems. Scaffolds derived from mitochondrial DNA, ribosomal DNA, and yeast plasmids can be identified by their high coverage. Contaminating data, such as cross-contamination from other samples in a multiplex sequencing run, can be identified by its low coverage. Scaffolds derived from the bacteriophage PhiX174 and Lambda DNAs that are frequently used as molecular standards in Illumina protocols can also be detected. Assemblies of yeast genomes with high heterozygosity, such as interspecies hybrids, often contain two types of scaffold: regions of the genome where the two alleles assembled into two separate scaffolds and each has a coverage level <em>C</em>, and regions where the two alleles co-assembled (collapsed) into a single scaffold that has a coverage level 2<em>C</em>. Visualizing the data with Coverage-<em>vs.</em>-Length (CVL) plots, which can be done using Microsoft Excel or Google Sheets, provides a simple method to understand the structure of a genome assembly and detect aberrant scaffolds or contigs. We provide a Python script that allows assemblies to be filtered to remove contaminants identified in CVL plots.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-3">It is now easy and cheap to sequence a yeast genome using the Illumina platform. Genome sequencing projects are usually described as either resequencing projects or <em>de novo</em> assembly projects. In resequencing projects, Illumina reads from the strain to be studied are mapped onto a reference genome from the same species (<em>e.g.</em>, <em>Saccharomyces cerevisiae</em> S288C), in order to find nucleotide polymorphisms or mutations. In <em>de novo</em> assembly projects, the Illumina reads from the strain to be studied are assembled without using a reference, to produce a set of contigs or scaffolds for the strain. <em>De novo</em> assembly is used in several specific situations: (i), where no suitable reference genome sequence exists (<em>e.g.</em>, if the species has not been sequenced before); (ii), where the researcher chooses to ignore the reference (<em>e.g.</em>, to explore the pan-genome in natural isolates of <em>S. cerevisiae</em>); (iii), where the strain being studied comes from an unknown species; or (iv), for mixed samples, such as metagenomics projects. For any newly-discovered yeast species, in order to publish a formal taxonomic description in the <em>International Journal of Systematic and Evolutionary Microbiology</em>, authors are now requested to include a (<em>de novo</em>) sequence assembly of its genome (<a href="http://ijs.microbiologyresearch.org/content/journal/ijsem/about">http://ijs.microbiologyresearch.org/content/journal/ijsem/about</a>).</p><p id="p-4">One of the most widely-used assembly programs for <em>de novo</em> yeast genome sequences is SPAdes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bankevich <em>et al.</em> 2012</a>). SPAdes assembles genomes into contigs or scaffolds, depending on the type of Illumina data that is input (single reads, paired-end reads, or mate-pair reads). Contigs are contiguous stretches of assembled genome sequence. Scaffolds are groups of contigs, whose relative order and orientation is known, but which are separated by short gaps of unknown sequence (represented by poly-N regions in the scaffold sequence). A typical yeast genome sequencing project will generate about 1.5 gigabases of raw data (<em>e.g.</em>, 10 million Illumina reads of 150 bp each), which can be assembled by SPAdes in a few hours on a standard laptop, giving more than 50x coverage of the genome.</p><p id="p-5">As the capacity of Illumina machines has grown, it has become increasingly common that samples are sequenced in multiplex, <em>i.e.</em>, genomic libraries are constructed from pooled samples, using a different index (or pair of indexes) for each sample, and then sequenced together in the same flowcell. For example, an Illumina HiSeq 4000 machine can sequence 96 yeast samples in multiplex in a single flowcell. It should be noted that the Illumina multiplex methodology uses up to four separate sequencing reactions to obtain data from a single flowcell spot containing a genomic DNA fragment: the library index(es) are sequenced in 1-2 reactions using different primers than are used to sequence one or both ends of the genomic DNA fragment (<a href="https://support.illumina.com/downloads/indexed-sequencing-overview-15057455.html">https://support.illumina.com/downloads/indexed-sequencing-overview-15057455.html</a>). There is therefore some potential for mix-ups in which genomic DNA reads are assigned to the wrong index and hence to the wrong sample.</p><p id="p-6">The drawback to our ability to sequence genomes faster and cheaper is that the amount of time and care spent on human examination of each sequence must inevitably decrease (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Lu and Salzberg 2018</a>). Gross errors such as the assignment of a genome sequence to the wrong species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Stavrou <em>et al.</em> 2018</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Watanabe <em>et al.</em> 2018</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Shen <em>et al.</em> 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Pavlov <em>et al.</em> 2018</a>), or yeast genome sequences that include many contigs from contaminating bacteria (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Donovan <em>et al.</em> 2018</a>), are increasingly present in public databases. In addition, the genome sequences of some yeast isolates have proven difficult to assemble because they are highly heterozygous, in some cases because they are interspecies hybrids (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Pryszcz <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Pryszcz and Gabaldon 2016</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Schröder <em>et al.</em> 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Braun-Galleani <em>et al.</em> 2018</a>). In this report, we present a simple method that can be used for quality control of an Illumina genome assembly, by making use of the information that SPAdes produces about the length and coverage of each contig or scaffold.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><p id="p-7">All the example assemblies we present were made using SPAdes version 3.11.1 (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bankevich <em>et al.</em> 2012</a>), downloaded from <a href="http://cab.spbu.ru/software/spades/">http://cab.spbu.ru/software/spades/</a>. Illumina FASTQ files were obtained by our laboratory using commercial DNA sequencing services, except for <em>Hanseniaspora osmophila</em> NCYC58 for which we downloaded FASTQ files from <a href="http://opendata.ifr.ac.uk/NCYC/">http://opendata.ifr.ac.uk/NCYC/</a> and assembled them (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We did not carry out any data preprocessing steps.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492532/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492532/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of Illumina datasets analyzed, and their SPAdes assemblies</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">Instructions for how to make a CVL plot from a SPAdes output file using common spreadsheet programs are given in Box 1.</p><div class="boxed-text" id="boxed-text-1"><span class="boxed-text-label"></span> <div id="sec-2" class="subsection"><h4>Box 1. How to make a CVL plot using Microsoft Excel, Google Sheets, or R.</h4></div><div id="sec-3" class="subsection"><h4>(A) Instructions for Excel.</h4><p id="p-12">These instructions are based on Excel for Mac 2011. Other versions of Excel may be slightly different. A video demonstration of these instructions is available at <a href="https://tinyurl.com/CVLexcel">https://tinyurl.com/CVLexcel</a></p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-13">Import the scaffolds.fasta file into a new blank Excel sheet:</p><ul class="list-simple " id="list-2"><li id="list-item-2"><p id="p-14">• In Excel, choose File → Open → scaffolds.fasta (Click Next and Finish in the dialog box).</p></li><li id="list-item-3"><p id="p-15">• Select Column A.</p></li><li id="list-item-4"><p id="p-16">• Choose Data → Sort → Sort by Column A (in A-to-Z order). All the header lines go to the top of the Excel sheet, with all the sequence rows below them.</p></li><li id="list-item-5"><p id="p-17">• Delete all the sequence data: Find the first row of sequence data and click on its row. Then scroll to the bottom of the file and shift-click on the last row of sequence data. Then choose Edit → Delete.</p></li><li id="list-item-6"><p id="p-18">• You are left with all the header lines. Save the file as an Excel (.xlsx) file.</p></li></ul></li><li id="list-item-7"><p id="p-19">Split the information in the header lines into separate columns:</p><ul class="list-simple " id="list-3"><li id="list-item-8"><p id="p-20">• Choose Data → Text to Columns. Click Next once.</p></li><li id="list-item-9"><p id="p-21">• In the Delimiters box, check Other and type an underscore character (_) in the box. Then click Next and Finish.</p></li><li id="list-item-10"><p id="p-22">• You should see the scaffold length in column D, and the coverage data in column F.</p></li></ul></li><li id="list-item-11"><p id="p-23">Make a scatterplot:</p><ul class="list-simple " id="list-4"><li id="list-item-12"><p id="p-24">• Select column D and command-click to select column F as well.</p></li><li id="list-item-13"><p id="p-25">• Insert → Chart… → Scatter → Marked Scatter</p></li><li id="list-item-14"><p id="p-26">• Change the Y-axis (coverage) to a log scale by double-clicking on the Y-axis and ticking the “Logarithmic scale” checkbox.</p></li><li id="list-item-15"><p id="p-27">• You can identify the scaffold associated with any point by moving the mouse over it.</p></li><li id="list-item-16"><p id="p-28">• It’s easier to see individual points if you make them smaller: Double-click on any point, choose Marker Style, and reduce the marker size to 3 point.</p></li></ul></li></ol></div><div id="sec-4" class="subsection"><h4>(B) Instructions for Google Sheets.</h4><p id="p-29">A video demonstration of these instructions is available at <a href="https://tinyurl.com/CVLgsheets">https://tinyurl.com/CVLgsheets</a></p><ol class="list-ord " id="list-5"><li id="list-item-17"><p id="p-30">Rename the scaffolds.fasta file scaffolds.txt</p></li><li id="list-item-18"><p id="p-31">Import the scaffolds.txt file into a new blank Google Sheet:</p><ul class="list-simple " id="list-6"><li id="list-item-19"><p id="p-32">• Choose File → Import… → Upload, and select the file scaffolds.txt</p></li><li id="list-item-20"><p id="p-33">• Click Import Data (leave all the options as their default values), and wait while the data are imported (it can take 1-2 min).</p></li><li id="list-item-21"><p id="p-34">• Select Column A.</p></li><li id="list-item-22"><p id="p-35">• Choose Data → Sort Sheet by Column A, A→Z, and wait while the data are sorted. All the header lines go to the top of the sheet, with all the sequence rows below them. Wait until the progress bar on the top right has finished.</p></li><li id="list-item-23"><p id="p-36">• Delete all the sequence data: Find the first row of sequence data and click on its row. Then scroll to the bottom of the file and shift-click on the last row of sequence data. Then choose Edit → Delete Rows.</p></li><li id="list-item-24"><p id="p-37">• You are left with all the header lines.</p></li></ul></li><li id="list-item-25"><p id="p-38">Split the information in the header lines into separate columns:</p><ul class="list-simple " id="list-7"><li id="list-item-26"><p id="p-39">• Select Column A.</p></li><li id="list-item-27"><p id="p-40">• Choose Data → Split Text into Columns...</p></li><li id="list-item-28"><p id="p-41">• A box labeled Separator pops up near the bottom of the screen. Pull down the menu in this box, choose Custom, and type an underscore character (_) in the box. Then hit Enter on the keyboard.</p></li><li id="list-item-29"><p id="p-42">• You should see the scaffold length in column D, and the coverage data in column F.</p></li></ul></li><li id="list-item-30"><p id="p-43">Make a scatterplot:</p><ul class="list-simple " id="list-8"><li id="list-item-31"><p id="p-44">• Select column D and control-click to select column F as well.</p></li><li id="list-item-32"><p id="p-45">• Insert → Chart…</p></li><li id="list-item-33"><p id="p-46">• In the Chart Editor panel (on the right of the screen), under the Data tab, choose Chart Type → Scatter</p></li><li id="list-item-34"><p id="p-47">• In the Chart Editor panel, under the Customize tab, click on Vertical Axis and tick the checkbox beside Logarithmic Scale.</p></li><li id="list-item-35"><p id="p-48">• You can identify the scaffold associated with any point by clicking on the point.</p></li><li id="list-item-36"><p id="p-49">• It’s easier to see individual points if you make them smaller. In the Chart Editor, under the Customize tab, choose Series, and reduce the Point size to 2 px.</p></li></ul></li></ol></div><div id="sec-5" class="subsection"><h4>(C) Instructions for R.</h4><p id="p-50">We provide an R script for making CVL plots at <a href="https://github.com/APDLS/CVLFilter">https://github.com/APDLS/CVLFilter</a>.</p></div></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-51">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7575536">https://doi.org/10.25387/g3.7575536</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-52">SPAdes (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Bankevich <em>et al.</em> 2012</a>) is a popular assembly program because it is simple to install, runs in a few hours on a laptop, and makes good assemblies ‘out of the box’ with its default settings. It produces two main output files, called contigs.fasta and scaffolds.fasta. The scaffolds.fasta file may include some poly-N regions where two contigs have been joined with a gap between them, because their relative order and orientation is known. For single-read and paired-end reads data, the two files are very similar and we consider only the scaffolds.fasta output file here.</p><p id="p-53">The scaffolds.fasta file contains all the assembled scaffolds that are larger than a minimum size. The minimum size is <em>k</em>+1 bp, where <em>k</em> is the largest <em>k</em>-mer size that SPAdes used for the assembly. For 150-bp reads, <em>k</em> is usually 77 bp so the smallest scaffolds reported are only 78 bp long. In the file, the scaffolds are given names such as NODE_1, NODE_2, etc, and are sorted in decreasing order of length so that NODE_1 is the longest scaffold in the assembly. Each scaffold in the file has a FASTA header that includes its name, its length in basepairs, and its coverage (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/879/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Organization of a scaffolds.fasta file containing a SPAdes genome assembly. The header line for each scaffold begins with a &#x201C;&gt;&#x201D; character, which is standard for data in FASTA format. The header for each scaffold shows the scaffold&#x2019;s node number, its length, and its k-mer coverage. In this example, NODE_1 is 1,401,924 bp long and has 73.35x coverage. The header line is followed by NODE_1&#x2019;s DNA sequence, and then the header and sequence of the next scaffold (NODE_2). The file can contain thousands of scaffold sequences, one after the other, down to the minimum scaffold size. The scaffolds are always named and sorted in descending order of length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-428856176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Organization of a scaffolds.fasta file containing a SPAdes genome assembly. The header line for each scaffold begins with a &#x201C;&gt;&#x201D; character, which is standard for data in FASTA format. The header for each scaffold shows the scaffold&#x2019;s node number, its length, and its &lt;em&gt;k-mer&lt;/em&gt; coverage. In this example, NODE_1 is 1,401,924 bp long and has 73.35x coverage. The header line is followed by NODE_1&#x2019;s DNA sequence, and then the header and sequence of the next scaffold (NODE_2). The file can contain thousands of scaffold sequences, one after the other, down to the minimum scaffold size. The scaffolds are always named and sorted in descending order of length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/879/F1.medium.gif" width="440" height="159"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/879/F1.medium.gif" width="440" height="159"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/879/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/879/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492467" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-54" class="first-child">Organization of a scaffolds.fasta file containing a SPAdes genome assembly. The header line for each scaffold begins with a “&gt;” character, which is standard for data in FASTA format. The header for each scaffold shows the scaffold’s node number, its length, and its <em>k-mer</em> coverage. In this example, NODE_1 is 1,401,924 bp long and has 73.35x coverage. The header line is followed by NODE_1’s DNA sequence, and then the header and sequence of the next scaffold (NODE_2). The file can contain thousands of scaffold sequences, one after the other, down to the minimum scaffold size. The scaffolds are always named and sorted in descending order of length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">The coverage values reported by SPAdes are properly called <em>k</em>-mer coverage. <em>k</em>-mer coverage is directly proportional to the more familiar concept of base coverage, which is the average number of Illumina reads that covered each nucleotide site in the scaffold when its consensus sequence was generated. The two quantities are related by this formula (<a href="https://www.ebi.ac.uk/%E2%88%BCzerbino/velvet/Manual.pdf">https://www.ebi.ac.uk/∼zerbino/velvet/Manual.pdf</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/879/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/879/embed/mml-math-1.gif"/></noscript></span></span>where <em>C<sub>k</sub></em> is <em>k</em>-mer coverage, <em>C</em> is base coverage, <em>L</em> is the read length, and <em>k</em> is the largest <em>k</em>-mer size that was used by SPAdes to make the assembly. <em>C<sub>k</sub></em> for a scaffold is the average number of times the <em>k</em>-mers that make up the scaffold were seen among the reads. For 150-bp reads and <em>k</em>-mers of 77 bp, the <em>k</em>-mer coverage is approximately half the base coverage. Throughout this paper, when we refer to “coverage” we mean <em>k</em>-mer coverage.</p><div id="sec-8" class="subsection"><h3>Example 1 – A typical yeast Coverage-<em>vs.</em>-Length (CVL) plot</h3><p id="p-56">In this example, we used a commercial genome sequencing service to sequence a yeast strain constructed from <em>Komagataella phaffii</em> strain CBS7435 (this species was formerly called <em>Pichia pastoris</em>). Two high-quality reference genome sequences are available for CBS7435 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Sturmberger <em>et al.</em> 2016</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Love <em>et al.</em> 2016</a>). Our raw data consisted of 11.1 million paired-end reads of 150 bp each. SPAdes assembled the data into almost 25,000 scaffolds, but most of these are very short and only 614 scaffolds are larger than 500 bp (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The N50 of the assembly is 642 kb. Researchers would typically discard very short scaffolds from the assembly, but here we retained them for analysis.</p><p id="p-57"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> shows a coverage-<em>vs.</em>-length (CVL) plot for this assembly. To make a CVL plot, we simply extract the coverage and length data from all the headers in the scaffolds.fasta file (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and plot them against each other. Each point in the plot is a scaffold. The X-axis is scaffold length, plotted on a linear scale. The Y-axis is the coverage (<em>k</em>-mer coverage), plotted on a log scale. The longest scaffold in the <em>K. phaffii</em> assembly (NODE_1) is 1.4 Mb long and has a coverage of 73x. The other large scaffolds (&gt;100 kb) have similar levels of coverage (between 70x and 90x). The variation in coverage levels increases on shorter scaffolds. As expected, all the large scaffolds matched the reference nuclear genome sequence available for <em>K. phaffii</em>. A few scaffolds stand out because they have coverage that is much higher than the 70-90x seen in the large scaffolds from the nuclear genome. NODE_19 is the mitochondrial genome (36 kb, 1493x). NODE_26 is a cytoplasmic linear DNA plasmid related to killer plasmids (12 kb, 557x). Coverage levels vary widely in the smaller scaffolds, shown in a zoomed-in view in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>. Five small scaffolds with very high coverage (1600x – 3000x) come from the ribosomal DNA array. Thus, scaffolds that stand out on the basis of unusually high coverage are those that do not come from typical parts of the nuclear genome.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/879/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CVL plot of a SPAdes assembly of data from Komagataella phaffii strain CBS7435, showing (A) all scaffolds, and (B) scaffolds shorter than 20 kb. The sources of the longest 100 scaffolds were identified using BLASTN searches against the NCBI database and are color coded. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text. The green lines in B indicate 65x coverage and 500 bp length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-428856176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CVL plot of a SPAdes assembly of data from &lt;em&gt;Komagataella phaffii&lt;/em&gt; strain CBS7435, showing (A) all scaffolds, and (B) scaffolds shorter than 20 kb. The sources of the longest 100 scaffolds were identified using BLASTN searches against the NCBI database and are color coded. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text. The green lines in B indicate 65x coverage and 500 bp length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/879/F2.medium.gif" width="277" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/879/F2.medium.gif" width="277" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/879/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/879/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492366" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-58" class="first-child">CVL plot of a SPAdes assembly of data from <em>Komagataella phaffii</em> strain CBS7435, showing (A) all scaffolds, and (B) scaffolds shorter than 20 kb. The sources of the longest 100 scaffolds were identified using BLASTN searches against the NCBI database and are color coded. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text. The green lines in B indicate 65x coverage and 500 bp length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">We used BLASTN searches against the NCBI nonredundant nucleotide database to identify the origins of each of the 100 longest scaffolds (<em>i.e.</em>, all the scaffolds longer than 1160 bp), and found that none of the ones with low coverage come from <em>K. phaffii</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The most common source of contamination (25 scaffolds) was <em>Verticillium dahliae</em> (a fungal pathogen of plants), including its nuclear rDNA sequence (NODE_33, 3326 bp at 60x coverage) as well as multiple scaffolds containing parts of its mitochondrial genome (∼3x coverage) and a nuclear retrotransposon (∼1x). We also identified seven contaminating scaffolds from cow (all were nuclear satellite DNAs, of which the most abundant was NODE_76, corresponding to the bovine ‘1.715’ satellite, 1397 bp at 29x coverage), and five from plant species (nuclear repetitive DNA and chloroplast DNA). We suspect that the source of these diverse contaminating sequences was other sequencing projects being carried out at the commercial sequencing center, probably in multiplex on the same Illumina flowcell as our sample. We do not think that our <em>K. phaffii</em> assembly is unusually highly contaminated, but rather that the situation we describe is commonplace.</p><p id="p-60">A bioinformatician handling our <em>K. phaffii</em> assembly would normally discard the shorter scaffolds before analysis or submission of the data to public databases. Typical practice would be to discard scaffolds &lt; 500 bp or &lt; 1 kb. However, as <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> shows, our assembly includes some scaffolds up to 4 kb long that are contaminants. The CVL plot allows the existence of scaffolds with aberrant coverage levels to be detected. In this example, applying a filter to keep only scaffolds with coverage &gt;63x and length &gt;500 bp would exclude all the contaminants and retain almost all the genuine <em>K. phaffii</em> scaffolds (green lines in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Importantly, these cutoffs could have been chosen simply by looking at the CVL plot and using BLASTN to check the most anomalous scaffolds (NODES 76, 33, and 30; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>).</p></div><div id="sec-9" class="subsection"><h3>Example 2 – Aneuploidy, and bacteriophage contamination</h3><p id="p-61"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows a CVL plot of a SPAdes assembly from <em>Hanseniaspora osmophila</em> strain NCYC58. This strain was sequenced by the UK National Collection of Yeast Cultures at the Earlham Institute as part of a systematic strain sequencing program (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dujon and Louis 2017</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Wu <em>et al.</em> 2017</a>), and assembled by us (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). A high-quality whole-genome shotgun assembly of a reference <em>H. osmophila</em> strain (AWRI3579) is also available (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Sternes <em>et al.</em> 2016</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/879/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CVL plot of a SPAdes assembly of data from Hanseniaspora osmophila strain NCYC58 showing (A) all scaffolds, and (B) scaffolds shorter than 10 kb. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-428856176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CVL plot of a SPAdes assembly of data from &lt;em&gt;Hanseniaspora osmophila&lt;/em&gt; strain NCYC58 showing (A) all scaffolds, and (B) scaffolds shorter than 10 kb. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/879/F3.medium.gif" width="439" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/879/F3.medium.gif" width="439" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/879/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/879/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492447" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-62" class="first-child">CVL plot of a SPAdes assembly of data from <em>Hanseniaspora osmophila</em> strain NCYC58 showing (A) all scaffolds, and (B) scaffolds shorter than 10 kb. Numbers beside points indicate scaffold (NODE) numbers mentioned in the text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-63">The plot of the NCYC58 assembly shows many large scaffolds with coverage of ∼35x (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). It also shows many small scaffolds with coverage &lt;10x (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), which in this case are contamination from other yeast species (probably due to cross-contamination from multiplexed yeast samples). From BLASTN searches, we found that the largest of these contaminating scaffolds (NODE_223, 8087 bp, 3x coverage) comes from <em>Saccharomyces cerevisiae</em> mitochondrial DNA. The second-largest (NODE_260, 4308 bp, 3x coverage) comes from <em>Wickerhamomyces anomalus</em> mitochondrial DNA. The most abundant yeast contaminant (NODE_300, 2033 bp, 9x coverage) comes from <em>S. cerevisiae</em> ribosomal DNA. There is a clear gap in coverage between the contaminants and the least-abundant genuine scaffold from <em>H. osmophila</em> (NODE_316, 1472 bp, 21x coverage), so the contaminating yeast scaffolds could be removed by excluding scaffolds with &lt;15x coverage (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-64">The CVL plot for NCYC58 also shows evidence of aneuploidy in this strain. Although most of the large scaffolds have coverage ∼35x, some scaffolds have twofold higher coverage at ∼70x (for example, NODES 3, 12, 15 and 17 in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>). By BLASTN searches against the reference sequence of <em>H. osmophila</em> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Sternes <em>et al.</em> 2016</a>), we found that the NCYC58 scaffolds with ∼70x coverage all map to the same chromosome in the reference assembly (AWRI3579 scaffold10). Therefore, we suggest that there is an extra copy of this chromosome in strain NCYC58.</p><p id="p-65">The high-coverage scaffolds in the CVL plot for NCYC58 include the mitochondrial genome and the ribosomal DNA, as was seen in <em>K. phaffii</em>. Another high-coverage scaffold is a contaminant: the bacteriophage ΦX174 genome (NODE_244, scaffold length 5441 bp, coverage 477x). ΦX174 DNA is used as an internal positive control for the sequencing reagents in some Illumina protocols and is a frequent contaminant in microbial genome assemblies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Mukherjee <em>et al.</em> 2015</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Lu and Salzberg 2018</a>). In CVL plots from some other yeast genome assemblies (not illustrated here) we have also found the bacteriophage λ genome at high coverage (scaffold length 48-49 kb). Bacteriophage λ DNA is used as a standard for DNA quantification in an Illumina protocol for making indexed paired-end libraries (<a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_trusight/trusight-rapid-capture-reference-guide-15043291-01.pdf">https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/samplepreps_trusight/trusight-rapid-capture-reference-guide-15043291-01.pdf</a>).</p></div><div id="sec-10" class="subsection"><h3>Example 3 – Assemblies of highly heterozygous or hybrid genomes</h3><p id="p-66"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> shows the CVL plot of a SPAdes genome assembly from <em>Torulaspora delbrueckii</em> strain L17. The plot shows that two major types of scaffold are present in L17: ones with coverage ∼33x, and ones with coverage ∼16x. Both types of scaffold have high similarity to other <em>Torulaspora</em> sequences in databases, so they are not contaminants. By BLASTN searches against the genome sequence of the type strain of <em>T. delbrueckii</em> (CBS1146; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gordon <em>et al.</em> 2011</a>) we found that L17 is a highly heterozygous diploid strain. Heterozygosity is unusual, because most strains of <em>T. delbrueckii</em> are haploid and their assemblies do not show two types of scaffold (<em>e.g.</em>, strain L16 in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/879/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Highly heterozygous genomes. (A) CVL plot of the assembly from Torulaspora delbrueckii strain L17, which is highly heterozygous. (B) CVL plot of the assembly from T. delbrueckii strain L16, which is not heterozygous. (C) Graphical view of a BLASTN search result. The database is the assembly of the L17 genome, and the query is the first 3 kb of the DYN1 gene of the T. delbrueckii reference strain CBS1146. Each horizontal red bar indicates a scaffold that contains a region of DYN1, relative to the 3-kb scale. Black arrows mark allelic pairs of scaffolds from non-collapsed regions, and magenta arrows mark single collapsed scaffolds. For clarity, this diagram shows only the first 3 kb of the 12 kb DYN1 gene. In total, DYN1 maps onto 68 scaffolds in the L17 assembly: 22 collapsed scaffolds and 23 pairs. (D) CVL plot of the assembly from Metschnikowia strain UCD127." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-428856176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Highly heterozygous genomes. (A) CVL plot of the assembly from &lt;em&gt;Torulaspora delbrueckii&lt;/em&gt; strain L17, which is highly heterozygous. (B) CVL plot of the assembly from &lt;em&gt;T. delbrueckii&lt;/em&gt; strain L16, which is not heterozygous. (C) Graphical view of a BLASTN search result. The database is the assembly of the L17 genome, and the query is the first 3 kb of the &lt;em&gt;DYN1&lt;/em&gt; gene of the &lt;em&gt;T. delbrueckii&lt;/em&gt; reference strain CBS1146. Each horizontal red bar indicates a scaffold that contains a region of &lt;em&gt;DYN1&lt;/em&gt;, relative to the 3-kb scale. Black arrows mark allelic pairs of scaffolds from non-collapsed regions, and magenta arrows mark single collapsed scaffolds. For clarity, this diagram shows only the first 3 kb of the 12 kb &lt;em&gt;DYN1&lt;/em&gt; gene. In total, &lt;em&gt;DYN1&lt;/em&gt; maps onto 68 scaffolds in the L17 assembly: 22 collapsed scaffolds and 23 pairs. (D) CVL plot of the assembly from &lt;em&gt;Metschnikowia&lt;/em&gt; strain UCD127.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/879/F4.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/3/879/F4.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/879/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/879/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492368" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-67" class="first-child">Highly heterozygous genomes. (A) CVL plot of the assembly from <em>Torulaspora delbrueckii</em> strain L17, which is highly heterozygous. (B) CVL plot of the assembly from <em>T. delbrueckii</em> strain L16, which is not heterozygous. (C) Graphical view of a BLASTN search result. The database is the assembly of the L17 genome, and the query is the first 3 kb of the <em>DYN1</em> gene of the <em>T. delbrueckii</em> reference strain CBS1146. Each horizontal red bar indicates a scaffold that contains a region of <em>DYN1</em>, relative to the 3-kb scale. Black arrows mark allelic pairs of scaffolds from non-collapsed regions, and magenta arrows mark single collapsed scaffolds. For clarity, this diagram shows only the first 3 kb of the 12 kb <em>DYN1</em> gene. In total, <em>DYN1</em> maps onto 68 scaffolds in the L17 assembly: 22 collapsed scaffolds and 23 pairs. (D) CVL plot of the assembly from <em>Metschnikowia</em> strain UCD127.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">Detailed analysis shows how the SPAdes assembler produced this pattern from the L17 data. In some sections of the genome, SPAdes ‘collapsed’ (merged) the sequences of the two alleles into a single consensus sequence, making only one scaffold, with ∼33x coverage. In other sections of the genome, SPAdes kept the two alleles separate, resulting in two scaffolds from these regions, with ∼16x coverage each. The two types of sections alternate along the genome, and some of the sections are quite small – only a few hundred basepairs. For example, a BLASTN search using part of the large gene <em>DYN1</em> as the query, and the SPAdes assembly of the L17 genome as the database, produces hits with the pattern shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>, alternating between hits to single scaffolds, and hits to pairs of scaffolds with lower coverage. Strain L17 contains two copies (alleles) of <em>DYN1</em>, but the assembler generated 68 separate scaffolds from them. In the parts of <em>DYN1</em> where separate allelic scaffolds were produced, the sequence divergence between alleles is approximately 7%. One of the alleles is almost identical to the <em>T. delbrueckii</em> CBS1146 reference genome, while the other is almost identical to the allele present in strain L16. Similar results are seen when other genes or genomic regions are used as BLASTN queries against the L17 assembly.</p><p id="p-69">The CVL plot for UCD127, a strain from the <em>Metschnikowia pulcherrima</em> subclade that we isolated from nature, shows a similar pattern with two major types of scaffold with a twofold difference in their coverage levels (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>). UCD127 is either a highly heterozygous diploid or an interspecies hybrid (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Venkatesh <em>et al.</em> 2018</a>). In this example, a large contaminant scaffold (22 kb, 4x coverage) was the mitochondrial genome of <em>Nadsonia starkeyi-henricii</em>, a yeast that we sequenced in the same multiplex job (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">O’Boyle <em>et al.</em> 2018</a>), which demonstrates that cross-contamination occurred in the flowcell.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-70">CVL plots provide a simple way to detect scaffolds or contigs in an assembly that have aberrant levels of sequence coverage. In some cases these scaffolds represent DNA that is truly present in the organism targeted for sequencing, at a copy-number that is higher or lower than average, which allows CVL plots to be used to detect mitochondrial DNA, ribosomal DNA, plasmids or other extrachromosomal elements, and aneuploidy. In other cases, the scaffolds with unusual coverage represent contaminants, which can stem from several different sources: cross-contamination from other DNA samples sequenced in the same multiplex experiment; bacterial contamination in the DNA sample used for sequencing; and bacteriophage DNA deliberately introduced for quality control in the sequencing process.</p><p id="p-71">In most of the examples we presented, we used assemblies from yeast strains where a reference genome assembly was already available from the same species. We did this so that we could validate our interpretations of the CVL plots by using BLAST searches. However, it should be emphasized that CVL plots allow most contaminant scaffolds to be detected and removed, even when no reference is available for the genome being sequenced, and even when the contaminant’s source species is not in genome databases. Contaminants can be detected and removed even without knowing what they are.</p><p id="p-72">The low-coverage contaminants we detected correspond to sequences that are probably the highest copy-number sequences in their source genomes: ribosomal DNAs and mitochondrial DNAs (<em>e.g.</em>, from <em>Verticillium</em>), satellite DNAs (bovine), retrotransposons (<em>Verticillium</em>), and chloroplast DNA (plants). This result is to be expected if cross-contamination occurs in a multiplex sequencing run; even a small number of sequencing reads from a eukaryotic genome can be assembled into consensus sequences for the most abundant repeat families in that genome (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Neuveglise <em>et al.</em> 2002</a>). As sequencing costs fall, the coverage level to which genomes are sequenced tends to increase. But as coverage increases, the lengths of scaffolds assembled from contaminating DNA will also increase.</p><p id="p-73">We have illustrated CVL plots with examples that used the SPAdes assembler, but in principle CVL plots could be made using the output from any assembler that reports coverage levels for individual scaffolds or contigs. SPAdes is particularly well suited to CVL plots because one of its characteristic behaviors is that contigs terminate whenever they enter repeat sequences, so that repeat sequences and the unique sequences that flank them are reported as separate contigs. This behavior means that each contig or scaffold reported by SPAdes has a fairly uniform level of coverage along its length. For comparison, we also generated CVL plots using two other popular assembly programs, ABySS 2.0 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Jackman <em>et al.</em> 2017</a>) and SOAPdenovo2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Luo <em>et al.</em> 2012</a>), with the same <em>K. phaffii</em> Illumina raw data as was used in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>. The CVL plots from these assemblers (Figs. S1, S2) are similar to those we obtained from SPAdes, in terms of allowing low-coverage contaminants to be detected. CVL plots may be a useful way to compare the quality of assemblies obtained using different assemblers, or different parameter values for one assembler, from the same raw data. For example, Figure S2 shows that shows SOAPdenovo produced a more fragmented assembly than SPAdes and ABySS.</p><p id="p-74">A researcher who has assembled a genome sequence will usually want to ‘clean up’ the assembly before submitting it to a public database. The current standard cleanup step is to remove all scaffolds smaller than a cutoff length, which is usually 500 bp or 1 kb. However, as we have shown, many contaminants would still be retained after such a length filter alone. Even without doing any BLAST searches, a CVL plot will enable the researcher to choose a cutoff value for minimum coverage. Applying cutoffs to both coverage and length will allow the researcher to produce an assembly whose level of contamination is greatly reduced, albeit at the risk of discarding a few genuine contigs. CVL plots can be made from SPAdes assemblies by using spreadsheet software without any programming steps (Box 1). We also provide a simple Python script that researchers can use to filter a SPAdes scaffolds.fasta or contigs.fasta file so that only the scaffolds/contigs that have coverage and length above minimum values specified by the user are retained (<a href="https://github.com/APDLS/CVLFilter">https://github.com/APDLS/CVLFilter</a>). Additionally, we provide an Rscript script that allows the automatic production of CVL plots with cutoff lines for coverage and length specified by the user.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-75">This work was supported by the Wellcome Trust PhD program in Computational Infection Biology at University College Dublin (105341/Z/14/Z). We thank Caroline Wilde (Lallemand, Inc.) for strains L16 and L17. The NCYC58 sequence data were produced by the National Collection of Yeast Cultures (UK), in partnership with the Earlham Institute, using funding awarded to the Institute of Food Research, Norwich, UK by the Biotechnology and Biological Sciences Research Council.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-76">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7575536">https://doi.org/10.25387/g3.7575536</a>.</p></li><li class="fn" id="fn-5"><p id="p-77"><em>Communicating editor: M. Cherry</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-18"><span class="received-label">Received </span>September 18, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-16"><span class="accepted-label">Accepted </span>January 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Douglass <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.1" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2012.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.2" data-doi="10.1093/femsyr/foy079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun-Galleani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz-Merino</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title"><em>Zygosaccharomyces pseudobailii</em>, another yeast interspecies hybrid that regained fertility by damaging one of its <em>MAT</em> loci.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">foy079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/femsyr/foy079</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft.volume%253D18%26rft.spage%253Dfoy079%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffoy079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/foy079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.3" data-doi="10.1371/journal.pone.0192898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identification of fungi in shotgun metagenomics datasets.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0192898</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0192898</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0192898%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0192898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0192898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.4" data-doi="10.1534/genetics.116.199216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome diversity and evolution in the budding yeasts (Saccharomycotina).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199216%26rft_id%253Dinfo%253Apmid%252F28592505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.5" data-doi="10.1073/pnas.1112808108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armisén</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proux-Wéra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">ÓhÉigeartaigh</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolutionary erosion of yeast sex chromosomes by mating-type switching accidents.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">20024</span>–<span class="cit-lpage">20029</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1112808108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112808108%26rft_id%253Dinfo%253Apmid%252F22123960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzUwLzIwMDI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.6" data-doi="10.1101/gr.214346.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandervalk</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohamadi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">777</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.214346.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.214346.116%26rft_id%253Dinfo%253Apmid%252F28232478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.7" data-doi="10.1186/s12864-016-2876-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative genomics and transcriptomics of <em>Pichia pastoris</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">550</span> (erratum: BMC Genomics 17: 762).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2876-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2876-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2876-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.8" data-doi="10.1371/journal.pcbi.1006277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Removing contaminants from databases of draft genomes.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1006277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1006277%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.9" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span> (erratum: Gigascience 4: s13742-015-0069-2).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2047-217X-1-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.10" data-doi="10.1186/1944-3277-10-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huntemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivanova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kyrpides</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pati</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Large-scale contamination of microbial isolate genomes by Illumina PhiX control.</span> <abbr class="cit-jnl-abbrev">Stand. Genomic Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1944-3277-10-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStand.%2BGenomic%2BSci.%26rft.volume%253D10%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1944-3277-10-18%26rft_id%253Dinfo%253Apmid%252F26203331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1944-3277-10-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26203331&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.11" data-doi="10.1101/gr.219202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neuveglise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaregola</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genomic evolution of the long terminal repeat retrotransposons in hemiascomycetous yeasts.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">930</span>–<span class="cit-lpage">943</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.219202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNeuveglise%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D930%26rft.epage%253D943%26rft.atitle%253DGenomic%2BEvolution%2Bof%2Bthe%2BLong%2BTerminal%2BRepeat%2BRetrotransposons%2Bin%2BHemiascomycetous%2BYeasts%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.219202%26rft_id%253Dinfo%253Apmid%252F12045146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.12" data-doi="10.1128/genomeA.00549-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Boyle</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergin</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussey</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddell</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Draft genome sequence of the yeast <em>Nadsonia starkeyi-henricii</em> UCD142, isolated from forest soil in Ireland.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00549–18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/genomeA.00549-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D6%26rft.spage%253D0054918e%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00549-18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.00549-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.13" data-doi="10.1002/jobm.201800066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlov</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bobrov</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sumacheva</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masharsky</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polev</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title"><em>Scytalidium candidum</em> 3C is a new name for the <em>Geotrichum candidum</em> Link 3C strain.</span> <abbr class="cit-jnl-abbrev">J. Basic Microbiol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">891</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jobm.201800066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBasic%2BMicrobiol.%26rft.volume%253D58%26rft.spage%253D883%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjobm.201800066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jobm.201800066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.14" data-doi="10.1093/nar/gkw294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Redundans: an assembly pipeline for highly heterozygous genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw294</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw294%26rft_id%253Dinfo%253Apmid%252F27131372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27131372&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.15" data-doi="10.1371/journal.pgen.1005626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryszcz</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemeth</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ksiezopolska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedusova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic aftermath of hybridization in the opportunistic pathogen <em>Candida metapsilosis</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005626</span> (erratum: PLoS Genet. 12: e1006202).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005626%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005626%26rft_id%253Dinfo%253Apmid%252F26517373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26517373&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.16" data-doi="10.1371/journal.pgen.1006404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schröder</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez de San Vicente</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prandini</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiple origins of the pathogenic yeast <em>Candida orthopsilosis</em> by separate hybridizations between two parental species.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006404%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.17" data-doi="10.1534/g3.116.034744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kominek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtzman</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hittinger</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reconstructing the backbone of the Saccharomycotina yeast phylogeny using genome-scale data.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3927</span>–<span class="cit-lpage">3939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.034744</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.034744%26rft_id%253Dinfo%253Apmid%252F27672114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTIvMzkyNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy84NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.18" data-doi="10.1002/yea.3303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stavrou</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mixao</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boekhout</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Misidentification of genome assemblies in public databases: The case of <em>Naumovozyma dairenensis</em> and proposal of a protocol to correct misidentifications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.3303</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D35%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.19" data-doi="10.1128/genomeA.01287-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sternes</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kutyna</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borneman</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome sequences of three species of <em>Hanseniaspora</em> isolated from spontaneous wine fermentations.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e01287–16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/genomeA.01287-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D4%26rft.spage%253De0128716%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.01287-16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.01287-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.20" data-doi="10.1016/j.jbiotec.2016.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturmberger</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chappell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krainer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Refined <em>Pichia pastoris</em> reference genome sequence.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">235</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jbiotec.2016.04.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiotechnol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2016.04.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2016.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.21" data-doi="10.1128/genomeA.00550-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinn Farrington</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maher</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Draft genome sequence of a highly heterozygous yeast strain from the <em>Metschnikowia pulcherrima</em> subclade, UCD127.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00550–18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/genomeA.00550-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D6%26rft.spage%253De0055018%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.00550-18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/genomeA.00550-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.22" data-doi="10.1128/AEM.01845-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uehara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukioka</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Can interspecies hybrid <em>Zygosaccharomyces rouxii</em> produce an allohaploid gamete?</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">e01845–17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.01845-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.volume%253D84%26rft.spage%253De0184517%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01845-17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/AEM.01845-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.879.23" data-doi="10.1038/s41598-017-14641-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elliston</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gall</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colquhoun</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">S. R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Yeast diversity in relation to the production of fuels and chemicals.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">14259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-14641-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D14259%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-14641-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-14641-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/879.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/867" title="Identification of Suppressor of Clathrin Deficiency-1 (SCD1) and Its Connection to Clathrin-Mediated Endocytosis in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/889" title="Turning Observed Founder Alleles into Expected Relationships in an Intercross Population" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/879" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uNeM4M89pzJqUWi5DzAXNAPhc_O88j7V7j-Wr3-C5R0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/879" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/879" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/879" class="active" data-icon-position="" data-hide-link-title="0">Coverage-Versus-Length Plots, a Simple Quality Control Step for de Novo Yeast Genome Sequence Assemblies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492416" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LGzkt7xInSft7Rt4IoJp9KA2gc4jdMHzF_DjTVz-efk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/879.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/879" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3R_wgnXai6W8MOmhKF2-_oZ4RBjXwLj7YGW5HCVsdkI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/879" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/879.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492416" id="node492416--2" data-pisa="ggg;9/3/879" data-pisa-master="ggg;g3.118.200745" data-apath="/ggg/9/3/879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Coverage-Versus-Length Plots, a Simple Quality Control Step for <em>de Novo</em> Yeast Genome Sequence Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander P. Douglass</span>, <span class="highwire-citation-author" data-delta="1">Caoimhe E. O’Brien</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Offei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aisling Y. Coughlan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raúl A. Ortiz-Merino</span>, <span class="highwire-citation-author" data-delta="5">Geraldine Butler</span>, <span class="highwire-citation-author" data-delta="6">Kevin P. Byrne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kenneth H. Wolfe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">879-887;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492416--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander P. Douglass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BP.%2BDouglass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglass%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BP.%2BDouglass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Caoimhe E. O’Brien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caoimhe%2BE.%2BO%E2%80%99Brien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Brien%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaoimhe%2BE.%2BO%25E2%2580%2599Brien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Offei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BOffei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Offei%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BOffei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aisling Y. Coughlan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aisling%2BY.%2BCoughlan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coughlan%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAisling%2BY.%2BCoughlan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aisling Y. Coughlan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Raúl A. Ortiz-Merino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ra%C3%BAl%2BA.%2BOrtiz-Merino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ortiz-Merino%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARa%25C3%25BAl%2BA.%2BOrtiz-Merino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raúl A. Ortiz-Merino</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Geraldine Butler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geraldine%2BButler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butler%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeraldine%2BButler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin P. Byrne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BByrne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byrne%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BByrne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth H. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BH.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20KH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BH.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kenneth H. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.wolfe@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">kenneth.wolfe@ucd.ie</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/879" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/879&amp;text=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/879&amp;t=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/879&amp;title=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492416/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492416" id="node492416--3" data-pisa="ggg;9/3/879" data-pisa-master="ggg;g3.118.200745" data-apath="/ggg/9/3/879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Coverage-Versus-Length Plots, a Simple Quality Control Step for <em>de Novo</em> Yeast Genome Sequence Assemblies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander P. Douglass</span>, <span class="highwire-citation-author" data-delta="1">Caoimhe E. O’Brien</span>, <span class="highwire-citation-author" data-delta="2">Benjamin Offei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aisling Y. Coughlan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Raúl A. Ortiz-Merino</span>, <span class="highwire-citation-author" data-delta="5">Geraldine Butler</span>, <span class="highwire-citation-author" data-delta="6">Kevin P. Byrne</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kenneth H. Wolfe</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">879-887;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200745 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492416--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander P. Douglass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BP.%2BDouglass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Douglass%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BP.%2BDouglass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Caoimhe E. O’Brien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caoimhe%2BE.%2BO%E2%80%99Brien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Brien%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaoimhe%2BE.%2BO%25E2%2580%2599Brien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Benjamin Offei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BOffei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Offei%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BOffei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Aisling Y. Coughlan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aisling%2BY.%2BCoughlan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coughlan%20AY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAisling%2BY.%2BCoughlan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8614-7780" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aisling Y. Coughlan</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Raúl A. Ortiz-Merino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ra%C3%BAl%2BA.%2BOrtiz-Merino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ortiz-Merino%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARa%25C3%25BAl%2BA.%2BOrtiz-Merino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4186-8941" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raúl A. Ortiz-Merino</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Geraldine Butler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biomolecular and Biomedical Sciences, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geraldine%2BButler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butler%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeraldine%2BButler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin P. Byrne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BByrne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Byrne%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BByrne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kenneth H. Wolfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Medicine, UCD Conway Institute, University College Dublin, Dublin 4, Ireland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BH.%2BWolfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wolfe%20KH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKenneth%2BH.%2BWolfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4992-4979" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kenneth H. Wolfe</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kenneth.wolfe@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">kenneth.wolfe@ucd.ie</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=879&amp;rft.epage=887&amp;rft.atitle=Coverage-Versus-Length%20Plots%2C%20a%20Simple%20Quality%20Control%20Step%20for%20de%20Novo%20Yeast%20Genome%20Sequence%20Assemblies&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Douglass%2C+Alexander+P.&amp;rft.au=O%E2%80%99Brien%2C+Caoimhe+E.&amp;rft.au=Offei%2C+Benjamin&amp;rft.au=Coughlan%2C+Aisling+Y.&amp;rft.au=Ortiz-Merino%2C+Ra%C3%BAl+A.&amp;rft.au=Butler%2C+Geraldine&amp;rft.au=Byrne%2C+Kevin+P.&amp;rft.au=Wolfe%2C+Kenneth+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492416/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492416/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492416/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492416/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492416/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492416/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492416/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492416/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492416/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492416/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492416/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492416/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/879" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/879.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/879.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/879.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/879.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
