<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/675" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/675.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/675.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/675.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/675/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/675.atom" />
<meta name="HW.pisa" content="ggg;9/3/675" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.118.200646" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Santantonio et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7311797" />
<meta name="DC.Description" content="Epistasis is an important contributor to genetic variance. In inbred populations, pairwise epistasis is present as additive by additive interactions. Testing for epistasis presents a multiple testing problem as the pairwise search space for modest numbers of markers is large. Single markers do not necessarily track functional units of interacting chromatin as well as haplotype based methods do. To harness the power of multiple markers while minimizing the number of tests conducted, we present a low resolution test for epistatic interactions across whole chromosome arms. Epistasis covariance matrices were constructed from the additive covariances of individual chromosome arms. These covariances were subsequently used to estimate an epistatic variance parameter while correcting for background additive and epistatic effects. We find significant epistasis for 2% of the interactions tested for four agronomic traits in a winter wheat breeding population. Interactions across homeologous chromosome arms were identified, but were less abundant than other chromosome arm pair interactions. The homeologous chromosome arm pair 4BL/4DL showed a strong negative relationship between additive and interaction effects that may be indicative of functional redundancy. Several chromosome arms appeared to act as hubs in an interaction network, suggesting that they may contain important regulatory factors. The differential patterns of epistasis across different traits demonstrate that detection of epistatic interactions is robust when correcting for background additive and epistatic effects in the population. The low resolution epistasis mapping method presented here identifies important epistatic interactions with a limited number of statistical tests at the cost of low precision." />
<meta name="DC.Contributor" content="Nicholas Santantonio" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Mark Sorrells" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Epistasis is an important contributor to genetic variance. In inbred populations, pairwise epistasis is present as additive by additive interactions. Testing for epistasis presents a multiple testing problem as the pairwise search space for modest numbers of markers is large. Single markers do not necessarily track functional units of interacting chromatin as well as haplotype based methods do. To harness the power of multiple markers while minimizing the number of tests conducted, we present a low resolution test for epistatic interactions across whole chromosome arms. Epistasis covariance matrices were constructed from the additive covariances of individual chromosome arms. These covariances were subsequently used to estimate an epistatic variance parameter while correcting for background additive and epistatic effects. We find significant epistasis for 2% of the interactions tested for four agronomic traits in a winter wheat breeding population. Interactions across homeologous chromosome arms were identified, but were less abundant than other chromosome arm pair interactions. The homeologous chromosome arm pair 4BL/4DL showed a strong negative relationship between additive and interaction effects that may be indicative of functional redundancy. Several chromosome arms appeared to act as hubs in an interaction network, suggesting that they may contain important regulatory factors. The differential patterns of epistasis across different traits demonstrate that detection of epistatic interactions is robust when correcting for background additive and epistatic effects in the population. The low resolution epistasis mapping method presented here identifies important epistatic interactions with a limited number of statistical tests at the cost of low precision.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/675" />
<meta name="citation_id" content="9/3/675" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/675" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/675.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/675.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/675.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200646" />
<meta name="citation_pmid" content="30455184" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="675" />
<meta name="citation_lastpage" content="684" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicholas Santantonio" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and" />
<meta name="citation_author_email" content="ns722@cornell.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4351-4023" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and" />
<meta name="citation_author_institution" content="USDA ARS, Robert W. Holley Center for Agriculture &amp; Health, Ithaca, NY 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4849-628X" />
<meta name="citation_author" content="Mark Sorrells" />
<meta name="citation_author_institution" content="Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7367-2663" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=K. L. Adams;citation_author=J. F. Wendel;citation_title=Polyploidy and genome evolution in plants.;citation_pages=135-141;citation_volume=8;citation_year=2005;citation_issue=2;citation_pmid=15752992;citation_doi=10.1016/j.pbi.2005.01.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_author=J. Isidro-Sánchez;citation_title=Locally epistatic models for genome-wide prediction and association by importance sampling.;citation_pages=74;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0348-8" />
<meta name="citation_reference" content="Akdemir, D. and U. G. Okeke, 2015 EMMREML: Fitting mixed models with known covariance structures. R package version 3.1." />
<meta name="citation_reference" content="citation_title=Mendel’s principles of heredity;citation_author=W. Bateson;citation_year=1909" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. A. Birchler;citation_author=H. Yao;citation_author=S. Chudalayandi;citation_author=D. Vaiman;citation_author=R. A. Veitia;citation_title=Heterosis.;citation_volume=22;citation_year=2010;citation_pmid=20622146;citation_doi=10.1105/tpc.110.076133" />
<meta name="citation_reference" content="Butler, D., 2009 asreml: asreml() fits the linear mixed model. R package version 3.0." />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Ö. Carlborg;citation_author=C. S. Haley;citation_title=Epistasis: too often neglected in complex trait studies?;citation_pages=618-625;citation_volume=5;citation_year=2004;citation_issue=8;citation_pmid=15266344;citation_doi=10.1038/nrg1407" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Ö. Carlborg;citation_author=L. Jacobsson;citation_author=P. Åhgren;citation_author=P. Siegel;citation_author=L. Andersson;citation_title=Epistasis and the release of genetic variation during long-term selection.;citation_pages=418-420;citation_volume=38;citation_year=2006;citation_issue=4;citation_pmid=16532011;citation_doi=10.1038/ng1761" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. S. Carlson;citation_author=M. A. Eberle;citation_author=L. Kruglyak;citation_author=D. A. Nickerson;citation_title=Mapping complex disease loci in whole-genome association studies.;citation_pages=446-452;citation_volume=429;citation_year=2004;citation_issue=6990;citation_pmid=15164069;citation_doi=10.1038/nature02623" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=Z. J. Chen;citation_title=Molecular mechanisms of polyploidy and hybrid vigor.;citation_pages=57-71;citation_volume=15;citation_year=2010;citation_issue=2;citation_pmid=20080432;citation_doi=10.1016/j.tplants.2009.12.003" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=Z. J. Chen;citation_title=Genomic and epigenetic insights into the molecular bases of heterosis.;citation_pages=471-482;citation_volume=14;citation_year=2013;citation_pmid=23752794;citation_doi=10.1038/nrg3503" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. Choulet;citation_author=A. Alberti;citation_author=S. Theil;citation_author=N. Glover;citation_author=V. Barbe;citation_title=Structural and functional partitioning of bread wheat chromosome 3B;citation_pages=1249721-1249721;citation_volume=345;citation_year=2014;citation_issue=6194;citation_pmid=25035497;citation_doi=10.1126/science.1249721" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Cowman;citation_author=M. Koyutürk;citation_title=Prioritizing tests of epistasis through hierarchical representation of genomic redundancies.;citation_pages=e131;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx505" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Crawford;citation_author=P. Zeng;citation_author=S. Mukherjee;citation_author=X. Zhou;citation_title=Detecting epistasis with the marginal epistasis test in genetic mapping studies of quantitative traits.;citation_pages=e1006869;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006869" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de Los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="Dahl, D. B., 2016 xtable: Export Tables to LaTeX or HTML. R package version 1.8–2." />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_title=Reproducing kernel hilbert spaces regression: a general framework for genetic evaluation.;citation_volume=87;citation_year=2009;citation_pmid=19213705;citation_doi=10.2527/jas.2008-1259" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Doebley;citation_author=A. Stec;citation_author=C. Gustus;citation_title=teosinte branched1 and the origin of maize: evidence for epistasis and the evolution of dominance.;citation_pages=333-346;citation_volume=141;citation_year=1995;citation_issue=1;citation_pmid=8536981" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. N. Doust;citation_author=L. Lukens;citation_author=K. M. Olsen;citation_author=M. Mauro-Herrera;citation_author=A. Meyer;citation_title=Beyond the single gene: How epistasis and gene-by-environment effects influence crop domestication.;citation_volume=111;citation_year=2014;citation_pmid=24753598;citation_doi=10.1073/pnas.1308940110" />
<meta name="citation_reference" content="citation_journal_title=Earth and Environmental Science Transactions of the Royal Society of Edinburgh;citation_author=R. A. Fisher;citation_title=Xv.–the correlation between relatives on the supposition of mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1919;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. K. Forsberg;citation_author=J. S. Bloom;citation_author=M. J. Sadhu;citation_author=L. Kruglyak;citation_author=Ö. Carlborg;citation_title=Accounting for genetic interactions improves modeling of individual quantitative trait phenotypes in yeast.;citation_pages=497-503;citation_volume=49;citation_year=2017;citation_pmid=28250458;citation_doi=10.1038/ng.3800" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Association for the Advancement of Animal Breeding and Genetics;citation_author=A. Gilmour;citation_title=Asreml for testing fixed effects and estimating multiple trait variance components.;citation_pages=386-390;citation_volume=12;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Gu;citation_author=L. Gu;citation_author=R. Eils;citation_author=M. Schlesner;citation_author=B. Brors;citation_title=circlize implements and enhances circular visualization in r.;citation_volume=30;citation_year=2014;citation_pmid=24930139;citation_doi=10.1093/bioinformatics/btu393" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. F. Hansen;citation_title=Why epistasis is important for selection and adaptation.;citation_pages=3501-3511;citation_volume=67;citation_year=2013;citation_pmid=24299403;citation_doi=10.1111/evo.12214" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. Henderson;citation_title=Best Linear Unbiased Prediction of Nonadditive Genetic Merits in Noninbred Populations;citation_pages=111-117;citation_volume=60;citation_year=1985;citation_issue=1;citation_doi=10.2527/jas1985.601111x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=W. G. Hill;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=Data and theory point to mainly additive genetic variance for complex traits.;citation_pages=e1000008-e1000008;citation_volume=4;citation_year=2008;citation_issue=2;citation_pmid=18454194;citation_doi=10.1371/journal.pgen.1000008" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=W. Huang;citation_author=T. F. Mackay;citation_title=The genetic architecture of quantitative traits cannot be inferred from variance component analysis.;citation_pages=e1006421;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006421" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Huang;citation_author=S. Richards;citation_author=M. A. Carbone;citation_author=D. Zhu;citation_author=R. R. Anholt;citation_title=Epistasis dominates the genetic architecture of drosophila quantitative traits.;citation_volume=109;citation_year=2012;citation_pmid=22949659;citation_doi=10.1073/pnas.1213423109" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome;citation_pages=1251788-1251788;citation_volume=345;citation_year=2014;citation_issue=6194;citation_pmid=25035500;citation_doi=10.1126/science.1251788" />
<meta name="citation_reference" content="citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=R. H. Schmidt;citation_author=J. C. Reif;citation_title=Haplotype-based genome-wide prediction models exploit local epistatic interactions among markers. G3: Genes, Genomes;citation_pages=1687-1699;citation_volume=8;citation_year=2018;citation_doi=10.1534/g3.117.300548" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. Jiang;citation_author=R. H. Schmidt;citation_author=Y. Zhao;citation_author=J. C. Reif;citation_title=A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.;citation_pages=1741-1746;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3974" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Keshet;citation_author=J. Lieman-Hurwitz;citation_author=H. Cedar;citation_title=DNA methylation affects the formation of active chromatin.;citation_pages=535-543;citation_volume=44;citation_year=1986;citation_issue=4;citation_pmid=3456276;citation_doi=10.1016/0092-8674(86)90263-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Kusterer;citation_author=J. Muminovic;citation_author=H. F. Utz;citation_author=H.-P. Piepho;citation_author=S. Barth;citation_title=Analysis of a triple testcross design with recombinant inbred lines reveals a significant role of epistasis in heterosis for biomass-related traits in arabidopsis.;citation_volume=175;citation_year=2007;citation_pmid=17287529;citation_doi=10.1534/genetics.106.069005" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. R. Lamkey;citation_author=B. J. Schnicker;citation_author=A. E. Melchinger;citation_title=Epistasis in an Elite Maize Hybrid and Choice of Generation for Inbred Line Development;citation_pages=1272-1281;citation_volume=35;citation_year=1995;citation_issue=5;citation_doi=10.2135/cropsci1995.0011183X003500050004x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. A. Lee;citation_author=C. C. Cockerham;citation_author=F. Smith;citation_title=THE INHERITANCE OF GOSSYPOL LEVEL IN GOSSYPIUM I. ADDITIVE, DOMINANCE, EPISTATIC, AND MATERNAL EFFECTS ASSOCIATED WITH SEED GOSSYPOL IN TWO VARIETIES OF GOSSYPIUM HIRSUTUM L;citation_pages=285-298;citation_volume=59;citation_year=1968;citation_issue=2;citation_pmid=17248408" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Li;citation_author=K. Lu;citation_author=Z. Chen;citation_author=T. Mu;citation_author=Z. Hu;citation_title=Dominance, over-dominance and epistasis condition the heterosis in two heterotic rice hybrids.;citation_volume=180;citation_year=2008;citation_pmid=18791236;citation_doi=10.1534/genetics.108.091942" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=D. Lin;citation_author=D. Zeng;citation_title=Likelihood-based inference on haplotype effects in genetic association studies.;citation_pages=89-104;citation_volume=101;citation_year=2006;citation_doi=10.1198/016214505000000808" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=L. N. Lukens;citation_author=J. Doebley;citation_title=Epistatic and environmental interactions for quantitative trait loci involved in maize evolution.;citation_pages=291-302;citation_volume=74;citation_year=1999;citation_doi=10.1017/S0016672399004073" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. L. Malmberg;citation_author=S. Held;citation_author=A. Waits;citation_author=R. Mauricio;citation_title=Epistasis for fitness-related quantitative traits in arabidopsis thaliana grown in the field and in the greenhouse.;citation_volume=171;citation_year=2005;citation_pmid=16157670;citation_doi=10.1534/genetics.105.046078" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. Martini;citation_author=V. Wimmer;citation_author=M. Erbe;citation_author=H. Simianer;citation_title=Epistasis and covariance: how gene interaction translates into genomic relationship.;citation_pages=963-976;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2675-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. McClelland;citation_author=M. Nelson;citation_author=E. Raschke;citation_title=Effect of site-specific modification on restriction endonucleases and DNA modification methyltransferases;citation_pages=3640-3659;citation_volume=22;citation_year=1994;citation_issue=17;citation_pmid=7937074;citation_doi=10.1093/nar/22.17.3640" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Melchinger;citation_author=H. Geiger;citation_author=F. Schnell;citation_title=Epistasis in maize (zea mays l.).;citation_pages=231-239;citation_volume=72;citation_year=1986;citation_doi=10.1007/BF00266997" />
<meta name="citation_reference" content="citation_title=Microsoft R Open;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Nagamine;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=V. Vitart;citation_author=C. Hayward;citation_title=Localising Loci underlying Complex Trait Variation Using Regional Genomic Relationship Mapping.;citation_pages=e46501-e46501;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23077511;citation_doi=10.1371/journal.pone.0046501" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. A;citation_author=J. Neyman;citation_author=E. S. Pearson;citation_title=Ix. on the problem of the most efficient tests of statistical hypotheses.;citation_volume=231;citation_year=1933;citation_doi=10.1098/rsta.1933.0009" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. G. Okeke;citation_author=D. Akdemir;citation_author=I. Rabbi;citation_author=P. Kulakow;citation_author=J.-L. Jannink;citation_title=Regional heritability mapping provides insights into dry matter content in african white and yellow cassava populations.;citation_pages=1-18;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.06.0050" />
<meta name="citation_reference" content="Paux, E., P. Sourdille, J. Salse, C. Saintenac, F. Choulet, et al., 2008 A physical map of the 1-gigabase bread wheat chromosome 3b. Science 322: 101–104." />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. C. Phillips;citation_title=Epistasis--the essential role of gene interactions in the structure and evolution of genetic systems.;citation_pages=855-867;citation_volume=9;citation_year=2008;citation_issue=11;citation_pmid=18852697;citation_doi=10.1038/nrg2452" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. T. Resende;citation_author=M. D. V. Resende;citation_author=F. F. Silva;citation_author=C. F. Azevedo;citation_author=E. K. Takahashi;citation_title=Regional heritability mapping and genome-wide association identify loci for complex growth, wood and disease resistance traits in eucalyptus.;citation_pages=1287-1300;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14266" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=V. Riggio;citation_author=R. Pong-Wong;citation_title=Regional heritability mapping to identify loci underlying genetic variation of complex traits.;citation_pages=S3;citation_volume=8;citation_year=2014;citation_doi=10.1186/1753-6561-8-S5-S3" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=M. D. Ritchie;citation_title=Using biological knowledge to uncover the mystery in the search for epistasis in genome-wide association studies.;citation_pages=172-182;citation_volume=75;citation_year=2011;citation_issue=1;citation_pmid=21158748;citation_doi=10.1111/j.1469-1809.2010.00630.x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Prediction of subgenome additive and interaction effects in allohexaploid wheat.;citation_volume=9;citation_year=2018;citation_pmid=30455185;citation_doi=10.1534/g3.118.200613" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.;citation_year=2018;citation_doi=10.1101/376731" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=G. Shen;citation_author=W. Zhan;citation_author=H. Chen;citation_author=Y. Xing;citation_title=Dominance and epistasis are the main contributors to heterosis for plant height in rice.;citation_pages=11-18;citation_volume=215-216;citation_year=2014;citation_doi=10.1016/j.plantsci.2013.10.004" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. Shirali;citation_author=R. Pong-Wong;citation_author=P. Navarro;citation_author=S. Knott;citation_author=C. Hayward;citation_title=Regional heritability mapping method helps explain missing heritability of blood lipid traits in isolated populations.;citation_pages=333-338;citation_volume=116;citation_year=2016;citation_doi=10.1038/hdy.2015.107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Stuber;citation_author=R. Moll;citation_title=EPISTASIS IN MAIZE (ZEA MAYS L.). II: COMPARISON OF SELECTED WITH UNSELECTED POPULATIONS;citation_pages=137-149;citation_volume=67;citation_year=1971;citation_issue=1;citation_pmid=17248515" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=A. Legarra;citation_title=On the additive and dominant variance and covariance of individuals within the genomic selection scope.;citation_volume=195;citation_year=2013;citation_pmid=24121775;citation_doi=10.1534/genetics.113.155176" />
<meta name="citation_reference" content="citation_title=Plant molecular evolution;citation_author=J. F. Wendel;citation_title=Genome evolution in polyploids;citation_year=2000;citation_doi=10.1007/978-94-011-4221-2_12" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=D. P. Wolf;citation_author=R. Hallauer;citation_title=Triple Testcross Analysis to Detect Epistasis in Maize;citation_pages=763-770;citation_volume=37;citation_year=1997;citation_issue=3;citation_doi=10.2135/cropsci1997.0011183X003700030012x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. D. Wolfe;citation_author=P. Kulakow;citation_author=I. Y. Rabbi;citation_author=J.-L. Jannink;citation_title=Marker-based estimates reveal significant non-additive effects in clonally propagated cassava (manihot esculenta): implications for the prediction of total genetic value and the selection of varieties.;citation_volume=6;citation_year=2016;citation_pmid=27587297;citation_doi=10.1534/g3.116.033332" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=X. Wu;citation_author=X. Chang;citation_author=R. Jing;citation_title=Genetic insight into yield-associated traits of wheat grown in multiple rain-fed environments.;citation_pages=e31249-e31249;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363596;citation_doi=10.1371/journal.pone.0031249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_author=Z. Jia;citation_title=Genomewide analysis of epistatic effects for quantitative traits in barley.;citation_volume=175;citation_year=2007;citation_pmid=17277367;citation_doi=10.1534/genetics.106.066571" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Yu;citation_author=J. Li;citation_author=C. Xu;citation_author=Y. Tan;citation_author=Y. Gao;citation_title=Importance of epistasis as the genetic basis of heterosis in an elite rice hybrid;citation_pages=9226-9231;citation_volume=94;citation_year=1997;citation_issue=17;citation_pmid=11038567;citation_doi=10.1073/pnas.94.17.9226" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=L. Zhang;citation_author=R. Liu;citation_author=Z. Wang;citation_author=D. A. Culver;citation_author=R. Wu;citation_title=Modeling haplotype-haplotype interactions in case-control genetic association studies.;citation_pages=2-2;citation_volume=3;citation_year=2012;citation_pmid=22303409;citation_doi=10.3389/fgene.2012.00002" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Epistasis is an important contributor to genetic variance. In inbred populations, pairwise epistasis is present as additive by additive interactions. Testing for epistasis presents a multiple testing problem as the pairwise search space for modest numbers of markers is large. Single markers do not necessarily track functional units of interacting chromatin as well as haplotype based methods do. To harness the power of multiple markers while minimizing the number of tests conducted, we present a low resolution test for epistatic interactions across whole chromosome arms. Epistasis covariance matrices were constructed from the additive covariances of individual chromosome arms. These covariances were subsequently used to estimate an epistatic variance parameter while correcting for background additive and epistatic effects. We find significant epistasis for 2% of the interactions tested for four agronomic traits in a winter wheat breeding population. Interactions across homeologous chromosome arms were identified, but were less abundant than other chromosome arm pair interactions. The homeologous chromosome arm pair 4BL/4DL showed a strong negative relationship between additive and interaction effects that may be indicative of functional redundancy. Several chromosome arms appeared to act as hubs in an interaction network, suggesting that they may contain important regulatory factors. The differential patterns of epistasis across different traits demonstrate that detection of epistatic interactions is robust when correcting for background additive and epistatic effects in the population. The low resolution epistasis mapping method presented here identifies important epistatic interactions with a limited number of statistical tests at the cost of low precision." />
<meta name="og-title" property="og:title" content="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/675" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Epistasis is an important contributor to genetic variance. In inbred populations, pairwise epistasis is present as additive by additive interactions. Testing for epistasis presents a multiple testing problem as the pairwise search space for modest numbers of markers is large. Single markers do not necessarily track functional units of interacting chromatin as well as haplotype based methods do. To harness the power of multiple markers while minimizing the number of tests conducted, we present a low resolution test for epistatic interactions across whole chromosome arms. Epistasis covariance matrices were constructed from the additive covariances of individual chromosome arms. These covariances were subsequently used to estimate an epistatic variance parameter while correcting for background additive and epistatic effects. We find significant epistasis for 2% of the interactions tested for four agronomic traits in a winter wheat breeding population. Interactions across homeologous chromosome arms were identified, but were less abundant than other chromosome arm pair interactions. The homeologous chromosome arm pair 4BL/4DL showed a strong negative relationship between additive and interaction effects that may be indicative of functional redundancy. Several chromosome arms appeared to act as hubs in an interaction network, suggesting that they may contain important regulatory factors. The differential patterns of epistasis across different traits demonstrate that detection of epistatic interactions is robust when correcting for background additive and epistatic effects in the population. The low resolution epistasis mapping method presented here identifies important epistatic interactions with a limited number of statistical tests at the cost of low precision." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GplCO1tlIvaOGoIjuzg8F7KritKK1a26mDFrSklRg3M","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492438","apath":"\/ggg\/9\/3\/675.atom","pisa":"ggg;9\/3\/675","ac":{"\/ggg\/9\/3\/675.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/675.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/675"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/675"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/675.full":true,"\/content\/9\/3\/675":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223hKUSXXrzFM9DKYnUBC4NTXZKFDnkEuAg2RxwUR\\\\\\\/aSrILNujoTZqzr7O+HE86CLr1IzAZSPRfWIwv+0thYV3pERYamsCvQaIPF9DzKt0q+hnGLsCSSL857d\\\\\\\/kQg0NHUqBCyruSQfCD3P7mZWR2k+YazEuqnQHFeUN8ZrOCGpTGNiY30ki+mgYuqBDgAev9zbRQqFs60mP7aA8EMlAYSJc8cgN0FbLy5CRLmWu6sCbEo5VqX\\\\\\\/\\\\\\\/U79IpJ6nQnI8E6oPn9jb1GTMTCHXjJqEq7pSqEuMun326ql\\\\\\\/FKtAdYeeNEtZc6M1n0NGlHUnQvsfP1ebHOulf8QeY6NXi8BMgDylmiTDJEj3u+4NytjewtkTzFFqPF\\\\\\\/i1JD5x1dt+7V7wN43BzH\\\\\\\/stxZJ5C7ma\\\\\\\/Zb+HCJ6tVZ6RVbaf+1xVq50Ja7o83VEkgBN6n7DC4rJTr+e6FrbLV5yO8UXiFeE8\\\\\\\/a2HfqKQH8LLC+NYpZyim8WRfYrHXgL4Dgz3vWnu00glT9HRfbLiHxJwgM\\\\\\\/QhsugnBjX9JEiyvl0WnHcZjGApWKGtO3Tt4yW7MwDvWJUHRBUaiRNekwNYEMtXg\\\\\\\/x6oPchxskj66po4aqJgzHzQAIBV90ov7pacgw5VP4+7XUYmQ+1VoKP7aWCxMlT0USmXIslNZ6gm4U3RGFE87GDPhed7lA9ntRNNV1V4KVZFBoCNnFAtP0T0gIsxEZUnFb4zhZgo92bmodryAkpQ2Wdrn\\\\\\\/nsaDrPL9LHdaskaxqpny3sKRDqWYlQ1\\\\\\\/ZNnOY5tOVUOZ9CJenjmUclVAOo0zYKouVHNWYijSHMoBWpDI1iPWKjx\\\\\\\/zcZ\\\\\\\/XPvtiqZ4jhj0Xrm3KahFi84Mj51dpx3WlpQD5kIBcbkcc1vH6srtJ4tuI7y0sIu8aj+WnqIjC+oE4NHBVnViFVCpkDUC8FYjhGAzrsa6+8gL1TUs8dIJqJZdDLOMU5FVcHvhC4Lg\\\\\\\/JVRJ3Gh9gz880hJzApCOjTw3El1e6m8+Pgsuq3WvXDBOcBmAn20kQSI+0n7W4aJYZRkONnnS3PQ4LqrwaxNfI3W8H3wWX7XEW5341GDhnpUSl\\\\\\\/DfnyG6hWU\\\\\\\/1YUTXyq5pjipIV7W\\\\\\\/IPy25pawUa6vcutT4TiniTx4X2A0sBp8lMI1BfWIv4+uyQGddKvjN7nKDhofB9JZTjeVs136PgzZtksCGHA7VlnjoP73fyJ8bf2VG5EF8dgHxCDPamHr6RnnYE\\\\\\\/udK8pVIULxmH01x63BLgYv7DWX3bijwizs+xr+du8188avRYVNV\\\u0022,\\\u0022iv\\\u0022:\\\u00223lpFc4RVjm0s+PREBuf7fQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022720f91bd07b39c1e4b4b2bf81c1f7d3efda445d520a2038200810ca3ae2ec914\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hi2u+lG97Rh6hi8Y+50GEcmJwhJ5V4+YUKtm05Mm\\\\\\\/hr37LYn2ZQ4ftl\\\\\\\/7x4W\\\\\\\/izps1zMG3s+naZYLIU7O+w7MYCLA9GJXQ7nxXFPYWkpEbYHZ2tI1rEBSFUysyozmNkXi3IVoPoq+6o4WKEb2NjtX55vEXcSTYMMNYUmzkVGVGJgC7nNwHEzPr0eG3l1PFoqJk9wtjGAvwWldcvnShCVn\\\\\\\/neaTg23RWLRIF0bFuzkLIKq+VqNYtJNcO18hvwLpliY+gTdlEdD4Sh19JuM90PGwHA10HiMC+TqOyO9\\\\\\\/G+yytw+KVMBptLtTrwO+39W9C57\\\\\\\/QwTNdj1RrtVIKODYkn2Qqfl7JUsY3peOAB1qGHKyMQvNc9oTjgDIdPGfN4Y66wWD1p076PI+ZO0uHJ3dZOjo5rATmgRW5AD0uwEte8vgx5jBER4C64hSS9eEeBrZIpq+W7tSftje8MY9crx3FRrcayOPXwoPNKGlJq+kx60KQccVpOBv7sa8TEp2gR8P7Tb4dH6htnWEwI9fe+ejm3bOSBZ9C0J2v9IMV+AkCSvV2pSGBVOmMIzTDKEW8m3wLwOhensM1Pc+Dsg4jruhaFqgTVYguLV+7tbxdIjsL4wydXRT+xpGJHhWhQgygCisoMS9vSs3pD6HbcQs5rxcqZoFBmy2J6\\\\\\\/AvYhUUW85YJQ9c1hQjQG3LwV0R79avS2kyh3OuPuyTvw9NSUtnDXhxBqC\\\\\\\/x7Wndi5o6Az9fU7HvQCHMmhWPuRmaNV\\\\\\\/UITBSTAcGzh3ycH3HgQiBH8jUg8JH8XdwKutov2GmNHwlvxqZEQNkvKIOjwwieZ4J6IVcCu8TOkXd\\\\\\\/S4qC1yj+2GRNfPhltJu3ExQErvQLR1d85PhANtJD7JnYmGRXOG640Gx6NXdY7HLILw9LeGzG544lghI8\\\\\\\/6MWXemxyBykv0LzF5aX0V2m64XsG8pDLRVw2mTMgy4lchVxndxcBJySM7OVdpQrsUKoZJugcA7hwiFiw8IdejgNYRNg4Qr\\\\\\\/vUQAQNq6W720Z3UesJoOLeJ4POiaWI6\\\\\\\/CH0ulMAAskPTW\\\\\\\/Qhxjxzs+arWA8Pjjxrp8dTvd5khfKrdEZ7NJQ0gt1cCD7F7uNIlQmd3N3PPx7Y98OsGqUAr9u0pMFKawJqCx5vUUifyZIlbrw8AGR5NWgerAov96Bk5tII0A+XoeMmBfZL9nppA+ylkHVjrOH9\\\\\\\/+uRw5tO2MaNvfKVi15hq57sXeKth8qDQB9si1grI3lPt3BTjjjy9L8zv0OXM9O4IaySrIWnyRboFkTuZ56sOz2uAZLt+CA05G0GLEnjoT33g4gmaTXSupNY7fFgBqBmPEqWkdOaBokeLrBsfi1aUIFNR7J3CpLQ2orXIg7yyGuDfS9WLZzSkqhuetxNUzTuualgIB0ceHn23XU3KwRZj10MIISLXKlSoDEBW2w\\\\\\\/dBrLldzkZKRdC1Sq24s\\\\\\\/Ki5NGFcDXEZhOafE7N7yMEeXfp0Q3hSkYIUypsgCKNQ+h7CM54\\\\\\\/GWCHJhF6qEBcCtXjl+ngQe3m9VB7X3wa0OfGwK1lzrj9RvfVIEZD8+J9ZM6V51wQOD5JhCmOrvgotuNjPTLU0WCuZRfqX4JmJK0ATlJbuf\\\\\\\/8tMraDv660xnZzX9bduQFWIvnQZWeEILpL3NeLKX9B9y9YkNgpW0Ry0JZGgkEkwcxBr2Z+wUfmaIBBWYjATsWwHTlOta8XGLBBOgUCqrJ8j92XWE\\\\\\\/l+B3KUe1kCQRmBZ4QaF4KCyyHBcoY9tcF7C0jGk35OiywEdSqGlDxNpcpGSmeHY8h8VFFd7PVDHoeb2\\\\\\\/CqNlZTswgR8mCYouqWy18nQvNevSD7EoHbJHq8gGlDHe7VwvnwB5G+LFnPm+rwIIyjekDJjmTdMy\\\\\\\/Kbqv15bZtmeCPkSf\\\\\\\/iwl8TxENTWH\\\\\\\/\\\\\\\/2M9vYxwvIUnAThT2jyTbp9LHZbMkG1aolmD7p5h+w5xUjrknPJUAUDyTdHZ543oHcBVeJtE4OI+umt3AAsA0t6KZLZ\\\\\\\/d53z6MB+5C9VVZwhqD2uhalWstIMH330Vdp\\\\\\\/gzoXh+gsM4\\\\\\\/\\\\\\\/\\\\\\\/PkdLisimbSp0VW3ZFXRyl8vXk1AcF7JY7mgZrEpQxHwcpG2ujXTrIzZ71MNWJvzJ5nnaNiN7a4dBUThljh1bKDfBVlObIJpR76SRlEa3ScxsKz6oI+yEvs1syJ7MkCSKBVh27JFxAqBQ49DL6agW\\\\\\\/5OSo\\\\\\\/l61EnFAGMNFwt34pJLctaTSV6+jhBu15ZcUCsNuQr5z2dzkB64YHOkOGxfQeH5TIUAOlaK5j9NH7HbAm6sRndbV21pREPodRS84GGHzBpH3H\\\\\\\/8BNSQAChOyIVOEQ5u+9IsefuBVBzBbprKC4\\\\\\\/nYf1M0UYWHnDJal68QhttBxTw54Ahek8e7kI1hUTRRTWgd61TwT7SqBKYBdEh3J+0TR0zCQu7G5KtXZwE76o8s\\\\\\\/G1IE0Tn23KZxdWixZHgj3pXSP+bljGWxQKwSZZ7EOPSoCS5PxJDzPkL5ktEfkqSsr3p\\\\\\\/uAZqBOWivYlvpTanbYO89INcLropQ408IDWRHqoh5sZDOEbm7XGE8ZAujWI\\\\\\\/0iHNnjIth8d2kV15j5C5vbSkISyoHEhtyTGlhIeI\\\\\\\/DhlgBaiargQz6FDRA8H1jbGMNlAiZn7N9RHaV7R73kG5fbA85wMbPJ6fMpC8QjoCBCbEOxCidPIWdwmhp0q5PcxjTtzWcLPiqYUqR4vR4m\\\\\\\/z86gA1aPVvHk5PDBnRxCIOsNoyaT\\\\\\\/fekI8Vya6cwyIK9CNSXNPmCrtCliaRBQK6fBc\\\\\\\/QHD6+8HimogottdOoHk7MLpZN8Bl5mxIRx\\\\\\\/pVLkXhP1TcfQmtaHmxN1xq5arZL3nJWJmcg8es5nNQB8vKhWUiDmXKl6kWUfocdoc8487BZPw3hCE4l9TpAL16rwGFbEKIssiLtjAM3iptMTmde5kgpTNB43r7wJXZEtoESldrzpDAeKlK0G1vfl2ctzSL\\\\\\\/x4n\\\\\\\/QPzXK0Y5CDzG9\\\\\\\/zOC5GjTE5H71LHPv\\\\\\\/KkDsEuHyKcaV\\\\\\\/8BLpaCbw4v71\\\\\\\/66DOEo78zJhCIZKmN2q9kpk3LESbqGAK6LsV2qd8nyRA7EtIuQ8FtVdZwFagzkNL6k8S5xpK7Z9XUCyEsqPwyty9S4bOUvkVaYDi8Eg0skxavg+oXO3ZvF7Fu8GCUP4iJqVRQnbL\\\\\\\/X3rGhcP0sP+8\\\\\\\/D4bODizObAUA7sm5SUNtjeVURtB1LTOzfg99uI2bdDjtwbK\\\\\\\/HETUo\\\\\\\/foX+TquywrjoXyySS3wnD8H3GaomC5ONG49PfOhlFN3oMFd\\\\\\\/NV0084trQzGbGutD+SkDXmEPp5RV9sRxUUYG8nnaBKlwVFv26\\\\\\\/xIjhlwPCakRfbSpWJ2INzWCmu6XgCuI6AKyWJ5CmYr3xQPIqDn6Z4j+pDbx3j9vdC84vRbREAqx\\\\\\\/HvUdUrzbErLcB3Gm7SeqScoUSp1pZCiI1+Pxy2xgXjZpVz1sHVwb7fa2U1UWcfR2XpSO5iOG0woyJ+0HQ2eqRRt329U431wIKnNYzT5KcF7nzRSDww+gf1WhH\\\\\\\/A9CoBBGVTWydzsC5RfoN6N\\\\\\\/FpHoGgUm8xcinOK3NNQH+50vp0Nu\\\\\\\/pZoop\\\\\\\/0iBGftBAfQNvWIZOEewp87fKKUyL1f8697PEbx\\\\\\\/0A43VybyhSNEZz7PWz9gDbjVWotlE63gJztAmOz4RPuDvehNhzDdyUcblNVtT7DzU868q0g6FFYwaN2geYMKQ6fjbKu4Dor5aPDW9RLp56++TkZr3nxJ80EXgXGBrpsVRbUZmUgsU9QufKupi5NVlxsvjPP8HUzFd\\\\\\\/Hfnx\\\\\\\/K6XH0AFqkkmKddDFMXzhMorwWyCDjdLDNx0QQEmyIFWUtGp6JimrTSz4HXTpSPWusA0csbtAmeS1RSUOxT4h+6E6JEHf\\\\\\\/Ak4LZeBKnMWxS4QQAHEra9XiGJ0d\\\\\\\/aZ+ygfhWLUt6XLO9LidZ1hn\\\\\\\/Vg2K6wOoKs5KTjBOi4RSSM5epGSwGHDji9wlaBqxo2vLgi3TIMuOLzz3z7IGHIYlBGYAlrMTI28AltoQoddzxwxC9mg1fAtHLNpnow25yyWNCqG8EX4pCUXriETHpizG3+ewTlbVJkEHZ\\\\\\\/dhlhqnoAjp\\\\\\\/RSJKtz1yprCqIFDYIMb9tTwvKPsqjQoRvoYMpusNJHVx3UWih0dvkX4mupd7kI50RXrMUAlid\\\\\\\/4JTIGPQn85nU4P0a1jDs6MeIrZERcmFvfiYotgL54uVc1Wwjaz\\\\\\\/X2VgEvccpMRYiMtX1j2Yi6JUJKzESUz6o\\\\\\\/a6Sqln+6bfUkSxaQI5cD9icWO0ZiPycsiMOE8tUGG44l7MlEwJpCdlMZJvhYEwZ42mHyeKTLkrQ9ghhrYTm5VtpbFE8mGiiLrcFJ02eBwisCmx+17+83vfrmnN2z3Zos4en9nAUJvdSrMPM57XDS7JCUxfego7VWHK1QHMkbU7stSF69KGxla3dr8\\\\\\\/j\\\\\\\/8\\\\\\\/K+SoiCsV2KCGNpOG8o7rtgiM5Vv9pnYMIxJ\\\\\\\/ss4HY6ku5ZL2\\\\\\\/ncMKMZ\\\\\\\/YZChUt8Oy2kRYfd+OpK5nyXW6BBJiX6VcMbgp6m0N8zQHOTl5PeH9c5eK1WS2s2jBmPnm7nezlVMmu\\\\\\\/T0TiTPi4BD6UrwofSn8Myy9bvTFAPOvdjcO+hNY31Kvxcfcaky+hu13hUoGObPxm1gGnheb3zqa3uPibp4a\\\\\\\/4wf1ILU0KC1MFFYTxe9A2ADM6+4N440l2X2zX\\\\\\\/By5KqnOp+2acWC4\\\\\\\/qF4PiWZMJFP4vOVEQBaDB2AkLSUkQ7dCSaBOLPZNpdXjXWE8Ifa8T7d9xsCPU7hPXwv2xrNzf76TobEVqDJirkCWzIqRnqe0yrNLs7KdGdkI5JEtJU0\\\\\\\/ciQhD2EPvfB1QaFzsyPmxl4dZH+UE99Y8Ad\\\\\\\/wgONUXeJi+vIsIXElsV50PwC+GjQPNtU8sV14wmUSAgoKRY3t8XB1TKhx4EtrfzAfgzEuQ8\\\\\\\/4y\\\\\\\/pJW3eTMLeGqNRJXtDF9JBy8h3vz1+PNhrTM++3jQrTQineQzkjqwcZ5S7HTJcN0F4+unYxiNL2t8kyrzxqd\\\\\\\/ftJlHkxZgc\\\\\\\/hFt3iqdpm7fYwMqFByp3P8z0LWhLVbtejRFL+BOFcHMxAQMlM\\\\\\\/gCS63GsRaw1M0S\\\\\\\/8+nzRIUlv9eUU5T1qneEyCYIEW\\\\\\\/M2YfV5XH\\\\\\\/vcUdrOhMIYK1M8xzaYlJFt5hXi3fMMQXYoEPOJfeY+xzeaLt5H2fnQxSvslAu6hI6hzIVTMKA+IvIK2frG\\\\\\\/ez+g\\\\\\\/KDtxaB\\\\\\\/9UCUDt3OXNCDkxrkcQXAJK6S5KpgmTRbAr3T5HYHJhNZReBMnpS9zPLDIiCwhMJa03NC+iGRvJYz5e\\\\\\\/ofMj1vGoGaOeS1yi5oj0vvZQrGVhlxvbgIEYyLnb\\\\\\\/MG9UoNUq8v+DJdMpC+fdDFS9AXANp3tEN6HmcjxFJ0vdlxjicIY46vQAqAPeTVl51PkmDTQC5U3o6DIo7TJ08doM93yEh57djaDyTB1gSw6c0M7q9litWqzGyrJY2xd3HbSOL\\\\\\\/OHl0idA2u262dlvzdjwQJiR0iFyPpVZ1IljxNzfplt3tLa2rU+qAJklR28mRyTapPd8k87pZdfkYmBCtnqY796w0TJ1izgaQnqRrb8I1DvzDAWTe\\\\\\\/pROoAhjJS5qBK1BkmdLyZD0T2sYpavEPDawFLLAtM48C6CgHRkCLOFxOuGQIL+2Do9xGPsFQLPpVQh60qLSRX0brOQs2gznPGwIkj\\\\\\\/mAskyUkxZj8WZi\\\\\\\/Lxy2q5FsUXh\\\\\\\/jhAZIFxVhr\\\\\\\/ipthdrOi2tlMskXXP+FD7o+qt7HJyAUvtCFTE4RB4Cf6dUASJgXEJJczoBGtnFXg0LtDFnaqGHXCPnvaDiIAir+DHzUCMnK+0oNE3JKLADBVx50sqFSlJTWjWQfmdnkfZPqSf0Kn9wInVEvPfPyY9VQCzev7uyCP3GF+JrnpiAoOuM+7rCceNyGTbVKS4Py09PRlH7AUv25RsmZJwc0KRaoCYD5UHV+cMGDW8Ou4GHfvpdOVGcpJhhYNSmHIudT\\\\\\\/Yu7GwxKUAc3214ZR9XzohnRCFBwNYzTpKeWMzya\\\\\\\/ceccAj7BXz1ZNnNAino3wCJRpov7w0bA1pp25hs8EbFoaR1jeHCa7dxVPpHT20EjtnpqPGOHJJae1ad9dyv0rQsDk72MgpE9Jz+6qCS+EbIFRCR0wl92GUQ9rY7+JMaaaA36M4ay+AeRfOMxOUf4Q4s5WP1OsqPKNBlATL6tvSqu+LCehQWrZuz1HYC16qC8utfNxC5BmrtXW0iTRFu7oFc8u7N1cebfX+DZkovb1ZzjyGc44r5KGK42R806PbkOxYocekeEBcC9amel82Vwc6YVj2p3qWS\\\\\\\/GQWorbVEwxSTqz7MOJwqPTFbLLsOnBwxr59bwCLBNrv62aX3S8sulTPR7t5g\\\\\\\/OFhD0zXfvPEeGsElLOY7xO81otyH2DHGwISrUf0j\\\\\\\/9gNpMWQ\\\\\\\/0fdzO83b+qgJsmHEWLNtz92E8Tb2qzzglSXzocrWr7vtIa09GcUpfdgVDtq8iLtsn4UpsEgo2BJLkbLi5QU9wztVE87urUksqiCl5e1sWE7aEUXSHH9luqFAwqEQmxYeNGzLdW8Ym7kGNsYL6GhqvfSoVeDlnsFTNjnHYdv+NIurEgBiEX\\\\\\\/hBryUuf+EzgimWGh1QlJR6kpvULD0ewPTGsmiOFN1qZZzOisVjtYLLM6q6OOrxiMBeAKciUUmTY0FmYM3vGOmCo5LKKcBbutYwHuVM8JW9yb8in9mdTiQQ9g5GloUBGu5wF7MygIxB0+tq650WAGxRupz2\\\\\\\/qaEpVDVW3yU26SQt6of6104EMZ1iB0KXULL7IAuBMU4eFSc6QEINDbEGWu2MG5612LwzzuJoD\\\\\\\/aSjIIK1AdWeaE0UFzDwAq0UsruDRmbjdRJjJ+dQenLjLXfTUyjpXJErG9MSaL4pFAb7PE2znJA+DPZUdF5X5\\\\\\\/4uGJWvsvBR+mAlvLw7YXq3IP\\\\\\\/F8X3YIroESKkWKxuthz5pMa\\\\\\\/HxHJarp5CAU2C1KYvDZLGIY8p5dlWG9WNMtYBWcn0BoXxXTSOcfbhk8wkixHSvq41fntQ1Cnt2DDPLbvLWvnlPHvwJRIoLY72\\\\\\\/aZIiXY3UrZZvJG3rYrMl6V5CPnWO9ZD8klmtktIiqtdR8llxAgG+UKaXUSzqxgB+RFaQEhVe8nSgqDDOtCv+1vAw7nrrATmHMqoydOQqP0JThQiC+ozV5mNObZnJTaEBN+VahxUXVFMr92LyKQRzwldVthhHYH0V9G0jRfONrpNm4B6ClJcmlm1ZGCmekWYXtb0vAMj+LCGRJWUCx0Ws5zNzBZs45oilwKJ6b6NXF6MreugP2r5uSqCtQa0QPJnPryX9WqpKw5ux0ly5bKQ3Y+RhkuFcBsdhZHrjwQE51addLrAc7Lc7YQVxnWd6ye0nCSJraDrT7TnjtNNQTgDw0829mKZnit4tG8jG3rywWfC3BcBOR\\\\\\\/wMXXnVI8IXwoU03Z94v1MSHHV4hQJWG0+5RiiwgXj3n8ByFUkzJ3lEMCfKZQG4EUi3uM7qU9DmIWH46P51\\\\\\\/mR1GxQixQQMDvtuxgzwIhZhUAVCFfNxlbS9PL4lJdiisoBC6oc90n3t0yMKovr0v97Oggj35b1EZK7USnEI0jRgxJQj3HADMaJzT\\\\\\\/jqOI8T3pKUYUcWXilZdv7MBa8i05ogo2z8oX9AN20YGDRPAuPvQFKE5v3+H+JXIAzb3AOb1dIboiKaCVHhafe03Jcw4kQXy1XyMzV2+mudCxq56F9UltZZ1vniBZQ+QqvaPRDcYCVbODgY05845pLqfTcZ9E+sHkJyRdmxiePROlFmc4Vk47VSXlLfeo\\\\\\\/a95YHow1HTWYw0KZvs1WTW4LZ+xXfTmzqIGRzlZ30CFJtVSXF6bvYH7G02jVIar0YW7GLXuHOEMqWAhIvidiQN2nGgzv0\\\\\\\/o9PcnFZUywJ\\\\\\\/wtHwLabVwYUtjf7Q5HuBRoV6nN5Vm\\\\\\\/+jrAT29XH5nu5ISjGNPrFxryat5RFHEnuau\\\\\\\/7Bv33\\\\\\\/zxo0rUmLnO9+SxFkiqKfNE5DjalVeWp3\\\\\\\/IbNP7oz7E+urTigf0lC7q8F7oriAwKXaCpoNEKzzF4T8O9yrO9T7kp8JNzpVNwaMqGLuxPRdipUCPVLQkyzUh36Ocf8tVzDsKirTdz5xE4JX9k+1KT2haCCiftBV5qPRljrSWmUY11vnfShVyrcABPxlgbS+Tm60Qdjeru4EHKssVTYhK7eccrdntUv+WxCfHAvWJND57gR3fsg5bjw8olUiLOvjUBK4O5fYL9qLECDw6ci3MB53mGjIlmp2K3Y6rTjXivw\\\\\\\/QSXaJx9eCigl2KC7MbK3d1vdxTbtDUGMgP5+6NCqSrTAEs7ZJPgLoktPpcqWs5pt16cXpA636XxyogMpwytfd2XaP\\\\\\\/zycbmNwWrfv0ZQHKwshF9HP\\\\\\\/Qjq\\\\\\\/TAxhMxZ0wNkMq5GHjXONkas5KWQgmQVYI+m7igZak+bwgOhWlTsiuwxL1dMmotNfE6gpgDRO\\\\\\\/31dc\\\\\\\/FEge3tLn6JEJkzk9USpUSINPjM9WTIz4yxkF76sWJFEIU8V082q0fycvTm+X2kiUD32XnhDJQEfHVOdeSSKG1DBfJrpQvRbvS809RKn0yeNKWbRjY85BMymw8J9VkVZGpwNVeKpbemhebiDsgDyxVXvtNK1FLzI6rNMfEGMuBRvv9c5Hnkd+SRQSF1MEaHW3O0cKfOHwkSyMugXzLc4L9VUtKAKzw9lyMqO7YMakwYgWQE11LawNmM2+gLSVnPZJQ8xOOsjVHRhZ7gS7s9oTEXtfZSwCdyoJ5U9qDA5abcER\\\\\\\/sO3+6IZ92v9REq7540ub\\\\\\\/eU9ocu4f26lnAUfnhZd7l6P4Zq+8tzoHOGoLqVKkmxgj5CKGjPsoxOQ7DwipziLT2lrAxBoAnfc2tadRQUSeHWXdmmMyZdoKob1WXSs2pNoJ44EyAomIdgZiglL6oX+T4ugrjA87K0EaDUtlyFuEReB5uuMo7mynAHX4KkPxPGQ51VU3fZsXFsdC895juK7ktYLk7cMfxH+lyvSI3pgIVu9e5QXzXS3TVMdtsetL3X2599ALJOLmrAXKwRNO+tKKZ1doyWmgxnOqtmlY5\\\\\\\/TMOZaSbA2063HcbgjRTbAcVJE1xIQM1J+xXYE\\\\\\\/d7iof9bZKkZ3IMQPQV5u95F\\\\\\\/yawgltGdQGmlD0suikjHvSiSvmX4ZOX1FdU5MtmKqhVINIdeb5GgteuWHaLzfDuyRO6\\\\\\\/AoYYNTymOaADuwrAbqUR7EOsSaEJ9XAHxHL9vwbEl1xMOykFru3uFsvU5lZd03CVGTIC6wZLXXx1XiYUfqDrIqH1YRdnFKQLK8efsczPau+mKqGTQzlRAG+FPVRU2OFkZ7+hbrPF4WRmX7SsmWk4JQpiNbe7f+nJFn\\\\\\\/L9iO56z6uD0BHyUHA5qAbdtRLs47cYmwnofV+y1LGuTsL+Cj7gCRB8tAnPM3G1DaRzwQ4V66yn3g7mph+TeBuiifoHMHeWyOO+V1\\\\\\\/hxTOCbAJ6nBv+4iDIXRauwlaiHpxE5rQzYx9h\\\\\\\/dph3QY0fP8gxaQnn2MaK1ZoDSCqvUhlPEGVJ1GyDcUn+yrm8dvRj7jjMXOhjsa7h8KZqeD0viPILpP0VXq7o55HmbKCLigFVNQy0Ybbkp7kIgIB1pI6sStrEaj\\\\\\\/uTxOlE37aNgYn1e3WiMfgffNyEAw7aMU1NxBTtM3uXBxgbmrVFqTM\\\\\\\/hAQfhBz4oN5kN5fxZYjQ8rM8JydDDsOfBFcgDPgXJXPDJt\\\\\\\/JMqHGvYO9V85adY6SClU4DE23HexnnHPTsjPpl\\\\\\\/qJ8mM3HVg\\\\\\\/eAkT5cZIAzwuATREP++wX6\\\\\\\/IkeB0fiFSh9uHABaFw9sAQiAylFgo1QzbO8EN+DWe6iIGb6fqZNgYBDLpjSsoxgKI2lp7ZB1sj+\\\\\\\/HzR+fNrMuZ10eIHrusfdUfkf9BVzt1jhkyLKNB1IKIy1d34l9mXxHJ7YzJSrKYobmqc69nRo1zpcQsHUR4pZaBbw35ORhSj0aZpqVUfE3NIu\\\\\\\/hXb8dk+6HXz\\\\\\\/iZY\\\\\\\/rCvrvBJ7xDfTB\\\\\\\/Pi2pq4FSnlw4+yY2sotQO8LpaxLSOOBmcRFZlDh6VcXrTwEUglyDSzE5PKFXE8QDHOzc5XmrpjGv0aPBtLPQX8MIBFdd7FGifeZSTPIgH3wqJDFUUSTZVIWemQdp3dTULbxdh9QGQgOwlefmyuinYeF3PcizJUo13g6ytoEyQ+oThy9dUFJyB8+xqui056L\\\\\\\/e4\\\\\\\/cG4ftITKbG3Fr2+qKfqw71bbm7sj9Kc6QZzILDm80g31yku2uTIUnas\\\\\\\/TayXpBcOMCnTTi1ZMnYpk9X1lgDVNu04L0F0HmHZnqHhXBakEGgjsHiJTQROq7x3K4zDiyw+IJhWw8FWDarpKX5ZMqkpdBXHqahJDXdom0+4UyzrqDOElshZ4wOOvR0tDeZeADidbuCib0bv2TWGK3Z5K6uO75MjwQMWVVhpitAtoXL1rh7aefTRfmpaZfDT7VzN0IqOKBT6On3mN1aLTiTk9aheAvgVSFLacPwOK+99QzqFcIHRdZMPtKSab0AJhGUcpR2HMz6N79+QIb89j9PG8mKaw3TBTksb6\\\\\\\/yh2kNzB16NzbMothBIyO4hR+XpjLNJwQD1g6tSWfyDhCRh+ntqPxwRG5T8oSXlAGChwWk8Iul+QuOjn1N84fzKF628kQ+6ZtBMzrA7zRBCHGeWM\\\\\\\/c7VXhm3S3CnB\\\\\\\/gnsP\\\\\\\/0WEGoid6CmyzqNpcmjAYt799VrRzv8CeJKFdaA39Cwtrd\\\\\\\/\\\\\\\/yN9vw7qjapsKaw+3A2ouJLaxP3Dtq2i5P+yvaaEvTGxZrSUhCQ483ANOMwKK429jITtVYaVtd1l\\\\\\\/RXtJ1QgngzZeWHPUtCEwn+k\\\\\\\/FFwUAvbacqKOnUSAWWp3DU86f9Vg7XY74+2YWsC59fdfM8Us5SDbcob17BO4aiYOLWIggHDQ+MbiF\\\\\\\/MReTkEBVYVVe7eaupo8epmqZ2m6B+4Dni3dn+QQEdtNp7pyn+bfjg2fDU351mOuh+PX6aZsVuLAAGgms6J0wAoo7iw2lZaN1dm7gfDCb1kwsSMXXQ8Ilqs3O7wQNfKx13TcEVYVVsCZMEXoj7bWiREPX5Q+7J2\\\\\\\/Eqm4Rl1hI4duyZ3tcUP9PqwG0erOmw5C+B4gr9okdmtPNDn6TcqMjtObIXSGhmd7M6BMM0DIgInKUzqq8ohnExGETuOXkJ9oyOuE+dV4cCUVS0NEatHbwgJDVHTVCdXRjV9YXkJQCQoLtU5iafBr+\\\\\\\/81g7J0X6xz3zFwhilzleEzv0ajl3sApRqAiDFXnAAnFx\\\\\\\/ZgHZyA5zu9Z2\\\\\\\/1h7buJ8NEwPUxUQp+IH0gFOC\\\\\\\/74oekyUkRyVZhvRANKplh0hNnpRq8nfDUF4\\\\\\\/5cCj1CvVMkZeXsELSC7GhV9cSLLh6ObYLUem\\\\\\\/A8ssxRJxw7D1uktOjS2RiKuC8H0qO6Ksbgg28rtUXI4nROBdDdTkrfmTHEyYpWrd9e1Ln9APCUv6i0AAu+Oh0d6Xhxw4Y4AhQBXhCye8tWfIracfnS2fGWBMXh8DuaaNlzigkHIrXOY94EAYpTr4uCnF4ymTB\\\\\\\/7kpzCxglRi+JJm7e0dXw5zrrRgfUeXwMb9bfZsVMxJqLThKLfjqyKuBzMJYfQoO2b2yhDUiZiIb04k+o54NlHG\\\\\\\/Kh\\\\\\\/4NyNHR0ewjBVFtQDjUBKeD1SIulXCWf5+y9DeNVtIKP8mwX+\\\\\\\/40QD+\\\\\\\/CjWuTG45zT6GYMdEyzCIGbQYm3xcLL2aGoMaVzjrKepLPX+a1o0uMVYb9PvhfrDEaXaDfu0jTQuem4tlEi6AxOTTFBpJ3KyXToiJk\\\\\\\/g3f5aaDkRbK8tSSGQdKbI7la1w8SfF49H1BD4AJx+oZSSuJO6rAB8kDLWdbe2Uw7CS\\\\\\\/pVsl0oJkCK8l+bIogfFPj4eYLPCSaTBFLLyBqTpKefOJXqvf+VgDvPZ+s2CEnG+cO1BPacxFWcCnwqt9AXqdmguGm6rX6vViAc\\\\\\\/x7xI9LcBCdxG5LVokDo1esJinUKOh6MPNKbyNNHXtVyeevPrdtFUyAaOzAGK9grAMrIP6C4gZox2Ui0CaHAvgVItqWFbT0pAL6854TGkcumCRDNnJzrMI9gywLQycp7H2qcQuuH9f7ov3NUYvr5adorfaUAWQ0xN7IoIIWznuIENEuYnnzTA1xj0vOmqwg17MUS6aP968juRR9rLiBp8PV77Dz2FfVKpSbMjDIxsyiTlvz\\\\\\\/YDLXFFASvLMGSTMYnUuhhmTCJYjrxwEokB2DazOaBRO8J6NIgGCiLqRa1ahHEBioy5SNjelfy2TvHGMjZCHPblBwketoKqf3cxZvbzwOWJ\\\\\\\/iSRKFrbKMhDeX7O6Ceo01vrlKp01cGp94basHeY6vywKFYcS4pkAa1RguHHtdEo4codzQBsUrGGIh\\\\\\\/Tpm4XCTPH9lQfNXziM+PI7r1rPmxxyg\\\\\\\/HNvHU\\\\\\\/3ib0Vqt8pwSJhGR0xHYur3so0\\\\\\\/m4LgT48QuIyZ7lb9jLxjKJFfjDRPZfh5IgslzjSQ\\\\\\\/oVTYKJgi42XprCBe2juaWnF7vOQQu7Jh9bpeqFdTTF4r7AL64L2YjfS4xY5JRS4THE54DPamWrSSGUKDX44CgIBXfGKICJeBD455fwdOslRPsJnO5RkpfCY7U0qihrcXGWm4osVyBPkfJR8kswskB25AR+ienyVer1WyJ53ZyO4kKttssrLkucS+pBRx5dWyjtd7AwjdHa4X8krpfKM4qZppfTAMaobKux8sfdZgp+9VfmF46e4V3EyfKKrzlacLUlJb9g5fQTJHr3rOP\\\\\\\/gjFVm2SF5aBaRFqzMIbouLtczyR5gD5Zfokwy8a4NhVJnu\\\\\\\/cuGOykgMqLUECXDHnG7TtdzGu4eJGy\\\\\\\/qBA6xqDe\\\\\\\/Qv9uTB+3DgfqzV32dqdCUNWdzBcsb72Sntjk7TlqRh8l3jMaJCC0MvoVJACQZxmSqLEPXAZ5TGwAFxJNH5RD2c9Vz\\\\\\\/6b8hj5iu2yjdZU77BO3QpDEnXU85ENJlMaelxKugpPIqCNNsaj3is69NapR+xeXTTROK68xymWVjkbHukD9jf8wdmYZKf9welVhhxCVGjOEfCDXQYHXYY99xKnFFIZfZZ7BF1Ahi9KwKEcr90tMgvt9NAiZYU7FaPUa9Bk7tTRAA+RGXMyMbksWnA8yITq7DB\\\\\\\/XCzosL+33OlUgdk7WHWMGmDgrIi05W0nhwdOBT1tBAl3DiHI2g\\\\\\\/BuOKh4POAb\\\\\\\/O4TvTm8yrO8lNkAAInAphFwkESC4Mu59zZi\\\\\\\/Kb\\\\\\\/g8r7NfCr\\\\\\\/2HZA5UcAebRHSFNO6Kj+U4BaYf4goOQUxy\\\\\\\/EEByVS7Ub\\\\\\\/C4vRnQVNM1I2z2OngDstoSsUyg2eXz2O57FwAJsxSljP8UtbzkgFU5vQNyTf2PHuHfoJrCUahV8EbKDLnFnqVjJeH7IqZzkACc5ZbLZcycV2Xw2wp6alg5AF0bJOTBCkzDa9ujLdyPtpWIRvcIAxNuhs0NcNpxldxuDrh5XHGTgdRm4ZoUoZ+aasWrBx0+ax2ac1Cy6IJdxrpKfQobsXZjFd7uHC2KyqP0HxdiDt5yFTVs7g1U\\\\\\\/YSjYOPWNgUx6S9Ry1dk0ZXnVB5SuQt9JNoQkdwdwVCAehE9UwoMFZF+TOL77bLZlOcVBBjhmz24i54c7LvnJEs3PtQf1XRQ03vEEcBWQm5PVWfTVwJbRsLVVNhoU2D5ZZP6QkzYZuNTwE3Bkdcg9Che9LBtlApmBqRfMqJDD1M4EfMZR6JfXEetByPm5nbVl3RaxX78rqmXkmTg8ZP7zA0GvnoA04pS9kgb5uoETu9pFwdE4Im\\\\\\\/gJ7xCNrylbV6sVBk7pBFu\\\\\\\/HYVyUda2C\\\\\\\/cH6aFPqYb\\\\\\\/52w5lNw0OR+43lyUOKh+RVJcQ7O7iVsjCT9SSXuLf2uFjnfzwcVNB2X2p3OjGpoASIAH8xSc2xLeDOCF2D2N\\\\\\\/azifbRhiiXKfUvE+xdS6b+QIrWONUyhku3ro5wPR2qGbh0yuo4Lg0M5n6f4yTC8uh0VOCQ3FasRrcw4vPUlJmkGZ\\\\\\\/SurZ7Oef5H0lQydcXPZjCWTX80aPfwC1zCALtngQoDQ+hiQT2Pb8tlTpboBxcoBmowBO\\\\\\\/j1nlOCLLxPkORxEjlgVpC5Vk7C66lQg+K0rnNg5jzwrIX3SZeMwVG6X28EJ+CJdZfrhpJhytHnhUSA6AdqgcvCMdqWHr2PSygG3o\\\\\\\/jS\\\\\\\/0f1i6\\\\\\\/gLPJL0kOSSZ4flwyZMdW8kIwLhjAl7XabOCDr+mZ7HFWZZCvlNWeyUyFYoDIYh\\\\\\\/2N15gRAYVxqennbcWWeyYpzBfaFkGVeXbsjJIcofs4QQt5H1okEOPLBkdprn+ntyjC1ZceuflaihlysUBKOCnK79W7YSfsLydJT01WIxbL4a2vjwlNLK5p4w5\\\\\\\/jilwX1Vgbq8OIbJQYIyxvfTjyn\\\\\\\/FLfCKjJum6Mck+B1\\\\\\\/diCAAwt3H93DJ+r\\\\\\\/0oLsZ4pFbJobhLuxu\\\\\\\/ZdFow1uyL8QeiT6pxo8bTk8kts4Q+0OA9WBEcmFWQoaxXG8hBicfoFCiU7SRgBpst9PjUdjOhYMcIm6iolEfE+YwwFhQJeY1vI\\\\\\\/L2doc65QsxHruAWjd1f8gU693pCZDxGxWyXdaJQWFBKZ1zRI4yahdKa7NIGUDNI+RzM4Yp8ZOOqqkpIB\\\\\\\/oqQd8VBP8MCFqTjjurPCQWHL77Bg5vpyooSfaeptsQkN6SuUuqNpKEEagrLLG2XNclwIiVBeNjYUYWEuNP2BAZs0tH7YnVe4CMHdG3j46hq0pGEpvmS6adlUfR2vEoWxoqF7ssEHZuuJxCUsrc9sIKNIUmLmiYluI0\\\\\\\/41RcfvjuD47ti1OPshF48ob8Kz2+KaRG33YbvRxfls7XnZUSCw5h3XuGe177KPUVvynY12ngw3X9W4LZeD1ZG7LDbGCXn1pFxx5MfoQwYRIWrw+QZZpe9MWoMhO+Cwwq3AkM1y0DnQHFGlWgPW9Z9NWst7T6sD6O70L7Ol0zEvZ5Oser3291X9SWgTzKWbDMDJgftq7zLl9UJkB\\\\\\\/nFVAnW1ZAIzgocED8xmuSwCJsnqGncOOx\\\\\\\/s37H0cxMeWOWzP17G+hE6UsMzc2sM4IoLqsn95PgpCKM4ELz1JD1DM888XFmyC7ph4QEVC6s4uo\\\\\\\/EEMMegJTUoB6offe2tIkqhtM2iieRjHGLAukYpahGkSVC8LwAqSPPbBR3Bv9hWngG9SXcOkw9kj2\\\\\\\/Km6ZgLLrNMgmiKEfZrXPK4NIAEe74peFdwrpAvhsXTDlZDFNdLsU4NX88OXUdnS0lnnplk\\\\\\\/3PANVwLAxiAJ+ieZMQKc2eus6sWyjoJWQoP+faGDU2CyK7mPc6OrCxuHtTcTeS7s8yc7UON7kQzgdTDdcbYtbl4rL1tqaCk+QLViU6HPmgoUYw7efEkjxJMhlGw9BOroDrksCaX7mTQfWuCsefttSd+bo\\\\\\\/cuFrnSlXOlYZv8SJWr5BsqYQ4oiN2CocQ4rJPOGZOMuLUcAc8IrOG06y50UkfVfe0Yz1vKoNRmSEGwf9ok+rjkim6oWCWubAc\\\\\\\/yFErwWief3UDN\\\\\\\/w7m4JABGA9OVirJTJP8rXBd7ScsGm+gcuWJ1gSOImQCAhi1FMLKA1+9XHWrjzyZIIHH7ivHjF\\\\\\\/dmAm\\\\\\\/U00eBxgdgwU25UMMvVKWE\\\\\\\/4S15JxQwvEtGlZhn6GuhvwEcXX78cjwDlsbVJJwnaFvJgSfmfJlRQrXrsz1QldIi5K5Y8i\\\\\\\/0lvJXPR7AKDsugCjKQ+0nOci88UIkdT2WQJzKlDFPSYEfB3s0Wp4A5Mo9Pe83p3Q72XqvSpp9GN9kGdNrJASUp\\\\\\\/YIV1MtCLfCLsHUDgN9\\\\\\\/fZImKkfwZmNdy+2L1VGum6hCZvIQ6GoBRw+kM7OVpZAMPwZ8H3WNW07BvLa9Uw3rfkTn\\\\\\\/SAdvd6RLyx2sSt8LOtarlXTHZKZACfOW0zSt0bjHdB1otgD3O+y38vvb6WzqAD3aS6npCqduLmpBIYYmoNROXJpZHgr\\\\\\\/8Vx1MYmgw+\\\\\\\/418MGcGhXSSOSyXvyJkRGW81SPSV8hO+KeyCmL7qsaQvwH5SzXUB7RItwShbKdfX62xekUd9Q+SzZkClUkUtmlior27p18tn14WhF6xHXPseCvMkd9F6\\\\\\\/eygeZAWk\\\\\\\/5CC7Jz6i1oB8E0AAtJg7dPr1iGZkGita9nSeGhSjiLfBBigHvKO+hoa+hTTu3LyzytU6amcUdLz1x8jhOxz\\\\\\\/9q1Fhv0FrZwMXT5G6iF7f8tfqGInfNSmn5wMCQOPpMc3rVlWB+r9\\\\\\\/\\\\\\\/StwLGxRRbOZ7ZAjhL9q7gnnOPIDKMYeI66pPgBQ4UOqPPtZI8vihvgPhtCPw\\\\\\\/LVuounUb5jEHPO9tZDF8i6IdXAt7O+Kna\\\\\\\/y4KkN0K8tZYO4BCE6DiLtgk\\\\\\\/OQjuquarO2TT+5NjWuabB7UTRjUTQS2mb94wdGIRoav6VoSupAIgsdJS3\\\\\\\/+27kDXjWyJT0qTUaY46GEtbFXnm0vKXB69s3vEBcbygh6cVRV0tjqpOn9OkjWxRXnvYEaXsAXRv6s6Rx+dnJqPkuRqm\\\\\\\/DhN5gyQrhznSv\\\\\\\/DQkDrZR9W3G\\\\\\\/HZ82W6fJr6PCqHyva6NkInGBfK9NlAOzFLnddfSBdBNde7qANqLwRI70f0J9+wYrVx0ZV\\\\\\\/3VDswywQ\\\\\\\/ZB9gFEtdybvVHpUAM3jkJIQogo8L8SPZVvZ3LNJBohEH\\\\\\\/wrhAP7S3OIRDNJVz1Y0GhwJ95YSt9e2dTAbqyyJ\\\\\\\/yVffTRX4YZUEcMOnHbB+T50xyFmoAYFySZNLfhUCGAvTyYq5brv5zaTYErHpmVxKcB90zCIJS++3ei0+fXure8iIg84fTTHdgclC\\\\\\\/qvVnxLy89iHy2w4tF3GWctPm\\\\\\\/B3cHpQmVcnVEX\\\\\\\/dwaYMfGzK7Z7uUkDyg7oaz9zbA5EIgDqIk0dHlN7iRtQOAVyuyFLLN1swrml8\\\\\\\/uFYD0xcS7SSkPcSDL9HOg5euf0ETqo3x7Br\\\\\\\/PuiYPG1rAwxfmgXBFqlKp0ML57zY28LOsD4sjo8NJ5zXaIbMLjKEiBnZhOCSfA1UiKpWH08oGqpCJRZSzewCzSmpJMnaVh3Hry7IX9reCAA6R6xk45XiZW\\\\\\\/6hexxrfE\\\\\\\/PnY7zqrpX6L5CmFb3ac0zgcyVfn9+Tc+JJ0YZYfv\\\\\\\/y8As1akJzSkU3yV3LbfQUP+2eVGIWVRn9vpnfY1go04jHSaherpLQOI8+s3J1X3CcqNAlXW+Bs21Jy59n7O2hBpd3YmUayknzjkKDGLVyibFDSIetLvn\\\\\\\/c+Eb4kHn4ToELJ\\\\\\\/G9l8nfeMdqpi\\\\\\\/gB9LcotoqHgGJKoOcyi8j9qKxzutmSA1DvQU0Ja1fCsalle5Ti\\\\\\\/uoPVG12kYyJGQZnXPATwQHU\\\\\\\/agALp+LQCTo5DSYP0Eb\\\\\\\/cC9KQxjXzI2mWcT3CWeQ51qCSg4MyS2i6R+lV++LeSlNSEhG008XOWkjYUyZzNj\\\\\\\/OgS2bNlbQBI6myQOtOOk2phcmxTc3Knsj1npPpcvCxKfXPeSErAFWYEWfIOFYe8Pj1TPFARESWlVOcx9LjDZikGJGLaypZN+Q12d56BGSTx0t5PvnYC2NQfi0Gw9cHWfXP+T8zTbusgiP2605USCD4PHL2MrRSI3F0LhuP8mzmLlFpGJRClcSmPyYdOve+NZC\\\\\\\/vRQNK+nipqSokUNNijibjs4HX6EIAfyAxxOo5rilR4dIqecKuMmWZvFix6s6sv9LvC3230E9nveV7VB2tTH+kMx6FGVdAYGp\\\\\\\/TyjHrjnZ8PM\\\\\\\/7xhXO+nvRw5ZLduzQHRLX1+w4Uchf3Ni6ULRodmgIie5JvhuBQqw7e439ZYVkqfRGLZ5CjF+NNw0EpP5cKWg0X2H0z8jdAEhuowooX4qstcOwAteeQFoDdoDNfwInBN3AyAatideC0x\\\\\\\/Mb90r33ag+kyeoub\\\\\\\/N9lzTuQveXMF\\\\\\\/u9NkHRSfgi2v5jcxIQ6S\\\\\\\/SRxhJYitzZExNTjcUjS4CbvUfSP+FPOY4ZinO4rMVFj4FfHSRZlG81gaXah1gO2yJDQV+kID5Ug6t\\\\\\\/JxGKWXby1kU6mfGAdUdRWrGSi+TYfe9OdTAILRpilC6cjfByS1u\\\\\\\/9eFuZnZ5aeUfhUoyIdK9JUsbWlWXJyUgnxVlnUB0k4ohPBSfdnke0vAEsXsAOUxzYA8k3bv9qoM0c4mtTAsjrjk89AQTTVvM1TV82yxVarSDj7BxURIqE4PxvUD1cnAKSLkGwuCD\\\\\\\/UvFPomBH88NdWQeS4eoMzBcaqKIji0aMFn0qTZ23qtmxlEfrVcMpBombNN8K8UAkxeoyA37RroU2u5XCy8\\\\\\\/3WySKhdT4aa8hcIsdW4mFCsHyZD3vZa4Sx73DEq0fMHhzUukpHNpz9DACcG6lZIb8n2IULHi8AvnhCzsKyrpDkLlhlNCN3wnvRFInfwqum3xjwY4uvuuZtG+oQxHVQElLwJpit5f4YGJV\\\\\\\/ALrkRRuqqd9tFBxjLvRQAHd7pwzMOb2N\\\\\\\/XDN1eQ5l32zR1fCRJT9ioUynk5L9XSMEjeaQlFXQzk5wb0EYNwn\\\\\\\/kq496LR77Gyclrjfuu0h0bx3Tg85GKUASrLb9jz+3mW77rKFAGsT2DQ95tN04mSpyZHu75xbGqrjcb3GC9Q5uUZT7vRk0k\\\\\\\/UjIt5NkEfiMqdza5oFjdyJsdvQg+SFIOA60o\\\\\\\/ZNdwjCNhjOPGpe8t9BkI1eOXsDcHU2V0KxrNDC5l2uqx+w9A8dZfkI5iBGxJtedtPICHOOPhd1njQcuUz0O7KPVCQ6i3Z+KM0IK\\\\\\\/uBHuVbP08TFzLBAvTgI8Z24lSAh+EjpnzV7I+NSPgF7Q7RE2U0UBNQuzL2dwN5UNEkRnfknq8N1j0ZWyHQzN5pE+YyaSc3q\\\\\\\/vw6oux7fIN0D7njDAUKz9KQQsVHYjWeAIrUAUW+foqmFbguxONTKgrwbKazjN57k3xdxv3E+eEklI\\\\\\\/loepKspWNkIQ7ohMwIkOt0qfyHZQdCBRjA0isqDgrKhc3w+1et8hDUViDcJLHbsq0p5dP6RDi4QrA3gyobW3Da+udJPa1HILzJO0IxnC2LSvH9+bVoqTyru0Evd1+z7eUZzalvHzx93qvehCkcWQ4YVGzR8XtNtoxXEkxn2dZJ0QRVUXHmX6OX+08ajI4lLrIilOa+J8wdvET\\\\\\\/wRb5Ti0uzXtBJ1OTtFyk1fWI62DqwaQGPBz3Y7A6\\\\\\\/cHsOApIGk13rcntQXCRa3vuyzNDltXUY0++r0Kogq\\\\\\\/yfhE6xd9RJ5cyDTk26CYdKgjj7\\\\\\\/At\\\\\\\/W87eZ94FDklqgQ+aybAhGPfDdgsK4SKTyJ9MWJICgmIrtFmzaKHTIe4G4MGjK7JTtxGsKyAQofQMYi9XiMveu6tJTlXzUjWvDyqn81oymBs41QH9\\\\\\\/9MIHwHnkuVZo7fvETyTuWf6xgdkLPAkvBJv7jVaR0BuzbTQC0dKXxBWyLwJdrZaedvyd4HWoALAZMXTHzAcYswihxxVOLRl5eVzHRqNDf340Eqw+tiau3\\\\\\\/O5m\\\\\\\/1rDfdxKiCUTvb8qlOi6nlFhfVSB1LzuPayOyDH5+44pbxSdX496c\\\\\\\/0nsOs7OIf4l5bweqBISDkrr4hNAJ3kKmhO2jAj7lgOBGnYsQAd2xapLn52ZkijTBsgHype9LLuYmzX1ykm9dlss2eqwmmFluDTRjqY6UngULVXDyee0BHfcV7LMCYPIROxw0VIEd6XWpqGgWDc6iaeni0yUeYW\\\\\\\/jj1fVgCjTmPVy6a7mJplKc7B3LCZ1uO8mEGktIr3QeQAHxWjX7bCQzEvfaQGmXzsm8IPNFWyZWmQNB3hfuRkt1gOWP5recDTYkMXYhkZHCIBKnaIh2aCqIx5+tv07TSiP2kw8i7LneHABVTV\\\\\\\/7FStUZ75D2olq1LDy9t7lQuShC8HvFvV8wvjh3vLUEMPDb+\\\\\\\/yfybHexQVDMOMlgNbiWPfxWvv\\\\\\\/ks\\\\\\\/y1\\\\\\\/SiyJApZR2W\\\\\\\/jk8+35gednmEt\\\\\\\/5hO1Mitucf98P3OH5A4lQaLKwUk7bhk7uU24zRUSdr5IRmBKn7n+ACbFFePqgyuwwFVGNvFj7xmrWM7FsZypX8owH+k5pkuRn0bEuYgDeCIjybmOw\\\\\\\/w8bSELz03G7WilRj12MI9M3Q+WgbTd4+HhHK8+cC+do44bVMpLJgYsBzOU0IHznOuHAxNtUkq9rJBfDy6+91RfwY82ET4a3+XU74Mrjis\\\\\\\/sdipZtlCxldwdPFSrtS+2hPEZnAU3W3vc2VcqhGWEQsWKTrJ7qkhKRDBlVMt4\\\\\\\/Z3G95XzAxwVZAKPnX1gHXHyjC2y\\\\\\\/7pKItuKOGV7+geLnXJhzpP+wQKTVlbxf+PuTq6fLRIUQeVxmReYskg017ICvuILbU1F2DUUedqC7HO8aZRyDZNmoZz4ISuTKMRVbsHc1gbOXM9J4g6wiIx8IAx0V+AW76XSEe0LqMe4qcTaa2Vqsytm4M7yUXNzf05puiovgeMwE1P7Bkz8Lg9n+6UBz1rPEOcK5bfGciA\\\\\\\/4xdyKTTqG6L4Br0k9zubFdN1CXgvZIJ2J9rlyhVWRh4nW4+2JH0O7fplREiZlNnJI9Ffex7hNmrZ7hz6eU8MXqxjgSFlJJaepRPy5dtuL\\\\\\\/KZeGqm3HWa4rN5iQeGW0yvQa+1sgSoZacY0NTnP2Oeq+aFPuXmIiv39+iLjBL5hlI1wDB3AVHERK4Qdlcgf1UOZ+MMDkKcoRKYOYePHfReaxr5V7oFtTS3AXmk1W7GK1OCGeoc2sWXAB0ejBQeu43kTlQs6gw2ucCpob8mbGvYfakb0nVLQBeDCwVMTIUUQ9UtxiC9RRu\\\\\\\/YLcOwg2C\\\\\\\/BNmWAPFnoF9Ui\\\\\\\/MqMaPX42uWhOoEYRBpXpe+iJrqHxTXzK7rm2mtK7SGEAUlRzvXAO2E8l1NIKTFHnOzRq6TMKJ31OtedLi8sx8KdQsWf77G33PxCBCpSUGdnLH\\\\\\\/pfLwfGWEIiS0npehsxWG11Dnsqcaiv21smsorkIkwurKWDXcGtNZ81Q4jH3Wkf3S9uSrytAe5ydOThVkCFlbVRXalW7dYjblXR7FYyag19xFhxgHxe7rq9MmbUWny2x+jinyaqRpmRSh71rEExt\\\\\\\/5MSAv\\\\\\\/wJI8ApobdKp6SaJv90Bu\\\\\\\/h7fBReNntpBVWkRJK85IbHBGoZ6q2YqpDze+KwCHdZgkrOVLasjnfWykK7sCNHtb3aEQSVueuGHLr0gxMsG8DcdEIJV+jNg5wIkCCxdE9+opMooElWG\\\\\\\/yA5LPi97+fp4tFnG0+XcWAAaUjIDQnGqPVNO7GGNGKjpQM9Y3zS2hoo6jWEpiYS1K2Nox1AOD+PQNkLJ7bo5Uaw6YPVqlJGQovD2ZVopcGE7Li8HNnFWSsgERM9DU4lcc\\\\\\\/0X9frRghTELWoozqYvMy8jkqJqVw4W+La2kH\\\\\\\/b+QjY+iewHmI7+XUAif\\\\\\\/7PYTIVMvRAwHaAFy6EY5\\\\\\\/AUPJX5v5CqWpTXGJNcnlzgfElfMpU5OmvW7YMVBwjS0RgEE0SmqrfgmspiXOy2pIrmuomCUDc2tOYpxG1rWkSswGbyzEWTNMUhWyCTYhfesn289mroh3JVNaQz0xc6oZ1lwFuHsvTIiLmKHYp714nRYnFDQJxP8p0F6FXew6QT+dcJsgZJoZPEpWeIgRG7bMzdg2FBLXv9t7Kcr6TPoGNW1ZPG8Uty6gIRW2qHgJ6BVDTLJ1x3N1eEC4NIvtOGALA4Y\\\\\\\/4sNhDyT7Ok6DinB49vshPqv7RaXym+uGtV8DBMJng9QhLqjG4oyejfF7p\\\\\\\/Vx703IFSFiHz66\\\\\\\/4hm+rlOVUzSIPxg9UPukJ0RZPzzGkvhXhD2u35+ozkjB1BzEgW3rBWwBUExK+P\\\\\\\/\\\\\\\/eeTNTZ3jA22a8eofEvXCUz4FDHSHMIJwi28awWa6ievizLliwfCt953Vf02fI8hX49m9MiqFnPuHIe+F8R\\\\\\\/5v9MGatQzose0NMTJOMA8f1yrPR2PtiD0iKclb0Bpi4+ScxC3tghIeahx0fL6I\\\\\\\/Q1FyhyjfcnMKJPr8weqOMVyGpHtyuka5wMGKv\\\\\\\/j8wdM\\\\\\\/ABBp\\\\\\\/Ax50PM+UvSNCK4W3mhYHnWOiYFPN8KBUcMla3fY2McGyN9ABH3YUNrePWnL+SaoXyRxagt+SiG+JFChsa5JDdp9Dtk04zLhH7LLSLYsSP6WHcvyY+1TSCmsUB9C6YN5xSQGhVX3ZDJQnI6e+jTxNg46cv0NlUIXSn\\\\\\\/m0kvn0El2Jj5U1+AoUu3hTSl1ctFt9fLDtC8\\\\\\\/n35O9EF9Z3wD+fIVf13oWaS3A1u0P9voVkI7QSPTTpTd5LxAFr+9AopAWzxrekNLAIvxQXGEhVL+ZX1JxX+mDavXdUhS+4LWBi\\\\\\\/BbVcwEOWjE4u4jaCjmCS0vtT8\\\\\\\/BB1eVbXtjSzjfCkyj5h49SCY1Atly3ALEkMPO8973nJDEQ2Vpl+5y7Sw46cgpN7jgQwXzf8egE\\\\\\\/jYZvYhnWgT9XmhApJqOvnWmuAmucM4OOu4rcsIPxRPergfy1mpCYQUHEoSky0mAFde8rpK\\\\\\\/KnJI0ULr4fBHkWXhm6BS24UvKGWDPncOBzf6d+mrfHArRzq4s8ouKvNbad3tL1eXGxK2XP1QA8X6dLUfXByWGeYZwXaexA+nFzLblQERop3Yod79oQLwn1g1XNFCiE8pFNNb2QHnubUXju3roZzLOrqLiZ6lWRxGlvrw+w4k\\\\\\\/SG3IzuKuG4+yoKiA35dljdL1OURteiVHbxFngxCnnA\\\\\\\/fKzgdPKVBJ6Xr4zFkXpiJW938vz8\\\\\\\/b7PC2nQIsJyulSlRbFHvIfr+VNIiZQwWC4D4C5YZ5P7BoAA9ToKUqjfvSs1TazNgXCCkthRei\\\\\\\/w5uQJq0PqxYuI1WYK4bygc5M1Jgr3bRWdwcG9\\\\\\\/Lpi8UIIIOrMWetwMsObgrk0eo6tYijYrTEY3+cfBJH2za3yppDIpwx5INFtNcL2cddoy9yWEmPqPHi+35QHqIoEoMsSPFbTwuXZcX2t3FgCxDQnF5036AfHzLcEJ68fzw6\\\\\\\/WpvVlW60FmSc9KwTbDSjQZqemljLBLf6xoMM08XJKXdVFljkyR3tZEaF2+wG6aU+H\\\\\\\/IH1xRxxXODW\\\\\\\/eZT7ObELYFZvIKk1eig9sxT6B30iybk93Jb+\\\\\\\/wGmmZyH\\\\\\\/p87HpJPucdL3xBFEyB\\\\\\\/6XBLuRcUBj1MeDzKp5FWQUTyL4J2cJHlspAxqHpilceOwmhh2D2W3a9wlNvECKcGSkED9tCY+Hi0ZEw1PWhQWubzlA33PLN3namG1xD9qEGr\\\\\\\/pzFQRm5gp1XafzOd9D\\\\\\\/bMxbLel9n40gYoCjZthPDQC\\\\\\\/4kSot1KMopi9LFm2d+OdtMrv7JVyAltmH7LfaenVZ1MZW6pqlGqXbDCC3h\\\\\\\/Jo+AefNlEkSAxjvG5szmU6Muoy3hTlJ8nqR3zWY3R97pCu1tSTx2qnMy1LrvoF0dzUtagBHNP0hAB1Ev7MJevGHWbM1cWmRsGbVGktjNBor+M4BM6jIzyLWPklfZSTKfIB6VY7PlWdo3PojOeGSFmrBzqxT6QzJ4GWFNh\\\\\\\/nfSRfbpVZw+JoEQcOZuPVljKwnWeL4ci2yk14+1mVMc4wD87R4ZUvYuhE0mrhP0M3IjrUjertDaLoYfLp7jD0C1tzUOrPkFd5HH53Tgt4m1RKuAHkKpn1eLDqv+53yIPnuF2CRgYdXz\\\\\\\/dCNNnHfvmMqlX7ERpPy7iLernL5kXnoCv1U0xf3J7fWh2X4gPOVJaeW+VBkyihOEa61EROg5IIJCM2+1RuZR9n8nHjwEf7UfV3ygPSjMFa8Lhfd3Hq3EMcO7ntoSWLsWEtsrEomlUG0d0GzUgJpiB\\\\\\\/wHQ4uSh+NTxSKQ0sOI9s8O6kDTqFnGdDdLT5tNkMqJ+HlKjUlUdGfJFrvBBW2rrp35c4dT5VaWYu9hdQaSeswuCWuojQLOMbeHA2qWmQd780\\\\\\\/6\\\\\\\/taC093esT2CWR3AJEzPD3QWZ8cb+k6icT\\\\\\\/TDczL3AhO5EneikhkvoWH1BnZ1tkmJzyBX2GTvUoiKn2T2j2MTXCVqTU0e1PtEf98S3JLX3K9YZm6V75Lq5kT4VPQE9NATrc08osmghhuj9xKz1cnE6IzfV\\\\\\\/doRtQJYqO00iuoTk3k+dUlxWppXW14fi0X+cmlWEIASc15JfIYm16wzgsXX8abgSNmsDJhqgHUf8ZUAyTfEWJSgbeKOYkTKUhXKRUbj5JrBQ+w9N+QN6XVV5fTO76OKRBnm0cgMfqPZsB5S4pk7X1esuuG\\\\\\\/y7AUd7IQ7Afgc0WDeOqNZ1zKEvWaokRKW1xZua+PZMzGBbF\\\\\\\/OMJN19nH9HD\\\\\\\/I+r2MJhaCsRa8lK3qSMYFOUoRAtmGELkQxJts\\\\\\\/C7ffAruBRyFaFrGeZcSqa3gHQV4I3Hqgz+vBPZpfKSV6lxGKJFx27qb+e1OZvou8poLW7P566qRgrR+dc1tqaHyHa1Y8bcg1M7XZFRvftuRlxi9hzk5yq\\\\\\\/ql3a11mXtyvogiO8ahuIQiBoUbUvJytEPZXy7y94fPpe1sutTSmDYraLtraZekuvtV8PtYzxJ7ZGhKkixMidaCz7QkeoiLbaA\\\\\\\/DMC+6856MiRqHjIT2d+Ot6wfjPKJKFg6FC3OTZXssJ1bP3cgWDGG20UFvBb0i1uELR1andfO8DB1cnWmxNRLNHJqZIiSKV9eXSQfSwDqUa7vy6\\\\\\\/fa2l+6pmMrqws2HawTCogS5oWzjyznOSlDqvBhpTv5+b7MHjulab5ogtbWyjLF8BmNhWTOiZ0\\\\\\\/Pv39KMKXud\\\\\\\/JwQ+gUjmjDgxzFlylgRMuUXaHORGr13fBtbscKsToXYxybj4CIadSv3Fy3s26eXd5VHQkEdhut92jBqYaaZX5mCfpaAvk5UbzLbj63cfCIwhiqn+hNjue4fiykkND9NL2QvO4FkyExya7Jt7bkYohYz92uIWkIrlLzK2zLP1YWCHFED0mmhFOePX+c4w+UAbAW1lvWcejlhNz0qHi+p8M9swaFuhT66x1enltpuyriTTW\\\\\\\/Ro0ZlT8Jbg0ZIyplyLsL3jdY2vkInXQuyzB9Ntwg5jWDZVsMytFko6sgu+BwG7fIMMtZqe8Uof+vctMf6v2T6\\\\\\\/Ik6E8ZE1ScPXMCHiEO9gB7pgCDBxtBKcDXeNDLwToSzsMOu1Gq3yxBs2KA\\\\\\\/9WjSFw7cmWyV5PlS62h13VtGFz1D\\\\\\\/Pqv6oSbQxOR1nKn\\\\\\\/WK5OvkzpL8nZrdjRA\\\\\\\/QuJBNJ\\\\\\\/deL4+B2Vl2qF0awmacP7Grk8BWnicu\\\\\\\/T\\\\\\\/wYwaaEP8Tble+0jsopdQnxNZu5TWV2l9++0M5Hn6V0UKRAglVTOOxBhwXJ\\\\\\\/TxvnoqoVcbniGHovL3Pmza8s78cICBgh58q8dHJbLsEoU7AazFXItvV1wfUQJIDGqNRRZc0PmAuE59Y5FogYU0b+fee3h9wq8G9frMqqufLKAqCCGBzSLMYtVSzv4A7VjduUD9J37F\\\\\\\/g0jo+Y8Tg8a6oPMAPmQoqB\\\\\\\/VYdZuFrEmigOH1JzewSkMcS04S0dSkfF79exbTiEF877DcxaqpUac5BT4Boss8v9kzfKf\\\\\\\/DJiUL+cRmTf\\\\\\\/M15G4yv\\\\\\\/8UqEellLwjgFD4963loVuLzjlQSXjaHRW+hAj0HMHLddte8QwkkDZCBxUeVXgO53NJMCeSZttlERXJqrQt1Uyo9o5p6KDnb8ajptSwIviKS5YEkKNhTeg\\\\\\\/U1T7usblOdcL5FeHx2xVg4HPcZZx9WqP1SdY7KI7LLAGMfLczvSyBHGJljfvJq3MLUirytBNYIKD7EdyKORT3So3VjZnktpDxHcQblFknsCMDxdrhX8g3kECj05iTO85y9ncak05xMv6EPcLEG7TrIQD8jChC+94hvFLIQmDUoPiWvoNxX6WNDxkjq+vrO\\\\\\\/+NyWdLD8Q0YyheCXT5m6NMQFiWdonlw2o8vDxmAoHDxRitiQ3VNuAyWzEbLEqaD5cWllTL+ta8X0HVN8UgEorKdxDsUZ3+XCFFDBiOT8Gx6VTU5X\\\\\\\/PZSjTiU7as1zyWLHI4hEpwGvexGwdW+EG\\\\\\\/z63zOU\\\\\\\/8j9xljXln1\\\\\\\/zaWvnxkJWxyBKrcNTfwvZs6fPmUxh5xGZyK1xuCg4Xi9mCapinkxlnqZTNw9hwAbcF+zAWQ1wm9LW6cWtzGkI4N0G4KwhYm9KsJ5At2UrIpvVVgiYNLSHyiDNxYkca3diR+T3lDxPaQ1S9FbexVqb\\\\\\\/FPZjQQWx4RoYEvUv2iVANzkMUe7TgHBs0npgJG11hORPpEJ7F4MD6jgzbAh6CuQ4LmGLHVAVCCyh60V+4WZ3WGhIDfKyuKX3LrO+\\\\\\\/0FNlNukKo9CGRBdYoZOZTE3rIbne4ZV4an6nBN9ZgAvL8KNV7V6CvfCCmyYC7QCanAiUoZdLCzPR50UcjZcb9ysR0Iwlb0b3\\\\\\\/nYFW1lk8Zp6ChQ6MZARQJ5EUnPhv\\\\\\\/NX0\\\\\\\/oFXvkh\\\\\\\/gUXvA\\\\\\\/07cn894m35gxXoVQAGZc4lPVhp\\\\\\\/fDgs7yk9IeSKMzakh5ERF+cvS9OSg+pwztbSIqaE+k14oLIuw6OLikvPrd0QgJDprgs7AadGFNYuEPYi5p7UGzMRZJn\\\\\\\/1jCh1pKEG8a+axnG+mzOF31stQX7hoAN1urjGOd1+QJM7UvJyGl2ZD6fSfoyQG+EJx4wdg7+sMCvX56UKkcqP160n7x5Q+8MYeWvFE3KNLJURBAcM\\\\\\\/mcIhR1hjWsMRyM5esnu1FG1WFi09pYndiYW06ENmXft3CD+pCopkxnXXR78HCBE+R61ivSbwLpeL1aQiGoEkBvoMDHzT71FYPKHFH7fyO5n7+Z\\\\\\\/5W7VNP+RsS+2vHxmRUbHghetIvmKBqJykcID9ALnigfm6S3QSwyDnbjTLLe6ugI4VgZbJiKIYx\\\\\\\/E7h+TYlfX3sFFHWvukJrdZBgrjXckryxi8pCA5IaB0Cw\\\\\\\/8PNvKz\\\\\\\/zp2KTSo3bH7q0juigo2F01AOfhd00prlTmy8F95xyVQEORiTc57ISrwQIdw4IiKcW43BNlWl1sh52HHuw\\\\\\\/UU\\\\\\\/n6wbXcb3PQQRpWeSg3\\\\\\\/av1Q1Vt+T78cZyHR0cs5pyudO3CcOFYxgUMGmCewwG\\\\\\\/r4AQ06bmrLsqyYXjOj71mm6Lhk+HLCz9bd9wp+69jrbFftFBbx5GXoUruX+ocyxM1DFJBarHL4twDE7ARqAJuk1qqQOBRPS0jO6OWXXVCnOBL+EJHQIqG\\\\\\\/wMGdMA1ezhBvrN3k6mM4\\\\\\\/JL\\\\\\\/HIQRUSZ7n7r7COAcgV1KVh\\\\\\\/NaJVPloYUPNNnpm5UB8TauM2QYt6fozomLZuqFaTuIwGf3rQ\\\\\\\/s8SArZFio1JBvAGFPYJFxgKFNK2KXD3YY6gs7b9lE9bFDXx831XGL3mMgJvla+Y9KrdJrq3ilA0GSKELOfKsNe3nnsZxvTdIsYzaY70q8P+hrBChJXodpJciYToyHdsMLgoOlwWqrpTEAFF1eKJ2GYuivJHsj0Lk2fz2YeiTmw7RpAG1GHHqmTcV5ldf+HzkcDcOfVUIeb\\\\\\\/Nw64hI7SdS1Vkq6C7ttcpfcF3nj5Cju+582xcXLuEWp+d1te8OvElxqW87d3+hHNfMQc0BT6r6R9RUe2zFb7h5CvRFqw0ilKxjiENamoah0iwL1EXbYHYKjF3aJM86+hCuX7NkBb8yOenQClk5dDLrQbFKz6eFp\\\\\\\/JYJqwOV1gFikvrIeRfIm+ewuzi1O0OHZB9ZWGhk\\\\\\\/mfD7ffqlNU1rPJ+hDwWUEn7fqbeDVMS+vIR3eHjI7INpgkZenELIg8L3kabw4Ei4vywPn0KUyTjuxNeLH12JONbfhGt9h47Wn\\\\\\\/I9h+dFUROnb1tP7GiXHFv4tS+MuuMgNY\\\\\\\/dcERhdoAENXDNZnhBAJP+M3ol+gDdaULWOV5isGsOlDTqKqIUXZWh0fvDlvS0ATpm7H4keitBZxuO32iFx7pN1a\\\\\\\/1EJVq\\\\\\\/+gmr+VHLvhMekhveRox\\\\\\\/n4asRdm4OQI647taETSooZU2BjUib\\\\\\\/72yWX2apYnxp5vKOJLRrU+WPmoOeMO8rN+lrTeafzl\\\\\\\/6hjif3k3bw3l1VMt6jOfCgo+dnDnjm\\\\\\\/r3n5RZ6ZjpWGOqF0y1dJdj4Q\\\u0022,\\\u0022iv\\\u0022:\\\u0022QVPVTNeYC5Plf5uEYoLOzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b8cf2d3d4fd9ddd7bb5c98a32a49c2b235b6908f4c13f3f2587ccbb056c8bf18\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jL2VpBegQmHVZx4Ol0zWgNul\\\\\\\/6Z+keRl3yc\\\\\\\/JzXLp90GSEBI5qdDlQVqNQNXXQ7Fna0Xpvo\\\\\\\/mDX8U798GUFrFbLAthuTyRB2ZUFVzKDshPw+YHVkeeoMPdYC9TvN67kq+x3JQE22id6vQG9fk\\\\\\\/7JBM18k1Bik7kzvr\\\\\\\/UiHfrycoqV\\\\\\\/BSrCZViHt2HYOYKW4xzp8PpZokp\\\\\\\/R+LnsNbNzLqK7Z1AIZpni5nKuf0N44NEE4BqbeanDNgvT4n2o+3rZQLyfYw0JKnJMWPoLXf6Nj0nCcl7TqhFUiUPWGLpO\\\\\\\/XeWUzvLMEBQOW5LCuIqplCnuB9DOzs\\\\\\\/mhZuM+So\\\\\\\/7m03VKtuwwhivCB3bsBttT3wIq5yWrQIILRz5wuYGcYBOPnvY1uBLoq4y4aT59nRnA8o+5N5aTTOZ0Ouk2Up+Kd84SPwD9DaXxlX7TwjaW9cLEA9rJA10MG8tjiouIQS9uQZeSFENuHXzUgpRd2Z11KFMetR0z4OYFZ1d7i5wvNjg55MJaNJOg0r49rdXuOtm32hezWhu5RZ0sra2TITq\\\\\\\/CZBteH13GgWkUgz5bNaTPJ0itts0g9YKBKiatpjdJ6ZLU8FaJJ3YbI+8\\\\\\\/XomMxd8DC1LUcvH9q+EqpMNFa5mCXGhd2hZ\\\\\\\/Np9rxpIpm9FXsz2Z4YE4VePcqT\\\\\\\/75KDSCj8eZSuMjun2xokuXkqc\\\\\\\/ujdoyM7YUb1KI4D30d5VPHohzTjQnvSMPk8Oaago6qwY0meMW9kASgOh1920X8oPRGVthhzf9XVmRz655D1g67abNIqZPcsJNDMCgGa54dLdlXZFqWJBuGmUghpqz9Jp1dTFjpa4LlPYmqGHSGmKxMmUYoDjcTYwFiM5xwEP\\\\\\\/yBzgNucLukOaDkcOEQOwXm\\\\\\\/FMBBXw+TIiIA62\\\\\\\/j0R07BNWDnD0CoQvpqcQxO9zqPs\\\\\\\/heItW8jrQTjnG5jLUYq0CWMfOijqiEPXCb4lqYV44YHM0foOouOBMLR7Ax5i\\\\\\\/JwuDaXpkzCWJY0EgLTyFjhmmV4xLdtBLsu5PxOKZIWIUQqNoXDQ+nUWl74H6TpjB9\\\\\\\/GSNlBsHs3kRkAhmcruZL3IqPnr5CtVxtxX84rp\\\\\\\/RtaU0h+8nKss\\\\\\\/X8K0InV29Z1DWRpoNKmxqwr7Ugr7oIjTPifPa0X6e9Kd8JmJ7rMlw+yfDabQV9MdcqpfpcEtqwlowRJIXmPbkkyKFQyGhQo6cECA1wLdcDI0xRgSR\\\\\\\/DCoElaitDJldxCbaBByTWk+8kijeeH+UNhJCcNyay2mPfEZ7D3AONVesOmhT88KusnKmDBxcbZ9NZuhmsjWHUq6W3zfJlD6etVf4TdrwNyjIFmt0vIWlgQ9AhTBtAmakmksVmtCp97hM90ynCwinFLU6ZZjnI5IVya6EGT8GDDTM5m3A2ccRaXrVO35OLFhsb95dZLl0cKA\\\\\\\/6ffh3MM7N\\\\\\\/b8Knc3anf7ybFszk\\\\\\\/bRam\\\\\\\/vz6i65rLb3+aWDrfcLelVtDTEIqSc4aAW9h21lvlAWDZocnH829DX8VDxvXejIy2Veelcuv536MAtWM1OK0Nb986THGlj03Tupl42DmdTruGw4KogYCi\\\\\\\/McM5Lrj+tty0Kn4UOvSx9DMjSEi0h8\\\\\\\/elIzm7lXrW0vywql+f8wvV+alQto8H071iwUxw9g\\\\\\\/TKYSy4eC3nsblxQXS6UxegEwzC\\\\\\\/mQAK5aFW\\\\\\\/tO4XV\\\\\\\/d5mdS8hAeepEX16pBderQ1BcKyrqdHIGkMHIGPQIktL+TqtNpTML7rHUzHVWfXleIXjM97W549iOS8rhMlzvMqCY6vaN0KaffGphYrBD5eU6dKtgKwI\\\\\\\/MPotL5GfKbA\\\\\\\/iHDsNo6XNy043G5mlN6YyJmIQIONv0JNB6Gr2kFufsLO+HiTCWvf8mXZmS2t+2S\\\\\\\/pc5aErnA1E+Dg3ehlb13abticK3TLj2ZY+dEw2quoDYeT5+BnBoGjwbDHYD456sJq5fWSkUX06EOLu1NaJAeXIXhHT109uPirhP9wkQ6qLsFRYfvibLF6bdIqgK7viP4b2ePbhQfG+CzzF0E=\\\u0022,\\\u0022iv\\\u0022:\\\u00227g2ld+VgyJq8MXVQLKvjww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd71f32012091e374b6bc03379ab0b42969aca0fd4133194c4814fb8d084145b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492438 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/675" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K9BpYyVl9KGYYNR94uTLA13a3w9a5YY23h8lUnD5cgw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/675" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K1ZlNNP4H8hJ_EzaRfjDscYCADBqNzAzAzHE3Uj5coQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/663" title="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/685" title="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492438" id="node492438" data-pisa="ggg;9/3/675" data-pisa-master="ggg;g3.118.200646" data-apath="/ggg/9/3/675.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">675-684;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200646 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492438" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/675" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492438" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492438/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/675.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492438" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492438/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/675.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492438" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492438/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/675.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492438" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492438/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/675.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492438" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492438/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Epistasis is an important contributor to genetic variance. In inbred populations, pairwise epistasis is present as additive by additive interactions. Testing for epistasis presents a multiple testing problem as the pairwise search space for modest numbers of markers is large. Single markers do not necessarily track functional units of interacting chromatin as well as haplotype based methods do. To harness the power of multiple markers while minimizing the number of tests conducted, we present a low resolution test for epistatic interactions across whole chromosome arms. Epistasis covariance matrices were constructed from the additive covariances of individual chromosome arms. These covariances were subsequently used to estimate an epistatic variance parameter while correcting for background additive and epistatic effects. We find significant epistasis for 2% of the interactions tested for four agronomic traits in a winter wheat breeding population. Interactions across homeologous chromosome arms were identified, but were less abundant than other chromosome arm pair interactions. The homeologous chromosome arm pair 4BL/4DL showed a strong negative relationship between additive and interaction effects that may be indicative of functional redundancy. Several chromosome arms appeared to act as hubs in an interaction network, suggesting that they may contain important regulatory factors. The differential patterns of epistasis across different traits demonstrate that detection of epistatic interactions is robust when correcting for background additive and epistatic effects in the population. The low resolution epistasis mapping method presented here identifies important epistatic interactions with a limited number of statistical tests at the cost of low precision.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/allopolyploidy" class="hw-term hw-article-keyword hw-article-keyword-allopolyploidy" rel="nofollow">allopolyploidy</a></li><li class="kwd"><a href="/keyword/multiple-test-correction" class="hw-term hw-article-keyword hw-article-keyword-multiple-test-correction" rel="nofollow">multiple test correction</a></li><li class="kwd"><a href="/keyword/heritability-mapping" class="hw-term hw-article-keyword hw-article-keyword-heritability-mapping" rel="nofollow">heritability mapping</a></li></ul><p id="p-3">Epistasis is the interaction of alleles, or variants, at two or more loci. Early observations of epistasis by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bateson (1909)</a> were mostly qualitative, noting that certain loci could mask the effects at other loci. Quantitative epistasis was first suggested and defined by <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fisher (1919)</a> who coined the term ‘epistasy’. Statistically, epistasis is the deviation from an additive expectation of two or more loci, often described as a change in the slope of one locus conditional on the genotype at another locus (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fisher 1919</a>). Variance due to quantitative epistasis has been shown to be an important contributor to the genetic variance in populations of model organisms such as <em>Arabidopsis</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Malmberg <em>et al.</em> 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kusterer <em>et al.</em> 2007</a>), as well as crop species such as maize (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stuber and Moll 1971</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Melchinger <em>et al.</em> 1986</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lamkey <em>et al.</em> 1995</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Wolf and Hallauer 1997</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lukens and Doebley 1999</a>) and rice (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yu <em>et al.</em> 1997</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Shen <em>et al.</em> 2014</a>). Significant epistasis has also been reported in allopolyploid crops like cotton (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lee <em>et al.</em> 1968</a>) and wheat (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> 2017</a>). Epistasis across subgenomes may be indicative of interactions between homeologous loci, analogous to dominance in diploids, and a possible contributor to that adaptation of these crops to a wide landscape (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wendel 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen 2010</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2013</a>). However, there is still little direct evidence that epistasis between homeologous loci is a large contributor to the total genetic variance in allopolyploids (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> 2018a</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">b</a>).</p><p id="p-4">Epistasis has also been shown to be an important contributor to evolution (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Doebley <em>et al.</em> 1995</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Lukens and Doebley 1999</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Phillips 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hansen 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Doust <em>et al.</em> 2014</a>). There has been considerable effort over the past several decades to incorporate these non-additive genetic factors into the genotype to phenotype map. More recently these effects have been incorporated into whole genome prediction models (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Vitezica <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Martini <em>et al.</em> 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jiang and Reif 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir and Jannink 2015</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wolfe <em>et al.</em> 2016</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akdemir <em>et al.</em> 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> 2017</a>).</p><p id="p-5">In practice, detecting epistatic interactions is difficult. The pairwise search space is large even for modest numbers of markers. For example, a population genotyped with 100 markers would require 4,950 tests for pairwise epistasis. With advances in genotyping technologies, the number of DNA markers available is typically much larger, in the tens to hundreds of thousands, and more recently in the millions. In this study, 11,604 markers were available, which would result in approximately 67 million tests for pairwise epistasis. A 0.05 genome-wide Bonferroni significance threshold for all pairwise epistasis tests in this study would then be <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-1.gif"/></noscript></span></span></p><p id="p-6">Several methods have been proposed to reduce the multiple testing problem. Epistasis is partitioned in part to the additive variance, particularly when allele frequencies differ from 0.5 at either locus (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hill <em>et al.</em> 2008</a>). Therefore, genome-wide scans can be used to first identify variants with a significant additive effect, then test only all pairwise variants identified in the scan (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carlson <em>et al.</em> 2004</a>). This can greatly reduce the number of epistatic tests performed, while increasing the likelihood that epistasis will be identified. Other methods include relaxing the multiple test correction threshold (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamini and Hochberg 1995</a>), or reducing the marker pairs tested based on some criteria such as biological function (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ritchie 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cowman and Koyutürk 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Crawford <em>et al.</em> 2017</a>).</p><p id="p-7">The multiple test correction problem is not the only challenge to identifying epistatic interactions. Allele frequency, linkage disequilibrium and the number of alleles at a given locus can all reduce the efficacy of pairwise marker epistasis detection. Low allele frequencies at either locus reduce the epistatic effect, partitioning it to the additive instead (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hill <em>et al.</em> 2008</a>). Less than perfect linkage disequilibrium between the markers and causal mutations also reduces the apparent effect size, limiting detection much as it does for additive effects (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Carlson <em>et al.</em> 2004</a>). Single nucleotide polymorphism (SNP) markers are typically considered bi-allelic, despite the potential for numerous alleles at a single locus in the population. The impact of these factors can be reduced by using multiple linked markers to determine haplotypes. Haplotypes have been shown to be powerful in the detection of additive and interaction effects by accurately tracking larger segments of DNA in high or perfect linkage disequilibrium (LD), and allowing multiple alleles at every locus (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lin and Zeng 2006</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jiang <em>et al.</em> 2018</a>). While allele frequencies are typically reduced using haplotypes (<em>i.e.</em>, the frequency of two alleles will be higher than the frequency of three alleles), the added power from accurately tracking relevant LD blocks make these methods attractive.</p><p id="p-8">Haplotypes do not need to be assigned directly to gain an advantage from using multiple markers to identify regions associated with complex traits. Regional heritability mapping (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nagamine <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Riggio and Pong-Wong 2014</a>) has been used to identify additive effects of rare and common variants in humans (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nagamine <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Shirali <em>et al.</em> 2016</a>) as well as plants species like eucalyptus (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Resende <em>et al.</em> 2017</a>) and cassava (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Okeke <em>et al.</em> 2018</a>). These methods employ the estimation of additive covariance between individuals based on markers in a given region of chromatin, and are used in a mixed model to estimate the genetic variance attributable to the region. Variance components can then be tested to determine if they are greater than zero using a likelihood ratio test.</p><p id="p-9">We propose a method to greatly reduce the number of statistical tests while taking advantage of multiple markers to determine importance of epistatic interactions across chromosome arms of an allohexaploid wheat population. This method is similar to the “divide and conquer” method of <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Akdemir and Jannink (2015)</a>, but models interactions across chromosomes instead of local epistasis. Epistatic covariances can be formed using the Hadamard product of component additive or dominance covariance matrices (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Henderson 1985</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Jiang and Reif 2015</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Martini <em>et al.</em> 2016</a>). Additive by additive epistatic interactions between disjoint sets of related (<em>i.e.</em>, linked) markers can be modeled by first calculating an additive covariance for each marker set, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-3.gif"/></noscript></span></span> , and using <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-4.gif"/></noscript></span></span> as the covariance estimate of the epistatic term between these sets. We define marker sets by the chromosome arm to which they belong, and estimate the epistatic variance component between the two arms using restricted maximum likelihood (REML) while correcting for background additive and epistatic effects.</p><p id="p-10">Common wheat is an important allohexaploid crop with three subgenomes, A, B and D, resulting from hybridization events approximately 500 thousand and 10 thousand years ago. Due to the allopolyploid nature of wheat, we were interested in identifying interactions across homeologous loci. Interactions at homeologous loci are analogous to dominance effects in diploid hybrids, and could be used to fix favorable homeoallelic interactions in inbred lines (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Wendel 2000</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams and Wendel 2005</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Birchler <em>et al.</em> 2010</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen 2010</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">2013</a>). Of the 21 chromosomes of wheat, chromosome arms pairs include <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-5.gif"/></noscript></span></span> homeologous pairs, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-6.gif"/></noscript></span></span> within subgenome pairs, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-7.gif"/></noscript></span></span> across subgenome arm pairs.</p><p id="p-11">Each chromosome arm of the wheat genome was sequenced independently using flow cytometry to assist in the assembly of the large complex genome (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">International Wheat Genome Sequencing Consortium 2014</a>). The lone exception was chromosome 3B, which was sequenced and assembled in its entirety before the other chromosomes of wheat (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Paux <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Choulet <em>et al.</em> 2014</a>). Therefore, assigning markers to a chromosome arm is feasible, but their position along that arm may not be well defined if the number of scaffolds is large, as was the case with the first wheat survey sequence (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">International Wheat Genome Sequencing Consortium 2014</a>). Using markers across an entire chromosome arm known to be homeologous to another chromosome arms may therefore be a better strategy than attempting to assign single homeologous marker pairs (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>). If interactions are detected across homeologous regions, this may provide evidence of beneficial homeoallelic interactions indicative of inter-genomic heterosis.</p><p id="p-12">We demonstrate the low resolution epistasis mapping methodology using the CNLM winter wheat dataset from <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> (2018a)</a>, and show that epistasis can be detected between homeologous and non-homeologous chromosome arms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Plant Materials</h3><p id="p-13">Details of the CNLM population used in this study can be found in <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> (2018a)</a>. Briefly, the dataset consists of 8,692 observations of 1,447 soft winter wheat breeding lines evaluated for four traits, grain yield (GY), plant height (PH), test weight (TW) and heading date (HD), in 26 environments across 10 years in an unbalanced design. The population was genotyped with 11,604 genotyping by sequencing (GBS) markers distributed throughout the genome, albeit with fewer markers on the D subgenome relative to the A and B subgenomes.</p></div><div id="sec-3" class="subsection"><h3>Chromosome centromere positions</h3><p id="p-14">Chromosome centromere positions were provided by the International Wheat Genome Sequencing Consortium (IWGSC) for all chromosomes except 3B (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">IWGSC 2018</a>, personal communication). Those positions were assigned by determining where chromosome arm library reads aligned to the final assembly. Each chromosome arm was sequenced independently using flow-cytometry to remove the chromosome arm from a series of aneuploid stocks, each containing an extra arm. The lone exception was chromosome 3B, which was sequenced in its entirety, so no centromere position was available for the 3B chromosome. Centromere start and stop addresses provided by IWGSC are shown in Supplemental Table S1.</p><p id="p-15">While restriction sites are expected to be uniformly distributed throughout the genome, methylation of cytosine is not. One of the restriction enzymes used to generate GBS libraries, <em>MspI</em>, is sensitive to DNA methylation, digesting unmethylated DNA at a much higher rate than methylated DNA (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McClelland <em>et al.</em> 1994</a>). Methylation is an important regulator of chromatin structure, where euchromatin tends to contain few methylation sites relative to heterochromatin (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Keshet <em>et al.</em> 1986</a>). Therefore restriction sites in heterochromatin with high levels of methylation, such as at the centromere, are less likely to be retained as GBS markers because digestion is less likely to happen at these sites. This means that the GBS markers can be used to roughly assign a centromere position using the density of GBS markers along the chromosome.</p><p id="p-16">To determine the centromere position of 3B, we employed kernel density estimation using the density() function of the ‘stats’ package in R to determine the smoothed density of GBS marker positions. We then assigned the 3B centromere interval to the chromosome positions flanking the second position for which the derivative of the density was zero. We performed this operation for all chromosomes to determine the efficacy of this method for determining the centromere position (Supplementary Figure S1). The positions provided by the IWGSC were used for all chromosomes other than our estimate for the 3B centromere position for all additional analyses.</p></div><div id="sec-4" class="subsection"><h3>Regional epistasis mapping</h3><p id="p-17">The low resolution epistasis mapping approach employed here uses markers from two defined regions, <em>i</em> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-8.gif"/></noscript></span></span> to calculate additive covariance between individuals (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">VanRaden 2008</a>, method I) based on those regions (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-10.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-11.gif"/></noscript></span></span>). The Hadamard product of these additive covariance matrices can be used to produce the pairwise additive by additive epistatic relationship, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-12.gif"/></noscript></span></span> between these two regions (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Henderson 1985</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Jiang and Reif 2015</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Martini <em>et al.</em> 2016</a>). In this study, we defined regions as the short (<em>S</em>) and long (<em>L</em>) arms of each chromosome, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-13.gif"/></noscript></span></span>. Variance components for each region and their respective interaction were estimated by fitting the following nested models<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where y is the phenotype vector, 1 is a vector of ones, <em>μ</em> is the population mean, X is the environment incidence matrix, β is the vector of fixed environmental effects, and Z is the genotype incidence matrix. Residuals were assumed to be normally distributed such that <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-17.gif"/></noscript></span></span> Chromosome arm additive effects were assumed to be <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-19.gif"/></noscript></span></span> while the chromosome arm interaction effect was assumed to be <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-20.gif"/></noscript></span></span> Background additive, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-21.gif"/></noscript></span></span> and epistatic, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-22.gif"/></noscript></span></span> effects were included to account for population structure. The covariances of the background effects were calculated as described in <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> (2018a</a>, equation 5), but with markers belonging to region <em>i</em> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-23.gif"/></noscript></span></span> omitted from the calculation.</p><p id="p-18">Sequential nested likelihood ratio tests were used to determine if the additive (model 2 <em>vs.</em> model 1) and interaction (model 3 <em>vs.</em> model 2) variance estimates of the chromosome arms were greater than zero. From the Neyman-Pearson lemma (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Neyman and Pearson 1933</a>), the likelihood ratio test statistic is defined as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-24.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-25.gif"/></noscript></span></span> and is uniformly most powerful (UMP).</p><p id="p-19">Best linear unbiased predictors (BLUPs) of each region were subsequently used to look for patterns between additive and interaction effects for the chromosome arm pair. The pairwise product of the additive chromosome arm BLUPs was then compared to the chromosome arm interaction BLUPs, in a manner analogous to the Additive × Additive single locus model (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>). Negative associations should indicate a less-than-additive model, whereas positive relationships would demonstrate a greater than additive epistatic effect.</p><p id="p-20">For the 14 three-way homeologous arm sets, a three-way interaction was included and tested against a model with only the three two-way interaction terms. We did not attempt to run all three-way chromosome arm combinations, as this would have been computationally infeasible, with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-26.gif"/></noscript></span></span> combinations. The Hadamard product of the three additive covariance matrices was used to produce the three-way additive by additive by additive epistatic relationship, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-27.gif"/></noscript></span></span> The following two models were fit to test the three-way interaction.<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>A likelihood ratio test was then used to determine if adding the three-way interaction term significantly improved the model fit beyond the two-way interaction terms.</p><p id="p-21">In summary, this method estimates a random additive effect for each region and a random interaction effect between these regions for all individuals while correcting for the background genetic effects of the remainder of the genome. The likelihood ratio test is used to determine if the variability of the interaction effects is greater than zero. If the interaction effects do significantly deviate from 0, then we reject the null hypothesis that there are no interacting loci between the two regions.</p></div><div id="sec-5" class="subsection"><h3>Software</h3><p id="p-22">Variance component estimation was accomplished using restricted maximum likelihood (REML) implemented in ‘ASReml-R’ (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gilmour 1997</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Butler 2009</a>). Other computation, analyses and figures were made using base R (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">R Core Team 2015</a>) implemented in the Microsoft Open R environment 3.3.2 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Microsoft 2017</a>) unless noted otherwise. The ‘circlize’ R package (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gu <em>et al.</em> 2014</a>) was used to make <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2</a> and S4. LaTeX tables were generated using the R package ‘xtable’ (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dahl 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Data Availability</h3><p id="p-23">All data used for this study can be found in <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> (2018a</a>). Additionally, we provide an example script that uses a custom R package, ‘lre’, to fit the models for the 4B and 4D chromosome pair (Supplementary File S1.tar.gz). The package relies on the freely available R package ‘EMMREML’ for solving multi-kernel mixed models (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Akdemir and Okeke 2015</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7311797">https://doi.org/10.25387/g3.7311797</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Centromere positions</h3><p id="p-24">Most of the GBS marker density estimates of centromere locations agreed well with the positions provided by the IWGSC (Supplemental Figure S1). Chromosomes 1D and 4A were exceptions. We estimated the 3B centromere to be positioned between 344.4 Mbp and 345.0 Mbp (Supplemental Table S1).</p></div><div id="sec-9" class="subsection"><h3>Model fit and p-value distribution</h3><p id="p-25">Homeologous chromosome arm pair models each had five random genetic effects and therefore five covariance structures for the two-way interaction models. All models converged, but some variance parameter estimates were often close to the parameter boundary and were considered to be zero. Variance component estimates on the boundary did not occur for the background additive or epistatic effects, but often occurred for one or both of the additive chromosome arm effects or the interaction effect. This resulted in a relatively large number of additive and interaction variance component tests with a p-value of 1. As a result, p-value distributions were heavily skewed toward 0 and 1 (Supplementary Figures S2 and S3). Most chromosome arms had low additive effect p-values, whereas most interaction p-values were high, indicating that the majority of chromosome arm pairs do not have effect interactions large enough to detect.</p></div><div id="sec-10" class="subsection"><h3>Homeologous arm tests</h3><p id="p-26">The U-shaped distribution of the p-values suggested that when the true variance was very small or zero, the average information algorithm estimated the parameter on the boundary (<em>i.e.</em>, 0), and when it was positive, the p-value tended to be low. Larger sample sizes may be necessary to obtain uniform p-value distributions when the null hypothesis is true. We therefore considered all homeologous arm pairs with an interaction variance p-value less than 0.05 that also had positive additive variance component estimates to determine the relationship between additive chromosome arm effects and their interaction.</p><p id="p-27">Seventeen homeologous chromosome arm pairs had significant interaction effects for at least one of the four traits (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Supplemental Figure S4). Interactions involving homeologs 4 and 7 were overrepresented, with 14 of the 22 significant interactions identified between one of these two homeologs. Chromosome arm pair tests failed to detect the significant homeologous marker set interactions found on chromosome homeologs 1 and 5 for HD and homeolog 3 for PH (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>). The failure to detect these regions using the chromosome arm test suggests that the associations detected by <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> (2018b)</a> were spurious, or their signal is being washed out by the abundance of uninformative markers on those chromosome arms. The lack of a two-way arm PH interaction on chromosome arm 3S agrees with the homeologous marker set identified there, where only the three-way homeologous marker set interaction term was significant.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492518/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492518/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Table of significant homeologous chromosome arm interactions. The proportion of genetic variance attributed to each arm and their corresponding interaction are shown with statistical significance from a nested likelihood ratio test</span><div class="sb-div caption-clear"></div></div></div><p id="p-31">The test for three-way homeologous chromosome arm interactions only revealed three sets of homeologous arms that had a significant three-way interaction at p <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-100.gif"/></noscript></span></span> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The three-way 3S chromosome arm interaction for PH was found to have a positive three-way arm interaction variance parameter estimate with a p-value of p <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-101.gif"/></noscript></span></span> supporting the evidence from <a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> (2018b)</a> that found a significant 3-way interaction on 3S using homeologous markers. The 7L three-way arm interaction term was also found to have a low p-value for TW of p <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-102.gif"/></noscript></span></span> confirming another significant three-way homeologous marker interaction found by <a id="xref-ref-59-7" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> (2018b)</a>. None of these three-way tests passed a Bonferroni significance threshold.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492401/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492401/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Table of significant three-way homeologous chromosome arm interactions. The proportion of genetic variance attributed to each arm and their corresponding interaction are shown with statistical significance from a nested likelihood ratio test indicated by asterisks</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">Many interactions were detected on chromosome arms where no homeologous marker sets were identified with a significant interaction effect (<a id="xref-ref-59-8" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>). Notably, a strong interaction effect was identified on homeolog 6S for HD, and two regions for GY on 5S and 7L, where no significant homeologous interaction sets were identified. Neither of the interacting pairs for GY had a p-value lower than a homeologous arm Bonferroni correction of 0.05 / 42 = 0.0012.</p><p id="p-35">Relationships between chromosome arm additive and interaction effects were only considered for the ten chromosome arm pair trait combinations that had all chromosome arm additive and interaction effects with significant non-zero variance components. Of these ten, six had significant correlations between the additive product and the interaction with an absolute value <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-121.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Four of these showed positive relationships, while the other two showed negative relationships. By far the strongest relationship detected was between 4BL and 4DL for PH (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-122.gif"/></noscript></span></span> <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), indicating that individuals with high or low additive values for both arms tended to have genotypic values less than expected by additivity alone. Conversely, the same 4BL/4DL pair had a weak, yet positive relationship for TW (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-123.gif"/></noscript></span></span> Supplemental Figure S5). The 4BS/4DS pair, where the <em>Rht</em> genes are known to reside, had a weak, yet significant, positive correlation for PH (Supplemental Figure S6).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/675/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction effect of chromosome 4BL by 4DL plotted against the product of the additive effects for 4BL and 4DL for PH. &#x3C1; indicates the Pearson correlation coefficient." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385385277" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interaction effect of chromosome 4BL by 4DL plotted against the product of the additive effects for 4BL and 4DL for PH. &lt;em&gt;&#x3C1;&lt;/em&gt; indicates the Pearson correlation coefficient.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/675/F1.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/675/F1.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/675/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/675/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492420" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">Interaction effect of chromosome 4BL by 4DL plotted against the product of the additive effects for 4BL and 4DL for PH. <em>ρ</em> indicates the Pearson correlation coefficient.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>All pairwise arm tests</h3><p id="p-37">For all <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-124.gif"/></noscript></span></span> pairwise chromosome arm pairs, we only consider those tests that passed a Bonferroni threshold of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-125.gif"/></noscript></span></span> for the interaction term in this section. Seventy-nine chromosome arm interaction variance components were declared significantly greater than zero for at least one trait, representing about 2% of the number tested (Supplementary Tables S2 and S3). Of these, interactions for the PH trait were the most prevalent, representing 49 (62%), of the interactions detected. HD and TW accounted for the remaining 13 (16%) and 17 (22%) interactions, respectively. No chromosome arm interactions were detected for GY at the Bonferroni significance threshold. No interactions were detected for any of the traits involving chromosome arms 1AS, 1DL, 2AS, 2DL, 3DL, 4AS, 5AS, 5BL, 5DL, 6BL, 6DS, and 7BS at this threshold.</p><p id="p-38">There were several chromosome arms that appeared to be interacting with multiple loci (Supplemental Table S4). Of these, several clearly stand out (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Chromosome arms 1AL, 2AL, 2DS, 4BS, 4DS, 4DL, 6AS and 7AL were involved in five or more interacting pairs for PH, with 2DS, 4DS and 4DL involved in 10 or more significant pairs. The 4D chromosome in particular was involved in almost half (21) of the interacting arm pairs for PH. 7DL was involved in all but three of the interacting pairs detected for the TW trait. Arm interactions for HD did not cluster to one or a few arms in the same way as PH and TW, but 6AS and 7BL were each involved in five interacting pairs for this trait.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/675/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome arm interactions significant at a Bonferroni correction of  Blue and red bridges indicate interactions with a significant positive or negative correlation between the product of the additive effects and their interaction effect, respectively. Black bridges indicate significant interactions that did not have a significant correlation between additive products and the interaction effect." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385385277" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosome arm interactions significant at a Bonferroni correction of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-126&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Blue and red bridges indicate interactions with a significant positive or negative correlation between the product of the additive effects and their interaction effect, respectively. Black bridges indicate significant interactions that did not have a significant correlation between additive products and the interaction effect.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/675/F2.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/675/F2.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/675/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/675/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492377" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-39" class="first-child">Chromosome arm interactions significant at a Bonferroni correction of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/F2/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/F2/embed/mml-math-126.gif"/></noscript></span></span> Blue and red bridges indicate interactions with a significant positive or negative correlation between the product of the additive effects and their interaction effect, respectively. Black bridges indicate significant interactions that did not have a significant correlation between additive products and the interaction effect.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Most correlations between the additive products and the epistatic effect were low in magnitude (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-127.gif"/></noscript></span></span>), particularly for the TW and HD traits. Notable exceptions include the 4BL/4DL pair for PH, which had a highly negative correlation, as previously noted. Pairs with moderate magnitude tended to also include the 4DL chromosome arm, but other pairs with moderate correlations between the product of their additive and interaction effects included the 1AL/2AL, 1AL/7AL, and 3AS/6AS arm pairs.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Centromere positions</h3><p id="p-41">While our assigned position for the 3B centromere position is an estimate, most of the other chromosome estimates were close to the centromere position provided by the IWGSC (Supplemental Figure S1). The centromere position estimate reported here should be sufficient to assign most of the 3B markers to the correct chromosome arm for the subsequent analyses.</p></div><div id="sec-14" class="subsection"><h3>Model fit and p-value distribution</h3><p id="p-42">The distribution of p-values from the likelihood ratio test should be uniform if no true interactions exist. If interactions are important, then we would expect to see a skewed distribution with many small p-values. However, the p-values were often calculated to be 1 because the variance components were estimated on the parameter boundary (<em>i.e.</em>, zero), resulting in the U-shaped distribution. When variance parameters are estimated on the parameter boundary, the p-value becomes 1 simply due to the fact that the variance component is zero. This is likely due to a lack of sufficient population size to distinguish and resolve multiple small variance components. Perhaps another explanation may be provided by the use of the the average information algorithm to fit the mixed model, which may lose a small portion of information by avoiding the calculation of the second derivative of the likelihood function. While other algorithms exist for solving REML problems, the computational burden of resolving multiple variance components with dense covariance structures may be restrictive. Further investigation is necessary to determine how large a population need be to resolve multiple genetic variance parameters with magnitudes of 1% or less of the total variance.</p></div><div id="sec-15" class="subsection"><h3>Homeologous arm tests</h3><p id="p-43">Most of the homeologous chromosome arm interactions detected across all traits involved homeologs 4 and 7. The less-than-additive trend observed for the 4BL/4DL pair for PH may suggest a significant degree of gene functional redundancy between these two arms. Functional gene redundancy between homeologous alleles should result in a less-than-additive effect similar to partial dominance at alleles. Despite having a weak positive additive genetic trait correlation between PH and TW (0.3, <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> 2018a</a>), the 4BL/4DL pair had a weak, yet positive relationship for TW. This provides evidence that the observed pattern is not simply a genetic artifact and may indicate differential gene function for these two traits.</p><p id="p-44">A negative correlation for PH was not observed for the 4BS/4DS chromosome arm pair, as would be expected from previous results for the <em>Rht</em> genes that reside on those chromosome arms (<a id="xref-ref-59-9" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>). In that study, a significant less-than-additive effect of the wildtype <em>Rht</em> homeologs suggested that the two genes have partial functional redundancy. The mutant alleles are insensitive to the plant hormone Gibberellic acid (GA), and result in a semi-dwarf plant stature which greatly reduces lodging in high nitrogen environments. Plants with both mutant alleles are far shorter than expected based on the additive effect of the two mutant alleles. From the perspective of functionality, one functional homeolog is able to recover most of the gene pathway function, with relatively little gained from the addition of a second functional homeolog. Therefore, the loss of one gene does not result in complete loss of the gene pathway, but merely a non-additive reduction in its total activity.</p><p id="p-45">The lack of a negative correlation between the additive and interaction effects for the 4BS/4DS pair cast some doubt on the usefulness of these correlations to infer the direction of the epistatic effect. The relationship between the product of the additive effects and the interaction was thought to mirror the <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/675/embed/mml-math-128.gif"/></noscript></span></span> Additive × Additive epistatic model using a multi-locus approach (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-59-10" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> 2018b</a>), but it is unclear what is driving these trends. Further investigation into the relationships between regional additive and epistatic effects in warranted.</p><p id="p-46">For inbred allopolyploids, multi-subunit protein complexes can be comprised of genes from a single subgenome, or from multiple subgenomes. If functional copies of subunits exist on both genomes, the formation of subgenome hetero-complexes may occur. Protein complexes comprised of evolutionarily divergent subunits may have increased or, more likely, decreased functionality. If hetero-complexes display decreased functionality, then we would expect the relationship between the additive and epistatic effects to be negative.</p><p id="p-47">It is unlikely that all homeologous interactions are so large in effect that they are quickly fixed after the hybridization event. The distribution of epistatic effects is likely similar in shape to the distribution of additive effects. These distributions will change based on the complexity of the trait. If a trait is governed by relatively few loci, the relatively few epistatic interactions may have larger effects, and may be easier to detect. In contrast, a large number of small effect additive loci may also result in a large number of small effect interactions that are too small to detect in populations of moderate size.</p></div><div id="sec-16" class="subsection"><h3>All pairwise arm tests</h3><p id="p-48">PH appears to exhibit a higher degree of epistasis than either TW or HD. However, the number of interacting loci or chromosome arms detected was not directly related to the observed increase in genomic prediction accuracy by inclusion of epistatic predictors. <a id="xref-ref-58-7" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> (2018a)</a> found HD to have the largest percent increase in accuracy from the additive model by including all pairwise additive by additive interactions, yet had the fewest detectable interacting chromosome arms, other than GY.</p><p id="p-49">GY showed no evidence of important epistatic interactions in this study, as has been previously shown (<a id="xref-ref-58-8" class="xref-bibr" href="#ref-58">Santantonio <em>et al.</em> 2018a</a>,<a id="xref-ref-59-11" class="xref-bibr" href="#ref-59">b</a>). This may be due to one of two explanations. The first and most obvious is that grain yield is not subject to epistatic gene action. This would mean that all genes contribute additively to the collection and allocation of resources to vegetative tissue, and then reallocation to the ear during flowering and grain fill. The second and more likely explanation is that GY is the culmination of essentially all the genes working in concert to produce the final outcome, and interactions with such small effects may simply be too small to detect (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Xu and Jia 2007</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2012</a>). Differential response to environmental stress across years and locations may further reduce the ability to detect interactions if they are only important in certain environments.</p><p id="p-50">While we corrected for population structure on both the additive and epistatic levels (<em>i.e.</em>, using additive and additive by additive genetic covariance terms), it is possible that residual structure is causing the observed additive and epistasis relationships. The drastically different patterns in the arm pair test results for each trait suggests otherwise. If these interactions were due to population structure, we would expect to see similar patterns of significance across all traits. When we omitted the background epistatic effect, most of the 861 interactions were significant (results not shown). We deemed this to be due to chromosome arm epistatic relationship matrices modeling close relationships in the population regardless of which unit of chromatin was used to determine those relationships. However, it is possible that these interactions are far more prevalent than suggested here, and that correction for background epistatic effects is diluting true genetic signal.</p><p id="p-51">The prevalence of a few chromosome arms interacting with many other arms is of particular interest, due to the potential for one site to influence the expression of so many other sites. These sites appear to act as hubs in interaction networks, and have been shown to be prevalent in yeast (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Forsberg <em>et al.</em> 2017</a>). <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> (2017)</a> observed a large proportion of the epistatic interactions affecting GY involved chromosomes 4A and 7D in a large population of hybrid wheat. While we did not detect a large number of interactions involving 4A, 7D was particularly important for TW. However, the interactions that they detected appear to be on the short arm of chromosome 7D, instead of the long arm as we observed. It appears that the hub loci detected in this study are not the same as those of <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> (2017)</a>, although they used a different genome assembly than used in this report. The signal detected for hub arms may be due to the presence of functional and non-functional alleles at important upstream regulators, such as transcription factors. In this case, a non-functional transcription factor could cause the suppression of differential additive alleles.</p><p id="p-52">The detection of chromosome arm interactions not identified in the homeologous marker sets of <a id="xref-ref-59-12" class="xref-bibr" href="#ref-59">Santantonio <em>et al.</em> (2018b)</a> suggests that single marker sets may miss important interactions. It is unclear if these interactions would have been detected if they had tested all pairwise epistatic interactions between markers. While all possible tests can be conducted, this increases the multiple testing problem drastically and may result in the loss of ability to detect any interactions. It is unclear how large the effect sizes of a single pair of interacting loci would need to be to show up in a variance component estimated from multiple loci. While this method may not work well for a single large effect interaction, it may work well for many small effect interactions as might be expected for homeologous regions.</p><p id="p-53">It should be noted that epistatic relationships formed from the Hadamard product of covariance matrices have the property of shrinking distant relationships while emphasizing close ones. For example, two lines with an additive covariance of 0.1 will have an epistatic covariance of 0.01, whereas two lines with an additive covariance of 0.9 will have an epistatic covariance of 0.81. It may be that there are several levels of relatedness that must be considered to properly account for genetic relatedness. The pedigree is an example of a covariance estimation procedure that emphasizes close relationships and deemphasizes more distant ones. Considering both pedigree and marker based covariance matrices has been shown to be more predictive than using either alone (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">de los Campos <em>et al.</em> 2009b</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 2010</a>). Other methods, including Reproducing Kernel Hilbert Spaces (RKHS), can be used to model these various degrees of genetic relatedness (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">de los Campos <em>et al.</em> 2009a</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Crossa <em>et al.</em> 2010</a>), but may have less genetic interpretability than the method presented here.</p><p id="p-54">The utility of the low resolution approach will depend on the magnitude and direction of effects, as well as the number and distribution of causal interacting loci. This is further complicated by the relative LD between these loci, the markers tagging them, and the loci with which they interact. While single markers may not be in high LD with causal loci, the combination of some markers can form haplotypes that are in high LD with one or more causal loci. Haplotype combinations across regions may then flag true interacting loci that would be otherwise undetectable using single marker interactions. While our method does not explicitly define these haplotypes, it does capture these relationships through the additive genetic covariance of the region. We suspect that this method will be the most appropriate for traits with moderate genetic complexity, as demonstrated by the lack of interacting regions detected for perhaps the most complex trait, grain yield.</p></div><div id="sec-17" class="subsection"><h3>Conclusion</h3><p id="p-55">The interacting pairs presented here do not have the precision to make claims of interacting genes. Nor are these interactions necessarily targets for selection. They do, however, demonstrate that there appears to be global patterns of epistasis across the genome. Seemingly additive only traits have often been shown to be under a high degree of epistasis when careful investigation is used to elucidate the trait (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Forsberg <em>et al.</em> 2017</a>). Some have argued that much, if not most, of genetic variation is subject to epistasis (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carlborg and Haley 2004</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Forsberg <em>et al.</em> 2017</a>), where the rest of the genome must be functional to express additive differences in alleles.</p><p id="p-56">This is evident when we consider the the complexity of the cell, where no genes truly work independently of one another. In order to create the complex structure of the cell, proteins may interact with other proteins, both alike and dislike to them, to form multi-subunit complexes. Therefore allelic variation alone should be sufficient to produce epistatic variation. It is merely our inability to separate this variation from “additive” variation under classic parameterizations that leads many to conclude that epistasis is not important (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Hill <em>et al.</em> 2008</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Huang and Mackay 2016</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Forsberg <em>et al.</em> 2017</a>).</p><p id="p-57">Further research into this methodology might be used to identify meaningful haplotypes. Once interacting segments are identified, they can each be split into multiple pieces for further refinement of the method, while nominally increasing the number of tests performed. The low resolution epistasis mapping approach presented here emphasizes the power of using multiple genetic markers to test for interacting genomic regions, albeit at the cost of low precision.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">Funding of this research was provided by the USDA National Needs Fellowship for N. Santantonio, in partial fulfillment of the requirements for a Ph.D in Plant Breeding and Genetics at Cornell University. Additionally, the field trials comprising the phenotypic data for the CNLM population were funded in part by the Hatch Project # 149-447. Genotyping was funded by the Wheat Coordinated Agricultural Project (WheatCAP). The authors thank the International Wheat Genome Sequencing Consortium for prepublication access to IWGSC RefSeq v1.0 and Martin Mascher at the Leibniz-Institute of Plant Genetics and Crop Plant Research IPK (collaborating with IWGSC) for providing chromosome centromere positions.</p><p id="p-59"><em>Note added in proof:</em> See Santantonio <em>et al.</em> 2019 (pp. 285–298) in this issue and (pp. 1105-1122) in the Genetics March 2019 issue, for related works.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-60">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7311797">https://doi.org/10.25387/g3.7311797</a>.</p></li><li class="fn" id="fn-7"><p id="p-61"><em>Communicating editor: J. Wendel</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-08-06"><span class="received-label">Received </span>August 6, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-11-05"><span class="accepted-label">Accepted </span>November 5, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Santantonio <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.1" data-doi="10.1016/j.pbi.2005.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polyploidy and genome evolution in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2005.01.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DAdams%26rft.auinit1%253DK.%2BL.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D141%26rft.atitle%253DPolyploidy%2Band%2Bgenome%2Bevolution%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2005.01.001%26rft_id%253Dinfo%253Apmid%252F15752992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2005.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15752992&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227932600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.2" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.173658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.3" data-doi="10.1186/s12711-017-0348-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isidro-Sánchez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Locally epistatic models for genome-wide prediction and association by importance sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0348-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0348-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0348-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.675.4"><div class="cit-metadata unstructured">Akdemir, D. and U. G. Okeke, 2015 EMMREML: <em>Fitting mixed models with known covariance structures</em>. R package version 3.1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.675.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bateson</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1909</span> <span class="cit-source">Mendel’s principles of heredity</span>, <span class="cit-publ-name">Cambridge Univ. Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.6" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.7" data-doi="10.1105/tpc.110.076133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chudalayandi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaiman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veitia</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Heterosis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.076133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.076133%26rft_id%253Dinfo%253Apmid%252F20622146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMjEwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.675.8"><div class="cit-metadata unstructured">Butler, D., 2009 <em>asreml: asreml() fits the linear mixed model</em>. R package version 3.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.9" data-doi="10.1038/nrg1407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis: too often neglected in complex trait studies?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D618%26rft.epage%253D625%26rft.atitle%253DEpistasis%253A%2Btoo%2Boften%2Bneglected%2Bin%2Bcomplex%2Btrait%2Bstudies%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1407%26rft_id%253Dinfo%253Apmid%252F15266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223039200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.10" data-doi="10.1038/ng1761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Åhgren</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Epistasis and the release of genetic variation during long-term selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D418%26rft.epage%253D420%26rft.atitle%253DEpistasis%2Band%2Bthe%2Brelease%2Bof%2Bgenetic%2Bvariation%2Bduring%2Blong-term%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1761%26rft_id%253Dinfo%253Apmid%252F16532011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236340500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.11" data-doi="10.1038/nature02623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mapping complex disease loci in whole-genome association studies.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">429</span>: <span class="cit-fpage">446</span>–<span class="cit-lpage">452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02623</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D429%26rft.issue%253D6990%26rft.spage%253D446%26rft.epage%253D452%26rft.atitle%253DMapping%2Bcomplex%2Bdisease%2Bloci%2Bin%2Bwhole-genome%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02623%26rft_id%253Dinfo%253Apmid%252F15164069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15164069&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221644600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.12" data-doi="10.1016/j.tplants.2009.12.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular mechanisms of polyploidy and hybrid vigor.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2009.12.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DChen%26rft.auinit1%253DZ.%2BJ.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D57%26rft.epage%253D71%26rft.atitle%253DMolecular%2Bmechanisms%2Bof%2Bpolyploidy%2Band%2Bhybrid%2Bvigor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2009.12.003%26rft_id%253Dinfo%253Apmid%252F20080432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2009.12.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080432&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275214100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.13" data-doi="10.1038/nrg3503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic and epigenetic insights into the molecular bases of heterosis.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3503%26rft_id%253Dinfo%253Apmid%252F23752794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23752794&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.14" data-doi="10.1126/science.1249721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choulet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alberti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theil</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbe</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Structural and functional partitioning of bread wheat chromosome 3b.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1249721</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1249721</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChoulet%26rft.auinit1%253DF.%26rft.volume%253D345%26rft.issue%253D6194%26rft.spage%253D1249721%26rft.epage%253D1249721%26rft.atitle%253DStructural%2Band%2Bfunctional%2Bpartitioning%2Bof%2Bbread%2Bwheat%2Bchromosome%2B3B%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1249721%26rft_id%253Dinfo%253Apmid%252F25035497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDUvNjE5NC8xMjQ5NzIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.15" data-doi="10.1093/nar/gkx505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cowman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koyutürk</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Prioritizing tests of epistasis through hierarchical representation of genomic redundancies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">e131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx505</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De131%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.16" data-doi="10.1371/journal.pgen.1006869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Detecting epistasis with the marginal epistasis test in genetic mapping studies of quantitative traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006869</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006869</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006869%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.17" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.675.18"><div class="cit-metadata unstructured">Dahl, D. B., 2016 <em>xtable: Export Tables to LaTeX or HTML</em>. R package version 1.8–2.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.19" data-doi="10.2527/jas.2008-1259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Reproducing kernel hilbert spaces regression: a general framework for genetic evaluation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1883</span>–<span class="cit-lpage">1887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2008-1259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2008-1259%26rft_id%253Dinfo%253Apmid%252F19213705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2008-1259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19213705&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.20" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustus</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">teosinte branched1 and the origin of maize: evidence for epistasis and the evolution of dominance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D333%26rft.epage%253D346%26rft.atitle%253Dteosinte%2Bbranched1%2Band%2Bthe%2Borigin%2Bof%2Bmaize%253A%2Bevidence%2Bfor%2Bepistasis%2Band%2Bthe%2Bevolution%2Bof%2Bdominance.%26rft_id%253Dinfo%253Apmid%252F8536981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzEvMzMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.22" data-doi="10.1073/pnas.1308940110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doust</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lukens</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauro-Herrera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Beyond the single gene: How epistasis and gene-by-environment effects influence crop domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">6178</span>–<span class="cit-lpage">6183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1308940110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1308940110%26rft_id%253Dinfo%253Apmid%252F24753598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE3LzYxNzgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.23" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">Xv.–the correlation between relatives on the supposition of mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Earth and Environmental Science Transactions of the Royal Society of Edinburgh</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0080456800012163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2Band%2BEnvironmental%2BScience%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BEdinburgh%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.24" data-doi="10.1038/ng.3800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadhu</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accounting for genetic interactions improves modeling of individual quantitative trait phenotypes in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3800</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3800%26rft_id%253Dinfo%253Apmid%252F28250458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28250458&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Asreml for testing fixed effects and estimating multiple trait variance components.</span> <abbr class="cit-jnl-abbrev">Proceedings of the Association for the Advancement of Animal Breeding and Genetics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">386</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BAssociation%2Bfor%2Bthe%2BAdvancement%2Bof%2BAnimal%2BBreeding%2Band%2BGenetics%26rft.volume%253D12%26rft.spage%253D386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.26" data-doi="10.1093/bioinformatics/btu393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eils</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brors</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">circlize implements and enhances circular visualization in r.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2811</span>–<span class="cit-lpage">2812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu393%26rft_id%253Dinfo%253Apmid%252F24930139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24930139&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343082900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.27" data-doi="10.1111/evo.12214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Why epistasis is important for selection and adaptation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3501</span>–<span class="cit-lpage">3511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.12214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D3501%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12214%26rft_id%253Dinfo%253Apmid%252F24299403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24299403&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327572400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.28" data-doi="10.2527/jas1985.601111x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Best linear unbiased prediction of nonadditive genetic merits in noninbred populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1985.601111x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D60%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BNonadditive%2BGenetic%2BMerits%2Bin%2BNoninbred%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1985.601111x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1985.601111x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AAM9700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.29" data-doi="10.1371/journal.pgen.1000008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Data and theory point to mainly additive genetic variance for complex traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000008%26rft.epage%253De1000008%26rft.atitle%253DData%2Band%2Btheory%2Bpoint%2Bto%2Bmainly%2Badditive%2Bgenetic%2Bvariance%2Bfor%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000008%26rft_id%253Dinfo%253Apmid%252F18454194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18454194&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.30" data-doi="10.1371/journal.pgen.1006421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of quantitative traits cannot be inferred from variance component analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1006421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006421%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.31" data-doi="10.1073/pnas.1213423109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Epistasis dominates the genetic architecture of drosophila quantitative traits.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">15553</span>–<span class="cit-lpage">15559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1213423109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213423109%26rft_id%253Dinfo%253Apmid%252F22949659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5LzE1NTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.32" data-doi="10.1126/science.1251788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Wheat Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A chromosome-based draft sequence of the hexaploid bread wheat (triticum aestivum) genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1251788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1251788</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BInternational%2BWheat%2BGenome%2BSequencing%2BConsorti%26rft.auinit1%253D%2B%26rft.volume%253D345%26rft.issue%253D6194%26rft.spage%253D1251788%26rft.epage%253D1251788%26rft.atitle%253DA%2Bchromosome-based%2Bdraft%2Bsequence%2Bof%2Bthe%2Bhexaploid%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2529%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251788%26rft_id%253Dinfo%253Apmid%252F25035500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDUvNjE5NC8xMjUxNzg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">IWGSC</span></li></ol><cite>, I. W. G. S. C., <span class="cit-pub-date">2018</span> Shifting the limits in wheat research and breeding using a fully annotated reference genome by the international wheat genome sequencing consortium (iwgsc). Science 361: eaar7191.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.34" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.35" data-doi="10.1534/g3.117.300548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Haplotype-based genome-wide prediction models exploit local epistatic interactions among markers. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1687</span>–<span class="cit-lpage">1699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D8%26rft.spage%253D1687%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.117.300548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.36" data-doi="10.1038/ng.3974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1741</span>–<span class="cit-lpage">1746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.37" data-doi="10.1016/0092-8674(86)90263-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keshet</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieman-Hurwitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cedar</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Dna methylation affects the formation of active chromatin.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">543</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0092-8674(86)90263-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKeshet%26rft.auinit1%253DI.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D535%26rft.epage%253D543%26rft.atitle%253DDNA%2Bmethylation%2Baffects%2Bthe%2Bformation%2Bof%2Bactive%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990263-1%26rft_id%253Dinfo%253Apmid%252F3456276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90263-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3456276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986A397100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.38" data-doi="10.1534/genetics.106.069005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kusterer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muminovic</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barth</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Analysis of a triple testcross design with recombinant inbred lines reveals a significant role of epistasis in heterosis for biomass-related traits in arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">2009</span>–<span class="cit-lpage">2017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.069005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.069005%26rft_id%253Dinfo%253Apmid%252F17287529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzIwMDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.39" data-doi="10.2135/cropsci1995.0011183X003500050004x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamkey</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnicker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Epistasis in an elite maize hybrid and choice of generation for inbred line development.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1272</span>–<span class="cit-lpage">1281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1995.0011183X003500050004x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLamkey%26rft.auinit1%253DK.%2BR.%26rft.volume%253D35%26rft.issue%253D5%26rft.spage%253D1272%26rft.epage%253D1281%26rft.atitle%253DEpistasis%2Bin%2Ban%2BElite%2BMaize%2BHybrid%2Band%2BChoice%2Bof%2BGeneration%2Bfor%2BInbred%2BLine%2BDevelopment%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1995.0011183X003500050004x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1995.0011183X003500050004x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RR62100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">The inheritance of gossypol level in gossypium i. additive, dominance, epistatic, and maternal effects associated with seed gossypol in two varieties of gossypium hirsutum l.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D59%26rft.issue%253D2%26rft.spage%253D285%26rft.epage%253D298%26rft.atitle%253DTHE%2BINHERITANCE%2BOF%2BGOSSYPOL%2BLEVEL%2BIN%2BGOSSYPIUM%2BI.%2BADDITIVE%252C%2BDOMINANCE%252C%2BEPISTATIC%252C%2BAND%2BMATERNAL%2BEFFECTS%2BASSOCIATED%2BWITH%2BSEED%2BGOSSYPOL%2BIN%2BTWO%2BVARIETIES%2BOF%2BGOSSYPIUM%2BHIRSUTUM%2BL%26rft_id%253Dinfo%253Apmid%252F17248408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI1OS8yLzI4NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.41" data-doi="10.1534/genetics.108.091942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dominance, over-dominance and epistasis condition the heterosis in two heterotic rice hybrids.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1725</span>–<span class="cit-lpage">1742</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.091942</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.091942%26rft_id%253Dinfo%253Apmid%252F18791236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzE3MjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.42" data-doi="10.1198/016214505000000808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Likelihood-based inference on haplotype effects in genetic association studies.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214505000000808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D101%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214505000000808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214505000000808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235958400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.43" data-doi="10.1017/S0016672399004073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lukens</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Epistatic and environmental interactions for quantitative trait loci involved in maize evolution.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672399004073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D74%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000085417000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.44" data-doi="10.1534/genetics.105.046078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malmberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waits</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauricio</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Epistasis for fitness-related quantitative traits in arabidopsis thaliana grown in the field and in the greenhouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2013</span>–<span class="cit-lpage">2027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.046078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.046078%26rft_id%253Dinfo%253Apmid%252F16157670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzIwMTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.45" data-doi="10.1007/s00122-016-2675-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Epistasis and covariance: how gene interaction translates into genomic relationship.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2675-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D963%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2675-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2675-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.46" data-doi="10.1093/nar/22.17.3640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClelland</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raschke</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Effect of site-specific modification on restriction endonucleases and dna modification methyltransferases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3640</span>–<span class="cit-lpage">3659</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/22.17.3640</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMcClelland%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D17%26rft.spage%253D3640%26rft.epage%253D3659%26rft.atitle%253DEffect%2Bof%2Bsite-specific%2Bmodification%2Bon%2Brestriction%2Bendonucleases%2Band%2BDNA%2Bmodification%2Bmethyltransferases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.17.3640%26rft_id%253Dinfo%253Apmid%252F7937074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.17.3640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7937074&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PJ87300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.47" data-doi="10.1007/BF00266997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnell</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Epistasis in maize (zea mays l.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00266997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D72%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00266997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00266997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986C467000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.675.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Microsoft</span></li></ol><cite> and R Core Team, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Microsoft R Open</span>. <span class="cit-publ-name">Microsoft Corporation</span>, <span class="cit-publ-loc">Redmond, Washington</span>. <a href="https://mran.microsoft.com">https://mran.microsoft.com</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.49" data-doi="10.1371/journal.pone.0046501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitart</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Localising loci underlying complex trait variation using regional genomic relationship mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNagamine%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46501%26rft.epage%253De46501%26rft.atitle%253DLocalising%2BLoci%2Bunderlying%2BComplex%2BTrait%2BVariation%2BUsing%2BRegional%2BGenomic%2BRelationship%2BMapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046501%26rft_id%253Dinfo%253Apmid%252F23077511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23077511&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.50" data-doi="10.1098/rsta.1933.0009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neyman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1933</span> <span class="cit-article-title">Ix. on the problem of the most efficient tests of statistical hypotheses.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. A</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsta.1933.0009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsta.1933.0009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsta.1933.0009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.51" data-doi="10.3835/plantgenome2017.06.0050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okeke</span>  <span class="cit-name-given-names">U. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Regional heritability mapping provides insights into dry matter content in african white and yellow cassava populations.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.06.0050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.06.0050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.06.0050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.675.52"><div class="cit-metadata unstructured">Paux, E., P. Sourdille, J. Salse, C. Saintenac, F. Choulet, <em>et al.</em>, 2008 A physical map of the 1-gigabase bread wheat chromosome 3b. Science 322: 101–104.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.53" data-doi="10.1038/nrg2452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Epistasis–the essential role of gene interactions in the structure and evolution of genetic systems.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">867</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPhillips%26rft.auinit1%253DP.%2BC.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D855%26rft.epage%253D867%26rft.atitle%253DEpistasis--the%2Bessential%2Brole%2Bof%2Bgene%2Binteractions%2Bin%2Bthe%2Bstructure%2Band%2Bevolution%2Bof%2Bgenetic%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2452%26rft_id%253Dinfo%253Apmid%252F18852697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852697&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260162500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.675.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.55" data-doi="10.1111/nph.14266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azevedo</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Regional heritability mapping and genome-wide association identify loci for complex growth, wood and disease resistance traits in eucalyptus.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">1287</span>–<span class="cit-lpage">1300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14266</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D1287%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.56" data-doi="10.1186/1753-6561-8-S5-S3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riggio</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Regional heritability mapping to identify loci underlying genetic variation of complex traits.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">S3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753-6561-8-S5-S3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D8%26rft.spage%253DS3%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-8-S5-S3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-8-S5-S3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.57" data-doi="10.1111/j.1469-1809.2010.00630.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Using biological knowledge to uncover the mystery in the search for epistasis in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.2010.00630.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DRitchie%26rft.auinit1%253DM.%2BD.%26rft.volume%253D75%26rft.issue%253D1%26rft.spage%253D172%26rft.epage%253D182%26rft.atitle%253DUsing%2Bbiological%2Bknowledge%2Bto%2Buncover%2Bthe%2Bmystery%2Bin%2Bthe%2Bsearch%2Bfor%2Bepistasis%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2010.00630.x%26rft_id%253Dinfo%253Apmid%252F21158748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2010.00630.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21158748&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.58" data-doi="10.1534/g3.118.200613"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Prediction of subgenome additive and interaction effects in allohexaploid wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">685</span>–<span class="cit-fpage">698</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200613</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200613%26rft_id%253Dinfo%253Apmid%252F30455185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjkvMy82ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.59" data-doi="10.1101/376731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">376731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/376731</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F376731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIzNzY3MzF2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.60" data-doi="10.1016/j.plantsci.2013.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dominance and epistasis are the main contributors to heterosis for plant height in rice.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">215-216</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2013.10.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D215216%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2013.10.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2013.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.61" data-doi="10.1038/hdy.2015.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirali</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Regional heritability mapping method helps explain missing heritability of blood lipid traits in isolated populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stuber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Epistasis in maize (zea mays l.). ii: Comparison of selected with unselected populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStuber%26rft.auinit1%253DC.%2BW.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D137%26rft.epage%253D149%26rft.atitle%253DEPISTASIS%2BIN%2BMAIZE%2B%2528ZEA%2BMAYS%2BL.%2529.%2BII%253A%2BCOMPARISON%2BOF%2BSELECTED%2BWITH%2BUNSELECTED%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F17248515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2Ny8xLzEzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.63" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.64" data-doi="10.1534/genetics.113.155176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">On the additive and dominant variance and covariance of individuals within the genomic selection scope.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155176%26rft_id%253Dinfo%253Apmid%252F24121775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS80LzEyMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.675.65" data-doi="10.1007/978-94-011-4221-2_12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome evolution in polyploids</span>, pp. <span class="cit-fpage">225</span>–<span class="cit-lpage">249</span> in <span class="cit-source">Plant molecular evolution</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-011-4221-2_12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-011-4221-2_12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-011-4221-2_12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.66" data-doi="10.2135/cropsci1997.0011183X003700030012x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Triple testcross analysis to detect epistasis in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1997.0011183X003700030012x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWolf%26rft.auinit1%253DD.%2BP.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D763%26rft.epage%253D770%26rft.atitle%253DTriple%2BTestcross%2BAnalysis%2Bto%2BDetect%2BEpistasis%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1997.0011183X003700030012x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1997.0011183X003700030012x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XD19200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.67" data-doi="10.1534/g3.116.033332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulakow</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Marker-based estimates reveal significant non-additive effects in clonally propagated cassava (manihot esculenta): implications for the prediction of total genetic value and the selection of varieties.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3497</span>–<span class="cit-lpage">3506</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.033332</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.033332%26rft_id%253Dinfo%253Apmid%252F27587297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjYvMTEvMzQ5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.68" data-doi="10.1371/journal.pone.0031249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic insight into yield-associated traits of wheat grown in multiple rain-fed environments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e31249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0031249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWu%26rft.auinit1%253DX.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De31249%26rft.epage%253De31249%26rft.atitle%253DGenetic%2Binsight%2Binto%2Byield-associated%2Btraits%2Bof%2Bwheat%2Bgrown%2Bin%2Bmultiple%2Brain-fed%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0031249%26rft_id%253Dinfo%253Apmid%252F22363596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0031249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363596&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.69" data-doi="10.1534/genetics.106.066571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomewide analysis of epistatic effects for quantitative traits in barley.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1955</span>–<span class="cit-lpage">1963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.066571</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.066571%26rft_id%253Dinfo%253Apmid%252F17277367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE5NTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.70" data-doi="10.1073/pnas.94.17.9226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Importance of epistasis as the genetic basis of heterosis in an elite rice hybrid.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">9226</span>–<span class="cit-lpage">9231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.94.17.9226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYu%26rft.auinit1%253DS.%2BB.%26rft.volume%253D94%26rft.issue%253D17%26rft.spage%253D9226%26rft.epage%253D9231%26rft.atitle%253DImportance%2Bof%2Bepistasis%2Bas%2Bthe%2Bgenetic%2Bbasis%2Bof%2Bheterosis%2Bin%2Ban%2Belite%2Brice%2Bhybrid%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.17.9226%26rft_id%253Dinfo%253Apmid%252F11038567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTcvOTIyNiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.675.71" data-doi="10.3389/fgene.2012.00002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culver</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Modeling haplotype-haplotype interactions in case-control genetic association studies.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2012.00002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DModeling%2Bhaplotype-haplotype%2Binteractions%2Bin%2Bcase-control%2Bgenetic%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00002%26rft_id%253Dinfo%253Apmid%252F22303409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22303409&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/675.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/663" title="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/685" title="Prediction of Subgenome Additive and Interaction Effects in Allohexaploid Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/675.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-N2WtPr_hU9IEKQ9YksrYQgcUt78q3GUWGhfRJP9jYkE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/675" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/675" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/675" class="active" data-icon-position="" data-hide-link-title="0">A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492438" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-c8JBafYSBT5IrjXvQed1EapiYTvaX9K9aDaAmUgYTgk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/675.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/675" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nmg5FpcnqRZGT-T5vmgdPNAyhsKSUacJp5wOoDBT0-Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/675" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/675.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492438" id="node492438--2" data-pisa="ggg;9/3/675" data-pisa-master="ggg;g3.118.200646" data-apath="/ggg/9/3/675.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">675-684;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200646 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492438--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/675" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/675&amp;text=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/675&amp;t=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/675&amp;title=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492438/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492438" id="node492438--3" data-pisa="ggg;9/3/675" data-pisa-master="ggg;g3.118.200646" data-apath="/ggg/9/3/675.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicholas Santantonio</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jean-Luc Jannink</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark Sorrells</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">675-684;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200646 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492438--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Santantonio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BSantantonio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santantonio%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BSantantonio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4351-4023" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicholas Santantonio</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ns722@cornell.edu" class="" data-icon-position="" data-hide-link-title="0">ns722@cornell.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA ARS, Robert W. Holley Center for Agriculture & Health, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4849-628X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jean-Luc Jannink</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Cornell University, Plant Breeding and Genetics Section, School of Integrated Plant Sciences, College of Agriculture and Life Sciences, Ithaca, NY 14853 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7367-2663" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark Sorrells</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=675&amp;rft.epage=684&amp;rft.atitle=A%20Low%20Resolution%20Epistasis%20Mapping%20Approach%20To%20Identify%20Chromosome%20Arm%20Interactions%20in%20Allohexaploid%20Wheat&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Santantonio%2C+Nicholas&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Sorrells%2C+Mark"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492438/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492438/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492438/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492438/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492438/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492438/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492438/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492438/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492438/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492438/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492438/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492438/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/675" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/675.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/675.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/675.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/675.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
