<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/3/663" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/3/663.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/3/663.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/3/663.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/3/663/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/3/663.atom" />
<meta name="HW.pisa" content="ggg;9/3/663" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" />
<meta name="DC.Identifier" content="10.1534/g3.118.200913" />
<meta name="DC.Date" content="2019-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Clark et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Low or uneven read depth is a common limitation of genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), resulting in high missing data rates, heterozygotes miscalled as homozygotes, and uncertainty of allele copy number in heterozygous polyploids. Bayesian genotype calling can mitigate these issues, but previously has only been implemented in software that requires a reference genome or uses priors that may be inappropriate for the population. Here we present several novel Bayesian algorithms that estimate genotype posterior probabilities, all of which are implemented in a new R package, polyRAD. Appropriate priors can be specified for mapping populations, populations in Hardy-Weinberg equilibrium, or structured populations, and in each case can be informed by genotypes at linked markers. The polyRAD software imports read depth from several existing pipelines, and outputs continuous or discrete numerical genotypes suitable for analyses such as genome-wide association and genomic prediction." />
<meta name="DC.Contributor" content="Lindsay V. Clark" />
<meta name="DC.Contributor" content="Alexander E. Lipka" />
<meta name="DC.Contributor" content="Erik J. Sacks" />
<meta name="article:published_time" content="2019-03-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Low or uneven read depth is a common limitation of genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), resulting in high missing data rates, heterozygotes miscalled as homozygotes, and uncertainty of allele copy number in heterozygous polyploids. Bayesian genotype calling can mitigate these issues, but previously has only been implemented in software that requires a reference genome or uses priors that may be inappropriate for the population. Here we present several novel Bayesian algorithms that estimate genotype posterior probabilities, all of which are implemented in a new R package, polyRAD. Appropriate priors can be specified for mapping populations, populations in Hardy-Weinberg equilibrium, or structured populations, and in each case can be informed by genotypes at linked markers. The polyRAD software imports read depth from several existing pipelines, and outputs continuous or discrete numerical genotypes suitable for analyses such as genome-wide association and genomic prediction.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/03/01" />
<meta name="citation_mjid" content="ggg;9/3/663" />
<meta name="citation_id" content="9/3/663" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/3/663" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/3/663.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/3/663.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/3/663.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200913" />
<meta name="citation_pmid" content="30655271" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="663" />
<meta name="citation_lastpage" content="673" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lindsay V. Clark" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_author_email" content="lvclark@illinois.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3881-9252" />
<meta name="citation_author" content="Alexander E. Lipka" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1571-8528" />
<meta name="citation_author" content="Erik J. Sacks" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. R. Andrews;citation_author=J. M. Good;citation_author=M. R. Miller;citation_author=G. Luikart;citation_author=P. A. Hohenlohe;citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics.;citation_pages=81-92;citation_volume=17;citation_year=2016;citation_pmid=26729255;citation_doi=10.1038/nrg.2015.28" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. D. Blischak;citation_author=L. S. Kubatko;citation_author=A. D. Wolfe;citation_title=SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data.;citation_pages=407-415;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/btx587" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. M. Bourke;citation_author=G. van Geest;citation_author=R. E. Voorrips;citation_author=J. Jansen;citation_author=T. Kranenburg;citation_title=polymapR—linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.;citation_pages=3496-3502;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty371" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=C. Maliepaard;citation_title=Tools for genetic studies in experimental populations of polyploids.;citation_pages=513;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00513" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. A. Buerkle;citation_author=Z. Gompert;citation_title=Population genomics based on low coverage sequencing: How low should we go?;citation_pages=3028-3035;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12105" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Chagné;citation_author=R. N. Crowhurst;citation_author=M. Troggio;citation_author=M. W. Davey;citation_author=B. Gilmore;citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.;citation_pages=e31745-e31745;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363718;citation_doi=10.1371/journal.pone.0031745" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot.;citation_author=L. V. Clark;citation_author=J. E. Brummer;citation_author=K. Głowacka;citation_author=M. C. Hall;citation_author=K. Heo;citation_title=A footprint of past climate change on the diversity and population structure of Miscanthus sinensis.;citation_volume=114;citation_year=2014;citation_pmid=24918203;citation_doi=10.1093/aob/mcu084" />
<meta name="citation_reference" content="citation_journal_title=Source Code Biol. Med.;citation_author=L. V. Clark;citation_author=E. J. Sacks;citation_title=TagDigger: user-friendly extraction of read counts from GBS and RAD-seq data.;citation_pages=11;citation_volume=11;citation_year=2016;citation_doi=10.1186/s13029-016-0057-7" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. W. Davey;citation_author=T. Cezard;citation_author=P. Fuentes-Utrilla;citation_author=C. Eland;citation_author=K. Gharbi;citation_title=Special features of RAD Sequencing data: implications for genotyping.;citation_pages=3151-3164;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12084" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=K. G. Dodds;citation_author=J. C. McEwan;citation_author=R. Brauning;citation_author=R. M. Anderson;citation_author=T. C. van Stijn;citation_title=Construction of relatedness matrices using genotyping-by-sequencing data.;citation_pages=1047;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2252-3" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="Garrison, E., and G. Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv 1207.3907." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gerard;citation_author=L. F. V. Ferrão;citation_author=A. A. F. Garcia;citation_author=M. Stephens;citation_title=Genotyping polyploids from messy sequencing data.;citation_volume=210;citation_year=2018;citation_pmid=30185430;citation_doi=10.1534/genetics.118.301468" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=F. Grandke;citation_author=P. Singh;citation_author=H. C. M. Heuven;citation_author=J. R. de Haan;citation_author=D. Metzler;citation_title=Advantages of continuous genotype values over genotype classes for GWAS in higher polyploids: A comparative study in hexaploid chrysanthemum.;citation_pages=672;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2926-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Practical issues in imputation-based association mapping.;citation_pages=e1000279-e1000279;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19057666;citation_doi=10.1371/journal.pgen.1000279" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Hamilton;citation_author=C. N. Hansey;citation_author=B. R. Whitty;citation_author=K. Stoffel;citation_author=A. N. Massa;citation_title=Single nucleotide polymorphism discovery in elite North American potato germplasm.;citation_pages=302-302;citation_volume=12;citation_year=2011;citation_pmid=21658273;citation_doi=10.1186/1471-2164-12-302" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: Analysis of Next Generation Sequencing Data.;citation_pages=356;citation_volume=15;citation_year=2014;citation_pmid=25420514;citation_doi=10.1186/s12859-014-0356-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=Glob. Change Biol. Bioenergy;citation_author=S. Liu;citation_author=L. V. Clark;citation_author=K. Swaminathan;citation_author=J. M. Gifford;citation_author=J. A. Juvik;citation_title=High density genetic map of Miscanthus sinensis reveals inheritance of zebra stripe.;citation_pages=616-630;citation_volume=8;citation_year=2016;citation_doi=10.1111/gcbb.12275" />
<meta name="citation_reference" content="citation_title=PLOS Genet.;citation_author=X. Liu;citation_author=M. Huang;citation_author=B. Fan;citation_author=E. S. Buckler;citation_author=Z. Zhang;citation_title=Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome-Wide Association Studies.;citation_pages=e1005767;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005767" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. Maruki;citation_author=M. Lynch;citation_title=Genotype Calling from Population-Genomic Sequencing Data.;citation_volume=7;citation_year=2017;citation_pmid=28108551;citation_doi=10.1534/g3.117.039008" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Ann. N. Y. Acad. Sci.;citation_author=G. D. Moghe;citation_author=S. H. Shiu;citation_title=The causes and molecular consequences of polyploidy in flowering plants.;citation_pages=16-34;citation_volume=1320;citation_year=2014;citation_pmid=24903334;citation_doi=10.1111/nyas.12466" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Money;citation_author=K. Gardner;citation_author=Z. Migicovsky;citation_author=H. Schwaninger;citation_author=G.-Y. Zhong;citation_title=LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms.;citation_volume=5;citation_year=2015;citation_pmid=26377960;citation_doi=10.1534/g3.115.021667" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Nakamura;citation_author=T. Oshima;citation_author=T. Morimoto;citation_author=S. Ikeda;citation_author=H. Yoshikawa;citation_title=Sequence-specific error profile of Illumina sequencers.;citation_volume=39;citation_year=2011;citation_pmid=21576222;citation_doi=10.1093/nar/gkr344" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=J. S. Paul;citation_author=A. Albrechtsen;citation_author=Y. S. Song;citation_title=Genotype and SNP calling from next-generation sequencing data.;citation_pages=443-451;citation_volume=12;citation_year=2011;citation_issue=6;citation_pmid=21587300;citation_doi=10.1038/nrg2986" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Obenchain;citation_author=M. Lawrence;citation_author=V. Carey;citation_author=S. Gogarten;citation_author=P. Shannon;citation_title=VariantAnnotation: A Bioconductor package for exploration and annotation of genetic variants.;citation_volume=30;citation_year=2014;citation_pmid=24681907;citation_doi=10.1093/bioinformatics/btu168" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=I. de B. Oliveira;citation_author=M. F. Resende;citation_author=F. Ferrao;citation_author=R. Amadeu;citation_author=J. Endelman;citation_title=Genomic prediction of autotetraploids; influence of relationship matrices, allele dosage, and continuous genotyping calls in phenotype prediction.;citation_year=2018;citation_doi=10.1101/432179" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. A. Poland;citation_author=T. W. Rife;citation_title=Genotyping-by-sequencing for plant breeding and genetics.;citation_pages=92-102;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.05.0005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. K. Ray;citation_author=N. D. Mueller;citation_author=P. C. West;citation_author=J. A. Foley;citation_title=Yield Trends Are Insufficient to Double Global Crop Production by 2050.;citation_pages=e66428;citation_volume=8;citation_year=2013;citation_pmid=23840465;citation_doi=10.1371/journal.pone.0066428" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=S. Renny-Byfield;citation_author=J. F. Wendel;citation_title=Doubling down on genomes: Polyploidy and crop plants.;citation_volume=101;citation_year=2014;citation_pmid=25090999;citation_doi=10.3732/ajb.1400119" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. F. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="Shiryaev, A. N., 2011 Bayes formula. Encycl. Math. Available at: https://www.encyclopediaofmath.org//index.php?title=Bayes_formula&amp;oldid=16075." />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=W. da Silva;citation_author=J. Ingram;citation_author=C. A. Hackett;citation_author=J. J. Coombs;citation_author=D. Douches;citation_title=Mapping Loci That Control Tuber and Foliar Symptoms Caused by PVY in Autotetraploid Potato (Solanum tuberosum L.).;citation_volume=7;citation_year=2017;citation_pmid=28903982;citation_doi=10.1534/g3.117.300264" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=H. N. De Silva;citation_author=A. J. Hall;citation_author=E. Rikkerink;citation_author=M. A. McNeilage;citation_author=L. G. Fraser;citation_title=Estimation of allele frequencies in polyploids under certain patterns of inheritance.;citation_pages=327-334;citation_volume=95;citation_year=2005;citation_issue=4;citation_pmid=16094298;citation_doi=10.1038/sj.hdy.6800728" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=G. T. Slavov;citation_author=R. Nipper;citation_author=P. Robson;citation_author=K. Farrar;citation_author=G. G. Allison;citation_title=Genome-wide association studies and prediction of 17 traits related to phenology, biomass and cell wall composition in the energy grass Miscanthus sinensis.;citation_pages=1227-1239;citation_volume=201;citation_year=2014;citation_pmid=24308815;citation_doi=10.1111/nph.12621" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Fingerprinting Soybean Germplasm and Its Utility in Genomic Research.;citation_volume=5;citation_year=2015;citation_pmid=26224783;citation_doi=10.1534/g3.115.019000" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Stacklies;citation_author=H. Redestig;citation_author=M. Scholz;citation_author=D. Walther;citation_author=J. Selbig;citation_title=pcaMethods - A bioconductor package providing PCA methods for incomplete data.;citation_volume=23;citation_year=2007;citation_pmid=17344241;citation_doi=10.1093/bioinformatics/btm069" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet;citation_title=Correction: Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome-Wide Association Studies;citation_pages=e1005957;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1005957" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. A. Tinker;citation_author=W. A. Bekele;citation_author=J. Hattori;citation_title=Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis.;citation_volume=6;citation_year=2016;citation_pmid=26818073;citation_doi=10.1534/g3.115.024596" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
<meta name="twitter:description" content="Low or uneven read depth is a common limitation of genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), resulting in high missing data rates, heterozygotes miscalled as homozygotes, and uncertainty of allele copy number in heterozygous polyploids. Bayesian genotype calling can mitigate these issues, but previously has only been implemented in software that requires a reference genome or uses priors that may be inappropriate for the population. Here we present several novel Bayesian algorithms that estimate genotype posterior probabilities, all of which are implemented in a new R package, polyRAD. Appropriate priors can be specified for mapping populations, populations in Hardy-Weinberg equilibrium, or structured populations, and in each case can be informed by genotypes at linked markers. The polyRAD software imports read depth from several existing pipelines, and outputs continuous or discrete numerical genotypes suitable for analyses such as genome-wide association and genomic prediction." />
<meta name="og-title" property="og:title" content="polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/3/663" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Low or uneven read depth is a common limitation of genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), resulting in high missing data rates, heterozygotes miscalled as homozygotes, and uncertainty of allele copy number in heterozygous polyploids. Bayesian genotype calling can mitigate these issues, but previously has only been implemented in software that requires a reference genome or uses priors that may be inappropriate for the population. Here we present several novel Bayesian algorithms that estimate genotype posterior probabilities, all of which are implemented in a new R package, polyRAD. Appropriate priors can be specified for mapping populations, populations in Hardy-Weinberg equilibrium, or structured populations, and in each case can be informed by genotypes at linked markers. The polyRAD software imports read depth from several existing pipelines, and outputs continuous or discrete numerical genotypes suitable for analyses such as genome-wide association and genomic prediction." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3.cover-source.jpg" />
    <title>polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__-8wSh-loC8aZQjVpkiWXeUnH4axtmjVhZGuSdz8kVhA__4doWch-QYHBi-iQ7qgOafMkMn_zUNZJNYpe8oI0O6J8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"5JPKM-ooO9X6tLiYx10aTnLyr2-L9ZlvN8bAd3mpq2U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492359","apath":"\/ggg\/9\/3\/663.atom","pisa":"ggg;9\/3\/663","ac":{"\/ggg\/9\/3\/663.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/3\/663.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/3\/663"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/3\/663"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/3\/663.abstract":true,"\/content\/9\/3\/663":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J2ngVH\\\\\\\/rjATnPlslksKqbiWp4mh+vgDkos0uD3i6l9g70xTl\\\\\\\/O2UNQszlUsYD+2r11zizU5fAkqMHErhlt1MqS0j2ov63gdoTpUWII4RcUC15\\\\\\\/w4KlajmbODVkR1CzpbEm9VdYqqdrAAQTLTqe1Pfkn9JSWbqKjKTKETCm8pMpaoHy\\\\\\\/GDFSQVA\\\\\\\/GCcH5WsnOSp2Bvyb6VLvO1aDbrJwhMXii6RhIVdA+g9z\\\\\\\/v0ffNzDc\\\\\\\/elcPPDFA0mnVlMmI\\\\\\\/oMIY2BkapmkRHZ0cNhWIQZQYZeGJszIqEwoY5sMkgmVQ47Z48HwuGydx5aRP\\\\\\\/8+TN1JcFJDGuRyc4uQ5orZ2qdVdLPcyHiAr5UsAvEo3ZgMLV\\\\\\\/m74MA2IlDk\\\\\\\/9gQKi07sizoa\\\\\\\/FkvxRs\\\\\\\/dX35mekDadjO3EZsvdfsAEoQm0hEbclFrW57a0CCpzjTAQ76TdPS1m7ZWdBAp0o4Gi0u5aX5VjeIx+s2eay3\\\\\\\/eH3zNwtrCoNR9VxJmnW15WNGUg3l7\\\\\\\/Uc5iBAbSErh3dwnRdkpWNqBTIp6RZyXNEsElJnEPjm1QQ9mXG9dQLu+Ix16Fzd+8Gunt3PLe7asikdT+8aFsJ7jehkmOB6nnTwHc6NFMPK7zotmCeluat4\\\\\\\/F0hc0\\\\\\\/7eaHnRjw44KZQtLwbg2AnsIruVfSpHMRjFZe0dHy3kRc+AzR0oJdTr+3xrQMM3PG3KGLB34t4xEGYVqYTj2r8fhEEzZf9jZp1\\\\\\\/4X2Db4SOODUf8KxFQEPlNrp8IY\\\\\\\/kQIeS8pSoS2AoqiFrW2W6mBaKsQZn3zfGUbCxrms8th1EtY6mhnkrJJ7A2CcLjCXHOAYab5Jo6XZwRskd1PqXQJXuFMqFwt0GqLp2kZiGEQ+DWA+OVho7ul7YZPvYTQQ7kqLeCqFpoONjDK4210H864dnYDNvc8Zayq0K8cgCIprsWujZq1AIhgKrFycbn1bg72i\\\\\\\/e9gyOLyId86iWWq\\\\\\\/qeEThSF4VDL\\\\\\\/dSWVP+SEH537tJlwmjZDG611kp8xKqXXcDqQAc5gyQFGfGM32Dogp1P+LTlMC9LvpyANvQpJcoqyyOv5IQ3LploMSLKajGLEykSWpBipEbVUoh0JTUhDOW+vpEUD3278P+uI8NTammQQfBAtOZuE9TnIbQgI+E\\\\\\\/gy79Kvu+CW9rcrhNX8WTeYOpP3jVoMgE7RX2YsVVrFs3acUJJcyXILIIcYHHuBsTwFZyftfYa6bWAgZ8Gc7d2UdLcUyT8EpIJrbGMNyrjMpPkPzJ\\\\\\\/ACaiKUn2S0i7sHe4\\\\\\\/TMY5Ms\\\u0022,\\\u0022iv\\\u0022:\\\u0022ARJVcsJ\\\\\\\/WXY8oHYBnAXDdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229882280fe11becdb04f4985144f0d02e3dbdd1951a21897d667cfa4fd9f66350\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IvsnJ3gFI3\\\\\\\/+eparghiCmDaV7PkI2ASXIbL0d9Q9h86raj9SREXDfOV3kD4fwl0jz3J4Yxx8ViMEMAxC7a2Q7RFeOuNIUDMj92mEy9570gO0KTZpGYfUJ2eWWOMjf+PWwfLxtfh\\\\\\\/c939+aNKwbsOMjjF69xy09\\\\\\\/nD9iJINqTbZJAxbHvnSspKFzd1J3rNQhfmfp0zD\\\\\\\/h44rclUKk5Gyevo0aPgCNJcscvshsWpZzvVjOuuCWZc8pwoN2kvz3enQ\\\\\\\/NBy4Hdl0qn9uVZL2pQ1yKZ8Fq845seqBe5E5WnLKHNQrva4Fn0P7YQGz5hfB2CQ7N0j3zOp1Kz1ZhC26QzFmVTTf3VQgRwZBqXvkov2IrEmStGvRv1fRIGOGp+B61musD52sPDtgySO4HXY0ASNS9KJ1++T+TU75l+GlnxdullKQL0d4EGqcMzwgU4Er7ZtVhh\\\\\\\/WbTYKDk3MuEdNviPZOU03DG4HPf85gGbFwGm0uZKbCn5bHODwHebSjXzP0g7nvq1xfC1EIshrOgPaUfWKRSZHzrElYoM2jPUoToGM2e3FKSt2HiFTNl2MyVFREtxx6zSnQ0zvshbCU0JdsI2P5BbnwDdhb11nnVLxBy9VtoncMS2AWAuOBwn3VGGRKRnagR2bKQNAetFajDVwtBzjAWsTmwiEAX0yda9VQICfMqFMHF9kVmfuIpVhRmb5ZF358UeUCQY\\\\\\\/E7+b64dUEeB8ikPch8gCRQr5LBRkFM3VTosC1jwmcHzIgcxzdH54pwNbMd3BQOGtuqd34E2QuUJbKdxttVHUPHv4fzKplUnVVoWX+IIVM+d0XKeONrefagydfsFuZ+WV+ukV5so\\\\\\\/s8YeDzSUuI26YPv0tELAaNPtjj\\\\\\\/nizuC7fD5JJtxBNTTPU3SZrpFiMaIrIrmJ9oenwTpAlGN\\\\\\\/7ywGKKJ13LV6Hi7IthjQdzTIxe9xN+d\\\\\\\/H\\\\\\\/IJ5F4uxmWfiE2OpolebrKL3zeclmrOVDCjTEaF2Az\\\\\\\/c6aM\\\\\\\/nWOm01T+SmlxGW5XoHX34v3s1Nce\\\\\\\/bt5jKuDHfG9b23rsCcK2WBYf+JzO1SKhUxmhmukXvnDkpFH0FswgID48IymzUBfOBhUtnxkuWp0p5ZB1tTNSoR9\\\\\\\/I5VvmazpxsL9HUMC1ypkVYVdbGbGf8mVf56R7Kihdb4fxwJfjULu6XyJPZSg+ADE\\\\\\\/8VXw5QFmw+lLbMxuElP38MglbunQstE\\\\\\\/itNqy\\\\\\\/WhsyPRUDJC83hTATBaMWJaXW8EMmE2VhwTtuMGHtHwzuyBTFXUZ\\\\\\\/oZkM0\\\\\\\/L+AXadkQ5ZZI079gRB31bBYPMybxfTykPqyu0P6OOs5fNL+w9fisDfsGOUZAKk4CVGxUt0rYr4FnLq4sYc1hKgJGE06t2QSyhkC2ps5Bnqke0Au5PRok3nQa2xv0Co1e0qOYy7krGLKMUZp+HzHyc+0jQghNKE+163+eax3JC1OPX2Hr1njtljzU11Yaz8TdB\\\\\\\/xYhzQ3Ce5O8\\\\\\\/VKOd4RvJ\\\\\\\/Ax3XlB3\\\\\\\/DF\\\\\\\/GhbtXhJU+rpg+pcHr3KvaH+ceI6zACBWWx5siURbYgMcrGYqKqqlNKly5vo1FJnnfQ1fxQ6V0ADr4NrxjQbD7j9nToamrYpMgGqVlw4LXoBtIeM8SgnuPBtSn+i00W5wYWOwifcVc\\\\\\\/jc7OGLpO12Xsqk9zeeZT7vkES\\\\\\\/zDyqD3JYCy8Hg1rkE795Oss8gKc\\\\\\\/gQH6Xcp7dZ4EEFLp5hFBi0ModavfoMpUEJAsR45N4jvowWsGoCUpA9zaX5zELEA21X8Ufyt7vjK1CUpNn6uDZv9WoTjMVgFTvrhLnwNrRMmBoLeEsPBXtxmEK5NVczcCsJhRTukJETpBtMt\\\\\\\/7mg\\\\\\\/2gdyefv99NbgmvV+0VXl+s5DRs0PNWruS942mb+VtCUriYrWZ\\\\\\\/4EmCON+AUOFrmVGP71xtY5oxD5X8Tu4dh3bpbLi6SjCP4FYsBs6tctrTb\\\\\\\/zucLf9mw42qafGvy9DlaX1yCoJpsIlZkxvqVDtLfU9YrXL5gMH8puyotlOf7r5F4kfv9+BG6mze\\\\\\\/LM9qW98HLKHMJovQ\\\\\\\/8jtHtPJi17t\\\\\\\/ecQdfH8E8XViBtKkOgpVbXqqrJUpNTMUxwOpintS\\\\\\\/HGqKobc6rMwxPNH1gvoJ7DKd48mVocegwleA0EMAcFpqJlVUAWsjRXU639iuy01zcmrAgmbUUSlpnGc2\\\\\\\/e58n0mNU4q4sOpPpB8JgEfBJqzLdKplOeVjv9Mnj3TKSxCIsMC04eQ4Tts8MqRP10o\\\\\\\/02GYM8pQElt8SvevWhPGX3YUBNkZjHI\\\\\\\/W4P3k2SpBG6ZAo1G2BWbTyll6MtnH8623SxzBep8lo1XI8zgwJ9e\\\\\\\/zCK\\\\\\\/+X9I5OF9HX5XgxsrwJtaKnWs0g2mKsKDFWDjpmikBMbpyFmwXn6tZvlrWtkjrp9ben7r2XT5ewHSxLZg7cY7vj0U3KAon\\\\\\\/o5DltFFEpKlrUTeJAaRNtj0Lj8zh8oVBhSEnR0GRjZFgN8B4Ll5xxmpqexiAHYKkdRA3Yb8033ZwExb6aRr5FdAfpiqK8sBOfhCI1Zb+zdlp1sMZQCzxleJuLQq7STIX0gTYIokoLmGXkTZMJpk\\\\\\\/6SeOeMxIgh+AaYSqxnbmFAzx7TSDDcT6Q0\\\\\\\/rmxIqpLf+HGKE2n9hal72vmKZQIIg3ba1OzG6y5A0mOTl76BLH4LtJjWUezgkH\\\\\\\/axfZLnInvvNtsZkZNM3wuN+QOptEO7REtcJkGL3cKwbV1alK7XtXHashe2A+XwKMY28\\\\\\\/LmhhzGtv7iDyDclQMGn2tMoWsqRMbq+922gChT1xhZo6IdhJZhejDlFwkiZ4OW2RmMybi374t+VOy2ZtRW9g3OTq\\\\\\\/xX3T8wkXZDC7tCAsRpRsPmSjJHOJ1NyqXFhzrN4vE1dW9TASFTWlvN8HQ5+EH0KWLu20b4PpLH6YAQkwXBYfKYbT8v\\\\\\\/oGDX3wkImTu6SQXqpbba6lL5y9b7milG2AvOx\\\\\\\/Qxpw0n2MVQ+XJDiLJKql0QjE6XE7pcHVsigfc9B7bE47fQLADRE8xFIFXzl6\\\\\\\/8YZzIYjfA5GCv7ehwgUzGnsRAtfFt8C6+AuM\\\\\\\/ZYdoZmJzLAy6P1\\\\\\\/Yi2Z0pHnPNUfkeHK9mywj+fGIY\\\\\\\/eUKHQe2A89u747aegvUiNYtuYeIlWR+fbaQWIo3OFcL7Z2vw8cMOm\\\\\\\/2GBeTmovgGl6AcoDxe2Z06NnQG6l6rIE6T0l2xXxbu0SWZHavOKhKM33UhHF9y06KQJGHM90cet6YHb6J7C7L+LF\\\\\\\/pU27y8+UDIVAd5EDOgfnoNP5SVq1LfaEFTz9ROebXWqK2VHThj8kRKhi9gE0K2qitJCcpDPl672GwdghIAD1WutRoU3YmDcSLQSKmAfpS2Tfb\\\\\\\/0w3DYcFixoHlAVXrJE1zOny90vh8hd23aMPEHWnevPhVNCP+1AlcbEAudsA43vVdLFOI+eIHP\\\\\\\/H9m2aKqUdxcRF3Vmumjq+Zw2GTQZJ9Zwgho9g0wLCMDGsQFKPmNsk8cmpwfA7Q4AFUUBedOEiOAlgX6AO9l7DqRAW0nY3QqqAY3FhDn+7clE3q6sgGYXZLtSXRzX0Ix6BrstG1VrZ\\\\\\\/wYG2dazMaT3E4ay+j4WNVjGF1IuYXJe4dTCXAIZibsCag2dvGdXGn3doxsVeoBhulRKJydZ+8008IbcbQeNVd3CDMJ6yA9jNOROkFfpWHZS3G6pALAJ\\\\\\\/XA1sbC1GndDkoQ9xUvhEU7MvDIgL9id+nHrOKz+l7cDMaWU0zvejTvkM9WHi7OVosSCTzmt4tMQ8DD8tGsdB6qyX6KV\\\\\\\/08Wu9Cbb9YsORuXtcVLT4c5dfAXM\\\\\\\/ljr3st5a\\\\\\\/9LF9YjUrdn91U0mcuoZnOB1qscpCwErJ54Ou+uWUmJXgLUchzLAJC+nO3ddS2lgeeAcXSyf17WcKIk3cbb7CtRPEDafMdCWkYl0RM9AyBHk53JJdPCsdoBZ0FuR0V\\\\\\\/drOwBDl06RRGcfGqppghQKdA5tgIefc+t0uBIqjYRk4lKLXRh9B6sXLHmrUQOMGMZ7r0nkj2Fy6VInEhuMCLyy7JkQtk87e4jsXBS2fHeiS5d22dDMgwiJYhuETUNmlGUhdUMJJ632eM\\\\\\\/aBaqfQBFINYcgCZKS3mKmHM4cgeemm9UMSqEhRY2ZmS3kVqZfkHiPY0wy8lk0ZkA16DjH1H2qjyfArsswrn3ILDhYJst5lN6GozcWD2+muyOHGyB8b1tcbry+BKHcZ+SjFxlWt4Mrmi8SHMZC\\\\\\\/Wm2bst3blYWpO89jzykhbzb40bf3v8cg1yQGv8HdsaFKMn0BQNPONX5wr9xX61XVAu6oH73KQVKkWahL+i42SVzymaWa5XaftvRmZZbefVHhIWKT07gOLSy3XwTrUAaOXfeEYHMIo3+esWFEngOrOM0UKuxUxpofs\\\\\\\/uxnn\\\\\\\/rCrRS6ZXNcS7\\\\\\\/gj28oAP2hhVOwa8HlrxuveSCYeYzVrlNffOQnyHaMWc2Vr9sgfSYtiwkmg1yi116Ii5YuoEn2nBr86eevggCL42AVxnva5TGGgJGGO\\\\\\\/i\\\\\\\/Bgch6+kQVMiGhRzMZQ+oUiCARG8VU2T\\\\\\\/dds0yOW95OL4Ax0m+9ffii7brXpoQMvYSlVo4kejktXjht35dVkv8bYCpZgEIY3I\\\\\\\/pV+Dt6+9VAl\\\\\\\/UojUaqsY28CEB8voWXLxdj4lriCio8LIROSs8\\\\\\\/D6cW67lCeS25u9k0LL9j6lCPvQ6UxHrzwAW9PwyDwsWwnNfYvH3E3ZeznOcfR2FiP+G9kT5+eGTGgU2SCiW3W7wOziHQMEHEcfnTVUmizCx2A\\\\\\\/Ly4UvH7Ur\\\\\\\/CfNP2eNWswCB9meu\\\\\\\/EOA5WNY9xlFXprxgAQcX+ursulSBDl+cvt8kwVOXF\\\\\\\/gLJIfZEVNnZuoXc60pafj45cXJ8UkR2uk\\\\\\\/Df2ui+9mEYGOhSQN3YbgiwP980q4mJm6TDHIWktiNh\\\\\\\/vmRPfOlSiRFUMOH1fyo+3KA5yq3ufVzwejGL4VaMpfpwo4gBpgSFm8a8eQKODQZtiZ7FB5TxtJ9I0l2ppb3xcY7hgk5xppv3lvDLMfctYJIIL13dxwNYzH0iuMpTQoS8xbB1s8OBl5KIVXo7DfqIIHIR7SYnh708LyovlslK0wudLLIiV8y4r+TQWHRPVOrcmePOvSKU6cYpb659ChmdRM2RjmIoraorlYEN4PpGMRw7vpaLQfiT3gDtEbB0bSXjWO2wIXksxE8mEh4ulaWXUw0MqjVrSmBJYRv5czFJRHMBmyS08DWBDVfbZziAKCprizFXdr2zXdYy7StYo4L3m6f2bgm9zK8SkCPtYv9YGRJFXyAAaxRmYIclKL7b6u4QH5KuYN9UVkHmMNq28P\\\\\\\/7lnoAETHX7alBOx19jP\\\\\\\/sD7Oae4Jzv\\\\\\\/j77KiYQ80iisi5jbgxpbdoC9JYrLzW6YS+8jSfydgs\\\\\\\/PmjgLz5V\\\\\\\/QuphMZWBk8ki4ytPgtTVcQEOa2286onkKIzNiglqFtM6LuwLxLj9IuG5QK4F4H3uSoJS6bA7M7FJFoYHUdPdMqQFq\\\\\\\/sbLgDglfUSZVqNVcC2XW5L4aonMH4t1QvIkbnRYGhDI0jmD52CKdJZwlCP9Q9G3EvjI6wjw3XJlrvGs0fJSqfjB10sN3K4pCoqMZ1N01f9mpM3ySI5r6hwfPX\\\\\\\/is+jOzen+hzmXCtpyBXeqTBH7BcuybVJetdpyTFOW3nfgfPhcnuMmqt+iQBMU2fO7PatHVQVUuchMmRX6TvA1P0se8uDRHcquUNvjEvEu\\\\\\\/Efdz0Hq42w4aIsZ+VSgvv\\\\\\\/lcrbcKWUA\\\\\\\/tp8Svg5nfQ17pSKV6K9omeVJuwZOd30WutymnvOw0OmuaAPZU0XmS4jsqksVHFtxVavEC+54gW2Ll7e7rDWLToaMTGX6rU+Zkcn3vqrBfSDs3dA+3AUJEcxS5Hzgj1IbwLvNokdv3Dl+5DGCuF\\\\\\\/l4R7sMWsWe0jNSwvBMjizdVnwAxh3W1Xx90f8wgUKYMirETbTW\\\\\\\/RpQOeSBEpkdE+EPtH\\\\\\\/TXok89KxKtMR36\\\\\\\/wxYlZc\\\\\\\/S\\\\\\\/SL2cj\\\\\\\/5P6OCkhTamM\\\\\\\/9LfzWIXSI13WiyPtWvCf25UKm3MI9iDVAT1P4Vd7Jc6xS4cFyxwW7poz+e+\\\\\\\/Hhevo5UR\\\\\\\/MKMK4TMjAUlKnLbh1tghaDAnt9Qsq5DM0ahUhaOVIbzroouOoL5+YtTVg5TW9be5ZeqaOq+7Gx\\\\\\\/11gM0IAUEOr1At0TEwMWG0fgn5go\\\\\\\/q1zZlHODXUAL0nYmFaP+0NEin70z55ajvk2qm9wxO+HgQlURXAJckPO2775nxx+ZPC6WOLKS9yvaYcqxqj+ngzr07ZdiVux8cTlYvxnlcPzFbRoLb\\\\\\\/O36O+lKjBbq9KIRre5rJnlC6G1blfPxwnurnjnImcuXgP+7FoKHq5vxVjxNsKGic2\\\\\\\/CgH8oD7DvXYRYP83YUy14Qts8p\\\\\\\/\\\\\\\/Sjqc+mha\\\\\\\/DA5rudXvLEzpgJaiAsBIHiER7RTbnm0yVUcNLh1S\\\\\\\/xkin5QB0NRBSmB6xIwZtKkNvO4BYDroMfiUneVuGv9qV0ZHO8tPecgQPHT4CZAQ1c3bmmKaNui9dfBTDQxA0T9CsgPhJYPIN7\\\\\\\/XjPu0TGzAu8AWKUubJQdduCkP7tLErhOK1HzRp2LXxExakGN4SYRRuu4uFgaDHKLZQu7DFHYvOTAly0UlrHmZg9nNpyQCZ\\\\\\\/1ub6AaYdTULbre+K6eB2aVEW8M+JED1+OAx7OocyOFKQVEcS1yNv9iOTGEfs4Rq9sjz1SmSvyfLfp\\\\\\\/Zqt9vAw1zyODwX4MTQn0+Mf6vanNCkK5Kf38uukS13lpdjXkHw7704uhSUx61kjuwhYSYAa1M37vBCG+MUBNnX5jRKZY0B7b61WhfNN1KZvs4NFZ9EvSDftkpnElOQePMCOuGk0hWeOPFPV3k8\\\\\\\/9\\\\\\\/VwBtyKcchsL9nMNzsCyiXYzl7Uf8HezrgfPojmm+6UWasjtX5AiHr1F0dv9ii2+6xjfpIHXMiYrodPvWA0nNe0wtVJ\\\\\\\/S8Lqi2RonRoApqQRqVNoqwZBEdv74Gr+PAnuN88doRf0k2a95aZotzu6ipH1yf2Trd2oyms4WnoEyr616TEtudZgJuFV8Fx4RyclVRHgK\\\\\\\/YI+KE+Q7sLBuRMEhL2ZU7D2Wf\\\\\\\/3oQXD6nJsZ0qBTIAPcXFufYuGfTtrV2zNE+K8IbZ7JKeLjTTwa84YGpcO7KcoRpLAlvuRKQlk86sGlg1VXdcG7LDBqzoENOAbOTy+FvEPYYlvBRYJqVCyMuFyJIh2moQSIKItF4C1RwAExIohtYVKG+vCQEuef9Pie\\\\\\\/Tz5nunzdUjQ70GMGam+ozmLQnORWg4MMaZ6EvBRRSBheZ8y6H0yVsC1fZQfmbamVT7RrV5Nw4EpSOunrjX\\\\\\\/mswpstew\\\\\\\/HgiYxK8p6MGEGiwBzeE6jGvHOiQPxa\\\\\\\/iFXWQyUWpU\\\\\\\/9CUFiqScy2N1ZXaXRUJG8LzwwKsZmd6Xqrztu\\\\\\\/TxOJm27bTBhZY8cC439O2JvCktIlkE39hUnjUyt+Vr2gSHd7SFRbdUBljRQGXcBi2GpPbR1C9FrKrz\\\\\\\/9krZsy5TcEFdRhknLoO7pmt0VQl6bQd3CuKSdB96AvsGNes6kxU3JqBTao7x3T1hehZjBPPJT8U39FW3pN\\\\\\\/PUIunWZLQrUm7ao53Lu\\\\\\\/A3uNBnNkMC9qNVu60TFIBrpKZY9GzT27I3UAcE9MlY8fQnFmQAVj6SbOgPNTxyPFvN6rAQwVkEyGzfv5LGUUsd\\\\\\\/5QSNtSLrYqDPWDRkz0\\\\\\\/36Ew\\\\\\\/RTINGo9MwNUImA0iJVrxQSQKinFDjMygqVtDRj76I3l\\\\\\\/t8uBW9iiqKK4aF9ijYMBPSch5Thi5HssI4baY2N1QJWZrM9vWFlD8MzL4az6vMMrkwAe3WptPVysmt8BaHD9Q\\\\\\\/CLRwkVayhMKD0XJ8Ix4i0p2vG1dLfBIVJFp5wQxTVTddk6LiMIcmiWT2EeiN9mksV4XRmaQRcSP17zH8re\\\\\\\/e5NxSf+PBjx5DvuaHP3dSZj+VFC289LhDzdDG6Fc4PnH0nhhztAJhM0m0v2BAnOp0157dzWWd6QMffHurbBDN88P5I4\\\\\\\/CZO\\\\\\\/\\\\\\\/0i8xUT1MdHYN9RMvHkKtb547MChXM6rk91uY0b8QHjL3CAIYCkMyATWisfTn6U8xW5lC8ChbMuXQB65jxw1ifkZ9+G4AasCBXnY12wjMA6rsTBKF+gBJbPU3qTb1bpKeNaujs+mEQLV3aq2pNAwvU2ZNiNrRjnXk\\\\\\\/Tw2h4OgGK8FG171IsywWLEGGVufvNAdLxf1mg6gs1xsqBzcWAdeggDP9wM8n3QHZ1QsZKM6rMwkokFS3DlxYSjzv2TPG7BL3H44jYGygvrYm4Xg8K3QkW1jiyGLW8XUW\\\\\\\/+hxXiKMykJEclbB47I+hzM+mph8d3VXG\\\\\\\/rXXJiVGB4JCBKa3eB3ACjxd34xGqWE+m3vHUQU1+HCjyhqQp\\\\\\\/suLhLYRx2uXPwA6IouaRiHzVTZWH+r7O144BdVXbt\\\\\\\/7O0VOKsXbS8rkr1o9KovSve2XDMI\\\\\\\/c1atr5glpZLH3rD9HROg9tAcImG+PPTxfMOR7LyO\\\\\\\/TBQHt2cyb9dDIzb3Slvj9Lf2XINsXKABQ0c8jPEw\\\\\\\/HnA9imNN4ibU0TJkwhgNso8bBujCRHe4gm2oZW9dxtqp8UIhQwza0On3QGAbzWQes0CzaM66Pg7aQNYUifqGCxcuQnkmBOc+SqdXU1F5VzTxLTC1R4ZE6MAmwn\\\\\\\/zTk5WbhXedVZqnkCZ5daiU3XUXlt2tg+xU2fLeM2Dxpc\\\\\\\/QCPADr3+IDjb9eqJkXmRIgWKDR8YUCsqDCsF3fHMROBqEFm3DaQrYiCzP9BZJLeJNkXDlm2Lsk\\\\\\\/+TjPSxjdb4XBeFwW02AtxlgchnuyeTMCn4OYalaQA12BX3Ql26IoM7m3UU28f2SUHnQ4ASBku0NHgAyGLlgUCL0XN\\\\\\\/PvkimkVAbI6AqmNlhmLULPl3zEHmozfSOueO1rVhFlv6oBAFo8NvZchPp80kkxMycWdlZDtSItdSFB\\\\\\\/JAGXA94dHHUs4S6uHaK\\\\\\\/X20\\\\\\\/O2rCI\\\\\\\/b3hHd9VCXxMxJLqUuDyeqgX3KHd6doHePxZ+GORjHNqC8XrEdHLxTEYOtfzVuxtUFEeEmpnhr8IXdMv2ukg7r2UaL7y\\\\\\\/dpSuyDMwZ0RHZsjCEi14A72pDTwZQcjxx8Sp+hIWXPsHKW0pmBy7cnjEWGWtXsucsYKNoPy+\\\\\\\/qvaLPmwlJ85iZxlFFSlVP8kLxbozOmAUJ18yaD+bO2eNdJ6dnsP1i9rloSjcSagYas3SYLRlhyJNALYkL21lj1t+pVPoqNtO3xxedLH9TbfkeFixgCctZScftOe7EXLwSfdl6iNKx6tiooilrWICzCTSNFA7KCWxcC1YwCzz9KcSsGZcXAo23cig4a+FDRpBFXfAwzNUr89CCyxCjQ3JWxvnktKm7Pd0GPII+IxJfwRjKG9XMQLmbaH1memdFfjstWzrXzYpC+wnb17QZ1KrltQsKZdggvWdZup4dUlTBPZAcGwEE4Fp2ryVaqZZ6Vl22m1rQy93vNWsISeJQQKbbGL+YqdaHk52\\\\\\\/KCHLIcwfKEdCGa6MKcSnNlDMmgD\\\\\\\/UN+JNuppr7h\\\\\\\/9wxqfHUIQaZN25mgxdxjmlQHvn87ICjhYuQNnyB1ciPf5aXZwpFEqTqOdVq8WQMjdIhYrtrjU8Fj1B7U7hOC1P8WGQT\\\\\\\/SFR3vwlmGgWBLiX+KAOLfGCTElrNRhgXG3sReCooQ8d7TK8IAQDaFOuYCq8ntJv6ASi1CIYWhAl5JVgkkwTmb3VngZez5HF5lvmNBndANv9lUxtp650B45YDfkqsVgJyY++jD88CdRj795eoR3cjMmjC3yZkUJvtZcqlwkF4E2612juBzzpA9Sz1Vwptk8RRnNSCZR4j2uDHP0Dqo8fOe3\\\\\\\/9gdXUrTEp7O\\\\\\\/QGvpc3EQsdCRXvPNDS+\\\\\\\/o5v0SY6bfsX\\\\\\\/f8RiviIxdm02pUeBRna0CVeFRdKSsef37aynHPVqlY84Jpp0Lf0VhdU8FoskQifTJbu0kHPBX7djChoNGjsMVOGByPkD\\\\\\\/MM3D+TcfAyeYUrB1JU9cltk4hcKVSop1nkhjjSofrK8QoD9bAbyyQBiIfb01f7D33zgQ2XRk7uaHM+dnekmG35Dpm77T4iM\\\\\\\/GRIvhVVqTG+vWsijw8tEdtD5CMCRehJAke2AEFMItstCQ2QG1mLECDbug3pM2HpK7Hp0Uguh7oC+hKkvmqTvD5jP+nIMR++4bgvJ1ZKjn0n9\\\\\\\/8rJOAVg2I44FzUMt473SpNKwvdj2kS45rE9yMzFHSXl3Ft4\\\\\\\/XwXnCVSzFXDpdH3ucEeedV3nXDSi+fuMi1LYcxXxgf6vFKROZpA4AFIx7ZP60QfWsEQjHkESbs9YGRUwvVCE5BKz3KgOzxzDaAdzZDrZw5rJkK2t6lxabKzoomKgH85AxjAAOqmWs7tf4usjgBhvDZaTCx84DzbVsPtP03O5mRRNWeAbXXibd+rxSO7M96ug8HLKrZsAWKi2AEIMX+5VAA8Hwpwk8tMPhfgtKwy0xMAWz0iL+ORCIz2z1jP4W9C3fEBE\\\\\\\/BpU7NxdXZLCn2YWOCDHzmzh3hckhOFkrA6Lrp0I0lWix87JIkgMapX+33zWu1LKKcjDSlYHYZZzYCL7U05INdNc+EuFWUs+U1kn9TvY5LZKvHHohWgGQRE\\\\\\\/4eFxWDsSWGvEtKVi+o57ox7pLnO6ST+e8Qt68nPjdn\\\\\\\/fua+A1aPR\\\\\\\/t1HT5fuv6RrGOnP+doASe9gpr3uCpa7iqQCSmwHV1658h9oTCAV29hfIy32GcRibxIWmQj1MgjxhKLtCw3lrkeznIMBihZmwNoygbf\\\\\\\/2MnTV8HBQrH2NdDaRm5PY\\\\\\\/NKXPnOW36bi8o8cfgUsDESA0h4AvjBQt1opWG3lagXIxUVRPsM0ZEK1qkNACjV9XnWg9CvWasoB91LBmzKgGa3HXcjotTKlG49tMrGMdJwQE\\\\\\\/G10bYlmi68BKxZl+rEDrKW\\\\\\\/iyUSlp2ALcvIGMAK35YtQJKfXl2wBsl1uWBsdZlKp\\\\\\\/xKL0KGCu1+fpfyLD0TTCuYAABitElNN0PbO5AyLFgZL+PZPMP4Qo+C+nODLk+PzHQc21djlT29BaocKVH2\\\\\\\/Hip\\\\\\\/ky1b3NURdae21o+mN2tVwck2uOsL5XwU7H6l5yfR7Upy7EMWV\\\\\\\/j\\\\\\\/2xDIZN3yUgT9iVJBBNPEndrxajMni4ixNv\\\\\\\/f2dQpIif8NncdmtQbJT0OXucT9oQ8ypTRNDWWfGu9AJdlHfBe3rC3skzHyAy9PvPP3Nr6lVS7W8vwEg9ki9cpDwFYCqN4aC2BzVD1B4qKqrKZp8UQ0OBqdS5UCINOFSk5khB8UCqayMLveZG4h9fN8YLjP2lLANWcK\\\\\\\/LlYNyKGCWmjtdoca00Gbj++WpsJkWk8lxbI4QVo4R9Cpm3hRE+gJ6ldRPa5UIIpCS\\\\\\\/2N+bcIKwkurbkqB+khnhlu3pRX54eOdu0TCS7VOsRlWhsTKXKQt3kHuiTcszoAUcYCmCKIsC3SU6ayQccHex25k\\\\\\\/iYxfW0wSf+xnCHp6oHl8fMGQZIcoof34gEr7y0lQb7OY8ZF5uTOmxrBBoUjAnIn6VI4rvxjxMk0rxZF0cqCnkvZE3\\\\\\\/kakhbkUjp3QvZqr4yElqx\\\\\\\/ZwKR2ouWpf+rE2DLO15O8\\\\\\\/r1iCdW1DR63Ss2vp8gSvjgCEDCVMTd7HMucwbKpnOLYvuxs6VTIGd6xxGTeTeOTdEcRz3ghoTxs8xOjt5YRblQc3VVxKMidZadguRHOp0DWTnf4hNdC2mVaYPf8zVFNgl\\\\\\\/6rPKbWjvXfimOpEM+5xXuvs7aGDzBani4\\\\\\\/OLWrQYl09q1vLYXn0KV2vfdc\\\\\\\/B96eA8M18+jjDJAFrIRtx3SM\\\\\\\/kuAiYf0vZnH1oMrAN0CWtZFNjOaJKxjBaSCZd3oNQVz5z\\\\\\\/nCn0ngnpBYWgjJd7lK9UhEgwH2MtBsPiXeW4RXFwYaAgaVlN2rp7U+4APBuzN915gWiRYLXPVxmfuwxV5FAgIv\\\\\\\/oY1uxo5J3xPQn2irvw6Iu8ieLZYHDchFxy\\\\\\\/xaBYsWAvBqd6FiZldKDJuJGGVF\\\\\\\/DsqNZ\\\\\\\/\\\\\\\/3hnsVSJz5kSGYl7kqRucZHyrK7j3UrIF9zQr6nn\\\\\\\/lsKhecREUIAMuWpao\\\\\\\/vtARIgpewW2RYEOBmiFDsXmL7eAo9j2pDI16nfrrULukZRzv602rSsGhHhvEqrwTgU0zXz6pUTIsOeN5DC4AYY6U4Mz+ZZxLVBQAmRz3oZnbKEepccStLYX0gbdrNIT08R29qEgADlRZha9YAKX2ilNSzXH0F8JlEb0qCo8G0cSseb1UxM792bgM62V7Xls7kIAG7RbASI9So8wVmkA6UNQprwL08IFov+30E6vmL3bl1SlcnWmG5zctOdHD6So9Isn0FmWuLhrrzuNLwiAB2eJe5dG7n53M6mmjRpEhWHWcbRCIsEClmdgJz36LMvItJGNOh6FeXxJ4MoH31kNBg8n9jx4UW6zNgqjf+G7vB76GDicxGxxAlpjFYAiDqRFaO9hRpNW4O\\\\\\\/dTBclo9JKGMa3mhkhso3WmOIyQfIG\\\\\\\/KhKNGo458Q4X9NY1SkwSr+1dN+h\\\\\\\/wox3oBQ2f4Gm2p3osHKBRzFpEOG5P5P2MAJ7tUm2JHQl4meyNQtKwspIHGOlyq4Hj3qpiUYSWmv16+sZZqpfbDGm3kFDAoSVJCZaT+Pvkpp1w5IKGjbRsxbD\\\\\\\/c6CahiifKQ9GdqWwg8L1Ewfy+6Y\\\\\\\/DVPBVJUzjt5cXIcUOPl5YMUdk36aw3RKaNq8kLhNd76u2pW4AHuLPxMzj1HleVLO50a2Qee1e5Cn07v5TzsUz2N876BcoC3RXbFAGiS3ZifdVLULdtvyNiJ0l41I29ZjjBhXEsr0ZJpzYwCq\\\\\\\/\\\\\\\/yjt+4QwYT7SLQ++QOl5mzX878riRHH+4Ei0SY3AeV8pwI2Unqcng3j8pjG0iUY+ofVBfJ2d7jxHAztV7HT6JRx90MlmvFCJ2RT2p2ba3ftO7\\\\\\\/ISVs5O\\\\\\\/tojU2BJYEn77K8uLYtHysS64LjcMuCUTPTw\\\\\\\/8KMSLTiM\\\\\\\/evOzXhrKrojVNDDc1J+ju8R4jmMoUvAnFhl+qyuEoSiOQCwUk1Jwfwd3iJ0KubOh6BXnqCLlvbYxiI\\\\\\\/idtZvSK38g3fkPvIBZTqO1FJIDCdiLQ0KdP4r9D3wuqOttQqrdPtXQaD7UIV+OiMmR7Abnbp1lHDU6USLV0Xk2AeRggkMX1IkEPjNCXd9s22oqiYU3z2bcUgx0vgjfLOhGORWzYcdZswlyYkYmxkvO1tJfhkWIEPOfJU8pMbYc02OIe5vzkXwPAO9RWC05136RxAw63XC6K4S96TVRBTV9wGue7x1TUi5DU76mERjgIC4e\\\\\\\/c8tHIJFcfmL2Bt+xJsc83cr7FQN0sUsXNxEzYwcQOInMQkPgXB5G36vdkZRH7KG30W9o4LC34vtCzpPAvCL26HeQcPf5tyx5arEnXtb30kP5vOX3qRJHGC+xCADHizz4VJ0XsTuhWi4MF0QrroZGCF3tRCRXyFg6tcN2NYvDR7Sy9q1jm5kfc7evI1km\\\\\\\/1sSwAnPMrJiny3leD+C9rbyHAA7bZQh\\\\\\\/d0G6ipy3Y2sy+hQ4PwPiP82FpLg25rb96e3EJALV5SbVThQakwS+xoUzJDR8nEwd1ua1Ejd9HciqqSEjECkfqSIe6yCqsuBKV4iBoVePwiK+1bAhNtgbXk1KMHlFVbMD2GUj+2+8R2s4ddUVKw2iesgSghLx3GjE9+PLQ7sRcQtawEibUz\\\\\\\/W0QFKILwA5HQ7am6VRcMr+98nOFkeoBKEKaqmGs+EXBJPJlzxfXeCzGYuSXwVfRATmHgdiNhcCk\\\\\\\/6HVY\\\\\\\/JCpdHJuYEL9M9VrmCA8k4b4EehSgl7PUvRT\\\\\\\/PjcMFfVZi3zOxeG6q61NbunLLEsDM9mQhIB5WxeZ8s5VDtZdCSYHPwsaa5\\\\\\\/DjjT+kfwR1Ka7AJO+2l0RqIt2tL5bPGGAMpgyUQj4EEtANl+wGGDb44TjWEwr+PJYKqvWDVzXDMO7UGFMkuad6XFb8wh5XU675Q1YgnVtKYlhlNERbNktB9wfu8kC\\\\\\\/cv6ULY6NUXj0unvGoQFpzWQqyCiWekDOLCfgnzuTMxuPdIN\\\\\\\/8aUa89oadCd4IqeSQT1QUpgfKHMJZ0RTpMe+SVe0sOpXnRp4X3FaCgw15EeNwT7eKwxfUvtvGHo1UvaNysv9NsxSZpBuKlEtCOfbp27vJpSTNt0uI8c2MYvkpVTosWJkjRviM30Hj+LUCRLlshhirwvZlmYyI4I2gHh2qDOtVgkdsCd0OvBOBb4g0RMavhGxVNI6iflLjUAVw3BhTS5RVBLu7\\\\\\\/j2dwzaalVsGzBaGtgosC97InG+Jrx0trV6kCPQCt0S7EfHih1pr0ndPutd3MSuoSoqMc9QFaOKdoV77fvcLgIks70c1rqGW1mICEHQH7jWMnKUqlErvQxsT7d673zKold0iXxiJYG38cyAgbPi6iiTeLOLRSKGyGmOk1rg0ZJeSR4ZIRPuFy4FrCVmRJNqVY6r6FmGzoaPfky5Y0UBvRRoask9NI7eOPzeIa2awSHxMsiOY5M7S4mNbf\\\\\\\/u3ptUatj+xuUN9Sh8k6ADqwK00H3vvhIUGjnt0zOBAzqHM7ZymBX0QePKGoWY7qq805qCXlEr2e0O2B85ODswJAJnvUpr2AlgIa+uWbem5ADrJQNPh1W9aWcj1zy54+9PAWaqQjMsHhR+SJie8Iw3E03Z1Fu5tv\\\\\\\/BQbaK8haZhHMI13NbL616ZRLKwLCN2MN0F+Y34zR1eanSBMxiY1c61wdN1G0POswVpcBWn1zHngy3uhFGarm4Rt84tKCdm6qpK5Lg4TRatAhhsfX+H2WyurYK00WK+Y4TlTfK5EEMMSCedO08atHI2yUaS0EYr7HfckhxTnzgI6kLD86StefOsYEl13pMhv8W7J9nI21t2i4jJLkzzuTCPzuut2eBrOi4LQGdfWAFKTYlKslkNg32F0w1iT19E6JFV6neXDYJIUHeUEPDoT+z8Xhhuaj4faSofPdOE1l4ZooIMELC1IlnIE5nEZA0f4Inbg4cAC1698nNcixn3GrmeZ4XATSM3e567lDXkfy8CI38zYLAMgHl2eoAfov+nebfOHUo7Xc9\\\\\\\/iGg0Lsmy5y2lBI82w1InFJwGIDkhu0NVcE3GBD7OgfXeSUAfPBzQKBTlOYctGHnxyzMVdI+DJQRe9TbebK+rRMAaV2kOrBxg6EZivT2\\\\\\\/ecOEEzLlEDd86dOcyD18FhrMPBmQnUBZGD1KQo2sazulUUGFWoCj0oESRZ1LbbEBH9QPaEAmrtRpBFOppiSo+tulSPNNXI+Hs2vitoatsOLyEInFU3+stVZZH4VVabjrMjNoRf4ftuh2w+nulkWxyKC4a7\\\\\\\/ce6KAK4nNxYqA7oWA3shxF0XWvmKAK\\\\\\\/LAwhXBehD43GrRkb0V\\\\\\\/zvqtGlT+xllAEOLqEX2MeXkD07P0Y0W46usWuLRizonloyeV\\\\\\\/8rBWNYIUkbyizTS\\\\\\\/RZoAhr+lyjSEu12LmH6N3mKzUx8IhUNhilVNRDbGWb0+G3j+RdqmRavQLHElLCxvVeRDiQWKP\\\\\\\/stJQkAhkPiDDSX9SlpGoKyufRr3v\\\\\\\/xBLai1IkcMFr0XiozXO5+MZK8uWKrpVpoBaDoeghWFbx3s2IpnnHid2Um5pegpigsqd4HCl2HoQYgmyiMwIx6fVvSdRhiPLp\\\\\\\/PFxvoejN1KpulrA5t+dsWKtrnuMUW8Yr0Hm4DHqBQoJ6FfeBzjfkytRAKSF6Is7QYx4hmI2L22bqBqhx6pp9S87qrOUc0A6QXmg7EYXllml6pgUjoeGKjEmXwgX0cUL6vMJ3CYCrdXgg2fWFqLk1JM9FO4rh8wjsbBYfwguB8Z+xwp44I\\\\\\\/nQt4C\\\\\\\/1uWytNhd0F7Ev\\\\\\\/stS\\\\\\\/vYy0ldFL+\\\\\\\/+FTdL30myEpN5wcKKIPnfaLkOJNdoCOMQsmKdjKX1RycYfS6fBatSodU\\\\\\\/9Dbt0ZYf2D4T4950G8TKl0iAZQGV6JaZUjV7eEIg3XBeuUpXS1Xy7x2b3v3vL150POpSyevp5D9pnFJGqxmECZRB4GKHR0wTU2G6kcUMqfs3POfHKxm4pCTmVh24yAukDpks+H63ZKyGTZJUtv6W3RTez4krcIg37LR2S\\\\\\\/P8gGZYgnrLgPb\\\\\\\/W67z29t9mUrhERCLxFq8UQj9Z8995v87inCzWCg1CYriQXxawVxOWNBHSwsGMYTjK0GR41yYyYIPgORhun5cnBMCsMK8BE4tkMMpivsH+RU8xCS4dKmRAEvV19J0ZfuqdCkwBViHTN+GbI0BRVDZAEYD10SVQW\\\\\\\/zOcYLT2fcY2KIwLtYgxZOnxMRHkOtNHczVS+8Ddvt\\\\\\\/9L1UjRQPaT79OfXlTdfCvx8DOA7WjIAElJuVQJCXNXHDqPb3iEd0d+j\\\\\\\/4Hwy\\\\\\\/bmLonrlSEhp43y2BfyahHCP44hwe35zHvxGk+mJZtxaPzNx4C+rupyLrnFRRAujcOuk544ZPiPm6xMEPF4BRBFNHZeVoob0vY4CnPHU8OFu7FJugMTJE2olqSaIHJm+hxinNjT2PrPdap41fcJbS+xlLXFs\\\\\\\/CzLN6huIl6CFaaWAJuSjjiEon0rWY6Ick4xY5Rw4J64IsQqlhxYXZIo8eX0bOKhHUIr56ta8WZJ+1FdJvmY8+Jc9NB8ZCY1PnaGrRm4QZWQM0hLCKarvP9TcsqtpyM0UsDj7NAVSoOtBA2SHmKgk+n9gHBKsPoJEZec3Q8iLWPnn0InbeQjugDNCg5UBt2dOsL9WYizHVUwOuFnW98z3q5\\\\\\\/wWUeWnhOuRoEKbZ9DFJH9UiWRvXedR7R2L4lnuATX4CQBdjyJIctGsIYAGXzrvNboixCRaSgNSLqUwp1lvtLb0xT5XLwEh9OBTfrPJLENyxwuHFyMoVwClSmW7ImvoyrSWK2CprlNElGyjah2ZNTHx7WjRgArp7IJP9Wpe3HzTdQJO7wNKj118lQfPD4J2q\\\\\\\/AAD\\\\\\\/WG85Is6f8umgb\\\\\\\/fKYmkJVPd6FG7zQKAvspYVBAycQpyr6QppqVxYKkTe5PPs3oMNwi68\\\\\\\/\\\\\\\/A50ussbE7nuxMFsLTb\\\\\\\/QNaWuRd+CFL19cgYoIyVO5ANpHzwBJnEot4xKXY2pvNSYq+V47InSnJiEDVttJCDXGha\\\\\\\/6X2kZCaOMKR\\\\\\\/nEohWbO8H4bExwXNRwg3J5OaARH61\\\\\\\/G8mGIVsSUUgLvuNi+UpH2Kyv+P9ExNQJaoY46jXd1OuS9bwzZv46z6CMhw8\\\\\\\/LbIhvUFsP7KpuTHMPg82vNGmT2rUh9QE2JZdCvm1Bo01rFXRe0\\\\\\\/3XuB9\\\\\\\/PL5bbhQDgOcY0fPd7izLf37h7E9gvXjGuMZIeZ3t6Pz1SNn8dyE1\\\\\\\/WhzJUGs9vLJwIbEfC0WMjJIr34YK+c7ZSsLhehNwUQIsQGNS0zUQG6qMMIempOQaCq2zWtya392hBeAwwF5YnMU2DfWQZiGJOWLLLSP2bCUWM0aQ\\\\\\\/qtYbudKk0g3KZPH1Qc1r4oCM+isLnUZH2HCwjohNVSHtRF49LHUBeRvPCCMz0CSKyKAvdp5xj\\\\\\\/Zu9FSLkZ90uPS1FI+i\\\\\\\/3Td0rEMrkBIamtkZkLjQGcPyhFa2GBOQrPoUghy\\\\\\\/qm+ZvLep\\\\\\\/M4XIoUqTrWWVTHMxehPrdS3ME4X0XkrUYFZIG0n8aOz2c4Z4d2wlXS6+ZYUyIm76\\\\\\\/7fRFa4iNe5irSly1Ggq9OsMCbWE11JfnPzOCMM+bnYAD9wiuhR9WyTgNXqv3MGCBbPzFMnr2yXNGk79dv7hxvtZusCu66zRg0ToA0xLgkN9mczYwvy6v+uux5zz71RD2HW8s1wyFC0Fh9cyFUlx0uM5HjJBxZoBtKgE4VQKvzJJ9R6nMIlilO2NZclLpDUQxz0KpMYy87oZYGR5X6+Jkhufw+VxCppTPeV6Z9eie1UYiuhWoM1J64f9bNBfJi8RsB0xlNMqEiA2Y+euWnhQ443HRlfEI5+UAQwrtch2S3FlcYCqdFrm2xVSfqeDFV2ivp1kMnz1Vl2t5cJzeuNoM1+cgVcC4ojTGlRNH5CXHU4aeesSOV0RgW39CAF+kfoET\\\\\\\/WZf3WFZ43nrKPFxLTI3x9hwM0y9z25+Ft0YRf\\\\\\\/5I8igK8hfJbUcHsE3pKgdc\\\\\\\/BJSSYdLTK+xKpxbGRjhug8j2kTvN2VWHzOtDZnesaQqg31xgrc3v7wRwWYR2mJF+p\\\\\\\/qAbTTYiokMY2yjY36x22a+X8\\\\\\\/gEhT4oc0eiaZU7EjhL7JWbSEksW\\\\\\\/licHlt8ijypoI25wN+nCWT\\\\\\\/D9th\\\\\\\/0CFvXg3+CoeRAR3eUDYSoTSiRuQSwBUq19aDyMR3NjsmhF\\\\\\\/V2dVNx5SdBmDIB\\\\\\\/onX7ad4TCsG4DcgqslYeGV3sSkHDQSWV12YrbCoxbvlc50OApwgnfvYjoKiiOZ5DMivOYU13Wjq9EmEBnikbzcTkwhs20TyFqfLLiOfCdPsSeVji\\\\\\\/ZEdnObGFZAdlvnmjWEMn+rZv9gltr8P1tx\\\\\\\/8kNKjbBbZJEc4insmFPqAb78ycmSk4L1qZPs4rCnE8QCXgDt3Bj2HZtQw5hvnvT6c0cswPA5jGljO0yMiUCBUJgKGcff81OUHmco\\\\\\\/7LnEEVEv0GdPb9c4NYKGUZXO7ezyB1OisBgiQoATT0nlysE594mpgTQqpYNj0gXtpb5wEmdyeR4JIk8hnt8zky8LPAiQOILZ08xCtHKlhX7PGPrPNE6RLMHxHM7qviQhxh7mx196JHccWu\\\\\\\/rQZPcAzqNH2yAO7ucglTRtWfEuoE\\\\\\\/bBZZ78gMfIhC+yqR1PBryh7fPSySenx6f6O893RUGK07mXssVpgQ+n1uJpkvHf4AZSvGS672OkPfo+i5xpTave7P1L9gwQxixSFFALn4ba+JMEFhBydkb3v+4xxTswB\\\\\\\/khPjD+96wqfrb9+yu4T++ONXlmWS98dk\\\\\\\/9HstmXcnzdH+Qypnyt35hC9TuBRabi\\\\\\\/xOk095FmlDhiYX\\\\\\\/rJtptztQGdiJ25UZhEMi4fr9oeQDdp3urG12zWlAxQ2tJ+ftBwVimQB2Ek4BodF8ZE9KldeaQ+XazmjjICdg5kMHxDLTA9U\\\\\\\/zBfHkt\\\\\\\/KmuoxUybK0FrJDobHa2B2Bo5ccCF\\\\\\\/7ABqRCVqrQCs2xphL91Y3Eszlh5uu8Ijjlc6ELy1\\\\\\\/8uyx9bHlRnIL28OZ8Pw6Is\\\\\\\/mmXi\\\\\\\/jA\\\\\\\/T7\\\\\\\/ScTly8+VHIIoTiDoOqRWNLkXgM1WoshTBIe+VIm7Gu1z++vIZOMuIbxT1klUFTCIREmxoAsO2vCmPbUmwre5woxiR7cXLM6gp\\\\\\\/\\\\\\\/1bNTA1eiMogfLg9HkL8GZvsaRJ+mdMt9TbxuRVdBlQ0GfMqAxQ8QQ+3CqBZC52U2Rjc9ZMV8xQvs5urT2cM4+LnfferNCFc6eGMX7yZvd4eoxkqwCXbtx2WHjNuimt06D0sqdz0zqHdS7PXeTTsNjKgKhAeg+M2VZenv\\\\\\\/arzIKKEfrFyJhs2ShGZlf9AOoyp6XI2eDimR0chsh\\\\\\\/3Loffj09O\\\\\\\/ym5SmKqH9EK30J+nDBYJcydFV696B5eSfRGkdcqtukam2I7lALitL3QzcqObtGAh7VQ\\\\\\\/\\\\\\\/2xCGdMmyWuYj4oBjuSV0vtowUSVEfGNPXnmc23zDj\\\\\\\/kBZuCFYq0d\\\\\\\/jICVGg+T8oixZmrPb9beM0oIm308lFyDWMqNIHNB1YDYq0jl+Dhs3ODuneMgleLdIrlhFABaVysz6cE5CDWx7VYvGc\\\\\\\/AGo244HKrt6uqjxDHE0sDDVdB31OGOoRxu1tdQfNYf6Vsj1cWxWBx+fCHBBoKVORRzbBtTvEWZTr5wrikp6C946ctK3GCmGG8R02QB7scamZzKvRYWz9eS\\\\\\\/kxIwYJKw2Qy81v6pGCimWttbdw+yVaCF5VB4BsluzP9LgleYCPG+brS+YXh0Zr71n4PG0mVGNgC\\\\\\\/Sj8W8girNwTyFmHI4yfvI29\\\\\\\/XALrdxXXyCL3BGee7gPr1lsnk5oAsHay5Gw0zety\\\\\\\/6zi3t+AGJhqG0A+doaZl4bumG5dKlIcs9FJSmGNsPSEmqe0aRGoWMsjCRmt\\\\\\\/Za9aMolWzQFQPa2bVc\\\\\\\/d3Obp9YHncepAR2Urb0MsYWppe9+X7WlvCOUifPXV\\\\\\\/X\\\\\\\/zcLrsvHlWUV0EfFXrXXwy0Tq14pwGKiVtjiWSJgZmuMaaPtrgRaHphGDvspLolxo7kJjP4tP1djCsnNPSRDoNxPYX5sVMnXIpImzF3mKXbEiCMImFT20f0JrgmeRLI4uDjozwfEmC8VHj5kZMTHLHcS19JhzEsJl166Wn0wx0\\\\\\\/IclxihKg0QK5UUnb700HSV6yDjifeVn4rOLxZueCJrOjlemjRU1xznhSP7y6kGONMr8+YMFOBzQ+QNurXDikEhTJWgbL0Ye\\\\\\\/ENyslrkuRENLgP64Pt59UAqg1\\\\\\\/pTHx5FFC6XCorXPoCP+pQtnWdGAl7PHc3NcGaz35tsiU4y4OnYrri+6EcOmiri8SGUX0yHtxgiTV5L8fOVagWqqDgqU6tT+zUctGOv2QacBiekAH9CwM4QJotOED2wSN4KaLEEI4lJIZXUV941JgNwkywalAHi+T\\\\\\\/4It88I6oNL4szEZpbabZboyPeOQd6CJDaR0fsiFFx6bAWA5Pwv09Pl1tL31URbphETjdBxSQnc5sl7TXIOWkXrz1u3gSKUT7YDRFv4L4F88o30FaLwVp\\\\\\\/47P4ZZ4Lvf1NBEF2DeQOybXzvC9ABQ7tJybmJIsEIY8VzfXbJ1v9vck2wlddG7np1+PvxD4qg30TSlVeDbOqcQrOR9AQvadb4nubbot6fApNquA3F1ITrjF8gPDEs6sRsPjVCX1G4AVWK7ZWz0ufbE+clgzjq5TMTwDAXoBg6929T2HJmInrzSHt8SVRoibQvJQEtZQXme1KA8A8h5DBWAW0w6M2XoO9OEkqRMoWJH6OhgY4zu5pF6YOWno99XzqyoLbnuGlbrgk\\\\\\\/d4oMQmQCP8WHmSBK31O4eYmo+C+RM+M992Dj2mqlvA5l\\\\\\\/9LH\\\\\\\/BiicHxC9mXp2elgcZ0iVWmFSSFaBW5LmNLpbvYUwAvn3egTaZL6DDNIfRrU6D6Ub9+wiUIMkUDQzZ1XFI5o+tARhjMWHuL9OsaQf1+0Ml\\\\\\\/NGyufyM1uXnFxUfwMoxiRZ58HeerzpeJEkLkrMfZoZkR\\\\\\\/Jn+hIvlaqnOTx\\\\\\\/SnVhG29F0xD9xSqN0JedDDr4xY4pC8xzVruw5qCX5fBSVwsnPtJQa9YjU+s\\\\\\\/LbgmE\\\\\\\/Jl6OgxhjvXov\\\\\\\/n\\\\\\\/mQzErQxgVLYqqxv8\\\\\\\/Wa9zzWi23XUctCYO\\\\\\\/Y4qXjWMqHU6OfNruQxQgvYiW59qvXg4bkGXdN+8mwjylVjjF2Xf17lo0bFdaNiEGJvZ7C\\\\\\\/dVYXPA32oi1zafNoGIeVrVWA1408dsKuTrvFV+oGVelk1HiM15rY5n83PVJd12cF9M0pVyrjqgv+eIfsIXBE8U3cSP9iEi1AudPNG0R9s1p+6Jq\\\\\\\/2LNLoNgX3gf3S3x47931e5m9uNisbIG+nHWqW9t2FZphkzDXYzKCuiAoI1LBcHU\\\\\\\/l1c5GxjfjuCahoQcpPAGaYWYKJ2P7V7KlAi3UgXN4uaZrqtDnzfTkqi523HVLIa94oeGWWLnE95dS704MJqtTGXLsQzWxrc7HXICxfZSLVk37e6yoLTIeSiJUMvvPA0g7PzXap6Ny9jZKp2TmT5C6yH8VstsyYwABbQRwTW6JdEioUule0Gm0B5EC5zAuoETTYDqIuTqlmaKYT++tDLZKivi+4azwkurZJ83HLyTBVAUdHUT4exFelJGONU\\\\\\\/8Akm4XrVDxewHlVwMyRkVVcZRcylA4WftBhe349BTcWsaMzlvrzt3Su+TH4EgVPKyhf1bNV4iDwfMwk\\\\\\\/4\\\\\\\/mWQHXeqThJRr\\\\\\\/1PLUOGklUXyvNGuQlCao1awFxr3nSIegQWei31nEBLcinS5xqWEPGitbOpODzD81Z65TPOuJxxHQnP9RN+bpEXOJgEyWi3FqOuXgywmtyKm+RddRkYrutrWXWcaOTztPRMrXwF7e8DodW7LdOf\\\\\\\/8hhN6ReBc\\\\\\\/4sAyALHo1x1naXVULt8BDdB+V4\\\\\\\/NQNTrEdGbiX37WfrnfVrEr+FRsrqg8UBOozCdQEU7OFSqbZ2EfiUXJuvylaC0PFcOZeX75mv2NZIO6DdQoSt+o0d2VUabT4JPtbqiy0\\\\\\\/3oGgE33Nk\\\\\\\/1+oB4kBnKYBmbuiZ7mD\\\\\\\/puRwpdOIQ3k0U4yAidkRbC5cKmuIL3TrV9BYlvjjddef4GBSLYqhvREDFCVTatVn2mrEFCZV3w\\\\\\\/SZCSOpQz+M26cTMI+rX6+dethk6gdnRwS4ip2ekKRYxiWKJfzq5L1zdT6JFgEjjusCAz6pDEPlxiYCPcpFJfa2ldwHUwuad\\\\\\\/7CAsMeUs+h9CB\\\\\\\/g4tB1LpGyovSoAEX7R9PyP0J5LsLqB7EKq3f2Wp8lsWZ2L3S1slhLT\\\\\\\/Nv3EKHeEyoCv7B+\\\\\\\/xM6XfBy1L7jRU+Q0zJBImvNi2964dQ7xeNasPa046TBStGa3OhbWtYFfTkTB2pjbKg0LdTGTY1nRKaWoO2iYUeBBAv+ivoxtAWUtoAy7XhBcUdd2a3rTdQx56LMYreO3Y7lLmc3+0ClM9rppBAzFCf\\\\\\\/gICF7NDka5GduHoLYR\\\\\\\/ovZwguSDY4+F\\\\\\\/AcjHQpg\\\\\\\/wp1w8xI0FAVqjEFb\\\\\\\/tP7a6+0vMzhkqlggQ+xR3JxdNGqsWfmhNQBA18q+Gcyz3Mb\\\\\\\/PkFPcnMMWPfCd7nYFWybQuTZIbK608R8QSWKFFTR+Gi2bdbykwl8d+YMg3jcTP1UVhvhsldFdCtvTQSHPLtOnCvqq3uCNqK2rcLVm+alqhilk1vtCGynmlyrzf4K9wjlVshhwmr31WFmwpmZfNt1m9tXUvvAvJXCTRmCaDUFSCubah3TYVixSO\\\\\\\/Grob860WHu3THSma9mpjHIP8fHj6xCCP1NpR2G3Gvw4EtXEqpAcGvsMixKL2KVV9GOCO\\\\\\\/KCda9y5QFPsSLLHx5ayvU4sPSP\\\\\\\/I\\\\\\\/3xVJV+\\\\\\\/4R2JG3p3zn+ReWTDySwGoTrUi8gpbRy\\\\\\\/qMsExHHDic7hKjKVFR48vd33PvtyOqOv\\\\\\\/AOaZOuVItyTbpHHpQyDfmM9c2I3mJ8ETNvEMhAZLY6\\\\\\\/isly2\\\\\\\/2uMaG9V0BsRy6JiEkL5OGIWTb3liXV0eIjSYTmYQhYQ+CDX9ldJGwTBIoJlkrLwylFmSpNcFve9PsrjDy6v+BTKx7HafDnCxyVFkoqTZubbYL0d0t5DyxPK1DOFAAzbz\\\\\\\/f93Y3oHKQ306FAgqtYgJbvYQTtj6fXVYNmTSweyw4izim6Mw5\\\\\\\/wtpxaL7kdJvAw2bRpGisSaV4mP8zOaKjlKhzB5VD8chZEBMr+zP\\\\\\\/RbKgb6zy2j5NLQgFFHzrmr\\\\\\\/QYiA7sSLFm7geQ8QBv1OSNYtgu6wEiyprgKOLQ72UERX1MrvMStC8YQP1yRDuNBfOwpYqK5166DEaOIMuQyWx4x5qyMLZHEg90y20nQE+OlZKmS50gzHIbflbFCMsdkvMIZfN4g8F9Pac9n+P3eG89nm0BOpmR6mTjj+XHI0oxKrcenVR56bplFOzNOHqZgq18jqoSzeJcM084K\\\\\\\/nQNbY0YbTzQjHHkpu0IdMJDqZDvfXgbIayfGScufqCbj4F92N2NrSjnoAMeZ0QXQ6BPNeFT+U7GEmCfu0KzuXGIj9xH6F4ZEQn5JpXYLg42F5irXQi3eFeJtpJd1HZUtRAHeCgLB7XEd2qoxcA5jHHNkUm8TrdY0DPpE1TDWlV6H+NXsLwd63xOYNA4AOg5nq6NhFGmgd4wNwqLftGJRH900xS8VYomyeDMOf\\\\\\\/ha7e+4m3dSqBOcXPUwUbVuzaQAPqEPFTAljffod4CLc8L7mg5\\\\\\\/ZEC\\\\\\\/qvbAPmAJWVCnzhZ3tcx35esL\\\\\\\/\\\\\\\/4TgcrbqsHnhzwsLW+oQqto8B4zLOMQFOjODcRf1AK65LRcsxIQ22efI9aj8bBN2DF5QMqWnwErqWaXBBDbIPgYR\\\\\\\/helnUmg5r\\\\\\\/\\\\\\\/QYeyrhwDrFhhw0BMoqOPOOFTIEshcnCJPUgHGxb89csX2pjVH+GChTR7cjhmDLk8VQoen\\\\\\\/fCrPZka6GDKLKXjqAMH7QoJDf38nhnQlC\\\\\\\/J4U2knHwzzZc9HP\\\\\\\/J1y8e0TKiKPRcDlRToJWWnsw\\\\\\\/PwmUd1cnpw9TgUxOitiJxwhlXb5sJU5Wo1ZYG+MdAPpZ9aSyrOLOPmqLUBMwn6O82rgnKs1h01hiWDWdHbXQ4SLCPRr0bcEHEtP4BZIAx7fTdHPgZZscZnKoGsHGOqM72x7MZ9y9W3t7uwss3K0KZaMezOuc5jNcUQp6ydgVsDXkD34dr6TBdnbOk9148Ks+A\\\\\\\/4vY7NFouIFu0oHHxsmQF8CGNyO\\\\\\\/U7y+0EBiLC+QE3ZEX2Yl+HoOZwWX2nWrntJPSU3CQvXr4rSfOu+RDv\\\\\\\/KEExtpUxDmu0PgLAnzhvwr6wH1U+b0AsRtSP+JB5ewHRjGOA3lbXmdRlY3Hp5MDh0ev+OXUwm4hMhPwOoLpsIOFQTMvYr4JHer\\\\\\\/8NxRbmPmGgHbAtVLuzZ1UEcIGZ7Y\\\\\\\/EyezmmroKkPKZb7hsHRySd6Q5gRtbTX30piE6yr4wnNe4tp\\\\\\\/q51MNWoU096nJbdKNiU4+vFUXrzCjzmYiOelEnOqg+06Tj46ch6JBOziiF46rlI6m2dQ0IoCoI3vRLeV9KGDz8X4iSAmF\\\\\\\/6EK7exG7FvyjCxg8rrpY6IysnZaXc1SCeFVZHzVVeN2+Lv9DTdxvSVyrtNZKh\\\\\\\/YfieGKjQup8viA3TwC5+42yV3QlhFr0llGNvhcnBG\\\\\\\/uud5RV7q8LDDkdZbiBXWVYurMIX+CiJ7fvuHBXptMmzoEH6JLV2EWJMLGzR6XlmQqAq3iMF0lawwJGM75\\\\\\\/mDFaxYwtlRAHDZA9Gc4kYX0Pgmv2J6TaOgWY0ppMHuXtfCpFB71tKSXct2LG6mM0gsBtdM+B+3T0cdAuVTDUNINXVbSs2Q38xfSNyQGJQKu52FdHj2eFsYRAxBbk1CwY093aJ4gOx8hdcWT2y5OrzbjXmtRdvwA1UiIZRpHeSvyNBlBi++h5nC9aBQiFI8zN9t61lAkqQwx2\\\\\\\/aiDwhw2JzsoMy6dS5LgVYbHQyLGY12YbH+efE9Msa6Y95BeksYnP6gs9HoTjMsGMeBbovnBiggebSamyFOu2zNoT06g7jm1oMOg9p4FG6ZcmkbSZIatgvCtAENTOs3MD8h7KIaIIuAubkdoj3uaL6NOJX8FsUqrOUZ7m1H+oAuDo9c0WZY1MbmKyErJuvLvwRHHvfhGkF0lVXqgcF66k91KpJuP2XIycgjY0K7AyzlS4p4\\\\\\\/aHlx\\\\\\\/Rzr+YBDv+p0g7r7oD8oyEfhh5FLnIRk8LHJ+SuVpAis8Pi230j4q\\\\\\\/eZif1wkIdbSCyzUBLIrUQ\\\\\\\/xEuWl0ZliFUAff9NuSUYxtSHLRyNK7u50xtkdSTHnKgkB2OquFuuxg542Ptt6Sg32iHMs4ydZ5ZFFSnfAla9Wbn4SdkoxgvO+kLxAUBDB9iQMDjqvgkhJQlRG5g9orcx6M\\\\\\\/XoRj8CvSzUy6EFFA9qV8YmE55ugAKFici5uYFR8RpFDOv\\\\\\\/5Hyuj\\\\\\\/km9ojcQfJUuriKAqdRdkLEE1MuoKqW7fswS20v9vYHEBgEscYNQJmAw6aw27KbGic+wY\\\\\\\/GbKCVKZtXd8xUxOVEFbu82qmXwnR1mrTjTmrJzuqtkhB1elutSBf8UxXbWCDtwmPbK0J4Dyd1JXNG588B0\\\\\\\/XZ8TT+0WzEalwJvIPn+IdcCy5aE5oc4HS0gEoNAnkN9\\\\\\\/s2mAB6vrHlpAzzObdmsNwxy4AzVLNEGUWN7PUHS9ZyYguAi48w7ZAIemhstBkaGjwAjH1bAAJQJFA7+I4o6FtoY\\\\\\\/42tbc4\\\\\\\/LMP7fmCQjUZuoerLAXy7RLx+2s7lw9FqWZYWoFI4mUeGZRRW5rmPWvs6eo8ttgIuGZQxjNJnNfwwmC7MknXIhcQCSDzdLvzEx05qxpMVtWoAFizM0Dclg435VxpZ6c\\\\\\\/cXXv2tOjy3fQvuavYTDp4P4G85bFMuhNH\\\\\\\/l3\\\\\\\/6UAGu4v7Sp4PYRoCVxqnnx+mN9WgcRTWtcu3ZljDrDyyJiKl\\\\\\\/Dq9icQhsYhtI2I+eU+g4pM+2E5WH5sxGIjfNqjne2Pv\\\\\\\/hnuIClyG\\\\\\\/xWkeZ\\\\\\\/Xxwyl\\\\\\\/FI8msSFSAj23b0dzxIlmLJHPBxp4dgVrYT0K5VXlh8dJCmlGfGGTRECNva5hyIPsS90Y3YDZhEEyf2kutK+VuaCxRcwj\\\\\\\/i3ItCBuBUKh56+yfnucwhRyJAImoxxS+1ZNxOOkPgXtCPT4zuLdu5dlOYfMEgwDymRVr3I9S+CfI8cSYTI7Uk2mQGxv\\\\\\\/tS+DqhJoMPJfJugLweTojDyGRGBcwzqtwLDrucZV8jtU83Vvgf\\\\\\\/dhFhjuKWf2uZ8DlL3ns7qRA1LYAv+zGdMFIGwx6b+2Ijn3U7LLDK+9GvZZlNl6GV6K\\\\\\\/cF5\\\\\\\/FR9aYteAuooSBlp8lEvdYQsZ75ofXJl7aAc\\\\\\\/R1xnneOXJyyepIUBUQdil4kbryL4PNXlT0UbzY0PNjR971FGYwx78tjJbRWzfDYV9OocOj0vm9jTVObu0be5bcYtmez3rUyzB31k\\\\\\\/3MwZ7XYo\\\\\\\/8z54GQgMlA3ZLMl4SUlR1D1gJ2WB2GaRpQzE5ENotlg21Nmr+8P6PYFdiWRyHgRv1kyzdff3veufGv3lpOcwlPHHcbnke8T8c2XHQ8pmnv4J5cwoiUEL8Saxq2rg+tS\\\\\\\/x1j6qzfZY5UsrwO0IeongAGu1GvIgkth5DWLsXVJBUwNqFYgg090blugdKUah+xNtk3ObMRqjU+DK7m07Ez8SUmQQkthS5nwc9fZcRkrG0Er5tauVX63tZNzpsiKafASnrkV9gEAdebIsSMCpLp1Gqf4QaCHlV1Ku62emG34gNiVLUTFCEM546ts3hTc3Y9KWZUAUOypeY5VUIHfIrea\\\\\\\/BMnKa71CRthwnW2DyoJQNhkgyX6UQMioBTQT0QbveEs5PDxMe347inSwH6incmSfpN5N+eeM2G\\\\\\\/OafbbUu5dNJQFltLXkZBdI+hEbF3g\\\\\\\/fEHAa365BHJ46EvcRHUhpVikSz6\\\\\\\/76Fl66QJS32ePmE0R9oxnHp7IUyju3OPUA\\\\\\\/MtAQMhkxfI7vmjNPImTzOhdNW8ImnNxMZAfvWhHjdApyREVzEOyxGA\\\\\\\/naP1fc\\\\\\\/me0KmSJsoOR02Ixtc0HSmVt\\\\\\\/DBHc5KbBJINr\\\\\\\/TxlK+nGwXlbvU44EVmHjkbOJk3GqeXeNbb7L\\\\\\\/zFMVffKIH6j7pF21IYX1RGXzF9TONqEshBSEPwjXvr4GW4IniuQ33cUDkBSsxr9NVEA11Udpz7Cgln40YLZ\\\\\\\/ej7SLJEB59z\\\\\\\/JO0zz7PEUN1KCFOcUV9aMXYUC6bJx\\\\\\\/HLlHVpxNTrQQqIjOxay2JikIEMvj+Y1SY6V6bFMHKUUe0nfo8ZXI7g9ltJZ8C1aXylB8MTNaV4Oj+cWL\\\\\\\/mtpRAX4jw2jsk\\\\\\\/jeilwvZkcStHL8zB8Z5cL7tEj\\\\\\\/6lgrh\\\\\\\/TGhEGpBo1u\\\\\\\/GljaKdSXodG0UEuCWMmhOlZ2GwB5c3jt2sdiCs99dJ+S3Qu+e\\\\\\\/\\\\\\\/Fi3d+PAQFTIi4uK6++GcIFlg60IHIgQK+1V3E5JaEJzyOR2NITEh58p51E0nuhlQY5Agyw72OBupdhx9U\\\\\\\/UbiTQ4pLjJzE9MU0nt+f+aRNyHcmQ6WTesLtOkFHg484I0qNRjn3X4QY5hPLYRDkOacX+j+OnL0+KDinNokQdaVl9kMDAYP\\\\\\\/SgHwEO3fq1Wr8tAFKzRBoG\\\\\\\/4JLN\\\\\\\/nvaMpxyIcYH6Z62\\\\\\\/h3OQD5Nt0cpoPj3jr0tnsb8Nha57USak7CPyFa9D4g3xgeJzxIS7MRhDcW8Vy9KNFEDkpuOPMcdhTsJdXxbBYdE8cRcwWSBXnUioms\\\\\\\/XKLIFKdIzWzFMqy1U+I0kCaIct24gELcmX0pi11QH\\\\\\\/RRBlMUjycFGn9pqeM+amFbJ4xGIMjp43mn3MtNyp4\\\\\\\/H7k8831g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022v5w8SHp+fA+07cMPjU6uDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b092200fbebf4fafd686207c35656c19309b7d18c0e41eb9b500e83d4dc378ff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qpoaUGEaGaV\\\\\\\/e3Jn9WIOOttLHgbY+GGdmN0s9kiYvmW2qSJITMx\\\\\\\/k7E0xXQYmS6qVXlC\\\\\\\/ME2jFTwoBsM+UIs\\\\\\\/mzn3TaS2O2xySiD9TxTtyYoDZLe1mGEXvBMVMgGZqpVb25XF8VJKZL0U2dfYhHRVNjELdDcFOohlEgX657AbV4MhtUFdkwTtNIlrXQvEd58uI6Gchh21E3tfeqI2fd+8qLLb7XMbMXeY2lUNTFdNFCR38um\\\\\\\/3yuqR72aG+OWep7o4iZ1vF96GK51PqImGK4lCGF6ENl3s0tB0t3euJv4tAN1qiY7gwgf5avATkYo3p3yKXwDNm6olHKqltcOCWDCVWqdb2md6h1XQXuqM32O9dTXHMHmywJUvhOMDferIduPCDY2vCvcKLoXuxX0SeWiaW+NobifnklPhbxFBtfCqq88X2htl4cAWOXpc5jbCR5aXC4A3om485eG2iBAqFDS6fPCDVlOCQsYm9HyyWZYxSrUE1HfAEuN8vzBsYvUzw1XvuI+TYp\\\\\\\/icqoJiyGG\\\\\\\/9UdlFxrhORJ9cQEYb84WwKzSS0Bi5AfR\\\\\\\/JjYTffftxcrhsrVkA9tSegL6XeAofmFaxJTHaVuSrO0bfkOOdqy4EK275wB2NjEEM8GMCf94oOzsmLb1\\\\\\\/axGCV\\\\\\\/WQK66VKwrFSDiOKu0LvBqOoTdQSk\\\\\\\/QU8XNe7sVvt4+Aar\\\\\\\/9KltIr20DY+yohMqVN4RZbC4WE9PNzmsuS9poA6hlUrJVaZE1zWBaOXnmzVwYclMl20fsiSZxTiNLmHM\\\\\\\/Bp2t22oqdoOtk8hS6mJ1orOfOY\\\\\\\/d\\\\\\\/ukynFUPjj\\\\\\\/\\\\\\\/kGD9moB3QQbMXlobisCcEphpXozp\\\\\\\/gHyW3x5P40DrREv8w1cTKuZfjHzgSMjA7aQek\\\\\\\/L5LZoxu9W3J\\\\\\\/w+ljB3S\\\\\\\/kvi06Vq4Vi8CvVF5YEby1kF7y8K5bHv1Bfimw2vRKU\\\\\\\/UZGPijTeuWvv3g\\\\\\\/SnLTAL4YzcohfKfb2Lo5ONGCTBWJvHP\\\\\\\/rgHoeBCP+1WytYYwTTzZwdVjMwQ7LU0q4IECmrXvtheNA3epjQLcIHcK4EuSiU05EEqBGfHOXcFU0i8zjofbYw13K6kM5Tt6Fnz0rkja4RIGeXxQXRM8MMGf46X5saynClLDNU4Kr4J5M3P9v9kTbxceXtbr4H0ouiGyRzk6mmcf336eWl2b2FWEb2gRibwWvqbsW0LeSExuKalxNCFAxiQFXyiaAhriNEiI6uXZWI9viKalZ\\\\\\\/Pcpm737BrrwB+zgD3URBXpcBM\\\\\\\/lLGLdc72hf9OQGKOWmkGNp+e9JAS6JHxq5QDsVJlFTA+OyojrD6162\\\\\\\/hXA\\\\\\\/DTaC6rVNnJByhL316eb4YTB8uhkFn1vQMjxDGe1VoJyM7D15kd155JQY4NezZvPPfMOzjLwSDzAaWRuW7bALnlqlP2gtRqF8FKZ85v8TnRxKnbVYDKGaEhmAtaw9OCmS\\\\\\\/CaOEuYuayDePUoCvB8PxR6N\\\\\\\/2gnhBLzM4emq0T8WEEqTaR39wQ+dxtVWdrH+QLJ1xUyujBWY6QUrhUEJEP5wLJN9WRpPo6Utl6v0aK13kSC9GATomhffL2pERU+HYBmshO7oSrNgRtKZ1Yb4CthTUCI6xPAnqhurNXL\\\\\\\/c5RPLMMUIhyPEoi1KyORoYZbtAE4d\\\\\\\/+2a5ElLeyu1GPQRXdGjqFJJ\\\\\\\/q\\\\\\\/Z8W0wXIrPkqKoW1EjUMoN9cxKNNwYri8APdKFUURuPym5Yjb7a73tyVVXxVIFMM5\\\\\\\/XUan3rtzcjZKYXoNSAEiwgb82QqXUd9q1LpZ4ta5h7V\\\\\\\/YCJX6BxQgM7MblTH9M5oJhPRi6K2wEIEOGNGqD4xnnvQ3JdH8w2yd2V29fVdWQZF\\\\\\\/ohlleG1FZr7tdH\\\\\\\/xEIxqCwcjSZkMX6D\\\\\\\/X0Sw6zX\\\\\\\/P54j+UFGJ4QyvVYlGx8WobTTTJaOxDzp8C7YWlN\\\\\\\/Xu0putQZYIP1CE1Ffh7cZ+gyS7Di\\\\\\\/Uu1T\\\\\\\/keVTqfzupG8UjXLgSw4lB3f\\\\\\\/wyfg0ZLDDuJH46kMcR2DbAw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V38XnkZQeeUC7lLwBCpaqQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229322580c1ea8343a5298ce2ce13126a13fa104a70eb6a7b6ee44190bd86ac710\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492359 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/663" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-99XUsCPMY6UPtLNiIpmb9zRBf74mubLSr-tycH2lQS8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/663" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r1g7WFiTbDu9DH1239hxntdrHZfkoN7GkB7Hhg5D0Ok" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/651" title="Evaluation of INSeq To Identify Genes Essential for Pseudomonas aeruginosa PGPR2 Corn Root Colonization" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/675" title="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492359" id="node492359" data-pisa="ggg;9/3/663" data-pisa-master="ggg;g3.118.200913" data-apath="/ggg/9/3/663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lindsay V. Clark</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="2">Erik J. Sacks</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">663-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492359" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay V. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BV.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20LV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALindsay%2BV.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lindsay V. Clark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvclark@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">lvclark@illinois.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erik J. Sacks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BJ.%2BSacks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sacks%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BJ.%2BSacks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/3/663" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492359" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/663.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492359" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/663.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492359" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/3/663.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492359" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/3/663.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492359" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Low or uneven read depth is a common limitation of genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), resulting in high missing data rates, heterozygotes miscalled as homozygotes, and uncertainty of allele copy number in heterozygous polyploids. Bayesian genotype calling can mitigate these issues, but previously has only been implemented in software that requires a reference genome or uses priors that may be inappropriate for the population. Here we present several novel Bayesian algorithms that estimate genotype posterior probabilities, all of which are implemented in a new R package, polyRAD. Appropriate priors can be specified for mapping populations, populations in Hardy-Weinberg equilibrium, or structured populations, and in each case can be informed by genotypes at linked markers. The polyRAD software imports read depth from several existing pipelines, and outputs continuous or discrete numerical genotypes suitable for analyses such as genome-wide association and genomic prediction.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/next-generation-dna-sequencing" class="hw-term hw-article-keyword hw-article-keyword-next-generation-dna-sequencing" rel="nofollow">next-generation DNA sequencing</a></li><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">polyploidy</a></li><li class="kwd"><a href="/keyword/bayesian-genotype-calling" class="hw-term hw-article-keyword hw-article-keyword-bayesian-genotype-calling" rel="nofollow">Bayesian genotype calling</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single nucleotide polymorphism</a></li><li class="kwd"><a href="/keyword/genotype-imputation" class="hw-term hw-article-keyword hw-article-keyword-genotype-imputation" rel="nofollow">genotype imputation</a></li></ul><p id="p-3">Approximately 70% of vascular plant species are recent polyploids, yet genomic resources and bioinformatics tools for polyploids typically lag behind those for diploids (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Moghe and Shiu 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Renny-Byfield and Wendel 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bourke <em>et al.</em> 2018b</a>). Reduced representation DNA sequencing methods, such as genotyping-by-sequencing (GBS) and restriction site-associated DNA sequencing (RAD-seq), have made high-density genotyping considerably more accessible and affordable (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Poland and Rife 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2013</a>). However, the two most popular pipelines for processing GBS and RAD-seq data, Stacks (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Catchen <em>et al.</em> 2013</a>) and TASSEL (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Glaubitz <em>et al.</em> 2014</a>), do not output polyploid genotypes. Though pipelines for polyploids are available, each have limitations that prevent their general application. For example, the UNEAK pipeline is designed for diploidized polyploids only (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lu <em>et al.</em> 2013</a>). HaploTag is specialized for self-fertilizing polyploids (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tinker <em>et al.</em> 2016</a>). FreeBayes and GATK can output polyploid genotypes, but require a reference genome (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>). The software EBG imports read depth from other pipelines to estimate auto- or allopolyploid genotypes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blischak <em>et al.</em> 2018</a>) but requires allele frequency estimations from the parent species for allopolyploids. The R package updog estimates polyploid genotypes from read depth, modeling preferential pairing and accounting for multiple technical issues that can arise with sequencing data, and can output posterior mean genotypes reflecting genotype uncertainty (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gerard <em>et al.</em> 2018</a>), but requires excessive amounts of computational time to run. SuperMASSA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Serang <em>et al.</em> 2012</a>) and fitPoly (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Voorrips <em>et al.</em> 2011</a>) were originally designed for calling polyploid genotypes from fluorescence-based SNP assays and have been adapted for sequencing data, but fail to call genotypes when low read depth results in high variance of read depth ratios. Thus, important staple crops such as wheat, potato, sweet potato, yam, and plantain are underserved by existing genotyping software, limiting our ability to perform marker-assisted selection, while yield increases from breeding are not keeping pace with projected food demands (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ray <em>et al.</em> 2013</a>).</p><p id="p-4">We present a new R package, polyRAD, for genotype estimation from read depth in polyploids and diploids. The software polyRAD is designed on the principle originally proposed by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li (2011)</a> that it is not necessary to call genotypes with complete certainty in order to make useful inferences from sequencing data. Initially, SNP discovery is performed by other software such as TASSEL (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Glaubitz <em>et al.</em> 2014</a>) or Stacks (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Catchen <em>et al.</em> 2013</a>), with or without a reference genome, then allelic read depth is imported into polyRAD from those pipelines or the read counting software TagDigger (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clark and Sacks 2016</a>). In polyRAD, one or several ploidies can be specified, including any level of auto- and/or allopolyploidy, allowing inheritance modes to vary across the genome. Genotype probabilities are estimated by polyRAD under a Bayesian framework, where priors are based on mapping population design, Hardy-Weinberg equilibrium (HWE), or population structure, with or without linkage disequilibrium (LD) and/or self-fertilization. Multi-allelic loci (haplotypes) are allowed, and are in fact encouraged because LD within the span of one RAD tag is not informative for genotype imputation. In addition to exporting the most probable genotype for each individual and locus, continuous numerical genotypes can be exported reflecting the relative probabilities of all possible allele copy numbers, and can then be used for genome-wide association or genomic prediction in software such as GAPIT (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lipka <em>et al.</em> 2012</a>), FarmCPU (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2016b</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">The PLoS Genetics Staff, 2016</a>), TASSEL (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradbury <em>et al.</em> 2007</a>), or rrBLUP (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Endelman 2011</a>). Discrete genotypes can also be exported for polymapR (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bourke <em>et al.</em> 2018a</a>). polyRAD is the first Bayesian genotype caller to incorporate population structure and multiple inheritance modes, as well as the first with an option for mapping population designs other than F1 and F2. It is available at <a href="https://github.com/lvclark/polyRAD">https://github.com/lvclark/polyRAD</a> and <a href="https://CRAN.R-project.org/package=polyRAD">https://CRAN.R-project.org/package=polyRAD</a>.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Overview</h3><p id="p-5">polyRAD implements Bayesian genotype estimation, similar to that proposed and implemented by several other groups (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Li 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nielsen <em>et al.</em> 2011</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Korneliussen <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Maruki and Lynch 2017</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gerard <em>et al.</em> 2018</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Blischak <em>et al.</em> 2018</a>). In all polyRAD pipelines, genotype prior probabilities (<em>P</em>(<em>G<sub>i</sub></em>)) represent, for a given allele and individual, the probability that <em>i</em> is the true allele copy number, before taking allelic read depth into account. Genotype prior probabilities are specified from population parameters, and optionally from genotypes at linked markers (see Supplementary Methods).</p><p id="p-6">For a given individual and locus, consider every sequencing read to be a Bernoulli trial, where the read either matches a given allele (success) or some other allele (failure). The probability of success is: <span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-1.gif"/></noscript></span></span>where <em>c</em> is the cross-contamination rate, <em>i</em> is the allele copy number in the genotype, <em>k</em> is the ploidy, and <em>p</em> is the allele frequency in the population. The <em>c</em> parameter is important for identifying homozygotes that could otherwise be misidentified as heterozygotes. For GBS and RAD-seq data, <em>c</em> is estimated by including a negative control in library preparation, <em>i.e.</em>, of the set of ligation reactions with barcoded adapters, one that has no genomic DNA added. The sequence read depth for this blank barcode is then divided by the mean read depth of non-blank barcodes in order to estimate <em>c</em>. Our model assumes <em>c</em> to be constant across loci, under the assumption that most errors are due to contamination during library preparation. In practice we have found <em>c</em> to typically be 1/1000 (unpublished data), and expect it to be more substantial than errors arising from the sequencing technology, which will tend to produce haplotypes not found elsewhere in the data set. Therefore, although it is known that sequencing error can vary from locus to locus depending on sequence context (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nakamura <em>et al.</em> 2011</a>), polyRAD does not estimate sequence error on a per-locus basis. Rare loci with very high sequencing error rates may exhibit underestimated likelihoods of homozygosity.</p><p id="p-7"><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gerard <em>et al.</em> (2018)</a> observed overdispersion in the distribution of sequence read depth, indicating that in reality <em>π<sub>i</sub></em> varies from sample to sample. We have observed the same in our datasets, likely due to factors such as differing contamination rates among samples, restriction cut site variation, and differences in size selection among libraries. Therefore, following <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Gerard <em>et al.</em> (2018)</a>, we model allelic read depth as following a beta-binomial distribution rather than a binomial distribution. For every possible allele copy number at a given locus and individual, the following equation is used to estimate the likelihood of the observed read depth using the beta-binomial probability mass function: <span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-2.gif"/></noscript></span></span>where <em>a</em> is the number of reads for a given allele at a given locus, <em>b</em> is the number of reads for other alleles at that locus, <em>G<sub>i</sub></em> is the state in which a locus has <em>i</em> copies of a given allele, <em>B</em> is the beta function, and <em>d</em> is the overdispersion parameter. The parameter <em>d</em> is set to nine by default given our observations of overdispersion in empirical data, and can be increased to model less overdispersion and vice versa. The function <em>TestOverdispersion</em> is included in polyRAD to assist the user in determining the optimal value of <em>d</em>. Although overdispersion is likely to vary from locus to locus, polyRAD uses a single estimate in order to save computational time. The lower <em>d</em> is, the more influence genotype prior probabilities have on genotype estimates.</p><p id="p-8">From the priors and likelihoods, a posterior probability can then be estimated for each possible allele copy number for each individual and allele using Bayes’ theorem (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shiryaev 2011</a>): <span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-3.gif"/></noscript></span></span>where all terms are as previously described.</p><p id="p-9">Bayesian genotype estimation allows correction of genotyping errors in diploids and polyploids, <em>i.e.</em>, when an individual is truly heterozygous but only one allele was sequenced, or when an individual appears heterozygous due to sequencing error or contamination but is truly homozygous. It also enables estimation of allele dosage in heterozygous polyploid genotypes. Moreover, genotype posterior probabilities are more influenced by priors when read depth is low, and by genotype likelihoods derived from allelic read depth when read depth is high. When read depth is zero for a given individual and locus, genotype posterior probabilities are equal to priors, and thus missing and non-missing data are handled within one coherent paradigm. It is therefore not necessary to impute missing genotypes in a second step if the priors are sufficiently informative.</p><p id="p-10">For export to other software, as well as iteration within the polyRAD pipelines, a given allele’s posterior mean genotype (<em>pmg</em>) is a mean of the number of copies of that allele, with the posterior genotype probabilities (Equation 3) serving as weights, as in <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guan and Stephens (2008)</a>. Thus, for an individual and allele, <em>pmg</em> is calculated as: <span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/3/663/embed/mml-math-4.gif"/></noscript></span></span>where all terms are as previously described. Additional details and equations for specification of prior genotype probabilities and estimation of other parameters are provided in Supplementary Materials. A flow chart of how this Bayesian genotypic estimation is implemented into polyRAD is displayed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In brief, for mapping populations, genotype priors are specified based on parental genotypes and progeny allele frequencies, and all parameters are estimated once. For diversity panels, genotype priors are adjusted and parameters re-estimated iteratively until allele frequencies converge. Source code is available at <a href="https://github.com/lvclark/polyRAD">https://github.com/lvclark/polyRAD</a>, archived at Zenodo (doi: <a href="https://10.5281/zenodo.1143744">10.5281/zenodo.1143744</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/663/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of polyRAD algorithms for genotype estimation. Genotype posterior probabilities are estimated iteratively until allele frequencies converge, except in the case of mapping populations, where allele frequencies are only estimated once. Purple boxes indicate inputs to the pipeline (read depth, contamination rate, and optionally, genomic positions of loci). Blue boxes indicate estimated parameters (allele frequencies, genotype likelihoods and prior and posterior probabilities, linkage between alleles, and probability of sampling each allele). Green boxes indicate alternative methodologies for genotype prior probability estimation (mapping, HWE, and population structure). Priors for the HWE and population structure models can be adjusted for self-fertilization according to De Silva et al. (2005). Orange boxes indicate sample &#xD7; allele matrices indicating approximate allele copy number. Dashed arrows indicate steps that happen only once at the beginning or end of the pipeline, whereas solid arrows indicate iterative steps. Circular arrows highlight cycles of iteration. Equations 1-4 are provided in the main manuscript, and Equations 5-19 are provided in Supplemental Materials." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1284579180" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of polyRAD algorithms for genotype estimation. Genotype posterior probabilities are estimated iteratively until allele frequencies converge, except in the case of mapping populations, where allele frequencies are only estimated once. Purple boxes indicate inputs to the pipeline (read depth, contamination rate, and optionally, genomic positions of loci). Blue boxes indicate estimated parameters (allele frequencies, genotype likelihoods and prior and posterior probabilities, linkage between alleles, and probability of sampling each allele). Green boxes indicate alternative methodologies for genotype prior probability estimation (mapping, HWE, and population structure). Priors for the HWE and population structure models can be adjusted for self-fertilization according to De Silva &lt;em&gt;et al.&lt;/em&gt; (2005). Orange boxes indicate sample &#xD7; allele matrices indicating approximate allele copy number. Dashed arrows indicate steps that happen only once at the beginning or end of the pipeline, whereas solid arrows indicate iterative steps. Circular arrows highlight cycles of iteration. Equations 1-4 are provided in the main manuscript, and Equations 5-19 are provided in Supplemental Materials.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/663/F1.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/3/663/F1.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/663/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/663/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492449" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Overview of polyRAD algorithms for genotype estimation. Genotype posterior probabilities are estimated iteratively until allele frequencies converge, except in the case of mapping populations, where allele frequencies are only estimated once. Purple boxes indicate inputs to the pipeline (read depth, contamination rate, and optionally, genomic positions of loci). Blue boxes indicate estimated parameters (allele frequencies, genotype likelihoods and prior and posterior probabilities, linkage between alleles, and probability of sampling each allele). Green boxes indicate alternative methodologies for genotype prior probability estimation (mapping, HWE, and population structure). Priors for the HWE and population structure models can be adjusted for self-fertilization according to <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">De Silva <em>et al.</em> (2005)</a>. Orange boxes indicate sample × allele matrices indicating approximate allele copy number. Dashed arrows indicate steps that happen only once at the beginning or end of the pipeline, whereas solid arrows indicate iterative steps. Circular arrows highlight cycles of iteration. Equations 1-4 are provided in the main manuscript, and Equations 5-19 are provided in Supplemental Materials.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Example use</h3><p id="p-12">Executable examples are provided in the vignette and manual distributed with polyRAD. Here we provide an additional brief example. <a id="xref-boxed-text-1-1" class="xref-boxed-text" href="#boxed-text-1">Box 1</a> illustrates the use of polyRAD on a diversity panel of a generic tetraploid species with a reference genome. Tools from the Bioconductor package VariantAnnotation (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Obenchain <em>et al.</em> 2014</a>) are used by the polyRAD function <em>VCF2RADdata</em> for import of a VCF file to the polyRAD-specific “RADdata” format. SNP filtering criteria are specified with the <em>min.ind.with.reads</em> and <em>min.ind.with.minor.allele</em> arguments to indicate the minimum number of individuals that must have more than zero reads of a locus, and the minimum number of individuals that must have reads of the minor allele, respectively. The <em>possiblePloidies</em> argument indicates that the inheritance mode could be allotetraploid (<em>c(2,2)</em>) or autotetraploid (<em>4</em>). Any ploidy may be specified with <em>possiblePloidies</em>, for example <em>8</em> for auto-octoploid, with the only limitation that all subgenomes in an allopolyploid must have the same ploidy. By default, <em>VCF2RADdata</em> groups SNP alleles into haplotypes that appear to have come from the same RAD tag, the size of which is specified by <em>tagsize</em>, in basepairs. Negative controls are indicated with <em>SetBlankTaxa</em>, and the contamination rate is estimated with <em>EstimateContaminationRate</em>. The function <em>IteratePopStructLD</em> is then used for genotype estimation, taking both population structure and LD into account. The probabilistic principal components analysis method from the Bioconductor package pcaMethods (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stacklies <em>et al.</em> 2007</a>) is used internally by <em>IteratePopStructLD</em> in order to estimate population structure. The <em>LDdist</em> argument indicates the distance in basepairs within which to search for alleles at other loci that can help predict copy number of a given allele. Once genotype posterior probabilities are estimated, other parameters are cleared from memory using the <em>StripDown</em> function. Continuous numerical genotypes are then formatted for GAPIT (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lipka <em>et al.</em> 2012</a>) using the <em>ExportGAPIT</em> function. Alternative functions are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. A very similar script could be used for a species without a reference genome, with <em>IteratePopStruct</em> in place of <em>IteratePopStructLD</em>, and a different import function for the appropriate non-reference pipeline.</p><div class="boxed-text" id="boxed-text-1"><h3>Box 1. Example R script using polyRAD. Read depth is imported from a VCF file, genotypes are estimated using population structure and LD, and continuous numerical genotypes are formatted for GAPIT.</h3><p id="p-13"><kbd>library(polyRAD)</kbd></p><p id="p-14"><kbd>library(VariantAnnotation)</kbd></p><p id="p-15"><kbd># prepare the VCF file for import</kbd></p><p id="p-16"><kbd>myvcf &lt;- “somegenotypes.vcf”</kbd></p><p id="p-17"><kbd>myvcfbg &lt;- bgzip(myvcf)</kbd></p><p id="p-18"><kbd>indexTabix(myvcfbz, format = “vcf”)</kbd></p><p id="p-19"><kbd># import VCF into a RADdata object</kbd></p><p id="p-20"><kbd>myRAD &lt;- VCF2RADdata(myvcfbg,</kbd></p><p id="p-21"><kbd>tagsize = 64,</kbd></p><p id="p-22"><kbd>min.ind.with.reads = 300,</kbd></p><p id="p-23"><kbd>min.ind.with.minor.allele = 15,</kbd></p><p id="p-24"><kbd>possiblePloidies = list(c(2,2), 4))</kbd></p><p id="p-25"><kbd># estimate contamination rate</kbd></p><p id="p-26"><kbd>myRAD &lt;- SetBlankTaxa(myRAD, c(“blank1”, “blank2”))</kbd></p><p id="p-27"><kbd>myRAD &lt;- EstimateContaminationRate(myRAD)</kbd></p><p id="p-28"><kbd># genotype estimation with pop. structure pipeline</kbd></p><p id="p-29"><kbd>myRAD &lt;- IteratePopStructLD(myRAD, LDdist = 5e4)</kbd></p><p id="p-30"><kbd># free up memory</kbd></p><p id="p-31"><kbd>myRAD &lt;- StripDown(myRAD)</kbd></p><p id="p-32"><kbd># export for GAPIT</kbd></p><p id="p-33"><kbd>myGM_GD &lt;- ExportGAPIT(myRAD)</kbd></p></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492481/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492481/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Overview of main polyRAD functions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Testing</h3><p id="p-35">To test the accuracy of polyRAD, we used datasets from three previously studied populations: 1) RAD-seq data and GoldenGate SNP genotypes from a diversity panel (n = 565) of the outcrossing, diploidized allotetraploid grass <em>Miscanthus sinensis</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clark <em>et al.</em> 2014</a>), 2) RAD-seq data and GoldenGate SNP genotypes from a bi-parental F<sub>1</sub> mapping population (n = 275) of <em>M. sinensis</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Liu <em>et al.</em> 2016a</a>), and 3) SNP array genotypes from a biparental F<sub>1</sub> mapping population of autotetraploid potato (n = 238) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">da Silva <em>et al.</em> 2017</a>). Allelic read depth at simulated RAD-seq markers was generated from the GoldenGate or SNP array genotypes, with overall locus depth drawn from a gamma distribution to resemble depth of actual RAD-seq markers (shape = 2 and scale = 5). The read depth for an individual genotype was also sampled from a gamma distribution, with the shape equal to the locus depth divided by 10, and scale = 10. The read depth for each allele was then sampled from the beta-binomial distribution as described in Equation 2, with <em>d</em> = 9 and <em>c</em> = 0.001. The <em>M. sinensis</em> diversity panel included 395 GoldenGate markers, plus real RAD-seq data for those same individuals across 3290 tag locations within 20 kb of any GoldenGate markers, called with the TASSEL GBS v2 pipeline (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Glaubitz <em>et al.</em> 2014</a>) using the <em>M. sinensis</em> v7.1 reference genome (DOE-JGI, <a href="http://phytozome.jgi.doe.gov/">http://phytozome.jgi.doe.gov/</a>). Additionally, to test the effect of ploidy within the <em>M. sinensis</em> diversity panel, tetraploidy was simulated by summing GoldenGate genotypes and RAD-seq read depth of each individual with the individual with the most similar read depth to it out of the ten individuals most closely related to it. The <em>M. sinensis</em> mapping population included 241 GoldenGate markers genotyped across 83 individuals, plus 3062 RAD-seq markers called with the UNEAK pipeline (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lu <em>et al.</em> 2013</a>) across those 83 individuals plus an additional 192 individuals. The potato mapping population included genotypes at 2538 markers. Additional simulations using data from diversity panels of soybean (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Song <em>et al.</em> 2015</a>), apple (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chagné <em>et al.</em> 2012</a>), and potato (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hamilton <em>et al.</em> 2011</a>) are presented in Figs. S1-S4. In each population, the simulated and real RAD-seq data were used for genotype calling with polyRAD, EBG (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Blischak <em>et al.</em> 2018</a>), updog (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Gerard <em>et al.</em> 2018</a>), and fitPoly (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Voorrips <em>et al.</em> 2011</a>), and missing genotypes from the EBG output were imputed with LinkImpute (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Money <em>et al.</em> 2015</a>) and/or rrBLUP (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Endelman 2011</a>) as appropriate. To estimate the accuracy of genotype calling and imputation, the root mean squared error (RMSE) was calculated between numeric genotypes (ranging from zero to the ploidy) at each simulated RAD-seq marker and at the GoldenGate or SNP array marker from which it was derived.</p></div><div id="sec-5" class="subsection"><h3>Data Availability</h3><p id="p-36">Data and scripts for analysis are available at <a href="https://doi.org/10.13012/B2IDB-9729830_V2">https://doi.org/10.13012/B2IDB-9729830_V2</a>. Supplementary text, equations, and figures have been deposited at Figshare: <a href="https://doi.org/10.25387/g3.7370999">https://doi.org/10.25387/g3.7370999</a>.</p></div></div><div class="section results-discussion" id="sec-6"><h2 class="">Results and discussion</h2><div id="sec-7" class="subsection"><h3>Accuracy of polyRAD</h3><p id="p-37">In the <em>M. sinensis</em> diversity panel, polyRAD showed improved genotype accuracy over the HWE, disequilibrium, and GATK methods implemented in EBG, as well as fitPoly, particularly at low read depths (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">3A</a>). polyRAD also showed a modest improvement in accuracy across all read depths as compared to updog (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3A</a>) while needing approximately two orders of magnitude less processing time than updog. Under the HWE model in polyRAD with discrete genotypes output, errors in genotypes with more than zero reads were similar to those from the HWE model of EBG in both diploid and tetraploid systems (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3A</a>). However, when priors in polyRAD were based on population structure, errors decreased, particularly in tetraploids and at low read depth (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2A</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3A</a>). In diploids and tetraploids respectively using the polyRAD population structure model with discrete genotypes, error (RMSE) was reduced by 14.6% (SE 1.0%) and 23.5% (SE 0.6%) relative to the GATK model, by 10.5% (SE 0.9%) and 11.8% (SE 0.5%) relative to the EBG HWE model, by 26.0% (SE 1.2%) and 25.6% (SE 0.6%) relative fitPoly, and by 8.0% (SE 1.0%) and 18.0% (SE 0.7%) relative to discrete genotype output by the updog “norm” model. Given the known population structure in <em>M. sinensis</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Clark <em>et al.</em> 2014</a>), it is unsurprising that a population structure-aware genotyping method would be more accurate than those based on HWE or otherwise not accounting for population structure. For genotypes with zero reads, imputation was most accurate when it accounted for population structure, using either polyRAD or rrBLUP (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">3B</a>). Although modeling LD did not improve accuracy in <em>M. sinensis</em> (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">3</a>), likely due to low LD as a result of outcrossing (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Slavov <em>et al.</em> 2014</a>), modeling LD did improve accuracy in wild soybean, apple, and a simulated inbreeding allohexaploid (Figures S1, S2, and S3, and Supporting Results). In a diversity panel of tetraploid potato, accuracy was improved by modeling population structure but not LD (Figures S4 and Supporting Results).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/663/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in a diversity panel of 565 diploid Miscanthus sinensis. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;norm&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1284579180" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in a diversity panel of 565 diploid &lt;em&gt;Miscanthus sinensis&lt;/em&gt;. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;norm&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/663/F2.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/3/663/F2.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/663/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/663/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492335" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-38" class="first-child">Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in a diversity panel of 565 diploid <em>Miscanthus sinensis</em>. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The “norm” model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/663/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping error of EBG, fitPoly, updog, polyRAD, and rrBLUP in a simulated tetraploid diversity panel derived from genotypes of 565 diploid Miscanthus sinensis. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;norm&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type. LinkImpute was not included given that it works for diploids only." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1284579180" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotyping error of EBG, fitPoly, updog, polyRAD, and rrBLUP in a simulated tetraploid diversity panel derived from genotypes of 565 diploid &lt;em&gt;Miscanthus sinensis&lt;/em&gt;. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;norm&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type. LinkImpute was not included given that it works for diploids only.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/663/F3.medium.gif" width="413" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/3/663/F3.medium.gif" width="413" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/663/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/663/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Genotyping error of EBG, fitPoly, updog, polyRAD, and rrBLUP in a simulated tetraploid diversity panel derived from genotypes of 565 diploid <em>Miscanthus sinensis</em>. The benefits of incorporating population structure into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 395 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD PopStruct LD method is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The “norm” model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type. LinkImpute was not included given that it works for diploids only.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">In diploid <em>M. sinensis</em> and tetraploid potato F1 mapping populations, polyRAD outperformed the GATK method, fitPoly, and updog, particularly when linked markers were used for informing the priors in polyRAD (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4A</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5A</a>). In diploids and tetraploids respectively using genotypes with non-zero read depth, error (RMSE) using the polyRAD linkage model with discrete genotypes was reduced by 31.6% (SE 2.2%) and 48.0% (SE 0.4%) with respect to the GATK model, and 1.5% (SE 3.1%) and 17.1% (SE 0.6%) with respect to the updog “f1” model with discrete genotypes. For diploids, error was reduced by 39.8% (SE 2.5%) using polyRAD with respect to fitPoly, while for tetraploids fitPoly failed for all markers. For imputation, polyRAD using the linkage model performed similarly to LinkImpute and rrBLUP (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4B</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">5B</a>). Although only F1 populations are presented here, many other population types are supported in polyRAD.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/663/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in an F1 mapping population of 83 diploid Miscanthus sinensis. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 241 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;f1&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1284579180" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in an F1 mapping population of 83 diploid &lt;em&gt;Miscanthus sinensis&lt;/em&gt;. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 241 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;f1&#x201D; model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/663/F4.medium.gif" width="356" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/3/663/F4.medium.gif" width="356" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/663/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/663/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492466" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Genotyping error of EBG, fitPoly, updog, polyRAD, LinkImpute, and rrBLUP in an F1 mapping population of 83 diploid <em>Miscanthus sinensis</em>. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 2. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 241 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The “f1” model was used with updog. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/3/663/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping error of EBG, updog, polyRAD, and rrBLUP in an F1 mapping population of tetraploid potato with 238 progeny. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 2538 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;f1&#x201D; model was used with updog. fitPoly results are omitted since it failed for all markers, and LinkImpute was not run since LinkImpute is for diploids only. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1284579180" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotyping error of EBG, updog, polyRAD, and rrBLUP in an F1 mapping population of tetraploid potato with 238 progeny. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 2538 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The &#x201C;f1&#x201D; model was used with updog. fitPoly results are omitted since it failed for all markers, and LinkImpute was not run since LinkImpute is for diploids only. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/3/663/F5.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/3/663/F5.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/3/663/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/3/663/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492369" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Genotyping error of EBG, updog, polyRAD, and rrBLUP in an F1 mapping population of tetraploid potato with 238 progeny. The benefits of incorporating linkage into the genotyping model and using continuous rather than discrete genotypes are illustrated. Genotypes were coded on a scale of 0 to 4. Root mean squared error (RMSE) was calculated between actual genotypes and genotypes ascertained from simulated RAD-seq reads at 2538 SNP markers (lower RMSE = higher accuracy). Each point represents one SNP. Median read depth is indicated by color, including genotypes with zero reads. The RMSE for continuous genotypes output by the polyRAD mapping method with linkage is shown on the x-axis, and the RMSE of other methods and types of genotypes (continuous or discrete) is shown on the y-axis. The dashed line indicates the ordinary least-squares regression with slope and intercept estimates, with standard errors. The “f1” model was used with updog. fitPoly results are omitted since it failed for all markers, and LinkImpute was not run since LinkImpute is for diploids only. (A) RMSE calculated using only genotypes with more than zero reads. (B) RMSE calculated using only genotypes with zero reads, by genotyping or imputation method and genotype type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">Genotyping error was also reduced 10–15% in most cases by exporting genotypes as continuous numerical variables (posterior mean genotypes), rather than discrete values (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figures 2</a>-<a id="xref-fig-5-3" class="xref-fig" href="#F5">5</a>). For example, in a diploid, a true heterozygote (numeric value of 1) with reads only for the reference allele might erroneously be called as zero (homozygous for the reference allele) if only the most probable genotype is exported. However, the genotype could be called 0.4 if continuous genotypes are allowed, indicating that there is a 60% chance of it being a homozygote and 40% chance of it being a heterozygote, and thereby reducing the error from 1.0 to 0.6. Similarly in polyploids, continuous numerical genotypes can correct for errors in allele copy number estimation of heterozygotes.</p></div><div id="sec-8" class="subsection"><h3>Downstream applications and implications for sequencing strategies</h3><p id="p-44">The genotyping methods implemented in polyRAD will have the most benefit for marker analysis where 1) the accuracy of individual genotypes is important, and 2) genotypes can be treated as continuous rather than discrete variables. The use of continuous <em>vs.</em> discrete genotypes has been demonstrated to increase power for genome-wide association studies (GWAS) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grandke <em>et al.</em> 2016</a>) and genomic prediction (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Oliveira <em>et al.</em> 2018</a>) in polyploids. More generally, we anticipate that analyses that seek to quantify marker-trait associations in a population of individuals, including GWAS, quantitative trait locus mapping, and genomic prediction methods involving variable selection, will especially benefit from polyRAD. By reducing genotyping error, polyRAD will increase the power of these methods to detect true associations. Analyses that will benefit less from polyRAD genotyping are those where an average is taken across many genotypes in order to estimate a statistic, such as allele frequencies in a population or overall relatedness of individuals (including kinship-based methods of genomic prediction), because genotyping errors generally are not biased toward one allele or the other and tend to balance out over many individuals and loci (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buerkle and Gompert 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dodds <em>et al.</em> 2015</a>).</p><p id="p-45">The advantages of polyRAD for accurate genotyping at low sequence read depth alter the economics of sequence-based genotyping, enabling researchers to purchase fewer sequencing lanes, multiplex more samples per lane, and/or retain more markers during filtering. In particular, for protocols using restriction enzymes where read depth varies considerably from locus to locus depending on fragment size (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beissinger <em>et al.</em> 2013</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrews <em>et al.</em> 2016</a>), there are diminishing returns on increasing the per-sample read depth, because some loci receive far more reads than are needed for accurate genotyping while other loci remain poor quality. Using population structure and linkage between loci, polyRAD uses information from high-depth markers to improve genotyping accuracy of low-depth markers, helping to maximize the useful information that is obtained from sequencing data. This advance is expected to improve breeding efficiency and economics.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">This material is based upon work supported by the National Science Foundation under Grant No. 1661490. The authors thank Marcus Olatoye, Per McCord, Joyce Njuguna, Wittney Mays, and Jiale He for testing the polyRAD software, and Associate Editor Daniel Runcie and two anonymous reviewers for comments on earlier versions of this manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-47"><em>Communicating editor: D. Runcie</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-21"><span class="received-label">Received </span>November 21, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-09"><span class="accepted-label">Accepted </span>January 9, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Clark <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.1" data-doi="10.1038/nrg.2015.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg.2015.28</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2015.28%26rft_id%253Dinfo%253Apmid%252F26729255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2015.28&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26729255&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.2" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.3" data-doi="10.1093/bioinformatics/btx587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blischak</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubatko</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">SNP genotyping and parameter estimation in polyploids using low-coverage sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx587</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.4" data-doi="10.1093/bioinformatics/bty371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Geest</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kranenburg</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">polymapR—linkage analysis and genetic map construction from F1 populations of outcrossing polyploids.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3496</span>–<span class="cit-lpage">3502</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty371</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3496%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.5" data-doi="10.3389/fpls.2018.00513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Tools for genetic studies in experimental populations of polyploids.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.6" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.7" data-doi="10.1111/mec.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buerkle</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gompert</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics based on low coverage sequencing: How low should we go?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3028</span>–<span class="cit-lpage">3035</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3028%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.8" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.9" data-doi="10.1371/journal.pone.0031745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chagné</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowhurst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmore</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e31745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0031745</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChagne%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De31745%26rft.epage%253De31745%26rft.atitle%253DGenome-wide%2BSNP%2Bdetection%252C%2Bvalidation%252C%2Band%2Bdevelopment%2Bof%2Ban%2B8K%2BSNP%2Barray%2Bfor%2Bapple.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0031745%26rft_id%253Dinfo%253Apmid%252F22363718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0031745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363718&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.10" data-doi="10.1093/aob/mcu084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brummer</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Głowacka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heo</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A footprint of past climate change on the diversity and population structure of <em>Miscanthus sinensis</em>.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/aob/mcu084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcu084%26rft_id%253Dinfo%253Apmid%252F24918203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcu084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24918203&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.11" data-doi="10.1186/s13029-016-0057-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacks</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">TagDigger: user-friendly extraction of read counts from GBS and RAD-seq data.</span> <abbr class="cit-jnl-abbrev">Source Code Biol. Med.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13029-016-0057-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSource%2BCode%2BBiol.%2BMed.%26rft.volume%253D11%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13029-016-0057-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13029-016-0057-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.12" data-doi="10.1111/mec.12084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes-Utrilla</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Special features of RAD Sequencing data: implications for genotyping.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3151</span>–<span class="cit-lpage">3164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12084</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3151%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.13" data-doi="10.1186/s12864-015-2252-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwan</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brauning</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Stijn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of relatedness matrices using genotyping-by-sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2252-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1047%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2252-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2252-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.14" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-other" id="cit-9.3.663.15"><div class="cit-metadata unstructured">Garrison, E., and G. Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv 1207.3907.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.16" data-doi="10.1534/genetics.118.301468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genotyping polyploids from messy sequencing data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301468%26rft_id%253Dinfo%253Apmid%252F30185430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.17" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.18" data-doi="10.1186/s12864-016-2926-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandke</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuven</span>  <span class="cit-name-given-names">H. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Advantages of continuous genotype values over genotype classes for GWAS in higher polyploids: A comparative study in hexaploid chrysanthemum.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2926-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D672%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2926-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2926-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.19" data-doi="10.1371/journal.pgen.1000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Practical issues in imputation-based association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000279%26rft.epage%253De1000279%26rft.atitle%253DPractical%2Bissues%2Bin%2Bimputation-based%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000279%26rft_id%253Dinfo%253Apmid%252F19057666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19057666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.20" data-doi="10.1186/1471-2164-12-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitty</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single nucleotide polymorphism discovery in elite north American potato germplasm.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHamilton%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D12%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bin%2Belite%2BNorth%2BAmerican%2Bpotato%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-302%26rft_id%253Dinfo%253Apmid%252F21658273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21658273&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.21" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: Analysis of Next Generation Sequencing Data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-014-0356-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.22" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.23" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22796960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308532300066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.24" data-doi="10.1111/gcbb.12275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swaminathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juvik</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">High density genetic map of <em>Miscanthus sinensis</em> reveals inheritance of zebra stripe.</span> <abbr class="cit-jnl-abbrev">Glob. Change Biol. Bioenergy</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gcbb.12275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BChange%2BBiol.%2BBioenergy%26rft.volume%253D8%26rft.spage%253D616%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcbb.12275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcbb.12275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-9.3.663.25" data-doi="10.1371/journal.pgen.1005767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome-Wide Association Studies.</span> <span class="cit-source">PLOS Genet.</span> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005767</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005767%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.26" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.27" data-doi="10.1534/g3.117.039008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genotype Calling from Population-Genomic Sequencing Data.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1393</span>–<span class="cit-lpage">1404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.039008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.039008%26rft_id%253Dinfo%253Apmid%252F28108551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xMzkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOS8zLzY2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.28" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.29" data-doi="10.1111/nyas.12466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The causes and molecular consequences of polyploidy in flowering plants.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1320</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nyas.12466</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1320%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.12466%26rft_id%253Dinfo%253Apmid%252F24903334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.12466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24903334&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.30" data-doi="10.1534/g3.115.021667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Money</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Migicovsky</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwaninger</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">LinkImpute: Fast and Accurate Genotype Imputation for Nonmodel Organisms.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2383</span>–<span class="cit-lpage">2390</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.021667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.021667%26rft_id%253Dinfo%253Apmid%252F26377960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTEvMjM4MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.31" data-doi="10.1093/nar/gkr344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshikawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sequence-specific error profile of Illumina sequencers.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr344%26rft_id%253Dinfo%253Apmid%252F21576222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21576222&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.32" data-doi="10.1038/nrg2986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype and SNP calling from next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DGenotype%2Band%2BSNP%2Bcalling%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2986%26rft_id%253Dinfo%253Apmid%252F21587300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21587300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.33" data-doi="10.1093/bioinformatics/btu168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Obenchain</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">VariantAnnotation: A Bioconductor package for exploration and annotation of genetic variants.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2076</span>–<span class="cit-lpage">2078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu168%26rft_id%253Dinfo%253Apmid%252F24681907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24681907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339814300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.34" data-doi="10.1101/432179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">I. de B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amadeu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction of autotetraploids; influence of relationship matrices, allele dosage, and continuous genotyping calls in phenotype prediction.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">432179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/432179</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F432179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/432179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.35" data-doi="10.3835/plantgenome2012.05.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rife</span>  <span class="cit-name-given-names">T. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotyping-by-sequencing for plant breeding and genetics.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.05.0005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D5%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.05.0005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.05.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.36" data-doi="10.1371/journal.pone.0066428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Yield Trends Are Insufficient to Double Global Crop Production by 2050.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e66428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0066428</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De66428%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0066428%26rft_id%253Dinfo%253Apmid%252F23840465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0066428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23840465&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.37" data-doi="10.3732/ajb.1400119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renny-Byfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Doubling down on genomes: Polyploidy and crop plants.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1711</span>–<span class="cit-lpage">1725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1400119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400119%26rft_id%253Dinfo%253Apmid%252F25090999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjExOiIxMDEvMTAvMTcxMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.38" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for Bayesian SNP genotyping in polyploids.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-9.3.663.39"><div class="cit-metadata unstructured">Shiryaev, A. N., 2011 Bayes formula. Encycl. Math. Available at: <a href="https://www.encyclopediaofmath.org//index.php?title=Bayes_formula&amp;#x0026;oldid=16075">https://www.encyclopediaofmath.org//index.php?title=Bayes_formula&amp;oldid=16075</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.40" data-doi="10.1534/g3.117.300264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mapping Loci That Control Tuber and Foliar Symptoms Caused by PVY in Autotetraploid Potato (<em>Solanum tuberosum</em> L.).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3587</span>–<span class="cit-lpage">3595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.300264</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300264%26rft_id%253Dinfo%253Apmid%252F28903982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjcvMTEvMzU4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.41" data-doi="10.1038/sj.hdy.6800728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Silva</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rikkerink</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeilage</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">L. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of allele frequencies in polyploids under certain patterns of inheritance.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6800728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DDe%2BSilva%26rft.auinit1%253DH.%2BN.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D327%26rft.epage%253D334%26rft.atitle%253DEstimation%2Bof%2Ballele%2Bfrequencies%2Bin%2Bpolyploids%2Bunder%2Bcertain%2Bpatterns%2Bof%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800728%26rft_id%253Dinfo%253Apmid%252F16094298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16094298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232184900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.42" data-doi="10.1111/nph.12621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slavov</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nipper</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association studies and prediction of 17 traits related to phenology, biomass and cell wall composition in the energy grass <em>Miscanthus sinensis</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D201%26rft.spage%253D1227%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12621%26rft_id%253Dinfo%253Apmid%252F24308815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24308815&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.43" data-doi="10.1534/g3.115.019000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fingerprinting Soybean Germplasm and Its Utility in Genomic Research.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.019000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019000%26rft_id%253Dinfo%253Apmid%252F26224783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTk5OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzkvMy82NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.44" data-doi="10.1093/bioinformatics/btm069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stacklies</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redestig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selbig</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">pcaMethods - A bioconductor package providing PCA methods for incomplete data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm069%26rft_id%253Dinfo%253Apmid%252F17344241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17344241&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246773300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.45" data-doi="10.1371/journal.pgen.1005957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The PLoS Genetics Staff</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Correction: Iterative Usage of Fixed and Random Effect Models for Powerful and Efficient Genome-Wide Association Studies</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005957</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D12%26rft.spage%253De1005957%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.46" data-doi="10.1534/g3.115.024596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekele</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Haplotag: Software for Haplotype-Based Genotyping-by-Sequencing Analysis.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">863</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.024596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024596%26rft_id%253Dinfo%253Apmid%252F26818073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvNC84NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy85LzMvNjYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.3.663.47" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gort</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F3%2F663.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/3/663.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/3/651" title="Evaluation of INSeq To Identify Genes Essential for Pseudomonas aeruginosa PGPR2 Corn Root Colonization" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/3/675" title="A Low Resolution Epistasis Mapping Approach To Identify Chromosome Arm Interactions in Allohexaploid Wheat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/3"><span>Volume 9 Issue 3, March 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/3.cover-source.jpg?itok=HDVm34vd" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/3/663.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ah1CyrJdTKjqJYpIy1GCvCQMXdu2Gyljr_EbU5lUmvY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/3/663" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/3/663" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/3/663" class="active" data-icon-position="" data-hide-link-title="0">polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492359" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IZhH8w_1YLejVw2Xus5k509_AehjunGKncMFXFrt5sk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/663.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/3/663" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RZUrfdd7fxMNrvH3gTbotJUXR9Mu4lDD8vrRjA7rByI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/3/663" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/3/663.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492359" id="node492359--2" data-pisa="ggg;9/3/663" data-pisa-master="ggg;g3.118.200913" data-apath="/ggg/9/3/663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lindsay V. Clark</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="2">Erik J. Sacks</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">663-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492359--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay V. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BV.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20LV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALindsay%2BV.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lindsay V. Clark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvclark@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">lvclark@illinois.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erik J. Sacks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BJ.%2BSacks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sacks%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BJ.%2BSacks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/3/663" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/3/663&amp;text=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/3/663&amp;t=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/3/663&amp;title=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492359/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492359" id="node492359--3" data-pisa="ggg;9/3/663" data-pisa-master="ggg;g3.118.200913" data-apath="/ggg/9/3/663.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">polyRAD: Genotype Calling with Uncertainty from Sequencing Data in Polyploids and Diploids</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lindsay V. Clark</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span> and <span class="highwire-citation-author" data-delta="2">Erik J. Sacks</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">663-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200913 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492359--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay V. Clark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BV.%2BClark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Clark%20LV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALindsay%2BV.%2BClark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3881-9252" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lindsay V. Clark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lvclark@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">lvclark@illinois.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Erik J. Sacks </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BJ.%2BSacks%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sacks%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BJ.%2BSacks%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=663&amp;rft.epage=673&amp;rft.atitle=polyRAD%3A%20Genotype%20Calling%20with%20Uncertainty%20from%20Sequencing%20Data%20in%20Polyploids%20and%20Diploids&amp;rft.volume=9&amp;rft.issue=3&amp;rft.date=2019-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Clark%2C+Lindsay+V.&amp;rft.au=Lipka%2C+Alexander+E.&amp;rft.au=Sacks%2C+Erik+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492359/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492359/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492359/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492359/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492359/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492359/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492359/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492359/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492359/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492359/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492359/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492359/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/933" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/3/663" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/3/663.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/3/663.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/3/663.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/3/663.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
