<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/11/1415" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/11/1415.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/11/1415.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/11/1415.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/11/1415.atom" />
<meta name="HW.pisa" content="ggg;2/11/1415" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" />
<meta name="DC.Identifier" content="10.1534/g3.112.003830" />
<meta name="DC.Date" content="2012-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 The C. elegans Deletion Mutant Consortium. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The nematode Caenorhabditis elegans is a powerful model system to study contemporary biological problems. This system would be even more useful if we had mutations in all the genes of this multicellular metazoan. The combined efforts of the C. elegans Deletion Mutant Consortium and individuals within the worm community are moving us ever closer to this goal. At present, of the 20,377 protein-coding genes in this organism, 6764 genes with associated molecular lesions are either deletions or null mutations (WormBase WS220). Our three laboratories have contributed the majority of mutated genes, 6841 mutations in 6013 genes. The principal method we used to detect deletion mutations in the nematode utilizes polymerase chain reaction (PCR). More recently, we have used array comparative genome hybridization (aCGH) to detect deletions across the entire coding part of the genome and massively parallel short-read sequencing to identify nonsense, splicing, and missense defects in open reading frames. As deletion strains can be frozen and then thawed when needed, these strains will be an enduring community resource. Our combined molecular screening strategies have improved the overall throughput of our gene-knockout facilities and have broadened the types of mutations that we and others can identify. These multiple strategies should enable us to eventually identify a mutation in every gene in this multicellular organism. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes." />
<meta name="DC.Contributor" content="The C. elegans Deletion Mutant Consortium" />
<meta name="article:published_time" content="2012-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The nematode &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; is a powerful model system to study contemporary biological problems. This system would be even more useful if we had mutations in all the genes of this multicellular metazoan. The combined efforts of the &lt;i&gt;C. elegans&lt;/i&gt; Deletion Mutant Consortium and individuals within the worm community are moving us ever closer to this goal. At present, of the 20,377 protein-coding genes in this organism, 6764 genes with associated molecular lesions are either deletions or null mutations (WormBase WS220). Our three laboratories have contributed the majority of mutated genes, 6841 mutations in 6013 genes. The principal method we used to detect deletion mutations in the nematode utilizes polymerase chain reaction (PCR). More recently, we have used array comparative genome hybridization (aCGH) to detect deletions across the entire coding part of the genome and massively parallel short-read sequencing to identify nonsense, splicing, and missense defects in open reading frames. As deletion strains can be frozen and then thawed when needed, these strains will be an enduring community resource. Our combined molecular screening strategies have improved the overall throughput of our gene-knockout facilities and have broadened the types of mutations that we and others can identify. These multiple strategies should enable us to eventually identify a mutation in every gene in this multicellular organism. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/11/01" />
<meta name="citation_mjid" content="ggg;2/11/1415" />
<meta name="citation_id" content="2/11/1415" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/11/1415" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/11/1415.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/11/1415.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/11/1415.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003830" />
<meta name="citation_pmid" content="23173093" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1415" />
<meta name="citation_lastpage" content="1425" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="The C. elegans Deletion Mutant Consortium" />
<meta name="citation_author_institution" content="Molecular and Cell Biology, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma, 73104" />
<meta name="citation_author_institution" content="Department of Physiology, Tokyo Women’s Medical University School of Medicine, Tokyo 162-8666, Japan" />
<meta name="citation_author_institution" content="Department of Zoology and Michael Smith Laboratories, University of British Columbia, Vancouver, British Columbia, V6T 1Z3 Canada" />
<meta name="citation_author_email" content="moerman@zoology.ubc.ca" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=Z. F. Altun;citation_author=B. Chen;citation_author=Z. W. Wang;citation_author=D. H. Hall;citation_title=High resolution map of Caenorhabditis elegans gap junction proteins.;citation_pages=1936-1950;citation_volume=238;citation_year=2009;citation_issue=8;citation_pmid=19621339;citation_doi=10.1002/dvdy.22025" />
<meta name="citation_reference" content="citation_title=C. elegans II;citation_title=Heterochronic genes;citation_author=V. Ambros;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. Ambros;citation_title=The functions of animal microRNAs.;citation_pages=350-355;citation_volume=431;citation_year=2004;citation_issue=7006;citation_pmid=15372042;citation_doi=10.1038/nature02871" />
<meta name="citation_reference" content="citation_author=A. Antebi;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Ashrafi;citation_author=F. Y. Chang;citation_author=J. L. Watts;citation_author=A. G. Fraser;citation_author=R. S. Kamath;citation_title=Genome-wide RNAi analysis of Caenorhabditis elegans fat regulatory genes.;citation_pages=268-272;citation_volume=421;citation_year=2003;citation_issue=6920;citation_pmid=12529643;citation_doi=10.1038/nature01279" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Avery;citation_title=The genetics of feeding in Caenorhabditis elegans;citation_pages=897-917;citation_volume=133;citation_year=1993;citation_pmid=8462849" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. I. Bargmann;citation_title=Neurobiology of the Caenorhabditis elegans Genome;citation_pages=2028-2033;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851919;citation_doi=10.1126/science.282.5396.2028" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in chemical biology;citation_journal_abbrev=Curr Opin Chem Biol;citation_author=R. Barstead;citation_title=Genome-wide RNAi.;citation_pages=63-66;citation_volume=5;citation_year=2001;citation_issue=1;citation_pmid=11166650;citation_doi=10.1016/S1367-5931(00)00173-3" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=R. J. Barstead;citation_author=D. G. Moerman;citation_title=C. elegans deletion mutant screening.;citation_pages=51-58;citation_volume=351;citation_year=2006;citation_pmid=16988425" />
<meta name="citation_reference" content="citation_author=J.-L. Bessereau;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. Boch;citation_title=TALEs of genome targeting.;citation_pages=135-136;citation_volume=29;citation_year=2011;citation_issue=2;citation_pmid=21301438;citation_doi=10.1038/nbt.1767" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. J. Bogdanove;citation_author=D. F. Voytas;citation_title=TAL Effectors: Customizable Proteins for DNA Targeting;citation_pages=1843-1846;citation_volume=333;citation_year=2011;citation_issue=6051;citation_pmid=21960622;citation_doi=10.1126/science.1204094" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Brenner;citation_title=The genetics of Caenorhabditis elegans;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_pmid=4366476" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology;citation_pages=2012-2018;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851916;citation_doi=10.1126/science.282.5396.2012" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Cermak;citation_author=E. L. Doyle;citation_author=M. Christian;citation_author=L. Wang;citation_author=Y. Zhang;citation_title=Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting;citation_volume=39;citation_year=2011;citation_pmid=21493687;citation_doi=10.1093/nar/gkr218" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. A. Chervitz;citation_author=L. Aravind;citation_author=G. Sherlock;citation_author=C. A. Ball;citation_author=E. V. Koonin;citation_title=Comparison of the Complete Protein Sets of Worm and Yeast: Orthology and Divergence;citation_pages=2022-2028;citation_volume=282;citation_year=1998;citation_issue=5396;citation_pmid=9851918;citation_doi=10.1126/science.282.5396.2022" />
<meta name="citation_reference" content="citation_journal_title=Methods in cell biology;citation_journal_abbrev=Methods Cell Biol;citation_author=M. L. Edgley;citation_author=D. L. Baillie;citation_author=D. l. Riddle;citation_author=A. M. Rose;citation_title=Genetic balancers.;citation_pages=147-184;citation_volume=48;citation_year=1995;citation_pmid=8531724" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. L. Edgley;citation_author=A. D’Souza;citation_author=G. Moulder;citation_author=S. McKay;citation_author=B. Shen;citation_title=Improved detection of small deletions in complex pools of DNA;citation_pages=e52-e52;citation_volume=30;citation_year=2002;citation_issue=12;citation_pmid=12060690;citation_doi=10.1093/nar/gnf051" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Fire;citation_author=S. Xu;citation_author=M. K. Montgomery;citation_author=S. A. Kostas;citation_author=S. E. Driver;citation_title=Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans.;citation_pages=806-811;citation_volume=391;citation_year=1998;citation_issue=6669;citation_pmid=9486653;citation_doi=10.1038/35888" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=I. Chaudry;citation_author=J. Taylor;citation_author=S. E. Neil;citation_title=Whole-genome profiling of mutagenesis in Caenorhabditis elegans;citation_pages=431-441;citation_volume=185;citation_year=2010;citation_pmid=20439774;citation_doi=10.1534/genetics.110.116616" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Fraser;citation_author=R. S. Kamath;citation_author=P. Zipperlen;citation_author=M. Martinez-Campos;citation_author=M. Sohrmann;citation_title=Functional genomic analysis of C. elegans chromosome I by systematic RNA interference.;citation_pages=325-330;citation_volume=408;citation_year=2000;citation_issue=6810;citation_pmid=11099033;citation_doi=10.1038/35042517" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Frøkjaer-Jensen;citation_author=M. W. Davis;citation_author=C. E. Hopkins;citation_author=B. J. Newman;citation_author=J. M. Thummel;citation_title=Single-copy insertion of transgenes in Caenorhabditis elegans.;citation_pages=1375-1383;citation_volume=40;citation_year=2008;citation_issue=11;citation_pmid=18953339;citation_doi=10.1038/ng.248" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Frøkjaer-Jensen;citation_author=M. W. Davis;citation_author=G. Hollopeter;citation_author=J. Taylor;citation_author=T. W. Harris;citation_title=Targeted gene deletions in C. elegans using transposon excision.;citation_pages=451-453;citation_volume=7;citation_year=2010;citation_issue=6;citation_pmid=20418868;citation_doi=10.1038/nmeth.1454" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=K. Gengyo-Ando;citation_author=S. Mitani;citation_title=Characterization of mutations induced by ethyl methanesulfonate, UV, and trimethylpsoralen in the nematode Caenorhabditis elegans.;citation_pages=64-69;citation_volume=269;citation_year=2000;citation_issue=1;citation_pmid=10694478;citation_doi=10.1006/bbrc.2000.2260" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Gönczy;citation_author=C. Echeverri;citation_author=K. Oegema;citation_author=A. Coulson;citation_author=S. J. Jones;citation_title=Functional genomic analysis of cell division in C. elegans using RNAi of genes on chromosome III.;citation_pages=331-336;citation_volume=408;citation_year=2000;citation_issue=6810;citation_pmid=11099034;citation_doi=10.1038/35042526" />
<meta name="citation_reference" content="citation_author=M. B. Goodman;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Hutter;citation_author=M. P. Ng;citation_author=N. Chen;citation_title=GExplore: a web server for integrated queries of protein domains, gene expression and mutant phenotypes.;citation_pages=529-529;citation_volume=10;citation_year=2009;citation_pmid=19917126;citation_doi=10.1186/1471-2164-10-529" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. Jansen;citation_author=E. Hazendonk;citation_author=K. L. Thijssen;citation_author=R. H. Plasterk;citation_title=Reverse genetics by chemical mutagenesis in Caenorhabditis elegans.;citation_pages=119-121;citation_volume=17;citation_year=1997;citation_issue=1;citation_pmid=9288111;citation_doi=10.1038/ng0997-119" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. S. Kamath;citation_author=A. G. Fraser;citation_author=Y. Dong;citation_author=G. Poulin;citation_author=R. Durbin;citation_title=Systematic functional analysis of the Caenorhabditis elegans genome using RNAi.;citation_pages=231-237;citation_volume=421;citation_year=2003;citation_issue=6920;citation_pmid=12529635;citation_doi=10.1038/nature01278" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=E. J. Kaufman;citation_author=E. A. Miska;citation_title=The microRNAs of Caenorhabditis elegans.;citation_pages=728-737;citation_volume=21;citation_year=2010;citation_issue=7;citation_pmid=20637886;citation_doi=10.1016/j.semcdb.2010.07.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. W. Knight;citation_author=B. L. Bass;citation_title=A role for the RNase III enzyme DCR-1 in RNA interference and germ line development in Caenorhabditis elegans;citation_volume=293;citation_year=2001;citation_pmid=11486053;citation_doi=10.1126/science.1062039" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Krause;citation_author=M. Wild;citation_author=B. Rosenzweig;citation_author=D. Hirsh;citation_title=Wild-type and mutant actin genes in Caenorhabditis elegans.;citation_pages=381-392;citation_volume=208;citation_year=1989;citation_issue=3;citation_pmid=2795655;citation_doi=10.1016/0022-2836(89)90503-2" />
<meta name="citation_reference" content="citation_author=C. Li;citation_author=K. Kim;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Li;citation_author=S. Huang;citation_author=W. Z. Jiang;citation_author=D. Wright;citation_author=M. H. Spalding;citation_title=TAL nucleases (TALNs): hybrid proteins composed of TAL effectors and FokI DNA-cleavage domain;citation_volume=39;citation_year=2011;citation_pmid=20699274;citation_doi=10.1093/nar/gkq704" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. X. Liu;citation_author=J. M. Spoerke;citation_author=E. L. Mulligan;citation_author=J. Chen;citation_author=B. Reardon;citation_title=High-Throughput Isolation of Caenorhabditis elegans Deletion Mutants;citation_pages=859-867;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508845;citation_doi=10.1101/gr.9.9.859" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. J. MacQueen;citation_author=J. J. Baggett;citation_author=N. Perumov;citation_author=R. A. Bauer;citation_author=T. Januszewski;citation_title=ACT-5 is an essential Caenorhabditis elegans actin required for intestinal microvilli formation;citation_volume=16;citation_year=2005;citation_pmid=15872090;citation_doi=10.1091/mbc.E04-12-1061" />
<meta name="citation_reference" content="citation_author=G. Manning;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=I. Mano;citation_author=M. Driscoll;citation_title=DEG/ENaC channels: a touchy superfamily that watches its salt.;citation_pages=568-578;citation_volume=21;citation_year=1999;citation_issue=7;citation_pmid=10472184;citation_doi=10.1002/(SICI)1521-1878(199907)21:7&lt;568::AID-BIES5&gt;3.0.CO;2-L" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. S. Maydan;citation_author=S. Flibotte;citation_author=M. L. Edgley;citation_author=J. Lau;citation_author=R. R. Selzer;citation_title=Efficient high-resolution deletion discovery in Caenorhabditis elegans by array comparative genomic hybridization;citation_volume=17;citation_year=2007;citation_pmid=17267812;citation_doi=10.1101/gr.5690307" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. S. Maydan;citation_author=A. Lorch;citation_author=M. L. Edgley;citation_author=S. Flibotte;citation_author=D. G. Moerman;citation_title=Copy number variation in the genomes of twelve natural isolates of Caenorhabditis elegans.;citation_pages=62-62;citation_volume=11;citation_year=2010;citation_pmid=20100350;citation_doi=10.1186/1471-2164-11-62" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. J. McKay;citation_author=S. J. M. Jones;citation_title=AcePrimer: automation of PCR primer design based on gene structure;citation_pages=1538-1539;citation_volume=18;citation_year=2002;citation_issue=11;citation_pmid=12424127;citation_doi=10.1093/bioinformatics/18.11.1538" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. A. Miska;citation_author=E. Alvarez-Saavedra;citation_author=A. L. Abbott;citation_author=N. C. Lau;citation_author=A. B. Hellman;citation_title=Most Caenorhabditis elegans microRNAs are individually not essential for development or viability.;citation_pages=e215-e215;citation_volume=3;citation_year=2007;citation_issue=12;citation_pmid=18085825;citation_doi=10.1371/journal.pgen.0030215" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics Proteomics;citation_author=D. G. Moerman;citation_author=R. J. Barstead;citation_title=Towards a mutation in every gene in Caenorhabditis elegans;citation_volume=7;citation_year=2008;citation_pmid=18417533;citation_doi=10.1093/bfgp/eln016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. P. O’Brien;citation_author=M. Remm;citation_author=E. L. Sonnhammer;citation_title=Inparanoid: a comprehensive database of eukaryotic orthologs;citation_pages=D476-D480;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608241;citation_doi=10.1093/nar/gki107" />
<meta name="citation_reference" content="citation_author=J. Pettitt;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=P. Phelan;citation_author=T. A. Starich;citation_title=Innexins get into the gap.;citation_pages=388-396;citation_volume=23;citation_year=2001;citation_issue=5;citation_pmid=11340620;citation_doi=10.1002/bies.1057" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. S. Reece-Hoyes;citation_author=B. Deplancke;citation_author=J. Shingles;citation_author=C. A. Grove;citation_author=I. A. Hope;citation_title=A compendium of Caenorhabditis elegans regulatory transcription factors: a resource for mapping transcription regulatory networks.;citation_pages=R110-R110;citation_volume=6;citation_year=2005;citation_issue=13;citation_pmid=16420670;citation_doi=10.1186/gb-2005-6-13-r110" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=V. Robert;citation_author=J. Bessereau;citation_title=Targeted engineering of the Caenorhabditis elegans genome following Mos1-triggered chromosomal breaks.;citation_pages=170-183;citation_volume=26;citation_year=2007;citation_issue=1;citation_pmid=17159906;citation_doi=10.1038/sj.emboj.7601463" />
<meta name="citation_reference" content="citation_author=H. M. Robertson;citation_author=J. H. Thomas;citation_year=2006" />
<meta name="citation_reference" content="citation_author=L. Salkoff;citation_author=A. D. Wei;citation_author=B. Baban;citation_author=A. Butler;citation_author=G. Fawcett;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. Prabhu;citation_author=M. M. O’Meara;citation_author=I. Pe’er;citation_author=O. Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. E. Sulston;citation_author=H. R. Horvitz;citation_title=Transcription of the nuclear and chloroplast genomes during the vegetative cell cycle in Chlamydomonas reinhardi.;citation_pages=11-23;citation_volume=56;citation_year=1977;citation_issue=1;citation_pmid=838128;citation_doi=10.1016/0012-1606(77)90151-8" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. E. Sulston;citation_author=E. Schierenberg;citation_author=J. G. White;citation_author=J. N. Thomson;citation_title=The embryonic cell lineage of the nematode Caenorhabditis elegans.;citation_pages=64-119;citation_volume=100;citation_year=1983;citation_issue=1;citation_pmid=6684600;citation_doi=10.1016/0012-1606(83)90201-4" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. D. Urnov;citation_author=E. J. Rebar;citation_author=M. C. Holmes;citation_author=H. S. Zhang;citation_author=P. D. Gregory;citation_title=Genome editing with engineered zinc finger nucleases.;citation_pages=636-646;citation_volume=11;citation_year=2010;citation_issue=9;citation_pmid=20717154;citation_doi=10.1038/nrg2842" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Wei;citation_author=A. Yuan;citation_author=G. Fawcett;citation_author=A. Butler;citation_author=T. Davis;citation_title=Efficient isolation of targeted Caenorhabditis elegans deletion strains using highly thermostable restriction endonucleases and PCR;citation_pages=e110-e110;citation_volume=30;citation_year=2002;citation_issue=20;citation_pmid=12384612;citation_doi=10.1093/nar/gnf109" />
<meta name="citation_reference" content="citation_journal_title=Sub-cellular biochemistry;citation_journal_abbrev=Subcell Biochem;citation_author=M. T. Weirauch;citation_author=T. R. Hughes;citation_title=A catalogue of eukaryotic transcription factor types, their evolutionary origin, and species distribution.;citation_pages=25-73;citation_volume=52;citation_year=2011;citation_pmid=21557078;citation_doi=10.1007/978-90-481-9069-0_3" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. G. White;citation_author=E. Southgate;citation_author=J. N. Thomson;citation_author=S. Brenner;citation_title=The Structure of the Ventral Nerve Cord of Caenorhabditis elegans;citation_pages=327-348;citation_volume=275;citation_year=1976;citation_issue=938;citation_pmid=8806;citation_doi=10.1098/rstb.1976.0086" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. H. Willis;citation_author=E. Munro;citation_author=R. Lyczak;citation_author=B. Bowerman;citation_title=Conditional dominant mutations in the Caenorhabditis elegans gene act-2 identify cytoplasmic and muscle roles for a redundant actin isoform;citation_volume=17;citation_year=2006;citation_pmid=16407404;citation_doi=10.1091/mbc.E05-09-0886" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. J. Wood;citation_author=T. W. Lo;citation_author=B. Zeitler;citation_author=C. S. Pickle;citation_author=E. J. Ralston;citation_title=Targeted genome editing across species using ZFNs and TALENs;citation_volume=333;citation_year=2011;citation_pmid=21700836;citation_doi=10.1126/science.1207773" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. R. Zwaal;citation_author=A. Broeks;citation_author=J. van Meurs;citation_author=J. T. Groenen;citation_author=R. H. Plasterk;citation_title=Target-selected gene inactivation in Caenorhabditis elegans by using a frozen transposon insertion mutant bank.;citation_pages=7431-7435;citation_volume=90;citation_year=1993;citation_issue=16;citation_pmid=8395047;citation_doi=10.1073/pnas.90.16.7431" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="twitter:description" content="The nematode Caenorhabditis elegans is a powerful model system to study contemporary biological problems. This system would be even more useful if we had mutations in all the genes of this multicellular metazoan. The combined efforts of the C. elegans Deletion Mutant Consortium and individuals within the worm community are moving us ever closer to this goal. At present, of the 20,377 protein-coding genes in this organism, 6764 genes with associated molecular lesions are either deletions or null mutations (WormBase WS220). Our three laboratories have contributed the majority of mutated genes, 6841 mutations in 6013 genes. The principal method we used to detect deletion mutations in the nematode utilizes polymerase chain reaction (PCR). More recently, we have used array comparative genome hybridization (aCGH) to detect deletions across the entire coding part of the genome and massively parallel short-read sequencing to identify nonsense, splicing, and missense defects in open reading frames. As deletion strains can be frozen and then thawed when needed, these strains will be an enduring community resource. Our combined molecular screening strategies have improved the overall throughput of our gene-knockout facilities and have broadened the types of mutations that we and others can identify. These multiple strategies should enable us to eventually identify a mutation in every gene in this multicellular organism. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes." />
<meta name="og-title" property="og:title" content="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/11/1415" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The nematode Caenorhabditis elegans is a powerful model system to study contemporary biological problems. This system would be even more useful if we had mutations in all the genes of this multicellular metazoan. The combined efforts of the C. elegans Deletion Mutant Consortium and individuals within the worm community are moving us ever closer to this goal. At present, of the 20,377 protein-coding genes in this organism, 6764 genes with associated molecular lesions are either deletions or null mutations (WormBase WS220). Our three laboratories have contributed the majority of mutated genes, 6841 mutations in 6013 genes. The principal method we used to detect deletion mutations in the nematode utilizes polymerase chain reaction (PCR). More recently, we have used array comparative genome hybridization (aCGH) to detect deletions across the entire coding part of the genome and massively parallel short-read sequencing to identify nonsense, splicing, and missense defects in open reading frames. As deletion strains can be frozen and then thawed when needed, these strains will be an enduring community resource. Our combined molecular screening strategies have improved the overall throughput of our gene-knockout facilities and have broadened the types of mutations that we and others can identify. These multiple strategies should enable us to eventually identify a mutation in every gene in this multicellular organism. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
    <title>Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9Ncg2h-7bNESUAGNhIkOpfkrJW2nqLJM8j6kpjgstFI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459350","apath":"\/ggg\/2\/11\/1415.atom","pisa":"ggg;2\/11\/1415","ac":{"\/ggg\/2\/11\/1415.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/11\/1415.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1415"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1415"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/11\/1415":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xp+wJYYay8Vm3hsCbWTvtoCvFSrc51hX4s+Y9L5dV80sWatNmUqrE4i0XHOeHMul4PaD4hUoEA6mzPq2G\\\\\\\/oHLQQb27mwlatWHPYrYeQR7Usc\\\\\\\/i8ukOud0I+FXBgGKXcYobufzrgGpi6cuby2RuUHl+xGHLMq3KAO+NK8zoGzuNJxBDGs8Z7UQBNquKiJ7xCpbCMSJgJ\\\\\\\/WE3kWgI5o0eyuUSaFuJGHgtxQiqH9DNZUwmPNqX6BDZFzApEGe1\\\\\\\/i8DtUFos80A+Z6ZFuozbRObu02Xxzf3HS6\\\\\\\/hDBn4EYNKfGdC\\\\\\\/0yX5QbEwghoRrCoOrnhefop9zVvUkV87T3TxLpfQWeqh3eeOaCXNRy9c+EIcziT\\\\\\\/\\\\\\\/IDOwy4ZqTbW2Ih\\\\\\\/bVYbdEsCoohuCJADvnz+BdO0HlGEzDIXLEGF+MJE0xcN1x\\\\\\\/8u9PeAqoj7qQjXxx6sj8ZD5MLMsN8Tow\\\\\\\/Qqx4cTrO5DnnAoljshBqXokLm0G1QTTNAIK+8F3odKkdq6D8nGM8hwjOm\\\\\\\/vza0zGAmkLN+zVQPGDRic9ObRUL5DP17QdxrKR13EJ29ydyLt+1xvBaclhBPq5dsLVn3AbWeYb3ySzGdFhbLHXCbyB75eY\\\\\\\/2LOJTMQyG+8Kk5qbC78QcMdKp1DevC37j6RajRYuyreSrTejlnxb7ZjfLjrNL9IKy1zy1QIWRIq89kBkPkr4gQnb00b5yFxfqljfz+pbQWGnHE+6ClHuMl17225On82YjlNU\\\\\\\/mBhlQ48mmeP3DGwWungip83IMxAqEWYkIYli239XFyk3y7hEger+FWwuy0D4bAaesNP1GEZ1hcUr8NaZb+wto+ZGJftP6fG\\\\\\\/Ip7maMBpwZpE8kDSksna4iuAIWZN8cSM6bFJ7ckLveCTUGCZHSGAKyOZ50I+JaDM9ygjnYInTJdx7e3iOu8ENHF9V8q4H9eyPAyBJ1sIbMW8M4Bwod7V\\\\\\\/1wLSTFBL6UuUSVxYqTUa22xEamWEa4cxTwhgxfv9cusX5pLXZ8tb4dHoeNLpN0fdiV6Ttq7zxZmaOLzNvjd4WYbWVopRZ8IoTRIK7XZCSI+eLtUovwVFoXaK6ON03sZ9jtnEW2n9z\\\\\\\/ZnTuTdGSeT7fbeOjaokNneqUhcRWhtYgB5C5Zg0ZeMEGvTlmKYv6\\\\\\\/Zn1jk6OD\\\\\\\/r9SrmUJdHzRqdVWFA8Z9b2BRsfdCkLk6xD0RwP3IH7Mm+VAqvFsCKUp9vLlperuyMG+0uqJI7KTTnizbygqBlLWBXgm+MyJdErTOfmXml5\\\\\\\/XlBmBVPcd8uBS\\\u0022,\\\u0022iv\\\u0022:\\\u0022tTHF34RJ7vzhYqWFeApvbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002241cfcca4168bd859650bbe48611eecf0b67289f4bc198d22b45d86301b2b086a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LVmJUJXxWIiYMbvldG6tGGoXSIEDLaLbqwNV9gFC\\\\\\\/AZpbVZTAh0pN0\\\\\\\/5NiJK4nnIA\\\\\\\/zFagufMAOAY920PxM2InV9Z008KXwhoVAezlyOu3AeAbA5HY2y\\\\\\\/LRVxHC3wVOXB8her53WS\\\\\\\/M0WAoRVjF+3sC0t7\\\\\\\/gAzlFWW+r4XmBp1+4JoXVa6rbuYeyq4ulBEOHUlvgiVe0qzGPha3jV5U8s5MMgTDm7xHoStjJ7GsShGPTGzA2KPksrzcadlftkkfHhGuexnEfgPVs3dqgv99utKiRkafKVqrEBnSmoa7cHfESzuYXVlN3GG21Z7rzBEJFCUaaTRt4zwy++pptfNwAxF+\\\\\\\/uHvc0Fi6cKkjTKbragqbvgLRewt49vWeA78xRVW4jMGHLINtgZ9b0bIpnX0BW\\\\\\\/sDDQ84ZZ8wja\\\\\\\/HHSgArjtIGewJJvgE+A4erwq\\\\\\\/5\\\\\\\/2cy2+MNTqQ\\\\\\\/eoP\\\\\\\/h1LnUGqZtyk5BXeGgpCeGP9jr58ao8Tqkpqvh66hfgNwa1Ov7bCHC7vu93lEZdSsBTEZIayPUqmHajAdq9d\\\\\\\/a3uoSq4w2ImP9K5WEqydUxYstaGwGwmygeFMrOdTy7GksuSD5fDIINs\\\\\\\/P6qrUzaopcP8xfpaPxX36ofOwwW5nQ\\\\\\\/WjCNiTbM0klH+1Ovtpal\\\\\\\/Yo4BoIazDI3yQH3ToMetwgoV+plknpZgdFJqBHZMsTjRT\\\\\\\/+jNiPs+KnXBh0w9V6q6f3cn5gqGrPtc5NitvN8xWzEHhz+bNvA+ioWLrCLFMhufjBp7rIUYR3OOwkdKy8Z98ymOCVy8oK+6RoLHUs7FIBHw0Vs6IU320pUVkwssoZM4cOLamYEOukDNo5s22n43dkrZu0WYqP4gCmxPJgPn1R5ohccHOxmnWNdpd+xAIiKbXJG2pL7fcUYsJdgoDKvt2qOoXkoFHzQIE\\\\\\\/GFOD3YOvFxDuvjR7N533yKKH8r8NmpqXhmXzzpiKffvc+Sa4je6Y8EauVyKbsLfiln4OjJAYJCywXsEdSze3lY7CMP6kAV8UA8KKiF6\\\\\\\/8xezaVOuQ\\\\\\\/5JG6x8bEQcOZX3YkcPfaTkOearhJ4HsG+dWL0Y2wpvDoBGlYQKZT2EfmzaxsE+9NlkAzRlK7AusKGw3H4OhdM8+\\\\\\\/ipl4AreaEwT1rkbcw19QkL6UV5XmPrmzasQmgn1lRJ6dIAKM97+2hSxGlWiiiTpb9e3i7j4imGvzx0qLacTFDxDdh9BASUQjl3\\\\\\\/VMayFNluC2XQTsFfwGQPu8nkemYniG6fnTFQTZOS3TxC4ko64zDyr9mArRx0guoIJ9m8GGkxSph7z+PbGGX824+csxKe+XosAmqkhjlMyhivjpuJULCDph88Zxe5GYcK+xYcFXVKTx40zNcTPO6I45atE5IlaHs2kmYvG+qSYTJ1cc9A+uZQH+zToMdU\\\\\\\/KeNoJp2jmdy11WD2b0JQoXvsa2IKqL5I60IXHf9BZYRt02duBabHIS+1tMv3Prel4WsLz48kgj5LmRWnncGChsaqJc9tnPlmtoyVN+xOzmlvSjFoz\\\\\\\/KQQY9Dejs9ddm8YOd7FXRJS+kMAgeKa54Ebt5T8arF9UuCzLMVnlCEnL75uldr2tyotdDsqLwRCZL6g2WGRQatKm3sh7GpBhkiIdI2FLfe\\\\\\\/h\\\\\\\/Dh0RcN8mQaVjlzu+5gCloJ33T0hEJsyWnjJ7gEsP5W9m0YRvov43vNI1p7slIkjq\\\\\\\/A+LU1OAEN\\\\\\\/SrMrhkWko5lC85juuyInfuIifYYQSkYtY2n+wZvdmCDfkIXuwAl6lhkI0shrlg7MxY1Z4m6v5Q32CpUe1v7fJbbnoZAgY2iBQfNVxiUsmhrbAU5fjQiN6KvLC1Y9bDVrCvK3wAiMFgJtxQcOVdwPJZ8r8oGoxHH6F1kawh1kgfVDY+cLikE7eXDxy25A0EzrJejo0zjVV1CZ56qp74Gg9q\\\\\\\/jljd1qQD8WTK0W8QbenOApRDZ7okeJHWSS6mwXUFoBtA7VsSokR5t5FwA97DzGnQfO81Pj4TxlJefJJJLbFJcIruKvowIG8aQrhUe5EnqHTU6K1+qgSDgFnwbvlRZRxP7394BfUS+DbLWwEuaNBUWExW6SfpmvsM\\\\\\\/z4mi78NZIaEujuviaXxSuNNO7Rv5gXNjUq7+qVnDoEnDAJztHMEmEVTXBTjn\\\\\\\/VVWg7Yq698ZZv0M\\\\\\\/w6whBru+a8BYcbA1AdgjlUJ04Pe99NoeuIYps3DbMeUsId\\\\\\\/duZExvMpuBhK574nb3eqvnHYxtTcuK6M2p8+XkrnXRGkHr3++vrH5JkCnR\\\\\\\/c5FwswGTsX0pzt+f3y+4tfp9LeIsUw8ih74jfGUcQUpwYX+zTW28Qb8nBNMTDXtHgmNO3cc8tNLZhm3+B57WEExwpvc4XJ9bC7Q4KPwVnG9leyk0kmQUwG2UMBZb6K6YyVh7YJJToJEidCrYSWdNG+oAP5OWWFUDPTlAFOotTRuF15VzAv1P8vPlDhSisS+VJ1XXz5AnypOQcNdJznQsPaaanqH3qTT+gHCY0FBHO9lBXi6EetPMhm19mNv1h8UBcTG2IBeKmM2Q7fP3Gwd45mx2dOB3MDFH71cIKvizy3v8Evw8EMNZY\\\\\\\/MlSF0UzRn8Xk0w6vkuwwGBKO4Fn3TZLCN0JeSeuL1K82GJ0DUhn88eLPG3gtEx\\\\\\\/I57DLol9ntIvmXAO6fCUr0Ei+oGtLAFfYpYz48EebR8G0axD88TUfTMTsjwRL1T\\\\\\\/zo26iEM\\\\\\\/GabvrNEFyRwrtfrRlymOwF6H+zrp6J\\\\\\\/Go7qgBO+bmdgI610bmZkWyRRmmCA7+83IeZjjaNorCyJDZOSjUPMgo6\\\\\\\/\\\\\\\/2O6kQMY1sRgVcat8Hh7QmHwYuhiJ2\\\\\\\/a+wAGVFI2rcA6QED0lfF3D9DecpOTqiaQIuI9QYXJ7HZHgxBQlPkAfyrmyKIGj6TxnDHf5sib+mviheZLqHLaqMYPOpa749RjKd1o223et+zFy4Ci0HFcGOD+IWM+QmDpunySnTrPCsf0tJazRthh0Ba3v2ICi8y3JnytnFIDhTPTx0D1CS9D\\\\\\\/YByMT8gFnkc9ia6IZz29a+C03u4PmUPlJGyyYcbpnjcargQjP33hQq6+zOt7NdVXbz4JCWkV3HAeJjI\\\\\\\/hQhzypkfinwi49Ke7tKOfUwxETZNhk\\\\\\\/hJ+Ic0tR46imr8CkPPaKGg11ZTNBjwjhv1S+0m0LoE\\\\\\\/4IRHnb6ktAynxqpxQjaSqU3qtwJTO3zRPdVFUWhj5T21UUVSjeBskKmUPNCXOEpseoc552KUnONUGEOsb9J+4zt\\\\\\\/HYzJUjBF8SuHyWol7im\\\\\\\/0NNebCFsPbZ8t74cOHxXhKPpflCnTUZ+p2S4RicJQxCTPUAupy1TtEOqU03siZz97GsClgjG6QmVx4S1i0igiybOHhQeXufy01tpZ+uV5srOBOCA6k6kMyPTrglyqO+lZYgYZT7SN\\\\\\\/JaIulF+chifZ+timGYHiELaOpDtJMElYTbb25wNt7XP4StlLvQX2IlUXN\\\\\\\/5xlYE2DDduBEak8BIrseU0D0pVn8retyAaFkDhpRmNo\\\\\\\/QlDYQVtpmoh4b8719QWsxUem6QBQJNgNVWv7mJQAQV3q1QI2C6CO4oePOyrG6PQVQ6ICZAPKWXcKs+LwKyAbtmELnLg8aBe3yMsfJmtBt2oBQ8gc3vFMMKs3h8VM6atX2OTiLjHJ9XMS4eL3+sQbjF3TKPMtm8XaLBySBPo+t\\\\\\\/RQ1FBYc5GJ19VbwBNbEFo\\\\\\\/X3f7PAKxuOFG+geuWeyI9Ef34FThCXTo1Yy9wHKYV3JLL7IgoV3Z8Z5DModYSR\\\\\\\/YxmXXaQZ3SmS0r1o1dr382+vNFI0rUODNhkxi\\\\\\\/RXYIKUL08J\\\\\\\/wKFY4OXEhAG4lPsIeeTsGJV81b06tctFrmAYW+4KCIDghkeQcLrQaFncPKHvzqiVTFpEot\\\\\\\/ad5BrISIIcetQzMZMPA4s\\\\\\\/zljT0MFdDRuK0VWQWY3VgAihLDqGPXHmHePQzNYlYF5HBrPv5H0ahkyt95boz4nByZINqfz+F+uaB0lwvbYhEuSoUh\\\\\\\/fz63PeXhQN7uUXdrbNH552\\\\\\\/gXysJQMNcxq\\\\\\\/uafnkQUeC8miMGipVKVAAjF3DiZigl7hrwQ+egH61IozHwPhpmehRVfXlVj253StDZO5CP2tCGp5SK1RDb2wKsNpJQGPlDiYYmkRPkKE9LAXswgzpMMWa7EnrGSMXDbKJZcWeuSZV4iBGX0HkZ1O7IU+qCixqrjIFMXvnpycwYTyFZk6DdtqHrZfP97V6SusgO9sX\\\\\\\/BSi7zrOP\\\\\\\/ul4ekgF2q+tAH\\\\\\\/M+88r9BiVigSd+K5OVnddrn4NAFAgDlzintC5MxfjtOZY+3dKbCHPTZpMb6MuRV6\\\\\\\/3vLuvD64kmr6P+e0HGh69PGXjzGBzFnY9BQniVj6TgFRHg4PLurC7r1\\\\\\\/TQkJODg24Pi6Q17Jbs0KvAUirW61RAiKhj2W1RitIYlLKvh\\\\\\\/\\\\\\\/n8WcASDUnDK2GD7H8L+86gkhUKzFZbYGwEyghojij6esXswoj1QKcMi0xXgkLpI7vjFOQfPT3nZjeFHGOf7iy\\\\\\\/\\\\\\\/FB0DullzwtEhUiFanR8YkRRWN951mT9GrqAkVm0wdjITA87JhSlY\\\\\\\/rxgAwZ4uvNYr+oYD6AfvS3GFySGuiOAQ0twPUfgI4dWlHW84KC\\\\\\\/9fS2pgld6QlQ14tknR5DWv+HCLmVoNrBnLbCi+DkycbtYaHG8niIkidL\\\\\\\/ExSKTHLKu5fG1cA1RAjg\\\\\\\/5QfzzQSwgeP4fLmx2qrOAdjhLmSOCfMu\\\\\\\/YM3INnuhPeNEY257ABlGnP9f\\\\\\\/0tnW5nNYN3HCTB4hBU+NqXYkwOnA453xgErybl9gQ4pmwsi2hKDZcNWi\\\\\\\/VPzoNs\\\\\\\/zf8NBSN6fiL1ugq9Xd7ocTom2KmbwuBeouKV3WbipH9ws1nttWVzUQp9alt12wZIstD+gvPsiIPSC2XBgfJLjjam12HUf26b+kHPx9iAx7\\\\\\\/RhZETNgTH7UjJAxSoqkc2Rq7xifMVB0J7WFXwBQEy9qGxuetfaypzoaVrsNI7HG0fkUmKrdYg6RrYgg5dliyKkzRD3EDjhXr7aKkvxKiqfdlHrpRHUVinIRM1lXq80UVmdDVq\\\\\\\/dfCT7c6auxpd32NO4sfZlHSvoTark5Z3Tkk1WGH3VE1DPxwE8UtjycrXMGorW6iF+Me8Gm2lDk4KU7StYFFhYWQverO4Imrn1X+UnZk+3N4X956M4ISWIFTKOUsPLqrgNhlZNtZmvJTbdadtP5JWNuZWfq0AMAPMJ2BDeQcJHl5RZnvks3Yh6IM+ERfr+GBkbsVnj72fvG3UyrNtcxAfkjETfuAVmGHhfzEm+9UDIgLJltgBhZBCgEA6spMzE75Z5saPP8+OBNklrcZzSGsH4dGpiFe+OTPGbg77J+aok6N9m2yuSAidEhUXHwSQoAT5y3BCiqui3EDGzqzksuQFGSMbxUITTbTkxk9ghXg\\\\\\\/TVgoPvs2B3T3DEpT5nwcQyDTovF2JtZRtw+Endglt7wXVL63RpsXBNrK5k\\\\\\\/Dkdbmc2\\\\\\\/TL6khS4Tth8GfIu4mH\\\\\\\/Rq0rl20dAHTrNZKVHzGHjnWAOQBi+IApeycBL4ofIuI0ESQZj9xeG1ovIolH0F2GkrBrJKvayArphc2TbWlZSMj3GFRUxSX\\\\\\\/FWRu\\\\\\\/hTtcQGr\\\\\\\/Z\\\\\\\/FwEGQYYjqilYC5EFQl5R2v8XHQXyoyoWZ7n85U6i2MNPIfwTX2\\\\\\\/twVD5rh+TXh9zXraF+VRAK4qqQDxQo2Dk8KeTx3Y1FeTdddr\\\\\\\/EschD9g0rkUZezBhDwB1W4hsWhcrsL9zTqRAU2s5Iyt81ViQiEYn6lqAkB9JsdDt1\\\\\\\/jh2+L2zLy49Z3\\\\\\\/tWeGs4o0\\\\\\\/IVEnJVS9KPTyVLd9qlIz7FaK0fBoPGEt+hkURogsJZaiYd3nbvQl+NG5JWY8x2m68srM79xAsJgkyx+IVRmmJIDAh3xqvYJ+1\\\\\\\/upGwF4RWdph+H6zkrZgy3WRHThQwqZfq2fhrlpjDYWOec7pByz8e9vCqRnD0a7g\\\\\\\/2sXcGzma9gKwd3D+IGjIrc8Jj7jvGbf+zdNhznk9npy\\\\\\\/5EBwwT2UglzG0SL99q9djydholXCW7P\\\\\\\/V4fUSXuefB\\\\\\\/MVB\\\\\\\/sVL6gVqIdCOBLHxsT5IGsugavcamqhcAwT9z233KXJgck0ZT5Rq2fO4et3Bop3QpshELKjHwkhVEnqZ6M3NIB\\\\\\\/TLjDI4Lv5UDYDaVAKToeHsU8TSWS4liBOkJ824iUt+VBJLbxH75OCgapij0uxXZqJ4Sy5f2jTFSR2oGsM578437mEXs3FR2RRoTyvvMxFGOJ0K51thbv4Y\\\\\\\/QVEvWoLRHq89J\\\\\\\/bRnqTqkIigdKiBXW5RXsWNgpYCyq3oBdA8ntW9IcNXuXalsz2B\\\\\\\/SF5HydE3yi+VaYlSjRixxLlLuSVpbaYJIYQjxOpOtU9DXzPiByANjB+ZJJNteNky\\\\\\\/7U7obe+MDfmB9bZ4TBxB3jkX4HzcaP0qo2G8XfUpgalR9mNM7aFwoyJmaKmPAWAR\\\\\\\/qr31KmbxF0BMLeHKkggOgy2stLfeO0gD1vGJz2xm7oh3NCT4oYUZBykDO8Ft1wJLL3+BG2QxY\\\\\\\/UaWuWpm+RG9cNojEXk2Qm1VVeDt1cfJFGx+7AyYVaLO+D5cqbxWBdfPD3NcVDsSUqhny+E3dQAuhMOCitudVbcqtJLJ3buxk4QUqszHfbuI+yT0u2Fn79pQzAi6SsYIdCE7n0PJydIyrrBPm\\\\\\\/TYzGYlD0ZLAd+RBB1jcZ8h5yV56GhR2H0WkT08JnR3TQdJcrBR76WiG5GBqOlBLrizHUR\\\\\\\/qEWKYbjQcOX4nRg2jjl5i0awqgxFojrfLPRlf7KLTMMOebhT3NvhR6HmiyFQKWjvPuY2Y+\\\\\\\/rCMBh0NtxL0A+\\\\\\\/0Ej2izV81GwfVM+tpbdylbl\\\\\\\/jZMxNarEesFOuiy6O4i6CInouRodIUuBaFjQd2NpFEZFCT1MpsvlqIX0tVRlvqVinK+z7hZNUXueZWSCYc4zhAuPr7chYoKYMefg9\\\\\\\/F6AyjB6YlETA+vH7n4pcTlhOf6SaiNxiWvSe7yshbtwZYXj7wsB1ycDgXqjb6wMTQKQE0hBfaID1EUjwHTOYFp\\\\\\\/SQiyFxHegw7NTpMAuBzYIEfu883isU7EO6vVe+\\\\\\\/PJGZkCQKZp3LEoVCiiduHUKSZcPYISJATnGdtywmJ1pvv57bdqtkaIhP0KRTtbIC95MpL6EytB8tQSlnQzOTqs4GmCbJDnNCqulnX+gMtBqxCNE4N2Bu\\\\\\\/uLo55F2gOtZuSvgqOxahC0A5gj40JR0KZqmxeJ2GMLbDAElGpIrBuilTxQYttZHrZBjOMoezHTqsHTduMs+n43p4+Rf6nr1qXaCrQJRSFPKiiT9HtQELQ75QrSTn1piGsPmoSn+u3IjsQuyHDRVXYCD7hCix13KTf0fOCKP+EdzPmomHgg307jZOPCH1\\\\\\\/1NOltuBU7rOMfQWt+9pcPsaZwlfIor+lEEP7vnYeQeoP71IV4D4ZDbSZZL0tDSyH92J8jN41b0Bh5V6X1ck7HsmivzchGMTJDEaeWaGCAImxRyJUdyILMySrmWfx3wlvwx2I6XCWx2xeKxfh27quY1rqGc\\\\\\\/RUKUhJNUIanA9iRFVDbIZixkj8oSjxm6ufmh7LYY208iikNgDFSPeiHos7qVi4Ys8ZmjhAStPyw5bqX38gLl\\\\\\\/jrGYFMfXvd\\\\\\\/oXaajVTE8x06c3tbpk6hEGz2W\\\\\\\/P4U1s6kYlijHeNYP91aYduDg6o75nWR9dEl6rePW9kh3wYJ1tMmvsNeTHSYAjzeRjjs7ew+lY7ZfLcihg+bTa1Ju0lbDuXRKhSVDlnpDc1Awezd8DGuiO\\\\\\\/DUFkI4pB470ANFtapj7blOahfOr6WosI+RLbtVVXAAVIbLenaMH5rZu+uFtZCGlgv2skVzF6Nivy3jjM0RXR4umcJxKwQ\\\\\\\/8E3NTY0Ss\\\\\\\/YITVr4XTsMt3i83VoHHTxiQg0TJOzWykEwbCCi4yfkekGzbsLZMOEalvb1fCM\\\\\\\/aonZfISzpT2sG5NOg3Ps1zqsf\\\\\\\/jNQ9I7lkoc3ATzWABMzPvfmFbMeabx5hs\\\\\\\/eTBKaPPaLyDcU2f8EaMZY3StJH0Cir1TTK26FKQgNEgDojRfpcb61s71Dg8EvlHRFYoDq7Pepx1AEVEccfVi4wuA93mdEZi6Jxhdy6MvFeKisqF4jgJ0q80loNkcujTOCOcw36Jbms3wLxe6iIkH8C4r9AJ59ttlHzGYOHhl1bF+Lw2RbR6btckdGeMf0HMGUPosNUquD8NkAZLZ\\\\\\\/2727Z7b3K8cqePWLce85O994PBxJiqxJvAHklraAKa0sm0Yir\\\\\\\/IjOzALwUvqqzClElY+AZ7ThfOOasNQcW1V+HC8dMPxWpkg2tVlQnULJ0NI3N9HCQc0cxVZS46a+mKwFWIssCohDhO+qCK74fBDkSbEAjwjg7cTJhID3rGpEHbmtvKLhPukjYMw9Lx\\\\\\\/m64YNEFc4AWqMn7BK6HNhE5a5G\\\\\\\/kggrYktFqeImInRI9JnQaHtB18XK1wCHygVK+Rmope5MebvMWl37lBaDwzB0K6Fr1Upcj7aiH4VQcnZ1Og44SPzck6Z\\\\\\\/VnOBpBtx8DpjKhdnqIUQn82K1LH1ZiSZyP1o8obDrimMCw7TIAGQxyOaJkHDoO3Z+kov8NJxjBl1LmB\\\\\\\/ASj9XNU2j45fF36xomWGdvr5bBCiwgp\\\\\\\/BX2kz4Bg5LSjfnfeFeWBHfqikQbb\\\\\\\/rSj9Cic0c26XEiFKfJCyVvDiIrMR86b+iqe4MunWzQINGju2wwmpEYNY83kicW+rFn6u3akjcygDjx3NUUXvwSM8ryr70n3RHlqp4zeg6N1a0K7DDf0afaew7+JTPFxuIg3rxORDtx40LH+2wDabGH5F+fshIkLgEFg3tDkPD94zm0tsA+HUlRqg4BiHJX4wd+LH63BJPvwJcBCCG5bMCeRmc4eKfUn\\\\\\\/vKnJcj6Wn6WDM5PRJa2UZrKpOQD+tbNy89TahVXrePFXRCXu0m0LIvoIH54nBq2ac4YSnDNUf8Vp4eagPWSh2phR8XIAYY1DCzu0+8N7z0cNkEFrcUISudKHy3n0TMUAJZ\\\\\\\/0w1el7Zk+b4EuPcEIWPkexTUgyabmoPu+gW+jgDWR85yeHUy\\\\\\\/ftqInbtwh2\\\\\\\/0Xaz7wRm2zU5UX1kkHXSZRmMwbnumI6vpTzW4VFasAKI9A6dLOg44asfdjf3GGjQyr7zzXbYILGekNyT5v6Z7ZqhJP5Zk20727Vr6H28qg31RdyHji3HAX09+PEHaGpumaz1OZ\\\\\\\/5S\\\\\\\/odAIsL0kQ8U0x27tKWlUyjIcOKGQICgH1Hx82qV7hkcRxFFXPYvZYLuccQ1pKz\\\\\\\/exFb\\\\\\\/SXRQ07cm3RS4tKn2whzVQpAl0qC7n9WZkMxwwzD63dzKDN0n7pwKi5LVmekENOKCGsCVOgEq3JwQJb2zME14fJbdt9t5aaCuTJe5mnzfhPnOKuk6\\\\\\\/UogvC1TWeNeGJD8cFdhOGntrlCcFs0Zz6usFwInxmU9x1wI9OxJpZZ4Yjvqe\\\\\\\/Pv5w2T7gRpUxvTzAb\\\\\\\/1glzyu8ZyNR756caniBQaAxPBjzOeWM\\\\\\\/reOKaRDBza\\\\\\\/pnhcxFvKSuaCeYfQrAMqCymJMInJxI5AYeNyAlGfYNOGG3PtMcG66vE8pmY7p5isXOoN2ETObZ5KOEVEWGo0Z4pjWpWnOeAZquR3o+z+sLUvmLokx6dIPGjQccfY83WtckQVlDJn\\\\\\\/Ry+oPzyrpnwygfIAIszb1GeEGYuiaL7jbw\\\\\\\/viSuDtZ2u9M9R2NPy9vA9oxdVu6mVf\\\\\\\/Z1PL+4BOXFguTbLoYS1AQacuA9RTH+2kuGrjnm6fozm44qkbJYD3P2tQC08JUCNLxhiv3OTKQjOiRtQDnFQEz4TDiiKVSUrsfLtBU4BAvUJZC4mmjvlwGY5xu2+7XFvplXD4zFXry5tLRL4xAnktl7XGrTEv7AA1zyJnfjC4TVvSg\\\\\\\/YWpbbAqchs9LHEcSTpJDphqunamFHYPMm8dA7FPHWa2TlCyqVTrSOXKQ64pK+rhqQuum5cD2303GGjl9O6GpfE27j6NROZdek46RItzgNouIfRFLxx8mSqRryLzaGfiJDR39YZONVhYPgDqPM1vW4GHVvuR4tC7RpH9rUuIThF38sD9uJu0Hj0b+vCd8jR9JFy5zu43kcY7SecQtuK+vNjG5LdutsfhJHMz\\\\\\\/vusMywQ4IiLgfQk09Y71UJv4htl9rPcVFgFUwxt2f1TRQnBaWSFZTb6wItc32LAOsjwtlOOB\\\\\\\/0qrojRYc4ybprBC0lc9upphCAfxN6g3FZbcb2oI9KhsUMav25vy2O2GlFAzcPxFWEdKurtKmloDr0x8lVoAlAhgt49wZGNM6ANMXGEWfy2cUsvM3+X0+\\\\\\\/0LABJorKfeGHbU9Rjjafmww0d+K5iZLOS1eFtzOeGm6xZ3E\\\\\\\/3h6pB7XNhDeqF6fLpPjMkL25VDYTxQw4wm75uDa\\\\\\\/9waVBFXsoVFlcuwbCDlN1uOTu+VJC57Et2Yr53HYx\\\\\\\/t7IkeBHuzgxQDSehKqXIF\\\\\\\/rC2mdakeXXX0d+qfSBucslwLOr7Df8h1Vg2oE4xJAhQIaRbRbHq5T+FRWo4ErOINQlv8YmTK+1LPdJ9aCLa1ue14p0uy04JS3DzJSU4ko3PJf1nLM8ADLIwuLprXhaAdzDuorDcCayWFbmRKAEj0G4CIihmBvsC\\\\\\\/6SPFV2dJSFwuQ8l\\\\\\\/qkik4JmVOMpgZgePu8Ys17FgfGz\\\\\\\/8TUTdofdk4AQlQrfHbb2jlNNmZGcLRC4CynIL4fjcqbtPNH9kPDxL6o\\\\\\\/Yl0cJWkvzuMsmBQ6kM5oXNNadpEMn641qJ0R3GOvHeXUcaOTNBjhOnOQOulwnzue4QM\\\\\\\/utABmZ2HVM6m9ZWC7DCJW7pq6PyzqOjxPAc\\\\\\\/tPiD2bUuNVFdIEgK5HrMC1+M5XtCf4vMybvfX4I9k0BWbOFPzTc4wzXeXyySOr35Ti9cu1X6otF\\\\\\\/GUY\\\\\\\/JdChAHyOsGSnKdvbe8o\\\\\\\/VM7uCfL7y9kjrP36DbAwDNI3ngpNIGJcKl7end+MGXpptyjLDCDLKxseux\\\\\\\/d0dRbGCTAeZ5XuTzNLee2KwthxpRUc9jf1fCNakLbBhosmonCSTbW3cCAubUBtAxJKsuk9w9YK73jL2Af7Y+EowPJrvXnhRCfaJ0m4nhoCKrKzSReSzGeXZmQkDZf3PmJvzoKmuhk9a1cWSrJFku94eps2rNfR0ZEDmw7bcmwKPP6lr1hSroGveEvA5vqPrhn3is7hZrk7+rI\\\\\\\/R2zMoiJF9cANQS0rINauso9Evb1Lhp+v\\\\\\\/6sGVQ7WNxwEOclynfIck4Dr5qlqgRgf57qz7v85KtybmI5jiebZY\\\\\\\/JqH6b1Lt4u7OR76x6Rp7WyG5Bv4NYZ95tu9kkWL7Wjb90jAsBm4l0V3Ol0c6uC91V6JH88KKj2M3\\\\\\\/q1v6nGqUcWUXdUvPekHbnAD000y\\\\\\\/4sceeoFFcD61YnGASBnmdYgVNu5ZRUfLo7qD9wjh0ZRN3QOYQ0nGjsLBofbyUe9hyoNC8WoC8iDWsVNbzWazBa1gmoeztsMgrAFErOghjSxVWFHrG3njRCx4MhAYkpVh8kZVbpHe8P7U6r2MUaPdJprjKHeobt\\\\\\\/dIIaJOlsUn7OsVDkD+pPT3NF4srVBWjjNUsmZ4iafSYCN+4KljYz7z4o9Net6EKJP8WyF86OAr7YshatQO9gJNebpuxjh+Gd0nuobRAzcijoNoYKbL\\\\\\\/QHwZQ7lju0I1uA9YjB2nuRqP6mHCPnLSsgpcmWVd81XrvVsCX0l4\\\\\\\/3JczGQgizYshv0SpsZZgbEj1JyL\\\\\\\/hFp+CPr6HvLQyk6s6mUfGZH1M3cMeVVbOKwIHsh673qMkdNtYJ3yk9I8+MNnmTA+qRGTSJs68ineMIISFeWFrw9oRfruFYfKbNpa6TPyp+6LZ86eYGgG62\\\\\\\/4mk59apKOQIb0G1Hr6HuSQTCyMb7SpZVIy5lbnzgmzJfORfU2s963a+gT1YdfKViq8+L+wJ9KoQdsR9sgomIi+FmquruCqztnR4pSwr1+Vyi+uuNw8dyiLWhxAh+UBnXy8xgmPt2NWTUZnJwOdXZOFaQ17vmHiee6G6Zqwiidcxx7QqIpJOvflZk41Lkjh0TN258xSfanySXg3+ppbP4qzmMNTnIXtVWjwhJaOfpHVp05KzphuQBoSbni7AnjgSN4l50glvfIKVxlqNHvTxKG6+oTyyz0qz41ibrmIA7Ca0zHqcSzBFGoimuDydeDEltgAC8+B9cJ+ZAwYDGGPRHqm+JIeIY3a3OrhO2S6XcEVlQZGdvkJmm8247+qHMdY5XT6BSbp9dLJnfjrapT\\\\\\\/RLJiATl4iBl8TBxT3HKR2h4EoJYyw1oG7TXwam7V6rlZBRCnD0gF4O3Chi7\\\\\\\/KMvVhXtLEzRFSu\\\\\\\/Wehwy1gN1oAphO+dOH3hfxbZxYs0wxOTZFb3qWwKN7A6SVdr6JMob0T1JKD+icV8baloqMySc1mw0Mqsf7K4kxJD4kW5dU\\\\\\\/sN\\\\\\\/4hNzmAu8FFlBJpIItehWNtUzc11W33iUqDmgqKzs2LhRbqoxxM2ZN\\\\\\\/0rdoPr18ZKh7FONoysfJwRqVmm3fzknGnKcWCy6Yyij8uMfJjlXAXRyGlYFw5tbZtYIrc706oCrbwajnFqslmXeJkUL4lP+t04gOQPc7yeuOQJbBtjaHHCD3iDRTytta4z2m9VCVX4Iq+HYmyExbTvgUcprGUBrnYAWIB1vSPXadx529fKazho439qvYrAl3lk9BONNK3WA\\\\\\\/DcqKOnDKaLh53Pw5vDslxhaxWJGSlE4EsQyqEi5Ap09Gf7NJD2RoFLuLbEKFIuq5Z5Y8GzXFJeomGoR6Ja2GJD7kSDSzi5H5EId5N9xHHC9H7e+xlc0RtE1\\\\\\\/qlHYtJkghCBixu3\\\\\\\/flFe8WgTIs9FfbkLBMio0rdfhMPZG7f5OpJjVPBXe\\\\\\\/vbLMM7ROAm1A7UxjlEjClrzzq2bi1Et2keSZhdb1A6ZxEiTWIuI6NgDHjS6p25Si38znxr7yQk2eSTVfwUc2VKtVLq8kU\\\\\\\/QLHK1fwY04M4OxG7Y92pKFQ++8nDN0XLy4HPrI7edaHhgdjPgC8jl5nV1X9Ea1\\\\\\\/uy300RwnSEqGSOYLFmqiEfjkUkMJZS3zDLIye8x6Fa799lVoUxe4SGceoQn6SCVcuPJ08cIDtwhe4bJJqrtuMvu9f4raW8w2KAhe+Sr1ypmulhR9Lh5GYpjFq6voqtxp9kcWXnAyn\\\\\\\/2zTrPqhBSXPt\\\\\\\/IXx5a\\\\\\\/rfEi\\\\\\\/mC92jGzT0aAv8erLpHH55Zk1QV5jy4r88jIZsjNmtzDW++jlWucfiVA3MJg5YzEWL1Y4UbfE0R9lqQhlm45kASanQIkm+5Fhu50uXwM5bfpQRVoP\\\\\\\/OMHS9bbokidnan4DpCi6VZEny7IRANXkO34D3sHZOjIt5u9C0YuEV196GbJIpxXDAL\\\\\\\/EvlyjpeOl9ZSwmxDRG\\\\\\\/colgT6wK8RqnDS9x5Dl1ulpI5vum\\\\\\\/jXnC65qrSX52shf+50L0dZCTUHmcTHTArn6J4m6pYCo3ob+wkKFQdUL439bhAttmZ9KNZKA+BRtuPgzwE\\\\\\\/tKQWS7p5eMsDpTFkYVvV\\\\\\\/JVToJNpNR7dhCbJP2S63ARuTXpWTHa\\\\\\\/hxH\\\\\\\/a2M36BNC2+SGgcK7uhHSdpAEmclh8qfwo344HTYQ1tBc9ADT\\\\\\\/M\\\\\\\/GoWI2nrBWzrSdq830HN8WdXFzEeC+EU0kYMx+q17XS\\\\\\\/RYPUkCt1I9BOpADOEUgxSEy21smdR0Cgl8vnjq6DAatuxNwChYFfOSXyEUAvUQ3L95Rb+G0evMq0XfFiPcY+NDck0hi6M9PoSkD+g2quq7\\\\\\\/u\\\\\\\/sGCD0+05isbCTZ+ic+ehs9PXtL52CM7KA8KyFpHCwIzUuqwmR3McNFMOsRwc4pNkzQfunpZ4k\\\\\\\/owLK6ApFC8IJ8TXZk\\\\\\\/m3ZExRbjSWTDYvkDBgA5LACSBsRaxWOJpDGMYFFJAmLs5BFiQ5X7Hda5FGpjf7URnO9KyCtR8II1uxnR9v+CWhKM+c587QgT4bIDywOhXl0Gp9DUHcbsb8KSXfdAVqoilOcGO4Vs4IVHWgHBBsQ1UvRXojIb89qlpXwOj4eidNcWn0mNwzIu6Ofy8Fqg8B+dYRW4OXcPshhFjWInBowzg\\\\\\\/JOo4FOizp8ckit4GXuo+9mnyHDhQJ550EN9DzEfdIj76unDE+Ha2845kyEuvfBWLdUw87hM4YGU+bqa++qu4\\\\\\\/pGePk8xNNmh4jo8p\\\\\\\/Y7Qotq24O0V5egH0vMxrp\\\\\\\/+WwqfvqZCQvQQnGGfs4H09TlaPz+nd+yREQiACSQ9EOPS4ZQ2SeIGdV85CYVI5\\\\\\\/A\\\\\\\/bhll7gXa1kQPdhTigm555r7tTDVxHAY8rfkOu0oTUekBqElVM3wh3MgbvM\\\\\\\/NnDmr\\\\\\\/7IHdIE+uQlx1a82Lok9S3L4dSNyil2MQ+jicpxFZUHYf6UKdrQ4vkxyhTgsS5POGlmInfFhhIcK+dT0KwmDV95pDOg6C1NcCC\\\\\\\/p8HLJO2L1aVMYztS1baejEbF90n1Mo8eQIztf95OKDPyK0rKi2HAHTNxpjrRt9s6cB\\\\\\\/ps7PnT14GBLzgfpbCs2gAsIKXfPAH1toTU8KlItA71Pxz\\\\\\\/X+o1IH75ka1fmDQQRz9a1VhC8ZXtbf917BpIb7aQGjXc9H5TCc9yOX6vG3\\\\\\\/98j60r8I1FlOGFhglQ7RZnbD1bGna9erzcVbv996fBPryyN5Q9wS1Eh1GrZhlKEpBIiNShwCRxl4s4DQORkZ96KPCRr0um7Om4SAptTmmTm\\\\\\\/yrXHQtlh\\\\\\\/jpgoM1xE5X8XXW5cSAuRnt1yqVAwLFehYQ+9sOMa2gM9IMkU41EEe7REPOql9RVFoFeJj1Xg1CynhQzTMCwM7yNt2c2PNJp67JoUxowEhLz\\\\\\\/TmCGvjvVnaD9tw\\\\\\\/+DIv\\\\\\\/GNjPNAcqXEqinYwEacaA5kxkPBSuYdJvCZN+5c\\\\\\\/+UUT45twDDRSqjedJWMRL28JmBb+ZPAJ8SM1+LRz3Rs2rz4cRNgxDjDNharwvU3EcGF3K\\\\\\\/QvKlMnEy9ZBNru27ZtUHeYUvVY1BzZS6lj4Z7wQcom70RalFxz7n8imYwuLSTILhSguCt3yypZoeu65IMLaLIq+T7IwkVROnIi289+ZtOHYTs7jVCfSGMoD1GSHpMM699OvUWUq9iXcmBs5c\\\\\\\/P39Cbsy\\\\\\\/JQaQcYOv0eUFzcxTj5BY5ULJs2SnOS+gGgTKjalHBmCVo3FkmWbz4lqy7UcQby69KfvqybSluCbIPGpytkjusjhjkjVW7LKEL\\\\\\\/a8T3vSfW0gb+btpa9kqJcOxhtLso3o\\\\\\\/8Cql0xet3PAT6TmHTJ6HViElFPOF4z1VNyaD9d0UMeTQUNUmwDcWsnEoFOS3Gxf9W6grzbFMjKS41\\\\\\\/MOVMyBg1qyMT9f\\\\\\\/StIAXtNmTegc9R+cX2DeozZxmtMi562nxiNLm9N7pEYWmGLo95wdnZnHnAHH3SJAY+UmEHc9VgwG\\\\\\\/37btycbhXjgqUil\\\\\\\/Fxo4OxcBv2xF1Rzc3IJonpib1VtEa9Jh+J+vp1Slvwngc2rc39zR6ael8HWs5SUx10dZpx9J2iuXjdTMLrae3RP8LROTz+8veYYSVCvFHrzNOAY5cv9tjv0Wz0mtVF5jk69zBrmSUoDtNvopUFNnXiQc64sWCpiyHcDy79yRZ+oOfl8gmbX5L4scBYudeKkVwuGmd2LMiY4w9euPLXi7ilGePnbjXoEUWttLFd9koZjb7agozAiyVZGrxbJVJ7cyYyZ798jMQDQHLroXF30ruA7j3+UyAyMyFAFbXwE9ui1B02w0mtCWT4h1\\\\\\\/rLuGYMr9uKJg5aACBa1P0R+EyWxjkJ47tgU9bREh7+Lr8NSePd6n0nF+zmEseeuSf15v3Loioa8159blt10HadfAEUGoVnq0AOX6SVyGsaVxmyC7mLyPRWic\\\\\\\/d36TdMqeb2j6MUkAVEnucZjywnZemS+zmIn9\\\\\\\/umxms9yE6FFIL9z9w1HaoKbvQEVINP\\\\\\\/4uPkTKU9oRlWq\\\\\\\/hNXNDPqNkPVuWcifMy4suALGHtgL9Ui3thi2yQSWfiBl4Zww7w0wWEFb7TNjXlXzA6l1Sdgn2fuA\\\\\\\/SghyvoGU9FQibHbZvBgddfVFWLMIYW5YmCbvgbZiJdMOm\\\\\\\/\\\\\\\/mdbfw+zqGQXtQ0eK2C7okztJ71xd7b+7iyPq+66cA4vY+LM7lasY3hG6lC56k1jJXjjFSHj47bbGpJYKKExIA8gEqRRiDyGHNN3HGqJjAVOxZfXZXsQsr08CTUEtUWqWrhGcRY+RuBXVL94N9mq8AWHQCZvB4wacfqbVuS89Er2df7Fl2XVR7FN5iCvuCw\\\\\\\/Z9oeSgVS2t6dU7wht8T7EIO6bgw3rZkvdUfscfSP0zCYvrNF3o9\\\\\\\/abAu6PJCcBMtV4S74gb\\\\\\\/50D7BdyEHKPUnNAd6Mi7bKvAnn\\\\\\\/30N5y1pUOOW\\\\\\\/kMTdTV8f2e5pZHYdcMfjTPYxnMmKKENTDjaheDhgaE4RpEuPVIkAvH07i7rWwPsPb6iv\\\\\\\/ilxHtptIzu87ryvggIE8PuCjMZX8bf17bY+5uuupOY7CqU+BrkuxKIBvL+EnUKFpci1j153nyfyZ\\\\\\\/Er9sz7kUl3JMUhMCU\\\\\\\/yQJg8h6rku3BF++16YCng5bghjbpV\\\\\\\/g1buEwT3TlMyr32\\\\\\\/LQEmbYtFx4DiVmG\\\\\\\/L5VyDMgMFOI5ST1puxPzvga4cqq7hhzjq1zFJRvd5PABSDKRHwt3FnKXnFf5Vl5pHNWsEtVtOadlExavLdESnB23ZalC7de3dBJ\\\\\\\/HOnfrmd9vXyRVmaCWcpBsOOGGL6OVovJ8qYkC5itI+rptFr29qWBvFe21cdzjhb66R1eAkm2vwuLVI1fqdtmOnJd60khEX42ioiNYZznrNCCzeT8\\\\\\\/mY1uNvXxpvdMbzUxv6pGMrgKzdq0BaThHLq5UV2Vbkq6X0Z6Cmio7nijSoLuNZCcD74Ce\\\\\\\/yQiOA2AcFBvn8zpfzrfCXAfsEK2S0HmDMtlOlW8SPFr8Adakb3qrVKst9sQnUZv0LpuNiyVQ1+rW7qJi2MnIwAM6t7mBFwz9+We3wy\\\\\\\/uZC+qhwRx42hPgDE\\\\\\\/rSA62popxhPZf3iH8G60GKXPMV5A5G7LO6Nh103pJRqm6N1lenO5gNfA44Hyzi2cXZVBClf9lpbnt7ZS6z1bc6ZmRP2cHVjo4di9B4KHYzpEM1jSKaWPOVXGOs+ZS3MqITIp\\\\\\\/qK+YCeNfmav\\\\\\\/uKgHWn70bYHYEaNLaAteSPPWsjulA9OrOvBIvOaxZwcfufFM61auB+5ENcVVcFgjFwt0Ck\\\\\\\/L17Ngqyb307RHXqhZEnLjkTosZjQYhMcjaTNHobFjZ33ghZWQU3HmUywhAS6SKgR\\\\\\\/EtqbyG5OeLIiI+N+e6RrQrsxTP3RhxgSSHUuFAbH5d0OSfKaUecySS98YEK9rl18fdjiuvaJCK8Bx8MIMegO\\\\\\\/I79DiFegAjqJIpYe2jFw+Ny6oN9t9g1W9d0rIRJevUL3NbeXRmZPLbnFy3Hz90F4Uu7NdO524tZq9XyteWcqQPhZrArr3LhReEOEa5Q182ZwmnmEJxmbd7FEbkcp589WGJnHpzVFwzjvN4HE+PXhUu6xFqFNbdx48fSiCQQRENd0zgHF3x7xp4gSLZeRxysoOee9CEiYIOMw6TL1W3HfT2xPHSp41VUpmcRPRLJCwB7\\\\\\\/E7ejxLChmYPl9Ur7XMOjCR85lsw+WhNIW8xQes8c4A5SGN6DEpIESqhI6dOa0ZPuMHLg37bcfT0JSiiwMRayJQc2vU5KMoHG2yVbZXn0wpF2NdGqzU5y0XeSLkjQxu2abguMhbvrpqoz6GNEHNQTD5pqe6dNbO+60W9WRTf7mUiGnemf4aKh2XfDYBzQ10Smtn8ZzXyuadC3U3pPNi2weXR8yI7YCca8j13PmZhbPF\\\\\\\/54RhtAvzXubEucIUCroF4LDRCA+w+32xOIVrvmjIXMoRq+i8qTIs02fyg2dVEQwH6CJG60v\\\\\\\/dIYY9DOp\\\\\\\/Eq2kh6rkyIz1A9xhklPjcMETBMAhobooe3+S1teyoYgKP\\\\\\\/c+iBQofmOaFdpZaG2U5Bht4Fj2\\\\\\\/qqxGi1xrDpZ7XMibf92D3Lv3EM\\\\\\\/olr9e70hG\\\\\\\/BliX8lv\\\\\\\/wuyOYFQ9HZCTj3JOu2YOEgRFK3nNX78CFtuK2CXb0cB484nBkfflBI2442fkmTX9Zd3OeMonmvmoexfFGWAL+e5vP4uUJEdnh1ktHaYR6n2pFzgdNGIiUQjssIKK7YJkFv2MxvthAT8hrH71\\\\\\\/mDpfxM0xYVk61fDwPdAx4e8jN4A\\\\\\\/YUos6D01aYLlCQ5omICYUOb+woiHRUoyfEpcr9mzKQeaUBpgV\\\\\\\/jdcbPbq5OXbRux46MRny7uXSSoi\\\\\\\/KWZZPMjWw8rBDZUUzVhDSm16t0+d5bYB4punfR\\\\\\\/L5hdhnXUe\\\\\\\/5JetAaI6S9L3+5KrfAE83Xlh3dmn4ZbkMqIr+wa2KsoLtPoy0VHQNUNwyrmlbzedQBTHuMEovgLISQg1c8vgFRYVF\\\\\\\/xxAV\\\\\\\/I2AT3I8bB8k9ynduuQDBbHZGMvWr3PsJw8jBpZq+VE3gH7I2d2jJcn4\\\\\\\/D2JKioJxau0dDxPkgnl6dAtea2LvCwev3qRLddVKl85YV6RjwrtZW++bnQUNM8RXfzN2OwVXVALl+VeNCuvIQFoXvOMNPjbOl7c\\\\\\\/hZR\\\\\\\/71jWbqS9ECaX9pNGb05jji8JSTWYy+LZWtlmCEceF5cMuHJZXevSilC1uiHGcxSd2BJ9xTn4Jcj\\\\\\\/NgKBc1u0esgeOwtMCmuuwGGoXdtHkkPqSp7HH1M4+3nN4wyHO6P7H7OoI3FaL57Ut\\\\\\\/ohEiiRq1+mLAFsiNaTWtJGe239yEdHN12FyjPKdCcYMf3yCIYDKNJbprj0CDC6YoEVcZrjTziqwBohCI24ussbMEk0HT+idAMFisHToTx1KjxFGnJILPjl70O2UIniw9KFp3B4wHCdGKCG4beY3UnGIkcZOMGijMwXP5LuJM5PReheFRrw7jenVD+f\\\\\\\/DFzBTIG90afYwenX3KZZa8awaojPYZxnjM6A9jDKoJGzJiT5U4oJmPvvz5SzzHnKPxMIeAu3sH8YkfURjho092aWjZPtiXekZQm+RxnfvZuoZhLTtXTBzW\\\\\\\/kTCRFdlv1YlKB+TIVVpsYVt6Q1TEvadAYuGtj+e\\\\\\\/7\\\\\\\/mY45xIXgwsm7yTzEJnJp7hZYsWkk6xm1PYjuxZvntC+FxwYWwKXSfiw7Iy37Ht6ZCrVwLBxV4JLYudGwLjKKqWOqaLsf29K8FhfymT37P9Y7yzuQV\\\\\\\/i1RNp3jWGZSiAubu5BnSUiARf2gXbQjy1dqBTKoKYcHvCGasu8R8vYN0IlpPT1EKK+evX1lan9oRINe178nendKhGoj\\\\\\\/3uVzLExqLY5r1suc8XHHKfsoW+1ThLVxUA\\\\\\\/toqqSQnQTn5JVMlpmeQnVWU0Fw\\\\\\\/qqgXJCxJjwAM1HevVw4sRl6IS5JCevH1P3Eg+gMCc1LsfH5i11TlQKzwD1bO+PuQSNSB27k\\\\\\\/9YCoBcQT4QGIHLOA6znZv7WipYMduO2IOUizrkBORKAeKpHTYwc7EWfqkMrHZ\\\\\\\/0DKgeEu6nrs05g8RhrCrpcD\\\\\\\/015xVld4d3JOzvPtB0p0xFq7gW4c2VfgC6iHDWWxrNABHVjJSc2c1zdMFd+T84EF\\\\\\\/g5sdVmWdBTxOfl35f\\\\\\\/UQ7GtuhqheazzwTSYdTXqmWlW3JlpfjlJc6RlJdTEciYOBga+N4fgTtOHd5RAU64Evj8rBS3ImQiHVC9fcPZq54e0POX2O+lh0m4qmEkosc9a+Ogn2eFrP1ayLwhWP1CpeXjK5CSyDBfELv\\\\\\\/0aTQDmA3ZSpbg7BI2Sm5Z5jqdPBhOfS\\\\\\\/mP865Hfy1X91TQirv105vLaIbFNkPwSRUYu7uFxIZ8kxj3Ewh7qAmOTxGIm2vnhBEVqNatruq\\\\\\\/mcnZEHyUN7Sc+YyoZ0JsCnjlMD0jlnZB7rEqSt4Yt5CWrent2hZnenZCL9qzxyTg6cQhuGfJ\\\\\\\/RpDYqTwZcWUfbOKQdAmRuWvRm6khK9d00eHKS7SIR4S10v4znZJVX8erb6sw6xFOel0h8Zjw0voLsScV9fz6ng5Rsm5EjKV3d0Wk\\\\\\\/7g67Q+HFlYtgrDdml\\\\\\\/US6r9+gzl4FZBaAmsm2AS7KFxn28N3OfVtp9HULVnhAxnOK5ptblAoE1BKlGxQS6tHbC1zDsbdsxcEnmqzqqRp5YZmvzkgmkvW5\\\\\\\/MLgXx+ObranVH\\\\\\\/r4cvqHl3I\\\\\\\/e5y\\\\\\\/fUlU57S3nHT2rvoThQa0TK+7o7K5IjzGMuFHWzlMLlu\\\\\\\/FX1jfLhvH2JhY41207TwiGhSDv6ecTbHpSQ8JYmLHAx5NqIVr1u4Lpg5uXG+KQYFa7ri6DCQ7we5Itqe4gDfBIkJ+Roe2UOio4UcwBmH+wM4da23CfRIPvCATCMuFYtgfdsrE8+UjfrDjPTUJegpzAtjbP4G6Clrx8pqhr3q4NcwMYs7KkB3bHM\\\\\\\/alnrrmFDb86B0WuzATi5ChaI15fOVJS6aeiDEEkr9GZAAmlY2nDQvckaY5fBFZ4ZYDbnoHhHUnaa+oBG\\\\\\\/slBiCJ9nJH9B9Us5Y5hP+5YTP6dg0VHOOwidhgxI1flJGxSXEGgHPQih1x0xep+PEQqM8xGPFFd6YmaicAbVIzQ6iyCoQF7e3LnTCI+fGDc1HZwx4IfGOG57D4BBViKOg2iEQZYxGYt5RDUbtLp6YGkZdcuCk7pIMBf2jt+7nz01X71GDUhF7CKdwkbMqsm6HPW5VFsLY5uo\\\\\\\/kKdshSCcjkfS7vDqY+5cuXTOfVue0U\\\\\\\/LkhtXEQDHWAUQRbMo6CmnYQsi83MgRmoRv\\\\\\\/Gr6pF+gQS5hhCtr1DJXOMudPOYx+d0VLe\\\\\\\/KXQzDGoOCHzZYJGLKIXAETOiZtZpUGya+\\\\\\\/a7DwD58RXPUWfwsxEI56krkdhZenRykAOtpIrce0eMRblPV5RFMWtLVp5gUNX7YwE5k6j1OXn5vFiVQXoc6sDklxk9uoYFnOE1gAQIe08lZf1IxEIijP4ulM8lD1JFK\\\\\\\/moQ78\\\\\\\/9nnV6wx+b23kk8USIDSXSqpt1zY10dE\\\\\\\/36d+t70IX3PwiV6vBUVUOHdY+jyH88Uce0rRyZcNuiBzgqYBARFsWPGqjkVcBvOBNWzHmQZtKJsYMEiZrIaUvBPgdCm\\\\\\\/EPKujEu143s36kslqennH5eBJbDKLoZag\\\\\\\/BQkEGjFnQfRUORYJAVhOu+FI\\\\\\\/WNuchZIYleUSNRHp9PRqvClhNvmDJll93NIkjIARxOMu0RuTT2h2SyOwXrsudvaS8SDmWCn1kucHKvM6z+PuUdD9JzgW0RMzWvPb\\\\\\\/fCnuJnz4UlNbUJuK4gbUBczd1JJZ19ZhE9JXZenuXILLNGC8mIFNciuvWcVPXjh\\\\\\\/aZQ6af2aqtrKyxe4qVl1d5Ai3S+sTj4+4MNAU1ubTW0E+8c5w7HKq2YRFaGsmVh7j7fleUUdZo\\\\\\\/fi0ncAVmQXdB+cVBZPrser1mBlSfCmgb0V2EfOlxpPcmHxG9tfhyOrf4y\\\\\\\/77ii1a3TNsLZJvXvvHIiPSHAL0FnRVZhrX8mY7ECkWOusQJvrrROzSZw3vaoItNxhfbqrhQcnPWjrpV8v7n1F2WG0S5WNdY0VuIPziDv95sfA5PHRgVlAWvo8AfbryGZ+sK4xIoSs8D6UDvhFiwIyIaRPqgwYVE3hwJuJiMKggglVL6PeeNDHER\\\\\\\/LAljGATJs+X5tIHbDsu\\\\\\\/HdJoA39n20eFdtODnqY0oXlDSi9ZawqFY479muynNilzAPRrReex0hvfFTp9eXvtYIh3+B1hVBC4PY\\\\\\\/JCjIgdGqlIYMO8pqqKqQuDGlpjtR13rN37vxAmHXPsGeHjHA1N4if2hV7B4zSXY0k0JDJx4Ek6owWpYCpZBF4oMKX+klOsUNXnMvshStkp9G\\\\\\\/uy2r69HQTvIY\\\\\\\/8TSNfGourR0zblpO20UA54LiUPF\\\\\\\/wDIvQs9DCwbC6MsopzaE67orFyinZjsM2dcJcfliad61gTEAXzCzTx2aMpH0GdhkjMRnuwgXtH+KsDDnw9VprFhmKDvLnz5mvy05aX1BSS1OcoNcf4rJkOwbUzwPUCytjNv5JtTJZpcBCJ5BmcLmDrebkdIiCje9kt+HEVwS+xxYGPMWjLW63u0VTiksZpPjwzX7RwSReuo9bsbETY3KTsHUelJCaSRShi4iY9k0oFU20INgGiCMLWRoOPi6cVpYXCZNF\\\\\\\/0J1QsI1Wg8Xn6F7iVZnUCcHChT+FtAqoksZQuVbnNgrhP8eQphdtw1kZAyp\\\\\\\/bpfWyZDwG5QvzJnn\\\\\\\/aK5P6yTMypCTFBFU+dGszTUef8IiokAe95SUhTBIlEznqiOevElo\\\\\\\/fdFS9LqeqJoe8Z+vwmGoSFSQlSsWi9EXMYsb4d6ofhiGNmeLLbAQ5\\\\\\\/HZ1qHlWu4rxWdqsLvy8oq3\\\\\\\/KRcbHCesvtayG38cRc3OiyDwyfesy31JZyBKBk+EpAria6sOUQF5tK7yo9I9TFiv+vh5VBsuxDLQz+peKsjODDGiTMU97ZLHh1A8Ysr70FQRAyhDjYJd8Upbm\\\\\\\/6Ru1uTmBCYTxz2BjkOOZPntwLDY6TsW6mTL0BYCX1Kge2MKnHsdBs89wpwyUmwm3epAyC86R6+9Bz6ms973xEcX2U7IB1SaMWqEhv4quqHR0CIwNslrJ0Sid9zY4feiMDtmvoIl3+wUV+evHj0JlD6wgMyT8BY3+lOXrC+uqlOraaIMF02Rwn6FJrPTSZrAp5YS9WoJO40\\\\\\\/GUgV92prxoU3hBZDLns3iNlsP9ZbB0bijmc3nOiajynTUDvKRQfHbnWZnbuAaLNU\\\\\\\/0lqghRN3R5rsjhOsXxK2gtIoTjS3va0tk3Cnj1NueTKod1QTki32hIr0v5srU58ZPw84gd1LdHaAanOxKbCgQ668OIj39NOoPJvbCFDuLuvfluoKxWzeU7NrLAmOaOS4fa4Qx+0GSs3oy3Xpu39DayW7k65fw\\\\\\\/y5ppvKV1PA2ibaR4Xx7k4lBnrKGNecsVRPnFlsPnmnC6mGcJX+V6SefJzTGc1k3iwy7yOFQnEN5eeubMRG1IvdSDfoAphoy0Lfob8g6MmNsNcp1znceU31uqwGTSegpKua8P7AM8HRT2IRvmjD51qrdC7Bz7mBVznSptk2oWZajU3+DHiVoRPUzpeMH5UDwrMPYNM3yuow8Ui9HGHfd8u13eIRyOcp7G+VpU\\\\\\\/iOlGHSVCH\\\\\\\/dzKJp+R3ZDpus31B4MBKZwnhx8VZkBrp1ZNez4NO3n+9AHQytb8r4okVczVUy4qi9OXCvPg8HNjH+gQN3d+GGYfA7JcH6VprHVmYmvAlgwtpR9xMfSGqKEkL6qUVaQzKvcXBnTwZzgfnlcYXCKJFpkeV4XLM\\\\\\\/iVQlrcDqGoHUOhVDjjWVx8b8w2JFLht21ducvdc7SlbUSr0Z5DF3IkEjqNc24NjYjmd\\\\\\\/0Kl1KZQLwx4OtRGHagAaZw8ZtmBIjGDP4qoaTdnvp0wCsImB1cccp6RwK37Uo4s27vFGBlNnxYS\\\\\\\/dB0NMJad0SDBRBz18CSuUJMajHJxUunB9BnfttUWyFw\\\\\\\/PyWMddEVXU7JOdKuGAD4QfZzs2KIovPVBOGdrcqmkZWmmBVwZfZvViS4H6tLet0ElIgBq41AJg\\\\\\\/h96eAl3OIlHaSG2y8HJZfzEBFjzOS6v89DGign14OKmsYlYcvBLZACD4Kj8kuryZSEP8LzpdXXf9lv+oVqen3dBskJIpT\\\\\\\/Vhb2v3McYRg1bgZsQET\\\\\\\/Q3\\\\\\\/h14IN9OPllsZB00ygkcQeM4veDv7w488nCdIQQuN1BNlvMJTf+kdEmbibLHX6Vk6T\\\\\\\/uUGAizYilXzxg4F1\\\\\\\/WAqlJ+UAbBcvFXO\\\\\\\/9N1AgHPnTFoVnNi7qKFRYIDAp3epLWiVDYnh74om4HhXMYRsRxNUbC7lljanMpzKVzgSDzyblJeXUJK30dpURF0UqWwwpvfhTInlBt61CS3Ov2nK7dOkBufZZdVfRhNuiwTdLsjUl\\\\\\\/nWJjb\\\\\\\/trSB56krM6nGNtaL53nPKItbPy02X5f6UCZ3TkU22Bets\\\\\\\/OD5zNTvOGt9ZQZ8hai++8DMeL6+jfuVy4TdtrqOLNBushIVKdHwJEPl3m0MHWCSFp8DJsg9eLpu1ny1r2zVlZt634rUr3RQL\\\\\\\/K3vbOIBbeI5ulKmOXmG36vq1uNtaWAejxceJtR+f9GnDCkPZWcmMbVHNyGbvLS8W15vRA8k9XtvCk26DHx2EwWOLnPbLBFLc+OaEgMQqUslNubb92gXmzDq7Sl2rhYW8twMbzgo15A1hdH8sdzsyP\\\\\\\/QATD409d+nTpahnMR0FrO3eAQ1fS2T16qLeZS5EFLwmtDB6dJnwwM4bXtt4hd5gNgwJ4iuKuiAFQ6216uXfiiNWQaGY8A5sQ+wxSeHzCw\\\\\\\/voxcSx+squTs0o2Difq7kfuwVU9murYo00voLlH8DVegCwqqTkN45Uk11H1dzx82c36Dt2BJyJ1hw1nMWPeGHp4SpDrxN7vPRXUCMTJ4NDtiB4+hjYtR+oBxBZfffhOpnvdOFC3XDVq02s6YwW6tSa9zWZO1oGIntnToeU3gOdJNyixsLasV4Xp0ndD9jcTv6V3BrWZKva5FEgGV8icmX+ZMRURvZfjHifV7N5s0XgGm6qihByU1FqngFYU+MFGQPvQCw5Ch0TslfloYRCikd9fpKNl3Y7g6AkuwR26MnRpjUacjy72mG3H2UUYbzniILWUeqc2dM\\\\\\\/oxjx+oIsk3pGEtwadLXN\\\\\\\/HN0X3UTK43ISzwR9G7h5oA3Lo\\\\\\\/nV0BtPks8TZC34l7NS43Ddd+T5nf5vqka6ZBwTXUigpY1rZkyN3+Ipcw+pTdRYKnXyv1rW3uuDk8FbJw6gQ1T0fAQNRkt9mcP06iaHZiwexv5THczLQ5kfdD3WJ2fL5qE1b3rsDCkcIkFBvlDdgLKOObHFCjG15+PeupSX\\\\\\\/\\\\\\\/hId2YCmeAlJvnn2PIPasdp5I4Rk759zmmNueJ6Y1grpc+zS2t+P8cvW1DH3Oz1d56jaRFmJUCIauDgj83E4Wt7GEP6QOrBkTO+gn2JVUR6rYq6dDK8yShubWweb6D2fKrKJch2O7V8lt\\\\\\\/Upa0Ao\\\\\\\/QWrmb9ON2bbQDLm2D8uzkXfib1E8965mCbPoND5na3cfqBGk5DreHJw8PsYYprWBwoZKQblAmX+lbnBznOmw2ZwsH7GP56uKOIhzP\\\\\\\/s5n8m9aFk7AZdFPBFbpStH3UKeQip3jVZLjnvvrDx3+Xe1sKU0V+2WDt1eRAyM0UR\\\\\\\/lEQhXHZb1mUH\\\\\\\/N91dEhdVn33fgKSiLOVyh86H6llMs3UsmwTwzUsD4dxDI9dkVXA+UPgR+CB2BJnavpPADhex5Qjcqc3Ah4lH5U+xWcfMlR0Y4IHbXz2z7VGVFcCZagSMbx3FZkwLcsUXmj0I6aXEfHOB2L3l+1rTjKGt94Ix4flbMPwz1oZ6uI8vyjow+irbOJB3tVyA\\\\\\\/Uc+fy6dLioAW1bn4KPr855cX6xvFgb3mObTTO6fk+f\\\\\\\/vm2\\\\\\\/Uq0JEDUtgA6Plek\\\\\\\/TNZzlMqfwFfubr+rG\\\\\\\/Mzf5oP8KW7jHRqc7Eno6Zy13mqeaAmZ\\\\\\\/kmQuFfOpi3AwK+BeBIywiChkKs2QejVcNHAlmBYKVYCF9n5KA2XEYZnLjsGGsVDMtYRW18Cd2gFtSt4hjB0kfjaXd7ENQ8kypUZD1mraC0m+QuvuG5I5TXqaiqp654RjX93uS+m5y6kp2oZN3VNMiltqpWgRmIEoWi9ibwdcjaAsEqnOBjRwHPyywcB1fDmpyhfsjeZZq5iKWqzBPKlrTCzO8ja5tsPl0yEsSBtf\\\\\\\/GuX6TjVmqJ2dwAQntfauTeCPrgqoLlKJK8UiTt9EVS0SxnLvEZ9IHbIgpWmLSUqTf63jjAfcH4ZgbikWuJ0JusKkb8KSf\\\\\\\/c2gsXDd5v3sj8v5DnagrQXNOJOcD\\\\\\\/OCmy+0AVEl60O+UtqHUq07ErfCzdItbLwzT0foMaWQe0xAx6q3PR08mFTOpBAQA040PThJfshcP6+a5JmBcacvEKJsVKKfwUxCS7kyzu\\\\\\\/BO\\\\\\\/1jelqEWjsrNvyBb0StGNNm+SymLiCI1pT3toVufDt7lbXAlY+nYlCgSQA9oO+bISHQP1XfYh11loANbP\\\\\\\/i21EM4VSlnuSYt\\\\\\\/mEqwTVvrLUEoXzsjQPsXRfmUYuSTgh78\\\\\\\/6w8flerVCCLFxDbRFHZ4yqQ1CiW5mz4s\\\\\\\/ojsDk397FfrxrcgFjNOR++81OTZM7zfVAYJD7UWgk0TNDiqREtJkKaWTAvYtRDeWcPB+5Ej2C55Ya5Vc5pIjZON8YwmBRhfN0znN7B2Nh7BlXNWk1X9l10l2v4B91mQj0tcpNvkq9ff0GOZgFhSpbEsvoMOXcN3Fq2G4YMFvuLMC6wQSlfu+WzgLH1KNvm59eBlMLjJRDx0lOVBgtTenTgmT67tpvozf9LtksDoXg6p58sJk6kxIjwewKe53A0\\\\\\\/Jmq5uixy3Kt+dTfddrYQCm2wrUO0uZBsVWmTWqkNpnzEupGuXHiiJdzkuaMmKnxiZ5RDZv2PRQUHpfXhZCqpKJr2a2v9di3QAvhzPsprG6Rf\\\\\\\/M3udUQ5kc9ywY8xFmx9xuUo8961FCzEmx\\\\\\\/LpDjKAOjqF4rBIfuphN6eNz8hJlVmYO3fOsS2HQX0g6u\\\\\\\/znCSwCmS0ELl6p8sm2tng8vlge2sEhlaQNz8j66I8Lhp0TM\\\\\\\/RUwCfy09n0ZNv7kXym2aFZNGWmAaXKZGY3R\\\\\\\/JUdY46Mc+gREH4k9HBNqVa4hatFQED44PdlMFCEWe9AenOywbxIEPh\\\\\\\/kDhIBjySli6lMO9WXPq5Is1IyNjWOp0MtLP\\\\\\\/JOa8lSj6bkFm+gx81FxcM5tvC+fjjcp4o62abklVroShWJmG8+tfSnm2r2rkHzRF76XTlwbxZ8QmA3glK\\\\\\\/8QFg9AwCasAuST9NwsZW2Guiko+IipFTRkPOg3zVYqRbGmoRb5AxGtciGJ5b7\\\\\\\/gayf3htPJbb0DeFDpgU\\\\\\\/rFp3jQHvLmEgURfv19i3B7YB24Dz2UV5S0vqTm6w4N+2TX1BHgs8we9nHzNv5n2xdOeOTTi5YaY2gOUBvz95\\\\\\\/ifPphoOdntyKyTSIyJ87qr1BlRp3Zq4JE\\\\\\\/KDJOiyWbHvqm5\\\\\\\/giC91bf1VFYKRa3s+jQMTEj6vbbmdJFVNgy+cLuYjx9G2HMitfFlcYcMMS0CmCcCKnJzfjR3OJnd00KEtf9oI8VEqmSpSu7dz4wiaLmnxBF16DV+Ch61lJnMa5MErfotLHNtSmUiUtYfiVqae2SMF9lX4gvjZW0t0m9W6nNzGKsKrOZz8id4rBCImzvaQ+OUq1UIhZHWuYL1cybkHsPJYCbywGXOwi+nl38mxMy2lrQLHGzfAAubVvaNHmMF+nWSvfQTu+RpMojXygQFzS76MMGP5nXgk0zzisjNiCMK590GoZmtFaxLq+\\\\\\\/HNgQPg8k\\\\\\\/Nu6KQ2\\\\\\\/xsXNGHCxCHtDQASUPtG9BEvBbo1esZZJROYgbzAu4\\\\\\\/TmOHzSogOCwAk+4L+o74L6S4YUvMtvWbaJTGH9qxWQUyW6tg5WrcqJNldCVH4ophhDDW7LR2UfhCG3wkUMmhYnjcQw1WFOnws3mzynInSbKqb1lZdLA7dtTTBx5mAyCHNUxD+++cucj1goryr41ephG6lHrFlfN4Uabq12iupCS7+F9zeQ6UT+ewAIv+D\\\\\\\/Yylk4+xb8kBs1eSD7jRZgGmWCkjKX9hn\\\\\\\/O0FH\\\\\\\/nlGLn1YXheN7b5U6\\\\\\\/Jwdiru1Z94w7EGhydSYCGmEYJpGrcA3hO\\\\\\\/fo07R4tePTWkVQYbmG4lDbAN2+bLAGhYpxxzjZD6iXyLRfMf5fvYdcHawiDa6\\\\\\\/tIashV+LmlZa2beMEwuYfsb+V+5KBJcEiYLMechgRU1KWKsYIa7+gn9vGJ8TGUnWs7mYRj6M7blXyueT0rbTyYY\\\\\\\/vcrQvumqvHuSv3tmScqCNSZMhjGwA0hbAYz8qe2z9M53yp9Uc09HggXPMjpJ2wAT9aCasjS4XeuDj\\\\\\\/ZED3bEIO88UFu0Joovve8ZnV5S3u\\\u0022,\\\u0022iv\\\u0022:\\\u0022vMj4k\\\\\\\/N91255slEuHNtfsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220fa7436fe2031e73d978683a8c6ec03fd9678c8b376746a8558765854f536658\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WyY\\\\\\\/pSi3BqqEbt7eHpnbsWuznOHk4t83lrirmMGd1Ua0SM1aDsu2HuFMN6czTMV6HEQXVtCkMo8Xa9Ip5ICP\\\\\\\/Lg9cYnl\\\\\\\/Pb74iXyg4bNjUinnbCKyvbZa+W481LbvLzBVIE0\\\\\\\/LGKwv5Q0qOrW+kbHgHNp3BkSWhPjAqJQojoTqtRCEbJiD0ZxQ4ZzgGNp1roToWZpoOmUbDKH9QeqNgcyoSuYq1DcUxm6EK8wEWBtYEa8a6ipfT0m9lezUAMEbHUkPFPrmhE8t6QmjISYxVoJR6qYTn\\\\\\\/G2SK75ElbJBPID403lkxdeFNSROIKQ8q12S0547lWWqqMLvLxgx2r2y2PQfBj3WJjQ88DSzYfHz5l6+p3OlL+RS8EnTKThHjfsv\\\\\\\/GrlVOi6ugtvzh0pBbqVTWVx3G8y3nfpxODlyaVYctRd+NeKvNTmsNZICFq1Oc1WTU3IXrAYMkx5b2RsdUnJ4eu16iSq3g5+2PieyRJ8zx1hO9ie2CcPPcvBvAGTHA99QFlU2Dyc96+b7az0gRYff4tKKWjWIwAl6S9P+HonqLkDErmo8DsEdpKOafAwpELbPXrO8C8rey0S29mt+yk+lU0Eg8oalkMAOu7107RfEL8AJHZ\\\\\\\/cbLdRcun5dCJT\\\\\\\/IoNCu1BrfzjgpNIZZ3mj0IZ0xeXRpZ9H7CUbPyPE0LzjT0kHeDJX5q+v+iRuoqXjCMSyKCAXdf\\\\\\\/DiBsC\\\\\\\/vk3zWPUNEl8SSFwrDcHHBoicPI\\\\\\\/whtsKcbqXmgKavaFnxTVUOaJg3F+nt+OZGTqhisN1u0rKGAGyLoeKmVm01hj9fieNe9f1KEZZw\\\\\\\/zWWlm9dyybx3KxMXMqpf35975T4FUf+SnaFj2KFBHSKK5rE4IcBnZlO0QFjxFioZI4gRwNgctQf8JBl7FjAyHuLQ1J6Wm4XFwtTD44HQg1Cvow70uqral0kAfdMJUSZr5MGeYK88OzUle2QiIHZdSk7odJ2e71Kqy+GvGnMzMC1MGaEyZGFtjnAryrmZEdH+Jrz9fOyplOqmv0DykIgzLuwNeBBXeFUHAoMWc1UIKdEMvpAamhjUPgtxiZXOsAYSVNDsoGJoOBDkfY4qH\\\\\\\/eiq9jYt55WQ2spYgZPF9yaBuMNhIeEKqnHZ7aFrCm+qIdW1gxrpIFsGXYVkLG5mc6zf6bhMYgFel9wrGer4LPYWhDEJNWFn4XXqy\\\\\\\/11NJKlZwpXrlbvWsKuKFkf5Xhv1ZsC5UfnqdbrqpFKfzens3Oq0fR8F9FGY6Al6t2VVLyniuWd9NO\\\\\\\/HYVCIGUF9ikCD1nGx2bm3oSLlt8RLxuwjcXrYAnZ4Y7viKFzMPkJYdxLO7Mou4XXNP3\\\\\\\/kdkAubyIhzh5KeGRvtVarYnMYf6C2DcN7g\\\\\\\/\\\\\\\/3MCLynL8H6cLbpAopbkWuyWzuS391qOMw9ccqdE1eFaduojXwqsExtpACuhmFLcgfgZD0ooMG5ZLa+g0\\\\\\\/7Cl1c24XbVPLX3ivAkUh4vPKxhdtVtfTDCkhsoMN0A5X67TVXnRCGWnxIYh3l0Ea8lXPxAhr7jcsn52wEgtzn0KRwRG4O7WdDv6+gTZSzFYR3Kda\\\\\\\/QfYWwYv4VviaVWODhOECidBx9vv6\\\\\\\/3WrPr0dWgzKL+jHRNwZfeULPMUIWwrieVi+hPtGHoMHxuRrLcnA5fFk+Cx0UjV4p8AXboTRjgpt5q\\\\\\\/hH3bcXJlafm\\\\\\\/H1446CRhSGB6PcolhT6y++05idFgphAR65O2kTobuUsUMLfQTjEbEt5HOhs3fnYNdcvBJBWj55gp+8OPBCvXWJwLePNd4SX\\\\\\\/wOtFuiar5QLoK3+3BTXEs3Bh\\\\\\\/3FaFumE82zDgVufd9TQbQkcP5ocV2RXX2RuqVQr0ZQOskqF6zDiz\\\\\\\/VNQXKChrImQXioSv9e57cXSoNoZJnib5A7BaPHN4YTTcFFIJrHV3UloIA7Dm9ClTZ7LaoKeU\\\\\\\/BoqRFAu4QL3WHe8NZfcYr\\\\\\\/EiDnTrZecPUkUGgCOF8BSz0UTjs0IAOCVLjVFfiDnE9M0mI+cyNMnZt4CIwF4IdbbjOrzckeGZKJBwQ9nMH\\\\\\\/LRr4c\\\u0022,\\\u0022iv\\\u0022:\\\u0022X93Ro\\\\\\\/ebALsrkV+qTTPBzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226d20981b3933c0d2e52298ab222316dea159a34d31b52600c5092d325061ed1c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459350 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1415" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IBKG4YblTkUe2M6dVpBJMqrhutpjCH70OwmxPAwNyDo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1415" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AhsBzFIydSe3ZVRTuX5xHyEKmQh0w7FHNNOrqD_NX9E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1405" title="Shrinkage Estimation of the Realized Relationship Matrix" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1427" title="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459350" id="node459350" data-pisa="ggg;2/11/1415" data-pisa-master="ggg;2/11/1415" data-apath="/ggg/2/11/1415.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Screening for Targeted Knockouts in the <em>Caenorhabditis elegans</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">The <span class="nlm-italic">C. elegans</span> Deletion Mutant Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1415-1425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459350" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cell Biology, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma, 73104</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology, Tokyo Women’s Medical University School of Medicine, Tokyo 162-8666, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology and Michael Smith Laboratories, University of British Columbia, Vancouver, British Columbia, V6T 1Z3 Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-corresp-email-link first last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/11/1415" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459350" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459350/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1415.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459350" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459350/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1415.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459350" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459350/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1415.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459350" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459350/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/11/1415.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459350" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459350/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The nematode <em>Caenorhabditis elegans</em> is a powerful model system to study contemporary biological problems. This system would be even more useful if we had mutations in all the genes of this multicellular metazoan. The combined efforts of the <em>C. elegans</em> Deletion Mutant Consortium and individuals within the worm community are moving us ever closer to this goal. At present, of the 20,377 protein-coding genes in this organism, 6764 genes with associated molecular lesions are either deletions or null mutations (WormBase WS220). Our three laboratories have contributed the majority of mutated genes, 6841 mutations in 6013 genes. The principal method we used to detect deletion mutations in the nematode utilizes polymerase chain reaction (PCR). More recently, we have used array comparative genome hybridization (aCGH) to detect deletions across the entire coding part of the genome and massively parallel short-read sequencing to identify nonsense, splicing, and missense defects in open reading frames. As deletion strains can be frozen and then thawed when needed, these strains will be an enduring community resource. Our combined molecular screening strategies have improved the overall throughput of our gene-knockout facilities and have broadened the types of mutations that we and others can identify. These multiple strategies should enable us to eventually identify a mutation in every gene in this multicellular organism. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/knockouts" class="hw-term hw-article-keyword hw-article-keyword-knockouts" rel="nofollow">knockouts</a></li><li class="kwd"><a href="/keyword/deletion-mutations" class="hw-term hw-article-keyword hw-article-keyword-deletion-mutations" rel="nofollow">deletion mutations</a></li><li class="kwd"><a href="/keyword/multi-gene-families" class="hw-term hw-article-keyword hw-article-keyword-multi-gene-families" rel="nofollow">multi-gene families</a></li></ul><p id="p-5">It has been fourteen years since the <em>C. elegans</em> Sequencing Consortium published the landmark paper on the sequence of the genome of this small roundworm (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14"><em>C. elegans</em> Sequencing Consortium 1998</a>). <em>C. elegans</em> was the first metazoan to reach this historic milestone, and as such, it offered the unprecedented opportunity for a comprehensive genetic and molecular dissection of the development and physiology of different cell types and tissues and of the genesis of organs. In the intervening years, studies on this nematode have not disappointed, largely because of the accumulated availability of thousands of mutations throughout the genome.</p><p id="p-6">Although it was clear to many in 1998 that to fully exploit the potential of this model system we would need to obtain mutations in all the genes, the way forward was not immediately obvious. This was because, unlike yeast, there was no homologous recombination system available to quickly generate targeted gene disruptions. Fourteen years later, the yeast <em>Saccharomyces cerevisiae</em> is still the only eukaryote with a set of deletions for all genes in the genome (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Winzeler <em>et al.</em> 1999</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Giaever <em>et al.</em> 2002</a>).</p><p id="p-7">In May 1998, a meeting was convened at the Sanger Center in Hinxton, England, to discuss the feasibility of mounting a large-scale <em>Caenorhabditis elegans</em> project to mutate every single gene in this organism. Many of those present were working on strategies to obtain deletions in individual genes, and the relative merits and cost of each strategy was discussed. What emerged from the meeting was the formation of a consortium of laboratories dedicated to providing the research community with gene knockouts (<em>i.e.</em> deletions or KOs) upon request [reviewed in <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Moerman and Barstead (2008)</a>]. Initiatives similar in concept and ultimate goals to the <em>C. elegans</em> deletion project are underway for several model organisms. The international Knockout Mouse Consortium (<a href="http://www.knockoutmouse.org/">http://www.knockoutmouse.org/</a>) and the Zebrafish Mutation Project (<a href="http://www.sanger.ac.uk/Projects/D_rerio/zmp/">http://www.sanger.ac.uk/Projects/D_rerio/zmp/</a>) are two excellent examples.</p><p id="p-8">The <em>C. elegans</em> Deletion Mutant Consortium as it stands today consists of three laboratories: the laboratory of Robert Barstead, located at the Oklahoma Medical Research Foundation (OMRF) in Oklahoma; the laboratory of Shohei Mitani, located at Tokyo Women’s Medical University, Tokyo, Japan; and the laboratory of Donald Moerman, located at the University of British Columbia (UBC) in Vancouver, Canada. While differing in some details, the fundamental screening strategy for deletions by each consortium lab is the same: populations of animals are exposed to a mutagen, and polymerase chain reaction (PCR) DNA amplification is used to identify deletions at target loci (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Through a sib selection process over several generations, a single animal is eventually isolated harboring the deletion. Standard protocols for PCR/deletion screening are described in detail elsewhere (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zwaal <em>et al.</em> 1993</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jansen <em>et al.</em> 1997</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 1999</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gengyo-Ando and Mitani 2000</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Barstead and Moerman 2006</a>; also see <a href="http://www.zoology.ubc.ca/%7Edgmweb/research1_pcr.htm">http://www.zoology.ubc.ca/%7Edgmweb/research1_pcr.htm</a> and the summary of this article). More recently, we have added comparative genome hybridization and massively parallel short-read sequencing technologies (whole-genome sequencing) to obtain gene mutations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> 2007</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Flibotte <em>et al.</em> 2010</a>). However, for this report, the vast majority of mutations generated are the result of PCR/deletion screening.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCR/deletion screening. Worms were grown in liquid culture in arrays of 96. A portion of each well was harvested for DNA preparation while the rest was frozen for later retrieval of animals. DNA was pooled into a grid of rows and columns, which could later be used to address individual wells. Nested PCR primers were designed for amplification of target region. Primers were designed using the program Aceprimer (McKay and Jones 2002). Shown are four slab gels stained with SYBR Gold used to identify deletions (faster running bands) from the wild-type DNA (slower bands). The four panels are (A) the initial screen, (B) first sib, (C) second sib, and (D) third sib. The WT band for this primer set is 2099 bp (long arrow), and the marker is a 100-bp molecular weight ladder with strong intensity bands at 1 kb and 3 kb (marked by horizontal bars on right hand side of image). The screening image (A) shows two different hits (in duplicate screening, short arrows), one at just over 1 kb (18th set of four lanes, in first two lanes of the four), and one at about 1.6 kb (23rd set of four lanes, in first two lanes). The one at 1 kb passed addressing but was not recovered in sib selection, so the remaining images (B&#x2013;D) are for the 1.6 kb candidate. Through rounds of sib selection, one enriches for animals segregating the deletion band. This enrichment proceeds from initial detection in (A), a mix of hundreds to thousands of animals, to first-round sibbing in (B), tens of animals, and finally single animal picks in sib2 (C) and sib3 (D), where one has single animals segregating the deletion band. For sib3 (D), there are 4 &#xD7; 24 single-worm populations, 1&#x2013;4. Populations 1 and 2 are in the first half of the comb, and 3 and 4 are in the second half. Set 3 is actually 24/24 positive, whereas the others are less than that, so set 3 was picked to go forward as it was homozygous at that point. The example shown is gk3287 in the gene F11E6.1 (gba-3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1230738519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PCR/deletion screening. Worms were grown in liquid culture in arrays of 96. A portion of each well was harvested for DNA preparation while the rest was frozen for later retrieval of animals. DNA was pooled into a grid of rows and columns, which could later be used to address individual wells. Nested PCR primers were designed for amplification of target region. Primers were designed using the program Aceprimer (McKay and Jones 2002). Shown are four slab gels stained with SYBR Gold used to identify deletions (faster running bands) from the wild-type DNA (slower bands). The four panels are (A) the initial screen, (B) first sib, (C) second sib, and (D) third sib. The WT band for this primer set is 2099 bp (long arrow), and the marker is a 100-bp molecular weight ladder with strong intensity bands at 1 kb and 3 kb (marked by horizontal bars on right hand side of image). The screening image (A) shows two different hits (in duplicate screening, short arrows), one at just over 1 kb (18&lt;sup&gt;th&lt;/sup&gt; set of four lanes, in first two lanes of the four), and one at about 1.6 kb (23&lt;sup&gt;rd&lt;/sup&gt; set of four lanes, in first two lanes). The one at 1 kb passed addressing but was not recovered in sib selection, so the remaining images (B&#x2013;D) are for the 1.6 kb candidate. Through rounds of sib selection, one enriches for animals segregating the deletion band. This enrichment proceeds from initial detection in (A), a mix of hundreds to thousands of animals, to first-round sibbing in (B), tens of animals, and finally single animal picks in sib2 (C) and sib3 (D), where one has single animals segregating the deletion band. For sib3 (D), there are 4 &#xD7; 24 single-worm populations, 1&#x2013;4. Populations 1 and 2 are in the first half of the comb, and 3 and 4 are in the second half. Set 3 is actually 24/24 positive, whereas the others are less than that, so set 3 was picked to go forward as it was homozygous at that point. The example shown is gk3287 in the gene F11E6.1 (gba-3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1415/F1.medium.gif" width="219" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1415/F1.medium.gif" width="219" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461995" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-9" class="first-child">PCR/deletion screening. Worms were grown in liquid culture in arrays of 96. A portion of each well was harvested for DNA preparation while the rest was frozen for later retrieval of animals. DNA was pooled into a grid of rows and columns, which could later be used to address individual wells. Nested PCR primers were designed for amplification of target region. Primers were designed using the program Aceprimer (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McKay and Jones 2002</a>). Shown are four slab gels stained with SYBR Gold used to identify deletions (faster running bands) from the wild-type DNA (slower bands). The four panels are (A) the initial screen, (B) first sib, (C) second sib, and (D) third sib. The WT band for this primer set is 2099 bp (long arrow), and the marker is a 100-bp molecular weight ladder with strong intensity bands at 1 kb and 3 kb (marked by horizontal bars on right hand side of image). The screening image (A) shows two different hits (in duplicate screening, short arrows), one at just over 1 kb (18<sup>th</sup> set of four lanes, in first two lanes of the four), and one at about 1.6 kb (23<sup>rd</sup> set of four lanes, in first two lanes). The one at 1 kb passed addressing but was not recovered in sib selection, so the remaining images (B–D) are for the 1.6 kb candidate. Through rounds of sib selection, one enriches for animals segregating the deletion band. This enrichment proceeds from initial detection in (A), a mix of hundreds to thousands of animals, to first-round sibbing in (B), tens of animals, and finally single animal picks in sib2 (C) and sib3 (D), where one has single animals segregating the deletion band. For sib3 (D), there are 4 × 24 single-worm populations, 1–4. Populations 1 and 2 are in the first half of the comb, and 3 and 4 are in the second half. Set 3 is actually 24/24 positive, whereas the others are less than that, so set 3 was picked to go forward as it was homozygous at that point. The example shown is gk3287 in the gene F11E6.1 (gba-3).</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Nematode culture, mutagenesis, and DNA preparation</h3><p id="p-10">Nematodes were generally grown as previously described (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brenner 1974</a>), with various modifications to suit the needs of our individual laboratories. All mutagenesis was carried out on the canonical wild-type strain <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, with each group using its own subculture of the strain. Mutagens employed include ethyl methanesulfonate (EMS), formaldehyde, and trimethyl psoralen treatment followed by UV irradiation (UV/TMP), generally under standard protocols. Modifications included variation of EMS dose; variation of TMP concentration and the length of UV irradiation; preparation of TMP concentrate in acetone or DMSO; and use of various types of equipment to deliver UV at 340 uW/cm2. Genomic DNA preparation for PCR deletion screens was usually as crude Proteinase K lysates of samples from library populations. DNA preparation for other procedures was performed using the PureGene Genomic DNA Tissue Kit (Qiagen catalog number 158622), following a supplementary Qiagen protocol for nematodes.</p></div><div id="sec-3" class="subsection"><h3>Deletion discovery by polymerase chain reaction (PCR)</h3><p id="p-11">Our three laboratories followed a basic protocol that underwent various types of development, fine-tuning, and specialization throughout the period of its application. In its simplest form, the protocol involves (1) design and synthesis of nested primer sets to drive detection of deletions in a large set of interesting target genes; (2) generation of a worm library representing anywhere from 125,000 to 1.4 million mutagenized genomes; (3) sampling of the library to yield enough DNA for wide screening, while preserving enough of the original populations that recovery of mutant animals was not compromised; (4) preparation of population DNA samples by crude Proteinase K lysis; (5) pooling of population DNAs to reduce the number of PCRs necessary to screen the entire library for deletions; (6) screening by nested PCR and agarose gel analysis to identify pools containing deletion PCR products (nested PCR provides both high sensitivity in complex pools and high specificity); (7) population addressing PCR and gel analysis to identify a single population conaining each particular deletion detected in pools; (8) recovery of surviving worms from individual library populations; (9) recovery of single animals heterozygous for each deletion through a stepwise program of sibling selection (several rounds of expansion by regrowth, sampling, DNA preparation, PCR, and gel analysis at progressively lower initial seed density until single-parent deletion populations were identified); (10) creation of stable deletion lines by establishment of homozygosity or construction of genetically balanced recessive lethal deletion strains; and (11) elucidation of deletion breakpoints by Sanger sequencing of PCR deletion products.</p><p id="p-12">Various alterations to this protocol were made by our individual laboratories in several areas, including mutagenesis methods and agents, library complexity, use of frozen or live libraries, use of the poison primer PCR method (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Edgley <em>et al.</em> 2002</a>), and development of robotic solutions for various processing steps. Details for some of these variations can be found in published work (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gengyo-Ando and Mitani 2000</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Barstead and Moerman 2006</a>) or on the Moerman lab website (<a href="http://www.zoology.ubc.ca/~dgmweb/research1.htm">http://www.zoology.ubc.ca/~dgmweb/research1.htm</a>).</p></div><div id="sec-4" class="subsection"><h3>Deletion discovery by comparative genome hybridization and whole-genome sequencing</h3><p id="p-13">Comparative genome hybridzation (CGH) allows copy number interrogation of an entire mutant genome in a single experiment. For this work, we applied the method to several different types of nematode strains to identify new deletions: wild <em>C. elegans</em> isolates (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> 2007</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">2010</a>); balanced lethals isolated after mutagenesis (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Edgley <em>et al.</em> 1995</a>); unmarked lines resulting from mutagenesis and clonal propagation (“anti-twitchers”); and homozygous deletion lines resulting from standard PCR screening (primarily <em>gk</em> alleles identified in the Moerman lab).</p><p id="p-14">CGH protocols generally followed those of <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> (2007)</a>, except that processing steps for nearly all experiments were performed in-house instead of at Roche NimbleGen. For whole-genome sequencing (WGS), we followed the protocol previously described (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Flibotte <em>et al.</em> 2010</a>). “Anti-twitcher” strains for both CGH and WGS were generated and analyzed using the basic protocol for isolation of <em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene">unc-22</a></em> strains, except that F2 non-twitchers were picked in 1% nicotine and these were propagated clonally through the F10 generation. (Isolation of F1 heterozygous <em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene">unc-22</a></em> animals, called twitchers because they vibrate in 1% nicotine solution, ensured that resulting lines were adequately mutagenized, and selection of non-<em><a href="http://www.wormbase.org/db/get?name=unc-22;class=Gene">unc-22</a></em> animals at F2, the anti-twitcher screen, produced lines without obvious morphological phenotypes.) Homozygous viable <em>gk</em> deletion strains isolated from standard PCR screening (protocols listed on the Moerman lab website; <a href="http://www.zoology.ubc.ca/~dgmweb/research1.htm">http://www.zoology.ubc.ca/~dgmweb/research1.htm</a>) were analyzed by CGH both for validation of the deletion isolated by PCR (see below) and to determine whether extra deletions unrelated to the PCR screening target were present in the genome.</p></div><div id="sec-5" class="subsection"><h3>Elucidation of deletion breakpoints</h3><p id="p-15">Deletion breakpoints were determined by Sanger sequencing of deletion PCR products and analysis by BLAST against the <em>C. elegans</em> genome. PCR products from deletion-positive reactions were pooled and purified using standard PCR-cleanup spin columns (for example, the Qiagen Qiaquick PCR Purification Kit, catalog number 28106) and subjected to Sanger sequencing from both ends using the left and right internal primers from the nested set used for isolation. Note that unoptimized nested PCR typically yields only the shorter deletion product from reactions on heterozygotes, so it was not necessary to obtain pure homozygous samples to get good quality sequence. The sequence data were analyzed with standard nucleotide BLAST (for example, using the BLAST server at <a href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>), and deletions were identified as discontinuities in the matches between query and subject in the correct genomic region (that is, between the PCR primers used for isolation) and of a size consistent with the observed band shift on agarose gels.</p><p id="p-16">Deletion breakpoints were found to be of three basic types: clean breaks, breaks with one or more bases that could be assigned to either side of the breakpoint (“ambiguous” breaks), and breaks with one or more bases of inserted material (“insertion” breaks). Graphical display of breakpoints in WormBase requires a discrete pair of flanking sequences for each deletion, so we developed a standard for reporting ambiguous and insertion breaks. For ambiguous breaks, we calculated the left breakpoint at the rightmost possible position; for insertion breaks, we calculated breaks to maximize the left and right matching portions within the amplicon and to minimize the insertion size.</p></div><div id="sec-6" class="subsection"><h3>Deletion validation</h3><p id="p-17">Some deletions isolated by the PCR method were discovered to be non-mutant (various investigator reports, data not shown), and in at least some cases, it was shown that under certain conditions a wild-type PCR product from flanking primers could be generated. We undertook a program of deletion validation to improve the overall quality of the materials generated by our projects.</p><p id="p-18">The initial method for this validation was a diagnostic PCR, in which homozygous viable deletion strains were subjected to PCR with flanking primers to confirm the presence of the deletion, and a PCR on deletion and wild-type templates with one primer internal to the deletion and one external (the “diagnostic” pair - a product of predicted size should result from the wild-type template but not from the deletion template). Presence of a predicted product from a deletion template was taken to indicate that the wild-type gene was still present in the genome and presumably functional. Deletion strains failing this test were discarded.</p><p id="p-19">The PCR diagnostic was abandoned in favor of CGH validation when we began to analyze PCR deletion strains for extra deletions. In this CGH diagnostic, both homozygous and balanced heterozygous deletions could be validated if CGH probes existed on our standard array design, as single-copy and two-copy losses are readily detectable. For this assay standard, CGH protocols were employed, but the data were analyzed both for new deletions and for confirmation that the deletion detected by PCR was missing by CGH criteria.</p></div><div id="sec-7" class="subsection"><h3>Strain and data distribution</h3><p id="p-20">Strains produced by our three laboratories are made available to the research community in two different ways. The Mitani lab handles primary strain distribution itself, subject to a Materials Transfer Agreement, and it distributes both homozygous and unstabilized heterozygous lines. The Barstead and Moerman labs rely on the Caenorhabditis Genetics Center (CGC) for primary strain distribution, which requires that strains be stabilized. To accommodate stabilzation, a system was developed in which both laboratories did primary PCR screening and isolated homozygous viable lines, but persistent heterozygotes were shipped to the Moerman lab for stabilization by further sib-selection, leading to homozygous lines or genetic balancing of true lethal mutants. As a result, the Barstead lab contributed homozygous <em>ok</em> deletion strains directly to the CGC, and the Moerman lab contributed to the CGC all strains stabilized in Vancouver, whether they carried <em>gk</em> or <em>ok</em> deletion alleles. For strains submitted to the CGC, we pioneered shipment of frozen strains on dry ice to minimize handling steps and increase the number of strains that could be sent at one time.</p><p id="p-21">Deletion breakpoint data were generated primarily in the Mitani lab (for <em>tm</em> alleles) and the Moerman lab (for <em>gk</em> and <em>ok</em> alleles). We worked closely with staff at WormBase to develop a graphical display of deletion extents in the genome browser, and to streamline data submission protocols to minimize the time between submission and appearance. For strains submitted to the CGC, complete database entries were prepared in the format of their in-house system to speed incorporation in their on-line strain list and thus get materials into the research community faster.</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Targeting knockouts is largely driven by user requests</h3><p id="p-22">As it was clear early in the project that our efforts would be labor intensive, we did not want to spend valuable resources obtaining mutations in genes that would not be utilized by the research community. Therefore, we decided that our search for gene deletions would be motivated primarily by requests from <em>C. elegans</em> researchers. The wisdom of this decision can be seen in the approximately 1500 publications that utilized alleles generated by our group. Until recently, all requests were handled through two websites, one at the OMRF in Oklahoma and one in Tokyo, Japan. Going forward, all requests should be submitted through the website in Japan at <a href="http://shigen.lab.nig.ac.jp/c.elegans/index.jsp">http://shigen.lab.nig.ac.jp/c.elegans/index.jsp</a>. The only priority for screening is date of submission. Genes are screened repeatedly against new mutation libraries, with different primer sets if necessary, until a deletion is obtained. The only exception to the request guideline is that we, after discussion with our Scientific Advisory Board and the community, are making a concerted effort to obtain mutations in all 941 transcription factors and 416 kinases (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Reece-Hoyes <em>et al.</em> 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Weirauch and Hughes 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Manning 2005</a>).</p><p id="p-23">A survey of the 20,377 protein-coding genes in WormBase (WS220) reveals 6764 genes with associated molecular lesions that are either deletions or nonsense mutations. Our laboratories are responsible for 6841 mutations in 6013 genes. To place this number in perspective, in 1998 there were fewer than 500 genes with associated molecular lesions. The bulk of the mutations identified by our group are deletions identified after PCR screening for requested genes. There are also several deletions identified by CGH screening of mutagenized animals for either viable or lethal deletions (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> 2007</a>). We have included single-gene and multi-gene deletions in large multi-gene families from CGH screens of wild strains of <em>C. elegans</em> (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Maydan <em>et al.</em> 2007</a>, <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">2010</a>; denoted as <em>niDf</em>, natural isolate deletions in WormBase). We have also included nonsense mutations and splicing defects derived from our WGS pilot project in the current report (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Flibotte <em>et al.</em> 2010</a>). Note, however, that we have not included missense mutations or any resequencing data beyond curated WormBase WS220 genes. Our calculation of 6841 mutations in 6013 genes is exclusive of about 500 small deletions we have identified that are limited to introns. If a deletion does not extend across at least one exon boundary, we consider it a silent allele, and it is not included in our estimate of mutated genes.</p></div><div id="sec-10" class="subsection"><h3>Quality control and strain and data archiving</h3><p id="p-24">Once a mutation is identified and a homozygous or persistent heterozygous strain is established, quality control (QC) occurs at a number of levels. All deletion mutations are sequenced. As the mutagens employed can cause double-strand breaks in the DNA, the mutations we identify through PCR are most commonly generated through non-homologous end joining during DNA repair. Consequently, we observe a range in deletion sizes as well as deletions accompanied by duplications of flanking sequences and even insertions of DNA from elsewhere in the genome. Of 4101 <em>gk</em> and <em>ok</em> deletions, 1097 are accompanied by additional duplicated or insertion sequences. This additional DNA is often only a single or at most a few bases, but occasionally can be as large as a 2 kb insert. There are 87 cases of inserts over 100 bases in length among the 1097 deletions with accompanying insertions. Although there is a range in size of the deletions, they are all less than 3 kb due to the placement of the flanking PCR primers. Occasionally, we generate strains containing both an intact and deleted target gene. Based on reports from others and our own experimental data where we have tested 183 strains by array CGH, this happens in less than 1% of mutant strains.</p><p id="p-25">A bonus of using CGH as a QC step on deletion strains isolated after PCR screening is that we often find additional deletions in the same strain. For example, in the set of 183 CGH validation strains, we found 57 additional gene deletions, including 1 strain with 5 deletions in total. This result with CGH analysis and our recent finding of over 300 mutations in strains after standard mutagenesis procedures using WGS (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Flibotte <em>et al.</em> 2010</a>) should serve as a warning to those using the strains that the strains need to be outcrossed extensively to remove extraneous mutations. We urge and indeed rely on the nematode user community to remove extraneous background mutations before a phenotypic characterization of these strains.</p><p id="p-26">Once strains from the Vancouver and Oklahoma groups pass QC, they are shipped to the Caenorhabditis Genetics Center (CGC) in Minneapolis, Minnesota, for distribution (<a href="http://www.cbs.umn.edu/CGC/">http://www.cbs.umn.edu/CGC/</a>). All strains isolated in Tokyo are available by request from the Mitani Lab (<a href="http://shigen.lab.nig.ac.jp/c.elegans/index.jsp">http://shigen.lab.nig.ac.jp/c.elegans/index.jsp</a>). All three laboratories submit data on each mutation, including sequence, flanking primers used for PCR and sequencing, mutagen employed, and strain background to WormBase (<a href="http://www.wormbase.org/">http://www.wormbase.org/</a>). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> illustrates the type of detailed information relevant to each mutation available through WormBase.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="WormBase view of image and annotation for deletions. (A) Screenshot of four genes on chromosome V. Red bars denote deletions, and the length of the bar indicates size of the deletion. (B) The red bars in (A) are hot links to text describing the deletions in greater detail. Besides the details on deletion breakpoints, primers used to amplify the deletion region are listed. Depicted here is the link for deletion tm1530." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1230738519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;WormBase view of image and annotation for deletions. (A) Screenshot of four genes on chromosome V. Red bars denote deletions, and the length of the bar indicates size of the deletion. (B) The red bars in (A) are hot links to text describing the deletions in greater detail. Besides the details on deletion breakpoints, primers used to amplify the deletion region are listed. Depicted here is the link for deletion &lt;em&gt;tm1530&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1415/F2.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1415/F2.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-27" class="first-child">WormBase view of image and annotation for deletions. (A) Screenshot of four genes on chromosome V. Red bars denote deletions, and the length of the bar indicates size of the deletion. (B) The red bars in (A) are hot links to text describing the deletions in greater detail. Besides the details on deletion breakpoints, primers used to amplify the deletion region are listed. Depicted here is the link for deletion <em>tm1530</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Identifying and stabilizing lethal mutants</h3><p id="p-28">The objective after PCR deletion screening and several rounds of sib selection is to obtain a homozygous strain bearing the deletion, but in a significant fraction of cases, it is not possible to derive a homozygous mutant strain. This usually indicates that the mutation resides in an essential gene. Often published RNAi studies give us forewarning that a gene is essential, although RNAi analysis and actual deletion phenotypes do not always agree (our unpublished results). Where feasible, we use chromosomal inversions or translocation chromosomes to balance recessive lethal deletion chromosomes. As over 80% of the genome is covered by such rearrangements, we can handle the bulk of lethal stains in this manner (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Edgley <em>et al.</em> 1995</a>). In the remaining cases, we use flanking double mutations or chromosomes with green fluorescent protein (GFP) insertions as local balancers. In this way, we were able to balance 1436 strains from the Oklahoma and Vancouver labs. We do little characterization of the lethal strains other than to determine arrest stage (embryonic or larval arrest, or adult sterility). It is important when working with deletion strains harboring mutations in so-called essential genes to remember that we have not ruled out the possibility the lethality is due to a tightly linked background mutation. It is the responsibility of the user to do a transgenic rescue experiment.</p><p id="p-29">Out of the 6013 genes mutated in this study 1436, about 24%, can be mutated to a lethal phenotype. Of these 1436 genes, 946 are single-copy essential genes in the nematode. We did find at least one paralog for the remaining 490 genes. We were curious to determine the distribution along the chromosome of genes that can be mutated to a lethal phenotype. We found that both essential and non-essential genes are more or less evenly distributed along each chromosome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Regions of the chromosome arms where multi-gene families are enriched, particularly on chromosomes II and V, are the only regions with a somewhat reduced number of essential genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of distribution of all the mutations (black) and only the lethal mutations (red) throughout the whole genome. This figure is based on 6764 total genes and 1436 essential genes (WS220)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1230738519" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of distribution of all the mutations (black) and only the lethal mutations (red) throughout the whole genome. This figure is based on 6764 total genes and 1436 essential genes (WS220).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1415/F3.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1415/F3.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1415/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462008" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-30" class="first-child">Comparison of distribution of all the mutations (black) and only the lethal mutations (red) throughout the whole genome. This figure is based on 6764 total genes and 1436 essential genes (WS220).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">As <em>C. elegans</em> shares a large number of orthologous gene pairs with the yeast <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chervitz <em>et al.</em> 1998</a>), we were interested in determining whether they also share a significant overlap in essential genes. We used the program InParanoid (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">O’Brien <em>et al.</em> 2005</a>) to identify 1905 pairs of orthologous genes between <em>C. elegans</em> and yeast. From the set of 1436 lethal genes in the nematode, only 413 have an ortholog in yeast. Of the 1193 essential genes in <em>Saccharomyces cerevisiae</em>, 678 have an ortholog in the nematode. The intersection of those 413 and 678 orthologous pairs yields 193 genes that are essential in both <em>C. elegans</em> and <em>Saccharomyces cerevisiae</em>. Although this is not a large number, some inferences can be made from this analysis. All 193 shared, essential genes between these two organisms are involved in core biological functions, such as DNA metabolism, protein synthesis, and energy production (see supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003830/-/DC1/TableS1.xls">Table S1</a> and GO annotation), a perhaps expected result when comparing the genome of a single-cell organism to a multicellular organism. It is also not too difficult to envisage a scenario where 485 genes essential in yeast are no longer essential in the worm, possibly through gene duplication and functional redundancy. As we only found paralogs in the nematode for 111 of these genes, gene duplication cannot be the answer in the majority of cases. More puzzling are the 220 orthologous genes not essential in yeast that are essential in the worm. Examination of the GO annotation for these orthologs did not distinguish them from the group of genes that are lethal in both organisms. When we examined the annotation for loss of function alleles in the Sacharomyces Genome Database (SGD; <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>), these yeast genes often had one or more of the following terms associated with the mutated state: vegetative growth decreased, colony sectoring, viability decreased, and/or slow growth. Perhaps what is marginal and on the edge of viability in yeast is terminal in the nematode.</p></div><div id="sec-12" class="subsection"><h3>Targeting multi-gene families for knockouts</h3><p id="p-32">One significant difference between the genomes of <em>C. elegans</em> and <em>Saccharomyces cerevisiae</em> that presents a particular challenge to a biologist studying gene function is the expansion of shared gene families and the derivation of whole new gene families as one moves from a single-cell organism to the complexity of a multicellular organism. The degree of overlap in domains, the expansion of domain families, and the number of new domains in the nematode relative to yeast was first described by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Chervitz <em>et al.</em> (1998)</a> in their comparative analysis of the sequenced genomes of both model organisms. In addition to user requests for knockouts, we have endeavored to identify mutations in all members of certain gene families so the relative contribution of each gene to the function and phenotype of the animal can be determined.</p><p id="p-33">Actin and actin-related proteins (<em>arp</em>) are examples of small gene families. While the Arp2/3 complex has a one-to-one ratio of genes between worms and yeast, actin itself is present as a single-copy gene in yeast, whereas there are five copies of the gene in the worm. There is a combination of similar and disparate tissue and temporal expression for these five actins (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Krause <em>et al.</em> 1989</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Avery 1993</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">MacQueen <em>et al.</em> 2005</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Willis <em>et al.</em> 2006</a>). While we have provided additional mutations to the existing actin mutant collection, our contribution has been more critical for the actin-related proteins, where we have provided the only alleles for three of the seven actin-related genes. This still leaves three members without mutations.</p><p id="p-34">Other gene families with shared domains between yeast and nematodes have undergone a substantial expansion. Some examples of expanded gene families are as follows: protein kinases, which have expanded from 118 genes in yeast to 416 in the nematode; phosphatases, which have gone from 3 genes in yeast to 98 in the worm; helicases in yeast, while prominent at 17 copies, have ballooned to 85 genes in the nematode; PDZ-containing proteins, which have expanded from 2 genes in yeast to 64 in worms; Fibronectin type II domain–containing proteins have expanded from 2 genes in yeast to 47 in the nematode; LIM domain proteins, which have expanded from 3 genes in yeast to 30 in <em>C. elegans</em>; and MATH domain proteins, which have expanded from 1 gene in yeast to 86 in the nematode [all data from <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Chervitz <em>et al.</em> (1998)</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hutter <em>et al.</em> (2009)</a>, GExplore (<a href="http://genome.sfu.ca/gexplore/">http://genome.sfu.ca/gexplore/</a>), and WormBase (<a href="http://www.wormbase.org/">http://www.wormbase.org/</a>)]. As can be seen in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, we have obtained mutations in several genes for a diverse set of these expanded gene families, but we do not have mutations in all the members for any of the larger families.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462020/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462020/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mutations in multigene families in <em>C. elegans</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Mutations in all, or at least most, members of a gene family provide researchers with a powerful resource to study the functional importance of a particular gene in development and to determine its role in a variety of different tissues.</p><p id="p-43">Innexins are an example of a gene family not found in yeast but only in multicellular organisms. These proteins are functionally analogous but not structurally homologous to connexins, vertebrate gap junction proteins. Innexins appear to perform the same function in many invertebrate species [reviewed in <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Phelan and Starich (2001)</a>]. In <em>C. elegans</em>, there are 25 innexins, and between our efforts and those of the community, there are now mutations in all 25 members of the family (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Our facilities provided mutations for 22 of these genes, and in 18 instances, we provided the canonical allele. Members of this gene family are expressed in different combinations, in different tissues, and at different times in development (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altun <em>et al.</em> 2009</a>). Mutations in all 25 innexin genes provide the foundation to determine how these proteins contribute to all aspects of development. The cadherins are another family of proteins not found in yeast but that are critical for cell-cell adhesion in multicellular organisms (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pettitt 2005</a>). In the nematode, there are 12 family members, and 11 have been successfully mutated. For 5 of these 12 genes, the only mutations are those provided by the Deletion Consortium.</p><p id="p-44">Often a requesting laboratory asks for all members of a small gene family to be targeted. The transforming growth factor (TGF) beta protein family is a case in point. There are 5 members of the family, and although the community had already obtained mutations in 3 of the genes, we were able to provide mutations in the remaining 2, as well as additional alleles of the previously mutated 3. The nuclear hormone receptor (NHR) family with 275 genes is one of the larger gene families in the nematode. These genes have diverse roles in metabolism, homeostasis, and development, but only a few have so far been characterized in the worm [reviewed in <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Antebi (2006)</a>]. We provided deletion mutations in 245 of the 250 mutated NHR genes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). With mutations in over 90% of these genes, there is an opportunity to determine the role of these proteins in the context of complex endocrine and other signaling networks. Our progress in obtaining mutations in several other gene families unique to metazoans is shown in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>.</p></div><div id="sec-13" class="subsection"><h3>Genes expressed in the nervous system</h3><p id="p-45">Because there is a complete cell lineage and a complete wiring diagram of the nervous system (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sulston and Horvitz 1977</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sulston <em>et al.</em> 1983</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">White <em>et al.</em> 1976</a>), <em>C. elegans</em> has emerged as an organism of choice for studies in neurobiology. These unprecedented cell-mapping resources mean that, in the worm, the origin of each neuron is known and all synaptic connections are mapped. With the addition of a sequenced genome, it was revealed that many molecular pathways are shared between the nematode and vertebrate nervous systems (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bargmann 1998</a>). Animal movement, feeding, and behavior are all regulated through the nervous system, and this requires about one third of all somatic cells. In cooperation with the <em>C. elegans</em> community, we have strived to obtain mutations in genes and gene families contributing to nervous system function, with the expectation that this will facilitate efforts to describe in detail how neural networks control behavior in this organism.</p><p id="p-46">A group of 30 genes encodes a class of sodium channels named degenerins. The best-studied members of this gene family are involved with touch or stretch sensation (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mano and Driscoll 1999</a>). Within this family, there are 16 genes with mutations identified only by our group. The nematode also has voltage-activated calcium channels and other types of calcium channels, as well as a large and varied family of potassium channels. The community, with some support from our group, has identified mutations in most of the voltage-activated calcium channel genes, and we aided in identifying mutations in two calcium leak channels, <em><a href="http://www.wormbase.org/db/get?name=unc-77;class=Gene">unc-77</a></em> (aka <em><a href="http://www.wormbase.org/db/get?name=nca-1;class=Gene">nca-1</a></em>) and <em><a href="http://www.wormbase.org/db/get?name=nca-2;class=Gene">nca-2</a></em>. Obtaining mutations in all 70-plus potassium channel genes has proven to be more elusive. The genes encoding potassium channels are divided into at least eight conserved families, and many have mammalian orthologs. These families fall into three groups: 2TM (transmembrane) potassium channels (3 genes), 4TM potassium channels (47 genes), and 6TM potassium channels (22 genes) [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Salkoff <em>et al.</em> (2005)</a>; revised numbers from GExplore and WormBase]. There are now mutations in 40 of the 72 genes, but the <em>twk</em> 4TM family is still lacking mutations in half its members. The 6TM family is better covered, partially through our efforts, but primarily through the efforts of individual laboratories also using PCR deletion screening strategies [see, for example, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wei <em>et al.</em> (2002)</a>]. Further selected examples of knockouts for genes expressed in the nervous system are described below and listed in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-47">The transient receptor potential (TRP) channel family is composed of a diverse set of proteins involved in mechanosensation and other responses to the environment (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goodman 2006</a>). For this family, there are mutations in all 21 genes, and several of these only have a knockout allele provided by the Deletion Consortium. Neuropeptides are another group of proteins important in the nervous system for modulating behavior. There are 114 genes encoding neuropeptides, and these comprise three gene families: the insulin family of 40 genes; the FMRamide family of 30 genes; and the non-insulin and non-FMRamide family of 44 genes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Kim 2008</a>). The availability of mutations in at least 75 of these genes will allow a detailed investigation into the modulatory role of this important class of peptides in <em>C. elegans</em> behavior.</p><p id="p-48">There are an estimated 101 ligand-gated ion channels in the nematode. These include excitatory receptors for acetylcholine and glutamate and inhibitory receptors for GABA and glutamate [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Robertson and Thomas (2006)</a>; revised numbers from GExplore and WormBase]. The combined efforts of the community and our group have identified mutations in over 60% of these genes. It should be noted that targeted deletions for many of these genes are absolutely essential, as the knockout phenotype of most of these genes is very weak or requires specific assay conditions to detect. Mutations in these genes would not be detected in most forward mutation screens. This is true in general for most genes in the larger gene families, but it is an especially acute problem for the nervous system, where one is often dealing with very subtle phenotypes.</p><p id="p-49"><em>C. elegans</em> has an inordinate fondness for 7TM G-protein–coupled receptors (GPCR). As pointed out by both <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bargmann (1998)</a> and <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Robertson and Thomas (2006)</a>, this class of protein accounts for 5–7% of all <em>C. elegans</em> genes. Two types of GPCR protein are described, those with clear homologs in other animals (139 genes), and “orphan” or worm-specific receptors, a group consisting of over a thousand genes in several subfamilies. For the group with clear homologs in other organisms, we have obtained mutations in about half of the genes (74 of 139). The number of functional genes within the orphan receptors is not clear as many have stop codon and missense mutations, but estimates are of about 1300 intact genes and more than 400 pseudogenes (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Robertson and Thomas 2006</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Hutter <em>et al.</em> 2009</a>; GExplore). The estimate of 400 pseudogenes is based on examination only of the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol strain, so many of these genes could be functional in other wild isolates. Our group has provided all but 5 of 286 lesions in the intact genes, and about half of these are due to <em>niDf</em> variants in wild-type populations (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Maydan <em>et al.</em> 2010</a>).</p></div><div id="sec-14" class="subsection"><h3>RNA metabolism, regulation, and miRNA</h3><p id="p-50">A very active area of research within the <em>C. elegans</em> community involves the maturation, regulation, and turnover of mRNA. Proteins with RNA recognition motif (RRM) domains often regulate message splicing and other aspects of mRNA function. There are over a hundred proteins containing RRM domains in <em>C. elegans</em>, and we as well as the community have identified mutations in over half of them (numbers from GExplore; <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-51">The regulation of mRNA expression via micro RNAs (miRNA) complementary to the 3′UTR of messages is an active area of research with new examples being discovered frequently. The interaction of the <em><a href="http://www.wormbase.org/db/get?name=lin-4;class=Gene">lin-4</a></em> and <em><a href="http://www.wormbase.org/db/get?name=lin-14;class=Gene">lin-14</a></em> genes was the first example of this type of regulation [reviewed in <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ambros (1997)</a>]. Since then, hundreds of miRNA genes have been discovered in many species (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ambros 2004</a>). The current estimate for the total number of miRNA genes in <em>C. elegans</em> is 207 [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kaufman and Miska 2010</a>; MirBase (<a href="http://www.mirbase.org/cgi-bin/mirna_summary.pl?org=cel">http://www.mirbase.org/cgi-bin/mirna_summary.pl?org=cel</a>)], but it should be noted that not all of these genes are confirmed, as it is difficult to identify miRNA genes through bioinformatic approaches alone. Just over 40% of these genes have been deleted, and most do not display an overt phenotype (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Miska <em>et al.</em> 2007</a>). Although the Deletion Consortium has provided deletions for only a portion of these genes, our group developed the “poison primer” protocol used to obtain deletions in miRNA genes (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Edgley <em>et al.</em> 2002</a>). These genes are difficult to target due to their small size, and the poison primer method proved invaluable for detecting deletions in specific small regions.</p><p id="p-52">When the Deletion Consortium was formed, its mission was to provide targeted mutations, which at the time was the only route to systematically examine loss-of-function phenotypes. The development of RNA interference (RNAi) as a convenient and rapid epigenetic method to generate mutant phenocopies in <em>C. elegans</em> is a popular and viable alternative to gene knockouts (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fire <em>et al.</em> 1998</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fraser <em>et al.</em> 2000</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gönczy <em>et al.</em> 2000</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ashrafi <em>et al.</em> 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kamath <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Barstead 2001</a>). RNAi and deletions each serve a purpose, and often these approaches complement one another. Many laboratories perform RNAi screens in sensitized backgrounds, which usually require a null allele of a gene in a particular pathway of interest. Often the follow-up to an RNAi experiment is a request to a participating laboratory in the Deletion Consortium for a deletion allele. Determining the molecular details of RNAi itself require knocking out many of the genes involved, including Dicer and other Argonaute proteins, all members of the PAZ domain protein family [<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>; for example, see <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Knight and Bass (2001)</a>].</p></div><div id="sec-15" class="subsection"><h3>Transcription factors and kinases</h3><p id="p-53">A major goal of our three laboratories is to obtain mutations for all 941 genes encoding transcription factors and the 416 genes encoding kinases. Our reason for targeting these two large gene families is that together they represent two major levels of developmental control within an organism. Although we have not completed either set, we are close in both cases, with mutations in 866 (90%) transcription factor genes and 380 (91%) kinase genes (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). When combined with the current efforts of modENCODE (<a href="http://www.modencode.org/">http://www.modencode.org/</a>), the mutations in transcription factors should be especially valuable over the coming years to help dissect the transcriptional networks controlling development in this organism. Likewise, unraveling the complexity of intracellular signaling cascades will be greatly aided with mutations in all the kinases. The capacity for this type of analysis is unprecedented for any other metazoan.</p></div><div id="sec-16" class="subsection"><h3>Summary</h3><p id="p-54">Deletion strains are an enduring community resource because worm stocks can be frozen and then thawed when needed. Public-domain generation of knockouts by dedicated projects (such as the Deletion Consortium) and availability of the stocks from central distribution nodes (the CGC or Tokyo) reduces unnecessary redundancy that could result if the targeted production of <em>C. elegans</em> mutants were left solely to individual investigators. Actual request and distribution statistics reveal the magnitude of wasteful duplication of effort that has been avoided. First, on average, the targets on our list have been requested by at least two investigators. In fact, we have received up to 10 requests for the same target. Second, strains from the CGC are shipped on average to three or four investigators.</p><p id="p-55">The current state of our efforts provides a wide range of new research opportunities into fundamental questions in biology. A drawback for <em>C. elegans</em> researchers in the past has been the lack of tools to directly alter a specific locus. Our deletion mutation collection partially offsets this limitation. As well, several technical developments, including the use of <em>Drosophila mauritiana Mos1</em> transposons (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bessereau 2006</a>), engineered zinc-finger containing DNA binding proteins (ZFN; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Urnov <em>et al.</em> 2010</a>), and transcription activator-like effector domain nucleases (TALEN; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boch 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bogdanove and Voytas 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2011</a>) will help to circumvent these limitations and are already changing the landscape for performing gene deletion and replacement experiments. The toolkit for <em>Mos1</em> manipulation for specific gene deletions and modifications is impressive and relies on transgene conversion of a site after <em>Mos1</em> excision has generated a double-strand DNA break. Variations on the theme include <em>Mos1</em> excision–induced transgene-instructed gene conversion (MosTIC) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Robert and Bessereau 2007</a>); <em>Mos1</em>-mediated single-copy insertion (MosSCI) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Frøkjaer-Jensen <em>et al.</em> 2008</a>); and <em>Mos1</em>-mediated deletion (MosDEL) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Frøkjaer-Jensen <em>et al.</em> 2010</a>). TALENs offer many of the same features and do not require a resident nearby transposon. Successful gene targeting in the worm using transcription activator-like effector domain was recently reported (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wood <em>et al.</em> 2011</a>). The addition of a toolkit to custom design and make TALENs will make this a popular method to generate deletions and gene modifications in several model systems (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cermak <em>et al.</em> 2011</a>). In addition to these techniques, massively parallel short-read sequencing is becoming more widely adopted (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sarin <em>et al.</em> 2008</a>; <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Flibotte <em>et al.</em> 2010</a>). For an example of how this technique can be applied to obtain single base alterations and indels across a whole genome, see the Million Mutation project (<a href="http://genome.sfu.ca/mmp/about.html">http://genome.sfu.ca/mmp/about.html</a>).</p><p id="p-56">Over the next few years, the pace of obtaining identified mutations in genes will increase as these new approaches for obtaining and identifying mutations are applied to this organism. The combination of these diverse approaches in <em>C. elegans</em> should eventually lead to mutations in all genes. This knowledge will usher in a new age of metazoan genetics in which the contribution to any biological process can be assessed for all genes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">We thank the staff of WormBase, and especially Mary Ann Tuli, for posting and hosting the deletion and strain descriptions. We thank the CGC, especially Aric Daul, who have provided a home for this resource and have sent out several thousand KO strains to the community. We also thank Daphne Cheng, Justine Fair, Christine Lee, and Henry Ng for technical assistance on this project. We thank Eurie Hong from SGD for providing the list of <em>Saccharomyces cerevisiae</em> essential genes. We thank John Reece-Hoyes and Mathew Weirauch for an updated list of nematode transcription factors. Harald Hutter and two anonymous reviewers made many helpful editorial suggestions. D.G.M. thanks Douglas Kilburn and the Michael Smith Laboratories for nurturing this project at its inception and for their continued support of the <em>C. elegans</em> Reverse Genetics Facility over the years. D.G.M. also thanks David Baillie, Ann Rose, and Terrence Snutch for their early support of the facility. We thank our scientific advisory board members, Robert Waterston, Robert Horvitz, Donna Albertson, Paul Sternberg, Richard Durbin, and Yuji Kohara for their support and guidance over the past several years. Research in the laboratory of D.G.M. was supported by Genome Canada, Genome British Columbia, the Michael Smith Research Foundation and the Canadian Institute for Health Research. Research in the laboratory of R.J.B. was supported by the United States National Institutes of Health (P41-HG-003652). The Japan National BioResource Project funded research in the laboratory of S.M. D.G.M. is a fellow of Canadian Institute for Advanced Research.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX</h2><div class="section" id="sec-17"><h3>PARTICIPANTS IN THE <em>C. ELEGANS</em> DELETION MUTANT CONSORTIUM</h3><p id="p-58">Barstead Laboratory, Molecular and Cell Biology, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104.</p><p id="p-59">Robert Barstead, Gary Moulder, Beth Cobb, Stephen Frazee, Diane Henthorn, Jeff Holmes, Daniela Jerebie, Martin Landsdale, Jamie Osborn, Cherilyn Pritchett, James Robertson, John Rummage, Ed Stokes, and Malani Vishwanathan.</p><p id="p-60">Mitani Laboratory, Department of Physiology, Tokyo Women’s Medical University School of Medicine, Tokyo 162-8666, Japan.</p><p id="p-61">Shohei Mitani, Keiko Gengyo-Ando, Osamu Funatsu, Sayaka Hori, Rieko Imae, Eriko Kage-Nakadai, Hiroyuki Kobuna, Etsuko Machiyama, Tomoko Motohashi, Muneyoshi Otori, Yuji Suehiro, and Sawako Yoshina.</p><p id="p-62">Moerman Laboratory, Department of Zoology and Michael Smith Laboratories, University of British Columbia, Vancouver, BC, V6T 1Z3 Canada.</p><p id="p-63">Donald Moerman, Mark Edgley, Ryan Adair, BJ Allan, Vinci Au, Iasha Chaudhry, Rene Cheung, Owen Dadivas, Simon Eng, Lisa Fernando, Angela Fisher, Stephane Flibotte, Erin Gilchrist, Allison Hay, Peter Huang, Rebecca Worsley Hunt, Christine Kwitkowski, Joanne Lau, Norris Lee, Lucy Liu, Adam Lorch, Candy Luck, Jason Maydan, Sheldon McKay, Angela Miller, Greg Mullen, Candice Navaroli, Sarah Neil, Rebecca Hunt-Newbury, Mikhaela Partridge, Jaryn Perkins, Anna Rankin, Greta Raymant, Nadereh Rezania, Alexandra Rogula, Bin Shen, Greg Stegeman, Angela Tardif, Jon Taylor, Mariana Veiga, Tina Wang, and Rick Zapf.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003830/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003830/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> The <em>C. elegans</em> Deletion Mutant Consortium members are listed in the Appendix.</p></li><li class="fn" id="fn-10"><p id="p-64"><em>Communicating editor: D. S. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-23"><span class="received-label">Received </span>July 23, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-09-11"><span class="accepted-label">Accepted </span>September 11, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 The <em>C. elegans</em> Deletion Mutant Consortium</li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.1" data-doi="10.1002/dvdy.22025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altun</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High resolution map of <em>Caenorhabditis elegans</em> gap junction proteins</span>. <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">1936</span>–<span class="cit-lpage">1950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DAltun%26rft.auinit1%253DZ.%2BF.%26rft.volume%253D238%26rft.issue%253D8%26rft.spage%253D1936%26rft.epage%253D1950%26rft.atitle%253DHigh%2Bresolution%2Bmap%2Bof%2BCaenorhabditis%2Belegans%2Bgap%2Bjunction%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.22025%26rft_id%253Dinfo%253Apmid%252F19621339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.22025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19621339&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1415.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ambros</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Heterochronic genes</span>, <span class="cit-source">C. elegans II</span>, <span class="cit-edition">Ed. 2</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Blumenthal</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Priess</span>  <span class="cit-name-given-names">J. R.</span></span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.3" data-doi="10.1038/nature02871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ambros</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The functions of animal microRNAs</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">350</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAmbros%26rft.auinit1%253DV.%26rft.volume%253D431%26rft.issue%253D7006%26rft.spage%253D350%26rft.epage%253D355%26rft.atitle%253DThe%2Bfunctions%2Bof%2Banimal%2BmicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02871%26rft_id%253Dinfo%253Apmid%252F15372042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15372042&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223864000051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antebi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> Nuclear hormone receptors in <em>C. elegans</em> (January 3, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.64.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.5" data-doi="10.1038/nature01279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watts</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide RNAi analysis of <em>Caenorhabditis elegans</em> fat regulatory genes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAshrafi%26rft.auinit1%253DK.%26rft.volume%253D421%26rft.issue%253D6920%26rft.spage%253D268%26rft.epage%253D272%26rft.atitle%253DGenome-wide%2BRNAi%2Banalysis%2Bof%2BCaenorhabditis%2Belegans%2Bfat%2Bregulatory%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01279%26rft_id%253Dinfo%253Apmid%252F12529643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12529643&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180397600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avery</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The genetics of feeding in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">897</span>–<span class="cit-lpage">917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D133%26rft.spage%253D897%26rft_id%253Dinfo%253Apmid%252F8462849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8462849&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KT99900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.7" data-doi="10.1126/science.282.5396.2028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bargmann</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Neurobiology of the <em>Caenorhabditis elegans</em> genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2028</span>–<span class="cit-lpage">2033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBargmann%26rft.auinit1%253DC.%2BI.%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2028%26rft.epage%253D2033%26rft.atitle%253DNeurobiology%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2028%26rft_id%253Dinfo%253Apmid%252F9851919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.8" data-doi="10.1016/S1367-5931(00)00173-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barstead</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genome-wide RNAi</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Chem. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bchemical%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BChem%2BBiol%26rft.aulast%253DBarstead%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D66%26rft.atitle%253DGenome-wide%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1367-5931%252800%252900173-3%26rft_id%253Dinfo%253Apmid%252F11166650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1367-5931(00)00173-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11166650&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167051500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barstead</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title"><em>C. elegans</em> deletion mutant screening</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBarstead%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D351%26rft.spage%253D51%26rft.epage%253D58%26rft.atitle%253DC.%2Belegans%2Bdeletion%2Bmutant%2Bscreening.%26rft_id%253Dinfo%253Apmid%252F16988425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16988425&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bessereau</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> Transposons in <em>C. elegans</em> (January 18, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.70.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.11" data-doi="10.1038/nbt.1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">TALEs of genome targeting</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBoch%26rft.auinit1%253DJ.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D135%26rft.epage%253D136%26rft.atitle%253DTALEs%2Bof%2Bgenome%2Btargeting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1767%26rft_id%253Dinfo%253Apmid%252F21301438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21301438&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.12" data-doi="10.1126/science.1204094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bogdanove</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">TAL effectors: customizable proteins for DNA targeting</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">1843</span>–<span class="cit-lpage">1846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBogdanove%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D333%26rft.issue%253D6051%26rft.spage%253D1843%26rft.epage%253D1846%26rft.atitle%253DTAL%2BEffectors%253A%2BCustomizable%2BProteins%2Bfor%2BDNA%2BTargeting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1204094%26rft_id%253Dinfo%253Apmid%252F21960622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA1MS8xODQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D77%26rft.spage%253D71%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4366476&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T582100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.14" data-doi="10.1126/science.282.5396.2012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">C. elegans Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genome sequence of the nematode C. elegans: a platform for investigating biology</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2012</span>–<span class="cit-lpage">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BC.%2Belegans%2BSequencing%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2012%26rft.epage%253D2018%26rft.atitle%253DGenome%2BSequence%2Bof%2Bthe%2BNematode%2BC.%2Belegans%253A%2BA%2BPlatform%2Bfor%2BInvestigating%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2012%26rft_id%253Dinfo%253Apmid%252F9851916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDEyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.15" data-doi="10.1093/nar/gkr218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cermak</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Efficient design and assembly of custom TALEN and other TAL effector-based constructs for DNA targeting</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr218%26rft_id%253Dinfo%253Apmid%252F21493687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM5LzEyL2U4MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.16" data-doi="10.1126/science.282.5396.2022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chervitz</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Comparison of the complete protein sets of worm and yeast: orthology and divergence</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">2022</span>–<span class="cit-lpage">2028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChervitz%26rft.auinit1%253DS.%2BA.%26rft.volume%253D282%26rft.issue%253D5396%26rft.spage%253D2022%26rft.epage%253D2028%26rft.atitle%253DComparison%2Bof%2Bthe%2BComplete%2BProtein%2BSets%2Bof%2BWorm%2Band%2BYeast%253A%2BOrthology%2Band%2BDivergence%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5396.2022%26rft_id%253Dinfo%253Apmid%252F9851918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODIvNTM5Ni8yMDIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baillie</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riddle</span>  <span class="cit-name-given-names">D. l.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic Balancers</span>. <abbr class="cit-jnl-abbrev">Methods Cell Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bcell%2Bbiology%26rft.stitle%253DMethods%2BCell%2BBiol%26rft.aulast%253DEdgley%26rft.auinit1%253DM.%2BL.%26rft.volume%253D48%26rft.spage%253D147%26rft.epage%253D184%26rft.atitle%253DGenetic%2Bbalancers.%26rft_id%253Dinfo%253Apmid%252F8531724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8531724&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.18" data-doi="10.1093/nar/gnf051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Souza</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moulder</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Improved detection of small deletions in complex pools of DNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">e52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgley%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.issue%253D12%26rft.spage%253De52%26rft.epage%253De52%26rft.atitle%253DImproved%2Bdetection%2Bof%2Bsmall%2Bdeletions%2Bin%2Bcomplex%2Bpools%2Bof%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnf051%26rft_id%253Dinfo%253Apmid%252F12060690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMwLzEyL2U1MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.19" data-doi="10.1038/35888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostas</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driver</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Potent and specific genetic interference by double-stranded RNA in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">391</span>: <span class="cit-fpage">806</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFire%26rft.auinit1%253DA.%26rft.volume%253D391%26rft.issue%253D6669%26rft.spage%253D806%26rft.epage%253D811%26rft.atitle%253DPotent%2Band%2Bspecific%2Bgenetic%2Binterference%2Bby%2Bdouble-stranded%2BRNA%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35888%26rft_id%253Dinfo%253Apmid%252F9486653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9486653&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072089500057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.20" data-doi="10.1534/genetics.110.116616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaudry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neil</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome profiling of mutagenesis in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116616%26rft_id%253Dinfo%253Apmid%252F20439774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.116616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20439774&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281905200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.21" data-doi="10.1038/35042517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zipperlen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Campos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sohrmann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Functional genomic analysis of <em>C. elegans</em> chromosome I by systematic RNA interference</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFraser%26rft.auinit1%253DA.%2BG.%26rft.volume%253D408%26rft.issue%253D6810%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DFunctional%2Bgenomic%2Banalysis%2Bof%2BC.%2Belegans%2Bchromosome%2BI%2Bby%2Bsystematic%2BRNA%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35042517%26rft_id%253Dinfo%253Apmid%252F11099033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35042517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11099033&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165296000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.22" data-doi="10.1038/ng.248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frøkjaer-Jensen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thummel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Single-copy insertion of transgenes in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DFrokjaer-Jensen%26rft.auinit1%253DC.%26rft.volume%253D40%26rft.issue%253D11%26rft.spage%253D1375%26rft.epage%253D1383%26rft.atitle%253DSingle-copy%2Binsertion%2Bof%2Btransgenes%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.248%26rft_id%253Dinfo%253Apmid%252F18953339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18953339&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260501500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.23" data-doi="10.1038/nmeth.1454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frøkjaer-Jensen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollopeter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Targeted gene deletions in <em>C. elegans</em> using transposon excision</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DFrokjaer-Jensen%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D451%26rft.epage%253D453%26rft.atitle%253DTargeted%2Bgene%2Bdeletions%2Bin%2BC.%2Belegans%2Busing%2Btransposon%2Bexcision.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1454%26rft_id%253Dinfo%253Apmid%252F20418868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20418868&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278150900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.24" data-doi="10.1006/bbrc.2000.2260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gengyo-Ando</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitani</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of mutations induced by ethyl methanesulfonate, UV, and trimethylpsoralen in the nematode <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DGengyo-Ando%26rft.auinit1%253DK.%26rft.volume%253D269%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D69%26rft.atitle%253DCharacterization%2Bof%2Bmutations%2Binduced%2Bby%2Bethyl%2Bmethanesulfonate%252C%2BUV%252C%2Band%2Btrimethylpsoralen%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fbbrc.2000.2260%26rft_id%253Dinfo%253Apmid%252F10694478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/bbrc.2000.2260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10694478&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085918600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.25" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.26" data-doi="10.1038/35042526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gönczy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Echeverri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oegema</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Functional genomic analysis of cell division in <em>C. elegans</em> using RNAi of genes on chromosome III</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DG%25C3%25B6nczy%26rft.auinit1%253DP.%26rft.volume%253D408%26rft.issue%253D6810%26rft.spage%253D331%26rft.epage%253D336%26rft.atitle%253DFunctional%2Bgenomic%2Banalysis%2Bof%2Bcell%2Bdivision%2Bin%2BC.%2Belegans%2Busing%2BRNAi%2Bof%2Bgenes%2Bon%2Bchromosome%2BIII.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35042526%26rft_id%253Dinfo%253Apmid%252F11099034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35042526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11099034&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165296000037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> Mechanosensation (January 6, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.62.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.28" data-doi="10.1186/1471-2164-10-529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GExplore: a web server for integrated queries of protein domains, gene expression and mutant phenotypes</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHutter%26rft.auinit1%253DH.%26rft.volume%253D10%26rft.spage%253D529%26rft.epage%253D529%26rft.atitle%253DGExplore%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bintegrated%2Bqueries%2Bof%2Bprotein%2Bdomains%252C%2Bgene%2Bexpression%2Band%2Bmutant%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-529%26rft_id%253Dinfo%253Apmid%252F19917126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917126&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.29" data-doi="10.1038/ng0997-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazendonk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thijssen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Reverse genetics by chemical mutagenesis in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJansen%26rft.auinit1%253DG.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D121%26rft.atitle%253DReverse%2Bgenetics%2Bby%2Bchemical%2Bmutagenesis%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0997-119%26rft_id%253Dinfo%253Apmid%252F9288111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0997-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288111&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU72400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.30" data-doi="10.1038/nature01278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamath</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Systematic functional analysis of the <em>Caenorhabditis elegans</em> genome using RNAi</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">421</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKamath%26rft.auinit1%253DR.%2BS.%26rft.volume%253D421%26rft.issue%253D6920%26rft.spage%253D231%26rft.epage%253D237%26rft.atitle%253DSystematic%2Bfunctional%2Banalysis%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Busing%2BRNAi.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01278%26rft_id%253Dinfo%253Apmid%252F12529635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12529635&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180397600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.31" data-doi="10.1016/j.semcdb.2010.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miska</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The microRNAs of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DKaufman%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D728%26rft.epage%253D737%26rft.atitle%253DThe%2BmicroRNAs%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2010.07.001%26rft_id%253Dinfo%253Apmid%252F20637886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2010.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20637886&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.32" data-doi="10.1126/science.1062039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bass</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A role for the RNase III enzyme DCR-1 in RNA interference and germ line development in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">2269</span>–<span class="cit-lpage">2271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1062039%26rft_id%253Dinfo%253Apmid%252F11486053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTMvNTUzOC8yMjY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.33" data-doi="10.1016/0022-2836(89)90503-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wild</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsh</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Wild-type and mutant actin genes in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrause%26rft.auinit1%253DM.%26rft.volume%253D208%26rft.issue%253D3%26rft.spage%253D381%26rft.epage%253D392%26rft.atitle%253DWild-type%2Band%2Bmutant%2Bactin%2Bgenes%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252889%252990503-2%26rft_id%253Dinfo%253Apmid%252F2795655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(89)90503-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2795655&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AK17600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> Neuropeptides (September 25, 2008), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.142.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.35" data-doi="10.1093/nar/gkq704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">TAL nucleases (TALNs): hybrid proteins composed of TAL effectors and FokI DNA-cleavage domain</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq704%26rft_id%253Dinfo%253Apmid%252F20699274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM5LzEvMzU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.36" data-doi="10.1101/gr.9.9.859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spoerke</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulligan</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reardon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">High-throughput isolation of <em>Caenorhabditis elegans</em> deletion mutants</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLiu%26rft.auinit1%253DL.%2BX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D859%26rft.epage%253D867%26rft.atitle%253DHigh-Throughput%2BIsolation%2Bof%2BCaenorhabditis%2Belegans%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.859%26rft_id%253Dinfo%253Apmid%252F10508845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.37" data-doi="10.1091/mbc.E04-12-1061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacQueen</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baggett</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perumov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Januszewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">ACT-5 is an essential <em>Caenorhabditis elegans</em> actin required for intestinal microvilli formation</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3247</span>–<span class="cit-lpage">3259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E04-12-1061%26rft_id%253Dinfo%253Apmid%252F15872090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE2LzcvMzI0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> Genomic overview of protein kinases (December 13, 2005), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community Wormbook, doi/10.1895/wormbook.1.60.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.39" data-doi="10.1002/(SICI)1521-1878(199907)21:7&lt;568::AID-BIES5&gt;3.0.CO;2-L"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driscoll</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">DEG/ENaC channels: a touchy superfamily that watches its salt</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DMano%26rft.auinit1%253DI.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D568%26rft.epage%253D578%26rft.atitle%253DDEG%252FENaC%2Bchannels%253A%2Ba%2Btouchy%2Bsuperfamily%2Bthat%2Bwatches%2Bits%2Bsalt.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291521-1878%2528199907%252921%253A7%253C568%253A%253AAID-BIES5%253E3.0.CO%253B2-L%26rft_id%253Dinfo%253Apmid%252F10472184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1521-1878(199907)21:7&lt;568::AID-BIES5&gt;3.0.CO;2-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10472184&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081291400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.40" data-doi="10.1101/gr.5690307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maydan</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selzer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Efficient high-resolution deletion discovery in <em>Caenorhabditis elegans</em> by array comparative genomic hybridization</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5690307%26rft_id%253Dinfo%253Apmid%252F17267812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvMzM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.41" data-doi="10.1186/1471-2164-11-62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maydan</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edgley</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flibotte</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Copy number variation in the genomes of twelve natural isolates of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMaydan%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D11%26rft.spage%253D62%26rft.epage%253D62%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bgenomes%2Bof%2Btwelve%2Bnatural%2Bisolates%2Bof%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-62%26rft_id%253Dinfo%253Apmid%252F20100350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100350&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.42" data-doi="10.1093/bioinformatics/18.11.1538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1538</span>–<span class="cit-lpage">1539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMcKay%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D1538%26rft.epage%253D1539%26rft.atitle%253DAcePrimer%253A%2Bautomation%2Bof%2BPCR%2Bprimer%2Bdesign%2Bbased%2Bon%2Bgene%2Bstructure%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.11.1538%26rft_id%253Dinfo%253Apmid%252F12424127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTgvMTEvMTUzOCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.43" data-doi="10.1371/journal.pgen.0030215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miska</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez-Saavedra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellman</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Most <em>Caenorhabditis elegans</em> microRNAs are individually not essential for development or viability</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMiska%26rft.auinit1%253DE.%2BA.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De215%26rft.epage%253De215%26rft.atitle%253DMost%2BCaenorhabditis%2Belegans%2BmicroRNAs%2Bare%2Bindividually%2Bnot%2Bessential%2Bfor%2Bdevelopment%2Bor%2Bviability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030215%26rft_id%253Dinfo%253Apmid%252F18085825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18085825&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.44" data-doi="10.1093/bfgp/eln016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moerman</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barstead</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Towards a mutation in every gene in <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics Proteomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Feln016%26rft_id%253Dinfo%253Apmid%252F18417533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI3LzMvMTk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.45" data-doi="10.1093/nar/gki107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Inparanoid: a comprehensive database of eukaryotic orthologs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D476</span>–<span class="cit-lpage">D480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DO%2527Brien%26rft.auinit1%253DK.%2BP.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD476%26rft.epage%253DD480%26rft.atitle%253DInparanoid%253A%2Ba%2Bcomprehensive%2Bdatabase%2Bof%2Beukaryotic%2Borthologs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki107%26rft_id%253Dinfo%253Apmid%252F15608241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzMy9zdXBwbF8xL0Q0NzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettitt</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> The cadherin superfamily (December 29, 2005), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.50.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.47" data-doi="10.1002/bies.1057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phelan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starich</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Innexins get into the gap</span>. <abbr class="cit-jnl-abbrev">Bioessays</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DPhelan%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D388%26rft.epage%253D396%26rft.atitle%253DInnexins%2Bget%2Binto%2Bthe%2Bgap.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.1057%26rft_id%253Dinfo%253Apmid%252F11340620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.1057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11340620&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168435600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.48" data-doi="10.1186/gb-2005-6-13-r110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reece-Hoyes</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deplancke</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shingles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hope</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A compendium of <em>Caenorhabditis elegans</em> regulatory transcription factors: a resource for mapping transcription regulatory networks</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DReece-Hoyes%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D6%26rft.issue%253D13%26rft.spage%253DR110%26rft.epage%253DR110%26rft.atitle%253DA%2Bcompendium%2Bof%2BCaenorhabditis%2Belegans%2Bregulatory%2Btranscription%2Bfactors%253A%2Ba%2Bresource%2Bfor%2Bmapping%2Btranscription%2Bregulatory%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-13-r110%26rft_id%253Dinfo%253Apmid%252F16420670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-13-r110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16420670&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.49" data-doi="10.1038/sj.emboj.7601463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bessereau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Targeted engineering of the <em>Caenorhabditis elegans</em> genome following Mos1-triggered chromosome breaks</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DRobert%26rft.auinit1%253DV.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D183%26rft.atitle%253DTargeted%2Bengineering%2Bof%2Bthe%2BCaenorhabditis%2Belegans%2Bgenome%2Bfollowing%2BMos1-triggered%2Bchromosomal%2Bbreaks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601463%26rft_id%253Dinfo%253Apmid%252F17159906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17159906&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243730600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> The putative chemoreceptor families of C. elegans (January 6, 2006), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.66.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salkoff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baban</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> Potassium channels in <em>C. elegans</em> (December 30, 2005), <em>WormBook</em>, ed. The <em>C. elegans</em> Research Community WormBook, doi/10.1895/wormbook.1.42.1, <a href="http://www.wormbook.org">http://www.wormbook.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.52" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Meara</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.53" data-doi="10.1016/0012-1606(77)90151-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulston</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Post-embryonic cell lineages of the nematode <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DHowell%26rft.auinit1%253DS.%2BH.%26rft.volume%253D56%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D23%26rft.atitle%253DTranscription%2Bof%2Bthe%2Bnuclear%2Band%2Bchloroplast%2Bgenomes%2Bduring%2Bthe%2Bvegetative%2Bcell%2Bcycle%2Bin%2BChlamydomonas%2Breinhardi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252877%252990151-8%26rft_id%253Dinfo%253Apmid%252F838128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(77)90151-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=838128&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.54" data-doi="10.1016/0012-1606(83)90201-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulston</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The embryonic cell lineage of the nematode <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSulston%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D100%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D119%26rft.atitle%253DThe%2Bembryonic%2Bcell%2Blineage%2Bof%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252883%252990201-4%26rft_id%253Dinfo%253Apmid%252F6684600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(83)90201-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6684600&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RN28500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.55" data-doi="10.1038/nrg2842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urnov</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rebar</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome editing with engineered zinc finger nucleases</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">636</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DUrnov%26rft.auinit1%253DF.%2BD.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D636%26rft.epage%253D646%26rft.atitle%253DGenome%2Bediting%2Bwith%2Bengineered%2Bzinc%2Bfinger%2Bnucleases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2842%26rft_id%253Dinfo%253Apmid%252F20717154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20717154&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281115600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.56" data-doi="10.1093/nar/gnf109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Efficient isolation of targeted <em>Caenorhabditis elegans</em> deletion strains using highly thermostable restriction endonucleases and PCR</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">e110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWei%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D20%26rft.spage%253De110%26rft.epage%253De110%26rft.atitle%253DEfficient%2Bisolation%2Bof%2Btargeted%2BCaenorhabditis%2Belegans%2Bdeletion%2Bstrains%2Busing%2Bhighly%2Bthermostable%2Brestriction%2Bendonucleases%2Band%2BPCR%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnf109%26rft_id%253Dinfo%253Apmid%252F12384612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8yMC9lMTEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.57" data-doi="10.1007/978-90-481-9069-0_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A catalogue of eukaryotic transcription factor types, their evolutionary origin, and species distribution</span>. <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSub-cellular%2Bbiochemistry%26rft.stitle%253DSubcell%2BBiochem%26rft.aulast%253DWeirauch%26rft.auinit1%253DM.%2BT.%26rft.volume%253D52%26rft.spage%253D25%26rft.epage%253D73%26rft.atitle%253DA%2Bcatalogue%2Bof%2Beukaryotic%2Btranscription%2Bfactor%2Btypes%252C%2Btheir%2Bevolutionary%2Borigin%252C%2Band%2Bspecies%2Bdistribution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-90-481-9069-0_3%26rft_id%253Dinfo%253Apmid%252F21557078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-90-481-9069-0_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21557078&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1415.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.58" data-doi="10.1098/rstb.1976.0086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southgate</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">The structure of the ventral nerve cord of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DWhite%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D275%26rft.issue%253D938%26rft.spage%253D327%26rft.epage%253D348%26rft.atitle%253DThe%2BStructure%2Bof%2Bthe%2BVentral%2BNerve%2BCord%2Bof%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1976.0086%26rft_id%253Dinfo%253Apmid%252F8806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjExOiIyNzUvOTM4LzMyNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.59" data-doi="10.1091/mbc.E05-09-0886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyczak</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowerman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Conditional dominant mutations in the <em>Caenorhabditis elegans</em> gene <em>act-2</em> identify cytoplasmic and muscle roles for a redundant actin isoform</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1051</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-09-0886%26rft_id%253Dinfo%253Apmid%252F16407404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE3LzMvMTA1MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.60" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.61" data-doi="10.1126/science.1207773"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeitler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickle</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralston</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Targeted genome editing across species using ZFNs and TALENs</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">333</span>: <span class="cit-fpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1207773%26rft_id%253Dinfo%253Apmid%252F21700836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzMvNjA0MC8zMDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1415.62" data-doi="10.1073/pnas.90.16.7431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwaal</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broeks</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Meurs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groenen</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Target-selected gene inactivation in <em>Caenorhabditis elegans</em> by using a frozen transposon insertion mutant bank</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7431</span>–<span class="cit-lpage">7435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZwaal%26rft.auinit1%253DR.%2BR.%26rft.volume%253D90%26rft.issue%253D16%26rft.spage%253D7431%26rft.epage%253D7435%26rft.atitle%253DTarget-selected%2Bgene%2Binactivation%2Bin%2BCaenorhabditis%2Belegans%2Bby%2Busing%2Ba%2Bfrozen%2Btransposon%2Binsertion%2Bmutant%2Bbank.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.16.7431%26rft_id%253Dinfo%253Apmid%252F8395047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTYvNzQzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzIvMTEvMTQxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/11/1415.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1405" title="Shrinkage Estimation of the Realized Relationship Matrix" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1427" title="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/11"><span>Volume 2 Issue 11, November 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/11.cover-source.jpg?itok=czJv9iCa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1415" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sGe1hzeQ8HwRF6npZtDPMcsJ2s6WfAm3om9OEV2ROBk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/11/1415" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/11/1415" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/11/1415" class="active" data-icon-position="" data-hide-link-title="0">Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459350" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XJorl7uZQxttoa0nzql-jsyp8BWhhtRcWOLJjMLALSw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1415.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/11/1415" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-dhMk4UsfI_DevaS7iXC05ifYwrcxbTrevO9XceI4tGY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/11/1415" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1415.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459350" id="node459350--2" data-pisa="ggg;2/11/1415" data-pisa-master="ggg;2/11/1415" data-apath="/ggg/2/11/1415.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Screening for Targeted Knockouts in the <em>Caenorhabditis elegans</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">The <span class="nlm-italic">C. elegans</span> Deletion Mutant Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1415-1425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459350--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cell Biology, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma, 73104</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology, Tokyo Women’s Medical University School of Medicine, Tokyo 162-8666, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology and Michael Smith Laboratories, University of British Columbia, Vancouver, British Columbia, V6T 1Z3 Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-corresp-email-link first last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/11/1415" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/11/1415&amp;text=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/11/1415&amp;t=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/11/1415&amp;title=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459350/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459350" id="node459350--3" data-pisa="ggg;2/11/1415" data-pisa-master="ggg;2/11/1415" data-apath="/ggg/2/11/1415.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Screening for Targeted Knockouts in the <em>Caenorhabditis elegans</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-collab" data-xlink-type="simple" data-hwp-id="collab-1">The <span class="nlm-italic">C. elegans</span> Deletion Mutant Consortium</span></span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1415-1425;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003830 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459350--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cell Biology, Oklahoma Medical Research Foundation, Oklahoma City, Oklahoma, 73104</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Physiology, Tokyo Women’s Medical University School of Medicine, Tokyo 162-8666, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Zoology and Michael Smith Laboratories, University of British Columbia, Vancouver, British Columbia, V6T 1Z3 Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-corresp-email-link first last"><span>For correspondence: 
<a href="mailto:moerman@zoology.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">moerman@zoology.ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1415&amp;rft.epage=1425&amp;rft.atitle=Large-Scale%20Screening%20for%20Targeted%20Knockouts%20in%20the%20Caenorhabditis%20elegans%20Genome&amp;rft.volume=2&amp;rft.issue=11&amp;rft.date=2012-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=%2C+"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459350/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459350/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459350/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459350/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459350/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459350/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459350/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459350/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459350/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459350/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459350/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459350/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/11/1415" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/11/1415.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/11/1415.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/11/1415.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/11/1415.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
