<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/11/1405" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/11/1405.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/11/1405.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/11/1405.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/11/1405/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/11/1405.atom" />
<meta name="HW.pisa" content="ggg;2/11/1405" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Shrinkage Estimation of the Realized Relationship Matrix" />
<meta name="DC.Identifier" content="10.1534/g3.112.004259" />
<meta name="DC.Date" content="2012-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Endelman, Jannink. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The additive relationship matrix plays an important role in mixed model prediction of breeding values. For genotype matrix X (loci in columns), the product XX′ is widely used as a realized relationship matrix, but the scaling of this matrix is ambiguous. Our first objective was to derive a proper scaling such that the mean diagonal element equals 1+ f , where f is the inbreeding coefficient of the current population. The result is a formula involving the covariance matrix for sampling genomic loci, which must be estimated with markers. Our second objective was to investigate whether shrinkage estimation of this covariance matrix can improve the accuracy of breeding value (GEBV) predictions with low-density markers. Using an analytical formula for shrinkage intensity that is optimal with respect to mean-squared error, simulations revealed that shrinkage can significantly increase GEBV accuracy in unstructured populations, but only for phenotyped lines; there was no benefit for unphenotyped lines. The accuracy gain from shrinkage increased with heritability, but at high heritability (&gt; 0.6) this benefit was irrelevant because phenotypic accuracy was comparable. These trends were confirmed in a commercial pig population with progeny-test-estimated breeding values. For an anonymous trait where phenotypic accuracy was 0.58, shrinkage increased the average GEBV accuracy from 0.56 to 0.62 (SE &lt; 0.00) when using random sets of 384 markers from a 60K array. We conclude that when moderate-accuracy phenotypes and low-density markers are available for the candidates of genomic selection, shrinkage estimation of the relationship matrix can improve genetic gain." />
<meta name="DC.Contributor" content="Jeffrey B. Endelman" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="article:published_time" content="2012-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Shrinkage Estimation of the Realized Relationship Matrix" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The additive relationship matrix plays an important role in mixed model prediction of breeding values. For genotype matrix &lt;b&gt;X&lt;/b&gt; (loci in columns), the product &lt;b&gt;XX′&lt;/b&gt; is widely used as a realized relationship matrix, but the scaling of this matrix is ambiguous. Our first objective was to derive a proper scaling such that the mean diagonal element equals 1+&lt;i&gt;f&lt;/i&gt;, where &lt;i&gt;f&lt;/i&gt; is the inbreeding coefficient of the current population. The result is a formula involving the covariance matrix for sampling genomic loci, which must be estimated with markers. Our second objective was to investigate whether shrinkage estimation of this covariance matrix can improve the accuracy of breeding value (GEBV) predictions with low-density markers. Using an analytical formula for shrinkage intensity that is optimal with respect to mean-squared error, simulations revealed that shrinkage can significantly increase GEBV accuracy in unstructured populations, but only for phenotyped lines; there was no benefit for unphenotyped lines. The accuracy gain from shrinkage increased with heritability, but at high heritability (&amp;gt; 0.6) this benefit was irrelevant because phenotypic accuracy was comparable. These trends were confirmed in a commercial pig population with progeny-test-estimated breeding values. For an anonymous trait where phenotypic accuracy was 0.58, shrinkage increased the average GEBV accuracy from 0.56 to 0.62 (SE &amp;lt; 0.00) when using random sets of 384 markers from a 60K array. We conclude that when moderate-accuracy phenotypes and low-density markers are available for the candidates of genomic selection, shrinkage estimation of the relationship matrix can improve genetic gain.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/11/01" />
<meta name="citation_mjid" content="ggg;2/11/1405" />
<meta name="citation_id" content="2/11/1405" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/11/1405" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/11/1405.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/11/1405.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/11/1405.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004259" />
<meta name="citation_pmid" content="23173092" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1405" />
<meta name="citation_lastpage" content="1413" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jeffrey B. Endelman" />
<meta name="citation_author_institution" content="Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author_email" content="j.endelman@gmail.com" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Statistical Inference;citation_author=G. Casella;citation_author=R. L. Berger;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=G3: Genes, Genomes, Genetics;citation_author=M. A. Cleveland;citation_author=J. M. Hickey;citation_author=S. Forni;citation_title=A common dataset for genomic analysis of livestock populations;citation_pages=429-435;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Cook;citation_author=M. D. McMullen;citation_author=J. B. Holland;citation_author=F. Tian;citation_author=P. Bradbury;citation_title=Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels;citation_volume=158;citation_year=2012;citation_pmid=22135431;citation_doi=10.1104/pp.111.185033" />
<meta name="citation_reference" content="citation_journal_title=Adv. Math.;citation_author=B. Efron;citation_title=Biased versus unbiased estimation;citation_pages=259-277;citation_volume=16;citation_year=1975;citation_doi=10.1016/0001-8708(75)90114-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_title=Average excess and average effect of a gene substitution;citation_pages=53-63;citation_volume=11;citation_year=1941;citation_doi=10.1111/j.1469-1809.1941.tb02272.x" />
<meta name="citation_reference" content="citation_title=Advances in Statistical Methods for Genetic Improvement of Livestock;citation_title=Discussion summary part VI: selection and non-random mating;citation_author=M. Goddard;citation_pages=474-475;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The impact of genetic relationship information on genome-assisted breeding values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_pmid=18073436" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_author=C. R. Henderson;citation_year=1984" />
<meta name="citation_reference" content="citation_title=Proceedings of the Fourth Berkeley Symposium on Mathematical Statistics and Probability;citation_author=W. James;citation_author=C. Stein;citation_title=Estimation with quadratic loss;citation_pages=361-380;citation_volume=Vol. 1;citation_year=1961" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient control of population structure in model organism association mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_title=An Introduction to Genetic Statistics;citation_author=O. Kempthorne;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=J. Multivariate Anal.;citation_author=O. Ledoit;citation_author=M. Wolf;citation_title=A well-conditioned estimator for large-dimensional covariance matrices;citation_pages=365-411;citation_volume=88;citation_year=2004;citation_doi=10.1016/S0047-259X(03)00096-4" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. P. Piepho;citation_title=Ridge Regression and Extensions for Genomewide Selection in Maize;citation_pages=1165-1176;citation_volume=49;citation_year=2009;citation_issue=4;citation_doi=10.2135/cropsci2008.10.0595" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=H. P. Piepho;citation_author=J. Möhring;citation_author=A. E. Melchinger;citation_author=A. Büchse;citation_title=BLUP for phenotypic selection in plant breeding and variety testing;citation_pages=209-228;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9449-8" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies;citation_pages=800-805;citation_volume=11;citation_year=2010" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_author=R Development Core Team;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=J. Schäfer;citation_author=K. Strimmer;citation_title=A shrinkage approach to large-scale covariance matrix estimation and implications for functional genomics;citation_pages=32;citation_volume=4;citation_year=2005" />
<meta name="citation_reference" content="citation_title=Linear Models;citation_author=S. R. Searle;citation_year=1971" />
<meta name="citation_reference" content="citation_title=Variance Components;citation_author=S. R. Searle;citation_author=G. C. Casella;citation_author=C. E. McCulloch;citation_year=1992" />
<meta name="citation_reference" content="citation_title=Proceedings of the Third Berkeley Symposium on Mathematical Statistics and Probability;citation_author=C. Stein;citation_title=Inadmissibility of the usual estimator for the mean of a multivariate normal distribution;citation_pages=197-206;citation_volume=Vol. 1;citation_year=1956" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Cordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C.-W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Shrinkage Estimation of the Realized Relationship Matrix" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="twitter:description" content="The additive relationship matrix plays an important role in mixed model prediction of breeding values. For genotype matrix X (loci in columns), the product XX′ is widely used as a realized relationship matrix, but the scaling of this matrix is ambiguous. Our first objective was to derive a proper scaling such that the mean diagonal element equals 1+ f , where f is the inbreeding coefficient of the current population. The result is a formula involving the covariance matrix for sampling genomic loci, which must be estimated with markers. Our second objective was to investigate whether shrinkage estimation of this covariance matrix can improve the accuracy of breeding value (GEBV) predictions with low-density markers. Using an analytical formula for shrinkage intensity that is optimal with respect to mean-squared error, simulations revealed that shrinkage can significantly increase GEBV accuracy in unstructured populations, but only for phenotyped lines; there was no benefit for unphenotyped lines. The accuracy gain from shrinkage increased with heritability, but at high heritability (&gt; 0.6) this benefit was irrelevant because phenotypic accuracy was comparable. These trends were confirmed in a commercial pig population with progeny-test-estimated breeding values. For an anonymous trait where phenotypic accuracy was 0.58, shrinkage increased the average GEBV accuracy from 0.56 to 0.62 (SE &lt; 0.00) when using random sets of 384 markers from a 60K array. We conclude that when moderate-accuracy phenotypes and low-density markers are available for the candidates of genomic selection, shrinkage estimation of the relationship matrix can improve genetic gain." />
<meta name="og-title" property="og:title" content="Shrinkage Estimation of the Realized Relationship Matrix" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/11/1405" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The additive relationship matrix plays an important role in mixed model prediction of breeding values. For genotype matrix X (loci in columns), the product XX′ is widely used as a realized relationship matrix, but the scaling of this matrix is ambiguous. Our first objective was to derive a proper scaling such that the mean diagonal element equals 1+ f , where f is the inbreeding coefficient of the current population. The result is a formula involving the covariance matrix for sampling genomic loci, which must be estimated with markers. Our second objective was to investigate whether shrinkage estimation of this covariance matrix can improve the accuracy of breeding value (GEBV) predictions with low-density markers. Using an analytical formula for shrinkage intensity that is optimal with respect to mean-squared error, simulations revealed that shrinkage can significantly increase GEBV accuracy in unstructured populations, but only for phenotyped lines; there was no benefit for unphenotyped lines. The accuracy gain from shrinkage increased with heritability, but at high heritability (&gt; 0.6) this benefit was irrelevant because phenotypic accuracy was comparable. These trends were confirmed in a commercial pig population with progeny-test-estimated breeding values. For an anonymous trait where phenotypic accuracy was 0.58, shrinkage increased the average GEBV accuracy from 0.56 to 0.62 (SE &lt; 0.00) when using random sets of 384 markers from a 60K array. We conclude that when moderate-accuracy phenotypes and low-density markers are available for the candidates of genomic selection, shrinkage estimation of the relationship matrix can improve genetic gain." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
    <title>Shrinkage Estimation of the Realized Relationship Matrix | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-S3MQ8CUvgIxN8uWe0U2iNu2pA-bN7PBO-MmDZdtVfY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459347","apath":"\/ggg\/2\/11\/1405.atom","pisa":"ggg;2\/11\/1405","ac":{"\/ggg\/2\/11\/1405.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/11\/1405.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1405"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1405"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/11\/1405":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iis3\\\\\\\/MSJWf5vBKwvTB6qw6gQU+p+zj4ushbV4rMSBYBBnTyXBEHrI0aQ\\\\\\\/xCJqPJPIz7krrTDZc33rxV1KnHftXYcWi7y355GzT0fbTKpGHCSVYzkSNZmOzpFGLfheX7RDe1mhEHhlkQ\\\\\\\/HbnDQfO0tTN8BtRniDKfJ54IowV61fK3MJkhkoOG2tGQoRrmsnvnJsewdNi\\\\\\\/qi4j6vUcF+3WvvR3qAwMCEKv4AT5PucDFitxDX2c805oz0xc8tM1OR8Upo4azTx28J+v03x86KSjrwad3WDpN2\\\\\\\/r45VHii\\\\\\\/T5A9wQj2WxAbBfZl9hVPjQuBrSBrgH8NguVu084v9TsDE0+UPdtUDqmQXWMiPo7WmDUi7UN9KMCJcigmnlFhIFpcOtHwkfC3lf28l4kJwhCzQsBJSWP2lxpUuZdT3Fpwo8FM11HOFS+9bHfH6FDJW9G9gLWIOeZy4Tr6eqWCMoDRE\\\\\\\/ST9hko1xlYiTuqxBkvSzsrGrC62jmT5K2nAgprxmKt8wFbMWd6mNOLDu38w7bzMl6D\\\\\\\/+O+SNVuXCatz\\\\\\\/CjDjw5yVCDgv0peBcTvlp5CwjyMWu8yHNYg2TNDTxLqlBHjalDoL6VxATRRfUo6gX0fQaHnjuuKBsRyfHcUlFoLJcyG5QXxDnBTvh7rcHtFUW6vsePYa1keTfzyra0p4JG4w+271RFtnybBY1HmQWXcMmxGlUr\\\\\\\/1DPx5W39Hw5I+2WdZrqyro\\\\\\\/T4eJ19ZNaqluBUW2vfTtcIohaar\\\\\\\/FF+Z5eutwBc+fcTKF9tDM+TCXmGou5hb9HKPVPT9VHt1BlGg\\\\\\\/EWcPlmSGtlcmilL0hkuk1u+jPGnDMuaqhpvsfswS0Z0Knii8WpzVnfpsT7Rs8wBjNs1wpaj22fCLN9BHqb034Z6t\\\\\\\/IM9P8WK0+ukR+\\\\\\\/aapUYgFu4zz\\\\\\\/ssWc0J1rIpQQmO+2lnMI+7DU19gRGKWw72YcezwsMbNBMcUINkKPSmXmZyi8Tznfp1B3dxFQLeolDklgGCyqJfcSg9dgab8+uOU4+IoGpeq4jE1DXVGOtFNEB5ukvbzcp81ZWZQolIk4AAJlo51FyKtGXRDn0eGqbMRNRb7bP0tajQiAMKFXpqSI5KqC7+M\\\\\\\/ccCgePscBhkYH890NO3zZA47Azu7zmwF2onVjJw\\\\\\\/6eDrwsqs1RBklAoo6YfTZlK+Kl76iP+kivSye9rOD0wgC1tkr+vVtOZSzNN6I6xLlJcld4xMtR+4r7yPZUJnHEYqX1kAzE7502YWM4s1e3Ji6266EI9+wx7juOYNr\\\u0022,\\\u0022iv\\\u0022:\\\u0022zeFD9\\\\\\\/QknwIF2ApP+x4VGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277b6c6d006c6456a1418d8e5b920bdbc1440efdcd755008211b44762dd4a10d8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227bBS7xa5sEDgc6n6\\\\\\\/tJ6Myq9NU1mhPSQiNEF7UxXwthUnqcP8Ch45ZSxOp+KRA+GULQg2rATp5hAr6E6mrl+UFfBeujg+jGLXDKepG5wHWpEGIMHqHTnaml8sTMx+tdZeztd86wlCi2EqOYGDZ4Bf7P53peRxjWJ2APS2Mv4Bd8zyiGADIl5cTWohF5p7zmztpaKZe7e6\\\\\\\/zbZfASbQMoNkOcTT+jLpUJVZY4+WH6YpG0nXAiCa4cFxqIAAU+ZhTnE\\\\\\\/\\\\\\\/CSf+GABFHEJUL8OnXwMljzQtjnw1zCoOr0EApo7KLWHXlqXbCEGkliQTQ+v3bVg869dA0KYNOeMESuZDlSTqAMY+thgCJKGY2h9oh2bEKpE0Z2QWZKCtxLxNBcjQciv8uV6xrGlV842uod8wDZbfDS85jPxg7TB77sz2h\\\\\\\/PVxSmXRKhG6qGd5XRG19TXgQmmFJLMLu1BaHdtV3rGu3UncQEbmpYWdxnsVCJhAu8NWAbZEuY+fcFjiV1k4NkX4GD64M\\\\\\\/Pinbrgh97GyUYpL73uqfv3kmtcz2KSTrNnb8HHDhbjarYNZx7750v7NAClGek6MjcIj0\\\\\\\/N5KiIrprx1I0rDxCWTlvRnOdwis12VTGhnbZjwhTv2tHu2lfR6uvtvlW4DFkvfCGNltGjdKksiaBs3lgTqh1gMyiKI7wP3IOKFeUEZWWlzhSy9Qx0HMyP+5Nd5LtOn+LA\\\\\\\/rUcML0P3P0fdBXD\\\\\\\/roC6ZG\\\\\\\/dMaK8jNqfD+ubbzRMQHnmEdFp0bGx\\\\\\\/OapvdzwH\\\\\\\/t3GXkAtwAivRqEA8ecY65ffmkl\\\\\\\/ox1GvD3mMAL6OxR5a1HIM97MGUgdVbyrS6G9GbavgBagLXWslrg9pUMBJQt2f8FFaHRYU7qnlkO6XlfZKi+VqpgfuXH\\\\\\\/TrH7w7WN\\\\\\\/mKdglJvezDTFIcVKLgm3lYLPlU0RU+38Tf9\\\\\\\/GHBG9OQ+FdEndSgrhzBWoaC5hQ1ZPETJCti19Y1JTyhy0R2Wnjl5ouBaVOJ3L+nkg3Ah9m5R4sEV+hQnhWfeMf8fmsoKBqIjbnt\\\\\\\/C98ZvJAhQ5EELMqro3MDRgN5xOQhl1nVRG54u5vdn4knlxWcdRYAIac2iuKKSqoFHjBT1Oas7ZLVswd9R\\\\\\\/sI\\\\\\\/5fTGdejeDCc4x85PWLhiayVptLjuuYAFBLtnfsyIbQmCs34HPyXB6qaXeeWfzOVb++j2sgA7HXigiCcrIi\\\\\\\/43C\\\\\\\/W+OWy1O0oMx5MoMzIBwqGmUrQNeZZ69HFqWLtFHPuU1pjZmiq5uEgeKdCDLMP0vfc60C7LfQdMw64amJ5Au5XPaEulzGogwOImhVWEFscn3RqAu9IKjUq5sNcOS7aXz2NK4hSbT76rwEsVf3Zp+IaWcmwbWFALD7RlSGDVYx7ivUhy04x4J6+cELmMhkCgYw\\\\\\\/jvX+FUfBKZItPeyML8FUyToNp7SitRHaDKK80q2vemdlbXGe5UZnOlHmwwlyChZhE5KVPnSd7vZPv2WQ5\\\\\\\/RsOvvaYEi\\\\\\\/jcEOAOd5mqjwT1Us4my13BtyyhBvfl\\\\\\\/E1gq\\\\\\\/NOO7lFV+OCHIHTr+xa9lNEbzy07DfBln3C5UOAh1BwnSCiEgnykuH4zNDwkokwGu0ywcKc9X6dTRb2EvYGCfGlDjOhXQeVpZaorhbzEoY5dgiM5y4KBKs+O1M2g4qs4ccODWFiT\\\\\\\/Xm1+WYIpZUlWlXM75GoL7AcJJZreIcZVueB1F4BG48dU5YpWTiG98FCnGB2LYdwB+j1KWaGeq28N5Jo8fuLHMopUSjaTc1AQQcPhtju9sW7YgnSiSVOCTn\\\\\\\/CernnNHMFTNOvYEm8dVf7Nx8wSrZYSwAj33IaWsRqUPDYq2BWuKEQlAesHFAq53Zp57caK2GHpA6zIZlHpsm906VuM0w95r0S3xjdN\\\\\\\/WhreL+o1PJzProrLNiKdgzmD0QusMiDdhqvHW+7E+mk9HV7GwD3bAQupSq2sWQQpu97dsv8Oxt0gc4J483AhAknMvWebtdVq85gykqNG1QLtRuPsryKxC+YOGp5ZoosSmTIGB9eXp3Cml\\\\\\\/HNijec0EcQn6P+65oegEthtPPkGUjtVTi0MRFXKidxMpDF5Ui7FZBOxuuKIwTAwhUDBrEHPj7gUvB3xCmC7WnimAdf2E4bfrJOfHHJnK\\\\\\\/\\\\\\\/WRDYjzKLsqodinY2h0mjwpo3yI5SWTDZQcqE9LuHYyf68Cd59115fmRWuUxcY72gdg331V2Muauy2Vwww19BJMdBxlI6qGCzRKnjxyJVYGinbk5C5h3SGyo92NB96UmNdhr+ky7Bj9UY2naLWUnX2Mouph8Lw+bl+DQJTzOMKNzkuw1H0wrJ\\\\\\\/y0DPeoNmdEufC6cnKVeU629GONJi0MfEy2P5n3324zQmDkrxTs7MmqbnEdCsaufamUEEYlSylrVvPw8eWfceNXfxvSwakMTvB81+DD97mbJS\\\\\\\/jTFBK\\\\\\\/LlD0AayBmPG6pamKv2zbGPIUKUrj+DTZL5o\\\\\\\/+ZCuW2\\\\\\\/1ygN4nEgdmeFNb\\\\\\\/vUuzMxoaaPHzgQFRe44fe96WbUHlznpZI7TvfPPbW\\\\\\\/p2fVM8lrmcTYuwGhAL5FXPp4JSCXIbZV9zQgpEs3NfUDi1ukFeVtRGqNt50afYg9\\\\\\\/YhLfH2yXD\\\\\\\/6krQmJVZ\\\\\\\/d\\\\\\\/1yVI7mQAtfUKzl09TbnNzwiTCvb2pSUybz4+nELENY5L0qCNEagD\\\\\\\/w8AnPqXv6C\\\\\\\/A5PDbogZ7a+SYEU6xXoA5dN5onCU+ybrOU2pFrRvXtS9uEpgiIdQaHXHqoKHNs0gdGaDRj8JadckSHLH6PJUkKaepSxLO5VklTAJ08E2LESjKl4bZSfPunAvlJXJdD2BRJ58ZtEzNR5z7WWaGKhXSxX8DfX1vx7weLpkjzhs++mKlDwqZPh8j5OJ13B9HEH5m+6PJhSkCuLBhG57UAEYgTgdbNygNrX0z2Aatk\\\\\\\/n64WCj6H7fZRQl0fSYi0lcpvhIEyfkUd6weNRYXwym9bFh+3a+iQuDUyS2zN2f++6kY+XxfI7xy0aLP4bn4lHhGD2xnZxTPlkzV9IweLr+80Pg0qzhk23E4UtTIyEB3zDzddXseqSkAK5pm6Hx1gLzStQ56Bgz+wbVQiS61igxDrxp0E05wh+Iingzll1EI5akC+gcy0AUyuDzXuVf03oLIrk6n094HBCrY81OA+egkoTeViSADqS25l0P+8nWmdsBPQ32nyOkzUHps3o90tjg4nWaSHXfCigXYG44zeoXDOEo\\\\\\\/MIrv3dFnhz4Cz2lPXkQiU1HOTF2PjF7KEn\\\\\\\/ASAR2NCneBycdDPEYzpPQpFOdzeG0oCCSaYg8ihvtsLbbPvh99mbr\\\\\\\/9hPyEo1aazDlQIXO9I6aYg4SnIJZ6wUXAIcrZi8nc6tGWF26dacliczo7C0ib3ObIWt5tjOYHVhu8PesvYmS+9HyIupH6WoiMRRUHJ\\\\\\\/bipvNQClRCFZqzJHAJnDBkdVBWnNxi\\\\\\\/qg2hp97zX7c6R92CPTe7pu93cG2+g1+KOH0h9WeorcuN9PenDIJynfsNM6PQvFAV96UHTTiPHDushLC\\\\\\\/FWaG2+2Ctl6bl54Vesflc8ckMLfKKsPVrXXFQ\\\\\\\/KiDAyRWvrk4uDYR0MdRbc30xI1p8P\\\\\\\/Gzx4w\\\\\\\/x\\\\\\\/QaIeLSDrnNPhdQ5+11tRa326E6C3x5NTK2kkgjzBxHphqOXwSUGi3LbrzuFzFqpExCrLQmRoCV7CxfdUOwBRPqC0Lwg+41Q39qyE1h8lF3hZGsIqe4Wqzp3cPwdfDqzhs3FvpWfKzFxFw9cS6kIKjQpawdZfQ\\\\\\\/MEqpdLfGvXmD\\\\\\\/g69w1fSqszDh2S8+7xteM5cW7JPg4a4oQ9lJn7eIN4zX6Fw+pW1dOof0PLW70isjeyZoMYoxNGSmWaUbCW6EhXAqT5RN+ndIAm9OwSDjru1hCpxtwR4pTwdtsBPE2+K7BQo5dn223CTrUFme5NV2FpoOBJzLdfCa8Rk6xrUsrxy6PUjFRcJDekCJaxGByDEFjQSVA7q3IKsIyOM\\\\\\\/tpjmeGadDS2Wv+5eIr8yLYoqCnHxYJq55uLRLWk+M6dVczY1GtNyPVv6wIeox6PmZhWsRxVvYVDmAmDSlsH6nmQUokjVAVX3EDURXq5GAd8Y2v2DsZmEXEq6OoAjaY73u9yXgxCVqYhCu0VWsKzJTfNOFoXGLe\\\\\\\/gVcy6HYDj1b32WbrPyHEKdHkRZeG1mLk5+O5jIy5bIaagTjJjBu3C1awcGl4OlGrwG63sZKFNVMnEhuNfilU1NT79QzDgHtyJwQA05OTCCh0C2bySX6Bf+LC3CfS6m1EH50khthgHwMW4ff3ewRuDVdyROGbSItq4jNHlVbit+ShbFUcca7nP9UuXtK4DvR3iVhtddyaar9VGoa+LxkTmg4JW4xNGnNEjjHwv6cNLtZa8GF3I6G4qrxddpSwDD7Wq\\\\\\\/D9C+YH3kKqwkkO\\\\\\\/2P8QSJGTrpznMX3YvWj9fz4U70kyV1zf4MrhpRnLBup6i3Mf6Y1h\\\\\\\/aqFrrC1RrJ4TxdTwf2XM1DNAM0kc+OJtZNItQF9+ER5EUK0RM0cXN4rqG4qen8P+H7uzLB3UIUoV8knDgGhQ66XKtkFRZdeEC5Ig0g09UmkVoUiGxYOJdYA4VVc3o+0PPv52LCsCgnxx9a2rhIYhYQRXTMviFLalxSSbjkd2yOOxSyr3yqNTp++zfj39NbYTHqg0ZfucZvISixjaptUCqGZ8gT9X84IqtbxCuFJf6IWrgBbjWCMsibGjummu2TNDHfSgXBSJRyW4y9Jdo3e0m9dJ7VsicIZapxh6eVlOzjDeoaoFA9X+01AUH8yjS6YNMMsmQp6O1yePv2NwiRU4GgSP6UJTc1sjDJBO7aspFaKr\\\\\\\/nCAHMwP\\\\\\\/gReZa\\\\\\\/22rRk3Z+YecAe3O8z+AfqtgApDzfPh1pUh03B+BFgl3b6I8y\\\\\\\/Y5NEd5KVnlJT05GU8ujJpbdQZHTEj22hlp4tgfeFr\\\\\\\/bMhqc2TqX8M4meckGgFWkIEsXhn8FQSTK3B47kcXTm\\\\\\\/3WUjUMrq4xmEW\\\\\\\/OcYuu0rN6tWJuBSm3s35JVUZnqBeEp0vdDLyrvGpQJ5vsx5ptb\\\\\\\/Ad3x4k5ras9nfAHLUGnexHTkpRh8JkSsAig\\\\\\\/YVQmMxMt8YaqkKqD41lB1+QHQshu87E0nP7eya3FyAGYfgPB5a\\\\\\\/Rko8o1Et+fOl31FEI4fcqUoVAV1wAl4QK8GusJ9QHOyLliA3mSnYE2YrZT1YVozdqLbLqc1RITb4aUbAIHebpxdXuYbeBXDlnYh1cJllwn5SH4Kp8h0ZURi92lmEKSYntTPLTgH3ky6Cx4h5yWXPdTWGE10NLVc8lNNI9pY0CEoEciCiiEtRWxnZAEaj3t\\\\\\\/yJut8s0kb8kUD1O\\\\\\\/12kOgQ3YKTYQK6Pft63L2cmuQRgfbNHOEXnby289BFjs9SaT2VgL5hec298FLJinvjMUOeMxqvE3zmEJY+9mqFGyQ4IcgSgtAaxuMEiWp9mNw\\\\\\\/ZDmiJykMYge+1OVZLqIR1tT4nQriLxZ43RXreuD2hEArQyePyEvWonOnr2DxaoBxQ2VLnGSEBWykR1X8vM+9iPc8iW7ROV5FphfKjCzo3Zfa6qBjrcGe5tH1vlUrHq6yGeu0qslTouvahsXbZe6oKqhkuNDuHchjg88d3opeIzgiLrirllSdkbxuJlIqxQ0MdFkP23Rzds3o+lFb+ZFo3cL48DzzZ0MjL6Q4mNrMhEW428DdeMGl+T3im7KvVqsMk4Sw9mHxmBCbHLsLIp3FLBcI1npAaXubzNfDKwGNY21cvGxsvVR2c93oXm5NA6cUsmKASVegEYQVXzt4XQtMHBEcnnur3PLz65sADPFDfFSFX2dQgBE2Havuij2QgHpyUap0AhOfzR2tdDjIUiiEeEnV2E6Lu0NjH1NvxP+pYI+5MxvwyGaUL5ADoyPxvmtuxNCtvjby2r3Yh6lzG73ybeMt+3FGQ9+O+0vTcKinBR\\\\\\\/y8WUHLc3NM6ivTWrUbAw\\\\\\\/B2KaDz4xLDRqEPPUAJJqEPy6O12UvVmuraCeFiqRDPKE9lL8utuJTbIjGhLE2msZjB2v5g2FlRNCUJcFJzaDj40SosO8bOfdZGmwD3OEVTyn54aistlOdCS6Lu6JIyyZn9XDxizP3NaoaoEXys\\\\\\\/Q04ac8iI666k8oWvMGW9gZ2B1XkC5YRNQHp7CKR8mTe\\\\\\\/yEUTLuFX75LjvcOfSrUB3k3PFSw6LpgGMnj4MM0bvmRI+TbiKHsCHoWntWwo41rlcaMqjyG43rXFopULijkzOKVLyOiv9xb8MYUt0uVoq10S1Z2Ky3EtNDqX2hwlj9K1s51rpbsG1reOTRompeq9jsu0ZgxBeDqwUYleC6xQM1iSYczppQfrj6nyiKVqb+hhMBu6jatcM0SqgFpSvEsbi0RdU\\\\\\\/x3AcUW0146QdyUX7aJyktJQ8c1sZSsrl8qcb7hG+0vfEJe6\\\\\\\/LtVxugoEGPfzA91ePyU6wTmH0u4ZqlYS1dcHvKIVRGvEGsEQdF+wnu2PQL3lzEU1FO24Kz4XKZP3hGw3Y2T5Eey\\\\\\\/ClAK4dimO\\\\\\\/VhqlQw6q+dmpCw7JWMbViGzfHa4M2XjXLN0WvFw+h1vE0oZSnFgSmOTiLmLLkVCgUTnQ1RrMOEYzEQ7bjoQxL8kQzv7gsXJmVHcHtCeQskF7V3FOeEyhmM1npYqSpDiCgVih67JXZK4p\\\\\\\/IGGWnIraOV1WKFrAHK7LlKVP\\\\\\\/kC3lcrS0j1bLcu1IvPx7XNyMtnatXuoOUVrCfsogSnxxrRnbYR+AvyJHYE\\\\\\\/appPQOYyy3WVlvsfgrCTL4cdhGlW\\\\\\\/Pd72BzDEOWLMiphL3e46g\\\\\\\/Vn8BXMVUyrsaDn34CM0qVfT6NmoubaUcnPJdmBZPKEJhic\\\\\\\/6iOjd7fXqR5E5Nmd50ADpN5HZiPv0ZJNTsP0+H86QfUpbVdHHrH2+LxchCVSTaK4GLGYjBmjQJuvsFQPPyNsH7DB9zbRew\\\\\\\/s\\\\\\\/qh+Qd4N1Y9iCg344zsrYcws9FL6JHuTq5KSuGNW7A+FayOIqyMiqaIcKdoAgXOsBhYQtuTXmRuAzrXsrDEkhESmeZS4Gq9wZTEw47lYQwF5vynPV8NHMhh24zTqBZ1EtJRcWMqWapfafudOrGKoGhoDpnLjrcPdJzPiwuImzJjiW2JP1ggsw5bXQMwwZCrqCM4CVrsK2XTJuh0XVgxcnhS+DJhBKt1PaHDAYyViV6z6Z\\\\\\\/xUlEmVdsQ3NuuR8tjBIz+P7zruF0k+GA5f4W4YjSPw9BdUe\\\\\\\/Kxx35rxTJJPF2ZtInT2TAdEw\\\\\\\/CK+8yNBi0sHDUSFA4qYjfTVNyNNQmnZ+VTt0bkr1H3csUtxkpryCIBYyuzTkaXjRxkcSRX98w5brFdlwptpbproV8NTMz1FXVlWd1YbsIggCSI16xBKUX2yHXCmFmO0DkS4TljE\\\\\\\/oVXxX5igzafs8HwKbp7R5gBu2IdsUFeWt6eOcVuxwdVd7uLGw7ZJEWd02mQLtODKDZYcKfMg\\\\\\\/ZsQCPrH78uB51WyXoU25Ik1+N\\\\\\\/TcUupXyDOTRex8XIY0crHkTzyi9XNsMI1LZsDk9EqjuKd7xsh18Y73Wd8vo4Oj+71v\\\\\\\/XoG7GGPH9ll6WO3XDQLLYpVannJQtT8lQO6xAotq+AilKwP8GZQ4TZ2PAgz3pmf60iBbXpp0YzsaQtDQvCPlehI03+0LKESHn8RfJi2yK\\\\\\\/XrqthTV+4m03I+IA+w8CgaJQemHcWw1CaUmH40JFZ\\\\\\\/\\\\\\\/Q7m7QlZJoheanb0KrC7051D3LAp68k9ArdC3wBsMJJ9NgrLDctyk1D+IR8a+KAfFPN\\\\\\\/LlscbrzHo831akRei\\\\\\\/SHxqd+ZjSA6O3kRHxBwDCTJ4dCL6ony8Z+uvaZSdxnoZioixxWNHizOdyQjBZcKAOvnaBRpOOZGJx\\\\\\\/AXkeyjLKw2U6mO1IKr2te+y3\\\\\\\/t01s6wvERXc\\\\\\\/D4r\\\\\\\/PabYMS+ym+s2bWWpWG0TbYZBz5lVxq2oLNLX1zeXnDKPEyvafgbJcgptA2odAYrAhdxvehmEkKh8fv74s7k96p2lZ0Lokcc42wPLD19LrJbMaLJDibKxCAJaNoW9hW75vhp\\\\\\\/JpYID32u+Nddp\\\\\\\/VTRpM2v0sZtmIOKjfGKn\\\\\\\/4iAdr8HRIJ6UukvpiH\\\\\\\/\\\\\\\/olYZBR3dvz+K3928ROhwabXzTxRbyM0chmoZspXU0695TsKN9YYVyp3G9Y85QSWzrTFpR0VeQVr89XOqRn1+w8\\\\\\\/9Ym\\\\\\\/C1oJNEKk2+JH1FSa876KSaUpZPtkDV54RYiwSmLxKqpRixx9yVNMx9R4hsXUi0a9Zg7CzLM+4cGjANYeZkDd4EKDh9ydcXbcgvYGUv\\\\\\\/\\\\\\\/e1ncmF++5yCBg+ygffgbadITvpW5AJcKFmwS6DYCoHsss7tVlMYoYnVIZ5Ik2E3RP3H6mnh5USf0zYVEYpz6RZxnrxI2MCsbdOZlESf486JmJ08IQvgo7DM2pFATDm6xNDxLQ3oE+WaTs86vUQ5wQWKxAXzQJ16cwIBj3Uu1k6+rZUCLxpMPCkGIP26LaUNv827LAzj90EJDoEkL4VPxAyN\\\\\\\/cKp5459pHJOPcjvr4ujHYsrPzscvGK7ERV4BuKR9VvLwqNyyZWq6zws+X2+8dL\\\\\\\/M+21Eg6kK\\\\\\\/vthRKJN+nfrSuWbOs7PWGJrs61O8sODZd8vczk2KutC+Y4oVyev1kLGj0UPPvuhhjaCtZF1Rg6eXcI4hQ+wbdBjssu3T8grTzRM7VZgJUCpPsQwNK8PGbtX8JJFwzHOToogxMpQpxLKvHsFSC5YmjU0\\\\\\\/mooqa4bavnrOBCrBCZDP2QerV1kJDZ0OWFoLg9nAfDErBYfAcNDYL+rjIuVwVQB4igYitelcUjOoM0OA2uZ\\\\\\\/bpw69icK3mHEXU\\\\\\\/NwRlkJr\\\\\\\/TCtZVkWXAK1Eq7bI2pqsKpbzkFJRYN4tLwXiLS9EhSFr7q8uiD5N7qjXkzJJfhlXdS2RH7BKXTwgCEIPC96eSfn1pflWKYo2eUqQ1PgzoRms58aJihhhJTVGFGY7xkdJBFbfz0x2GN\\\\\\\/rSqcnATZvuwo\\\\\\\/ukgi4vPwGqFknsazz1pdhHbjfdylXAQ9a2YCJFtlR\\\\\\\/Rqd0h97k7OSdPDuMbmeFdOG4GwZOzzQ8hqFir6VLqhlchHY4CSm3RFLEUg+aZgBqPvSEvMvgbW\\\\\\\/iW4WrOyfPiHI6bFsG\\\\\\\/d97z1IyS4Eg1Ua6r3mFKpUf7iYM\\\\\\\/uxYIfpxnMnffQLYPzgACBlLw6wR0phDereHFN+huYBrDgc45brBBYqoWO8OWeYADmdGDDTJGamJYYK4agZWkfNVwoM7SANkDJjowJ2pSC4CcRZrZ+sQUZcJy\\\\\\\/G6OhJZqNKYUd1mxqCBTkndctP3+LD+UktO+xM9s\\\\\\\/6nDb\\\\\\\/ZSuPn5yShZPdEEaD8PP3e5divJEaVh4Dl59cX4wFoPpcSdqLfWuZlY4lP9ycPskPzcQPKzw2NMEXiOYiRidvTKA3f5uO\\\\\\\/z7hbTXsgU0PN3Rcev3s7v0O9WxgzPqwj\\\\\\\/cLDWbUewt8hdJc+KdzcKVafbhE6IgURT9L450lNbzWk\\\\\\\/FGMhJo\\\\\\\/tsPJULLfe\\\\\\\/wSapxAqWtaPPbuf4iLndViXzMy+5AJDn0VnE4cug+W381TPFzaGKIkETNTQSbHyM8b6vEXA5GnL9fppscSrjOrWekzQs8QJcOMGzYpA\\\\\\\/chz\\\\\\\/YzQjLSkli4dHRnbymhF2g65q+G\\\\\\\/M99u\\\\\\\/LMVl1D1tgmWSosvGBi3PXWCHc542DwB+KdHC689XgY6xFSbdiuv9aSwuRMPe9btJb+PlgbwDrR3dZb\\\\\\\/sGZYpyfDYYHd9sEYZoSBKMFHv0FW\\\\\\\/UUORYPxS7d266+\\\\\\\/tqeNibOXHidPUY1iixqCOgOK6Tb\\\\\\\/CyLgnGxk5KPnVaUVeu\\\\\\\/IvG0xLAjk4PVw8uHJZkYwLbk0A0ISezKX+e0fSRjjWc7\\\\\\\/Sp\\\\\\\/TZD7zMmJsdRVQJglT1NM3M3BYFIrsyhg1r4HljwKN9rm1NQbVsZpEh1xkd87DhCOG0Zy8VoPhATGBmQMCAJLxsCchbc7ud3kvSKDg3zoGVbppceSNItbBkXGeYmuvI2P85MIwa5VfIf2uJVYtpBR6\\\\\\\/9V\\\\\\\/CJpWmFzVxO\\\\\\\/08Kah8A+NVKHmSE5pTTdi1b4jV+qBAEdiwRmPS+PLrjmMlRm9Cn9D5wRiWooNsNSrSTtRQJtCzFSN+Kmv0+FDnAP6klhc3TGpXHeet4mm9wiQE+WJaPwrXUh5MsZK+6I6yxtUuvJCzJuEiTqQy0Z5uv147qPKb0gKjZQs0eGG42zgyD9PIezDXj6uPQ\\\\\\\/JILInxxvRC4\\\\\\\/X+VSB6ioTB78zdSmxF7rX2ghECIlASWQVItfrbPcaJp4yU\\\\\\\/mblqRLrmvD4yDADAamjzpPjH7GiZgLhqMtrhHsk3em6+DIgMwxmfxPiPkMMdoMRAH1Yvm3PUbicLLgtjBrF3a66VIBqx2Hc3xv9RVnmttQv9jdlgIrr9bxhpD1fCx7pypkfGcIEC1B0flRdkt189V79xIgYM8DV+4gPZco4A6xOSCX2SSp8DOqevuGOFIOjpcwltBaTbiK7oAD4hk9NKYI\\\\\\\/C+wS3jZJlQvnlEVx60WyVSL9fcOsJlQT+AKsffHRjvbSPpUA5wwkz4bAe3SrMDc6vi1FTDGh3gH0lyU4BZP\\\\\\\/DuvP9awXIVQ1S\\\\\\\/90NlKztgC6dOW+Wo6hmBa5I7IhJHTrtvQDQHi2qLXvEAT6Gq2\\\\\\\/zFtrODaL7UNHSC9QAk2DhWD08\\\\\\\/qHzIYLlq4CGoAGmNVBnQS3GWoEuNsSETVmsQ8GjdXDAx6+zXyxBRWahWYnGj\\\\\\\/qDkG6DpFdy\\\\\\\/a0guOnvWxQGCfpzpGG954J0dbjq0sS1n7NPfeFosHtbklYE8N+1uV5IIV8UgQa2NdjTbc8bsHHe87YJvektUytS7+4sWjSPeQKcKo7UxDDHnwaAryiVq6Un1h2zTA+hAqahlCbI+l2AFqH7VTa3S46HxPlFb9GZdzvXclhPPP0xgeZ\\\\\\\/zL3OekChqWqSeYr3tPynvtbKbhpQoNyRac3EOp3r\\\\\\\/hSQcd5ANUa48WBJ+cjjyML6jWuJcaZaLeyaegJsSneKej7aLjj2eazCffTqlHgUltFnmDAnY94biXj033\\\\\\\/SHQtYM2CA1jEgqfu6cQIuBsmzz7ZRsfE0fBIkEZ0nHSHJBH\\\\\\\/Cb1j37I\\\\\\\/SiiYxfZs+162ZrII9tNxaWCWiBOCrnnBjrnPSfWHk\\\\\\\/BJB8X5dCmEs4IzmhlM8onxk2qv+1UPiEpsgN4K24RS2cNYdozAtfHzHwQI\\\\\\\/rRpDSCC1X0\\\\\\\/SUSgsi3RGqvBtZESClJOU2\\\\\\\/fNj1kpsCb83VRNLhStvBXsxt4pH5MwlxD2LcqbAOKxzJbXAXNlRY4PGC5PJFqX+cf+6NEfgjRqTpcvRKavotuFqQ3QdzKOPd+PmfASsoNw13fa8\\\\\\\/1LatXitWZ2b1vKMv5fCH1FNV7JcIcsWzjZPa8p0Y3X5Y9ffiO9jwCenoN1sb+ChaLoGRK6HSSpNMV7GhunKON7jEbAbhJAJ8XavE7hHRNe+hNxUjHl1D34JqDx7NgkpuMkvoNux81pBxyXQVRCyTq8Pum3nJB1oSJAf1pTbUKtJOvMy+yapVkZ4ezV+8ZhfGkGNJuUOsgyQsxzwLFAIWn4govgCxH5hjxXc2D2\\\\\\\/l8rnskIHcAt5YvTnixlDltQ1ayjEmbADZf0aZ3sULsF2E3XynyFQu2IYFHoNo7sqeU\\\\\\\/JCJ8uE2i5fZhgiwoPfGB8kPfXhEiM9ljdlXfxba68leIoiPBQpQY+6Wu983FRGzy5YElim6wNS00vt09qfxdGKr2Ce5oyH1fB5CCscaiL6t0DDw6jR4cvJifrgFYE6ZaYt3l9P7J3cqa9E1rVLyEnRQFEpPdk\\\\\\\/jtxyyhWbfjolL71tcGuTVlEudEPllbpi+btAQftOkJrrKx4ID1KZZ\\\\\\\/LsVnZIiUyFcZMcuLzJpF+ST+LfFQKOC25hIDiJtpkDBAUQk3QkIcBFzOBdBTopz56AWYvUQLDGWvpSAn7Y2RxmF4PlnU3MB9ZjEcIRI15MD5683Dpjq8k+7PSuw1sOok2UhwcrMB\\\\\\\/+Ylwtkxu1fbKBU4wutzbHoWgKEJKEFb3crLh136ykCzioRpKf1ajYAJFNZk08Ouc0+omZlry6RjmJxZv4mRck5XYqt\\\\\\\/\\\\\\\/gl1C64Xq7vFS9htiyXzz+hOLIczlFSf1qjJQUBmQL\\\\\\\/yiVkLCNKDVoBCfHwZS4lHSr5LwX4EZQci4I7FWuVZzyRec22ptmeH7tMioBgtdlC5tSdJWUtXWhl6WUetJUxGMDRNGG3aMwYqgHqkkNPNwmr1efShgAW1Bzwkw8M5N3wmoJ5qCbhdmwF41DKcna9jEjwPwRPqbqSsADj++wVlf+Cq4X1E5P09lTmdXU02\\\\\\\/vZZaBxCuwWLkTu9IYNyza9VXrUSMQ6qdk28mUlPTikmT7iJnPQX9lRLUEzYSI3k3uRftKV9\\\\\\\/83OGMS7zJVje6m4YEYwBKK6KG3rMdy5\\\\\\\/JSYzQJzcqIzLEVd7vihXdXVF3P1ywbjUJd\\\\\\\/D9yMKq1vO10gskYyC7yEvHJ+TxvQkwbiE5mGZzEYaOFoF5rNiqjZ5hgz00xd+PO1oW0f1J4Wb7C2oktgoGzXBK5VOVuJrErF\\\\\\\/QjG12ztwEPKGsMJ6WqR6yhI14OZQDj+AbfuiB2\\\\\\\/fOCJevZ9UOo55IfNnna4iNxB94QGFne+e3E\\\\\\\/AzSDOoKRD\\\\\\\/nLfKvHwIGznyClaHCAJ2y+eZmLLbiFHGJMS1V3iwrQueeKJS8bLCQeTqo9RADHE0mTcvuvoyk62ed2yHLgZo6GAzOgt5efYy8SRmJXV3iuvbIrHDhdU2+lYMuivhJdLlWWJr7jujC+q2L8h+LgXf3hwyf66cv0O0jPPfgRMJiCoF5LevRd6Dx+70X6j6MiB27wUdLmEqg+p7iy7JCgX96OpcLS14lSvoNgXWJw4SNjY0qhfd4EeSw5bWqAHDKCCMsMVE9BD+tPtCxZW3DCjBSoisqGoNXZaeE\\\\\\\/kAc6AfH6AAp\\\\\\\/ymQX+8byF5WzxP4cDp2h6o4yNMJEvSgOMj7v+w3iMm1pO3B7MXVFfSDjn3l+7SAYn1wWDkjci90+fkL+1kWOCanrwOD3nkdJmXa3cg9bjos7ITgTG2Q3KlDbBN54MJx1LJI6hgLskFKkaqpPSuo8+JX3nlsem0eAkjhsYwzEysj8jAD0k+dY3XXE+Adzsepf9hkT0IhyBS65hEdLIwrDULEx9bLDjSlBuhoEO1tl7dlX3nxH9otWtg4Jtlit219K58XG8NPLtdj\\\\\\\/hFA\\\\\\\/uTbq1VawbO22C\\\\\\\/bcp1Zmdl\\\\\\\/+Uy5UAWUZoPx6NUAVDaaXKcyvuProHL4dpSP1HJ\\\\\\\/T1hKScdL8j5eVUKkMphylcUiqn4wQhHCQGqqXPLnQ2HZow13RJzFzLRt7w2sOjMrST7ijW48utNI6+vDiEh2ca+Z9ca3otMS1S1PHkrUXhJ0MGZFRectFkj3Zks7pGw3kWbm7OC4gXzdwriweB3vY+rcUuAcGkoU\\\\\\\/tun22ABa9cLtu8i7fSljX9UGwo5Wmi6Q6KraphGucDp9xmgJAQfKYVKTzC7ITUFC\\\\\\\/qb0bEOMaew8FRLiU43nMM3GK2NlrThM9vL79LiKoeNmj5FlawxqpeCn0WQqVUXrPKh8PkwjxLPZ4KfkDYRpdEmyqwDKIxmLpMLd8xBVROM7MHHLm1hNVszJVgd98liA8pNDjgjoMAMf8TQo9VOFyvi3KB+Gdn9tqKt5QRxnw9CAkjHAdD199Gilx6YtKMyE9z8xhjY8bgTPV5t8bgKJHQWphec9LxZv92\\\\\\\/zOq8cr2OyPNWZtzZfREr8Zk\\\\\\\/UNwyNuCclR5gBXWvlY+IuN0HRTdiAJ9qay7Ho3uW5TvYetwi+Xo2hQGecrug1YkwDkX4\\\\\\\/gzPGGYeKzK2I2dBfn+qxIJ0+\\\\\\\/sRt5Pen9sXsuMiC0\\\\\\\/17kRmbPtP1+3sj10wuLDaYIsfH87oUChbql7wbXe8bFuESwheDCYicuUDxjjDWofiNV2O8N5R1jbHbCHdU8HqHzl9PSoVbCou6K+4K\\\\\\\/IYlu4UG33HxRGwTiMAza2CN9LzeIaiiXOlDxp\\\\\\\/ESqreSQGuN4ud5yhAEN0uvJFxbzNeFACLtVZKFX2U\\\\\\\/JSIgLFOWt62JKbmdd02+fwQJJtrvipobzp26hOzyZ4qN58KBFBwARyaU\\\\\\\/1Xu8ysiZath7MmZ4XkIqRloVXOeKtdWa7cxIV9qbsEybE3VVM0w\\\\\\\/DB+gGnhnDxHyk0tzBeSleC7vRL\\\\\\\/1VHdVDWo4xyfMYl6f7BKLxhZX+G9UO1izzL\\\\\\\/FnL+GQ\\\\\\\/nq6mkTEG3IgyNH0wfdHJyp\\\\\\\/J4+0u8ArBP9BMa0IymlrF7ZYIEsGdxix\\\\\\\/lpuncQ9vBc08IfVf9pVMWyw6MZHZYwQ0YDBCvj3Q2iQXXyvas4SZSY4sh5sVWuViEmRHVWuyGYGCbyOKfO1FkuYoAidDu7YFxuOIgViuRQ+jLUb+SQ34CCtotTrk\\\\\\\/6wD0Wi\\\\\\\/f5D1R+ETtsQPy+Exrk3m7s2wnMB8eIkHSHd2fGqy\\\\\\\/hIQmJX45JSo1j3ytLiQY5CNtA76Dn8SD+pFai1cAJMfB57sc6dUu9h\\\\\\\/CfAe7IJXLWMojnfZtSA\\\\\\\/E\\\\\\\/bO4DEDtfihOfiBkPq15DJCEL7QR\\\\\\\/onCt8QX5nAM3PKuGD+bNuOPz\\\\\\\/Dd7YPujcwsH19OuCRzHE7Nx6lbS974xPIohhBP2PBzycecR3WFYX7fhWubw1LfTMz8Fz7YPb3rUZiQ1QV5AgabaM2f2H7AmHhqAEyB8QaSo+yrE2S0ceYothlnXdGJR5gakzMWKyoy8IGsdSDXioZRuCM4CWgDgtUzM9Bg50X1ucIShfk11956yKJ7cUpkej46X5TrebfWl+XLFNPWv7Nn\\\\\\\/v8QjWCEkP4kxQUUhLHUp5e4L6aDx1V5+FBWl0MfGW03vJCMTBhaudCee777Z5Ryspqz6yXJ3iTSxz7yb9gW6jTvoRIiv4O3GCRhXN8fS3DFHbiGTnz8W+sz7oVPL8HlF6yUT3qnUha2ka7mIMvyTEOWoO1AzfVoh1gWIY5tb5pmPEnnOO9DibAHfqzO0jDkLnsLghWHqW33ZaxkIKCOiVzWK7Y\\\\\\\/4F8u\\\\\\\/8lt\\\\\\\/bDdGbeXm+tl95IiSEk1IkWa1svhoI+eOt8epLD3CCFA7DUsO75zQoQt8reAMUk1npmeDAsHx+cyI4Sp4J563y0LZ9PUY9hB5XOgcny9UChyPHRedZjk2zA89i6IGXgaDSKuMk5oAIPCRLnUx0GOsiFQq14kfiEdVpfDcfngWaP7nBc5KIeEJvT9NOhkW47VMEo0EJOxhoXqfABJOf96xlX\\\\\\\/fVSKYLVeWMbUX41LxbDmoRDBNKkS1PWODG9H2qGzd+q375LDEbXKs1UYJ3p5sQtv3w01x+8TlgBdCMao6mlfj0NWLEEXB3lhZOaO6kqBSvkgOBjn+XAIWQbKaYpgFlfHsXmAlsVTdf5Q4UuKBMBrKSFhFSiR8rrevc\\\\\\\/VsM6yOgwnJ92LwxGvAy9ycqPPZ4RyXtIC5n0udPMIrXI2FkZQoSdaa7TnGP0Et+hGT2vCcyK7dXy72YP3Pj96iUSctSkuW4rufarzh2wTRlXzpxKFQbmgbBFeU0FZPnFjEU7sMHo8GXa8yAPh7eVIDhu+PS+yBI\\\\\\\/mvF94C01DNUdnUAQJoDChPcLBk\\\\\\\/4LP+VdpTJVusUyzkCiLui60yduCFxeA5eXQF4F4fYjJ3KyV0Plkl6bx1oAYncyAK02IQptSg85lsTabw2slvdVyGYUIThyyKdl\\\\\\\/90B0uM4XpFDxId5esYT4sVXQlVKA7XzC\\\\\\\/FVjBc6Ao5YFPfn1NeQ47\\\\\\\/kYui\\\\\\\/WQCMOQd7sJLKXMjxP6UdKIgR48I4\\\\\\\/9neZqk7oBg+TKSW8SnSFDnTXf5j+YkfZGhm\\\\\\\/j5WmvJp0zHNj0DymqclXjnbxGcXkGuzBcNS15gNBbbQOnEFzFi7Nu4fGyjEwarH5mJ\\\\\\\/0YGA7IOyBefJ6op4rcqbL6f+TwvUhfhKaEM\\\\\\\/pDsbR3lgf+WIIj84yH+YjUICB1rz3a+aHyEvA5IsB85kXnSDICCYlqxk60ub7hHLoYnNFnL8wctYEl1dGbdymTT8PaE1MoolI6ynoIem8AEBGDo\\\\\\\/GTRNP8zWWuyCPNtzReB1phB8bqlPI1SKTQnI1Rm0D6qdL03wZtZuO4RcAjJqI7bUFnXoMVpSnJagBAcr+1ZmOeYnNMS4rAlJA3tcK64Rr1rqNgBRnU0eG9jiUs7r1sTFNJ84nRuKjGd2ICs4OeNc2IxmDMR9akam5wU7BmDxRc1MunyAQYoEA\\\\\\\/yIWsUOyvH4kkqT3y4+MP0sOK944cpC\\\\\\\/bw6ATyD9rRB5ByzhaMscfeoyq1KUvByRfWJy7eVCLW1haIBA8thNMEkkF3sCnkCJkXu8RBvCpVHZrqZ+uM4duYSBCvBSSOshO+zyO5O0mRaNog4s9WCafvv90GNqO\\\\\\\/ip14G67a\\\\\\\/dH\\\\\\\/4HLT8fLGQ\\\\\\\/xgGtaC2T4YyknBw2HtRh5VkSWBSzintQCNSnA6yfKMv4sfZSeI6buD0rBMHMHbB1eXxScwri\\\\\\\/CkziSE7FVec0QCUOgwE+J3AFDfYDAWdPT+LmTAvP1OYccCr9OHB1cSIwbykgtUFSDlpmCdUFgxpkozcYbeiyoxgn\\\\\\\/qUs8+4KNtGK2zKPsJxa4g\\\\\\\/Hs+NH+9WZkJi2jJDVdYEzLdhvycmnMcxI09+TXE5OM6U3VW5MU\\\\\\\/WgCt4GTBQQ\\\\\\\/4wlrJD9HqphtimNgapHLhWa0vGK0DRdaT7plNm7\\\\\\\/+\\\\\\\/bSEWtzCroBU00F9iazRI1aiGapWAreGgT3yP5W3jQK4wnbysg9xBFaqAujt64pfT3b3kEWLsK9+wsgm66gHFeE7l5dX6J7h8i7qUe2rnCYnFENETOwpISdZ3w3\\\\\\\/Hiry+Fp5312IJ\\\\\\\/yz5hWl4qRunpm4KE+UqHCw4nsMtoc4QifUjUB3VBugG5VqC1gUNDJoqgn1MIasqBwg92puGgwn6eT01NS0p1eS8T1w6M2blf8VXQuypbMRUXq5ESwr9MEyUZjUz9A\\\\\\\/cgx3Ybi5jut57aEs5pdp0CdLBbiU84lSLzCy4yX3E0xYYwrPLFgVQxhU5aOfpLMXKe5yHK1ZNuLU8sxQ\\\\\\\/uoMgE+ivMmQnZp8XTQ7X4UCeIKp94UEG6iDzo5P6pBtdSmXEGGrVxe1G1OwAZDF2nji48EXSyFqb6fnLZWkWCVNIVKLVHlSfApgvGR9FgtCHax880b2VmCQ2XzYeKHIJ2mXzy+xgILyYNs5KKWC78upa1+RYoi+HQIr71lY+WKiJJ+dZk2tONfDK\\\\\\\/izfVz+71oo9FW1zRs6720N9dD+eL329v6Ly04DIBgbmAN8J9pZTXPICp2+7k23SxPEki7VUCkFFz6LiyBXRkac6RM6wXfA5nfUoLPY8\\\\\\\/gKwNKUtUUoGGnF9mQYyiYau4RsK\\\\\\\/RrMz+uGBqHhO8JLVoD1Ipk0uNVv7HJAMzz9jFnJO8DrubTaa3T7dP++DFYYPnQ8x0XydYUzCh\\\\\\\/VWZjJEO+BEDOR8V1mD8b25MR0UkgYW7hyXZzx4oVuP\\\\\\\/Ua5vg6oI\\\\\\\/XyiRVfOjs5P0tr3OAI1oKOebMW32rs68XkvO3MFVaOVSr\\\\\\\/UZ6oBICTGttjbZEoBV\\\\\\\/2AydMFAmSzv1bx0oAZ6MLWNynhn3e8txD4Gm1OSEDQYkU\\\\\\\/H5Oi0BqCEONb29TBPltjBOsBzUiPr+fD3iOo+wuPaPmmQgPH4zm23b5mQome51+E+LcXP\\\\\\\/x0GPeq\\\\\\\/hPe6iUeSZLlDJLsolLu2l3v3n9yKsP8Jb7JpabfpJjaCgA1HV5oYx2qoOxG\\\\\\\/C1bd6wF4T1vG1q0KbwOhChtLVV723DYKcBUK1IRq+9CydJFWRqEQV4Kczi8V9M2dbmIOhfOvnc1QkURAgPmjRiUXHQM9ZZ8q67x\\\\\\\/\\\\\\\/XmpkkpQ+z1GHfJ5roFSBRH1V+hsjh4O3qrGXBYWSIe9wc3fPRJhevmUHCy\\\\\\\/hxtB5YDFlGP8ycTwfOvLCPKui8kQelpsXsyWe0ycphGzrwobRNpClEFqzuNPghjOT+Mq+KbiUuNHYQdn71ldgc1Z8OuIDwU3EZG+QKBR\\\\\\\/pxK5jKHLEazLJGGF885y6pLRqEDMzFRX8qjfJjLPIpRIS6B6b4mpc7oJrOrK0\\\\\\\/8DlMpT0phI78l4+CdrIjPPfVntsgaGpk5weiOaz9\\\\\\\/PO0svpwpzXTJhkVfRsBARFUqzXNbA74MvedU6oRmHTXidV8UIALkBnH5wnPULawGPf1\\\\\\\/rAscjyrRWXSep5wXt7JsBzX85x1pyPeQqwqZyMzZMge5RXIhI+9aPRo6wwWOi2c0td5iCRfe+99JCNjkWLQX1KTfDLQ8yU5q6aDb3ANRBprG7baUY8cNLfjuuOjsXfDKKNjcuCwZ7hnowEes86RfE+O+DekWfGYAx0426ocM\\\\\\\/8tyN89W+k550ARByWRDQxEC6TFwV2Qo8g9uLmqoU0O5A4QHq\\\\\\\/+d0TY2lrjnoNmrEHQsrU0Ab4c\\\\\\\/V0e21scYo6bBAbGyet\\\\\\\/x3xVt1XX5rm7SynjP\\\\\\\/6JlsJ1FYp3DV4d2FFUZqpxTOcQSpydVmtDtRLi+e1vr4ZxGSFeFInNZeRjzzk6hPkAw6XvzvF5jZvtVWbIdgT3r21wRRgKzDk4EHdilUFohJSTOVaosywPrhBY2oZAQt7VT7vEKcm\\\\\\\/WHgwXKf7vNeCknY6eKFTlZ9+NmbqDzBow6+p4y9mDl6WCtQG5lVxrm9gIQcz2SGtM70xvxq\\\\\\\/ayUXD4mw\\\\\\\/G2PJhza9R8ggHyJxRY7PtMqQJoU\\\\\\\/AVjaOXrfnmDArYeqRpuG3UB+OfeUkA\\\\\\\/pHAqnlU9faKKSTQxqc6fH\\\\\\\/K5yt5P30cqbcbEfh2YS6US6OetfnBfWgu08BTKECN70DjyebDkri8WTMOV\\\\\\\/nBDGPJQGvXhw70t5MhFImq99licfnBVZhDly52I6mBlYVnWmIMwhcw\\\\\\\/Dv3ZJ0VBIG9Z+TG6AmRJynjsINqOvTDm91+JvoiWcOz3KXdFgnfdHmFoIiuuXrd\\\\\\\/0n+VsrZDx1qb2\\\\\\\/zDApobsonAJuca6w\\\\\\\/YpWgsQ7ixKjtmLzsFyV0cprGcmTVhL2YB1bAuHAUBC\\\\\\\/Zl1So\\\\\\\/ocsaE+7dfJf86AQSRQWcy6vHfTqOQ7lL5fOpDlK1aP5lDkvCQdxE0uo7mjZ1xZstFEeki+AiNH6dO1YVEM9hap6Epo0C9tg6wg3\\\\\\\/fUKzecOl5aJwUJhBkFB6YJMJSLY7K9+4f4zdlbbJkULB9cX1tr8BmXCX9xgfPxE4Lcop3aHC7KwCgxl4ty8da0+bgdCkSR4OGXKcATAhR9WENsMK7pHLM2SteCXLu9LWUXkOWaR+CgdBeaBsVBYvv4cnTQ\\\\\\\/yMpukTI1hHHs0KKfclWtI41XoWiPIIR\\\\\\\/Oy2fier81ZgwkzWTQQ6vhF5\\\\\\\/WK4XlN\\\\\\\/4jB9ABHLbaW6rOmMLC92UPE765KDmqhr8hIDDjTzgsb86TdDcuXd1Mtcr0dOSelkO29SxawC\\\\\\\/LRJA2ROpq6iF0sAyJQtdwbTsqPY7QcWarDUd49g\\\\\\\/gitbIrnNfkFQhIkVaY4tLsgmNjmB8h6gqVEwRY7HdD2YJvKRlVk\\\\\\\/oVhI9OEk8Nt6Xg\\\\\\\/ZI03hmrT7MMWah\\\\\\\/Um3OLgmLh7BCXbdNgsqUwb2POz4PbZ5zaloC7GQ3kXzb3cJjpeXtJGqy52k9VKAAhkQSHU\\\\\\\/Xc3CJEp7LS4Jm5ZElzXcMUqylkjfRJVX4sgtFPl\\\\\\\/Majud1o+3euVJkiPd5wDA2\\\\\\\/JgNj\\\\\\\/9OLoBQ\\\\\\\/tPTYnAedOnl8aPyP9ozJD7+f\\\\\\\/fcfHAt6jVWQZCA7YUrhOGBsTuk5XtpMbtHecsSf\\\\\\\/qTmTjFF+zUU1xkUyvmjV0uXZLo0LRSDMBhhNRwUQU6xufvAzzOTpbZ3HkJ6AB+0lSGXlR+XuERTJ+sTYujjQ+IK5VLvl6ft5HR\\\\\\\/K6jjRRYp2QPJXF2lKtELhfNRm\\\\\\\/J755RpF1B+tP9whGqQS24rB\\\\\\\/3+JOwFgBOmu7c4ChB1RoTXzkozM6ShIU+kupMymFoS2OGVZ6zEcz2B10NEM47ZyBboUlptAVuK692pd52Bfcr1l7EWpKA4OfzdBLRR7ppgfzRcS0Xlt5+jdVXF1cckMOetYhO\\\\\\\/6NNffor3n9au0K3aLTZyB98iFakp8vB2b1JQ3ab22dCunusQMENSAQ4xN0N1BESE4\\\\\\\/TXPCSLb381J8iHsbPxwfShfqW7AyL8fx5tl2z5AJLI883ZuoYDI0szx22lTwHma0XpwHvbj1y1kLJhEqtxqTb7EpvL91jM52EXo9P6+ZdlQr\\\\\\\/DMS5bnUwz7IkIEtVT5dg7WfqQMbmhZxOXJVaKapUoVCf3pLPz+zkVN9PBMjgsXawcG5alyR+5VoccTVJ9L++HI9RoQCsU9HD0mMnTrT0nFVndAmOpRwp8MWnbhDQIvCakR4oieBluNUw0ZcA1HExNh04iu0PblUHQpHEo+bMtH\\\\\\\/S4Fweo73DaQXzN0UR3+KZZYfdyZlQsCH2BhfxhO34LplrAJvDEuRgJHzp9gTMMybFFtpWKwJViB3pMYf5RaaCFCuDutmgJ6B\\\\\\\/B5cgLBaNUFlWahFzHQNq0OjCQaM\\\\\\\/MBUF3UODA4cSkW2bbFc0WTT1DAa5bu92A7UroCq1yhPcEN8LkoUlFus5u5wMsn3jESsELlMf93l7nWceUG+Q4zjVR8DVfUiDxTi4p\\\\\\\/FGLdigZtIUe\\\\\\\/uYFGNmSsYeD7M\\\\\\\/INurQ3ntGRwhOX1MBPPYxssMO92TWQ09Y+wK9gFmlb2vSRU4N+RjVUZbnZWRC3umiNL7PMWMjOLcBmRw650z3r9CbfWWhpjeT72VgL3RpF2JSy0TF6ZnP+LcQCT4n2WJW2tzBlxFHCAfYm42xf+Gs6XM6+1V1Vgw7d+FfemtvnZU5GmWPgeo2zyg3JbHaTU9qNg1BGNFjtZahblMaTRS57os59UWYjqrslIiStM5QGGjZ8f841TXfjO2c6jYIUao4WcBHUfdVO9jqIVdZp6mxDUpDjMWY4zpLIKBgvEyu9Y9Tp4MR87ExEmxGH9KYFsZiUc4IuB08ChYijvklmcb2r4Jzo\\\\\\\/Nr6N5zDELbQK\\\\\\\/DP0vTzDe6XWJz+C9Nrdepcb8PJpvuGjvVZnTus6\\\\\\\/BjApO7wrs36+DzZ3oPTigG6hmIlcqol1bZi3vkc+UN3ZJ1mC2lwgBP3sGz8f34yYl2dfU+uaAoTrPQPLwHhBxduqOwM8B\\\\\\\/k4OJEI7fBwTCOF+6UFxqKTWDFjM4RSgrwiTaSZYEGtm71xQuzHxevPbmZXI681eZzMe+v9FLl6cLR2BrVLal1ETmXxKQjp+qMxChnm5n21pVB6anyIqzBAv7X5rqC5nncMofqcuUyn2\\\\\\\/JoaLjoiDHlT2IvD5RbengNxmMvPpecjjQtIp5XmzulLdCkI2cmXiNRn14CCf+vfJE2GzrQRMI5sRL2WoFKl9gs8zMA2tMAGKaHOLD6x2CTcHdZ+5AtczHvnmiBON\\\\\\\/f4\\\\\\\/oy0y6QidqZgG39x5FEMRa8OQI+g5iz5CIzwNQy8+fY4YPjLA0GeCuhwFU1hh4aFUwxnBg9D5LADeantLuyaiE9xCGa2+p4lSqyyOosSNJBkV3\\\\\\\/8hfXtSZXOOHgPl5jxqwqSrvWSI6TbGU5cNuQNpVMv0R6leXw1KbRJwfLSpdw1kSRUtTo8i9LhC0PsrI+NbgLuWrYsI7jFE9QZw8Re2aGOG1gBzozYzts67cfOxNWlkuCKWiAxp98mwOqsxdXTOXSXJVaRsf\\\\\\\/D1xuVooFQY+rf32SH9uQsLmXsxIlS7S3auFQ9SRHlMif+OP4UUdYyR2X31cfCzkGQu2lYVDfitMiKkaGXGCVdDSsVORUi5NiShnKl5pCkP2IC7aFbRLnakqfiCkhaAWYykp120KR6avkhUdjfR+85zK3FQKTm+747kGWt5tE4P\\\\\\\/Iu5ztALsuuG0ispodQmtc1xFQs550qHvchxhZpJm78D1rF3kJTK3kxgsoTtGsHohmtAufblMkP1N\\\\\\\/fKo8ztmsb4oXpWCIzLaZf+cGGfuMat3+7IpwPSGwtGPypEgig1EwSGth466j1S3iTxK37fZsu3\\\\\\\/V3Ecgi\\\\\\\/0+YLJfIrwYQ3NKaQAP4qABqdrf1RgMCeIsl1dZfTrnI\\\\\\\/5qmR\\\\\\\/o7TxmUIbxi37Ci0ITn2zdxAMb\\\\\\\/SgPOVMUqsDL0Lhl8DbYy4Cukm7xoKSIUk6hhwDMwiaa5oZeaHH2vMGNzhs2Z1kw1GPR4WvvHXuAWzftx5joNvNBzbnbLdfFSUjtPTmaG8dgPNRdmIE+yHMmYRRmyTrs2SdMjz1hbnGDxCTHWH0WqHcZnGYQR8emPLkBI1RsUAlDnN97Egd3nZdM7YKNT+4+pV69b0BDtKOVZVRaZTTaF+3KFNSWTnraoYi8xKO0VvF8c\\\\\\\/VJ27dhb\\\\\\\/tsV870Qj\\\\\\\/emnuXE96qVWiKoj88lsbI4hUf2RdyByvbauHXYSO9yvnBMZnVWvoPfOOvms3SYw1PbroIouGWMYIX9h6znpgOzry\\\\\\\/3yhQg\\\\\\\/i9sL4GlXjVJqSV7ewh9ACNwzoeGGkdqttEliAfr6RxnQN3Ri1K8wdFuwWfSf3L9dZRgtKKNRefEED3hKNrnb9bPoQbVoUNTsuN3jzmcL9HQMp1xJkFZnOlWvV9135VuUiyx83Ro1ACQnfi40GdPwP3xd4kzFfSqXITkcrHZLnPk800oLrLMECNRoWmIqJEeGZDYIUzWcwHysGnAe6\\\\\\\/oCj3Sl\\\\\\\/epDZaLYJcKFNBLlD6tP9sWCrkdoQbe1lGPChHhuVeWaNgjCWUAakTROyv6CTmUJBHqUoTOXknpYw25y6AnqoJ0hpQBeqdiGgf0LeIDRZQugoQHvergH6YmMRDszRPNXYEUf048\\\\\\\/1Kug5NWsZNu57Qf+lxWtrS4tAeomz5ONnqboWjYA1N+1kIr86ja7AqGJD8ug8W9ESz2tHlAv5P19Gv1B8TZzVzfw+CbG5cl36S6\\\\\\\/SZZyY48EyBoyQeeAIW1D5TU\\\\\\\/1ZADCb6yYVBasKA4t4fxBEsU7UGAVxAm\\\\\\\/r57jFMDQQyZVDzg1JY95T5hwy8K+3kxtvNuXGZcaM+sSxBbQ1359Rp6Tu8j85pEt72qp\\\\\\\/RqmUWgSeJlZq8u5TetPeVKetBF19ePKOaPy+d\\\\\\\/X5sWpJ3HyLRZHT7lYojvB3UTcN9yb4I8kuEUa0bs526CeouBi4pOQgq\\\\\\\/ufwu\\\\\\\/PbEQmAhtg4Ri2ryBiJc+0maGvzeizw\\\\\\\/wtpTFq1I6fHWNWID7z+6aCH0dZxew9Sm0faN76I\\\\\\\/LdJAtroJqli224WYHQz1NwF9KeuWyqQ4sxiieKcUZZNDMgUP7Kw50try4\\\\\\\/22iluXprl9z0P7W0BM5IRR4mf9Z+3+5eALWl1gIsm5KEkwkGtr2YWHkA6nYVbob2pzkKnhtdv56PmYdhkzE+1NcnjqcKsEkRd7tQmaawF+T6bKpGPc8UJ6WRFSGj\\\\\\\/iDS4T8f\\\\\\\/DcdrddHzU2q69o6khOnOaXsHJtYgn\\\\\\\/\\\\\\\/MtStJMYmT0E8Y5AG5ODV\\\\\\\/so2xzHAfT+lvmEaQE\\\\\\\/XpkqdBpOjag3hFzqqnsOCp2M7R8jF+eA5PJUMcdJphm16PGYLvMxCugrMR8QwgM628Vdt5LuX4i2SOo+lbs7WrJO7sBvfYYYjX57gN5T11dEw++JMxQaiPt2BHzDw1fQrAMnSIuMBoYa0vDko1UFfqSJX1gqgWH1pFUlDrYbbiAmS8dNemlZbzVwZqgQ2vsbaZv9nHDlXAJH9OsLOsAaq5MK8YynnlkAzg7lnzIQwIZT0co71fFN50V5YvwM1zK5OgfO9FsFKXcwPqvio+VPMizzyuEof3y+3jUtvmkMOc+s06LPf9cHnZsr8hE43LzxqIQwyhbQPk8sJQxUVvnU0eGUzIhjZzjctA8KojurbrfGfG7hd84xnPFEvBtn3AzjVmN3bhtNCwOrcbrxxsNqYRQo1AnrCxV9soHE4Sbk1afKLPi0ohcxIFFkC3PG94vO8OMgkN8WXEUBt8scGUcDehY3oA\\\\\\\/ef4PF5ycCvyKtgRungfWQCqqDDdcLW+zFQ9Py5UlePmjVlDtHXEKXoV2eCjlsa\\\\\\\/iJiMStpW8to7oH7Xuh1LrTFpXwpBTvTqZqMsKwv2xGQa\\\\\\\/zeybtW98bYr2WTrace6U3G96p4tyBM7PdqXrnX4bNun8wHqneN4LOI4\\\\\\\/WtbCdYTvnKDjDrGYksyertl70V5s\\\\\\\/GRy+lIl6Z+y0vm1MsfKV+JKHyydj0VpFkMXCDnGG9FE4FSwvO\\\\\\\/kzbHAWdZiTzMmpgvG9fCEjce2XeISmy9h68xI\\\\\\\/05BmVMN3w2hlT\\\\\\\/Un3CrblnMvxf6w3YxpJa66eGCxu044jr5sQNk\\\\\\\/lSPjFduYUyHokaI3lC8B6HTFrEYKB7TmYv+B0XmC4Ihsx4OCXD9Qxf3giJSI4yBpBAFsx3xdg\\\\\\\/luDHUr3KGcWK0munNCE8w9HdXywUkaQPp4xXZJosFumqJiFFu8jhyzLT\\\\\\\/\\\\\\\/dp\\\\\\\/oi\\\\\\\/A8lhAYn3b09t6wj8R8gUgOMRiriVWTrKAEjqMtz7gc6z9NpwGZDP6YSl1h66SRYtFedQdNpVWRukdUTDt8UmnqPkX0jQw+uXzvYkv0HiExvX2pDuYXnylQWheu+0gmf9+HHhOM1sg8q2qG4JT+wVXp+ctnS5JHo4m8oIIQD03zs3iIi15s\\\\\\\/dQJysL8S5DdzQvxgLNDucX7wIbXOhDC6XRW8OrjLZmZSWnYtzf0YtnESxkiBfroEOE6Xzg4Gw9PfYlo2gzzL8pKzeYimQ7aXDqV+0XPB5Z9LZlFF6Si8Lbraub3vmPf17lpUPYogg1+NNFYajzvUsSO7pczZIJZDa5Pgdzzkk9DYiB1LdUQ6Pq72jtJValghKsTtyYZg3gqSbMYFkGpeCBLH7Ta98zmar\\\\\\\/\\\\\\\/QQHYZxFA+CYGLdCjOEr+k58cgcxkH+\\\\\\\/8Fo6\\\\\\\/U1eYh00eVHE9LwO60wJctZbkLCFCkv4UWOmVeYVljyyexJegbPAQKY9\\\\\\\/+lLsm6+tuW8PN8QG+EX6cHXhEaoGIABrAXskEUvLEJic5GuVhUPf8ZBZ4LHlQVLl5R4RWE2spCS+it8beZbVfVSanDX\\\\\\\/hUuWRhU82AUGthMeYjJ+tRXQTw0a88MUk9l2OOOsidO6sVq7J2RAZbDf3a5B6o2gHqwunhxHGPlpJyqTuThX515oxHqtzsxeVfu33rDLTWGs+PHjBV4RXDCTjxg6j8yTX6u9VGZJmmD13UCMlNKQbw+L\\\\\\\/OVYE+pqynWngWmYljl8eE4ZbhuR1jGpDYEKTBozhI6JQ1sUxPD7RInsC+VQ9+wPoFYmF9SpUsfKJzUldOuG0sNRmdSpxLQxhuGZ93VsKroqt7NKrfQbVGR2j0vuYUcPX4LUH45wgwpQYcv244EyVJodBQTrgNW9goMqwze7FaMhojT2siXRirQLGupm8vrlk\\\\\\\/EwuwIb41hAAuBDxRRns5fSZiPHlzZtxuqHZbe6mNGt3hqbADt3TcDDDca\\\\\\\/yCDxLnogyZjCl6sztdzTOE922nqCSZwUZFlqkMBVmG8foo90r05UpYydc64MYCMDNHonCbDo0\\\\\\\/B9nxbV5qr2IGrsYhsYY3cMP3cRJhWU3mt2dctS\\\\\\\/iK6Cmwy8h6zQ1pCoUiZZtBQlBElqjeON6PO87qworts9yd3UeEHrMbObA9bMpbgAxfSSHgoRTtiXRklwZBfZl5CgvnQlFV5CAbigasngmPA2GM9bSqeaMpqdGZHwQWeiBuZegVCSGrmMahhtOj6V8Tz\\\\\\\/mO0ybjO6lYtL3EB\\\\\\\/IKJpj9LKL4cukj8cRbpl3f5q0lR5Zvykh5o58EXRosmm2NVMQ8\\\\\\\/6GGo7CBKbwk0uYVzRrL2Z3gLmY9KLJ2uVseYFR7cErFHmuyQpy2CUZuw7Q4F1k9fkKW\\\\\\\/k5Dqx\\\\\\\/kSXoa8gb+5lq2BV5qQO5AYEAxFJFI2vuHR7aQ0Ed34OP0OS4E22q\\\\\\\/AZD20kU7EwzesCzPYjQCArQiZ0L8RoQ9xUqFaxqf\\\\\\\/MbgNO8To+xNBWmZAxUviZhEvWm3psV+4cnA8TEt8yH6oEObwg5TiiC50pOhCPGpSRtBL+fZoxEj2B7lApzykQ5QfG5OKrCI3uRtTJnbpEpUUMnPwAJJv1Vwp\\\\\\\/WCRmlAiiJsalJanhZbaIHEx1fj42vh4tgxT\\\\\\\/mW4+b2WgipUJgC4jnp\\\\\\\/Gdxm49OrAHgpme7DRloXkvWd1fYAU2K6jd9Nq17T3HUR\\\\\\\/etHgSfLxs4dgXJ2C3wl4m0O2MVnHHNLBLfQSHtQgJOTK+0fo\\\\\\\/rxjtWdM8HZUIbstlID3TQ7tgAHaDjF3m\\\\\\\/jja9RlXh5EkG\\\\\\\/Hw4BVR6K\\\\\\\/\\\\\\\/4sEcxRRa0uTfVZ2XTj4T6BbfSSVgcJoTL+s1eQp6YUFJ2NvsbZSJYj5vkoilw3irrUCBmYF7OeTP8aAYyZpIeYuZ4hXjyUKOkEY+PTV49XFJOG+YaTMyoYNtIqpk+8UoRvjcBp0tZstsVrM8DBbR7MmKMjHGGyiENWUlq+8CWzY\\\\\\\/p7xDxRvVIf2CVXPyGpYH2H2DZestaNGJBMEfcPdiEOQGvg0rCFqDJHnrMWJBMABU9dW5I78XIxVWs\\\\\\\/nB9SWk0yUTEKMAPiRu0zv0XDR7c7wHLq\\\\\\\/b1Vk5SnVamjMgrvkvaD6dyEVeQUXbQxCOQ+epArGavZVZKAfDcrjud5TOM\\\\\\\/CnWMqpX4Co4TGYF3NGrHubkk58cuW0J6blVaIMECqdV3XTP0yfMfQuB8TvIOnJ4v1aIlVpEexfnvzCkIJyckO5+schjR+\\\\\\\/0E8o93wyw3TKwGhBP9Mlk2uM\\\\\\\/tEep32zSEgmsAB4Z1jzFHtu\\\\\\\/RdlUFhC6uDgYGCvvweGWXwip1Iw+zqdrgoq7wYYAj3k75sxwDc80nmx\\\\\\\/KMRi7WK4VJUyetEG\\\\\\\/9WEKg3ns6w+6pdDM9T7RPsqBs7GCikfeg4ftfC1LaCzEoVY1gFrZvf9Gzu46oRoT62YescCLedGG0gmuoXMX\\\\\\\/5Xk6pa1HB95PFIPyGSb8d9vEBdDpoG26gDSEvkRxz\\\\\\\/lxvHNOCKiq7kdPY0yA74XTADn60eyBtAcIfsCtNnPZa7kFk+QdDfzUvO3fNP8mdo4bzUaL9q7LXoILDEYkoC4j6CuUkxtREWsjLY1+gcY4i0xiPbmaxWSEqrhF+CIXxo6jAxoZ5tr9spzG3cAer2MK1RMSrB+YVAaIr3qJuT\\\\\\\/TL22JkUkpOikVC7XDooU9DmTFTpkb1SMi09rRgU6oejfI4p6zfkKmq3Mbs\\\\\\\/jNTdZUtcWGHMc\\\\\\\/4QFqsioaC6w2DYzW45pbut7klHr98efRNFZgmPTKDRoMdh7Wvo2GpFCAaLp3pLq36jFSuCTJoQns0dYNCMQJHp8rDhqur\\\\\\\/VotnTcBUdBfVXtUkDmoTiZBuUeV+dzFFzZzMnxzr2mXJtAKaMR\\\\\\\/ynzS7Oz\\\\\\\/IKTlkfZnvygRtV8K\\\\\\\/6a62eejPGbGN83v5WYd296QIutb4mdC2u4UtCpRrZMSS0jrDm8Rwza1CEJhJbUS5+VEeq38gffJ1SufQ5wTLtWja4QrrQGqwX70hu3LA1OeH+lOe6sG2Fm\\\\\\\/59T9YyDC4Jp5wgMQcv6ITqZB7972e2NjgTwD6ZULbH9Ux2B2kS\\\\\\\/qsTwo1aksmnIRjzZ+K2qTUTJM+LAzjo4LzC7hI00qUC5xrhjnpdpZZk1AE+PheWs1UMXZpwkWuR2kChJiGEmqe7EXISoHPHUXt8yFdN74t0MvJZDXkp4Q5SJAAqfLuD1SqMPHLllzH\\\\\\\/qvGx8OqC7Z0y9JK+gRRvOx8qgOTUotozwqyWq7a4ne30lnn9wvlEop8TPo9E437\\\\\\\/EE0zLlJoAQHpmab6Xu38bTji\\\\\\\/UY6MMkHaBDjc8fZ0AyTqiaSj6u1LYILCkVdFDslJXUQBAN1C77wJ6U6FTVL5KI3TqmLvPYtUwTvmu7R9yBBvQWjb8iKEx+2OiXthB9exIE+ZdpyA8tCBoxjiikmWA\\\\\\\/JJiLw5AlcsFuH0Q9A+NmxMVZ5kzkC78K\\\\\\\/y\\\\\\\/Y\\\\\\\/gMq5NWWyyYzWVDGWpqjTJdoulY6VCTqmhG4FR8FPb4nAgLlUp1yXbdsnJjY9gHX65w+nKpw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TdP95nRv3DwYv\\\\\\\/6zUmh7vg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002261d2cbc686b4b2d2ded53f06b7062f6390d50907aaeb749678bbcb457d99351d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nkAHE6uWNy\\\\\\\/NXyF5LFzZaYdo8FqBdusX47lcEGy+XiMPQM8QkjqKnXwNwe4B21d6jNLzEWbC9GXoEVW3Rd7zfwgVS\\\\\\\/OcX\\\\\\\/1PPSwD5CnOMwt6nOzPDaZ1s0u8M1RMWG4l1n\\\\\\\/eB03SmhlQo5P4ejDXkd7btGkghrj9jF9qPa2X39FyC6DbgEv9dWtpNrvYeSnR02ER2PB0QcVOwrPo8oqgoNsYu8kE6ZZFo3nHcLzsxBktGD8Zw4Ymy\\\\\\\/sQveYEeze8k1pmD\\\\\\\/rkpCuM2hhM8dis\\\\\\\/QczM4coNqRbsRgQMvEqWk0SEbnjA+Jp8CuXsoqbOW8AUOs9yC3WE\\\\\\\/GQWWasdZNFhZO7uRdN\\\\\\\/zJrhqr\\\\\\\/\\\\\\\/e8Q8FAPyEVnreauIIwPNgf3X+lMOHem\\\\\\\/34+UBFb2dvis1SBigQQZjkcTSVnt7LjE5luof5c\\\\\\\/CwGewbdSV\\\\\\\/abOnfS5ECcLYcjYWWicEMr7sR8pWVZF3TyrxlM2SqdpuOvk0h4FRuFwufkWJz0V6ripWF\\\\\\\/w22tb5exG8oXfwlZiQApHMqI\\\\\\\/P5XXg5t1JZPHCiq8zPk+PaZrZIT5NgSVB9qKuXHYe15d8eTU+4AGnXA0Yu6ciOJ1IR+XZ\\\\\\\/wQM6ZPEvK4I7t79PBye0wa2NRxxeHw0YUeArJ9cK1msC0Y6TOZzS5Vl9i0XRjwdsI1z6xwNTeSgamMdBYX345+aeNQRHioSfO5M\\\\\\\/ULoygrF3KAyl1SvlMo3BGmbPIXkcYHxi1AWRSh1+pq9Ky9mvugNEnNstP24tfvMWkK2NVUDfdobjxKojv3ZGZatWAyfNcPvfBuxcavJNmvdXNGgrLlkZdj1e\\\\\\\/93c+RqOZEDcgYZLmLra\\\\\\\/rw27ayXGlpzMoyftlYikd0JxEkVQIRQylxFhC0tjAdsoDkVmEd+JbCtTLeZP0YIK0M5sBwzNC1HbZ38BEQ+92dSkDStMO9aUawLCnncj64I9TX27FonwWBP3uXxvJOsp6NtJJBMwvKJ7oAdnmyxUUryVVOyK3NQDBvm9ONBzRKZFtHToc9ufHB1uUuTexIibQPcKMUvRdtQsMOMi7E8MqgIjvbf10jA2HOHn+Yd+CkQqgc8Qn\\\\\\\/9lwp7bwKUpD9rhcnwtKqFtU+yO9aj7IqRL4+\\\\\\\/mB9oCYVCDhkKfW8avvTF9m4fmAvhjQCTAywo0TsLcpdJiFiYKFiUph3Sf7J5w2XoKzgb1NTJl4n\\\\\\\/hmaWvrNY1n8jBJzEaHOmLTIQ0aDL3Ta0oDxXVkOXg5urw7f4+qRHs1URhI1SJNv0j5XV+pkpxDx+878bFycpVs9IJCUJy4uLHEmWJBR7ejcRQxgHp6l4yd4NaK8ZQ+iGFva\\\\\\\/wiRFi9Fs+xJ1jTOY3C\\\\\\\/YSunyYYE57ehegmSe7n4oFEvE54tsB\\\\\\\/HrTOgT\\\\\\\/NirnByIaWTGUm+nBkZ43JacbEet8TfHuxHMtMeRD\\\\\\\/FNXHgjKSecBvv\\\\\\\/B4ySYgRSbSCZE7vpORU6wGHdZw3ujjK8DF8LCaplUZwyUMnxyupBHG9zgfgNEFyltP9H7Ef8NrtMGYm8IkygaMSSaMWjlq8QOizxhKzmgbI5T6rJxAXBcEBqf9OF\\\\\\\/tXCyZ1AnJSyX09kC\\\\\\\/66UaAMBZY0jxvYfnj6d28xXzZQ4zp9gguomNR8mj2UTeVs3YG2YeUj9v2HFbFWW5NqUyqIKttvZ7AjtjSbmNMTwcA6WDv5kkEkxTB64B388MseiCbP7idA3ed7XmbDWTRYgZ5BtdLWayHMwev\\\\\\\/iENkTWlZOuSYz6DuvO1JZinXk8iDM++8Ex7olsbgH8Npq3Qecc3RLeXrYTYya9hLUHpP3xeaSWaBfSJWKwCYSnwz7AoCZZta8AdWICzg3IVrBSGHHW0NJA9Qrq5OkBokV8g\\\\\\\/20PlUuyznzFvJ7Y3hao4Ifu3d0eQbJXjL+SQRP14jZ+CmqTvje6mdz03XKVnq7CAyuBYAEg\\\\\\\/cnSnwPXdqgTlItfIdqOmW9D4Ie3eBUrjv5TNu74crdT5YLM9GTsm+r3G0AQVvFFVsdAQAA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EV\\\\\\\/YTZOvgHr4gnUVqEe22Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c24cfca33762c0466863f64f904ff4099177894d56559023bc2a8c6f1f48c046\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459347 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1405" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ytc9lmaHAG83j6V_NXdUw1Hwji-P2aFvJTA-cvyW0Zc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H4Sx1ixziWX6TK1HTU8zhxs9EUWRO6xKiakDGE6W568" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1397" title="Maintaining Sufficient Nanos Is a Critical Function for Polar Granule Component in the Specification of Primordial Germ Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1415" title="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459347" id="node459347" data-pisa="ggg;2/11/1405" data-pisa-master="ggg;2/11/1405" data-apath="/ggg/2/11/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Shrinkage Estimation of the Realized Relationship Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey B. Endelman</span> and <span class="highwire-citation-author" data-delta="1">Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459347" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.endelman@gmail.com" class="" data-icon-position="" data-hide-link-title="0">j.endelman@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/11/1405" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459347" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1405.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459347" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1405.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459347" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1405.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459347" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/11/1405.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459347" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The additive relationship matrix plays an important role in mixed model prediction of breeding values. For genotype matrix <strong>X</strong> (loci in columns), the product <strong>XX′</strong> is widely used as a realized relationship matrix, but the scaling of this matrix is ambiguous. Our first objective was to derive a proper scaling such that the mean diagonal element equals 1+<em>f</em>, where <em>f</em> is the inbreeding coefficient of the current population. The result is a formula involving the covariance matrix for sampling genomic loci, which must be estimated with markers. Our second objective was to investigate whether shrinkage estimation of this covariance matrix can improve the accuracy of breeding value (GEBV) predictions with low-density markers. Using an analytical formula for shrinkage intensity that is optimal with respect to mean-squared error, simulations revealed that shrinkage can significantly increase GEBV accuracy in unstructured populations, but only for phenotyped lines; there was no benefit for unphenotyped lines. The accuracy gain from shrinkage increased with heritability, but at high heritability (&gt; 0.6) this benefit was irrelevant because phenotypic accuracy was comparable. These trends were confirmed in a commercial pig population with progeny-test-estimated breeding values. For an anonymous trait where phenotypic accuracy was 0.58, shrinkage increased the average GEBV accuracy from 0.56 to 0.62 (SE &lt; 0.00) when using random sets of 384 markers from a 60K array. We conclude that when moderate-accuracy phenotypes and low-density markers are available for the candidates of genomic selection, shrinkage estimation of the relationship matrix can improve genetic gain.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/realized-relationship-matrix" class="hw-term hw-article-keyword hw-article-keyword-realized-relationship-matrix" rel="nofollow">realized relationship matrix</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/breeding-value-prediction" class="hw-term hw-article-keyword hw-article-keyword-breeding-value-prediction" rel="nofollow">breeding value prediction</a></li><li class="kwd"><a href="/keyword/shrinkage-estimation" class="hw-term hw-article-keyword hw-article-keyword-shrinkage-estimation" rel="nofollow">shrinkage estimation</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-4">Mixed models play an important role in the prediction of breeding values for plants and animals. Under the assumption that the breeding values are multivariate normal with genetic covariance <strong>G</strong>, best linear unbiased prediction (BLUP) can be used to calculate the breeding values from phenotypic data (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Henderson 1984</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernardo 2010</a>). In the absence of molecular markers, genetic covariance can be estimated via the approximation<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where each element of the numerator relationship matrix <strong>A</strong> is twice the coefficient of coancestry and depends on the probability of identity-by-descent (IBD) from a base population with additive genetic variance <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-2.gif"/></noscript></span></span> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kempthorne 1957</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lynch and Walsh 1998</a>). We regard <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> as an approximation because it depends on a number of population genetic assumptions that rarely hold in breeding populations, particularly in plant breeding (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Goddard 1986</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Lynch and Walsh 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Piepho <em>et al.</em> 2008</a>).</p><p id="p-5">When molecular markers are available, it is often assumed that the goal is to estimate the probability of IBD, but in fact, the goal is to estimate the genetic covariance, which depends on the genotypes of the causal loci and is fundamentally a state property. It follows that for a complex trait for which the infinitesimal model is a suitable approximation, <strong>G</strong> depends on the probability that the alleles at a random locus are identical in state, or IBS (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Powell <em>et al.</em> 2010</a>). Our first objective was to develop a theoretical framework for estimating the (realized) relationship matrix that is suitable for inbred lines and consistent with the IBS approach (<em>i.e.</em> without invoking a base population).</p><p id="p-6">As the number of markers increases, the probability of IBS at a random marker approaches the probability of IBS at random genomic loci. This limiting behavior is the basis for estimators of the form (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Habier <em>et al.</em> 2007</a>)<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-3.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <strong>X</strong> is the <em>n</em>×<em>m</em> genotype matrix for <em>m</em> markers and <em>n</em> lines, and the proportionality constant θ is fit by maximum likelihood (or REML). To see this connection more explicitly, note that when <em>m</em> bi-allelic markers are coded as {−1,0,1}, the <em>n</em>×<em>n</em> matrix of IBS coefficients (for the markers) is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-4.gif"/></noscript></span></span>, where <strong>J</strong> is a matrix of ones (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Piepho 2009</a>).</p><p id="p-7">Although <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> is sufficient for breeding value prediction, to define a realized relationship matrix, a convention is needed concerning the scaling of the matrix. By analogy with the numerator relationship matrix, we propose that the mean of the diagonal elements equals 1+<em>f</em>, where <em>f</em> is the inbreeding coefficient of the current population<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span></p><p id="p-8">(Beginning with <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>, the symbol <strong>A</strong> denotes the IBS relationship matrix, and the angular brackets denote the average with respect to an index, in this case, <em>i</em>.) <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a> requires a concept for inbreeding that is consistent with the IBS framework. Following <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Powell <em>et al.</em> (2010)</a>, we define the inbreeding coefficient for a single locus as the intra-individual gametic correlation, but our extension to the multi-locus case is different and emerges as an algebraic necessity during the derivation.</p><p id="p-9">The strategy embodied in <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a>, in which the IBS properties of the markers are used as a proxy for the IBS properties of any two genomic loci, requires the number of markers to be much larger than the number of lines (<em>m</em> &gt;&gt; <em>n</em>). However, to minimize genotyping costs in breeding programs, it is common to use low-density (<em>e.g.</em> 384) SNP arrays, in which case the number of lines may exceed the number of markers. To develop a suitable estimator for this situation, we express the realized relationship matrix in terms of the <em>n</em>×<em>n</em> variance-covariance matrix (<strong>Σ</strong>) for genomic loci (<em>i.e.</em> when sampling columns of the genotype matrix). <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation 2</a> is equivalent to estimating <strong>Σ</strong> with the sample covariance <strong>S</strong>, which in the large <em>m</em> limit is asymptotically optimal with respect to mean-squared error (MSE) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Casella and Berger 2002</a>).</p><p id="p-10">When the number of lines exceeds the number of markers, the MSE of the sample covariance matrix is no longer optimal because there are too many parameters to estimate (<em>n</em><sup>2</sup>/2) relative to the number of marker data points (<em>nm</em>). This type of phenomenon is well known in the statistics literature under the name Stein’s paradox (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Stein 1956</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Efron 1975</a>), and it was <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">James and Stein (1961)</a> who first proposed shrinkage to reduce the MSE. <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> (2010)</a> have proposed a shrinkage estimator for the realized relationship matrix, but it does not preserve <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>. We propose an alternative estimator that does not shrink the inbreeding coefficient, and we investigate its impact on the accuracy of breeding value prediction in rice, barley, maize, and pig populations.</p><div class="section other1" id="sec-1"><h2 class="">THEORY</h2><div id="sec-2" class="subsection"><h3>Derivation of A in terms of causal loci</h3><p id="p-11">Initially we work with causal loci, using an overscript tilde for variables. Eventually we will work with a marker matrix and use the same symbols without the tilde. Consider a trait with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-6.gif"/></noscript></span></span> causal bi-allelic loci, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-7.gif"/></noscript></span></span> is the allele content at locus <em>k</em> in line <em>i</em> (the assignment of alleles is arbitrary). For an additive trait, the genetic value of line <em>i</em> can be written as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where the random genetic effects <em>u<sub>k</sub></em> are taken to be physiological parameters of the causal loci, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-9.gif"/></noscript></span></span> is the centered genotype matrix. By centering the allele content of each locus (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-10.gif"/></noscript></span></span> are the allele frequencies), the genetic values are expressed relative to the population mean <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-11.gif"/></noscript></span></span>. Furthermore, as proved in <em>Appendix 1</em>, the additive genetic value in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a> equals the breeding value, that is, twice the mean progeny value when the current population (regardless of its structure) is randomly mated.</p><p id="p-12">As in the introduction, we denote the variance-covariance matrix for the breeding values by <strong>G</strong><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span></p><p id="p-13">The subscript <em>u</em> on the variance operator indicates that it is with respect to the random genetic effects and not the genotypes—the latter are simply given and not assumed to follow any distribution. For the last step in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a>, we have assumed the <em>u<sub>k</sub></em> are i.i.d. with constant variance <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-13.gif"/></noscript></span></span>, which is appropriate for a complex trait with many causal loci of comparable effect (<em>i.e.</em> well described by the infinitesimal model). The variance per locus is scaled by <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-14.gif"/></noscript></span></span> so that <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-15.gif"/></noscript></span></span> is an intensive property that does not depend on the number of causal loci.</p><p id="p-14">We are now in a position to decompose <strong>G</strong> as <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-16.gif"/></noscript></span></span>, where <strong>A</strong> is the (IBS) relationship matrix satisfying <a id="xref-disp-formula-3-4" class="xref-disp-formula" href="#disp-formula-3">Equation 3</a>. This convention for the scaling of <strong>A</strong> implies that<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where the trace operator, tr(•), sums the diagonal elements of a matrix. Genetic formulas for both the population inbreeding coefficient <em>f</em> and the variance parameter σ<sup>2</sup> emerge upon applying the trace operator to <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span></p><p id="p-15"><a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation 7</a> follows from the identity <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-21.gif"/></noscript></span></span>, which holds for any matrix <strong>Z</strong> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Searle 1971</a>). <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Equation 8</a> follows by writing the diploid genotype as the sum of its two gametes: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-22.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-24.gif"/></noscript></span></span> are binary variables. <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 9</a>, which follows from several algebraic manipulations, introduces the notation <em>q</em> = 1–<em>p</em> as well as the intra-individual gametic correlation <em>f<sub>k</sub></em> at a single locus (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Powell <em>et al.</em> 2010</a>)<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span></p><p id="p-16">In <em>Appendix 2</em>, we show that <em>f<sub>k</sub></em> is also the deviation from Hardy-Weinberg proportions and thus interpretable as the inbreeding coefficient for the population.</p><p id="p-17">Upon comparing <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation 6</a> with <a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">Equation 9</a>, we see that the coefficient of the <strong>A</strong> matrix is<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>and the inbreeding coefficient <em>f</em> is a weighted average across loci<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span></p><p id="p-18">Dividing <strong>G</strong> (<a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a>) by σ<sup>2</sup> (<a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation 11</a>) yields the following formula for the relationship matrix<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span></p><p id="p-19">As expected, the parameter <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-29.gif"/></noscript></span></span> cancels out and does not appear in the formula for <strong>A</strong>. The second step in <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">Equation 13</a> is an identity from matrix algebra, in which <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-30.gif"/></noscript></span></span> has been written as a sum over <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-31.gif"/></noscript></span></span> matrices with dimension <em>n</em>×<em>n</em>, formed from the outer product of the columns of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-32.gif"/></noscript></span></span> (denoted by <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-33.gif"/></noscript></span></span>). In the limit of the infinitesimal model, the averages in <a id="xref-disp-formula-13-2" class="xref-disp-formula" href="#disp-formula-13">Equation 13</a> converge to the corresponding expected values under random sampling of genomic loci. Letting the random variables <strong>w</strong> and <em>p</em> denote the centered genotype and allele frequency, respectively, the result is<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span></p></div><div id="sec-3" class="subsection"><h3>Estimating A from markers</h3><p id="p-20">The <em>n</em>×<em>m</em>-centered genotype matrix for the markers <strong>W</strong> (without a tilde) represents <em>m</em> realizations of the random variable <strong>w</strong> and can be used to estimate the parameters in <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation 14</a>, the most important being the genome-wide covariance matrix <strong>Σ =</strong> var[<strong>w</strong>]. If the markers are an unbiased sample of genomic loci, then the sample covariance matrix <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-35.gif"/></noscript></span></span> is an unbiased estimator of the genomic parameter <strong>Σ</strong>. Moreover, when the number of markers is large compared to the number of lines (<em>m</em> ≫ <em>n</em>), <strong>S</strong> is optimal with respect to mean-squared error (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Casella and Berger 2002</a>). This leads to the following estimator for <strong>A</strong><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span></p><p id="p-21">As the number of markers decreases, the sample covariance is no longer an optimal estimator for the genome-wide covariance matrix. By shrinking the estimate, although this introduces bias, the estimation error can be decreased. One type of shrinkage estimator is the weighted average<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>where the shrinkage intensity <em>δ</em> ranges from 0 to 1. When <em>δ</em> = 0, there is no shrinkage and the estimator equals the sample covariance <strong>S</strong>. When <em>δ</em> = 1, the estimate is completely shrunk to a target <strong>T</strong>. The target represents a low-dimensional model that can be estimated with greater precision than <strong>Σ</strong> because it has fewer parameters (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Schäfer and Strimmer 2005</a>).</p><p id="p-22">A common target is <strong>T</strong> = <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-38.gif"/></noscript></span></span><strong>I</strong> where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-39.gif"/></noscript></span></span> is the mean of the diagonal elements of <strong>S</strong> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ledoit and Wolf 2004</a>). Substituting this formula into <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">Equation 16</a>, one can verify that the total variance is estimated without bias: <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-40.gif"/></noscript></span></span>. Shrinkage is not needed for estimating the total variance because it is a single parameter. Similarly, even with as few as <em>m</em> = 96 markers, the row means of the marker matrix <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-41.gif"/></noscript></span></span> will be a near-optimal estimator for the genome-wide parameter E[<strong>w</strong>] because there are only <em>n</em> parameters to estimate from <em>nm</em> data points. Our shrinkage estimator (denoted with an asterisk) is thus<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span></p><p id="p-23">To select the shrinkage intensity, we make use of results from <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Ledoit and Wolf (2004)</a>, who derived an analytical formula for the shrinkage intensity that minimizes the expected MSE for the covariance matrix<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span></p><p id="p-24">(The squared Frobenius norm ‖⋅‖<sup>2</sup> is the sum of the squared elements of a matrix.) In <em>Methods</em>, we give the asymptotic solution to <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">Equation 18</a>. From this solution, a useful heuristic can be derived for when shrinkage is expected to be negligible (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Ledoit and Wolf 2004</a>)<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-44.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>where CV is the coefficient of variation of the eigenvalues of <strong>S</strong>. <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">Equation 19</a> indicates that when the ratio of lines to markers is small compared with the dispersion of eigenvalues (<em>n</em>/<em>m</em> ≪ CV<sup>2</sup>), there is no need for shrinkage. This formula is revisited in <em>Results</em>.</p></div></div><div class="section methods" id="sec-4"><h2 class="">METHODS</h2><div id="sec-5" class="subsection"><h3>Data sets</h3><p id="p-25">Genotypes for several publicly available populations were used in this study:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-26"><span class="list-label">(1) </span>Maize diversity panel (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cook <em>et al.</em> 2012</a>) (available at <a href="http://www.panzea.org/dynamic/derivative_data/Cook_etal_2012_SNP50K_maize282_AGPv1-111202.zip">http://www.panzea.org/dynamic/derivative_data/Cook_etal_2012_SNP50K_maize282_AGPv1-111202.zip</a>)</p></li><li id="list-item-2"><p id="p-27"><span class="list-label">(2) </span>Rice diversity panel (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Zhao <em>et al.</em> 2011</a>) (available at <a href="ftp://ftp.gramene.org/pub/gramene/CURRENT_RELEASE/data/diversity/data_download/hapmap_plink_files/div_rice34.RiceDiversity44K.hapmap.tar.gz">ftp://ftp.gramene.org/pub/gramene/CURRENT_RELEASE/data/diversity/data_download/hapmap_plink_files/div_rice34.RiceDiversity44K.hapmap.tar.gz</a>)</p></li><li id="list-item-3"><p id="p-28"><span class="list-label">(3) </span>Commercial pig population (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cleveland <em>et al.</em> 2012</a>) (available at <a href="http://www.g3journal.org/content/suppl/2012/04/06/2.4.429.DC1/FileS1.zip">http://www.g3journal.org/content/suppl/2012/04/06/2.4.429.DC1/FileS1.zip</a>)</p></li><li id="list-item-4"><p id="p-29"><span class="list-label">(4) </span>Advanced breeding lines from the North Dakota State University 2006–2009 two-row and six-row barley breeding programs (available by querying the database at <a href="http://hordeumtoolbox.org">http://hordeumtoolbox.org</a>)</p></li></ul><p id="p-30">For the pig population, we also used phenotypes and progeny-test-estimated breeding values (pEBV) for three anonymous traits, downloaded from the same source. Genotypes were curated by eliminating markers with more than 10% missing data and lines with more than 15% missing data. The number of lines and markers after curation are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Missing marker scores were imputed with the population mean for each marker.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462051/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462051/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Populations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Shrinkage intensity</h3><p id="p-35">Let the <em>n</em>×<em>m</em> matrix <strong>Z</strong> constitute <em>m</em> independent observations of an <em>n</em>-variate random variable <strong>z</strong> with mean <strong>0</strong>, for which the sample covariance matrix is <strong>S</strong> = <em>m</em><sup>−1</sup><strong>ZZ</strong>′. <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Ledoit and Wolf (2004)</a> proved that the following shrinkage intensity produces an estimator that is asymptotically optimal with respect to MSE (<a id="xref-disp-formula-18-2" class="xref-disp-formula" href="#disp-formula-18">Equation 18</a>)<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span></p><p id="p-36">For convenience, we rewrite the numerator in <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">Equation 20</a> as<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-47.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-48.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span><span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span></p><p id="p-37"><a id="xref-disp-formula-22-1" class="xref-disp-formula" href="#disp-formula-22">Equations 22</a> and <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">23</a> follow from cyclic permutation properties of the trace and the definition of <strong>S</strong>. <a id="xref-disp-formula-24-1" class="xref-disp-formula" href="#disp-formula-24">Equation 24</a> introduces the matrix<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-50.gif"/></noscript></span><span class="disp-formula-label">(25)</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-51.gif"/></noscript></span></span> is the matrix formed by squaring the elements of <strong>Z</strong>. In practice, the shrinkage intensity is confined to the interval [0,1] when <a id="xref-disp-formula-20-2" class="xref-disp-formula" href="#disp-formula-20">Equation 20</a> yields a value outside this range (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Ledoit and Wolf 2004</a>). When applying these formulas to the genomic data, for <strong>Z</strong> we used the centered genotype matrix <strong>W</strong> adjusted to have zero row means: <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-52.gif"/></noscript></span></span>.</p><p id="p-38">This shrinkage algorithm has been implemented as part of the rrBLUP package for R, version 4.0 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Endelman 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">R Development Core Team 2011</a>).</p></div><div id="sec-7" class="subsection"><h3>Simulation and analysis</h3><p id="p-39">Simulated traits were constructed by first generating additive genetic values from the multivariate normal distribution, with variance equal to the full-marker relationship matrix (hence, σ<sup>2</sup> = 1). Independent normal deviates with variance <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-53.gif"/></noscript></span></span> were added to generate phenotypes, and the <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-54.gif"/></noscript></span></span> parameter was modulated to simulate traits with different phenotypic accuracies. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> was generated with <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-55.gif"/></noscript></span></span> = 3 for the plant species and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-56.gif"/></noscript></span></span> = 2 for the pigs. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> is based on 10,000 simulations, with log<sub>2</sub><span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-57.gif"/></noscript></span></span> chosen from a uniform(−1,7) distribution, and results were binned by realized phenotypic accuracy in 0.1 increments.</p><p id="p-40">Mixed model prediction of breeding values was conducted with the model<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(26)</span></span>where <strong>y</strong> is the vector of phenotypes, <em>μ</em> is a fixed effect, <strong>1</strong> is a column vector of ones, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-59.gif"/></noscript></span></span> is the vector of breeding values with estimated relationship matrix <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-60.gif"/></noscript></span></span>, and the residuals are <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-61.gif"/></noscript></span></span>. Computations were done with R package rrBLUP (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Endelman 2011</a>), which estimates variance components by REML using the eigenvalue decomposition algorithm of <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kang <em>et al.</em> (2008)</a>. With this algorithm, the inverse phenotypic covariance matrix <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-62.gif"/></noscript></span></span> is readily generated, after which the BLUE and BLUP solutions for the fixed and random effects, respectively, can be calculated using standard formulas (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Searle <em>et al.</em> 1992</a>)<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-63.gif"/></noscript></span></span></p><p id="p-41">Accuracy was defined as the Pearson correlation coefficient between the genomic estimated breeding values (GEBV = <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-64.gif"/></noscript></span></span>) and either the true breeding values (in the simulation) or the progeny-test-estimated breeding values (pEBV) for the pig traits.</p></div></div><div class="section results" id="sec-8"><h2 class="">RESULTS</h2><p id="p-42"><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> lists several attributes of the five populations used in this study. The population sizes ranged from <em>n</em> = 274 (maize) to <em>n</em> = 3534 (pig). The pig, maize, and rice populations had 30–50K SNPs, whereas only 2K SNPs were available for the barley populations. The inbreeding coefficient (<em>f</em>) for each of the four plant populations, calculated from the mean diagonal element of the relationship matrix, was near 1 as expected for inbred lines (imputing missing markers with the population mean introduced low levels of heterozygosity). The pig population was outbred with <em>f</em> = 0.03.</p><p id="p-43">Both structured and unstructured populations were included. The rice population was a diverse panel of several distinct types (<em>indica</em>, <em>japonica</em>, <em>Aus</em>, <em>etc</em>.) identifiable with principal component (PC) analysis (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Zhao <em>et al.</em> 2011</a>). The observation that 34% of the total variation was captured by the first PC indicates its highly structured nature (1<sup>st</sup> PC in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). We intentionally grouped the 2-row and 6-row barley lines, which as separate populations are unstructured (1<sup>st</sup> PC &lt; 10%) and derived from different breeding programs, into one population to create a second structured population for analysis (32% explained by 1<sup>st</sup> PC). The pig and maize populations were relatively unstructured (1<sup>st</sup> PC &lt; 10%).</p><p id="p-44">Population structure can also be detected from a histogram of the realized relationship coefficients. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> contrasts the unstructured 2-row barley population with the structured 2+6-row population. Because the relationship coefficients are expressed relative to the current population, the mean of the off-diagonal elements (left panel) is −(1 + <em>f</em>)/<em>n</em>, which is essentially 0 for populations with hundreds of lines or more. Despite having the same mean, the histogram for the 2-row population is unimodal, whereas that for the 2+6-row population is bimodal. The positive peak in the bimodal distribution arises from relationships between lines with the same row number, while the negative peak corresponds to relationships between lines with different row numbers. The highly structured rice population also has a diffuse distribution of off-diagonal elements, whereas the pig and maize distributions are unimodal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1/004259SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histograms of entries in the realized relationship matrix for the 2-row and 2+6-row barley populations. The diagonal elements have a mean of 1 + f &#x2248; 2 for inbred lines, while the off-diagonal elements have a mean of &#x2212;(1 + f)/n &#x2248; 0. The bimodal distribution of the off-diagonal elements reveals the highly structured nature of the 2+6-row barley population. The positive peak contains relationships between lines with the same row number, while the negative peak is between lines with different row numbers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423507909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histograms of entries in the realized relationship matrix for the 2-row and 2+6-row barley populations. The diagonal elements have a mean of 1 + &lt;em&gt;f&lt;/em&gt; &#x2248; 2 for inbred lines, while the off-diagonal elements have a mean of &#x2212;(1 + &lt;em&gt;f&lt;/em&gt;)/&lt;em&gt;n&lt;/em&gt; &#x2248; 0. The bimodal distribution of the off-diagonal elements reveals the highly structured nature of the 2+6-row barley population. The positive peak contains relationships between lines with the same row number, while the negative peak is between lines with different row numbers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1405/F1.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1405/F1.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-45" class="first-child">Histograms of entries in the realized relationship matrix for the 2-row and 2+6-row barley populations. The diagonal elements have a mean of 1 + <em>f</em> ≈ 2 for inbred lines, while the off-diagonal elements have a mean of −(1 + <em>f</em>)/<em>n</em> ≈ 0. The bimodal distribution of the off-diagonal elements reveals the highly structured nature of the 2+6-row barley population. The positive peak contains relationships between lines with the same row number, while the negative peak is between lines with different row numbers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">The right panel in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows the distribution of diagonal elements in the realized relationship matrices for the 2-row and 2+6-row barley populations. Although the mean of the diagonal elements is 1+<em>f</em> and thus at most 2, the individual coefficients can be larger than 2, unlike the diagonal elements of the numerator relationship matrix. The interpretation of the diagonal coefficients in terms of inbreeding is discussed below.</p><div id="sec-9" class="subsection"><h3>Shrinkage to minimize MSE</h3><p id="p-47">For each of the five populations, relationship matrices were estimated from random subsets of markers, with the shrinkage intensity chosen to minimize the expected MSE. As shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, for every population, the shrinkage intensity approached zero as marker number increased, but there were clear differences in the amount of shrinkage at low marker density. With 384 markers, the two structured populations (rice and 2+6-barley) had less than 3% shrinkage compared with nearly 20% shrinkage for the 2-row barley and over 30% shrinkage for the maize and pig populations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Shrinkage intensity to minimize the expected MSE. Each point is the mean from 20 random subsets of markers (SE &lt; 0.01). As expected, the optimal shrinkage decreased as the number of markers increased. There was little shrinkage for the structured populations (rice, 2+6-row barley) because of their high eigenvalue dispersion (see CV in Table 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423507909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Shrinkage intensity to minimize the expected MSE. Each point is the mean from 20 random subsets of markers (SE &lt; 0.01). As expected, the optimal shrinkage decreased as the number of markers increased. There was little shrinkage for the structured populations (rice, 2+6-row barley) because of their high eigenvalue dispersion (see CV in Table 1).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1405/F2.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1405/F2.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461941" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-48" class="first-child">Shrinkage intensity to minimize the expected MSE. Each point is the mean from 20 random subsets of markers (SE &lt; 0.01). As expected, the optimal shrinkage decreased as the number of markers increased. There was little shrinkage for the structured populations (rice, 2+6-row barley) because of their high eigenvalue dispersion (see CV in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">These trends can be understood in terms of the heuristic in <a id="xref-disp-formula-19-2" class="xref-disp-formula" href="#disp-formula-19">Equation 19</a>, in which (for a given marker density) the shrinkage intensity depends on the ratio <em>n</em>/CV<sup>2</sup> between population size (<em>n</em>) and the coefficient of variation (CV) for the eigenvalues of the <em>n×n</em> covariance matrix. Because the leading principal components in a structured population account for a large amount of the total variation, such populations have high eigenvalue CV. As shown in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>, the rice and 2+6-row barley populations had the highest CV values (7.2 and 9.0, respectively), while the maize population had the lowest at 1.3. The final column in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> shows the ratio <em>n</em>/CV<sup>2</sup> relative to the pig population (= 1). Although the pig population was nearly 13 times the size of the maize population, its CV was 3.6 times larger, leading to nearly identical <em>n</em>/CV<sup>2</sup> ratios and shrinkage intensities in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The two structured populations had the smallest <em>n</em>/CV<sup>2</sup> ratios and thus also the least shrinkage in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. The 2-row barley population was intermediate between these extremes.</p><p id="p-50">The shrinkage intensities in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> were based on minimizing the <em>expected</em> MSE, as determined from a reduced marker set. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> (using <em>m</em> = 384 markers) shows that this approach did in fact minimize the <em>actual</em> MSE between the full-marker relationship matrix and that based on the reduced marker set (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1/FigureS2.pdf">Figure S2</a> for 2+6-row barley). The solid lines show the MSE as a function of the shrinkage intensity (in 0.05 increments), and in every case, the minimum was attained near the value indicated in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>. For the rice and 2+6-row barley populations, the minimum MSE was attained at 5% shrinkage <em>vs.</em> 2–3% shrinkage based on the expected MSE. The correspondence was equally good for the unstructured populations: 2-row barley = 15% actual <em>vs.</em> 19% expected; pig = 35% actual <em>vs.</em> 34% expected; maize = 30% actual <em>vs.</em> 32% expected.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximizing accuracy vs. minimizing MSE. At shrinkage intensities ranging from 0 to 0.7, with 0.05 increments, the relationship matrix was calculated for random sets of 384 markers. In each replicate, the MSE was calculated relative to the full marker relationship matrix (MSE = n&#x2212;2&#x2016;A384 &#x2212; Afull&#x2016;2), and GEBV accuracy was estimated using simulated phenotypes. The two curves (dashed = accuracy, solid = MSE) show the mean from 40 simulations (SE less than 3% of the mean)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423507909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximizing accuracy &lt;em&gt;vs.&lt;/em&gt; minimizing MSE. At shrinkage intensities ranging from 0 to 0.7, with 0.05 increments, the relationship matrix was calculated for random sets of 384 markers. In each replicate, the MSE was calculated relative to the full marker relationship matrix (MSE = &lt;em&gt;n&lt;/em&gt;&lt;sup&gt;&#x2212;2&lt;/sup&gt;&#x2016;&lt;strong&gt;A&lt;/strong&gt;&lt;sub&gt;384&lt;/sub&gt; &#x2212; &lt;strong&gt;A&lt;/strong&gt;&lt;sub&gt;full&lt;/sub&gt;&#x2016;&lt;sup&gt;2&lt;/sup&gt;), and GEBV accuracy was estimated using simulated phenotypes. The two curves (dashed = accuracy, solid = MSE) show the mean from 40 simulations (SE less than 3% of the mean).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1405/F3.medium.gif" width="415" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1405/F3.medium.gif" width="415" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-51" class="first-child">Maximizing accuracy <em>vs.</em> minimizing MSE. At shrinkage intensities ranging from 0 to 0.7, with 0.05 increments, the relationship matrix was calculated for random sets of 384 markers. In each replicate, the MSE was calculated relative to the full marker relationship matrix (MSE = <em>n</em><sup>−2</sup>‖<strong>A</strong><sub>384</sub> − <strong>A</strong><sub>full</sub>‖<sup>2</sup>), and GEBV accuracy was estimated using simulated phenotypes. The two curves (dashed = accuracy, solid = MSE) show the mean from 40 simulations (SE less than 3% of the mean).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Maximizing accuracy</h3><p id="p-52">Minimizing the MSE, although theoretically tractable, is not in itself particularly useful. A more meaningful criterion is maximizing the accuracy of breeding value prediction. The dashed curves in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> show the effect of shrinkage on prediction accuracy, as measured by the correlation between GEBV (using the shrunken relationship matrix and all phenotypes for training) and true breeding values simulated with the full marker matrix. The results indicate that shrinkage based on minimizing MSE is somewhat conservative with respect to maximizing accuracy. This follows from the observation that the maximum in the accuracy curve occurred at higher shrinkage than where MSE was minimized. For the maize, rice, and pig populations, the shrinkage intensity needed to minimize MSE was 0.20–0.25 less than for maximizing accuracy. This difference was somewhat smaller for the barley populations, but they only had 2K markers for estimating the full marker relationship matrix.</p><p id="p-53"><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> compares GEBV accuracy against phenotypic accuracy in the maize population for a range of simulated heritabilities. The three curves correspond to (1) using all 44K markers, (2) using a random set of 384 SNPs with shrinkage, and (3) using 384 SNPs without shrinkage. For all three methods, the maximum GEBV accuracy relative to phenotypic accuracy was observed at a phenotypic accuracy of 0.3 (SE &lt; 0.004). Comparing the two lower curves, one sees that shrinkage improved GEBV accuracy with 384 markers, and the accuracy gain increased with heritability. At a phenotypic accuracy of 0.9, shrinkage improved GEBV accuracy by 0.07 on average.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracy for simulated phenotypes in the maize population. The three curves show the difference between GEBV accuracy and phenotypic accuracy as a function of phenotypic accuracy (SE &lt; 0.004 not shown). GEBV accuracy was highest using all markers, followed by 384 SNPs with shrinkage. All three prediction methods peaked when phenotypic accuracy was 0.3, while the accuracy gain due to shrinkage increased monotonically with phenotypic accuracy. Phenotypic accuracies between 0.4 and 0.6 represented a &#x201C;sweet spot&#x201D; for shrinkage: in this range, heritability was high enough for shrinkage to substantially improve GEBV accuracy but not so high that phenotypes were more accurate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423507909" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction accuracy for simulated phenotypes in the maize population. The three curves show the difference between GEBV accuracy and phenotypic accuracy as a function of phenotypic accuracy (SE &lt; 0.004 not shown). GEBV accuracy was highest using all markers, followed by 384 SNPs with shrinkage. All three prediction methods peaked when phenotypic accuracy was 0.3, while the accuracy gain due to shrinkage increased monotonically with phenotypic accuracy. Phenotypic accuracies between 0.4 and 0.6 represented a &#x201C;sweet spot&#x201D; for shrinkage: in this range, heritability was high enough for shrinkage to substantially improve GEBV accuracy but not so high that phenotypes were more accurate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1405/F4.medium.gif" width="440" height="365"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1405/F4.medium.gif" width="440" height="365"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1405/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461924" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-54" class="first-child">Prediction accuracy for simulated phenotypes in the maize population. The three curves show the difference between GEBV accuracy and phenotypic accuracy as a function of phenotypic accuracy (SE &lt; 0.004 not shown). GEBV accuracy was highest using all markers, followed by 384 SNPs with shrinkage. All three prediction methods peaked when phenotypic accuracy was 0.3, while the accuracy gain due to shrinkage increased monotonically with phenotypic accuracy. Phenotypic accuracies between 0.4 and 0.6 represented a “sweet spot” for shrinkage: in this range, heritability was high enough for shrinkage to substantially improve GEBV accuracy but not so high that phenotypes were more accurate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55"><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> also illustrates that phenotypic accuracy can be superior to GEBV accuracy for highly heritable phenotypes. When phenotypic accuracy was above 0.6, it surpassed GEBV accuracy using random sets of 384 SNPs without shrinkage, and the crossover with shrinkage occurred at phenotypic accuracy equal to 0.8. This phenomenon arises because low-density markers sample the genome incompletely, leading to discrepancy between the true and estimated relationship matrices. If the sampling error is large enough, the accuracy of the phenotypes is corrupted rather than improved through the mixed model analysis. The “sweet spot” for shrinkage in this simulation was at phenotypic accuracies between 0.4 and 0.6. In this range, GEBV accuracy was substantially improved by shrinkage and was also higher than phenotypic accuracy.</p><p id="p-56">These trends were confirmed by our analysis of three anonymous traits in the pig population, for which progeny-test-estimated breeding values (pEBV) are available to calculate accuracy (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cleveland <em>et al.</em> 2012</a>). <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> compares the accuracy of phenotypes, high-density SNPs (53K), and low-density SNPs (random sets of 384), both with and without shrinkage. The top row for each trait shows the accuracy for individuals with measured phenotypes; the bottom row is for individuals without a measured phenotype. Looking at the last two columns, one sees a clear benefit to using shrinkage for predicting the breeding value of phenotyped individuals, and this benefit increased with heritability. For trait T3 (<em>h</em><sup>2</sup> = 0.38), shrinkage increased 384 SNP GEBV accuracy from 0.56 to 0.62, a gain of 0.06 (<em>P</em> &lt; 10<sup>−10</sup> by paired <em>t</em>-test). For traits T4 and T5 (<em>h</em><sup>2</sup> ≈ 0.6), the accuracy gain from shrinkage was 0.09 and 0.10, respectively, but phenotypic accuracy was still higher. With a phenotypic accuracy of 0.58, trait T3 appears to be in the sweet spot: GEBV accuracy was improved by shrinkage and was also higher than phenotypic accuracy.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462005/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462005/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Prediction accuracies for pig traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-62"><a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows that shrinkage did not improve GEBV accuracy for the unphenotyped pigs, nor have we observed any benefit in simulations. For example, even with as few as 96 markers, where the gains in GEBV accuracy were 0.1–0.2 in the maize population when training on all phenotypes, there was no accuracy gain when predicting unphenotyped lines.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">DISCUSSION</h2><p id="p-63">Although we have built upon the work of <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Powell <em>et al.</em> (2010)</a> and <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> (2010)</a>, our results are different from their unified additive relationship, or UAR, model. The UAR model assumed a genetic model with standardized coefficients for the causal loci<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(27)</span></span></p><p id="p-64"><a id="xref-disp-formula-28-1" class="xref-disp-formula" href="#disp-formula-28">Equation 27</a> has the undesirable property that the genetic values of lines possessing a rare causal allele tend to infinity as the allele frequency approaches zero. The marker-based estimate of the off-diagonal elements in the UAR matrix is<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-66.gif"/></noscript></span><span class="disp-formula-label">(28)</span></span>which also tends to infinity for lines possessing a rare marker allele as its frequency approaches zero. Such divergent behavior does not occur in the estimators we have derived. Our formula for high-density markers (<a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation 15</a>) is identical to the first formula proposed by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">VanRaden (2008)</a> for use with an unselected, outbred base population. Our IBS derivation provides rigorous justification for using this formula in any population when the number of markers is much larger than the number of lines.</p><p id="p-65">In the numerator relationship matrix, each diagonal element equals one plus the probability that the two alleles at a randomly chosen locus are IBD from the base population. As this probability lies between 0 and 1, the diagonal elements in the numerator relationship matrix range from 1 to 2. It was evident from <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> that the diagonal elements in the realized relationship matrix can fall outside this range. In the UAR model, the diagonal elements have been modified to lie in the range 0–2 (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> 2010</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Powell <em>et al.</em> 2010</a>), but this has the effect of creating an improper covariance matrix for the breeding values (<em>i.e.</em> it may no longer be positive semidefinite).</p><p id="p-66">From the formula for the realized relationship matrix in <a id="xref-disp-formula-13-3" class="xref-disp-formula" href="#disp-formula-13">Equation 13</a>, the analog to the inbreeding coefficient for an individual is<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-67.gif"/></noscript></span><span class="disp-formula-label">(29)</span></span></p><p id="p-67">To gain insight into this formula, note that if <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-68.gif"/></noscript></span></span> = 1/2 for all loci, <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">Equation 29</a> simplifies to 2<em>ψ</em> − 1, where <em>ψ</em> is the fraction of homozygous loci. In the context of an IBS model, homozygosity is an appropriate state quantity for measuring the inbreeding of an individual. The overall inbreeding coefficient <em>f</em> can be written as an average over individuals or over loci<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(30)</span></span>where the weights <em>β<sub>k</sub></em> are given in <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">Equation 12</a>.</p><p id="p-68">Because the allele content at each locus is centered by the population mean, our realized relationship matrix is positive semidefinite but not strictly positive definite (there is at least one zero eigenvalue). This means the breeding values follow a singular normal distribution, but this poses no problem from the perspective of mixed model theory (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Searle <em>et al.</em> 1992</a>).</p><div id="sec-12" class="subsection"><h3>Heritability</h3><p id="p-69">When the genetic covariance is written as proportional to the numerator relationship matrix, the proportionality constant is the additive genetic variance in the outbred base population. Because the IBS-relationship matrix uses the current population as the “base,” one might expect its proportionality constant, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-70.gif"/></noscript></span></span> (<a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">Equation 11</a>), to equal the genetic variance of the current population, but this is not true for inbred lines. As originally shown by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fisher (1941)</a> [see also <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Kempthorne (1957)</a> and <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Lynch and Walsh (1998)</a>], the additive genetic variance for a single locus with no dominance is <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-71.gif"/></noscript></span></span>. Compared with the coefficient of the relationship matrix, the additive genetic variance is larger by a factor of (1+<em>f</em>).</p><p id="p-70">This fact has implications for estimating heritability in the narrow sense. If the additive genetic values in the mixed model are breeding values (<em>i.e.</em> twice the mean progeny value; see <em>Appendix 1</em>), heritability can be defined using parent-offspring regression as<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-72.gif"/></noscript></span><span class="disp-formula-label">(31)</span></span></p><p id="p-71">Replacing the breeding values (and phenotypic mean <em>μ</em>) in <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">Equation 31</a> with their predicted values provides an immediate estimator for heritability. By taking the expected value of <a id="xref-disp-formula-32-2" class="xref-disp-formula" href="#disp-formula-32">Equation 31</a>, heritability can be related to the variance components of the mixed model (<a id="xref-disp-formula-26-1" class="xref-disp-formula" href="#disp-formula-26">Equation 26</a>). In <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1/FileS1.pdf">File S1</a>, we show that for large populations<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-73.gif"/></noscript></span><span class="disp-formula-label">(32)</span></span></p><p id="p-72"><a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">Equation 32</a> can also be used to estimate <em>h</em><sup>2</sup> by replacing the variance components with their ML or REML estimates.</p></div><div id="sec-13" class="subsection"><h3>Shrinkage</h3><p id="p-73"><a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Yang <em>et al.</em> (2010)</a> proposed using the identity matrix as a low-dimensional target when shrinking the estimate of the relationship matrix: <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-74.gif"/></noscript></span></span>. For inbred populations, this estimator is not ideal because it shrinks the off-diagonal and diagonal elements with the same intensity. By contrast, our estimator does not shrink the inbreeding coefficient.</p><p id="p-74">Using both real and simulated phenotypes, we have demonstrated that shrinkage can substantially increase the accuracy of GEBVs for phenotyped individuals (or lines), but not for unphenotyped ones. Although the term “genomic selection” is typically used in the context of predicting unphenotyped individuals, it is also encompasses the selection of phenotyped individuals for mating based on GEBV, which is important in plant and animal breeding. In plant breeding, we also see potential to use the realized relationship matrix with single-replicate or unbalanced multi-environment yield trials to more accurately advance lines for variety or hybrid development, and shrinkage may be beneficial in these applications.</p></div><div id="sec-14" class="subsection"><h3>Conclusion</h3><p id="p-75">There were two objectives in this study. The first was to formulate the realized relationship matrix based on identity-by-state at causal loci and by requiring the mean diagonal element to equal 1+<em>f</em> for the current population. For high-density markers, the optimal estimator of this relationship matrix is equivalent to the first formula of <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">VanRaden (2008)</a>. The second objective was to explore shrinkage estimation of the relationship matrix at low marker density. In unstructured populations with more lines than markers, shrinkage estimation can increase the accuracy of GEBVs for phenotyped lines; there is no benefit without phenotypes. Particularly when phenotypes have moderate accuracy, <em>e.g.</em> from preliminary yield trials in plant breeding, shrinkage estimation has the potential to improve the selection of lines as parents or for variety development.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-76">Support for this research was provided by the USDA-ARS and the Bill and Melinda Gates Foundation.</p></div><div class="section app" id="app-1"><h2>Appendix</h2><div class="section" id="sec-15"><h3>Appendix 1</h3><p id="p-77">We prove that, for an additive trait in an arbitrary population, the formula in <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a> for the genetic value of line <em>i</em><span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-75.gif"/></noscript></span></span>equals two times the mean genetic value of its progeny under random mating, expressed relative to the population mean. In other words, the additive genetic value is the breeding value. By assuming the genetic effects <em>u<sub>k</sub></em> are physiological parameters of the causal loci (rather than defined in a least-squares sense), we can write the progeny genetic value as <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-76.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-77.gif"/></noscript></span></span> is the allele dosage at locus <em>k</em> in progeny <em>j</em>. The mean progeny value for line <em>i</em> is thus<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-78.gif"/></noscript></span><span class="disp-formula-label">(33)</span></span>where the expectation is with respect to the random processes of gamete segregation and mating. The former contributes <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-79.gif"/></noscript></span></span> and the latter contributes <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-80.gif"/></noscript></span></span> for a mean progeny value of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-81.gif"/></noscript></span></span>. Subtracting the population mean <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-82.gif"/></noscript></span></span> and multiplying by two produces <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a>. Note that <a id="xref-disp-formula-35-1" class="xref-disp-formula" href="#disp-formula-35">Equation 33</a> does not require the causal loci to be in linkage equilibrium (the linearity of the expectation operator does not require statistical independence).</p></div><div class="section" id="sec-16"><h3>Appendix 2</h3><p id="p-78">We prove that for a single locus, the inbreeding coefficient defined by <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation 10</a> (written here without overscript tildes)<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-83.gif"/></noscript></span></span>also quantifies the deviation from Hardy-Weinberg proportions. For allele content <em>X<sub>i</sub></em> ∈ {0,1,2}, the proportion of heterozygotes (<em>H</em>) is<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(34)</span></span></p><p id="p-79">Upon substituting <em>X<sub>i</sub></em> = <em>x<sub>i</sub></em><sub>1</sub> + <em>x<sub>i</sub></em><sub>2</sub> into <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">Equation 34</a>, where <em>x<sub>i</sub></em><sub>1</sub> and <em>x<sub>i</sub></em><sub>2</sub> are binary variables denoting the two gametes in individual <em>i</em>, the result is<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-85.gif"/></noscript></span><span class="disp-formula-label">(35)</span></span></p><p id="p-80">Similarly, the proportion of <em>X<sub>i</sub></em> = 2 homozygotes is (omitting several steps)<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1405/embed/mml-math-86.gif"/></noscript></span><span class="disp-formula-label">(36)</span></span></p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004259/-/DC1</a>.</p></li><li class="fn" id="fn-11"><p id="p-81"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-06-22"><span class="received-label">Received </span>June 22, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-09-10"><span class="accepted-label">Accepted </span>September 10, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Endelman, Jannink</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for Quantitative Traits in Plants</span>, <span class="cit-comment">Ed. 2</span>. <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">Statistical Inference</span>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Duxbury</span>, <span class="cit-publ-loc">Pacific Grove, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A common dataset for genomic analysis of livestock populations</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D2%26rft.spage%253D429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.4" data-doi="10.1104/pp.111.185033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.185033%26rft_id%253Dinfo%253Apmid%252F22135431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi84MjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.5" data-doi="10.1016/0001-8708(75)90114-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Biased versus unbiased estimation</span>. <abbr class="cit-jnl-abbrev">Adv. Math.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BMath.%26rft.volume%253D16%26rft.spage%253D259%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0001-8708%252875%252990114-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0001-8708(75)90114-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.6" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.7" data-doi="10.1111/j.1469-1809.1941.tb02272.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1941</span> <span class="cit-article-title">Average excess and average effect of a gene substitution</span>. <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D11%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1941.tb02272.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1941.tb02272.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Discussion summary part VI: selection and non-random mating</span>, pp. <span class="cit-fpage">474</span>–<span class="cit-lpage">475</span> in <span class="cit-source">Advances in Statistical Methods for Genetic Improvement of Livestock</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hammond</span>  <span class="cit-name-given-names">K.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D2389%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18073436&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251949800035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>, Ed. 3. <span class="cit-publ-name">University of Guelph</span>, <span class="cit-publ-loc">Guelph</span>, <span class="cit-publ-loc">Ontario</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-confproc" id="cit-2.11.1405.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-article-title">Estimation with quadratic loss</span>, pp. <span class="cit-fpage">361</span>–<span class="cit-lpage">380</span> in <span class="cit-source">Proceedings of the Fourth Berkeley Symposium on Mathematical Statistics and Probability</span>, <span class="cit-vol">Vol. 1</span>. <span class="cit-publ-name">University of California Press,</span> <span class="cit-publ-loc">Berkeley</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BFourth%2BBerkeley%2BSymposium%2Bon%2BMathematical%2BStatistics%2Band%2BProbability%26rft.volume%253D1%26rft.spage%253D361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.12" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.080101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18385116&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254921600050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-source">An Introduction to Genetic Statistics</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.14" data-doi="10.1016/S0047-259X(03)00096-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ledoit</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A well-conditioned estimator for large-dimensional covariance matrices</span>. <abbr class="cit-jnl-abbrev">J. Multivariate Anal.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMultivariate%2BAnal.%26rft.volume%253D88%26rft.spage%253D365%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0047-259X%252803%252900096-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0047-259X(03)00096-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.16" data-doi="10.2135/cropsci2008.10.0595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ridge regression and extensions for genomewide selection in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPiepho%26rft.auinit1%253DH.%2BP.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D1165%26rft.epage%253D1176%26rft.atitle%253DRidge%2BRegression%2Band%2BExtensions%2Bfor%2BGenomewide%2BSelection%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.10.0595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.10.0595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000268109800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.17" data-doi="10.1007/s10681-007-9449-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Büchse</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">BLUP for phenotypic selection in plant breeding and variety testing</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D209%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9449-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9449-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254877800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D11%26rft.spage%253D800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.19"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2011</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schäfer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A shrinkage approach to large-scale covariance matrix estimation and implications for functional genomics</span>. <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D4%26rft.spage%253D32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-source">Linear Models</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-2.11.1405.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Variance Components</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-confproc" id="cit-2.11.1405.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">Inadmissibility of the usual estimator for the mean of a multivariate normal distribution</span>, pp. <span class="cit-fpage">197</span>–<span class="cit-lpage">206</span> in <span class="cit-source">Proceedings of the Third Berkeley Symposium on Mathematical Statistics and Probability</span>, <span class="cit-vol">Vol. 1</span>. <span class="cit-publ-name">University of California Press</span>, <span class="cit-publ-loc">Berkeley</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BThird%2BBerkeley%2BSymposium%2Bon%2BMathematical%2BStatistics%2Band%2BProbability%26rft.volume%253D1%26rft.spage%253D197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.24" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.25" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.11.1405.26" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em></span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/11/1405.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1397" title="Maintaining Sufficient Nanos Is a Critical Function for Polar Granule Component in the Specification of Primordial Germ Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1415" title="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/11"><span>Volume 2 Issue 11, November 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/11.cover-source.jpg?itok=czJv9iCa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wF-7kRz6LIg5bpmSk9QqUAABXCk2UDs9zAMk8AT80XY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/11/1405" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/11/1405" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/11/1405" class="active" data-icon-position="" data-hide-link-title="0">Shrinkage Estimation of the Realized Relationship Matrix</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459347" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fz6jQ3stsEV_sQEzB6Xt7BdsLvFztaoG48AI1rjxNBc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1405.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/11/1405" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qFmKWNhRqAFZzKyuYsSS1vfGCigp0NpadkLb-lc8Cvg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/11/1405" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1405.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459347" id="node459347--2" data-pisa="ggg;2/11/1405" data-pisa-master="ggg;2/11/1405" data-apath="/ggg/2/11/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Shrinkage Estimation of the Realized Relationship Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey B. Endelman</span> and <span class="highwire-citation-author" data-delta="1">Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459347--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.endelman@gmail.com" class="" data-icon-position="" data-hide-link-title="0">j.endelman@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/11/1405" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/11/1405&amp;text=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/11/1405&amp;t=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/11/1405&amp;title=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459347/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459347" id="node459347--3" data-pisa="ggg;2/11/1405" data-pisa-master="ggg;2/11/1405" data-apath="/ggg/2/11/1405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Shrinkage Estimation of the Realized Relationship Matrix</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jeffrey B. Endelman</span> and <span class="highwire-citation-author" data-delta="1">Jean-Luc Jannink</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1405-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004259 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459347--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.endelman@gmail.com" class="" data-icon-position="" data-hide-link-title="0">j.endelman@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Robert W. Holley Center for Agriculture and Health, USDA-ARS, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1405&amp;rft.epage=1413&amp;rft.atitle=Shrinkage%20Estimation%20of%20the%20Realized%20Relationship%20Matrix&amp;rft.volume=2&amp;rft.issue=11&amp;rft.date=2012-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Endelman%2C+Jeffrey+B.&amp;rft.au=Jannink%2C+Jean-Luc"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459347/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459347/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459347/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459347/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459347/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459347/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459347/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459347/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459347/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459347/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459347/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459347/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/11/1405" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">THEORY</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">METHODS</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/11/1405.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/11/1405.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/11/1405.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/11/1405.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
