<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/11/1427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/11/1427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/11/1427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/11/1427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/11/1427/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/11/1427.atom" />
<meta name="HW.pisa" content="ggg;2/11/1427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" />
<meta name="DC.Identifier" content="10.1534/g3.112.003699" />
<meta name="DC.Date" content="2012-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2012 Windhausen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic prediction is expected to considerably increase genetic gains by increasing selection intensity and accelerating the breeding cycle. In this study, marker effects estimated in 255 diverse maize ( Zea mays L. ) hybrids were used to predict grain yield, anthesis date, and anthesis-silking interval within the diversity panel and testcross progenies of 30 F2-derived lines from each of five populations. Although up to 25% of the genetic variance could be explained by cross validation within the diversity panel, the prediction of testcross performance of F2-derived lines using marker effects estimated in the diversity panel was on average zero. Hybrids in the diversity panel could be grouped into eight breeding populations differing in mean performance. When performance was predicted separately for each breeding population on the basis of marker effects estimated in the other populations, predictive ability was low ( i.e. , 0.12 for grain yield). These results suggest that prediction resulted mostly from differences in mean performance of the breeding populations and less from the relationship between the training and validation sets or linkage disequilibrium with causal variants underlying the predicted traits. Potential uses for genomic prediction in maize hybrid breeding are discussed emphasizing the need of (1) a clear definition of the breeding scenario in which genomic prediction should be applied ( i.e. , prediction among or within populations), (2) a detailed analysis of the population structure before performing cross validation, and (3) larger training sets with strong genetic relationship to the validation set." />
<meta name="DC.Contributor" content="Vanessa S. Windhausen" />
<meta name="DC.Contributor" content="Gary N. Atlin" />
<meta name="DC.Contributor" content="John M. Hickey" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Mark E. Sorrells" />
<meta name="DC.Contributor" content="Babu Raman" />
<meta name="DC.Contributor" content="Jill E. Cairns" />
<meta name="DC.Contributor" content="Amsal Tarekegne" />
<meta name="DC.Contributor" content="Kassa Semagn" />
<meta name="DC.Contributor" content="Yoseph Beyene" />
<meta name="DC.Contributor" content="Pichet Grudloyma" />
<meta name="DC.Contributor" content="Frank Technow" />
<meta name="DC.Contributor" content="Christian Riedelsheimer" />
<meta name="DC.Contributor" content="Albrecht E. Melchinger" />
<meta name="article:published_time" content="2012-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic prediction is expected to considerably increase genetic gains by increasing selection intensity and accelerating the breeding cycle. In this study, marker effects estimated in 255 diverse maize (&lt;i&gt;Zea mays L.&lt;/i&gt;) hybrids were used to predict grain yield, anthesis date, and anthesis-silking interval within the diversity panel and testcross progenies of 30 F&lt;sub&gt;2&lt;/sub&gt;-derived lines from each of five populations. Although up to 25% of the genetic variance could be explained by cross validation within the diversity panel, the prediction of testcross performance of F&lt;sub&gt;2&lt;/sub&gt;-derived lines using marker effects estimated in the diversity panel was on average zero. Hybrids in the diversity panel could be grouped into eight breeding populations differing in mean performance. When performance was predicted separately for each breeding population on the basis of marker effects estimated in the other populations, predictive ability was low (&lt;i&gt;i.e.&lt;/i&gt;, 0.12 for grain yield). These results suggest that prediction resulted mostly from differences in mean performance of the breeding populations and less from the relationship between the training and validation sets or linkage disequilibrium with causal variants underlying the predicted traits. Potential uses for genomic prediction in maize hybrid breeding are discussed emphasizing the need of (1) a clear definition of the breeding scenario in which genomic prediction should be applied (&lt;i&gt;i.e.&lt;/i&gt;, prediction among or within populations), (2) a detailed analysis of the population structure before performing cross validation, and (3) larger training sets with strong genetic relationship to the validation set.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/11/01" />
<meta name="citation_mjid" content="ggg;2/11/1427" />
<meta name="citation_id" content="2/11/1427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/11/1427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/11/1427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/11/1427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/11/1427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003699" />
<meta name="citation_pmid" content="23173094" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1427" />
<meta name="citation_lastpage" content="1436" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vanessa S. Windhausen" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Gary N. Atlin" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="John M. Hickey" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Mark E. Sorrells" />
<meta name="citation_author_institution" content="United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Babu Raman" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Jill E. Cairns" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Amsal Tarekegne" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Kassa Semagn" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Yoseph Beyene" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico" />
<meta name="citation_author" content="Pichet Grudloyma" />
<meta name="citation_author_institution" content="Nakhon Sawan Field Crops Research Center, Nakhon Sawan 60190, Thailand" />
<meta name="citation_author" content="Frank Technow" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Christian Riedelsheimer" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author" content="Albrecht E. Melchinger" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany" />
<meta name="citation_author_email" content="melchinger@uni-hohenheim.de" />
<meta name="citation_reference" content="citation_journal_title=TAG;citation_author=T. Albrecht;citation_author=V. Wimmer;citation_author=H.-J. Auinger;citation_author=M. Erbe;citation_author=C. Knaak;citation_title=Genome-based prediction of testcross values in maize;citation_pages=339-350;citation_volume=123;citation_year=2011;citation_pmid=21505832;citation_doi=10.1007/s00122-011-1587-7" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=J. L. Araus;citation_author=G. A. Slafer;citation_author=C. Royo;citation_author=M. D. Serret;citation_title=Breeding for yield potential and stress adaptation in cereals;citation_pages=377-412;citation_volume=27;citation_year=2008;citation_doi=10.1080/07352680802467736" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Bernardo;citation_title=Correlation between testcross performance of lines at early and late selfing generations;citation_pages=17-21;citation_volume=82;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. BurgueÃ±o;citation_author=G. D. L. Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype Ã environment interaction using pedigree and dense molecular markers;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_title=ASReml-R reference Manual.;citation_author=D. G. Butler;citation_author=B. R. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Charcosset;citation_author=L. Essioux;citation_title=The effect of population structure on the relationship between heterosis and heterozygosity at marker loci;citation_pages=336-343;citation_volume=89;citation_year=1994;citation_pmid=24177851" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=H. D. Daetwyler;citation_author=J. H. van der Werf;citation_title=Optimising multistage dairy cattle breeding schemes including genomic selection using decorrelated or optimum selection indices.;citation_pages=1-1;citation_volume=44;citation_year=2012;citation_pmid=22252172;citation_doi=10.1186/1297-9686-44-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. De Los Campos;citation_author=P. PÃ©rez;citation_author=D. Gianola;citation_author=J. BurgueÃ±o;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers;citation_pages=713-724;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. PÃ©rez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=P. Bijma;citation_author=J. A. Woolliams;citation_title=Inbreeding in genome-wide selection.;citation_pages=369-376;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076474" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. D. Daetwyler;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=Accuracy of predicting the genetic risk of disease using a genome-wide approach.;citation_pages=e3395-e3395;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852893;citation_doi=10.1371/journal.pone.0003395" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree;citation_pages=375-385;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Reliability of genomic predictions across multiple populations;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_title=Theoret. Appl. Genet.;citation_author=J. M. GonzÃ¡lez-Camacho;citation_author=G. de los Campos;citation_author=P. PÃ©rez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=D. Habier;citation_author=J. Tetens;citation_author=F. R. Seefried;citation_author=P. Lichtner;citation_author=G. Thaller;citation_title=Modeling relationships between calving traits: a comparison between standard and recursive mixed models.;citation_pages=1-1;citation_volume=42;citation_year=2010;citation_pmid=20100345;citation_doi=10.1186/1297-9686-42-1" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics in Maize Breeding.;citation_author=A. R. Hallauer;citation_author=M. J. Carena;citation_author=J. B. Miranda Filho;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=GSE is now an open access journal published by BioMed Central.;citation_pages=1-1;citation_volume=41;citation_year=2009;citation_pmid=19284684;citation_doi=10.1186/1297-9686-41-1" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient control of population structure in model organism association mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-GraniÃ©;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice;citation_pages=611-618;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley;citation_pages=1609;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. E. Melchinger;citation_author=R. K. Gumber;citation_author=R. B. Leipert;citation_author=M. Vuylsteke;citation_author=M. Kuiper;citation_title=Prediction of testcross means and variances among F3 progenies of F1 crosses from testcross means and genetic distances of their parents in maize;citation_pages=503-512;citation_volume=96;citation_year=1998;citation_pmid=24710890;citation_doi=10.1007/s001220050767" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=E. Porcher;citation_author=T. Giraud;citation_author=I. Goldringer;citation_author=C. Lavigne;citation_title=Experimental demonstration of a causal relationship between heterogeneity of selection and genetic differentiation in quantitative traits;citation_pages=1434-1445;citation_volume=58;citation_year=2004;citation_pmid=15341147" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=B. Pujol;citation_author=A. J. Wilson;citation_author=R. I. C. Ross;citation_author=J. R. Pannell;citation_title=Are Q(ST)-F(ST) comparisons for natural populations meaningful?;citation_pages=4782-4785;citation_volume=17;citation_year=2008;citation_issue=22;citation_pmid=19140971;citation_doi=10.1111/j.1365-294X.2008.03958.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. Saatchi;citation_author=M. C. McClure;citation_author=S. D. McKay;citation_author=M. M. Rolf;citation_author=J. Kim;citation_title=Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information.;citation_pages=1-1;citation_volume=43;citation_year=2011;citation_pmid=21208445;citation_doi=10.1186/1297-9686-43-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. SchÃ¶n;citation_author=H. F. Utz;citation_author=S. Groh;citation_author=B. Truberg;citation_author=S. Openshaw;citation_title=Quantitative Trait Locus Mapping Based on Resampling in a Vast Maize Testcross Experiment and Its Relevance to Quantitative Genetics for Complex Traits;citation_pages=485-498;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166171;citation_doi=10.1534/genetics.167.1.485" />
<meta name="citation_reference" content="citation_journal_title=Biologiske Skripter;citation_author=T. Sorensen;citation_title=A method of establishing groups of equal amplitude in plant sociology based on similarity of species content and its application to analyses of the vegetation on Danish commons;citation_pages=1-34;citation_volume=5;citation_year=1948" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Tuberosa;citation_author=S. Salvi;citation_author=S. Giuliani;citation_author=M. C. Sanguineti;citation_author=M. Bellotti;citation_title=Genome-wide approaches to investigate and improve maize response to drought;citation_pages=120-141;citation_volume=47;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=W. Wen;citation_author=J. L. Araus;citation_author=T. Shah;citation_author=J. Cairns;citation_author=G. Mahuku;citation_title=Molecular characterization of a diverse maize inbred line collection and its potential utilization for stress tolerance improvement;citation_pages=2569-2581;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.08.0465" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. C. Whitlock;citation_author=F. Guillaume;citation_title=Testing for spatially divergent selection: comparing QST to FST;citation_pages=1055-1063;citation_volume=183;citation_year=2009;citation_pmid=19687138;citation_doi=10.1534/genetics.108.099812" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thompson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. K. Wong;citation_author=R. Bernardo;citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.;citation_pages=815-824;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18219476;citation_doi=10.1007/s00122-008-0715-5" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. WÃ¼rschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=F. H. Longin;citation_author=T. WÃ¼rschum;citation_author=N. Ranc;citation_title=Impact of selective genotyping in the training population on accuracy and bias of genomic selection.;citation_pages=707-713;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22481121;citation_doi=10.1007/s00122-012-1862-2" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
<meta name="twitter:description" content="Genomic prediction is expected to considerably increase genetic gains by increasing selection intensity and accelerating the breeding cycle. In this study, marker effects estimated in 255 diverse maize ( Zea mays L. ) hybrids were used to predict grain yield, anthesis date, and anthesis-silking interval within the diversity panel and testcross progenies of 30 F2-derived lines from each of five populations. Although up to 25% of the genetic variance could be explained by cross validation within the diversity panel, the prediction of testcross performance of F2-derived lines using marker effects estimated in the diversity panel was on average zero. Hybrids in the diversity panel could be grouped into eight breeding populations differing in mean performance. When performance was predicted separately for each breeding population on the basis of marker effects estimated in the other populations, predictive ability was low ( i.e. , 0.12 for grain yield). These results suggest that prediction resulted mostly from differences in mean performance of the breeding populations and less from the relationship between the training and validation sets or linkage disequilibrium with causal variants underlying the predicted traits. Potential uses for genomic prediction in maize hybrid breeding are discussed emphasizing the need of (1) a clear definition of the breeding scenario in which genomic prediction should be applied ( i.e. , prediction among or within populations), (2) a detailed analysis of the population structure before performing cross validation, and (3) larger training sets with strong genetic relationship to the validation set." />
<meta name="og-title" property="og:title" content="Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/11/1427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic prediction is expected to considerably increase genetic gains by increasing selection intensity and accelerating the breeding cycle. In this study, marker effects estimated in 255 diverse maize ( Zea mays L. ) hybrids were used to predict grain yield, anthesis date, and anthesis-silking interval within the diversity panel and testcross progenies of 30 F2-derived lines from each of five populations. Although up to 25% of the genetic variance could be explained by cross validation within the diversity panel, the prediction of testcross performance of F2-derived lines using marker effects estimated in the diversity panel was on average zero. Hybrids in the diversity panel could be grouped into eight breeding populations differing in mean performance. When performance was predicted separately for each breeding population on the basis of marker effects estimated in the other populations, predictive ability was low ( i.e. , 0.12 for grain yield). These results suggest that prediction resulted mostly from differences in mean performance of the breeding populations and less from the relationship between the training and validation sets or linkage disequilibrium with causal variants underlying the predicted traits. Potential uses for genomic prediction in maize hybrid breeding are discussed emphasizing the need of (1) a clear definition of the breeding scenario in which genomic prediction should be applied ( i.e. , prediction among or within populations), (2) a detailed analysis of the population structure before performing cross validation, and (3) larger training sets with strong genetic relationship to the validation set." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11.cover-source.jpg" />
    <title>Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tiyjM1LGfjSymRmqriQkdf-dJykHgeku2eSJvfdGmds","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459349","apath":"\/ggg\/2\/11\/1427.atom","pisa":"ggg;2\/11\/1427","ac":{"\/ggg\/2\/11\/1427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/11\/1427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/11\/1427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/11\/1427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wHoUz9P42EWA17ejHzCgGl6FoGMKEUVXYuYj9CNL2GWaEyc+cz+Lw9bkrWcSfmiqPHj+rXtbWIUvRc9UQE24ZqAuvZWAkIYyYjUtTG9ofDaJa6jeOMagn7IUqjfZwVu9S0l2sdSMZcHnRaibGvTz3ZyVEbHujeRhI5JNV1D65SDYIKsfCAVqB4D6\\\\\\\/Xoka0F0+QLdS2VqRM\\\\\\\/40qL6Mzta61GpszzkBtYrYADZXq327BFF6yfRy1YVTTwHkDL4U7ZFcuqqSJIQ34t3f3f1zkdaRQmur96hYU2BgbJMSNpGEN0v7kN45UYWjvnV2t8mhyrpe\\\\\\\/vx6eYS8VeOIeqZZdYt+M8sD+rRU53OHzX9P3NPqVC+DLuQL2fYT1enFZ3slQR6Ro521xmV6bEiZoq1ONkFUcDRn+xxn0yxrimR2kw9QOu2QR0SlH1K6JU++RiGGiJT8fVH9Hx\\\\\\\/VXmIPf7DO\\\\\\\/cprbna5fvSvLsMBSFT1DlomkOamrdq2b35mXtleD4Yr7AXZttB9XUrRqs3jRlZRSVWdFYM5W2HOVVsYCbQJWYdGt8tfFMd0Xhnhg+Ple1URGSUz+Jy\\\\\\\/lSY2qX4DEOGTc4nLAMu0lq1nChzUx3\\\\\\\/ESzMTR2YSTB9sgH1opy72FlbGIvaxJrEiA0A5a9KaChThp5XlwHA19hMra5XmNCCWjCNEsIH4Cm0HyPgNVomMMXXGviQC5aOcqs6dZhaO\\\\\\\/oI+kDJl40JMQqviZYo9DJKzv7Zbiz\\\\\\\/hf6F5y0q559PG0bqWujcqyf4WoU+nEPhk0rHaeKWRGJE3Is7nGV4cetcMqMLpdPkOknQP6Xx29o8o9PQeU5TKqFpxnOI4k7a66XHNRWB0j+tEReWAD8KxOx4g7GGuMee8thnMTpJAgrLq5Uf5V7zjAhjiGYB1jK0iLruEgu4UkCr9wCilfd\\\\\\\/X5+BxN3QFxPAIzWC+VFYjutq0yq4qfUcgpzUHES6Tihtn0C1Nk6Reln3JS5\\\\\\\/b6YnG0nBv6rj50h4JS5GuzKMy+AtuxH5zZduNOlxMBlmVCmlobZglyzIA4WnxW0r7vEm\\\\\\\/a2esbldDiTaOZFC8CDvVIAMGshkGAKNPPwql8MOS3PrmSJxGoUVV1ZGUU8ojRNfAwtwTjbe41kWyc9C8xg\\\\\\\/JujZw7bo\\\\\\\/A2XvA6SRCFQ1ZY373nt9oqK2GClkPac+gE6i2FuMkj\\\\\\\/WOCe2z2M7Jb2zBG0E68fUfL8ON9YoNXILEjS5m8qR3idpQD5x+QJDcQZYdZinn\\\\\\\/+SCYfsWi+IC9DUPCXIkc8PKs0\\\u0022,\\\u0022iv\\\u0022:\\\u0022376H4syXEtjzgnbVYUzwnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227da356e0b8be073eae5994d796375063132f1e6e995d7a8f6c3d5cf136fd64b4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022C54TgmzHocmXyhtPT5V9ff8sXAzU91X\\\\\\\/WqhUidO8sfmZaIP2GR99NE7cM+F5MSOWMDYNSjRQ9GqIpuuggeamdwL8VFWZLch\\\\\\\/zmUdJraq+TmDsUbup\\\\\\\/fQT53ftk60I8Tn2GNR6eC797wTbQ8Url+T8Rp1AQrbZdcFSy+XkxfXr6IPzTD6oULo2Y85+YAx2hEcEoFYo7FgV84XidxqMYKUOyDBb9q3GWLV8d59yAcacdhhnixP0A\\\\\\\/AbaoUK3Dcmi4AVJM+RIUV4whgLqHnavzkweK3cLzSOtynHA9RDjENm6j1MxMsX0NodxZ7oxiGXLRArhtvpBIFfv7yR6M7ot4d9nNISB6l2qzsshykeVY6O\\\\\\\/qPCuufK6Y3fvfhoIwM2Mjyr0wpceCw0sHCK1IV2Am3rRnN0+48U3Bf+N+N9MxbfNjdQd0yWmV1v+HZUb\\\\\\\/l0xl67vLr4Y9sDVpqVZGqA4GPQRbZFwii2MtGnatBaxoNCVvQuxMuGVm9JbgOVWbiTO\\\\\\\/okRpFUcAHvTtnx2e2g2mgeeMfLKy9sz3pkZhl59vMukOm6Od5hJ\\\\\\\/JL+xBMEyIUyCNrvESkueYjERS2iXr2EOdOk0\\\\\\\/3UsQsnKhMm\\\\\\\/y2W0xdmY7tjS0QyApsOODLysq0196WMddzkP6UDSjtooCMByUe7CZBDlkDULzHpfbu825fWw8WemKK19HnznNYPd1ceEvSShbwcooQxzscDy\\\\\\\/eaEjqRvSXShoFOgByE4B\\\\\\\/ld\\\\\\\/b0GmAI0U4dwsEzkyp\\\\\\\/wR775ANWZV6bxh3OoeeBJoWnted5HgzIQZJMWSfmhfQ9wkhVq6nRHRbwfqf4gxbGnq7ItZqifCOP9m5glIB\\\\\\\/oTb8dUHGKwVPVTNC1UFi1Rv64QSBfK2ctFDxuKL\\\\\\\/LLbPh4pM\\\\\\\/IHe9hiAxhcu\\\\\\\/AQ0VRAavsWmXYMGVe\\\\\\\/M8lagKKO\\\\\\\/Osmx0MuyXha\\\\\\\/3KgIKcYfpkMmsKL0WD4tQFF\\\\\\\/Lnf6UuTeL8v6WHw3rZ9D\\\\\\\/Iin8kEYBxemq8uHrzrzBUowK9d+b\\\\\\\/0xhSwO\\\\\\\/8g4UUwFkbJaS1bsoiRIMGMLlKh4SlnSXP0AxDgTgWX4pOQ+75TjDbfLx7MClOZL\\\\\\\/NlW2hvcSt42bK7ixayTUTkMcFkioVv4jpZ2YpvIT\\\\\\\/0BePclVOt0+rf72dnMu11RZqe0BclevxW\\\\\\\/ZR0+zM\\\\\\\/Z5he\\\\\\\/Svb7YgH00PWfJ2L4K6G0\\\\\\\/G22X212uKfH+9uiUND7TWRL8UpOmFUSNFhJEWIUS47D6yGQy+ci\\\\\\\/UgsjXhClccsEw2Aiv+d9+\\\\\\\/1FVTchmMaoocWO+KxZlv5n4DhLcPqz01+I+iI1x3trpn2LX+J\\\\\\\/qbCZ242mwaC8w6w5qUWKetPDN1x6\\\\\\\/edIN\\\\\\\/6rEsmuYmPkCrh1NpntqyUT9YpnHUCcD3LDGmXP39L7fFky4ZsHFPGe3qtSKRjUJlh1X94e7gOwk0NvWWOynELrXqhkf8\\\\\\\/KMU6Sj9pL38vGRZI\\\\\\\/ZvFfVKi6O+bdC3lADqyEIg4HjK5YiAxdcBCsCpsp5s4VxG4hhFNkiYLycfP7p+pDRN\\\\\\\/IgPjs5xdUY1KPFFIOHBagGLH7diD7cRkbznC06LO3F2Jb72EzfZObTBM+DGdSiH7GLrUBamxTn9nYfykJ4kWF7YNNu5sXHg0HK7nYpfRY1PZDcV3oLdaSrlVIAeDBpE1S6pIk\\\\\\\/bMqf9MNipT\\\\\\\/I7T9AxjyieI12Pygb43C\\\\\\\/jyYVf5c\\\\\\\/XMptK\\\\\\\/KNso\\\\\\\/tE25nmt6cJSuItHYhj8aoafpiNNKz\\\\\\\/b5yac2GeF94STCJsy93DoSKdN1wP6cOOls7hjDp6FL5CyS7qiac7rTNti+SF58M4zTFvcEQN5efOmRHeOUpYlk2Ff8DaJNsnCjr2RTU7vLkzJa\\\\\\\/LSsZUZda6x9d5tUBBSMXFZPOYaOaS\\\\\\\/YJaxdUVFjIRiz23QFEz30YdMADbmUc+feLVRy3AVl8S4wNpNLnHm8KvjF8B6aPVZJKJC5F8CqnMbQ+21KSgLaaQuHDpiUh73FfL4Sz9iv+jPCyNXqWMZ0PJLHHOzfjb2JM4iAnqr7GBpQLDdeBcDQ35is1\\\\\\\/AH7pHZFE42ko0gI2jpfplB0PmNpk+YTEMOOx5Miduld6igKzyLFCYwmHURrpWCDNftxsuY1V0dIQaO3sGePobZzuaGdBB2aqsXkqVUP\\\\\\\/9rIXWT4ktPFE4QVBSs2eyjhXfOnRrEV6aj3m8yVOaXVHto0MNbAQ7RTqQHs2gnQYCOkG2DuyvAhG8u9EQTNNRXiKVFiYkbKWGPD6NBLMAx3FA8Pc8FgX2BxCOZ3BpXQracNC4OshvZmIvkm1MbjGBok7gxbyiHEnMGMxrSXCkaL4pzO6RB0yHAzgy8HBWPcQjmSo8qmUIhyBnCBcDCT\\\\\\\/WEGF9ulUwSbKBhufqlKrx0Hui9ALTGZCi4iREDxHD5esz1pP+QojFDUoh+ElsCl0YPxpkd0uRHqQvG5EBVTQ369Wk4Fsq+V2bvqU\\\\\\\/ipZp+sep0opAUZqQ2ZaInnKFTwMHBrFizx0iaaBxQgcv\\\\\\\/F\\\\\\\/OXh93LiRezJRs7FQRU3lsv\\\\\\\/nrYwpYun\\\\\\\/cURVqJfHqWdjaGwNgB3FqTMEpsLRO6AmsPJT\\\\\\\/SzxECY9Rpidk2zvEx8pAk2pq\\\\\\\/jwwq189OB0218Nzhqfja\\\\\\\/EH8Ts2Viy3ByxZ\\\\\\\/gjH9IDzMvuAlOv2WpR8wMLHWXZB8zZO2IRAlMluIkYrkA1DiX4BzOPKR57BGoYtV9UtSH3jDdN2\\\\\\\/4mWkdozRXA50iglrtw3ICDEEIu\\\\\\\/ijNpxhvcOZjv0byrJRw+vosX1wkh2BhjkjlFGoOmy\\\\\\\/BpyOi\\\\\\\/PljTg1y3Mg+WagZjRaXqSwzA\\\\\\\/Ijnr2mDOAwso5MFbMBjhaA2lQs+ttit9FRVL3dgZG7fmquVKwENu+IK5ZFwWR6k+8CHF4pLZiz0JWKdeUSkAFhnEIBuTOvex0QI4lOo\\\\\\\/o51WzsH2SsQbgofuB7W7Pkv3OBCp1424A9\\\\\\\/r8lcpDWm6OiSDHZlUi6B8btAJ+BbtZlVFTnc5Nj7lAFW2jxfqj8oKpTSwrmjCrLPfFGhg3qhG8G3y4vYCFPL6ODr+CcsLUTvVkSGjAdt3866oy0wOfeRf66niIvLnZ5v2jWbiTD7Z3aqrdjLzYXkhxKJQ3pe0fvkXNhBYppwkeqdRMA7UIjkr8fqOboo2\\\\\\\/MAzQwpO4xIwdZGFiYTxex4d6is9XQ2Is7Deba3Mp9Wbv\\\\\\\/k+n1fQu2OdCpk1AxD3LTuREDG+MsWvIyYkd5KDJTub3OShzSht4q30rn+YoB74ihBR8gz+UDRmh7lIpFuRRpeCdS2rFpM1Fm25UhFhpvj6yXp6y+Y+rt4UQuYhBf9lmXxgz5Vcke+qshAMpGLpa1UuJYN\\\\\\\/45H0G3F5ls6weBQKhroOwGxYADZMJqLcaEHQPRUev\\\\\\\/GFryPyUX1rEdzpDLWil9RL5l20Fk8KT+t1h5PVLRiXsziT9uh8FlcqcCjoa1DPeJ7esWURrXCO1h0jFE1EtmE84N00mw8z500qIhGKNg2WYYxVcrKPh4eys49sJwv41Oe3A5mLGv0qFPDqImj7Cf6313Y0rxAjzv9X0M9ENSisMMSDh7xBK9kThfDLbxkSAOSYka4NW\\\\\\\/D7Gwtfku8r8SdMOVKu\\\\\\\/F1G2OmSApzQAhCTTqxe8wEFINpRFAA3Z0kskTyNGQ9uH+SeukjgkVFiPcomtWnueV4fZ0YQmmLWBvOrA+61nVmsMvsELZSkR\\\\\\\/qr5RlaalEOYKvBAZddznYlZn8JirYkLibkKP2fpVklI4CUg\\\\\\\/+xjZ+X0f2eppJytyoRLicbES8gvwnNNZSx9ZSMTzGV4pqNhH73MchzUvi3XQ\\\\\\\/ODPb7k\\\\\\\/jpyXFSxdkMmqLQcX2HEUqLhAv7e5VhqoZ8hDyqjPIoIVzExy5OP2\\\\\\\/woQpUAvjkDiQTPWtGz27IYaapcn0an8ac46RSAVPx33SoRvAoasbjtKFW6AXJJvkuAl9OKd5lLVXlXiQv+fXZ2i4ZDiNb+Ehu0KRT0\\\\\\\/\\\\\\\/5cwS8hbXQgzGFhDN\\\\\\\/NNuEMKE8MV8H9nsABqb9dPgxghSRL0ef0\\\\\\\/QWzQgd+4s4OxsmUinEVU3lwC\\\\\\\/tbLrUmlEhnkG6jO+YV9+E78qwSp0V43qrAdMXgOX1ZIChONhWjFZlcdgYmq63E2GZYHBj+UOiPty7TBWZhmRYrZAJeLtqVRhcm3i1oBL42+wLqyrnXzbfyaNfUGE2FtuvqhNuBgSfeD3twbsA\\\\\\\/v\\\\\\\/A5cWv0TFuNCytGTUPYRDsNOlpZoAunhxmj2skTUwZmL1NT91+7ZUZhkrSeTDloxRyVefmZVYr3cE41KBBmd8UWBCla2uqiqUA6Pv6ovtn2KYQAkF2KEnn2qf8EqlI3ETCwNzXuLWpt1zgAebVBZDnlIoAk9+d+zkAbzE9lfeBactZhEULbeONDT9p4vI4L+31GcFm6iVFfBfjaG6Ck09jRp8VgbTLK\\\\\\\/qJhIlLccuxIEYXh4Ome3BFYt7AfiuQlvHPbLUJhJYxKFz\\\\\\\/JJJdJG4W12vonnkXLxSMZ627ALL+Z6fhID\\\\\\\/FyskvFaANFoq0EElsSuSMpI6JJt4wHlHzCED7nuccPSlz0\\\\\\\/58mJE+5GjhTMh\\\\\\\/jAA4epdHUabnpub0OSbXAPCYTvvTJAjBBt5LwAy6O\\\\\\\/XpiB5DNYwk2WoM6cdreZYv\\\\\\\/ArEihu7YYXCBP8Zrk\\\\\\\/kWrROKqQED\\\\\\\/wWZSMPPNkpy7VgoiwjgKVfbjvDo21aUrNecxGvGOm3\\\\\\\/a9P4XU\\\\\\\/+B4Ui1iFHg4Ivy\\\\\\\/nzf7b6\\\\\\\/KoIHkdZalIag29ruTDbdAUKe16+JH5NFAJQbgm2HrcVhRP794SSZTFjriDkbUrAvA7XJ+vSYXrhG4nilfuZ9PLK01KNamXnaTx88iKcw08iW3bJXSW660Heu52VrYwyKlWlPPmoNqcgRtgs\\\\\\\/ld0kcdJ85qERp3mjcFX2kSes8IdG6Gikn7C1VHdEVlHWY1gQPXSimbNFCjrCpM9Gh6bPYco7oD4ZGqEA2rZf4z0FmZYCSdN6wjEAxB28om6\\\\\\\/UihTgeNYsOSsba2janRGm2+U22nFzuOeMa9z9ULEeVKsrmRhoIZ5\\\\\\\/p6\\\\\\\/9z5L3JIktmY5Bx3Ec8Szn+WtYg2w\\\\\\\/zc\\\\\\\/fBr4M+yClFZCGAPxQyzW6GHlHLNWvNz6Lc2wrt\\\\\\\/tRedNiUGB3u0crjwWDRfq4Nc7TSmHh9rPQpLc4g9zq5jitf8aBT98s3WqQl00V+QPzFpFfnwjlQNMJ3twF3odUpjz5AOcGrs5VpvKAagh1w8axTlf0eIyrwOShI6f5g6GKpR7E4ZPgiB6\\\\\\\/ze1m0EPCsOeW+haCk100FlOlGw6I5m96avP7YaW6Rgi1BelfIVYMzO42fL2ipG5Rp\\\\\\\/cqyyY3FAbkZAFYzKEfQuHnUlb05zxEboYlvss2+ddD7X29pP3gET9aciW7paJ9xfVdMwqBGfq98Q2agWfAX+ooTcQzarAL33a7wu2jDa\\\\\\\/90DzvluQxyUhIXA5V+vIubdnjDUGglLgMkv89ksu3wDcrPqwE8dbHvAShgv2loRQdM2bIU5O5EU+8sZM5lPlJ6srf4qEh0glIgnz\\\\\\\/LRMKpejF\\\\\\\/qXwlVZVps00B2q60jv8UVxwa67FfdtXylTelYsk3d4+OhIPYyIlQdthlcNB0azKcz9FBFeQeYQCPz7EGpweFgQv6x97Kwhi0Wo2qwGHNGNBMFRkuP6yJ1g+TAAqes4uVIDDEag7LExNfgYQ7XqoiIa95cFfjUvcmyzvgE+st1JBd+mNxVAD693ggqeu076Tdu46U\\\\\\\/RFUM7JsLdwKzE1gNJn8fR\\\\\\\/y8YOGt1ZRhE3FPJVqzOoa1mxEYUq6ZfeZrPvz1zAhttbB0Edv8w6gywwUjr9wZlp6LOqBQi6f\\\\\\\/2c4HARS0lMVxNTVH\\\\\\\/lmGV9PP3JSq\\\\\\\/rjLE+5cXetO526UgiXphsJX5MADzExOseJYo2HqCfO\\\\\\\/4FuwdaBLe0Yr0PrDQwacMwA661WhT+g2n4jfntwUA0T8ELfh+LOvzOZKtIMNSRWfk5y1Df\\\\\\\/dguZpfhPHdF4T1zx14UhoeQVVYFAICVw\\\\\\\/ukIBk5VImBhzbWik0elnGitqcv5jbyYlTlVwOziyE7Xrc4tBPy3wC769FtTtP8mM8yVAo1QQ75VCiOXjWPGt4hyjLme8i5x9ZWGrXs5uivdRqD+\\\\\\\/EBBhDivXheQ6GzYb8L3E+qdKn+p9IPaF5xqRFJtVVgyiQoND\\\\\\\/z47YPpwjnwBOTf7csE9udKe3Yd0Jml0X07CzgUAv12bdYZ1vtxaLVuQDWTWfawBcGY194UnCAGCDHiePc2fzvEn949cmzgJmJmMm0dQerV4y3hJxhPcBlslhhyO4rAHp9sgxAVLdaVR0ZObIfhusUlpfp2UAa5qYu4+S8YwTxJfDRnkSIIP9ekHuxo5oAU9V9R4gkPvWidXgjH8dpCEKmfCUhzJ\\\\\\\/UCgjq\\\\\\\/gyzhxTcMitYUsH5MLPPQYOSEEnKn161Y3L+old7Mq4hRjvIcWSV9NbpBxOPRbYRxphwGDUx7r7WlMyfPjwJ7ZbFOI00VL0I5W1dRpyAc\\\\\\\/csiXt+hS2cpA0QRS7O3Xr7lXG1v68UiboFOeOvTjTEs96IM1IcQB1OnykVjLuHbaCBas9fF+C1OBAyFnbT25NaJQWC9HFcgjcx8SqGzSnPudGBXizsHlQ4QyODsAcpA++jLNbwHfB\\\\\\\/vxkg6v\\\\\\\/yqSOmvHR29srTZcO6zQS8a1fMJDEhRtqP0sH13ZXZGuielOxgc14NGiY+9rLeEodaiLXzTxPWGCUP5VUaUN9jUEd\\\\\\\/SgAiEdLUzmeJe8UsbL\\\\\\\/a+I6NG\\\\\\\/ENUE8bk1Q4pXoOOu9BWuzbf6iS6SxsJFBXIM4ms6vKI2jFERQFSy\\\\\\\/rjfM2MCpLcXPt1XYAqW5co\\\\\\\/lkkLWDB6d6CfqnkR\\\\\\\/unnz4AWmAzr7I2sEyODDzn95kKluHIFH8lwXIuQP0raYpfCj82T7QWVHD34b5d1FUL7F5NsgQN7PN6KgcHNdrKcB1efmI4SxeGLFGDmPmr4YYDjlrwGvx6ZVMGjq246s8dY1XgLczlZhhK+\\\\\\\/ziN4LIcFxvirkecv7\\\\\\\/ZrqTrXpHg9FnvDxXDoeQt02ehQFu0VSBjFKTKGLtl63OMjeLZPKlwAXM3mxnFG++7EMwQyeVYsUiq6tuHVKFcEVNb+NhLs8vQiB7YUu76ZPkJAc9xeSGIQwM\\\\\\\/U3Jz7BIUn0MXKU13gdcN5epDlhpVsdxQy02LkuNzK7wAchix7Dc+RXVKD1AA8fvn3VsreYW71aabFQI4dXf\\\\\\\/cMpznv+mGpK6GqPjCDZtoMJCZ64L1joi\\\\\\\/IO0UN2v5iiIa0zBLzJHnQ8q\\\\\\\/u1nneCjiXfolpLct1BFR04iP70OLUxOvJHTwXjF15N3nIqR5y5\\\\\\\/Yhyks2rV6UC+gYVDskmgQ9bR4en9n5CdGBp6lV8zCYFYhKZA6cFQ60ssxjO8Cst0plLrhep57U3bglaQbv\\\\\\\/U3tYXMKvQH8jOvcOHHNw+8mbIVFXt7GgQJMfUiKjUU8ajqDyFBZWDesz957d\\\\\\\/Zojl+9jooNKE8kxW2OEOeCdoqkvMl+EUy\\\\\\\/R4p1q\\\\\\\/08EVrHBtFwR+7cNakx2OStn+21mUmJ\\\\\\\/b\\\\\\\/3qsYFnW2sKnnHNVfuVE0yQRIKlbYWn9BMc1ycAc2+e6szPHnr3bX5Q5erH4YgjSfZyEP5Y3E28fXpXbaQ5a7m5sq9LvjWJNhM+sx5KW96aRaN8kYn4lkjcBAsU3bP16t7NR6yP95CvnsRkI4RFk9Ks0wcOiUREqq6i5pWw+kSBEoxzpjQNly3i\\\\\\\/kRB5nEKcUSG081aXzPlUG9oSyifNmjDAWuqcAhXZNbLlDpmta3tous7hkJ9nNQmgac4AqITOQncc4yDP3IWYcp4rbRyqO+2uIK0UYLZJ4mIRkfMw7jgqulDTBg99tCfgaTKoqVgAB0bhUHQ+VLiSxL1roFAtHI6GrX5os+tJZLG6yImJfbP41nccwGsArYEZFTIK2Wn3pIe2swsMwTLAqPhDulhcGYRi5BOHo3H8qhC\\\\\\\/YbbawyVztD8+NDBXWh6tcBSCaDGqkhwK9nOopGTQNxDL4keBOlY4kcDZifyjGL6D\\\\\\\/96VWKWN8oiXoiUCD7h8s7G2c6GlJc4qslQLoGlt+4EJF2hUh36JCpWxwdrs1+hSWBObQfKN8FisFvoa2QoMoybY\\\\\\\/+j67SgFA3bGYOTRfol1nXY8lhPubXUucYWiHzfRPTAsWqWlFn0nlydM1v2Zw1jQ8qokx0MZwFta1Hid\\\\\\\/ai7n6pgZ6RzH59aXPUHKtcX8KZTOqP0dzviFrlgEUazJElcQNoKcLZ22lv8JPZ4NOuuNWqPJY2l5Ed7kQKCF+AAXZ8td9NHi5EDfZ5fCnahy8TDtauUeLVlWIgz5kjXWHZkLksHnpD9+6AsleKLDJvldYAttromQwgJxW8BfobNwXdm8UvtefXJ30gr8by90z\\\\\\\/Xyte6s32kE8IGWUl2P5YIU4b9c5OIqO2xLX8DWPS2jCa5f2uz4VPcmbmTC7FRVSAwF8k4tUJqmz8byK30CGx+sO5PZyLm5LCwOEH9J8uDds6n\\\\\\\/J3aZeZi7cCvNElfR8RTYJJ7HYjGCu54bqLv9Bq\\\\\\\/HFl3VOivEQKXDbQvH44\\\\\\\/q3uHTmJdHmNyOD6bBK+IJoDTJDTUpNrj8q73scLpmshIKu18kX6VZ0evl1BR9zd+jWQIL5CnyA47K94naB0irte1\\\\\\\/S7PTBFhJqKWxPYgA6JgUQWaewdGxPbQz7vAW2bHQsMUaspisCe0erxFVzSZ2p0PkAf0UFoA9\\\\\\\/NAOatb+XpGWSaMtzCQxJsgEEfbkEjkYkZcdeFWl6xw7bNAE3BjKjFl7kOZmdnYOaRnE8Tp09wmD5MGNkigGPartlGKE4E2kwMqWNjz+1Hf2CZLxRwpJY1WhgBnsEQr1wQ8Xf3hw8oRCu1Ew+1\\\\\\\/zBIsVfinKLcatWkNkaGfAg9BNNwR9df\\\\\\\/CShmvzfjCu59SkIDAqE87e9zH7\\\\\\\/dQhy\\\\\\\/QAQ4QCfvncVZqtitdBHFHht6j7pgWlU1uCTE7X\\\\\\\/rqEGNvM\\\\\\\/AQqhLggFLjBpNgv30uhvcwnu+u+SQlGqC\\\\\\\/3YCdr6bZs1hfbl2OgcMDC0fP+8FyB2+uQWUtWr1C90RSBW9qGQke74Q0xYn\\\\\\\/6WhUIPcXV7l5scpBxyBtGwuntNheQpiRiPeFQVUVMTrgPYakXHqVtYWxyIJ\\\\\\\/XB+2poMUdc1ZVnViFJgaovDDacfmxitxINx\\\\\\\/sTlBXJFB9IcdlkV2mNnvGo78SxjCADrIlBGtkC65\\\\\\\/SpetcMiuhtZ0BcSz56uXO9cvO6topgQTlkb2lvzrzNLnjg9wZwAwD3\\\\\\\/WbWU9Y+G+BE43PZ\\\\\\\/NB8MNSyENp53b\\\\\\\/Xv4gez+4rRtB\\\\\\\/LNjtBlP9Ff9jPClWilQtFBz9Y7ZvPrMdsUFJI0LUK49wzyI5lwj9xP3TLgDEioCcF52gbgAUcEQwMyiehmRDUb1q48tpWswIYVYBb3zs+qI5s0\\\\\\\/rZXqKLBs1eE2vz8Zd6+VPnG9a5GAfmwi6n4cr5DUMbQajIKHtKwsu4CvI0IrlnEkXPUvqe7EQJ++VwHIsrSAVu7eUT5LyjDK0TTD7wR04ypdRiswiP34ykQX4ZytRc+4kGuwEn61ir3NrRa9ULPu+QDKiH\\\\\\\/ygd+2IqReIxkfIfMC9m8DavPXkolTk5Pduy8BBkLAQWkOSufwa7AAGEkGdIL27G0IeSYqRDsw2zNglUwA3K5awUdlUucY\\\\\\\/w+JKMorO8U36RTU\\\\\\\/lQqTm8JlvVMHrBA07PPk\\\\\\\/eo\\\\\\\/\\\\\\\/pU0zT6UeQBsatKPPMsH\\\\\\\/+OcGX7V69y7vuZ1U4rp\\\\\\\/CbusF8ILrukBIcbHdU5U2SrVOLh2Rz9Ol0ehqDCYa6vKlSb8+KN1I4W4q1AHbPhhABrBFNNN3Wq1WcSTnhw+eVrM0i\\\\\\\/COat+gK+lV25hHs1G0A+pw48rKhSHqgmXcfXz3IH6VaCQI0mTv8iXBTNYl+HyALhz5GJn2H1xCvhnHeMlgxv\\\\\\\/1iMFREd55E9iRPMfT9lTlVwVJQyAJNocU0IePkBY6PH28nvaBrlBXzuywFCbcJ\\\\\\\/DzQpkDyF5RfBdf+MceaYLWtO8p\\\\\\\/f6zxOuT5DSxey\\\\\\\/F5So3OCXRGaj\\\\\\\/A+vXq3uV9arQgzSSZm30ra\\\\\\\/YLrvUdIrqbP5cky\\\\\\\/SCMHJLR2Wt8zxB4HfN7z1TjpoetzPpn8PiqhWTdOyJjIgqwDIs+SQqGii8RFBOafFIm6VBf9gzXL+fEqxa\\\\\\\/tP2hjqxiQbtnnrw08w7Ls\\\\\\\/2wMeBy\\\\\\\/jXlccjUTRjShgeyZvg\\\\\\\/5E0rJUrEsbWU2uKIsl1fPeX1uA+Og1ee8U1hycqEs7RgwjIf4i2CeY7l7XR2lt6JH5mQQgBky81EXNrGkNDer5E0QhmoeTCYfYBPMGrISQ8eQ9868BRFG25\\\\\\\/N5uh+0WmN9MD1IgL71aah4nR3OYJgFM2\\\\\\\/fQyyU2aeyO6\\\\\\\/vmtRzWaBHqGI5HCuQaB\\\\\\\/sZZDwp8ir6XngstZy7sKCsjsIVTmwLrzPKw5z9mYCtjy5pp8RNclpL0vgMXQffs8p8\\\\\\\/1Rnz7wO11Ac2cJQ719YIB3NBhMrsabJEVl+nytKNegKHaIJIFKh\\\\\\\/7Lvh0F\\\\\\\/v5gSvybPyAXMAfnCmAirX0jx3ePx9xy+h9La6ks8pSsSmtmmyTmBd2B86zpKjdecXF51bvvSxeXn9kHg0Abb\\\\\\\/MdCv5CUEnt+uCQ+FxyAO8X+5xwQYc7hPjNgKU2rPw+dKX5dcGBxtM+9xHL6WhBDHRHyzXx9lb06z9VP6UZbpc+ctjbXp9n8A1YEKrJHDwcjhqhuXGM7j5bXHinZ4bKj1nVV0u9uFMOD3iB4+0+swbzGquDHLkSgF7fbJjkbfC\\\\\\\/6Wnst2p6wxFpXpat4ZSZ3BLYLoqmpNpltH+5TjL0ymOU8Acli6hq6OGSFECGOhBKBGbwgppAZ4ggNghHQq6RjVPbHzXVEKc5zPlBiILI0Da8z38ijdtN6gc2asjrsi88kKtac50ebwO0lQq6ArKgNr8yUpNCRYVUx5lGSRJm+fC\\\\\\\/DatrXUOH6jANcv9wBRfDJ8SJ08ZrH4oBZpJ8IdVYJ\\\\\\\/0NJdOsijIvtRyGoBNPL4ENQEkBMPhPKZfCJozntmg2Ms91zaWL4QuHWVp+Qd3KZ8WTvlkU7awMl46hKTNcj4LLrsmoi2Cih6s1jsPwL9PwzICwW8og2x\\\\\\\/pJ8XthX8Sy3hOnN6PBXhwKMXpztfCwLeacTpP5b7j\\\\\\\/Ko41LZFOhJrpjM5w2s8Mk4JyeRmy45D03tGJErxGEWggxYdqqnwZyPHwnBh9\\\\\\\/S1b\\\\\\\/YUD2MjVlELtMo1CbGTHkhd5Ck2tY0huUOrYSCb9LHldAP9V8HuDd1gdM4o\\\\\\\/O8hWZtk+EslcccMhdQOtmnsha+CgXVo1BJ2o945kGh8iVwe36sekGSu1Ud0VR\\\\\\\/2myXe5CgfBMGyC5Bei3TqmRoqkEX\\\\\\\/WbjD6Y21u6irtXZgYsCbsq8TsHKDlk9buLGKhsW\\\\\\\/kySj5HDh4cziWFQdeHNyrcdbp+XawthwOR3p\\\\\\\/\\\\\\\/nBD5DrjAIUJqbknKcnSyXGb0Q42xPLm8QedNTmy511\\\\\\\/BmXEyBz\\\\\\\/E7RDUEBjgpDSsfE1+10jYLma22f51mPclqgGbOGkpFmbXxqu4titP3Hbr6Si2PmcvYBPBg1UmJr\\\\\\\/giFZVBaMF\\\\\\\/X1c\\\\\\\/kKpJSXgKQ4MjvtTmU1IYMLXY5r\\\\\\\/yvr+ELY6hQBJ088\\\\\\\/yO3a6+p56RxDO\\\\\\\/zzVIDjZGe6+IcPFkkYrl9H1Jzl7U8yvPrAByzBZIEdisX1OxVqOroP3qgcFigCdgsdxC4czagRn+vorvPMQfoBkcdGnOST23snOkYJu1pbRbc7So6nm190Qf0rg2y0S\\\\\\\/JAoWpSXcAUXRup4wN5Ug3auUqh5zWfwJYb1BX+uy\\\\\\\/0odmljjLClp+316RTwXmr1VWipCzVcc9DlBc9ycPscVaFGjx\\\\\\\/wqeUuhtkJwv3WcakWI3wBbh6n3jZJqnA6TD9o0qYCnquVjKVJPyhQ9ebm7i9hJzuw7w91DAavWMnTDATdBGbiHxjc\\\\\\\/J3LQGk67M3j7LEjlrMY7y\\\\\\\/tlzkRiuUssZmhg\\\\\\\/NmF\\\\\\\/k5CdoB429TCtwazZlvb4BhS5g8C6wAOodwP99sAvsGMTSx+KQy60waG27mCMutOaJb5ZTUhhIuEJTBGu5L07LUH5xFllS\\\\\\\/ZQ9X6wfzlKFJcuYIBbGyIq3gFvyP30Xg+DckRcVepmB0dC2uZ5LMO+luklVxFFtRUMFmLSFgxmjKc5MJyuago0445U877IALJWe3X5+N4IaPpvebXQiXEr9hlVp8iEMvi19lT5G4u8D45MP1lqsNRU7iSYwQa2zwb7miOhREXzROSnn6KLyictyk06thgXVajjA1W3fYXE3A9sWNS+bBcuTjhWS1jl5Bxo8jGtEFbxOmMkbMr9USoEaoK49IXyEuOLikpSh57WQvfgRjwfLKNSYN+xg\\\\\\\/B8jO82u7qFDaS6J80zzyE7NofRWKmfPv7XlWB5gQ4c73xXB+8BdcSM2Xc9AnDGd7+kzq069iD6br2bICbl463EuAZPKKUfcQJjWvZbbrjgrBEk4LXeu0fIIRHtbzCIYQKlzsKP0KgXIPZ1P+nuvrjDenc7zMy1HLacmRroKkAxmf7RyYcXmaSgffMfm2mSHjYkSpjkgoKemM5gFqhwnf9u5WSpTPF2wn8BykjT1GR4yDxYQH9MxTo4YSqG4KFAcNboGRX0chtbBdSqfsMbpp6Px\\\\\\\/H6Mz7SZDQO8M6scsd1sP0lOc+X+mYdCe4o5ZkOlJYru2FzT3pcbG9ul709HJQrm5DC4B0EgF5U2wo1p6iSR6KgJVwmsCdUcjp9jErZrdeKMzkYdwNuVJcrW6nuuZPHeEmZefRBF9O\\\\\\\/QQVpgpB+CWXdPWDeb8pLkjnc\\\\\\\/nSv\\\\\\\/eNq6fWBk7wqV3uA8lG3kFi5okfxcu9AcFaTd7aE0ZIfzsb7TaSgybTauzd31Lc0tHSqeKS4W0\\\\\\\/m7NAaM6ATpOL8Rh8K3gQUBaU1AKqFhLkFju8fGETKRdygNZidJWbH9lguXM1SkQXEM\\\\\\\/CIVZalHsNbGjBdMuwqCZLsYhz8Wd1d\\\\\\\/PJpxa3Vw5Ub\\\\\\\/8tgdRa35nhtL2bODdo+39GIqqtuqQ4toKRCF4sXcqs9ziOOFoEEmZbTesyfHJvFqbvOWrTxXG4ihPAqRa+bROj2TjWJxD32Buy1vc05AusVSBrTS5\\\\\\\/2mD8gatVZmxwo8WmgVp9pmgVn+7wsYN8hi5dYm2uPmPj+T9h4pi57BY9zMgjklG5KQpdKBc01b5mf7+cCrEaNr4n7Ak\\\\\\\/Paust3RQxjs+XLlaHGKuleuG\\\\\\\/nuRaSIUcxyLIRrZSHVn3GaETw9Ua+n\\\\\\\/uqlm0HqRD3NUUGksheEA4OvDBKVKWNiNZ89zx\\\\\\\/YOrnYeMQoGWQ4lz9FcrfdtEazNGM7flKtNxsWAVQxVocKXxImbg60VIVqWZX0BMZMwQV25nIdLi2W8FD2TM\\\\\\\/ToZM052Nlsseyl9JCiS5gnWi7Cg8I9wmg42SAVndKSVtmCj0pUXxuskL1d4VXELcxVQPZ+WC\\\\\\\/jc25cr7HUgLHy4min+2jcBK+4umFpSpV+iKUDhr6N5nRscYeV+H2YFqPv9J5DTmaMvm+bHbzDfKo9G3bR3kOWNYiIsBvHza58j3UTKb+WiTJbFWd9j8J4Yep5mhiEVHYndtaf+FzQ+550TIA7vB69qCk3BQn\\\\\\\/z481\\\\\\\/ItudAFYGvSc68Edfqqdu3nqtG+K6Xh11eJTAiNt8Z39O+UyLPAevgV4KxoQvX8bJRZuvaG+G7Xbmqw6ZoACOOYhk7D8SctEzcC6DRz6ObvHvNBuFlPYtsl10wxAnT2Wb1GkEakQxNI4p8FtyObahuO7E3nDZOyganHc8nh9dXrkq0vEJa3BOXGkPQgNg15spP90G2VQWN1AJwfG6qy9Wj7lRx+o5y6NfeFa79biRyMyaZlTILcxDXDOR0iyuHo+xDroZci+BjIJConjMal8v9rGUv57GvOtLk2Au5hyD5v2ugxRt8l35vmh22\\\\\\\/OjmsON\\\\\\\/fl\\\\\\\/8Aac50TZZgPw26qr1gHPNMQH+Hn3lE9rC6iVVp0Bgof4nZYmSjs6xyIqJ0FLwYUZ00\\\\\\\/tHk9yWTc9LTyckMKuYaIetZqT7FVfaa+SYYL\\\\\\\/GiPrw2t1HyfgjByio177fGAjs\\\\\\\/+l+HoZoyf5e9trwdpZSa4Mlh1+BtqdJQ2PEjEM1CT0srBnY9FN8tCZJr7DOsAilNwEB2EL9jdtT9vXNXzoB2ArrZhFQpwq90KZunIBzfSC9hnTJwGi9R3vJHPZDwJs0Qjl+OcBnw3zG6cUI46rTKU5cSsolDLnhcWsgAMvKlWcxRYJlPzv1r6nBIqxNRGDIQ03Kn1FElKpgcALU+eL+pEdivhQEtjzb5e4vWks5F1Sm5aum+6z6P1CVOuZtmMzPX9oxM97G9iDD1DHZyuIY2PLo4hMSyX8P1gBMcdIxWyLsd9Eo6+l4EtZj0Zi3x24\\\\\\\/P99NZnmDDzmBnmig\\\\\\\/FS3XOwuTCFA2cMQgdCcOwCQ04xLeaPlMAp1GnXsMwMCLCFpOgQHZeb8epfaCR\\\\\\\/K3TSJpT0qdrIEGItI9+cUh04It7zW4L\\\\\\\/xmXnsZKSwxRcz3XsaGd07Shru0D3t9jBUE6K9BoN55CxODxZbcnJJAKuUvhxOS2uaHk89Qh+5RDQ0UlxTbQpyYu8\\\\\\\/FMWKBbYUN6uwczKNA4AB43YBP98CkfJaxWh+5GCQJuBz9GeJWnkF5uw3drNeBMYfY1DZa6XN+xpCHFJ6\\\\\\\/0dCFpURn7PVe6Qn0AIqI1Pxv0ohe5feWKTdkHz4RCCg1hsYS8BI2ae0Ui0z5BfRL7KFJQVj\\\\\\\/4TWJnuyQvynzQO9IMI+iedidukOByTJ3AyNxeumsiQFPE2ktZHbHYPXptAhvaP2RQYqTJ10BgHjPgmgoINl6szLQo0\\\\\\\/zpo7GZR\\\\\\\/n5NZ2eyqig0Pvpq030X9EdsnhhK99BKwUgiuZrfTM0IoYU+zk61Oh7LMizvAS6ya27en3RdE1zKSm5Y4TPumZosVkQvAsemo4FLfVxlknUnRsurKB\\\\\\\/eAY8Iv9sEJfYeQOeAZva9ovSv4jSyRfWg7oyPswcdWnCCxSwuyDAlEs4Fplrps0qeOO9vvGp3a1BjS2+J861jh2Lq9JXALrMBapcryWt6nK7cuRnbiG0XoBWO2BcgHgIfpBcRwcv5FQAekjfOM0x+3gdoE09t60Uu\\\\\\\/gchlgC4zAC2ETJ+Ra7+AjEV1R9NpMH7XPdn1jXtP7LT52ShVEEijkrYsggRiJdRVEw5fdOzTZSS64h++kK5AAE1iXE3lCNanVWc1yFR8+4w\\\\\\\/3lhV0hG21Mc1HkM8Oh5vg899s7p6UX0D7XsYHiMWkGYGpNkgnYKs0D2xjtGtXrfy\\\\\\\/CRF6xqz+0kLmZvTG4F\\\\\\\/LzmzpEKZw+spwl4onEA6L+Nugq65FyeyA32jT5liPNuBPEaW3loxInbjhTxXLwN\\\\\\\/vLnFfuCmgTkRWPZPpP4QZMKqUH3sZuZiM7A\\\\\\\/OdjDz6vKSJeSBqRjCg\\\\\\\/A+UUSyhHz3AvwoVM\\\\\\\/oxyONyzRjVOyHwzDiTAPd6TXnhGUkzryWvOPXzvbSWyxPtYdXwp+2vaeyACQIa0uvoz9AI1nBDW2RVCDrdet6LjkTOM1FR5SfqdU+cJVLRbvmkDUVeX\\\\\\\/K9n4QOFEoQ453CzGGOeQhjJ94mwG6NM4wvBwADo53GIir4cew4gckj24F\\\\\\\/zhDRWEr6nBVI+N+442KcJh0UablyBgo4pS9JdwC\\\\\\\/w60p3T7KW3GjcNJMwfjPce4bhtV9v3DaNeva8PNdJHPOPAgWa1NVbtv5hlqTbH3Wq88MGvl5BsvYFoXwtTrQ8ubyEVQnNnYd5rvImq2+AeZMZB7YLv1A1GDaoDmv0nRI2l2cBHy297brLZiutPyo43kCzcvrnwR97A1OY2\\\\\\\/OuP+peyJ9BeZ\\\\\\\/VHNSy0qeoCw9R2hcPW3hUCA74L+OheIuf8MtW6GpxavnqN+3rwdDRm7m5i+rN+4VmxbiJIxiJIOiO2UrvQpHzS\\\\\\\/UjIhYDOOtAfKxHUfcRrztNnFkRbFO2xhNZWuaIYw9TySBXvX+6+otdzJxfU6IvgiMXeWnaGxjx+ClcECDqpop6wTlkMS2kM3FQai0Ms\\\\\\\/0PX5Vshxau7xg4BkDkYlHsUfJrWXlJVJ3vPSxaPSuBUQAfIXKkKTlDTOvgvMeOghAYIqUzPPp5MKt+vzv9HOP509hhRMuFX6tlUHiB4jdW9Yfse3tPWrbpTsvyxot1JfFUgZrEFwIvjreqtZ9cE+8tRu7YMDmjo8Vmymm6F9QBBbsMeXmsLRgM5No7UhUJHt1Pj2nDxZzwmO\\\\\\\/VQaXgPN1X53Ps7DFC3EkPE9JIvRc4HGfgOl87GMTZcGzBT9hvVb0lzMlrwx+ItiV\\\\\\\/VMNdQ2n5Negx8v20pioTMy\\\\\\\/dkzYhcc3IksYNqEPOJ1iUi+Tl9IDA367JgYlgfzOQ\\\\\\\/wIau3Uj\\\\\\\/GSQtQZUJPRzzYstYDUZZcmDDKY4wgvyJ2TuGWhUV5KXimyqEryiYbTsoAVdqgz5JhFI7RKt3K7JADjYCUw+ybBshEyiv481CeXVOYluaZ7b2YOMNRFwqFLrFg5ogq97SMXIp5xyAsUXlU6G7jiUO9JJ0ve84y5gA65rX8TbpqdbZfQ+63a\\\\\\\/PvhUbUFpJctypBRQS0eEbnOWmmiW1HYjCso75Cn71N6mgKn5yeIueDTcYnvCKGpOxA5M\\\\\\\/Pwhpnf5LHBt1+rQnMSU0Y2YBZGAuK\\\\\\\/5SShrgfNiWglIQMTSU3oEoApMcPVEjCDt8DUX2SvMlLG+d0oveQCKJWNkseqNRCT27Oq6UpEI6rdoVSADtuCrHCUGr8TJUzNXCBRfnVw8nGlWw\\\\\\\/GnI4QsBakjreYndWoGyg6hF7RIo7H2SF8qak6GV8rJxncMde+MNH6Zmtc5iMAp0hyEQmkI2G2eU7dmI9iG1Q0LAd+1Csipv4vzbqZj0dVKCNQQD9rBBs5Wp7es0f6Lc81XGvMsxSSlA8\\\\\\\/TdrBmFOdUY+co9BeFiQhdZ\\\\\\\/3U+1oTXFoaD2BjF5YEaAzgBJIHrqSEvEygO6iG0Hn2w5qZB9xn5RPTPvIaQsTwPGjsyKRbymTXAluqy9cqz4Ghz5Q\\\\\\\/J1I04B5DibgD4cP\\\\\\\/D4DqYiSWGjtq+tzWJMWlOOUVElsPO3\\\\\\\/SDtY0p7DygbdH+bqtjFJdYcHLdRB8RmfiFNCxmjWRYAQdYugh86JhRobnSnrAex6fg1HrfeTCRY2bySwDVavxy7IU1NCIxfhcIfEZWbljGFeXygXDeaI7y\\\\\\\/vgZj+d7okupAtLUuMZ\\\\\\\/GFZYrvjFd50tRrsoN+iCIAvMsXN6GcZTRwAm1df7LGU3+0J4SlFG4xrZUFjXipJMf+YPln2jg4ui6FndOcERa+fhKgxezaBbVICxeLKpoRJPWgJLXi9YZ4yw1NZG7o0P4W5wF0ydwtmo4rMNm5jnlmXQ+i2RB4i3M9EtVlvWQCH9LBo48RdvVQAqEvtdcs5WgGCxJyxOhtf\\\\\\\/mfLo8nwsxdK8gFw5EXrCMePeEL6D3HW6KJi6xiiLuaY8FP\\\\\\\/57IkGLnovt163jSJWdq4w4CMYmZl93roaAzPM31ypCkzqfJleAnmAd0ze\\\\\\\/4sGaOZ4XChArPTITq2uczjhxbWjImScWl2VIEAbABsCEsg\\\\\\\/RrOyjs1MJI0vWHbBU9wmOIIzMOWqUcJsHW2veCVGtyayFPzM7s5AQgC0PMyti\\\\\\\/iI\\\\\\\/5ukcCWPvpnvraoT4g7fe3gIsqEClGqF2yjcrkwpUT88OUlcUIIiukvToECvsHA4y1Qp4C7FsEWIRUwow0sKCtXDnbfsANlf\\\\\\\/4cTmD7gjo99VViQFeFVYufQBw31u0ebC+EP5WZ0HDeEi1wzmz13nB\\\\\\\/70jQTwq+xt0Yn1zqiYy25xJ0KxcKValE4sHOJZk3NAv0sy48QyrmzjObGEJ5\\\\\\\/3zQSDUYjtcEXALguXLNSfdANigsROSuXZoObCfewZWXKoWOYRx1fKgWgqVPApiYFHRO7btpSi5PwzmEp\\\\\\\/NOHhWnRIBuW+X3lhfNQW6Ymz7VCnoRIg+DRer6zYukP1gAOPUr2kXzheR8hWOPWpAwgSayMxYOGt2TU2b6bPv9m2FF8eaB9GL45QavNCsg86wUX8p10f0x92XJPXW6yUHHdqqDjFuR3bG31KCVw8WI+Vw3W4j16qAMsKdBnyS029pbOSYBxV3eebqS2iBLI4siRATjQ+vOOCKq\\\\\\\/M50q7QqTznztTANx5qr\\\\\\\/czb7jD8n42daxLbkdJ9MI65n9Vmx81Qpd12bioI5bUIZuq3ZLqXeJkJspfsnHekZPg5uw2LbHthIqaXU4mw0rgSRgqhUBraqWWWpm\\\\\\\/0rX2OhLtmhsxrOu9FBfegoMtB4pSNo\\\\\\\/lHZn0pKgzEXWrKD2wcM19pCxgg93Ze5DEUiASUg\\\\\\\/BHdbu4Ki0Y63jD5cJB9oL9pkZwiUueDb9SXpUrwSJUd\\\\\\\/CMSqki3TfJma6GDfykhUpvYlON+CP60XFf7KEC6I9N56iFVkyhdtTxOZ4DwP8VMEQsXPjn8YHzo6UuBU7XpG\\\\\\\/m\\\\\\\/SepCrDtcnU6PYBejB0Jg7Sla5ZqmFCRq0rXW31FPj3PuD7lWXfzR\\\\\\\/Nd0TyaZPf7Fl11wfMQFvOW6G64mwQudKQxjDfHmYWdwG\\\\\\\/ctmdgvJ2ZdxpF0YyBLhjwI5pSgFvVRx\\\\\\\/iDnqHCN4wuaqk973VQstl1Aaw+GVLGRBykeEX2PXXjmkiJBD38ZbF1enWkIyOmtTTy54WgnSonh914c4rLyycdiipzdNcr4WLTXSdfca5Usw9fBmUcO0ai2JSa+RljYbO7W+2GayJqzIqyRf3uaknGdkHVf6R+5v0TkizzZNR\\\\\\\/Eot0QdQrGzUxltdEmmLiz6TGWluvzUFRf8q9cTeYPBGam2ZewoNcDVzl+B4pFsaM3VOTInXabH7ohZDiMUdorV8atjbpd6xNuYnQIOVvtp42qYbUxONpmTVBDGO0jDOi1r7kc+aOBRzXi1W8Cia1MTM38GCRRUFkgIVNFmxdhx2Qshsk\\\\\\\/i0w6QNQQybiMhEBLt7rdZz18k845wzCcEtT7crYNeA1o8ulxKskLpitAwhqCrFE83wY6vnlwqlq3Hi4x1sGGq1\\\\\\\/\\\\\\\/5Xto3BQVjRHO\\\\\\\/OhF88Ud\\\\\\\/NGLaTQHv11Fi4x+zbOWJK+lyLWTDzbf4+WzpjQ59ZkPkk9o6xpSTTkjl8L4\\\\\\\/jZKWWG\\\\\\\/vTmWKMzpIPdI2PVrwt\\\\\\\/xz3c7YfV1zBFF9EDKDIY0XNDRloc5HJ\\\\\\\/eD3Jo5SH3w7287lkCFHXKOtFj1Cujafyw2zbNHtxXGIcOgK6DojQJOZZktXNAjz99tPtDFUkue87NrOsNM6DrhIhcWZYVXqCK9UewawnUsn4XmUFQ0FO4CHFd1YbOKOG73Eb+NJgZfHJfagLC759atanbEjHo+q+Kwzd3UlUROmbQWYRV6hsyiz7VrOQIuxPry1OOdBkdQHB5sxkDdiTwRyGHRCoRlwgbQPKt+ISwo0BOgV7zWK4JXfSVZ1l7n5QO98jwmY0X8QBIYx60vabcbOfs62GwBqt8q7B3R\\\\\\\/rwWItAG4ne82\\\\\\\/f2\\\\\\\/T\\\\\\\/suuTqsMjJWlbVyptrvJdwVxgN16I6iZwvKVkWy4RA9aZu0vldCDHk0\\\\\\\/H2QQzyfYJRV20cPgojw2x1VltG5wBP6aN8wSo+7Vxll4Li47ezJNeSRRwrLgPH0A5SEdUkOPeWVFbJJRGOJvAPCRKFIxCmcTPnSPiY5oihSs+TsxXFqYYAH6PcA+XFlJNkC52Z036XgsUDvDPSSdreIyWUgBLVqrlIxiO8vL\\\\\\\/Yrsn0+jSSUrvD3VmwpiJNwWyF5TfWinOrD5byiseA38TTFxnExcJca65Pdg19PZ1PywPOpbvDybq\\\\\\\/xqqPBNgz0APqRtXJVKS\\\\\\\/E0qnJ2UcTsfFV1xaAAk9UjQkQ1SAac7VzouEBWqDqLHmmjXCLuVhjaenCT18EmjfRRTwmZERsg\\\\\\\/3W4LzYqC\\\\\\\/D7MVeKm4zAlip1INeVSwwKX\\\\\\\/0NcGu0gCEgvMY5NAJTaK24Cub+3WgPzC+IXvYfvAUerNTTSnpGPJmoIZQFi5nqIBQStY929MF\\\\\\\/7ohOKBOedp2HJPIwvAhpFxpDdC9pgvxMEVR69APKeDnsB3nT05lTTkYurN\\\\\\\/o\\\\\\\/OYxRMX3yAX5eeFfwn4NQkM\\\\\\\/fuOyLzSmWes3bVu+JigK9Xl\\\\\\\/awwvXBYa1QLu4wp1k7hEw4M1STMkb7z63r5qVC9I\\\\\\\/yELSvFHcoAoRCW7AHanjQGIDLC6cR+vIjhiYc7qPzF\\\\\\\/AlWIkvXC\\\\\\\/0nDG73DBcu9uhhgEZd0kyzmJmH\\\\\\\/AgsLMszLsvi2RkeaQEc1jmN+Uwpg+qmnuKkAOVs8HIh9unRZ+a0PvJC7aMu2Aw8wOFIjMLbc7QnUnRihHdyu5NMXpcWit4HPg3wc4mrDEoA4BTsGbQLlTalhxHNPwrLKh20skhft\\\\\\\/79AJdjUcYXVJOm0WNO+icfdp6VXGotAT5Hh4GY8JBq\\\\\\\/vZL1BzYdIjfgsEF8r0SD\\\\\\\/KFhctjUj4fgXtiqlNTVol4hINcXVCmSDOcRO1WnuVt8\\\\\\\/kq1h2z7ODt9ZUt3hvs2kB6x7rAPIRp+iUm1ZEShN\\\\\\\/EaUG9r3j7dVzXNU7Rk5G4iHj64zhqv7sOx2Cp4r4YPfOHeVCrMQeat7G0ac+YVV7DtXaF57xuBCFq+idSF6Lo1FkJjykf+EjgnaHcl6M+Bok0UHf4eEEOD8lKB0wjUFyEdOU1VNrXufdThFZQol1J8SI5fFNPOkfjQfdsCy7DQ0BiIVC7lOax2W3nCQd+jdKXs7BbzThUOZWdotDls\\\\\\\/t8w\\\\\\\/UCc5pxVw6nqWih79uDYGPpR4L4dq9xRtD7xYklL+dvx6AtQy+To6AeNuM7CgmdCGV+W19gpEF67V06MrJ6umQzxUV0lHz8w5oKAsxcATGRvteS\\\\\\\/wUzBmoyEuYdzWb4zSB+jTPl8OdWEycmMr1drLVpe0Yxxi\\\\\\\/ubz4MeVDTu\\\\\\\/Vz013q9lG7WGeokCOSVkJ4dcTvuGEeh6Jk8Jven2I0yw5jivhG86sCshVrPwVswngMB0cppgIIj7zWqRS1LloDypV6Wbak3Vva5iH\\\\\\\/ilx9uiLOKibeQaimeUWEg20a8f+3RgnJ8jXzpsw1BF5PaBlDfGRy9Mj8zPnEC2NxmEpHbrzE8ohWGZ9O3nii2EjfjRPGh5AMESFHGpduUKoRb0TBfXSwkN6PiWcpUUMgcyepZlzbSNXJoEXM\\\\\\\/+aOIsn5\\\\\\\/+S5fS6X1cXWwA9BjMlRmdOhryWErh50i4z4ZrlDiATX0L6YZuq+b3aYG9VZGRjnW71yRspMGCnLB9f3DeSRHcZtvfghp5M\\\\\\\/xckL2jNRNoLtH3pMyZTWVoQ6S6Zjdh1V5b09JL+MDzl94SyY\\\\\\\/MPkadK9BlIhrCVqd1sHhJ6fG0WCr1dEiG9oAJZ5JH4ACN4K5diYDgGzpnyCWZ7CaKlHgfvrsVYdMsfxAEEMXH8Kkztzzu4obgb2aILXjN5vtKnrK7pSyIwZKil3r7FUf8YE32wQAyQtzcViZEowkdwzrstiFb9wLM6I5Vxf\\\\\\\/g\\\\\\\/vtU15E0EcaMCralRpJQPIiP2mzI0XCZvW2EPH\\\\\\\/1hqwwIqydF4aoFYnJmWbaSnxak+TkfbMiGumbcYZFNRa9KuyMh+4LhGSRz7Fb\\\\\\\/Ur7SEqzMVrDrflyL5nAsU4W\\\\\\\/fBhH25ZIu9M+ts4nezieD0ciQ3bt+w7\\\\\\\/YRmrKhvx9Y5dNLnmphGX1C2XL9XklL98ORWTLRPNfOM7fBfDk\\\\\\\/oo01gONFrY3Xg7lWZ\\\\\\\/1okkTeDFaVdKNoNxHsPHK74rT7Iy6Y\\\\\\\/Carr3jlBDoJhcG0GEbV0jAYVKp2yzHI0R3HIACH8XvRhfzvdfXJX6YQuWuSP6o3UQ1OyjQBY6u6KPmuS3UbqGanklgkfqYSGfPnmlpKl1GVUSe9rt8X687ZxoW56+ZHRxGuTc4uTgN1e68b7EH4bdAUZvEf27lhflJd26sD1hyGDoNRvG\\\\\\\/Gd\\\\\\\/\\\\\\\/4FviLop7Fg35fshGTt1P63jeqsw3ZhQ\\\\\\\/hRjX5HbouIA76WKrgGbajcjulqhzUOAY7ZZJqJ5iANyd7HGDFUf0kAGk51oZvQZq+9JA2cXNExuubCZn2DTX0oNvS4lP7p1nINhxl6kYHiyCYZwv3JsmGT3VmoNRhtlCMBtGY1JtYZQTL1PHu2JAD6d02lMsdM5Gi6O+92u7R8z2W+jL0EZB+YUkniyHNiBe\\\\\\\/v\\\\\\\/63SJuX5pZgG8XKE\\\\\\\/X3H\\\\\\\/b3yfLvrl\\\\\\\/x5wHJkuLFNtf4xyhc3hjbzyuqZ5BOKGbKCidhbKbhoRLdNTaD3DaoxY+l4H8cXPlbtpaKuvOZ1fOcE9ZR5QQkDWyeUWOepw6G+XgwY5UNGY0RfxJKswVEBOX0dfQSF12iqfX4wgLZJO3DVm3K0tOBPUV8EgE3Gnzedf887j9p2cmc+cJgXCoef7TDADSeMI4VgMwR4LmhGwXh2+TVYZD6UWpYhXAdbvgz\\\\\\\/2dHJXC9OEL0enPsSA8fdE7ZcMquLrM68EpG7niD7SJ8WeG6b\\\\\\\/vSu5g9KWi9GU0cTdfxJxdTniNNejXu8ZAPWDLulfJyliDsJW\\\\\\\/jbQUnDrswxpqbOT5rWedcdbge7pk766D05eIUdCh75qlsj4GTGvDZptgZ+vbFC4gdZGuai9tp6tJfMK8S5t5L11FKmU2PDbT1EA8\\\\\\\/ttFv2CfLmMbnUeTlUTle8NnxSI1tA5jUgOOb7OFwYcsjZ96nbOoAuN6BCgnLEor\\\\\\\/kg2RIdW0T25186Ac+p+HQPfn1E8bO0kpuJ4VmiPrhlwhujSTw0NPADC6WXS21QXglko0gXT5A39bjlWwkschu90bNXz\\\\\\\/gmKYcVwC3IW2MWlwrU8GN\\\\\\\/0QV\\\\\\\/umnpcu5jnMiZKb5HHTbi4mfqnlf+fs+e0Zehc2KFAPD16sYhu1ezJmEVBhSPlhv\\\\\\\/MNDWPjOvH0nxMaVZXF9ccSEdnbw7YxTsItUtvSTcgHtO58lRFFbOoozhfT4oSs0XATlNq3A0Utxlr5wRH6U\\\\\\\/W5r6HDcfany5MGdp2gSjI0OJNAbuK++vnHFZ+Y\\\\\\\/WNTesu9d631qcJV\\\\\\\/lJN+ClLmvll1VSRzV+DVFDZERVXjx3s\\\\\\\/oCLAsayNpHzvHneWVz2TFhGEtYnQNh\\\\\\\/OoN1uX6AnsHFe5S0W+i8n4nr6Rt6fsvVMPCJdpXkY8MO\\\\\\\/pIp4gZXpb9qV\\\\\\\/vA1dsFVmtfA934cFP\\\\\\\/41Zp4MjukO7cnuXABastpEQv2\\\\\\\/Cy7eULI0b5ECHQz5fgNAsTHWv10gcCVsL5T\\\\\\\/DaPs9uu6oPKF0szGITbRemVkkDo5XCAyA7iGVO\\\\\\\/0eY7w5xuHx0Zs4wMuAb61rkptytYlJRhoF+3Fg9ViLQeBgzz1oVykXgDnaWOjokD8sp0m6GdIAUPgo5LBiWmWAPed3eQ2gHbdYqtfMw8Sa5oZ6P90ALvonhm4DkAIbFsznZQp5MnJz5TkaLxpIHCMzjo3whIx1jcZt5rXA9txjkLeGfB\\\\\\\/gkMttYEslDCgvxUd2+ntdmKPHOEeLydHi+8K+iAEDqJaauM\\\\\\\/0PhVnt3OHzMyOtbG4Fk5GQPLSs5EJpbG+KYJhs7yHBJ+IvOMxE5BMCgHSoPuRe+pIF+hY\\\\\\\/RzLu0k4exAR0QmZu5epLPBiRNZxZmc3qwk9PXJHS2h\\\\\\\/vr5LkcdQUQ1P5EYewpc9hophCym6+p6WUIoUWEMOaJRuMAXxoXG6hJQevxiF9wB6ak8DM4bm2aFYM1zQomzGf2QyC7+HX3tqVJrMQ11HyJCP5N8A\\\\\\\/XxNuNEvA+HBxzkd8s93yygkAb7gv1Cb5GF5mW8NhNrFwkqwYxAy\\\\\\\/e2Z0Hgq+sRf7lu4o65zgaPPdxmosRbdejZd\\\\\\\/kldL6EVs7dgixflj7E6vw4ayE\\\\\\\/P0sz0n\\\\\\\/uvjygAPD8u6mNG7WyMfeA57JFw0v\\\\\\\/rrO93iKUuN4k8gB0\\\\\\\/QLDeyRrcmVfoL4FPb+RU7w9\\\\\\\/siB5msi6mlcxzYe55vQH8hfGVYhjj3XD2azbA+QXEJtzgV2cc1h6GCyN3uCidEso6Dftw\\\\\\\/xNV+BDPoaP1pqGb1Lti2GEY3npctGk3N50bYKW1ODUmShW8DPtxNkz\\\\\\\/h3iLgwy0Yy3MnzoQZyWucAlz97AdXV5q+1SubXSgSsSIH6HkQQBUzby6VGOLmTJ6VFH+GGkxcS0pFyeMW9qs8VH3Ir2vN\\\\\\\/G27ndAjTdCSunX2cF0xXUGxyDSYHKLSWaxmwDlDdcbLu7JWT2E01aZ1n58FoF7pMVPCR6pibD18nShiIuKN9EzKEfbppEvhmEdiIv\\\\\\\/DcCTI78UKuCOeEPnrvDCqWYx1ui3VzOSaymVC9Z0msIe\\\\\\\/TYsNE8PdpsvPPfNV+As4jTtpiwg65FNAwfBHaiBRIWZ9uzarDSFSK39RtP8VfYJZ8nNeL1jfdTjinkGbcrhM3qy0gp4id\\\\\\\/BPT76BNxg1pMWuArzEyfmnVF75NypuFf2yBOOu9MoUWXMLwsesoMas1eJyRDO65wANtGm4xMI4TaOToJ2HHExLfxMweWNLVaBIqqx0DBdyjXhuiG\\\\\\\/j\\\\\\\/IQ2DLN5Q5GuJ7oK3g7Gi+c88AwVmI6c18rxdxvt0M5pzq6P\\\\\\\/E8JiTrGCatJRE9aYpXfp2Zho8X1UU5vvDee6\\\\\\\/OnQYxtTw2h+AjV6Gfs6xb7k72Jc6XkkN+TngoWxQp04QLiG3ejYQpadoW7+inp7zoXgQb2k1fkFxe6rt78zIbttDV9Yzxfjb8W61K1zdc21RtYKIhklto8phfyyzTav77VEpTbVXZPa6HuOkK\\\\\\\/yrLm+FvEmUi3K+GyDaBoiGjqTvg+BmmEQm7BIGtro9QULJUGZPItMVo11mUJ6jIpKrZpkQev1CdQNgz46wmyEXcCXC\\\\\\\/dsUzOOZNu1FXNDXDqpRG1osMdwLXLBaB4CIp0xO8lKOdFyCb+ktUAOCjcr\\\\\\\/gr3JakZyXmRFojsyD6SQFPSCYWVLN9DZ4QdhT3wBkKyuwAmXoBgpvLsq4ouZjf9YRedPQol6riBpie6scYFo1OAaLfRLM1V9Bk7W7MjdYpo7KEFITwS\\\\\\\/QMjFOjB70HWIEDYukN346\\\\\\\/YbnqkkhZbu7zb31W6ZV4PuL5216O\\\\\\\/kNlfOUrTZrJsusGWWQI\\\\\\\/z9rql6c0yPpSSz0jeHMP2+RRctJdR95pNCvKJHS94loQpldMWKrF61kkNXu4tpcb4INMsc3nPpXAFE56Eonewd\\\\\\\/k5gCW3G6NNt8e++WdqPtLdP4TGn14DRoB0Il501jIVd1Abo1TzKc7Rni1nfYHuno2f6e9BHy13vj2xkrotl+U7YK2LB0fHDhK4wM\\\\\\\/Dw2en5IkLEHiOpzDH6AcDeaxA03VZgQt7X+QgrF0DPiAYY0FGQNvvaYkGZ+jH8bIUGCdzDIUjXuvx\\\\\\\/\\\\\\\/+EbYBfdPuwkImZ\\\\\\\/\\\\\\\/IMtzB+2cnph84DVYR\\\\\\\/GRr57c+PzDjdK\\\\\\\/jTCRQ5ZhiF6gIdy9VIAFFjoKafQMrh15Ji\\\\\\\/sEWF4\\\\\\\/IISEbIUjUrU59MvQ6uky98jOTnmU6ph4fkW+vGDtlmQkQS0OroeUrD+hBvqi4AMutXTE+7DtAifamvEvdXHw4xwG25Frfa4Kpb0tfRXCRUX6RWgxEijpvhXnHr9H61Csr6IABXsJ7aRQ3nTad3zzzZYY0pIRL8qZvB9fcFsAOfJaadvRSA0coIwdcTzS7HPZEwa+AmAk2wLBabRdMPJXaJgGnxaTVSwO6HH0qRM4MdkfI0mg3qO2HxzZ7hU10LFjRoqHhU6JZixBCjRl68MNJl+TIu5DCZylilzp882sNw7IUonHxIXUpI\\\\\\\/GRkx\\\\\\\/071E\\\\\\\/drkJDBNKiNMLaZHMFX\\\\\\\/U8\\\\\\\/+74P+u1viOPyXlrd+cyYUhfujCzkBpY\\\\\\\/5pWmQNCha1XnP3vpMRJFGVFJOYfJVR1Ks4eJOkMVuCZ4T6vmA+JmvAVZNcvuDTUap54BKpygWJ9+88dfA6tS7zBKdSV\\\\\\\/6s6B2cGmAuoPVFvKRYSYlJu7mPitVrr+RhzD4ZGzf5LYWO7LNxcYeOl7X7p6GnQ1jCNON86sMXCx2P1Z3O+Nr5pQPqv2vnpMei\\\\\\\/pIFjKrR\\\\\\\/H0Yzxyd03O9yFavGfL1JGWgi2hwj\\\\\\\/2OjsZabyMVMXJCpYJ1E2SxWNhk+QfWjZjY1IGjxw6Tfzqmh7NXuPK507wvYnFG4CJ88K9c6GTeUZewaNP9wJwr2V4luqzdtxcG2LLPg1wdmXPKSpzHIqUucKsNMiPjBvphdmx2I0Ei054OCVsbMXZWI+r3I6SR5P8Ma5Y8PoybI4atXy9SywAUGOdtmwER5bkuUkB9fem3jXQNP\\\\\\\/NFQm\\\\\\\/QNbLdnZkJeczgrBmIvCS6urThEs6EdlQ3S4ayVm0F6WKfRKLFiorNkAu3YL4D\\\\\\\/nkioOOTumL+rQShriqA1TObmLuEBh\\\\\\\/1AG5XokZNrnlI2k9OxoPiCPhtSxymWIv9SxIl8YL69mcrjRjVa3ojy\\\\\\\/R17eWeuBQJduavEuUh0wYDiujjW8cNFQVmhBLgBhRUl0qFogT4nYS9Td1U0TzMY2ESxMtsw4UfIer0NdHH9AH7tokdaPHVCpJwCUbDN7yxPHflDS45OpVGFuj2d36fyoLRMOD62bobCaNAFensT4R80pps3QKUAzsIRIzN4p5XGLe2UhvZNc+ZCgVyR9+K1yxvATg46HoZQPPOuBnLhD9KJvFLivrJPPKLHEIiECHVXhASA8Z4WuFlMi0O1tHNM88eNx7bnqabRiqg+g0hmwFJ+sB7xoqDqWoBF+jfD5VFD1\\\\\\\/qDjhJ\\\\\\\/GNOgEZYfWVZ6yf6QFqO4fmMJKmN4OTBMOa7FtEdpRNOn6qcxEifxBTZKR6fBgxLqvjFAXzF956fAxxqoSOfGlGhkx\\\\\\\/eW7RCU0mpymlf\\\\\\\/t2T8b+HJzySGDN1AEeHD+OEHccrbB8S5LLUEyqqtgX1JmJTtro\\\\\\\/pXtxnJmb3MEcjk6L4jAeBOxWh2ckZBcEDx8NGNep349dfw0+EWgg0rL\\\\\\\/temE7vXxbC\\\\\\\/DLYeCnTIC0aCD4hEW\\\\\\\/+qIQb2yuahUW92bNZDzrmScqMDLO1Xz3HKZoWMxkKgxQaU3MC3VB0q2geoxfqO8ibkseMs7tDjQ71x11LA6j5gt8r40OFhmxMMl9U3GDOjJ3UYUBRHm0Sc\\\\\\\/pY09gmMvkn4Qj+x3NwsiYgL2xQyDos5coX9U96KQwfXeVeVF\\\\\\\/Jr0FXnAka7LO8Wp6KSwCNl3Fh8+\\\\\\\/NSra+s380jpUYMZkcSxe8LhX5kp2NtEu1XAJ+6ri97IkYIJWPGu5Jou1g1oVHmCZgdM840bCnL4LuV8+ifp3VSTmQF7IvKuYF095TifMsFnPUiqpCy8qTJc3tQIYBjuWF5ess+SggpbZ2379q5JJAsYzKoeZMBvaAmwTIx8wFnFy82vyphFQTmfSaHuzahYsOCvHvz95F4YPZuberhOpUHe2H+fbHvJpvvygOn3nFh8W7F37mgdZQLNkX48aRhBBJsDsAUMGsVuHbFVugvwgoN5HUUhwmmHA+VLnsetK2tWfbcl7JhlmxipQdOqL6OHA8jLx2TAuZAoZEu50l7VY8n1rOkrbOw7kzfdnDo2QLR2TheOsqh7szDt7vcdVVSoZ3\\\\\\\/9A5PUZx4vibGJFMjrT0WBpUCFyzxD3CCBlQIEThoRctwNuV4pani0owUISFILItCPC8BQDQ7gdkAoA+GZGaXji9SoA6AW5xWDbKMJByXRqIhwnGtMLG0ozkjoc\\\\\\\/WOQhio0\\\\\\\/3BbPtQxGX0fAETPV+usiNdTYRkzITe9AMBQjTJCoHjlUdZ2jm2+h82\\\\\\\/KhRey6OPR7C+U64xZoxuJIZM29Dt2qi+Owo8QpIQd3lbyjq9PfsenTjswv+cLixrxI\\\\\\\/MHJ38aWGc1qic8PDOn9o8CrWPa8lhEi59niPjhDCGvt5B0opwNhilxkXuqzGtdiO74HAjGAZ+GeXxHrx6+PHWUZlOeHpwgaZ6QDWcwL\\\\\\\/5Lv8KXY9hlQr6MPVSF2SjXxEgaraVqzCTcmfpKCDoBqSVXYSrUYijwWEqgsgLZKvGwVL2dBsxypTEGjwtFP2vAnRIid\\\\\\\/IwJ7gLJRDdw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DVox8YmtrlTgZ5rO\\\\\\\/J2Ofg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2c7d33e607af1350b22e1f42451689949510b0e29c723acaee6e6b1dbc4b441\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CWhCVmIBs1hrjDXZsX9Esg4FcemV8SRr5D79tD+PobqzrOo3wV5Suw\\\\\\\/m\\\\\\\/Hp9U0GWVO3CqK0jSMh4pbpxrIbMCFHrZ1htmNOQGLyh8IOguwNmSIZKpgDbqlUjHn7r3iTAVMeEUdWnZZjWE2OzsKrqXwBDGQXaUR8S6EJvfBuleynCdeUvMlBPMfbmfRtW5O52sbxOx0QuTBYjC3sQHmhAgONtDhpx0TJaSd0tv5FETA+mJxPqiXEeqVUiw0EX+SLJ8L8ShTVAC6mvMM7KJUccM2VQQYW\\\\\\\/r7vfijnw5eLWn2uKwOoV342etKBfQDdBCRBshUjRyNl\\\\\\\/IAOn+U1HOr7e0AOP3S94j7J29xF\\\\\\\/IcC9MRegE1HmgB3IsL4oghnBZPvWMWBvJpKuimmUaFMz+S38rXsG32lAqsO5PdfswTxbkpCWrLyoP4\\\\\\\/u7m7O+ol8U6jnHkYJkx5izPUbPf4o3mpVI8TZEw2b+8TKHP8Io139z+OpdkuRmIUBeWEQKdyLECWMT4bQ1APuyCHD27QD2t8TeyUHvnA+KXf97vqHS2eFdvSe0fJO808zrbAbq7iBXUuDsYFgqmOlEgBmxJSLGsTieEY6b1q98YLFNDPpwdi\\\\\\\/ZHpNqEjS1mYkNMSPXFNzv9XPILaR7VO92ldyM\\\\\\\/098Ol\\\\\\\/KCRvjBQ4141Dqi+b0hB9A3KA4bpdRzdF+R9mxl+pa3NaymuUaVX\\\\\\\/Ws8oZAk2iVPK+DxIEZ3+ShiOWjxjEUawEHddjNS8RymlwYrnbG4Ag69RuX0EETQRrpyHuUrqIdA4bfbtFg5+2YFt\\\\\\\/EgV6X1Fk5\\\\\\\/A1chuO9uNjdn2SrZnK4v0a3pSPeqItB6YCQroA4Wf\\\\\\\/wV8sqBOS23NXpKOc5J1bKqbXtOsEaH+C5z7aS3P5z\\\\\\\/UxNCIIVCoCfMYfQZsfl\\\\\\\/s3ltlgXbr\\\\\\\/hyefM5SDEiAXevXPMaJIIrfsdTUxQMIjYrUSBNyMXqrC5HJfv6s4zG16RU4CrPzhZf1ePDMpFkyA8vlduJaPqwsBQWCTVeDrpR0SqVUK7QQeobitlcTYAl8zT\\\\\\\/lCy+BkCGdyW\\\\\\\/FWAH0rbr\\\\\\\/+uDhfntHba7iALqfeGOnsX0e1qk3V4MNsAH8bSQRbyUlK9OFwHz53ZWJ65eX5aqJOEP6TVfmCQbwTu8iiJrU1f4KEOG2EH3w54h2bKpMSaY8+uqPo9cyLIICWaD+\\\\\\\/9+T8c97ZNPq25N667iiW8DpW2alGzxtUpg8hc65rzF+ohwt+FUvn2CKiqgxmjq2+vbUtZD32nqZU981+ue\\\\\\\/CVqqpN7znXWWuU4jN3z\\\\\\\/lwNRZNcSPSd0iMhslEx6iPT\\\\\\\/yOsg94XxGmbitOvaQT0cCwys1Yy3WRi5pV+6cxjQaWiw+gpFhtngygtHvrhgoBqDVJ8GCRJ2J1F1xcYkQiwuszPkyIWB5MPkInbypdOqqI\\\\\\\/JORz6kVBmkbcJ28kplF3bk45coiniU01L0GoUpO5tYjLpcL+BtCgg4HF14QmWx8AGjXuDhu\\\\\\\/t6vdXKsfy0Yr18gkJ\\\\\\\/OkxhqN\\\\\\\/MaS9w9m1\\\\\\\/uCbYHk1S1N5BbDE+rZYyexKY3r1IQJAVjo9gXyE9B5sU8qQ1YMxtG6T4cDQ7Met1S3aO8uhopkH3X8nCOojf4vd19tJw9leYASLruWbrN8Ovw3+TfXQjEd8+goiLAaH6IEiS6jskMo2Rq2CCX2NOw\\\\\\\/kDOEoT76ylCkJGmyh7WI15+DuRaI3RAkbm9Dx8\\\\\\\/OwjQAt2X0WH3b46Nli4g8f8bpT1NtEPssUX3fP3UcTAcxblqV8WnBUh5IT5d5LkhYFrDyW3ZXm\\\\\\\/EcTptkmC\\\\\\\/4ExTszW+j9j5yO1J6CTGGjUELRbobhLLeJ9vppUpqWF67kIK9ONYXH3RJuG2CZyKRVbwlyGoodmEqpzj3u5He9mH\\\\\\\/1sNlAuRo9FrCrD9IoblNzOperSjlLaNz2JXAyT4eyOcqMZ6KlMTSX56D\\\\\\\/KOmqHNb0g1fCiHwlGleD8+RmLcbvzb+1M8EZpBtYom61ibtFeTMX1w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022h7gEUDNWimLzdefYVVUBgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221662739cf19898f1dbb23ad9827f646ee1890142fe686dd432f0e6067ca758f1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459349 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8Ye0nZ0-0I111qygxqbsBKrTUkxcCcaTcBkNzMGdUNk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Aw6QrecOCLa8_02WiIgbyvDfSgMeRoFrqla5cHmRP2M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1415" title="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1437" title="Identification of Gene Expression Changes Associated With Long-Term Memory of Courtship Rejection in Drosophila Males" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459349" id="node459349" data-pisa="ggg;2/11/1427" data-pisa-master="ggg;2/11/1427" data-apath="/ggg/2/11/1427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="1">Gary N. Atlin</span>, <span class="highwire-citation-author" data-delta="2">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="3">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="5">Mark E. Sorrells</span>, <span class="highwire-citation-author" data-delta="6">Babu Raman</span>, <span class="highwire-citation-author" data-delta="7">Jill E. Cairns</span>, <span class="highwire-citation-author" data-delta="8">Amsal Tarekegne</span>, <span class="highwire-citation-author" data-delta="9">Kassa Semagn</span>, <span class="highwire-citation-author" data-delta="10">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="11">Pichet Grudloyma</span>, <span class="highwire-citation-author" data-delta="12">Frank Technow</span>, <span class="highwire-citation-author" data-delta="13">Christian Riedelsheimer</span> and <span class="highwire-citation-author" data-delta="14">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1427-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003699 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459349" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary N. Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BN.%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BN.%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Babu Raman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Babu%2BRaman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raman%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABabu%2BRaman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jill E. Cairns </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BE.%2BCairns%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cairns%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJill%2BE.%2BCairns%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Amsal Tarekegne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amsal%2BTarekegne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarekegne%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmsal%2BTarekegne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Pichet Grudloyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Nakhon Sawan Field Crops Research Center, Nakhon Sawan 60190, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pichet%2BGrudloyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grudloyma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APichet%2BGrudloyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Christian Riedelsheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BRiedelsheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riedelsheimer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BRiedelsheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/11/1427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459349" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459349" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459349" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/11/1427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459349" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/11/1427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459349" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genomic prediction is expected to considerably increase genetic gains by increasing selection intensity and accelerating the breeding cycle. In this study, marker effects estimated in 255 diverse maize (<em>Zea mays L.</em>) hybrids were used to predict grain yield, anthesis date, and anthesis-silking interval within the diversity panel and testcross progenies of 30 F<sub>2</sub>-derived lines from each of five populations. Although up to 25% of the genetic variance could be explained by cross validation within the diversity panel, the prediction of testcross performance of F<sub>2</sub>-derived lines using marker effects estimated in the diversity panel was on average zero. Hybrids in the diversity panel could be grouped into eight breeding populations differing in mean performance. When performance was predicted separately for each breeding population on the basis of marker effects estimated in the other populations, predictive ability was low (<em>i.e.</em>, 0.12 for grain yield). These results suggest that prediction resulted mostly from differences in mean performance of the breeding populations and less from the relationship between the training and validation sets or linkage disequilibrium with causal variants underlying the predicted traits. Potential uses for genomic prediction in maize hybrid breeding are discussed emphasizing the need of (1) a clear definition of the breeding scenario in which genomic prediction should be applied (<em>i.e.</em>, prediction among or within populations), (2) a detailed analysis of the population structure before performing cross validation, and (3) larger training sets with strong genetic relationship to the validation set.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li><li class="kwd"><a href="/keyword/grain-yield" class="hw-term hw-article-keyword hw-article-keyword-grain-yield" rel="nofollow">grain yield</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li></ul><p id="p-4">In a hybrid maize breeding program, numerous crosses between inbred lines and testers need to be evaluated in extensive field trials to identify hybrids with greater yield potential in the target environment. Most crosses are discarded after field evaluation due to low general performance. To save resources, it would be advantageous to select inbred lines with high general combining ability by the use of molecular markers, because line performance <em>per se</em> is a poor predictor of hybrid performance (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Melchinger <em>et al.</em> 1998</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hallauer <em>et al.</em> 2010</a>). Although a large number of quantitative trait loci (QTL) have been identified, the impact of marker-assisted selection for improving maize hybrid performance in low- and high-yielding environments has been marginal (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tuberosa <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Araus <em>et al.</em> 2008</a>). This is primarily attributed to the small effects of the detected QTL and the fact that many detected QTL are specific to a particular genetic background. Genomic prediction provides an alternative method to use genomic information in breeding decisions. Rather than using only significant marker-trait associations to build up the prediction model, genomic prediction uses all markers simultaneously. The resulting genomic estimated breeding value (GEBV) is the sum of all marker effects (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Meuwissen <em>et al.</em> 2001</a>). After successful implementation of genomic prediction of breeding values of Holstein and Jersey dairy cattle (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009a</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goddard and Hayes 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2010</a>) and genetic risk of human diseases (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2008</a>), it is now beginning to be used in plant breeding programs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lorenzana and Bernardo 2009</a>).</p><p id="p-5">Using genomic prediction, simulation studies and initial experimental results indicate that grain or biomass yield of maize hybrids can be predicted with high accuracy utilizing one of several different prediction models (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">GonzÃ¡lez-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhao <em>et al.</em> 2012a</a>). This suggests that rapid increases in rates of genetic gain are possible because prediction accuracy of GEBVs is linearly related to the response to selection. Ideally, a training set composed of genetically diverse individuals, such as different animal breeds (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009a</a>), would be used for prediction. This would reduce the cost of implementing genomic prediction in breeding programs considerably as the training set would have wide applicability. Nevertheless, more validation experiments are necessary to investigate whether published high prediction accuracies can be applied with as much success in populations different from those in which the marker effects were estimated (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Goddard and Hayes 2007</a>). Prediction accuracy of genotypes originating from different populations may be lower than reported in previous studies using genotypes originating from the same population, particularly, if (1) the sample size of the training set is small, (2) broad-sense heritability (<em>H</em>) of the trait of interest is low, (3) information from close relatives is not available (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Saatchi <em>et al.</em> 2011</a>), and/or (4) linkage phases between single-nucleotide polymorphism (SNP) markers and QTL change in sign as suggested for heterotic pools that evolved separately over a long time (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charcosset and Essioux 1994</a>).</p><p id="p-6">The accuracy of genomic prediction is estimated by the correlation between the true breeding value and the GEBV. To date, prediction accuracy has been estimated by evaluating training and validation sets in single and/or the same set of environments. Multienvironment models can benefit from genetic correlations between environments (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">BurgueÃ±o <em>et al.</em> 2012</a>). However, it is unknown whether marker effects estimated in a set of environments are predictive of genotype performance in a different set of environments. Furthermore, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> (2012)</a> and <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Saatchi <em>et al.</em> (2011)</a> indicated that population structure might affect prediction accuracies. If the genotype set can be subdivided into several clusters or breeding populations that differ in performance level, the correlation between the true breeding value and the GEBV is likely, in part, to be driven by these differences as was also reported for marker assisted selection (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kang <em>et al.</em> 2008</a>) and genomic prediction (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Saatchi <em>et al.</em> 2011</a>).</p><p id="p-7">The objectives of this study were to (1) investigate the effects of sample size and number of test environments on prediction accuracy and to evaluate the prediction accuracies in a diversity panel of maize single crosses with the training and validation set drawn from either the same or different environments; (2) examine the prospects for genomic prediction based on testcross data from a diversity panel with a given tester to predict the performance of testcross progeny from segregating biparental populations derived from crosses of lines included or not included in the training set in combination with a different tester in different environments; (3) evaluate prediction accuracy in the presence of population structure; and (4) discuss potential uses for genomic prediction in maize hybrid breeding.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotypes and experimental design</h3><p id="p-8">The study used data from two experiments. In Experiment 1, a set of 255 diverse maize inbred lines was used. To summarize in brief, lines were selected to represent the genetic diversity across drought, low-N, soil acidity, and pest and disease resistance breeding programs of the International Maize and Wheat Improvement Center (CIMMYT) and the International Institute of Tropical Agriculture (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wen <em>et al.</em> 2011</a>). The lines could be grouped into eight breeding populations based on pedigree information, environmental adaptation, and main breeding target (F. San Vicente, personal communication): lines from the regional CIMMYT breeding program in Zimbabwe (n = 36), from the CIMMYT acid soil tolerance breeding program in Colombia (n = 24), from the CIMMYT insect resistance breeding program (n = 39), from the CIMMYT physiology breeding populations selected for drought tolerance, including the drought tolerant population white (DTPW C9, n= 17) and yellow (DTPY, n = 15) as well as the La Posta SequÃ­a breeding population (n = 39), and from CIMMYTâs subtropical (n =37) and tropical breeding programs (n = 38) in Mexico. For the remaining 10 genotypes, no information on the breeding origin was available. Lines were separated into early- and late-flowering maturity groups and crossed with tester CML312. In total, six trials were conducted in 2010 to 2011 in Mexico and Thailand for both maturity groups.</p><p id="p-9">Experiment 2 comprised five biparental F<sub>2</sub> populations generated using nine parental lines, four of which were part of Experiment 1. The other five parental lines were distantly related to the lines comprising Experiment 1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1/003699SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1/FigureS1.pdf">Figure S1</a>). One hundred fifty test cross progenies were generated by crossing 30 F<sub>2</sub>-derived lines from each cross with tester CML395/CML444 and evaluated in four trials conducted in 2011 in Zimbabwe and Kenya.</p><p id="p-10">All trials were conducted using alpha-lattice designs with two replicates in the dry season under well-watered conditions. Hybrids were evaluated for grain yield, anthesis date, and anthesis-silking interval. Grain yield was recorded in t/ha and adjusted to 12.5% moisture content. Anthesis date was recorded in days after sowing when 50% of plants within a plot shed pollen. Anthesis-silking interval was estimated as the number of days between silking and anthesis date.</p></div><div id="sec-3" class="subsection"><h3>SNP genotyping and marker selection</h3><p id="p-11">All 255 inbred lines in Experiment 1 and 30 F<sub>2</sub>-derived lines per population in Experiment 2 (n = 150) were genotyped with the MaizeSNP50 Bead Chip from Illumina, Inc. SNP markers were preprocessed according to the following criteria: (1) less than 5% missing values, and (2) minor allele frequency greater than 5% to exclude SNPs with a high rate of genotyping error and low frequency. A total of 37,403 SNPs met these criteria in Experiment 1 and were subsequently used for validation within Experiment 1. Across Experiment 1 and 2, 18,695 SNP markers were in common after SNP preprocessing. This set of markers was used for validation between Experiments 1 and 2.</p></div><div id="sec-4" class="subsection"><h3>Statistical analysis</h3><div id="sec-5" class="subsection"><h4>Variance components and heritability:</h4><p id="p-12">Variance components were estimated treating all effects as random effects. Two genotypes were in common across maturity groups.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-1.gif"/></noscript></span>, <span class="disp-formula-label">[1]</span></span>where <em>Y</em> is the mean performance of a certain genotype, <em>Î¼</em> is the overall mean, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-2.gif"/></noscript></span></span> the effect of genotype <em>i</em>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-3.gif"/></noscript></span></span> the effect of the environment <em>j</em>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-4.gif"/></noscript></span></span> the interaction between genotype <em>i</em> and environment <em>j</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-5.gif"/></noscript></span></span> the effect of the maturity group <em>k</em> nested in environment <em>j</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-6.gif"/></noscript></span></span> the effect of replicate <em>l</em> nested within maturity group <em>k</em> and environment <em>j</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-7.gif"/></noscript></span></span> the effect of block <em>m</em> nested within replicate <em>l</em>, maturity group <em>k</em> and environment <em>j</em>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-8.gif"/></noscript></span></span> the residual associated with a single plot. The genetic variance among and within breeding populations and clusters (Qst) was estimated by partitioning the genotype effect in model [1] into the effect of the group (breeding population or cluster) and that of the genotype nested within the group. The environment was defined as the year-site combination in which the trials were conducted. It should be noted that individual trials were treated as random samples from the target environment as the purpose of hybrid testing was to predict future performance in farmersâ fields.</p><p id="p-13">Broad-sense heritability (<em>H</em>) was estimated across <em>e</em> environments and <em>r</em> replicates (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hallauer <em>et al.</em> 2010</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-9.gif"/></noscript></span>, <span class="disp-formula-label">[2]</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-10.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-11.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/embed/mml-math-12.gif"/></noscript></span></span> are the genetic, genotype-by-environment, and residual variance components, respectively. <em>H</em> was estimated for means over all environments (e = 6) as well as in pairs of e = 4 and e = 2 environments.</p><p id="p-14">On the basis of best linear unbiased estimation, hybrid means were derived in each set of environments (e = 6, 4 or 2) applying model [1] treating the genotype main effects as fixed and all other effects as random.</p></div><div id="sec-6" class="subsection"><h4>Genetic relationship between lines:</h4><p id="p-15">The genetic relationship matrix was estimated by applying method 1 reported by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">VanRaden (2008)</a>. The resulting estimate was divided by two to obtain the kinship among lines. Mean kinship within breeding populations was estimated across all off-diagonal elements. Lines were grouped by specifying the desired number of clusters to n = 5, 10, and 15 using the complete linkage method (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sorensen 1948</a>). Furthermore, the molecular variance among and within breeding populations and clusters (Fst) was assessed applying an analysis of molecular variance.</p><p id="p-16">We investigated the linkage disequilibrium (LD) structure in the largest three breeding populations (<em>i.e.</em>, La Posta SequÃ­a, Zimbabwe, and Entomology) by fitting second-order natural smoothing splines onto the scatter plot of LD <em>vs.</em> the physical distances between markers on the same chromosome. Only markers with a marker allele frequency &gt;0.05 within the respective breeding population were considered for computing the LD. Furthermore, we investigated the persistence of linkage phases across the three breeding populations following <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Technow <em>et al.</em> (2012)</a>. Here, only markers with an allele frequency &gt;0.05 within both breeding populations in the comparison were considered.</p></div><div id="sec-7" class="subsection"><h4>Genomic prediction:</h4><p id="p-17">Hybrid performance was predicted for grain yield, anthesis date, and anthesis-silking interval using ridge regression best linear unbiased prediction (rrBLUP). BLUPs of allelic effects were estimated by assuming that all effects have the same prior distribution and shrinking them toward zero by the same magnitude (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Whittaker <em>et al.</em> 2000</a>). We define predictive ability [r(Å·,g)] as the Pearson correlation between the phenotype and the GEBV. The prediction accuracy [r(Ä,g)] was estimated as the correlation between the true breeding value and the GEBV, obtained by dividing the predictive ability in each run by the square root of <em>H</em> of the target trait evaluated in the respective set of environments (e = 6, 4, or 2). Different validation (V) procedures were used to evaluate the effect of different factors on genomic prediction for hybrid performance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1/FigureS2.pdf">Figure S2</a>):</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-18">(V1) Effect of sample size and number of test environments: Fivefold cross validation was conducted by subdividing the 255 hybrids of Experiment 1 randomly into five disjoint subsets. One subset was left out for validation whereas the other four subsets were used as training set. This procedure was replicated 20 times, yielding in total 100 runs. Marker effects were estimated in the training set to predict the performance of the validation set evaluated in the same set of environments. The sample size of the training set was varied (n = 204, 156, or 108) as well as the number of environments in which the training and validations set were evaluated (e = 6, 4, or 2).</p></li><li id="list-item-2"><p id="p-19">(V2) Effect of evaluating training and validation sets across different environments: Marker effects were estimated in the training set evaluated in four environments to predict performance of the validation set evaluated in two different environments applying a fivefold cross-validation as described in V1.</p></li><li id="list-item-3"><p id="p-20">(V3) Effect of evaluating training and validation sets with low degree of relationship across different environments, using a different tester: Performance of hybrids generated by crossing 30 F<sub>2</sub>-derived lines with a different tester (Experiment 2) was predicted using marker effects estimated in 255 hybrids (Experiment 1) evaluated in different environments.</p></li><li id="list-item-4"><p id="p-21">(V4) Effect of ânoâ relationship between training and validation set: Performance of one half of the genotypes in one focal breeding population or cluster was predicted based on marker effects estimated in the remaining breeding populations or clusters. This procedure was replicated 20 times. In each replication a different set of genotypes were placed into the two halves of the focal breeding population or cluster.</p></li><li id="list-item-5"><p id="p-22">(V5) Effect of including relationship between training and validation set: Performance of one half of the genotypes in a focal breeding population or cluster was predicted based on marker effects estimated from a combination of the remaining breeding populations or clusters and the other half of the genotypes in the focal group. This procedure was repeated 20 times as described in V4.</p></li><li id="list-item-6"><p id="p-23">(V6) Prediction based on group means, without the use of markers effects: In each V1 run, the mean of each breeding population or cluster in the training set was used to predict the performance of the genotypes in the validation set. The group mean was estimated across all genotypes of each breeding population and was as such independent of the mean performance of the validation set.</p></li></ul><p id="p-24">All analyses were performed using the R software version 2.12.2. For estimation of variance components and hybrid means, the <em>ASREML</em> package version 3 was used (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Butler <em>et al.</em> 2009</a>). Breeding values were predicted using the <em>rrBLUP</em> package version 2 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Endelman 2011</a>).</p></div></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Variance components and heritability</h3><p id="p-25">Mean grain yield of hybrids was 6.88 t/ha in Experiment 1 and 7.02 t/ha in Experiment 2 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Mean anthesis date was 71 days after flowering. The early and late maturity group differed in mean anthesis date by 2.6 days (data not shown). The ratio between genotype-by-environment variance and the genetic variance ranged between 0.48 and 1.21, with the greatest values observed for grain yield. <em>H</em> across trials was moderate to high for all traits evaluated in Experiments 1 and 2 (0.61-0.85). Within breeding populations, it ranged between 0.34 and 0.84 for grain yield, 0.32 and 0.90 for anthesis date, and 0.31 and 0.71 for anthesis-silking interval (data not shown).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462048/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462048/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mean and standard error of grain yield anthesis date, and anthesis-silking interval, their variance components and broad-sense heritability estimated for 255 hybrids evaluated in six environments (Experiment1) and for 150 testcross progenies of 30 F<sub>2</sub>-derived lines from each population evaluated in 4 environments (Experiment 2)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genetic relationship and LD</h3><p id="p-27">Mean kinship within breeding populations of Experiment 1 was between 0.10 and 0.16 for the Colombia acid soil tolerant, La Posta SequÃ­a, DTPW C9, and DTPY C9 breeding populations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For lines derived from the Entomology and Zimbabwe breeding populations, mean kinship was 0.05 and 0.09, respectively. Mean kinship was lowest for the Mexico subtropical and Mexico tropical breeding population. Generally, the relationship within a specific breeding population was greater than among breeding populations. This was particularly true for La Posta SequÃ­a, which had a low kinship to all other breeding populations, as also reported in a previous study using the same genotype set (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Wen <em>et al.</em> 2011</a>). LD decayed rapidly with physical distance between markers (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Furthermore, LD was greater within La Posta SequÃ­a than within the Zimbabwe and Entomology breeding population. The proportion of identical linkage phases across breeding populations was considerably lower than 1 and quickly declined to values close to 0.5 with increasing marker distance.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of the kinship matrix of 255 lines assigned to 8 breeding populations (Experiment 1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121060862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heat map of the kinship matrix of 255 lines assigned to 8 breeding populations (Experiment 1).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1427/F1.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1427/F1.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462041" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1â</span> <p id="p-28" class="first-child">Heat map of the kinship matrix of 255 lines assigned to 8 breeding populations (Experiment 1).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Second-order smoothing spline fits of LD (r2) vs. the distance in mega base pairs (Mbp) between markers on the same chromosome, within the La Posta Sequ&#xED;a (1), Zimbabwe (2), and Entomology (3) breeding population. (B) Second-order smoothing spline fits of proportion of marker pairs with equal linkage phase vs. the distance in marker base pairs between markers on the same chromosome. The horizontal line indicates a linkage phase of 0.5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1121060862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Second-order smoothing spline fits of LD (r&lt;sup&gt;2&lt;/sup&gt;) &lt;em&gt;vs.&lt;/em&gt; the distance in mega base pairs (Mbp) between markers on the same chromosome, within the La Posta Sequ&#xED;a (1), Zimbabwe (2), and Entomology (3) breeding population. (B) Second-order smoothing spline fits of proportion of marker pairs with equal linkage phase &lt;em&gt;vs.&lt;/em&gt; the distance in marker base pairs between markers on the same chromosome. The horizontal line indicates a linkage phase of 0.5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/11/1427/F2.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/11/1427/F2.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/11/1427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2â</span> <p id="p-29" class="first-child">(A) Second-order smoothing spline fits of LD (r<sup>2</sup>) <em>vs.</em> the distance in mega base pairs (Mbp) between markers on the same chromosome, within the La Posta SequÃ­a (1), Zimbabwe (2), and Entomology (3) breeding population. (B) Second-order smoothing spline fits of proportion of marker pairs with equal linkage phase <em>vs.</em> the distance in marker base pairs between markers on the same chromosome. The horizontal line indicates a linkage phase of 0.5.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Effects of sample size, different environments, and tester on genomic prediction</h3><p id="p-30">When genotypes were randomly assigned to the training and validation sets and evaluated in the same environments, predictive ability ranged between 0.30 and 0.45 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, V1). Predictive ability declined slightly with decreasing number of environments but remained stable when the size of the training set was reduced from 204 to 108 genotypes. Prediction accuracy ranged between 0.43 and 0.50.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461985/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461985/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean and standard deviation of predictive ability [r(Å·,g)] and prediction accuracy [r(Ä,g)] of genomic prediction in Experiment 1 obtained with different number of genotypes (n) and environments (e) in which the training and/or validation set were evaluated</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Prediction accuracy of performance in two environments was between 0.47 and 0.49, when based on marker effects estimated in four environments including the two environments of the validation set (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, row 3 in V1). Predictive ability decreased by 0.10 (26%), 0.06 (14%), and 0.04 (9%) for grain yield, anthesis date and anthesis-silking interval, respectively, when the same set of environments were used to predict performance in two different environments (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, V2).</p><p id="p-33">Predictive ability for performance of 30 F<sub>2</sub>-derived lines per population (Experiment 2) was between â0.37 and 0.49 based on marker effects estimated in Experiment 1 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, V3). Average predictive ability across populations varied around zero.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461952/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461952/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Predictive ability for testcross progenies of 30 F<sub>2</sub>-derived lines from each population evaluated in environments (Experiment 2) using marker effects estimated from the 255 inbred lines and phenotypic data of their testcross progenies evaluated in environments (Experiment 1)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Genomic prediction among and within breeding populations and clusters</h3><p id="p-35">In Experiment 1, predictive ability for performance in a specific group (breeding population or cluster) using marker effects estimated in the other groups, ranged between 0.12 to 0.21 for grain yield â0.01 to 0.23 for anthesis date and â0.03 to 0.02 for anthesis-silking interval with high standard deviations (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>, V4). Predictive ability decreased when increasing the number of clusters from 5 to 10 to 15 but was lowest when grouping the genotypes into breeding populations. When 50% of the genotypes in the validation set were included in the training set (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, V5), predictive ability increased for all traits. This increase was greater for anthesis date and anthesis-silking interval than for grain yield.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461946/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461946/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Predictive ability for grain yield, anthesis date, and anthesis-silking interval under validation schemes V4-V6 in Experiment 1</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">Breeding populations differed considerably in their mean performance. The difference between the least- and greatest-yielding population was large (1.15 t/ha, <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1/TableS1.pdf">Table S1</a>) whereas the standard error of means was only between 0.01 and 0.04 (data not shown). Breeding population La Posta SequÃ­a was high yielding, late flowering, and had a shorter anthesis-silking interval (<em>e.g.</em>, better flowering synchrony) relative to the other breeding populations. Cross validation methods V1 and V2 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) partitioned lines from different breeding populations into both the training and validation sets, such that some of the predictive ability was driven by the difference in mean performance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1/FigureS3.pdf">Figure S3</a>). When the mean of each breeding population in the training set was used to predict performance of the genotypes in the validation set (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, V6), predictive abilities were similar to or even greater than in V1, which used markers to predict performance. Even when the genotype set was divided into 15 clusters, genotypes of different breeding populations were placed into the same cluster. This implied that validation in each cluster was conducted across different breeding population means which led to higher predictive ability than when predicting the performance of each breeding population separately.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462071/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462071/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Minimum and maximum of grain yield, anthesis date, anthesis-silking interval, and the genetic and molecular variance among (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/T5/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/T5/embed/mml-math-19.gif"/></noscript></span></span>) and within (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/T5/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/11/1427/T5/embed/mml-math-20.gif"/></noscript></span></span>) clusters or breeding populations in Experiment 1</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">Analysis of genetic variance revealed that dividing the genotype set by breeding populations maximized variance among populations while minimizing variance within populations (Qst; <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). For grain yield, the variance among breeding populations explained 26% of the genetic variance while the variance among 15 clusters explained only 16% of the genetic variance. This difference was not observed when estimating the molecular variance (Fst). Here, no matter how many clusters or breeding populations were used to group lines, the variance among groups explained about 10% of the molecular variance.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Genomic prediction of performance within a diversity panel and testcross progenies of F<sub>2</sub>-derived lines</h3><p id="p-39">Within the diversity panel of Experiment 1, the performance of untested genotypes could be predicted, explaining up to 25% of the genetic variance by randomly assigning genotypes to the training and validation set. Much greater prediction accuracies have been reported in previous studies in diversity panels (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> 2012</a>) and segregating populations (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Zhao <em>et al.</em> 2012a</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">b</a>). Regarding the fact that resources need to be allocated to phenotyping and/or genotyping, we examined the effect of the sample size and the number of test environments on prediction accuracy under validation scheme V1 (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Contrary to theoretical expectations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">SchÃ¶n <em>et al.</em> 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Daetwyler <em>et al.</em> 2007</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goddard and Hayes 2009</a>), prediction accuracy remained almost constant when reducing the sample size from 204 to 108 and the number of test environments from six to two, which suggests that besides LD and relatedness, other factors, <em>i.e.</em>, population structure, contributed to the high prediction accuracy values under validation scheme V1.</p><p id="p-40">By using the diversity panel in Experiment 1 as training set and the F<sub>2</sub>-derived lines of five crosses in Experiment 2 as a validation set, we examined a situation commonly encountered in breeding, where the environments and the tester used in the training set differ from those in the validation set, and where the lines to be predicted have limited relationship with the training set. The predictive abilities observed in validation scheme V3 were disappointing because they varied around zero even for crosses of lines included in the training set (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). According to theoretical results (A. E. Melchinger, unpublished data), the prediction accuracy expected when changing from tester T<sub>1</sub> in the training set to tester T<sub>2</sub> in the validation set is obtained as the product of the prediction accuracy with the same tester in the training and validation set and the genetic correlation between the testcross performance of the lines with the two testers. Using the same tester in Experiment 1, predictive ability estimates obtained under validation schemes V1 and V2 were similar using four environments for the training set and two common or different environments for the validation set. Thus, the different environments could not explain the drop in predictive ability observed under V3. Estimates of genetic correlation among two testers were reported to range between 0.6 and 0.9 for grain yield (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 1991</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Melchinger <em>et al.</em> 1998</a>). The genetic correlation among the two testers used in the current study is probably of the same order of magnitude but could not be estimated because no testcross data were available with common genotypes. The extent to which line-by-tester interactions contribute to low predictive ability warrants further research.</p></div><div id="sec-15" class="subsection"><h3>Implications of hidden or apparent population structure on genomic prediction</h3><p id="p-41">In segregating maize populations (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zhao <em>et al.</em> 2012b</a>) and different full-sib families in mice (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Legarra <em>et al.</em> 2008</a>), prediction accuracies were low when the training and validation set comprised genotypes from different crosses or families. Similar to those studies, we investigated whether part of the drop in predictive ability observed under V3 relative to V1 is attributable to population structure. Based on breedersâ information, the 255 lines included in Experiment 1 originated from eight different breeding populations. Mean kinship among breeding populations was low (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), especially for La Posta SequÃ­a, where LD was higher than within the Zimbabwe and Entomology breeding populations. Differences in LD levels between breeding populations hamper the transferability of marker effects from one breeding population to another, even when the linkage phases are identical. The proportion of identical linkage phases across breeding populations quickly declined with increasing physical distance between markers to values close to 0.5 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Because of differences in LD and linkage phases, marker effects estimated in one breeding population cannot easily be transferred to another, and this at least partly explains the low accuracies observed within breeding populations using marker effects estimated in the other breeding populations (V4 and V5). Interestingly, between two distinct heterotic pools of maize (flint and dent) used for hybrid breeding in Europe, linkage phases decreased less, to a minimum of about 0.6, even though a minimum of 0.5 could have been expected given the long separation of the two pools (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Technow <em>et al.</em> 2012</a>). The steeper decrease of linkage phases with physical distance between markers in the current study may relate to the smaller sample sizes but also to the fact that the lines in Experiment 1 were developed from rather broad based populations by pedigree breeding accompanied by selection for <em>per se</em> and testcross performance with emphasis on different adaptive traits (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Wen <em>et al.</em> 2011</a>).</p><p id="p-42">Partitioning of the genetic variance across the testcrosses into the variance among and within breeding populations revealed that the former explained 26% of the variance for grain yield (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). This was also reflected by the large difference in the population means of 1.15 t/ha. Reduced genetic distance among lines originating from the same breeding population as compared to those from different breeding populations also was reflected by the heat map of kinship values based on SNP data (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Interestingly, in the analysis of molecular variance, the proportion of variance among populations in the total molecular variance was much smaller compared with the subdivision based on the genetic variance of the agronomic traits. Furthermore, the ratio between genetic variance among and within populations was almost three times greater when estimated based on phenotypic data (Qst) than based on marker data (Fst). This finding suggests that SNPs do not fully capture the differences among the lines from different breeding populations. Possibly, selection by breeders results in greater differences at the phenotypic level than reflected by genome-wide markers (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Porcher <em>et al.</em> 2004</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pujol <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Whitlock and Guillaume 2009</a>), an observation that warrants further research.</p><p id="p-43">To further investigate the effects of population structure on predictive ability under validation scheme V1, we grouped lines into different numbers of clusters based on the relationship matrix. Including information from relatives into the training set improved within-group prediction substantially for simple traits like anthesis date and anthesis-silking interval, but less so for grain yield. In all instances, predictive ability values including genetic relationship between training and validations sets (V5) were considerably lower compared with V1. Interestingly, when predictions for the lines were solely based on the means of the respective breeding population (V6), we achieved similar or even higher prediction accuracies than with the high-density, SNP-based genomic prediction in V1. Consequently, prediction accuracy across breeding populations resulted mostly from differences in mean performance and less from the relationship between the training and validation set or linkage phases between breeding populations, as also reported in cattle (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2010</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Saatchi <em>et al.</em> 2011</a>). The implications of this result depend on whether previous knowledge of population structure is available and whether one is interested in predicting performance within or among breeding populations. This will be discussed in detail in the next section, <em>Potential uses for genomic prediction in maize hybrid development</em>.</p></div><div id="sec-16" class="subsection"><h3>Potential uses for genomic prediction in maize hybrid development</h3><p id="p-44">Before incorporating genomic prediction in a plant breeding program, one has to clearly define the breeding scenario in which genomic prediction will be applied. The following scenarios may be differentiated:</p><div id="sec-17" class="subsection"><h4>Training and validation set comprise lines from a diversity panel:</h4><p id="p-45">One application of genomic prediction is the performance prediction of new lines in a pedigree breeding program from a large, diverse training set of lines with a low average coparentage with the lines under selection. GEBV accuracy in such populations would result from exploiting LD between high-density markers and QTL controlling the trait. To be effective, this strategy will likely require much larger training sets and denser marker maps than methods depending on close relationships. Simulations for a full sib family indicate that at least 1000 genotypes are required to achieve a prediction accuracy of approximately 0.75 with <em>H</em> of the trait of 0.5 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hayes <em>et al.</em> 2009b</a>). Nevertheless, it has to be regarded that greater prediction accuracies are likely to be achieved if the training set is large and includes lines related to the validation set (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2010</a>). In six-row barley, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lorenz <em>et al.</em> (2012)</a> found little-to-no increase in prediction accuracy when combining distantly related breeding populations to increase the size of the training population. The importance of genetic relationship between training and validation set is discussed in further detail in breeding scenario C.</p><p id="p-46">Prediction accuracy depends on the prediction problem that the breeder is attempting to address. If the goal is to predict within a population that comprises groups of related genotypes with differences in mean performance, results of this study indicate that this can lead to false conclusions regarding the prospects of genomic prediction within groups, which is likely to be the most common application. Prediction accuracy determined with validation scheme V1 in the presence of different groups with different performance levels would only be helpful to breeders if no information on those groups is available, <em>i.e.</em>, at the very beginning in breeding for a specific trait like biogas production (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Riedelsheimer <em>et al.</em> 2012</a>). If no reduction in accuracy is found by reducing the sample size in the training set, this can be taken as an indication for the presence of hidden population structure. In this case, genotyping could be applied to identify groups of related lines. Subsequently, phenotyping a representative sample of lines from each group would be sufficient to determine differences in the performance level of the different groups. If groups are present, it is recommended to take this into account in the validation scheme. Further research is needed on the effect of the number of distinct populations <em>vs.</em> the number of lines needed to achieve reliable prediction, as our results show that predictions based on small, highly structured training sets will not achieve useful accuracy. <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">BurgueÃ±o <em>et al.</em> (2012)</a> showed that for correlated environments, some of the benefits in predictive accuracy come from borrowing information from correlated environments and from using information regarding pedigree and genetic markers. These results indicate that the impact of environmental structure in combination with population structure on prediction accuracy should be considered.</p></div><div id="sec-18" class="subsection"><h4>Training and validation set are segregating progenies from the same cross:</h4><p id="p-47">One application of genomic prediction already used in commercial maize breeding (A. Gordillo, personal communication) is the prediction of performance of double haploid lines which have not been phenotyped, on the basis of a training set derived from the same cross. Similar within bi-parental family predictions were originally envisioned by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bernardo and Yu (2007)</a>. This approach would be similar to training and validation within each of the five crosses of Experiment 2, which could not be assessed in the current study due to the low sample size for each population (n = 30). Because multilocation phenotyping is more expensive than one-time genotyping, this approach would allow breeders to generate large full-sib families of doubled haploid lines (<em>i.e.</em>, n = 200), phenotype only a small fraction of lines, but large enough to provide reasonably accurate GEBVs (<em>e.g.</em>, n = 50), and advance both the best of the phenotyped and unphenotyped full sibs to the next testing stage, based on phenotype and GEBV, respectively. GEBVs are likely to provide moderate accuracy for this application because of the close relationship between the training and validation set and high LD within full-sib families even at low marker density and small population sizes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wong and Bernardo 2008</a>).</p></div><div id="sec-19" class="subsection"><h4>Training and validation set include related and unrelated genotypes:</h4><p id="p-48">As illustrated by the comparison of predictive ability under validation schemes V4 and V5 including genotypes from the same group in the training set helps to improve predictive ability in the validation set. In maize (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> 2011</a>), cattle (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de Roos <em>et al.</em> 2009</a>) and sheep (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2012</a>), it was reported that when the cross-validation scheme allowed for a high degree of relatedness, prediction accuracy increased by 0.26, 0.12, and 0.09, respectively, relative to that achieved across distantly related families. This increase depends on the degree of relatedness between the groups and also whether the LD between markers and QTL is stable across different groups. The latter will depend on the marker density and the breeding history of the groups. If the groups trace back to different races of maize and have been kept separate for a long time and selected with emphasis on different traits, chances are high that LD between adjacent markers is low even with a high marker density. This is similar to the situation in animal breeding, where marker effects estimated in Holstein dairy cattle did not predict accurately GEBVs of Jersey dairy cattle, and <em>vice versa</em> (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hayes <em>et al.</em> 2009a</a>). An open question in this context is how many groups should be included and how many individuals per group are required to obtain high predictive ability in validation schemes V4 and V5.</p></div><div id="sec-20" class="subsection"><h4>Recurrent selection with closed synthetic populations of key inbreds:</h4><p id="p-49">Another potential application of genomic prediction is rapid-cycle, marker-based recurrent selection in closed populations, like in La Posta SequÃ­a but with a sample size &gt;100, that will serve as sources of inbred lines. The objectives of such a recurrent selection program are to generate an improved population by increasing the frequency of favorable alleles while maintaining sufficient genetic variation for subsequent cycles of selection. One cycle of phenotypic recurrent selection consists of (1) the development of progenies from a population, (2) phenotypic evaluation of the progenies, and (3) selection and recombination of the best selected individuals to form a new population that will form the base material for the next cycle. Genomic prediction would be implemented by genotyping and phenotyping individuals in step (2) and estimating marker effects to predict hybrid performance in the subsequent recurrent cycles and recombine the best lines based on GEBVs alone. Phenotyping would only be used to re-estimate marker effects by evaluating the phenotype of selected parental lines each third recurrent cycle, thus substantially reducing both monetary and time costs associated with phenotyping (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heffner <em>et al.</em>, 2009</a>). If these populations were derived from a limited number of parents, high LD between markers and QTL alleles should persist for several cycles of selection, allowing increased genetic gain through acceleration of the breeding cycle with selection based on GEBV alone.</p></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank all collaborators involved in conducting the trials. Also, we thank H. F. Utz and T. Schrag for their help during the statistical analysis. This study was supported in part by the Rockefeller Foundation, the Bill and Melinda Gates Foundation through the drought-tolerant maize for Africa (DTMA) and water-efficient maize for Africa (WEMA) projects, and the Bundesministerium fÃ¼r wirtschaftliche Zusammenarbeit und Entwicklung (BMZ) 08.78603-001.00 research project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003699/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-51"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-07-10"><span class="received-label">Received </span>July 10, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-09-14"><span class="accepted-label">Accepted </span>September 14, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2012 Windhausen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.1" data-doi="10.1007/s00122-011-1587-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genome-based prediction of testcross values in maize</span>. <abbr class="cit-jnl-abbrev">TAG</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">339</span>â<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%26rft.volume%253D123%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1587-7%26rft_id%253Dinfo%253Apmid%252F21505832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1587-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505832&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.2" data-doi="10.1080/07352680802467736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slafer</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serret</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Breeding for yield potential and stress adaptation in cereals</span>. <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">377</span>â<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D27%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352680802467736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352680802467736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">Correlation between testcross performance of lines at early and late selfing generations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">17</span>â<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D82%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1991FT42100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.4" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>â<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.5" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">BurgueÃ±o</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">G. D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genomic prediction of breeding values when modeling genotype Ã environment interaction using pedigree and dense molecular markers</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>â<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-book" id="cit-2.11.1427.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-source">ASReml-R reference Manual.</span> <span class="cit-publ-name">Department of Primary Industries and Fisheries</span>, <span class="cit-publ-loc">Brisbane, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Essioux</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">The effect of population structure on the relationship between heterosis and heterozygosity at marker loci</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">336</span>â<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D89%26rft.spage%253D336%26rft_id%253Dinfo%253Apmid%252F24177851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24177851&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.8" data-doi="10.1186/1297-9686-44-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBorner%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimising%2Bmultistage%2Bdairy%2Bcattle%2Bbreeding%2Bschemes%2Bincluding%2Bgenomic%2Bselection%2Busing%2Bdecorrelated%2Bor%2Boptimum%2Bselection%2Bindices.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-1%26rft_id%253Dinfo%253Apmid%252F22252172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22252172&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.9" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">BurgueÃ±o</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>â<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D713%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.118521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20813882&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282807400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.10" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Genomic selection and prediction in plant breeding</span>. <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>â<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Inbreeding in genome-wide selection</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">369</span>â<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D369%26rft.epage%253D376%26rft.atitle%253DInbreeding%2Bin%2Bgenome-wide%2Bselection.%26rft_id%253Dinfo%253Apmid%252F18076474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18076474&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.12" data-doi="10.1371/journal.pone.0003395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Accuracy of predicting the genetic risk of disease using a genome-wide approach</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3395%26rft.epage%253De3395%26rft.atitle%253DAccuracy%2Bof%2Bpredicting%2Bthe%2Bgenetic%2Brisk%2Bof%2Bdisease%2Busing%2Ba%2Bgenome-wide%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003395%26rft_id%253Dinfo%253Apmid%252F18852893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852893&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.13" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>â<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.101501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19293140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.14" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Reliability of genomic predictions across multiple populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>â<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.15" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP</span>. <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>â<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.16" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genomic selection</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>â<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.17" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>â<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-other" id="cit-2.11.1427.18" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">GonzÃ¡lez-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks</span>. <span class="cit-source">Theoret. Appl. Genet.</span> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>â<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoret.%2BAppl.%2BGenet.%26rft.volume%253D125%26rft.spage%253D759%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.19" data-doi="10.1186/1297-9686-42-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tetens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seefried</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichtner</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaller</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The impact of genetic relationship information on genomic breeding values in German Holstein cattle</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLopez%2Bde%2BMaturana%26rft.auinit1%253DE.%26rft.volume%253D42%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DModeling%2Brelationships%2Bbetween%2Bcalving%2Btraits%253A%2Ba%2Bcomparison%2Bbetween%2Bstandard%2Band%2Brecursive%2Bmixed%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-1%26rft_id%253Dinfo%253Apmid%252F20100345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20100345&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-book" id="cit-2.11.1427.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carena</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda Filho</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-source">Quantitative Genetics in Maize Breeding.</span> <span class="cit-publ-name">Iowa State University Press</span>, <span class="cit-publ-loc">Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.21" data-doi="10.1186/1297-9686-41-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span>â<span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DH.%26rft.volume%253D41%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DGSE%2Bis%2Bnow%2Ban%2Bopen%2Baccess%2Bjournal%2Bpublished%2Bby%2BBioMed%2BCentral.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-1%26rft_id%253Dinfo%253Apmid%252F19284684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284684&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.22" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span>â<span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>â<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.23" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genomic selection for crop improvement</span>. <abbr class="cit-jnl-abbrev">Crop Science</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.24" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.080101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18385116&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254921600050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.25" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-GraniÃ©</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Performance of genomic selection in mice</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>â<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D180%26rft.spage%253D611%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.088575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18757934&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259758500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.26" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.27" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>â<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.28" data-doi="10.1007/s001220050767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gumber</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leipert</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuylsteke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuiper</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Prediction of testcross means and variances among F3 progenies of F1 crosses from testcross means and genetic distances of their parents in maize</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">503</span>â<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D96%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050767%26rft_id%253Dinfo%253Apmid%252F24710890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24710890&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>â<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8yLzExLzE0MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porcher</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giraud</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldringer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavigne</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Experimental demonstration of a causal relationship between heterogeneity of selection and genetic differentiation in quantitative traits</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1434</span>â<span class="cit-lpage">1445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D58%26rft.spage%253D1434%26rft_id%253Dinfo%253Apmid%252F15341147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15341147&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.31" data-doi="10.1111/j.1365-294X.2008.03958.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">R. I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pannell</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Are Q(ST)-F(ST) comparisons for natural populations meaningful?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4782</span>â<span class="cit-lpage">4785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPujol%26rft.auinit1%253DB.%26rft.volume%253D17%26rft.issue%253D22%26rft.spage%253D4782%26rft.epage%253D4785%26rft.atitle%253DAre%2BQ%2528ST%2529-F%2528ST%2529%2Bcomparisons%2Bfor%2Bnatural%2Bpopulations%2Bmeaningful%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03958.x%26rft_id%253Dinfo%253Apmid%252F19140971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03958.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19140971&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260735300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.32" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>â<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.33" data-doi="10.1186/1297-9686-43-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saatchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClure</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rolf</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Accuracies of genomic breeding values in American Angus beef cattle using K-means clustering for cross-validation</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DForni%26rft.auinit1%253DS.%26rft.volume%253D43%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DDifferent%2Bgenomic%2Brelationship%2Bmatrices%2Bfor%2Bsingle-step%2Banalysis%2Busing%2Bphenotypic%252C%2Bpedigree%2Band%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-1%26rft_id%253Dinfo%253Apmid%252F21208445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208445&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.34" data-doi="10.1534/genetics.167.1.485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SchÃ¶n</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Openshaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Quantitative trait locus mapping based on resampling in a vast maize testcross experiment and its relevance to quantitative genetics for complex traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">485</span>â<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchon%26rft.auinit1%253DC.%2BC.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D485%26rft.epage%253D498%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BMapping%2BBased%2Bon%2BResampling%2Bin%2Ba%2BVast%2BMaize%2BTestcross%2BExperiment%2Band%2BIts%2BRelevance%2Bto%2BQuantitative%2BGenetics%2Bfor%2BComplex%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.485%26rft_id%253Dinfo%253Apmid%252F15166171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvNDg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMi8xMS8xNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span>â<span class="cit-article-title">A method of establishing groups of equal amplitude in plant sociology based on similarity of species content and its application to analyses of the vegetation on Danish commons</span>. <abbr class="cit-jnl-abbrev">Biologiske Skripter</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiologiske%2BSkripter%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.36" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrag</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>â<span class="cit-lpage">1194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giuliani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanguineti</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Genome-wide approaches to investigate and improve maize response to drought</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">120</span>â<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D47%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.38" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>â<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.39" data-doi="10.2135/cropsci2010.08.0465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Molecular characterization of a diverse maize inbred line collection and its potential utilization for stress tolerance improvement</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2569</span>â<span class="cit-lpage">2581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2569%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.08.0465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.08.0465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.40" data-doi="10.1534/genetics.108.099812"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Testing for spatially divergent selection: comparing QST to FST</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1055</span>â<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D183%26rft.spage%253D1055%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.099812%26rft_id%253Dinfo%253Apmid%252F19687138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.108.099812&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19687138&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.41" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Marker-assisted selection using ridge regression</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>â<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.42" data-doi="10.1007/s00122-008-0715-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">815</span>â<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWong%26rft.auinit1%253DC.%2BK.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D815%26rft.epage%253D824%26rft.atitle%253DGenomewide%2Bselection%2Bin%2Boil%2Bpalm%253A%2Bincreasing%2Bselection%2Bgain%2Bper%2Bunit%2Btime%2Band%2Bcost%2Bwith%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0715-5%26rft_id%253Dinfo%253Apmid%252F18219476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0715-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18219476&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254262600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.43" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">WÃ¼rschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span>â<span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>â<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-2.11.1427.44" data-doi="10.1007/s00122-012-1862-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">WÃ¼rschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranc</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span>â<span class="cit-article-title">Impact of selective genotyping in the training population on accuracy and bias of genomic selection</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">707</span>â<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D707%26rft.epage%253D713%26rft.atitle%253DImpact%2Bof%2Bselective%2Bgenotyping%2Bin%2Bthe%2Btraining%2Bpopulation%2Bon%2Baccuracy%2Band%2Bbias%2Bof%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1862-2%26rft_id%253Dinfo%253Apmid%252F22481121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1862-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481121&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F11%2F1427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/11/1427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/11/1415" title="Large-Scale Screening for Targeted Knockouts in the Caenorhabditis elegans Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/11/1437" title="Identification of Gene Expression Changes Associated With Long-Term Memory of Courtship Rejection in Drosophila Males" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/11"><span>Volume 2 Issue 11, November 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/11.cover-source.jpg?itok=czJv9iCa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/11/1427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CQL__WSle-AQ57HLHbLElVTB6Pe9MwjYWpl5qWldbTI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/11/1427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/11/1427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/11/1427" class="active" data-icon-position="" data-hide-link-title="0">Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459349" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NVx_ZVDXsf-4TNG_5q-s0z11BHWnRdJqeFx6LP2y1ek" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/11/1427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-PN2hReF-N8XdMizkWBxcqMrsXyzQUDcA-V7c9RrLle0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/11/1427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/11/1427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459349" id="node459349--2" data-pisa="ggg;2/11/1427" data-pisa-master="ggg;2/11/1427" data-apath="/ggg/2/11/1427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="1">Gary N. Atlin</span>, <span class="highwire-citation-author" data-delta="2">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="3">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="5">Mark E. Sorrells</span>, <span class="highwire-citation-author" data-delta="6">Babu Raman</span>, <span class="highwire-citation-author" data-delta="7">Jill E. Cairns</span>, <span class="highwire-citation-author" data-delta="8">Amsal Tarekegne</span>, <span class="highwire-citation-author" data-delta="9">Kassa Semagn</span>, <span class="highwire-citation-author" data-delta="10">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="11">Pichet Grudloyma</span>, <span class="highwire-citation-author" data-delta="12">Frank Technow</span>, <span class="highwire-citation-author" data-delta="13">Christian Riedelsheimer</span> and <span class="highwire-citation-author" data-delta="14">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1427-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003699 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459349--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary N. Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BN.%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BN.%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Babu Raman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Babu%2BRaman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raman%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABabu%2BRaman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jill E. Cairns </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BE.%2BCairns%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cairns%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJill%2BE.%2BCairns%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Amsal Tarekegne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amsal%2BTarekegne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarekegne%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmsal%2BTarekegne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Pichet Grudloyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Nakhon Sawan Field Crops Research Center, Nakhon Sawan 60190, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pichet%2BGrudloyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grudloyma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APichet%2BGrudloyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Christian Riedelsheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BRiedelsheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riedelsheimer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BRiedelsheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/11/1427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/11/1427&amp;text=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/11/1427&amp;t=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/11/1427&amp;title=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459349/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459349" id="node459349--3" data-pisa="ggg;2/11/1427" data-pisa-master="ggg;2/11/1427" data-apath="/ggg/2/11/1427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="1">Gary N. Atlin</span>, <span class="highwire-citation-author" data-delta="2">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="3">Jose Crossa</span>, <span class="highwire-citation-author" data-delta="4">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="5">Mark E. Sorrells</span>, <span class="highwire-citation-author" data-delta="6">Babu Raman</span>, <span class="highwire-citation-author" data-delta="7">Jill E. Cairns</span>, <span class="highwire-citation-author" data-delta="8">Amsal Tarekegne</span>, <span class="highwire-citation-author" data-delta="9">Kassa Semagn</span>, <span class="highwire-citation-author" data-delta="10">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="11">Pichet Grudloyma</span>, <span class="highwire-citation-author" data-delta="12">Frank Technow</span>, <span class="highwire-citation-author" data-delta="13">Christian Riedelsheimer</span> and <span class="highwire-citation-author" data-delta="14">Albrecht E. Melchinger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1427-1436;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003699 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459349--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary N. Atlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BN.%2BAtlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Atlin%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BN.%2BAtlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark E. Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>United States Department of Agricultural Research Service and Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BE.%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20ME&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BE.%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Babu Raman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Babu%2BRaman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raman%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABabu%2BRaman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jill E. Cairns </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jill%2BE.%2BCairns%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cairns%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJill%2BE.%2BCairns%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Amsal Tarekegne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amsal%2BTarekegne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarekegne%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmsal%2BTarekegne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kassa Semagn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kassa%2BSemagn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Semagn%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKassa%2BSemagn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>International Maize and Wheat Improvement Center (CIMMYT), El Batan 56130, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Pichet Grudloyma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Nakhon Sawan Field Crops Research Center, Nakhon Sawan 60190, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pichet%2BGrudloyma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grudloyma%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APichet%2BGrudloyma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Frank Technow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frank%2BTechnow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Technow%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrank%2BTechnow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Christian Riedelsheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BRiedelsheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riedelsheimer%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BRiedelsheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Albrecht E. Melchinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim, 70599 Stuttgart, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Albrecht%2BE.%2BMelchinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melchinger%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlbrecht%2BE.%2BMelchinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:melchinger@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">melchinger@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1427&amp;rft.epage=1436&amp;rft.atitle=Effectiveness%20of%20Genomic%20Prediction%20of%20Maize%20Hybrid%20Performance%20in%20Different%20Breeding%20Populations%20and%20Environments&amp;rft.volume=2&amp;rft.issue=11&amp;rft.date=2012-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Windhausen%2C+Vanessa+S.&amp;rft.au=Atlin%2C+Gary+N.&amp;rft.au=Hickey%2C+John+M.&amp;rft.au=Crossa%2C+Jose&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Sorrells%2C+Mark+E.&amp;rft.au=Raman%2C+Babu&amp;rft.au=Cairns%2C+Jill+E.&amp;rft.au=Tarekegne%2C+Amsal&amp;rft.au=Semagn%2C+Kassa&amp;rft.au=Beyene%2C+Yoseph&amp;rft.au=Grudloyma%2C+Pichet&amp;rft.au=Technow%2C+Frank&amp;rft.au=Riedelsheimer%2C+Christian&amp;rft.au=Melchinger%2C+Albrecht+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459349/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459349/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459349/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459349/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459349/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459349/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459349/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459349/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459349/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459349/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459349/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459349/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype Ã Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/11/1427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/11/1427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/11/1427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/11/1427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/11/1427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
