<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4169" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4169.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4169.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4169.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4169/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4169.atom" />
<meta name="HW.pisa" content="ggg;9/12/4169" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" />
<meta name="DC.Identifier" content="10.1534/g3.119.400545" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Tourrette et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Meiotic recombination generates genetic diversity but in most species the number of crossovers per meiosis is limited. Previous modeling studies showed that increasing recombination can enhance response to selection. However, such studies did not assume a specific method of modifying recombination. Our objective was to test whether two methods used to increase recombination in plants could increase genetic gain in a population undergoing recurrent selection such as in genomic selection programs. The first method, in Oryza sativa , used a mutant of anti-crossover genes, increasing global recombination without affecting the recombination landscape shape. The second one used the ploidy level of a cross between Brassica rapa and Brassica napus , increasing recombination especially in pericentromeric regions. Our modeling framework used these recombination landscapes and sampled quantitative trait loci positions from the actual gene distributions. We simulated selection programs with initially a cross between two inbred lines, for two species. Increased recombination enhanced the response to selection. The amount of enhancement in the cumulative gain largely depended on the species and the number of quantitative trait loci (2, 10, 20, 50, 200 or 1000 per chromosome). Genetic gains were increased up to 30% after 20 generations. Furthermore, increasing recombination in cold regions was the most effective: the gain was larger by 25% with the first method and 34% with the second one in B. rapa , and 12% compared to 16% in O. sativa . In summary, increased recombination enhances the genetic gain in long-term selection programs, with visible effects after four to five generations." />
<meta name="DC.Contributor" content="Elise Tourrette" />
<meta name="DC.Contributor" content="Rex Bernardo" />
<meta name="DC.Contributor" content="Matthieu Falque" />
<meta name="DC.Contributor" content="Olivier C. Martin" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Meiotic recombination generates genetic diversity but in most species the number of crossovers per meiosis is limited. Previous modeling studies showed that increasing recombination can enhance response to selection. However, such studies did not assume a specific method of modifying recombination. Our objective was to test whether two methods used to increase recombination in plants could increase genetic gain in a population undergoing recurrent selection such as in genomic selection programs. The first method, in &lt;i&gt;Oryza sativa&lt;/i&gt;, used a mutant of anti-crossover genes, increasing global recombination without affecting the recombination landscape shape. The second one used the ploidy level of a cross between &lt;i&gt;Brassica rapa&lt;/i&gt; and &lt;i&gt;Brassica napus&lt;/i&gt;, increasing recombination especially in pericentromeric regions. Our modeling framework used these recombination landscapes and sampled quantitative trait loci positions from the actual gene distributions. We simulated selection programs with initially a cross between two inbred lines, for two species. Increased recombination enhanced the response to selection. The amount of enhancement in the cumulative gain largely depended on the species and the number of quantitative trait loci (2, 10, 20, 50, 200 or 1000 per chromosome). Genetic gains were increased up to 30% after 20 generations. Furthermore, increasing recombination in cold regions was the most effective: the gain was larger by 25% with the first method and 34% with the second one in &lt;i&gt;B. rapa&lt;/i&gt;, and 12% compared to 16% in &lt;i&gt;O. sativa&lt;/i&gt;. In summary, increased recombination enhances the genetic gain in long-term selection programs, with visible effects after four to five generations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4169" />
<meta name="citation_id" content="9/12/4169" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4169" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4169.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4169.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4169.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400545" />
<meta name="citation_pmid" content="31628152" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4169" />
<meta name="citation_lastpage" content="4181" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elise Tourrette" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0822-5330" />
<meta name="citation_author" content="Rex Bernardo" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3323-4690" />
<meta name="citation_author" content="Matthieu Falque" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6444-858X" />
<meta name="citation_author" content="Olivier C. Martin" />
<meta name="citation_author_institution" content="GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and" />
<meta name="citation_author_email" content="olivier.c.martin@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5295-5963" />
<meta name="citation_reference" content="citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Transplantation;citation_journal_abbrev=Transplantation;citation_author=D. Bailey;citation_title=Recombinant-inbred strains. An aid to finding identity, linkage, and function of histocompatibility and other genes.;citation_pages=325-327;citation_volume=11;citation_year=1971;citation_issue=3;citation_pmid=5558564;citation_doi=10.1097/00007890-197103000-00013" />
<meta name="citation_reference" content="citation_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_title=Why sex and recombination?;citation_pages=187-195;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Battagin;citation_author=G. Gorjanc;citation_author=A.-M. Faux;citation_author=S. E. Johnston;citation_author=J. M. Hickey;citation_title=Effect of manipulating recombination rates on response to selection in livestock breeding programs.;citation_pages=44;citation_volume=48;citation_year=2016;citation_pmid=27335010;citation_doi=10.1186/s12711-016-0221-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Bauer;citation_author=M. Falque;citation_author=H. Walter;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Intraspecific variation of recombination rate in maize.;citation_pages=R103;citation_volume=14;citation_year=2013;citation_pmid=24050704;citation_doi=10.1186/gb-2013-14-9-r103" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. Bernardo;citation_title=Should maize doubled haploids be induced among F(1) or F (2) plants?;citation_pages=255-262;citation_volume=119;citation_year=2009;citation_issue=2;citation_pmid=19396574;citation_doi=10.1007/s00122-009-1034-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=R. Bernardo;citation_title=Prospective targeted recombination and genetic gains for quantitative traits in maize.;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0118" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Blary;citation_author=E. Jenczewski;citation_title=Manipulation of crossover frequency and distribution for plant breeding.;citation_pages=575-592;citation_volume=132;citation_year=2019;citation_doi=10.1007/s00122-018-3240-1" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=S. P. Brandariz;citation_author=R. Bernardo;citation_title=Predicted Genetic Gains from Targeted Recombination in Elite Biparental Maize Populations.;citation_pages=180062;citation_volume=12;citation_year=2019;citation_doi=10.3835/plantgenome2018.08.0062" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Bulmer;citation_title=The effect of selection on genetic variability.;citation_pages=201-211;citation_volume=105;citation_year=1971;citation_doi=10.1086/282718" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Burt;citation_title=Perspective: sex, recombination, and the efficacy of selection--was Weismann right?;citation_pages=337-351;citation_volume=54;citation_year=2000;citation_issue=2;citation_pmid=10937212;citation_doi=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2" />
<meta name="citation_reference" content="Choulet, F., A. Alberti, S. Theil, N. Glover, V. Barbe et al., 2014 Structural and functional partitioning of bread wheat chromosome 3B. Science 345: 1249721." />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. M. Comeron;citation_author=A. Williford;citation_author=R. Kliman;citation_title=The Hill-Robertson effect: evolutionary consequences of weak selection and linkage in finite populations.;citation_pages=19-31;citation_volume=100;citation_year=2008;citation_issue=1;citation_pmid=17878920;citation_doi=10.1038/sj.hdy.6801059" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Crismani;citation_author=C. Girard;citation_author=N. Froger;citation_author=M. Pradillo;citation_author=J. L. Santos;citation_title=FANCM Limits Meiotic Crossovers;citation_pages=1588-1590;citation_volume=336;citation_year=2012;citation_issue=6088;citation_pmid=22723424;citation_doi=10.1126/science.1220381" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Felsenstein;citation_title=The evolutionary advantage of recombination.;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Fernandes;citation_author=M. Seguela-Arnaud;citation_author=C. Larchevêque;citation_author=A. Lloyd;citation_author=R. Mercier;citation_title=Unleashing meiotic crossovers in hybrid plants.;citation_volume=115;citation_year=2018;citation_pmid=29183972;citation_doi=10.1073/pnas.1713078114" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Girard;citation_author=L. Chelysheva;citation_author=S. Choinard;citation_author=N. Froger;citation_author=N. Macaisne;citation_title=AAA-ATPase FIDGETIN-LIKE 1 and helicase FANCM antagonize meiotic crossovers by distinct mechanisms.;citation_pages=e1005369;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1005369" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Giraut;citation_author=M. Falque;citation_author=J. Drouaud;citation_author=L. Pereira;citation_author=O. C. Martin;citation_title=Genome-wide crossover distribution in Arabidopsis thaliana meiosis reveals sex-specific patterns along chromosomes.;citation_pages=e1002354-e1002354;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072983;citation_doi=10.1371/journal.pgen.1002354" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. R. Goddard;citation_author=H. C. J. Godfray;citation_author=A. Burt;citation_title=Sex increases the efficacy of natural selection in experimental yeast populations.;citation_pages=636-640;citation_volume=434;citation_year=2005;citation_issue=7033;citation_pmid=15800622;citation_doi=10.1038/nature03405" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. Gonen;citation_author=M. Battagin;citation_author=S. E. Johnston;citation_author=G. Gorjanc;citation_author=J. M. Hickey;citation_title=The potential of shifting recombination hotspots to increase genetic gain in livestock breeding.;citation_pages=55;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0330-5" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S. Griffiths;citation_author=J. Simmonds;citation_author=M. Leverington;citation_author=Y. Wang;citation_author=L. Fish;citation_title=Meta-QTL analysis of the genetic control of crop height in elite European winter wheat germplasm.;citation_pages=159-171;citation_volume=29;citation_year=2012;citation_doi=10.1007/s11032-010-9534-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. Hayes;citation_author=M. E. Goddard;citation_title=The distribution of the effects of genes affecting quantitative traits in livestock.;citation_pages=209-229;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11403745;citation_doi=10.1051/gse:2001117" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Filler-Hayut;citation_author=C. M. Bessudo;citation_author=A. A. Levy;citation_title=Targeted recombination between homologous chromosomes for precise breeding in tomato.;citation_pages=15605;citation_volume=8;citation_year=2017;citation_doi=10.1038/ncomms15605" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Leflon;citation_author=L. Grandont;citation_author=F. Eber;citation_author=V. Huteau;citation_author=O. Coriton;citation_title=Crossovers get a boost in Brassica allotriploid and allotetraploid hybrids.;citation_volume=22;citation_year=2010;citation_pmid=20622148;citation_doi=10.1105/tpc.110.075986" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=C. Lister;citation_author=C. Dean;citation_title=Recombinant inbred lines for mapping RFLP and phenotypic markers in Arabidopsis thaliana.;citation_pages=745-750;citation_volume=4;citation_year=1993;citation_doi=10.1046/j.1365-313X.1993.04040745.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Lloyd;citation_author=C. Morgan;citation_author=F. C. H. Franklin;citation_author=K. Bomblies;citation_title=Plasticity of meiotic recombination rates in response to temperature in Arabidopsis.;citation_volume=208;citation_year=2018;citation_pmid=29496746;citation_doi=10.1534/genetics.117.300588" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. T. Lovell;citation_author=K. Grogan;citation_author=T. F. Sharbel;citation_author=J. K. McKay;citation_title=Mating system and environmental variation drive patterns of adaptation in Boechera spatifolia (Brassicaceae).;citation_pages=4486-4497;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12879" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. S. Mason;citation_author=M. Rousseau-Gueutin;citation_author=J. Morice;citation_author=P. E. Bayer;citation_author=N. Besharat;citation_title=Centromere locations in Brassica A and C genomes revealed through half-tetrad analysis.;citation_volume=202;citation_year=2016;citation_pmid=26614742;citation_doi=10.1534/genetics.115.183210" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. McClosky;citation_author=S. D. Tanksley;citation_title=The impact of recombination on short-term selection gain in plant breeding experiments.;citation_pages=2299-2312;citation_volume=126;citation_year=2013;citation_doi=10.1007/s00122-013-2136-3" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=C. McPhee;citation_author=A. Robertson;citation_title=The effect of suppressing crossing-over on the response to selection in Drosophila melanogaster.;citation_pages=1-16;citation_volume=16;citation_year=1970;citation_issue=1;citation_pmid=5488984;citation_doi=10.1017/S0016672300002238" />
<meta name="citation_reference" content="citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=R. Mercier;citation_author=C. Mézard;citation_author=E. Jenczewski;citation_author=N. Macaisne;citation_author=M. Grelon;citation_title=The molecular biology of meiosis in plants.;citation_volume=66;citation_year=2015;citation_pmid=25494464;citation_doi=10.1146/annurev-arplant-050213-035923" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=D. Mieulet;citation_author=G. Aubert;citation_author=C. Bres;citation_author=A. Klein;citation_author=G. Droc;citation_title=Unleashing meiotic crossovers in crops.;citation_pages=1010-1016;citation_volume=4;citation_year=2018;citation_doi=10.1038/s41477-018-0311-x" />
<meta name="citation_reference" content="citation_title=Rice Genomics, Genetics and Breeding;citation_title=Composition and Structure of Rice Centromeres and Telomeres;citation_pages=37-52;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=J. L. Modliszewski;citation_author=G. P. Copenhaver;citation_title=Meiotic recombination gets stressed out: CO frequency is plastic under pressure.;citation_pages=95-102;citation_volume=36;citation_year=2017;citation_doi=10.1016/j.pbi.2016.11.019" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=H. J. Muller;citation_title=The relation of recombination to mutational advance.;citation_pages=2-9;citation_volume=1;citation_year=1964;citation_doi=10.1016/0027-5107(64)90047-8" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. P. Otto;citation_author=N. H. Barton;citation_title=Selection for recombination in small populations.;citation_pages=1921-1931;citation_volume=55;citation_year=2001;citation_issue=10;citation_pmid=11761054;citation_doi=10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Pelé;citation_author=M. Falque;citation_author=G. Trotoux;citation_author=F. Eber;citation_author=S. Nègre;citation_title=Amplifying recombination genome-wide and reshaping crossover landscapes in Brassicas.;citation_pages=e1006794;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006794" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=D. Phillips;citation_author=G. Jenkins;citation_author=M. Macaulay;citation_author=C. Nibau;citation_author=J. Wnetrzak;citation_title=The effect of temperature on the male and female recombination landscape of barley.;citation_pages=421-429;citation_volume=208;citation_year=2015;citation_pmid=26255865;citation_doi=10.1111/nph.13548" />
<meta name="citation_reference" content="R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Ru;citation_author=R. Bernardo;citation_title=Targeted recombination to increase genetic gain in self-pollinated species.;citation_pages=289-300;citation_volume=132;citation_year=2019;citation_doi=10.1007/s00122-018-3216-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. A. Salomé;citation_author=K. Bomblies;citation_author=R. A. Laitinen;citation_author=L. Yant;citation_author=R. Mott;citation_title=Genetic architecture of flowering-time variation in Arabidopsis thaliana.;citation_volume=188;citation_year=2011;citation_pmid=21406681;citation_doi=10.1534/genetics.111.126607" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Séguéla-Arnaud;citation_author=W. Crismani;citation_author=C. Larchevêque;citation_author=J. Mazel;citation_author=N. Froger;citation_title=Multiple mechanisms limit meiotic crossovers: TOP3α and two BLM homologs antagonize crossovers in parallel to FANCM.;citation_volume=112;citation_year=2015;citation_pmid=25825745;citation_doi=10.1073/pnas.1423107112" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Serra;citation_author=C. Lambing;citation_author=C. H. Griffin;citation_author=S. D. Topp;citation_author=D. C. Nageswaran;citation_title=Massive crossover elevation via combination of HEI10 and recq4a recq4b during Arabidopsis meiosis.;citation_volume=115;citation_year=2018;citation_pmid=29463699;citation_doi=10.1073/pnas.1713071115" />
<meta name="citation_reference" content="citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=L. Suay;citation_author=D. Zhang;citation_author=F. Eber;citation_author=H. Jouy;citation_author=M. Lodé;citation_title=Crossover rate between homologous chromosomes and interference are regulated by the addition of specific unpaired chromosomes in Brassica.;citation_pages=645-656;citation_volume=201;citation_year=2014;citation_pmid=24117470;citation_doi=10.1111/nph.12534" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=P. B. Talbert;citation_author=S. Henikoff;citation_title=Centromeres convert but don't cross.;citation_pages=e1000326-e1000326;citation_volume=8;citation_year=2010;citation_issue=3;citation_pmid=20231873;citation_doi=10.1371/journal.pbio.1000326" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Technow;citation_author=C. Riedelsheimer;citation_author=T. A. Schrag;citation_author=A. E. Melchinger;citation_title=Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.;citation_pages=1181-1194;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22733443;citation_doi=10.1007/s00122-012-1905-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. van Ooijen;citation_title=Accuracy of mapping quantitative trait loci in autogamous species.;citation_pages=803-811;citation_volume=84;citation_year=1992;citation_pmid=24201478;citation_doi=10.1007/BF00227388" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. T. Webster;citation_author=L. D. Hurst;citation_title=Direct and indirect consequences of meiotic recombination: implications for genome evolution.;citation_pages=101-109;citation_volume=28;citation_year=2012;citation_issue=3;citation_pmid=22154475;citation_doi=10.1016/j.tig.2011.11.002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. Zeyl;citation_author=G. Bell;citation_title=The advantage of sex in evolving yeast populations.;citation_pages=465-468;citation_volume=388;citation_year=1997;citation_issue=6641;citation_pmid=9242403;citation_doi=10.1038/41312" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=P. A. Ziolkowski;citation_author=C. J. Underwood;citation_author=C. Lambing;citation_author=M. Martinez-Garcia;citation_author=E. J. Lawrence;citation_title=Natural variation and dosage of the HEI10 meiotic E3 ligase control Arabidopsis crossover recombination.;citation_volume=31;citation_year=2017;citation_pmid=28223312;citation_doi=10.1101/gad.295501.116" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Meiotic recombination generates genetic diversity but in most species the number of crossovers per meiosis is limited. Previous modeling studies showed that increasing recombination can enhance response to selection. However, such studies did not assume a specific method of modifying recombination. Our objective was to test whether two methods used to increase recombination in plants could increase genetic gain in a population undergoing recurrent selection such as in genomic selection programs. The first method, in Oryza sativa , used a mutant of anti-crossover genes, increasing global recombination without affecting the recombination landscape shape. The second one used the ploidy level of a cross between Brassica rapa and Brassica napus , increasing recombination especially in pericentromeric regions. Our modeling framework used these recombination landscapes and sampled quantitative trait loci positions from the actual gene distributions. We simulated selection programs with initially a cross between two inbred lines, for two species. Increased recombination enhanced the response to selection. The amount of enhancement in the cumulative gain largely depended on the species and the number of quantitative trait loci (2, 10, 20, 50, 200 or 1000 per chromosome). Genetic gains were increased up to 30% after 20 generations. Furthermore, increasing recombination in cold regions was the most effective: the gain was larger by 25% with the first method and 34% with the second one in B. rapa , and 12% compared to 16% in O. sativa . In summary, increased recombination enhances the genetic gain in long-term selection programs, with visible effects after four to five generations." />
<meta name="og-title" property="og:title" content="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4169" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Meiotic recombination generates genetic diversity but in most species the number of crossovers per meiosis is limited. Previous modeling studies showed that increasing recombination can enhance response to selection. However, such studies did not assume a specific method of modifying recombination. Our objective was to test whether two methods used to increase recombination in plants could increase genetic gain in a population undergoing recurrent selection such as in genomic selection programs. The first method, in Oryza sativa , used a mutant of anti-crossover genes, increasing global recombination without affecting the recombination landscape shape. The second one used the ploidy level of a cross between Brassica rapa and Brassica napus , increasing recombination especially in pericentromeric regions. Our modeling framework used these recombination landscapes and sampled quantitative trait loci positions from the actual gene distributions. We simulated selection programs with initially a cross between two inbred lines, for two species. Increased recombination enhanced the response to selection. The amount of enhancement in the cumulative gain largely depended on the species and the number of quantitative trait loci (2, 10, 20, 50, 200 or 1000 per chromosome). Genetic gains were increased up to 30% after 20 generations. Furthermore, increasing recombination in cold regions was the most effective: the gain was larger by 25% with the first method and 34% with the second one in B. rapa , and 12% compared to 16% in O. sativa . In summary, increased recombination enhances the genetic gain in long-term selection programs, with visible effects after four to five generations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nLEIZtOI5d1eSatKVFH9S8V4H8iR5h3IQrmNkzWb6VU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494989","apath":"\/ggg\/9\/12\/4169.atom","pisa":"ggg;9\/12\/4169","ac":{"\/ggg\/9\/12\/4169.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4169.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4169"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4169"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4169":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pw0WtwiynAk0oRzQ7WtKJ\\\\\\\/BcvR6XjTW1KhiWn71xPSVXou3XNdqrB3NIDQluHEOOwRuCoichzt0ivuqPQ82yKu6tEC6+srjxTscFbQBEC2AWRAiVjwYf5ZW\\\\\\\/iGGj82nbb1iTR+kDRMwXk329Q20WO3588MWM7Za6n48oSg7hF9wHVwAYfx7N+COH0I4PiHGrk2uFK7IeNox41Q25aUcx8ZofBbtGsaPXQYc+e50xXEjB2pKOm+QRetwzw0PVkEAZ9EP8ahPNtsMHx4umpay1mvgMC3xly\\\\\\\/\\\\\\\/PSaOu3Pmy2MBkr1xp0eVNrlgLp6AuST4JGOXPrVkczDIgGPVZSXhhrMNb8R3GwdFP7VDNTN2N0OKX5IJg8XOtu6I4TMHQSG0Z8Spmj6cPpvt6\\\\\\\/iOxi+ruCndK7R3YHhQBrsPgnGaAW\\\\\\\/Iatc6uunaxVeoq9ToGM4CDG+NcfIrgCOcVNtj+OtTaeA\\\\\\\/Zf3UuQ3M+mc8AnmlkqZk+iHG7sGxCiWqtF1J7sIIsFuFcUAzl+WzyR\\\\\\\/2f+y0OnIcsH35gHYVsAR3PI7eaBtT1lrd9D\\\\\\\/u+krWFlDJZpXPTgyVnitQlPHvOIjPKN9XnZ0H5vmRhzKK0\\\\\\\/NECd2elzwZz5wZYQpfAK84bu9n+Mv6owidnjj5VXpIOcU\\\\\\\/y4D8omB6YJywmsc61naX5rFK5cF08Kg8biR1p49vve0PEusXAvXKaSiEHHL1Os\\\\\\\/2oz6BwmgpL\\\\\\\/io+HRAwicYz\\\\\\\/XxNF4MdT9TUsJgpG7FnDltZ9FfSRR+\\\\\\\/ZR7GzRxi32EV\\\\\\\/2pcEVLsfGKEy61K1PLcLn4LyQ8zfYuBzXY9G7ZEYUR3QJCgdHMFKa+nuMnfVv2Uy3foXxCGLUFsmYsGZV8+UU\\\\\\\/SL\\\\\\\/K3cUeJevP0ThjyyuQ\\\\\\\/ssuX\\\\\\\/8yGYUMNAacWLhQcjvBG4GeRUA9IfsobHC0J78qduvjWh5clAevbr7+qtoJJwK6AcfFHLrHLST7b9Zi83JsLQJ+TJdWazNBK\\\\\\\/zfhicpQ2DvE6h96nNDrkcF+C7ugc5KKhXukcrrbCQ9IMxUvWmbKavvrhLqRfe6E6+t+f5qxAifkzRh63RdtrNzID0VEUEzpUUvTk+4d923EVSiJUs5FeVM5rwzntIWc5IMdEWtinU\\\\\\\/wt9\\\\\\\/b9zXEWvVgOev+MK8DnS72dkUcqArt4P8YK+4iplvRwXayAC5Rk2TGYqO2LHI3sOVQIXo1HPVE0FYCwoQj\\\\\\\/TRdoUKdqGNOx2Bv\\\\\\\/8vWPY4XB0bBFvE6\\\\\\\/DXFMSDOpC8RvpAz8lA93hsWuqVmu5\\\\\\\/I\\\u0022,\\\u0022iv\\\u0022:\\\u0022miVvoPnmxTV1V2Tz5tuw6A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224f110ed68ffcd4c6ef23644a0e6ff810ea659a418d544e2fc3e5132247c85d74\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bRZKwCNtayvz9CpojKuVtlOWPhhC3dB7eqa+rAmqv36DxsSKPIvXld7GnK5zKYzgg1fA+2tMzS\\\\\\\/0Hlt2WxyJLVv1pvY2sWRyD8lIJo4O0V+D\\\\\\\/b8Myne8z1ezRYeyecEPsxEBpE63fWk\\\\\\\/SUOZqYi49+v\\\\\\\/M2l52WcUV1jYiMauU5DoHnCmWVI4Cc2MaUR5l8zqT+K29ZId6AZ\\\\\\\/ClLfR55ocg+8wpmR8FooLlxkOeg2ktimWo2hCJJpphEJ4X8LXYrEuUQ5pnb8liJUEubYIufWq83mha94PApsVBaErQiccu767o4pqbzLLqmt9Gcp5r86rNmxcpECuu02QHkaW0TiQKvNpu5A8p6o1csrCZOdj14jtm8lhZekXU4aOx4Xja2xIgHganB8nfq9EpuZC0vcVxT1AgNRLyk3QgmEt3AQ6D17gn30vM1Ez8PKs00F1cUdFX9mOr2cREZAGKZ+d7doH3Y5rbhWA99iWvrapyVcrJ9uoQ8zJRUO8Qm4R4kMWEJV5quqJfM3WUlPm1xLWaIWsW52HhvLMeWdOBotHIwjZHPWbcG3YT5U7xBU9AaaQiSygfe3nsCJ1sSmR\\\\\\\/xWyPFLTXy4g2TSgIhfwcOuErljzS6nnbDg4pWSp4aiHUemou2iNlDUxw7o4LF1WwB6WhYGEU+adAHbEi7hXOsMZnbqcPBMps9JwGPoBXiMDqBqZaZOXldG8cvvXScLRf0m2ozDqNECvSrnXpn97ABXjqpx7gUlUYOLM5N+SBCUFuePrbhuhjAlawZEgujjFTUEMl2j3SYyh9uUMbmrlttFkTsqbg35Wyy9lgdU66JTy+Q80Wh4EgfaI6fUQz\\\\\\\/24A8HYoRRsuG90MzjcVPy0F7ucWF0vGhEh4pnrPjKkjVOQKylypSiCIxQoTgwqzm0nBgIullW6hkA4YS6IKBQDbnS1zBcnWNl3byj\\\\\\\/5VZO2Ek\\\\\\\/Mt5k7ZTxdB\\\\\\\/t5AFcK2nKx8BtFe7\\\\\\\/HjMnwK7FHBFB93X27zuiPaDRKonCpoiY0iQ+9vR5XjXYpFiCG\\\\\\\/f3YJNPsxpAKe1y+7t+2BAZP2ZwPlhUWdgL3gVvCgR4GvWWaYSjPp2GiTvz3NQ\\\\\\\/3osSCCaYoNM3ac86xsvJ9Q2Vd0JbDaEpsnZKO6aBJSI86nQqk+BuAWf6EoxaSmS\\\\\\\/cduT5gNJjj3rKrDmFjGLwdH4CpAQhVOVFI0TbW4xtWUtCPHnvcls1PN0k5WVRCICjdC4enp2ZCfTOCH3h0payElhPM0Ajk1ve6nJt70r5rpjBF1XPJ80Iyk\\\\\\\/DQ7cfo\\\\\\\/PuokdCADckaEXQTfPJIV996RXOWmKeO3Ruwbeeb5tO94MC\\\\\\\/yRJrYPNWgQXTe8NU5PJqX4jbISu4E\\\\\\\/kfeES0Ve5\\\\\\\/69FEtaN9TsOWUNuDMSYLc6CfnfxDzlUGyvcJR8HP0FKHoPnHqutcU6mi+mcuEyKMr2WmcGbF00lPernVSIDr1TbUqfwJmNuHIGcFSso2KAYmN5pZf8v9R9ajI+wssEcYAt9c88MIXwgH5lieb851uUexC4xGxYnWPm9pOj0zg6oixguv+qHbP1sZJbtHL9zHA9D50YGQ0nn1uL\\\\\\\/aRWcA1G8HKrxrGvUFpdRvc2lU4XYFUz9gfuZRplf1LriuYjUtXaKOoQoVZHLCH2O\\\\\\\/WuMvCN64Lia1p4dd0VVKwmVVU53JmT7DSTiXvrAtfXK5A\\\\\\\/0+jMsB1lkdENQrso5ilPDzwY2e\\\\\\\/lsglggOmU2MydlIzyDW\\\\\\\/0ocDxWedItXtWBg19SRG6OAsCeHq\\\\\\\/uqW8ieUvqZuZ3m\\\\\\\/hc6CHbNzMVWDBGVx3bR5cODYLO11L3QLmnSIWntn2zS0QQZ09NQBQNRgU\\\\\\\/HuipLyWO8nGIFyM3WMLrJvYzJ8WiMZXKL77E9POCtd0r7440wtyrbUWoB+A1OYW\\\\\\\/TIvGSAxALuC1RDHQe0\\\\\\\/bMbflFMstU0ReY97wnBUYEryL9p+PzsMTI4BRIPx1By1x9B9DnMj1kLuswXTIXSbuljXb4DMkbpEE5rBs8hDyxiHTiw292Y1Pt1hOKmfqsp8\\\\\\\/4ERVZMFjW4e9HR6oHZgkkq6akXqsORs9LpIjNG7pF5+DNyfmeihageq+q1e1r8iJlqFBNw04Me4kcJxL5D0xCiLoCkVnClkEDhRj+xk1JrzZp\\\\\\\/JJeMHy\\\\\\\/khZWc\\\\\\\/7auj0nat8WSb6GOHMboD8QGI3siFb1PzrPtXFvc1PcnzcoNaPQslaQGzZk+4yquBuTKNScE+FXBBfi2CPfCh2dEHhTdPvDcDk3RAB9GXe+BUphWqFLFzgaoniZN7PYjngZKXHFdJwc9JzfTt8V+MrmVHeloz0xveO+GykToJpTIfpUzb\\\\\\\/I\\\\\\\/Epp1U0cyLBAO94bfq5Xpe03sp1twKdIrrCnJ2363Z7AiO3p5buiiGpBjdQuQuAyGJOGA7qRcMjiefR6IN7ur5hfbgCazJNyVzEMUFxMCxHOmJjgXE1vUanRgiMaFTjg+QGiKHskGvL7r4HGNRPrbx3RP0Mii6vVy7BsgFO9I0aow7pp6Sc1Fz1A\\\\\\\/21o1FX+LDwOLGXXTkLPJNu+9shSdevZF0M5ombAqrQAybwrrogrR2SVBCddihWy\\\\\\\/VrH1\\\\\\\/CV8m0xilmnplH5sHGsmtPdP4wL2NHB\\\\\\\/quBSkEgIDALFh\\\\\\\/vuVYXYRfqks2kIXI1l0Jyi6lbolf\\\\\\\/qGaWDICp9aqLcjGcnd9ooHdQGHTt3PdAPBfOnleTeuG7oIZDbBiclB+ajbXkEAMq5ZZo+0uqPYSSWFuJ46+pctTJpMwhey31e2tPeiaQk0R13QH2S9EkuAye3pzTOpHOqoL0EpMGRlqzBY0Q\\\\\\\/jowZ7hBz1qYy7wqx7Iy8Y5Y9yDhT51lQoIBSOp3RNQMJFMQK8SJrPthhOLae4c9UsAmrAMeo08w+g8M5kZE8jP7dwfN\\\\\\\/Gd63mTMS9NZvqh0Xi3aFdBtGE619O\\\\\\\/v5daNuQiwYFO801zns4w8UN64v\\\\\\\/X68sJ+W2jg9kq\\\\\\\/CVQT3UfxQEhkaLtSEF7xzlRN42\\\\\\\/bD4DW\\\\\\\/QIzILMGImiYch7zmgAuizb3tI29DM\\\\\\\/JwdtF8wbUJbuIL3wTtgp0+q6UWb\\\\\\\/M41N9wMxqnNtcdrg6+HRpQuuzI34q\\\\\\\/WxcJcm\\\\\\\/JidVX92qw7CqfLj5KNltZhMFv22V4zSfzsFFeIPA42XPeKa4\\\\\\\/VJ20x+F+ZCWlg1QIAaKzZK1Us4uPIXIuXR\\\\\\\/z8RL9F1Obz8yWHvuTwta5jrm3H+d0osOyTDYofXPIlz6PtqT\\\\\\\/bDpz56dRmci9xLpjYbWGtfnvl3J3toLZQXP23GRuddPa7BQZJEn4b\\\\\\\/CGDt2LhExx1fxvkVY4QBrs0WJQHu2rKc6eOIOM7XTtCU5I0Uz3Ed1BWeoFWTngutmymfXtCWk2hwULJt21LLfUxoyA90y\\\\\\\/iNWvzhGPw3SdTOaZ\\\\\\\/YL\\\\\\\/FIjKzUcAjUt++38BrBA+WcM537noovwLO96AY6dpK+PaxAX1rmvxAbUIfw9ou5jqXKgAHL85fMnjSNydADNZzoCNT2zUJlRYxRIG83rrA6oGzV8YQ8SOqk7pdepZb+hMvoccwIsmgsGUbt+fuJ7ASFMuJcvC3o0lS5EJcfpD4pGGxeSStXr0HHWQaQb5IWVQLfDiFW7rPzz9EgY6WnMvUIPpldMeNZ71oEKjWSbNGm8D\\\\\\\/t0UbRBymMnbcVxrVB2RlYmQe9Es\\\\\\\/8oW4YxAIGoFFHWdNDAXDqyFQ6X7OWAKLcZmnO+j6BhRJtQCLVNUti38m+ckEFJCHQoFFOlZ72AvzjJzwtf+nWosC1ApSo+c1dGlZ\\\\\\\/fwJ8aupQ0woW7c4llXwBC3lHhJAqk51JMdNI\\\\\\\/UtuW6O+EBmI\\\\\\\/cJmRRYtALH8o4poZRYcEePQ0eGVgmkCKQqYYgWcS2oCopa\\\\\\\/S5GEtFsYK9bXiyEOfTT5PosPPODWEgVyR9z7Vrl6KshrcektMkKR4Zuspm8mCCSV9k3rQzglMoUraJWgHtdRQXO2YQpJk\\\\\\\/0S00gdOkSgUZ25MHS8wIJ7F\\\\\\\/maELU7zqrOb95ybM6MQBjkSZAN1UMukqnqahriQrhCcfPVx9h9FZHLmrOvVfp36F4WB1OP0IU10h3t4T2SlqJMoqOXFUY5nl66EHy6a\\\\\\\/xRWIRPbFd79kX9k+y5JNeJQE3\\\\\\\/3AnD4rKfLvrxCdLjtNBqlBSZMcxcUdFcSaJ+4wvEYf\\\\\\\/aZF5sMRHSP1mfCS5UzhgN2lT9RCe1leq8a0\\\\\\\/it32tu3sQDXRAgwtaVuKopAIcpv2Yo6M7qZBFoifJ+bDPLR45eiufKMxZXjKRiTg7XVXTW77jdGQwmFYQ9yRCjfvy9Az9kDtLPA29C+OwZya0o1xUtBE32EGbMTSQoF3d8avTAsdFtXq2H7KtiORwfq94\\\\\\\/yNkzlfMBi2P\\\\\\\/aT9lhvYAtxkE+hMsyqVMh2rQrOcz2eGy\\\\\\\/JvNHDgBnJxaYk50K3KcBAp2rHhYdK5BcLdg5+NV\\\\\\\/E+U66WICSsCrTgv+LnDOZWLcNw9sOCmEk3NEMhoR655nVBVk7KP3x4DgusxDnxqhatT4s0v+MEey3gJ5nbx0WQcYRI1LZGcPJE+QHYTmoOcC2iXEIw4HUO1iuCXiIG+2kMIF\\\\\\\/OkXzlJuEQ5J9i\\\\\\\/3\\\\\\\/PPxCk7lyaflhRpn0WnxYbq\\\\\\\/Ff2DQvMk\\\\\\\/fz3PxzohNhRTsyeTI4WALiOs4vgKj4xGEgKRdV4mhmx73swHTH4Yxl2\\\\\\\/HLajyx1cqjMuBaUZUszHEz1UYAzrRygz1Zg34N4hFkoSmH0QTW+whje7mZ6sruipqwBajVtiZ5krV4cpoEVaDVZw7zeHUws3e1zttmuRsljM8x\\\\\\\/biLHDEzvtDr3c5957LWltCveDxhk38\\\\\\\/xUV96Lzz6Ev\\\\\\\/XC4X8bRj67eo3Jr6WPJti1lMrxuJ0IsOmb8nywMT3NFckhGneBOlTW6UNpDtdJu8dMlIlh4f7dts7mBuEPaBTEepnrP9U\\\\\\\/ixm\\\\\\\/MGRcUvK0OgXjN5q0bKDJ7+Xq9uGhjacxYObG7MoaYpe6ryatrvP+847NH0ouSMUw5ma8FD5HweWzFQILRRi7y5TukxCkTRe0AdKfZvXYX28r4A\\\\\\\/vxsmj9mnixH3\\\\\\\/tJ1UM+YhwUOZwXF6gxtLvQAx9xFMyadPZ1L\\\\\\\/5ZX5Hf815fka0fkywoRYAJFe\\\\\\\/ZtoeofnzNJG22koesudYBFIUYEbT4Dxxou0ss3VwoabR3J6aKF+tsDTiEXkQ8spHKllUe5vwyKwPry\\\\\\\/nVT27MPBHXmAiJAF70zldKmJjZsN96cXTrAC+gCvIiSax3uPvcCoYd4rsZ7DCrg\\\\\\\/2n3GVJlcfDoJzMRu1zZCEBjN8114MYTb4VOuUb7gaCTZLfoLvaYJlDyK\\\\\\\/v4KNkGkFvzkNn8rBZZAi2HuNgfNbteVe4IndL6w5wBGkJBIYCfoA3l5xwTZ+Qe9ZKjMr9UVBr8I48eJcjkV90jTLVFQne0CkDHSDI8ySGuA4za5tE0FFTDMUleFYooe3f7Q02Fte1aoV\\\\\\\/9cMeujuuHuTxLA7HuX6gbfclPdC3UW6eSktWuMIoncpUiCEN1HJHTjSTMHZk9Zx25+3Wl3BkN2\\\\\\\/89atS+dwr58b13zjriIqBHpFdDdjGJiO88pDnZZvwWPdoh0numIg0LqdLd3M4QiPjhw4MDcJVgXAwE+\\\\\\\/JoP8qHozv03uQ4BApujrjqjlnPljjF95NkLxZIKb3H9w\\\\\\\/0VxT1DW6BVnyH11k3\\\\\\\/ZnFmnUTYNkhlxBGFhu7LeHATemuYrpp1DypqCipiiPFFeVGsmQLT4DXKpC9bxqs9Rhwagdk5gMH1baiekCTKYluzSEOjRyPXqRaI7GbwuFH0l8BxDx9tRyykenYtK4w9NOG0JenCJ\\\\\\\/k8vnkaUsE1bIlNpsm1s2HsZ+FdWCAc+7c1e3\\\\\\\/BP3RUvS1ZolFLllBExM8\\\\\\\/LuyTqm13jBWe\\\\\\\/gU+Z1T1kKK+VGretpXnHBvR7KavOqyMXRwTfcdRTIeP9UJ+YMA\\\\\\\/0Kbf7fo3M++NFQRBB\\\\\\\/txx6LFS+vmpFU3Whhz2qC9MoFDoTEL4DxlEkQW8j9fRqp+XD0vJrJJNTCjauV\\\\\\\/FDq2jcIez6P9nIqx0EVeE05NKrRRVIwgCB1hYoz0\\\\\\\/wKbJT9P6\\\\\\\/day4qA2jPDE7LE6VgqkpcZNfYLX\\\\\\\/SeLLbqOEvVpQxKBcXV8bfyO4R14C4CObumGuTu21QrZ7vGNi3CGoJROi60kBUfS3KeLNjxjdNHlQp5tKMWoByl73wI2ZvWUMx1TjQeISIsKMD5z2Ufq4K4WKK+PASe1e+b\\\\\\\/5BzpzCZuZ5m1tsEO5S6mJgfoS1mgdNUCBCAKqZ0XnnNMycmWH5mvvOtkY7xnb4yxudEzP3uWfLtx2cnNfPVZKYhWLnqUA4so5nOYT+acO0zkUtwDPfOdn95ti+c0gdlTf1i0+Py5ILou5qRCiSkQpbuL8R+U7p\\\\\\\/fPbU\\\\\\\/PoudgQaKo9p6jvRUdJWRAKWvVTODZdgKN52rbXlT9XSIcQi\\\\\\\/A0yy4wtUA2ZscmhEs+euGbI9Lz50YdYkQdAGDuYfUStBEDd3r5zDMS+0LmWE2fXJEG4yM0QuaxOWLVd6DB+VhmkmBE\\\\\\\/5wDZ6BSgqreWFBUfzJ+sYk5FxR5+A3yJSRM1yIqmic\\\\\\\/KIARKvF0gsYzsk5e09D2\\\\\\\/PHCcNgtX89QwQDI2AB+Nr2OuAdxJaiXLlcpxBwPTD9G3cGRYFm6NaRIT5rKI26ASV74F2fn1PXnaFm+cY10FXZJb+biMwK60hsdG\\\\\\\/m8IAYpUyF\\\\\\\/6gkA6blTDodVpEorn20IgUWcqck0F1f24\\\\\\\/d\\\\\\\/lG3Pm84oNSk\\\\\\\/8DPHnsxIapkHLjdpcuupk39byZLqH4xSUdoL3Mg\\\\\\\/fTKCDdhLeXzWOXI1tEMG6X0+RUDl06x7BusU8p8DL1bMPyZard7MpI3R6T6ny7Ski4WWoyQHLqxl7Oc1wMlJhX4QO3quAcSMCBX3Li3x+Tzetuga4iLbPeBgTAsd19QFx798zimgjKtjHd1YuSwIMEanUq2V0nNqXMb9x6r3PxtukuEZIL38KJQPKbmD97M2MoOcv+0xJOQpXBNm82Von0HTFz5T2Uco+L+dd4jOu3voou6CfuyxUhxx74k\\\\\\\/Ez3YcQgAmCl5nE+DxlY29uNRkkX\\\\\\\/ZkvQLoszPXkNKLvgKXVnXiOh9UOzvaoy4iIGNxN8i4Ga1ODXO1luCKIJCn3RWwoQbrhmn29F4q4Do5JrMeeQlefzOmSPT1MTpuOD1FFEcg+cjYWSMtSI8DLvW9\\\\\\\/eGIl9h0axdxMtHrfXqsMQNsmCKgvjZeh2ZFKq\\\\\\\/sJe\\\\\\\/822vBKlzRzmZqXiZ09yqaAShg8LXTler+0emBJm1dbmHVbFCzDbrmXWn3kW5t53777HS5+SJ0e\\\\\\\/s9SzPiAPrSQfoI9ADUbY7VuQs0ZajOQMMQkKp\\\\\\\/2FU6SQYDqJOxVzjXu9III2xkHY7QeLC7qOS7OalI3NAfTVFdoyzW97nJAxy2tgKC16ElKy40yJ2xqVabYfIjan7Xz8+cfOxNZnfybP+8V6BxGrdUKL+nr+cIxOAJe6TCzPT\\\\\\\/aAJx7JyZNk+1uI8FMYv8r3nTxWAO6jA6y2QOmoFMmfQRsjkUuodmp1HFzpYbXKvRfSAT0ZpE691YuT8LBRSdYGrYWT23vwCGy4fhcYccIDnwTH8g1rohRdOxfDmVB+yvlZUCHo5\\\\\\\/DV+Eedcj2B7CQ3aCdIoilvG1\\\\\\\/n6tAS476FLEz4ELi6nOpDWhBHpmW9ADGTFASGIZMaxtCdtYfXe44gEep8h2J6T51olk5vHQJZGflecfkMTm\\\\\\\/XbQZtyA83xjfyX0\\\\\\\/LkZRrAQ94FzeQ1D+b8ji8OrWSFA8RMToIrCg1Nqw7QYUcx\\\\\\\/8LWtmMAPQg7hDL6OEwxpgA8Nv5HTRTjr3\\\\\\\/xDATkVXf\\\\\\\/PQmWMxJWKMAkf+Z4a4c1lgN2pvhtiKI0ABa17U06Aw8WVIMOfXor3ySBmowZoAUGHtoOtSvkCcmdSLMsWZDsVP+vJ6ns8qCPIzMxOT2K7dV2Gshi1OX7djHhb0x\\\\\\\/KB55a2bMiRm0bz6be6XUfzM5H1OgfTc5eSNuPwUxMdDQspvImydU55M88U39Olay6fAmzImfM5LEvgaRi0cyS7Eq1SfZHhYLzfiVKHFV+CtHm9\\\\\\\/7aaVdJ8RMRB3bB1joZ+OGTZ2uK1dgzRB4\\\\\\\/EW2JqBEexDMQjGT0wy9l4WRvI3fyu0uIo+RAOHC32wDHwiWLhwmXioEg6YBzZQsznHjXPi9t1T4kS+bPXAHLDFvkbjLOWVAps3xgCNPqU\\\\\\\/ybrhoMnEPspmqEb\\\\\\\/1h1uGmsKD0TV+43UJj+1oUU9+2mcGON4TuohOhh4artbj8nZf1qdgGOz2a0RQVq+xW\\\\\\\/7h3rUSAYZ3mG97Co3ERyq269geV5+BfD+epQXctDvKXpGlyCaWdeTgsnWYablyybWzr5+iJuaGDx7ZkyBazLEFADDyXLBYztvIhNMt2rkR5mU7NKJs4QRN0EDt\\\\\\\/+2YRg4RLsW10P1+hqaLSjrcUM4dHccSorSvGVkoWRPivV+2CL9GRPYHcHkuy5e0lMUwkzPe1c46fyraiv586Zb3LfdtlxcWM08pdB8kk1BKp6RyOGIlxnEeQNPRTDhD8xhzmcnzJSSzOh+6YBalNDfr6F5aq3dfhvDEIqHXR5rZ9eAme0RY8JQh8LvWKq+r6wKu8om1hXniODvcwMPHrpWrTLE5akcXWtBQEcSAKAFW\\\\\\\/goiorZszjGBJgOBdXrHrlmPHZGaz7sYphAgWeZnqtXoMMIBWGJsC1GYKMfeioIw3Ujs3RuB+QuFIS++9AUAFGJH5QJr1GBUrcjva4ZMvgQXr6Rh5Db+8supInydw\\\\\\\/oPb8V9i8OFfAjEXO98uAannM0RKcWiQKJhVI9n3xO2lzjZsnrxWXo7vvkcHGc9w6lIB5HCjstsukZcxEOhN1AZAB5BDLznSeoQUIkyYyB6RGGi9pE\\\\\\\/6gJBnu+zMIESYYJJ3YnnYxkmtk7t7UqrCS728SjdmKw5Yo5osYX7lAozCSTyGspw6m6ti0Op9vMmrv14o6I0tZEj6IGmByfwzSJhMlxguQ6uZoy1DcV22z+JyR5S+tE9dGy9FNvzkyVkA7UQjzKi3LshrAe5P97R5iCfd1t1oLDR6qPZy1MmICvjsjmtA5+jJsLkOmQ8OCjyWzFlDVLKK0\\\\\\\/AzthAjlQ7l6Da4me\\\\\\\/8knOl2mLPbSsGRrC1S2VVWAxmx2KLMlJgP2gKm6PsssZWheau7oXHBD7NlihbHEDhZ2czMiem9AnUEIQMmOt5oW2SxbA8y15EQjt2YFaXjLMBF9XCC4h85i5AoBWmaLKtmdExCNUSo4AzDMmCGpLTvuJUwAX+8DStBKX9wlXjwc4IPDCijkp\\\\\\\/iO+VYUzH7n+ZuLlnCDj24057l5g6ET53bCfxd6RTxtSJY0+3oF8RB3YUga1tVKhXEjk1WPMmrAQ906nor0GXNeH60OZsoJI18o5pLjeIYa7iH\\\\\\\/8iRCsuA4EDXRaFjDV2vFShug72+ClxWkg2mRhN\\\\\\\/mWoc2CTNr0qH+Zxsu7En4dWoynVAPJmEQqDRsvYWsTqd+hGb3Ya4WQcaNPkz2lc8DE1Zl2L8FGrlq0UWxywaTd24l5lOqADrjKy79VfgV\\\\\\\/qRQL6GKRXMkH5+1aZxFs1Tc9LYmb5UL3BqCRwPITZtUV13fC9wgazV1tx4pronBvsVGrtEct8I8Vj36\\\\\\\/RKm3kLOORmSKRTTDzCnyetZ9Aq2A4DibkAgS66VVopghXUrb9SVCmI0m7S+uabLpnAT2nJkagZ1C\\\\\\\/4g8jfXLcatqSVkRmBMhuMb98M+qHYX+DlwY\\\\\\\/lGSqMmFHWmw5x9sKNoEcfilqZBG3RwbAPCkjzYgAka\\\\\\\/Gesld3+J9c5J1hCclQmApFc+YnRUMT\\\\\\\/b1rBhmt0F5KjzLfRq5r\\\\\\\/s+WUPQudsKzERslZf3v0Wy+uRf6LpWBQ7hmKKVXxf1VK+B\\\\\\\/C9ayLmIVkoe9q5BvoQ9CJagVn37J79N0YUWxsEjYkCysGT\\\\\\\/E6cAw1xreyz4mD0Wl4ja+KEJxSMPpaHoXAGt73kkTDvgs63b303veCYOZhP864\\\\\\\/Uj530JSH8Nj+duwkaFnA0XaLVOaXJzdK4qcoLroBJGXbx83v2OKk2sThak\\\\\\\/1fex70MMj\\\\\\\/AfNoot+3mcEllqBBtUE3jF0xuNkH7RkjKqBi\\\\\\\/N7pmCD5SBLWqj2zKxm9oZeFcL2jQN4WuRon6llvfhYerPNFeZPznyZcoH\\\\\\\/Lb0Nj2FCdyisEsh+kx+7674ggrope1Ln4sNVfcAjPusJlQenSQlrMsW7MDDsi0gWOfJa+\\\\\\\/vdHOh9nwxP46y2ML6WaJd9H4woSC9B6knZMGL4cyCDmMSlPsx1ML5tC9qYgo4TXchat2PVEQfV2hNFKXSgamad7nfObH3taBkDuEIWZ683Wj1GLt\\\\\\\/w2Gkct88EJdQJrLq\\\\\\\/AtJOihQc2a0RF1FNUH6VHdd9Yg7cV+nOfeOLdZMAhlyyNvnARDK9qMx4DIBibHK8s5AmA0GF3fygG0iZO\\\\\\\/0W55APx8Ruxcp8KjYIV3JTeWse9sppn3fMB5U+18\\\\\\\/\\\\\\\/cIPPeHlPsQj3J4JOiEy9nrpGzD0ewL+bvkgakKE0uqvfjCyI1e4WlEb5d3voIUa065RwkkvOMDhBHtt1sOuyikA0jK5\\\\\\\/ImerBWC5o7IDAdrx\\\\\\\/TMgQnnxQqlj9K8oUVJ16fLpyNz9sDH+W8gV6nkLj6b8NAW3esMKpYNktdxFsRLaDa2pFWF+2iSaPYPu517HBkwv0me27s8Y52JtfVItqqWAIQNtddNvdNA3JljEWIHboqS1jLqyhIbug2K3\\\\\\\/Io+sP7wthChql5HS89ojs7DzTaHWjSCuMM\\\\\\\/qSKZZZXxIILS3M9xTRmN6xSzthymHfd0eUy9eR1M+hTZB\\\\\\\/b7k0eJBKqik\\\\\\\/7pZM6vXhKK\\\\\\\/E32ibv\\\\\\\/XYnJOpmYynLrhqGZ3lKjA8XnIIOIL3WAsWMHx3VShCSH6kctD0t1DSkqXCCCH4QLeNDNFgY0zP8d18QTkYFU6ghqRCIjke\\\\\\\/ZsqkEYUMZ63ojbG9NNJcNegpVh9TQuFJr7g6FolqVTYr2WhRWfLIy8BGJhbpk9\\\\\\\/RcTk0E\\\\\\\/M5gDVmecapV3nA3Th4B6ebt5+KhYHeVi1zhPCR1Zh2iFB5W9fbt8FaHFL7gTdrk4GGnFRyMPNGeJ0nfWjp5PzKsuPBR1CeusDkzfCCi+iP7qdQgLq5lctudXbH\\\\\\\/xWtV2Z9kwDFu2aLVoYO7yTz7m0W2pAFPS3wgkp0NEvmqc9Q5YnletmzDfPRZ7VNU4lCmWUEJDRobc1yRAaT\\\\\\\/Xj+hWXnO0FbsWbwjaHQLc2\\\\\\\/BcXRH6VIZzbTnltvKJZx45IRVHXiA8citzYfvrnxPLLBoHW81ABBlTTrVGdwmljufWuc4nEzxiZnKIF6tHLcnmj2dj92U\\\\\\\/ppuhim997dVgxJk4vdHznp8mhZtDaNP1x\\\\\\\/7qJf\\\\\\\/PuWHfG\\\\\\\/claBWMM++UsKUVGhh\\\\\\\/iQlgzyKUwrxK\\\\\\\/HMpht+2gA5q3NZbwN8njgnZYuaQQZ62lraefplABUgIRbdFqKw6Jsdzx0Pz+Sq0lQm8Q380eUu3msfalin6fN3caatw4v\\\\\\\/wbyoc4H90T4riHWXKgRUsXdqk3amrbcXYOVP2gA1UHWqXmXj63UMO9dlNSo17vzV0GLYHETZIL21L7UsXn4UEsNXje\\\\\\\/ZFQM8N2TaI5O3xx20Kqmyl0C8lelie0cUQ6CemHkPPzn+IANor\\\\\\\/AlOWK\\\\\\\/icCG8aNpuRssAyZVgHDg2KABS3TdZUBBnGUL5S\\\\\\\/NY8P0cHLJeGopDl2Yrl0AH0x8jAQ+auBo8IwWJdzlFIB00EGcOQR471QoFBDno5Nnn8Pn7qwH+sY9i2Be+2YTo6KPBR06lbtiTvgx4ICp524pzJdaK6rvRVA8he\\\\\\\/VJhu7qcVarEds9jVme+OM7uE6uyXCErEzudyzwqJK6y91SGV8yYFulCLudYw1RqIwyLqfANqJVjJezAEz\\\\\\\/2ng\\\\\\\/HzmTs0\\\\\\\/3NJfw5WY4KBLXjB2svY6ABxT3MLH9\\\\\\\/Rv87hevo2cUmsiFs5WuNXDwM5R2KeoitR81mFU+PFRclatdB57ECGoqYtVdcAN8FGyG0l3OvMRbYUJu+6f+D4WBdiHW+CWDYM1y1h0EfnxiK1WuZNskwgkOJDOvjU5Pq1Gr73TDzVU28SQa7Yzt8yFepRipZHatlim\\\\\\\/pATGkQw6NBjej9NW75jEHYq0VSAn+AlZLg6mf0EaiH5pvrF\\\\\\\/h9mbpJQ+fcclUxSSHzB09L1pb3oBAlaH7BntTtNTNg2B2I55luORn3nwYeN4jotT5ns7NJ0MuXC7RI+fgVe9DaTRvKY0QfV2FNYFhepRKfdtkUrAtNodiEQjvueWYTgkYA0xgrZ8PB5b16KPIPCZs8Uu6ovCpALJRn6SInGuN5qyvWPrdV\\\\\\\/sH9cnZACPiaQeI4wGM8xwITQh9PxwgA9fLmUSNKAjQB9h9PZWiwAmpdZv\\\\\\\/Wr9k6m61s3j7Wzfpa15w9plPnZnxcbtJIx2kOGrz\\\\\\\/KCy02xZDToyLYRb+FZnkKhAKi+4ZDo8guQiDu98ycHlXQv8qkI+lmV2RhwFUmEZcmKTMJMTNGH0kQ+B7yIfb29gohlaBYZCaQZbaBcIz54ANXpKmqhBKTaxRPrjqlMIDi7EMi5TZtsjDv+OrHyyTb5bO3WvdRL4zlusBQrgNlGBqkT9dHd5hrnoki+Xgx\\\\\\\/hCVqZ1U873tHAjQ3oUT57BxD0FdaZcxu2hF0NcEdJ12WuLhO5i6YDAXm4fw1DaWM0z+uzPVcLjYN9E5myd72Mb6yEnZd+3MspuZw7IwoNdrwWbO6yX1qY3oSMJOY2bK0TF9cjGnoxS03f\\\\\\\/1kgvm5taUxI4QGGisB6rXjwofvxjCrB7K5WyJ7VsVWnstNI9KF2d8LfNHqkWDcTGZ6JvxbzsJ09V5fTOktKmrzJXNP8\\\\\\\/ztgqk23LogxLLXCmzn2TgojKXV\\\\\\\/8b8Hvnv8D4247l4W8OgbMBbHc51HTmK9Dg4II0AbEEB2LPTGagXyojDO10\\\\\\\/tqD0iLhAx9x\\\\\\\/PwXrRWLdWQgauZhrzluJaVaU7eGbaXTwyQTd\\\\\\\/293RdVmQNqas9wqYWB32UUkgCwyM+H7RivJZeAlgPGXqVW+f+mNZCCYK2kwuXWcxKKbcfHwPrf1oZYNgsINpa6LEF\\\\\\\/0jLQRqv9N6MCDzOwbUJxM1CRCIFUWh+ZjqXneUrLvh2b0FtaA5vQLvPzelkEeA6898IW\\\\\\\/N3mV8bGIziloSX1mt1K+WuZYP9ytWAPlXBojzFIxfJi101Pi4CRoLg+eKtFMlOxkpIYTH3Lq9kzSZt3ywksAbtHg5KcrxBSg2V9NJjcgX8FLFt3kXlVwA0zTe3g3cjoTu0NiTlJQgo5\\\\\\\/x7jP6T1KG+QZVPAL1jFrYv6XZjim\\\\\\\/wwjfKn5LUzv\\\\\\\/M1pLZOjnk962VB8yhxGSPHgVJImd9unJHBhJPVnbajMFYCZ+EjgJCS+m28O\\\\\\\/stEDsOVC\\\\\\\/vnAkp5Gq77HPuWTH0DbrnPQMxzf4TTpI9TP44RcJXg8+pqXrqbNF7S\\\\\\\/FYfgeQnQW+t27gB5Mbr8jBtxFOokDpkzXYNxkQBdbQZ+cqzcZTU098DZoUaZlX1RkpxvUHEnz6MjweefKOGP4CDAz7vboIDMBIQSuBf2Sh0RKBi2VL85\\\\\\\/lyQIf5JVuQIZQo2F8s\\\\\\\/XGE+br048PN7SJ3yo6hPS7VdtsvyQcirFhsRjtHTGYrrkSrb3GE5Xyfqxw6mF4IGTzGEFJSrPVAvg1InWixSU7REbma5LYPEnuJSrLuNV3YRjvZS2Tmtey0aaaMHDS8MEINHhVhzsFrv+rNLX9i+apTSVvFb9\\\\\\\/v1OO43PFtxXh\\\\\\\/Hh9pzaRFNtnw38SRwNpNRY8hhzOX9OTC78O+ECfQhI\\\\\\\/BtAGNPOby2uMtc\\\\\\\/1veg1m\\\\\\\/lU8Ie1PwtDaF86WdY7OvEaW4mOpDgJLQyX9IvuH07PJX8r0k5dUMb33TJH4g3FPSDAdkVwYdXrzN2kO2yFXrPodtzws70PrlSZfck6zNo\\\\\\\/dtfMQmnV3bQB3fDRIaLC0l+4TQYoXdVHoDzpP60hwyRRT1tKShv7wYLtcL6dUEodDtK1v\\\\\\\/71NV0+ebKNbl\\\\\\\/XxZt3XkoSP5Z0ZGwBdnJi\\\\\\\/MP9xZBytI\\\\\\\/SLi924X0GwDvsX93Tb6JBcXvvVSg51\\\\\\\/Wok7IK5MVPF3N7cZwxpztXUVGG2lT98fr7Jc75rX\\\\\\\/QZO6TdEmfeQCE\\\\\\\/cZr6tQ1Ea+FUGNJvzQDrVsWvvJP4kSaWhzfjREOfKZNEHrrY77WKLmbb0vsPyWOM9RQMiopqx3vDnlvBqYn8TTyJaTsqsomx+mLVzFFvJg61MuuKPIW38\\\\\\\/8apBP4hHwYFMml3gZNN\\\\\\\/uRTjew3fV6C98rFAJ2bnnDGRbKfaMa2OtUJ0E8ugUeQwv9bvrn02TO2pTP+w5Emz0TxpdE7+Du0+OV+JpV05PqvPfGLQqRMSw7\\\\\\\/5S9s2ZYWOKlq+ealtgMbO4lxnjdtcqkvbxOniKtM7yRM9JYKTnTbFLFP7UWY\\\\\\\/yrPhAnBlBEivZoz0K2EdyqO6Q3WDQFzzJInQNtCKdGc+TC1aCjmcYMOOZ+dWJixuhhrJbEOr+v\\\\\\\/r9Y5FMorFSm6T0ov3EKp2gJ1EL22uJ7kNh0kJsnbaQXZ8iylyfTe\\\\\\\/awNU9um5m8jXiH0VNpTbutJm8lJMSetASjiMqIMNiTNz8VW8KBpSKjRw71ZL4MdMtAs4M6mvDWdoLbLNaj9zgRruaEhs5SJgMuUFEa7KxjrZA+AYDmXTzYhQq2J0L27e\\\\\\\/X99yXu9KxJjdOlt2CPQSvV8yfPqoVd1w8yuh9UuZ2e\\\\\\\/BKeNkAHQb4mJeynrpu5JCh7VfsulzVmYIuJXUtRgyO3BmFhntrGfy5G9DG8XICMl3D\\\\\\\/U5BcdeAIF2Ycl0\\\\\\\/hOhcY7lzSVlolTtZmGyVMhWsoIh+O9QpG4NmLXgGB3od9iw2tEmQyTHiYz6RmFBjV1dBgOAiw0C69I1CXhR5R5Y4ohEpfNmNKDo2U9TBckwURcMY\\\\\\\/G0Iaj8KuAKmWobjj9WV3ZezD0OnGZ2hxtOWx5buWFnutvzCuT0Ld16+\\\\\\\/LITAl4cR2Tq\\\\\\\/F61v2zdcEpDBXUM4JEQ7gSKGtqT4D9GFNSloKYHM\\\\\\\/pR0q5CqRsEoH+Yi6+BLt3Lz9LS6T+mwwIWxq44p4XNDZSCPrZj\\\\\\\/ZN6TEdt9xhA\\\\\\\/HBeTOvU73lKOhmxYAeKgAXcDqigvzjvVE3ip4tM2mLDq9GoMX5SCoHHj0FvGg141yL3NCyUI\\\\\\\/X+soGMM0gcyLiViqB6Lbjr+xsBIjDlNPMLWDvQa5OnRWSBGFeYBVfCN9DDsKP0qPzWkvNboubmxi8dQhRjRicCmBkZ99a6pjZLqDabTMjDi93Ik6qOuo\\\\\\\/4HBPiT3j7+uQNpBJp\\\\\\\/bY7hl477dv3xX35cVIc2jfnhUF7KfUXLQPoQMnkFuM\\\\\\\/627VtFyT+7KUSxnElqcyJYNxMJKVhcKLWMixz7FwHdL5igpUqE2PBA6ExFdXuKFezf3U6K2Mfx22Gl8GcOMtmiu9msT84gvVuW080PqNWb0HkPJgTFaI7xukxO08Ad\\\\\\\/XAB54AlU0WmScDbeymwo1WsB59p2pKczV9I\\\\\\\/6LmKjD6\\\\\\\/Xg04dQYURoyFlOM\\\\\\\/IWPdMNxtm\\\\\\\/++YxvHt80IYpTR3Wqd8T3yUvJAFy2plJc4y3kBDzo4vHS0v2HMNRt4hyyh75DXwPsMdB+aMb4+FtLYMbJBWLd5B+qG6HwpolbIORtFAMy6BrS\\\\\\\/TuarBMxNu3jTcNoqHmumo3\\\\\\\/+1m+sM+9FKLn8w\\\\\\\/vFMb\\\\\\\/AD\\\\\\\/hjzthZW34U56vXXvF6ixtRvfBcfuOW1zT62dWNO+9aqh56O\\\\\\\/YQ4P+gqnCji5AdRSNMX+aaTCydVwquPCROnS6GkaFGZmTEi8Q1rFNVoiZ0iorrgsX7341ghn4RP6lYuethSxt+x6jTrkzug38s\\\\\\\/FFXxmOV143Na3sBTqOVj7I4GgQA2NXCTSyQY570ZZEf9JNuWJU2erHk4hnsQJgK3Pbx\\\\\\\/YGGqXMLyvOzDfLKvXUUAI4w45JfD3FGkTmlNukot4ul5q2w91FBMGKfsiM17l0Bwaa0nBjxfsml59X1F0n\\\\\\\/zmLpWP2hSv42FRUat332\\\\\\\/nCymDtS\\\\\\\/LLLoNAsp5iW4MVPRqY0cjHHC8ErsBZEaioWC59vsigv1nKwX+\\\\\\\/+De4NA6Lmd5xwzFL7sqLwmd2WqEutb\\\\\\\/YlEKkxfZT8YBuNQHbOtEOw9bV\\\\\\\/qw5yAQ2A42E1dvhYvKWeXjhGEeIW5f3YX9Hkq7GmLvOvTsr6NrclkzUbcGrbCgJWwMDJodavP+rHXygAMbuJAfqhJI35MoGB6CDK3OYqnx2VuNmtOThnBb633tG3L4nkI+LSieXXkXJ+wAZCfN87UFYUXLwb3oy46KY3Uh5JC0lxzLHWnYYfCrEWWqprjQ0RkWCiYHxnBnNnjvAPd+d3sULxrHjD3MFmfExRjo5FlbfTY9T26zKsmYFNXicId88DIXdTPQNiwuw+4cTlPmBkE20DhWRVzNLcmqLj6BU+uxksCfnvCobW602qMFijWRr+K5KXETbAE3Pse0AzOwJU2P\\\\\\\/aNlaw0sDG71HwfFPu8OkUo\\\\\\\/3BpQKiq2wnwdB2cu1oxEUDzkWFpA7CSNFyDHPqQCRuE5EUwqVL2lJO0FaGwYpiL6xAzE+bQDtkyJitfoXxUoQRlSF8IlKlBa5RqqtQ2e1zJkNXVAxRIi9z0\\\\\\\/RoGWZfA4eHaapN6uz+j3N7lAqVdKhsUwakhWYZkdqWEYw7slxMRoFHfdTDCcgum3UPfyOnCUbN7mLIqtQ\\\\\\\/Xe2eCv6oYdYspkkB00rUCG\\\\\\\/Irjon69o6W4zB+lDDIlYAyvPzvvl4ZTcK15hbcmn\\\\\\\/ejN8aUayooiic34KMvDJVahTvui0VEOlwDV0aV7KqRljc3bBVujgOSiJnXSMyupH+0oWN4Cff\\\\\\\/dOx7kbgH+57dJs0d+eUickUtbn+MUFK6nLAteaVgzc2jV5nlP3JfCak7TnNbO9HJsUebOmq55kObhwP3P2yYrov0jAZwcxAA8iqJ0mgVuC\\\\\\\/Qwjg9cXB0qvD9XoZkcyDn+XLx7qqqZXc4lIvJ6qpHJqLT9ZJAMATCUbE+dqhIvdtA88hWJATpck2LhiDZEOoICAmmJBBZQZfGrVdom8VoSy2uQFBUTyXmajzKNpFoLhfxkEqKVoIwGIVFwkCtFU5g\\\\\\\/QI8V2NfhA12Yu45FWE1XGrM0Fca7owsIWr+knKnR19JOYPB2ORnxOqMzR\\\\\\\/gifc\\\\\\\/JOaHEnGR1IY2GBj0awnlgiP7dPxkrpNDlfCCF0OVzhlbG71vJZAY3O+yrfSTTHyP1XOgyfEeCTf1jLdcCzzwXCwAOsqcKrfTg5yeTzn6icb1fZ45BRHjdr\\\\\\\/UxlBK9whEiQt\\\\\\\/7D7DL8YfgsweCGPP+cQC6VueSDL\\\\\\\/zs+0N59she8nJszulofrTnkXcyUkEmCWJnYBWVBvka9GDlESZ\\\\\\\/Gxa5L42Y\\\\\\\/kPBJk9ij+kCd5FmisBzVZkT02KulkIHFYgw7dcx2iKxdfM\\\\\\\/8hKxbE3SKW3b52SOQnR5orlOvtjnkcZIm7HmiV4iXgZMv\\\\\\\/HgKvoqNETXNbaTiaEIIQErYKVzbAaVvdV+TIKAgs\\\\\\\/Cn2j1nyPD2S8F9T2ML\\\\\\\/gYqWnrSi8TVxlJxS2hjt3WfQaTTt4pJvxP6t6Wcoz1L0in67HIuS7PojNVuznf0tS1FheL47PB4JuzhRa6\\\\\\\/qklvPxbF1Pm+54qacL+zRUO8JB7HQOEUM3KSzA78HxfyjU\\\\\\\/3SvIcxFioymNMQbqnp+LbYmFlV4Dvc7CIqVRQkhP1BGXRI0VZHJe7ged4EbN1B6EA6CAvU1ztTv4GASPYzZV6mTci0z1iOLCqyfCg\\\\\\\/3xaWrzq28EHEbVLTThT0BwvT8uYL7NiIGEjFc6hJxKphNj3WLCGJYDpAjIrWr\\\\\\\/oDKvs7ICpWtaOSgajFYLXbW4zhZlAZldUWxQc8dLeWK7I+NbsvjD0TJX4kmytsezBxD8q+RoIdoT\\\\\\\/kUzjVFK2cl3hca3rpK3lPFqRQPWASbPpTJJknbgvH+CQ\\\\\\\/T0lVTEnRFEZlA2AXFT6faWoCek7YLAG5B2KaYBgH1rx0\\\\\\\/uoKwmDBPCm2ECh8kXm7+\\\\\\\/g2YFCbFIL7Tv9AGjn3JzEymClBmoQqv\\\\\\\/+2rjQau1tKPMrZKi1kvZggBzlFy9jUM5fwd0ACW0896dCnwnAaeMbHpgDwAkVQ2QWFLKykD3vJhJh6AKaQ5FBtX5MxERC2EZOksV84nk5RcDWhsPTcxXEX6pmy+Y5K+\\\\\\\/hLzpPB4F+Gki5Lo45LuCOqhaqRh1HptVFSNqVpZfrLgOnbTOxUa5CUxKJUAquby3SkjCQWDKnVGbKqq4CJwZsfCBJHecSXkmi2g1ocKZhiKgH9MCUmpHFDWJgqf+uRtwgTr1YaMzi\\\\\\\/U0KvMdmwxRRqAKn1jAitSkYGDsWpdx0+ZeTuMoW2hqFLVvisVcCBQUJSVAIr8TSLIIjS65oHwXLx97F2ygOHF\\\\\\\/KW\\\\\\\/HQsZN7+YWxrusOpxqxZAH9QhlwEFz5vsgz9N2Q7v\\\\\\\/6OSDybpsN8YF4kgYMdWDyBMKieVUKcQxgJSKwRlmfIIH+xON57KQQyZGgd\\\\\\\/ZXRYgymiIJ55j0Qklw0akmWkiUCk\\\\\\\/o++qh0B\\\\\\\/b9hw9h2HSJrsW5+4hKur2fZDpAc\\\\\\\/VIViYokZb7+zAolnoFn4cceBcQp5EB2HTDpSyNFMMbfgzaz3dqnLzrR\\\\\\\/YPAnzzDBa6X47pmzzNN1Rnm+pyLL8TsousHi9cxgVR8btfOjZ+bM\\\\\\\/LpB5uw+KRb3X9tbXoYuvxQLPm8o8BXswa7C+sHSxLOmbEDz\\\\\\\/W68b575AvVyXF7UHJRwNKy6zEG6vjv+Vco\\\\\\\/qAI7cgpM2juH1sAp2fICb5AgxmCPdMUWfWX8lL5g\\\\\\\/533omAkslX4FFO\\\\\\\/PCEPjUms6RhcZyDTnPRdsaG3GxjH5\\\\\\\/Nchh+\\\\\\\/kEcNlFXBmjJNNBx1gLmngnpHrkMbSDKY+IPutHfuMkXzOcgDESojQvc3b0ZZxRb9Vb587FeOkfrLqYRxM6LgXsDQcEOLX0IsuKmmV8mxOiqrSrW6T+E2uWNwfnVhUhY+6H47QAwkOP2k5p\\\\\\\/P\\\\\\\/doF\\\\\\\/iRAdk8hKsIdLj\\\\\\\/AJwyZpaNeHbKgdE39LD4nQ8H\\\\\\\/wscGmjkITZdm9QNi2CIqzWPciu7yziaPzcg3XVqylDlOg3qQnk1yVlUyc8PLClfpq3NEehqUstO1vtX2+DMYvTZknYadp\\\\\\\/ihzMV839MjplvbVXKU9RB\\\\\\\/8PR\\\\\\\/DH8G1s7qq2TOPrOKdPlIBLO2bOfP\\\\\\\/V1fRVC\\\\\\\/wJVu1DZpE7lY63XcSB2T0gqLmem1DfOACkdT8Qmo1csRFF8LS3znKusTfJDNOxfbY1xcfcniJNvAYHYxKb3HjF4Yxvn7cqFqS1DxBDGFRyCBPD6t9A\\\\\\\/fQ+yBLQbKuBuyFMXqdXs9fnar5DvEUOvqYtH8oedESgN+Eq5npEFxlpqUdUvQMUjxmp6IVdlVfSznnL9f+ziTntFv34gbKtgn2Eq93lhugmY0fAepbL1kGq8PpC0ejcS62NsvRxrK0ibcr+wdDSpRcWW6BAw620GrMaKahrxNrX6k5qsmr6WR8lf66+PItIOtEElxG+PFDMeew8Xz0KtTBRAlgDlHROZ3m0CUvYpnEr6NAOuDS6iNdGx9\\\\\\\/BBnRFaSqyelNqnDARgMZW0A7RqXTKtXrxccBPui0RJcAgnPjaBj7qnxeUckJ1f08Effya60FYcRNUmF2TZH5mAL\\\\\\\/3Wspok1CTu6x2Q6S7coJA\\\\\\\/FhbElp2peiYj31XjjhG9U0vUKb5RgHxTZbE\\\\\\\/qVAR\\\\\\\/cJMPZQndBXfloBt3Ghkedx7npqPvRVkCAygcm0K+vsE3hNPIATBMsI6J4bpch8hZueaHHNnaPiDd8YxKxnFEg\\\\\\\/A2xuGHrfXmBUP3goXFmPELPojfz52JauPSwXMP2wz2D2B1IRdwF848TsXoGkn4quTPinQ13ErQKlMBbAZtmlrO\\\\\\\/MoOqIQZymQB\\\\\\\/1SpOtkp+vZzF4rJ0WDfk8FC3dNr+nsPHM7V0bIDPm+cuQsN8sKsP0PimVP2nFPUZhte1cp6VEV5Zg5lm1htDa0rFJRZ59zvEfZc6Tuvedkv0gDtYDX6o7YeNIYNuMglWUEgbnjFhj31oKspm2Iw4awZ64T4D7L29EGrARpM9CVFgqKRfw3YHtmkzViMWbp5SrMPCEAI3vb+8aAGUFQBE7jiQk91acuWU4eCpjXqthD8Bo\\\\\\\/bSfY8Lb+D66+a5PvekGAKEcPpjV+3QfhEa0LnTN5nvltkR4tGX854vSvkyqxWnJXHC79oE+w0boTt29ytsOzwU7JwkLlrpx2Wx\\\\\\\/39XlJ49AjsVYpGtPUoUJgOA+qj5KbVNgNMR7T1z6nmNvGhSOVyKEgUxUvLKIJXuV8OPBTFK00ixkCH4YjEQm0o6wE\\\\\\\/x3x6CVEj97xz+6br5zOjnAl4yTjeuCRfsYL2xO2rUIi1IkhzdoHbs3bHFHGnVo6Ue5cuKhfeJJaAH\\\\\\\/fhvhuPxhItiv1eDlBDolLnIhTy6gktgazD5i6Fitk3U2CPLo6usalSaD4u1BhXrBleQhMX8oTVnE\\\\\\\/cv5urSYFcVQOavHvzaE0Qw4g3TmtoTry7eqmZg9ToxOsI9R+vG\\\\\\\/v\\\\\\\/kz4\\\\\\\/qSOUgLZ0vVBEG04YxY6gkM1EjogQ\\\\\\\/zbJBUSVHcpYIJEkC3ciS78fRyXMlei7pqdUsGP0C+yrS5Vaxh8BDf0ikW7Ve5VdkCcFY8NImtaf7cWqdWlvtW8\\\\\\\/Y5DYadHZVvidAcK0fcT4wrwLfpxtA5ZP6dGGs4TYaJITW3duHFWaNv2X3JhD2QvAWQEPxVCEm2hiQewYqO9SAsy3f7sfSyyYBVnfdjDqlQQoEZtFXhFe\\\\\\\/6J\\\\\\\/J6sgCQXMJddTDO\\\\\\\/5d32USnQeFjP7PsL0ZIZjkWvvzqj6cjizlIn0XxOt\\\\\\\/JJ+unKs6F1D4Gi8bsm6zBjFR6nmfBJMKhnp3h1VpCzcPHsDpKeN3J5Ul8CzPP7Dbm5lSSbhdTLyS\\\\\\\/uwUx0puEwNuCWnk1iO4mfJiImyWuiZGOqv2JxkWRy6wVVGv63sMaDihKkFOik7m1e095X+AzpTdpbnqOYD4lqr7fTyi6lzpZaCexRfRDZLlXV21cQB33TVaq4LHoiI6VqvPB415V8whCCQTY5P7MAyaZtSpBKzAwh0WBLwPsQtwOnIWv0swoUQZcWj4n6cNW2rZoZD2Wup37LVyx6WAsvBoEbT3CBznCPw7f+FZcqd5otrtCZc1uncmfW0HUPgQm6ccQLejtPl\\\\\\\/0pNcvqRklIErT1DJMf4dxSa8\\\\\\\/nz4EI\\\\\\\/HXtI1sg9tvsPbJmWhAaDFJezbhrMBwzS9CLPfF9s3AU4AVpPSvezN52DPnGyudDMjyOlZbrTWkJKCO\\\\\\\/vhzUUoeiCc+ycotvfQ0AtJ3DUtewFqGOPIQlZrZ2zGFxc9XDn3qqokNXPOLJLAUm1Uxg3mRhUsFsa5R1GqO5evKAQitN80HJp8ZjDgMw4Gk244o7eUYVpr+ejyyRV6ffeT54K8VnJx+zdnNozhZwNZKpZUz6QabS3DCVF0CEIqNm0TRQSyJwDZuvhB8h906a50XP8Ih1WChDZTKOv75Ii96WVfRwCElMD3zjPjR+Jlhy8XvyqCixONRx6M92YUF4EXryKPqVCvAGM9vkC1vHkER9TjqpFaDWK3+HLoDv2+gSypwnkn74E6VP6V27539EB4eh9uIRR3iPcc5yUhgYjtpDSp5ikPLmzV2I2GU\\\\\\\/A4D5nst+aD28AweW68tnX+BZ2uNT9xqHmXph7LsQZMbRkUvwWPlDF6WUEu6nD+yoT\\\\\\\/GFp0QB5fW3lmNhMmUwFYjc0Z3Nt5hwCD62LhtsifIJawAUvv3x83ixZsYLUbu\\\\\\\/kwVmgwhd2iRvhXDCq2uoC0fflTGrHg+DV23tJ7cSb\\\\\\\/Px5YdMDrDXP\\\\\\\/3bWr+Id3uVL4kfcJoamP3ExYi0zI0vYr4CrobleBAPS3G+PyPSbnJ6W4Qd8BDUx+1EjLYdqKq6MB6RtRyAB4v0gqq8gLWbLoxk2W+kTwGqaEigcqSMCJFgcE1vBMHUvPX6OxZq6q68IPEHNZ88ulDa9A3TCTMZb8NFoGqvQQXK4UKkx3pJInfOg7xZFcaWoRWvieGNxMP6Knfp\\\\\\\/VMCFyew2nxBg81UUAOkwHIPRMzmFGZGKL6A+Ae+0r2moB3CCYr7gPGFdREB32Vw\\\\\\\/6pzJ8uDNklHJ0bhTc43y7Ho3pVj\\\\\\\/cnpUwnylLwRcZjdtysDzEdEutIGK5T2f1yf3VIxyjkktkT39WZVEy7YZgIeGxJ6qOEK7nJncwPci01o6whBTMj0A2693U9qWWY+a83WOtM5EkyZmNpVNBiRf4743qv2glIOJQjiXe6AeUuoIkBGmNl8EGX\\\\\\\/Cjgt3UhDFIfvPG435m7JoZ9+y4nwVGmp9xrlKG4qmRzki4p8zoX8F1UCfcxjAJbGz1sAanJFjuRCR7JrS5NBJyvsqwahVjcQudySqsCm4hOvMfGVC4RWrHSp1Za+xXdp8TctiymbtpA1uuDIp4qh5brsso2Kj0zfJGHLKFpwUhaRchixWDIubucq3X4J8+4o1tmekOZeGTzJbFDTUhmXwEmbDk1uT0lH71QZFMjfBp1DNKda2Cerv+6Q5Dbr1g\\\\\\\/Itn3Ztec\\\\\\\/rvurhvkNcKgUduaQmSDKY9ofGAjLH\\\\\\\/5PAlJ12mOpuNT7+3fUw9gIZtigfe+jF7ItKWNuy1mkwUNontiOmjh9SycAVBCNHACWbFE55cSvbDw447RJqp861i1jg6t8RIRniCCt1Ml8DflXX9IfaHZyu74+4N40pPtCbOp6sCUxoIf95wmzPvbTYi6N0tTcj9g0Jajv38\\\\\\\/iLovwdnaWtL\\\\\\\/NMMlL32rmVnieyleQr\\\\\\\/oBtwVDidGVbmfatvWEdwc6MBbsI400fwLpyxdzC9sVJqe\\\\\\\/LCCX7ahKrwK6UbenyQjV1ucsSVDj4\\\\\\\/fM9v4Mp88NlpyEjqJT5XFPuRmfxEfZIcJ4xkW55BUiJUZunFZp7ZaJTf75x5BIEtg4r5LSLO8P+mOtfqxlc28BzVhjsQAGDALuYSB2GpiAPgHxARAtXHQqFSWVISbE3Gsom3gmyGnb62WHHlM1qsKyppBIehZCl5I8JGnavZIMY0XtWWJucGvGW2jwtU\\\\\\\/SnSYx9dy\\\\\\\/ITK\\\\\\\/X2x9czLfQrNLpbKhsPL9RhdPswSPkGe9NGlQOjNDCdDC2QuOK9GtHDWPSjXtD4D3J4fMwprkPWp8z\\\\\\\/V9DeDPLzyAqGKW8x1214Ad68GIEAz\\\\\\\/vDrE8aky6uFyxdxhfMUnfEV9uYaACR3q4UCq1+wD2zlFF7WCl\\\\\\\/Qjwq+In9OZ\\\\\\\/3HEi\\\\\\\/QaITg+mQX7N5YC\\\\\\\/SP7BE1TXNFddwy19hORZZpqtKUQdTxreXfTte7Y3BoEwmDlAlnRw4DmgQFg9nM+u0fVp8U0mGMPytlXh\\\\\\\/ShYhhR7rxPkInsa+Pmyap3nuPBf9ahNJledU+bAl9UsJmvR4NTJUDjQFE1CzFNRpOyU9Fv+Zu9X\\\\\\\/8NgE8rZja4Qty5QBEC\\\\\\\/rFJxyb52lJziVpeQkBSjc2Lrr7aWi9SGAH6DsPy4mzrEUjMQk1F3kaGbaDFXq1ekw66JUJCaEcCaKmBrx92NdQW8JGTrr7R8+RFto5ipUbwuYZJty26PP4QsaMHPvE9+yGU17cBa9tls8Fd\\\\\\\/ruWIrvVlcEvzskF4455AnfgJJi6wrqPz6ycvVOloUZzKxtGm4DNoASV4ewXk\\\\\\\/O\\\\\\\/3ePkIemXb019Cj19fLXAhCOm9MRFTBPvXEA6PuTyWAsuJ0IPlhIwkGm\\\\\\\/c2\\\\\\\/s6UjW7wFQchcLNsjR85w0tF0Zbp327AZqfCI9qhyz30a+BTjKLPvPYo8NnDxpM9hpT3VsjtrNjxTGsnW5htkvfPIAK\\\\\\\/K1U\\\\\\\/JnVUlnM4KgvkyjwG94Ov1kqffMt2cnDpaYI75DUjVtvp6UlTgARbvF0C4RJr0itld74jKptesWvDoFLsmsI31Wm74JGOKgCAiQYud9YLIrqbtmp0hZIBaCgZKgd9bMWugRNu2DsUfRc6RNJFcV74AGd9uT8KObCkGx0gTKZwIBq8wWbbycVWJ0\\\\\\\/YlvclycjJWzq7aANQ134iqh0mXnTAPh\\\\\\\/NbkZntgBzM61ZPP0htMOub0nCp7cxWyQ9EAZsEQ9sxxmwQ6SobOcJhTmAPYTkYQWrGe+6997MTujJVEIMJO1d7WQa3j\\\\\\\/qgMXVXYp4sRuUgj\\\\\\\/33fgh8Ygb9MMNfO5x473g1krkNP1Hgk67Cju7fSOI+bCoFxx4h+3xdRYwgNfaGrK9t16xdH9bQ5SuhAwfYrYEFlNfOCXKDGshFW3pf21od98ZD9ULxXhUJXeff8Y4Myx\\\\\\\/yVrFHvSUBV2JpGjghlKehukxWTvycaMA\\\\\\\/+C9r8bhTyeE3SY9CBovYn2NwBHTW8Onz4y6SAErIJMQHaNupK+rI9H54VuChxtW2SyCKS+zaSe3eVuPkOmSQaIk88FH0qv7qOPcd1lWIGQCgnPQYKsL1ihNOrP0dULkKr80t2TcsSqb8O+JlpAfpiG\\\\\\\/R7\\\\\\\/kEdYhogVQs5dm93CXIUmTjuCRyckmIPIkFDZs+13k0rgubMaqqTh2jykcKNJBpuB+7pO+0+Nmx1nYcPSyhVh9EgapF5tX6QSkLV8ACN0siqfM7JAHw7t0D0AAy\\\\\\\/r6wcEbMYy5FV4tuU9wTmh99pubqXuHKBoKwyUxVLAzuqO5BorUW7sO85RO8YKMF+ZKAAoI736PyA2IYQErWIaFy821l9LekHSEnDl11JukE7X76qdjlEEQw6OTaGB1MVVWuUVWXzCI7J8mAdBB0lm6uX2TX1ksVW8l+OyG77qcuyrtHtfTKrpplSqphBQekHv9RjhHF681x3xLkWtdgvq9cpNYh\\\\\\\/mv4Rc1slbInXx50sZGY93J7ZzrwmppNtUW\\\\\\\/YIpOn1lu8C50nWfvl67\\\\\\\/+JekzXcRGgofirJhtNHjxSebjeqmR1Leh\\\\\\\/sdUMp17RkztHW5Xu1OL0Ev3TKMAr0+oSVkqczHnBivCtofxrmCXxxaMk3OAyZeZ+p9v5h1rdtd24Dsv2tVml9Tt7YYTmdJ7CjCtKeZpLI0882ZOVbcmxCNr8lW+swuYb5y2zNmwMaJlBFegY2eQqXOevTec8NnGdhpx9jr4YOS1nFrMAPINz3PHR1rHJvJF1yUvwRJ6GLHU38Kc0VOBxC5FGa7NTnPUfso4jDfasp38UrhdFRxjMVUN+mr2\\\\\\\/169\\\\\\\/pseFc7s4rto+XdV+BCX2g5+WZru50feqQYzPSXhckRQ+eSVKBvQfSCj4pSsOaJhDHuBM7k3m8G\\\\\\\/nBsknPnXYzwHNV1+POeRmE7urKyATSxw1Uj7yz+mdqe3QUtHLK7bvvKZXlybXY9syPtkjJ1nNLVJvSYwzkzft795Rf\\\\\\\/B0JCpfTNKyVTnQL9RYeWQPSt6PZoiCyDhEgqtCstL0loW4Z0M\\\\\\\/mJvwq\\\\\\\/tQYQPh92sOZLqwOEaRc4FDKRuB8N0INMelTmqaeIiRy50XnU8F+N51j338bxSNvDZRKUrWkw43ZJEPZzgzD67yDXquYM8T+lkm4VqW8IU\\\\\\\/5QmFbsg43irAZfezJlD6P\\\\\\\/90hmC2s+ilHGikSO+n8u\\\\\\\/rmRgGSiikKdEjdYUjY5LXM8LRCANjVqdx12Q2WFRCnH3MtdsXY\\\\\\\/Qhqel9q01tHdha8lxy+SEFq8XanYSJHcsmKhO6EgqwzvesDniiEsa80sbAECr9CtdIkqO6tZS7Fk0be5qZT42D1CNV1JptCORumg9\\\\\\\/znhw50zVfpfRC8uFjlSD6wTniFyTyHYiJtPBVlNRCODD+swfyLiskiI5nlMsoZXgOmnW++WhAcKAMOv5qtcdPWCZJZGRV1aKOEDlAo63EqtoEX7h6LXsSil2EgjFkmqpl0CsOBXNlaLcyDcV01hPatlFQNhD77I6nzAEfu0f6hjiFyVXqNPFgMXCWba3E76JbtdQsEMibzncKZhld+3YcROnH1+PUDZvyy8b9lF3Xz8UHpG5Z9RtPrpOHzoTkUwTotZZwWeRpSUKpEAXReWM5akdITkkPdMB8jOTRFjhmx9d7ofvN\\\\\\\/+Q48RN629\\\\\\\/pn+M1CmvPoGI65l8oXutPcH5T2hja2tKB\\\\\\\/TQIW1lU2M0dxO1wudSe3+Ry4mhxT2APVGNbL+AhKp2oBC9dRGU8znpS4aiOcyekCph9KryqSBIwqBxqvLU86sxJ99rWibWG5XkphzkQO+5OSB\\\\\\\/k39nLPy+4rn+5EDaWiYRoSRr5YQ2Jarn6o5Xpd5bz0e3Qrb7kRbKOQX++zo6wADJIWVWUgzSnvrSQrY7BRDhMLdLO+R3MPcLVn8dC6MQoX15WblurvH5DL8QJakCTW1PKw5aKqXbP6LgdYOA8sA7S4dNDFJw45CRsG+QEU8aj\\\\\\\/ZGimHgx83r3BYYOhp2uxNoV28g5BmCm7adI6MOdDf+e1b2fr5R3bBvDT2HGhzzbEHg8iy1XYbKI7BpmUAzdXSXUfhsMVXzB6ijkM8ubDaZaPsNlWOoIVubzRXIZfkmlFa2gPDot8b+XGwDxFx9jF1TqlA4sKIjGuNiXlJ680JROkU+EIxBKcgXNG8HVY9uTPCfl5SfodLmW1Wof66\\\\\\\/SMDeROUQJupx6Ps8G+ToMosGdLVDeW+pkEPvBMkfugIcyrA4wTzQZ4DVty63d9GaEtthP5U6Sq7TlH1wulId\\\\\\\/JCBz3+NOerjbjKTZ5CMOESPcOHJLyy8aOQgzCWoOwATCH7KkpU1isInanPeJRY\\\\\\\/Fe9Ga81isyki9cA84M27XVwtSaH5IlTOS9c+9bBFskgRZ5273cl6m9hlbqOTrNjyR3GirP7g\\\\\\\/TJTDr51Rka6233GTs4hukvHlpYPYKlso2y\\\\\\\/1Zkg5rWILXJdpe3YMdNclvnAqPezsnEiekQ5eKW+6QMg\\\\\\\/koCDKTsirl6pxPptC7SBRN2LtYTH3nxtQyo2rPC1wLjCy5P9\\\\\\\/9N4uwrcw9GS0+H\\\\\\\/ounmB8gUxu3Z77x+IiGKMONE5XOEIHGPifNhq0gMhBrRu0F0HzwsU1TgbZsAkoPTsQ0h11fL5sMl3m4arV\\\\\\\/jdMPkon\\\\\\\/hNhDnc+xR84a3HznSGL0tq3a1TsdmS6pGzyHD9BYl0TRZDIO7jDfHj16UwV4Kt2Z\\\\\\\/86vhwUUQxjRHxQLFL8g26byRG\\\\\\\/5BHplt+BDJlgyvCmnQi9yztHjga7Fj4qBo83VL2p4ksThAlJJBlD6his3oxoDwkD2H0HmbtKPfYrFz9Kxix+HwNGiKPiWJBepvLGVbUp9R5sl\\\\\\\/25BRdeYI0NWKTXLeBFFAGAj44ZsnGrGHalAg+TkpTxVZV8DDuaXNj8W7tLOM8Ige8n1o\\\\\\\/424E11XXFnA8keVCGI1qDvpBnlP+ZSMXfP98TpxuPH6bzF9UnHflYro30KeU60WnsT7AzzGjs6QgGigHIJF68OKRlVE9YEjrfwquV4lqzJP9DFbNahJ\\\\\\\/CtlKdFcY+8K+58dCluCqZ2XdvvRFRwK09OHb\\\\\\\/vHomtpPFszjxK0p93FmqlIV0\\\\\\\/dhgSOniLTP9o0l2L91M6dPaPKw\\\\\\\/Wgz3Rsoebzb+E+xS781kyqI0tSUQhSFFr+gZ9C5yFT5XWLIsjLUrGiuu0hBTcLT0fCx4n9Zcvy4Q3LABtNjuCS6yFUSQ9K+A5G1mEVi8nUhaBrlMw6ggVb4sCX1VJym4GLMStZFg2RSwL989ZrPhfkNxrYnNuvQKvL2Ld3ok7rJJdJJsjjmg5ZwhzQQ6Eg1ghX9fKB38uNGqfXGins+aHjB1I5bOWprx281HHCan1oZRLrdWu0dweoV8jMF+qRc2QUZlKtfGk2JXY1VDdaPAzMKl4LdIZ99MPCc32WowD0ITXj\\\\\\\/jpovtCAXmW2yc6hoFDAhb7QK4Zx1dwESnJ4EokweIcN58dE++8\\\\\\\/Q9ot0vjdsA4B8aQtM6PCaifQf7h4LFvStrg26Gzx32kXmiMbeN3qB0Ky5hv531Y9coLTmGUSutDNYsZV\\\\\\\/0D37RbFPPkUPqLUFJrWkIQRFiUInePQof3T\\\\\\\/BnDZ2Dg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kDi+URzJIcb2ihEY+CkEeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b967aadfc02b9c339c59c0b3788a0558d51dd63ede621071b9bdcf14c721debf\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pf7TKr8OIu9KB4wiSAevtbR8ZPhZo1k6oEEEjSiR2XLOyCgd2\\\\\\\/xnagJvmBtk4lPigcmh5ZN34QICsfmXJB\\\\\\\/\\\\\\\/dcbd9bHFVwMmaPS1fJdXnnGespoU5jBrg1fFDM010yIjzr4Uf8IUy8diUApDBWoQcgEaiwUA5jq6khTfJd\\\\\\\/Rr9YxT3u562\\\\\\\/YhHNO\\\\\\\/yHOZEqtq9RxgFH1m0fMRnQ+DDvaGrVm7pdMxvOQgqDiEp04oI9A1zbha1ssHMW0d5f38\\\\\\\/3oqx9gWvuqrfCczcBrL9xKV4E\\\\\\\/7sMCm\\\\\\\/V6RzS3taHU4FYWeGEdKpAEmehNPkj7oKopNhe1PbQa2LVykdZn3C0RoQjP2BV7HwaV83M0ZmxBsxLTtHbZqixEgGGEmuZsm\\\\\\\/wJ1hoVEQh4t4R6Y+p1ugW6o85UxMKu4aSgj74ONn52Il980dD1AbVDxg7dDynvelN2bEXbR1EWHJUfayAafHW+8zZtOlrFA2YBzbDM2lM+6J9fITqLo6HtXzWI\\\\\\\/PjRLsDWG8T7tqdjTUBkDWRi6fmRzhH+BbsPrr3qK7mQJ7PeM0e5qoaYA13dpS2Go03OvrhvCArIsOpcmFugBMJEBMkgoYb\\\\\\\/6srOTcY+mPYZcV9vR\\\\\\\/syPWMTORP9Q\\\\\\\/btRB5w859u9BGRnlGewxRmdQWXFat19KCylpGhAC3IUe1+v0IGhaDhOexG1WOUlX6WpWsdQsggVJ+ZNuBzG1B2yvbWQwLT3qqAWul+fM4WYXL5+\\\\\\\/IRFvYN5X\\\\\\\/UsoikAU+nxDFpHUtRe47PKQcU8g4Qyp9yn45836SYRkpH5xs5P\\\\\\\/6gpltr7LxApjNkWpv0Tdvlc529Ox8p7i2TxAnEKg1XgJA0trbQVALD0wUYUQnBejCfCMyrkoXpk5G8sMGycR0ITZ\\\\\\\/oy99NVCVirMbHL50IOwHEargoAObs8Bov+pucZo7YVWyF7NKQUWxoa8HapbEJSRTpXVEha7VJrZeunL0jYTGWoMFnQZpokXfhsOP1dCPRmZ368yEafPun0RYtZNeUM3v6haX2CoaNvekF47hTLPq3x4YVWQNUb7MTwszRCv71Pcg9udQw2v01TDFpMws08WIfRiOc9+nM1Xzk0AtKoZjATc9RHYv7f9Z4YhAh3BItW+lnSGqB4dv9iQQQnFvPVdLNgsRK1\\\\\\\/9dEdXebjoyztlH7uUxBEmQB7En2heeyHfvruTgCIwdFPfV9r3snICGAK0wEdSW2YsHn4YX+aGpVAFLecOyvVLuWe\\\\\\\/6gs5pAHFs\\\\\\\/ChZNdmRiJzfVTBMkGwj58NffXan7h\\\\\\\/7n7TwIUv6kX7STwhxzwRK755sOt0bQCZcoPZ1taThu3RhaYM7OPGMuvxIXq9qrFbq6MBl3PWlGrJmCjYNMWdKN2FeHa0Iy6uge65Q2eO\\\\\\\/BqP0yP3cmewJ0dJvQMlB8F4Y\\\\\\\/htR0yF0ZdBQ7HXLX9lHUbk8SpoEsmHTXFXvWp67n+PYlymF8pwGb9CvdYhRcz6lfA9b8h1CkS639LVXv2Ln8Ls53cm2KHUzUqFurWzCvYxymfhl+KctXwyHWafP0XFBZSY1luONwm3eUE9Spco7HDciXQdprEy88haFn5BNNknIQkydlNYKsXnBOcGK4TO\\\\\\\/q24Jdji6HIIbE3bjOLXoasEdoeaGwJJIBaVIuz4jMl6mqrIwp9j5TZ4Jkyz3i9zs4mQGq+dhxj\\\\\\\/9r3qJxCqaVY6ierG6kcfuzAVm8n3l4jvVedeCLhOxZHL4DrxgSifBWs2wWWjSazFDPBGnEXVLal\\\\\\\/3FudgKtff4Z6z3TP7Ysu+A+snZq39Rvbomw9Wu1QMskfwfbEPUw2008AGP6Vh6UO8UfE9Lk3MafCeXzkY2v35oVTmymXP21aIoDwLMkIumNtPl9bQ3puZ93uaJWuxaQCzB6Q5hArAOtvmenHFuqxHIZfxyfl7Ngtm722crIN5ecw3npp5SeZQMy7\\\\\\\/jCRkjMWDeI7yCa+LNap+rGiF8k\\\\\\\/TqW+O7Nc5BeTVd6KQztncPjkkqxpHON\\\\\\\/aRGE\\\\\\\/dw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tZ7B90gDMgvuku9jAscIXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d61fc92927f429cdd86ed6e51a22dacf76bf20698dcdecce8b5b83bc4e089f20\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494989 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4169" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nGwPJO6aAo3kquzIKqp0mLKFFLMuptDkvSwwC0uPcCA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4169" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VycArciTP_1jnHejMHodOdZ5tSksF7vM0YyfAWH-2W4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4159" title="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4183" title="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494989" id="node494989" data-pisa="ggg;9/12/4169" data-pisa-master="ggg;g3.119.400545" data-apath="/ggg/9/12/4169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of <em>Oryza sativa</em> and <em>Brassica rapa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elise Tourrette</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rex Bernardo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthieu Falque</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier C. Martin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4169-4181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400545 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494989" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elise Tourrette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elise%2BTourrette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tourrette%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElise%2BTourrette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elise Tourrette</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rex Bernardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rex%2BBernardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernardo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARex%2BBernardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rex Bernardo</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthieu Falque</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Olivier C. Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BC.%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20OC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BC.%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier C. Martin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:olivier.c.martin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">olivier.c.martin@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4169" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494989" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494989/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4169.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494989" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494989/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4169.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494989" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494989/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4169.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494989" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494989/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Meiotic recombination generates genetic diversity but in most species the number of crossovers per meiosis is limited. Previous modeling studies showed that increasing recombination can enhance response to selection. However, such studies did not assume a specific method of modifying recombination. Our objective was to test whether two methods used to increase recombination in plants could increase genetic gain in a population undergoing recurrent selection such as in genomic selection programs. The first method, in <em>Oryza sativa</em>, used a mutant of anti-crossover genes, increasing global recombination without affecting the recombination landscape shape. The second one used the ploidy level of a cross between <em>Brassica rapa</em> and <em>Brassica napus</em>, increasing recombination especially in pericentromeric regions. Our modeling framework used these recombination landscapes and sampled quantitative trait loci positions from the actual gene distributions. We simulated selection programs with initially a cross between two inbred lines, for two species. Increased recombination enhanced the response to selection. The amount of enhancement in the cumulative gain largely depended on the species and the number of quantitative trait loci (2, 10, 20, 50, 200 or 1000 per chromosome). Genetic gains were increased up to 30% after 20 generations. Furthermore, increasing recombination in cold regions was the most effective: the gain was larger by 25% with the first method and 34% with the second one in <em>B. rapa</em>, and 12% compared to 16% in <em>O. sativa</em>. In summary, increased recombination enhances the genetic gain in long-term selection programs, with visible effects after four to five generations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-genetics" class="hw-term hw-article-keyword hw-article-keyword-quantitative-genetics" rel="nofollow">quantitative genetics</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">plant breeding</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li></ul><p id="p-3">Recombination is one of the processes generating genetic diversity, creating new <em>allelic combinations</em> through sexual reproduction. Hence, recombination is a cornerstone for obtaining genetic progress during selection. Aside from providing genetic diversity driving genetic progress, recombination is key to a number of practical applications. For instance, recombination intensity controls the resolution of genetic mapping and QTL analyses (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lander and Botstein 1989</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">van Ooijen 1992</a>) and it is particularly important for creating specific mapping populations such as Recombinant Inbred Lines (RILs) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bailey 1971</a> for the development of RILs in mouse, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lister and Dean 1993</a> for the development of RILs in <em>A. thaliana</em>).</p><p id="p-4">Theoretically, it has been shown that recombination is essential for the efficiency of selection, by breaking the linkage between QTL (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hill and Robertson 1966</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Felsenstein 1974</a>). Indeed, when two QTL are linked, they tend to be inherited together. If the alleles of these QTL have opposite effects on the traits of interest, that can impede the efficiency of selection. Such negative linkage disequilibrium can be generated in populations due to selection (Bulmer effect, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bulmer 1971</a>) or genetic drift (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barton 2009</a>).</p><p id="p-5">A consequence of increased recombination during recurrent selection is the faster accumulation of advantageous mutations in evolving populations. In the limit, for instance, of a population without recombination, it is difficult to get rid of deleterious mutations even in the presence of selection (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Muller 1964</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Felsenstein 1974</a>). The relevance of modifying recombination has also been seen in some experimental studies on the rate of response to selection (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">McPhee and Robertson 1970</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zeyl and Bell 1997</a>, see <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Burt 2000</a> for a review, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goddard <em>et al.</em> 2005</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lovell <em>et al.</em> 2014</a>). These studies showed what would happen if one were to suppress recombination. Inversely, could one have an increase in the efficiency of selection by <em>increasing</em> recombination rate? This question is all the more of interest as recombination rate is limited in most species, there almost never being more than three crossovers (COs) per meiosis per chromosome pair regardless of the chromosome physical size (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mercier <em>et al.</em> 2015</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fernandes <em>et al.</em> 2018</a>). Several studies using simulations looked at the effects of increasing recombination on genetic gain during selection programs as follows.</p><p id="p-6"><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McClosky and Tanksley (2013)</a> simulated a plant breeding experiment with a biparental initial population. They first looked at what could be gained by having all loci segregating independently, hereafter referred to as “free recombination”, compared to normal recombination: they found that the average gain was 11%. They then thought about ways to either accumulate recombination events or increase recombination. To accumulate recombination events, they did more rounds of random mating before selection. To increase recombination, they assumed it was a quantitative trait and thus selected higher recombinant individuals in parallel to selecting on their trait of interest. In this way, they found they could attain up to half of the genetic progress that was obtained in the case of free recombination and they concluded that the most effective technique was to perform this simultaneous double selection on both the trait of interest and recombination rate. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Battagin <em>et al.</em> (2016)</a> simulated a livestock breeding experiment in which there were two successive phases, with first the production of a historical population to get a genetic structure similar to that of an actual population used for breeding, and then a breeding program based on that population. They looked at different values for the recombination rates by increasing the genetic lengths of the chromosomes. For a twofold increase of the genetic lengths, they found that the gain was 12.5% higher when considering a program covering 40 generations, for a 10-fold increase the gain was 28.7% higher while for a 20-fold increase the gain was 33.4% higher. Hence, to get a high increase of the gain, they had to increase the recombination rate by 10 to 20-fold. The study by Battagin <em>et al.</em> found a higher increase than the one by McClosky and Tanksley, which could be explained by the population structure (initial diversity) or by the nature of the selection program (in livestock, distinction between males and females, selection on the males only). Lastly, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gonen <em>et al.</em> (2017)</a>, while not interested in increased recombination rate, looked at shifting recombination hotspots to have access to more allelic variations. They observed an increase in the genetic gain when shifting QTL from outside to inside hotspots. That led to a similar conclusion as the previous studies, namely that it may be beneficial to increase recombination rates everywhere in the genome.</p><p id="p-7">Since these simulation studies were performed, different methods of modifying recombination rate along the genome have been developed (see <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blary and Jenczewski (2019)</a> for a review of the different methods). By modifying recombination, we mean either globally increasing the recombination rate (more relevant to the first two simulation studies) or modifying the recombination landscape (more relevant to the third simulation study mentioned above). A first method, the so-called HyperRec technology, is based on suppressing genes that have an anti-crossover role. To date, three associated pathways have been used, corresponding to the knock-out of three genes: <em>FANCM</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Crismani <em>et al.</em> 2012</a>), <em>RECQ4</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Séguéla-Arnaud <em>et al.</em> 2015</a>) and <em>FIGL1</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Girard <em>et al.</em> 2015</a>). When these genes are knocked out, one sees major increases in the number of crossovers, without changing much the recombination landscape, be it in <em>A. thaliana</em> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fernandes <em>et al.</em> 2018</a>), pea, rice or tomato (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mieulet <em>et al.</em> 2018</a>). The increase in recombination rate is many-fold: for instance, the double mutant <em>recq4 figl1</em> showed 7.8-fold increase in the number of COs compared to the wild type in <em>A. thaliana</em>. Another situation leading to increased recombination has been discovered in polyploids. Specifically, it has been found that the ploidy levels in <em>Brassica</em> species have a strong influence both on global recombination rates and on recombination landscapes. For example allotriploid hybrids AAC (single copy of the C genome), obtained by crossing <em>B. rapa</em> (diploid AA) with <em>B. napus</em> (allotetraploid AACC), produced crossovers in the A genome at rates about three times those arising in <em>B. rapa</em> diploids. More interestingly, it led to a tremendous increase in the recombination rate within the pericentromeric regions where neither <em>B. rapa</em> nor <em>B. napus</em> recombine (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Leflon <em>et al.</em> 2010</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Suay <em>et al.</em> 2014</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> 2017</a>). This is potentially important for breeding applications because genes are present in these regions (although gene density is less than in other parts of the genome). For instance, multiple QTL relevant to breeding are localized in bread wheat centromeres (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Griffiths <em>et al.</em> 2012</a> for crop height, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Choulet <em>et al.</em> 2014</a>).</p><p id="p-8">Other methods for increasing recombination rates include genome editing to target COs to specific genomic sites (see <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Filler-Hayut <em>et al.</em> 2017</a> for an application of genome editing in tomato, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bernardo 2017</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ru and Bernardo 2019</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brandariz and Bernardo 2019</a>, for assessing the potential use of targeted recombination in plant breeding via simulations) or over-expressing pro-crossover genes such as HE10 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ziolkowski <em>et al.</em> 2017</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Serra <em>et al.</em> 2018</a>). It is also known that modifications of the growth environment can affect recombination rates. For example, recombination rates depend on the temperature (see <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Phillips <em>et al.</em> 2015</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lloyd <em>et al.</em> 2018</a> for examples in barley and <em>A. thaliana</em>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Modliszewski and Copenhaver 2017</a> for a review on different stresses affecting recombination). There is also some variability across individuals within a given species, for instance associated with the sex context of meiosis (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Giraut <em>et al.</em> 2011</a>, <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Phillips <em>et al.</em> 2015</a> for examples in <em>A. thaliana</em> and barley respectively) or coming from genetic differences between the individuals as revealed by intra-specific variability of recombination rates and landscapes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Salomé <em>et al.</em> 2011</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bauer <em>et al.</em> 2013</a>, for examples in <em>A. thaliana</em> and maize respectively).</p><p id="p-9">In this study, we investigate to what extent modified recombination can be used in breeding programs to maintain genetic diversity and increase genetic gain, considering the first two experimental methods mentioned above, that is (i) using mutants of anti-crossover genes and (ii) changing ploidy levels in <em>Brassiceae</em>. For that we work with the actual recombination profiles experimentally observed with each of these two methods. For our simulations, we expand on the approach used by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">McClosky and Tanksley (2013)</a>, using a biparental population and parameters in a range similar to the ones they used. We investigate the effects of a number of parameters, in particular the number of QTL (2, 10, 20, 50, 200 and 1000 per chromosome), the presence of coupling or repulsion between QTL, crossover interference, and the level of trait heritability (h<sup>2</sup> = 0.2, 0.5, 0.8 and 1). Finally, instead of using only the phenotype for selection, we also test the consequences of modified recombination when using genomic selection.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">We investigated the effects of modifying recombination on genetic gain in selection programs by mathematical and computer modeling. All our codes were written in the programming language R (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">R Core Team 2018</a>), both for producing individual-based simulations of forward-in-time selection programs and for all the associated analyses.</p><div id="sec-2" class="subsection"><h3>Experimental landscapes for the HyperRec and boosted recombination approaches</h3><p id="p-11">The normal and modified recombination landscapes used in our simulations were taken from the published results in <em>Brassica rapa</em> (turnip, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> 2017</a>) and <em>Oryza sativa</em> (rice, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Mieulet <em>et al.</em> 2018</a>). In the case of <em>B. rapa</em>, the modified recombination landscape leads to about 20-fold increased recombination rates in the pericentromeric regions which are otherwise poor in crossovers. For <em>O. sativa</em>, the recombination rate is globally increased but without significant changes in the recombination profile, the pericentromeric regions still being poor in crossovers.</p><p id="p-12">For <em>B. rapa</em>, the modified recombination was called boosted recombination and for <em>O. sativa</em> it was called HyperRec (HR) recombination. As <em>O. sativa</em> and <em>B. rapa</em> had different genetic parameters (number of chromosomes, genetic lengths), a fair comparison of the boosted and HR approaches requires one to use the two methods in each species separately. Since there are no experimental measurements of recombination landscapes in <em>B. rapa</em> under HR nor in <em>O. sativa</em> under boosted recombination, those two landscapes had to be produced <em>in silico</em>. Our procedure for doing so is explained in the paragraph “<em>Construction of the HyperRec recombination landscape in B. rapa and the boosted recombination landscape in O. sativa</em>” (see further, toward the middle of this Materials and Methods section). The three recombination landscapes (normal, boosted and HR) are presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for <em>B. rapa</em> and in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> for <em>O. sativa</em>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="B. rapa recombination landscape. Recombination landscape for each chromosome of B. rapa for normal (black), HR (green) and boosted (red) recombination. The HR landscape was defined so that its profile was the same as for normal recombination, while the normal and boosted landscapes were from Pel&#xE9; et al. (2017). In blue, B. rapa centromere positions from Mason et al. (2016)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;B. rapa&lt;/em&gt; recombination landscape. Recombination landscape for each chromosome of &lt;em&gt;B. rapa&lt;/em&gt; for normal (black), HR (green) and boosted (red) recombination. The HR landscape was defined so that its profile was the same as for normal recombination, while the normal and boosted landscapes were from Pel&#xE9; &lt;em&gt;et al.&lt;/em&gt; (2017). In blue, &lt;em&gt;B. rapa&lt;/em&gt; centromere positions from Mason &lt;em&gt;et al.&lt;/em&gt; (2016).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F1.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4169/F1.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495099" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child"><em>B. rapa</em> recombination landscape. Recombination landscape for each chromosome of <em>B. rapa</em> for normal (black), HR (green) and boosted (red) recombination. The HR landscape was defined so that its profile was the same as for normal recombination, while the normal and boosted landscapes were from <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> (2017)</a>. In blue, <em>B. rapa</em> centromere positions from <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mason <em>et al.</em> (2016)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="O. sativa recombination landscape. Recombination landscape for each chromosome of O. sativa for normal (black), HR (green) and boosted (red) recombination. The boosted recombination landscape was defined via an additive increase everywhere while the normal and HR landscapes were from Mieulet et al. (2018). In blue, the centromere positions taken from Mizuno et al. (2018)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;O. sativa&lt;/em&gt; recombination landscape. Recombination landscape for each chromosome of &lt;em&gt;O. sativa&lt;/em&gt; for normal (black), HR (green) and boosted (red) recombination. The boosted recombination landscape was defined &lt;em&gt;via&lt;/em&gt; an additive increase everywhere while the normal and HR landscapes were from Mieulet &lt;em&gt;et al.&lt;/em&gt; (2018). In blue, the centromere positions taken from Mizuno &lt;em&gt;et al.&lt;/em&gt; (2018).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F2.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4169/F2.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child"><em>O. sativa</em> recombination landscape. Recombination landscape for each chromosome of <em>O. sativa</em> for normal (black), HR (green) and boosted (red) recombination. The boosted recombination landscape was defined <em>via</em> an additive increase everywhere while the normal and HR landscapes were from <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Mieulet <em>et al.</em> (2018)</a>. In blue, the centromere positions taken from <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mizuno <em>et al.</em> (2018)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>QTL effects</h3><p id="p-15">For each chromosome, we allowed for 2, 10, 20, 50, 200, or 1000 loci carrying biallelic QTL affecting the trait considered. We considered these different cases for the numbers of QTL to test different genetic architectures underlying our trait.</p><p id="p-16">The amplitude of each QTL effect was drawn from a gamma distribution; our choice for the shape parameter <em>k = 0.4</em> leads to an L-shaped distribution so that most of the QTL had a weak effect (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hayes and Goddard 2001</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Technow <em>et al.</em> 2012</a>). The gamma distribution’s scale parameter θ was chosen such that the variance of this QTL effect (equal to <em>kθ<sup>2</sup></em>) was one and thus <em>θ = 1.58</em>. As the population is biallelic, each allelic effect was equal to half the QTL’ amplitude. A sign was then randomly assigned for one allelic effect and the opposite sign was taken for the other one. The distributions of absolute allelic effects for 2, 10, 20, 50 and 200 QTL per chromosome are shown in FigS1.</p></div><div id="sec-4" class="subsection"><h3>Allowing for coupling or repulsion between QTL</h3><p id="p-17">Recombination should benefit selection by breaking the linkage between the QTL having an opposite allelic effect on the trait (repulsion case). On the contrary, recombination is expected to be detrimental to selection by breaking the positive correlations between two QTL having both positive or both negative effects (coupling case). Hence, we also looked at the effect of coupling or repulsion on the efficiency of boosted recombination. To model the coupling (respectively repulsion), we introduced a probability of changing (respectively keeping) the sign of the allelic effect when going from one QTL to the next as one goes along the chromosome of a founding parent. This probability <em>P</em> is equal to <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4169/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4169/embed/mml-math-1.gif"/></noscript></span></span>, <em>∆x</em> being the genetic distance between the two QTL and <em>L</em> being the characteristic genetic distance over which coupling (respectively repulsion) is significant. The larger <em>L</em>, the further one will maintain high correlations between allelic signs. Under coupling, <em>P</em> is the probability that the sign will <em>change</em> between adjacent QTL; under repulsion, <em>P</em> is the probability that the sign will stay the same. When <em>L</em> tends to infinity or <em>∆x</em> tends to 0, <em>P = 0</em>. On the other hand, when <em>L</em> tends to 0 or <em>∆x</em> tends to infinity, <em>P = 1/2</em> so that there is no coupling nor repulsion between adjacent QTL and the sign of an allele will be assigned independently of the sign of the allele for the previous QTL. In our situation, we chose a characteristic length <em>L = 5 cM</em>, which resulted in moderate coupling/repulsion. Such levels were undone rather quickly under increased recombination.</p></div><div id="sec-5" class="subsection"><h3>QTL positions</h3><p id="p-18">The positions of the QTL were drawn independently with a density proportional to the gene density of each species (taking gene distributions without transposable elements; see FigS2 for the gene distribution of <em>B. rapa</em> and FigS3 for <em>O. sativa</em>). Gene physical positions were downloaded from public databases (for <em>O. sativa</em>: <a href="https://rapdb.dna.affrc.go.jp/download/irgsp1.html">https://rapdb.dna.affrc.go.jp/download/irgsp1.html</a>, file “Gene structure information in GFF format” for the computationally predicted genes; and for <em>B. rapa</em>: <a href="http://brassicadb.org/brad/datasets/pub/BrassicaceaeGenome/Brassica_rapa/V3.0/">http://brassicadb.org/brad/datasets/pub/BrassicaceaeGenome/Brassica_rapa/V3.0/</a>, file Brapa_genome_v3.0_genes.gff3.gz). QTL physical positions were thus <em>sampled</em> at random among the gene physical positions, for each replicate of our simulations. As only genetic positions in centiMorgans were used during simulations, we had to convert the QTL physical positions into their genetic positions by using the genetic map for each situation (normal, HR, and boosted recombinations for both <em>O. sativa</em> and <em>B. rapa</em>).</p></div><div id="sec-6" class="subsection"><h3>Construction of the HyperRec recombination landscape in B. rapa and the boosted recombination landscape in O. sativa</h3><p id="p-19">The genetic maps of <em>O. sativa</em> for the wild type (WT, with normal recombination) and <em>Osrecq4</em> mutant (HR recombination) were set using marker genetic and physical positions given as supplementary material in <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Mieulet <em>et al.</em> (2018)</a>. The ones for <em>B. rapa</em> A<sub>n</sub>A<sub>r’</sub> (female) diploid (normal recombination) and A<sub>n</sub>A<sub>r’</sub>C<sub>n</sub> allotriploid (boosted recombination) were set using the marker physical positions taken from <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> (2017)</a> and the genetic positions kindly communicated by Pelé.</p><p id="p-20">Since the experimental landscapes of boosted recombination in <em>O. sativa</em> and HR recombination in <em>B. rapa</em> have not yet been measured experimentally, they were produced <em>in silico</em>. The boosted recombination landscape was defined by <em>adding</em> a constant to the recombination rate (in cM/Mb) of the normal recombination landscape (same additive increase of recombination everywhere along the chromosome). This constant was <em>(GL<sub>HR</sub> – GL<sub>NR</sub>)/PL</em>, with <em>GL<sub>HR</sub></em> and <em>GL<sub>NR</sub></em> being the genetic lengths in centiMorgans under HR and normal recombination, and <em>PL</em> being the physical length of the chromosome in megabases. Analogously, the HR recombination landscape was defined by <em>multiplying</em> the normal recombination rate by a constant, <em>GL<sub>BR</sub>/GL<sub>NR</sub></em>, with <em>GL<sub>BR</sub></em> and <em>GL<sub>NR</sub></em> being the genetic lengths under boosted and normal recombination (thus the increase of recombination is proportional to the initial recombination). Both of these constants were chosen to obtain, for each chromosome, the same total increase of recombination for HR and boosted recombination compared to normal recombination.</p><p id="p-21">Given that the landscapes are specified experimentally by markers, it is necessary to interpolate between them to be able to simulate crossovers at arbitrary positions; for that, the relationships between genetic and physical positions were modeled using splines (R function smooth.spline with spar = 0.1 to remove noise in the data), from which QTL genetic positions were obtained. Gene physical positions used for sampling were restricted to the physical range of the genetic map as we do not have the relation between genetic and physical positions outside of this mapped region.</p><p id="p-22">We took the same QTL <em>physical</em> positions and effects for normal and modified recombination, only changing the simulated QTL <em>genetic</em> positions based on the different recombination profiles (our simulations operate in genetic space).</p><p id="p-23">The genetic lengths used in the different situations were given by the genetic maps and are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495109/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495109/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genetic lengths, in centiMorgans, for the 12 chromosomes of <em>O. sativa</em> and the 10 chromosomes of <em>B. rapa</em> under normal and increased recombination (HR and boosted)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Simulating crossover formation</h3><p id="p-24">Meiotic recombination was simulated by generating crossovers in genetic space. The number of crossovers was drawn from a Poisson distribution, as occurs in the no-interference model of Haldane (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">McPeek and Speed 1995</a>). The genetic distance between two adjacent crossovers was drawn from an exponential distribution, with an average of 1 crossover per 100 cM.</p><p id="p-25">We also performed a study including crossover interference which was modeled following the single-pathway (only interfering crossovers) Gamma model (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">McPeek and Speed 1995</a>), using a gamma distribution for the distance between adjacent crossovers. Hence, using this model with interference, it is possible to modify the “repulsion” of crossovers generated within a single meiosis, the strength of which depends on the <em>nu</em> parameter of the distribution. When this parameter is equal to 1, it is equivalent to the model of Haldane without interference. The values of the <em>nu</em> parameter for each chromosome in the diploid and in the allotriploid hybrid of <em>B. rapa</em> (normal and boosted recombination) were taken from <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> (2017)</a>. The values are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table2</a>. In <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> (2017)</a>, interference was estimated using two-pathways instead of the single-pathway. Contrary to the single-pathway, both interfering and non-interfering crossovers are taken into account for the estimation of the <em>nu</em> parameter by those authors. As a result, by using the interference parameters of the two-pathway model in our single pathway approach, we probably overestimated the effect of interference (<em>nu</em> parameters higher than they should be).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495086/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495086/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Values of the <em>nu</em> parameters of the gamma distribution controlling the level of interference for the diploid (normal recombination) and allotriploid hybrids (boosted recombination) of <em>B. rapa</em>.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Neutral markers</h3><p id="p-26">We also introduced 1000 biallelic neutral markers per chromosome (having no effect on the trait). They were taken to be equally spaced, the genetic distance between adjacent markers being given by the ratio between the chromosome’s genetic length and the number of neutral markers minus 1. The number of these markers was kept constant while the genetic lengths changed according to the modified or not recombination landscapes.</p></div><div id="sec-9" class="subsection"><h3>The quantitative genetics framework and selection schemes</h3><p id="p-27">For each individual we denote its genotypic value by <em>G</em>. As only additivity was considered, <em>G</em> was equal to the allelic effects present in the considered individual, summed over all QTL and thus over all pairs of homologous chromosomes.</p><p id="p-28">The initial population, biparental, was generated from two homozygous lines P1 and P2 (generation <em>g = 0</em>), with P1 having one of the two alleles for each QTL and P2 the other one. P1 and P2 were crossed to get an F1 individual (generation <em>g = 1</em>).</p><p id="p-29">Two different selection schemes were tested: a “Fn selection scheme” and a “DH selection scheme”.</p><p id="p-30">In the Fn selection scheme, the F1 was selfed to produce a population of 250 F2 individuals (<em>g = 2</em>). From this F2 generation, each further generation was obtained via (i) a step of “truncation selection” to keep the “best” individuals (see below for the criteria used) within the population based on their breeding value, and (ii) a step of random mating between these selected individuals to produce the offspring that form the individuals of the next generation. We simulated this selection program over a total of 20 generations. The populations had a constant size of 250 individuals and the selection pressure was 2%, meaning that only the 5 best individuals were kept to create the next generation.</p><p id="p-31">In the DH selection scheme, the F1 was used to get 250 doubled haploids (DH, <em>g = 2</em>), among which the 5 best individuals were then selected and crossed to get a new heterozygous population of 250 individuals. Each one of these individuals was then used to produce one DH, and among these DHs the best ones were again selected and crossed for the next generation. These steps, producing respectively homozygous and heterozygous populations, were alternated during 20 generations, corresponding to 10 cycles of selection. The selection intensity was the same as in the Fn selection scheme, <em>i.e.</em>, 5 individuals among the 250 and the population size of 250 individuals was also constant.</p><p id="p-32">One cycle of the Fn selection scheme is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> while one cycle of the DH selection scheme is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. As we performed up to 20 generations of selection, we also considered the selection of 10 and 20 individuals among the 250 (selection intensity of 4% and 8% respectively) to include more realistic situations and study the influence of selection intensity.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of one cycle of selection in the Fn selection scheme." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of one cycle of selection in the Fn selection scheme.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F3.medium.gif" width="440" height="71"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4169/F3.medium.gif" width="440" height="71"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495083" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Schematic representation of one cycle of selection in the Fn selection scheme.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of one cycle of selection in the DH selection scheme." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic representation of one cycle of selection in the DH selection scheme.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F4.medium.gif" width="440" height="55"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4169/F4.medium.gif" width="440" height="55"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495016" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Schematic representation of one cycle of selection in the DH selection scheme.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">We considered two selection criteria where the breeding value corresponded either to the phenotypic value <em>P</em> (in the case of phenotypic selection) or to the genomic estimated breeding value <em>GEBV</em> (in the case of genomic selection).</p><p id="p-36">For phenotypic selection, <em>P</em> was calculated as the genotypic value <em>G</em> plus an environmental noise <em>E</em>. E was taken from a normal distribution with null mean and variance <em>V<sub>E</sub></em>. To set <em>V<sub>E</sub></em>, we first estimated the genetic variance <em>V<sub>G</sub></em> in the F2 population, and calculated <em>V<sub>E</sub></em> to get the desired value of the heritability defined as <em>h<sup>2</sup> = V<sub>G</sub>/V<sub>P</sub> = V<sub>G</sub>/(V<sub>G</sub>+V<sub>E</sub>)</em> (V<sub>P</sub> being the phenotypic variance). Four different values of <em>h<sup>2</sup></em> were tested: 1, 0.8, 0.5 and 0.2. Note that h<sup>2</sup> is defined based on the F2 population, the later populations having different variances also had different values of <em>h<sup>2</sup></em>.</p><p id="p-37">In the case of genomic selection, <em>GEBV</em> was calculated using the estimated marker effects and the population mean. Both were estimated in the F2 population and then every fourth generation (training populations) with ridge regression best linear unbiased prediction (RR-BLUP) using the rrBLUP R package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Endelman 2011</a>). This corresponds to the mixed model <em>Y = Xβ + Zu + e</em>, where <em>Y</em> is the vector (Nx1) of phenotypic values calculated as for phenotypic selection, <em>X</em> a vector (Nx1) of 1s, <em>Z</em> the genotype matrix (NxN<sub>marker</sub>) and e the vector of residuals. The genotype matrix was coded as follows: <em>Z<sub>ij</sub> = -1</em> if the marker j of individual i is homozygous for the allele of parent 1, <em>Z<sub>ij</sub> = 1</em> if it is homozygous for the allele of parent 2, and <em>Z<sub>ij</sub> = 0</em> if it is heterozygous. The population mean <em>β</em> and the vector of marker effects u were then estimated and used in the next generations to calculate <em>GEBV = β + Z<sub>g</sub>u</em>, where <em>Z<sub>g</sub></em> was the genotype matrix of the generation g, coded as previously described.</p><p id="p-38">An important consideration in genomic selection is the frequency with which one re-calibrates the estimated marker effects along generations of selection. In this work we considered three such cases: a single calibration arising in the F2 generation, calibrations every fourth generation and calibrations every second generation.</p></div><div id="sec-10" class="subsection"><h3>Parameter choices</h3><p id="p-39">Clearly, many parameters values had to be set in any framework such as ours. In the Results section, unless otherwise stated, the parameters used were those of the Fn selection scheme, under genomic selection with the marker effects estimated every fourth generation. Furthermore, we used 200 QTL per chromosome and a trait heritability of 0.5, and no crossover interference. The selection intensity was set to 2% and we imposed no coupling or repulsion. As justified in the Results section, the analyses were mainly focused on the species <em>B. rapa</em>. To understand the role of the different parameters in our framework, we considered them one at a time. Specifically, starting with the default values just given above, we saw what happened when changing one of our choices, such as the number of QTL or the frequency of calibration of the genomic selection model.</p></div><div id="sec-11" class="subsection"><h3>Using replicates in the simulations and statistical analyses</h3><p id="p-40">For each generation, the genetic gain was calculated as the difference between the population’s mean genotypic value at generation g and at generation 2 (first generation subject to selection) <em>i.e.</em>, <em>G<sub>g</sub></em> - <em>G<sub>g</sub></em> <sub>=</sub> <em><sub>2</sub></em> where <em>G<sub>g</sub></em> denotes the population average of the genotypic values at generation <em>g</em>. Note that for the same species and same number of QTL, the genetic variance at generation 2 was identical whatever the level of recombination. Thus, we did not need to normalize the difference of genotypic values by dividing it by the genetic variance at generation 2. We also calculated at each generation the genetic variance of the population as the variance of the vector of genotypic values. The <em>efficiency</em> of having modified recombination was calculated as the ratio of genetic gains: <em>G<sub>modified recombination</sub>/ G<sub>normal recombination</sub></em>. We use this ratio when comparing situations having different genetic variances at generation 2 (<em>e.g.</em>, when considering the effect of varying the number of QTL).</p><p id="p-41">To assess the stochastic variability of the process, we used 1000 replicates for each study (choice of selection program and parameters) and calculated, for each replicate, the genetic gain, genetic variance and relative gain. Confidence intervals were then calculated via the standard error on the mean values of these quantities, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4169/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4169/embed/mml-math-2.gif"/></noscript></span></span>.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-42">The scripts for the simulations are available as a R package on <a href="https://sourcesup.renater.fr/frs/download.php/latestfile/2217/CAREB_R.tar.gz">https://sourcesup.renater.fr/frs/download.php/latestfile/2217/CAREB_R.tar.gz</a>. The supplementary figures S1 to S10 have been deposited on figshare: <a href="https://doi.org/10.25387/g3.8858624">https://doi.org/10.25387/g3.8858624</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><p id="p-43">We hypothesize that increased recombination can be beneficial for selection by (i) breaking linkage between QTL and (ii) providing access to genetic diversity by introducing recombination into cold regions. To test the first hypothesis, we studied different numbers of QTL per chromosome: the more QTL per chromosome there are, the tighter the linkage. Hence, as the number of QTL per chromosome increases, the effect of increased recombination should also increase. Note that we are more interested in the number of QTL per chromosome than in the total number of QTL genome-wide: indeed, recombination arises within and across chromosomes but both HR and boosting act only on the former. To test the second hypothesis, we compared the increase of recombination by HR and boosted recombination, where the first method does not change significantly the recombination landscape (no crossovers in the cold regions in both normal and HR) while the second one changes the recombination landscape and introduces crossovers into the cold regions. Our modeling and simulational approaches are explained in the Methods section; a set of reference parameters are used (<em>e.g.</em>, population size, number of QTL, heritability, level of selection, etc) to illustrate the effects of increased recombination but of course we also explore the roles of these different parameters, showing in particular that the effects are almost always robust to changing the values of the parameters.</p><div id="sec-14" class="subsection"><h3>Increased recombination leads to higher genetic gain</h3><p id="p-44">We find that increasing recombination slows down both the loss of genetic diversity and the decrease of genetic variance (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This results in an increase of the genetic gain and a delay in the fixation of the QTL alleles, the plateau for the genetic gain (due to fixation) being reached later (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic variance for B. rapa and O. sativa for the three different levels of recombination. Genetic variance as a function of generation, for B. rapa and O. sativa under normal (black), HR (green) and boosted (red) recombination for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The first generation shown is the F2 generation (g = 2). The error bars represent 95% confidence intervals on the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic variance for &lt;em&gt;B. rapa&lt;/em&gt; and &lt;em&gt;O. sativa&lt;/em&gt; for the three different levels of recombination. Genetic variance as a function of generation, for &lt;em&gt;B. rapa&lt;/em&gt; and &lt;em&gt;O. sativa&lt;/em&gt; under normal (black), HR (green) and boosted (red) recombination for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The first generation shown is the F2 generation (g = 2). The error bars represent 95% confidence intervals on the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F5.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/12/4169/F5.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495005" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Genetic variance for <em>B. rapa</em> and <em>O. sativa</em> for the three different levels of recombination. Genetic variance as a function of generation, for <em>B. rapa</em> and <em>O. sativa</em> under normal (black), HR (green) and boosted (red) recombination for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The first generation shown is the F2 generation (g = 2). The error bars represent 95% confidence intervals on the mean.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic gain for B. rapa and O. sativa for the three different levels of recombination. Genetic gain (sum of QTL effects) as a function of the generation number, for B. rapa and O. sativa under normal (black), HR (green) and boosted (red) recombination, for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The gain ratios between increased and normal recombination at g = 20 are as follows. In B. rapa, 1.34 for boosted recombination and 1.25 for HR. In O. sativa, 1.16 for boosted recombination and 1.12 for HR. The first generation shown is the F2 generation (g = 2). The error bars represent the 95% confidence intervals on the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic gain for &lt;em&gt;B. rapa&lt;/em&gt; and &lt;em&gt;O. sativa&lt;/em&gt; for the three different levels of recombination. Genetic gain (sum of QTL effects) as a function of the generation number, for &lt;em&gt;B. rapa&lt;/em&gt; and &lt;em&gt;O. sativa&lt;/em&gt; under normal (black), HR (green) and boosted (red) recombination, for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The gain ratios between increased and normal recombination at g = 20 are as follows. In &lt;em&gt;B. rapa&lt;/em&gt;, 1.34 for boosted recombination and 1.25 for HR. In &lt;em&gt;O. sativa&lt;/em&gt;, 1.16 for boosted recombination and 1.12 for HR. The first generation shown is the F2 generation (g = 2). The error bars represent the 95% confidence intervals on the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F6.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/12/4169/F6.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495063" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-46" class="first-child">Genetic gain for <em>B. rapa</em> and <em>O. sativa</em> for the three different levels of recombination. Genetic gain (sum of QTL effects) as a function of the generation number, for <em>B. rapa</em> and <em>O. sativa</em> under normal (black), HR (green) and boosted (red) recombination, for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, 200 QTL per chromosome, crossovers formed without interference, no coupling nor repulsion. The gain ratios between increased and normal recombination at g = 20 are as follows. In <em>B. rapa</em>, 1.34 for boosted recombination and 1.25 for HR. In <em>O. sativa</em>, 1.16 for boosted recombination and 1.12 for HR. The first generation shown is the F2 generation (g = 2). The error bars represent the 95% confidence intervals on the mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">At g = 20, for <em>B. rapa</em>, the genetic gain (sum of the QTL effects) is 237 for normal recombination, to be compared to 313 (respectively 293) for boosted (respectively HR) recombination, leading to a gain ratio of 1.34 (respectively 1.25) when using increased recombination. Similarly, at g = 20, for <em>O. sativa</em>, the genetic gain is 322 for normal recombination and 367 (respectively 356) for boosted (respectively HR) recombination, leading to gain ratios of 1.16 (respectively 1.12) for that species. We can also note that increased recombination has a stronger effect for <em>B. rapa</em> than for <em>O. sativa</em>.</p><p id="p-48">For <em>B. rapa</em>, the allelic fixation arises around g = 15 (null genetic variance) under normal recombination, while the genetic gain continues to increase for boosted and HR recombination at later generations.</p><p id="p-49">Note that the effect of increased recombination on the genetic gain is not immediate. It happens at around g = 7 for <em>B. rapa</em> and g = 11 for <em>O. sativa</em>. Nevertheless, the effect of increased recombination gets stronger with time. At g = 10, the gain ratio is 1.15 for <em>B. rapa</em> under boosted recombination while at g = 20, it is 1.34.</p></div><div id="sec-15" class="subsection"><h3>Allowing recombination in cold regions is most effective</h3><p id="p-50">Under boosted recombination, there is an increase of the recombination rate in the cold regions, while under HR recombination there is no change in the recombination profile and the recombination is thus mainly increased in the hot regions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-51">Increasing recombination in the cold regions gives us access to more genetic diversity compared to normal recombination. As expected, the effect of increased recombination is stronger under boosted recombination than under HR. For instance, in the case of <em>B. rapa</em> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), the gain ratio is 1.34 for the first <em>vs.</em> 1.25 for the second. This differential arises even though the gene density in these cold regions is lower than in the rest of the genome (FigS2).</p><p id="p-52">Given the expectation that increased recombination is most effective if it arises where (normal) recombination rates are lowest, in the following, we choose to concentrate on <em>B. rapa</em> under boosted recombination and compare it to normal recombination.</p></div><div id="sec-16" class="subsection"><h3>Heritability does not affect the advantage of increasing recombination</h3><p id="p-53">Consider now the influence of heritability on genetic gains in our framework. We see in our study of increased recombination that the dependencies on heritability are very small (FigS4), with gain ratios of 1.40, 1.38, 1.32, and 1.24 for h<sup>2</sup> = 1, 0.8, 0.5 and 0.2 at g = 20 in the case of phenotypic selection. The dependencies on heritability are even weaker in the case of genomic selection where the gain ratios are 1.33, 1.37 1.34, and 1.28 at g = 20 for those same heritabilities (FigS5). The smaller effect of heritability under genomic selection compared to phenotypic selection could have been anticipated because genomic selection estimates the genotypic value and thus attenuates effects of environmental noise.</p></div><div id="sec-17" class="subsection"><h3>Increasing recombination is more efficient with many QTL</h3><p id="p-54">We find that increased recombination is more effective for traits controlled by a large number of QTL. For instance, there is essentially no effect when there are just 2 QTL per chromosome, result which can be compared to the gain ratio of 1.34 for 200 QTL per chromosome in the context of <em>B. rapa</em> under boosted recombination at g = 20 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gain ratio for different numbers of QTL. Gain ratio (genetic gain with increased recombination over genetic gain with normal recombination), as a function of the number of QTL per chromosome, for boosted (red) and HR (green) compared to normal recombination, for B. rapa (full lines) and O. sativa (dotted lines) for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, crossovers formed without interference, no coupling nor repulsion. The error bars represent 95% confidence intervals on the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-674121954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gain ratio for different numbers of QTL. Gain ratio (genetic gain with increased recombination over genetic gain with normal recombination), as a function of the number of QTL per chromosome, for boosted (red) and HR (green) compared to normal recombination, for &lt;em&gt;B. rapa&lt;/em&gt; (full lines) and &lt;em&gt;O. sativa&lt;/em&gt; (dotted lines) for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, crossovers formed without interference, no coupling nor repulsion. The error bars represent 95% confidence intervals on the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4169/F7.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/12/4169/F7.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4169/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495106" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-55" class="first-child">Gain ratio for different numbers of QTL. Gain ratio (genetic gain with increased recombination over genetic gain with normal recombination), as a function of the number of QTL per chromosome, for boosted (red) and HR (green) compared to normal recombination, for <em>B. rapa</em> (full lines) and <em>O. sativa</em> (dotted lines) for genomic selection with the marker effects estimated every fourth generation, a heritability of 0.5, selection on heterozygotes with an intensity of 2%, crossovers formed without interference, no coupling nor repulsion. The error bars represent 95% confidence intervals on the mean.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Frequent recalibrations improves the benefit of increasing recombination</h3><p id="p-56">We also considered the influence of the frequency of recalibrating the genomic selection model. Clearly the best results are expected when the model is recalibrated often, and that is what we see (FigS6). But since recalibration requires time and financial resources, the question is to what extent such recalibrations can be spread out in time. In our system, we find that the gain ratio at g = 20 between boosted and normal recombination is 1.27 when the model is calibrated <em>only</em> in the F2 generation, 1.34 when it is recalibrated once every four generations and 1.35 when recalibrated once every two generations. The difference between these last two cases is not significant, leading us to conclude that there is a regime where one can benefit from the advantages of genomic selection yet not pay a too high price.</p></div><div id="sec-19" class="subsection"><h3>Increased recombination is more effective under strong selection</h3><p id="p-57">Depending on the number of generations used in the selection program, we can afford to select more or less strongly. As up to 20 generations were performed, selection intensities of 2% (5 individuals selected at each generation), 4% (10 selected) and 8% (20 selected) were considered (FigS7). In the short-term, higher selection pressures result in higher genetic gains (until generation 7 for normal recombination and generation 10 for boosted recombination). However, fixation happens earlier as a strong selection intensity depletes the diversity faster (fixation at g = 15 under normal recombination with selection at 2%, while fixation is not visible in the other situations). Thus, decreasing the selection intensity and/or increasing recombination delays the fixation. For the last generations, under normal recombination, selecting 4% or 8% of the individuals makes a significant difference while that is not the case under boosted recombination. Hence boosted recombination may be sufficient to conserve a certain level of diversity and thus allow for stronger selection intensity. Increasing recombination is more efficient with a strong selection intensity, the gain ratio between boosted and normal recombination being 1.34 for 2% of selection, 1.30 for 4% and 1.23 for 8% at g = 20.</p></div><div id="sec-20" class="subsection"><h3>Increasing recombination similarly impacts DH and heterozygous selection strategies</h3><p id="p-58">All of the previous results were obtained when the selection was done in heterozygous populations (Fn scheme). But in some crops, selection can be performed on doubled haploids (DH) homozygous individuals. Hence, we asked whether our conclusions on the usefulness of increased recombination might extend to the case of selection using DH populations. We find that, for the same number of generations (but half the number of cycles of DH), there is hardly any difference between the two types of populations under selection: at g = 20, the gain ratio is 1.34 for the Fn selection scheme and 1.31 for the DH selection scheme (FigS8).</p></div><div id="sec-21" class="subsection"><h3>Increased recombination is beneficial under repulsion but can be detrimental under coupling</h3><p id="p-59">So far we supposed a situation without coupling nor repulsion. Since recombination can be expected to be unfavorable in the presence of coupling, it was necessary to quantify the consequences of increased recombination when the alleles were either in coupling or in repulsion (FigS9). As expected, under coupling, increasing recombination is detrimental, with a ratio of the gains between boosted and normal recombination of 0.80, while under repulsion, it is beneficial, with a ratio of 1.38 that is more than the ratio of 1.34 under a random situation (neither coupling nor repulsion). Interestingly, we found that the effect of coupling is strong whereas that of repulsion is weak.</p></div><div id="sec-22" class="subsection"><h3>The benefit of increased recombination is insensitive to the presence of interference</h3><p id="p-60">All of the previous results were produced assuming that there was no interference. Having interfering crossovers will lead to fewer events where two adjacent crossovers are close in the same meiosis but it is plausible that this feature has little consequence when considering multiple generations. And this is what transpires from our simulations: increasing recombination still leads to a higher gain (FigS10). The effect of boosted recombination is slightly weaker when incorporating interference as observed in the experimental data: the ratio of the gains is 1.31 compared to 1.34 when interference is ignored. This result is partly justified by the fact that crossover interference is lower in the allotriploid AAC compared to the diploid AA of <em>B. rapa</em> (lower <em>nu</em> parameter of the gamma distribution, <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> 2017</a>). Another result shown by FigS10 is that genetic gain <em>per se</em> is hardly affected by interference.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><div id="sec-24" class="subsection"><h3>Increased recombination generally enhances genetic gain across generations</h3><p id="p-61">For the range of parameters studied, increased recombination, whether HyperRec or boosted recombination, had a positive effect and led to higher genetic gain than normal recombination.</p><p id="p-62">This effect was not observed immediately but took at least three to four generations. However, it got stronger with time for both boosted and HR recombination, which is concordant with the fact that increased recombination slowed down the loss of genetic variance. The early genetic gains (arising during the first generations) may be driven more by random chromosomal assortment than increased numbers of crossovers (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">McClosky and Tanksley 2013</a>). Nevertheless, under recurrent selection, it is more advantageous to increase recombination at the beginning than at the end since recombination acts only when genetic diversity is present. By increasing recombination at the end of program, the effect, if any, would be marginal as most of the genetic diversity would already have been used up.</p><p id="p-63">In our study, the choice of different parameters was important for the size of the effect produced by increased recombination. Among them, the number of QTL had one of the highest effects. We found that the genetic gain increased with the number of QTL, as had already been observed by others (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">McClosky and Tanksley 2013</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Battagin <em>et al.</em> 2016</a>). Such a behavior is expected as the more QTL one has, the tighter their linkage and the more recombination will be needed to break that linkage. Besides, considering our choice of population size (250 non-independent individuals descending from two homozygous lines), we cannot neglect genetic drift. Hence, the fixation will happen early when there are only a few QTL, independently of the level of recombination, and increased recombination will have less time to act. It is worth noting that the interaction between genetic drift and selection tends to give an advantage to higher recombination (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Otto and Barton 2001</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Comeron <em>et al.</em> 2008</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Webster and Hurst 2012</a>) and thus probably enhances the positive effect of increased recombination. Moreover, having an initial population consisting of two parental lines and having the selection start on the F2 population resulted in a high initial linkage disequilibrium, which could also heighten the effect of increased recombination. The strong selection will also heighten the effect of increased recombination as the genetic diversity will be depleted faster and recombination will be needed to keep a certain level of diversity for the selection to act upon. Increasing recombination is thus more interesting in situations where the genetic diversity is limited (initially or due to genetic drift or selection).</p><p id="p-64">The presence of coupling or repulsion also had a significant effect on the efficiency of increased recombination. This holds true especially for coupling, which we speculate is due to the strong selection intensity (2%). Indeed, under coupling, the selection will result in directly keeping the individuals with large blocks of positive alleles and the gain will have a fast increase. Increasing recombination will break these blocks and lead to a much slower increase of the gain. On the contrary, under repulsion, most of the individuals will have a genotypic value close to zero initially as positive and negative alleles alternate along the chromosomes. Hence, building blocks of positive alleles needs more time, and this is what happens to some extent in the random situation. Nevertheless, in the random situation, some blocks will be in repulsion while others will be in coupling, leading to a higher gain compared to the situation of pure repulsion.</p><p id="p-65">Lastly, the choice of the species also had an impact on the efficiency of modifying recombination.</p><p id="p-66">We observed a greater efficiency of increased recombination in <em>B. rapa</em> compared to <em>O</em>. sativa. Since recombination arises both within and across chromosomes, the number of chromosomes (10 for <em>B. rapa</em> and 12 for <em>O. sativa</em>) will inevitably affect the genetic gain. For instance, in the limit of no intra-chromosomal recombination, it will be advantageous to have more chromosomes. Because of this effect, comparing the genetic gains across these two species is subject to caution. It is possible that the difference in the effect of increased recombination between <em>B. rapa</em> and <em>O. sativa</em> we see is because of the difference in chromosomal genetic lengths, with an average of 80 cM and 275 cM for normal and increased recombination for <em>B. rapa</em> and an average of 145 and 475 cM for normal and increased recombination for <em>O. sativa</em>. For the same number of QTL per chromosome, the QTL in <em>B. rapa</em> will be more linked compared to <em>O. sativa</em> and so it is plausible that <em>B. rapa</em> will respond more strongly to increased recombination. The maximum values of the gains being limited by the alleles present in the population, the gain ratios cannot increase indefinitely with increased recombination; for instance the gain ratio between very high and simply high recombination will be close to 1. Thus, the initial level of recombination can be expected to significantly affect the gain ratios.</p><p id="p-67">When considering the other parameters (heritability, selection scheme and types, crossover interference) aside from the number of QTL, we found that they had only a modest effect on the gain ratio. In the case of heritability, this was already observed for phenotypic selection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo 2009</a>, <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">McClosky and Tanksley 2013</a>). In the case of interference, our single pathway modeling overestimated its strength (see Material and Method section) and thus the actual effects of interference are probably even weaker than what is shown in FigS10.</p><p id="p-68">Overall, increasing recombination in plant breeding programs should increase the genetic gain and retain genetic diversity longer. This is particularly true in populations with restricted initial diversity or subject to selection schemes depleting the diversity at a high rate (for instance when there is strong selection).</p></div><div id="sec-25" class="subsection"><h3>Consequences of increasing recombination in cold regions of the recombination landscape</h3><p id="p-69">The highest gain ratio occurred when the regions that are normally cold for recombination were transformed into warm regions. This enhancement to the genetic gain occurred even though the gene density is low in those regions. This result is coherent with the observations made by <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gonen <em>et al.</em> (2017)</a> on the usefulness of shifting recombination hotspots toward the cold regions of the landscapes. Indeed, the absence of recombination in some regions will impede selection there (Hill-Robertson effect, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hill and Robertson 1966</a>). By significantly increasing recombination in cold regions, the efficiency of selection in those regions and thus the total effect of selection will increase. It follows that the effect of increased recombination will be most favorable when that increase occurs in cold regions, as observed in our simulations. The major cold regions in many crops are the pericentromeric ones. In fact, these regions can extend to half of a chromosome’s physical length and are almost completely devoid of crossovers (<em>cf</em>. examples by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sherman and Stack 1995</a> in tomato, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson <em>et al.</em> 2003</a> in maize and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Choulet <em>et al.</em> 2014</a> in wheat). Our modeling thus predicts that increasing the number of crossovers in these regions will lead to significant gains. Note that these crossovers must not be too close to the centromeres if one is to avoid segregation problems and thus loss of fertility (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Talbert and Henikoff 2010</a>). Such problems might explain the difference of viability and number of seed per plants observed by <a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Pelé <em>et al.</em> (2017)</a> in the allotriploids (modified recombination landscape) compared to the diploids. On the other hand, with the HR technology, which almost does not change the recombination landscape and in particular does not put additional crossovers in the cold regions, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Fernandes <em>et al.</em> (2018)</a> did not observe growth or developmental defects in <em>A. thaliana</em> but did observe some fertility defects, which were not correlated with the level of increase of recombination.</p></div><div id="sec-26" class="subsection"><h3>Feasibility of increased recombination</h3><p id="p-70">If increased recombination is to be used in practice, different questions will arise such as when and how should it be applied, and is it worth the cost of doing so? Increasing recombination would incur additional cost, such as producing the mutant(s) for HyperRec or the needed crosses for boosted recombination. As some fertility defects may be expected with increased recombination, one may need to increase the population sizes compared to normal recombination. While HyperRec has been developed in several crops such as pea and tomato (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Mieulet <em>et al.</em> 2018</a>), with varying degrees of increased recombination, boosted recombination has so far only been obtained in <em>B. rapa</em>. It is also possible to consider other means for increasing recombination such as controlling the growth temperature; however such approaches have led so far to only quite modest increases in recombination rates. One could also try to generate increased recombination through parallel selection for plants with greater number of crossovers as suggested by <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">McClosky and Tanksley (2013)</a> but to date there is little experimental evidence that this approach can be effective. Regardless of the approach chosen, our modeling predicts that there will be significant benefits for breeding only if the recombination rate is increased many-fold and if the selection is applied during enough generations. Depending on the number of generations wanted, as well as other parameters such as trait architecture, it may not be worthwhile to introduce increased recombination using the kind of methodologies we tested. Instead, it may be more cost effective to resort to methods that have the potential to target recombination to precise locations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bernardo 2017</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ru and Bernardo 2019</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brandariz and Bernardo 2019</a>). Such approaches are most promising when the trait of interest is controlled by a major cluster of QTL or when the goal is to produce the proper recombinants in just one generation. However, in this situation of targeted recombination, the precise location of the crossover(s) needs to be determined beforehand. Finally, even though our modeling and simulations indicate how one may best exploit increased recombination (small populations, strong selection intensities, traits controlled by numerous QTL), it may prove difficult to provide quantitatively reliable predictions because the genetic architecture is typically uncertain and the importance of genetic and environmental interactions that were ignored in our study are generally unknown.</p></div><div id="sec-27" class="subsection"><h3>Consequences for genomic selection programs</h3><p id="p-71"><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Battagin <em>et al.</em> (2016)</a> wondered about the use of increased recombination for genomic selection programs and speculated that large datasets would be needed. Indeed, the calibration of genomic selection models is dependent on genomic blocks shared across individuals in the population; increasing recombination reduces the sizes of these blocks. Consequently, the calibration of the genomic selection model is made more difficult and more importantly, the predictive power of the model decreases faster as one accumulates crossovers throughout generations. Nevertheless, our results show a benefit of increased recombination for genomic selection programs even when using a relatively modest training population of 250 individuals. We did observe as anticipated that the accuracy of genomic prediction was reduced under increased recombination (faster break-down of the linkage blocks). But we speculate that under higher recombination the slower loss of genetic variance may compensate the loss of prediction accuracy over generations, justifying the higher genetic gain for boosted and HR recombination than for normal recombination. As <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Battagin <em>et al.</em> (2016)</a> expected, increased recombination generated smaller linkage blocks. This led to the estimation of marker effects being closer to the true marker effects compared to normal recombination even though, in the generation used for the calibration, the prediction of genomic value was lower than for normal recombination. In effect, it was the greater reliability of the predictions <em>at the level of the markers</em> that resulted in the better genomic predictions at later generations when using increased recombination. This was particularly striking in our simulations when no re-estimation was done during selection, the prediction relying entirely on the calibration produced at the F2 generation.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We thank S. Mezmouk, E. Jenczewski, J.C. Motamayor, D. Livingston, C.E. Durel and R. Mercier for constructive comments and discussion. This work has benefited from a French State grant (LabEx Saclay Plant Sciences-SPS, ANR-10- LABX-0040-SPS), managed by the French National Research Agency under an “Investments for the Future” program (ANR-11-IDEX-0003-02) and KWS, MARS, and Secobra which co-funded the salary of ET and Ecole Doctorale FIRE - Program Bettencourt for support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-73">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8858624">https://doi.org/10.25387/g3.8858624</a>.</p></li><li class="fn" id="fn-2"><p id="p-74"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-15"><span class="received-label">Received </span>July 15, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-15"><span class="accepted-label">Accepted </span>October 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Tourrette <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. G.</span> <span class="cit-name-surname">Doyle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Brigham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Carter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">Hooker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> High-resolution crossover maps for each bivalent of Zea mays using recombination nodules. Genetics165: 849–865.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.2" data-doi="10.1097/00007890-197103000-00013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Recombinant inbred strains.</span> <abbr class="cit-jnl-abbrev">Transplantation</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1097/00007890-197103000-00013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransplantation%26rft.stitle%253DTransplantation%26rft.aulast%253DBailey%26rft.auinit1%253DD.%2BW.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D327%26rft.atitle%253DRecombinant-inbred%2Bstrains.%2BAn%2Baid%2Bto%2Bfinding%2Bidentity%252C%2Blinkage%252C%2Band%2Bfunction%2Bof%2Bhistocompatibility%2Band%2Bother%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00007890-197103000-00013%26rft_id%253Dinfo%253Apmid%252F5558564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00007890-197103000-00013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5558564&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971I836800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4169.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Why sex and recombination?</span> pp. <span class="cit-fpage">187</span>–<span class="cit-lpage">195</span> in <span class="cit-source">Cold Spring Harbor Symposia on Quantitative Biology</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.4" data-doi="10.1186/s12711-016-0221-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Battagin</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gorjanc</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.-M.</span> <span class="cit-name-surname">Faux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Johnston</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Effect of manipulating recombination rates on response to selection in livestock breeding programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-016-0221-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-016-0221-1%26rft_id%253Dinfo%253Apmid%252F27335010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-016-0221-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27335010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.5" data-doi="10.1186/gb-2013-14-9-r103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Falque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Walter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bauland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Camisan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Intraspecific variation of recombination rate in maize.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-9-r103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR103%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r103%26rft_id%253Dinfo%253Apmid%252F24050704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24050704&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.6" data-doi="10.1007/s00122-009-1034-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Should maize doubled haploids be induced among F 1 or F 2 plants?</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1034-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D119%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D262%26rft.atitle%253DShould%2Bmaize%2Bdoubled%2Bhaploids%2Bbe%2Binduced%2Bamong%2BF%25281%2529%2Bor%2BF%2B%25282%2529%2Bplants%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1034-1%26rft_id%253Dinfo%253Apmid%252F19396574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1034-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19396574&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.7" data-doi="10.3835/plantgenome2016.11.0118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Prospective targeted recombination and genetic gains for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.8" data-doi="10.1007/s00122-018-3240-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blary</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Jenczewski</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Manipulation of crossover frequency and distribution for plant breeding.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3240-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D132%26rft.spage%253D575%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3240-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3240-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.9" data-doi="10.3835/plantgenome2018.08.0062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandariz</span>,  <span class="cit-name-given-names">S. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Predicted Genetic Gains from Targeted Recombination in Elite Biparental Maize Populations.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">12</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.08.0062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D12%26rft.spage%253D180062%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.08.0062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.08.0062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.10" data-doi="10.1086/282718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">The effect of selection on genetic variability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D105%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J801400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.11" data-doi="10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Perspective: sex, recombination, and the efficacy of selection was Weismann right?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBurt%26rft.auinit1%253DA.%26rft.volume%253D54%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D351%26rft.atitle%253DPerspective%253A%2Bsex%252C%2Brecombination%252C%2Band%2Bthe%2Befficacy%2Bof%2Bselection--was%2BWeismann%2Bright%253F%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282000%2529054%255B0337%253APSRATE%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F10937212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2000)054[0337:PSRATE]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10937212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086886500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.12"><div class="cit-metadata unstructured">Choulet, F., A. Alberti, S. Theil, N. Glover, V. Barbe <em>et al.</em>, 2014 Structural and functional partitioning of bread wheat chromosome 3B. Science 345: 1249721.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.13" data-doi="10.1038/sj.hdy.6801059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Williford</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kliman</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Hill–Robertson effect: evolutionary consequences of weak selection and linkage in finite populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6801059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D100%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D31%26rft.atitle%253DThe%2BHill-Robertson%2Beffect%253A%2Bevolutionary%2Bconsequences%2Bof%2Bweak%2Bselection%2Band%2Blinkage%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801059%26rft_id%253Dinfo%253Apmid%252F17878920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17878920&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251727900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.14" data-doi="10.1126/science.1220381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crismani</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Girard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Froger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pradillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Santos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FANCM limits meiotic crossovers.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>: <span class="cit-fpage">1588</span>–<span class="cit-lpage">1590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1220381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCrismani%26rft.auinit1%253DW.%26rft.volume%253D336%26rft.issue%253D6088%26rft.spage%253D1588%26rft.epage%253D1590%26rft.atitle%253DFANCM%2BLimits%2BMeiotic%2BCrossovers%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1220381%26rft_id%253Dinfo%253Apmid%252F22723424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzYvNjA4OC8xNTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.15" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D78%26rft.spage%253D737%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4448362&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974V365800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.17" data-doi="10.1073/pnas.1713078114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seguela-Arnaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Larchevêque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lloyd</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mercier</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Unleashing meiotic crossovers in hybrid plants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">2431</span>–<span class="cit-lpage">2436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1713078114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1713078114%26rft_id%253Dinfo%253Apmid%252F29183972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzEwLzI0MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.18" data-doi="10.1371/journal.pgen.1005369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Girard</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Chelysheva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Choinard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Froger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Macaisne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">AAA-ATPase FIDGETIN-LIKE 1 and helicase FANCM antagonize meiotic crossovers by distinct mechanisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005369</span>. Erratum: e1005448.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005369%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.19" data-doi="10.1371/journal.pgen.1002354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraut</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Falque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drouaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pereira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. C.</span> <span class="cit-name-surname">Martin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide crossover distribution in Arabidopsis thaliana meiosis reveals sex-specific patterns along chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGiraut%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002354%26rft.epage%253De1002354%26rft.atitle%253DGenome-wide%2Bcrossover%2Bdistribution%2Bin%2BArabidopsis%2Bthaliana%2Bmeiosis%2Breveals%2Bsex-specific%2Bpatterns%2Balong%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002354%26rft_id%253Dinfo%253Apmid%252F22072983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072983&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.20" data-doi="10.1038/nature03405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. C. J.</span> <span class="cit-name-surname">Godfray</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Burt</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sex increases the efficacy of natural selection in experimental yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">636</span>–<span class="cit-lpage">640</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BR.%26rft.volume%253D434%26rft.issue%253D7033%26rft.spage%253D636%26rft.epage%253D640%26rft.atitle%253DSex%2Bincreases%2Bthe%2Befficacy%2Bof%2Bnatural%2Bselection%2Bin%2Bexperimental%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03405%26rft_id%253Dinfo%253Apmid%252F15800622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15800622&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228011000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.21" data-doi="10.1186/s12711-017-0330-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Battagin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gorjanc</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The potential of shifting recombination hotspots to increase genetic gain in livestock breeding.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0330-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0330-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0330-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.22" data-doi="10.1007/s11032-010-9534-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Simmonds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Leverington</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fish</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meta-QTL analysis of the genetic control of crop height in elite European winter wheat germplasm.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-010-9534-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D29%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-010-9534-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-010-9534-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000299084500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.23" data-doi="10.1051/gse:2001117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of the effects of genes affecting quantitative traits in livestock.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-33-3-209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D229%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bthe%2Beffects%2Bof%2Bgenes%2Baffecting%2Bquantitative%2Btraits%2Bin%2Blivestock.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2001117%26rft_id%253Dinfo%253Apmid%252F11403745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2001117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11403745&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169502100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.24" data-doi="10.1038/ncomms15605"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Filler-Hayut</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Bessudo</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Levy</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Targeted recombination between homologous chromosomes for precise breeding in tomato.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">15605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms15605</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D15605%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.25" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">W. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300010156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Botstein</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D121%26rft.spage%253D185%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2563713&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989R676700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.27" data-doi="10.1105/tpc.110.075986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leflon</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Grandont</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Huteau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Coriton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Crossovers get a boost in Brassica allotriploid and allotetraploid hybrids.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2253</span>–<span class="cit-lpage">2264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.110.075986</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.075986%26rft_id%253Dinfo%253Apmid%252F20622148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMjI1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.28" data-doi="10.1046/j.1365-313X.1993.04040745.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Dean</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Recombinant inbred lines for mapping RFLP and phenotypic markers in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-313X.1993.04040745.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D4%26rft.spage%253D745%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1993.04040745.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1993.04040745.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MC86200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.29" data-doi="10.1534/genetics.117.300588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Morgan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C. H.</span> <span class="cit-name-surname">Franklin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bomblies</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Plasticity of meiotic recombination rates in response to temperature in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300588</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300588%26rft_id%253Dinfo%253Apmid%252F29496746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwOC80LzE0MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.30" data-doi="10.1111/mec.12879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lovell</span>,  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Grogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Sharbel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">McKay</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mating system and environmental variation drive patterns of adaptation in <em>Boechera spatifolia</em> (Brassicaceae).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4486</span>–<span class="cit-lpage">4497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12879</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D4486%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.31" data-doi="10.1534/genetics.115.183210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>,  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rousseau-Gueutin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Morice</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">Bayer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Besharat</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Centromere locations in Brassica A and C genomes revealed through half-tetrad analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.183210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.183210%26rft_id%253Dinfo%253Apmid%252F26614742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNTEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.32" data-doi="10.1007/s00122-013-2136-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClosky</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Tanksley</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The impact of recombination on short-term selection gain in plant breeding experiments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2299</span>–<span class="cit-lpage">2312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2136-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2299%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2136-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2136-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.33" data-doi="10.1017/S0016672300002238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPhee</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Robertson</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">The effect of suppressing crossing-over on the response to selection in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300002238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMcPhee%26rft.auinit1%253DC.%2BP.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D16%26rft.atitle%253DThe%2Beffect%2Bof%2Bsuppressing%2Bcrossing-over%2Bon%2Bthe%2Bresponse%2Bto%2Bselection%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300002238%26rft_id%253Dinfo%253Apmid%252F5488984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300002238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5488984&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H384300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>,  <span class="cit-name-given-names">M. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Speed</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> Modeling interference in genetic recombination. Genetics139: 1031–1044.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.35" data-doi="10.1146/annurev-arplant-050213-035923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Mézard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Jenczewski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Macaisne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Grelon</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The molecular biology of meiosis in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-arplant-050213-035923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-050213-035923%26rft_id%253Dinfo%253Apmid%252F25494464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-050213-035923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25494464&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.36" data-doi="10.1038/s41477-018-0311-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mieulet</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Aubert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bres</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Klein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Droc</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Unleashing meiotic crossovers in crops.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1010</span>–<span class="cit-lpage">1016</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41477-018-0311-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D4%26rft.spage%253D1010%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41477-018-0311-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41477-018-0311-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4169.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mizuno</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Matsumoto</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wu</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Composition and Structure of Rice Centromeres and Telomeres</span>. pp. <span class="cit-fpage">37</span>–<span class="cit-lpage">52</span> in <span class="cit-source"><em>Rice Genomics</em>, <em>Genetics and Breeding</em></span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Singapore</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.38" data-doi="10.1016/j.pbi.2016.11.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Modliszewski</span>,  <span class="cit-name-given-names">J. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Copenhaver</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Meiotic recombination gets stressed out: CO frequency is plastic under pressure.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2016.11.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D36%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2016.11.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2016.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.39" data-doi="10.1016/0027-5107(64)90047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">The relation of recombination to mutational advance.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0027-5107(64)90047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D1%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252864%252990047-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(64)90047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.40" data-doi="10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selection for recombination in small populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1921</span>–<span class="cit-lpage">1931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2001.tb01310.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D55%26rft.issue%253D10%26rft.spage%253D1921%26rft.epage%253D1931%26rft.atitle%253DSelection%2Bfor%2Brecombination%2Bin%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1921%253ASFRISP%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11761054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1921:SFRISP]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11761054&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172308600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.41" data-doi="10.1371/journal.pgen.1006794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelé</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Falque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Trotoux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nègre</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Amplifying recombination genome-wide and reshaping crossover landscapes in Brassicas.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1006794</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006794</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006794%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.42" data-doi="10.1111/nph.13548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jenkins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Macaulay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Nibau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wnetrzak</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The effect of temperature on the male and female recombination landscape of barley.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13548</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13548%26rft_id%253Dinfo%253Apmid%252F26255865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26255865&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-9.12.4169.43"><div class="cit-metadata unstructured">R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.44" data-doi="10.1007/s00122-018-3216-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ru</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Targeted recombination to increase genetic gain in self-pollinated species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3216-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D132%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3216-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3216-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.45" data-doi="10.1534/genetics.111.126607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bomblies</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Laitinen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mott</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic architecture of flowering-time variation in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.126607</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.126607%26rft_id%253Dinfo%253Apmid%252F21406681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.46" data-doi="10.1073/pnas.1423107112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Séguéla-Arnaud</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Crismani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Larchevêque</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mazel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Froger</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multiple mechanisms limit meiotic crossovers: TOP3α and two BLM homologs antagonize crossovers in parallel to FANCM.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">4713</span>–<span class="cit-lpage">4718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1423107112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1423107112%26rft_id%253Dinfo%253Apmid%252F25825745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE1LzQ3MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.12.4169.47" data-doi="10.1073/pnas.1713071115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lambing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Griffin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Topp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Nageswaran</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Massive crossover elevation via combination of HEI10 and recq4a recq4b during Arabidopsis meiosis.</span> <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">115</span>: <span class="cit-fpage">2437</span>–<span class="cit-lpage">2442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1713071115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1713071115%26rft_id%253Dinfo%253Apmid%252F29463699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzEwLzI0MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>,  <span class="cit-name-given-names">J. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Stack</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> Two-dimensional spreads of synaptonemal complexes from solanaceous plants. VI. High-resolution recombination nodule map for tomato (<em>Lycopersicon esculentum</em>). Genetics141: 683–708.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.49" data-doi="10.1111/nph.12534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suay</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jouy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lodé</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Crossover rate between homologous chromosomes and interference are regulated by the addition of specific unpaired chromosomes in Brassica.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">656</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D201%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12534%26rft_id%253Dinfo%253Apmid%252F24117470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24117470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.50" data-doi="10.1371/journal.pbio.1000326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>,  <span class="cit-name-given-names">P. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Henikoff</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Centromeres convert but don’t cross.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000326</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTalbert%26rft.auinit1%253DP.%2BB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1000326%26rft.epage%253De1000326%26rft.atitle%253DCentromeres%2Bconvert%2Bbut%2Bdon%2527t%2Bcross.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000326%26rft_id%253Dinfo%253Apmid%252F20231873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20231873&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.51" data-doi="10.1007/s00122-012-1905-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Riedelsheimer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Schrag</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of hybrid performance in maize with models incorporating dominance and population specific marker effects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1181</span>–<span class="cit-lpage">1194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1905-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1181%26rft.epage%253D1194%26rft.atitle%253DGenomic%2Bprediction%2Bof%2Bhybrid%2Bperformance%2Bin%2Bmaize%2Bwith%2Bmodels%2Bincorporating%2Bdominance%2Band%2Bpopulation%2Bspecific%2Bmarker%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1905-8%26rft_id%253Dinfo%253Apmid%252F22733443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1905-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22733443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.52" data-doi="10.1007/BF00227388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>,  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Accuracy of mapping quantitative trait loci in autogamous species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00227388</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D84%26rft.spage%253D803%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00227388%26rft_id%253Dinfo%253Apmid%252F24201478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00227388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24201478&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JT30000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.53" data-doi="10.1016/j.tig.2011.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>,  <span class="cit-name-given-names">M. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Hurst</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Direct and indirect consequences of meiotic recombination: implications for genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2011.11.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DWebster%26rft.auinit1%253DM.%2BT.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D101%26rft.epage%253D109%26rft.atitle%253DDirect%2Band%2Bindirect%2Bconsequences%2Bof%2Bmeiotic%2Brecombination%253A%2Bimplications%2Bfor%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.11.002%26rft_id%253Dinfo%253Apmid%252F22154475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22154475&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301635300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.54" data-doi="10.1038/41312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeyl</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bell</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The advantage of sex in evolving yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">388</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/41312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZeyl%26rft.auinit1%253DC.%26rft.volume%253D388%26rft.issue%253D6641%26rft.spage%253D465%26rft.epage%253D468%26rft.atitle%253DThe%2Badvantage%2Bof%2Bsex%2Bin%2Bevolving%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F41312%26rft_id%253Dinfo%253Apmid%252F9242403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/41312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9242403&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XN55300046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4169.55" data-doi="10.1101/gad.295501.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ziolkowski</span>,  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Underwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lambing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Martinez-Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Lawrence</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Natural variation and dosage of the HEI10 meiotic E3 ligase control Arabidopsis crossover recombination.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.295501.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.295501.116%26rft_id%253Dinfo%253Apmid%252F28223312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMzEvMy8zMDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4169.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4159" title="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4183" title="Classification-Based Inference of Dynamical Models of Gene Regulatory Networks" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4169" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4B14JhtbpujdeHRbEIdeZ_rUo-yFC8p-03MBIL5ZfII" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4169" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4169" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4169" class="active" data-icon-position="" data-hide-link-title="0">Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494989" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qaH6FgJMA7TOFs3sJ8DcKaiJpmXByY9dJAstIFpXKoQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4169.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4169" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_NuDg3NN7lAuNAZ9kHGdz_hY5QC3t6vJS_zXa-_Rj_E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4169" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4169.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494989" id="node494989--2" data-pisa="ggg;9/12/4169" data-pisa-master="ggg;g3.119.400545" data-apath="/ggg/9/12/4169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of <em>Oryza sativa</em> and <em>Brassica rapa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elise Tourrette</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rex Bernardo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthieu Falque</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier C. Martin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4169-4181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400545 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494989--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elise Tourrette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elise%2BTourrette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tourrette%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElise%2BTourrette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elise Tourrette</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rex Bernardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rex%2BBernardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernardo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARex%2BBernardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rex Bernardo</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthieu Falque</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Olivier C. Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BC.%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20OC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BC.%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier C. Martin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:olivier.c.martin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">olivier.c.martin@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4169" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4169&amp;text=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4169&amp;t=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4169&amp;title=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494989/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494989" id="node494989--3" data-pisa="ggg;9/12/4169" data-pisa-master="ggg;g3.119.400545" data-apath="/ggg/9/12/4169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of <em>Oryza sativa</em> and <em>Brassica rapa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elise Tourrette</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rex Bernardo</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthieu Falque</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier C. Martin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4169-4181;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400545 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494989--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elise Tourrette </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elise%2BTourrette%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tourrette%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElise%2BTourrette%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0822-5330" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elise Tourrette</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Rex Bernardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rex%2BBernardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bernardo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARex%2BBernardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3323-4690" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rex Bernardo</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthieu Falque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthieu%2BFalque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Falque%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthieu%2BFalque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6444-858X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthieu Falque</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Olivier C. Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>GQE - Le Moulon, INRA, Univ. Paris-Sud, CNRS, AgroParisTech, Université Paris-Saclay, Gif-sur-Yvette, France and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BC.%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20OC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BC.%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5295-5963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier C. Martin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:olivier.c.martin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">olivier.c.martin@inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4169&amp;rft.epage=4181&amp;rft.atitle=Assessing%20by%20Modeling%20the%20Consequences%20of%20Increased%20Recombination%20in%20Recurrent%20Selection%20of%20Oryza%20sativa%20and%20Brassica%20rapa&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tourrette%2C+Elise&amp;rft.au=Bernardo%2C+Rex&amp;rft.au=Falque%2C+Matthieu&amp;rft.au=Martin%2C+Olivier+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494989/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494989/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494989/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494989/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494989/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494989/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494989/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494989/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494989/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494989/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494989/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494989/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4169" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4169.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4169.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4169.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
