<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4159" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4159.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4159.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4159.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4159/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4159.atom" />
<meta name="HW.pisa" content="ggg;9/12/4159" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" />
<meta name="DC.Identifier" content="10.1534/g3.119.400755" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Tilk et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Evolve-and-resequence (E+R) experiments leverage next-generation sequencing technology to track the allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here, we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of bi-allelic SNPs in populations of any model organism founded with sequenced homozygous strains. Using both experimentally-pooled and simulated samples of Drosophila melanogaster , we show that haplotype inference can improve allele frequency accuracy by orders of magnitude for up to 50 generations of recombination, and is robust to moderate levels of missing data, as well as different selection regimes. Finally, we show that a simple linear model generated from these simulations can predict the accuracy of haplotype-derived allele frequencies in other model organisms and experimental designs. To make these results broadly accessible for use in E+R experiments, we introduce HAF-pipe, an open-source software tool for calculating haplotype-derived allele frequencies from raw sequencing data. Ultimately, by reducing sequencing costs without sacrificing accuracy, our method facilitates E+R designs with higher replication and resolution, and thereby, increased power to detect adaptive alleles." />
<meta name="DC.Contributor" content="Susanne Tilk" />
<meta name="DC.Contributor" content="Alan Bergland" />
<meta name="DC.Contributor" content="Aaron Goodman" />
<meta name="DC.Contributor" content="Paul Schmidt" />
<meta name="DC.Contributor" content="Dmitri Petrov" />
<meta name="DC.Contributor" content="Sharon Greenblum" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Evolve-and-resequence (E+R) experiments leverage next-generation sequencing technology to track the allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&amp;gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here, we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&amp;lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of bi-allelic SNPs in populations of any model organism founded with sequenced homozygous strains. Using both experimentally-pooled and simulated samples of &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, we show that haplotype inference can improve allele frequency accuracy by orders of magnitude for up to 50 generations of recombination, and is robust to moderate levels of missing data, as well as different selection regimes. Finally, we show that a simple linear model generated from these simulations can predict the accuracy of haplotype-derived allele frequencies in other model organisms and experimental designs. To make these results broadly accessible for use in E+R experiments, we introduce HAF-pipe, an open-source software tool for calculating haplotype-derived allele frequencies from raw sequencing data. Ultimately, by reducing sequencing costs without sacrificing accuracy, our method facilitates E+R designs with higher replication and resolution, and thereby, increased power to detect adaptive alleles.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4159" />
<meta name="citation_id" content="9/12/4159" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4159" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4159.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4159.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4159.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400755" />
<meta name="citation_pmid" content="31636085" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4159" />
<meta name="citation_lastpage" content="4168" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Susanne Tilk" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford CA 94305," />
<meta name="citation_author_email" content="greensi@lbl.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9156-9360" />
<meta name="citation_author" content="Alan Bergland" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford CA 94305," />
<meta name="citation_author_institution" content="Department of Biology, University of Virginia, Charlottesville VA 22904, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7145-7575" />
<meta name="citation_author" content="Aaron Goodman" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford CA 94305," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0768-7592" />
<meta name="citation_author" content="Paul Schmidt" />
<meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia PA 19104" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8076-6705" />
<meta name="citation_author" content="Dmitri Petrov" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford CA 94305," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3664-9130" />
<meta name="citation_author" content="Sharon Greenblum" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford CA 94305," />
<meta name="citation_author_email" content="greensi@lbl.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6148-3016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. G. Baldwin-Brown;citation_author=A. D. Long;citation_author=K. R. Thornton;citation_title=The power to detect quantitative trait loci using resequenced, experimentally evolved populations of diploid, sexual organisms.;citation_volume=31;citation_year=2014;citation_pmid=24441104;citation_doi=10.1093/molbev/msu048" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. Barghi;citation_author=R. Tobler;citation_author=V. Nolte;citation_author=C. Schlötterer;citation_title=Drosophila simulans: A Species with Improved Resolution in Evolve and Resequence Studies.;citation_volume=7;citation_year=2017;citation_pmid=28546383;citation_doi=10.1534/g3.117.043349" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Barrick;citation_author=D. S. Yu;citation_author=S. H. Yoon;citation_author=H. Jeong;citation_author=T. K. Oh;citation_title=Genome evolution and adaptation in a long-term experiment with Escherichia coli.;citation_pages=1243-1247;citation_volume=461;citation_year=2009;citation_issue=7268;citation_pmid=19838166;citation_doi=10.1038/nature08480" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Baym;citation_author=S. Kryazhimskiy;citation_author=T. D. Lieberman;citation_author=H. Chung;citation_author=M. M. Desai;citation_title=Inexpensive multiplexed library preparation for megabase-sized genomes.;citation_pages=e0128036;citation_volume=10;citation_year=2015;citation_pmid=26000737;citation_doi=10.1371/journal.pone.0128036" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=E. L. Behrman;citation_author=V. M. Howick;citation_author=M. Kapun;citation_author=F. Staubach;citation_author=A. O. Bergland;citation_title=Rapid seasonal evolution in innate immunity of wild Drosophila melanogaster.;citation_volume=285;citation_year=2018;citation_pmid=29321302;citation_doi=10.1098/rspb.2017.2599" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. K. Burke;citation_author=J. P. Dunham;citation_author=P. Shahrestani;citation_author=K. R. Thornton;citation_author=M. R. Rose;citation_title=Genome-wide analysis of a long-term evolution experiment with Drosophila.;citation_pages=587-590;citation_volume=467;citation_year=2010;citation_issue=7315;citation_pmid=20844486;citation_doi=10.1038/nature09352" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=M. K. Burke;citation_title=How does adaptation sweep through the genome? Insights from long-term selection experiments.;citation_volume=279;citation_year=2012;citation_pmid=22833271;citation_doi=10.1098/rspb.2012.0799" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. K. Burke;citation_author=G. Liti;citation_author=A. D. Long;citation_title=Standing genetic variation drives repeatable experimental evolution in outcrossing populations of Saccharomyces cerevisiae.;citation_volume=31;citation_year=2014;citation_pmid=25172959;citation_doi=10.1093/molbev/msu256" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C.-C. Cao;citation_author=X. Sun;citation_title=Accurate estimation of haplotype frequency from pooled sequencing data and cost-effective identification of rare haplotype carriers by overlapping pool sequencing.;citation_volume=31;citation_year=2015;citation_pmid=25304780;citation_doi=10.1093/bioinformatics/btu670" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. H. Chandler;citation_title=Parallel genome-wide fixation of ancestral alleles in partially outcrossing experimental populations of Caenorhabditis elegans.;citation_volume=4;citation_year=2014;citation_pmid=24989146;citation_doi=10.1534/g3.114.012914" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. M. Comeron;citation_author=R. Ratnappan;citation_author=S. Bailin;citation_title=The Many Landscapes of Recombination in Drosophila melanogaster.;citation_pages=e1002905-e1002905;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071443;citation_doi=10.1371/journal.pgen.1002905" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. E. Cook;citation_author=S. Zdraljevic;citation_author=J. P. Roberts;citation_author=E. C. Andersen;citation_title=CeNDR, the Caenorhabditis elegans natural diversity resource.;citation_volume=45;citation_year=2017;citation_pmid=27701074;citation_doi=10.1093/nar/gkw893" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. F. Feder;citation_author=D. A. Petrov;citation_author=A. O. Bergland;citation_title=LDx: Estimation of Linkage Disequilibrium from High-Throughput Pooled Resequencing Data.;citation_pages=e48588-e48588;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23152785;citation_doi=10.1371/journal.pone.0048588" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Fracassetti;citation_author=P. C. Griffin;citation_author=Y. Willi;citation_title=Validation of Pooled Whole-Genome Re-Sequencing in Arabidopsis lyrata.;citation_pages=e0140462;citation_volume=10;citation_year=2015;citation_pmid=26461136;citation_doi=10.1371/journal.pone.0140462" />
<meta name="citation_reference" content="citation_author=S. U. Franssen;citation_author=N. H. Barton;citation_author=C. Schlötterer;citation_year=2017;citation_pmid=27702776;citation_doi=10.1093/molbev/msw210" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. L. Graves;citation_author=K. L. Hertweck;citation_author=M. A. Phillips;citation_author=M. V. Han;citation_author=L. G. Cabral;citation_title=Genomics of Parallel Experimental Evolution in Drosophila.;citation_pages=831-842;citation_volume=34;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ràmia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster Genetic Reference Panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_author=C. Illingworth;citation_author=L. Parts;citation_author=S. Schiffels;citation_author=G. Liti;citation_author=V. Mustonen;citation_year=2011;citation_pmid=22114362;citation_doi=10.1093/molbev/msr289" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=G. H. Jajamovich;citation_author=A. Iliadis;citation_author=D. Anastassiou;citation_author=X. Wang;citation_title=Maximum-parsimony haplotype frequencies inference based on a joint constrained sparse representation of pooled DNA.;citation_pages=270;citation_volume=14;citation_year=2013;citation_pmid=24010487;citation_doi=10.1186/1471-2105-14-270" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. R. Jha;citation_author=C. M. Miles;citation_author=N. R. Lippert;citation_author=C. D. Brown;citation_author=K. P. White;citation_title=Whole-Genome Resequencing of Experimental Populations Reveals Polygenic Basis of Egg-Size Variation in Drosophila melanogaster.;citation_volume=32;citation_year=2015;citation_pmid=26044351;citation_doi=10.1093/molbev/msv136" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. Kessner;citation_author=T. L. Turner;citation_author=J. Novembre;citation_title=Maximum Likelihood Estimation of Frequencies of Known Haplotypes from Pooled Sequence Data.;citation_volume=30;citation_year=2013;citation_pmid=23364324;citation_doi=10.1093/molbev/mst016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Kessner;citation_author=J. Novembre;citation_title=forqs: forward-in-time simulation of recombination, quantitative traits and selection.;citation_volume=30;citation_year=2014;citation_pmid=24336146;citation_doi=10.1093/bioinformatics/btt712" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Kessner;citation_author=J. Novembre;citation_title=Power analysis of artificial selection experiments using efficient whole genome simulation of quantitative traits.;citation_volume=199;citation_year=2015;citation_pmid=25672748;citation_doi=10.1534/genetics.115.175075" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Kofler;citation_author=C. Schlötterer;citation_title=A guide for the design of evolve and resequencing studies.;citation_volume=31;citation_year=2014;citation_pmid=24214537;citation_doi=10.1093/molbev/mst221" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Kolaczkowski;citation_author=A. D. Kern;citation_author=A. K. Holloway;citation_author=D. J. Begun;citation_title=Genomic differentiation between temperate and tropical Australian populations of Drosophila melanogaster.;citation_volume=187;citation_year=2011;citation_pmid=21059887;citation_doi=10.1534/genetics.110.123059" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. B. Lack;citation_author=J. D. Lange;citation_author=A. D. Tang;citation_author=R. B. Corbett-Detig;citation_author=J. E. Pool;citation_title=A Thousand Fly Genomes: An Expanded Drosophila Genome Nexus.;citation_volume=33;citation_year=2016;citation_pmid=27687565;citation_doi=10.1093/molbev/msw195" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Q. Long;citation_author=D. C. Jeffares;citation_author=Q. Zhang;citation_author=K. Ye;citation_author=V. Nizhynska;citation_title=PoolHap: inferring haplotype frequencies from pooled samples by next generation sequencing.;citation_pages=e15292-e15292;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21264334;citation_doi=10.1371/journal.pone.0015292" />
<meta name="citation_reference" content="citation_author=A. Long;citation_author=G. Liti;citation_author=A. Luptak;citation_author=O. Tenaillon;citation_pages=567;citation_volume=16;citation_year=2015;citation_pmid=26347030;citation_doi=10.1038/nrg3937" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Lynch;citation_author=D. Bost;citation_author=S. Wilson;citation_author=T. Maruki;citation_author=S. Harrison;citation_title=Population-genetic inference from pooled-sequencing data.;citation_volume=6;citation_year=2014;citation_pmid=24787620;citation_doi=10.1093/gbe/evu085" />
<meta name="citation_reference" content="citation_author=P. Nouhaud;citation_author=R. Tobler;citation_author=V. Nolte;citation_author=C. Schlötterer;citation_year=2016;citation_doi=10.1002/ece3.2402" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Orozco-terWengel;citation_author=M. Kapun;citation_author=V. Nolte;citation_author=R. Kofler;citation_author=T. Flatt;citation_title=Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.;citation_pages=4931-4941;citation_volume=21;citation_year=2012;citation_issue=20;citation_pmid=22726122;citation_doi=10.1111/j.1365-294X.2012.05673.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Pirinen;citation_title=Estimating population haplotype frequencies from pooled SNP data using incomplete database information.;citation_volume=25;citation_year=2009;citation_pmid=19861356;citation_doi=10.1093/bioinformatics/btp584" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. N. Pitt;citation_author=A. R. Ferré-D’Amaré;citation_title=Rapid Construction of Empirical RNA Fitness Landscapes;citation_pages=376-379;citation_volume=330;citation_year=2010;citation_issue=6002;citation_pmid=20947767;citation_doi=10.1126/science.1192001" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. Schirmer;citation_author=R. D’Amore;citation_author=U. Z. Ijaz;citation_author=N. Hall;citation_author=C. Quince;citation_title=Illumina error profiles: resolving fine-scale variation in metagenomic sequencing data.;citation_pages=125;citation_volume=17;citation_year=2016;citation_pmid=26968756;citation_doi=10.1186/s12859-016-0976-y" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. Schlötterer;citation_author=R. Kofler;citation_author=E. Versace;citation_author=R. Tobler;citation_author=S. U. Franssen;citation_title=“Combining Experimental Evolution with next-Generation Sequencing: A Powerful Tool to Study Adaptation from Standing Genetic Variation.”;citation_pages=431-40;citation_volume=114;citation_year=2015;citation_pmid=25269380;citation_doi=10.1038/hdy.2014.86" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=Z. Sheng;citation_author=M. E. Pettersson;citation_author=C. F. Honaker;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Standing genetic variation as a major contributor to adaptation in the Virginia chicken lines selection experiment.;citation_pages=219;citation_volume=16;citation_year=2015;citation_pmid=26438066;citation_doi=10.1186/s13059-015-0785-z" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Teotónio;citation_author=I. M. Chelo;citation_author=M. Bradić;citation_author=M. R. Rose;citation_author=A. D. Long;citation_title=Experimental evolution reveals natural selection on standing genetic variation.;citation_pages=251-257;citation_volume=41;citation_year=2009;citation_issue=2;citation_pmid=19136954;citation_doi=10.1038/ng.289" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. C. Turchin;citation_author=C. W. K. Chiang;citation_author=C. D. Palmer;citation_author=S. Sankararaman;citation_author=D. Reich;citation_title=Evidence of widespread selection on standing variation in Europe at height-associated SNPs.;citation_pages=1015-1019;citation_volume=44;citation_year=2012;citation_issue=9;citation_pmid=22902787;citation_doi=10.1038/ng.2368" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. L. Turner;citation_author=P. M. Miller;citation_title=Investigating natural variation in Drosophila courtship song by the evolve and resequence approach.;citation_volume=191;citation_year=2012;citation_pmid=22466043;citation_doi=10.1534/genetics.112.139337" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. A. Wichman;citation_author=M. R. Badgett;citation_author=L. A. Scott;citation_author=C. M. Boulianne;citation_author=J. J. Bull;citation_title=Different Trajectories of Parallel Evolution During Viral Adaptation;citation_pages=422-424;citation_volume=285;citation_year=1999;citation_issue=5426;citation_pmid=10411508;citation_doi=10.1126/science.285.5426.422" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Zhu;citation_author=A. O. Bergland;citation_author=J. González;citation_author=D. A. Petrov;citation_title=Empirical validation of pooled whole genome population re-sequencing in Drosophila melanogaster.;citation_pages=e41901-e41901;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22848651;citation_doi=10.1371/journal.pone.0041901" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Evolve-and-resequence (E+R) experiments leverage next-generation sequencing technology to track the allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here, we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of bi-allelic SNPs in populations of any model organism founded with sequenced homozygous strains. Using both experimentally-pooled and simulated samples of Drosophila melanogaster , we show that haplotype inference can improve allele frequency accuracy by orders of magnitude for up to 50 generations of recombination, and is robust to moderate levels of missing data, as well as different selection regimes. Finally, we show that a simple linear model generated from these simulations can predict the accuracy of haplotype-derived allele frequencies in other model organisms and experimental designs. To make these results broadly accessible for use in E+R experiments, we introduce HAF-pipe, an open-source software tool for calculating haplotype-derived allele frequencies from raw sequencing data. Ultimately, by reducing sequencing costs without sacrificing accuracy, our method facilitates E+R designs with higher replication and resolution, and thereby, increased power to detect adaptive alleles." />
<meta name="og-title" property="og:title" content="Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4159" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Evolve-and-resequence (E+R) experiments leverage next-generation sequencing technology to track the allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here, we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of bi-allelic SNPs in populations of any model organism founded with sequenced homozygous strains. Using both experimentally-pooled and simulated samples of Drosophila melanogaster , we show that haplotype inference can improve allele frequency accuracy by orders of magnitude for up to 50 generations of recombination, and is robust to moderate levels of missing data, as well as different selection regimes. Finally, we show that a simple linear model generated from these simulations can predict the accuracy of haplotype-derived allele frequencies in other model organisms and experimental designs. To make these results broadly accessible for use in E+R experiments, we introduce HAF-pipe, an open-source software tool for calculating haplotype-derived allele frequencies from raw sequencing data. Ultimately, by reducing sequencing costs without sacrificing accuracy, our method facilitates E+R designs with higher replication and resolution, and thereby, increased power to detect adaptive alleles." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"D3M9krgvJf9id3gv-sI4Tl0VRPOE8tWXL2UX2ySXrzQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494900","apath":"\/ggg\/9\/12\/4159.atom","pisa":"ggg;9\/12\/4159","ac":{"\/ggg\/9\/12\/4159.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4159.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4159"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4159"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4159":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Gl+dCHWm5DVsG\\\\\\\/G50Ccg0dKLe4FvjLoMgFJY760U5WiHvZhF8hzGJVOCdREpl7taUPZtup2dnj427qkzPZTyYUC3JGS22IeOBnLepISDelhnztuV\\\\\\\/z3NkYm\\\\\\\/hWU+U7egnPZylAOicmzRs36onhsa3E9AbxP\\\\\\\/3e1J\\\\\\\/2q2rjB3lX4wEtN8KBVU6dzxKLhl1mffRb23nJGKl30v1tMOVH8oUOFW+U0h9BVqIjtDK+WBbr3AHWwatNqvnaJELFnhR9Df13wFzVeLJ8e3Cf7BR\\\\\\\/Ti7JQQD+JvVBtHph1HUBXGaD1RcY+RYJ07RhLxGDBO+cb2WFP6d2GJVGY+iSas\\\\\\\/J1q+DBkzlwBbN2V1iLOsvOKn49\\\\\\\/0Z98\\\\\\\/OvOzPv+5hjyOuqcNqzEFjnrbv7qhj6Pa\\\\\\\/oTrP2pjCgsGoSPqXz5saP8rz2B4xbDYyyYL5tyClUIZbGOCiPJrYqAFLoofGiuOUvcsZK2souWS0r7jjbSICesTijNZGgrf6hL5h6F4NEsC0I1gFXEwE7nx9tzQqpIPr722YAP9am\\\\\\\/pVNms3EXQUMaEih2ywRsH9XtH7\\\\\\\/CqZR0UDRNGLL\\\\\\\/8DoigxAGmej0yjyjrfuhUTPaHom0kHU5RTGQ0VDv9sdoFZFl9l99Ge791Z3NH6haDpY2SEe3kfcfkwSVX2fdHFfQJUP0qbUKsAji4QjK9RLET9ElKah4iTDh+dzOwLpWbIqZ9agnKSsMNJFDy+6vXvzSvyWwpPtN9Z\\\\\\\/QWicb5i0M1bHZykYSBXQd\\\\\\\/iuy4cKqxvuRrTYZcTe3FR0dr4HGSoWgZyv82hcw4ax7G15lgIoJirYftnKdDvGs2sTOj5\\\\\\\/Tnb2lG1XTbqNQePa1hPoOds3AXTzLNaXs2fgwXvGIYpMWacycX+IySvu79j93YM\\\\\\\/zOJ4AwfPxvK6YFOVearml6hmMrMELf5QnHSEQBnLuXOGcMv6SPK8+8RqpgJrAPzmdOqfe6veb71thph9HUQL1dLdO7xfOBsg1E1M6glYUzVUoLxumrgsz2rIxEILTfz9mCFbUp3oA1hMm90PGV0NGefP\\\\\\\/VPpiliuK5r0SmdPPzu4USjwIHibIdvWxd\\\\\\\/2DSKOu1YMFI\\\\\\\/7imU6WBu5usWZtod\\\\\\\/O\\\\\\\/G2tonaEnDkCmGg+FaRCR9Cuhh3F7eiG4+zNKbGkW0\\\\\\\/TGo5i2AZo8snJJVGpf67sYlixsUHFOMPlDuExGH+PwWkg67cdU7n\\\\\\\/tj0py3orObisFNl3uoeyMc9gPhZ9l2sSl7080rKpOll0vjVoalNZ79yj+zhHFlyp2u07\\\u0022,\\\u0022iv\\\u0022:\\\u0022LGhZTWA\\\\\\\/VJ9ANybQqxgy9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226abcfb7bd1d7ff1cd8c8d51445905d246a573ca46bd3d955d976aa84884bc17f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LbYVxExtR6CGt7c4QeyuckKLsOBJ0oB7eme4KvlCcJYuE2KDF1W2KKIbFgtScVtIEmGmMLENWlhZ18pm9vrVGEcGlL54ldiJYDS7+SPFgfu\\\\\\\/ZPX2IHE2mltpSMhSvbbGwcS5y2b1XGEMTUk+HOOwujtL56DayODoPVET4cq60xD1QV4TgSiehz1wMw+\\\\\\\/MGJFZbgj+pwoxKqNtXT1f9M+KecRtLTFQl3zECZsw1wCZbHad44dFWF7dyHbu\\\\\\\/NK0sxRDUd+\\\\\\\/eAJTun\\\\\\\/bb4M9K\\\\\\\/RFfatKx+MzehO9jGbS5ilBQpsKnJSFTL5qfCpcicr7A2GP0koiYp0ok9GVyxbCioGhHUEf1CAX16YoAcwYPILVTKmR\\\\\\\/GZfIES\\\\\\\/juevbyxhooFDwS4tpuiALVM\\\\\\\/WzCnRP79myZDhAb97MeDjVTuOgT7AS5iBCDZ\\\\\\\/1QVIqPy1N2FjDbqWIMsm\\\\\\\/kD5hPHo7A0a1jZasIpNtZp7NJrXERwCkEJ5rvvv4FlHr9SfrbISdefq6cZT1\\\\\\\/Hx9yGQtXTYcC3oD0DiohI4e\\\\\\\/1zpDUxEF0GtZPC+\\\\\\\/Cwh\\\\\\\/kSKGPoAeIoWSHGs2dUxEi7hVJ+3kOIc2UE3eGjOlCDotbp4cpmeEAoMCPec81MKiCM2A\\\\\\\/jNvuAmnEfdvXyc9DNVLH1pLL+GZU8GSq974HzN5eeIWct0XSw+I2N0bvRmNb+BR5ZsCsLT8UbrxwJfn5gOie5eTHYv3fxUNcF\\\\\\\/zlfpeZkt6wfsgeXguv6q\\\\\\\/Xd7cYz\\\\\\\/dnIJzswzdZlzMK\\\\\\\/alvA1LPO8Iymv\\\\\\\/1O0eXpTc8I+ziKX6qBYvRs5N0yVp91ClTiNFuexqcTgE6enrQsOikWKQB3GeWGb3bH85RWA68qmVofvy35lXtK\\\\\\\/VuIVGSwWQkGnzNXxOttU1y88ur5uw4FihHjRH0DagHZXEED3QzHtNBmxHXUCyVVivg6\\\\\\\/Wstv3ccLm0iuPz07GCQZgzCzdHMK2a7GSJo8tOKbl4HCyZeSM3\\\\\\\/QZ17YhZniCIy\\\\\\\/SqUhOjxOTYplvoG3mZq7fFebcAaDpXNCFr4IKduH\\\\\\\/iSO6T9HLFHDIioyxDyaCRxpT9gU0H2+kmDe\\\\\\\/ObS6cpsM+TH3c1UtpWdNDrSwFmLkEFa+pEIFF5x9aJeAThkY6UNMg8T31YSe6467hKg99EHIJpKBhWqF521amGmxBTxu3+xGeEgGlyqGD0FMh9GnutRj0DVHlBfd8\\\\\\\/XoLIc4OVA0hOQr9XFVXeDXIE0cdrtsubG2Urruxyh3kMRB4nIszHs00C3r5CHuh8rPbp6Ic30usJGJIwpk6uAHUj2B\\\\\\\/84APSgqb33IZBTdJVUe3lsu1OXuLsDxP0GzbfJrrCstWaZAW0eWb4oVQjsMtkhbg8f2je04fUrcAlKk7Rpvtp0+Q42MRGPnD3FuPZyWN1maYc5GugmxspjpYcP+N\\\\\\\/mdb1fBlXwDJSTABeWCtCMYstdjVH61f0pG85z0Iq6ocv\\\\\\\/cvs+v9oCwYjAvSCd8yfBLpZwYBXO9yAoriCJusY9pcOf7gthMZqbqDFpA04FMz5nR5H82HnV5YuXvCzcuiGBEN5Da9dDB4v9Tjn9OBsiI6Xk\\\\\\\/4sgNVCBhLzuk5rdQ5fJkD25Pg4uwuh5oJKFINLq\\\\\\\/yvjScI5hlmMecZOuEC6ZyohnJb1wsmTWnKFqcvg9jZzU3z8zP0SM6Mh7j40fAXrJHkzBdvDexN4XyoVvqrdcPpy4tktOojMxExklFLzuh\\\\\\\/VMHmGwut6TzMQNPXVQ0T1ojqDojpnTNE88+w6pfu7GipiCGtRTo41lOQGsHbFg5MUG7YV56qiAVrnZ0d7LWUu4QCq6vVJlh4ZPAwstLpDNWq9N7RJ0ZTM1uDHWbVhSENu9ppeSDKzf2n+dyhNtJiSLzLcIAyHkPRU7Hzx\\\\\\\/9apU03bVv8mbRnOPU7lu1zPuIPA2zan5c9Rr\\\\\\\/TQULhZpuyN0RLXPRItH4YEr+HinO\\\\\\\/b7eZ+8NbAHpANjWQwczURKrWQG4FHuEk2QW2\\\\\\\/zwANSzs6P6dH67wZlxGn7ugONDDrkv6kVIQaVGXs47oR2RQARNZ8BAswR11eagxMTLdPzccXKxRsfwMJj8LBRyDOTMDIoxyJ4SQ7OPWJRtoQu5ZKBKkshvxggbdWypsIrYjdja11IyszxCQIwfpHrsG1DB\\\\\\\/2H+rJySS+K6ad\\\\\\\/R7kRE3y8fve3A9V8qy\\\\\\\/nKZgtyZEZPvqaE2SRNezup0Dj8KJJU23BlYfn\\\\\\\/+WVNZX+2j362\\\\\\\/XiviNU6pmpNWBMeVfwjn2AmcgwQjARjm1NnsAsswPwR2TnFWxz1eNDX9wPGLMiIDnxw60+7cylnt1JqlQYNpjGWueZVqEybu0b14A6BTTJ206R81KVnfTszfQuJVShkvYHDNxrVB10e8sXnMu8YiziZ7UvHC67W19qhAZ66\\\\\\\/\\\\\\\/DXvn+nXyATs67JxyxXurGXcES1l8aFBVpxFS\\\\\\\/eMm5PMiQwezGsmysX0yjOjIdCOJ6v\\\\\\\/03nz3Sal+FQFUQRayB9XpnJBthGNpXCmmhddUJK+QzshPAAJNi5OtrraCS\\\\\\\/eR48OyRKMWjpD\\\\\\\/+6tR+2BjcW2jql7RI1w2VhuWlnE0CU15\\\\\\\/KPZ19QxkANoo99Saw9s8s6MJkObC4u2YEOtJMFiI0en\\\\\\\/0yiGsA+MiYOOwxHbWaJ6TIuDKq9O4LEycFbfmGR4ymeiJjLjVIpMt7dOrKb91a\\\\\\\/biTTYeMhgXhVScgMDhRXqQ0RIGj88WRDejNWOa8BIZ\\\\\\\/CxeNjQpM2\\\\\\\/vfRRtaBaSCxoz0X2c8E8fanUDDVDCguh5EMs019YxJiOwxJERNgRBJ1DVdXV+MfooK7zqIo7Eh3AEc1McSWyFtDkofXuU2dWnJvyZ\\\\\\\/Gv6DJ7uD4ImL6e9KK\\\\\\\/rND\\\\\\\/sCPpfuvsLhvIHXcj6N5kqQDyLfxTfeD78encfUV6ww\\\\\\\/Hpm0CWxeFQgYF5fr1duCZXyscQZt+oP5AyatxOux4yJXpwDrDirzx7\\\\\\\/wC8Td7huneCbhUMwygOtjpfSps76iA3Ufw1p1LBxix1SsELDEMaQ+gtOJqk79H3V93liI09d7IxF\\\\\\\/XVhxyKzfclAotRc0brwgvCbb+qKCYVK\\\\\\\/Kdkr+Pa\\\\\\\/s2AgHAYTP6G0mNjKh58QC3QWwiWpG5ZSYPQ9bMTH0uH6yEP95xXEnWREKp6iuOtBub\\\\\\\/DhAAv7DKSNSaW\\\\\\\/bf4gV2kN0vvbbXnYyoL\\\\\\\/GA7UmIwAsmBeM2HnumYy\\\\\\\/rRwGITMRQO2Iil87XnbmJBqwmwFYaNd7vKFGCLrgXvRXvYlYJwu1TpcBBiookILHBrJCeX0iSqd64IenHrOiSkFVTan0oyXxhKDg2CD15Uqw5iunw+WtxM7Pp8iyHdoEZL1jiLFmYda6uMKnlK8fTm\\\\\\\/fkVzxNXUhDvisbDf3ZH8NCWtS15EBDYeurvRNA66WIJA25NWvFNQGYHhCemzdsl2D7nFZMpIN5B2G+MWsXs37dCJaWme1oPFZgqU6ScFo7PJ5yqoQdvSQbW18amZwyk5r5C\\\\\\\/UdhPMjtPAlSVHClRFuasBLSahwQTJHpB3vq\\\\\\\/ondJeP3NunUAbYjY8nQOkWxJzpMG5cbRMqW9nnWirb9kpizY2rWAsLaNTsX9IJIHSqWdzZ0GeK8vgYJvIHd8Tq7Dxk+ZNVuBbpiyhkxKl6WILkzGJdJLmXxmhnmmWoVDPKVeu7XRXyzfot+3h9M7It3Sy02C8umRvH1e9g5NejPWnc8ixHSNllzjOhdqzOqodPCb9RXcZjo9xGf5g4jRPT7jqhJkTd5OOyxCQEG0NTJpvFH2iUETRaZB8u6uACyGuZsWan1IJZ56nTwYd\\\\\\\/JUre\\\\\\\/D6HZiTp4gDLLM6VP2Z2G\\\\\\\/FSryQR\\\\\\\/t5uoHz2GsJzHYG2nn68Uc99LnErFVZMxKro2x8kmgn4swGvnViifx0LnVLahXC2t4hjM23qzxCZlYL\\\\\\\/WY3JMJSHHoMOtu5cjDqTzAp9Ho2eavsIsFaXfIBcV4YvjykqllHYeUhPyNuW5dzBYzr2poF9DKfeo2p\\\\\\\/E+IqlBtbON7n+e+E17Zr8YLGYESW1rQmgrjQBGWt4NsLEHo\\\\\\\/T3cNwty8TdsywYKXvKAg5\\\\\\\/4ABiuXc105v73luVACYSDTU82ce\\\\\\\/p0hQnrwHE0uZ9HINhWepZ8BM7tZmPrTuNpiKRBm6MNb4TWQBvAxWKs0Bwrip\\\\\\\/X9h+3WMTUjQEmUHfnmTdsTAsoigVihve\\\\\\\/gPk+gzpgt4tLQuTbjCtdRTUeNP0otSUY3Ps3c7aa1TiCyuiq0IPiUcqCHGsFQVUi+oWYzfJLOcaPQSGaBy0GL650zoEO5kq2P0UtHfShCX3RsfqmPn2vxnhCMqzormWF4043kqrz7JIWh\\\\\\\/01o8LsIKGtNYUebG2zkWaqxsK+X7OZepcTtqKY0v41GWefe9A31ddGHorDOxsQLUpAW+aH8qq5EdvOADZpFQ0eAu\\\\\\\/VvcGnVCnvhEtj1jSivfGoQuL1upmr2zWIsgHI\\\\\\\/vWYai1OkeZ6zFup0yJuaClfG7FPwAgIkgcPs0CtlrkrSqQ4aapWrW+jUs5\\\\\\\/qegBA717de+Y5+A09alnHSHplAHXV2pjQe9vL9ALrUvv\\\\\\\/hKVoHZj58sRO0ZG2OnNxxiCvIEVWq1ef6duQB6Ql5V9I16pmGY3P7sxiYhRtho\\\\\\\/6H\\\\\\\/0OiEndUfzwaXIbVQXdJ26YsDZmTpH+3EhD76v56naM16xIAEfB+O4bFB5JMkHrzoueOa8+BgcpIoa\\\\\\\/8Wf1Hr6naYGBtBqalmSUTTzhG4iqUkaSRmTYC8KzWETdcUz9B1qM4gomgYi2aHRCDaZ2tY23\\\\\\\/aasgSswXnjYiyGKUAnxJShJkkZ+Knpno+93Z\\\\\\\/gZzuAoJH191qcLbkwCzQzKQTkbA2bFglKhsqGazp2lnDatf4MzsAQsZ2OXy2uz6O8aaJPxS9Yqs+RAUc6SIDpeNfXwf2twQJx0SbwpqdDxicLikttE+e6SJ50743nOjw78xipnQJ9NKclbOmuZ9oWvhMgoEk4X7\\\\\\\/v18AkfgNC+CpH31zHfqk7X5541AkZywPyikh3Q1TzgzNZoBMhfkgiDY\\\\\\\/736N1UxtCTM4V10TKfkq0eZ+qEJaslxgPgrGy7nQahLrpa4rdchRVLi+6Wuh8BGS2Yx5tK9ZcRUpDnyo1Do+fFyeOOe\\\\\\\/os5dnlk2noJ5FhtZbniENvMqXAOxu0G6KFFHKm4GGDx+VEGU3H7bpdg2hI6EhwH\\\\\\\/jLagaEPAH9HhceRPNEOkh\\\\\\\/ci84qPV\\\\\\\/zhgSmMM4jRZmLQfO6Z5m4+TaN5gezKgAQtQl77tKcEZr8+jW3SFoCEuL6ZWcEtjOjKLnqXrcKIxcWadIKqPLEYget0MISXLATOsfYjlf1atrCo8+a0BOJK1cwlgXntGodIUrhNMCEGp8gp0pvSUWaMzXaA\\\\\\\/o3dijtbSSlgc9pPKFI597rMbGb\\\\\\\/7i+HnZl81tKjN1lmu91ZQQfZ5mHmZ3KjRDhjVvA+kHX2FUyj7p9NqJoYOZBF36GlUFVkLN0u53iNfMEo3feLlrc5oK\\\\\\\/OqxszONNgwJnRkaJwD4JQjA9wBC\\\\\\\/39ks2n6OFubRn8PUShyhmPznWWmmJSM\\\\\\\/UjeHdVMGbDcmARd7I0bCwe6zt0QuzM7n8NExfNN8Quv+5kbMISDDrUIZrO1EB+dUF89iaGwQJSgUcb8\\\\\\\/\\\\\\\/9FMyc6ME4aMwsSHlv5BjTXvkuLdE+ZNyJvuwiUBfRGWl5U4x9mqfrylryLNzGyLkj0Qjyi3ABzHsl\\\\\\\/1xtQrdlw0Dvnn4oe4Zz+wjr7NQPIRdz1Oznp8zRSmX5BuzLysh7+f3STr4n9yNNuXZVwb8L+u5muS7Vq4GVjMp6tWc8\\\\\\\/3DBcebqwEaIT\\\\\\\/jVbTwjzkfHWum5ZzRXUauJGBNI\\\\\\\/HDWPpvRggABzO2bDG9k7mlHKIZF90lf3rhvAJZM\\\\\\\/njTlXgMQ\\\\\\\/Et2kwMOVS02R1wy8EhyUAFpbrSyx42RTylaQzyini4+AK4SI05CQ7PmoVQvyGjtqW9KPaN\\\\\\\/yaN6R4JQf693yGaZZ\\\\\\\/sVPL84zQwDqFUOxAisYTC7UzbIBeA3djcWxDaZ\\\\\\\/ReUPXt5JCnyWtxeADIB5pj4FxXhH87X3ecSU0bxLfAL1ODoIL66J+2g2l5TqkSqCmilVG6kiUtnCTS81\\\\\\\/9UP5C84QRV3vKnmoXkHwajhgnfVx17nRVEoT+7by5Qv+Z+46HHIIdRANWwfKknFkiU7NytKm0KOdYvrHv21SzIKoQNkIe9yny+PqfH8DSHX++NFACBIj94Sz8Fo8loNDf6OL5lgnRRXhnSNuFTfVzJJ5\\\\\\\/JE+TQucgQXPo2kT6JZuGVtixiFQU6dYvguiYu8EffVa71Vpyl0WF9ny+t3ftXbtqF3VXkdf5XGG84KqBUgpzLWVkZjBCWc6aEaNIefyd7SFlg7Og04NUOqCoQTuv0UdjK6MfwXHOa5OKvegiuGjQLVKqajnESUrTSFtxxG1PmZnjFcL6yTf5F2FLutSDV8b+oyUd1Km1YmJrtcyl0oxH9tkRKKv5qIG\\\\\\\/ZLX26RXwMsgok50eS85pjdxCc91SKubGK2oykrVkEVbnA5anT6Jp9WWL8YaIrcHgZZkaWd\\\\\\\/O3rmu3I3KBMfr7xoRy49APrjMUvRu7ft0q+FTD7Y6bkVsa9Wy5SkJmcdK6qrn1Eka22T5K7JPsgAU\\\\\\\/5L\\\\\\\/tJqgpZQjCCC\\\\\\\/pkQXtRKzPzoyKxkNnf3\\\\\\\/X8dKgMjtSnoP9PHtVe2PuEjFEae5R\\\\\\\/INXfBZ0wCgsWXZScEQDxiNdqQ1forAMRBW+mvyo8YOu+bScXeDcKyEWszMljfLHEZlflqxntZSvx9DNPdpv+WapqrgpuvYpIwlG3Q4nhIUTbHZjlP1i2JTOCLxei\\\\\\\/0ZYioam1S2txLavpNrm5WnkbFajnPlfIwLDh6ROoyRF6VpV8heRWBOhIWylYToITaZpdNqyp8vqRmnXmfpZIe2QGfmp76izGN069H\\\\\\\/9Uh2SlE0U3P\\\\\\\/+RURzi0sFgQ3fYq8P5wiMCyx5cJ47DFKQN50npwWaAatOYQRmaRbJP50UVqd+J3YwnoMu4j85QlNuz7MAYrFDb\\\\\\\/fFzKAyLLsfvRTwSW9ftt5kAsLaasEy9wnIRC93rkAaOZYHs3PnMjZBXKvAw8e6OkqQvZLf0rkz6H6UcfJPjHFFzDnaPsgzn+JwZrlCOBRHyFujpsJvI62SR7yllXEG6LtAT1NncxW3f8jowq63xXsl2uxxc1e3WA\\\\\\\/MskkkcYRLD+lGKNheEEzOMfxra2mfZVFapbPV0p4+p1IA1daYNOiQfQgS8BMKj2FUfzU\\\\\\\/MnitCr9XSWqBnk0XUL7+029ScKpm56JW4TFUGo+\\\\\\\/9hEApavDi8vrxoh2xVqw2+RXDR5HZaTg+QaE8EI4aftlt\\\\\\\/9cWuLC4+agPmQC7ETNn5DefHy\\\\\\\/1H4fqEzg6py8cujnJL9Lx+pCiYwaAwJVe06fJz53dGPWAI8WevyiKv9ZlAqCHnLYKJ1Bze6IiySF0f4FOamOEDlTiimpCryy19JCyKH50NsPnVbu\\\\\\\/UYkxkG8lxVI+sCKzm\\\\\\\/V+LdPifLUz4rGnmckaOJzqytzz1HztBUnus1nwPeg\\\\\\\/TJ1rWGu3aJauzfsG9CcoX4Kw7PHwcpS1xtaSSjdV++eaUCVFNy8r7OzpG4Xp\\\\\\\/v8KpKfuaigD1RbTpDx++tfp8+0QpNV1wTcs8KfcmRjy4nvzGd1sevcIDgFyuVPDptuTcpA75+rhyqmJtBRL+5OZ+eZ78AyyB48jzrD6fC1xduGRG2e16dc4hm5SqlQOkJkIxC40I+nSCJ6JYUOVlN0UV1EqqqhXXFfQ6tTKOI4YqpJsfFrY1vJZ8CcjzlwcP\\\\\\\/pS4rMm27PepEOjWfGj+ja2XNw64AKiLF3f81Vk1BZwMxIpt9d2MF1nmEgIBhEPsbczDDKDJTZRaqV36ikxz94NOf4OlIL+VV63UPyvws5wS1lo8rxSOEVMmQwNJyUV9VnU8AyFIlA54cfVdmuICKAwzPpBOARZ2n2nXCcRW7cxpwe3IkkzACaSL2sFSNVznLiLHmeBLeuSh+89IluLvgSWl7rLmMQDPWQrFnwi7wXa00a6UOJJ5tunrCUI1+E9SrhfWUbbjdfyqCgdz1poUFfJQJgBfX7BOYzjCToUNBxCBV55tOhzdYop4Kx76C0ol3LDysUxi\\\\\\\/+epMjqTcPri3++LB+iGuSyBKRVaDhnrgNUadv\\\\\\\/SiNhKlRvqzCks1yuVzgf\\\\\\\/Sv+r7D9VuspBCgZMRIi5jh7J0w3H\\\\\\\/rrN3\\\\\\\/v\\\\\\\/sqTsOLvIypQGhpyJmNL8gxv4Efwyh6vCMJeNwX74jhrGcQqKBVPC+z51H6ygFv9aIoTj0H7JyqYfCs8dhbHv8\\\\\\\/jE+3mq\\\\\\\/VEBZzR4\\\\\\\/I1AIU9TTp6gZkJU6Qgw01AgiHcITz\\\\\\\/FiKx\\\\\\\/drYHsksch\\\\\\\/cQJ1fDZxxuHbJP8xMdNCDQMUauDTq+C4wqdw4PeUtplP\\\\\\\/oNNAinOWrmkLY6GVRwGobRDJncy5rR0bc6RN4hXGyB8q2sOTiqJgFBLIoxn+twnk31pA4ryPGlqpsalHA6Az2yMK7yeuIgrPxZFKJY8kMzyzgLzGLWrezu7p88B4oK57uzf4BYxj\\\\\\\/UPV6JEo7xw0zkAR11q2Sw5gA6\\\\\\\/7plnzMYoTAbPRSTMq+28OrivimZtM0xCXSYJHXNHMOn\\\\\\\/hYp5m+NVPmdhr61TSaSwlAIBEsVweKAXjf15OmRboiUU1MmIjS5zs3BjwYzcMpCyW3onpV06IQe7A3jdBkfb9\\\\\\\/kt3toNzqlhjh8ztNir3acBC4Yt0shnWsJmm7XI0QyqaM06M07XEuZGGVn3O4F4ZdE296avnsOIZ76mANnFZCbWbWltCUUfC+wTS0nht8vbySihJxMhRsRGWWCjOdq\\\\\\\/Gcl+vgGCjF3IPzCk6St5iORe6CnD+6WVGBTNo1lq1V4GxUX5uZGFuGUKQcDv3YKkzm1jRQXYdkfMeKnx1p6ygR6aBBTt5wgQwYJHux3XXlu4DEpKy3UXJgQ7\\\\\\\/9XLEItpd48FQltr\\\\\\\/jBg61AG\\\\\\\/oqVKThS5O3Gqdy6CG1c9a3XJBdRTOCObA7xHDsS1r9GPoBTqtURVdAz21CPy2hZvyaXqjTgJUN7SjA6ueE8LIb\\\\\\\/b\\\\\\\/6trVNqO6Q7W\\\\\\\/LX7nkkSEQaoVHmceldpnf73l90doPwtF0inSzJIAyyC6eyhSV3Wa87xUEo0Mnn6bngBUrHHKPY3IDKLd5X0aKkVIQW4XHW5tPmHj50ATyOJCxMWRsY0EKyRAagaSVGNYKdLITuZ+FJnaKUCpki4Zx1dungB1raciSUpSoCwMXtunMe9HS610vscyIlY876PoTSGNXWW9hGTVPlrm3SSNfzMgaGgkQWQNCAB7Ap5awC6zkt1oPeY5fV2V+1Dh0bSZOEoNmpp9RjdQfFb2GeJwPCrMJVh4i2RMgSDdyKQusf2\\\\\\\/eWAwIZdl7uEhW1cb15z6vRRUbxSKIVg90xh36c0y7YP\\\\\\\/akZ0ba9ejNzt2xlG8UeaPuID87d1hysT6lAqFAcvqRt\\\\\\\/c2VyU1jl3+\\\\\\\/NIqE0lBO5d\\\\\\\/ULsMUb5daARwZK3NyDMLsHp6g1rPo6OPbCeNhvBH4A0W5\\\\\\\/b+9mP17qxxnJG4iLqwK8WFtKAmVKBsIttix586TLW\\\\\\\/E+pq+kUfzJGLO+73BcKUqUrEVVv6RO+Ck6XgU2gBgStVjtI+BOJ+xTH0m8fMX+trECLJcezDFZ8k2IxRWRdJLt8beKQYhSGmQIrRIDknB7K6lHuizjnv+RrF2ekoB4KMI+kM97GTkjftt46XyK9b4rJMSIoDQSXOp6p8qRXXDoaQ5mlYnVlla2pokKvr0Gahaekj+8ipZT3LbhpQo4cwp03BYlcfPXWqBHRvHY4WT1RxIjE4DnBOLyhpFTklc7h2VuwUEB6Z0jcS6FXZZ+VqwzZznr1X236nIejV+EdFLYapTM6wLOO0sDu+oIpoBoxiqJkdi7EoLvuhxRpm0Cy73acI\\\\\\\/S9BVXKgaRld5rGsQVS2QGWELUEkgheyD8QtHA63XUA2tUK\\\\\\\/Oz36sWXb1VJEwIrTFLZ91BEOZ9JFDbt\\\\\\\/x\\\\\\\/B5tqqAzQR\\\\\\\/Gpw6d\\\\\\\/dSTSn\\\\\\\/ABk31fyeT5Tlh1LE7vL3e5YY1ts0XIZMsLxFPsPlky0EB0vR7Red3GzW3g79pgUPN8iyiMebqyUKCwwH4YV5mv7tGyjo4AeyqLbtuUE3Z7j8+eYyOvN0ZHWFH0w9jwxj9R5uePJfZeYKKyItHPyWV8UZT6usqd4poHVTLPM+9vWhnia1BDwfs4q9J8OJN8Ew42A3A6FrDkrHXr+NXqDEZRxueNwUaPLfrdzClUjM4+rkUxzL\\\\\\\/3cBTtaXGYUg4NRS2H9r3t6tV+4HSbWSLYp80nKEeUDDU2mBW+G3340XRJrJupthYWj7i36wvTlK0lC4TDAmz0C\\\\\\\/o91Xl\\\\\\\/mxBGekJabhz2pc53mdWSTroBb\\\\\\\/4fEYUP6ruQBPF9PaQ9FW8eKhJ8yFO2Z77MTKQsPupJet4r5xVqj5ZSejE1uZGILNUgOwDqmRD5B9crkSWWHbFgkUuNUxmsbyXxuW2CGofTge7JWd5DUH4tHQbjSQxgS29xcB0++TWUNfmdSTszIdVVeRBBtbQY7EiZr5RtPltds67TOMeSmD4LNjBh7KYXozxof5BGee5aDlGYPSmQntULFqF5QTfb1qCv3OZ\\\\\\\/cMT3DP+61QnuQuWmlcBW6K9YaTYWyVOio4TSYsyKacFav+kpiGww4FarCafHtBuPfV83IL6W0A+xkSue2cj8h777fXeAmduC9pMxflQVjsRjNpzEiOjz1nEftEDWLRVfq45JDbiiV\\\\\\\/3mN6ESRnofMFvR7Cb5gKS\\\\\\\/W73YkiIj6QVzbylqmoXigihVVb\\\\\\\/aa6E8Cep3I7xVThfQlfot5kRQ9jMdtMoOnPrtDit+Qwtd22G0NTJMECYdF3BMyzF6JoPqNg2wrWA3H9X7viTDtQpbvowC+le8kgWZXtHx++tmfWgw9aK3PxlkMYrahDt2uV8IDp4+Fa+JisvOww3gUsyj0ksUQ44t1WqMmg0pACLXvJumHUyksaDy7pr5iJJS8TQudjnsTKdPwPnchUlrEkxhegEYiSvYjivTvfbOQCxiFhKmL7u3uamCZ6adbwOmlYkojX1qRc4BB5oG4bhQ\\\\\\\/qrQFpSZCiXX2RkKg68C9go5oiHjlTMOaOFuE8DgwHIXAuUgf6FDvUPKTeGRssRzMc1g9uRRp6icbnhNiISsxGHhMej9Fpbj4V71kLiFThBsPSKs5jCp5\\\\\\\/DPurJA+\\\\\\\/QnXAvH2E4MTEc07RPvBQzhpdoktYUyCVtQJXU74HvLzu42PdsmN0cvJ3iZP74XYLdcV5\\\\\\\/hEpWyDJxFN5+BDTe93EFMb6WR2\\\\\\\/DWJGyOvGSILKRB5UL+Qoc24UI1iuUOws5YmoVSN6zlIyyJ\\\\\\\/S7U6w9V57NkO3OeCOCxkv8uUgfVm2nDyPUKXGpBulaUiLYkxlk2djf\\\\\\\/a\\\\\\\/GScMAheAxeSFtFg185fPbn8R10zSHLrOLErqYORKZqlEdgT8Mq21ImtDA4wrjrOqZng+\\\\\\\/\\\\\\\/kRrXczJ0RQa6GDjtc7DT8\\\\\\\/0CMBycPgdsWrQQI5lV0NpLN\\\\\\\/BT+39cDzTKFTl8Fx3fl7S5LiQ0+40VWUXe9rytGmBdOpL5O2gKrNmu+GClJQSvM3alGVLACjB\\\\\\\/PEhgTHSKD5Sz8gVRZ2amvxZNGpPe4G\\\\\\\/lZqeL4F20igttjtLnPI8Tb8X\\\\\\\/yKpPRhjfNeFt9wEM7JwPG5eEquhcOVDN4134PoGQCGOiH\\\\\\\/vXc64mBdRcSwJQ5PzFFv13xmSVtbMz8W1TYBkTBQfPdyH1hlqPlP8G3Ibg0RVoQnlRny8uixfWV6gNEqW6gSfCMmyb5X1yE+TS50I\\\\\\\/yiYEX1lfbuTduw8x\\\\\\\/8A72AS\\\\\\\/VTYNpjJILnRy+lfJQdF4GES2asreviKGzc+mTNVmuAMq6cJLEwv7zJXD8O3aqngxRdgXh1jBq50GMv4h4cMtD+ej5bAzg5\\\\\\\/MEyu6VkGzpfKleU7VRcEvYP44YLunzPNk\\\\\\\/LGItAIObT2WNQEu\\\\\\\/OHWaudbkwWm0Q13j1N30HdGoUeyvuCBs9KLjApGWZZ8gOV0j7x\\\\\\\/i0ms2VgzYU8W3mg1BWU2FdSNjc8vHR+6IjC8HibwQk7LQhh1+WiH99e+oHFjuiKhll3181054tOFlWYhZGP4SUpGFgWfavsdUebLpIYTQ9fkjghzwBpZ2RnDG9t\\\\\\\/A8VRV6l1Z2vRpRNTp7KuhrfMuoo0kE37q\\\\\\\/emk2Gep+QKOzgsD4EsKv4McX1BWHzCQ7jUw74DynvEgZ72QkQKv4mF6\\\\\\\/cOj8meU7fyjGkOLCwNRbR6Ve+MYdPSrGBmMEVgg9\\\\\\\/+bPedXgzuMr5y5aznJD1g9+LHz0Nvegw7thrAsnZXLNH7ik\\\\\\\/Q0PI48aHfxIpJ4RkHujGiecOD\\\\\\\/ECTh7TNdPncUgIX+HpiVI8I5uKYvlh9W7QhJ4lwC+hQSpP0R4BfhaE2QeFbZtaC7OxiIj35qpqfsybLQlvIaLwmvUqkm\\\\\\\/Ng5JuDU1p1Y2izHvGfFjmA5NeMwL8QJJRwVBvJkpSVuDF8t5zbY7wHS6aa8MxB47tsCzNDQeAYP2ndvcYaydClOB35ZND5T2OQZHPwx\\\\\\\/p46FvbExYuH0k5PuuPmWAdjhgx79dovZjS3a18uXTxQqT6ADsffrYru3TrPc4\\\\\\\/aiYSX7FGSX2e9QzDO0aY1yDhurEwVTnENaVpibeSh3uos4DO\\\\\\\/DO9ZRmqB9y34RO594R90TnrWxv1Ijt\\\\\\\/rwWPMBLnaD\\\\\\\/A7MT3LaFnH35+DkiIDwe5IhJrSx8TcwQLQVEUOpq29D4qCt3beOFLr47nWTlzk+a2AtduwD5AfYG1\\\\\\\/ziD6qbXK4t4WyBWJuNgi4+2XIcDFtcPFt2ZP0DSiquqchrkvGzjKFFI3ld54WxvjqiNs5QTHOz\\\\\\\/Bejv7bkYU8wqAul0x4pcpRu0hDSm83T\\\\\\\/X7AXQQShClet2yfRX8Vc13j92oWu0xvthCafJPRkmbEmxU25vJlJlbb6j6ql4oAbovJ\\\\\\\/TldrHh92hxQOmjwVMAgQJZYIOKOAdbdsbIc0gX9pJfNZJwffleNtGa3LMnCvTbmvvnLfDS4nT\\\\\\\/U1\\\\\\\/rlvwJCNhHSyPa9VyA86NE4rrtxV7z+QKsBGKr7nb\\\\\\\/J9vIoERYU0GEoQCrqvrf48cbgRD7VsHC5DYo2OuqYmuAhbcR6x4\\\\\\\/EnqdvaI79wE9O5kEsmzENmBZ4xlBS2Takh7vv66WkQkjkWA3+WoWmDX\\\\\\\/r+a\\\\\\\/3VenEwYDv6psZk0H1sz6XlL83EtK4XwKyNaivUpUW0JY2BFt6c1L1XxYb9lRip2z6w9oS2rGYhsnu9yCHoLhHIQ8usKJ8HmgjMRQ+GToWbT0LpVCRWI6xAo+pB7s7WnyndfQ1mXu31+ZpDpMjACFwn351m5RbSCGl31ZeaTFdSmdAVwooUV6XTwCFF9jXgP\\\\\\\/2QkrfJF4mqE4MBfl3okEgCjIF5DjzSXnrysV66WMJXQHmnrGaB+E+uoKf6TliCY8+K8XtLjvzzt+DsruwsJk1H8PY8piX7TDxE7gjTvvLuLNDCgEAckYuKxKwJikS2rj2gXgxrcp8NvJC6E+EY8jiNHUouJt26W+CtbSPayTzOXjNdQAJ48igOk7c\\\\\\\/wx7RcgGBLpxMJjrx1tju4f60Vm6VYgDQJG2Y+5bPSjldNr4sjapgJkozwf63qjdEdBvRxQQ4f0SQ\\\\\\\/pr6bMp5uM8ILt56hgT91KhRCIJACQTNjMlH5m8k3TnoeVMDvhzDkhuyTX+cC1ZYJ2\\\\\\\/rMNCooT16SXff7kYWQIJKmreCw5I1uh3pYrTsHvAvkDSzGqJ2dyxHzluRRZNvody4fccpH\\\\\\\/dBKR8LvZBgpQZvRx7ityPTvyiDT+rQZAOh2QnquAPIlRnH8UKbs4h0ehpQF6iHuZgUe\\\\\\\/4TEJC\\\\\\\/s\\\\\\\/0Vz+JdkrddrXb3LLzMjvJ4QntwsNtheO0R9L0w5q4gOtiF6taLgifauVWvx\\\\\\\/KnXawj6oIbDNanAhLFZx+xpv1fpqL\\\\\\\/J+3xtiOip5RL9+0WXxsZ3GdggXXkT\\\\\\\/wYkmltKt\\\\\\\/\\\\\\\/sdbotLdOmOowcLyN5CCD2uG7iJJ3Bm0qAcp5vz7xKGoINjz7hx6dYn9Fi6GBgD3uIWgG7IglITQzQ8kI1m4pYvI4fwEK\\\\\\\/qXU1rOPQfA9XkL4vF4Xl7Y7gBAl\\\\\\\/1b8owxVOJFN+UIpgw8Wkyzc1Gkj5\\\\\\\/\\\\\\\/elC6igTDtXMIIC\\\\\\\/Yso5W0H4EADEmg1gM0unOhOfqWHRUeK5Z1pX6T7bMz0G95ZcdeNFglOdZ4oPX8kN0NMpF0Q76ZWNl5pN+YSSHE\\\\\\\/CD9ZzTiORdg7PWy0SfHu0X+1mVwrwlr1\\\\\\\/YyE3EXF30iiqVcuDe3t0Q6zIe5bDeYa7jrJIp08l2v4fvGW08pEbMGDSyCxyQjNQFSC50ITtSnRkx+ApXyynmYN8r44TmgxsnPRqrMQacnwiZVKtkI10gjEwwtm5Nyd3+gsq4bMYzFdyWVRUiXqJ8agoD7w4cMSo\\\\\\\/D4JSeaR5omjQ2QBlWD+8a4f27UX6yRD9rebzqhelkkANtlCAa1Z+0XqxeO5o08P8GzrjIf6c7O0o7jUWZALT25SAkaBiN6DINnLyUZA4\\\\\\\/hFbeiVWmJ9szLJVA74YEKFaynzFu8Xh6jqAfyONBDmZWHGn+X1011+NfFnLhe42PLTjkMKadNdhnmfNozdPdMBx\\\\\\\/TxRKd+0Djph91o7O9HiXvF5466bpdtQIlr0Qv6kC3pfHEWo2QcPbXN2E+ZseEi3Ms22FtFiOc9rWr\\\\\\\/tFqxwBvlNcsqu2ApVfRPi5fK6TY8XyppZbrL+\\\\\\\/CGzmL\\\\\\\/xN+ZPfoZgRE8hdg7PoOsNiBM6AspooryzFPoB8\\\\\\\/jMs\\\\\\\/mQX20pJLZziOsgMlfbSniW3vqnn96ZySa46rAFN+TPbf4gxqHstBZQctuwrWEZkJXRJ1u\\\\\\\/DqXXRF16\\\\\\\/tkYLlAwgpQtFNAQWNGOvg5sQkO6WKjpoktgVccbPx11qUHEDgKkmd7Ct2X7KFivLc5RTTUDZF+KD3dMU6ssLJQkpVkXfnSvVCe9Uwc4ccnVX2zVMfWhoZvNvCXaFE7zyvYVRJPbE5kSCLimW8F7d9IA9EjLiNe4mxmjp4lXTcDX\\\\\\\/HKN20qAvvH\\\\\\\/37HooIF3OKwkYVBJcZhjRPUYg+d3hsKX05Kp16+6jnqJzhx4U0MhJrq+20300ufwYc0jhJb1R53dZWT2b30ZVBUxfQFF+a3aQZjuXrKvmfv68ZuUvqHucsFXRREUR3KT35FzN+BHZ1vqVwBSUnb4pKgIEa+riH0zFSMN1NvkuJ0j\\\\\\\/Po+dvm7KqC32Ssysmmx9fxux8e1IUqgD5jceKiS8rtNla2QmoULNzMbB2U55Y3tvXEhIUist4wK9o3M\\\\\\\/3Z5rKNoGFXkOUcboQ6a9sA52UC11o2ceTWM0\\\\\\\/iqb31D7oBfoentaogaIwMAUWzoBKQa8ll9Mu59Cv6etTdG\\\\\\\/bnZhOoRdBg406EosUfimsc2Neq48lExbCv45+cSyj1yB6V486wExie6vX5snA1HAJ8KlO69kqSnn+899BHgWjgNCJtokG2NGnp8OeyMHwV+h26XbU65XPNrQXRjjwjZ1XYfvs8bOROFH2dNMzX6LCDlUd0Wf4Ocp+iIV1jECuFle6AA+k4V3XEtZTz02VWP5ywvlFdMKgyhaYQVLA3jN+J7jVoU\\\\\\\/lM3RV5SMANslRzPpn39OOgFNl5foHutLEMwvSSNcJvlenxzMR3texEYXw8LyMc+FS7rVYaz4jHYMSD+2jVQ5+BK\\\\\\\/5\\\\\\\/GSLfb7e39sQeA6uS3OKXU0y17PbQucReVoc6ohTyR+2TeZmwJMSKbmvU9ghoTBUGqcC3Zb8vSxfVuPSduhGlmLt3fMkCY2z46ypG2x+g4YvxZ\\\\\\\/7X2aMeWCCk5jwRA15dkjp69M2uQ3dEugMutdmnqDa8hcE6STccVqp7h2HTpat+26byiWw9xu5EgJVqYyDmtWLrKfWL34qL23kK0MGCHCdgtGZ9peCLgoI7Hg2VtEFrxnb3fC2sfAIAKSUC+mCtznwNzab\\\\\\\/DH9uxtFiCW8wI2Av7hHRj3KHdOP0idgfAF0p0tiJMH21HFVsiiH39cy6NzfFRH5priZg0IJW7XETYGoAu1mITDt3feQewo9yXFp7KR3nnr9kLaxBUYKlpAZVWwMR+Yo12LatfgLz\\\\\\\/9frG7a7i0UcXqc1VPM4\\\\\\\/NulmakQWEO0GeU75\\\\\\\/D7z9eGTfzKU9\\\\\\\/2NsFxybNdv6r\\\\\\\/u4tRPGjJRTG2ceNAt4QY1H4AGTdbZ\\\\\\\/3tBxuCAgrcqGopj8dmdtbXFMaT14OSCvhvs9B8qyfo9L7pU2PWOeRZJo84S2A3uWEqxa2Pp2CKXLbKFEncgr9trDjVgu6XNlWqX2FBokW4jfQRAgXKZiMGexfUiHMshorK4Q1IvOoHVvhXe7UCGa8eRQAJr0rvNtSZug6rcQ4cjPLrt\\\\\\\/Y7CCUdiyUfin7+iWKltr1R33QvtaNSoLeIRZdDQcTjPYD31qPZcFe\\\\\\\/glH7OrUkm\\\\\\\/wsaPo14aQ7p7+A3zAxtCPwjGOSnLxVYnYy2iJKnLfKTtSb6VJeIsdd8pbjRvOXNKSEQcWRByG\\\\\\\/9lOr\\\\\\\/dXJJjgyP3G8b5NOHdNvxIOW8SpsQvr3vmn3JFYnKBHIBDFJtTemVrVHvdxJhBgHoZS9ueEfLK1DU2yPF3tjod085cPGhyAD0Mx9b0OqOOrbol1yiEZyZ9nmcGNzFurKcHf0KcHLxDVuVqYj4WfiJEoRx8C715Fc3i+DTupWJvibt3vbjg2I\\\\\\\/DZAaAioJzBQVm3\\\\\\\/uX7Q3AdUNprQxRG9fORKDS3SpzMivwBbPpCuigpZRjjs\\\\\\\/7MB+VTMO9hCLgzQKkIo1DLa8axW9mb3m45pNPMNuvkuCuJlYgYuzEGhQfaBL8BHi8I0mreMjgyUoCjptiXrTb9t5AlVvt96SLMC2rVPPy2MvjjhMjG9LeX0EOrv4vElLDsJVpdS2TbAvTJ8liK8kXH4obRZQuvfg\\\\\\\/HwqUG3NVGQUO31rDmK7gmlJP3V3LVsM+9KsHa8u55VemF\\\\\\\/wLwHPBQ8QP4W\\\\\\\/n5Jc1+eptY99MwqXgHWF4KXbNUEVAlOzXcdCqtEPwDDliUH3GAL8t9r4cl+9kx9EocTTrNd8+3vW+yrjlK1LbcmXD9N+ZEOBQBN+HPIu15lWwa+ovdDvNhh9AvZpgGU2KDa52t\\\\\\\/lB\\\\\\\/Sm\\\\\\\/N8sqMPTIIynPSY94Bu+VUSbZisYuJwFJTFE906MLIwy3vil8EU8vm0xiUVWIJq7HYxJw+f\\\\\\\/7naLdVsqegXayKX+UIrVI\\\\\\\/0AtZYQzGXw5rya2Xa8v79BzESRTk2J\\\\\\\/r038XTCysTD6qX5vqbMFQy97oi2jU71ItrVdH6juKWzt\\\\\\\/2H1Sp3kugU88Q3wudbMR3aegVToSGGNaFu0hQ1ooFOKwvePw0nZl8zi8sCW0cSUu22CHBKFhyRpaeHjY5joYw\\\\\\\/6jm\\\\\\\/Jp5P2W4EoCDuNBCAulgdkKyJAyCxLICZsIBsZza2ENw4NFEhDutnEj5QCS70VOSfx\\\\\\\/nxGMwF4SfJ3ZjErQTekbUrk6EMsPBTI5JcwjobGDPetNLHHahiXFJDVJpJdb9ZevL1+y5YfvAvVlAzZajSLyargnFALts6TIG5esxw\\\\\\\/l\\\\\\\/psF9UGVP2YeJbN0ODOjvrau5miGtnchh3RFZvNvPlwXgzCDSEiI5WulkxYJ6H4mqrJcFnZNReFdkV7yYSq66MzZkve+HcakzUlvq3umZP\\\\\\\/flf6P1dylzusq3Z8tQQoQmjSjSLmgJb4z8gHeYKqW+4H5NwibHR5GlAN5MDB53Z7qRkvUiUyQqAosfDF0mpswW+yrbUYgbtoZEJrM9qeodL77YErFwvP0gt4xSlnSyKnM8EUoaF07FTp0v36S7WDWMhWIlDZhvdk5m\\\\\\\/D1UDAQg++AN1x5yRRCx7l6flRcM0YvhJsFq6pBnMYKBjdykzfnADWTEOeAYnVgu7z6f1NTvyRddunaPnQZ2NfnP8tR6Qq6v35ijSj5OdF5YmizLtsZwwEc9DtHWgTrI4RSF7UhSPlMF62zsisrqVj+gDmMx5MsOzNieB\\\\\\\/kROjk4uj8jk3QYB6lWXU9ixozjYY1A+Uu9GhPFhffuMnfUr2zLxwPvjMbncXh4YIL2ZxS9D0iA3Ucjy5jZNQUeca1fyrzOD5lvHAuxDUWpzFGkYnMcgpV+2zCEvEfDmm\\\\\\\/4AUGAGjE7LTs8AnN17tQ9gsXXmgFP2COzNjj\\\\\\\/GYBt0OlF7eezXynm9ROk9t++fjdhuXZL7IMQi8XTATepYxiHqGQgTNc9WHLHewh6+56k\\\\\\\/l13DtyzaSrcEWTrGVV8jeBQv+\\\\\\\/+Yn+S71q7roCC0vY8LpBStCBU2dkEJ63N3zEQgswKV9rfbAOyx9yFO0L+\\\\\\\/CtejXAR+b0YSJYnGsRcQK3I5eryHWBqruoXCM9GQEV6V27YSeY4CEtdw0hfeSuTjrkcDd7+Qqb\\\\\\\/+yCJojMsI19dPJwjcIJ5mMQ7Vj\\\\\\\/YRfcPRe+WP4mcyqRV62s2ULL4dyuLDovj19qT7J5F2jMr73gJjYr91JaV8h4606Vc40lay9SbjpR9z0prnuPRmepdbGooksLRF8fkpFsw6RYyaQmSKg3BVZPkgsBbUdI3dohAnJmjqp+NW8Xix7yIw3ZxGHnmHAIcjM15M9\\\\\\\/ihX6\\\\\\\/BXgjbVQXJPCEUkzrhOA2lpDk79a5+rhODidYem8AtIqtR4mw77z86gFFnIL6W5qvrVng31gKKTydIbKEgoCH8PHfoljG7TPmhNFArC\\\\\\\/Eg\\\\\\\/C4809bIYc6Bzy4KWaVGTm8x9c8bW7sUxM5FdzZ0I9bHQNNTtHk4tcDNXrVgAhCXb5uldK+ehLxWCYe1zUsKIR+GYXYzw3WUNqNXc4ZmltwPu4koiMmV2PFZU+FBccfOdpQ43wbMhA8N0IA8eV8ayRf54H1032N35fPHsehcVF1\\\\\\\/BV5\\\\\\\/wI4Z1TQgy2fEFeij043v+ke1jI+B6XXTwLMFE6Mm\\\\\\\/aYdUYdI\\\\\\\/VakMFx7POMUl+xIWZ0JbCprpF1W90NaUlyiCABaFuHQS0X16kx8szk7l\\\\\\\/x4Ve99d4mqKaAoSY4WLq+dA2aM6ZMmiYaToP1GHkSrcG+xQosn28yYfRcUoU8\\\\\\\/WVg5RfGLZU+b96ncuKUUA8zysm4VmZq1G7IhmT7rTNm0AcfItoKx6arfEj969V0RDIfAjiGtIAsniuzkVwoscT0wuRWaIwe3R+jwJU5Tr4eQ7xKDkrm5h6xvH7B7GggKkC4GKncvry9XxzVnsLtgfT70xgQm3rkdqZPuK5VYCEqbrCeiBwUXggwGVzmNFEZoXXXrf70ov1eQQmrUS1TAKliYp116Ea7Ae2W+2z8Utijyq2cl\\\\\\\/a3Rx419KG9XTKJXNbZspBEm\\\\\\\/mW6Z\\\\\\\/u8zjHW1he3XkY4qxfNS5l0xUA2oovucogWugrkl5EF\\\\\\\/8WOlAF3nduJ4\\\\\\\/5STHrLR18\\\\\\\/MDknzgV62shoCUFlEJ1wiHbWqWPSRIg597lwUzg9nbUges15vS1ALz6Mwvr0l2Rotseh29xR9TmczDkXiRsMrTi48mlxSPor\\\\\\\/4IxdsiTM1kSnNFw6jmo2MX6MYpMwX81GQkQB5RRW2nw1hOb9R8n6tNg23tECZhT7DAXAj7ahJ3TmxX58rrQmAY+GGbLDIJ+uCc3zCkdGsORy8NOSi7F46nbKIn4rppMmDkSYs4Oe6dpeyn47EdDLQx07HWxt22QKFYQMhvLIgsFqRKCEo\\\\\\\/ACo1o7LOLyaZUHR+mUtfvvcG+t5nSPSjwsSB5253nL6wansmNZ4891bxZ2cMc30V9mWVnB9ZIazYB07jyP43sYHCRzhh6NEPBhHoQGmkFIlELd1Svbndn3fEHlbyYaxlCHt2cy6M4fButjh6w0lwSws1dGlU\\\\\\\/ZwtjH4483VCvHpCD8AxGDLqh\\\\\\\/FFZzzbU\\\\\\\/lDtNbl68kU0Ej6qma+SrYRCTxLW2NkWORf02Fxv4DdxT+Xq7A5djcIj550Nnfwtat4Rz9XtQbf64uphlZZHsmVAnMpes2UI1iLBMO1IkC7IPnuvngpiZ5PwFSrjIiiVBEBK7NDlnNTV9VL+GrEu7\\\\\\\/VEizmUyiu7OLQ9PRL97hVxZVdTtyQ8NSFgaJDgEl16kGTUwvpDJKSpQimlsxMUG8ONCtIxYbDbxQ+7Ui6CFTdSMcaLFZkx0hjMPMgIJpO4cPeZdlDPEZlJR6bWccmNgFLgdG00o+0sxB7dcfJT0NxXPnKqEvx1cc7RqB3DOUX1XZshlZrvnUl1J1yaRTF6t0D+CBAVlqMDu48joHMXxUDAQjI5\\\\\\\/b9kaf+umqY5zfnAVoWB9\\\\\\\/NWCR441INxvndZsHLyOpDlaFrvceW3S8G2SFKrmuWNeeJmBwRlu9qhLKSAnEiGrPFQg0ZzMepU62YGFiIFo4jJ4Obe2tb5azZPh7Cp7aMTIc7Nlq9Js5i9g\\\\\\\/\\\\\\\/TmcWFdybq2d\\\\\\\/0q7W7J5zY\\\\\\\/Vh5WBnf6Qy17yb7AMKWwWZNEMXqthuO1m9ROariPDZZ388iKmtrX+UhBUE65uLhab6ObNM3pyNHdQG5yhLufOn8PUBP7GUWJOaMB7k9Wa5\\\\\\\/cnQ3z\\\\\\\/OudnPemRpZsh4ZeZo4E9cZsZj9Uy8lnDQsPwwYUc63onu9CXyZdJMk6uCrSpPqHUjXjIs4bXyTrivTrvdb+6NsW44PV88l3ai44So+OYZkT8gTt6wsNKa2hoCzf9iwvYXGPyKAYPUYPwjYCzwKEQS1M3p9cG22CRrxfJCXakHzvbEA9xPPY4sogCfghzj7G6VdqjrLfkymWjKj86z3zUqYnIqpa5pT1ZTlwyHU2\\\\\\\/\\\\\\\/IWlTrTbcF0NfmP9EIGL57v6flOu9XPjGY4erXQJXqI8+XIz\\\\\\\/wFuCNIrSbdGjEbT4waEOuW3xo1SC9X3G2U+GJiwYWjVgZJiQ7gV1VqtfBRwoMS3aLKdWidtldaWCWHkuwfiyYclGvOiyEIc7QB\\\\\\\/jYMdsiiXMAt2+aeTjj7iz9Y9DBWuSDWmHNDpdgcrPvfJBS1TmD6C2C2\\\\\\\/cPx4SGwPBzQsakLTCy8Z0KwP8HTTIC2vJLCzejCwYxD4z+pgZumONkcIC0+srRdOIy7KxE5OX6yk8T5SjdIibtf\\\\\\\/Btf23QvtU5K+ueVM6j7mbRfje6nEgzdaInc8pWLfzVjn0wCdwaFdiHEIBC020WOIZ1I37kmDbv7hkXTZQqS\\\\\\\/nCvPoYk3ThAur03HHjZ2VPraChIdNWvup+BeYr13L1u0Y4hE66XU94ZKrRoWZTYw0FBp423uHXVVzER4k\\\\\\\/t9D1j63e1btb0to6cjxOJk8bW4NHjRojJeepI84oYjeE3It7Q9gISKp8BDK0oVbSK6ffArhWgtdTQShZ3rV7zvZvsS0dRSVU1xcrIlNBDClVTG5g\\\\\\\/DzjXk0TuEdBo4ljWKHBKFmCKT3TKp3LCVTbEby8gag9GcMXLyuC5JK9w0+h2sNlpHKGuq3pQbn\\\\\\\/AuO2nF4v8sZHIqrdSnVsm57LRrE+I7x\\\\\\\/k2E+pYHsVHNR+RJO+ZHeQLlp6tQZHNz4hpIW9YfgXML8ES5vk7kPb2AC+HySODCIRkJIOBigkWFvsPluy1e+s77ZGJ3sfvev9M4sXxvNDTBDVr+a9tZhDV6nekBGcLomczfVDtOF6CxkZTM3\\\\\\\/vfc9UK3LQMfqpAXrRlxV9SnIc+H8FJwrWQx3DLkVg1OxSmgMjlBlxxKuQDXkmgnDRAoFwiYezIaAopLUjegEWQ5mJ1qi3kitgjS6BISXdg5cUaxATOx1PxgNwWXIVXj8fWnJ4013sRTbIDMZW0ZyHrnmdury6idQpCo7YZ9kTM+ksh\\\\\\\/6hipT5d4spuT5M6X8oBf9WMdBCI+KVUsXwX37s3FXlo942HmpzHIa64+2CxxIME2n11D966v7utPp7F2eOKTiBz3NHkM5DxS+c9XZ8\\\\\\\/4CeFwiRKl4jPy2i+5ywrJlN+0lPCrTGC9t0SIokz\\\\\\\/MZkuvwL1Sa0W0a1H2FN\\\\\\\/OWZxcdblBKbBNoPweT25Hfnx0kL7vzIig7XGqr3s5A7fkbE1Eb89I\\\\\\\/dBhM+m50PIi+iu+ZCuFQ8OD9M96WCD\\\\\\\/ytFB9lo81bIb22hoxGV\\\\\\\/Qf9bqNF68PnBtxyNjjJOMFjILNUkWrYhwFFo9W+vXgdBQtenb3pngTHbi1e7LE4evJHmpZJ0hCzjM9KGCDWHNDvq4XLvXThW\\\\\\\/GHVUgKXbcdX3Bj06GZTqhjT7LAZ2lkPA0aS4s8L3fyteIIYP27taE0q5JKfQrulTjAciXTsb7MNTOkAO8pBGdEdlSRfWm55NaOHQbWncc7\\\\\\\/1aDAlINuYjsNgw+9J3bSWIrpuq4ZEhKwg0v53Mvr3XC1M9aiQT9HDLpaVUfgSRRWS1VKvJZ+YGOQcdXBJRAPYWKItSSrFAy\\\\\\\/95+3x\\\\\\\/igpSHT6m6aK\\\\\\\/8a9ex6XR30sbIwG2TUM+n+CL6qGNsCicYUlZDDTQ13A2cRsN\\\\\\\/26kj8nuNCpfKUhzH\\\\\\\/PTwaEaAuVG36Ui6UY18RyvS8GT8MJ64EMxpzpgVaDkAcXaBe2lGnyG6b6ywcPEl+eeBtyOQQZvK+Ngx7Mv1LKAJOSndaCvHl\\\\\\\/vEJPkB2o578YJogvjX3pDOQhVFftM\\\\\\\/N7Zdj8zKcrpr17TdJ36GsDVlx3pp12EjGaNLj6U1RiPKONtUPwA2PzikkZbcEi3RH\\\\\\\/lAizk4im7rLKHp8HuQWQBMUwEIZz5\\\\\\\/kQjASLyYnyBTz5CXJurSF7ikxw0\\\\\\\/4\\\\\\\/KG+r2cvZL\\\\\\\/uOs13LSk37FJK5OHV+Zb3UhCYw8KJBfcto0OQsOhtx5q6DOWLwehl3UNKZM+Sp3vN1lvTuv4s8vj\\\\\\\/1FpA5Mj18Az8ViXq6ndcW7oTCSH\\\\\\\/TIGffFegILb\\\\\\\/3bo9cqyDKdFAtbb69wlmt8tmHF2wTJ2\\\\\\\/ucb0hZDXFR6Qvu7JNyticjrBjBkvRbE2UgJ54VZhQddGobWKFyId3Sy+FC0sLJ1SEnPwawdJVW8SqJ1p6F7a+lWP71+gBjtYI8RyBSBcOM8Nl0E7+tLtFsByYZs\\\\\\\/amRY1Y0BK\\\\\\\/rJWjAtjuxVkTEEgoz9ROmndb3MycTWs2LBaarMZ2fg+xMeTbbh4NMm\\\\\\\/OMBZuKvrlh822nfNpfbwY8WfvbmC\\\\\\\/L4qDk8PKaQX6UM1S4Hqj7lb2fgsC2J7j08FuOAnDttYb9ZTpmOmzHdVlitjGs4QaYbTgKk1wIpm\\\\\\\/NMjXAOTgSYInwOMHwD475LvSHO0DPBgGhcsfgIWWkOZfENO3foU47ZS7CMgNlSlNnZ15X1lDLNWjYAniZn7QfS1CZVAnfAvX1FqRqj3\\\\\\\/g6kQ7qurZEt3pRhRwIQx1o2cdwm1tZnXhBT\\\\\\\/FEaeEtw59mZhgDjgZqEG4q+G5yyygp\\\\\\\/uQBPaZnG56PEbdfHK\\\\\\\/3k\\\\\\\/2Dn4rvRV43PzIf6nfbtGRQeQ0qALONNLVzfpNXKE212EclByn8UcgdwftJPXjetJBqsoIScaxXsizIL8PQ15aCN9Yc7mQewlWFdgbp8W9oVijzTgEtnmFQIJXjmP3Vgd9DoEbj+5rZq9\\\\\\\/oEeRF7xGbMVopv2Nw8oE2rXEkdyggS+PUsbTqj\\\\\\\/ABpxf2EG5dyl7LfKXLZ8jmckE8M0h5zXgF\\\\\\\/Ol3D7dXM8W\\\\\\\/BsKKAghUSq038avKDyincRsfiWSh8pg2gTulp9IXtZ8as97V+qOlRj\\\\\\\/RnhGHY+stlBp3PRPFJC3JAkI\\\\\\\/qqlbuX2T4sB5w9RdbFmPqVsBNk934zTBqAd6RhGi0McwauKzme8P1LhJK8+eaWhrl85RkSayPVY5SGq6pHOsGUbUFx7w9F+sPPDdEw3EQvLzW8xst2DBqmAzETcVcWRpwN8wpr6Bxo3HtsEZRVzjcGKJJ2Ken6UYvGupvtoyNOCd291ODEwN\\\\\\\/872K0L0IUZw2s2dS0fE5ox+RgLcmSDTo0Tt79E8iEvgdA9G6Z65zKsMQukvSaJYxV5r1ASj0ILc4uNrd3cB4ybE83sYxxRmg6S7Os1qVKkpcyqeYj0hLPAdemUPDb7FRkr4pyEupNqK5lJa9IyUdOFb2ZAoGhXDxT5NBeMISD2o8aVXE\\\\\\\/Mw\\\\\\\/NAI2CdrB2rvABkGD+tugB\\\\\\\/5RA7udEq4wCTmz4Ffnf15gfRfH\\\\\\\/o+z0\\\\\\\/6V1HuPHs61CK5f5i+v0W+ZvWobrq0u6D6lf0z+YM8E+eUZaB07ccCEjtlpC6K\\\\\\\/Pv0sLgUZ2m3dmL0honc1ZHNd4KmnQwIwYgclVFZ7SWVb82gtQ3FLtm+\\\\\\\/wzvzDXzTKTdIYCspdB9iRKwEkIEmXPvDU3SfupFSymmbdEX2+7XCzUeydxbW1pehVQG86zNgyfIhYl7Bdp4a4VeySddCsrmOYIZvfd1oSRln9r1xXAHydU1BS3prcZWjVo1wELFr7T1IgzFxw9F2w2ZmE3TMhXYWOj\\\\\\\/J37FTK3NKJpZJVNzpvVGWtn4sK0anyl8cdbUdFOhPVtqeNuA6ZEuf+Ut+eoQvew3ROG3QCz+3VN6t4ZHj6UvZBqlmlVTYO\\\\\\\/3Z+A5ZMrjZFISp3wmQ9mQGjc3QAr8VOfsBYGkeCgArAVDyazSSTgdc9SWdHDHGURg+FpXPH8csUWxfNbG3VwSFW5VCEA77hYqUm8GHZQKnj0PhssDrls09byj9iKj8yEBwD7\\\\\\\/UI3+G55A5k4hPCQ3W5Gbh64yTg35YNewU+bkSYRsVTe2lsTiS8mdBl9NdBqobBVZlkz0PBmHxoRIxsQDKDgvKV9YqEFf\\\\\\\/ubLe2qlPtUyc+M\\\\\\\/fhJ3BQEYseOF\\\\\\\/H\\\\\\\/3FfuV1eAuZzjNF8irnGg9F9I6sNUbIN1WLeqhBLmtFa\\\\\\\/qVrxYO7XtZwBnOiyU2xHWS9hL8Yve3yW3ZlEXv6zMvu\\\\\\\/dDLw6DfRz66rvutoj\\\\\\\/asxnXx7BazrhTbc7xMCMC+4kssQtm6jJLV\\\\\\\/DV919l3G3ykbW\\\\\\\/qHLQ4zIqMMJsX3WTLAC\\\\\\\/CcgMSn+IXhcOX9L4Ew4RYWALlT0LiZHVYw1xel7kXUrruyQ2+0Jflj9ojMAmrybutfaleTQmQe0brK6cpd8m8jWwRfquQTPsDfqjFCM68rUQKQ7GrId8xuQkOGox3zTy98HyuFCBTyEW9qdGg\\\\\\\/S55q4GCfm8C3Uik1OlrFlPczMJnjF7uX1X7G3HrZTgwEkqfhVjDGxKn2AAy1dQO3KeW9LteJyODTPP+RHpAnwLx66eW9sM4tVfTvSmjsYcl0mOqepE4acJAxIxP9jnG4Z4cluNsTiJKSfXRp0XdCvCFHLv7GL+SCfjCmjje5MUFjIxGeZicTenj\\\\\\\/D0ZVpfm6bjU0lps49rKEfnLMC+kHaiVKbsDwmxsHuX0b1gQfqaJh3FYj7OArDmDzpC+2NPDoWLQrbP8lHh3HtWf0V\\\\\\\/JKjjbqekjRW3ZgG4dMPZqIf2vvw21H58Zwc7NHm8npRiyYEJ\\\\\\\/1N2m6hGX9LTXqir753ocsT7bu0Qb8ZmXBCqSrt25\\\\\\\/w5ekuFkbguBxVIrQPuLZOMQaXw\\\\\\\/kr1Kk9Au8GxqS8ytxGC2ehrHAVMvxuCysLr51JlL+KRSQGhFD\\\\\\\/AK8SXAGt8jQorRBBPGJotGiIPNmENkHlnYsW8lCefemIUUEJHzcY199gwITZtWYriXYus3TxW+Smw\\\\\\\/WalZiVKblVoImgkEury99HOqWnRWMFUHT6F51zwwgVQejhDFvx8mTqC2NcLTGEpqS4TxlSazz+MMRJbvmFizW7QbqzsJj2BuGHDmHVbTLwPztzBYJh8wpmSVNg3GjySW03aNXpfxAg+MMkcKTg5URjDhniqW2a1OAqR551b3wuqXd0hT1xvsEHaXcAiUu+s31Np53jrPBUQ7MICz37eLnrVwBGJQ8OsgZBxZNo3N4aGD4UoT8clpJfdadW5+1dwVGjbsEOg9jdWREQ9FNm1ohcBMrsYDGWKwlPrBKNkYoSYt6XYugDw8LvqMOPZLOQuECEKOo+DmuRHRTdlOAugudEJBDF27ES23qVU1mKY9I5x6txiQs7Z7qWPRKPHT217NcceaMTx3EMXY6MHg4NfgCIlx\\\\\\\/dDIjKWKQRQhdku2z5HxWGvDV0s2mhjx8bYizmQVIJEn7A2KRCFyjbKcBuJteHMta2KknHkfMYS0mHBVHMT1uvfy4IstVF\\\\\\\/5f3VoKZs4EkZA5blcjtNxhURxNUMqnSvMaM6fPiXIfYE7XtU3Jnbc9LQCfjN+tpWRlkk00wt1ZpEzOTA0XKMeGPEwHCQfEmjGwY0tg0AXUi3f5F8U3vhTPR4H+0GbEIrxxqdUrAOyAHJLsonEIFqWJyeO+ZkSe2NZbe0TQu74xPUGEB\\\\\\\/DEE0PlW41DDgzIdjynYf7UR9GSCvwrTCvjxurvQDasSBwlIA41ClJTqy90DDWvMa7NQxD+cMEGawhGZxy1EP8uaZ+Rnzlea2hFUZ174eCyK\\\\\\\/NK74YZW2I0gIXBkn40gxEWtdCfO5LjjCONRzQG091JDj3PzymYWcwaXj0SYhuC97LHDi\\\\\\\/DpGCu2MlKkSowXGuBQNC9mWJSSDcpJJ0M2xMade9q5r40hMUe32joNSzbKuuQr7FzIviqoqDrSygy8BbcVkAiqc4QZ98Bzw4ICfjnB1XE6ND8pu8ruGwv3Esb90qAiwtgsFJ\\\\\\\/tyF4lEMoDpMKl23d\\\\\\\/zmQ4x1aCrOua9y6p6vPHLFw1eXwFmlzUPm4Ho+Tw97jYIbpLf+t6e7h1bnJ2BY3vpIcyPGdESJSczSe1vjHW4n7sLDyAO5OIQv+dyQG+ufkTTEUOJ5RLUKOGuVxrKU4WujJ\\\\\\\/Ce02BPlcy1u6sJ+Nx0ITgUbr6o5YMdgjQ7Mw+p9GmM2ko2GB9vNPr7u5gQmUtKGm43eHQ9Oj232saZvEem9c\\\\\\\/KmJC0o+u6GkxGbGaxEMyw9vC0QeHFt1WD\\\\\\\/H8tQK1ijLjn+97crrWrap4DmxYx7s3Jm+bSedinaQ+w6kim1o3Z3zG+gJEEFQi2xDushr6eVmow5xdkhW1g8HLId7NChs4JSg7OtT8pVLWo+0i6\\\\\\\/eN21JECQtCM4gmzrEg9a8ceBS5o1j3RWBnDHtfgXlhEZJI9ud4ewNr5Oky+cset5rp3GxOjN4GVd1IBOCGc2hLMwC\\\\\\\/G3mpzl7vi1hAc3TOQnPtzp3fnIZKLr8i1ZyzTNwI\\\\\\\/Ikftb2oU0fEIfz7RG91R1xELUqO8MUUqwGv8RgrTHc184tuNowC+meZe+hF0bbKIdL4gRmHjhP+L+Ke1feLOwAPbOvNW1j2s1sq+a9HZVAhC9yzd5tJVcl28qBTR6W0j7yOSUXLlRIbgu5GYq5lV\\\\\\\/r3QzwdAqsdNWPC+fd5lRoaKv0mu9MBmcFhdzB7\\\\\\\/YS1dkDU\\\\\\\/SDJRgulrHEBcK9o5G65tOZ66SPzyUblh7qtKxouzLTAws55wV3P2x3akd86kth0MPrb49FPf95xYsJh1orczsVKBkAVxMSXsiIViOnrZTcL7BQ4sRPI8g+z0oMkIVzfJeshqCTU2CBNH9GSeTWdR\\\\\\\/VPWlALXMfli31xtLshUiOHP986saS0gEARcfihMfaXWupIg+Sujd8aR+LEZPuFegIv57edBgGgbHUQnM\\\\\\\/3VE5JtDpWPUSVbUJ4Q+OOfX9jIiilsC91enXuil1xH6Hea2ykmFv0gZPLJgZ4eRt7WUqClNPQIxlrcTGlpcoxiV7lLaexWzFqcB57N+fxA01ekR9ysSwGD5K9vxQUWCssK3uhlqI1e2XBya\\\\\\\/MwBNt\\\\\\\/M2aGXTJc7PKbMAo2JxlZkv3zH+BlPYYXj4JowZvWI3z5pGI\\\\\\\/rW5ZKFWhggHGSQvEiAYYFX1k45fhJjgu2GicpHwn4jXHTBx+LNCNmMck79F4bMB6Y7o0KdwgvP0+5EtZvIFXyzyC2dc924FNdO7dUYADcWD6wR54RhH+HUVfUpkMV71ngWFjiPrfYEohAI8\\\\\\\/W990XzD73u\\\\\\\/jEM6jOZKUh3Y0nWcvBZH+yPgH5aPJ+JlNW8aeKvD7DG1jGGaMTw2MaO6aam2cjhpfWWg0T6VVzEIS\\\\\\\/+OaMFkswWxYFxhBgY8ifH1vzE+ql4gPEcuLA8OwofmO\\\\\\\/BUr++Il96DNdkZ0BkiOATnSSduLY27cVsO401MXYkaseVcwG+gnn7AymhtJQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022uvMNAMAYyrVlxNagBxgI+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224d3d55d49b87f1608ce0c2b2466b860fa99089ae9add953268820b092d3e9bb3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hw3HOnBzpHDB\\\\\\\/JAoMNjAWTF\\\\\\\/sjsJsEwjwIlxy\\\\\\\/5RJMfZpieTs72JIm0VaGfLjzVEfD8wbFH8kVAL6shYUc9GY4dwiA4fSHA3Vyw+qc8GitLpuOnXSVrpOwFetVJIQqRldkyEU07wFIppwe4AqisKV0sgBpIvMiaNMivUnHUyI6TkTWgFquzQyzAjzbKCrXJi5w7NxTGAzGohmvH\\\\\\\/OssnIIxdsUQnC61QfpBdcsJl+k1RZL3ylY41MaRqaFKzXe2Tpj\\\\\\\/kN8GgGX2Nr1oKNwa2RJDT3HfwY51ahCPLo3bs37elJvnpfEYmLQ1PxD5aDku38ZlH6y9h\\\\\\\/zM+5N+YCDyEI3oz6xxdUSAH0mWT+gGzUNoEJ1D9H1JDTyabLalUmldahADVQnzytgh79rRFxi9sJpvYn8j9CNEebadJclsFMj01\\\\\\\/0ZdQHaZROTyRJjMpKdjjS+lCy5W8HchH8pZ6JBKUl2ycLHsOy6gP1S3RftWIALQlm2VRJljtXd0hliVZ9lfSoOPDplhWnvJ2Vxr5IsZvMTm7h+wAkqz6Fw7cAqNMcY1YCIJ2L9OmLWmBRmXoMmw685lNAp06DIBO4jZuKWxicDXQhu9pcNPbRJr+ZkD0pUTtkTXeEll86Clb0xgsTK0nH6v6UDZj+\\\\\\\/Edshe7Mb7EQVB3AwJpN6iSG\\\\\\\/q3tYUu6U6zOxtnNM1q7ioYtwIlp3Vujq9rK9fVO3PuWWT86oEZDCX9Tz6JlugmefbnAvhc\\\\\\\/qoxUsy8QnEs0snNVEQDGr5TxYIPlRIP\\\\\\\/xJlsp71Z2zvyxZFBzHWu1PDvq84vAz7OsOK58ksLHz+fArMd7C2xBTIlUXPNuoOQsKd5izmFL81mLW5s1VEHx71Ug8btknx5t4+6hM\\\\\\\/1JeRQM44aIehk5t0NmlPldYiySVZrFRMskKYTiUYHrgkI\\\\\\\/exxSozdtrqw3nOAMFVSCC9+9ypnGdmkfFxjAjgt1ATrZY1hAy1bRT4IhTMhMmYNyi6ZrsRV9rzwvEiqUUtR7xsGMQ+CIVDHF6HmXK+9Z8KFSvG2xdprriPZQ9hub+b119jpK9HcONC64QwmCRY9XvyG5s8h9eZdAHq8ZPU0hRzRV7YDzqEAW9SUTA\\\\\\\/EkjYsIlgB\\\\\\\/UqHFU46rzJo2JzhKpj6l+dTThfxxrRdyVuu22CuedCIxqdZG1siVteHQl\\\\\\\/lgMPU\\\\\\\/\\\\\\\/\\\\\\\/9BDeLAG22Gw4J62pCNuvHCfp3Q4r59rknkqy6NuZzN7SKlvUWVU\\\\\\\/Xa45ja5LH1HTISI7ZEYh7BpTT4cJSsUL4nbXN1gfT4WTj0kIiT0bL33hkZzovw+aIVx8kawl6GSeRu2bHo2Dm+Z0\\\\\\\/tqfWS36IRcQfrmF0vqAXp6WLJjZHHEzIz1\\\\\\\/A+3X+7NMBHymIH16CvFLQqevUKT+Haxzm8R3MNtfJOzrQMUP6ja9lUSkFGRgIov+PWkX2HaTlhDjqAV\\\\\\\/g5IXenv4uxbJrX5Weuf7dd280zB5+GWr8QkLn53c2FbOppZEFXejSPEHueMBnSp5kedZFVSLedMJR1ePyqbR\\\\\\\/DCxf8lsWbrhnkEu2Fs2enh8RTZicDB8CoLQKIZK9LewWSqNgVVbyi6Z5TfS88lcXVkztQepfbUgrnFwCIQOG6wZfFDIOaMU8AY4Vcgvl\\\\\\\/n6RA4+\\\\\\\/sFshtgEQbly3qmx80zPAcAcz1hn7na9borD4y4PWbxblYPs0f9LvV8xaPDEejen\\\\\\\/kLr8UCJMM\\\\\\\/9SuzEgYmM6he4l0NumTPb+ymG81d5oEasBHJTsLqB4wMQP4Lqz07K11ED66\\\\\\\/\\\\\\\/kR5xqALWXEsNmQYqRWgNLZcAGD9yUxayNYBwa82yOJtfFVgo5YgZq+p1RXqjmSDuE\\\\\\\/kSZPawgEJXnU1Sk071wwbtcDQtjA6tWPiaWSdp5JeIxncwuf63qfDL52bjAbaPt3b4PvzPaW1P971m7XaZIF5Inn\\\\\\\/SQ+5z2HsZ3rHZYq3sS0iQNJNnJiN4x8DHBokb2HB6IID5tm3d4CgOeT2ZZPNx3hNiD0v4g1Vhw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hhkL4MhngmzpqFU98KIs\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002280cfcd4c0a2498157dd362bc0da7d8c40b2817752fa4179f9a1d8bb2d46964a9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494900 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4159" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZwSKFv5a4FW8BQTCMMrfwHDeOzu0wZ8cRNbLStbz63I" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4159" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--6QFhDMNFibiGbc4eyNTvOCjAi-llW8tgwB1J9RO7gU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4149" title="Loss of rps9 in Zebrafish Leads to p53-Dependent Anemia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4169" title="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494900" id="node494900" data-pisa="ggg;9/12/4159" data-pisa-master="ggg;g3.119.400755" data-apath="/ggg/9/12/4159.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susanne Tilk</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alan Bergland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron Goodman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Schmidt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri Petrov</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharon Greenblum</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4159-4168;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400755 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494900" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susanne Tilk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BTilk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusanne%2BTilk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susanne Tilk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan Bergland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Virginia, Charlottesville VA 22904, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BBergland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergland%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlan%2BBergland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alan Bergland</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Goodman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BGoodman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BGoodman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron Goodman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Pennsylvania, Philadelphia PA 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Schmidt</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dmitri Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri Petrov</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sharon Greenblum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BGreenblum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenblum%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BGreenblum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharon Greenblum</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4159" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494900" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494900/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4159.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494900" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494900/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4159.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494900" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494900/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4159.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494900" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494900/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Evolve-and-resequence (E+R) experiments leverage next-generation sequencing technology to track the allele frequency dynamics of populations as they evolve. While previous work has shown that adaptive alleles can be detected by comparing frequency trajectories from many replicate populations, this power comes at the expense of high-coverage (&gt;100x) sequencing of many pooled samples, which can be cost-prohibitive. Here, we show that accurate estimates of allele frequencies can be achieved with very shallow sequencing depths (&lt;5x) via inference of known founder haplotypes in small genomic windows. This technique can be used to efficiently estimate frequencies for any number of bi-allelic SNPs in populations of any model organism founded with sequenced homozygous strains. Using both experimentally-pooled and simulated samples of <em>Drosophila melanogaster</em>, we show that haplotype inference can improve allele frequency accuracy by orders of magnitude for up to 50 generations of recombination, and is robust to moderate levels of missing data, as well as different selection regimes. Finally, we show that a simple linear model generated from these simulations can predict the accuracy of haplotype-derived allele frequencies in other model organisms and experimental designs. To make these results broadly accessible for use in E+R experiments, we introduce HAF-pipe, an open-source software tool for calculating haplotype-derived allele frequencies from raw sequencing data. Ultimately, by reducing sequencing costs without sacrificing accuracy, our method facilitates E+R designs with higher replication and resolution, and thereby, increased power to detect adaptive alleles.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/evolve-and-resequence-0" class="hw-term hw-article-keyword hw-article-keyword-evolve-and-resequence" rel="nofollow">evolve-and-resequence</a></li><li class="kwd"><a href="/keyword/allele-frequencies" class="hw-term hw-article-keyword hw-article-keyword-allele-frequencies" rel="nofollow">allele frequencies</a></li><li class="kwd"><a href="/keyword/coverage" class="hw-term hw-article-keyword hw-article-keyword-coverage" rel="nofollow">coverage</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/haplotypes" class="hw-term hw-article-keyword hw-article-keyword-haplotypes" rel="nofollow">haplotypes</a></li></ul><p id="p-3">A major barrier to understanding the genetic basis of rapid adaptation has been the lack of robust experimental frameworks for assaying allele frequency dynamics. Recently, evolve and re-sequence (E+R) experiments (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Long <em>et al.</em> 2015</a>), which leverage next-generation sequencing technology to track real-time genome-wide allele frequency changes during evolution, have become a powerful step forward in studying adaptation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burke 2012</a>). In most E+R studies, replicate populations are evolved over tens to hundreds of generations in an artificial or natural selection regime and allele frequency measurements from multiple time-points are used to identify genomic targets of selection. To date, E+R approaches have already been successfully applied in a variety of model systems, including RNA molecules, viruses, <em>Escherichia coli</em>, <em>Saccharomyces cerevisiae</em>, <em>Caenorhabditis elegans</em> and <em>Drosophila melanogaster</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrick <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burke <em>et al.</em> 2014</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pitt and Ferré-D’Amaré 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Orozco-terWengel <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wichman <em>et al.</em> 1999</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chandler 2014</a>). The ability to concurrently observe both phenotypic and genomic changes across multiple systems offers the potential to answer long-standing questions in molecular evolution. Careful analysis of the patterns and magnitude of allele frequency change may reveal the extent of the genome that is under selection, how interacting alleles contribute to adaptive traits, and the speed of adaptation in different evolutionary regimes.</p><p id="p-4">Crucially, however, the power to address such questions depends on the replication, time-resolution, and accuracy of allele frequency trajectories, with tradeoffs between these often incurred due to high sequencing costs. Recommended E+R schemes with even minimal power to detect selection involve sampling tens to hundreds of individuals from at least three replicate populations over a minimum of ten generations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kofler and Schlötterer 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schlötterer <em>et al.</em> 2016</a>). Since individual-based, genome-wide DNA sequencing at sufficient coverages is generally cost-prohibitive, most E+R studies rely instead on pooled sequencing (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burke <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Illingworth <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Graves <em>et al.</em> 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barghi <em>et al.</em> 2017</a>) of all individuals sampled from a given time-point and replicate. While this approach sacrifices information about individual genotypes and linkage, pooled sequencing has been shown to provide a reliable measure of population-level allele frequencies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhu <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fracassetti <em>et al.</em> 2015</a>). Still, forward-in-time simulations suggest that each pooled sample must be sequenced at a minimum of 50x coverage to detect strong selection (s &gt; 0.1) and even higher coverage to detect weaker selection (s = 0.025) (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schlötterer <em>et al.</em> 2016</a>). Given that optimized experimental designs often involve &gt;100 samples, total costs for <em>D. melanogaster</em> E+R experiments that achieve reasonable detection power can reach well above $25,000 at current sequencing costs. Thus, achieving sufficient accuracy remains a major limiting factor in capitalizing on the promise of E+R.</p><p id="p-5">The short timescales for which E+R is most appropriate may, however, facilitate ways to reduce sequencing costs without sacrificing experimental power. First, there is a growing body of evidence that in sexual populations, the bulk of short-term adaptation, especially in fairly small populations, relies on standing genetic variation rather than new mutations (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Graves <em>et al.</em> 2017</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Burke <em>et al.</em> 2014</a>). Many E+R schemes involve experimental populations derived from a fixed number of inbred founder lines (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Orozco-terWengel <em>et al.</em> 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Turner and Miller 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jha <em>et al.</em> 2015</a>), so the identity, starting frequency, and haplotype structure of all segregating variants are often either already well-known or can easily be obtained by sequencing each founder line. Tracking only the frequencies of these validated variants can still provide enough power to detect selection, while no longer requiring the high depths of sequencing needed to differentiate new mutations from sequencing errors.</p><p id="p-6">Second, at short timescales haplotype structure can be leveraged to provide more accurate allele frequency estimates. In the time frame of most E+R experiments, recombination does not fully break apart haplotype blocks and disrupt linkage, and thus genomes in an evolving population are each expected to be a mosaic of founder haplotypes. In this scenario, recently developed haplotype inference tools (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cao and Sun 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Q. Long <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pirinen 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jajamovich <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Franssen <em>et al.</em> 2017</a>) can integrate information from sequencing reads across multiple nearby sites to efficiently infer the relative frequency of each founder haplotype within small genomic windows. These haplotype frequencies can then be used as weights to calculate pooled allele frequencies for local segregating variants. With this approach, the accuracy of an allele frequency estimate depends less on the number of mapped reads at the individual site, and instead relies on the discriminatory power of all mapped reads in the surrounding genomic window when inferring haplotype frequencies. Haplotype inference methods such as harp (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>) have been shown to accurately predict haplotype frequencies at coverages as low as 25x, and simulations of pool-seq data from a small genomic region at fixed read depths indicate that the use of haplotype frequency information increases the power to detect selection compared to raw allele frequencies alone (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lynch <em>et al.</em> 2014</a>). However, these tools have not yet been used to infer allele frequencies from real pooled samples in an E+R framework, nor has a thorough analysis been performed to fully examine how the accuracy of haplotype-derived allele frequency estimates scales with empirical pooled coverage, across many parameters relevant for E+R.</p><p id="p-7">Here, we focus on defining the accuracy of haplotype-derived allele frequencies (HAFs) specifically in the context of E+R experiments, in which populations are generally initiated from tens of founder lines and are evolved for tens of generations. Since haplotype inference will be affected by 1) read depths throughout genomic windows, 2) recombination events, and 3) missing founder genotypes, we begin by leveraging both simulated and experimental data from <em>D. melanogaster</em> to assess how the accuracy of HAFs scales with each of these parameters. To do so, we introduce a new metric, ‘effective coverage’, that equates the error from HAF estimates to the expected error from binomial sampling during pooled sequencing at a given read depth. We find that haplotype inference can significantly increase the accuracy of allele frequency estimates over multiple generations of recombination with selection and with varying completeness of founder genotype knowledge. Finally, we extend these simulations to accurately predict effective coverage in other model organisms, such as <em>C. elegans</em>. We show as a proof of principle that a simple linear model can predict effective coverage with an R<sup>2</sup> value of 0.875 and only requires four parameters: generations of recombination between population founding and sampling, average recombination rate, percent of unknown founder genotypes, and empirical read depth of the sequenced sample. Additionally, to facilitate the use of haplotype inference in E+R experiments, we introduce a software tool to calculate HAFs, HAF-pipe (<a href="https://github.com/petrov-lab/HAFpipe-line">https://github.com/petrov-lab/HAFpipe-line</a>), and a shiny app for predicting HAF accuracy in any model organism (<a href="https://ec-calculator.shinyapps.io/shinyapp/">https://ec-calculator.shinyapps.io/shinyapp/</a>). We conclude our findings by offering recommendations about the most powerful way to integrate haplotype inference into E+R experimental schemes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Establishment and sequencing of founder set</h3><p id="p-8">207 iso-female <em>Drosophila melanogaster</em> lines were established from wild individuals sampled from Maine and Pennsylvania (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Behrman <em>et al.</em> 2018</a>), and inbred for ∼20 generations of full-sibling mating to produce viable, fertile inbred lines. 30-50 individuals from each line were pooled for DNA extraction. Whole flies were homogenized with lysis buffer and 1mm beads, and DNA was precipitated from the homogenate before resuspension in TE buffer. Libraries were prepared with a modified Nextera protocol (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baym <em>et al.</em> 2015</a>). All samples were indexed with Illumina’s TruSeq Dual Index Sequencing Primer Kit (PE-121-1003) and pooled equimolarly into 3 sets of ∼70 samples each. Each set of pooled DNA libraries were purified using Ampure XP and size-selected to 450-500 bp with a SizeSelect E-Gel. After an additional 5 rounds of PCR, DNA libraries were purified using Ampure XP beads, quantified, and diluted to the appropriate concentration before sequencing on the HiSeq 3000. All sequences were deposited in SRA (BioProject PRJNA383555). Adapter sequences were trimmed (Trimmomatic v0.33) and overlapping reads were merged (PEAR v0.9.6), then reads were mapped (bwa v 0.7.9) to the <em>D. melanogaster</em> reference genome (v5.39) using default parameters. PCR duplicates were removed using PicardTools (v1.12). Base quality score recalibration, indel realignment, and novel SNP discovery were carried out using GATK’s HaplotypeCaller. Only bi-allelic SNPs segregating in the 99 lines pooled for resequencing in this study were used to generate a founder SNP table, simulate reads, and estimate haplotype frequencies.</p></div><div id="sec-3" class="subsection"><h3>Generating experimentally pooled samples</h3><p id="p-9">One male each was selected from each of 99 inbred strains, and all 99 individuals were pooled for re-sequencing. A second biological replicate was constructed from 99 additional individuals. DNA isolation was performed as described above. Three separate libraries were prepared from each of the two biological replicates using different library prep methods: [1] according to protocols described in Nextera DNA Library Prep Reference Guide (15027987 v01); [2] a modified Nextera protocol (as described above) and [3] a Covaris shearing protocol. Final results from the 3 library prep methods were similar. All libraries were size-selected and PCR amplified using two replicate PCR reactions and a high volume of template DNA to prevent PCR-jackpotting. DNA was purified, quantified, and diluted before sequencing on the HiSeq 3000. Raw, 150bp pair-end reads were trimmed for adapter sequences with Skewer (version 0.1.127). Read merging, mapping, and PCR duplicate removal was performed as above.</p></div><div id="sec-4" class="subsection"><h3>Generating simulated pooled samples</h3><p id="p-10">150-bp paired end pre-aligned reads were simulated from a table of founder genotypes and the <em>D. melanogaster</em> reference genome with simreads, a software tool included with the harp package (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>). All reads were simulated with an error rate of 0.2% (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schirmer <em>et al.</em> 2016</a>), with simulated sequencing errors receiving a lower simulated base quality score. No read trimming or PCR duplicate removal was done. All SNP tables with missing genotypes were imputed before read simulation.</p></div><div id="sec-5" class="subsection"><h3>Haplotype frequency estimation</h3><p id="p-11">All haplotype frequencies were estimated with harp - Haplotype Analysis of Reads in Pools (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>) in a two-step process in which 1) a likelihood model is built by probabilistically assigning all reads to haplotypes, and 2) maximum likelihood estimates of haplotype frequencies are calculated in discrete chromosomal windows, given local read assignments. An assumption of this method is that there are no recombination breakpoints within a window used for haplotype frequency estimation. However, with a fixed window size, this assumption breaks down as the lengths of unrecombined fragments decrease. The distribution of fragment lengths at a given generation can be modeled as an exponential distribution with rate λ, which can be calculated with:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-1.gif"/></noscript></span></span>where L is chromosome length, R is recombination rate, and G is the number of generations of recombination between the initiation of the founding population and sampling. The <em>q</em>th quantile of this distribution can then be calculated in R with the function qexp(<em>q</em>,<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-2.gif"/></noscript></span></span>).</p><p id="p-12">We allowed window sizes to shrink over successive generations of recombination, such that only 18% of sampled unrecombined fragment lengths were expected to be smaller than the window length. Various quantiles from 5-25 were tested before choosing this parameter (Figure S4). Note that haplotype frequencies for fully unrecombined chromosomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) were evaluated in 1000kb, 100kb and 10kb windows. To further reduce error, we used overlapping inference windows, with a step size equal to 10% of the window size. Thus, the vast majority of sites fall within 10 separate overlapping inference windows. Finally, in order to balance local relevance with maximal information, we created likelihood models in sliding windows 10x the size of frequency estimation windows, with a step size equal to half the likelihood window size.</p><p id="p-13">For reference, inferring haplotype frequencies for 99 founder lines at 283k segregating sites on chromosome 2L in 1000kb windows took 8 min and required 450Mb RAM for samples sequenced at 5x empirical coverage and took 15 min and required 860Mb RAM for samples sequenced at 10x. Using 100kb windows took 9.5 min / 70Mb and 17.5 min / 132Mb for 5x and 10x samples, respectively.</p></div><div id="sec-6" class="subsection"><h3>HAF estimations</h3><p id="p-14">The haplotype-derived allele frequency (HAF) for a given biallelic site was calculated by summing founder haplotypes containing the alternate allele, each weighted by their average estimated haplotype frequency in all haplotype inference windows overlapping the site. Founder haplotypes with missing genotypes were given a fractional alternate allele count equal to the proportion of genotyped founders with alternate alleles.</p></div><div id="sec-7" class="subsection"><h3>Accuracy estimations using effective coverage</h3><p id="p-15">Effective coverage was used as a metric to assess the accuracy of all HAFs and raw AFs. For a given set of allele frequency estimates <em>p<sub>estimated</sub></em> at <em>n</em> sites, for which true frequency <em>p<sub>true</sub></em> is known, we first calculate the root mean squared error (<em>RMSE<sub>estimated</sub></em>) across all sites, where<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-3.gif"/></noscript></span></span>Next, we solve for the coverage <em>C<sub>effective</sub></em> at which <em>RMSE<sub>theoretical</sub></em> from binomial sampling would be equal to <em>RMSE<sub>estimated</sub></em>, where<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-4.gif"/></noscript></span></span>Solving for <em>C<sub>effective</sub></em> yields,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-5.gif"/></noscript></span></span>which is the theoretical coverage at which binomial sampling of reads would be expected to contain the observed amount of error from estimated frequencies.</p></div><div id="sec-8" class="subsection"><h3>Recombination</h3><p id="p-16">Forward-in-time simulations of recombination were performed with the software tool forqs (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kessner and Novembre 2014</a>) using a <em>D. melanogaster</em> recombination map (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comeron <em>et al.</em> 2012</a>). forqs simulates recombination of haplotype chunks for chromosomes of user-specified lengths for a randomly mating population, using a user-supplied recombination map, and non-overlapping generations. As a conservative metric, in our simulations we referred to the female <em>D. melanogaster</em> recombination rate. Since male <em>D. melanogaster</em> do not undergo recombination, our estimates of the number of recombination events per generation are higher than that expected in real populations and our estimates of effective coverage serve as a lower bound on effective coverage for the same number of generations in real populations. Three rounds of simulation were performed for each selection regime. In each round, an initial population of 1,000 individuals was created, with each individual assigned to a randomly selected homozygous founder strain. 5-10 sites were randomly chosen to be under selection and the genotypes of each individual (determined by the genotype of the corresponding founder strain) at these sites was supplied to forqs via an ms file. Homozygous reference, heterozygous, and homozygous alternate genotypes were assigned fitness advantages equal to 0, <em>s</em>, or 2<em>s</em> respectively, where s was a specified selection coefficient (either s = 0.025 or s = 0.1 in our simulations). The chosen loci each contributed independently to a single additive trait, with environmental variance equal to 0.01. At each generation, a fitness value was calculated by forqs for each individual based on their genotypes at the selected sites, with fitness decaying linearly with distance from the optimum trait value of 1. Individuals were selected to contribute to the next generation probabilistically based on their fitness value. Recombination breakpoints were simulated for evolutionary trajectories up to 50 generations in 5 replicate populations with a constant population size of 1,000 individuals. Within each round, each replicate contained the same selected sites and selection coefficient. At specific generations, 100 sets of recombination breakpoints (each representing a pair of evolved ‘chromosomes’) were randomly selected from the forqs output and were used to reconstruct ‘sampled chromosome genotypes’ from corresponding segments of the 99 founder genotype calls. This set of sampled genotypes was used to directly calculate ‘true’ allele frequencies for the sampled pool and was also used as input for read simulations with simreads. The resulting reads were then used for HAF calculation.</p></div><div id="sec-9" class="subsection"><h3>Generating a predictive model of effective coverage</h3><p id="p-17">We plotted the relationship between effective coverage (EC), and two key simulation parameters: the average number of sequencing reads overlapping SNP sites in each haplotype inference window (Rwin), and the percent of founder genotype calls that were missing (M) (Figure S7). While we observed non-linear relationships between effective coverage and both parameters, the log-log relationships were fairly linear. This suggested that a reasonable simple model would have the following format:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4159/embed/mml-math-6.gif"/></noscript></span></span>We used the R function ‘nls’ to solve for the coefficients <em>a</em>, <em>b</em>, <em>and c</em> in this formula, using all <em>D. melanogaster</em> simulations described in Results.</p></div><div id="sec-10" class="subsection"><h3>HAFS with an alternate founder set</h3><p id="p-18">For the DGRP founder set, SNP information was obtained for 205 strains initially isolated from Raleigh, NC that were independently sequenced as part of freeze 2 of the <em>Drosophila</em> Genetic Reference Panel (DGRP) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> 2014</a>). Genotype data were downloaded directly from <a href="http://dgrp2.gnets.ncsu.edu">http://dgrp2.gnets.ncsu.edu</a>. For the <em>C. elegans</em> founder set, a soft-filtered VCF file (<em>v. 20170531</em>) of genotype calls for 249 sequenced strains (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 2017</a>) was downloaded from the CeNDR website (<a href="https://www.elegansvariation.org/data/release/20170531">https://www.elegansvariation.org/data/release/20170531</a>), and was converted to a SNP table including genotypes for 100 randomly selected lines at all segregating biallelic SNP sites.</p><p id="p-19">After constructing the appropriate SNP table, read simulation, haplotype inference, and effective coverage calculations were carried out as described in the sections above.</p></div><div id="sec-11" class="subsection"><h3>Code accessibility</h3><p id="p-20">Scripts to calculate HAFs are available at <a href="https://github.com/petrov-lab/HAFpipe-line">https://github.com/petrov-lab/HAFpipe-line</a>. At minimum, the pipeline requires a) a vcf file or table of called genotypes at biallelic variants from sequenced founder lines, and b) mapped reads from one or more pool-seq samples. The pipeline uses harp for haplotype inference. Founder genotype data are additionally available at <a href="https://github.com/petrov-lab/HAFpipe-line/blob/master/99.clean.SNP.HARP.segregating.gz">https://github.com/petrov-lab/HAFpipe-line/blob/master/99.clean.SNP.HARP.segregating.gz</a>. Code used to generate the simulated data are provided at <a href="https://github.com/petrov-lab/HAFpipe-line/tree/master/simulations">https://github.com/petrov-lab/HAFpipe-line/tree/master/simulations</a>.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-21">File S1 contains a BAM file from pooled sequencing of the 99 inbred <em>D. melanogaster</em> lines. File S2 contains the founder genotypes from individual sequencing of the same set of 99 inbred lines (raw sequencing data available at SRA BioProject PRJNA383555). File S3 and File S4 contain SRA accessions and bam files of each of the 99 founder lines. Strains are available upon request. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9888008">https://doi.org/10.25387/g3.9888008</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Overview of HAF calculation method</h3><p id="p-22">Using haplotype inference for E+R experiments requires a genotyped founder set of inbred lines. Here, we begin by focusing on populations derived from a founder set of 99 sequenced <em>D. melanogaster</em> inbred lines (see Methods, ‘Establishment and sequencing of founder set’), and for simplicity limit our analysis to the 283,437 known segregating bi-allelic sites on chromosome 2L. In the analyses below, we estimate raw and haplotype-derived allele frequencies (referred to as raw AFs and HAFs, respectively) for real and simulated pools of ∼100 individuals sampled from populations derived from these founder haplotypes.</p><p id="p-23">Each sample is first subjected to pooled sequencing and all reads are mapped to the <em>D. melanogaster</em> reference genome. Raw AFs at each of the 283k sites are simply calculated as the fraction of mapped reads containing the alternate allele, after removing reads with neither the reference nor the alternate allele. To calculate HAFs, we performed haplotype frequency estimation with harp (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>), a haplotype inference tool that uses sequence identity and base quality scores from each sequenced read in a bam file, as well as a table of founder genotypes, to obtain maximum likelihood estimates of haplotype frequencies in discrete chromosomal windows. We determined that missing calls in the founder genotype table can bias haplotype frequency estimation, and therefore, we first impute all missing genotypes before running harp (Supplemental Text, Figure S1). The frequency of each founder haplotype is inferred within sliding genomic windows (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Methods) with extensive overlap to mitigate erroneous haplotype frequency assignments at the edges of inference windows. After inferring founder haplotype frequencies, we calculate HAFs at each SNP site by taking the weighted sum of local haplotype frequencies for founders containing the alternate allele (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, Methods).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of HAF calculation method. In an evolve-and-resequence experiment, the evolving population at any time point can be considered a mosaic of founder genomes. If founder genotypes are known, the relative frequency of each founder haplotype can be inferred within small genomic windows by leveraging pooled sequencing data and existing bioinformatic tools (i.e., harp (Kessner et al. 2013)). Allele frequencies can then be calculated from the weighted sum of founder haplotypes, rather than directly from mapped reads at each site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1867286919" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of HAF calculation method. In an evolve-and-resequence experiment, the evolving population at any time point can be considered a mosaic of founder genomes. If founder genotypes are known, the relative frequency of each founder haplotype can be inferred within small genomic windows by leveraging pooled sequencing data and existing bioinformatic tools (&lt;em&gt;i.e.&lt;/em&gt;, harp (Kessner &lt;em&gt;et al.&lt;/em&gt; 2013)). Allele frequencies can then be calculated from the weighted sum of founder haplotypes, rather than directly from mapped reads at each site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4159/F1.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4159/F1.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494944" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Overview of HAF calculation method. In an evolve-and-resequence experiment, the evolving population at any time point can be considered a mosaic of founder genomes. If founder genotypes are known, the relative frequency of each founder haplotype can be inferred within small genomic windows by leveraging pooled sequencing data and existing bioinformatic tools (<em>i.e.</em>, harp (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>)). Allele frequencies can then be calculated from the weighted sum of founder haplotypes, rather than directly from mapped reads at each site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To determine the accuracy of HAFs and raw AFs, estimated allele frequencies were compared to ‘true’ allele frequencies derived from the known composition of founder haplotypes in the sample. Chromosome-wide accuracy of HAFs and raw AFs was quantified using a new metric, effective coverage, which represents the theoretical coverage at which the root-mean-square error (RMSE) from binomial read sampling equals the RMSE from observed allele frequency estimates (Figure S2). While expected error from extreme allele frequencies is lower than that of intermediate allele frequencies under a binomial model, by taking the ratio of the expected error and the observed error for the same set of true allele frequencies, effective coverage values do not depend on the underlying allele frequency spectrum (Methods). Note that while this metric specifically focuses on variance from the sampling of reads from pooled chromosomes, in practice, the ability of both HAFs and raw AFs to accurately reflect true population-level allele frequencies will also depend on variance from the sampling of individuals from the population. This independent source of error has however been well-treated elsewhere (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feder <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kolaczkowski <em>et al.</em> 2011</a>) and will not be impacted by haplotype inference.</p><p id="p-26">In the following sections, we explore how the accuracy of HAFs differs from raw AFs, and how it scales with empirical coverage, number of generations of recombination, and missing founder genotypes.</p></div><div id="sec-15" class="subsection"><h3>Haplotype inference significantly increases the accuracy of allele frequency estimations</h3><p id="p-27">We begin by analyzing the very simplest scenario, in which a sample consists of un-recombined chromosomes, one from each founder line. To examine the accuracy of HAFs and AFs in this scenario, we created two biological replicate samples of 99 <em>D. melanogaster</em> individuals, each from a different homozygous inbred strain (Methods) and performed high-coverage pooled sequencing of each sample. True allele frequencies for each sample were then calculated, incorporating estimates of uneven pooling during sequencing (Figure S3, Supplemental Text).</p><p id="p-28">The accuracy of HAFs depends on the power to estimate haplotype frequencies, which in turn is affected by the coverage of mapped reads throughout the genomic window used for haplotype inference. In order to compare the accuracy of HAFs to raw AFs and test how each scales with empirical coverage, reads from the two biological replicates originally sequenced at ∼140x were down-sampled to chromosome-wide empirical coverages of 1x to 100x, and then used to calculate the effective coverage of estimated allele frequencies for each replicate (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Haplotype inference was initially performed using 100kb sliding windows, and accuracy was assessed only at the 27k sites with known genotype information for every founder. As expected, effective coverage of both HAFs and raw AFs is similar within the two biological replicates, and increases with higher chromosome-wide empirical coverage. Yet for all empirical coverages tested, HAFs have strikingly higher effective coverages than raw AFs. This substantial gain in accuracy from haplotype inference was most prominent at lower empirical coverages, with a &gt;40-fold increase in accuracy at 10x empirical coverage (from 10x to effectively &gt;400x). At higher empirical coverages, haplotype inference appears to produce diminishing returns and effective coverage begins to plateau.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy of HAFs and raw AFs for biological and simulated samples. A) Effective coverage of allele frequencies estimated with and without haplotype inference for the two biological replicates down-sampled to empirical coverages from 1-100x. (R1 = replicate 1, R2 = replicate 2; HAFs calculated with 100kb inference windows) B-D) Effective coverages of HAFs for biological replicates (blue) and simulated samples (red) using 10kb, 100kb, or 1000kb inference windows at empirical coverages of 1-10x. Orange line indicates a zoomed-in section of the same biological replicate values as shown in A. In all panels, a dashed line of 100x indicates the accuracy threshold required to detect strong selection (Kofler and Schl&#xF6;tterer 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1867286919" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy of HAFs and raw AFs for biological and simulated samples. A) Effective coverage of allele frequencies estimated with and without haplotype inference for the two biological replicates down-sampled to empirical coverages from 1-100x. (R1 = replicate 1, R2 = replicate 2; HAFs calculated with 100kb inference windows) B-D) Effective coverages of HAFs for biological replicates (blue) and simulated samples (red) using 10kb, 100kb, or 1000kb inference windows at empirical coverages of 1-10x. Orange line indicates a zoomed-in section of the same biological replicate values as shown in A. In all panels, a dashed line of 100x indicates the accuracy threshold required to detect strong selection (Kofler and Schl&#xF6;tterer 2014).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4159/F2.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4159/F2.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494965" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Accuracy of HAFs and raw AFs for biological and simulated samples. A) Effective coverage of allele frequencies estimated with and without haplotype inference for the two biological replicates down-sampled to empirical coverages from 1-100x. (R1 = replicate 1, R2 = replicate 2; HAFs calculated with 100kb inference windows) B-D) Effective coverages of HAFs for biological replicates (blue) and simulated samples (red) using 10kb, 100kb, or 1000kb inference windows at empirical coverages of 1-10x. Orange line indicates a zoomed-in section of the same biological replicate values as shown in A. In all panels, a dashed line of 100x indicates the accuracy threshold required to detect strong selection (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kofler and Schlötterer 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">We next tested the effect of using smaller (10kb) or larger (1000kb) windows for haplotype inference at empirical coverages up to 10x. We find that as expected, larger window sizes produce the most accurate HAFs, since more reads are available to infer haplotype frequencies in each window. Specifically, 1000kb HAFs derived from empirical coverages of 1x and 5x reached effective coverages of &gt;400x and &gt;900x, respectively.</p><p id="p-31">We also confirmed that similar results would be achieved by simulated samples with known sources of error. To do so, we simulated pooled synthetic reads with a standard Illumina sequencing error rate of 0.002 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schirmer <em>et al.</em> 2016</a>) and corresponding base quality scores (<a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Kessner <em>et al.</em> 2013</a>) from the same proportions of the 99 founder lines included in both biological replicates, and calculated effective coverage with the same empirical coverages and window sizes as above. Effective coverages for these simulated samples closely mirror effective coverages obtained from matched experimental samples (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A-B</a>). Slight differences at higher empirical coverages and larger window sizes are most likely caused by compounded experimental error from DNA extractions, PCR reactions and sequencing, as well as ambiguity in the ‘true’ genotypes estimated for individually sequenced lines. We explore the effects of founder genotype ambiguity further below (see “HAF accuracy is impacted by missing founder genotypes”).</p><p id="p-32">Together, these results suggest that HAFs derived from multiple biological samples sequenced at low empirical coverages can be orders of magnitude more accurate than raw AFs, and that simulated samples capture the magnitude of this effect quite well. In the following analyses we focus on simulated data from these 99 founder lines in order to precisely and reliably test how recombination and missing founder genotypes affect HAFs in realistic E+R scenarios.</p></div><div id="sec-16" class="subsection"><h3>Incorporation recombination and selection over short time scales</h3><p id="p-33">In the first section we showed that HAFs are accurate for samples of unrecombined chromosomes with very similar allele frequencies to the founder population. However, in a realistic E+R scenario sampled chromosomes will be recombined mosaics of the founders and selection may substantially shift allele frequencies over time. Thus, in the remainder of our analyses, we incorporate selection and recombination using a forward-in-time simulator, forqs (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kessner and Novembre 2014</a>). We simulated recombination for 50 generations in a population of 1,000 randomly-mating individuals using a <em>D. melanogaster</em> recombination map (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Comeron <em>et al.</em> 2012</a>), and tracked the breakpoints and haplotype of origin for all recombined segments at every generation. At specific generations, we randomly selected 200 recombined chromosomes (<em>i.e.</em>, 100 diploid individuals) from the simulated population and reconstructed the full sequences of these ‘sampled’ chromosomes from corresponding segments of the 99 individually sequenced founder haplotypes. Reads were simulated from the pooled set of 200 reconstructed chromosomes.</p><p id="p-34">We tested the accuracy of HAFs in three different selection regimes: five randomly chosen sites with selection strength s = 0.025 (<em>i.e.</em>, weak selection), 10 randomly chosen sites with selection strength s = 0.025, and 5 randomly chosen sites with selection strength s = 0.1 (<em>i.e.</em>, strong selection). In each case, the selected sites contributed additively to a single quantitative trait (Methods). We ran each simulation in three rounds, picking a different set of selected sites in each round, and simulated 5 replicate populations for each round. We calculated HAFs for each simulated sample, adjusting the window size for haplotype inference each generation based on the expected length of unrecombined haplotype blocks (Methods, Figure S4). As recombination proceeds, these windows become smaller, with fewer reads available to inform haplotype frequency estimation, and thus we expect that accuracy will decline.</p><p id="p-35">Our results show that while accuracy does decline over time, HAFs in general maintain &gt;100x effective coverage even after 50 generations in the presence of 5-10 selected sites per chromosome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We note that the three selection regimes tested all perform comparably well, though effective coverage is slightly higher with more selected sites (<em>i.e.</em>, 10 <em>vs.</em> 5) and larger selection coefficients (<em>i.e.</em>, s = 0.1 <em>vs.</em> s = 0.025). To assess the utility of HAFs for longer-term experiments, we also conducted three separate simulations with weak selection and 5 selected sites for 200 generations, and note that effective coverage of HAFs remains above the 100x threshold for detecting strong selection across 100-150 generations of recombination (Figure S5). Overall, we find that increasing empirical coverage has a diminishing returns effect on accuracy, while decreasing the generations of recombination has an approximately linear effect on accuracy.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effective coverage for recombination simulations with 99 inbred founder lines. In each simulation, recombination was simulated for a population of 1,000 individuals initiated from a panel of 99 fully sequenced inbred D. melanogaster lines, with a randomly chosen set of selected sites from among the 283,437 segregating sites on chromosome 2L. At multiple generations, 100 recombined individuals were sampled in silico from each population for simulated sequencing, HAF calculation, and error estimation. A) 5 or 10 sites were under weak or strong selection (panels), all sequencing was simulated at 5x, and HAFs were calculated with no missing genotypes. B) 5 sites were under weak selection, sequencing was simulated at multiple read depths (panels), and HAFs were calculated with various fractions of missing founder genotypes (color)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1867286919" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effective coverage for recombination simulations with 99 inbred founder lines. In each simulation, recombination was simulated for a population of 1,000 individuals initiated from a panel of 99 fully sequenced inbred &lt;em&gt;D. melanogaster&lt;/em&gt; lines, with a randomly chosen set of selected sites from among the 283,437 segregating sites on chromosome 2L. At multiple generations, 100 recombined individuals were sampled &lt;em&gt;in silico&lt;/em&gt; from each population for simulated sequencing, HAF calculation, and error estimation. A) 5 or 10 sites were under weak or strong selection (panels), all sequencing was simulated at 5x, and HAFs were calculated with no missing genotypes. B) 5 sites were under weak selection, sequencing was simulated at multiple read depths (panels), and HAFs were calculated with various fractions of missing founder genotypes (color).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4159/F3.medium.gif" width="440" height="125"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4159/F3.medium.gif" width="440" height="125"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494919" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Effective coverage for recombination simulations with 99 inbred founder lines. In each simulation, recombination was simulated for a population of 1,000 individuals initiated from a panel of 99 fully sequenced inbred <em>D. melanogaster</em> lines, with a randomly chosen set of selected sites from among the 283,437 segregating sites on chromosome 2L. At multiple generations, 100 recombined individuals were sampled <em>in silico</em> from each population for simulated sequencing, HAF calculation, and error estimation. A) 5 or 10 sites were under weak or strong selection (panels), all sequencing was simulated at 5x, and HAFs were calculated with no missing genotypes. B) 5 sites were under weak selection, sequencing was simulated at multiple read depths (panels), and HAFs were calculated with various fractions of missing founder genotypes (color).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>HAF accuracy is impacted by missing founder genotype</h3><p id="p-37">In addition to the effects of recombination and selection on HAF accuracy, ambiguity in the genotypes of founders (either due to missing genotype information or residual heterozygosity) can also present challenges. We tested how setting founder genotypes to be ambiguous (<em>i.e.</em>, from a called ‘A’ allele to an uncalled ‘N’) influences the accuracy of HAFs. For each simulation, 1–10% of all genotype calls in the founder table were randomly selected to be assigned as ‘N’. Genotypes at these sites were then imputed prior to estimating haplotype frequencies (Supplemental Text). For each percentage of missing calls tested, three rounds of simulation were performed.</p><p id="p-38">We find that missing genotype calls can significantly reduce effective coverage (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). However, the vast majority of the parameter space tested — 73.3% of all simulations — still yielded effective coverage values greater than 100x. Furthermore, 92.1% of simulations with less than 10% missing genotypes and fewer than 50 generations of recombination yielded effective coverage values greater than 100x.</p><p id="p-39">Importantly, these high overall chromosome-wide effective coverages extend to the selected sites themselves. Even with moderate levels of missing founder calls (up to 3% of missing sites), HAFs at selected sites still track closely with true allele frequencies; this is crucial for correctly detecting alleles under selection in E+R (Figure S6).</p></div><div id="sec-18" class="subsection"><h3>Estimating effective coverage with different founder sets and other model organisms</h3><p id="p-40">Finally, we explored how the utility of HAFs may extend to other founder sets with known SNPs and known recombination rates. Specifically, we tested whether a simple linear model based on the simulations above could accurately predict effective coverage for other experimental designs and other model organisms. The regimes tested in the simulations above can be collapsed into two independent parameters that affect HAF accuracy: 1) the number of reads used for haplotype inference in each window and 2) the percent of founder genotypes that are missing. The first parameter is a combination of SNP density, read depth, and window size – while the window size itself is a function of the recombination rate and the number of generations of recombination since population initiation.</p><p id="p-41">We calculated regression coefficients of a log-linear model (Figure S7) using all simulations described in the sections above, which focused on a single founder set across many experimental regimes. To test whether this model could accurately predict HAF accuracy for other founder sets, we simulated three rounds of evolutionary trajectories (with 5 weakly selected sites in each round) for two entirely different founder sets composed of 1) a widely-used reference panel of 205 <em>D. melanogaster</em> lines known as the DGRP (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> 2014</a>), and 2) 100 genotyped <em>C. elegans</em> strains from a reference panel known as CeNDR (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 2017</a>). For each founder set we simulated pooled samples for various generations and empirical coverages and calculated HAFs with various levels of missing founder information. We then used our model (trained only on the original simulations) to predict effective coverage for each of these new samples (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Observed effective coverage vs. effective coverage predicted by a simple 2-parameter log-linear model. A) A model built on samples from the simulated experimental evolution of 99 inbred Drosophila melanogaster lines described in the sections above was used to predict effective coverage in the simulated experimental evolution of different founder panels (205 DGRP lines) and different founder species (100 CeNDR lines). B) Effective coverage was well-predicted across a range of simulation parameters, including read depth, number of generations, percent missing founder genotypes, and selection regimes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1867286919" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Observed effective coverage &lt;em&gt;vs.&lt;/em&gt; effective coverage predicted by a simple 2-parameter log-linear model. A) A model built on samples from the simulated experimental evolution of 99 inbred &lt;em&gt;Drosophila melanogaster&lt;/em&gt; lines described in the sections above was used to predict effective coverage in the simulated experimental evolution of different founder panels (205 DGRP lines) and different founder species (100 CeNDR lines). B) Effective coverage was well-predicted across a range of simulation parameters, including read depth, number of generations, percent missing founder genotypes, and selection regimes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4159/F4.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4159/F4.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495054" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Observed effective coverage <em>vs.</em> effective coverage predicted by a simple 2-parameter log-linear model. A) A model built on samples from the simulated experimental evolution of 99 inbred <em>Drosophila melanogaster</em> lines described in the sections above was used to predict effective coverage in the simulated experimental evolution of different founder panels (205 DGRP lines) and different founder species (100 CeNDR lines). B) Effective coverage was well-predicted across a range of simulation parameters, including read depth, number of generations, percent missing founder genotypes, and selection regimes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We find that across all samples tested, our simple two-parameter model has an R<sup>2</sup> value of 0.875. Predicted effective coverage values for simulations with the original 99-lines founder set differ from true effective coverage values by 25% on average, with the largest source of error due to random effects between simulation rounds. Overall accuracy for DGRP and CeNDR samples was slightly lower, with average deviations of 37% in both founder sets. We confirmed that the model does not systematically over- or under- predict the effective coverage of DGRP samples or CeNDR samples nor any other set of parameters included in our simulations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). This suggests this model is broadly applicable for predicting HAF accuracy across many founder sets. Thus, given a set sequencing budget and a founder set with known sequencing quality, this model may be useful as a guideline for devising experimental schemes and distribution of resources that would maximize detection power. To this end, we have created a shiny app to help experimentalists predict effective coverage for their particular set of parameters (freely available at <a href="https://ec-calculator.shinyapps.io/shinyapp/">https://ec-calculator.shinyapps.io/shinyapp/</a>), as well as a table of requirements for HAF estimation (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of requirements for using HAFs to estimate allele frequencies in E+R experiments. Recommendations of timescale are based on simulations with D. melanogaster." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1867286919" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of requirements for using HAFs to estimate allele frequencies in E+R experiments. Recommendations of timescale are based on simulations with &lt;em&gt;D. melanogaster.&lt;/em&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4159/F5.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/12/4159/F5.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4159/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494940" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Schematic of requirements for using HAFs to estimate allele frequencies in E+R experiments. Recommendations of timescale are based on simulations with <em>D. melanogaster.</em></p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section other1" id="sec-19"><h2 class="">Disscussion</h2><p id="p-45">E+R experiments have become a powerful tool to assay the underpinnings of rapid adaptation by tracking allele frequency trajectories within populations over time. Previous studies have shown that the greatest power to detect adaptive variants comes from an optimized experimental design that tracks allele frequencies in multiple replicate populations, samples each replicate population at multiple timepoints, and maximizes the coverage of each pooled sample. Incorporating all of these factors into an E+R framework, however, can present significant financial challenges. Here, we offer a way to mitigate these high sequencing costs without sacrificing statistical power.</p><p id="p-46">Our framework uses haplotype inference to increase the accuracy of pooled allele frequency estimates at low coverages. Since the accuracy of haplotype-derived allele frequencies relies on the total informative value of reads across a genomic window, rather than coverage at a single site, this approach allows us to sequence less but still maintain high accuracy in allele frequency estimations. In this vein, the window-based techniques used by HAFs have an advantage over raw AFs in that the accuracy of an individual SNP with low read depth will benefit from reads at surrounding sites in the window. Overall, our method is capable of achieving the same accuracy as would be expected from sequencing each sample at 100x (as recommended in order to reliably detect strong selection), while only requiring empirical coverage of 1x or less, bringing total sequencing costs from &gt;$25,000 down to less than $200.</p><p id="p-47">There are, however, limitations to this approach. First, as presented, this framework requires the founder population to be derived entirely from fully-inbred lines. As a result, the population dynamics of loci under selection may differ slightly from trajectories in natural populations due to the genetic diversity lost in the inbreeding process (<em>i.e.</em>, natural haplotypes, homozygous lethal mutations, and rare variants), as well as higher levels of linkage disequilibrium compared to non-inbred lines. Reconstituting an outbred population using inbred lines, however, can be an effective way to mitigate the effects of the inbreeding process, and has been experimentally shown to have negligible bias and effect on adaptive dynamics (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nouhaud <em>et al.</em> 2016</a>). Additionally, in reality even with many generations of inbreeding, residual heterozygosity may still exist. In our HAF calculation method, we treat heterozygous sites as missing calls, and impute the genotype as reference or alternate based on called genotypes before inferring haplotype frequencies. While we test the impact of these missing calls in our simulations by placing them randomly across the genome, in reality heterozygous sites are often non-randomly dispersed which in turn, could lead to variation in effective coverage not accounted for here. Fortunately, both of these issues may be addressed in futures studies with more sophisticated founder sequencing approaches that incorporate phasing. Newer long-read technologies may already make this an achievable reality for a number of systems.</p><p id="p-48">Second, this approach requires a reliable and comprehensive account of the variants present in each founder line. Since previous studies recommend upwards of 100 founders, sequencing each individual founder line to a sufficiently high depth (in our work, we have found sequencing coverages &gt;10x to be sufficient) may present a high upfront cost. However, this cost represents a one-time investment, which can be applied toward all future experiments using the same set of founders. Furthermore, a number of consortiums already maintain publicly available stocks of large numbers of <em>Drosophila</em> lines and other model organisms with full, high-quality genome sequences (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lack <em>et al.</em> 2016</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Huang <em>et al.</em> 2014</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Cook <em>et al.</em> 2017</a>). We anticipate that these resources will continue to rapidly expand, facilitating experiments with even greater haplotype diversity at minimal costs.</p><p id="p-49">In addition, this approach is limited to studying short-term adaptation on the scale of tens of generations. In fact, an assumption of our method is that within an inference window, recombination breakpoints minimally affect the ability to accurately call haplotype frequencies. For a given window size however, this assumption becomes less valid as recombination proceeds and haplotypes blocks decay. Conversely however, decreasing the window size reduces the information used for haplotype inference, which at the extreme renders HAFs no more accurate than raw AFs. In our pipeline, we attempt to balance these effects by scaling window sizes at any generation by the expected average unrecombined fragment length. While our results here demonstrate that even with this scaling procedure, recombination will limit the ability to detect adaptation on timescales of more than tens of generations, the short-term adaptive dynamics that best fit E+R studies fall well within this range. Furthermore, it is at these short timescales, when large numbers of replicate populations are critical to reliably detect selection, that the cost savings associated with haplotype inference methods will be most beneficial.</p><p id="p-50">Finally, this approach relies on tracking the trajectories of known bi-allelic polymorphisms derived from the founder population, and thus, <em>de novo</em> mutations will not be assayed in this framework. Nonetheless our approach should sufficiently capture the salient features of short-term adaptive dynamics, as there is a growing body of experimental evidence suggesting that selection acts primarily on standing genetic variation in sexual organisms, and that <em>de novo</em> beneficial mutations do not play a large role in rapid adaptation (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Burke <em>et al.</em> 2014</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Teotónio <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sheng <em>et al.</em> 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Turchin <em>et al.</em> 2012</a>). Additionally, by tracking only known well-validated polymorphisms, the approach is largely robust to error from small non-SNP chromosomal variants such as indels.</p><p id="p-51">Despite the above limitations, collectively our results show that integrating haplotype inference into future E+R experiments is a cost-effective way to achieve accuracy in allele frequency estimates, which will directly improve the ability to detect genome-wide signatures of adaptation. Consequently, we offer specific recommendations for future E+R experimental schemes that take advantage of this approach. First, each founder line should be initially sequenced to a sufficient depth that minimizes any missing genotypes. If missing genotype calls do exist in founder lines, imputing sites prior to haplotype inference can mitigate some of this error.</p><p id="p-52">Together, these guidelines and the analysis above form a framework for achieving effective coverages of close to 100x with empirical coverages as low as 1x even after 50 generations of recombination in <em>D. melanogaster</em>, reducing sequencing costs by 100-fold. Ultimately, these cost savings, which can be extended to experiments with a variety of model organisms, will facilitate E+R frameworks that can incorporate large numbers of replicate populations. These improvements may be crucial to the statistical power to distinguish between beneficial and neutral alleles (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kessner and Novembre 2015</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baldwin-Brown <em>et al.</em> 2014</a>) and ultimately the future of E+R as a practical and reliable experimental tool.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We wish to thank the members of the Petrov lab and the reviewers for their insightful comments and suggestions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-54">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9888008">https://doi.org/10.25387/g3.9888008</a>.</p></li><li class="fn" id="fn-2"><p id="p-55"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-20"><span class="received-label">Received </span>September 20, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-15"><span class="accepted-label">Accepted </span>October 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Tilk <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.1" data-doi="10.1093/molbev/msu048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baldwin-Brown</span>,  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Thornton</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The power to detect quantitative trait loci using resequenced, experimentally evolved populations of diploid, sexual organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu048%26rft_id%253Dinfo%253Apmid%252F24441104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24441104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333283200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.2" data-doi="10.1534/g3.117.043349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barghi</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tobler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nolte</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Drosophila simulans: A Species with Improved Resolution in Evolve and Resequence Studies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2337</span>–<span class="cit-lpage">2343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.043349</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.043349%26rft_id%253Dinfo%253Apmid%252F28546383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8yMzM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.3" data-doi="10.1038/nature08480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Yoon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jeong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K.</span> <span class="cit-name-surname">Oh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome evolution and adaptation in a long-term experiment with Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08480</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBarrick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D461%26rft.issue%253D7268%26rft.spage%253D1243%26rft.epage%253D1247%26rft.atitle%253DGenome%2Bevolution%2Band%2Badaptation%2Bin%2Ba%2Blong-term%2Bexperiment%2Bwith%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08480%26rft_id%253Dinfo%253Apmid%252F19838166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19838166&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271190800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.4" data-doi="10.1371/journal.pone.0128036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baym</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kryazhimskiy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Lieberman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Desai</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inexpensive multiplexed library preparation for megabase-sized genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0128036</span>. Erratum: 10: e0131262.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0128036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0128036%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0128036%26rft_id%253Dinfo%253Apmid%252F26000737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0128036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000737&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.5" data-doi="10.1098/rspb.2017.2599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behrman</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Howick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kapun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Staubach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Bergland</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rapid seasonal evolution in innate immunity of wild Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">285</span>: <span class="cit-elocation-id">20172599</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2017.2599</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2017.2599%26rft_id%253Dinfo%253Apmid%252F29321302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2017.2599&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29321302&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.6" data-doi="10.1038/nature09352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Dunham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Shahrestani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. R.</span> <span class="cit-name-surname">Thornton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Rose</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of a long-term evolution experiment with Drosophila.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature09352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBurke%26rft.auinit1%253DM.%2BK.%26rft.volume%253D467%26rft.issue%253D7315%26rft.spage%253D587%26rft.epage%253D590%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Ba%2Blong-term%2Bevolution%2Bexperiment%2Bwith%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09352%26rft_id%253Dinfo%253Apmid%252F20844486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20844486&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282273100038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.7" data-doi="10.1098/rspb.2012.0799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">How does adaptation sweep through the genome? Insights from long-term selection experiments.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">5029</span>–<span class="cit-lpage">5038</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2012.0799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.0799%26rft_id%253Dinfo%253Apmid%252F22833271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.0799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22833271&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.8" data-doi="10.1093/molbev/msu256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Liti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Standing genetic variation drives repeatable experimental evolution in outcrossing populations of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3228</span>–<span class="cit-lpage">3239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu256%26rft_id%253Dinfo%253Apmid%252F25172959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25172959&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000345777600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.9" data-doi="10.1093/bioinformatics/btu670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">C.-C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Sun</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accurate estimation of haplotype frequency from pooled sequencing data and cost-effective identification of rare haplotype carriers by overlapping pool sequencing.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu670%26rft_id%253Dinfo%253Apmid%252F25304780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25304780&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.10" data-doi="10.1534/g3.114.012914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>,  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parallel genome-wide fixation of ancestral alleles in partially outcrossing experimental populations of Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1657</span>–<span class="cit-lpage">1665</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.012914</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012914%26rft_id%253Dinfo%253Apmid%252F24989146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvOS8xNjU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.11" data-doi="10.1371/journal.pgen.1002905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comeron</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ratnappan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bailin</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The many landscapes of recombination in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002905</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002905</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DComeron%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002905%26rft.epage%253De1002905%26rft.atitle%253DThe%2BMany%2BLandscapes%2Bof%2BRecombination%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002905%26rft_id%253Dinfo%253Apmid%252F23071443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.12" data-doi="10.1093/nar/gkw893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>,  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zdraljevic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Roberts</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Andersen</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CeNDR, the Caenorhabditis elegans natural diversity resource.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D650</span>–<span class="cit-lpage">D657</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw893%26rft_id%253Dinfo%253Apmid%252F27701074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27701074&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.13" data-doi="10.1371/journal.pone.0048588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>,  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Bergland</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">LDx: estimation of linkage disequilibrium from high-throughput pooled resequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e48588</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0048588</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFeder%26rft.auinit1%253DA.%2BF.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48588%26rft.epage%253De48588%26rft.atitle%253DLDx%253A%2BEstimation%2Bof%2BLinkage%2BDisequilibrium%2Bfrom%2BHigh-Throughput%2BPooled%2BResequencing%2BData.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048588%26rft_id%253Dinfo%253Apmid%252F23152785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23152785&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.14" data-doi="10.1371/journal.pone.0140462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fracassetti</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Griffin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Willi</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Validation of Pooled Whole-Genome Re-Sequencing in Arabidopsis lyrata.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0140462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0140462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0140462%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140462%26rft_id%253Dinfo%253Apmid%252F26461136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26461136&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.15" data-doi="10.1093/molbev/msw210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franssen</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Barton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Reconstruction of Haplotype-Blocks Selected during Experimental Evolution. Mol. Biol. Evol. 34: 174–184–184.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw210%26rft_id%253Dinfo%253Apmid%252F27702776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27702776&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>,   <span class="cit-name-suffix">Jr</span>.,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Hertweck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Phillips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Cabral</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomics of Parallel Experimental Evolution in Drosophila.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.17" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Massouras</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Inoue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Peiffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ràmia</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 Drosophila melanogaster Genetic Reference Panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.171546.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDE1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.18" data-doi="10.1093/molbev/msr289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Illingworth</span>  <span class="cit-name-given-names">C.</span>   <span class="cit-name-suffix">JR</span>.</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parts</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schiffels</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Liti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Mustonen</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Quantifying selection acting on a complex trait using allele frequency time series data. Mol. Biol. Evol. 29: 1187–1197.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr289</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr289%26rft_id%253Dinfo%253Apmid%252F22114362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22114362&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302018100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.19" data-doi="10.1186/1471-2105-14-270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jajamovich</span>,  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Iliadis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Anastassiou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Maximum-parsimony haplotype frequencies inference based on a joint constrained sparse representation of pooled DNA.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">270</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-270%26rft_id%253Dinfo%253Apmid%252F24010487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24010487&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.20" data-doi="10.1093/molbev/msv136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Miles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Lippert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">White</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Whole-Genome Resequencing of Experimental Populations Reveals Polygenic Basis of Egg-Size Variation in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2616</span>–<span class="cit-lpage">2632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv136%26rft_id%253Dinfo%253Apmid%252F26044351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26044351&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.21" data-doi="10.1093/molbev/mst016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Turner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Maximum Likelihood Estimation of Frequencies of Known Haplotypes from Pooled Sequence Data.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1145</span>–<span class="cit-lpage">1158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst016%26rft_id%253Dinfo%253Apmid%252F23364324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23364324&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318165700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.22" data-doi="10.1093/bioinformatics/btt712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">forqs: forward-in-time simulation of recombination, quantitative traits and selection.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt712</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt712%26rft_id%253Dinfo%253Apmid%252F24336146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24336146&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.23" data-doi="10.1534/genetics.115.175075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Power analysis of artificial selection experiments using efficient whole genome simulation of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">991</span>–<span class="cit-lpage">1005</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.175075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175075%26rft_id%253Dinfo%253Apmid%252F25672748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzQvOTkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.24" data-doi="10.1093/molbev/mst221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A guide for the design of evolve and resequencing studies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst221</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst221%26rft_id%253Dinfo%253Apmid%252F24214537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst221&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24214537&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330836900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.25" data-doi="10.1534/genetics.110.123059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolaczkowski</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Holloway</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Begun</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic differentiation between temperate and tropical Australian populations of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.123059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123059%26rft_id%253Dinfo%253Apmid%252F21059887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzEvMjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.26" data-doi="10.1093/molbev/msw195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lack</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Lange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Corbett-Detig</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Pool</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Thousand Fly Genomes: An Expanded Drosophila Genome Nexus.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3308</span>–<span class="cit-lpage">3313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw195%26rft_id%253Dinfo%253Apmid%252F27687565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27687565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.27" data-doi="10.1371/journal.pone.0015292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Jeffares</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nizhynska</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoolHap: inferring haplotype frequencies from pooled samples by next generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e15292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0015292</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLong%26rft.auinit1%253DQ.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15292%26rft.epage%253De15292%26rft.atitle%253DPoolHap%253A%2Binferring%2Bhaplotype%2Bfrequencies%2Bfrom%2Bpooled%2Bsamples%2Bby%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015292%26rft_id%253Dinfo%253Apmid%252F21264334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21264334&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.28" data-doi="10.1038/nrg3937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Liti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Luptak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Tenaillon</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Elucidating the molecular architecture of adaptation via evolve and resequence experiments. Nat. Rev. Genet. 16: 567–82–82.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3937</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D16%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3937%26rft_id%253Dinfo%253Apmid%252F26347030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26347030&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.29" data-doi="10.1093/gbe/evu085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bost</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wilson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Maruki</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Harrison</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population-genetic inference from pooled-sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1210</span>–<span class="cit-lpage">1218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evu085</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu085%26rft_id%253Dinfo%253Apmid%252F24787620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24787620&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.30" data-doi="10.1002/ece3.2402"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nouhaud</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tobler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nolte</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Ancestral population reconstitution from isofemale lines as a tool for experimental evolution. Ecol. Evol. 6: 7169–7175.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.2402</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.2402&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.31" data-doi="10.1111/j.1365-294X.2012.05673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kapun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nolte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kofler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Flatt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Adaptation of Drosophila to a novel laboratory environment reveals temporally heterogeneous trajectories of selected alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4931</span>–<span class="cit-lpage">4941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2012.05673.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOrozco-Terwengel%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D20%26rft.spage%253D4931%26rft.epage%253D4941%26rft.atitle%253DAdaptation%2Bof%2BDrosophila%2Bto%2Ba%2Bnovel%2Blaboratory%2Benvironment%2Breveals%2Btemporally%2Bheterogeneous%2Btrajectories%2Bof%2Bselected%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05673.x%26rft_id%253Dinfo%253Apmid%252F22726122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22726122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309888200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.32" data-doi="10.1093/bioinformatics/btp584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pirinen</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating population haplotype frequencies from pooled SNP data using incomplete database information.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3296</span>–<span class="cit-lpage">3302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp584%26rft_id%253Dinfo%253Apmid%252F19861356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19861356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272464000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.33" data-doi="10.1126/science.1192001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pitt</span>,  <span class="cit-name-given-names">J. N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Ferré-D’Amaré</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid construction of empirical RNA fitness landscapes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1192001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPitt%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D376%26rft.epage%253D379%26rft.atitle%253DRapid%2BConstruction%2Bof%2BEmpirical%2BRNA%2BFitness%2BLandscapes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192001%26rft_id%253Dinfo%253Apmid%252F20947767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.34" data-doi="10.1186/s12859-016-0976-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schirmer</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">D’Amore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U. Z.</span> <span class="cit-name-surname">Ijaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Hall</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Quince</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Illumina error profiles: resolving fine-scale variation in metagenomic sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">125</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-0976-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-0976-y%26rft_id%253Dinfo%253Apmid%252F26968756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-0976-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26968756&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.35" data-doi="10.1038/hdy.2014.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kofler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Versace</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tobler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. U.</span> <span class="cit-name-surname">Franssen</span></span></li></ol><cite>. <span class="cit-pub-date">2015</span> <span class="cit-article-title">“Combining Experimental Evolution with next-Generation Sequencing: A Powerful Tool to Study Adaptation from Standing Genetic Variation.”</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">40</span>. Erratum: 116: 248.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.86</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.86%26rft_id%253Dinfo%253Apmid%252F25269380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25269380&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.36" data-doi="10.1186/s13059-015-0785-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheng</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Pettersson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Honaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Siegel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Ö.</span> <span class="cit-name-surname">Carlborg</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Standing genetic variation as a major contributor to adaptation in the Virginia chicken lines selection experiment.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">219</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0785-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0785-z%26rft_id%253Dinfo%253Apmid%252F26438066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0785-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26438066&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.37" data-doi="10.1038/ng.289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teotónio</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Chelo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bradić</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Rose</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Experimental evolution reveals natural selection on standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.289</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTeotonio%26rft.auinit1%253DH.%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D251%26rft.epage%253D257%26rft.atitle%253DExperimental%2Bevolution%2Breveals%2Bnatural%2Bselection%2Bon%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.289%26rft_id%253Dinfo%253Apmid%252F19136954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19136954&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263091300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.38" data-doi="10.1038/ng.2368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turchin</span>,  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W. K.</span> <span class="cit-name-surname">Chiang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Palmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sankararaman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence of widespread selection on standing variation in Europe at height-associated SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2368</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTurchin%26rft.auinit1%253DM.%2BC.%26rft.volume%253D44%26rft.issue%253D9%26rft.spage%253D1015%26rft.epage%253D1019%26rft.atitle%253DEvidence%2Bof%2Bwidespread%2Bselection%2Bon%2Bstanding%2Bvariation%2Bin%2BEurope%2Bat%2Bheight-associated%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2368%26rft_id%253Dinfo%253Apmid%252F22902787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22902787&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.39" data-doi="10.1534/genetics.112.139337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">T. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Miller</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Investigating natural variation in Drosophila courtship song by the evolve and resequence approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">642</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.139337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139337%26rft_id%253Dinfo%253Apmid%252F22466043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzIvNjMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.40" data-doi="10.1126/science.285.5426.422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wichman</span>,  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Badgett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Scott</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Boulianne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Bull</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Different trajectories of parallel evolution during viral adaptation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">424</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.285.5426.422</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWichman%26rft.auinit1%253DH.%2BA.%26rft.volume%253D285%26rft.issue%253D5426%26rft.spage%253D422%26rft.epage%253D424%26rft.atitle%253DDifferent%2BTrajectories%2Bof%2BParallel%2BEvolution%2BDuring%2BViral%2BAdaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5426.422%26rft_id%253Dinfo%253Apmid%252F10411508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyNi80MjIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQxNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4159.41" data-doi="10.1371/journal.pone.0041901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Bergland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">González</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Empirical validation of pooled whole genome population re-sequencing in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e41901</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0041901</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhu%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41901%26rft.epage%253De41901%26rft.atitle%253DEmpirical%2Bvalidation%2Bof%2Bpooled%2Bwhole%2Bgenome%2Bpopulation%2Bre-sequencing%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041901%26rft_id%253Dinfo%253Apmid%252F22848651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22848651&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4159.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4159.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4149" title="Loss of rps9 in Zebrafish Leads to p53-Dependent Anemia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4169" title="Assessing by Modeling the Consequences of Increased Recombination in Recurrent Selection of Oryza sativa and Brassica rapa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4159" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bZpHGjKLxrOmqNqK1zTOxGok9O-huNl4byprYrEce4o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4159" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4159" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4159" class="active" data-icon-position="" data-hide-link-title="0">Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494900" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hs1i_K1HKWhsqxyMzBnDeCLsPlrEmqLi8jCgQq3m9G8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4159.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4159" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EjTmMccmbPY1-lNsrL4aLGrRLw1uOG4wdDi2MrDWqDk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4159" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4159.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494900" id="node494900--2" data-pisa="ggg;9/12/4159" data-pisa-master="ggg;g3.119.400755" data-apath="/ggg/9/12/4159.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susanne Tilk</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alan Bergland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron Goodman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Schmidt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri Petrov</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharon Greenblum</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4159-4168;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400755 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494900--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susanne Tilk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BTilk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusanne%2BTilk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susanne Tilk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan Bergland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Virginia, Charlottesville VA 22904, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BBergland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergland%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlan%2BBergland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alan Bergland</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Goodman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BGoodman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BGoodman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron Goodman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Pennsylvania, Philadelphia PA 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Schmidt</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dmitri Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri Petrov</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sharon Greenblum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BGreenblum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenblum%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BGreenblum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharon Greenblum</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4159" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4159&amp;text=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4159&amp;t=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4159&amp;title=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494900/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494900" id="node494900--3" data-pisa="ggg;9/12/4159" data-pisa-master="ggg;g3.119.400755" data-apath="/ggg/9/12/4159.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accurate Allele Frequencies from Ultra-low Coverage Pool-Seq Samples in Evolve-and-Resequence Experiments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Susanne Tilk</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alan Bergland</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aaron Goodman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Schmidt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dmitri Petrov</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharon Greenblum</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4159-4168;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400755 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494900--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Susanne Tilk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BTilk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilk%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusanne%2BTilk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9156-9360" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Susanne Tilk</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alan Bergland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Virginia, Charlottesville VA 22904, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BBergland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bergland%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlan%2BBergland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7145-7575" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alan Bergland</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Goodman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BGoodman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BGoodman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0768-7592" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aaron Goodman</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, University of Pennsylvania, Philadelphia PA 19104</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8076-6705" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Schmidt</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dmitri Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri Petrov</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sharon Greenblum </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Stanford University, Stanford CA 94305,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BGreenblum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenblum%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharon%2BGreenblum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6148-3016" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharon Greenblum</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:greensi@lbl.gov" class="" data-icon-position="" data-hide-link-title="0">greensi@lbl.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4159&amp;rft.epage=4168&amp;rft.atitle=Accurate%20Allele%20Frequencies%20from%20Ultra-low%20Coverage%20Pool-Seq%20Samples%20in%20Evolve-and-Resequence%20Experiments&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tilk%2C+Susanne&amp;rft.au=Bergland%2C+Alan&amp;rft.au=Goodman%2C+Aaron&amp;rft.au=Schmidt%2C+Paul&amp;rft.au=Petrov%2C+Dmitri&amp;rft.au=Greenblum%2C+Sharon"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494900/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494900/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494900/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494900/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494900/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494900/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494900/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494900/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494900/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494900/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494900/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494900/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4159" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Disscussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4159.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4159.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4159.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
