<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/4223" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/4223.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/4223.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/4223.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/12/4223/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/4223.atom" />
<meta name="HW.pisa" content="ggg;9/12/4223" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice" />
<meta name="DC.Identifier" content="10.1534/g3.119.400740" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Tyler et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic mapping is a primary tool of genetics in model organisms; however, many quantitative trait loci (QTL) contain tens or hundreds of positional candidate genes. Prioritizing these genes for validation is often ad hoc and biased by previous findings. Here we present a technique for prioritizing positional candidates based on computationally inferred gene function. Our method uses machine learning with functional genomic networks, whose links encode functional associations among genes, to identify network-based signatures of functional association to a trait of interest. We demonstrate the method by functionally ranking positional candidates in a large locus on mouse Chr 6 (45.9 Mb to 127.8 Mb) associated with histamine hypersensitivity (Histh). Histh is characterized by systemic vascular leakage and edema in response to histamine challenge, which can lead to multiple organ failure and death. Although Histh risk is strongly influenced by genetics, little is known about its underlying molecular or genetic causes, due to genetic and physiological complexity of the trait. To dissect this complexity, we ranked genes in the Histh locus by predicting functional association with multiple Histh-related processes. We integrated these predictions with new single nucleotide polymorphism (SNP) association data derived from a survey of 23 inbred mouse strains and congenic mapping data. The top-ranked genes included Cxcl12 , Ret , Cacna1c , and Cntn3 , all of which had strong functional associations and were proximal to SNPs segregating with Histh. These results demonstrate the power of network-based computational methods to nominate highly plausible quantitative trait genes even in challenging cases involving large QTL and extreme trait complexity." />
<meta name="DC.Contributor" content="Anna L. Tyler" />
<meta name="DC.Contributor" content="Abbas Raza" />
<meta name="DC.Contributor" content="Dimitry N. Krementsov" />
<meta name="DC.Contributor" content="Laure K. Case" />
<meta name="DC.Contributor" content="Rui Huang" />
<meta name="DC.Contributor" content="Runlin Z. Ma" />
<meta name="DC.Contributor" content="Elizabeth P. Blankenhorn" />
<meta name="DC.Contributor" content="Cory Teuscher" />
<meta name="DC.Contributor" content="J. Matthew Mahoney" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic mapping is a primary tool of genetics in model organisms; however, many quantitative trait loci (QTL) contain tens or hundreds of positional candidate genes. Prioritizing these genes for validation is often &lt;i&gt;ad hoc&lt;/i&gt; and biased by previous findings. Here we present a technique for prioritizing positional candidates based on computationally inferred gene function. Our method uses machine learning with functional genomic networks, whose links encode functional associations among genes, to identify network-based signatures of functional association to a trait of interest. We demonstrate the method by functionally ranking positional candidates in a large locus on mouse Chr 6 (45.9 Mb to 127.8 Mb) associated with histamine hypersensitivity (Histh). Histh is characterized by systemic vascular leakage and edema in response to histamine challenge, which can lead to multiple organ failure and death. Although Histh risk is strongly influenced by genetics, little is known about its underlying molecular or genetic causes, due to genetic and physiological complexity of the trait. To dissect this complexity, we ranked genes in the &lt;i&gt;Histh&lt;/i&gt; locus by predicting functional association with multiple Histh-related processes. We integrated these predictions with new single nucleotide polymorphism (SNP) association data derived from a survey of 23 inbred mouse strains and congenic mapping data. The top-ranked genes included &lt;i&gt;Cxcl12&lt;/i&gt;, &lt;i&gt;Ret&lt;/i&gt;, &lt;i&gt;Cacna1c&lt;/i&gt;, and &lt;i&gt;Cntn3&lt;/i&gt;, all of which had strong functional associations and were proximal to SNPs segregating with Histh. These results demonstrate the power of network-based computational methods to nominate highly plausible quantitative trait genes even in challenging cases involving large QTL and extreme trait complexity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/4223" />
<meta name="citation_id" content="9/12/4223" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/4223" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/4223.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/4223.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/4223.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400740" />
<meta name="citation_pmid" content="31645420" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="4223" />
<meta name="citation_lastpage" content="4233" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anna L. Tyler" />
<meta name="citation_author_institution" content="The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8371-2377" />
<meta name="citation_author" content="Abbas Raza" />
<meta name="citation_author_institution" content="Department of Medicine," />
<meta name="citation_author" content="Dimitry N. Krementsov" />
<meta name="citation_author_institution" content="Department of Biomedical and Health Sciences," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2709-7466" />
<meta name="citation_author" content="Laure K. Case" />
<meta name="citation_author_institution" content="The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609," />
<meta name="citation_author" content="Rui Huang" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and" />
<meta name="citation_author" content="Runlin Z. Ma" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and" />
<meta name="citation_author" content="Elizabeth P. Blankenhorn" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Drexel University College of Medicine, Philadelphia, PA," />
<meta name="citation_author" content="Cory Teuscher" />
<meta name="citation_author_institution" content="Department of Medicine," />
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Vermont Larner College of Medicine, Burlington, VT, 05405," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9236-8843" />
<meta name="citation_author" content="J. Matthew Mahoney" />
<meta name="citation_author_institution" content="Department of Neurological Sciences," />
<meta name="citation_author_institution" content="Department of Computer Science," />
<meta name="citation_author_email" content="John.M.Mahoney@uvm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1425-5939" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. J. Baker;citation_author=J. J. Jay;citation_author=J. A. Bubier;citation_author=M. A. Langston;citation_author=E. J. Chesler;citation_title=GeneWeaver: a web-based system for integrative functional genomics.;citation_volume=40;citation_year=2012;citation_pmid=22080549;citation_doi=10.1093/nar/gkr968" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Barrett;citation_author=S. E. Wilhite;citation_author=P. Ledoux;citation_author=C. Evangelista;citation_author=I. F. Kim;citation_title=NCBI GEO: archive for functional genomics data sets—update.;citation_volume=41;citation_year=2012;citation_pmid=23193258;citation_doi=10.1093/nar/gks1193" />
<meta name="citation_reference" content="citation_journal_title=Mol. Vis.;citation_author=M. D. Benson;citation_author=C. C. Khor;citation_author=P. J. Gage;citation_author=O. J. Lehmann;citation_title=A targeted approach to genome-wide studies reveals new genetic associations with central corneal thickness.;citation_pages=952;citation_volume=23;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. A. Bogue;citation_author=S. C. Grubb;citation_author=D. O. Walton;citation_author=V. M. Philip;citation_author=G. Kolishovski;citation_title=Mouse Phenome Database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice.;citation_pages=D843-D850;citation_volume=46;citation_year=2017;citation_doi=10.1093/nar/gkx1082" />
<meta name="citation_reference" content="citation_journal_title=Tissue antigens;citation_journal_abbrev=Tissue Antigens;citation_author=M. Bunce;citation_author=C. O’neill;citation_author=M. Barnardo;citation_author=P. Krausa;citation_author=M. Browning;citation_title=Phototyping: comprehensive DNA typing for HLA-A, B, C, DRB1, DRB3, DRB4, DRB5 &amp;amp; DQB1 by PCR with 144 primer mixes utilizing sequence-specific primers (PCR-SSP).;citation_pages=355-367;citation_volume=46;citation_year=1995;citation_issue=5;citation_pmid=8838344;citation_doi=10.1111/j.1399-0039.1995.tb03127.x" />
<meta name="citation_reference" content="citation_journal_title=J. Immunol.;citation_author=S. Carpenter;citation_author=T. Carlson;citation_author=J. Dellacasagrande;citation_author=A. Garcia;citation_author=S. Gibbons;citation_title=TRIL, a functional component of the TLR4 signaling complex, highly expressed in brain.;citation_volume=183;citation_year=2009;citation_pmid=19710467;citation_doi=10.4049/jimmunol.0901518" />
<meta name="citation_reference" content="citation_journal_title=Complex Syst.;citation_author=G. Csardi;citation_author=T. Nepusz;citation_title=The igraph software package for complex network research. InterJournal;citation_pages=1-9;citation_volume=1695;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=J. R. Davis;citation_author=L. Fresard;citation_author=D. A. Knowles;citation_author=M. Pala;citation_author=C. D. Bustamante;citation_title=An efficient multiple-testing adjustment for eQTL studies that accounts for linkage disequilibrium between variants.;citation_pages=216-224;citation_volume=98;citation_year=2016;citation_pmid=26749306;citation_doi=10.1016/j.ajhg.2015.11.021" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. Durinck;citation_author=Y. Moreau;citation_author=A. Kasprzyk;citation_author=S. Davis;citation_author=B. De Moor;citation_title=BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis;citation_pages=3439-3440;citation_volume=21;citation_year=2005;citation_issue=16;citation_pmid=16082012;citation_doi=10.1093/bioinformatics/bti525" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=S. Durinck;citation_author=P. T. Spellman;citation_author=E. Birney;citation_author=W. Huber;citation_title=Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt.;citation_pages=1184-1191;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19617889;citation_doi=10.1038/nprot.2009.97" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Edgar;citation_author=M. Domrachev;citation_author=A. E. Lash;citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository;citation_pages=207-210;citation_volume=30;citation_year=2002;citation_issue=1;citation_pmid=11752295;citation_doi=10.1093/nar/30.1.207" />
<meta name="citation_reference" content="citation_journal_title=Journal of vascular surgery :  official publication, the Society for Vascular Surgery [and] International Society for Cardiovascular Surgery, North American Chapter;citation_journal_abbrev=J Vasc Surg;citation_author=J. R. Elmore;citation_author=M. A. Obmann;citation_author=H. Kuivaniemi;citation_author=G. Tromp;citation_author=G. S. Gerhard;citation_title=Identification of a genetic variant associated with abdominal aortic aneurysms on chromosome 3p12.3 by genome wide association.;citation_pages=1525-1531;citation_volume=49;citation_year=2009;citation_issue=6;citation_pmid=19497516;citation_doi=10.1016/j.jvs.2009.01.041" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=L. Fagerberg;citation_author=B. M. Hallström;citation_author=P. Oksvold;citation_author=C. Kampf;citation_author=D. Djureinovic;citation_title=Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics.;citation_volume=13;citation_year=2014;citation_pmid=24309898;citation_doi=10.1074/mcp.M113.035600" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=The Gene Ontology resource: 20 years and still GOing strong.;citation_pages=D330-D338;citation_volume=47;citation_year=2018;citation_doi=10.1093/nar/gky1055" />
<meta name="citation_reference" content="citation_journal_title=International archives of allergy and immunology;citation_journal_abbrev=Int Arch Allergy Immunol;citation_author=M. Ghannadan;citation_author=A. W. Hauswirth;citation_author=G.-H. Schernthaner;citation_author=M. R. Müller;citation_author=W. Klepetko;citation_title=Detection of novel CD antigens on the surface of human mast cells and basophils.;citation_pages=299-307;citation_volume=127;citation_year=2002;citation_issue=4;citation_pmid=12021549;citation_doi=10.1159/000057747" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Goya;citation_author=A. K. Wong;citation_author=V. Yao;citation_author=A. Krishnan;citation_author=M. Homilius;citation_title=FNTM: a server for predicting functional networks of tissues in mouse.;citation_volume=43;citation_year=2015;citation_pmid=25940632;citation_doi=10.1093/nar/gkv443" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. S. Greene;citation_author=A. Krishnan;citation_author=A. K. Wong;citation_author=E. Ricciotti;citation_author=R. A. Zelaya;citation_title=Understanding multicellular function and disease with human tissue-specific networks.;citation_pages=569-576;citation_volume=47;citation_year=2015;citation_pmid=25915600;citation_doi=10.1038/ng.3259" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=Y. Guan;citation_author=C. L. Ackert-Bicknell;citation_author=B. Kell;citation_author=O. G. Troyanskaya;citation_author=M. A. Hibbs;citation_title=Functional genomics complements quantitative genetics in identifying disease-gene associations.;citation_pages=e1000991-e1000991;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=21085640;citation_doi=10.1371/journal.pcbi.1000991" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=P. L. Hedley;citation_author=P. Jørgensen;citation_author=S. Schlamowitz;citation_author=J. Moolman-Smook;citation_author=J. K. Kanters;citation_title=The genetic basis of Brugada syndrome: a mutation update.;citation_pages=1256-1266;citation_volume=30;citation_year=2009;citation_issue=9;citation_pmid=19606473;citation_doi=10.1002/humu.21066" />
<meta name="citation_reference" content="citation_journal_title=Pharmacological Reviews;citation_journal_abbrev=Pharmacol. Rev.;citation_author=S. Hill;citation_author=C. Ganellin;citation_author=H. Timmerman;citation_author=J. Schwartz;citation_author=N. Shankley;citation_title=International Union of Pharmacology. XIII. Classification of Histamine Receptors;citation_pages=253-278;citation_volume=49;citation_year=1997;citation_issue=3;citation_pmid=9311023" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=J. O. Hiltunen;citation_author=A. Laurikainen;citation_author=M. S. Airaksinen;citation_author=M. Saarma;citation_title=GDNF family receptors in the embryonic and postnatal rat heart and reduced cholinergic innervation in mice hearts lacking ret or GFRalpha2.;citation_pages=28-39;citation_volume=219;citation_year=2000;citation_issue=1;citation_pmid=10974669;citation_doi=10.1002/1097-0177(2000)9999:9999&lt;::AID-DVDY1031&gt;3.0.CO;2-P" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=P. D. Hsu;citation_author=E. S. Lander;citation_author=F. Zhang;citation_title=Development and applications of CRISPR-Cas9 for genome engineering.;citation_pages=1262-1278;citation_volume=157;citation_year=2014;citation_pmid=24906146;citation_doi=10.1016/j.cell.2014.05.010" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Ju;citation_author=C. S. Greene;citation_author=F. Eichinger;citation_author=V. Nair;citation_author=J. B. Hodgin;citation_title=Defining cell-type specificity at the transcriptional level in human disease.;citation_volume=23;citation_year=2013;citation_pmid=23950145;citation_doi=10.1101/gr.155697.113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.;citation_author=A. Krishnan;citation_author=R. Zhang;citation_author=V. Yao;citation_author=C. L. Theesfeld;citation_author=A. K. Wong;citation_title=Genome-wide prediction and functional characterization of the genetic basis of autism spectrum disorder.;citation_pages=1454-1462;citation_volume=19;citation_year=2016;citation_doi=10.1038/nn.4353" />
<meta name="citation_reference" content="citation_journal_title=Curr. Allergy Asthma Rep.;citation_author=P. Lieberman;citation_author=L. H. Garvey;citation_title=Mast cells and anaphylaxis.;citation_pages=20;citation_volume=16;citation_year=2016;citation_pmid=26857018;citation_doi=10.1007/s11882-016-0598-5" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Z. Liu;citation_author=A. F. McRae;citation_author=D. R. Nyholt;citation_author=S. E. Medland;citation_author=N. R. Wray;citation_title=A versatile gene-based test for genome-wide association studies.;citation_pages=139-145;citation_volume=87;citation_year=2010;citation_issue=1;citation_pmid=20598278;citation_doi=10.1016/j.ajhg.2010.06.009" />
<meta name="citation_reference" content="Meyer, D., E. Dimitriadou, K. Hornik, A. Weingessel, and F. Leisch, 2018 e1071: Misc Functions of the Department of Statistics, Probability Theory Group (Formerly: E1071), TU Wien. R package version 1.7–0." />
<meta name="citation_reference" content="citation_title=GeneReviews;citation_title=Timothy syndrome;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Physical review. E, Statistical, nonlinear, and soft matter physics;citation_journal_abbrev=Phys Rev E Stat Nonlin Soft Matter Phys;citation_author=M. E. J. Newman;citation_title=Fast algorithm for detecting community structure in networks.;citation_pages=066133-066133;citation_volume=69;citation_year=2004;citation_issue=6 Pt 2;citation_pmid=15244693;citation_doi=10.1103/PhysRevE.69.066133" />
<meta name="citation_reference" content="citation_journal_title=Genes Brain Behav.;citation_author=C. C. Parker;citation_author=G. Sokoloff;citation_author=E. Leung;citation_author=S. L. Kirkpatrick;citation_author=A. A. Palmer;citation_title=A large QTL for fear and anxiety mapped using an F2 cross can be dissected into multiple smaller QTLs.;citation_pages=714-722;citation_volume=12;citation_year=2013;citation_pmid=23876074" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. M. Petkov;citation_author=Y. Ding;citation_author=M. A. Cassell;citation_author=W. Zhang;citation_author=G. Wagner;citation_title=An Efficient SNP System for Mouse Genome Scanning and Elucidating Strain Relationships;citation_pages=1806-1811;citation_volume=14;citation_year=2004;citation_issue=9;citation_pmid=15342563;citation_doi=10.1101/gr.2825804" />
<meta name="citation_reference" content="Raza, A., E. C. Chan, W.-S. Chen, Z. Xie, L. M. Scott, et al., 2019 A natural mouse model reveals genetic determinants of Systemic Capillary Leak Syndrome (Clarkson disease). Communications Biology." />
<meta name="citation_reference" content="Reimand, J., R. Kolde, and T. Arak, 2018 gProfileR: Interface to the ’g:Profiler’ Toolkit. R package version 0.6.7." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Rusmini;citation_author=P. Griseri;citation_author=F. Lantieri;citation_author=I. Matera;citation_author=K. L. Hudspeth;citation_title=Induction of RET dependent and independent pro-inflammatory programs in human peripheral blood mononuclear cells from Hirschsprung patients.;citation_pages=e59066;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0059066" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Heart Assoc.;citation_author=J. C. Schisler;citation_author=T. J. Grevengoed;citation_author=F. Pascual;citation_author=D. E. Cooper;citation_author=J. M. Ellis;citation_title=Cardiac energy dependence on glucose increases metabolites related to glutathione and activates metabolic genes controlled by mechanistic target of rapamycin.;citation_volume=4;citation_year=2015;citation_pmid=25713290;citation_doi=10.1161/JAHA.114.001136" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. L. Smith;citation_author=J. T. Eppig;citation_title=The Mammalian Phenotype Ontology as a unifying standard for experimental and high-throughput phenotyping data.;citation_pages=653-668;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22961259;citation_doi=10.1007/s00335-012-9421-3" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Sudweeks;citation_author=J. A. Todd;citation_author=E. P. Blankenhorn;citation_author=B. B. Wardell;citation_author=S. R. Woodward;citation_title=Locus controlling Bordetella pertussis-induced histamine sensitization (Bphs), an autoimmune disease-susceptibility gene, maps distal to T-cell receptor beta-chain gene on mouse chromosome 6.;citation_pages=3700-3704;citation_volume=90;citation_year=1993;citation_issue=8;citation_pmid=8475118;citation_doi=10.1073/pnas.90.8.3700" />
<meta name="citation_reference" content="citation_journal_title=International archives of allergy and applied immunology;citation_journal_abbrev=Int Arch Allergy Appl Immunol;citation_author=N. M. Vaz;citation_author=C. M. de Souza;citation_author=M. M. Hornbrook;citation_author=D. G. Hanson;citation_author=N. R. Lynch;citation_title=Sensitivity to intravenous injections of histamine and serotonin in inbred mouse strains.;citation_pages=545-554;citation_volume=53;citation_year=1977;citation_issue=6;citation_pmid=863521;citation_doi=10.1159/000231796" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Wang;citation_author=J. Huang;citation_author=Y. Liu;citation_author=L. Ma;citation_author=J. B. Potash;citation_title=COMBAT: a combined association test for genes using summary statistics.;citation_volume=207;citation_year=2017;citation_pmid=28878002;citation_doi=10.1534/genetics.117.300257" />
<meta name="citation_reference" content="citation_journal_title=Life Sci.;citation_author=M. Wang;citation_author=T. Shibamoto;citation_author=M. Tanida;citation_author=Y. Kuda;citation_author=Y. Kurata;citation_title=Mouse anaphylactic shock is caused by reduced cardiac output, but not by systemic vasodilatation or pulmonary vasoconstriction, via PAF and histamine.;citation_pages=98-105;citation_volume=116;citation_year=2014;citation_doi=10.1016/j.lfs.2014.09.010" />
<meta name="citation_reference" content="citation_journal_title=J. Immunol.;citation_author=P. Wochal;citation_author=V. A. Rathinam;citation_author=A. Dunne;citation_author=T. Carlson;citation_author=W. Kuang;citation_title=TRIL is involved in cytokine production in the brain following escherichia coli infection.;citation_volume=193;citation_year=2014;citation_pmid=25015823;citation_doi=10.4049/jimmunol.1302392" />
<meta name="citation_reference" content="citation_journal_title=Rare Dis.;citation_author=Z. Xie;citation_author=V. Nagarajan;citation_author=D. E. Sturdevant;citation_author=S. Iwaki;citation_author=E. Chan;citation_title=Genome-wide SNP analysis of the systemic capillary leak syndrome (Clarkson disease).;citation_pages=27445e;citation_volume=1;citation_year=2013;citation_doi=10.4161/rdis.27445" />
<meta name="citation_reference" content="citation_journal_title=Med. Sci.;citation_author=J. Yan;citation_author=Q. Li;citation_author=Y. Luo;citation_author=S. Yan;citation_author=Y. He;citation_title=Association of CACNA1C gene genetic polymorphism with the susceptibility as well as prognosis for chronic spontaneous urticaria. Zhong nan da xue xue bao. Yi xue ban= Journal of Central South University.;citation_pages=929-936;citation_volume=43;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. N. Yazbek;citation_author=D. A. Buchner;citation_author=J. M. Geisinger;citation_author=L. C. Burrage;citation_author=S. H. Spiezio;citation_title=Deep congenic analysis identifies many strong, context-dependent QTLs, one of which, Slc35b4, regulates obesity and glucose homeostasis.;citation_volume=21;citation_year=2011;citation_pmid=21507882;citation_doi=10.1101/gr.120741.111" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Genetic mapping is a primary tool of genetics in model organisms; however, many quantitative trait loci (QTL) contain tens or hundreds of positional candidate genes. Prioritizing these genes for validation is often ad hoc and biased by previous findings. Here we present a technique for prioritizing positional candidates based on computationally inferred gene function. Our method uses machine learning with functional genomic networks, whose links encode functional associations among genes, to identify network-based signatures of functional association to a trait of interest. We demonstrate the method by functionally ranking positional candidates in a large locus on mouse Chr 6 (45.9 Mb to 127.8 Mb) associated with histamine hypersensitivity (Histh). Histh is characterized by systemic vascular leakage and edema in response to histamine challenge, which can lead to multiple organ failure and death. Although Histh risk is strongly influenced by genetics, little is known about its underlying molecular or genetic causes, due to genetic and physiological complexity of the trait. To dissect this complexity, we ranked genes in the Histh locus by predicting functional association with multiple Histh-related processes. We integrated these predictions with new single nucleotide polymorphism (SNP) association data derived from a survey of 23 inbred mouse strains and congenic mapping data. The top-ranked genes included Cxcl12 , Ret , Cacna1c , and Cntn3 , all of which had strong functional associations and were proximal to SNPs segregating with Histh. These results demonstrate the power of network-based computational methods to nominate highly plausible quantitative trait genes even in challenging cases involving large QTL and extreme trait complexity." />
<meta name="og-title" property="og:title" content="Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/4223" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic mapping is a primary tool of genetics in model organisms; however, many quantitative trait loci (QTL) contain tens or hundreds of positional candidate genes. Prioritizing these genes for validation is often ad hoc and biased by previous findings. Here we present a technique for prioritizing positional candidates based on computationally inferred gene function. Our method uses machine learning with functional genomic networks, whose links encode functional associations among genes, to identify network-based signatures of functional association to a trait of interest. We demonstrate the method by functionally ranking positional candidates in a large locus on mouse Chr 6 (45.9 Mb to 127.8 Mb) associated with histamine hypersensitivity (Histh). Histh is characterized by systemic vascular leakage and edema in response to histamine challenge, which can lead to multiple organ failure and death. Although Histh risk is strongly influenced by genetics, little is known about its underlying molecular or genetic causes, due to genetic and physiological complexity of the trait. To dissect this complexity, we ranked genes in the Histh locus by predicting functional association with multiple Histh-related processes. We integrated these predictions with new single nucleotide polymorphism (SNP) association data derived from a survey of 23 inbred mouse strains and congenic mapping data. The top-ranked genes included Cxcl12 , Ret , Cacna1c , and Cntn3 , all of which had strong functional associations and were proximal to SNPs segregating with Histh. These results demonstrate the power of network-based computational methods to nominate highly plausible quantitative trait genes even in challenging cases involving large QTL and extreme trait complexity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-oP0gc3yhiA7EntWtue5DpEgmHD7FW8TF4IK26wc1gI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494987","apath":"\/ggg\/9\/12\/4223.atom","pisa":"ggg;9\/12\/4223","ac":{"\/ggg\/9\/12\/4223.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/4223.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4223"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/4223"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/4223":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Te8e9P0W1VkkKI3o00HkVGzLLKidkIRGksE52uoACs4Mzh0DFxcaL\\\\\\\/xVVl\\\\\\\/KVeYu4EdAKsWTxwpIG1H8QyVTl\\\\\\\/yTsrj7ogfoJIQGZ3KUn0munRjwuJDpiLyNykMEqIp3WBiRUFvwBwCsDU0qgNVqRRpHmXgLITjqmeFGw7bUw+VgXRMLxljuoa9QBlyyyYOKwgqh8C28lCXRGVaFi7\\\\\\\/PGr0P5XnlDgt1Di+B59DtN2Ghj3s8RtIJQo6+XIyeu4xgdjRdwhfRXZ0JkziRNvXtxjc4N0COtUfewbHGlRAVAEEVq3vQpWCy1CS3ABSigX8\\\\\\\/CKjybMoWLbmJ+dUJzt2j17skQf\\\\\\\/C4Y44mW78t6xuIw8tgjRS8GIVZffyUsFrOF7AKLVMMUdIKx7SGw9uzCU0oCrqknAMbzb4fpFdPMWhFRYdMIYfAIsJNx\\\\\\\/1emC0YG2mOoXprebWw1Vq2X8jjPvIYWzHk07OESr3mdOqmjRPgFSniir1\\\\\\\/m2sr5FMS2vsbtUpmpQK86IX6LPzeKLVxh\\\\\\\/hohix\\\\\\\/QNbfYZEpNWj9Z\\\\\\\/eSdJO\\\\\\\/okVpaNlyowTZcw1jM+V45t8PR5ABM+H5l5NI0y9O4\\\\\\\/htp+\\\\\\\/Yk+WWtXoihd1fPgN1CUE9ayODhDX7Agd6Z+16RMp98Eun6kamyHysdRMh9qzujgz5ebMbND1HB8WoE\\\\\\\/uUXMPEOAlviP3Ufk5P44AVN40bHInvbUAWbdVl9J7QhLxPJxkj8g+QCzY1\\\\\\\/YFEV\\\\\\\/ckA3H916Zj0KPBC05W5Ur\\\\\\\/xUpewPx2H\\\\\\\/hCxu2qwDbfDK0CtNhOuGwBIi3BW6oTPmsm2YC4RJzkJExAH1L7j\\\\\\\/hWhMhzb5Za4DRiJGbWyRgxslqGFBrjqzZkF4TBZ2g7jhV\\\\\\\/1eirtpuBZo5j+1v+teyrUcyPQuIzX9yhkFSuJhfTYU+7UOc54w565MZmI+QH9Z5EsZeSqKWq2LpMYunIe8t0TAnuOaaEYvBOirf5KSzHB2UNje2UFP4Zo0NZ0jfOs5VjJSzHECIblcyA7Rl2YxhVsm+ol8rZtpd5+A0SvwTJ5rkL7Cgi8zm3Ye3U8TgyIoKLbBTva8ZPCOeUmr+\\\\\\\/dbyv+XG0KByy8PX06b\\\\\\\/vZO0J3mx5UK3vCB137I7tMNK9E9X37DMAkJG71WMLYp\\\\\\\/\\\\\\\/Cbw8R5Me9HbO9x4QQH03IQCRLPC2dOHLIb6gUSBDnNqoYp7FIA9mfH1X24wCzkFTU9iqAgeTAkYfy4F1rSkulyi6AMft4jkMV2PvzhY+HCJQNuzeJMWhpg\\\\\\\/wCdQbSGz0nJ3\\\u0022,\\\u0022iv\\\u0022:\\\u0022e9+dE\\\\\\\/UXLLhR4RRCDtLYag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222364fbc0ead583853f08f076cbfbec0004b621dfebc8bae8146f74db8212ca10\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fjsUzsuA5NFzr66JgAllQOA402i9NWjhTibJhS5ymUBbOETx7t7kgAywmFFudMIeVgrxojbck06NMvVX1Z\\\\\\\/v98k7Rcae+GVj88HvTxZhh1C9NFjdE31\\\\\\\/AE\\\\\\\/rbSePXUR8+XE+RvR9oLl86pQqdK5Ab7x9FF21gHOdr8dpXwyF5LBJlGVGl351r9\\\\\\\/Iehm1uBt9uYIe1VgIdh7XuJbI7XE0nO\\\\\\\/HW63OZ0HUYBOEP\\\\\\\/FBvgi32AZNVbCkYZPI3Da07rIU5mU0ohlG\\\\\\\/6FPjjOF8A62sP5ydb6Qow4dTqtp3gUaS966cid2wact0eqYQxN4O41psJAZteXevfcYHUC+6Mko0UNHDw9RIpcaRfqhbUzcgnYvbPTfVzF7CHqpkDGEXX8QXrplKjhmun4\\\\\\\/pY1G4yQ4cD5BghR9UQhHZMm32gbLLeUzvP9OK5xyWyX3y\\\\\\\/PECjXhOl+7k+qgVXdPjWKf2JVUMa2fYMZYvzdkb4AsWPDg+c3r6xrJjoy1BXQqarUm7SqsGTFHe1TeUDoIIPwAfqRGyQ9XYZOzwJUzNUHsj\\\\\\\/wWpGFd9DNnRfkyDqOUqnoyg+8FEmqIxb76FIRiX+oRaDg0KfrGY\\\\\\\/nKQ5mGhNf\\\\\\\/cWAqFxJ6M6JBokO68er8FtoIfIdylIpOHAEZpZFFuqV9L+16n1hJVFX8j3h\\\\\\\/NwL4b89nxio5ryhRn7sTolIsiodFhkHtlr+YJ7TKx05bw0b\\\\\\\/kcNMXhli1v7wM74AJT5wYKdJOi2xtFB7V2HRx+vC3cpvedAE3oLzmrNdpb2K1CUwa03gAmV3vRymyfYBJZw0Co2YD\\\\\\\/PpXcJvZZ1eWcBfU\\\\\\\/kAPezSy0iSZjEfFIOF9naJUhrMQYNZiBQ70qwrLB+UKUFySpNYKcmsf8cEHGIBKw07OEs\\\\\\\/qJ42pdDhnIjg45lneZMewCqkMPlvdSeZUMGABKgmIAe9oJN3ygXRVo9rh9A0iD\\\\\\\/b\\\\\\\/4r8yeu11hNE3zTh1+oSOUSMxE+rVTiWbR1tlZq\\\\\\\/JusLEw+kofbzkVIxgIST9AziRbaxa+Qn40\\\\\\\/U8KQAgd1b\\\\\\\/3JsDAn6oRyBnhxgVg08chxIg3gUZs5v8vjL8cJUyx\\\\\\\/PHU5GOlA4oH9qbVctAyW02mr9LBbDvNv8GqYN8TCxCOVkFO5nED4hHEzKK\\\\\\\/V8dMt9jIKT5ZGCBSW1x8IUQHNUwjdYzfi9SgtIeIg3gCsu0jmBvt9+NVYeSflChlpPuS5+c59WAxTREf+Pis2fXB\\\\\\\/o\\\\\\\/d2nCW1lRQWFn3XcWFhM+Oim6XoaC9puO2eHxXH2XcdhM+In0SRU5JgS6q2btSd2+jQmbS8EX5c79JsB9LsYNfTsv2CjIi7QxxL\\\\\\\/qGxKFwNzm0UZ+oU9T0KCqPsMQxI8mxwVTv\\\\\\\/FFvYyLWzzrrPDE9nfWms0esKvswPtlxIbJaEcE3Hug2zasguDNL8Q1zCxVLsRkrOb3FCH4g\\\\\\\/dYRYIce7i8S5WWF2Xp+K2XUX3cC54SIhoYDdcy3Amn4vdXmx1WJh0XV2VkElF32+rSreYFwxaRVRSoGWeBkyNVs2R2ajo3J\\\\\\\/YqVjataLSLb4b3kO3TaZQcU73gfJsfQGZewz5WxZrQevfAggA1Vj3HtGkOXhwJ1MhqFvCp1aImIoE6DK5J+w\\\\\\\/8eO1MiB2SZzQbq70O3+Mk6qKET11YCXxL9f\\\\\\\/HXhSPwF2FvOUQ0Y8f9bkX9TOsGDzS95aDl039xlG+kRyyMrzj6U5ivRqlAyb3RF8RBrTndwxQTcSR88Nng1yGfYvo03RGEalQ9UfRJ9O\\\\\\\/xJaMVTTNiE\\\\\\\/CqDf2+XGXa22tPAt++KhAGf6HloKv7LBw752AI36CnO1GayCHaGU1R93t59ytsokoB5k+z86Tkukr7XAs0tWr9QGds3GJ6FEsIzCQ9qif3EU5FKHL4g92Vtn1aiOXxSgDH47LIfRf2f5Ie\\\\\\\/Urc2bHwMVMJXgvXyTslnV20zkP6ynWn9AW79OOF7w1\\\\\\\/FWsN5OO4TNRaox+EBkRSY8znbzSPYuygv8on9WyKfi8B19jT+b\\\\\\\/ByCWMFmiW9ULgakn3j4iKkrgGyFvJUFBfEAEYQGuZagIuS35C7fSPRdfKur0csuW8qjZV33SKbl6SYr7hzHR5VNKLjFx4cpZy0ktlhW1hGZ114AvkgE0lv\\\\\\\/WW7R8AAOvADniKnmukcehHAp\\\\\\\/XFE7knpGq6BtVqJ3JrcqZur40AtTDWGtv\\\\\\\/aVmNSOR+Jd6aQGM5nSP3xoooM1qbuYdbsIpTVMj0FwYik4aIu9yReiPQ2UBY3XdURDSLWw22rWQDliSW4DixmPx9zcxeYTuy5dIwmznPD4od+XAghn4yMobxw\\\\\\\/51S2Lhps\\\\\\\/lvyiuk76TX4hJlKqsbBfvVsAz6P86PVmUP4rGAE\\\\\\\/ySZlOT11ujrp31VUH\\\\\\\/kyjw1z1b+WSNqfPUmTv6T3FK1G1uw5hc7jBTGSzoHJwrmCTxfDcRNQGq+SrPblD+mtxcU6XQkcIbWwjcb73eaMrGBmUADSIfBlkPptTCvAZBYx4obcPjPwCxoI\\\\\\\/JEoDfTZBqa5tdJHETJyfe0RYrGHP5ZgXrbr3Tw0aad6jVM7kOHCMs5t7OJdPUqNUahYwuDER9om3JIm6Ki9YrtAJMP1vU0k\\\\\\\/cC7OY62zXtiQarqIP\\\\\\\/RuBKxYeR1HsveqgjlVVCbcErd8o2DeK7A2rzS0PWuzsyyP9+NgY7KtMrTzJm1u8JjIxczHlCjKdw2UcqT7EbHaX+qEEm8Hf3L\\\\\\\/sKwm1kVodyplfNRbdNMU4i45ynlQVi8D0wKQdZNbqunD\\\\\\\/ty5bTGZEYV1fo+GN5CCw8Cpoi2x1p8g4ssj6we60fWkJB\\\\\\\/mt1T2V7ZrFGu61MMHya8EnUAMs38p9qZZVK9liyLp+CRS2hPPE02fcCj1cZ6cBP8LJjqv8VOcl68IN7Vb5qEkiRZPkIhzWAjoLRgOyQbatoef34bxtBwOyvbf33CfRNBKZfcQRbQyPOJn3dvcqD4Um\\\\\\\/JxVshN4YRRj+VkJUAf8Re4ISIpAP9yE\\\\\\\/XsIRruWZ3Xh\\\\\\\/cneTMwNV5dciqhDRrR2pNhw5BboJw8JuLJgyuJF5mQchD7K5QYsBD15ON6Gnpx6QnzQfQltVHH0WLGWQy\\\\\\\/XnwjqVpJu\\\\\\\/Yq72m8Kg10qqnpZtIED5px6YklPqchPEnQz3Z2ocKpAf\\\\\\\/lbDKZTrTyKfnP4dkhXrS1JaILLSFBtLOa9cX5DRGp2NLWjYlS3uv6Lrr2ZNNVSb6Eg9vyxwR6ZKL+xs4qDVX5qRD5a\\\\\\\/RBEoRaxqSAiktwAZ93BJ1vIlYap0bqrkQuuT2rSc\\\\\\\/GWX7ujT2UuB9\\\\\\\/CMMCSwKC1iswAqfL8CrhWgGoBZb9ZUiLc2vTpNmypN2cQZkNxiOq4QEI6yTpSd3XEqg8hX7LEAUQu3\\\\\\\/p7oNZ7dQfL\\\\\\\/RySNab0TSrP8ZSP1bveCoGS4yG7z2JKw4mpBa6fInT13J34aVufmsSr7x5EFdLyFt5dSHFwLauZtJfR6p0mvdeXjUfYXr2k9V4Jh42WAzoEjLLofSMrXGy+i4xcBHXpBoYkSdHZf1AT\\\\\\\/3c7o9hvQXRmNPV4T+TfG+3sITita\\\\\\\/Me1\\\\\\\/kHJmiF9OFQBfHDAX3a\\\\\\\/K0cmnmcuJBDqxUlLUlOGoxDb3VE8qjMuFWp8tJaMoMBH9xMFKvqhdtois3hh7ntX7SvY98pTTl7ZmLLkotaw5Tzyu59VextswiMsEB\\\\\\\/pjqwoe4BJzTv4HPiBkw78ZvbIaL4HWqK8EqwXQIj6zL9wd5NAAcxkV4aofp6ylfQKSPMSJO\\\\\\\/pvSgaVFKnD1N9E2eDBuCVP3q4+aCu9g5wezWybqWtn+11Ug8s5tIqS6NUOzkyPyRNd+cuhLRZidpT1clPgPiMKDXteF9AjNAvukTW3SlcLtPCpnAxqmQ+mW\\\\\\\/CxAZ\\\\\\\/tHs4DtcFXxQi7JTtrqMmx05rNJdhAnrU6PFNS0WwEMf09XUyLpDE\\\\\\\/C+aYHtr3CLN5fH2U25zXt9ClVdlCykWJTECv7tZs2jl\\\\\\\/UYjoOs8l60Tn9ElMlwh4\\\\\\\/dM3AKKJ5G7ADtu5vNs9TEoe734Vy4bM7WTivWdxWiysJDeqUKZzBySDFEAw4zEN0qi3XrX0QGDxaXozjnjfM78YZVZSCcFeh632JuRanRy7uM8z3JMPiB\\\\\\\/\\\\\\\/zBZa021NbXWh9Iqyuc7R4nZBHFNJlpFDPvXLA4Fu6pedGJKhYWMFP0gvTebFREkZbhlaK5YfuewmJa7ylMD\\\\\\\/7ZHrwgmtRbuLjX3Bk4iSDBwjNZVy7dlDPF3vF1Qhqa1Jl5QI9jA20oJyKQeCMYa02IOyLOgVn9SMBBGV3IlI1ipTuxQQAwOYUc6LuYoBfdUOKgc54UB7Vuje1tqJ\\\\\\\/LTYoDAhCI3vGCNlZ3a17zlISCj5Nf9amJuofQV\\\\\\\/ycAE0eFX1GzgFU3s0KV1wqakcVdx6EMlowivd\\\\\\\/8yGx9kUPW5SQaOiF8SbcG9B4DGd3CAZgQjzaDE0i\\\\\\\/vxq7wrbokoZvsAEaYSB87AJX2gM+Qcom70+gmTR7tTE3lLACBFH5atwf+kFDvJzN45yEyqZnKB1pyfzlYnSP+N11x1jFUO\\\\\\\/EwEpytPFlMIULStJFs0Pgksv6zpZceitDP4OS48Agk8ZcIo6YAiVFJQVd7F\\\\\\\/xxDxiISbZfoZK6S67l1UU+ifhCp0IyO7rT8a\\\\\\\/TVTuHHYqeAFZ+UuBrFrDXWjt4grlM4C9jgqxCwVA0uScsMtskwOvK12suikJA5eUI2u8h3kxoCupFioON3Sg8nYj7KuAljkdcj5x8wwLH\\\\\\\/64n4NCHtu8w57lB7xliDWjWIJevsJPBqzQEvEGYDNqv0RmDe2NhUf2WkOQZUjN8Bxlyd1R8F0dwcHwLKsZTVTxW5EHM7CriaEnD5fc5npSEGrbckD2xqjPtPMyqG3kG0ss3PR650I6ntHOL3dYXbwwPpRfhhzQPjK7EfjCQIt\\\\\\\/7pOrPFeLmVecpa8Va1MWPhlHdSLftO2xWVr4lUJsoTRpoaItIF9skUVRo3tdoT9g+OHqSb\\\\\\\/8vnJi5T2XxoP36DIQD0gGUNcDM4AyjDNLi5ONdCUKByL4RDGor3AQEgSJRbHheQIbz9W+4UXFAvGwUZF9UtXOWkQOLE7F+\\\\\\\/cE+e9xK5OkvxoK+ShR89c9PG8\\\\\\\/Ic4E4qMIG+iXj\\\\\\\/glZNUi6VEA3UzfggdbPZv7cR39q6p2c8kqONuQ8n+8Ad3hZ2AASnfeak1Z2M4MdO3x3VTslDcy39sRSly0gvAP+10Ct1epDxxMXDZT0rzTZOsHtGBK8c2KjsMK8guroNqJv\\\\\\\/z6DQS+rmCguemtiXqT9cLPcBNScxJZ0DNWjlb5XgeZs5lsPllpNGGcM4mwP9PS8A3MOjcvyDt1k2EXQBrBncdbhMxo1ucqCrVYeKMtWP4Nrsgf70ta6R1jRi3L9EbkoJOak03Z3EfXMYa\\\\\\\/DjrmpXTVOXGpGrEbJhdD0nGPdEOyWKQh5oZl8GQEuyuVlkETkrcdtU5Y\\\\\\\/D6Bt7PQY5y+lzSbfY59+4dVAGPeab1lZ36IDQCxagUV2ian5FUKKGL77WkKEUM93Av4HsnGoTh57282bcWEg7YvIs7D\\\\\\\/9Uz63As3xceS3IAq+7hfrovwmAjp7VsQvDJoNh5g6kDvB+s1q1lys2I4YFSBiFYBIipfNLhz\\\\\\\/fEG4jnDfLin\\\\\\\/+pmJ7w4Kxx5R0oblHnpKbyzED4uSAyu51NQOXtFEiwg7P2I3PvFwTL23RBcvM5pR4iye3I32+szobNqcFJDlvO3o\\\\\\\/fZggygpECLZBRR2Wfm5FOjrAfCypcZHF1SSXTcmZuA40ixti372KfPHCvXqUIiVwNp9VNK9W7vjuEt\\\\\\\/a1J5DljRQumu+AiWr3qeK3HcLgQIEQe0BY3XJXah+Lmy7eBeecJSwqA96I70dpkX\\\\\\\/x5z7QjYrjJiu4C5\\\\\\\/35V\\\\\\\/lUUj6cBUwut4NDW\\\\\\\/MhY4YrMq2j1dWrTW7s28t3Yn5yS1H4R4Opq8kutleyRM9Er4WlB578vePMpTpPZTEG1Pgg20h+PO6S6x3sdLok\\\\\\\/US4aslDrSoe2+O1T5678FDvO1kMLW2r2YFr+i\\\\\\\/oHJEwGOCfo6yg2u4iPwZ88NqWI\\\\\\\/62RC1XdwouPbtYcLQokqknlsg75b9jCBiQSzAlRLOnTXLi0SEy18MaWfiJYEddIqvmoRGYhgjPPn4cVueyj3riGAIx+ZjbDiQtFNvCUFb1eFEi6nsuabL03+0QRQxkVd8oMDsV+GN3mY76FwS1bEe3zZHOSYM8BdblsKZJUO\\\\\\\/Tbs9ImtzBPPOfvKq2R1P+Eq3eEkQBg1VsPBpE\\\\\\\/fJSg8JCn+bwytL8FDETq1WqomYonWQV9rQW5JoJZ2rpNbsytYFF9x6K0PjUh3\\\\\\\/XaslMu2VT5gzEWaN9DzdW0nyzianJgzOj\\\\\\\/l3anBVgs4\\\\\\\/QH9OZDglB0IFzRIc64oHigU43A+5tx7ib8GmhKYzpXtrEXM0AGpiadsLQkakkgKQPF2V5GlGIMeYdiKbk1LEl28IZuglOAzSgLn37sitMcEOAcNgdQqkfbIfu38RZr6COXkfo\\\\\\\/7j6J8isJpoEcvTTkU\\\\\\\/4pR73a1ZZV37Q7RkWjfIpXid5\\\\\\\/+Q72HRaC4RtHibmoE\\\\\\\/Rsl1HJiQwiA5d3CI1qikBCxSx2hJvLCeNNMDDXE0hoUM2jw4G+kooSBpjBL4aZb1JGcitWsf4zRmJUs1fucZnZhGgEzomL9Mdr1wG3a+BYckM2qJcauAIWrF6Xj3ddpVGZgKJCZ+d6lUTYkS+DI1ErhvJvVkKYNLTSy7gXmrpdU8anF9K7eecxYXEW8ykTJEXry8pr4pyCkznrh6JQPTm2eE5ulmd9SjUzKBslLPU2WGQArpOI1eX\\\\\\\/44JV3QbqVc\\\\\\\/DZ42uwvm0hWwjf9oN3T6hFHEB\\\\\\\/JlY8HK83QWeeZqDlUR3meaKiyyrZ642Z\\\\\\\/QbuDxfuVuPD748JWDzACr0i\\\\\\\/rkG5B1Al8\\\\\\\/ak6tECVJ5wJ8uVrx78VCsIpKPxagL9OaFDWAd6nYjJly9fkwcJQ2JTrNiIdKAhreJOCnK1+VsNGHIWgzH1g\\\\\\\/2IG\\\\\\\/0x7iQzwcysUa\\\\\\\/xSpA0CTZL39jofcxDasKf+y3FVel7vFRoGUYFyQMCu6RQHUsmSBIYX6llEdndB9PLFxAjpW3tt5msabQZP9xQi2ku1vxqrQoihzykRDB+gd4kaUo1Dx0DQPyGbGJdVRvpnp8rbdR2sFcMhwogdyFkujLfVwmYxIpJ\\\\\\\/uo5QLSMXF1PPo1+3p+K\\\\\\\/O2jWTKEZOFIAxhlyO\\\\\\\/F9H9+xqX0MD59NYETy29qdREgZUAO4OsoxSdSTCT9VsupRlWriE7x21yyeqRXt3BHIpQrtPl20TEC+fveuNm6aP+8uMcFeGDzM+zBcF23NOibIyLzCSgzurkyOiiTpL8IG5PCTtJtIGduh4pRVAng1xO5\\\\\\\/i1CG6rMSsW331RIYZoF0dQe5PZFDMedTgqzaKeMhdf23SsgIfsZ9OuHN6aYrGNNr37g5w8eCLYUqfsyLrL9Qx3dRXAIUtyCFYlKgADl8neMjpEwneegrUnBA7pNXaOvjmkTHlovWkdrg6V1+ExDFYhNs0hHVhWcMtmEWNQypzEvpRjT6lBSnMMriR\\\\\\\/Jg1qvFvp5UoUGTNDXDkJVUZC6op0fSsuA+3EUeHHqs2kur22kh0qugl47eyUhg7RJ2qxJjdV5UOr4FoyUDQdLvebtyycC3c38a21Vu44XQYDJS4I6CIQ\\\\\\\/y2QiiBweQpTcvrQBuEVaD4THS8a3LL+0NfKsykqVGbKBagIejgIMUFLke3llB0BBY0BKQJdKAKEXUkh14y4\\\\\\\/PqO9ILRv750PpYcr37JLC0ECapASi2XgUiDC\\\\\\\/obTvB+wdB2M+Y9vwIDS4lM7cmLHA5neCk2zJ4vBTC9ttNlHlMJhYr6uI3miwWLghDWeex9wMzOOw\\\\\\\/TZN0dgTZwkEHunV0UOLksQpANMD5qQa5pZShBb5P7jVBsBog1tredrjOrLBUDImO7bCXCa5XhRTcaei3YJo7MCoacp5ttSZHjbfZtgSJgMpernL42U7Kx7U9A58Jpon4SvC41glTNLr71ff9xPwqCNDMWWqETuzb\\\\\\\/cNEA6XqVJZxNsZzFc7QAvhID7sAX5b2ZpWSev432541XWLx\\\\\\\/yRmBl3UbyK+L9HHBJLcL09hk4thj2sglOKyX99RuPNFTfThLsR3XYEx+7nUcOPnagaIvp1hJaC+mnGHjPQfbnePyEyaQjp4UbziSzI9bsPeF1gqSloI1ZpMHg\\\\\\\/r78gP+OibsLI7DBQPt5EMDZpdmv6uzmTb210M8ekCRIJ\\\\\\\/Q5fnZHwCLxwWuvNbBwdshDp\\\\\\\/74lbDQZWAhlRBqodIZ40EHG2cE9wAlwKDDjyFxNttnnFi\\\\\\\/f67yvVib\\\\\\\/x916f8oLTPBXIxoIy2oi9sn7kS3Qfc3d24hBdWpM78WuXQjeS8eNhO7kooEIn21kvwVItNMJTttHASvSK6xwk864V9prTVpGB5\\\\\\\/+lJBG0jnaLvgjQDC6eFekI6wN7WA9+6bs1JrWD2A89\\\\\\\/Ees6+RM60g9mNWfvSQJwwTgK8Wn7kn9tpQ9K5ZvDmIT8BaN6W1wYknjxoRZRW92EaiTyASDWkZNEz\\\\\\\/By6\\\\\\\/5bAfn3sIfnCutpVu1OEj3WA6dPeLlZdC2\\\\\\\/zFjfFPD3C976tH6VSHwfAt1XknqBjEAuzyYwMj\\\\\\\/aBFkNwg0C3vlxHIpgRMil\\\\\\\/\\\\\\\/QSCeZprGtNPMe+cdzfhyp63pGYuec79hmar6YO18lKu4RE7SqG1TFSZ4Nzuuue8gc4122Sx53o6lDcqa\\\\\\\/iCo1zcJjRvNdIFJYZbFiRgHjMmsVbr3mzRdLPYrigG6xL+hlsnthjTqPppiw7SUsApvFRDJQkvgDFlCqGnPTq1zXG\\\\\\\/RQuVcacRBBpb8vu7A6CVWLLBCoF3Xi0Xk2RT0WQmrwCE0Gu9RXmyURclAwkyfyJxrEX0wFQxDINjW28s6oTXqqyxd5ilKBbW0dXvdgNNkjwdo1+VFggQAsrA4yHMzaAzg0BLcrehCsGZEVXzUjQLF9iBKzqrXeVJtHWSnrNo\\\\\\\/zSGITyeLeP9vKa5vnQe\\\\\\\/Ia7ElbwLTTpYnmd+CvSgyKmEjsw4Ggc\\\\\\\/355KCEJz4dbi7HtPAlj0IKRdhRY++ylT+04xtEO\\\\\\\/6mPO4htaTquO597EefPvT2CCYU3dEvJhQlstOL6BRHeaMLfrMmNE+TjzrwvGqMF7t1avVrgY2YpP+NSlE\\\\\\\/+Zhy2IyA33dTzozID5hzpAEpuPVJNNVO8JwuOwLDHmP8nor8C8qbzdZVqwzLCSjVrraMH7gyXD3iGoG5jiFAX1ELlEZlOz\\\\\\\/IT21eQT+sKve5Jt3H9lQWMnimwaAI9eJnjeDQqxOJ9Lrf4jRi2IZXtZL7AnSrd4jnfXhKADPA6oCvVuHa1QG6RG1UDd3uei70WwyyL3iv3nrGlushmYrdKuC7b9hC0X6gAL6D4ttPmSgZ5oHnqAPHXNZu2VNG3hEQILqtVpl3FKS5mtrxTPBiMJm8cppJInYvp0bQtwzEaOwzdqmYRrStDMmgOy3t\\\\\\\/yC8IPBik7TnO5clLCTjAikT2vuGe8HYXhs\\\\\\\/QQnO\\\\\\\/6UxTwjCOIi39mjMpIEtYKVMNFbneLLxi1feCmoDSAcv2tzF6+cFgpbfYhU\\\\\\\/Epejjb9gh0D7rtkQiDjN4ziOyBD96gGs5ymXttLCjxZv092BXL+gSd+HDur0qBj1LR\\\\\\\/\\\\\\\/K31V1z7esrimSMdh+ZSjWwJulzXBHlwDN\\\\\\\/fK4cIzF0AE46ojsYyxP\\\\\\\/LygU2ENNsCHUfQfGszP\\\\\\\/AsJenHH6ppjFVdatl3P+jtdc4YL2NcEwRms7HTvmGrNpLqZyTcUwqwVPGsKl\\\\\\\/QdpobuWWr8Ups6w+jMHDnAVQcro\\\\\\\/E27DiJZd\\\\\\\/0Jikb48HxwwUJmxix4JJ6i5Occfo93YPqcFG\\\\\\\/UtU0bgr+Car6XQDFYgcgHDMSY0qZdM60pvAu+6MrE2w3\\\\\\\/H5by2sdcbtjdO6r9bwIW8CB6srSCk23AgedmX0pkqWNbsqPWKJWdSTY5EeE8CZ1XRu74Y4VW1LUIXQIWUjFVfxxUY26QpA+8o+yuZtKzdxnd90XP8wsqWuXWkdhfoUmHaL2O9Jkedz2ltQU5PjezRxsddM+AnXE6giHzBtYUEXYMsTIWqk24ye30ueMO58ArPX7vTTRFJP8Pbsr\\\\\\\/okwCK\\\\\\\/RnJO1U9zOPVpg1cjGPKg1OWdYPdar6AkgBgDpWxfwkgTOPqOuTOvq0z4Qi3GbZvQZo520t\\\\\\\/3smF58qY+RH6o2EzRvzRABD3Wl+Mzc1yPEGzEEuW6mnyRfT2mLUMvFklR6XixejCpFlHREkKG2grwrO28TY4vB2QV9dzf8H0f3QXJLK4zrtkPRbgAxfXVAaegOEKyfsSR89pPca8hJNAUN0xbDStJBmkW29iu5LToAK94YMy1Q3nkS9N3H4ud3eMTvRIoJOFHgEFvM7B2x5Bk+\\\\\\\/CdtK6Lxd3mWvXr7BRQPO18Vlpm+UOZcVOXnKNvMXgBaBqOzcisrGM9fy\\\\\\\/ZUcaimtvJOlPwnFBeBu\\\\\\\/tgbMN7oOvf0jsFngRbsW8LsV6nxX321YF2t0fc19aDKCD0ChqxiA67K+17dM9KXJBe47fZy1D71P5SHlkH5T49v2GJIjCiWDBPCCbaOyqcF\\\\\\\/mBmoPGvXDcOM3W8\\\\\\\/pM0ld8qlScAlAIEI2lNXKDn+a\\\\\\\/b5+AvVnQMymbpg1i4LNN6gSsgxTaa5jHo8OnEzaSCmTm6FeM2G8brBdXqc8D84bj4h+ThByj5RwC0mM+BfPb6IE3eJ7sMfe8a9N8jwvFGg5xmoR9LXUxkMcWswT3i75cUeiLMi3jStB\\\\\\\/yz\\\\\\\/3pas7L56grjJid7GkiFAZF52vpMNhsGgaDoh4zCiNushletVwdNLfBQfgmUjTd8Sgikk3th9qSYc0kNOl3yrATLClrH6ptDSYSUX\\\\\\\/kAQS5GuJthcdE\\\\\\\/4oxw8R3+BAVxv8R7vYlKgH2hubaYA5xR38OU3Kfl+71si2Qz1vxC9mP5Omij9ZhXeoAGxwFSKmGUOo8sdosgorHNxn3RMckW0AkLSMrMMiIQIw3w\\\\\\\/LDhivqaKfhBP+l\\\\\\\/PDLlUwRzn2JpLx+nA9w7qjQvQJcijgUDfwg8CW0DaLyKZjoGaKW3lTLKFjIUALOeP8Vi+zX\\\\\\\/heOZR1HWvjen0ixiYzpTrSR42ge6qZKgASE9Ig4TXJXXz17QPVLJuXKz2Hh7YkTTUOxtyL86au5XvKR73D1QHalZ2+PZrOQm1psPDvSMiqCs2t12\\\\\\\/Hps1fwGwHUbKVzP2PZkGtclCjwfz76ErBhQKCgcg3BnwIEDqPWYtvItu017zwIgl4Z15x6AsmK2USd8uk0k+PVIyfdSF0vPVJik26VKpjZG8WeOdJO3L7nLqg6LEp\\\\\\\/ivyoTp2o+CJJaCONVYXZXZwM63jTx8b8wfQr1N+Z7SxTZh5TOf6goZMBy5PULcd4+9\\\\\\\/ZpPzNa2Ue8\\\\\\\/qJzQaT\\\\\\\/uH06tGpHxvmu9qmz8OEv5LdblYKdxjuEN\\\\\\\/BCvAjxn6GTS14eCSS0XG5IEi8XilWRckXn405qF3Q+PxhNHs4BEnEQOAh2b2PysMDa3NKXQjqK\\\\\\\/PInGu5J7LiwJMvWWSfk2qvQ+sPiqX11i7daL\\\\\\\/YZLhR9TPtflbE2KF110xhJwRQ929NxrfuR\\\\\\\/JwcOiXNQxMBniWdWZx8K53mzAaG\\\\\\\/+YSyP+KHuEw72r\\\\\\\/Egee1pbADve4xebP2DVnYulHIkObv24GaK6bYvv3HmWH6mKJK3rhgMQw2sjjiSV7edJcwHaOrTPRLCjhZBr5B+v3PiuL3boabw7A6Dq4cD+mdOwi\\\\\\\/B8wusu23fgQzsNWLqQ8T9A49SMqXk9NyR1qf+500Tj0C9kuShEtlrPsy7ETc8DkbSuEPeoFwFxF0DdMAXacY8UWxpiEwi2\\\\\\\/iBssvKMMslq00CAqVjvksDwq6cyDGdH\\\\\\\/J9Bb2hYvlRHz2iohLfzG1jA2HboZ3v6Lo64BoDHkvryReDgtqrMbvK6TJUFDLIim9Ni3aCoz+lMbvmOPFu6if4xbMf1P3x5yQIiejgysjU2NrE6L8x4NyckJ63aE\\\\\\\/otEpFVgoQvN6S9uLH1xUFwugUxzKCABPLceKDgBVjWNoi4oOl\\\\\\\/0en+j2vr6BM9O11NF0Tg8F\\\\\\\/hm684\\\\\\\/\\\\\\\/aWaoF4PoRWdiaOXUa\\\\\\\/OIpC\\\\\\\/PGHfiwFa2q5Y0iPUfvc\\\\\\\/fla8pw++Io31RwYtzOROSoQdbNAA\\\\\\\/vJGDE9SzcaWckP+Liof\\\\\\\/KFsRkIqiAnzzcQwQDirQExBeyVYtLdfQATtcuxyI\\\\\\\/ANeMfwxtCLxTLmLGTxKH+VGNNHa+bAW71HHRUpiwVqxv8Mjqp0LA6oXpEKxsLhDvsJcuPfBqpdDHsOcf9ifuMk7FhmeLNd9MGBn\\\\\\\/wwjU9BBSrJ0HWqxj1qdDPcqob23Xb3zt1D\\\\\\\/DYVR6LppUK2kRoO2gwHT7ISEE2hBGoT76jESmGc2KW0DFjwaUEb5zNbuG32n3w8BHmdJEYamJHMjBXY83CpHs4qo1eHNFD2puZM+pw0lQfoKMAnvh4HTcQize5fkvX+FIJBxITlSkk7tSm0wqYYI6sTNvSMz6KJEENHXJnCVOFDi12DxcVwqNjyBG6f8ohVGQeDlgzghFvQ\\\\\\\/62WJHjOtF8REVYItODhny2pa6F93wb\\\\\\\/e7HTD7yjETOmOLkyYYtvDvF\\\\\\\/yqiW1gFYOYWeiOHekxLXfAk\\\\\\\/CRdSZ4a63y2qkrRcuDiLEQmGyVwkhJ2dWV1JWTHxu24X5bVciB\\\\\\\/SdhQuqpGGMcOUDR\\\\\\\/OeE14yTLLzd\\\\\\\/DQF7JZ\\\\\\\/mS0AC04GlZ9+D2BB3egbI8B+pCbO+2hfg1Yu+bzK843\\\\\\\/Hmpf7UMzbLk5bFx0jb7x9G271PUkwuKYEpEFTCeuq4uyoWQebcEKvtks2Hl4aFw5cvOubdxivK6mP95o3RwaGalROxRcwurnCGvSFPh\\\\\\\/urrhS5\\\\\\\/iTPeElQklBJPt75FLwfeIEHhYiuRdd5dEgqVi6qQyhNSdJK2WIkXBhbrZkc6mfd4I+hvKxCcpbKU7qK4EYDOFKNpC2uA6iS8HozEJPADZjLXyl3HvPDAK1AYo38hw9Tt\\\\\\\/pMbUHU95epkQ\\\\\\\/bt4J4ie154AQOWSAsU7isFi46YBVF9w4xvfNDlxmWMh1DXEaqtQ16\\\\\\\/J9ZssH5CFv2tmzH1ojPkt1PQIqKY+zq7vgc6fmVDrpytAGa9Kw62w7M4jnIwbH\\\\\\\/GklrrEq9JFQwPTsUlsDXIIYJC3saoteR8KWSwDWqYNouv1d8g3vOKqnSXhQyDmvMEyqgUcLeQuhzlQ9JzaAXjuLo4K4Dp8O7+F1011vC0VVBalQG1IfWNoiHHSZYol1GtvjkGKTLzjlL2tGcSyyz+ePjiTqnwZFC83fK\\\\\\\/C97R38fqSiVsp6D5HisSFluB57P9SXWekAQTET8wxI44N2TIeMI9JyRoyyTxCbdPi9HxbtntZAvmH2wIFHJJOHzqZOppND6617Ps5gQcaMLIUJVdCHdi\\\\\\\/bPgx8s9DNVIzrINDd6cCFthMhTrs2IORY\\\\\\\/DRI1+qDZCwA2ca4tdAioC2gHaVppiCek3DDM\\\\\\\/Hw7mx0xC2HYFvjj3aZ506+3zBi7c5zo\\\\\\\/qk22gVVUI3Xy0v7+1T8e8ob0k+4tcltFGQtvkFifWU1Cga\\\\\\\/e8wofZQAcPztv40xgA+eHixXBTQMiRbaaBMuVEmktr5aKTGTtoyKkZncnv4bEsbzmrZ+tJcSudQMKive+klOQB7l2P24MMV5UeQ2A9v6CZRfq4HcYb9V\\\\\\\/BsNd9b7euEf8zLLEmURpads612Tzoh0LaG7ofTNqKDp3MH5bFllllwQ0gvvy7YEQxliZLuEukELXinUC6vSrDjf2veImX+A7iaX3FQSnBVOIQrI+YS7kMU62iQykGwx0+\\\\\\\/KAIRN4\\\\\\\/g7nJtHqmCYn17Xc\\\\\\\/2DJUMMG55qP5XlxDqq94uKiQPsN\\\\\\\/wZxArYaWnGxqORSRhIi6F8meE1Ztwr8V0lT\\\\\\\/il4OEY\\\\\\\/5Tx5+0vb73UFqP4fH9g1aeGCsGYAnNBy44TRJ\\\\\\\/Icsom5+1FUMxqQhv5LIhaHs8I0PZE3rPUeYeTbP4pQojhq4MR6fPgdBp+2iQNCnbOsVpglUHTe4OHXXFUqXAm7oy3oASi0aPPricpZrcTtDqv92QV3YOpcM9GVlmQilwwt3EXl7b4WDmuCqao8ZGrQezMlZjCVBKdIAag1MkfAbl+jw+\\\\\\\/7GfjELBuHUZBTwAFiiyiKLruoupxGSdfY6PZeDMLK4MI615zAnpZjoM+fw8ZXTM6+5UjoVxGJf04oAe9mgIZhRXFPWA5tNRMBzrtlPh3GoOfCazwx+7Q8hnMg1wyNifcHcjFPSCwZwKz+G\\\\\\\/MW2JEB+IUlNtjvJ8LbG+yDdsr0L+2RJCtOAIB\\\\\\\/mFZrQ0FE\\\\\\\/AudJrdxC\\\\\\\/Gg5uSxOicGeAXYItfgaouxMOP7plxxmKtCGIUzICryZdkVzIIncO85RZeiiOHXhO0GyDyiwfSjkAFGN8ts3PUdXThvK67lXRcFFMLsoWrJXT4XgAk4Jqde9IRrFBZKyG+kDa9ixwyiEZaTnidxRwOgtolpnt4ZER8QXrOhs2biJ5VbZWfR1riXXm9S1X0iZd9F0aahqd6spfmYD9AD+KZ6Yn9nDkT8DhQKuSi1M402X3kLtu35EWEkQYw8RtbQn1f92EroD91VYB4aD\\\\\\\/Ap3gXMv6M19yUrMOKCoAbdqh7L6xeFRBN+whw1aO3xqRpTfE7ncw0LrGQV2VjhNtJXepgwNNhuJgh1Px8jM8E3UdfSgnKS32aJFXwaPDtbiIT+RnljD0r5Adif+Qbg7xIO9H83uSo7WTeu02F0diy7\\\\\\\/MdGcycOr4UmFCt4V8H28yX0gcE2Hr26bC6u9vA8XkTor470hy0iN3jQce2iV\\\\\\\/\\\\\\\/7Ue+aQBuw0g3sq4Tl+u\\\\\\\/gmLdgmqesh8owIlVJbS6z9A2LXmrlv4jWUQye3GNUuojmz8NBKUYxbDeIxxJPZWwUZvMbSVIdcOb4i4XNIiRVJZPCjZQ38DLdICBVZBJGT66xTvD5oabP4MXV\\\\\\\/7UkwiQ+6cmD7mQ8s0aq3hpMBZorbWrQgpUspLgUyf719\\\\\\\/EvxkXqHlVC9m1JNI9oLRIExrV7REHQjZDCLTuaFLXRKzsNyT8UzKao7mKTRIPlYHYR3kzvMH8gMkdj0NUR1ihWyyp9tr9NlcWlHzpFYq2b\\\\\\\/jtimbSnHnfrTgwNL2BzsYtafLDNNDQc0ZxJDmJrkYShAqMm+yQU+3O04bTQNkb3MY3Lpm2n2IOZ\\\\\\\/QGGn1bxBulnheF3TE8CAfBoTQFP8i3ankJ+yQ3VpouQQMRekOmCRSKi0YUKeeg6veWxpvKhnStU2FOZuP67GWWkXmDD2+RVTZZY4ZuJiIyf\\\\\\\/Rquku5DFCGZxgc5hr3Ad16y67QA7tBGNLxQyNNa+yQYA4sGoHVLiievYL+qfc\\\\\\\/\\\\\\\/z2zZGQmAHka6xS7Fre36Cyr67r+woXF1I1cSV7JbabTeLLaIrFILJpewFaWH9LTWqAmw3RNbF5d9iii1sVRSbDT9QswkxwXzeo\\\\\\\/LOAnqJ+SiftorU3b0aFdrnHZ\\\\\\\/5yHzQM5D1PpE1aHksWgUm4hsMknXYOTkHE+vZV2MvO7j2pjwnMaCJWm08NZFzLRfqLUpdKRT8yneDLoVNO3\\\\\\\/Mx056Cdz4lvt3kpmF+nKmLWZgQtXSG61zBuYf3y7rQtUGVTLqqpDlrNq9ZV8Zu8wSNPBq6nnDNkXkhoFB2uogBQSH3SJ5Eq3kQjEqS+vH3z0DkWgPTv3ubuWefjVrAxsJ+Q0nYHw7k0FCy7ZyE9nQVo3e21DHJ61Tmqbuk9TN\\\\\\\/k\\\\\\\/KEAQDzNYI5S8Kiyv5FU6sdefK01d9s\\\\\\\/AMthvkXr7NHje1FtkEqE9kH4vi9kfT7zUmY4M12l3HDgNwq\\\\\\\/tU5KOGNVFHrXp+t0DQZSkfQ364uVutNs6DZaMJPJbz2Nf0tYCR8F1gyfodP+WhUCXKzDK9V8917NLE2BbTp3Io74svBfNZw4\\\\\\\/hQLQqZ+Oft6yMNu+KGqROP6cTLeqMNwseS8bHUHOx8vpUtg8hnNGdBoAv92npEjoEMHYNXYDuRe3qiVkDHTnA\\\\\\\/WsXtjpfJ5Qqh+R7nh98znGqBYF\\\\\\\/wIWLh2Fxsosb4M3S67ImP5BXRvvPQj7Ejoe0L3dWCWkM5T5ie\\\\\\\/j4XOioYwuxHujLvSbZ3fDGIPWcie3EE7RcdNEuBgW9Cf50VmV4hDUwzvxLzwQeeT6+eFrAr7q5LIw+9Mw2R1Ium3ZXok8mok63Db8UGi99skK0VNUs\\\\\\\/G8Iv07N\\\\\\\/5L9oeJ\\\\\\\/m4jIE2N2vZI0CuJX268JtemTj47C+5VyrySxtdU8EU+5Sg0DTl5F+mtvvw1ljQpMJ+94dswy12eqF7O9zTsQ86kg4tmi4DksEWVhHtNXz8OY2SS4erdAjkvvxfQWJQb\\\\\\\/W2rj4YRpnteVeR0qK8WWVVkwuiyhsvz4WEoMM+cYnkJriaZje7GOEm6OcKWgvK1ZGGNJI+utzw2Kp\\\\\\\/+q5kSgK5k+9CqSoTKQ+ZU8RPKvGfbeYZBqc3Pr7oSYIlLA3CxOgAMZPxCuV\\\\\\\/vWHKu2ACbRhjKREy+m9ozk3mg3OvcCMD4lugaMkJAUXbsTjXP6D1pfXXVTlyz+ACFyDbMeqF4MtcrkFm1VxvE3hiLyJ2NQoli6s+VboJukqNZ5i9ZfoXfRB8VS6lTEnyvCWq2\\\\\\\/3IQkHHx9aLmQGZlt7LcnsUpTtLgVRvcfWDC9dxfWe7OEcDel0c1H0hab+BA3WYqzhvWYdWtAZP27j30ioo8kBZJrPrTchFDMmKBmGcHj17EGAQPzdtI83GxMgoqcWgyv11PqyQn6+tIgTDxc0CKY23YjO1+0fIIq+ZVHPr4Tvk2cwRzSBF8V7ZTegW\\\\\\\/bI0dS+WhQx3QdeUuOHfOgQuLw5Qo4P+GUgRxBmVlb\\\\\\\/IA1AHgdBxyc7973dEcyUmz21XvyRlz77yOAz3rpqZBDIO9fryDdxkZsJouH2MvIRlDYZ+xFzrItTGF9jZpt\\\\\\\/pV6ddZ9GyI5c0Q8phGuyQUZnVBQx\\\\\\\/uMfzLEikxWmWVpf1o7QL4nqbbu1MTwCQ5h4OA7CBsbif2oXTgmoY9EReE9tVtnidtNqGZgtFGOzhLkYBx53eDhiDkHlvHkPIYx\\\\\\\/Pk3Na5Q\\\\\\\/BKuaNP+4uT8PxGo4DzuIM9N8ZTVElox\\\\\\\/96s27xJB2L9nh9UzvZ9vDIUGUAVRJLZcg2H\\\\\\\/xtH62Y+ubelLoiG0PaaRmD1wTBrCps5OwPjaVe\\\\\\\/yB7DyNDYtipsmOvGSrTUaP6\\\\\\\/OgZx8K\\\\\\\/N96gfpxMcfbJyTYnNnQ3Ffk3jyUPqvVnIIzZOPflwDVWLYMHaKLfPvFCbYzrdAee23GsQIZhfx\\\\\\\/jFf0ydEq5Sb5F4SfVjawLUaTAgzZapt6vyu12D4Ws6eARNfcdYuQ+kARkf4FlKdq83gnv1y\\\\\\\/9M\\\\\\\/eJXPa2VyJ5YkUesFdZu3h+HhfklPwsqEO87itGm\\\\\\\/hQ2LTI01DIXfRahVwC\\\\\\\/o7TWv3K3iJMnC9pGTJvaO3Lzb5VY+JFe5EsLZIdsVmmKrh8QTtTejTRxKbO1Q4wOK2inGH8akEAgIKZCibBk4S6eAxj9GisKlrMQuSpfL\\\\\\\/JXPHsVHaEjtyYvmhY9zsA3\\\\\\\/mt5f\\\\\\\/NSI3i\\\\\\\/QJBZ31YiK2PM3XvivVaQhEgZ9MSv\\\\\\\/ZbF\\\\\\\/zEiaELuy+KqmN6+iyJwu2br0+T0tSFbnlCjnJJ7OxckMCVfgnjY4Nl1Ea5\\\\\\\/MXlS5jFyaM4GpHVxGQLcHM5R35n+C3DiPvxYIEkXonPGo\\\\\\\/UQgUSnf82VCc5Ed668\\\\\\\/RB78ch\\\\\\\/2qWh+SfKxGx033EWqrjlNQBH\\\\\\\/P7aRElstS1UWg8pEkUGs3DwSZ1o6nS34wIojzKvaWOj7\\\\\\\/L1wcJJYEGHAaDMDznmmN7TypIWlTYogAYcVvvghTcFC1aT7bTYV34HQ9zSrFl2FMFf+UBZm2+AB4nCVYwGre4GNGaRO1yJgewK\\\\\\\/YTFiz+GX8+yEJw70AG2VZcO16vT\\\\\\\/KV45Adm3BoyhPsiMQbWcGhWVY4F0GaGx5wLqebc1b5Yc28p99gN+2iBcGqIrDMXdrHF4U+aA+qbS\\\\\\\/ahLN3TwMCpDCeR9MTVuEQl31RnVRiSTWacPNTURh1yDsvlzhE\\\\\\\/m7k98Ps3qCP6j9TvA5vTdT19PanvuHtklLBkDiiJlhKwlXupPowEMFpfC32JZLSx4rGPJLKxoylakU3gLi5RjZV9TpBD6u+5pUIlk1fjHsNuPQCHB+rMa4r0At514nT76GggrFjifQwtYesoz+vLT4dUje0lInTrwEHlkXuvUlJXKH0geXB98tYG32TLaORJM0wIgXPRFDQx6dzQeF2\\\\\\\/TcTWoLHuY7Xiv\\\\\\\/czzc4\\\\\\\/HgxoIlxfeuJ+a27Mnvg7F2KFueVU4wE3cddSsA\\\\\\\/2Ei60mTtfDHv7Am00TAI8e0Uu\\\\\\\/dPno+7pMhW7Y8\\\\\\\/BylPwL8u7WPym9otVfrue5O4oJfdcTT7x0K76dZ5nW5H+W8dVUOR2Eu+vI346t\\\\\\\/oL9luVBus4aznlhUNpLNQfLd77HfTJ5kfh9KvGcA6DhEr+qCUpLpz97LH6Vx4ghWaqxdiPGkNyB\\\\\\\/usXreYeLaukRadu+flJ5w+NEXepx4FrduRrmj5jVxiNOGH+ElINXSq5o1yPKlKz+vT7jKsmu16z6i4t5C0iv+722d28pre0paXI0EzaF5s1mcr8ic\\\\\\\/o2FsfWgPO6VcM1WvIUY7y76GEEFUyh357VH7jScIfsYEdjKLx1T9GiqtZD\\\\\\\/clQ2UPD8AHgR6Miy6zinf9t5\\\\\\\/OsUYRpvBgXzNivbxM5qh46JdGVjMC6xjbpjoRM9ibqBv6k8OMGGMDh\\\\\\\/FT6KbcjuTZtf77KUy+84tKWM8jV+nKdcXWd4dpefxA2XDru5lf\\\\\\\/bmHiVmaNrTtX\\\\\\\/4ZuJCgRg3BHvJiAPqklJBf3Y81tDEmFsXAGbxqdLel75geXpKILSqE\\\\\\\/no9mFiqin4+r\\\\\\\/V7qqAeyB\\\\\\\/YdGMcqjO2Y4TlUaiY6ej+GTRig3wgH0CoHhQ\\\\\\\/XPiOUsiTwxHVM3rM\\\\\\\/aXpHt5hDuz1feOfGZP0\\\\\\\/HkWbnivAoSLornUjq4kLO+v7OYQQSsHD6UqZNi0i4aUVtrDup9RqusYxFSUZEIHqCn\\\\\\\/qVSGAYFGYYunZoORNIU1QSpuxHU1qjBUzXqzDVYo40IhX4VMNNsH8vwSUwbO8i5Rg+AbGrtizEgN7NZst1LilgiL1687yp5nKmFvq5LWi5Ra8d8jCKK4NQpzROIsuGu0PQZQ7Dv257oPIUHefCLDtz5JIcoyz+HiDewxhU9keIKS+sMQIYds4A7euHTej8EL7tFDg0UINs2U1gDwmINaw0KeD7m19BlKB4HKSeMbvBF2sDpqSM\\\\\\\/MbZBpb3Sft4nDmBFFlAEYxaPNLiS1P6VxqsWW4g+qWUHE4DzBF89Z7aWVuiMYsRs7+FJdTRpuCjyPa+cJvAokRQDN6jIv7Bs+RfwCukxX3oHbq\\\\\\\/W7hPYTiwd8+ifXGFJpRHgLgnn+eiCEesm4FBkFKjplpjT5u4d95Gs5RNqlYlBfenRTUZ9AmQxKfWM8vPG1EcM5aQx4W7sWj8LlevR8GKXm+IaE7KTuQgGcx5wu7HK8EsCCTIqojjMXSvePMP9K4Hynr+NTB1uhrh5GnfqT8nwPnpbHR5kHLBaM5g6WdqTVfE5s0\\\\\\\/fsVMaslUlQHbzX9bx1H5uskkBH8Tvgeo+JovLO5YcANUDzyPSZPirk\\\\\\\/VNBOBbDX2gnPUo3XdgjtPc6LvT9cLVn2EieqK\\\\\\\/YNS+Y52QujVjWjlVuycDYFAtWossYgteMLXS6q16Dz3yb3vaay2bKbqg6nOYkyVqIZzy3xDngKPItCJHaKqVrHaxLK3ugEuUlP5qegZV3765gnqFP2vjTwlzYKJCT7\\\\\\\/TWShE9i8wOw4lGPRzts86MXvNkWIVctYGO+\\\\\\\/LojzZAG29LPbRsO7CtSYz7XWBcXkJeaUUz6uiBam1U94ejkKKYCd77dIZVb0VSTuu0fgQex2vLdYEfZ128PYyenhhrYioZlCOskjdqeKfBdR1vUB1gdON+uJnnKooDvGStyn\\\\\\\/f6yVOMmYMzcZ2\\\\\\\/7DMXa3zgxmNoTAiU3iFl24uCvL2rxs+M1jVBoUKw5Ml93XEHj9gQLvg7XtY34qWyxUf8rpxZXfdk2lsLVDEadpS8AcAVzhEfCoErQy\\\\\\\/ffzS7bBsWnB5YK1zLAeoQa0Rcbc3JnEeK8Q1e+ZzhNHPOYcWTrQxqyJu6ZwNUFYZdFESiEKMjjZURcU8d8ZXszvAi13F9x7JONBQ9\\\\\\\/xCVlYZfF1redAnVzRldyPcJCLMMYST+K2SweIWUNrRZAyfp3f5c\\\\\\\/piob6W0QnqbMZqTwL7r4XwJfPCQ8bVwD1P8LQv2ybp2HIUSZqm3r2fvD4LbzxKPc1QzzBbjKe1bu4VcPPRk2VJnBvQUlV2Xv1k8jss2qaIb4WQKyuE3UxWArYWNzZsrmzUwV6mYtDnZ5aGqFCuLhu2rVJ3lYrJSMh3kfauMQ0gyxRpS5lTED69igECTCP5\\\\\\\/xRvQH+5L59fkBPgiRpaaTyoXCmHLhvh1mQDPjTqwdUKSQ+CSEXGRwzV1XDfbhpD5r0xx73LJYBfRosDeqsYEkTWtemi0MfrOPk94vLPJr2+qcCpG6\\\\\\\/wiUqEgG7w2LJKa7p8AXULSWOVe19BGy23xokJmPszKDczQxze\\\\\\\/btp9BTB6Fv2nmK5vQL26Fh9+rAd1p++iovrJ\\\\\\\/JpGMqlaAuVRRkJMNxRHtrrDjAEi4VEab7EuJSkkaFjHPchXNkYvNRVQZcVv8FaUCfaqpiBpstYYvyVfbFpC+tJE+a812npUwcK9mC+At9xDJlxbFe60\\\\\\\/VNkpKnc\\\\\\\/LyruRtyZGwoSjZTkXziJhEmYdU+ONDvyTgJQkO0yy0cHEMMb1R6Q6yP7WoctVlKx7FbwE76Q5kC0Krjlyw+2Dwn5YNP1dVUNgfv5e4oYDhfTqjNWdKTp1vT7r9zb2cK1JM9DfjRjRv4D\\\\\\\/8UoeRNYddj2L5raBngKd4f18Ga+QpS3kgsgySsrsnBfJQbj884FS24cV2EoBB35\\\\\\\/vAgV88WzkadIw8RXLes8\\\\\\\/UNpv4ygvIqJt7nIaM+fxOfrzVYS2ewFucZbgpEi5RuPGIIzF\\\\\\\/a3H8S\\\\\\\/NVvzrQls6wqFp4bjQSFiMUn5FJsxcHgJJz+9p2DNj765lzNDsbEvp9\\\\\\\/AeZ13LIbJpb1VapI2dia93APtD6SjHdhq+51YFueratKs+t1sYEP0WJIjjwfDwZ1ppe9WKU6KiiWZbKUuSH9FP5re\\\\\\\/wDiSFkQPJaTMmywrxlrvbPjwBCCbnxFhQdRPYj5zlLVZ8suEJbcpQiZ3aCZlbHsW+M9K3jOK6umyakxYZoOFec27xvT3X\\\\\\\/AP1zioRAocX+d9ol\\\\\\\/kzSN4TUBG9DZwBMjo1G9SeR76VoDfmYWcFB+UAUoqGxwxX6sFDuf3FVm4PQ+5pHOCMydBRJnDtOR6TIfGx92Fpl9qQxgcawIA6T4iPr8tN1FvlJ+JYLMhczO3szm32UjKdQtBW9z1AAy02mmnCfDaYq6HmyFR2fchGKIPFMLRKbpFN8cllNFJRx7mexr7ZSnEfO4aFNHGU0+2U+wV9Ce9QH5V\\\\\\\/ypRSLOL8UcIx+8uAOiI\\\\\\\/0BPjO\\\\\\\/iDyIVe+S1mwos5MARLM7zg7JXzJ1EkaR4x6aMQA+TvTmtofmqgmA1gEH8+HZnjrLut52KNWLgxu+aRMP3hsAMHSyElPcDg9+xOVMlzxCVlVNYzvefpfrMCuNZp40uZkNx04KA3aKRgPDx0GKBdIpeqGB9IS9u7y4+erwTsMFcy1WeUFtLfGu1Aa7NDCu+1zGKGK4rWRQYZCeyCVtL6o8f+J\\\\\\\/j4OMmDjVJh3WGuQIUT3Gfd5ANN9qvCDx1M0ogw4gD7CxI6X3GlqF5AZAogrwuIssSLEydlTfj4B28HkYs3jt0QKB9eTCAAsBA+52onpaYEJ879kvExaYeZA0GYsdd6OhQ6NZP1lrt2zcNKWWRDB0FJiRjnFH+9SaR891FVRS1+bK+h\\\\\\\/wHcxyd8GC365S2ZUDbdu+X7ti2qCwbmSuWSGI3tBA060vlx6ZnXTB5Y4Va6qmcGpqA+zyteIO33iXQwpPdOxkPiOkTsrTEOGhOn6uiqrsdcMQxO6jU0xHfCBkq68i9QC2ClCtBA45pbn1G2mpLBDNVOEifjWE2QU3neADXl9w4N2t0YSsh+9x8E+Ko4e7PkJC50sVR7oHn\\\\\\\/Lid4FceesVn+HrOpXIMKXAj+30poGZyKx3cdBQg2Sjua6LKypLMBrJ4r+OEvBBlmZJ\\\\\\\/vyJFiMg2ICfXtmNKLtlt6U9bAZEo\\\\\\\/niueueQH9N7\\\\\\\/F4NZb9csMY9wxpM9yeyyEzrop2phEvZnANeQCiWa35JNA950PBIyvrIX0YafKx80q\\\\\\\/pLAK5fs7iH\\\\\\\/OBti2Ki+5OmDHjrDgUwNu\\\\\\\/Lvj1I1oY2DRmOFiyog74WF6mbtNWuMQind5P1\\\\\\\/bwE1+G9z0WKewGXxL8aXP1r\\\\\\\/oYXZfa6H22+aMFXl0b0nOYBwXQ\\\\\\\/RT0ZynmKs6mbh3Z7J7OB94NYXyinG+ysSYUs8jxqzUDiWv2c8UzSiABNxJb7BMFHAlDviRvGFbA2xHyv40z49\\\\\\\/jzvMALg+0TMYmktgElfZzfB7CkiJBwf5EVHuqi30lQ2W4MKueLM6gbSWwEFOwPdvgILLLIBAGDjJUkIJBFod0QUWSTKwnfpOX5Qa0Xvr9IUF77LKztjG2EcrcwoPALeQSdD6+SOoaSMbEwAH7QEK98\\\\\\\/jyohFcr2hcsoj1nfXdTgOck1LfhabwTfb8Powcbx3eXuO8OQmrU+g+ijzOe3cFpTG06SjhZqU6fQNu8lEz+\\\\\\\/ig2\\\\\\\/d7pwg5\\\\\\\/aMwrRvauy\\\\\\\/uv+50kvZJKKKY1l\\\\\\\/91n3leCMLeVoWPJ6wElKkojA5Jid7NfnXd8f2vDpBSpzIealc273CHSwpwPuoWwWl1bAOx5EwTNTQY2\\\\\\\/zUuR1OQzRRs+X3Dah07yUAkXx9Y\\\\\\\/\\\\\\\/BwhUjzitU05vbXaOZzIDPnnxgzwLTWMgGOmuZLxNPYfttFYNPiAp4CD8Io0KbqpxlBoEuYSzJPK1JWGC3Gi7rsu+GTtEwzGi+T3t8vgo7QAqgWYT+gXfXyUYcHLLC32hsXqQVG35tJRwQzoxqCrwGiRNKEEBsGqe2evA4XPCKfJ8VJVzPC8mGZEgq4QcFdh8Mohzuf\\\\\\\/B1+i4aIJvY4WH4axZSxkQgRis2UCvN4xY+l\\\\\\\/SMlWXKYJev+NCgelpa4TC6UNIbghYZrA1sn86Vx4bSjobppl2jlR55cATHHsthZ53N8GAxIY+gmLk53f3XBI9vkSIfc8VAP1gP2FOaVyNkjOxarki1HJ7+SV9NybiiBah8zF3cSHpjMH2JrzgYwDTu3X3Wc+OYezhPOnoUuf3mWCkXaajSNmAih8k08qdUzI2clzRierm8jLio8NvlckEFsKasEYC98DokcwWHruxmGSU5pYz20bJhIjJx2ep2c21qu5v1SB1FcAnyBKi+JLjlEDnwi6zCnvdvTYc1JcQSpgF67D3mFqoQe9cSOljkz0EpdMaBSq0MGdWyNZP+yfa4hEJpUIX7+PzeNryBss3hD9uLK0jHZESIpshXUJWTwRW1iMV\\\\\\\/VXa1XVrqPshtFpoZPV2xHA1IhUsN3j+V0VdXIP2pxpkuaAtqtaYfLFc1g0QDGgyUIh2oFniLI1vy99iaf0459\\\\\\\/cTuFMUWc9JH6lRU04XKk41\\\\\\\/0T140sI7Bk3zMeY\\\\\\\/D4ZNBu8\\\\\\\/ojeHhGG3RrG1EV5rCI41eOnrKyaiu4d2MryYgXCrHRnYrOI+dF6xNIe3NlyZitRP2xcVVD5qCm\\\\\\\/TOja1XWYfOovQQ6EybhRk\\\\\\\/cgXqPMUwNqvgZlxkY30v3tzOniRrmJWfl1d7W+0zSGBBH8mFFqGwHR2r1yzuhW4WRHcVj9nbvNvxhpsyude5Lk7BnxiEp1qfY\\\\\\\/eAYvoMre4ew6jIYH5S1FM\\\\\\\/dJgUVZWjib5gn9xYV2O7RaaMeM8uiGRckHsZjofXPZYb9WIDM55fLYTfesEly3AIX58lhSLgpRG7fHQmJH16d2kpq286eUFsJ\\\\\\\/V5Bv30XHKRbVLOhpf1M4rv8n0AoNimblTR9Bvz1mrg1fwf\\\\\\\/pGY53A3nTq13RsqP2FInqO9mSGORJ9j2qBNkZJU2xnYLECutKPil6OPZkn4n8+0wMt9rzsZL0nxkPjT1thtoknrCgAS3zhxnMBE4LiZIJJxQ\\\\\\\/ffVy5A7BPndQdMbtmMYO6\\\\\\\/BQvyB\\\\\\\/GLtj2T4DJJp5TvxjOfQrf\\\\\\\/G7rYGQ8A8x6gONRqtB54gC2lJ8L4dWB8MquBITgqLSEJSObrEZ9HBs7vtcM4qZhrXC8MSLp+ZTQDhy4PG8k0972QVPqop5sZPUuNp3kbdyDDXUPFyAsZnnswwCdSCN7n9K\\\\\\\/kTOjjsWaOzKB5pdxaDBYK0eY+ShouNTUcte\\\\\\\/lOgXYhR3mJd9IDXPx5RXnyaJ0A4KDNc81fuxg7PV3athSW\\\\\\\/+XH+IcVHox2Gr9EQSStXk26AXkyKH80j5gP6SruRQlEu2MlWaOkhqlGNvGH6rU7Boj6q4XcQ9cRQzaIiVpFnbYaL6Q413fs3p0H9yDnJnsC3K7ewjn7cNojKh+oKbsCfYd7Wes+5KXKOILFvDUP2yZE63qv1x49ime1iOx0hjjzk1hRc+uQqob8c25Iu5f\\\\\\\/hvMR7mcP3oItW9Si5pPZp3zaIm0na0BNky2n3RMO2vh\\\\\\\/BKr4\\\\\\\/gRslD+9eH5LQLPzppsnt62r65eTH4ptTY2b+dBTWRpzp0CL\\\\\\\/hYopONYpWNIojELnDF1ZYf2iVuMSYcDGi4s\\\\\\\/xfBRxlwODhJ1WGVZ3gfipEy2uMgIYLhjEhQXNebS0bp1h8CPlxW3XPlDlccI5EM\\\\\\\/zx+p5gdBAiHwuvJKUCM2t79KYsnndz1eZw\\\\\\\/piRK2YLkzVDwp3Y4qSbXhcnQZM+9xLgLhnokgOQQIsJlIP\\\\\\\/zBYI\\\\\\\/m4nvQwRlFr18EOvL3e4+RJxLq0zHAaCKPBhXnRv+dEaf8A3sJPLBF7sVmRxQ9ddMJcIUEYJHMS19eCvG+L1ChBjexWzyWQPn7\\\\\\\/62Ft+ksStYgBfU11gOsSRQQY+i6IbNDMVnugJfSyNt5R1R7lk0woxpEnpeHYganc7pAqSjAUnqoczq0aQFDSW0oQjsO2IzxQzzcZCqWV77\\\\\\\/vKk\\\\\\\/K9qFwJ+\\\\\\\/aeYKPDulsNqhQu\\\\\\\/N+abUiD8X9bi41eUKz9o3+\\\\\\\/ARPyNMWPhX12bOwZH1DOuxNHi0h41fxbKJJwyJZ4U4vUmBReyPD7YLSFY0IDlLCIZxheOPxaR4a98c9AOXE3bEWRv6K5sMPzodhEArA5poEOhtXAfQvP7UcJo5Q7o9Dqh7ALOSOGlypY9NzFsIgNiTUwXwbZXWxl5qRprHQ1+zVPznI3YaTR6kDQog1\\\\\\\/7xhq6kzAh7f\\\\\\\/\\\\\\\/CvpO3u6210g\\\\\\\/xYscledv9b+xFQfkVxhnxNW6EIR+KWTEOKtXEBEFxql2NSJ+n5BcUMVV32\\\\\\\/JvLDgrM\\\\\\\/LPaYHSbVzVR+hv101xuuBcCAsyLbk9D6thdF6bYRR4gibbgsuizTgmnZQykNAXxNpIszQ61htGy0ftJk2lzemvRV9mjSJcRajF1kqZls9RRx4lE2A7Aqj7LNQL\\\\\\\/y3l8ohhZf\\\\\\\/yg8GpPmJOduz4yoIR87813FGbr8JNFKPvfyy9IhjfAdZPy2hVpc4smTUlhcdO7+SGtpp0VBokBLky+oyf+AeWpjwZ8uX8A+czyCxAlN3ryoSSNenleHNV4dBHlLpHktZj0kX8wW4EIJjJxPupQqO23kUmpYh24r1OQVlyNDlIdGSxZRBJh63sArICmodIHbHthT9\\\\\\\/FhUbM+3y+PrRU8NgTFN3ulawY3ACzwZ8W2VdoGT6yM\\\\\\\/FI9M5oEWOOw67vm2OzEetW9I5lDq2fxb0Igwx9SuLlVLLJcyhuvOO3X2RTGiBmMMENFACu7bPE1fNzTZYKxhOrRlQqxJyiIh0pNKq97X7WDkzBTGK12B8GkaX4HOKyELv2SC0zFxtLE3UFTrY7M4bmuvvI1EiOrXy5WdLUiPRXn4Ocja6S7w+E8zz3+V7+6Ssj8msBJ9\\\\\\\/vmJ1zNxrzXepG+U4tPkG\\\\\\\/bQGaEsTXz+Fwti\\\\\\\/THmbhoyfL3kEsDzmct6BEFisacqMSEtPCCATQN4Nq9SCQmxXFLdS+aWdK0wOOzSCBYZLMm1XS4F2qahXcHaJ+bWpPw9lA+YEoaVQj9mDyjVHhJaRjro7WY6RYRugqajKJspEfg3kQwMW4oSereA70c1lLNrM092MWhMtf2ntR2QYQFrMYJn0JKuLy8gw8CR\\\\\\\/gQVlsmXvPCQG4R3vBRn\\\\\\\/YtoVwLIarFIYCgh4mNuLtlYGcelaMYEKX\\\\\\\/iLT0ix3+lUW4iOkrxBzYM+cGuKrGKyYd2zz2Jxxgvqobm\\\\\\\/4neZLHA1D2T\\\\\\\/jnP9PclW1+s\\\\\\\/vFNVqfjlOs5w66\\\\\\\/1Hf9wlXDEw6LANUlgMrQhDa2rkBkisbhNyQJ4L1F0hQYh6mRqLEn798Bx19Rk30B2HG+D4YUPdZ6ufvkVmM3INMfO23dWQqprhDF7E89EfPGn6O0YQdiJD1dp45Vqfw5TWmGvgQ9Cee0FAH1QseeakhvQx+YKI0WvbTS1QUdE2m6V7Pmvt705eDeQCeLr7+FvjbuGS1n2F0HLTxVBlESyXdakB6Nsne7nvDIH9303tr+MFaebhH9EJTdoawYGbzm9kfbonuTISJrjdGPkESSKmIA+7z\\\\\\\/uPlPC9RXt8omEvTmpKlNrIkxpFQuFfujV9gABow7YWOMwKsJXips56f4LROnhXjoRwrfwCNwj590w0kfbL4qkLbhtbjjHX11cR6kNZxVtRek+AYi4dT9txorkQeGTNQlqwpFyi+jEDdxmx3VprjZyFt38XOj95wA7T6ayv3kf9PP\\\\\\\/4wTUzzzp35ddBOLHndYF89ZnfRlMbjWu1UxuwA10nRQzH9kktl\\\\\\\/EjUuQdoVyGVH0xZp8n7IcorZRJsOQ8IGj9fE9tgq1Tgl1DhMQC8KH6+ujwRCEJF+QSS0ZQtYJHoBJ4dyFYMXKU6rualogfPla8NreTjT7OavbXoLIQXo+m13+rftuyXmy\\\\\\\/1LsrlZMOcI3u7F8pE8591SjzI8Y+PBtUUxoNws6XiN71C1G04ux0XmdOehYn2XcBc7kF4KaP+nrVQFjFyu\\\\\\\/6vqfrqvcDyPqyDlqqD8f6FiGcWnYGKejCYb0nA7AevociOJoQZ7+z649YOSzgr8NmNWaEtAl15OtkGX7rEaztnWW\\\\\\\/\\\\\\\/9Mz2+uNh\\\\\\\/ROzJvv1Rz9pS+\\\\\\\/kfJwO\\\\\\\/NAWzHaQ\\\\\\\/LEUJNywq2xghNmkepFyT0vEM+y6N\\\\\\\/gJ0OXry9mhwlNZastD21OR\\\\\\\/l2PLXVmKAoMjTdQkjJk296hSgTnZrreAWY3OlSGptZuGF0zNfHyjbf1LbTFN85WvQje\\\\\\\/hBtCkgIygOHBY+I2LZ64B3KBsr7+XpNHoGXQ5uwZm5vJ6TKjuOGMIULR1ugv0\\\\\\\/EZXMuUSknx+5WseOZx4nCRTUXcFYAZCRv0uy7pKWg2e53RfQPejnYmK8gl1STkHpGWb+\\\\\\\/IUTihXWKW5+nOgXFHGCnvyrwXOLnbpLmdgcKKg1C4z2N3FZNAiCzksEX0zu4Z9lhS\\\\\\\/d5JNBAWQeZWQgQHOd\\\\\\\/MafB0nJnk8TJVdL78qjAMEVnYWyCG70uhGeO+ArFuRSM+kZeVw5H2mDDJLI4l7JWRJyFV0QGCt+JqJY1DAw\\\\\\\/XOPuNfP8E3wfA1FkAxQD0y12+yEGfNamsVRGkEkqjxQRFx7WVc76Mpnsgqa5jtIUH7HdujdQZREEceaLjCO3Oc8pcEnLSO82XsrY6DFWRx4SoGG1X+Euo4iVOHElvtWZtRxuAfsxVhXEtRR0gg4oO5Sg03mXKIgWU+iiT9QkhMo+PRspuuBuDPt1ROzXe5aUjpVTMv3KyLP5k6tpGlaPwEH6SMqJPUOadqK2PDdi6DQSRcFJ9z+W\\\\\\\/0Ck\\\\\\\/CeGiTM9o4xcd51ICzEM517z9zH+PgtsCdm8\\\\\\\/ptzXf4M3ZPumgScySeGdwePvi5QN0+FV5dTw==\\\u0022,\\\u0022iv\\\u0022:\\\u00220JNSEjzcSywLfZcDdfczNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ba9202c96593cede84234608e47421c9065a905eb9f44b1dc48b320dbb10f840\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022z0WjJ+dT+tYSirX1Zv5Cap7zID1x2AYLc9Am8KwwhOCuXN+Bc6fJFObvSOFUF0TIpWk9JlTkvQUu38bM+pxYALRZt3Q46A1QMkVSr8W35uWuQmpPyb18U0qn7kn\\\\\\\/itiiXdD7LAQ8agZqWhQAeXf2mxJ0gKCXEKXuuXARMZ9Merzt1S72Ztfozvj4ugEHWw6KoIdQVPWYotivb3Ec1F+Px+rTMDifLrgJIfTzZ3VDZEi29bU+bIFEKg64CaxRHWC3c4Mi01t58pescVeLoLAd2CShp8OkpflI9gUcVw\\\\\\\/MvbKppXiavXy3szcLIfkCLqIJHvEtFbUubvLragHfbnHPGXuHcNk6dC0EiJ1Q\\\\\\\/9ypV3+B\\\\\\\/kSU+fP2oXKM97ooL4l\\\\\\\/prDDXZuh1TBf\\\\\\\/usq2sbX++40kJB1SjVF+eA4KbYxwR\\\\\\\/MXkauECOYOZ7LKrXWR4VoBfGK05jAYfCJD5dzzzceWRX8+wVHO7fyi5KxrP93SRU11ZEYLtP1fdmgHs9UMxAfWUKNQgYZCaiHwdXXimxLBjk0mMhLkGtmsxatZY5e+ox3QnJaGEV4\\\\\\\/8fxSP8i3lD7UgFhzRSqcIm7SZZaODqr38Tu8Z7wh0+82YEdXxWNnGJ2TeIOdd9WQnmXBVLYh2O9eDY2AKBdTlQobUFHsOtV7JQKf7BHNj9N1+qPqHQ\\\\\\\/PWsrahVKzsFSI0oNenngMrkLOaibuuEsnkIHqjV5I0oot0ZW62WU0rra81V3OAfAu7wsAD9e3oBSqWaxFp2KujfbDaMs0K5MoDCMmVx00sObZEKGVMjbX653v7WW145m8HRIEsZbWgQoK67jl69z3zf23jnc5qvlmaWqYZveB8nexAGxdhZvMRbAZ+ANiWIwOmZXlbL1DTnjLjjAr1IpRxrOl+GTEnaddbWvsUeOz4MBTMSFjneQbqf5MYkGrP5GTRjpHru+g0OHyod9yzwcMmnmde5uqwdyJ97wb+BxpacbaC+8mQgEW5tz8l6v3vU7CnSbwgizEa0gp1InYqUufMTL54gzTsbwIhIdzzA0VSSMydW5szDUfjy+qn8CpvrPAizy46vE+vL8L0wuoY95lpend9wudSnKbNHaiZc\\\\\\\/NlCy6+QWzjwt0iipo66ZBGSP\\\\\\\/Sy5VpKOuZej5Mo\\\\\\\/I170fdPg1mPeGk0qBiFxT7UWvrtXg5ZNdqRuiTs05pikVwy5pn4a6CAVLxeeDwrXSxhi9e3iZ7cPzyuMZEVp8cDjfBdxRN3MwKsMA4\\\\\\\/RfpOS23n6WkCFyVrNaJVIY3Fr7jtTlOE7cn6ofW1UzdR+KM58rGqlnko4HUrF8Tlf+Fa6gUAgcZiCmNP317mNgwVNMELhHij58Cqzk1WI+4kai0J2wOcQamoZ3mPrB1f+U8pLYwVOVqvdus7xXkkz1S0aCt4+24fARNmMrfyO+K6zj3ga37UN8nSxNB4jl8igxkSqPhiy1ATPSMl+2Ffd3inP8q5A\\\\\\\/\\\\\\\/uVgeFpVr4x45M1fsOY0AI0YikauCREG9ThisUmkgCL66OKz3NG+wbv\\\\\\\/PjxHSUfS73sstVnUf4RZh5XBsmCaKSnm5+ly4nRejJ3hHrdX332KcWpAWF1Ji0F8tPcSyyWoNxHBvOanNMIbCYdceMbFE0CN1EC4OyXV+Hy4e2p4jcNTFeW8zTNEkZm6u0nnSlvYbHB60WhojrRxnAEjTwuDbHDWp2J4LJuRgfux4\\\\\\\/xgAsie8wXeQbTs\\\\\\\/XMrUzVznzaZyVD9j9LvR77aOxzAATbySrGP7iOoT+JqpKtBl3j8UTG46eXvvmlEfCDeL9nvM0\\\\\\\/kOIkTuW4ba9Z4dtjn276NGZeLSilYSBIQTOduD5HTt4HuAq5Om073B\\\\\\\/2nhbwK9qnupQKG9XRRw8adyWP5u1J+hh9LM3OCDQEBSvwKY736dGsMXQ3o4Qg7iwo228+VISTl0ekgCMyzFfGF2TM82IleLb6t\\\\\\\/PP9SaCTBuUmvJCYZX17UYwr2wtWiQnAWQnYQZ4Ue2boUmPQzHHikcFhdVSNw8yNQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PdJjGwGfng1BE9V4OIJz9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002271016a72705bad56de48b26f164c4a89a336d6c8c297ca2723b99a1fdfb138be\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494987 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4223" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TgafQ82bI3Omgx3KSCkL62TU0cMeZfZHedfJ3ygRJ4E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JvXCeEB7Fvs2-9Y0CPE4z8ILyWXds6QYvTtB6OVfMaQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4209" title="Marker-Trait Associations for Enhancing Agronomic Performance, Disease Resistance, and Grain Quality in Synthetic and Bread Wheat Accessions in Western Siberia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4235" title="Machine Learning Enables High-Throughput Phenotyping for Analyses of the Genetic Architecture of Bulliform Cell Patterning in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494987" id="node494987" data-pisa="ggg;9/12/4223" data-pisa-master="ggg;g3.119.400740" data-apath="/ggg/9/12/4223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anna L. Tyler</span>, <span class="highwire-citation-author" data-delta="1">Abbas Raza</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dimitry N. Krementsov</span>, <span class="highwire-citation-author" data-delta="3">Laure K. Case</span>, <span class="highwire-citation-author" data-delta="4">Rui Huang</span>, <span class="highwire-citation-author" data-delta="5">Runlin Z. Ma</span>, <span class="highwire-citation-author" data-delta="6">Elizabeth P. Blankenhorn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cory Teuscher</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Matthew Mahoney</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4223-4233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400740 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494987" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna L. Tyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BL.%2BTyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tyler%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BL.%2BTyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anna L. Tyler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abbas Raza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abbas%2BRaza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raza%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbbas%2BRaza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitry N. Krementsov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biomedical and Health Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitry%2BN.%2BKrementsov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krementsov%20DN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADimitry%2BN.%2BKrementsov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dimitry N. Krementsov</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laure K. Case </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laure%2BK.%2BCase%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Case%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaure%2BK.%2BCase%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Runlin Z. Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Runlin%2BZ.%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20RZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARunlin%2BZ.%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elizabeth P. Blankenhorn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Immunology, Drexel University College of Medicine, Philadelphia, PA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BP.%2BBlankenhorn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blankenhorn%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BP.%2BBlankenhorn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cory Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, University of Vermont Larner College of Medicine, Burlington, VT, 05405,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cory%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACory%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cory Teuscher</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">J. Matthew Mahoney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Neurological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Computer Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BMatthew%2BMahoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mahoney%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BMatthew%2BMahoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Matthew Mahoney</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:John.M.Mahoney@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">John.M.Mahoney@uvm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/4223" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494987" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494987/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4223.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494987" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494987/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/4223.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494987" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494987/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/4223.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494987" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494987/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic mapping is a primary tool of genetics in model organisms; however, many quantitative trait loci (QTL) contain tens or hundreds of positional candidate genes. Prioritizing these genes for validation is often <em>ad hoc</em> and biased by previous findings. Here we present a technique for prioritizing positional candidates based on computationally inferred gene function. Our method uses machine learning with functional genomic networks, whose links encode functional associations among genes, to identify network-based signatures of functional association to a trait of interest. We demonstrate the method by functionally ranking positional candidates in a large locus on mouse Chr 6 (45.9 Mb to 127.8 Mb) associated with histamine hypersensitivity (Histh). Histh is characterized by systemic vascular leakage and edema in response to histamine challenge, which can lead to multiple organ failure and death. Although Histh risk is strongly influenced by genetics, little is known about its underlying molecular or genetic causes, due to genetic and physiological complexity of the trait. To dissect this complexity, we ranked genes in the <em>Histh</em> locus by predicting functional association with multiple Histh-related processes. We integrated these predictions with new single nucleotide polymorphism (SNP) association data derived from a survey of 23 inbred mouse strains and congenic mapping data. The top-ranked genes included <em>Cxcl12</em>, <em>Ret</em>, <em>Cacna1c</em>, and <em>Cntn3</em>, all of which had strong functional associations and were proximal to SNPs segregating with Histh. These results demonstrate the power of network-based computational methods to nominate highly plausible quantitative trait genes even in challenging cases involving large QTL and extreme trait complexity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-prioritization" class="hw-term hw-article-keyword hw-article-keyword-gene-prioritization" rel="nofollow">Gene prioritization</a></li><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine learning</a></li><li class="kwd"><a href="/keyword/quantitative-trait-locus" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-locus" rel="nofollow">quantitative trait locus</a></li><li class="kwd"><a href="/keyword/histamine-hypersensitivity" class="hw-term hw-article-keyword hw-article-keyword-histamine-hypersensitivity" rel="nofollow">histamine hypersensitivity</a></li><li class="kwd"><a href="/keyword/clarkson%E2%80%99s-disease" class="hw-term hw-article-keyword hw-article-keyword-clarkson&#x2019;s-disease" rel="nofollow">Clarkson’s Disease</a></li></ul><p id="p-3">Identifying causal variants within quantitative trait loci (QTL) is a central problem of genetics, but genetic linkage often prevents narrowing QTL to less than several megabases (Mb). Thus, QTL may contain hundreds of candidate genes. Instead of revealing the exact gene (or genes) responsible for trait variation, QTL mapping produces positional candidate genes. Rigorously narrowing a QTL by fine mapping with congenic strains can take years or decades, particularly in organisms like mice that have long generation times. Moreover, high-resolution congenic mapping often reveals that the overall QTL effect is due to multiple linked genes within the QTL rather than a single gene (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Parker <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yazbek <em>et al.</em> 2011</a>). Thus, positional data alone are generally insufficient to nominate candidate genes for subsequent biological follow up. To overcome the limitations of mapping data, researchers look within a QTL for plausible candidate genes. However, these selections are typically done by <em>ad hoc</em> criteria using prior knowledge or a literature search. This strategy is strongly biased toward prior knowledge and is highly error prone due to missing annotations. There is a need for rigorous and systematic strategies to distinguish among positional candidate genes for mechanistic follow up.</p><p id="p-4">We developed a novel approach to rank positional candidates based on functional association with a trait. To avoid annotation or literature bias, we use functional genomic networks (FGNs), which encode predicted functional associations among all genes in the genome. FGNs such as the Functional Networks of Tissues in Mouse (FNTM) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goya <em>et al.</em> 2015</a>) and HumanBase (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Greene <em>et al.</em> 2015</a>), are Bayesian integration networks that combine gene co-expression, protein-protein binding data, ontology annotation and other data to predict functional associations among genes. With these networks we can expand on known gene-trait associations to identify genes that were not previously associated with the trait.</p><p id="p-5">Recent studies with functional genomic networks have demonstrated their power to generate novel associations between genes and specific phenotype terms (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> 2010</a>) or biological processes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ju <em>et al.</em> 2013</a>). For example, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> (2010)</a> used a support vector machine (SVM) classifier to identify a gene network associated with bone mineralization. They predicted and validated novel associations between genes and bone mineralization phenotypes for genes that lay outside of all published QTL for bone mineralization phenotypes (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> 2010</a>). Subsequent studies using similar network-based techniques have made novel predictions of hypertension- and autism-associated genes (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Greene <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Krishnan <em>et al.</em> 2016</a>). We have expanded these methods to rank genes in a mapped QTL based on multiple putative functional terms and to integrate these rankings with genetic association <em>p</em> values from strain surveys. We generated a ranked list for all genes in the QTL that incorporated both the functional and positional scores of each candidate gene.</p><p id="p-6">Our strategy first built trait-associated gene lists from structured biological ontologies (<em>e.g.</em>, the Gene Ontology (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ashburner <em>et al.</em> 2000</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gene Ontology Consortium 2018</a>) and the Mammalian Phenotype Ontology (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Smith and Eppig 2012</a>)) and public transcriptomic data from the Gene expression Omnibus (GEO) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar <em>et al.</em> 2002</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett <em>et al.</em> 2012</a>). We then applied machine learning classifiers to the functional networks of tissues in mice (FNTM) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goya <em>et al.</em> 2015</a>) to identify network-based signatures of the trait-related gene lists. This strategy allowed us to predict gene-trait associations that were not annotated within a structured ontology, overcoming the missing annotation problem.</p><p id="p-7">We applied our approach to a large QTL associated with histamine hypersensitivity (Histh) in mice. Histh in mice is a lethal response to a histamine injection. In insensitive mice, a histamine injection produces an inflammatory response that resolves without further treatment. Mice with the Histh response develop excitation and ear blanching, followed by progressive respiratory distress, vasodilation, anaphylactic shock, and death (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vaz <em>et al.</em> 1977</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2014</a>). Histh can be induced in a subset of mouse strains by sensitization with Complete Freund’s Adjuvant (CFA). Histh also develops spontaneously in SJL/J animals older than six months of age.</p><p id="p-8">We previously mapped Histh to a locus on Chr 6 (45.9 Mb to 127.8 Mb; the <em>Histh</em> locus), which was confirmed using a congenic line (B10.S-<em>Histh<sup>SJL</sup></em>) (Raza <em>et al.</em> In Press). Because of the large size of this locus, additional information is required to identify causal variants. To narrow down candidates, we integrated novel genetic association data from interval-specific congenic recombinant lines (ISCRLs) and an inbred strain survey with our network-based functional predictions of Histh-related genes. By augmenting positional data with functional predictions, we dramatically reduced the candidate gene list to a tractable set of high-quality candidates that are implicated in Histh-related processes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">As a supplement to the computational portion of the methods section, this paper includes an executable workflow (See Data Availability). An outline of the computational workflow is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The workflow includes all files and parameters required to recreate the computational portions of this study.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow Overview. The workflow is broken into blocks by color, each with a bolded title. Each block shows how data (blue rectangles) were operated on (gray rectangles) to achieve results (green rectangles). Arrows show the general flow of work and dependence (and independence) of individual analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Workflow Overview. The workflow is broken into blocks by color, each with a bolded title. Each block shows how data (blue rectangles) were operated on (gray rectangles) to achieve results (green rectangles). Arrows show the general flow of work and dependence (and independence) of individual analyses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F1.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/4223/F1.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Workflow Overview. The workflow is broken into blocks by color, each with a bolded title. Each block shows how data (blue rectangles) were operated on (gray rectangles) to achieve results (green rectangles). Arrows show the general flow of work and dependence (and independence) of individual analyses.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-11">A total of 23 mouse strains (129X1/SvJ, A/J, AKR/J, B10.S-<em>H</em>2s/SgMcdJ (B10.S), BALB/cJ, BPL/1J, BPN/3J, C3H/HeJ, C57BL/6J, C57BL/10J, CBA/J, CZECHII/EiJ, DBA/1J, DBA/2J, FVB/NJ, JF1/MsJ, MOLF/EiJ, MRL/MpJ, NOD/ShiLtJ, NU/J, PWD/PhJ, PWK/PhJ, SJL/J and SWR/J were purchased from the Jackson Laboratory (Bar Harbor, ME). All mice, including B10.S-<em>Histh</em><sup><em>SJL</em></sup> and B10.S-<em>Histh</em><sup><em>SJL</em></sup> ISRC lines, were generated and maintained under specific pathogen-free conditions in the vivarium of the Given Medical Building at the University of Vermont according to National Institutes of Health guidelines. All animal studies were approved by the Institutional Animal Care and Use Committee of the University of Vermont.</p></div><div id="sec-3" class="subsection"><h3>Histh Phenotyping</h3><p id="p-12">On day 0 mice were injected at two s.q. sites with complete Freund’s adjuvant (CFA) (Sigma-Aldrich, St. Louis, MO) supplemented with 200 <em>μ</em>g of <em>Mycobacterium tuberculosis</em> H37Ra (Difco Laboratories, Detroit, MI). On D30 histamine hypersensitivity was determined by i.v. injection of histamine (mg/kg dry weight free base) in phosphate buffered saline (PBS). Deaths were recorded at 30 min post injection and the data are reported as the number of animals dead over the number of animals studied. Significance of observed differences was determined by Chi-square with <em>p</em>-values &lt;0.05 significant.</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and genotyping</h3><p id="p-13">DNA was isolated from mouse tail clippings as previously described (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sudweeks <em>et al.</em> 1993</a>). Briefly, individual tail clippings were incubated with <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-1.gif"/></noscript></span></span>L cell lysis buffer (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-2.gif"/></noscript></span></span> g/mL proteinase K, 100 mM NaCl, 10mM Tris-HCl (pH 8.3), 10 mM EDTA, 100mM KCl, 0.50% SDS) overnight at <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-3.gif"/></noscript></span></span>. The next day, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-4.gif"/></noscript></span></span>L of 6M NaCl were added followed by centrifugation for 10 min at <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-5.gif"/></noscript></span></span>. The supernatant layer was transferred to a fresh tube containing <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-6.gif"/></noscript></span></span>L of isopropanol. After centrifuging for two minutes, the supernatant was discarded, and pellet washed with 70% ethanol. After a final two-minute centrifugation, the supernatant was discarded, and DNA was air dried and resuspended in <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-7.gif"/></noscript></span></span>L TE.</p><div id="sec-5" class="subsection"><h4>Genotyping:</h4><p id="p-14">Genotyping was performed using either microsatellite markers in a standard PCR reaction or sequence-specific SNP primers in a phototyping reaction. Polymorphic microsatellites were selected to have a minimum polymorphism of 8bp for optimal identification by agarose gel electrophoresis. Briefly, primers were synthesized by IDT-DNA (Coralville, IA) and diluted to a concentration of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-8.gif"/></noscript></span></span>M. PCR amplification was performed using Promega GoTaq. The cycling conditions included a two-minute initial denaturation step at 94° followed by 35 cycles of 94° for 30 sec, 55° for 30 sec and 72° for 30 sec followed by a final extension step at 72° for five minutes. Amplicons were subjected to 2% agarose gel electrophoresis and visualized by ethidium bromide and UV light.</p></div><div id="sec-6" class="subsection"><h4>Phototyping:</h4><p id="p-15">Genotyping was performed using sequence-specific primers that differ only at the 3’ nucleotide corresponding to each allele of the identified SNP (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bunce <em>et al.</em> 1995</a>). Each primer set was designed using Primer3 to have a Tm of 58-60∘C, synthesized by IDT-DNA (Coralville, IA), and used at a concentration of 100<em>μ</em>M (primer sequences are available in Supplemental File 1). PCR reactions were subjected to multistage (high, medium and low stringency) cycling conditions as described in Supplemental File 2 and if found to be necessary, the cycle conditions at each stage were adjusted to accommodate the optimal annealing temperature. Amplicons were electrophoresed with <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-9.gif"/></noscript></span></span>L Orange G loading buffer on a 1.5% agarose gel stained with ethidium bromide and visualized by UV light. The presence of a SNP-specific allele was scored by observing an amplicon of the expected size in either reaction. Cycling conditions are available in Supplemental File 2.</p></div></div><div id="sec-7" class="subsection"><h3>Generation of Histh congenic lines and GigaMUGA</h3><p id="p-16">B10.S-<em>Histh<sup>SJL</sup></em> ISRC lines were generated by identifying recombinant haplotypes across the <em>Histh</em> interval among (B10.S-<em>Histh<sup>SJL</sup></em> × B10.S) × B10.S BC1 mice and then fixed as homozygous lines (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). To identify potential contaminating background loci segregating among the strains and to further refine the recombination break points of each line, the lines were further genotyped using GigaMUGA arrays (143,259 markers) by the commercial service of Neogen/Geneseek (Lincoln, NE).</p></div><div id="sec-8" class="subsection"><h3>Targeted genetic association testing</h3><p id="p-17">We retrieved genotype data (both coding and non-coding) of the 23 mouse strains from the databases at the Sanger Institute (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Keane <em>et al.</em> 2011</a>) (<a href="https://www.sanger.ac.uk/science/data/mouse-genomes-project">https://www.sanger.ac.uk/science/data/mouse-genomes-project</a>) and The Jackson Laboratory (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bogue <em>et al.</em> 2017</a>) (<a href="https://phenome.jax.org/">https://phenome.jax.org/</a>). The lack of representation of wild-derived strains <em>e.g.</em>, MOLF/EiJ and others, in these databases was compensated for by genotyping using Illumina Hiseq 2500 platform <a href="https://www.illumina.com/">https://www.illumina.com/</a>. For detailed methods see Supplemental File 3. All these data sources were collated to generate genotype information for a total of 15,302 SNPs across the <em>Histh</em> locus (45-128 Mbp, Supplemental File 4).</p><p id="p-18">To calculate associations between genetic polymorphisms and Histh, we used efficient mixed-model association (EMMA) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kang <em>et al.</em> 2008</a>). This method treats genetic relatedness as a random variable in a linear mixed model to account for population structure, thereby reducing false associations between SNPs and the measured trait. To calculate the kinship matrix, we compiled a set of 470,365 SNPs across all strains from the Mouse Genome Informatics database (Mouse Genome Informatics Mouse Genome Informatics Web Site). We removed the SNPs in the congenic interval from the kinship estimation, and generated the kinship matrix with the remaining 455,068 SNPs. We used the likelihood ratio test function (emma.ML.LRT) to generate <em>p</em> values. Significance was defined with a Bonferroni correction (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-10.gif"/></noscript></span></span>). Genomic coordinates included for each SNP using the latest mouse genome build GRCm38.p5/mm10.</p></div><div id="sec-9" class="subsection"><h3>Trait-related gene sets</h3><p id="p-19">The positional candidate genes were ranked based on their predicted association with seven functional terms related to the Histh phenotype: “aging”, “<em>Mycobacterium tuberculosis</em>”, “cardiac”, “G-protein coupled receptor”, “histamine”, “inflammation”, “type I hypersensitivity”, and “vascular permeability.” We used Gene Weaver (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baker <em>et al.</em> 2012</a>) to generate gene sets associated with each term. To do this, we entered each term into the Gene Weaver homepage (<a href="https://geneweaver.org">https://geneweaver.org</a>). We restricted the search to human, rat, and mouse genes, and to curated lists only. Mouse orthologs for each gene were retrieved using the batch query tool in MGI (<a href="http://www.informatics.jax.org/batch_data.shtml">http://www.informatics.jax.org/batch_data.shtml</a>). In addition, we used Gene Expression Omnibus (GEO) and PubMed to retrieve expression data sets for each phenotype term. We removed all positional candidates from the gene lists used for training such that no true-positive positional candidates were used for training. The gene lists used are available as a set of zipped text files in Supplemental File 5.</p></div><div id="sec-10" class="subsection"><h3>FNTM network</h3><p id="p-20">We trained support vector machines (SVMs) to classify genes in each gene list using features derived from FNTM (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Goya <em>et al.</em> 2015</a>). In this functional genomic network, genes are nodes, and the edge weights between them are continuous values between 0 and 1 predicting the degree to which each pair of genes is functionally related. Larger values indicate higher predicted functional relatedness. Functional relatedness in this network was predicted through Bayesian integration of data sets from multiple sources, including gene expression, protein-protein interaction data, and annotation to GO terms (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Goya <em>et al.</em> 2015</a>). We downloaded the top edges of the mouse network on January 15, 2018 from <a href="http://fntm.princeton.edu">http://fntm.princeton.edu</a>.</p></div><div id="sec-11" class="subsection"><h3>Clustering gene sets</h3><p id="p-21"><a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> (2010)</a> observed that support vector machines trained on 200 to 300 genes yielded the best classification accuracy. Two of our gene lists had fewer than 100 genes. For all lists over 400 genes, we reduced the size of our training sets by clustering each term gene set into modules using the fast greedy (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Newman 2004</a>) algorithm in the R/igraph package (Csardi 2006). We applied the fast greedy algorithm iteratively until all modules comprised fewer than 400 genes. Using a maximum modules size of 300 overly fragmented the networks yielding many modules with fewer than 100 genes.</p></div><div id="sec-12" class="subsection"><h3>Machine learning</h3><p id="p-22">To classify positional candidates as belonging to a functional module, we trained SVMs using the connection weights in the FNTM network as features, as described in <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> (2010)</a>. Briefly, an annotated set of genes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, blue nodes) is used as a set of known positives for the corresponding functional module. Other genes in this module are expected to be strongly functionally connected to these known positives, <em>i.e.</em>, have a high probability of functionally interacting with known positives. Each gene, therefore, is represented as a <em>feature vector</em> of connection weights to the known positives, which can be visualized as a sub-matrix of the <em>adjacency matrix</em> of the network (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Correspondingly, the rows of this matrix are labeled as either known positive or not (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, blue dots <em>vs.</em> gray dots). We used the e1071 package in R (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meyer <em>et al.</em> 2018</a>) to train SVMs to distinguish the known positive genes from an equal-sized set of genes selected at random from outside the known positive list using the network-based feature vectors (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). The trained model can then annotate novel genes as belonging to the functional module by classifying any gene in the genome (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, green bordered nodes).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Network-based machine learning for functionally annotating genes. A Known-positive genes annotated to a functional term (blue nodes) are typically densely interconnected in a functional network. B The adjacency matrix of a network is a tabular representation of the connectivity structure of the network in which each row/column corresponds to a node of the network, and connected pairs of nodes have non-zero values in the corresponding cell of the matrix. Note that in general the connections are weighted, but for display we are only showing present/absent links (white/black cells). The connections from every gene in the genome to the known positives form a sub-matrix of the adjacency matrix called the feature matrix (vertical red lines), whose rows are the feature vectors for each gene. C Using the network-based feature vectors for each gene, we train SVMs to distinguish known positives (blue dots) from random genes in the genome (gray dots) to identify the full sub-network corresponding to the true positive genes (green bordered dots and dotted red lines in panels A,B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Network-based machine learning for functionally annotating genes. A Known-positive genes annotated to a functional term (blue nodes) are typically densely interconnected in a functional network. B The adjacency matrix of a network is a tabular representation of the connectivity structure of the network in which each row/column corresponds to a node of the network, and connected pairs of nodes have non-zero values in the corresponding cell of the matrix. Note that in general the connections are weighted, but for display we are only showing present/absent links (white/black cells). The connections from every gene in the genome to the known positives form a sub-matrix of the adjacency matrix called the feature matrix (vertical red lines), whose rows are the feature vectors for each gene. C Using the network-based feature vectors for each gene, we train SVMs to distinguish known positives (blue dots) from random genes in the genome (gray dots) to identify the full sub-network corresponding to the true positive genes (green bordered dots and dotted red lines in panels A,B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F2.medium.gif" width="440" height="167"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/4223/F2.medium.gif" width="440" height="167"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495113" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Network-based machine learning for functionally annotating genes. A Known-positive genes annotated to a functional term (blue nodes) are typically densely interconnected in a functional network. B The adjacency matrix of a network is a tabular representation of the connectivity structure of the network in which each row/column corresponds to a node of the network, and connected pairs of nodes have non-zero values in the corresponding cell of the matrix. Note that in general the connections are weighted, but for display we are only showing present/absent links (white/black cells). The connections from every gene in the genome to the known positives form a sub-matrix of the adjacency matrix called the feature matrix (vertical red lines), whose rows are the feature vectors for each gene. C Using the network-based feature vectors for each gene, we train SVMs to distinguish known positives (blue dots) from random genes in the genome (gray dots) to identify the full sub-network corresponding to the true positive genes (green bordered dots and dotted red lines in panels A,B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">We trained 100 SVMs on each module selecting a new set of random genes for each run. We used a linear kernel and 10-fold cross-validation for each SVM. We trained each SVM over a series of cost parameters. We started with the sequence <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-11.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-12.gif"/></noscript></span></span> by factors of 10, and iteratively narrowed the range of cost parameters until we found a series of eight cost parameters that maximized the accuracy of the SVM (see Workflow).</p><p id="p-25">We calculated the area under the ROC curves (AUC) over all runs in the following way: For a sequence ranging from the minimum SVM score to the maximum SVM score, we quantified all true positives (TP), true negative (TN), false positives (FP) and false negatives (FN). The TP genes in this case were those genes from the known positives that were correctly classified as being in the module (above the SVM score cutoff). TN genes in this case were those genes outside the module that were correctly classified as being outside the module (below the SVM score cutoff). We calculated the AUC across the average curve for all 100 SVMs for each module.</p></div><div id="sec-13" class="subsection"><h3>Positional Candidate Scoring</h3><p id="p-26">We used the trained SVMs to score each positional candidate gene in the <em>Histh</em> locus. The score for each gene gave an estimate of how functionally related each gene was to each module based on its connection weights to the known module genes in the FNTM mouse network. SVMs classify inputs into two classes, and inputs receive either a positive or a negative score depending on which class the SVM places them. The larger the magnitude of this score, the more confident the classification. Genes with large positive scores were predicted by the SVMs to interact functionally with the genes in the module, while genes with large negative scores were predicted to <em>not</em> functionally interact with the module genes. The scale of these SVM scores can vary widely between models. Thus, to compare SVM scores across different trained models, we calculated a false positive rate (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-13.gif"/></noscript></span></span>) for each gene. The FPR lies between 0 and 1 and can be compared across different models. For each gene’s SVM score we calculated the number of true positives (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-14.gif"/></noscript></span></span>), true negatives (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-15.gif"/></noscript></span></span>), false positives (<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-16.gif"/></noscript></span></span>) and false negatives (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-17.gif"/></noscript></span></span>) classified by the SVM. The <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-18.gif"/></noscript></span></span> for a given SVM score was calculated as <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-19.gif"/></noscript></span></span>.</p><p id="p-27">The final functional score for each gene was the <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-20.gif"/></noscript></span></span> across all modules. This meant that genes with a high functional score for a single module, but low functional scores for other modules, received higher overall scores than genes with moderately high scores across all modules.</p></div><div id="sec-14" class="subsection"><h3>Combined Gene Score</h3><p id="p-28">High-quality candidate genes in the locus should not only be functionally related to the trait of interest, but should also segregate with the trait of interest. We thus defined a combined gene score (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-21.gif"/></noscript></span></span>) that combined these two aspects of the analysis:</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-22.gif"/></noscript></span></span><p id="p-29">where the denominators of the two terms on the right hand side are the maximum values of <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-23.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-24.gif"/></noscript></span></span> over all positional candidates in <em>Histh</em>, respectively, which normalizes the functional and positional scores to be comparable to each other. EMMA <em>p</em> values for SNPs were assigned to the nearest gene within 1 megabase using the R package biomaRt (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Durinck <em>et al.</em> 2005</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2009</a>) (Supplemental Table 1). Genes for which more than one SNP was assigned were given the maximum <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-25.gif"/></noscript></span></span> across all SNPs associated with that gene. The full matrix of combined scores across all gene sets is in Supplemental Table 2. The rows of this matrix are sorted by the maximum gene score across all gene lists.</p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-30">A reproducible workflow in R markdown is available on GitHub (<a href="https://github.com/MahoneyLab/HhsFunctionalRankings">https://github.com/MahoneyLab/HhsFunctionalRankings</a>). This workflow contains all code required to reproduce the figures and results presented in this manuscript.</p><p id="p-31">The data used as input for the workflow, as well as intermediate and final results, are available on figshare (<a href="https://figshare.com/articles/Data_required_to_run_HhsFunctionalRankings_workflow/8205356">https://figshare.com/articles/Data_required_to_run_HhsFunctionalRankings_workflow/8205356</a>). The data set is called under HhsFunctionalRankings. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9989117">https://doi.org/10.25387/g3.9989117</a>.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>Generation of interval specific recombinant congenic lines (ISRCL) across the Histh locus</h3><p id="p-32">In prior work, we mapped the genetic locus regulating susceptibility to age- and/or inflammation (CFA)-dependent sensitivity to histamine on Chr 6 in SJL/J mice (Raza <em>et al.</em> In Press). The B10.S-<em>Histh<sup>SJL</sup></em> congenic mice exhibited Histh and carry a large <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-26.gif"/></noscript></span></span> Mb region of SJL/J between 45.9 Mb to 127.8 Mb on the resistant B10.S background (Raza <em>et al.</em> In Press) (MGI:6360897). This large QTL includes 628 protein coding genes. To narrow this region, we generated five ISRCLs using B10.S-<em>Histh<sup>SJL</sup></em> x B10.S backcross mice and assessed their susceptibility to Histh (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Under an additive model, these data suggest that <em>Histh</em> is composed of four sub-QTL which we have designated <em>Histh1</em> (MGI: 6362992), <em>Histh2</em> (MGI: 6362994), <em>Histh3</em> (MGI: 6362996), and <em>Histh4</em> (MGI: 6362997), each contributing 17%, 19%, 14% and 10%, respectively, to the overall penetrance. Importantly, for each sub-QTL this makes positional candidate gene identification using interactive high-resolution congenic mapping impractical.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interval specific recombinant congenic line (ISRCL) mapping identified four linked QTL controlling Histh. ISRCLs were injected (D0) with CFA and subsequently challenged (D30) by i.v. injection of 75 mg/kg histamine to determine histamine hypersensitivity. Deaths were recorded at 30 min post injection and the data are reported as the number of animals dead over the number of animals studied. Significance of observed differences was determined by a  test with p-values  considered significant." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Interval specific recombinant congenic line (ISRCL) mapping identified four linked QTL controlling Histh. ISRCLs were injected (D0) with CFA and subsequently challenged (D30) by i.v. injection of 75 mg/kg histamine to determine histamine hypersensitivity. Deaths were recorded at 30 min post injection and the data are reported as the number of animals dead over the number of animals studied. Significance of observed differences was determined by a &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; test with &lt;em&gt;p&lt;/em&gt;-values &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; considered significant.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F3.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/4223/F3.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494917" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Interval specific recombinant congenic line (ISRCL) mapping identified four linked QTL controlling Histh. ISRCLs were injected (D0) with CFA and subsequently challenged (D30) by i.v. injection of 75 mg/kg histamine to determine histamine hypersensitivity. Deaths were recorded at 30 min post injection and the data are reported as the number of animals dead over the number of animals studied. Significance of observed differences was determined by a <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F3/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F3/embed/mml-math-27.gif"/></noscript></span></span> test with <em>p</em>-values <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F3/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F3/embed/mml-math-28.gif"/></noscript></span></span> considered significant.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Inbred strain survey of Histh</h3><p id="p-34">To investigate whether the Histh phenotype is unique to SJL, we assessed histamine responses for 23 inbred mouse strains (including SJL/J and B10.S; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These strains were chosen using haplotype structure across the Histh interval to identify additional mouse strains that are likely to share a susceptible <em>Histh</em> allele (data not shown). 129X1/SvJ, ALR/LtJ, BPN/3J, FVB/NJ, NOD/ShiLtJ, NU/J, SJL/BmJ and SWR/J mice were identified as having similar haplotype structure as SJL/J at the <em>Histh</em> locus. ALR/LtJ and SJL/BmJ mice required embryo recovery and were therefore not included. Histh phenotyping identified FVB/NJ, SWR/J, and NU/J mice as Histh-susceptible, whereas 129X1/SvJ, NOD/ShiLtJ, and BPN/3J were resistant. Taken together with our earlier data, these results indicate that Histh susceptibility segregates among a unique subset of SJL/J-related strains (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Petkov <em>et al.</em> 2004</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494966/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494966/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">A survey Histh phenotypes across 23 inbred mouse strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Targeted genetic association analysis for Histh</h3><p id="p-36">Our result from previous linkage analysis (Raza <em>et al.</em> In Press) and congenic mapping localized <em>Histh</em> to an <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-29.gif"/></noscript></span></span> Mb region on Chr 6 between 45.9 Mb to 127.8 Mb. Given that Histh-susceptibility is restricted to a unique subset of inbred strains, particularly the closely related SJL/J, FVB/NJ, and SWR/J, we performed a targeted association analysis between SNPs in the <em>Histh</em> locus across all 23 inbred strains (<em>cf</em>. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benson <em>et al.</em> (2017)</a>).</p><p id="p-37">We tested the association of 15,302 SNPs across the <em>Histh</em> locus using efficient mixed-model association (EMMA) (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kang <em>et al.</em> 2008</a>). A total of 84 SNPs in 23 genes showed significant associations (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-30.gif"/></noscript></span></span>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, Supplemental Table 3). The majority of the significant hits were intronic (71%), non-coding (12%), intergenic (4%) or regulatory (5%) variants. Interestingly, there was overlap between three of the four <em>Histh</em> sub-QTL (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) and SNP-association peaks.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Targeted genetic association analysis for Histh. Negative log-transformed p values of SNP associations with Histh. Genomic coordinates (mm10 Mbp) of each SNP are shown along the x-axis. Each circle denotes a single SNP. Gene names are included for SNPs that crossed p-value threshold of  shown with a red dotted line. The location of Histh sub-QTL are shown at the top of the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Targeted genetic association analysis for Histh. Negative log-transformed &lt;em&gt;p&lt;/em&gt; values of SNP associations with Histh. Genomic coordinates (mm10 Mbp) of each SNP are shown along the &lt;em&gt;x&lt;/em&gt;-axis. Each circle denotes a single SNP. Gene names are included for SNPs that crossed &lt;em&gt;p&lt;/em&gt;-value threshold of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; shown with a red dotted line. The location of Histh sub-QTL are shown at the top of the figure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F4.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/12/4223/F4.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495027" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Targeted genetic association analysis for Histh. Negative log-transformed <em>p</em> values of SNP associations with Histh. Genomic coordinates (mm10 Mbp) of each SNP are shown along the <em>x</em>-axis. Each circle denotes a single SNP. Gene names are included for SNPs that crossed <em>p</em>-value threshold of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F4/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F4/embed/mml-math-31.gif"/></noscript></span></span> shown with a red dotted line. The location of Histh sub-QTL are shown at the top of the figure.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Network-based prediction of Histh-associated genes</h3><p id="p-39">To predict functional candidates among the positional candidates in the <em>Histh</em> locus, we delineated a list of Histh-associated biological processes and trained machine learning classifiers to identify sub-networks of functional genomic networks associated with each of these processes. An overview of our workflow is in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. We first defined gene sets that were related to seven terms that are functionally related to the Histh phenotype.</p><p>The terms and their justifications are as follows:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-41"><em>Type I hypersensitivity/Anaphylaxis</em>: The death response following systemic histamine challenge exhibits symptoms of type I hypersensitivity/anaphylaxis including respiratory distress, vasodilation, and anaphylactic shock (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Vaz <em>et al.</em> 1977</a>).</p></li><li id="list-item-2"><p id="p-42"><em>Cardiac</em>: There is evidence suggesting that anaphylactic shock in mice is associated with decreased cardiac output, rather than solely a function of systemic vasodilation (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2014</a>).</p></li><li id="list-item-3"><p id="p-43"><em>Histamine</em>: Histh is elicited by a systemic histamine challenge (Raza <em>et al.</em> In Press).</p></li><li id="list-item-4"><p id="p-44"><em>G-protein coupled receptor</em>: Histamine receptor H<sub>1</sub> (<em>Hrh1</em>) signaling is required for the Histh phenotype, and all histamine receptors belong to the family of G-protein coupled receptors (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hill <em>et al.</em> 1997</a>).</p></li><li id="list-item-5"><p id="p-45"><em>Aging</em>: Spontaneous Histh develops after six months of age in sensitive mouse strains (Raza <em>et al.</em> In Press).</p></li><li id="list-item-6"><p id="p-46"><em>Inflammation</em>: Treatment with pro-inflammatory CFA induces Histh in sensitive mouse strains.</p></li><li id="list-item-7"><p id="p-47"><em>Tuberculosis</em>: Histh is induced in some mouse strains by CFA, which contains inactivated <em>Mycobacterium tuberculosis</em> (Raza <em>et al.</em> In Press).</p></li><li id="list-item-8"><p id="p-48"><em>Vascular permeability</em>: The Histh response includes vascular leakage in skin and skeletal muscles as assessed by Miles’assay (Raza <em>et al.</em> In Press).</p></li></ul><p>We used Gene Weaver, the Gene Expression Omnibus (GEO), and PubMed to retrieve gene sets associated with each of these terms (see Materials and Methods). The gene sets ranged in size from 651 to 1466 genes. Because <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Guan <em>et al.</em> (2010)</a> found that SVMs trained on gene sets with around 300 genes performed best for network-based functional prediction, we clustered large gene sets into modules of approximately 300 genes and analyzed each module separately (see Materials and Methods). Supplemental Table 4 shows the number of genes in each module, as well as the top five enrichment terms for each using the R package gProfileR (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Reimand <em>et al.</em> 2018</a>). Multiple members of these gene sets are encoded in the <em>Histh</em> locus. For example, <em>Hrh1</em> was a member of the Anaphylaxis gene set. To reduce bias in classification, we removed all such genes from each gene set before SVM training. We then trained an ensemble of 100 SVMs on each module gene set. We calculated ROC curves for each model to quantify the ability of each set of SVMs to distinguish genes inside the module gene set from all genes outside the module gene set. AUCs ranged from 0.9 to 0.975 indicating that the SVMs were able to classify the genes in each list robustly. In other words, each gene set used to define a putative Histh-related process forms a distinct subnetwork of the full functional genomic network. We then applied the trained SVM models to the positional candidate genes in the <em>Histh</em> locus. By classifying each positional candidate, we can identify genes that are likely to be functionally associated with each module gene set. For example, for the Anaphylaxis module gene set, the histamine receptor <em>Hrh1</em> received a positive score indicating that the SVMs predicted that it belonged to the Anaphylaxis gene set despite its absence from the training set. This example provides a positive control and shows that the SVMs identify biologically relevant patterns in the functional genomic network. In addition to the SVM score, we calculated a false positive rate (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-32.gif"/></noscript></span></span>) for each gene (see Materials and Methods). Low <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-33.gif"/></noscript></span></span>s indicate high confidence in the classification. The details of this analysis are described in an executable workflow as a companion to this paper (see Data Availability).</p></div><div id="sec-21" class="subsection"><h3>Integration of functional enrichment with genetic association</h3><p id="p-49">Genes that are predicted to be highly functionally related to the trait may not have functionally variant alleles in the study population, and may therefore be unlikely to drive the observed strain differences in Histh. To identify genes that were likely to have functionally relevant polymorphisms, we integrated functional scores with SNP association <em>p</em> values to focus only on those candidates that satisfied both criteria. By plotting the maximum functional score for a gene, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-34.gif"/></noscript></span></span> <em>vs.</em> the <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-35.gif"/></noscript></span></span> (normalized to the max values; see Materials and Methods), we can identify genes that were predicted to be both highly functionally related to Histh phenotype and likely to have functional polymorphisms that segregated with Histh susceptibility (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The blue line in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> traces along the Pareto front of the gene set in this space. For any gene on this line, finding a gene with a stronger functional association means finding a gene with lower SNP <em>p</em> value, and <em>vice versa</em>. The genes near the Pareto front have either segregating polymorphisms or are predicted to be functionally related to Histh, or both. All such genes are potentially good candidates for experimental follow-up.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two axes of gene scoring. Gene names are plotted by their  on the x-axis and the  on the y-axis. Both scores were scaled by their maximum value for better comparison. Genes farther to the right were associated with SNPs that segregated with Histh. Genes higher up on the y-axis have stronger functional association with gene modules. The blue line marks the Pareto front. Genes on this line maximize the two scores and are the best candidates based on the combination of both scores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Two axes of gene scoring. Gene names are plotted by their &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; on the &lt;em&gt;x&lt;/em&gt;-axis and the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; on the &lt;em&gt;y&lt;/em&gt;-axis. Both scores were scaled by their maximum value for better comparison. Genes farther to the right were associated with SNPs that segregated with Histh. Genes higher up on the &lt;em&gt;y&lt;/em&gt;-axis have stronger functional association with gene modules. The blue line marks the Pareto front. Genes on this line maximize the two scores and are the best candidates based on the combination of both scores.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F5.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/12/4223/F5.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494946" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Two axes of gene scoring. Gene names are plotted by their <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F5/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F5/embed/mml-math-36.gif"/></noscript></span></span> on the <em>x</em>-axis and the <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F5/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F5/embed/mml-math-37.gif"/></noscript></span></span> on the <em>y</em>-axis. Both scores were scaled by their maximum value for better comparison. Genes farther to the right were associated with SNPs that segregated with Histh. Genes higher up on the <em>y</em>-axis have stronger functional association with gene modules. The blue line marks the Pareto front. Genes on this line maximize the two scores and are the best candidates based on the combination of both scores.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">To rank the candidates with a single score, we defined a final gene score (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-38.gif"/></noscript></span></span>) for each gene, which is the sum of the (normalized) <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-39.gif"/></noscript></span></span> and the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-40.gif"/></noscript></span></span> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This score prioritizes candidates in the upper right quadrant with simultaneously high positional and functional scores. The genes in the upper right quadrant—<em>Cxcl12</em>, <em>Ret</em> and <em>Cacna1c</em>—had near-maximal scores along both axes and were therefore ranked as the best candidates for follow-up. The full table of gene scores by module can be seen in Supplemental Table 2.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Final gene scores. Gene functional values were combined with SNP associations to assign each gene a final gene score (). Higher gene scores indicate better candidates. A Heat map showing the final score of each of the top 20 ranked genes for each gene module. To aid visualization of the strongest candidates, asterisks in each cell indicate where candidate genes were associated with a module with an . B The top panel shows individual SNPs plotted at their genomic location (x-axis) and their  (y-axis). All SNPs with nominally significant p value ( are plotted. The horizontal line indicates the Bonferroni corrected significance cutoff (). The four sub-QTL are demarcated by background color and are labeled at the top of the figure. The bottom panel shows genes plotted at their genomic location (x-axis) and their final gene score () (y-axis) to demonstrate how the final ranked genes align with the SNP association data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1816083752" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Final gene scores. Gene functional values were combined with SNP associations to assign each gene a final gene score (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-41&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Higher gene scores indicate better candidates. A Heat map showing the final score of each of the top 20 ranked genes for each gene module. To aid visualization of the strongest candidates, asterisks in each cell indicate where candidate genes were associated with a module with an &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. B The top panel shows individual SNPs plotted at their genomic location (&lt;em&gt;x&lt;/em&gt;-axis) and their &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (&lt;em&gt;y&lt;/em&gt;-axis). All SNPs with nominally significant &lt;em&gt;p&lt;/em&gt; value (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are plotted. The horizontal line indicates the Bonferroni corrected significance cutoff (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The four sub-QTL are demarcated by background color and are labeled at the top of the figure. The bottom panel shows genes plotted at their genomic location (&lt;em&gt;x&lt;/em&gt;-axis) and their final gene score (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) (&lt;em&gt;y&lt;/em&gt;-axis) to demonstrate how the final ranked genes align with the SNP association data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/4223/F6.medium.gif" width="268" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/12/4223/F6.medium.gif" width="268" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/4223/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495033" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-52" class="first-child">Final gene scores. Gene functional values were combined with SNP associations to assign each gene a final gene score (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-41.gif"/></noscript></span></span>). Higher gene scores indicate better candidates. A Heat map showing the final score of each of the top 20 ranked genes for each gene module. To aid visualization of the strongest candidates, asterisks in each cell indicate where candidate genes were associated with a module with an <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-42.gif"/></noscript></span></span>. B The top panel shows individual SNPs plotted at their genomic location (<em>x</em>-axis) and their <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-43.gif"/></noscript></span></span> (<em>y</em>-axis). All SNPs with nominally significant <em>p</em> value (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-44.gif"/></noscript></span></span> are plotted. The horizontal line indicates the Bonferroni corrected significance cutoff (<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-45.gif"/></noscript></span></span>). The four sub-QTL are demarcated by background color and are labeled at the top of the figure. The bottom panel shows genes plotted at their genomic location (<em>x</em>-axis) and their final gene score (<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/F6/embed/mml-math-46.gif"/></noscript></span></span>) (<em>y</em>-axis) to demonstrate how the final ranked genes align with the SNP association data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">In addition to identifying the top-ranked gene over the full <em>Histh</em> locus, we identified a top-ranked gene for each sub-QTL identified through congenic mapping. <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a> shows the functional associations across all modules of the top 20 genes ordered by final gene score (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12/4223/embed/mml-math-47.gif"/></noscript></span></span>). The full matrix of scores for all ranked genes can be found in Supplemental Table 2.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-54">In this analysis, we identified a small set of positional candidate genes in a large locus by combining SNP associations with predicted functional associations to Histh. SNP association tests resulted in groups of SNPs with many significant and indistinguishable <em>p</em> values. <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> shows a group of genes in the lower right corner that have maximally significant <em>p</em> values. These genes are indistinguishable by <em>p</em> value alone, but the functional score in this pipeline differentiated these genes based on relatedness to the processes of interest. Furthermore, multiple genes that were highly ranked by the full pipeline did not receive maximally significant <em>p</em> values from the SNP association tests. We assigned SNPs to genes based on proximity, but this assignment can produce false associations between genes and phenotypes. For example, a SNP could tag a regulatory variant that influences multiple nearby genes, only some of which are associated to the trait. Moreover, the SNP may not completely segregate with the causal variant. Likewise, the complexities of genetic architecture, including epistasis, may result in reduced significance for univariate SNP associations for truly causal SNPs. Thus, genes with less than optimal <em>p</em> values may be overlooked if <em>p</em> value were the only ranking criterion. By adding a functional criterion to the ranking process, we can buffer against mapping complexities and gene assignment errors and identify which of the positional candidates are also likely to be functionally related to Histh. The final list of genes was highly plausible and can be followed up relatively easily with modern genetic editing techniques.</p><div id="sec-23" class="subsection"><h3>High-quality candidates for Histh</h3><p id="p-55">Three genes in the final ranked list deserve particular attention: <em>Cxcl12</em>, <em>Ret</em>, and <em>Cacna1c</em>. These genes did not have the most significant <em>p</em> values across the locus, but were predicted to be highly functionally related to Histh-related processes (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The top-ranked gene, <em>Cxcl12</em> (also known as stromal cell-derived factor 1 (Sdf1)), is chemotactic for mast cells via the chemokine receptor <em>Cxcr4</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ghannadan <em>et al.</em> 2002</a>). Mast cells are major drivers of pathological events in anaphylaxis (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lieberman and Garvey 2016</a>), demonstrating that the final predictions are highly relevant to Histh. Additionally, CXCL12/SDF1 secreted by tumor cells is associated with increased endothelial permeability, both locally and systemically. The second-ranked gene <em>Ret</em> encodes the ret proto-oncogene, a cellular tyrosine kinase transmembrane receptor, the activation of which stimulates multiple downstream pathways involved in cell differentiation, growth, migration, and survival, inflammation (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rusmini <em>et al.</em> 2013</a>) and the development of the cardiovascular system (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hiltunen <em>et al.</em> 2000</a>). Alleles of this gene could conceivably modify multiple processes underlying Histh, including the both the anatomical background susceptible to Histh and the acute response to histamine. <em>Ret</em> was significantly associated with multiple functional gene sets (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>). The third-ranked gene, <em>Cacna1c</em>, calcium channel, voltage-dependent, L type, alpha 1C subunit, which is expressed in the heart, blood vessels, and central nervous system (Mouse Genome Informatics Mouse Genome Informatics Web Site). Mutations in <em>Cacna1c</em> are associated with electrophysiological alterations in the heart (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Napolitano <em>et al.</em> 2015</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hedley <em>et al.</em> 2009</a>) suggesting a possible role for <em>Cacna1c</em> in impaired cardiac function in Histh. Interestingly, SNPs in human <em>CACNA1C</em> were recently associated with chronic spontaneous urticaria (<em>i.e.</em>, spontaneous episodes of hives and/or angioedema) and antihistamine drug response (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yan <em>et al.</em> 2018</a>). These results suggest a direct connection between <em>Cacna1c</em> and the histamine response.</p><p id="p-56">All of the above genes lie in the <em>Histh4</em> locus, which accounts for only a portion of the total variation in the Histh phenotype. In the <em>Histh3</em> locus, the highest-ranked candidate gene was <em>Cntn3</em>, which encodes for contactin 3, also called brain-derived immunoglobulin superfamily protein 1 (BIG-1) or plasmacytoma-associated neuronal glycoprotein (PANG), was first reported in 1994 in rats (Connelly <em>et al.,</em> 1994; Yoshihara <em>et al.,</em> 1994). Genetic variants of human <em>CNTN3</em> are associated with an autism spectrum disorder (Morrow <em>et al.,</em> 2008; Hussman <em>et al.,</em> 2011) and vascular abnormalities underlying abdominal aortic aneurysms (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elmore <em>et al.</em> 2009</a>). The latter suggests a potential connection to impaired cardiac function during histamine challenge (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Elmore <em>et al.</em> 2009</a>). Intriguingly, <em>CNTN3</em> is near a segregating SNP for Systemic Capillary Leak Syndrome (SCLS) from a human GWAS. SCLS is an extremely rare disease characterized by transient but potentially lethal episodes of diffuse vascular leakage of proteins and fluids into peripheral tissues, resulting in massive whole-body edema and hypotensive shock. The pathological mechanisms and genetic basis for SCLS remain elusive (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Xie <em>et al.</em> 2013</a>), but SCLS shares many phenotypic properties with Histh in mice. In particular, SCLS attacks are diagnosed based on the clinical triad of hypotension, elevated hematocrit, and hypoalbuminemia, all of which naturally occur in the Histh-sensitive SJL mouse strain (Raza <em>et al.</em> In Press). The potential association between <em>CNTN3</em> and SCLS, therefore, lends credence to its possible functional role in Histh as well. Indeed, <em>CNTN3</em> was not only a positional candidate in the SCLS GWAS, but was contained within functional terms that were enriched among the top positional candidate genes (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Xie <em>et al.</em> 2013</a>), indicating that CNTN3 functions in concert with other genetic risk factors for SCLS.</p><p id="p-57">In the <em>Histh1</em> locus, the top hits in were <em>Creb5</em> and <em>Tril</em>. <em>Creb5</em> codes for cAMP responsive element binding protein 5. <em>Creb5</em> has high expression in the heart (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fagerberg <em>et al.</em> 2014</a>) and has been implicated in cardiac function and pathology (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schisler <em>et al.</em> 2015</a>). Additionally, CREB5 is as component of the age-associated inflammatory network underlying disregulated cytokine expression, i.e., inflammaging. <em>Tril</em> is Tlr4 interactor with leucine-rich repeats and is a functional component of Tlr4 complex involved with LPS signaling and is highly expressed in the kidney (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carpenter <em>et al.</em> 2009</a>), indicating a potential role for <em>Tril</em> in blood pressure regulation. <em>Tril</em>(−/−) mice also produce lower levels of multiple proinflammatory cytokines and chemokines within the brain after <em>E. coli</em> and LPS challenge (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wochal <em>et al.</em> 2014</a>) and is required for Tlr3 signaling, suggesting a potential role in immune modulation. There were no significant hits in the <em>Histh2</em> locus.</p><p id="p-58">None of the prioritized genes here have been tested specifically for association with Histh, and further experimental validation will be required. However, the above genes each have compelling functional associations that can inform follow-up studies.</p></div><div id="sec-24" class="subsection"><h3>Computation and quantitative trait gene prediction</h3><p id="p-59">Definitive functional validation of a quantitative trait gene (QTG) has traditionally required either congenic mapping to resolve an extremely narrow QTL, or <em>ad hoc</em> nomination of a candidate gene for direct experimentation. The advent of modern genetic technologies, such as CRISPR/Cas9 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hsu <em>et al.</em> 2014</a>), allow relatively fast and inexpensive allelic manipulations, so the burden of QTG prediction is moving toward a regime in which a small handful of strong candidates can be followed up individually. Importantly, many QTL, including <em>Histh</em>, contain multiple causal variants, so fine-mapping alone cannot provide definitive validation. Therefore, computational tools that can identify a small number of reasonable candidates can be a significant aid in biological follow-up. We have presented an integrative strategy for ranking genes in a QTL by combining predicted functional associations to the trait with SNP associations. Our method produces a full ranked list of genes in the locus providing researchers with the potential to validate multiple targets. To this end, the <em>Histh</em> QTL represents an extreme use case for QTG prediction–a large, polygenic QTL associated with a physiologically complex trait.</p><p id="p-60">One major limitation to our approach is the decision of which functional terms to include for network-based prediction. The better tailored this set is to the trait of interest, the greater confidence we can have in the final predictions. In principle, the inclusion of a spurious functional term could skew the rankings toward genes that are functionally associated with the spurious term but irrelevant to the trait of interest. One potential way around this issue is to use functional data, such as transcriptomics, directly from the mapping population. However, in some cases, including Histh, the relevant tissue in which to measure gene expression may not be obvious. Alternatively, one could consider distinct rankings for each functional term. In any case, the researcher will have to exercise some measure of judgment in the prioritization process. However, by transferring the judgments from a large list of positional candidate genes to a smaller and more tractable list of trait-related biological processes, we have shown that we can arrive at a strong set of follow up candidates that would have evaded naive <em>p</em> value filters and are relatively unbiased by findings published in the literature.</p><p id="p-61">Another limitation to this approach is our method of assigning <em>p</em> values to genes. We currently use the <em>p</em> value from the single best SNP within 1Mb of the gene body, and these SNPs may or may not be tagging the gene they are assigned to. There exist multiple methods for assigning gene-based <em>p</em> values based on multiple SNP <em>p</em> values, such as the Versatile Gene-Based Test for Genome-wide Association Studies (VEGAS) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2010</a>), Combined Association Test for Genes (COMBAT) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2017</a>), and eigenMT (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Davis <em>et al.</em> 2016</a>). However, none of these methods are directly applicable to the current experiment, and deriving a gene-based test statistic that is applicable here is beyond the scope of this study. We will be investigating this possibility for future implementations of this method.</p><p id="p-62">The final output of our method, a ranked list of positional candidate genes, is easy to interpret, and provides researchers with a clear set of hypotheses to test in the lab. While this approach cannot definitively identify the causal gene or genes in a locus, it does provide a much-reduced set of plausible candidates to test.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">We thank Laura Cort for supervising students during genotyping of congenic mice. ALT and JMM are supported by a grant (R21 LM012615) from the National Library of Medicine of the United States National Institutes of Health (NIH). AR, DNK, EPB, and CT were supported by grants from the NIH and the National Multiple Sclerosis Society (NMSS). DNK was supported by NIH grants from the National Institute of Neurological Disease and Stroke (R01 NS097596), National Institute of Allergy and Infectious Disease (R21 AI145306), and the NMSS (RR-1602-07780).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-64">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9989117">https://doi.org/10.25387/g3.9989117</a>.</p></li><li class="fn" id="fn-3"><p id="p-65"><em>Communicating editor: T. Mackay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-26"><span class="received-label">Received </span>July 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-20"><span class="accepted-label">Accepted </span>October 20, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Tyler <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.1" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Ball</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Blake</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Botstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Butler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Gene ontology: tool for the unification of biology.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/75556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.2" data-doi="10.1093/nar/gkr968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Jay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Bubier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Langston</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Chesler</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GeneWeaver: a web-based system for integrative functional genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1067</span>–<span class="cit-lpage">D1076</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr968</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr968%26rft_id%253Dinfo%253Apmid%252F22080549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22080549&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300160&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.3" data-doi="10.1093/nar/gks1193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Wilhite</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ledoux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Evangelista</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. F.</span> <span class="cit-name-surname">Kim</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NCBI GEO: archive for functional genomics data sets—update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D991</span>–<span class="cit-lpage">D995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks1193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1193%26rft_id%253Dinfo%253Apmid%252F23193258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23193258&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300141&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Khor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Gage</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. J.</span> <span class="cit-name-surname">Lehmann</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A targeted approach to genome-wide studies reveals new genetic associations with central corneal thickness.</span> <abbr class="cit-jnl-abbrev">Mol. Vis.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BVis.%26rft.volume%253D23%26rft.spage%253D952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.5" data-doi="10.1093/nar/gkx1082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Grubb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. O.</span> <span class="cit-name-surname">Walton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Philip</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kolishovski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mouse Phenome Database: an integrative database and analysis suite for curated empirical phenotype data from laboratory mice.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D843</span>–<span class="cit-lpage">D850</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD843%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.6" data-doi="10.1111/j.1399-0039.1995.tb03127.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bunce</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">O’neill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Barnardo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Krausa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Browning</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Phototyping: comprehensive DNA typing for HLA-A, B, C, DRB1, DRB3, DRB4, DRB5 &amp; DQB1 by PCR with 144 primer mixes utilizing sequence-specific primers PCR-SSP.</span> <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1399-0039.1995.tb03127.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DBunce%26rft.auinit1%253DM.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D355%26rft.epage%253D367%26rft.atitle%253DPhototyping%253A%2Bcomprehensive%2BDNA%2Btyping%2Bfor%2BHLA-A%252C%2BB%252C%2BC%252C%2BDRB1%252C%2BDRB3%252C%2BDRB4%252C%2BDRB5%2B%2526amp%253B%2BDQB1%2Bby%2BPCR%2Bwith%2B144%2Bprimer%2Bmixes%2Butilizing%2Bsequence-specific%2Bprimers%2B%2528PCR-SSP%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-0039.1995.tb03127.x%26rft_id%253Dinfo%253Apmid%252F8838344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-0039.1995.tb03127.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8838344&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TK99900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.7" data-doi="10.4049/jimmunol.0901518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Carlson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dellacasagrande</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gibbons</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TRIL, a functional component of the TLR4 signaling complex, highly expressed in brain.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">3989</span>–<span class="cit-lpage">3995</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4049/jimmunol.0901518</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.0901518%26rft_id%253Dinfo%253Apmid%252F19710467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4My82LzM5ODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.12.4223.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Csardi</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nepusz</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The igraph software package for complex network research. InterJournal</span>. <abbr class="cit-jnl-abbrev">Complex Syst.</abbr> <span class="cit-vol">1695</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComplex%2BSyst.%26rft.volume%253D1695%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.9" data-doi="10.1016/j.ajhg.2015.11.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fresard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Knowles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pala</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Bustamante</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An efficient multiple-testing adjustment for eQTL studies that accounts for linkage disequilibrium between variants.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.021%26rft_id%253Dinfo%253Apmid%252F26749306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26749306&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.10" data-doi="10.1093/bioinformatics/bti525"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durinck</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Moreau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kasprzyk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">De Moor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3439</span>–<span class="cit-lpage">3440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti525</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DDurinck%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D16%26rft.spage%253D3439%26rft.epage%253D3440%26rft.atitle%253DBioMart%2Band%2BBioconductor%253A%2Ba%2Bpowerful%2Blink%2Bbetween%2Bbiological%2Bdatabases%2Band%2Bmicroarray%2Bdata%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti525%26rft_id%253Dinfo%253Apmid%252F16082012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti525&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16082012&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231360600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.11" data-doi="10.1038/nprot.2009.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durinck</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Spellman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1184</span>–<span class="cit-lpage">1191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2009.97</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DDurinck%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D1184%26rft.epage%253D1191%26rft.atitle%253DMapping%2Bidentifiers%2Bfor%2Bthe%2Bintegration%2Bof%2Bgenomic%2Bdatasets%2Bwith%2Bthe%2BR%252FBioconductor%2Bpackage%2BbiomaRt.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.97%26rft_id%253Dinfo%253Apmid%252F19617889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19617889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.12" data-doi="10.1093/nar/30.1.207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Domrachev</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Lash</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/30.1.207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D210%26rft.atitle%253DGene%2BExpression%2BOmnibus%253A%2BNCBI%2Bgene%2Bexpression%2Band%2Bhybridization%2Barray%2Bdata%2Brepository%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.1.207%26rft_id%253Dinfo%253Apmid%252F11752295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.1.207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11752295&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173077100055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.13" data-doi="10.1016/j.jvs.2009.01.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elmore</span>,  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Obmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kuivaniemi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tromp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Gerhard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of a genetic variant associated with abdominal aortic aneurysms on chromosome 3p12.3 by genome wide association.</span> <abbr class="cit-jnl-abbrev">J. Vasc. Surg.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jvs.2009.01.041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvascular%2Bsurgery%2B%253A%2B%2Bofficial%2Bpublication%252C%2Bthe%2BSociety%2Bfor%2BVascular%2BSurgery%2B%255Band%255D%2BInternational%2BSociety%2Bfor%2BCardiovascular%2BSurgery%252C%2BNorth%2BAmerican%2BChapter%26rft.stitle%253DJ%2BVasc%2BSurg%26rft.aulast%253DElmore%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D49%26rft.issue%253D6%26rft.spage%253D1525%26rft.epage%253D1531%26rft.atitle%253DIdentification%2Bof%2Ba%2Bgenetic%2Bvariant%2Bassociated%2Bwith%2Babdominal%2Baortic%2Baneurysms%2Bon%2Bchromosome%2B3p12.3%2Bby%2Bgenome%2Bwide%2Bassociation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jvs.2009.01.041%26rft_id%253Dinfo%253Apmid%252F19497516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jvs.2009.01.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19497516&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.14" data-doi="10.1074/mcp.M113.035600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fagerberg</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Hallström</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Oksvold</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kampf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Djureinovic</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/mcp.M113.035600</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M113.035600%26rft_id%253Dinfo%253Apmid%252F24309898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjg6IjEzLzIvMzk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.15" data-doi="10.1093/nar/gky1055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Gene Ontology Consortium</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The Gene Ontology resource: 20 years and still GOing strong.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">D330</span>–<span class="cit-lpage">D338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD330%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky1055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.16" data-doi="10.1159/000057747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghannadan</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Hauswirth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.-H.</span> <span class="cit-name-surname">Schernthaner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Müller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Klepetko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detection of novel CD antigens on the surface of human mast cells and basophils.</span> <abbr class="cit-jnl-abbrev">Int. Arch. Allergy Immunol.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000057747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Barchives%2Bof%2Ballergy%2Band%2Bimmunology%26rft.stitle%253DInt%2BArch%2BAllergy%2BImmunol%26rft.aulast%253DGhannadan%26rft.auinit1%253DM.%26rft.volume%253D127%26rft.issue%253D4%26rft.spage%253D299%26rft.epage%253D307%26rft.atitle%253DDetection%2Bof%2Bnovel%2BCD%2Bantigens%2Bon%2Bthe%2Bsurface%2Bof%2Bhuman%2Bmast%2Bcells%2Band%2Bbasophils.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000057747%26rft_id%253Dinfo%253Apmid%252F12021549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000057747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12021549&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.17" data-doi="10.1093/nar/gkv443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goya</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Yao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Krishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Homilius</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FNTM: a server for predicting functional networks of tissues in mouse.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">W182</span>–<span class="cit-lpage">W187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv443%26rft_id%253Dinfo%253Apmid%252F25940632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25940632&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.18" data-doi="10.1038/ng.3259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Krishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ricciotti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Zelaya</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Understanding multicellular function and disease with human tissue-specific networks.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3259%26rft_id%253Dinfo%253Apmid%252F25915600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25915600&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.19" data-doi="10.1371/journal.pcbi.1000991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Ackert-Bicknell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. G.</span> <span class="cit-name-surname">Troyanskaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Hibbs</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Functional genomics complements quantitative genetics in identifying disease-gene associations.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000991</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1000991</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1000991%26rft.epage%253De1000991%26rft.atitle%253DFunctional%2Bgenomics%2Bcomplements%2Bquantitative%2Bgenetics%2Bin%2Bidentifying%2Bdisease-gene%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000991%26rft_id%253Dinfo%253Apmid%252F21085640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.20" data-doi="10.1002/humu.21066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedley</span>,  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jørgensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schlamowitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Moolman-Smook</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Kanters</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic basis of Brugada syndrome: a mutation update.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.21066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DHedley%26rft.auinit1%253DP.%2BL.%26rft.volume%253D30%26rft.issue%253D9%26rft.spage%253D1256%26rft.epage%253D1266%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2BBrugada%2Bsyndrome%253A%2Ba%2Bmutation%2Bupdate.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.21066%26rft_id%253Dinfo%253Apmid%252F19606473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.21066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19606473&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269675400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ganellin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Timmerman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schwartz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Shankley</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">International union of pharmacology. XIII. classification of histamine receptors.</span> <abbr class="cit-jnl-abbrev">Pharmacol. Rev.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacological%2BReviews%26rft.stitle%253DPharmacol.%2BRev.%26rft.aulast%253DHill%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D253%26rft.epage%253D278%26rft.atitle%253DInternational%2BUnion%2Bof%2BPharmacology.%2BXIII.%2BClassification%2Bof%2BHistamine%2BReceptors%26rft_id%253Dinfo%253Apmid%252F9311023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoicGhhcm1yZXYiO3M6NToicmVzaWQiO3M6ODoiNDkvMy8yNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.22" data-doi="10.1002/1097-0177(2000)9999:9999&lt;::AID-DVDY1031&gt;3.0.CO;2-P"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hiltunen</span>,  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Laurikainen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Airaksinen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Saarma</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">GDNF family receptors in the embryonic and postnatal rat heart and reduced cholinergic innervation in mice hearts lacking RET or GFRα2.</span> <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">219</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/1097-0177(2000)9999:9999&lt;::AID-DVDY1031&gt;3.0.CO;2-P</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253DHiltunen%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D219%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D39%26rft.atitle%253DGDNF%2Bfamily%2Breceptors%2Bin%2Bthe%2Bembryonic%2Band%2Bpostnatal%2Brat%2Bheart%2Band%2Breduced%2Bcholinergic%2Binnervation%2Bin%2Bmice%2Bhearts%2Blacking%2Bret%2Bor%2BGFRalpha2.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0177%25282000%25299999%253A9999%253C%253A%253AAID-DVDY1031%253E3.0.CO%253B2-P%26rft_id%253Dinfo%253Apmid%252F10974669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1097-0177(2000)9999:9999&lt;::AID-DVDY1031&gt;3.0.CO;2-P&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10974669&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089062600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.23" data-doi="10.1016/j.cell.2014.05.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>,  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Lander</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and applications of CRISPR-Cas9 for genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1262</span>–<span class="cit-lpage">1278</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.05.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1262%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.05.010%26rft_id%253Dinfo%253Apmid%252F24906146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.05.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24906146&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340881400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.24" data-doi="10.1101/gr.155697.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ju</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Greene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eichinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nair</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Hodgin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Defining cell-type specificity at the transcriptional level in human disease.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1862</span>–<span class="cit-lpage">1873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.155697.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.155697.113%26rft_id%253Dinfo%253Apmid%252F23950145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xODYyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.25" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Zaitlen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Wade</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kirby</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Heckerman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.26" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Goodstadt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Danecek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.27" data-doi="10.1038/nn.4353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Yao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Theesfeld</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Wong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide prediction and functional characterization of the genetic basis of autism spectrum disorder.</span> <abbr class="cit-jnl-abbrev">Nat. Neurosci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1454</span>–<span class="cit-lpage">1462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nn.4353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BNeurosci.%26rft.volume%253D19%26rft.spage%253D1454%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nn.4353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.28" data-doi="10.1007/s11882-016-0598-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lieberman</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. H.</span> <span class="cit-name-surname">Garvey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mast cells and anaphylaxis.</span> <abbr class="cit-jnl-abbrev">Curr. Allergy Asthma Rep.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11882-016-0598-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BAllergy%2BAsthma%2BRep.%26rft.volume%253D16%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11882-016-0598-5%26rft_id%253Dinfo%253Apmid%252F26857018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11882-016-0598-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26857018&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.29" data-doi="10.1016/j.ajhg.2010.06.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">McRae</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Nyholt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Medland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A versatile gene-based test for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.06.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DLiu%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D145%26rft.atitle%253DA%2Bversatile%2Bgene-based%2Btest%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.06.009%26rft_id%253Dinfo%253Apmid%252F20598278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.06.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20598278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280029200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.30"><div class="cit-metadata unstructured">Meyer, D., E. Dimitriadou, K. Hornik, A. Weingessel, and F. Leisch, 2018 e1071: Misc Functions of the Department of Statistics, Probability Theory Group (Formerly: E1071), TU Wien. R package version 1.7–0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-9.12.4223.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Napolitano</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Splawski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Timothy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bloise</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Priori</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Timothy syndrome</span>, <span class="cit-source">GeneReviews</span>, <span class="cit-publ-name">University of Washington</span>, <span class="cit-publ-loc">Seattle</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.32" data-doi="10.1103/PhysRevE.69.066133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>,  <span class="cit-name-given-names">M. E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Fast algorithm for detecting community structure in networks.</span> <abbr class="cit-jnl-abbrev">Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</abbr> <span class="cit-vol">69</span>: <span class="cit-elocation-id">066133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1103/PhysRevE.69.066133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysical%2Breview.%2BE%252C%2BStatistical%252C%2Bnonlinear%252C%2Band%2Bsoft%2Bmatter%2Bphysics%26rft.stitle%253DPhys%2BRev%2BE%2BStat%2BNonlin%2BSoft%2BMatter%2BPhys%26rft.aulast%253DNewman%26rft.auinit1%253DM.%2BE.%26rft.volume%253D69%26rft.issue%253D6%2BPt%2B2%26rft.spage%253D066133%26rft.epage%253D066133%26rft.atitle%253DFast%2Balgorithm%2Bfor%2Bdetecting%2Bcommunity%2Bstructure%2Bin%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1103%252FPhysRevE.69.066133%26rft_id%253Dinfo%253Apmid%252F15244693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1103/PhysRevE.69.066133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15244693&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Leung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Kirkpatrick</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A large QTL for fear and anxiety mapped using an F2 cross can be dissected into multiple smaller QTLs.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BBrain%2BBehav.%26rft.volume%253D12%26rft.spage%253D714%26rft_id%253Dinfo%253Apmid%252F23876074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23876074&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.34" data-doi="10.1101/gr.2825804"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petkov</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ding</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Cassell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wagner</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An efficient SNP system for mouse genome scanning and elucidating strain relationships.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1806</span>–<span class="cit-lpage">1811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.2825804</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DPetkov%26rft.auinit1%253DP.%2BM.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D1806%26rft.epage%253D1811%26rft.atitle%253DAn%2BEfficient%2BSNP%2BSystem%2Bfor%2BMouse%2BGenome%2BScanning%2Band%2BElucidating%2BStrain%2BRelationships%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2825804%26rft_id%253Dinfo%253Apmid%252F15342563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzkvMTgwNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.12.4223.35"><div class="cit-metadata unstructured">Raza, A., E. C. Chan, W.-S. Chen, Z. Xie, L. M. Scott, <em>et al.</em>, 2019 A natural mouse model reveals genetic determinants of Systemic Capillary Leak Syndrome (Clarkson disease). Communications Biology.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-9.12.4223.36"><div class="cit-metadata unstructured">Reimand, J., R. Kolde, and T. Arak, 2018 <em>gProfileR: Interface to the ’g:Profiler’ Toolkit</em>. R package version 0.6.7.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.37" data-doi="10.1371/journal.pone.0059066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rusmini</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Griseri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lantieri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Matera</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Hudspeth</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Induction of RET dependent and independent pro-inflammatory programs in human peripheral blood mononuclear cells from Hirschsprung patients.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e59066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0059066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De59066%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0059066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0059066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.38" data-doi="10.1161/JAHA.114.001136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schisler</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Grevengoed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Pascual</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Cooper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ellis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cardiac energy dependence on glucose increases metabolites related to glutathione and activates metabolic genes controlled by mechanistic target of rapamycin.</span> <abbr class="cit-jnl-abbrev">J. Am. Heart Assoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e001136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1161/JAHA.114.001136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BHeart%2BAssoc.%26rft_id%253Dinfo%253Adoi%252F10.1161%252FJAHA.114.001136%26rft_id%253Dinfo%253Apmid%252F25713290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiYWhhb2EiO3M6NToicmVzaWQiO3M6MTE6IjQvMi9lMDAxMTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.39" data-doi="10.1007/s00335-012-9421-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">C. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Eppig</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Mammalian Phenotype Ontology as a unifying standard for experimental and high-throughput phenotyping data.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">653</span>–<span class="cit-lpage">668</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9421-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSmith%26rft.auinit1%253DC.%2BL.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D653%26rft.epage%253D668%26rft.atitle%253DThe%2BMammalian%2BPhenotype%2BOntology%2Bas%2Ba%2Bunifying%2Bstandard%2Bfor%2Bexperimental%2Band%2Bhigh-throughput%2Bphenotyping%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9421-3%26rft_id%253Dinfo%253Apmid%252F22961259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9421-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22961259&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.40" data-doi="10.1073/pnas.90.8.3700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudweeks</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Todd</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. P.</span> <span class="cit-name-surname">Blankenhorn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Wardell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Woodward</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Locus controlling bordetella pertussis-induced histamine sensitization (Bphs), an autoimmune disease-susceptibility gene, maps distal to T-cell receptor beta-chain gene on mouse chromosome 6.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">3700</span>–<span class="cit-lpage">3704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.90.8.3700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSudweeks%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D90%26rft.issue%253D8%26rft.spage%253D3700%26rft.epage%253D3704%26rft.atitle%253DLocus%2Bcontrolling%2BBordetella%2Bpertussis-induced%2Bhistamine%2Bsensitization%2B%2528Bphs%2529%252C%2Ban%2Bautoimmune%2Bdisease-susceptibility%2Bgene%252C%2Bmaps%2Bdistal%2Bto%2BT-cell%2Breceptor%2Bbeta-chain%2Bgene%2Bon%2Bmouse%2Bchromosome%2B6.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.8.3700%26rft_id%253Dinfo%253Apmid%252F8475118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MC84LzM3MDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.41" data-doi="10.1159/000231796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaz</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">de Souza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Hornbrook</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Hanson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Lynch</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Sensitivity to intravenous injections of histamine and serotonin in inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Int. Arch. Allergy Immunol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000231796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Barchives%2Bof%2Ballergy%2Band%2Bapplied%2Bimmunology%26rft.stitle%253DInt%2BArch%2BAllergy%2BAppl%2BImmunol%26rft.aulast%253DVaz%26rft.auinit1%253DN.%2BM.%26rft.volume%253D53%26rft.issue%253D6%26rft.spage%253D545%26rft.epage%253D554%26rft.atitle%253DSensitivity%2Bto%2Bintravenous%2Binjections%2Bof%2Bhistamine%2Band%2Bserotonin%2Bin%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000231796%26rft_id%253Dinfo%253Apmid%252F863521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000231796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=863521&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F4223.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.42" data-doi="10.1534/genetics.117.300257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Potash</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">COMBAT: a combined association test for genes using summary statistics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">891</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300257</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300257%26rft_id%253Dinfo%253Apmid%252F28878002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzMvODgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi80MjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.43" data-doi="10.1016/j.lfs.2014.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shibamoto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tanida</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kuda</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kurata</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mouse anaphylactic shock is caused by reduced cardiac output, but not by systemic vasodilatation or pulmonary vasoconstriction, via PAF and histamine.</span> <abbr class="cit-jnl-abbrev">Life Sci.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.lfs.2014.09.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLife%2BSci.%26rft.volume%253D116%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.lfs.2014.09.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.lfs.2014.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.44" data-doi="10.4049/jimmunol.1302392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wochal</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. A.</span> <span class="cit-name-surname">Rathinam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dunne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Carlson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Kuang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TRIL is involved in cytokine production in the brain following <em>escherichia coli</em> infection.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1911</span>–<span class="cit-lpage">1919</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4049/jimmunol.1302392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1302392%26rft_id%253Dinfo%253Apmid%252F25015823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzE5MTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzQyMjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.45" data-doi="10.4161/rdis.27445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Nagarajan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Sturdevant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Iwaki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Chan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide SNP analysis of the systemic capillary leak syndrome (Clarkson disease).</span> <abbr class="cit-jnl-abbrev">Rare Dis.</abbr> <span class="cit-vol">1</span>: e27445.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/rdis.27445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRare%2BDis.%26rft.volume%253D1%26rft.spage%253D27445e%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frdis.27445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rdis.27445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Luo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">He</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Association of CACNA1C gene genetic polymorphism with the susceptibility as well as prognosis for chronic spontaneous urticaria. Zhong nan da xue xue bao. Yi xue ban= Journal of Central South University.</span> <abbr class="cit-jnl-abbrev">Med. Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed.%2BSci.%26rft.volume%253D43%26rft.spage%253D929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.4223.47" data-doi="10.1101/gr.120741.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yazbek</span>,  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Buchner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Geisinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Burrage</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Spiezio</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deep congenic analysis identifies many strong, context-dependent QTLs, one of which, <em>Slc35b4</em>, regulates obesity and glucose homeostasis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1073</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.120741.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.120741.111%26rft_id%253Dinfo%253Apmid%252F21507882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTA2NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvNDIyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/4223.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/4209" title="Marker-Trait Associations for Enhancing Agronomic Performance, Disease Resistance, and Grain Quality in Synthetic and Bread Wheat Accessions in Western Siberia" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/4235" title="Machine Learning Enables High-Throughput Phenotyping for Analyses of the Genetic Architecture of Bulliform Cell Patterning in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/4223" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wZt8hTZiKIiMZ1ofuY-UY0hubEaPu78ROafCaZ_zeSM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/4223" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/4223" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/4223" class="active" data-icon-position="" data-hide-link-title="0">Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494987" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-uoGIocGMld2UBC-bmXbiyjghnoTScGwEjhNcKXjyT4Y" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4223.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/4223" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-R6jDijXKPnBnmNZQXXn8H5Swu07QgmYyMpzDl3RC1_w" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/4223" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/4223.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494987" id="node494987--2" data-pisa="ggg;9/12/4223" data-pisa-master="ggg;g3.119.400740" data-apath="/ggg/9/12/4223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anna L. Tyler</span>, <span class="highwire-citation-author" data-delta="1">Abbas Raza</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dimitry N. Krementsov</span>, <span class="highwire-citation-author" data-delta="3">Laure K. Case</span>, <span class="highwire-citation-author" data-delta="4">Rui Huang</span>, <span class="highwire-citation-author" data-delta="5">Runlin Z. Ma</span>, <span class="highwire-citation-author" data-delta="6">Elizabeth P. Blankenhorn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cory Teuscher</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Matthew Mahoney</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4223-4233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400740 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494987--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna L. Tyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BL.%2BTyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tyler%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BL.%2BTyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anna L. Tyler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abbas Raza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abbas%2BRaza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raza%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbbas%2BRaza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitry N. Krementsov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biomedical and Health Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitry%2BN.%2BKrementsov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krementsov%20DN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADimitry%2BN.%2BKrementsov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dimitry N. Krementsov</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laure K. Case </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laure%2BK.%2BCase%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Case%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaure%2BK.%2BCase%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Runlin Z. Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Runlin%2BZ.%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20RZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARunlin%2BZ.%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elizabeth P. Blankenhorn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Immunology, Drexel University College of Medicine, Philadelphia, PA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BP.%2BBlankenhorn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blankenhorn%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BP.%2BBlankenhorn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cory Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, University of Vermont Larner College of Medicine, Burlington, VT, 05405,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cory%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACory%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cory Teuscher</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">J. Matthew Mahoney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Neurological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Computer Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BMatthew%2BMahoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mahoney%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BMatthew%2BMahoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Matthew Mahoney</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:John.M.Mahoney@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">John.M.Mahoney@uvm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/4223" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/4223&amp;text=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/4223&amp;t=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/4223&amp;title=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494987/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494987" id="node494987--3" data-pisa="ggg;9/12/4223" data-pisa-master="ggg;g3.119.400740" data-apath="/ggg/9/12/4223.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Network-Based Functional Prediction Augments Genetic Association To Predict Candidate Genes for Histamine Hypersensitivity in Mice</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anna L. Tyler</span>, <span class="highwire-citation-author" data-delta="1">Abbas Raza</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dimitry N. Krementsov</span>, <span class="highwire-citation-author" data-delta="3">Laure K. Case</span>, <span class="highwire-citation-author" data-delta="4">Rui Huang</span>, <span class="highwire-citation-author" data-delta="5">Runlin Z. Ma</span>, <span class="highwire-citation-author" data-delta="6">Elizabeth P. Blankenhorn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cory Teuscher</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. Matthew Mahoney</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">4223-4233;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400740 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494987--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna L. Tyler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BL.%2BTyler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tyler%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BL.%2BTyler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8371-2377" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anna L. Tyler</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abbas Raza </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abbas%2BRaza%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raza%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbbas%2BRaza%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitry N. Krementsov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biomedical and Health Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitry%2BN.%2BKrementsov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krementsov%20DN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADimitry%2BN.%2BKrementsov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2709-7466" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dimitry N. Krementsov</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Laure K. Case </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory, 600 Main St. Bar Harbor, ME, 04609,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laure%2BK.%2BCase%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Case%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaure%2BK.%2BCase%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Rui Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rui%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARui%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Runlin Z. Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>School of Life Sciences, University of Chinese Academy of Sciences, Beijing 100049, China, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Runlin%2BZ.%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20RZ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARunlin%2BZ.%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Elizabeth P. Blankenhorn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Immunology, Drexel University College of Medicine, Philadelphia, PA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BP.%2BBlankenhorn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blankenhorn%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BP.%2BBlankenhorn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Cory Teuscher </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Medicine,</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Pathology and Laboratory Medicine, University of Vermont Larner College of Medicine, Burlington, VT, 05405,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cory%2BTeuscher%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Teuscher%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACory%2BTeuscher%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9236-8843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cory Teuscher</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">J. Matthew Mahoney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Neurological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Department of Computer Science,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BMatthew%2BMahoney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mahoney%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BMatthew%2BMahoney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1425-5939" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. Matthew Mahoney</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:John.M.Mahoney@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">John.M.Mahoney@uvm.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=4223&amp;rft.epage=4233&amp;rft.atitle=Network-Based%20Functional%20Prediction%20Augments%20Genetic%20Association%20To%20Predict%20Candidate%20Genes%20for%20Histamine%20Hypersensitivity%20in%20Mice&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tyler%2C+Anna+L.&amp;rft.au=Raza%2C+Abbas&amp;rft.au=Krementsov%2C+Dimitry+N.&amp;rft.au=Case%2C+Laure+K.&amp;rft.au=Huang%2C+Rui&amp;rft.au=Ma%2C+Runlin+Z.&amp;rft.au=Blankenhorn%2C+Elizabeth+P.&amp;rft.au=Teuscher%2C+Cory&amp;rft.au=Mahoney%2C+J.+Matthew"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494987/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494987/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494987/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494987/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494987/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494987/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494987/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494987/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494987/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494987/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494987/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494987/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/4223" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/4223.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/4223.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/4223.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
