<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/12/3933" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/12/3933.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/12/3933.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/12/3933.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/12/3933.atom" />
<meta name="HW.pisa" content="ggg;9/12/3933" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species" />
<meta name="DC.Identifier" content="10.1534/g3.119.400529" />
<meta name="DC.Date" content="2019-12-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Stam et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Wild tomato species, like Solanum chilense , are important germplasm resources for enhanced biotic and abiotic stress resistance in tomato breeding. S. chilense also serves as a model to study adaptation of plants to drought and the evolution of seed banks. The absence of a well-annotated reference genome in this compulsory outcrossing, very diverse species limits in-depth studies on the genes involved.

We generated ∼134 Gb of DNA and 157 Gb of RNA sequence data for S chilense , which yielded a draft genome with an estimated length of 914 Mb, encoding 25,885 high-confidence predicted gene models, which show homology to known protein-coding genes of other tomato species. Approximately 71% of these gene models are supported by RNA-seq data derived from leaf tissue samples. Benchmarking with Universal Single-Copy Orthologs (BUSCO) analysis of predicted gene models retrieved 93.3% of BUSCO genes. To further verify the genome annotation completeness and accuracy, we manually inspected the NLR resistance gene family and assessed its assembly quality. We find subfamilies of NLRs unique to S. chilense . Synteny analysis suggests significant degree of the gene order conservation between the S. chilense , S. lycopersicum and S. pennellii genomes .

We generated the first genome and transcriptome sequence assemblies for the wild tomato species Solanum chilense and demonstrated their value in comparative genomics analyses. These data are an important resource for studies on adaptation to biotic and abiotic stress in Solanaceae , on evolution of self-incompatibility and for tomato breeding." />
<meta name="DC.Contributor" content="Remco Stam" />
<meta name="DC.Contributor" content="Tetyana Nosenko" />
<meta name="DC.Contributor" content="Anja C. Hörger" />
<meta name="DC.Contributor" content="Wolfgang Stephan" />
<meta name="DC.Contributor" content="Michael Seidel" />
<meta name="DC.Contributor" content="José M. M. Kuhn" />
<meta name="DC.Contributor" content="Georg Haberer" />
<meta name="DC.Contributor" content="Aurelien Tellier" />
<meta name="article:published_time" content="2019-12-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Wild tomato species, like &lt;i&gt;Solanum chilense&lt;/i&gt;, are important germplasm resources for enhanced biotic and abiotic stress resistance in tomato breeding. &lt;i&gt;S. chilense&lt;/i&gt; also serves as a model to study adaptation of plants to drought and the evolution of seed banks. The absence of a well-annotated reference genome in this compulsory outcrossing, very diverse species limits in-depth studies on the genes involved.&lt;/p&gt;&lt;p&gt;We generated ∼134 Gb of DNA and 157 Gb of RNA sequence data for &lt;i&gt;S chilense&lt;/i&gt;, which yielded a draft genome with an estimated length of 914 Mb, encoding 25,885 high-confidence predicted gene models, which show homology to known protein-coding genes of other tomato species. Approximately 71% of these gene models are supported by RNA-seq data derived from leaf tissue samples. Benchmarking with Universal Single-Copy Orthologs (BUSCO) analysis of predicted gene models retrieved 93.3% of BUSCO genes. To further verify the genome annotation completeness and accuracy, we manually inspected the NLR resistance gene family and assessed its assembly quality. We find subfamilies of NLRs unique to &lt;i&gt;S. chilense&lt;/i&gt;. Synteny analysis suggests significant degree of the gene order conservation between the &lt;i&gt;S. chilense&lt;/i&gt;, &lt;i&gt;S. lycopersicum&lt;/i&gt; and &lt;i&gt;S. pennellii genomes&lt;/i&gt;.&lt;/p&gt;&lt;p&gt;We generated the first genome and transcriptome sequence assemblies for the wild tomato species &lt;i&gt;Solanum chilense&lt;/i&gt; and demonstrated their value in comparative genomics analyses. These data are an important resource for studies on adaptation to biotic and abiotic stress in &lt;i&gt;Solanaceae&lt;/i&gt;, on evolution of self-incompatibility and for tomato breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/12/01" />
<meta name="citation_mjid" content="ggg;9/12/3933" />
<meta name="citation_id" content="9/12/3933" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/12/3933" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/12/3933.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/12/3933.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/12/3933.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400529" />
<meta name="citation_pmid" content="31604826" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="12" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3933" />
<meta name="citation_lastpage" content="3941" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Remco Stam" />
<meta name="citation_author_institution" content="Phytopathology, Technical University Munich, Germany," />
<meta name="citation_author_institution" content="Population Genetics, Technical University Munich, Germany," />
<meta name="citation_author_email" content="stam@wzw.tum.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3444-6954" />
<meta name="citation_author" content="Tetyana Nosenko" />
<meta name="citation_author_institution" content="Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany," />
<meta name="citation_author_institution" content="Environmental Simulations, Helmholtz Center of Munich, Germany," />
<meta name="citation_author" content="Anja C. Hörger" />
<meta name="citation_author_institution" content="Department of Biosciences, University of Salzburg, Austria," />
<meta name="citation_author" content="Wolfgang Stephan" />
<meta name="citation_author_institution" content="Evolutionary Biology, LMU Munich and Natural History Museum Berlin, Germany, and" />
<meta name="citation_author" content="Michael Seidel" />
<meta name="citation_author_institution" content="Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany," />
<meta name="citation_author" content="José M. M. Kuhn" />
<meta name="citation_author_institution" content="Evolutionary Biology and Ecology, Albert-Ludwig University of Freiburg, Germany" />
<meta name="citation_author" content="Georg Haberer" />
<meta name="citation_author_institution" content="Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany," />
<meta name="citation_author" content="Aurelien Tellier" />
<meta name="citation_author_institution" content="Population Genetics, Technical University Munich, Germany," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8895-0785" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. E. Allen;citation_author=S. L. Salzberg;citation_title=JIGSAW: integration of multiple sources of evidence for gene prediction.;citation_volume=21;citation_year=2005;citation_pmid=16076884;citation_doi=10.1093/bioinformatics/bti609" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=G. Andolfo;citation_author=F. Jupe;citation_author=K. Witek;citation_author=G. J. Etherington;citation_author=M. R. Ercolano;citation_title=Defining the full tomato NB-LRR resistance gene repertoire using genomic and cDNA RenSeq.;citation_pages=120;citation_volume=14;citation_year=2014;citation_pmid=24885638;citation_doi=10.1186/1471-2229-14-120" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=U. Arunyawat;citation_author=W. Stephan;citation_author=T. Städler;citation_title=Using multilocus sequence data to assess population structure, natural selection, and linkage disequilibrium in wild tomatoes.;citation_volume=24;citation_year=2007;citation_pmid=17675653;citation_doi=10.1093/molbev/msm162" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=E. Baggs;citation_author=G. Dagdas;citation_author=K. Krasileva;citation_title=NLR diversity, helpers and integrated domains: making sense of the NLR IDentity.;citation_pages=59-67;citation_volume=38;citation_year=2017;citation_pmid=28494248;citation_doi=10.1016/j.pbi.2017.04.012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Boetzer;citation_author=C. V. Henkel;citation_author=H. J. Jansen;citation_author=D. Butler;citation_author=W. Pirovano;citation_title=Scaffolding pre-assembled contigs using SSPACE.;citation_volume=27;citation_year=2011;citation_pmid=21149342;citation_doi=10.1093/bioinformatics/btq683" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Boetzer;citation_author=W. Pirovano;citation_title=Toward almost closed genomes with GapFiller.;citation_pages=R56-R56;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22731987;citation_doi=10.1186/gb-2012-13-6-r56" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina Sequence Data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Bolger;citation_author=F. Scossa;citation_author=M. E. Bolger;citation_author=C. Lanz;citation_author=F. Maumus;citation_title=The genome of the stress-tolerant wild tomato species Solanum pennellii.;citation_pages=1034-1038;citation_volume=46;citation_year=2014;citation_pmid=25064008;citation_doi=10.1038/ng.3046" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. B. Böndel;citation_author=H. Lainer;citation_author=T. Nosenko;citation_author=M. Mboup;citation_author=A. Tellier;citation_title=North–South colonization associated with local adaptation of the wild tomato species Solanum chilense.;citation_volume=32;citation_year=2015;citation_pmid=26232423;citation_doi=10.1093/molbev/msv166" />
<meta name="citation_reference" content="citation_journal_title=R. Soc. Open Sci.;citation_author=K. B. Böndel;citation_author=T. Nosenko;citation_author=W. Stephan;citation_title=Signatures of natural selection in abiotic stress-responsive genes of Solanum chilense.;citation_volume=5;citation_year=2018;citation_doi=10.1098/rsos.171198" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Plant Genomics;citation_author=A. Conesa;citation_author=S. Götz;citation_title=Blast2GO: A comprehensive suite for functional analysis in plant genomics.;citation_pages=619832-619832;citation_volume=2008;citation_year=2008;citation_pmid=18483572;citation_doi=10.1155/2008/619832" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=I. Fischer;citation_author=K. A. Steige;citation_author=W. Stephan;citation_author=M. Mboup;citation_title=Sequence evolution and expression regulation of stress-responsive genes in natural populations of wild tomato.;citation_pages=e78182;citation_volume=8;citation_year=2013;citation_pmid=24205149;citation_doi=10.1371/journal.pone.0078182" />
<meta name="citation_reference" content="Gilbert, D., 2013 Gene-omes built from mRNA seq not genome DNA, in 7th annual arthropod genomics symposium. Notre Dame" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Inf. Softw. Technol.;citation_author=G. Gremme;citation_author=V. Brendel;citation_author=M. E. Sparks;citation_author=S. Kurtz;citation_title=Engineering a software tool for gene structure prediction in higher organisms.;citation_pages=965-978;citation_volume=47;citation_year=2005;citation_doi=10.1016/j.infsof.2005.09.005" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=J.-F. Dufayard;citation_author=V. Lefort;citation_author=M. Anisimova;citation_author=W. Hordijk;citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.;citation_volume=59;citation_year=2010;citation_pmid=20525638;citation_doi=10.1093/sysbio/syq010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=J. R. Wortman;citation_author=S. L. Salzberg;citation_title=DAGchainer: a tool for mining segmental genome duplications and synteny.;citation_volume=20;citation_year=2004;citation_pmid=15247098;citation_doi=10.1093/bioinformatics/bth397" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Igic;citation_author=W. A. Smith;citation_author=K. A. Robertson;citation_author=B. A. Schaal;citation_author=J. R. Kohn;citation_title=Studies of self-incompatibility in wild tomatoes: I. S-allele diversity in Solanum chilense (Dun.) Reiche [corrected] (Solanaceae).;citation_pages=553-561;citation_volume=99;citation_year=2007;citation_issue=5;citation_pmid=17700636;citation_doi=10.1038/sj.hdy.6801035" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. D. G. Jones;citation_author=R. E. Vance;citation_author=J. L. Dangl;citation_title=Intracellular innate immune surveillance devices in plants and animals;citation_pages=aaf6395-aaf6395;citation_volume=354;citation_year=2016;citation_issue=6316;citation_pmid=27934708;citation_doi=10.1126/science.aaf6395" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Li;citation_author=A. Godzik;citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Liao;citation_author=G. K. Smyth;citation_author=W. Shi;citation_title=featureCounts: an efficient general purpose program for assigning sequence reads to genomic features.;citation_volume=30;citation_year=2014;citation_pmid=24227677;citation_doi=10.1093/bioinformatics/btt656" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. Lin;citation_author=G. Zhu;citation_author=J. Zhang;citation_author=X. Xu;citation_author=Q. Yu;citation_title=Genomic analyses provide insights into the history of tomato breeding.;citation_pages=1220-1226;citation_volume=46;citation_year=2014;citation_pmid=25305757;citation_doi=10.1038/ng.3117" />
<meta name="citation_reference" content="citation_journal_title=Genome Biology 15;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.;citation_volume=21;citation_year=2011;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. H. Majoros;citation_author=M. Pertea;citation_author=S. L. Salzberg;citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.;citation_volume=20;citation_year=2004;citation_pmid=15145805;citation_doi=10.1093/bioinformatics/bth315" />
<meta name="citation_reference" content="citation_journal_title=Exp. Agric.;citation_author=J.-P. Martínez;citation_author=A. Antúnez;citation_author=R. Pertuzé;citation_author=M. D. P. Acosta;citation_author=X. Palma;citation_title=Effects of saline water on water status, yield and fruit quality of wild (Solanum chilense) and domesticated (Solanum lycopersicum var. cerasiforme) tomatoes.;citation_pages=573-586;citation_volume=48;citation_year=2012;citation_doi=10.1017/S001447971200066X" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. W. Michelmore;citation_author=B. C. Meyers;citation_title=Clusters of Resistance Genes in Plants Evolve by Divergent Selection and a Birth-and-Death Process;citation_pages=1113-1130;citation_volume=8;citation_year=1998;citation_issue=11;citation_pmid=9847076;citation_doi=10.1101/gr.8.11.1113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Moriya;citation_author=M. Itoh;citation_author=S. Okuda;citation_author=A. C. Yoshizawa;citation_author=M. Kanehisa;citation_title=KAAS: an automatic genome annotation and pathway reconstruction server.;citation_volume=35;citation_year=2007;citation_pmid=17526522;citation_doi=10.1093/nar/gkm321" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=T. Nosenko;citation_author=K. B. Böndel;citation_author=G. Kumpfmüller;citation_author=W. Stephan;citation_title=Adaptation to low temperatures in the wild tomato species Solanum chilense.;citation_pages=2853-2869;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13637" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. Parra;citation_author=E. Blanco;citation_author=R. Guigó;citation_title=GeneID in Drosophila;citation_pages=511-515;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779490;citation_doi=10.1101/gr.10.4.511" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. B. Pease;citation_author=D. C. Haak;citation_author=M. W. Hahn;citation_author=L. C. Moyle;citation_title=Phylogenomics reveals three sources of adaptive variation during a rapid radiation.;citation_pages=e1002379;citation_volume=14;citation_year=2016;citation_pmid=26871574;citation_doi=10.1371/journal.pbio.1002379" />
<meta name="citation_reference" content="citation_journal_title=Syst. Bot. Monogr.;citation_author=I. E. Peralta;citation_author=D. M. Spooner;citation_author=S. Knapp;citation_title=Taxonomy of wild tomatoes and their relatives (Solanum sect. Lycopersicoides, sect. Juglandifolia, sect. Lycopersicon; Solanaceae).;citation_pages=186;citation_volume=84;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Proost;citation_author=J. Fostier;citation_author=D. De Witte;citation_author=B. Dhoedt;citation_author=P. Demeester;citation_title=i-ADHoRe 3.0—fast and sensitive detection of genomic homology in extremely large data sets.;citation_volume=40;citation_year=2012;citation_pmid=22102584;citation_doi=10.1093/nar/gkr955" />
<meta name="citation_reference" content="Rambaut, A., and A. Drummond, 2009 FigTree v1. 3.1." />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=L. E. Rose;citation_author=L. Grzeskowiak;citation_author=A. C. Hörger;citation_author=M. Groth;citation_author=W. Stephan;citation_title=Targets of selection in a disease resistance network in wild tomatoes.;citation_pages=921-927;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21726387;citation_doi=10.1111/j.1364-3703.2011.00720.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. H. Schulz;citation_author=D. R. Zerbino;citation_author=M. Vingron;citation_author=E. Birney;citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.;citation_volume=28;citation_year=2012;citation_pmid=22368243;citation_doi=10.1093/bioinformatics/bts094" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Städler;citation_author=U. Arunyawat;citation_author=W. Stephan;citation_title=Population Genetics of Speciation in Two Closely Related Wild Tomatoes (Solanum Section Lycopersicon);citation_pages=339-350;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202377;citation_doi=10.1534/genetics.107.081810" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. Stam;citation_author=D. Scheikl;citation_author=A. Tellier;citation_title=Pooled enrichment sequencing identifies diversity and evolutionary pressures at NLR resistance genes within a wild tomato population.;citation_volume=8;citation_year=2016;citation_pmid=27189991;citation_doi=10.1093/gbe/evw094" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=R. Stam;citation_author=D. Scheikl;citation_author=A. Tellier;citation_title=The wild tomato species Solanum chilense shows variation in pathogen resistance between geographically distinct populations.;citation_pages=e2910;citation_volume=5;citation_year=2017;citation_doi=10.7717/peerj.2910" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=R. Stam;citation_author=G. A. Silva-Arias;citation_author=A. Tellier;citation_title=Subsets of NLR genes show differential signatures of adaptation during colonization of new habitats.;citation_volume=224;citation_year=2019;citation_doi=10.1111/nph.16017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=O. Keller;citation_author=I. Gunduz;citation_author=A. Hayes;citation_author=S. Waack;citation_title=AUGUSTUS: ab initio prediction of alternative transcripts.;citation_pages=W435-W439;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845043;citation_doi=10.1093/nar/gkl200" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Steuernagel;citation_author=F. Jupe;citation_author=K. Witek;citation_author=J. D. G. Jones;citation_author=B. B. H. Wulff;citation_title=NLR-parser: rapid annotation of plant NLR complements.;citation_volume=31;citation_year=2015;citation_pmid=25586514;citation_doi=10.1093/bioinformatics/btv005" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=Z. Tabaeizadeh;citation_author=Z. Agharbaoui;citation_author=H. Harrak;citation_author=V. Poysa;citation_title=Transgenic tomato plants expressing a Lycopersicon chilense chitinase gene demonstrate improved resistance to Verticillium dahliae race 2.;citation_pages=197-202;citation_volume=19;citation_year=1999;citation_doi=10.1007/s002990050733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. D. Tanksley;citation_author=M. W. Ganal;citation_author=J. P. Prince;citation_author=M. C. Devicente;citation_author=M. W. Bonierbale;citation_title=High density molecular linkage maps of the tomato and potato genomes.;citation_pages=1141-1160;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1360934" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Tellier;citation_author=S. J. Y. Laurent;citation_author=H. Lainer;citation_author=P. Pavlidis;citation_author=W. Stephan;citation_title=Inference of seed bank parameters in two wild tomato species using ecological and genetic data.;citation_volume=108;citation_year=2011;citation_pmid=21949404;citation_doi=10.1073/pnas.1111266108" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_title=Exploring genetic variation in the tomato (Solanum section Lycopersicon) clade by whole-genome sequencing.;citation_pages=136-148;citation_volume=80;citation_year=2014;citation_pmid=25039268;citation_doi=10.1111/tpj.12616" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. G. Verlaan;citation_author=S. F. Hutton;citation_author=R. M. Ibrahem;citation_author=R. Kormelink;citation_author=R. G. F. Visser;citation_title=The tomato yellow leaf curl virus resistance genes Ty-1 and Ty-3 are allelic and code for DFDGD-class RNA–dependent RNA polymerases.;citation_pages=e1003399;citation_volume=9;citation_year=2013;citation_pmid=23555305;citation_doi=10.1371/journal.pgen.1003399" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. W. Vurture;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=C. J. Underwood;citation_author=H. Fang;citation_title=GenomeScope: fast reference-free genome profiling from short reads.;citation_pages=2202-2204;citation_volume=33;citation_year=2017;citation_pmid=28369201;citation_doi=10.1093/bioinformatics/btx153" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="Wang, W., G. Haberer, H. Gundlach, C. Gläßer, T. Nussbaumer et al., 2014 The Spirodela polyrhiza genome reveals insights into its neotenous reduction fast growth and aquatic lifestyle. Nat. Commun. 5: ncomms4311." />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=H. Xia;citation_author=L. Camus-Kulandaivelu;citation_author=W. Stephan;citation_author=A. Tellier;citation_author=Z. Zhang;citation_title=Nucleotide diversity patterns of local adaptation at drought-related candidate genes in wild tomatoes.;citation_pages=4144-4154;citation_volume=19;citation_year=2010;citation_doi=10.1111/j.1365-294X.2010.04762.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Xie;citation_author=G. Wu;citation_author=J. Tang;citation_author=R. Luo;citation_author=J. Patterson;citation_title=SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads.;citation_volume=30;citation_year=2014;citation_pmid=24532719;citation_doi=10.1093/bioinformatics/btu077" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. M. Zdobnov;citation_author=R. Apweiler;citation_title=InterProScan - an integration platform for the signature-recognition methods in InterPro;citation_pages=847-848;citation_volume=17;citation_year=2001;citation_issue=9;citation_pmid=11590104;citation_doi=10.1093/bioinformatics/17.9.847" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Zhang;citation_author=K. Kobert;citation_author=T. Flouri;citation_author=A. Stamatakis;citation_title=PEAR: a fast and accurate Illumina Paired-End reAd mergeR.;citation_volume=30;citation_year=2014;citation_pmid=24142950;citation_doi=10.1093/bioinformatics/btt593" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=S. Zhou;citation_author=R. J. Sauvé;citation_author=Z. Liu;citation_author=S. Reddy;citation_author=S. Bhatti;citation_title=Identification of Salt-induced Changes in Leaf and Root Proteomes of the Wild Tomato, Solanum chilense.;citation_pages=288-302;citation_volume=136;citation_year=2011;citation_doi=10.21273/JASHS.136.4.288" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=E. Zuriaga;citation_author=J. Blanca;citation_author=F. Nuez;citation_title=Classification and phylogenetic relationships in Solanum section Lycopersicon based on AFLP and two nuclear gene sequences.;citation_pages=663-678;citation_volume=56;citation_year=2008;citation_doi=10.1007/s10722-008-9392-0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
<meta name="twitter:description" content="Wild tomato species, like Solanum chilense , are important germplasm resources for enhanced biotic and abiotic stress resistance in tomato breeding. S. chilense also serves as a model to study adaptation of plants to drought and the evolution of seed banks. The absence of a well-annotated reference genome in this compulsory outcrossing, very diverse species limits in-depth studies on the genes involved.

We generated ∼134 Gb of DNA and 157 Gb of RNA sequence data for S chilense , which yielded a draft genome with an estimated length of 914 Mb, encoding 25,885 high-confidence predicted gene models, which show homology to known protein-coding genes of other tomato species. Approximately 71% of these gene models are supported by RNA-seq data derived from leaf tissue samples. Benchmarking with Universal Single-Copy Orthologs (BUSCO) analysis of predicted gene models retrieved 93.3% of BUSCO genes. To further verify the genome annotation completeness and accuracy, we manually inspected the NLR resistance gene family and assessed its assembly quality. We find subfamilies of NLRs unique to S. chilense . Synteny analysis suggests significant degree of the gene order conservation between the S. chilense , S. lycopersicum and S. pennellii genomes .

We generated the first genome and transcriptome sequence assemblies for the wild tomato species Solanum chilense and demonstrated their value in comparative genomics analyses. These data are an important resource for studies on adaptation to biotic and abiotic stress in Solanaceae , on evolution of self-incompatibility and for tomato breeding." />
<meta name="og-title" property="og:title" content="The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/12/3933" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Wild tomato species, like Solanum chilense , are important germplasm resources for enhanced biotic and abiotic stress resistance in tomato breeding. S. chilense also serves as a model to study adaptation of plants to drought and the evolution of seed banks. The absence of a well-annotated reference genome in this compulsory outcrossing, very diverse species limits in-depth studies on the genes involved.

We generated ∼134 Gb of DNA and 157 Gb of RNA sequence data for S chilense , which yielded a draft genome with an estimated length of 914 Mb, encoding 25,885 high-confidence predicted gene models, which show homology to known protein-coding genes of other tomato species. Approximately 71% of these gene models are supported by RNA-seq data derived from leaf tissue samples. Benchmarking with Universal Single-Copy Orthologs (BUSCO) analysis of predicted gene models retrieved 93.3% of BUSCO genes. To further verify the genome annotation completeness and accuracy, we manually inspected the NLR resistance gene family and assessed its assembly quality. We find subfamilies of NLRs unique to S. chilense . Synteny analysis suggests significant degree of the gene order conservation between the S. chilense , S. lycopersicum and S. pennellii genomes .

We generated the first genome and transcriptome sequence assemblies for the wild tomato species Solanum chilense and demonstrated their value in comparative genomics analyses. These data are an important resource for studies on adaptation to biotic and abiotic stress in Solanaceae , on evolution of self-incompatibility and for tomato breeding." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/12.cover-source.jpg" />
    <title>The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__49R9wsCpiKJS6e6ZOiXRsC0989bO7FF1XdNTikSRKGM__k1m2vpL1Nu9n43bJnV24EQ0Z_UWo-Iq0zZ2FkQLL-iU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OrDUhUmV-vUNPMVf1PLwr5ATboteJ4lt3BqE4XYgvzc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"494901","apath":"\/ggg\/9\/12\/3933.atom","pisa":"ggg;9\/12\/3933","ac":{"\/ggg\/9\/12\/3933.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/12\/3933.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/12\/3933"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/12\/3933"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/12\/3933":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BkLnYXvMG743UQ1a0iF+M5ycvz\\\\\\\/QDTtNKuTz0pLj3PpUZlBKtNK+xeXv0Su8Cf12TXwQJVKU8jz8qc\\\\\\\/Wsn7uG33+C9ekw+S2qu\\\\\\\/UEHVQ98g\\\\\\\/bFD1BVfTZf3otNNvypTWJsOzgYgXuwfX1v5fsW0fZAfjXDvJKF66LGygJdHUHBq61qeyk9g7WewDFTxKlmrJ+MeEk+ZSU6uwqaPHdwFNuEewyE9O8lv\\\\\\\/dbQ2cSTWn3TMjk\\\\\\\/492zJjz4MvsxI0t2o4SmXtwrZf+a5LdM4phGh1j3lS7uB1MfUc9kdN4VMmRFK8VkjLPUtHjRVA9iU8eciWtH+31Ized3WWbG4lrKVy30GfPFWN0yvH1mAxBqwJTlxNDjJEiRYEDvLntLxdwnBsMGQLZahuvv\\\\\\\/POe1sGwQ5NLsHLsyvNNgnA+6YM6kJFL7XFsfIw+kNV+szkPOQLWQPCPCMauWR3MGggQz+1arsDZtPuAGkJkn4C+ekSS\\\\\\\/Nl4OJnrnJi+a9\\\\\\\/56p8BFvoaVZXBfatp8nQGi9PeX0G9\\\\\\\/zbRFSSeRm7NC51sHYj3Vu\\\\\\\/rOSTA3rG76G8Oc4UExwQ63nhsvHhHZD3zaHniDVjrLJ4aU3Y3JN0+gNtBewlN05mjp+Gs5qT8ueGyofBggzPmvJh6g5qimie\\\\\\\/wYX3SgzGcDJcDq7bMjicZMpq9KmX35YAORfdp33IgG2kWZiEfa1AH1ag9AZD7iiPqYxsUdP\\\\\\\/flqt1JfOFPPxcOchGjY9khRkJd+STYoqHtIlTbXDEZBoaH+9f6wpeYn6CJX8jF79zWRLSaIZ648aqMmRLSrOZG7Ik8v8Bi\\\\\\\/9WMWiEm6ijUmOU6ObAlfxEEiwOjg4BV5\\\\\\\/DDX\\\\\\\/G0fzyWc+9+QCYGYDRRcgePhgvzYU8IlGn2maXX2\\\\\\\/y3jbJf6l9Nnix6NHF5sbOH8tTrus7c0\\\\\\\/h\\\\\\\/oS01e4vdXhNB\\\\\\\/xxtinlLrT6Kwtza5ZzIyjd1kVZaCFgjnBoJR3F9WsMPj6kpTGkSR9BinEjfptAL30rZOUZHEq1mazrEl9kxHPszYav9BcCR321WimYQUhGOjPy2KEEInsWTPXdArFwMCakYa1LU5f8iMVqfMqlHQCE\\\\\\\/63hPiWCgoXJoI2nu38NiKZvorz4kBsZb+3t+U7vHleNNkaQWTvFMUfLs+7PMXkcUz8SAqQjb2jiUislqHKjDiAdbK6RrWGCX7N0fJdLPdAF2ax7A9gu9h1NsarR5UuybBrLZXfdMSjw11N6AT5UV46c6QkHbVhqplE39rHSMFv+pcukCWI8QpLGflmm\\\u0022,\\\u0022iv\\\u0022:\\\u0022NTxin\\\\\\\/hmvU6tKNog6Mk73w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e86637f3427101d1cf19ea9b3a55f873fcc78250312e8ebbf6dbcf1fbe86318a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SDLJDwlwBU4STcZcUHQt\\\\\\\/JMGWkA5RTfmi2n\\\\\\\/nPUvpEGSo1kdroyBkPsm8nDLmwnvO48xk6uBU9wfYojjahGGqyln6GIkaRvPQdRu0i9a09f8ce9FvjH7uRIm0pHhJ+s5SjncH0T4wEZ4nngCoiZm99XOHQFlpes8xOZvdzCV2Ru7ilz2EnPyyRxCuLLMOV+RSr2+t9NE\\\\\\\/tj2Wg9gKRzYTYdWTKQnfLdN+HIaUN1aG0LnRUDSSWg8yj6AheImdj+Egzjlb3CE0AvpEWJNVjQpB680pO1WmyMylsZbUoPBhv10esQeQXycPAi5xlHQWLJDkWi1vzlh5GWtiFTHPgFuhBk1fspAaAylWgb2TzcVGIMjh\\\\\\\/LRxeE+REL6keCRjzD5ggFDCBWYNdEn9UAp2fl92QmpBXjTIlFS7ylkbsBxAeI2MJM4J\\\\\\\/wvIguAFlbIq3\\\\\\\/qEJmawbApBNHJxq98RznADLVewoBLK0PFP7lOShca\\\\\\\/q9H\\\\\\\/DTJYMjuANU9bHkot+yUXsSywbhTkNt4jo3465ZF\\\\\\\/DanSpDclTE+3JkoVFxCBuIMQjBHcOTUFKzG6YWdMZiFK5t7Vr8riHAtoNtUEVq8yTvCSJ4sAxd8HRDuvYrgbMdbyC9Csm1P9FGXXvHkCDXAe32CtldWknyGwHaYAYBc2ADCgnjm6pYwTNweYu15vqbk4REZqzosAbXlg1hAZhGUgiWc4xtxsT2akWo2QJiaZIVjb+1+cfDn7L9OvPn44IqefLlhF3fGO40GoQnACnmQqLvhaolE9z7n+3Lgt7DvGTmwLbHsl4FhB4BkVvXklXL4aeByF5dw635cpJDLQwrQ3SGUy+MlcGlf2yF+Fk8vSeXuicAcYERaP15L2eF772aY+zph0UgSoajvO1tq9CKzG0qjBpcLob\\\\\\\/burrZi3Hzt8W7L6neDWuFk4jKK+ue67LRLFXYZgbBsPFhlVitbZwvlPOYhF7ocydjWSvdgn5Juxk4sGPmOEkQVP05QSqpXbKO3goF3fNDMvi9WvifsZrjGDrDPcivAIxoRSfDvWpxpMVG9mlkDQSets0Rhc6+\\\\\\\/PSa\\\\\\\/HL7lqn4EIc6VnUyLbr3Os3WPdCCluXh3UMrdHQmZ3re9yFIgBKF0R+XytmKZgsOM9HPEZIreB1u74z2qbbRSD0w1gr00Cnz4cpIy7GD7eVrGxZTKzC555SCyxac46ituICrlxYi3RzyfT4RFi3AlAV1mJxJxOcJSqKl7j35XE55uokxQuzpm7lBZu6MBibDRnNad0ttNApyUXEwqzIe3ce4X35ySBWaFB6HM8EyOJc1lDO5QKqLoOD4usygXV30ja2tTMBqoyEUk+y81pTFgPr9mnGYVtMg4q34Z9tEXrT4ypzH9fP2x4pcERka1Wl928DSF451N0omiH8nRi3J+ENlSak1ucWHykC1kRWVvxd70VF0WxF8XRcptrZx8Vr6Gj4YQNpvnYpHNoR1p+B7X5uX2ZEBPi0qNYnYt\\\\\\\/e4ewCtjldd\\\\\\\/cH\\\\\\\/tS6wxq8Dta7eO69KLat6EjbGLGrixu4SVNkaGsS9vJ\\\\\\\/A5l5xMDjiAogzG0Jr8cOrBh3RQTD6H6N\\\\\\\/uV4YBFeSGzZdK5lDXvmoZERnjA0+AH6LC\\\\\\\/SVktG2h3QLCxLu4x7Zt2tIuQSVYjHwsTu7z8vQ\\\\\\\/oXCzDVh4NDWo45OMT6Xqp3A1QwV3OT4JLVO+OoH0Fvi6dodIdqITHw0kQ8MZNny5pI6M9X8v5SA1uLCZKEZbNxY\\\\\\\/7xm\\\\\\\/YzpX7YgaY0MxnA\\\\\\\/EGp9YcXeKz+2AbzgqTFn1zeVOVquRTK3OT1dp7RoDyXk5RfOAptH0ea9i0Dc1ISUMBus8\\\\\\\/45GYMgDJsHbbilkZLdEO1GC6mBKvoVeNok3rq8c4aCHhgH71zI5JaoWimVgztTAlsNIZPnnkuI9MRUzNH\\\\\\\/w0p99HP13Sp3+aiHNMaK88reHHc5Zo2u\\\\\\\/3mSZTUv0uSPImPOtzeeCBlH3iQcDnMJ9L2ruD\\\\\\\/YRR3X99NA+aXEFe2cAb1pbtJJxAMKoXD4tHh05sJmfvyfyp7Ti7TzQe5PDWbLHqI+V1NC2tQ1qJmoWHo299VOxlLjyR0YeVhV1xYDrDjfhnzJQPBTXc\\\\\\\/xULd5osZEwLX82bvPZA\\\\\\\/IbM3MWEY0nx7Sb2j0gR\\\\\\\/sgCG11oR\\\\\\\/8dXw0OaQ2IOmB2OoS0Qmy8H1r3DS3ony5YSl4+vF3SwXOJLsd5ZRT9Sxgs35ATl1ihPKFgpPgu1Ztjh0hZKG4nTuYY3UI61v3hHmAiD6p+BpIu8vviH7jbaCSeSNC5YRiWoHxv9in6sY7KCEUYi2Zd4MeqQa8tGPgDRzfttD10esi5LWzvAPgBh+WObWn8mt\\\\\\\/D3H9jn+0oNmx9gvnG7XkqqejQsfs96dB9Ez4a1Vip3TSghIODtuiVuY2K4ibT29iEkOfCZ4QnnZlMnXsnEBsT+SRoU\\\\\\\/BoNOLd9oQKjpV1kEmc\\\\\\\/xGBTFpj+cL9ncmdvHR89ZDhipK7oJsAICPgDhhg7fOGPgrrwoBH67HvhowOdEmw+7CBVESNY8sWgdEfFNM+wJuI53UTdbJJGg3DYuSa18ePv8oTRweOvmG9YH+crVb9uYhxVQb5maDExeIPfU9Wfzg0Z9\\\\\\\/NQSq+GdbLAr9eCuA4KfurkNO2ZMN5xJ29ePL8YWh69lNlnAqUpCtYa2\\\\\\\/8B3XOf\\\\\\\/GkhW\\\\\\\/zsUN31fH9FyLzR+U+s8jzy23LUrmaMebyHzSpEtZZn59jl7ajMbhyq5XJDfqvNMqrlRkE7oA05U+8yojHm8rd4bOxhnQC0k86FxjnP3qrri4VIjHPgunwNvOYPhuB87pFYXJ3T8dD6FhF2NewTF1mXQzlhCLW+qheJJiXx68kqidW3IWqeHqoDTbEqEDmcIQ1Va+VzXbaKlGSmlIzAtQVqQPrgy6ykFfzMcoV0TyQ7ma\\\\\\\/f5sWfZUuHT9yQs2Mm2CWTP5EkEO2bB8P4KLF6HsYU3WcEdvhTzAWxfnrq\\\\\\\/uTa0hrMsUCS53jbygi1NYeY0VCsbgKEeYAtRcGsgWOotldSzXjfkBeC0q1tJiAw+9etiUxThJd0oZtU3k2IU42w2bF1vMQVoBu0TV39g4oECWVDWGr8VzNPQG0AGG0cegBB2J\\\\\\\/WDQCUq9TDfGQuQf0W\\\\\\\/6RZHQBmEDmLbZt\\\\\\\/ol4lcEaGHWzMWD4f06F4kzmh6Q4qCg+78PgX8mwN2LWsxdJfhOzFgXStKIF4wOIboMsMllh+E+eK\\\\\\\/TdufTyOmFgjWGH9h56Db6l4Q5S02On7E1qOAwOKMcxs7DtP+RzwtYVShwM7MnxYNMKzauwgMqWE69eI4H6TjzwjUOSnu\\\\\\\/e3JOWStNDixVuEtWD4EUHZ5Zi\\\\\\\/YVq+Yn1S4biR8+w3JzsIednq1LmUX6zTt3NUQTRslkqkdA\\\\\\\/BEbuk+3oX9PkEL20zg2J3R+RlN6ua8Vdjj8br11VC9eaCbTH4DwPi58L5Wzd1QkfLtSNUU5PcM0g3nyeMWcz0MQM0BhpwLk564sf6RnxKZBfEX3GVGocPgtQJ2faYkJSUnx9fpwNYFhGx5AgWa7nrSmGbpGLoRNzatDNsQh42r8QEBodfYgOZTsyDjUqpdFBESfQJpByLrAq\\\\\\\/LSmc7bqWSXS2IX0pa00b+heuT+nXFfPehDTJSDBFKzopJZSQAX1557xEvTYwU8aKGkMDfydlECc3GQ4ZUmgw+KIzNKBMRk9yYO6n+CeWPBDWVVIvJjh8yPZSbC9e04yTw18yvrZzt8TZN5wgV\\\\\\\/fl5R\\\\\\\/0SBHnn\\\\\\\/OM6iA\\\\\\\/1KJ6KjRBCZkIWldMoVKg1Zdh6zkpvL\\\\\\\/I36WuPUjTtlU43VyaofDacJQDzjvyggP0d0VhAfDfqm3rWzaxbfFdsl0wea9SEXj2OCklZBcGk3ttnzW+TVWuiXncXcAmk5ADgKySkWmMCWmWCDgkdjbcgJ3GQNGxdiKhEx5pzDRO1oz2wKK1nqtVS+JK5uPZUJTi3IOmrMfKA0YNOURT4Jrl1QxvvGJneW8GVZV17UpDos18pP5cdSNl6CbBuZv21gIYepGoSQIVYg\\\\\\\/xP2wCdulvPtk9wdZgKgtlGzjleRmpj0x3KDr9lJ6VMfdLTnnGi4EWaMzXmKgkcVNw+3w544k8x8buGCygvtvg1KkoohYLkbj2tm+kcTA7y4Ocelzca8zbkeZnEN7JqMfhIy234hYmbFFhclE7ptzyQ1n9pI85BajoDERJeHM5f64+\\\\\\\/N5eRwl1uUuReUsMuG10ktGoMdzJnpH\\\\\\\/+B9\\\\\\\/buC0MO+Z428OZVNpBHkqqL5XW2WOUcvE\\\\\\\/P9MVI0TyTVT0C0rhgeyiI29gLfLq1q5yL39eV\\\\\\\/pZ2tdLQbQFoqJ0vw8QcI5Acye1VnjuXSRXQplEaEaB2K\\\\\\\/BIzNs\\\\\\\/0UCyRKnBa39X7VT77L5r200u9r5dJZFDrPkK\\\\\\\/t9YXEdudc25X3Psuct\\\\\\\/DysQIm3DssJhjeWnozh8ZlkcQvsRhMS3nXfRLbYKi+fT1AwL\\\\\\\/0CloWC2YwB4bK\\\\\\\/r4U7GP1L1TE9PC73pPYblXga+d7CFI8Pe8GLxL0LgB8abRPis1WW8M55oiEI8gJslH8uH7aQKRkllwKrU8iwrvleIaUZDVa10zmgmBNIAkN9Zp+oUU9wevrY2C6yIb2vPeKGwCokaWO3gjcizP\\\\\\\/Tby4\\\\\\\/vQ966YJ+yM5qL3FWaG8ca3qi1YPZwkXNEY6\\\\\\\/wT7tbsNbW\\\\\\\/V28sxo8shsWh+hquBPYsEsDRJBhr0W0Rhr13PN34J7MoGHpawm5eR0Dn8qDtAIdxaA5vl\\\\\\\/2+z95dMEpJPsD0dnDsMyjq2bM\\\\\\\/CZgUNhaZMTdOX970lpuCazIUWwNaIdNyeSgx+k3zACWn+XKCUW6k0MKhKl9vEqbT0rJF0LEtBkqduw0ImuX7lQYnDez8lwjWFGwRWHsABUnHYCe9fg5tO2Nx9XLGuE+sKBuUZsilI5TYgSPcLAMmlE4niVFOAw66tMU5EMXrtDeo5Blv7CVPG8+7R1qgziZZ+GAtGkkCWZz4XD3khHv+HZox\\\\\\\/lU3uypFA+IhOQjKMNA4CLz\\\\\\\/pnyZNCVQOqv8SG6Rqllr9GDCRyPZ6G4EHYtzaPlhVTM6w6nnMO2BT\\\\\\\/vvuUp\\\\\\\/rVNdKGcCI3nOyscUg5CvrIIoSY0A78nW8XIpU3\\\\\\\/gmOdLuxNcoSDMFgQrBKTew7RzZjfHzsgw5qSWS5xKlzHYlj\\\\\\\/cRgslhu3ZMnuio7TqA6R5UZnfANpLDBHQF7LpbfJGfYGb2GSvbqTMQm3YanzrcNkeVV8hlgUUr95EL7w956ba7g+PmUyAxGmAf+YFv2GENMuNT\\\\\\\/Yhk0WJIIqxzKU0KdQDAXhV0CDF7uv\\\\\\\/7CdfhutnB9nnD6KZj4WXIQOrvjZEXAyNKuirbQWyh7U057CDTUiDOqnnU3H8ilxspV\\\\\\\/0mGVc8+5mqRn7UwqUUnjiy\\\\\\\/m6xYw+VwwZYkusiVf7eYGqFb3Req\\\\\\\/YNDlTygqa8QKdk6sAcZpe9pxBTY3GG42fT3MZrL4Qw8UuPm6+s7kwM4HvZrUE1XYb\\\\\\\/5dW23BwtX\\\\\\\/\\\\\\\/0szxYEKwknbNE3jBB7hE\\\\\\\/3QUziVUlM24xV7rmUD3y4qWfnQj1Fn3Q9r\\\\\\\/bHhC7HjJ70gAylo7xx8HoE2HwTvzyBpNijdQZlhsyxwKm2MtgLjrRpIZeEmjXrQl2w06vQy56tXtDkLlShMf5JfLplgGU5fpU3C97hNcJfhsjoPoKXbrqi7WoqZpXMOHJYzNjhKpdOT5D9UCw3rB5VbQJ3NLCgLQE2b0kc3+1gfRtTdGskOg\\\\\\\/un1Efrj9XIvkOA52sBqvMde+Rvvo6hyOC7rH9rjzd5eclxNn4t2FmBPoKjntA0pdu0JwRXeSz8Rx+Vm8lSNMeznkaBqspS9yAtyO7j2LOfcr+kWeIrsT\\\\\\\/MHsRC\\\\\\\/nTREcHoY7ZI6SBfHVAqc9dK+7fdigfHYJiOFNr7dj94RSEKTpevUbTXI4pJvydjlMsWkK2tNSOW3gu8jX9Vsk4JtFMQvh2ac8D6SrOgvDq2qGWSc3uWdpOCWnx3nP3vx2wbZaTkiNuH2UK+uMy5O50vgGrHcQDYDyfHoal7f59iHPfmXHAF7u3k3Q5IZSX33D4GcJyfNr3LetlTtXfTqNPOqlTznD+oWk7SN842r\\\\\\\/jev\\\\\\\/bQMvV9cIfL4b1UVuOd0ZVNwAr2VKzsnWExM2DuqjZnoNBrFXI8gaGtMFJMV54fe5Klb3ta1ZiTJuJsA4dsn7fWp7vQ1x7jtxjk7lW5ExNlPzypV0epHkWvYzN6m0SalDASkS5d\\\\\\\/vlH+ft\\\\\\\/kSOQR7ScX6kmun7\\\\\\\/O1h0jOrQbZNwhgDUt0dUchlwrtoQh6dwACXnjWqGqFWDOZpwYMyXkrXpOjyXNWduoN6HTkOkfHqTMVnVZiGClZrVQIBHoMpU4tCvHhHn4DBs+K9ptri8C6JE7AxuXb2Vh2QwoJRpKUIrxNoIJgALPOTPoeRS2eI6Wn6TdKNpmki4HNWvzvmrutFXHA32cz+5UsmuKMg8jrxOTCNyEC7O23RSdK8fQRKBSIqeT7355J1lwgoBkd\\\\\\\/h0wOGAeKdE4wpfKsUHCRPnxwrRaXOLGSwKzwtdPPNonLHgQkq6eTD7N9DnNN6XSU\\\\\\\/1yHohvWZjmEVRHFP6\\\\\\\/AhHy+Zu8PoVq85Ly1OgmCgBD7Mu9z+KGJNXoLatj1E5TWQELYmd7vrvsoMPKY\\\\\\\/p52vYhAlmm2EyjS68MhiNjr0H3tMs2mVwqDnCc+mb5uQh1asMwKGf2PyISUss8sNxAGxZhtKJ1lBnXzSDEMatsxMD7oWpmJsNEnJW9HUzoo771zTv+7tjklj3NyH9ZC4Ec3AQI9RgKcOfnityAa65YiEee61z2BULFa6twobAeNLCs4pSnpKE68TcJfmVmdu3ptfwryNcGpoGUgJmRtUjMdqDqnGU7QrIGhJq4QckIDO4YuhP\\\\\\\/WdEbUyikQLHHgRpvIbi61TXd+CjhUF\\\\\\\/psWkERoZ\\\\\\\/ldErf8D6G+Yl1rBQEfgZvxMyKQr7IsCZlR08L03I2ne\\\\\\\/OxCQ+t2Y+oxnJeVz7GKjtRF\\\\\\\/fVfT25PSjxWFeKse3qLi9GBtVmpagdOqUYtxCULt3l47BzLjS2\\\\\\\/1IS2PLEnKiYCOAMfPLqo06byRPjDsRX6iO\\\\\\\/j6YrgI4HlveD0J6SbEctW\\\\\\\/Fhfsi9F6LYYBvBuGfzhM6\\\\\\\/Lmf34ahgPUbDrxA2AvgLaW5ugWDiVhy4SXX2IEfKi0DKosBkvLW5J9tPDw0+3y9Q807\\\\\\\/uvps91Gu19e+c1CMT2Tx0FuQvWZR4AzUX82nygpyoWcPzDdD+gjjMpXSk4LfA6puhaW7+3NYvmIwE\\\\\\\/JacRRJRIEpGE1p97x\\\\\\\/sZR+o8MWmQKkp+RmXErIOmhwES8zrlpFgdUPKlmPHwexMZNBQeVM8DXflCi4hzexY9\\\\\\\/9\\\\\\\/zU11JgyjjzH9iNwWx2hKYZ9UzNN3d2LxhJahQWu8xG+T+HJFMtF6pK51PduSDgbSg5BiWta1+wzK0jyA9CE6Qf53hoW97mDHRRk7+xptBnbZ9INghHtgM0FR7uoJ4Y8F2lG80UP91NTbYt2N73EVgwLGvUcCRM4YHhxoOaj5Rm1KwZHSJTRxc3GkflSWFJWu8IMdGmU+n2uqlKJnXYaynoMKNRJXGiNCE1wmRKG+2taJrkcH3XvdQ5MtL30QTl+2\\\\\\\/zXNu3ZB6F2FLeqUapMSGj03X\\\\\\\/7IKs49EcXnB11ePYiz9BMBN9WnL7crAqF48BHUsWNEbwq6NNavZGBWLoHOsrKyQXQxk97qLK1eqxje8REBxESyXFb1KIC09mYJKnmLrny1NZYeu47MEdCgkgq\\\\\\\/Is07ZmjKAr947Nn97ktjnXollT+NYFEPq2xOLLv1Gl5gvm7q+oLoiTKtIjvttLBBZ7TLk\\\\\\\/VMh3mjQnBFm3FAwWMYkhdlTmGSv4MXLF6H7EaRrDVIT5MblCxzQVrzpOcMzqoiPIVr\\\\\\\/li0ucgVDB5Za2P6WOimQWtWPL34S+bsLuD\\\\\\\/87jBzmQkNODMtO1UNWj6FkDCgXCR0BLkEM84YmhR+va6KM5K1caGZHBxJLatAugcmHZqUzqNDvokBdyv2BCGdV78UVCuv0hggWPBO9yhRMrirJr9KicID0pQI4BlcFuT25hXMUV2Qp8LG8iO5VIoKL2xZWcigIKjfyE5pdQppAAry5wbnMFSETsDuo\\\\\\\/GD0aOb0ZAx+EAGAYlhCSNqLYCZKMlxWfkLONsSDvnPabrnilJRnG59+SBgaqCfZi50AyqAFDydudEeWHm3h+vOUqjzjQlnk0qKI7SffJAS0QRPmRyCJkj4wmb8L1As8Eub5JRyean8+ZtdUDHIG5jyWnd\\\\\\\/hFpH2ezu5M+tsWpAhRIO\\\\\\\/BQ1UOYYJnu6Yh6c7slc6DgL9MdDJTPXAN58lcFwhomKmhDIi5j8bw9VV+AK9aAf\\\\\\\/YM\\\\\\\/d6xrHIkfQfdxFZ\\\\\\\/wxvhhEJso9A7M9nxiJUT56F0IU8OHLTJdnfiOUYXS2ocioT1X5g+1SW1JwL2qM3QD8YCVRV821SDIiELs9fPufHQsr2uHKzXFFptURtRYCAIwjVhQKuhaboFsbjs\\\\\\\/QEGzzotEzXjyaw2p8U\\\\\\\/0zQ9GmVlKDW07eDtHpJRBaX+elxvnIX3xyzcZY+m6WX3+yTXMGI6FyGaeQP2nsMf6\\\\\\\/dfCo1clDAraK\\\\\\\/ovHc4nesRAtPj8tjmAX+RLLk8Ff9FUdv3nbSy0xVxbMlgsqsSB9DDrfqpJUfXJB8RpMMz921brvEFnq1KvE9MgzFHrVy0zP+xroR2BsP73\\\\\\\/rDWhJmr4beu\\\\\\\/+xw0H8Q08h5WkNX46VqY1rwlqUVsMQvRPKslfZ9WT0SuvH9\\\\\\\/v7otmC3pQHKKw9kUowx1980UbqIBSPBft97\\\\\\\/gzHHL9WqJpfLJlVSvcROfuAidd883Vf+9SUb2APhqVOTq+QTyESM7RCMyg8fz0r7Ge7CMzcUEU2Zpzhf8PnAOdgOcArWrg51WmlXmjM0LiAWSx4nkQfSJ0xMm4lP2kHPZIg\\\\\\\/ZwLkhnAqkhyQIaP6G+mxxx4bG1B6Wovk9SUs7Keb5DQULAABqbs2JAqMjFNiEuQRxqLaEB0\\\\\\\/PR\\\\\\\/XGqCG6JD6HMxUX\\\\\\\/mW08MtNlzEaPaU72vRpqcmTJ+a57xbQwSG723OUuVaF36mXfUDsD2cbWjH9WIgwMEcw8aG92aG4+GQT3JDgXYHYhFYpUepsfoztCM\\\\\\\/GkD\\\\\\\/NjFCkTWDoyvUBPE5lSWyjFYxVrEi05jCsTBl5eA4+1DP\\\\\\\/t2BjIrhPsSd4qIxW6E5hk9hn4nnfG7a9NBr9GpIruLmTtu4CXhiQx0KP5XkOVnECZ\\\\\\\/D0n52QwbIf6fGdbKiyBr1eEZw+kExToU7DShVShrNm1k9ag2ClNn25Fky7yaf9p0GWTmmH7kkBBfqGp1zXWBS66c+8CiIe49+8P+T3cOtiQGf6hEk9ipqG0vxn+5o7AxZWna\\\\\\\/+JMoF8dwMHaynGdaWF0QYzEBV97Ddk1MPVelDP99hyUYDe6LnA9QqbJYwaZGwfAqDwSZ1WKdNPzc2NN61F5h1Cv7pbWQeFYU5BU4DZfcGg5JUUuFIphb\\\\\\\/yi0wgulzNjyI3ZuqYdhHskUT0AoDSfSg+bYOv86gIobKN+CzFdF1CHvNgGgIebO7fjiuqpy6QxUHAFZnakzjj2HHHodxap4hYoL8OOOW9boxdDpOGFT3IozT3Sh2aWlvX2zx4KRH4I2yoWotZNn43TxqGdgE+2UqJB4Tmf0iaoOalS9wYtNfUKIZU0UDoZ7\\\\\\\/irOLoNbGXDdQFjG6dm\\\\\\\/7SJ+Ls\\\\\\\/L04GX2\\\\\\\/zLKAXkrBZekKB3mKBDfQW\\\\\\\/w4di48C+AWeD9NfSxi6dvn7NyrlpJ0sZrNVWxLX31fAkQEjSVgXp477MLQHJ\\\\\\\/nzbqctPxaHrdbrArzHSZKGoYoVnWv+YcOnbIxxRfKq4v3nZeWQzyemOSI37bqKDHqEz32rYEw879BwqNikuyYpKQlY8bqCx6p2yoPqm10Yicq23uLFdnqm5WKc2HxwmeIp4Va61ogmE4Vdy1TsGNBZQF7SXCtJGCXTAUMPuXjpR1Lvc6UDVgaKBOHGkhslbziExLAaDuy9WfkqEFq43eCG97CZKf\\\\\\\/74sZWb\\\\\\\/3cuYlzMauTU3IUmpyVF5RVgn2YHnpK3Tm6wRPSUW4QQBZxwzdE6cHymyV7peE97TWQiCqyc+5OfSeqZMbkAFmPv7W00\\\\\\\/PuKr2+o\\\\\\\/tvklxTXZP2frR4euzyXbIOilmMTKrT5ljkjZ8BFX74Po9U3rkfXkXVNGbe3oSDNsLf4U88kZp+IDlVwvsyCj0EabDZemdfxW4aO0Sry3MOdWKX5\\\\\\\/lveHcQVC7DbjyZW8WQysfOwd41PT7J3ZAtiEybYxmRehOgbta6bi5SY3Jh\\\\\\\/t\\\\\\\/\\\\\\\/aF5dgmpcBjFYJZFZf9YguHBfIMuliOgT1E5yTCLxvfslwYafs\\\\\\\/6eCugmbDGb10fd2OrD\\\\\\\/zfwtNQPVg29OIPJ3CWhVy3FN9HOKLgXOTkJvAaHRzOSiiuMB0NqTVCbTEooeOmFFVJcGxQST1+vwczdQJmM480SbL29c\\\\\\\/HPpefu55DGIO3geqwSO\\\\\\\/dj7s1tvTKed0vPdO\\\\\\\/pKXhmm8YKOX1GVk05eKJ+P+w63O27ZiS\\\\\\\/L+9XzgaJ68lbOy0KWaBecXUeTPRlFs3mC3UvbRe21kMfEuff76Cnxvz96fl3JqiMSwKnswJ+QJWwCM9J7le9cFum1+b5fzMUDoKTMhG0vVAkK65sO+np33\\\\\\\/W6o3hiNSEeX7Pfus0LAbT6Y8+b8dpZ9bDqZVf8pgvA\\\\\\\/1n+exOth6OyvlAj+lpTU30kB1p1AbyLD8xTieCLqzi1giR133Z1VjNXRdOB6Mn6NXgu93pptpcM8iVAIzptNJfEww9kWyu4wsupSKVilZevpEMvMv+b\\\\\\\/YVAqvvAU6tKyRelMnkiwGgYjiNjLdLfncIREP9mMkY8dw2mxCLnLqrHPsLatbLuD7wJz4tVRIh8faPscVHs6bTNKX6i4NmyjW+qVg2hNdfSfIbotexKCL4AdBsJI8xvkwcVakeGURS9C9aK8B4b1WG12AbNhmzQx0zqajpb\\\\\\\/fFdDcI3NH\\\\\\\/ZGYPBPgZhM+VrwsZZ5DZrSnxOUXUu4Q3TQ3DPMm3pouLB9eu\\\\\\\/kehvIOUUTjqSdFd8kBpDeo32ngFrjan1yxiw7dLrAqf+wblWZO9qiUdJm8XpOu+ka1UDvqKEQrdVW1f8CnTNOZNsE\\\\\\\/Z6pe22fmkvYHtHJiVKP40WJuO98nEQJpeexHayRB8u\\\\\\\/yzv51iis478m4QvmsSY25FkCchk9ZgbOoc5SHz21P98iwboMjHVxksG6D7SxZqgAQQpX3uFoboWP9VX\\\\\\\/P7tTCrEVBLnjienctrvDMEpFw7Q5BjlZNf0soCDJSELD6ArgHRZ\\\\\\\/+CnuC5HlCyz1mszqfXNE1pzIs1UyAP34TwDBMEXosimzaGYZCMw99wgZLFH4t6ipbsmUkIWhLBxe+6UkFzehxs+NW8BCkrpBMVRW\\\\\\\/4oqFFmfbekIUCMAtv2Fg2NIHDWstgHxyZItVN0hKZMdG7wW5KqT0yOeXeM0o5drmkwfaK93M6H9IbuSIVqjWcC1XohuHGb23XVgDQ0hJq1BT1ivdw4oUPsntuWWFKrw2XXiV5f9CIzEhvCp+yE5wbw7vLGtVoMF4yJQ0ta6C1VWwywV3t1bdOxHANxkFslYMwTvzsYgfiliLvPctf6AIEBpsP6\\\\\\\/DbhgGT7YodeCVjfSx5Q5Rjz2JUiqg+Gf6esI3uhsSafS+aSWfF8dtikoi\\\\\\\/JanuqDgrI4GHESxAZEZES1g\\\\\\\/2\\\\\\\/sxJYYHRGkStHWpIFw1T64Hmjx8UP3Hf\\\\\\\/bHSAeRE6w7eTSFzR5vqluwR\\\\\\\/JLmvirRoXjPg4BTcgiFoWdvMHL0J0lVRTmB60KalnZlw168KiWOAYltmOFk16BhFmfpaj4A3AzFWm6JT5ckm\\\\\\\/J\\\\\\\/JAc36aTUulEEmz+FeQiADzx3E5aSjZ6SFqRAPMjD6ikAUGDWYoSJ92pv\\\\\\\/c900CXAdMBzIpqWqGkWSNjEDtf6r+D6nLjuwFDnvFHa4I83iFoAFFlVkwLMLy2G0QwM0nfG4wGxRQ5N+ac1DIWW0r3zdR2rbtefHzFbCs5uk4fwxndXXo9t35AJz26x4D8eSKnbIq55WYXeaImjuIGD0coItnsSLxpedjjpZPPMR7Agmh8Uzn9Hf3H923vZp3GXsMPnpAL3R+O0F9+\\\\\\\/2KLX\\\\\\\/UBu+FeTwjF61hlG69RAeIXPb7JTnuyY9EZHjqnKyDMYyuYD1R7R4t\\\\\\\/eEQY932vXzas2hspWr7d1BRhkjajrUQ0sFL7P63c4rQkuEA3TGvfQ15nbK\\\\\\\/2DvXuXmlWahaKAKHGJWd90bWHPhT2vv\\\\\\\/d2InL28DDZ+oJw\\\\\\\/UQodkuV7Mse0cf7ibXogrASTImrauPu4GbsDCr9uai0pFtj5t1Lv2NO2n9x32ilcPBmxDxCe36l\\\\\\\/mXOdKXGyI0woYes8TkHarHEbdz\\\\\\\/EwfWT9fTPdQWnoYGzoV7jZsiPqJk4R66G8fbGZG9oz2ePSL9dwkJcQRFJX+GuzCg+JBHZl1xdIf\\\\\\\/6ycPaqhnptCf3qpCnwa0daU7UW9PUW+wdR4zLxh966bDuJtgfvOThe3JhAK7nNym6dP\\\\\\\/r2xm3vbTG4xx7ZuW6zAWbcDY2+UGUt9jAxj7z206QpXs55XigmFpIHtCto0vFzFANq2WuSWNDnxluRtfGpCn6CJZj\\\\\\\/gjnlb\\\\\\\/yDhXHCH\\\\\\\/QBgYjvfwpYs6XF91Q3zXdAsTycVcmO256\\\\\\\/7e+2fVob3sXjIU3bLNzN05nZp00Av5IrweAwcH8nidyMVnUCpAPruVWTMEoo6FTND7SrsmqpfT6jGvnP0kxCs2bggbVrnksKZMDjKGWuW5sW7rIftYJE+UnFtXwVYJeWvzX1xdrzuh\\\\\\\/nbYfrkQVDjuf1y\\\\\\\/fbiTnDr5vEHq+6J+prFxIBE0e+uW5Pf8SFQ2ddDPlVyKNLct\\\\\\\/2LUUmsUmPrKUtjAisayN4YanqCYB7fiSPTPDi9lVSG8ODFe+CRha4ATBYRYk58TXLOFuD+L+W7ha8dAsCuNPIMr3uJFsKz0gembsmbAJYBk1fj1poLQxJTQgsF2J9wZNrXdsCiPzfZUmsokD9g73n3PwebEKv4Plqefy907OjieNV\\\\\\\/7Ixm69YEtSZAnJjv+oLXdlyHLP3WlDoANcihHB9OUsCxkS5UdqQDyHy\\\\\\\/hYM4egSbok9u43i13he2zOm6M2D0I+HWnoS4aaOg3GgtJsteJPko2s\\\\\\\/mxMCPe3nBpksi6ITJx\\\\\\\/J37KXL0euoWKRBs9Q6OzAXlbRwtADqVj5OZDYftmco2pPIAHWAeTW22qauD6zzVkB5xpXYcVhQbhGrJlgYpfpn7g62lyjXDiHme2AGeXfYF0Bmne7J50sWfAauQ19HUkkcMdFCpzGfLUoYB3L0Z6g68WT81TRfj\\\\\\\/YtYpqD7i5d8KXzFofOSkXoPpE5YZAwNxjLzTyfSFfbXiBwiHAYSej5yqKTUW+J5vFLR\\\\\\\/2mOotjtv0jOlSFScNgWw0Pn6\\\\\\\/3AddHer5KT\\\\\\\/1f4aIJ6oMSqF2JGh5yDqy5TNOe8bONoaw+KG0l2uFaUkxmEiCykW7Nza2Ta4+gTZ9FxIY\\\\\\\/4oV7qIJM9QTAIszeLRtDPdlOHmK6EaIl7s6h8WoI8JzJSBUI3X3FkMscpL03y6so\\\\\\\/8IDZ9FO90v+qJu2eUbXBfITj6Wx1OWiCarqr8wGryQ3zlMjHa6VyTD8Ln0NgYrkF\\\\\\\/0IgpLyVj+f88mhlzYBp8DmjtFVUH0OkLmBn3JW97SAlYVNKkWyxrgQWyQ3W5RrdqAiwmXRJ3jDGziXMqI5Q0K4kSeSdtmzBDGdTdqnXjC3ZIY0kojAcm7xfNSVGM8KWOfd6HRyRgddLRIvT+J7QTpVNT000w56KbZRzQmDN0zccSO4VJBTY1s7oumwa9L\\\\\\\/CleJfemCz\\\\\\\/p5dLtcRW+medLUHYHr+ElF+p8VNc4+nC90sU5hHq\\\\\\\/wr40MwCqsKbXkywMuPZWpDrlIBQS4EytBIAwnyfYeteTusNmKYR0VX7RNttc7mKbRlmGMDoeZSiaKxU\\\\\\\/NAVFcpyZiymaab4XR4BCC+I76EutmDehdNoD2ao8Gj+\\\\\\\/pCDhjPOMjoPKX6DGYogscIL03eQGPgfc5gL+v7P+E53dNnV2FZSEwB8txPQ6bbVDI78wJUmV5UaxgFWuizqw0NkmOnlbXky6iylT4GH3BLsoCPfHrDBJcXuua6LbqCDWB9l77RiTwJRJmZxb4M9rXnYJ9FUiX2IeN7CEF+TjA5BazcN33zFAJu7hECUBdJnrhUpQEJE8Dt6dTHOC23Hh6CIvJHrDjRim\\\\\\\/DO9o7cGdn31K8IolMElyX\\\\\\\/NexaCB2LSWx1DK+6KIZIinri7+WHDdY\\\\\\\/40CG7pte3D72ctUARRpE95PwMgHSmegCgqA0RpOyxjGDHsQURS1f5Gr3fTe4ul3ZTlXtZbyal6OPw+DR3ktfIxbDM8C52h23QwtRSi9uSDL58Ddwu\\\\\\\/x65oZizBci5\\\\\\\/bcVxAuG5dvrkfoR3lJ1VfZ\\\\\\\/uz7FmaWQApibcKGCPPoty7HtftwrHkhT\\\\\\\/8U9Ly+3pOyks7PNcLPaMLCKY8CW9NcWgUtKmIgtweIzBWAMM4LxhHLyNKv0sJc5iifnArpCjrbE\\\\\\\/3qt5+1o+WCgrxWdbr9COrJ7qCWjmcaup1liDicDNErOpXef8ZBm47NikaiwbyfVV3czsqmH7sgIvVgmZZSP4dXAOD\\\\\\\/jSR07A52C\\\\\\\/TligfP7Z6kS2huwSP5YVKT1K2TJhjesAMFuTwJzoG0Q+943gtaLvtbra1NjuH6P6gFGmBSVMxexzv61bUQQEmb0bRy1048qnisarC4hg\\\\\\\/YKAA2XzgSzruoYzSo5WSBM9pyw5yYyBUoyWKxYz5IlVkXyV91P\\\\\\\/8sDIfTioQ1KOzTyKq06Eiii7JorQBNLuzJOjGtghb+jLJFcp2MUl2hBzlgDJJfz0\\\\\\\/JJwVzFfXXkpSodtRtrfBAciBD\\\\\\\/684eUFI4cPOhaeoKjJ5F7qwzK0YxGMHEPUvW5BbYCxCAJXVA73p0ugfvdEBuXCYQJhCat6PGEO9C3yW530BGrRYEVM5RpEndv\\\\\\\/8wMsBo1viUdtiP5N76GnRLxrJoIbk7cvE65VUOJXtop\\\\\\\/XP7ph1wYiHd8JsdmVjSHXnLBmGpV4THPz2I3BIaCRTFPs8A3w08QX6rpq4eSetK6qZYamd\\\\\\\/SnJQPHxuUjLwAUhAjxp3FN7kUsHcPsgIDZaVFh0OGUEOdy3f5NvVnKNwzKhGc8K7Ya+k9chvgybAnHh9QeCNlXKH3ysTrouiYXDmc7hASrWzWcLg4O24D8+KuUlUAnjmnXUfSHLULWj5f\\\\\\\/Iggv3SkzDnjCObYWgzuxkluk4cnR7RxPVfT8JtJPNh2pwJoMYYvCHQYLOLhAB9x1LgF02\\\\\\\/wfao0fYRjEkVqBMC0O2nelBf1pspz+UM1Pae6BJLlfKLQLNaspIGWM6UNmu4mmswRXrhAxx22GwO5FesX3YBmkb0HaAwZ\\\\\\\/i\\\\\\\/MO7gY8J1ixzJHWTtREH7CUrsXSeX7rmNgE6cxSZqII54OuQDZhM3QuY4rDD8MCPpKxU8H9beiMy9mS7vD3ssHuC5pdqnR0yyM4UhrfLeh2kiAVC9FFiFfHObeoLstGHMjESOIiy3S3Sjinxd6CrMKwN6\\\\\\\/ofSzwozoo9t8s1zSlV0T5Cl\\\\\\\/gR6PbzzT07qmR13Yjl+GGMavh\\\\\\\/1f\\\\\\\/yqO8lh23M9pUnsjjoJr95cpLZIDh2iSKhzYLEjVrI9BgRwGH2wxidnY5Qw+yJlQKgREBD5vCY+iUhPirS23POwpWs5aFxItNLdmY369M6fz+pZYw\\\\\\\/mBR4tpIgAPN+6MX9efXkCxk9S94\\\\\\\/ASwC6MktsMnac8tyK1z4Dl+3zo48bQBPFsTXTYwG3BXJTfKekgfuNaIJThBrIdopxQyn2pmAvOdiFLbIUSWJqlIRNtrEvw4dSGScavpj2IQDCaNVKX\\\\\\\/lZDhRs3DXi+AHDar5D2d\\\\\\\/YkMpt1+8vv90fb77e4hNjGbpehgS6vEWQEaZ0gyCAxaEKMH7r2yWXd5TdYtfiAjs9fmKn7lC8d9Bkr\\\\\\\/dRm5yvibZYxEQqnS7W7PVbdJdfZlAQKPVkamO9HI3W5f4EpXfRs7hm4PcxD8adPrxOECEz8lwWvBV+0GunS2KZQVoJnkZIf3BD13mdrVHxAR8Vbun\\\\\\\/tQuls5rCoS35Ka0bnAGXXe+FfPaBXkmRdj+Y+7n9naiNYLSl2YEbZ0gIo89ZkQw+ry7L2G86bQrgrYkCamdkfQffkTg0Qhx+T9+Qqb3Vk4o8fJFL\\\\\\\/eRiVNtxWzg82nqGTytHqFXVGUOz+hq\\\\\\\/i3TZxHsgBNWbFK6pBh8Kbsm92JBqkrcOHDn1cTxSGuVbndnoTtKEIcW6sYTKbfiyLQHEnh\\\\\\\/u8+Sw4w0DYzud6se1iShkBvehG789OJh3Zr0sHWJvsv4TiFVTpmj0Luybm6lsXoAOVjchslMGkXA1cxHYbzcF4tiapc1d3R5cSZC9UPmEXoTlzqpXq9ClW8TKNzhFjci2E8XZxI6WW\\\\\\\/RWlutfIf2V2ZpJG0hUhY1xaFdEMF\\\\\\\/\\\\\\\/QVhGb8nkAgZZssJp\\\\\\\/Zh\\\\\\\/Plp3ZeFmBvLr4IiVewZzFJLyC\\\\\\\/Z212XbhnYkCHxji2zPGFpzP6kI8oKUFOV36ygOpLwhS\\\\\\\/LSd2HhdW9GM9fb8IQIBl\\\\\\\/3ftKymf3JH4IsteIVesw+k4r1cOQWvbeSnnCDFnYLzPTIvO+krSTQ+vvwkn9ppUIWWaExCU+OP8lUTSZuENJhwh8dgP8CKdsSt\\\\\\\/iauSL0+Lmp4FNjkR9ZLpBh96+cTg6dISZAvp0W0O2F\\\\\\\/nrzK2tzDSAd2iw1sHNn36crP4i04dckxavuAUZ2qwmzDPMXrGGYclPyr0Wr1bQOwEan3Fed93EhmKgJnSneABjXWt0JoGbcQGDdtDVWpdyI5FsjAFt+TTwyS40iv+4HUmHv5qM0vcz1qJc8XoETBqw1SlsG9aTvUPJwlprDeFC+rRX\\\\\\\/fzVProKut0Z15nggdSs+92lMzWv842\\\\\\\/adZCCc3uLBU77teN43J3PX6vabMyMbnOkw20\\\\\\\/m+iDCd1B8VP+vWMzSb1Abf91OBRhcFaA93dj6ZJ5EsIaq\\\\\\\/4eg496hpJJUw9rhec3uooXqR5sb3O8A7QCif79O7dT6eld88k3DRGU0YG5r6ywmp\\\\\\\/4f+zY7Ic\\\\\\\/2L4KW1eklRbZ\\\\\\\/jzKA6sm1ZueFweIqzakQ6AUw+Z4vXnhqxXi9wL4n1Qx9Gd06P17fbCCil3XLsWCluE35rmyAqqoGp4VnS76R6M3qaCxfVrtKL8PXWWZhZ2WcQDE7U8avXbg7zaQYFmIE9DUnJxdcBj7ZR9TltdFOn+HXCAkm9BWMBfQ2mor7J1YEE3VKZ+obrXY6WLm4SkDbEFCRaEJAPxDwzTY6m0A\\\\\\\/dRKERy56dmd6wJ\\\\\\\/adsD3lm7e6fWzbijcVPGOuEBREQSdEkpoc+x1OVHHf4gzABsqqbvlbg3+Pz1SpFzN3jbryYYCkyoaSXfYR8jMsH\\\\\\\/O\\\\\\\/GvV93f0I8isD7QvGmZk6NppwTUrc1kHQf6K8Io6r\\\\\\\/J7ychtFzoVsVcB59HnLcVer0fz2uAYwrkYW8hW1E+SjStLpeNMZgZwzPwWHy2Aa0i5TDS1Fyj36il78jjQHO1wpBNu5rIpJVga9ttwELb25k19ziVIUkNdKR2zD1TXAbQRkSbDSG3eQ6ZGNEOUx4MBpSZlp\\\\\\\/b55gXwWLbBnulg67T7oWI3U2Co7uu3YzHqIdMFMOrFPQaTTi4B+AVsQoYrR726IOS26mt5qp+7S36Kc9my1vH8hFlxJYIbiBTHr\\\\\\\/3\\\\\\\/0sXG\\\\\\\/9i6aGqO4gy78osYUnvFP9nZh2sgPwqqdgtB1I+favi37WaWCwz8UsDkyHnRZ1GolSVWxWwSA\\\\\\\/pap25CFWC4KIzg+NqVqFz8zS40T\\\\\\\/al4CtqUnh2N6FNGpgVT9PuahiD1Lqe37Jo3vxec5itWIMVGjw3nBd\\\\\\\/BxAuW3EKew3Z+zB4\\\\\\\/b7YeLlWSk966LiN\\\\\\\/P4kFV35sgmHd0g61vo+\\\\\\\/II6iKTvbw547lQiPbKVVIEnmPZN9A1ERg2NlwkDOB93JY1Vvs179\\\\\\\/gQcdraR3qLl+DA7RqC01gFfvX+NSq4NPpdbcIdjOBdWXfmeM10Ztn9OClpxKl1SVd0rhNKjDaQkKkfki7gmCggSgHI1ZL9WpkF5OI8\\\\\\\/AqOCR6U+pSc7lvYeRjA9Ia5VrKd\\\\\\\/TKPH2Wcwfqa9vc+mYDV0q0xASSHrSAOp\\\\\\\/4SiVGDoaMvWrvh6SEOMo9qv0t3rKz4+9JMxBAax8MM2zM\\\\\\\/4MGiF7tlA\\\\\\\/JWrb+udiIZ2c\\\\\\\/ralYNnDI+wyKYDYdawphlAlIo0HqJba+hBvDGX25\\\\\\\/w9LbiwS43MlmngyY40\\\\\\\/penY\\\\\\\/6XLEhG2abZaslYbuL8gD1GZgvb7mPGX2P4SGYfsMF+fxpKHW2TaqBe6j0vDDNF1fKUEWg03zBoZrckMiwSd+VMoYEtf8ktNJcwaQH8vkGmnl1naRFWIvCQJ8Ubp6d\\\\\\\/uTcimqg3Bd4SfE9LgOSxfe6TOMQ9ZG84Cl++CzgJZ4l9SMhgjBYtTFBro0b6p5e4o4JrmC1j4Y22mhSSj6NBFegymoEefM\\\\\\\/96b2TzkFUDAYaeR3LzHa\\\\\\\/qVf0Wzc0FGJckdWbGzLTnNGhLt4f41lHkvvRzd36UrUNu24LpK576eXKw278B8tjCQBFQ8R0nVv5i2ZnaxVz28Y0EhqFtr8RdTrnNVo3Bf0\\\\\\\/+jWxxIuHDMWaKzregCjW3OOW3RQe6ajSAu\\\\\\\/3gGAgvHGmQFKvixvS+9sk8Xt3SHT\\\\\\\/hkvpFGsVJHWXe6cOimhA4DL7kpu3FngwMnVJiT5c6eT4UHiwhUWaVU+0U8eeEPrXg0KUdYaV9Zbl1mQyoe7QYzTmUP1Jri0wPNfUsKTsAHguEkwUR5CeMP+i6\\\\\\\/bLajRtDeoe63XlrPhwKqMJwiBC2ivBpzbUuEeul+gTIueJsJBXq01mDYd7yKq2t5XnHpUJVKRQHoCqHfTpOST\\\\\\\/wtS2Vtxfb3ThRXbiNesLDQ5HlDzVC+ScENlzn8wpnq0S6ZQFYhYPkIYIJDkTqNTCNf8eyQjL8zlQHS7BtW\\\\\\\/R+Z20c1qyy4kbBO5r4lL8eRc9+\\\\\\\/B7cQyiUBjfPjuNEW9kJO49ICETFK1dIJgx3SOK83zW\\\\\\\/L+XS1lXB+pXfk6o3LYNhp08qzM0LYz\\\\\\\/clQsMHIxp78Nap9Gq0R9VjF8N1aPC9FpBoUuzqdbxnN1MkdadarLZggfbl1Lr4yiUe4EkO6WWNDnTBjowCPxtHn7qDHctLFh9QdpfxT+p41bl9EMI3i0cTjXGOvlBL1fmOF\\\\\\\/WnWGrbk3C0KFAnlUdmjhkVxEQ\\\\\\\/0CKaMl+HPWRTQydmSBm8nPOXKPndehJxGtmaAfAYP4t1yH5U6nzdefMLq3ssLDfxOWWflADZj\\\\\\\/X7VT1VUsAyyKaKMLvP+QG7KXPXm7YoTo2pkIgp28D9i3E6goTW97hJxB5507KuSB9JLU09HK+frrpPxSwBufmYJ8G4oXUn9RZ90upqdQsC2\\\\\\\/YwRSy3rxovllSH4IVUU1wnBvd+e9sQRKi5x4DVm9hlBOSiAFLWLje4lKMVNnOIAdVPr2n3O8CPPOrn8JnpaQLfgGRUbLWTbBGKzHrTU\\\\\\\/XkRY8FxjVbuuHtcjEUE87Yaqz5NmfAGSAQYam+VAuyPyiqq\\\\\\\/0eM9NWJG7TSZY9SIoK6e9c5XWBWcrY4bzBL+tkDU4+mlIbvyoSmGjoP1Nirlf4LQIylybiFyg3P\\\\\\\/1RShXPwx3Vv1h9pe2uGUTEfmh1CkYCcpGs7sxA\\\\\\\/q5k3bUjzVUWWLxcSF93e79fAT\\\\\\\/fGuCk2zQ8Lmz7PaFA7EyGx43opOSz+ivlH8sfeONe5n0D9FzQ3TQRGLzDzTyBUeCUYo6OWhylsRjuFLOAUNAyiENScLmV7fA8871K0UEOOYHKA0e0T0\\\\\\\/K0eQ57\\\\\\\/7RguFWOALBbVMuj3GTre88RAZGSMIp59WkL94dUSmiTZYL25HYHuQMNJ20oHg7KqI2T26e8BMj\\\\\\\/2KYzA412+4lgFMuuVphSYyI10mz+gS5P2EOW+pxL6Si\\\\\\\/Gp5CgdPi+\\\\\\\/Q3DjHOvFjG6AnA22HdAt0sQ09wt3I02LPqqvIB82AH\\\\\\\/R6UjgVHnfKy7oXLGvJfBLuQbHDGBCMZvvVwgnh\\\\\\\/f1FDUlzhVcH7aJJEL3wvVEH549vrDAH1F8ZBzDKHbod66zPzi5X3ug0+rDC6NGm9csLshhWKm3gyprKJQoYOjLXU8vmEPXr3\\\\\\\/giZwCuQRUcCD9wGEJ5k0mTgouC9l0GQWqeNq8JYnQJliMYkoMQNHEDmvbDdBEK5GBOySeBj4kM6TEwdQ3JkZOxa8h5wqX3953WRHTDmJE2gnARtLpMhPPX7n5h0gbR\\\\\\\/c0XwNC4Vi7S2Ij4mkYCZMmJv\\\\\\\/7sZA2B6QVZ9xYdQv\\\\\\\/KoiGu65qPGLMdtHq7DdwzTocsDDP3zqN40ctIVJdGyEl76tXN9SSi56n6aCa2c5pPqg5Xdqutwzuk2VPXURdvT6uikHPeFwk39wQi1QcxYwqvHJZ1aaAI0t8ren2qvApJayKD2GbCN1ZpoEWiIfxk2JaXvg5eGAWGI68I1kHhDcDJr6Np7H6s7wUl\\\\\\\/TX2jz5\\\\\\\/pveqBcpcoKj23a9O+WLHuSnUTwVVYBQ6qu1noY2a+J8c\\\\\\\/s9PpXla1PyiLq9yEOcl5TB4XRBBcY+eVc\\\\\\\/sxD0FpRS+vjRgR5wgjZELkEgy1HOjpnQtuP8cbOCqqdLZUv7CjJNXdYn+w0MUy7jjkki1N8M95EPrwJL18BF1wr5mhsVTI9gEBchnhNWwNU2TA0I9bBNvg6WYuO7+eKgE520th6MyCN5UiVH3qDFzsEkPSSEkGpo+EYFosbdVHf\\\\\\\/1LhPafpLrhFFWji19lVCq8BKJ4Ax3L27otmBv+vrbWJFJIpoGlrZ+FA0YXhjpxHYxXYKeZTqTRpiKRJeC1mVLJoxrS0\\\\\\\/m62jjNMkltjNQ21njr5SPakM0jtbtAnUBqPXgmF38Il66mdLTAOS2NvswbDf3QYSy02\\\\\\\/hc96xrNYBaXKISUue1H7S+5AJu3dUQ84lPSlXvPdc76yFzoS4owNG7iliu5Q317tWwdGm6sllyMqQQdeENqwEIe0p+eyHVCMc5y7Swykhb7EwCaYWREUSlVtUdbukG8SXmzSyjTzeQUxwRAYaS4xSp8Ybjk4qFeGiIUaDw0lzTmLHGyHbCafT7qScdRHTFo6xukisOTomW6NIDbFTVMv3eQMGV5Shjp39rQngYA6u0XhHshfTzvTUnKiKSghqXlbfJeN7DhOZ61MlVvZEMvT0vmnAI0u0xQYTjhJdVA4v3fIFUyxn2bC8DXWN+sRQW3rR8sHQAewzip+qrAi967AgA6fnVRamMFH5bBPOTAuXe1UEuSKvjBZnOYWQbPJiIid0efF6k6NXJ\\\\\\\/WPZyDhM85eqaIXvxH9fesgCGOumSll4i8Rhroe2E+bkaGmhNoUsTx+UijSbGMtAExtqBWkfDbnss3JWtZiLhFZq0gj3EUj7751WQm40fYqstFn9kvYOtg7R5CZNt5jrGPhygRKGf8725VCnsMFqv2uumdGsTpBFalSRbQnhc4sTjCIq7PIjBP6CADr5M16ggx6SMPQoqG5C\\\\\\\/sMBl\\\\\\\/kbbkJ8JrDXkg4Ds+wJCtigPWCDauqXlbirhPsvRN2GLUIJ8M8s7HC7XHaARGWWvNDNBWe2raZ60rnifMMwjOxmozyOpA\\\\\\\/lR8MoSTS2Cp16bGA3uEkzTFS6oiBD+vEv4vgmu69SXSvlc6noljcPC2O9LYgdoiSTW3WDsLFx1bI4wCNKYAYvn5UsqJJyqPd7bRbrQwiq57st13dHiaxjVA2vldfMxLQ+VkWJY4JXs\\\\\\\/8TLbDOD2aGcgXkyW3jhOTEYalP\\\\\\\/Z3UEwTVe2\\\\\\\/or9lP64x+1DYkDrfytKzr9FFYuwn36PWWg7D4yGhRe\\\\\\\/jfPK3S0iYIyEsL5NFc1X2kaWD5FHxRamruMvv0OeubHSwdAba6vJGysiigQNGj4Sw8qwtmLIVVGhu9Nbjb\\\\\\\/JCe8bRVYFGyYOg9gv57Cmg8cB3iJOJv9PD\\\\\\\/QezAieh14UDolMlHL0U2OvN84dErUw6StaZiObw8V5u2es7tdEBuDYce4o6FC12oVg1WgvLhi\\\\\\\/1PpmsislekpvbHei2BAd8cedt2+uYmXyfstZgidnxL7KkjiQnzMn34EAJpnd\\\\\\\/xYJzcMeBmXwo\\\\\\\/PIEQqbFP\\\\\\\/ubBtPmgm\\\\\\\/1o1Dk0QhfG38YL82A4bU7wVeKIsSzD77qmKAMb1Tjce55XcQv4ib\\\\\\\/x666POX1x\\\\\\\/gPZRUywsKJtJ4UNcWNUn8JyL5LVVJCdYtcu1Da5l+lmmC8Az6Ry51lyVcBo7GDc\\\\\\\/2rWOp72N44wtfRCl7cOG2R0H8gyITCsWeY1sd9kh5GNogYx0uBO2iSwtFuu9y6Yot1XQWU14WnHbWluAN0G\\\\\\\/yk9Ijwfv\\\\\\\/XJKVeZnvOHx8aM64q4FKO4ES9aw5U++rxTdMSCKDLzTKZ0EELGU2bDscOwcn9rLvvze+y2NXBjf4M+VBUVemQVcvxR8g+bS16irZhLynh1bfgwuJ7wAsyh0mxgvwBnzFYiQFYBYt+lhRH0mfqcko3Lu1TYpbyDAVvJl7zszWAUlWP\\\\\\\/whHIO3BhbTXQUDwpJcYCGKAQsXwfhcG7BzA7BDganh8EUQrD4fpTpqcAoIt3JDwZ+einvYGUFOSLgyiqy7eMPK4sxE6uwfVLw1Xtq\\\\\\\/NhV+Eb3j9NxL\\\\\\\/qUJ9DJIFvsXZxlDGUWvofGakbRrRcLi2CfLglci5vLxu\\\\\\\/l\\\\\\\/NMK7Mmn1OHKbTOrkRxmEUI4Orb7vc6q4UunNAjAHcoQ+UDQBqroX6djlS\\\\\\\/5g7v2JWV+xT8\\\\\\\/xdy73pGC2pinDeP070o4MbFU1YoEPXYT4IQW5vmK8D8IUpoXLEoezJbxb3+3oe5XKF+dlEAq2b\\\\\\\/uYbVSHXmN08e6En+aVY1kum+W46+D2QUIxae\\\\\\\/Uru7c9rp49MlBIqeXu9e1dJQbAQNdDup7GvwmS4apymEibTIQOENRur6FNbYLZtMs8798Bvc3AzoKGISuo0fiAxeGgu8GDmT7kKmYss2TodoivkHAx1DKPaAD8TOpjcC0CvS3jCYFN3qeAUSVmR5kcJVx\\\\\\\/fxZ\\\\\\\/nvS0eaVOxaWmwH1nbcprYHjEQEiXkh3TnrhsOu74hfw1yV23ysuM9rzTFKZy3apzLxIS4oO7349uyPRYsSVGGcOKanAaPr41JWhr+tirBoGmS8DJjOxXd1Js36p2ymL1umJJoMGRmLxXQoLjFVeu8uvX18WHmDuOoUSoeI7I6GNM2lBA2QWejHfyGV8IolJtYMTVWEUNa0QC4+OaCq9oWCf4VepYNK0ZaMMBWnOr2SvMVWxZYS13Sfe2Bd7Syj5ySVnf8UvCyd+zb+u8oXtW1CSE3I5V0MAT\\\\\\\/skpKbwQpX1sv8+OgZvucDGlG+H5GHyaDp\\\\\\\/k3e5hoFpPnSyYwTzRX9yFNZtWvWYwjSiW9pncimWRXkR2IkCvnVoNdhXMQj7v2BKN90h71MzsqL3syvq7nhtU5nJrbokewRunyy8xFbDVq4cuPMkXjTJKbR0JCXLUTmOWw6WJQXN+PeJu1WHOOCoYNEzuG4yLdDPgmasyAF4\\\\\\\/gZeTwz0Ri0UsqyCpqQeMc8p3YfMm\\\\\\\/+Ena7pKMfegsQ6hJSKXhR\\\\\\\/O9DExI6PElAQU7725XMzAO0Ml359Zsdb3\\\\\\\/9Ag6BsC\\\\\\\/Z3KmWqvUgOeO6ghHcv8O81K4OMUSSYrlL+UKbu5ofE3xbm2JgFsY8bK1TAf\\\\\\\/953Dr4qiGyJ6cFMIVRxpJuDt7GaI6PdrlBUhe1cf3gz4A7mKLdAOphPPI+NcUS\\\\\\\/sw0xZAnApjOL0cEQDixONoss4xVtUDtk4FoN2BnteX0OKRIGmWkMycaiuhVEVmQIe8C2rM80nAyDObC2\\\\\\\/nHIeVDi8hf7MtAof702uJHzhyHfGpskE6A3LmMdaKrK3yIS10smWpe4gAKEn+nx4W8YZ\\\\\\\/FjiiJOkBnFTneVEsHBJHWQfJyMRFellhyw7QSPiY+CJgLKBebAIfK1DKJprLHzCCdTkwBY68NdE1Bs8+yCftWB+ZZCJFgmPevR3YPNk4HMOLGlkvd82tdSNGK5FR9b6pfLuWb2L85H2409hySkJHsk9UbF4jBFVA06SD98IzvU8IIo7FvSwBUnSLFva7WljknkecvAx0jM8SZ+2jZJIDdVR9qYVMHnD4I7cQsJhjTMlugUP924VmHNXa86xS0ZgNv\\\\\\\/ikldwjeDwTO5rTlmjIxlqJQ55p7IEhy1YqicOUDbYeFGlkfZ428KqWALnVpFChB7Pz0jmEWwX7hmu9s1RUhwjPC39x6W71JDA9oSpvRHkDDGDx8iu8ytXyvoWVjz\\\\\\\/fjcBj7tMUG\\\\\\\/17gp0h3+HXaf0jZKtu8s1eErTRhgpYMA+p5q91aUmrVvHYAYuKSTslQnLcIFUPfS\\\\\\\/yLjaT3DF8EEXZnr\\\\\\\/0z0hr4\\\\\\\/YownzvNJlJkTFQ\\\\\\\/5jDOyiNx3TN46s3t40Gw598fghUpm8xcFht0PPYKyNqUidGqZ8qrH131ne\\\\\\\/H3QoorcSxQhPw502V\\\\\\\/Fk4X\\\\\\\/2N8GwcnCqyF7pbBN0sDKuUf2DDSWzZEl8ToQe13H8mRPiaRJdH1bU+jOa57gbeMbaoQlVYtnKNkntRvgDmRILakjVSdeIPaivrCSqAnOIPCyyr6dIpay\\\\\\\/ufDkbZPaRrzFwOs7\\\\\\\/L88wkPZNw+Rf0nV9Osldjpadho+yOhvLGNcGz4LrUeHWbT1igbIXervXczj5DRgcBMDGq0mNXqUrJmpOpyynfvkWJJ1IHEQ7ANBKLwzOdofQRWveL1vJYK6Ay02oI3dwiQdCED5lRpjlmDJomHH33IlErKVyGfnsEXQeA8TQwKOiDDmf7eudGGMGjFgpK3w8pTNEnWKqRVFoFZPHSFB1ZS6wqkAJvaF8ndw\\\\\\\/eq6x8F2gf1\\\\\\\/yKV8Qyj1un5BxKkHaZQ1j64tBCYb0dkLVoXga1AtSJvih65DMlX\\\\\\\/B2NvdncVOwo8jAteAfrW4stOv0siaL90MHCH52oRxQ7PxLaUiDz+yxhomMmrBbKroz+1MsueHJ+6ODumD4v4pz\\\\\\\/5Lpa7zfdd3mFVBBVvOF6Uj7bVbOmiIVCU7hlmesi33CrV600LXoxN4klRinwNqTv13dQSbRXljFNhr105qgt76ObjKNVs0HrYhAZeqRbtsOq7rBMSa4HQVcJBh6Bu+c39hW7\\\\\\\/pN1iLBb7VRiy5x8UuUeNoDeSJKCsqv3wJUCMUQYSOMhFnVVRthDd\\\\\\\/X8w1rtgawVXIBrWbJXchV3Kw\\\\\\\/xu4RJF7nCVVTb\\\\\\\/rOChnNZUx40iBQXDLktP\\\\\\\/8QhfGSh5+P\\\\\\\/cQGhj\\\\\\\/Q2\\\\\\\/oP8UU\\\\\\\/XrSm0xVZ9zMby1hM8oB+OnLV3gcRq71VmiijfGywNiT+Z2IJeaBRCq7onhGoi3evPZqzupKaEHE6tiaKA+BNZiv9G6ILIly\\\\\\\/Hq5z95OD+mp3p0G5CopyirUB3v3Twd7Hpt5rTQivUKMVzrqwXnfmh+pJHWHZ2JnFm+8W3Jgw7725QLbC1iarycnsYDtc+0RNkOazsOo6eVQ0NXtwk\\\\\\\/iX3KjucpMELdz87ZRbnV+6u+ZnpJaBUaklM1aG0r4xLiHhnkLKydWgd5H99ATfzMtRkoykmhe4KXGINfOmJ3DSloXjBTvaiquN9OE6PJgoKjjjtF4vG07\\\\\\\/X\\\\\\\/awSb0NSmGPuMbq3PcoVonxQax5iLv\\\\\\\/nRGhpSkK+g6rlkCHAJXl9QZVbntQui8ENSbMHBkNnbmKG0bSw0dzM9VvRAvQB9gVOnSgB0EfObh5EwxhzVpVHMO1XxtQkmwQRRyQyxFq1eMz\\\\\\\/6pR8InAYczJaGVe0XO16gBxDuKx3LNbVQ7gTemW9ZUoy2wdjGVSLft9u5mAgu7ayzeNn1sG0J14TevZ9A2cb+oDX3Rtv3iJmIOovzmWdpQamPOOT1agBaCrqBBe7uHENN5SqiRAu+py6uvSyxdcqAtkX+eSvU0LfjkO9VVEjPYyuul5\\\\\\\/Ccf4d2oUV2Snv58e7OIQXOmThMc1kS+Au0Fu\\\\\\\/ncAWgWBQZMsMeDU\\\\\\\/LKw9tTv6Qff6g2LmDhaH+8pMNDgC5oTOAHYyDw2aZzRo\\\\\\\/x1zptEt2qWh2KzuKMxYxKvCT8QK3dgYJdTSImjFUTfu97ve70ZQG2SY3hKvXlToyqPSjZN++vgeK8seFY78pDljd\\\\\\\/hDk1oOrAHndH1ozVk4vQNzB3Sr3uMlLduPaS2oca1GrJ5ngpx8e9pn1UPj21rTl+hbf4NWfNYeA1nJrWKyvBKNTZy3sJTEW7sH3YeB6V1qj24DSu+riovS2Ej78PfBHirkanEdy5ofOT05f2qCy\\\\\\\/3Nyjk\\\\\\\/IqNHdD2QMAj9auYp9PXhIez+KQPi3\\\\\\\/8lIag5n9ZFXTlc\\\\\\\/IyW+5JntDOtSSkrEJvabYu\\\\\\\/fEEcSpnqcH7TFrMypr5w3xdJ5zi1MLT+3NWJO4Nj2y0+YWYYdK+eaNbbSDXzofcqNi8goEloPY\\\\\\\/0g+11oPR0aprmhSFrrdQ8\\\\\\\/ZXGSwaM7NpBkvQ+DqO4FqPXZ0kp69QsCqNtcZjdTfXwrUqI4PCl0N6q9Exsu8TTZBqbUyN0Sg+XEMQKWFlvnW23eFet1NcGBwOMmlBbQEpCDU3uYY3md4\\\\\\\/tLwCir\\\\\\\/NpZnAqMdYGHShZxa2m8JOwcFQ1v8qzNM10fZ8uqX9mbmfft6q6GJ4AgTJpDIUE7EsqP3XPJ0RZQkmSqmEtAG8uYrzM6lZBDxakTW5g4ZyEmF3ljX8TSZ2PBYQGNZ1I0Rc4\\\\\\\/AZ9xN+QGrKHVbwqjw7r2J1u4skNyvIRFaprzxWEEdBbR1NLBMGRruPJKuZYzCOGSf9e419ltxG9OSBmZkGl7rWsMea7dn8gu0GK2N7hI3WoMg9A7I3Nr8TVqFpwcWHvWQInzuivE6dmrdXbP9C8KzUyFharST1+W0lYhnGhaLj8NXkCymNdkLlNdV1SWQcvt8M3jCEnrb206dgHfp7RJ65DqU4ig+SoOj8FKm6UdwXkCymehXEsyMnPQbfPBf895qNSI+f4WB9vdTB7\\\\\\\/pzQoxDEp5hxP+7CtXUqL+Ihd0v8wExFBISnoSYJ39StB64KaGithLOmvTzHVaf2a+0DIzMcQxzO75agrRQUbRPt851MjGq+l9XBOmRFfHID\\\\\\\/1Yu71XyfxZQb3EIdvVBqhDjHah6K9Ey5Ym1xxXhfgXuSAgK\\\\\\\/a3xZOuxjy3qki7NdxChOXOhVqWPhaQ\\\\\\\/SWhD4KPAAzt2N4YOt8Lli8BSsx\\\\\\\/P+mTXvA11HXP3D2RFxJWcLIIFT7CFSC8JvOoYO9kyL6sNgrDMxFiwCmU18J0oaYIKzJXp8l5lZHza5kLY5U8saMJdESlJKMIUcTZc54udDRV6aC9cQhpV4bh8XA+PDVGjUhSzjZ7o2Tjncna6kQqTkVhgmgCH\\\\\\\/StKGLYukb5SOkIIYqNW3wHeSNyiiF5XZvapVFedIm7e1LEstg3NxjuhrHf4yHEmOatiRkzSdnbNvn1LyplZZS6FOrKhtVRCk0K+QEkd1+IVh9yma\\\\\\\/eVU+qfnMku1V4dJMTcUTL\\\\\\\/gdQ7hf75PA+7AaburHebyr21Cv9yUVAMGj2+kFW92dB\\\\\\\/s0TFahFPROwQXaRS5SAx22Mf\\\\\\\/YoGsMAzkP35XvkuQxpqNngosfcYrak0BcVa0jegMOV405VBeRfwuMAmXysnHPpKvzxmvMhhZw6QbGkbkJjuSk12RM81Colq+IiLnnOpxEZboteEc\\\\\\\/G\\\\\\\/LNaKzoc+NKoIRMiXplKk1KLErW88jJPIFojITcQOm50uGWj9sLhzxcNzNcwWzIoleZY8HiGpX1eEwOhgJvzUNXTATryYvsmn8adBtApZgy\\\\\\\/jm0qKZSX0qvRqzCBr4gRsqyPPKd5ulHemRZ6A6uLU443YGSgk+gaUjKjFHhSEnGHKE13jkUlIouGQ3Nr+mM9PzhAFqer3GDaqTWrftpAbfF7j7j58VD9BJ32yHASNtY6khkqjQEOxmI5XcGvsCNZd7T2+IEfwPwmBUrw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GVIRrT9EVCrwt7EFKtPq+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d203cb2d8f94347482eecbcac1790d577291e4ad73cd8e6b8c728ad43006d1a6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qLZLuP\\\\\\\/k7SqK0lIsBHNN5Imd3YQl7GOAPMt7gGVpxumPoA+JoGUK4C2\\\\\\\/rEdpePVJ92F4idMfbDfIjuRhvaWixd4ioCvrAKOo5TboPuLIo8hXjevZ3PPKAm8gvLJJLNFe1JayYdpQd0xKgMzGr2hmweF9gIGqatkApEUIGgUbI6\\\\\\\/f15xyn\\\\\\\/MhHa2+9zLV0BgiEqymlk0d+4x2pWwPObIuLnJcbvHXESjc8foJbPzrrguHZ8Q+N4Y75uje9Qbyo2yic2mjQBlRKc22VwthtO3m1eLj2MoOrNe7h\\\\\\\/Q+c+20J9eiguKY0e9icI6cTUcvdJDXKDfCF14lr5qIG2MTwCCyh\\\\\\\/3SR0+bkFQ8raZqcO05bP3W\\\\\\\/waE+CZECSkXjHLm\\\\\\\/dq8hNczDflTKAeoxP7iTGvTwCu1U\\\\\\\/1GlWD34LliZz\\\\\\\/afnvzs3pTqVOVHU0RgGjqslLYChKEA6ZG52eBidL0bVgOwkExO6EHTpOWwtnkOfMj5j9svQQG2bht94FHo7SyFDGpcBVQzhZvJ8zCmFsI9iahqy9WYy\\\\\\\/rfNK6soEoHbU284InRs\\\\\\\/CGU0y2Lm2op010tt5Zf6onGiT9+Z+cOCFzlU0yZKJtChJdsKaO5\\\\\\\/g1RA7PK3Wz+GwRKgqMS1tzP4kii7M+T0Ou8x20sFU1ImjcAv200JV7+RBQaToBVEjlXXzLISALy6TwGLDffSGl6eGjhZCTq2GFrUItkcd4qH9IYkVVl5APlt+1GiKD7iHNUWtRouGLD+6F96Uivp3ra69itg\\\\\\\/H1Bm3XfpOyXBTv+s4nalKdG1\\\\\\\/+bXTFT1\\\\\\\/Q9GuXzl2yeEb4Ljjfe6hjyJXWDzVFmqjM94rlp1rcnx8yQZOKK2VJtz\\\\\\\/mET2utz3BdWON6e0fs0Ze9uTzDY0lUdqIeSis4BU6p48rA7gVupH4g556uoJqjMzLdIPI13oourcgHdeeYP3dbQ9gtnw8BTJaMPVkQFD2sxyhAWKJy1CpZi8knlHwSO0\\\\\\\/QD+eQrzSBTTX9jnog8n4IwxzW\\\\\\\/5WM\\\\\\\/MX4eCHGuuzM0iWblRdAZQrZ24iHky7Xpj9fkFs9QEnhtKCgzdNKVAZNumXLrGtiNV9g22RO43XmuMdEzuYS9hBd69usv3+Urmf7WJWjgplnARBSFU1UPDMfpyvLJnCocdCXxhizdW8kt4tCeeIQv3AwMAetG0tuA25mHl15VP5oOLXp0kmfOtpuAvKyxF\\\\\\\/JnlsiNx32tsVDfzjVNZ+xllsnnrMVpVJMSLTj9sII62gB5nqeQgF+x1swnTwdlefoFTMVSP2uyjFjNUcFp5qC0aCNotCROmdeqSlEwsq\\\\\\\/THeZwZ95bsz6vEe0iogsAb38VbM+pR8jJBkjc2UJVjeQhw8S+qKoY8p98GBCE8R\\\\\\\/iNEZHALv0YRuR9zZ\\\\\\\/FBwiGw32Uq0yZdSkhQb1w+z8kP2xZ5H\\\\\\\/50dEVyc\\\\\\\/HDdPfV+NmnpYmmEmsBLakGkbveecWLedDO0D13sFGV60D1ssMVzZSk5e45j\\\\\\\/0Gm8g57UwGPUGZyLcLcX5mjsTFSzU3l+UquBij06CK0CVUZgCUSCb8gDBTnYnS0ERYdEfnQCCJy5onnEmlO5pYDfD92QUS04f7XQKn841oIBvBZ7kF4ulS3uo97Wo9V9zg1hR3CQtUhRw3SN8QJk\\\\\\\/jxnMsqeEphuKNI6pDCxYeeRw\\\\\\\/3Wjxk+9RWq9uMj0C1CbMzxr04tqyu2eVeSufIgKc1QyhXElR4OrdgOSDmD+fWF9jDjAjgcq2k74y8OSN9NIT9xYIf6Ty74Yxx\\\\\\\/rYSNNwXd31lT8U4reYLZEZpAjzbs3LHHDNwHn8j3EEe5NByvxsqfR3eHQ5+dAvpAKAo2i5gsGiXzGh7VKyeekpW09sl1aDTkYI7kBPVp50TZ60R0Jp83bHJ54LzifI9\\\\\\\/nOihjNu51xvVQ95fgtEeU9waxJfSWK\\\\\\\/mqbzwJQv403uoRd+tMSqN82CMY+kvbimm6VHiHzZRswlHAKoJzn0uERYRPPm8XFykzILEqhznUoTkCI+BEA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022optHSnwyVKcdyGJM5+guyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c48c8f1d86c6a54dff497cc2411e2ff144c0fbf74f489e907a2e7ec7fdc582ee\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-494901 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3933" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YVDRvQfbQyYtBk5mPEOkUDRc6HbOMMQyZmzeMR46_ks" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3933" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7X6RBWHe9pvDm7QCajM5j6q-dnXFAWryK-ddnmkqEHk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/3929" title="A Highly Contiguous Reference Genome for Northern Bobwhite (Colinus virginianus)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/3943" title="The Genome Sequence of the Eastern Woodchuck (Marmota monax) – A Preclinical Animal Model for Chronic Hepatitis B" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494901" id="node494901" data-pisa="ggg;9/12/3933" data-pisa-master="ggg;g3.119.400529" data-apath="/ggg/9/12/3933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>de Novo</em> Reference Genome and Transcriptome Assemblies of the Wild Tomato Species <em>Solanum chilense</em> Highlights Birth and Death of NLR Genes Between Tomato Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Remco Stam</span>, <span class="highwire-citation-author" data-delta="1">Tetyana Nosenko</span>, <span class="highwire-citation-author" data-delta="2">Anja C. Hörger</span>, <span class="highwire-citation-author" data-delta="3">Wolfgang Stephan</span>, <span class="highwire-citation-author" data-delta="4">Michael Seidel</span>, <span class="highwire-citation-author" data-delta="5">José M. M. Kuhn</span>, <span class="highwire-citation-author" data-delta="6">Georg Haberer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurelien Tellier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3933-3941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400529 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494901" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Remco Stam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Phytopathology, Technical University Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Remco%2BStam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stam%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARemco%2BStam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Remco Stam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stam@wzw.tum.de" class="" data-icon-position="" data-hide-link-title="0">stam@wzw.tum.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tetyana Nosenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Environmental Simulations, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetyana%2BNosenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nosenko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetyana%2BNosenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anja C. Hörger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biosciences, University of Salzburg, Austria,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anja%2BC.%2BH%C3%B6rger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=H%C3%B6rger%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnja%2BC.%2BH%25C3%25B6rger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Evolutionary Biology, LMU Munich and Natural History Museum Berlin, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José M. M. Kuhn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evolutionary Biology and Ecology, Albert-Ludwig University of Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BM.%2BKuhn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhn%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BM.%2BKuhn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Georg Haberer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BHaberer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haberer%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BHaberer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aurelien Tellier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurelien%2BTellier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tellier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAurelien%2BTellier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurelien Tellier</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/12/3933" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:494901" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:494901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/3933.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:494901" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:494901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/12/3933.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:494901" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:494901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/12/3933.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:494901" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:494901/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Wild tomato species, like <em>Solanum chilense</em>, are important germplasm resources for enhanced biotic and abiotic stress resistance in tomato breeding. <em>S. chilense</em> also serves as a model to study adaptation of plants to drought and the evolution of seed banks. The absence of a well-annotated reference genome in this compulsory outcrossing, very diverse species limits in-depth studies on the genes involved.</p><p id="p-4">We generated ∼134 Gb of DNA and 157 Gb of RNA sequence data for <em>S chilense</em>, which yielded a draft genome with an estimated length of 914 Mb, encoding 25,885 high-confidence predicted gene models, which show homology to known protein-coding genes of other tomato species. Approximately 71% of these gene models are supported by RNA-seq data derived from leaf tissue samples. Benchmarking with Universal Single-Copy Orthologs (BUSCO) analysis of predicted gene models retrieved 93.3% of BUSCO genes. To further verify the genome annotation completeness and accuracy, we manually inspected the NLR resistance gene family and assessed its assembly quality. We find subfamilies of NLRs unique to <em>S. chilense</em>. Synteny analysis suggests significant degree of the gene order conservation between the <em>S. chilense</em>, <em>S. lycopersicum</em> and <em>S. pennellii genomes</em>.</p><p id="p-5">We generated the first genome and transcriptome sequence assemblies for the wild tomato species <em>Solanum chilense</em> and demonstrated their value in comparative genomics analyses. These data are an important resource for studies on adaptation to biotic and abiotic stress in <em>Solanaceae</em>, on evolution of self-incompatibility and for tomato breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-sequence-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-sequence-assembly" rel="nofollow">Genome sequence assembly</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">Transcriptome</a></li><li class="kwd"><a href="/keyword/evolutionary-genomics" class="hw-term hw-article-keyword hw-article-keyword-evolutionary-genomics" rel="nofollow">Evolutionary Genomics</a></li><li class="kwd"><a href="/keyword/tomato" class="hw-term hw-article-keyword hw-article-keyword-tomato" rel="nofollow">Tomato</a></li><li class="kwd"><a href="/keyword/nlr-genes" class="hw-term hw-article-keyword hw-article-keyword-nlr-genes" rel="nofollow">NLR genes</a></li></ul><p id="p-6">Tomato (<em>Solanum lycopersicum</em>) is an important vegetable crop. Together with its wild relatives it is a model sytstem for tolerance to abiotic and biotic stresses (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tomato-Genome-Consortium 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lin <em>et al.</em> 2014</a>). Tomato breeders have often used germplasm of wild relatives to enhance stress tolerance (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bolger <em>et al.</em> 2014b</a>). Genome assemblies exist for tomato relatives <em>S. habrochaites</em>, <em>S. pimpinellifolium</em> and <em>S. pennellii</em>. Yet, fully accessible and annotated reference genome sequences are to date only available for <em>S. lycopersicum</em> (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Tomato-Genome-Consortium 2012</a>) and the selfing wild tomato relative <em>S. pennellii</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bolger <em>et al.</em> 2014b</a>) Here, we present a reference genome assembly, annotation and additional <em>de novo</em> leaf transcriptome assemblies for the stress tolerant and outcrossing species, <em>S. chilense</em>.</p><p id="p-7"><em>S. chilense</em> occurs on the southern edge of the wild tomato species range, in southern Peru and northern Chile. It belongs to the section Peruvianum, which contains four closely related wild tomato species, of which <em>S. chilense</em> forms a monophyletic subclade (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pease <em>et al.</em> 2016</a>). <em>S. chilense</em> split from <em>S. peruvianum</em> about 1 mya (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arunyawat <em>et al.</em> 2007</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Städler <em>et al.</em> 2008</a>). Since then, the species has migrated southward and colonized diverse arid habitats both in mountainous and coastal terrain bordering the Atacama desert characterized by low temperatures or extreme aridity, respectively (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Böndel <em>et al.</em> 2015</a>)(<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Solanum chilense populations in their natural habitat (by R. Stam). The top panels show coastal and lowland habitats; the lower panels show typical mountain habitats. LA3111 originates from a mountainous habitat similar to the last panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-381269854" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Solanum chilense&lt;/em&gt; populations in their natural habitat (by R. Stam). The top panels show coastal and lowland habitats; the lower panels show typical mountain habitats. LA3111 originates from a mountainous habitat similar to the last panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3933/F1.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/12/3933/F1.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495050" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child"><em>Solanum chilense</em> populations in their natural habitat (by R. Stam). The top panels show coastal and lowland habitats; the lower panels show typical mountain habitats. LA3111 originates from a mountainous habitat similar to the last panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9"><em>S. chilense</em> has been used in studies on drought (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Xia <em>et al.</em> 2010</a>), salt (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhou <em>et al.</em> 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Martínez <em>et al.</em> 2012</a>) and cold tolerance (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nosenko <em>et al.</em> 2016</a>), pathogen resistance and resistance gene evolution (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stam <em>et al.</em> 2017</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2019</a>), as well as adaptation to extreme environments (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fischer <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Böndel <em>et al.</em> 2018</a>). As an outcrossing species it has been used to understand self-incompatibility in the tomato clade (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Igic <em>et al.</em> 2007</a>). The species is characterized by high levels of genetic diversity (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Arunyawat <em>et al.</em> 2007</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Städler <em>et al.</em> 2008</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Böndel <em>et al.</em> 2015</a>) probably due to existence of seed banking (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tellier <em>et al.</em> 2011</a>).</p><p id="p-10"><em>S. chilense</em> is also being used as a resource in tomato breeding; genes from <em>S. chilense</em> have been successfully introgressed to enhance resistance to the fungal pathogen <em>Verticilium dahliae</em> (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tabaeizadeh <em>et al.</em> 1999</a>) and to the Tomato Mosaic Virus Y (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Verlaan <em>et al.</em> 2013</a>).</p><p id="p-11">To corroborate the quality of our reference genome and to demonstrate its value for future genomic studies, we compared the NLR family in <em>S. chilense</em> with those in <em>S. lycopersicum</em> and <em>S. pennellii</em>. Canonical pathogen resistance genes in plants belong to the NLR family (Nod-like receptor or Nucleotide binding site, leucine rich repeat containing receptor) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jones <em>et al.</em> 2016</a>). NLRs are modular and contain an N-terminal domain, which can be a Toll-Interleukin Receptor (TIR) or a Coiled Coil (CC) domain, followed by a Nucleotide Binding Site (NBS) domain and several Leucine Rich Repeats (LRR). Complete NLRs have all three domains, whereas partial NLRs lack one or the other. Complete as well as some partial NLRs are involved in signaling of the plant immune system (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baggs <em>et al.</em> 2017</a>). TIR- and CC-domain-containing NLRs are called TNL or CNL, respectively; CNL are often grouped in subclasses.</p><p id="p-12">Comparative studies in <em>S. lycopersicum</em> and some wild relatives revealed interspecific differences in NLRs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Andolfo <em>et al.</em> 2014</a>). For example, <em>S. lycopersicum</em> and closely related <em>S. pimpinellifolium</em>, contain respectively 326 and 355 NLRs, while <em>S. pennellii</em> contains only 216 putative NLRs (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stam <em>et al.</em> 2016</a>). These differences in the NLR repertoire are thought to be the result of a birth and death process (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Michelmore and Meyers 1998</a>) and could possibly be explained by differences in pathogen pressure.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><div id="sec-2" class="subsection"><h3>De novo genome sequence assembly for S. chilense LA3111</h3><p id="p-13">Seeds were obtained through the tomato genome resource center (TGRC, UC Davis). DNA was extracted from leaves of one adult plant from accession LA3111 (plant number 3111_t13) using the Qiagen DNAeasy kit following the instructions of the supplier. The sequencing was conducted at Eurofins Genomics (Ebersberg, Germany) using standard library preparation protocols for four different libraries. Two standard paired-end libraries were produced with insert sizes of 300 bp and 500-550 bp and two libraries were prepared for mate pair sequencing, with insert sizes of 8 kb and 20 kb. The 500 bp fragment library was sequenced using a MiSeq protocol, and overlapping paired-end reads (∼55%) were stitched to longer single reads using the software PEAR v0.9.8 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhang <em>et al.</em> 2014</a>). Remaining unstitched clusters (45%) were retained as paired-end reads. The other three libraries were sequenced on Illumina HiSeq2500 sequencers at Eurofins Genomics. Construction of the mate-pair-like library was done at Eurofins Genomics using their proprietary protocol. Other libraries were constructed using commercially available kits (NEBNext Ultra DNA Library Prep Kit for Illumina, article number E7370) according to the manufacturer’s instructions. In brief, 1 µg of DNA was fragmented using a Covaris Instrument (Covaris Inc., Woburn, MA) according to manufacturer’s instructions. End-repair, A-tailing and ligation of indexed Illumina Adapter, size selection and amplification was performed accordingly. The resulting fragments were cleaned up and quantified. Libraries were loaded on the cBot (Illumina, San Diego, CA) and cluster generation was performed using manufacturer’s instructions. Sequencing of paired-end reads of 125 bp read length was performed on a HiSeq2500 machine (HiSeq Control Software 2.2.38) using HiSeq Flow Cell v4 and TruSeq SBS Kit v4. Raw data were processed using RTA v.1.18.61. CASAVA v.1.8.4 was used to generate FASTQ-files.</p><p id="p-14">In total we generated ∼134 Gb of raw data (Table S1), which would correspond to &gt; 130x coverage assuming a ∼950 Mb genome size, as was estimated by Eurofins Genomics using k-mer analysis using their proprietary protocol. In addition we used GenomeScope (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Vurture <em>et al.</em> 2017</a>) on our paired-end illumina HiSeq read data (recommended settings, k-mer size 27). We used the Celera assembler (CAv8.3; <a href="https://sourceforge.net/projects/wgs-assembler/files/wgs-assembler/wgs-8.3">https://sourceforge.net/projects/wgs-assembler/files/wgs-assembler/wgs-8.3</a>) and stitched and unassembled MiSeq read data to generate contigs. The fragment correction module and the bogart unitigger of the Celera assembler were applied with a graph and merge error rate of 5%. Minimal overlap length, overlap and merge error rates were set to 50 bp and 6% each, respectively. The resulting contigs were linked to scaffolds by SSPACE v2 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boetzer <em>et al.</em> 2011</a>) using all four libraries available for la3111_t13. Scaffolds were further processed by five iterations of GapFiller v2.1.1 and corrected by Pilon v1.21 in full-correction mode (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boetzer and Pirovano 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Walker <em>et al.</em> 2014</a>).</p><p id="p-15">Scaffolds containing exclusively organellar genes were detected using BLAST similarity searches (ncbi-blast v2.6.0+; e-value 1e-30) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) against the database containing chloroplast- and mitochondria-encoded genes of three <em>Solanaceae species</em>: <em>S. lycopersicum</em>, <em>S. pennellii</em> and <em>N. tabacum</em>.</p></div><div id="sec-3" class="subsection"><h3>De novo assembly of S. chilense leaf transcriptome</h3><p id="p-16">Twenty four Illumina paired-end read RNA-Seq libraries were generated for 12 <em>S. chilense</em> plants from populations LA3111 and LA2750. Replicates were obtained by propagating plants vegetatively. Total RNA was extracted from leaf tissue samples from multiple mature plants under normal and stress (chilling, 6h at 4°) conditions using the RNeasy Plant Mini Kit (Qiagen GmbH, Hilden, Germany) and purified from DNA using the TURBO DNA-free Kit (Ambion, Darmstadt, Germany). RNA concentration and integrity were assessed using a Bioanalyzer 2100 (Agilent Technologies, Waldbroon, Germany). The preparation of random primed cDNA libraries has been performed by GATC Biotech AG (Konstanz, Germany) according to their internal and proprietary SOP. Sequencing on a HiSeq 2500 in paired end mode with a read length of 2 × 100 bases was also conducted by GATC Biotech AG.</p><p id="p-17">RNA-Seq library contamination with non-target species and organellar RNA was assessed using FastQ Screen v0.5.2 (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/">http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/</a>) with 100000 read sub-sample and a database consisting of UniVec database and reference genome sequences of <em>Homo sapiens</em>, <em>Escherichia coli</em>, <em>Nicotiana tabacum</em> (NCBI) and <em>Fusarium graminearum</em> (<a href="ftp://ftpmips.helmholtz-muenchen.de/fungi/Fusarium/F_graminearum_PH1_v32">ftp://ftpmips.helmholtz-muenchen.de/fungi/Fusarium/F_graminearum_PH1_v32</a>). Quality of all RNA-Seq libraries were assessed using FastQC v0.11.2 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andrews 2010</a>). Adapters, reads with average quality below 30 and low quality (&lt; 30) bases at the read termini were trimmed using Trimmomatic v0.35 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bolger <em>et al.</em> 2014a</a>). Only paired reads with minimum after-processing length of 70 bp were retained and used for assembling transcriptomes. Data for each population (six individuals and 12 RNA_seq libraries per population) were assembled <em>de novo</em> using Trinity v2.3.2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grabherr <em>et al.</em> 2011</a>), SOAPdenovo-Trans v1.0.4 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Xie <em>et al.</em> 2014</a>) and Oases-Velvet (v0.2.08/v1.2.08) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schulz <em>et al.</em> 2012</a>); the redundancy acquired from pooling the three assemblies was reduced using the EvidentialGene pipeline (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gilbert 2013</a>); SOAPdenovo-Trans assembly was conducted using k-mer sizes from 29 to 79 with a step size of 10. Oases-Velvet assembler was run with k-mer sizes from 25 to 73 with a step size of 12. Trinity assembly was conducted using the k-mer size of 25, the default and the only size supported by this assembler. Oases and SOAP assemblies obtained with each k-mer size were pulled together and clustered using CD-HIT-EST v4.6.5 and identity parameter set at 100% (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Godzik 2006</a>). The contigs output by CD-HIT-EST and contigs resulting form the Trinity assembly were combined together and clustered using the EvidentialGene pipeline. Because FastQ Screen analyses identified organellar (chloroplast and mitochondrial) RNA as the major source of the RNA-Seq library contamination (up to 15%; Figure S1), contigs resulting from the assembly of organellar sequences were identified using BLAST similarity searches (ncbi-blast v2.6.0+; e-value 1e-70) against the chloroplast- and mitochondria-encoded proteins of three <em>Solanaceae species</em>: <em>S. lycopersicum</em>, <em>S. pennellii</em> and <em>N. tabacum</em>, and were excluded from the resulting <em>de novo</em> transcriptome assemblies.</p></div><div id="sec-4" class="subsection"><h3>Gene model prediction</h3><p id="p-18">We applied a previously described consensus approach (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2014</a>) to derive gene structures from the <em>S. chilense</em> draft genome. Briefly, <em>de novo</em> gene finders Augustus v3.2.3 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stanke <em>et al.</em> 2006</a>), Snap v1.0beta (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Korf 2004</a>) and GeneID v1.4.4 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Parra <em>et al.</em> 2000</a>) were trained on a set of high confidence models that were derived from the LA3111 and LA2750 transcriptome assemblies. Existing matrices for eudicots and <em>S. lycopersicum</em> were used for predictions with Fgenesh v1.6 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Salamov and Solovyev 2000</a>) and GlimmerHMM v3.0.1 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Majoros <em>et al.</em> 2004</a>), respectively. Predictions were weighted by a decision tree using the JIGSAW software v3.2.10 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen and Salzberg 2005</a>). Spliced alignments of known proteins and <em>S. chilense</em> transcripts of this study were generated by the GenomeThreader tool v1.6.6 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gremme <em>et al.</em> 2005</a>). We used current proteome releases (status of August 2016) of <em>Arabidopsis thaliana</em>, <em>Medicago truncatula</em>, <em>Ricinus communis</em>, <em>S. lycopersicum</em>, <em>Glycine max</em>, <em>Nicotiana benthiamiana</em>, <em>Cucumis sativa</em> and <em>Vitis vinifera</em>. Spliced alignments required a minimum alignment coverage of 50% and a maximum intron size of 50 kb under the <em>Arabidopsis</em> splice site model. Next, <em>de novo</em> and homology predictions were merged to top-scoring consensus models by their matches to a custom reference blastp database comprising <em>Arabidopsis</em>, <em>Medicago</em>, <em>S. lycopersicum</em> and <em>S. pennellii</em> proteins. In a last step, we annotated the top-scoring models using the AHRD (“A human readable description”)-pipeline (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2014</a>) and InterProScan v5.21 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zdobnov and Apweiler 2001</a>) to identify and remove gene models containing transposon signatures. The resulting final models were then classified into high scoring models if they showed at least 90% of alignment consistency for both query (<em>S. chilense</em>) and subject (custom plant database gene as described above), an evalue of 1e-30 or better and at least 60% of the bitscore of the blastp comparison.</p><p id="p-19">To obtain an additional RNA-seq support for the predicted gene models, raw RNA-seq data were processed (adapter and quality trimming) using Trimmomatic v0.35 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bolger <em>et al.</em> 2014a</a>) and aligned to the <em>S. chilense</em> genome sequence assembly using STAR v2.5 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dobin <em>et al.</em> 2013</a>). Read pairs aligned to exonic regions of predicted gene models were summarized per gene using featureCounts (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liao <em>et al.</em> 2014</a>). To account for differences in library sizes, the resulting count matrix was normalized using DeSeq2 (Love <em>et al.</em> 2014). Genes represented by at least ten normalized counts (<em>i.e.</em>, read pairs) in at least three samples were considered expressed.</p><p id="p-20">Functional gene annotation and assignment to the GO term categories were performed using Blast2GO v 4.1 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conesa and Götz 2008</a>) based on the results of InterProScan v5.21 (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Zdobnov and Apweiler 2001</a>) and BLAST (ncbi-blast v2.6.0+) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1997</a>) similarity searches against the NCBI non-redundant sequence database. KEGG pathway orthology assignment of protein-coding genes was conducted using KAAS (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Moriya <em>et al.</em> 2007</a>).</p><p id="p-21">The completeness of the <em>S. chilense</em> genome and transcriptome sequence assemblies and annotation was assessed using Benchmarking with Universal Single-Copy Orthologs (BUSCO v3) analysis (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Simão <em>et al.</em> 2015</a>). As an additional criterion for assessing genome assembly and annotation, we evaluated the composition and quality of the NLR gene model prediction in <em>S. chilense</em>.</p><p id="p-22">Loci encoding putative NLR genes were identified using NLRParser (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Steuernagel <em>et al.</em> 2015</a>) with cut-off thresholds as described before (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Stam <em>et al.</em> 2016</a>). We manually inspected all regions with NLR motifs and updated the annotated open reading frames where this was required. The improved annotation was based on presence of NLR motifs and expression evidence (from the RNA-seq data). For example, in some cases the automated annotations contained introns that led to exclusion of NLR domains or missed adjacent annotated NLR domains, whereas our RNA-Seq confirmed that these domains were transcribed. In these cases we either removed annotated introns or selected alternative predicted ORFs to contain the largest number of NLR domains, without introducing frame shifts. Only 15 NLR genes required manual curation based on the RNA-Seq data aligned to the reference genome. In ten instances, frame shifts made it impossible to enhance the gene model. For these genes, the computationally predicted CDS were retained. The remaining 211 predicted NLR gene models showed to be well resolved and did not require any correction. Functional clades were assigned based on protein sequences of the NBS domain. BLASTp searches (cutoff e-value 10<sup>−30</sup>) were used to match the <em>S. chilense</em> NLRs with the nearest previously assigned NLRs (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Andolfo <em>et al.</em> 2014</a>). For example, the clade of the best hit in other tomato species was assigned to the <em>S. chilense</em> gene. In the majority of cases, this gave unequivocal results. In one instance, members of our new clade matched two previously defined clades equally well; this clade thus has double naming (CNL1/CNL9). The NLRs in two identified clusters did not match any NLRs that had been clustered previously. In these cases, new cluster numbers were assigned (CNL20, CNL21).</p><p id="p-23">To assess synteny between the <em>S. chilense</em>, <em>S. lycopersicum</em> and <em>S. pennellii</em> genomes, orthologous pairs of protein-coding genes were identified using reciprocal BLAST searches with an e-value threshold of 10<sup>−30</sup> and maximum target sequence number 50. For <em>S. lycopersicum</em> and <em>S. pennellii</em>, the longest splice variant for each gene was used as a BLAST input. Duplicated BLAST matches were filtered out. Only gene pairs confirmed by bidirectional BLAST searches were retained and used as an input for synteny analyses. A spatial distribution of resulting orthologous gene pairs was analyzed and gene blocks conserved between genomes (syntenic) were identified using iADHoRe v3.0.0.1 (hybrid mode with minimum syntenic block size = 3; (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Proost <em>et al.</em> 2012</a>)). For tandem arrays of genes, a single representative was retained in syntenic blocks. To compare, we also conducted synteny analyses using DAGchainer (r02-06-2008; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Haas <em>et al.</em> 2004</a>) with the following parameters: a constant match score of value 12 (-Z 12), gap penalty equal to 1 (-g 1), minimum chain length equal five colinear genes (-A 5) and allowing neighboring genes within a single chain to be no more than ten genes apart (-D 10).</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-24">We mapped our sequence data as well as data from all nine publicly available <em>S. peruvianum</em> and presumed <em>S. chilense</em> data (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">100 Tomato Genome Sequencing Consortium 2014</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lin <em>et al.</em> 2014</a>) (SRA numbers: ERR418084, ERR418094, ERR418097, ERR418098, SRR1572692, SRR1572692, SRR1572694, SRR1572695 and SRR1572696, origins described in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) against the <em>S. pennellii</em> reference genome (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Bolger <em>et al.</em> 2014b</a>) using STAMPY v1.0.2 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lunter and Goodson 2011</a>) (substitution rate 0.01, insert size 500). The SNP calling and filtering was done using samtools v0.1.19 (mpileup, call -m with default parameters) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2009</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Maximum Likelihood (ML) phylogenetic tree constructed based on six CT loci (nuclear genes) extracted from the S. chilense LA3111 sample sequenced in this study (in bold; marked with gray rectangle) and previously sequenced S. peruvianum and S. chilense samples. For the samples from whole genome data, sequences were aligned to the S. pennellii reference genome and the sequence data for the CT loci was extracted. Single-gene alignments were concatenated; the resulting super alignment was used as in input for RaxML to construct the ML tree (1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. S. ochranthum was used as an outgroup. The sequence IDs containing chil and peru represent Sanger sequences from S. chilense and S. peruvianum individuals, respectively, followed by the accession/ individual number. The sequences with IDs containing SRR- and ERR-numbers followed by the accession number were extracted from previously generated whole genome data. B) Phylogeny constructed based on chloroplast SNP data extracted from S. chilense LA3111 (in bold and marked with gray rectangle) and previously sequenced S. peruvianum whole genome sequence data. Chloroplast sequences were aligned to the S. pennellii reference genome. The tree was built using resulting alignments and PhyML (GTR, NNI, BioNJ, 1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. aSequences of individuals with IDs containing SRR1572692, SRR1572693, SRR1572694, SRR1572695 and SRR1572696 were obtained from Lin et al. (2014). bSequences with IDs containing ERR418084, ERR418094, ERR418097 and ERR418098 originate from 100 Tomato Genome Sequencing Consortium (2014). Individual SRR1572696 was reported as S. chilense in the main text of the paper (ref) and as S. peruvianum in the supplementary, which contain all original data. The original classification of the sequences with IDs ERR418097 and ERR418098 as S. chilense has been later withdrawn from the CGN database." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-381269854" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) Maximum Likelihood (ML) phylogenetic tree constructed based on six CT loci (nuclear genes) extracted from the &lt;em&gt;S. chilense&lt;/em&gt; LA3111 sample sequenced in this study (in bold; marked with gray rectangle) and previously sequenced &lt;em&gt;S. peruvianum&lt;/em&gt; and &lt;em&gt;S. chilense&lt;/em&gt; samples. For the samples from whole genome data, sequences were aligned to the &lt;em&gt;S. pennellii&lt;/em&gt; reference genome and the sequence data for the CT loci was extracted. Single-gene alignments were concatenated; the resulting super alignment was used as in input for RaxML to construct the ML tree (1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. &lt;em&gt;S. ochranthum&lt;/em&gt; was used as an outgroup. The sequence IDs containing &lt;em&gt;chil&lt;/em&gt; and &lt;em&gt;peru&lt;/em&gt; represent Sanger sequences from &lt;em&gt;S. chilense&lt;/em&gt; and &lt;em&gt;S. peruvianum&lt;/em&gt; individuals, respectively, followed by the accession/ individual number. The sequences with IDs containing SRR- and ERR-numbers followed by the accession number were extracted from previously generated whole genome data. B) Phylogeny constructed based on chloroplast SNP data extracted from &lt;em&gt;S. chilense&lt;/em&gt; LA3111 (in bold and marked with gray rectangle) and previously sequenced &lt;em&gt;S. peruvianum&lt;/em&gt; whole genome sequence data. Chloroplast sequences were aligned to the &lt;em&gt;S. pennellii&lt;/em&gt; reference genome. The tree was built using resulting alignments and PhyML (GTR, NNI, BioNJ, 1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. &lt;sup&gt;a&lt;/sup&gt;Sequences of individuals with IDs containing SRR1572692, SRR1572693, SRR1572694, SRR1572695 and SRR1572696 were obtained from Lin &lt;em&gt;et al.&lt;/em&gt; (2014). &lt;sup&gt;b&lt;/sup&gt;Sequences with IDs containing ERR418084, ERR418094, ERR418097 and ERR418098 originate from 100 Tomato Genome Sequencing Consortium (2014). Individual SRR1572696 was reported as &lt;em&gt;S. chilense&lt;/em&gt; in the main text of the paper (ref) and as &lt;em&gt;S. peruvianum&lt;/em&gt; in the supplementary, which contain all original data. The original classification of the sequences with IDs ERR418097 and ERR418098 as &lt;em&gt;S. chilense&lt;/em&gt; has been later withdrawn from the CGN database.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3933/F2.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/12/3933/F2.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/494972" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">A) Maximum Likelihood (ML) phylogenetic tree constructed based on six CT loci (nuclear genes) extracted from the <em>S. chilense</em> LA3111 sample sequenced in this study (in bold; marked with gray rectangle) and previously sequenced <em>S. peruvianum</em> and <em>S. chilense</em> samples. For the samples from whole genome data, sequences were aligned to the <em>S. pennellii</em> reference genome and the sequence data for the CT loci was extracted. Single-gene alignments were concatenated; the resulting super alignment was used as in input for RaxML to construct the ML tree (1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. <em>S. ochranthum</em> was used as an outgroup. The sequence IDs containing <em>chil</em> and <em>peru</em> represent Sanger sequences from <em>S. chilense</em> and <em>S. peruvianum</em> individuals, respectively, followed by the accession/ individual number. The sequences with IDs containing SRR- and ERR-numbers followed by the accession number were extracted from previously generated whole genome data. B) Phylogeny constructed based on chloroplast SNP data extracted from <em>S. chilense</em> LA3111 (in bold and marked with gray rectangle) and previously sequenced <em>S. peruvianum</em> whole genome sequence data. Chloroplast sequences were aligned to the <em>S. pennellii</em> reference genome. The tree was built using resulting alignments and PhyML (GTR, NNI, BioNJ, 1000 bootstrap replicates). The branch length is shown as expected number of substitutions per site and bootstrap values are reported on each tree node. <sup>a</sup>Sequences of individuals with IDs containing SRR1572692, SRR1572693, SRR1572694, SRR1572695 and SRR1572696 were obtained from <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lin <em>et al.</em> (2014)</a>. <sup>b</sup>Sequences with IDs containing ERR418084, ERR418094, ERR418097 and ERR418098 originate from 100 Tomato Genome Sequencing Consortium (2014). Individual SRR1572696 was reported as <em>S. chilense</em> in the main text of the paper (ref) and as <em>S. peruvianum</em> in the supplementary, which contain all original data. The original classification of the sequences with IDs ERR418097 and ERR418098 as <em>S. chilense</em> has been later withdrawn from the CGN database.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We extracted two types of loci. First, we extracted the sequence at six CT loci (CT066, CT093, CT166, CT179, CT198, CT268) for all 12 accessions. These are single-copy cDNA markers developed and mapped in <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tanksley <em>et al.</em> (1992)</a>, which have previously been used to investigate the evolutionary relationships between wild tomato species (<em>e.g.</em>, <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Arunyawat <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rose <em>et al.</em> 2011</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Böndel <em>et al.</em> 2015</a>). To account for heterozygosity, two alleles were constructed randomly per individual. A concatenated alignment was prepared and manually checked. We added to this alignment 53 sequences obtained by Sanger sequencing in previous work on <em>S. chilense</em> and <em>S. peruvianum</em> (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Städler <em>et al.</em> 2008</a>). These sequences originate from <em>S. chilense</em> or <em>S. peruvianum</em> accessions as identified by the TGRC (UC Davis, USA) according to the taxonomic key in Peralta <em>et al.</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Peralta <em>et al.</em> 2008</a>). <em>S. ochranthum</em> (accession LA2682) was used as an outgroup. The phylogentic reconstruction (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>) was obtained by the Maximum Likelihood method (ML; with GTR+Gamma+I model and 1000 bootstrap replicates) as implemented in RaxML (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stamatakis 2014</a>).</p><p id="p-27">Second, we reconstructed the sequences for the coding regions of the chloroplast for each of the 12 samples listed above and our LA3111 sample. These sequences were aligned and a phylogenetic tree was constructed using PhyML (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guindon <em>et al.</em> 2010</a>) (ML, GTR, 1000 bootstraps, Best of NNI&amp;SPR, BioNJ). The resulting tree was visualized and edited using Figtree v1.4.4 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rambaut and Drummond 2009</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-28">The <em>S. chilense</em> genome data and raw RNA-Seq data generated for this study are deposited to the NCBI Short Read Archive under the BioProject IDs PRJNA508893 and PRJNA474106. All supplementary figures and tables presented in the manuscript, as well as the raw output files for BUSCO and KEGG, the NLR synteny and the .phy files used in the phylogenies are available on figshare. The <em>S. chilense</em> genome sequence assembly and annotation, CDS and protein models and <em>de novo</em> leaf transcriptome assemblies (for the accessions LA3111 and LA2750) are available via NCBI (assembly ASM601370v1) and through the Sol Genomics Network.</p><p id="p-29">(<a href="ftp://ftp.solgenomics.net/genomes/Solanum_chilense/Stam_et_al_2019/">ftp://ftp.solgenomics.net/genomes/Solanum_chilense/Stam_et_al_2019/</a>). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8223794">https://doi.org/10.25387/g3.8223794</a>.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><div id="sec-8" class="subsection"><h3>First S. chilense genome sequence assembly</h3><p id="p-30">Species within the Peruvianum group have diverged relatively recently and exhibit high intraspecific genetic and phenotypic diversity (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pease <em>et al.</em> 2016</a>). Hence, species assignment of individuals from this complex can be ambiguous (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zuriaga <em>et al.</em> 2008</a>). To confirm that our newly sequenced plant is indeed <em>S. chilense</em>, we performed phylogenetic comparisons of our sequenced individual and publicly available sequence data from <em>S. chilense</em> and <em>S. peruvianum</em>. We find that all robustly assigned <em>S. chilense</em> accessions (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Städler <em>et al.</em> 2008</a>) and our LA3111 individual cluster together into a well-supported monophyletic group (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>), while the recently sequenced accessions from <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">100 Tomato Genome Sequencing Consortium (2014)</a> and <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Lin <em>et al.</em> (2014)</a> form a polyphyletic group with known <em>S. peruvianum</em> samples.</p><p id="p-31">Additionally, we reconstructed the chloroplast phylogeny of the members of the <em>S. peruvianum</em> clade. All previously sequenced samples formed a polyphyletic group, which is a topology known for the species <em>S. peruvianum</em>, whereas our <em>S. chilense</em> sample forms a separated branch (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Thus, phylogenetic analyses of both nuclear- and plastid-encoded genes confirm that data presented in this study are the first instance of a <em>S. chilense</em> genome sequence assembly.</p></div><div id="sec-9" class="subsection"><h3>S. chilense LA3111 genome and transcriptome statistics</h3><p id="p-32">The final contig assembly comprised 150,750 contigs ranging from 1 to 162 kb totalling ∼717.7 Mb of assembled genome sequence with a N50 of 9,755 bp. After further processing, the 81,307 final scaffolds span a total size of 914 Mb with a N50 of 70.6 kb (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Of them, 111 scaffolds of total length ∼730 kb contain exclusively organellar genes and potentially represent the <em>S. chilense</em> chloroplast and mitochondrial genomes (Table S7). Our total genome size is very similar to the size calculation using k-mer analysis (∼950 Mb) and a bit larger than the estimates obtained by GenomeScope (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Vurture <em>et al.</em> 2017</a>) (721 Mb, Figure S2). The genomes for <em>S. lycopersicum</em> (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Tomato-Genome-Consortium 2012</a>) and <em>S. pennellii</em> (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Bolger <em>et al.</em> 2014b</a>) and are estimated to be ∼900 and ∼940 Mb, respectively. GenomeScope has been shown to underestimate the size for very heterozygous species like <em>S. chilense</em> (<em>e.g.</em>, for oyster, which is also highly heterozygous, flow cytometry, k-mer analyses and GenomeScope analyses esitmated 637, 545 and 471 Mb respectively (GenomeScope manual; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Vurture <em>et al.</em> 2017</a>). Therefore, we expect the genome size of <em>S. chilense</em> to be in the range of 900-950 Mb.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/494934/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/494934/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">S. chilense genome assembly</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">The resulting transcriptome assemblies contain 41,666 and 35,470 transcripts for LA3111 and LA2750, respectively (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Despite the fact that both <em>S. chilense</em> transcriptome assemblies were derived from a single tissue and only two conditions (normal and cold-stress), they are 93.3% (LA3111) and 94.2% (LA2750) complete according to the BUSCO analyses (Table S2). Our data included RNA-Seq libraries for 24 <em>S. chilense</em> individuals. High variation of gene expression within- and between populations in the wild plant species might have contributed to the relative richness of the resulting transcriptome assemblies.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495029/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495029/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">S. chilense <em>de novo</em> transcriptome assemblies</span><div class="sb-div caption-clear"></div></div></div><p id="p-34">We predicted 25,885 high-confidence (hc) gene loci that show good coverage and high homology to known proteins of other tomato species. Besides their support by homology, approximately 71% (18,290) of the hc genes are additionally supported by RNA-seq data derived from leaf tissue samples. Complementary to the set of hc models, we report the presence of 41,481 low confidence (lc) loci to maximize gene content information. Functionality for some of the lc models (6,569) is suggested by transcriptome evidence from the leaf RNA-seq data.</p></div><div id="sec-10" class="subsection"><h3>Completeness and gene model validation</h3><p id="p-35">The completeness of the assembled genome was assessed using BUSCO (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Simão <em>et al.</em> 2015</a>) and was at 91.8%. This number is lower than the scores found for the previously annotated <em>S. lycopersicum</em> and <em>S. pennellii</em> genomes (Table S3), which is an expected difference between genome assemblies generated from the highly heterozygous individuals of self-incompatible species and inbred individuals of selfing species.</p><p id="p-36">Synteny analyses using both iADHORE and DAGchainer showed conserved gene order between the genomes of the three tomato species: <em>S. chilense</em> (this study), <em>S. lycopersicum</em> (NCBI genome annotation release 102, ITAG2.4) and <em>S. pennellii</em> (NCBI genome annotation release 100, v2). We found that our <em>S</em>. <em>chilense</em> gene models (hc and lc) show homology to respectively 24,651 <em>S. lycopersicum</em> and 25,695 <em>S. pennellii</em> genes. By different estimates, from 2,364 to 3,216 (iADHoRe and DAGchainer, respectively) syntenic blocks containing total from 12,984 to 19,090 unique gene pairs were formed between the <em>S. chilense</em> and <em>S. pennellii</em> genomes (Table S4). Comparison with the <em>S. lycopersicum</em> genome output similar results: from 2,535 to 3,225 syntenic blocks containing from 14,013 to 19,114 gene pairs, respectively (Table S5). To compare, 977 syntenic gene blocks were detected between <em>S. lycopersicum</em> and <em>S. pennellii</em> genomes using the same i-ADHoRe parameters consisting of 18,064 and 17,904 gene models, respectively (Table S6). According to the results of the DAGchainer analysis, which has a higher block selection stringency relative to i-ADHoRe, syntenic blocks between <em>S. chilense</em> and both <em>S. pennellii</em> and <em>S. lycopersicum</em> contain at average 5.9 gene pairs; over 69% <em>S. chilense</em> scaffolds are represented by a single syntenic block containing at average 8.5 genes (maximum 136 genes; for both comparisons). Results of these analyses show preservation of positional association between genes among the three <em>Solanum</em> genomes. However, relatively high fragmentation of the <em>S. chilense</em> genome assembly does not allow us to assess large chromosomal re-arrangements in this species relative to <em>S. pennellii</em> and <em>S. lycopersicum</em>.</p></div><div id="sec-11" class="subsection"><h3>NLR annotation and comparison</h3><p id="p-37">In total we found 236 putative NLRs, of which 139 are CNLs and 35 TNLs. Sixty two NLRs cannot be assigned to either class. Most CDS were supported by all three measures used for the annotation. The total number of NLRs identified in <em>S. chilense the S. chilense genome</em> is lower than in cultivated tomato (355) and more similar to <em>S. pennellii</em> (216), (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Stam <em>et al.</em> 2016</a>). The syntenic blocks identified between the <em>S. chilense</em> and the <em>S. lycopersicum</em> and <em>S. pennellii</em> genomes include 69 and 50 NLR genes, respectively, and show that NLRs are distributed across all 12 chromosomes (Supplementary Material). Except for several short tandems of identical or nearly identical gene copies, NLRs do not tend to form any positional clusters in tomato genomes. Only 30% of <em>S. chilense</em> NLRs belong to syntenic gene blocks (compared to <em>S. lycopersicum</em> and <em>S. pennellii</em>) showing the fast evolution and genomic organization of this gene family at the phylogenetic time scale (over millions of years).</p><p id="p-38">To further confirm the relative completeness of the NLR set in <em>S. chilense</em>, we reconstructed a phylogeny for the gene family based on the NBS protein sequences of the NLRs. All major NLR clades found in <em>S. lycopersium</em> and <em>S. pennellii</em> are present in the <em>S. chilense</em> genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). There are some small but interesting differences with the other tomato species. The CNL-4 and CNL-15 clades contained four or five members in <em>S. lycopersicum</em>, yet in <em>S. chilense</em> each had only one member. In addition, we identified two new clades: CNL20 and CNL21. A direct comparison shows that some clades have more members in <em>S. pennellii</em>, while others have more members in <em>S. chilense</em> (Figure S3). Similar differences can be seen between <em>S. pennellii</em> and <em>S. lycopersicum</em> (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Stam <em>et al.</em> 2016</a>) Using the definitions of <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Michelmore and Meyers (1998)</a>, our data suggest the birth and death of NLR genes between tomato species.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum Likelihood (ML) phylogenetic tree of the NLR genes identified in S. chilense. The tree was made as described in Stam et al. 2016. Clades with high (&gt;80%) bootstrap values are collapsed. Most previously described clades can be identified and are indicated as such. The TNL family is highlighted in yellow. Several previously identified NLR genes from different species are included for comparison and Apaf1.1 and Ced4 are used as an outgroup, similar as in Andolfo et al. (2014) and Stam et al. (2016). A list of these genes and their species of origin can be found in Table S8. Clades marked with an asterisk are NRC-dependent. NLR with orthologs (based on reciprocal best blast hits) in S. pennellii are in bold. The branch length is shown as expected number of substitutions per site. Clades CNL20 and CNL21 are new for S. chilense." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-381269854" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum Likelihood (ML) phylogenetic tree of the NLR genes identified in &lt;em&gt;S. chilense&lt;/em&gt;. The tree was made as described in Stam &lt;em&gt;et al.&lt;/em&gt; 2016. Clades with high (&gt;80%) bootstrap values are collapsed. Most previously described clades can be identified and are indicated as such. The TNL family is highlighted in yellow. Several previously identified NLR genes from different species are included for comparison and Apaf1.1 and Ced4 are used as an outgroup, similar as in Andolfo &lt;em&gt;et al.&lt;/em&gt; (2014) and Stam &lt;em&gt;et al.&lt;/em&gt; (2016). A list of these genes and their species of origin can be found in Table S8. Clades marked with an asterisk are NRC-dependent. NLR with orthologs (based on reciprocal best blast hits) in &lt;em&gt;S. pennellii&lt;/em&gt; are in bold. The branch length is shown as expected number of substitutions per site. Clades CNL20 and CNL21 are new for &lt;em&gt;S. chilense&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/12/3933/F3.medium.gif" width="435" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/12/3933/F3.medium.gif" width="435" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/12/3933/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495073" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Maximum Likelihood (ML) phylogenetic tree of the NLR genes identified in <em>S. chilense</em>. The tree was made as described in Stam <em>et al.</em> 2016. Clades with high (&gt;80%) bootstrap values are collapsed. Most previously described clades can be identified and are indicated as such. The TNL family is highlighted in yellow. Several previously identified NLR genes from different species are included for comparison and Apaf1.1 and Ced4 are used as an outgroup, similar as in <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Andolfo <em>et al.</em> (2014)</a> and <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Stam <em>et al.</em> (2016)</a>. A list of these genes and their species of origin can be found in Table S8. Clades marked with an asterisk are NRC-dependent. NLR with orthologs (based on reciprocal best blast hits) in <em>S. pennellii</em> are in bold. The branch length is shown as expected number of substitutions per site. Clades CNL20 and CNL21 are new for <em>S. chilense</em>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-12"><h2 class="">Conclusions</h2><p id="p-40">We present the first draft genome sequence assembly and <em>de novo</em> transcriptome assemblies of the wild tomato species <em>S. chilense</em>. Using several complementary methods, including comparative analyses for a large and complex gene family such as the NLR-family, we show that quality of this genome assembly and annotation satisfy requirements for a reference genome for comparative genomics studies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">RS was supported by the Alexander von Humboldt foundation. <em>S. chilense</em> genome sequencing was funded by DFG grant TE 809/7-1 to AT. Generating and sequencing of the <em>S. chilense</em> RNA-Seq data were supported by the DFG grant STE 325/15 to WS. We thank the TGRC at UC Davis (USA) for providing the plant material.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-42">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.8223794">https://doi.org/10.25387/g3.8223794</a>.</p></li><li class="fn" id="fn-3"><p id="p-43"><em>Communicating editor: M. Hufford</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-09"><span class="received-label">Received </span>July 9, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-08"><span class="accepted-label">Accepted </span>October 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Stam <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.1" data-doi="10.1093/bioinformatics/bti609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">J. E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">JIGSAW: integration of multiple sources of evidence for gene prediction.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3596</span>–<span class="cit-lpage">3603</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti609%26rft_id%253Dinfo%253Apmid%252F16076884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16076884&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.2" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Madden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schaffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.3" data-doi="10.1186/1471-2229-14-120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfo</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jupe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Witek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Etherington</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Ercolano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Defining the full tomato NB-LRR resistance gene repertoire using genomic and cDNA RenSeq.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-14-120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-120%26rft_id%253Dinfo%253Apmid%252F24885638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24885638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite> <span class="cit-pub-date">2010</span> <span class="cit-comment">FastQC: a quality control tool for high throughput sequence data. Available at: <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.5" data-doi="10.1093/molbev/msm162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arunyawat</span>,  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Städler</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using multilocus sequence data to assess population structure, natural selection, and linkage disequilibrium in wild tomatoes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2310</span>–<span class="cit-lpage">2322</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm162%26rft_id%253Dinfo%253Apmid%252F17675653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17675653&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250437000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.6" data-doi="10.1016/j.pbi.2017.04.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baggs</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Dagdas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Krasileva</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">NLR diversity, helpers and integrated domains: making sense of the NLR IDentity.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2017.04.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D38%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2017.04.012%26rft_id%253Dinfo%253Apmid%252F28494248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2017.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28494248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.7" data-doi="10.1093/bioinformatics/btq683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. V.</span> <span class="cit-name-surname">Henkel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Jansen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Butler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pirovano</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scaffolding pre-assembled contigs using SSPACE.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">579</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq683</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq683%26rft_id%253Dinfo%253Apmid%252F21149342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21149342&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287246000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.8" data-doi="10.1186/gb-2012-13-6-r56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boetzer</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Pirovano</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toward almost closed genomes with GapFiller.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2012-13-6-r56</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBoetzer%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253DR56%26rft.epage%253DR56%26rft.atitle%253DToward%2Balmost%2Bclosed%2Bgenomes%2Bwith%2BGapFiller.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-6-r56%26rft_id%253Dinfo%253Apmid%252F22731987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-6-r56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22731987&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.9" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina Sequence Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.10" data-doi="10.1038/ng.3046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Scossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Bolger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lanz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Maumus</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">The genome of the stress-tolerant wild tomato species <em>Solanum pennellii</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1038</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3046%26rft_id%253Dinfo%253Apmid%252F25064008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25064008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.11" data-doi="10.1093/molbev/msv166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Böndel</span>,  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lainer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nosenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mboup</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tellier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">North–South colonization associated with local adaptation of the wild tomato species <em>Solanum chilense</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2932</span>–<span class="cit-lpage">2943</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv166%26rft_id%253Dinfo%253Apmid%252F26232423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26232423&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.12" data-doi="10.1098/rsos.171198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Böndel</span>,  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nosenko</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Signatures of natural selection in abiotic stress-responsive genes of <em>Solanum chilense</em>.</span> <abbr class="cit-jnl-abbrev">R. Soc. Open Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">171198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsos.171198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR.%2BSoc.%2BOpen%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.171198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsos.171198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.13" data-doi="10.1155/2008/619832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Götz</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Blast2GO: A comprehensive suite for functional analysis in plant genomics.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2008</span>: <span class="cit-elocation-id">619832</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2008/619832</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DConesa%26rft.auinit1%253DA.%26rft.volume%253D2008%26rft.spage%253D619832%26rft.epage%253D619832%26rft.atitle%253DBlast2GO%253A%2BA%2Bcomprehensive%2Bsuite%2Bfor%2Bfunctional%2Banalysis%2Bin%2Bplant%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2008%252F619832%26rft_id%253Dinfo%253Apmid%252F18483572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2008/619832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18483572&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.14" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Schlesinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drenkow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zaleski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.15" data-doi="10.1371/journal.pone.0078182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Steige</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mboup</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence evolution and expression regulation of stress-responsive genes in natural populations of wild tomato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e78182</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0078182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De78182%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0078182%26rft_id%253Dinfo%253Apmid%252F24205149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0078182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24205149&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.12.3933.16"><div class="cit-metadata unstructured">Gilbert, D., 2013 Gene-omes built from mRNA seq not genome DNA, in <em>7th annual arthropod genomics symposium. Notre Dame</em></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.17" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yassour</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Z.</span> <span class="cit-name-surname">Levin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Thompson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1883</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.18" data-doi="10.1016/j.infsof.2005.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gremme</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Brendel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sparks</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kurtz</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Engineering a software tool for gene structure prediction in higher organisms.</span> <abbr class="cit-jnl-abbrev">Inf. Softw. Technol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.infsof.2005.09.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInf.%2BSoftw.%2BTechnol.%26rft.volume%253D47%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.infsof.2005.09.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.infsof.2005.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234322400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.19" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-F.</span> <span class="cit-name-surname">Dufayard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Lefort</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Anisimova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hordijk</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/syq010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.20" data-doi="10.1093/bioinformatics/bth397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Delcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Wortman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">DAGchainer: a tool for mining segmental genome duplications and synteny.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3643</span>–<span class="cit-lpage">3646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth397%26rft_id%253Dinfo%253Apmid%252F15247098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15247098&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225786600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.21" data-doi="10.1038/sj.hdy.6801035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igic</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Robertson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Schaal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Kohn</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Studies of self-incompatibility in wild tomatoes: I. S-allele diversity in <em>Solanum chilense</em> Dun. (Solanaceae).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>. Erratum: 562.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6801035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DIgic%26rft.auinit1%253DB.%26rft.volume%253D99%26rft.issue%253D5%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DStudies%2Bof%2Bself-incompatibility%2Bin%2Bwild%2Btomatoes%253A%2BI.%2BS-allele%2Bdiversity%2Bin%2BSolanum%2Bchilense%2B%2528Dun.%2529%2BReiche%2B%255Bcorrected%255D%2B%2528Solanaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801035%26rft_id%253Dinfo%253Apmid%252F17700636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17700636&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250382000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.22" data-doi="10.1126/science.aaf6395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J. D. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Vance</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Dangl</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Intracellular innate immune surveillance devices in plants and animals.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">354</span>: <span class="cit-elocation-id">aaf6395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaf6395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BD.%2BG.%26rft.volume%253D354%26rft.issue%253D6316%26rft.spage%253Daaf6395%26rft.epage%253Daaf6395%26rft.atitle%253DIntracellular%2Binnate%2Bimmune%2Bsurveillance%2Bdevices%2Bin%2Bplants%2Band%2Banimals%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf6395%26rft_id%253Dinfo%253Apmid%252F27934708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTQvNjMxNi9hYWY2Mzk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.23" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>,  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.24" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.25" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Godzik</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.26" data-doi="10.1093/bioinformatics/btt656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. K.</span> <span class="cit-name-surname">Smyth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shi</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">930</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt656</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt656%26rft_id%253Dinfo%253Apmid%252F24227677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24227677&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.27" data-doi="10.1038/ng.3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Yu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic analyses provide insights into the history of tomato breeding.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1220</span>–<span class="cit-lpage">1226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3117%26rft_id%253Dinfo%253Apmid%252F25305757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25305757&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.12.3933.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anders</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biology 15</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.29" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goodson</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.111120.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.30" data-doi="10.1093/bioinformatics/bth315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majoros</span>,  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pertea</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2878</span>–<span class="cit-lpage">2879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bth315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth315%26rft_id%253Dinfo%253Apmid%252F15145805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15145805&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225250100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.31" data-doi="10.1017/S001447971200066X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>,  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Antúnez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pertuzé</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. P.</span> <span class="cit-name-surname">Acosta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Palma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effects of saline water on water status, yield and fruit quality of wild (<em>Solanum chilense</em>) and domesticated (<em>Solanum lycopersicum</em> var. cerasiforme) tomatoes.</span> <abbr class="cit-jnl-abbrev">Exp. Agric.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S001447971200066X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BAgric.%26rft.volume%253D48%26rft.spage%253D573%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS001447971200066X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S001447971200066X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.32" data-doi="10.1101/gr.8.11.1113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>,  <span class="cit-name-given-names">R. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Meyers</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Clusters of resistance genes in plants evolve by divergent selection and a birth-and-death process.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.8.11.1113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D1113%26rft.epage%253D1130%26rft.atitle%253DClusters%2Bof%2BResistance%2BGenes%2Bin%2BPlants%2BEvolve%2Bby%2BDivergent%2BSelection%2Band%2Ba%2BBirth-and-Death%2BProcess%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.11.1113%26rft_id%253Dinfo%253Apmid%252F9847076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjgvMTEvMTExMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzkvMTIvMzkzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.33" data-doi="10.1093/nar/gkm321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Itoh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Okuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Yoshizawa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kanehisa</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">KAAS: an automatic genome annotation and pathway reconstruction server.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W182</span>–<span class="cit-lpage">W185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkm321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm321%26rft_id%253Dinfo%253Apmid%252F17526522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17526522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255311500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.34" data-doi="10.1111/mec.13637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nosenko</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Böndel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kumpfmüller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Adaptation to low temperatures in the wild tomato species <em>Solanum chilense</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2853</span>–<span class="cit-lpage">2869</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2853%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.35" data-doi="10.1101/gr.10.4.511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Blanco</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Guigó</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">GeneID in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DParra%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DGeneID%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.511%26rft_id%253Dinfo%253Apmid%252F10779490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.36" data-doi="10.1371/journal.pbio.1002379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pease</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Haak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Moyle</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Phylogenomics reveals three sources of adaptive variation during a rapid radiation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1002379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1002379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D14%26rft.spage%253De1002379%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002379%26rft_id%253Dinfo%253Apmid%252F26871574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26871574&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peralta</span>,  <span class="cit-name-given-names">I. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Spooner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Knapp</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Taxonomy of wild tomatoes and their relatives (<em>Solanum</em> sect. Lycopersicoides, sect. Juglandifolia, sect. Lycopersicon; Solanaceae).</span> <abbr class="cit-jnl-abbrev">Syst. Bot. Monogr.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBot.%2BMonogr.%26rft.volume%253D84%26rft.spage%253D186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.38" data-doi="10.1093/nar/gkr955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proost</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fostier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">De Witte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dhoedt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Demeester</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">i-ADHoRe 3.0—fast and sensitive detection of genomic homology in extremely large data sets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr955%26rft_id%253Dinfo%253Apmid%252F22102584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22102584&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.39"><div class="cit-metadata unstructured">Rambaut, A., and A. Drummond, 2009 <em>FigTree v1. 3.1</em>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.40" data-doi="10.1111/j.1364-3703.2011.00720.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>,  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Grzeskowiak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Hörger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Groth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Targets of selection in a disease resistance network in wild tomatoes.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">927</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2011.00720.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DRose%26rft.auinit1%253DL.%2BE.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D921%26rft.epage%253D927%26rft.atitle%253DTargets%2Bof%2Bselection%2Bin%2Ba%2Bdisease%2Bresistance%2Bnetwork%2Bin%2Bwild%2Btomatoes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2011.00720.x%26rft_id%253Dinfo%253Apmid%252F21726387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2011.00720.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21726387&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.41" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>,  <span class="cit-name-given-names">A. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. V.</span> <span class="cit-name-surname">Solovyev</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in Drosophila genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.42" data-doi="10.1093/bioinformatics/bts094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>,  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Zerbino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vingron</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts094%26rft_id%253Dinfo%253Apmid%252F22368243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22368243&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302806900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.43" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.44" data-doi="10.1534/genetics.107.081810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Städler</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Arunyawat</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population genetics of speciation in two closely related wild tomatoes (<em>Solanum</em> section Lycopersicon).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStadler%26rft.auinit1%253DT.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DPopulation%2BGenetics%2Bof%2BSpeciation%2Bin%2BTwo%2BClosely%2BRelated%2BWild%2BTomatoes%2B%2528Solanum%2BSection%2BLycopersicon%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081810%26rft_id%253Dinfo%253Apmid%252F18202377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvMzM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.45" data-doi="10.1093/gbe/evw094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Scheikl</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tellier</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Pooled enrichment sequencing identifies diversity and evolutionary pressures at NLR resistance genes within a wild tomato population.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1501</span>–<span class="cit-lpage">1515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw094%26rft_id%253Dinfo%253Apmid%252F27189991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27189991&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.46" data-doi="10.7717/peerj.2910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Scheikl</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tellier</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The wild tomato species <em>Solanum chilense</em> shows variation in pathogen resistance between geographically distinct populations.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e2910</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.2910</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D5%26rft.spage%253De2910%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.2910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.2910&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.47" data-doi="10.1111/nph.16017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Silva-Arias</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tellier</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Subsets of NLR genes show differential signatures of adaptation during colonization of new habitats.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">224</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.16017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.16017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.16017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.48" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.49" data-doi="10.1093/nar/gkl200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Keller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Gunduz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Waack</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">AUGUSTUS: ab initio prediction of alternative transcripts.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W435</span>–<span class="cit-lpage">W439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW435%26rft.epage%253DW439%26rft.atitle%253DAUGUSTUS%253A%2Bab%2Binitio%2Bprediction%2Bof%2Balternative%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl200%26rft_id%253Dinfo%253Apmid%252F16845043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845043&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200087&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.50" data-doi="10.1093/bioinformatics/btv005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steuernagel</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jupe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Witek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D. G.</span> <span class="cit-name-surname">Jones</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. B. H.</span> <span class="cit-name-surname">Wulff</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">NLR-parser: rapid annotation of plant NLR complements.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv005%26rft_id%253Dinfo%253Apmid%252F25586514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25586514&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.51" data-doi="10.1007/s002990050733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabaeizadeh</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Agharbaoui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Harrak</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Poysa</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Transgenic tomato plants expressing a <em>Lycopersicon chilense</em> chitinase gene demonstrate improved resistance to Verticillium dahliae race 2.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002990050733</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D19%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002990050733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002990050733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084314200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>,  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Ganal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Prince</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Devicente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Bonierbale</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">High-density molecular linkage maps of the tomato and potato genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1160%26rft.atitle%253DHigh%2Bdensity%2Bmolecular%2Blinkage%2Bmaps%2Bof%2Bthe%2Btomato%2Band%2Bpotato%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F1360934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzExNDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy85LzEyLzM5MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.53" data-doi="10.1073/pnas.1111266108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J. Y.</span> <span class="cit-name-surname">Laurent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lainer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pavlidis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inference of seed bank parameters in two wild tomato species using ecological and genetic data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">17052</span>–<span class="cit-lpage">17057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1111266108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111266108%26rft_id%253Dinfo%253Apmid%252F21949404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQxLzE3MDUyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOS8xMi8zOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.54" data-doi="10.1111/tpj.12616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">100 Tomato Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Exploring genetic variation in the tomato (Solanum section Lycopersicon) clade by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D80%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12616%26rft_id%253Dinfo%253Apmid%252F25039268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25039268&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342721900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.55" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Tomato-Genome-Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.56" data-doi="10.1371/journal.pgen.1003399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verlaan</span>,  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Hutton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Ibrahem</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kormelink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G. F.</span> <span class="cit-name-surname">Visser</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The tomato yellow leaf curl virus resistance genes Ty-1 and Ty-3 are allelic and code for DFDGD-class RNA–dependent RNA polymerases.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003399%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003399%26rft_id%253Dinfo%253Apmid%252F23555305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.57" data-doi="10.1093/bioinformatics/btx153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vurture</span>,  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Underwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">GenomeScope: fast reference-free genome profiling from short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2202</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2202%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx153%26rft_id%253Dinfo%253Apmid%252F28369201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28369201&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.58" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Abeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Priest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Abouelliel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.59"><div class="cit-metadata unstructured">Wang, W., G. Haberer, H. Gundlach, C. Gläßer, T. Nussbaumer <em>et al.</em>, 2014 The <em>Spirodela polyrhiza</em> genome reveals insights into its neotenous reduction fast growth and aquatic lifestyle. Nat. Commun. 5: ncomms4311.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.60" data-doi="10.1111/j.1365-294X.2010.04762.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Camus-Kulandaivelu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tellier</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleotide diversity patterns of local adaptation at drought-related candidate genes in wild tomatoes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">4144</span>–<span class="cit-lpage">4154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2010.04762.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D19%26rft.spage%253D4144%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04762.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04762.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000282180500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.61" data-doi="10.1093/bioinformatics/btu077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Luo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Patterson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1660</span>–<span class="cit-lpage">1666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu077%26rft_id%253Dinfo%253Apmid%252F24532719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24532719&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338109200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.62" data-doi="10.1093/bioinformatics/17.9.847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>,  <span class="cit-name-given-names">E. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Apweiler</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">InterProScan–an integration platform for the signature-recognition methods in InterPro.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">848</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/17.9.847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DZdobnov%26rft.auinit1%253DE.%2BM.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D848%26rft.atitle%253DInterProScan%2B-%2Ban%2Bintegration%2Bplatform%2Bfor%2Bthe%2Bsignature-recognition%2Bmethods%2Bin%2BInterPro%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.9.847%26rft_id%253Dinfo%253Apmid%252F11590104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.9.847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11590104&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171566400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.63" data-doi="10.1093/bioinformatics/btt593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kobert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Flouri</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stamatakis</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PEAR: a fast and accurate Illumina Paired-End reAd mergeR.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt593</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt593%26rft_id%253Dinfo%253Apmid%252F24142950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24142950&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F12%2F3933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332259300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.64" data-doi="10.21273/JASHS.136.4.288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Sauvé</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Reddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bhatti</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of Salt-induced Changes in Leaf and Root Proteomes of the Wild Tomato, <em>Solanum chilense</em>.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.21273/JASHS.136.4.288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D136%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.21273%252FJASHS.136.4.288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21273/JASHS.136.4.288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.12.3933.65" data-doi="10.1007/s10722-008-9392-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuriaga</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Blanca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Nuez</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Classification and phylogenetic relationships in Solanum section Lycopersicon based on AFLP and two nuclear gene sequences.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10722-008-9392-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D56%26rft.spage%253D663%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-008-9392-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-008-9392-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/12/3933.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/12/3929" title="A Highly Contiguous Reference Genome for Northern Bobwhite (Colinus virginianus)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/12/3943" title="The Genome Sequence of the Eastern Woodchuck (Marmota monax) – A Preclinical Animal Model for Chronic Hepatitis B" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/12"><span>Volume 9 Issue 12, December 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/12.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/12.cover-source.jpg?itok=0XEjjYNw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (12)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/12/3933" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TW8ie7bWZjr7C5QfsTLo_KTD06DjoqO9zBY08odtLLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/12/3933" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/12/3933" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/12/3933" class="active" data-icon-position="" data-hide-link-title="0">The de Novo Reference Genome and Transcriptome Assemblies of the Wild Tomato Species Solanum chilense Highlights Birth and Death of NLR Genes Between Tomato Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/494901" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nwB0xuyGMuyQyiFz_WYV3UPIMU-c1ZJnwmbXBXmpcxo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/3933.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/12/3933" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jArhZWbzuqDBtydgRdgl6Ral-5WlngCWXKGZWKfxuzk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/12/3933" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/12/3933.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494901" id="node494901--2" data-pisa="ggg;9/12/3933" data-pisa-master="ggg;g3.119.400529" data-apath="/ggg/9/12/3933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>de Novo</em> Reference Genome and Transcriptome Assemblies of the Wild Tomato Species <em>Solanum chilense</em> Highlights Birth and Death of NLR Genes Between Tomato Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Remco Stam</span>, <span class="highwire-citation-author" data-delta="1">Tetyana Nosenko</span>, <span class="highwire-citation-author" data-delta="2">Anja C. Hörger</span>, <span class="highwire-citation-author" data-delta="3">Wolfgang Stephan</span>, <span class="highwire-citation-author" data-delta="4">Michael Seidel</span>, <span class="highwire-citation-author" data-delta="5">José M. M. Kuhn</span>, <span class="highwire-citation-author" data-delta="6">Georg Haberer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurelien Tellier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3933-3941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400529 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494901--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Remco Stam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Phytopathology, Technical University Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Remco%2BStam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stam%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARemco%2BStam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Remco Stam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stam@wzw.tum.de" class="" data-icon-position="" data-hide-link-title="0">stam@wzw.tum.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tetyana Nosenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Environmental Simulations, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetyana%2BNosenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nosenko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetyana%2BNosenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anja C. Hörger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biosciences, University of Salzburg, Austria,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anja%2BC.%2BH%C3%B6rger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=H%C3%B6rger%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnja%2BC.%2BH%25C3%25B6rger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Evolutionary Biology, LMU Munich and Natural History Museum Berlin, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José M. M. Kuhn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evolutionary Biology and Ecology, Albert-Ludwig University of Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BM.%2BKuhn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhn%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BM.%2BKuhn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Georg Haberer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BHaberer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haberer%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BHaberer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aurelien Tellier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurelien%2BTellier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tellier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAurelien%2BTellier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurelien Tellier</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/12/3933" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/12/3933&amp;text=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/12/3933&amp;t=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/12/3933&amp;title=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/494901/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494901" id="node494901--3" data-pisa="ggg;9/12/3933" data-pisa-master="ggg;g3.119.400529" data-apath="/ggg/9/12/3933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The <em>de Novo</em> Reference Genome and Transcriptome Assemblies of the Wild Tomato Species <em>Solanum chilense</em> Highlights Birth and Death of NLR Genes Between Tomato Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Remco Stam</span>, <span class="highwire-citation-author" data-delta="1">Tetyana Nosenko</span>, <span class="highwire-citation-author" data-delta="2">Anja C. Hörger</span>, <span class="highwire-citation-author" data-delta="3">Wolfgang Stephan</span>, <span class="highwire-citation-author" data-delta="4">Michael Seidel</span>, <span class="highwire-citation-author" data-delta="5">José M. M. Kuhn</span>, <span class="highwire-citation-author" data-delta="6">Georg Haberer</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aurelien Tellier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">December 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 12 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3933-3941;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400529 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node494901--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Remco Stam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Phytopathology, Technical University Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Remco%2BStam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stam%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARemco%2BStam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3444-6954" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Remco Stam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stam@wzw.tum.de" class="" data-icon-position="" data-hide-link-title="0">stam@wzw.tum.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tetyana Nosenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Environmental Simulations, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tetyana%2BNosenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nosenko%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATetyana%2BNosenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Anja C. Hörger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biosciences, University of Salzburg, Austria,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anja%2BC.%2BH%C3%B6rger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=H%C3%B6rger%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnja%2BC.%2BH%25C3%25B6rger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Wolfgang Stephan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Evolutionary Biology, LMU Munich and Natural History Museum Berlin, Germany, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wolfgang%2BStephan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWolfgang%2BStephan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael Seidel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeidel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seidel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeidel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José M. M. Kuhn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Evolutionary Biology and Ecology, Albert-Ludwig University of Freiburg, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BM.%2BKuhn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhn%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BM.%2BKuhn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Georg Haberer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Plant Genome and Systems Biology, Helmholtz Center of Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Georg%2BHaberer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haberer%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorg%2BHaberer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aurelien Tellier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Population Genetics, Technical University Munich, Germany,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurelien%2BTellier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tellier%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAurelien%2BTellier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8895-0785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aurelien Tellier</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3933&amp;rft.epage=3941&amp;rft.atitle=The%20de%20Novo%20Reference%20Genome%20and%20Transcriptome%20Assemblies%20of%20the%20Wild%20Tomato%20Species%20Solanum%20chilense%20Highlights%20Birth%20and%20Death%20of%20NLR%20Genes%20Between%20Tomato%20Species&amp;rft.volume=9&amp;rft.issue=12&amp;rft.date=2019-12-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Stam%2C+Remco&amp;rft.au=Nosenko%2C+Tetyana&amp;rft.au=H%C3%B6rger%2C+Anja+C.&amp;rft.au=Stephan%2C+Wolfgang&amp;rft.au=Seidel%2C+Michael&amp;rft.au=Kuhn%2C+Jos%C3%A9+M.+M.&amp;rft.au=Haberer%2C+Georg&amp;rft.au=Tellier%2C+Aurelien"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/494901/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/494901/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/494901/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/494901/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/494901/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/494901/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/494901/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/494901/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/494901/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/494901/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/494901/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/494901/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/12/3933" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/12/3933.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/12/3933.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/12/3933.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
